{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "810d9f10-ec19-4b09-8f90-e983e460b319",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true
   },
   "source": [
    "<center>\n",
    "<img src=\"https://laelgelcpublic.s3.sa-east-1.amazonaws.com/lael_50_years_narrow_white.png.no_years.400px_96dpi.png\" width=\"300\" alt=\"LAEL 50 years logo\">\n",
    "<h3>APPLIED LINGUISTICS GRADUATE PROGRAMME (LAEL)</h3>\n",
    "</center>\n",
    "<hr>"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "888c2c96-2fc3-4a1a-995b-c388036a2a15",
   "metadata": {},
   "source": [
    "# Corpus Linguistics - Study 2 - Phase 1_1 - eyamrog"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c2b2a221-54af-4c1c-a7fd-c80cda278dcc",
   "metadata": {},
   "source": [
    "The aim of this phase is to check."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a685d8b0-7715-45a6-9489-2d3db9b346c8",
   "metadata": {},
   "source": [
    "## Required Python packages"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b526a82b-22a6-4d28-afa2-eb6e7bcca4fb",
   "metadata": {},
   "source": [
    "- beautifulsoup4\n",
    "- lxml\n",
    "- pandas\n",
    "- requests\n",
    "- selenium\n",
    "- tqdm"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "aa922755-c4d6-4008-9aad-d35e33b18ed7",
   "metadata": {},
   "source": [
    "## Importing the required libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "182dbbab-65ee-4695-a418-3e9d9d8599e8",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import re\n",
    "import pandas as pd\n",
    "import os\n",
    "import sys\n",
    "from tqdm import tqdm\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.edge.service import Service\n",
    "from selenium.webdriver.common.by import By\n",
    "from selenium.webdriver.support.ui import WebDriverWait\n",
    "from selenium.webdriver.support import expected_conditions as EC"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef3ad412-2346-43d3-8607-08705487f1b2",
   "metadata": {},
   "source": [
    "## Defining input variables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a7657874-dc0a-4a7b-ae4b-bb3ca3fd7fa0",
   "metadata": {},
   "outputs": [],
   "source": [
    "input_directory = 'cl_st2_ph1_eyamrog'\n",
    "output_directory = 'cl_st2_ph11_eyamrog'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d32b4a24-8a37-4302-9c3e-790aa6dee914",
   "metadata": {},
   "source": [
    "## Creating output directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "5dce68e3-44e7-4a7e-bc25-73c21ca6fbcb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output directory successfully created.\n"
     ]
    }
   ],
   "source": [
    "# Check if the output directory already exists. If it does, do nothing. If it doesn't exist, create it.\n",
    "if os.path.exists(output_directory):\n",
    "    print('Output directory already exists.')\n",
    "else:\n",
    "    try:\n",
    "        os.makedirs(output_directory)\n",
    "        print('Output directory successfully created.')\n",
    "    except OSError as e:\n",
    "        print('Failed to create the directory:', e)\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6b8c5274-7933-44ef-a148-d1aece85cf52",
   "metadata": {},
   "source": [
    "## Web Scraping [Annual Review of Plant Biology](https://www.annualreviews.org/content/journals/arplant)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "37b0cb13-bf9e-4bf0-a137-6998ecf03801",
   "metadata": {},
   "source": [
    "### Defining input variables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "762ad942-72fa-46c5-9221-91b506c16347",
   "metadata": {},
   "outputs": [],
   "source": [
    "ar_plant_biology_prefix = 'ar_plant_biology'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "572549d6-3221-4fb7-916b-b3e78c2851f4",
   "metadata": {},
   "source": [
    "### Importing the data into a DataFrame"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "bb7c9f4b-6de3-48ae-a018-646361dbb8a2",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_ar_plant_biology = pd.read_json(f\"{input_directory}/{ar_plant_biology_prefix}.jsonl\", lines=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "9fe7040b-c321-48f8-b9c3-7cd5d0494f65",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Title', 'URL', 'Authors', 'Vol/Year/Page Range', 'DOI',\n",
       "       'Area of Knowledge'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_ar_plant_biology.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "56691dbb-853d-4920-b6dd-52b154f60278",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Vol. 73 \\n(2022),pp.1–16', 'Vol. 73 \\n(2022),pp.17–42',\n",
       "       'Vol. 73 \\n(2022),pp.43–65', 'Vol. 73 \\n(2022),pp.67–92',\n",
       "       'Vol. 73 \\n(2022),pp.93–121', 'Vol. 73 \\n(2022),pp.123–148',\n",
       "       'Vol. 73 \\n(2022),pp.149–172', 'Vol. 73 \\n(2022),pp.173–200',\n",
       "       'Vol. 73 \\n(2022),pp.201–225', 'Vol. 73 \\n(2022),pp.227–254',\n",
       "       'Vol. 73 \\n(2022),pp.255–291', 'Vol. 73 \\n(2022),pp.293–321',\n",
       "       'Vol. 73 \\n(2022),pp.323–353', 'Vol. 73 \\n(2022),pp.355–378',\n",
       "       'Vol. 73 \\n(2022),pp.379–403', 'Vol. 73 \\n(2022),pp.405–432',\n",
       "       'Vol. 73 \\n(2022),pp.433–455', 'Vol. 73 \\n(2022),pp.457–474',\n",
       "       'Vol. 73 \\n(2022),pp.475–494', 'Vol. 73 \\n(2022),pp.495–521',\n",
       "       'Vol. 73 \\n(2022),pp.523–551', 'Vol. 73 \\n(2022),pp.553–584',\n",
       "       'Vol. 73 \\n(2022),pp.585–616', 'Vol. 73 \\n(2022),pp.617–648',\n",
       "       'Vol. 73 \\n(2022),pp.649–672', 'Vol. 73 \\n(2022),pp.673–702',\n",
       "       'Vol. 73 \\n(2022),pp.703–728', 'Vol. 72 \\n(2021),pp.1–16',\n",
       "       'Vol. 72 \\n(2021),pp.17–46', 'Vol. 72 \\n(2021),pp.47–76',\n",
       "       'Vol. 72 \\n(2021),pp.77–103', 'Vol. 72 \\n(2021),pp.105–131',\n",
       "       'Vol. 72 \\n(2021),pp.133–153', 'Vol. 72 \\n(2021),pp.155–184',\n",
       "       'Vol. 72 \\n(2021),pp.185–216', 'Vol. 72 \\n(2021),pp.217–244',\n",
       "       'Vol. 72 \\n(2021),pp.245–271', 'Vol. 72 \\n(2021),pp.273–296',\n",
       "       'Vol. 72 \\n(2021),pp.297–323', 'Vol. 72 \\n(2021),pp.325–356',\n",
       "       'Vol. 72 \\n(2021),pp.357–385', 'Vol. 72 \\n(2021),pp.387–410',\n",
       "       'Vol. 72 \\n(2021),pp.411–435', 'Vol. 72 \\n(2021),pp.437–462',\n",
       "       'Vol. 72 \\n(2021),pp.463–495', 'Vol. 72 \\n(2021),pp.497–524',\n",
       "       'Vol. 72 \\n(2021),pp.525–550', 'Vol. 72 \\n(2021),pp.551–580',\n",
       "       'Vol. 72 \\n(2021),pp.581–614', 'Vol. 72 \\n(2021),pp.615–639',\n",
       "       'Vol. 72 \\n(2021),pp.641–676', 'Vol. 72 \\n(2021),pp.677–702',\n",
       "       'Vol. 72 \\n(2021),pp.703–736', 'Vol. 72 \\n(2021),pp.737–760',\n",
       "       'Vol. 72 \\n(2021),pp.761–791', 'Vol. 72 \\n(2021),pp.793–822',\n",
       "       'Vol. 72 \\n(2021),pp.823–846', 'Vol. 72 \\n(2021),pp.847–866',\n",
       "       'Vol. 72 \\n(2021),pp.867–891', 'Vol. 71 \\n(2020),pp.1–38',\n",
       "       'Vol. 71 \\n(2020),pp.39–69', 'Vol. 71 \\n(2020),pp.71–102',\n",
       "       'Vol. 71 \\n(2020),pp.103–129', 'Vol. 71 \\n(2020),pp.131–156',\n",
       "       'Vol. 71 \\n(2020),pp.157–182', 'Vol. 71 \\n(2020),pp.183–215',\n",
       "       'Vol. 71 \\n(2020),pp.217–245', 'Vol. 71 \\n(2020),pp.247–272',\n",
       "       'Vol. 71 \\n(2020),pp.273–302', 'Vol. 71 \\n(2020),pp.303–326',\n",
       "       'Vol. 71 \\n(2020),pp.327–353', 'Vol. 71 \\n(2020),pp.355–378',\n",
       "       'Vol. 71 \\n(2020),pp.379–402', 'Vol. 71 \\n(2020),pp.403–433',\n",
       "       'Vol. 71 \\n(2020),pp.435–460', 'Vol. 71 \\n(2020),pp.461–485',\n",
       "       'Vol. 71 \\n(2020),pp.487–515', 'Vol. 71 \\n(2020),pp.517–546',\n",
       "       'Vol. 71 \\n(2020),pp.547–573', 'Vol. 71 \\n(2020),pp.575–603',\n",
       "       'Vol. 71 \\n(2020),pp.605–629', 'Vol. 71 \\n(2020),pp.631–658',\n",
       "       'Vol. 71 \\n(2020),pp.659–687', 'Vol. 71 \\n(2020),pp.689–712',\n",
       "       'Vol. 71 \\n(2020),pp.713–739', 'Vol. 71 \\n(2020),pp.741–765',\n",
       "       'Vol. 71 \\n(2020),pp.767–788', 'Vol. 71 \\n(2020),pp.789–816'],\n",
       "      dtype=object)"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_ar_plant_biology['Vol/Year/Page Range'].unique()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6b7867ee-091b-4556-adab-1a28fa94edf8",
   "metadata": {},
   "source": [
    "### Extracting the `Posted` dates from the column `Vol/Year/Page Range`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "20fb8cda-a59d-4110-94f8-69cc8db0943a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Extract year using RegEx\n",
    "df_ar_plant_biology['Posted'] = df_ar_plant_biology['Vol/Year/Page Range'].str.extract(r'^Vol. .+ \\n\\((\\d{4})\\).+')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "f5d59024-5240-42a8-a3e2-5b741d78d505",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_ar_plant_biology['Posted'] = pd.to_datetime(df_ar_plant_biology['Posted'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "d2414715-7e2d-42d5-b920-3ece81771bd3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Title                          object\n",
       "URL                            object\n",
       "Authors                        object\n",
       "Vol/Year/Page Range            object\n",
       "DOI                            object\n",
       "Area of Knowledge              object\n",
       "Posted                 datetime64[ns]\n",
       "dtype: object"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_ar_plant_biology.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "c324b187-92fa-4101-bdd9-b99170fb75b5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Title</th>\n",
       "      <th>URL</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Vol/Year/Page Range</th>\n",
       "      <th>DOI</th>\n",
       "      <th>Area of Knowledge</th>\n",
       "      <th>Posted</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Adventures in Life and Science, from Light to ...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Elaine Tobin</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.1–16</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-090921...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Phosphorus Acquisition and Utilization in Plants</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Hans Lambers</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.17–42</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102720...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Meeting in the Middle: Lessons and Opportuniti...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Mae Antonette Mercado, , andAnthony J. Studer</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.43–65</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102720...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Plant Proteome Dynamics</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Julia Mergner, , andBernhard Kuster</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.67–92</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102620...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Evolution and Functions of Plant U-Box Protein...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Jana Trenner, ,Jacqueline Monaghan, ,Bushra Sa...</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.93–121</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102720...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>Phenotyping: New Windows into the Plant for Br...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Michelle Watt, ,Fabio Fiorani, ,Björn Usadel, ...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.689–712</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-042916...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>The Genomics ofCannabisand Its Close Relatives</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>I. Kovalchuk, ,M. Pellino, ,P. Rigault, ,R. va...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.713–739</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-081519...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>Sequencing and Analyzing the Transcriptomes of...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Gane Ka-Shu Wong, ,Douglas E. Soltis, ,Jim Lee...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.741–765</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-042916...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>Engineering Synthetic Signaling in Plants</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Alexander R. Leydon, ,Hardik P. Gala, ,Sarah G...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.767–788</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-081519...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>Novel Imaging Modalities Shedding Light on Pla...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Natalie M. Clark, ,Lisa Van den Broeck, ,Marjo...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.789–816</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-050718...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>88 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                Title  \\\n",
       "0   Adventures in Life and Science, from Light to ...   \n",
       "1    Phosphorus Acquisition and Utilization in Plants   \n",
       "2   Meeting in the Middle: Lessons and Opportuniti...   \n",
       "3                             Plant Proteome Dynamics   \n",
       "4   Evolution and Functions of Plant U-Box Protein...   \n",
       "..                                                ...   \n",
       "83  Phenotyping: New Windows into the Plant for Br...   \n",
       "84     The Genomics ofCannabisand Its Close Relatives   \n",
       "85  Sequencing and Analyzing the Transcriptomes of...   \n",
       "86          Engineering Synthetic Signaling in Plants   \n",
       "87  Novel Imaging Modalities Shedding Light on Pla...   \n",
       "\n",
       "                                                  URL  \\\n",
       "0   https://www.annualreviews.org/content/journals...   \n",
       "1   https://www.annualreviews.org/content/journals...   \n",
       "2   https://www.annualreviews.org/content/journals...   \n",
       "3   https://www.annualreviews.org/content/journals...   \n",
       "4   https://www.annualreviews.org/content/journals...   \n",
       "..                                                ...   \n",
       "83  https://www.annualreviews.org/content/journals...   \n",
       "84  https://www.annualreviews.org/content/journals...   \n",
       "85  https://www.annualreviews.org/content/journals...   \n",
       "86  https://www.annualreviews.org/content/journals...   \n",
       "87  https://www.annualreviews.org/content/journals...   \n",
       "\n",
       "                                              Authors  \\\n",
       "0                                        Elaine Tobin   \n",
       "1                                        Hans Lambers   \n",
       "2       Mae Antonette Mercado, , andAnthony J. Studer   \n",
       "3                 Julia Mergner, , andBernhard Kuster   \n",
       "4   Jana Trenner, ,Jacqueline Monaghan, ,Bushra Sa...   \n",
       "..                                                ...   \n",
       "83  Michelle Watt, ,Fabio Fiorani, ,Björn Usadel, ...   \n",
       "84  I. Kovalchuk, ,M. Pellino, ,P. Rigault, ,R. va...   \n",
       "85  Gane Ka-Shu Wong, ,Douglas E. Soltis, ,Jim Lee...   \n",
       "86  Alexander R. Leydon, ,Hardik P. Gala, ,Sarah G...   \n",
       "87  Natalie M. Clark, ,Lisa Van den Broeck, ,Marjo...   \n",
       "\n",
       "            Vol/Year/Page Range  \\\n",
       "0      Vol. 73 \\n(2022),pp.1–16   \n",
       "1     Vol. 73 \\n(2022),pp.17–42   \n",
       "2     Vol. 73 \\n(2022),pp.43–65   \n",
       "3     Vol. 73 \\n(2022),pp.67–92   \n",
       "4    Vol. 73 \\n(2022),pp.93–121   \n",
       "..                          ...   \n",
       "83  Vol. 71 \\n(2020),pp.689–712   \n",
       "84  Vol. 71 \\n(2020),pp.713–739   \n",
       "85  Vol. 71 \\n(2020),pp.741–765   \n",
       "86  Vol. 71 \\n(2020),pp.767–788   \n",
       "87  Vol. 71 \\n(2020),pp.789–816   \n",
       "\n",
       "                                                  DOI      Area of Knowledge  \\\n",
       "0   https://doi.org/10.1146/annurev-arplant-090921...  Agricultural Sciences   \n",
       "1   https://doi.org/10.1146/annurev-arplant-102720...  Agricultural Sciences   \n",
       "2   https://doi.org/10.1146/annurev-arplant-102720...  Agricultural Sciences   \n",
       "3   https://doi.org/10.1146/annurev-arplant-102620...  Agricultural Sciences   \n",
       "4   https://doi.org/10.1146/annurev-arplant-102720...  Agricultural Sciences   \n",
       "..                                                ...                    ...   \n",
       "83  https://doi.org/10.1146/annurev-arplant-042916...  Agricultural Sciences   \n",
       "84  https://doi.org/10.1146/annurev-arplant-081519...  Agricultural Sciences   \n",
       "85  https://doi.org/10.1146/annurev-arplant-042916...  Agricultural Sciences   \n",
       "86  https://doi.org/10.1146/annurev-arplant-081519...  Agricultural Sciences   \n",
       "87  https://doi.org/10.1146/annurev-arplant-050718...  Agricultural Sciences   \n",
       "\n",
       "       Posted  \n",
       "0  2022-01-01  \n",
       "1  2022-01-01  \n",
       "2  2022-01-01  \n",
       "3  2022-01-01  \n",
       "4  2022-01-01  \n",
       "..        ...  \n",
       "83 2020-01-01  \n",
       "84 2020-01-01  \n",
       "85 2020-01-01  \n",
       "86 2020-01-01  \n",
       "87 2020-01-01  \n",
       "\n",
       "[88 rows x 7 columns]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_ar_plant_biology"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3a95cebe-f937-4504-ba67-ab277899eedc",
   "metadata": {},
   "source": [
    "### Creating the column `Text ID`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "01ea7c99-6083-4f86-a99f-cbafbd34b233",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_ar_plant_biology['Text ID'] = ar_plant_biology_prefix + df_ar_plant_biology.index.astype(str).str.zfill(6)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d470693b-18f1-45f1-89fa-dc36f23b5f83",
   "metadata": {},
   "source": [
    "### Inspecting a few samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "a788abb6-2ada-4ba1-88b9-b758f89cec43",
   "metadata": {},
   "outputs": [],
   "source": [
    "url_sample = df_ar_plant_biology.at[87, 'URL']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "022ac30f-ee8d-44e5-b7ee-ebc929208569",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.annualreviews.org/content/journals/10.1146/annurev-arplant-050718-100038'"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url_sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "1831d68c-851a-4da7-86d7-e7cb6605f7c0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Setting up the WebDriver (make sure you have downloaded the Microsoft Edge WebDriver executable)\n",
    "# https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/\n",
    "service = Service(r'C:\\Users\\eyamr\\OneDrive\\Documentos\\0-Technology\\laelgelc\\edgedriver_win64\\msedgedriver.exe')\n",
    "driver = webdriver.Edge(service=service)\n",
    "\n",
    "# Navigating to target URL 1 and saving its web page\n",
    "driver.get(url_sample)\n",
    "wait = WebDriverWait(driver, 10)\n",
    "wait.until(EC.presence_of_element_located((By.ID, 'google_esf')))\n",
    "document_page_sample = driver.page_source\n",
    "\n",
    "with open(f\"{output_directory}/{ar_plant_biology_prefix}_sample.html\", 'w', encoding='utf8', newline='\\n') as file:\n",
    "    file.write(document_page_sample)\n",
    "\n",
    "# Closing the WebDriver\n",
    "driver.quit()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e00ba1fa-10bb-46c8-8288-e3de983a6dee",
   "metadata": {},
   "source": [
    "### Scraping the paragraphs of the articles into TXT format"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "9bc8fef5-fffb-4a21-9562-2d7d7839bb21",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Iterating over the rows of the column 'URL' to scrape paragraphs from each article\n",
    "for index, row in df_ar_plant_biology.iterrows():\n",
    "    url = row['URL']\n",
    "    text_id = row['Text ID']\n",
    "    \n",
    "    # Setting up the WebDriver\n",
    "    service = Service(r'C:\\Users\\eyamr\\OneDrive\\Documentos\\0-Technology\\laelgelc\\edgedriver_win64\\msedgedriver.exe')\n",
    "    driver = webdriver.Edge(service=service)\n",
    "    driver.get(url)\n",
    "    \n",
    "    # Wait for page to load\n",
    "    wait = WebDriverWait(driver, 10)\n",
    "    wait.until(EC.presence_of_element_located((By.ID, 'google_esf')))\n",
    "    \n",
    "    page = driver.page_source\n",
    "    soup = BeautifulSoup(page, 'lxml')\n",
    "    paragraphs = soup.find_all('p')\n",
    "    \n",
    "    # Extract text from paragraphs\n",
    "    article_content = '\\n'.join(p.get_text(strip=True) for p in paragraphs)\n",
    "    \n",
    "    # Saving each article's content to a text file\n",
    "    with open(f\"{output_directory}/{text_id}.txt\", 'w', encoding='utf-8') as file:\n",
    "        file.write(article_content)\n",
    "        \n",
    "    # Closing the WebDriver\n",
    "    driver.quit()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a140e522-a944-4f8d-b60c-625de224a0b2",
   "metadata": {},
   "source": [
    "### Adding the column `Text` with the text extracted from each article"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "1aba86f9-2fd3-420a-b61c-2e14927aca93",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function to read the content of a TXT file\n",
    "def read_txt_file(file_path):\n",
    "    with open(file_path, 'r', encoding='utf-8') as file:\n",
    "        return file.read()\n",
    "\n",
    "# Iterating through each row in the DataFrame and add the text content\n",
    "texts = []\n",
    "for index, row in df_ar_plant_biology.iterrows():\n",
    "    text_id = row['Text ID']\n",
    "    txt_file_path = os.path.join(output_directory, f\"{text_id}.txt\")\n",
    "    if os.path.exists(txt_file_path):\n",
    "        text_content = read_txt_file(txt_file_path)\n",
    "    else:\n",
    "        text_content = None  # or you can set it to an empty string or any default value\n",
    "    texts.append(text_content)\n",
    "\n",
    "# Add the 'Text' column to the DataFrame\n",
    "df_ar_plant_biology['Text'] = texts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "58dd5800-7afc-4b38-9f17-247b980e6734",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Title</th>\n",
       "      <th>URL</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Vol/Year/Page Range</th>\n",
       "      <th>DOI</th>\n",
       "      <th>Area of Knowledge</th>\n",
       "      <th>Posted</th>\n",
       "      <th>Text ID</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Adventures in Life and Science, from Light to ...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Elaine Tobin</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.1–16</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-090921...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "      <td>ar_plant_biology000000</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Phosphorus Acquisition and Utilization in Plants</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Hans Lambers</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.17–42</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102720...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "      <td>ar_plant_biology000001</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Meeting in the Middle: Lessons and Opportuniti...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Mae Antonette Mercado, , andAnthony J. Studer</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.43–65</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102720...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "      <td>ar_plant_biology000002</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Plant Proteome Dynamics</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Julia Mergner, , andBernhard Kuster</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.67–92</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102620...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "      <td>ar_plant_biology000003</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Evolution and Functions of Plant U-Box Protein...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Jana Trenner, ,Jacqueline Monaghan, ,Bushra Sa...</td>\n",
       "      <td>Vol. 73 \\n(2022),pp.93–121</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-102720...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2022-01-01</td>\n",
       "      <td>ar_plant_biology000004</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>Phenotyping: New Windows into the Plant for Br...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Michelle Watt, ,Fabio Fiorani, ,Björn Usadel, ...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.689–712</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-042916...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "      <td>ar_plant_biology000083</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>The Genomics ofCannabisand Its Close Relatives</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>I. Kovalchuk, ,M. Pellino, ,P. Rigault, ,R. va...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.713–739</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-081519...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "      <td>ar_plant_biology000084</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>Sequencing and Analyzing the Transcriptomes of...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Gane Ka-Shu Wong, ,Douglas E. Soltis, ,Jim Lee...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.741–765</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-042916...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "      <td>ar_plant_biology000085</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>Engineering Synthetic Signaling in Plants</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Alexander R. Leydon, ,Hardik P. Gala, ,Sarah G...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.767–788</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-081519...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "      <td>ar_plant_biology000086</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>Novel Imaging Modalities Shedding Light on Pla...</td>\n",
       "      <td>https://www.annualreviews.org/content/journals...</td>\n",
       "      <td>Natalie M. Clark, ,Lisa Van den Broeck, ,Marjo...</td>\n",
       "      <td>Vol. 71 \\n(2020),pp.789–816</td>\n",
       "      <td>https://doi.org/10.1146/annurev-arplant-050718...</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>2020-01-01</td>\n",
       "      <td>ar_plant_biology000087</td>\n",
       "      <td>We usecookiesto track usage and preferences.I ...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>88 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                Title  \\\n",
       "0   Adventures in Life and Science, from Light to ...   \n",
       "1    Phosphorus Acquisition and Utilization in Plants   \n",
       "2   Meeting in the Middle: Lessons and Opportuniti...   \n",
       "3                             Plant Proteome Dynamics   \n",
       "4   Evolution and Functions of Plant U-Box Protein...   \n",
       "..                                                ...   \n",
       "83  Phenotyping: New Windows into the Plant for Br...   \n",
       "84     The Genomics ofCannabisand Its Close Relatives   \n",
       "85  Sequencing and Analyzing the Transcriptomes of...   \n",
       "86          Engineering Synthetic Signaling in Plants   \n",
       "87  Novel Imaging Modalities Shedding Light on Pla...   \n",
       "\n",
       "                                                  URL  \\\n",
       "0   https://www.annualreviews.org/content/journals...   \n",
       "1   https://www.annualreviews.org/content/journals...   \n",
       "2   https://www.annualreviews.org/content/journals...   \n",
       "3   https://www.annualreviews.org/content/journals...   \n",
       "4   https://www.annualreviews.org/content/journals...   \n",
       "..                                                ...   \n",
       "83  https://www.annualreviews.org/content/journals...   \n",
       "84  https://www.annualreviews.org/content/journals...   \n",
       "85  https://www.annualreviews.org/content/journals...   \n",
       "86  https://www.annualreviews.org/content/journals...   \n",
       "87  https://www.annualreviews.org/content/journals...   \n",
       "\n",
       "                                              Authors  \\\n",
       "0                                        Elaine Tobin   \n",
       "1                                        Hans Lambers   \n",
       "2       Mae Antonette Mercado, , andAnthony J. Studer   \n",
       "3                 Julia Mergner, , andBernhard Kuster   \n",
       "4   Jana Trenner, ,Jacqueline Monaghan, ,Bushra Sa...   \n",
       "..                                                ...   \n",
       "83  Michelle Watt, ,Fabio Fiorani, ,Björn Usadel, ...   \n",
       "84  I. Kovalchuk, ,M. Pellino, ,P. Rigault, ,R. va...   \n",
       "85  Gane Ka-Shu Wong, ,Douglas E. Soltis, ,Jim Lee...   \n",
       "86  Alexander R. Leydon, ,Hardik P. Gala, ,Sarah G...   \n",
       "87  Natalie M. Clark, ,Lisa Van den Broeck, ,Marjo...   \n",
       "\n",
       "            Vol/Year/Page Range  \\\n",
       "0      Vol. 73 \\n(2022),pp.1–16   \n",
       "1     Vol. 73 \\n(2022),pp.17–42   \n",
       "2     Vol. 73 \\n(2022),pp.43–65   \n",
       "3     Vol. 73 \\n(2022),pp.67–92   \n",
       "4    Vol. 73 \\n(2022),pp.93–121   \n",
       "..                          ...   \n",
       "83  Vol. 71 \\n(2020),pp.689–712   \n",
       "84  Vol. 71 \\n(2020),pp.713–739   \n",
       "85  Vol. 71 \\n(2020),pp.741–765   \n",
       "86  Vol. 71 \\n(2020),pp.767–788   \n",
       "87  Vol. 71 \\n(2020),pp.789–816   \n",
       "\n",
       "                                                  DOI      Area of Knowledge  \\\n",
       "0   https://doi.org/10.1146/annurev-arplant-090921...  Agricultural Sciences   \n",
       "1   https://doi.org/10.1146/annurev-arplant-102720...  Agricultural Sciences   \n",
       "2   https://doi.org/10.1146/annurev-arplant-102720...  Agricultural Sciences   \n",
       "3   https://doi.org/10.1146/annurev-arplant-102620...  Agricultural Sciences   \n",
       "4   https://doi.org/10.1146/annurev-arplant-102720...  Agricultural Sciences   \n",
       "..                                                ...                    ...   \n",
       "83  https://doi.org/10.1146/annurev-arplant-042916...  Agricultural Sciences   \n",
       "84  https://doi.org/10.1146/annurev-arplant-081519...  Agricultural Sciences   \n",
       "85  https://doi.org/10.1146/annurev-arplant-042916...  Agricultural Sciences   \n",
       "86  https://doi.org/10.1146/annurev-arplant-081519...  Agricultural Sciences   \n",
       "87  https://doi.org/10.1146/annurev-arplant-050718...  Agricultural Sciences   \n",
       "\n",
       "       Posted                 Text ID  \\\n",
       "0  2022-01-01  ar_plant_biology000000   \n",
       "1  2022-01-01  ar_plant_biology000001   \n",
       "2  2022-01-01  ar_plant_biology000002   \n",
       "3  2022-01-01  ar_plant_biology000003   \n",
       "4  2022-01-01  ar_plant_biology000004   \n",
       "..        ...                     ...   \n",
       "83 2020-01-01  ar_plant_biology000083   \n",
       "84 2020-01-01  ar_plant_biology000084   \n",
       "85 2020-01-01  ar_plant_biology000085   \n",
       "86 2020-01-01  ar_plant_biology000086   \n",
       "87 2020-01-01  ar_plant_biology000087   \n",
       "\n",
       "                                                 Text  \n",
       "0   We usecookiesto track usage and preferences.I ...  \n",
       "1   We usecookiesto track usage and preferences.I ...  \n",
       "2   We usecookiesto track usage and preferences.I ...  \n",
       "3   We usecookiesto track usage and preferences.I ...  \n",
       "4   We usecookiesto track usage and preferences.I ...  \n",
       "..                                                ...  \n",
       "83  We usecookiesto track usage and preferences.I ...  \n",
       "84  We usecookiesto track usage and preferences.I ...  \n",
       "85  We usecookiesto track usage and preferences.I ...  \n",
       "86  We usecookiesto track usage and preferences.I ...  \n",
       "87  We usecookiesto track usage and preferences.I ...  \n",
       "\n",
       "[88 rows x 9 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_ar_plant_biology"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7dd02d31-3a36-4220-b46e-479fa30aca9b",
   "metadata": {},
   "source": [
    "### Inspecting a few samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "d46241f5-906e-4c07-b2ff-1c087929d6df",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"We usecookiesto track usage and preferences.I Understand\\nThe acquisition of quantitative information on plant development across a range of temporal and spatial scales is essential to understand the mechanisms of plant growth. Recent years have shown the emergence of imaging methodologies that enable the capture and analysis of plant growth, from the dynamics of molecules within cells to the measurement of morphometricand physiological traits in field-grown plants. In some instances, these imaging methods can be parallelized across multiple samples to increase throughput. When high throughput is combined with high temporal and spatial resolution, the resulting image-derived data sets could be combined with molecular large-scale data sets to enable unprecedented systems-level computational modeling. Such image-driven functional genomics studies may be expected to appear at an accelerating rate in the near future given the early success of the foundational efforts reviewed here. We present new imaging modalities and review how they have enabled a better understanding of plant growth from the microscopic to the macroscopic scale.\\nArticle metrics loading...\\nafn1*These authors contributed equally to this article\\nThe acquisition of quantitative information on plant development across a range of temporal and spatial scales is essential to understand the mechanisms of plant growth. Recent years have shown the emergence of imaging methodologies that enable the capture and analysis of plant growth, from the dynamics of molecules within cells to the measurement of morphometricand physiological traits in field-grown plants. In some instances, these imaging methods can be parallelized across multiple samples to increase throughput. When high throughput is combined with high temporal and spatial resolution, the resulting image-derived data sets could be combined with molecular large-scale data sets to enable unprecedented systems-level computational modeling. Such image-driven functional genomics studies may be expected to appear at an accelerating rate in the near future given the early success of the foundational efforts reviewed here. We present new imaging modalities and review how they have enabled a better understanding of plant growth from the microscopic to the macroscopic scale.\\nMany important biological processes within an organism occur on different spatial and temporal scales. For example, cell division is triggered by specific gene and protein networks. As cells divide, they form larger tissues and organs that can be influenced by external factors, such as chemical and physical conditions or biological interactions. To better understand how these processes proceed over different spatial scales, researchers have developed imaging methods that range from microscopic intra- and intercellular interactions to mesoscopic tissue- and organ-level processes to the macroscopic scale, including whole organisms or even populations. Some of these imaging techniques can continuously sample at an appropriate rate and in response to variations in experimental conditions, which enables the quantification of the dynamics of developmental processes. For example, live imaging (imaging the activity as it is occurring) provides unprecedented levels of temporal detail that encompass the study of plant biology, including tissue patterning, cell division, cell specification, hormone distribution, and plant architecture. Methods like fluorescence correlation spectroscopy (FCS) (34),Förster resonance energy transfer coupled with fluorescence lifetime imaging (FRET-FLIM)(164), and super-resolution microscopy (137) have been developed to study molecular dynamics within a single cell, such as protein binding, as well as between different cells, such as protein movement (Figure 1). Moving to a larger scale, methods such asmagnetic resonance imaging (MRI), fluorescence molecular tomography (FMT), X-ray microcomputed tomography (μCT), positron emission tomography (PET), and ultrasound have been used to obtain images of whole organs and tissues, which have proven to be useful tools in medical imaging (Figure 1) (54,118,168). The combination of data from these imaging modalities draws an integrated picture of plant biology, from the single cell to the entire organism. On an even larger scale, new techniques have been developed, allowing for quantitative imaging of entire plant populations. These macroscale techniques are especially important for surveying crop quality and determining how environmental conditions impact yield (Figure 1).\\n\\nFigure 1Click to view\\n\\n\\nWhile most imaging techniques had their original applications in animal systems, many have been adapted to plants (132). Here, we describe recent imaging techniques that have been successfully used to analyze plant biology on increasing spatial scales, ranging from microscopic to mesoscopic and macroscopic (Figure 1). Starting at the molecular level, we detail how FRET-FLIM and scanning fluorescence correlation spectroscopy (FCS) methods have been used to quantify protein interactions and movement in vivo within different plant cell types. Next, at the mesoscopic scale, we present methods that either monitor cellular events in the context of whole growing organs with light sheet fluorescence microscopy (LSFM) or image tissues from multiple plants simultaneously under controlled environmental conditions using microfluidics devices. Finally, at the macroscopic scale, we present methods used to image entire plant organs or even different populations of plants grown together in a field. Techniques such as PET, X-ray μCT, and MRI imaging, co-opted from the medical imaging field, have been used to noninvasively study root systems growing in soil. Additionally, we detail how ultrasound, infrared imaging, andlight detection and ranging (LiDAR)have been coupled with technologies such as unmanned aerial vehicles to allow for high-throughput imaging of entire plant populations. Looking to the future, we believe that developing methodologies that combine multiscale imaging data will allow for a more accurate picture of how biological processes occur over space and time within and across organisms.\\nFluorescence microscopy is an essential tool in cell biology. The use of fluorescence imaging technologies has contributed tremendously to solving biological questions such as the visualization of tissue dynamics and morphogenesis, determination of subcellular localization, changes in protein turnover in time and space and under different stimuli, and formation and stoichiometry of protein complexes. In the last 25 years, the performance of fluorescence imaging has increased dramatically in the biological sciences, especially with the new generation of confocal microscopes equipped with higher-sensitivity detectors and genetic engineering advances in generating fluorophores with higher brightness, improved quantum yield and fluorescence lifetime, and increased sensitivity (10,52,53,141). With these tools, in vivo imaging can be implemented noninvasively and as a routine technology. A great example is the determination of subcellular protein localization within living cells and at high resolution in a living organism (62,85,86).\\nIn 1946, Theodor Förster introduced Förster resonance energy transfer (FRET) (46). FRET describes the phenomena in which, upon excitation, energy is transferred from a donor fluorophore to an acceptor fluorophore present within a range of 1–10 nm. FRET occurrence results in a decrease in the intensity of the donor's fluorescence and an increase in the acceptor's fluorescence intensity. FRET can also be measured by determining the lifetime of the donor fluorophore using fluorescence lifetime imaging microscopy (FLIM) (25). The fluorescence lifetime can be described as the average time that a molecule stays in an excited state prior to its return to the ground state. The fluorophore lifetime can be displayed as images, which include the spatial distribution of excited-state lifetimes (20). During measurements, FLIM images are taken and lifetime values can be extracted by analyzing a region of interest (ROI) drawn within a pseudocolor lifetime map. Interactions can then be detected by measuring changes in the lifetime between ROIs within the same cell, tissue, or organ. These continuous advances allow the detection of protein localization as well as its physical association with relevant partners with a high spatial and temporal resolution, with high specificity, and without perturbing cell state, making it possible to follow protein association at the subcellular resolution in living multicellular organisms.\\nIn plant tissues, FRET-FLIM has been widely used to validate, detect, and visualize protein interactions (Figure 2), especially with the increasing amount of protein interactions identified by yeast two-hybrid screens or mass spectrometry (MS)-based proteomics. Interactions of the MADS-box proteins were confirmed by FRET-FLIM in plant-derived protoplasts (72,73,153). The plasmodesmata-localized reticulon (RTNLB) family proteins are additional examples of proteins identified by coimmunoprecipitation MS (87). FRET-FLIM was also used to show homodimerization of the transcription factor family AUXIN RESPONSE FACTORs (ARFs), which have important roles in auxin response and plant growth and development (12), as well as heterodimerization with auxin/indole-3-acetic acid (AUX/IAA) proteins (e.g., ARF9 and IAA10) (122). In the case of receptor kinases, implementation of FRET-FLIM was insightful in showing that ARABIDOPSIS CRINKLY4 (ACR4) forms homomers and heteromers with CLAVATA 1 (CLV1) at the plasma membrane or in plasmodesmata in a concentration-dependent manner (145). FRET-FLIM analysis of the ligand-perceiving receptor for brassinosteroids, BRASSINOSTEROID INSENSITIVE 1 (BRI1) with its receptor SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASE (SERK3), revealed interactions at the plasma membrane in the epidermis of livingArabidopsisroots (13). Intriguingly, despite colocalization on the plasma membrane, interactions were detected only in a limited domain of the plasma membrane, indicating that only a small fraction of the BRI1 and SERK3 proteins are in close proximity (13). In addition to resolving interactions with proteins involved in signaling processes within living tissues, FRET-FLIM was successfully used to detect transcription factor associations at cellular resolution in theArabidopsisroot meristem. Recent studies showed that the BIRD protein JACKDAW (C2H2-type transcription factor) forms a ternary complex with the mobile protein and cell fate regulator SHORTROOT (SHR) and its target SCARECROW (SCR) (92). Optimizing FRET-FLIM technology in livingArabidopsisroots showed that physical proximity between these proteins is cell-type dependent (92). These observations reflect the advantage of using this technology to detect differential protein associations within the same as well as between different subcellular compartments.\\n\\nFigure 2Click to view\\n\\n\\nAnother extension of FRET-FLIM techniques involvesscanning FCSmethodologies. Tagging a protein with a fluorescent molecule allows scanning FCS to detect and quantify the constant fluctuations in fluorescence and protein dynamics (38). Fluorescence fluctuations over time correlate with the movement or kinetics of the fluorescent molecule while the amplitude of the fluctuations correlates with the protein concentration (33,117,126). Scanning FCS has been developed to determine the oligomeric state of a protein complex, quantify protein interactions, and map protein movement between and within cells. Although scanning FCS has mainly been used in the animal kingdom to quantify, for example, protein aggregates, an emerging role for scanning FCS in the plant kingdom has become apparent in the last few years (107,126).\\nRather than operating independently, proteins interact with each other to form functional complexes in order to exert their molecular roles during plant development. As such, the stability, activity, and functionality of a protein can depend on its interaction partner. Analyzing the composition or oligomeric state of a protein complex as well as the dynamics of the formed complexes is thus important in the study of plant development and possible through the development of scanning FCS methodologies. To measure the oligomerization state of single proteins and heterologous protein complexes, one can use scanning FCS in combination withnumber and brightness (N&B)analysis (Figure 2) (32). For this, the protein of interest is tagged with a fluorescent molecule. By detecting fluorescence intensities over time with raster scan images, the average and variance of intensity can be measured and used to calculate the N&B, i.e., the intrinsic intensity of the fluorescent molecule (38). N&B allows for the detection of different oligomeric states of fluorescent-tagged proteins while taking into account immobile fluorescent-tagged proteins, autofluorescence, and bleaching in a living cell (23,32). If two proteins are tagged with different fluorophores [e.g., green fluorescent protein (GFP) and mCherry], N&B can quantify protein complex stoichiometry based on the coinciding appearance of two fluorophores in a process calledcross-correlation N&B (cross N&B)or cross-correlation FCS (Figure 2) (4,138). For cross N&B, the fluctuations of fluorescence are calculated from the intensity measured over time from two or more detection channels, depending on the number of different fluorophores (64).\\nIn the root stem cell niche ofArabidopsis thaliana, N&B analyses have been used to determine the oligomeric state of several key regulators in stem cell maintenance (21,22). First, by analyzing raster images of specific stem cells in the root stem cell niche, one can determine the cell specificity and distribution of protein complexes and their oligomeric state. As such, the authors (21,22) reported that the stoichiometry, as well as the abundance of the well-known SHR-SCR complex, varies between the quiescent center and the cortex endodermis initials, which contributed to the fundamental understanding of the functional role of this complex in stem cell maintenance. The SHR-SCR complex acts to spatially restrictSHRexpression to the endodermis, leading to cell-specific control of cell division (84,144). In another example, N&B was used in the root to show that ARF7 and ARF19 form higher-order protein assemblies (up to 10-mers) that limit auxin responsiveness (120). Thus, cross N&B provides a complementary approach toFRET-FLIM, and using both techniques together provides the most information about protein binding and protein–protein complex stoichiometries within a sample. Second, the dynamics of protein complex composition can be analyzed by imaging over multiple time points during plant development. Third, protein complex information can be built in computational models to more accurately fit them to experimental data and vice versa and can be used to validate model predictions.\\nComputational models also often integrate kinetic data, which are generally more difficult to experimentally determine, but the difficulty can be addressed withscanning FCS. By using the same raster scan for N&B analysis in a process calledraster image correlation spectroscopy (RICS), scanning FCS can also quantify protein movement (Figure 2) (31). The diffusion of a fluorescent protein can be calculated from these time series images by removing the fluctuations of fluorescence that result from the speed of the scan, an intrinsic setting of the confocal microscope (31). As such, the diffusion rate of SHR between the vasculature initials, stem cells that expressSHR, and the neighboring cortex endodermis initial and quiescent center cells was determined (22). As with N&B, RICS can also correlate two fluorophores in a process called cross-correlation RICS (cross RICS) to measure how the binding between two proteins affects their movement (35). Thus, RICS allows for the measurement of protein movement in different regions of a single cell or in different cell types of a multicellular organism.\\nOne of the drawbacks of RICS is that it cannot detect whether movement is restricted at barriers within cells, such as the nuclear envelope or organelles, or between cells, such as cell membranes and walls. To address this drawback, thepair correlation function (pCF)was developed to measure protein movement through a barrier (67). pCF uses a single line scanned across the potential diffusion barrier to calculate the probability of fluorescence fluctuations in the case of free movement and then correlates this probability with the measured fluorescence (33). When the pCF analysis shows no autocorrelation at the diffusion barrier, the protein cannot cross the barrier, while autocorrelation suggests that the protein can pass the barrier (Figure 2) (33). Therefore, pCF can reveal whether protein movement within or between cells is restricted in any manner. Specifically, pCF has been used to unravel the molecular mechanisms underlying stem cell maintenance in the root stem cell niche. The stem cell identity in the root stem cell niche is maintained through the controlled regulation of asymmetric cell divisions by the SHR-SCR complex, among others (144). With pCF analyses, the authors (22) were able to detect SHR movement from the vasculature initials to the cortex endodermis initials, where it forms a complex with SCR that in turn restricts the diffusion of SHR back to the vasculature. The molecular mechanisms regulating stem cell maintenance were even further unraveled with pCF, revealing that WUSCHELRELATED HOMEOBOX5 (WOX5), a central regulator in maintaining the dormant state of the quiescent center, can diffuse to the vasculature initials where it regulates downstream targets such as SHR (21).\\nLSFMis a recent powerful addition to the arsenal of modalities available to study development at the mesoscopic scale (148). LSFM uses two distinct optical axes. The first one illuminates a thin slice of the specimen with a sheet of laser light while keeping the rest of the sample in the dark. The second optical axis is orthogonal to the illumination axis and collects the light emitted by the fluorophores in the illuminated volume of the specimen. By moving the sample stepwise across the sheet and taking an image at each position, a stack of images encapsulating the whole volume of the specimen is collected (Figure 3) (71,167).\\n\\nFigure 3Click to view\\n\\n\\nThe LSFM design combines the speed and sensitivity of wide-field fluorescence microscopy with the optical sectioning ability of confocal microscopy, making LSFM particularly amenable to imaging of organs and organisms while minimizing photodamage. At any given time, only a small volume of the specimen is illuminated; therefore, during a three-dimensional (3D) scan, the total amount of laser light received by the whole specimen is minimal (one to two times the amount received by a single plane). In a confocal microscope, for each focal plane imaged, the whole specimen is illuminated, and the emitted light coming from regions above and below it is eliminated by the pinhole placed in front of the detector. Consequently, the total amount of light received by the sample during a 3D scan is proportional to the number of planes imaged. In addition, for laser scanning confocal microscopy, the pinhole imposes a point-by-point scanned detection. In LSFM, images are acquired by modern electron-multiplying charge-coupled device or scientific complementary metal-oxide-semiconductor cameras that have higher signal-to-noise ratios than the photomultipliers found in most laser scanning confocal microscopy. This combination of high sensitivity, speed of image acquisition, and parsimonious use of the energy illuminating the sample makes LSFM particularly suited for imaging biological processes over extended periods of time and/or at high frequency. Many variations of the basic LSFM design have been implemented over the years in lab-built and commercial solutions. LSFM designs differ in the type of illumination (single-photon versus two-photon, static versus scanned versus Bessel beams or lattices), the number of sheets and cameras used simultaneously, their positioning (horizontal versus vertical), and the use of adaptive optics. These variations provide specific advantages for each application and specimen but also illustrate that LSFM setups can appear too specialized: Changing magnification is not easily done, and usually only a single specimen can be imaged at a time (119,124). This limitation inspired the development of a multisample holder that allows simultaneous imaging of several specimens (28).\\nApplication of LSFM to plant biology has bloomed in the last seven years. Most LSFM designs (custom built and commercial) use one or two illumination sheets arranged horizontally with the plant positioned vertically in the imaging chamber (8). Imitation of close to natural conditions during imaging is achieved by the illumination of aerial tissue with a light-emitting diode system mimicking sunlight that can simulate the light patterns of long or short days. In addition, a perfusion system exchanges media in the chamber, ensuring a constant supply of nutrients as well as the removal of contaminants or generated toxic compounds (97). This permits imaging over the course of several days with no apparent damage (160,163). Mounting the plant on the LSFM device to ensure stability over time is specific for each application (8,111,160,162). To date, the majority of LSFM imaging in plants has been on theArabidopsisroots because of their ideal geometry and good optical properties. Yet, imaging of other organs (leaves, flowers, shoots meristems) and species, including crops (Medicago, rice, tomato), has been achieved (see112for a comprehensive review).\\nA strength of LSFM is its ability to bridge microscopic and mesoscopic scales. LSFM can image fast subcellular processes in the context of intact organs and over the course of several hours or days. LSFM was used inArabidopsisto relate cellular proliferation with the growth of the primary root (28,97,139). LSFM was used to quantify the abundance of the cytoskeletal END BINDING PROTEIN 1C (EB1c), an atypical and plant-specific microtubule plus-end-tracking protein in the growingArabidopsisroot (108). It revealed that EB1c, previously only studied in the context of dividing cells, resides in the nucleus between divisions, where its accumulation correlates with the nuclear surface as the cells differentiate. This use of LSFM illustrates how quantitative analysis can uncover new links between the cytoskeleton, nuclear size, and differentiation. LSFM was also used to reveal that the MITOGEN-ACTIVATED PROTEIN KINASE 6 (MPK6) associates with the phragmoplasts where it controls proper division of root cells (143). At subcellular levels, LSFM has been used to observe Ca2+dynamics where it characterized calcium oscillations associated with growth rate and auxin effects on growth (18). LSFM also enabled the imaging of cytosolic and nuclear Ca2+waves deep into a growing root, revealing the existence of a long-postulated process of Ca2+percolation across tissues (26).\\nLSFM is particularly well suited to investigate the development of plant organs over several days and elucidate, by bridging the cell to the organ scale, how plant organs are shaped. LSFM was used to imageArabidopsislateral root development, to uncover and quantify the ellipse-to-round shape transition of the lateral root primordium (93), and to precisely analyze the patterns of cell divisions (161). These studies uncovered that the typical dome shape aspect of the lateral root primordium and its internal organization in layers are the result of the accommodation of the overlying tissue to the primordium outgrowth and the geometric pattern of cell division, respectively. LSFM uncovered that the swelling of the lateral root founder cells deep in the primary root is an important signal perceived by the endodermis, which signals back to these cells and allows their further development (160). LSFM helped researchers to understand how the growth angle of a lateral root in regard to the gravity vector results from the asymmetric distribution of auxin yielding to the asymmetric expansion of epidermis cells and the bending of the young lateral root (131).\\nPlant growth and development, as well as physiology and metabolism, are sensitive to environmental stress conditions, which means that specimen handling and variable conditions during live imaging may affect the validity and reproducibility of obtained results. Therefore, technical innovations are needed to reduce sample handling while still allowing for chemical and biological treatments. A classic example is the Fåhraeus slide, with roots growing between a microscope slide and a cover glass that are separated by a spacer (9,41). The use of these slides revealed the effect of bacterial nodulation factors on root hair deformation during the onset of symbiosis between rhizobia and legumes (43,63).\\nOver the past decade, microdevices fabricated from the silicone-based organic polymer polydimethylsiloxane (PDMS) have become increasingly popular tools for the cultivation, perfusion, and imaging of plant cells or organs (Figure 4). Biologically inert and gas permeable, PDMS has been widely used in microfluidiclab-on-a-chip devicesthat are cast and fixed onto optical glass to create chambers that facilitate microscopic observation of live specimens (170). Due to their design flexibility, lab-on-a-chip-based microdevices have a wide range of applications for the study of plants with diverse morphologies and their development under different environmental conditions. The first plant microdevices were developed to cultivate primaryArabidopsisroots (17,58,100),CamelliaorToreniapollen tubes (68,134,173),Arabidopsisovules (115), or the filamentous protonemal tissue of the moss modelPhyscomitrella patens(6) under controlled conditions at the microscope for experiments lasting hours to days or weeks. Recently, this technology has been adapted for roots of larger plants, such asOryza sativaandPopulus tremuloides(57,70). As we will illustrate in the following paragraphs, several discoveries in plant physiology, developmental and cell biology that were recently made using microdevices demonstrate their potential for the unveiling of yet unknown molecular mechanisms in plants.\\n\\nFigure 4Click to view\\n\\n\\nA main advantage of microfluidic imaging platforms over other sample mounting and perfusion systems is the minimal sample handling. Pollen grains, plant protoplasts, blended moss tissue, and even ovules can be manually introduced as suspension directly into a dedicated germination chamber of the device or immobilized into casted traps (Figure 4) (115,133,135,140). Seedling roots grow into the device autonomously, which prevents mechanical damage (17,58). Another significant improvement is the increase in throughput that can be achieved through the parallelization of experiments within such miniaturized platforms. The RootChip was initially developed for the parallel imaging of 8 (58) and later 16 roots (77) at a time, which enabled fluorescent sensor-based measurements of small molecule dynamics in living tissue. The RootArray enabled the analysis of gene expression patterns in up to 64 roots ofArabidopsisin a single chamber (17). Upon changing growth conditions, such as a drop in pH from 5.7 to 4.6 or a deficiency in iron or sulfur, the authors observed dramatic but transient changes in gene expression. For example, the homeodomain transcription factor WOX5, typically expressed exclusively in the quiescent center, was transiently detectable within the root elongation zone (17). By leveraging and combining the technical advances made in microfluidic devices and LSFM, researchers developed a 3D-printed device, the Multi-sample Arabidopsis Growth and Imaging Chamber (MAGIC), that is able to image whole organs for a longer period of time (28). MAGIC has been custom-designed for a commercial, dual-sided illumination light sheet microscope and it allows researchers to grow and image fourArabidopsisroots simultaneously. Moreover, with the design of MAGIC, shoots grow outside of the agar, allowing gas exchange during the imaging process, which leads to successful imaging of cell division in the root meristem over a period of 48 hours (28). Thus, the expression of pCYCB1;1:GFP, a marker for cell division, was tracked for 24 hours under conditions of iron sufficiency, iron deficiency, heat stress, and combined iron deficiency and heat stress (14). Surprisingly, the change in the number and timing of cell divisions under the combined stress differed from the additive effect of the individual stresses, indicating that combinatorial stresses trigger unique stress responses (14).\\nTaking advantage of trapping up to 40Arabidopsisovules in a microcage array (115) and employing two-photon microscopy enabled the first live imaging of embryogenesis over three days from the first zygote division to a heart-shaped 64-cell embryo (55). Following cell fate specification, this study established a cell lineage tree and revealed cell damage-induced cell fate conversion. Upon laser-ablation of the apical cell of a proembryo within the microcage, the authors revealed a compensatory change in cell identity in a neighboring nonembryonic cell toward an embryonic fate as indicated by cell-type specific transcriptional reporters (55).\\nThe applications of microfluidic lab-on-a-chip devices for plant studies go beyond providing a platform for specimen mounting for microscopy; they include chemical, biotic, and mechanical stimulation as well as the probing of mechanical properties of cells and tissues (Figure 4). While a main motivation for the use of perfusion devices is to keep conditions constant, devices such as the RootChip contain micromechanical valves (158) and thereby enable pulsed treatments with a complete exchange of the chamber condition in less than 10 s (58,59). Several studies took advantage of such on-chip valve systems in combination with genetically encoded fluorescent biosensors to visualize molecular dynamics during membrane transport (58,88), to reveal stimulus-specific calcium signaling in response to biotic or abiotic stress elicitors (30,80), or to uncover the dynamic distribution of the phytohormones abscisic acid (77) and gibberellin (127) in growingArabidopsisroots. While most microscope setups only allow for horizontal specimen mounting, one study employed a vertically mounted RootChip to investigate the inhibitory effect of auxin on root cell elongation, while avoiding interference by gravitropic responses (44). The authors found that, upon auxin treatment, roots responded within 30 s by slowing down their growth, reaching a 70% slower growth rate after 10 min (44). By comparing the response of mutants lacking auxin signaling components such as the auxin efflux carrier AUX1 or the auxin coreceptors TIR1, AFB2, and AFB3, the authors were able to provide further strong evidence for the existence of an auxin signaling pathway that acts independently of transcriptional regulation (44).\\nIn comparison to larger perfusion systems, the small chamber volumes of microdevices allow for a reduced consumption of applied compounds and a more rapid and uniform treatment of the specimen. However, specialized channel architectures can also enable local or asymmetric stimulation of specific parts of tissues or single cells and thereby facilitate studies on mechanisms of stimulus perception, local and systemic adaptive responses, and chemotaxis. An early pioneering work offered the possibility to apply an auxin treatment perpendicular to the root axis, with the stimulation stream focused by multilaminar flow and thereby locally inducing outgrowth of root hairs (100). To investigate pollen tube chemotaxis toward female gametophytes, two studies examined the response to ovule-derived attractants by offering growing pollen tubes a choice to grow toward an empty chamber or one filled with ovules (68,173). Instead of perpendicular stimulation, asymmetric conditions can also be generated in parallel to the specimen's growth axis. A device for pollen tubes allowed researchers to subject a growing cell to asymmetric growth conditions, revealing that growth direction follows the optimal calcium concentration in the environment (135). Asymmetric conditions are also highly relevant for studies on root development, as environmental heterogeneity is a hallmark of soil. The dual-flow-RootChip device used a micropillar array to guideArabidopsisroot growth in the middle of the observation chamber and took advantage of two laminar streams of the medium separated by the root in the center (147). Experiments involving the asymmetric availability of inorganic phosphate on either side of the root (2.4 mM versus 0.01 mM) resulted in rapid, cell-autonomous stimulation of root hair growth on the inorganic-phosphate-rich side that preceded rather than followed transcriptional regulation (147). Together with the aforementioned transcription-independent response to auxin (44), these studies point to yet unknown, fast, nongenomic regulatory mechanisms of cellular growth. The unveiling of these mechanisms will greatly benefit from the ability to perform long-term live-cell imaging in microfluidic perfusion systems.\\nAnother major aim for root biology is to understand biotic interactions with other organisms in the rhizosphere.Lab-on-a-chip-based devices offer solutions to accomplish live imaging of the infection and colonization of roots by symbiotic or pathogenic microbiota (146). However, very few studies have so far explored the potential of plant–microbe or plant–herbivore interactions with lab-on-a-chip-based devices (3,98,114). The Tracking Root Interactions System enabled, for example, the study of the preferential association ofBacillus subtiliswith the elongation zones of roots of differentArabidopsisgenotypes in a double-chamber device housing two parallel roots that were separated by a perforated barrier, allowing for free movement of bacteria (98).\\nWhile chemical conditions and biotic interactions mainly influence root growth and development at the metabolic level, mechanical forces exerted by and on the root play a more direct role in root growth rate and direction and, as a consequence, root system architecture (37). To explore the intrinsic forces exerted by growing plant cells on their environment, several strategies have been developed in microdevices that take advantage of known physical properties of PDMS. Using the deformation of sidewalls in narrow growth channels as readout, researchers determined a dilating pressure ofCamelliapollen tubes of 0.15 mPa (134). Alternative approaches used to determine growth force in pollen tubes were the application of capacitive force sensors (16) or microcantilevers at the end of growth channels that became deflected by the tube growing against them (51). Furthermore, when growing pollen tubes against cantilevers, the authors observed frequent bursting of the pollen tubes when the cantilevers approached maximum deflection, which indicated that the reduced growth rate caused by the counterforce of the cantilever induced a cellular compensation mechanism that led to a weakening of the apical cell wall (51). A cantilever-based approach had also been applied to measure the growth force ofA. thaliana,Nicotiana benthamiana, andCapsella rubellaroots in a microdevice, which yielded species-specific values (65,113).\\nBeyond the intrinsic forces exerted by plant cells during growth, extrinsic forces play a significant role in modulating growth behavior. To understand how extrinsic mechanical forces are perceived by pollen tubes, investigators have developed microdevices such as TipChip that include obstacles and wavy channel architectures for pollen tubes (2,50). Using devices with specialized compression valves that crossed the observation channels and allowed for actuated mechanical stimulation of pollen tubes enabled the quantification of mechanical properties of tip-growing cells and the demonstration of the interplay between turgor pressure and cell wall stiffness during tip growth (69). In another study, pollen tubes and root hairs were forced to grow through gaps with a width of less than one-tenth of the cell's normal diameter (172), revealing a surprising capability of tip-growing cells to overcome such obstacles without stalling growth or bursting.\\nThough powerful, lab-on-a-chip-based devices do not reach the level of structural complexity attained by soil on roots. Synthetic, particle-rich substrates like transparent soil, consisting of irregularly shaped polymer particles with the same refractive index as water, have been shown to produce similar root system architectures as soil or sand (36). Since the material is virtually invisible, it represents a promising way for root phenotyping at microscopic detail, taking soil structure and other environmental cues into account (109). Great potential for new discoveries lies in the vast possibilities to create synthetic environments that combine multiple iterations of the above-mentioned features and thereby to model ecosystems and unveil trade-off and decision-making mechanisms during the growth of plant cells and the development of plant organs (142,146).\\nAccurate phenotyping of agronomic traits is critical for increased crop production and the development of new plant varieties. While the affordability and efficiency of genotyping are increasing, the cost and efficiency of phenotyping remain bottlenecks. Manual phenotyping at high spatial and temporal resolution is impractical and time-consuming (47), motivating the development of automated, imaging-based phenotyping approaches aimed at quantifying roots and shoots.\\nPhenotyping methods for root systems can be divided into two broad classes: noninvasive and destructive. While destructive methods are still the primary means for capturing images of field-grown root systems, noninvasive methods for plants grown in solid substrates have become increasingly available for greenhouse and growth-chamber-grown plants. The improved phenotyping capabilities afforded by noninvasive methods are necessary to accurately quantify dynamic root traits for crop improvement. There is no single ideal solution for macroscopic root imaging, and the choice of platform depends on the biological question, experimental parameters, and laboratory resource constraints. In addition, each imaging approach requires significant image postprocessing and analysis, with specific software being developed for many methods.\\n2D noninvasive root phenotyping methods include growth on agar plates (102) or in gellan gum (74), germination paper (rhizoslides) (89) or pouches (155), or nylon fabric in hydroponics (rhizoponics) (99). In these methods, roots are generally imaged with either a flatbed scanner or a digital camera. The major advantages of these approaches are that they provide root growth data at low cost, are accessible to most laboratories, and are relatively high throughput. Some (e.g., growth in gellan gum) can also be converted into 3D information (24,154). Using this approach, 89 quantitative trait loci for 25 2D and 3D root traits were identified in a rice recombinant inbred line population grown in gellan gum, which provide breeding targets for specific root architecture traits (154). A drawback of these methods is that the use of artificial media can impact the root architecture, and thus extrapolation to plants grown in soil is challenging. 2D methods that use soil or soil-like substrates include the clear pot method (125), which images roots that grow at the periphery of soil-filled containers; growth in rhizoboxes such as Glo-Roots (123), where plants expressing luciferase reporters are grown in soil between polycarbonate plastic plates and imaged in a luminescence imaging system; and transparent soil methods (36,94).\\nOne of the biggest challenges in root phenotyping is the nondestructive 3D imaging of soil-grown roots. Recent advances in tomographic methods make them a promising choice for nondestructive imaging in three dimensions. In tomographic imaging methods, an object-penetrating wave or beam creates 2D image cross sections that are stitched together to generate 3D images. MRI, X-ray μCT, and PET are three of the more common tomographic methods used for noninvasive 3D root phenotyping in soil (Figure 5b). The major advantage of these methods is their ability to view root system architecture or function nondestructively in native environments.\\n\\nFigure 5Click to view\\n\\n\\nMRI uses nuclear magnetic resonance to detect signals from atomic nuclei, usually1H (protons) in plants. Magnetic fields are used to manipulate the nuclear magnetic moment or rotational spin of protons. Ultimately this information is processed into 3D image data sets. MRI has been used to examine root–nematode interactions (66), storage roots (101), and cereal root development (159), but it can also be used to view the distribution of root water uptake (149) and mobility (130,136,171). MRI is sensitive to the soil substrate, and distinguishing nuclear magnetic resonance signals originating from the water in soil versus the water in roots can be difficult. These challenges can be overcome by adjusting the experimental and acquisition parameters on the MRI to increase root-to-soil contrast (129). Although MRI imaging time is long and thus not suitable for large populations, under the right experimental conditions the contrast between root and soil is sufficiently robust so that image processing is relatively straightforward (159).\\nIn recent years, X-ray μCT has moved to the forefront for noninvasive 3D imaging in soil. As X-rays pass through a sample, a portion of the X-rays are absorbed, and X-ray intensity decreases. This decrease in intensity is known as attenuation and is a function of the density of the sample (95). One major challenge in X-ray μCT is that the attenuation values of roots and soil overlap, which creates significant challenges for data analysis (156), and attenuation values may vary in different zones of the root. There is also a trade-off between sample size and image resolution, but recent improvements in X-ray μCT scanners and image processing algorithms have significantly improved the methodology (104). X-ray μCT has been used to examine the impact of soil and substrate density on root growth and architecture (15,128), root–root interactions (96), potato tuber growth (116), and root–fungal interaction (151). This technology also enhanced our understanding of lateral root development in soil. Using X-ray μCT, Orman-Ligeza et al. (110) found that maize and barley roots suppress lateral root formation when roots grow through air spaces in soil and are not in contact with water. This observation led to additional experiments that showed that transient abscisic acid application suppressed lateral root formation by altering lateral root prebranch sites near the root tip.\\nThere appear to be minimal deleterious effects of X-ray μCT at low total dosages (<30 Gy) on root growth in most plants or on microbial population biomass (11,45,175), but dosage effects on plant growth vary by plant species (reviewed in175). Another type of X-ray CT, synchrotron X-ray computed tomography, uses high-intensity X-rays from a synchrotron to noninvasively view roots. This method offers micrometer-level resolution and is useful for noninvasively viewing root cellular anatomy. Synchrotron X-ray computed tomography has been used to assess root hydraulics at the cellular level (27) and root hair–soil interactions (81).\\nUnlike MRI and X-ray μCT, PET is used not for observing root structures, but for investigating their function. PET quantifies the distribution of a positron-emitting radioisotope (such as11C or13N) in a plant over time with spatial resolution of approximately 1–5 mm. This allows observation of the kinetics of nutrient uptake, transport, and movement throughout the plant in three dimensions. Plant PET scanners (7,75,166) have been used to examine photoassimilate transport and allocation in sorghum (78) and the impact of root herbivory on auxin biosynthesis and flux in maize roots (121). By measuring the time of arrival of carbon-11-labeled IAA in two different regions of maize roots, auxin transport speed was shown to decrease in maize roots after Western corn rootworm attack (121). PET has also been used in conjunction with MRI or X-ray μCT for structure-function studies (49,75). While PET is a very good tool for addressing questions regarding root function, it is limited by high cost, low accessibility, and low throughput.\\nFor many researchers, nondestructive root imaging in the field is the ultimate goal, but methods for this have yet to be fully realized. Current methods include rhizotrons and minirhizotrons and ground-penetrating radar. Rhizotrons are similar to underground walk-in chambers with windows that look at the soil (103), while minirhizotrons are transparent tubes that can be inserted into the soil at various places and angles throughout a field. Rhizotrons and minirhizotrons allow successive measurements of the same roots over time and are well-suited to investigate questions about fine roots (19,39,76,165) and nodules. Minirhizotron imaging was used to identify the effects of high carbon dioxide concentration (CO2) and reduced precipitation on nodule distribution and number on field-grown soybean roots (56). This work demonstrated the complex, interactive effects of high CO2and reduced precipitation, which caused an increase in nodules per length of root and volume of soil. Surprisingly, given the increase in nodules, soybean leaves grown in high CO2and reduced precipitation did not have increased nitrogen on a leaf-mass basis. This may have been because the increased nodules were distributed on roots found in shallow, drier soil, which may have reduced the N2-fixing activity of each nodule (56). One drawback of rhizotrons and minirhizotrons is that they capture only a small portion of the root system and do not provide a 3D view. In contrast, ground-penetrating radar uses high-frequency radio waves to obtain images of belowground structures. Ground-penetrating radar is high cost and low resolution and has limited use in crop systems. To date, it has been used to image large tree roots (5,150), cassava tubers (29), winter wheat, and energy cane (91).\\nWhile many of the tomography approaches used for belowground imaging have also been used for aboveground imaging, the size of the instruments has limited these analyses to lab- or greenhouse-based experiments. In contrast to belowground imaging, the goal of nondestructive approaches in the field has proven more attainable for aboveground traits. Various field-based phenotyping platforms have partially automated the process of collecting field data, providing a means for rapid phenotyping to identify target plant traits (169). The challenge for aboveground field-based plant imaging has been the choice of imaging approach and the method of deploying that imaging platform in the field.\\nAs with root imaging, there is no single ideal solution for aboveground macroscopic imaging, and the approach depends on the biological question, experimental parameters, and laboratory resource constraints. The simplest approach to aboveground phenotyping is the acquisition of 2D images obtained from the visible light spectrum [red, green, blue (RGB) cameras] and near-infrared light spectrum (Figure 5) (40). These methods are reliable for detecting discoloration and thermal properties of individual plants (90) and can be used to extract measures of plant phenology and other morphological traits (60).\\nOther 2D imaging approaches take advantage of the spectral properties of plant tissues via imaging spectroscopy (e.g., multispectral andhyperspectral). Imaging spectroscopy combines digital imaging and spectroscopy to provide spatial maps of reflectance for a given wavelength of light, often visualized as a heatmap (Figure 5). Multispectral cameras acquire data from discrete and defined wide bandwidths (usually 5 to 12 bands), whereas hyperspectral cameras are capable of capturing thousands of narrow 5–20-nm spectral bands (1). Hyperspectral cameras provide higher resolution spectral data than multispectral cameras, but they are significantly more expensive, and much of the additional wavelength data has yet to be related to specific plant traits. In both systems, maintaining the appropriate calibration is critical for data quality and consistency of spectral measurements (90). Imaging spectroscopy has enabled the quantification of agronomically relevant plant traits (169). For example, the normalized difference vegetation index, which quantifies the amount of green vegetation (157), and photochemical reflectance index, which quantifies the photosynthetic efficiency of plants (48), were established to relate multispectral values to agronomic traits.\\nApproaches for the 3D reconstruction and quantification of aerial tissues include ranging sensors,time-of-flight (ToF) cameras, andstereography(Figure 5). These imaging approaches, often representing data in the form of point clouds, are gaining popularity for their ability to directly measure plant geometry via remote sensing. Ranging sensors typically emit either sound or light and measure the time for an accompanying detector to receive a reflection, which makes direct observations of plant geometry possible without physical contact. Two examples of ranging sensors include ultrasonic andLiDAR. Ultrasonic sensors are susceptible to diffusion even over short distances, so they are mainly used for overall plant height measurements (42), whereas LiDAR can resolve submillimeter geometries at high temporal resolution (152). Ranging approaches can generate dense 3D reconstructions of an entire field. However, the data sets are often sizable and require significant postprocessing, and moderately priced LiDAR devices can cost between $10,000 and $20,000. In contrast, ToF cameras use reflected light in a similar way as ranging sensors and cost significantly less but have much lower resolution. Light emitted for ToF sensing is typically in the infrared spectrum, which is directly affected by incident light, making these sensors noisy and unreliable, especially in areas with high levels of sunlight penetration. Despite the drawbacks, ToF has been shown to be a practical method for field-based data collection (83). Finally, stereo cameras, which are in the same price range as ToF cameras, bridge the gap between visual and 3D imaging. Stereo cameras are composed of two cameras separated by a known distance, which allows the triangulation of object distance based on the perceived image shift between the two cameras (82). Stereo cameras are useful because they are robust in well-lit areas and have significantly higher resolution when compared to ToF cameras; however, they suffer from image correspondence errors between the two cameras. Overall, ToF cameras have been shown to have favorable results over stereo vision despite their sensitivity to ambient light (79). ToF sensors are quite promising and in intensive development for increasing the throughput of data acquisition for plant breeding.\\nImaging sensors can be mounted on different data collection platforms depending on the plant trait and resolution required. Options for deploying imaging sensors to capture stalk and leaf phenotypes at high resolution include ground mobile robots (105,174) and existing field equipment. For example, imaging sensors have been mounted on irrigation systems to reduce cost and provide high-resolution overhead imaging (61). Unmanned aerial vehicles can also be outfitted with (generally small/lightweight) sensors as an alternative, low-cost method for imaging from overhead but are subject to flight restrictions and are power and payload constrained (176). Satellite imaging is another technique for measuring RGB and near-infrared light that is cost effective for imaging large areas, although the results have relatively low temporal and spatial resolution (106). The choice of data collection platform depends primarily on the target phenotype and resolution.\\nAdvances in macroscopic plant imaging techniques have been significant; however, major knowledge gaps have to be filled before we are able to bridge the gap between genotype and phenotype. Challenges in root phenotyping include nondestructively imaging root systems in the field. Challenges in shoot phenotyping include the advancement of dynamic deployment platforms. For both types of phenotyping, image analysis and trait data extraction (although not discussed here) remain bottlenecks. Ultimately, tying these traits back to plant function and crop improvement remains at the forefront of motivation for macroscopic plant imaging.\\nAbove we describe different imaging modalities used to collect both microscopic and macroscopic data on plant cells, tissues, organs, and populations. Images from a microscopic level can provide information about the mode of action of proteins, cell-to-cell communication, and dynamical cellular processes. Mesoscopic imaging informs us about the dynamics of cell behavior in tissues and organs in their environment. On a macroscopic level, images can yield morphometric phenotypic data, such as leaf size and root architecture, but also intrinsic plant traits such as chlorophyll content. The collection of images obtained across these different scales generates large data sets for which meta-analysis is the key next step. To enable meta-analysis, we need to further develop centralized databases to store imaging data and develop query tools to extract data from these images to optimally take advantage of available imaging resources. Future work should focus on employing computational methods such as machine learning, deep learning, and neural networks, which can extract and integrate large quantities of information from these collections of images. Additionally, other large-scale sources of data such as cell-type specific and/or organism-wide transcriptomics and proteomics should be integrated with these imaging data to allow scientists to connect morphological phenotypes with molecular phenotypes. As our biological models continue to grow and integrate processes from different temporal and spatial scales, the sustained development and refinement of these multiscale imaging methods are necessary.\\n\\n\\nThe authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.\\nI.B., N.M.C., L.V.d.B., and R.S. contributed to the writing of the section titled FRET-FLIM and Scanning FCS as Tools to Determine Protein Dynamics at Cellular Resolution, of which I.B. wrote the FRET-FLIM paragraphs and N.M.C., L.V.d.B., and R.S. wrote the scanning FCS paragraphs. A.M. contributed to the writing of the section titled Using Light Sheet Fluorescence Microscopy to Understand the Dynamics Underlying Plant Growth. M.G. and G.G. contributed to the writing of the section titled Microdevices for Imaging Growth and Organ Development. A.S., H.G.T., A.S.I.P., and E.E.S. contributed to the writing of the section titled Nondestructive Imaging Methods to Quantify Below- and Aboveground Macroscopic Plant Traits. All authors contributed to the writing of the introduction and conclusions and to the design of the figures.\\nWe apologize to all colleagues whose work could not be included in this review due to space limitations. We thank Lindsay Erndwein of Illustrations by LindZeaMays for illustrating the figures of this article. Research in the Sozzani lab was provided by the National Science Foundation (NSF) (CAREER MCB-1453130), NSF/Biotechnology and Biological Sciences Research Council (BBSRC) (NSF MCB 1517058), and the North Carolina Agricultural & Life Sciences Research Foundation in the College of Agriculture and Life Sciences at North Carolina State University (to R.S.). Research in the Maizel lab is supported by DFG FOR2581, the Land Baden-Württemberg, the Chica und Heinz Schaller Stiftung, the Cluster of Excellence CellNetworks, and the Boehringer Ingelheim Stiftung Plus3 program. E.E.S. is supported by grants from the Delaware Biosciences Center for Advanced Technology, the University of Delaware Research Foundation, and the Thomas Jefferson Fund. A.S. is supported by a University of Delaware Graduate Student Fellowship. Research in the A.S.I.P. lab is supported by grants from the Foundation for Food and Agriculture Research (FFAR), the NSF, Purdue University start-up funds, and USDA Hatch Funds (IND011293). G.G. is supported by research group funds from the Cluster of Excellence CellNetworks and a grant from the Deutsche Forschungsgemeinschaft (GR 4559/3-1). I.B. was supported by King Abdullah University of Science and Technology (KAUST).\\nMulti-sampleArabidopsisGrowth and Imaging Chamber (MAGIC) for the ZEISS light sheet Z.1 microscope:https://www.youtube.com/watch?v=spcUzefr35U; shows the preparation ofArabidopsissamples for long-term imaging in the MAGIC\\nThe Root Array:https://www.youtube.com/watch?v=Q9H3F94xHPg; describes a microfluidics device, developed by the Benfey Lab at Duke University, in which plants can be grown and their roots imaged by confocal microscopy over time and without manual intervention; this device, the Root Array, allows for high-throughput, high-resolution data acquisition, making it an important tool for analyzing dynamic gene expression in a living organ\\nPlant Image Analysis:https://quantitative-plant.org; maintained by Dr. Guillaume Lobet, provides an excellent resource for image analysis software\\nLiterature Cited\\nData & Media loading...\\nLiterature Cited\\n\\n\\n\""
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_ar_plant_biology.at[87, 'Text']"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "67d55358-33a0-4796-87c6-50ca4177fc6e",
   "metadata": {},
   "source": [
    "### Exporting to a file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "25ecc560-1829-44f0-805e-2f57137fa6cd",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_ar_plant_biology.to_json(f\"{output_directory}/{ar_plant_biology_prefix}.jsonl\", orient='records', lines=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cbd1bd53-2ca0-4d44-b565-9477d48ef510",
   "metadata": {},
   "source": [
    "## Web Scraping [Nature Food](https://www.nature.com/natfood/)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e1a17cd7-747a-4b10-81f3-ccd41e7a4081",
   "metadata": {},
   "source": [
    "### Defining input variables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "7ce38827-120b-49dc-841f-0f9c01b8c7c0",
   "metadata": {},
   "outputs": [],
   "source": [
    "nature_food_prefix = 'nature_food'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dc84350e-8381-4a31-b273-309f668a7711",
   "metadata": {},
   "source": [
    "### Importing the data into a DataFrame"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "728afeb1-6c51-4538-b4d0-bb299cc13fbc",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_nature_food_open_access = pd.read_json(f\"{input_directory}/{nature_food_prefix}_open_access.jsonl\", lines=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "1df521de-eace-4644-907e-021deb06e435",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_nature_food_open_access['Published'] = pd.to_datetime(df_nature_food_open_access['Published'], unit='ms')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a246fe08-ad2d-49f0-8b26-2f63d2920538",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Title</th>\n",
       "      <th>URL</th>\n",
       "      <th>Authors (compact list)</th>\n",
       "      <th>Published</th>\n",
       "      <th>PDF URL</th>\n",
       "      <th>Open Access</th>\n",
       "      <th>Area of Knowledge</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Transfer of cannabinoids into the milk of dair...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Bettina Wagner, Pietro Gerletti, Robert Pieper</td>\n",
       "      <td>2022-11-14</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A systematic scoping review of the sustainabil...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>A. Charlotte Bunge, Amanda Wood, Line J. Gordon</td>\n",
       "      <td>2022-11-03</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Rights and representation support justice acro...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Christina C. Hicks, Jessica A. Gephart, Rosamo...</td>\n",
       "      <td>2022-10-18</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Global dietary quality in 185 countries from 1...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Victoria Miller, Patrick Webb, Rubina Hakeem</td>\n",
       "      <td>2022-09-19</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Food system by-products upcycled in livestock ...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Vilma Sandström, Anna Chrysafi, Matti Kummu</td>\n",
       "      <td>2022-09-19</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Global food insecurity and famine from reduced...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Lili Xia, Alan Robock, Ryan Heneghan</td>\n",
       "      <td>2022-08-15</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>A shift from cattle to camel and goat farming ...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Jaber Rahimi, Erwann Fillol, Klaus Butterbach-...</td>\n",
       "      <td>2022-07-21</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Time management governs climate resilience and...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Andrew J. McDonald, Balwinder-Singh, R. K. Malik</td>\n",
       "      <td>2022-07-21</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>A hybrid inorganic–biological artificial photo...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Elizabeth C. Hann, Sean Overa, Robert E. Jinke...</td>\n",
       "      <td>2022-06-23</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>A systematic review of the definitions and int...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Kajsa Resare Sahlin, Joanna Trewern</td>\n",
       "      <td>2022-06-23</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Crop harvests for direct food use insufficient...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Deepak K. Ray, Lindsey L. Sloat, Wei Xie</td>\n",
       "      <td>2022-05-12</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Southeast Asia must narrow down the yield gap ...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-004...</td>\n",
       "      <td>Shen Yuan, Alexander M. Stuart, Patricio Grassini</td>\n",
       "      <td>2022-03-24</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Establishing long-term nitrogen response of gl...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Hans J. M. van Grinsven, Peter Ebanyat, Hein F...</td>\n",
       "      <td>2022-01-31</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Circularity in animal production requires a ch...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Benjamin van Selm, Anita Frehner, Hannah H. E....</td>\n",
       "      <td>2022-01-06</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Global maps of cropland extent and change show...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Peter Potapov, Svetlana Turubanova, Jocelyn Co...</td>\n",
       "      <td>2021-12-23</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Diet and lifestyle behaviour disruption relate...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Mohsen Mazidi, Emily R. Leeming, Sarah E. Berry</td>\n",
       "      <td>2021-12-16</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Short- and long-term warming effects of methan...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Ignacio Pérez-Domínguez, Agustin del Prado, Ma...</td>\n",
       "      <td>2021-12-13</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Spatial frameworks for robust estimation of yi...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Juan I. Rattalino Edreira, José F. Andrade, Pa...</td>\n",
       "      <td>2021-09-30</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>A scoping review of the contributions of farme...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>Livia Bizikova, Ephraim Nkonya, Beth Timmers</td>\n",
       "      <td>2020-10-12</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>A systematic review of employment outcomes fro...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>W. H. Eugenie Maïga, Mohamed Porgo, Gracian Ch...</td>\n",
       "      <td>2020-10-12</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                Title  \\\n",
       "0   Transfer of cannabinoids into the milk of dair...   \n",
       "1   A systematic scoping review of the sustainabil...   \n",
       "2   Rights and representation support justice acro...   \n",
       "3   Global dietary quality in 185 countries from 1...   \n",
       "4   Food system by-products upcycled in livestock ...   \n",
       "5   Global food insecurity and famine from reduced...   \n",
       "6   A shift from cattle to camel and goat farming ...   \n",
       "7   Time management governs climate resilience and...   \n",
       "8   A hybrid inorganic–biological artificial photo...   \n",
       "9   A systematic review of the definitions and int...   \n",
       "10  Crop harvests for direct food use insufficient...   \n",
       "11  Southeast Asia must narrow down the yield gap ...   \n",
       "12  Establishing long-term nitrogen response of gl...   \n",
       "13  Circularity in animal production requires a ch...   \n",
       "14  Global maps of cropland extent and change show...   \n",
       "15  Diet and lifestyle behaviour disruption relate...   \n",
       "16  Short- and long-term warming effects of methan...   \n",
       "17  Spatial frameworks for robust estimation of yi...   \n",
       "18  A scoping review of the contributions of farme...   \n",
       "19  A systematic review of employment outcomes fro...   \n",
       "\n",
       "                                                  URL  \\\n",
       "0   https://www.nature.com/articles/s43016-022-006...   \n",
       "1   https://www.nature.com/articles/s43016-022-006...   \n",
       "2   https://www.nature.com/articles/s43016-022-006...   \n",
       "3   https://www.nature.com/articles/s43016-022-005...   \n",
       "4   https://www.nature.com/articles/s43016-022-005...   \n",
       "5   https://www.nature.com/articles/s43016-022-005...   \n",
       "6   https://www.nature.com/articles/s43016-022-005...   \n",
       "7   https://www.nature.com/articles/s43016-022-005...   \n",
       "8   https://www.nature.com/articles/s43016-022-005...   \n",
       "9   https://www.nature.com/articles/s43016-022-005...   \n",
       "10  https://www.nature.com/articles/s43016-022-005...   \n",
       "11  https://www.nature.com/articles/s43016-022-004...   \n",
       "12  https://www.nature.com/articles/s43016-021-004...   \n",
       "13  https://www.nature.com/articles/s43016-021-004...   \n",
       "14  https://www.nature.com/articles/s43016-021-004...   \n",
       "15  https://www.nature.com/articles/s43016-021-003...   \n",
       "16  https://www.nature.com/articles/s43016-021-003...   \n",
       "17  https://www.nature.com/articles/s43016-021-003...   \n",
       "18  https://www.nature.com/articles/s43016-020-001...   \n",
       "19  https://www.nature.com/articles/s43016-020-001...   \n",
       "\n",
       "                               Authors (compact list)  Published  \\\n",
       "0      Bettina Wagner, Pietro Gerletti, Robert Pieper 2022-11-14   \n",
       "1     A. Charlotte Bunge, Amanda Wood, Line J. Gordon 2022-11-03   \n",
       "2   Christina C. Hicks, Jessica A. Gephart, Rosamo... 2022-10-18   \n",
       "3        Victoria Miller, Patrick Webb, Rubina Hakeem 2022-09-19   \n",
       "4         Vilma Sandström, Anna Chrysafi, Matti Kummu 2022-09-19   \n",
       "5                Lili Xia, Alan Robock, Ryan Heneghan 2022-08-15   \n",
       "6   Jaber Rahimi, Erwann Fillol, Klaus Butterbach-... 2022-07-21   \n",
       "7    Andrew J. McDonald, Balwinder-Singh, R. K. Malik 2022-07-21   \n",
       "8   Elizabeth C. Hann, Sean Overa, Robert E. Jinke... 2022-06-23   \n",
       "9                 Kajsa Resare Sahlin, Joanna Trewern 2022-06-23   \n",
       "10           Deepak K. Ray, Lindsey L. Sloat, Wei Xie 2022-05-12   \n",
       "11  Shen Yuan, Alexander M. Stuart, Patricio Grassini 2022-03-24   \n",
       "12  Hans J. M. van Grinsven, Peter Ebanyat, Hein F... 2022-01-31   \n",
       "13  Benjamin van Selm, Anita Frehner, Hannah H. E.... 2022-01-06   \n",
       "14  Peter Potapov, Svetlana Turubanova, Jocelyn Co... 2021-12-23   \n",
       "15    Mohsen Mazidi, Emily R. Leeming, Sarah E. Berry 2021-12-16   \n",
       "16  Ignacio Pérez-Domínguez, Agustin del Prado, Ma... 2021-12-13   \n",
       "17  Juan I. Rattalino Edreira, José F. Andrade, Pa... 2021-09-30   \n",
       "18       Livia Bizikova, Ephraim Nkonya, Beth Timmers 2020-10-12   \n",
       "19  W. H. Eugenie Maïga, Mohamed Porgo, Gracian Ch... 2020-10-12   \n",
       "\n",
       "                                              PDF URL  Open Access  \\\n",
       "0   https://www.nature.com/articles/s43016-022-006...  Open Access   \n",
       "1   https://www.nature.com/articles/s43016-022-006...  Open Access   \n",
       "2   https://www.nature.com/articles/s43016-022-006...  Open Access   \n",
       "3   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "4   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "5   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "6   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "7   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "8   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "9   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "10  https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "11  https://www.nature.com/articles/s43016-022-004...  Open Access   \n",
       "12  https://www.nature.com/articles/s43016-021-004...  Open Access   \n",
       "13  https://www.nature.com/articles/s43016-021-004...  Open Access   \n",
       "14  https://www.nature.com/articles/s43016-021-004...  Open Access   \n",
       "15  https://www.nature.com/articles/s43016-021-003...  Open Access   \n",
       "16  https://www.nature.com/articles/s43016-021-003...  Open Access   \n",
       "17  https://www.nature.com/articles/s43016-021-003...  Open Access   \n",
       "18  https://www.nature.com/articles/s43016-020-001...  Open Access   \n",
       "19  https://www.nature.com/articles/s43016-020-001...  Open Access   \n",
       "\n",
       "        Area of Knowledge  \n",
       "0   Agricultural Sciences  \n",
       "1   Agricultural Sciences  \n",
       "2   Agricultural Sciences  \n",
       "3   Agricultural Sciences  \n",
       "4   Agricultural Sciences  \n",
       "5   Agricultural Sciences  \n",
       "6   Agricultural Sciences  \n",
       "7   Agricultural Sciences  \n",
       "8   Agricultural Sciences  \n",
       "9   Agricultural Sciences  \n",
       "10  Agricultural Sciences  \n",
       "11  Agricultural Sciences  \n",
       "12  Agricultural Sciences  \n",
       "13  Agricultural Sciences  \n",
       "14  Agricultural Sciences  \n",
       "15  Agricultural Sciences  \n",
       "16  Agricultural Sciences  \n",
       "17  Agricultural Sciences  \n",
       "18  Agricultural Sciences  \n",
       "19  Agricultural Sciences  "
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_nature_food_open_access"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "488c0fb8-5dc2-470e-a6c6-017b3337ded4",
   "metadata": {},
   "source": [
    "### Creating the column `Text ID`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "608606ef-1071-4d19-aac9-eb0ba9d95ca5",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_nature_food_open_access['Text ID'] = nature_food_prefix + df_nature_food_open_access.index.astype(str).str.zfill(6)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9f0d5f03-0ad1-4164-bb25-664d4f0758b9",
   "metadata": {},
   "source": [
    "### Inspecting a few samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "b94fff61-3093-4347-8e1f-a528da0ad484",
   "metadata": {},
   "outputs": [],
   "source": [
    "url_sample = df_nature_food_open_access.at[19, 'URL']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "afe0cc1a-ad64-4e57-b6d6-49face28869c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.nature.com/articles/s43016-020-00172-x'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url_sample"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "1394220e-1d34-4fbb-80c4-1f1d9079ed8f",
   "metadata": {},
   "outputs": [],
   "source": [
    "document_page_sample = requests.get(url_sample)\n",
    "\n",
    "with open(f\"{output_directory}/{nature_food_prefix}_sample.html\", 'wb') as file:\n",
    "    file.write(document_page_sample.content)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "521a40ec-5e1b-4dd5-b978-c2808f0e9c73",
   "metadata": {},
   "source": [
    "### Scraping the paragraphs of the articles into TXT format"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "61aabf1d-089c-49b4-9a10-baf06e84b87d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Iterating over the rows of the column 'URL' to scrape paragraphs from each article\n",
    "for index, row in df_nature_food_open_access.iterrows():\n",
    "    url = row['URL']\n",
    "    text_id = row['Text ID']\n",
    "    page = requests.get(url).content\n",
    "    soup = BeautifulSoup(page, 'lxml')\n",
    "    \n",
    "    # Finding all <p> elements\n",
    "    paragraphs = soup.find_all('p')\n",
    "    \n",
    "    # Extract text from paragraphs\n",
    "    article_content = '\\n'.join(p.get_text(strip=True) for p in paragraphs)\n",
    "    \n",
    "    # Saving each article's content to a text file\n",
    "    with open(f\"{output_directory}/{text_id}.txt\", 'w', encoding='utf-8') as file:\n",
    "        file.write(article_content)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "05c050cf-fe3f-4518-a58d-e292582f329f",
   "metadata": {},
   "source": [
    "### Adding the column `Text` with the text extracted from each article"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "f2095026-24dc-4775-b9cc-c3bce4a5d134",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function to read the content of a TXT file\n",
    "def read_txt_file(file_path):\n",
    "    with open(file_path, 'r', encoding='utf-8') as file:\n",
    "        return file.read()\n",
    "\n",
    "# Iterating through each row in the DataFrame and add the text content\n",
    "texts = []\n",
    "for index, row in df_nature_food_open_access.iterrows():\n",
    "    text_id = row['Text ID']\n",
    "    txt_file_path = os.path.join(output_directory, f\"{text_id}.txt\")\n",
    "    if os.path.exists(txt_file_path):\n",
    "        text_content = read_txt_file(txt_file_path)\n",
    "    else:\n",
    "        text_content = None  # or you can set it to an empty string or any default value\n",
    "    texts.append(text_content)\n",
    "\n",
    "# Add the 'Text' column to the DataFrame\n",
    "df_nature_food_open_access['Text'] = texts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "e6e25439-128a-4346-b7a1-441df4a110bf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Title</th>\n",
       "      <th>URL</th>\n",
       "      <th>Authors (compact list)</th>\n",
       "      <th>Published</th>\n",
       "      <th>PDF URL</th>\n",
       "      <th>Open Access</th>\n",
       "      <th>Area of Knowledge</th>\n",
       "      <th>Text ID</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Transfer of cannabinoids into the milk of dair...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Bettina Wagner, Pietro Gerletti, Robert Pieper</td>\n",
       "      <td>2022-11-14</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000000</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A systematic scoping review of the sustainabil...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>A. Charlotte Bunge, Amanda Wood, Line J. Gordon</td>\n",
       "      <td>2022-11-03</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000001</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Rights and representation support justice acro...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Christina C. Hicks, Jessica A. Gephart, Rosamo...</td>\n",
       "      <td>2022-10-18</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-006...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000002</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Global dietary quality in 185 countries from 1...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Victoria Miller, Patrick Webb, Rubina Hakeem</td>\n",
       "      <td>2022-09-19</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000003</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Food system by-products upcycled in livestock ...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Vilma Sandström, Anna Chrysafi, Matti Kummu</td>\n",
       "      <td>2022-09-19</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000004</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Global food insecurity and famine from reduced...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Lili Xia, Alan Robock, Ryan Heneghan</td>\n",
       "      <td>2022-08-15</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000005</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>A shift from cattle to camel and goat farming ...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Jaber Rahimi, Erwann Fillol, Klaus Butterbach-...</td>\n",
       "      <td>2022-07-21</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000006</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Time management governs climate resilience and...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Andrew J. McDonald, Balwinder-Singh, R. K. Malik</td>\n",
       "      <td>2022-07-21</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000007</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>A hybrid inorganic–biological artificial photo...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Elizabeth C. Hann, Sean Overa, Robert E. Jinke...</td>\n",
       "      <td>2022-06-23</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000008</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>A systematic review of the definitions and int...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Kajsa Resare Sahlin, Joanna Trewern</td>\n",
       "      <td>2022-06-23</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000009</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Crop harvests for direct food use insufficient...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Deepak K. Ray, Lindsey L. Sloat, Wei Xie</td>\n",
       "      <td>2022-05-12</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-005...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000010</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Southeast Asia must narrow down the yield gap ...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-004...</td>\n",
       "      <td>Shen Yuan, Alexander M. Stuart, Patricio Grassini</td>\n",
       "      <td>2022-03-24</td>\n",
       "      <td>https://www.nature.com/articles/s43016-022-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000011</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Establishing long-term nitrogen response of gl...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Hans J. M. van Grinsven, Peter Ebanyat, Hein F...</td>\n",
       "      <td>2022-01-31</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000012</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Circularity in animal production requires a ch...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Benjamin van Selm, Anita Frehner, Hannah H. E....</td>\n",
       "      <td>2022-01-06</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000013</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Global maps of cropland extent and change show...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Peter Potapov, Svetlana Turubanova, Jocelyn Co...</td>\n",
       "      <td>2021-12-23</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-004...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000014</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Diet and lifestyle behaviour disruption relate...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Mohsen Mazidi, Emily R. Leeming, Sarah E. Berry</td>\n",
       "      <td>2021-12-16</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000015</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Short- and long-term warming effects of methan...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Ignacio Pérez-Domínguez, Agustin del Prado, Ma...</td>\n",
       "      <td>2021-12-13</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000016</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Spatial frameworks for robust estimation of yi...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Juan I. Rattalino Edreira, José F. Andrade, Pa...</td>\n",
       "      <td>2021-09-30</td>\n",
       "      <td>https://www.nature.com/articles/s43016-021-003...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000017</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>A scoping review of the contributions of farme...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>Livia Bizikova, Ephraim Nkonya, Beth Timmers</td>\n",
       "      <td>2020-10-12</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000018</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>A systematic review of employment outcomes fro...</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>W. H. Eugenie Maïga, Mohamed Porgo, Gracian Ch...</td>\n",
       "      <td>2020-10-12</td>\n",
       "      <td>https://www.nature.com/articles/s43016-020-001...</td>\n",
       "      <td>Open Access</td>\n",
       "      <td>Agricultural Sciences</td>\n",
       "      <td>nature_food000019</td>\n",
       "      <td>Thank you for visiting nature.com. You are usi...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                Title  \\\n",
       "0   Transfer of cannabinoids into the milk of dair...   \n",
       "1   A systematic scoping review of the sustainabil...   \n",
       "2   Rights and representation support justice acro...   \n",
       "3   Global dietary quality in 185 countries from 1...   \n",
       "4   Food system by-products upcycled in livestock ...   \n",
       "5   Global food insecurity and famine from reduced...   \n",
       "6   A shift from cattle to camel and goat farming ...   \n",
       "7   Time management governs climate resilience and...   \n",
       "8   A hybrid inorganic–biological artificial photo...   \n",
       "9   A systematic review of the definitions and int...   \n",
       "10  Crop harvests for direct food use insufficient...   \n",
       "11  Southeast Asia must narrow down the yield gap ...   \n",
       "12  Establishing long-term nitrogen response of gl...   \n",
       "13  Circularity in animal production requires a ch...   \n",
       "14  Global maps of cropland extent and change show...   \n",
       "15  Diet and lifestyle behaviour disruption relate...   \n",
       "16  Short- and long-term warming effects of methan...   \n",
       "17  Spatial frameworks for robust estimation of yi...   \n",
       "18  A scoping review of the contributions of farme...   \n",
       "19  A systematic review of employment outcomes fro...   \n",
       "\n",
       "                                                  URL  \\\n",
       "0   https://www.nature.com/articles/s43016-022-006...   \n",
       "1   https://www.nature.com/articles/s43016-022-006...   \n",
       "2   https://www.nature.com/articles/s43016-022-006...   \n",
       "3   https://www.nature.com/articles/s43016-022-005...   \n",
       "4   https://www.nature.com/articles/s43016-022-005...   \n",
       "5   https://www.nature.com/articles/s43016-022-005...   \n",
       "6   https://www.nature.com/articles/s43016-022-005...   \n",
       "7   https://www.nature.com/articles/s43016-022-005...   \n",
       "8   https://www.nature.com/articles/s43016-022-005...   \n",
       "9   https://www.nature.com/articles/s43016-022-005...   \n",
       "10  https://www.nature.com/articles/s43016-022-005...   \n",
       "11  https://www.nature.com/articles/s43016-022-004...   \n",
       "12  https://www.nature.com/articles/s43016-021-004...   \n",
       "13  https://www.nature.com/articles/s43016-021-004...   \n",
       "14  https://www.nature.com/articles/s43016-021-004...   \n",
       "15  https://www.nature.com/articles/s43016-021-003...   \n",
       "16  https://www.nature.com/articles/s43016-021-003...   \n",
       "17  https://www.nature.com/articles/s43016-021-003...   \n",
       "18  https://www.nature.com/articles/s43016-020-001...   \n",
       "19  https://www.nature.com/articles/s43016-020-001...   \n",
       "\n",
       "                               Authors (compact list)  Published  \\\n",
       "0      Bettina Wagner, Pietro Gerletti, Robert Pieper 2022-11-14   \n",
       "1     A. Charlotte Bunge, Amanda Wood, Line J. Gordon 2022-11-03   \n",
       "2   Christina C. Hicks, Jessica A. Gephart, Rosamo... 2022-10-18   \n",
       "3        Victoria Miller, Patrick Webb, Rubina Hakeem 2022-09-19   \n",
       "4         Vilma Sandström, Anna Chrysafi, Matti Kummu 2022-09-19   \n",
       "5                Lili Xia, Alan Robock, Ryan Heneghan 2022-08-15   \n",
       "6   Jaber Rahimi, Erwann Fillol, Klaus Butterbach-... 2022-07-21   \n",
       "7    Andrew J. McDonald, Balwinder-Singh, R. K. Malik 2022-07-21   \n",
       "8   Elizabeth C. Hann, Sean Overa, Robert E. Jinke... 2022-06-23   \n",
       "9                 Kajsa Resare Sahlin, Joanna Trewern 2022-06-23   \n",
       "10           Deepak K. Ray, Lindsey L. Sloat, Wei Xie 2022-05-12   \n",
       "11  Shen Yuan, Alexander M. Stuart, Patricio Grassini 2022-03-24   \n",
       "12  Hans J. M. van Grinsven, Peter Ebanyat, Hein F... 2022-01-31   \n",
       "13  Benjamin van Selm, Anita Frehner, Hannah H. E.... 2022-01-06   \n",
       "14  Peter Potapov, Svetlana Turubanova, Jocelyn Co... 2021-12-23   \n",
       "15    Mohsen Mazidi, Emily R. Leeming, Sarah E. Berry 2021-12-16   \n",
       "16  Ignacio Pérez-Domínguez, Agustin del Prado, Ma... 2021-12-13   \n",
       "17  Juan I. Rattalino Edreira, José F. Andrade, Pa... 2021-09-30   \n",
       "18       Livia Bizikova, Ephraim Nkonya, Beth Timmers 2020-10-12   \n",
       "19  W. H. Eugenie Maïga, Mohamed Porgo, Gracian Ch... 2020-10-12   \n",
       "\n",
       "                                              PDF URL  Open Access  \\\n",
       "0   https://www.nature.com/articles/s43016-022-006...  Open Access   \n",
       "1   https://www.nature.com/articles/s43016-022-006...  Open Access   \n",
       "2   https://www.nature.com/articles/s43016-022-006...  Open Access   \n",
       "3   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "4   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "5   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "6   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "7   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "8   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "9   https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "10  https://www.nature.com/articles/s43016-022-005...  Open Access   \n",
       "11  https://www.nature.com/articles/s43016-022-004...  Open Access   \n",
       "12  https://www.nature.com/articles/s43016-021-004...  Open Access   \n",
       "13  https://www.nature.com/articles/s43016-021-004...  Open Access   \n",
       "14  https://www.nature.com/articles/s43016-021-004...  Open Access   \n",
       "15  https://www.nature.com/articles/s43016-021-003...  Open Access   \n",
       "16  https://www.nature.com/articles/s43016-021-003...  Open Access   \n",
       "17  https://www.nature.com/articles/s43016-021-003...  Open Access   \n",
       "18  https://www.nature.com/articles/s43016-020-001...  Open Access   \n",
       "19  https://www.nature.com/articles/s43016-020-001...  Open Access   \n",
       "\n",
       "        Area of Knowledge            Text ID  \\\n",
       "0   Agricultural Sciences  nature_food000000   \n",
       "1   Agricultural Sciences  nature_food000001   \n",
       "2   Agricultural Sciences  nature_food000002   \n",
       "3   Agricultural Sciences  nature_food000003   \n",
       "4   Agricultural Sciences  nature_food000004   \n",
       "5   Agricultural Sciences  nature_food000005   \n",
       "6   Agricultural Sciences  nature_food000006   \n",
       "7   Agricultural Sciences  nature_food000007   \n",
       "8   Agricultural Sciences  nature_food000008   \n",
       "9   Agricultural Sciences  nature_food000009   \n",
       "10  Agricultural Sciences  nature_food000010   \n",
       "11  Agricultural Sciences  nature_food000011   \n",
       "12  Agricultural Sciences  nature_food000012   \n",
       "13  Agricultural Sciences  nature_food000013   \n",
       "14  Agricultural Sciences  nature_food000014   \n",
       "15  Agricultural Sciences  nature_food000015   \n",
       "16  Agricultural Sciences  nature_food000016   \n",
       "17  Agricultural Sciences  nature_food000017   \n",
       "18  Agricultural Sciences  nature_food000018   \n",
       "19  Agricultural Sciences  nature_food000019   \n",
       "\n",
       "                                                 Text  \n",
       "0   Thank you for visiting nature.com. You are usi...  \n",
       "1   Thank you for visiting nature.com. You are usi...  \n",
       "2   Thank you for visiting nature.com. You are usi...  \n",
       "3   Thank you for visiting nature.com. You are usi...  \n",
       "4   Thank you for visiting nature.com. You are usi...  \n",
       "5   Thank you for visiting nature.com. You are usi...  \n",
       "6   Thank you for visiting nature.com. You are usi...  \n",
       "7   Thank you for visiting nature.com. You are usi...  \n",
       "8   Thank you for visiting nature.com. You are usi...  \n",
       "9   Thank you for visiting nature.com. You are usi...  \n",
       "10  Thank you for visiting nature.com. You are usi...  \n",
       "11  Thank you for visiting nature.com. You are usi...  \n",
       "12  Thank you for visiting nature.com. You are usi...  \n",
       "13  Thank you for visiting nature.com. You are usi...  \n",
       "14  Thank you for visiting nature.com. You are usi...  \n",
       "15  Thank you for visiting nature.com. You are usi...  \n",
       "16  Thank you for visiting nature.com. You are usi...  \n",
       "17  Thank you for visiting nature.com. You are usi...  \n",
       "18  Thank you for visiting nature.com. You are usi...  \n",
       "19  Thank you for visiting nature.com. You are usi...  "
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_nature_food_open_access"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8d10d5eb-d24f-42f6-bab4-848defe53961",
   "metadata": {},
   "source": [
    "### Inspecting a few samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "44799302-20a8-47e5-ae2c-3496fbd2db7f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain\\n            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in\\n            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles\\n            and JavaScript.\\nAdvertisement\\nNature Foodvolume1,pages605–619 (2020)Cite this article\\n11kAccesses\\n15Citations\\n16Altmetric\\nMetricsdetails\\nAPublisher Correctionto this article was published on 20 October 2020\\nThis article has beenupdated\\nEngagement of youth in agriculture in low- and middle-income countries may offer opportunities to curb underemployment, urban migration, disillusionment of youth and social unrest, as well as to lift individuals and communities from poverty and hunger. Lack of education or skills training has been cited as a challenge to engage youth in the sector. Here we systematically interrogate the literature for the evaluation of skills training programmes for youth in low- and middle-income countries. Sixteen studies—nine quantitative, four qualitative and three mixed methods—from the research and grey literature documented the effects of programmes on outcomes relating to youth engagement, including job creation, income, productivity and entrepreneurship in agriculture. Although we find that skills training programmes report positive effects on our chosen outcomes, like previous systematic reviews we find the topic to chronically lack evaluation. Given the interest that donors and policymakers have in youth engagement in agriculture, our systematic review uncovers a gap in the knowledge of their effectiveness.\\nYouth in low- and middle-income countries (LMIC) disproportionately experience working poverty. In 2019, about 21% of employed youth in LMIC were living on less than US$2 a day, compared with 16% of the overall working population1. In sub-Saharan Africa, nearly 70% of working youth were found to be living in poverty; in South Asia, close to 50% were living in poverty2. Issues of youth unemployment and underemployment are linked to greater likelihood of future unemployment, decreased future job satisfaction, lower income and poorer health in adulthood3. National consequences include greater costs to support public programmes (such as public work programmes that provide temporary jobs) and indirect costs of lower earnings such as loss of investment in education4,5. Furthermore, youth underemployment is linked to disillusionment and the possibility of social unrest6.\\nThe working-age population in LMIC is predicted to double in the next 35 years7and while this presents challenges, many LMIC are currently experiencing a demographic dividend phase where there is a high ratio of working-age population to dependents. This offers unique prospects for economic development with concomitant reductions in poverty and food insecurity. Addressing unemployment and underemployment is, therefore, a major policy priority for LMIC6, and a key sector for the creation of employment opportunities, especially in Africa and Asia, is agriculture6,8,9.\\nMany people in LMIC rely on agriculture for their livelihoods (32% in 2019)10, either directly, as farmers, or indirectly in sectors that derive their existence from agricultural production8,9,11. Agricultural development is estimated to be up to 3.2 times more effective in alleviating poverty in low-income, resource-rich countries than any other sector12. Due to the close links between poverty and food insecurity13,14,15, agricultural development could also have positive consequences for the alleviation of hunger, particularly for women, as their empowerment in agriculture improves households’ food security and nutrition16,17,18.\\nHowever, there has been a declining trend of youth participation in agriculture since 2000, mainly in favour of the service sector6,19,20, which precipitates migration from rural to urban areas. Increased educational attainment for rural youth coupled with inability to rent or own land is a driver of urban migration21. In addition, the increasing ageing farmer population in rural areas exacerbates the demographic pressure on land at the expense of the youth22.\\nA further constraint on youth engagement in agriculture is a lack of education in disciplines related to agriculture or skills training23,24,25. A study among Thailand’s youth reported that 71% identified knowledge of farming practices as a pre-requisite to setting up a viable farm23. In rural Ethiopia, government initiatives to increase skills and productivity, and introduce improved and modern farming methods have generated interest among youth in joining the sector, and in Indonesia, vocational training was noted as increasing the likelihood of a successful career in agriculture26. A study in Zambia on rural youth aspirations, opinions and perceptions on agriculture documented high interest among youth in more productive forms of farming, such as the use of draught animals, electricity and the increased application of fertilizers24. Such findings challenge an assumption common in policy proposals that youth are not interested in agriculture25. Today, with the development of information and communication technology (ICT), young people have more opportunities to strengthen their skills and access relevant information and are therefore well positioned to understand market dynamics, and institutional and financial systems, enabling them to initiate and capitalize on processes of change in the agricultural sector27,28. Human capital theory predicts a positive correlation between human capital accumulation and labour productivity. On that basis, skills training can be used to improve agricultural employment outcomes29. Where governments and policy interventions support skills training for youth, there is a real possibility for entrepreneurship, a competitive economy and ultimately national growth. But, despite the implementation of skills training interventions, generally via youth employment programmes30, few specifically target agricultural skills training in LMIC and very little is known about the effectiveness of youth agricultural interventions30,31.\\nHere we systematically review skills-based training interventions that aim to increase youth engagement in agricultural employment in LMIC to better inform investment decisions made by donors and policymakers. The interventions include agriculture-related courses, on-the-job training, technical or vocational education and training in agriculture, as well as general skills training including entrepreneurship, financial literacy and life skills for engagement in agriculture. The outcomes of interest we started out with were: employment along an agricultural value chain; employment in agribusiness; engagement in contract farming; development of agricultural entrepreneurship; agricultural business performance (productivity, profit, income, marketing rate); involvement in agricultural extension service provision. After data extraction, the outcomes of interest found in the selected studies are jobs created in the agricultural sector, self-employment and entrepreneurship, provision of and employment in extension services, profit/income/earnings from an agricultural activity or job, farm productivity, and the accessibility of employment opportunities in the sector. These outcomes pertain to the categories of jobs that can be found along the agricultural value chain.\\nWe found among the studies yielded from the systematic literature search that skills training interventions reported employment in agriculture, agribusiness or agriculture-related activities, development of agricultural entrepreneurship, agricultural business performances (productivity, profit, income) and involvement in agricultural extension service provision for young participants.However, we also found a chronic lack of evaluation of the effectiveness of interventions designed to enhance agricultural opportunities and engagement for young people in LMIC, a finding previously shown31.\\nSixteen studies were identified for review based on a priori inclusion and exclusion criteria (Fig.1) detailed in our Preferred Reporting Items for Systematic Reviews and Meta-Analyses Protocol, PRISMA-P (Supplementary Material1, summarized inMethodsand published on Open Science Framework,https://osf.io/bhegq//).\\nInclusion criteria were youth as the target population; inclusion of one or more outcome of interest (employment along an agricultural value chain; employment in agribusiness; engagement in contract farming; development of agricultural entrepreneurship; agricultural business performance (productivity, profit, income, marketing rate); involvement in agricultural extension service provision); agriculture sector as field of study; skills training as an intervention; publication in English or French between 1990 and 2019; original research or review of existing research or institutional reports; targets low- and middle-income country or countries as area(s) of study (see list of World Bank country classifications (Supplementary Table1); a clear and well-accepted methodology (studies were excluded if there was no clear method on sampling, data analysis or discussion of results). Studies meeting the inclusion criteria and targeting mixed group (youth and other demographic groups) were also retained in the search strategy. A double-blind title and abstract screening were performed on 4,789 articles that were uploaded to systematic review software, Covidence, for title and abstract screening. Each article was reviewed by two independent reviewers and discrepancies were resolved by a third independent author within the team. After title and abstract screening, 261 articles remained. From title and abstract screening, 16 articles met a priori inclusion criteria.\\nA data extraction template (Supplementary Table2) was used to document all information of interest from each of the 16 studies, overviewed in Table1.\\nEleven of the studies were based in Africa32,33,34,35,36,37,38,39,40,41,42and five in Asia43,44,45,46,47. Twelve of the studies were published in peer-reviewed journals33,34,35,36,39,40,41,42,44,45,46,47and the rest originated from the grey literature, including one dissertation38, one report37and two working papers32,43.\\nWith regard to the study design, nine of the included studies were quantitative32,33,34,35,36,37,43,44,45, four were qualitative41,42,46,47and three used mixed non-experimental38,39,40methods. Only one study used randomized control trial (RCT) as a study design method of evaluation32. Quasi-experimental impact methods (difference-in-differences (DID) and propensity score matching (PSM)) and quantitative non-experimental methods (statistical and econometric methods) were used in two33,43and six34,35,36,37,44,45studies, respectively. Nine of the included studies relied on survey data32,33,34,35,36,37,43,44,45, one study used data from interviews47, one study used data from focus groups42and the rest of the studies used mixed sources of data38,39,40(Supplementary Table3).\\nTable2collates information from the selected studies on the basis of types of intervention and participant characteristics. Technical education/training35,41,42,46and vocational training37,40,44,45constituted half of the interventions (four, each); youth programmes, agriculture-related courses and on-the-job training were identified as interventions in three33,34,38, two39,47and one36of the studies, respectively, and the remainder of the studies combined two types of intervention32,43. Twelve of the interventions were implemented through public policies33,34,35,37,38,39,41,42,43,44,45,47; non-governmental organizations (NGOs) and a mix of institutions (public and private) were each identified as implementers in two32,36and one46of the studies, respectively, and one study reported intervention implemented by an international institution40.\\nNine of the studies solely targeted youth32,33,34,35,37,38,43,45,46, and seven targeted mixed groups of youth and others36,39,40,41,42,44,47. In fourteen studies, the participants were from all genders. In nine of the studies, participants were a mixed group of those already and not yet engaged in agriculture32,34,37,39,41,42,43,44,46; in five of the studies, participants were already engaged in agriculture before receiving skills training interventions35,36,37,45,47; there was not enough information to determine whether the participants were already engaged in agriculture in two studies33,40. Six of the studies indicated that the participants resided in rural areas33,34,35,36,46,47, while participants located in urban areas and in both rural and urban areas were identified in four32,38,40,45and five37,39,41,43,44of the studies, respectively; there was not enough information to determine the location of the participants in one42study. The population targeted in the studies was both educated and non-educated youth. Among the nine studies32,33,34,35,37,38,43,45,46that focused exclusively on youth, two targeted youth with a secondary education background34,46, one45targeted youth with a university background and six32,33,35,37,38,43of the studies targeted youth with a mixed educational background.\\nWe evaluated the risk of bias of the included studies based on a previous approach48. The domains of risk retained are (1) the sampling technique used for the study, (2) the type of intervention, (3) the choice of the area of study, (4) the population targeted, (5) the method of data collection, (6) the method of data analysis, (7) the measurement of outcome and (8) the statistical significance of the effect. For each domain of risk, the criteria evaluated were defined and rated by their relevance for assessing the effectiveness of the interventions. Supplementary Table4summarizes the criteria of each domain of risk and its assessment and rating.\\nUsing this scale, 15% of our included studies are at low risk of bias, 60% at moderate risk of bias and the remaining 25% at serious risk of bias. The outcome of the risk of bias assessment of the included studies in this systematic review is presented in Table3.\\nThe risk of bias assessment process highlighted differences in focus, methods used and standards of evidence across the included studies. Weaknesses in study design, survey methods and method of evaluation of the impact of the interventions were common in most of the studies (with the exception of the studies ranked at low risk of bias), leading to weak results and limited generalizability.\\nThe youth employment outcomes of interest to this systematic review are job creation, self-employment, engagement in entrepreneurship, provision of extension services, productivity of the farm/agriculture-related activities, profits/income, and job search or employment opportunity in agriculture-related activities. Here we elaborate on the study design and risk of bias of all studies, and highlight the effects on outcomes of interest for a selection of low and moderate risk studies.\\nEight studies32,38,39,40,41,42,43,45looked at job creation in agriculture as an outcome. Among those studies, three are quantitative studies32,43,45, two are qualitative studies41,42and three are mixed-methods studies38,39,40.\\nIn one quantitative study, deemed at low risk of bias (Table3), 1,700 workers and 1,500 firms were followed over four years to compare the effects of offering workers vocational training and offering firms wage subsidies to train workers on-the-job (firm training) in Uganda32. The results showed that both interventions allowed participants to acquire sector-specific skills and firm-specific skills leading to higher employment rates post-training for each type of worker, but the effect was greater for vocational training compared with firm training (21% versus 14% post-training employment rate) and their total earnings rose by more compared with the firm-training intervention (34% versus 20%). The qualitative studies41,42, although not designed to assess the effectiveness of an intervention, highlighted a link between skills training and employment outcome. However, both studies were deemed at serious risk of bias. A mixed-methods study38on youth programmes in Ghana showed that about 86.4% of young people still pursued maize farming a year after exiting the Youth in Agriculture Programme (YIAP). This public intervention was implemented to address youth unemployment in Ghana with the goal of getting young people to engage in the agricultural sector. The four main components of the programme were crops/block farm, livestock and poultry, fisheries/aquaculture, and agribusiness. The study focuses on evaluating the crops/block farm component. The crops cultivated under the YIAP include maize (seed and grain), sorghum, soybean, tomato and onion. This study is ranked at moderate risk of bias.\\nSix studies36,39,41,45,46,47indicated that skills training interventions resulted in self-employment in agriculture. Out of these studies, two studies are quantitative36,45, three are qualitative41,46,47and one is a mixed-methods study39.\\nIn one quantitative study36, self-employment was stimulated by a skills training radio campaign on growing orange-fleshed sweet potatoes in Ghana, Tanzania, Burkina Faso and Uganda. A survey of the local communities where the radio campaign was run found that households that reported hearing the educational radio campaign in Ghana, Tanzania, Burkina Faso and Uganda were 8.9, 2.3, 1.7 and 1.1 times more likely, respectively, to engage in growing orange-fleshed sweet potatoes, than households that did not. This study is deemed at moderate risk of bias.\\nFive studies34,38,39,41,42showed that skills training interventions encourage youth engagement or entrepreneurship in agriculture. Among these studies, one is quantitative34, two are qualitative41,42and two are mixed-methods studies38,39. In the quantitative study, a youth programme including agriculture content (training in livestock production, crop production and dairy farming) in South Africa indicated that youth engagement or self-employment in agriculture is eight times higher when agricultural programmes that specifically target the youth are implemented compared with when agricultural programmes are not available. This study is deemed at moderate risk of bias. Regarding the mixed-methods studies, one study38, deemed at moderate risk of bias with youth programme (YIAP in Ghana) as intervention, showed that after exiting the programme, 86.4% of beneficiaries were still involved in farming within a year. The qualitative studies were deemed at serious risk of bias.\\nTwo studies35,41found that skills training interventions lead to higher productivity of the farms. One of the studies is quantitative35and the other is qualitative41. In the quantitative study, estimated to be at moderate risk of bias, the National Agricultural Extension and Research Liaison Services (NAERLS) rural youth extension programmes (RUYEP) helped 84.2% of beneficiaries achieve yields that exceed one tonne per hectare for maize in Nigeria, compared with 66% of non-participants35. The qualitative study41, outlined in Table1, is deemed at serious risk of bias.\\nTen studies32,33,35,38,40,41,42,43,44,47looked at profit/income earning of the farm as an outcome. Among those studies, five are quantitative32,33,35,43,44, three are qualitative41,42,47and two38,40are mixed-methoda studies. In one of the quantitative studies, the Training for Rural Economic Empowerment (TREE) programme increased beneficiaries’ income by US$787 compared with non-beneficiaries over the 2011–2014 programme implementation period33. This study is deemed at low risk of bias. Another quantitative study44, deemed at moderate risk of bias, found that the continued adopters of beekeeping and mushroom growing had increased their family income by 49% and 24%, respectively. The three qualitative studies, not described here but outlined in Table1, are deemed at serious risk of bias41,42,47. The mixed-methods study40showed that the creation of a company that recycled livestock by-product (bone crafts and soap production) allowed vulnerable women and youths to earn an additional US$44.6 from bone crafts and US$50.2 from soap production weekly. This study is at moderate risk of bias.\\nThree studies39,41,42investigated the effect of skills training on this outcome. One study is a mixed-methods design39and two41,42are qualitative. All of these studies, not described here but outlined in Table1, are deemed at serious risk of bias.\\nOne study39investigated on the effects of skills interventions on provision of agricultural extension service and found that the majority of graduates who benefited from student–farmer attachment and/or the Supervised Student Enterprise Project (SSEP) were engaged in extension work. This study, outlined in Table1, is deemed at serious risk of bias.\\nTwo studies39,47used agriculture-related courses as interventions.One of these studies is a mixed-methods study39and the other is qualitative47. The mixed-methods study investigated several outcomes in agriculture, namely, job creation, entrepreneurship, self-employment, provision of agricultural extension service and job search opportunity, which were found to improve with the skills training interventions. The interventions consisted of introducing innovations in agricultural training curricula (community engagement and agri-enterprise development) at Gulu University in Uganda. The community engagement took the form of a one year (or less) placement of undergraduate students to work with smallholder farmers and farmer groups within a 10\\u2009km radius of the university. The agri-enterprise development consisted of having the students design business plans; the best plans were rewarded with start-up capital. The employment rate among the graduates was 84% six months after graduation and increased to 90% after one year; less than 2% of the graduates created their own businesses. The qualitative study47investigated two outcomes in agriculture, self-employment and income, which were found to increase after skills training on ready food mixes, maize products and mango products. The two studies are deemed to be at serious risk of bias.\\nFour studies35,41,42,46used technical education/training as interventions. Only one of these studies is quantitative35; the others are qualitative41,42,46. The quantitative study35investigated productivity and income of the farm, and found both to increase after the intervention. The NAERLS RUYEP objectives are to provide technical advisory services to boost agricultural production and raise living standards of the youth. The results showed that the intervention allowed 84.2% of beneficiaries to achieve yields that exceed one tonne per hectare for maize in Nigeria, compared with 66% of non-participants. This study is deemed at moderate risk of bias. Among the qualitative studies, one46looked at self-employment as an outcome and found a positive association with the intervention. The other two qualitative studies are deemed of serious risk of bias.\\nYouth programmes are programmes that target youth and train them in either specific skills (agricultural skills, ICT skills and so on) or broad skills (decision-making skills, business skills and so on) to enhance their employability. These have been used as interventions in three studies33,34,38. One of these studies is mixed methods38and the two others are quantitative33,34. The mixed-methods study38investigated the following outcomes in agriculture: job creation, engagement and income; a positive association was found between youth programme and both engagement and income. The results showed that about 86.4% of young people still pursued maize farming one year after exiting the programme and the mean income of GH¢758 obtained by beneficiaries was found to be greater than the national mean annual per capita income of GH¢734. Among the two quantitative studies33,34, one investigated the income of beneficiaries33and the other34looked at engagement in agriculture; both found a positive effect of the intervention on their outcome. The study that investigated the income of beneficiaries as an outcome revealed that the TREE programme increased beneficiaries’ income by US$787 compared with non-beneficiaries over the 2011–2014 programme implementation period33. In the other study34, a youth programme including agriculture content (training in livestock production, crop production and dairy farming) in South Africa indicated that youth engagement or self-employment in agriculture is eight times higher when agricultural programmes that specifically target the youth are implemented compared with when agricultural programmes are not available. Given that all three studies are at moderate or low risk of bias, we can conclude that the findings suggest that youth programmes have the potential to influence youth engagement in agriculture.\\nOnly one study36looked at on-the-job training as an intervention. The outcome investigated is self-employment, on which the intervention had a positive effect. The results showed that households that reported listening to an educational radio campaign in Ghana, Tanzania, Burkina Faso and Uganda were 8.9, 2.3, 1.7 and 1.1 times more likely, respectively, to engage in growing orange-fleshed sweet potatoes, than households that did not. The study was deemed at moderate risk of bias.\\nVocational training has been used as an intervention by four studies37,40,44,45. Among these studies, three are quantitative37,44,45and one is a mixed-methods study40. One quantitative study44investigated income as an outcome, on which positive effects of the intervention were found in India. The findings indicated that vocational training programmes have resulted in continued adoption of beekeeping and mushroom cultivation enterprises by 20% and 51% of trained farmers, respectively, and increased their family income by 49% and 24%, respectively. The second quantitative study investigated job creation and self-employment as outcomes and found positive links with the training45. The results of the study highlighted that vocational training in agriculture in Iran resulted in employment of more than half of graduates. The third quantitative study found a positive effect of the intervention on job creation, the sole outcome it had investigated37. The study showed that vocational training for a youth employment programme in Ghana resulted in the creation of 16,383 jobs in agribusiness. All four studies are deemed at moderate risk of bias (Table3); however, the use of descriptive methods in some of these studies preclude us from concluding that they are effective in improving employment outcomes for youth in the agricultural sector.\\nOne study43investigated the combination of vocational training and technical training as an intervention. The outcomes investigated are job creation and income, on which the intervention had a positive effect. The study indicated that vocational training and technical training in agriculture (poultry technician) resulted in an increase in employment of 34.2% among the 41 beneficiaries who were trained as poultry technicians in Nepal. This study is deemed at low risk of bias, suggesting that combining vocational training and technical training may be a way of improving job prospects and income for youth in the agricultural sector.\\nOne study32investigated the combination of vocational training and on-the-job training as an intervention. The outcomes investigated are job creation and earnings, on which the intervention had a positive effect. The results showed that both interventions allowed participants to acquire sector-specific skills and firm-specific skills, leading to higher employment rates post-training for vocational-trained workers compared with firm-trained workers (21% versus 14% post-training employment rate) and their total earnings rose by more compared with the firm-trained workers (34% versus 20%). This study is deemed at low risk of bias.\\nTen studies out of the 16 overviewed in Table1presented information on the duration of training. Eight of these have programmes that last one year or less. The remaining studies indicated a training duration between two and five years. This suggests that training programmes predominantly have a relatively short-term duration, which is consistent with many interventions taking the form of technical and vocational education/training. The popularity of technical and vocational/education training as a model of intervention may be due to the relatively short-term nature of the training, or due to the nature of technical and vocational training, which is well suited for out-of-school youth, which are found in large numbers in LMIC49.\\nIssues facing youth engagement in agriculture today are relatively well documented, including educational attainment, matrimonial status, gender, household size, parental income and occupation, membership in social organization, access to ICT, land tenure system and access to state-run agricultural youth programmes50,51,52. This present systematic review, which focused solely on interventions to engage youth in agriculture, yielded a limited set of studies—nine quantitative, four qualitative and three mixed-methods studies—so generalizable conclusions are difficult to draw. The risk of bias assessment yielded three studies32,33,43deemed at low risk of bias, nine studies34,35,36,37,38,40,44,45,46deemed at moderate risk of bias and four studies deemed at serious of risk bias39,41,42,47.\\nThe results of our systematic review generally are in line with those found by the systematic review of Kluve et al.53on interventions to improve the labour market outcomes of youth. That systematic review of 107 interventions, including skills training, in 31 countries, found small positive effects for promoting entrepreneurship and skills training—especially integrated skills training programmes—but not for employment services and subsidized employment.\\nOur systematic review also demonstrated that in general, skills interventions seeking to motivate youth’s engagement in agriculture do not undergo a thorough evaluation for effectiveness, with hard outcomes related to employment. Our selected studies included case studies and qualitative methods, which are not adequate methods of evaluating impact and effectiveness of interventions. Only one study used an RCT32. The two studies relying on a quasi-experimental approach used DID and PSM methods33,43. Indeed, the results of the risk of bias assessment indicated the studies relying on RCT and quasi-experimental impact evaluation methods were at low risk of bias. However, these study designs are expensive to conduct. We found that of the studies that evaluate interventions, the majority did not use state-of-the-art impact evaluation methods. This has been corroborated by other studies30,31, showing a chronic lack of evaluation of interventions that aim to provide agricultural skills to youth.\\nTraining on ICT is an important aspect for attracting and retaining youth in the agricultural sector46. ICT offers a method of delivering training to a large number of farmers, which could enhance the performance of the youth already in agriculture and attract new youth to the sector36. Radio campaigns have been shown to be effective in spurring adoption and consumption of orange-fleshed potatoes in Ghana, Tanzania, Burkina Faso and Uganda36. A study conducted in the Philippines found that ICT training helps motivate secondary school students whose parents are engaged in agriculture to work within the sector, especially when combined with offline activities such as exposure and hands-on experience as well as creative and motivational actitivites46.\\nIt is important to note that heterogeneity in gender and education are not accounted for in the analysis of the impacts of education on youth participation in agriculture. Our systematic review revealed that most of the included studies failed to address the effectiveness of targeting the population of interest—educated and uneducated youth. Illiteracy and gender heterogeneity were not addressed in the included studies. Indeed, no studies assessed the effects of training interventions on illiterate youth. This calls for investigations to focus on this vulnerable group of society, which represent about 25% of youth in sub-Saharan Africa and 11% in Southern Asia54. Failing to account for such variation in the background of the youth participants limits the ability to assess the effectiveness of skills training interventions.\\nThe absence of robust research and lack of effective evaluation of the available data on the effectiveness of agricultural youth employment interventions has notable consequences on potential investment. Ultimately, the commitment of policymakers is necessary to ensure the sustainability and success of interventions to boost youth’s engagement in agriculture. It is encouraging that the majority of interventions (12 studies out of 16) studied originated from public policy, compared with three originating from non-public policy programmes (NGOs, international institution) and one from mixed policies (public and non-public policies). However, to provide a compelling basis on which to convince governments and donors to fund future interventions, as well as encourage young people to partake in training, cost-effectiveness analysis and estimates of returns on investment in training programmes is necessary. Indeed, a 2018 stocktaking of the evidence on the effectiveness of youth employment interventions in Africa found that for the agricultural sector in particular, “there is very little literature and virtually no evaluation evidence to inform policymakers about what types of interventions can improve the prospects of young people in the [agricultural] sector”31. Our study supports this conclusion. Moreover, to ensure that the skills training provides long-term opportunities for youth, it is crucial to establish a periodic follow-up to assess how trainees are performing after completion of a training programme. This aspect was missing in most of the interventions reviewed in this systematic review, yet it is important to check that the youth who engage in agriculture after receiving skills training are still involved and thrive in their agriculture-related business in the long term.\\nIn summary, there is a need to foster youth skills training programmes and more importantly to evaluate more rigourously these programmes so that knowledge on good practices may be generated and transferred from one developing country to another. Estimates of returns to investment of agricultural skills training programmes are warranted as they could provide governments and donors with the evidence and cost-based analysis to continue and increase support for such programmes. Interventions also need to account for heterogeneity in gender and educational background of the youth to foster sustainability in agricultural value chains, inform inclusive policy design and ultimately contribute to reducing poverty and food insecurity in LMIC.\\nThis systematic review was prepared following guidelines from Petticrew and Roberts55. The approach comprises five steps: identifying the research question; identifying relevant studies; study selection; extracting and charting the data; and collating, summarizing and reporting the results. The protocol for this study was registered on the Open Science Framework before study selection and can be accessed athttps://osf.io/bhegq//. The guiding question for this systematic review was: What are the effects of skills training interventions on educated and non-educated youth employment outcomes in agricultural value chains, agribusiness or contract farming in LMIC? The inclusion and exclusion criteria to identify and then select the relevant studies are shown in Table4.\\nRegarding the risk of bias assessment, each study was assessed following the criteria of the eight domains of risk of bias we considered. The maximum score a study can obtain in terms of minimizing all domains of risk of bias is 23 stars, which is 100% of the stars. A study is deemed to be at low risk of bias across all domains if its total score is in the interval 75–100%. If the total score is in the interval 50–75%, the study is said to be at moderate risk of bias across all domains. A study is at serious risk of bias if its score falls within the interval 25–50%. When the total score ranges from 0 to 25%, the study is deemed to be at critical risk of bias across all domains. See Supplementary Table4for details on the criteria used.\\nAn exhaustive search strategy was developed and tested in CAB Abstracts to identify all available research pertaining to the effects of skills training interventions on educated and non-educated youth employment outcomes in agriculture in LMIC. Search terms were developed to address variations of the key concepts in the research question: skills training, youth, employment or engagement, and agriculture. Searches were performed on 9 May 2019 in the following electronic databases: CAB Abstracts (access via OVID); Web of Science Core Collection (access via Web of Science); EconLit (access via ProQuest); Agricola (access via OVID); and Scopus (access via Elsevier). Full search strategies for each database, including grey literature, can be accessed in their entirety athttps://osf.io/xv56k/.\\nA comprehensive search of grey literature sources was also conducted. A list of the resources that were searched can be found athttps://osf.io/xv56k/. The grey literature searches were performed using custom web-scraping scripts. The search strings were tested per website before initiating web-scraping. An existing Google Chrome extension was needed to scrape dynamically generated websites.\\nThe results from the databases and the grey literature searches were combined and de-duplicated using a Python script. Duplicates were detected using title, abstract and same year of publication, where year of publication was a match, where title cosine similarity was greater than 85%, and where abstracts cosine similarity was greater than 80% or one of the abstracts (or both) was empty. When duplicates were found, the results from the databases and the grey literature searches were combined and duplicates were removed.\\nFollowing de-duplication, each citation was analysed using a machine-learning model. The model added more than 30 new metadata fields, such as identifying populations, geographies, interventions and outcomes of interest. This allowed for accelerated identification of potential articles for exclusion at the title/abstract screening stage.\\nSystematic review software, Covidence, was used for both title/abstract and full-text screening decision-making with two independent reviewers evaluating each item. Citations were included in this study if they met all of the inclusion criteria noted above. Studies that did not meet all the inclusion criteria were excluded. Exclusion criteria were the inverse of the inclusion criteria. Each citation that met one of the exclusion criteria at the title, abstract or full-text screening phases were excluded. Studies included in the full-text screening stage were those that met all inclusion criteria and none of the exclusion criteria, or those whose eligibility could not be established during title/abstract screening. Reasons for exclusion were documented at the full-text screening phase.\\nThe retrieval of hundreds of PDFs for full-text screening was done with a combination of automated and manual methods. For the automated method, a Python script was created that would handle the tasks of PDF discovery, download and file renaming using Google Scholar. The script read the bibliographic data from an Excel spreadsheet and then executed a script to retrieve the full-text PDF. If the article is spotted in the search results, the download link is clicked, and the article will be auto-renamed and marked as being downloaded. Manual methods were employed for those items that were not retrieved using the script.\\nA total of 245 records were identified for full-text screening. This screening process led to the identification of 16 studies that were considered adequate regarding the content and methodological rigour. The PRISMA flow diagram (Fig.1) shows the steps followed during the screening process and the number of items that resulted after each step.\\nData extraction was based on interventions and outcomes established in the research question and exclusion criteria. The data extraction focused on the outcomes of the studies, the methods used to obtain the outcomes, and the validity and reliability of those methods using a data-extraction form. To reduce risk of bias related to the extracted data, two separate researchers extracted data from each included study in the full-text review step. When disagreements occurred between researchers on data extracted from a study, a third researcher was engaged to resolve conflict by extracting data again from the study and the results were compared with those found previously. In total, 31 conflicts were solved among the 261 reviews. The critical appraisal of individual sources of evidence gave an indication of the strength of evidence provided and informed the standards followed for this systematic review.\\nFurther information on research design is available in theNature Research Reporting Summarylinked to this article.\\nThe data that support the findings of this study are available from the corresponding author upon reasonable request.\\nThe code used in this study is available upon request.\\nAn amendment to this paper has been published and can be accessed via a link at the top of the paper.\\nWorld Employment and Social Outlook 2019: Trends for Youth(International Labour Office, 2019).\\nWorld Youth Report: Youth and the 2030 Agenda for Sustainable Development(United Nations, 2018);https://www.un.org/development/desa/youth/wp-content/uploads/sites/21/2018/12/WorldYouthReport-2030Agenda.pdf\\nKilimani, N. Youth employment in developing economies: evidence on policies and interventions.IDS Bull.48, 13–32 (2017).\\nGoogle Scholar\\nGlobal Employment Trends for Youth: Special Issue on the Impact of the Global Economic Crisis on Youth(ILO, 2010).\\nPieters, J.Youth Employment in LMIC(Institute of Labor Economics, 2013).\\nYeboah, F. K. & Jayne, T. S. Africa’s evolving employment trends.J. Dev. Stud.54, 803–832 (2018).\\nArticleGoogle Scholar\\nWorld Population Prospects 2019: HighlightsST/ESA/SER. A/423 (UN DESA, 2019).\\nEllis, F.Rural Livelihoods and Diversity in LMIC(Oxford Univ. Press, 2000).\\nPellegrini, L. & Tasciotti, L. Crop diversification, dietary diversity and agricultural income: empirical evidence from eight LMIC.Can. J. Dev. Stud.35, 211–227 (2014).\\nArticleGoogle Scholar\\nEmployment by sex and age—ILO modelled estimates.ILOSTAT Database(International Labour Organization, accessed 14 February 2020);https://ilostat.ilo.org/data\\nCarletto, G. et al. Rural income generating activities in LMIC: re-assessing the evidence.Electron. J. Agric. Dev. Econ.4, 146–193 (2007).\\nGoogle Scholar\\nChristiaensen, L., Demery, L. & Kuhl, J. The (evolving) role of agriculture in poverty reduction—an empirical perspective.J. Dev. Econ.96, 239–254 (2011).\\nArticleGoogle Scholar\\nReducing Poverty and Hunger: The Critical Role of Financing for Food, Agriculture and Rural Development(FAO, 2002).\\nEicher, C. K. InStrategies for African Development(eds Berg, R. J. & Whitaker, J. S.) 242–275 (Univ. California Press, 1986).\\nde Alan, B. & MH, S. Linkages between poverty, food security and undernutrition: evidence from China and India.China Agric. Econ. Rev.7, 655–667 (2015).\\nArticleGoogle Scholar\\nSraboni, E., Malapit, H. J., Quisumbing, A. R. & Ahmed, A. U. Women’s empowerment in agriculture: what role for food security in Bangladesh?World Dev.61, 11–52 (2014).\\nArticleGoogle Scholar\\nSharaunga, S., Mudhara, M. & Bogale, A. Effects of ‘women empowerment’ on household food security in rural KwaZulu‐Natal province.Dev. Policy Rev.34, 223–252 (2016).\\nArticleGoogle Scholar\\nMalapit, H. J. L. & Quisumbing, A. R. What dimensions of women’s empowerment in agriculture matter for nutrition in Ghana?Food Policy52, 54–63 (2015).\\nArticleGoogle Scholar\\nSumberg, J., Anyidoho, N. A., Leavy, J., te Lintelo, D. J. & Wellard, K. Introduction: the young people and agriculture ‘problem’ in Africa.IDS Bull.43, 1–8 (2012).\\nArticleGoogle Scholar\\nBezu, S. & Holden, S. Are rural youth in Ethiopia abandoning agriculture?World Dev.64, 259–272 (2014).\\nArticleGoogle Scholar\\nTadele, G. & Gella, A. A.Becoming a Young Farmer in Ethiopia: Processes and ChallengesWorking Paper 83 (Future Agricultures, 2014).\\nLindsjö, K., Mulwafu, W., Andersson Djurfeldt, A. & Joshua, M. K. Generational dynamics of agricultural intensification in Malawi: challenges for the youth and elderly smallholder farmers.Int. J. Agric. Sustain.https://doi.org/10.1080/14735903.2020.1721237(2020).\\nSalvago, M. R., Phiboon, K., Faysse, N. & Nguyen, T. P. L. Young people’s willingness to farm under present and improved conditions in Thailand.Outlook Agric.48, 282–291 (2019).\\nArticleGoogle Scholar\\nDaum, T. Of bulls and bulbs: aspirations, opinions and perceptions of rural adolescents and youth in Zambia.Dev. Pract.29, 882–897 (2018).\\nArticleGoogle Scholar\\nYeboah, T. et al. Hard work and hazard: young people and agricultural commercialisation in Africa.J. Rural Stud.76, 142–151 (2020).\\nArticleGoogle Scholar\\nLeavy, J. & Hossain, N. Who wants to farm? Youth aspirations, opportunities and rising food prices.IDS Working Papershttps://doi.org/10.1111/j.2040-0209.2014.00439.x(2014).\\nRoser, M. & Ortiz-Ospina, E. Global education.Our World in Datahttps://ourworldindata.org/global-education(2016).\\nGyimah-Brempong, K. & Kimenyi, M. S.Youth Policy and the Future of African Development(African Growth Initiative, 2013).\\nFoster, A. D. & Rosenzweig, M. R. Learning by doing and learning from others: human capital and technical change in agriculture.J. Political Econ.103, 1176–1209 (1995).\\nArticleGoogle Scholar\\nEichhorst, W. & Rinne, U.An Assessment of the Youth Employment Inventory and Implications for Germany’s Development Policy(Institute of Labor Economics, 2015).\\nBetcherman, G. & Khan, T. Jobs for Africa’s expanding youth cohort: a stocktaking of employment prospects and policy interventions.IZA J. Dev. Migr.8, 13 (2018).\\nArticleGoogle Scholar\\nAlfonsi, L. et al.Tackling Youth Unemployment: Evidence from a Labour Market Experiment in Uganda(STICERD, LSE, 2017).\\nLachaud, M. A., Bravo-Ureta, B. E., Fiala, N. & Gonzalez, S. P. The impact of agri-business skills training in Zimbabwe: an evaluation of the Training for Rural Economic Empowerment (TREE) programme.J. Dev. Effect.10, 373–391 (2018).\\nArticleGoogle Scholar\\nCheteni, P. Youth participation in agriculture in the Nkonkobe District Municipality, South Africa.J. Hum. Ecol.55, 207–213 (2016).\\nArticleGoogle Scholar\\nGambo Akpoko, J. & Kudi, T. M. Impact assessment of university-based rural youth agricultural extension out-reach program in selected villages of Kaduna-State, Nigeria.J. Apppl. Sci.7, 3292–3296 (2007).\\nArticleGoogle Scholar\\nHudson, H. E., Leclair, M., Pelletier, B. & Sullivan, B. Using radio and interactive ICTs to improve food security among smallholder farmers in sub-Saharan Africa.Telecomm. Policy41, 670–684 (2017).\\nArticleGoogle Scholar\\nGhana—Job Creation and Skills Development: Main Report(World Bank, 2009).\\nBaah, C.Assessment of the Youth in Agriculture Programme in Ejura-Sekyedumase District(Kwame Nkrumah Univ. Science and Technology, 2014).\\nOdongo, W., Kalule, S. W., Kule, E. K., Ndyomugyenyi, E. & Ongeng, D. Responsiveness of agricultural training curricula in African universities to labour market needs: the case of Gulu University in Uganda.African J. Rural Dev.2, 67–76 (2017).\\nGoogle Scholar\\nKinyanjui, W. & Noor, M. S. From waste to employment opportunities and wealth creation: a case study of utilization of livestock by-products in Hargeisa, Somaliland.Food Nutr. Sci.4, 1287–1292 (2013).\\nGoogle Scholar\\nLatopa, A.-L. A. & Rashid, S. N. S. A. The impacts of integrated youth training farm as a capacity building center for youth agricultural empowerment in Kwara State, Nigeria.Mediterr. J. Soc. Sci.6, 524–532 (2015).\\nGoogle Scholar\\nShoulders, C. W., Barrick, R. K. & Myers, B. E. An assessment of the impact of internship programs in the agricultural technical schools of Egypt as perceived by participant groups.J. Int. Agric. Ext. Educ.18, 18–29 (2011).\\nGoogle Scholar\\nChakravarty, S., Lundberg, M., Nikolov, P. & Zenker, J.The Role of Training Programs for Youth Employment in Nepal: Impact Evaluation Report on the Employment Fund(World Bank, 2016).\\nSingh, K., Peshin, R. & Saini, S. K. Evaluation of the agricultural vocational training programmes conducted by the Krishi Vigyan Kendras (Farm Science Centres) in Indian Punjab.J. Agric. Rural Dev. Trop. Subtrop.111, 65–77 (2010).\\nGoogle Scholar\\nKhosravipour, B. & Soleimanpour, M. R. Comparison of students’ entrepreneurship spirit in agricultural scientific-applied higher education centers of Iran.Am. J. Agric. Environ. Sci.12, 1012–1015 (2012).\\nGoogle Scholar\\nManalo, J. A. IV, Balmeo, K. P., Domingo, O. C. & Saludez, F. M. Young allies of agricultural extension: the infomediary campaign in Aurora, Philippines.Philipp. J. Crop Sci.39, 30–40 (2014).\\nGoogle Scholar\\nChannal, G. P., Kotikal, Y. K. & Pattar, P. S. Empowering rural women as a successful entrepreneur—through Krishi Vigyan Kendra.Agric. Update12, 498–501 (2017).\\nArticleGoogle Scholar\\nSterne, J. A. et al. ROBINS-I: a tool for assessing risk of bias in non-randomised studies of interventions.BMJ355, i4919 (2016).\\nArticleGoogle Scholar\\nHow Does the Short-Term Training Program Contribute to Skills Development in Bangladesh? A Tracer Study of the Short-Term Training GraduatesSouth Asia Region, Education Global Practice Discussion Paper (World Bank, 2015).\\nNnadi, F. N. & Akwiwu, C. D. Determinants of youths’ participation in rural agriculture in Imo State, Nigeria.J. Appl. Sci.8, 328–333 (2008).\\nArticleADSGoogle Scholar\\nAkpan, S. B., Patrick, I. V., James, S. U. & Agom, D. I. Determinants of decision and participation of rural youth in agricultural production: a case study of youth in southern region of Nigeria.Russ. J. Agric. Socio-Econ. Sci.7, 35–48 (2015).\\nGoogle Scholar\\nAdesugba, M. & Mavrotas, G.Youth Employment, Agricultural Transformation, and Rural Labour Dynamics in Nigeria(International Food Policy Research Institute, 2016).\\nKluve, J. et al.Interventions to Improve the Labour Market Outcomes of Youth: A Systematic Review of Training, Entrepreneurship Promotion, Employment Services and Subsidized Employment Interventions(The Campbell Collaboration, 2017).\\nLiteracy Rates Continues to Rise from One Generation to the Next(UNESCO Institute for Statistics, 2017).\\nPetticrew, M. & Roberts, H.Systematic Reviews in the Social Sciences: A Practical Guide(John Wiley & Sons, 2008).\\nDownload references\\nWe thank J.-A. Porciello and M. Eber-Rose for helpful comments on earlier drafts of this manuscript. We gratefully acknowledge funding support from Bundesministerium für Wirtschaftliche Zusammenarbeit und Entwicklung (Federal Ministry for Economic Cooperation and Development in Germany) and The Bill and Melinda Gates Foundation as part of Ceres2030: Sustainable Solutions to End Hunger, a project administered by Cornell University, USA.\\nUniversité Norbert Zongo, Koudougou, Burkina Faso\\nW. H. Eugenie Maïga\\nUniversité Thomas Sankara, Ouagadougou, Burkina Faso\\nMohamed Porgo,\\xa0Pam Zahonogo,\\xa0Doubahan Adeline Coulibaly\\xa0&\\xa0Salimata Traoré\\nUniversité Nationale d’Agriculture de Kétou, Kétou, Benin\\nCocou Jaurès Amegnaglo\\nInstitute of Development Studies–University of Sussex, Brighton, UK\\nJustin Flynn\\nCentre Universitaire Polytechnique de Kaya, Kaya, Burkina Faso\\nWindinkonté Seogo\\nUniversity of Minnesota, Saint Paul, MN, USA\\nJulia A. Kelly\\nInformation Training and Outreach Centre for Africa, Centurion, South Africa\\nGracian Chimwaza\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nYou can also search for this author inPubMedGoogle Scholar\\nW.H.E.M., M.P. and P.Z. developed the research question. J.A.K. and G.C. conducted the literature search. All authors drafted the PRISMA-P protocol for this study. W.H.E.M., M.P., P.Z, C.J.A, D.A.C., J.F., W.S. and S.T. conducted the full-text reviews and drafted the paper, and all authors contributed to the writing.\\nCorrespondence toW. H. Eugenie Maïga.\\nThe authors declare no competing interests.\\nPublisher’s noteSpringer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\nSupplementary Tables 1–4.\\nPRISMA protocol for the systematic review.\\nOpen AccessThis article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visithttp://creativecommons.org/licenses/by/4.0/.\\nReprints and permissions\\nMaïga, W.H.E., Porgo, M., Zahonogo, P.et al.A systematic review of employment outcomes from youth skills training programmes in agriculture in low- and middle-income countries.Nat Food1, 605–619 (2020). https://doi.org/10.1038/s43016-020-00172-x\\nDownload citation\\nReceived:23 December 2019\\nAccepted:16 September 2020\\nPublished:12 October 2020\\nIssue Date:October 2020\\nDOI:https://doi.org/10.1038/s43016-020-00172-x\\nAnyone you share the following link with will be able to read this content:\\nSorry, a shareable link is not currently available for this article.\\n\\nProvided by the Springer Nature SharedIt content-sharing initiative\\nCABI Agriculture and Bioscience(2024)\\nNature Food(2024)\\nDe Economist(2024)\\nAgriculture & Food Security(2023)\\nNature Machine Intelligence(2020)\\nCollection\\nAdvertisement\\nNature Food (Nat Food)ISSN2662-1355(online)\\n© 2024 Springer Nature Limited\\nSign up for theNature Briefing: Anthropocenenewsletter — what matters in anthropocene research, free to your inbox weekly.'"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_nature_food_open_access.at[19, 'Text']"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9d161738-8de5-4180-81eb-123d5fe00f79",
   "metadata": {},
   "source": [
    "### Exporting to a file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "0fdbd3fa-9410-4cd1-80ab-9a396798dbb0",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_nature_food_open_access.to_json(f\"{output_directory}/{nature_food_prefix}_open_access.jsonl\", orient='records', lines=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7ab41f75-0a5b-4c12-8e44-5129fc1e4b15",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
