<html lang="en-US" class="toolbar-stuck"><head>
      <meta name="citation_pii" content="S0047248420301135">
<meta name="citation_id" content="102852">
<meta name="citation_issn" content="0047-2484">
<meta name="citation_volume" content="146">
<meta name="citation_publisher" content="Academic Press">
<meta name="citation_firstpage" content="102852">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_journal_title" content="Journal of Human Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.jhevol.2020.102852">
<meta name="dc.identifier" content="10.1016/j.jhevol.2020.102852">
<meta name="citation_article_type" content="Full-length article">
<meta property="og:description" content="Wallace's Line demarcates a biogeographical boundary between the Indomalaya and Australasian ecoregions. Most placental mammalian genera, for example,â€¦">
<meta property="og:image" content="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420X0008X-cov150h.gif">
<meta name="citation_title" content="Mitogenomics of macaques (Macaca) across Wallace's Line in the context of modern human dispersals">
<meta property="og:title" content="Mitogenomics of macaques (Macaca) across Wallace's Line in the context of modern human dispersals">
<meta name="citation_publication_date" content="2020/09/01">
<meta name="citation_online_date" content="2020/08/08">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOCACHE,NOODP,NOYDIR">
      <title>Mitogenomics of macaques (Macaca) across Wallace's Line in the context of modern human dispersals - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S0047248420301135">
      <meta name="tdm-reservation" content="1">
      <meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">
      <meta property="og:type" content="article">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team">
      <script async="" src="https://cdn.pendo.io/agent/static/d6c1d995-bc7e-4e53-77f1-2ea4ecbb9565/pendo.js"></script><script type="text/javascript">(function newRelicBrowserProSPA() {
  ;
  window.NREUM || (NREUM = {});
  NREUM.init = {
    privacy: {
      cookies_enabled: false
    },
    ajax: {
      deny_list: ["bam-cell.nr-data.net"]
    }
  };
  ;
  NREUM.loader_config = {
    accountID: "2128461",
    trustKey: "2038175",
    agentID: "1118783207",
    licenseKey: "7ac4127487",
    applicationID: "814814845"
  };
  ;
  NREUM.info = {
    beacon: "bam.nr-data.net",
    errorBeacon: "bam.nr-data.net",
    licenseKey: "7ac4127487",
    applicationID: "814814845",
    sa: 1
  };
  ; /*! For license information please see nr-loader-spa-1.238.0.min.js.LICENSE.txt */
  (() => {
    "use strict";

    var e,
      t,
      r = {
        5763: (e, t, r) => {
          r.d(t, {
            P_: () => f,
            Mt: () => p,
            C5: () => s,
            DL: () => v,
            OP: () => T,
            lF: () => D,
            Yu: () => y,
            Dg: () => h,
            CX: () => c,
            GE: () => b,
            sU: () => _
          });
          var n = r(8632),
            i = r(9567);
          const o = {
              beacon: n.ce.beacon,
              errorBeacon: n.ce.errorBeacon,
              licenseKey: void 0,
              applicationID: void 0,
              sa: void 0,
              queueTime: void 0,
              applicationTime: void 0,
              ttGuid: void 0,
              user: void 0,
              account: void 0,
              product: void 0,
              extra: void 0,
              jsAttributes: {},
              userAttributes: void 0,
              atts: void 0,
              transactionName: void 0,
              tNamePlain: void 0
            },
            a = {};
          function s(e) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            if (!a[e]) throw new Error("Info for ".concat(e, " was never set"));
            return a[e];
          }
          function c(e, t) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            a[e] = (0, i.D)(t, o), (0, n.Qy)(e, a[e], "info");
          }
          var u = r(7056);
          const d = () => {
              const e = {
                blockSelector: "[data-nr-block]",
                maskInputOptions: {
                  password: !0
                }
              };
              return {
                allow_bfcache: !0,
                privacy: {
                  cookies_enabled: !0
                },
                ajax: {
                  deny_list: void 0,
                  block_internal: !0,
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                distributed_tracing: {
                  enabled: void 0,
                  exclude_newrelic_header: void 0,
                  cors_use_newrelic_header: void 0,
                  cors_use_tracecontext_headers: void 0,
                  allowed_origins: void 0
                },
                session: {
                  domain: void 0,
                  expiresMs: u.oD,
                  inactiveMs: u.Hb
                },
                ssl: void 0,
                obfuscate: void 0,
                jserrors: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                metrics: {
                  enabled: !0
                },
                page_action: {
                  enabled: !0,
                  harvestTimeSeconds: 30
                },
                page_view_event: {
                  enabled: !0
                },
                page_view_timing: {
                  enabled: !0,
                  harvestTimeSeconds: 30,
                  long_task: !1
                },
                session_trace: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                harvest: {
                  tooManyRequestsDelay: 60
                },
                session_replay: {
                  enabled: !1,
                  harvestTimeSeconds: 60,
                  sampleRate: .1,
                  errorSampleRate: .1,
                  maskTextSelector: "*",
                  maskAllInputs: !0,
                  get blockClass() {
                    return "nr-block";
                  },
                  get ignoreClass() {
                    return "nr-ignore";
                  },
                  get maskTextClass() {
                    return "nr-mask";
                  },
                  get blockSelector() {
                    return e.blockSelector;
                  },
                  set blockSelector(t) {
                    e.blockSelector += ",".concat(t);
                  },
                  get maskInputOptions() {
                    return e.maskInputOptions;
                  },
                  set maskInputOptions(t) {
                    e.maskInputOptions = {
                      ...t,
                      password: !0
                    };
                  }
                },
                spa: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                }
              };
            },
            l = {};
          function f(e) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            if (!l[e]) throw new Error("Configuration for ".concat(e, " was never set"));
            return l[e];
          }
          function h(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            l[e] = (0, i.D)(t, d()), (0, n.Qy)(e, l[e], "config");
          }
          function p(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            var r = f(e);
            if (r) {
              for (var n = t.split("."), i = 0; i < n.length - 1; i++) if ("object" != typeof (r = r[n[i]])) return;
              r = r[n[n.length - 1]];
            }
            return r;
          }
          const g = {
              accountID: void 0,
              trustKey: void 0,
              agentID: void 0,
              licenseKey: void 0,
              applicationID: void 0,
              xpid: void 0
            },
            m = {};
          function v(e) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            if (!m[e]) throw new Error("LoaderConfig for ".concat(e, " was never set"));
            return m[e];
          }
          function b(e, t) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            m[e] = (0, i.D)(t, g), (0, n.Qy)(e, m[e], "loader_config");
          }
          const y = (0, n.mF)().o;
          var w = r(385),
            A = r(6818);
          const x = {
              buildEnv: A.Re,
              bytesSent: {},
              queryBytesSent: {},
              customTransaction: void 0,
              disabled: !1,
              distMethod: A.gF,
              isolatedBacklog: !1,
              loaderType: void 0,
              maxBytes: 3e4,
              offset: Math.floor(w._A?.performance?.timeOrigin || w._A?.performance?.timing?.navigationStart || Date.now()),
              onerror: void 0,
              origin: "" + w._A.location,
              ptid: void 0,
              releaseIds: {},
              session: void 0,
              xhrWrappable: "function" == typeof w._A.XMLHttpRequest?.prototype?.addEventListener,
              version: A.q4,
              denyList: void 0
            },
            E = {};
          function T(e) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            if (!E[e]) throw new Error("Runtime for ".concat(e, " was never set"));
            return E[e];
          }
          function _(e, t) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            E[e] = (0, i.D)(t, x), (0, n.Qy)(e, E[e], "runtime");
          }
          function D(e) {
            return function (e) {
              try {
                const t = s(e);
                return !!t.licenseKey && !!t.errorBeacon && !!t.applicationID;
              } catch (e) {
                return !1;
              }
            }(e);
          }
        },
        9567: (e, t, r) => {
          r.d(t, {
            D: () => i
          });
          var n = r(50);
          function i(e, t) {
            try {
              if (!e || "object" != typeof e) return (0, n.Z)("Setting a Configurable requires an object as input");
              if (!t || "object" != typeof t) return (0, n.Z)("Setting a Configurable requires a model to set its initial properties");
              const r = Object.create(Object.getPrototypeOf(t), Object.getOwnPropertyDescriptors(t)),
                o = 0 === Object.keys(r).length ? e : r;
              for (let a in o) if (void 0 !== e[a]) try {
                "object" == typeof e[a] && "object" == typeof t[a] ? r[a] = i(e[a], t[a]) : r[a] = e[a];
              } catch (e) {
                (0, n.Z)("An error occurred while setting a property of a Configurable", e);
              }
              return r;
            } catch (e) {
              (0, n.Z)("An error occured while setting a Configurable", e);
            }
          }
        },
        6818: (e, t, r) => {
          r.d(t, {
            Re: () => i,
            gF: () => o,
            q4: () => n
          });
          const n = "1.238.0",
            i = "PROD",
            o = "CDN";
        },
        385: (e, t, r) => {
          r.d(t, {
            FN: () => a,
            IF: () => u,
            Nk: () => l,
            Tt: () => s,
            _A: () => o,
            il: () => n,
            pL: () => c,
            v6: () => i,
            w1: () => d
          });
          const n = "undefined" != typeof window && !!window.document,
            i = "undefined" != typeof WorkerGlobalScope && ("undefined" != typeof self && self instanceof WorkerGlobalScope && self.navigator instanceof WorkerNavigator || "undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope && globalThis.navigator instanceof WorkerNavigator),
            o = n ? window : "undefined" != typeof WorkerGlobalScope && ("undefined" != typeof self && self instanceof WorkerGlobalScope && self || "undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope && globalThis),
            a = "" + o?.location,
            s = /iPad|iPhone|iPod/.test(navigator.userAgent),
            c = s && "undefined" == typeof SharedWorker,
            u = (() => {
              const e = navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);
              return Array.isArray(e) && e.length >= 2 ? +e[1] : 0;
            })(),
            d = Boolean(n && window.document.documentMode),
            l = !!navigator.sendBeacon;
        },
        1117: (e, t, r) => {
          r.d(t, {
            w: () => o
          });
          var n = r(50);
          const i = {
            agentIdentifier: "",
            ee: void 0
          };
          class o {
            constructor(e) {
              try {
                if ("object" != typeof e) return (0, n.Z)("shared context requires an object as input");
                this.sharedContext = {}, Object.assign(this.sharedContext, i), Object.entries(e).forEach(e => {
                  let [t, r] = e;
                  Object.keys(i).includes(t) && (this.sharedContext[t] = r);
                });
              } catch (e) {
                (0, n.Z)("An error occured while setting SharedContext", e);
              }
            }
          }
        },
        8e3: (e, t, r) => {
          r.d(t, {
            L: () => d,
            R: () => c
          });
          var n = r(8325),
            i = r(1284),
            o = r(4322),
            a = r(3325);
          const s = {};
          function c(e, t) {
            const r = {
              staged: !1,
              priority: a.p[t] || 0
            };
            u(e), s[e].get(t) || s[e].set(t, r);
          }
          function u(e) {
            e && (s[e] || (s[e] = new Map()));
          }
          function d() {
            let e = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : "",
              t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : "feature";
            if (u(e), !e || !s[e].get(t)) return a(t);
            s[e].get(t).staged = !0;
            const r = [...s[e]];
            function a(t) {
              const r = e ? n.ee.get(e) : n.ee,
                a = o.X.handlers;
              if (r.backlog && a) {
                var s = r.backlog[t],
                  c = a[t];
                if (c) {
                  for (var u = 0; s && u < s.length; ++u) l(s[u], c);
                  (0, i.D)(c, function (e, t) {
                    (0, i.D)(t, function (t, r) {
                      r[0].on(e, r[1]);
                    });
                  });
                }
                delete a[t], r.backlog[t] = null, r.emit("drain-" + t, []);
              }
            }
            r.every(e => {
              let [t, r] = e;
              return r.staged;
            }) && (r.sort((e, t) => e[1].priority - t[1].priority), r.forEach(e => {
              let [t] = e;
              a(t);
            }));
          }
          function l(e, t) {
            var r = e[1];
            (0, i.D)(t[r], function (t, r) {
              var n = e[0];
              if (r[0] === n) {
                var i = r[1],
                  o = e[3],
                  a = e[2];
                i.apply(o, a);
              }
            });
          }
        },
        8325: (e, t, r) => {
          r.d(t, {
            A: () => c,
            ee: () => u
          });
          var n = r(8632),
            i = r(2210),
            o = r(5763);
          class a {
            constructor(e) {
              this.contextId = e;
            }
          }
          var s = r(3117);
          const c = "nr@context:".concat(s.a),
            u = function e(t, r) {
              var n = {},
                s = {},
                d = {},
                f = !1;
              try {
                f = 16 === r.length && (0, o.OP)(r).isolatedBacklog;
              } catch (e) {}
              var h = {
                on: g,
                addEventListener: g,
                removeEventListener: function (e, t) {
                  var r = n[e];
                  if (!r) return;
                  for (var i = 0; i < r.length; i++) r[i] === t && r.splice(i, 1);
                },
                emit: function (e, r, n, i, o) {
                  !1 !== o && (o = !0);
                  if (u.aborted && !i) return;
                  t && o && t.emit(e, r, n);
                  for (var a = p(n), c = m(e), d = c.length, l = 0; l < d; l++) c[l].apply(a, r);
                  var f = b()[s[e]];
                  f && f.push([h, e, r, a]);
                  return a;
                },
                get: v,
                listeners: m,
                context: p,
                buffer: function (e, t) {
                  const r = b();
                  if (t = t || "feature", h.aborted) return;
                  Object.entries(e || {}).forEach(e => {
                    let [n, i] = e;
                    s[i] = t, t in r || (r[t] = []);
                  });
                },
                abort: l,
                aborted: !1,
                isBuffering: function (e) {
                  return !!b()[s[e]];
                },
                debugId: r,
                backlog: f ? {} : t && "object" == typeof t.backlog ? t.backlog : {}
              };
              return h;
              function p(e) {
                return e && e instanceof a ? e : e ? (0, i.X)(e, c, () => new a(c)) : new a(c);
              }
              function g(e, t) {
                n[e] = m(e).concat(t);
              }
              function m(e) {
                return n[e] || [];
              }
              function v(t) {
                return d[t] = d[t] || e(h, t);
              }
              function b() {
                return h.backlog;
              }
            }(void 0, "globalEE"),
            d = (0, n.fP)();
          function l() {
            u.aborted = !0, u.backlog = {};
          }
          d.ee || (d.ee = u);
        },
        5546: (e, t, r) => {
          r.d(t, {
            E: () => n,
            p: () => i
          });
          var n = r(8325).ee.get("handle");
          function i(e, t, r, i, o) {
            o ? (o.buffer([e], i), o.emit(e, t, r)) : (n.buffer([e], i), n.emit(e, t, r));
          }
        },
        4322: (e, t, r) => {
          r.d(t, {
            X: () => o
          });
          var n = r(5546);
          o.on = a;
          var i = o.handlers = {};
          function o(e, t, r, o) {
            a(o || n.E, i, e, t, r);
          }
          function a(e, t, r, i, o) {
            o || (o = "feature"), e || (e = n.E);
            var a = t[o] = t[o] || {};
            (a[r] = a[r] || []).push([e, i]);
          }
        },
        3239: (e, t, r) => {
          r.d(t, {
            bP: () => s,
            iz: () => c,
            m$: () => a
          });
          var n = r(385);
          let i = !1,
            o = !1;
          try {
            const e = {
              get passive() {
                return i = !0, !1;
              },
              get signal() {
                return o = !0, !1;
              }
            };
            n._A.addEventListener("test", null, e), n._A.removeEventListener("test", null, e);
          } catch (e) {}
          function a(e, t) {
            return i || o ? {
              capture: !!e,
              passive: i,
              signal: t
            } : !!e;
          }
          function s(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2],
              n = arguments.length > 3 ? arguments[3] : void 0;
            window.addEventListener(e, t, a(r, n));
          }
          function c(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2],
              n = arguments.length > 3 ? arguments[3] : void 0;
            document.addEventListener(e, t, a(r, n));
          }
        },
        3117: (e, t, r) => {
          r.d(t, {
            a: () => n
          });
          const n = (0, r(4402).Rl)();
        },
        4402: (e, t, r) => {
          r.d(t, {
            Ht: () => u,
            M: () => c,
            Rl: () => a,
            ky: () => s
          });
          var n = r(385);
          const i = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";
          function o(e, t) {
            return e ? 15 & e[t] : 16 * Math.random() | 0;
          }
          function a() {
            const e = n._A?.crypto || n._A?.msCrypto;
            let t,
              r = 0;
            return e && e.getRandomValues && (t = e.getRandomValues(new Uint8Array(31))), i.split("").map(e => "x" === e ? o(t, ++r).toString(16) : "y" === e ? (3 & o() | 8).toString(16) : e).join("");
          }
          function s(e) {
            const t = n._A?.crypto || n._A?.msCrypto;
            let r,
              i = 0;
            t && t.getRandomValues && (r = t.getRandomValues(new Uint8Array(31)));
            const a = [];
            for (var s = 0; s < e; s++) a.push(o(r, ++i).toString(16));
            return a.join("");
          }
          function c() {
            return s(16);
          }
          function u() {
            return s(32);
          }
        },
        7056: (e, t, r) => {
          r.d(t, {
            Bq: () => n,
            Hb: () => o,
            oD: () => i
          });
          const n = "NRBA",
            i = 144e5,
            o = 18e5;
        },
        7894: (e, t, r) => {
          function n() {
            return Math.round(performance.now());
          }
          r.d(t, {
            z: () => n
          });
        },
        7243: (e, t, r) => {
          r.d(t, {
            e: () => o
          });
          var n = r(385),
            i = {};
          function o(e) {
            if (e in i) return i[e];
            if (0 === (e || "").indexOf("data:")) return {
              protocol: "data"
            };
            let t;
            var r = n._A?.location,
              o = {};
            if (n.il) t = document.createElement("a"), t.href = e;else try {
              t = new URL(e, r.href);
            } catch (e) {
              return o;
            }
            o.port = t.port;
            var a = t.href.split("://");
            !o.port && a[1] && (o.port = a[1].split("/")[0].split("@").pop().split(":")[1]), o.port && "0" !== o.port || (o.port = "https" === a[0] ? "443" : "80"), o.hostname = t.hostname || r.hostname, o.pathname = t.pathname, o.protocol = a[0], "/" !== o.pathname.charAt(0) && (o.pathname = "/" + o.pathname);
            var s = !t.protocol || ":" === t.protocol || t.protocol === r.protocol,
              c = t.hostname === r.hostname && t.port === r.port;
            return o.sameOrigin = s && (!t.hostname || c), "/" === o.pathname && (i[e] = o), o;
          }
        },
        50: (e, t, r) => {
          function n(e, t) {
            "function" == typeof console.warn && (console.warn("New Relic: ".concat(e)), t && console.warn(t));
          }
          r.d(t, {
            Z: () => n
          });
        },
        2587: (e, t, r) => {
          r.d(t, {
            N: () => c,
            T: () => u
          });
          var n = r(8325),
            i = r(5546),
            o = r(8e3),
            a = r(3325);
          const s = {
            stn: [a.D.sessionTrace],
            err: [a.D.jserrors, a.D.metrics],
            ins: [a.D.pageAction],
            spa: [a.D.spa],
            sr: [a.D.sessionReplay, a.D.sessionTrace]
          };
          function c(e, t) {
            const r = n.ee.get(t);
            e && "object" == typeof e && (Object.entries(e).forEach(e => {
              let [t, n] = e;
              void 0 === u[t] && (s[t] ? s[t].forEach(e => {
                n ? (0, i.p)("feat-" + t, [], void 0, e, r) : (0, i.p)("block-" + t, [], void 0, e, r), (0, i.p)("rumresp-" + t, [Boolean(n)], void 0, e, r);
              }) : n && (0, i.p)("feat-" + t, [], void 0, void 0, r), u[t] = Boolean(n));
            }), Object.keys(s).forEach(e => {
              void 0 === u[e] && (s[e]?.forEach(t => (0, i.p)("rumresp-" + e, [!1], void 0, t, r)), u[e] = !1);
            }), (0, o.L)(t, a.D.pageViewEvent));
          }
          const u = {};
        },
        2210: (e, t, r) => {
          r.d(t, {
            X: () => i
          });
          var n = Object.prototype.hasOwnProperty;
          function i(e, t, r) {
            if (n.call(e, t)) return e[t];
            var i = r();
            if (Object.defineProperty && Object.keys) try {
              return Object.defineProperty(e, t, {
                value: i,
                writable: !0,
                enumerable: !1
              }), i;
            } catch (e) {}
            return e[t] = i, i;
          }
        },
        1284: (e, t, r) => {
          r.d(t, {
            D: () => n
          });
          const n = (e, t) => Object.entries(e || {}).map(e => {
            let [r, n] = e;
            return t(r, n);
          });
        },
        4351: (e, t, r) => {
          r.d(t, {
            P: () => o
          });
          var n = r(8325);
          const i = () => {
            const e = new WeakSet();
            return (t, r) => {
              if ("object" == typeof r && null !== r) {
                if (e.has(r)) return;
                e.add(r);
              }
              return r;
            };
          };
          function o(e) {
            try {
              return JSON.stringify(e, i());
            } catch (e) {
              try {
                n.ee.emit("internal-error", [e]);
              } catch (e) {}
            }
          }
        },
        3960: (e, t, r) => {
          r.d(t, {
            K: () => a,
            b: () => o
          });
          var n = r(3239);
          function i() {
            return "undefined" == typeof document || "complete" === document.readyState;
          }
          function o(e, t) {
            if (i()) return e();
            (0, n.bP)("load", e, t);
          }
          function a(e) {
            if (i()) return e();
            (0, n.iz)("DOMContentLoaded", e);
          }
        },
        8632: (e, t, r) => {
          r.d(t, {
            EZ: () => u,
            Qy: () => c,
            ce: () => o,
            fP: () => a,
            gG: () => d,
            mF: () => s
          });
          var n = r(7894),
            i = r(385);
          const o = {
            beacon: "bam.nr-data.net",
            errorBeacon: "bam.nr-data.net"
          };
          function a() {
            return i._A.NREUM || (i._A.NREUM = {}), void 0 === i._A.newrelic && (i._A.newrelic = i._A.NREUM), i._A.NREUM;
          }
          function s() {
            let e = a();
            return e.o || (e.o = {
              ST: i._A.setTimeout,
              SI: i._A.setImmediate,
              CT: i._A.clearTimeout,
              XHR: i._A.XMLHttpRequest,
              REQ: i._A.Request,
              EV: i._A.Event,
              PR: i._A.Promise,
              MO: i._A.MutationObserver,
              FETCH: i._A.fetch
            }), e;
          }
          function c(e, t, r) {
            let i = a();
            const o = i.initializedAgents || {},
              s = o[e] || {};
            return Object.keys(s).length || (s.initializedAt = {
              ms: (0, n.z)(),
              date: new Date()
            }), i.initializedAgents = {
              ...o,
              [e]: {
                ...s,
                [r]: t
              }
            }, i;
          }
          function u(e, t) {
            a()[e] = t;
          }
          function d() {
            return function () {
              let e = a();
              const t = e.info || {};
              e.info = {
                beacon: o.beacon,
                errorBeacon: o.errorBeacon,
                ...t
              };
            }(), function () {
              let e = a();
              const t = e.init || {};
              e.init = {
                ...t
              };
            }(), s(), function () {
              let e = a();
              const t = e.loader_config || {};
              e.loader_config = {
                ...t
              };
            }(), a();
          }
        },
        7956: (e, t, r) => {
          r.d(t, {
            N: () => i
          });
          var n = r(3239);
          function i(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] && arguments[1],
              r = arguments.length > 2 ? arguments[2] : void 0,
              i = arguments.length > 3 ? arguments[3] : void 0;
            return void (0, n.iz)("visibilitychange", function () {
              if (t) return void ("hidden" == document.visibilityState && e());
              e(document.visibilityState);
            }, r, i);
          }
        },
        1214: (e, t, r) => {
          r.d(t, {
            em: () => b,
            u5: () => j,
            QU: () => O,
            _L: () => I,
            Gm: () => H,
            Lg: () => L,
            BV: () => G,
            Kf: () => K
          });
          var n = r(8325),
            i = r(3117);
          const o = "nr@original:".concat(i.a);
          var a = Object.prototype.hasOwnProperty,
            s = !1;
          function c(e, t) {
            return e || (e = n.ee), r.inPlace = function (e, t, n, i, o) {
              n || (n = "");
              const a = "-" === n.charAt(0);
              for (let s = 0; s < t.length; s++) {
                const c = t[s],
                  u = e[c];
                d(u) || (e[c] = r(u, a ? c + n : n, i, c, o));
              }
            }, r.flag = o, r;
            function r(t, r, n, s, c) {
              return d(t) ? t : (r || (r = ""), nrWrapper[o] = t, function (e, t, r) {
                if (Object.defineProperty && Object.keys) try {
                  return Object.keys(e).forEach(function (r) {
                    Object.defineProperty(t, r, {
                      get: function () {
                        return e[r];
                      },
                      set: function (t) {
                        return e[r] = t, t;
                      }
                    });
                  }), t;
                } catch (e) {
                  u([e], r);
                }
                for (var n in e) a.call(e, n) && (t[n] = e[n]);
              }(t, nrWrapper, e), nrWrapper);
              function nrWrapper() {
                var o, a, d, l;
                try {
                  a = this, o = [...arguments], d = "function" == typeof n ? n(o, a) : n || {};
                } catch (t) {
                  u([t, "", [o, a, s], d], e);
                }
                i(r + "start", [o, a, s], d, c);
                try {
                  return l = t.apply(a, o);
                } catch (e) {
                  throw i(r + "err", [o, a, e], d, c), e;
                } finally {
                  i(r + "end", [o, a, l], d, c);
                }
              }
            }
            function i(r, n, i, o) {
              if (!s || t) {
                var a = s;
                s = !0;
                try {
                  e.emit(r, n, i, t, o);
                } catch (t) {
                  u([t, r, n, i], e);
                }
                s = a;
              }
            }
          }
          function u(e, t) {
            t || (t = n.ee);
            try {
              t.emit("internal-error", e);
            } catch (e) {}
          }
          function d(e) {
            return !(e && e instanceof Function && e.apply && !e[o]);
          }
          var l = r(2210),
            f = r(385);
          const h = {},
            p = f._A.XMLHttpRequest,
            g = "addEventListener",
            m = "removeEventListener",
            v = "nr@wrapped:".concat(n.A);
          function b(e) {
            var t = function (e) {
              return (e || n.ee).get("events");
            }(e);
            if (h[t.debugId]++) return t;
            h[t.debugId] = 1;
            var r = c(t, !0);
            function i(e) {
              r.inPlace(e, [g, m], "-", o);
            }
            function o(e, t) {
              return e[1];
            }
            return "getPrototypeOf" in Object && (f.il && y(document, i), y(f._A, i), y(p.prototype, i)), t.on(g + "-start", function (e, t) {
              var n = e[1];
              if (null !== n && ("function" == typeof n || "object" == typeof n)) {
                var i = (0, l.X)(n, v, function () {
                  var e = {
                    object: function () {
                      if ("function" != typeof n.handleEvent) return;
                      return n.handleEvent.apply(n, arguments);
                    },
                    function: n
                  }[typeof n];
                  return e ? r(e, "fn-", null, e.name || "anonymous") : n;
                });
                this.wrapped = e[1] = i;
              }
            }), t.on(m + "-start", function (e) {
              e[1] = this.wrapped || e[1];
            }), t;
          }
          function y(e, t) {
            let r = e;
            for (; "object" == typeof r && !Object.prototype.hasOwnProperty.call(r, g);) r = Object.getPrototypeOf(r);
            for (var n = arguments.length, i = new Array(n > 2 ? n - 2 : 0), o = 2; o < n; o++) i[o - 2] = arguments[o];
            r && t(r, ...i);
          }
          var w = "fetch-",
            A = w + "body-",
            x = ["arrayBuffer", "blob", "json", "text", "formData"],
            E = f._A.Request,
            T = f._A.Response,
            _ = "prototype";
          const D = {};
          function j(e) {
            const t = function (e) {
              return (e || n.ee).get("fetch");
            }(e);
            if (!(E && T && f._A.fetch)) return t;
            if (D[t.debugId]++) return t;
            function r(e, r, i) {
              var o = e[r];
              "function" == typeof o && (e[r] = function () {
                var e,
                  r = [...arguments],
                  a = {};
                t.emit(i + "before-start", [r], a), a[n.A] && a[n.A].dt && (e = a[n.A].dt);
                var s = o.apply(this, r);
                return t.emit(i + "start", [r, e], s), s.then(function (e) {
                  return t.emit(i + "end", [null, e], s), e;
                }, function (e) {
                  throw t.emit(i + "end", [e], s), e;
                });
              });
            }
            return D[t.debugId] = 1, x.forEach(e => {
              r(E[_], e, A), r(T[_], e, A);
            }), r(f._A, "fetch", w), t.on(w + "end", function (e, r) {
              var n = this;
              if (r) {
                var i = r.headers.get("content-length");
                null !== i && (n.rxSize = i), t.emit(w + "done", [null, r], n);
              } else t.emit(w + "done", [e], n);
            }), t;
          }
          const C = {},
            N = ["pushState", "replaceState"];
          function O(e) {
            const t = function (e) {
              return (e || n.ee).get("history");
            }(e);
            return !f.il || C[t.debugId]++ || (C[t.debugId] = 1, c(t).inPlace(window.history, N, "-")), t;
          }
          var S = r(3239);
          const P = {},
            R = ["appendChild", "insertBefore", "replaceChild"];
          function I(e) {
            const t = function (e) {
              return (e || n.ee).get("jsonp");
            }(e);
            if (!f.il || P[t.debugId]) return t;
            P[t.debugId] = !0;
            var r = c(t),
              i = /[?&](?:callback|cb)=([^&#]+)/,
              o = /(.*)\.([^.]+)/,
              a = /^(\w+)(\.|$)(.*)$/;
            function s(e, t) {
              if (!e) return t;
              const r = e.match(a),
                n = r[1];
              return s(r[3], t[n]);
            }
            return r.inPlace(Node.prototype, R, "dom-"), t.on("dom-start", function (e) {
              !function (e) {
                if (!e || "string" != typeof e.nodeName || "script" !== e.nodeName.toLowerCase()) return;
                if ("function" != typeof e.addEventListener) return;
                var n = (a = e.src, c = a.match(i), c ? c[1] : null);
                var a, c;
                if (!n) return;
                var u = function (e) {
                  var t = e.match(o);
                  if (t && t.length >= 3) return {
                    key: t[2],
                    parent: s(t[1], window)
                  };
                  return {
                    key: e,
                    parent: window
                  };
                }(n);
                if ("function" != typeof u.parent[u.key]) return;
                var d = {};
                function l() {
                  t.emit("jsonp-end", [], d), e.removeEventListener("load", l, (0, S.m$)(!1)), e.removeEventListener("error", f, (0, S.m$)(!1));
                }
                function f() {
                  t.emit("jsonp-error", [], d), t.emit("jsonp-end", [], d), e.removeEventListener("load", l, (0, S.m$)(!1)), e.removeEventListener("error", f, (0, S.m$)(!1));
                }
                r.inPlace(u.parent, [u.key], "cb-", d), e.addEventListener("load", l, (0, S.m$)(!1)), e.addEventListener("error", f, (0, S.m$)(!1)), t.emit("new-jsonp", [e.src], d);
              }(e[0]);
            }), t;
          }
          const k = {};
          function H(e) {
            const t = function (e) {
              return (e || n.ee).get("mutation");
            }(e);
            if (!f.il || k[t.debugId]) return t;
            k[t.debugId] = !0;
            var r = c(t),
              i = f._A.MutationObserver;
            return i && (window.MutationObserver = function (e) {
              return this instanceof i ? new i(r(e, "fn-")) : i.apply(this, arguments);
            }, MutationObserver.prototype = i.prototype), t;
          }
          const z = {};
          function L(e) {
            const t = function (e) {
              return (e || n.ee).get("promise");
            }(e);
            if (z[t.debugId]) return t;
            z[t.debugId] = !0;
            var r = t.context,
              i = c(t),
              a = f._A.Promise;
            return a && function () {
              function e(r) {
                var n = t.context(),
                  o = i(r, "executor-", n, null, !1);
                const s = Reflect.construct(a, [o], e);
                return t.context(s).getCtx = function () {
                  return n;
                }, s;
              }
              f._A.Promise = e, Object.defineProperty(e, "name", {
                value: "Promise"
              }), e.toString = function () {
                return a.toString();
              }, Object.setPrototypeOf(e, a), ["all", "race"].forEach(function (r) {
                const n = a[r];
                e[r] = function (e) {
                  let i = !1;
                  [...(e || [])].forEach(e => {
                    this.resolve(e).then(a("all" === r), a(!1));
                  });
                  const o = n.apply(this, arguments);
                  return o;
                  function a(e) {
                    return function () {
                      t.emit("propagate", [null, !i], o, !1, !1), i = i || !e;
                    };
                  }
                };
              }), ["resolve", "reject"].forEach(function (r) {
                const n = a[r];
                e[r] = function (e) {
                  const r = n.apply(this, arguments);
                  return e !== r && t.emit("propagate", [e, !0], r, !1, !1), r;
                };
              }), e.prototype = a.prototype;
              const n = a.prototype.then;
              a.prototype.then = function () {
                var e = this,
                  o = r(e);
                o.promise = e;
                for (var a = arguments.length, s = new Array(a), c = 0; c < a; c++) s[c] = arguments[c];
                s[0] = i(s[0], "cb-", o, null, !1), s[1] = i(s[1], "cb-", o, null, !1);
                const u = n.apply(this, s);
                return o.nextPromise = u, t.emit("propagate", [e, !0], u, !1, !1), u;
              }, a.prototype.then[o] = n, t.on("executor-start", function (e) {
                e[0] = i(e[0], "resolve-", this, null, !1), e[1] = i(e[1], "resolve-", this, null, !1);
              }), t.on("executor-err", function (e, t, r) {
                e[1](r);
              }), t.on("cb-end", function (e, r, n) {
                t.emit("propagate", [n, !0], this.nextPromise, !1, !1);
              }), t.on("propagate", function (e, r, n) {
                this.getCtx && !r || (this.getCtx = function () {
                  if (e instanceof Promise) var r = t.context(e);
                  return r && r.getCtx ? r.getCtx() : this;
                });
              });
            }(), t;
          }
          const M = {},
            B = "setTimeout",
            F = "setInterval",
            U = "clearTimeout",
            q = "-start",
            Z = "-",
            V = [B, "setImmediate", F, U, "clearImmediate"];
          function G(e) {
            const t = function (e) {
              return (e || n.ee).get("timer");
            }(e);
            if (M[t.debugId]++) return t;
            M[t.debugId] = 1;
            var r = c(t);
            return r.inPlace(f._A, V.slice(0, 2), B + Z), r.inPlace(f._A, V.slice(2, 3), F + Z), r.inPlace(f._A, V.slice(3), U + Z), t.on(F + q, function (e, t, n) {
              e[0] = r(e[0], "fn-", null, n);
            }), t.on(B + q, function (e, t, n) {
              this.method = n, this.timerDuration = isNaN(e[1]) ? 0 : +e[1], e[0] = r(e[0], "fn-", this, n);
            }), t;
          }
          var W = r(50);
          const X = {},
            Q = ["open", "send"];
          function K(e) {
            var t = e || n.ee;
            const r = function (e) {
              return (e || n.ee).get("xhr");
            }(t);
            if (X[r.debugId]++) return r;
            X[r.debugId] = 1, b(t);
            var i = c(r),
              o = f._A.XMLHttpRequest,
              a = f._A.MutationObserver,
              s = f._A.Promise,
              u = f._A.setInterval,
              d = "readystatechange",
              l = ["onload", "onerror", "onabort", "onloadstart", "onloadend", "onprogress", "ontimeout"],
              h = [],
              p = f._A.XMLHttpRequest = function (e) {
                const t = new o(e),
                  n = r.context(t);
                try {
                  r.emit("new-xhr", [t], n), t.addEventListener(d, (a = n, function () {
                    var e = this;
                    e.readyState > 3 && !a.resolved && (a.resolved = !0, r.emit("xhr-resolved", [], e)), i.inPlace(e, l, "fn-", A);
                  }), (0, S.m$)(!1));
                } catch (e) {
                  (0, W.Z)("An error occurred while intercepting XHR", e);
                  try {
                    r.emit("internal-error", [e]);
                  } catch (e) {}
                }
                var a;
                return t;
              };
            function g(e, t) {
              i.inPlace(t, ["onreadystatechange"], "fn-", A);
            }
            if (function (e, t) {
              for (var r in e) t[r] = e[r];
            }(o, p), p.prototype = o.prototype, i.inPlace(p.prototype, Q, "-xhr-", A), r.on("send-xhr-start", function (e, t) {
              g(e, t), function (e) {
                h.push(e), a && (m ? m.then(w) : u ? u(w) : (v = -v, y.data = v));
              }(t);
            }), r.on("open-xhr-start", g), a) {
              var m = s && s.resolve();
              if (!u && !s) {
                var v = 1,
                  y = document.createTextNode(v);
                new a(w).observe(y, {
                  characterData: !0
                });
              }
            } else t.on("fn-end", function (e) {
              e[0] && e[0].type === d || w();
            });
            function w() {
              for (var e = 0; e < h.length; e++) g(0, h[e]);
              h.length && (h = []);
            }
            function A(e, t) {
              return t;
            }
            return r;
          }
        },
        7825: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.ajax;
        },
        6660: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.jserrors;
        },
        3081: (e, t, r) => {
          r.d(t, {
            gF: () => o,
            mY: () => i,
            t9: () => n,
            vz: () => s,
            xS: () => a
          });
          const n = r(3325).D.metrics,
            i = "sm",
            o = "cm",
            a = "storeSupportabilityMetrics",
            s = "storeEventMetrics";
        },
        4649: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.pageAction;
        },
        7633: (e, t, r) => {
          r.d(t, {
            Dz: () => i,
            OJ: () => a,
            qw: () => o,
            t9: () => n
          });
          const n = r(3325).D.pageViewEvent,
            i = "firstbyte",
            o = "domcontent",
            a = "windowload";
        },
        9251: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.pageViewTiming;
        },
        3614: (e, t, r) => {
          r.d(t, {
            BST_RESOURCE: () => i,
            END: () => s,
            FEATURE_NAME: () => n,
            FN_END: () => u,
            FN_START: () => c,
            PUSH_STATE: () => d,
            RESOURCE: () => o,
            START: () => a
          });
          const n = r(3325).D.sessionTrace,
            i = "bstResource",
            o = "resource",
            a = "-start",
            s = "-end",
            c = "fn" + a,
            u = "fn" + s,
            d = "pushState";
        },
        7836: (e, t, r) => {
          r.d(t, {
            BODY: () => x,
            CB_END: () => E,
            CB_START: () => u,
            END: () => A,
            FEATURE_NAME: () => i,
            FETCH: () => _,
            FETCH_BODY: () => v,
            FETCH_DONE: () => m,
            FETCH_START: () => g,
            FN_END: () => c,
            FN_START: () => s,
            INTERACTION: () => f,
            INTERACTION_API: () => d,
            INTERACTION_EVENTS: () => o,
            JSONP_END: () => b,
            JSONP_NODE: () => p,
            JS_TIME: () => T,
            MAX_TIMER_BUDGET: () => a,
            REMAINING: () => l,
            SPA_NODE: () => h,
            START: () => w,
            originalSetTimeout: () => y
          });
          var n = r(5763);
          const i = r(3325).D.spa,
            o = ["click", "submit", "keypress", "keydown", "keyup", "change"],
            a = 999,
            s = "fn-start",
            c = "fn-end",
            u = "cb-start",
            d = "api-ixn-",
            l = "remaining",
            f = "interaction",
            h = "spaNode",
            p = "jsonpNode",
            g = "fetch-start",
            m = "fetch-done",
            v = "fetch-body-",
            b = "jsonp-end",
            y = n.Yu.ST,
            w = "-start",
            A = "-end",
            x = "-body",
            E = "cb" + A,
            T = "jsTime",
            _ = "fetch";
        },
        5938: (e, t, r) => {
          r.d(t, {
            W: () => o
          });
          var n = r(5763),
            i = r(8325);
          class o {
            constructor(e, t, r) {
              this.agentIdentifier = e, this.aggregator = t, this.ee = i.ee.get(e, (0, n.OP)(this.agentIdentifier).isolatedBacklog), this.featureName = r, this.blocked = !1;
            }
          }
        },
        9144: (e, t, r) => {
          r.d(t, {
            j: () => m
          });
          var n = r(3325),
            i = r(5763),
            o = r(5546),
            a = r(8325),
            s = r(7894),
            c = r(8e3),
            u = r(3960),
            d = r(385),
            l = r(50),
            f = r(3081),
            h = r(8632);
          function p() {
            const e = (0, h.gG)();
            ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease", "addPageAction", "setCurrentRouteName", "setPageViewName", "setCustomAttribute", "interaction", "noticeError", "setUserId", "setApplicationVersion"].forEach(t => {
              e[t] = function () {
                for (var r = arguments.length, n = new Array(r), i = 0; i < r; i++) n[i] = arguments[i];
                return function (t) {
                  for (var r = arguments.length, n = new Array(r > 1 ? r - 1 : 0), i = 1; i < r; i++) n[i - 1] = arguments[i];
                  let o = [];
                  return Object.values(e.initializedAgents).forEach(e => {
                    e.exposed && e.api[t] && o.push(e.api[t](...n));
                  }), o.length > 1 ? o : o[0];
                }(t, ...n);
              };
            });
          }
          var g = r(2587);
          function m(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : {},
              m = arguments.length > 2 ? arguments[2] : void 0,
              v = arguments.length > 3 ? arguments[3] : void 0,
              {
                init: b,
                info: y,
                loader_config: w,
                runtime: A = {
                  loaderType: m
                },
                exposed: x = !0
              } = t;
            const E = (0, h.gG)();
            y || (b = E.init, y = E.info, w = E.loader_config), (0, i.Dg)(e, b || {}), (0, i.GE)(e, w || {}), y.jsAttributes ??= {}, d.v6 && (y.jsAttributes.isWorker = !0), (0, i.CX)(e, y);
            const T = (0, i.P_)(e);
            A.denyList = [...(T.ajax?.deny_list || []), ...(T.ajax?.block_internal ? [y.beacon, y.errorBeacon] : [])], (0, i.sU)(e, A), p();
            const _ = function (e, t) {
              t || (0, c.R)(e, "api");
              const h = {};
              var p = a.ee.get(e),
                g = p.get("tracer"),
                m = "api-",
                v = m + "ixn-";
              function b(t, r, n, o) {
                const a = (0, i.C5)(e);
                return null === r ? delete a.jsAttributes[t] : (0, i.CX)(e, {
                  ...a,
                  jsAttributes: {
                    ...a.jsAttributes,
                    [t]: r
                  }
                }), A(m, n, !0, o || null === r ? "session" : void 0)(t, r);
              }
              function y() {}
              ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease"].forEach(e => h[e] = A(m, e, !0, "api")), h.addPageAction = A(m, "addPageAction", !0, n.D.pageAction), h.setCurrentRouteName = A(m, "routeName", !0, n.D.spa), h.setPageViewName = function (t, r) {
                if ("string" == typeof t) return "/" !== t.charAt(0) && (t = "/" + t), (0, i.OP)(e).customTransaction = (r || "http://custom.transaction") + t, A(m, "setPageViewName", !0)();
              }, h.setCustomAttribute = function (e, t) {
                let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2];
                if ("string" == typeof e) {
                  if (["string", "number"].includes(typeof t) || null === t) return b(e, t, "setCustomAttribute", r);
                  (0, l.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t, "> was provided."));
                } else (0, l.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e, "> was provided."));
              }, h.setUserId = function (e) {
                if ("string" == typeof e || null === e) return b("enduser.id", e, "setUserId", !0);
                (0, l.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e, "> was provided."));
              }, h.setApplicationVersion = function (e) {
                if ("string" == typeof e || null === e) return b("application.version", e, "setApplicationVersion", !1);
                (0, l.Z)("Failed to execute setApplicationVersion. Expected <String | null>, but got <".concat(typeof e, ">."));
              }, h.interaction = function () {
                return new y().get();
              };
              var w = y.prototype = {
                createTracer: function (e, t) {
                  var r = {},
                    i = this,
                    a = "function" == typeof t;
                  return (0, o.p)(v + "tracer", [(0, s.z)(), e, r], i, n.D.spa, p), function () {
                    if (g.emit((a ? "" : "no-") + "fn-start", [(0, s.z)(), i, a], r), a) try {
                      return t.apply(this, arguments);
                    } catch (e) {
                      throw g.emit("fn-err", [arguments, this, e], r), e;
                    } finally {
                      g.emit("fn-end", [(0, s.z)()], r);
                    }
                  };
                }
              };
              function A(e, t, r, i) {
                return function () {
                  return (0, o.p)(f.xS, ["API/" + t + "/called"], void 0, n.D.metrics, p), i && (0, o.p)(e + t, [(0, s.z)(), ...arguments], r ? null : this, i, p), r ? void 0 : this;
                };
              }
              function x() {
                r.e(111).then(r.bind(r, 7438)).then(t => {
                  let {
                    setAPI: r
                  } = t;
                  r(e), (0, c.L)(e, "api");
                }).catch(() => (0, l.Z)("Downloading runtime APIs failed..."));
              }
              return ["actionText", "setName", "setAttribute", "save", "ignore", "onEnd", "getContext", "end", "get"].forEach(e => {
                w[e] = A(v, e, void 0, n.D.spa);
              }), h.noticeError = function (e, t) {
                "string" == typeof e && (e = new Error(e)), (0, o.p)(f.xS, ["API/noticeError/called"], void 0, n.D.metrics, p), (0, o.p)("err", [e, (0, s.z)(), !1, t], void 0, n.D.jserrors, p);
              }, d.il ? (0, u.b)(() => x(), !0) : x(), h;
            }(e, v);
            return (0, h.Qy)(e, _, "api"), (0, h.Qy)(e, x, "exposed"), (0, h.EZ)("activatedFeatures", g.T), _;
          }
        },
        3325: (e, t, r) => {
          r.d(t, {
            D: () => n,
            p: () => i
          });
          const n = {
              ajax: "ajax",
              jserrors: "jserrors",
              metrics: "metrics",
              pageAction: "page_action",
              pageViewEvent: "page_view_event",
              pageViewTiming: "page_view_timing",
              sessionReplay: "session_replay",
              sessionTrace: "session_trace",
              spa: "spa"
            },
            i = {
              [n.pageViewEvent]: 1,
              [n.pageViewTiming]: 2,
              [n.metrics]: 3,
              [n.jserrors]: 4,
              [n.ajax]: 5,
              [n.sessionTrace]: 6,
              [n.pageAction]: 7,
              [n.spa]: 8,
              [n.sessionReplay]: 9
            };
        }
      },
      n = {};
    function i(e) {
      var t = n[e];
      if (void 0 !== t) return t.exports;
      var o = n[e] = {
        exports: {}
      };
      return r[e](o, o.exports, i), o.exports;
    }
    i.m = r, i.d = (e, t) => {
      for (var r in t) i.o(t, r) && !i.o(e, r) && Object.defineProperty(e, r, {
        enumerable: !0,
        get: t[r]
      });
    }, i.f = {}, i.e = e => Promise.all(Object.keys(i.f).reduce((t, r) => (i.f[r](e, t), t), [])), i.u = e => "nr-spa.1097a448-1.238.0.min.js", i.o = (e, t) => Object.prototype.hasOwnProperty.call(e, t), e = {}, t = "NRBA-1.238.0.PROD:", i.l = (r, n, o, a) => {
      if (e[r]) e[r].push(n);else {
        var s, c;
        if (void 0 !== o) for (var u = document.getElementsByTagName("script"), d = 0; d < u.length; d++) {
          var l = u[d];
          if (l.getAttribute("src") == r || l.getAttribute("data-webpack") == t + o) {
            s = l;
            break;
          }
        }
        s || (c = !0, (s = document.createElement("script")).charset = "utf-8", s.timeout = 120, i.nc && s.setAttribute("nonce", i.nc), s.setAttribute("data-webpack", t + o), s.src = r), e[r] = [n];
        var f = (t, n) => {
            s.onerror = s.onload = null, clearTimeout(h);
            var i = e[r];
            if (delete e[r], s.parentNode && s.parentNode.removeChild(s), i && i.forEach(e => e(n)), t) return t(n);
          },
          h = setTimeout(f.bind(null, void 0, {
            type: "timeout",
            target: s
          }), 12e4);
        s.onerror = f.bind(null, s.onerror), s.onload = f.bind(null, s.onload), c && document.head.appendChild(s);
      }
    }, i.r = e => {
      "undefined" != typeof Symbol && Symbol.toStringTag && Object.defineProperty(e, Symbol.toStringTag, {
        value: "Module"
      }), Object.defineProperty(e, "__esModule", {
        value: !0
      });
    }, i.p = "https://js-agent.newrelic.com/", (() => {
      var e = {
        801: 0,
        92: 0
      };
      i.f.j = (t, r) => {
        var n = i.o(e, t) ? e[t] : void 0;
        if (0 !== n) if (n) r.push(n[2]);else {
          var o = new Promise((r, i) => n = e[t] = [r, i]);
          r.push(n[2] = o);
          var a = i.p + i.u(t),
            s = new Error();
          i.l(a, r => {
            if (i.o(e, t) && (0 !== (n = e[t]) && (e[t] = void 0), n)) {
              var o = r && ("load" === r.type ? "missing" : r.type),
                a = r && r.target && r.target.src;
              s.message = "Loading chunk " + t + " failed.\n(" + o + ": " + a + ")", s.name = "ChunkLoadError", s.type = o, s.request = a, n[1](s);
            }
          }, "chunk-" + t, t);
        }
      };
      var t = (t, r) => {
          var n,
            o,
            [a, s, c] = r,
            u = 0;
          if (a.some(t => 0 !== e[t])) {
            for (n in s) i.o(s, n) && (i.m[n] = s[n]);
            if (c) c(i);
          }
          for (t && t(r); u < a.length; u++) o = a[u], i.o(e, o) && e[o] && e[o][0](), e[o] = 0;
        },
        r = self["webpackChunk:NRBA-1.238.0.PROD"] = self["webpackChunk:NRBA-1.238.0.PROD"] || [];
      r.forEach(t.bind(null, 0)), r.push = t.bind(null, r.push.bind(r));
    })(), (() => {
      var e = i(50);
      class t {
        addPageAction(t, r) {
          (0, e.Z)("Call to agent api addPageAction failed. The session trace feature is not currently initialized.");
        }
        setPageViewName(t, r) {
          (0, e.Z)("Call to agent api setPageViewName failed. The page view feature is not currently initialized.");
        }
        setCustomAttribute(t, r, n) {
          (0, e.Z)("Call to agent api setCustomAttribute failed. The js errors feature is not currently initialized.");
        }
        noticeError(t, r) {
          (0, e.Z)("Call to agent api noticeError failed. The js errors feature is not currently initialized.");
        }
        setUserId(t) {
          (0, e.Z)("Call to agent api setUserId failed. The js errors feature is not currently initialized.");
        }
        setApplicationVersion(t) {
          (0, e.Z)("Call to agent api setApplicationVersion failed. The agent is not currently initialized.");
        }
        setErrorHandler(t) {
          (0, e.Z)("Call to agent api setErrorHandler failed. The js errors feature is not currently initialized.");
        }
        finished(t) {
          (0, e.Z)("Call to agent api finished failed. The page action feature is not currently initialized.");
        }
        addRelease(t, r) {
          (0, e.Z)("Call to agent api addRelease failed. The agent is not currently initialized.");
        }
      }
      var r = i(3325),
        n = i(5763);
      const o = Object.values(r.D);
      function a(e) {
        const t = {};
        return o.forEach(r => {
          t[r] = function (e, t) {
            return !1 !== (0, n.Mt)(t, "".concat(e, ".enabled"));
          }(r, e);
        }), t;
      }
      var s = i(9144);
      var c = i(5546),
        u = i(385),
        d = i(8e3),
        l = i(5938),
        f = i(3960);
      class h extends l.W {
        constructor(e, t, r) {
          let n = !(arguments.length > 3 && void 0 !== arguments[3]) || arguments[3];
          super(e, t, r), this.auto = n, this.abortHandler, this.featAggregate, this.onAggregateImported, n && (0, d.R)(e, r);
        }
        importAggregator() {
          let t = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : {};
          if (this.featAggregate || !this.auto) return;
          const r = u.il && !0 === (0, n.Mt)(this.agentIdentifier, "privacy.cookies_enabled");
          let o;
          this.onAggregateImported = new Promise(e => {
            o = e;
          });
          const a = async () => {
            let n;
            try {
              if (r) {
                const {
                  setupAgentSession: e
                } = await i.e(111).then(i.bind(i, 3228));
                n = e(this.agentIdentifier);
              }
            } catch (t) {
              (0, e.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.", t);
            }
            try {
              if (!this.shouldImportAgg(this.featureName, n)) return (0, d.L)(this.agentIdentifier, this.featureName), void o(!1);
              const {
                  lazyFeatureLoader: e
                } = await i.e(111).then(i.bind(i, 8582)),
                {
                  Aggregate: r
                } = await e(this.featureName, "aggregate");
              this.featAggregate = new r(this.agentIdentifier, this.aggregator, t), o(!0);
            } catch (t) {
              (0, e.Z)("Downloading and initializing ".concat(this.featureName, " failed..."), t), this.abortHandler?.(), o(!1);
            }
          };
          u.il ? (0, f.b)(() => a(), !0) : a();
        }
        shouldImportAgg(e, t) {
          return e !== r.D.sessionReplay || !!n.Yu.MO && !1 !== (0, n.Mt)(this.agentIdentifier, "session_trace.enabled") && (!!t?.isNew || !!t?.state.sessionReplay);
        }
      }
      var p = i(7633),
        g = i(7894);
      class m extends h {
        static featureName = p.t9;
        constructor(e, t) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          if (super(e, t, p.t9, i), ("undefined" == typeof PerformanceNavigationTiming || u.Tt) && "undefined" != typeof PerformanceTiming) {
            const t = (0, n.OP)(e);
            t[p.Dz] = Math.max(Date.now() - t.offset, 0), (0, f.K)(() => t[p.qw] = Math.max((0, g.z)() - t[p.Dz], 0)), (0, f.b)(() => {
              const e = (0, g.z)();
              t[p.OJ] = Math.max(e - t[p.Dz], 0), (0, c.p)("timing", ["load", e], void 0, r.D.pageViewTiming, this.ee);
            });
          }
          this.importAggregator();
        }
      }
      var v = i(1117),
        b = i(1284);
      class y extends v.w {
        constructor(e) {
          super(e), this.aggregatedData = {};
        }
        store(e, t, r, n, i) {
          var o = this.getBucket(e, t, r, i);
          return o.metrics = function (e, t) {
            t || (t = {
              count: 0
            });
            return t.count += 1, (0, b.D)(e, function (e, r) {
              t[e] = w(r, t[e]);
            }), t;
          }(n, o.metrics), o;
        }
        merge(e, t, r, n, i) {
          var o = this.getBucket(e, t, n, i);
          if (o.metrics) {
            var a = o.metrics;
            a.count += r.count, (0, b.D)(r, function (e, t) {
              if ("count" !== e) {
                var n = a[e],
                  i = r[e];
                i && !i.c ? a[e] = w(i.t, n) : a[e] = function (e, t) {
                  if (!t) return e;
                  t.c || (t = A(t.t));
                  return t.min = Math.min(e.min, t.min), t.max = Math.max(e.max, t.max), t.t += e.t, t.sos += e.sos, t.c += e.c, t;
                }(i, a[e]);
              }
            });
          } else o.metrics = r;
        }
        storeMetric(e, t, r, n) {
          var i = this.getBucket(e, t, r);
          return i.stats = w(n, i.stats), i;
        }
        getBucket(e, t, r, n) {
          this.aggregatedData[e] || (this.aggregatedData[e] = {});
          var i = this.aggregatedData[e][t];
          return i || (i = this.aggregatedData[e][t] = {
            params: r || {}
          }, n && (i.custom = n)), i;
        }
        get(e, t) {
          return t ? this.aggregatedData[e] && this.aggregatedData[e][t] : this.aggregatedData[e];
        }
        take(e) {
          for (var t = {}, r = "", n = !1, i = 0; i < e.length; i++) t[r = e[i]] = x(this.aggregatedData[r]), t[r].length && (n = !0), delete this.aggregatedData[r];
          return n ? t : null;
        }
      }
      function w(e, t) {
        return null == e ? function (e) {
          e ? e.c++ : e = {
            c: 1
          };
          return e;
        }(t) : t ? (t.c || (t = A(t.t)), t.c += 1, t.t += e, t.sos += e * e, e > t.max && (t.max = e), e < t.min && (t.min = e), t) : {
          t: e
        };
      }
      function A(e) {
        return {
          t: e,
          min: e,
          max: e,
          sos: e * e,
          c: 1
        };
      }
      function x(e) {
        return "object" != typeof e ? [] : (0, b.D)(e, E);
      }
      function E(e, t) {
        return t;
      }
      var T = i(8632),
        _ = i(4402),
        D = i(4351);
      var j = i(7956),
        C = i(3239),
        N = i(9251);
      class O extends h {
        static featureName = N.t;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, N.t, r), u.il && ((0, n.OP)(e).initHidden = Boolean("hidden" === document.visibilityState), (0, j.N)(() => (0, c.p)("docHidden", [(0, g.z)()], void 0, N.t, this.ee), !0), (0, C.bP)("pagehide", () => (0, c.p)("winPagehide", [(0, g.z)()], void 0, N.t, this.ee)), this.importAggregator());
        }
      }
      var S = i(3081);
      class P extends h {
        static featureName = S.t9;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, S.t9, r), this.importAggregator();
        }
      }
      var R = i(6660);
      class I {
        constructor(e, t, r, n) {
          this.name = "UncaughtError", this.message = e, this.sourceURL = t, this.line = r, this.column = n;
        }
      }
      class k extends h {
        static featureName = R.t;
        #e = new Set();
        constructor(e, t) {
          let n = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, R.t, n);
          try {
            this.removeOnAbort = new AbortController();
          } catch (e) {}
          this.ee.on("fn-err", (e, t, n) => {
            this.abortHandler && !this.#e.has(n) && (this.#e.add(n), (0, c.p)("err", [this.#t(n), (0, g.z)()], void 0, r.D.jserrors, this.ee));
          }), this.ee.on("internal-error", e => {
            this.abortHandler && (0, c.p)("ierr", [this.#t(e), (0, g.z)(), !0], void 0, r.D.jserrors, this.ee);
          }), u._A.addEventListener("unhandledrejection", e => {
            this.abortHandler && (0, c.p)("err", [this.#r(e), (0, g.z)(), !1, {
              unhandledPromiseRejection: 1
            }], void 0, r.D.jserrors, this.ee);
          }, (0, C.m$)(!1, this.removeOnAbort?.signal)), u._A.addEventListener("error", e => {
            this.abortHandler && (this.#e.has(e.error) ? this.#e.delete(e.error) : (0, c.p)("err", [this.#n(e), (0, g.z)()], void 0, r.D.jserrors, this.ee));
          }, (0, C.m$)(!1, this.removeOnAbort?.signal)), this.abortHandler = this.#i, this.importAggregator();
        }
        #i() {
          this.removeOnAbort?.abort(), this.#e.clear(), this.abortHandler = void 0;
        }
        #t(e) {
          return e instanceof Error ? e : void 0 !== e?.message ? new I(e.message, e.filename || e.sourceURL, e.lineno || e.line, e.colno || e.col) : new I("string" == typeof e ? e : (0, D.P)(e));
        }
        #r(e) {
          let t = "Unhandled Promise Rejection: ";
          if (e?.reason instanceof Error) try {
            return e.reason.message = t + e.reason.message, e.reason;
          } catch (t) {
            return e.reason;
          }
          if (void 0 === e.reason) return new I(t);
          const r = this.#t(e.reason);
          return r.message = t + r.message, r;
        }
        #n(e) {
          return e.error instanceof Error ? e.error : new I(e.message, e.filename, e.lineno, e.colno);
        }
      }
      var H = i(2210);
      let z = 1;
      const L = "nr@id";
      function M(e) {
        const t = typeof e;
        return !e || "object" !== t && "function" !== t ? -1 : e === u._A ? 0 : (0, H.X)(e, L, function () {
          return z++;
        });
      }
      function B(e) {
        if ("string" == typeof e && e.length) return e.length;
        if ("object" == typeof e) {
          if ("undefined" != typeof ArrayBuffer && e instanceof ArrayBuffer && e.byteLength) return e.byteLength;
          if ("undefined" != typeof Blob && e instanceof Blob && e.size) return e.size;
          if (!("undefined" != typeof FormData && e instanceof FormData)) try {
            return (0, D.P)(e).length;
          } catch (e) {
            return;
          }
        }
      }
      var F = i(1214),
        U = i(7243);
      class q {
        constructor(e) {
          this.agentIdentifier = e;
        }
        generateTracePayload(e) {
          if (!this.shouldGenerateTrace(e)) return null;
          var t = (0, n.DL)(this.agentIdentifier);
          if (!t) return null;
          var r = (t.accountID || "").toString() || null,
            i = (t.agentID || "").toString() || null,
            o = (t.trustKey || "").toString() || null;
          if (!r || !i) return null;
          var a = (0, _.M)(),
            s = (0, _.Ht)(),
            c = Date.now(),
            u = {
              spanId: a,
              traceId: s,
              timestamp: c
            };
          return (e.sameOrigin || this.isAllowedOrigin(e) && this.useTraceContextHeadersForCors()) && (u.traceContextParentHeader = this.generateTraceContextParentHeader(a, s), u.traceContextStateHeader = this.generateTraceContextStateHeader(a, c, r, i, o)), (e.sameOrigin && !this.excludeNewrelicHeader() || !e.sameOrigin && this.isAllowedOrigin(e) && this.useNewrelicHeaderForCors()) && (u.newrelicHeader = this.generateTraceHeader(a, s, c, r, i, o)), u;
        }
        generateTraceContextParentHeader(e, t) {
          return "00-" + t + "-" + e + "-01";
        }
        generateTraceContextStateHeader(e, t, r, n, i) {
          return i + "@nr=0-1-" + r + "-" + n + "-" + e + "----" + t;
        }
        generateTraceHeader(e, t, r, n, i, o) {
          if (!("function" == typeof u._A?.btoa)) return null;
          var a = {
            v: [0, 1],
            d: {
              ty: "Browser",
              ac: n,
              ap: i,
              id: e,
              tr: t,
              ti: r
            }
          };
          return o && n !== o && (a.d.tk = o), btoa((0, D.P)(a));
        }
        shouldGenerateTrace(e) {
          return this.isDtEnabled() && this.isAllowedOrigin(e);
        }
        isAllowedOrigin(e) {
          var t = !1,
            r = {};
          if ((0, n.Mt)(this.agentIdentifier, "distributed_tracing") && (r = (0, n.P_)(this.agentIdentifier).distributed_tracing), e.sameOrigin) t = !0;else if (r.allowed_origins instanceof Array) for (var i = 0; i < r.allowed_origins.length; i++) {
            var o = (0, U.e)(r.allowed_origins[i]);
            if (e.hostname === o.hostname && e.protocol === o.protocol && e.port === o.port) {
              t = !0;
              break;
            }
          }
          return t;
        }
        isDtEnabled() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.enabled;
        }
        excludeNewrelicHeader() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.exclude_newrelic_header;
        }
        useNewrelicHeaderForCors() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !1 !== e.cors_use_newrelic_header;
        }
        useTraceContextHeadersForCors() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.cors_use_tracecontext_headers;
        }
      }
      var Z = i(7825),
        V = ["load", "error", "abort", "timeout"],
        G = V.length,
        W = n.Yu.REQ,
        X = n.Yu.XHR;
      class Q extends h {
        static featureName = Z.t;
        constructor(e, t) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, Z.t, i), (0, n.OP)(e).xhrWrappable && (this.dt = new q(e), this.handler = (e, t, r, n) => (0, c.p)(e, t, r, n, this.ee), (0, F.u5)(this.ee), (0, F.Kf)(this.ee), function (e, t, i, o) {
            function a(e) {
              var t = this;
              t.totalCbs = 0, t.called = 0, t.cbTime = 0, t.end = E, t.ended = !1, t.xhrGuids = {}, t.lastSize = null, t.loadCaptureCalled = !1, t.params = this.params || {}, t.metrics = this.metrics || {}, e.addEventListener("load", function (r) {
                _(t, e);
              }, (0, C.m$)(!1)), u.IF || e.addEventListener("progress", function (e) {
                t.lastSize = e.loaded;
              }, (0, C.m$)(!1));
            }
            function s(e) {
              this.params = {
                method: e[0]
              }, T(this, e[1]), this.metrics = {};
            }
            function c(t, r) {
              var i = (0, n.DL)(e);
              i.xpid && this.sameOrigin && r.setRequestHeader("X-NewRelic-ID", i.xpid);
              var a = o.generateTracePayload(this.parsedOrigin);
              if (a) {
                var s = !1;
                a.newrelicHeader && (r.setRequestHeader("newrelic", a.newrelicHeader), s = !0), a.traceContextParentHeader && (r.setRequestHeader("traceparent", a.traceContextParentHeader), a.traceContextStateHeader && r.setRequestHeader("tracestate", a.traceContextStateHeader), s = !0), s && (this.dt = a);
              }
            }
            function d(e, r) {
              var n = this.metrics,
                i = e[0],
                o = this;
              if (n && i) {
                var a = B(i);
                a && (n.txSize = a);
              }
              this.startTime = (0, g.z)(), this.listener = function (e) {
                try {
                  "abort" !== e.type || o.loadCaptureCalled || (o.params.aborted = !0), ("load" !== e.type || o.called === o.totalCbs && (o.onloadCalled || "function" != typeof r.onload) && "function" == typeof o.end) && o.end(r);
                } catch (e) {
                  try {
                    t.emit("internal-error", [e]);
                  } catch (e) {}
                }
              };
              for (var s = 0; s < G; s++) r.addEventListener(V[s], this.listener, (0, C.m$)(!1));
            }
            function l(e, t, r) {
              this.cbTime += e, t ? this.onloadCalled = !0 : this.called += 1, this.called !== this.totalCbs || !this.onloadCalled && "function" == typeof r.onload || "function" != typeof this.end || this.end(r);
            }
            function f(e, t) {
              var r = "" + M(e) + !!t;
              this.xhrGuids && !this.xhrGuids[r] && (this.xhrGuids[r] = !0, this.totalCbs += 1);
            }
            function h(e, t) {
              var r = "" + M(e) + !!t;
              this.xhrGuids && this.xhrGuids[r] && (delete this.xhrGuids[r], this.totalCbs -= 1);
            }
            function p() {
              this.endTime = (0, g.z)();
            }
            function m(e, r) {
              r instanceof X && "load" === e[0] && t.emit("xhr-load-added", [e[1], e[2]], r);
            }
            function v(e, r) {
              r instanceof X && "load" === e[0] && t.emit("xhr-load-removed", [e[1], e[2]], r);
            }
            function b(e, t, r) {
              t instanceof X && ("onload" === r && (this.onload = !0), ("load" === (e[0] && e[0].type) || this.onload) && (this.xhrCbStart = (0, g.z)()));
            }
            function y(e, r) {
              this.xhrCbStart && t.emit("xhr-cb-time", [(0, g.z)() - this.xhrCbStart, this.onload, r], r);
            }
            function w(e) {
              var t,
                r = e[1] || {};
              if ("string" == typeof e[0] ? 0 === (t = e[0]).length && u.il && (t = "" + u._A.location.href) : e[0] && e[0].url ? t = e[0].url : u._A?.URL && e[0] && e[0] instanceof URL ? t = e[0].href : "function" == typeof e[0].toString && (t = e[0].toString()), "string" == typeof t && 0 !== t.length) {
                t && (this.parsedOrigin = (0, U.e)(t), this.sameOrigin = this.parsedOrigin.sameOrigin);
                var n = o.generateTracePayload(this.parsedOrigin);
                if (n && (n.newrelicHeader || n.traceContextParentHeader)) if (e[0] && e[0].headers) s(e[0].headers, n) && (this.dt = n);else {
                  var i = {};
                  for (var a in r) i[a] = r[a];
                  i.headers = new Headers(r.headers || {}), s(i.headers, n) && (this.dt = n), e.length > 1 ? e[1] = i : e.push(i);
                }
              }
              function s(e, t) {
                var r = !1;
                return t.newrelicHeader && (e.set("newrelic", t.newrelicHeader), r = !0), t.traceContextParentHeader && (e.set("traceparent", t.traceContextParentHeader), t.traceContextStateHeader && e.set("tracestate", t.traceContextStateHeader), r = !0), r;
              }
            }
            function A(e, t) {
              this.params = {}, this.metrics = {}, this.startTime = (0, g.z)(), this.dt = t, e.length >= 1 && (this.target = e[0]), e.length >= 2 && (this.opts = e[1]);
              var r,
                n = this.opts || {},
                i = this.target;
              "string" == typeof i ? r = i : "object" == typeof i && i instanceof W ? r = i.url : u._A?.URL && "object" == typeof i && i instanceof URL && (r = i.href), T(this, r);
              var o = ("" + (i && i instanceof W && i.method || n.method || "GET")).toUpperCase();
              this.params.method = o, this.txSize = B(n.body) || 0;
            }
            function x(e, t) {
              var n;
              this.endTime = (0, g.z)(), this.params || (this.params = {}), this.params.status = t ? t.status : 0, "string" == typeof this.rxSize && this.rxSize.length > 0 && (n = +this.rxSize);
              var o = {
                txSize: this.txSize,
                rxSize: n,
                duration: (0, g.z)() - this.startTime
              };
              i("xhr", [this.params, o, this.startTime, this.endTime, "fetch"], this, r.D.ajax);
            }
            function E(e) {
              var t = this.params,
                n = this.metrics;
              if (!this.ended) {
                this.ended = !0;
                for (var o = 0; o < G; o++) e.removeEventListener(V[o], this.listener, !1);
                t.aborted || (n.duration = (0, g.z)() - this.startTime, this.loadCaptureCalled || 4 !== e.readyState ? null == t.status && (t.status = 0) : _(this, e), n.cbTime = this.cbTime, i("xhr", [t, n, this.startTime, this.endTime, "xhr"], this, r.D.ajax));
              }
            }
            function T(e, t) {
              var r = (0, U.e)(t),
                n = e.params;
              n.hostname = r.hostname, n.port = r.port, n.protocol = r.protocol, n.host = r.hostname + ":" + r.port, n.pathname = r.pathname, e.parsedOrigin = r, e.sameOrigin = r.sameOrigin;
            }
            function _(e, t) {
              e.params.status = t.status;
              var r = function (e, t) {
                var r = e.responseType;
                return "json" === r && null !== t ? t : "arraybuffer" === r || "blob" === r || "json" === r ? B(e.response) : "text" === r || "" === r || void 0 === r ? B(e.responseText) : void 0;
              }(t, e.lastSize);
              if (r && (e.metrics.rxSize = r), e.sameOrigin) {
                var n = t.getResponseHeader("X-NewRelic-App-Data");
                n && (e.params.cat = n.split(", ").pop());
              }
              e.loadCaptureCalled = !0;
            }
            t.on("new-xhr", a), t.on("open-xhr-start", s), t.on("open-xhr-end", c), t.on("send-xhr-start", d), t.on("xhr-cb-time", l), t.on("xhr-load-added", f), t.on("xhr-load-removed", h), t.on("xhr-resolved", p), t.on("addEventListener-end", m), t.on("removeEventListener-end", v), t.on("fn-end", y), t.on("fetch-before-start", w), t.on("fetch-start", A), t.on("fn-start", b), t.on("fetch-done", x);
          }(e, this.ee, this.handler, this.dt), this.importAggregator());
        }
      }
      var K = i(3614);
      const {
        BST_RESOURCE: Y,
        RESOURCE: J,
        START: ee,
        END: te,
        FEATURE_NAME: re,
        FN_END: ne,
        FN_START: ie,
        PUSH_STATE: oe
      } = K;
      var ae = i(7836);
      const {
        FEATURE_NAME: se,
        START: ce,
        END: ue,
        BODY: de,
        CB_END: le,
        JS_TIME: fe,
        FETCH: he,
        FN_START: pe,
        CB_START: ge,
        FN_END: me
      } = ae;
      var ve = i(4649);
      class be extends h {
        static featureName = ve.t;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, ve.t, r), this.importAggregator();
        }
      }
      new class extends t {
        constructor(t) {
          let r = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : (0, _.ky)(16);
          super(), u._A ? (this.agentIdentifier = r, this.sharedAggregator = new y({
            agentIdentifier: this.agentIdentifier
          }), this.features = {}, this.desiredFeatures = new Set(t.features || []), this.desiredFeatures.add(m), Object.assign(this, (0, s.j)(this.agentIdentifier, t, t.loaderType || "agent")), this.start()) : (0, e.Z)("Failed to initial the agent. Could not determine the runtime environment.");
        }
        get config() {
          return {
            info: (0, n.C5)(this.agentIdentifier),
            init: (0, n.P_)(this.agentIdentifier),
            loader_config: (0, n.DL)(this.agentIdentifier),
            runtime: (0, n.OP)(this.agentIdentifier)
          };
        }
        start() {
          const t = "features";
          try {
            const n = a(this.agentIdentifier),
              i = [...this.desiredFeatures];
            i.sort((e, t) => r.p[e.featureName] - r.p[t.featureName]), i.forEach(t => {
              if (n[t.featureName] || t.featureName === r.D.pageViewEvent) {
                const i = function (e) {
                  switch (e) {
                    case r.D.ajax:
                      return [r.D.jserrors];
                    case r.D.sessionTrace:
                      return [r.D.ajax, r.D.pageViewEvent];
                    case r.D.sessionReplay:
                      return [r.D.sessionTrace];
                    case r.D.pageViewTiming:
                      return [r.D.pageViewEvent];
                    default:
                      return [];
                  }
                }(t.featureName);
                i.every(e => n[e]) || (0, e.Z)("".concat(t.featureName, " is enabled but one or more dependent features has been disabled (").concat((0, D.P)(i), "). This may cause unintended consequences or missing data...")), this.features[t.featureName] = new t(this.agentIdentifier, this.sharedAggregator);
              }
            }), (0, T.Qy)(this.agentIdentifier, this.features, t);
          } catch (r) {
            (0, e.Z)("Failed to initialize all enabled instrument classes (agent aborted) -", r);
            for (const e in this.features) this.features[e].abortHandler?.();
            const n = (0, T.fP)();
            return delete n.initializedAgents[this.agentIdentifier]?.api, delete n.initializedAgents[this.agentIdentifier]?.[t], delete this.sharedAggregator, n.ee?.abort(), delete n.ee?.get(this.agentIdentifier), !1;
          }
        }
        addToTrace(t) {
          (0, e.Z)("Call to agent api addToTrace failed. The page action feature is not currently initialized.");
        }
        setCurrentRouteName(t) {
          (0, e.Z)("Call to agent api setCurrentRouteName failed. The spa feature is not currently initialized.");
        }
        interaction() {
          (0, e.Z)("Call to agent api interaction failed. The spa feature is not currently initialized.");
        }
      }({
        features: [Q, m, O, class extends h {
          static featureName = re;
          constructor(e, t) {
            if (super(e, t, re, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !u.il) return;
            const n = this.ee;
            let i;
            (0, F.QU)(n), this.eventsEE = (0, F.em)(n), this.eventsEE.on(ie, function (e, t) {
              this.bstStart = (0, g.z)();
            }), this.eventsEE.on(ne, function (e, t) {
              (0, c.p)("bst", [e[0], t, this.bstStart, (0, g.z)()], void 0, r.D.sessionTrace, n);
            }), n.on(oe + ee, function (e) {
              this.time = (0, g.z)(), this.startPath = location.pathname + location.hash;
            }), n.on(oe + te, function (e) {
              (0, c.p)("bstHist", [location.pathname + location.hash, this.startPath, this.time], void 0, r.D.sessionTrace, n);
            });
            try {
              i = new PerformanceObserver(e => {
                const t = e.getEntries();
                (0, c.p)(Y, [t], void 0, r.D.sessionTrace, n);
              }), i.observe({
                type: J,
                buffered: !0
              });
            } catch (e) {}
            this.importAggregator({
              resourceObserver: i
            });
          }
        }, P, be, k, class extends h {
          static featureName = se;
          constructor(e, t) {
            if (super(e, t, se, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !u.il) return;
            if (!(0, n.OP)(e).xhrWrappable) return;
            try {
              this.removeOnAbort = new AbortController();
            } catch (e) {}
            let r,
              i = 0;
            const o = this.ee.get("tracer"),
              a = (0, F._L)(this.ee),
              s = (0, F.Lg)(this.ee),
              c = (0, F.BV)(this.ee),
              d = (0, F.Kf)(this.ee),
              l = this.ee.get("events"),
              f = (0, F.u5)(this.ee),
              h = (0, F.QU)(this.ee),
              p = (0, F.Gm)(this.ee);
            function m(e, t) {
              h.emit("newURL", ["" + window.location, t]);
            }
            function v() {
              i++, r = window.location.hash, this[pe] = (0, g.z)();
            }
            function b() {
              i--, window.location.hash !== r && m(0, !0);
              var e = (0, g.z)();
              this[fe] = ~~this[fe] + e - this[pe], this[me] = e;
            }
            function y(e, t) {
              e.on(t, function () {
                this[t] = (0, g.z)();
              });
            }
            this.ee.on(pe, v), s.on(ge, v), a.on(ge, v), this.ee.on(me, b), s.on(le, b), a.on(le, b), this.ee.buffer([pe, me, "xhr-resolved"], this.featureName), l.buffer([pe], this.featureName), c.buffer(["setTimeout" + ue, "clearTimeout" + ce, pe], this.featureName), d.buffer([pe, "new-xhr", "send-xhr" + ce], this.featureName), f.buffer([he + ce, he + "-done", he + de + ce, he + de + ue], this.featureName), h.buffer(["newURL"], this.featureName), p.buffer([pe], this.featureName), s.buffer(["propagate", ge, le, "executor-err", "resolve" + ce], this.featureName), o.buffer([pe, "no-" + pe], this.featureName), a.buffer(["new-jsonp", "cb-start", "jsonp-error", "jsonp-end"], this.featureName), y(f, he + ce), y(f, he + "-done"), y(a, "new-jsonp"), y(a, "jsonp-end"), y(a, "cb-start"), h.on("pushState-end", m), h.on("replaceState-end", m), window.addEventListener("hashchange", m, (0, C.m$)(!0, this.removeOnAbort?.signal)), window.addEventListener("load", m, (0, C.m$)(!0, this.removeOnAbort?.signal)), window.addEventListener("popstate", function () {
              m(0, i > 1);
            }, (0, C.m$)(!0, this.removeOnAbort?.signal)), this.abortHandler = this.#i, this.importAggregator();
          }
          #i() {
            this.removeOnAbort?.abort(), this.abortHandler = void 0;
          }
        }],
        loaderType: "spa"
      });
    })();
  })();
})()</script>
      <link rel="shortcut icon" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/103/images/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/103/images/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/arp.css">
      <link href="//cdn.pendo.io" rel="dns-prefetch">
      <link href="https://cdn.pendo.io" rel="preconnect" crossorigin="anonymous">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com">
      <script async="" id="reading-assistant-script-tag" src="/feature/assets/ai-components?componentVersion=V14" type="text/javascript"></script>
      <script type="text/javascript">
        
        window.appData = window.appData || [];
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Journal of Human Evolution","auth":"AE"};
        window.arp = {
          config: {"adobeSuite":"elsevier-sd-prod","arsUrl":"https://ars.els-cdn.com","assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1","recommendationsFeedback":{"enabled":true,"url":"https://feedback.recs.d.elsevier.com/raw/events","timeout":60000},"googleMapsApiKey":"AIzaSyCBYU6I6lrbEU6wQXUEIte3NwGtm3jwHQc","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","xocsMediaPlayerBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/147/js/ableplayer","strictMode":false,"seamlessAccess":{"enableSeamlessAccess":true,"scriptUrl":"https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js","persistenceUrl":"https://service.seamlessaccess.org/ps/","persistenceContext":"seamlessaccess.org","scienceDirectUrl":"https://www.sciencedirect.com","shibAuthUrl":"https://auth.elsevier.com/ShibAuth/institutionLogin"},"reaxys":{"apiUrl":"https://reaxys-sdlc.reaxys.com","origin":"sciencedirect","queryBuilderHostPath":"https://www.reaxys.com/reaxys/secured/hopinto.do","url":"https://www.reaxys.com"},"oneTrustCookie":{"enabled":true},"ssrn":{"url":"https://papers.ssrn.com","path":"/sol3/papers.cfm"},"aiNavigation":{"enabled":true,"version":"V1"}},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
        window.addEventListener('beforeprint', () => pendo.onGuideDismissed());
      </script>
    <script data-cfasync="false" src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" data-document-language="true" data-domain-script="865ea198-88cc-4e41-8952-1df75d554d02"></script><meta http-equiv="origin-trial" content="AlK2UR5SkAlj8jjdEc9p3F3xuFYlF6LYjAML3EOqw1g26eCwWPjdmecULvBH5MVPoqKYrOfPhYVL71xAXI1IBQoAAAB8eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiV2ViVmlld1hSZXF1ZXN0ZWRXaXRoRGVwcmVjYXRpb24iLCJleHBpcnkiOjE3NTgwNjcxOTksImlzU3ViZG9tYWluIjp0cnVlfQ=="><meta http-equiv="origin-trial" content="Amm8/NmvvQfhwCib6I7ZsmUxiSCfOxWxHayJwyU1r3gRIItzr7bNQid6O8ZYaE1GSQTa69WwhPC9flq/oYkRBwsAAACCeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiV2ViVmlld1hSZXF1ZXN0ZWRXaXRoRGVwcmVjYXRpb24iLCJleHBpcnkiOjE3NTgwNjcxOTksImlzU3ViZG9tYWluIjp0cnVlfQ=="><meta http-equiv="origin-trial" content="A9wSqI5i0iwGdf6L1CERNdmsTPgVu44ewj8QxTBYgsv1LCPUVF7YmWOvTappqB1139jAymxUW/RO8zmMqo4zlAAAAACNeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiRmxlZGdlQmlkZGluZ0FuZEF1Y3Rpb25TZXJ2ZXIiLCJleHBpcnkiOjE3MzY4MTI4MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A+d7vJfYtay4OUbdtRPZA3y7bKQLsxaMEPmxgfhBGqKXNrdkCQeJlUwqa6EBbSfjwFtJWTrWIioXeMW+y8bWAgQAAACTeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiRmxlZGdlQmlkZGluZ0FuZEF1Y3Rpb25TZXJ2ZXIiLCJleHBpcnkiOjE3MzY4MTI4MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><script src="https://securepubads.g.doubleclick.net/pagead/managed/js/gpt/m202506050101/pubads_impl.js" async=""></script><link href="https://securepubads.g.doubleclick.net/pagead/managed/dict/m202506100101/gpt" rel="compression-dictionary"><script src="https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script src="https://cdn.cookielaw.org/scripttemplates/202504.1.0/otBannerSdk.js" async="" type="text/javascript"></script><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
</style><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-button-order-0{order:0}#onetrust-banner-sdk .ot-button-order-1{order:1}#onetrust-banner-sdk .ot-button-order-2{order:2}#onetrust-banner-sdk #onetrust-close-btn-container svg{height:10px;width:10px;pointer-events:none}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .pc-logo svg,#onetrust-pc-sdk .ot-pc-logo svg{height:60px;width:180px}#onetrust-pc-sdk #close-pc-btn-handler>svg{margin:auto;display:block;height:12px;width:12px}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo svg,#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.8em;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px}#onetrust-banner-sdk .ot-optout-signal svg.ot-source-sprite,#onetrust-pc-sdk .ot-optout-signal svg.ot-source-sprite{position:relative;bottom:-3px}#onetrust-banner-sdk .ot-optout-signal svg:not(.ot-source-sprite),#onetrust-pc-sdk .ot-optout-signal svg:not(.ot-source-sprite){transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg:not(.ot-source-sprite) path,#onetrust-pc-sdk .ot-optout-signal svg:not(.ot-source-sprite) path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk #onetrust-sprite-svg{display:none}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk img,#onetrust-pc-sdk img,#ot-sdk-cookie-policy img,#ot-sync-ntfy img{overflow:hidden !important}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-group-container{position:relative}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px;margin-right:5px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (max-width: 550px){#onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler,#onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler{margin-right:0}#onetrust-banner-sdk .has-reject-all-button div#onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler,#onetrust-banner-sdk .has-reject-all-button div#onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button div#onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler{margin-right:0}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk #onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler{width:auto}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container{display:inline-flex;flex-wrap:wrap}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler{float:none}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container *[class*=ot-button-order-]:nth-of-type(1){margin-right:auto !important}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group.ot-button-order-container{width:auto}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:80%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! Extra code to blur out background */
.onetrust-pc-dark-filter{
background:rgba(0,0,0,.5);
z-index:2147483646;
width:100%;
height:100%;
overflow:hidden;
position:fixed;
top:0;
bottom:0;
left:0;
backdrop-filter: initial
}

/*! v6.12.0 2021-01-19 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #eb6500!important;outline:1px solid transparent;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007398!important;border:2px solid #007398!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#eb6500!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007398!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color: #2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007398!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #eb6500!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:#2e2e2e!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:#2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #eb6500;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:#2e2e2e;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#eb6500;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#eb6500;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #eb6500!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{width:100%}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk #ot-pc-hdr{margin-left:24px}div#onetrust-pc-sdk .ot-lst-subhdr{width:calc(100% - 24px)}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,div#onetrust-pc-sdk button#filter-apply-handler,div#onetrust-pc-sdk button#filter-cancel-handler{height:2em!important;padding-left:14px!important;padding-right:14px!important}div#onetrust-pc-sdk #ot-fltr-cnt{box-shadow:0 0;border:1px solid #8e8e8e;border-radius:0}div#onetrust-pc-sdk .ot-fltr-scrlcnt{max-height:calc(100% - 80px)}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}div#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:16px}div#onetrust-pc-sdk #ot-lst-cnt{margin-left:24px;width:calc(100% - 48px)}div#onetrust-pc-sdk #ot-anchor{display:none!important}

/* 2023-12-04  Fix for button order in mobile view*/
@media (max-width: 550px) {
  #onetrust-accept-btn-handler {order: 1;  }
  #onetrust-reject-all-handler { order: 2;  }
  #onetrust-pc-btn-handler { order: 3;  }
}


/*! Extra code to blur our background */
.onetrust-pc-dark-filter{
backdrop-filter: blur(3px)
}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;position:relative;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #ot-filter-list-header{margin-top:20px;margin-bottom:10px;float:left;max-width:150px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px 1px 1px 15px;overflow:hidden;text-overflow:ellipsis;white-space:nowrap}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .ot-pc-refuse-all-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .save-preference-btn-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container #accept-recommended-btn-handler{width:auto;margin-bottom:1rem}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:10px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-accordion-layout .ot-pgph-link{margin-top:5px}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .ot-pc-refuse-all-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .save-preference-btn-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container #accept-recommended-btn-handler{width:100%}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! Extra code to blur out background */
.onetrust-pc-dark-filter{
background:rgba(0,0,0,.5);
z-index:2147483646;
width:100%;
height:100%;
overflow:hidden;
position:fixed;
top:0;
bottom:0;
left:0;
backdrop-filter: initial
}

/*! v6.12.0 2021-01-19 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #eb6500!important;outline:1px solid transparent;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007398!important;border:2px solid #007398!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#eb6500!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007398!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color: #2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007398!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #eb6500!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:#2e2e2e!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:#2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #eb6500;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:#2e2e2e;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#eb6500;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#eb6500;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #eb6500!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{width:100%}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk #ot-pc-hdr{margin-left:24px}div#onetrust-pc-sdk .ot-lst-subhdr{width:calc(100% - 24px)}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,div#onetrust-pc-sdk button#filter-apply-handler,div#onetrust-pc-sdk button#filter-cancel-handler{height:2em!important;padding-left:14px!important;padding-right:14px!important}div#onetrust-pc-sdk #ot-fltr-cnt{box-shadow:0 0;border:1px solid #8e8e8e;border-radius:0}div#onetrust-pc-sdk .ot-fltr-scrlcnt{max-height:calc(100% - 80px)}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}div#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:16px}div#onetrust-pc-sdk #ot-lst-cnt{margin-left:24px;width:calc(100% - 48px)}div#onetrust-pc-sdk #ot-anchor{display:none!important}

/* 2023-12-04  Fix for button order in mobile view*/
@media (max-width: 550px) {
  #onetrust-accept-btn-handler {order: 1;  }
  #onetrust-reject-all-handler { order: 2;  }
  #onetrust-pc-btn-handler { order: 3;  }
}


/*! Extra code to blur our background */
.onetrust-pc-dark-filter{
backdrop-filter: blur(3px)
}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://assets.adobedtm.com/4a848ae9611a/032db4f73473/launch-a6263b31083f.min.js" type="text/javascript" async=""></script><script src="https://assets.adobedtm.com/extensions/EP31dbb9c60e404ba1aa6e746d49be6f29/AppMeasurement.min.js" async=""></script><script src="https://assets.adobedtm.com/extensions/EP31dbb9c60e404ba1aa6e746d49be6f29/AppMeasurement_Module_ActivityMap.min.js" async=""></script><script src="https://assets.adobedtm.com/4a848ae9611a/032db4f73473/93571d51095c/RCa16d232f95a944c0aabdea6621a2ef94-source.min.js" async=""></script><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style></head>
    <body data-sd-ui-layer-boundary="true"><div id="MathJax_Message" style="display: none;"></div>
      <script type="text/javascript">
        window.__PRELOADED_STATE__ = {"abstracts":{"content":[{"$$":[{"$":{"id":"sectitle0010"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Wallace's Line demarcates a biogeographical boundary between the Indomalaya and Australasian ecoregions. Most placental mammalian genera, for example, occur to the west of this line, whereas most marsupial genera occur to the east. However, macaque monkeys are unusual because they naturally occur on both western and eastern sides. To further explore this anomalous distribution, we analyzed 222 mitochondrial genomes from âˆ¼20 macaque species, including new genomes from 60 specimens. These comprise a population sampling of most Sulawesi macaques, "},{"#name":"italic","_":"Macaca fascicularis"},{"#name":"__text__","_":" (long-tailed macaques) specimens that were collected by Alfred R. Wallace and specimens that were recovered during archaeological excavations at Liang Bua, a cave on the Indonesian island of Flores. In "},{"#name":"italic","_":"M.Â fascicularis"},{"#name":"__text__","_":", three mitochondrial lineages span the southernmost portion of Wallace's Line between Bali and Lombok, and divergences within these lineages are contemporaneous with, and possibly mediated by, past dispersals of modern human populations. Near the central portion of Wallace's Line between Borneo and Sulawesi, a more ancient dispersal of macaques from mainland Asia to Sulawesi preceded modern human colonization, which was followed by rapid dispersal of matrilines and was subsequently influenced by recent interspecies hybridization. In contrast to previous studies, we find no strong signal of recombination in most macaque mitochondrial genomes. These findings further characterize macaque evolution before and after modern human dispersal throughout Southeast Asia and point to possible effects on biodiversity of ancient human cultural diasporas."}],"$":{"view":"all","id":"abspara0010"},"#name":"simple-para"}],"$":{"view":"all","id":"abssec0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"abs0010","lang":"en","class":"author"},"#name":"abstract"}],"floats":[],"footnotes":[],"attachments":[]},"accessbarConfig":{"fallback":false,"id":"accessbar","version":"0.0.1","analytics":{"location":"accessbar","eventName":"ctaImpression"},"label":{},"ariaLabel":{"accessbar":"Download options and search","componentsList":"PDF Options"},"banners":[{"id":"BannerSsrn"}],"components":[{"target":"_blank","analytics":[{"ids":["accessbar:fta:single-article"],"eventName":"ctaClick"}],"label":"View&nbsp;**PDF**","ariaLabel":"View PDF. Opens in a new window.","id":"ViewPDF"},{"analytics":[{"ids":["accessbar:fta:full-issue"],"eventName":"ctaClick"}],"label":"Download full issue","id":"DownloadFullIssue"},{"id":"ViewClinicalKeyFullText","analytics":[{"eventName":"ctaClick","ids":["accessbar:fta:clinical-key"]}],"label":"View full text","rel":"noopener noreferrer","target":"_blank","href":"https://www.clinicalkey.com/#!/content/journal/1-s2.0-S0047248420301135"}],"search":{"inputPlaceHolder":"Search ScienceDirect","ariaLabel":{"input":"Search ScienceDirect","submit":"Submit search"},"formAction":"/search#submit","analytics":[{"ids":["accessbar:search"],"eventName":"searchStart"}],"id":"QuickSearch"}},"adobeTarget":{},"article":{"analyticsMetadata":{"accountId":"228598","accountName":"ScienceDirect Guests","loginStatus":"anonymous","userId":"12975512","isLoggedIn":false},"article-number":"102852","cid":"272603","content-family":"serial","copyright-line":"Â© 2020 The Author(s). Published by Elsevier Ltd.","cover-date-years":["2020"],"cover-date-start":"2020-09-01","cover-date-text":"September 2020","document-subtype":"fla","document-type":"article","entitledToken":"7FA278CD5175C59871BA1D8C98DECCB2606BF876EF5FC9CEACC7C320568485B98C673EFE3F668494","genAiToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJnZW5BaUFwcHMiLCJwaWkiOiJTMDA0NzI0ODQyMDMwMTEzNSIsInN1YiI6IjIyODU5OCIsImlzcyI6ImFycCIsInNlc3Npb25JZCI6IjRlMDllN2I1NjViYWEwNDg5NTU5OWI0N2M1NzMyNjNiNDgyZmd4cnFhIiwiZXhwIjoxNzQ5NjAzODA4LCJpYXQiOjE3NDk1OTY2MDgsInZlcnNpb24iOjEsImp0aSI6IjU5MjU4MmQ5LWU3MGUtNGI3My1iZTNiLWI3OTI3Y2ZmN2E0ZiJ9.vi0lkpYirSZofcCtiZfRDe3KEeYukudWSLaOju2lj5w","eid":"1-s2.0-S0047248420301135","doi":"10.1016/j.jhevol.2020.102852","first-fp":"102852","hub-eid":"1-s2.0-S0047248420X0008X","issuePii":"S0047248420X0008X","item-weight":"FULL-TEXT","language":"en","last-author":{"#name":"last-author","$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"$$":[{"#name":"author","$":{"id":"au11","orcid":"0000-0002-4760-558X","author-id":"S0047248420301135-a8d4ce801189d851e063a11bc1688235"},"$$":[{"#name":"given-name","_":"Matthias"},{"#name":"surname","_":"Meyer"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0095"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]}]},"normalized-first-auth-initial":"B","normalized-first-auth-surname":"EVANS","open-research":{"#name":"open-research","$":{"xmlns:xocs":true},"$$":[{"#name":"or-embargo-opening-date","_":"2021-08-08T00:00:00.000Z"}]},"pages":[{"first-page":"102852"}],"pii":"S0047248420301135","self-archiving":{"#name":"self-archiving","$":{"xmlns:xocs":true},"$$":[{"#name":"sa-start-date","_":"2021-08-08T00:00:00.000Z"},{"#name":"sa-user-license","_":"http://creativecommons.org/licenses/by-nc-nd/4.0/"}]},"srctitle":"Journal of Human Evolution","suppl":"C","timestamp":"2022-06-22T00:55:09.604948Z","title":{"content":[{"#name":"title","$":{"id":"title0010"},"$$":[{"#name":"__text__","_":"Mitogenomics of macaques ("},{"#name":"italic","_":"Macaca"},{"#name":"__text__","_":") across Wallace's Line in the context of modern human dispersals"}]}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"146","vol-iss-suppl-text":"Volume 146","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":true,"blockFullTextForAnonymousAccess":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":true,"document-references":108,"freeHtmlGiven":false,"ssoUrls":["//acw.clinicalkey.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y","//acw.scopus.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y","//acw.sciencedirect.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y","//acw.elsevier.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y"],"userProfile":{"departmentName":"ScienceDirect Guests","accessType":"GUEST","accountId":"228598","webUserId":"12975512","accountName":"ScienceDirect Guests","departmentId":"291352","userType":"NORMAL","hasMultipleOrganizations":false,"accountNumber":"C000228598"},"access":{"openAccess":true,"openArchive":false,"license":"http://creativecommons.org/licenses/by-nc-nd/4.0/"},"accessTypeLabel":"Open access","aipType":"none","articleEntitlement":{"entitled":true,"isCasaUser":false,"usageInfo":"(12975512,U|291352,D|228598,A|3,P|2,PL)(SDFE,CON|4e09e7b565baa04895599b47c573263b482fgxrqa,SSO|ANON_GUEST,ACCESS_TYPE)","entitledByAccount":false,"entitlementOrigin":"SD"},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"dates":{"Available online":"8 August 2020","Received":"6 April 2020","Revised":[],"Accepted":"28 June 2020","Publication date":"1 September 2020","Version of Record":"8 August 2020"},"documentTypeLabel":"Research article","downloadFullIssue":true,"entitlementReason":"openaccess","features":["aamAttachments","keywords","references","preview"],"hasBody":true,"has-large-authors":false,"hasScholarlyAbstract":true,"headerConfig":{"contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","userEmail":"","orgName":"ScienceDirect Guests","webUserId":"12975512","libraryBanner":null,"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasInstitutionalAssociation":false,"hasMultiOrg":false,"userType":"GUEST","userAnonymity":"ANON_GUEST","allowCart":true,"environment":"prod","cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com"},"isCorpReq":false,"isPdfFullText":false,"issn":"00472484","issn-primary-formatted":"0047-2484","issRange":"","isThirdParty":false,"licenseLink":{"prefix":"Under a Creative Commons ","text":"license","url":"http://creativecommons.org/licenses/by-nc-nd/4.0/"},"pageCount":14,"pdfDownload":{"isPdfFullText":false,"urlMetadata":{"queryParams":{"md5":"e4092d04c9edc296a398559bd6e896b0","pid":"1-s2.0-S0047248420301135-main.pdf"},"pii":"S0047248420301135","pdfExtension":"/pdfft","path":"science/article/pii"}},"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":165,"openArchiveStatus":false,"openArchiveArticleCount":64,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S0047248420X0008X-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S0047248420X0008X/cover/DOWNSAMPLED200/image/gif/5d8bbcd8ebdcab4be0d175fab03d3543/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"21362","pixel-height":"200","pixel-width":"150"},{"attachment-eid":"1-s2.0-S0047248420X0008X-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S0047248420X0008X/cover/DOWNSAMPLED/image/gif/38be25b04145a932e4a89d82e745157d/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"13197","pixel-height":"150","pixel-width":"113"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S00472484.gif","title":"journal-of-human-evolution","contentTypeCode":"JL","images":{"coverImage":"https://ars.els-cdn.com/content/image/1-s2.0-S0047248420X0008X-cov150h.gif","logo":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/image/elsevier-non-solus.svg","logoAltText":"Elsevier"},"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S0047248420X0008X-cov150h.gif"},"rightsLink":"https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&contentID=S0047248420301135&orderBeanReset=true","volRange":"146","titleString":"Mitogenomics of macaques (Macaca) across Wallace's Line in the context of modern human dispersals","ssrn":{},"renderingMode":"Article","isAbstract":false,"isContentVisible":false,"ajaxLinks":{"referenceLinks":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true,"citingArticles":true,"authorMetadata":true},"pdfEmbed":false,"displayViewFullText":false},"authors":{"content":[{"#name":"author-group","$":{"id":"augrp0010"},"$$":[{"#name":"author","$":{"id":"au1","orcid":"0000-0002-9512-8845","author-id":"S0047248420301135-7f7a7dea1d7c763fb8e48ed7ce318c4d"},"$$":[{"#name":"given-name","_":"Ben J."},{"#name":"surname","_":"Evans"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"crosref0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âˆ—"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMnhtbG5zJTNBeGxpbmslMjIlM0F0cnVlJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTJDJTIyaHJlZiUyMiUzQSUyMm1haWx0byUzQWV2YW5zYiU0MG1jbWFzdGVyLmNhJTIyJTJDJTIyaWQlMjIlM0ElMjJlYWRkMDAxMCUyMiU3RCUyQyUyMl8lMjIlM0ElMjJldmFuc2IlNDBtY21hc3Rlci5jYSUyMiU3RA=="}]},{"#name":"author","$":{"id":"au2","author-id":"S0047248420301135-9a8e468964583152f0640691aa5270e4"},"$$":[{"#name":"given-name","_":"Marie-Theres"},{"#name":"surname","_":"Gansauge"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"author","$":{"id":"au3","author-id":"S0047248420301135-4b6525603c3b204e983a51a94bbb76a0"},"$$":[{"#name":"given-name","_":"Matthew W."},{"#name":"surname","_":"Tocheri"},{"#name":"cross-ref","$":{"refid":"aff3","id":"crosref0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]},{"#name":"cross-ref","$":{"refid":"aff4","id":"crosref0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"cross-ref","$":{"refid":"aff5","id":"crosref0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]},{"#name":"author","$":{"id":"au4","author-id":"S0047248420301135-ebf69ff89ad6277d9f643ab4cc9e863a"},"$$":[{"#name":"given-name","_":"Michael A."},{"#name":"surname","_":"Schillaci"},{"#name":"cross-ref","$":{"refid":"aff6","id":"crosref0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"f"}]}]},{"#name":"author","$":{"id":"au5","author-id":"S0047248420301135-b115fe5b2f4f35f9d132b13230c19370"},"$$":[{"#name":"given-name","_":"Thomas"},{"#name":"surname","_":"Sutikna"},{"#name":"cross-ref","$":{"refid":"aff5","id":"crosref0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]},{"#name":"cross-ref","$":{"refid":"aff7","id":"crosref0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"g"}]},{"#name":"cross-ref","$":{"refid":"aff8","id":"crosref0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"h"}]}]},{"#name":"author","$":{"id":"au6","author-id":"S0047248420301135-70954b20c96e1d6be6300913dcac3dcf"},"$$":[{"#name":"surname","_":"Jatmiko"},{"#name":"cross-ref","$":{"refid":"aff5","id":"crosref0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]},{"#name":"cross-ref","$":{"refid":"aff8","id":"crosref0065"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"h"}]}]},{"#name":"author","$":{"id":"au7","author-id":"S0047248420301135-63270c7ec25e48139326b57d2524690f"},"$$":[{"#name":"given-name","_":"E. Wahyu"},{"#name":"surname","_":"Saptomo"},{"#name":"cross-ref","$":{"refid":"aff5","id":"crosref0070"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]},{"#name":"cross-ref","$":{"refid":"aff8","id":"crosref0075"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"h"}]}]},{"#name":"author","$":{"id":"au8","author-id":"S0047248420301135-10342571e485f5914c81564b9379ee9c"},"$$":[{"#name":"given-name","_":"Amy"},{"#name":"surname","_":"Klegarth"},{"#name":"cross-ref","$":{"refid":"aff9","id":"crosref0080"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"i"}]}]},{"#name":"author","$":{"id":"au9","orcid":"0000-0002-7935-7840","author-id":"S0047248420301135-4b593c6c9a96c06d329076793e7476c8"},"$$":[{"#name":"given-name","_":"Anthony J."},{"#name":"surname","_":"Tosi"},{"#name":"cross-ref","$":{"refid":"aff10","id":"crosref0085"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"j"}]}]},{"#name":"author","$":{"id":"au10","author-id":"S0047248420301135-81d70331d51ba13d17f4e3f9e543694b"},"$$":[{"#name":"given-name","_":"Don J."},{"#name":"surname","_":"Melnick"},{"#name":"cross-ref","$":{"refid":"aff11","id":"crosref0090"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"k"}]}]},{"#name":"author","$":{"id":"au11","orcid":"0000-0002-4760-558X","author-id":"S0047248420301135-a8d4ce801189d851e063a11bc1688235"},"$$":[{"#name":"given-name","_":"Matthias"},{"#name":"surname","_":"Meyer"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0095"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"affiliation","$":{"id":"aff1","affiliation-id":"S0047248420301135-ce27be7078e3c3607a4555d2a255b728"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Department of Biology, McMaster University, Life Sciences Building Room 328, 1280 Main Street West, Hamilton, ON, L8S 4K1, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Biology"},{"#name":"organization","_":"McMaster University"},{"#name":"address-line","_":"Life Sciences Building Room 328"},{"#name":"address-line","_":"1280 Main Street West"},{"#name":"city","_":"Hamilton"},{"#name":"state","_":"ON"},{"#name":"postal-code","_":"L8S 4K1"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0010"},"_":"Department of Biology, Life Sciences Building Room 328, McMaster University, 1280 Main Street West, Hamilton, ON, L8S 4K1, Canada"}]},{"#name":"affiliation","$":{"id":"aff2","affiliation-id":"S0047248420301135-59a76e98276afe82383247a5fad44224"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Max Planck Institute for Evolutionary Anthropology, Deutscher Platz 6, D-04103, Leipzig, Germany"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Max Planck Institute for Evolutionary Anthropology"},{"#name":"address-line","_":"Deutscher Platz 6"},{"#name":"city","_":"Leipzig"},{"#name":"postal-code","_":"D-04103"},{"#name":"country","_":"Germany"}]},{"#name":"source-text","$":{"id":"srct0015"},"_":"Max Planck Institute for Evolutionary Anthropology, Deutscher Platz 6, D-04103 Leipzig, Germany"}]},{"#name":"affiliation","$":{"id":"aff3","affiliation-id":"S0047248420301135-262ce999680b98293a9a1af02ab62f9e"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Department of Anthropology, Lakehead University, 955 Oliver Road, Thunder Bay, ON, P7B 5E1, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"Lakehead University"},{"#name":"address-line","_":"955 Oliver Road"},{"#name":"city","_":"Thunder Bay"},{"#name":"state","_":"ON"},{"#name":"postal-code","_":"P7B 5E1"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0020"},"_":"Department of Anthropology, Lakehead University, 955 Oliver Road, Thunder Bay, ON, P7B 5E1, Canada"}]},{"#name":"affiliation","$":{"id":"aff4","affiliation-id":"S0047248420301135-0c7cd7155d980c04a40199d90c7183d9"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Human Origins Program, National Museum of Natural History, Smithsonian Institution, Washington, DC, 20013, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Human Origins Program"},{"#name":"organization","_":"National Museum of Natural History"},{"#name":"organization","_":"Smithsonian Institution"},{"#name":"city","_":"Washington"},{"#name":"state","_":"DC"},{"#name":"postal-code","_":"20013"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0025"},"_":"Human Origins Program, National Museum of Natural History, Smithsonian Institution, Washington DC 20013, USA"}]},{"#name":"affiliation","$":{"id":"aff5","affiliation-id":"S0047248420301135-ae9553eb6adcbe9761914749858114c0"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","_":"Australian Research Council Centre of Excellence for Australian Biodiversity and Heritage, University of Wollongong, Wollongong, New South Wales, 2522, Australia"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Australian Research Council Centre of Excellence for Australian Biodiversity and Heritage"},{"#name":"organization","_":"University of Wollongong"},{"#name":"city","_":"Wollongong"},{"#name":"state","_":"New South Wales"},{"#name":"postal-code","_":"2522"},{"#name":"country","_":"Australia"}]},{"#name":"source-text","$":{"id":"srct0030"},"_":"Australian Research Council Centre of Excellence for Australian Biodiversity and Heritage, University of Wollongong, Wollongong, New South Wales 2522, Australia"}]},{"#name":"affiliation","$":{"id":"aff6","affiliation-id":"S0047248420301135-3e9bad0d3ceec56e22f97babed15e83c"},"$$":[{"#name":"label","_":"f"},{"#name":"textfn","_":"Department of Anthropology, University of Toronto Scarborough, 1265 Military Trail, Toronto, ON, M1C 1A4, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"University of Toronto Scarborough"},{"#name":"address-line","_":"1265 Military Trail"},{"#name":"city","_":"Toronto"},{"#name":"state","_":"ON"},{"#name":"postal-code","_":"M1C 1A4"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0035"},"_":"Department of Anthropology, University of Toronto Scarborough, 1265 Military Trail, Toronto, ON, M1C 1A4, Canada"}]},{"#name":"affiliation","$":{"id":"aff7","affiliation-id":"S0047248420301135-0dfa9ee0df03ab63725b08ee93cac2e2"},"$$":[{"#name":"label","_":"g"},{"#name":"textfn","_":"Centre for Archaeological Science, School of Earth and Environmental Sciences, University of Wollongong, Wollongong, New South Wales, 2522, Australia"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Centre for Archaeological Science"},{"#name":"organization","_":"School of Earth and Environmental Sciences"},{"#name":"organization","_":"University of Wollongong"},{"#name":"city","_":"Wollongong"},{"#name":"state","_":"New South Wales"},{"#name":"postal-code","_":"2522"},{"#name":"country","_":"Australia"}]},{"#name":"source-text","$":{"id":"srct0040"},"_":"Centre for Archaeological Science, School of Earth and Environmental Sciences, University of Wollongong, Wollongong, New South Wales 2522, Australia"}]},{"#name":"affiliation","$":{"id":"aff8","affiliation-id":"S0047248420301135-17921b07e5fad932adfb6154b2dda1e9"},"$$":[{"#name":"label","_":"h"},{"#name":"textfn","_":"Pusat Penelitian Arkeologi Nasional, Jalan Raya Condet Pejatan No. 4, Jakarta, 12510, Indonesia"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Pusat Penelitian Arkeologi Nasional"},{"#name":"address-line","_":"Jalan Raya Condet Pejatan No. 4"},{"#name":"city","_":"Jakarta"},{"#name":"postal-code","_":"12510"},{"#name":"country","_":"Indonesia"}]},{"#name":"source-text","$":{"id":"srct0045"},"_":"Pusat Penelitian Arkeologi Nasional, Jalan Raya Condet Pejatan No. 4, Jakarta 12510 Indonesia"}]},{"#name":"affiliation","$":{"id":"aff9","affiliation-id":"S0047248420301135-9eedcfd689179878ddd71789721ce52f"},"$$":[{"#name":"label","_":"i"},{"#name":"textfn","_":"Department of Anthropology, University of Washington, 14 Denny Hall, Box 353100, Seattle, WA, 98195-3100, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"University of Washington"},{"#name":"address-line","_":"14 Denny Hall"},{"#name":"address-line","_":"Box 353100"},{"#name":"city","_":"Seattle"},{"#name":"state","_":"WA"},{"#name":"postal-code","_":"98195-3100"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0050"},"_":"Department of Anthropology, University of Washington, 14 Denny Hall, Box 353100, Seattle, WA 98195-3100, USA"}]},{"#name":"affiliation","$":{"id":"aff10","affiliation-id":"S0047248420301135-e68a7cc3bb0e63b371e51a239d88a202"},"$$":[{"#name":"label","_":"j"},{"#name":"textfn","_":"Department of Anthropology, Kent State University, 750 Hilltop Drive, 226 Lowry Hall, Kent, OH, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"Kent State University"},{"#name":"address-line","_":"750 Hilltop Drive"},{"#name":"address-line","_":"226 Lowry Hall"},{"#name":"city","_":"Kent"},{"#name":"state","_":"OH"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0055"},"_":"Department of Anthropology, 750 Hilltop Drive, 226 Lowry Hall, Kent State University, Kent, OH, USA"}]},{"#name":"affiliation","$":{"id":"aff11","affiliation-id":"S0047248420301135-044a7c5dd8442c838831844bf1f57ead"},"$$":[{"#name":"label","_":"k"},{"#name":"textfn","_":"Department of Ecology, Evolution, and Environmental Biology, Columbia University, 10th Floor Schermerhorn Extension, 116th Street and Broadway, New York, NY, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Ecology, Evolution, and Environmental Biology"},{"#name":"organization","_":"Columbia University"},{"#name":"address-line","_":"10th Floor Schermerhorn Extension"},{"#name":"address-line","_":"116th Street and Broadway"},{"#name":"city","_":"New York"},{"#name":"state","_":"NY"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0060"},"_":"Department of Ecology, Evolution, and Environmental Biology, 10th Floor Schermerhorn Extension, 116th Street and Broadway, Columbia University, New York, NY, USA"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"âˆ—"},{"#name":"text","_":"Corresponding author."}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1","affiliation-id":"S0047248420301135-ce27be7078e3c3607a4555d2a255b728"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Department of Biology, McMaster University, Life Sciences Building Room 328, 1280 Main Street West, Hamilton, ON, L8S 4K1, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Biology"},{"#name":"organization","_":"McMaster University"},{"#name":"address-line","_":"Life Sciences Building Room 328"},{"#name":"address-line","_":"1280 Main Street West"},{"#name":"city","_":"Hamilton"},{"#name":"state","_":"ON"},{"#name":"postal-code","_":"L8S 4K1"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0010"},"_":"Department of Biology, Life Sciences Building Room 328, McMaster University, 1280 Main Street West, Hamilton, ON, L8S 4K1, Canada"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2","affiliation-id":"S0047248420301135-59a76e98276afe82383247a5fad44224"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Max Planck Institute for Evolutionary Anthropology, Deutscher Platz 6, D-04103, Leipzig, Germany"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Max Planck Institute for Evolutionary Anthropology"},{"#name":"address-line","_":"Deutscher Platz 6"},{"#name":"city","_":"Leipzig"},{"#name":"postal-code","_":"D-04103"},{"#name":"country","_":"Germany"}]},{"#name":"source-text","$":{"id":"srct0015"},"_":"Max Planck Institute for Evolutionary Anthropology, Deutscher Platz 6, D-04103 Leipzig, Germany"}]},"aff3":{"#name":"affiliation","$":{"id":"aff3","affiliation-id":"S0047248420301135-262ce999680b98293a9a1af02ab62f9e"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Department of Anthropology, Lakehead University, 955 Oliver Road, Thunder Bay, ON, P7B 5E1, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"Lakehead University"},{"#name":"address-line","_":"955 Oliver Road"},{"#name":"city","_":"Thunder Bay"},{"#name":"state","_":"ON"},{"#name":"postal-code","_":"P7B 5E1"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0020"},"_":"Department of Anthropology, Lakehead University, 955 Oliver Road, Thunder Bay, ON, P7B 5E1, Canada"}]},"aff4":{"#name":"affiliation","$":{"id":"aff4","affiliation-id":"S0047248420301135-0c7cd7155d980c04a40199d90c7183d9"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Human Origins Program, National Museum of Natural History, Smithsonian Institution, Washington, DC, 20013, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Human Origins Program"},{"#name":"organization","_":"National Museum of Natural History"},{"#name":"organization","_":"Smithsonian Institution"},{"#name":"city","_":"Washington"},{"#name":"state","_":"DC"},{"#name":"postal-code","_":"20013"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0025"},"_":"Human Origins Program, National Museum of Natural History, Smithsonian Institution, Washington DC 20013, USA"}]},"aff5":{"#name":"affiliation","$":{"id":"aff5","affiliation-id":"S0047248420301135-ae9553eb6adcbe9761914749858114c0"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","_":"Australian Research Council Centre of Excellence for Australian Biodiversity and Heritage, University of Wollongong, Wollongong, New South Wales, 2522, Australia"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Australian Research Council Centre of Excellence for Australian Biodiversity and Heritage"},{"#name":"organization","_":"University of Wollongong"},{"#name":"city","_":"Wollongong"},{"#name":"state","_":"New South Wales"},{"#name":"postal-code","_":"2522"},{"#name":"country","_":"Australia"}]},{"#name":"source-text","$":{"id":"srct0030"},"_":"Australian Research Council Centre of Excellence for Australian Biodiversity and Heritage, University of Wollongong, Wollongong, New South Wales 2522, Australia"}]},"aff6":{"#name":"affiliation","$":{"id":"aff6","affiliation-id":"S0047248420301135-3e9bad0d3ceec56e22f97babed15e83c"},"$$":[{"#name":"label","_":"f"},{"#name":"textfn","_":"Department of Anthropology, University of Toronto Scarborough, 1265 Military Trail, Toronto, ON, M1C 1A4, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"University of Toronto Scarborough"},{"#name":"address-line","_":"1265 Military Trail"},{"#name":"city","_":"Toronto"},{"#name":"state","_":"ON"},{"#name":"postal-code","_":"M1C 1A4"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0035"},"_":"Department of Anthropology, University of Toronto Scarborough, 1265 Military Trail, Toronto, ON, M1C 1A4, Canada"}]},"aff7":{"#name":"affiliation","$":{"id":"aff7","affiliation-id":"S0047248420301135-0dfa9ee0df03ab63725b08ee93cac2e2"},"$$":[{"#name":"label","_":"g"},{"#name":"textfn","_":"Centre for Archaeological Science, School of Earth and Environmental Sciences, University of Wollongong, Wollongong, New South Wales, 2522, Australia"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Centre for Archaeological Science"},{"#name":"organization","_":"School of Earth and Environmental Sciences"},{"#name":"organization","_":"University of Wollongong"},{"#name":"city","_":"Wollongong"},{"#name":"state","_":"New South Wales"},{"#name":"postal-code","_":"2522"},{"#name":"country","_":"Australia"}]},{"#name":"source-text","$":{"id":"srct0040"},"_":"Centre for Archaeological Science, School of Earth and Environmental Sciences, University of Wollongong, Wollongong, New South Wales 2522, Australia"}]},"aff8":{"#name":"affiliation","$":{"id":"aff8","affiliation-id":"S0047248420301135-17921b07e5fad932adfb6154b2dda1e9"},"$$":[{"#name":"label","_":"h"},{"#name":"textfn","_":"Pusat Penelitian Arkeologi Nasional, Jalan Raya Condet Pejatan No. 4, Jakarta, 12510, Indonesia"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Pusat Penelitian Arkeologi Nasional"},{"#name":"address-line","_":"Jalan Raya Condet Pejatan No. 4"},{"#name":"city","_":"Jakarta"},{"#name":"postal-code","_":"12510"},{"#name":"country","_":"Indonesia"}]},{"#name":"source-text","$":{"id":"srct0045"},"_":"Pusat Penelitian Arkeologi Nasional, Jalan Raya Condet Pejatan No. 4, Jakarta 12510 Indonesia"}]},"aff9":{"#name":"affiliation","$":{"id":"aff9","affiliation-id":"S0047248420301135-9eedcfd689179878ddd71789721ce52f"},"$$":[{"#name":"label","_":"i"},{"#name":"textfn","_":"Department of Anthropology, University of Washington, 14 Denny Hall, Box 353100, Seattle, WA, 98195-3100, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"University of Washington"},{"#name":"address-line","_":"14 Denny Hall"},{"#name":"address-line","_":"Box 353100"},{"#name":"city","_":"Seattle"},{"#name":"state","_":"WA"},{"#name":"postal-code","_":"98195-3100"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0050"},"_":"Department of Anthropology, University of Washington, 14 Denny Hall, Box 353100, Seattle, WA 98195-3100, USA"}]},"aff10":{"#name":"affiliation","$":{"id":"aff10","affiliation-id":"S0047248420301135-e68a7cc3bb0e63b371e51a239d88a202"},"$$":[{"#name":"label","_":"j"},{"#name":"textfn","_":"Department of Anthropology, Kent State University, 750 Hilltop Drive, 226 Lowry Hall, Kent, OH, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"Kent State University"},{"#name":"address-line","_":"750 Hilltop Drive"},{"#name":"address-line","_":"226 Lowry Hall"},{"#name":"city","_":"Kent"},{"#name":"state","_":"OH"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0055"},"_":"Department of Anthropology, 750 Hilltop Drive, 226 Lowry Hall, Kent State University, Kent, OH, USA"}]},"aff11":{"#name":"affiliation","$":{"id":"aff11","affiliation-id":"S0047248420301135-044a7c5dd8442c838831844bf1f57ead"},"$$":[{"#name":"label","_":"k"},{"#name":"textfn","_":"Department of Ecology, Evolution, and Environmental Biology, Columbia University, 10th Floor Schermerhorn Extension, 116th Street and Broadway, New York, NY, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Ecology, Evolution, and Environmental Biology"},{"#name":"organization","_":"Columbia University"},{"#name":"address-line","_":"10th Floor Schermerhorn Extension"},{"#name":"address-line","_":"116th Street and Broadway"},{"#name":"city","_":"New York"},{"#name":"state","_":"NY"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0060"},"_":"Department of Ecology, Evolution, and Environmental Biology, 10th Floor Schermerhorn Extension, 116th Street and Broadway, Columbia University, New York, NY, USA"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"âˆ—"},{"#name":"text","_":"Corresponding author."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"authorMetadata":[],"banner":{"expanded":false},"biographies":{},"body":{},"chapters":{"toc":[],"isLoading":false},"changeViewLinks":{"showFullTextLink":false,"showAbstractLink":false},"citingArticles":{},"clientConfig":{"adobeSuite":"elsevier-sd-prod","arsUrl":"https://ars.els-cdn.com","assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1","recommendationsFeedback":{"enabled":true,"url":"https://feedback.recs.d.elsevier.com/raw/events","timeout":60000},"googleMapsApiKey":"AIzaSyCBYU6I6lrbEU6wQXUEIte3NwGtm3jwHQc","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","xocsMediaPlayerBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/147/js/ableplayer","strictMode":false,"seamlessAccess":{"enableSeamlessAccess":true,"scriptUrl":"https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js","persistenceUrl":"https://service.seamlessaccess.org/ps/","persistenceContext":"seamlessaccess.org","scienceDirectUrl":"https://www.sciencedirect.com","shibAuthUrl":"https://auth.elsevier.com/ShibAuth/institutionLogin"},"reaxys":{"apiUrl":"https://reaxys-sdlc.reaxys.com","origin":"sciencedirect","queryBuilderHostPath":"https://www.reaxys.com/reaxys/secured/hopinto.do","url":"https://www.reaxys.com"},"oneTrustCookie":{"enabled":true},"ssrn":{"url":"https://papers.ssrn.com","path":"/sol3/papers.cfm"},"aiNavigation":{"enabled":true,"version":"V1"}},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"lang":"en","id":"kwrds0010","view":"all","class":"keyword"},"$$":[{"#name":"section-title","$":{"id":"sectitle0015"},"_":"Keywords"},{"#name":"keyword","$":{"id":"kwrd0010"},"$$":[{"#name":"text","_":"Mitochondrial genomes"}]},{"#name":"keyword","$":{"id":"kwrd0015"},"$$":[{"#name":"text","_":"Primate evolution"}]},{"#name":"keyword","$":{"id":"kwrd0020"},"$$":[{"#name":"text","_":"Human cultural diasporas"}]},{"#name":"keyword","$":{"id":"kwrd0025"},"$$":[{"#name":"text","_":"Introduced species"}]},{"#name":"keyword","$":{"id":"kwrd0030"},"$$":[{"#name":"text","_":"Recombination in mitochondrial DNA"}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"crossMark":{"isOpen":false},"domainConfig":{"assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1","cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com"},"downloadIssue":{"openOnPageLoad":false,"isOpen":false,"downloadCapOpen":false,"articles":[],"selected":[]},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"entitledRecommendations":{"openOnPageLoad":false,"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"exam":{},"helpText":{"keyDates":{"html":"<div class=\"key-dates-help\"><h3 class=\"u-margin-s-bottom u-h4\">Publication milestones</h3><p class=\"u-margin-m-bottom\">The dates displayed for an article provide information on when various publication milestones were reached at the journal that has published the article. Where applicable, activities on preceding journals at which the article was previously under consideration are not shown (for instance submission, revisions, rejection).</p><p class=\"u-margin-xs-bottom\">The publication milestones include:</p><ul class=\"key-dates-help-list u-margin-m-bottom u-padding-s-left\"><li><span class=\"u-text-italic\">Received</span>: The date the article was originally submitted to the journal.</li><li><span class=\"u-text-italic\">Revised</span>: The date the most recent revision of the article was submitted to the journal. Dates corresponding to intermediate revisions are not shown.</li><li><span class=\"u-text-italic\">Accepted</span>: The date the article was accepted for publication in the journal.</li><li><span class=\"u-text-italic\">Available online</span>: The date a version of the article was made available online in the journal.</li><li><span class=\"u-text-italic\">Version of Record</span>: The date the finalized version of the article was made available in the journal.</li></ul><p>More information on publishing policies can be found on the <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://www.elsevier.com/about/policies-and-standards/publishing-ethics\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">Publishing Ethics Policies</span></span></a> page. View our <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://www.elsevier.com/researcher/author/submit-your-paper\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">Publishing with Elsevier: step-by-step</span></span></a> page to learn more about the publishing process. For any questions on your own submission or other questions related to publishing an article, <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://service.elsevier.com/app/phone/supporthub/publishing\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">contact our Researcher support team.</span></span></a></p></div>","title":"What do these dates mean?"}},"glossary":{},"issueNavigation":{"previous":{},"next":{}},"linkingHubLinks":{},"metrics":{"metricGroup":{"citations":[],"captures":[],"mentions":[],"socialMedia":[]},"isLoading":false,"error":false,"hasMetrics":false},"preview":{},"rawtext":"","recommendations":{},"references":{},"referenceLinks":{"internal":[],"internalLoaded":false,"external":[]},"refersTo":{},"referredToBy":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenVirtualSpecialIssueLink":false,"isOpenRecommendations":true,"isOpenSubstances":true,"citingArticles":[false,false,false,false,false,false],"recommendations":[false,false,false,false,false,false]},"seamlessAccess":{},"specialIssueArticles":{},"substances":{},"supplementaryFilesData":[],"tableOfContents":{"showEntitledTocLinks":true},"tail":{},"transientError":{"isOpen":false},"sidePanel":{"openState":1},"viewConfig":{"articleFeature":{"aiNavigation":false},"pathPrefix":""},"virtualSpecialIssue":{"showVirtualSpecialIssueLink":false},"usageProps":{"itemStage":"S300","isAip":false,"tombAip":"0","sample":false},"userCookiePreferences":{"STRICTLY_NECESSARY":true,"PERFORMANCE":false,"FUNCTIONAL":false,"TARGETING":false}};
      </script>
      <noscript>
      JavaScript is disabled on your browser.
      Please enable JavaScript to use all the features on this page.
      <img src=https://smetrics.elsevier.com/b/ss/elsevier-sd-prod/1/G.4--NS/1749596608062?pageName=sd%3Aproduct%3Ajournal%3Aarticle&c16=els%3Arp%3Ast&c2=sd&v185=img&v33=ae%3AANON_GUEST&c1=ae%3A228598&c12=ae%3A12975512 />
    </noscript>
      <a class="anchor sr-only sr-only-focusable u-display-inline anchor-primary" href="#screen-reader-main-content"><span class="anchor-text-container"><span class="anchor-text">Skip to main content</span></span></a><a class="anchor sr-only sr-only-focusable u-display-inline anchor-primary" href="#screen-reader-main-title"><span class="anchor-text-container"><span class="anchor-text">Skip to article</span></span></a>
      <div id="root"><div class="App" id="app" data-aa-name="root"><div class="page"><div class="sd-flex-container"><div class="sd-flex-content"><header id="gh-cnt"><div id="gh-main-cnt" class="u-flex-center-ver u-position-relative u-padding-s-hor u-padding-l-hor-from-xl"><a id="gh-branding" class="u-flex-center-ver" href="/" aria-label="ScienceDirect home page" data-aa-region="header" data-aa-name="ScienceDirect"><img class="gh-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/24/images/elsevier-non-solus-new-grey.svg" alt="Elsevier logo" height="48" width="54"><svg xmlns="http://www.w3.org/2000/svg" version="1.1" height="15" viewBox="0 0 190 23" role="img" class="gh-wordmark u-margin-s-left" aria-labelledby="gh-wm-science-direct" focusable="false" aria-hidden="true" alt="ScienceDirect Wordmark"><title id="gh-wm-science-direct">ScienceDirect</title><g><path fill="#EB6500" d="M3.81 6.9c0-1.48 0.86-3.04 3.7-3.04 1.42 0 3.1 0.43 4.65 1.32l0.13-2.64c-1.42-0.63-2.97-0.96-4.78-0.96 -4.62 0-6.6 2.44-6.6 5.45 0 5.61 8.78 6.14 8.78 9.93 0 1.48-1.15 3.04-3.86 3.04 -1.72 0-3.4-0.56-4.72-1.39l-0.36 2.64c1.55 0.76 3.57 1.06 5.15 1.06 4.26 0 6.7-2.48 6.7-5.51C12.59 11.49 3.81 10.76 3.81 6.9M20.27 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.14 0-4.55-1.71-4.55-5.91C17.93 10.2 20.01 9.18 20.27 9.01"></path><rect x="29.42" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M30.67 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.68-0.96 1.68-1.88C32.35 1.55 31.56 0.7 30.67 0.7M48.06 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H48.06M39.91 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C38.56 10.27 39.71 9.37 39.91 9.18zM58.82 6.57c-2.24 0-3.63 1.12-4.85 2.61l-0.4-2.21h-2.34l0.13 1.19c0.1 0.76 0.13 1.78 0.13 2.97v10.79h2.54V11.88c0.69-0.96 2.15-2.48 2.48-2.64 0.23-0.13 1.29-0.4 2.08-0.4 2.28 0 2.48 1.15 2.54 3.43 0.03 1.19 0.03 3.17 0.03 3.17 0.03 3-0.1 6.47-0.1 6.47h2.54c0 0 0.07-4.49 0.07-6.96 0-1.48 0.03-2.97-0.1-4.46C63.31 7.43 61.49 6.57 58.82 6.57M72.12 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C69.77 10.2 71.85 9.18 72.12 9.01M92.74 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H92.74M84.59 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C83.24 10.27 84.39 9.37 84.59 9.18zM103.9 1.98h-7.13v19.93h6.83c7.26 0 9.77-5.68 9.77-10.03C113.37 7.33 110.93 1.98 103.9 1.98M103.14 19.8h-3.76V4.1h4.09c5.38 0 6.96 4.39 6.96 7.79C110.43 16.87 108.19 19.8 103.14 19.8zM118.38 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.69-0.96 1.69-1.88C120.07 1.55 119.28 0.7 118.38 0.7"></path><rect x="117.13" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M130.2 6.6c-1.62 0-2.87 1.45-3.4 2.74l-0.43-2.37h-2.34l0.13 1.19c0.1 0.76 0.13 1.75 0.13 2.9v10.86h2.54v-9.51c0.53-1.29 1.72-3.7 3.17-3.7 0.96 0 1.06 0.99 1.06 1.22l2.08-0.6V9.18c0-0.03-0.03-0.17-0.06-0.4C132.8 7.36 131.91 6.6 130.2 6.6M145.87 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.89-1.95-4.89-5.51v-0.49H145.87M137.72 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C136.37 10.27 137.52 9.37 137.72 9.18zM153.23 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.14-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.69 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C150.89 10.2 152.97 9.18 153.23 9.01M170 19.44c-0.92 0.36-1.72 0.69-2.51 0.69 -1.16 0-1.58-0.66-1.58-2.34V8.95h3.93V6.97h-3.93V2.97h-2.48v3.99h-2.71v1.98h2.71v9.67c0 2.64 1.39 3.73 3.33 3.73 1.15 0 2.54-0.39 3.43-0.79L170 19.44M173.68 5.96c-1.09 0-2-0.87-2-1.97 0-1.1 0.91-1.97 2-1.97s1.98 0.88 1.98 1.98C175.66 5.09 174.77 5.96 173.68 5.96zM173.67 2.46c-0.85 0-1.54 0.67-1.54 1.52 0 0.85 0.69 1.54 1.54 1.54 0.85 0 1.54-0.69 1.54-1.54C175.21 3.13 174.52 2.46 173.67 2.46zM174.17 5.05c-0.09-0.09-0.17-0.19-0.25-0.3l-0.41-0.56h-0.16v0.87h-0.39V2.92c0.22-0.01 0.47-0.03 0.66-0.03 0.41 0 0.82 0.16 0.82 0.64 0 0.29-0.21 0.55-0.49 0.63 0.23 0.32 0.45 0.62 0.73 0.91H174.17zM173.56 3.22l-0.22 0.01v0.63h0.22c0.26 0 0.43-0.05 0.43-0.34C174 3.28 173.83 3.21 173.56 3.22z"></path></g></svg></a><div class="gh-nav-cnt u-hide-from-print"><div class="gh-nav-links-container gh-nav-links-container-h u-hide-from-print gh-nav-content-container"><nav aria-label="links" class="gh-nav gh-nav-links gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item gh-move-to-spine"><a class="anchor gh-nav-action text-s anchor-secondary anchor-medium" href="/browse/journals-and-books" id="gh-journals-books-link" data-aa-region="header" data-aa-name="Journals &amp; Books"><span class="anchor-text-container"><span class="anchor-text">Journals &amp; Books</span></span></a></li></ul></nav><nav aria-label="utilities" class="gh-nav gh-nav-utilities gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-help text-s u-flex-center-ver u-gap-6 gh-nav-action"><div class="gh-move-to-spine gh-help-button gh-help-icon gh-nav-item"><div class="popover" id="gh-help-icon-popover"><div id="popover-trigger-gh-help-icon-popover"><input type="hidden"><button class="button-link button-link-secondary gh-icon-btn button-link-medium button-link-icon-left" type="button" aria-expanded="false"><svg focusable="false" viewBox="0 0 114 128" height="20" width="20" class="icon icon-help gh-icon"><path d="M57 8C35.69 7.69 15.11 21.17 6.68 40.71c-8.81 19.38-4.91 43.67 9.63 59.25 13.81 15.59 36.85 21.93 56.71 15.68 21.49-6.26 37.84-26.81 38.88-49.21 1.59-21.15-10.47-42.41-29.29-52.1C74.76 10.17 65.88 7.99 57 8zm0 10c20.38-.37 39.57 14.94 43.85 34.85 4.59 18.53-4.25 39.23-20.76 48.79-17.05 10.59-40.96 7.62-54.9-6.83-14.45-13.94-17.42-37.85-6.83-54.9C26.28 26.5 41.39 17.83 57 18zm-.14 14C45.31 32.26 40 40.43 40 50v2h10v-2c0-4.22 2.22-9.66 8-9.24 5.5.4 6.32 5.14 5.78 8.14C62.68 55.06 52 58.4 52 69.4V76h10v-5.56c0-8.16 11.22-11.52 12-21.7.74-9.86-5.56-16.52-16-16.74-.39-.01-.76-.01-1.14 0zM52 82v10h10V82H52z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Help</span></span></button></div></div></div></li><li class="gh-nav-search text-s u-flex-center-ver u-gap-6 gh-nav-action"><div class="gh-search-toggle gh-nav-item search-button-link"><a class="anchor button-link-secondary anchor-secondary u-margin-l-left gh-nav-action gh-icon-btn anchor-medium anchor-icon-left anchor-with-icon" href="/search" id="gh-search-link" data-aa-button="search-in-header-opened-from-article" role="button"><svg focusable="false" viewBox="0 0 100 128" height="20" class="icon icon-search gh-icon"><path d="M19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6L73.23 80.19c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96S19.9 18.47 12.19 26.18C4.47 33.89.22 44.15.22 55.06s4.25 21.17 11.97 28.88C19.9 91.65 30.16 95.9 41.07 95.9c9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg><span class="anchor-text-container"><span class="anchor-text">Search</span></span></a></div></li></ul></nav></div></div><div class="gh-profile-container gh-move-to-spine u-hide-from-print"><a class="anchor text-s u-clr-grey8 u-margin-l-left gh-icon-btn anchor-primary anchor-medium anchor-icon-left anchor-with-icon" href="/user/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0047248420301135&amp;from=globalheader" id="gh-myaccount-btn" data-aa-region="header" data-aa-name="personalsignin"><svg focusable="false" viewBox="0 0 106 128" height="20" aria-hidden="true" class="icon icon-person gh-cta-btn-icon"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><span class="anchor-text-container"><span class="anchor-text">My account</span></span></a></div><a class="anchor text-s u-clr-grey8 gh-move-to-spine u-hide-from-print u-margin-l-left anchor-secondary gh-icon-btn anchor-medium anchor-icon-left anchor-with-icon" href="/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0047248420301135" id="gh-institutionalsignin-btn" data-aa-region="header" data-aa-name="institutionalsignin"><svg focusable="false" viewBox="0 0 106 128" height="20" aria-hidden="true" class="icon icon-institution gh-cta-btn-icon"><path d="M84 98h10v10H12V98h10V52h14v46h10V52h14v46h10V52h14v46zM12 36.86l41-20.84 41 20.84V42H12v-5.14zM104 52V30.74L53 4.8 2 30.74V52h10v36H2v30h102V88H94V52h10z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Sign in</span></span></a><div id="gh-mobile-menu" class="mobile-menu u-hide-from-print"><div class="gh-hamburger u-fill-grey7"><button class="button-link u-flex-center-ver button-link-primary button-link-icon-left" type="button" aria-label="Toggle mobile menu" aria-expanded="false"><svg class="gh-hamburger-svg-el gh-hamburger-closed" role="img" aria-hidden="true" height="20" width="20"><path d="M0 14h40v2H0zm0-7h40v2H0zm0-7h40v2H0z"></path></svg></button></div><div id="gh-overlay" class="mobile-menu-overlay u-overlay u-display-none" role="button" tabindex="-1"></div><div id="gh-drawer" aria-label="Mobile menu" class="" role="navigation"></div></div></div></header><div id="sd-ai-navigation-container"></div><div id="reading-assistant-main-body-section"></div><div class="Article" id="mathjax-container" role="main"><div class="accessbar-sticky"><div id="screen-reader-main-content"></div><div role="region" aria-label="Download options and search"><div class="accessbar"><div class="accessbar-label"></div><ul aria-label="PDF Options"><li class="ViewPDF"><a class="link-button accessbar-utility-component accessbar-utility-link link-button-primary link-button-icon-left" target="_blank" aria-label="View PDF. Opens in a new window." href="/science/article/pii/S0047248420301135/pdfft?md5=e4092d04c9edc296a398559bd6e896b0&amp;pid=1-s2.0-S0047248420301135-main.pdf" rel="nofollow"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="link-button-text-container"><span class="link-button-text"><span>View&nbsp;<strong>PDF</strong></span></span></span></a></li><li class="DownloadFullIssue"><button class="button-link accessbar-utility-component button-link-primary" type="button" aria-label="Download full issue"><span class="button-link-text-container"><span class="button-link-text"><span>Download full issue</span></span></span></button></li></ul><form class="QuickSearch" action="/search#submit" method="get" aria-label="form"><div class="search-input"><div class="search-input-container search-input-container-no-label"><label class="search-input-label u-hide-visually" for="article-quick-search">Search ScienceDirect</label><input type="search" id="article-quick-search" name="qs" class="search-input-field" aria-describedby="article-quick-search-description-message" aria-invalid="false" aria-label="Search ScienceDirect" placeholder="Search ScienceDirect" value=""></div><div class="search-input-message-container"><div class="search-input-validation-error" aria-live="polite"></div><div id="article-quick-search-description-message"></div></div></div><button type="submit" class="button u-margin-xs-left button-primary small button-icon-only" aria-disabled="false" aria-label="Submit search"><svg focusable="false" viewBox="0 0 100 128" height="20" class="icon icon-search"><path d="M19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6L73.23 80.19c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96S19.9 18.47 12.19 26.18C4.47 33.89.22 44.15.22 55.06s4.25 21.17 11.97 28.88C19.9 91.65 30.16 95.9 41.07 95.9c9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></button><input type="hidden" name="origin" value="article"><input type="hidden" name="zone" value="qSearch"></form></div></div></div><div class="article-wrapper grid row"><div role="navigation" class="u-display-block-from-lg col-lg-6 u-padding-s-top sticky-table-of-contents" aria-label="Table of contents"><div class="TableOfContents" lang="en"><div class="Outline" id="toc-outline"><h2 class="u-h4">Outline</h2><ol class="u-padding-xs-bottom"><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#abs0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Abstract"><span class="anchor-text-container"><span class="anchor-text">Abstract</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#kwrds0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Keywords"><span class="anchor-text-container"><span class="anchor-text">Keywords</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec1" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="1. Introduction"><span class="anchor-text-container"><span class="anchor-text">1. Introduction</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec2" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="2. Methods"><span class="anchor-text-container"><span class="anchor-text">2. Methods</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec3" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="3. Results"><span class="anchor-text-container"><span class="anchor-text">3. Results</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec4" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="4. Discussion"><span class="anchor-text-container"><span class="anchor-text">4. Discussion</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec5" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="5. Conclusions"><span class="anchor-text-container"><span class="anchor-text">5. Conclusions</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec6" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Data accessibility"><span class="anchor-text-container"><span class="anchor-text">Data accessibility</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec7" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Conflict of interest"><span class="anchor-text-container"><span class="anchor-text">Conflict of interest</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#ack0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Acknowledgments"><span class="anchor-text-container"><span class="anchor-text">Acknowledgments</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#appsec1" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Supplementary Online Material"><span class="anchor-text-container"><span class="anchor-text">Supplementary Online Material</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#cebib0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="References"><span class="anchor-text-container"><span class="anchor-text">References</span></span></a></li></ol><button class="button-link u-margin-xs-top u-margin-s-bottom button-link-primary button-link-icon-right" type="button" aria-expanded="false" data-aa-button="sd:product:journal:article:type=menu:name=show-full-outline"><span class="button-link-text-container"><span class="button-link-text">Show full outline</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="CitedBy" id="toc-cited-by"><h2 class="u-h4"><a class="anchor anchor-primary" href="#section-cited-by"><span class="anchor-text-container"><span class="anchor-text">Cited by (23)</span></span></a></h2><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2 class="u-h4">Figures (5)</h2><ol class="u-margin-s-bottom"><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig1" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure1. Macaque distributions in Southeast Asia" class="u-display-block" height="164px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr1.sml" width="176px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig2" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure2. Inferred evolutionary relationships among mitochondrial genomes offer insightsâ€¦" class="u-display-block" height="163px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr2.sml" width="114px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig3" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure3. Detailed relationships among clade B of M" class="u-display-block" height="162px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr3.sml" width="58px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig4" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure4. Inferred evolutionary relationships among mitochondrial genomes of silenusâ€¦" class="u-display-block" height="164px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr4.sml" width="132px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig5" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure5. Two alternative dispersal scenarios for the ancestors of M" class="u-display-block" height="100px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr5.sml" width="219px"></a></li></ol><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2 class="u-h4">Tables (1)</h2><ol class="u-padding-s-bottom"><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl1" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="14C ages for bone and tooth fragments of long-tailed macaques recovered from Liang Bua (Flores, Indonesia)."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a></li></ol><div class="PageDivider"></div></div><div class="Extras" id="toc-extras"><h2 class="u-h4">Extras (2)</h2><ol class="u-padding-s-bottom"><li><span class="download-all-supplemental-data"><span class="in-toc"><iframe title="download_all" name="download_all" class="u-hide"></iframe><form method="post" action="/sdfe/pdf/download/S0047248420301135/attachments" target="download_all"><input name="eids" type="hidden" value="1-s2.0-S0047248420301135-mmc1.docx"><input name="eids" type="hidden" value="1-s2.0-S0047248420301135-mmc2.zip"><button class="button-link download-all-button u-font-sans u-margin-xs-bottom button-link-primary button-link-icon-left" type="submit"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-download"><path d="M77.38 56.18l-6.6-7.06L54 66.36V26H44v40.34L26.28 49.1l-7.3 7.08 29.2 29.32 29.2-29.32M88 74v20H10V74H0v30h98V74H88"></path></svg><span class="button-link-text-container"><span class="button-link-text"><span class="download-all-title">Download all </span></span></span></button></form></span></span></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#mmc1" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Multimedia component 1"><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Multimedia component 1</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#mmc2" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Multimedia component 2"><svg focusable="false" viewBox="0 0 95 128" height="20" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Multimedia component 2</span></span></a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right u-padding-s-top" lang="en"><div class="Publication" id="publication"><div class="publication-brand u-display-block-from-sm"><a class="anchor u-display-flex anchor-primary" href="/journal/journal-of-human-evolution" title="Go to Journal of Human Evolution on ScienceDirect"><span class="anchor-text-container"><span class="anchor-text"><img class="publication-brand-image" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/image/elsevier-non-solus.svg" alt="Elsevier"></span></span></a></div><div class="publication-volume u-text-center"><h2 class="publication-title u-h3" id="publication-title"><a class="anchor anchor-secondary publication-title-link" href="/journal/journal-of-human-evolution" title="Go to Journal of Human Evolution on ScienceDirect"><span class="anchor-text-container"><span class="anchor-text">Journal of Human Evolution</span></span></a></h2><div class="text-xs"><a class="anchor anchor-primary" href="/journal/journal-of-human-evolution/vol/146/suppl/C" title="Go to table of contents for this volume/issue"><span class="anchor-text-container"><span class="anchor-text">Volume 146</span></span></a>, <!-- -->September 2020<!-- -->, 102852</div></div><div class="publication-cover u-display-block-from-sm"><a class="anchor u-display-flex anchor-primary" href="/journal/journal-of-human-evolution/vol/146/suppl/C"><span class="anchor-text-container"><span class="anchor-text"><img class="publication-cover-image" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420X0008X-cov150h.gif" alt="Journal of Human Evolution"></span></span></a></div></div><h1 id="screen-reader-main-title" class="Head u-font-serif u-h2 u-margin-s-ver"><span class="title-text">Mitogenomics of macaques (<em>Macaca</em>) across Wallace's Line in the context of modern human dispersals</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div aria-live="polite"></div><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au1"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Ben J.</span> <span class="text surname">Evans</span> </span><span class="author-ref" id="baff1"><sup>a</sup></span><svg focusable="false" viewBox="0 0 106 128" height="20" title="Correspondence author icon" class="icon icon-person react-xocs-author-icon u-fill-grey8"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><svg focusable="false" viewBox="0 0 102 128" height="20" title="Author email or social media contact details icon" class="icon icon-envelope react-xocs-author-icon u-fill-grey8"><path d="M55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0L17.58 34h69.54L55.8 57.19zM0 32.42l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-.98 9.42-2.93L102 34.34V24H0zM92 88.9L73.94 66.16l-8.04 5.95L83.28 94H18.74l18.38-23.12-8.04-5.96L10 88.94V51.36L0 42.9V104h102V44.82l-10 8.46V88.9"></path></svg></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au2"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Marie-Theres</span> <span class="text surname">Gansauge</span> </span><span class="author-ref" id="baff2"><sup>b</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au3"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Matthew W.</span> <span class="text surname">Tocheri</span> </span><span class="author-ref" id="baff3"><sup>c</sup></span> <span class="author-ref" id="baff4"><sup>d</sup></span> <span class="author-ref" id="baff5"><sup>e</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au4"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Michael A.</span> <span class="text surname">Schillaci</span> </span><span class="author-ref" id="baff6"><sup>f</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au5"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Thomas</span> <span class="text surname">Sutikna</span> </span><span class="author-ref" id="baff5"><sup>e</sup></span> <span class="author-ref" id="baff7"><sup>g</sup></span> <span class="author-ref" id="baff8"><sup>h</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au6"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="text surname">Jatmiko</span> </span><span class="author-ref" id="baff5"><sup>e</sup></span> <span class="author-ref" id="baff8"><sup>h</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au7"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">E. Wahyu</span> <span class="text surname">Saptomo</span> </span><span class="author-ref" id="baff5"><sup>e</sup></span> <span class="author-ref" id="baff8"><sup>h</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au8"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Amy</span> <span class="text surname">Klegarth</span> </span><span class="author-ref" id="baff9"><sup>i</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au9"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Anthony J.</span> <span class="text surname">Tosi</span> </span><span class="author-ref" id="baff10"><sup>j</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au10"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Don J.</span> <span class="text surname">Melnick</span> </span><span class="author-ref" id="baff11"><sup>k</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au11"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Matthias</span> <span class="text surname">Meyer</span> </span><span class="author-ref" id="baff2"><sup>b</sup></span></span></span></button></div></div></div><button class="button-link u-margin-s-ver button-link-primary button-link-icon-right" type="button" id="show-more-btn" data-aa-button="icon-expand"><span class="button-link-text-container"><span class="button-link-text">Show more</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="banner-options u-padding-xs-bottom"><div class="toc-button-wrap u-display-inline-block u-display-none-from-lg u-margin-s-right"><button class="button-link button-link-secondary button-link-icon-left button-link-has-colored-icon" type="button"><svg focusable="false" viewBox="0 0 128 128" height="20" class="icon icon-list"><path d="M23 26a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V30zM23 56a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V60zM23 86a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V90z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Outline</span></span></button></div><button class="button-link AddToMendeley button-link-secondary u-margin-s-right u-display-inline-flex-from-md button-link-icon-left button-link-has-colored-icon" type="button"><svg focusable="false" viewBox="0 0 86 128" height="20" class="icon icon-plus"><path d="M48 58V20H38v38H0v10h38v38h10V68h38V58z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Add to Mendeley</span></span></button><div class="Social u-display-inline-block" id="social"><div class="popover social-popover" id="social-popover"><div id="popover-trigger-social-popover"><button class="button-link button-link-secondary u-margin-s-right button-link-icon-left button-link-has-colored-icon" type="button" aria-expanded="false" aria-haspopup="true"><svg focusable="false" viewBox="0 0 114 128" height="20" class="icon icon-share"><path d="M90 112c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zM24 76c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zm66-60c6.62 0 12 5.38 12 12s-5.38 12-12 12-12-5.38-12-12 5.38-12 12-12zm0 62c-6.56 0-12.44 2.9-16.48 7.48L45.1 70.2c.58-1.98.9-4.04.9-6.2s-.32-4.22-.9-6.2l28.42-15.28C77.56 47.1 83.44 50 90 50c12.14 0 22-9.86 22-22S102.14 6 90 6s-22 9.86-22 22c0 1.98.28 3.9.78 5.72L40.14 49.1C36.12 44.76 30.38 42 24 42 11.86 42 2 51.86 2 64s9.86 22 22 22c6.38 0 12.12-2.76 16.14-7.12l28.64 15.38c-.5 1.84-.78 3.76-.78 5.74 0 12.14 9.86 22 22 22s22-9.86 22-22-9.86-22-22-22z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Share</span></span></button></div></div></div><div class="ExportCitation u-display-inline-block" id="export-citation"><div class="popover export-citation-popover" id="export-citation-popover"><div id="popover-trigger-export-citation-popover"><button class="button-link button-link-secondary button-link-icon-left button-link-has-colored-icon" type="button" aria-expanded="false" aria-haspopup="true"><svg focusable="false" viewBox="0 0 104 128" height="20" class="icon icon-cited-by-66"><path d="M2 58.78V106h44V64H12v-5.22C12 40.28 29.08 32 46 32V22C20.1 22 2 37.12 2 58.78zM102 32V22c-25.9 0-44 15.12-44 36.78V106h44V64H68v-5.22C68 40.28 85.08 32 102 32z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Cite</span></span></button></div></div></div></div></div><div class="ArticleIdentifierLinks u-margin-xs-bottom text-xs" id="article-identifier-links"><a class="anchor doi anchor-primary" href="https://doi.org/10.1016/j.jhevol.2020.102852" target="_blank" rel="noreferrer noopener" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier"><span class="anchor-text-container"><span class="anchor-text">https://doi.org/10.1016/j.jhevol.2020.102852</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor rights-and-content anchor-primary" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0047248420301135&amp;orderBeanReset=true" target="_blank" rel="noreferrer noopener"><span class="anchor-text-container"><span class="anchor-text">Get rights and content</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div><div class="LicenseInfo text-xs u-margin-xs-bottom"><div class="License"><span>Under a Creative Commons </span><a class="anchor anchor-primary" href="http://creativecommons.org/licenses/by-nc-nd/4.0/" target="_blank" rel="noreferrer noopener"><span class="anchor-text-container"><span class="anchor-text">license</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div><div class="AccessLabel"><span class="access-indicator"></span>Open access</div></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts u-font-serif" id="abstracts"><div class="abstract author" id="abs0010" lang="en"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Abstract</h2><div id="abssec0010"><div class="u-margin-s-bottom" id="abspara0010"><span>Wallace's Line demarcates a biogeographical boundary between the Indomalaya and Australasian ecoregions. Most placental mammalian genera, for example, occur to the west of this line, whereas most marsupial genera occur to the east. However, macaque monkeys are unusual because they naturally occur on both western and eastern sides. To further explore this anomalous distribution, we analyzed 222 <a href="/topics/agricultural-and-biological-sciences/mitochondrial-genome" title="Learn more about mitochondrial genomes from ScienceDirect's AI-generated Topic Pages" class="topic-link">mitochondrial genomes</a> from âˆ¼20 macaque species, including new genomes from 60 specimens. These comprise a population sampling of most Sulawesi macaques, </span><span><em><a href="/topics/agricultural-and-biological-sciences/macaca-fascicularis" title="Learn more about Macaca fascicularis from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca fascicularis</a></em></span> (long-tailed macaques) specimens that were collected by Alfred R. Wallace and specimens that were recovered during archaeological excavations at Liang Bua, a cave on the Indonesian island of Flores. In <em>M.&nbsp;fascicularis</em><span>, three mitochondrial <a href="/topics/agricultural-and-biological-sciences/lineages" title="Learn more about lineages from ScienceDirect's AI-generated Topic Pages" class="topic-link">lineages</a> span the southernmost portion of Wallace's Line between Bali and Lombok, and divergences within these lineages are contemporaneous with, and possibly mediated by, past dispersals of modern human populations. Near the central portion of Wallace's Line between Borneo and Sulawesi, a more ancient dispersal of macaques from mainland Asia to Sulawesi preceded modern human colonization, which was followed by rapid dispersal of matrilines and was subsequently influenced by recent interspecies hybridization. In contrast to previous studies, we find no strong signal of recombination in most macaque mitochondrial genomes. These findings further characterize macaque evolution before and after modern human dispersal throughout Southeast Asia and point to possible effects on biodiversity of ancient human cultural diasporas.</span></div></div></div></div><ul id="issue-navigation" class="issue-navigation u-margin-s-bottom u-bg-grey1"><li class="previous move-left u-padding-s-ver u-padding-s-left"><a class="button-alternative button-alternative-tertiary u-display-flex button-alternative-icon-left" href="/science/article/pii/S0047248420301123"><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-left"><path d="M1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text-container"><span class="button-alternative-text">Previous <span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></span></a></li><li class="next move-right u-padding-s-ver u-padding-s-right"><a class="button-alternative button-alternative-tertiary u-display-flex button-alternative-icon-right" href="/science/article/pii/S0047248420301160"><span class="button-alternative-text-container"><span class="button-alternative-text">Next <span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></span><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-right"><path d="M1 99l38-38L1 23l7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords u-font-serif"><div class="keywords-section" id="kwrds0010"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Keywords</h2><div class="keyword" id="kwrd0010"><span>Mitochondrial genomes</span></div><div class="keyword" id="kwrd0015"><span>Primate evolution</span></div><div class="keyword" id="kwrd0020"><span>Human cultural diasporas</span></div><div class="keyword" id="kwrd0025"><span>Introduced species</span></div><div class="keyword" id="kwrd0030"><span>Recombination in mitochondrial DNA</span></div></div></div><div class="Body u-font-serif" id="body"><div><section id="sec1"><h2 id="sectitle0020" class="u-h4 u-margin-l-top u-margin-xs-bottom">1. Introduction</h2><div class="u-margin-s-bottom"><div id="p0010"><span>Huxley's modification of Wallace's Line lies between multiple <a href="/topics/agricultural-and-biological-sciences/southeast-asians" title="Learn more about Southeast Asian from ScienceDirect's AI-generated Topic Pages" class="topic-link">Southeast Asian</a><span> islands on the eastern margin of the Sunda <a href="/topics/agricultural-and-biological-sciences/continental-shelf" title="Learn more about continental shelf from ScienceDirect's AI-generated Topic Pages" class="topic-link">continental shelf</a>, including Bali and Lombok in the south, Borneo and Sulawesi in the middle, and Palawan/Busuanga and the oceanic islands of the Philippines in the north. This line demarcates a formidable biogeographical filter that impedes faunal migration between Asia and Australia via the intervening region of Wallacea, which includes Sulawesi, the Lesser Sunda Islands, and the Molucca Islands (</span></span><a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>; <a class="anchor anchor-primary" href="#bib42" name="bbib42" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib42"><span class="anchor-text-container"><span class="anchor-text">Huxley, 1868</span></span></a>; <a class="anchor anchor-primary" href="#bib102" name="bbib102" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib102"><span class="anchor-text-container"><span class="anchor-text">Wallace, 1863</span></span></a>). Huxley's modification of Wallace's Line and Wallace's Line (the northern portion of which runs between Sulawesi and the Philippines) demarcate distinct biodiversity hot spots (<a class="anchor anchor-primary" href="#bib64" name="bbib64" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib64"><span class="anchor-text-container"><span class="anchor-text">Myers et&nbsp;al., 2000</span></span></a>) and ecoregions (<a class="anchor anchor-primary" href="#bib72" name="bbib72" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib72"><span class="anchor-text-container"><span class="anchor-text">Olson et&nbsp;al., 2001</span></span></a>). For example, most terrestrial placental mammalian genera occur naturally only to the west of Wallace's Line (<a class="anchor anchor-primary" href="#bib78" name="bbib78" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib78"><span class="anchor-text-container"><span class="anchor-text">Raven, 1935</span></span></a>; <a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Holt et&nbsp;al., 2013</span></span></a>). Turnover of bird species is also marked across Wallace's Line, although less so for amphibians (<a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Holt et&nbsp;al., 2013</span></span></a><span>). Even in modern humans, <a href="/topics/agricultural-and-biological-sciences/genetic-divergence" title="Learn more about genetic variation from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic variation</a> is strongly structured east and west of this line, with a greater Australo-Melanesian ancestry component in the east, a greater Asian ancestry component in the west, and a gradient of admixture of these components as one moves east or west away from this line (</span><a class="anchor anchor-primary" href="#bib107" name="bbib107" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib107"><span class="anchor-text-container"><span class="anchor-text">Xu et&nbsp;al., 2012</span></span></a>; <a class="anchor anchor-primary" href="#bib40" name="bbib40" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib40"><span class="anchor-text-container"><span class="anchor-text">Hudjashov et&nbsp;al., 2017</span></span></a>).</div><figure class="figure text-xs" id="fig1"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr1.jpg" height="517" alt="Figure&nbsp;1" aria-describedby="cap0010"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr1_lrg.jpg" target="_blank" download="" title="Download high-res image (859KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (859KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr1.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0010"><p id="fspara0010"><span class="label">Figure&nbsp;1</span>. Macaque distributions in Southeast Asia. In bright red are three focal features of this study: Huxley's modification of Wallace's Line, Sulawesi Island, and the Lesser Sunda Islands. Bali and Penida are part of the Lesser Sunda Islands but here and elsewhere these islands are grouped or shaded with the Sunda Region because it lies west of Wallace's Line. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div><div class="u-margin-s-bottom" id="p0015">Given these sharp biogeographical boundaries, it is notable that macaque monkeys (<em>Macaca</em> spp.) managed to successfully disperse across Huxley's modification of Wallace's Line at least three separate times (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>; <a class="anchor anchor-primary" href="#bib25" name="bbib25" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib25"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1969</span></span></a>, <a class="anchor anchor-primary" href="#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text">1975</span></span></a>, <a class="anchor anchor-primary" href="#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text">1995</span></span></a>). For each of these three macaque dispersal events, higher diversity and divergence of genetic variation are observed in populations west of this line (<a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>), indicating that dispersal occurred in an eastward direction. The earliest of these three dispersal events occurred âˆ¼3 million years ago (Ma) or more, when a population (presumably from Borneo) ancestral to extant <span><em><a href="/topics/agricultural-and-biological-sciences/macaca-nemestrina" title="Learn more about Macaca nemestrina from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca nemestrina</a></em></span> (Sunda pigtail macaques) on Borneo dispersed to Sulawesi (<a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2010</span></span></a>, <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">2017</span></span></a>; <a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a>), ultimately giving rise to as many as eight endemic species: <em>Macaca nigra</em>, <em>Macaca nigrescens</em>, <em>Macaca hecki</em>, <em>Macaca tonkeana</em>, <em>Macaca togeanus, Macaca maura, Macaca ochreata</em>, and <em>Macaca brunnescens</em> (<a class="anchor anchor-primary" href="#bib25" name="bbib25" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib25"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1969</span></span></a>; <a class="anchor anchor-primary" href="#bib30" name="bbib30" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib30"><span class="anchor-text-container"><span class="anchor-text">Froehlich and Supriatna, 1996</span></span></a>); <em>M.&nbsp;togeanus</em> is sometimes considered to be a population of <em>M.&nbsp;tonkeana</em>, and <em>M.&nbsp;brunnescens</em> is sometimes considered to be a subspecies of <em>M.&nbsp;ochreata</em> (<a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Groves, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib80" name="bbib80" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib80"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2014</span></span></a>). Another dispersal occurred further north (again presumably from Borneo) to the oceanic islands of the Philippines, possibly during the penultimate glacial maximum âˆ¼190 thousand years ago (ka) and led to a distinct subspecies of <span><em><a href="/topics/agricultural-and-biological-sciences/macaca-fascicularis" title="Learn more about Macaca fascicularis from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca fascicularis</a></em></span>, <em>M.&nbsp;f. philippinensis</em> (<a class="anchor anchor-primary" href="#bib27" name="bbib27" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib27"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1991</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>; but see <a class="anchor anchor-primary" href="#bib86" name="bbib86" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib86"><span class="anchor-text-container"><span class="anchor-text">Smith et&nbsp;al., 2014</span></span></a>). A third dispersal event, the timing of which is less clear, occurred further south with the dispersal of <em>M.&nbsp;fascicularis</em> to the Lesser Sunda Islands (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>; <a class="anchor anchor-primary" href="#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1995</span></span></a>, <a class="anchor anchor-primary" href="#bib29" name="bbib29" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib29"><span class="anchor-text-container"><span class="anchor-text">2006</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>) and established the easternmost extent of the genus <em>Macaca</em> (see <a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">Supplementary Online Material [SOM]</span></span></a> for more background with regard to the diversification of <em>Macaca</em>).</div><div class="u-margin-s-bottom" id="p0020">Macaque dispersal to the Lesser Sunda Islands has been interpreted by some as a relatively recent event (âˆ¼4â€“3 ka), possibly the direct result of translocation by modern humans (<a class="anchor anchor-primary" href="#bib29" name="bbib29" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib29"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2006</span></span></a>; <a class="anchor anchor-primary" href="#bib35" name="bbib35" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib35"><span class="anchor-text-container"><span class="anchor-text">Heinsohn, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib53" name="bbib53" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib53"><span class="anchor-text-container"><span class="anchor-text">Long, 2003</span></span></a>), in part owing to similarities in dorsal pelage color among populations from Java and the Lesser Sunda Islands and a lack of archaeological or fossil evidence of macaques on the Lesser Sunda Islands before the mid-Holocene (<a class="anchor anchor-primary" href="#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1995</span></span></a>). Consistent with this hypothesis, partial mitochondrial sequences from <em>M.&nbsp;fascicularis</em> specimens from Flores and Bali, which lie to the east and west of Wallace's Line, respectively (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>), are very similar to one another (<a class="anchor anchor-primary" href="#bib47" name="bbib47" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib47"><span class="anchor-text-container"><span class="anchor-text">Klegarth et&nbsp;al., 2017</span></span></a><span>). However, the most recent common ancestor of other mitochondrial <a href="/topics/agricultural-and-biological-sciences/lineages" title="Learn more about lineages from ScienceDirect's AI-generated Topic Pages" class="topic-link">lineages</a> from the Lesser Sunda Islands (specifically from Timor) and lineages from the Sunda region is âˆ¼1&nbsp;Ma (</span><a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>), opening the possibility that the Lesser Sunda population of <em>M.&nbsp;fascicularis</em> was initially established long before modern humans reached this region. One possibility is that the deep âˆ¼1&nbsp;Ma mitochondrial divergence time observed in previous studies is due to unsampled (or extinct) lineages whose divergences across Wallace's Line are actually more shallow (<a class="anchor anchor-primary" href="#bib85" name="bbib85" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib85"><span class="anchor-text-container"><span class="anchor-text">Schillaci et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>). Thus, although exactly when and how <em>M.&nbsp;fascicularis</em> dispersed to the Lesser Sunda Islands remains unresolved, these questions could potentially be addressed with increased sampling.</div><section id="sec1.1"><h3 id="sectitle0025" class="u-h4 u-margin-m-top u-margin-xs-bottom">1.1. Molecular ecology of the <em>silenus</em> group of macaques</h3><div class="u-margin-s-bottom" id="p0025"><span>In macaques, interspecies hybridization in combination with incomplete lineage sorting frequently contributes to discordant phylogenetic affinities between mitochondrial and <a href="/topics/agricultural-and-biological-sciences/nuclear-genome" title="Learn more about nuclear genomes from ScienceDirect's AI-generated Topic Pages" class="topic-link">nuclear genomes</a> (</span><a class="anchor anchor-primary" href="#bib22" name="bbib22" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib22"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib58" name="bbib58" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib58"><span class="anchor-text-container"><span class="anchor-text">Melnick and Hoelzer, 1992</span></span></a>; <a class="anchor anchor-primary" href="#bib88" name="bbib88" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib88"><span class="anchor-text-container"><span class="anchor-text">Stevison and Kohn, 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib93" name="bbib93" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib93"><span class="anchor-text-container"><span class="anchor-text">Tosi and Coke, 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib94" name="bbib94" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib94"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2002</span></span></a>, <a class="anchor anchor-primary" href="#bib95" name="bbib95" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib95"><span class="anchor-text-container"><span class="anchor-text">2003</span></span></a>). For example, partial mitochondrial DNA sequences support a closer relationship between <em>M.&nbsp;nemestrina</em> from Borneo and macaque species from Sulawesi than between <em>M.&nbsp;nemestrina</em> from Borneo and Sumatra (<a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2010</span></span></a><span>). However, reduced representation genome sequences of nuclear DNA strongly support <a href="/topics/agricultural-and-biological-sciences/monophyly" title="Learn more about monophyly from ScienceDirect's AI-generated Topic Pages" class="topic-link">monophyly</a> of </span><em>M.&nbsp;nemestrina</em> with respect to Sulawesi macaques (<a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>). These nuclear data also strongly support reciprocal monophyly of Sulawesi macaques from the northern peninsula with respect to those from the rest of Sulawesi (<a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>), but strong support for this relationship has not been recovered from partial mitochondrial DNA sequences (<a class="anchor anchor-primary" href="#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 1999</span></span></a>, <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">2010</span></span></a>; <a class="anchor anchor-primary" href="#bib95" name="bbib95" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib95"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2003</span></span></a>). Thus, it also remains an open question whether full mitochondrial genome sequences from <em>M.&nbsp;nemestrina</em><span> and Sulawesi macaques, including a diversity of <a href="/topics/agricultural-and-biological-sciences/intraspecific-variation" title="Learn more about intraspecific variation from ScienceDirect's AI-generated Topic Pages" class="topic-link">intraspecific variation</a> and increased geographic sampling, would provide results more consistent with those of nuclear DNA.</span></div></section><section id="sec1.2"><h3 id="sectitle0030" class="u-h4 u-margin-m-top u-margin-xs-bottom">1.2. Recombination in mitochondrial DNA</h3><div class="u-margin-s-bottom" id="p0030">Recombination in mitochondrial DNA has been directly illustrated in humans and flies (<a class="anchor anchor-primary" href="#bib49" name="bbib49" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib49"><span class="anchor-text-container"><span class="anchor-text">Kraytsberg et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib54" name="bbib54" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib54"><span class="anchor-text-container"><span class="anchor-text">Ma and O'Farrell, 2015</span></span></a>), and several studies suggest it occurs in other organisms as well, including macaques (<a class="anchor anchor-primary" href="#bib73" name="bbib73" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib73"><span class="anchor-text-container"><span class="anchor-text">Piganeau et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib96" name="bbib96" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib96"><span class="anchor-text-container"><span class="anchor-text">Tsaousis et&nbsp;al., 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib98" name="bbib98" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib98"><span class="anchor-text-container"><span class="anchor-text">Ujvari et&nbsp;al., 2007</span></span></a>). What is not clear, however, is how frequently recombination occurs and the degree to which detection of recombination in public sequence data reflects biological phenomena as opposed to statistical or data-related artifacts (<a class="anchor anchor-primary" href="#bib106" name="bbib106" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib106"><span class="anchor-text-container"><span class="anchor-text">White et&nbsp;al., 2013</span></span></a>). Simulations indicate that analysis of recombined data using phylogenetic methods that assume no recombination has the potential to lead to phylogenetic misinference, exaggeration of terminal branch lengths, overestimation of substitution rate heterogeneity, and inference of non-clock-like evolutionary rates and star-like topologies (<a class="anchor anchor-primary" href="#bib76" name="bbib76" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib76"><span class="anchor-text-container"><span class="anchor-text">Posada and Crandall, 2002</span></span></a>; <a class="anchor anchor-primary" href="#bib83" name="bbib83" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib83"><span class="anchor-text-container"><span class="anchor-text">Schierup and Hein, 2000a</span></span></a>, <a class="anchor anchor-primary" href="#bib84" name="bbib84" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib84"><span class="anchor-text-container"><span class="anchor-text">b</span></span></a>). Given its potential implications for phylogenetic inference, tests for recombination in mitochondrial DNA are a valuable prelude to phylogenetic analysis, with the caveat that the performance of methods to detect recombination varies under different conditions, such as variation in levels of polymorphism and demographic fluctuations (<a class="anchor anchor-primary" href="#bib75" name="bbib75" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib75"><span class="anchor-text-container"><span class="anchor-text">Posada and Crandall, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib106" name="bbib106" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib106"><span class="anchor-text-container"><span class="anchor-text">White et&nbsp;al., 2013</span></span></a>).</div></section><section id="sec1.3"><h3 id="sectitle0035" class="u-h4 u-margin-m-top u-margin-xs-bottom">1.3. Goals of the present study</h3><div class="u-margin-s-bottom" id="p0035"><span>In this study, we explore macaque mitogenomics across Wallace's Line using 162 previously published <a href="/topics/agricultural-and-biological-sciences/mitochondrial-genome" title="Learn more about mitochondrial genomes from ScienceDirect's AI-generated Topic Pages" class="topic-link">mitochondrial genomes</a> and those of 60 additional specimens. These new samples include two specimens collected by Alfred R. Wallace (one each from Lombok and Flores), three recovered during archaeological excavations at Liang Bua (Flores), all Sulawesi macaque species except </span><em>M.&nbsp;ochreata</em>, and novel geographic representation of <em>M.&nbsp;nemestrina</em> and <em>M.&nbsp;fascicularis</em>. Our aim is to resolve two long-standing questions about macaque evolution and dispersal: (1) when did <em>M.&nbsp;fascicularis</em> reach the Lesser Sunda Islands, and was this dispersal natural or mediated by modern human behavior? and (2) what are the phylogenetic relationships among mitochondrial genomes of <em>M.&nbsp;nemestrina</em> and Sulawesi macaques, and are they congruent with inferences from nuclear DNA? To accomplish these goals, we also test an important assumption of the phylogenetic methods that we deploy (<a class="anchor anchor-primary" href="#bib76" name="bbib76" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib76"><span class="anchor-text-container"><span class="anchor-text">Posada and Crandall, 2002</span></span></a>), namely, that recombination does not occur among these mitochondrial genomes.</div></section></section><section id="sec2"><h2 id="sectitle0040" class="u-h4 u-margin-l-top u-margin-xs-bottom">2. Methods</h2><section id="sec2.1"><h3 id="sectitle0045" class="u-h4 u-margin-m-top u-margin-xs-bottom">2.1. Samples</h3><div class="u-margin-s-bottom" id="p0040"><span><span>We obtained 60 new macaque <a href="/topics/agricultural-and-biological-sciences/mitochondrial-genome" title="Learn more about mitochondrial genome from ScienceDirect's AI-generated Topic Pages" class="topic-link">mitochondrial genome</a> sequences from either whole-genome shotgun (WGS) sequencing of </span><a href="/topics/agricultural-and-biological-sciences/dna-extraction" title="Learn more about DNA extractions from ScienceDirect's AI-generated Topic Pages" class="topic-link">DNA extractions</a> from 29 blood samples (including six samples of </span><em>M.&nbsp;nemestrina</em> and 23 from seven species of Sulawesi macaque) or from targeted capture sequencing of DNA extractions from 31 museum or archaeological specimens (including 23 <em>M.&nbsp;fascicularis</em> and eight <em>M.&nbsp;nemestrina</em> specimens). All but one of these samples (a museum specimen labeled as <em>M.&nbsp;nemestrina</em> from â€˜Asiaâ€™) have specific sampling locality information, minimally by island. The WGS samples generally have even more precise locations (usually to the nearest town or city). All of the museum specimens were collected within the past 150 years. The two <em>M.&nbsp;fascicularis</em> specimens from Lombok and Flores that were collected by Alfred R. Wallace were registered in the Natural History Museum, London (then known as the British Museum of Natural History), in 1864 and 1920, respectively, but their actual field collection dates remain unknown to us. Except as noted in the following paragraphs, species identification for the blood and museum specimen samples was unambiguous and based on provenance and external characteristics recorded during sampling. Additional information on samples analyzed in this study is presented in <a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S1</span></span></a>.</div><div class="u-margin-s-bottom" id="p0045">Four bone and tooth fragments recovered during archaeological excavations of Liang Bua, a cave site on Flores, and identified based on morphology as <em>Macaca</em><span> sp. were selected for the screening of ancient DNA preservation and potential species identification. Portions of these same fragments were also sent for direct accelerator mass spectrometry (AMS) <a href="/topics/agricultural-and-biological-sciences/radiocarbon-dating" title="Learn more about radiocarbon dating from ScienceDirect's AI-generated Topic Pages" class="topic-link">radiocarbon dating</a><span> at the Research Laboratory for Archaeology and the History of Art at the University of Oxford, using chemical <a href="/topics/agricultural-and-biological-sciences/pretreatment" title="Learn more about pretreatment from ScienceDirect's AI-generated Topic Pages" class="topic-link">pretreatment</a>, target preparation, and measurement protocols described elsewhere (</span></span><a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Brock et&nbsp;al., 2010</span></span></a>; <a class="anchor anchor-primary" href="#bib7" name="bbib7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib7"><span class="anchor-text-container"><span class="anchor-text">Bronk Ramsey et&nbsp;al., 2002</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2004</span></span></a>). DNA analysis of the archaeological and museum specimens was carried out at the Max Planck Institute for Evolutionary Anthropology (Leipzig, Germany).</div><div class="u-margin-s-bottom" id="p0050"><span>Detailed information on DNA extraction, targeted capture methodology, sequencing, and <a href="/topics/agricultural-and-biological-sciences/genome-assembly" title="Learn more about genome assembly from ScienceDirect's AI-generated Topic Pages" class="topic-link">genome assembly</a> of WGS and targeted capture data is provided in </span><a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM (Fig.&nbsp;S1 and Table&nbsp;S1)</span></span></a>. We combined our sample of 60 new mitochondrial genomes with 162 genomes published previously. One genome sequence from <span><em><a href="/topics/agricultural-and-biological-sciences/macaca-mulatta" title="Learn more about Macaca mulatta from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca mulatta</a></em></span> that was probably incorrectly annotated as <em>M.&nbsp;nigra</em> (GenBank accession <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) and one from <em>M.&nbsp;fascicularis</em> (<a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FJ906803.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">FJ906803.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) were initially considered, but were subsequently excluded based on tests for recombination discussed further in section <a class="anchor anchor-primary" href="#sec2.3" name="bsec2.3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec2.3"><span class="anchor-text-container"><span class="anchor-text">2.3</span></span></a>. Provenance information and GenBank accession numbers for all data analyzed in this study are provided in <a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;1</span></span></a>.</div></section><section id="sec2.2"><h3 id="sectitle0050" class="u-h4 u-margin-m-top u-margin-xs-bottom">2.2. Phylogenetic analysis</h3><div class="u-margin-s-bottom" id="p0055">We aligned the combined sample of 222 mitochondrial genomes from GenBank with a program for multiple alignment using fast Fourier transform (MAFFT) version 7.205 (<a class="anchor anchor-primary" href="#bib46" name="bbib46" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib46"><span class="anchor-text-container"><span class="anchor-text">Katoh and Standley, 2013</span></span></a>) and then modified this alignment manually. Mitochondrial genomes from a gelada (accession <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FJ785426.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">FJ785426.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) and two hamadryas baboons (accessions <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=NC001992.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">NC001992.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=JX946201.2" target="_blank"><span class="anchor-text-container"><span class="anchor-text">JX946201.2</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) were used as out-groups. We included in our analysis regions with some missing data, but excluded from all samples two regions of ambiguous alignment â€” a portion of the nongenic region that corresponds to positions 7697â€“7761 in <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KJ567054.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KJ567054.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, and a portion of the D-loop, that corresponds to positions 15686â€“15741 in <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KJ567054.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KJ567054.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a> â€” leaving a total of 16,575 base pairs (including some positions with gaps) in the alignment of 222 sequences. Four sets of sequences in our analysis were identical: <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KM851028.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KM851028.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>&nbsp;+&nbsp;<a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KM851029.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KM851029.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>; <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KM851027.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KM851027.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>&nbsp;+&nbsp;<a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KM851019.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KM851019.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>; the new <em>M.&nbsp;nemestrina</em> mitochondrial genomes from samples PM665 and PM1206; and the new <em>M.&nbsp;maura</em> mitochondrial genomes from samples PF615, PM614, and PM616. All uncertain positions were converted to missing data before phylogenetic analysis. A maximum likelihood phylogeny was estimated using a model of evolution (TIM3+I+G4) favored by the Bayesian Information Criterion (BIC), as implemented by IQTREE version 1.4.0 (<a class="anchor anchor-primary" href="#bib65" name="bbib65" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib65"><span class="anchor-text-container"><span class="anchor-text">Nguyen et&nbsp;al., 2015</span></span></a>). The TIM3 model uses empirical base frequencies; the reversible transition rate parameter between A and C is equal to that between C and G, and the same parameter between A and T is equal to that between G and T. This model also has a parameter that represents the proportion of invariant sites (I) and model heterogeneity in the evolutionary rate of different sites with a gamma distribution approximated by four rate categories (G4). We evaluated support for each node using 1,000 replicates of ultrafast bootstrap as implemented by IQTREE.</div><div class="u-margin-s-bottom" id="p0060">We used BEAST version 2 (<a class="anchor anchor-primary" href="#bib16" name="bbib16" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib16"><span class="anchor-text-container"><span class="anchor-text">Drummond and Rambaut, 2007</span></span></a><span>) to estimate a time-calibrated phylogeny, using a random starting tree and assuming a relaxed log-normal clock with an estimated rate and a birth-death model for the tree prior. Six independent <a href="/topics/agricultural-and-biological-sciences/markov-chain" title="Learn more about Markov chain from ScienceDirect's AI-generated Topic Pages" class="topic-link">Markov chain</a> Monte Carlo runs were performed, each for âˆ¼15 million generations and sampling every 10,000 generations. We used previously published estimated relationships among fossil and extant species and estimated dates for fossils to establish priors for the ages of two nodes in our phylogeny. We set the upper (youngest) boundary for the age of extant macaques at 5.3&nbsp;Ma (</span><a class="anchor anchor-primary" href="#bib1" name="bbib1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib1"><span class="anchor-text-container"><span class="anchor-text">Alba et&nbsp;al., 2014</span></span></a>) based on the upper age range of the oldest fossil European macaques, which are considered to be closely related to <span><em><a href="/topics/agricultural-and-biological-sciences/macaca-sylvanus" title="Learn more about Macaca sylvanus from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca sylvanus</a></em></span> (<a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Delson, 1980</span></span></a>; <a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a>). We set the lower (oldest) boundary for the age of extant macaques at 6.2&nbsp;Ma based on the earliest age for the first appearance of an unambiguous macaque, <em>Macaca libyca</em>, from Wadi Natrun, Egypt (<a class="anchor anchor-primary" href="#bib104" name="bbib104" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib104"><span class="anchor-text-container"><span class="anchor-text">Werdelin, 2010</span></span></a>). <em>Macaca libyca</em> may have diverged before the diversification of African and Asian macaques rather than being ancestral to this clade (<a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Delson, 1980</span></span></a><span>); however, this lower boundary for extant macaque diversification is also consistent with the proposal that African and Asian macaque <a href="/topics/agricultural-and-biological-sciences/lineages" title="Learn more about lineages from ScienceDirect's AI-generated Topic Pages" class="topic-link">lineages</a> diverged in association with the Messinian Salinity Crisis between 5.9&nbsp;Ma and 5.3&nbsp;Ma (</span><a class="anchor anchor-primary" href="#bib1" name="bbib1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib1"><span class="anchor-text-container"><span class="anchor-text">Alba et&nbsp;al., 2014</span></span></a>), while also allowing for the possibility (âˆ¼0.3 million years) of a pre-Messinian divergence (<a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a>). The timing of divergence between geladas and baboons was recently estimated to be between 4.5 and 4.2&nbsp;Ma (<a class="anchor anchor-primary" href="#bib77" name="bbib77" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib77"><span class="anchor-text-container"><span class="anchor-text">Pugh and Gilbert, 2018</span></span></a>); however, we conservatively used a slightly lower limit of 5.5&nbsp;Ma (Pugh personal communication), which is closer to but not as old as other estimates (<a class="anchor anchor-primary" href="#bib51" name="bbib51" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib51"><span class="anchor-text-container"><span class="anchor-text">Leakey, 1993</span></span></a>; <a class="anchor anchor-primary" href="#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text">Delson, 2000</span></span></a>; <a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a>). A uniform prior was set for each of these calibrations.</div><div class="u-margin-s-bottom" id="p0065">For the BEAST analyses, we used a more parameterized model (GTR+I+G4, with empirical base frequencies) than the one that was selected by the BIC as implemented by IQTREE, as detailed previously. The GTR model has an independently estimated reversible transition rate between each of the four nucleotides. Preliminary analyses recovered no well-supported differences when a model with fewer parameters than the BIC-selected model was used (TN93). Fifty percent of the Markov chain Monte Carlo run was discarded as burn-in based on inspection of the posterior likelihood trace using Tracer version 1.71 (<a class="anchor anchor-primary" href="#bib16" name="bbib16" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib16"><span class="anchor-text-container"><span class="anchor-text">Drummond and Rambaut, 2007</span></span></a>), enabling quantification of the effective sample sizes of model parameter estimates. The effective sample size values for these analyses exceeded 200. Well-supported relationships (&gt; 95% posterior probabilities) within the BEAST analysis were identical to the maximum likelihood analysis.</div></section><section id="sec2.3"><h3 id="sectitle0055" class="u-h4 u-margin-m-top u-margin-xs-bottom">2.3. Recombination</h3><div class="u-margin-s-bottom" id="p0070">Previously, a signature of recombination was detected in partial mitochondrial sequences from <em>M.&nbsp;nemestrina</em> from Borneo and Sulawesi macaques (<a class="anchor anchor-primary" href="#bib73" name="bbib73" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib73"><span class="anchor-text-container"><span class="anchor-text">Piganeau et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib96" name="bbib96" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib96"><span class="anchor-text-container"><span class="anchor-text">Tsaousis et&nbsp;al., 2005</span></span></a>). To evaluate whether this signature was present in the mitochondrial genomes studied here, we analyzed these data using nine methods (RDP, GENECONV, Bootscan, Maxchi, Chimaera, SiSscan, PhylPro, LARD, 3Seq) as implemented by the Recombination Detection Program (<a class="anchor anchor-primary" href="#bib55" name="bbib55" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib55"><span class="anchor-text-container"><span class="anchor-text">Martin and Rybicki, 2000</span></span></a>), version 4.97.</div></section></section><section id="sec3"><h2 id="sectitle0060" class="u-h4 u-margin-l-top u-margin-xs-bottom">3. Results</h2><div class="u-margin-s-bottom" id="p0075">As the bulk of the sequences analyzed here have been published elsewhere (<a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a>), we present our results in terms of the new insights gained from the additional 60 new genomes we collected. Specifically, these new data provide novel perspectives into (1) the timing and nature of <em>M.&nbsp;fascicularis</em> dispersal across Wallace's Line into the Lesser Sunda Islands and (2) phylogenetic resolution among and within <em>M.&nbsp;nemestrina</em> and Sulawesi macaques using mitochondrial genomes.</div><section id="sec3.1"><h3 id="sectitle0065" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.1. Macaca fascicularis</h3><section id="sec3.1.1"><h4 id="sectitle0070" class="u-margin-m-top u-margin-xs-bottom">Ancient DNA and radiocarbon ages of archaeological macaque samples from Liang Bua</h4><div class="u-margin-s-bottom"><div id="p0080"><span>All four archaeological samples from Liang Bua tested positive for ancient DNA belonging to <a href="/topics/agricultural-and-biological-sciences/cercopithecidae" title="Learn more about Cercopithecidae from ScienceDirect's AI-generated Topic Pages" class="topic-link">Cercopithecidae</a> and yielded direct AMS radiocarbon ages (</span><a class="anchor anchor-primary" href="#tbl1" name="btbl1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a>); however, only three of these samples had DNA preservation suitable for subsequent sequencing of mitochondrial genomes (see <a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM and Table&nbsp;S2</span></span></a>). The resulting three genome sequences belong to <em>M.&nbsp;fascicularis</em>. The AMS radiocarbon ages were calibrated using the CALIB 7.1 program (<a class="anchor anchor-primary" href="http://calib.org" target="_blank"><span class="anchor-text-container"><span class="anchor-text">http://calib.org</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) with the offset for the Southern Hemisphere (SHCal13; <a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Holt et&nbsp;al., 2013</span></span></a>) and converted into a median probability age estimate with conventional calendar-year age ranges using 95% confidence intervals (CIs) (<a class="anchor anchor-primary" href="#tbl1" name="btbl1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a>; <a class="anchor anchor-primary" href="#bib89" name="bbib89" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib89"><span class="anchor-text-container"><span class="anchor-text">Stuiver et&nbsp;al., 2020</span></span></a>). Hereafter, we refer only to the calibrated <sup>14</sup>C ages and their 95% CI ranges. Because the <sup>14</sup>C ages for these archaeological <em>M.&nbsp;fascicularis</em> samples were all within the past 2,500 years, these ages were not used as tip dates in the BEAST analysis.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl1"><span class="captions text-s"><span id="cap0035"><p id="tspara0010"><span class="label">Table&nbsp;1</span>. <sup>14</sup>C ages for bone and tooth fragments of long-tailed macaques recovered from Liang Bua (Flores, Indonesia).</p></span></span><div class="groups"><table><thead><tr class="valign-top"><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><th scope="col" class="rowsep-1" colspan="3">Age</th></tr><tr class="rowsep-1 valign-top"><th scope="col">Sample code</th><th scope="col">Sequence code</th><th scope="col">Excavated sector</th><th scope="col">Depth (cm)</th><th scope="col">Stratigraphic unit</th><th scope="col">Methods<a class="anchor anchor-primary" href="#tbl1fna" name="btbl1fna" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1fna"><span class="anchor-text-container"><span class="anchor-text"><sup>a</sup></span></span></a></th><th scope="col"><sup>14</sup>C (yr BP)</th><th scope="col" class="rowsep-1">Calibrated median probability (yr cal. BP)</th><th scope="col" class="rowsep-1">Calibrated range, 95% CI<a class="anchor anchor-primary" href="#tbl1fnb" name="btbl1fnb" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1fnb"><span class="anchor-text-container"><span class="anchor-text"><sup>b</sup></span></span></a> (ka)</th></tr></thead><tbody><tr class="valign-top"><td>OxA-37730</td><td>D1638</td><td>XXVI</td><td class="align-char">âˆ¼40</td><td class="align-char">8C</td><td>AMS</td><td class="align-char">190&nbsp;Â±&nbsp;23</td><td class="align-char">188</td><td class="align-char">0.23â€“0.14</td></tr><tr class="valign-top"><td>OxA-37731</td><td>D1647</td><td>XXVI</td><td class="align-char">âˆ¼70</td><td class="align-char">8C</td><td>AMS</td><td class="align-char">1,605&nbsp;Â±&nbsp;24</td><td class="align-char">1,457</td><td class="align-char">1.53â€“1.40</td></tr><tr class="valign-top"><td>OxA-37732</td><td>D1541</td><td>XXVI</td><td class="align-char">âˆ¼100</td><td class="align-char">8C</td><td>AMS</td><td class="align-char">1,619&nbsp;Â±&nbsp;25</td><td class="align-char">1,470</td><td class="align-char">1.54â€“1.41</td></tr><tr class="valign-top"><td>OxA-39515</td><td>SP3918<a class="anchor anchor-primary" href="#tbl1fnc" name="btbl1fnc" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1fnc"><span class="anchor-text-container"><span class="anchor-text"><sup>c</sup></span></span></a></td><td>XXVI</td><td class="align-char">âˆ¼130</td><td class="align-char">8C</td><td>AMS</td><td class="align-char">2,482&nbsp;Â±&nbsp;19</td><td class="align-char">2,483</td><td class="align-char">2.54â€“2.36</td></tr></tbody></table></div><dl class="footnotes"><dt id="tbl1fna">a</dt><dd><div class="u-margin-s-bottom" id="ntpara0010"><span>AMS, accelerator mass spectrometry; for details of the chemical <a href="/topics/agricultural-and-biological-sciences/pretreatment" title="Learn more about pretreatment from ScienceDirect's AI-generated Topic Pages" class="topic-link">pretreatment</a>, target preparation, and AMS measurement, see </span><a class="anchor anchor-primary" href="#bib7" name="bbib7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib7"><span class="anchor-text-container"><span class="anchor-text">Bronck Ramsey et&nbsp;al. (2002</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2004)</span></span></a>.</div></dd><dt id="tbl1fnb">b</dt><dd><div class="u-margin-s-bottom" id="ntpara0015">CI, confidence interval.</div></dd><dt id="tbl1fnc">c</dt><dd><div class="u-margin-s-bottom" id="ntpara0020">Complete genome sequencing not attempted for this sample.</div></dd></dl></div></div></section><section id="sec3.1.2"><h4 id="sectitle0075" class="u-margin-m-top u-margin-xs-bottom">Intraspecific mitogenomics</h4><div class="u-margin-s-bottom"><div id="p0085"><span>Several recent studies have used molecular markers, including mitochondrial genomes, to explore the mitochondrial <a href="/topics/agricultural-and-biological-sciences/phylogeography" title="Learn more about phylogeography from ScienceDirect's AI-generated Topic Pages" class="topic-link">phylogeography</a> of macaques, especially </span><em>M.&nbsp;fascicularis</em> in Southeast Asia (<a class="anchor anchor-primary" href="#bib4" name="bbib4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib4"><span class="anchor-text-container"><span class="anchor-text">Blancher et&nbsp;al., 2008</span></span></a>; <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib59" name="bbib59" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib59"><span class="anchor-text-container"><span class="anchor-text">Mohd Salleh et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib85" name="bbib85" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib85"><span class="anchor-text-container"><span class="anchor-text">Schillaci et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib86" name="bbib86" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib86"><span class="anchor-text-container"><span class="anchor-text">Smith et&nbsp;al., 2014</span></span></a>; <a class="anchor anchor-primary" href="#bib93" name="bbib93" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib93"><span class="anchor-text-container"><span class="anchor-text">Tosi and Coke, 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib94" name="bbib94" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib94"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2002</span></span></a>, <a class="anchor anchor-primary" href="#bib95" name="bbib95" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib95"><span class="anchor-text-container"><span class="anchor-text">2003</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>). Similar to this previous research, we find that the mitochondrial DNA of <em>M.&nbsp;fascicularis</em> consists of two major clades, clade A and clade B (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a>; <a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>). Our results suggest that these two clades diverged from one another âˆ¼1.67&nbsp;Ma (this and all subsequent dates are median estimates from the posterior distribution), with a 95% CI of 1.95â€“1.41&nbsp;Ma (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S3</span></span></a>), similar to previous results (<a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a>). Clade A is distributed across mainland Asia, northern Sumatra, and the surrounding islands, except for one mitochondrial genome that putatively originated in Mindanao, Philippines, which may be a recent introduction or mislabeled sample (<a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>). Clade B, which includes the 23 new <em>M.&nbsp;fascicularis</em> genomes from this study, is distributed across southern Sumatra, Borneo, Java, the Philippines, and the Lesser Sunda Islands. Clade B diversified 0.90&nbsp;Ma (95% CI&nbsp;= 1.05â€“0.77&nbsp;Ma), again similar to previously published estimates (<a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>).</div><figure class="figure text-xs" id="fig2"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr2.jpg" height="1019" alt="Figure&nbsp;2" aria-describedby="cap0015"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr2_lrg.jpg" target="_blank" download="" title="Download high-res image (1MB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (1MB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr2.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0015"><p id="fspara0015"><span class="label">Figure&nbsp;2</span>. Inferred evolutionary relationships among mitochondrial genomes offer insights into macaque evolution in Southeast Asia. Consensus tree from Bayesian (BEAST) analysis of mitochondrial genome sequences; 95% confidence intervals for key divergence times are indicated with red bars above each node and also provided in <a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S3</span></span></a>. Posterior probabilities of each node are depicted based on the legend. Scale bar indicates Ma, and branches and labels are colored according to the geographic localities in the inset. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure><figure class="figure text-xs" id="fig3"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr3.jpg" height="994" alt="Figure&nbsp;3" aria-describedby="cap0020"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr3_lrg.jpg" target="_blank" download="" title="Download high-res image (861KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (861KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr3.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0020"><p id="fspara0020"><span class="label">Figure&nbsp;3</span>. Detailed relationships among clade B of <em>M.&nbsp;fascicularis</em> demonstrate that multiple lineages occur on both sides of the southern portion of Wallace's Line. Samples collected by Wallace (W) and from Liang Bua are labeled. Except for very recent relationships, posterior probabilities are depicted following <a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;2</span></span></a>. Asterisks denote new mitochondrial genomes reported here; 95% confidence intervals for key divergence times are indicated with blue bars above each node. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div><div class="u-margin-s-bottom" id="p0090">Within clade B, there is strong support for five distinct subclades (labeled B1 thru B5; <a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a>) and weak support for a clade containing mitochondrial genomes from the islands of Nias and Simeulue (west of north Sumatra), but the relationships among these clades are not well resolved. Clade B5 comprises specimens from southern Sumatra, Borneo, and the Philippines, whereas clade B4 comprises specimens from Java and Mauritius. The other three subclades (B1â€“B3) included specimens that span the southern portion of Wallace's Line from Java, Bawean, Bali, and Penida Islands in the west and the Lesser Sunda Islands of Lombok, Sumbawa, Flores, and Timor in the east. Although previously known only in specimens from Timor (eight sequences; <a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib85" name="bbib85" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib85"><span class="anchor-text-container"><span class="anchor-text">Schillaci et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>), our results show that clade B1 is also found on Flores (eight sequences, including one specimen collected by Alfred R. Wallace and all three archaeological specimens from Liang Bua), Sumbawa (three sequences), and Java (four sequences; <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>). Similarly, mitochondrial genome sequences within subclades B2 and B3 were formerly known only from west of Wallace's Line, on Bali and the nearby Sundaic island of Penida (B2), and on Java, Bali, and the nearby Sundaic island of Bawean (B3). However, we found that mitochondrial DNA of a <em>M.&nbsp;fascicularis</em> specimen that was sampled by Alfred R. Wallace on Lombok, which is immediately east of Wallace's Line, is from clade B2, and mitochondrial DNA from another museum specimen from Lombok is from clade B3.</div><div class="u-margin-s-bottom" id="p0095">Our results suggest that extant sequence divergence within clade B1 began 80 ka (95% CI&nbsp;= 110â€“60 ka), within clade B2 began 140 ka (95% CI&nbsp;= 180â€“100 ka), and within clade B3 began 210 ka (95% CI&nbsp;= 270â€“160 ka). Within clade B1, there are two subclades (B1-a, B1-b; <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>). Clade B1-b began diversifying 70 ka (90â€“50 ka) and is restricted to the Lesser Sunda Islands based on the currently available data. Clade B1-b comprises three clades that are respectively exclusive to the islands of Flores, Flores&nbsp;+&nbsp;Sumbawa, or Timor, although support for the Flores&nbsp;+&nbsp;Sumbawa clade differed between the Bayesian and maximum likelihood analyses (46% and 100% posterior probability and bootstrap support, respectively). Clade B1-a includes four sequences from Java and two from Flores. Within clade B1, subclade B1-a1 (<a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>) contains genomes from Java and Flores.</div><div class="u-margin-s-bottom" id="p0100">The divergence times between closely related genomes from either side of Wallace's Line provide estimates for the timing of possible eastward dispersal events of <em>M.&nbsp;fascicularis</em>: (i) the most recent common ancestor of subclade B1-a1 is estimated to have lived 30 ka ago (95% CI&nbsp;= 40â€“20 ka); (ii) within clade B2, subclade B2-a includes two sequences from Penida, which is west of Wallace's Line, and from Lombok, which is east; the most recent common ancestor of this clade is 60 ka (90â€“40 ka); and (iii) within clade B3, subclade B3-a includes four sequences from Bali and one from Lombok; the most recent common ancestor of subclade B3-a is 10 ka (20â€“0 ka). Dispersal scenarios associated with these divergence times and also the divergence between subclades B1-a, and B1-b (<a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>) are presented in the <a class="anchor anchor-primary" href="#sec4" name="bsec4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec4"><span class="anchor-text-container"><span class="anchor-text">Discussion</span></span></a> section.</div></section></section><section id="sec3.2"><h3 id="sectitle0080" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.2. <em>Macaca leonina</em> and <em>Macaca nemestrina</em></h3><div class="u-margin-s-bottom"><div id="p0105"><span>Consistent with several previous studies based on partial mitochondrial DNA sequences, our results support mitochondrial DNA <a href="/topics/agricultural-and-biological-sciences/paraphyly" title="Learn more about paraphyly from ScienceDirect's AI-generated Topic Pages" class="topic-link">paraphyly</a> of </span><em>M.&nbsp;nemestrina</em> (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>; <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2010</span></span></a>). Mitochondrial genomes of macaques from mainland Asia that belong to the <span><em><a href="/topics/agricultural-and-biological-sciences/silenus" title="Learn more about silenus from ScienceDirect's AI-generated Topic Pages" class="topic-link">silenus</a></em></span> group (<span><em><a href="/topics/agricultural-and-biological-sciences/macaca-silenus" title="Learn more about Macaca silenus from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca silenus</a></em></span>, which is from the Western Ghats of India, and <em>Macaca leonina</em>, which is from portions of mainland Southeast Asia including northeastern India, Bangladesh, Thailand, and Vietnam) form a clade that is sister to a <em>M.&nbsp;nemestrina</em> clade made up of specimens from Sumatra and Peninsular Malaysia as well as a museum specimen of unknown origin (SMF16836, from â€˜Asiaâ€™). Other <em>M.&nbsp;nemestrina</em> mitochondrial haplotypes from Borneo (<em>n</em>&nbsp;=&nbsp;12) form a clade that is sister to a clade of Sulawesi macaques.</div><figure class="figure text-xs" id="fig4"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr4.jpg" height="716" alt="Figure&nbsp;4" aria-describedby="cap0025"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr4_lrg.jpg" target="_blank" download="" title="Download high-res image (850KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (850KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr4.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0025"><p id="fspara0025"><span class="label">Figure&nbsp;4</span>. Inferred evolutionary relationships among mitochondrial genomes of <em>silenus</em> group samples support paraphyly of <em>M.&nbsp;nemestina</em> but fail to resolve early relationships among the Sulawesi macaques. Labeling follows <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>, except that 95% confidence intervals for key divergence times are indicated with red bars above each node. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div><div class="u-margin-s-bottom" id="p0110">At least four major mitochondrial lineages are present in <em>M.&nbsp;nemestrina</em> from Borneo. The provenance of captive specimens in three of these clades suggests these lineages are represented by <em>M.&nbsp;nemestrina</em> from northeast, southeast, and east Borneo, respectively (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>). A fourth clade may be present in west Borneo, based on sequence similarity to a partial sequence in GenBank (<a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=AY206515.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">AY206515.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) of another captive specimen from Sanggau, Kalimantan Barat, in Indonesia. Within <em>silenus</em> group macaques, our results suggest sequence divergence between Sulawesi and Borneo macaques and those from Asia and Sumatra occurred 3.93&nbsp;Ma (95% CI&nbsp;= 4.43â€“3.46&nbsp;Ma). Our results also suggest the mitochondrial genomes of <em>M.&nbsp;nemestrina</em> from Borneo diverged from those of Sulawesi macaques 2.45&nbsp;Ma (95% CI&nbsp;= 2.79â€“2.13&nbsp;Ma), which provides an estimate for when macaques (presumably from Borneo) dispersed east across Wallace's Line to Sulawesi.</div></section><section id="sec3.3"><h3 id="sectitle0085" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.3. Sulawesi macaques</h3><div class="u-margin-s-bottom" id="p0115">Within Sulawesi macaques, mitochondrial DNA lineages are monophyletic and cluster by species, with one exception wherein a mitochondrial genome from a <em>M.&nbsp;tonkeana</em> specimen (PF511) clusters with mitochondrial genomes from <em>M.&nbsp;hecki</em> (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a><span>). This female macaque was sampled from near the <a href="/topics/agricultural-and-biological-sciences/hybrid-zone" title="Learn more about hybrid zone from ScienceDirect's AI-generated Topic Pages" class="topic-link">hybrid zone</a> between </span><em>M.&nbsp;tonkeana</em> and <em>M.&nbsp;hecki</em> (<a class="anchor anchor-primary" href="#bib11" name="bbib11" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib11"><span class="anchor-text-container"><span class="anchor-text">Bynum et&nbsp;al., 1997</span></span></a>; <a class="anchor anchor-primary" href="#bib20" name="bbib20" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib20"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2003a</span></span></a>). Based on photographs that were taken at the time of sampling, this specimen displayed narrow, kidney-shaped ischial callosities and relatively modestly sized brow ridges, which are morphological features of <em>M.&nbsp;tonkeana</em> and not <em>M.&nbsp;hecki</em>, in which these features are more oval and more prominent, respectively (<a class="anchor anchor-primary" href="#bib25" name="bbib25" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib25"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1969</span></span></a><span>). Preliminary analyses of WGS data from the <a href="/topics/agricultural-and-biological-sciences/nuclear-genome" title="Learn more about nuclear genome from ScienceDirect's AI-generated Topic Pages" class="topic-link">nuclear genome</a> of this specimen are consistent with this species' identification; at this time, however, we do not attempt to quantify the extent of </span><em>M.&nbsp;hecki</em> ancestry in the nuclear genome of this specimen. The uncorrected pairwise distance between the mitochondrial genome of sample PF511 and the most closely related <em>M.&nbsp;hecki</em> mitochondrial genome (PF643) is 0.0024.</div><div class="u-margin-s-bottom" id="p0120">Relationships among mitochondrial lineages of each Sulawesi species generally have short internal branches, and some are poorly supported (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>). However, support in the BEAST analysis for some early-branching relationships among Sulawesi macaques is surprising from a phylogeographical perspective. For example, a clade containing the mitochondrial genomes of <em>M.&nbsp;nigra</em> and <em>M.&nbsp;tonkeana</em>&nbsp;+&nbsp;<em>M.&nbsp;togeanus</em> is well supported (99% posterior probability) although <em>M.&nbsp;nigra</em> and <em>M.&nbsp;nigrescens</em> have a strongly supported sister relationship based on nuclear DNA and are distributed in parapatry on the distal end of the northern peninsula (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;1</span></span></a>, <a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>; <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>). <em>Macaca tonkeana</em> occurs in central Sulawesi, with the range of <em>M.&nbsp;hecki</em> between it and <em>M.&nbsp;nigra</em>&nbsp;+&nbsp;<em>M.&nbsp;nigrescens</em> (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>; <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>). Likewise, a clade containing the mitochondrial genomes of <em>M.&nbsp;nigrescens</em>, <em>M.&nbsp;maura</em>, <em>M.&nbsp;brunnescens</em>, and <em>M.&nbsp;hecki</em> has modest support (88% posterior probability) but is incongruent with supported relationships among nuclear DNA sequences (<a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a>). These nodes were present with modest support in the maximum likelihood analysis: the <em>M.&nbsp;nigra</em>, <em>M.&nbsp;tonkeana, Macaca togenaus</em> clade has a bootstrap support value of 77%, and the <em>M.&nbsp;nigrescens</em>, <em>M.&nbsp;maura</em>, <em>M.&nbsp;brunnescens</em>, <em>M.&nbsp;hecki</em> clade has a bootstrap support value of 61%. To explore the effect of analyzing only closely related mitochondrial genomes, we estimated a phylogeny from only the <em>silenus</em> group and Sulawesi mitochondrial genomes using BEAST. All nodes with &gt;85% posterior probability in the <em>silenus</em> group in the main analysis (depicted in <a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>) were also present in the consensus topology of this reduced analysis (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Fig.&nbsp;S2</span></span></a>). With the exception of the <em>M.&nbsp;nigra</em>, <em>M.&nbsp;tonkeana, M. togenaus</em> clade, which had a 96% posterior probability, other early-branching relationships among Sulawesi lineages were not strongly supported in the <em>silenus</em> group and Sulawesi-only analysis, which is similar to results of the main analysis.</div><div class="u-margin-s-bottom" id="p0125">Consistent with inferences based on partial mitochondrial sequences (<a class="anchor anchor-primary" href="#bib20" name="bbib20" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib20"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2003a</span></span></a>), substantial intraspecific divergence is observed within <em>M.&nbsp;hecki</em><span> between the population in the southern portion of its distribution and that in the northeastern region. By contrast, very little <a href="/topics/agricultural-and-biological-sciences/intraspecific-variation" title="Learn more about intraspecific variation from ScienceDirect's AI-generated Topic Pages" class="topic-link">intraspecific variation</a> is observed between the two </span><em>M.&nbsp;nigrescens</em> samples.</div></section><section id="sec3.4"><h3 id="sectitle0090" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.4. Recombination tests</h3><div class="u-margin-s-bottom" id="p0130">A total of 29 putatively recombined regions in 40 genomes were detected in the data by at least one of the nine recombination methods implemented by RDP (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S4</span></span></a>). Twenty-five of these 29 regions were flagged as possibly being â€˜caused by an evolutionary process other than recombination,â€™ including four of the new mitochondrial genomes reported here: <em>M.&nbsp;hecki</em> sample PF644 (one 335 base pair region detected by five tests), <em>M.&nbsp;nigrescens</em> sample PM1011 (two regions that were 191 and 343 base pairs in size and detected by two tests and one test, respectively), and two <em>M.&nbsp;fascicularis</em> samples SP3942 (two regions that were 496 and 1099 base pairs in size and detected by three tests and one test, respectively) and SP3934 (one region that was 1099 base pairs in size and detected by one test). Recombination tests generally have a high rate of false positives and low power (<a class="anchor anchor-primary" href="#bib75" name="bbib75" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib75"><span class="anchor-text-container"><span class="anchor-text">Posada and Crandall, 2001</span></span></a><span>), and none of these four putatively recombined regions were particularly convincing based on visual inspection. For example, the putatively recombined region of PF644 included uncalled bases at 13 positions and unique <a href="/topics/agricultural-and-biological-sciences/single-nucleotide-polymorphism" title="Learn more about SNPs from ScienceDirect's AI-generated Topic Pages" class="topic-link">SNPs</a> in this genome at 14 positions that were not present in any putative parental genomes; this pattern is suggestive of false positives owing to rapid evolution of patches of sequence in a subset of individuals (patchytachy; </span><a class="anchor anchor-primary" href="#bib90" name="bbib90" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib90"><span class="anchor-text-container"><span class="anchor-text">Sun et&nbsp;al., 2011</span></span></a>). Both of the putatively recombined regions in the <em>M.&nbsp;fascicularis</em> genomes from Liang Bua (SP3942 and SP3934) largely comprised missing data. Visual inspection identified no discernable sign of recombination in the putatively recombined region of the <em>M.&nbsp;nigrescens</em> PM1011 genome.</div><div class="u-margin-s-bottom" id="p0135">Four recombination regions that were not flagged by RDP as possibly being caused by an evolutionary process other than recombination were all within two GenBank genomes that we excluded from phylogenetic analysis. One of these genomes was <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, which is annotated as a <em>M.&nbsp;nigra</em> genome, but phylogenetically positioned within a clade containing <em>M.&nbsp;mulatta</em> genomes based on preliminary analyses (not shown). One putatively recombined region in <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a> was 1203 base pairs long and was identified by seven of nine tests. Visual inspection supports the inference of recombination based on high similarity with the minor parental sequences. Another region of <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a> was identified by six tests, but recombination was not strongly evident based on visual inspection.</div><div class="u-margin-s-bottom" id="p0140">The second GenBank genome with a significant signature of recombination in seven tests was <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FJ906803.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">FJ906803.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, which is annotated as <em>M.&nbsp;fascicularis</em> of unknown origin. Visual inspection of two putatively recombined regions in this genome did not identify a marked indication of recombination because the regions were substantially diverged from the putative parental sequences. Because recombination can influence phylogenetic reconstruction, these two GenBank genomes were not included in the phylogenetic analyses presented here, and to our knowledge, neither has been included in a publication elsewhere. The signal of recombination in <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a> and <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FJ906803.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">FJ906803.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a> could have a technical explanation (e.g., contamination of nuclear insertions of mitochondrial origin, mapping or genotyping errors) or a biological basis. For <a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, a technical explanation for inferred recombination event 1 (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S4</span></span></a>) is supported by the geographically nonoverlapping distributions and relatively distant phylogenetic relationships of putative major (<em>M.&nbsp;mulatta</em>, <em>Macaca cyclopis</em>) and minor (<em>M.&nbsp;nigra</em>) parental sequences that are inferred to have contributed the larger and smaller fractions of sequence to the putative recombined sequence. The major parent of recombination event 3 (<a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=KP072068.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">KP072068.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) and events 2 and 5 (<a class="anchor anchor-primary" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FJ906803.1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">FJ906803.1</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>) was unknown, and the minor parents were <em>M.&nbsp;mulatta</em> (event 3) and <em>M.&nbsp;fascicularis</em> (events 2 and 5).</div></section></section><section id="sec4"><h2 id="sectitle0095" class="u-h4 u-margin-l-top u-margin-xs-bottom">4. Discussion</h2><div class="u-margin-s-bottom" id="p0145">In this study, we conducted a phylogenetic analysis of 222 mitochondrial genomes from papionin monkeys, 219 of which are from the genus <em>Macaca</em> and 60 of which are previously unpublished. These 60 new mitochondrial genomes are novel in the sense that they include (i) species not previously sampled for whole mitochondrial genome sequencing (<em>M.&nbsp;nigrescens</em>, <em>M.&nbsp;hecki</em>, <em>M.&nbsp;togeanus</em>, <em>M.&nbsp;brunnescens</em>), (ii) novel geographical regions for which full genomes and/or intraspecific variation were not previously available (<em>M.&nbsp;fascicularis</em> from several islands in the Lesser Sunda Islands, <em>M.&nbsp;nemestrina</em> from several locations on Borneo Island, all Sulawesi macaques except <em>M.&nbsp;ochreata</em> and <em>M.&nbsp;brunnescens</em>), and (iii) prehistoric specimens of <em>M.&nbsp;fascicularis</em> as old as âˆ¼2 ka that were recovered during archaeological excavations on Flores. The increased taxon sampling and broader geographic and temporal coverage afforded by these new mitochondrial genomes provide insights into macaque evolution, particularly as it relates to macaque dispersals east across Wallace's Line.</div><section id="sec4.1"><h3 id="sectitle0100" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.1. Recombination in macaque mitochondria?</h3><div class="u-margin-s-bottom" id="p0150">One concern about previous surveys of recombination in mitochondria using Sanger sequences (e.g., <a class="anchor anchor-primary" href="#bib73" name="bbib73" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib73"><span class="anchor-text-container"><span class="anchor-text">Piganeau et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib96" name="bbib96" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib96"><span class="anchor-text-container"><span class="anchor-text">Tsaousis et&nbsp;al., 2005</span></span></a>) is that these data may be affected by nuclear insertions of mitochondrial sequence (NUMTs). The very high coverage of our WGS data (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Fig.&nbsp;S1</span></span></a>) suggests that the influence of NUMTs is almost certainly small in the new data analyzed here. We identified a convincing signature of four recombination events in two mitochondrial genomes from GenBank that were excluded from our analysis, but not in the other 222 genomes we included. For one of these four events (event 1), there is a lack of geographical proximity in extant ranges and diverged phylogenetic relationships of the putative parental sequences. In three others (events 2â€“3, 5) a major sequence was not identified in the data despite a relatively robust sample of extant variation in terms of interspecific and intraspecific variation for some macaque species. The characteristics thus cast doubt on a biological (as opposed to a technical) basis for the recombination signatures in these two GenBank sequences. In the 222 other genomes we analyzed, any signal of recombination that is present was insufficiently strong to be consistently detected by the methods we deployed. This could be because recombination is rare or absent or because demographic factors, such as rapid expansion of matrilines on Sulawesi, limited our ability to detect recombination.</div></section><section id="sec4.2"><h3 id="sectitle0105" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.2. Dispersal of <em>M.&nbsp;fascicularis</em> to the Lesser Sunda Islands</h3><div class="u-margin-s-bottom" id="p0155">There are competing hypotheses for explaining the timing and nature of the dispersal of <em>M.&nbsp;fascicularis</em> to the Lesser Sunda Islands (<a class="anchor anchor-primary" href="#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1995</span></span></a>; <a class="anchor anchor-primary" href="#bib35" name="bbib35" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib35"><span class="anchor-text-container"><span class="anchor-text">Heinsohn, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib53" name="bbib53" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib53"><span class="anchor-text-container"><span class="anchor-text">Long, 2003</span></span></a>; <a class="anchor anchor-primary" href="#bib85" name="bbib85" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib85"><span class="anchor-text-container"><span class="anchor-text">Schillaci et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>). One of these is based on divergence in mitochondrial genomes and posits that this dispersal occurred naturally âˆ¼1&nbsp;Ma, which is before modern humans and possibly even other hominins reached this region (<a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>). <em>Macaca fascicularis</em> can swim (<a class="anchor anchor-primary" href="#bib101" name="bbib101" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib101"><span class="anchor-text-container"><span class="anchor-text">van Schaik et&nbsp;al., 1996</span></span></a>), and the capacity of macaques to cross significant marine barriers is evidenced by the Sulawesi macaques, whose ancestors arrived on the island long before hominins reached this region (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;2</span></span></a>; <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2010</span></span></a>, <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">2017</span></span></a>). Evidence of archaic hominins in Island Southeast Asia extends to âˆ¼1.5&nbsp;Ma based on a juvenile fossil hominin cranium found at Mojokerto, Java, in 1936, which is typically attributed to <em>Homo erectus</em> (<a class="anchor anchor-primary" href="#bib61" name="bbib61" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib61"><span class="anchor-text-container"><span class="anchor-text">Morwood et&nbsp;al., 2003</span></span></a>; <a class="anchor anchor-primary" href="#bib41" name="bbib41" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib41"><span class="anchor-text-container"><span class="anchor-text">Huffman et&nbsp;al., 2006</span></span></a>). Other evidence of <em>H.&nbsp;erectus</em> in Java includes fossils and stone artifacts found at other sites in Java including Sangiran (âˆ¼1.3â€“0.9&nbsp;Ma; <a class="anchor anchor-primary" href="#bib56" name="bbib56" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib56"><span class="anchor-text-container"><span class="anchor-text">Matsu'ura et&nbsp;al., 2020</span></span></a>), Trinil (âˆ¼0.5&nbsp;Ma; <a class="anchor anchor-primary" href="#bib45" name="bbib45" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib45"><span class="anchor-text-container"><span class="anchor-text">Joordens et&nbsp;al., 2015</span></span></a>), and Ngandong (âˆ¼0.55â€“0.11&nbsp;Ma; <a class="anchor anchor-primary" href="#bib79" name="bbib79" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib79"><span class="anchor-text-container"><span class="anchor-text">Rizal et&nbsp;al., 2020</span></span></a>). East of Huxley's modification of Wallace's line, archaic hominins are known from the islands of Luzon in the Philippines and Flores in Indonesia (<a class="anchor anchor-primary" href="#bib62" name="bbib62" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib62"><span class="anchor-text-container"><span class="anchor-text">Morwood et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib8" name="bbib8" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib8"><span class="anchor-text-container"><span class="anchor-text">Brown et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib15" name="bbib15" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib15"><span class="anchor-text-container"><span class="anchor-text">Detroit et&nbsp;al., 2019</span></span></a>). In addition, stone artifacts of unknown hominins dated to between âˆ¼200 and 100 ka have been recovered on Sulawesi at Talepu (<a class="anchor anchor-primary" href="#bib100" name="bbib100" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib100"><span class="anchor-text-container"><span class="anchor-text">van den Bergh et&nbsp;al., 2016b</span></span></a>), well before the earliest skeletal evidence of modern humans in Island Southeast Asia (73â€“63 ka in Sumatra; <a class="anchor anchor-primary" href="#bib105" name="bbib105" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib105"><span class="anchor-text-container"><span class="anchor-text">Westaway et&nbsp;al., 2017</span></span></a>). On Luzon, skeletal remains of <em>Homo luzonensis</em> are dated to minimum ages of 50 ka and 67 ka (<a class="anchor anchor-primary" href="#bib15" name="bbib15" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib15"><span class="anchor-text-container"><span class="anchor-text">Detroit et&nbsp;al., 2019</span></span></a>), whereas stone artifacts and butchered rhinoceros bones derive from sediments dated to 777â€“631 ka (<a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Ingicco et&nbsp;al., 2018</span></span></a>). On Flores, skeletal and cultural remains of <em>Homo floresiensis</em><span>, an extinct human species that stood about 106 cm tall, weighed âˆ¼27.5&nbsp;kg, and had a chimpanzee-sized <a href="/topics/agricultural-and-biological-sciences/protocerebrum" title="Learn more about brain from ScienceDirect's AI-generated Topic Pages" class="topic-link">brain</a> (âˆ¼420&nbsp;cm</span><sup>3</sup>), are dated to âˆ¼190â€“50 ka at Liang Bua (<a class="anchor anchor-primary" href="#bib8" name="bbib8" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib8"><span class="anchor-text-container"><span class="anchor-text">Brown et&nbsp;al., 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib24" name="bbib24" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib24"><span class="anchor-text-container"><span class="anchor-text">Falk et&nbsp;al., 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kubo et&nbsp;al., 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib33" name="bbib33" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib33"><span class="anchor-text-container"><span class="anchor-text">Grabowski et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib92" name="bbib92" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib92"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2016</span></span></a>). <em>Homo floresiensis</em>-like teeth and lower jaw fragments dated to âˆ¼700 ka are known from Mata Menge (<a class="anchor anchor-primary" href="#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text">Brumm et&nbsp;al., 2016</span></span></a>; <a class="anchor anchor-primary" href="#bib99" name="bbib99" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib99"><span class="anchor-text-container"><span class="anchor-text">van den Bergh et&nbsp;al., 2016a</span></span></a>), and stone artifacts from Wolo Sege are at least 1&nbsp;Ma (<a class="anchor anchor-primary" href="#bib9" name="bbib9" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib9"><span class="anchor-text-container"><span class="anchor-text">Brumm et&nbsp;al., 2010</span></span></a>). Thus, it is certainly plausible that <em>M.&nbsp;fascicularis</em> dispersed to the Lesser Sunda Islands around the same time as early hominins.</div><div class="u-margin-s-bottom" id="p0160">An alternative hypothesis is based on a lack of substantial morphological differentiation across the southern portion of Wallace's Line and a dearth of fossil evidence of macaques in the Lesser Sunda Islands and posits that <em>M.&nbsp;fascicularis</em> dispersed to these islands more recently and possibly was introduced by modern humans (<a class="anchor anchor-primary" href="#bib63" name="bbib63" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib63"><span class="anchor-text-container"><span class="anchor-text">Musser, 1981</span></span></a>; <a class="anchor anchor-primary" href="#bib35" name="bbib35" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib35"><span class="anchor-text-container"><span class="anchor-text">Heinsohn, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib29" name="bbib29" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib29"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2006</span></span></a>; <a class="anchor anchor-primary" href="#bib91" name="bbib91" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib91"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2018</span></span></a>). Numerous examples exist of modern humanâ€“mediated introduction of macaques in Southeast Asia, including <em>M.&nbsp;nigra</em> to the Bacan Islands in the Moluccas (<a class="anchor anchor-primary" href="#bib103" name="bbib103" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib103"><span class="anchor-text-container"><span class="anchor-text">Wallace, 1869</span></span></a>; <a class="anchor anchor-primary" href="#bib82" name="bbib82" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib82"><span class="anchor-text-container"><span class="anchor-text">Rosenbaum et&nbsp;al., 1998</span></span></a>), <em>M.&nbsp;togeanus</em> to Malenge Island in the Togian archipelago (<a class="anchor anchor-primary" href="#bib30" name="bbib30" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib30"><span class="anchor-text-container"><span class="anchor-text">Froehlich and Supriatna, 1996</span></span></a>), and <em>M.&nbsp;fascicularis</em> to many locations, including Kabaena Island, Mauritius, Angaur Island, New Guinea, and elsewhere (<a class="anchor anchor-primary" href="#bib74" name="bbib74" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib74"><span class="anchor-text-container"><span class="anchor-text">Poirier and Smith, 1974</span></span></a>; <a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Froehlich et&nbsp;al., 2003</span></span></a>; <a class="anchor anchor-primary" href="#bib53" name="bbib53" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib53"><span class="anchor-text-container"><span class="anchor-text">Long, 2003</span></span></a>). Although modern humans have been present in this region for tens of thousands of years, it is possible that transport of macaques and other non-endemic animals to the Lesser Sunda Islands did not take place until recent migrations associated with Neolithic culture &lt; 5 ka (<a class="anchor anchor-primary" href="#bib63" name="bbib63" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib63"><span class="anchor-text-container"><span class="anchor-text">Musser, 1981</span></span></a>; <a class="anchor anchor-primary" href="#bib35" name="bbib35" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib35"><span class="anchor-text-container"><span class="anchor-text">Heinsohn, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib29" name="bbib29" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib29"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2006</span></span></a>; <a class="anchor anchor-primary" href="#bib91" name="bbib91" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib91"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2018</span></span></a>). Evidence of modern humans in the Lesser Sunda Islands extends back to âˆ¼46â€“40 ka (<a class="anchor anchor-primary" href="#bib60" name="bbib60" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib60"><span class="anchor-text-container"><span class="anchor-text">Morley et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">O'Connor et&nbsp;al., 2010</span></span></a>, <a class="anchor anchor-primary" href="#bib69" name="bbib69" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib69"><span class="anchor-text-container"><span class="anchor-text">2011</span></span></a>; <a class="anchor anchor-primary" href="#bib91" name="bbib91" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib91"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2018</span></span></a>) and in Australia extends back to as early as âˆ¼65 ka (<a class="anchor anchor-primary" href="#bib12" name="bbib12" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib12"><span class="anchor-text-container"><span class="anchor-text">Clarkson et&nbsp;al., 2017</span></span></a>; but see <a class="anchor anchor-primary" href="#bib68" name="bbib68" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib68"><span class="anchor-text-container"><span class="anchor-text">O'Connell et&nbsp;al., 2018</span></span></a>). Genetic analyses of living humans in the Lesser Sunda Islands and the surrounding regions show that they are the descendants of early Australo-Melanesian populations that admixed with Austronesian populations (<a class="anchor anchor-primary" href="#bib40" name="bbib40" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib40"><span class="anchor-text-container"><span class="anchor-text">Hudjashov et&nbsp;al., 2017</span></span></a>). This relatively recent dispersal of Austronesian populations into Southeast Asia originated in Taiwan and reached the Lesser Sunda Islands, possibly by way of Luzon (in the Philippines) and Sulawesi (<a class="anchor anchor-primary" href="#bib40" name="bbib40" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib40"><span class="anchor-text-container"><span class="anchor-text">Hudjashov et&nbsp;al., 2017</span></span></a>), and roughly coincides with the earliest evidence of Neolithic cultural practices on these Wallacean islands (<a class="anchor anchor-primary" href="#bib3" name="bbib3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib3"><span class="anchor-text-container"><span class="anchor-text">Bellwood et&nbsp;al., 1995</span></span></a>; <a class="anchor anchor-primary" href="#bib2" name="bbib2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib2"><span class="anchor-text-container"><span class="anchor-text">Bellwood, 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib87" name="bbib87" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib87"><span class="anchor-text-container"><span class="anchor-text">Spriggs, 2012</span></span></a>; <a class="anchor anchor-primary" href="#bib48" name="bbib48" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib48"><span class="anchor-text-container"><span class="anchor-text">Ko et&nbsp;al., 2014</span></span></a>; <a class="anchor anchor-primary" href="#bib70" name="bbib70" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib70"><span class="anchor-text-container"><span class="anchor-text">O'Connor, 2015</span></span></a>).</div><div class="u-margin-s-bottom" id="p0165"><span>Direct evidence of Neolithic cultural practices, such as pottery, evidence of <a href="/topics/agricultural-and-biological-sciences/agricultural-science" title="Learn more about agriculture from ScienceDirect's AI-generated Topic Pages" class="topic-link">agriculture</a>, and putatively introduced animals, appears in Southeast Asia with variable timing across this region, perhaps before âˆ¼3,000 years ago in portions of Borneo, the Philippines, the Lesser Sunda Islands, and the Moluccas (</span><a class="anchor anchor-primary" href="#bib70" name="bbib70" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib70"><span class="anchor-text-container"><span class="anchor-text">O'Connor, 2015</span></span></a>). No evidence of macaques older than âˆ¼4 ka has been recovered on Timor (<a class="anchor anchor-primary" href="#bib32" name="bbib32" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib32"><span class="anchor-text-container"><span class="anchor-text">Glover, 1986</span></span></a>; <a class="anchor anchor-primary" href="#bib70" name="bbib70" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib70"><span class="anchor-text-container"><span class="anchor-text">O'Connor, 2015</span></span></a>) or Liang Bua (âˆ¼190 ka to present; <a class="anchor anchor-primary" href="#bib91" name="bbib91" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib91"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2018</span></span></a>) or at older sites on Flores (âˆ¼1.0â€“0.7&nbsp;Ma; <a class="anchor anchor-primary" href="#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text">Brumm et&nbsp;al., 2016</span></span></a>). Similarly, on Sumba, which was not sampled in this study but is inhabited by <em>M.&nbsp;fascicularis</em> (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>), faunal remains have been recovered from Pleistocene deposits âˆ¼1&nbsp;Ma and from Holocene deposits âˆ¼3.5â€“2.0 ka, but neither of these included macaques (<a class="anchor anchor-primary" href="#bib97" name="bbib97" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib97"><span class="anchor-text-container"><span class="anchor-text">Turvey et&nbsp;al., 2017</span></span></a><span>). On Timor, macaque remains have only ever been recovered from archaeological contexts that are younger than âˆ¼4 ka and that also include pottery and other putatively introduced mammals such as <a href="/topics/agricultural-and-biological-sciences/pig" title="Learn more about pigs from ScienceDirect's AI-generated Topic Pages" class="topic-link">pigs</a>, but none are found in deposits older than âˆ¼5 ka (</span><a class="anchor anchor-primary" href="#bib32" name="bbib32" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib32"><span class="anchor-text-container"><span class="anchor-text">Glover, 1986</span></span></a>). Macaques may have been intentionally introduced to the Lesser Sunda Islands by modern humans as a potential source of food to be hunted (<a class="anchor anchor-primary" href="#bib82" name="bbib82" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib82"><span class="anchor-text-container"><span class="anchor-text">Rosenbaum et&nbsp;al., 1998</span></span></a>). Macaques are also kept as pets in Indonesia and elsewhere (<a class="anchor anchor-primary" href="#bib44" name="bbib44" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib44"><span class="anchor-text-container"><span class="anchor-text">Jones-Engel et&nbsp;al., 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib47" name="bbib47" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib47"><span class="anchor-text-container"><span class="anchor-text">Klegarth et&nbsp;al., 2017</span></span></a>; <a class="anchor anchor-primary" href="#bib66" name="bbib66" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib66"><span class="anchor-text-container"><span class="anchor-text">Nijman et&nbsp;al., 2017</span></span></a><span>), and this may also have provided a rationale for transport of macaques between islands in the past. Escaped or released animals could thus contribute to the distribution and <a href="/topics/agricultural-and-biological-sciences/genetic-divergence" title="Learn more about genetic variation from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic variation</a> of modern macaque populations.</span></div><div class="u-margin-s-bottom"><div id="p0170">Our results have implications for discerning between these two hypotheses, but fail to conclusively reject either one. We found that the most recent divergences within three <em>M.&nbsp;fascicularis</em> mitochondrial clades that span the southern portion of Wallace's Line have 95% CIs of 40â€“20, 90â€“40, and 20â€“0 ka for clades B1-a1, B2-a, and B3-a, respectively (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S3</span></span></a>). The phylogenetic placement of clades B2 and B3 points to recent (&lt; 90 ka) eastward dispersal from Bali or Penida across Wallace's Line to Lombok, and these dispersal events may have occurred separately or concurrently with a polymorphic population. However, there are multiple interpretations of clade B1, summarized in <a class="anchor anchor-primary" href="#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;5</span></span></a>, that are conceivably consistent either with ancient (âˆ¼1&nbsp;Ma) or more recent dispersal. One possibility is that <em>M.&nbsp;fascicularis</em> dispersed to the Lesser Sunda Islands âˆ¼1&nbsp;Ma, which initiated the divergence of clade B1 from the other lineages in clade B, and then more recently, âˆ¼80 ka, descendants of these early migrants dispersed back to Java from Lombok, initiating divergence of clade B1 into B1-a1 and B1-b. The latter divergence would then have been followed by a recent dispersal from Java back to Lombok, which then initiated divergence within subclade B1-a1 âˆ¼30 ka (<a class="anchor anchor-primary" href="#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;5</span></span></a><span>; left). This scenario, however, is at odds with the archaeological/fossil record from the Lesser Sunda Islands and with the shallow divergence between lineages on separate islands of the Lesser Sunda Islands (i.e., divergence âˆ¼70 ka lineages within clade B1-b from Timor, Sumbawa, Flores), unless the effective <a href="/topics/agricultural-and-biological-sciences/population-size" title="Learn more about population size from ScienceDirect's AI-generated Topic Pages" class="topic-link">population size</a> of clade B1 mitochondrial genomes from the Lesser Sunda Islands was very small.</span></div><figure class="figure text-xs" id="fig5"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr5.jpg" height="174" alt="Figure&nbsp;5" aria-describedby="cap0030"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr5_lrg.jpg" target="_blank" download="" title="Download high-res image (161KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (161KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-gr5.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0030"><p id="fspara0030"><span class="label">Figure&nbsp;5</span>. Two alternative dispersal scenarios for the ancestors of <em>M.&nbsp;fascicularis</em> in the Lesser Sunda Islands, under the assumption that well-supported relationships among mitochondrial genomes estimated in this study are accurate. The phylogenies on the left and right are a simplified subset of relationships from <a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a>. On the left is a scenario involving an early dispersal (âˆ¼1 Mya) followed by more recent ones (thousands of years ago). On the right is a scenario with one or more recent dispersals and no early one; a dashed lineage represents an unsampled or extinct lineage from the Sunda region.</p></span></span></figure></div><div class="u-margin-s-bottom" id="p0175">A second interpretation of clade B1, which is also not consistent with the archaeological/fossil record, is that diversification of this clade began when macaques dispersed more recently (&lt; 80 ka) from Java/Bali/Penida across Wallace's Line to the Lesser Sunda Islands, bringing with them lineages within clade B1 (<a class="anchor anchor-primary" href="#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;5</span></span></a>; right). This dispersal could have involved a polymorphic population (or multiple separate ones) carrying subclades B1-a and B1-b (and perhaps clades B2 or B3 as well). In other words, in addition to subclades B3-a and B2-a, early diversification of clade B1 also could have occurred on Java or elsewhere in the Sunda region, with Javan <em>M.&nbsp;fascicularis</em> belonging to subclade B1-b either being extinct or not yet sampled. Overall, a recent dispersal of <em>M.&nbsp;fascicularis</em> to the Lesser Sunda Islands tens of thousands, rather than several hundreds of thousands, of years ago is more parsimonious in the sense that a recent dispersal minimally involves four lineages dispersing across Wallace's Line, whereas an ancient dispersal (âˆ¼1&nbsp;Ma) involves at least five lineages (<a class="anchor anchor-primary" href="#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;5</span></span></a>). If correct, a &lt; 80-ka-old dispersal of <em>M.&nbsp;fascicularis</em> to the Lesser Sunda Islands may have been mediated either intentionally or unintentionally by early modern human movements through Wallacea.</div><div class="u-margin-s-bottom" id="p0180">Some perspective on divergence times can be gleaned from the ancient samples from Liang Bua on Flores. Three of the four fragments of <em>Macaca</em> sp. from Liang Bua sampled for this study were positively identified as <em>M.&nbsp;fascicularis</em> based on their mitochondrial genomes, and all three are nested within subclade B1-b (<a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>). These specimens yielded calibrated <sup>14</sup>C ages of âˆ¼1.5 ka (two samples) and âˆ¼0.2 ka (one sample; <a class="anchor anchor-primary" href="#tbl1" name="btbl1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a><span>). DNA from the fourth specimen was identified as <a href="/topics/agricultural-and-biological-sciences/cercopithecidae" title="Learn more about Cercopithecidae from ScienceDirect's AI-generated Topic Pages" class="topic-link">Cercopithecidae</a> but was not preserved enough for sequencing, although it did yield a calibrated </span><sup>14</sup>C age of âˆ¼2.5 ka. After excluding missing data, the available sequences from the three genomes from Liang Bua (15,619 base pairs) are identical, despite their different ages, which limits their utility for calibration purposes. Nonetheless, this provides a coarse expectation (based on a small sample size) that one substitution typically takes more than âˆ¼1400 years to occur. Within clade B1-a, there are&nbsp;âˆ¼10 differences between <em>M.&nbsp;fascicularis</em><span> mitochondrial genomes from Sumbawa&nbsp;+&nbsp;Flores compared with those from Timor. It follows that these lineages diverged more than 14,000 years ago, which is consistent with the phylogenetically estimated divergence time for this node of 30 ka (40â€“20 ka). Overall, these divergence estimates seem consistent with macaque residency in the Lesser Sunda Islands before âˆ¼4 ka, but perhaps not reaching back to âˆ¼1&nbsp;Ma. With caveats discussed in the following paragraphs, most 95% CIs for divergence times suggest that our sample is not substantially affected by very recent dispersals or recently escaped or released <a href="/topics/agricultural-and-biological-sciences/captive-animals" title="Learn more about captive animals from ScienceDirect's AI-generated Topic Pages" class="topic-link">captive animals</a>, or if it possibly was (e.g., clade B3-a; </span><a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S3</span></span></a>), the geographic scope of this effect is limited.</div><div class="u-margin-s-bottom" id="p0185">Our tests for recombination suggest that these data do not appear to be substantially affected by recombination in mitochondrial DNA, which increases branch lengths and leads to exaggerated divergence times. However, there are several cautionary notes with respect to the divergence estimates in this study. Perhaps most importantly (and as illustrated by this study compared to previous ones), unsampled mitochondrial diversity from extant and prehistoric macaque populations on these islands could identify new variation that influences phylogeographic interpretations.</div><div class="u-margin-s-bottom" id="p0190"><span>Another concern is that the rate of evolution at the tips of a phylogeny is generally higher than the long-term rate of substitution at deeper portions of a phylogeny because some variants at tips are polymorphisms that will be removed by <a href="/topics/agricultural-and-biological-sciences/natural-selection" title="Learn more about natural selection from ScienceDirect's AI-generated Topic Pages" class="topic-link">natural selection</a> before they contribute to divergence (</span><a class="anchor anchor-primary" href="#bib36" name="bbib36" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib36"><span class="anchor-text-container"><span class="anchor-text">Ho et&nbsp;al., 2011</span></span></a>). We did not observe a dramatic difference in the rate estimates at the tips of the tree compared with the older branches (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM</span></span></a><span>), but this likely is an artifact of our calibration points, which are both old. To explore this possibility further, we estimated the rate ratio of nonsynonymous and <a href="/topics/agricultural-and-biological-sciences/synonymous-substitution" title="Learn more about synonymous substitutions from ScienceDirect's AI-generated Topic Pages" class="topic-link">synonymous substitutions</a> per site (dN/dS) on internal and terminal branches (</span><a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM</span></span></a>). This rate ratio (Ï‰) was significantly higher on terminal branches (0.1232) than on internal branches (0.0928; <em>p</em>&nbsp;=&nbsp;0.01). This finding is consistent with the proposals that (i) the rate of evolution is higher on terminal than on interior branches and (ii) our divergence estimates for terminal nodes (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM Table&nbsp;S3</span></span></a>) may be overestimates.</div><div class="u-margin-s-bottom" id="p0195">Thus, the disparity between the archaeological and paleontological evidence and the divergence estimates reported here can be partially explained by overestimation of divergence times at the tips of the phylogeny (<a class="anchor anchor-primary" href="#bib36" name="bbib36" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib36"><span class="anchor-text-container"><span class="anchor-text">Ho et&nbsp;al., 2011</span></span></a>). Another factor that could cause this disparity is that divergence of mitochondrial lineages often can be older than the divergence time of descendant populations or species, and the disparity between gene divergence and species divergence increases with the population size, as does the probability of incomplete lineage sorting (<a class="anchor anchor-primary" href="#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text">Edwards and Beerli, 2000</span></span></a><span>). Female <a href="/topics/agricultural-and-biological-sciences/philopatry" title="Learn more about philopatry from ScienceDirect's AI-generated Topic Pages" class="topic-link">philopatry</a> and a female-skewed adult sex ratio in macaques also have the potential to cause divergence times in mitochondrial DNA to exceed the divergence times of populations (</span><a class="anchor anchor-primary" href="#bib58" name="bbib58" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib58"><span class="anchor-text-container"><span class="anchor-text">Melnick and Hoelzer, 1992</span></span></a>; <a class="anchor anchor-primary" href="#bib37" name="bbib37" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib37"><span class="anchor-text-container"><span class="anchor-text">Hoelzer, 1997</span></span></a>; <a class="anchor anchor-primary" href="#bib38" name="bbib38" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib38"><span class="anchor-text-container"><span class="anchor-text">Hoelzer et&nbsp;al., 1998</span></span></a>). For these reasons, and in light of the archaeological and paleontological record, we do not consider our findings strong enough to reject the hypothesis that macaques were introduced in association with the Neolithic cultural diaspora in this region (i.e., &lt; 5 ka; <a class="anchor anchor-primary" href="#bib32" name="bbib32" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib32"><span class="anchor-text-container"><span class="anchor-text">Glover, 1986</span></span></a>; <a class="anchor anchor-primary" href="#bib91" name="bbib91" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib91"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2018</span></span></a>), although we do interpret the data to favor a scenario of more recent (thousands of years ago) rather than ancient (several hundreds of thousands of years ago) dispersal to the Lesser Sunda Islands. Future work could further explore these scenarios by searching for evidence of macaques on the Lesser Sunda Islands that are more than a few thousand years old. In tandem, collection and analysis of more macaque genetic data, including better geographical sampling of mitochondrial DNA and multilocus analysis of nuclear DNA, would permit more precise estimates of key parameters associated with this dispersal, such as ancestral population size, dispersal time, and the presence or absence of migration after the initial dispersal to the Lesser Sunda Islands. This is because far higher rates of recombination in nuclear DNA allow one to sample independent coalescent histories of molecular variation to a greater extent than in mitochondrial DNA.</div></section><section id="sec4.3"><h3 id="sectitle0110" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.3. Rapid dispersal on Sulawesi, fragmentation, and hybridization</h3><div class="u-margin-s-bottom" id="p0200"><span>New data and analyses reported here support previous inferences based on partial mitochondrial genomes including mitochondrial <a href="/topics/agricultural-and-biological-sciences/paraphyly" title="Learn more about paraphyly from ScienceDirect's AI-generated Topic Pages" class="topic-link">paraphyly</a> of </span><em>M.&nbsp;nemestrina</em> with respect to Sulawesi macaques, but they also fail to resolve phylogenetic relationships within Sulawesi (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>; <a class="anchor anchor-primary" href="#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 1999</span></span></a>; <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2010</span></span></a>). Although our conclusions are similar to previous studies, the present study pushes forward our understanding of macaque evolution in this region by increasing the size and likely the quality of mitochondrial sequences being considered. For example, Sanger sequences of partial mitochondrial genomes by <a class="anchor anchor-primary" href="#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al. (1999</span></span></a>, <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">2010)</span></span></a> comprised&nbsp;&lt; 14% of the genome and, as mentioned previously, are more likely to be influenced by NUMTs than the current data.</div><div class="u-margin-s-bottom" id="p0205">Similar to <em>M.&nbsp;fascicularis</em>, the deepest diversification within <em>silenus</em> group macaques is found in the Sunda region. Because we did not sequence a mitochondrial genome of a <em>M.&nbsp;nemestrina</em> specimen from south Sumatra, we were unable to determine whether this early bifurcation occurs in mitochondrial DNA of <em>M.&nbsp;nemestrina</em> from northern and southern Sumatra, which is the case with <em>M.&nbsp;fascicularis</em> (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a>; <a class="anchor anchor-primary" href="#bib108" name="bbib108" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib108"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a>), or from Sumatra and Borneo. On Sulawesi, mitochondrial diversity generally clusters by species, but with poorly resolved relationships among some of these clusters. This phylogenetic pattern with short internodes and long terminal branches matches expectations for an initially expanding population (<a class="anchor anchor-primary" href="#bib67" name="bbib67" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib67"><span class="anchor-text-container"><span class="anchor-text">Nordborg, 2001</span></span></a><span><span>). <a href="/topics/agricultural-and-biological-sciences/monophyly" title="Learn more about Monophyly from ScienceDirect's AI-generated Topic Pages" class="topic-link">Monophyly</a> of mitochondrial genomes (outside of hybrid zones) is consistent with the proposal that evolution of much of Sulawesi's terrestrial fauna was influenced by </span><a href="/topics/agricultural-and-biological-sciences/forest-fragmentation" title="Learn more about habitat fragmentation from ScienceDirect's AI-generated Topic Pages" class="topic-link">habitat fragmentation</a> (</span><a class="anchor anchor-primary" href="#bib21" name="bbib21" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib21"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2003b</span></span></a>).</div><div class="u-margin-s-bottom" id="p0210">Mitochondrial paraphyly in <em>M.&nbsp;mulatta</em>, <em>M.&nbsp;fascicularis</em>, and <em>M.&nbsp;nemestrina</em> (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;2</span></span></a>) has been reported previously (<a class="anchor anchor-primary" href="#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 1999</span></span></a>, <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">2010</span></span></a>; <a class="anchor anchor-primary" href="#bib57" name="bbib57" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib57"><span class="anchor-text-container"><span class="anchor-text">Matsudaira et&nbsp;al., 2018</span></span></a>; <a class="anchor anchor-primary" href="#bib94" name="bbib94" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib94"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2002</span></span></a>, <a class="anchor anchor-primary" href="#bib95" name="bbib95" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib95"><span class="anchor-text-container"><span class="anchor-text">2003</span></span></a>) and is possibly a consequence of sex-specific dispersal and other demographic characteristics of macaques (<a class="anchor anchor-primary" href="#bib58" name="bbib58" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib58"><span class="anchor-text-container"><span class="anchor-text">Melnick and Hoelzer, 1992</span></span></a>). Incomplete lineage sorting could account for several inferred relationships among mitochondrial genomes of Sulawesi macaques (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>, <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>) that are discordant with those among nuclear genomes, although with equivocal statistical support between the Bayesian and maximum likelihood analyses of the mitochondrial genomes.</div><div class="u-margin-s-bottom" id="p0215">We identified one female <em>M.&nbsp;tonkeana</em> specimen (PF511) that carried a mitochondrial genome that is almost identical to that carried by <em>M.&nbsp;hecki</em> specimens<em>.</em> PF511 was previously incorrectly classified as <em>M.&nbsp;hecki</em> based on partial mitochondrial DNA sequences (<a class="anchor anchor-primary" href="#bib20" name="bbib20" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib20"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2003a</span></span></a>), but a reassessment of photos and consideration of whole-genome sequences indicates that the genetic ancestry of this individual is mostly <em>M.&nbsp;tonkeana</em> (data not shown). In this case, incomplete lineage sorting is an unlikely explanation for this observation because pairwise divergence between other <em>M.&nbsp;tonkeana</em> and <em>M.&nbsp;hecki</em> mitochondrial genomes is âˆ¼20 times higher (<a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;2</span></span></a>). A more plausible explanation is that this specimen has recent mixed ancestry stemming from hybridization between these two species, followed by several generations of backcrossing with <em>M.&nbsp;tonkeana</em> individuals.</div></section></section><section id="sec5"><h2 id="sectitle0115" class="u-h4 u-margin-l-top u-margin-xs-bottom">5. Conclusions</h2><div class="u-margin-s-bottom" id="p0220">New mitochondrial genome sequences presented here provide new insights into macaque evolution in Southeast Asia. Many of these insights are possible because DNA was accessible from museum specimens and three archaeological specimens from Flores. Analysis of new samples and geographic localities of <em>M.&nbsp;fascicularis</em> yielded substantially more recent divergence estimates across the southern portion of Wallace's Line that suggest that the period of <em>M.&nbsp;fascicularis</em><span><span> dispersal to the Lesser Sunda Islands probably occurred within the past 100,000 years. Divergence times across the southern portion of Wallace's Line generally antedate the Neolithic. However, <a href="/topics/agricultural-and-biological-sciences/molecular-evolution" title="Learn more about molecular evolution from ScienceDirect's AI-generated Topic Pages" class="topic-link">molecular evolution</a> and </span><a href="/topics/agricultural-and-biological-sciences/population-genetics" title="Learn more about population genetic from ScienceDirect's AI-generated Topic Pages" class="topic-link">population genetic</a> considerations and the possibility of unsampled variation bring these observations in closer alignment with the archaeological/paleontological record and with the hypothesis that macaques dispersed to the Lesser Sunda Islands fairly recently and possibly with the arrival of Neolithic culture. On Sulawesi, new data presented here offer glimpses into a dynamic evolutionary history including early arrival and rapid colonization (short internodes, possible incomplete lineage sorting), fragmentation (monophyly of species-specific lineages), and at least one instance of recent mitochondrial gene flow (at the hybrid zone between </span><em>M.&nbsp;tonkeana</em> and <em>M.&nbsp;hecki</em>). Overall, this study offers unique insights into the dynamic evolution of macaques and implications of human cultural diasporas on biodiversity.</div></section><section id="sec6"><h2 id="sectitle0120" class="u-h4 u-margin-l-top u-margin-xs-bottom">Data accessibility</h2><div class="u-margin-s-bottom" id="p0225">The 60 new mitochondrial genome sequences have been deposited in GenBank accessions MT300196â€“MT300255.</div></section><section id="sec7"><h2 id="sectitle0120a" class="u-h4 u-margin-l-top u-margin-xs-bottom">Conflict of interest</h2><div class="u-margin-s-bottom" id="p0225a">There is no competing interest.</div></section></div><section id="ack0010"><h2 id="sectitle0125" class="u-h4 u-margin-l-top u-margin-xs-bottom">Acknowledgments</h2><div class="u-margin-s-bottom" id="p0230">We thank Svante PÃ¤Ã¤bo and the Max Planck Institute for Evolutionary Anthropology for hosting B.J.E. during a sabbatical, during which some of the work for this project was performed. We thank Roberto Portela Miguez (Natural History Museum), Katrin Krohmann (Senckenberg Museum), Verity Mathis (Florida Museum of Natural History), and I Made Geria (Pusat Penelitian Arkeologi Nasional) for granting permission to sample specimens in their collection. We thank Birgit Nickel for performing laboratory work associated with mitochondrial DNA capture, Ben Vernot for assistance with design of capture probes, David LÃ³pez HerrÃ¡ez for logistical assistance with carbon dating, Nicolas Dierckxsens for advice on NOVOPlasty, Remco Bouckaert and Michael Matschiner for advice on BEAST, Janet Kelso and Emil Karpinski for helpful feedback, and Brian Golding for access to computational resources. We thank Eric Delson, Masanaru Takai, Kelsey Pugh, and John Barry for providing information with regard to the primate fossil record. We also thank three anonymous reviewers and the handling editor for constructive comments on an earlier draft of this manuscript. We acknowledge and thank the <span id="gs1">Natural Sciences and Engineering Research Council of Canada</span> (<a class="anchor anchor-primary" href="#gs1"><span class="anchor-text-container"><span class="anchor-text">RGPIN-2017-05770</span></span></a>), <span id="gs2">Compute Canada Resource Allocation Competitions</span>, the <span id="gs3">Social Sciences and Humanities Research Council of Canada</span> (No. <a class="anchor anchor-primary" href="#gs3"><span class="anchor-text-container"><span class="anchor-text">435-2017-1234</span></span></a>), and the <span id="gs4">Max Planck Society</span> for supporting this work. Open-access publication costs were supplied by The <span id="gs5">Social Sciences and Humanities Research Council of Canada</span> (no. <a class="anchor anchor-primary" href="#gs5"><span class="anchor-text-container"><span class="anchor-text">435-2017-1234</span></span></a>). T.S. is supported by the <span id="gs6">ARC Centre of Excellence for Australian Biodiversity and Heritage</span> (<a class="anchor anchor-primary" href="#gs6"><span class="anchor-text-container"><span class="anchor-text">CE170100015</span></span></a>).</div></section><div class="Appendices"><section id="appsec1"><h2 id="sectitle0130" class="u-h4 u-margin-l-top u-margin-xs-bottom">Supplementary Online Material</h2><span class="download-all-supplemental-data"><span class="article-attachment"><iframe title="download_all" name="download_all" class="u-hide"></iframe><form method="post" action="/sdfe/pdf/download/S0047248420301135/attachments" target="download_all"><input name="eids" type="hidden" value="1-s2.0-S0047248420301135-mmc1.docx"><input name="eids" type="hidden" value="1-s2.0-S0047248420301135-mmc2.zip"><button class="button-link download-all-button u-font-sans button-link-primary button-link-icon-left" type="submit"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-download"><path d="M77.38 56.18l-6.6-7.06L54 66.36V26H44v40.34L26.28 49.1l-7.3 7.08 29.2 29.32 29.2-29.32M88 74v20H10V74H0v30h98V74H88"></path></svg><span class="button-link-text-container"><span class="button-link-text"><span class="download-all-title">Download all </span>supplementary files<span class="u-show-inline-from-sm"> included with this article</span></span></span></button></form><a class="anchor help-link u-font-sans u-margin-s-left anchor-primary anchor-has-colored-icon anchor-icon-left anchor-with-icon" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank" title="Whatâ€™s this? (Opens in new window)"><svg focusable="false" viewBox="0 0 114 128" height="20" class="icon icon-help"><path d="M57 8C35.69 7.69 15.11 21.17 6.68 40.71c-8.81 19.38-4.91 43.67 9.63 59.25 13.81 15.59 36.85 21.93 56.71 15.68 21.49-6.26 37.84-26.81 38.88-49.21 1.59-21.15-10.47-42.41-29.29-52.1C74.76 10.17 65.88 7.99 57 8zm0 10c20.38-.37 39.57 14.94 43.85 34.85 4.59 18.53-4.25 39.23-20.76 48.79-17.05 10.59-40.96 7.62-54.9-6.83-14.45-13.94-17.42-37.85-6.83-54.9C26.28 26.5 41.39 17.83 57 18zm-.14 14C45.31 32.26 40 40.43 40 50v2h10v-2c0-4.22 2.22-9.66 8-9.24 5.5.4 6.32 5.14 5.78 8.14C62.68 55.06 52 58.4 52 69.4V76h10v-5.56c0-8.16 11.22-11.52 12-21.7.74-9.86-5.56-16.52-16-16.74-.39-.01-.76-.01-1.14 0zM52 82v10h10V82H52z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Whatâ€™s this?</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></span></span><div class="u-margin-s-bottom" id="p0235">The following is the Supplementary online material to this article:<span class="display"><span class="e-component e-component-mmc1" id="mmc1"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-mmc1.docx" target="_blank" download="" title="Download Word document (711KB)"><svg focusable="false" viewBox="0 0 92 128" height="20" width="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Word document (711KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Multimedia component 1</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc2" id="mmc2"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248420301135-mmc2.zip" target="_blank" download="" title="Download zip file (80KB)"><svg focusable="false" viewBox="0 0 95 128" height="20" width="20" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download zip file (80KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Multimedia component 2</span>. </p></span></span></span></span></div></section></div></div><div class="related-content-links u-display-none-from-md"><button class="button-link button-link-primary button-link-small" type="button"><span class="button-link-text-container"><span class="button-link-text">Recommended articles</span></span></button></div><div class="Tail"></div><div><section class="bibliography u-font-serif text-s" id="cebib0010"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">References</h2><section class="bibliography-sec" id="cebibsec0010"><ol class="references" id="reference-links-cebibsec0010"><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib1" id="ref-id-bib1" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Alba et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref1"><div class="contribution"><div class="authors u-font-sans">D.M. Alba, E. Delson, G. Carnevale, S. Colombero, M. Delfino, P. Giuntelli, M. Pavia, G. Pavia</div><div id="ref-id-sref1" class="title text-m">First joint record of <em>Mesopithecus</em> and cf. <em>Macaca</em> in the Miocene of Europe</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 67 (2014), pp. 1-18</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248413002406/pdfft?md5=4d9a0e35a9eebab98485a3688865deb5&amp;pid=1-s2.0-S0047248413002406-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref1"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248413002406" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84893796429&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=First%20joint%20record%20of%20Mesopithecus%20and%20cf.%20Macaca%20in%20the%20Miocene%20of%20Europe&amp;publication_year=2014&amp;author=D.M.%20Alba&amp;author=E.%20Delson&amp;author=G.%20Carnevale&amp;author=S.%20Colombero&amp;author=M.%20Delfino&amp;author=P.%20Giuntelli&amp;author=M.%20Pavia&amp;author=G.%20Pavia" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib2" id="ref-id-bib2" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bellwood, 2005</span></span></a></span><span class="reference" id="sref2"><div class="contribution"><div class="authors u-font-sans">P. Bellwood</div><div id="ref-id-sref2" class="title text-m">Examining the farming/language dispersal hypothesis in the East Asian context</div></div><div class="host u-font-sans">R. Blench, L. Sagart, A. Sanchez-Mazas (Eds.), The peopling of East Asia: Putting together archaeology, linguistics and genetics, Routledge, London, UK (2005), pp. 41-54</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Examining%20the%20farminglanguage%20dispersal%20hypothesis%20in%20the%20East%20Asian%20context&amp;publication_year=2005&amp;author=P.%20Bellwood" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref2"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib3" id="ref-id-bib3" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bellwood et&nbsp;al., 1995</span></span></a></span><span class="reference" id="sref3"><div class="contribution"><div class="authors u-font-sans">P. Bellwood, J.J. Fox, D. Tryon</div><div id="ref-id-sref3" class="title text-m">The Austronesians in history: common origins and diverse transformations</div></div><div class="host u-font-sans">P. Bellwood, J.J. Fox, D. Tryon (Eds.), The Austronesians: historical and comparative perspectives, Australian National University, Canberra, Australia (1995), pp. 1-16</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20Austronesians%20in%20history%3A%20common%20origins%20and%20diverse%20transformations&amp;publication_year=1995&amp;author=P.%20Bellwood&amp;author=J.J.%20Fox&amp;author=D.%20Tryon" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib4" id="ref-id-bib4" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Blancher et&nbsp;al., 2008</span></span></a></span><span class="reference" id="sref4"><div class="contribution"><div class="authors u-font-sans">A. Blancher, M. Bonhomme, B. Crouau-Roy, K. Terao, T. Kitano, N. Saitou</div><div id="ref-id-sref4" class="title text-m">Mitochondrial DNA sequence phylogeny of 4 populations of the widely distributed cynomolgus macaque (<em>Macaca fascicularis fascicularis</em>)</div></div><div class="host u-font-sans">J.&nbsp;Hered., 99 (2008), pp. 254-264</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/jhered/esn003" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-43549114766&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20sequence%20phylogeny%20of%204%20populations%20of%20the%20widely%20distributed%20cynomolgus%20macaque%20&amp;publication_year=2008&amp;author=A.%20Blancher&amp;author=M.%20Bonhomme&amp;author=B.%20Crouau-Roy&amp;author=K.%20Terao&amp;author=T.%20Kitano&amp;author=N.%20Saitou" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib5" id="ref-id-bib5" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Brock et&nbsp;al., 2010</span></span></a></span><span class="reference" id="sref5"><div class="contribution"><div class="authors u-font-sans">F. Brock, T. Higham, P. Ditchfield, C.B. Ramsey</div><div id="ref-id-sref5" class="title text-m">Current Pretreatment Methods for Ams Radiocarbon Dating at the Oxford Radiocarbon Accelerator Unit (Orau)</div></div><div class="host u-font-sans">Radiocarbon, 52 (2010), pp. 103-112</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-77953655023&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Current%20Pretreatment%20Methods%20for%20Ams%20Radiocarbon%20Dating%20at%20the%20Oxford%20Radiocarbon%20Accelerator%20Unit%20&amp;publication_year=2010&amp;author=F.%20Brock&amp;author=T.%20Higham&amp;author=P.%20Ditchfield&amp;author=C.B.%20Ramsey" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib6" id="ref-id-bib6" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bronk Ramsey et&nbsp;al., 2004</span></span></a></span><span class="reference" id="sref6"><div class="contribution"><div class="authors u-font-sans">C. Bronk Ramsey, T. Higham, A. Bowles, R.E.M. Hedges</div><div id="ref-id-sref6" class="title text-m">Improvements to the pretreatment of bone at Oxford</div></div><div class="host u-font-sans">Radiocarbon, 46 (2004), pp. 155-163</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85095441543&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Improvements%20to%20the%20pretreatment%20of%20bone%20at%20Oxford&amp;publication_year=2004&amp;author=C.%20Bronk%20Ramsey&amp;author=T.%20Higham&amp;author=A.%20Bowles&amp;author=R.E.M.%20Hedges" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib7" id="ref-id-bib7" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bronk Ramsey et&nbsp;al., 2002</span></span></a></span><span class="reference" id="sref7"><div class="contribution"><div class="authors u-font-sans">C. Bronk Ramsey, T.H.F. Higham, D.C. Owen, A.W.G. Pike, R.E.M. Hedges</div><div id="ref-id-sref7" class="title text-m">Radiocarbon dates from the Oxford AMS system: Archaeometry Datelist 31</div></div><div class="host u-font-sans">Archaeometry, 44 (2002), pp. 1-149</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Radiocarbon%20dates%20from%20the%20Oxford%20AMS%20system%3A%20Archaeometry%20Datelist%2031&amp;publication_year=2002&amp;author=C.%20Bronk%20Ramsey&amp;author=T.H.F.%20Higham&amp;author=D.C.%20Owen&amp;author=A.W.G.%20Pike&amp;author=R.E.M.%20Hedges" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref7"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib8" id="ref-id-bib8" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Brown et&nbsp;al., 2004</span></span></a></span><span class="reference" id="sref8"><div class="contribution"><div class="authors u-font-sans">P. Brown, T. Sutikna, M.J. Morwood, R.P. Soejono, Jatmiko, E.W. Saptomo, R.A. Due</div><div id="ref-id-sref8" class="title text-m">A&nbsp;new small-bodied hominin from the Late Pleistocene of Flores, Indonesia</div></div><div class="host u-font-sans">Nature, 431 (2004), pp. 1055-1061</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-7744243968&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0new%20small-bodied%20hominin%20from%20the%20Late%20Pleistocene%20of%20Flores%2C%20Indonesia&amp;publication_year=2004&amp;author=P.%20Brown&amp;author=T.%20Sutikna&amp;author=M.J.%20Morwood&amp;author=R.P.%20Soejono&amp;author=Jatmiko&amp;author=E.W.%20Saptomo&amp;author=R.A.%20Due" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib9" id="ref-id-bib9" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Brumm et&nbsp;al., 2010</span></span></a></span><span class="reference" id="sref9"><div class="contribution"><div class="authors u-font-sans">A. Brumm, G.M. Jensen, G.D. van den Bergh, M.J. Morwood, I. Kurniawan, F. Aziz, M. Storey</div><div id="ref-id-sref9" class="title text-m">Hominins on Flores, Indonesia, by one million years ago</div></div><div class="host u-font-sans">Nature, 464 (2010), pp. 748-752</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature08844" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref9"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-77950439473&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref9"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Hominins%20on%20Flores%2C%20Indonesia%2C%20by%20one%20million%20years%20ago&amp;publication_year=2010&amp;author=A.%20Brumm&amp;author=G.M.%20Jensen&amp;author=G.D.%20van%20den%20Bergh&amp;author=M.J.%20Morwood&amp;author=I.%20Kurniawan&amp;author=F.%20Aziz&amp;author=M.%20Storey" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref9"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib10" id="ref-id-bib10" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Brumm et&nbsp;al., 2016</span></span></a></span><span class="reference" id="sref10"><div class="contribution"><div class="authors u-font-sans">A. Brumm, G.D. van den Bergh, M. Storey, I. Kurniawan, B.V. Alloway, R. Setiawan, E. Setiyabudi, R. Grun, M.W. Moore, D. Yurnaldi, M.R. Puspaningrum, U.P. Wibowo, H. Insani, I. Sutisna, J.A. Westgate, N.J. Pearce, M. Duval, H.J. Meijer, F. Aziz, T. Sutikna, S. van der Kaars, S. Flude, M.J. Morwood</div><div id="ref-id-sref10" class="title text-m">Age and context of the oldest known hominin fossils from Flores</div></div><div class="host u-font-sans">Nature, 534 (2016), pp. 249-253</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature17663" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84973643966&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Age%20and%20context%20of%20the%20oldest%20known%20hominin%20fossils%20from%20Flores&amp;publication_year=2016&amp;author=A.%20Brumm&amp;author=G.D.%20van%20den%20Bergh&amp;author=M.%20Storey&amp;author=I.%20Kurniawan&amp;author=B.V.%20Alloway&amp;author=R.%20Setiawan&amp;author=E.%20Setiyabudi&amp;author=R.%20Grun&amp;author=M.W.%20Moore&amp;author=D.%20Yurnaldi&amp;author=M.R.%20Puspaningrum&amp;author=U.P.%20Wibowo&amp;author=H.%20Insani&amp;author=I.%20Sutisna&amp;author=J.A.%20Westgate&amp;author=N.J.%20Pearce&amp;author=M.%20Duval&amp;author=H.J.%20Meijer&amp;author=F.%20Aziz&amp;author=T.%20Sutikna&amp;author=S.%20van%20der%20Kaars&amp;author=S.%20Flude&amp;author=M.J.%20Morwood" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib11" id="ref-id-bib11" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bynum et&nbsp;al., 1997</span></span></a></span><span class="reference" id="sref11"><div class="contribution"><div class="authors u-font-sans">E.L. Bynum, D.Z. Bynum, J. Supriatna</div><div id="ref-id-sref11" class="title text-m">Confirmation and location of the hybrid zone between wild populations of <em>Macaca tonkeana</em> and <em>Macaca hecki</em> in Central Sulawesi, Indonesia</div></div><div class="host u-font-sans">Am. J. Primatol., 43 (1997), pp. 181-209</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0030659096&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Confirmation%20and%20location%20of%20the%20hybrid%20zone%20between%20wild%20populations%20of%20Macaca%20tonkeana%20and%20Macaca%20hecki%20in%20Central%20Sulawesi%2C%20Indonesia&amp;publication_year=1997&amp;author=E.L.%20Bynum&amp;author=D.Z.%20Bynum&amp;author=J.%20Supriatna" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib12" id="ref-id-bib12" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Clarkson et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref12"><div class="contribution"><div class="authors u-font-sans">C. Clarkson, Z. Jacobs, B. Marwick, R. Fullagar, L. Wallis, M. Smith, R.G. Roberts, E. Hayes, K. Lowe, X. Carah, S.A. Florin, J. McNeil, D. Cox, L.J. Arnold, Q. Hua, J. Huntley, H.E.A. Brand, T. Manne, A. Fairbairn, J. Shulmeister, L. Lyle, M. Salinas, M. Page, K. Connell, G. Park, K. Norman, T. Murphy, C. Pardoe</div><div id="ref-id-sref12" class="title text-m">Human occupation of northern Australia by 65,000 years ago</div></div><div class="host u-font-sans">Nature, 547 (2017), pp. 306-310</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature22968" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85025177688&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Human%20occupation%20of%20northern%20Australia%20by%2065%2C000%20years%20ago&amp;publication_year=2017&amp;author=C.%20Clarkson&amp;author=Z.%20Jacobs&amp;author=B.%20Marwick&amp;author=R.%20Fullagar&amp;author=L.%20Wallis&amp;author=M.%20Smith&amp;author=R.G.%20Roberts&amp;author=E.%20Hayes&amp;author=K.%20Lowe&amp;author=X.%20Carah&amp;author=S.A.%20Florin&amp;author=J.%20McNeil&amp;author=D.%20Cox&amp;author=L.J.%20Arnold&amp;author=Q.%20Hua&amp;author=J.%20Huntley&amp;author=H.E.A.%20Brand&amp;author=T.%20Manne&amp;author=A.%20Fairbairn&amp;author=J.%20Shulmeister&amp;author=L.%20Lyle&amp;author=M.%20Salinas&amp;author=M.%20Page&amp;author=K.%20Connell&amp;author=G.%20Park&amp;author=K.%20Norman&amp;author=T.%20Murphy&amp;author=C.%20Pardoe" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib13" id="ref-id-bib13" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Delson, 1980</span></span></a></span><span class="reference" id="sref13"><div class="contribution"><div class="authors u-font-sans">E. Delson</div><div id="ref-id-sref13" class="title text-m">Fossil macaques, phyletic relationships and a scenario of development</div></div><div class="host u-font-sans">D.G. Lindburg (Ed.), The Macaques: Studies in Ecology, Behavior, and Evolution, Van Nostrand Reinhold, New York (1980), pp. 10-30</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Fossil%20macaques%2C%20phyletic%20relationships%20and%20a%20scenario%20of%20development&amp;publication_year=1980&amp;author=E.%20Delson" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib14" id="ref-id-bib14" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Delson, 2000</span></span></a></span><span class="reference" id="sref14"><div class="contribution"><div class="authors u-font-sans">E. Delson</div><div id="ref-id-sref14" class="title text-m">Cercopithecinae</div></div><div class="host u-font-sans">E. Delson, I. Tattersall (Eds.), Encyclopedia of Human Evolution and Prehistory, Garland Publishing, New York (2000), pp. 166-171</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Cercopithecinae&amp;publication_year=2000&amp;author=E.%20Delson" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref14"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib15" id="ref-id-bib15" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Detroit et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref15"><div class="contribution"><div class="authors u-font-sans">F. Detroit, A.S. Mijares, J. Corny, G. Daver, C. Zanolli, E. Dizon, E. Robles, R. Grun, P.J. Piper</div><div id="ref-id-sref15" class="title text-m">A&nbsp;new species of <em>Homo</em> from the Late Pleistocene of the Philippines</div></div><div class="host u-font-sans">Nature, 568 (2019), pp. 181-186</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/s41586-019-1067-9" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85064257068&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0new%20species%20of%20Homo%20from%20the%20Late%20Pleistocene%20of%20the%20Philippines&amp;publication_year=2019&amp;author=F.%20Detroit&amp;author=A.S.%20Mijares&amp;author=J.%20Corny&amp;author=G.%20Daver&amp;author=C.%20Zanolli&amp;author=E.%20Dizon&amp;author=E.%20Robles&amp;author=R.%20Grun&amp;author=P.J.%20Piper" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib16" id="ref-id-bib16" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Drummond and Rambaut, 2007</span></span></a></span><span class="reference" id="sref16"><div class="contribution"><div class="authors u-font-sans">A.J. Drummond, A. Rambaut</div><div id="ref-id-sref16" class="title text-m">BEAST: Bayesian evolutionary analysis by sampling trees</div></div><div class="host u-font-sans">BMC Evol. Biol., 7 (2007), p. 214</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1186/1471-2148-7-214" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-38649110663&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=BEAST%3A%20Bayesian%20evolutionary%20analysis%20by%20sampling%20trees&amp;publication_year=2007&amp;author=A.J.%20Drummond&amp;author=A.%20Rambaut" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib17" id="ref-id-bib17" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Edwards and Beerli, 2000</span></span></a></span><span class="reference" id="sref17"><div class="contribution"><div class="authors u-font-sans">S.V. Edwards, P. Beerli</div><div id="ref-id-sref17" class="title text-m">Gene divergence, population divergence, and variance in coalescence time in phylogeographic studies</div></div><div class="host u-font-sans">Evolution, 53 (2000), pp. 1839-1954</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Gene%20divergence%2C%20population%20divergence%2C%20and%20variance%20in%20coalescence%20time%20in%20phylogeographic%20studies&amp;publication_year=2000&amp;author=S.V.%20Edwards&amp;author=P.%20Beerli" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref17"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib18" id="ref-id-bib18" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 1999</span></span></a></span><span class="reference" id="sref18"><div class="contribution"><div class="authors u-font-sans">B.J. Evans, J.C. Morales, J. Supriatna, D.J. Melnick</div><div id="ref-id-sref18" class="title text-m">Origin of the Sulawesi macaques (Cercopithecidae, <em>Macaca</em>) as inferred from a mitochondrial DNA phylogeny</div></div><div class="host u-font-sans">Biol. J. Linn. Soc., 66 (1999), pp. 539-560</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0024406698902929/pdf?md5=9d0fd13bb64a8122850ac40e25f50952&amp;pid=1-s2.0-S0024406698902929-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref18"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0024406698902929" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0032953991&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20the%20Sulawesi%20macaques%20%20as%20inferred%20from%20a%20mitochondrial%20DNA%20phylogeny&amp;publication_year=1999&amp;author=B.J.%20Evans&amp;author=J.C.%20Morales&amp;author=J.%20Supriatna&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib19" id="ref-id-bib19" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2010</span></span></a></span><span class="reference" id="sref19"><div class="contribution"><div class="authors u-font-sans">B.J. Evans, L. Pin, D.J. Melnick, S.I. Wright</div><div id="ref-id-sref19" class="title text-m">Sexâ€“linked inheritance in macaque monkeys: implications for effective population size and dispersal to Sulawesi</div></div><div class="host u-font-sans">Genetics, 185 (2010), pp. 923-937</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1534/genetics.110.116228" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-78349286357&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Sexlinked%20inheritance%20in%20macaque%20monkeys%3A%20implications%20for%20effective%20population%20size%20and%20dispersal%20to%20Sulawesi&amp;publication_year=2010&amp;author=B.J.%20Evans&amp;author=L.%20Pin&amp;author=D.J.%20Melnick&amp;author=S.I.%20Wright" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib20" id="ref-id-bib20" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2003a</span></span></a></span><span class="reference" id="sref20"><div class="contribution"><div class="authors u-font-sans">B.J. Evans, J. Supriatna, N. Andayani, D.J. Melnick</div><div id="ref-id-sref20" class="title text-m">Diversification of Sulawesi macaque monkeys: decoupled evolution of mitochondrial and autosomal DNA</div></div><div class="host u-font-sans">Evolution, 57 (2003), pp. 1931-1946</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0041783471&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20Sulawesi%20macaque%20monkeys%3A%20decoupled%20evolution%20of%20mitochondrial%20and%20autosomal%20DNA&amp;publication_year=2003&amp;author=B.J.%20Evans&amp;author=J.%20Supriatna&amp;author=N.%20Andayani&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib21" id="ref-id-bib21" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2003b</span></span></a></span><span class="reference" id="sref21"><div class="contribution"><div class="authors u-font-sans">B.J. Evans, J. Supriatna, N. Andayani, M.I. Setiadi, D.C. Cannatella, D.J. Melnick</div><div id="ref-id-sref21" class="title text-m">Monkeys and toads define areas of endemism on Sulawesi</div></div><div class="host u-font-sans">Evolution, 57 (2003), pp. 1436-1443</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0038641057&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref21"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Monkeys%20and%20toads%20define%20areas%20of%20endemism%20on%20Sulawesi&amp;publication_year=2003&amp;author=B.J.%20Evans&amp;author=J.%20Supriatna&amp;author=N.%20Andayani&amp;author=M.I.%20Setiadi&amp;author=D.C.%20Cannatella&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref21"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib22" id="ref-id-bib22" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2001</span></span></a></span><span class="reference" id="sref22"><div class="contribution"><div class="authors u-font-sans">B.J. Evans, J. Supriatna, D.J. Melnick</div><div id="ref-id-sref22" class="title text-m">Hybridization and population genetics of two macaque species in Sulawesi, Indonesia</div></div><div class="host u-font-sans">Evolution, 55 (2001), pp. 1685-1702</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Hybridization%20and%20population%20genetics%20of%20two%20macaque%20species%20in%20Sulawesi%2C%20Indonesia&amp;publication_year=2001&amp;author=B.J.%20Evans&amp;author=J.%20Supriatna&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib23" id="ref-id-bib23" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Evans et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref23"><div class="contribution"><div class="authors u-font-sans">B.J. Evans, A.J. Tosi, K. Zeng, J. Dushoff, A. Corvelo, D.J. Melnick</div><div id="ref-id-sref23" class="title text-m">Speciation over the edge: gene flow among non-human primate species across a formidable biogeographic barrier</div></div><div class="host u-font-sans">R.&nbsp;Soc. Open Sci., 4 (2017)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Speciation%20over%20the%20edge%3A%20gene%20flow%20among%20non-human%20primate%20species%20across%20a%20formidable%20biogeographic%20barrier&amp;publication_year=2017&amp;author=B.J.%20Evans&amp;author=A.J.%20Tosi&amp;author=K.%20Zeng&amp;author=J.%20Dushoff&amp;author=A.%20Corvelo&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref23"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib24" id="ref-id-bib24" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Falk et&nbsp;al., 2009</span></span></a></span><span class="reference" id="sref24"><div class="contribution"><div class="authors u-font-sans">D. Falk, C. Hildebolt, K. Smith, M.J. Morwood, T. Sutikna, Jatmiko, E. Wayhu Saptomo, F. Prior</div><div id="ref-id-sref24" class="title text-m">LB1's virtual endocast, microcephaly, and hominin brain evolution</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 57 (2009), pp. 597-607</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248408002418/pdfft?md5=b82207ce36b740faf8c229cc37629002&amp;pid=1-s2.0-S0047248408002418-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref24"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248408002418" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-70449117517&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=LB1s%20virtual%20endocast%2C%20microcephaly%2C%20and%20hominin%20brain%20evolution&amp;publication_year=2009&amp;author=D.%20Falk&amp;author=C.%20Hildebolt&amp;author=K.%20Smith&amp;author=M.J.%20Morwood&amp;author=T.%20Sutikna&amp;author=Jatmiko&amp;author=E.%20Wayhu%20Saptomo&amp;author=F.%20Prior" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib25" id="ref-id-bib25" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1969</span></span></a></span><span class="reference" id="sref25"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref25" class="title text-m">Taxonomy and evolution of the monkeys of Celebes (Primates: Cercopithecidae)</div></div><div class="host u-font-sans">S. Karger, Switzerland (1969)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Taxonomy%20and%20evolution%20of%20the%20monkeys%20of%20Celebes%20&amp;publication_year=1969&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref25"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib26" id="ref-id-bib26" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1975</span></span></a></span><span class="reference" id="sref26"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref26" class="title text-m">Taxonomy and evolution of liontail and pigtail macaques (Primates: Cercopithecidae)</div></div><div class="host u-font-sans">Fieldiana Zool., 67 (1975), pp. 1-169</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Taxonomy%20and%20evolution%20of%20liontail%20and%20pigtail%20macaques%20&amp;publication_year=1975&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref26"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib27" id="ref-id-bib27" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1991</span></span></a></span><span class="reference" id="sref27"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref27" class="title text-m">Systematic review of Philippine macaques (Primates, Cercopithecidae: <em>Macaca fascicularis</em> subspp.)</div></div><div class="host u-font-sans">Fieldiana Zool., 64 (1991), pp. 1-46</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Systematic%20review%20of%20Philippine%20macaques%20&amp;publication_year=1991&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref27"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib28" id="ref-id-bib28" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1995</span></span></a></span><span class="reference" id="sref28"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref28" class="title text-m">Systematic review of southeast Asian longtail macaques, <em>Macaca fascicularis</em> (Raffles, (1821))</div></div><div class="host u-font-sans">Fieldiana Zool., 81 (i-vi) (1995), pp. 1-206</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Systematic%20review%20of%20southeast%20Asian%20longtail%20macaques%2C%20Macaca%20fascicularis%20&amp;publication_year=1995&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref28"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib29" id="ref-id-bib29" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2006</span></span></a></span><span class="reference" id="sref29"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref29" class="title text-m">Comparative review of <em>fascicularis</em>-group species of macaques (Primates: <em>Macaca</em>)</div></div><div class="host u-font-sans">Fieldiana Zool., 107 (2006), pp. 1-43</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Comparative%20review%20of%20fascicularis-group%20species%20of%20macaques%20&amp;publication_year=2006&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref29"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib30" id="ref-id-bib30" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Froehlich and Supriatna, 1996</span></span></a></span><span class="reference" id="sref30"><div class="contribution"><div class="authors u-font-sans">J. Froehlich, J. Supriatna</div><div id="ref-id-sref30" class="title text-m">Secondary intergradation between <em>Macaca maurus</em> and <em>M.&nbsp;tonkeana</em> in South Sulawesi, and the species status of <em>M.&nbsp;togeanus</em></div></div><div class="host u-font-sans">J. Fa, D.G. Lindburg (Eds.), Evolution and Ecology of Macaque Societies, Cambridge University Press, Cambridge (1996), pp. 43-70</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0030470127&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Secondary%20intergradation%20between%20Macaca%20maurus%20and%20M.%C2%A0tonkeana%20in%20South%20Sulawesi%2C%20and%20the%20species%20status%20of%20M.%C2%A0togeanus&amp;publication_year=1996&amp;author=J.%20Froehlich&amp;author=J.%20Supriatna" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib31" id="ref-id-bib31" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Froehlich et&nbsp;al., 2003</span></span></a></span><span class="reference" id="sref31"><div class="contribution"><div class="authors u-font-sans">J.W. Froehlich, M.A. Schillaci, L. Jones-Engel, D.J. Froehlich, B. Pullen</div><div id="ref-id-sref31" class="title text-m">A&nbsp;Sulawesi beachhead by longtail monkeys (<em>Macaca fascicularis</em>) on Kaneba Island, Indonesia</div></div><div class="host u-font-sans">Anthropologie, XLI (2003), pp. 17-24</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0Sulawesi%20beachhead%20by%20longtail%20monkeys%20%20on%20Kaneba%20Island%2C%20Indonesia&amp;publication_year=2003&amp;author=J.W.%20Froehlich&amp;author=M.A.%20Schillaci&amp;author=L.%20Jones-Engel&amp;author=D.J.%20Froehlich&amp;author=B.%20Pullen" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref31"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib32" id="ref-id-bib32" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Glover, 1986</span></span></a></span><span class="reference" id="sref32"><div class="contribution"><div class="authors u-font-sans">I. Glover</div><div id="ref-id-sref32" class="title text-m">Archaeology in eastern Timor, 1966-67</div></div><div class="host u-font-sans">Terra Aust., 11 (1986)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Archaeology%20in%20eastern%20Timor%2C%201966-67&amp;publication_year=1986&amp;author=I.%20Glover" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref32"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib33" id="ref-id-bib33" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Grabowski et&nbsp;al., 2015</span></span></a></span><span class="reference" id="sref33"><div class="contribution"><div class="authors u-font-sans">M. Grabowski, K.G. Hatala, W.L. Jungers, B.G. Richmond</div><div id="ref-id-sref33" class="title text-m">Body mass estimates of hominin fossils and the evolution of human body size</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 85 (2015), pp. 75-93</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248415001311/pdfft?md5=3d2a8dc8b40cf7ce08d645d345f7fe72&amp;pid=1-s2.0-S0047248415001311-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref33"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248415001311" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84938482950&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Body%20mass%20estimates%20of%20hominin%20fossils%20and%20the%20evolution%20of%20human%20body%20size&amp;publication_year=2015&amp;author=M.%20Grabowski&amp;author=K.G.%20Hatala&amp;author=W.L.%20Jungers&amp;author=B.G.%20Richmond" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib34" id="ref-id-bib34" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Groves, 2001</span></span></a></span><span class="reference" id="sref34"><div class="contribution"><div class="authors u-font-sans">C.P. Groves</div><div id="ref-id-sref34" class="title text-m">Primate Taxonomy</div></div><div class="host u-font-sans">Smithsonian Institution Press, Washington D. C., USA (2001)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Primate%20Taxonomy&amp;publication_year=2001&amp;author=C.P.%20Groves" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib35" id="ref-id-bib35" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Heinsohn, 2001</span></span></a></span><span class="reference" id="sref35"><div class="contribution"><div class="authors u-font-sans">T.E. Heinsohn</div><div id="ref-id-sref35" class="title text-m">Human influences on vertebrate zoogeography: animal translocation and biological invasions across and to the east of Wallace's Line</div></div><div class="host u-font-sans">I. Metcalfe, J.M.B. Smith, M. Morwood, I. Davidson (Eds.), Faunal and floral migrations and evolution in SE Asia-Australia, A. A. Balkema Publishers, Lisse (2001), pp. 153-170</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Human%20influences%20on%20vertebrate%20zoogeography%3A%20animal%20translocation%20and%20biological%20invasions%20across%20and%20to%20the%20east%20of%20Wallaces%20Line&amp;publication_year=2001&amp;author=T.E.%20Heinsohn" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref35"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib36" id="ref-id-bib36" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ho et&nbsp;al., 2011</span></span></a></span><span class="reference" id="sref36"><div class="contribution"><div class="authors u-font-sans">S.Y. Ho, R. Lanfear, L. Bromham, M.J. Phillips, J. Soubrier, A.G. Rodrigo, A. Cooper</div><div id="ref-id-sref36" class="title text-m">Time-dependent rates of molecular evolution</div></div><div class="host u-font-sans">Mol. Ecol., 20 (2011), pp. 3087-3101</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-79960629871&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Time-dependent%20rates%20of%20molecular%20evolution&amp;publication_year=2011&amp;author=S.Y.%20Ho&amp;author=R.%20Lanfear&amp;author=L.%20Bromham&amp;author=M.J.%20Phillips&amp;author=J.%20Soubrier&amp;author=A.G.%20Rodrigo&amp;author=A.%20Cooper" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib37" id="ref-id-bib37" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hoelzer, 1997</span></span></a></span><span class="reference" id="sref37"><div class="contribution"><div class="authors u-font-sans">G.A. Hoelzer</div><div id="ref-id-sref37" class="title text-m">Inferring phylogenies from mtDNA variation: Mitochondrial-gene trees versus nuclear-gene trees revisited</div></div><div class="host u-font-sans">Evolution, 51 (1997), pp. 622-626</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Inferring%20phylogenies%20from%20mtDNA%20variation%3A%20Mitochondrial-gene%20trees%20versus%20nuclear-gene%20trees%20revisited&amp;publication_year=1997&amp;author=G.A.%20Hoelzer" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref37"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib38" id="ref-id-bib38" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hoelzer et&nbsp;al., 1998</span></span></a></span><span class="reference" id="sref38"><div class="contribution"><div class="authors u-font-sans">G.A. Hoelzer, J. Wallman, D.J. Melnick</div><div id="ref-id-sref38" class="title text-m">The effects of social structure, geographical structure, and population size on the evolution of mitochondrial DNA: II. Molecular clocks and the lineage sorting period</div></div><div class="host u-font-sans">J.&nbsp;Mol. Evol., 47 (1998), pp. 21-31</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0031781638&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref38"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20effects%20of%20social%20structure%2C%20geographical%20structure%2C%20and%20population%20size%20on%20the%20evolution%20of%20mitochondrial%20DNA%3A%20II.%20Molecular%20clocks%20and%20the%20lineage%20sorting%20period&amp;publication_year=1998&amp;author=G.A.%20Hoelzer&amp;author=J.%20Wallman&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref38"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib39" id="ref-id-bib39" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Holt et&nbsp;al., 2013</span></span></a></span><span class="reference" id="sref39"><div class="contribution"><div class="authors u-font-sans">B.G. Holt, J.P. Lessard, M.K. Borregaard, S.A. Fritz, M.B. Araujo, D. Dimitrov, P.H. Fabre, C.H. Graham, G.R. Graves, K.A. Jonsson, D. Nogues-Bravo, Z. Wang, R.J. Whittaker, J. Fjeldsa, C. Rahbek</div><div id="ref-id-sref39" class="title text-m">An update of Wallace's zoogeographic regions of the world</div></div><div class="host u-font-sans">Science, 339 (2013), pp. 74-78</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1126/science.1228282" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref39"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84871922008&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref39"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=An%20update%20of%20Wallaces%20zoogeographic%20regions%20of%20the%20world&amp;publication_year=2013&amp;author=B.G.%20Holt&amp;author=J.P.%20Lessard&amp;author=M.K.%20Borregaard&amp;author=S.A.%20Fritz&amp;author=M.B.%20Araujo&amp;author=D.%20Dimitrov&amp;author=P.H.%20Fabre&amp;author=C.H.%20Graham&amp;author=G.R.%20Graves&amp;author=K.A.%20Jonsson&amp;author=D.%20Nogues-Bravo&amp;author=Z.%20Wang&amp;author=R.J.%20Whittaker&amp;author=J.%20Fjeldsa&amp;author=C.%20Rahbek" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref39"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib40" id="ref-id-bib40" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hudjashov et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref40"><div class="contribution"><div class="authors u-font-sans">G. Hudjashov, T.M. Karafet, D.J. Lawson, S. Downey, O. Savina, H. Sudoyo, J.S. Lansing, M.F. Hammer, M.P. Cox</div><div id="ref-id-sref40" class="title text-m">Complex patterns of admixture across the Indonesian Archipelago</div></div><div class="host u-font-sans">Mol. Biol. Evol., 34 (2017), pp. 2439-2452</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/molbev/msx196" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref40"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85030698321&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref40"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Complex%20patterns%20of%20admixture%20across%20the%20Indonesian%20Archipelago&amp;publication_year=2017&amp;author=G.%20Hudjashov&amp;author=T.M.%20Karafet&amp;author=D.J.%20Lawson&amp;author=S.%20Downey&amp;author=O.%20Savina&amp;author=H.%20Sudoyo&amp;author=J.S.%20Lansing&amp;author=M.F.%20Hammer&amp;author=M.P.%20Cox" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref40"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib41" id="ref-id-bib41" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Huffman et&nbsp;al., 2006</span></span></a></span><span class="reference" id="sref41"><div class="contribution"><div class="authors u-font-sans">O.F. Huffman, Y. Zaim, J. Kappelman, D.R. Ruez Jr., J. de Vos, Y. Rizal, F. Aziz, C. Hertler</div><div id="ref-id-sref41" class="title text-m">Relocation of the 1936 Mojokerto skull discovery site near Perning, East Java</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 50 (2006), pp. 431-451</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248405001995/pdfft?md5=18831fac4fa2fe86212344e47de0c98e&amp;pid=1-s2.0-S0047248405001995-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref41"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248405001995" aria-describedby="ref-id-sref41"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33644858867&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref41"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Relocation%20of%20the%201936%20Mojokerto%20skull%20discovery%20site%20near%20Perning%2C%20East%20Java&amp;publication_year=2006&amp;author=O.F.%20Huffman&amp;author=Y.%20Zaim&amp;author=J.%20Kappelman&amp;author=D.R.%20Ruez%20Jr.&amp;author=J.%20de%20Vos&amp;author=Y.%20Rizal&amp;author=F.%20Aziz&amp;author=C.%20Hertler" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref41"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib42" id="ref-id-bib42" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Huxley, 1868</span></span></a></span><span class="reference" id="sref42"><div class="contribution"><div class="authors u-font-sans">T.H. Huxley</div><div id="ref-id-sref42" class="title text-m">On the classification and distribution of the Alectoromorphae and Heteromorphae</div></div><div class="host u-font-sans">Proc. Zool. Soc. Lond., 1868 (1868), pp. 294-319</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=On%20the%20classification%20and%20distribution%20of%20the%20Alectoromorphae%20and%20Heteromorphae&amp;publication_year=1868&amp;author=T.H.%20Huxley" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref42"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib43" id="ref-id-bib43" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ingicco et&nbsp;al., 2018</span></span></a></span><span class="reference" id="sref43"><div class="contribution"><div class="authors u-font-sans">T. Ingicco, G.D. van den Bergh, C. Jago-On, J.J. Bahain, M.G. Chacon, N. Amano, H. Forestier, C. King, K. Manalo, S. Nomade, A. Pereira, M.C. Reyes, A.M. Semah, Q. Shao, P. Voinchet, C. Falgueres, P.C.H. Albers, M. Lising, G. Lyras, D. Yurnaldi, P. Rochette, A. Bautista, J. de Vos</div><div id="ref-id-sref43" class="title text-m">Earliest known hominin activity in the Philippines by 709 thousand years ago</div></div><div class="host u-font-sans">Nature, 557 (2018), pp. 233-237</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/s41586-018-0072-8" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref43"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85046669438&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref43"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Earliest%20known%20hominin%20activity%20in%20the%20Philippines%20by%20709%20thousand%20years%20ago&amp;publication_year=2018&amp;author=T.%20Ingicco&amp;author=G.D.%20van%20den%20Bergh&amp;author=C.%20Jago-On&amp;author=J.J.%20Bahain&amp;author=M.G.%20Chacon&amp;author=N.%20Amano&amp;author=H.%20Forestier&amp;author=C.%20King&amp;author=K.%20Manalo&amp;author=S.%20Nomade&amp;author=A.%20Pereira&amp;author=M.C.%20Reyes&amp;author=A.M.%20Semah&amp;author=Q.%20Shao&amp;author=P.%20Voinchet&amp;author=C.%20Falgueres&amp;author=P.C.H.%20Albers&amp;author=M.%20Lising&amp;author=G.%20Lyras&amp;author=D.%20Yurnaldi&amp;author=P.%20Rochette&amp;author=A.%20Bautista&amp;author=J.%20de%20Vos" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref43"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib44" id="ref-id-bib44" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Jones-Engel et&nbsp;al., 2005</span></span></a></span><span class="reference" id="sref44"><div class="contribution"><div class="authors u-font-sans">L. Jones-Engel, M.A. Schillaci, G.A. Engel, U. Paputungan, D.J. Froehlich</div><div id="ref-id-sref44" class="title text-m">Characterizing primate pet ownership in Sulawesi: Implications for disease&nbsp;transmission</div></div><div class="host u-font-sans">J. Patterson, J. Wallis (Eds.), Primate Commensalism: The primate-human interface, American Society of Primatologists (2005), pp. 196-221</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Characterizing%20primate%20pet%20ownership%20in%20Sulawesi%3A%20Implications%20for%20disease%C2%A0transmission&amp;publication_year=2005&amp;author=L.%20Jones-Engel&amp;author=M.A.%20Schillaci&amp;author=G.A.%20Engel&amp;author=U.%20Paputungan&amp;author=D.J.%20Froehlich" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref44"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib45" id="ref-id-bib45" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Joordens et&nbsp;al., 2015</span></span></a></span><span class="reference" id="sref45"><div class="contribution"><div class="authors u-font-sans">J.C. Joordens, F. d'Errico, F.P. Wesselingh, S. Munro, J. de Vos, J. Wallinga, C. Ankjaergaard, T. Reimann, J.R. Wijbrans, K.F. Kuiper, H.J. Mucher, H. Coqueugniot, V. Prie, I. Joosten, B. van Os, A.S. Schulp, M. Panuel, V. van der Haas, W. Lustenhouwer, J.J. Reijmer, W. Roebroeks</div><div id="ref-id-sref45" class="title text-m"><em>Homo erectus</em> at Trinil on Java used shells for tool production and engraving</div></div><div class="host u-font-sans">Nature, 518 (2015), pp. 228-231</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature13962" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref45"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84925283338&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref45"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Homo%20erectus%20at%20Trinil%20on%20Java%20used%20shells%20for%20tool%20production%20and%20engraving&amp;publication_year=2015&amp;author=J.C.%20Joordens&amp;author=F.%20d'Errico&amp;author=F.P.%20Wesselingh&amp;author=S.%20Munro&amp;author=J.%20de%20Vos&amp;author=J.%20Wallinga&amp;author=C.%20Ankjaergaard&amp;author=T.%20Reimann&amp;author=J.R.%20Wijbrans&amp;author=K.F.%20Kuiper&amp;author=H.J.%20Mucher&amp;author=H.%20Coqueugniot&amp;author=V.%20Prie&amp;author=I.%20Joosten&amp;author=B.%20van%20Os&amp;author=A.S.%20Schulp&amp;author=M.%20Panuel&amp;author=V.%20van%20der%20Haas&amp;author=W.%20Lustenhouwer&amp;author=J.J.%20Reijmer&amp;author=W.%20Roebroeks" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref45"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib46" id="ref-id-bib46" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Katoh and Standley, 2013</span></span></a></span><span class="reference" id="sref46"><div class="contribution"><div class="authors u-font-sans">K. Katoh, S.M. Standley</div><div id="ref-id-sref46" class="title text-m">MAFFT multiple sequence alignment software version 7: Improvements in performance and usability</div></div><div class="host u-font-sans">Mol. Biol. Evol., 30 (2013), pp. 772-780</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/molbev/mst010" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref46"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84875619226&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref46"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20Improvements%20in%20performance%20and%20usability&amp;publication_year=2013&amp;author=K.%20Katoh&amp;author=S.M.%20Standley" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref46"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib47" id="ref-id-bib47" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Klegarth et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref47"><div class="contribution"><div class="authors u-font-sans">A.R. Klegarth, S.A. Sanders, A.D. Gloss, K.E. Lane-deGraaf, L. Jones-Engel, A. Fuentes, H. Hollocher</div><div id="ref-id-sref47" class="title text-m">Investigating biogeographic boundaries of the Sunda shelf: A phylogenetic analysis of two island populations of <em>Macaca fascicularis</em></div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 163 (2017), pp. 658-670</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.23235" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref47"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85018882713&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref47"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Investigating%20biogeographic%20boundaries%20of%20the%20Sunda%20shelf%3A%20A%20phylogenetic%20analysis%20of%20two%20island%20populations%20of%20Macaca%20fascicularis&amp;publication_year=2017&amp;author=A.R.%20Klegarth&amp;author=S.A.%20Sanders&amp;author=A.D.%20Gloss&amp;author=K.E.%20Lane-deGraaf&amp;author=L.%20Jones-Engel&amp;author=A.%20Fuentes&amp;author=H.%20Hollocher" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref47"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib48" id="ref-id-bib48" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ko et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref48"><div class="contribution"><div class="authors u-font-sans">A.M.S. Ko, C.Y. Chen, Q.M. Fu, F. Delfin, M.K. Li, H.L. Chiu, M. Stoneking, Y.C. Ko</div><div id="ref-id-sref48" class="title text-m">Early Austronesians: Into and out of Taiwan</div></div><div class="host u-font-sans">Am. J. Hum. Genet., 94 (2014), pp. 426-436</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0002929714000615/pdfft?md5=9353878e1f5359315afe26234a4c0397&amp;pid=1-s2.0-S0002929714000615-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref48"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0002929714000615" aria-describedby="ref-id-sref48"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84896789927&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref48"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Early%20Austronesians%3A%20Into%20and%20out%20of%20Taiwan&amp;publication_year=2014&amp;author=A.M.S.%20Ko&amp;author=C.Y.%20Chen&amp;author=Q.M.%20Fu&amp;author=F.%20Delfin&amp;author=M.K.%20Li&amp;author=H.L.%20Chiu&amp;author=M.%20Stoneking&amp;author=Y.C.%20Ko" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref48"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib49" id="ref-id-bib49" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Kraytsberg et&nbsp;al., 2004</span></span></a></span><span class="reference" id="sref49"><div class="contribution"><div class="authors u-font-sans">Y. Kraytsberg, M. Schwartz, T.A. Brown, K. Ebralidse, W.S. Kunz, D.A. Clayton, J. Vissing, K. Khrapko</div><div id="ref-id-sref49" class="title text-m">Recombination of human mitochondrial DNA</div></div><div class="host u-font-sans">Science, 304 (2004), p. 981</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-2442609784&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref49"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Recombination%20of%20human%20mitochondrial%20DNA&amp;publication_year=2004&amp;author=Y.%20Kraytsberg&amp;author=M.%20Schwartz&amp;author=T.A.%20Brown&amp;author=K.%20Ebralidse&amp;author=W.S.%20Kunz&amp;author=D.A.%20Clayton&amp;author=J.%20Vissing&amp;author=K.%20Khrapko" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref49"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib50" id="ref-id-bib50" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Kubo et&nbsp;al., 2013</span></span></a></span><span class="reference" id="sref50"><div class="contribution"><div class="authors u-font-sans">D. Kubo, R.T. Kono, Y. Kaifu</div><div id="ref-id-sref50" class="title text-m">Brain size of <em>Homo floresiensis</em> and its evolutionary implications</div></div><div class="host u-font-sans">Proc. Roy. Soc. Lond. B, 280 (2013), p. 20130338</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rspb.2013.0338" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref50"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Brain%20size%20of%20Homo%20floresiensis%20and%20its%20evolutionary%20implications&amp;publication_year=2013&amp;author=D.%20Kubo&amp;author=R.T.%20Kono&amp;author=Y.%20Kaifu" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref50"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib51" id="ref-id-bib51" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Leakey, 1993</span></span></a></span><span class="reference" id="sref51"><div class="contribution"><div class="authors u-font-sans">M.G. Leakey</div><div id="ref-id-sref51" class="title text-m">Evolution of <em>Theropithecus</em> in the Turkana Basin</div></div><div class="host u-font-sans">N.G. Jablonski (Ed.), <em>Theropithecus</em>, the Rise and Fall of a Primate Genus, Cambridge University Press, Cambridge (1993), pp. 85-124</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20Theropithecus%20in%20the%20Turkana%20Basin&amp;publication_year=1993&amp;author=M.G.%20Leakey" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref51"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib52" id="ref-id-bib52" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Liedigk et&nbsp;al., 2015</span></span></a></span><span class="reference" id="sref52"><div class="contribution"><div class="authors u-font-sans">R. Liedigk, J. Kolleck, K.O. Boker, E. Meijaard, B.M. Md-Zain, M.A. Abdul-Latiff, A. Ampeng, M. Lakim, P. Abdul-Patah, A.J. Tosi, M. Brameier, D. Zinner, C. Roos</div><div id="ref-id-sref52" class="title text-m">Mitogenomic phylogeny of the common long-tailed macaque (<em>Macaca fascicularis fascicularis</em>)</div></div><div class="host u-font-sans">BMC Genom., 16 (2015), p. 222</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84925304379&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref52"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Mitogenomic%20phylogeny%20of%20the%20common%20long-tailed%20macaque%20&amp;publication_year=2015&amp;author=R.%20Liedigk&amp;author=J.%20Kolleck&amp;author=K.O.%20Boker&amp;author=E.%20Meijaard&amp;author=B.M.%20Md-Zain&amp;author=M.A.%20Abdul-Latiff&amp;author=A.%20Ampeng&amp;author=M.%20Lakim&amp;author=P.%20Abdul-Patah&amp;author=A.J.%20Tosi&amp;author=M.%20Brameier&amp;author=D.%20Zinner&amp;author=C.%20Roos" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref52"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib53" id="ref-id-bib53" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Long, 2003</span></span></a></span><span class="reference" id="sref53"><div class="contribution"><div class="authors u-font-sans">J.L. Long</div><div id="ref-id-sref53" class="title text-m">Introduced Mammals of the World: Their History, Distribution, and Influence</div></div><div class="host u-font-sans">CSIRO Publishing, Collingwood, Australia (2003)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Introduced%20Mammals%20of%20the%20World%3A%20Their%20History%2C%20Distribution%2C%20and%20Influence&amp;publication_year=2003&amp;author=J.L.%20Long" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref53"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib54" id="ref-id-bib54" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ma and O'Farrell, 2015</span></span></a></span><span class="reference" id="sref54"><div class="contribution"><div class="authors u-font-sans">H. Ma, P.H. O'Farrell</div><div id="ref-id-sref54" class="title text-m">Selections that isolate recombinant mitochondrial genomes in animals</div></div><div class="host u-font-sans">Elife, 4 (2015)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Selections%20that%20isolate%20recombinant%20mitochondrial%20genomes%20in%20animals&amp;publication_year=2015&amp;author=H.%20Ma&amp;author=P.H.%20O'Farrell" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref54"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib55" id="ref-id-bib55" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Martin and Rybicki, 2000</span></span></a></span><span class="reference" id="sref55"><div class="contribution"><div class="authors u-font-sans">D. Martin, E. Rybicki</div><div id="ref-id-sref55" class="title text-m">RDP: Detection of recombination amongst aligned sequences</div></div><div class="host u-font-sans">Bioinformatics, 16 (2000), pp. 562-563</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033824415&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref55"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=RDP%3A%20Detection%20of%20recombination%20amongst%20aligned%20sequences&amp;publication_year=2000&amp;author=D.%20Martin&amp;author=E.%20Rybicki" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref55"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib56" id="ref-id-bib56" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Matsu'ura et&nbsp;al., 2020</span></span></a></span><span class="reference" id="sref56"><div class="contribution"><div class="authors u-font-sans">S. Matsu'ura, M. Kondo, T. Danhara, S. Sakata, H. Iwano, T. Hirata, I. Kurniawan, E. Setiyabudi, Y. Takeshita, M. Hyodo, I. Kitaba, M. Sudo, Y. Danhara, F. Aziz</div><div id="ref-id-sref56" class="title text-m">Age control of the first appearance datum for Javanese <em>Homo erectus</em> in the Sangiran area</div></div><div class="host u-font-sans">Science, 367 (2020), pp. 210-214</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1126/science.aau8556" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref56"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85077697364&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref56"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Age%20control%20of%20the%20first%20appearance%20datum%20for%20Javanese%20Homo%20erectus%20in%20the%20Sangiran%20area&amp;publication_year=2020&amp;author=S.%20Matsu'ura&amp;author=M.%20Kondo&amp;author=T.%20Danhara&amp;author=S.%20Sakata&amp;author=H.%20Iwano&amp;author=T.%20Hirata&amp;author=I.%20Kurniawan&amp;author=E.%20Setiyabudi&amp;author=Y.%20Takeshita&amp;author=M.%20Hyodo&amp;author=I.%20Kitaba&amp;author=M.%20Sudo&amp;author=Y.%20Danhara&amp;author=F.%20Aziz" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref56"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib57" id="ref-id-bib57" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Matsudaira et&nbsp;al., 2018</span></span></a></span><span class="reference" id="sref57"><div class="contribution"><div class="authors u-font-sans">K. Matsudaira, Y. Hamada, S. Bunlungsup, T. Ishida, A.M. San, S. Malaivijitnond</div><div id="ref-id-sref57" class="title text-m">Whole mitochondrial genomic and Y-chromosomal phylogenies of Burmese long-tailed macaque (<em>Macaca fascicularis aurea</em>) suggest ancient hybridization between <em>fascicularis</em> and <em>sinica</em> species groups</div></div><div class="host u-font-sans">J.&nbsp;Hered., 109 (2018), pp. 360-371</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/jhered/esx108" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref57"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85047062813&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref57"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Whole%20mitochondrial%20genomic%20and%20Y-chromosomal%20phylogenies%20of%20Burmese%20long-tailed%20macaque%20%20suggest%20ancient%20hybridization%20between%20fascicularis%20and%20sinica%20species%20groups&amp;publication_year=2018&amp;author=K.%20Matsudaira&amp;author=Y.%20Hamada&amp;author=S.%20Bunlungsup&amp;author=T.%20Ishida&amp;author=A.M.%20San&amp;author=S.%20Malaivijitnond" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref57"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib58" id="ref-id-bib58" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Melnick and Hoelzer, 1992</span></span></a></span><span class="reference" id="sref58"><div class="contribution"><div class="authors u-font-sans">D.J. Melnick, G.A. Hoelzer</div><div id="ref-id-sref58" class="title text-m">Differences in male and female macaque dispersal lead to contrasting distributions of nuclear and mitochondrial DNA variation</div></div><div class="host u-font-sans">Int. J. Primatol., 13 (1992), pp. 379-393</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-51249166471&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref58"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20male%20and%20female%20macaque%20dispersal%20lead%20to%20contrasting%20distributions%20of%20nuclear%20and%20mitochondrial%20DNA%20variation&amp;publication_year=1992&amp;author=D.J.%20Melnick&amp;author=G.A.%20Hoelzer" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref58"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib59" id="ref-id-bib59" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Mohd Salleh et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref59"><div class="contribution"><div class="authors u-font-sans">F. Mohd Salleh, J. Ramos-Madrigal, F. Penaloza, S. Liu, S.S. Mikkel-Holger, P.P. Riddhi, R. Martins, D. Lenz, J. Fickel, C. Roos, M.S. Shamsir, M.S. Azman, K.L. Burton, J.R. Stephen, A. Wilting, M.T.P. Gilbert</div><div id="ref-id-sref59" class="title text-m">An expanded mammal mitogenome dataset from Southeast Asia</div></div><div class="host u-font-sans">GigaScience, 6 (2017), pp. 1-8</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1017/S0025315417001734" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref59"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=An%20expanded%20mammal%20mitogenome%20dataset%20from%20Southeast%20Asia&amp;publication_year=2017&amp;author=F.%20Mohd%20Salleh&amp;author=J.%20Ramos-Madrigal&amp;author=F.%20Penaloza&amp;author=S.%20Liu&amp;author=S.S.%20Mikkel-Holger&amp;author=P.P.%20Riddhi&amp;author=R.%20Martins&amp;author=D.%20Lenz&amp;author=J.%20Fickel&amp;author=C.%20Roos&amp;author=M.S.%20Shamsir&amp;author=M.S.%20Azman&amp;author=K.L.%20Burton&amp;author=J.R.%20Stephen&amp;author=A.%20Wilting&amp;author=M.T.P.%20Gilbert" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref59"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib60" id="ref-id-bib60" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Morley et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref60"><div class="contribution"><div class="authors u-font-sans">M.W. Morley, P. Goldberg, T. Sutikna, M.W. Tocheri, L.C. Prinsloo, E.W. Saptomo, S. Wasisto, R.G. Roberts</div><div id="ref-id-sref60" class="title text-m">Initial micromorphological results from Liang Bua, Flores (Indonesia): site formation processes and hominin activities at the type locality of <em>Homo floresiensis</em></div></div><div class="host u-font-sans">Archaeol. Sci., 77 (2017), pp. 125-142</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0305440316300838/pdfft?md5=7a7477111fa039f89e20d30f7db49244&amp;pid=1-s2.0-S0305440316300838-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref60"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0305440316300838" aria-describedby="ref-id-sref60"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84978472912&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref60"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Initial%20micromorphological%20results%20from%20Liang%20Bua%2C%20Flores%20%3A%20site%20formation%20processes%20and%20hominin%20activities%20at%20the%20type%20locality%20of%20Homo%20floresiensis&amp;publication_year=2017&amp;author=M.W.%20Morley&amp;author=P.%20Goldberg&amp;author=T.%20Sutikna&amp;author=M.W.%20Tocheri&amp;author=L.C.%20Prinsloo&amp;author=E.W.%20Saptomo&amp;author=S.%20Wasisto&amp;author=R.G.%20Roberts" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref60"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib61" id="ref-id-bib61" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Morwood et&nbsp;al., 2003</span></span></a></span><span class="reference" id="sref61"><div class="contribution"><div class="authors u-font-sans">M.J. Morwood, P. O'Sullivan, E.E. Susanto, F. Aziz</div><div id="ref-id-sref61" class="title text-m">Revised age for Mojokerto 1, an early <em>Homo erectus</em> cranium from East Java</div></div><div class="host u-font-sans">Aust. Archaeol., 57 (2003), pp. 1-4</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1080/03122417.2003.11681757" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref61"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33644860677&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref61"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Revised%20age%20for%20Mojokerto%201%2C%20an%20early%20Homo%20erectus%20cranium%20from%20East%20Java&amp;publication_year=2003&amp;author=M.J.%20Morwood&amp;author=P.%20O'Sullivan&amp;author=E.E.%20Susanto&amp;author=F.%20Aziz" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref61"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib62" id="ref-id-bib62" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Morwood et&nbsp;al., 2004</span></span></a></span><span class="reference" id="sref62"><div class="contribution"><div class="authors u-font-sans">M.J. Morwood, R.P. Soejono, R.G. Roberts, T. Sutikna, C.S. Turney, K.E. Westaway, W.J. Rink, J.X. Zhao, G.D. van den Bergh, R.A. Due, D.R. Hobbs, M.W. Moore, M.I. Bird, L.K. Fifield</div><div id="ref-id-sref62" class="title text-m">Archaeology and age of a new hominin from Flores in eastern Indonesia</div></div><div class="host u-font-sans">Nature, 431 (2004), pp. 1087-1091</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-7744228241&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref62"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Archaeology%20and%20age%20of%20a%20new%20hominin%20from%20Flores%20in%20eastern%20Indonesia&amp;publication_year=2004&amp;author=M.J.%20Morwood&amp;author=R.P.%20Soejono&amp;author=R.G.%20Roberts&amp;author=T.%20Sutikna&amp;author=C.S.%20Turney&amp;author=K.E.%20Westaway&amp;author=W.J.%20Rink&amp;author=J.X.%20Zhao&amp;author=G.D.%20van%20den%20Bergh&amp;author=R.A.%20Due&amp;author=D.R.%20Hobbs&amp;author=M.W.%20Moore&amp;author=M.I.%20Bird&amp;author=L.K.%20Fifield" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref62"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib63" id="ref-id-bib63" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Musser, 1981</span></span></a></span><span class="reference" id="sref63"><div class="contribution"><div class="authors u-font-sans">G.G. Musser</div><div id="ref-id-sref63" class="title text-m">The giant rat of Flores and its relativees east of Borneo and Bali</div></div><div class="host u-font-sans">Bull. Am. Mus. Nat. Hist., 169 (1981), pp. 67-176</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20giant%20rat%20of%20Flores%20and%20its%20relativees%20east%20of%20Borneo%20and%20Bali&amp;publication_year=1981&amp;author=G.G.%20Musser" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref63"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib64" id="ref-id-bib64" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Myers et&nbsp;al., 2000</span></span></a></span><span class="reference" id="sref64"><div class="contribution"><div class="authors u-font-sans">N. Myers, R.A. Mittermeier, C.G. Mittermeier, G.A.B. da Fonseca, J. Kent</div><div id="ref-id-sref64" class="title text-m">Biodiversity hotspots for conservation priorities</div></div><div class="host u-font-sans">Nature, 403 (2000), pp. 853-858</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0034708178&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref64"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Biodiversity%20hotspots%20for%20conservation%20priorities&amp;publication_year=2000&amp;author=N.%20Myers&amp;author=R.A.%20Mittermeier&amp;author=C.G.%20Mittermeier&amp;author=G.A.B.%20da%20Fonseca&amp;author=J.%20Kent" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref64"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib65" id="ref-id-bib65" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Nguyen et&nbsp;al., 2015</span></span></a></span><span class="reference" id="sref65"><div class="contribution"><div class="authors u-font-sans">L.T. Nguyen, H.A. Schmidt, A. von Haeseler, B.Q. Minh</div><div id="ref-id-sref65" class="title text-m">IQ-TREE: A fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</div></div><div class="host u-font-sans">Mol. Biol. Evol., 32 (2015), pp. 268-274</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/molbev/msu300" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref65"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=IQ-TREE%3A%20A%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;publication_year=2015&amp;author=L.T.%20Nguyen&amp;author=H.A.%20Schmidt&amp;author=A.%20von%20Haeseler&amp;author=B.Q.%20Minh" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref65"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib66" id="ref-id-bib66" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Nijman et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref66"><div class="contribution"><div class="authors u-font-sans">V. Nijman, D. Spaan, E.J. Rode-Margono, Wirdateti, K.A.I. Nekaris</div><div id="ref-id-sref66" class="title text-m">Changes in the primate trade in Indonesian wildlife markets over a 25-year period: fewer apes and langurs, more macaques, and slow lorises</div></div><div class="host u-font-sans">Am. J. Primatol., 79 (2017)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Changes%20in%20the%20primate%20trade%20in%20Indonesian%20wildlife%20markets%20over%20a%2025-year%20period%3A%20fewer%20apes%20and%20langurs%2C%20more%20macaques%2C%20and%20slow%20lorises&amp;publication_year=2017&amp;author=V.%20Nijman&amp;author=D.%20Spaan&amp;author=E.J.%20Rode-Margono&amp;author=Wirdateti&amp;author=K.A.I.%20Nekaris" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref66"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib67" id="ref-id-bib67" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Nordborg, 2001</span></span></a></span><span class="reference" id="sref67"><div class="contribution"><div class="authors u-font-sans">M. Nordborg</div><div id="ref-id-sref67" class="title text-m">Coalescent theory</div></div><div class="host u-font-sans">D. Balding, M. Bishop, C. Cannings (Eds.), Handbook of Statistical Genetics, Wiley, Chichester (2001), pp. 1-37</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Coalescent%20theory&amp;publication_year=2001&amp;author=M.%20Nordborg" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref67"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib68" id="ref-id-bib68" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">O'Connell et&nbsp;al., 2018</span></span></a></span><span class="reference" id="sref68"><div class="contribution"><div class="authors u-font-sans">J.F. O'Connell, J. Allen, M.A.J. Williams, A.N. Williams, C.S.M. Turney, N.A. Spooner, J. Kamminga, G. Brown, A. Cooper</div><div id="ref-id-sref68" class="title text-m">When did <em>Homo sapiens</em> first reach Southeast Asia and Sahul?</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 115 (2018), pp. 8482-8490</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1808385115" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref68"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85051825953&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref68"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=When%20did%20Homo%20sapiens%20first%20reach%20Southeast%20Asia%20and%20Sahul&amp;publication_year=2018&amp;author=J.F.%20O'Connell&amp;author=J.%20Allen&amp;author=M.A.J.%20Williams&amp;author=A.N.%20Williams&amp;author=C.S.M.%20Turney&amp;author=N.A.%20Spooner&amp;author=J.%20Kamminga&amp;author=G.%20Brown&amp;author=A.%20Cooper" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref68"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib69" id="ref-id-bib69" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">O'Connor et&nbsp;al., 2011</span></span></a></span><span class="reference" id="sref69"><div class="contribution"><div class="authors u-font-sans">S. O'Connor, R. Ono, C. Clarkson</div><div id="ref-id-sref69" class="title text-m">Pelagic fishing at 42,000 years before the present and the maritime skills of modern humans</div></div><div class="host u-font-sans">Science, 334 (2011), pp. 1117-1121</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1126/science.1207703" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref69"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-82255169476&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref69"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Pelagic%20fishing%20at%2042%2C000%20years%20before%20the%20present%20and%20the%20maritime%20skills%20of%20modern%20humans&amp;publication_year=2011&amp;author=S.%20O'Connor&amp;author=R.%20Ono&amp;author=C.%20Clarkson" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref69"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib70" id="ref-id-bib70" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">O'Connor, 2015</span></span></a></span><span class="reference" id="sref70"><div class="contribution"><div class="authors u-font-sans">S. O'Connor</div><div id="ref-id-sref70" class="title text-m">Rethinking the Neolithic in Island Southeast Asia, with particular reference to the archaeology of Timor-Leste and Sulawesi</div></div><div class="host u-font-sans">Archipel, 90 (2015), pp. 15-47</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.4000/archipel.362" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref70"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84948986175&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref70"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Rethinking%20the%20Neolithic%20in%20Island%20Southeast%20Asia%2C%20with%20particular%20reference%20to%20the%20archaeology%20of%20Timor-Leste%20and%20Sulawesi&amp;publication_year=2015&amp;author=S.%20O'Connor" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref70"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib71" id="ref-id-bib71" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">O'Connor et&nbsp;al., 2010</span></span></a></span><span class="reference" id="sref71"><div class="contribution"><div class="authors u-font-sans">S. O'Connor, A. Barham, M. Spriggs, P. Veth, K. Aplin, E. St Pierre</div><div id="ref-id-sref71" class="title text-m">Cave archaeology and sampling issues in the tropics: a case study from Lene Hara Cave, a 42,000 year old occupation site in East Timor, Island Southeast Asia</div></div><div class="host u-font-sans">Aust. Archaeol., 71 (2010), pp. 29-40</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1080/03122417.2010.11689382" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref71"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-78650814464&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref71"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Cave%20archaeology%20and%20sampling%20issues%20in%20the%20tropics%3A%20a%20case%20study%20from%20Lene%20Hara%20Cave%2C%20a%2042%2C000%20year%20old%20occupation%20site%20in%20East%20Timor%2C%20Island%20Southeast%20Asia&amp;publication_year=2010&amp;author=S.%20O'Connor&amp;author=A.%20Barham&amp;author=M.%20Spriggs&amp;author=P.%20Veth&amp;author=K.%20Aplin&amp;author=E.%20St%20Pierre" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref71"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib72" id="ref-id-bib72" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Olson et&nbsp;al., 2001</span></span></a></span><span class="reference" id="sref72"><div class="contribution"><div class="authors u-font-sans">D.M. Olson, E. Dinerstein, E.D. Wikramanayake, N.D. Burgess, G.V. Powell, E.C. Underwood, J.A. D'amico, I. Itoua, H.E. Strand, J.C. Morrison, C.J. Loucks</div><div id="ref-id-sref72" class="title text-m">Terrestrial Ecoregions of the World: a New Map of Life on Earth</div></div><div class="host u-font-sans">Bioscience, 51 (2001), pp. 933-938</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0035181573&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref72"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Terrestrial%20Ecoregions%20of%20the%20World%3A%20a%20New%20Map%20of%20Life%20on%20Earth&amp;publication_year=2001&amp;author=D.M.%20Olson&amp;author=E.%20Dinerstein&amp;author=E.D.%20Wikramanayake&amp;author=N.D.%20Burgess&amp;author=G.V.%20Powell&amp;author=E.C.%20Underwood&amp;author=J.A.%20D'amico&amp;author=I.%20Itoua&amp;author=H.E.%20Strand&amp;author=J.C.%20Morrison&amp;author=C.J.%20Loucks" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref72"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib73" id="ref-id-bib73" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Piganeau et&nbsp;al., 2004</span></span></a></span><span class="reference" id="sref73"><div class="contribution"><div class="authors u-font-sans">G. Piganeau, M. Gardner, A. Eyre-Walker</div><div id="ref-id-sref73" class="title text-m">A&nbsp;broad survey of recombination in animal mitochondria</div></div><div class="host u-font-sans">Mol. Biol. Evol., 21 (2004), pp. 2319-2325</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-8444235408&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref73"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0broad%20survey%20of%20recombination%20in%20animal%20mitochondria&amp;publication_year=2004&amp;author=G.%20Piganeau&amp;author=M.%20Gardner&amp;author=A.%20Eyre-Walker" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref73"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib74" id="ref-id-bib74" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Poirier and Smith, 1974</span></span></a></span><span class="reference" id="sref74"><div class="contribution"><div class="authors u-font-sans">F.E. Poirier, E.O. Smith</div><div id="ref-id-sref74" class="title text-m">The crab-eating macaques (<em>Macaca fascicularis</em>) of Angaur Island, Palau, Micronesia</div></div><div class="host u-font-sans">Folia Primatol., 22 (1974), pp. 258-306</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1159/000155631" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref74"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0016143694&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref74"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20crab-eating%20macaques%20%20of%20Angaur%20Island%2C%20Palau%2C%20Micronesia&amp;publication_year=1974&amp;author=F.E.%20Poirier&amp;author=E.O.%20Smith" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref74"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib75" id="ref-id-bib75" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Posada and Crandall, 2001</span></span></a></span><span class="reference" id="sref75"><div class="contribution"><div class="authors u-font-sans">D. Posada, K.A. Crandall</div><div id="ref-id-sref75" class="title text-m">Evaluation of methods for detecting recombination from DNA sequences: computer simulations</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 98 (2001), pp. 13757-13762</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0035923699&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref75"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20methods%20for%20detecting%20recombination%20from%20DNA%20sequences%3A%20computer%20simulations&amp;publication_year=2001&amp;author=D.%20Posada&amp;author=K.A.%20Crandall" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref75"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib76" id="ref-id-bib76" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Posada and Crandall, 2002</span></span></a></span><span class="reference" id="sref76"><div class="contribution"><div class="authors u-font-sans">D. Posada, K.A. Crandall</div><div id="ref-id-sref76" class="title text-m">The effect of recombination on the accuracy of phylogeny estimation</div></div><div class="host u-font-sans">J.&nbsp;Mol. Evol., 54 (2002), pp. 396-402</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s00239-001-0034-9" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref76"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0036183042&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref76"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20recombination%20on%20the%20accuracy%20of%20phylogeny%20estimation&amp;publication_year=2002&amp;author=D.%20Posada&amp;author=K.A.%20Crandall" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref76"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib77" id="ref-id-bib77" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Pugh and Gilbert, 2018</span></span></a></span><span class="reference" id="sref77"><div class="contribution"><div class="authors u-font-sans">K.D. Pugh, C.C. Gilbert</div><div id="ref-id-sref77" class="title text-m">Phylogenetic relationships of living and fossil African papionins: combined evidence from morphology and molecules</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 123 (2018), pp. 35-51</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248417304669/pdfft?md5=494a9a8f1d8075adb7c82b69943dfeee&amp;pid=1-s2.0-S0047248417304669-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref77"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248417304669" aria-describedby="ref-id-sref77"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85050378462&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref77"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20of%20living%20and%20fossil%20African%20papionins%3A%20combined%20evidence%20from%20morphology%20and%20molecules&amp;publication_year=2018&amp;author=K.D.%20Pugh&amp;author=C.C.%20Gilbert" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref77"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib78" id="ref-id-bib78" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Raven, 1935</span></span></a></span><span class="reference" id="sref78"><div class="contribution"><div class="authors u-font-sans">H.C. Raven</div><div id="ref-id-sref78" class="title text-m">Wallace's Line and the distribution of Indo-Australian mammals</div></div><div class="host u-font-sans">Bull. Am. Mus. Nat. Hist., 68 (1935), pp. 179-293</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Wallaces%20Line%20and%20the%20distribution%20of%20Indo-Australian%20mammals&amp;publication_year=1935&amp;author=H.C.%20Raven" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref78"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib79" id="ref-id-bib79" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Rizal et&nbsp;al., 2020</span></span></a></span><span class="reference" id="sref79"><div class="contribution"><div class="authors u-font-sans">Y. Rizal, K.E. Westaway, Y. Zaim, G.D. van den Bergh, E.A. Bettis 3rd, M.J. Morwood, O.F. Huffman, R. Grun, R. Joannes-Boyau, R.M. Bailey, Sidarto, M.C. Westaway, I. Kurniawan, M.W. Moore, M. Storey, F. Aziz, Suminto, J.X. Zhao, Aswan, M.E. Sipola, R. Larick, J.P. Zonneveld, R. Scott, S. Putt, R.L. Ciochon</div><div id="ref-id-sref79" class="title text-m">Last appearance of <em>Homo erectus</em> at Ngandong, Java, 117,000-108,000 years ago</div></div><div class="host u-font-sans">Nature, 577 (2020), pp. 381-385</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/s41586-019-1863-2" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref79"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85077068003&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref79"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Last%20appearance%20of%20Homo%20erectus%20at%20Ngandong%2C%20Java%2C%20117%2C000-108%2C000%20years%20ago&amp;publication_year=2020&amp;author=Y.%20Rizal&amp;author=K.E.%20Westaway&amp;author=Y.%20Zaim&amp;author=G.D.%20van%20den%20Bergh&amp;author=E.A.%20Bettis%203rd&amp;author=M.J.%20Morwood&amp;author=O.F.%20Huffman&amp;author=R.%20Grun&amp;author=R.%20Joannes-Boyau&amp;author=R.M.%20Bailey&amp;author=Sidarto&amp;author=M.C.%20Westaway&amp;author=I.%20Kurniawan&amp;author=M.W.%20Moore&amp;author=M.%20Storey&amp;author=F.%20Aziz&amp;author=Suminto&amp;author=J.X.%20Zhao&amp;author=Aswan&amp;author=M.E.%20Sipola&amp;author=R.%20Larick&amp;author=J.P.%20Zonneveld&amp;author=R.%20Scott&amp;author=S.%20Putt&amp;author=R.L.%20Ciochon" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref79"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib80" id="ref-id-bib80" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref80"><div class="contribution"><div class="authors u-font-sans">C. Roos, R. Boonratana, J. Supriatna, J. Fellowes, C. Groves, S. Nash, A. Rylands, R. Mittermeier</div><div id="ref-id-sref80" class="title text-m">An updated taxonomy and conservation status review of Asian primates</div></div><div class="host u-font-sans">Asian Primates J., 4 (2014), pp. 2-38</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=An%20updated%20taxonomy%20and%20conservation%20status%20review%20of%20Asian%20primates&amp;publication_year=2014&amp;author=C.%20Roos&amp;author=R.%20Boonratana&amp;author=J.%20Supriatna&amp;author=J.%20Fellowes&amp;author=C.%20Groves&amp;author=S.%20Nash&amp;author=A.%20Rylands&amp;author=R.%20Mittermeier" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref80"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib81" id="ref-id-bib81" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Roos et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref81"><div class="contribution"><div class="authors u-font-sans">C. Roos, M. Kothe, D.M. Alba, E. Delson, D. Zinner</div><div id="ref-id-sref81" class="title text-m">The radiation of macaques out of Africa: evidence from mitogenome divergence times and the fossil record</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 133 (2019), pp. 114-132</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248418302458/pdfft?md5=009ab5104cde19a97ff15c2fcd07f374&amp;pid=1-s2.0-S0047248418302458-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref81"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248418302458" aria-describedby="ref-id-sref81"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85068038631&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref81"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20radiation%20of%20macaques%20out%20of%20Africa%3A%20evidence%20from%20mitogenome%20divergence%20times%20and%20the%20fossil%20record&amp;publication_year=2019&amp;author=C.%20Roos&amp;author=M.%20Kothe&amp;author=D.M.%20Alba&amp;author=E.%20Delson&amp;author=D.%20Zinner" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref81"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib82" id="ref-id-bib82" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Rosenbaum et&nbsp;al., 1998</span></span></a></span><span class="reference" id="sref82"><div class="contribution"><div class="authors u-font-sans">B. Rosenbaum, T.G. O'Brien, M. Kinnaird, J. Supriatna</div><div id="ref-id-sref82" class="title text-m">Population densities of Sulawesi crested black macaques (<em>Macaca nigra</em>) on Bacan and Sulawesi, Indonesia: effects of habitat disturbance and hunting</div></div><div class="host u-font-sans">Am. J. Primatol., 44 (1998), pp. 89-106</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0031931011&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref82"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Population%20densities%20of%20Sulawesi%20crested%20black%20macaques%20%20on%20Bacan%20and%20Sulawesi%2C%20Indonesia%3A%20effects%20of%20habitat%20disturbance%20and%20hunting&amp;publication_year=1998&amp;author=B.%20Rosenbaum&amp;author=T.G.%20O'Brien&amp;author=M.%20Kinnaird&amp;author=J.%20Supriatna" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref82"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib83" id="ref-id-bib83" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Schierup and Hein, 2000a</span></span></a></span><span class="reference" id="sref83"><div class="contribution"><div class="authors u-font-sans">M.H. Schierup, J. Hein</div><div id="ref-id-sref83" class="title text-m">Consequences of recombination on traditional phylogenetic analysis</div></div><div class="host u-font-sans">Genetics, 156 (2000), pp. 879-891</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/genetics/156.2.879" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref83"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033793521&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref83"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Consequences%20of%20recombination%20on%20traditional%20phylogenetic%20analysis&amp;publication_year=2000&amp;author=M.H.%20Schierup&amp;author=J.%20Hein" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref83"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib84" id="ref-id-bib84" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Schierup and Hein, 2000b</span></span></a></span><span class="reference" id="sref84"><div class="contribution"><div class="authors u-font-sans">M.H. Schierup, J. Hein</div><div id="ref-id-sref84" class="title text-m">Recombination and the molecular clock</div></div><div class="host u-font-sans">Mol. Biol. Evol., 17 (2000), pp. 1578-1579</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/oxfordjournals.molbev.a026256" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref84"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033805038&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref84"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Recombination%20and%20the%20molecular%20clock&amp;publication_year=2000&amp;author=M.H.%20Schierup&amp;author=J.%20Hein" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref84"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib85" id="ref-id-bib85" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Schillaci et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref85"><div class="contribution"><div class="authors u-font-sans">M.A. Schillaci, A.R. Klegarth, W.M. Switzer, M.R. Shattuck, B.P.Y.-H. Lee, H. Hollocher</div><div id="ref-id-sref85" class="title text-m">Evolutionary relationships of <em>Macaca fascicularis</em> (Raffles 1821) (Primates: Cercopithecidae) from Singapore revealed by Bayesian analysis of mitohchondrial DNA</div></div><div class="host u-font-sans">Raffles Bull. Zool., 65 (2017), pp. 3-19</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85012011494&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref85"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20relationships%20of%20Macaca%20fascicularis%20%20%20from%20Singapore%20revealed%20by%20Bayesian%20analysis%20of%20mitohchondrial%20DNA&amp;publication_year=2017&amp;author=M.A.%20Schillaci&amp;author=A.R.%20Klegarth&amp;author=W.M.%20Switzer&amp;author=M.R.%20Shattuck&amp;author=B.P.Y.-H.%20Lee&amp;author=H.%20Hollocher" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref85"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib86" id="ref-id-bib86" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Smith et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref86"><div class="contribution"><div class="authors u-font-sans">D.G. Smith, J. Ng, D. George, J.S. Trask, P. Houghton, B. Singh, J. Villano, S. Kanthaswamy</div><div id="ref-id-sref86" class="title text-m">A&nbsp;genetic comparison of two alleged subspecies of Philippine cynomolgus macaques</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 155 (2014), pp. 136-348</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84905900564&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref86"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0genetic%20comparison%20of%20two%20alleged%20subspecies%20of%20Philippine%20cynomolgus%20macaques&amp;publication_year=2014&amp;author=D.G.%20Smith&amp;author=J.%20Ng&amp;author=D.%20George&amp;author=J.S.%20Trask&amp;author=P.%20Houghton&amp;author=B.%20Singh&amp;author=J.%20Villano&amp;author=S.%20Kanthaswamy" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref86"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib87" id="ref-id-bib87" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Spriggs, 2012</span></span></a></span><span class="reference" id="sref87"><div class="contribution"><div class="authors u-font-sans">M. Spriggs</div><div id="ref-id-sref87" class="title text-m">Is the Neolithic spread in Island Southeast Asia really as confusing as the archaeologists (and some linguists) make it seem?</div></div><div class="host u-font-sans">M.L. Tjoa-Bonatz, A. Reinecke, D. Bonatz (Eds.), Selected papers from the 13th international conference of the European Association of Southeast Asian archaeologists, NUS Press, Singapore (2012)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Is%20the%20Neolithic%20spread%20in%20Island%20Southeast%20Asia%20really%20as%20confusing%20as%20the%20archaeologists%20%20make%20it%20seem&amp;publication_year=2012&amp;author=M.%20Spriggs" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref87"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib88" id="ref-id-bib88" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Stevison and Kohn, 2009</span></span></a></span><span class="reference" id="sref88"><div class="contribution"><div class="authors u-font-sans">L.S. Stevison, M.H. Kohn</div><div id="ref-id-sref88" class="title text-m">Divergence population genetic analysis of hybridization&nbsp;between rhesus and cynomolgus macaques</div></div><div class="host u-font-sans">Mol. Ecol., 18 (2009), pp. 2457-2475</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-65949110584&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref88"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Divergence%20population%20genetic%20analysis%20of%20hybridization%C2%A0between%20rhesus%20and%20cynomolgus%20macaques&amp;publication_year=2009&amp;author=L.S.%20Stevison&amp;author=M.H.%20Kohn" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref88"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib89" id="ref-id-bib89" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Stuiver et&nbsp;al., 2020</span></span></a></span><span class="reference" id="sref89"><div class="contribution"><div class="authors u-font-sans">M. Stuiver, P.J. Reimer, R. Reimer</div><div id="ref-id-sref89" class="title text-m">CALIB 7.1</div></div><div class="host u-font-sans"> (2020)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=CALIB%207.1&amp;publication_year=2020&amp;author=M.%20Stuiver&amp;author=P.J.%20Reimer&amp;author=R.%20Reimer" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref89"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib90" id="ref-id-bib90" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Sun et&nbsp;al., 2011</span></span></a></span><span class="reference" id="sref90"><div class="contribution"><div class="authors u-font-sans">S. Sun, B.J. Evans, G.B. Golding</div><div id="ref-id-sref90" class="title text-m">'Patchy-tachy' leads to false positives for recombination</div></div><div class="host u-font-sans">Mol. Biol. Evol., 28 (2011), pp. 2549-2559</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/molbev/msr076" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref90"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-80052155692&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref90"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Patchy-tachy%20leads%20to%20false%20positives%20for%20recombination&amp;publication_year=2011&amp;author=S.%20Sun&amp;author=B.J.%20Evans&amp;author=G.B.%20Golding" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref90"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib91" id="ref-id-bib91" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2018</span></span></a></span><span class="reference" id="sref91"><div class="contribution"><div class="authors u-font-sans">T. Sutikna, M.W. Tocheri, J.T. Faith, Jatmiko, R.D. Awe, H.J.M. Meijer, E.W. Saptomo, R.G. Roberts</div><div id="ref-id-sref91" class="title text-m">The spatio-temporal distribution of archaeological and faunal finds at Liang Bua (Flores, Indonesia) in light of the revised chronology for <em>Homo floresiensis</em></div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 124 (2018), pp. 52-74</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248417303160/pdfft?md5=17002f3bc187f44c2d5ff8de02831edd&amp;pid=1-s2.0-S0047248417303160-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref91"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248417303160" aria-describedby="ref-id-sref91"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85054010564&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref91"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20spatio-temporal%20distribution%20of%20archaeological%20and%20faunal%20finds%20at%20Liang%20Bua%20%20in%20light%20of%20the%20revised%20chronology%20for%20Homo%20floresiensis&amp;publication_year=2018&amp;author=T.%20Sutikna&amp;author=M.W.%20Tocheri&amp;author=J.T.%20Faith&amp;author=Jatmiko&amp;author=R.D.%20Awe&amp;author=H.J.M.%20Meijer&amp;author=E.W.%20Saptomo&amp;author=R.G.%20Roberts" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref91"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib92" id="ref-id-bib92" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Sutikna et&nbsp;al., 2016</span></span></a></span><span class="reference" id="sref92"><div class="contribution"><div class="authors u-font-sans">T. Sutikna, M.W. Tocheri, M.J. Morwood, E.W. Saptomo, Jatmiko, R.D. Awe, S. Wasisto, K.E. Westaway, M. Aubert, B. Li, J.X. Zhao, M. Storey, B.V. Alloway, M.W. Morley, H.J.M. Meijer, G.D. van den Bergh, R. Grun, A. Dosseto, A. Brumm, W.L. Jungers, R.G. Roberts</div><div id="ref-id-sref92" class="title text-m">Revised stratigraphy and chronology for <em>Homo floresiensis</em> at Liang Bua in Indonesia</div></div><div class="host u-font-sans">Nature, 532 (2016), pp. 366-369</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature17179" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref92"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84964426590&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref92"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Revised%20stratigraphy%20and%20chronology%20for%20Homo%20floresiensis%20at%20Liang%20Bua%20in%20Indonesia&amp;publication_year=2016&amp;author=T.%20Sutikna&amp;author=M.W.%20Tocheri&amp;author=M.J.%20Morwood&amp;author=E.W.%20Saptomo&amp;author=Jatmiko&amp;author=R.D.%20Awe&amp;author=S.%20Wasisto&amp;author=K.E.%20Westaway&amp;author=M.%20Aubert&amp;author=B.%20Li&amp;author=J.X.%20Zhao&amp;author=M.%20Storey&amp;author=B.V.%20Alloway&amp;author=M.W.%20Morley&amp;author=H.J.M.%20Meijer&amp;author=G.D.%20van%20den%20Bergh&amp;author=R.%20Grun&amp;author=A.%20Dosseto&amp;author=A.%20Brumm&amp;author=W.L.%20Jungers&amp;author=R.G.%20Roberts" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref92"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib93" id="ref-id-bib93" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Tosi and Coke, 2007</span></span></a></span><span class="reference" id="sref93"><div class="contribution"><div class="authors u-font-sans">A.J. Tosi, C.S. Coke</div><div id="ref-id-sref93" class="title text-m">Comparative phylogenetics offer new insights into the biogeographic history of <em>Macaca fascicularis</em> and the origin of the Mauritian macaques</div></div><div class="host u-font-sans">Mol. Phylogenet. Evol., 42 (2007), pp. 498-504</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S1055790306003095/pdfft?md5=28aa68c3740cc67cb5f8240c10f62d06&amp;pid=1-s2.0-S1055790306003095-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref93"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S1055790306003095" aria-describedby="ref-id-sref93"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33751014882&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref93"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Comparative%20phylogenetics%20offer%20new%20insights%20into%20the%20biogeographic%20history%20of%20Macaca%20fascicularis%20and%20the%20origin%20of%20the%20Mauritian%20macaques&amp;publication_year=2007&amp;author=A.J.%20Tosi&amp;author=C.S.%20Coke" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref93"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib94" id="ref-id-bib94" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2002</span></span></a></span><span class="reference" id="sref94"><div class="contribution"><div class="authors u-font-sans">A.J. Tosi, J.C. Morales, D.J. Melnick</div><div id="ref-id-sref94" class="title text-m">Y-chromosome and mitochondrial markers in <em>Macaca fascicularis</em> indicate introgression with Indochinese <em>M.&nbsp;mulatta</em> and a biogeographic barrier in the Isthmus of Kra</div></div><div class="host u-font-sans">Int. J. Primatol., 23 (2002), pp. 161-178</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0036145725&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref94"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Y-chromosome%20and%20mitochondrial%20markers%20in%20Macaca%20fascicularis%20indicate%20introgression%20with%20Indochinese%20M.%C2%A0mulatta%20and%20a%20biogeographic%20barrier%20in%20the%20Isthmus%20of%20Kra&amp;publication_year=2002&amp;author=A.J.%20Tosi&amp;author=J.C.%20Morales&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref94"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib95" id="ref-id-bib95" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2003</span></span></a></span><span class="reference" id="sref95"><div class="contribution"><div class="authors u-font-sans">A.J. Tosi, J.C. Morales, D.J. Melnick</div><div id="ref-id-sref95" class="title text-m">Paternal, maternal, and biparental molecular markers provide unique windows onto the evolutionary history of macaque monkeys</div></div><div class="host u-font-sans">Evolution, 57 (2003), pp. 1419-1435</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0038641049&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref95"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Paternal%2C%20maternal%2C%20and%20biparental%20molecular%20markers%20provide%20unique%20windows%20onto%20the%20evolutionary%20history%20of%20macaque%20monkeys&amp;publication_year=2003&amp;author=A.J.%20Tosi&amp;author=J.C.%20Morales&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref95"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib96" id="ref-id-bib96" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Tsaousis et&nbsp;al., 2005</span></span></a></span><span class="reference" id="sref96"><div class="contribution"><div class="authors u-font-sans">A.D. Tsaousis, D.P. Martin, D. Ladoukakis, D. Posada, E. Zouros</div><div id="ref-id-sref96" class="title text-m">Widespread recombination in published animal mtDNA sequences</div></div><div class="host u-font-sans">Mol. Biol. Evol., 22 (2005), pp. 925-933</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/molbev/msi084" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref96"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-16344387680&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref96"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Widespread%20recombination%20in%20published%20animal%20mtDNA%20sequences&amp;publication_year=2005&amp;author=A.D.%20Tsaousis&amp;author=D.P.%20Martin&amp;author=D.%20Ladoukakis&amp;author=D.%20Posada&amp;author=E.%20Zouros" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref96"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib97" id="ref-id-bib97" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Turvey et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref97"><div class="contribution"><div class="authors u-font-sans">S.T. Turvey, J.J. Crees, J. Hansford, T.E. Jeffree, N. Crumpton, I. Kurniawan, E. Setiyabudi, T. Guillerme, U. Paranggarimu, A. Dosseto, G.D. van den Bergh</div><div id="ref-id-sref97" class="title text-m">Quaternary vertebrate faunas from Sumba, Indonesia: implications for Wallacean biogeography and evolution</div></div><div class="host u-font-sans">Proc. Roy. Soc. Lond. B, 284 (2017)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Quaternary%20vertebrate%20faunas%20from%20Sumba%2C%20Indonesia%3A%20implications%20for%20Wallacean%20biogeography%20and%20evolution&amp;publication_year=2017&amp;author=S.T.%20Turvey&amp;author=J.J.%20Crees&amp;author=J.%20Hansford&amp;author=T.E.%20Jeffree&amp;author=N.%20Crumpton&amp;author=I.%20Kurniawan&amp;author=E.%20Setiyabudi&amp;author=T.%20Guillerme&amp;author=U.%20Paranggarimu&amp;author=A.%20Dosseto&amp;author=G.D.%20van%20den%20Bergh" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref97"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib98" id="ref-id-bib98" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ujvari et&nbsp;al., 2007</span></span></a></span><span class="reference" id="sref98"><div class="contribution"><div class="authors u-font-sans">B. Ujvari, M. Dowton, T. Madsen</div><div id="ref-id-sref98" class="title text-m">Mitochondrial DNA recombination in a free-ranging Australian lizard</div></div><div class="host u-font-sans">Biol. Lett., 3 (2007), pp. 189-192</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rsbl.2006.0587" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref98"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-35248880386&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref98"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20recombination%20in%20a%20free-ranging%20Australian%20lizard&amp;publication_year=2007&amp;author=B.%20Ujvari&amp;author=M.%20Dowton&amp;author=T.%20Madsen" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref98"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib99" id="ref-id-bib99" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">van den Bergh et&nbsp;al., 2016a</span></span></a></span><span class="reference" id="sref99"><div class="contribution"><div class="authors u-font-sans">G.D. van den Bergh, Y. Kaifu, I. Kurniawan, R.T. Kono, A. Brumm, E. Setiyabudi, F. Aziz, M.J. Morwood</div><div id="ref-id-sref99" class="title text-m"><em>Homo floresiensis</em>-like fossils from the early Middle Pleistocene of Flores</div></div><div class="host u-font-sans">Nature, 534 (2016), pp. 245-248</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature17999" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref99"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84973643880&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref99"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Homo%20floresiensis-like%20fossils%20from%20the%20early%20Middle%20Pleistocene%20of%20Flores&amp;publication_year=2016&amp;author=G.D.%20van%20den%20Bergh&amp;author=Y.%20Kaifu&amp;author=I.%20Kurniawan&amp;author=R.T.%20Kono&amp;author=A.%20Brumm&amp;author=E.%20Setiyabudi&amp;author=F.%20Aziz&amp;author=M.J.%20Morwood" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref99"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib100" id="ref-id-bib100" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">van den Bergh et&nbsp;al., 2016b</span></span></a></span><span class="reference" id="sref100"><div class="contribution"><div class="authors u-font-sans">G.D. van den Bergh, B. Li, A. Brumm, R. Grun, D. Yurnaldi, M.W. Moore, I. Kurniawan, R. Setiawan, F. Aziz, R.G. Roberts, Suyono, M. Storey, E. Setiabudi, M.J. Morwood</div><div id="ref-id-sref100" class="title text-m">Earliest hominin occupation of Sulawesi, Indonesia</div></div><div class="host u-font-sans">Nature, 529 (2016), pp. 208-211</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature16448" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref100"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84954501860&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref100"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Earliest%20hominin%20occupation%20of%20Sulawesi%2C%20Indonesia&amp;publication_year=2016&amp;author=G.D.%20van%20den%20Bergh&amp;author=B.%20Li&amp;author=A.%20Brumm&amp;author=R.%20Grun&amp;author=D.%20Yurnaldi&amp;author=M.W.%20Moore&amp;author=I.%20Kurniawan&amp;author=R.%20Setiawan&amp;author=F.%20Aziz&amp;author=R.G.%20Roberts&amp;author=Suyono&amp;author=M.%20Storey&amp;author=E.%20Setiabudi&amp;author=M.J.%20Morwood" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref100"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib101" id="ref-id-bib101" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">van Schaik et&nbsp;al., 1996</span></span></a></span><span class="reference" id="sref101"><div class="contribution"><div class="authors u-font-sans">C.P. van Schaik, A. van Amerongen, M.A. van Noordwijk</div><div id="ref-id-sref101" class="title text-m">Riverine refuging by wild Sumatran long-tailed macaques (<em>Macaca fascicularis</em>)</div></div><div class="host u-font-sans">J.E. Fa, D.G. Lindburg (Eds.), Evolution and ecology of macaque societies, Cambridge University Press, Cambridge (1996), pp. 160-181</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0030469126&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref101"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Riverine%20refuging%20by%20wild%20Sumatran%20long-tailed%20macaques%20&amp;publication_year=1996&amp;author=C.P.%20van%20Schaik&amp;author=A.%20van%20Amerongen&amp;author=M.A.%20van%20Noordwijk" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref101"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib102" id="ref-id-bib102" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Wallace, 1863</span></span></a></span><span class="reference" id="sref102"><div class="contribution"><div class="authors u-font-sans">A.R. Wallace</div><div id="ref-id-sref102" class="title text-m">On the physical geography of the Malay Archipelago</div></div><div class="host u-font-sans">J.&nbsp;R. Geo. Soc., 1863 (1863), pp. 217-234</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.2307/1798448" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref102"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=On%20the%20physical%20geography%20of%20the%20Malay%20Archipelago&amp;publication_year=1863&amp;author=A.R.%20Wallace" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref102"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib103" id="ref-id-bib103" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Wallace, 1869</span></span></a></span><span class="reference" id="sref103"><div class="contribution"><div class="authors u-font-sans">A.R. Wallace</div><div id="ref-id-sref103" class="title text-m">The Malay Archipelago; The land of the orangutan and the bird of paradise</div></div><div class="host u-font-sans">Macmillan Publishers, New York (1869)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20Malay%20Archipelago%3B%20The%20land%20of%20the%20orangutan%20and%20the%20bird%20of%20paradise&amp;publication_year=1869&amp;author=A.R.%20Wallace" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref103"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib104" id="ref-id-bib104" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Werdelin, 2010</span></span></a></span><span class="reference" id="sref104"><div class="contribution"><div class="authors u-font-sans">L. Werdelin</div><div id="ref-id-sref104" class="title text-m">Chronology of Neogene mammal localities</div></div><div class="host u-font-sans">L. Werdelin, W.J. Sanders (Eds.), Cenozoic mammals of Africa, University of California Press, Berkeley, Los Angeles &amp; London (2010), pp. 27-43</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Chronology%20of%20Neogene%20mammal%20localities&amp;publication_year=2010&amp;author=L.%20Werdelin" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref104"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib105" id="ref-id-bib105" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Westaway et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref105"><div class="contribution"><div class="authors u-font-sans">K.E. Westaway, J. Louys, R.D. Awe, M.J. Morwood, G.J. Price, J.X. Zhao, M. Aubert, R. Joannes-Boyau, T.M. Smith, M.M. Skinner, T. Compton, R.M. Bailey, G.D. van den Bergh, J. de Vos, A.W.G. Pike, C. Stringer, E.W. Saptomo, Y. Rizal, J. Zaim, W.D. Santoso, A. Trihascaryo, L. Kinsley, B. Sulistyanto</div><div id="ref-id-sref105" class="title text-m">An early modern human presence in Sumatra 73,000â€“63,000 years ago</div></div><div class="host u-font-sans">Nature, 548 (2017), pp. 322-325</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature23452" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref105"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85027852151&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref105"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=An%20early%20modern%20human%20presence%20in%20Sumatra%2073%2C00063%2C000%20years%20ago&amp;publication_year=2017&amp;author=K.E.%20Westaway&amp;author=J.%20Louys&amp;author=R.D.%20Awe&amp;author=M.J.%20Morwood&amp;author=G.J.%20Price&amp;author=J.X.%20Zhao&amp;author=M.%20Aubert&amp;author=R.%20Joannes-Boyau&amp;author=T.M.%20Smith&amp;author=M.M.%20Skinner&amp;author=T.%20Compton&amp;author=R.M.%20Bailey&amp;author=G.D.%20van%20den%20Bergh&amp;author=J.%20de%20Vos&amp;author=A.W.G.%20Pike&amp;author=C.%20Stringer&amp;author=E.W.%20Saptomo&amp;author=Y.%20Rizal&amp;author=J.%20Zaim&amp;author=W.D.%20Santoso&amp;author=A.%20Trihascaryo&amp;author=L.%20Kinsley&amp;author=B.%20Sulistyanto" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref105"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib106" id="ref-id-bib106" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">White et&nbsp;al., 2013</span></span></a></span><span class="reference" id="sref106"><div class="contribution"><div class="authors u-font-sans">D.J. White, D. Bryant, N.J. Gemmell</div><div id="ref-id-sref106" class="title text-m">How good are indirect tests at detecting recombination in human mtDNA?</div></div><div class="host u-font-sans">G3 (Bethesda), 3 (2013), pp. 1095-1104</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1534/g3.113.006510" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref106"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84883225901&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref106"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=How%20good%20are%20indirect%20tests%20at%20detecting%20recombination%20in%20human%20mtDNA&amp;publication_year=2013&amp;author=D.J.%20White&amp;author=D.%20Bryant&amp;author=N.J.%20Gemmell" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref106"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib107" id="ref-id-bib107" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Xu et&nbsp;al., 2012</span></span></a></span><span class="reference" id="sref107"><div class="contribution"><div class="authors u-font-sans">S. Xu, I. Pugach, M. Stoneking, M. Kayser, L. Jin, Consortium, H.P.-A.S.</div><div id="ref-id-sref107" class="title text-m">Genetic dating indicates that the Asian-Papuan admixture through Eastern Indonesia corresponds to the Austronesian expansion</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 109 (2012), pp. 4574-4579</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1118892109" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref107"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84858690407&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref107"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Genetic%20dating%20indicates%20that%20the%20Asian-Papuan%20admixture%20through%20Eastern%20Indonesia%20corresponds%20to%20the%20Austronesian%20expansion&amp;publication_year=2012&amp;author=S.%20Xu&amp;author=I.%20Pugach&amp;author=M.%20Stoneking&amp;author=M.%20Kayser&amp;author=L.%20Jin&amp;author=Consortium%2C%20H.P.-A.S." target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref107"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib108" id="ref-id-bib108" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Yao et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref108"><div class="contribution"><div class="authors u-font-sans">L. Yao, H. Li, R.D. Martin, C.S. Moreau, R.S. Malhi</div><div id="ref-id-sref108" class="title text-m">Tracing the phylogeographic history of Southeast Asian long-tailed macaques through mitogenomes of museum specimens</div></div><div class="host u-font-sans">Mol. Phylogenet. Evol., 116 (2017), pp. 227-238</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S1055790316303621/pdfft?md5=e33cc96ca2f752b360a8dc3974fd4bc0&amp;pid=1-s2.0-S1055790316303621-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref108"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S1055790316303621" aria-describedby="ref-id-sref108"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85029582424&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref108"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Tracing%20the%20phylogeographic%20history%20of%20Southeast%20Asian%20long-tailed%20macaques%20through%20mitogenomes%20of%20museum%20specimens&amp;publication_year=2017&amp;author=L.%20Yao&amp;author=H.%20Li&amp;author=R.D.%20Martin&amp;author=C.S.%20Moreau&amp;author=R.S.%20Malhi" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref108"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li></ol></section></section></div><div id="section-cited-by"><section aria-label="Cited by" class="ListArticles preview"><div class="PageDivider"></div><header id="citing-articles-header"><h2 class="u-h4 u-margin-l-ver u-font-serif">Cited by (23)</h2></header><div aria-describedby="citing-articles-header"><div class="citing-articles u-margin-l-bottom"><ul><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-0-title"><a class="anchor anchor-primary" href="https://doi.org/10.1017/S0031182021000639" target="_blank"><span class="anchor-text-container"><span class="anchor-text">A new subspecies of Trypanosoma cyclops found in the Australian terrestrial leech Chtonobdella bilineata</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2021, Parasitology</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-1-title"><a class="anchor anchor-primary" href="https://doi.org/10.1093/zoolinnean/zlac018" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Diversity and systematics of Trichomycterus Valenciennes 1832 (Siluriformes: Trichomycteridae) in the Rio Doce Basin: iterating DNA, phylogeny and classical taxonomy</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2023, Zoological Journal of the Linnean Society</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-2-title"><a class="anchor anchor-primary" href="https://doi.org/10.3390/ani13010048" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Pigs as Pets: Early Human Relations with the Sulawesi Warty Pig (Sus celebensis)</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2023, Animals</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-3-title"><a class="anchor anchor-primary" href="https://doi.org/10.1093/sysbio/syac044" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Wallacean and Melanesian Islands Promote Higher Rates of Diversification within the Global Passerine Radiation Corvides</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2022, Systematic Biology</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-4-title"><a class="anchor anchor-primary" href="https://doi.org/10.1007/s00335-021-09922-z" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Genomic resources for rhesus macaques (Macaca mulatta)</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2022, Mammalian Genome</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-5-title"><a class="anchor anchor-primary" href="https://doi.org/10.24272/J.ISSN.2095-8137.2020.279" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Multilocus phylogeny suggests a distinct species status for the Nepal population of Assam macaques (Macaca assamensis): Implications for evolution and conservation</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2021, Zoological Research</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li></ul><a class="button-alternative button-alternative-secondary large-alternative button-alternative-icon-left" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-85089543743&amp;md5=509696b592a3ef6f7c328496d379f" target="_blank" id="citing-articles-view-all-btn"><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-right"><path d="M1 99l38-38L1 23l7-7 45 45-45 45z"></path></svg><span class="button-alternative-text-container"><span class="button-alternative-text">View all citing articles on Scopus</span></span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></a></div></div></section></div><div class="Copyright"><span class="copyright-line">Â© 2020 The Author(s). Published by Elsevier Ltd.</span></div></article><div class="u-display-block-from-md col-lg-6 col-md-8 pad-right u-padding-s-top"><aside class="RelatedContent u-clr-grey8" aria-label="Related content"><section class="RelatedContentPanel u-margin-s-bottom"><header id="recommended-articles-header" class="related-content-panel-header u-margin-s-bottom"><button class="button-link button-link-secondary related-content-panel-toggle is-up button-link-icon-right button-link-has-colored-icon" type="button" aria-expanded="true" data-aa-button="sd:product:journal:article:location=recommended-articles:type=close"><span class="button-link-text-container"><span class="button-link-text"><h2 class="section-title u-h4"><span class="related-content-panel-title-text">Recommended articles</span></h2></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles" class="text-xs"><ul><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article0-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S0277379116305510" title="Archaeological implications of a widespread 13th Century tephra marker across the central Indonesian Archipelago"><span class="anchor-text-container"><span class="anchor-text"><span>Archaeological implications of a widespread 13th Century tephra marker across the central Indonesian Archipelago</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Quaternary Science Reviews, Volume 155, 2017, pp. 86-99</div></div><div class="authors"><span>Brent V.</span> <span>Alloway</span>, â€¦, <span>Quan</span> <span>Hua</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article1-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S0965206X18300238" title="Editorial"><span class="anchor-text-container"><span class="anchor-text"><span>Editorial</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Journal of Tissue Viability, Volume 27, Issue 1, 2018, p. 1</div></div><div class="authors"><span>Amit</span> <span>Gefen</span>, <span>Dan</span> <span>Bader</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article2-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S104061821600015X" title="Thai-Malay Peninsula and South China Sea networks (500 BCâ€“AD 200), based on a reappraisal of â€œSa Huynh-Kalanayâ€-related ceramics"><span class="anchor-text-container"><span class="anchor-text"><span>Thai-Malay Peninsula and South China Sea networks (500 BCâ€“AD 200), based on a reappraisal of â€œSa Huynh-Kalanayâ€-related ceramics</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Quaternary International, Volume 416, 2016, pp. 219-227</div></div><div class="authors"><span>Aude</span> <span>Favereau</span>, <span>BÃ©rÃ©nice</span> <span>Bellina</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article3-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S105579031730489X" title="Ancient hybridization and admixture in macaques (genus Macaca) inferred from whole genome sequences"><span class="anchor-text-container"><span class="anchor-text"><span>Ancient hybridization and admixture in macaques (genus <em>Macaca</em>) inferred from whole genome sequences</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Molecular Phylogenetics and Evolution, Volume 127, 2018, pp. 376-386</div></div><div class="authors"><span>Zhenxin</span> <span>Fan</span>, â€¦, <span>Jing</span> <span>Li</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article4-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S1040618215011696" title="New insights on the late Pleistoceneâ€“Holocene lithic industry in East Kalimantan (Borneo): The contribution of three rock shelter sites in the karstic area of the Mangkalihat peninsula"><span class="anchor-text-container"><span class="anchor-text"><span>New insights on the late Pleistoceneâ€“Holocene lithic industry in East Kalimantan (Borneo): The contribution of three rock shelter sites in the karstic area of the Mangkalihat peninsula</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Quaternary International, Volume 416, 2016, pp. 126-150</div></div><div class="authors"><span>Michel</span> <span>Grenet</span>, â€¦, <span>FranÃ§ois-Xavier</span> <span>Ricaut</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article5-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S1055790316302731" title="Phylogenetic position of the white-cheeked macaque (Macaca leucogenys), a newly described primate from southeastern Tibet"><span class="anchor-text-container"><span class="anchor-text"><span>Phylogenetic position of the white-cheeked macaque (<em>Macaca leucogenys</em>), a newly described primate from southeastern Tibet</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Molecular Phylogenetics and Evolution, Volume 107, 2017, pp. 80-89</div></div><div class="authors"><span>Pengfei</span> <span>Fan</span>, â€¦, <span>Ming</span> <span>Li</span></div></div><div class="buttons"></div></li></ul></div><button class="button-link more-recommendations-button u-margin-s-bottom button-link-primary button-link-icon-right" type="button"><span class="button-link-text-container"><span class="button-link-text">Show 3 more articles</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></section><section class="RelatedContentPanel u-margin-s-bottom"><header id="metrics-header" class="related-content-panel-header u-margin-s-bottom"><button class="button-link button-link-secondary related-content-panel-toggle is-up button-link-icon-right button-link-has-colored-icon" type="button" aria-expanded="true"><span class="button-link-text-container"><span class="button-link-text"><h2 class="section-title u-h4"><span class="related-content-panel-title-text">Article Metrics</span></h2></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><div class="plumX-metrics"><h3 class="text-s metric-title metric-title-citations">Citations</h3><ul><li class="text-xs metrics"><span>Citation Indexes</span><span>23</span></li></ul><h3 class="text-s metric-title metric-title-captures">Captures</h3><ul><li class="text-xs metrics"><span>Mendeley Readers</span><span>56</span></li></ul><h3 class="text-s metric-title metric-title-mentions">Mentions</h3><ul><li class="text-xs metrics"><span>News Mentions</span><span>1</span></li><li class="text-xs metrics"><span>References</span><span>7</span></li></ul><h3 class="text-s metric-title metric-title-social-media">Social Media</h3><ul><li class="text-xs metrics"><span>Shares, Likes &amp; Comments</span><span>3</span></li></ul><div class="metrics u-margin-m-top u-margin-s-bottom"><img src="https://cdn.plu.mx/3ba727faf225e19d2c759f6ebffc511d/plumx-logo.png" class="plumX-logo" alt="PlumX Metrics Logo"><a class="anchor text-xs anchor-primary" href="https://plu.mx/plum/a/?doi=10.1016%2Fj.jhevol.2020.102852&amp;theme=plum-sciencedirect-theme&amp;hideUsage=true" target="_blank"><span class="anchor-text-container"><span class="anchor-text">View details</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></div></div></section></aside></div></div></div></div><footer role="contentinfo" class="els-footer u-bg-white text-xs u-padding-s-hor u-padding-m-hor-from-sm u-padding-l-hor-from-md u-padding-l-ver u-margin-l-top u-margin-xl-top-from-sm u-margin-l-top-from-md"><div class="els-footer-elsevier u-margin-m-bottom u-margin-0-bottom-from-md u-margin-s-right u-margin-m-right-from-md u-margin-l-right-from-lg"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.elsevier.com/" target="_blank" aria-label="Elsevier home page (opens in a new tab)"><img class="footer-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/47/images/elsevier-non-solus-new-with-wordmark.svg" alt="Elsevier logo with wordmark" height="64" width="58" loading="lazy"></a></div><div class="els-footer-content"><div class="u-remove-if-print"><ul class="els-footer-links u-margin-xs-bottom" style="list-style:none"><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/solutions/sciencedirect" target="_blank" id="els-footer-about-science-direct" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">About ScienceDirect</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0047248420301135" id="els-footer-remote-access" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Remote access</span></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsmediakits.com" target="_blank" id="els-footer-advertise" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Advertise</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" target="_blank" id="els-footer-contact-support" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Contact and support</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" target="_blank" id="els-footer-terms-condition" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Terms and conditions</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/legal/privacy-policy" target="_blank" id="els-footer-privacy-policy" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Privacy policy</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li></ul></div><p id="els-footer-cookie-message" class="u-remove-if-print">Cookies are used by this site. <!-- --> <button class="button-link ot-sdk-show-settings cookie-btn button-link-primary button-link-small" type="button" id="ot-sdk-btn">Cookie Settings</button></p><p id="els-footer-copyright">All content on this site: Copyright Â© <!-- -->2025<!-- --> Elsevier B.V., its licensors, and contributors. All rights are reserved, including those for text and data mining, AI training, and similar technologies. For all open access content, the relevant licensing terms apply.</p></div><div class="els-footer-relx u-margin-0-top u-margin-m-top-from-xs u-margin-0-top-from-md"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.relx.com/" target="_blank" aria-label="RELX home page (opens in a new tab)" id="els-footer-relx"><img class="" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/60/images/logo-relx-tm.svg" alt="RELX group home page" height="20" width="93" loading="lazy"></a></div></footer></div></div></div></div>
      <div id="floating-ui-node" class="floating-ui-node" data-sd-ui-floating-ui="true"></div>
      <iframe style="display: none" src="//acw.clinicalkey.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y" tabindex="-1"></iframe><iframe style="display: none" src="//acw.scopus.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y" tabindex="-1"></iframe><iframe style="display: none" src="//acw.sciencedirect.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y" tabindex="-1"></iframe><iframe style="display: none" src="//acw.elsevier.com/SSOCore/update?acw=4e09e7b565baa04895599b47c573263b482fgxrqa%7C%24%7C82F48B7AC2A967352A30A80E6946FAD28108954C484FB7A342B3B4E19D55700D02AC7FDC2E8213F95EEB9DD526858074C1D32131EAEE4E613FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9346-3b1d31c5791885488d7-d5c7266cd5731631-71y" tabindex="-1"></iframe>
      
      
<script type="text/javascript">
    window.pageData = {"content":[{"contentType":"JL","format":"MIME-XHTML","id":"sd:article:pii:S0047248420301135","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"0047-2484","volumeNumber":"146","suppl":"C","provider":"elsevier","entitlementType":"openaccess"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productAppVersion":"full-direct","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1749596608044,"loadTime":"","testId":"sd:article-page:current-ui"},"visitor":{"accessType":"ae:ANON_GUEST","accountId":"ae:228598","accountName":"ae:ScienceDirect Guests","loginStatus":"anonymous","userId":"ae:12975512","ipAddress":"189.120.74.179","appSessionId":"85f31e29-64eb-4a59-85f4-d8a059f32dac"}};
    window.pageData.page.loadTime = performance ? Math.round(performance.now()).toString() : '';

    try {
      appData.push({
      event: 'pageLoad',
      page: pageData.page,
      visitor: pageData.visitor,
      content: pageData.content
      })
    } catch(e) {
        console.warn("There was an error loading or running Adobe DTM: ", e);
    }
</script>
      <script nomodule="" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/73/js/core-js/3.20.2/core-js.es.minified.js" type="text/javascript"></script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/108/js/react/18.3.1/react.production.min.js" type="text/javascript"></script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/108/js/react-dom/18.3.1/react-dom.production.min.js" type="text/javascript"></script>
      <script async="" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/arp.js" type="text/javascript"></script>
      <script type="text/javascript">
    const pendoData = {"visitor":{"pageName":"SD:product:journal:article","pageType":"CP-CA","pageProduct":"SD","pageLanguage":"en","pageEnvironment":"prod","accessType":"ae:ANON_GUEST","countryCode":"BR"},"account":{"id":"ae:228598","name":"ae:ScienceDirect Guests"},"events":{}};;
    pendoData.events = {
      ready: function () {
        pendo.addAltText();
      },
    };
    function runPendo(data, options) {
  const {
    firstDelay,
    maxRetries,
    urlPrefix,
    urlSuffix,
    apiKey
  } = options;
  (function (apiKey) {
    (function (p, e, n, d, o) {
      var v, w, x, y, z;
      o = p[d] = p[d] || {};
      o._q = [];
      v = ['initialize', 'identify', 'updateOptions', 'pageLoad'];
      for (w = 0, x = v.length; w < x; ++w) (function (m) {
        o[m] = o[m] || function () {
          o._q[m === v[0] ? 'unshift' : 'push']([m].concat([].slice.call(arguments, 0)));
        };
      })(v[w]);
      y = e.createElement(n);
      y.async = !0;
      y.src = urlPrefix + apiKey + urlSuffix;
      z = e.getElementsByTagName(n)[0];
      z.parentNode.insertBefore(y, z);
    })(window, document, 'script', 'pendo');
    pendo.addAltText = function () {
      var target = document.querySelector('body');
      var observer = new MutationObserver(function (mutations) {
        mutations.forEach(function (mutation) {
          if (mutation?.addedNodes?.length) {
            if (mutation.addedNodes[0]?.className?.includes("_pendo-badge")) {
              const badge = mutation.addedNodes[0];
              const altText = badge?.attributes['aria-label']?.value || 'Feedback';
              const pendoBadgeImage = pendo.dom(`#${badge?.attributes?.id.value} img`);
              if (pendoBadgeImage.length) {
                pendoBadgeImage[0]?.setAttribute('alt', altText);
              }
            }
          }
        });
      });
      var config = {
        attributeFilter: ['data-layout'],
        attributes: true,
        childList: true,
        characterData: true,
        subtree: false
      };
      observer.observe(target, config);
    };
  })(apiKey);
  (function watchAndSetPendo(nextDelay, retryAttempt) {
    if (data.visitor.id) {
      console.debug(`initializing pendo`);
      pendo.initialize(data);
    } else if (typeof pageDataTracker === 'object' && typeof pageDataTracker.getVisitorId === 'function' && pageDataTracker.getVisitorId()) {
      data.visitor.id = pageDataTracker.getVisitorId();
      console.debug(`initializing pendo`);
      pendo.initialize(data);
    } else {
      if (retryAttempt > 0) {
        return setTimeout(function () {
          watchAndSetPendo(nextDelay * 2, retryAttempt - 1);
        }, nextDelay);
      }
      pendo.initialize(data);
      console.debug(`gave up ... pendo initialized`);
    }
  })(firstDelay, maxRetries);
}
    window.addPendoScript = () => {
      runPendo(pendoData, {
        firstDelay: 100,
        maxRetries: 5,
        urlPrefix: 'https://cdn.pendo.io/agent/static/',
        urlSuffix: '/pendo.js',
        apiKey: 'd6c1d995-bc7e-4e53-77f1-2ea4ecbb9565',
      });
      window.addPendoScript = () => {};
    };
    addEventListener('load', () => window.addPendoScript());
  </script>
      <span id="pendo-answer-rating"></span>
      <script type="text/x-mathjax-config;executed=true">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=MML_SVG" type="text/javascript"></script>
      <script async="" src="https://www.googletagservices.com/tag/js/gpt.js" type="text/javascript"></script>
      <script async="" src="https://scholar.google.com/scholar_js/casa.js" type="text/javascript"></script>
      <script data-cfasync="false">
      (function initOneTrust()  {
        const monitor = {
  init: () => {},
  loaded: () => {},
};
        function enableGroup(group) {
  document.querySelectorAll(`script[type*="ot-${group}"]`).forEach(script => {
    script.type = 'text/javascript';
    document.head.appendChild(script);
  });
}
        function runOneTrustCookies(doClear, monitor) {
  const oneTrustConsentSdkId = 'onetrust-consent-sdk';
  const emptyNodeSelectors = 'h3.ot-host-name, h4.ot-host-desc, button.ot-host-box';
  const ariaLabelledByButtonNodes = 'div.ot-accordion-layout > button';
  const ariaAttribute = 'aria-labelledby';
  function adjustOneTrustDOM() {
    const oneTrustRoot = document.getElementById('onetrust-consent-sdk');

    /* remove empty nodes */
    [...(oneTrustRoot?.querySelectorAll(emptyNodeSelectors) ?? [])].filter(e => e.textContent === '').forEach(e => e.remove());

    /* remove invalid aria-labelledby values */
    oneTrustRoot?.querySelectorAll(ariaLabelledByButtonNodes).forEach(e => {
      const presentIdValue = e.getAttribute(ariaAttribute)?.split(' ').filter(label => document.getElementById(label)).join(' ');
      if (presentIdValue) {
        e.setAttribute(ariaAttribute, presentIdValue);
      }
    });
  }
  function observeOneTrustLoaded(shouldSetOTDefaults, isConsentPresent) {
    const cb = (mutationList, observer) => {
      const oneTrustRoot = mutationList.filter(mutationRecord => mutationRecord.type === 'childList' && mutationRecord.addedNodes.length).map(mutationRecord => [...mutationRecord.addedNodes]).flat().find(e => e.id === oneTrustConsentSdkId);
      if (oneTrustRoot && typeof OneTrust !== 'undefined') {
        monitor.loaded(true);
        OneTrust.OnConsentChanged(() => {
          const perfAllowed = decodeURIComponent(document.cookie.match('(^| )OptanonConsent=([^;]+)')?.[2])?.match('groups=([0-9:0|1,?]+)&?')?.[1]?.match('2:([0|1])')[1] === '1';
          if (perfAllowed) {
            enableGroup('performance');
          }
        });
        if (!isConsentPresent && (shouldSetOTDefaults || OneTrust.GetDomainData().ConsentModel.Name === 'implied consent')) {
          OneTrust.UpdateConsent(`Category`, `1:1`);
          OneTrust.UpdateConsent(`Category`, `2:1`);
          OneTrust.UpdateConsent(`Category`, `3:1`);
          OneTrust.UpdateConsent(`Category`, `4:0`);
        }
        document.dispatchEvent(new CustomEvent('@sdtech/onetrust/loaded', {}));
        observer.disconnect();
        adjustOneTrustDOM();
      }
    };
    const observer = new MutationObserver(cb);
    observer.observe(document.querySelector('body'), {
      childList: true
    });
  }
  if (doClear) {
    document.cookie = 'OptanonAlertBoxClosed=; expires=Thu, 01 Jan 1970 00:00:00 UTC; samesite=lax; path=/';
  }
  const isConsentPresent = !!decodeURIComponent(document.cookie.match('(^| )OptanonConsent=([^;]+)')?.[2])?.match('groups=([0-9:0|1,?]+)&?')?.[1];
  const shouldSetOTDefaults = 'true' === 'false' && !document.cookie?.match('OptanonAlertBoxClosed=');
  if (shouldSetOTDefaults) {
    const date = new Date();
    date.setFullYear(date.getFullYear() + 1);
    document.cookie = `OptanonAlertBoxClosed=${new Date().toISOString()}; expires=${date.toUTCString()}; samesite=lax; path=/; domain=sciencedirect.com`;
  }
  observeOneTrustLoaded(shouldSetOTDefaults, isConsentPresent, monitor);
  window.addOTScript = () => {
    const otSDK = document.createElement('script');
    otSDK.setAttribute('data-cfasync', 'false');
    otSDK.setAttribute('src', 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js');
    otSDK.setAttribute('data-document-language', 'true');
    otSDK.setAttribute('data-domain-script', '865ea198-88cc-4e41-8952-1df75d554d02');
    window.addOTScript = () => {};
    document.head.appendChild(otSDK);
    monitor.init();
  };
  window.addEventListener('load', () => window.addOTScript());
}
        if (document.location.host.match(/.sciencedirect.com$/)) {
          runOneTrustCookies(true, monitor);
        }
        else {
          window.addEventListener('load', (event) => {
            enableGroup('performance');
          });
        }
      }());
    </script>
      
    <script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc808e99d44eef',t:'MTc0OTU5NjYwOC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe>
  <iframe id="ps_gsaqhrt" src="https://service.seamlessaccess.org/ps/" style="display: none; position: absolute; top: -999px; left: -999px;"></iframe><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-hide ot-fade-in" style="display: none;visibility: hidden;
                    opacity: 0;transition: visibility 0s 400ms, opacity 400ms linear;"></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region" style="display: none;visibility: hidden;
                    opacity: 0;transition: visibility 0s 400ms, opacity 400ms linear;"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler" ot-button-order="0" style="display: inline-block;">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Strictly Necessary Cookies - Cookie Details List" data-parent-id="1">Cookie Details Listâ€Ž</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" checked="" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Performance Cookies - Cookie Details List" data-parent-id="2">Cookie Details Listâ€Ž</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Targeting Cookies - Cookie Details List" data-parent-id="4">Cookie Details Listâ€Ž</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Searchâ€¦" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><div id="ot-filter-list-header"></div><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler" ot-button-order="2">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite">Your Privacy [`dialog closed`]</span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><script>
var pageDataTracker = {
    eventCookieName: 'eventTrack',
    debugCookie: 'els-aa-debugmode',
    debugCounter: 1,
    warnings: [],
    measures: {},
    timeoffset: 0

    ,trackPageLoad: function(data) {
        if (window.pageData && ((pageData.page && pageData.page.noTracking == 'true') || window.pageData_isLoaded)) {
            return false;
        }

        this.updatePageData(data);

        this.initWarnings();
        if(!(window.pageData && pageData.page && pageData.page.name)) {
            console.error('pageDataTracker.trackPageLoad() called without pageData.page.name being defined!');
            return;
        }

        this.processIdPlusData(window.pageData);

        if(window.pageData && pageData.page && !pageData.page.loadTime) {
          pageData.page.loadTime = performance ? Math.round((performance.now())).toString() : '';
        }

        if(window.pageData && pageData.page) {
            var localTime = new Date().getTime();
            if(pageData.page.loadTimestamp) {
                // calculate timeoffset
                var serverTime = parseInt(pageData.page.loadTimestamp);
                if(!isNaN(serverTime)) {
                    this.timeoffset = pageData.page.loadTimestamp - localTime;
                }
            } else {
                pageData.page.loadTimestamp = localTime;
            }
        }

        this.validateData(window.pageData);

        try {
            var cookieTest = 'aa-cookie-test';
            this.setCookie(cookieTest, cookieTest);
            if(this.getCookie(cookieTest) != cookieTest) {
                this.warnings.push('dtm5');
            }
            this.deleteCookie(cookieTest);
        } catch(e){
            this.warnings.push('dtm5');
        }

        this.registerCallbacks();
        this.setAnalyticsData();

        // handle any cookied event data
        this.getEvents();

        window.pageData_isLoaded = true;

        this.debugMessage('Init - trackPageLoad()', window.pageData);

        _satellite.track('eventDispatcher', JSON.stringify({
          eventName: 'newPage',
          eventData: {eventName: 'newPage'},
          pageData: window.pageData
        }));
    }

    ,trackEvent: function(event, data, callback) {
        if (window.pageData && pageData.page && pageData.page.noTracking == 'true') {
            return false;
        }
        
        if(!window.pageData_isLoaded) {
            if(this.isDebugEnabled()) {
                console.log('[AA] pageDataTracker.trackEvent() called without calling trackPageLoad() first.');
            }
            return false;
        }

        if (event) {
            this.initWarnings();
            if(event === 'newPage') {
                // auto fillings
                if(data && data.page && !data.page.loadTimestamp) {
                    data.page.loadTimestamp = ''+(new Date().getTime() + this.timeoffset);
                }
                this.processIdPlusData(data);
            }

            window.eventData = data ? data : {};
            window.eventData.eventName = event;
            if(!_satellite.getVar('blacklisted')) {
                this.handleEventData(event, data);
    
                if(event === 'newPage') {
                    this.validateData(window.pageData);
                }
                this.debugMessage('Event: ' + event, data);
    
                _satellite.track('eventDispatcher', JSON.stringify({
                  eventName: event,
                  eventData: window.eventData,
                  pageData: window.pageData
                }));
            } else {
                this.debugMessage('!! Blocked Event: ' + event, data);
            }
        }

        if (typeof(callback) == 'function') {
            callback.call();
        }
    }

    ,processIdPlusData: function(data) {
        if(data && data.visitor && data.visitor.idPlusData) {
            var idPlusFields = ['userId', 'accessType', 'accountId', 'accountName'];
            for(var i=0; i < idPlusFields.length; i++) {
                if(typeof data.visitor.idPlusData[idPlusFields[i]] !== 'undefined') {
                    data.visitor[idPlusFields[i]] = data.visitor.idPlusData[idPlusFields[i]];
                }
            }
            data.visitor.idPlusData = undefined;
        }
    }

    ,validateData: function(data) {
        if(!data) {
            this.warnings.push('dv0');
            return;
        }

        // top 5
        if(!(data.visitor && data.visitor.accessType)) {
            this.warnings.push('dv1');
        }
        if(data.visitor && (data.visitor.accountId || data.visitor.accountName)) {
            if(!data.visitor.accountName) {
                this.warnings.push('dv2');
            }
            if(!data.visitor.accountId) {
                this.warnings.push('dv3');
            }
        }
        if(!(data.page && data.page.productName)) {
            this.warnings.push('dv4');
        }
        if(!(data.page && data.page.businessUnit)) {
            this.warnings.push('dv5');
        }
        if(!(data.page && data.page.name)) {
            this.warnings.push('dv6');
        }

        // rp mandatory
        if(data.page && data.page.businessUnit && (data.page.businessUnit.toLowerCase().indexOf('els:rp:') !== -1 || data.page.businessUnit.toLowerCase().indexOf('els:rap:') !== -1)) {
            if(!(data.page && data.page.loadTimestamp)) {
                this.warnings.push('dv7');
            }
            if(!(data.page && data.page.loadTime)) {
                this.warnings.push('dv8');
            }
            if(!(data.visitor && data.visitor.ipAddress)) {
                this.warnings.push('dv9');
            }
            if(!(data.page && data.page.type)) {
                this.warnings.push('dv10');
            }
            if(!(data.page && data.page.language)) {
                this.warnings.push('dv11');
            }
        }

        // other
        if(data.page && data.page.environment) {
            var env = data.page.environment.toLowerCase();
            if(!(env === 'dev' || env === 'cert' || env === 'prod')) {
                this.warnings.push('dv12');
            }
        }
        if(data.content && data.content.constructor !== Array) {
            this.warnings.push('dv13');
        }

        if(data.visitor && data.visitor.accountId && data.visitor.accountId.indexOf(':') == -1) {
            this.warnings.push('dv14');
            data.visitor.accountId = "data violation"
        }
    }

    ,initWarnings: function() {
        this.warnings = [];
        try {
            var hdn = document.head.childNodes;
            var libf = false;
            for(var i=0; i<hdn.length; i++) {
                if(hdn[i].src && (hdn[i].src.indexOf('satelliteLib') !== -1 || hdn[i].src.indexOf('launch') !== -1)) {
                    libf = true;
                    break;
                }
            }
            if(!libf) {
                this.warnings.push('dtm1');
            }
        } catch(e) {}

        try {
            for (let element of document.querySelectorAll('*')) {
                // Check if the element has a src attribute and if it meets the criteria
                if (element.src && element.src.includes('assets.adobedtm.com') && element.src.includes('launch')) {
                    if (element.tagName.toLowerCase() !== 'script') {
                        this.warnings.push('dtm5');
                    }
                    if (!element.hasAttribute('async')) {
                        this.warnings.push('dtm4');
                    }
                }
            }
        } catch (e) { }
    }

    ,getMessages: function() {
        return ['v1'].concat(this.warnings).join('|');
    }
    ,addMessage: function(message) {
        this.warnings.push(message);
    }

    ,getPerformance: function() {
        var copy = {};
        for (var attr in this.measures) {
            if(this.measures.hasOwnProperty(attr)) {
                copy[attr] = this.measures[attr];
            }
        }

        this.measures = {};
        return copy;
    }

    ,dtmCodeDesc: {
        dtm1: 'satellite-lib must be placed in the <head> section',
        dtm2: 'trackPageLoad() must be placed and called before the closing </body> tag',
        dtm3: 'trackEvent() must be called at a stage where Document.readyState=complete (e.g. on the load event or a user event)',
        dtm4: 'Embed codes need to be loaded in async mode',
        dtm5: 'Embed codes not in type script',
        dv1: 'visitor.accessType not set but mandatory',
        dv2: 'visitor.accountName not set but mandatory',
        dv3: 'visitor.accountId not set but mandatory',
        dv4: 'page.productName not set but mandatory',
        dv5: 'page.businessUnit not set but mandatory',
        dv6: 'page.name not set but mandatory',
        dv7: 'page.loadTimestamp not set but mandatory',
        dv8: 'page.loadTime not set but mandatory',
        dv9: 'visitor.ipAddress not set but mandatory',
        dv10: 'page.type not set but mandatory',
        dv11: 'page.language not set but mandatory',
        dv12: 'page.environment must be set to \'prod\', \'cert\' or \'dev\'',
        dv13: 'content must be of type array of objects',
        dv14: 'account number must contain at least one \':\', e.g. \'ae:12345\''
    }

    ,debugMessage: function(event, data) {
        if(this.isDebugEnabled()) {
            console.log('[AA] --------- [' + (this.debugCounter++) + '] Web Analytics Data ---------');
            console.log('[AA] ' + event);
            console.groupCollapsed("[AA] AA Data: ");
            if(window.eventData) {
                console.log("[AA] eventData:\n" + JSON.stringify(window.eventData, true, 2));
            }
            if(window.pageData) {
                console.log("[AA] pageData:\n" + JSON.stringify(window.pageData, true, 2));
            }
            console.groupEnd();
            if(this.warnings.length > 0) {
                console.groupCollapsed("[AA] Warnings ("+this.warnings.length+"): ");
                for(var i=0; i<this.warnings.length; i++) {
                    var error = this.dtmCodeDesc[this.warnings[i]] ? this.dtmCodeDesc[this.warnings[i]] : 'Error Code: ' + this.warnings[i];
                    console.log('[AA] ' + error);
                }
                console.log('[AA] More can be found here: https://confluence.cbsels.com/display/AA/AA+Error+Catalog');
                console.groupEnd();
            }
            console.log("This mode can be disabled by calling 'pageDataTracker.disableDebug()'");
        }
    }

    ,getTrackingCode: function() {
      var campaign = _satellite.getVar('Campaign - ID');
      if(!campaign) {
        campaign = window.sessionStorage ? sessionStorage.getItem('dgcid') : '';
      }
      return campaign;
    }

    ,isDebugEnabled: function() {
        if(typeof this.debug === 'undefined') {
            this.debug = (document.cookie.indexOf(this.debugCookie) !== -1) || (window.pageData && pageData.page && pageData.page.environment && pageData.page.environment.toLowerCase() === 'dev');
            //this.debug = (document.cookie.indexOf(this.debugCookie) !== -1);
        }
        return this.debug;
    }

    ,enableDebug: function(expire) {
        if (typeof expire === 'undefined') {
            expire = 86400;
        }
        console.log('You just enabled debug mode for Adobe Analytics tracking. This mode will persist for 24h.');
        console.log("This mode can be disabled by calling 'pageDataTracker.disableDebug()'");
        this.setCookie(this.debugCookie, 'true', expire, document.location.hostname);
        this.debug = true;
    }

    ,disableDebug: function() {
        console.log('Debug mode is now disabled.');
        this.deleteCookie(this.debugCookie);
        this.debug = false;
    }

    ,setAnalyticsData: function() {
        if(!(window.pageData && pageData.page && pageData.page.productName && pageData.page.name)) {
            return;
        }
        pageData.page.analyticsPagename = pageData.page.productName + ':' + pageData.page.name;

        var pageEls = pageData.page.name.indexOf(':') > -1 ? pageData.page.name.split(':') : [pageData.page.name];
        pageData.page.sectionName = pageData.page.productName + ':' + pageEls[0];
    }

    ,getEvents: function() {
        pageData.savedEvents = {};
        pageData.eventList = [];

        var val = this.getCookie(this.eventCookieName);
        if (val) {
            pageData.savedEvents = val;
        }

        this.deleteCookie(this.eventCookieName);
    }

    ,updatePageData(data) {
        window.pageData = window.pageData || {};
        if (data && typeof(data) === 'object') {
            for (var x in data) {
                if(data.hasOwnProperty(x) && data[x] instanceof Array) {
                    pageData[x] = data[x];
                } else if(data.hasOwnProperty(x) && typeof(data[x]) === 'object') {
                    if(!pageData[x]) {
                        pageData[x] = {};
                    }
                    for (var y in data[x]) {
                        if(data[x].hasOwnProperty(y)) {
                            pageData[x][y] = data[x][y];
                        }
                    }
                }
            }
        }
    }

    ,handleEventData: function(event, data) {
        var val;
        switch(event) {
            case 'newPage':
                this.updatePageData(data);
                this.setAnalyticsData();
            case 'saveSearch':
            case 'searchResultsUpdated':
                if (data) {
                    // overwrite page-load object
                    if (data.search && typeof(data.search) == 'object') {
                        window.eventData.search.resultsPosition = '';
                        pageData.search = pageData.search || {};
                        var fields = ['advancedCriteria', 'criteria', 'currentPage', 'dataFormCriteria', 'facets', 'resultsByType', 'resultsPerPage', 'sortType', 'totalResults', 'type', 'database',
                        'suggestedClickPosition','suggestedLetterCount','suggestedResultCount', 'autoSuggestCategory', 'autoSuggestDetails','typedTerm','selectedTerm', 'channel',
                        'facetOperation', 'details'];
                        for (var i=0; i<fields.length; i++) {
                            if (data.search[fields[i]]) {
                                pageData.search[fields[i]] = data.search[fields[i]];
                            }
                        }
                    }
                }
                this.setAnalyticsData();
                break;
            case 'navigationClick':
                if (data && data.link) {
                    window.eventData.navigationLink = {
                        name: ((data.link.location || 'no location') + ':' + (data.link.name || 'no name'))
                    };
                }
                break;
            case 'autoSuggestClick':
                if (data && data.search) {
                    val = {
                        autoSuggestSearchData: (
                            'letterct:' + (data.search.suggestedLetterCount || 'none') +
                            '|resultct:' + (data.search.suggestedResultCount || 'none') +
                            '|clickpos:' + (data.search.suggestedClickPosition || 'none')
                        ).toLowerCase(),
                        autoSuggestSearchTerm: (data.search.typedTerm || ''),
                        autoSuggestTypedTerm: (data.search.typedTerm || ''),
                        autoSuggestSelectedTerm: (data.search.selectedTerm || ''),
                        autoSuggestCategory: (data.search.autoSuggestCategory || ''),
                        autoSuggestDetails: (data.search.autoSuggestDetails || '')
                    };
                }
                break;
            case 'linkOut':
                if (data && data.content && data.content.length > 0) {
                    window.eventData.linkOut = data.content[0].linkOut;
                    window.eventData.referringProduct = _satellite.getVar('Page - Product Name') + ':' + data.content[0].id;
                }
                break;
            case 'socialShare':
                if (data && data.social) {
                    window.eventData.sharePlatform = data.social.sharePlatform || '';
                }
                break;
            case 'contentInteraction':
                if (data && data.action) {
                    window.eventData.action.name = pageData.page.productName + ':' + data.action.name;
                }
                break;
            case 'searchWithinContent':
                if (data && data.search) {
                    window.pageData.search = window.pageData.search || {};
                    pageData.search.withinContentCriteria = data.search.withinContentCriteria;
                }
                break;
            case 'contentShare':
                if (data && data.content) {
                    window.eventData.sharePlatform = data.content[0].sharePlatform;
                }
                break;
            case 'contentLinkClick':
                if (data && data.link) {
                    window.eventData.action = { name: pageData.page.productName + ':' + (data.link.type || 'no link type') + ':' + (data.link.name || 'no link name') };
                }
                break;
            case 'contentWindowLoad':
            case 'contentTabClick':
                if (data && data.content) {
                    window.eventData.tabName = data.content[0].tabName || '';
                    window.eventData.windowName = data.content[0].windowName || '';
                }
                break;
            case 'userProfileUpdate':
                if (data && data.user) {
                    if (Object.prototype.toString.call(data.user) === "[object Array]") {
                        window.eventData.user = data.user[0];
                    }
                }
                break;
            case 'videoStart':
                if (data.video) {
                    data.video.length = parseFloat(data.video.length || '0');
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.open(data.video.id, data.video.length, s.Media.playerName);
                    s.Media.play(data.video.id, data.video.position);
                }
                break;
            case 'videoPlay':
                if (data.video) {
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.play(data.video.id, data.video.position);
                }
                break;
            case 'videoStop':
                if (data.video) {
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.stop(data.video.id, data.video.position);
                }
                break;
            case 'videoComplete':
                if (data.video) {
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.stop(data.video.id, data.video.position);
                    s.Media.close(data.video.id);
                }
                break;
            case 'addWebsiteExtension':
                if(data && data.page) {
                    val = {
                        wx: data.page.websiteExtension
                    }
                }
                break;
        }

        if (val) {
            this.setCookie(this.eventCookieName, val);
        }
    }

    ,registerCallbacks: function() {
        var self = this;
        if(window.usabilla_live) {
            window.usabilla_live('setEventCallback', function(category, action, label, value) {
                if(action == 'Campaign:Open') {
                    self.trackEvent('ctaImpression', {
                        cta: {
                            ids: ['usabillaid:' + label]
                        }
                    });
                } else if(action == 'Campaign:Success') {
                    self.trackEvent('ctaClick', {
                        cta: {
                            ids: ['usabillaid:' + label]
                        }
                    });
                }
            });
        }
    }

    ,getConsortiumAccountId: function() {
        var id = '';
        if (window.pageData && pageData.visitor && (pageData.visitor.consortiumId || pageData.visitor.accountId)) {
            id = (pageData.visitor.consortiumId || 'no consortium ID') + '|' + (pageData.visitor.accountId || 'no account ID');
        }

        return id;
    }

    ,getSearchClickPosition: function() {
        if (window.eventData && eventData.search && eventData.search.resultsPosition) {
            var pos = parseInt(eventData.search.resultsPosition), clickPos;
            if (!isNaN(pos)) {
                var page = pageData.search.currentPage ? parseInt(pageData.search.currentPage) : '', perPage = pageData.search.resultsPerPage ? parseInt(pageData.search.resultsPerPage) : '';
                if (!isNaN(page) && !isNaN(perPage)) {
                    clickPos = pos + ((page - 1) * perPage);
                }
            }
            return clickPos ? clickPos.toString() : eventData.search.resultsPosition;
        }
        return '';
    }

    ,getSearchFacets: function() {
        var facetList = '';
        if (window.pageData && pageData.search && pageData.search.facets) {
            if (typeof(pageData.search.facets) == 'object') {
                for (var i=0; i<pageData.search.facets.length; i++) {
                    var f = pageData.search.facets[i];
                    facetList += (facetList ? '|' : '') + f.name + '=' + f.values.join('^');
                }
            }
        }
        return facetList;
    }

    ,getSearchResultsByType: function() {
        var resultTypes = '';
        if (window.pageData && pageData.search && pageData.search.resultsByType) {
            for (var i=0; i<pageData.search.resultsByType.length; i++) {
                var r = pageData.search.resultsByType[i];
                resultTypes += (resultTypes ? '|' : '') + r.name + (r.results || r.values ? '=' + (r.results || r.values) : '');
            }
        }
        return resultTypes;
    }

    ,getJournalInfo: function() {
        var info = '';
        if (window.pageData && pageData.journal && (pageData.journal.name || pageData.journal.specialty || pageData.journal.section || pageData.journal.issn || pageData.journal.issueNumber || pageData.journal.volumeNumber || pageData.journal.family || pageData.journal.publisher)) {
            var journal = pageData.journal;
            info = (journal.name || 'no name') 
            + '|' + (journal.specialty || 'no specialty') 
            + '|' + (journal.section || 'no section') 
            + '|' + (journal.issn || 'no issn') 
            + '|' + (journal.issueNumber || 'no issue #') 
            + '|' + (journal.volumeNumber || 'no volume #')
            + '|' + (journal.family || 'no family')
            + '|' + (journal.publisher || 'no publisher');

        }
        return info;
    }

    ,getBibliographicInfo: function(doc) {
        if (!doc || !(doc.publisher || doc.indexTerms || doc.publicationType || doc.publicationRights || doc.volumeNumber || doc.issueNumber || doc.subjectAreas || doc.isbn)) {
            return '';
        }

        var terms = doc.indexTerms ? doc.indexTerms.split('+') : '';
        if (terms) {
            terms = terms.slice(0, 5).join('+');
            terms = terms.length > 100 ? terms.substring(0, 100) : terms;
        }

        var areas = doc.subjectAreas ? doc.subjectAreas.split('>') : '';
        if (areas) {
            areas = areas.slice(0, 5).join('>');
            areas = areas.length > 100 ? areas.substring(0, 100) : areas;
        }

        var biblio	= (doc.publisher || 'none')
            + '^' + (doc.publicationType || 'none')
            + '^' + (doc.publicationRights || 'none')
            + '^' + (terms || 'none')
            + '^' + (doc.volumeNumber || 'none')
            + '^' + (doc.issueNumber || 'none')
            + '^' + (areas || 'none')
            + '^' + (doc.isbn || 'none');

        return this.stripProductDelimiters(biblio).toLowerCase();
    }

    ,getContentItem: function() {
        var docs = window.eventData && eventData.content ? eventData.content : pageData.content;
        if (docs && docs.length > 0) {
            return docs[0];
        }
    }

    ,getFormattedDate: function(ts) {
        if (!ts) {
            return '';
        }

        var d = new Date(parseInt(ts) * 1000);

        // now do formatting
        var year = d.getFullYear()
            ,month = ((d.getMonth() + 1) < 10 ? '0' : '') + (d.getMonth() + 1)
            ,date = (d.getDate() < 10 ? '0' : '') + d.getDate()
            ,hours = d.getHours() > 12 ? d.getHours() - 12 : d.getHours()
            ,mins = (d.getMinutes() < 10 ? '0' : '') + d.getMinutes()
            ,ampm = d.getHours() > 12 ? 'pm' : 'am';

        hours = (hours < 10 ? '0' : '') + hours;
        return year + '-' + month + '-' + date;
    }

    ,getVisitorId: function() {
        var orgId = '4D6368F454EC41940A4C98A6@AdobeOrg';
        if(Visitor && Visitor.getInstance(orgId)) {
            return Visitor.getInstance(orgId).getMarketingCloudVisitorID();
        } else {
            return ''
        }
    }

    ,setProductsVariable: function() {
        var prodList = window.eventData && eventData.content ? eventData.content : pageData.content
            ,prods = [];
        if (prodList) {
            for (var i=0; i<prodList.length; i++) {
                if (prodList[i].id || prodList[i].type || prodList[i].publishDate || prodList[i].onlineDate) {
                    if (!prodList[i].id) {
                        prodList[i].id = 'no id';
                    }
                    var prodName = (pageData.page.productName || 'xx').toLowerCase();
                    if (prodList[i].id.indexOf(prodName + ':') != 0) {
                        prodList[i].id = prodName + ':' + prodList[i].id;
                    }
                    prodList[i].id = this.stripProductDelimiters(prodList[i].id);
                    var merch = [];
                    if (prodList[i].format) {
                        merch.push('evar17=' + this.stripProductDelimiters(prodList[i].format.toLowerCase()));
                    }
                    if (prodList[i].type) {
                        var type = prodList[i].type;
                        if (prodList[i].accessType) {
                            type += ':' + prodList[i].accessType;
                        }
                        merch.push('evar20=' + this.stripProductDelimiters(type.toLowerCase()));

                        if(type.indexOf(':manuscript') > 0) {
                            /*
                            var regex = /[a-z]+:manuscript:id:([a-z]+-[a-z]-[0-9]+-[0-9]+)/gmi;
                            var m = regex.exec(prodList[i].id);
                            if(m) {
                                merch.push('evar200=' + m[1]);
                            }
                            merch.push('evar200=' + prodList[i].id);
                            */
                            a = prodList[i].id.lastIndexOf(':');
                            if(a>0) {
                                merch.push('evar200=' + prodList[i].id.substring(a+1).toUpperCase());
                            }
                        } else if(type.indexOf(':submission') > 0) {
                            merch.push('evar200=' + prodList[i].id);
                        }
                    }
                    if(!prodList[i].title) {
                        prodList[i].title = prodList[i].name;
                    }
                    if (prodList[i].title) {
                        merch.push('evar75=' + this.stripProductDelimiters(prodList[i].title.toLowerCase()));
                    }
                    if (prodList[i].breadcrumb) {
                        merch.push('evar63=' + this.stripProductDelimiters(prodList[i].breadcrumb).toLowerCase());
                    }
                    var nowTs = new Date().getTime()/1000;
                    if (prodList[i].onlineDate && !isNaN(prodList[i].onlineDate)) {
                        if(prodList[i].onlineDate > 32503680000) {
                            prodList[i].onlineDate = prodList[i].onlineDate/1000;
                        }
                        merch.push('evar122=' + this.stripProductDelimiters(pageDataTracker.getFormattedDate(prodList[i].onlineDate)));
                        var onlineAge = Math.floor((nowTs - prodList[i].onlineDate) / 86400);
                        onlineAge = (onlineAge === 0) ? 'zero' : onlineAge;
                        merch.push('evar128=' + onlineAge);
                    }
                    if (prodList[i].publishDate && !isNaN(prodList[i].publishDate)) {
                        if(prodList[i].publishDate > 32503680000) {
                            prodList[i].publishDate = prodList[i].publishDate/1000;
                        }
                        merch.push('evar123=' + this.stripProductDelimiters(pageDataTracker.getFormattedDate(prodList[i].publishDate)));
                        var publishAge = Math.floor((nowTs - prodList[i].publishDate) / 86400);
                        publishAge = (publishAge === 0) ? 'zero' : publishAge;
                        merch.push('evar127=' + publishAge);
                    }
                    if (prodList[i].onlineDate && prodList[i].publishDate) {
                        merch.push('evar38=' + this.stripProductDelimiters(pageDataTracker.getFormattedDate(prodList[i].onlineDate) + '^' + pageDataTracker.getFormattedDate(prodList[i].publishDate)));
                    }
                    if (prodList[i].mapId) {
                        merch.push('evar70=' + this.stripProductDelimiters(prodList[i].mapId));
                    }
					if (prodList[i].relevancyScore) {
						merch.push('evar71=' + this.stripProductDelimiters(prodList[i].relevancyScore));
					}
                    if (prodList[i].status) {
                        merch.push('evar73=' + this.stripProductDelimiters(prodList[i].status));
                    }
                    if (prodList[i].previousStatus) {
                        merch.push('evar111=' + this.stripProductDelimiters(prodList[i].previousStatus));
                    }
                    if (prodList[i].entitlementType) {
                        merch.push('evar80=' + this.stripProductDelimiters(prodList[i].entitlementType));
                    }
                    if (prodList[i].recordType) {
                        merch.push('evar93=' + this.stripProductDelimiters(prodList[i].recordType));
                    }
                    if (prodList[i].exportType) {
                        merch.push('evar99=' + this.stripProductDelimiters(prodList[i].exportType));
                    }
                    if (prodList[i].importType) {
                        merch.push('evar142=' + this.stripProductDelimiters(prodList[i].importType));
                    }
                    if (prodList[i].section) {
                        merch.push('evar100=' + this.stripProductDelimiters(prodList[i].section));
                    }
                    if (prodList[i].detail) {
                        merch.push('evar104=' + this.stripProductDelimiters(prodList[i].detail.toLowerCase()));
                    } else if(prodList[i].details) {
                        merch.push('evar104=' + this.stripProductDelimiters(prodList[i].details.toLowerCase()));
                    }
                    if (prodList[i].position) {
                        merch.push('evar116=' + this.stripProductDelimiters(prodList[i].position));
                    }
                    if (prodList[i].publicationTitle) {
                        merch.push('evar129=' + this.stripProductDelimiters(prodList[i].publicationTitle));
                    }
                    if (prodList[i].specialIssueTitle) {
                        merch.push('evar130=' + this.stripProductDelimiters(prodList[i].specialIssueTitle));
                    }
                    if (prodList[i].specialIssueNumber) {
                        merch.push('evar131=' + this.stripProductDelimiters(prodList[i].specialIssueNumber));
                    }
                    if (prodList[i].referenceModuleTitle) {
                        merch.push('evar139=' + this.stripProductDelimiters(prodList[i].referenceModuleTitle));
                    }
                    if (prodList[i].referenceModuleISBN) {
                        merch.push('evar140=' + this.stripProductDelimiters(prodList[i].referenceModuleISBN));
                    }
                    if (prodList[i].volumeTitle) {
                        merch.push('evar132=' + this.stripProductDelimiters(prodList[i].volumeTitle));
                    }
                    if (prodList[i].publicationSection) {
                        merch.push('evar133=' + this.stripProductDelimiters(prodList[i].publicationSection));
                    }
                    if (prodList[i].publicationSpecialty) {
                        merch.push('evar134=' + this.stripProductDelimiters(prodList[i].publicationSpecialty));
                    }
                    if (prodList[i].issn) {
                        merch.push('evar135=' + this.stripProductDelimiters(prodList[i].issn));
                    }
                    if (prodList[i].id2) {
                        merch.push('evar159=' + this.stripProductDelimiters(prodList[i].id2));
                    }
                    if (prodList[i].id3) {
                        merch.push('evar160=' + this.stripProductDelimiters(prodList[i].id3));
                    }
                    if (prodList[i].provider) {
                        merch.push('evar164=' + this.stripProductDelimiters(prodList[i].provider));
                    }
                    if (prodList[i].citationStyle) {
                        merch.push('evar170=' + this.stripProductDelimiters(prodList[i].citationStyle));
                    }

                    var biblio = this.getBibliographicInfo(prodList[i]);
                    if (biblio) {
                        merch.push('evar28=' + biblio);
                    }

                    if (prodList[i].turnawayId) {
                        pageData.eventList.push('product turnaway');
                    }

                    var price = prodList[i].price || '', qty = prodList[i].quantity || '', evts = [];
                    if (price && qty) {
                        qty = parseInt(qty || '1');
                        price = parseFloat(price || '0');
                        price = (price * qty).toFixed(2);

                        if (window.eventData && eventData.eventName && eventData.eventName == 'cartAdd') {
                            evts.push('event20=' + price);
                        }
                    }

                    var type = window.pageData && pageData.page && pageData.page.type ? pageData.page.type : '', evt = window.eventData && eventData.eventName ? eventData.eventName : '';
                    if (type.match(/^CP\-/gi) !== null && (!evt || evt == 'newPage' || evt == 'contentView')) {
                        evts.push('event181=1');
                    }
                    if (evt == 'contentDownload' || type.match(/^CP\-DL/gi) !== null) {
                        evts.push('event182=1');
                    }
                    if (evt == 'contentDownloadRequest') {
                        evts.push('event319=1');
                    }
                    if (evt == 'contentExport') {
                        evts.push('event184=1');
                    }
                    if (this.eventFires('recommendationViews')) {
                        evts.push('event264=1');
                    }

                    if(prodList[i].datapoints) {
                        evts.push('event239=' + prodList[i].datapoints);
                    }
                    if(prodList[i].documents) {
                        evts.push('event240=' + prodList[i].documents);
                    }
                    if(prodList[i].size) {
                        evts.push('event335=' + prodList[i].size);
                        evts.push('event336=1')
                    }

                    if(evt == 'genAIContentUpdated') {
                        evts.push('event51=1');
                    }

                    prods.push([
                        ''					// empty category
                        ,prodList[i].id		// id
                        ,qty				// qty
                        ,price				// price
                        ,evts.join('|')		// events
                        ,merch.join('|')	// merchandising eVars
                    ].join(';'));
                }
            }
        }

        return prods.join(',');
    }
    ,eventFires: function(eventName) {
      var evt = window.eventData && eventData.eventName ? eventData.eventName : '';
      if(evt == eventName) {
        return true;
      }
      // initial pageload and new pages
      if((!window.eventData || evt == 'newPage') && window.pageData && window.pageData.trackEvents) {
        var tEvents = window.pageData.trackEvents;
        for(var i=0; i<tEvents.length; i++) {
          if(tEvents[i] == eventName) {
            return true;
          }
        }
      }
      return false;
    }

    ,md5: function(s){function L(k,d){return(k<<d)|(k>>>(32-d))}function K(G,k){var I,d,F,H,x;F=(G&2147483648);H=(k&2147483648);I=(G&1073741824);d=(k&1073741824);x=(G&1073741823)+(k&1073741823);if(I&d){return(x^2147483648^F^H)}if(I|d){if(x&1073741824){return(x^3221225472^F^H)}else{return(x^1073741824^F^H)}}else{return(x^F^H)}}function r(d,F,k){return(d&F)|((~d)&k)}function q(d,F,k){return(d&k)|(F&(~k))}function p(d,F,k){return(d^F^k)}function n(d,F,k){return(F^(d|(~k)))}function u(G,F,aa,Z,k,H,I){G=K(G,K(K(r(F,aa,Z),k),I));return K(L(G,H),F)}function f(G,F,aa,Z,k,H,I){G=K(G,K(K(q(F,aa,Z),k),I));return K(L(G,H),F)}function D(G,F,aa,Z,k,H,I){G=K(G,K(K(p(F,aa,Z),k),I));return K(L(G,H),F)}function t(G,F,aa,Z,k,H,I){G=K(G,K(K(n(F,aa,Z),k),I));return K(L(G,H),F)}function e(G){var Z;var F=G.length;var x=F+8;var k=(x-(x%64))/64;var I=(k+1)*16;var aa=Array(I-1);var d=0;var H=0;while(H<F){Z=(H-(H%4))/4;d=(H%4)*8;aa[Z]=(aa[Z]| (G.charCodeAt(H)<<d));H++}Z=(H-(H%4))/4;d=(H%4)*8;aa[Z]=aa[Z]|(128<<d);aa[I-2]=F<<3;aa[I-1]=F>>>29;return aa}function B(x){var k="",F="",G,d;for(d=0;d<=3;d++){G=(x>>>(d*8))&255;F="0"+G.toString(16);k=k+F.substr(F.length-2,2)}return k}function J(k){k=k.replace(/rn/g,"n");var d="";for(var F=0;F<k.length;F++){var x=k.charCodeAt(F);if(x<128){d+=String.fromCharCode(x)}else{if((x>127)&&(x<2048)){d+=String.fromCharCode((x>>6)|192);d+=String.fromCharCode((x&63)|128)}else{d+=String.fromCharCode((x>>12)|224);d+=String.fromCharCode(((x>>6)&63)|128);d+=String.fromCharCode((x&63)|128)}}}return d}var C=Array();var P,h,E,v,g,Y,X,W,V;var S=7,Q=12,N=17,M=22;var A=5,z=9,y=14,w=20;var o=4,m=11,l=16,j=23;var U=6,T=10,R=15,O=21;s=J(s);C=e(s);Y=1732584193;X=4023233417;W=2562383102;V=271733878;for(P=0;P<C.length;P+=16){h=Y;E=X;v=W;g=V;Y=u(Y,X,W,V,C[P+0],S,3614090360);V=u(V,Y,X,W,C[P+1],Q,3905402710);W=u(W,V,Y,X,C[P+2],N,606105819);X=u(X,W,V,Y,C[P+3],M,3250441966);Y=u(Y,X,W,V,C[P+4],S,4118548399);V=u(V,Y,X,W,C[P+5],Q,1200080426);W=u(W,V,Y,X,C[P+6],N,2821735955);X=u(X,W,V,Y,C[P+7],M,4249261313);Y=u(Y,X,W,V,C[P+8],S,1770035416);V=u(V,Y,X,W,C[P+9],Q,2336552879);W=u(W,V,Y,X,C[P+10],N,4294925233);X=u(X,W,V,Y,C[P+11],M,2304563134);Y=u(Y,X,W,V,C[P+12],S,1804603682);V=u(V,Y,X,W,C[P+13],Q,4254626195);W=u(W,V,Y,X,C[P+14],N,2792965006);X=u(X,W,V,Y,C[P+15],M,1236535329);Y=f(Y,X,W,V,C[P+1],A,4129170786);V=f(V,Y,X,W,C[P+6],z,3225465664);W=f(W,V,Y,X,C[P+11],y,643717713);X=f(X,W,V,Y,C[P+0],w,3921069994);Y=f(Y,X,W,V,C[P+5],A,3593408605);V=f(V,Y,X,W,C[P+10],z,38016083);W=f(W,V,Y,X,C[P+15],y,3634488961);X=f(X,W,V,Y,C[P+4],w,3889429448);Y=f(Y,X,W,V,C[P+9],A,568446438);V=f(V,Y,X,W,C[P+14],z,3275163606);W=f(W,V,Y,X,C[P+3],y,4107603335);X=f(X,W,V,Y,C[P+8],w,1163531501);Y=f(Y,X,W,V,C[P+13],A,2850285829);V=f(V,Y,X,W,C[P+2],z,4243563512);W=f(W,V,Y,X,C[P+7],y,1735328473);X=f(X,W,V,Y,C[P+12],w,2368359562);Y=D(Y,X,W,V,C[P+5],o,4294588738);V=D(V,Y,X,W,C[P+8],m,2272392833);W=D(W,V,Y,X,C[P+11],l,1839030562);X=D(X,W,V,Y,C[P+14],j,4259657740);Y=D(Y,X,W,V,C[P+1],o,2763975236);V=D(V,Y,X,W,C[P+4],m,1272893353);W=D(W,V,Y,X,C[P+7],l,4139469664);X=D(X,W,V,Y,C[P+10],j,3200236656);Y=D(Y,X,W,V,C[P+13],o,681279174);V=D(V,Y,X,W,C[P+0],m,3936430074);W=D(W,V,Y,X,C[P+3],l,3572445317);X=D(X,W,V,Y,C[P+6],j,76029189);Y=D(Y,X,W,V,C[P+9],o,3654602809);V=D(V,Y,X,W,C[P+12],m,3873151461);W=D(W,V,Y,X,C[P+15],l,530742520);X=D(X,W,V,Y,C[P+2],j,3299628645);Y=t(Y,X,W,V,C[P+0],U,4096336452);V=t(V,Y,X,W,C[P+7],T,1126891415);W=t(W,V,Y,X,C[P+14],R,2878612391);X=t(X,W,V,Y,C[P+5],O,4237533241);Y=t(Y,X,W,V,C[P+12],U,1700485571);V=t(V,Y,X,W,C[P+3],T,2399980690);W=t(W,V,Y,X,C[P+10],R,4293915773);X=t(X,W,V,Y,C[P+1],O,2240044497);Y=t(Y,X,W,V,C[P+8],U,1873313359);V=t(V,Y,X,W,C[P+15],T,4264355552);W=t(W,V,Y,X,C[P+6],R,2734768916);X=t(X,W,V,Y,C[P+13],O,1309151649);Y=t(Y,X,W,V,C[P+4],U,4149444226);V=t(V,Y,X,W,C[P+11],T,3174756917);W=t(W,V,Y,X,C[P+2],R,718787259);X=t(X,W,V,Y,C[P+9],O,3951481745);Y=K(Y,h);X=K(X,E);W=K(W,v);V=K(V,g)}var i=B(Y)+B(X)+B(W)+B(V);return i.toLowerCase()}
    ,stripProductDelimiters: function(val) {
        if (val) {
            return val.replace(/\;|\||\,/gi, '-');
        }
    }

    ,setCookie: function(name, value, seconds, domain) {
        domain = document.location.hostname;
        var expires = '';
        var expiresNow = '';
        var date = new Date();
        date.setTime(date.getTime() + (-1 * 1000));
        expiresNow = "; expires=" + date.toGMTString();

        if (typeof(seconds) != 'undefined') {
            date.setTime(date.getTime() + (seconds * 1000));
            expires = '; expires=' + date.toGMTString();
        }

        var type = typeof(value);
        type = type.toLowerCase();
        if (type != 'undefined' && type != 'string') {
            value = JSON.stringify(value);
        }

        // fix scoping issues
        // keep writing the old cookie, but make it expire
        document.cookie = name + '=' + value + expiresNow + '; path=/';

        // now just set the right one
        document.cookie = name + '=' + value + expires + '; path=/; domain=' + domain;
    }

    ,getCookie: function(name) {
        name = name + '=';
        var carray = document.cookie.split(';'), value;

        for (var i=0; i<carray.length; i++) {
            var c = carray[i];
            while (c.charAt(0) == ' ') {
                c = c.substring(1, c.length);
            }
            if (c.indexOf(name) == 0) {
                value = c.substring(name.length, c.length);
                try {
                    value = JSON.parse(value);
                } catch(ex) {}

                return value;
            }
        }

        return null;
    }

    ,deleteCookie: function(name) {
        this.setCookie(name, '', -1);
        this.setCookie(name, '', -1, document.location.hostname);
    }

    ,mapAdobeVars: function(s) {
        var vars = {
            pageName		: 'Page - Analytics Pagename'
            ,channel		: 'Page - Section Name'
            ,campaign		: 'Campaign - ID'
            ,currencyCode	: 'Page - Currency Code'
            ,purchaseID		: 'Order - ID'
            ,prop1			: 'Visitor - Account ID'
            ,prop2			: 'Page - Product Name'
            ,prop4			: 'Page - Type'
            ,prop6			: 'Search - Type'
            ,prop7			: 'Search - Facet List'
            ,prop8			: 'Search - Feature Used'
            ,prop12			: 'Visitor - User ID'
            ,prop13			: 'Search - Sort Type'
            ,prop14			: 'Page - Load Time'
            ,prop15         : 'Support - Topic Name'
            ,prop16			: 'Page - Business Unit'
            ,prop21			: 'Search - Criteria'
            ,prop24			: 'Page - Language'
            ,prop25			: 'Page - Product Feature'
            ,prop28         : 'Support - Search Criteria'
            ,prop30			: 'Visitor - IP Address'
            ,prop33         : 'Page - Product Application Version'
            ,prop34         : 'Page - Website Extensions'
            ,prop60			: 'Search - Data Form Criteria'
            ,prop63			: 'Page - Extended Page Name'
            ,prop65         : 'Page - Online State'
            ,prop67         : 'Research Networks'
            ,prop40: 'Page - UX Properties'

            ,eVar3			: 'Search - Total Results'
            ,eVar7			: 'Visitor - Account Name'
            ,eVar15			: 'Event - Search Results Click Position'
            ,eVar19			: 'Search - Advanced Criteria'
            ,eVar21			: 'Promo - Clicked ID'
            ,eVar22			: 'Page - Test ID'
            ,eVar47			: 'Page - Test Subject ID'
            ,eVar27			: 'Event - AutoSuggest Search Data'
            ,eVar157		: 'Event - AutoSuggest Search Typed Term'
            ,eVar156		: 'Event - AutoSuggest Search Selected Term'
            ,eVar162		: 'Event - AutoSuggest Search Category'
            ,eVar163		: 'Event - AutoSuggest Search Details'
            ,eVar33			: 'Visitor - Access Type'
            ,eVar34			: 'Order - Promo Code'
            ,eVar39			: 'Order - Payment Method'
            ,eVar41			: 'Visitor - Industry'
            ,eVar42			: 'Visitor - SIS ID'
            ,eVar43			: 'Page - Error Type'
            ,eVar44			: 'Event - Updated User Fields'
            ,eVar48			: 'Email - Recipient ID'
            ,eVar51			: 'Email - Message ID'
            ,eVar52			: 'Visitor - Department ID'
            ,eVar53			: 'Visitor - Department Name'
            ,eVar60			: 'Search - Within Content Criteria'
            ,eVar61			: 'Search - Within Results Criteria'
            ,eVar62			: 'Search - Result Types'
            ,eVar74			: 'Page - Journal Info'
            ,eVar59			: 'Page - Journal Publisher'
            ,eVar76			: 'Email - Broadlog ID'
            ,eVar78			: 'Visitor - Details'
            ,eVar80         : 'Visitor - Usage Path Info'
            ,eVar102		: 'Form - Name'
            ,eVar103        : 'Event - Conversion Driver'
            ,eVar105        : 'Search - Current Page'
            ,eVar106        : 'Visitor - App Session ID'
            ,eVar107        : 'Page - Secondary Product Name'
            ,eVar117        : 'Search - Database'
            ,eVar126        : 'Page - Environment'
            ,eVar141        : 'Search - Criteria Original'
            ,eVar143        : 'Page - Tabs'
            ,eVar161        : 'Search - Channel'
            ,eVar169        : 'Search - Facet Operation'
            ,eVar173        : 'Search - Details'
            ,eVar174        : 'Campaign - Spredfast ID'
            ,eVar175        : 'Visitor - TMX Device ID'
            ,eVar176        : 'Visitor - TMX Request ID'
            ,eVar148        : 'Visitor - Platform Name'
            ,eVar149        : 'Visitor - Platform ID'
            ,eVar152        : 'Visitor - Product ID'
            ,eVar153        : 'Visitor - Superaccount ID'
            ,eVar154        : 'Visitor - Superaccount Name'
            ,eVar177        : 'Page - Context Domain'
            ,eVar189    : 'Page - Experimentation User Id'
            ,eVar190    : 'Page - Identity User'
            ,eVar199    : 'Page - ID+ Parameters'

            ,list2			: 'Page - Widget Names'
            ,list3			: 'Promo - IDs'
        };

        for (var i in vars) {
            s[i] = s[i] ? s[i] : _satellite.getVar(vars[i]);
        }
    }
};

// async support fallback
(function(w) {
	var eventBuffer = [];
	if(w.appData) {
		if(Array.isArray(w.appData)) {
			eventBuffer = w.appData;
		} else {
			console.error('Elsevier DataLayer "window.appData" must be specified as array');
			return;
		}
    }

	w.appData = [];

	var oldPush = w.appData.push;

	var appDataPush = function() {
        oldPush.apply(w.appData, arguments);
        for(var i=0; i<arguments.length; i++) {
            var data = arguments[i];
            if(data.event) {
                if(data.event == 'pageLoad') {
                    w.pageDataTracker.trackPageLoad(data);
                } else {
                    w.pageDataTracker.trackEvent(data.event, data);
                }
            }
        }
	};

	w.appData.push = appDataPush;
	for(var i=0; i<eventBuffer.length; i++) {
	    var data = eventBuffer[i];
	    w.appData.push(data);
	}
})(window);

</script><script>_satellite["_runScript1"](function(event, target, Promise) {
_satellite.logger.log("eventDispatcher: clearing tracking state");try{s.events="",s.linkTrackVars="",s.linkTrackEvents=""}catch(e){_satellite.logger.log("eventDispatcher: s object - could not reset state.")}try{dispatcherData=JSON.parse(event.detail),window.ddqueue=window.ddqueue||[],window.ddqueue.push(dispatcherData),window.eventData=dispatcherData.eventData,window.pageData=dispatcherData.pageData,_satellite.track(dispatcherData.eventName)}catch(e){_satellite.logger.log("eventDispatcher: exception"),_satellite.logger.log(e)}
});</script><script>_satellite["_runScript2"](function(event, target, Promise) {
_satellite.logger.log("eventDispatcher: clearing tracking state");try{s.events="",s.linkTrackVars="",s.linkTrackEvents=""}catch(e){_satellite.logger.log("eventDispatcher: s object - could not reset state.")}try{dispatcherData=JSON.parse(event.detail),window.ddqueue=window.ddqueue||[],window.ddqueue.push(dispatcherData),window.eventData=dispatcherData.eventData,window.pageData=dispatcherData.pageData,_satellite.track(dispatcherData.eventName)}catch(e){_satellite.logger.log("eventDispatcher: exception"),_satellite.logger.log(e)}
});</script></body></html>