<html lang="en-US" class="toolbar-stuck"><head>
      <meta name="citation_pii" content="S0047248421001019">
<meta name="citation_id" content="103049">
<meta name="citation_issn" content="0047-2484">
<meta name="citation_volume" content="159">
<meta name="citation_publisher" content="Academic Press">
<meta name="citation_firstpage" content="103049">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_journal_title" content="Journal of Human Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.jhevol.2021.103049">
<meta name="dc.identifier" content="10.1016/j.jhevol.2021.103049">
<meta name="citation_article_type" content="Full-length article">
<meta property="og:description" content="Ancient DNA analyses have shown that interbreeding between hominin taxa occurred multiple times. Although admixture is often reflected in skeletal phe…">
<meta property="og:image" content="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421X00097-cov150h.gif">
<meta name="citation_title" content="Effects of hybridization on pelvic morphology: A macaque model">
<meta property="og:title" content="Effects of hybridization on pelvic morphology: A macaque model">
<meta name="citation_publication_date" content="2021/10/01">
<meta name="citation_online_date" content="2021/08/26">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOCACHE,NOODP,NOYDIR">
      <title>Effects of hybridization on pelvic morphology: A macaque model - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S0047248421001019">
      <meta name="tdm-reservation" content="1">
      <meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">
      <meta property="og:type" content="article">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team">
      <script async="" src="https://cdn.pendo.io/agent/static/d6c1d995-bc7e-4e53-77f1-2ea4ecbb9565/pendo.js"></script><script type="text/javascript">(function newRelicBrowserProSPA() {
  ;
  window.NREUM || (NREUM = {});
  NREUM.init = {
    privacy: {
      cookies_enabled: false
    },
    ajax: {
      deny_list: ["bam-cell.nr-data.net"]
    }
  };
  ;
  NREUM.loader_config = {
    accountID: "2128461",
    trustKey: "2038175",
    agentID: "1118783207",
    licenseKey: "7ac4127487",
    applicationID: "814814845"
  };
  ;
  NREUM.info = {
    beacon: "bam.nr-data.net",
    errorBeacon: "bam.nr-data.net",
    licenseKey: "7ac4127487",
    applicationID: "814814845",
    sa: 1
  };
  ; /*! For license information please see nr-loader-spa-1.238.0.min.js.LICENSE.txt */
  (() => {
    "use strict";

    var e,
      t,
      r = {
        5763: (e, t, r) => {
          r.d(t, {
            P_: () => f,
            Mt: () => p,
            C5: () => s,
            DL: () => v,
            OP: () => T,
            lF: () => D,
            Yu: () => y,
            Dg: () => h,
            CX: () => c,
            GE: () => b,
            sU: () => _
          });
          var n = r(8632),
            i = r(9567);
          const o = {
              beacon: n.ce.beacon,
              errorBeacon: n.ce.errorBeacon,
              licenseKey: void 0,
              applicationID: void 0,
              sa: void 0,
              queueTime: void 0,
              applicationTime: void 0,
              ttGuid: void 0,
              user: void 0,
              account: void 0,
              product: void 0,
              extra: void 0,
              jsAttributes: {},
              userAttributes: void 0,
              atts: void 0,
              transactionName: void 0,
              tNamePlain: void 0
            },
            a = {};
          function s(e) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            if (!a[e]) throw new Error("Info for ".concat(e, " was never set"));
            return a[e];
          }
          function c(e, t) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            a[e] = (0, i.D)(t, o), (0, n.Qy)(e, a[e], "info");
          }
          var u = r(7056);
          const d = () => {
              const e = {
                blockSelector: "[data-nr-block]",
                maskInputOptions: {
                  password: !0
                }
              };
              return {
                allow_bfcache: !0,
                privacy: {
                  cookies_enabled: !0
                },
                ajax: {
                  deny_list: void 0,
                  block_internal: !0,
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                distributed_tracing: {
                  enabled: void 0,
                  exclude_newrelic_header: void 0,
                  cors_use_newrelic_header: void 0,
                  cors_use_tracecontext_headers: void 0,
                  allowed_origins: void 0
                },
                session: {
                  domain: void 0,
                  expiresMs: u.oD,
                  inactiveMs: u.Hb
                },
                ssl: void 0,
                obfuscate: void 0,
                jserrors: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                metrics: {
                  enabled: !0
                },
                page_action: {
                  enabled: !0,
                  harvestTimeSeconds: 30
                },
                page_view_event: {
                  enabled: !0
                },
                page_view_timing: {
                  enabled: !0,
                  harvestTimeSeconds: 30,
                  long_task: !1
                },
                session_trace: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                harvest: {
                  tooManyRequestsDelay: 60
                },
                session_replay: {
                  enabled: !1,
                  harvestTimeSeconds: 60,
                  sampleRate: .1,
                  errorSampleRate: .1,
                  maskTextSelector: "*",
                  maskAllInputs: !0,
                  get blockClass() {
                    return "nr-block";
                  },
                  get ignoreClass() {
                    return "nr-ignore";
                  },
                  get maskTextClass() {
                    return "nr-mask";
                  },
                  get blockSelector() {
                    return e.blockSelector;
                  },
                  set blockSelector(t) {
                    e.blockSelector += ",".concat(t);
                  },
                  get maskInputOptions() {
                    return e.maskInputOptions;
                  },
                  set maskInputOptions(t) {
                    e.maskInputOptions = {
                      ...t,
                      password: !0
                    };
                  }
                },
                spa: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                }
              };
            },
            l = {};
          function f(e) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            if (!l[e]) throw new Error("Configuration for ".concat(e, " was never set"));
            return l[e];
          }
          function h(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            l[e] = (0, i.D)(t, d()), (0, n.Qy)(e, l[e], "config");
          }
          function p(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            var r = f(e);
            if (r) {
              for (var n = t.split("."), i = 0; i < n.length - 1; i++) if ("object" != typeof (r = r[n[i]])) return;
              r = r[n[n.length - 1]];
            }
            return r;
          }
          const g = {
              accountID: void 0,
              trustKey: void 0,
              agentID: void 0,
              licenseKey: void 0,
              applicationID: void 0,
              xpid: void 0
            },
            m = {};
          function v(e) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            if (!m[e]) throw new Error("LoaderConfig for ".concat(e, " was never set"));
            return m[e];
          }
          function b(e, t) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            m[e] = (0, i.D)(t, g), (0, n.Qy)(e, m[e], "loader_config");
          }
          const y = (0, n.mF)().o;
          var w = r(385),
            A = r(6818);
          const x = {
              buildEnv: A.Re,
              bytesSent: {},
              queryBytesSent: {},
              customTransaction: void 0,
              disabled: !1,
              distMethod: A.gF,
              isolatedBacklog: !1,
              loaderType: void 0,
              maxBytes: 3e4,
              offset: Math.floor(w._A?.performance?.timeOrigin || w._A?.performance?.timing?.navigationStart || Date.now()),
              onerror: void 0,
              origin: "" + w._A.location,
              ptid: void 0,
              releaseIds: {},
              session: void 0,
              xhrWrappable: "function" == typeof w._A.XMLHttpRequest?.prototype?.addEventListener,
              version: A.q4,
              denyList: void 0
            },
            E = {};
          function T(e) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            if (!E[e]) throw new Error("Runtime for ".concat(e, " was never set"));
            return E[e];
          }
          function _(e, t) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            E[e] = (0, i.D)(t, x), (0, n.Qy)(e, E[e], "runtime");
          }
          function D(e) {
            return function (e) {
              try {
                const t = s(e);
                return !!t.licenseKey && !!t.errorBeacon && !!t.applicationID;
              } catch (e) {
                return !1;
              }
            }(e);
          }
        },
        9567: (e, t, r) => {
          r.d(t, {
            D: () => i
          });
          var n = r(50);
          function i(e, t) {
            try {
              if (!e || "object" != typeof e) return (0, n.Z)("Setting a Configurable requires an object as input");
              if (!t || "object" != typeof t) return (0, n.Z)("Setting a Configurable requires a model to set its initial properties");
              const r = Object.create(Object.getPrototypeOf(t), Object.getOwnPropertyDescriptors(t)),
                o = 0 === Object.keys(r).length ? e : r;
              for (let a in o) if (void 0 !== e[a]) try {
                "object" == typeof e[a] && "object" == typeof t[a] ? r[a] = i(e[a], t[a]) : r[a] = e[a];
              } catch (e) {
                (0, n.Z)("An error occurred while setting a property of a Configurable", e);
              }
              return r;
            } catch (e) {
              (0, n.Z)("An error occured while setting a Configurable", e);
            }
          }
        },
        6818: (e, t, r) => {
          r.d(t, {
            Re: () => i,
            gF: () => o,
            q4: () => n
          });
          const n = "1.238.0",
            i = "PROD",
            o = "CDN";
        },
        385: (e, t, r) => {
          r.d(t, {
            FN: () => a,
            IF: () => u,
            Nk: () => l,
            Tt: () => s,
            _A: () => o,
            il: () => n,
            pL: () => c,
            v6: () => i,
            w1: () => d
          });
          const n = "undefined" != typeof window && !!window.document,
            i = "undefined" != typeof WorkerGlobalScope && ("undefined" != typeof self && self instanceof WorkerGlobalScope && self.navigator instanceof WorkerNavigator || "undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope && globalThis.navigator instanceof WorkerNavigator),
            o = n ? window : "undefined" != typeof WorkerGlobalScope && ("undefined" != typeof self && self instanceof WorkerGlobalScope && self || "undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope && globalThis),
            a = "" + o?.location,
            s = /iPad|iPhone|iPod/.test(navigator.userAgent),
            c = s && "undefined" == typeof SharedWorker,
            u = (() => {
              const e = navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);
              return Array.isArray(e) && e.length >= 2 ? +e[1] : 0;
            })(),
            d = Boolean(n && window.document.documentMode),
            l = !!navigator.sendBeacon;
        },
        1117: (e, t, r) => {
          r.d(t, {
            w: () => o
          });
          var n = r(50);
          const i = {
            agentIdentifier: "",
            ee: void 0
          };
          class o {
            constructor(e) {
              try {
                if ("object" != typeof e) return (0, n.Z)("shared context requires an object as input");
                this.sharedContext = {}, Object.assign(this.sharedContext, i), Object.entries(e).forEach(e => {
                  let [t, r] = e;
                  Object.keys(i).includes(t) && (this.sharedContext[t] = r);
                });
              } catch (e) {
                (0, n.Z)("An error occured while setting SharedContext", e);
              }
            }
          }
        },
        8e3: (e, t, r) => {
          r.d(t, {
            L: () => d,
            R: () => c
          });
          var n = r(8325),
            i = r(1284),
            o = r(4322),
            a = r(3325);
          const s = {};
          function c(e, t) {
            const r = {
              staged: !1,
              priority: a.p[t] || 0
            };
            u(e), s[e].get(t) || s[e].set(t, r);
          }
          function u(e) {
            e && (s[e] || (s[e] = new Map()));
          }
          function d() {
            let e = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : "",
              t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : "feature";
            if (u(e), !e || !s[e].get(t)) return a(t);
            s[e].get(t).staged = !0;
            const r = [...s[e]];
            function a(t) {
              const r = e ? n.ee.get(e) : n.ee,
                a = o.X.handlers;
              if (r.backlog && a) {
                var s = r.backlog[t],
                  c = a[t];
                if (c) {
                  for (var u = 0; s && u < s.length; ++u) l(s[u], c);
                  (0, i.D)(c, function (e, t) {
                    (0, i.D)(t, function (t, r) {
                      r[0].on(e, r[1]);
                    });
                  });
                }
                delete a[t], r.backlog[t] = null, r.emit("drain-" + t, []);
              }
            }
            r.every(e => {
              let [t, r] = e;
              return r.staged;
            }) && (r.sort((e, t) => e[1].priority - t[1].priority), r.forEach(e => {
              let [t] = e;
              a(t);
            }));
          }
          function l(e, t) {
            var r = e[1];
            (0, i.D)(t[r], function (t, r) {
              var n = e[0];
              if (r[0] === n) {
                var i = r[1],
                  o = e[3],
                  a = e[2];
                i.apply(o, a);
              }
            });
          }
        },
        8325: (e, t, r) => {
          r.d(t, {
            A: () => c,
            ee: () => u
          });
          var n = r(8632),
            i = r(2210),
            o = r(5763);
          class a {
            constructor(e) {
              this.contextId = e;
            }
          }
          var s = r(3117);
          const c = "nr@context:".concat(s.a),
            u = function e(t, r) {
              var n = {},
                s = {},
                d = {},
                f = !1;
              try {
                f = 16 === r.length && (0, o.OP)(r).isolatedBacklog;
              } catch (e) {}
              var h = {
                on: g,
                addEventListener: g,
                removeEventListener: function (e, t) {
                  var r = n[e];
                  if (!r) return;
                  for (var i = 0; i < r.length; i++) r[i] === t && r.splice(i, 1);
                },
                emit: function (e, r, n, i, o) {
                  !1 !== o && (o = !0);
                  if (u.aborted && !i) return;
                  t && o && t.emit(e, r, n);
                  for (var a = p(n), c = m(e), d = c.length, l = 0; l < d; l++) c[l].apply(a, r);
                  var f = b()[s[e]];
                  f && f.push([h, e, r, a]);
                  return a;
                },
                get: v,
                listeners: m,
                context: p,
                buffer: function (e, t) {
                  const r = b();
                  if (t = t || "feature", h.aborted) return;
                  Object.entries(e || {}).forEach(e => {
                    let [n, i] = e;
                    s[i] = t, t in r || (r[t] = []);
                  });
                },
                abort: l,
                aborted: !1,
                isBuffering: function (e) {
                  return !!b()[s[e]];
                },
                debugId: r,
                backlog: f ? {} : t && "object" == typeof t.backlog ? t.backlog : {}
              };
              return h;
              function p(e) {
                return e && e instanceof a ? e : e ? (0, i.X)(e, c, () => new a(c)) : new a(c);
              }
              function g(e, t) {
                n[e] = m(e).concat(t);
              }
              function m(e) {
                return n[e] || [];
              }
              function v(t) {
                return d[t] = d[t] || e(h, t);
              }
              function b() {
                return h.backlog;
              }
            }(void 0, "globalEE"),
            d = (0, n.fP)();
          function l() {
            u.aborted = !0, u.backlog = {};
          }
          d.ee || (d.ee = u);
        },
        5546: (e, t, r) => {
          r.d(t, {
            E: () => n,
            p: () => i
          });
          var n = r(8325).ee.get("handle");
          function i(e, t, r, i, o) {
            o ? (o.buffer([e], i), o.emit(e, t, r)) : (n.buffer([e], i), n.emit(e, t, r));
          }
        },
        4322: (e, t, r) => {
          r.d(t, {
            X: () => o
          });
          var n = r(5546);
          o.on = a;
          var i = o.handlers = {};
          function o(e, t, r, o) {
            a(o || n.E, i, e, t, r);
          }
          function a(e, t, r, i, o) {
            o || (o = "feature"), e || (e = n.E);
            var a = t[o] = t[o] || {};
            (a[r] = a[r] || []).push([e, i]);
          }
        },
        3239: (e, t, r) => {
          r.d(t, {
            bP: () => s,
            iz: () => c,
            m$: () => a
          });
          var n = r(385);
          let i = !1,
            o = !1;
          try {
            const e = {
              get passive() {
                return i = !0, !1;
              },
              get signal() {
                return o = !0, !1;
              }
            };
            n._A.addEventListener("test", null, e), n._A.removeEventListener("test", null, e);
          } catch (e) {}
          function a(e, t) {
            return i || o ? {
              capture: !!e,
              passive: i,
              signal: t
            } : !!e;
          }
          function s(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2],
              n = arguments.length > 3 ? arguments[3] : void 0;
            window.addEventListener(e, t, a(r, n));
          }
          function c(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2],
              n = arguments.length > 3 ? arguments[3] : void 0;
            document.addEventListener(e, t, a(r, n));
          }
        },
        3117: (e, t, r) => {
          r.d(t, {
            a: () => n
          });
          const n = (0, r(4402).Rl)();
        },
        4402: (e, t, r) => {
          r.d(t, {
            Ht: () => u,
            M: () => c,
            Rl: () => a,
            ky: () => s
          });
          var n = r(385);
          const i = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";
          function o(e, t) {
            return e ? 15 & e[t] : 16 * Math.random() | 0;
          }
          function a() {
            const e = n._A?.crypto || n._A?.msCrypto;
            let t,
              r = 0;
            return e && e.getRandomValues && (t = e.getRandomValues(new Uint8Array(31))), i.split("").map(e => "x" === e ? o(t, ++r).toString(16) : "y" === e ? (3 & o() | 8).toString(16) : e).join("");
          }
          function s(e) {
            const t = n._A?.crypto || n._A?.msCrypto;
            let r,
              i = 0;
            t && t.getRandomValues && (r = t.getRandomValues(new Uint8Array(31)));
            const a = [];
            for (var s = 0; s < e; s++) a.push(o(r, ++i).toString(16));
            return a.join("");
          }
          function c() {
            return s(16);
          }
          function u() {
            return s(32);
          }
        },
        7056: (e, t, r) => {
          r.d(t, {
            Bq: () => n,
            Hb: () => o,
            oD: () => i
          });
          const n = "NRBA",
            i = 144e5,
            o = 18e5;
        },
        7894: (e, t, r) => {
          function n() {
            return Math.round(performance.now());
          }
          r.d(t, {
            z: () => n
          });
        },
        7243: (e, t, r) => {
          r.d(t, {
            e: () => o
          });
          var n = r(385),
            i = {};
          function o(e) {
            if (e in i) return i[e];
            if (0 === (e || "").indexOf("data:")) return {
              protocol: "data"
            };
            let t;
            var r = n._A?.location,
              o = {};
            if (n.il) t = document.createElement("a"), t.href = e;else try {
              t = new URL(e, r.href);
            } catch (e) {
              return o;
            }
            o.port = t.port;
            var a = t.href.split("://");
            !o.port && a[1] && (o.port = a[1].split("/")[0].split("@").pop().split(":")[1]), o.port && "0" !== o.port || (o.port = "https" === a[0] ? "443" : "80"), o.hostname = t.hostname || r.hostname, o.pathname = t.pathname, o.protocol = a[0], "/" !== o.pathname.charAt(0) && (o.pathname = "/" + o.pathname);
            var s = !t.protocol || ":" === t.protocol || t.protocol === r.protocol,
              c = t.hostname === r.hostname && t.port === r.port;
            return o.sameOrigin = s && (!t.hostname || c), "/" === o.pathname && (i[e] = o), o;
          }
        },
        50: (e, t, r) => {
          function n(e, t) {
            "function" == typeof console.warn && (console.warn("New Relic: ".concat(e)), t && console.warn(t));
          }
          r.d(t, {
            Z: () => n
          });
        },
        2587: (e, t, r) => {
          r.d(t, {
            N: () => c,
            T: () => u
          });
          var n = r(8325),
            i = r(5546),
            o = r(8e3),
            a = r(3325);
          const s = {
            stn: [a.D.sessionTrace],
            err: [a.D.jserrors, a.D.metrics],
            ins: [a.D.pageAction],
            spa: [a.D.spa],
            sr: [a.D.sessionReplay, a.D.sessionTrace]
          };
          function c(e, t) {
            const r = n.ee.get(t);
            e && "object" == typeof e && (Object.entries(e).forEach(e => {
              let [t, n] = e;
              void 0 === u[t] && (s[t] ? s[t].forEach(e => {
                n ? (0, i.p)("feat-" + t, [], void 0, e, r) : (0, i.p)("block-" + t, [], void 0, e, r), (0, i.p)("rumresp-" + t, [Boolean(n)], void 0, e, r);
              }) : n && (0, i.p)("feat-" + t, [], void 0, void 0, r), u[t] = Boolean(n));
            }), Object.keys(s).forEach(e => {
              void 0 === u[e] && (s[e]?.forEach(t => (0, i.p)("rumresp-" + e, [!1], void 0, t, r)), u[e] = !1);
            }), (0, o.L)(t, a.D.pageViewEvent));
          }
          const u = {};
        },
        2210: (e, t, r) => {
          r.d(t, {
            X: () => i
          });
          var n = Object.prototype.hasOwnProperty;
          function i(e, t, r) {
            if (n.call(e, t)) return e[t];
            var i = r();
            if (Object.defineProperty && Object.keys) try {
              return Object.defineProperty(e, t, {
                value: i,
                writable: !0,
                enumerable: !1
              }), i;
            } catch (e) {}
            return e[t] = i, i;
          }
        },
        1284: (e, t, r) => {
          r.d(t, {
            D: () => n
          });
          const n = (e, t) => Object.entries(e || {}).map(e => {
            let [r, n] = e;
            return t(r, n);
          });
        },
        4351: (e, t, r) => {
          r.d(t, {
            P: () => o
          });
          var n = r(8325);
          const i = () => {
            const e = new WeakSet();
            return (t, r) => {
              if ("object" == typeof r && null !== r) {
                if (e.has(r)) return;
                e.add(r);
              }
              return r;
            };
          };
          function o(e) {
            try {
              return JSON.stringify(e, i());
            } catch (e) {
              try {
                n.ee.emit("internal-error", [e]);
              } catch (e) {}
            }
          }
        },
        3960: (e, t, r) => {
          r.d(t, {
            K: () => a,
            b: () => o
          });
          var n = r(3239);
          function i() {
            return "undefined" == typeof document || "complete" === document.readyState;
          }
          function o(e, t) {
            if (i()) return e();
            (0, n.bP)("load", e, t);
          }
          function a(e) {
            if (i()) return e();
            (0, n.iz)("DOMContentLoaded", e);
          }
        },
        8632: (e, t, r) => {
          r.d(t, {
            EZ: () => u,
            Qy: () => c,
            ce: () => o,
            fP: () => a,
            gG: () => d,
            mF: () => s
          });
          var n = r(7894),
            i = r(385);
          const o = {
            beacon: "bam.nr-data.net",
            errorBeacon: "bam.nr-data.net"
          };
          function a() {
            return i._A.NREUM || (i._A.NREUM = {}), void 0 === i._A.newrelic && (i._A.newrelic = i._A.NREUM), i._A.NREUM;
          }
          function s() {
            let e = a();
            return e.o || (e.o = {
              ST: i._A.setTimeout,
              SI: i._A.setImmediate,
              CT: i._A.clearTimeout,
              XHR: i._A.XMLHttpRequest,
              REQ: i._A.Request,
              EV: i._A.Event,
              PR: i._A.Promise,
              MO: i._A.MutationObserver,
              FETCH: i._A.fetch
            }), e;
          }
          function c(e, t, r) {
            let i = a();
            const o = i.initializedAgents || {},
              s = o[e] || {};
            return Object.keys(s).length || (s.initializedAt = {
              ms: (0, n.z)(),
              date: new Date()
            }), i.initializedAgents = {
              ...o,
              [e]: {
                ...s,
                [r]: t
              }
            }, i;
          }
          function u(e, t) {
            a()[e] = t;
          }
          function d() {
            return function () {
              let e = a();
              const t = e.info || {};
              e.info = {
                beacon: o.beacon,
                errorBeacon: o.errorBeacon,
                ...t
              };
            }(), function () {
              let e = a();
              const t = e.init || {};
              e.init = {
                ...t
              };
            }(), s(), function () {
              let e = a();
              const t = e.loader_config || {};
              e.loader_config = {
                ...t
              };
            }(), a();
          }
        },
        7956: (e, t, r) => {
          r.d(t, {
            N: () => i
          });
          var n = r(3239);
          function i(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] && arguments[1],
              r = arguments.length > 2 ? arguments[2] : void 0,
              i = arguments.length > 3 ? arguments[3] : void 0;
            return void (0, n.iz)("visibilitychange", function () {
              if (t) return void ("hidden" == document.visibilityState && e());
              e(document.visibilityState);
            }, r, i);
          }
        },
        1214: (e, t, r) => {
          r.d(t, {
            em: () => b,
            u5: () => j,
            QU: () => O,
            _L: () => I,
            Gm: () => H,
            Lg: () => L,
            BV: () => G,
            Kf: () => K
          });
          var n = r(8325),
            i = r(3117);
          const o = "nr@original:".concat(i.a);
          var a = Object.prototype.hasOwnProperty,
            s = !1;
          function c(e, t) {
            return e || (e = n.ee), r.inPlace = function (e, t, n, i, o) {
              n || (n = "");
              const a = "-" === n.charAt(0);
              for (let s = 0; s < t.length; s++) {
                const c = t[s],
                  u = e[c];
                d(u) || (e[c] = r(u, a ? c + n : n, i, c, o));
              }
            }, r.flag = o, r;
            function r(t, r, n, s, c) {
              return d(t) ? t : (r || (r = ""), nrWrapper[o] = t, function (e, t, r) {
                if (Object.defineProperty && Object.keys) try {
                  return Object.keys(e).forEach(function (r) {
                    Object.defineProperty(t, r, {
                      get: function () {
                        return e[r];
                      },
                      set: function (t) {
                        return e[r] = t, t;
                      }
                    });
                  }), t;
                } catch (e) {
                  u([e], r);
                }
                for (var n in e) a.call(e, n) && (t[n] = e[n]);
              }(t, nrWrapper, e), nrWrapper);
              function nrWrapper() {
                var o, a, d, l;
                try {
                  a = this, o = [...arguments], d = "function" == typeof n ? n(o, a) : n || {};
                } catch (t) {
                  u([t, "", [o, a, s], d], e);
                }
                i(r + "start", [o, a, s], d, c);
                try {
                  return l = t.apply(a, o);
                } catch (e) {
                  throw i(r + "err", [o, a, e], d, c), e;
                } finally {
                  i(r + "end", [o, a, l], d, c);
                }
              }
            }
            function i(r, n, i, o) {
              if (!s || t) {
                var a = s;
                s = !0;
                try {
                  e.emit(r, n, i, t, o);
                } catch (t) {
                  u([t, r, n, i], e);
                }
                s = a;
              }
            }
          }
          function u(e, t) {
            t || (t = n.ee);
            try {
              t.emit("internal-error", e);
            } catch (e) {}
          }
          function d(e) {
            return !(e && e instanceof Function && e.apply && !e[o]);
          }
          var l = r(2210),
            f = r(385);
          const h = {},
            p = f._A.XMLHttpRequest,
            g = "addEventListener",
            m = "removeEventListener",
            v = "nr@wrapped:".concat(n.A);
          function b(e) {
            var t = function (e) {
              return (e || n.ee).get("events");
            }(e);
            if (h[t.debugId]++) return t;
            h[t.debugId] = 1;
            var r = c(t, !0);
            function i(e) {
              r.inPlace(e, [g, m], "-", o);
            }
            function o(e, t) {
              return e[1];
            }
            return "getPrototypeOf" in Object && (f.il && y(document, i), y(f._A, i), y(p.prototype, i)), t.on(g + "-start", function (e, t) {
              var n = e[1];
              if (null !== n && ("function" == typeof n || "object" == typeof n)) {
                var i = (0, l.X)(n, v, function () {
                  var e = {
                    object: function () {
                      if ("function" != typeof n.handleEvent) return;
                      return n.handleEvent.apply(n, arguments);
                    },
                    function: n
                  }[typeof n];
                  return e ? r(e, "fn-", null, e.name || "anonymous") : n;
                });
                this.wrapped = e[1] = i;
              }
            }), t.on(m + "-start", function (e) {
              e[1] = this.wrapped || e[1];
            }), t;
          }
          function y(e, t) {
            let r = e;
            for (; "object" == typeof r && !Object.prototype.hasOwnProperty.call(r, g);) r = Object.getPrototypeOf(r);
            for (var n = arguments.length, i = new Array(n > 2 ? n - 2 : 0), o = 2; o < n; o++) i[o - 2] = arguments[o];
            r && t(r, ...i);
          }
          var w = "fetch-",
            A = w + "body-",
            x = ["arrayBuffer", "blob", "json", "text", "formData"],
            E = f._A.Request,
            T = f._A.Response,
            _ = "prototype";
          const D = {};
          function j(e) {
            const t = function (e) {
              return (e || n.ee).get("fetch");
            }(e);
            if (!(E && T && f._A.fetch)) return t;
            if (D[t.debugId]++) return t;
            function r(e, r, i) {
              var o = e[r];
              "function" == typeof o && (e[r] = function () {
                var e,
                  r = [...arguments],
                  a = {};
                t.emit(i + "before-start", [r], a), a[n.A] && a[n.A].dt && (e = a[n.A].dt);
                var s = o.apply(this, r);
                return t.emit(i + "start", [r, e], s), s.then(function (e) {
                  return t.emit(i + "end", [null, e], s), e;
                }, function (e) {
                  throw t.emit(i + "end", [e], s), e;
                });
              });
            }
            return D[t.debugId] = 1, x.forEach(e => {
              r(E[_], e, A), r(T[_], e, A);
            }), r(f._A, "fetch", w), t.on(w + "end", function (e, r) {
              var n = this;
              if (r) {
                var i = r.headers.get("content-length");
                null !== i && (n.rxSize = i), t.emit(w + "done", [null, r], n);
              } else t.emit(w + "done", [e], n);
            }), t;
          }
          const C = {},
            N = ["pushState", "replaceState"];
          function O(e) {
            const t = function (e) {
              return (e || n.ee).get("history");
            }(e);
            return !f.il || C[t.debugId]++ || (C[t.debugId] = 1, c(t).inPlace(window.history, N, "-")), t;
          }
          var S = r(3239);
          const P = {},
            R = ["appendChild", "insertBefore", "replaceChild"];
          function I(e) {
            const t = function (e) {
              return (e || n.ee).get("jsonp");
            }(e);
            if (!f.il || P[t.debugId]) return t;
            P[t.debugId] = !0;
            var r = c(t),
              i = /[?&](?:callback|cb)=([^&#]+)/,
              o = /(.*)\.([^.]+)/,
              a = /^(\w+)(\.|$)(.*)$/;
            function s(e, t) {
              if (!e) return t;
              const r = e.match(a),
                n = r[1];
              return s(r[3], t[n]);
            }
            return r.inPlace(Node.prototype, R, "dom-"), t.on("dom-start", function (e) {
              !function (e) {
                if (!e || "string" != typeof e.nodeName || "script" !== e.nodeName.toLowerCase()) return;
                if ("function" != typeof e.addEventListener) return;
                var n = (a = e.src, c = a.match(i), c ? c[1] : null);
                var a, c;
                if (!n) return;
                var u = function (e) {
                  var t = e.match(o);
                  if (t && t.length >= 3) return {
                    key: t[2],
                    parent: s(t[1], window)
                  };
                  return {
                    key: e,
                    parent: window
                  };
                }(n);
                if ("function" != typeof u.parent[u.key]) return;
                var d = {};
                function l() {
                  t.emit("jsonp-end", [], d), e.removeEventListener("load", l, (0, S.m$)(!1)), e.removeEventListener("error", f, (0, S.m$)(!1));
                }
                function f() {
                  t.emit("jsonp-error", [], d), t.emit("jsonp-end", [], d), e.removeEventListener("load", l, (0, S.m$)(!1)), e.removeEventListener("error", f, (0, S.m$)(!1));
                }
                r.inPlace(u.parent, [u.key], "cb-", d), e.addEventListener("load", l, (0, S.m$)(!1)), e.addEventListener("error", f, (0, S.m$)(!1)), t.emit("new-jsonp", [e.src], d);
              }(e[0]);
            }), t;
          }
          const k = {};
          function H(e) {
            const t = function (e) {
              return (e || n.ee).get("mutation");
            }(e);
            if (!f.il || k[t.debugId]) return t;
            k[t.debugId] = !0;
            var r = c(t),
              i = f._A.MutationObserver;
            return i && (window.MutationObserver = function (e) {
              return this instanceof i ? new i(r(e, "fn-")) : i.apply(this, arguments);
            }, MutationObserver.prototype = i.prototype), t;
          }
          const z = {};
          function L(e) {
            const t = function (e) {
              return (e || n.ee).get("promise");
            }(e);
            if (z[t.debugId]) return t;
            z[t.debugId] = !0;
            var r = t.context,
              i = c(t),
              a = f._A.Promise;
            return a && function () {
              function e(r) {
                var n = t.context(),
                  o = i(r, "executor-", n, null, !1);
                const s = Reflect.construct(a, [o], e);
                return t.context(s).getCtx = function () {
                  return n;
                }, s;
              }
              f._A.Promise = e, Object.defineProperty(e, "name", {
                value: "Promise"
              }), e.toString = function () {
                return a.toString();
              }, Object.setPrototypeOf(e, a), ["all", "race"].forEach(function (r) {
                const n = a[r];
                e[r] = function (e) {
                  let i = !1;
                  [...(e || [])].forEach(e => {
                    this.resolve(e).then(a("all" === r), a(!1));
                  });
                  const o = n.apply(this, arguments);
                  return o;
                  function a(e) {
                    return function () {
                      t.emit("propagate", [null, !i], o, !1, !1), i = i || !e;
                    };
                  }
                };
              }), ["resolve", "reject"].forEach(function (r) {
                const n = a[r];
                e[r] = function (e) {
                  const r = n.apply(this, arguments);
                  return e !== r && t.emit("propagate", [e, !0], r, !1, !1), r;
                };
              }), e.prototype = a.prototype;
              const n = a.prototype.then;
              a.prototype.then = function () {
                var e = this,
                  o = r(e);
                o.promise = e;
                for (var a = arguments.length, s = new Array(a), c = 0; c < a; c++) s[c] = arguments[c];
                s[0] = i(s[0], "cb-", o, null, !1), s[1] = i(s[1], "cb-", o, null, !1);
                const u = n.apply(this, s);
                return o.nextPromise = u, t.emit("propagate", [e, !0], u, !1, !1), u;
              }, a.prototype.then[o] = n, t.on("executor-start", function (e) {
                e[0] = i(e[0], "resolve-", this, null, !1), e[1] = i(e[1], "resolve-", this, null, !1);
              }), t.on("executor-err", function (e, t, r) {
                e[1](r);
              }), t.on("cb-end", function (e, r, n) {
                t.emit("propagate", [n, !0], this.nextPromise, !1, !1);
              }), t.on("propagate", function (e, r, n) {
                this.getCtx && !r || (this.getCtx = function () {
                  if (e instanceof Promise) var r = t.context(e);
                  return r && r.getCtx ? r.getCtx() : this;
                });
              });
            }(), t;
          }
          const M = {},
            B = "setTimeout",
            F = "setInterval",
            U = "clearTimeout",
            q = "-start",
            Z = "-",
            V = [B, "setImmediate", F, U, "clearImmediate"];
          function G(e) {
            const t = function (e) {
              return (e || n.ee).get("timer");
            }(e);
            if (M[t.debugId]++) return t;
            M[t.debugId] = 1;
            var r = c(t);
            return r.inPlace(f._A, V.slice(0, 2), B + Z), r.inPlace(f._A, V.slice(2, 3), F + Z), r.inPlace(f._A, V.slice(3), U + Z), t.on(F + q, function (e, t, n) {
              e[0] = r(e[0], "fn-", null, n);
            }), t.on(B + q, function (e, t, n) {
              this.method = n, this.timerDuration = isNaN(e[1]) ? 0 : +e[1], e[0] = r(e[0], "fn-", this, n);
            }), t;
          }
          var W = r(50);
          const X = {},
            Q = ["open", "send"];
          function K(e) {
            var t = e || n.ee;
            const r = function (e) {
              return (e || n.ee).get("xhr");
            }(t);
            if (X[r.debugId]++) return r;
            X[r.debugId] = 1, b(t);
            var i = c(r),
              o = f._A.XMLHttpRequest,
              a = f._A.MutationObserver,
              s = f._A.Promise,
              u = f._A.setInterval,
              d = "readystatechange",
              l = ["onload", "onerror", "onabort", "onloadstart", "onloadend", "onprogress", "ontimeout"],
              h = [],
              p = f._A.XMLHttpRequest = function (e) {
                const t = new o(e),
                  n = r.context(t);
                try {
                  r.emit("new-xhr", [t], n), t.addEventListener(d, (a = n, function () {
                    var e = this;
                    e.readyState > 3 && !a.resolved && (a.resolved = !0, r.emit("xhr-resolved", [], e)), i.inPlace(e, l, "fn-", A);
                  }), (0, S.m$)(!1));
                } catch (e) {
                  (0, W.Z)("An error occurred while intercepting XHR", e);
                  try {
                    r.emit("internal-error", [e]);
                  } catch (e) {}
                }
                var a;
                return t;
              };
            function g(e, t) {
              i.inPlace(t, ["onreadystatechange"], "fn-", A);
            }
            if (function (e, t) {
              for (var r in e) t[r] = e[r];
            }(o, p), p.prototype = o.prototype, i.inPlace(p.prototype, Q, "-xhr-", A), r.on("send-xhr-start", function (e, t) {
              g(e, t), function (e) {
                h.push(e), a && (m ? m.then(w) : u ? u(w) : (v = -v, y.data = v));
              }(t);
            }), r.on("open-xhr-start", g), a) {
              var m = s && s.resolve();
              if (!u && !s) {
                var v = 1,
                  y = document.createTextNode(v);
                new a(w).observe(y, {
                  characterData: !0
                });
              }
            } else t.on("fn-end", function (e) {
              e[0] && e[0].type === d || w();
            });
            function w() {
              for (var e = 0; e < h.length; e++) g(0, h[e]);
              h.length && (h = []);
            }
            function A(e, t) {
              return t;
            }
            return r;
          }
        },
        7825: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.ajax;
        },
        6660: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.jserrors;
        },
        3081: (e, t, r) => {
          r.d(t, {
            gF: () => o,
            mY: () => i,
            t9: () => n,
            vz: () => s,
            xS: () => a
          });
          const n = r(3325).D.metrics,
            i = "sm",
            o = "cm",
            a = "storeSupportabilityMetrics",
            s = "storeEventMetrics";
        },
        4649: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.pageAction;
        },
        7633: (e, t, r) => {
          r.d(t, {
            Dz: () => i,
            OJ: () => a,
            qw: () => o,
            t9: () => n
          });
          const n = r(3325).D.pageViewEvent,
            i = "firstbyte",
            o = "domcontent",
            a = "windowload";
        },
        9251: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.pageViewTiming;
        },
        3614: (e, t, r) => {
          r.d(t, {
            BST_RESOURCE: () => i,
            END: () => s,
            FEATURE_NAME: () => n,
            FN_END: () => u,
            FN_START: () => c,
            PUSH_STATE: () => d,
            RESOURCE: () => o,
            START: () => a
          });
          const n = r(3325).D.sessionTrace,
            i = "bstResource",
            o = "resource",
            a = "-start",
            s = "-end",
            c = "fn" + a,
            u = "fn" + s,
            d = "pushState";
        },
        7836: (e, t, r) => {
          r.d(t, {
            BODY: () => x,
            CB_END: () => E,
            CB_START: () => u,
            END: () => A,
            FEATURE_NAME: () => i,
            FETCH: () => _,
            FETCH_BODY: () => v,
            FETCH_DONE: () => m,
            FETCH_START: () => g,
            FN_END: () => c,
            FN_START: () => s,
            INTERACTION: () => f,
            INTERACTION_API: () => d,
            INTERACTION_EVENTS: () => o,
            JSONP_END: () => b,
            JSONP_NODE: () => p,
            JS_TIME: () => T,
            MAX_TIMER_BUDGET: () => a,
            REMAINING: () => l,
            SPA_NODE: () => h,
            START: () => w,
            originalSetTimeout: () => y
          });
          var n = r(5763);
          const i = r(3325).D.spa,
            o = ["click", "submit", "keypress", "keydown", "keyup", "change"],
            a = 999,
            s = "fn-start",
            c = "fn-end",
            u = "cb-start",
            d = "api-ixn-",
            l = "remaining",
            f = "interaction",
            h = "spaNode",
            p = "jsonpNode",
            g = "fetch-start",
            m = "fetch-done",
            v = "fetch-body-",
            b = "jsonp-end",
            y = n.Yu.ST,
            w = "-start",
            A = "-end",
            x = "-body",
            E = "cb" + A,
            T = "jsTime",
            _ = "fetch";
        },
        5938: (e, t, r) => {
          r.d(t, {
            W: () => o
          });
          var n = r(5763),
            i = r(8325);
          class o {
            constructor(e, t, r) {
              this.agentIdentifier = e, this.aggregator = t, this.ee = i.ee.get(e, (0, n.OP)(this.agentIdentifier).isolatedBacklog), this.featureName = r, this.blocked = !1;
            }
          }
        },
        9144: (e, t, r) => {
          r.d(t, {
            j: () => m
          });
          var n = r(3325),
            i = r(5763),
            o = r(5546),
            a = r(8325),
            s = r(7894),
            c = r(8e3),
            u = r(3960),
            d = r(385),
            l = r(50),
            f = r(3081),
            h = r(8632);
          function p() {
            const e = (0, h.gG)();
            ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease", "addPageAction", "setCurrentRouteName", "setPageViewName", "setCustomAttribute", "interaction", "noticeError", "setUserId", "setApplicationVersion"].forEach(t => {
              e[t] = function () {
                for (var r = arguments.length, n = new Array(r), i = 0; i < r; i++) n[i] = arguments[i];
                return function (t) {
                  for (var r = arguments.length, n = new Array(r > 1 ? r - 1 : 0), i = 1; i < r; i++) n[i - 1] = arguments[i];
                  let o = [];
                  return Object.values(e.initializedAgents).forEach(e => {
                    e.exposed && e.api[t] && o.push(e.api[t](...n));
                  }), o.length > 1 ? o : o[0];
                }(t, ...n);
              };
            });
          }
          var g = r(2587);
          function m(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : {},
              m = arguments.length > 2 ? arguments[2] : void 0,
              v = arguments.length > 3 ? arguments[3] : void 0,
              {
                init: b,
                info: y,
                loader_config: w,
                runtime: A = {
                  loaderType: m
                },
                exposed: x = !0
              } = t;
            const E = (0, h.gG)();
            y || (b = E.init, y = E.info, w = E.loader_config), (0, i.Dg)(e, b || {}), (0, i.GE)(e, w || {}), y.jsAttributes ??= {}, d.v6 && (y.jsAttributes.isWorker = !0), (0, i.CX)(e, y);
            const T = (0, i.P_)(e);
            A.denyList = [...(T.ajax?.deny_list || []), ...(T.ajax?.block_internal ? [y.beacon, y.errorBeacon] : [])], (0, i.sU)(e, A), p();
            const _ = function (e, t) {
              t || (0, c.R)(e, "api");
              const h = {};
              var p = a.ee.get(e),
                g = p.get("tracer"),
                m = "api-",
                v = m + "ixn-";
              function b(t, r, n, o) {
                const a = (0, i.C5)(e);
                return null === r ? delete a.jsAttributes[t] : (0, i.CX)(e, {
                  ...a,
                  jsAttributes: {
                    ...a.jsAttributes,
                    [t]: r
                  }
                }), A(m, n, !0, o || null === r ? "session" : void 0)(t, r);
              }
              function y() {}
              ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease"].forEach(e => h[e] = A(m, e, !0, "api")), h.addPageAction = A(m, "addPageAction", !0, n.D.pageAction), h.setCurrentRouteName = A(m, "routeName", !0, n.D.spa), h.setPageViewName = function (t, r) {
                if ("string" == typeof t) return "/" !== t.charAt(0) && (t = "/" + t), (0, i.OP)(e).customTransaction = (r || "http://custom.transaction") + t, A(m, "setPageViewName", !0)();
              }, h.setCustomAttribute = function (e, t) {
                let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2];
                if ("string" == typeof e) {
                  if (["string", "number"].includes(typeof t) || null === t) return b(e, t, "setCustomAttribute", r);
                  (0, l.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t, "> was provided."));
                } else (0, l.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e, "> was provided."));
              }, h.setUserId = function (e) {
                if ("string" == typeof e || null === e) return b("enduser.id", e, "setUserId", !0);
                (0, l.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e, "> was provided."));
              }, h.setApplicationVersion = function (e) {
                if ("string" == typeof e || null === e) return b("application.version", e, "setApplicationVersion", !1);
                (0, l.Z)("Failed to execute setApplicationVersion. Expected <String | null>, but got <".concat(typeof e, ">."));
              }, h.interaction = function () {
                return new y().get();
              };
              var w = y.prototype = {
                createTracer: function (e, t) {
                  var r = {},
                    i = this,
                    a = "function" == typeof t;
                  return (0, o.p)(v + "tracer", [(0, s.z)(), e, r], i, n.D.spa, p), function () {
                    if (g.emit((a ? "" : "no-") + "fn-start", [(0, s.z)(), i, a], r), a) try {
                      return t.apply(this, arguments);
                    } catch (e) {
                      throw g.emit("fn-err", [arguments, this, e], r), e;
                    } finally {
                      g.emit("fn-end", [(0, s.z)()], r);
                    }
                  };
                }
              };
              function A(e, t, r, i) {
                return function () {
                  return (0, o.p)(f.xS, ["API/" + t + "/called"], void 0, n.D.metrics, p), i && (0, o.p)(e + t, [(0, s.z)(), ...arguments], r ? null : this, i, p), r ? void 0 : this;
                };
              }
              function x() {
                r.e(111).then(r.bind(r, 7438)).then(t => {
                  let {
                    setAPI: r
                  } = t;
                  r(e), (0, c.L)(e, "api");
                }).catch(() => (0, l.Z)("Downloading runtime APIs failed..."));
              }
              return ["actionText", "setName", "setAttribute", "save", "ignore", "onEnd", "getContext", "end", "get"].forEach(e => {
                w[e] = A(v, e, void 0, n.D.spa);
              }), h.noticeError = function (e, t) {
                "string" == typeof e && (e = new Error(e)), (0, o.p)(f.xS, ["API/noticeError/called"], void 0, n.D.metrics, p), (0, o.p)("err", [e, (0, s.z)(), !1, t], void 0, n.D.jserrors, p);
              }, d.il ? (0, u.b)(() => x(), !0) : x(), h;
            }(e, v);
            return (0, h.Qy)(e, _, "api"), (0, h.Qy)(e, x, "exposed"), (0, h.EZ)("activatedFeatures", g.T), _;
          }
        },
        3325: (e, t, r) => {
          r.d(t, {
            D: () => n,
            p: () => i
          });
          const n = {
              ajax: "ajax",
              jserrors: "jserrors",
              metrics: "metrics",
              pageAction: "page_action",
              pageViewEvent: "page_view_event",
              pageViewTiming: "page_view_timing",
              sessionReplay: "session_replay",
              sessionTrace: "session_trace",
              spa: "spa"
            },
            i = {
              [n.pageViewEvent]: 1,
              [n.pageViewTiming]: 2,
              [n.metrics]: 3,
              [n.jserrors]: 4,
              [n.ajax]: 5,
              [n.sessionTrace]: 6,
              [n.pageAction]: 7,
              [n.spa]: 8,
              [n.sessionReplay]: 9
            };
        }
      },
      n = {};
    function i(e) {
      var t = n[e];
      if (void 0 !== t) return t.exports;
      var o = n[e] = {
        exports: {}
      };
      return r[e](o, o.exports, i), o.exports;
    }
    i.m = r, i.d = (e, t) => {
      for (var r in t) i.o(t, r) && !i.o(e, r) && Object.defineProperty(e, r, {
        enumerable: !0,
        get: t[r]
      });
    }, i.f = {}, i.e = e => Promise.all(Object.keys(i.f).reduce((t, r) => (i.f[r](e, t), t), [])), i.u = e => "nr-spa.1097a448-1.238.0.min.js", i.o = (e, t) => Object.prototype.hasOwnProperty.call(e, t), e = {}, t = "NRBA-1.238.0.PROD:", i.l = (r, n, o, a) => {
      if (e[r]) e[r].push(n);else {
        var s, c;
        if (void 0 !== o) for (var u = document.getElementsByTagName("script"), d = 0; d < u.length; d++) {
          var l = u[d];
          if (l.getAttribute("src") == r || l.getAttribute("data-webpack") == t + o) {
            s = l;
            break;
          }
        }
        s || (c = !0, (s = document.createElement("script")).charset = "utf-8", s.timeout = 120, i.nc && s.setAttribute("nonce", i.nc), s.setAttribute("data-webpack", t + o), s.src = r), e[r] = [n];
        var f = (t, n) => {
            s.onerror = s.onload = null, clearTimeout(h);
            var i = e[r];
            if (delete e[r], s.parentNode && s.parentNode.removeChild(s), i && i.forEach(e => e(n)), t) return t(n);
          },
          h = setTimeout(f.bind(null, void 0, {
            type: "timeout",
            target: s
          }), 12e4);
        s.onerror = f.bind(null, s.onerror), s.onload = f.bind(null, s.onload), c && document.head.appendChild(s);
      }
    }, i.r = e => {
      "undefined" != typeof Symbol && Symbol.toStringTag && Object.defineProperty(e, Symbol.toStringTag, {
        value: "Module"
      }), Object.defineProperty(e, "__esModule", {
        value: !0
      });
    }, i.p = "https://js-agent.newrelic.com/", (() => {
      var e = {
        801: 0,
        92: 0
      };
      i.f.j = (t, r) => {
        var n = i.o(e, t) ? e[t] : void 0;
        if (0 !== n) if (n) r.push(n[2]);else {
          var o = new Promise((r, i) => n = e[t] = [r, i]);
          r.push(n[2] = o);
          var a = i.p + i.u(t),
            s = new Error();
          i.l(a, r => {
            if (i.o(e, t) && (0 !== (n = e[t]) && (e[t] = void 0), n)) {
              var o = r && ("load" === r.type ? "missing" : r.type),
                a = r && r.target && r.target.src;
              s.message = "Loading chunk " + t + " failed.\n(" + o + ": " + a + ")", s.name = "ChunkLoadError", s.type = o, s.request = a, n[1](s);
            }
          }, "chunk-" + t, t);
        }
      };
      var t = (t, r) => {
          var n,
            o,
            [a, s, c] = r,
            u = 0;
          if (a.some(t => 0 !== e[t])) {
            for (n in s) i.o(s, n) && (i.m[n] = s[n]);
            if (c) c(i);
          }
          for (t && t(r); u < a.length; u++) o = a[u], i.o(e, o) && e[o] && e[o][0](), e[o] = 0;
        },
        r = self["webpackChunk:NRBA-1.238.0.PROD"] = self["webpackChunk:NRBA-1.238.0.PROD"] || [];
      r.forEach(t.bind(null, 0)), r.push = t.bind(null, r.push.bind(r));
    })(), (() => {
      var e = i(50);
      class t {
        addPageAction(t, r) {
          (0, e.Z)("Call to agent api addPageAction failed. The session trace feature is not currently initialized.");
        }
        setPageViewName(t, r) {
          (0, e.Z)("Call to agent api setPageViewName failed. The page view feature is not currently initialized.");
        }
        setCustomAttribute(t, r, n) {
          (0, e.Z)("Call to agent api setCustomAttribute failed. The js errors feature is not currently initialized.");
        }
        noticeError(t, r) {
          (0, e.Z)("Call to agent api noticeError failed. The js errors feature is not currently initialized.");
        }
        setUserId(t) {
          (0, e.Z)("Call to agent api setUserId failed. The js errors feature is not currently initialized.");
        }
        setApplicationVersion(t) {
          (0, e.Z)("Call to agent api setApplicationVersion failed. The agent is not currently initialized.");
        }
        setErrorHandler(t) {
          (0, e.Z)("Call to agent api setErrorHandler failed. The js errors feature is not currently initialized.");
        }
        finished(t) {
          (0, e.Z)("Call to agent api finished failed. The page action feature is not currently initialized.");
        }
        addRelease(t, r) {
          (0, e.Z)("Call to agent api addRelease failed. The agent is not currently initialized.");
        }
      }
      var r = i(3325),
        n = i(5763);
      const o = Object.values(r.D);
      function a(e) {
        const t = {};
        return o.forEach(r => {
          t[r] = function (e, t) {
            return !1 !== (0, n.Mt)(t, "".concat(e, ".enabled"));
          }(r, e);
        }), t;
      }
      var s = i(9144);
      var c = i(5546),
        u = i(385),
        d = i(8e3),
        l = i(5938),
        f = i(3960);
      class h extends l.W {
        constructor(e, t, r) {
          let n = !(arguments.length > 3 && void 0 !== arguments[3]) || arguments[3];
          super(e, t, r), this.auto = n, this.abortHandler, this.featAggregate, this.onAggregateImported, n && (0, d.R)(e, r);
        }
        importAggregator() {
          let t = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : {};
          if (this.featAggregate || !this.auto) return;
          const r = u.il && !0 === (0, n.Mt)(this.agentIdentifier, "privacy.cookies_enabled");
          let o;
          this.onAggregateImported = new Promise(e => {
            o = e;
          });
          const a = async () => {
            let n;
            try {
              if (r) {
                const {
                  setupAgentSession: e
                } = await i.e(111).then(i.bind(i, 3228));
                n = e(this.agentIdentifier);
              }
            } catch (t) {
              (0, e.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.", t);
            }
            try {
              if (!this.shouldImportAgg(this.featureName, n)) return (0, d.L)(this.agentIdentifier, this.featureName), void o(!1);
              const {
                  lazyFeatureLoader: e
                } = await i.e(111).then(i.bind(i, 8582)),
                {
                  Aggregate: r
                } = await e(this.featureName, "aggregate");
              this.featAggregate = new r(this.agentIdentifier, this.aggregator, t), o(!0);
            } catch (t) {
              (0, e.Z)("Downloading and initializing ".concat(this.featureName, " failed..."), t), this.abortHandler?.(), o(!1);
            }
          };
          u.il ? (0, f.b)(() => a(), !0) : a();
        }
        shouldImportAgg(e, t) {
          return e !== r.D.sessionReplay || !!n.Yu.MO && !1 !== (0, n.Mt)(this.agentIdentifier, "session_trace.enabled") && (!!t?.isNew || !!t?.state.sessionReplay);
        }
      }
      var p = i(7633),
        g = i(7894);
      class m extends h {
        static featureName = p.t9;
        constructor(e, t) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          if (super(e, t, p.t9, i), ("undefined" == typeof PerformanceNavigationTiming || u.Tt) && "undefined" != typeof PerformanceTiming) {
            const t = (0, n.OP)(e);
            t[p.Dz] = Math.max(Date.now() - t.offset, 0), (0, f.K)(() => t[p.qw] = Math.max((0, g.z)() - t[p.Dz], 0)), (0, f.b)(() => {
              const e = (0, g.z)();
              t[p.OJ] = Math.max(e - t[p.Dz], 0), (0, c.p)("timing", ["load", e], void 0, r.D.pageViewTiming, this.ee);
            });
          }
          this.importAggregator();
        }
      }
      var v = i(1117),
        b = i(1284);
      class y extends v.w {
        constructor(e) {
          super(e), this.aggregatedData = {};
        }
        store(e, t, r, n, i) {
          var o = this.getBucket(e, t, r, i);
          return o.metrics = function (e, t) {
            t || (t = {
              count: 0
            });
            return t.count += 1, (0, b.D)(e, function (e, r) {
              t[e] = w(r, t[e]);
            }), t;
          }(n, o.metrics), o;
        }
        merge(e, t, r, n, i) {
          var o = this.getBucket(e, t, n, i);
          if (o.metrics) {
            var a = o.metrics;
            a.count += r.count, (0, b.D)(r, function (e, t) {
              if ("count" !== e) {
                var n = a[e],
                  i = r[e];
                i && !i.c ? a[e] = w(i.t, n) : a[e] = function (e, t) {
                  if (!t) return e;
                  t.c || (t = A(t.t));
                  return t.min = Math.min(e.min, t.min), t.max = Math.max(e.max, t.max), t.t += e.t, t.sos += e.sos, t.c += e.c, t;
                }(i, a[e]);
              }
            });
          } else o.metrics = r;
        }
        storeMetric(e, t, r, n) {
          var i = this.getBucket(e, t, r);
          return i.stats = w(n, i.stats), i;
        }
        getBucket(e, t, r, n) {
          this.aggregatedData[e] || (this.aggregatedData[e] = {});
          var i = this.aggregatedData[e][t];
          return i || (i = this.aggregatedData[e][t] = {
            params: r || {}
          }, n && (i.custom = n)), i;
        }
        get(e, t) {
          return t ? this.aggregatedData[e] && this.aggregatedData[e][t] : this.aggregatedData[e];
        }
        take(e) {
          for (var t = {}, r = "", n = !1, i = 0; i < e.length; i++) t[r = e[i]] = x(this.aggregatedData[r]), t[r].length && (n = !0), delete this.aggregatedData[r];
          return n ? t : null;
        }
      }
      function w(e, t) {
        return null == e ? function (e) {
          e ? e.c++ : e = {
            c: 1
          };
          return e;
        }(t) : t ? (t.c || (t = A(t.t)), t.c += 1, t.t += e, t.sos += e * e, e > t.max && (t.max = e), e < t.min && (t.min = e), t) : {
          t: e
        };
      }
      function A(e) {
        return {
          t: e,
          min: e,
          max: e,
          sos: e * e,
          c: 1
        };
      }
      function x(e) {
        return "object" != typeof e ? [] : (0, b.D)(e, E);
      }
      function E(e, t) {
        return t;
      }
      var T = i(8632),
        _ = i(4402),
        D = i(4351);
      var j = i(7956),
        C = i(3239),
        N = i(9251);
      class O extends h {
        static featureName = N.t;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, N.t, r), u.il && ((0, n.OP)(e).initHidden = Boolean("hidden" === document.visibilityState), (0, j.N)(() => (0, c.p)("docHidden", [(0, g.z)()], void 0, N.t, this.ee), !0), (0, C.bP)("pagehide", () => (0, c.p)("winPagehide", [(0, g.z)()], void 0, N.t, this.ee)), this.importAggregator());
        }
      }
      var S = i(3081);
      class P extends h {
        static featureName = S.t9;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, S.t9, r), this.importAggregator();
        }
      }
      var R = i(6660);
      class I {
        constructor(e, t, r, n) {
          this.name = "UncaughtError", this.message = e, this.sourceURL = t, this.line = r, this.column = n;
        }
      }
      class k extends h {
        static featureName = R.t;
        #e = new Set();
        constructor(e, t) {
          let n = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, R.t, n);
          try {
            this.removeOnAbort = new AbortController();
          } catch (e) {}
          this.ee.on("fn-err", (e, t, n) => {
            this.abortHandler && !this.#e.has(n) && (this.#e.add(n), (0, c.p)("err", [this.#t(n), (0, g.z)()], void 0, r.D.jserrors, this.ee));
          }), this.ee.on("internal-error", e => {
            this.abortHandler && (0, c.p)("ierr", [this.#t(e), (0, g.z)(), !0], void 0, r.D.jserrors, this.ee);
          }), u._A.addEventListener("unhandledrejection", e => {
            this.abortHandler && (0, c.p)("err", [this.#r(e), (0, g.z)(), !1, {
              unhandledPromiseRejection: 1
            }], void 0, r.D.jserrors, this.ee);
          }, (0, C.m$)(!1, this.removeOnAbort?.signal)), u._A.addEventListener("error", e => {
            this.abortHandler && (this.#e.has(e.error) ? this.#e.delete(e.error) : (0, c.p)("err", [this.#n(e), (0, g.z)()], void 0, r.D.jserrors, this.ee));
          }, (0, C.m$)(!1, this.removeOnAbort?.signal)), this.abortHandler = this.#i, this.importAggregator();
        }
        #i() {
          this.removeOnAbort?.abort(), this.#e.clear(), this.abortHandler = void 0;
        }
        #t(e) {
          return e instanceof Error ? e : void 0 !== e?.message ? new I(e.message, e.filename || e.sourceURL, e.lineno || e.line, e.colno || e.col) : new I("string" == typeof e ? e : (0, D.P)(e));
        }
        #r(e) {
          let t = "Unhandled Promise Rejection: ";
          if (e?.reason instanceof Error) try {
            return e.reason.message = t + e.reason.message, e.reason;
          } catch (t) {
            return e.reason;
          }
          if (void 0 === e.reason) return new I(t);
          const r = this.#t(e.reason);
          return r.message = t + r.message, r;
        }
        #n(e) {
          return e.error instanceof Error ? e.error : new I(e.message, e.filename, e.lineno, e.colno);
        }
      }
      var H = i(2210);
      let z = 1;
      const L = "nr@id";
      function M(e) {
        const t = typeof e;
        return !e || "object" !== t && "function" !== t ? -1 : e === u._A ? 0 : (0, H.X)(e, L, function () {
          return z++;
        });
      }
      function B(e) {
        if ("string" == typeof e && e.length) return e.length;
        if ("object" == typeof e) {
          if ("undefined" != typeof ArrayBuffer && e instanceof ArrayBuffer && e.byteLength) return e.byteLength;
          if ("undefined" != typeof Blob && e instanceof Blob && e.size) return e.size;
          if (!("undefined" != typeof FormData && e instanceof FormData)) try {
            return (0, D.P)(e).length;
          } catch (e) {
            return;
          }
        }
      }
      var F = i(1214),
        U = i(7243);
      class q {
        constructor(e) {
          this.agentIdentifier = e;
        }
        generateTracePayload(e) {
          if (!this.shouldGenerateTrace(e)) return null;
          var t = (0, n.DL)(this.agentIdentifier);
          if (!t) return null;
          var r = (t.accountID || "").toString() || null,
            i = (t.agentID || "").toString() || null,
            o = (t.trustKey || "").toString() || null;
          if (!r || !i) return null;
          var a = (0, _.M)(),
            s = (0, _.Ht)(),
            c = Date.now(),
            u = {
              spanId: a,
              traceId: s,
              timestamp: c
            };
          return (e.sameOrigin || this.isAllowedOrigin(e) && this.useTraceContextHeadersForCors()) && (u.traceContextParentHeader = this.generateTraceContextParentHeader(a, s), u.traceContextStateHeader = this.generateTraceContextStateHeader(a, c, r, i, o)), (e.sameOrigin && !this.excludeNewrelicHeader() || !e.sameOrigin && this.isAllowedOrigin(e) && this.useNewrelicHeaderForCors()) && (u.newrelicHeader = this.generateTraceHeader(a, s, c, r, i, o)), u;
        }
        generateTraceContextParentHeader(e, t) {
          return "00-" + t + "-" + e + "-01";
        }
        generateTraceContextStateHeader(e, t, r, n, i) {
          return i + "@nr=0-1-" + r + "-" + n + "-" + e + "----" + t;
        }
        generateTraceHeader(e, t, r, n, i, o) {
          if (!("function" == typeof u._A?.btoa)) return null;
          var a = {
            v: [0, 1],
            d: {
              ty: "Browser",
              ac: n,
              ap: i,
              id: e,
              tr: t,
              ti: r
            }
          };
          return o && n !== o && (a.d.tk = o), btoa((0, D.P)(a));
        }
        shouldGenerateTrace(e) {
          return this.isDtEnabled() && this.isAllowedOrigin(e);
        }
        isAllowedOrigin(e) {
          var t = !1,
            r = {};
          if ((0, n.Mt)(this.agentIdentifier, "distributed_tracing") && (r = (0, n.P_)(this.agentIdentifier).distributed_tracing), e.sameOrigin) t = !0;else if (r.allowed_origins instanceof Array) for (var i = 0; i < r.allowed_origins.length; i++) {
            var o = (0, U.e)(r.allowed_origins[i]);
            if (e.hostname === o.hostname && e.protocol === o.protocol && e.port === o.port) {
              t = !0;
              break;
            }
          }
          return t;
        }
        isDtEnabled() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.enabled;
        }
        excludeNewrelicHeader() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.exclude_newrelic_header;
        }
        useNewrelicHeaderForCors() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !1 !== e.cors_use_newrelic_header;
        }
        useTraceContextHeadersForCors() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.cors_use_tracecontext_headers;
        }
      }
      var Z = i(7825),
        V = ["load", "error", "abort", "timeout"],
        G = V.length,
        W = n.Yu.REQ,
        X = n.Yu.XHR;
      class Q extends h {
        static featureName = Z.t;
        constructor(e, t) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, Z.t, i), (0, n.OP)(e).xhrWrappable && (this.dt = new q(e), this.handler = (e, t, r, n) => (0, c.p)(e, t, r, n, this.ee), (0, F.u5)(this.ee), (0, F.Kf)(this.ee), function (e, t, i, o) {
            function a(e) {
              var t = this;
              t.totalCbs = 0, t.called = 0, t.cbTime = 0, t.end = E, t.ended = !1, t.xhrGuids = {}, t.lastSize = null, t.loadCaptureCalled = !1, t.params = this.params || {}, t.metrics = this.metrics || {}, e.addEventListener("load", function (r) {
                _(t, e);
              }, (0, C.m$)(!1)), u.IF || e.addEventListener("progress", function (e) {
                t.lastSize = e.loaded;
              }, (0, C.m$)(!1));
            }
            function s(e) {
              this.params = {
                method: e[0]
              }, T(this, e[1]), this.metrics = {};
            }
            function c(t, r) {
              var i = (0, n.DL)(e);
              i.xpid && this.sameOrigin && r.setRequestHeader("X-NewRelic-ID", i.xpid);
              var a = o.generateTracePayload(this.parsedOrigin);
              if (a) {
                var s = !1;
                a.newrelicHeader && (r.setRequestHeader("newrelic", a.newrelicHeader), s = !0), a.traceContextParentHeader && (r.setRequestHeader("traceparent", a.traceContextParentHeader), a.traceContextStateHeader && r.setRequestHeader("tracestate", a.traceContextStateHeader), s = !0), s && (this.dt = a);
              }
            }
            function d(e, r) {
              var n = this.metrics,
                i = e[0],
                o = this;
              if (n && i) {
                var a = B(i);
                a && (n.txSize = a);
              }
              this.startTime = (0, g.z)(), this.listener = function (e) {
                try {
                  "abort" !== e.type || o.loadCaptureCalled || (o.params.aborted = !0), ("load" !== e.type || o.called === o.totalCbs && (o.onloadCalled || "function" != typeof r.onload) && "function" == typeof o.end) && o.end(r);
                } catch (e) {
                  try {
                    t.emit("internal-error", [e]);
                  } catch (e) {}
                }
              };
              for (var s = 0; s < G; s++) r.addEventListener(V[s], this.listener, (0, C.m$)(!1));
            }
            function l(e, t, r) {
              this.cbTime += e, t ? this.onloadCalled = !0 : this.called += 1, this.called !== this.totalCbs || !this.onloadCalled && "function" == typeof r.onload || "function" != typeof this.end || this.end(r);
            }
            function f(e, t) {
              var r = "" + M(e) + !!t;
              this.xhrGuids && !this.xhrGuids[r] && (this.xhrGuids[r] = !0, this.totalCbs += 1);
            }
            function h(e, t) {
              var r = "" + M(e) + !!t;
              this.xhrGuids && this.xhrGuids[r] && (delete this.xhrGuids[r], this.totalCbs -= 1);
            }
            function p() {
              this.endTime = (0, g.z)();
            }
            function m(e, r) {
              r instanceof X && "load" === e[0] && t.emit("xhr-load-added", [e[1], e[2]], r);
            }
            function v(e, r) {
              r instanceof X && "load" === e[0] && t.emit("xhr-load-removed", [e[1], e[2]], r);
            }
            function b(e, t, r) {
              t instanceof X && ("onload" === r && (this.onload = !0), ("load" === (e[0] && e[0].type) || this.onload) && (this.xhrCbStart = (0, g.z)()));
            }
            function y(e, r) {
              this.xhrCbStart && t.emit("xhr-cb-time", [(0, g.z)() - this.xhrCbStart, this.onload, r], r);
            }
            function w(e) {
              var t,
                r = e[1] || {};
              if ("string" == typeof e[0] ? 0 === (t = e[0]).length && u.il && (t = "" + u._A.location.href) : e[0] && e[0].url ? t = e[0].url : u._A?.URL && e[0] && e[0] instanceof URL ? t = e[0].href : "function" == typeof e[0].toString && (t = e[0].toString()), "string" == typeof t && 0 !== t.length) {
                t && (this.parsedOrigin = (0, U.e)(t), this.sameOrigin = this.parsedOrigin.sameOrigin);
                var n = o.generateTracePayload(this.parsedOrigin);
                if (n && (n.newrelicHeader || n.traceContextParentHeader)) if (e[0] && e[0].headers) s(e[0].headers, n) && (this.dt = n);else {
                  var i = {};
                  for (var a in r) i[a] = r[a];
                  i.headers = new Headers(r.headers || {}), s(i.headers, n) && (this.dt = n), e.length > 1 ? e[1] = i : e.push(i);
                }
              }
              function s(e, t) {
                var r = !1;
                return t.newrelicHeader && (e.set("newrelic", t.newrelicHeader), r = !0), t.traceContextParentHeader && (e.set("traceparent", t.traceContextParentHeader), t.traceContextStateHeader && e.set("tracestate", t.traceContextStateHeader), r = !0), r;
              }
            }
            function A(e, t) {
              this.params = {}, this.metrics = {}, this.startTime = (0, g.z)(), this.dt = t, e.length >= 1 && (this.target = e[0]), e.length >= 2 && (this.opts = e[1]);
              var r,
                n = this.opts || {},
                i = this.target;
              "string" == typeof i ? r = i : "object" == typeof i && i instanceof W ? r = i.url : u._A?.URL && "object" == typeof i && i instanceof URL && (r = i.href), T(this, r);
              var o = ("" + (i && i instanceof W && i.method || n.method || "GET")).toUpperCase();
              this.params.method = o, this.txSize = B(n.body) || 0;
            }
            function x(e, t) {
              var n;
              this.endTime = (0, g.z)(), this.params || (this.params = {}), this.params.status = t ? t.status : 0, "string" == typeof this.rxSize && this.rxSize.length > 0 && (n = +this.rxSize);
              var o = {
                txSize: this.txSize,
                rxSize: n,
                duration: (0, g.z)() - this.startTime
              };
              i("xhr", [this.params, o, this.startTime, this.endTime, "fetch"], this, r.D.ajax);
            }
            function E(e) {
              var t = this.params,
                n = this.metrics;
              if (!this.ended) {
                this.ended = !0;
                for (var o = 0; o < G; o++) e.removeEventListener(V[o], this.listener, !1);
                t.aborted || (n.duration = (0, g.z)() - this.startTime, this.loadCaptureCalled || 4 !== e.readyState ? null == t.status && (t.status = 0) : _(this, e), n.cbTime = this.cbTime, i("xhr", [t, n, this.startTime, this.endTime, "xhr"], this, r.D.ajax));
              }
            }
            function T(e, t) {
              var r = (0, U.e)(t),
                n = e.params;
              n.hostname = r.hostname, n.port = r.port, n.protocol = r.protocol, n.host = r.hostname + ":" + r.port, n.pathname = r.pathname, e.parsedOrigin = r, e.sameOrigin = r.sameOrigin;
            }
            function _(e, t) {
              e.params.status = t.status;
              var r = function (e, t) {
                var r = e.responseType;
                return "json" === r && null !== t ? t : "arraybuffer" === r || "blob" === r || "json" === r ? B(e.response) : "text" === r || "" === r || void 0 === r ? B(e.responseText) : void 0;
              }(t, e.lastSize);
              if (r && (e.metrics.rxSize = r), e.sameOrigin) {
                var n = t.getResponseHeader("X-NewRelic-App-Data");
                n && (e.params.cat = n.split(", ").pop());
              }
              e.loadCaptureCalled = !0;
            }
            t.on("new-xhr", a), t.on("open-xhr-start", s), t.on("open-xhr-end", c), t.on("send-xhr-start", d), t.on("xhr-cb-time", l), t.on("xhr-load-added", f), t.on("xhr-load-removed", h), t.on("xhr-resolved", p), t.on("addEventListener-end", m), t.on("removeEventListener-end", v), t.on("fn-end", y), t.on("fetch-before-start", w), t.on("fetch-start", A), t.on("fn-start", b), t.on("fetch-done", x);
          }(e, this.ee, this.handler, this.dt), this.importAggregator());
        }
      }
      var K = i(3614);
      const {
        BST_RESOURCE: Y,
        RESOURCE: J,
        START: ee,
        END: te,
        FEATURE_NAME: re,
        FN_END: ne,
        FN_START: ie,
        PUSH_STATE: oe
      } = K;
      var ae = i(7836);
      const {
        FEATURE_NAME: se,
        START: ce,
        END: ue,
        BODY: de,
        CB_END: le,
        JS_TIME: fe,
        FETCH: he,
        FN_START: pe,
        CB_START: ge,
        FN_END: me
      } = ae;
      var ve = i(4649);
      class be extends h {
        static featureName = ve.t;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, ve.t, r), this.importAggregator();
        }
      }
      new class extends t {
        constructor(t) {
          let r = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : (0, _.ky)(16);
          super(), u._A ? (this.agentIdentifier = r, this.sharedAggregator = new y({
            agentIdentifier: this.agentIdentifier
          }), this.features = {}, this.desiredFeatures = new Set(t.features || []), this.desiredFeatures.add(m), Object.assign(this, (0, s.j)(this.agentIdentifier, t, t.loaderType || "agent")), this.start()) : (0, e.Z)("Failed to initial the agent. Could not determine the runtime environment.");
        }
        get config() {
          return {
            info: (0, n.C5)(this.agentIdentifier),
            init: (0, n.P_)(this.agentIdentifier),
            loader_config: (0, n.DL)(this.agentIdentifier),
            runtime: (0, n.OP)(this.agentIdentifier)
          };
        }
        start() {
          const t = "features";
          try {
            const n = a(this.agentIdentifier),
              i = [...this.desiredFeatures];
            i.sort((e, t) => r.p[e.featureName] - r.p[t.featureName]), i.forEach(t => {
              if (n[t.featureName] || t.featureName === r.D.pageViewEvent) {
                const i = function (e) {
                  switch (e) {
                    case r.D.ajax:
                      return [r.D.jserrors];
                    case r.D.sessionTrace:
                      return [r.D.ajax, r.D.pageViewEvent];
                    case r.D.sessionReplay:
                      return [r.D.sessionTrace];
                    case r.D.pageViewTiming:
                      return [r.D.pageViewEvent];
                    default:
                      return [];
                  }
                }(t.featureName);
                i.every(e => n[e]) || (0, e.Z)("".concat(t.featureName, " is enabled but one or more dependent features has been disabled (").concat((0, D.P)(i), "). This may cause unintended consequences or missing data...")), this.features[t.featureName] = new t(this.agentIdentifier, this.sharedAggregator);
              }
            }), (0, T.Qy)(this.agentIdentifier, this.features, t);
          } catch (r) {
            (0, e.Z)("Failed to initialize all enabled instrument classes (agent aborted) -", r);
            for (const e in this.features) this.features[e].abortHandler?.();
            const n = (0, T.fP)();
            return delete n.initializedAgents[this.agentIdentifier]?.api, delete n.initializedAgents[this.agentIdentifier]?.[t], delete this.sharedAggregator, n.ee?.abort(), delete n.ee?.get(this.agentIdentifier), !1;
          }
        }
        addToTrace(t) {
          (0, e.Z)("Call to agent api addToTrace failed. The page action feature is not currently initialized.");
        }
        setCurrentRouteName(t) {
          (0, e.Z)("Call to agent api setCurrentRouteName failed. The spa feature is not currently initialized.");
        }
        interaction() {
          (0, e.Z)("Call to agent api interaction failed. The spa feature is not currently initialized.");
        }
      }({
        features: [Q, m, O, class extends h {
          static featureName = re;
          constructor(e, t) {
            if (super(e, t, re, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !u.il) return;
            const n = this.ee;
            let i;
            (0, F.QU)(n), this.eventsEE = (0, F.em)(n), this.eventsEE.on(ie, function (e, t) {
              this.bstStart = (0, g.z)();
            }), this.eventsEE.on(ne, function (e, t) {
              (0, c.p)("bst", [e[0], t, this.bstStart, (0, g.z)()], void 0, r.D.sessionTrace, n);
            }), n.on(oe + ee, function (e) {
              this.time = (0, g.z)(), this.startPath = location.pathname + location.hash;
            }), n.on(oe + te, function (e) {
              (0, c.p)("bstHist", [location.pathname + location.hash, this.startPath, this.time], void 0, r.D.sessionTrace, n);
            });
            try {
              i = new PerformanceObserver(e => {
                const t = e.getEntries();
                (0, c.p)(Y, [t], void 0, r.D.sessionTrace, n);
              }), i.observe({
                type: J,
                buffered: !0
              });
            } catch (e) {}
            this.importAggregator({
              resourceObserver: i
            });
          }
        }, P, be, k, class extends h {
          static featureName = se;
          constructor(e, t) {
            if (super(e, t, se, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !u.il) return;
            if (!(0, n.OP)(e).xhrWrappable) return;
            try {
              this.removeOnAbort = new AbortController();
            } catch (e) {}
            let r,
              i = 0;
            const o = this.ee.get("tracer"),
              a = (0, F._L)(this.ee),
              s = (0, F.Lg)(this.ee),
              c = (0, F.BV)(this.ee),
              d = (0, F.Kf)(this.ee),
              l = this.ee.get("events"),
              f = (0, F.u5)(this.ee),
              h = (0, F.QU)(this.ee),
              p = (0, F.Gm)(this.ee);
            function m(e, t) {
              h.emit("newURL", ["" + window.location, t]);
            }
            function v() {
              i++, r = window.location.hash, this[pe] = (0, g.z)();
            }
            function b() {
              i--, window.location.hash !== r && m(0, !0);
              var e = (0, g.z)();
              this[fe] = ~~this[fe] + e - this[pe], this[me] = e;
            }
            function y(e, t) {
              e.on(t, function () {
                this[t] = (0, g.z)();
              });
            }
            this.ee.on(pe, v), s.on(ge, v), a.on(ge, v), this.ee.on(me, b), s.on(le, b), a.on(le, b), this.ee.buffer([pe, me, "xhr-resolved"], this.featureName), l.buffer([pe], this.featureName), c.buffer(["setTimeout" + ue, "clearTimeout" + ce, pe], this.featureName), d.buffer([pe, "new-xhr", "send-xhr" + ce], this.featureName), f.buffer([he + ce, he + "-done", he + de + ce, he + de + ue], this.featureName), h.buffer(["newURL"], this.featureName), p.buffer([pe], this.featureName), s.buffer(["propagate", ge, le, "executor-err", "resolve" + ce], this.featureName), o.buffer([pe, "no-" + pe], this.featureName), a.buffer(["new-jsonp", "cb-start", "jsonp-error", "jsonp-end"], this.featureName), y(f, he + ce), y(f, he + "-done"), y(a, "new-jsonp"), y(a, "jsonp-end"), y(a, "cb-start"), h.on("pushState-end", m), h.on("replaceState-end", m), window.addEventListener("hashchange", m, (0, C.m$)(!0, this.removeOnAbort?.signal)), window.addEventListener("load", m, (0, C.m$)(!0, this.removeOnAbort?.signal)), window.addEventListener("popstate", function () {
              m(0, i > 1);
            }, (0, C.m$)(!0, this.removeOnAbort?.signal)), this.abortHandler = this.#i, this.importAggregator();
          }
          #i() {
            this.removeOnAbort?.abort(), this.abortHandler = void 0;
          }
        }],
        loaderType: "spa"
      });
    })();
  })();
})()</script>
      <link rel="shortcut icon" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/103/images/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/103/images/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/arp.css">
      <link href="//cdn.pendo.io" rel="dns-prefetch">
      <link href="https://cdn.pendo.io" rel="preconnect" crossorigin="anonymous">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com">
      <script async="" id="reading-assistant-script-tag" src="/feature/assets/ai-components?componentVersion=V14" type="text/javascript"></script>
      <script type="text/javascript">
        
        window.appData = window.appData || [];
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Journal of Human Evolution","auth":"AE"};
        window.arp = {
          config: {"adobeSuite":"elsevier-sd-prod","arsUrl":"https://ars.els-cdn.com","assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1","recommendationsFeedback":{"enabled":true,"url":"https://feedback.recs.d.elsevier.com/raw/events","timeout":60000},"googleMapsApiKey":"AIzaSyCBYU6I6lrbEU6wQXUEIte3NwGtm3jwHQc","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","xocsMediaPlayerBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/147/js/ableplayer","strictMode":false,"seamlessAccess":{"enableSeamlessAccess":true,"scriptUrl":"https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js","persistenceUrl":"https://service.seamlessaccess.org/ps/","persistenceContext":"seamlessaccess.org","scienceDirectUrl":"https://www.sciencedirect.com","shibAuthUrl":"https://auth.elsevier.com/ShibAuth/institutionLogin"},"reaxys":{"apiUrl":"https://reaxys-sdlc.reaxys.com","origin":"sciencedirect","queryBuilderHostPath":"https://www.reaxys.com/reaxys/secured/hopinto.do","url":"https://www.reaxys.com"},"oneTrustCookie":{"enabled":true},"ssrn":{"url":"https://papers.ssrn.com","path":"/sol3/papers.cfm"},"aiNavigation":{"enabled":true,"version":"V1"}},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
        window.addEventListener('beforeprint', () => pendo.onGuideDismissed());
      </script>
    <script data-cfasync="false" src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" data-document-language="true" data-domain-script="865ea198-88cc-4e41-8952-1df75d554d02"></script><meta http-equiv="origin-trial" content="AlK2UR5SkAlj8jjdEc9p3F3xuFYlF6LYjAML3EOqw1g26eCwWPjdmecULvBH5MVPoqKYrOfPhYVL71xAXI1IBQoAAAB8eyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiV2ViVmlld1hSZXF1ZXN0ZWRXaXRoRGVwcmVjYXRpb24iLCJleHBpcnkiOjE3NTgwNjcxOTksImlzU3ViZG9tYWluIjp0cnVlfQ=="><meta http-equiv="origin-trial" content="Amm8/NmvvQfhwCib6I7ZsmUxiSCfOxWxHayJwyU1r3gRIItzr7bNQid6O8ZYaE1GSQTa69WwhPC9flq/oYkRBwsAAACCeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiV2ViVmlld1hSZXF1ZXN0ZWRXaXRoRGVwcmVjYXRpb24iLCJleHBpcnkiOjE3NTgwNjcxOTksImlzU3ViZG9tYWluIjp0cnVlfQ=="><meta http-equiv="origin-trial" content="A9wSqI5i0iwGdf6L1CERNdmsTPgVu44ewj8QxTBYgsv1LCPUVF7YmWOvTappqB1139jAymxUW/RO8zmMqo4zlAAAAACNeyJvcmlnaW4iOiJodHRwczovL2RvdWJsZWNsaWNrLm5ldDo0NDMiLCJmZWF0dXJlIjoiRmxlZGdlQmlkZGluZ0FuZEF1Y3Rpb25TZXJ2ZXIiLCJleHBpcnkiOjE3MzY4MTI4MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><meta http-equiv="origin-trial" content="A+d7vJfYtay4OUbdtRPZA3y7bKQLsxaMEPmxgfhBGqKXNrdkCQeJlUwqa6EBbSfjwFtJWTrWIioXeMW+y8bWAgQAAACTeyJvcmlnaW4iOiJodHRwczovL2dvb2dsZXN5bmRpY2F0aW9uLmNvbTo0NDMiLCJmZWF0dXJlIjoiRmxlZGdlQmlkZGluZ0FuZEF1Y3Rpb25TZXJ2ZXIiLCJleHBpcnkiOjE3MzY4MTI4MDAsImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9"><script src="https://securepubads.g.doubleclick.net/pagead/managed/js/gpt/m202506050101/pubads_impl.js" async=""></script><link href="https://securepubads.g.doubleclick.net/pagead/managed/dict/m202506100101/gpt" rel="compression-dictionary"><script src="https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script src="https://cdn.cookielaw.org/scripttemplates/202504.1.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-button-order-0{order:0}#onetrust-banner-sdk .ot-button-order-1{order:1}#onetrust-banner-sdk .ot-button-order-2{order:2}#onetrust-banner-sdk #onetrust-close-btn-container svg{height:10px;width:10px;pointer-events:none}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .pc-logo svg,#onetrust-pc-sdk .ot-pc-logo svg{height:60px;width:180px}#onetrust-pc-sdk #close-pc-btn-handler>svg{margin:auto;display:block;height:12px;width:12px}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo svg,#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.8em;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px}#onetrust-banner-sdk .ot-optout-signal svg.ot-source-sprite,#onetrust-pc-sdk .ot-optout-signal svg.ot-source-sprite{position:relative;bottom:-3px}#onetrust-banner-sdk .ot-optout-signal svg:not(.ot-source-sprite),#onetrust-pc-sdk .ot-optout-signal svg:not(.ot-source-sprite){transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg:not(.ot-source-sprite) path,#onetrust-pc-sdk .ot-optout-signal svg:not(.ot-source-sprite) path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk #onetrust-sprite-svg{display:none}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk img,#onetrust-pc-sdk img,#ot-sdk-cookie-policy img,#ot-sync-ntfy img{overflow:hidden !important}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-group-container{position:relative}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px;margin-right:5px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (max-width: 550px){#onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler,#onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler{margin-right:0}#onetrust-banner-sdk .has-reject-all-button div#onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler,#onetrust-banner-sdk .has-reject-all-button div#onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button div#onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler{margin-right:0}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk #onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler{width:auto}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container{display:inline-flex;flex-wrap:wrap}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container #onetrust-pc-btn-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container #onetrust-accept-btn-handler{float:none}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group.ot-button-order-container *[class*=ot-button-order-]:nth-of-type(1){margin-right:auto !important}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group.ot-button-order-container{width:auto}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:80%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! Extra code to blur out background */
.onetrust-pc-dark-filter{
background:rgba(0,0,0,.5);
z-index:2147483646;
width:100%;
height:100%;
overflow:hidden;
position:fixed;
top:0;
bottom:0;
left:0;
backdrop-filter: initial
}

/*! v6.12.0 2021-01-19 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #eb6500!important;outline:1px solid transparent;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007398!important;border:2px solid #007398!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#eb6500!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007398!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color: #2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007398!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #eb6500!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:#2e2e2e!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:#2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #eb6500;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:#2e2e2e;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#eb6500;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#eb6500;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #eb6500!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{width:100%}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk #ot-pc-hdr{margin-left:24px}div#onetrust-pc-sdk .ot-lst-subhdr{width:calc(100% - 24px)}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,div#onetrust-pc-sdk button#filter-apply-handler,div#onetrust-pc-sdk button#filter-cancel-handler{height:2em!important;padding-left:14px!important;padding-right:14px!important}div#onetrust-pc-sdk #ot-fltr-cnt{box-shadow:0 0;border:1px solid #8e8e8e;border-radius:0}div#onetrust-pc-sdk .ot-fltr-scrlcnt{max-height:calc(100% - 80px)}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}div#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:16px}div#onetrust-pc-sdk #ot-lst-cnt{margin-left:24px;width:calc(100% - 48px)}div#onetrust-pc-sdk #ot-anchor{display:none!important}

/* 2023-12-04  Fix for button order in mobile view*/
@media (max-width: 550px) {
  #onetrust-accept-btn-handler {order: 1;  }
  #onetrust-reject-all-handler { order: 2;  }
  #onetrust-pc-btn-handler { order: 3;  }
}


/*! Extra code to blur our background */
.onetrust-pc-dark-filter{
backdrop-filter: blur(3px)
}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;position:relative;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #ot-filter-list-header{margin-top:20px;margin-bottom:10px;float:left;max-width:150px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px 1px 1px 15px;overflow:hidden;text-overflow:ellipsis;white-space:nowrap}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .ot-pc-refuse-all-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .save-preference-btn-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container #accept-recommended-btn-handler{width:auto;margin-bottom:1rem}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:10px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-accordion-layout .ot-pgph-link{margin-top:5px}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .ot-pc-refuse-all-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container .save-preference-btn-handler,#onetrust-pc-sdk .ot-btn-container.ot-button-order-container #accept-recommended-btn-handler{width:100%}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! Extra code to blur out background */
.onetrust-pc-dark-filter{
background:rgba(0,0,0,.5);
z-index:2147483646;
width:100%;
height:100%;
overflow:hidden;
position:fixed;
top:0;
bottom:0;
left:0;
backdrop-filter: initial
}

/*! v6.12.0 2021-01-19 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #eb6500!important;outline:1px solid transparent;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007398!important;border:2px solid #007398!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#eb6500!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007398!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color: #2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007398!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #eb6500!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:#2e2e2e!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:#2e2e2e!important;text-decoration-color:#eb6500!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #eb6500;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:#2e2e2e;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#eb6500;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#eb6500;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #eb6500!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{width:100%}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk #ot-pc-hdr{margin-left:24px}div#onetrust-pc-sdk .ot-lst-subhdr{width:calc(100% - 24px)}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,div#onetrust-pc-sdk button#filter-apply-handler,div#onetrust-pc-sdk button#filter-cancel-handler{height:2em!important;padding-left:14px!important;padding-right:14px!important}div#onetrust-pc-sdk #ot-fltr-cnt{box-shadow:0 0;border:1px solid #8e8e8e;border-radius:0}div#onetrust-pc-sdk .ot-fltr-scrlcnt{max-height:calc(100% - 80px)}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}div#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:16px}div#onetrust-pc-sdk #ot-lst-cnt{margin-left:24px;width:calc(100% - 48px)}div#onetrust-pc-sdk #ot-anchor{display:none!important}

/* 2023-12-04  Fix for button order in mobile view*/
@media (max-width: 550px) {
  #onetrust-accept-btn-handler {order: 1;  }
  #onetrust-reject-all-handler { order: 2;  }
  #onetrust-pc-btn-handler { order: 3;  }
}


/*! Extra code to blur our background */
.onetrust-pc-dark-filter{
backdrop-filter: blur(3px)
}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://assets.adobedtm.com/4a848ae9611a/032db4f73473/launch-a6263b31083f.min.js" type="text/javascript" async=""></script><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
</style><script src="https://assets.adobedtm.com/extensions/EP31dbb9c60e404ba1aa6e746d49be6f29/AppMeasurement.min.js" async=""></script><script src="https://assets.adobedtm.com/extensions/EP31dbb9c60e404ba1aa6e746d49be6f29/AppMeasurement_Module_ActivityMap.min.js" async=""></script><script src="https://assets.adobedtm.com/4a848ae9611a/032db4f73473/93571d51095c/RCa16d232f95a944c0aabdea6621a2ef94-source.min.js" async=""></script><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style><style>
            input[data-pendo-field-valid="false"] {
                border-color: red !important;
                outline-color: red !important;
            }
        </style></head>
    <body data-sd-ui-layer-boundary="true"><div id="MathJax_Message" style="display: none;"></div>
      <script type="text/javascript">
        window.__PRELOADED_STATE__ = {"abstracts":{"content":[{"$$":[{"$":{"id":"sectitle0010"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Ancient DNA analyses have shown that interbreeding between hominin taxa occurred multiple times. Although admixture is often reflected in skeletal phenotype, the relationship between the two remains poorly understood, hampering interpretation of the hominin fossil record. Direct study of this relationship is often impossible due to the paucity of hominin fossils and difficulties retrieving ancient genetic material. Here, we use a sample of known ancestry hybrids between two closely related nonhuman primate taxa (Indian and Chinese "},{"#name":"italic","_":"Macaca mulatta"},{"#name":"__text__","_":") to investigate the effect of admixture on skeletal morphology. We focus on pelvic shape, which has potential fitness implications in hybrids, as mismatches between maternal pelvic and fetal cranial morphology are often fatal to mother and offspring. As the pelvis is also one of the skeletal regions that differs most between "},{"#name":"italic","_":"Homo sapiens"},{"#name":"__text__","_":" and Neanderthals, investigating the pelvic consequences of interbreeding could be informative regarding the viability of their hybrids. We find that the effect of admixture in "},{"#name":"italic","_":"M. mulatta"},{"#name":"__text__","_":" is small and proportional to the relatively small morphological difference between the parent taxa. Sexual dimorphism appears to be the main determinant of pelvic shape in "},{"#name":"italic","_":"M. mulatta"},{"#name":"__text__","_":". The lack of difference in pelvic shape between Chinese and Indian "},{"#name":"italic","_":"M. mulatta"},{"#name":"__text__","_":" is in contrast to that between Neanderthals and "},{"#name":"italic","_":"H. sapiens"},{"#name":"__text__","_":", despite a similar split time (in generations) between the hybridizing pairs. Greater phenotypic divergence between hominins may relate to adaptations to disparate environments but may also highlight how the unique degree of cultural buffering in hominins allowed for greater neutral divergence. In contrast to some previous work identifying extreme morphologies in first- and second-generation hybrids, here the relationship between pelvic shape and admixture is linear. This linearity may be because most sampled animals have a multigenerational admixture history or because of relatively high constraints on the pelvis compared with other skeletal regions."}],"$":{"view":"all","id":"abspara0010"},"#name":"simple-para"}],"$":{"view":"all","id":"abssec0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"abs0010","lang":"en","class":"author"},"#name":"abstract"}],"floats":[],"footnotes":[],"attachments":[]},"accessbarConfig":{"fallback":false,"id":"accessbar","version":"0.0.1","analytics":{"location":"accessbar","eventName":"ctaImpression"},"label":{},"ariaLabel":{"accessbar":"Download options and search","componentsList":"PDF Options"},"banners":[{"id":"BannerSsrn"}],"components":[{"target":"_blank","analytics":[{"ids":["accessbar:fta:single-article"],"eventName":"ctaClick"}],"label":"View&nbsp;**PDF**","ariaLabel":"View PDF. Opens in a new window.","id":"ViewPDF"},{"analytics":[{"ids":["accessbar:fta:full-issue"],"eventName":"ctaClick"}],"label":"Download full issue","id":"DownloadFullIssue"},{"target":"_blank","analytics":[{"ids":["accessbar:article:chorus-am"],"eventName":"ctaClick"}],"label":"View Open Manuscript","href":"/science/article/am/pii/S0047248421001019","id":"OpenManuscript"},{"id":"ViewClinicalKeyFullText","analytics":[{"eventName":"ctaClick","ids":["accessbar:fta:clinical-key"]}],"label":"View full text","rel":"noopener noreferrer","target":"_blank","href":"https://www.clinicalkey.com/#!/content/journal/1-s2.0-S0047248421001019"}],"search":{"inputPlaceHolder":"Search ScienceDirect","ariaLabel":{"input":"Search ScienceDirect","submit":"Submit search"},"formAction":"/search#submit","analytics":[{"ids":["accessbar:search"],"eventName":"searchStart"}],"id":"QuickSearch"}},"adobeTarget":{},"article":{"analyticsMetadata":{"accountId":"228598","accountName":"ScienceDirect Guests","loginStatus":"anonymous","userId":"12975512","isLoggedIn":false},"article-number":"103049","cid":"272603","content-family":"serial","copyright-line":"© 2021 The Author(s). Published by Elsevier Ltd.","cover-date-years":["2021"],"cover-date-start":"2021-10-01","cover-date-text":"October 2021","document-subtype":"fla","document-type":"article","entitledToken":"66B54AB14D6CC7914AF5A05FA69AD71EFBEC5B607A99667DA2BAA88100A19580AD98FF98FE50F4FD","genAiToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJnZW5BaUFwcHMiLCJwaWkiOiJTMDA0NzI0ODQyMTAwMTAxOSIsInN1YiI6IjIyODU5OCIsImlzcyI6ImFycCIsInNlc3Npb25JZCI6ImZiMDU0YjFlNTE2YzU3NDVlMjI5ZTkxMjVjODU1OTdiM2YxZGd4cnFhIiwiZXhwIjoxNzQ5NjAzOTE0LCJpYXQiOjE3NDk1OTY3MTQsInZlcnNpb24iOjEsImp0aSI6IjJmMjg1ODcxLTE3YjItNDY5Ni1iYTM1LTI0NjRiOGU1NWU1ZSJ9.xfabZqO6jfwow50xr1nab3Gkr6vz78_BB2hIW7Vt7iw","eid":"1-s2.0-S0047248421001019","doi":"10.1016/j.jhevol.2021.103049","first-fp":"103049","hub-eid":"1-s2.0-S0047248421X00097","issuePii":"S0047248421X00097","item-weight":"FULL-TEXT","language":"en","last-author":{"#name":"last-author","$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"$$":[{"#name":"author","$":{"id":"au6","author-id":"S0047248421001019-394f7f3f5a66ce80bd222704882d198c"},"$$":[{"#name":"given-name","_":"Timothy D."},{"#name":"surname","_":"Weaver"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]}]},"normalized-first-auth-initial":"L","normalized-first-auth-surname":"BUCK","open-research":{"#name":"open-research","$":{"xmlns:xocs":true},"$$":[{"#name":"or-agreements","$$":[{"#name":"or-agreement","$$":[{"#name":"or-agreement-name","_":"CHU_NSF"},{"#name":"or-agreement-terms","_":"publishAcceptedManuscriptIndexable"}]}]},{"#name":"or-agreement-license","_":"http://www.elsevier.com/open-access/userlicense/1.0/"}]},"pages":[{"first-page":"103049"}],"pii":"S0047248421001019","self-archiving":{"#name":"self-archiving","$":{"xmlns:xocs":true},"$$":[{"#name":"sa-start-date","_":"2022-08-26T00:00:00.000Z"},{"#name":"sa-user-license","_":"http://creativecommons.org/licenses/by-nc-nd/4.0/"}]},"srctitle":"Journal of Human Evolution","suppl":"C","timestamp":"2022-06-22T07:44:42.204453Z","title":{"content":[{"#name":"title","$":{"id":"title0010"},"_":"Effects of hybridization on pelvic morphology: A macaque model"}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"159","vol-iss-suppl-text":"Volume 159","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":true,"blockFullTextForAnonymousAccess":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":true,"document-references":91,"freeHtmlGiven":false,"ssoUrls":["//acw.clinicalkey.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u","//acw.scopus.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u","//acw.sciencedirect.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u","//acw.elsevier.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u"],"userProfile":{"departmentName":"ScienceDirect Guests","accessType":"GUEST","accountId":"228598","webUserId":"12975512","accountName":"ScienceDirect Guests","departmentId":"291352","userType":"NORMAL","hasMultipleOrganizations":false,"accountNumber":"C000228598"},"access":{"openAccess":true,"openArchive":false,"license":"http://creativecommons.org/licenses/by-nc-nd/4.0/"},"accessTypeLabel":"Open access","aipType":"none","articleEntitlement":{"entitled":true,"isCasaUser":false,"usageInfo":"(12975512,U|291352,D|228598,A|3,P|2,PL)(SDFE,CON|fb054b1e516c5745e229e9125c85597b3f1dgxrqa,SSO|ANON_GUEST,ACCESS_TYPE)","entitledByAccount":false,"entitlementOrigin":"SD"},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"dates":{"Available online":"26 August 2021","Received":"10 November 2020","Revised":[],"Accepted":"25 June 2021","Publication date":"1 October 2021","Version of Record":"26 August 2021"},"documentTypeLabel":"Research article","downloadFullIssue":true,"entitlementReason":"openaccess","features":["aamAttachments","keywords","references","preview"],"hasBody":true,"has-large-authors":false,"hasScholarlyAbstract":true,"headerConfig":{"contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","userEmail":"","orgName":"ScienceDirect Guests","webUserId":"12975512","libraryBanner":null,"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasInstitutionalAssociation":false,"hasMultiOrg":false,"userType":"GUEST","userAnonymity":"ANON_GUEST","allowCart":true,"environment":"prod","cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com"},"isCorpReq":false,"isPdfFullText":false,"issn":"00472484","issn-primary-formatted":"0047-2484","issRange":"","isThirdParty":false,"licenseLink":{"prefix":"Under a Creative Commons ","text":"license","url":"http://creativecommons.org/licenses/by-nc-nd/4.0/"},"pageCount":13,"pdfDownload":{"isPdfFullText":false,"urlMetadata":{"queryParams":{"md5":"08c4f8c5a04c804cb19a38c138b63424","pid":"1-s2.0-S0047248421001019-main.pdf"},"pii":"S0047248421001019","pdfExtension":"/pdfft","path":"science/article/pii"}},"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":165,"openArchiveStatus":false,"openArchiveArticleCount":64,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S0047248421X00097-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S0047248421X00097/cover/DOWNSAMPLED200/image/gif/0ff6b21f8f30e226a2fedd20091a7de1/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"21148","pixel-height":"200","pixel-width":"150"},{"attachment-eid":"1-s2.0-S0047248421X00097-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S0047248421X00097/cover/DOWNSAMPLED/image/gif/d8a3b8c491a8065be6e970dec1cf2548/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"13047","pixel-height":"150","pixel-width":"113"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S00472484.gif","title":"journal-of-human-evolution","contentTypeCode":"JL","images":{"coverImage":"https://ars.els-cdn.com/content/image/1-s2.0-S0047248421X00097-cov150h.gif","logo":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/image/elsevier-non-solus.svg","logoAltText":"Elsevier"},"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S0047248421X00097-cov150h.gif"},"rightsLink":"https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&contentID=S0047248421001019&orderBeanReset=true","volRange":"159","titleString":"Effects of hybridization on pelvic morphology: A macaque model","openManuscriptUrl":"/science/article/am/pii/S0047248421001019","ssrn":{},"renderingMode":"Article","isAbstract":false,"isContentVisible":false,"ajaxLinks":{"referenceLinks":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true,"citingArticles":true,"authorMetadata":true},"pdfEmbed":false,"displayViewFullText":false},"authors":{"content":[{"#name":"author-group","$":{"id":"augrp0010"},"$$":[{"#name":"author","$":{"id":"au1","orcid":"0000-0002-1768-9049","author-id":"S0047248421001019-735e68958ee3905b00831b5c49ac4518"},"$$":[{"#name":"given-name","_":"Laura T."},{"#name":"surname","_":"Buck"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"crosref0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMnhtbG5zJTNBeGxpbmslMjIlM0F0cnVlJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTJDJTIyaHJlZiUyMiUzQSUyMm1haWx0byUzQWwuYnVjayU0MGxqbXUuYWMudWslMjIlMkMlMjJpZCUyMiUzQSUyMmVhZGQwMDEwJTIyJTdEJTJDJTIyXyUyMiUzQSUyMmwuYnVjayU0MGxqbXUuYWMudWslMjIlN0Q="}]},{"#name":"author","$":{"id":"au2","author-id":"S0047248421001019-aed8f57b067c0c2cbb8cbdea82ccdc92"},"$$":[{"#name":"given-name","_":"David C."},{"#name":"surname","_":"Katz"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"aff3","id":"crosref0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au3","author-id":"S0047248421001019-4830af2779b3207e9159e91c337c9b42"},"$$":[{"#name":"given-name","_":"Rebecca Rogers"},{"#name":"surname","_":"Ackermann"},{"#name":"cross-ref","$":{"refid":"aff4","id":"crosref0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"cross-ref","$":{"refid":"aff5","id":"crosref0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]},{"#name":"author","$":{"id":"au4","orcid":"0000-0003-0189-6390","author-id":"S0047248421001019-f5f93ae1c3f9f9885d0673748f92b1ca"},"$$":[{"#name":"given-name","_":"Leslea J."},{"#name":"surname","_":"Hlusko"},{"#name":"cross-ref","$":{"refid":"aff6","id":"crosref0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"f"}]},{"#name":"cross-ref","$":{"refid":"aff7","id":"crosref0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"g"}]}]},{"#name":"author","$":{"id":"au5","author-id":"S0047248421001019-b821242a67ec09f18716e2f000bc52cb"},"$$":[{"#name":"given-name","_":"Sree"},{"#name":"surname","_":"Kanthaswamy"},{"#name":"cross-ref","$":{"refid":"aff8","id":"crosref0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"h"}]}]},{"#name":"author","$":{"id":"au6","author-id":"S0047248421001019-394f7f3f5a66ce80bd222704882d198c"},"$$":[{"#name":"given-name","_":"Timothy D."},{"#name":"surname","_":"Weaver"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"affiliation","$":{"id":"aff1","affiliation-id":"S0047248421001019-c0f39cf8509d6c346e9f43cdb6de73cd"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"School of Biological and Environmental Sciences, Liverpool John Moores University, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"School of Biological and Environmental Sciences"},{"#name":"organization","_":"Liverpool John Moores University"},{"#name":"country","_":"UK"}]},{"#name":"source-text","$":{"id":"srct0010"},"_":"School of Biological and Environmental Sciences, Liverpool John Moores University, UK"}]},{"#name":"affiliation","$":{"id":"aff2","affiliation-id":"S0047248421001019-b5b2a8f4d7307826ad372e317fa0fce2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Department of Anthropology, University of California Davis, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"University of California Davis"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0015"},"_":"Department of Anthropology, University of California Davis, USA"}]},{"#name":"affiliation","$":{"id":"aff3","affiliation-id":"S0047248421001019-afd722a16bd4077a81d5628185758c82"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"University of Calgary, Cumming School of Medicine, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"University of Calgary"},{"#name":"organization","_":"Cumming School of Medicine"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0020"},"_":"University of Calgary Cumming School of Medicine, Canada"}]},{"#name":"affiliation","$":{"id":"aff4","affiliation-id":"S0047248421001019-9227196579a0c8e6e26614872fa2ef22"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Department of Archaeology, University of Cape Town, South Africa"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Archaeology"},{"#name":"organization","_":"University of Cape Town"},{"#name":"country","_":"South Africa"}]},{"#name":"source-text","$":{"id":"srct0025"},"_":"Department of Archaeology, University of Cape Town, South Africa"}]},{"#name":"affiliation","$":{"id":"aff5","affiliation-id":"S0047248421001019-d873a39736717be42c6abdc4a7c59756"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","_":"Human Evolution Research Institute, University of Cape Town, South Africa"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Human Evolution Research Institute"},{"#name":"organization","_":"University of Cape Town"},{"#name":"country","_":"South Africa"}]},{"#name":"source-text","$":{"id":"srct0030"},"_":"Human Evolution Research Institute, University of Cape Town, South Africa"}]},{"#name":"affiliation","$":{"id":"aff6","affiliation-id":"S0047248421001019-44bd7265f7ffe1742d19bdcfe9217cbe"},"$$":[{"#name":"label","_":"f"},{"#name":"textfn","_":"Department of Integrative Biology, University of California Berkeley, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Integrative Biology"},{"#name":"organization","_":"University of California Berkeley"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0035"},"_":"Department of Integrative Biology, University of California Berkeley, USA"}]},{"#name":"affiliation","$":{"id":"aff7","affiliation-id":"S0047248421001019-89665a70cb6de53dbfe25708537bbe66"},"$$":[{"#name":"label","_":"g"},{"#name":"textfn","_":"Centro Nacional de Investigación sobre la Evolución Humana (CENIEH), Burgos, Spain"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Centro Nacional de Investigación sobre la Evolución Humana (CENIEH)"},{"#name":"city","_":"Burgos"},{"#name":"country","_":"Spain"}]},{"#name":"source-text","$":{"id":"srct0040"},"_":"Centro Nacional de Investigacion sobre la Evolucion Humana (CENIEH), Burgos, Spain"}]},{"#name":"affiliation","$":{"id":"aff8","affiliation-id":"S0047248421001019-b3f10cf96dcb306381f659c1c2677d0c"},"$$":[{"#name":"label","_":"h"},{"#name":"textfn","_":"School of Natural and Mathematical Sciences, Arizona State University, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"School of Natural and Mathematical Sciences"},{"#name":"organization","_":"Arizona State University"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0045"},"_":"School of Natural and Mathematical Sciences, Arizona State University, USA"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"∗"},{"#name":"text","_":"Corresponding author."}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1","affiliation-id":"S0047248421001019-c0f39cf8509d6c346e9f43cdb6de73cd"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"School of Biological and Environmental Sciences, Liverpool John Moores University, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"School of Biological and Environmental Sciences"},{"#name":"organization","_":"Liverpool John Moores University"},{"#name":"country","_":"UK"}]},{"#name":"source-text","$":{"id":"srct0010"},"_":"School of Biological and Environmental Sciences, Liverpool John Moores University, UK"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2","affiliation-id":"S0047248421001019-b5b2a8f4d7307826ad372e317fa0fce2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Department of Anthropology, University of California Davis, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Anthropology"},{"#name":"organization","_":"University of California Davis"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0015"},"_":"Department of Anthropology, University of California Davis, USA"}]},"aff3":{"#name":"affiliation","$":{"id":"aff3","affiliation-id":"S0047248421001019-afd722a16bd4077a81d5628185758c82"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"University of Calgary, Cumming School of Medicine, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"University of Calgary"},{"#name":"organization","_":"Cumming School of Medicine"},{"#name":"country","_":"Canada"}]},{"#name":"source-text","$":{"id":"srct0020"},"_":"University of Calgary Cumming School of Medicine, Canada"}]},"aff4":{"#name":"affiliation","$":{"id":"aff4","affiliation-id":"S0047248421001019-9227196579a0c8e6e26614872fa2ef22"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Department of Archaeology, University of Cape Town, South Africa"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Archaeology"},{"#name":"organization","_":"University of Cape Town"},{"#name":"country","_":"South Africa"}]},{"#name":"source-text","$":{"id":"srct0025"},"_":"Department of Archaeology, University of Cape Town, South Africa"}]},"aff5":{"#name":"affiliation","$":{"id":"aff5","affiliation-id":"S0047248421001019-d873a39736717be42c6abdc4a7c59756"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","_":"Human Evolution Research Institute, University of Cape Town, South Africa"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Human Evolution Research Institute"},{"#name":"organization","_":"University of Cape Town"},{"#name":"country","_":"South Africa"}]},{"#name":"source-text","$":{"id":"srct0030"},"_":"Human Evolution Research Institute, University of Cape Town, South Africa"}]},"aff6":{"#name":"affiliation","$":{"id":"aff6","affiliation-id":"S0047248421001019-44bd7265f7ffe1742d19bdcfe9217cbe"},"$$":[{"#name":"label","_":"f"},{"#name":"textfn","_":"Department of Integrative Biology, University of California Berkeley, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Integrative Biology"},{"#name":"organization","_":"University of California Berkeley"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0035"},"_":"Department of Integrative Biology, University of California Berkeley, USA"}]},"aff7":{"#name":"affiliation","$":{"id":"aff7","affiliation-id":"S0047248421001019-89665a70cb6de53dbfe25708537bbe66"},"$$":[{"#name":"label","_":"g"},{"#name":"textfn","_":"Centro Nacional de Investigación sobre la Evolución Humana (CENIEH), Burgos, Spain"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Centro Nacional de Investigación sobre la Evolución Humana (CENIEH)"},{"#name":"city","_":"Burgos"},{"#name":"country","_":"Spain"}]},{"#name":"source-text","$":{"id":"srct0040"},"_":"Centro Nacional de Investigacion sobre la Evolucion Humana (CENIEH), Burgos, Spain"}]},"aff8":{"#name":"affiliation","$":{"id":"aff8","affiliation-id":"S0047248421001019-b3f10cf96dcb306381f659c1c2677d0c"},"$$":[{"#name":"label","_":"h"},{"#name":"textfn","_":"School of Natural and Mathematical Sciences, Arizona State University, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"School of Natural and Mathematical Sciences"},{"#name":"organization","_":"Arizona State University"},{"#name":"country","_":"USA"}]},{"#name":"source-text","$":{"id":"srct0045"},"_":"School of Natural and Mathematical Sciences, Arizona State University, USA"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"∗"},{"#name":"text","_":"Corresponding author."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"authorMetadata":[],"banner":{"expanded":false},"biographies":{},"body":{},"chapters":{"toc":[],"isLoading":false},"changeViewLinks":{"showFullTextLink":false,"showAbstractLink":false},"citingArticles":{},"clientConfig":{"adobeSuite":"elsevier-sd-prod","arsUrl":"https://ars.els-cdn.com","assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1","recommendationsFeedback":{"enabled":true,"url":"https://feedback.recs.d.elsevier.com/raw/events","timeout":60000},"googleMapsApiKey":"AIzaSyCBYU6I6lrbEU6wQXUEIte3NwGtm3jwHQc","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","xocsMediaPlayerBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/147/js/ableplayer","strictMode":false,"seamlessAccess":{"enableSeamlessAccess":true,"scriptUrl":"https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js","persistenceUrl":"https://service.seamlessaccess.org/ps/","persistenceContext":"seamlessaccess.org","scienceDirectUrl":"https://www.sciencedirect.com","shibAuthUrl":"https://auth.elsevier.com/ShibAuth/institutionLogin"},"reaxys":{"apiUrl":"https://reaxys-sdlc.reaxys.com","origin":"sciencedirect","queryBuilderHostPath":"https://www.reaxys.com/reaxys/secured/hopinto.do","url":"https://www.reaxys.com"},"oneTrustCookie":{"enabled":true},"ssrn":{"url":"https://papers.ssrn.com","path":"/sol3/papers.cfm"},"aiNavigation":{"enabled":true,"version":"V1"}},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"lang":"en","id":"kwrds0010","view":"all","class":"keyword"},"$$":[{"#name":"section-title","$":{"id":"sectitle0015"},"_":"Keywords"},{"#name":"keyword","$":{"id":"kwrd0010"},"$$":[{"#name":"text","_":"Hybridization"}]},{"#name":"keyword","$":{"id":"kwrd0015"},"$$":[{"#name":"text","_":"Human evolution"}]},{"#name":"keyword","$":{"id":"kwrd0020"},"$$":[{"#name":"text","_":"Primate"}]},{"#name":"keyword","$":{"id":"kwrd0025"},"$$":[{"#name":"text","_":"Skeletal morphology"}]},{"#name":"keyword","$":{"id":"kwrd0030"},"$$":[{"#name":"text","_":"Geometric morphometric methods"}]},{"#name":"keyword","$":{"id":"kwrd0035"},"$$":[{"#name":"text","$$":[{"#name":"italic","_":"Macaca mulatta"}]}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"crossMark":{"isOpen":false},"domainConfig":{"assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1","cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com"},"downloadIssue":{"openOnPageLoad":false,"isOpen":false,"downloadCapOpen":false,"articles":[],"selected":[]},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"entitledRecommendations":{"openOnPageLoad":false,"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"exam":{},"helpText":{"keyDates":{"html":"<div class=\"key-dates-help\"><h3 class=\"u-margin-s-bottom u-h4\">Publication milestones</h3><p class=\"u-margin-m-bottom\">The dates displayed for an article provide information on when various publication milestones were reached at the journal that has published the article. Where applicable, activities on preceding journals at which the article was previously under consideration are not shown (for instance submission, revisions, rejection).</p><p class=\"u-margin-xs-bottom\">The publication milestones include:</p><ul class=\"key-dates-help-list u-margin-m-bottom u-padding-s-left\"><li><span class=\"u-text-italic\">Received</span>: The date the article was originally submitted to the journal.</li><li><span class=\"u-text-italic\">Revised</span>: The date the most recent revision of the article was submitted to the journal. Dates corresponding to intermediate revisions are not shown.</li><li><span class=\"u-text-italic\">Accepted</span>: The date the article was accepted for publication in the journal.</li><li><span class=\"u-text-italic\">Available online</span>: The date a version of the article was made available online in the journal.</li><li><span class=\"u-text-italic\">Version of Record</span>: The date the finalized version of the article was made available in the journal.</li></ul><p>More information on publishing policies can be found on the <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://www.elsevier.com/about/policies-and-standards/publishing-ethics\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">Publishing Ethics Policies</span></span></a> page. View our <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://www.elsevier.com/researcher/author/submit-your-paper\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">Publishing with Elsevier: step-by-step</span></span></a> page to learn more about the publishing process. For any questions on your own submission or other questions related to publishing an article, <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://service.elsevier.com/app/phone/supporthub/publishing\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">contact our Researcher support team.</span></span></a></p></div>","title":"What do these dates mean?"}},"glossary":{},"issueNavigation":{"previous":{},"next":{}},"linkingHubLinks":{},"metrics":{"metricGroup":{"citations":[],"captures":[],"mentions":[],"socialMedia":[]},"isLoading":false,"error":false,"hasMetrics":false},"preview":{},"rawtext":"","recommendations":{},"references":{},"referenceLinks":{"internal":[],"internalLoaded":false,"external":[]},"refersTo":{},"referredToBy":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenVirtualSpecialIssueLink":false,"isOpenRecommendations":true,"isOpenSubstances":true,"citingArticles":[false,false,false,false,false,false],"recommendations":[false,false,false,false,false,false]},"seamlessAccess":{},"specialIssueArticles":{},"substances":{},"supplementaryFilesData":[],"tableOfContents":{"showEntitledTocLinks":true},"tail":{},"transientError":{"isOpen":false},"sidePanel":{"openState":1},"viewConfig":{"articleFeature":{"aiNavigation":false},"pathPrefix":""},"virtualSpecialIssue":{"showVirtualSpecialIssueLink":false},"usageProps":{"itemStage":"S300","isAip":false,"tombAip":"0","sample":false},"userCookiePreferences":{"STRICTLY_NECESSARY":true,"PERFORMANCE":false,"FUNCTIONAL":false,"TARGETING":false}};
      </script>
      <noscript>
      JavaScript is disabled on your browser.
      Please enable JavaScript to use all the features on this page.
      <img src=https://smetrics.elsevier.com/b/ss/elsevier-sd-prod/1/G.4--NS/1749596714512?pageName=sd%3Aproduct%3Ajournal%3Aarticle&c16=els%3Arp%3Ast&c2=sd&v185=img&v33=ae%3AANON_GUEST&c1=ae%3A228598&c12=ae%3A12975512 />
    </noscript>
      <a class="anchor sr-only sr-only-focusable u-display-inline anchor-primary" href="#screen-reader-main-content"><span class="anchor-text-container"><span class="anchor-text">Skip to main content</span></span></a><a class="anchor sr-only sr-only-focusable u-display-inline anchor-primary" href="#screen-reader-main-title"><span class="anchor-text-container"><span class="anchor-text">Skip to article</span></span></a>
      <div id="root"><div class="App" id="app" data-aa-name="root"><div class="page"><div class="sd-flex-container"><div class="sd-flex-content"><header id="gh-cnt"><div id="gh-main-cnt" class="u-flex-center-ver u-position-relative u-padding-s-hor u-padding-l-hor-from-xl"><a id="gh-branding" class="u-flex-center-ver" href="/" aria-label="ScienceDirect home page" data-aa-region="header" data-aa-name="ScienceDirect"><img class="gh-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/24/images/elsevier-non-solus-new-grey.svg" alt="Elsevier logo" height="48" width="54"><svg xmlns="http://www.w3.org/2000/svg" version="1.1" height="15" viewBox="0 0 190 23" role="img" class="gh-wordmark u-margin-s-left" aria-labelledby="gh-wm-science-direct" focusable="false" aria-hidden="true" alt="ScienceDirect Wordmark"><title id="gh-wm-science-direct">ScienceDirect</title><g><path fill="#EB6500" d="M3.81 6.9c0-1.48 0.86-3.04 3.7-3.04 1.42 0 3.1 0.43 4.65 1.32l0.13-2.64c-1.42-0.63-2.97-0.96-4.78-0.96 -4.62 0-6.6 2.44-6.6 5.45 0 5.61 8.78 6.14 8.78 9.93 0 1.48-1.15 3.04-3.86 3.04 -1.72 0-3.4-0.56-4.72-1.39l-0.36 2.64c1.55 0.76 3.57 1.06 5.15 1.06 4.26 0 6.7-2.48 6.7-5.51C12.59 11.49 3.81 10.76 3.81 6.9M20.27 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.14 0-4.55-1.71-4.55-5.91C17.93 10.2 20.01 9.18 20.27 9.01"></path><rect x="29.42" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M30.67 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.68-0.96 1.68-1.88C32.35 1.55 31.56 0.7 30.67 0.7M48.06 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H48.06M39.91 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C38.56 10.27 39.71 9.37 39.91 9.18zM58.82 6.57c-2.24 0-3.63 1.12-4.85 2.61l-0.4-2.21h-2.34l0.13 1.19c0.1 0.76 0.13 1.78 0.13 2.97v10.79h2.54V11.88c0.69-0.96 2.15-2.48 2.48-2.64 0.23-0.13 1.29-0.4 2.08-0.4 2.28 0 2.48 1.15 2.54 3.43 0.03 1.19 0.03 3.17 0.03 3.17 0.03 3-0.1 6.47-0.1 6.47h2.54c0 0 0.07-4.49 0.07-6.96 0-1.48 0.03-2.97-0.1-4.46C63.31 7.43 61.49 6.57 58.82 6.57M72.12 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C69.77 10.2 71.85 9.18 72.12 9.01M92.74 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H92.74M84.59 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C83.24 10.27 84.39 9.37 84.59 9.18zM103.9 1.98h-7.13v19.93h6.83c7.26 0 9.77-5.68 9.77-10.03C113.37 7.33 110.93 1.98 103.9 1.98M103.14 19.8h-3.76V4.1h4.09c5.38 0 6.96 4.39 6.96 7.79C110.43 16.87 108.19 19.8 103.14 19.8zM118.38 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.69-0.96 1.69-1.88C120.07 1.55 119.28 0.7 118.38 0.7"></path><rect x="117.13" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M130.2 6.6c-1.62 0-2.87 1.45-3.4 2.74l-0.43-2.37h-2.34l0.13 1.19c0.1 0.76 0.13 1.75 0.13 2.9v10.86h2.54v-9.51c0.53-1.29 1.72-3.7 3.17-3.7 0.96 0 1.06 0.99 1.06 1.22l2.08-0.6V9.18c0-0.03-0.03-0.17-0.06-0.4C132.8 7.36 131.91 6.6 130.2 6.6M145.87 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.89-1.95-4.89-5.51v-0.49H145.87M137.72 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C136.37 10.27 137.52 9.37 137.72 9.18zM153.23 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.14-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.69 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C150.89 10.2 152.97 9.18 153.23 9.01M170 19.44c-0.92 0.36-1.72 0.69-2.51 0.69 -1.16 0-1.58-0.66-1.58-2.34V8.95h3.93V6.97h-3.93V2.97h-2.48v3.99h-2.71v1.98h2.71v9.67c0 2.64 1.39 3.73 3.33 3.73 1.15 0 2.54-0.39 3.43-0.79L170 19.44M173.68 5.96c-1.09 0-2-0.87-2-1.97 0-1.1 0.91-1.97 2-1.97s1.98 0.88 1.98 1.98C175.66 5.09 174.77 5.96 173.68 5.96zM173.67 2.46c-0.85 0-1.54 0.67-1.54 1.52 0 0.85 0.69 1.54 1.54 1.54 0.85 0 1.54-0.69 1.54-1.54C175.21 3.13 174.52 2.46 173.67 2.46zM174.17 5.05c-0.09-0.09-0.17-0.19-0.25-0.3l-0.41-0.56h-0.16v0.87h-0.39V2.92c0.22-0.01 0.47-0.03 0.66-0.03 0.41 0 0.82 0.16 0.82 0.64 0 0.29-0.21 0.55-0.49 0.63 0.23 0.32 0.45 0.62 0.73 0.91H174.17zM173.56 3.22l-0.22 0.01v0.63h0.22c0.26 0 0.43-0.05 0.43-0.34C174 3.28 173.83 3.21 173.56 3.22z"></path></g></svg></a><div class="gh-nav-cnt u-hide-from-print"><div class="gh-nav-links-container gh-nav-links-container-h u-hide-from-print gh-nav-content-container"><nav aria-label="links" class="gh-nav gh-nav-links gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item gh-move-to-spine"><a class="anchor gh-nav-action text-s anchor-secondary anchor-medium" href="/browse/journals-and-books" id="gh-journals-books-link" data-aa-region="header" data-aa-name="Journals &amp; Books"><span class="anchor-text-container"><span class="anchor-text">Journals &amp; Books</span></span></a></li></ul></nav><nav aria-label="utilities" class="gh-nav gh-nav-utilities gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-help text-s u-flex-center-ver u-gap-6 gh-nav-action"><div class="gh-move-to-spine gh-help-button gh-help-icon gh-nav-item"><div class="popover" id="gh-help-icon-popover"><div id="popover-trigger-gh-help-icon-popover"><input type="hidden"><button class="button-link button-link-secondary gh-icon-btn button-link-medium button-link-icon-left" type="button" aria-expanded="false"><svg focusable="false" viewBox="0 0 114 128" height="20" width="20" class="icon icon-help gh-icon"><path d="M57 8C35.69 7.69 15.11 21.17 6.68 40.71c-8.81 19.38-4.91 43.67 9.63 59.25 13.81 15.59 36.85 21.93 56.71 15.68 21.49-6.26 37.84-26.81 38.88-49.21 1.59-21.15-10.47-42.41-29.29-52.1C74.76 10.17 65.88 7.99 57 8zm0 10c20.38-.37 39.57 14.94 43.85 34.85 4.59 18.53-4.25 39.23-20.76 48.79-17.05 10.59-40.96 7.62-54.9-6.83-14.45-13.94-17.42-37.85-6.83-54.9C26.28 26.5 41.39 17.83 57 18zm-.14 14C45.31 32.26 40 40.43 40 50v2h10v-2c0-4.22 2.22-9.66 8-9.24 5.5.4 6.32 5.14 5.78 8.14C62.68 55.06 52 58.4 52 69.4V76h10v-5.56c0-8.16 11.22-11.52 12-21.7.74-9.86-5.56-16.52-16-16.74-.39-.01-.76-.01-1.14 0zM52 82v10h10V82H52z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Help</span></span></button></div></div></div></li><li class="gh-nav-search text-s u-flex-center-ver u-gap-6 gh-nav-action"><div class="gh-search-toggle gh-nav-item search-button-link"><a class="anchor button-link-secondary anchor-secondary u-margin-l-left gh-nav-action gh-icon-btn anchor-medium anchor-icon-left anchor-with-icon" href="/search" id="gh-search-link" data-aa-button="search-in-header-opened-from-article" role="button"><svg focusable="false" viewBox="0 0 100 128" height="20" class="icon icon-search gh-icon"><path d="M19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6L73.23 80.19c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96S19.9 18.47 12.19 26.18C4.47 33.89.22 44.15.22 55.06s4.25 21.17 11.97 28.88C19.9 91.65 30.16 95.9 41.07 95.9c9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg><span class="anchor-text-container"><span class="anchor-text">Search</span></span></a></div></li></ul></nav></div></div><div class="gh-profile-container gh-move-to-spine u-hide-from-print"><a class="anchor text-s u-clr-grey8 u-margin-l-left gh-icon-btn anchor-primary anchor-medium anchor-icon-left anchor-with-icon" href="/user/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0047248421001019&amp;from=globalheader" id="gh-myaccount-btn" data-aa-region="header" data-aa-name="personalsignin"><svg focusable="false" viewBox="0 0 106 128" height="20" aria-hidden="true" class="icon icon-person gh-cta-btn-icon"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><span class="anchor-text-container"><span class="anchor-text">My account</span></span></a></div><a class="anchor text-s u-clr-grey8 gh-move-to-spine u-hide-from-print u-margin-l-left anchor-secondary gh-icon-btn anchor-medium anchor-icon-left anchor-with-icon" href="/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0047248421001019" id="gh-institutionalsignin-btn" data-aa-region="header" data-aa-name="institutionalsignin"><svg focusable="false" viewBox="0 0 106 128" height="20" aria-hidden="true" class="icon icon-institution gh-cta-btn-icon"><path d="M84 98h10v10H12V98h10V52h14v46h10V52h14v46h10V52h14v46zM12 36.86l41-20.84 41 20.84V42H12v-5.14zM104 52V30.74L53 4.8 2 30.74V52h10v36H2v30h102V88H94V52h10z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Sign in</span></span></a><div id="gh-mobile-menu" class="mobile-menu u-hide-from-print"><div class="gh-hamburger u-fill-grey7"><button class="button-link u-flex-center-ver button-link-primary button-link-icon-left" type="button" aria-label="Toggle mobile menu" aria-expanded="false"><svg class="gh-hamburger-svg-el gh-hamburger-closed" role="img" aria-hidden="true" height="20" width="20"><path d="M0 14h40v2H0zm0-7h40v2H0zm0-7h40v2H0z"></path></svg></button></div><div id="gh-overlay" class="mobile-menu-overlay u-overlay u-display-none" role="button" tabindex="-1"></div><div id="gh-drawer" aria-label="Mobile menu" class="" role="navigation"></div></div></div></header><div id="sd-ai-navigation-container"></div><div id="reading-assistant-main-body-section"></div><div class="Article" id="mathjax-container" role="main"><div class="accessbar-sticky"><div id="screen-reader-main-content"></div><div role="region" aria-label="Download options and search"><div class="accessbar"><div class="accessbar-label"></div><ul aria-label="PDF Options"><li class="accessbar-item-show-from-initial accessbar-item-show-from-xs accessbar-item-show-from-md ViewPDF"><a class="link-button accessbar-utility-component accessbar-utility-link link-button-primary link-button-icon-left" target="_blank" aria-label="View PDF. Opens in a new window." href="/science/article/pii/S0047248421001019/pdfft?md5=08c4f8c5a04c804cb19a38c138b63424&amp;pid=1-s2.0-S0047248421001019-main.pdf" rel="nofollow"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="link-button-text-container"><span class="link-button-text"><span>View&nbsp;<strong>PDF</strong></span></span></span></a></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md DownloadFullIssue"><button class="button-link accessbar-utility-component button-link-primary" type="button" aria-label="Download full issue"><span class="button-link-text-container"><span class="button-link-text"><span>Download full issue</span></span></span></button></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md Divider"><span class="accessbar-divider"></span></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md OpenManuscript"><a class="anchor accessbar-utility-component accessbar-utility-link anchor-primary" href="/science/article/am/pii/S0047248421001019" target="_blank" aria-label="View Open Manuscript"><span class="anchor-text-container"><span class="anchor-text"><span>View Open Manuscript</span></span></span></a></li><li class="accessbar-item-show-from-initial accessbar-item-show-from-xs accessbar-item-hide-from-md OverflowPopover"><div class="popover accessbar-overflow-popover" id="OverflowPopoverAnchor"><div id="popover-trigger-OverflowPopoverAnchor"><button class="button-link accessbar-utility-component button-link-primary button-link-icon-right" type="button" aria-label="Other access options"><span class="button-link-text-container"><span class="button-link-text"><span>Other access options</span></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div></li></ul><form class="QuickSearch" action="/search#submit" method="get" aria-label="form"><div class="search-input"><div class="search-input-container search-input-container-no-label"><label class="search-input-label u-hide-visually" for="article-quick-search">Search ScienceDirect</label><input type="search" id="article-quick-search" name="qs" class="search-input-field" aria-describedby="article-quick-search-description-message" aria-invalid="false" aria-label="Search ScienceDirect" placeholder="Search ScienceDirect" value=""></div><div class="search-input-message-container"><div class="search-input-validation-error" aria-live="polite"></div><div id="article-quick-search-description-message"></div></div></div><button type="submit" class="button u-margin-xs-left button-primary small button-icon-only" aria-disabled="false" aria-label="Submit search"><svg focusable="false" viewBox="0 0 100 128" height="20" class="icon icon-search"><path d="M19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6L73.23 80.19c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96S19.9 18.47 12.19 26.18C4.47 33.89.22 44.15.22 55.06s4.25 21.17 11.97 28.88C19.9 91.65 30.16 95.9 41.07 95.9c9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></button><input type="hidden" name="origin" value="article"><input type="hidden" name="zone" value="qSearch"></form></div></div></div><div class="article-wrapper grid row"><div role="navigation" class="u-display-block-from-lg col-lg-6 u-padding-s-top sticky-table-of-contents" aria-label="Table of contents"><div class="TableOfContents" lang="en"><div class="Outline" id="toc-outline"><h2 class="u-h4">Outline</h2><ol class="u-padding-xs-bottom"><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#abs0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Abstract"><span class="anchor-text-container"><span class="anchor-text">Abstract</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#kwrds0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Keywords"><span class="anchor-text-container"><span class="anchor-text">Keywords</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec1" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="1. Introduction"><span class="anchor-text-container"><span class="anchor-text">1. Introduction</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec2" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="2. Materials and methods"><span class="anchor-text-container"><span class="anchor-text">2. Materials and methods</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec3" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="3. Results"><span class="anchor-text-container"><span class="anchor-text">3. Results</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec4" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="4. Discussion"><span class="anchor-text-container"><span class="anchor-text">4. Discussion</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#sec5" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="5. Conclusions"><span class="anchor-text-container"><span class="anchor-text">5. Conclusions</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#p00101" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Conflict of interest statement"><span class="anchor-text-container"><span class="anchor-text">Conflict of interest statement</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#ack0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Acknowledgments"><span class="anchor-text-container"><span class="anchor-text">Acknowledgments</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#appsec1" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Appendix A. Supplementary Online Material"><span class="anchor-text-container"><span class="anchor-text">Appendix A. Supplementary Online Material</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="#cebib0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="References"><span class="anchor-text-container"><span class="anchor-text">References</span></span></a></li></ol><button class="button-link u-margin-xs-top u-margin-s-bottom button-link-primary button-link-icon-right" type="button" aria-expanded="false" data-aa-button="sd:product:journal:article:type=menu:name=show-full-outline"><span class="button-link-text-container"><span class="button-link-text">Show full outline</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="CitedBy" id="toc-cited-by"><h2 class="u-h4"><a class="anchor anchor-primary" href="#section-cited-by"><span class="anchor-text-container"><span class="anchor-text">Cited by (11)</span></span></a></h2><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2 class="u-h4">Figures (7)</h2><ol class="u-margin-s-bottom"><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig1" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure1. Landmarks and semilandmarks used in the analyses presented here viewed from…" class="u-display-block" height="164px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr1.sml" width="153px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig2" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure2. Semilandmark curves and wireframe used in the analyses presented here, viewed…" class="u-display-block" height="100px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr2.sml" width="219px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig3" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure3. Distributions of pairwise Procrustes distances generated directly from the data" class="u-display-block" height="164px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr3.sml" width="157px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig4" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure4. Plot of regression scores showing the relationship between pelvic shape and…" class="u-display-block" height="163px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr4.sml" width="129px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig5" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure5. PCA plot showing sex-adjusted shape differences between full-bred species" class="u-display-block" height="164px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr5.sml" width="185px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="#fig6" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure6. Regression score plot showing the relationship between sex-adjusted shape…" class="u-display-block" height="164px" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr6.sml" width="147px"></a></li></ol><button class="button-link u-margin-xs-top u-margin-s-bottom button-link-primary button-link-icon-right" type="button" data-aa-button="sd:product:journal:article:type=menu:name=show-figures"><span class="button-link-text-container"><span class="button-link-text">Show 1 more figure</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2 class="u-h4">Tables (8)</h2><ol class="u-padding-s-bottom"><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl1" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Breakdown of sample (n=138) by Chinese ancestry and sex."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl2" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Numbers and definitions of all landmarks (1–48) and semilandmarks (49–204)."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;2</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl3" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Comparison (analysis of variance) of interactive and additive models for the relationship between sex, admixture (percent Chinese ancestry), and shape in the complete sample (n=138)."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;3</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl4" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Model comparison (analysis of variance) of three-knot B spline and simple additive models for the relationship between shape and admixture in the complete sample (n= 138, sex held constant for both va..."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;4</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl5" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Analysis of variance of differences in centroid size between full-bred and admixed animals in the complete sample (n=138)."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;5</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#tbl6" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Procrustes variances for admixture categories in the complete sample (n=138)."><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;6</span></span></a></li></ol><button class="button-link u-margin-xs-top u-margin-s-bottom button-link-primary button-link-icon-right" type="button" data-aa-button="sd:product:journal:article:type=menu:name=show-tables"><span class="button-link-text-container"><span class="button-link-text">Show all tables</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Extras" id="toc-extras"><h2 class="u-h4">Extras (3)</h2><ol class="u-padding-s-bottom"><li><span class="download-all-supplemental-data"><span class="in-toc"><iframe title="download_all" name="download_all" class="u-hide"></iframe><form method="post" action="/sdfe/pdf/download/S0047248421001019/attachments" target="download_all"><input name="eids" type="hidden" value="1-s2.0-S0047248421001019-mmc1.pdf"><input name="eids" type="hidden" value="1-s2.0-S0047248421001019-mmc2.zip"><input name="eids" type="hidden" value="1-s2.0-S0047248421001019-mmc3.zip"><button class="button-link download-all-button u-font-sans u-margin-xs-bottom button-link-primary button-link-icon-left" type="submit"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-download"><path d="M77.38 56.18l-6.6-7.06L54 66.36V26H44v40.34L26.28 49.1l-7.3 7.08 29.2 29.32 29.2-29.32M88 74v20H10V74H0v30h98V74H88"></path></svg><span class="button-link-text-container"><span class="button-link-text"><span class="download-all-title">Download all </span></span></span></button></form></span></span></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#mmc1" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Multimedia component 1"><svg focusable="false" viewBox="0 0 95 128" height="20" class="icon icon-pdf-download"><path d="M82 108H12V59c0-6.08 4.92-11 11-11h17V28h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06H82v40h10V10H35.6c-5.38 0-10.62 1.92-14.76 5.4C11.74 23.08 2 35.54 2 47.5V118h90v-18H82v8zM56.06 68.6c-.84-.38-1.84-.56-2.98-.56h-9.04v23.6h5.94v-9h2.18c2.48 0 4.36-.62 5.66-1.84s1.92-3.06 1.92-5.5c0-1.04-.12-2-.4-2.88-.26-.88-.66-1.64-1.22-2.3s-1.22-1.14-2.06-1.52zm-3.12 8.94c-.4.46-.98.7-1.74.7h-1.22v-5.76h1.22c1.56 0 2.34.96 2.34 2.88 0 .98-.2 1.72-.6 2.18zm21.84-8.52c-.96-.66-2.32-.98-4.06-.98H62v23.6h8.72c1.74 0 3.1-.32 4.06-.98s1.7-1.52 2.18-2.62.78-2.34.88-3.76.16-2.9.16-4.44-.06-3.02-.16-4.44-.4-2.68-.88-3.76a5.94 5.94 0 0 0-2.18-2.62zM72 83.68c-.06.96-.18 1.72-.38 2.24s-.48.88-.84 1.04-.86.24-1.48.24h-1.32V72.46h1.32c.62 0 1.12.08 1.48.24s.64.52.84 1.04.32 1.28.38 2.24c.06.98.08 2.24.08 3.84s-.02 2.9-.08 3.86zm13.98-6.58v-4.02h7.74v-5.04h-13.7v23.6h5.96v-9.72h7.26V77.1z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Multimedia component 1</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#mmc2" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Multimedia component 2"><svg focusable="false" viewBox="0 0 95 128" height="20" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Multimedia component 2</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="#mmc3" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Multimedia component 3"><svg focusable="false" viewBox="0 0 95 128" height="20" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Multimedia component 3</span></span></a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right u-padding-s-top" lang="en"><div class="Publication" id="publication"><div class="publication-brand u-display-block-from-sm"><a class="anchor u-display-flex anchor-primary" href="/journal/journal-of-human-evolution" title="Go to Journal of Human Evolution on ScienceDirect"><span class="anchor-text-container"><span class="anchor-text"><img class="publication-brand-image" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/image/elsevier-non-solus.svg" alt="Elsevier"></span></span></a></div><div class="publication-volume u-text-center"><h2 class="publication-title u-h3" id="publication-title"><a class="anchor anchor-secondary publication-title-link" href="/journal/journal-of-human-evolution" title="Go to Journal of Human Evolution on ScienceDirect"><span class="anchor-text-container"><span class="anchor-text">Journal of Human Evolution</span></span></a></h2><div class="text-xs"><a class="anchor anchor-primary" href="/journal/journal-of-human-evolution/vol/159/suppl/C" title="Go to table of contents for this volume/issue"><span class="anchor-text-container"><span class="anchor-text">Volume 159</span></span></a>, <!-- -->October 2021<!-- -->, 103049</div></div><div class="publication-cover u-display-block-from-sm"><a class="anchor u-display-flex anchor-primary" href="/journal/journal-of-human-evolution/vol/159/suppl/C"><span class="anchor-text-container"><span class="anchor-text"><img class="publication-cover-image" src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421X00097-cov150h.gif" alt="Journal of Human Evolution"></span></span></a></div></div><h1 id="screen-reader-main-title" class="Head u-font-serif u-h2 u-margin-s-ver"><span class="title-text">Effects of hybridization on pelvic morphology: A macaque model</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div aria-live="polite"></div><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au1"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Laura T.</span> <span class="text surname">Buck</span> </span><span class="author-ref" id="baff1"><sup>a</sup></span> <span class="author-ref" id="baff2"><sup>b</sup></span><svg focusable="false" viewBox="0 0 106 128" height="20" title="Correspondence author icon" class="icon icon-person react-xocs-author-icon u-fill-grey8"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><svg focusable="false" viewBox="0 0 102 128" height="20" title="Author email or social media contact details icon" class="icon icon-envelope react-xocs-author-icon u-fill-grey8"><path d="M55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0L17.58 34h69.54L55.8 57.19zM0 32.42l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-.98 9.42-2.93L102 34.34V24H0zM92 88.9L73.94 66.16l-8.04 5.95L83.28 94H18.74l18.38-23.12-8.04-5.96L10 88.94V51.36L0 42.9V104h102V44.82l-10 8.46V88.9"></path></svg></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au2"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">David C.</span> <span class="text surname">Katz</span> </span><span class="author-ref" id="baff2"><sup>b</sup></span> <span class="author-ref" id="baff3"><sup>c</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au3"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Rebecca Rogers</span> <span class="text surname">Ackermann</span> </span><span class="author-ref" id="baff4"><sup>d</sup></span> <span class="author-ref" id="baff5"><sup>e</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au4"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Leslea J.</span> <span class="text surname">Hlusko</span> </span><span class="author-ref" id="baff6"><sup>f</sup></span> <span class="author-ref" id="baff7"><sup>g</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au5"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Sree</span> <span class="text surname">Kanthaswamy</span> </span><span class="author-ref" id="baff8"><sup>h</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" type="button" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au6"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Timothy D.</span> <span class="text surname">Weaver</span> </span><span class="author-ref" id="baff2"><sup>b</sup></span></span></span></button></div></div></div><button class="button-link u-margin-s-ver button-link-primary button-link-icon-right" type="button" id="show-more-btn" data-aa-button="icon-expand"><span class="button-link-text-container"><span class="button-link-text">Show more</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="banner-options u-padding-xs-bottom"><div class="toc-button-wrap u-display-inline-block u-display-none-from-lg u-margin-s-right"><button class="button-link button-link-secondary button-link-icon-left button-link-has-colored-icon" type="button"><svg focusable="false" viewBox="0 0 128 128" height="20" class="icon icon-list"><path d="M23 26a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V30zM23 56a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V60zM23 86a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V90z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Outline</span></span></button></div><button class="button-link AddToMendeley button-link-secondary u-margin-s-right u-display-inline-flex-from-md button-link-icon-left button-link-has-colored-icon" type="button"><svg focusable="false" viewBox="0 0 86 128" height="20" class="icon icon-plus"><path d="M48 58V20H38v38H0v10h38v38h10V68h38V58z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Add to Mendeley</span></span></button><div class="Social u-display-inline-block" id="social"><div class="popover social-popover" id="social-popover"><div id="popover-trigger-social-popover"><button class="button-link button-link-secondary u-margin-s-right button-link-icon-left button-link-has-colored-icon" type="button" aria-expanded="false" aria-haspopup="true"><svg focusable="false" viewBox="0 0 114 128" height="20" class="icon icon-share"><path d="M90 112c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zM24 76c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zm66-60c6.62 0 12 5.38 12 12s-5.38 12-12 12-12-5.38-12-12 5.38-12 12-12zm0 62c-6.56 0-12.44 2.9-16.48 7.48L45.1 70.2c.58-1.98.9-4.04.9-6.2s-.32-4.22-.9-6.2l28.42-15.28C77.56 47.1 83.44 50 90 50c12.14 0 22-9.86 22-22S102.14 6 90 6s-22 9.86-22 22c0 1.98.28 3.9.78 5.72L40.14 49.1C36.12 44.76 30.38 42 24 42 11.86 42 2 51.86 2 64s9.86 22 22 22c6.38 0 12.12-2.76 16.14-7.12l28.64 15.38c-.5 1.84-.78 3.76-.78 5.74 0 12.14 9.86 22 22 22s22-9.86 22-22-9.86-22-22-22z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Share</span></span></button></div></div></div><div class="ExportCitation u-display-inline-block" id="export-citation"><div class="popover export-citation-popover" id="export-citation-popover"><div id="popover-trigger-export-citation-popover"><button class="button-link button-link-secondary button-link-icon-left button-link-has-colored-icon" type="button" aria-expanded="false" aria-haspopup="true"><svg focusable="false" viewBox="0 0 104 128" height="20" class="icon icon-cited-by-66"><path d="M2 58.78V106h44V64H12v-5.22C12 40.28 29.08 32 46 32V22C20.1 22 2 37.12 2 58.78zM102 32V22c-25.9 0-44 15.12-44 36.78V106h44V64H68v-5.22C68 40.28 85.08 32 102 32z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Cite</span></span></button></div></div></div></div></div><div class="ArticleIdentifierLinks u-margin-xs-bottom text-xs" id="article-identifier-links"><a class="anchor doi anchor-primary" href="https://doi.org/10.1016/j.jhevol.2021.103049" target="_blank" rel="noreferrer noopener" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier"><span class="anchor-text-container"><span class="anchor-text">https://doi.org/10.1016/j.jhevol.2021.103049</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor rights-and-content anchor-primary" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0047248421001019&amp;orderBeanReset=true" target="_blank" rel="noreferrer noopener"><span class="anchor-text-container"><span class="anchor-text">Get rights and content</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div><div class="LicenseInfo text-xs u-margin-xs-bottom"><div class="License"><span>Under a Creative Commons </span><a class="anchor anchor-primary" href="http://creativecommons.org/licenses/by-nc-nd/4.0/" target="_blank" rel="noreferrer noopener"><span class="anchor-text-container"><span class="anchor-text">license</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div><div class="AccessLabel"><span class="access-indicator"></span>Open access</div></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider border-black-mobile"></div><div class="Abstracts u-font-serif" id="abstracts"><div class="abstract author" id="abs0010" lang="en"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Abstract</h2><div id="abssec0010"><div class="u-margin-s-bottom" id="abspara0010"><span><span><span>Ancient DNA analyses have shown that interbreeding between <a href="/topics/biochemistry-genetics-and-molecular-biology/hominin" title="Learn more about hominin from ScienceDirect's AI-generated Topic Pages" class="topic-link">hominin</a> taxa occurred multiple times. Although admixture is often reflected in skeletal phenotype, the relationship between the two remains poorly understood, hampering interpretation of the </span><a href="/topics/biochemistry-genetics-and-molecular-biology/fossil-hominin" title="Learn more about hominin fossil from ScienceDirect's AI-generated Topic Pages" class="topic-link">hominin fossil</a> record. Direct study of this relationship is often impossible due to the paucity of hominin fossils and difficulties retrieving ancient </span><a href="/topics/biochemistry-genetics-and-molecular-biology/genetics" title="Learn more about genetic from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic</a> material. Here, we use a sample of known ancestry hybrids between two closely related nonhuman primate taxa (Indian and Chinese </span><span><span><a href="/topics/agricultural-and-biological-sciences/macaca-mulatta" title="Learn more about Macaca mulatta from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca mulatta</a></span></span>) to investigate the effect of admixture on skeletal morphology. We focus on pelvic shape, which has potential fitness implications in hybrids, as mismatches between maternal pelvic and fetal cranial morphology are often fatal to mother and offspring. As the pelvis is also one of the skeletal regions that differs most between <span><span><a href="/topics/biochemistry-genetics-and-molecular-biology/human" title="Learn more about Homo sapiens from ScienceDirect's AI-generated Topic Pages" class="topic-link">Homo sapiens</a></span></span> and Neanderthals, investigating the pelvic consequences of interbreeding could be informative regarding the viability of their hybrids. We find that the effect of admixture in <em>M.&nbsp;mulatta</em><span> is small and proportional to the relatively small morphological difference between the parent taxa. <a href="/topics/biochemistry-genetics-and-molecular-biology/sex-difference" title="Learn more about Sexual dimorphism from ScienceDirect's AI-generated Topic Pages" class="topic-link">Sexual dimorphism</a> appears to be the main determinant of pelvic shape in </span><em>M.&nbsp;mulatta</em>. The lack of difference in pelvic shape between Chinese and Indian <em>M.&nbsp;mulatta</em> is in contrast to that between Neanderthals and <em>H.&nbsp;sapiens</em>, despite a similar split time (in generations) between the hybridizing pairs. Greater phenotypic divergence between hominins may relate to adaptations to disparate environments but may also highlight how the unique degree of cultural buffering in hominins allowed for greater neutral divergence. In contrast to some previous work identifying extreme morphologies in first- and second-generation hybrids, here the relationship between pelvic shape and admixture is linear. This linearity may be because most sampled animals have a multigenerational admixture history or because of relatively high constraints on the pelvis compared with other skeletal regions.</div></div></div></div><ul id="issue-navigation" class="issue-navigation u-margin-s-bottom u-bg-grey1"><li class="previous move-left u-padding-s-ver u-padding-s-left"><a class="button-alternative button-alternative-tertiary u-display-flex button-alternative-icon-left" href="/science/article/pii/S0047248421001068"><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-left"><path d="M1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text-container"><span class="button-alternative-text">Previous <span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></span></a></li><li class="next move-right u-padding-s-ver u-padding-s-right"><a class="button-alternative button-alternative-tertiary u-display-flex button-alternative-icon-right" href="/science/article/pii/S004724842100097X"><span class="button-alternative-text-container"><span class="button-alternative-text">Next <span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></span><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-right"><path d="M1 99l38-38L1 23l7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords u-font-serif"><div class="keywords-section" id="kwrds0010"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Keywords</h2><div class="keyword" id="kwrd0010"><span>Hybridization</span></div><div class="keyword" id="kwrd0015"><span>Human evolution</span></div><div class="keyword" id="kwrd0020"><span>Primate</span></div><div class="keyword" id="kwrd0025"><span>Skeletal morphology</span></div><div class="keyword" id="kwrd0030"><span>Geometric morphometric methods</span></div><div class="keyword" id="kwrd0035"><span><em>Macaca mulatta</em></span></div></div></div><div class="Body u-font-serif" id="body"><div><section id="sec1"><h2 id="sectitle0020" class="u-h4 u-margin-l-top u-margin-xs-bottom">1. Introduction</h2><div class="u-margin-s-bottom" id="p0010">Elucidating the effects of hybridization is critical for understanding the context of human evolution and the processes driving it (<a class="anchor anchor-primary" href="#bib4" name="bbib4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib4"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2016</span></span></a>). The past few years have seen great methodological advances in ancient DNA (aDNA) analyses, enabling the detection of interbreeding events and resulting in an increased appreciation of their frequency and importance (for reviews, refer to <a class="anchor anchor-primary" href="#bib35" name="bbib35" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib35"><span class="anchor-text-container"><span class="anchor-text">Gopalan et&nbsp;al., 2021</span></span></a>; <a class="anchor anchor-primary" href="#bib86" name="bbib86" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib86"><span class="anchor-text-container"><span class="anchor-text">Wolf and Akey, 2018</span></span></a><span><span>). Despite these advances, we are still far from understanding the importance of gene flow between <a href="/topics/biochemistry-genetics-and-molecular-biology/hominin" title="Learn more about hominin from ScienceDirect's AI-generated Topic Pages" class="topic-link">hominin</a> </span><a href="/topics/agricultural-and-biological-sciences/lineages" title="Learn more about lineages from ScienceDirect's AI-generated Topic Pages" class="topic-link">lineages</a> in determining morphology and from recognizing hybrids in the fossil record (</span><a class="anchor anchor-primary" href="#bib80" name="bbib80" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib80"><span class="anchor-text-container"><span class="anchor-text">Warren et&nbsp;al., 2018</span></span></a>). Claims of hybrid Neanderthal/<span><span><a href="/topics/biochemistry-genetics-and-molecular-biology/human" title="Learn more about Homo sapiens from ScienceDirect's AI-generated Topic Pages" class="topic-link">Homo sapiens</a></span></span><span> fossils were made for decades before supporting <a href="/topics/biochemistry-genetics-and-molecular-biology/genetics" title="Learn more about genetic from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic</a> evidence was available (</span><a class="anchor anchor-primary" href="#bib25" name="bbib25" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib25"><span class="anchor-text-container"><span class="anchor-text">Duarte et&nbsp;al., 1999</span></span></a>; <a class="anchor anchor-primary" href="#bib78" name="bbib78" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib78"><span class="anchor-text-container"><span class="anchor-text">Trinkaus et&nbsp;al., 2003a</span></span></a>, <a class="anchor anchor-primary" href="#bib79" name="bbib79" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib79"><span class="anchor-text-container"><span class="anchor-text">2003b</span></span></a>; <a class="anchor anchor-primary" href="#bib65" name="bbib65" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib65"><span class="anchor-text-container"><span class="anchor-text">Rougier et&nbsp;al., 2007</span></span></a><span>). When the data are limited to fossil material and its archaeological context, however, these assertions are almost inevitably subject to counterclaims, such as those asserting that intermediate morphology is within the boundaries of <a href="/topics/biochemistry-genetics-and-molecular-biology/intraspecific-variation" title="Learn more about intraspecific variation from ScienceDirect's AI-generated Topic Pages" class="topic-link">intraspecific variation</a> in </span><em>H.&nbsp;sapiens</em> (e.g., <a class="anchor anchor-primary" href="#bib74" name="bbib74" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib74"><span class="anchor-text-container"><span class="anchor-text">Tattersall and Schwartz, 1999</span></span></a>)<em>.</em></div><div class="u-margin-s-bottom" id="p0015">In many cases, even now, preservation issues mean there is limited potential for aDNA evidence to resolve the question of which putative (morphological) hybrids are actual (genetic) hybrids. There have nonetheless been some spectacular finds. One such was the aDNA analysis of the Oase 1 mandible from Romania, which showed that this <em>H.&nbsp;sapiens</em> individual did indeed have a recent Neanderthal ancestor (<a class="anchor anchor-primary" href="#bib33" name="bbib33" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib33"><span class="anchor-text-container"><span class="anchor-text">Fu et&nbsp;al., 2015</span></span></a>), as had been suggested from its morphology (<a class="anchor anchor-primary" href="#bib78" name="bbib78" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib78"><span class="anchor-text-container"><span class="anchor-text">Trinkaus et&nbsp;al., 2003a</span></span></a>, <a class="anchor anchor-primary" href="#bib79" name="bbib79" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib79"><span class="anchor-text-container"><span class="anchor-text">2003b</span></span></a>; <a class="anchor anchor-primary" href="#bib65" name="bbib65" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib65"><span class="anchor-text-container"><span class="anchor-text">Rougier et&nbsp;al., 2007</span></span></a>). Although the exact relationship between Oase 1's hybrid ancestry and suggested Neanderthal traits is not clear, the fact that it was identified as a hybrid based on both morphology and genetics supports a growing body of research, indicating that signatures of hybridization are detectable in skeletal morphology.</div><div class="u-margin-s-bottom" id="p0020">Here, we build on existing research using nonhuman primates to inform our understanding of hominin hybrids (e.g., <a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>; <a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Boel et&nbsp;al., 2019</span></span></a>; <a class="anchor anchor-primary" href="#bib54" name="bbib54" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib54"><span class="anchor-text-container"><span class="anchor-text">Martinez et&nbsp;al., 2019</span></span></a><span>). We use a large sample of captive full-bred and admixed Chinese and Indian <a href="/topics/biochemistry-genetics-and-molecular-biology/rhesus-monkey" title="Learn more about rhesus macaques from ScienceDirect's AI-generated Topic Pages" class="topic-link">rhesus macaques</a> (</span><span><em><a href="/topics/agricultural-and-biological-sciences/macaca-mulatta" title="Learn more about Macaca mulatta from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca mulatta</a></em></span>) to elucidate the expected morphological consequences of admixture between closely related primate species, including hominins. Of particular interest because Chinese and Indian <em>M.&nbsp;mulatta</em> have been mixing in this captive colony for decades, the study also examines how admixture effects on phenotype vary with ancestry contributions from the two subspecies lines.</div><section id="sec1.1"><h3 id="sectitle0025" class="u-h4 u-margin-m-top u-margin-xs-bottom">1.1. The effects of hybridization on the pelvis</h3><div class="u-margin-s-bottom" id="p0025">We focus on the effects of hybridization on pelvic morphology. In some primates, these effects may be particularly acute due to interactions between the functional and physiological constraints of locomotion and parturition (<a class="anchor anchor-primary" href="#bib76" name="bbib76" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib76"><span class="anchor-text-container"><span class="anchor-text">Trevathan, 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib49" name="bbib49" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib49"><span class="anchor-text-container"><span class="anchor-text">Kawada et&nbsp;al., 2020</span></span></a>). The pelvis is also a region of divergent shape in <em>H.&nbsp;sapiens</em> and <em>H</em>. <em>neanderthalensis</em> (<a class="anchor anchor-primary" href="#bib63" name="bbib63" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib63"><span class="anchor-text-container"><span class="anchor-text">Rak and Arensburg, 1987</span></span></a>; <a class="anchor anchor-primary" href="#bib83" name="bbib83" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib83"><span class="anchor-text-container"><span class="anchor-text">Weaver and Hublin, 2009</span></span></a>). Combined with neonatal crania that, although less distinct than the adult crania of these taxa, are also differently shaped (<a class="anchor anchor-primary" href="#bib61" name="bbib61" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib61"><span class="anchor-text-container"><span class="anchor-text">Ponce de León and Zollikofer, 2001</span></span></a>; <a class="anchor anchor-primary" href="#bib38" name="bbib38" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib38"><span class="anchor-text-container"><span class="anchor-text">Gunz et&nbsp;al., 2012</span></span></a>), this could have had implications for the viability of hybrid offspring. A more Neanderthal-like fetal cranium and a more <em>H.&nbsp;sapiens</em>-like birth canal could have led to increased likelihood of dystocia and the potential death of both mother and child. These severe consequences would have constituted a strong selective pressure likely to shape morphology, as has been suggested for the effect of caesarean sections on fetopelvic disproportion in recent humans (<a class="anchor anchor-primary" href="#bib57" name="bbib57" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib57"><span class="anchor-text-container"><span class="anchor-text">Mitteroecker et&nbsp;al., 2016</span></span></a>, <a class="anchor anchor-primary" href="#bib58" name="bbib58" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib58"><span class="anchor-text-container"><span class="anchor-text">2017</span></span></a>). To the best of our knowledge, the relationship between pelvic morphology and admixture in the fossil record has not yet been addressed, most likely due to the paucity of both suitable well-preserved fossils and postcranial material from extant known hybrids. Thus, it is a skeletal region from which we might learn a considerable amount, regarding the morphological impact of hybridization, by adopting an approach based on nonhuman primate analogs.</div></section><section id="sec1.2"><h3 id="sectitle0030" class="u-h4 u-margin-m-top u-margin-xs-bottom">1.2. Macaca mulatta taxonomy</h3><div class="u-margin-s-bottom" id="p0030">The genus <em>Macaca</em><span> is divided into four <a href="/topics/biochemistry-genetics-and-molecular-biology/species" title="Learn more about species groups from ScienceDirect's AI-generated Topic Pages" class="topic-link">species groups</a>. The </span><em>fascicularis</em> group, which contains <em>M.&nbsp;mulatta</em>, is the most widely distributed group (<a class="anchor anchor-primary" href="#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1976</span></span></a>, <a class="anchor anchor-primary" href="#bib32" name="bbib32" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib32"><span class="anchor-text-container"><span class="anchor-text">2006</span></span></a>). In 2000, Fooden reported that <em>M.&nbsp;mulatta</em> inhabited Southeast and South Asia with a geographic range of ∼15–36&nbsp;N° and ∼70–120 °E, although this range was historically wider and has likely been further reduced in the last two decades. Fossils show that <em>M.&nbsp;mulatta</em>, or a very similar precursor, was present in its current range by &gt;40 ka, notwithstanding the last glacial maximum, which likely caused local extinctions and subsequent postglacial in-migrations in some presently inhabited regions (<a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a>).</div><div class="u-margin-s-bottom" id="p0035">There is substantial disagreement over the systematics of <em>M.&nbsp;mulatta</em>, which appears to be polyphyletic: mitochondrial DNA (mtDNA) analyses cluster Chinese <em>M.&nbsp;mulatta</em> with <em>Macaca cyclopis</em> or <span><span><a href="/topics/agricultural-and-biological-sciences/macaca-fuscata" title="Learn more about Macaca fuscata from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca fuscata</a></span></span>, rather with than their supposed conspecifics, including Indian <em>M.&nbsp;mulatta</em> (<a class="anchor anchor-primary" href="#bib56" name="bbib56" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib56"><span class="anchor-text-container"><span class="anchor-text">Melnick et&nbsp;al., 1993</span></span></a>; <a class="anchor anchor-primary" href="#bib42" name="bbib42" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib42"><span class="anchor-text-container"><span class="anchor-text">Hayasaka et&nbsp;al., 1996</span></span></a>). mtDNA also suggests deep divergences between geographic groups of <em>M.&nbsp;mulatta</em> to at least a subspecific level, although there remains disagreement as to how many subspecies should be recognized (<a class="anchor anchor-primary" href="#bib88" name="bbib88" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib88"><span class="anchor-text-container"><span class="anchor-text">Zhang and Shi, 1993</span></span></a>; <a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>). Indeed, the divergence between Indian and Chinese <em>M.&nbsp;mulatta</em> mtDNA is deeper than between some well-accepted species pairs of macaques such as <em>Macaca assamensis</em> and <em>Macaca thibetana</em> (<a class="anchor anchor-primary" href="#bib42" name="bbib42" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib42"><span class="anchor-text-container"><span class="anchor-text">Hayasaka et&nbsp;al., 1996</span></span></a>; <a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>). Similar divisions between Chinese and Indian <em>M.&nbsp;mulatta</em><span> have also been found in studies of <a href="/topics/biochemistry-genetics-and-molecular-biology/protein-polymorphism" title="Learn more about protein polymorphisms from ScienceDirect's AI-generated Topic Pages" class="topic-link">protein polymorphisms</a><span>, <a href="/topics/agricultural-and-biological-sciences/microsatellite" title="Learn more about microsatellite from ScienceDirect's AI-generated Topic Pages" class="topic-link">microsatellite</a><span> <a href="/topics/agricultural-and-biological-sciences/single-nucleotide-polymorphism" title="Learn more about single nucleotide polymorphisms from ScienceDirect's AI-generated Topic Pages" class="topic-link">single nucleotide polymorphisms</a> (SNPs), and major histocompatibility-complex (MHC) alleles (</span></span></span><a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text">Ferguson et&nbsp;al., 2007</span></span></a>).</div><div class="u-margin-s-bottom" id="p0040">Here, we refer to Chinese and Indian <em>M.&nbsp;mulatta</em> as different subspecies solely to avoid more confusing terms, such as ‘group’ or ‘population’. Our analyses are not intended to address the validity or otherwise of that taxonomic designation. The differences between extant Chinese and Indian <em>M.&nbsp;mulatta</em> subspecies, particularly the molecular and physiological differences, have been studied extensively due to the species' role as a biomedical model. Chinese-derived <em>M.&nbsp;mulatta</em><span> has greater genetic <a href="/topics/agricultural-and-biological-sciences/heterozygosity" title="Learn more about heterozygosity from ScienceDirect's AI-generated Topic Pages" class="topic-link">heterozygosity</a> (</span><a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text">Ferguson et&nbsp;al., 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib48" name="bbib48" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib48"><span class="anchor-text-container"><span class="anchor-text">Kanthaswamy et&nbsp;al., 2009</span></span></a>) and morphological diversity (<a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a>) than Indian-derived animals. Indeed, <a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough (2005)</span></span></a> found pairwise differences between Indian <em>M.&nbsp;mulatta</em><span> populations to be half as great as between comparable pairs of human populations, despite the extremely low levels of <a href="/topics/agricultural-and-biological-sciences/genetic-divergence" title="Learn more about genetic variation from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic variation</a> in </span><em>H.&nbsp;sapiens</em> (<a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>). These low levels of variation may be due to a bottleneck experienced by Indian populations in their recent evolutionary past (<a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib48" name="bbib48" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib48"><span class="anchor-text-container"><span class="anchor-text">Kanthaswamy et&nbsp;al., 2009</span></span></a><span><span>). Because Neanderthals also appear to have had small <a href="/topics/biochemistry-genetics-and-molecular-biology/population-size" title="Learn more about population sizes from ScienceDirect's AI-generated Topic Pages" class="topic-link">population sizes</a> and low </span><a href="/topics/biochemistry-genetics-and-molecular-biology/genetic-divergence" title="Learn more about genetic diversity from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic diversity</a> (</span><a class="anchor anchor-primary" href="#bib37" name="bbib37" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib37"><span class="anchor-text-container"><span class="anchor-text">Green et&nbsp;al., 2010</span></span></a>; <a class="anchor anchor-primary" href="#bib12" name="bbib12" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib12"><span class="anchor-text-container"><span class="anchor-text">Bocquet-Appel and Degioanni, 2013</span></span></a>), this is another way in which this <em>M.&nbsp;mulatta</em> cross-sample is an interesting analog for hybridization during human evolution.</div><div class="u-margin-s-bottom" id="p0045">Relative to the known genetic differences, the extent of morphological variation within <em>M.&nbsp;mulatta</em> and between Chinese and Indian groups is less clear. Fooden (e.g., <a class="anchor anchor-primary" href="#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text">1976</span></span></a>, <a class="anchor anchor-primary" href="#bib29" name="bbib29" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib29"><span class="anchor-text-container"><span class="anchor-text">1982</span></span></a>, <a class="anchor anchor-primary" href="#bib30" name="bbib30" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib30"><span class="anchor-text-container"><span class="anchor-text">1995</span></span></a>, <a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">2000</span></span></a>, <a class="anchor anchor-primary" href="#bib32" name="bbib32" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib32"><span class="anchor-text-container"><span class="anchor-text">2006</span></span></a>) argues that although there is substantial geographic variation in the phenotype, the pattern of variation within <em>M.&nbsp;mulatta</em> is insufficiently clear to warrant subspecific divisions (<a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a>). <a class="anchor anchor-primary" href="#bib30" name="bbib30" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib30"><span class="anchor-text-container"><span class="anchor-text">Fooden's (1995</span></span></a>, <a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">2000)</span></span></a> taxonomy is based on a conservative designation requiring at least one trait with discontinuous variation between subspecies. Many other researchers, however, have noted substantial morphological differences between Indian- and Chinese-derived animals in particular (see discussion in <a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a>). For example, Chinese males are larger than Indian males, which may reflect adaptation to a colder climate, following Bergmann's rule (<a class="anchor anchor-primary" href="#bib9" name="bbib9" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib9"><span class="anchor-text-container"><span class="anchor-text">Bergmann, 1847</span></span></a>; <a class="anchor anchor-primary" href="#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text">Clarke and O'Neil, 1999</span></span></a>). Indeed, across the species' natural range, stature and mass tend to increase gradually with latitude, although there are regional exceptions to this trend (<a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a><span>). <a href="/topics/agricultural-and-biological-sciences/tail" title="Learn more about Tail from ScienceDirect's AI-generated Topic Pages" class="topic-link">Tail</a> length decreases with longitude, particularly east of 95 °E, and is, therefore, longer in Indian than in Chinese </span><em>M.&nbsp;mulatta</em> (<a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a>; <a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2005</span></span></a>). In captive-bred Chinese and Indian <em>M.&nbsp;mulatta</em><span> housed in Louisiana, USA (Tulane National Primate Research Center), there are differences in patterns of <a href="/topics/biochemistry-genetics-and-molecular-biology/sex-difference" title="Learn more about sexual dimorphism from ScienceDirect's AI-generated Topic Pages" class="topic-link">sexual dimorphism</a>, with greater sexual dimorphism in Chinese than in Indian monkeys (</span><a class="anchor anchor-primary" href="#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text">Clarke and O'Neil, 1999</span></span></a>). These studies suggest that size, allometric shape, and patterns of sexual dimorphism may be variables that differ between the two <em>M.&nbsp;mulatta</em> taxa and which therefore may vary in admixed individuals.</div><div class="u-margin-s-bottom" id="p0050">Chinese and Indian <em>M.&nbsp;mulatta</em> have been isolated from one another since at least the Late Pleistocene (<a class="anchor anchor-primary" href="#bib88" name="bbib88" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib88"><span class="anchor-text-container"><span class="anchor-text">Zhang and Shi, 1993</span></span></a>; <a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>), likely since ∼162 ka, based on genetic analyses of ∼1,500 SNPs (<a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a>). Given a generation time of approximately 7.5 years (<a class="anchor anchor-primary" href="#bib87" name="bbib87" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib87"><span class="anchor-text-container"><span class="anchor-text">Wolfe, 1986</span></span></a>) and the split time of ∼162 ka (<a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a>), about 21,600 generations passed between the divergence of Indian and Chinese <em>M.&nbsp;mulatta</em> and their interbreeding in the California National Primate Research Center (CNPRC) colony in the mid-1980s (see section <a class="anchor anchor-primary" href="#sec2.1" name="bsec2.1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec2.1"><span class="anchor-text-container"><span class="anchor-text">2.1</span></span></a>). The same calculation for <em>H.&nbsp;sapiens</em> and Neanderthals depends upon more assumptions and disputed dates, but an approximate estimation of the same variables is possible. If we use a relatively conservative split date of 550–765 ka (<a class="anchor anchor-primary" href="#bib90" name="bbib90" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib90"><span class="anchor-text-container"><span class="anchor-text">Meyer et&nbsp;al., 2016</span></span></a>), a generation length of 26–30 years (<a class="anchor anchor-primary" href="#bib92" name="bbib92" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib92"><span class="anchor-text-container"><span class="anchor-text">Moorjani et&nbsp;al., 2016</span></span></a>), and a date for interbreeding of 47–65 ka (<a class="anchor anchor-primary" href="#bib91" name="bbib91" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib91"><span class="anchor-text-container"><span class="anchor-text">Sankararaman et&nbsp;al., 2012</span></span></a><span>), the middle point of each range results in a rough estimate of 21,482 generations between divergence and interbreeding. The macaque and hominin taxa can thus be argued to have had similar degrees of divergence between taxon pairs at the time of <a href="/topics/agricultural-and-biological-sciences/introgression" title="Learn more about introgression from ScienceDirect's AI-generated Topic Pages" class="topic-link">introgression</a>, making Chinese and Indian </span><em>M.&nbsp;mulatta</em> an interesting model for examining Neanderthal/<em>H.&nbsp;sapiens</em> hybridization.</div></section><section id="sec1.3"><h3 id="sectitle0035" class="u-h4 u-margin-m-top u-margin-xs-bottom">1.3. Hybridization in nonhuman primates</h3><div class="u-margin-s-bottom" id="p0055">Hybridization appears to be commonplace in the primate order, occurring in all major lineages (<a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Cortés-Ortiz et&nbsp;al., 2019</span></span></a>). Some well-established macaque species, such as <span><em><a href="/topics/agricultural-and-biological-sciences/macaca-arctoides" title="Learn more about Macaca arctoides from ScienceDirect's AI-generated Topic Pages" class="topic-link">Macaca arctoides</a></em></span>, are even thought to have hybrid origins (<a class="anchor anchor-primary" href="#bib75" name="bbib75" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib75"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2000</span></span></a>). Ackermann and colleagues (e.g., <a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>, <a class="anchor anchor-primary" href="#bib1" name="bbib1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib1"><span class="anchor-text-container"><span class="anchor-text">2019</span></span></a>; <a class="anchor anchor-primary" href="#bib2" name="bbib2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib2"><span class="anchor-text-container"><span class="anchor-text">Ackermann and Bishop, 2010</span></span></a>) have used nonhuman primates as hominin analogs to study the potential effects of hybridization during human evolution. The majority of this work has been carried out on baboons, which are well studied, provide a good ecological analog for hominins (<a class="anchor anchor-primary" href="#bib47" name="bbib47" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib47"><span class="anchor-text-container"><span class="anchor-text">Jolly, 2001</span></span></a>; for review, see <a class="anchor anchor-primary" href="#bib27" name="bbib27" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib27"><span class="anchor-text-container"><span class="anchor-text">Fischer et&nbsp;al., 2019</span></span></a>), and are housed in several major captive colonies with known pedigree information. <a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al. (2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014)</span></span></a> showed that first- and second-generation olive-yellow baboon hybrids exhibit extreme size (heterosis) and size variation outside the parental ranges. They also display high frequencies of unusual nonmetric cranial traits and dental and sutural anomalies. This body of work, supported by other research on nonhuman primate hybridization (for reviews, see <a class="anchor anchor-primary" href="#bib1" name="bbib1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib1"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2019</span></span></a>; <a class="anchor anchor-primary" href="#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text">Cortés-Ortiz et&nbsp;al., 2019</span></span></a><span><span>), provides some important insights to underpin the present study. Notably, there is a skeletal signal of hybridization, and the effects of hybridization appear to be similar across primate taxa, albeit with an effect of <a href="/topics/biochemistry-genetics-and-molecular-biology/phylogeny" title="Learn more about phylogenetic from ScienceDirect's AI-generated Topic Pages" class="topic-link">phylogenetic</a> proximity, such that more extreme hybrids are more likely to result from interbreeding between more divergent taxa. It also provides some expectations to test with our macaque data: hybrid primates generally show increased variation and </span><a href="/topics/agricultural-and-biological-sciences/heterosis" title="Learn more about heterosis from ScienceDirect's AI-generated Topic Pages" class="topic-link">heterosis</a> compared with parent taxa.</span></div></section></section><section id="sec2"><h2 id="sectitle0040" class="u-h4 u-margin-l-top u-margin-xs-bottom">2. Materials and methods</h2><section id="sec2.1"><h3 id="sectitle0045" class="u-h4 u-margin-m-top u-margin-xs-bottom">2.1. Sample</h3><div class="u-margin-s-bottom" id="p0060">Our sample is composed of <em>M.&nbsp;mulatta</em> housed at the CNPRC, University of California, Davis. The CNPRC colony was initially composed entirely of Indian <em>M.&nbsp;mulatta</em><span>, but after a ban on exporting primates from India in 1978, the colony's managers introduced a small number of Chinese animals to maintain <a href="/topics/biochemistry-genetics-and-molecular-biology/genetics" title="Learn more about genetic from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic</a> variation within the population. The first interbreeding was recorded in 1983 (</span><a class="anchor anchor-primary" href="#bib48" name="bbib48" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib48"><span class="anchor-text-container"><span class="anchor-text">Kanthaswamy et&nbsp;al., 2009</span></span></a>). Unlike many nonhuman hybrid studies, which focus on the first few hybrid generations (e.g., <a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>; <a class="anchor anchor-primary" href="#bib67" name="bbib67" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib67"><span class="anchor-text-container"><span class="anchor-text">Savriama et&nbsp;al., 2018</span></span></a>; <a class="anchor anchor-primary" href="#bib80" name="bbib80" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib80"><span class="anchor-text-container"><span class="anchor-text">Warren et&nbsp;al., 2018</span></span></a>), interbreeding between Indian and Chinese <em>M.&nbsp;mulatta</em> has been occurring in this colony for at least four to five generations (based on a generation time of ∼7.5 years; <a class="anchor anchor-primary" href="#bib87" name="bbib87" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib87"><span class="anchor-text-container"><span class="anchor-text">Wolfe, 1986</span></span></a><span>). The colony now provides a sample with a distribution from full-bred Indian to full-bred Chinese animals, including many individuals with low percentages of Chinese ancestry. This range of admixture is a better representation of what we might expect in natural <a href="/topics/biochemistry-genetics-and-molecular-biology/hybrid-zone" title="Learn more about hybrid zones from ScienceDirect's AI-generated Topic Pages" class="topic-link">hybrid zones</a><span> and in the <a href="/topics/biochemistry-genetics-and-molecular-biology/fossil-hominin" title="Learn more about hominin fossil from ScienceDirect's AI-generated Topic Pages" class="topic-link">hominin fossil</a> record if hybridization leads to viable offspring (</span></span><a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a>). In effect, the Indian <em>M.&nbsp;mulatta</em> housed at the CNPRC are a population experiencing a relatively small amount of genetic input from another taxon (Chinese <em>M.&nbsp;mulatta</em>); this can be seen as analogous to a Pleistocene <em>H.&nbsp;sapiens</em> population undergoing a small number of time-limited interbreeding events with Neanderthals.</div><div class="u-margin-s-bottom"><div id="p0065"><span>Our sample data consisted of 138 full-body medical <a href="/topics/agricultural-and-biological-sciences/computed-tomography" title="Learn more about computed tomography from ScienceDirect's AI-generated Topic Pages" class="topic-link">computed tomography</a> (CT) scans from adult (&gt;6 years) </span><em>M.&nbsp;mulatta</em> housed at the CNPRC: 94 females and 44 males. Sex ratios in the CNPRC colony are deliberately kept unbalanced to reduce intramale fighting, resulting in unequal numbers of available males and females. To optimize the spread of admixture proportions (Chinese ancestry) in our sample, we included more females than males; however, we sought to maintain the same degree of female bias in both Indian and Chinese <em>M.&nbsp;mulatta</em> samples to the extent possible. We included 19 full-bred Indian <em>M.&nbsp;mulatta</em>, 17 full-bred Chinese <em>M.&nbsp;mulatta</em>, and hybrid animals with a range of admixture proportions (<a class="anchor anchor-primary" href="#tbl1" name="btbl1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a>). Estimates of Chinese ancestry were obtained from the CNPRC and are constructed from pedigree records. In our aim to obtain the best range of admixture in our sample, we included individuals with some arthritis if the original morphology was sufficiently unobscured as to allow confident landmark placement (see following paragraphs); however, we removed six animals with pathological pelves from our initial sample after examination of their CT scans, leaving the final sample of 138 (<a class="anchor anchor-primary" href="#tbl1" name="btbl1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a>).</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl1"><span class="captions text-s"><span id="cap0045"><p id="tspara0010"><span class="label">Table&nbsp;1</span>. Breakdown of sample (<em>n</em>&nbsp;=&nbsp;138) by Chinese ancestry and sex.</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Ancestry category</th><th scope="col">0%</th><th scope="col">12.5%</th><th scope="col">25%</th><th scope="col">37.5%</th><th scope="col">50%</th><th scope="col">62.5%</th><th scope="col">75%</th><th scope="col">87.5%</th><th scope="col">100%</th></tr></thead><tbody><tr class="valign-top"><td>Female</td><td class="align-char">11</td><td class="align-char">29</td><td class="align-char">15</td><td class="align-char">12</td><td class="align-char">5</td><td class="align-char">3</td><td class="align-char">7</td><td class="align-char">1</td><td class="align-char">11</td></tr><tr class="valign-top"><td>Male</td><td class="align-char">8</td><td class="align-char">15</td><td class="align-char">8</td><td class="align-char">3</td><td class="align-char">2</td><td class="align-char">2</td><td class="align-char">0</td><td class="align-char">0</td><td class="align-char">6</td></tr><tr class="valign-top"><td>Total</td><td class="align-char">19</td><td class="align-char">44</td><td class="align-char">23</td><td class="align-char">15</td><td class="align-char">7</td><td class="align-char">5</td><td class="align-char">7</td><td class="align-char">1</td><td class="align-char">17</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0015">Ancestry category reflects the percentage of Chinese ancestry: 0% Chinese ancestry&nbsp;=&nbsp;full-bred Indian, 100% Chinese ancestry&nbsp;=&nbsp;full-bred Chinese. Full-bred sample <em>n</em>&nbsp;=&nbsp;36.</div></div></div></div><div class="u-margin-s-bottom" id="p0070">All animals were CT scanned at the CNPRC by appropriately trained staff. CT scanning and all other experimental procedures for this study were approved by the <a href="/topics/agricultural-and-biological-sciences/institutional-animal-care-and-use-committee" title="Learn more about Institutional Animal Care and Use Committee from ScienceDirect's AI-generated Topic Pages" class="topic-link">Institutional Animal Care and Use Committee</a><span> at the University of California, Davis (protocol #19057). In most cases, monkeys were sedated and scanned in&nbsp;vivo. Although in a few cases monkeys were scanned as cadavers, no monkeys were sacrificed for this study. As they become available, macerated skeletal remains from cadavers of animals involved in this study are being curated in the Department of Anthropology, University of California, Davis. CT scans from the sample are also freely available on request via the MorphoSource digital archive (Morphosource.org; project name: The <a href="/topics/biochemistry-genetics-and-molecular-biology/rhesus-monkey" title="Learn more about rhesus macaque from ScienceDirect's AI-generated Topic Pages" class="topic-link">rhesus macaque</a> admixture project).</span></div></section><section id="sec2.2"><h3 id="sectitle0050" class="u-h4 u-margin-m-top u-margin-xs-bottom">2.2. Data collection</h3><div class="u-margin-s-bottom"><div id="p0075"><span>Using the CT scans, in Avizo v. 9.1 Lite (Thermo Fisher Scientific, Waltham), we virtually segmented the <a href="/topics/agricultural-and-biological-sciences/skeleton" title="Learn more about skeleton from ScienceDirect's AI-generated Topic Pages" class="topic-link">skeleton</a> from the other tissue types and the pelvis from the rest of the skeleton using a mixture of automatic and manual thresholding and segmentation tools. To collect landmark coordinates for analyses using geometric morphometric methods (GMM), we digitized landmarks and semilandmarks on virtual reconstructions (shaded isosurfaces and transparent volume renderings) using Avizo and exported the coordinates for analysis in R v. 3.5.2–3.6.3 (</span><a class="anchor anchor-primary" href="#bib21" name="bbib21" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib21"><span class="anchor-text-container"><span class="anchor-text">R Core Team, 2020</span></span></a>). We collected 48 true landmarks plus semilandmarks describing seven curves by digitizing many points along each curve (<a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a>; <a class="anchor anchor-primary" href="#tbl2" name="btbl2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl2"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;2</span></span></a>). These curve semilandmarks then were resampled in R using a custom script (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">Supplementary Online Material [SOM]</span></span></a>) so that each curve contained the same number of semilandmarks for each individual as follows: curve 1 (40 semilandmarks), curves 2 and 3 (18 semilandmarks each), curves 4 and 5 (30 landmarks each), and curves 6 and 7 (10 landmarks each; <a class="anchor anchor-primary" href="#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;2</span></span></a>). After processing, our data set consisted of 204 total landmarks: 48 true landmarks and 156 semilandmarks. We selected landmarks and semilandmarks with the intent to optimally characterize the size and shape of the pelvis. We examined intraobserver error by using ten repetitions each of two individuals digitized repeatedly, with more than a day elapsing between each digitizing session. We processed each landmark configuration as described in the following paragraphs and compared the Procrustes distances between iterations of the same individual versus those between different individuals (see <a class="anchor anchor-primary" href="#sec3" name="bsec3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec3"><span class="anchor-text-container"><span class="anchor-text">Results</span></span></a>).</div><figure class="figure text-xs" id="fig1"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr1.jpg" height="572" alt="Figure&nbsp;1" aria-describedby="cap0010"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr1_lrg.jpg" target="_blank" download="" title="Download high-res image (606KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (606KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr1.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0010"><p id="fspara0010"><span class="label">Figure&nbsp;1</span>. Landmarks and semilandmarks used in the analyses presented here viewed from (clockwise from top left) ventral, cranial, caudal, dorsal, and right lateral views. Black spheres: true landmarks, gray spheres: semilandmarks, lines: wireframe connecting landmarks and semilandmarks. For landmarks that correspond to numbers, refer to <a class="anchor anchor-primary" href="#tbl2" name="btbl2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl2"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;2</span></span></a>. For ease of visualization, only true landmarks are numbered, and they are numbered only in the view where they are most easily seen. In the visualization of results throughout the remaining article, only the wireframes are shown to better visualize differences between shapes.</p></span></span></figure><figure class="figure text-xs" id="fig2"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr2.jpg" height="243" alt="Figure&nbsp;2" aria-describedby="cap0015"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr2_lrg.jpg" target="_blank" download="" title="Download high-res image (183KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (183KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr2.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0015"><p id="fspara0015"><span class="label">Figure&nbsp;2</span>. Semilandmark curves and wireframe used in the analyses presented here, viewed from (left to right) anterior and posterior. 1: pelvic inlet (closed) curve, 2: right dorsal ischial curve, 3: left dorsal ischial curve, 4: right iliac blade curve, 5: left iliac blade curve, 6: right pubic symphysis curve, 7: left pubic symphysis curve.</p></span></span></figure><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl2"><span class="captions text-s"><span id="cap0050"><p id="tspara0020"><span class="label">Table&nbsp;2</span>. Numbers and definitions of all landmarks (1–48) and semilandmarks (49–204).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-center"><strong>Landmark number</strong></th><th scope="col" class="align-center"><strong>Definition</strong></th></tr></thead><tbody><tr class="valign-top"><td>1</td><td>Most cranial point on the sacral wing—right</td></tr><tr class="valign-top"><td>2</td><td>Most caudoventral point on the sacral wing, on linea terminalis—right</td></tr><tr class="valign-top"><td>3</td><td>Most lateral point on the cranial articular surface of S1—right</td></tr><tr class="valign-top"><td>4</td><td>Most ventral point on the cranial articular surface of S1 body—midline</td></tr><tr class="valign-top"><td>5</td><td>Most ventral point on the dorsal margin of articular cranial surface of S1 body—midline</td></tr><tr class="valign-top"><td>6</td><td>Most ventral point on the cranial articular surface of S2 body—midline</td></tr><tr class="valign-top"><td>7</td><td>Most caudal point on the sacrum—midline</td></tr><tr class="valign-top"><td>8</td><td>Dorsal caudal iliac spine—right</td></tr><tr class="valign-top"><td>9</td><td>Cranialmost point on the iliac spine—right</td></tr><tr class="valign-top"><td>10</td><td>Cranialmost point on the acetabular rim—right</td></tr><tr class="valign-top"><td>11</td><td>Medialmost point on cranial portion of the lunate—right</td></tr><tr class="valign-top"><td>12</td><td>Medialmost point on caudal portion of the lunate—right</td></tr><tr class="valign-top"><td>13</td><td>Center of the acetabulum—right</td></tr><tr class="valign-top"><td>14</td><td>Caudalmost point on the acetabular rim—right</td></tr><tr class="valign-top"><td>15</td><td>Dorsalmost point on the acetabular rim—right</td></tr><tr class="valign-top"><td>16</td><td>Ventrolateral point on the lateral ischial crest at min. breadth—right</td></tr><tr class="valign-top"><td>17</td><td>Iliac blade medial (most medial intersection of the iliac blade and sacrum)—right</td></tr><tr class="valign-top"><td>18</td><td>Dorsalmost point on the ischial spine—right</td></tr><tr class="valign-top"><td>19</td><td>Intersection of the dorsal ischial crest and ischial tuberosity—right</td></tr><tr class="valign-top"><td>20</td><td>Dorsalmost point on the ischial tuberosity—right</td></tr><tr class="valign-top"><td>21</td><td>Most ventrolateral point on the ischial tuberosity—right</td></tr><tr class="valign-top"><td>22</td><td>Most ventromedial point on the ischial tuberosity—right</td></tr><tr class="valign-top"><td>23</td><td>Cranialmost point on the obturator foramen—right</td></tr><tr class="valign-top"><td>24</td><td>Ventralmost point on the obturator foramen—right</td></tr><tr class="valign-top"><td>25</td><td>Caudalmost point on the obturator foramen—right</td></tr><tr class="valign-top"><td>26</td><td>Pubic symphysis cranial—right</td></tr><tr class="valign-top"><td>27</td><td>Most cranial point on the sacral wing—left</td></tr><tr class="valign-top"><td>28</td><td>Most caudoventral point on the sacral wing, on linea terminalis—left</td></tr><tr class="valign-top"><td>29</td><td>Most lateral point on the cranial articular surface of S1—left</td></tr><tr class="valign-top"><td>30</td><td>Dorsal caudal iliac spine—left</td></tr><tr class="valign-top"><td>31</td><td>Cranialmost point on the iliac spine—left</td></tr><tr class="valign-top"><td>32</td><td>Cranialmost point on the acetabular rim—left</td></tr><tr class="valign-top"><td>33</td><td>Medialmost point on cranial portion of the lunate—left</td></tr><tr class="valign-top"><td>34</td><td>Medialmost point on caudal portion of the lunate—left</td></tr><tr class="valign-top"><td>35</td><td>Center of the acetabulum—left</td></tr><tr class="valign-top"><td>36</td><td>Caudalmost point on the acetabular rim—left</td></tr><tr class="valign-top"><td>37</td><td>Dorsalmost point on the acetabular rim—left</td></tr><tr class="valign-top"><td>38</td><td>Ventrolateral point on the lateral ischial crest at min. breadth—left</td></tr><tr class="valign-top"><td>39</td><td>Iliac crest medial (most medial intersection of the iliac crest and sacrum)—left</td></tr><tr class="valign-top"><td>40</td><td>Dorsalmost point on the ischial spine—left</td></tr><tr class="valign-top"><td>41</td><td>Intersection of the dorsal ischial crest and ischial tuberosity—left</td></tr><tr class="valign-top"><td>42</td><td>Dorsalmost point on the ischial tuberosity—left</td></tr><tr class="valign-top"><td>43</td><td>Most ventrolateral point on the ischial tuberosity—left</td></tr><tr class="valign-top"><td>44</td><td>Most ventromedial point on the ischial tuberosity—left</td></tr><tr class="valign-top"><td>45</td><td>Cranialmost point on the obturator foramen—left</td></tr><tr class="valign-top"><td>46</td><td>Ventralmost point on the obturator foramen—left</td></tr><tr class="valign-top"><td>47</td><td>Most caudodorsal point on the obturator foramen—left</td></tr><tr class="valign-top"><td class="rowsep-1">48</td><td class="rowsep-1">Pubic symphysis cranial—left</td></tr></tbody></table><table><thead><tr class="valign-top"><th scope="col" class="rowsep-1 align-center"><strong>Semilandmark number</strong></th><th scope="col" class="rowsep-1 align-center"><strong>Definition</strong></th></tr></thead><tbody><tr class="valign-top"><td>49 to 88</td><td>Pelvic inlet curve</td></tr><tr class="valign-top"><td>89 to 106</td><td>Dorsal ischial curve—right</td></tr><tr class="valign-top"><td>107 to 124</td><td>Dorsal ischial curve—left</td></tr><tr class="valign-top"><td>125 to 154</td><td>Iliac blade curve—right</td></tr><tr class="valign-top"><td>155 to 184</td><td>Iliac blade curve—left</td></tr><tr class="valign-top"><td>185 to 194</td><td>Pubic symphysis curve—right</td></tr><tr class="valign-top"><td>195 to 204</td><td>Pubic symphysis curve—left</td></tr></tbody></table></div></div></div></section><section id="sec2.3"><h3 id="sectitle0055" class="u-h4 u-margin-m-top u-margin-xs-bottom">2.3. Methods of analysis</h3><div class="u-margin-s-bottom" id="p0080">All analyses were carried out in R v. 3.5.2–3.6.3 (<a class="anchor anchor-primary" href="#bib21" name="bbib21" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib21"><span class="anchor-text-container"><span class="anchor-text">R Core Team, 2020</span></span></a>) using the geomorph (<a class="anchor anchor-primary" href="#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text">Collyer and Adams, 2018</span></span></a>, <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">2020</span></span></a>; <a class="anchor anchor-primary" href="#bib7" name="bbib7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib7"><span class="anchor-text-container"><span class="anchor-text">Adams et&nbsp;al., 2020</span></span></a>) and Morpho (<a class="anchor anchor-primary" href="#bib68" name="bbib68" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib68"><span class="anchor-text-container"><span class="anchor-text">Schlager, 2017</span></span></a>) packages and base R functions (<a class="anchor anchor-primary" href="#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">SOM</span></span></a>). To register the shape data, we used the procSym function in Morpho (<a class="anchor anchor-primary" href="#bib68" name="bbib68" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib68"><span class="anchor-text-container"><span class="anchor-text">Schlager, 2017</span></span></a><span><span><span>) to perform a generalized Procrustes superimposition of the landmarks, to slide the semilandmarks along their curves using a criterion of minimizing bending energy, and to project the superimposed configurations into a tangent space. We performed <a href="/topics/biochemistry-genetics-and-molecular-biology/principal-component-analysis" title="Learn more about principal component analysis from ScienceDirect's AI-generated Topic Pages" class="topic-link">principal component analysis</a> (PCA) in geomorph (using the plotTangentSpace function) to summarize the high-dimensional data into axes of greatest explanatory power and to visualize variation within and between groups of interest on these axes. We then used geomorph's procD.lm function to perform Procrustes analysis of variance (ANOVA) to test for relationships between predictor and response variables of interest and to compare model fits. To test for the presence of different patterns of </span><a href="/topics/biochemistry-genetics-and-molecular-biology/sex-difference" title="Learn more about sexual dimorphism from ScienceDirect's AI-generated Topic Pages" class="topic-link">sexual dimorphism</a> in different admixture groups, we compared the fit of additive (shape&nbsp;∼&nbsp;admixture&nbsp;+&nbsp;sex) and interactive (shape&nbsp;∼&nbsp;admixture ∗ sex) models of shape, sex, and admixture. We also included a three-knot B spline model of admixture and sex-adjusted shape in the comparison to investigate whether departures from a linear relationship between sex and ancestry could result from </span><a href="/topics/biochemistry-genetics-and-molecular-biology/genetic-determinism" title="Learn more about genetic factors from ScienceDirect's AI-generated Topic Pages" class="topic-link">genetic factors</a> such as dominance or epistasis. The B-spline model allows for the relationship between the response and predictor variables to be described by a smooth curve.</span></div><div class="u-margin-s-bottom" id="p0085">Owing to sexual dimorphism in our sample (described in more detail in the following paragraphs), we used <a href="/topics/agricultural-and-biological-sciences/least-square" title="Learn more about least squares from ScienceDirect's AI-generated Topic Pages" class="topic-link">least squares</a> linear regression to account for the effects of sex on admixture and shape. Because sex ratios varied with Chinese ancestry percentages, it was necessary to adjust both variables to control for the effect of sex entirely. The sex-adjusted residuals were used in all subsequent analyses of the relationship between shape and admixture.</div><div class="u-margin-s-bottom" id="p0090">To quantify and visualize the relationship between admixture and pelvic shape, we regressed pelvic shape on admixture using least squares linear regression and then projected the sex-adjusted shape observations onto the resulting regression vector to compute admixture scores (<a class="anchor anchor-primary" href="#bib24" name="bbib24" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib24"><span class="anchor-text-container"><span class="anchor-text">Drake and Klingenberg, 2008</span></span></a>; <a class="anchor anchor-primary" href="#bib51" name="bbib51" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib51"><span class="anchor-text-container"><span class="anchor-text">Ledevin and Koyabu, 2019</span></span></a>; <a class="anchor anchor-primary" href="#bib70" name="bbib70" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib70"><span class="anchor-text-container"><span class="anchor-text">Sheratt et&nbsp;al., 2019</span></span></a>; <a class="anchor anchor-primary" href="#bib59" name="bbib59" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib59"><span class="anchor-text-container"><span class="anchor-text">Moore, 2021</span></span></a>). Displayed in a bivariate plot, the results provide a straightforward way to compare the magnitude of shape differences attributable to admixture (the regression trend) to the magnitude of shape differences not explained by the model (residuals about the trend). To visualize the relative contributions of sex and admixture variables to overall shape variation, we display distributions of pairwise Procrustes distances between specimens using different filters of group and sex membership. To test for differences in variance, we used the function morphol.disparity in geomorph (<a class="anchor anchor-primary" href="#bib7" name="bbib7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib7"><span class="anchor-text-container"><span class="anchor-text">Adams et&nbsp;al., 2020</span></span></a>; <a class="anchor anchor-primary" href="#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text">Collyer and Adams, 2020</span></span></a>), which calculates the Procrustes variance for each group, the pairwise distances between group variances, and the <em>p</em>-values of tests of significant difference between pairs (<em>p</em>&nbsp;&lt;&nbsp;0.05 throughout).</div></section></section><section id="sec3"><h2 id="sectitle0060a" class="u-h4 u-margin-l-top u-margin-xs-bottom">3. Results</h2><div class="u-margin-s-bottom" id="p0095">In our intraobserver error analysis, the mean Procrustes distances between iterations of the same individual were much smaller than those between different individuals, and the 95% confidence intervals (CIs) of their means did not overlap. The mean Procrustes distance between iterations of the same individual was 0.026 (95% CI: 0.023–0.029) and 0.036 (95% CI: 0.024–0.048); between different individuals, the mean Procrustes distance was 0.105 (95% CI: 0.101–0.109). We judged this to mean that levels of intraobserver error in our data were within acceptable limits.</div><section id="sec3.1"><h3 id="sectitle0060" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.1. Admixed macaques</h3><div class="u-margin-s-bottom"><div id="p0100">As expected, given the known sexual dimorphism in macaque species (e.g., <a class="anchor anchor-primary" href="#bib60" name="bbib60" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib60"><span class="anchor-text-container"><span class="anchor-text">Plavcan, 2001</span></span></a>) and the obstetric function of the pelvis, our initial Procrustes ANOVAs showed a relatively high degree of sexual dimorphism in the pelvic morphology of the complete sample: sex explains ∼18% of the variation in shape (F [1, 136]&nbsp;=&nbsp;30.13, r<sup>2</sup>&nbsp;=&nbsp;0.18, <em>p</em>&nbsp;&lt;&nbsp;0.05). To investigate the nature of sexual dimorphism within the sample, we compared the fit of additive and interaction models of sex and shape (<a class="anchor anchor-primary" href="#tbl3" name="btbl3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl3"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;3</span></span></a>). The interaction model of the relationship between sex, admixture (% Chinese ancestry), and shape does not fit the data significantly better than a simple additive model, suggesting that sexual dimorphism does not vary with degree of admixture. This being the case, we used additive regression models with sex as the predictor and shape or admixture as the response to produce residuals that allowed us to adjust for sex in both shape and admixture variables for use in further analyses. These residuals are used in all subsequent analyses performed on the complete sample.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl3"><span class="captions text-s"><span id="cap0055"><p id="tspara0025"><span class="label">Table&nbsp;3</span>. Comparison (analysis of variance) of interactive and additive models for the relationship between sex, admixture (percent Chinese ancestry), and shape in the complete sample (<em>n</em>&nbsp;=&nbsp;138).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-center">Model</th><th scope="col" class="align-center">Residual df</th><th scope="col">df</th><th scope="col">RSS</th><th scope="col">SS</th><th scope="col">r<sup>2</sup></th><th scope="col">F</th><th scope="col"><em>p</em></th></tr></thead><tbody><tr class="valign-top"><td>Shape&nbsp;∼&nbsp;sex&nbsp;+&nbsp;admix</td><td class="align-char">135</td><td class="align-char">2+</td><td class="align-char">0.508</td><td></td><td class="align-char">0.000</td><td></td><td></td></tr><tr class="valign-top"><td>Shape&nbsp;∼&nbsp;sex ∗ admix</td><td class="align-char">134</td><td class="align-char">3+</td><td class="align-char">0.504</td><td class="align-char">0.004</td><td class="align-char">0.007</td><td class="align-char">1.105</td><td class="align-char">0.292</td></tr><tr class="valign-top"><td>Total</td><td class="align-char">137</td><td></td><td class="align-char">0.635</td><td></td><td></td><td></td><td></td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0030">The larger interactive model is not a significantly better fit than the smaller additive model.</div></div></div></div><div class="u-margin-s-bottom"><div id="p0105">After adjusting for sex in both shape and admixture variables, the amount of Chinese ancestry explains a very small, although significant, portion of remaining shape variation: approximately 2% (F [1, 136]&nbsp;=&nbsp;3.07, r<sup>2</sup>&nbsp;=&nbsp;0.02, <em>p</em>&nbsp;&lt;&nbsp;0.05). The weak effect of admixture on shape, in contrast to sexual dimorphism, can be seen when pairwise Procrustes distances between the sexes in each Chinese ancestry category are compared with within-sex variation between categories (<a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>). Also shown in <a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a> is variation between members of the same sex within the same admixture category. This can be considered baseline interindividual variation resulting from a combination of many factors, likely including age, weight, and lifetime experiences in the colony. Levels of variation between those with different Chinese ancestry categories, whether adjacent or more distant, are indistinguishable from that baseline variation, demonstrating the very weak effect of admixture compared with other sources of interindividual variation.</div><figure class="figure text-xs" id="fig3"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr3.jpg" height="399" alt="Figure&nbsp;3" aria-describedby="cap0020"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr3_lrg.jpg" target="_blank" download="" title="Download high-res image (160KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (160KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr3.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0020"><p id="fspara0020"><span class="label">Figure&nbsp;3</span>. Distributions of pairwise Procrustes distances generated directly from the data. Distributions show distances between members of different sexes within a Chinese ancestry category (orange, dashed line), between individuals of the same sex in the same Chinese ancestry category (gray, solid line), in categories 12.5% apart/adjacent categories (turquoise, solid line), and in categories 50% apart (purple, solid line). Variation between individuals in the same sex and ancestry category (gray, solid line) may be considered baseline levels of variation against which to compare the effects of sex or admixture. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div><div class="u-margin-s-bottom"><div id="p0110">The effect of ancestry on pelvic shape (adjusting both for sex) is mostly evident in the shape of the iliac blades (<a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;4</span></span></a><span>). In animals with more Chinese ancestry, the cranial border of the iliac blades is of relatively equal height, ventral to dorsal, whereas in animals with less Chinese ancestry, the iliac blades extend further laterally and the cranial edges are more angled, extending further cranially at their ventral extent. The dorsal part of the iliac blades, in the region of the <a href="/topics/agricultural-and-biological-sciences/vertebra" title="Learn more about vertebrae from ScienceDirect's AI-generated Topic Pages" class="topic-link">vertebrae</a> S2 and S3 (</span><a class="anchor anchor-primary" href="#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;1</span></span></a><span>), also extends more caudally in animals with higher regression scores. There are further differences in the ischium: in higher scoring animals, the medial border of the ischial tuberosities is more laterally placed, compared with animals with lower scores, and the superior border of the pubic symphysis is more ventrally placed. Finally, the <a href="/topics/agricultural-and-biological-sciences/sacrum" title="Learn more about sacrum from ScienceDirect's AI-generated Topic Pages" class="topic-link">sacrum</a> differs between individuals who vary in regression scores: higher scoring individuals have a sacrum that is craniocaudally shorter and more ventrally orientated in the region of S2/S3. All of these differences are very slight, however.</span></div><figure class="figure text-xs" id="fig4"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr4.jpg" height="485" alt="Figure&nbsp;4" aria-describedby="cap0025"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr4_lrg.jpg" target="_blank" download="" title="Download high-res image (509KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (509KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr4.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0025"><p id="fspara0025"><span class="label">Figure&nbsp;4</span>. Plot of regression scores showing the relationship between pelvic shape and admixture, both adjusted for sex. Jitter is used to improve the visibility of separate points. Full-bred Indian animals (0% Chinese ancestry) in red; full-bred Chinese animals (100% Chinese ancestry) in dark blue; all other admixture percentages correspond to a color scale between the two extremes, as shown to the right of the graph. Wireframes below the graph show minimum (in red) and maximum (in dark blue) scoring shapes on the regression score axis with displacement magnified by two to aid visualization. Views shown (left to right) are ventral, cranial, caudal, and right lateral. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div><div class="u-margin-s-bottom"><div id="p0115">The relationship between regression scores and admixture displayed in <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a> appears to approach a linear or additive relationship, meaning that change in shape and increasing Chinese ancestry is relatively proportional. This is supported by model comparison, which shows that a three-knot B spline model does not fit the data significantly better than a simple linear model of shape and Chinese ancestry % (adjusted for sex; <a class="anchor anchor-primary" href="#tbl4" name="btbl4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl4"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;4</span></span></a>).</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl4"><span class="captions text-s"><span id="cap0060"><p id="tspara0035"><span class="label">Table&nbsp;4</span>. Model comparison (analysis of variance) of three-knot B spline and simple additive models for the relationship between shape and admixture in the complete sample (<em>n</em>&nbsp;= 138<em>,</em> sex held constant for both variables).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Model</th><th scope="col">Residual df</th><th scope="col">df</th><th scope="col">RSS</th><th scope="col">SS</th><th scope="col">r<sup>2</sup></th><th scope="col">F</th><th scope="col"><em>p</em></th></tr></thead><tbody><tr class="valign-top"><td>Additive</td><td class="align-char">135</td><td class="align-char">2+</td><td class="align-char">0.508</td><td></td><td class="align-char">0.000</td><td></td><td></td></tr><tr class="valign-top"><td>B spline</td><td class="align-char">130</td><td class="align-char">5</td><td class="align-char">0.489</td><td class="align-char">0.120</td><td class="align-char">0.307</td><td class="align-char">1.038</td><td class="align-char">0.388</td></tr><tr class="valign-top"><td>Total</td><td class="align-char">137</td><td></td><td class="align-char">0.635</td><td></td><td></td><td></td><td></td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0040">The B-spline model is not a significantly better fit than the smaller additive model.</div></div></div></div><div class="u-margin-s-bottom"><div id="p0120">In the complete sample, there is no significant difference in pelvic centroid size between any pairwise comparison of full-bred Indian, Chinese, or admixed groups (<a class="anchor anchor-primary" href="#tbl5" name="btbl5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl5"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;5</span></span></a>). Similarly, there are no significant differences between group variances, showing that no one group is more variable in pelvic shape than another (<a class="anchor anchor-primary" href="#tbl6" name="btbl6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl6"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;6</span></span></a>, <a class="anchor anchor-primary" href="#tbl7" name="btbl7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl7"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;7</span></span></a>). This suggests there are no transgressive sizes and no increase in shape variation in admixed individuals in this sample.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl5"><span class="captions text-s"><span id="cap0065"><p id="tspara0045"><span class="label">Table&nbsp;5</span>. Analysis of variance of differences in centroid size between full-bred and admixed animals in the complete sample (<em>n</em>&nbsp;=&nbsp;138).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><th scope="col">df</th><th scope="col">SS</th><th scope="col">MS</th><th scope="col">r<sup>2</sup></th><th scope="col">F</th><th scope="col"><em>P</em></th></tr></thead><tbody><tr class="valign-top"><td>Admixture status<a class="anchor anchor-primary" href="#tbl5fna" name="btbl5fna" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl5fna"><span class="anchor-text-container"><span class="anchor-text"><sup>a</sup></span></span></a></td><td class="align-char">2</td><td class="align-char">11377</td><td class="align-char">5688.70</td><td class="align-char">0.04</td><td class="align-char">2.58</td><td class="align-char">0.07<a class="anchor anchor-primary" href="#tbl5fnb" name="btbl5fnb" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl5fnb"><span class="anchor-text-container"><span class="anchor-text"><sup>b</sup></span></span></a></td></tr><tr class="valign-top"><td>Residuals</td><td class="align-char">135</td><td class="align-char">298013</td><td class="align-char">2207.50</td><td class="align-char">0.96</td><td></td><td></td></tr><tr class="valign-top"><td>Total</td><td class="align-char">137</td><td class="align-char">309391</td><td></td><td></td><td></td><td></td></tr></tbody></table></div><dl class="footnotes"><dt id="tbl5fna">a</dt><dd><div class="u-margin-s-bottom" id="ntpara0010">Admixture status groups: full-bred Indian, full-bred Chinese, admixed.</div></dd><dt id="tbl5fnb">b</dt><dd><div class="u-margin-s-bottom" id="ntpara0015">There is no significant difference (<em>p</em>&nbsp;&gt;&nbsp;0.05) in centroid size between groups with different admixture status.</div></dd></dl></div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl6"><span class="captions text-s"><span id="cap0070"><p id="tspara0050"><span class="label">Table&nbsp;6</span>. Procrustes variances for admixture categories in the complete sample (<em>n</em>&nbsp;=&nbsp;138).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Admixture status</th><th scope="col">Procrustes variance</th></tr></thead><tbody><tr class="valign-top"><td>Ind</td><td class="align-char">0.0043</td></tr><tr class="valign-top"><td>Chi</td><td class="align-char">0.0049</td></tr><tr class="valign-top"><td>Ad</td><td class="align-char">0.0046</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0055">Ind,&nbsp;full-bred Indian; Chi,&nbsp;full-bred Chinese; Ad,&nbsp;admixed.</div></div></div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl7"><span class="captions text-s"><span id="cap0075"><p id="tspara0060"><span class="label">Table&nbsp;7</span>. Test for significant differences in variance (<a class="anchor anchor-primary" href="#tbl6" name="btbl6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl6"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;6</span></span></a>) between full-bred and admixed animals in the complete sample (<em>n</em>&nbsp;=&nbsp;138).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><th scope="col">Ind</th><th scope="col">Chi</th><th scope="col">Ad</th></tr></thead><tbody><tr class="valign-top"><td>Ind</td><td>–</td><td>0.00054</td><td>0.00026</td></tr><tr class="valign-top"><td>Chi</td><td>0.34</td><td>–</td><td>0.00028</td></tr><tr class="valign-top"><td>Ad</td><td>0.53</td><td>0.54</td><td>–</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0065">Ind,&nbsp;full-bred Indian; Chi,&nbsp;full-bred Chinese; Ad,&nbsp;admixed. Pairwise differences between Procrustes variances are given above the trace; <em>p</em>-values of tests of significance of differences are given below the trace. None of the comparisons reaches significance (<em>p</em> &gt; 0.05).</div></div></div></div></section><section id="sec3.2"><h3 id="sectitle0065" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.2. Full-bred macaques</h3><div class="u-margin-s-bottom"><div id="p0125">In contrast to the full sample analysis (<a class="anchor anchor-primary" href="#tbl3" name="btbl3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl3"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;3</span></span></a>), analysis of solely the Indian and Chinese full-bred macaques indicates that an interaction model of the relationship between sex and shape is a significantly better fit for the data than an additive model (<a class="anchor anchor-primary" href="#tbl8" name="btbl8" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl8"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;8</span></span></a>). This suggests a difference in sexual dimorphism between the two subspecies that is masked by the greater variation in the complete sample (refer also <a class="anchor anchor-primary" href="#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text">Clarke and O'Neil, 1999</span></span></a><span>). Owing to these apparent differences in sexual dimorphism, for the full-bred sample we adjusted for differences in the effects of sex on shape separately for each subspecies, by adding the difference between the subspecies-specific male and female means to the female configurations for each subspecies. A <a href="/topics/biochemistry-genetics-and-molecular-biology/principal-component-analysis" title="Learn more about PCA from ScienceDirect's AI-generated Topic Pages" class="topic-link">PCA</a> showing the separation between the subspecies on the main axes of sex-adjusted shape can be seen in </span><a class="anchor anchor-primary" href="#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;5</span></span></a>. The sex-adjusted residual shapes were used as response variables in a Procrustes ANOVA to quantify the effect of subspecies designation on shape (F [1, 34]&nbsp;=&nbsp;1.47, r<sup>2</sup>&nbsp;=&nbsp;0.07, <em>p</em>&nbsp;&lt;&nbsp;0.05). After accounting for sexual dimorphism, there is a significant but fairly weak relationship between the remaining shape variation and subspecies designation (<a class="anchor anchor-primary" href="#fig6" name="bfig6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig6"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;6</span></span></a>). Designation as Indian or Chinese subspecies accounts for approximately 7% of pelvic shape variation in the full-bred sample.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl8"><span class="captions text-s"><span id="cap0080"><p id="tspara0070"><span class="label">Table&nbsp;8</span>. Comparison (analysis of variance) of additive and interactive models of the relationship between sex, subspecies designation (Indian or Chinese), and shape in the full-bred sample (<em>n</em>&nbsp;=&nbsp;36).</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Model</th><th scope="col">Residual Df</th><th scope="col">RSS</th><th scope="col">SS</th><th scope="col">R<sup>2</sup></th><th scope="col">F</th><th scope="col"><em>p</em></th></tr></thead><tbody><tr class="valign-top"><td>Shape&nbsp;∼&nbsp;sex&nbsp;+&nbsp;subspecies</td><td class="align-char">33</td><td class="align-char">0.149</td><td></td><td class="align-char">0.000</td><td></td><td></td></tr><tr class="valign-top"><td>Shape&nbsp;∼&nbsp;sex ∗ subspecies</td><td class="align-char">32</td><td class="align-char">0.140</td><td class="align-char">0.010</td><td class="align-char">0.059</td><td class="align-char">2.187</td><td class="align-char">0.03</td></tr><tr class="valign-top"><td>Total</td><td class="align-char">35</td><td class="align-char">0.160</td><td></td><td></td><td></td><td></td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0075">The interactive model is a significantly better fit than the smaller additive model, as denoted by the significant (&lt;0.05) <em>p</em>-value.</div></div></div><figure class="figure text-xs" id="fig5"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr5.jpg" height="340" alt="Figure&nbsp;5" aria-describedby="cap0030"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr5_lrg.jpg" target="_blank" download="" title="Download high-res image (134KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (134KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr5.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0030"><p id="fspara0030"><span class="label">Figure&nbsp;5</span>. PCA plot showing sex-adjusted shape differences between full-bred species. PC1 accounts for 25.2% variance and PC2 accounts for 13.0% variance. Full-bred Indian animals (0% Chinese ancestry) shown as red circles; full-bred Chinese animals (100% Chinese ancestry) shown as blue squares. This PCA shows the large degree of overlap in pelvic shape between the subspecies on the primary axes of variation. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.) PCA, principal component analysis.</p></span></span></figure><figure class="figure text-xs" id="fig6"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr6.jpg" height="428" alt="Figure&nbsp;6" aria-describedby="cap0035"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr6_lrg.jpg" target="_blank" download="" title="Download high-res image (417KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (417KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr6.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0035"><p id="fspara0035"><span class="label">Figure&nbsp;6</span>. Regression score plot showing the relationship between sex-adjusted shape associated with subspecies designation and subspecies designation. Jitter is used to improve the visibility of separate points. Full-bred Indian (0% Chinese ancestry) animals shown in red; full-bred Chinese (100% Chinese ancestry) animals shown in blue. Wireframes below graph show mean Indian shape (in red) and mean Chinese shape (in blue). Note that this differs from <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>, where the wireframes model the extremes of regression scores. As in <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>, displacement is magnified by two to aid visualization. Views shown (left to right) are ventral, cranial, caudal, and right lateral. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div><div class="u-margin-s-bottom" id="p0130"><a class="anchor anchor-primary" href="#fig6" name="bfig6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig6"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;6</span></span></a> shows the relationship between regression scores summarizing the proportion of sex-adjusted shape associated with subspecies designation and subspecies designation itself. As with the full sample, there is a difference in pelvic shape associated with Chinese ancestry. Chinese individuals generally have higher regression scores than Indian individuals, showing that they inhabit a different region of the shape space. Compared with the Chinese mean shape, the Indian mean shape is characterized by dorsoventrally and mediolaterally broader iliac blades, which are more angled across the top, running from high ventrally to low dorsally, whereas the tops of the mean Chinese iliac blades are more level. The mean Indian sacrum is more dorsocaudally placed between the ilia, and the sacral alae are shorter craniocaudally. The acetabulae of the mean Indian shape are placed more ventrolaterally, and the pubic symphysis of the mean Indian shape is more dorsally orientated. Finally, the ischial tuberosities of the mean Indian shape extend more caudomedially than those of the mean Chinese shape. These shape differences echo what are seen in <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a> and suggest that more Chinese ancestry leads to a pelvis that more closely approaches the mean Chinese shape.</div></section><section id="sec3.3"><h3 id="sectitle0070" class="u-h4 u-margin-m-top u-margin-xs-bottom">3.3. Projecting admixed individuals onto a vector of full-bred shape</h3><div class="u-margin-s-bottom"><div id="p0135">The effect of admixture on sex-adjusted pelvic shape in the complete sample is small, as emphasized by the comparison with baseline interindividual variation within a sex and ancestry category (<a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>). Our analyses show, however, that there is a significant relationship between Chinese ancestry and shape and suggest that this is relationship is broadly additive. To examine this possibility further, we created a vector of shape variation using only the sex-adjusted full-bred Indian and Chinese individuals (<em>n</em>&nbsp;=&nbsp;36) and projected the sex-adjusted admixed individuals (<em>n</em>&nbsp;=&nbsp;102) onto this vector (<a class="anchor anchor-primary" href="#fig7" name="bfig7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig7"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;7</span></span></a>). This figure shows an idealized, linear relationship between Chinese ancestry and sex-adjusted pelvic shape. Comparison of <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>, <a class="anchor anchor-primary" href="#fig7" name="bfig7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig7"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;7</span></span></a> shows there is very little difference between the modeled relationship and that obtained with the actual data, supporting the assertion that the proportion of Chinese ancestry affects pelvic shape in a predictable, proportional manner.</div><figure class="figure text-xs" id="fig7"><span><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr7.jpg" height="312" alt="Figure&nbsp;7" aria-describedby="cap0040"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr7_lrg.jpg" target="_blank" download="" title="Download high-res image (238KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (238KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-gr7.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0040"><p id="fspara0040"><span class="label">Figure&nbsp;7</span>. Regression score plot showing the relationship between sex-adjusted shape associated with Chinese ancestry and Chinese ancestry. Shapes of admixed individuals are projected onto a vector constructed using full-bred shape only. Jitter is used to improve the visibility of separate points. Full-bred Indian (0% Chinese ancestry) animals in red; Full-bred Chinese (100% Chinese ancestry) animals in dark blue; all other admixture percentages correspond to a color scale between the two extremes, as shown to the right of the graph. Compare with <a class="anchor anchor-primary" href="#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></span></span></figure></div></section></section><section id="sec4"><h2 id="sectitle0075" class="u-h4 u-margin-l-top u-margin-xs-bottom">4. Discussion</h2><div class="u-margin-s-bottom" id="p0140">The effect of admixture on pelvic shape variation in Indian and Chinese <em>M.&nbsp;mulatta</em> is very slight. To put this into context, shape variation between individuals from different Chinese ancestry categories is no greater than intracategory (within sex) variation (<a class="anchor anchor-primary" href="#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;3</span></span></a>). The latter shape variation probably results from multiple variables including age, weight, and stochastic variation. This is in clear contrast to the strong effect of sex on pelvic shape, which cuts across all Chinese ancestry categories.</div><section id="sec4.1"><h3 id="sectitle0080" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.1. The skeletal context</h3><div class="u-margin-s-bottom" id="p0145">The low levels of pelvic shape differentiation between members of different Chinese ancestry categories are not surprising, given the relatively small differences we found between the full-bred subspecies (<a class="anchor anchor-primary" href="#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Fig.&nbsp;5</span></span></a>). Given the research discussed earlier (see <a class="anchor anchor-primary" href="#sec1" name="bsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec1"><span class="anchor-text-container"><span class="anchor-text">Introduction</span></span></a>) suggesting that there are noticeable gross phenotypic differences between Indian and Chinese <em>M.&nbsp;mulatta</em> (<a class="anchor anchor-primary" href="#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text">Clarke and O'Neil, 1999</span></span></a>; <a class="anchor anchor-primary" href="#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a>; <a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2005</span></span></a>), it is possible that the similarity we see in the pelvic data is particular to that region. Skeletal regions display different degrees of canalization and variation (<a class="anchor anchor-primary" href="#bib15" name="bbib15" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib15"><span class="anchor-text-container"><span class="anchor-text">Buck et&nbsp;al., 2010</span></span></a>), and the strong selective pressures resulting from the requirements of parturition and locomotion might be expected to lead to stabilizing selection acting on the pelvis. The macaque fit between fetus and birth canal is relatively tight during birth, comparable to that seen in extant humans, and there is evidence in <em>M.&nbsp;mulatta</em> of selection on the pelvis to reduce cephalopelvic dystocia (<a class="anchor anchor-primary" href="#bib49" name="bbib49" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib49"><span class="anchor-text-container"><span class="anchor-text">Kawada et&nbsp;al., 2020</span></span></a>). The present study is the first analysis of hybrid pelvic morphology. In fact, to the best of our knowledge, it is the first study of any postcranial shape in known hybrids where skeletal shape has been analyzed directly, rather than through the use of proxy measurements taken by palpating bones through soft tissue in live animals (e.g., <a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a>). Thus, the effects of admixture more widely in human and nonhuman primate pelvic shape remain unknown, and the possibility of regional anatomical differences in admixture signature will be examined in forthcoming work on the crania and other bones from the same sample. If the phenotypic similarity between Chinese and Indian <em>M.&nbsp;mulatta</em><span> is replicated in other regions of the <a href="/topics/agricultural-and-biological-sciences/skeleton" title="Learn more about skeleton from ScienceDirect's AI-generated Topic Pages" class="topic-link">skeleton</a>, these taxa may be relatively phenotypically undifferentiated (skeletally), despite clear genetic differences between them (</span><a class="anchor anchor-primary" href="#bib88" name="bbib88" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib88"><span class="anchor-text-container"><span class="anchor-text">Zhang and Shi, 1993</span></span></a>; <a class="anchor anchor-primary" href="#bib71" name="bbib71" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib71"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a>; <a class="anchor anchor-primary" href="#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text">Ferguson et&nbsp;al., 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib48" name="bbib48" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib48"><span class="anchor-text-container"><span class="anchor-text">Kanthaswamy et&nbsp;al., 2009</span></span></a>).</div></section><section id="sec4.2"><h3 id="sectitle0085" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.2. The macaque context</h3><div class="u-margin-s-bottom" id="p0150">Morphological research on hybrids between Japanese (<em>M.&nbsp;fuscata</em>) and Taiwanese (<em>M.&nbsp;cyclopis</em>) macaques shows results comparable with those presented here (<a class="anchor anchor-primary" href="#bib40" name="bbib40" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib40"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2012</span></span></a>; <a class="anchor anchor-primary" href="#bib46" name="bbib46" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib46"><span class="anchor-text-container"><span class="anchor-text">Ito et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Boel et&nbsp;al., 2019</span></span></a>). In particular, hybridization between <em>M.&nbsp;fuscata</em> and <em>M.&nbsp;cyclopis</em><span> results in a linear correspondence between relative <a href="/topics/agricultural-and-biological-sciences/tail" title="Learn more about tail from ScienceDirect's AI-generated Topic Pages" class="topic-link">tail</a> length and degree of admixture, with no extreme phenotypes recorded (</span><a class="anchor anchor-primary" href="#bib40" name="bbib40" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib40"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2012</span></span></a>), contrary to what is seen for many other nonhuman primate hybrids (refer following paragraphs). <em>M.&nbsp;fuscata</em> and <em>M. cyclopsis</em> are estimated to have a divergence time of ∼170 ka (<a class="anchor anchor-primary" href="#bib16" name="bbib16" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib16"><span class="anchor-text-container"><span class="anchor-text">Chu et&nbsp;al., 2007</span></span></a>), close to the ∼162 ka estimated for Indian and Chinese <em>M.&nbsp;mulatta</em> (<a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a>). GMM-based cranial analyses of these hybrids show no greater levels of variation in hybrids than in parent species and no clear effects of hybrid status on the frequency of anomalous nonmetric traits (<a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Boel et&nbsp;al., 2019</span></span></a><span>). Qualitative comparison of PCAs of the <a href="/topics/agricultural-and-biological-sciences/morphospace" title="Learn more about morphospace from ScienceDirect's AI-generated Topic Pages" class="topic-link">morphospace</a> inhabited by the two full-bred samples, however, does seem to indicate a greater degree of separation between </span><em>M.&nbsp;fuscata</em> and <em>M.&nbsp;cyclopis</em> than between the <em>M.&nbsp;mulatta</em> subspecies and a tendency of all hybrids toward <em>M.&nbsp;fuscata</em> shape, rather than the linear relationship with admixture we describe for our sample (<a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Boel et&nbsp;al., 2019</span></span></a>; refer also analysis of linear measurements in; <a class="anchor anchor-primary" href="#bib46" name="bbib46" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib46"><span class="anchor-text-container"><span class="anchor-text">Ito et&nbsp;al., 2015</span></span></a>). The Japanese-Taiwanese hybrids were wild caught in Japan (<a class="anchor anchor-primary" href="#bib46" name="bbib46" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib46"><span class="anchor-text-container"><span class="anchor-text">Ito et&nbsp;al., 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">Boel et&nbsp;al., 2019</span></span></a>), at a higher latitude and in a more temperate climate than Taiwan. Therefore, there may be some adaptive advantage in <em>M.&nbsp;fuscata</em> morphology leading to the dominance of <em>M.&nbsp;fuscata</em>-like morphology in hybrids and also potentially selection for greater phenotypic divergence between the parent taxa after their separation in the Pleistocene, if their habitats are less similar than those of Indian and Chinese <em>M.&nbsp;mulatta</em>.</div></section><section id="sec4.3"><h3 id="sectitle0090" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.3. The nonhuman primate context</h3><div class="u-margin-s-bottom" id="p0155">In contrast to much of the previous research on nonhuman primates (<a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>; <a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a><span>) with the exception of other macaque studies (refer previous paragraphs), in this study, we found no increased shape variation and no <a href="/topics/agricultural-and-biological-sciences/heterosis" title="Learn more about heterosis from ScienceDirect's AI-generated Topic Pages" class="topic-link">heterosis</a> or transgressive size in the hybrid sample. There are several possible factors that may contribute to this difference from earlier work, including the methods used, the range of admixture of the hybrids making up each sample and the degree of similarity (phenotypic or genetic) between the parent taxa in each study.</span></div><div class="u-margin-s-bottom" id="p0160">In the present study, we used landmark-based GMM to compare overall pelvic shape, while by contrast, most studies of hybrid morphology in nonhuman primates (e.g., <a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>; <a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a>; <a class="anchor anchor-primary" href="#bib46" name="bbib46" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib46"><span class="anchor-text-container"><span class="anchor-text">Ito et&nbsp;al., 2015</span></span></a><span>) have used traditional morphometrics and nonmetric traits. This may, in part, account for differences in results between studies, as analysis of individual nonmetric traits or measurements can emphasize greater difference between specific regions that may be averaged out by analyses of overall shape using GMM. Similar GMM used to analyze shape in hybrid <a href="/topics/agricultural-and-biological-sciences/mouse" title="Learn more about mouse from ScienceDirect's AI-generated Topic Pages" class="topic-link">mouse</a> crania show patterns of increased variation and extreme phenotypes similar to those seen in baboons (</span><a class="anchor anchor-primary" href="#bib80" name="bbib80" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib80"><span class="anchor-text-container"><span class="anchor-text">Warren et&nbsp;al., 2018</span></span></a>), however, suggesting that GMM is not entirely responsible for the differences in results between studies.</div><div class="u-margin-s-bottom" id="p0165">Hybridization in the baboon colony studied by <a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al. (2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014)</span></span></a> was only a few generations deep, and many transgressive or anomalous morphologies characterized as the products of hybridization are most likely to occur in the F1 generation or in generations close to the interbreeding event (<a class="anchor anchor-primary" href="#bib62" name="bbib62" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib62"><span class="anchor-text-container"><span class="anchor-text">Prentis et&nbsp;al., 2008</span></span></a><span>). This likely contributes to our lack of such a signal in the current sample, which contains fewer early generation hybrids and only one F1 individual. More subtle manifestations of expected hybrid morphology, however, such as some individuals of transgressive size/shape and greater variation within admixed samples, have also been documented in naturally hybridizing populations of <a href="/topics/biochemistry-genetics-and-molecular-biology/callitrichinae" title="Learn more about marmosets from ScienceDirect's AI-generated Topic Pages" class="topic-link">marmosets</a> and howler monkeys, where far fewer individuals are likely to be early generation hybrids (</span><a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a>). This suggests that the admixture proportions sampled here may not completely explain the more linear relationship between admixture and pelvic morphology in our results.</div><div class="u-margin-s-bottom" id="p0170">A final possible reason for the novelty of our results, in relation to the majority of nonhuman primate hybrid research, may be relatively short divergence time between the <em>M.&nbsp;mulatta</em> parent taxa compared with the parent taxa investigated in previous work. Baboon taxonomy is much debated, but a recent article by <a class="anchor anchor-primary" href="#bib64" name="bbib64" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib64"><span class="anchor-text-container"><span class="anchor-text">Rogers et&nbsp;al. (2019)</span></span></a><span> suggests that the <a href="/topics/agricultural-and-biological-sciences/lineages" title="Learn more about lineages from ScienceDirect's AI-generated Topic Pages" class="topic-link">lineages</a><span> of the <a href="/topics/agricultural-and-biological-sciences/olives" title="Learn more about olive from ScienceDirect's AI-generated Topic Pages" class="topic-link">olive</a> and yellow baboon allotaxa studied by Ackermann and colleagues (</span></span><a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>) diverged about 1.4&nbsp;Ma. Black (<span><span><a href="/topics/biochemistry-genetics-and-molecular-biology/alouatta" title="Learn more about Alouatta from ScienceDirect's AI-generated Topic Pages" class="topic-link">Alouatta</a> pigra</span></span>) and mantled (<em>A.&nbsp;palliata</em>) howler monkeys, as studied by <a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz (2013)</span></span></a>, are still more phylogenetically divergent, with a split time of ∼3&nbsp;Ma (<a class="anchor anchor-primary" href="#bib22" name="bbib22" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib22"><span class="anchor-text-container"><span class="anchor-text">Cortés-Ortiz et&nbsp;al., 2003</span></span></a>), while <span><span><a href="/topics/agricultural-and-biological-sciences/callithrix" title="Learn more about Callithrix from ScienceDirect's AI-generated Topic Pages" class="topic-link">Callithrix</a></span></span> is a fairly young genus and the black-tufted and white-headed marmosets (<em>C</em>. <em>penicillata</em> and <em>C.&nbsp;geoffroyi</em>, respectively), whose hybrids have been studied by <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al. (2014)</span></span></a>, split ∼700 ka (<a class="anchor anchor-primary" href="#bib53" name="bbib53" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib53"><span class="anchor-text-container"><span class="anchor-text">Malukiewicz et&nbsp;al., 2017</span></span></a>). Based on these dates, the genetic distance between Indian and Chinese <em>M.&nbsp;mulatta</em>, estimated to have diverged ∼162 ka (<a class="anchor anchor-primary" href="#bib43" name="bbib43" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib43"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a>), is considerably less than that between these other taxa. There also remains the possibility that the rhesus macaques are less differentiated morphologically relative to their genetic distance, perhaps due to stabilizing selection (refer also to section <a class="anchor anchor-primary" href="#sec4.4" name="bsec4.4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec4.4"><span class="anchor-text-container"><span class="anchor-text">4.4</span></span></a><span>). This is an interesting possibility, which has not yet been investigated and which should be followed up by the comparison of further nonhuman primate taxa pairs with similar <a href="/topics/biochemistry-genetics-and-molecular-biology/phylogeny" title="Learn more about phylogenetic from ScienceDirect's AI-generated Topic Pages" class="topic-link">phylogenetic</a> divergence times.</span></div><div class="u-margin-s-bottom" id="p0175"><span><span>The differences between results presented here and previous work on nonhuman primate hybrids, as well as between previous studies, demonstrate the complex relationship between taxonomic divergence and hybrid morphology, which is not yet fully understood. Similarly, these differences indicate the need for further investigation to better understand which nonhuman primate species make the best models for <a href="/topics/biochemistry-genetics-and-molecular-biology/hominin" title="Learn more about hominin from ScienceDirect's AI-generated Topic Pages" class="topic-link">hominin</a> hybridization. It has been shown that more genetically divergent taxa lead to more transgressive hybrids, due to increasing fixation of different alleles in each parent taxon over time (i.e., fixed differences in alleles as opposed to differences in allele frequencies), which may lead to </span><a href="/topics/agricultural-and-biological-sciences/complementary-genes" title="Learn more about complementary gene from ScienceDirect's AI-generated Topic Pages" class="topic-link">complementary gene</a> action or epistasis in hybrid offspring (</span><a class="anchor anchor-primary" href="#bib72" name="bbib72" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib72"><span class="anchor-text-container"><span class="anchor-text">Stelkens and Seehausen, 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib73" name="bbib73" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib73"><span class="anchor-text-container"><span class="anchor-text">Stelkens et&nbsp;al., 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib20" name="bbib20" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib20"><span class="anchor-text-container"><span class="anchor-text">Comeault and Matute, 2018</span></span></a>; <a class="anchor anchor-primary" href="#bib8" name="bbib8" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib8"><span class="anchor-text-container"><span class="anchor-text">Allen et&nbsp;al., 2020</span></span></a>). The relationship between phenotypic divergence and transgressive phenotypes is less clear, however (<a class="anchor anchor-primary" href="#bib72" name="bbib72" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib72"><span class="anchor-text-container"><span class="anchor-text">Stelkens and Seehausen, 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib73" name="bbib73" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib73"><span class="anchor-text-container"><span class="anchor-text">Stelkens et&nbsp;al., 2009</span></span></a>), and relationships among divergence time and genetic and phenotypic differentiation appear to differ between groups of organisms (<a class="anchor anchor-primary" href="#bib72" name="bbib72" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib72"><span class="anchor-text-container"><span class="anchor-text">Stelkens and Seehausen, 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib55" name="bbib55" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib55"><span class="anchor-text-container"><span class="anchor-text">McGee et&nbsp;al., 2016</span></span></a>). Most vertebrate research on these relationships to date has been in fish (<a class="anchor anchor-primary" href="#bib72" name="bbib72" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib72"><span class="anchor-text-container"><span class="anchor-text">Stelkens and Seehausen, 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib73" name="bbib73" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib73"><span class="anchor-text-container"><span class="anchor-text">Stelkens et&nbsp;al., 2009</span></span></a>; <a class="anchor anchor-primary" href="#bib55" name="bbib55" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib55"><span class="anchor-text-container"><span class="anchor-text">McGee et&nbsp;al., 2016</span></span></a>). Thus, it is also unclear how well these results translate to primates, making this area ripe for further investigation.</div></section><section id="sec4.4"><h3 id="sectitle0095" class="u-h4 u-margin-m-top u-margin-xs-bottom">4.4. The paleoanthropological context</h3><div class="u-margin-s-bottom" id="p0180">From a paleoanthropological point of view, it is relevant that there appears to be considerably less variation in pelvic shape between Indian and Chinese <em>M.&nbsp;mulatta</em> than between Neanderthals and <em>H.&nbsp;sapiens</em>, a hybridizing taxon pair with reasonably comparable levels of genetic relatedness (see <a class="anchor anchor-primary" href="#sec1" name="bsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec1"><span class="anchor-text-container"><span class="anchor-text">Introduction</span></span></a>)<em>.</em> Further examination of other aspects of the <em>M.&nbsp;mulatta</em> skeleton and other pairs of nonhuman primate taxa will be required to determine whether, given their genetic distance, Indian and Chinese rhesus macaques are relatively phenotypically similar or whether <em>H.&nbsp;sapiens</em> and Neanderthals are more divergent than would be expected within the context of the primate order. If shown to be present throughout the skeleton, low levels of phenotypic differentiation between Chinese and Indian <em>M.&nbsp;mulatta</em><span> might suggest stabilizing selection, while the mechanism leading to relatively high differentiation between the hominin taxa could be either <a href="/topics/biochemistry-genetics-and-molecular-biology/directional-selection" title="Learn more about directional selection from ScienceDirect's AI-generated Topic Pages" class="topic-link">directional selection</a> or neutral evolution. If stabilizing selection has played a lesser role in the evolutionary history of the hominin pelvis than that of macaques, this could reflect a wider trend. For example, </span><a class="anchor anchor-primary" href="#bib69" name="bbib69" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib69"><span class="anchor-text-container"><span class="anchor-text">Schroeder and von Cramon Taubadel (2017)</span></span></a><span> have shown that across all sampled extant <a href="/topics/agricultural-and-biological-sciences/hominoids" title="Learn more about hominoid from ScienceDirect's AI-generated Topic Pages" class="topic-link">hominoid</a> taxa, hominoid cranial evolution is characterized by stabilizing selection with very few exceptions, one of which is marked directional selection in the hominin lineage after its split with the chimpanzee lineage. The pelvic evidence from the present study may suggest that this pattern holds throughout the skeleton. The presence of unusually high levels of phenotypic differentiation in hominins (</span><a class="anchor anchor-primary" href="#bib52" name="bbib52" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib52"><span class="anchor-text-container"><span class="anchor-text">Lynch, 1990</span></span></a>) is supported by comparison of rates of cranial evolution in the <em>Pan</em> clade versus the Neanderthal/<em>H.&nbsp;sapiens</em> clade (<a class="anchor anchor-primary" href="#bib82" name="bbib82" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib82"><span class="anchor-text-container"><span class="anchor-text">Weaver, 2014</span></span></a>) and by comparisons between Neanderthal/<em>H.&nbsp;sapiens</em> morphological divergence and that between other catarrhine pairs (<a class="anchor anchor-primary" href="#bib41" name="bbib41" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib41"><span class="anchor-text-container"><span class="anchor-text">Harvati et&nbsp;al., 2004</span></span></a>). It has been suggested that a unique human ecological niche, typified by cultural adaptation, has allowed the build-up of uncommonly great neutral variation in our lineage due to reduced selection (including stabilizing selection) compared with other organisms (<a class="anchor anchor-primary" href="#bib3" name="bbib3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib3"><span class="anchor-text-container"><span class="anchor-text">Ackermann and Cheverud, 2004</span></span></a>; <a class="anchor anchor-primary" href="#bib85" name="bbib85" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib85"><span class="anchor-text-container"><span class="anchor-text">Wells and Stock, 2007</span></span></a>; <a class="anchor anchor-primary" href="#bib84" name="bbib84" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib84"><span class="anchor-text-container"><span class="anchor-text">Weaver and Stringer, 2015</span></span></a>; <a class="anchor anchor-primary" href="#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text">Buck et&nbsp;al., 2019</span></span></a>).</div><div class="u-margin-s-bottom" id="p0185">There is, however, also a strong argument for selection leading to greater pelvic variation between <em>H.&nbsp;sapiens</em> and Neanderthals than between Indian and Chinese <em>M.&nbsp;mulatta</em>. Indian- and Chinese-derived <em>M.&nbsp;mulatta</em> are likely phenotypically similar, at least in part because they have inhabited similar habitats since their divergence (<a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2005</span></span></a>). This is in contrast to <em>H.&nbsp;sapiens</em> and Neanderthals, which adapted for hundreds of thousands of years to the disparate climates and habitats of Africa and Western Eurasia, respectively (<a class="anchor anchor-primary" href="#bib44" name="bbib44" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib44"><span class="anchor-text-container"><span class="anchor-text">Holliday, 1997a</span></span></a>, <a class="anchor anchor-primary" href="#bib45" name="bbib45" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib45"><span class="anchor-text-container"><span class="anchor-text">b</span></span></a>). Climate has played a role in extant human pelvic shape variation (<a class="anchor anchor-primary" href="#bib11" name="bbib11" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib11"><span class="anchor-text-container"><span class="anchor-text">Betti et&nbsp;al., 2014</span></span></a>), and it is also thought to have affected pelvic morphology in extinct hominins (<a class="anchor anchor-primary" href="#bib66" name="bbib66" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib66"><span class="anchor-text-container"><span class="anchor-text">Ruff, 1993</span></span></a>; <a class="anchor anchor-primary" href="#bib81" name="bbib81" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib81"><span class="anchor-text-container"><span class="anchor-text">Weaver, 2009</span></span></a>). In addition to, or as an alternative to the effects of climate, directional selection leading to cranial vault expansion in the hominin lineage, as reported by <a class="anchor anchor-primary" href="#bib69" name="bbib69" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib69"><span class="anchor-text-container"><span class="anchor-text">Schroeder and von Cramon Taubadel (2017)</span></span></a>, could play a role in pelvic change via the requirement for congruence during parturition. If this is the case, the simultaneous cranial expansion but divergent cranial shapes in the <em>H.&nbsp;sapiens</em> and Neanderthal lineages could have contributed to their differences in pelvic morphology.</div><div class="u-margin-s-bottom" id="p0190">The seeds of relatively great divergence in pelvic shape between hominin species may lie still deeper in hominin evolution. It has been suggested that integration has been relaxed in the human pelvis relative to other hominids, allowing for greater evolvability and selection for bipedal morphology (<a class="anchor anchor-primary" href="#bib36" name="bbib36" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib36"><span class="anchor-text-container"><span class="anchor-text">Grabowski et&nbsp;al., 2011</span></span></a>). This possibility is borne out by the high levels of intraspecific geographic variation in human pelvic morphology compared with other skeletal regions (<a class="anchor anchor-primary" href="#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text">Betti and Manica, 2018</span></span></a>). A breakdown in integration is likely governed by changes to the underlying genetic architecture controlling pelvic development and evidence that distinctive hominin pelvis shape results from a unique growth trajectory emerging relatively early in development also suggests modification of genetically determined pathways (<a class="anchor anchor-primary" href="#bib89" name="bbib89" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib89"><span class="anchor-text-container"><span class="anchor-text">Zirkle and Lovejoy, 2019</span></span></a>). Different genetic underpinning and levels of integration in macaques and hominins could mean that similar degrees of genetic difference could affect phenotypic outcomes in different ways. This scenario could facilitate the potential development of adaptive differences in one group (e.g., Neanderthals/<em>H.&nbsp;sapiens</em>) but neutral divergence in the other (e.g., Indian/Chinese <em>M.&nbsp;mulatta</em>). We note that the possibilities of directional selection and unusually high neutral variation are not mutually exclusive and could have combined to produce particularly phenotypically divergent taxa in <em>H.&nbsp;sapiens</em> and Neanderthals.</div><div class="u-margin-s-bottom" id="p0195">One could argue that, owing to their lower levels of phenotypic disparity relative to the time depth of their divergence, Indian/Chinese <em>M.&nbsp;mulatta</em> pelves are not a good model for the phenotypic effects of hybridization in hominins. It may, however, be difficult to find a pair of primate taxa with a similar divergence time to <em>H.&nbsp;sapiens</em>/Neanderthals and similar morphological disparity, highlighting the peculiarity of these hominin sister taxa. Most of the taxon pairs in the nonhuman primate hybrid literature have greater phylogenetic split times, and, although there is no way of directly comparing parent taxon phenotypic disparity between studies, the stronger admixture effects seen in previous studies may in part be related to greater phenotypic differentiation between parent taxa (see section <a class="anchor anchor-primary" href="#sec4.3" name="bsec4.3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="sec4.3"><span class="anchor-text-container"><span class="anchor-text">4.3</span></span></a>).</div><div class="u-margin-s-bottom" id="p0200">Although weak, the linear nature of the relationship between admixture and pelvic shape we find here is nonetheless informative. Greater Chinese ancestry leads to a pelvic shape which is closer to the mean shape for full-bred Chinese individuals, and the difference in Chinese ancestry is proportional to the difference in pelvic shape. If further investigation bears out the suggestion of the results we present here—that the morphology of hybrids between closely related primate taxa is proportional to the input of each parent taxon—this information can be used to further investigate the expected morphology of hominin hybrids. Models of expected hominin hybrid morphology can be constructed with the known morphology of parent taxa; for example, Neanderthal and <em>H.&nbsp;sapiens</em> crania as the endpoints of the hypothetical shape vector along which all admixed individuals would fall. Thus, the magnitude of disparity between parental shapes would not affect the pattern of the hybrids that fall between them, if the pattern of variation is a valid analog.</div></section></section><section id="sec5"><h2 id="sectitle0100" class="u-h4 u-margin-l-top u-margin-xs-bottom">5. Conclusions</h2><div class="u-margin-s-bottom" id="p0205">From a large sample of multigenerational hybrids and full-bred Indian and Chinese <em>M.&nbsp;mulatta</em>, we find the effects of admixture on pelvic shape to be very small, albeit significant, comparable with interindividual differences between members of the same sex and admixture category. The admixture signal we identify is linear, with greater Chinese ancestry resulting in a pelvic shape that more closely approximates that of full-bred Chinese <em>M.&nbsp;mulatta</em>. We find no evidence for an increase in size or shape variation nor transgressive morphologies in hybrids. Taken in the context of the existing nonhuman primate hybrid literature, the small effect and additive nature of the admixture signal in <em>M.&nbsp;mulatta</em> pelvic shape is likely affected by a combination of factors, including the phylogenetic closeness of the parent taxa relative to that between previously studied taxon pairs (<a class="anchor anchor-primary" href="#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a>, <a class="anchor anchor-primary" href="#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text">2014</span></span></a>; <a class="anchor anchor-primary" href="#bib50" name="bbib50" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib50"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a>; <a class="anchor anchor-primary" href="#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a>), the similarity of Indian and Chinese <em>M.&nbsp;mulatta</em> habitat and ecological niche since their divergence (<a class="anchor anchor-primary" href="#bib39" name="bbib39" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib39"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2005</span></span></a>), and the low number of early generation hybrids in the present study. Despite the interesting disparity in phenotypic divergence between the Indian and Chinese <em>M.&nbsp;mulatta</em> and Neanderthals/<em>H.&nbsp;sapiens</em>, which deserves further investigation, the relative similarity in phylogenetic closeness (refer also <a class="anchor anchor-primary" href="#bib8" name="bbib8" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib8"><span class="anchor-text-container"><span class="anchor-text">Allen et&nbsp;al., 2020</span></span></a>) and the multiple generations of hybridization in the current sample of <em>M.&nbsp;mulatta</em> have resulted in a useful analog for hominin hybridization. Models of hypothetical outcomes of hybridization between Neanderthals and <em>H.&nbsp;sapiens</em>, using the pattern revealed by these <em>M.&nbsp;mulatta</em> results, can now be tested against the fossil record to increase our understanding of the impacts of interbreeding on human evolution.</div></section></div><section id="p00101"><h2 id="p00102" class="u-h4 u-margin-l-top u-margin-xs-bottom">Conflict of interest statement</h2><div class="u-margin-s-bottom" id="p00102a">The authors declare no conflict of interest.</div></section><section id="ack0010"><h2 id="sectitle0105" class="u-h4 u-margin-l-top u-margin-xs-bottom">Acknowledgments</h2><div class="u-margin-s-bottom" id="p0210">This research was funded by the <span id="gs1">National Science Foundation</span> (BCS <a class="anchor anchor-primary" href="#gs1"><span class="anchor-text-container"><span class="anchor-text">1623366</span></span></a>, <a class="anchor anchor-primary" href="#gs2"><span class="anchor-text-container"><span class="anchor-text">1720128</span></span></a>) and the <span id="gs2">Leakey Foundation</span>, with further support to R.R.A. from the <span id="gs10">National Research Foundation of South Africa</span>. The authors thank the Editor, Associate Editor, and three anonymous reviewers for their constructive comments, which have much improved this article. The authors thank Mark Grote for his substantial statistical assistance. The authors are grateful to CNPRC staff, including Alice Tarantal for overseeing CT scanning, and Paul-Michael Sosa, Daniel Quiros Molina, Jennifer Short, and Sarah Davis for their extensive help with data collection. The authors thank Irene Englis and staff at the University of California Davis Museum of Wildlife and Fish Biology, for skeletonizing remains involved in this study. L.T.B. is grateful to Alex Cantó Pastor, Austin Reynolds, Mark Grabowski, Aurélien Mounier, and Florent Détroit for help with coding, and the authors thank Amber Parks and Sara Jhanjar for all their help with segmenting CT scans.</div></section><div class="Appendices"><section id="appsec1"><h2 id="sectitle0110" class="u-h4 u-margin-l-top u-margin-xs-bottom">Appendix A. Supplementary Online Material</h2><span class="download-all-supplemental-data"><span class="article-attachment"><iframe title="download_all" name="download_all" class="u-hide"></iframe><form method="post" action="/sdfe/pdf/download/S0047248421001019/attachments" target="download_all"><input name="eids" type="hidden" value="1-s2.0-S0047248421001019-mmc1.pdf"><input name="eids" type="hidden" value="1-s2.0-S0047248421001019-mmc2.zip"><input name="eids" type="hidden" value="1-s2.0-S0047248421001019-mmc3.zip"><button class="button-link download-all-button u-font-sans button-link-primary button-link-icon-left" type="submit"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-download"><path d="M77.38 56.18l-6.6-7.06L54 66.36V26H44v40.34L26.28 49.1l-7.3 7.08 29.2 29.32 29.2-29.32M88 74v20H10V74H0v30h98V74H88"></path></svg><span class="button-link-text-container"><span class="button-link-text"><span class="download-all-title">Download all </span>supplementary files<span class="u-show-inline-from-sm"> included with this article</span></span></span></button></form><a class="anchor help-link u-font-sans u-margin-s-left anchor-primary anchor-has-colored-icon anchor-icon-left anchor-with-icon" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank" title="What’s this? (Opens in new window)"><svg focusable="false" viewBox="0 0 114 128" height="20" class="icon icon-help"><path d="M57 8C35.69 7.69 15.11 21.17 6.68 40.71c-8.81 19.38-4.91 43.67 9.63 59.25 13.81 15.59 36.85 21.93 56.71 15.68 21.49-6.26 37.84-26.81 38.88-49.21 1.59-21.15-10.47-42.41-29.29-52.1C74.76 10.17 65.88 7.99 57 8zm0 10c20.38-.37 39.57 14.94 43.85 34.85 4.59 18.53-4.25 39.23-20.76 48.79-17.05 10.59-40.96 7.62-54.9-6.83-14.45-13.94-17.42-37.85-6.83-54.9C26.28 26.5 41.39 17.83 57 18zm-.14 14C45.31 32.26 40 40.43 40 50v2h10v-2c0-4.22 2.22-9.66 8-9.24 5.5.4 6.32 5.14 5.78 8.14C62.68 55.06 52 58.4 52 69.4V76h10v-5.56c0-8.16 11.22-11.52 12-21.7.74-9.86-5.56-16.52-16-16.74-.39-.01-.76-.01-1.14 0zM52 82v10h10V82H52z"></path></svg><span class="anchor-text-container"><span class="anchor-text">What’s this?</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></span></span><div class="u-margin-s-bottom" id="p0215">The following are the Supplementary online material to this article:<span class="display"><span class="e-component e-component-mmc1" id="mmc1"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-mmc1.pdf" target="_blank" download="" title="Download Acrobat PDF file (12KB)"><svg focusable="false" viewBox="0 0 95 128" height="20" width="20" class="icon icon-pdf-download"><path d="M82 108H12V59c0-6.08 4.92-11 11-11h17V28h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06H82v40h10V10H35.6c-5.38 0-10.62 1.92-14.76 5.4C11.74 23.08 2 35.54 2 47.5V118h90v-18H82v8zM56.06 68.6c-.84-.38-1.84-.56-2.98-.56h-9.04v23.6h5.94v-9h2.18c2.48 0 4.36-.62 5.66-1.84s1.92-3.06 1.92-5.5c0-1.04-.12-2-.4-2.88-.26-.88-.66-1.64-1.22-2.3s-1.22-1.14-2.06-1.52zm-3.12 8.94c-.4.46-.98.7-1.74.7h-1.22v-5.76h1.22c1.56 0 2.34.96 2.34 2.88 0 .98-.2 1.72-.6 2.18zm21.84-8.52c-.96-.66-2.32-.98-4.06-.98H62v23.6h8.72c1.74 0 3.1-.32 4.06-.98s1.7-1.52 2.18-2.62.78-2.34.88-3.76.16-2.9.16-4.44-.06-3.02-.16-4.44-.4-2.68-.88-3.76a5.94 5.94 0 0 0-2.18-2.62zM72 83.68c-.06.96-.18 1.72-.38 2.24s-.48.88-.84 1.04-.86.24-1.48.24h-1.32V72.46h1.32c.62 0 1.12.08 1.48.24s.64.52.84 1.04.32 1.28.38 2.24c.06.98.08 2.24.08 3.84s-.02 2.9-.08 3.86zm13.98-6.58v-4.02h7.74v-5.04h-13.7v23.6h5.96v-9.72h7.26V77.1z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Acrobat PDF file (12KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Multimedia component 1</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc2" id="mmc2"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-mmc2.zip" target="_blank" download="" title="Download zip file (8KB)"><svg focusable="false" viewBox="0 0 95 128" height="20" width="20" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download zip file (8KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Multimedia component 2</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc3" id="mmc3"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S0047248421001019-mmc3.zip" target="_blank" download="" title="Download zip file (1KB)"><svg focusable="false" viewBox="0 0 95 128" height="20" width="20" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download zip file (1KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Multimedia component 3</span>. </p></span></span></span></span></div></section></div></div><div class="related-content-links u-display-none-from-md"><button class="button-link button-link-primary button-link-small" type="button"><span class="button-link-text-container"><span class="button-link-text">Recommended articles</span></span></button></div><div class="Tail"></div><div><section class="bibliography u-font-serif text-s" id="cebib0010"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">References</h2><section class="bibliography-sec" id="cebibsec0010"><ol class="references" id="reference-links-cebibsec0010"><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib1" id="ref-id-bib1" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref1"><div class="contribution"><div class="authors u-font-sans">R.R. Ackermann, M.L. Arnold, M.D. Baiz, J.A. Cahill, L. Cortés-Ortiz, B.J. Evans, B.R. Grant, P.R. Grant, B. Hallgrimsson, R.A. Humphreys, C.J. Jolly, J. Malukiewicz, C.J. Percival, T.B. Ritzman, C. Roos, C.C. Roseman, L. Schroeder, F.H. Smith, K.A. Warren, R.K. Wayne, D. Zinner</div><div id="ref-id-sref1" class="title text-m">Hybridization in human evolution: insights from other organisms</div></div><div class="host u-font-sans">Evol. Anthropol., 28 (2019), pp. 189-209</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/evan.21787" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85067880952&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Hybridization%20in%20human%20evolution%3A%20insights%20from%20other%20organisms&amp;publication_year=2019&amp;author=R.R.%20Ackermann&amp;author=M.L.%20Arnold&amp;author=M.D.%20Baiz&amp;author=J.A.%20Cahill&amp;author=L.%20Cort%C3%A9s-Ortiz&amp;author=B.J.%20Evans&amp;author=B.R.%20Grant&amp;author=P.R.%20Grant&amp;author=B.%20Hallgrimsson&amp;author=R.A.%20Humphreys&amp;author=C.J.%20Jolly&amp;author=J.%20Malukiewicz&amp;author=C.J.%20Percival&amp;author=T.B.%20Ritzman&amp;author=C.%20Roos&amp;author=C.C.%20Roseman&amp;author=L.%20Schroeder&amp;author=F.H.%20Smith&amp;author=K.A.%20Warren&amp;author=R.K.%20Wayne&amp;author=D.%20Zinner" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib2" id="ref-id-bib2" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ackermann and Bishop, 2010</span></span></a></span><span class="reference" id="sref2"><div class="contribution"><div class="authors u-font-sans">R.R. Ackermann, J.M. Bishop</div><div id="ref-id-sref2" class="title text-m">Morphological and molecular evidence reveals recent hybridization between gorilla taxa</div></div><div class="host u-font-sans">Evolution, 64 (2010), pp. 271-290</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/j.1558-5646.2009.00858.x" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref2"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-73949083194&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref2"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Morphological%20and%20molecular%20evidence%20reveals%20recent%20hybridization%20between%20gorilla%20taxa&amp;publication_year=2010&amp;author=R.R.%20Ackermann&amp;author=J.M.%20Bishop" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref2"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib3" id="ref-id-bib3" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ackermann and Cheverud, 2004</span></span></a></span><span class="reference" id="sref3"><div class="contribution"><div class="authors u-font-sans">R.R. Ackermann, J.M. Cheverud</div><div id="ref-id-sref3" class="title text-m">Detecting genetic drift versus selection in human evolution</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 101 (2004), pp. 17946-17951</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.0405919102" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-11144240291&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Detecting%20genetic%20drift%20versus%20selection%20in%20human%20evolution&amp;publication_year=2004&amp;author=R.R.%20Ackermann&amp;author=J.M.%20Cheverud" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib4" id="ref-id-bib4" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2016</span></span></a></span><span class="reference" id="sref4"><div class="contribution"><div class="authors u-font-sans">R.R. Ackermann, A. Mackay, M.L. Arnold</div><div id="ref-id-sref4" class="title text-m">The hybrid origin of “Modern” Humans</div></div><div class="host u-font-sans">Evol. Biol., 43 (2016), pp. 1-11</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s11692-015-9348-1" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84958755370&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20hybrid%20origin%20of%20Modern%20Humans&amp;publication_year=2016&amp;author=R.R.%20Ackermann&amp;author=A.%20Mackay&amp;author=M.L.%20Arnold" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib5" id="ref-id-bib5" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2006</span></span></a></span><span class="reference" id="sref5"><div class="contribution"><div class="authors u-font-sans">R.R. Ackermann, J. Rogers, J.M. Cheverud</div><div id="ref-id-sref5" class="title text-m">Identifying the morphological signatures of hybridization in primate and human evolution</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 51 (2006), pp. 632-645</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248406001485/pdfft?md5=9cabbfe0b629582b0c56f0bcd59bf4f8&amp;pid=1-s2.0-S0047248406001485-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref5"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248406001485" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33751109947&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Identifying%20the%20morphological%20signatures%20of%20hybridization%20in%20primate%20and%20human%20evolution&amp;publication_year=2006&amp;author=R.R.%20Ackermann&amp;author=J.%20Rogers&amp;author=J.M.%20Cheverud" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib6" id="ref-id-bib6" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ackermann et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref6"><div class="contribution"><div class="authors u-font-sans">R.R. Ackermann, L. Schroeder, J. Rogers, J.M. Cheverud</div><div id="ref-id-sref6" class="title text-m">Further evidence for phenotypic signatures of hybridization in descendent baboon populations</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 76 (2014), pp. 54-62</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248414001109/pdfft?md5=141b7bf726e55041755004f3d29f8c52&amp;pid=1-s2.0-S0047248414001109-mainext.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref6"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248414001109" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84927692636&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Further%20evidence%20for%20phenotypic%20signatures%20of%20hybridization%20in%20descendent%20baboon%20populations&amp;publication_year=2014&amp;author=R.R.%20Ackermann&amp;author=L.%20Schroeder&amp;author=J.%20Rogers&amp;author=J.M.%20Cheverud" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib7" id="ref-id-bib7" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Adams et&nbsp;al., 2020</span></span></a></span><span class="reference" id="sref7"><div class="contribution"><div class="authors u-font-sans">D.C. Adams, M.L. Collyer, A. Kaliontzopoulou</div><div id="ref-id-sref7" class="title text-m">Geomorph: Software for geometric morphometric analyses</div></div><div class="host u-font-sans"> (2020)</div><div class="host u-font-sans"><a class="anchor anchor-primary" href="https://cran.r-project.org/package=geomorph" target="_blank"><span class="anchor-text-container"><span class="anchor-text">https://cran.r-project.org/package=geomorph</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, Ver. 3.2.1</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Geomorph%3A%20Software%20for%20geometric%20morphometric%20analyses&amp;publication_year=2020&amp;author=D.C.%20Adams&amp;author=M.L.%20Collyer&amp;author=A.%20Kaliontzopoulou" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref7"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib8" id="ref-id-bib8" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Allen et&nbsp;al., 2020</span></span></a></span><span class="reference" id="sref8"><div class="contribution"><div class="authors u-font-sans">R. Allen, H. Ryan, B.W. Davis, C. King, L. Frantz, E. Irving-pease, R. Barnett, A. Linderholm, L. Loog, J. Haile, O. Lebrasseur, M. White, A.C. Kitchener, W.J. Murphy, G. Larson, W.J. Murphy</div><div id="ref-id-sref8" class="title text-m">A&nbsp;mitochondrial genetic divergence proxy predicts the reproductive compatibility of mammalian hybrids</div></div><div class="host u-font-sans">Proc. Royal Soc. B Biol. Sci., 287 (2020), p. 20200690</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rspb.2020.0690" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85085908545&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0mitochondrial%20genetic%20divergence%20proxy%20predicts%20the%20reproductive%20compatibility%20of%20mammalian%20hybrids&amp;publication_year=2020&amp;author=R.%20Allen&amp;author=H.%20Ryan&amp;author=B.W.%20Davis&amp;author=C.%20King&amp;author=L.%20Frantz&amp;author=E.%20Irving-pease&amp;author=R.%20Barnett&amp;author=A.%20Linderholm&amp;author=L.%20Loog&amp;author=J.%20Haile&amp;author=O.%20Lebrasseur&amp;author=M.%20White&amp;author=A.C.%20Kitchener&amp;author=W.J.%20Murphy&amp;author=G.%20Larson&amp;author=W.J.%20Murphy" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib9" id="ref-id-bib9" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bergmann, 1847</span></span></a></span><span class="reference" id="sref9"><div class="contribution"><div class="authors u-font-sans">C.A. Bergmann</div></div><div class="host u-font-sans" id="ref-id-sref9">Uber die Verhaltnisee der Warmeokonomie der Thiere zu iher Grosse, vol. 1, Gottingen Studien (1847), pp. 595-708</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar?q=Bergmann%2C%20C.A.%2C%201847.%20Uber%20die%20verhaltnisee%20der%20warmeokonomie%20der%20thiere%20zu%20iher%20grosse.%20Gottingen%20Studien%201%2C%20595-708." target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref9"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib10" id="ref-id-bib10" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Betti and Manica, 2018</span></span></a></span><span class="reference" id="sref10"><div class="contribution"><div class="authors u-font-sans">L. Betti, A. Manica</div><div id="ref-id-sref10" class="title text-m">Human variation in the shape of the birth canal is significant and geographically structured</div></div><div class="host u-font-sans">Proc. Royal Soc. B Biol. Sci., 285 (2018), p. 20181807</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rspb.2018.1807" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85055600702&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Human%20variation%20in%20the%20shape%20of%20the%20birth%20canal%20is%20significant%20and%20geographically%20structured&amp;publication_year=2018&amp;author=L.%20Betti&amp;author=A.%20Manica" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib11" id="ref-id-bib11" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Betti et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref11"><div class="contribution"><div class="authors u-font-sans">L. Betti, N. von Cramon-Taubadel, A. Manica, S.J. Lycett</div><div id="ref-id-sref11" class="title text-m">The interaction of neutral evolutionary processes with climatically-driven adaptive changes in the 3D shape of the human os coxae</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 73 (2014), pp. 64-74</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248414001195/pdfft?md5=cd1dd8da9aff7a466b104b96ba6be7e7&amp;pid=1-s2.0-S0047248414001195-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref11"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248414001195" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84905754659&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20interaction%20of%20neutral%20evolutionary%20processes%20with%20climatically-driven%20adaptive%20changes%20in%20the%203D%20shape%20of%20the%20human%20os%20coxae&amp;publication_year=2014&amp;author=L.%20Betti&amp;author=N.%20von%20Cramon-Taubadel&amp;author=A.%20Manica&amp;author=S.J.%20Lycett" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib12" id="ref-id-bib12" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Bocquet-Appel and Degioanni, 2013</span></span></a></span><span class="reference" id="sref12"><div class="contribution"><div class="authors u-font-sans">J.P. Bocquet-Appel, A. Degioanni</div><div id="ref-id-sref12" class="title text-m">Neanderthal demographic estimates</div></div><div class="host u-font-sans">Curr. Anthropol., 54 (2013), pp. S202-S213</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1086/673725" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84893250945&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Neanderthal%20demographic%20estimates&amp;publication_year=2013&amp;author=J.P.%20Bocquet-Appel&amp;author=A.%20Degioanni" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib13" id="ref-id-bib13" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Boel et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref13"><div class="contribution"><div class="authors u-font-sans">C. Boel, D. Curnoe, Y. Hamada</div><div id="ref-id-sref13" class="title text-m">Craniofacial shape and nonmetric trait variation in hybrids of the Japanese macaque (<em>Macaca fuscata</em>) and the Taiwanese macaque (<em>Macaca cyclopis</em>)</div></div><div class="host u-font-sans">Int. J. Primatol., 40 (2019), pp. 214-243</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s10764-019-00081-2" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85062967484&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Craniofacial%20shape%20and%20nonmetric%20trait%20variation%20in%20hybrids%20of%20the%20Japanese%20macaque%20%20and%20the%20Taiwanese%20macaque%20&amp;publication_year=2019&amp;author=C.%20Boel&amp;author=D.%20Curnoe&amp;author=Y.%20Hamada" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib14" id="ref-id-bib14" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Buck et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref14"><div class="contribution"><div class="authors u-font-sans">L.T. Buck, I. De Groote, Y. Hamada, B.R. Hassett, T. Ito, J.T. Stock</div><div id="ref-id-sref14" class="title text-m">Evidence of different climatic adaptation strategies in humans and non-human primates</div></div><div class="host u-font-sans">Sci. Rep., 9 (2019), p. 11025</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85069946467&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref14"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20different%20climatic%20adaptation%20strategies%20in%20humans%20and%20non-human%20primates&amp;publication_year=2019&amp;author=L.T.%20Buck&amp;author=I.%20De%20Groote&amp;author=Y.%20Hamada&amp;author=B.R.%20Hassett&amp;author=T.%20Ito&amp;author=J.T.%20Stock" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref14"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib15" id="ref-id-bib15" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Buck et&nbsp;al., 2010</span></span></a></span><span class="reference" id="sref15"><div class="contribution"><div class="authors u-font-sans">L.T. Buck, J.T. Stock, R.a. Foley</div><div id="ref-id-sref15" class="title text-m">Levels of intraspecific variation within the catarrhine skeleton</div></div><div class="host u-font-sans">Int. J. Primatol., 31 (2010), pp. 779-795</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s10764-010-9428-0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-77957151217&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Levels%20of%20intraspecific%20variation%20within%20the%20catarrhine%20skeleton&amp;publication_year=2010&amp;author=L.T.%20Buck&amp;author=J.T.%20Stock&amp;author=R.a.%20Foley" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib16" id="ref-id-bib16" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Chu et&nbsp;al., 2007</span></span></a></span><span class="reference" id="sref16"><div class="contribution"><div class="authors u-font-sans">J.H. Chu, Y.S. Lin, H.Y. Wu</div><div id="ref-id-sref16" class="title text-m">Evolution and dispersal of three closely related macaque species, <em>Macaca mulatta, M. cyclopis</em>, and <em>M.&nbsp;fuscata</em>, in the eastern Asia</div></div><div class="host u-font-sans">Mol. Phylogenet. Evol., 43 (2007), pp. 418-429</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S1055790306004787/pdfft?md5=ef496947ce379d5ab397a0d7a97fce91&amp;pid=1-s2.0-S1055790306004787-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref16"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S1055790306004787" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-34247594848&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20dispersal%20of%20three%20closely%20related%20macaque%20species%2C%20Macaca%20mulatta%2C%20M.%20cyclopis%2C%20and%20M.%C2%A0fuscata%2C%20in%20the%20eastern%20Asia&amp;publication_year=2007&amp;author=J.H.%20Chu&amp;author=Y.S.%20Lin&amp;author=H.Y.%20Wu" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib17" id="ref-id-bib17" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Clarke and O'Neil, 1999</span></span></a></span><span class="reference" id="sref17"><div class="contribution"><div class="authors u-font-sans">M.R. Clarke, J.A.S. O'Neil</div><div id="ref-id-sref17" class="title text-m">Morphometric comparison of Chinese-origin and Indian-derived rhesus monkeys (<em>Macaca mulatta</em>)</div></div><div class="host u-font-sans">Am. J. Primatol., 74 (1999), pp. 335-346</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0032896282&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref17"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Morphometric%20comparison%20of%20Chinese-origin%20and%20Indian-derived%20rhesus%20monkeys%20&amp;publication_year=1999&amp;author=M.R.%20Clarke&amp;author=J.A.S.%20O'Neil" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref17"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib18" id="ref-id-bib18" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Collyer and Adams, 2018</span></span></a></span><span class="reference" id="sref18"><div class="contribution"><div class="authors u-font-sans">M.L. Collyer, D.C. Adams</div><div id="ref-id-sref18" class="title text-m">RRPP: An R package for fitting linear models to high-dimensional data using residual randomization</div></div><div class="host u-font-sans">Methods Ecol. Evol., 9 (2018), pp. 1772-1779</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/2041-210x.13029" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85047666760&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=RRPP%3A%20An%20R%20package%20for%20fitting%20linear%20models%20to%20high-dimensional%20data%20using%20residual%20randomization&amp;publication_year=2018&amp;author=M.L.%20Collyer&amp;author=D.C.%20Adams" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib19" id="ref-id-bib19" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Collyer and Adams, 2020</span></span></a></span><span class="reference" id="sref19"><div class="contribution"><div class="authors u-font-sans">M.L. Collyer, D.C. Adams</div><div id="ref-id-sref19" class="title text-m">RRPP: Linear model evaluation and randomized residuals in a permutation procedure</div></div><div class="host u-font-sans"> (2020)</div><div class="host u-font-sans"><a class="anchor anchor-primary" href="https://cran.r-project.org/package=RRPP" target="_blank"><span class="anchor-text-container"><span class="anchor-text">https://cran.r-project.org/package=RRPP</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a>, Ver. 0.5.2</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=RRPP%3A%20Linear%20model%20evaluation%20and%20randomized%20residuals%20in%20a%20permutation%20procedure&amp;publication_year=2020&amp;author=M.L.%20Collyer&amp;author=D.C.%20Adams" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib20" id="ref-id-bib20" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Comeault and Matute, 2018</span></span></a></span><span class="reference" id="sref20"><div class="contribution"><div class="authors u-font-sans">A.A. Comeault, D.R. Matute</div><div id="ref-id-sref20" class="title text-m">Genetic divergence and the number of hybridizing species affect the path to homoploid hybrid speciation</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 115 (2018), pp. 9761-9766</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1809685115" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85054023774&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Genetic%20divergence%20and%20the%20number%20of%20hybridizing%20species%20affect%20the%20path%20to%20homoploid%20hybrid%20speciation&amp;publication_year=2018&amp;author=A.A.%20Comeault&amp;author=D.R.%20Matute" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib22" id="ref-id-bib22" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Cortés-Ortiz et&nbsp;al., 2003</span></span></a></span><span class="reference" id="sref22"><div class="contribution"><div class="authors u-font-sans">L. Cortés-Ortiz, E. Bermingham, C. Rico, E. Rodríguez-Luna, I. Sampaio, M. Ruiz-García</div><div id="ref-id-sref22" class="title text-m">Molecular systematics and biogeography of the Neotropical monkey genus, <em>Alouatta</em></div></div><div class="host u-font-sans">Mol. Phylogenet. Evol., 26 (2003), pp. 64-81</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S1055790302003081/pdfft?md5=13282b1daddf5ec0ccb6da5c206ee941&amp;pid=1-s2.0-S1055790302003081-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref22"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S1055790302003081" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0346525130&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Molecular%20systematics%20and%20biogeography%20of%20the%20Neotropical%20monkey%20genus%2C%20Alouatta&amp;publication_year=2003&amp;author=L.%20Cort%C3%A9s-Ortiz&amp;author=E.%20Bermingham&amp;author=C.%20Rico&amp;author=E.%20Rodr%C3%ADguez-Luna&amp;author=I.%20Sampaio&amp;author=M.%20Ruiz-Garc%C3%ADa" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib23" id="ref-id-bib23" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Cortés-Ortiz et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref23"><div class="contribution"><div class="authors u-font-sans">L. Cortés-Ortiz, C. Roos, D. Zinner</div><div id="ref-id-sref23" class="title text-m">Introduction to special issue on primate hybridization and hybrid zones</div></div><div class="host u-font-sans">Int. J. Primatol., 40 (2019), pp. 1-8</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s10764-019-00076-z" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref23"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Introduction%20to%20special%20issue%20on%20primate%20hybridization%20and%20hybrid%20zones&amp;publication_year=2019&amp;author=L.%20Cort%C3%A9s-Ortiz&amp;author=C.%20Roos&amp;author=D.%20Zinner" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref23"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib24" id="ref-id-bib24" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Drake and Klingenberg, 2008</span></span></a></span><span class="reference" id="sref24"><div class="contribution"><div class="authors u-font-sans">A.G. Drake, C.P. Klingenberg</div><div id="ref-id-sref24" class="title text-m">The pace of morphological change: Historical transformation of skull shape in St. Bernard dogs</div></div><div class="host u-font-sans">Proc. Royal Soc. B Biol. Sci., 275 (2008), pp. 71-76</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rspb.2007.1169" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-36349010900&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20pace%20of%20morphological%20change%3A%20Historical%20transformation%20of%20skull%20shape%20in%20St.%20Bernard%20dogs&amp;publication_year=2008&amp;author=A.G.%20Drake&amp;author=C.P.%20Klingenberg" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib25" id="ref-id-bib25" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Duarte et&nbsp;al., 1999</span></span></a></span><span class="reference" id="sref25"><div class="contribution"><div class="authors u-font-sans">C. Duarte, J. Maurício, P.B. Pettitt, P. Souto, E. Trinkaus, H. Van Der Plicht, J. Zilhão</div><div id="ref-id-sref25" class="title text-m">The early Upper Paleolithic human skeleton from the Abrigo do Lagar Velho (Portugal) and modern human emergence in Iberia</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 96 (1999), pp. 7604-7609</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033594901&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref25"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20early%20Upper%20Paleolithic%20human%20skeleton%20from%20the%20Abrigo%20do%20Lagar%20Velho%20%20and%20modern%20human%20emergence%20in%20Iberia&amp;publication_year=1999&amp;author=C.%20Duarte&amp;author=J.%20Maur%C3%ADcio&amp;author=P.B.%20Pettitt&amp;author=P.%20Souto&amp;author=E.%20Trinkaus&amp;author=H.%20Van%20Der%20Plicht&amp;author=J.%20Zilh%C3%A3o" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref25"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib26" id="ref-id-bib26" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ferguson et&nbsp;al., 2007</span></span></a></span><span class="reference" id="sref26"><div class="contribution"><div class="authors u-font-sans">B. Ferguson, S.L. Street, H. Wright, C. Pearson, Y. Jia, S.L. Thompson, P. Allibone, C.J. Dubay, E. Spindel, R.B. Norgren</div><div id="ref-id-sref26" class="title text-m">Single nucleotide polymorphisms (SNPs) distinguish Indian-origin and Chinese-origin rhesus macaques (<em>Macaca mulatta</em>)</div></div><div class="host u-font-sans">BMC Genom., 8 (2007), p. 43</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33847652196&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref26"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphisms%20%20distinguish%20Indian-origin%20and%20Chinese-origin%20rhesus%20macaques%20&amp;publication_year=2007&amp;author=B.%20Ferguson&amp;author=S.L.%20Street&amp;author=H.%20Wright&amp;author=C.%20Pearson&amp;author=Y.%20Jia&amp;author=S.L.%20Thompson&amp;author=P.%20Allibone&amp;author=C.J.%20Dubay&amp;author=E.%20Spindel&amp;author=R.B.%20Norgren" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref26"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib27" id="ref-id-bib27" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fischer et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref27"><div class="contribution"><div class="authors u-font-sans">J. Fischer, J.P. Higham, S.C. Alberts, L. Barrett, J.C. Beehner, T.J. Bergman, A.J. Carter, A. Collins, S. Elton, J. Fagot, M.J. Ferreira da Silva, K. Hammerschmidt, P. Henzi, C.J. Jolly, S. Knauf, G.H. Kopp, J. Rogers, C. Roos, C. Ross, R.M. Seyfarth, J. Silk, N. Snyder-Mackler, V. Staedele, L. Swedell, M.L. Wilson, D. Zinner</div><div id="ref-id-sref27" class="title text-m">The natural history of model organisms: Insights into the evolution of social systems and species from baboon studies</div></div><div class="host u-font-sans">eLife, 8 (2019), Article e50989</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85074815713&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref27"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20natural%20history%20of%20model%20organisms%3A%20Insights%20into%20the%20evolution%20of%20social%20systems%20and%20species%20from%20baboon%20studies&amp;publication_year=2019&amp;author=J.%20Fischer&amp;author=J.P.%20Higham&amp;author=S.C.%20Alberts&amp;author=L.%20Barrett&amp;author=J.C.%20Beehner&amp;author=T.J.%20Bergman&amp;author=A.J.%20Carter&amp;author=A.%20Collins&amp;author=S.%20Elton&amp;author=J.%20Fagot&amp;author=M.J.%20Ferreira%20da%20Silva&amp;author=K.%20Hammerschmidt&amp;author=P.%20Henzi&amp;author=C.J.%20Jolly&amp;author=S.%20Knauf&amp;author=G.H.%20Kopp&amp;author=J.%20Rogers&amp;author=C.%20Roos&amp;author=C.%20Ross&amp;author=R.M.%20Seyfarth&amp;author=J.%20Silk&amp;author=N.%20Snyder-Mackler&amp;author=V.%20Staedele&amp;author=L.%20Swedell&amp;author=M.L.%20Wilson&amp;author=D.%20Zinner" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref27"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib28" id="ref-id-bib28" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1976</span></span></a></span><span class="reference" id="sref28"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref28" class="title text-m">Provisional classification and key to living species of macaques (Primates: <em>Macaca</em>)</div></div><div class="host u-font-sans">Folia Primatol., 25 (1976), pp. 225-236</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1159/000155715" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref28"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0016908692&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref28"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Provisional%20classification%20and%20key%20to%20living%20species%20of%20macaques%20&amp;publication_year=1976&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref28"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib29" id="ref-id-bib29" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1982</span></span></a></span><span class="reference" id="sref29"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref29" class="title text-m">Ecogeographic segregation of macaque species</div></div><div class="host u-font-sans">Primates, 23 (1982), pp. 574-579</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0000670324&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref29"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Ecogeographic%20segregation%20of%20macaque%20species&amp;publication_year=1982&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref29"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib30" id="ref-id-bib30" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 1995</span></span></a></span><span class="reference" id="sref30"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref30" class="title text-m">Systematic review of Southeast Asian longtail macaques, <em>Macaca fascicularis</em> (Raffles, 1821)</div></div><div class="host u-font-sans">Fieldiana Zool., 81 (1995), pp. 2-3</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Systematic%20review%20of%20Southeast%20Asian%20longtail%20macaques%2C%20Macaca%20fascicularis%20&amp;publication_year=1995&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib31" id="ref-id-bib31" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2000</span></span></a></span><span class="reference" id="sref31"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref31" class="title text-m">Zoology Systematic Review of the Rhesus Macaque, <em>Macaca mulatta</em> (Zimmermann, 1780)</div></div><div class="host u-font-sans">Field Museum of Natural History, Chicago (2000)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Zoology%20Systematic%20Review%20of%20the%20Rhesus%20Macaque%2C%20Macaca%20mulatta%20&amp;publication_year=2000&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref31"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib32" id="ref-id-bib32" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fooden, 2006</span></span></a></span><span class="reference" id="sref32"><div class="contribution"><div class="authors u-font-sans">J. Fooden</div><div id="ref-id-sref32" class="title text-m">Comparative review of <em>fascicularis</em>-group species of macaques (Primates: <em>Macaca</em>)</div></div><div class="host u-font-sans">Fieldiana Zool., 107 (2006), pp. 1-43</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Comparative%20review%20of%20fascicularis-group%20species%20of%20macaques%20&amp;publication_year=2006&amp;author=J.%20Fooden" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref32"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib33" id="ref-id-bib33" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fu et&nbsp;al., 2015</span></span></a></span><span class="reference" id="sref33"><div class="contribution"><div class="authors u-font-sans">Q. Fu, M. Hajdinjak, O.T. Moldovan, S. Constantin, S. Mallick, P. Skoglund, N. Patterson, N. Rohland, I. Lazaridis, B. Nickel, B. Viola, K. Prüfer, M. Meyer, J. Kelso, D. Reich, S. Pääbo</div><div id="ref-id-sref33" class="title text-m">An early modern human from Romania with a recent Neanderthal ancestor</div></div><div class="host u-font-sans">Nature, 524 (2015), pp. 216-219</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature14558" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84939496491&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=An%20early%20modern%20human%20from%20Romania%20with%20a%20recent%20Neanderthal%20ancestor&amp;publication_year=2015&amp;author=Q.%20Fu&amp;author=M.%20Hajdinjak&amp;author=O.T.%20Moldovan&amp;author=S.%20Constantin&amp;author=S.%20Mallick&amp;author=P.%20Skoglund&amp;author=N.%20Patterson&amp;author=N.%20Rohland&amp;author=I.%20Lazaridis&amp;author=B.%20Nickel&amp;author=B.%20Viola&amp;author=K.%20Pr%C3%BCfer&amp;author=M.%20Meyer&amp;author=J.%20Kelso&amp;author=D.%20Reich&amp;author=S.%20P%C3%A4%C3%A4bo" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib34" id="ref-id-bib34" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Fuzessy et&nbsp;al., 2014</span></span></a></span><span class="reference" id="sref34"><div class="contribution"><div class="authors u-font-sans">L.F. Fuzessy, I. de O. Silva, J. Malukiewicz, F.F.R. Silva, M. do C. Pônzio, V. Boere, R.R. Ackermann</div><div id="ref-id-sref34" class="title text-m">Morphological variation in wild marmosets (<em>Callithrix penicillata</em> and <em>C.&nbsp;geoffroyi</em>) and their hybrids</div></div><div class="host u-font-sans">Evol. Biol., 41 (2014), pp. 480-493</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s11692-014-9284-5" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84906069430&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Morphological%20variation%20in%20wild%20marmosets%20%20and%20their%20hybrids&amp;publication_year=2014&amp;author=L.F.%20Fuzessy&amp;author=I.%20de%20O.%20Silva&amp;author=J.%20Malukiewicz&amp;author=F.F.R.%20Silva&amp;author=M.%20do%20C.%20P%C3%B4nzio&amp;author=V.%20Boere&amp;author=R.R.%20Ackermann" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib35" id="ref-id-bib35" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Gopalan et&nbsp;al., 2021</span></span></a></span><span class="reference" id="sref35"><div class="contribution"><div class="authors u-font-sans">S. Gopalan, E.G. Atkinson, L.T. Buck, T.D. Weaver, B.M. Henn</div><div id="ref-id-sref35" class="title text-m">Inferring archaic introgression from hominin genetic data</div></div><div class="host u-font-sans">Evol. Anthropol., 30 (2021), pp. 199-220</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/evan.21895" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref35"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85105037913&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref35"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Inferring%20archaic%20introgression%20from%20hominin%20genetic%20data&amp;publication_year=2021&amp;author=S.%20Gopalan&amp;author=E.G.%20Atkinson&amp;author=L.T.%20Buck&amp;author=T.D.%20Weaver&amp;author=B.M.%20Henn" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref35"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib36" id="ref-id-bib36" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Grabowski et&nbsp;al., 2011</span></span></a></span><span class="reference" id="sref36"><div class="contribution"><div class="authors u-font-sans">M.W. Grabowski, J.D. Polk, C.C. Roseman</div><div id="ref-id-sref36" class="title text-m">Divergent patterns of integration and reduced constraint in the human hip and the origins of bipedalism</div></div><div class="host u-font-sans">Evolution, 65 (2011), pp. 1336-1356</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/j.1558-5646.2011.01226.x" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-79955137569&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Divergent%20patterns%20of%20integration%20and%20reduced%20constraint%20in%20the%20human%20hip%20and%20the%20origins%20of%20bipedalism&amp;publication_year=2011&amp;author=M.W.%20Grabowski&amp;author=J.D.%20Polk&amp;author=C.C.%20Roseman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib37" id="ref-id-bib37" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Green et&nbsp;al., 2010</span></span></a></span><span class="reference" id="sref37"><div class="contribution"><div class="authors u-font-sans">R.E. Green, J. Krause, A.W. Briggs, T. Maricic, U. Stenzel, M. Kircher, N. Patterson, H. Li, W. Zhai, M.H.-Y. Fritz, N.F. Hansen, E.Y. Durand, A.-S. Malaspinas, J.D. Jensen, T. Marques-Bonet, C. Alkan, K. Prüfer, M. Meyer, H.A. Burbano, J.M. Good, R. Schultz, A. Aximu-Petri, A. Butthof, B. Höber, B. Höffner, M. Siegemund, A. Weihmann, C. Nusbaum, E.S. Lander, C. Russ, N. Novod, J. Affourtit, M. Egholm, C. Verna, P. Rudan, D. Brajkovic, Z. Kucan, I. Gusic, V.B. Doronichev, L.V. Golovanova, C. Lalueza-Fox, M. de la Rasilla, J. Fortea, A. Rosas, R.W. Schmitz, P.L.F. Johnson, E.E. Eichler, D. Falush, E. Birney, J.C. Mullikin, M. Slatkin, R. Nielsen, J. Kelso, M. Lachmann, D. Reich, S. Pääbo</div><div id="ref-id-sref37" class="title text-m">A&nbsp;draft sequence of the Neandertal genome</div></div><div class="host u-font-sans">Science, 328 (2010), pp. 710-722</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1126/science.1188021" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref37"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-77952136530&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref37"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0draft%20sequence%20of%20the%20Neandertal%20genome&amp;publication_year=2010&amp;author=R.E.%20Green&amp;author=J.%20Krause&amp;author=A.W.%20Briggs&amp;author=T.%20Maricic&amp;author=U.%20Stenzel&amp;author=M.%20Kircher&amp;author=N.%20Patterson&amp;author=H.%20Li&amp;author=W.%20Zhai&amp;author=M.H.-Y.%20Fritz&amp;author=N.F.%20Hansen&amp;author=E.Y.%20Durand&amp;author=A.-S.%20Malaspinas&amp;author=J.D.%20Jensen&amp;author=T.%20Marques-Bonet&amp;author=C.%20Alkan&amp;author=K.%20Pr%C3%BCfer&amp;author=M.%20Meyer&amp;author=H.A.%20Burbano&amp;author=J.M.%20Good&amp;author=R.%20Schultz&amp;author=A.%20Aximu-Petri&amp;author=A.%20Butthof&amp;author=B.%20H%C3%B6ber&amp;author=B.%20H%C3%B6ffner&amp;author=M.%20Siegemund&amp;author=A.%20Weihmann&amp;author=C.%20Nusbaum&amp;author=E.S.%20Lander&amp;author=C.%20Russ&amp;author=N.%20Novod&amp;author=J.%20Affourtit&amp;author=M.%20Egholm&amp;author=C.%20Verna&amp;author=P.%20Rudan&amp;author=D.%20Brajkovic&amp;author=Z.%20Kucan&amp;author=I.%20Gusic&amp;author=V.B.%20Doronichev&amp;author=L.V.%20Golovanova&amp;author=C.%20Lalueza-Fox&amp;author=M.%20de%20la%20Rasilla&amp;author=J.%20Fortea&amp;author=A.%20Rosas&amp;author=R.W.%20Schmitz&amp;author=P.L.F.%20Johnson&amp;author=E.E.%20Eichler&amp;author=D.%20Falush&amp;author=E.%20Birney&amp;author=J.C.%20Mullikin&amp;author=M.%20Slatkin&amp;author=R.%20Nielsen&amp;author=J.%20Kelso&amp;author=M.%20Lachmann&amp;author=D.%20Reich&amp;author=S.%20P%C3%A4%C3%A4bo" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref37"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib38" id="ref-id-bib38" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Gunz et&nbsp;al., 2012</span></span></a></span><span class="reference" id="sref38"><div class="contribution"><div class="authors u-font-sans">P. Gunz, S. Neubauer, L. Golovanova, V. Doronichev, B. Maureille, J.J. Hublin</div><div id="ref-id-sref38" class="title text-m">A&nbsp;uniquely modern human pattern of endocranial development. Insights from a new cranial reconstruction of the Neandertal newborn from Mezmaiskaya</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 62 (2012), pp. 300-313</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248411002314/pdfft?md5=5993756915865e07a713722a0d18e829&amp;pid=1-s2.0-S0047248411002314-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref38"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248411002314" aria-describedby="ref-id-sref38"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84856549547&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref38"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0uniquely%20modern%20human%20pattern%20of%20endocranial%20development.%20Insights%20from%20a%20new%20cranial%20reconstruction%20of%20the%20Neandertal%20newborn%20from%20Mezmaiskaya&amp;publication_year=2012&amp;author=P.%20Gunz&amp;author=S.%20Neubauer&amp;author=L.%20Golovanova&amp;author=V.%20Doronichev&amp;author=B.%20Maureille&amp;author=J.J.%20Hublin" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref38"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib39" id="ref-id-bib39" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2005</span></span></a></span><span class="reference" id="sref39"><div class="contribution"><div class="authors u-font-sans">Y. Hamada, T. Watanabe, K. Chatani, S. Hayakawa, M. Iwamoto</div><div id="ref-id-sref39" class="title text-m">Morphometrical comparison between Indian- and Chinese-derived rhesus macaques (<em>Macaca mulatta</em>)</div></div><div class="host u-font-sans">Anthropol. Sci., 113 (2005), pp. 183-188</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1537/ase.03104" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref39"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-25644443048&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref39"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Morphometrical%20comparison%20between%20Indian-%20and%20Chinese-derived%20rhesus%20macaques%20&amp;publication_year=2005&amp;author=Y.%20Hamada&amp;author=T.%20Watanabe&amp;author=K.%20Chatani&amp;author=S.%20Hayakawa&amp;author=M.%20Iwamoto" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref39"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib40" id="ref-id-bib40" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hamada et&nbsp;al., 2012</span></span></a></span><span class="reference" id="sref40"><div class="contribution"><div class="authors u-font-sans">Y. Hamada, A. Yamamoto, Y. Kunimatsu, S. Tojima, T. Mouri, Y. Kawamoto</div><div id="ref-id-sref40" class="title text-m">Variability of tail length in hybrids of the Japanese macaque (<em>Macaca fuscata</em>) and the Taiwanese macaque (<em>Macaca cyclopis</em>)</div></div><div class="host u-font-sans">Primates, 53 (2012), pp. 397-411</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s10329-012-0317-3" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref40"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84867139964&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref40"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Variability%20of%20tail%20length%20in%20hybrids%20of%20the%20Japanese%20macaque%20%20and%20the%20Taiwanese%20macaque%20&amp;publication_year=2012&amp;author=Y.%20Hamada&amp;author=A.%20Yamamoto&amp;author=Y.%20Kunimatsu&amp;author=S.%20Tojima&amp;author=T.%20Mouri&amp;author=Y.%20Kawamoto" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref40"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib41" id="ref-id-bib41" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Harvati et&nbsp;al., 2004</span></span></a></span><span class="reference" id="sref41"><div class="contribution"><div class="authors u-font-sans">K. Harvati, S.R. Frost, K.P. McNulty</div><div id="ref-id-sref41" class="title text-m">Neanderthal taxonomy reconsidered: implications of 3D primate models of intra- and interspecific differences</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 101 (2004), pp. 1147-1152</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0842299590&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref41"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Neanderthal%20taxonomy%20reconsidered%3A%20implications%20of%203D%20primate%20models%20of%20intra-%20and%20interspecific%20differences&amp;publication_year=2004&amp;author=K.%20Harvati&amp;author=S.R.%20Frost&amp;author=K.P.%20McNulty" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref41"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib42" id="ref-id-bib42" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hayasaka et&nbsp;al., 1996</span></span></a></span><span class="reference" id="sref42"><div class="contribution"><div class="authors u-font-sans">K. Hayasaka, K. Fujii, S. Horai</div><div id="ref-id-sref42" class="title text-m">Molecular phylogeny of macaques: Implications of nucleotide sequences from an 896-base pair region of mitochondrial DNA</div></div><div class="host u-font-sans">Mol. Biol. Evol., 13 (1996), pp. 1044-1053</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1093/oxfordjournals.molbev.a025655" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref42"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0029816997&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref42"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogeny%20of%20macaques%3A%20Implications%20of%20nucleotide%20sequences%20from%20an%20896-base%20pair%20region%20of%20mitochondrial%20DNA&amp;publication_year=1996&amp;author=K.%20Hayasaka&amp;author=K.%20Fujii&amp;author=S.%20Horai" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref42"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib43" id="ref-id-bib43" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Hernandez et&nbsp;al., 2007</span></span></a></span><span class="reference" id="sref43"><div class="contribution"><div class="authors u-font-sans">R.D. Hernandez, M.J. Hubisz, D.A. Wheeler, D.G. Smith, B. Ferguson, J. Rogers, L. Nazareth, A. Indap, T. Bourquin, J. McPherson, D. Muzny, R. Gibbs, R. Nielsen, C.D. Bustamante</div><div id="ref-id-sref43" class="title text-m">Demographic histories and patterns of linkage disequilibrium in Chinese and Indian rhesus macaques</div></div><div class="host u-font-sans">Science, 316 (2007), pp. 240-243</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1126/science.1140462" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref43"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-34247323510&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref43"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Demographic%20histories%20and%20patterns%20of%20linkage%20disequilibrium%20in%20Chinese%20and%20Indian%20rhesus%20macaques&amp;publication_year=2007&amp;author=R.D.%20Hernandez&amp;author=M.J.%20Hubisz&amp;author=D.A.%20Wheeler&amp;author=D.G.%20Smith&amp;author=B.%20Ferguson&amp;author=J.%20Rogers&amp;author=L.%20Nazareth&amp;author=A.%20Indap&amp;author=T.%20Bourquin&amp;author=J.%20McPherson&amp;author=D.%20Muzny&amp;author=R.%20Gibbs&amp;author=R.%20Nielsen&amp;author=C.D.%20Bustamante" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref43"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib44" id="ref-id-bib44" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Holliday, 1997a</span></span></a></span><span class="reference" id="sref44"><div class="contribution"><div class="authors u-font-sans">T.W. Holliday</div><div id="ref-id-sref44" class="title text-m">Postcranial evidence of cold adaptation in European Neandertals</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 104 (1997), pp. 245-258</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0030688020&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref44"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Postcranial%20evidence%20of%20cold%20adaptation%20in%20European%20Neandertals&amp;publication_year=1997&amp;author=T.W.%20Holliday" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref44"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib45" id="ref-id-bib45" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Holliday, 1997b</span></span></a></span><span class="reference" id="sref45"><div class="contribution"><div class="authors u-font-sans">T.W. Holliday</div><div id="ref-id-sref45" class="title text-m">Body proportions in Late Pleistocene Europe and modern human origins</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 32 (1997), pp. 423-448</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248496901116/pdf?md5=3b4b9041c3487fa014dbf3ca9576dc7d&amp;pid=1-s2.0-S0047248496901116-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref45"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248496901116" aria-describedby="ref-id-sref45"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0031147898&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref45"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Body%20proportions%20in%20Late%20Pleistocene%20Europe%20and%20modern%20human%20origins&amp;publication_year=1997&amp;author=T.W.%20Holliday" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref45"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib46" id="ref-id-bib46" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ito et&nbsp;al., 2015</span></span></a></span><span class="reference" id="sref46"><div class="contribution"><div class="authors u-font-sans">T. Ito, Y. Kawamoto, Y. Hamada, T.D. Nishimura</div><div id="ref-id-sref46" class="title text-m">Maxillary sinus variation in hybrid macaques: Implications for the genetic basis of craniofacial pneumatization</div></div><div class="host u-font-sans">Biol. J. Linn. Soc., 115 (2015), pp. 333-347</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/bij.12528" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref46"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84928179939&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref46"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Maxillary%20sinus%20variation%20in%20hybrid%20macaques%3A%20Implications%20for%20the%20genetic%20basis%20of%20craniofacial%20pneumatization&amp;publication_year=2015&amp;author=T.%20Ito&amp;author=Y.%20Kawamoto&amp;author=Y.%20Hamada&amp;author=T.D.%20Nishimura" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref46"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib47" id="ref-id-bib47" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Jolly, 2001</span></span></a></span><span class="reference" id="sref47"><div class="contribution"><div class="authors u-font-sans">C.J. Jolly</div><div id="ref-id-sref47" class="title text-m">A&nbsp;proper study for mankind: analogies from the papionin monkeys and their implications for human evolution</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 116 (2001), pp. 177-204</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.10021" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref47"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0041407109&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref47"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0proper%20study%20for%20mankind%3A%20analogies%20from%20the%20papionin%20monkeys%20and%20their%20implications%20for%20human%20evolution&amp;publication_year=2001&amp;author=C.J.%20Jolly" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref47"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib48" id="ref-id-bib48" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Kanthaswamy et&nbsp;al., 2009</span></span></a></span><span class="reference" id="sref48"><div class="contribution"><div class="authors u-font-sans">S. Kanthaswamy, L. Gill, J. Satkoski, V. Goyal, V. Malladi, A. Kou, K. Basuta, L. Sarkisyan, D. George, D.G. Smith</div><div id="ref-id-sref48" class="title text-m">Development of a Chinese-Indian hybrid (Chindian) rhesus macaque colony at the California National primate research center by introgression</div></div><div class="host u-font-sans">J.&nbsp;Med. Primatol., 38 (2009), pp. 86-96</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/j.1600-0684.2008.00305.x" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref48"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-63049123844&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref48"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20Chinese-Indian%20hybrid%20%20rhesus%20macaque%20colony%20at%20the%20California%20National%20primate%20research%20center%20by%20introgression&amp;publication_year=2009&amp;author=S.%20Kanthaswamy&amp;author=L.%20Gill&amp;author=J.%20Satkoski&amp;author=V.%20Goyal&amp;author=V.%20Malladi&amp;author=A.%20Kou&amp;author=K.%20Basuta&amp;author=L.%20Sarkisyan&amp;author=D.%20George&amp;author=D.G.%20Smith" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref48"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib49" id="ref-id-bib49" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Kawada et&nbsp;al., 2020</span></span></a></span><span class="reference" id="sref49"><div class="contribution"><div class="authors u-font-sans">M. Kawada, M. Nakatsukasa, T. Nishimura, A. Kaneko, N. Morimoto</div><div id="ref-id-sref49" class="title text-m">Covariation of fetal skull and maternal pelvis during the perinatal period in rhesus macaques and evolution of childbirth in primates</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 117 (2020), p. 202002112</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Covariation%20of%20fetal%20skull%20and%20maternal%20pelvis%20during%20the%20perinatal%20period%20in%20rhesus%20macaques%20and%20evolution%20of%20childbirth%20in%20primates&amp;publication_year=2020&amp;author=M.%20Kawada&amp;author=M.%20Nakatsukasa&amp;author=T.%20Nishimura&amp;author=A.%20Kaneko&amp;author=N.%20Morimoto" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref49"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib50" id="ref-id-bib50" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Kelaita and Cortés-Ortiz, 2013</span></span></a></span><span class="reference" id="sref50"><div class="contribution"><div class="authors u-font-sans">M.A. Kelaita, L. Cortés-Ortiz</div><div id="ref-id-sref50" class="title text-m">Morphological variation of genetically confirmed <em>Alouatta pigra</em>&nbsp;× <em>A.&nbsp;palliata</em> hybrids from a natural hybrid zone in Tabasco, Mexico</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 150 (2013), pp. 223-234</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.22196" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref50"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84872327836&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref50"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Morphological%20variation%20of%20genetically%20confirmed%20Alouatta%20pigra%C2%A0%C3%97%20A.%C2%A0palliata%20hybrids%20from%20a%20natural%20hybrid%20zone%20in%20Tabasco%2C%20Mexico&amp;publication_year=2013&amp;author=M.A.%20Kelaita&amp;author=L.%20Cort%C3%A9s-Ortiz" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref50"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib51" id="ref-id-bib51" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ledevin and Koyabu, 2019</span></span></a></span><span class="reference" id="sref51"><div class="contribution"><div class="authors u-font-sans">R. Ledevin, D. Koyabu</div><div id="ref-id-sref51" class="title text-m">Patterns and constraints of craniofacial variation in colobine monkeys: Disentangling the effects of phylogeny, allometry and diet</div></div><div class="host u-font-sans">Evol. Biol., 46 (2019), pp. 14-34</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s11692-019-09469-7" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref51"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85060925498&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref51"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Patterns%20and%20constraints%20of%20craniofacial%20variation%20in%20colobine%20monkeys%3A%20Disentangling%20the%20effects%20of%20phylogeny%2C%20allometry%20and%20diet&amp;publication_year=2019&amp;author=R.%20Ledevin&amp;author=D.%20Koyabu" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref51"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib52" id="ref-id-bib52" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Lynch, 1990</span></span></a></span><span class="reference" id="sref52"><div class="contribution"><div class="authors u-font-sans">M. Lynch</div><div id="ref-id-sref52" class="title text-m">The rate of morphological evolution in mammals from the standpoint of the neutral expectation</div></div><div class="host u-font-sans">Am. Nat., 136 (1990), pp. 727-741</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0025635488&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref52"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20rate%20of%20morphological%20evolution%20in%20mammals%20from%20the%20standpoint%20of%20the%20neutral%20expectation&amp;publication_year=1990&amp;author=M.%20Lynch" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref52"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib53" id="ref-id-bib53" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Malukiewicz et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref53"><div class="contribution"><div class="authors u-font-sans">J. Malukiewicz, C.M. Hepp, K. Guschanski, A.C. Stone</div><div id="ref-id-sref53" class="title text-m">Phylogeny of the <em>jacchus</em> group of <em>Callithrix</em> marmosets based on complete mitochondrial genomes</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 162 (2017), pp. 157-169</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.23105" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref53"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85006380159&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref53"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20the%20jacchus%20group%20of%20Callithrix%20marmosets%20based%20on%20complete%20mitochondrial%20genomes&amp;publication_year=2017&amp;author=J.%20Malukiewicz&amp;author=C.M.%20Hepp&amp;author=K.%20Guschanski&amp;author=A.C.%20Stone" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref53"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib54" id="ref-id-bib54" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Martinez et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref54"><div class="contribution"><div class="authors u-font-sans">F.I. Martinez, C. Capelli, M.J. Ferreira da Silva, V. Aldeias, Z. Alemseged, W. Archer, M. Bamford, D. Biro, R. Bobe, D.R. Braun, J.M. Habermann, T. Lüdecke, H. Madiquida, J. Mathe, E. Negash, L.M. Paulo, M. Pinto, M. Stalmans, F. Tátá, S. Carvalho</div><div id="ref-id-sref54" class="title text-m">A&nbsp;missing piece of the <em>Papio</em> puzzle: Gorongosa baboon phenostructure and intrageneric relationships</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 130 (2019), pp. 1-20</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248418301106/pdfft?md5=57d817044cdb7ed1e2458c8480a3fed7&amp;pid=1-s2.0-S0047248418301106-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref54"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248418301106" aria-describedby="ref-id-sref54"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85062397145&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref54"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0missing%20piece%20of%20the%20Papio%20puzzle%3A%20Gorongosa%20baboon%20phenostructure%20and%20intrageneric%20relationships&amp;publication_year=2019&amp;author=F.I.%20Martinez&amp;author=C.%20Capelli&amp;author=M.J.%20Ferreira%20da%20Silva&amp;author=V.%20Aldeias&amp;author=Z.%20Alemseged&amp;author=W.%20Archer&amp;author=M.%20Bamford&amp;author=D.%20Biro&amp;author=R.%20Bobe&amp;author=D.R.%20Braun&amp;author=J.M.%20Habermann&amp;author=T.%20L%C3%BCdecke&amp;author=H.%20Madiquida&amp;author=J.%20Mathe&amp;author=E.%20Negash&amp;author=L.M.%20Paulo&amp;author=M.%20Pinto&amp;author=M.%20Stalmans&amp;author=F.%20T%C3%A1t%C3%A1&amp;author=S.%20Carvalho" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref54"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib55" id="ref-id-bib55" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">McGee et&nbsp;al., 2016</span></span></a></span><span class="reference" id="sref55"><div class="contribution"><div class="authors u-font-sans">M.D. McGee, R.Y. Neches, O. Seehausen</div><div id="ref-id-sref55" class="title text-m">Evaluating genomic divergence and parallelism in replicate ecomorphs from young and old cichlid adaptive radiations</div></div><div class="host u-font-sans">Mol. Ecol., 25 (2016), pp. 260-268</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/mec.13463" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref55"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84954057914&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref55"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evaluating%20genomic%20divergence%20and%20parallelism%20in%20replicate%20ecomorphs%20from%20young%20and%20old%20cichlid%20adaptive%20radiations&amp;publication_year=2016&amp;author=M.D.%20McGee&amp;author=R.Y.%20Neches&amp;author=O.%20Seehausen" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref55"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib56" id="ref-id-bib56" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Melnick et&nbsp;al., 1993</span></span></a></span><span class="reference" id="sref56"><div class="contribution"><div class="authors u-font-sans">D.J. Melnick, G.a Hoelzer, R. Absher, M.V. Ashley</div><div id="ref-id-sref56" class="title text-m">mtDNA diversity in rhesus monkeys reveals overestimates of divergence time and paraphyly with neighboring species</div></div><div class="host u-font-sans">Mol. Biol. Evol., 10 (1993), pp. 282-295</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0027536811&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref56"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=mtDNA%20diversity%20in%20rhesus%20monkeys%20reveals%20overestimates%20of%20divergence%20time%20and%20paraphyly%20with%20neighboring%20species&amp;publication_year=1993&amp;author=D.J.%20Melnick&amp;author=G.a%20Hoelzer&amp;author=R.%20Absher&amp;author=M.V.%20Ashley" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref56"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib90" id="ref-id-bib90" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Meyer et&nbsp;al., 2016</span></span></a></span><span class="reference" id="optUnzcNgwyxw"><div class="contribution"><div class="authors u-font-sans">M. Meyer, J.L. Arsuaga, C. de Filippo, S. Nagel, A. Aximu-Petri, B. Nickle, I. Martínez, A. Gracia, J.M.B. de Castro, E. Carbonell, B. Viola</div><div id="ref-id-optUnzcNgwyxw" class="title text-m">Nuclear DNA sequences from the Middle Pleistocene Sima de los Huesos hominins</div></div><div class="host u-font-sans">Nature, 531 (2016), pp. 504-507</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nature17405" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optUnzcNgwyxw"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84961725717&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optUnzcNgwyxw"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20sequences%20from%20the%20Middle%20Pleistocene%20Sima%20de%20los%20Huesos%20hominins&amp;publication_year=2016&amp;author=M.%20Meyer&amp;author=J.L.%20Arsuaga&amp;author=C.%20de%20Filippo&amp;author=S.%20Nagel&amp;author=A.%20Aximu-Petri&amp;author=B.%20Nickle&amp;author=I.%20Mart%C3%ADnez&amp;author=A.%20Gracia&amp;author=J.M.B.%20de%20Castro&amp;author=E.%20Carbonell&amp;author=B.%20Viola" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optUnzcNgwyxw"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib57" id="ref-id-bib57" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Mitteroecker et&nbsp;al., 2016</span></span></a></span><span class="reference" id="sref57"><div class="contribution"><div class="authors u-font-sans">P. Mitteroecker, S.M. Huttegger, B. Fischer, M. Pavlicev</div><div id="ref-id-sref57" class="title text-m">Cliff-edge model of obstetric selection in humans</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 113 (2016), pp. 14680-14685</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1612410113" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref57"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85006413287&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref57"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Cliff-edge%20model%20of%20obstetric%20selection%20in%20humans&amp;publication_year=2016&amp;author=P.%20Mitteroecker&amp;author=S.M.%20Huttegger&amp;author=B.%20Fischer&amp;author=M.%20Pavlicev" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref57"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib58" id="ref-id-bib58" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Mitteroecker et&nbsp;al., 2017</span></span></a></span><span class="reference" id="sref58"><div class="contribution"><div class="authors u-font-sans">P. Mitteroecker, S. Windhager, M. Pavlicev</div><div id="ref-id-sref58" class="title text-m">Cliff-edge model predicts intergenerational predisposition to dystocia and Caesarean delivery</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 114 (2017), pp. 11669-11672</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1712203114" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref58"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85032694808&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref58"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Cliff-edge%20model%20predicts%20intergenerational%20predisposition%20to%20dystocia%20and%20Caesarean%20delivery&amp;publication_year=2017&amp;author=P.%20Mitteroecker&amp;author=S.%20Windhager&amp;author=M.%20Pavlicev" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref58"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib59" id="ref-id-bib59" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Moore, 2021</span></span></a></span><span class="reference" id="sref59"><div class="contribution"><div class="authors u-font-sans">A.J. Moore</div><div id="ref-id-sref59" class="title text-m">Vertebral pneumaticity is correlated with serial variation in vertebral shape in storks</div></div><div class="host u-font-sans">J.&nbsp;Anat., 238 (2021), pp. 615-625</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/joa.13322" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref59"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85091419260&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref59"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Vertebral%20pneumaticity%20is%20correlated%20with%20serial%20variation%20in%20vertebral%20shape%20in%20storks&amp;publication_year=2021&amp;author=A.J.%20Moore" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref59"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib92" id="ref-id-bib92" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Moorjani et&nbsp;al., 2016</span></span></a></span><span class="reference" id="optZfYCqPwd4I"><div class="contribution"><div class="authors u-font-sans">P. Moorjani, S. Sankararaman, Q. Fu, M. Przeworski, N. Patterson, D. Reich</div><div id="ref-id-optZfYCqPwd4I" class="title text-m">A&nbsp;genetic method for dating ancient genomes provides a direct estimate of human generation interval in the last 45,000 years</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 113 (2016), pp. 5652-5657</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1514696113" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optZfYCqPwd4I"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84969871881&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optZfYCqPwd4I"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%C2%A0genetic%20method%20for%20dating%20ancient%20genomes%20provides%20a%20direct%20estimate%20of%20human%20generation%20interval%20in%20the%20last%2045%2C000%20years&amp;publication_year=2016&amp;author=P.%20Moorjani&amp;author=S.%20Sankararaman&amp;author=Q.%20Fu&amp;author=M.%20Przeworski&amp;author=N.%20Patterson&amp;author=D.%20Reich" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optZfYCqPwd4I"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib60" id="ref-id-bib60" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Plavcan, 2001</span></span></a></span><span class="reference" id="sref60"><div class="contribution"><div class="authors u-font-sans">J.M. Plavcan</div><div id="ref-id-sref60" class="title text-m">Sexual dimorphism in primate evolution</div></div><div class="host u-font-sans">Yearb. Phys. Anthropol., 44 (2001), pp. 25-53</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.10011" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref60"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84903707030&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref60"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Sexual%20dimorphism%20in%20primate%20evolution&amp;publication_year=2001&amp;author=J.M.%20Plavcan" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref60"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib61" id="ref-id-bib61" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ponce de León and Zollikofer, 2001</span></span></a></span><span class="reference" id="sref61"><div class="contribution"><div class="authors u-font-sans">M.S. Ponce de León, C.P.E. Zollikofer</div><div id="ref-id-sref61" class="title text-m">Neanderthal cranial ontogeny and its implications for late hominid diversity</div></div><div class="host u-font-sans">Nature, 412 (2001), pp. 534-538</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0035797392&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref61"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Neanderthal%20cranial%20ontogeny%20and%20its%20implications%20for%20late%20hominid%20diversity&amp;publication_year=2001&amp;author=M.S.%20Ponce%20de%20Le%C3%B3n&amp;author=C.P.E.%20Zollikofer" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref61"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib62" id="ref-id-bib62" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Prentis et&nbsp;al., 2008</span></span></a></span><span class="reference" id="sref62"><div class="contribution"><div class="authors u-font-sans">P.J. Prentis, J.R.U. Wilson, E.E. Dormontt, D.M. Richardson, A.J. Lowe</div><div id="ref-id-sref62" class="title text-m">Adaptive evolution in invasive species</div></div><div class="host u-font-sans">Trends Plant Sci., 13 (2008), pp. 288-294</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S136013850800126X/pdfft?md5=bd206119f2178bdb1a88762d2bbdc039&amp;pid=1-s2.0-S136013850800126X-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref62"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S136013850800126X" aria-describedby="ref-id-sref62"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-44649196460&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref62"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Adaptive%20evolution%20in%20invasive%20species&amp;publication_year=2008&amp;author=P.J.%20Prentis&amp;author=J.R.U.%20Wilson&amp;author=E.E.%20Dormontt&amp;author=D.M.%20Richardson&amp;author=A.J.%20Lowe" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref62"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib21" id="ref-id-bib21" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">R Core Team, 2020</span></span></a></span><span class="reference" id="sref21"><div class="contribution"><div class="authors u-font-sans">R Core Team</div><div id="ref-id-sref21" class="title text-m">R: A Language and Environment for Statistical Computing</div></div><div class="host u-font-sans">R Foundation for Statistical Computing, Vienna, Austria (2018)</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=R%3A%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&amp;publication_year=2018&amp;author=R%20Core%20Team" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref21"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib63" id="ref-id-bib63" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Rak and Arensburg, 1987</span></span></a></span><span class="reference" id="sref63"><div class="contribution"><div class="authors u-font-sans">Y. Rak, B. Arensburg</div><div id="ref-id-sref63" class="title text-m">Kebara 2 Neanderthal pelvis: first look at a complete inlet</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 73 (1987), pp. 227-231</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.1330730209" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref63"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0023279282&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref63"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Kebara%202%20Neanderthal%20pelvis%3A%20first%20look%20at%20a%20complete%20inlet&amp;publication_year=1987&amp;author=Y.%20Rak&amp;author=B.%20Arensburg" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref63"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib64" id="ref-id-bib64" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Rogers et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref64"><div class="contribution"><div class="authors u-font-sans">J. Rogers, M. Raveendran, R.A. Harris, T. Mailund, K. Leppälä, G. Athanasiadis, M.H. Schierup, J. Cheng, K. Munch, J.A. Walker, M.K. Konkel, V.E. Jordan, C.J. Steely, T.O. Beckstrom, C. Bergey, A. Burrell, D. Schrempf, A. Noll, M. Kothe, G.H. Kopp, Y. Liu, S. Murali, K. Billis, F.J. Martin, M. Muffato, L.A. Cox, J. Else, T. Disotell, D.M. Muzny, J. Phillips-Conroy, B. Aken, E.E. Eichler, T. Marques-Bonet, C. Kosiol, M.A. Batzer, M.W. Hahn, J. Tung, D. Zinner, C. Roos, C.J. Jolly, R.A. Gibbs, K.C. Worley, N. Archidiacono, O. Capozzi, C.R. Catacchio, H.H. Dinh, H.V. Doddapaneni, Y. Han, J. Huddleston, S.N. Jhangiani, A. Karimpour-Fard, V. Korchina, C.L. Kovar, L. Kuderna, S.L. Lee, X. Liu, A. Marra-Campanale, C.E. Mason, M. de M. Montero, K.A. Pagel, A. Palazzo, J. Pecotte, V. Pejaver, L. Pipes, V.S. Quick, P. Radivojac, A. Raja, B.J. Raney, K. Rice, M. Rocchi, J.M. Sikela, R. Stanyon, G.W.C. Thomas, M. Ventura, T.P. Vilgalys, T. Vinar, L. Walter</div><div id="ref-id-sref64" class="title text-m">The comparative genomics and complex population history of <em>Papio</em> baboons</div></div><div class="host u-font-sans">Sci. Adv., 5 (2019), Article eaau6947</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85060670885&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref64"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20comparative%20genomics%20and%20complex%20population%20history%20of%20Papio%20baboons&amp;publication_year=2019&amp;author=J.%20Rogers&amp;author=M.%20Raveendran&amp;author=R.A.%20Harris&amp;author=T.%20Mailund&amp;author=K.%20Lepp%C3%A4l%C3%A4&amp;author=G.%20Athanasiadis&amp;author=M.H.%20Schierup&amp;author=J.%20Cheng&amp;author=K.%20Munch&amp;author=J.A.%20Walker&amp;author=M.K.%20Konkel&amp;author=V.E.%20Jordan&amp;author=C.J.%20Steely&amp;author=T.O.%20Beckstrom&amp;author=C.%20Bergey&amp;author=A.%20Burrell&amp;author=D.%20Schrempf&amp;author=A.%20Noll&amp;author=M.%20Kothe&amp;author=G.H.%20Kopp&amp;author=Y.%20Liu&amp;author=S.%20Murali&amp;author=K.%20Billis&amp;author=F.J.%20Martin&amp;author=M.%20Muffato&amp;author=L.A.%20Cox&amp;author=J.%20Else&amp;author=T.%20Disotell&amp;author=D.M.%20Muzny&amp;author=J.%20Phillips-Conroy&amp;author=B.%20Aken&amp;author=E.E.%20Eichler&amp;author=T.%20Marques-Bonet&amp;author=C.%20Kosiol&amp;author=M.A.%20Batzer&amp;author=M.W.%20Hahn&amp;author=J.%20Tung&amp;author=D.%20Zinner&amp;author=C.%20Roos&amp;author=C.J.%20Jolly&amp;author=R.A.%20Gibbs&amp;author=K.C.%20Worley&amp;author=N.%20Archidiacono&amp;author=O.%20Capozzi&amp;author=C.R.%20Catacchio&amp;author=H.H.%20Dinh&amp;author=H.V.%20Doddapaneni&amp;author=Y.%20Han&amp;author=J.%20Huddleston&amp;author=S.N.%20Jhangiani&amp;author=A.%20Karimpour-Fard&amp;author=V.%20Korchina&amp;author=C.L.%20Kovar&amp;author=L.%20Kuderna&amp;author=S.L.%20Lee&amp;author=X.%20Liu&amp;author=A.%20Marra-Campanale&amp;author=C.E.%20Mason&amp;author=M.%20de%20M.%20Montero&amp;author=K.A.%20Pagel&amp;author=A.%20Palazzo&amp;author=J.%20Pecotte&amp;author=V.%20Pejaver&amp;author=L.%20Pipes&amp;author=V.S.%20Quick&amp;author=P.%20Radivojac&amp;author=A.%20Raja&amp;author=B.J.%20Raney&amp;author=K.%20Rice&amp;author=M.%20Rocchi&amp;author=J.M.%20Sikela&amp;author=R.%20Stanyon&amp;author=G.W.C.%20Thomas&amp;author=M.%20Ventura&amp;author=T.P.%20Vilgalys&amp;author=T.%20Vinar&amp;author=L.%20Walter" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref64"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib65" id="ref-id-bib65" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Rougier et&nbsp;al., 2007</span></span></a></span><span class="reference" id="sref65"><div class="contribution"><div class="authors u-font-sans">H. Rougier, Ş. Milota, R. Rodrigo, M. Gherase, L. Sarcinǎ, O. Moldovan, J. Zilhão, S. Constantin, R.G. Franciscus, C.P.E. Zollikofer, M.P. De León, E. Trinkaus</div><div id="ref-id-sref65" class="title text-m">Peştera cu Oase 2 and the cranial morphology of early modern Europeans</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 104 (2007), pp. 1165-1170</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.0610538104" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref65"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33846568560&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref65"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Pe%C5%9Ftera%20cu%20Oase%202%20and%20the%20cranial%20morphology%20of%20early%20modern%20Europeans&amp;publication_year=2007&amp;author=H.%20Rougier&amp;author=%C5%9E.%20Milota&amp;author=R.%20Rodrigo&amp;author=M.%20Gherase&amp;author=L.%20Sarcin%C7%8E&amp;author=O.%20Moldovan&amp;author=J.%20Zilh%C3%A3o&amp;author=S.%20Constantin&amp;author=R.G.%20Franciscus&amp;author=C.P.E.%20Zollikofer&amp;author=M.P.%20De%20Le%C3%B3n&amp;author=E.%20Trinkaus" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref65"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib66" id="ref-id-bib66" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Ruff, 1993</span></span></a></span><span class="reference" id="sref66"><div class="contribution"><div class="authors u-font-sans">C. Ruff</div><div id="ref-id-sref66" class="title text-m">Climatic adaptation and hominid evolution: the thermoregulatory imperative</div></div><div class="host u-font-sans">Evol. Anthropol., 2 (1993), pp. 53-60</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/evan.1360020207" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref66"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0003373264&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref66"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Climatic%20adaptation%20and%20hominid%20evolution%3A%20the%20thermoregulatory%20imperative&amp;publication_year=1993&amp;author=C.%20Ruff" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref66"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib91" id="ref-id-bib91" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Sankararaman et&nbsp;al., 2012</span></span></a></span><span class="reference" id="optDu5uLIbUqN"><div class="contribution"><div class="authors u-font-sans">S. Sankararaman, N. Patterson, H. Li, S. Pääbo, D. Reich</div><div id="ref-id-optDu5uLIbUqN" class="title text-m">The date of interbreeding between Neandertals and modern humans</div></div><div class="host u-font-sans">PLoS Genet., 8 (2012), p. e1002947</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1371/journal.pgen.1002947" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optDu5uLIbUqN"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84868089030&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optDu5uLIbUqN"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20date%20of%20interbreeding%20between%20Neandertals%20and%20modern%20humans&amp;publication_year=2012&amp;author=S.%20Sankararaman&amp;author=N.%20Patterson&amp;author=H.%20Li&amp;author=S.%20P%C3%A4%C3%A4bo&amp;author=D.%20Reich" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-optDu5uLIbUqN"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib67" id="ref-id-bib67" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Savriama et&nbsp;al., 2018</span></span></a></span><span class="reference" id="sref67"><div class="contribution"><div class="authors u-font-sans">Y. Savriama, M. Valtonen, J.I. Kammonen, P. Rastas, O.P. Smolander, A. Lyyski, T.J. Häkkinen, I.J. Corfe, S. Gerber, I. Salazar-Ciudad, L. Paulin, L. Holm, A. Löytynoja, P. Auvinen, J. Jernvall</div><div id="ref-id-sref67" class="title text-m">Bracketing phenogenotypic limits of mammalian hybridization</div></div><div class="host u-font-sans">Roy. Soc. Open Sci., 5 (2018), p. 180903</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rsos.180903" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref67"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85058365351&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref67"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Bracketing%20phenogenotypic%20limits%20of%20mammalian%20hybridization&amp;publication_year=2018&amp;author=Y.%20Savriama&amp;author=M.%20Valtonen&amp;author=J.I.%20Kammonen&amp;author=P.%20Rastas&amp;author=O.P.%20Smolander&amp;author=A.%20Lyyski&amp;author=T.J.%20H%C3%A4kkinen&amp;author=I.J.%20Corfe&amp;author=S.%20Gerber&amp;author=I.%20Salazar-Ciudad&amp;author=L.%20Paulin&amp;author=L.%20Holm&amp;author=A.%20L%C3%B6ytynoja&amp;author=P.%20Auvinen&amp;author=J.%20Jernvall" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref67"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib68" id="ref-id-bib68" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Schlager, 2017</span></span></a></span><span class="reference" id="sref68"><div class="contribution"><div class="authors u-font-sans">S. Schlager</div><div id="ref-id-sref68" class="title text-m">Morpho and Rvcg - shape analysis in R</div></div><div class="host u-font-sans">G. Zheng, S. Li, G. Szekely (Eds.), Statistical Shape and Deformation Analysis, Academic Press, Cambridge (2017), pp. 217-256</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/B9780128104934000110/pdfft?md5=a0b67662245eb0c805ec063d87441313&amp;pid=3-s2.0-B9780128104934000110-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref68"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/B9780128104934000110" aria-describedby="ref-id-sref68"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85032284776&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref68"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Morpho%20and%20Rvcg%20-%20shape%20analysis%20in%20R&amp;publication_year=2017&amp;author=S.%20Schlager" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref68"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib69" id="ref-id-bib69" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Schroeder and von Cramon-Taubadel, 2017</span></span></a></span><span class="reference" id="sref69"><div class="contribution"><div class="authors u-font-sans">L. Schroeder, N. von Cramon-Taubadel</div><div id="ref-id-sref69" class="title text-m">The evolution of hominoid cranial diversity: A quantitative genetic approach</div></div><div class="host u-font-sans">Evolution, 71 (2017), pp. 2634-2649</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/evo.13361" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref69"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85032915253&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref69"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20hominoid%20cranial%20diversity%3A%20A%20quantitative%20genetic%20approach&amp;publication_year=2017&amp;author=L.%20Schroeder&amp;author=N.%20von%20Cramon-Taubadel" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref69"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib70" id="ref-id-bib70" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Sheratt et&nbsp;al., 2019</span></span></a></span><span class="reference" id="sref70"><div class="contribution"><div class="authors u-font-sans">E. Sheratt, K.L. Sanders, A. Watson, M.N. Hutchinson, M.S. Lee, A. Palci</div><div id="ref-id-sref70" class="title text-m">Heterochronic shifts mediate ecomorphological convergence in skull shape of microcephalic sea snakes</div></div><div class="host u-font-sans">Integr. Comp. Biol., 59 (2019), pp. 616-624</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Heterochronic%20shifts%20mediate%20ecomorphological%20convergence%20in%20skull%20shape%20of%20microcephalic%20sea%20snakes&amp;publication_year=2019&amp;author=E.%20Sheratt&amp;author=K.L.%20Sanders&amp;author=A.%20Watson&amp;author=M.N.%20Hutchinson&amp;author=M.S.%20Lee&amp;author=A.%20Palci" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref70"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib71" id="ref-id-bib71" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Smith and Mcdonough, 2005</span></span></a></span><span class="reference" id="sref71"><div class="contribution"><div class="authors u-font-sans">D.G. Smith, J. Mcdonough</div><div id="ref-id-sref71" class="title text-m">Mitochondrial DNA variation in Chinese and Indian Rhesus macaques (<em>Macaca mulatta</em>)</div></div><div class="host u-font-sans">Am. J. Primatol., 65 (2005), pp. 1-25</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-13944271115&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref71"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20variation%20in%20Chinese%20and%20Indian%20Rhesus%20macaques%20&amp;publication_year=2005&amp;author=D.G.%20Smith&amp;author=J.%20Mcdonough" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref71"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib72" id="ref-id-bib72" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Stelkens and Seehausen, 2009</span></span></a></span><span class="reference" id="sref72"><div class="contribution"><div class="authors u-font-sans">R. Stelkens, O. Seehausen</div><div id="ref-id-sref72" class="title text-m">Genetic distance between species predicts novel trait expression in their hybrids</div></div><div class="host u-font-sans">Evolution, 63 (2009), pp. 884-897</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/j.1558-5646.2008.00599.x" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref72"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-62949103871&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref72"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Genetic%20distance%20between%20species%20predicts%20novel%20trait%20expression%20in%20their%20hybrids&amp;publication_year=2009&amp;author=R.%20Stelkens&amp;author=O.%20Seehausen" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref72"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib73" id="ref-id-bib73" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Stelkens et&nbsp;al., 2009</span></span></a></span><span class="reference" id="sref73"><div class="contribution"><div class="authors u-font-sans">R.B. Stelkens, C. Schmid, O. Selz, O. Seehausen</div><div id="ref-id-sref73" class="title text-m">Phenotypic novelty in experimental hybrids is predicted by the genetic distance between species of cichlid fish</div></div><div class="host u-font-sans">BMC Evol. Biol., 9 (2009), p. 283</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1186/1471-2148-9-283" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref73"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-72949118344&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref73"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20novelty%20in%20experimental%20hybrids%20is%20predicted%20by%20the%20genetic%20distance%20between%20species%20of%20cichlid%20fish&amp;publication_year=2009&amp;author=R.B.%20Stelkens&amp;author=C.%20Schmid&amp;author=O.%20Selz&amp;author=O.%20Seehausen" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref73"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib74" id="ref-id-bib74" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Tattersall and Schwartz, 1999</span></span></a></span><span class="reference" id="sref74"><div class="contribution"><div class="authors u-font-sans">I. Tattersall, J.H. Schwartz</div><div id="ref-id-sref74" class="title text-m">Commentary hominids and hybrids: The place of Neanderthals in human evolution</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 96 (1999), pp. 7117-7119</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033595005&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref74"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Commentary%20hominids%20and%20hybrids%3A%20The%20place%20of%20Neanderthals%20in%20human%20evolution&amp;publication_year=1999&amp;author=I.%20Tattersall&amp;author=J.H.%20Schwartz" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref74"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib75" id="ref-id-bib75" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Tosi et&nbsp;al., 2000</span></span></a></span><span class="reference" id="sref75"><div class="contribution"><div class="authors u-font-sans">A.J. Tosi, J.C. Morales, D.J. Melnick</div><div id="ref-id-sref75" class="title text-m">Comparison of Y chromosome and mtDNA phylogenies leads to unique inferences of macaque evolutionary history</div></div><div class="host u-font-sans">Mol. Phylogenet. Evol., 17 (2000), pp. 133-144</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S1055790300908340/pdf?md5=74dc5dda248e9ebc3fd68c9447645c5a&amp;pid=1-s2.0-S1055790300908340-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref75"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S1055790300908340" aria-describedby="ref-id-sref75"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033710127&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref75"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20Y%20chromosome%20and%20mtDNA%20phylogenies%20leads%20to%20unique%20inferences%20of%20macaque%20evolutionary%20history&amp;publication_year=2000&amp;author=A.J.%20Tosi&amp;author=J.C.%20Morales&amp;author=D.J.%20Melnick" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref75"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib76" id="ref-id-bib76" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Trevathan, 2015</span></span></a></span><span class="reference" id="sref76"><div class="contribution"><div class="authors u-font-sans">W. Trevathan</div><div id="ref-id-sref76" class="title text-m">Primate pelvic anatomy and implications for birth</div></div><div class="host u-font-sans">Philos. Trans. Roy. Soc. B Biol. Sci., 370 (2015), p. 20140065</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rstb.2014.0065" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref76"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Primate%20pelvic%20anatomy%20and%20implications%20for%20birth&amp;publication_year=2015&amp;author=W.%20Trevathan" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref76"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib78" id="ref-id-bib78" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Trinkaus et&nbsp;al., 2003a</span></span></a></span><span class="reference" id="sref78"><div class="contribution"><div class="authors u-font-sans">E. Trinkaus, Ş. Milota, R. Rodrigo, G. Mircea, O. Moldovan</div><div id="ref-id-sref78" class="title text-m">Early modern human cranial remains from the Peştera cu Oase, Romania</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 45 (2003), pp. 245-253</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248403001179/pdfft?md5=34bc51af5c8665cf13cf0115073d5829&amp;pid=1-s2.0-S0047248403001179-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref78"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248403001179" aria-describedby="ref-id-sref78"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0242662657&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref78"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Early%20modern%20human%20cranial%20remains%20from%20the%20Pe%C5%9Ftera%20cu%20Oase%2C%20Romania&amp;publication_year=2003&amp;author=E.%20Trinkaus&amp;author=%C5%9E.%20Milota&amp;author=R.%20Rodrigo&amp;author=G.%20Mircea&amp;author=O.%20Moldovan" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref78"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib79" id="ref-id-bib79" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Trinkaus et&nbsp;al., 2003b</span></span></a></span><span class="reference" id="sref79"><div class="contribution"><div class="authors u-font-sans">E. Trinkaus, O. Moldovan, Ş. Milota, A. Bîlgǎr, L. Sarcina, S. Athreya, S.E. Bailey, R. Rodrigo, G. Mircea, T. Higham, C. Bronk Ramsey, J. Van der Plicht</div><div id="ref-id-sref79" class="title text-m">An early modern human from the Peştera cu Oase, Romania</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 100 (2003), pp. 11231-11236</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0141706378&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref79"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=An%20early%20modern%20human%20from%20the%20Pe%C5%9Ftera%20cu%20Oase%2C%20Romania&amp;publication_year=2003&amp;author=E.%20Trinkaus&amp;author=O.%20Moldovan&amp;author=%C5%9E.%20Milota&amp;author=A.%20B%C3%AElg%C7%8Er&amp;author=L.%20Sarcina&amp;author=S.%20Athreya&amp;author=S.E.%20Bailey&amp;author=R.%20Rodrigo&amp;author=G.%20Mircea&amp;author=T.%20Higham&amp;author=C.%20Bronk%20Ramsey&amp;author=J.%20Van%20der%20Plicht" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref79"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib80" id="ref-id-bib80" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Warren et&nbsp;al., 2018</span></span></a></span><span class="reference" id="sref80"><div class="contribution"><div class="authors u-font-sans">K.A. Warren, T.B. Ritzman, R.A. Humphreys, C.J. Percival, B. Hallgrímsson, R.R. Ackermann</div><div id="ref-id-sref80" class="title text-m">Craniomandibular form and body size variation of first generation mouse hybrids: A model for hominin hybridization</div></div><div class="host u-font-sans">J.&nbsp;Hum. Evol., 116 (2018), pp. 57-74</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248416301725/pdfft?md5=bec6fc59999bb04f3c1d0dd181f29ff0&amp;pid=1-s2.0-S0047248416301725-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref80"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="/science/article/pii/S0047248416301725" aria-describedby="ref-id-sref80"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85044251881&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref80"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Craniomandibular%20form%20and%20body%20size%20variation%20of%20first%20generation%20mouse%20hybrids%3A%20A%20model%20for%20hominin%20hybridization&amp;publication_year=2018&amp;author=K.A.%20Warren&amp;author=T.B.%20Ritzman&amp;author=R.A.%20Humphreys&amp;author=C.J.%20Percival&amp;author=B.%20Hallgr%C3%ADmsson&amp;author=R.R.%20Ackermann" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref80"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib81" id="ref-id-bib81" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Weaver, 2009</span></span></a></span><span class="reference" id="sref81"><div class="contribution"><div class="authors u-font-sans">T.D. Weaver</div><div id="ref-id-sref81" class="title text-m">The meaning of Neandertal skeletal morphology</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 106 (2009), pp. 16028-16033</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.0903864106" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref81"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-70349507632&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref81"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20meaning%20of%20Neandertal%20skeletal%20morphology&amp;publication_year=2009&amp;author=T.D.%20Weaver" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref81"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib82" id="ref-id-bib82" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Weaver, 2014</span></span></a></span><span class="reference" id="sref82"><div class="contribution"><div class="authors u-font-sans">T.D. Weaver</div><div id="ref-id-sref82" class="title text-m">Quantitative- and molecular-genetic differentiation in humans and chimpanzees: Implications for the evolutionary processes underlying cranial diversification</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 154 (2014), pp. 615-620</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.22538" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref82"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84903976202&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref82"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Quantitative-%20and%20molecular-genetic%20differentiation%20in%20humans%20and%20chimpanzees%3A%20Implications%20for%20the%20evolutionary%20processes%20underlying%20cranial%20diversification&amp;publication_year=2014&amp;author=T.D.%20Weaver" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref82"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib83" id="ref-id-bib83" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Weaver and Hublin, 2009</span></span></a></span><span class="reference" id="sref83"><div class="contribution"><div class="authors u-font-sans">T.D. Weaver, J. Hublin</div><div id="ref-id-sref83" class="title text-m">Neandertal birth canal shape and the evolution of human childbirth</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 106 (2009), p. 8151</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.0812554106" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref83"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-66249109285&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref83"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Neandertal%20birth%20canal%20shape%20and%20the%20evolution%20of%20human%20childbirth&amp;publication_year=2009&amp;author=T.D.%20Weaver&amp;author=J.%20Hublin" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref83"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib84" id="ref-id-bib84" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Weaver and Stringer, 2015</span></span></a></span><span class="reference" id="sref84"><div class="contribution"><div class="authors u-font-sans">T.D. Weaver, C.B. Stringer</div><div id="ref-id-sref84" class="title text-m">Unconstrained cranial evolution in Neandertals and modern humans compared to common chimpanzees</div></div><div class="host u-font-sans">Proc. Royal Soc. B Biol. Sci., 282 (2015), p. 20151519</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1098/rspb.2015.1519" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref84"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84945219300&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref84"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Unconstrained%20cranial%20evolution%20in%20Neandertals%20and%20modern%20humans%20compared%20to%20common%20chimpanzees&amp;publication_year=2015&amp;author=T.D.%20Weaver&amp;author=C.B.%20Stringer" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref84"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib85" id="ref-id-bib85" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Wells and Stock, 2007</span></span></a></span><span class="reference" id="sref85"><div class="contribution"><div class="authors u-font-sans">J.C.K. Wells, J.T. Stock</div><div id="ref-id-sref85" class="title text-m">The biology of the colonizing ape</div></div><div class="host u-font-sans">Am. J. Phys. Anthropol., 134 (2007), pp. 191-222</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1002/ajpa.20735" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref85"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-36849024449&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref85"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20biology%20of%20the%20colonizing%20ape&amp;publication_year=2007&amp;author=J.C.K.%20Wells&amp;author=J.T.%20Stock" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref85"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib86" id="ref-id-bib86" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Wolf and Akey, 2018</span></span></a></span><span class="reference" id="sref86"><div class="contribution"><div class="authors u-font-sans">A.B. Wolf, J.M. Akey</div><div id="ref-id-sref86" class="title text-m">Outstanding questions in the study of archaic hominin admixture</div></div><div class="host u-font-sans">PLoS Genet., 14 (2018), Article e1007349</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1371/journal.pgen.1007349" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref86"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85048246658&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref86"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Outstanding%20questions%20in%20the%20study%20of%20archaic%20hominin%20admixture&amp;publication_year=2018&amp;author=A.B.%20Wolf&amp;author=J.M.%20Akey" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref86"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib87" id="ref-id-bib87" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Wolfe, 1986</span></span></a></span><span class="reference" id="sref87"><div class="contribution"><div class="authors u-font-sans">L.D. Wolfe</div><div id="ref-id-sref87" class="title text-m">Reproductive biology of rhesus and Japanese macaques</div></div><div class="host u-font-sans">Primates, 27 (1986), pp. 95-101</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0002807191&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref87"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Reproductive%20biology%20of%20rhesus%20and%20Japanese%20macaques&amp;publication_year=1986&amp;author=L.D.%20Wolfe" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref87"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib88" id="ref-id-bib88" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Zhang and Shi, 1993</span></span></a></span><span class="reference" id="sref88"><div class="contribution"><div class="authors u-font-sans">Y.-P. Zhang, L.-M. Shi</div><div id="ref-id-sref88" class="title text-m">Phylogeny of rhesus monkeys (<em>Macaca mulatta</em>) as revealed by mitochondrial DNA restriction enzyme analysis</div></div><div class="host u-font-sans">Int. J. Primatol., 14 (1993), pp. 587-605</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0004528240&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref88"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20rhesus%20monkeys%20%20as%20revealed%20by%20mitochondrial%20DNA%20restriction%20enzyme%20analysis&amp;publication_year=1993&amp;author=Y.-P.%20Zhang&amp;author=L.-M.%20Shi" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref88"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="#bbib89" id="ref-id-bib89" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">Zirkle and Lovejoy, 2019</span></span></a></span><span class="reference" id="sref89"><div class="contribution"><div class="authors u-font-sans">D. Zirkle, C.O. Lovejoy</div><div id="ref-id-sref89" class="title text-m">The hominid ilium is shaped by a synapomorphic growth mechanism that is unique within primates</div></div><div class="host u-font-sans">Proc. Natl. Acad. Sci. USA, 116 (2019), pp. 13915-13920</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.1905242116" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref89"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85068557587&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref89"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20hominid%20ilium%20is%20shaped%20by%20a%20synapomorphic%20growth%20mechanism%20that%20is%20unique%20within%20primates&amp;publication_year=2019&amp;author=D.%20Zirkle&amp;author=C.O.%20Lovejoy" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref89"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li></ol></section></section></div><div id="section-cited-by"><section aria-label="Cited by" class="ListArticles preview"><div class="PageDivider"></div><header id="citing-articles-header"><h2 class="u-h4 u-margin-l-ver u-font-serif">Cited by (11)</h2></header><div aria-describedby="citing-articles-header"><div class="citing-articles u-margin-l-bottom"><ul><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-0-title"><a class="anchor anchor-primary" href="/science/article/pii/S0047248422001671"><span class="anchor-text-container"><span class="anchor-text">Making meaning from fragmentary fossils: Early Homo in the Early to early Middle Pleistocene</span></span></a></h3><div>2023, Journal of Human Evolution</div><div class="CitedSection u-margin-s-top"><div class="u-margin-s-left"><div class="cite-header u-text-italic u-font-sans">Citation Excerpt :</div><p class="u-font-serif text-xs">A key issue is a lack of comparative data from extant taxa regarding how skeletal phenotype is influenced by environment and thus what expectations and conclusions can be drawn from the distribution of phenotypes of fossil fragments. Additionally, there is little comparative data for understanding what to expect from the skeleton in instances of hybridization (although work in this area has recently increased; e.g. Buck et al., 2021; Harvati and Ackermann, 2022). Likewise, alternate models for thinking about how to visualize sexual dimorphism based on size distribution and for thinking about what parameters might influence anatomy in time and space and how to account for that, aside from the fairly blunt instrument of looking at overall size variation and comparing this with our close relatives, would be useful.</p></div></div></div><div class="buttons"><button class="button-link button-link-primary button-link-icon-right" type="button" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-0-title" aria-controls="citing-articles-article-0" aria-expanded="false"><span class="button-link-text-container"><span class="button-link-text">Show abstract</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"><div class="u-margin-ver-m"><div class="u-margin-s-bottom" id="abspara0010">In celebration of the 50th anniversary of the Journal of Human Evolution, we re-evaluate the fossil record for early <em>Homo</em> (principally <em>Homo erectus</em>, <em>Homo habilis</em>, and <em>Homo rudolfensis</em>) from early diversification and dispersal in the Early Pleistocene to the ultimate demise of <em>H.&nbsp;erectus</em> in the early Middle Pleistocene. The mid-1990s marked an important historical turning point in our understanding of early <em>Homo</em> with the redating of key <em>H.&nbsp;erectus</em> localities, the discovery of small <em>H.&nbsp;erectus</em> in Asia, and the recovery of an even earlier presence of early <em>Homo</em> in Africa. As such, we compare our understanding of early <em>Homo</em> before and after this time and discuss how the order of fossil discovery and a focus on anchor specimens has shaped, and in many ways biased, our interpretations of early <em>Homo</em> species and the fossils allocated to them. Fragmentary specimens may counter conventional wisdom but are often overlooked in broad narratives. We recognize at least three different cranial and two or three pelvic morphotypes of early <em>Homo</em>. Just one postcranial morph aligns with any certainty to a cranial species, highlighting the importance of explicitly identifying how we link specimens together and to species; we offer two ways of visualizing these connections. Chronologically and morphologically <em>H.&nbsp;erectus</em> is a member of early <em>Homo</em>, not a temporally more recent species necessarily evolved from either <em>H.&nbsp;habilis</em> or <em>H.&nbsp;rudolfensis</em>. Nonetheless, an ancestral–descendant notion of their evolution influences expectations around the anatomy of missing elements, especially the foot. Weak support for long-held notions of postcranial modernity in <em>H.&nbsp;erectus</em> raises the possibility of alternative drivers of dispersal. New observations suggest that the dearth of faces in later <em>H.&nbsp;erectus</em> may mask taxonomic diversity in Asia and suggest various later mid-Pleistocene populations could derive from either Asia or Africa. Future advances will rest on the development of nuanced ways to affiliate fossils, greater transparency of implicit assumptions, and attention to detailed life history information for comparative collections; all critical pursuits for future research given the great potential they have to enrich our evolutionary reconstructions for the next fifty years and beyond.</div></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-1-title"><a class="anchor anchor-primary" href="/science/article/pii/S0047248422001567"><span class="anchor-text-container"><span class="anchor-text">Moving beyond the adaptationist paradigm for human evolution, and why it matters</span></span></a></h3><div>2023, Journal of Human Evolution</div><div class="CitedSection u-margin-s-top"><div class="u-margin-s-left"><div class="cite-header u-text-italic u-font-sans">Citation Excerpt :</div><p class="u-font-serif text-xs">These studies have provided a way of illuminating population history, understanding fitness, and potentially determining how selection on genes introduced through gene exchange shapes the phenotype, results of which can be applied to the human past. This research shows that new variation, even novelty, is produced through hybridization, and we can see it quite clearly in skeletons of numerous animals, including coyote-wolf hybrids (Zdjelar et al., 2021), baboons (Ackermann et al., 2006, 2014; Eichel and Ackermann, 2016), other primates (Ackermann and Bishop, 2010; Buck et al., 2021) and other mammals (Leamy, 1982; Thorpe and Leamy, 1983; Ackermann et al., 2010; Warren et al., 2018; Hanot et al., 2019). Baboons and canids represent particularly good analogs for hominins in terms of being medium-sized land-based mammals that are geographically widespread, as recognized by Hall and Jolly (and others).</p></div></div></div><div class="buttons"><button class="button-link button-link-primary button-link-icon-right" type="button" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-1-title" aria-controls="citing-articles-article-1" aria-expanded="false"><span class="button-link-text-container"><span class="button-link-text">Show abstract</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"><div class="u-margin-ver-m"><div class="u-margin-s-bottom" id="abspara0010">The Journal of Human Evolution (JHE) was founded 50 years ago when much of the foundation for how we think about human evolution was in place or being put in place, providing the main framework for how we consider our origins today. Here, we will explore historical developments, including early JHE outputs, as they relate to our understanding of the relationship between phenotypic variation and evolutionary process, and use that as a springboard for considering our current understanding of these links as applied to human evolution. We will focus specifically on how the study of variation itself has shifted us away from taxonomic and adaptationist perspectives toward a richer understanding of the processes shaping human evolutionary history, using literature searches and specific test cases to highlight this. We argue that natural selection, gene exchange, genetic drift, and mutation should not be considered individually when considering the production of hominin diversity. In this context, we offer suggestions for future research directions and reflect on this more complex understanding of human evolution and its broader relevance to society. Finally, we end by considering authorship demographics and practices in the last 50 years within JHE and how a shift in these demographics has the potential to reshape the science of human evolution going forward.</div></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-2-title"><a class="anchor anchor-primary" href="https://doi.org/10.7554/elife.78169" target="_blank"><span class="anchor-text-container"><span class="anchor-text">The Natural History of Model Organisms: The rhesus macaque as a success story of the Anthropocene</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2022, Elife</div></div><div class="buttons"><button class="button-link button-link-primary button-link-icon-right" type="button" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-2-title" aria-controls="citing-articles-article-2" aria-expanded="false"><span class="button-link-text-container"><span class="button-link-text">Show abstract</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"><div class="u-margin-ver-m"><div class="u-margin-s-bottom">Of all the non-human primate species studied by researchers, the rhesus macaque (<em>Macaca mulatta</em>) is likely the most widely used across biological disciplines. Rhesus macaques have thrived during the Anthropocene and now have the largest natural range of any non-human primate. They are highly social, exhibit marked genetic diversity, and display remarkable niche flexibility (which allows them to live in a range of habitats and survive on a variety of diets). These characteristics mean that rhesus macaques are well-suited for understanding the links between sociality, health and fitness, and also for investigating intra-specific variation, adaptation and other topics in evolutionary ecology.</div></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-3-title"><a class="anchor anchor-primary" href="https://doi.org/10.1002/ar.25366" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Testing the reliability of the rearticulation of osteological primate pelves in comparative morphological studies</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2024, Anatomical Record</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-4-title"><a class="anchor anchor-primary" href="https://doi.org/10.1002/ajpa.24634" target="_blank"><span class="anchor-text-container"><span class="anchor-text">A Pleistocene assemblage of near-modern Papio hamadryas from the Middle Awash study area, Afar Rift, Ethiopia</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2023, American Journal of Biological Anthropology</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-5-title"><a class="anchor anchor-primary" href="https://doi.org/10.1038/s41559-022-01875-z" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Merging morphological and genetic evidence to assess hybridization in Western Eurasian late Pleistocene hominins</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2022, Nature Ecology and Evolution</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li></ul><a class="button-alternative button-alternative-secondary large-alternative button-alternative-icon-left" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-85113619289&amp;md5=2698a6d2488982daf2f277b46bb6e71" target="_blank" id="citing-articles-view-all-btn"><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-right"><path d="M1 99l38-38L1 23l7-7 45 45-45 45z"></path></svg><span class="button-alternative-text-container"><span class="button-alternative-text">View all citing articles on Scopus</span></span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></a></div></div></section></div><div class="Copyright"><span class="copyright-line">© 2021 The Author(s). Published by Elsevier Ltd.</span></div></article><div class="u-display-block-from-md col-lg-6 col-md-8 pad-right u-padding-s-top"><aside class="RelatedContent u-clr-grey8" aria-label="Related content"><section class="RelatedContentPanel u-margin-s-bottom"><header id="recommended-articles-header" class="related-content-panel-header u-margin-s-bottom"><button class="button-link button-link-secondary related-content-panel-toggle is-up button-link-icon-right button-link-has-colored-icon" type="button" aria-expanded="true" data-aa-button="sd:product:journal:article:location=recommended-articles:type=close"><span class="button-link-text-container"><span class="button-link-text"><h2 class="section-title u-h4"><span class="related-content-panel-title-text">Recommended articles</span></h2></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles" class="text-xs"><ul><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article0-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S1471490615002719" title="CTLA-4 and Autoimmunity: New Twists in the Tale"><span class="anchor-text-container"><span class="anchor-text"><span>CTLA-4 and Autoimmunity: New Twists in the Tale</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Trends in Immunology, Volume 36, Issue 12, 2015, pp. 760-762</div></div><div class="authors"><span>Lucy S.K.</span> <span>Walker</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S1471490615002719/pdfft?md5=63dfe42d7f1934d3fe03b7e9e4a20b68&amp;pid=1-s2.0-S1471490615002719-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article0-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article1-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S026643561600005X" title="Electrochemical telomerase assay for screening for oral cancer"><span class="anchor-text-container"><span class="anchor-text"><span>Electrochemical telomerase assay for screening for oral cancer</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">British Journal of Oral and Maxillofacial Surgery, Volume 54, Issue 3, 2016, pp. 301-305</div></div><div class="authors"><span>Mana</span> <span>Hayakawa</span>, …, <span>Kazuhiro</span> <span>Tominaga</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article2-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S0047248421001561" title="First record of Macaca (Cercopithecidae, Primates) in the Middle Pleistocene of Greece"><span class="anchor-text-container"><span class="anchor-text"><span>First record of <em>Macaca</em> (Cercopithecidae, Primates) in the Middle Pleistocene of Greece</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Journal of Human Evolution, Volume 162, 2022, Article 103104</div></div><div class="authors"><span>George E.</span> <span>Konidaris</span>, …, <span>Katerina</span> <span>Harvati</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248421001561/pdfft?md5=da7fec5f161f6e6ffa10073be0f51f4c&amp;pid=1-s2.0-S0047248421001561-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article2-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article3-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S0047248416301683" title="3D enamel thickness in Neandertal and modern human permanent canines"><span class="anchor-text-container"><span class="anchor-text"><span>3D enamel thickness in Neandertal and modern human permanent canines</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Journal of Human Evolution, Volume 113, 2017, pp. 162-172</div></div><div class="authors"><span>Laura</span> <span>Buti</span>, …, <span>Stefano</span> <span>Benazzi</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="/science/article/pii/S0047248416301683/pdfft?md5=048949e33e360b97e66ca09bf00942b7&amp;pid=1-s2.0-S0047248416301683-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article3-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article4-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S004724841930020X" title="The dental remains from the Early Upper Paleolithic of Manot Cave, Israel"><span class="anchor-text-container"><span class="anchor-text"><span>The dental remains from the Early Upper Paleolithic of Manot Cave, Israel</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Journal of Human Evolution, Volume 160, 2021, Article 102648</div></div><div class="authors"><span>Rachel</span> <span>Sarig</span>, …, <span>Gerhard W.</span> <span>Weber</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article5-title"><a class="anchor u-clamp-2-lines anchor-primary" href="/science/article/pii/S0268003323000189" title="Body mass affects kinetic symmetry and inflammatory markers in adolescent knees during gait"><span class="anchor-text-container"><span class="anchor-text"><span>Body mass affects kinetic symmetry and inflammatory markers in adolescent knees during gait</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Clinical Biomechanics, Volume 102, 2023, Article 105887</div></div><div class="authors"><span>Sarah P.</span> <span>Shultz</span>, …, <span>Howard J.</span> <span>Hillstrom</span></div></div><div class="buttons"></div></li></ul></div><button class="button-link more-recommendations-button u-margin-s-bottom button-link-primary button-link-icon-right" type="button"><span class="button-link-text-container"><span class="button-link-text">Show 3 more articles</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></section><section class="RelatedContentPanel u-margin-s-bottom"><header id="metrics-header" class="related-content-panel-header u-margin-s-bottom"><button class="button-link button-link-secondary related-content-panel-toggle is-up button-link-icon-right button-link-has-colored-icon" type="button" aria-expanded="true"><span class="button-link-text-container"><span class="button-link-text"><h2 class="section-title u-h4"><span class="related-content-panel-title-text">Article Metrics</span></h2></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><div class="plumX-metrics"><h3 class="text-s metric-title metric-title-citations">Citations</h3><ul><li class="text-xs metrics"><span>Citation Indexes</span><span>11</span></li></ul><h3 class="text-s metric-title metric-title-captures">Captures</h3><ul><li class="text-xs metrics"><span>Mendeley Readers</span><span>20</span></li></ul><h3 class="text-s metric-title metric-title-social-media">Social Media</h3><ul><li class="text-xs metrics"><span>Shares, Likes &amp; Comments</span><span>6</span></li></ul><div class="metrics u-margin-m-top u-margin-s-bottom"><img src="https://cdn.plu.mx/3ba727faf225e19d2c759f6ebffc511d/plumx-logo.png" class="plumX-logo" alt="PlumX Metrics Logo"><a class="anchor text-xs anchor-primary" href="https://plu.mx/plum/a/?doi=10.1016%2Fj.jhevol.2021.103049&amp;theme=plum-sciencedirect-theme&amp;hideUsage=true" target="_blank"><span class="anchor-text-container"><span class="anchor-text">View details</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></div></div></section></aside></div></div></div></div><footer role="contentinfo" class="els-footer u-bg-white text-xs u-padding-s-hor u-padding-m-hor-from-sm u-padding-l-hor-from-md u-padding-l-ver u-margin-l-top u-margin-xl-top-from-sm u-margin-l-top-from-md"><div class="els-footer-elsevier u-margin-m-bottom u-margin-0-bottom-from-md u-margin-s-right u-margin-m-right-from-md u-margin-l-right-from-lg"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.elsevier.com/" target="_blank" aria-label="Elsevier home page (opens in a new tab)"><img class="footer-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/47/images/elsevier-non-solus-new-with-wordmark.svg" alt="Elsevier logo with wordmark" height="64" width="58" loading="lazy"></a></div><div class="els-footer-content"><div class="u-remove-if-print"><ul class="els-footer-links u-margin-xs-bottom" style="list-style:none"><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/solutions/sciencedirect" target="_blank" id="els-footer-about-science-direct" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">About ScienceDirect</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0047248421001019" id="els-footer-remote-access" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Remote access</span></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsmediakits.com" target="_blank" id="els-footer-advertise" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Advertise</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" target="_blank" id="els-footer-contact-support" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Contact and support</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" target="_blank" id="els-footer-terms-condition" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Terms and conditions</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/legal/privacy-policy" target="_blank" id="els-footer-privacy-policy" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Privacy policy</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li></ul></div><p id="els-footer-cookie-message" class="u-remove-if-print">Cookies are used by this site. <!-- --> <button class="button-link ot-sdk-show-settings cookie-btn button-link-primary button-link-small" type="button" id="ot-sdk-btn">Cookie Settings</button></p><p id="els-footer-copyright">All content on this site: Copyright © <!-- -->2025<!-- --> Elsevier B.V., its licensors, and contributors. All rights are reserved, including those for text and data mining, AI training, and similar technologies. For all open access content, the relevant licensing terms apply.</p></div><div class="els-footer-relx u-margin-0-top u-margin-m-top-from-xs u-margin-0-top-from-md"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.relx.com/" target="_blank" aria-label="RELX home page (opens in a new tab)" id="els-footer-relx"><img class="" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/60/images/logo-relx-tm.svg" alt="RELX group home page" height="20" width="93" loading="lazy"></a></div></footer></div></div></div></div>
      <div id="floating-ui-node" class="floating-ui-node" data-sd-ui-floating-ui="true"></div>
      <iframe style="display: none" src="//acw.clinicalkey.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u" tabindex="-1"></iframe><iframe style="display: none" src="//acw.scopus.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u" tabindex="-1"></iframe><iframe style="display: none" src="//acw.sciencedirect.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u" tabindex="-1"></iframe><iframe style="display: none" src="//acw.elsevier.com/SSOCore/update?acw=fb054b1e516c5745e229e9125c85597b3f1dgxrqa%7C%24%7C2142A2F44E06FFFB8FA3526695CA7981CA61B302D290706694F54566A2D1C1DF164588A55FAC4A5484738069F2EF9EF76376B0BBE194D0143FBA44D1BD4E4F2EB0469A67597464825D387A21AFA2E514&amp;utt=9c01-29d701c57917b71a9c6-c338e6a124245881-u" tabindex="-1"></iframe>
      
      
<script type="text/javascript">
    window.pageData = {"content":[{"contentType":"JL","format":"MIME-XHTML","id":"sd:article:pii:S0047248421001019","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"0047-2484","volumeNumber":"159","suppl":"C","provider":"elsevier","entitlementType":"openaccess"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productAppVersion":"full-direct","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1749596714470,"loadTime":"","testId":"sd:article-page:current-ui"},"visitor":{"accessType":"ae:ANON_GUEST","accountId":"ae:228598","accountName":"ae:ScienceDirect Guests","loginStatus":"anonymous","userId":"ae:12975512","ipAddress":"189.120.74.179","appSessionId":"442b58e3-bb78-4710-a5c5-e1f732f8c18b"}};
    window.pageData.page.loadTime = performance ? Math.round(performance.now()).toString() : '';

    try {
      appData.push({
      event: 'pageLoad',
      page: pageData.page,
      visitor: pageData.visitor,
      content: pageData.content
      })
    } catch(e) {
        console.warn("There was an error loading or running Adobe DTM: ", e);
    }
</script>
      <script nomodule="" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/73/js/core-js/3.20.2/core-js.es.minified.js" type="text/javascript"></script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/108/js/react/18.3.1/react.production.min.js" type="text/javascript"></script>
      <script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/108/js/react-dom/18.3.1/react-dom.production.min.js" type="text/javascript"></script>
      <script async="" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/0408e3544c5d8a31f29fd84f48c3b04cd8f090f1/arp.js" type="text/javascript"></script>
      <script type="text/javascript">
    const pendoData = {"visitor":{"pageName":"SD:product:journal:article","pageType":"CP-CA","pageProduct":"SD","pageLanguage":"en","pageEnvironment":"prod","accessType":"ae:ANON_GUEST","countryCode":"BR"},"account":{"id":"ae:228598","name":"ae:ScienceDirect Guests"},"events":{}};;
    pendoData.events = {
      ready: function () {
        pendo.addAltText();
      },
    };
    function runPendo(data, options) {
  const {
    firstDelay,
    maxRetries,
    urlPrefix,
    urlSuffix,
    apiKey
  } = options;
  (function (apiKey) {
    (function (p, e, n, d, o) {
      var v, w, x, y, z;
      o = p[d] = p[d] || {};
      o._q = [];
      v = ['initialize', 'identify', 'updateOptions', 'pageLoad'];
      for (w = 0, x = v.length; w < x; ++w) (function (m) {
        o[m] = o[m] || function () {
          o._q[m === v[0] ? 'unshift' : 'push']([m].concat([].slice.call(arguments, 0)));
        };
      })(v[w]);
      y = e.createElement(n);
      y.async = !0;
      y.src = urlPrefix + apiKey + urlSuffix;
      z = e.getElementsByTagName(n)[0];
      z.parentNode.insertBefore(y, z);
    })(window, document, 'script', 'pendo');
    pendo.addAltText = function () {
      var target = document.querySelector('body');
      var observer = new MutationObserver(function (mutations) {
        mutations.forEach(function (mutation) {
          if (mutation?.addedNodes?.length) {
            if (mutation.addedNodes[0]?.className?.includes("_pendo-badge")) {
              const badge = mutation.addedNodes[0];
              const altText = badge?.attributes['aria-label']?.value || 'Feedback';
              const pendoBadgeImage = pendo.dom(`#${badge?.attributes?.id.value} img`);
              if (pendoBadgeImage.length) {
                pendoBadgeImage[0]?.setAttribute('alt', altText);
              }
            }
          }
        });
      });
      var config = {
        attributeFilter: ['data-layout'],
        attributes: true,
        childList: true,
        characterData: true,
        subtree: false
      };
      observer.observe(target, config);
    };
  })(apiKey);
  (function watchAndSetPendo(nextDelay, retryAttempt) {
    if (data.visitor.id) {
      console.debug(`initializing pendo`);
      pendo.initialize(data);
    } else if (typeof pageDataTracker === 'object' && typeof pageDataTracker.getVisitorId === 'function' && pageDataTracker.getVisitorId()) {
      data.visitor.id = pageDataTracker.getVisitorId();
      console.debug(`initializing pendo`);
      pendo.initialize(data);
    } else {
      if (retryAttempt > 0) {
        return setTimeout(function () {
          watchAndSetPendo(nextDelay * 2, retryAttempt - 1);
        }, nextDelay);
      }
      pendo.initialize(data);
      console.debug(`gave up ... pendo initialized`);
    }
  })(firstDelay, maxRetries);
}
    window.addPendoScript = () => {
      runPendo(pendoData, {
        firstDelay: 100,
        maxRetries: 5,
        urlPrefix: 'https://cdn.pendo.io/agent/static/',
        urlSuffix: '/pendo.js',
        apiKey: 'd6c1d995-bc7e-4e53-77f1-2ea4ecbb9565',
      });
      window.addPendoScript = () => {};
    };
    addEventListener('load', () => window.addPendoScript());
  </script>
      <span id="pendo-answer-rating"></span>
      <script type="text/x-mathjax-config;executed=true">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=MML_SVG" type="text/javascript"></script>
      <script async="" src="https://www.googletagservices.com/tag/js/gpt.js" type="text/javascript"></script>
      <script async="" src="https://scholar.google.com/scholar_js/casa.js" type="text/javascript"></script>
      <script data-cfasync="false">
      (function initOneTrust()  {
        const monitor = {
  init: () => {},
  loaded: () => {},
};
        function enableGroup(group) {
  document.querySelectorAll(`script[type*="ot-${group}"]`).forEach(script => {
    script.type = 'text/javascript';
    document.head.appendChild(script);
  });
}
        function runOneTrustCookies(doClear, monitor) {
  const oneTrustConsentSdkId = 'onetrust-consent-sdk';
  const emptyNodeSelectors = 'h3.ot-host-name, h4.ot-host-desc, button.ot-host-box';
  const ariaLabelledByButtonNodes = 'div.ot-accordion-layout > button';
  const ariaAttribute = 'aria-labelledby';
  function adjustOneTrustDOM() {
    const oneTrustRoot = document.getElementById('onetrust-consent-sdk');

    /* remove empty nodes */
    [...(oneTrustRoot?.querySelectorAll(emptyNodeSelectors) ?? [])].filter(e => e.textContent === '').forEach(e => e.remove());

    /* remove invalid aria-labelledby values */
    oneTrustRoot?.querySelectorAll(ariaLabelledByButtonNodes).forEach(e => {
      const presentIdValue = e.getAttribute(ariaAttribute)?.split(' ').filter(label => document.getElementById(label)).join(' ');
      if (presentIdValue) {
        e.setAttribute(ariaAttribute, presentIdValue);
      }
    });
  }
  function observeOneTrustLoaded(shouldSetOTDefaults, isConsentPresent) {
    const cb = (mutationList, observer) => {
      const oneTrustRoot = mutationList.filter(mutationRecord => mutationRecord.type === 'childList' && mutationRecord.addedNodes.length).map(mutationRecord => [...mutationRecord.addedNodes]).flat().find(e => e.id === oneTrustConsentSdkId);
      if (oneTrustRoot && typeof OneTrust !== 'undefined') {
        monitor.loaded(true);
        OneTrust.OnConsentChanged(() => {
          const perfAllowed = decodeURIComponent(document.cookie.match('(^| )OptanonConsent=([^;]+)')?.[2])?.match('groups=([0-9:0|1,?]+)&?')?.[1]?.match('2:([0|1])')[1] === '1';
          if (perfAllowed) {
            enableGroup('performance');
          }
        });
        if (!isConsentPresent && (shouldSetOTDefaults || OneTrust.GetDomainData().ConsentModel.Name === 'implied consent')) {
          OneTrust.UpdateConsent(`Category`, `1:1`);
          OneTrust.UpdateConsent(`Category`, `2:1`);
          OneTrust.UpdateConsent(`Category`, `3:1`);
          OneTrust.UpdateConsent(`Category`, `4:0`);
        }
        document.dispatchEvent(new CustomEvent('@sdtech/onetrust/loaded', {}));
        observer.disconnect();
        adjustOneTrustDOM();
      }
    };
    const observer = new MutationObserver(cb);
    observer.observe(document.querySelector('body'), {
      childList: true
    });
  }
  if (doClear) {
    document.cookie = 'OptanonAlertBoxClosed=; expires=Thu, 01 Jan 1970 00:00:00 UTC; samesite=lax; path=/';
  }
  const isConsentPresent = !!decodeURIComponent(document.cookie.match('(^| )OptanonConsent=([^;]+)')?.[2])?.match('groups=([0-9:0|1,?]+)&?')?.[1];
  const shouldSetOTDefaults = 'true' === 'false' && !document.cookie?.match('OptanonAlertBoxClosed=');
  if (shouldSetOTDefaults) {
    const date = new Date();
    date.setFullYear(date.getFullYear() + 1);
    document.cookie = `OptanonAlertBoxClosed=${new Date().toISOString()}; expires=${date.toUTCString()}; samesite=lax; path=/; domain=sciencedirect.com`;
  }
  observeOneTrustLoaded(shouldSetOTDefaults, isConsentPresent, monitor);
  window.addOTScript = () => {
    const otSDK = document.createElement('script');
    otSDK.setAttribute('data-cfasync', 'false');
    otSDK.setAttribute('src', 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js');
    otSDK.setAttribute('data-document-language', 'true');
    otSDK.setAttribute('data-domain-script', '865ea198-88cc-4e41-8952-1df75d554d02');
    window.addOTScript = () => {};
    document.head.appendChild(otSDK);
    monitor.init();
  };
  window.addEventListener('load', () => window.addOTScript());
}
        if (document.location.host.match(/.sciencedirect.com$/)) {
          runOneTrustCookies(true, monitor);
        }
        else {
          window.addEventListener('load', (event) => {
            enableGroup('performance');
          });
        }
      }());
    </script>
      
    <script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc8327e86a4ee3',t:'MTc0OTU5NjcxNC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe>
  <iframe id="ps_frcldb" src="https://service.seamlessaccess.org/ps/" style="display: none; position: absolute; top: -999px; left: -999px;"></iframe><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-hide ot-fade-in" style="display: none;visibility: hidden;
                    opacity: 0;transition: visibility 0s 400ms, opacity 400ms linear;"></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region" style="display: none;visibility: hidden;
                    opacity: 0;transition: visibility 0s 400ms, opacity 400ms linear;"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler" ot-button-order="0" style="display: inline-block;">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Strictly Necessary Cookies - Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" checked="" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Performance Cookies - Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Targeting Cookies - Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><div id="ot-filter-list-header"></div><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler" ot-button-order="2">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite">Your Privacy [`dialog closed`]</span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><script>
var pageDataTracker = {
    eventCookieName: 'eventTrack',
    debugCookie: 'els-aa-debugmode',
    debugCounter: 1,
    warnings: [],
    measures: {},
    timeoffset: 0

    ,trackPageLoad: function(data) {
        if (window.pageData && ((pageData.page && pageData.page.noTracking == 'true') || window.pageData_isLoaded)) {
            return false;
        }

        this.updatePageData(data);

        this.initWarnings();
        if(!(window.pageData && pageData.page && pageData.page.name)) {
            console.error('pageDataTracker.trackPageLoad() called without pageData.page.name being defined!');
            return;
        }

        this.processIdPlusData(window.pageData);

        if(window.pageData && pageData.page && !pageData.page.loadTime) {
          pageData.page.loadTime = performance ? Math.round((performance.now())).toString() : '';
        }

        if(window.pageData && pageData.page) {
            var localTime = new Date().getTime();
            if(pageData.page.loadTimestamp) {
                // calculate timeoffset
                var serverTime = parseInt(pageData.page.loadTimestamp);
                if(!isNaN(serverTime)) {
                    this.timeoffset = pageData.page.loadTimestamp - localTime;
                }
            } else {
                pageData.page.loadTimestamp = localTime;
            }
        }

        this.validateData(window.pageData);

        try {
            var cookieTest = 'aa-cookie-test';
            this.setCookie(cookieTest, cookieTest);
            if(this.getCookie(cookieTest) != cookieTest) {
                this.warnings.push('dtm5');
            }
            this.deleteCookie(cookieTest);
        } catch(e){
            this.warnings.push('dtm5');
        }

        this.registerCallbacks();
        this.setAnalyticsData();

        // handle any cookied event data
        this.getEvents();

        window.pageData_isLoaded = true;

        this.debugMessage('Init - trackPageLoad()', window.pageData);

        _satellite.track('eventDispatcher', JSON.stringify({
          eventName: 'newPage',
          eventData: {eventName: 'newPage'},
          pageData: window.pageData
        }));
    }

    ,trackEvent: function(event, data, callback) {
        if (window.pageData && pageData.page && pageData.page.noTracking == 'true') {
            return false;
        }
        
        if(!window.pageData_isLoaded) {
            if(this.isDebugEnabled()) {
                console.log('[AA] pageDataTracker.trackEvent() called without calling trackPageLoad() first.');
            }
            return false;
        }

        if (event) {
            this.initWarnings();
            if(event === 'newPage') {
                // auto fillings
                if(data && data.page && !data.page.loadTimestamp) {
                    data.page.loadTimestamp = ''+(new Date().getTime() + this.timeoffset);
                }
                this.processIdPlusData(data);
            }

            window.eventData = data ? data : {};
            window.eventData.eventName = event;
            if(!_satellite.getVar('blacklisted')) {
                this.handleEventData(event, data);
    
                if(event === 'newPage') {
                    this.validateData(window.pageData);
                }
                this.debugMessage('Event: ' + event, data);
    
                _satellite.track('eventDispatcher', JSON.stringify({
                  eventName: event,
                  eventData: window.eventData,
                  pageData: window.pageData
                }));
            } else {
                this.debugMessage('!! Blocked Event: ' + event, data);
            }
        }

        if (typeof(callback) == 'function') {
            callback.call();
        }
    }

    ,processIdPlusData: function(data) {
        if(data && data.visitor && data.visitor.idPlusData) {
            var idPlusFields = ['userId', 'accessType', 'accountId', 'accountName'];
            for(var i=0; i < idPlusFields.length; i++) {
                if(typeof data.visitor.idPlusData[idPlusFields[i]] !== 'undefined') {
                    data.visitor[idPlusFields[i]] = data.visitor.idPlusData[idPlusFields[i]];
                }
            }
            data.visitor.idPlusData = undefined;
        }
    }

    ,validateData: function(data) {
        if(!data) {
            this.warnings.push('dv0');
            return;
        }

        // top 5
        if(!(data.visitor && data.visitor.accessType)) {
            this.warnings.push('dv1');
        }
        if(data.visitor && (data.visitor.accountId || data.visitor.accountName)) {
            if(!data.visitor.accountName) {
                this.warnings.push('dv2');
            }
            if(!data.visitor.accountId) {
                this.warnings.push('dv3');
            }
        }
        if(!(data.page && data.page.productName)) {
            this.warnings.push('dv4');
        }
        if(!(data.page && data.page.businessUnit)) {
            this.warnings.push('dv5');
        }
        if(!(data.page && data.page.name)) {
            this.warnings.push('dv6');
        }

        // rp mandatory
        if(data.page && data.page.businessUnit && (data.page.businessUnit.toLowerCase().indexOf('els:rp:') !== -1 || data.page.businessUnit.toLowerCase().indexOf('els:rap:') !== -1)) {
            if(!(data.page && data.page.loadTimestamp)) {
                this.warnings.push('dv7');
            }
            if(!(data.page && data.page.loadTime)) {
                this.warnings.push('dv8');
            }
            if(!(data.visitor && data.visitor.ipAddress)) {
                this.warnings.push('dv9');
            }
            if(!(data.page && data.page.type)) {
                this.warnings.push('dv10');
            }
            if(!(data.page && data.page.language)) {
                this.warnings.push('dv11');
            }
        }

        // other
        if(data.page && data.page.environment) {
            var env = data.page.environment.toLowerCase();
            if(!(env === 'dev' || env === 'cert' || env === 'prod')) {
                this.warnings.push('dv12');
            }
        }
        if(data.content && data.content.constructor !== Array) {
            this.warnings.push('dv13');
        }

        if(data.visitor && data.visitor.accountId && data.visitor.accountId.indexOf(':') == -1) {
            this.warnings.push('dv14');
            data.visitor.accountId = "data violation"
        }
    }

    ,initWarnings: function() {
        this.warnings = [];
        try {
            var hdn = document.head.childNodes;
            var libf = false;
            for(var i=0; i<hdn.length; i++) {
                if(hdn[i].src && (hdn[i].src.indexOf('satelliteLib') !== -1 || hdn[i].src.indexOf('launch') !== -1)) {
                    libf = true;
                    break;
                }
            }
            if(!libf) {
                this.warnings.push('dtm1');
            }
        } catch(e) {}

        try {
            for (let element of document.querySelectorAll('*')) {
                // Check if the element has a src attribute and if it meets the criteria
                if (element.src && element.src.includes('assets.adobedtm.com') && element.src.includes('launch')) {
                    if (element.tagName.toLowerCase() !== 'script') {
                        this.warnings.push('dtm5');
                    }
                    if (!element.hasAttribute('async')) {
                        this.warnings.push('dtm4');
                    }
                }
            }
        } catch (e) { }
    }

    ,getMessages: function() {
        return ['v1'].concat(this.warnings).join('|');
    }
    ,addMessage: function(message) {
        this.warnings.push(message);
    }

    ,getPerformance: function() {
        var copy = {};
        for (var attr in this.measures) {
            if(this.measures.hasOwnProperty(attr)) {
                copy[attr] = this.measures[attr];
            }
        }

        this.measures = {};
        return copy;
    }

    ,dtmCodeDesc: {
        dtm1: 'satellite-lib must be placed in the <head> section',
        dtm2: 'trackPageLoad() must be placed and called before the closing </body> tag',
        dtm3: 'trackEvent() must be called at a stage where Document.readyState=complete (e.g. on the load event or a user event)',
        dtm4: 'Embed codes need to be loaded in async mode',
        dtm5: 'Embed codes not in type script',
        dv1: 'visitor.accessType not set but mandatory',
        dv2: 'visitor.accountName not set but mandatory',
        dv3: 'visitor.accountId not set but mandatory',
        dv4: 'page.productName not set but mandatory',
        dv5: 'page.businessUnit not set but mandatory',
        dv6: 'page.name not set but mandatory',
        dv7: 'page.loadTimestamp not set but mandatory',
        dv8: 'page.loadTime not set but mandatory',
        dv9: 'visitor.ipAddress not set but mandatory',
        dv10: 'page.type not set but mandatory',
        dv11: 'page.language not set but mandatory',
        dv12: 'page.environment must be set to \'prod\', \'cert\' or \'dev\'',
        dv13: 'content must be of type array of objects',
        dv14: 'account number must contain at least one \':\', e.g. \'ae:12345\''
    }

    ,debugMessage: function(event, data) {
        if(this.isDebugEnabled()) {
            console.log('[AA] --------- [' + (this.debugCounter++) + '] Web Analytics Data ---------');
            console.log('[AA] ' + event);
            console.groupCollapsed("[AA] AA Data: ");
            if(window.eventData) {
                console.log("[AA] eventData:\n" + JSON.stringify(window.eventData, true, 2));
            }
            if(window.pageData) {
                console.log("[AA] pageData:\n" + JSON.stringify(window.pageData, true, 2));
            }
            console.groupEnd();
            if(this.warnings.length > 0) {
                console.groupCollapsed("[AA] Warnings ("+this.warnings.length+"): ");
                for(var i=0; i<this.warnings.length; i++) {
                    var error = this.dtmCodeDesc[this.warnings[i]] ? this.dtmCodeDesc[this.warnings[i]] : 'Error Code: ' + this.warnings[i];
                    console.log('[AA] ' + error);
                }
                console.log('[AA] More can be found here: https://confluence.cbsels.com/display/AA/AA+Error+Catalog');
                console.groupEnd();
            }
            console.log("This mode can be disabled by calling 'pageDataTracker.disableDebug()'");
        }
    }

    ,getTrackingCode: function() {
      var campaign = _satellite.getVar('Campaign - ID');
      if(!campaign) {
        campaign = window.sessionStorage ? sessionStorage.getItem('dgcid') : '';
      }
      return campaign;
    }

    ,isDebugEnabled: function() {
        if(typeof this.debug === 'undefined') {
            this.debug = (document.cookie.indexOf(this.debugCookie) !== -1) || (window.pageData && pageData.page && pageData.page.environment && pageData.page.environment.toLowerCase() === 'dev');
            //this.debug = (document.cookie.indexOf(this.debugCookie) !== -1);
        }
        return this.debug;
    }

    ,enableDebug: function(expire) {
        if (typeof expire === 'undefined') {
            expire = 86400;
        }
        console.log('You just enabled debug mode for Adobe Analytics tracking. This mode will persist for 24h.');
        console.log("This mode can be disabled by calling 'pageDataTracker.disableDebug()'");
        this.setCookie(this.debugCookie, 'true', expire, document.location.hostname);
        this.debug = true;
    }

    ,disableDebug: function() {
        console.log('Debug mode is now disabled.');
        this.deleteCookie(this.debugCookie);
        this.debug = false;
    }

    ,setAnalyticsData: function() {
        if(!(window.pageData && pageData.page && pageData.page.productName && pageData.page.name)) {
            return;
        }
        pageData.page.analyticsPagename = pageData.page.productName + ':' + pageData.page.name;

        var pageEls = pageData.page.name.indexOf(':') > -1 ? pageData.page.name.split(':') : [pageData.page.name];
        pageData.page.sectionName = pageData.page.productName + ':' + pageEls[0];
    }

    ,getEvents: function() {
        pageData.savedEvents = {};
        pageData.eventList = [];

        var val = this.getCookie(this.eventCookieName);
        if (val) {
            pageData.savedEvents = val;
        }

        this.deleteCookie(this.eventCookieName);
    }

    ,updatePageData(data) {
        window.pageData = window.pageData || {};
        if (data && typeof(data) === 'object') {
            for (var x in data) {
                if(data.hasOwnProperty(x) && data[x] instanceof Array) {
                    pageData[x] = data[x];
                } else if(data.hasOwnProperty(x) && typeof(data[x]) === 'object') {
                    if(!pageData[x]) {
                        pageData[x] = {};
                    }
                    for (var y in data[x]) {
                        if(data[x].hasOwnProperty(y)) {
                            pageData[x][y] = data[x][y];
                        }
                    }
                }
            }
        }
    }

    ,handleEventData: function(event, data) {
        var val;
        switch(event) {
            case 'newPage':
                this.updatePageData(data);
                this.setAnalyticsData();
            case 'saveSearch':
            case 'searchResultsUpdated':
                if (data) {
                    // overwrite page-load object
                    if (data.search && typeof(data.search) == 'object') {
                        window.eventData.search.resultsPosition = '';
                        pageData.search = pageData.search || {};
                        var fields = ['advancedCriteria', 'criteria', 'currentPage', 'dataFormCriteria', 'facets', 'resultsByType', 'resultsPerPage', 'sortType', 'totalResults', 'type', 'database',
                        'suggestedClickPosition','suggestedLetterCount','suggestedResultCount', 'autoSuggestCategory', 'autoSuggestDetails','typedTerm','selectedTerm', 'channel',
                        'facetOperation', 'details'];
                        for (var i=0; i<fields.length; i++) {
                            if (data.search[fields[i]]) {
                                pageData.search[fields[i]] = data.search[fields[i]];
                            }
                        }
                    }
                }
                this.setAnalyticsData();
                break;
            case 'navigationClick':
                if (data && data.link) {
                    window.eventData.navigationLink = {
                        name: ((data.link.location || 'no location') + ':' + (data.link.name || 'no name'))
                    };
                }
                break;
            case 'autoSuggestClick':
                if (data && data.search) {
                    val = {
                        autoSuggestSearchData: (
                            'letterct:' + (data.search.suggestedLetterCount || 'none') +
                            '|resultct:' + (data.search.suggestedResultCount || 'none') +
                            '|clickpos:' + (data.search.suggestedClickPosition || 'none')
                        ).toLowerCase(),
                        autoSuggestSearchTerm: (data.search.typedTerm || ''),
                        autoSuggestTypedTerm: (data.search.typedTerm || ''),
                        autoSuggestSelectedTerm: (data.search.selectedTerm || ''),
                        autoSuggestCategory: (data.search.autoSuggestCategory || ''),
                        autoSuggestDetails: (data.search.autoSuggestDetails || '')
                    };
                }
                break;
            case 'linkOut':
                if (data && data.content && data.content.length > 0) {
                    window.eventData.linkOut = data.content[0].linkOut;
                    window.eventData.referringProduct = _satellite.getVar('Page - Product Name') + ':' + data.content[0].id;
                }
                break;
            case 'socialShare':
                if (data && data.social) {
                    window.eventData.sharePlatform = data.social.sharePlatform || '';
                }
                break;
            case 'contentInteraction':
                if (data && data.action) {
                    window.eventData.action.name = pageData.page.productName + ':' + data.action.name;
                }
                break;
            case 'searchWithinContent':
                if (data && data.search) {
                    window.pageData.search = window.pageData.search || {};
                    pageData.search.withinContentCriteria = data.search.withinContentCriteria;
                }
                break;
            case 'contentShare':
                if (data && data.content) {
                    window.eventData.sharePlatform = data.content[0].sharePlatform;
                }
                break;
            case 'contentLinkClick':
                if (data && data.link) {
                    window.eventData.action = { name: pageData.page.productName + ':' + (data.link.type || 'no link type') + ':' + (data.link.name || 'no link name') };
                }
                break;
            case 'contentWindowLoad':
            case 'contentTabClick':
                if (data && data.content) {
                    window.eventData.tabName = data.content[0].tabName || '';
                    window.eventData.windowName = data.content[0].windowName || '';
                }
                break;
            case 'userProfileUpdate':
                if (data && data.user) {
                    if (Object.prototype.toString.call(data.user) === "[object Array]") {
                        window.eventData.user = data.user[0];
                    }
                }
                break;
            case 'videoStart':
                if (data.video) {
                    data.video.length = parseFloat(data.video.length || '0');
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.open(data.video.id, data.video.length, s.Media.playerName);
                    s.Media.play(data.video.id, data.video.position);
                }
                break;
            case 'videoPlay':
                if (data.video) {
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.play(data.video.id, data.video.position);
                }
                break;
            case 'videoStop':
                if (data.video) {
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.stop(data.video.id, data.video.position);
                }
                break;
            case 'videoComplete':
                if (data.video) {
                    data.video.position = parseFloat(data.video.position || '0');
                    s.Media.stop(data.video.id, data.video.position);
                    s.Media.close(data.video.id);
                }
                break;
            case 'addWebsiteExtension':
                if(data && data.page) {
                    val = {
                        wx: data.page.websiteExtension
                    }
                }
                break;
        }

        if (val) {
            this.setCookie(this.eventCookieName, val);
        }
    }

    ,registerCallbacks: function() {
        var self = this;
        if(window.usabilla_live) {
            window.usabilla_live('setEventCallback', function(category, action, label, value) {
                if(action == 'Campaign:Open') {
                    self.trackEvent('ctaImpression', {
                        cta: {
                            ids: ['usabillaid:' + label]
                        }
                    });
                } else if(action == 'Campaign:Success') {
                    self.trackEvent('ctaClick', {
                        cta: {
                            ids: ['usabillaid:' + label]
                        }
                    });
                }
            });
        }
    }

    ,getConsortiumAccountId: function() {
        var id = '';
        if (window.pageData && pageData.visitor && (pageData.visitor.consortiumId || pageData.visitor.accountId)) {
            id = (pageData.visitor.consortiumId || 'no consortium ID') + '|' + (pageData.visitor.accountId || 'no account ID');
        }

        return id;
    }

    ,getSearchClickPosition: function() {
        if (window.eventData && eventData.search && eventData.search.resultsPosition) {
            var pos = parseInt(eventData.search.resultsPosition), clickPos;
            if (!isNaN(pos)) {
                var page = pageData.search.currentPage ? parseInt(pageData.search.currentPage) : '', perPage = pageData.search.resultsPerPage ? parseInt(pageData.search.resultsPerPage) : '';
                if (!isNaN(page) && !isNaN(perPage)) {
                    clickPos = pos + ((page - 1) * perPage);
                }
            }
            return clickPos ? clickPos.toString() : eventData.search.resultsPosition;
        }
        return '';
    }

    ,getSearchFacets: function() {
        var facetList = '';
        if (window.pageData && pageData.search && pageData.search.facets) {
            if (typeof(pageData.search.facets) == 'object') {
                for (var i=0; i<pageData.search.facets.length; i++) {
                    var f = pageData.search.facets[i];
                    facetList += (facetList ? '|' : '') + f.name + '=' + f.values.join('^');
                }
            }
        }
        return facetList;
    }

    ,getSearchResultsByType: function() {
        var resultTypes = '';
        if (window.pageData && pageData.search && pageData.search.resultsByType) {
            for (var i=0; i<pageData.search.resultsByType.length; i++) {
                var r = pageData.search.resultsByType[i];
                resultTypes += (resultTypes ? '|' : '') + r.name + (r.results || r.values ? '=' + (r.results || r.values) : '');
            }
        }
        return resultTypes;
    }

    ,getJournalInfo: function() {
        var info = '';
        if (window.pageData && pageData.journal && (pageData.journal.name || pageData.journal.specialty || pageData.journal.section || pageData.journal.issn || pageData.journal.issueNumber || pageData.journal.volumeNumber || pageData.journal.family || pageData.journal.publisher)) {
            var journal = pageData.journal;
            info = (journal.name || 'no name') 
            + '|' + (journal.specialty || 'no specialty') 
            + '|' + (journal.section || 'no section') 
            + '|' + (journal.issn || 'no issn') 
            + '|' + (journal.issueNumber || 'no issue #') 
            + '|' + (journal.volumeNumber || 'no volume #')
            + '|' + (journal.family || 'no family')
            + '|' + (journal.publisher || 'no publisher');

        }
        return info;
    }

    ,getBibliographicInfo: function(doc) {
        if (!doc || !(doc.publisher || doc.indexTerms || doc.publicationType || doc.publicationRights || doc.volumeNumber || doc.issueNumber || doc.subjectAreas || doc.isbn)) {
            return '';
        }

        var terms = doc.indexTerms ? doc.indexTerms.split('+') : '';
        if (terms) {
            terms = terms.slice(0, 5).join('+');
            terms = terms.length > 100 ? terms.substring(0, 100) : terms;
        }

        var areas = doc.subjectAreas ? doc.subjectAreas.split('>') : '';
        if (areas) {
            areas = areas.slice(0, 5).join('>');
            areas = areas.length > 100 ? areas.substring(0, 100) : areas;
        }

        var biblio	= (doc.publisher || 'none')
            + '^' + (doc.publicationType || 'none')
            + '^' + (doc.publicationRights || 'none')
            + '^' + (terms || 'none')
            + '^' + (doc.volumeNumber || 'none')
            + '^' + (doc.issueNumber || 'none')
            + '^' + (areas || 'none')
            + '^' + (doc.isbn || 'none');

        return this.stripProductDelimiters(biblio).toLowerCase();
    }

    ,getContentItem: function() {
        var docs = window.eventData && eventData.content ? eventData.content : pageData.content;
        if (docs && docs.length > 0) {
            return docs[0];
        }
    }

    ,getFormattedDate: function(ts) {
        if (!ts) {
            return '';
        }

        var d = new Date(parseInt(ts) * 1000);

        // now do formatting
        var year = d.getFullYear()
            ,month = ((d.getMonth() + 1) < 10 ? '0' : '') + (d.getMonth() + 1)
            ,date = (d.getDate() < 10 ? '0' : '') + d.getDate()
            ,hours = d.getHours() > 12 ? d.getHours() - 12 : d.getHours()
            ,mins = (d.getMinutes() < 10 ? '0' : '') + d.getMinutes()
            ,ampm = d.getHours() > 12 ? 'pm' : 'am';

        hours = (hours < 10 ? '0' : '') + hours;
        return year + '-' + month + '-' + date;
    }

    ,getVisitorId: function() {
        var orgId = '4D6368F454EC41940A4C98A6@AdobeOrg';
        if(Visitor && Visitor.getInstance(orgId)) {
            return Visitor.getInstance(orgId).getMarketingCloudVisitorID();
        } else {
            return ''
        }
    }

    ,setProductsVariable: function() {
        var prodList = window.eventData && eventData.content ? eventData.content : pageData.content
            ,prods = [];
        if (prodList) {
            for (var i=0; i<prodList.length; i++) {
                if (prodList[i].id || prodList[i].type || prodList[i].publishDate || prodList[i].onlineDate) {
                    if (!prodList[i].id) {
                        prodList[i].id = 'no id';
                    }
                    var prodName = (pageData.page.productName || 'xx').toLowerCase();
                    if (prodList[i].id.indexOf(prodName + ':') != 0) {
                        prodList[i].id = prodName + ':' + prodList[i].id;
                    }
                    prodList[i].id = this.stripProductDelimiters(prodList[i].id);
                    var merch = [];
                    if (prodList[i].format) {
                        merch.push('evar17=' + this.stripProductDelimiters(prodList[i].format.toLowerCase()));
                    }
                    if (prodList[i].type) {
                        var type = prodList[i].type;
                        if (prodList[i].accessType) {
                            type += ':' + prodList[i].accessType;
                        }
                        merch.push('evar20=' + this.stripProductDelimiters(type.toLowerCase()));

                        if(type.indexOf(':manuscript') > 0) {
                            /*
                            var regex = /[a-z]+:manuscript:id:([a-z]+-[a-z]-[0-9]+-[0-9]+)/gmi;
                            var m = regex.exec(prodList[i].id);
                            if(m) {
                                merch.push('evar200=' + m[1]);
                            }
                            merch.push('evar200=' + prodList[i].id);
                            */
                            a = prodList[i].id.lastIndexOf(':');
                            if(a>0) {
                                merch.push('evar200=' + prodList[i].id.substring(a+1).toUpperCase());
                            }
                        } else if(type.indexOf(':submission') > 0) {
                            merch.push('evar200=' + prodList[i].id);
                        }
                    }
                    if(!prodList[i].title) {
                        prodList[i].title = prodList[i].name;
                    }
                    if (prodList[i].title) {
                        merch.push('evar75=' + this.stripProductDelimiters(prodList[i].title.toLowerCase()));
                    }
                    if (prodList[i].breadcrumb) {
                        merch.push('evar63=' + this.stripProductDelimiters(prodList[i].breadcrumb).toLowerCase());
                    }
                    var nowTs = new Date().getTime()/1000;
                    if (prodList[i].onlineDate && !isNaN(prodList[i].onlineDate)) {
                        if(prodList[i].onlineDate > 32503680000) {
                            prodList[i].onlineDate = prodList[i].onlineDate/1000;
                        }
                        merch.push('evar122=' + this.stripProductDelimiters(pageDataTracker.getFormattedDate(prodList[i].onlineDate)));
                        var onlineAge = Math.floor((nowTs - prodList[i].onlineDate) / 86400);
                        onlineAge = (onlineAge === 0) ? 'zero' : onlineAge;
                        merch.push('evar128=' + onlineAge);
                    }
                    if (prodList[i].publishDate && !isNaN(prodList[i].publishDate)) {
                        if(prodList[i].publishDate > 32503680000) {
                            prodList[i].publishDate = prodList[i].publishDate/1000;
                        }
                        merch.push('evar123=' + this.stripProductDelimiters(pageDataTracker.getFormattedDate(prodList[i].publishDate)));
                        var publishAge = Math.floor((nowTs - prodList[i].publishDate) / 86400);
                        publishAge = (publishAge === 0) ? 'zero' : publishAge;
                        merch.push('evar127=' + publishAge);
                    }
                    if (prodList[i].onlineDate && prodList[i].publishDate) {
                        merch.push('evar38=' + this.stripProductDelimiters(pageDataTracker.getFormattedDate(prodList[i].onlineDate) + '^' + pageDataTracker.getFormattedDate(prodList[i].publishDate)));
                    }
                    if (prodList[i].mapId) {
                        merch.push('evar70=' + this.stripProductDelimiters(prodList[i].mapId));
                    }
					if (prodList[i].relevancyScore) {
						merch.push('evar71=' + this.stripProductDelimiters(prodList[i].relevancyScore));
					}
                    if (prodList[i].status) {
                        merch.push('evar73=' + this.stripProductDelimiters(prodList[i].status));
                    }
                    if (prodList[i].previousStatus) {
                        merch.push('evar111=' + this.stripProductDelimiters(prodList[i].previousStatus));
                    }
                    if (prodList[i].entitlementType) {
                        merch.push('evar80=' + this.stripProductDelimiters(prodList[i].entitlementType));
                    }
                    if (prodList[i].recordType) {
                        merch.push('evar93=' + this.stripProductDelimiters(prodList[i].recordType));
                    }
                    if (prodList[i].exportType) {
                        merch.push('evar99=' + this.stripProductDelimiters(prodList[i].exportType));
                    }
                    if (prodList[i].importType) {
                        merch.push('evar142=' + this.stripProductDelimiters(prodList[i].importType));
                    }
                    if (prodList[i].section) {
                        merch.push('evar100=' + this.stripProductDelimiters(prodList[i].section));
                    }
                    if (prodList[i].detail) {
                        merch.push('evar104=' + this.stripProductDelimiters(prodList[i].detail.toLowerCase()));
                    } else if(prodList[i].details) {
                        merch.push('evar104=' + this.stripProductDelimiters(prodList[i].details.toLowerCase()));
                    }
                    if (prodList[i].position) {
                        merch.push('evar116=' + this.stripProductDelimiters(prodList[i].position));
                    }
                    if (prodList[i].publicationTitle) {
                        merch.push('evar129=' + this.stripProductDelimiters(prodList[i].publicationTitle));
                    }
                    if (prodList[i].specialIssueTitle) {
                        merch.push('evar130=' + this.stripProductDelimiters(prodList[i].specialIssueTitle));
                    }
                    if (prodList[i].specialIssueNumber) {
                        merch.push('evar131=' + this.stripProductDelimiters(prodList[i].specialIssueNumber));
                    }
                    if (prodList[i].referenceModuleTitle) {
                        merch.push('evar139=' + this.stripProductDelimiters(prodList[i].referenceModuleTitle));
                    }
                    if (prodList[i].referenceModuleISBN) {
                        merch.push('evar140=' + this.stripProductDelimiters(prodList[i].referenceModuleISBN));
                    }
                    if (prodList[i].volumeTitle) {
                        merch.push('evar132=' + this.stripProductDelimiters(prodList[i].volumeTitle));
                    }
                    if (prodList[i].publicationSection) {
                        merch.push('evar133=' + this.stripProductDelimiters(prodList[i].publicationSection));
                    }
                    if (prodList[i].publicationSpecialty) {
                        merch.push('evar134=' + this.stripProductDelimiters(prodList[i].publicationSpecialty));
                    }
                    if (prodList[i].issn) {
                        merch.push('evar135=' + this.stripProductDelimiters(prodList[i].issn));
                    }
                    if (prodList[i].id2) {
                        merch.push('evar159=' + this.stripProductDelimiters(prodList[i].id2));
                    }
                    if (prodList[i].id3) {
                        merch.push('evar160=' + this.stripProductDelimiters(prodList[i].id3));
                    }
                    if (prodList[i].provider) {
                        merch.push('evar164=' + this.stripProductDelimiters(prodList[i].provider));
                    }
                    if (prodList[i].citationStyle) {
                        merch.push('evar170=' + this.stripProductDelimiters(prodList[i].citationStyle));
                    }

                    var biblio = this.getBibliographicInfo(prodList[i]);
                    if (biblio) {
                        merch.push('evar28=' + biblio);
                    }

                    if (prodList[i].turnawayId) {
                        pageData.eventList.push('product turnaway');
                    }

                    var price = prodList[i].price || '', qty = prodList[i].quantity || '', evts = [];
                    if (price && qty) {
                        qty = parseInt(qty || '1');
                        price = parseFloat(price || '0');
                        price = (price * qty).toFixed(2);

                        if (window.eventData && eventData.eventName && eventData.eventName == 'cartAdd') {
                            evts.push('event20=' + price);
                        }
                    }

                    var type = window.pageData && pageData.page && pageData.page.type ? pageData.page.type : '', evt = window.eventData && eventData.eventName ? eventData.eventName : '';
                    if (type.match(/^CP\-/gi) !== null && (!evt || evt == 'newPage' || evt == 'contentView')) {
                        evts.push('event181=1');
                    }
                    if (evt == 'contentDownload' || type.match(/^CP\-DL/gi) !== null) {
                        evts.push('event182=1');
                    }
                    if (evt == 'contentDownloadRequest') {
                        evts.push('event319=1');
                    }
                    if (evt == 'contentExport') {
                        evts.push('event184=1');
                    }
                    if (this.eventFires('recommendationViews')) {
                        evts.push('event264=1');
                    }

                    if(prodList[i].datapoints) {
                        evts.push('event239=' + prodList[i].datapoints);
                    }
                    if(prodList[i].documents) {
                        evts.push('event240=' + prodList[i].documents);
                    }
                    if(prodList[i].size) {
                        evts.push('event335=' + prodList[i].size);
                        evts.push('event336=1')
                    }

                    if(evt == 'genAIContentUpdated') {
                        evts.push('event51=1');
                    }

                    prods.push([
                        ''					// empty category
                        ,prodList[i].id		// id
                        ,qty				// qty
                        ,price				// price
                        ,evts.join('|')		// events
                        ,merch.join('|')	// merchandising eVars
                    ].join(';'));
                }
            }
        }

        return prods.join(',');
    }
    ,eventFires: function(eventName) {
      var evt = window.eventData && eventData.eventName ? eventData.eventName : '';
      if(evt == eventName) {
        return true;
      }
      // initial pageload and new pages
      if((!window.eventData || evt == 'newPage') && window.pageData && window.pageData.trackEvents) {
        var tEvents = window.pageData.trackEvents;
        for(var i=0; i<tEvents.length; i++) {
          if(tEvents[i] == eventName) {
            return true;
          }
        }
      }
      return false;
    }

    ,md5: function(s){function L(k,d){return(k<<d)|(k>>>(32-d))}function K(G,k){var I,d,F,H,x;F=(G&2147483648);H=(k&2147483648);I=(G&1073741824);d=(k&1073741824);x=(G&1073741823)+(k&1073741823);if(I&d){return(x^2147483648^F^H)}if(I|d){if(x&1073741824){return(x^3221225472^F^H)}else{return(x^1073741824^F^H)}}else{return(x^F^H)}}function r(d,F,k){return(d&F)|((~d)&k)}function q(d,F,k){return(d&k)|(F&(~k))}function p(d,F,k){return(d^F^k)}function n(d,F,k){return(F^(d|(~k)))}function u(G,F,aa,Z,k,H,I){G=K(G,K(K(r(F,aa,Z),k),I));return K(L(G,H),F)}function f(G,F,aa,Z,k,H,I){G=K(G,K(K(q(F,aa,Z),k),I));return K(L(G,H),F)}function D(G,F,aa,Z,k,H,I){G=K(G,K(K(p(F,aa,Z),k),I));return K(L(G,H),F)}function t(G,F,aa,Z,k,H,I){G=K(G,K(K(n(F,aa,Z),k),I));return K(L(G,H),F)}function e(G){var Z;var F=G.length;var x=F+8;var k=(x-(x%64))/64;var I=(k+1)*16;var aa=Array(I-1);var d=0;var H=0;while(H<F){Z=(H-(H%4))/4;d=(H%4)*8;aa[Z]=(aa[Z]| (G.charCodeAt(H)<<d));H++}Z=(H-(H%4))/4;d=(H%4)*8;aa[Z]=aa[Z]|(128<<d);aa[I-2]=F<<3;aa[I-1]=F>>>29;return aa}function B(x){var k="",F="",G,d;for(d=0;d<=3;d++){G=(x>>>(d*8))&255;F="0"+G.toString(16);k=k+F.substr(F.length-2,2)}return k}function J(k){k=k.replace(/rn/g,"n");var d="";for(var F=0;F<k.length;F++){var x=k.charCodeAt(F);if(x<128){d+=String.fromCharCode(x)}else{if((x>127)&&(x<2048)){d+=String.fromCharCode((x>>6)|192);d+=String.fromCharCode((x&63)|128)}else{d+=String.fromCharCode((x>>12)|224);d+=String.fromCharCode(((x>>6)&63)|128);d+=String.fromCharCode((x&63)|128)}}}return d}var C=Array();var P,h,E,v,g,Y,X,W,V;var S=7,Q=12,N=17,M=22;var A=5,z=9,y=14,w=20;var o=4,m=11,l=16,j=23;var U=6,T=10,R=15,O=21;s=J(s);C=e(s);Y=1732584193;X=4023233417;W=2562383102;V=271733878;for(P=0;P<C.length;P+=16){h=Y;E=X;v=W;g=V;Y=u(Y,X,W,V,C[P+0],S,3614090360);V=u(V,Y,X,W,C[P+1],Q,3905402710);W=u(W,V,Y,X,C[P+2],N,606105819);X=u(X,W,V,Y,C[P+3],M,3250441966);Y=u(Y,X,W,V,C[P+4],S,4118548399);V=u(V,Y,X,W,C[P+5],Q,1200080426);W=u(W,V,Y,X,C[P+6],N,2821735955);X=u(X,W,V,Y,C[P+7],M,4249261313);Y=u(Y,X,W,V,C[P+8],S,1770035416);V=u(V,Y,X,W,C[P+9],Q,2336552879);W=u(W,V,Y,X,C[P+10],N,4294925233);X=u(X,W,V,Y,C[P+11],M,2304563134);Y=u(Y,X,W,V,C[P+12],S,1804603682);V=u(V,Y,X,W,C[P+13],Q,4254626195);W=u(W,V,Y,X,C[P+14],N,2792965006);X=u(X,W,V,Y,C[P+15],M,1236535329);Y=f(Y,X,W,V,C[P+1],A,4129170786);V=f(V,Y,X,W,C[P+6],z,3225465664);W=f(W,V,Y,X,C[P+11],y,643717713);X=f(X,W,V,Y,C[P+0],w,3921069994);Y=f(Y,X,W,V,C[P+5],A,3593408605);V=f(V,Y,X,W,C[P+10],z,38016083);W=f(W,V,Y,X,C[P+15],y,3634488961);X=f(X,W,V,Y,C[P+4],w,3889429448);Y=f(Y,X,W,V,C[P+9],A,568446438);V=f(V,Y,X,W,C[P+14],z,3275163606);W=f(W,V,Y,X,C[P+3],y,4107603335);X=f(X,W,V,Y,C[P+8],w,1163531501);Y=f(Y,X,W,V,C[P+13],A,2850285829);V=f(V,Y,X,W,C[P+2],z,4243563512);W=f(W,V,Y,X,C[P+7],y,1735328473);X=f(X,W,V,Y,C[P+12],w,2368359562);Y=D(Y,X,W,V,C[P+5],o,4294588738);V=D(V,Y,X,W,C[P+8],m,2272392833);W=D(W,V,Y,X,C[P+11],l,1839030562);X=D(X,W,V,Y,C[P+14],j,4259657740);Y=D(Y,X,W,V,C[P+1],o,2763975236);V=D(V,Y,X,W,C[P+4],m,1272893353);W=D(W,V,Y,X,C[P+7],l,4139469664);X=D(X,W,V,Y,C[P+10],j,3200236656);Y=D(Y,X,W,V,C[P+13],o,681279174);V=D(V,Y,X,W,C[P+0],m,3936430074);W=D(W,V,Y,X,C[P+3],l,3572445317);X=D(X,W,V,Y,C[P+6],j,76029189);Y=D(Y,X,W,V,C[P+9],o,3654602809);V=D(V,Y,X,W,C[P+12],m,3873151461);W=D(W,V,Y,X,C[P+15],l,530742520);X=D(X,W,V,Y,C[P+2],j,3299628645);Y=t(Y,X,W,V,C[P+0],U,4096336452);V=t(V,Y,X,W,C[P+7],T,1126891415);W=t(W,V,Y,X,C[P+14],R,2878612391);X=t(X,W,V,Y,C[P+5],O,4237533241);Y=t(Y,X,W,V,C[P+12],U,1700485571);V=t(V,Y,X,W,C[P+3],T,2399980690);W=t(W,V,Y,X,C[P+10],R,4293915773);X=t(X,W,V,Y,C[P+1],O,2240044497);Y=t(Y,X,W,V,C[P+8],U,1873313359);V=t(V,Y,X,W,C[P+15],T,4264355552);W=t(W,V,Y,X,C[P+6],R,2734768916);X=t(X,W,V,Y,C[P+13],O,1309151649);Y=t(Y,X,W,V,C[P+4],U,4149444226);V=t(V,Y,X,W,C[P+11],T,3174756917);W=t(W,V,Y,X,C[P+2],R,718787259);X=t(X,W,V,Y,C[P+9],O,3951481745);Y=K(Y,h);X=K(X,E);W=K(W,v);V=K(V,g)}var i=B(Y)+B(X)+B(W)+B(V);return i.toLowerCase()}
    ,stripProductDelimiters: function(val) {
        if (val) {
            return val.replace(/\;|\||\,/gi, '-');
        }
    }

    ,setCookie: function(name, value, seconds, domain) {
        domain = document.location.hostname;
        var expires = '';
        var expiresNow = '';
        var date = new Date();
        date.setTime(date.getTime() + (-1 * 1000));
        expiresNow = "; expires=" + date.toGMTString();

        if (typeof(seconds) != 'undefined') {
            date.setTime(date.getTime() + (seconds * 1000));
            expires = '; expires=' + date.toGMTString();
        }

        var type = typeof(value);
        type = type.toLowerCase();
        if (type != 'undefined' && type != 'string') {
            value = JSON.stringify(value);
        }

        // fix scoping issues
        // keep writing the old cookie, but make it expire
        document.cookie = name + '=' + value + expiresNow + '; path=/';

        // now just set the right one
        document.cookie = name + '=' + value + expires + '; path=/; domain=' + domain;
    }

    ,getCookie: function(name) {
        name = name + '=';
        var carray = document.cookie.split(';'), value;

        for (var i=0; i<carray.length; i++) {
            var c = carray[i];
            while (c.charAt(0) == ' ') {
                c = c.substring(1, c.length);
            }
            if (c.indexOf(name) == 0) {
                value = c.substring(name.length, c.length);
                try {
                    value = JSON.parse(value);
                } catch(ex) {}

                return value;
            }
        }

        return null;
    }

    ,deleteCookie: function(name) {
        this.setCookie(name, '', -1);
        this.setCookie(name, '', -1, document.location.hostname);
    }

    ,mapAdobeVars: function(s) {
        var vars = {
            pageName		: 'Page - Analytics Pagename'
            ,channel		: 'Page - Section Name'
            ,campaign		: 'Campaign - ID'
            ,currencyCode	: 'Page - Currency Code'
            ,purchaseID		: 'Order - ID'
            ,prop1			: 'Visitor - Account ID'
            ,prop2			: 'Page - Product Name'
            ,prop4			: 'Page - Type'
            ,prop6			: 'Search - Type'
            ,prop7			: 'Search - Facet List'
            ,prop8			: 'Search - Feature Used'
            ,prop12			: 'Visitor - User ID'
            ,prop13			: 'Search - Sort Type'
            ,prop14			: 'Page - Load Time'
            ,prop15         : 'Support - Topic Name'
            ,prop16			: 'Page - Business Unit'
            ,prop21			: 'Search - Criteria'
            ,prop24			: 'Page - Language'
            ,prop25			: 'Page - Product Feature'
            ,prop28         : 'Support - Search Criteria'
            ,prop30			: 'Visitor - IP Address'
            ,prop33         : 'Page - Product Application Version'
            ,prop34         : 'Page - Website Extensions'
            ,prop60			: 'Search - Data Form Criteria'
            ,prop63			: 'Page - Extended Page Name'
            ,prop65         : 'Page - Online State'
            ,prop67         : 'Research Networks'
            ,prop40: 'Page - UX Properties'

            ,eVar3			: 'Search - Total Results'
            ,eVar7			: 'Visitor - Account Name'
            ,eVar15			: 'Event - Search Results Click Position'
            ,eVar19			: 'Search - Advanced Criteria'
            ,eVar21			: 'Promo - Clicked ID'
            ,eVar22			: 'Page - Test ID'
            ,eVar47			: 'Page - Test Subject ID'
            ,eVar27			: 'Event - AutoSuggest Search Data'
            ,eVar157		: 'Event - AutoSuggest Search Typed Term'
            ,eVar156		: 'Event - AutoSuggest Search Selected Term'
            ,eVar162		: 'Event - AutoSuggest Search Category'
            ,eVar163		: 'Event - AutoSuggest Search Details'
            ,eVar33			: 'Visitor - Access Type'
            ,eVar34			: 'Order - Promo Code'
            ,eVar39			: 'Order - Payment Method'
            ,eVar41			: 'Visitor - Industry'
            ,eVar42			: 'Visitor - SIS ID'
            ,eVar43			: 'Page - Error Type'
            ,eVar44			: 'Event - Updated User Fields'
            ,eVar48			: 'Email - Recipient ID'
            ,eVar51			: 'Email - Message ID'
            ,eVar52			: 'Visitor - Department ID'
            ,eVar53			: 'Visitor - Department Name'
            ,eVar60			: 'Search - Within Content Criteria'
            ,eVar61			: 'Search - Within Results Criteria'
            ,eVar62			: 'Search - Result Types'
            ,eVar74			: 'Page - Journal Info'
            ,eVar59			: 'Page - Journal Publisher'
            ,eVar76			: 'Email - Broadlog ID'
            ,eVar78			: 'Visitor - Details'
            ,eVar80         : 'Visitor - Usage Path Info'
            ,eVar102		: 'Form - Name'
            ,eVar103        : 'Event - Conversion Driver'
            ,eVar105        : 'Search - Current Page'
            ,eVar106        : 'Visitor - App Session ID'
            ,eVar107        : 'Page - Secondary Product Name'
            ,eVar117        : 'Search - Database'
            ,eVar126        : 'Page - Environment'
            ,eVar141        : 'Search - Criteria Original'
            ,eVar143        : 'Page - Tabs'
            ,eVar161        : 'Search - Channel'
            ,eVar169        : 'Search - Facet Operation'
            ,eVar173        : 'Search - Details'
            ,eVar174        : 'Campaign - Spredfast ID'
            ,eVar175        : 'Visitor - TMX Device ID'
            ,eVar176        : 'Visitor - TMX Request ID'
            ,eVar148        : 'Visitor - Platform Name'
            ,eVar149        : 'Visitor - Platform ID'
            ,eVar152        : 'Visitor - Product ID'
            ,eVar153        : 'Visitor - Superaccount ID'
            ,eVar154        : 'Visitor - Superaccount Name'
            ,eVar177        : 'Page - Context Domain'
            ,eVar189    : 'Page - Experimentation User Id'
            ,eVar190    : 'Page - Identity User'
            ,eVar199    : 'Page - ID+ Parameters'

            ,list2			: 'Page - Widget Names'
            ,list3			: 'Promo - IDs'
        };

        for (var i in vars) {
            s[i] = s[i] ? s[i] : _satellite.getVar(vars[i]);
        }
    }
};

// async support fallback
(function(w) {
	var eventBuffer = [];
	if(w.appData) {
		if(Array.isArray(w.appData)) {
			eventBuffer = w.appData;
		} else {
			console.error('Elsevier DataLayer "window.appData" must be specified as array');
			return;
		}
    }

	w.appData = [];

	var oldPush = w.appData.push;

	var appDataPush = function() {
        oldPush.apply(w.appData, arguments);
        for(var i=0; i<arguments.length; i++) {
            var data = arguments[i];
            if(data.event) {
                if(data.event == 'pageLoad') {
                    w.pageDataTracker.trackPageLoad(data);
                } else {
                    w.pageDataTracker.trackEvent(data.event, data);
                }
            }
        }
	};

	w.appData.push = appDataPush;
	for(var i=0; i<eventBuffer.length; i++) {
	    var data = eventBuffer[i];
	    w.appData.push(data);
	}
})(window);

</script><script>_satellite["_runScript1"](function(event, target, Promise) {
_satellite.logger.log("eventDispatcher: clearing tracking state");try{s.events="",s.linkTrackVars="",s.linkTrackEvents=""}catch(e){_satellite.logger.log("eventDispatcher: s object - could not reset state.")}try{dispatcherData=JSON.parse(event.detail),window.ddqueue=window.ddqueue||[],window.ddqueue.push(dispatcherData),window.eventData=dispatcherData.eventData,window.pageData=dispatcherData.pageData,_satellite.track(dispatcherData.eventName)}catch(e){_satellite.logger.log("eventDispatcher: exception"),_satellite.logger.log(e)}
});</script><script>_satellite["_runScript2"](function(event, target, Promise) {
_satellite.logger.log("eventDispatcher: clearing tracking state");try{s.events="",s.linkTrackVars="",s.linkTrackEvents=""}catch(e){_satellite.logger.log("eventDispatcher: s object - could not reset state.")}try{dispatcherData=JSON.parse(event.detail),window.ddqueue=window.ddqueue||[],window.ddqueue.push(dispatcherData),window.eventData=dispatcherData.eventData,window.pageData=dispatcherData.pageData,_satellite.track(dispatcherData.eventName)}catch(e){_satellite.logger.log("eventDispatcher: exception"),_satellite.logger.log(e)}
});</script><script>_satellite["_runScript3"](function(event, target, Promise) {
_satellite.logger.log("eventDispatcher: clearing tracking state");try{s.events="",s.linkTrackVars="",s.linkTrackEvents=""}catch(e){_satellite.logger.log("eventDispatcher: s object - could not reset state.")}try{dispatcherData=JSON.parse(event.detail),window.ddqueue=window.ddqueue||[],window.ddqueue.push(dispatcherData),window.eventData=dispatcherData.eventData,window.pageData=dispatcherData.pageData,_satellite.track(dispatcherData.eventName)}catch(e){_satellite.logger.log("eventDispatcher: exception"),_satellite.logger.log(e)}
});</script></body></html>