<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="b303ccb8-0e89-4a23-b593-d611b7484797" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867421013295;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;issue:issue:pii\:S0092867420X00262;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Evolutionary assembly of cooperating cell types in an animal chemical defense system">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)01329-5">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/acea6313-f93b-41cd-92bc-518f0d45a5a6/fx1.jpg">
    


        <meta property="og:description" content="The rove beetle defense gland is the result of coevolution between two secretory cell
types that work together to produce a chemical cocktail, illuminating how cell types
evolve cooperative interactions generating organ-level behaviors.">
    











        <meta name="twitter:title" content="Evolutionary assembly of cooperating cell types in an animal chemical defense system">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/acea6313-f93b-41cd-92bc-518f0d45a5a6/fx1.jpg">
    


        <meta name="twitter:description" content="The rove beetle defense gland is the result of coevolution between two secretory cell
types that work together to produce a chemical cocktail, illuminating how cell types
evolve cooperative interactions generating organ-level behaviors.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)01329-5">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Evolutionary assembly of cooperating cell types in an animal chemical defense system: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Evolutionary assembly of cooperating cell types in an animal chemical defense system"> <meta name="citation_date" content="2021/12/09"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="25"> <meta name="citation_firstpage" content="6138"> <meta name="citation_lastpage" content="6156.e28"> <meta name="citation_pmid" content="34890552"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421013295"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421013295"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421013295"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.11.014"> <meta name="citation_pmid" content="34890552"> <meta name="citation_author" content="Adrian Brückner"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Jean M. Badroos"><meta name="citation_author_orcid" content="0000-0002-0867-3686"><meta name="citation_author_institution" content="Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Robert W. Learsch"><meta name="citation_author_orcid" content="0000-0001-6329-5879"><meta name="citation_author_institution" content="Division of Engineering and Applied Science, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Mina Yousefelahiyeh"><meta name="citation_author_orcid" content="0000-0003-0365-3018"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Sheila A. Kitchen"><meta name="citation_author_orcid" content="0000-0003-4402-8139"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_author" content="Joseph Parker"><meta name="citation_author_email" content="joep@caltech.edu"><meta name="citation_author_orcid" content="0000-0001-9598-2454"><meta name="citation_author_institution" content="Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA"><meta name="citation_keywords" content="cell type evolution; biosynthetic pathway evolution; enzyme evolution; organ function; evolution of cooperation; single cell RNAseq; rove beetles; Dalotia coriaria"><meta name="citation_reference" content="citation_title=Deciphering signatures of mutational processes operative in human cancer; citation_author=L.B. Alexandrov; citation_author=S. Nik-Zainal; citation_author=D.C. Wedge; citation_author=P.J. Campbell; citation_author=M.R. Stratton; citation_journal_title=Cell Rep.; citation_volume=3; citation_firstpage=246; citation_lastpage=259; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Basic local alignment search tool; citation_author=S.F. Altschul; citation_author=W. Gish; citation_author=W. Miller; citation_author=E.W. Myers; citation_author=D.J. Lipman; citation_journal_title=J.&nbsp;Mol. Biol.; citation_volume=215; citation_firstpage=403; citation_lastpage=410; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=A new method for non-parametric multivariate analysis of variance; citation_author=M.J. Anderson; citation_journal_title=Austral Ecol.; citation_volume=26; citation_firstpage=32; citation_lastpage=46; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=FastQC: a quality control tool for high throughput sequence data; citation_author=S. Andrews; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Laccase 2 is the phenoloxidase gene required for beetle cuticle tanning; citation_author=Y. Arakane; citation_author=S. Muthukrishnan; citation_author=R.W. Beeman; citation_author=M.R. Kanost; citation_author=K.J. Kramer; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=11337; citation_lastpage=11342; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=The origin and evolution of cell types; citation_author=D. Arendt; citation_author=J.M. Musser; citation_author=C.V.H. Baker; citation_author=A. Bergman; citation_author=C. Cepko; citation_author=D.H. Erwin; citation_author=M. Pavlicev; citation_author=G. Schlosser; citation_author=S. Widder; citation_author=M.D. Laubichler; citation_author=G.P. Wagner; citation_journal_title=Nat. Rev. Genet.; citation_volume=17; citation_firstpage=744; citation_lastpage=757; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Insect fat body: energy, metabolism, and regulation; citation_author=E.L. Arrese; citation_author=J.L. Soulages; citation_journal_title=Annu. Rev. Entomol.; citation_volume=55; citation_firstpage=207; citation_lastpage=225; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Mini-review an insect-specific system for terrestrialization: Laccase-mediated cuticle formation; citation_author=T. Asano; citation_author=Y. Seto; citation_author=K. Hashimoto; citation_author=H. Kurushima; citation_journal_title=Insect Biochem. Mol. Biol.; citation_volume=108; citation_firstpage=61; citation_lastpage=70; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genomic analysis of the tryptome reveals molecular mechanisms of gland cell evolution; citation_author=L.S. Babonis; citation_author=J.F. Ryan; citation_author=C. Enjolras; citation_author=M.Q. Martindale; citation_journal_title=Evodevo; citation_volume=10; citation_firstpage=23; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Molecular spandrels: tests of adaptation at the genetic level; citation_author=R.D.H. Barrett; citation_author=H.E. Hoekstra; citation_journal_title=Nat. Rev. Genet.; citation_volume=12; citation_firstpage=767; citation_lastpage=780; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Matrix: sparse and dense matrix classes and methods. R Package Version; citation_author=D. Bates; citation_author=M. Maechler; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The lme4 package; citation_author=D. Bates; citation_author=D. Sarkar; citation_author=M.D. Bates; citation_author=L. Matrix; citation_journal_title=R Package Version; citation_volume=2; citation_firstpage=74; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Dimensionality reduction for visualizing single-cell data using UMAP; citation_author=E. Becht; citation_author=L. McInnes; citation_author=J. Healy; citation_author=C.-A. Dutertre; citation_author=I.W.H. Kwok; citation_author=L.G. Ng; citation_author=F. Ginhoux; citation_author=E.W. Newell; citation_journal_title=Nat. Biotechnol.; citation_volume=37; citation_firstpage=38; citation_lastpage=44; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing; citation_author=Y. Benjamini; citation_author=Y. Hochberg; citation_journal_title=J R Stat Soc Series B Stat Methodol; citation_volume=57; citation_firstpage=289; citation_lastpage=300; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Biology of Rove Beetles (Staphylinidae); citation_author=O. Betz; citation_author=U. Irmler; citation_author=J. Klimaszewski; citation_publication_date=2018; citation_publisher=Springer; "><meta name="citation_reference" content="citation_title=Specialized cells tag sexual and species identity in Drosophila melanogaster; citation_author=J.-C. Billeter; citation_author=J. Atallah; citation_author=J.J. Krupp; citation_author=J.G. Millar; citation_author=J.D. Levine; citation_journal_title=Nature; citation_volume=461; citation_firstpage=987; citation_lastpage=991; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The Flow of Cream through Narrow Glass Tubes; citation_author=G.W.S. Blair; citation_author=J.C. Hening; citation_author=A. Wagstaff; citation_journal_title=J.&nbsp;Phys. Chem.; citation_volume=43; citation_firstpage=853; citation_lastpage=864; citation_publication_date=1939; "><meta name="citation_reference" content="citation_title=Chemical Ecology, Biochemistry, and Molecular Biology of Insect Hydrocarbons; citation_author=G.J. Blomquist; citation_author=M.D. Ginzel; citation_journal_title=Annu. Rev. Entomol.; citation_volume=66; citation_firstpage=45; citation_lastpage=60; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Chemical Defenses of Arthropods; citation_author=M. Blum; citation_publication_date=1981; citation_publisher=Academic Press; "><meta name="citation_reference" content="citation_title=Molecular Mechanism of the Two-Component Suicidal Weapon of Neocapritermes taracua Old Workers; citation_author=T. Bourguignon; citation_author=J. Šobotník; citation_author=J. Brabcová; citation_author=D. Sillam-Dussès; citation_author=A. Buček; citation_author=J. Krasulová; citation_author=B. Vytisková; citation_author=Z. Demianová; citation_author=M. Mareš; citation_author=Y. Roisin; citation_author=H. Vogel; citation_journal_title=Mol. Biol. Evol.; citation_volume=33; citation_firstpage=809; citation_lastpage=819; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Near-optimal probabilistic RNA-seq quantification; citation_author=N.L. Bray; citation_author=H. Pimentel; citation_author=P. Melsted; citation_author=L. Pachter; citation_journal_title=Nat. Biotechnol.; citation_volume=34; citation_firstpage=525; citation_lastpage=527; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Random Forests; citation_author=L. Breiman; citation_journal_title=Mach. Learn.; citation_volume=45; citation_firstpage=5; citation_lastpage=32; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=A chemo-ecologists’ practical guide to compositional data analysis; citation_author=A. Brückner; citation_author=M. Heethoff; citation_journal_title=Chemoecology; citation_volume=27; citation_firstpage=33; citation_lastpage=46; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Molecular evolution of gland cell types and chemical interactions in animals; citation_author=A. Brückner; citation_author=J. Parker; citation_journal_title=J.&nbsp;Exp. Biol.; citation_volume=223; citation_number=Pt, Suppl 1; citation_firstpage=jeb211938; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=De novo biosynthesis of simple aromatic compounds by an arthropod (Archegozetes longisetosus); citation_author=A. Brückner; citation_author=M. Kaltenpoth; citation_author=M. Heethoff; citation_journal_title=Proc. Biol. Sci.; citation_volume=287; citation_firstpage=20201429; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Molecular evolutionary trends and biosynthesis pathways in the Oribatida revealed by the genome of Archegozetes longisetosus; citation_author=A. Brückner; citation_author=A.A. Barnett; citation_author=I.A. Antoshechkin; citation_author=S.A. Kitchen; citation_journal_title=Biorxiv; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database; citation_author=T. Brůna; citation_author=K.J. Hoff; citation_author=A. Lomsadze; citation_author=M. Stanke; citation_author=M. Borodovsky; citation_journal_title=Nar. Genom. Bioinform.; citation_volume=3; citation_firstpage=lqaa108; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Fast and sensitive protein alignment using DIAMOND; citation_author=B. Buchfink; citation_author=C. Xie; citation_author=D.H. Huson; citation_journal_title=Nat. Methods; citation_volume=12; citation_firstpage=59; citation_lastpage=60; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Integrating single-cell transcriptomic data across different conditions, technologies, and species; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=P. Smibert; citation_author=E. Papalexi; citation_author=R. Satija; citation_journal_title=Nat. Biotechnol.; citation_volume=36; citation_firstpage=411; citation_lastpage=420; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=boot: Bootstrap R (S-Plus) functions; citation_author=A. Canty; citation_author=B. Ripley; citation_journal_title=R Package Version; citation_volume=1; citation_firstpage=3; citation_lastpage=20; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=1,4-Benzoquinone antimicrobial agents against Staphylococcus aureus and Mycobacterium tuberculosis derived from scorpion venom; citation_author=E.N. Carcamo-Noriega; citation_author=S. Sathyamoorthi; citation_author=S. Banerjee; citation_author=E. Gnanamani; citation_author=M. Mendoza-Trujillo; citation_author=D. Mata-Espinosa; citation_author=R. Hernández-Pando; citation_author=J.I. Veytia-Bucheli; citation_author=L.D. Possani; citation_author=R.N. Zare; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=12642; citation_lastpage=12647; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=AnnotationForge: tools for building SQLite-based annotation data packages. R package version 1.36.0; citation_author=M. Carlson; citation_author=H. Pages; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Third-generation in situ hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust; citation_author=H.M.T. Choi; citation_author=M. Schwarzkopf; citation_author=M.E. Fornace; citation_author=A. Acharya; citation_author=G. Artavanis; citation_author=J. Stegmaier; citation_author=A. Cunha; citation_author=N.A. Pierce; citation_journal_title=Development; citation_volume=145; citation_firstpage=dev165753; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Detection of bacterial 16S rRNA and identification of four clinically important bacteria by real-time PCR; citation_author=R.J. Clifford; citation_author=M. Milillo; citation_author=J. Prestwood; citation_author=R. Quintero; citation_author=D.V. Zurawski; citation_author=Y.I. Kwak; citation_author=P.E. Waterman; citation_author=E.P. Lesho; citation_author=P. Mc Gann; citation_journal_title=PloS one; citation_volume=7; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=MITE Tracker: an accurate approach to identify miniature inverted-repeat transposable elements in large genomes; citation_author=J.M. Crescente; citation_author=D. Zavallo; citation_author=M. Helguera; citation_author=L.S. Vanzetti; citation_journal_title=BMC Bioinformatics; citation_volume=19; citation_firstpage=348; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=On the Origin of Species by Means of Natural Selection, or, the Preservation of Favoured Races in the Struggle for Life; citation_author=C. Darwin; citation_publication_date=1859; citation_publisher=John Murray; "><meta name="citation_reference" content="citation_title=The Blind Watchmaker: Why the Evidence of Evolution Reveals a Universe Without Design; citation_author=R. Dawkins; citation_publication_date=1986; citation_publisher=WW Norton &amp; Company; "><meta name="citation_reference" content="citation_title=Isopropylesters as Wetting Agents From the Defensive Secretion of the Rove Beetle Coprophilus striatulus F. (Coleoptera, Staphylinidae); citation_author=K. Dettner; citation_journal_title=Insect Biochem.; citation_volume=14; citation_firstpage=383; citation_lastpage=390; citation_publication_date=1984; "><meta name="citation_reference" content="citation_title=Solvent-dependent Variability of Effectiveness of Quinone-defensive Systems of Oxytelinae Beetles (Coleoptera: Staphylinidae); citation_author=K. Dettner; citation_journal_title=Entomol. Gen.; citation_volume=15; citation_firstpage=275; citation_lastpage=292; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Defensive secretions and exocrine glands in free-living staphylinid beetles—their bearing on phylogeny (Coleoptera: Staphylinidae); citation_author=K. Dettner; citation_journal_title=Biochem. Syst. Ecol.; citation_volume=21; citation_firstpage=143; citation_lastpage=162; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Dabbing and shooting of benzo- and naphthoquinone secretions: Defensive strategies of bark-inhabiting aleocharine (col.: Staphylinidae) and tenebrionid (col.: Tenebrionidae) beetle larvae; citation_author=K. Dettner; citation_journal_title=J.&nbsp;Chem. Ecol.; citation_volume=19; citation_firstpage=1337; citation_lastpage=1354; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Sticky secretion from two pairs of defensive glands of rove beetleDeleaster dichrous (Grav.) (Coleoptera: Staphylinidae); citation_author=K. Dettner; citation_author=G. Schwinger; citation_author=P. Wunderle; citation_journal_title=J.&nbsp;Chem. Ecol.; citation_volume=11; citation_firstpage=859; citation_lastpage=883; citation_publication_date=1985; "><meta name="citation_reference" content="citation_title=Pinch-off dynamics and dripping-onto-substrate (DoS) rheometry of complex fluids; citation_author=J. Dinic; citation_author=L.N. Jimenez; citation_author=V. Sharma; citation_journal_title=Lab Chip; citation_volume=17; citation_firstpage=460; citation_lastpage=473; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Scaling in pinch-off of generalized Newtonian fluids; citation_author=P. Doshi; citation_author=R. Suryo; citation_author=O.E. Yildirim; citation_author=G.H. McKinley; citation_author=O.A. Basaran; citation_journal_title=J Non-Newton Fluid; citation_volume=113; citation_firstpage=1; citation_lastpage=27; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Package ‘data. table.’ Extension of ‘data; citation_author=M. Dowle; citation_author=A. Srinivasan; citation_author=J. Gorecki; citation_author=M. Chirico; citation_author=P. Stetsenko; citation_author=T. Short; citation_author=S. Lianoglou; citation_author=E. Antonyan; citation_author=M. Bonsch; citation_author=H. Parsonage; citation_publication_date=2019; citation_publisher=Frame; "><meta name="citation_reference" content="citation_title=Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt; citation_author=S. Durinck; citation_author=P.T. Spellman; citation_author=E. Birney; citation_author=W. Huber; citation_journal_title=Nat. Protoc.; citation_volume=4; citation_firstpage=1184; citation_lastpage=1191; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Structure–function relationship among bacterial, fungal and plant laccases; citation_author=U.N. Dwivedi; citation_author=P. Singh; citation_author=V.P. Pandey; citation_author=A. Kumar; citation_journal_title=J.&nbsp;Mol. Catal. B Enzym.; citation_volume=68; citation_firstpage=117; citation_lastpage=128; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The approximation of one matrix by another of lower rank; citation_author=C. Eckart; citation_author=G. Young; citation_journal_title=Psychometrika; citation_volume=1; citation_firstpage=211; citation_lastpage=218; citation_publication_date=1936; "><meta name="citation_reference" content="citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy; citation_author=D.M. Emms; citation_author=S. Kelly; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=157; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=OrthoFinder: phylogenetic orthology inference for comparative genomics; citation_author=D.M. Emms; citation_author=S. Kelly; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=238; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=An efficient algorithm for large-scale detection of protein families; citation_author=A.J. Enright; citation_author=S. Van Dongen; citation_author=C.A. Ouzounis; citation_journal_title=Nucleic Acids Res.; citation_volume=30; citation_firstpage=1575; citation_lastpage=1584; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Novelty and Innovation in the History of Life; citation_author=D.H. Erwin; citation_journal_title=Curr. Biol.; citation_volume=25; citation_firstpage=R930; citation_lastpage=R940; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A conceptual framework of evolutionary novelty and innovation; citation_author=D.H. Erwin; citation_journal_title=Biol. Rev. Camb. Philos. Soc.; citation_volume=96; citation_firstpage=1; citation_lastpage=15; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Origin and evolution of the CYP4G subfamily in insects, cytochrome P450 enzymes involved in cuticular hydrocarbon synthesis; citation_author=R. Feyereisen; citation_journal_title=Mol. Phylogenet. Evol.; citation_volume=143; citation_firstpage=106695; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Birth-and-death evolution of the fatty acyl-CoA reductase (FAR) gene family and diversification of cuticular hydrocarbon synthesis in Drosophila; citation_author=C. Finet; citation_author=K. Slavik; citation_author=J. Pu; citation_author=S.B. Carroll; citation_author=H. Chung; citation_journal_title=Genome Biol. Evol.; citation_volume=11; citation_firstpage=1541; citation_lastpage=1551; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Effect displays in R for generalised linear models; citation_author=J. Fox; citation_journal_title=J.&nbsp;Stat. Softw.; citation_volume=8; citation_firstpage=1; citation_lastpage=27; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Package ‘car’; citation_author=J. Fox; citation_author=S. Weisberg; citation_author=D. Adler; citation_author=D. Bates; citation_author=G. Baud-Bovy; citation_author=S. Ellison; citation_author=D. Firth; citation_author=M. Friendly; citation_author=G. Gorjanc; citation_author=S. Graves; citation_publication_date=2012; citation_publisher=R Foundation for Statistical Computing; "><meta name="citation_reference" content="citation_title=Chemical Signalling in Beetles; citation_author=W. Francke; citation_author=K. Dettner; citation_journal_title=Top. Curr. Chem.; citation_volume=240; citation_firstpage=85; citation_lastpage=166; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=CD-HIT: accelerated for clustering the next-generation sequencing data; citation_author=L. Fu; citation_author=B. Niu; citation_author=Z. Zhu; citation_author=S. Wu; citation_author=W. Li; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=3150; citation_lastpage=3152; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome; citation_author=M.G. Grabherr; citation_author=B.J. Haas; citation_author=M. Yassour; citation_author=J.Z. Levin; citation_author=D.A. Thompson; citation_author=I. Amit; citation_author=X. Adiconis; citation_author=L. Fan; citation_author=R. Raychowdhury; citation_author=Q. Zeng; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=29; citation_firstpage=644; citation_lastpage=652; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The placenta as a model for understanding the origin and evolution of vertebrate organs; citation_author=O.W. Griffith; citation_author=G.P. Wagner; citation_journal_title=Nat. Ecol. Evol.; citation_volume=1; citation_firstpage=0072; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments; citation_author=B.J. Haas; citation_author=S.L. Salzberg; citation_author=W. Zhu; citation_author=M. Pertea; citation_author=J.E. Allen; citation_author=J. Orvis; citation_author=O. White; citation_author=C.R. Buell; citation_author=J.R. Wortman; citation_journal_title=Genome Biol.; citation_volume=9; citation_firstpage=R7; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis; citation_author=B.J. Haas; citation_author=A. Papanicolaou; citation_author=M. Yassour; citation_author=M. Grabherr; citation_author=P.D. Blood; citation_author=J. Bowden; citation_author=M.B. Couger; citation_author=D. Eccles; citation_author=B. Li; citation_author=M. Lieber; citation_author=et al.; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=1494; citation_lastpage=1512; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=PAST: Paleontological statistics software package for education and data analysis; citation_author=O. Hammer; citation_author=D.A. Harper; citation_author=P.D. Ryan; citation_journal_title=Palaeontol. Electronica; citation_volume=4; citation_firstpage=9; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=Regional and Functional Differentiation in the Insect Fact Body; citation_author=N.H. Haunerland; citation_author=P.D. Shirk; citation_journal_title=Annu. Rev. Entomol.; citation_volume=40; citation_firstpage=121; citation_lastpage=145; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Terminal Selectors of Neuronal Identity; citation_author=O. Hobert; citation_journal_title=Curr. Top Dev. Biol.; citation_volume=116; citation_firstpage=455; citation_lastpage=475; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A hydrophobic ratchet entrenches molecular complexes; citation_author=G.K.A. Hochberg; citation_author=Y. Liu; citation_author=E.G. Marklund; citation_author=B.P.H. Metzger; citation_author=A. Laganowsky; citation_author=J.W. Thornton; citation_journal_title=Nature; citation_volume=588; citation_firstpage=503; citation_lastpage=508; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Advances in deciphering the genetic basis of insect cuticular hydrocarbon biosynthesis and variation; citation_author=H. Holze; citation_author=L. Schrader; citation_author=J. Buellesbach; citation_journal_title=Heredity; citation_volume=126; citation_firstpage=219; citation_lastpage=234; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Beetle horns evolved from wing serial homologs; citation_author=Y. Hu; citation_author=D.M. Linz; citation_author=A.P. Moczek; citation_journal_title=Science; citation_volume=366; citation_firstpage=1004; citation_lastpage=1007; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Benzoquinone reveals a cysteine-dependent desensitization mechanism of TRPA1; citation_author=Y. Ibarra; citation_author=N.T. Blair; citation_journal_title=Mol. Pharmacol.; citation_volume=83; citation_firstpage=1120; citation_lastpage=1132; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Quantitative Visualization of Gene Expression in Mucoid and Nonmucoid Pseudomonas aeruginosa Aggregates Reveals Localized Peak Expression of Alginate in the Hypoxic Zone; citation_author=P. Jorth; citation_author=M.A. Spero; citation_author=J. Livingston; citation_author=D.K. Newman; citation_journal_title=MBio; citation_volume=10; citation_firstpage=e02622; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Analysis of Drosophila TRPA1 reveals an ancient origin for human chemical nociception; citation_author=K. Kang; citation_author=S.R. Pulver; citation_author=V.C. Panzano; citation_author=E.C. Chang; citation_author=L.C. Griffith; citation_author=D.L. Theobald; citation_author=P.A. Garrity; citation_journal_title=Nature; citation_volume=464; citation_firstpage=597; citation_lastpage=600; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability; citation_author=K. Katoh; citation_author=D.M. Standley; citation_journal_title=Mol. Biol. Evol.; citation_volume=30; citation_firstpage=772; citation_lastpage=780; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Biosynthesis of coenzyme Q in eukaryotes; citation_author=M. Kawamukai; citation_journal_title=Biosci. Biotechnol. Biochem.; citation_volume=80; citation_firstpage=23; citation_lastpage=33; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Cerebellar nuclei evolved by repeatedly duplicating a conserved cell-type set; citation_author=J.M. Kebschull; citation_author=E.B. Richman; citation_author=N. Ringach; citation_author=D. Friedmann; citation_author=E. Albarran; citation_author=S.S. Kolluru; citation_author=R.C. Jones; citation_author=W.E. Allen; citation_author=Y. Wang; citation_author=S.W. Cho; citation_author=et al.; citation_journal_title=Science; citation_volume=370; citation_firstpage=eabd5059; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=GeMoMa: Homology-Based Gene Prediction Utilizing Intron Position Conservation and RNA-seq Data; citation_author=J. Keilwagen; citation_author=F. Hartung; citation_author=J. Grau; citation_journal_title=Methods Mol. Biol.; citation_volume=1962; citation_firstpage=161; citation_lastpage=177; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Cell type innovation at the tips of the animal tree; citation_author=Y. Kishi; citation_author=J. Parker; citation_journal_title=Curr. Opin. Genet. Dev.; citation_volume=69; citation_firstpage=112; citation_lastpage=121; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells; citation_author=A.M. Klein; citation_author=L. Mazutis; citation_author=I. Akartuna; citation_author=N. Tallapragada; citation_author=A. Veres; citation_author=V. Li; citation_author=L. Peshkin; citation_author=D.A. Weitz; citation_author=M.W. Kirschner; citation_journal_title=Cell; citation_volume=161; citation_firstpage=1187; citation_lastpage=1201; citation_publication_date=2015; "><meta name="citation_reference" content="Kluyver, T., Ragan-Kelley, B., Pérez, F., Granger, B.E., Bussonnier, M., Frederic, J., Kelley, K., Hamrick, J., Grout, J., Corlay, S., et al. (2016). Jupyter Notebooks-a publishing format for reproducible computational workflows. In proceedings of the 20th International Conference on Electronic Publishing. pp. 87-90."><meta name="citation_reference" content="citation_title=Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq; citation_author=D. Kotliar; citation_author=A. Veres; citation_author=M.A. Nagy; citation_author=S. Tabrizi; citation_author=E. Hodis; citation_author=D.A. Melton; citation_author=P.C. Sabeti; citation_journal_title=eLife; citation_volume=8; citation_firstpage=e43803; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Oxytocin/vasopressin-like peptide inotocin regulates cuticular hydrocarbon synthesis and water balancing in ants; citation_author=A. Koto; citation_author=N. Motoyama; citation_author=H. Tahara; citation_author=S. McGregor; citation_author=M. Moriyama; citation_author=T. Okabe; citation_author=M. Miura; citation_author=L. Keller; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=116; citation_firstpage=5597; citation_lastpage=5606; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genomic Resources for Goniozus legneri, Aleochara bilineata and Paykullia maculata, Representing Three Independent Origins of the Parasitoid Lifestyle in Insects.; citation_author=K. Kraaijeveld; citation_author=P. Neleman; citation_author=J. Mariën; citation_author=E. de Meijer; citation_author=J. Ellers; citation_journal_title=G3 (Bethesda); citation_volume=9; citation_firstpage=987; citation_lastpage=991; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=BlobTools: Interrogation of genome assemblies.; citation_author=D.R. Laetsch; citation_author=M.L. Blaxter; citation_journal_title=F1000Res.; citation_volume=6; citation_firstpage=1287; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Evolution of the vertebrate eye: opsins, photoreceptors, retina and eye cup.; citation_author=T.D. Lamb; citation_author=S.P. Collin; citation_author=E.N. Pugh; citation_journal_title=Nat. Rev. Neurosci.; citation_volume=8; citation_firstpage=960; citation_lastpage=976; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices.; citation_author=D. Li; citation_author=R. Luo; citation_author=C.-M. Liu; citation_author=C.-M. Leung; citation_author=H.-F. Ting; citation_author=K. Sadakane; citation_author=H. Yamashita; citation_author=T.-W. Lam; citation_journal_title=Methods; citation_volume=102; citation_firstpage=3; citation_lastpage=11; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Classification and regression by randomForest; citation_author=A. Liaw; citation_author=M. Wiener; citation_journal_title=R News; citation_volume=2/3; citation_firstpage=18; citation_lastpage=22; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Dual evolutionary origin of insect wings&nbsp;supported by an investigation of the abdominal wing serial homologs in Tribolium; citation_author=D.M. Linz; citation_author=Y. Tomoyasu; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E658; citation_lastpage=E667; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A molecular test of platyhelminth phylogeny: inferences from partial 28S rDNA sequences; citation_author=M.K. Litvaitis; citation_author=K. Rohde; citation_journal_title=Invertebrate Biology; citation_volume=118; citation_firstpage=42; citation_lastpage=56; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=How a neutral evolutionary ratchet can build cellular complexity; citation_author=J. Lukeš; citation_author=J.M. Archibald; citation_author=P.J. Keeling; citation_author=W.F. Doolittle; citation_author=M.W. Gray; citation_journal_title=IUBMB Life; citation_volume=63; citation_firstpage=528; citation_lastpage=537; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data; citation_author=A.T.L. Lun; citation_author=S. Riesenfeld; citation_author=T. Andrews; citation_author=T.P. Dao; citation_author=T. Gomes; citation_author=J.C. Marioni; citation_author=1st Human Cell Atlas Jamboree; citation_journal_title=Genome Biol.; citation_volume=20; citation_firstpage=63; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler; citation_author=R. Luo; citation_author=B. Liu; citation_author=Y. Xie; citation_author=Z. Li; citation_author=W. Huang; citation_author=J. Yuan; citation_author=G. He; citation_author=Y. Chen; citation_author=Q. Pan; citation_author=Y. Liu; citation_author=et al.; citation_journal_title=Gigascience; citation_volume=1; citation_firstpage=18; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The frailty of adaptive hypotheses for the origins of organismal complexity; citation_author=M. Lynch; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_number=Suppl 1; citation_firstpage=8597; citation_lastpage=8604; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The development and functions of oenocytes; citation_author=R. Makki; citation_author=E. Cinnamon; citation_author=A.P. Gould; citation_journal_title=Annu. Rev. Entomol.; citation_volume=59; citation_firstpage=405; citation_lastpage=425; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology; citation_author=J.C. Marioni; citation_author=D. Arendt; citation_journal_title=Annu. Rev. Cell Dev. Biol.; citation_volume=33; citation_firstpage=537; citation_lastpage=553; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Extensional rheometry with a handheld mobile device; citation_author=K.A. Marshall; citation_author=A.M. Liedtke; citation_author=A.H. Todt; citation_author=T.W. Walker; citation_journal_title=Exp. Fluids; citation_volume=58; citation_firstpage=69; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; citation_author=M. Martin; citation_journal_title=EMBnet. J.; citation_volume=17; citation_firstpage=10; citation_lastpage=12; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Deep-Time Convergence in Rove Beetle Symbionts of Army Ants; citation_author=M. Maruyama; citation_author=J. Parker; citation_journal_title=Curr. Biol.; citation_volume=27; citation_firstpage=920; citation_lastpage=926; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction; citation_author=L. McInnes; citation_author=J. Healy; citation_author=J. Melville; citation_journal_title=arXiv; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The barcode, UMI, set format and BUStools; citation_author=P. Melsted; citation_author=V. Ntranos; citation_author=L. Pachter; citation_journal_title=Bioinformatics; citation_volume=35; citation_firstpage=4472; citation_lastpage=4473; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Modular, efficient and constant-memory single-cell RNA-seq preprocessing; citation_author=P. Melsted; citation_author=A.S. Booeshaghi; citation_author=L. Liu; citation_author=F. Gao; citation_author=L. Lu; citation_author=K.H.J. Min; citation_author=E. da Veiga Beltrame; citation_author=K.E. Hjörleifsson; citation_author=J. Gehring; citation_author=L. Pachter; citation_journal_title=Nat. Biotechnol.; citation_volume=39; citation_firstpage=813; citation_lastpage=818; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Package ‘vcd.’ Visualizing Categorical Data. R Package Version 1–4; citation_author=D. Meyer; citation_author=A. Zeileis; citation_author=K. Hornik; citation_author=F. Gerber; citation_author=M. Friendly; citation_author=M.D. Meyer; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Control of compartment development by the engrailed gene in Drosophila; citation_author=G. Morata; citation_author=P.A. Lawrence; citation_journal_title=Nature; citation_volume=255; citation_firstpage=614; citation_lastpage=617; citation_publication_date=1975; "><meta name="citation_reference" content="citation_title=Package AnnotationHub; citation_author=M. Morgan; citation_author=M. Carlson; citation_author=D. Tenenbaum; citation_author=S. Arora; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=BUSpaRse: kallisto | bustools R utilities, R Package Version 1.4.2; citation_author=L. Moses; citation_author=L. Pachter; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Profiling cellular diversity in sponges informs animal cell type and nervous system evolution; citation_author=J.M. Musser; citation_author=K.J. Schippers; citation_author=M. Nickel; citation_author=G. Mizzon; citation_author=A.B. Kohn; citation_author=C. Pape; citation_author=J.U. Hammel; citation_author=F. Wolf; citation_author=C. Liang; citation_author=A. Hernández-Plaza; citation_author=et al.; citation_journal_title=Science; citation_volume=374; citation_firstpage=717; citation_lastpage=723; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies; citation_author=L.-T. Nguyen; citation_author=H.A. Schmidt; citation_author=A. von Haeseler; citation_author=B.Q. Minh; citation_journal_title=Mol. Biol. Evol.; citation_volume=32; citation_firstpage=268; citation_lastpage=274; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Cuticle formation and pigmentation in beetles; citation_author=M.Y. Noh; citation_author=S. Muthukrishnan; citation_author=K.J. Kramer; citation_author=Y. Arakane; citation_journal_title=Curr. Opin. Insect Sci.; citation_volume=17; citation_firstpage=1; citation_lastpage=9; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Furcation and fusion: The phylogenetics of evolutionary novelty; citation_author=T.H. Oakley; citation_journal_title=Dev. Biol.; citation_volume=431; citation_firstpage=69; citation_lastpage=76; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=How Complexity Originates: The Evolution of Animal Eyes; citation_author=T.H. Oakley; citation_author=D.I. Speiser; citation_journal_title=Annu. Rev. Ecol. Evol. Syst.; citation_volume=46; citation_firstpage=1; citation_lastpage=24; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Myrmecophily in beetles (Coleoptera): evolutionary patterns and biological mechanisms; citation_author=J. Parker; citation_journal_title=Myrmecol. News; citation_volume=22; citation_firstpage=65; citation_lastpage=108; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Staphylinids; citation_author=J. Parker; citation_journal_title=Curr. Biol.; citation_volume=27; citation_firstpage=R49; citation_lastpage=R51; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Hox-logic of body plan innovations for social insect symbiosis in rove beetles; citation_author=J. Parker; citation_author=K.T. Eldredge; citation_author=I. Thomas; citation_author=R. Coleman; citation_author=S. Davis; citation_journal_title=bioRxiv; citation_firstpage=198945; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Diversity and evolution of the transposable element repertoire in arthropods with particular reference to insects; citation_author=M. Petersen; citation_author=D. Armisén; citation_author=R.A. Gibbs; citation_author=L. Hering; citation_author=A. Khila; citation_author=G. Mayer; citation_author=S. Richards; citation_author=O. Niehuis; citation_author=B. Misof; citation_journal_title=BMC Evol. Biol.; citation_volume=19; citation_firstpage=11; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Gene knockdown analysis by double-stranded RNA injection; citation_inbook_title=Molecular methods for evolutionary genetics; citation_author=B.N. Philip; citation_author=Y. Tomoyasu; citation_firstpage=471; citation_lastpage=497; citation_publication_date=2012; citation_publisher=Humana Press; "><meta name="citation_reference" content="citation_title=A polyketide synthase-peptide synthetase gene cluster from an uncultured bacterial symbiont of Paederus beetles; citation_author=J. Piel; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=99; citation_firstpage=14002; citation_lastpage=14007; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Differential analysis of RNA-seq incorporating quantification uncertainty; citation_author=H. Pimentel; citation_author=N.L. Bray; citation_author=S. Puente; citation_author=P. Melsted; citation_author=L. Pachter; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=687; citation_lastpage=690; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Helping plants to deal with insects: the role of beneficial soil-borne microbes; citation_author=A. Pineda; citation_author=S.-J. Zheng; citation_author=J.J.A. van Loon; citation_author=C.M.J. Pieterse; citation_author=M. Dicke; citation_journal_title=Trends Plant Sci.; citation_volume=15; citation_firstpage=507; citation_lastpage=514; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Linear and nonlinear mixed effects models; citation_author=J. Pinheiro; citation_author=D. Bates; citation_journal_title=R Package Version; citation_volume=3; citation_firstpage=1; citation_lastpage=89; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Yeast and rat Coq3 and Escherichia coli UbiG polypeptides catalyze both O-methyltransferase steps in coenzyme Q biosynthesis; citation_author=W.W. Poon; citation_author=R.J. Barkovich; citation_author=A.Y. Hsu; citation_author=A. Frankel; citation_author=P.T. Lee; citation_author=J.N. Shepherd; citation_author=D.C. Myles; citation_author=C.F. Clarke; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=274; citation_firstpage=21665; citation_lastpage=21672; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=FastTree 2--approximately maximum-likelihood trees for large alignments; citation_author=M.N. Price; citation_author=P.S. Dehal; citation_author=A.P. Arkin; citation_journal_title=PLoS ONE; citation_volume=5; citation_firstpage=e9490; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; citation_author=K.D. Pruitt; citation_author=T. Tatusova; citation_author=D.R. Maglott; citation_journal_title=Nucleic Acids Res.; citation_volume=35; citation_firstpage=D61; citation_lastpage=D65; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Redundans: an assembly pipeline for highly heterozygous genomes; citation_author=L.P. Pryszcz; citation_author=T. Gabaldón; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_firstpage=e113; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=An insect-specific P450 oxidative decarbonylase for cuticular hydrocarbon biosynthesis; citation_author=Y. Qiu; citation_author=C. Tittiger; citation_author=C. Wicker-Thomas; citation_author=G. Le Goff; citation_author=S. Young; citation_author=E. Wajnberg; citation_author=T. Fricaux; citation_author=N. Taquet; citation_author=G.J. Blomquist; citation_author=R. Feyereisen; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=109; citation_firstpage=14858; citation_lastpage=14863; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The Shape of Life: Genes, Development, and the Evolution of Animal Form; citation_author=R. Raff; citation_publication_date=1996; citation_publisher=University of Chicago Press; "><meta name="citation_reference" content="citation_title=FigTree v1.4.1; citation_author=A. Rambaut; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Package ‘mass.’; citation_author=B. Ripley; citation_author=B. Venables; citation_author=D.M. Bates; citation_author=K. Hornik; citation_author=A. Gebhardt; citation_author=D. Firth; citation_author=M.B. Ripley; citation_journal_title=Cran R; citation_volume=538; citation_firstpage=113; citation_lastpage=120; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing. R Foundation for Statistical Computing; citation_author=R Core Team; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=VSEARCH: a versatile open source tool for metagenomics; citation_author=T. Rognes; citation_author=T. Flouri; citation_author=B. Nichols; citation_author=C. Quince; citation_author=F. Mahé; citation_journal_title=PeerJ; citation_volume=4; citation_firstpage=e2584; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Dripping-onto-substrate capillary breakup extensional rheometry of low-viscosity printing inks; citation_author=M. Rosello; citation_author=S. Sur; citation_author=B. Barbet; citation_author=J.P. Rothstein; citation_journal_title=J.&nbsp;Non-Newton. Fluid; citation_volume=266; citation_firstpage=160; citation_lastpage=170; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Channels and transporters in salivary glands; citation_author=E. Roussa; citation_journal_title=Cell Tissue Res.; citation_volume=343; citation_firstpage=263; citation_lastpage=287; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Lattice: Multivariate Data Visualization with R; citation_author=D. Sarkar; citation_publication_date=2008; citation_publisher=Springer Science &amp; Business Media; "><meta name="citation_reference" content="citation_title=Microbiology, genomics, and clinical significance of the Pseudomonas fluorescens species complex, an unappreciated colonizer of humans; citation_author=B.S. Scales; citation_author=R.P. Dickson; citation_author=J.J. LiPuma; citation_author=G.B. Huffnagle; citation_journal_title=Clin. Microbiol. Rev.; citation_volume=27; citation_firstpage=927; citation_lastpage=948; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A Short History of Nearly Every Sense-The Evolutionary History of Vertebrate Sensory Cell Types; citation_author=G. Schlosser; citation_journal_title=Integr. Comp. Biol.; citation_volume=58; citation_firstpage=301; citation_lastpage=316; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The Evolution of Organ Systems; citation_author=A. Schmidt-Rhaesa; citation_publication_date=2007; citation_publisher=Oxford University Press; "><meta name="citation_reference" content="citation_title=Deep homology and the origins of evolutionary novelty; citation_author=N. Shubin; citation_author=C. Tabin; citation_author=S. Carroll; citation_journal_title=Nature; citation_volume=457; citation_firstpage=818; citation_lastpage=823; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs; citation_author=F.A. Simão; citation_author=R.M. Waterhouse; citation_author=P. Ioannidis; citation_author=E.V. Kriventseva; citation_author=E.M. Zdobnov; citation_journal_title=Bioinformatics; citation_volume=31; citation_firstpage=3210; citation_lastpage=3212; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Repeat-Masker Open-3.0; citation_author=A.F. Smit; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=RepeatModeler Open-1.0; citation_author=A.F. Smit; citation_author=R. Hubley; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=The animal fatty acid synthase: one gene, one polypeptide, seven enzymes; citation_author=S. Smith; citation_journal_title=FASEB J.; citation_volume=8; citation_firstpage=1248; citation_lastpage=1259; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Structural and functional organization of the animal fatty acid synthase; citation_author=S. Smith; citation_author=A. Witkowski; citation_author=A.K. Joshi; citation_journal_title=Prog. Lipid Res.; citation_volume=42; citation_firstpage=289; citation_lastpage=317; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Measurement of relaxation times in extensional flow of weakly viscoelastic polymer solutions; citation_author=P.C. Sousa; citation_author=E.J. Vega; citation_author=R.G. Sousa; citation_author=J.M. Montanero; citation_author=M.A. Alves; citation_journal_title=Rheol. Acta; citation_volume=56; citation_firstpage=11; citation_lastpage=20; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel; citation_author=M. Stanke; citation_author=S. Waack; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=Suppl 2; citation_firstpage=ii215; citation_lastpage=ii225; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Fatty acids in insects: Composition, metabolism, and biological significance; citation_author=D.W. Stanley-Samuelson; citation_author=R.A. Jurenka; citation_author=C. Cripps; citation_author=G.J. Blomquist; citation_author=M. de Renobales; citation_journal_title=Arch. Insect Biochem.; citation_volume=9; citation_firstpage=1; citation_lastpage=33; citation_publication_date=1988; "><meta name="citation_reference" content="citation_title=Biochemistry of Mitochondrial Coenzyme Q Biosynthesis; citation_author=J.A. Stefely; citation_author=D.J. Pagliarini; citation_journal_title=Trends Biochem. Sci.; citation_volume=42; citation_firstpage=824; citation_lastpage=843; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance; citation_author=J.L.M. Steidle; citation_author=K. Dettner; citation_journal_title=Syst. Entomol.; citation_volume=18; citation_firstpage=149; citation_lastpage=168; citation_publication_date=1993; "><meta name="citation_reference" content="citation_title=Comprehensive Integration of Single-Cell Data; citation_author=T. Stuart; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=C. Hafemeister; citation_author=E. Papalexi; citation_author=W.M. Mauck; citation_author=Y. Hao; citation_author=M. Stoeckius; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1888; citation_lastpage=1902.e21; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Insights into how development and life-history dynamics shape the evolution of venom; citation_author=J.M. Surm; citation_author=Y. Moran; citation_journal_title=Evodevo; citation_volume=12; citation_firstpage=1; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Developmental regulation of silk gene expression in Bombyx mori; citation_inbook_title=Molecular Insect Science; citation_author=Y. Suzuki; citation_author=S. Takiya; citation_author=T. Suzuki; citation_author=C. Hui; citation_author=K. Matsuno; citation_author=M. Fukuta; citation_author=T. Nagata; citation_author=K. Ueno; citation_firstpage=83; citation_lastpage=89; citation_publication_date=1990; citation_publisher=SpringerLink; "><meta name="citation_reference" content="citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training; citation_author=V. Ter-Hovhannisyan; citation_author=A. Lomsadze; citation_author=Y.O. Chernoff; citation_author=M. Borodovsky; citation_journal_title=Genome Res.; citation_volume=18; citation_firstpage=1979; citation_lastpage=1990; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Larval RNAi in Tribolium (Coleoptera) for analyzing adult development; citation_author=Y. Tomoyasu; citation_author=R.E. Denell; citation_journal_title=Dev. Genes Evol.; citation_volume=214; citation_firstpage=575; citation_lastpage=578; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=The biosynthetic diversity of the animal world; citation_author=J.P. Torres; citation_author=E.W. Schmidt; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=294; citation_firstpage=17684; citation_lastpage=17692; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Animal biosynthesis of complex polyketides in a photosynthetic partnership; citation_author=J.P. Torres; citation_author=Z. Lin; citation_author=J.M. Winter; citation_author=P.J. Krug; citation_author=E.W. Schmidt; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=2882; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=New high-cloning-efficiency vectors for complementation studies and recombinant protein overproduction in Escherichia coli and Salmonella enterica; citation_author=C.M. VanDrisse; citation_author=J.C. Escalante-Semerena; citation_journal_title=Plasmid; citation_volume=86; citation_firstpage=1; citation_lastpage=6; citation_publication_date=2016; "><meta name="citation_reference" content="Van Rossum, G., &amp; Drake, F. L. (2000). Python reference manual. (iUniverse)."><meta name="citation_reference" content="citation_title=Habitat structure and prey aggregation determine the functional response in a soil predator–prey interaction; citation_author=O. Vucic-Pestic; citation_author=K. Birkhofer; citation_author=B.C. Rall; citation_author=S. Scheu; citation_author=U. Brose; citation_journal_title=Pedobiologia (Jena); citation_volume=53; citation_firstpage=307; citation_lastpage=312; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Evolutionary novelties; citation_author=G.P. Wagner; citation_author=V.J. Lynch; citation_journal_title=Curr. Biol.; citation_volume=20; citation_firstpage=R48; citation_lastpage=R52; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Benzoquinones in the defensive secretion of a bug (Pamillia behrensii): a common chemical trait retrieved in the Heteroptera; citation_author=J.M. Wagner; citation_author=T.H. Naragon; citation_author=A. Brückner; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.12.11.421891; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Fatty acid synthesis and its regulation; citation_author=S.J. Wakil; citation_author=J.K. Stoops; citation_author=V.C. Joshi; citation_journal_title=Annu. Rev. Biochem.; citation_volume=52; citation_firstpage=537; citation_lastpage=579; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=The Complexity of Making Ubiquinone; citation_author=Y. Wang; citation_author=S. Hekimi; citation_journal_title=Trends Endocrinol. Metab.; citation_volume=30; citation_firstpage=929; citation_lastpage=943; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Discovering Pathway and Cell Type Signatures in Transcriptomic Compendia with Machine Learning; citation_author=G.P. Way; citation_author=C.S. Greene; citation_journal_title=Annu. Rev. Biomed. Data Sci.; citation_volume=2; citation_firstpage=1; citation_lastpage=17; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Developmental Plasticity and Evolution; citation_author=M.J. West-Eberhard; citation_publication_date=2003; citation_publisher=Oxford University Press; "><meta name="citation_reference" content="citation_title=ggplot2; citation_author=H. Wickham; citation_journal_title=Wiley Interdiscip. Rev. Comput. Stat.; citation_volume=3; citation_firstpage=180; citation_lastpage=185; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=dbplyr: A “dplyr” Back End for Databases. R&nbsp;package Version 1.4. 0; citation_author=H. Wickham; citation_author=E. Ruiz; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Welcome to the Tidyverse; citation_author=H. Wickham; citation_author=M. Averick; citation_author=J. Bryan; citation_author=W. Chang; citation_author=L.D. McGowan; citation_author=R. François; citation_author=G. Grolemund; citation_author=A. Hayes; citation_author=L. Henry; citation_author=J. Hester; citation_journal_title=J.&nbsp;Open Source Softw.; citation_volume=4; citation_firstpage=1686; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Package ‘cowplot’: Streamlined Plot Theme and Plot Annotations for ‘ggplot2.’ CRAN; citation_author=C. Wilke; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=clusterProfiler: an R package for comparing biological themes among gene clusters; citation_author=G. Yu; citation_author=L.-G. Wang; citation_author=Y. Han; citation_author=Q.-Y. He; citation_journal_title=OMICS; citation_volume=16; citation_firstpage=284; citation_lastpage=287; citation_publication_date=2012; "><meta name="citation_abstract" content="<h2>Summary</h2><p>How the functions of multicellular organs emerge from the underlying evolution of cell types is poorly understood. We deconstructed evolution of an organ novelty: a rove beetle gland that secretes a defensive cocktail. We show how gland function arose via assembly of two cell types that manufacture distinct compounds. One cell type, comprising a chemical reservoir within the abdomen, produces alkane and ester compounds. We demonstrate that this cell type is a hybrid of cuticle cells and ancient pheromone and adipocyte-like cells, executing its function via a mosaic of enzymes from each parental cell type. The second cell type synthesizes benzoquinones using a chimera of conserved cellular energy and cuticle formation pathways. We show that evolution of each cell type was shaped by coevolution between the two cell types, yielding a potent secretion that confers adaptive value. Our findings illustrate how cooperation between cell types arises, generating new, organ-level behaviors.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=9314&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)01329-5&amp;be=3481&amp;fe=9152&amp;dc=6735&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749592101242,%22n%22:0,%22f%22:2852,%22dn%22:2853,%22dne%22:2853,%22c%22:2853,%22s%22:2853,%22ce%22:2853,%22rq%22:2853,%22rp%22:3465,%22rpe%22:4187,%22dl%22:3470,%22di%22:4630,%22ds%22:6735,%22de%22:6997,%22dc%22:9151,%22l%22:9151,%22le%22:9296%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><style></style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c1D442 {
  padding: 0.705em 0.763em 0.022em 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c35 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c41 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c3BC {
  padding: 0.442em 0.53em 0.216em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c1D706 {
  padding: 0.694em 0.583em 0.013em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c64 {
  padding: 0.694em 0.556em 0.011em 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c77 {
  padding: 0.431em 0.722em 0.011em 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c72 {
  padding: 0.442em 0.392em 0 0;
}

mjx-c.mjx-c65 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c67 {
  padding: 0.453em 0.5em 0.206em 0;
}

mjx-c.mjx-c75 {
  padding: 0.442em 0.556em 0.011em 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c61 {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c74 {
  padding: 0.615em 0.389em 0.011em 0;
}

mjx-c.mjx-c3A {
  padding: 0.431em 0.278em 0 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D457 {
  padding: 0.661em 0.412em 0.205em 0;
}

mjx-c.mjx-c3C {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c3B {
  padding: 0.431em 0.278em 0.193em 0;
}

mjx-c.mjx-c1D462 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c1D459 {
  padding: 0.694em 0.298em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c1D445 {
  padding: 0.683em 0.759em 0.022em 0;
}

mjx-c.mjx-c1D44C {
  padding: 0.683em 0.762em 0 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)01329-5"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)01329-5"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-541004" aria-expanded="false" aria-controls="login-links-541004" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-541004" id="login-links--541004" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-541004" data-opens="#links-dialog-541004" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-541004" aria-modal="true"><button data-closes="#links-dialog-541004" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.11.014">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)01329-5" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0026-2"><span>Volume 184</span>, <span>Issue 25</span></a></span><span class="meta-panel__pages"><span>p6138-6156.e28</span></span><span class="meta-panel__onlineDate">December 09, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Evolutionary assembly of cooperating cell types in an animal chemical defense system</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Adrian</span> <span property="familyName">Brückner</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Adrian</span> <span property="familyName">Brückner</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Br%C3%BCckner/Adrian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Jean M.</span> <span property="familyName">Badroos</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jean M.</span> <span property="familyName">Badroos</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0867-3686" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0867-3686</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Badroos/Jean+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Robert W.</span> <span property="familyName">Learsch</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Robert W.</span> <span property="familyName">Learsch</span> <a class="orcid-id" href="https://orcid.org/0000-0001-6329-5879" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-6329-5879</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Engineering and Applied Science, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Learsch/Robert+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Mina</span> <span property="familyName">Yousefelahiyeh</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mina</span> <span property="familyName">Yousefelahiyeh</span> <a class="orcid-id" href="https://orcid.org/0000-0003-0365-3018" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-0365-3018</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yousefelahiyeh/Mina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Sheila A.</span> <span property="familyName">Kitchen</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sheila A.</span> <span property="familyName">Kitchen</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4402-8139" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4402-8139</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kitchen/Sheila+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Joseph</span> <span property="familyName">Parker</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joseph</span> <span property="familyName">Parker</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9598-2454" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9598-2454</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:joep@caltech.edu" property="email" title="Send email to Joseph Parker">joep@caltech.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Parker/Joseph" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">4</a></sup> <a id="eadd0010" href="mailto:joep@caltech.edu" property="email" title="Send email to Joseph Parker">joep@caltech.edu</a></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Division of Biology and Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Division of Chemistry and Chemical Engineering, California Institute of Technology, Pasadena, CA 91125, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Division of Engineering and Applied Science, California Institute of Technology, Pasadena, CA 91125, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">4</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received June 23, 2021</span>; <span>Revised September 29, 2021</span>; <span>Accepted November 10, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.11.014" property="sameAs" target="_blank">10.1016/j.cell.2021.11.014 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421013295" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.11.014" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0026-2"><img src="/cms/asset/0540b9ed-f4ad-4863-9830-19ca05143514/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 25"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2901329-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901329-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.014">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2901329-5" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.11.014&amp;pii=S0092-8674%2821%2901329-5" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2901329-5" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.11.014">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.11.014" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421013295&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421013295&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=25&amp;pii=S0092-8674%2820%29X0026-2" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)01332-5" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)01327-1" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (7)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-relations"><div class="relation"><div class="relation--head">Correction: </div><div class="relation--body">Errata<div class="content--title"><a href="/cell/fulltext/S0092-8674(22)00323-3">Evolutionary assembly of cooperating cell types in an animal chemical defense system</a></div><div class="content--contributors">Adrian Brückner, Jean M. Badroos, Robert W. Learsch ...</div><div class="content--publishDate"> March 31, 2022</div></div></div></div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">A rove beetle defense gland reveals how evolution builds new organs</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Two secretory cell types work together, one making a solid toxin, the second a solvent</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">New cellular functions evolved by combining ancient transcriptome modules and pathways</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Evolution of each cell type was shaped by adaptive coevolution between the cell types</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">How the functions of multicellular organs emerge from the underlying evolution of cell types is poorly understood. We deconstructed evolution of an organ novelty: a rove beetle gland that secretes a defensive cocktail. We show how gland function arose via assembly of two cell types that manufacture distinct compounds. One cell type, comprising a chemical reservoir within the abdomen, produces alkane and ester compounds. We demonstrate that this cell type is a hybrid of cuticle cells and ancient pheromone and adipocyte-like cells, executing its function via a mosaic of enzymes from each parental cell type. The second cell type synthesizes benzoquinones using a chimera of conserved cellular energy and cuticle formation pathways. We show that evolution of each cell type was shaped by coevolution between the two cell types, yielding a potent secretion that confers adaptive value. Our findings illustrate how cooperation between cell types arises, generating new, organ-level behaviors.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/1a4067a7-cb35-4543-8b11-b0c51822b6b8/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.11.014/asset/acea6313-f93b-41cd-92bc-518f0d45a5a6/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/1a4067a7-cb35-4543-8b11-b0c51822b6b8/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;cell+type+evolution&quot;" alt="Search on this keyword" rel="nofollow">cell type evolution</a></li><li><a href="/action/doSearch?AllField=&quot;biosynthetic+pathway+evolution&quot;" alt="Search on this keyword" rel="nofollow">biosynthetic pathway evolution</a></li><li><a href="/action/doSearch?AllField=&quot;enzyme+evolution&quot;" alt="Search on this keyword" rel="nofollow">enzyme evolution</a></li><li><a href="/action/doSearch?AllField=&quot;organ+function&quot;" alt="Search on this keyword" rel="nofollow">organ function</a></li><li><a href="/action/doSearch?AllField=&quot;evolution+of+cooperation&quot;" alt="Search on this keyword" rel="nofollow">evolution of cooperation</a></li><li><a href="/action/doSearch?AllField=&quot;single+cell+RNAseq&quot;" alt="Search on this keyword" rel="nofollow">single cell RNAseq</a></li><li><a href="/action/doSearch?AllField=&quot;rove+beetles&quot;" alt="Search on this keyword" rel="nofollow">rove beetles</a></li><li><a href="/action/doSearch?AllField=&quot;Dalotia+coriaria&quot;" alt="Search on this keyword" rel="nofollow"><i>Dalotia coriaria</i></a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0035" role="paragraph">Animal diversification has been shaped by the evolution of new organs with novel functional properties (<span class="dropBlock reference-citations"><a id="body-ref-sref120" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120" aria-controls="sref120">Raff, 1996</a><div class="dropBlock__holder" data-db-target-of="sref120"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Raff, R.</div><div><strong>The Shape of Life: Genes, Development, and the Evolution of Animal Form</strong></div><div>University of Chicago Press, 1996</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.RaffThe+Shape+of+Life%3A+Genes%2C+Development%2C+and+the+Evolution+of+Animal+Form1996University+of+Chicago+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Structures such as eyes, skeletons and brains were multicellular innovations that opened new ecological frontiers and catalyzed major cladogenesis (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Erwin, 2021</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Erwin, D.H.</div><div><strong>A conceptual framework of evolutionary novelty and innovation</strong></div><div><em>Biol. Rev. Camb. Philos. Soc.</em> 2021; <strong>96</strong>:1-15</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.H.ErwinA+conceptual+framework+of+evolutionary+novelty+and+innovationBiol.+Rev.+Camb.+Philos.+Soc.962021115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). How the functions of complex organs originate has challenged biologists since Darwin (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Darwin, 1859</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Darwin, C.</div><div><strong>On the Origin of Species by Means of Natural Selection, or, the Preservation of Favoured Races in the Struggle for Life</strong></div><div>John Murray, 1859</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.5962/bhl.title.82303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5962%2Fbhl.title.82303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Dawkins, 1986</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Dawkins, R.</div><div><strong>The Blind Watchmaker: Why the Evidence of Evolution Reveals a Universe Without Design</strong></div><div>WW Norton &amp; Company, 1986</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.DawkinsThe+Blind+Watchmaker%3A+Why+the+Evidence+of+Evolution+Reveals+a+Universe+Without+Design1986WW+Norton+%26+Company" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89" aria-controls="sref89">Lynch, 2007</a><div class="dropBlock__holder" data-db-target-of="sref89"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Lynch, M.</div><div><strong>The frailty of adaptive hypotheses for the origins of organismal complexity</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:8597-8604</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LynchThe+frailty+of+adaptive+hypotheses+for+the+origins+of+organismal+complexityProc.+Natl.+Acad.+Sci.+USA104Suppl+1200785978604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Oakley and Speiser, 2012</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Oakley, T.H. ∙ Speiser, D.I.</div><div><strong>How Complexity Originates: The Evolution of Animal Eyes</strong></div><div><em>Annu. Rev. Ecol. Evol. Syst.</em> 2012; <strong>46</strong>:1-24</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.H.OakleyD.I.SpeiserHow+Complexity+Originates%3A+The+Evolution+of+Animal+EyesAnnu.+Rev.+Ecol.+Evol.+Syst.462012124" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref131-1" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-1" aria-controls="sref131-1">Shubin et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref131-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Shubin, N. ∙ Tabin, C. ∙ Carroll, S.</div><div><strong>Deep homology and the origins of evolutionary novelty</strong></div><div><em>Nature.</em> 2009; <strong>457</strong>:818-823</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.ShubinC.TabinS.CarrollDeep+homology+and+the+origins+of+evolutionary+noveltyNature4572009818823" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref155" href="#" role="doc-biblioref" data-xml-rid="bib155" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref155" aria-controls="sref155">West-Eberhard, 2003</a><div class="dropBlock__holder" data-db-target-of="sref155"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">162.</div><div>West-Eberhard, M.J.</div><div><strong>Developmental Plasticity and Evolution</strong></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.J.West-EberhardDevelopmental+Plasticity+and+Evolution2003Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Organs are composed of specialized cell types that perform dedicated roles within these structures (<span class="dropBlock reference-citations"><a id="body-ref-sref130" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130" aria-controls="sref130">Schmidt-Rhaesa, 2007</a><div class="dropBlock__holder" data-db-target-of="sref130"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Schmidt-Rhaesa, A.</div><div><strong>The Evolution of Organ Systems</strong></div><div>Oxford University Press, 2007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Schmidt-RhaesaThe+Evolution+of+Organ+Systems2007Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); yet, knowledge of how cell types are constructed at the molecular level during evolution, and how they cooperate to produce emergent, organ-level behaviors, remains fragmentary. At the gene regulatory level, new cell types are thought to arise from the evolution of unique, combinatorial transcription factor activities that specify novel cell identities (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Arendt et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Arendt, D. ∙ Musser, J.M. ∙ Baker, C.V.H. ...</div><div><strong>The origin and evolution of cell types</strong></div><div><em>Nat. Rev. Genet.</em> 2016; <strong>17</strong>:744-757</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArendtJ.M.MusserC.V.H.BakerA.BergmanC.CepkoD.H.ErwinM.PavlicevG.SchlosserS.WidderM.D.LaubichlerG.P.WagnerThe+origin+and+evolution+of+cell+typesNat.+Rev.+Genet.172016744757" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, a major problem is explaining how the downstream molecular components, which carry out the function of the differentiated cell, themselves originate. For a cell to execute its role, multiple gene products must be configured to work in concert. In a multicellular organ, this process must be coordinated across multiple cell types, fostering a division of labor that generates organ behavior (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Kishi and Parker, 2021</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Kishi, Y. ∙ Parker, J.</div><div><strong>Cell type innovation at the tips of the animal tree</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2021; <strong>69</strong>:112-121</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.KishiJ.ParkerCell+type+innovation+at+the+tips+of+the+animal+treeCurr.+Opin.+Genet.+Dev.692021112121" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0040" role="paragraph">The molecular changes leading to new cell type- and organ-level functions are obscured by the deep antiquity of many animal organs (<span class="dropBlock reference-citations"><a id="body-ref-sref131-2" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131-2" aria-controls="sref131-2">Shubin et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref131-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Shubin, N. ∙ Tabin, C. ∙ Carroll, S.</div><div><strong>Deep homology and the origins of evolutionary novelty</strong></div><div><em>Nature.</em> 2009; <strong>457</strong>:818-823</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.ShubinC.TabinS.CarrollDeep+homology+and+the+origins+of+evolutionary+noveltyNature4572009818823" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, insight may be gained from “evolutionary novelties”—lineage-specific structures that are absent in outgroups or serially homologous body parts (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Erwin, 2015</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Erwin, D.H.</div><div><strong>Novelty and Innovation in the History of Life</strong></div><div><em>Curr. Biol.</em> 2015; <strong>25</strong>:R930-R940</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.H.ErwinNovelty+and+Innovation+in+the+History+of+LifeCurr.+Biol.252015R930R940" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref150" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150" aria-controls="sref150">Wagner and Lynch, 2010</a><div class="dropBlock__holder" data-db-target-of="sref150"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">157.</div><div>Wagner, G.P. ∙ Lynch, V.J.</div><div><strong>Evolutionary novelties</strong></div><div><em>Curr. Biol.</em> 2010; <strong>20</strong>:R48-R52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.P.WagnerV.J.LynchEvolutionary+noveltiesCurr.+Biol.202010R48R52" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Vertebrate eyes (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Lamb et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Lamb, T.D. ∙ Collin, S.P. ∙ Pugh, Jr., E.N.</div><div><strong>Evolution of the vertebrate eye: opsins, photoreceptors, retina and eye cup.</strong></div><div><em>Nat. Rev. Neurosci.</em> 2007; <strong>8</strong>:960-976</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.D.LambS.P.CollinE.N.PughJr.Evolution+of+the+vertebrate+eye%3A+opsins%2C+photoreceptors%2C+retina+and+eye+cup.Nat.+Rev.+Neurosci.82007960976" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), insect wings (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Linz and Tomoyasu, 2018</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Linz, D.M. ∙ Tomoyasu, Y.</div><div><strong>Dual evolutionary origin of insect wings&nbsp;supported by an investigation of the abdominal wing serial homologs in <i>Tribolium</i></strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E658-E667</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.LinzY.TomoyasuDual+evolutionary+origin+of+insect+wings%C2%A0supported+by+an+investigation+of+the+abdominal+wing+serial+homologs+in+TriboliumProc.+Natl.+Acad.+Sci.+USA1152018E658E667" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), beetle horns (<span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Hu et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Hu, Y. ∙ Linz, D.M. ∙ Moczek, A.P.</div><div><strong>Beetle horns evolved from wing serial homologs</strong></div><div><em>Science.</em> 2019; <strong>366</strong>:1004-1007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HuD.M.LinzA.P.MoczekBeetle+horns+evolved+from+wing+serial+homologsScience366201910041007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the mammalian placenta (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Griffith and Wagner, 2017</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Griffith, O.W. ∙ Wagner, G.P.</div><div><strong>The placenta as a model for understanding the origin and evolution of vertebrate organs</strong></div><div><em>Nat. Ecol. Evol.</em> 2017; <strong>1</strong>:0072</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.W.GriffithG.P.WagnerThe+placenta+as+a+model+for+understanding+the+origin+and+evolution+of+vertebrate+organsNat.+Ecol.+Evol.120170072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) are paradigms that connect genomic and developmental changes to the establishment of new phenotypes. Nevertheless, knowledge of how the functions of novel organs arise at the cellular level has been limited. One hurdle has been the challenge of measuring the transcriptomic composition of cell types—critical for tracing how different subfunctions within a cell evolved. The advent of single-cell methods has now brought many questions regarding cell-type evolution within reach (<span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Marioni and Arendt, 2017</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Marioni, J.C. ∙ Arendt, D.</div><div><strong>How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2017; <strong>33</strong>:537-553</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.MarioniD.ArendtHow+Single-Cell+Genomics+Is+Changing+Evolutionary+and+Developmental+BiologyAnnu.+Rev.+Cell+Dev.+Biol.332017537553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). New tools permit decomposition of the transcriptome, enabling discovery of gene expression programs underlying cell identity and quantitative assessment of relationships between cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref154" href="#" role="doc-biblioref" data-xml-rid="bib154" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref154" aria-controls="sref154">Way and Greene, 2019</a><div class="dropBlock__holder" data-db-target-of="sref154"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">161.</div><div>Way, G.P. ∙ Greene, C.S.</div><div><strong>Discovering Pathway and Cell Type Signatures in Transcriptomic Compendia with Machine Learning</strong></div><div><em>Annu. Rev. Biomed. Data Sci.</em> 2019; <strong>2</strong>:1-17</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.P.WayC.S.GreeneDiscovering+Pathway+and+Cell+Type+Signatures+in+Transcriptomic+Compendia+with+Machine+LearningAnnu.+Rev.+Biomed.+Data+Sci.22019117" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we leverage these methods to trace the evolution of cooperating cell types in an animal organ.</div><div id="p0045" role="paragraph">Exocrine glands are archetypal organ novelties: unique, clade-specific glands have evolved convergently thousands of times in metazoans, transforming how animals interact via pheromones and chemical defenses, enabling specialized modes of prey capture and digestion, and aiding in substrate adhesion, desiccation avoidance, and antimicrobial protection (<span class="dropBlock reference-citations"><a id="body-ref-sref24-1" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-1" aria-controls="sref24-1">Brückner and Parker, 2020</a><div class="dropBlock__holder" data-db-target-of="sref24-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Brückner, A. ∙ Parker, J.</div><div><strong>Molecular evolution of gland cell types and chemical interactions in animals</strong></div><div><em>J.&nbsp;Exp. Biol.</em> 2020; <strong>223</strong>:jeb211938</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerJ.ParkerMolecular+evolution+of+gland+cell+types+and+chemical+interactions+in+animalsJ.%C2%A0Exp.+Biol.223Pt%2C+Suppl+12020jeb211938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each origin of a new gland involves the evolution of new secretory cell types that synthesize natural products (<span class="dropBlock reference-citations"><a id="body-ref-sref147" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147" aria-controls="sref147">Torres and Schmidt, 2019</a><div class="dropBlock__holder" data-db-target-of="sref147"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Torres, J.P. ∙ Schmidt, E.W.</div><div><strong>The biosynthetic diversity of the animal world</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2019; <strong>294</strong>:17684-17692</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.P.TorresE.W.SchmidtThe+biosynthetic+diversity+of+the+animal+worldJ.%C2%A0Biol.+Chem.29420191768417692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Venom glands of wasps, scorpions, and cone snails; the explosive weaponry of bombardier beetles; and diverse mammalian scent glands are multicellular novelties composed of taxon-restricted cell types found nowhere else. For many such organs, including vertebrate salivary glands (<span class="dropBlock reference-citations"><a id="body-ref-sref126" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126" aria-controls="sref126">Roussa, 2011</a><div class="dropBlock__holder" data-db-target-of="sref126"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Roussa, E.</div><div><strong>Channels and transporters in salivary glands</strong></div><div><em>Cell Tissue Res.</em> 2011; <strong>343</strong>:263-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.RoussaChannels+and+transporters+in+salivary+glandsCell+Tissue+Res.3432011263287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), moth silk glands (<span class="dropBlock reference-citations"><a id="body-ref-sref144" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144" aria-controls="sref144">Suzuki et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref144"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">149.</div><div>Suzuki, Y. ∙ Takiya, S. ∙ Suzuki, T. ...</div><div><strong>Developmental regulation of silk gene expression in Bombyx mori</strong></div><div><div>Hagedorn, H.H. ∙ Hildebrand, J.G. ∙ Kidwell, M.G. ... (Editors)</div></div><div><strong>Molecular Insect Science</strong></div><div>SpringerLink, 1990; 83-89</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.SuzukiS.TakiyaT.SuzukiC.HuiK.MatsunoM.FukutaT.NagataK.UenoDevelopmental+regulation+of+silk+gene+expression+in+Bombyx+moriH.H.HagedornJ.G.HildebrandM.G.KidwellJ.H.LawMolecular+Insect+Science1990SpringerLink8389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), cnidarian digestive glands (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Babonis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Babonis, L.S. ∙ Ryan, J.F. ∙ Enjolras, C. ...</div><div><strong>Genomic analysis of the tryptome reveals molecular mechanisms of gland cell evolution</strong></div><div><em>Evodevo.</em> 2019; <strong>10</strong>:23</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.S.BabonisJ.F.RyanC.EnjolrasM.Q.MartindaleGenomic+analysis+of+the+tryptome+reveals+molecular+mechanisms+of+gland+cell+evolutionEvodevo10201923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and venom glands of many species (<span class="dropBlock reference-citations"><a id="body-ref-sref143" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143" aria-controls="sref143">Surm and Moran, 2021</a><div class="dropBlock__holder" data-db-target-of="sref143"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">148.</div><div>Surm, J.M. ∙ Moran, Y.</div><div><strong>Insights into how development and life-history dynamics shape the evolution of venom</strong></div><div><em>Evodevo.</em> 2021; <strong>12</strong>:1</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.SurmY.MoranInsights+into+how+development+and+life-history+dynamics+shape+the+evolution+of+venomEvodevo1220211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the final secretion is a cocktail produced by distinct secretory cell types. Biosynthetic synergism between cell types is pronounced in many small molecule-based chemical defense systems. Here, an inactive toxin and its activating enzyme may be produced by separate glands; alternatively, a soluble toxin and its solvent are secreted into a common reservoir, yielding a bioactive secretion (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Kishi and Parker, 2021</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Kishi, Y. ∙ Parker, J.</div><div><strong>Cell type innovation at the tips of the animal tree</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2021; <strong>69</strong>:112-121</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.KishiJ.ParkerCell+type+innovation+at+the+tips+of+the+animal+treeCurr.+Opin.+Genet.+Dev.692021112121" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Such functional interdependence encapsulates how emergent, organ-level behaviors can arise from cooperation between cell types.</div><div id="p0050" role="paragraph">One animal clade notable for pervasive glandular innovation is the rove beetles (Staphylinidae)—a radiation of 64,000 predominantly soil-dwelling predators that comprise Metazoa’s largest family (<span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Betz et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Betz, O. ∙ Irmler, U. ∙ Klimaszewski, J.</div><div><strong>Biology of Rove Beetles (Staphylinidae)</strong></div><div>Springer, 2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.BetzU.IrmlerJ.KlimaszewskiBiology+of+Rove+Beetles+%28Staphylinidae%292018Springer" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Parker, 2017</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Parker, J.</div><div><strong>Staphylinids</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:R49-R51</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerStaphylinidsCurr.+Biol.272017R49R51" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Rove beetles exhibit widespread evolution of abdominal glands that produce diverse compounds including quinones, terpenes, alkaloids, iridoids, hydrocarbons and esters (<span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Francke and Dettner, 2005</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Francke, W. ∙ Dettner, K.</div><div><strong>Chemical Signalling in Beetles</strong></div><div><em>Top. Curr. Chem.</em> 2005; <strong>240</strong>:85-166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.FranckeK.DettnerChemical+Signalling+in+BeetlesTop.+Curr.+Chem.240200585166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A unique, flexible body enables rove beetles to target secretions at other organisms, fostering interspecies relationships from chemical defense to symbiosis (<span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Parker, 2016</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Parker, J.</div><div><strong>Myrmecophily in beetles (Coleoptera): evolutionary patterns and biological mechanisms</strong></div><div><em>Myrmecol. News.</em> 2016; <strong>22</strong>:65-108</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerMyrmecophily+in+beetles+%28Coleoptera%29%3A+evolutionary+patterns+and+biological+mechanismsMyrmecol.+News22201665108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The glands are often composite organs that produce multi-compound secretions via distinct cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Dettner, 1993a</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Dettner, K.</div><div><strong>Defensive secretions and exocrine glands in free-living staphylinid beetles—their bearing on phylogeny (Coleoptera: Staphylinidae)</strong></div><div><em>Biochem. Syst. Ecol.</em> 1993; <strong>21</strong>:143-162</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerDefensive+secretions+and+exocrine+glands+in+free-living+staphylinid+beetles%E2%80%94their+bearing+on+phylogeny+%28Coleoptera%3A+Staphylinidae%29Biochem.+Syst.+Ecol.211993143162" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although their chemistry and anatomy have been studied in many species (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Francke and Dettner, 2005</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Francke, W. ∙ Dettner, K.</div><div><strong>Chemical Signalling in Beetles</strong></div><div><em>Top. Curr. Chem.</em> 2005; <strong>240</strong>:85-166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.FranckeK.DettnerChemical+Signalling+in+BeetlesTop.+Curr.+Chem.240200585166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), how these organs function at the molecular level, and how they evolved in specific lineages, is unknown. Here, we examine the molecular architecture of one such gland, enabling us to infer how a novel organ and its constituent cell types evolved. We exploited the model species <i>Dalotia coriaria</i> of the megadiverse Aleocharinae, a subfamily that possess a defensive “tergal gland” on the dorsal abdomen (<span class="dropBlock reference-citations"><a id="body-ref-sref141-1" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-1" aria-controls="sref141-1">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Using single-cell transcriptomics, we uncover pathways for defensive compounds in tergal gland cell types. By inferring transcriptomic and pathway relationships with more ancient cell types, we retrace how the tergal gland was functionally assembled during evolution. We present evidence that evolution followed an accessible path toward cell-type cooperativity, building an organ capable of producing a multi-compound secretion that confers adaptive value.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>The tergal gland is an organ novelty</h3><div id="p0055" role="paragraph">When provoked, <i>Dalotia</i> flexes its abdomen to release the contents of a large chemical defense gland—the tergal gland—situated between abdominal segments A6 and A7 (<a id="crosref0225" href="#fig1">Figures 1</a>A and 1B) (<span class="dropBlock reference-citations"><a id="body-ref-sref141-2" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-2" aria-controls="sref141-2">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The secretion is smeared directly onto the perceived threat and contains three benzoquinones (BQs) (<a id="crosref0235" href="#fig1">Figure&nbsp;1</a>D)—aromatics that are common arthropod defensive compounds (<span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Blum, 1981</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Blum, M.</div><div><strong>Chemical Defenses of Arthropods</strong></div><div>Academic Press, 1981</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BlumChemical+Defenses+of+Arthropods1981Academic+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). BQs are topical irritants that bind TRPA1 channels (<span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Ibarra and Blair, 2013</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Ibarra, Y. ∙ Blair, N.T.</div><div><strong>Benzoquinone reveals a cysteine-dependent desensitization mechanism of TRPA1</strong></div><div><em>Mol. Pharmacol.</em> 2013; <strong>83</strong>:1120-1132</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.IbarraN.T.BlairBenzoquinone+reveals+a+cysteine-dependent+desensitization+mechanism+of+TRPA1Mol.+Pharmacol.83201311201132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), activating nociceptive pathways (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Kang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Kang, K. ∙ Pulver, S.R. ∙ Panzano, V.C. ...</div><div><strong>Analysis of Drosophila TRPA1 reveals an ancient origin for human chemical nociception</strong></div><div><em>Nature.</em> 2010; <strong>464</strong>:597-600</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KangS.R.PulverV.C.PanzanoE.C.ChangL.C.GriffithD.L.TheobaldP.A.GarrityAnalysis+of+Drosophila+TRPA1+reveals+an+ancient+origin+for+human+chemical+nociceptionNature4642010597600" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The adaptive value of <i>Dalotia</i>’s secretion is demonstrated by experimentally depleting the contents of the gland, disarming the beetle, and compromising its&nbsp;defense against predatory ants (<a id="crosref0255" href="#fig1">Figures 1</a>C and <a id="crosref0260" href="#figs1">S1</a>A). The BQs may also have antimicrobial properties (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Carcamo-Noriega et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Carcamo-Noriega, E.N. ∙ Sathyamoorthi, S. ∙ Banerjee, S. ...</div><div><strong>1,4-Benzoquinone antimicrobial agents against <i>Staphylococcus aureus</i> and <i>Mycobacterium tuberculosis</i> derived from scorpion venom</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12642-12647</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.N.Carcamo-NoriegaS.SathyamoorthiS.BanerjeeE.GnanamaniM.Mendoza-TrujilloD.Mata-EspinosaR.Hern%C3%A1ndez-PandoJ.I.Veytia-BucheliL.D.PossaniR.N.Zare1%2C4-Benzoquinone+antimicrobial+agents+against+Staphylococcus+aureus+and+Mycobacterium+tuberculosis+derived+from+scorpion+venomProc.+Natl.+Acad.+Sci.+USA11620191264212647" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/23322283-9a93-4480-b6a1-48d1b9998b5f/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.014/asset/b5b9ec2b-06bd-436f-a2c3-4ad5ef91b5cf/main.assets/gr1.jpg" height="420" width="743" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/23322283-9a93-4480-b6a1-48d1b9998b5f/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">The tergal gland</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) <i>Dalotia coriaria</i> with position of tergal gland.</div><div id="fspara0020" role="paragraph">(B) Cartoon of gland deployment.</div><div id="fspara0025" role="paragraph">(C) <i>Dalotia</i> survival with predatory <i>Liometopum</i> ants after 48&nbsp;h (armed versus disarmed glands; Mann-Whitney U test: p&nbsp;&lt; 0.001). The middle bar of each boxplot represents the median, the box denotes the interquartile range, while the whiskers show the minimum and maximum.</div><div id="fspara0030" role="paragraph">(D) Composition of gland secretion (n&nbsp;= 10 beetles) measured by GC-MS. The middle bar of each boxplot represents the median, the box denotes the interquartile range, while the whiskers show the minimum and maximum.</div><div id="fspara0035" role="paragraph">(E) Cartoon cross section of abdomen showing gland cell types: secretory epithelium (magenta) invaginated between segments A6–A7 forms a chemical reservoir; bulbous secretory cells (green) connect to reservoir via ducts. The secretion exudes via the opening between segments.</div><div id="fspara0040" role="paragraph">(F) Confocal image of internal segments A5–A7 of aleocharine abdomen (magenta, α-Engrailed antibody labels reservoir cell nuclei; green, bulbous secretory cells autofluorescence; blue, phalloidin-stained muscle).</div><div id="fspara0045" role="paragraph">(G) Confocal image of internal segments A5–A7 of glandless outgroup rove beetle (Tachyporinae: <i>Coproporus</i>).</div><div id="fspara0050" role="paragraph">See also <a id="crosref2425" href="#figs1">Figures S1</a> and <a id="crosref2430" href="#figs2">S2</a> and <a id="crosref2435" href="#mmc7">Table S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/df2f6aa3-4cac-4d40-b87a-968b0103529e/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.014/asset/06e85581-153f-46ba-8eac-3cf3df5515c9/main.assets/figs1.jpg" height="529" width="659" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/df2f6aa3-4cac-4d40-b87a-968b0103529e/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">CO<sub>2</sub> disarming, comparison to a glandless outgroup rove beetle, and compound biosynthesis despite microbial suppression, related to <a id="crosref2530" href="#fig1">Figures 1</a> and <a id="crosref2535" href="#fig2">2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0365" role="paragraph">A: Repeated treatment with small doses of CO<sub>2</sub> leads to depletion of the tergal gland content in <i>Dalotia</i> effectively disarming the beetles. Significance was assessed using a Mann-Whitney-U test (<sup>∗∗∗</sup>&nbsp;= p&nbsp;&lt; 0.001). B: Representative GC profile of the <i>Dalotia coriaria</i> gland content extracted in hexane. In order of retention: 1,4-benzoquinone (1,4-BQ), 2-methyl-1,4-BQ, undecane, 2-methoxy-3-methyl-1,4-BQ, ethyl decanoate, isopropyl decanoate, ethyl dodecanoate. C: Representative GC profile of <i>Tachinus</i> (Tachyporinae) extracted and measured using the same protocol. No gland compounds were detected. Asterisks denote in order of retention: C16:1-fatty acid (FA), C16-FA, C18-FA. D: Antibiotic treatment eliminates bacterial load in <i>Dalotia</i> as determined by qPCR quantification of the ratio of bacterial 16&nbsp;s rRNA to host 28&nbsp;s rRNA levels in antibiotic-treated and control beetles. E-H: Quantities of defensive compounds are unaffected by bacterial suppression: total secretion (E); undecane (F); the ester fraction (G); and the benzoquinones (H). Significance was assessed using Mann-Whitney-U tests (<sup>∗∗</sup>&nbsp;= p ≤ 0.01; ns&nbsp;= p ≥ 0.05). In all figures, the horizontal middle bars represents the median, while the vertical bars show the minimum and maximum.</div></div></div></div></figcaption></figure></div><div id="p0060" role="paragraph">Although BQs are highly noxious, each compound in isolation is a solid. Consequently, the secretion also contains four less toxic compounds: large amounts of a C11 alkane, <i>n</i>-undecane, low amounts of two C10 esters, ethyl decanoate and isopropyl decanoate, and trace levels of a C12 ester, ethyl dodecanoate (<a id="crosref0270" href="#fig1">Figure&nbsp;1</a>D). These compounds may function as solvents, combining with BQs to create a potent secretion, as well as promoting spreading and penetration into cuticles (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Blum, 1981</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Blum, M.</div><div><strong>Chemical Defenses of Arthropods</strong></div><div>Academic Press, 1981</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BlumChemical+Defenses+of+Arthropods1981Academic+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Dettner, 1984</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Dettner, K.</div><div><strong>Isopropylesters as Wetting Agents From the Defensive Secretion of the Rove Beetle Coprophilus striatulus F. (Coleoptera, Staphylinidae)</strong></div><div><em>Insect Biochem.</em> 1984; <strong>14</strong>:383-390</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerIsopropylesters+as+Wetting+Agents+From+the+Defensive+Secretion+of+the+Rove+Beetle+Coprophilus+striatulus+F.+%28Coleoptera%2C+Staphylinidae%29Insect+Biochem.141984383390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The secretion is thus a cocktail, the composition of which may be explained by possible synergism between its components. To produce this cocktail, the tergal gland consists of two anatomically distinct cell types: (1) a secretory epithelium, composed of columnar cells that are continuous with the epidermis and form a large reservoir inside the body into which they secrete directly (<a id="crosref0285" href="#fig1">Figures 1</a>E and 1F); and (2) large, bulbous secretory cells, 8–14 per animal, located posterior to the reservoir into which they feed via tubular ducts (<a id="crosref0290" href="#fig1">Figures 1</a>E and 1F). The tergal gland, and these two cell types, are specific to aleocharines (<span class="dropBlock reference-citations"><a id="body-ref-sref141-3" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-3" aria-controls="sref141-3">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); they are absent in outgroup rove beetles (subfamily Tachyporinae) that cannot produce defensive compounds (<a id="crosref0300" href="#fig1">Figures 1</a>G, <a id="crosref0305" href="#figs1">S1</a>B, and S1C). We sought to determine how these two cell types cooperate to synthesize <i>Dalotia</i>’s defensive secretion.</div></section><section id="sec-2-2"><h3>The solvent cell type</h3><div id="p0065" role="paragraph"><i>Dalotia</i>’s ability to produce defensive compounds is refractory to&nbsp;antibiotics (<a id="crosref0310" href="#figs1">Figures S1</a>D–S1H). We therefore reasoned that biosynthesis relies not on symbiotic bacteria, as in some other chemically defended rove beetles (<span class="dropBlock reference-citations"><a id="body-ref-sref111" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111" aria-controls="sref111">Piel, 2002</a><div class="dropBlock__holder" data-db-target-of="sref111"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Piel, J.</div><div><strong>A polyketide synthase-peptide synthetase gene cluster from an uncultured bacterial symbiont of Paederus beetles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:14002-14007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PielA+polyketide+synthase-peptide+synthetase+gene+cluster+from+an+uncultured+bacterial+symbiont+of+Paederus+beetlesProc.+Natl.+Acad.+Sci.+USA9920021400214007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but on enzymes encoded in the beetle’s genome and expressed within the gland. We dissected small numbers of each gland cell type and performed SMART-Seq RNA sequencing, creating cell type transcriptomes (<a id="crosref0320" href="#figs2">Figures S2</a>A–S2C). This approach led to recovery of candidate enzymes in each cell type. We focused initially on synthesis of the undecane and esters, hypothesizing that their hydrocarbon chains derive from fatty acids (<span class="dropBlock reference-citations"><a id="body-ref-sref139" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139" aria-controls="sref139">Stanley-Samuelson et&nbsp;al., 1988</a><div class="dropBlock__holder" data-db-target-of="sref139"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Stanley-Samuelson, D.W. ∙ Jurenka, R.A. ∙ Cripps, C. ...</div><div><strong>Fatty acids in insects: Composition, metabolism, and biological significance</strong></div><div><em>Arch. Insect Biochem.</em> 1988; <strong>9</strong>:1-33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.W.Stanley-SamuelsonR.A.JurenkaC.CrippsG.J.BlomquistM.de+RenobalesFatty+acids+in+insects%3A+Composition%2C+metabolism%2C+and+biological+significanceArch.+Insect+Biochem.91988133" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Fatty acids are built from acetyl-CoA and malonyl-CoA units produced by glycolysis (<span class="dropBlock reference-citations"><a id="body-ref-sref152" href="#" role="doc-biblioref" data-xml-rid="bib152" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref152" aria-controls="sref152">Wakil et&nbsp;al., 1983</a><div class="dropBlock__holder" data-db-target-of="sref152"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">159.</div><div>Wakil, S.J. ∙ Stoops, J.K. ∙ Joshi, V.C.</div><div><strong>Fatty acid synthesis and its regulation</strong></div><div><em>Annu. Rev. Biochem.</em> 1983; <strong>52</strong>:537-579</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.J.WakilJ.K.StoopsV.C.JoshiFatty+acid+synthesis+and+its+regulationAnnu.+Rev.+Biochem.521983537579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Rearing <i>Dalotia</i> on D-glucose-<sup>13</sup>C<sub>6</sub> food, we observed strong <sup>13</sup>C incorporation into the undecane and esters (<a id="crosref0335" href="#figs3">Figure&nbsp;S3</a>A), confirming they originate via the glycolytic pathway and fatty acid synthesis. In contrast, adding D<sub>23</sub>-dodecanoic acid to food led to negligible D<sub>23</sub> incorporation (<a id="crosref0340" href="#figs3">Figure&nbsp;S3</a>B). The compounds thus do not derive from consumed fatty acids.</div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/aad40f19-2ac3-4f63-9496-df101a1241bf/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.014/asset/08a3f340-bf19-4895-ba13-aad79b1c4379/main.assets/figs2.jpg" height="960" width="753" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/aad40f19-2ac3-4f63-9496-df101a1241bf/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Cell-type specific transcriptomes and functional screening pipeline, related to <a id="crosref2540" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0375" role="paragraph">A: To prepare cell-type specific transcriptomes of the two tergal gland cell types, cells were micro-dissected by hand and segment 6 tergite tissue was used as a control. Cells were lysed, and libraries prepared and sequenced to perform differential expression analysis. B: Differential expression analysis of solvent cells versus control cells. Heatmap showing the 75 most upregulated transcripts in the solvent cells (as assessed by q-value using sleuth; see <a id="intref0665" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). Expression in all individual sequencing libraries is shown. Darker squares denote a stronger mean expression value. A full list of transcripts can be found at CaltechData (<a id="intref0670" href="https://data.caltech.edu/records/1900" target="_blank">https://data.caltech.edu/records/1900</a>). C: Differential expression analysis of BQ cells versus control cells. Heatmap showing the 75 most upregulated transcripts in the benzoquinone cells (as assessed by q-value using sleuth; see <a id="intref0675" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). Expression in all individual sequencing libraries is shown. Darker squares denote a stronger mean expression value. A full list of transcripts can be found at CaltechData (<a id="intref0680" href="https://data.caltech.edu/records/1900" target="_blank">https://data.caltech.edu/records/1900</a>). D: Schematic workflow of the enzyme screening strategy. By injecting mature larvae with double stranded RNA (dsRNA) targeting specific mRNAs, transcription is silenced in gland cells with temporal control, during pupal and early adult life when defensive compound biosynthesis occurs. The chemical composition of the tergal gland secretion can then be quantified in the mature adult via gas chromatography-mass spectrometry (GC-MS).</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/7023b27a-b2fe-4449-8c25-e63d6fd07d48/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.014/asset/1eb1ea84-c9d7-43e1-8eff-a7906c8171da/main.assets/figs3.jpg" height="942" width="687" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/7023b27a-b2fe-4449-8c25-e63d6fd07d48/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Undecane and ester biosynthesis, related to <a id="crosref2545" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0385" role="paragraph">A: Representative mass spectra of molecular ion region of undecane, ethyl decanoate, isopropyl decanoate and ethyl dodecanoate from beetles fed with unlabeled <i>Dalotia</i> diet (control) or infused with D-<sup>13</sup>C<sub>6</sub>-glucose. Spectra recorded in single-ion mode; enriched ions were detected in all four spectra in beetles fed D-<sup>13</sup>C<sub>6</sub>-glucose. B: Representative mass spectra of molecular ion region of undecane from beetles fed with unlabeled <i>Dalotia</i> diet (control) or food infused with D<sub>23</sub>-dodecanoic acid. Spectra recorded in single-ion mode; no enriched ions were detected. C-F: RNAi silencing of solvent pathway enzymes. C: Amounts of undecane, ethyl decanoate, isopropyl decanoate and ethyl dodecanoate extracted from beetles injected with dsRNA against <i>GFP</i> (control) and <i>MFASN</i>. Silencing expression of <i>MFASN</i> resulted in near- complete loss of solvents (undecane and esters). D: Amounts of undecane, ethyl decanoate, dodecanal, isopropyl decanoate and ethyl dodecanoate extracted from beetles injected with dsRNA against <i>GFP</i> (control) and <i>TG-CYP4G</i>. Silencing <i>TG-CYP4G</i> caused loss of undecane, but a large increase in both dodecanal and ethyl dodecanoate without influencing levels of ethyl decanoate and isopropyl decanoate. E: Amounts of undecane, ethyl decanoate, isopropyl decanoate and ethyl dodecanoate extracted from beetles injected dsRNA against <i>GFP</i> (control) and <i>TG-FAR</i>. Silencing TG-FAR resulted in loss of undecane but a large increase in ethyl dodecanoate without influencing levels of ethyl decanoate and isopropyl decanoate. F: Amounts of ethyl decanoate, isopropyl decanoate and ethyl dodecanoate extracted from beetles injected with dsRNA against <i>GFP</i> (control) and <i>TG-αEst</i>. Silencing TG-αEst resulted in near complete loss of all esters. In all panels, significance was assessed using Mann-Whitney-U tests (<sup>∗∗∗</sup>&nbsp;= p ≤ 0.001; <sup>∗</sup>&nbsp;= p ≤ 0.05; ns&nbsp;= p ≥ 0.05). In (C–F), the horizontal middle bars represents the median, while the vertical bars show the minimum and maximum. G: Maximum-likelihood phylogeny of the CYP4G gene family (best model: LG+R8). Some clades have been collapsed for presentation purposes. The topology confirms that both <i>Dalotia CYP4G</i> copies are the product of a recent duplication within the CYP4G1 clade, known for their role in cuticular hydrocarbon synthesis in insects. <i>Dalotia</i> lacks a member of the CYP4G15 clade (red collapsed clade).</div></div></div></div></figcaption></figure></div><div id="p0070" role="paragraph">In animals, the multistep reaction to produce fatty acids is catalyzed by a multidomain fatty acid synthase (FASN) (<span class="dropBlock reference-citations"><a id="body-ref-sref135" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135" aria-controls="sref135">Smith, 1994</a><div class="dropBlock__holder" data-db-target-of="sref135"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">140.</div><div>Smith, S.</div><div><strong>The animal fatty acid synthase: one gene, one polypeptide, seven enzymes</strong></div><div><em>FASEB J.</em> 1994; <strong>8</strong>:1248-1259</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.SmithThe+animal+fatty+acid+synthase%3A+one+gene%2C+one+polypeptide%2C+seven+enzymesFASEB+J.8199412481259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref136-1" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-1" aria-controls="sref136-1">Smith et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref136-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Smith, S. ∙ Witkowski, A. ∙ Joshi, A.K.</div><div><strong>Structural and functional organization of the animal fatty acid synthase</strong></div><div><em>Prog. Lipid Res.</em> 2003; <strong>42</strong>:289-317</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.SmithA.WitkowskiA.K.JoshiStructural+and+functional+organization+of+the+animal+fatty+acid+synthaseProg.+Lipid+Res.422003289317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). FASNs generally synthesize the commonest fatty acids, C16 and C18 in length (<span class="dropBlock reference-citations"><a id="body-ref-sref136-2" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136-2" aria-controls="sref136-2">Smith et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref136-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Smith, S. ∙ Witkowski, A. ∙ Joshi, A.K.</div><div><strong>Structural and functional organization of the animal fatty acid synthase</strong></div><div><em>Prog. Lipid Res.</em> 2003; <strong>42</strong>:289-317</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.SmithA.WitkowskiA.K.JoshiStructural+and+functional+organization+of+the+animal+fatty+acid+synthaseProg.+Lipid+Res.422003289317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, <i>Dalotia</i>’s gland compounds are shorter (C10–C12). Notably, a predicted <i>FASN</i> was upregulated in the epithelial secretory cells comprising the reservoir (<a id="crosref0360" href="#figs2">Figure&nbsp;S2</a>B) but not in the bulbous secretory cells (<a id="crosref0365" href="#figs2">Figure&nbsp;S2</a>C). Strong expression in these cells was confirmed via <i>in situ</i> hybridization chain reaction (HCR) (<span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Choi et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Choi, H.M.T. ∙ Schwarzkopf, M. ∙ Fornace, M.E. ...</div><div><strong>Third-generation <i>in situ</i> hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust</strong></div><div><em>Development.</em> 2018; <strong>145</strong>:dev165753</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.M.T.ChoiM.SchwarzkopfM.E.FornaceA.AcharyaG.ArtavanisJ.StegmaierA.CunhaN.A.PierceThird-generation+in+situ+hybridization+chain+reaction%3A+multiplexed%2C+quantitative%2C+sensitive%2C+versatile%2C+robustDevelopment1452018dev165753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0375" href="#fig2">Figure&nbsp;2</a>F). To deduce this <i>FASN</i>’s function, we temporally silenced it with systemic RNA interference (RNAi) (<span class="dropBlock reference-citations"><a id="body-ref-sref109-1" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-1" aria-controls="sref109-1">Parker et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref109-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Parker, J. ∙ Eldredge, K.T. ∙ Thomas, I. ...</div><div><strong>Hox-logic of body plan innovations for social insect symbiosis in rove beetles</strong></div><div><em>bioRxiv.</em> 2018; 198945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerK.T.EldredgeI.ThomasR.ColemanS.DavisHox-logic+of+body+plan+innovations+for+social+insect+symbiosis+in+rove+beetlesbioRxiv2018198945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref146" href="#" role="doc-biblioref" data-xml-rid="bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146" aria-controls="sref146">Tomoyasu and Denell, 2004</a><div class="dropBlock__holder" data-db-target-of="sref146"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">151.</div><div>Tomoyasu, Y. ∙ Denell, R.E.</div><div><strong>Larval RNAi in Tribolium (Coleoptera) for analyzing adult development</strong></div><div><em>Dev. Genes Evol.</em> 2004; <strong>214</strong>:575-578</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.TomoyasuR.E.DenellLarval+RNAi+in+Tribolium+%28Coleoptera%29+for+analyzing+adult+developmentDev.+Genes+Evol.2142004575578" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0390" href="#figs2">Figure&nbsp;S2</a>D). Silencing caused near-total loss of undecane and all&nbsp;three esters without impacting the BQs (<a id="crosref0395" href="#fig2">Figures 2</a>A, 2B, and <a id="crosref0400" href="#figs3">S3</a>C). We name this enzyme “Master Fatty Acid Synthase” (MFASN), due to its upstream role in synthesizing fatty acid-derived compounds in the tergal gland. We infer that the epithelial secretory cells (magenta cell type in <a id="crosref0405" href="#fig2">Figure&nbsp;2</a>J) are the source of these compounds. This cell type is hence named the “solvent cell type.”</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/f8ab2808-32c5-4bb9-a9d1-3f7c6789a4ee/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.014/asset/c9ebf03a-a86d-4e60-a6dc-bed87f77c318/main.assets/gr2.jpg" height="394" width="758" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/f8ab2808-32c5-4bb9-a9d1-3f7c6789a4ee/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">The solvent cell type</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0060" role="paragraph">(A–E) Tergal gland secretion GC traces.</div><div id="fspara0065" role="paragraph">(A) Wild-type with compound peaks labeled.</div><div id="fspara0070" role="paragraph">(B) <i>MFASN</i> RNAi causes loss of undecane and esters but not BQs.</div><div id="fspara0075" role="paragraph">(C) <i>TG-CYP4G</i> RNAi removes undecane; dodecanal appears and C12 ester is strongly increased.</div><div id="fspara0080" role="paragraph">(D) <i>TG-FAR</i> RNAi removes undecane; dodecanal is absent and the C12 ester increases.</div><div id="fspara0085" role="paragraph">(E) <i>TG-</i>α<i>Est</i> RNAi removes esters without affecting undecane or BQs. For statistics related to (B)–(E), see <a id="crosref2440" href="#mmc7">Table S1</a>.</div><div id="fspara0090" role="paragraph">(F–I) HCR (magenta) of solvent pathway enzymes: <i>MFASN</i> (F), <i>TG-CYP4G</i> (G), <i>TG-FAR</i> (H), and <i>TG-</i>α<i>Est</i> (I). Green, WGA.</div><div id="fspara0095" role="paragraph">(J) The solvent pathway.</div><div id="fspara0100" role="paragraph">See also <a id="crosref2445" href="#figs1">Figures S1</a> and <a id="crosref2450" href="#figs3">S3</a>.</div></div></div></div></figcaption></figure></div><div id="p0075" role="paragraph">Further analysis of the solvent cell transcriptome led to discovery of a complete fatty acid pathway downstream of MFASN. The maximally expressed transcript is a predicted cytochrome P450 (CYP) (<a id="crosref0410" href="#figs2">Figure&nbsp;S2</a>B), cell-type-specific expression of which was confirmed by HCR (<a id="crosref0415" href="#fig2">Figure&nbsp;2</a>G). Phylogenetic analysis revealed that this CYP belongs to an insect-specific “CYP4G” class (<a id="crosref0420" href="#figs3">Figure&nbsp;S3</a>G), known to decarbonylate aldehydes to create alkanes (<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Feyereisen, 2020</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Feyereisen, R.</div><div><strong>Origin and evolution of the CYP4G subfamily in insects, cytochrome P450 enzymes involved in cuticular hydrocarbon synthesis</strong></div><div><em>Mol. Phylogenet. Evol.</em> 2020; <strong>143</strong>:106695</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.FeyereisenOrigin+and+evolution+of+the+CYP4G+subfamily+in+insects%2C+cytochrome+P450+enzymes+involved+in+cuticular+hydrocarbon+synthesisMol.+Phylogenet.+Evol.1432020106695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-1" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-1" aria-controls="sref119-1">Qiu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref119-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Qiu, Y. ∙ Tittiger, C. ∙ Wicker-Thomas, C. ...</div><div><strong>An insect-specific P450 oxidative decarbonylase for cuticular hydrocarbon biosynthesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:14858-14863</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.QiuC.TittigerC.Wicker-ThomasG.Le+GoffS.YoungE.WajnbergT.FricauxN.TaquetG.J.BlomquistR.FeyereisenAn+insect-specific+P450+oxidative+decarbonylase+for+cuticular+hydrocarbon+biosynthesisProc.+Natl.+Acad.+Sci.+USA10920121485814863" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with this function, silencing the tergal gland <i>CYP4G</i> (herein “<i>TG-CYP4G</i>”) caused selective loss of the undecane and the appearance of its C12 aldehyde precursor, dodecanal (<a id="crosref0435" href="#fig2">Figures 2</a>C and <a id="crosref0440" href="#figs3">S3</a>D). We infer that TG-CYP4G functions downstream of MFASN as a terminal enzyme that decarbonylates dodecanal to make undecane. To produce dodecanal, the activated C12 fatty acid (dodecanoyl-CoA) produced by MFASN must first be reduced to an aldehyde. Conspicuously, a predicted fatty acyl-CoA reductase (herein “<i>TG-FAR</i>”) was strongly expressed in solvent cells (<a id="crosref0445" href="#fig2">Figures 2</a>H and <a id="crosref0450" href="#figs2">S2</a>B). As with <i>TG-CYP4G</i>, silencing <i>TG-FAR</i> caused specific loss of undecane, this time without the appearance of dodecanal (<a id="crosref0455" href="#fig2">Figures 2</a>D and <a id="crosref0460" href="#figs3">S3</a>E). We infer that solvent cells make undecane via a pathway in which MFASN produces dodecanoic acid that is reduced by TG-FAR and decarbonylated by TG-CYP4G (<a id="crosref0465" href="#fig2">Figure&nbsp;2</a>J).</div><div id="p0080" role="paragraph">In addition to this alkane pathway, a second branch of the solvent pathway yields the esters. Silencing <i>MFASN</i> caused loss of the three esters (<a id="crosref0470" href="#fig2">Figure&nbsp;2</a>B). Because two of these are C10 compounds (isopropyl and ethyl decanoate), MFASN must produce their C10 fatty acid precursor, decanoic acid, in addition to the dodecanoic acid precursor of undecane (<a id="crosref0475" href="#fig2">Figure&nbsp;2</a>J). Notably, we observed strong upregulation of a single α-esterase enzyme in solvent cells (<a id="crosref0480" href="#fig2">Figures 2</a>I and <a id="crosref0485" href="#figs2">S2</a>B). Silencing this enzyme (“<i>TG-</i>α<i>Est</i>”) removed all three esters without affecting undecane (<a id="crosref0490" href="#fig2">Figures 2</a>E and <a id="optSh9cvAw1R5" href="#figs3">S3</a>F). TG-αEst is thus the key enzyme in the ester branch of the solvent pathway. TG-αEst esterifies both C10-CoA and C12-CoA to yield the three esters (<a id="crosref0500" href="#fig2">Figure&nbsp;2</a>J).</div></section><section id="sec-2-3"><h3>A cell atlas of <i>Dalotia</i> abdominal segments</h3><div id="p0085" role="paragraph">Solvent cells create a volatile solution that we hypothesize dissolves the BQs. To investigate how this function evolved, we reconstructed the evolutionary steps by which the solvent cells were pieced together. We applied single-cell RNA sequencing (scRNA-seq) using droplet microfluidics (10x Chromium v.3) to entire adult abdominal segments, enabling us to determine the transcriptomic relationships of solvent cells to other cell types in the <i>Dalotia</i> abdomen. We created two segmental atlases: “Segment 7” comprising segment A7 plus the solvent cell reservoir (<a id="crosref0505" href="#fig3">Figure&nbsp;3</a>A); and “Segment 6” encompassing segment A6&nbsp;minus the reservoir. Segment 6 is serially homologous but lacks the gland, approximating ancestral segment morphology (<a id="crosref0510" href="#fig3">Figures 3</a>A and 3B). We integrated five 10x runs (<a id="crosref0515" href="#figs4">Figures S4</a>A–S4C) and selected 3,000 transcripts that capture transcriptional heterogeneity across cells for subsequent analyses. Using Seurat 3.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ButlerP.HoffmanP.SmibertE.PapalexiR.SatijaIntegrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+speciesNat.+Biotechnol.362018411420" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we determined 26 cell clusters, assigning these to putative cell types using marker genes from <i>Drosophila</i> (see <a id="crosref0525" href="#sec-4">STAR Methods</a>). The atlas is shown in <a id="crosref0530" href="#fig3">Figure&nbsp;3</a>C as a UMAP.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/6b790b50-5b30-4f75-b717-6f1f2240d23b/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.014/asset/6f703b5f-ae7c-45bf-a3f7-11e41be245af/main.assets/gr3.jpg" height="596" width="771" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/6b790b50-5b30-4f75-b717-6f1f2240d23b/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Single cell atlas of <i>Dalotia</i> abdominal segments</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0110" role="paragraph">(A) Scheme for 10x scRNA-seq.</div><div id="fspara0115" role="paragraph">(B) UMAP of cells separated by segment (green, Segment 6; magenta, Segment 7). Cell type 6 is composed primarily of Segment 7 cells.</div><div id="fspara0120" role="paragraph">(C) UMAP of cell types and their grouping into cell classes.</div><div id="fspara0125" role="paragraph">(D–G) Expression of solvent pathway enzymes: <i>MFASN</i> (D), <i>TG-FAR</i> (E), <i>TG-CYP4G</i> (F), and <i>TG-</i>α<i>Est</i> (G). Numbers correspond to cell types in (C).</div><div id="fspara0130" role="paragraph">See also <a id="crosref2455" href="#figs4">Figure&nbsp;S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/81e2f6a1-5023-4144-a6a5-44cacfea880c/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.014/asset/d22ecbd5-e676-4add-8386-750362e20abe/main.assets/figs4.jpg" height="732" width="677" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/81e2f6a1-5023-4144-a6a5-44cacfea880c/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">scRNA-seq data processing and cell clustering and classification, related to <a id="crosref2550" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0395" role="paragraph">A: Variable feature plot showing the 3,000 transcripts that exhibit high cell-to-cell variation across the dataset (i.e., highly expressed in some cells and not others). These transcripts were used for all downstream analyses and clustering. B, C: The total number of cells (B) and number of transcripts per cell (C) from Segment 6 (control) and 7 (gland) of all concatenated 10x runs after filtering. In C, the middle bars of the boxplots represent the median, the boxes denote the interquartile range, while the whiskers show the minimum and maximum. D: Neighbor joining clustering (NJ) on a Bray-Curtis similarity matrix (n&nbsp;= 10,000 bootstraps) based on the normalized and scaled cluster average expression profiles of the 3000 most variable transcripts (A). Clades form the basis of further grouping the cell types into 14 higher order cell classes. E: Robustness of the 14 cell class assignments tested using a Random Forest classifier. The graph (top) shows the robustness (class error rate) of the cell class assignments running a Random Forest classifier over 1000 generations. The heatmap shows the pairwise-class error rates of the assigned groups after 1000 generations. Only cell classes with a relatively low cell number were observed to have a higher error rate. F: UMAP plot of <i>Dalotia</i> abdominal Segment 6 and 7 cell atlas showing 14 cell classes.</div></div></div></div></figcaption></figure></div><div id="p0090" role="paragraph">Using each cell type’s mean expression of the 3,000 transcript set, we inferred a tree of transcriptomic similarity, and used the topology to test whether the 26 cell types could be grouped into higher-order classes via a random forest classifier (<a id="crosref0535" href="#figs4">Figures S4</a>D and S4E). This classifier produced strong support for 14 higher groupings, herein called “cell classes” (<a id="crosref0540" href="#fig3">Figures 3</a>C, <a id="crosref0545" href="#figs4">S4</a>E, and S4F). Overall, 85% of all cells were correctly assigned to one cell class (<a id="crosref0550" href="#figs4">Figure&nbsp;S4</a>E). The 26 cell types and 14 cell classes thus represent a robust classification of cell types in abdominal segments 6 and 7 of <i>Dalotia</i>.</div></section><section id="sec-2-4"><h3>Ester branch evolution via co-option of fat body enzymes</h3><div id="p0095" role="paragraph">Within the abdominal cell atlas, a single cluster co-expressed the four solvent pathway enzymes (<a id="crosref0555" href="#fig3">Figures 3</a>D–3G and <a id="crosref0560" href="#figs5">S5</a>A–S5F). This cell type—cluster number 6—is thus the solvent cell type, a determination further supported it being almost exclusively restricted to segment 7 (<a id="crosref0565" href="#fig3">Figure&nbsp;3</a>B). To investigate solvent pathway evolution, we searched for features shared with other cell types. We observed that several cell types—1, 5, 13, 14, and 18—expressed <i>MFASN</i> to high levels, indicating fatty acid synthesis (<a id="crosref0570" href="#fig3">Figures 3</a>D and <a id="optLQ4rl3poUe" href="#figs5">S5</a>A). We deduced that these cell types comprise <i>Dalotia</i>’s fat body—an ancient insect organ sharing many functions with vertebrate liver and adipose tissue (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Arrese and Soulages, 2010</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Arrese, E.L. ∙ Soulages, J.L.</div><div><strong>Insect fat body: energy, metabolism, and regulation</strong></div><div><em>Annu. Rev. Entomol.</em> 2010; <strong>55</strong>:207-225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.L.ArreseJ.L.SoulagesInsect+fat+body%3A+energy%2C+metabolism%2C+and+regulationAnnu.+Rev.+Entomol.552010207225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Fat body cells are sites of fat and glucose storage, as well as lipid synthesis, and are identifiable as lipid droplet-containing cells forming loose tissue throughout the abdomen. HCR revealed strong <i>MFASN</i> expression in <i>Dalotia</i>’s fat body (<a id="crosref0580" href="#figs5">Figure&nbsp;S5</a>G). Phylogenetic analysis shows that <i>MFASN</i> is a single copy gene present in all beetles, as well as in <i>Drosophila</i> (<a id="crosref0585" href="#fig4">Figure&nbsp;4</a>L). <i>MFASN</i>’s deep conservation and broad expression across <i>Dalotia</i>’s fat body cell types imply an ancestral function in the insect fat body. We HCR-labeled <i>MFASN</i> in a distantly related outgroup beetle, <i>Tribolium</i> (Tenebrionidae), revealing strong, fat body-specific expression (<a id="crosref0590" href="#figs5">Figure&nbsp;S5</a>H). <i>MFASN</i>’s ancestral function was indeed in the fat body; production of C12 and C10 precursors evolved by co-opting <i>MFASN</i> into solvent cells.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/fcac9bf3-f69b-4f1a-af8f-d00fe996615b/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.014/asset/70cfaef6-e221-48c9-b287-35dda1da7da2/main.assets/gr4.jpg" height="821" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/fcac9bf3-f69b-4f1a-af8f-d00fe996615b/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Solvent pathway evolution</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0140" role="paragraph">(A) <i>TG-</i>α<i>Est</i> HCR (magenta) labels segmental bands of ventral fat body (VFB; green, WGA-stained membranes).</div><div id="fspara0145" role="paragraph">(B and C) BODIPY (magenta) labels lipid droplets in VFB (green, DAPI).</div><div id="fspara0150" role="paragraph">(D and E) <i>O-CYP4G</i> HCR (magenta) stains oenocytes in ventral abdomen. (D) Intermingling of oenocytes and VFB (green, WGA). (E) Single oenocyte with <i>O-CYP4G</i> expression (green, DAPI).</div><div id="fspara0155" role="paragraph">(F–J) GC traces of <i>Dalotia</i> CHCs. (F) Wild-type with major peaks identified. (G) <i>O-CYP4G</i> RNAi removes CHCs. (H) <i>O-FASN</i> RNAi removes most CHCs. (I) <i>O-FAR</i> RNAi removes CHCs. (J) <i>O-ELO</i> RNAi shifts CHCs to shorter chain lengths.</div><div id="fspara0160" role="paragraph">(K) Expression dot plot of <i>FASN</i>, <i>FAR</i>, <i>CYP4G</i> and <i>ELO</i> enzyme copies across abdominal cell types. Dot size, percent of cells of each type expressing transcript; color, mean expression level.</div><div id="fspara0165" role="paragraph">(L) ML tree of <i>FASN</i> genes from <i>Dalotia</i>, other beetles and <i>Drosophila</i> (LG+F+R6 model). Collapsed clades signify multiple paralogs. <i>Drusilla</i> and <i>Coproporus MFASN F1</i> and <i>F2</i> are fragments, likely of the same gene.</div><div id="fspara0170" role="paragraph">(M) ML tree of <i>CYP4G</i> genes (LG+I+G4 model). Duplication in Aleocharinae yields putative oenocyte and tergal gland (TG) clades.</div><div id="fspara0175" role="paragraph">(N) <i>Dalotia</i> genomic interval showing <i>TG-CYP4G</i> and <i>O-CYP4G</i> tandem copies.</div><div id="fspara0180" role="paragraph">(O) Model of solvent pathway assembly via co-option or duplication of VFB and oenocyte enzymes.</div><div id="fspara0185" role="paragraph">See also <a id="crosref2460" href="#figs5">Figure&nbsp;S5</a>, <a id="crosref2465" href="#mmc7">Table S1</a>, and <a id="crosref2470" href="#mmc1">Data S1</a> and <a id="crosref2475" href="#mmc2">S2</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/69afec1c-2edd-4ea1-b086-3310e602f42c/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.014/asset/154b76d6-11d8-422b-9226-f8efcf730795/main.assets/figs5.jpg" height="939" width="680" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/69afec1c-2edd-4ea1-b086-3310e602f42c/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Solvent cell evolution, related to <a id="crosref2555" href="#fig4">Figures 4</a> and <a id="crosref2560" href="#fig5">5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0405" role="paragraph">A-D: Violin plots showing expression of <i>MFASN</i> (A), <i>TG-CYP4G</i> (B), <i>TG-FAR</i> (C) and <i>TG-αEST</i> (D) in each cell across all cell types in the abdominal segment cell type atlas. E: Expression of <i>TG-CYP4G</i> by cell type in each of the five segment-specific 10x runs. <i>TG-CYP4G</i> shows a generally high expression across all cell types, but almost all expression in non-solvent cells comes from Segment 6 Run 1 and Segment 7 Run 1. This global expression, with very low levels in almost all cell types, is likely the result of the ‘ambient mRNA’-effect, which is typical of droplet-based methods. Ambient mRNAs are most likely derived from high-abundance transcripts that are leaked from dying cells and bind to beads during the microfluidic step in the 10x workflow. Only in solvent cells are the UMI count levels were high, indicating that the low levels mRNA detected in non-solvent cells likely represent ambient mRNAs. F: To independently evaluate the cell type specificity of <i>TG-CYP4G</i>, we used HCR and imaged the whole beetle abdomen, confirming that its transcription is highly specific to the solvent cells. G: HCR in <i>Dalotia</i> reveals <i>MFASN</i> in the solvent cells (left panel) and dorsal fat body (DFB; right panel). C: HCR of the <i>MFASN</i> ortholog in <i>T.&nbsp;castaneum</i> reveals strong expression in fat body. H: UMAP plot of <i>Dalotia</i> abdominal segment 6 and 7 cell atlas highlighting the dorsal (DFB; orange) and ventral fat body (VFB; red) cells. I’, I’’: Expression of the <i>Pumpless</i> (I’) and <i>Apolipophorin</i> (I’’) fat body marker genes show strong expression in DFB but not VFB.</div></div></div></div></figcaption></figure></div><div id="p0100" role="paragraph">Fat body cells can be highly heterogenous in insects (<span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Haunerland and Shirk, 1995</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Haunerland, N.H. ∙ Shirk, P.D.</div><div><strong>Regional and Functional Differentiation in the Insect Fact Body</strong></div><div><em>Annu. Rev. Entomol.</em> 1995; <strong>40</strong>:121-145</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.H.HaunerlandP.D.ShirkRegional+and+Functional+Differentiation+in+the+Insect+Fact+BodyAnnu.+Rev.+Entomol.401995121145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and this is so in <i>Dalotia</i>. First, cell types 1, 5, 14 and 18—but not 13—cluster-based on mean transcript expression (<a id="crosref0600" href="#figs4">Figure&nbsp;S4</a>D), implying two different fat body classes. The four former cell types also express <i>Drosophila</i> fat body markers including <i>pumpless</i> and <i>apolipophorin</i>, whereas cell type 13 does not (<a id="crosref0605" href="#fig5">Figures 5</a>I–5I″). Most strikingly, the two fat body classes differ in expression of α<i>-</i>esterases (<a id="crosref0610" href="#mmc1">Data S1</a>): all α<i>-</i>esterase copies except one are expressed in cell types 1, 5, 14, and 18 (and occasional other cell types) (<a id="crosref0615" href="#mmc1">Data S1</a>), whereas cell type 13 expresses only <i>TG-</i>α<i>Est</i>, shared exclusively with solvent cells (<a id="crosref0620" href="#fig3">Figure&nbsp;3</a>G; <a id="crosref0625" href="#mmc1">Data S1</a>). Using HCR of <i>TG-</i>α<i>Est</i>, we identified cell type 13 as subepidermal bands of fat body containing large lipid droplets (<a id="crosref0630" href="#fig4">Figures 4</a>A–4C). We refer to cell type 13 as&nbsp;“ventral fat body” (VFB) and other fat body types distributed more dorsally as “dorsal fat body” (DFB; cell types 1, 5, 14+18) (<a id="crosrefs0010" href="#figs4">Figures S4</a>D and <a id="crosref0635" href="#figs5">S5</a>I). Given the general function of α<i>-</i>esterases in the fat body, that sister paralogs of <i>TG-</i>α<i>Est</i> are expressed primarily in fat body, and that <i>TG-</i>α<i>Est</i> itself functions in the VFB, we infer that the solvent pathway’s ester branch evolved by recruiting a fat body α<i>-</i>esterase to function downstream of MFASN (<a id="crosref0640" href="#fig4">Figure&nbsp;4</a>O). Importantly, <i>TG-</i>α<i>Est</i> is an aleocharine-specific gene that is absent in other beetles, including outgroup Tachyporinae (<a id="crosref0645" href="#mmc1">Data S1</a>). Furthermore, the duplicate occurs in genomes of species known to produce esters, including <i>Dalotia</i> (tribe Athetini), <i>Drusilla</i>, and <i>Platyusa</i> (both Lomechusini), but is absent from <i>Aleochara</i> (Aleocharini) that does not (<a id="crosref0650" href="#mmc1">Data S1</a>).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/84df5429-e453-4df5-9f8f-6ea107e96036/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.014/asset/cd0a7829-150c-4141-94c6-dcc4505a900f/main.assets/gr5.jpg" height="644" width="752" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/84df5429-e453-4df5-9f8f-6ea107e96036/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Solvent cell evolution</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0195" role="paragraph">(A) GEP usage by cell type. Some cell types were grouped into classes for clarity and to facilitate computation.</div><div id="fspara0200" role="paragraph">(B) GEP9 and 17 usage by each cell type reveals solvent cells are a GEP9/17 hybrid.</div><div id="fspara0205" role="paragraph">(C) UMAP of cell types (from <a id="crosref2480" href="#fig3">Figure&nbsp;3</a>C) with cuticle cell class and solvent cells highlighted.</div><div id="fspara0210" role="paragraph">(D) GEP17 usage across cell types with cuticle cells and solvent cells indicated.</div><div id="fspara0215" role="paragraph">(E) UMAP with oenocytes, VFB, and solvent cells highlighted.</div><div id="fspara0220" role="paragraph">(F) GEP9 across cell types with oenocytes, VFB and solvent cells indicated (inset is magnified oenocyte cluster).</div><div id="fspara0225" role="paragraph">(G) Cartoon showing segmentally repeated VFB and oenocytes in ventral abdomen.</div><div id="fspara0230" role="paragraph">(H) Model of solvent cell type evolution: cuticle cells comprising intersegmental membrane between A6 and A7 express GEP17 as principal GEP; acquisition of GEP9 from oenocytes and VFB transformed them into solvent cells.</div><div id="fspara0235" role="paragraph">See also <a id="crosref2485" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref2490" href="#mmc3">Data S3</a> and <a id="crosref2495" href="#mmc4">S4</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>Alkane branch evolution via duplication of oenocyte enzymes</h3><div id="p0105" role="paragraph">Many aleocharines synthesize esters as the putative BQ solvent but lack alkanes, which may represent a more recent evolutionarily addition (<span class="dropBlock reference-citations"><a id="body-ref-sref141-4" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-4" aria-controls="sref141-4">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Recruitment of MFASN and TG-αEst may thus have been an early step in solvent pathway evolution, with the alkane branch added subsequently. We noted that one cell type—23—expressed a paralog of <i>TG-CYP4G</i> (<a id="crosref0660" href="#fig4">Figure&nbsp;4</a>K; <a id="crosref0665" href="#mmc2">Data S2</a>A). CYP4Gs have been shown to function canonically in oenocytes—ancient pheromone-producing cells found in all insects (<span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Makki et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Makki, R. ∙ Cinnamon, E. ∙ Gould, A.P.</div><div><strong>The development and functions of oenocytes</strong></div><div><em>Annu. Rev. Entomol.</em> 2014; <strong>59</strong>:405-425</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.MakkiE.CinnamonA.P.GouldThe+development+and+functions+of+oenocytesAnnu.+Rev.+Entomol.592014405425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Oenocytes synthesize very long-chain (C25–C40) alkanes and alkenes known as cuticular hydrocarbons (CHCs) that are secreted onto the body and encode a chemical signature of species identity (<span class="dropBlock reference-citations"><a id="body-ref-sref16" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16" aria-controls="sref16">Billeter et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Billeter, J.-C. ∙ Atallah, J. ∙ Krupp, J.J. ...</div><div><strong>Specialized cells tag sexual and species identity in Drosophila melanogaster</strong></div><div><em>Nature.</em> 2009; <strong>461</strong>:987-991</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.-C.BilleterJ.AtallahJ.J.KruppJ.G.MillarJ.D.LevineSpecialized+cells+tag+sexual+and+species+identity+in+Drosophila+melanogasterNature4612009987991" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). CHCs also form a waxy barrier against desiccation (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Koto et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Koto, A. ∙ Motoyama, N. ∙ Tahara, H. ...</div><div><strong>Oxytocin/vasopressin-like peptide inotocin regulates cuticular hydrocarbon synthesis and water balancing in ants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:5597-5606</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KotoN.MotoyamaH.TaharaS.McGregorM.MoriyamaT.OkabeM.MiuraL.KellerOxytocin%2Fvasopressin-like+peptide+inotocin+regulates+cuticular+hydrocarbon+synthesis+and+water+balancing+in+antsProc.+Natl.+Acad.+Sci.+USA116201955975606" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119-2" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-2" aria-controls="sref119-2">Qiu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref119-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Qiu, Y. ∙ Tittiger, C. ∙ Wicker-Thomas, C. ...</div><div><strong>An insect-specific P450 oxidative decarbonylase for cuticular hydrocarbon biosynthesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:14858-14863</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.QiuC.TittigerC.Wicker-ThomasG.Le+GoffS.YoungE.WajnbergT.FricauxN.TaquetG.J.BlomquistR.FeyereisenAn+insect-specific+P450+oxidative+decarbonylase+for+cuticular+hydrocarbon+biosynthesisProc.+Natl.+Acad.+Sci.+USA10920121485814863" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In oenocytes, CYP4G performs terminal decarbonylation to yield the secreted CHCs (<span class="dropBlock reference-citations"><a id="body-ref-sref119-3" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119-3" aria-controls="sref119-3">Qiu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref119-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Qiu, Y. ∙ Tittiger, C. ∙ Wicker-Thomas, C. ...</div><div><strong>An insect-specific P450 oxidative decarbonylase for cuticular hydrocarbon biosynthesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:14858-14863</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.QiuC.TittigerC.Wicker-ThomasG.Le+GoffS.YoungE.WajnbergT.FricauxN.TaquetG.J.BlomquistR.FeyereisenAn+insect-specific+P450+oxidative+decarbonylase+for+cuticular+hydrocarbon+biosynthesisProc.+Natl.+Acad.+Sci.+USA10920121485814863" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We HCR-labeled this <i>CYP4G</i>, revealing clusters of enlarged cells subepidermally in the ventral abdomen, often intermingled with VFB (<a id="crosref0695" href="#fig4">Figures 4</a>D and 4E). Silencing this locus led to near-total loss of <i>Dalotia</i>’s CHCs (<a id="crosref0700" href="#fig4">Figures 4</a>F and 4G; <a id="crosref0705" href="#mmc2">Data S2</a>E), confirming that these cells are the oenocytes. We name this enzyme <i>O-CYP4G</i> (<i>Oenocyte-CYP4G</i>).</div><div id="p0110" role="paragraph">Recent studies have delineated a conserved oenocyte CHC pathway (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Blomquist and Ginzel, 2021</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Blomquist, G.J. ∙ Ginzel, M.D.</div><div><strong>Chemical Ecology, Biochemistry, and Molecular Biology of Insect Hydrocarbons</strong></div><div><em>Annu. Rev. Entomol.</em> 2021; <strong>66</strong>:45-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.J.BlomquistM.D.GinzelChemical+Ecology%2C+Biochemistry%2C+and+Molecular+Biology+of+Insect+HydrocarbonsAnnu.+Rev.+Entomol.6620214560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This pathway bears striking similarity to the alkane branch of the solvent pathway: oenocytes use FASNs to produce fatty acids that are reduced to aldehydes by FARs before decarbonylation by CYP4G (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Blomquist and Ginzel, 2021</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Blomquist, G.J. ∙ Ginzel, M.D.</div><div><strong>Chemical Ecology, Biochemistry, and Molecular Biology of Insect Hydrocarbons</strong></div><div><em>Annu. Rev. Entomol.</em> 2021; <strong>66</strong>:45-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.J.BlomquistM.D.GinzelChemical+Ecology%2C+Biochemistry%2C+and+Molecular+Biology+of+Insect+HydrocarbonsAnnu.+Rev.+Entomol.6620214560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">Holze et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Holze, H. ∙ Schrader, L. ∙ Buellesbach, J.</div><div><strong>Advances in deciphering the genetic basis of insect cuticular hydrocarbon biosynthesis and variation</strong></div><div><em>Heredity.</em> 2021; <strong>126</strong>:219-234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.HolzeL.SchraderJ.BuellesbachAdvances+in+deciphering+the+genetic+basis+of+insect+cuticular+hydrocarbon+biosynthesis+and+variationHeredity1262021219234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—modifications that mirror exactly the steps for undecane synthesis (<a id="crosref0725" href="#fig2">Figure&nbsp;2</a>J). Consistent with the deep conservation of the insect CHC pathway, we find that, in addition to <i>O-CYP4G</i>, cell type 23 expresses three <i>FASN</i> paralogs and four <i>FAR</i>s (<a id="crosref0730" href="#fig4">Figure&nbsp;4</a>K; <a id="crosref0735" href="#mmc2">Data S2</a>B and S2C). As with <i>O-CYP4G</i>, silencing selected copies of these enzymes strongly diminished CHC production (<a id="crosref0740" href="#fig4">Figures 4</a>H and 4I; <a id="crosref0745" href="#mmc2">Data S2</a>F and S2G). Hence, <i>Dalotia</i> expresses parallel alkane pathways in ancient oenocytes and novel solvent cells, their close similarity implying common ancestry. Indeed, we find that <i>TG-CYP4G</i> and <i>O-CYP4G</i> are sister duplicates found only in Aleocharinae (<a id="crosref0750" href="#fig4">Figure&nbsp;4</a>M); in <i>Dalotia</i>, the two copies sit tandemly in the genome (<a id="crosref0755" href="#fig4">Figure&nbsp;4</a>N). The ancestral CYP4G was likely an oenocyte enzyme that synthesized CHCs; duplication gave rise to&nbsp;oenocyte and solvent cell copies (<a id="crosref0760" href="#fig4">Figure&nbsp;4</a>O). Indeed, <i>TG-CYP4G</i> is conserved in species that produce alkanes (<i>Aleochara</i>, <i>Drusilla</i>, <i>Platyusa</i>, and <i>Dalotia</i>) but is absent from <i>Lissagria</i> that does not (<a id="crosref0765" href="#fig4">Figure&nbsp;4</a>M). Whether duplication of an <i>O-FAR</i> led to <i>TG-FAR</i> is less clear; <i>FAR</i>s undergo extensive gene birth-and-death, so their history is challenging to infer (<span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Finet et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Finet, C. ∙ Slavik, K. ∙ Pu, J. ...</div><div><strong>Birth-and-death evolution of the fatty acyl-CoA reductase (FAR) gene family and diversification of cuticular hydrocarbon synthesis in Drosophila</strong></div><div><em>Genome Biol. Evol.</em> 2019; <strong>11</strong>:1541-1551</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.FinetK.SlavikJ.PuS.B.CarrollH.ChungBirth-and-death+evolution+of+the+fatty+acyl-CoA+reductase+%28FAR%29+gene+family+and+diversification+of+cuticular+hydrocarbon+synthesis+in+DrosophilaGenome+Biol.+Evol.11201915411551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>TG-FAR</i> is not a sister paralog of an <i>O-FAR</i> (<a id="crosref0775" href="#mmc2">Data S2</a>J), so its role in alkane synthesis may have arisen convergently. Regardless, the CHC pathway represented a pre-existing template: by recruiting a FAR and CYP4G downstream of MFASN, alkanes were added to the defensive secretion (<a id="crosref0780" href="#fig4">Figure&nbsp;4</a>O).</div><div id="p0115" role="paragraph">Despite similar enzyme logic, the products of the two pathways are markedly different: the CHC pathway produces very-long-chain waxy hydrocarbons, whereas the solvent pathway makes a medium-chain volatile liquid. Oenocytes make longer compounds via elongases (ELOs) (<span class="dropBlock reference-citations"><a id="body-ref-sref18-3" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-3" aria-controls="sref18-3">Blomquist and Ginzel, 2021</a><div class="dropBlock__holder" data-db-target-of="sref18-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Blomquist, G.J. ∙ Ginzel, M.D.</div><div><strong>Chemical Ecology, Biochemistry, and Molecular Biology of Insect Hydrocarbons</strong></div><div><em>Annu. Rev. Entomol.</em> 2021; <strong>66</strong>:45-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.J.BlomquistM.D.GinzelChemical+Ecology%2C+Biochemistry%2C+and+Molecular+Biology+of+Insect+HydrocarbonsAnnu.+Rev.+Entomol.6620214560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">Holze et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Holze, H. ∙ Schrader, L. ∙ Buellesbach, J.</div><div><strong>Advances in deciphering the genetic basis of insect cuticular hydrocarbon biosynthesis and variation</strong></div><div><em>Heredity.</em> 2021; <strong>126</strong>:219-234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.HolzeL.SchraderJ.BuellesbachAdvances+in+deciphering+the+genetic+basis+of+insect+cuticular+hydrocarbon+biosynthesis+and+variationHeredity1262021219234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0795" href="#fig4">Figure&nbsp;4</a>O). We find <i>Dalotia</i> oenocytes express five ELOs (<a id="crosref0800" href="#fig4">Figure&nbsp;4</a>K; <a id="crosref0805" href="#mmc2">Data S2</a>D), knockdown of which caused an altered profile with many shorter chain compounds (<a id="crosref0810" href="#fig4">Figure&nbsp;4</a>J; <a id="crosref0815" href="#mmc2">Data S2</a>I) without reducing total CHC levels (<a id="crosref0820" href="#mmc2">Data S2</a>H). In contrast, solvent cells express no ELOs (<a id="crosref0825" href="#fig4">Figure&nbsp;4</a>K; <a id="crosref0830" href="#mmc2">Data S2</a>D). Hence, the solvent pathway evolved via selective FAR and CYP4G recruitment without an ELO, enabling medium-chain biosynthesis (<a id="crosref0835" href="#fig4">Figure&nbsp;4</a>O).</div></section><section id="sec-2-6"><h3>Solvent cell evolution via transcriptomic hybridization of ancient cell types</h3><div id="p0120" role="paragraph">Solvent cells comprise part of the beetle’s cuticle, forming part&nbsp;of the intersegmental membrane joining segments A6 and&nbsp;A7 (<a id="opt1ZUVHJdtXP" href="#fig1">Figure 1</a>E). Developmentally, the cells derive from the epidermal posterior (P) compartment of A6 and express Engrailed (<a id="crosref0840" href="#fig1">Figure&nbsp;1</a>F; <a id="crosref0845" href="#mmc2">Data S2</a>K and S2L), the P compartment selector transcription factor (<span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Morata and Lawrence, 1975</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Morata, G. ∙ Lawrence, P.A.</div><div><strong>Control of compartment development by the engrailed gene in Drosophila</strong></div><div><em>Nature.</em> 1975; <strong>255</strong>:614-617</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.MorataP.A.LawrenceControl+of+compartment+development+by+the+engrailed+gene+in+DrosophilaNature2551975614617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Like the surrounding epidermis, solvent cells produce chitin, which forms an internal lining to the reservoir and is continuous with the rest of the exoskeleton (<span class="dropBlock reference-citations"><a id="body-ref-sref141-5" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-5" aria-controls="sref141-5">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Despite their epidermal identity, solvent cells are enlarged, columnar secretory cells that produce undecane and esters. To understand how solvent cells acquired this property, we studied their transcriptomic relationship to other cell types within the abdomen. We employed consensus non-negative matrix factorization (cNMF) (<span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), performing an unsupervised search for constellations of significantly co-expressed transcripts across all cells within the 10x dataset, irrespective of their cell type. cNMF applies iterative NMF treatments on a transcripts-by-transcripts matrix to identify groups of significantly co-expressed genes (“gene expression programs” or “GEPs”). GEPs discretize the transcriptome into building blocks that may be surrogates of cellular properties. A GEP used in one or a few cell types may confer aspects of cell identity; conversely, a GEP used by many cell types likely underlies a routine activity such as mitosis (<span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Employing cNMF, we determined that 20 GEPs accurately capture a decomposed representation of the total transcriptome of A6 and A7 (<a id="crosref0870" href="#mmc3">Data S3</a>). We calculated the proportional contribution of each GEP to each individual cell type’s transcriptome, depicted as a usage map (<a id="crosref0875" href="#fig5">Figure&nbsp;5</a>A). Visualizing transcriptome composition in this way, some GEPs appear cell-type-specific, such as GEP8 and GEP2—unique identity GEPs expressed in heart and midgut cells, respectively. Conversely, most cell types express GEPs 18 and 12, implying common cellular activities.</div><div id="p0125" role="paragraph">Of particular interest are cell types composed of combinations of GEPs (<a id="crosref0880" href="#fig5">Figure&nbsp;5</a>A). These include the solvent cells, recovered as a composite of GEPs 17 and 9 (<a id="crosref0885" href="#fig5">Figures 5</a>A and 5B). Consistent with solvent cells being part of the cuticular epidermis, GEP17 is the principal GEP expressed by the cuticular cell class (<a id="crosref0890" href="#fig5">Figures 5</a>B–5D). This cell class is composed of cell types 0, 9, 17, and 20 (<a id="crosref0895" href="#fig3">Figures 3</a>C and <a id="crosref0900" href="#figs4">S4</a>D), which express multiple cuticle proteins and Laccase 2 that functions in cuticle tanning (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Arakane et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arakane, Y. ∙ Muthukrishnan, S. ∙ Beeman, R.W. ...</div><div><strong>Laccase 2 is the phenoloxidase gene required for beetle cuticle tanning</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:11337-11342</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ArakaneS.MuthukrishnanR.W.BeemanM.R.KanostK.J.KramerLaccase+2+is+the+phenoloxidase+gene+required+for+beetle+cuticle+tanningProc.+Natl.+Acad.+Sci.+USA10220051133711342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref0910" href="#figs6">Figure&nbsp;S6</a>F). However, solvent cells are clearly divergent from other cuticular cells in their additional expression of GEP9, which accounts for ∼20% of the transcriptome (<a id="crosref0915" href="#fig5">Figure&nbsp;5</a>B). The composition of GEP9 reveals that it is constituted by 61 core transcripts (applying a stringent <i>Z</i> score filter of &lt;0.002) (<a id="crosref0920" href="#mmc4">Data S4</a>A) that show strong and significant enrichment for biological processes related to lipid metabolism and fatty acid biosynthesis (<a id="crosref0925" href="#mmc4">Data S4</a>B–S4D). GEP9 is thus a transcriptional module that may endow solvent cells with their capacity for high level fatty acid production and modification.</div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/584ea0aa-67d4-4f3b-8ea0-a0a2eaa0127b/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.014/asset/ea283830-bf0a-4ff5-af59-9b1904c6008e/main.assets/figs6.jpg" height="857" width="656" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/584ea0aa-67d4-4f3b-8ea0-a0a2eaa0127b/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">BQ cell evolution, related to <a id="crosref2565" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0415" role="paragraph">A: BQs are synthesized from dietary Tyr but not from glucose. Representative mass spectra of molecular ion region of 1,4-BQ, 2-methl-1,4-BQ and 2-methoxy-3-methyl-1,4,-BQ from beetles fed with unlabeled <i>Dalotia</i> diet (control), food infused with <sup>13</sup>C<sub>6</sub>-glucose or <sup>13</sup>C<sub>6</sub>-Tyr. Spectra were recorded in single-ion mode. No enrichment was detected in any of the three BQs for the <sup>13</sup>C<sub>6</sub>-glucose fed beetles. In contrast, a strong [M+6]<sup>+</sup> enrichment was detected for all BQs in <sup>13</sup>C<sub>6</sub>-Tyr fed beetles. B: Functional characterization of Dmd. Levels of 1,4-BQ, 2-methyl-1,4-BQ and 2-methoxy-3-methyl-1,4,-BQ extracted from beetles injected with dsRNA against <i>GFP</i> (control) and <i>dmd</i>. Silencing <i>dmd</i> resulted in near-complete loss of BQs in <i>Dalotia</i>’s secretion. The horizontal middle bars represents the median, while the vertical bars show the minimum and maximum. C-E: Adult beetle morphology after late larval injection with dsRNA against <i>GFP</i> (C); control), <i>lac2</i> (D) and <i>dmd</i> (E). (F): Expression of <i>laccase-2</i> across all cell classes in the 10x data reveals strong expression cuticle cells. (G): Expression of <i>dmd</i> across all cell classes in the 10x data shows no tissue specific expression of <i>dmd</i> as BQ-cells are not captured by 10x. In F and G, the horizontal middle bars represents the median, while the vertical bars show the minimum and maximum. H: Reaction of the three HQ substrates 1,4-HQ, 2-Me-1,4-HQ and 2-MeO-3-Me-1,4-HQ with purified Dmd <i>in&nbsp;vitro</i> as compared to a non-enzyme control. BQ levels were measured using GC-FID. Size of the bars represent the median, while error bars denote the SE.</div></div></div></div></figcaption></figure></div><div id="p0130" role="paragraph">Remarkably, GEP9 is the principal GEP of both oenocytes and VFB (<a id="crosref0930" href="#fig5">Figures 5</a>A, 5B, 5E, and 5F). This expression module therefore defines the two cell types from which solvent pathway enzymes were co-opted or duplicated. Unlike solvent cells, VFB and oenocytes show no pronounced GEP usage beyond GEP9 (aside from GEPs 18 and 12 that most cell types express) (<a id="crosref0935" href="#fig5">Figure&nbsp;5</a>A). GEP9 thus likely contributes to the functional identity of these two cell types, which are both specialized for fatty acid biosynthesis. It follows that GEP9 probably imparts this same function in solvent cells. A close relationship between solvent cells, VFB, and oenocytes is further supported by the three cell types forming a clade based on mean transcript expression (<a id="crosref0940" href="#figs4">Figure&nbsp;S4</a>D), and random forest classifies them into the same cell class (<a id="crosref0945" href="#figs4">Figures S4</a>E and S4F). That solvent cells are a novelty within <i>Dalotia</i>’s cuticle, whereas oenocytes and fat body cells are ancient, non-cuticular cell types in all insects, implies a specific evolutionary scenario (<a id="crosref0950" href="#fig5">Figures 5</a>G and 5H). We suggest that solvent cells arose via transcriptomic hybridization: they are a cuticular cell type, ancestrally comprising intersegmental membrane, that gained an expression module that evolved in oenocytes and fat body, equipping them for high-level fatty acid synthesis (<a id="crosref0955" href="#fig5">Figure&nbsp;5</a>H). As part of this process, the oenocytes and fat body also contributed distinct branches of the solvent pathway (<a id="crosref0960" href="#fig4">Figure&nbsp;4</a>O).</div></section><section id="sec-2-7"><h3>The BQ cell type</h3><div id="p0135" role="paragraph">We next focused on BQ synthesis. Although defensive BQs have arisen in myriapods, arachnids, earwigs, termites, cockroaches, grasshoppers, capsid bugs, and at least seven times in beetles (<span class="dropBlock reference-citations"><a id="body-ref-sref151" href="#" role="doc-biblioref" data-xml-rid="bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151" aria-controls="sref151">Wagner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref151"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">158.</div><div>Wagner, J.M. ∙ Naragon, T.H. ∙ Brückner, A.</div><div><strong>Benzoquinones in the defensive secretion of a bug (Pamillia behrensii): a common chemical trait retrieved in the Heteroptera</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.WagnerT.H.NaragonA.Br%C3%BCcknerBenzoquinones+in+the+defensive+secretion+of+a+bug+%28Pamillia+behrensii%29%3A+a+common+chemical+trait+retrieved+in+the+HeteropterabioRxiv202010.1101%2F2020.12.11.421891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), their mechanistic origins were hitherto unknown in any species. Aromatic compounds in animals are often acquired from dietary aromatic amino acids or symbiotic microbes, or sometimes synthesized <i>de novo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Brückner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Brückner, A. ∙ Kaltenpoth, M. ∙ Heethoff, M.</div><div><strong>De novo biosynthesis of simple aromatic compounds by an arthropod (<i>Archegozetes longisetosus</i>)</strong></div><div><em>Proc. Biol. Sci.</em> 2020; <strong>287</strong>:20201429</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerM.KaltenpothM.HeethoffDe+novo+biosynthesis+of+simple+aromatic+compounds+by+an+arthropod+%28Archegozetes+longisetosus%29Proc.+Biol.+Sci.287202020201429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref148" href="#" role="doc-biblioref" data-xml-rid="bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148" aria-controls="sref148">Torres et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref148"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Torres, J.P. ∙ Lin, Z. ∙ Winter, J.M. ...</div><div><strong>Animal biosynthesis of complex polyketides in a photosynthetic partnership</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2882</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.P.TorresZ.LinJ.M.WinterP.J.KrugE.W.SchmidtAnimal+biosynthesis+of+complex+polyketides+in+a+photosynthetic+partnershipNat.+Commun.1120202882" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To infer the synthetic route, we fed <i>Dalotia</i> D-glucose-<sup>13</sup>C<sub>6</sub> and observed negligible <sup>13</sup>C incorporation into the BQs (<a id="crosref0980" href="#figs6">Figure&nbsp;S6</a>A), arguing against complete <i>de&nbsp;novo</i> synthesis. Conversely, feeding Tyr-<sup>13</sup>C<sub>6</sub> or Phe-<sup>13</sup>C<sub>6</sub> led to strong <sup>13</sup>C incorporation, with molecular weights of all BQs increasing by exactly 6 (<a id="crosref0985" href="#figs6">Figure&nbsp;S6</a>A). The BQ’s benzene rings thus derive from dietary aromatic amino acids.</div><div id="p0140" role="paragraph">Beyond their use in chemical defense, quinones play key roles in insect metabolism: ubiquinone (coenzyme Q<sub>10</sub>) is a redox-active compound synthesized in mitochondria where it functions in electron transport (<span class="dropBlock reference-citations"><a id="body-ref-sref140" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140" aria-controls="sref140">Stefely and Pagliarini, 2017</a><div class="dropBlock__holder" data-db-target-of="sref140"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Stefely, J.A. ∙ Pagliarini, D.J.</div><div><strong>Biochemistry of Mitochondrial Coenzyme Q Biosynthesis</strong></div><div><em>Trends Biochem. Sci.</em> 2017; <strong>42</strong>:824-843</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.A.StefelyD.J.PagliariniBiochemistry+of+Mitochondrial+Coenzyme+Q+BiosynthesisTrends+Biochem.+Sci.422017824843" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); additionally, quinone intermediates arise during exoskeleton maturation (cuticle tanning) (<span class="dropBlock reference-citations"><a id="body-ref-sref104-1" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-1" aria-controls="sref104-1">Noh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref104-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Noh, M.Y. ∙ Muthukrishnan, S. ∙ Kramer, K.J. ...</div><div><strong>Cuticle formation and pigmentation in beetles</strong></div><div><em>Curr. Opin. Insect Sci.</em> 2016; <strong>17</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Y.NohS.MuthukrishnanK.J.KramerY.ArakaneCuticle+formation+and+pigmentation+in+beetlesCurr.+Opin.+Insect+Sci.17201619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In both contexts, aromatic amino acids are the precursors. We asked whether these ancient pathways could give clues to BQ synthesis in the tergal gland. In cuticle tanning, oxidation of Tyr-derived dopa and dopamine creates quinones that are pigment precursors and protein crosslinkers used for cuticle hardening (sclerotization) (<span class="dropBlock reference-citations"><a id="body-ref-sref104-2" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104-2" aria-controls="sref104-2">Noh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref104-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Noh, M.Y. ∙ Muthukrishnan, S. ∙ Kramer, K.J. ...</div><div><strong>Cuticle formation and pigmentation in beetles</strong></div><div><em>Curr. Opin. Insect Sci.</em> 2016; <strong>17</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Y.NohS.MuthukrishnanK.J.KramerY.ArakaneCuticle+formation+and+pigmentation+in+beetlesCurr.+Opin.+Insect+Sci.17201619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Oxidation is mediated by Laccase 2 (Lac2), a secreted multicopper oxidase (MCO) (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Asano et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Asano, T. ∙ Seto, Y. ∙ Hashimoto, K. ...</div><div><strong>Mini-review an insect-specific system for terrestrialization: Laccase-mediated cuticle formation</strong></div><div><em>Insect Biochem. Mol. Biol.</em> 2019; <strong>108</strong>:61-70</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.AsanoY.SetoK.HashimotoH.KurushimaMini-review+an+insect-specific+system+for+terrestrialization%3A+Laccase-mediated+cuticle+formationInsect+Biochem.+Mol.+Biol.10820196170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A predicted MCO was strongly upregulated in the second cell type within the gland—the bulbous secretory cells (<a id="crosref1010" href="#figs2">Figure&nbsp;S2</a>C). This transcript encodes a secreted protein with three cupredoxin domains in the same configuration as other laccases (<a id="crosref1015" href="#fig6">Figure&nbsp;6</a>A) (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Dwivedi et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Dwivedi, U.N. ∙ Singh, P. ∙ Pandey, V.P. ...</div><div><strong>Structure–function relationship among bacterial, fungal and plant laccases</strong></div><div><em>J.&nbsp;Mol. Catal. B Enzym.</em> 2011; <strong>68</strong>:117-128</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.N.DwivediP.SinghV.P.PandeyA.KumarStructure%E2%80%93function+relationship+among+bacterial%2C+fungal+and+plant+laccasesJ.%C2%A0Mol.+Catal.+B+Enzym.682011117128" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Silencing this laccase caused near-total loss of all three BQs, without affecting the solvents (<a id="crosref1025" href="#fig6">Figures 6</a>B and <a id="crosref1030" href="#figs6">S6</a>B). We name this laccase “Decommissioned” (Dmd) after its loss-of-function phenotype where the irritant is eliminated from the secretion.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/815e9a68-dd13-4177-8b9f-fa054eae0806/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.014/asset/d4bba9d5-0132-435e-accc-52c8c6313687/main.assets/gr6.jpg" height="941" width="732" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/815e9a68-dd13-4177-8b9f-fa054eae0806/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">The BQ cell type</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0245" role="paragraph">(A) Dmd domain structure with homology to Lac1 and 2.</div><div id="fspara0250" role="paragraph">(B) GC traces of gland secretions. Top: wild-type; bottom: <i>dmd</i> RNAi.</div><div id="fspara0255" role="paragraph">(C and D) <i>dmd</i> HCR (magenta; green, WGA).</div><div id="fspara0260" role="paragraph">(E and F) Single ion GC traces of HQ molecular ions 2-Me-1,4-HQ (<i>m/z</i>&nbsp;= 124) (E) and 2-MeO-3-Me-1,4-HQ (<i>m/z</i>&nbsp;= 154) (F) in wild-type (top) and <i>dmd</i>-RNAi beetles (bottom). Asterisks in (F) mark a non-target compound with <i>m/z</i> = 154.</div><div id="fspara0265" role="paragraph">(G) Unrooted ML tree of insect laccases with aleocharine-specific Dmd clade (LG+I+G4 model).</div><div id="fspara0270" role="paragraph">(H–J) Dmd activity assayed by GC-MS when purified protein is provided with HQs. GC traces show efficient BQ production in Dmd’s presence (upper trace); lower trace is no Dmd control. HQs: 1,4-HQ (H); 2-Me-1,4-HQ (I); 2-MeO-3-Me-1,4-HQ (J).</div><div id="fspara0275" role="paragraph">(K) GC quantification of BQs produced by Dmd, Lac1, and Lac2 with different HQ substrates (n&nbsp;= 3 replicates/enzyme). Size of bar represent the mean, while error bars are standard error of the mean.</div><div id="fspara0280" role="paragraph">(L) 2-MeO-3-Me-1,4-HQ versus ubiquinol.</div><div id="fspara0285" role="paragraph">(M) Diagnostic ions for 4-HB-<sup>13</sup>C<sub>6</sub> incorporation into 2-Me-1,4-BQ and 2-MeO-3-Me-1,4-BQ. Top trace: total ion current (TIC) wild-type chromatogram. Lower traces: diagnostic ions for BQs labeled with <sup>13</sup>C<sub>6</sub> (<i>m/z</i>&nbsp;= 128 for 2-Me-1,4-BQ and <i>m/z</i>&nbsp;= 158 for 2-MeO-3-Me-1,4-BQ). Intensity of <i>m/z</i>&nbsp;= 128 and <i>m/z</i>&nbsp;= 158 ions was magnified 100× in control and 4-HB-<sup>13</sup>C<sub>6</sub> fed beetles compared to TIC chromatogram.</div><div id="fspara0290" role="paragraph">(N) ML tree of CoQ3 enzymes (LG+I+G4 model) showing multiple copies in Aleocharinae (green boxed clade), including paralog MeOS.</div><div id="fspara0295" role="paragraph">(O) GC trace following <i>meos</i> RNAi shows specific loss of 2-MeO-3-Me-1,4-BQ.</div><div id="fspara0300" role="paragraph">(P) Model of BQ cell function: Tyr-derived 4-HB in mitochondria (MT) is modified to HQs via a ubiquinol-like pathway (Q). HQs are secreted into cell lumen for oxidation by Dmd. BQs travel to reservoir via duct.</div><div id="fspara0305" role="paragraph">(Q) Inferred HQ pathway (2-MeO-3-me-1,4-HQ as example); DC/OH, unidentified decarboxylase/hydroxylase.</div><div id="fspara0310" role="paragraph">See also <a id="crosref2500" href="#figs6">Figures S6</a> and <a id="crosref2505" href="#figs7">S7</a>, <a id="crosref2510" href="#mmc7">Table S1</a>, and <a id="crosref2515" href="#mmc6">Data S6</a>.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">We confirmed <i>dmd</i>’s expression in the bulbous secretory cells via HCR (<a id="crosref1035" href="#fig6">Figures 6</a>C and 6D), establishing the source of the BQs. We refer to these cells as the “BQ cell type.” Because Dmd is secreted, we hypothesized that it may be a terminal pathway enzyme that oxidizes secreted BQ precursors. Laccases are well known to oxidize hydroxyl groups, and studies in other aleocharine rove beetles have shown that in addition to BQs, trace levels of corresponding hydroquinones (HQs) can occasionally be detected in the secretion, consistent with HQs being unoxidized precursors (<span class="dropBlock reference-citations"><a id="body-ref-sref141-6" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-6" aria-controls="sref141-6">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In <i>Dalotia</i>, we sometimes recover trace levels of 2-methyl-1,4-HQ in the secretion, corresponding to the non-oxidized form of the highest abundance BQ. Strikingly, we find that although silencing <i>dmd</i> diminishes levels of 2-methyl-1,4-BQ, it leads to excess levels of 2-methyl-1,4-HQ (<a id="crosref1045" href="#fig6">Figure&nbsp;6</a>E). Further, a HQ precursor of a different BQ species, 2-methoxy-3-methyl-1,4-HQ, additionally appears (<a id="crosref1050" href="#fig6">Figure&nbsp;6</a>F). Accumulation of HQs following <i>dmd</i> silencing provides strong evidence that this laccase oxidizes secreted HQs, converting them into BQs. We relate this function to the BQ cell’s anatomy, where the cell envelopes a lumen connected directly to the duct (<a id="crosref1055" href="#fig6">Figure&nbsp;6</a>P). We posit that the lumen is an oxidation chamber into which Dmd and HQs are secreted and combine. The duct channels the resulting cytotoxic BQs into the solvent reservoir (<a id="crosref1060" href="#fig6">Figure&nbsp;6</a>P).</div><div id="p0150" role="paragraph">We recovered <i>dmd</i> orthologs only in genomes of aleocharines that possess a tergal gland and synthesize BQs (<a id="crosref1065" href="#fig6">Figure&nbsp;6</a>G). Phylogenetically, Dmd does not branch from inside either clade of conserved insect laccase, Lac1 and Lac2 (<a id="crosref1070" href="#fig6">Figure&nbsp;6</a>G). As in other insects, <i>Dalotia lac2</i> is expressed in cuticular cells (<a id="crosref1075" href="#figs6">Figure&nbsp;S6</a>F) and silencing it abolishes tanning (<a id="crosref1080" href="#figs6">Figures S6</a>C and S6D), whereas <i>dmd</i> is not expressed in the cuticle (<a id="crosref1085" href="#figs6">Figure&nbsp;S6</a>G) and is not involved in tanning (<a id="crosref1090" href="#figs6">Figures S6</a>C and S6E). Dmd thus defines an aleocharine-specific laccase clade that functions as a terminal oxidase in BQ synthesis. To corroborate this model, we synthesized Dmd protein and tested its ability to oxidize HQ precursors of all three BQs <i>in&nbsp;vitro</i>. Consistent with its inferred function, Dmd strongly catalyzed conversion of all three HQs to BQs (<a id="crosref1095" href="#fig6">Figures 6</a>H–6J and <a id="crosref1100" href="#figs6">S6</a>H). Further, we synthesized <i>Dalotia</i> Lac1 and Lac2 and found that although both enzymes exhibited some activity on at least one HQ substrate, neither were as efficient as Dmd (<a id="crosref1105" href="#fig6">Figure&nbsp;6</a>K). We conclude that Dmd is a catalytically specialized laccase that performs HQ oxidation in BQ cells.</div></section><section id="sec-2-8"><h3>HQ synthesis via evolution of a ubiquinone-like pathway</h3><div id="p0155" role="paragraph">What pathway produces the HQ precursors for Dmd? Our data indicate that BQs derive from Tyr (<a id="crosref1110" href="#figs6">Figure&nbsp;S6</a>A), but how the aromatic ring is hydroxylated to make HQs and decorated with methyl and methoxy groups is unknown. One eukaryotic pathway exists that integrates these steps: ubiquinone biosynthesis. Here, Tyr is converted to 4-hydroxybenzoic acid (4-HB), which is modified in the mitochondrion to yield an HQ, ubiquinol (<span class="dropBlock reference-citations"><a id="body-ref-sref153-1" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-1" aria-controls="sref153-1">Wang and Hekimi, 2019</a><div class="dropBlock__holder" data-db-target-of="sref153-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Wang, Y. ∙ Hekimi, S.</div><div><strong>The Complexity of Making Ubiquinone</strong></div><div><em>Trends Endocrinol. Metab.</em> 2019; <strong>30</strong>:929-943</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.WangS.HekimiThe+Complexity+of+Making+UbiquinoneTrends+Endocrinol.+Metab.302019929943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>Dalotia</i>’s BQs resemble less modified versions of ubiquinol, at most incorporating methyl and methoxy groups (<a id="crosref1120" href="#fig6">Figure&nbsp;6</a>L). To test whether HQs derive from a similar pathway, we fed <i>Dalotia</i> 4-HB-<sup>13</sup>C<sub>6</sub> and measured <sup>13</sup>C<sub>6</sub> incorporation into the BQs. Although the absolute level of incorporation was not high, we observed significant <sup>13</sup>C<sub>6</sub> enrichment into the two most abundant BQs (<a id="crosref1125" href="#fig6">Figure&nbsp;6</a>M). The magnitude of <sup>13</sup>C<sub>6</sub> incorporation was lower than observed on feeding <i>Dalotia</i> Tyr-<sup>13</sup>C<sub>6</sub> (<a id="crosref1130" href="#figs6">Figure&nbsp;S6</a>A), likely because 4-HB is a catabolite, not a dietary precursor, limiting its access from the gut to the correct cellular location. This result nevertheless identifies 4-HB as an intermediate in the conversion of Tyr to HQs.</div><div id="p0160" role="paragraph">Key steps in 4-HB’s conversion to ubiquinol are carried out by sequentially acting CoQ enzymes (<span class="dropBlock reference-citations"><a id="body-ref-sref153-2" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153-2" aria-controls="sref153-2">Wang and Hekimi, 2019</a><div class="dropBlock__holder" data-db-target-of="sref153-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Wang, Y. ∙ Hekimi, S.</div><div><strong>The Complexity of Making Ubiquinone</strong></div><div><em>Trends Endocrinol. Metab.</em> 2019; <strong>30</strong>:929-943</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.WangS.HekimiThe+Complexity+of+Making+UbiquinoneTrends+Endocrinol.+Metab.302019929943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1140" href="#figs7">Figure&nbsp;S7</a>J). Two CoQ enzymes are of potential relevance: CoQ5, a&nbsp;methyltransferase that adds the methyl group, and CoQ3, an O-methyltransferase that creates the methoxy groups. We asked whether these enzymes add methyl or methoxy groups to <i>Dalotia</i>’s BQs. In most eukaryotes, including insects, CoQ enzymes are encoded by conserved single copy genes (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Kawamukai, 2016</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Kawamukai, M.</div><div><strong>Biosynthesis of coenzyme Q in eukaryotes</strong></div><div><em>Biosci. Biotechnol. Biochem.</em> 2016; <strong>80</strong>:23-33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.KawamukaiBiosynthesis+of+coenzyme+Q+in+eukaryotesBiosci.+Biotechnol.+Biochem.8020162333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Due to their essential role in cellular respiration, studying them <i>in&nbsp;vivo</i> is challenging. For example, we observed ∼3-fold higher transcription of <i>CoQ5</i> in BQ cells (<a id="crosref1150" href="#figs7">Figure&nbsp;S7</a>B), but knockdown with even low dsRNA levels led to complete lethality. Unusually, however, we found that <i>CoQ3</i> has duplicated in <i>Dalotia</i>, as well as in genomes of all other aleocharines surveyed (in some cases it has duplicated twice) (<a id="crosref1155" href="#fig7">Figure&nbsp;7</a>F). Conspicuously, one of <i>Dalotia</i>’s <i>CoQ3</i>s is strongly upregulated in BQ cells (<a id="crosref1160" href="#figs7">Figure&nbsp;S7</a>D) as well as moderately so in putative sensory neurons (<a id="crosref1165" href="#figs7">Figure&nbsp;S7</a>H). Studies across eukaryotes have shown that CoQ3 adds two methoxy groups to ubiquinol using <i>S</i>-adenosylmethionine as the methyl donor (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Poon et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Poon, W.W. ∙ Barkovich, R.J. ∙ Hsu, A.Y. ...</div><div><strong>Yeast and rat Coq3 and Escherichia coli UbiG polypeptides catalyze both O-methyltransferase steps in coenzyme Q biosynthesis</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1999; <strong>274</strong>:21665-21672</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.W.PoonR.J.BarkovichA.Y.HsuA.FrankelP.T.LeeJ.N.ShepherdD.C.MylesC.F.ClarkeYeast+and+rat+Coq3+and+Escherichia+coli+UbiG+polypeptides+catalyze+both+O-methyltransferase+steps+in+coenzyme+Q+biosynthesisJ.%C2%A0Biol.+Chem.27419992166521672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An analogous reaction might yield the methoxy group of 2-methoxy-3-methyl-1,4-BQ. We fed <i>Dalotia</i> CD<sub>3</sub>-labeled methionine and observed direct incorporation of CD<sub>3</sub> into 2-methoxy-3-methyl-1,4-BQ, confirming that the same reaction takes place (<a id="crosref1175" href="#figs7">Figure&nbsp;S7</a>A). Remarkably, silencing the BQ cell-expressed <i>CoQ3</i> duplicate led to complete loss of 2-methoxy-3-methyl-1,4-BQ in the secretion (<a id="crosref1180" href="#fig6">Figures 6</a>O&nbsp;and <a id="crosref1185" href="#figs7">S7</a>E–S7G), without affecting <i>Dalotia</i>’s other BQs (<a id="crosref1190" href="#fig6">Figures 6</a>O and <a id="crosref1195" href="#figs7">S7</a>G). We name this <i>CoQ3</i> paralog “<i>methoxyless</i>” (<i>meos</i>) and deduce that it performs an identical modification to a defensive BQ as canonical CoQ3 does to ubiquinone. <i>meos</i> appears to be a single copy gene that is specific to aleocharines (<a id="crosref1200" href="#fig6">Figure&nbsp;6</a>N).</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/5897f4ac-a5b3-498a-b7ba-5f06ee1774ed/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.014/asset/9b541a00-aca9-439a-960b-31dac089d22b/main.assets/gr7.jpg" height="679" width="753" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/5897f4ac-a5b3-498a-b7ba-5f06ee1774ed/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Synergism between solvent and BQ cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0320" role="paragraph">(A) Survival with ants when BQ or solvent cells compromised (GLM, Χ<sup>2</sup><sub>2,33</sub>&nbsp;= 53.5, p&nbsp;&lt; 0.001). Dot size conveys number of armed beetles in arena included in statistical analysis. Black dots and bars are the estimates and SE of binomial model, respectively.</div><div id="fspara0325" role="paragraph">(B) Fly larva survival 1&nbsp;h post-immersion in synthetic gland components (GLM, Χ<sup>2</sup><sub>7,32</sub>&nbsp;= 5842.8, p&nbsp;&lt; 0.001). Letters indicate significant differences. The middle bar of each boxplot represents the median, the box denotes the interquartile range, while the whiskers show the minimum and maximum.</div><div id="fspara0330" role="paragraph">(C) <i>P.&nbsp;fluorescens</i> growth curves over 24&nbsp;h in presence of synthetic gland components (ANOVA<sub>endpoint</sub>, F<sub>7,96</sub>&nbsp;= 6.9, p&nbsp;&lt; 0.001; Welch’s ANOVA<sub>lag time</sub>, F<sub>7,96</sub>&nbsp;= 134.9, p&nbsp;&lt; 0.001). Lines represent the mean for each growth curve; error bars denote the 95% confidence interval at each time point.</div><div id="fspara0335" role="paragraph">(D) Dripping-onto-substrate (DoS) rheometer: synthetic secretion is dispensed via nozzle onto aluminum substrate. Droplet diameter <i>R</i> is measured until droplet pinch off.</div><div id="fspara0340" role="paragraph">(E) Video sequence of DoS measurement.</div><div id="fspara0345" role="paragraph">(F) Surface coating ability (K/σ × 100 in s<sup>n</sup>m<sup>−1</sup>) of mixtures. The middle bar of each boxplot represents the median, the box denotes the interquartile range, while the whiskers show the minimum and maximum.</div><div id="fspara0350" role="paragraph">(G) SCA and BQ solubility decrease with longer alkane chain length. See <a id="crosref2520" href="#mmc7">Table S1</a> for statistical analyses.</div><div id="fspara0355" role="paragraph">See also <a id="crosref2525" href="#mmc5">Data S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.11.014/asset/6c1dde58-4ad7-4f76-9252-aab13da99284/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.014/asset/a74801d7-ace2-4ff5-8d54-d09c0ff51bc9/main.assets/figs7.jpg" height="845" width="769" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/6c1dde58-4ad7-4f76-9252-aab13da99284/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Evidence for a mitochondrial HQ pathway, related to <a id="crosref2570" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0425" role="paragraph">A: Representative mass spectra of molecular ion region of 2-methoxy-3-methyl-1,4-BQ from beetles fed with unlabeled <i>Dalotia</i> diet (control) versus food infused with CD<sub>3</sub>-methionine. Spectra were recorded in single-ion mode. A strong [M+3]<sup>+</sup> enrichment indicates that the methoxy group is derived from dietary Met. B-D: Expression of <i>CoQ5</i> (J; Likelihood-Ratio test BQ-cell versus control: LR-test statistic&nbsp;= 5.37; q&nbsp;= 0.0374), <i>CoQ6</i> (K; Likelihood-Ratio test BQ-cell versus control: LR-test statistic&nbsp;= 9.27; q&nbsp;= 0.0082) and <i>meos</i> (L; Likelihood-Ratio test BQ-cell versus control: LR-test statistic&nbsp;= 33.74; q&nbsp;&lt; 0.0001) in BQ cells and control tissue based on SMARTseq. The horizontal middle bars represents the median, while the vertical bars show the minimum and maximum. E, F: Representative GC traces of gland compounds from wild-type (<i>GFP</i> RNAi; (E) and <i>meos</i> RNAi-injected (F) beetles. Silencing of <i>meos</i> leads to loss of 2-methoxy-3-methyl-1,4-BQ only, while both other BQs are still present. G: Amounts of 1,4-BQ, 2-methyl-1,4-BQ and 2-methoxy-3-methyl-1,4,-BQ extracted from <i>GFP</i> RNAi (control) and MeOS <i>RNAi</i> beetles. Significance was assessed using Mann-Whitney-U tests (<sup>∗∗</sup>&nbsp;= p ≤ 0.01). The horizontal middle bars represents the median, while the vertical bars show the minimum and maximum. H: Expression of <i>meos</i>&nbsp;across all cell classes in the 10x data shows tissue specific expression in the sensory neurons. Note that BQ-cells are not captured by 10x. I: MeOS contains a mitochondrial inner membrane (GO:0005743) signal peptide sequence. J: Proposed biosynthesis pathway of <i>Dalotia</i>’s BQs versus the mitochondrial ubiquinone pathway. Unique BQ pathway enzymes are in green; unique ubiquinone pathway enzymes are in orange; enzymes shared by the two pathways are in blue.</div></div></div></div></figcaption></figure></div><div id="p0165" role="paragraph">Canonical CoQ3 functions on the mitochondrial inner membrane, where MeOS is likewise predicted to localize (<a id="crosref1205" href="#figs7">Figure&nbsp;S7</a>I). Furthermore, another enzyme, CoQ6, which performs essential priming hydroxylation prior to methoxylation of ubiquinol by CoQ3, is also upregulated in BQ cells (<a id="crosref1210" href="#figs7">Figure&nbsp;S7</a>C), presumably permitting MeOS to methoxylate HQs (<a id="crosref1215" href="#figs7">Figure&nbsp;S7</a>J). However, silencing <i>CoQ6</i> led to beetle lethality. Discovery of MeOS therefore provides a serendipitous genetic window into a mitochondrial route where 4-HB is modified by CoQ enzymes to make HQs (<a id="crosref1220" href="#fig7">Figure&nbsp;7</a>I), closely paralleling ubiquinol synthesis (<a id="crosref1225" href="#figs7">Figure&nbsp;S7</a>J). Fully delineating the pathway <i>in&nbsp;vivo</i> is currently impossible owing to essential systemic functions of other CoQ enzymes. However, we postulate their co-option as a parsimonious model and conjecture that <i>Dalotia</i>’s three BQs arise via differential processing by MeOS and other CoQ enzymes (<a id="crosref1230" href="#fig6">Figures 6</a>Q and <a id="crosref1235" href="#figs7">S7</a>J).</div></section><section id="sec-2-9"><h3>Coevolution of BQ and solvent cells</h3><div id="p0170" role="paragraph">Why did cell type evolution follow the routes we have uncovered? The tergal gland secretion enhances beetle survival (<a id="crosref1240" href="#fig1">Figure&nbsp;1</a>C), so we reasoned that the BQ and solvent cells together underlie the gland’s adaptive value. To test this hypothesis, we performed a large-scale selection experiment, placing <i>Dalotia</i> in multiplexed arenas with predatory ants and quantifying relative survival of beetles that were either wild-type (GFP RNAi; 12 arenas, n&nbsp;= 120 beetles), <i>MFASN</i>-silenced to inhibit solvent production (12 arenas, n&nbsp;= 120 beetles), or <i>dmd</i>-silenced to inhibit BQ synthesis (12 arenas, n&nbsp;= 120 beetles). We allowed ants and beetles to interact for 48&nbsp;h before assaying survival. In both <i>MFASN</i>- and <i>dmd-</i>silenced treatments, we observed a comparable, significant reduction in survival (64% for GFP RNAi, 50% for <i>MFASN</i> RNAi, p&nbsp;&lt; 0.001, 45% for <i>dmd</i> RNAi; p&nbsp;&lt; 0.001) (<a id="crosref1245" href="#fig7">Figure&nbsp;7</a>A). Reduced survival appears to result from impaired chemical defense, because silencing <i>MFASN</i> or <i>dmd</i> without ant exposure did not significantly impact survival (<a id="crosref1250" href="#mmc5">Data S5</a>A). These results demonstrate each cell type’s adaptive value at the organismal level.</div><div id="p0175" role="paragraph">Because the BQ and solvent cells secrete into a common reservoir, their products may combine to make a bioactive secretion. We explored how the compounds interact to shape the secretion’s physicochemical properties. Two parameters capture salient properties of defensive secretions: the surface coating ability (SCA)—a measure of wetting potential, and extensional viscosity (EV)—a substance’s resistance to deformation when force is applied. The SCA and EV of arthropod secretions are species-specific and highly variable, likely arising as a balance of factors including safe containment in gland reservoirs, controlled exudation, efficient spread across target tissues, as well as final irritancy. We used a custom high-speed video rheometer to quantify SCA and EV of droplets of synthetic mixtures of different defensive compounds (<a id="crosref1255" href="#fig7">Figures 7</a>D and 7E). We found that SCAs of the BQ species alone or in combination are zero (<a id="crosref1260" href="#fig7">Figure&nbsp;7</a>F), whereas their EVs are virtually infinite (<a id="crosref1265" href="#mmc5">Data S5</a>B), because the three compounds remain in solid phase even when mixed. However, adding undecane dissolves the BQs, creating a moderately viscous liquid with high SCA (<a id="crosref1270" href="#fig7">Figure&nbsp;7</a>F; <a id="crosref1275" href="#mmc5">Data S5</a>B). Undecane thus acts as a solvent, unlocking the BQs. Equivalent SCA and EV are observed whether esters are present or not (<a id="crosref1280" href="#fig7">Figure&nbsp;7</a>F). Undecane is therefore the primary solvent and the main determinant of the secretion’s SCA and EV.</div><div id="p0180" role="paragraph">We examined the biological consequences of this chemical synergism by measuring toxicity against other organisms. Using viability of <i>Drosophila</i> larvae as a readout, we made synthetic secretions and measured survival rates 1&nbsp;h after drastic, whole-body immersion. Dipping fly larvae for 1&nbsp;s in a solution composed solely of undecane or esters, or both compound classes combined, led to only a small-to-modest survival reduction (<a id="crosref1285" href="#fig7">Figure&nbsp;7</a>B). Due to the BQs being solids, we could not immerse fly larvae in them, but found that completely bathing larvae in BQ powder caused a comparably limited drop in survival (<a id="crosref1290" href="#fig7">Figure&nbsp;7</a>B). However, when the BQs were combined with the undecane and esters at their natural ratios, mortality increased dramatically, reaching 80% (with 100% mortality in 2 of 5 replicates) (<a id="crosref1295" href="#fig7">Figure&nbsp;7</a>B). High mortality was also obtained if undecane was substituted for a higher ester fraction, or vice versa, demonstrating the critical effect of dissolving the BQs (<a id="crosref1300" href="#fig7">Figure&nbsp;7</a>B). These data indicate that the BQ and solvent cells are indeed engaged in biosynthetic synergism; only when their products are combined is a potent secretion with demonstrable adaptive value obtained (<a id="crosref1305" href="#fig1">Figures 1</a>C and <a id="crosref1310" href="#fig7">7</a>A).</div><div id="p0185" role="paragraph">We studied the antimicrobial effects of the secretion, noting that <i>Dalotia</i> self-applies the secretion topically to its own body. To do so, we assayed how synthetic combinations impact growth of the Gram-negative bacterium, <i>Pseudomonas fluorescens</i>—a common soil species and potential insect pathogen (<span class="dropBlock reference-citations"><a id="body-ref-sref113" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113" aria-controls="sref113">Pineda et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref113"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Pineda, A. ∙ Zheng, S.-J. ∙ van Loon, J.J.A. ...</div><div><strong>Helping plants to deal with insects: the role of beneficial soil-borne microbes</strong></div><div><em>Trends Plant Sci.</em> 2010; <strong>15</strong>:507-514</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.PinedaS.-J.ZhengJ.J.A.van+LoonC.M.J.PieterseM.DickeHelping+plants+to+deal+with+insects%3A+the+role+of+beneficial+soil-borne+microbesTrends+Plant+Sci.152010507514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref128" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128" aria-controls="sref128">Scales et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref128"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Scales, B.S. ∙ Dickson, R.P. ∙ LiPuma, J.J. ...</div><div><strong>Microbiology, genomics, and clinical significance of the Pseudomonas fluorescens species complex, an unappreciated colonizer of humans</strong></div><div><em>Clin. Microbiol. Rev.</em> 2014; <strong>27</strong>:927-948</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.S.ScalesR.P.DicksonJ.J.LiPumaG.B.HuffnagleMicrobiology%2C+genomics%2C+and+clinical+significance+of+the+Pseudomonas+fluorescens+species+complex%2C+an+unappreciated+colonizer+of+humansClin.+Microbiol.+Rev.272014927948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). When added to the <i>P.&nbsp;fluorescens</i> culture medium, no single compound inhibited growth (as measured by the OD500 of the culture at the endpoint of the assay) (<a id="crosref1325" href="#fig7">Figure&nbsp;7</a>C; <a id="crosref1330" href="#mmc7">Table S1</a>). Similarly, adding pairs of compound classes (undecane+esters, undecane+BQs, or esters+BQs) caused no effect (all pairwise tests p ≥ 0.18) (<a id="crosref1335" href="#fig7">Figure&nbsp;7</a>C; <a id="crosref1340" href="#mmc7">Table S1</a>). Remarkably, however, all three compound classes together caused dramatic growth suppression (all pairwise tests p ≤ 0.007) (<a id="crosref1345" href="#fig7">Figure&nbsp;7</a>C). This suppressive effect was also manifested in a significantly prolonged lag-time prior to the exponential growth phase of the <i>P.&nbsp;fluorescens</i> culture (all pairwise tests p&nbsp;&lt; 0.001) (<a id="crosref1350" href="#mmc5">Data S5</a>C). Antimicrobial efficacy arising from combining these compounds is unexpected and striking; to our knowledge it has not been previously reported. The effect is again consistent with the adaptive utility of the tergal gland arising from cooperation between BQ and solvent cells.</div><div id="p0190" role="paragraph"><i>Dalotia</i>’s tergal gland secretion is thus a multi-compound cocktail with emergent properties possessed by no single component. Although the BQs form the active defensive agent, the solvents provide a vehicle for the BQs while simultaneously conferring surface coating and tissue penetrating properties. We find two principal ways in which the solvent pathway may be specialized to meet these demands. First, the pathway favors high-level alkane and low-level ester synthesis. We noticed that silencing either <i>TG-CYP4G</i> or <i>TG-FAR</i> strongly increased levels of the C12 ester (“ester shift” in <a id="crosref1355" href="#fig2">Figures 2</a>H and 2I), consistent with TG-CYP4G and TG-FAR normally titrating most C12-CoA toward undecane, permitting TG-αEst to make only trace ethyl dodecanoate (<a id="crosref1360" href="#fig1">Figures 1</a>C and <a id="crosref1365" href="#fig2">2</a>F). Accordingly, silencing <i>TG-CYP4G</i> or <i>TG-FAR</i> frees up C12-CoA for TG-αEst, resulting in elevated ethyl dodecanoate (<a id="crosref1370" href="#fig2">Figures 2</a>H and 2I). The alkane branch is thus specialized for C12-CoA, whereas TG-αEst can use both C10- and C12-CoA. In this way, the solvent pathway produces large amounts of undecane as both the primary solvent and determinant of EV and SCA. This alkane bias strongly impacts the secretion: if undecane is replaced with the esters at their natural ratios, substantially higher EV and exceptionally high SCA result (<a id="crosref1375" href="#fig7">Figure&nbsp;7</a>F; <a id="crosref1380" href="#mmc5">Data S5</a>B). Such a sticky, surface active secretion may be challenging to secrete and difficult to contain in the reservoir. Low abundance esters are&nbsp;nevertheless critical for the secretion’s antimicrobial effect (<a id="crosref1385" href="#fig7">Figure&nbsp;7</a>C) and may also promote penetration into cuticles (<span class="dropBlock reference-citations"><a id="body-ref-sref38" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38" aria-controls="sref38">Dettner, 1991</a><div class="dropBlock__holder" data-db-target-of="sref38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Dettner, K.</div><div><strong>Solvent-dependent Variability of Effectiveness of Quinone-defensive Systems of Oxytelinae Beetles (Coleoptera: Staphylinidae)</strong></div><div><em>Entomol. Gen.</em> 1991; <strong>15</strong>:275-292</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerSolvent-dependent+Variability+of+Effectiveness+of+Quinone-defensive+Systems+of+Oxytelinae+Beetles+%28Coleoptera%3A+Staphylinidae%29Entomol.+Gen.151991275292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0195" role="paragraph">A second aspect is the use of undecane as opposed to a longer chain alkane as the solvent. Undecane stands in contrast to the very-long-chain CHCs produced in oenocytes. During solvent pathway evolution, the importance of selective recruitment of a reductase (FAR) and decarbonylase (CYP4G) but not an elongase (ELO) is underscored by insolubility of BQs in even the shortest of <i>Dalotia</i>’s CHCs, heptacosane (C27) (<a id="crosref1395" href="#fig7">Figures 7</a>F and 7G). However, the lack of ELO recruitment was not the only critical feature. Most insect FASNs produce C14–C18 fatty acids, rather than C12. We find that the corresponding alkanes (C13, C15, and C17), although only moderately longer than undecane, create secretions with much lower SCAs when mixed with the BQs, even in the presence of esters (<a id="crosref1400" href="#fig7">Figures 7</a>F and 7G). In the case of heptadecane (C17), adding BQs without esters caused the secretion to freeze (<a id="crosref1405" href="#fig7">Figure&nbsp;7</a>F), and BQ crystals were still evident when esters were included. Hence, synthesis of C12-CoA by MFASN is central to creating an alkane that can both solubilize the BQs and create a topical secretion (<a id="opt8z9LVwe0X0" href="#fig7">Figure 7</a>G).</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0200" role="paragraph">We have traced the evolution of new cellular functions comprising an organ novelty—the defensive tergal gland of rove beetles. First, we uncovered small molecule enzyme pathways that execute the biosynthetic functions of two taxon-restricted secretory cell types. Second, we presented evidence that these pathways—as well as a transcriptomic module expressed within one of the cell types—arose via repurposing components from more ancient cell types within the beetle. Third, we demonstrated the consequences of cell type evolution at the organ level, assigning adaptive value to the coordinated actions of the two cell types, as well as to specialized features of the solvent pathway that render the secretion effective in chemical defense. Our findings indicate that the functional evolution of animal cell types can be a constrained process, employing pre-existing pathway motifs and expression programs and hence potentially convergence-prone. Conversely, our study demonstrates that the route followed by cell-type evolution may also be a highly contingent process, conditional on other cell types within the organ and the fitness consequences of their collective output.</div><section id="sec-3-1"><h3>Evolution of new biosynthetic functions</h3><div id="p0205" role="paragraph">Using scRNA-seq, we pinpointed specific cell types from which&nbsp;solvent pathway enzymes had been ancestrally sourced, enabling us to reconstruct the assembly of the pathway via co-option or duplication of oenocyte and fat body enzymes. We also determined that a transcriptome module that defines oenocytes and VFB, GEP9, has been re-employed in solvent cells. Through these changes, an ancestral region of abdominal cuticle evolved into specialized secretory cells comprising a reservoir. That the beetle’s capacity to manufacture solvents is derived from its primary metabolism supports the view of genetic paths of least resistance to new animal chemistries—a notion stemming from pervasive convergence in the compounds that animals produce (<span class="dropBlock reference-citations"><a id="body-ref-sref24-2" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24-2" aria-controls="sref24-2">Brückner and Parker, 2020</a><div class="dropBlock__holder" data-db-target-of="sref24-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Brückner, A. ∙ Parker, J.</div><div><strong>Molecular evolution of gland cell types and chemical interactions in animals</strong></div><div><em>J.&nbsp;Exp. Biol.</em> 2020; <strong>223</strong>:jeb211938</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerJ.ParkerMolecular+evolution+of+gland+cell+types+and+chemical+interactions+in+animalsJ.%C2%A0Exp.+Biol.223Pt%2C+Suppl+12020jeb211938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The BQ pathway likewise points to an underlying genomic blueprint; here, HQ synthesis, although not yet fully delineated, owes at least part of its existence to the ubiquinone pathway from which the <i>CoQ3</i> paralog <i>meos</i> originated via duplication. We predict the connection will turn out to be still more extensive, with ubiquinone and HQ pathways sharing a subset of enzymes (<a id="crosref1415" href="#figs7">Figure&nbsp;S7</a>J). Similarly, the role of the laccase, Dmd, parallels or may derive from the conserved function of Lac2 in cuticle tanning (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Asano et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Asano, T. ∙ Seto, Y. ∙ Hashimoto, K. ...</div><div><strong>Mini-review an insect-specific system for terrestrialization: Laccase-mediated cuticle formation</strong></div><div><em>Insect Biochem. Mol. Biol.</em> 2019; <strong>108</strong>:61-70</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.AsanoY.SetoK.HashimotoH.KurushimaMini-review+an+insect-specific+system+for+terrestrialization%3A+Laccase-mediated+cuticle+formationInsect+Biochem.+Mol.+Biol.10820196170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The discovery of laccase involvement in BQ-based termite chemical defense (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Bourguignon et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Bourguignon, T. ∙ Šobotník, J. ∙ Brabcová, J. ...</div><div><strong>Molecular Mechanism of the Two-Component Suicidal Weapon of Neocapritermes taracua Old Workers</strong></div><div><em>Mol. Biol. Evol.</em> 2016; <strong>33</strong>:809-819</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.BourguignonJ.%C5%A0obotn%C3%ADkJ.Brabcov%C3%A1D.Sillam-Duss%C3%A8sA.Bu%C4%8DekJ.Krasulov%C3%A1B.Vytiskov%C3%A1Z.Demianov%C3%A1M.Mare%C5%A1Y.RoisinH.VogelMolecular+Mechanism+of+the+Two-Component+Suicidal+Weapon+of+Neocapritermes+taracua+Old+WorkersMol.+Biol.+Evol.332016809819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) hints at parallel use of laccases in BQ-producing arthropod clades. We propose that repurposing pathway motifs from ancient metabolic cell types—or entire expression programs that enable production of certain compounds—represent accessible routes to new biosynthetic functions in animal cell types. A consequence is that exploration of chemical space is constrained, leading to widespread convergence.</div></section><section id="sec-3-2"><h3>Cell-type evolution via transcriptomic hybridization</h3><div id="p0210" role="paragraph">Together with recruitment of solvent pathway enzymes, we propose that acquisition of GEP9 was causal in transforming the intersegmental membrane between A6 and A7 into a glandular epithelium. The membrane is naturally invaginated between segments in rove beetles (<a id="crosref1430" href="#fig5">Figure&nbsp;5</a>G); consequently, further growth created a reservoir into which the solvent cells could secrete directly (<a id="crosref1435" href="#fig5">Figure&nbsp;5</a>H). The antiquity of the oenocytes and fat body, which originated at the base of the Insecta, combined with GEP9 being the sole program recovered in these cell types, indicates that the polarity of recruitment was from these ancient cell types into solvent cells rather than vice versa. We cannot, however, rule out that some newer GEP9 components may have evolved functions in solvent cells and found secondary, pleiotropic utility in oenocytes or VFB.</div><div id="p0215" role="paragraph">The solvent cells shed mechanistic light on the phenomenon of “cell type fusion” where a novel cell type appears to take on features of two ancestral cell types (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Arendt et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Arendt, D. ∙ Musser, J.M. ∙ Baker, C.V.H. ...</div><div><strong>The origin and evolution of cell types</strong></div><div><em>Nat. Rev. Genet.</em> 2016; <strong>17</strong>:744-757</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArendtJ.M.MusserC.V.H.BakerA.BergmanC.CepkoD.H.ErwinM.PavlicevG.SchlosserS.WidderM.D.LaubichlerG.P.WagnerThe+origin+and+evolution+of+cell+typesNat.+Rev.+Genet.172016744757" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Oakley, 2017</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Oakley, T.H.</div><div><strong>Furcation and fusion: The phylogenetics of evolutionary novelty</strong></div><div><em>Dev. Biol.</em> 2017; <strong>431</strong>:69-76</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.H.OakleyFurcation+and+fusion%3A+The+phylogenetics+of+evolutionary+noveltyDev.+Biol.43120176976" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref129" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129" aria-controls="sref129">Schlosser, 2018</a><div class="dropBlock__holder" data-db-target-of="sref129"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Schlosser, G.</div><div><strong>A Short History of Nearly Every Sense-The Evolutionary History of Vertebrate Sensory Cell Types</strong></div><div><em>Integr. Comp. Biol.</em> 2018; <strong>58</strong>:301-316</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.SchlosserA+Short+History+of+Nearly+Every+Sense-The+Evolutionary+History+of+Vertebrate+Sensory+Cell+TypesIntegr.+Comp.+Biol.582018301316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Redeployment of a pre-existing transcriptional program in a new cellular context may be a common means for generating cellular- and organ-level novelties, and contrasts with duplication and divergence of sister cells within organs or repeating developmental fields such as body segments (<span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Marioni and Arendt, 2017</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Marioni, J.C. ∙ Arendt, D.</div><div><strong>How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2017; <strong>33</strong>:537-553</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.MarioniD.ArendtHow+Single-Cell+Genomics+Is+Changing+Evolutionary+and+Developmental+BiologyAnnu.+Rev.+Cell+Dev.+Biol.332017537553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or brain nuclei (<span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">Kebschull et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Kebschull, J.M. ∙ Richman, E.B. ∙ Ringach, N. ...</div><div><strong>Cerebellar nuclei evolved by repeatedly duplicating a conserved cell-type set</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:eabd5059</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.KebschullE.B.RichmanN.RingachD.FriedmannE.AlbarranS.S.KolluruR.C.JonesW.E.AllenY.WangS.W.ChoCerebellar+nuclei+evolved+by+repeatedly+duplicating+a+conserved+cell-type+setScience3702020eabd5059" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Transcriptomic hybridization may be mechanistically facile if expression programs conferring cell identity are controlled by one or a few transcription factors (i.e., “terminal selectors”) (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Hobert, 2016</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Hobert, O.</div><div><strong>Terminal Selectors of Neuronal Identity</strong></div><div><em>Curr. Top Dev. Biol.</em> 2016; <strong>116</strong>:455-475</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.HobertTerminal+Selectors+of+Neuronal+IdentityCurr.+Top+Dev.+Biol.1162016455475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). How GEP9 became expressed in solvent cells is unknown, but this abdominal location is a region of overlapping expression of the Abdominal A and Abdominal B Hox proteins, both of which are needed for solvent cell development (<span class="dropBlock reference-citations"><a id="body-ref-sref109-2" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-2" aria-controls="sref109-2">Parker et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref109-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Parker, J. ∙ Eldredge, K.T. ∙ Thomas, I. ...</div><div><strong>Hox-logic of body plan innovations for social insect symbiosis in rove beetles</strong></div><div><em>bioRxiv.</em> 2018; 198945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerK.T.EldredgeI.ThomasR.ColemanS.DavisHox-logic+of+body+plan+innovations+for+social+insect+symbiosis+in+rove+beetlesbioRxiv2018198945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Conceivably, a terminal selector controlling GEP9 in oenocytes and VFB could have come under Hox control and been co-opted into cuticle cells, creating a GEP9/17 hybrid transcriptome. Future studies may uncover the regulatory basis of BQ and solvent cell identities.</div></section><section id="sec-3-3"><h3>Cell-type evolution shaped by cooperative interactions at the organ level</h3><div id="p0220" role="paragraph">Demonstrating adaptive change at the molecular level depends on connecting such changes to phenotypic outcomes that differentially impact fitness (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Barrett and Hoekstra, 2011</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Barrett, R.D.H. ∙ Hoekstra, H.E.</div><div><strong>Molecular spandrels: tests of adaptation at the genetic level</strong></div><div><em>Nat. Rev. Genet.</em> 2011; <strong>12</strong>:767-780</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.D.H.BarrettH.E.HoekstraMolecular+spandrels%3A+tests+of+adaptation+at+the+genetic+levelNat.+Rev.+Genet.122011767780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our data connect the evolution of new cellular functions to a cooperative interaction between cell types that dictates whole organ performance and directly impacts animal survival. Hence, we propose that coevolution between the BQ and solvent cells has been driven, at least in part, by natural selection for organ-level properties. Our findings underscore how metazoan cell type and organ evolution are intrinsically coupled, with changes at the cellular level being comprehensible—and conferring adaptive value—only when their impact at the collective, multicellular organ level is considered (<span class="dropBlock reference-citations"><a id="body-ref-sref76-3" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-3" aria-controls="sref76-3">Kishi and Parker, 2021</a><div class="dropBlock__holder" data-db-target-of="sref76-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Kishi, Y. ∙ Parker, J.</div><div><strong>Cell type innovation at the tips of the animal tree</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2021; <strong>69</strong>:112-121</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.KishiJ.ParkerCell+type+innovation+at+the+tips+of+the+animal+treeCurr.+Opin.+Genet.+Dev.692021112121" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0225" role="paragraph">Inferring the steps leading to cooperativity between the solvent and BQ cells is informed by two observations. First, the recalcitrance of the solid BQs makes a “solvents first” scenario likely, where alkanes or esters, or fatty acid-derived progenitors thereof, initially arose. Such compounds may have conferred modest chemical defense or acted as pheromones or lubricants for the flexible abdomen. This reasoning is supported by the second observation: that the earliest-diverging branch of the tergal gland-bearing Aleocharinae—the tribe Hypocyphtini—produce fatty acid-derivatives but lack BQs. We deduce that the prior evolution of a fatty acid-based secretion set the stage for the subsequent evolution of BQs, the potency of which could be unlocked by the former compounds acting as effective solvents. Features of the solvent pathway such as its alkane bias, the use of medium chain compounds and the presence of low-abundance esters, can be interpreted as specializations to harness the BQs’ toxicity and manipulability (<a id="crosref1485" href="#fig7">Figure&nbsp;7</a>F<b>;</b> <a id="crosref1490" href="#mmc5">Data S5</a>). Hence, we suggest that the solvent cells in turn underwent adaptive modifications to make better use of the BQs, implying reciprocal coevolution between the two cell types. Across Aleocharinae, BQs are a constant feature of the secretion but the fatty acid derivatives vary extensively, including hydrocarbons, esters, and aldehydes of differing chain lengths and ratios (<span class="dropBlock reference-citations"><a id="body-ref-sref141-7" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-7" aria-controls="sref141-7">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These differences are expected to strongly influence the secretion’s physicochemical nature, further emphasizing reciprocal coevolution.</div><div id="p0230" role="paragraph">Cooperative behavior is a feature of biological systems at all&nbsp;scales of organization. The tergal gland, comprising the simplest case of only two cell types, presents a model for explaining the evolution of cooperativity at the organ level. According to this model, the solvent cells created a niche for the BQ cells, which enhanced the gland’s adaptive value. The dependence of the BQs on the product of the solvent cells, and the reciprocal dependence of the solvent cells on the BQs to enhance their own adaptive value, meant that the two cell types became “locked in” as a unit evolving within constraints set by performance at the organ level. This hypothesized route toward cooperativity contrasts with models for cooperativity within protein complexes. Here, dependencies between subunits can arise via entrenchment of binding interactions that were ancestrally selectively neutral (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Hochberg et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Hochberg, G.K.A. ∙ Liu, Y. ∙ Marklund, E.G. ...</div><div><strong>A hydrophobic ratchet entrenches molecular complexes</strong></div><div><em>Nature.</em> 2020; <strong>588</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.K.A.HochbergY.LiuE.G.MarklundB.P.H.MetzgerA.LaganowskyJ.W.ThorntonA+hydrophobic+ratchet+entrenches+molecular+complexesNature5882020503508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Lukeš et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Lukeš, J. ∙ Archibald, J.M. ∙ Keeling, P.J. ...</div><div><strong>How a neutral evolutionary ratchet can build cellular complexity</strong></div><div><em>IUBMB Life.</em> 2011; <strong>63</strong>:528-537</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.Luke%C5%A1J.M.ArchibaldP.J.KeelingW.F.DoolittleM.W.GrayHow+a+neutral+evolutionary+ratchet+can+build+cellular+complexityIUBMB+Life632011528537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the tergal gland, we posit that interdependence has likewise been enforced, but through the addition of a new cell type that is functionally contingent on a pre-existing cell type, which itself became obligately reliant on the second cell type to realize a relative selective advantage. Such a scenario could iterate through further cycles, informing how cooperative interactions between diverse cell types may arise generally within organs.</div></section><section id="sec-3-4"><h3>Limitations of study</h3><div id="p0235" role="paragraph">The BQ cells’ large diameter (∼30–40&nbsp;μm) and scarcity (∼8–14 cells/animal) precluded their recovery in the 10x scRNA-seq cell type atlas. We were thus unable to determine their GEP composition via cNMF, and thus their transcriptomic relationship to other cell types remains unclear at this time. Future studies of early-branching aleocharines are needed to gauge the conservation of BQ and solvent pathway enzymes across phylogeny; such efforts may further illuminate the tergal gland’s stepwise evolution.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0250" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr><td><i>Dalotia coriaria</i></td><td>Applied Bionomics (Canada)</td><td><a id="intref0050" href="https://www.appliedbio-nomics.com/products/dalotia/" target="_blank">https://www.appliedbio-nomics.com/products/dalotia/</a></td></tr><tr><td><i>Drosophila</i> <i>melanogaster</i></td><td>Dickinson Laboratory (Caltech)</td><td>N/A</td></tr><tr class="rowsep"><td><i>Liometopum occidentale</i></td><td>This study (field collection)</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and viral strains</b></td></tr><tr><td><i>Pseudomonas fluorescens</i> wild type (strain WCS365)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Jorth et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Jorth, P. ∙ Spero, M.A. ∙ Livingston, J. ...</div><div><strong>Quantitative Visualization of Gene Expression in Mucoid and Nonmucoid Pseudomonas aeruginosa Aggregates Reveals Localized Peak Expression of Alginate in the Hypoxic Zone</strong></div><div><em>MBio.</em> 2019; <strong>10</strong>:e02622</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.JorthM.A.SperoJ.LivingstonD.K.NewmanQuantitative+Visualization+of+Gene+Expression+in+Mucoid+and+Nonmucoid+Pseudomonas+aeruginosa+Aggregates+Reveals+Localized+Peak+Expression+of+Alginate+in+the+Hypoxic+ZoneMBio102019e02622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr class="rowsep"><td>BL21 competent <i>E.&nbsp;coli</i></td><td>provided by C. VanDrisse, Caltech</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>amoxicillin</td><td>Sigma-Aldrich</td><td>#Cat A8523-5G</td></tr><tr><td>streptomycin sulfate salt</td><td>Sigma-Aldrich</td><td>#Cat S6501-25G</td></tr><tr><td>tetracycline</td><td>Sigma-Aldrich</td><td>#Cat 87128-25G</td></tr><tr><td>sodium dodecyl sulfate ACS reagent, ≥99.0%</td><td>Sigma-Aldrich</td><td>#Cat 436143-100G</td></tr><tr><td>13C6-D-glucose ≥99 atom % 13C, ≥99% (CP)</td><td>Sigma-Aldrich</td><td>#Cat 389374-1G</td></tr><tr><td>13C6-tyrosine &gt;99 atom % 13C, 99% (CP)</td><td>Sigma-Aldrich</td><td>#Cat 489794-100MG</td></tr><tr><td>D3-methionine (methyl-D3, 98%)</td><td>Cambridge Isotope Laboratories</td><td>#Cat DLM-431-1</td></tr><tr><td>D23-dodecanoid acid ≥98 atom % D, ≥98% (CP)</td><td>Sigma-Aldrich</td><td>#Cat 451401-25MG</td></tr><tr><td>13C6-4-hydroxybenzoic acid (RING-13C6, 99%)</td><td>Cambridge Isotope Laboratories</td><td>#Cat CLM-273-0.5</td></tr><tr><td>hexane ReagentPlus, ≥99%</td><td>Sigma-Aldrich</td><td>#Cat 139386-500ML</td></tr><tr><td>diethyl pyrocarbonate (DEPC)</td><td>Sigma-Aldrich</td><td>#Cat 40718-25ML</td></tr><tr><td>1,4-benzoquinone, Certified Reference Material</td><td>Sigma-Aldrich</td><td>#Cat PHR1028-1G</td></tr><tr><td>2-methyl-1,4-benzoquinone 98%</td><td>Sigma-Aldrich</td><td>#Cat 211311-5G</td></tr><tr><td>2-methoxy-3-methyl-1,4-benzoquinone</td><td>This study; see below</td><td>N/A</td></tr><tr><td>1,4-hydroquinone ReagentPlus, 99%</td><td>Sigma-Aldrich</td><td>#Cat H17902-100G</td></tr><tr><td>2-methyl-1,4- hydroquinone 99%</td><td>Sigma-Aldrich</td><td>#Cat 112968-250G</td></tr><tr><td>2-methoxy-3-methyl-1,4- hydroquinone</td><td>Sigma-Aldrich</td><td>#Cat L300217-1G</td></tr><tr><td>n-undecane, reference substance for GC</td><td>Sigma-Aldrich</td><td>#Cat 1097940005</td></tr><tr><td>isopropyl decanoate AldrichCPR</td><td>Sigma-Aldrich</td><td>#Cat CDS000590</td></tr><tr><td>ethyl decanoate ReagentPlus, ≥99%</td><td>Sigma-Aldrich</td><td>#Cat 148970-100ML</td></tr><tr><td>ethyl dodecanoate ≥98.0% (GC)</td><td>Sigma-Aldrich</td><td>#Cat 61630-100ML</td></tr><tr><td>octadecane analytical standard</td><td>Sigma-Aldrich</td><td>#Cat 74691-5G</td></tr><tr><td>16% paraformaldehyde aqueous solution</td><td>Electron Microscopy Sciences</td><td>#Cat 15710</td></tr><tr><td>Trizol</td><td>Thermo Fisher</td><td>#Cat 15596026</td></tr><tr><td>Schneider’s <i>Drosophila</i> medium</td><td>Thermo Fisher</td><td>#Cat 21720024</td></tr><tr><td>FBS - fetal bovine serum</td><td>Thermo Fisher</td><td>#Cat A3160501</td></tr><tr><td>Corning 0.25% Trypsin 0.1% EDTA in HBSS</td><td>Fisher Scientific</td><td>#Cat MT25053CI</td></tr><tr><td>Gibco Defined Trypsin Inhibitor</td><td>Sigma-Aldrich</td><td>#Cat R007100</td></tr><tr><td>Dulbecco’s Phosphate Buffered Saline</td><td>Sigma-Aldrich</td><td>#Cat D8537-100ML</td></tr><tr><td>Bovine Serum Albumin solution</td><td>Sigma-Aldrich</td><td>#Cat B8667-1.25ML</td></tr><tr><td>Earle's Balanced Salt Solution</td><td>Sigma-Aldrich</td><td>#Cat E7510-100ML</td></tr><tr><td>Isopropyl-β-D-thiogalactoside (IPTG)</td><td>Sigma-Aldrich</td><td>#Cat 10724815001</td></tr><tr><td>Ni-NTA resin</td><td>Sigma-Aldrich</td><td>#Cat 70666-4</td></tr><tr><td>imidazole ReagentPlus, 99%</td><td>Sigma-Aldrich</td><td>#Cat I202-100G</td></tr><tr><td>DNase grade II, from bovine pancreas</td><td>Sigma-Aldrich</td><td>#Cat 10104159001</td></tr><tr><td>2-Morpholinoethanesulfonic acid monohydrate EMPROVE EXPERT</td><td>Sigma-Aldrich</td><td>#Cat 1370740250</td></tr><tr><td>SnakeSkin Dialysis Tubing, 10K MWCO, 16&nbsp;mm</td><td>Thermo Fisher</td><td>#Cat 68100</td></tr><tr><td>Copper (II) sulfate ReagentPlus, ≥99%</td><td>Sigma-Aldrich</td><td>#Cat C1297-100G</td></tr><tr><td>ABTS (2,2'-azino-bis(3-ethylbenzothiazoline-6-sulfonic acid))</td><td>Sigma-Aldrich</td><td>#Cat 10102946001</td></tr><tr><td>urea, BioReagent, for molecular biology,</td><td>Sigma-Aldrich</td><td>#Cat U5378-100G</td></tr><tr><td>sodium sulfate anhydrate - Na2SO4</td><td>Sigma-Aldrich</td><td>#Cat 238597-500G</td></tr><tr><td>dichloromethane, ≥99.8%, with amylene stabilizer</td><td>Sigma-Aldrich</td><td>#Cat 34856-1L</td></tr><tr><td>sodium periodate ACS reagent, ≥99.8%</td><td>Sigma-Aldrich</td><td>#Cat 311448-100G</td></tr><tr><td>tridecane analytical standard</td><td>Sigma-Aldrich</td><td>#Cat 91490-5ML</td></tr><tr><td>pentadecane analytical standard</td><td>Sigma-Aldrich</td><td>#Cat 442700</td></tr><tr><td>heptadecane analytical standard</td><td>Sigma-Aldrich</td><td>#Cat 51578-5ML</td></tr><tr><td>heptacosane analytical standard</td><td>Sigma-Aldrich</td><td>#Cat 51559-250MG</td></tr><tr><td>chloroform, ≥99.8%, with amylene stabilizer</td><td>Sigma-Aldrich</td><td>#Cat 372978-1L</td></tr><tr><td>Hoechst 33342 Solution (20&nbsp;mM)</td><td>Thermo Fisher</td><td>#Cat 62249</td></tr><tr><td>Alexa-488-Wheat Germ Agglutinin Conjugate</td><td>Thermo Fisher</td><td>#Cat W11261</td></tr><tr><td>Alexa-647-Wheat Germ Agglutinin Conjugate</td><td>Thermo Fisher</td><td>#Cat W32466</td></tr><tr><td>BODIPY 493/503</td><td>Thermo Fisher</td><td>#Cat D3922</td></tr><tr><td>ProLong Gold Antifade Mountant</td><td>Thermo Fisher</td><td>#Cat P36934</td></tr><tr class="rowsep"><td>RNAlater™ Stabilization Solution</td><td>Thermo Fisher</td><td>#Cat AM7020</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Chromium Single Cell 3′ GEM, Library &amp;<br>Gel Bead Kit v3</td><td>10x Genomics</td><td>#Cat 1000075</td></tr><tr><td>Chromium Chip B Single Cell Kit v3</td><td>10x Genomics</td><td>#Cat 1000073</td></tr><tr><td>Chromium i7 Multiplex Kit</td><td>10x Genomics</td><td>#Cat 120262</td></tr><tr><td>NEBNext Single Cell/Low Input RNA Library Prep Kit for Illumina</td><td>New England Biolabs</td><td>#Cat E6420L</td></tr><tr><td>NEBNext Multiplex Oligos for Illumina (Dual Index Primers Set 1)</td><td>New England Biolabs</td><td>#Cat E7600S</td></tr><tr><td>NEBNext Multiplex Oligos for Illumina (Dual Index Primers Set 2)</td><td>New England Biolabs</td><td>#Cat E7780S</td></tr><tr><td>Zymo Research Quick-DNA Miniprep Plus Kit</td><td>Zymo Research Corporation</td><td>#Cat D4069</td></tr><tr><td>Luna Universal qPCR Master Mix</td><td>New England Biolabs</td><td>#Cat M3003L</td></tr><tr><td>Qubit dsDNA HS Assay Kit</td><td>Thermo Fisher</td><td>#Cat Q32854</td></tr><tr><td>TOPO TA Cloning Kit</td><td>Thermo Fisher</td><td>#Cat 450641</td></tr><tr><td>MEGAscript T7 Transcription Kit</td><td>Thermo Fisher</td><td>#Cat AMB13345</td></tr><tr><td>In situ hybridization chain reaction (HCR) v3 kit for O-CYP4G-B1</td><td>Molecular Instruments</td><td>3752/D845</td></tr><tr><td>In situ HCR v3 kit for dmd-B1</td><td>Molecular Instruments</td><td>3752/D837</td></tr><tr><td>In situ HCR v3 kit for MFASN-B1</td><td>Molecular Instruments</td><td>3768/D873</td></tr><tr><td>In situ HCR v3 kit for TG-FAR-B1</td><td>Molecular Instruments</td><td>3752/D847</td></tr><tr><td>In situ HCR v3 kit for TG-CYP4G I-B1</td><td>Molecular Instruments</td><td>3834/E003 and 3752/D853</td></tr><tr><td>In situ HCR v3 kit for TG-CYP4G II-B4</td><td>Molecular Instruments</td><td>4096/E272 and 4096/E274</td></tr><tr><td>In situ HCR v3 kit for TG-α-EST-B1</td><td>Molecular Instruments</td><td>3834/E004 and 3752/D851</td></tr><tr><td>In situ HCR v3 kit for Tcas_MFASN-B1</td><td>Molecular Instruments</td><td>4096/E270</td></tr><tr><td>Illumina TruSeqTM mRNA</td><td>Illumina</td><td>#Cat 20020594</td></tr><tr><td>Agilent Bioanalyzer High Sensitivity DNA assay (Analysis Kit and DNA Chips)</td><td>Agilent</td><td>#Cat 5067-4626 / 5067-4627</td></tr><tr><td>LIVE/DEAD Viability/Cytotoxicity Kit</td><td>Thermo Fisher</td><td>#Cat L3224</td></tr><tr><td>Blood &amp; Cell Culture DNA Mini Kit (25)</td><td>Qiagen</td><td>#Cat 13323</td></tr><tr><td>Illumina TruSeqTM DNA kit v2</td><td>Illumina</td><td>#Cat FC-121-2001</td></tr><tr class="rowsep"><td>NEBNext Ultra II FS DNA Library Prep Kit for Illumina</td><td>New England Biolabs</td><td>#Cat E7805L</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>Univ16SRT-F: 5′-ACTCCTACGGGAGGCAGCAGT-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-optG10oc3bwOi-1" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optG10oc3bwOi-1" aria-controls="optG10oc3bwOi-1">Clifford et al., 2012</a><div class="dropBlock__holder" data-db-target-of="optG10oc3bwOi-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Clifford, R.J. ∙ Milillo, M. ∙ Prestwood, J. ...</div><div><strong>Detection of bacterial 16S rRNA and identification of four clinically important bacteria by real-time PCR</strong></div><div><em>PloS one.</em> 2012; <strong>7</strong>, e48558</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.J.CliffordM.MililloJ.PrestwoodR.QuinteroD.V.ZurawskiY.I.KwakP.E.WatermanE.P.LeshoP.Mc+GannDetection+of+bacterial+16S+rRNA+and+identification+of+four+clinically+important+bacteria+by+real-time+PCRPloS+one72012e48558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>Univ16SRT-R: 5′-TATTACCGCGGCTGCTGGC-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-optG10oc3bwOi-2" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optG10oc3bwOi-2" aria-controls="optG10oc3bwOi-2">Clifford et al., 2012</a><div class="dropBlock__holder" data-db-target-of="optG10oc3bwOi-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Clifford, R.J. ∙ Milillo, M. ∙ Prestwood, J. ...</div><div><strong>Detection of bacterial 16S rRNA and identification of four clinically important bacteria by real-time PCR</strong></div><div><em>PloS one.</em> 2012; <strong>7</strong>, e48558</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.J.CliffordM.MililloJ.PrestwoodR.QuinteroD.V.ZurawskiY.I.KwakP.E.WatermanE.P.LeshoP.Mc+GannDetection+of+bacterial+16S+rRNA+and+identification+of+four+clinically+important+bacteria+by+real-time+PCRPloS+one72012e48558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>D3A _F: 5′-GACCCGTCTTGAAACACGGA-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-optA1Pb6vGgHv-1" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optA1Pb6vGgHv-1" aria-controls="optA1Pb6vGgHv-1">Litvaitis and Rohde, 1999</a><div class="dropBlock__holder" data-db-target-of="optA1Pb6vGgHv-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Litvaitis, M.K. ∙ Rohde, K.</div><div><strong>A molecular test of platyhelminth phylogeny: inferences from partial 28S rDNA sequences</strong></div><div><em>Invertebrate Biology.</em> 1999; <strong>118</strong>:42-56</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.K.LitvaitisK.RohdeA+molecular+test+of+platyhelminth+phylogeny%3A+inferences+from+partial+28S+rDNA+sequencesInvertebrate+Biology11819994256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>D3B_R: 5′-TCGGAAGGAACCAGCTACTA-3′</td><td><span class="dropBlock reference-citations"><a id="body-ref-optA1Pb6vGgHv-2" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optA1Pb6vGgHv-2" aria-controls="optA1Pb6vGgHv-2">Litvaitis and Rohde, 1999</a><div class="dropBlock__holder" data-db-target-of="optA1Pb6vGgHv-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Litvaitis, M.K. ∙ Rohde, K.</div><div><strong>A molecular test of platyhelminth phylogeny: inferences from partial 28S rDNA sequences</strong></div><div><em>Invertebrate Biology.</em> 1999; <strong>118</strong>:42-56</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.K.LitvaitisK.RohdeA+molecular+test+of+platyhelminth+phylogeny%3A+inferences+from+partial+28S+rDNA+sequencesInvertebrate+Biology11819994256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>N/A</td></tr><tr><td>HCR v3 Amplifier Hairpin System Alexa546-B1</td><td>Molecular Instruments</td><td>N/A</td></tr><tr><td>HCR v3 Amplifier Hairpin System Alexa488-B4</td><td>Molecular Instruments</td><td>N/A</td></tr><tr><td>pTEV16 protein expression vector</td><td><span class="dropBlock reference-citations"><a id="body-ref-opt9Cj5loRWYd-1" href="#" role="doc-biblioref" data-xml-rid="bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opt9Cj5loRWYd-1" aria-controls="opt9Cj5loRWYd-1">VanDrisse and Escalante-Semerena, 2016</a><div class="dropBlock__holder" data-db-target-of="opt9Cj5loRWYd-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">154.</div><div>VanDrisse, C.M. ∙ Escalante-Semerena, J.C.</div><div><strong>New high-cloning-efficiency vectors for complementation studies and recombinant protein overproduction in Escherichia coli and Salmonella enterica</strong></div><div><em>Plasmid.</em> 2016; <strong>86</strong>:1-6</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.VanDrisseJ.C.Escalante-SemerenaNew+high-cloning-efficiency+vectors+for+complementation+studies+and+recombinant+protein+overproduction+in+Escherichia+coli+and+Salmonella+entericaPlasmid86201616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>provided by C. VanDrisse, Caltech</td></tr><tr><td><i>RNAi target primers</i></td><td>This study</td><td>Sequence (5′ – 3′)</td></tr><tr><td>O-CYP4G _F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CACTCCCTGTCGGAACCTTGGA</td></tr><tr><td>O-CYP4G _R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TTGCGACATCCTCCACAGACGT</td></tr><tr><td>O-CYP4G _F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>ACGTCTGTGGAGGATGTCGCAA</td></tr><tr><td>O-CYP4G _R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>ATCCAAAATCCCCGGACCCGAT</td></tr><tr><td>TG-CYP4G_F3</td><td>This study; Integrated DNA Technologies, Inc</td><td>GAACATCCTTGCGTTCCTCAAC</td></tr><tr><td>TG-CYP4G _R3</td><td>This study; Integrated DNA Technologies, Inc</td><td>ATCTGACAGAGAATGCACTTTTGC</td></tr><tr><td>TG-CYP4G _F4</td><td>This study; Integrated DNA Technologies, Inc</td><td>ATGGCCGTTGTAAAGCTTTCGG</td></tr><tr><td>TG-CYP4G _R4</td><td>This study; Integrated DNA Technologies, Inc</td><td>GGCGGTGGTATCATGACCTTCA</td></tr><tr><td>TG-CYP4G _F5</td><td>This study; Integrated DNA Technologies, Inc</td><td>CCAGATGTAATGAATGGCTAGGGC</td></tr><tr><td>TG-CYP4G _R5</td><td>This study; Integrated DNA Technologies, Inc</td><td>CCTGTGCGGAAGGCTAATATTG</td></tr><tr><td>ELO_708_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCCGCTCATGGGTAATTGCTCG</td></tr><tr><td>ELO_708_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TTTTGCCATCGAACATCACCGCA</td></tr><tr><td>ELO_708_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>AAATGGATGGCGAATAGACCCG</td></tr><tr><td>ELO_708_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>GCAGGTTGTGCGGGTTTTGTTT</td></tr><tr><td>ELO_804_F3</td><td>This study; Integrated DNA Technologies, Inc</td><td>GGCTGGGAGTGACAAAACTCGG</td></tr><tr><td>ELO_804_R3</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCCGCAGATGAGGTCCTAGTGC</td></tr><tr><td>ELO_804_F4</td><td>This study; Integrated DNA Technologies, Inc</td><td>TTGCTGGTGGCCATGCAACTTT</td></tr><tr><td>ELO_804_R4</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCTTGGACATGCCATCCGACGA</td></tr><tr><td>Dmd_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCATCCATAGCTCGTACCGTTG</td></tr><tr><td>Dmd_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TTAGTTTCTGTGACGTCCCTGG</td></tr><tr><td>Dmd_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>ATGTGCAGGCAAATCTTGATCG</td></tr><tr><td>Dmd_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>TGGCACGTGTTTGTTCATACAC</td></tr><tr><td>Dmd_F3</td><td>This study; Integrated DNA Technologies, Inc</td><td>CCAGGGACGTCACAGAAACTAA</td></tr><tr><td>Dmd_R3</td><td>This study; Integrated DNA Technologies, Inc</td><td>ATTGGACTATAAATGGCCGCCT</td></tr><tr><td>Lac_2 _long_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CCTGGAGGTGACCACGTCTTGA</td></tr><tr><td>Lac_2_long_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CTCCTATAGGCGTTGGCGGAGT</td></tr><tr><td>Lac_2_ short_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>GCCAGTTCCGTGATCCCAACAC</td></tr><tr><td>Lac_2_ short_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>GATCTTGACATCTGGACGCGCC</td></tr><tr><td>MeOS_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCCTGGTTTGGACAGTCTGCGA</td></tr><tr><td>MeOS_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CGTCGAGCGTCGAGCACTTATG</td></tr><tr><td>TG-α-EST_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CGGTTTCCTCAACCTCAAAGACAC</td></tr><tr><td>TG-α-EST_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>GTCTGCGTCGTTCTTTGGAGTG</td></tr><tr><td>TG-α-EST_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>TGCCCAAACACAACTTGCAACT</td></tr><tr><td>TG-α-EST_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>CTCCACAACCACCGAAACCATG</td></tr><tr><td>O-FASN_708_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>AGTACGCCAGGTTTCTCCGTGT</td></tr><tr><td>O-FASN_708_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>AACAGCGCAACCAGTTGTACCC</td></tr><tr><td>O-FASN_708_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>TGCCAATGGTCCTCCCCATGAA</td></tr><tr><td>O-FASN_708_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCAGCTCTGAGGACAGCTGCTC</td></tr><tr><td>O-FASN_1114_F3</td><td>This study; Integrated DNA Technologies, Inc</td><td>GTCTAGCATCACCAGCACGCTG</td></tr><tr><td>O-FASN_1114_R3</td><td>This study; Integrated DNA Technologies, Inc</td><td>AGTTCCCGATGCTCCTGAACGT</td></tr><tr><td>O-FASN_1114_F4</td><td>This study; Integrated DNA Technologies, Inc</td><td>GGGAGCTTCTTCTCCTCCGCTT</td></tr><tr><td>O-FASN_1114_R4</td><td>This study; Integrated DNA Technologies, Inc</td><td>AACGCGGGTCAAACCAACTACG</td></tr><tr><td>O-FASN_1220_F5</td><td>This study; Integrated DNA Technologies, Inc</td><td>GGTGGCACGTTGCAACAAAAGG</td></tr><tr><td>O-FASN_1220_R5</td><td>This study; Integrated DNA Technologies, Inc</td><td>CTGTAGTCCGCATCAACGCCAG</td></tr><tr><td>O-FASN_1220_F6</td><td>This study; Integrated DNA Technologies, Inc</td><td>GATCGACCACCCTGAGATCCCC</td></tr><tr><td>O-FASN_1220_R6</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCCGAGCGGGTATAACCAGTCG</td></tr><tr><td>O-FAR_934_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CAGGCGGAACTGGTTTCCTTGG</td></tr><tr><td>O-FAR_934_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>GTACAATGGCGACCGGAAGTGG</td></tr><tr><td>O-FAR_934_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>GTGTAACATTCAGCGTGGGCGT</td></tr><tr><td>O-FAR_934_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>ATGCGGCATCTTCCATTCTCGC</td></tr><tr><td>O-FAR_1142_F3</td><td>This study; Integrated DNA Technologies, Inc</td><td>ACGCATGCCACAATAACCACCC</td></tr><tr><td>O-FAR_1142_R3</td><td>This study; Integrated DNA Technologies, Inc</td><td>ACGCGACGAACATCCAATGGAC</td></tr><tr><td>O-FAR_1142_F4</td><td>This study; Integrated DNA Technologies, Inc</td><td>TGCCATAATCCCCAACAAGTTTGGC</td></tr><tr><td>O-FAR_1142_R4</td><td>This study; Integrated DNA Technologies, Inc</td><td>CGGATTTGTTGGTAGGGCGCTT</td></tr><tr><td>MFASN_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>AGCAATTTCTTGGCACCCGCTT</td></tr><tr><td>MFASN_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>TCTAGCAACTCTGGCTTGGGCA</td></tr><tr><td>TG-FAR_F1</td><td>This study; Integrated DNA Technologies, Inc</td><td>GCACTGCCAGGTTCAACGTC</td></tr><tr><td>TG-FAR_R1</td><td>This study; Integrated DNA Technologies, Inc</td><td>CCAACTGACCCTAGAGAATTACTC</td></tr><tr><td>TG-FAR_F2</td><td>This study; Integrated DNA Technologies, Inc</td><td>CGGAAGAGTTTGCTTGTCACTG</td></tr><tr class="rowsep"><td>TG-FAR_R2</td><td>This study; Integrated DNA Technologies, Inc</td><td>ACGCACTTGACTATATGCTGTACT</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>R v.3.6.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref123-1" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-1" aria-controls="sref123-1">R Core Team, 2018</a><div class="dropBlock__holder" data-db-target-of="sref123-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing. R Foundation for Statistical Computing</strong></div><div>2018</div><div><a id="intref0650" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing.+R+Foundation+for+Statistical+Computing2018https%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="https://cran.r-project.org/bin/windows/base/old/3.6.0/" target="_blank">https://cran.r-project.org/bin/windows/base/old/3.6.0/</a></td></tr><tr><td>Past: Paleontological Statistics Software Package for Education and Data Analysis 4.04</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">Hammer et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Hammer, O. ∙ Harper, D.A. ∙ Ryan, P.D.</div><div><strong>PAST: Paleontological statistics software package for education and data analysis</strong></div><div><em>Palaeontol. Electronica.</em> 2001; <strong>4</strong>:9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.HammerD.A.HarperP.D.RyanPAST%3A+Paleontological+statistics+software+package+for+education+and+data+analysisPalaeontol.+Electronica420019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://www.nhm.uio.no/english/research/infrastructure/past/downloads/past4.04.zip" target="_blank">https://www.nhm.uio.no/english/research/infrastructure/past/downloads/past4.04.zip</a></td></tr><tr><td>Phyton v3</td><td><span class="dropBlock reference-citations"><a id="body-ref-oref0020-1" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="oref0020-1" aria-controls="oref0020-1">Van Rossum and Drake, 2000</a><div class="dropBlock__holder" data-db-target-of="oref0020-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div>Van Rossum, G., &amp; Drake, F. L. (2000). <i>Python reference manual</i>. (iUniverse).</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Van+Rossum%2C+G.%2C+%26+Drake%2C+F.+L.+%282000%29.+Python+reference+manual.+%28iUniverse%29." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0065" href="https://www.python.org/download/releases/3.0/" target="_blank">https://www.python.org/download/releases/3.0/</a></td></tr><tr><td>TransDecoder</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Haas et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Haas, B.J. ∙ Papanicolaou, A. ∙ Yassour, M. ...</div><div><strong>De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:1494-1512</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasA.PapanicolaouM.YassourM.GrabherrP.D.BloodJ.BowdenM.B.CougerD.EcclesB.LiM.LieberDe+novo+transcript+sequence+reconstruction+from+RNA-seq+using+the+Trinity+platform+for+reference+generation+and+analysisNat.+Protoc.8201314941512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0070" href="https://github.com/TransDecoder/TransDecoder/wiki" target="_blank">https://github.com/TransDecoder/TransDecoder/wiki</a></td></tr><tr><td>cd-hit</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Fu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Fu, L. ∙ Niu, B. ∙ Zhu, Z. ...</div><div><strong>CD-HIT: accelerated for clustering the next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:3150-3152</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FuB.NiuZ.ZhuS.WuW.LiCD-HIT%3A+accelerated+for+clustering+the+next-generation+sequencing+dataBioinformatics28201231503152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="http://weizhongli-lab.org/cd-hit/" target="_blank">http://weizhongli-lab.org/cd-hit/</a></td></tr><tr><td>Orthofinder v2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref48-1" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-1" aria-controls="sref48-1">Emms and Kelly, 2015</a><div class="dropBlock__holder" data-db-target-of="sref48-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Emms, D.M. ∙ Kelly, S.</div><div><strong>OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:157</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.EmmsS.KellyOrthoFinder%3A+solving+fundamental+biases+in+whole+genome+comparisons+dramatically+improves+orthogroup+inference+accuracyGenome+Biol.162015157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref49-1" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-1" aria-controls="sref49-1">2019</a><div class="dropBlock__holder" data-db-target-of="sref49-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Emms, D.M. ∙ Kelly, S.</div><div><strong>OrthoFinder: phylogenetic orthology inference for comparative genomics</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:238</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.EmmsS.KellyOrthoFinder%3A+phylogenetic+orthology+inference+for+comparative+genomicsGenome+Biol.202019238" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0080" href="https://github.com/davidemms/OrthoFinder" target="_blank">https://github.com/davidemms/OrthoFinder</a></td></tr><tr><td>Diamond</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref28-1" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-1" aria-controls="sref28-1">Buchfink et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref28-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Buchfink, B. ∙ Xie, C. ∙ Huson, D.H.</div><div><strong>Fast and sensitive protein alignment using DIAMOND</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:59-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.BuchfinkC.XieD.H.HusonFast+and+sensitive+protein+alignment+using+DIAMONDNat.+Methods1220155960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="https://github.com/bbuchfink/diamond" target="_blank">https://github.com/bbuchfink/diamond</a></td></tr><tr><td>MCL clustering</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Enright et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Enright, A.J. ∙ Van Dongen, S. ∙ Ouzounis, C.A.</div><div><strong>An efficient algorithm for large-scale detection of protein families</strong></div><div><em>Nucleic Acids Res.</em> 2002; <strong>30</strong>:1575-1584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.EnrightS.Van+DongenC.A.OuzounisAn+efficient+algorithm+for+large-scale+detection+of+protein+familiesNucleic+Acids+Res.30200215751584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="https://metacpan.org/pod/Bio::Tools::Run::TribeMCL" target="_blank">https://metacpan.org/pod/Bio::Tools::Run::TribeMCL</a></td></tr><tr><td>MAFFT</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Katoh and Standley, 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Katoh, K. ∙ Standley, D.M.</div><div><strong>MAFFT multiple sequence alignment software version 7: improvements in performance and usability</strong></div><div><em>Mol. Biol. Evol.</em> 2013; <strong>30</strong>:772-780</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KatohD.M.StandleyMAFFT+multiple+sequence+alignment+software+version+7%3A+improvements+in+performance+and+usabilityMol.+Biol.+Evol.302013772780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0095" href="https://mafft.cbrc.jp/alignment/software/" target="_blank">https://mafft.cbrc.jp/alignment/software/</a></td></tr><tr><td>FastTree 2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref116-1" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-1" aria-controls="sref116-1">Price et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref116-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Price, M.N. ∙ Dehal, P.S. ∙ Arkin, A.P.</div><div><strong>FastTree 2--approximately maximum-likelihood trees for large alignments</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e9490</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.N.PriceP.S.DehalA.P.ArkinFastTree+2--approximately+maximum-likelihood+trees+for+large+alignmentsPLoS+ONE52010e9490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0100" href="http://www.microbesonline.org/fasttree/" target="_blank">http://www.microbesonline.org/fasttree/</a></td></tr><tr><td>IQ-TREE v1.6.12</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref103-1" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-1" aria-controls="sref103-1">Nguyen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref103-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Nguyen, L.-T. ∙ Schmidt, H.A. ∙ von Haeseler, A. ...</div><div><strong>IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:268-274</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.-T.NguyenH.A.SchmidtA.von+HaeselerB.Q.MinhIQ-TREE%3A+a+fast+and+effective+stochastic+algorithm+for+estimating+maximum-likelihood+phylogeniesMol.+Biol.+Evol.322015268274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0105" href="http://www.iqtree.org/" target="_blank">http://www.iqtree.org/</a></td></tr><tr><td>FigTree v1.4.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref121-1" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-1" aria-controls="sref121-1">Rambaut, 2012</a><div class="dropBlock__holder" data-db-target-of="sref121-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Rambaut, A.</div><div><strong>FigTree v1.4.1</strong></div><div>2012</div><div><a id="intref0645" href="http://Tree.Biol.Ed.Ac.Uk/Software/Figtree/" target="_blank">http://Tree.Biol.Ed.Ac.Uk/Software/Figtree/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.RambautFigTree+v1.4.12012http%3A%2F%2FTree.Biol.Ed.Ac.Uk%2FSoftware%2FFigtree%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0110" href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">http://tree.bio.ed.ac.uk/software/figtree/</a></td></tr><tr><td>REViGO tool</td><td><a id="intref0115" href="http://revigo.irb.hr" target="_blank">http://revigo.irb.hr</a></td><td><a id="intref0120" href="http://revigo.irb.hr/" target="_blank">http://revigo.irb.hr/</a></td></tr><tr><td>MonaGO tool</td><td><a id="intref0125" href="https://monago.erc.monash.edu/.s17" target="_blank">https://monago.erc.monash.edu/.s17</a></td><td><a id="intref0130" href="https://monago.erc.monash.edu/" target="_blank">https://monago.erc.monash.edu/</a></td></tr><tr><td>cNMF</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="https://github.com/dylkot/cNMF" target="_blank">https://github.com/dylkot/cNMF</a></td></tr><tr><td>Seurat v3.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ButlerP.HoffmanP.SmibertE.PapalexiR.SatijaIntegrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+speciesNat.+Biotechnol.362018411420" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref142-1" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-1" aria-controls="sref142-1">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref142-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.StuartA.ButlerP.HoffmanC.HafemeisterE.PapalexiW.M.Mauck3rdY.HaoM.StoeckiusP.SmibertR.SatijaComprehensive+Integration+of+Single-Cell+DataCell177201918881902.e21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0140" href="https://cran.r-project.org/web/packages/Seurat/index.html" target="_blank">https://cran.r-project.org/web/packages/Seurat/index.html</a></td></tr><tr><td>BUStools (v.0.39.3)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Melsted et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Melsted, P. ∙ Booeshaghi, A.S. ∙ Liu, L. ...</div><div><strong>Modular, efficient and constant-memory single-cell RNA-seq preprocessing</strong></div><div><em>Nat. Biotechnol.</em> 2021; <strong>39</strong>:813-818</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MelstedA.S.BooeshaghiL.LiuF.GaoL.LuK.H.J.MinE.da+Veiga+BeltrameK.E.Hj%C3%B6rleifssonJ.GehringL.PachterModular%2C+efficient+and+constant-memory+single-cell+RNA-seq+preprocessingNat.+Biotechnol.392021813818" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0145" href="https://bustools.github.io/about" target="_blank">https://bustools.github.io/about</a></td></tr><tr><td>sleuth 0.30.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref112-1" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-1" aria-controls="sref112-1">Pimentel et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref112-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Pimentel, H. ∙ Bray, N.L. ∙ Puente, S. ...</div><div><strong>Differential analysis of RNA-seq incorporating quantification uncertainty</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:687-690</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.PimentelN.L.BrayS.PuenteP.MelstedL.PachterDifferential+analysis+of+RNA-seq+incorporating+quantification+uncertaintyNat.+Methods142017687690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0150" href="https://github.com/pachterlab/sleuth" target="_blank">https://github.com/pachterlab/sleuth</a></td></tr><tr><td>kallisto 0.46.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.L.BrayH.PimentelP.MelstedL.PachterNear-optimal+probabilistic+RNA-seq+quantificationNat.+Biotechnol.342016525527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="https://pachterlab.github.io/kallisto/" target="_blank">https://pachterlab.github.io/kallisto/</a></td></tr><tr><td>BUSCO v4.1.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref132-1" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-1" aria-controls="sref132-1">Simão et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref132-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Simão, F.A. ∙ Waterhouse, R.M. ∙ Ioannidis, P. ...</div><div><strong>BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3210-3212</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.A.Sim%C3%A3oR.M.WaterhouseP.IoannidisE.V.KriventsevaE.M.ZdobnovBUSCO%3A+assessing+genome+assembly+and+annotation+completeness+with+single-copy+orthologsBioinformatics31201532103212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0160" href="https://gitlab.com/ezlab/busco" target="_blank">https://gitlab.com/ezlab/busco</a></td></tr><tr><td>BLAST</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Altschul et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Altschul, S.F. ∙ Gish, W. ∙ Miller, W. ...</div><div><strong>Basic local alignment search tool</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1990; <strong>215</strong>:403-410</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.AltschulW.GishW.MillerE.W.MyersD.J.LipmanBasic+local+alignment+search+toolJ.%C2%A0Mol.+Biol.2151990403410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0165" href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a></td></tr><tr><td>Trinity v2.8.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Grabherr et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Grabherr, M.G. ∙ Haas, B.J. ∙ Yassour, M. ...</div><div><strong>Full-length transcriptome assembly from RNA-Seq data without a reference genome</strong></div><div><em>Nat. Biotechnol.</em> 2011; <strong>29</strong>:644-652</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.G.GrabherrB.J.HaasM.YassourJ.Z.LevinD.A.ThompsonI.AmitX.AdiconisL.FanR.RaychowdhuryQ.ZengFull-length+transcriptome+assembly+from+RNA-Seq+data+without+a+reference+genomeNat.+Biotechnol.292011644652" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0170" href="https://github.com/trinityrnaseq/trinityrnaseq" target="_blank">https://github.com/trinityrnaseq/trinityrnaseq</a></td></tr><tr><td>STAR 2.7.0a</td><td><span class="dropBlock reference-citations"><a id="body-ref-optMaXjtQ2q4i-1" href="#" role="doc-biblioref" data-xml-rid="bib167" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optMaXjtQ2q4i-1" aria-controls="optMaXjtQ2q4i-1">Dobin et al., 2013</a><div class="dropBlock__holder" data-db-target-of="optMaXjtQ2q4i-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.DobinC.A.DavisF.SchlesingerJ.DrenkowC.ZaleskiS.JhaP.BatutM.ChaissonT.R.GingerasSTAR%3A+ultrafast+universal+RNA-seq+alignerBioinformatics2920131521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0175" href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR</a></td></tr><tr><td>JupyterLab</td><td><span class="dropBlock reference-citations"><a id="body-ref-oref0010-1" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="oref0010-1" aria-controls="oref0010-1">Kluyver et al., 2016</a><div class="dropBlock__holder" data-db-target-of="oref0010-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div>Kluyver, T., Ragan-Kelley, B., Pérez, F., Granger, B.E., Bussonnier, M., Frederic, J., Kelley, K., Hamrick, J., Grout, J., Corlay, S., et al. (2016). <i>Jupyter Notebooks-a publishing format for reproducible computational workflows. In proceedings of the 20th International Conference on Electronic Publishing.</i> pp. 87-90.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Kluyver%2C+T.%2C+Ragan-Kelley%2C+B.%2C+P%C3%A9rez%2C+F.%2C+Granger%2C+B.E.%2C+Bussonnier%2C+M.%2C+Frederic%2C+J.%2C+Kelley%2C+K.%2C+Hamrick%2C+J.%2C+Grout%2C+J.%2C+Corlay%2C+S.%2C+et+al.+%282016%29.+Jupyter+Notebooks-a+publishing+format+for+reproducible+computational+workflows.+In+proceedings+of+the+20th+International+Conference+on+Electronic+Publishing.+pp.+87-90." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0180" href="https://jupyter.org/" target="_blank">https://jupyter.org/</a></td></tr><tr><td>AnnotationForge 1.34.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref32-1" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-1" aria-controls="sref32-1">Carlson and Pages, 2019</a><div class="dropBlock__holder" data-db-target-of="sref32-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Carlson, M. ∙ Pages, H.</div><div><strong>AnnotationForge: tools for building SQLite-based annotation data packages. R package version 1.36.0</strong></div><div>2019</div><div><a id="intref0635" href="https://bioconductor.org/packages/AnnotationForge" target="_blank">https://bioconductor.org/packages/AnnotationForge</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.CarlsonH.PagesAnnotationForge%3A+tools+for+building+SQLite-based+annotation+data+packages.+R+package+version+1.36.02019https%3A%2F%2Fbioconductor.org%2Fpackages%2FAnnotationForge" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0185" href="https://bioconductor.org/packages/release/bioc/html/AnnotationForge.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/AnnotationForge.html</a></td></tr><tr><td>AnnotationHub 3.0.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Morgan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Morgan, M. ∙ Carlson, M. ∙ Tenenbaum, D. ...</div><div><strong>Package AnnotationHub</strong></div><div>2019</div><div><a id="intref0640" href="https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MorganM.CarlsonD.TenenbaumS.AroraPackage+AnnotationHub2019https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FAnnotationHub.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0190" href="https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html</a></td></tr><tr><td>Biomart 2.48</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Durinck et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Durinck, S. ∙ Spellman, P.T. ∙ Birney, E. ...</div><div><strong>Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:1184-1191</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.DurinckP.T.SpellmanE.BirneyW.HuberMapping+identifiers+for+the+integration+of+genomic+datasets+with+the+R%2FBioconductor+package+biomaRtNat.+Protoc.4200911841191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0195" href="https://bioconductor.org/packages/release/bioc/html/biomaRt.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/biomaRt.html</a></td></tr><tr><td>boot</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Canty and Ripley, 2017</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Canty, A. ∙ Ripley, B.</div><div><strong>boot: Bootstrap R (S-Plus) functions</strong></div><div><em>R Package Version.</em> 2017; <strong>1</strong>:3-20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CantyB.Ripleyboot%3A+Bootstrap+R+%28S-Plus%29+functionsR+Package+Version12017320" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0200" href="https://cran.r-project.org/web/packages/boot/boot.pdf" target="_blank">https://cran.r-project.org/web/packages/boot/boot.pdf</a></td></tr><tr><td>BUSpaRse 1.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref101-1" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-1" aria-controls="sref101-1">Moses and Pachter, 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Moses, L. ∙ Pachter, L.</div><div><strong>BUSpaRse: kallisto | bustools R utilities, R Package Version 1.4.2</strong></div><div>2021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.MosesL.PachterBUSpaRse%3A+kallisto+%7C+bustools+R+utilities%2C+R+Package+Version+1.4.22021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0205" href="https://bioconductor.org/packages/release/bioc/html/BUSpaRse.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/BUSpaRse.html</a></td></tr><tr><td>Car 3.0-10</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Fox et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Fox, J. ∙ Weisberg, S. ∙ Adler, D. ...</div><div><strong>Package ‘car’</strong></div><div>R Foundation for Statistical Computing, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxS.WeisbergD.AdlerD.BatesG.Baud-BovyS.EllisonD.FirthM.FriendlyG.GorjancS.GravesPackage+%E2%80%98car%E2%80%992012R+Foundation+for+Statistical+Computing" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0210" href="https://cran.r-project.org/web/packages/car/index.html" target="_blank">https://cran.r-project.org/web/packages/car/index.html</a></td></tr><tr><td>clusterProfiler 2.8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref160-1" href="#" role="doc-biblioref" data-xml-rid="bib160" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160-1" aria-controls="sref160-1">Yu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref160-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Yu, G. ∙ Wang, L.-G. ∙ Han, Y. ...</div><div><strong>clusterProfiler: an R package for comparing biological themes among gene clusters</strong></div><div><em>OMICS.</em> 2012; <strong>16</strong>:284-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.YuL.-G.WangY.HanQ.-Y.HeclusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clustersOMICS162012284287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0215" href="https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html</a></td></tr><tr><td>Cowplot 1.1.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref159-1" href="#" role="doc-biblioref" data-xml-rid="bib159" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref159-1" aria-controls="sref159-1">Wilke, 2017</a><div class="dropBlock__holder" data-db-target-of="sref159-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Wilke, C.</div><div><strong>Package ‘cowplot’: Streamlined Plot Theme and Plot Annotations for ‘ggplot2.’ CRAN</strong></div><div>2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.WilkePackage+%E2%80%98cowplot%E2%80%99%3A+Streamlined+Plot+Theme+and+Plot+Annotations+for+%E2%80%98ggplot2.%E2%80%99+CRAN2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0220" href="https://cran.r-project.org/web/packages/cowplot/index.html" target="_blank">https://cran.r-project.org/web/packages/cowplot/index.html</a></td></tr><tr><td>data.table 1.12.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Dowle et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Dowle, M. ∙ Srinivasan, A. ∙ Gorecki, J. ...</div><div><strong>Package ‘data. table.’ Extension of ‘data</strong></div><div>Frame, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DowleA.SrinivasanJ.GoreckiM.ChiricoP.StetsenkoT.ShortS.LianoglouE.AntonyanM.BonschH.ParsonagePackage+%E2%80%98data.+table.%E2%80%99+Extension+of+%E2%80%98data2019Frame" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0225" href="https://cran.r-project.org/web/packages/data.table/data.table.pdf" target="_blank">https://cran.r-project.org/web/packages/data.table/data.table.pdf</a></td></tr><tr><td>Dbplyr 1.4.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref157-1" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157-1" aria-controls="sref157-1">Wickham and Ruiz, 2019</a><div class="dropBlock__holder" data-db-target-of="sref157-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>Wickham, H. ∙ Ruiz, E.</div><div><strong>dbplyr: A “dplyr” Back End for Databases. R&nbsp;package Version 1.4. 0</strong></div><div>2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamE.Ruizdbplyr%3A+A+%E2%80%9Cdplyr%E2%80%9D+Back+End+for+Databases.+R%C2%A0package+Version+1.4.+02019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0230" href="https://dbplyr.tidyverse.org/" target="_blank">https://dbplyr.tidyverse.org/</a></td></tr><tr><td>DropletUtils 1.8.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Lun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ..., 1st Human Cell Atlas Jamboree</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.L.LunS.RiesenfeldT.AndrewsT.P.DaoT.GomesJ.C.Marioni1st+Human+Cell+Atlas+JamboreeEmptyDrops%3A+distinguishing+cells+from+empty+droplets+in+droplet-based+single-cell+RNA+sequencing+dataGenome+Biol.20201963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0235" href="https://bioconductor.org/packages/release/bioc/html/DropletUtils.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/DropletUtils.html</a></td></tr><tr><td>Effects 4.2-0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Fox, 2003</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Fox, J.</div><div><strong>Effect displays in R for generalised linear models</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2003; <strong>8</strong>:1-27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxEffect+displays+in+R+for+generalised+linear+modelsJ.%C2%A0Stat.+Softw.82003127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0240" href="https://cran.r-project.org/web/packages/effects/effects.pdf" target="_blank">https://cran.r-project.org/web/packages/effects/effects.pdf</a></td></tr><tr><td>ggplot2 3.3.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref156-1" href="#" role="doc-biblioref" data-xml-rid="bib156" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref156-1" aria-controls="sref156-1">Wickham, 2011</a><div class="dropBlock__holder" data-db-target-of="sref156-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Wickham, H.</div><div><strong>ggplot2</strong></div><div><em>Wiley Interdiscip. Rev. Comput. Stat.</em> 2011; <strong>3</strong>:180-185</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.Wickhamggplot2Wiley+Interdiscip.+Rev.+Comput.+Stat.32011180185" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0245" href="https://ggplot2.tidyverse.org/" target="_blank">https://ggplot2.tidyverse.org/</a></td></tr><tr><td>lattice 0.20-44</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref127-1" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-1" aria-controls="sref127-1">Sarkar, 2008</a><div class="dropBlock__holder" data-db-target-of="sref127-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Sarkar, D.</div><div><strong>Lattice: Multivariate Data Visualization with R</strong></div><div>Springer Science &amp; Business Media, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.SarkarLattice%3A+Multivariate+Data+Visualization+with+R2008Springer+Science+%26+Business+Media" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0250" href="https://cran.r-project.org/web/packages/lattice/lattice.pdf" target="_blank">https://cran.r-project.org/web/packages/lattice/lattice.pdf</a></td></tr><tr><td>lme4 1.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Bates et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bates, D. ∙ Sarkar, D. ∙ Bates, M.D. ...</div><div><strong>The lme4 package</strong></div><div><em>R Package Version.</em> 2007; <strong>2</strong>:74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.BatesD.SarkarM.D.BatesL.MatrixThe+lme4+packageR+Package+Version2200774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0255" href="https://cran.r-project.org/web/packages/lme4/lme4.pdf" target="_blank">https://cran.r-project.org/web/packages/lme4/lme4.pdf</a></td></tr><tr><td>MASS 7.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref122-1" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-1" aria-controls="sref122-1">Ripley et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref122-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Ripley, B. ∙ Venables, B. ∙ Bates, D.M. ...</div><div><strong>Package ‘mass.’</strong></div><div><em>Cran R.</em> 2013; <strong>538</strong>:113-120</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.RipleyB.VenablesD.M.BatesK.HornikA.GebhardtD.FirthM.B.RipleyPackage+%E2%80%98mass.%E2%80%99Cran+R5382013113120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0260" href="https://cran.r-project.org/web/packages/MASS/index.html" target="_blank">https://cran.r-project.org/web/packages/MASS/index.html</a></td></tr><tr><td>Matrix 1.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Bates and Maechler, 2010</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bates, D. ∙ Maechler, M.</div><div><strong>Matrix: sparse and dense matrix classes and methods. R Package Version</strong></div><div>2010</div><div><a id="intref0630" href="https://cran.r-project.org/web/packages/Matrix/index.html" target="_blank">https://cran.r-project.org/web/packages/Matrix/index.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.BatesM.MaechlerMatrix%3A+sparse+and+dense+matrix+classes+and+methods.+R+Package+Version2010https%3A%2F%2Fcran.r-project.org%2Fweb%2Fpackages%2FMatrix%2Findex.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0265" href="https://cran.r-project.org/web/packages/Matrix/Matrix.pdf" target="_blank">https://cran.r-project.org/web/packages/Matrix/Matrix.pdf</a></td></tr><tr><td>nlme 3.1-108</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref114-1" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-1" aria-controls="sref114-1">Pinheiro and Bates, 2007</a><div class="dropBlock__holder" data-db-target-of="sref114-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Pinheiro, J. ∙ Bates, D.</div><div><strong>Linear and nonlinear mixed effects models</strong></div><div><em>R Package Version.</em> 2007; <strong>3</strong>:1-89</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PinheiroD.BatesLinear+and+nonlinear+mixed+effects+modelsR+Package+Version32007189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0270" href="https://cran.r-project.org/web/packages/nlme/nlme.pdf" target="_blank">https://cran.r-project.org/web/packages/nlme/nlme.pdf</a></td></tr><tr><td>randomForest 4.6-16</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Liaw and Wiener, 2002</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Liaw, A. ∙ Wiener, M.</div><div><strong>Classification and regression by randomForest</strong></div><div><em>R News.</em> 2002; <strong>2/3</strong>:18-22</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.LiawM.WienerClassification+and+regression+by+randomForestR+News2%2F320021822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0275" href="https://cran.r-project.org/web/packages/randomForest/randomForest.pdf" target="_blank">https://cran.r-project.org/web/packages/randomForest/randomForest.pdf</a></td></tr><tr><td>tidyverse 1.2.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref158-1" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158-1" aria-controls="sref158-1">Wickham et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref158-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the Tidyverse</strong></div><div><em>J.&nbsp;Open Source Softw.</em> 2019; <strong>4</strong>:1686</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamM.AverickJ.BryanW.ChangL.D.McGowanR.Fran%C3%A7oisG.GrolemundA.HayesL.HenryJ.HesterWelcome+to+the+TidyverseJ.%C2%A0Open+Source+Softw.420191686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0280" href="https://www.tidyverse.org/" target="_blank">https://www.tidyverse.org/</a></td></tr><tr><td>vcd 1.3-2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref98-1" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-1" aria-controls="sref98-1">Meyer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref98-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Meyer, D. ∙ Zeileis, A. ∙ Hornik, K. ...</div><div><strong>Package ‘vcd.’ Visualizing Categorical Data. R Package Version 1–4</strong></div><div>2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.MeyerA.ZeileisK.HornikF.GerberM.FriendlyM.D.MeyerPackage+%E2%80%98vcd.%E2%80%99+Visualizing+Categorical+Data.+R+Package+Version+1%E2%80%9342020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0285" href="https://cran.r-project.org/web/packages/vcd/index.html" target="_blank">https://cran.r-project.org/web/packages/vcd/index.html</a></td></tr><tr><td>FastQC v0.11.8</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Andrews, S.</div><div><strong>FastQC: a quality control tool for high throughput sequence data</strong></div><div>2010</div><div><a id="intref0625" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+a+quality+control+tool+for+high+throughput+sequence+data2010https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0290" href="https://github.com/s-andrews/FastQC/blob/master/RELEASE_NOTES.txt" target="_blank">https://github.com/s-andrews/FastQC/blob/master/RELEASE_NOTES.txt</a></td></tr><tr><td>cutadapt v1.18</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Martin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnet. J.</em> 2011; <strong>17</strong>:10-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MartinCutadapt+removes+adapter+sequences+from+high-throughput+sequencing+readsEMBnet.+J.1720111012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0295" href="https://kbase.us/applist/apps/kb_cutadapt/remove_adapters/release" target="_blank">https://kbase.us/applist/apps/kb_cutadapt/remove_adapters/release</a></td></tr><tr><td>MEGAHIT v1.1.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Li et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Li, D. ∙ Luo, R. ∙ Liu, C.-M. ...</div><div><strong>MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices.</strong></div><div><em>Methods.</em> 2016; <strong>102</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.LiR.LuoC.-M.LiuC.-M.LeungH.-F.TingK.SadakaneH.YamashitaT.-W.LamMEGAHIT+v1.0%3A+A+fast+and+scalable+metagenome+assembler+driven+by+advanced+methodologies+and+community+practices.Methods1022016311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0300" href="https://github.com/voutcn/megahit" target="_blank">https://github.com/voutcn/megahit</a></td></tr><tr><td>Redundans v0.14a</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref118-1" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-1" aria-controls="sref118-1">Pryszcz and Gabaldón, 2016</a><div class="dropBlock__holder" data-db-target-of="sref118-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Pryszcz, L.P. ∙ Gabaldón, T.</div><div><strong>Redundans: an assembly pipeline for highly heterozygous genomes</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e113</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.P.PryszczT.Gabald%C3%B3nRedundans%3A+an+assembly+pipeline+for+highly+heterozygous+genomesNucleic+Acids+Res.442016e113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0305" href="https://github.com/lpryszcz/redundans" target="_blank">https://github.com/lpryszcz/redundans</a></td></tr><tr><td>GapCloser v1.12</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Luo et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Luo, R. ∙ Liu, B. ∙ Xie, Y. ...</div><div><strong>SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</strong></div><div><em>Gigascience.</em> 2012; <strong>1</strong>:18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.LuoB.LiuY.XieZ.LiW.HuangJ.YuanG.HeY.ChenQ.PanY.LiuSOAPdenovo2%3A+an+empirically+improved+memory-efficient+short-read+de+novo+assemblerGigascience1201218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0310" href="https://github.com/BGI-Qingdao/stLFR_GapCloser" target="_blank">https://github.com/BGI-Qingdao/stLFR_GapCloser</a></td></tr><tr><td>RepeatModeler v 1.0.11</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref134-1" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-1" aria-controls="sref134-1">Smit and Hubley, 2008</a><div class="dropBlock__holder" data-db-target-of="sref134-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Smit, A.F. ∙ Hubley, R.</div><div><strong>RepeatModeler Open-1.0</strong></div><div>2008</div><div><a id="intref0660" href="https://www.scienceopen.com/document?vid=c1050e3d-8dc1-48ba-81c8-b372af25be50" target="_blank">https://www.scienceopen.com/document?vid=c1050e3d-8dc1-48ba-81c8-b372af25be50</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.F.SmitR.HubleyRepeatModeler+Open-1.02008https%3A%2F%2Fwww.scienceopen.com%2Fdocument%3Fvid%3Dc1050e3d-8dc1-48ba-81c8-b372af25be50" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0315" href="https://www.repeatmasker.org/RepeatModeler/" target="_blank">https://www.repeatmasker.org/RepeatModeler/</a></td></tr><tr><td>MITE tracker</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Crescente et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Crescente, J.M. ∙ Zavallo, D. ∙ Helguera, M. ...</div><div><strong>MITE Tracker: an accurate approach to identify miniature inverted-repeat transposable elements in large genomes</strong></div><div><em>BMC Bioinformatics.</em> 2018; <strong>19</strong>:348</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.CrescenteD.ZavalloM.HelgueraL.S.VanzettiMITE+Tracker%3A+an+accurate+approach+to+identify+miniature+inverted-repeat+transposable+elements+in+large+genomesBMC+Bioinformatics192018348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0320" href="https://github.com/INTABiotechMJ/MITE-Tracker" target="_blank">https://github.com/INTABiotechMJ/MITE-Tracker</a></td></tr><tr><td>vsearch v 2.7.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref124-1" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-1" aria-controls="sref124-1">Rognes et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref124-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Rognes, T. ∙ Flouri, T. ∙ Nichols, B. ...</div><div><strong>VSEARCH: a versatile open source tool for metagenomics</strong></div><div><em>PeerJ.</em> 2016; <strong>4</strong>:e2584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.RognesT.FlouriB.NicholsC.QuinceF.Mah%C3%A9VSEARCH%3A+a+versatile+open+source+tool+for+metagenomicsPeerJ42016e2584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0325" href="https://bioweb.pasteur.fr/packages/pack@vsearch@2.7.1/" target="_blank">https://bioweb.pasteur.fr/packages/pack@vsearch@2.7.1/</a></td></tr><tr><td>RepeatMasker v 4.07</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref133-1" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-1" aria-controls="sref133-1">Smit, 2004</a><div class="dropBlock__holder" data-db-target-of="sref133-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Smit, A.F.</div><div><strong>Repeat-Masker Open-3.0</strong></div><div>2004</div><div><a id="intref0655" href="http://www.Repeatmasker.Org" target="_blank">http://www.Repeatmasker.Org</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.F.SmitRepeat-Masker+Open-3.02004http%3A%2F%2Fwww.Repeatmasker.Org" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0330" href="https://www.repeatmasker.org/" target="_blank">https://www.repeatmasker.org/</a></td></tr><tr><td>GeneMark-ES v.4.33</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref145-1" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-1" aria-controls="sref145-1">Ter-Hovhannisyan et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref145-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Ter-Hovhannisyan, V. ∙ Lomsadze, A. ∙ Chernoff, Y.O. ...</div><div><strong>Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training</strong></div><div><em>Genome Res.</em> 2008; <strong>18</strong>:1979-1990</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.Ter-HovhannisyanA.LomsadzeY.O.ChernoffM.BorodovskyGene+prediction+in+novel+fungal+genomes+using+an+ab+initio+algorithm+with+unsupervised+trainingGenome+Res.18200819791990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0335" href="http://exon.gatech.edu/GeneMark/" target="_blank">http://exon.gatech.edu/GeneMark/</a></td></tr><tr><td>Augustus v3.2.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref138-1" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-1" aria-controls="sref138-1">Stanke and Waack, 2003</a><div class="dropBlock__holder" data-db-target-of="sref138-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>Stanke, M. ∙ Waack, S.</div><div><strong>Gene prediction with a hidden Markov model and a new intron submodel</strong></div><div><em>Bioinformatics.</em> 2003; <strong>19</strong>:ii215-ii225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.StankeS.WaackGene+prediction+with+a+hidden+Markov+model+and+a+new+intron+submodelBioinformatics19Suppl+22003ii215ii225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0340" href="http://augustus.gobics.de/" target="_blank">http://augustus.gobics.de/</a></td></tr><tr><td>BRAKER v2.1.2</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">Brůna et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Brůna, T. ∙ Hoff, K.J. ∙ Lomsadze, A. ...</div><div><strong>BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database</strong></div><div><em>Nar. Genom. Bioinform.</em> 2021; <strong>3</strong>:lqaa108</div></div><div class="external-links"><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.014&amp;key=2-s2.0-85111209000" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.Br%C5%AFnaK.J.HoffA.LomsadzeM.StankeM.BorodovskyBRAKER2%3A+automatic+eukaryotic+genome+annotation+with+GeneMark-EP%2B+and+AUGUSTUS+supported+by+a+protein+databaseNar.+Genom.+Bioinform.32021lqaa108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0345" href="https://github.com/Gaius-Augustus/BRAKER" target="_blank">https://github.com/Gaius-Augustus/BRAKER</a></td></tr><tr><td>PASA v2.3.3</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Haas et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Haas, B.J. ∙ Salzberg, S.L. ∙ Zhu, W. ...</div><div><strong>Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasS.L.SalzbergW.ZhuM.PerteaJ.E.AllenJ.OrvisO.WhiteC.R.BuellJ.R.WortmanAutomated+eukaryotic+gene+structure+annotation+using+EVidenceModeler+and+the+Program+to+Assemble+Spliced+AlignmentsGenome+Biol.92008R7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0350" href="https://github.com/PASApipeline/PASApipeline/releases" target="_blank">https://github.com/PASApipeline/PASApipeline/releases</a></td></tr><tr><td>GeMoMA v1.6.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Keilwagen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Keilwagen, J. ∙ Hartung, F. ∙ Grau, J.</div><div><strong>GeMoMa: Homology-Based Gene Prediction Utilizing Intron Position Conservation and RNA-seq Data</strong></div><div><em>Methods Mol. Biol.</em> 2019; <strong>1962</strong>:161-177</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.KeilwagenF.HartungJ.GrauGeMoMa%3A+Homology-Based+Gene+Prediction+Utilizing+Intron+Position+Conservation+and+RNA-seq+DataMethods+Mol.+Biol.19622019161177" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0355" href="http://www.jstacs.de/index.php/GeMoMa" target="_blank">http://www.jstacs.de/index.php/GeMoMa</a></td></tr><tr><td>EVidenceModeler v.1.1.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Haas et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Haas, B.J. ∙ Salzberg, S.L. ∙ Zhu, W. ...</div><div><strong>Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasS.L.SalzbergW.ZhuM.PerteaJ.E.AllenJ.OrvisO.WhiteC.R.BuellJ.R.WortmanAutomated+eukaryotic+gene+structure+annotation+using+EVidenceModeler+and+the+Program+to+Assemble+Spliced+AlignmentsGenome+Biol.92008R7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0360" href="https://evidencemodeler.github.io/" target="_blank">https://evidencemodeler.github.io/</a></td></tr><tr class="rowsep"><td>Data analysis scripts</td><td>This study</td><td>CaltechData: <a id="intref0365" href="https://doi.org/10.22002/D1.1918" target="_blank">https://doi.org/10.22002/D1.1918</a></td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>processed scRNAseq 10x data</td><td>This study</td><td>CaltechData: <a id="intref0370" href="https://doi.org/10.22002/D1.1915" target="_blank">https://doi.org/10.22002/D1.1915</a></td></tr><tr><td>processed SMARTseq data</td><td>This study</td><td>CaltechData: <a id="intref0375" href="https://doi.org/10.22002/D1.1900" target="_blank">https://doi.org/10.22002/D1.1900</a></td></tr><tr><td>raw rheology video data</td><td>This study</td><td>CaltechData: <a id="intref0380" href="https://doi.org/10.22002/D1.1905" target="_blank">https://doi.org/10.22002/D1.1905</a></td></tr><tr><td>transcriptome data</td><td>This study</td><td>CaltechData: <a id="intref0385" href="https://doi.org/10.22002/D1.1914" target="_blank">https://doi.org/10.22002/D1.1914</a></td></tr><tr><td>RNAi experiments, survival assays, in&nbsp;vitro enzyme data</td><td>This study</td><td>CaltechData: <a id="intref0390" href="https://doi.org/10.22002/D1.1917" target="_blank">https://doi.org/10.22002/D1.1917</a></td></tr><tr><td>alignment and tree fasta files</td><td>This study</td><td>CaltechData: <a id="intref0395" href="https://doi.org/10.22002/D1.1916" target="_blank">https://doi.org/10.22002/D1.1916</a></td></tr><tr><td>genome and transcriptome assemblies of other rove beetles, including analysis script</td><td>This study</td><td>CaltechData: <a id="intref0400" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr><td><i>Drosophila</i> <i>melanogaster</i></td><td>NCBI nr (protein database)</td><td><a id="intref0405" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/215/GCF_000001215.4_Release_6_plus_ISO1_MT/GCF_000001215.4_Release_6_plus_ISO1_MT_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/215/GCF_000001215.4_Release_6_plus_ISO1_MT/GCF_000001215.4_Release_6_plus_ISO1_MT_protein.faa.gz</a></td></tr><tr><td><i>Agrilus planipennis</i></td><td>NCBI nr (protein database)</td><td><a id="intref0410" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/699/045/GCF_000699045.2_Apla_2.0/GCF_000699045.2_Apla_2.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/699/045/GCF_000699045.2_Apla_2.0/GCF_000699045.2_Apla_2.0_protein.faa.gz</a></td></tr><tr><td><i>Anoplophora glabripennis</i></td><td>NCBI nr (protein database)</td><td><a id="intref0415" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/390/285/GCF_000390285.2_Agla_2.0/GCF_000390285.2_Agla_2.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/390/285/GCF_000390285.2_Agla_2.0/GCF_000390285.2_Agla_2.0_protein.faa.gz</a></td></tr><tr><td><i>Leptinotarsa decemlineata</i></td><td>NCBI nr (protein database)</td><td><a id="intref0420" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/500/325/GCF_000500325.1_Ldec_2.0/GCF_000500325.1_Ldec_2.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/500/325/GCF_000500325.1_Ldec_2.0/GCF_000500325.1_Ldec_2.0_protein.faa.gz</a></td></tr><tr><td><i>Dendroctonus ponderosae</i></td><td>NCBI nr (protein database)</td><td><a id="intref0425" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/355/655/GCF_000355655.1_DendPond_male_1.0/GCF_000355655.1_DendPond_male_1.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/355/655/GCF_000355655.1_DendPond_male_1.0/GCF_000355655.1_DendPond_male_1.0_protein.faa.gz</a></td></tr><tr><td><i>Aethina tumida</i></td><td>NCBI nr (protein database)</td><td><a id="intref0430" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/937/115/GCF_001937115.1_Atum_1.0/GCF_001937115.1_Atum_1.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/937/115/GCF_001937115.1_Atum_1.0/GCF_001937115.1_Atum_1.0_protein.faa.gz</a></td></tr><tr><td><i>Tribolium castaneum</i></td><td>NCBI nr (protein database)</td><td><a id="intref0435" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/002/335/GCF_000002335.3_Tcas5.2/GCF_000002335.3_Tcas5.2_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/002/335/GCF_000002335.3_Tcas5.2/GCF_000002335.3_Tcas5.2_protein.faa.gz</a></td></tr><tr><td><i>Onthophagus taurus</i></td><td>NCBI nr (protein database)</td><td><a id="intref0440" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/648/695/GCF_000648695.1_Otau_2.0/GCF_000648695.1_Otau_2.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/648/695/GCF_000648695.1_Otau_2.0/GCF_000648695.1_Otau_2.0_protein.faa.gz</a></td></tr><tr><td><i>Nicrophorus vespilloides</i></td><td>NCBI nr (protein database)</td><td><a id="intref0445" href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/412/225/GCF_001412225.1_Nicve_v1.0/GCF_001412225.1_Nicve_v1.0_protein.faa.gz" target="_blank">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/001/412/225/GCF_001412225.1_Nicve_v1.0/GCF_001412225.1_Nicve_v1.0_protein.faa.gz</a></td></tr><tr><td><i>Coproporus</i> sp.</td><td>Genome sequencing; this study</td><td>Sequence Read Archive: SRR4301367; CaltechData: <a id="intref0450" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr><td><i>Tachinus</i> sp.</td><td>Genome sequencing; this study</td><td>Sequence Read Archive: SRR15992418; CaltechData: <a id="intref0455" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr><td><i>Aleochara bilineata</i></td><td>NCBI BioProject</td><td>NCBI BioProject: PRJNA378164; CaltechData: <a id="intref0460" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr><td><i>Drusilla canaliculata</i></td><td>Genome sequencing; this study</td><td>Sequence Read Archive: SRR5906249; CaltechData: <a id="intref0465" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr><td><i>Lissagria laeviuscula</i></td><td>RNA sequencing; this study</td><td>Sequence Read Archive: SRR15992419, SRR15992420; CaltechData: <a id="intref0470" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr><td><i>Platyusa sonomae</i></td><td>RNA sequencing; this study</td><td>Sequence Read Archive: SRR15992410, SRR15992411, SRR15992412, SRR15992413; CaltechData: <a id="intref0475" href="https://doi.org/10.22002/D1.1919" target="_blank">https://doi.org/10.22002/D1.1919</a></td></tr><tr class="rowsep"><td><i>Dalotia coriaria</i> reference genome assembly</td><td><a id="intref0480" href="http://131.215.78.39/genomebrowser/cgi-bin/hgGateway?genome=Dcor2&amp;hubUrl=http://131.215.78.39/genomebrowser/evolution/parkergroup/Dalotia_coriaria/hub2.txt" target="_blank">http://131.215.78.39/genomebrowser/cgi-bin/hgGateway?genome=Dcor2&amp;hubUrl=http://131.215.78.39/genomebrowser/evolution/parkergroup/Dalotia_coriaria/hub2.txt</a></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Other</b></td></tr><tr><td>ZB-5MS fused silica capillary column</td><td>Phenomenex</td><td>7HG-G010-11</td></tr><tr><td>pluriStrainer cell strainer</td><td>ImTec Diagnostics</td><td>43-50040-51</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2901329-5" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0255" role="paragraph">Further information and requests for reagents and resources should be directed to and will be fulfilled by the Lead Contact, Joseph Parker (<a id="intref0485" href="mailto:joep@caltech.edu">joep@caltech.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0260" role="paragraph">Primers and generated dsRNAs are available via request from the Lead Contact.</div></section></section><section id="sec-4-3"><h3>Method details</h3><section id="sec-4-3-1"><h4>Beetle husbandry</h4><div id="p0265" role="paragraph">The Greenhouse Rove Beetle (<i>Dalotia coriaria</i>, Kraatz) strain used in this study was originally donated by Applied Bionomics (Canada) and inbred for nine generations. Beetles were kept as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref109-3" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-3" aria-controls="sref109-3">Parker et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref109-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Parker, J. ∙ Eldredge, K.T. ∙ Thomas, I. ...</div><div><strong>Hox-logic of body plan innovations for social insect symbiosis in rove beetles</strong></div><div><em>bioRxiv.</em> 2018; 198945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerK.T.EldredgeI.ThomasR.ColemanS.DavisHox-logic+of+body+plan+innovations+for+social+insect+symbiosis+in+rove+beetlesbioRxiv2018198945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and fed with oat/poultry-rearing pellet-powder three times per week.</div></section><section id="sec-4-3-2"><h4>Microbial suppression</h4><div id="p0270" role="paragraph">Late third instar larvae were fed with a mixture of sterilized oat/poultry-rearing pellet-powder with amoxicillin, streptomycin and tetracycline antibiotics (2.5% w/w for each) or pure sterile oat/poultry-rearing pellet-powder as a control. Beetles were single-housed in 5&nbsp;cm Petri dishes with a thin layer of Plaster of Paris. Petri dishes were moisturized three times a week and fresh food was provided <i>ad libitum</i>. Ten days after adult eclosion, tergal gland contents were extracted in hexane and analyzed via GC-MS (see below). To assess the effect of antibiotic treatment on absolute numbers of bacteria associated with the beetles, bacterial 16S rRNA copy numbers determined by quantitative PCR (qPCR). Since many insects have bacteria on their cuticles, the beetles were surface washed in 5% (<i>v/v</i>) sodium dodecyl sulfate solution before bacterial quantification. For the control and antibiotic treatments, DNA was extracted from eight replicates of individual <i>Dalotia</i> using the Zymo Research Quick-DNA Miniprep Plus Kit according to manufacturer’s instructions. For qPCR, we used universal eubacterial 16S rRNA gene primers (Univ16SRT-F: 5′-ACTCCTACGGGAGGCAGCAGT-3′; Univ16SRT-R: 5′-TATTACCGCGGCTGCTGGC-3′) (<span class="dropBlock reference-citations"><a id="body-ref-optG10oc3bwOi-3" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optG10oc3bwOi-3" aria-controls="optG10oc3bwOi-3">Clifford et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="optG10oc3bwOi-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Clifford, R.J. ∙ Milillo, M. ∙ Prestwood, J. ...</div><div><strong>Detection of bacterial 16S rRNA and identification of four clinically important bacteria by real-time PCR</strong></div><div><em>PloS one.</em> 2012; <strong>7</strong>, e48558</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.J.CliffordM.MililloJ.PrestwoodR.QuinteroD.V.ZurawskiY.I.KwakP.E.WatermanE.P.LeshoP.Mc+GannDetection+of+bacterial+16S+rRNA+and+identification+of+four+clinically+important+bacteria+by+real-time+PCRPloS+one72012e48558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For quality assessment of DNA extracts and standardization of bacterial titers, qPCR with primers targeting host 28S rRNA was conducted simultaneously (D3A _F: 5′-GACCCGTCTTGAAACACGGA-3′; and D3B_R: 5′-TCGGAAGGAACCAGCTACTA-3′) (<span class="dropBlock reference-citations"><a id="body-ref-optA1Pb6vGgHv-3" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optA1Pb6vGgHv-3" aria-controls="optA1Pb6vGgHv-3">Litvaitis and Rohde, 1999</a><div class="dropBlock__holder" data-db-target-of="optA1Pb6vGgHv-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Litvaitis, M.K. ∙ Rohde, K.</div><div><strong>A molecular test of platyhelminth phylogeny: inferences from partial 28S rDNA sequences</strong></div><div><em>Invertebrate Biology.</em> 1999; <strong>118</strong>:42-56</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.K.LitvaitisK.RohdeA+molecular+test+of+platyhelminth+phylogeny%3A+inferences+from+partial+28S+rDNA+sequencesInvertebrate+Biology11819994256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). qPCR was performed on an Applied Biosystems 7300 Real Time PCR System in final reaction volumes of 25&nbsp;μl, including the following components: 1&nbsp;μl of DNA template, 2.5&nbsp;μl of each primer (10&nbsp;μM), 12.5&nbsp;μl of autoclaved distilled H<sub>2</sub>O, and 6.5&nbsp;μl of Luna® Universal qPCR Master Mix (New England BioLabs). PCR conditions were: 95°C for 5&nbsp;minutes, 40 cycles at 95°C for 10 s, 70°C for 15 s, and 72°C for 10 s. A melting curve analysis was performed by increasing the temperature from 60°C to 95°C within 20&nbsp;min. Standard curves were established for host 28S and bacterial 16S using PCR product as templates. A Qubit fluorometer (Thermo Fisher) was used to measure DNA concentrations to calibrate standard curves. The ratio between absolute copy numbers of bacterial 16S and host 28S (&nbsp;= bacterial/host copy ratio) was used as a standardized measure of bacterial abundance per beetle sample and the difference between the antibiotic-treatment and control group beetles was assessed with a Mann-Whitney-U-test.</div></section><section id="sec-4-3-3"><h4>Biochemical tracer experiments and SIM mass spectrometry</h4><div id="p0275" role="paragraph">Late third instar larvae were fed sterilized oat/poultry-rearing pellet-powder with amoxicillin, streptomycin and tetracycline antibiotics (2.5% w/w for each) plus 25% (<i>w/w</i>) stable isotope-labeled precursors: <sup>13</sup>C<sub>6</sub>-D-glucose, <sup>13</sup>C<sub>6</sub>-tyrosine (all &gt; 99% enrichment, Sigma-Aldrich), D<sub>3</sub>-methione, D<sub>23</sub>-dodecanoid acid and <sup>13</sup>C<sub>6</sub>-4-hydroxybenzoic acid (98%–99% enrichment, Cambridge Isotope Laboratories, Inc.) as well as a control with sterile oat/poultry-rearing pellet-powder. Beetles were single-housed and glands of adults were extracted ten days after eclosion using hexane. Crude hexane extracts were analyzed with a GC-MS as described in detail below. Electron ionization mass spectra of characteristic fragment ions were monitored in single ion mode (SIM) and at 70 eV.</div></section><section id="sec-4-3-4"><h4>Artificial disarming and survival biotest against ants</h4><div id="p0280" role="paragraph">We developed a protocol to artificially disarm <i>Dalotia</i>, creating beetles that lack the tergal gland secretion. Adult beetles were collected from laboratory stock populations and placed on a CO<sub>2</sub> fly pad; after the beetles were initially anesthetized the valve was closed and beetles could recover. Subsequently, we pulsed the beetles with low doses of CO<sub>2</sub> which initiated abdomen flexing and visible release of chemicals from the tergal gland. We repeated this cycle of anesthesia, recovery, and low pulses of CO<sub>2</sub> five times. To check success of the protocol, a subgroup of CO<sub>2</sub>-treated beetles, as well as control animals form the stock population beetles were individually extracted in 70&nbsp;μL hexane for 10&nbsp;min and their glandular compounds profiled with GC-MS (see below). Control group beetle glands contained 6.3 ± 5.6&nbsp;μg (mean ± SD) of secretion, while CO<sub>2</sub>-treated animals contained 0.5 ± 1&nbsp;ng (<a id="crosref1800" href="#figs1">Figure&nbsp;S1</a>A). CO<sub>2</sub>-treatment did not affect survival of the beetles and after 72&nbsp;h all 25 beetles in both the treated and control groups were still alive. For the survival biotest, ten <i>Dalotia</i> beetles were paired with five <i>Liometopum occidentale</i> Emery ants (collected: Chaney Canyon, Altadena, CA; 34°13’04.1”N 118°09’06.4”W). Beetles and ants were placed in a 100x100x50mm plastic box with 10mm Plaster of Paris and two rolled pieces of Kimwipe to create an artificial foraging space (<span class="dropBlock reference-citations"><a id="body-ref-sref149" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149" aria-controls="sref149">Vucic-Pestic et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref149"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Vucic-Pestic, O. ∙ Birkhofer, K. ∙ Rall, B.C. ...</div><div><strong>Habitat structure and prey aggregation determine the functional response in a soil predator–prey interaction</strong></div><div><em>Pedobiologia (Jena).</em> 2010; <strong>53</strong>:307-312</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.Vucic-PesticK.BirkhoferB.C.RallS.ScheuU.BroseHabitat+structure+and+prey+aggregation+determine+the+functional+response+in+a+soil+predator%E2%80%93prey+interactionPedobiologia+%28Jena%29532010307312" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In total, we prepared 14 boxes each of control and disarmed beetles (&nbsp;= 140 beetles and 70 ants per treatment). The experiment ran for 48&nbsp;h and the percentage of surviving beetles calculated for each box. Difference in survival between control and disarmed groups was analyzed using a Mann-Whitney-U-test.</div></section><section id="sec-4-3-5"><h4>Double-stranded RNA preparation and RNAi knockdown in Dalotia</h4><div id="p0285" role="paragraph">Double-stranded RNA was prepared from cDNA from pooled from all life-stages of the beetle. Regions of 450-650&nbsp;bp for locus were amplified using primers with T7 linkers (5′- TAATACGACTCACTATAGGG-3′). Fragments were cloned into a pCR4-TOPO Vector (TOPO TA Cloning Kit, ThermoFisher) using primers listed in the Key Resource Table.</div><div id="p0290" role="paragraph">The same primers were subsequently used to amplify template DNA from the TOPO vector for dsRNA synthesis, using the MEGAscript T7 Transcription Kit (ThermoFisher). After synthesis, dsRNA was cleaned using acid phenol/chloroform (50:50) and adjusted to a concentration of ∼6&nbsp;mg/ml. For injection, dsRNA stock was then diluted 1:1 in DEPC-treated 1x PBS and blue food dye following a previously published protocol (<span class="dropBlock reference-citations"><a id="body-ref-opt0vPPm0qdrV" href="#" role="doc-biblioref" data-xml-rid="bib166" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opt0vPPm0qdrV" aria-controls="opt0vPPm0qdrV">Philip and Tomoyasu, 2012</a><div class="dropBlock__holder" data-db-target-of="opt0vPPm0qdrV"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Philip, B.N. ∙ Tomoyasu, Y.</div><div><strong>Gene knockdown analysis by double-stranded RNA injection</strong></div><div><strong><i>Molecular methods for evolutionary genetics</i></strong></div><div>Humana Press, 2012; 471-497</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.N.PhilipY.TomoyasuGene+knockdown+analysis+by+double-stranded+RNA+injectionMolecular+methods+for+evolutionary+genetics2012Humana+Press471497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Following the same protocol, dsRNA against green fluorescent protein (<i>GFP</i>) was prepared and injected as a control. Late third instar larvae were collected from laboratory stock populations and microinjection was performed according to <span class="dropBlock reference-citations"><a id="body-ref-sref109-4" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-4" aria-controls="sref109-4">Parker et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref109-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Parker, J. ∙ Eldredge, K.T. ∙ Thomas, I. ...</div><div><strong>Hox-logic of body plan innovations for social insect symbiosis in rove beetles</strong></div><div><em>bioRxiv.</em> 2018; 198945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerK.T.EldredgeI.ThomasR.ColemanS.DavisHox-logic+of+body+plan+innovations+for+social+insect+symbiosis+in+rove+beetlesbioRxiv2018198945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. After injection, the larvae were individually placed into 5&nbsp;cm plastic Petri dishes with thick moistened filter paper. Larvae that died before pupation or did not pupate by the end of ten days were discarded. The filter paper was moistened three times per week. After adults had eclosed, beetles were additionally fed with frozen fruit flies and oat/poultry-rearing pellet-powder on the same schedule for ten days. Mature beetles were used for experiments chemical assays.</div></section><section id="sec-4-3-6"><h4>Extraction of gland compounds and gas chromatography/mass spectrometry</h4><div id="p0295" role="paragraph">To screen chemical phenotypes of dsRNA knockdown and dsGFP control beetles, single specimens were submersed (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Dettner, 1984</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Dettner, K.</div><div><strong>Isopropylesters as Wetting Agents From the Defensive Secretion of the Rove Beetle Coprophilus striatulus F. (Coleoptera, Staphylinidae)</strong></div><div><em>Insect Biochem.</em> 1984; <strong>14</strong>:383-390</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerIsopropylesters+as+Wetting+Agents+From+the+Defensive+Secretion+of+the+Rove+Beetle+Coprophilus+striatulus+F.+%28Coleoptera%2C+Staphylinidae%29Insect+Biochem.141984383390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">1993b</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Dettner, K.</div><div><strong>Dabbing and shooting of benzo- and naphthoquinone secretions: Defensive strategies of bark-inhabiting aleocharine (col.: Staphylinidae) and tenebrionid (col.: Tenebrionidae) beetle larvae</strong></div><div><em>J.&nbsp;Chem. Ecol.</em> 1993; <strong>19</strong>:1337-1354</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerDabbing+and+shooting+of+benzo-+and+naphthoquinone+secretions%3A+Defensive+strategies+of+bark-inhabiting+aleocharine+%28col.%3A+Staphylinidae%29+and+tenebrionid+%28col.%3A+Tenebrionidae%29+beetle+larvaeJ.%C2%A0Chem.+Ecol.19199313371354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref141-8" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-8" aria-controls="sref141-8">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in 70&nbsp;μL hexane (company, GC/MS analytical grade) containing 150&nbsp;ng/μl n-octadecane (Sigma Aldrich) as an internal standard; after 10&nbsp;min the solvent was separated from the insect, transferred into a new vial and frozen at −80°C for further analysis. A GCMSQP2020 gas chromatography/mass-spectrometry system (Shimadzu, Kyōto, Japan) equipped with a ZB-5MS fused silica capillary column (30&nbsp;m x 0.25&nbsp;mm ID, df&nbsp;= 0.25&nbsp;μm) from Phenomenex (Torrance, CA, USA) was used for chemical profiling. Crude hexane sample aliquots (2&nbsp;μl) were injected by using an AOC-20i autosampler system (Shimadzu) into a split/splitless-injector which operated in splitless-mode at a temperature of 310°C. Helium was used as the carrier-gas with a constant flow rate of 2.13&nbsp;ml/min. The chromatographic conditions were as follows: The column temperature at the start was 40°C with a 1-minute hold after which the temperature was initially increased 30°C/min to 250°C and further increased 50°C/min to a final temperature of 320°C and held for 5&nbsp;minutes. Electron impact ionization spectra were recorded at 70 eV ion source voltage, with a scan rate of 0.2 scans/sec from m/z 40 to 450. The ion source of the mass spectrometer and the transfer line were kept at 230°C and 320°C, respectively.</div><div id="p0300" role="paragraph">Compounds were identified based on their <i>m/z</i> fragmentation patters and by comparison to authentic standards (1,4-benzoquinone, 2-methyl-1,4-benzoquinone, undecane, isopropyl decanoate, ethyl decanoate, ethyl dodecanoate; all SigmaAldrich; 2-methoxy-3-methyl-1,4-benzoquninone was synthesized as outlined below). Additionally, compound identity was confirmed by comparison of retention indices and MS data with published literature (<span class="dropBlock reference-citations"><a id="body-ref-sref37-3" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-3" aria-controls="sref37-3">Dettner, 1984</a><div class="dropBlock__holder" data-db-target-of="sref37-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Dettner, K.</div><div><strong>Isopropylesters as Wetting Agents From the Defensive Secretion of the Rove Beetle Coprophilus striatulus F. (Coleoptera, Staphylinidae)</strong></div><div><em>Insect Biochem.</em> 1984; <strong>14</strong>:383-390</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerIsopropylesters+as+Wetting+Agents+From+the+Defensive+Secretion+of+the+Rove+Beetle+Coprophilus+striatulus+F.+%28Coleoptera%2C+Staphylinidae%29Insect+Biochem.141984383390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Dettner et&nbsp;al., 1985</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Dettner, K. ∙ Schwinger, G. ∙ Wunderle, P.</div><div><strong>Sticky secretion from two pairs of defensive glands of rove beetleDeleaster dichrous (Grav.) (Coleoptera: Staphylinidae)</strong></div><div><em>J.&nbsp;Chem. Ecol.</em> 1985; <strong>11</strong>:859-883</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerG.SchwingerP.WunderleSticky+secretion+from+two+pairs+of+defensive+glands+of+rove+beetleDeleaster+dichrous+%28Grav.%29+%28Coleoptera%3A+Staphylinidae%29J.%C2%A0Chem.+Ecol.111985859883" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref141-9" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-9" aria-controls="sref141-9">Steidle and Dettner, 1993</a><div class="dropBlock__holder" data-db-target-of="sref141-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The authentic standards were used to construct four-point calibrations curves for external standardization and quantification of BQs, esters and alkanes. Semi-quantification of bulk cuticular hydrocarbons (CHCs) was based on the internal C18 standard. We quantified the ion abundance and the absolute amounts of compounds in ng based on the internal or external standard (see above), as well as the relative composition of individual CHCs compared to ion abundance of the other CHCs. For quantifying gland compounds following RNAi silencing, we compared the target compound amounts between dsRNA knockdown and dsGFP control beetles with a Kruskal-Wallis test or Mann-Whitney-U-tests. Compositional data of CHCs was compared by non-metric multidimensional scaling (NMDS) ordination of a Bray-Curtis similarity matrix and analyzed using PERMANOVA (<span class="dropBlock reference-citations"><a id="body-ref-sref3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3" aria-controls="sref3">Anderson, 2001</a><div class="dropBlock__holder" data-db-target-of="sref3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Anderson, M.J.</div><div><strong>A new method for non-parametric multivariate analysis of variance</strong></div><div><em>Austral Ecol.</em> 2001; <strong>26</strong>:32-46</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.J.AndersonA+new+method+for+non-parametric+multivariate+analysis+of+varianceAustral+Ecol.2620013246" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Brückner and Heethoff, 2017</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Brückner, A. ∙ Heethoff, M.</div><div><strong>A chemo-ecologists’ practical guide to compositional data analysis</strong></div><div><em>Chemoecology.</em> 2017; <strong>27</strong>:33-46</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerM.HeethoffA+chemo-ecologists%E2%80%99+practical+guide+to+compositional+data+analysisChemoecology2720173346" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). GC-MS data related to RNAi can be found at CaltechData (<a id="intref0490" href="https://data.caltech.edu/records/1917" target="_blank">https://data.caltech.edu/records/1917</a>).</div></section><section id="sec-4-3-7"><h4><i>In situ</i> hybridization chain reaction (HCR)</h4><div id="p0305" role="paragraph">For <i>in situ</i> HCR, DNA probe sets, DNA HCR amplifier, HCR hairpins as well as hybridization, wash and amplification buffers were purchased from Molecular Instruments (Beckman Institute at Caltech; <a id="intref0495" href="https://www.molecularinstruments.com/" target="_blank">https://www.molecularinstruments.com/</a>) for each target transcript (<a id="crosref1855" href="#sec-4-1">Key Resources Table</a>). For dissections, adult beetles were immersed in ice-cold DEPC-treated PBS and their abdomens were removed. Ventral segments were removed with dissection scissors. Dorsal abdominal segments were fixed in 4% PFA in DEPC-treated PBST for 25&nbsp;minutes at room temperature and subsequently washed with DEPC-treated PBST (3x5&nbsp;min). Fixation was followed by a 5&nbsp;min proteinase K (10&nbsp;μg/mL) treatment at room temperature, and samples were then rinsed in DEPC-treated PBST (3x). Samples were postfixed in 4% PBST-PFA for 25&nbsp;minutes at room temperature and washed again with DEPC-treated PBST (3x5&nbsp;min). The amplification and detection stages followed published protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Choi et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Choi, H.M.T. ∙ Schwarzkopf, M. ∙ Fornace, M.E. ...</div><div><strong>Third-generation <i>in situ</i> hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust</strong></div><div><em>Development.</em> 2018; <strong>145</strong>:dev165753</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.M.T.ChoiM.SchwarzkopfM.E.FornaceA.AcharyaG.ArtavanisJ.StegmaierA.CunhaN.A.PierceThird-generation+in+situ+hybridization+chain+reaction%3A+multiplexed%2C+quantitative%2C+sensitive%2C+versatile%2C+robustDevelopment1452018dev165753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Probes were either initiated with B1-Alexa546 or B4-Alexa488 amplifiers. After amplification and before the final wash steps, DAPI or Hoechst 33342 (1:2000) to&nbsp;mark nuclei, Alexa 488- or Alexa-647-Wheat Germ Agglutinin Conjugate (WGA; 1:200) to label cell membranes and in selected samples BODIPY 493/503 (1:200) to stain for neutral lipids were added. Tissue samples were imaged as whole mounts of dorsal abdomens in ProLong Gold Antifade Mountant (ThermoFisher), using a Zeiss LSM 880 with Airyscan fast.</div></section><section id="sec-4-3-8"><h4>Fly toxicity biotest</h4><div id="p0310" role="paragraph">To test the toxicity of <i>Dalotia</i> tergal gland compounds, we followed an approach by <span class="dropBlock reference-citations"><a id="body-ref-sref37-4" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-4" aria-controls="sref37-4">Dettner (1984)</a><div class="dropBlock__holder" data-db-target-of="sref37-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Dettner, K.</div><div><strong>Isopropylesters as Wetting Agents From the Defensive Secretion of the Rove Beetle Coprophilus striatulus F. (Coleoptera, Staphylinidae)</strong></div><div><em>Insect Biochem.</em> 1984; <strong>14</strong>:383-390</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerIsopropylesters+as+Wetting+Agents+From+the+Defensive+Secretion+of+the+Rove+Beetle+Coprophilus+striatulus+F.+%28Coleoptera%2C+Staphylinidae%29Insect+Biochem.141984383390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. First, artificial test solutions were prepared using the main gland constituents mimicking the natural ratios. An all-compound mixture (∼100&nbsp;μl) was prepared by mixing 8&nbsp;mg 1,4- benzoquinone, 28&nbsp;mg 2-methyl-1,4-benzoquinone, 47&nbsp;mg undecane, 7&nbsp;mg isopropyl decanoate, 13&nbsp;mg ethyl decanoate and 7&nbsp;mg ethyl dodecanoate (all Sigma Aldrich). Subsequently, mixtures were prepared without certain compounds to create the following treatments: undecane+BQs, esters+BQs, all BQs, undecane+esters, undecane, esters; 1x PBS was used as control. For the survival tests a group of 25 <i>Drosophila melanogaster</i> third instar wandering larvae were immersed for 1&nbsp;s in 1&nbsp;mL of artificial secretion or dipped into solid BQ powder and subsequently moved into a fresh culture tube. This protocol was repeated five times for each of the eight different treatments. Numbers of surviving fly larvae were counted after 1 h. For statistical analysis we used a generalized linear model (GLM) with binomial error distribution and logit as link-function in R. We fitted ‘fly survival’ as a binomial response variable of surviving fly larvae and total number of fly larvae (25 individuals) in each replicate. Finally, we used the R-implemented ‘Simultaneous Tests for General Linear Hypotheses’ with Tukey contrast for pairwise comparisons.</div></section><section id="sec-4-3-9"><h4>Antimicrobial assay</h4><div id="p0315" role="paragraph">To assay the antimicrobial properties of the <i>Dalotia</i> tergal gland secretion, we performed bacterial growth assays in a standard 96-well plate design. Artificial test solutions that mimicked natural ratios of compounds, as well and individual and pairs of compound classes were prepared as above (Fly toxicity biotest) and diluted in LB broth to a concentration of ∼6&nbsp;μg/μl. 1x PBS was used as a control. Cultures of <i>Pseudomonas fluorescens</i> wild-type (strain WCS365) (<span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Jorth et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Jorth, P. ∙ Spero, M.A. ∙ Livingston, J. ...</div><div><strong>Quantitative Visualization of Gene Expression in Mucoid and Nonmucoid Pseudomonas aeruginosa Aggregates Reveals Localized Peak Expression of Alginate in the Hypoxic Zone</strong></div><div><em>MBio.</em> 2019; <strong>10</strong>:e02622</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.JorthM.A.SperoJ.LivingstonD.K.NewmanQuantitative+Visualization+of+Gene+Expression+in+Mucoid+and+Nonmucoid+Pseudomonas+aeruginosa+Aggregates+Reveals+Localized+Peak+Expression+of+Alginate+in+the+Hypoxic+ZoneMBio102019e02622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were grown on LB agar plates overnight at 30°C. Subsequently, ∼5&nbsp;μl of the bacterial colony was transferred into 500&nbsp;μl and the OD<sub>500</sub> was measured on a NanoDrop Spectrometer (Thermo Fisher). Based on the OD reading, the cell suspension was diluted to OD<sub>500</sub>&nbsp;= 0.1. For the growth assay, 15&nbsp;μl of the cell suspension was inoculated with 134&nbsp;μl LB and 1&nbsp;μl of test secretion, and overlayed with 50&nbsp;μl sterile mineral oil. Growth curves were recorded over 24h every 10&nbsp;min on a Biotek Synergy 4 running in continuous shaking mode at 30°C. Each treatment and the negative PBS control were replicated 13 times across two plates.</div><div id="p0320" role="paragraph">The OD<sub>500</sub> values were plotted over time, and we modeled the bacterial growth with a Bayesian regression using a normal likelihood with mean given by a logistic function:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper O upper D 500 left parenthesis t right parenthesis equals StartFraction upper A Over 1 plus e Superscript StartFraction 4 normal mu Super Subscript m Superscript Over upper A EndFraction left parenthesis lamda minus t right parenthesis plus 2 Baseline EndFraction" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(41 (40 0 39 (38 (3 1 2) 37 (7 4 5 6))) 8 (36 9 (35 10 11 (34 12 (33 (32 (20 (18 13 17 (16 14 15)) 19) 31 (28 22 (26 23 24 25) 27)) 29 30)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="41" data-semantic-children="40,36" data-semantic-content="8" data-semantic-owns="40 8 36" data-semantic-speech="upper O upper D 500 left parenthesis t right parenthesis equals StartFraction upper A Over 1 plus e Superscript StartFraction 4 normal mu Super Subscript m Superscript Over upper A EndFraction left parenthesis lamda minus t right parenthesis plus 2 Baseline EndFraction"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="40" data-semantic-children="0,38" data-semantic-content="39" data-semantic-parent="41" data-semantic-owns="0 39 38"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="40"><mjx-c class="mjx-c1D442">𝑂</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="39" data-semantic-parent="40" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-children="3,7" data-semantic-content="37,1" data-semantic-parent="40" data-semantic-owns="3 37 7"><mjx-msub data-semantic-type="subscript" data-semantic-role="simple function" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="38" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="appl"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.666em;">5</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn></mjx-script></mjx-msub><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="37" data-semantic-parent="38" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="7" data-semantic-children="5" data-semantic-content="4,6" data-semantic-parent="38" data-semantic-owns="4 5 6"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="41" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="36" data-semantic-children="9,35" data-semantic-parent="41" data-semantic-owns="9 35"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mtext size="s" data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-parent="36"><mjx-c class="mjx-c41">A</mjx-c></mjx-mtext></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="35" data-semantic-children="10,34" data-semantic-content="11" data-semantic-parent="36" data-semantic-owns="10 11 34"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="35"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="11" data-semantic-parent="35" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="34" data-semantic-children="12,33" data-semantic-parent="35" data-semantic-owns="12 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="34"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-script style="vertical-align: 0.504em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="33" data-semantic-children="32,30" data-semantic-content="29" data-semantic-parent="34" data-semantic-owns="32 29 30"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="32" data-semantic-children="20,28" data-semantic-content="31" data-semantic-parent="33" data-semantic-owns="20 31 28"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="20" data-semantic-children="18,19" data-semantic-parent="32" data-semantic-owns="18 19"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="13,16" data-semantic-content="17" data-semantic-parent="20" data-semantic-owns="13 17 16"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="18"><mjx-c class="mjx-c34">4</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="18" data-semantic-owns="14 15"><mjx-mo data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c3BC">μ</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="20"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="28" data-semantic-children="26" data-semantic-content="22,27" data-semantic-parent="32" data-semantic-owns="22 26 27"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="22" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-children="23,25" data-semantic-content="24" data-semantic-parent="28" data-semantic-owns="23 24 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="26"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="27" data-semantic-parent="28" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="29" data-semantic-parent="33" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="33"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math></mjx-container></div></div></div>Where λ describes the lag time (in h) before bacterial growth takes off, μ<sub>m</sub> is the maximal growth rate, t is the time since inoculation and A is the asymptote of the curve after growth levels off. We sampled these parameters for each growth event using the Stan implementation of Hamiltonian Monte Carlo and took the median sample value for each parameter for a given growth curve. A detailed script can be found on CaltechData (<a id="intref0500" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). We then compared the lag time λ and the OD<sub>500</sub>at the endpoint data after 24h for each treatment using a Welch’s ANOVA and one-way ANOVA, respectively, followed by a Tukey’s HDS pairwise posthoc test.</div></section><section id="sec-4-3-10"><h4>Ant selection experiment</h4><div id="p0325" role="paragraph">To test for an adaptive role of the BQ and solvent cells in chemical defense, we generated beetles with either BQ-free or solvent-free gland secretions by knocking down <i>dmd</i> or <i>MFASN</i>, respectively, by injecting dsRNA (∼3&nbsp;mg/ml) against these targets. As a control treatment, we injected dsRNA against <i>GFP</i> (∼3&nbsp;mg/ml). Due to some mortality post-injection, we injected ∼400-500 larvae for each dsRNA target. The detailed RNAi protocol, primer sequences and beetle care were the same as those used for pathway characterization (see below). 10-day old adults were used to set up the following survival assays: like previously, ten beetles were paired with five ants (collected: Chaney Canyon, Altadena, CA; 34°13’04.1”N 118°09’06.4”W) in a 100x15&nbsp;mm plastic Petri dish with a thin layer of Plaster of Paris and two rolled pieces of Kimwipe. In total, we prepared 12 dishes per treatment (&nbsp;= 120 beetles and 60 ants per treatment), running the assay for 48&nbsp;h before counted the number of surviving beetles. Because the amount of secretion per beetle cannot be determined before the experiment, we collected and extracted every beetle, regardless if dead or alive, in 30&nbsp;μl of a mixture of 1:1 (v/v) hexane:chloroform for 15&nbsp;min. We then ran a total of 360 beetle samples on the GC-MS using the same chromatographic conditions as outlined below. To determine if RNAi was successful in each individual beetle, we used single-ion detection of characteristic ions of 2-methyl-1,4-benzoquione (<i>m/z</i>&nbsp;= 122) and undecane (<i>m/z</i>&nbsp;= 156) to determine whether the glands of extracted beetles were benzoquinone-free (<i>dmd</i>-RNAi), solvent-free (<i>MFASN</i> RNAi) or not influenced (<i>GFP</i>). Only beetles with the correct gland composition were included in the final dataset; beetles with completely empty glands at the end of the experiment were removed from the dataset because it is possible they possessed empty glands at the start of the experiment. For statistical analysis of survival differences among BQ-free, solvent-free and control beetles, we again used a GLM with binomial error distribution and logit as link-function in R. We fitted ‘beetle survival’ as a binomial response variable of counted surviving beetles with a non-empty gland and total number of beetles (dead+alive) with a non-empty gland to account for the different sample sizes in each replicate. Finally, we used the R-implemented ‘Simultaneous Tests for General Linear Hypotheses’ with Tukey contrast for pairwise comparisons. The R script for this experiment can be found at CaltechData (<a id="intref0505" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). As a control to assess survival without ants, we injected 50 larvae for each construct (<i>MFASN</i>, <i>dmd</i> and <i>GFP</i>), raised them until maturity and recorded their survival for a period of twelve consecutive days.</div></section><section id="sec-4-3-11"><h4>Bulk RNaseq and transcriptome assembly</h4><div id="p0330" role="paragraph">For bulk RNA sequencing (3 biological and 3 technical replicates), a total of 105 gland (Segment A7) and control (Segment A6) segments from male beetles were dissected and separately collected for total RNA extraction with Trizol. Library preparation followed the Illumina TruSeq mRNA stranded kit protocol, including chemical shearing to obtain an average final library size of ∼300&nbsp;bp. Illumina 100&nbsp;bp paired-end sequencing was performed on a HiSeq2500 platform in which the twelve <i>Dalotia</i> gland libraries were multiplexed on the same lane. For genome-guided assembly of the transcriptome, a bam-file was created from the <i>Dalotia coriaria</i> reference genome assembly: (<a id="intref0510" href="http://131.215.78.39/genomebrowser/cgi-bin/hgGateway?genome=Dcor2&amp;hubUrl=http://131.215.78.39/genomebrowser/evolution/parkergroup/Dalotia_coriaria/hub2.txt" target="_blank">http://131.215.78.39/genomebrowser/cgi-bin/hgGateway?genome=Dcor2&amp;hubUrl=http://131.215.78.39/genomebrowser/evolution/parkergroup/Dalotia_coriaria/hub2.txt</a>) using STAR (<span class="dropBlock reference-citations"><a id="body-ref-optMaXjtQ2q4i-2" href="#" role="doc-biblioref" data-xml-rid="bib167" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="optMaXjtQ2q4i-2" aria-controls="optMaXjtQ2q4i-2">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="optMaXjtQ2q4i-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.DobinC.A.DavisF.SchlesingerJ.DrenkowC.ZaleskiS.JhaP.BatutM.ChaissonT.R.GingerasSTAR%3A+ultrafast+universal+RNA-seq+alignerBioinformatics2920131521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), while the approximately 240 million RNaseq reads were <i>in silico</i> normalized and subsequently used together with the bam-file to assemble transcripts using Trinity v2.8.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Grabherr et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Grabherr, M.G. ∙ Haas, B.J. ∙ Yassour, M. ...</div><div><strong>Full-length transcriptome assembly from RNA-Seq data without a reference genome</strong></div><div><em>Nat. Biotechnol.</em> 2011; <strong>29</strong>:644-652</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.G.GrabherrB.J.HaasM.YassourJ.Z.LevinD.A.ThompsonI.AmitX.AdiconisL.FanR.RaychowdhuryQ.ZengFull-length+transcriptome+assembly+from+RNA-Seq+data+without+a+reference+genomeNat.+Biotechnol.292011644652" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, transcript sequences were clustered and concatenated to remove duplications using CD-HIT (<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Fu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Fu, L. ∙ Niu, B. ∙ Zhu, Z. ...</div><div><strong>CD-HIT: accelerated for clustering the next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:3150-3152</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FuB.NiuZ.ZhuS.WuW.LiCD-HIT%3A+accelerated+for+clustering+the+next-generation+sequencing+dataBioinformatics28201231503152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with a sequence identity threshold of 0.95, yielding an assembly with a total length of 61.1 Mbp, an N50&nbsp;= 3097&nbsp;bp and a BUSCO score (<span class="dropBlock reference-citations"><a id="body-ref-sref132-2" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-2" aria-controls="sref132-2">Simão et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref132-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Simão, F.A. ∙ Waterhouse, R.M. ∙ Ioannidis, P. ...</div><div><strong>BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3210-3212</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.A.Sim%C3%A3oR.M.WaterhouseP.IoannidisE.V.KriventsevaE.M.ZdobnovBUSCO%3A+assessing+genome+assembly+and+annotation+completeness+with+single-copy+orthologsBioinformatics31201532103212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of C:97.6%[S:67.5%,D:30.4%],F:0.5%,M:1.6%. Annotations of transcripts were transferred from the respective gene models from the genome and additionally confirmed by blastx (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Altschul et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Altschul, S.F. ∙ Gish, W. ∙ Miller, W. ...</div><div><strong>Basic local alignment search tool</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1990; <strong>215</strong>:403-410</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.AltschulW.GishW.MillerE.W.MyersD.J.LipmanBasic+local+alignment+search+toolJ.%C2%A0Mol.+Biol.2151990403410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of all transcripts against the non-redundant NCBI protein database (<span class="dropBlock reference-citations"><a id="body-ref-sref117-1" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-1" aria-controls="sref117-1">Pruitt et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref117-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Pruitt, K.D. ∙ Tatusova, T. ∙ Maglott, D.R.</div><div><strong>NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D61-D65</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.D.PruittT.TatusovaD.R.MaglottNCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteinsNucleic+Acids+Res.352007D61D65" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Finally, the bulk-transcriptome was used to build a kallisto (<span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.L.BrayH.PimentelP.MelstedL.PachterNear-optimal+probabilistic+RNA-seq+quantificationNat.+Biotechnol.342016525527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) index file for pseudoalignment of single-cell RNaseq reads and cell-type specific, ultra-low input RNaseq (SMART-Seq). The assembled transcriptome can be found at CaltechData (<a id="intref0515" href="https://data.caltech.edu/records/1914" target="_blank">https://data.caltech.edu/records/1914</a>).</div></section><section id="sec-4-3-12"><h4>Cell-type specific transcriptome sequencing (SMART-seq)</h4><div id="p0335" role="paragraph">For microdissection of specific gland cell types, male <i>Dalotia</i> beetles were injected with Invitrogen Wheat Germ Agglutinin, Alexa Fluor 488 Conjugate, to visualize the glandular structure <i>in&nbsp;vivo</i>, using a microinjector. Under fluorescence microscopy, we dissected the BQ cells in groups of 3-7 cells, the solvent cell tissue (∼1000 cells) and entire tergite 6 (control tissue) in ice-cold DEPC-treated PBS. We used the NEBNext® Single Cell/Low Input RNA Library Prep Kit for Illumina® together with NEBNext® Multiplex Oligos for Illumina® (New England Biolab) for library preparation, including cell dissociation, reverse transcription of poly(A) RNA, amplification full-length cDNA, fragmentation, ligation and final library amplification according to the manufacturer’s protocol. We performed cDNA amplification for 18-20 PCR cycles and final library amplification for 8 PCR cycles. In total, we constructed 31 libraries (ten D1, ten D2 and eleven control). The quality and concentration of the resulting libraries were assessed using the Qubit High Sensitivity dsDNA kit (Thermo Scientific) and Agilent Bioanalyzer High Sensitivity DNA assay. Libraries were sequenced on an Illumina HiSeq2500 platform (single-end with read lengths of 50 nt) with ∼20-25 million reads per library. Illumina sequencing reads were pseudoaligned to the bulk transcriptome and quantified (100 bootstrap samples) with kallisto 0.46.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref21-3" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-3" aria-controls="sref21-3">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref21-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.L.BrayH.PimentelP.MelstedL.PachterNear-optimal+probabilistic+RNA-seq+quantificationNat.+Biotechnol.342016525527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-13"><h4>Cell-type specific transcriptome analysis</h4><div id="p0340" role="paragraph">For cell-type specific differential expression analysis, kallisto-quantified RNaseq data were processed with sleuth 0.30.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref112-2" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-2" aria-controls="sref112-2">Pimentel et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref112-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Pimentel, H. ∙ Bray, N.L. ∙ Puente, S. ...</div><div><strong>Differential analysis of RNA-seq incorporating quantification uncertainty</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:687-690</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.PimentelN.L.BrayS.PuenteP.MelstedL.PachterDifferential+analysis+of+RNA-seq+incorporating+quantification+uncertaintyNat.+Methods142017687690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using Likelihood Ratio tests in R 3.6.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref123-2" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-2" aria-controls="sref123-2">R Core Team, 2018</a><div class="dropBlock__holder" data-db-target-of="sref123-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing. R Foundation for Statistical Computing</strong></div><div>2018</div><div><a id="intref0650" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing.+R+Foundation+for+Statistical+Computing2018https%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To assign either up or downregulation for each transcript we extracted the normalized transcripts per million (tpm) from gland and control samples of sleuth object and defined up and downregulation as:<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="down negative regulated colon StartStartFraction StartFraction 1 Over n Subscript g Baseline EndFraction sigma summation Underscript i equals 1 Overscript n Subscript g Baseline Endscripts x Subscript i Baseline OverOver StartFraction 1 Over n Subscript c Baseline EndFraction sigma summation Underscript j equals 1 Overscript n Subscript c Baseline Endscripts x Subscript j Baseline EndEndFraction less than 1 semicolon u p minus r e g u l a t e d colon StartStartFraction StartFraction 1 Over n Subscript g Baseline EndFraction sigma summation Underscript i equals 1 Overscript n Subscript g Baseline Endscripts x Subscript i Baseline OverOver StartFraction 1 Over n Subscript c Baseline EndFraction sigma summation Underscript j equals 1 Overscript n Subscript c Baseline Endscripts x Subscript j Baseline EndEndFraction greater than 1" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(128 (113 0 (111 1 2)) 3 (114 (47 (25 (9 5 (8 6 7)) 24 (23 (18 10 (14 11 12 13) (17 15 16)) (21 19 20))) (46 (30 26 (29 27 28)) 45 (44 (39 31 (35 32 33 34) (38 36 37)) (42 40 41)))) 48 49) 50 (126 (116 51 115 52) 53 (125 55 117 56 118 57 119 58 120 59 121 60 122 61 123 62 124 63)) 64 (127 (108 (86 (70 66 (69 67 68)) 85 (84 (79 71 (75 72 73 74) (78 76 77)) (82 80 81))) (107 (91 87 (90 88 89)) 106 (105 (100 92 (96 93 94 95) (99 97 98)) (103 101 102)))) 109 110))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="128" data-semantic-children="113,3,114,50,126,64,127" data-semantic-content="3,50,64" data-semantic-owns="113 3 114 50 126 64 127" data-semantic-speech="down negative regulated colon StartStartFraction StartFraction 1 Over n Subscript g Baseline EndFraction sigma summation Underscript i equals 1 Overscript n Subscript g Baseline Endscripts x Subscript i Baseline OverOver StartFraction 1 Over n Subscript c Baseline EndFraction sigma summation Underscript j equals 1 Overscript n Subscript c Baseline Endscripts x Subscript j Baseline EndEndFraction less than 1 semicolon u p minus r e g u l a t e d colon StartStartFraction StartFraction 1 Over n Subscript g Baseline EndFraction sigma summation Underscript i equals 1 Overscript n Subscript g Baseline Endscripts x Subscript i Baseline OverOver StartFraction 1 Over n Subscript c Baseline EndFraction sigma summation Underscript j equals 1 Overscript n Subscript c Baseline Endscripts x Subscript j Baseline EndEndFraction greater than 1"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="113" data-semantic-children="0,111" data-semantic-parent="128" data-semantic-collapsed="(113 (c 112) 0 111)" data-semantic-owns="0 111"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="113"><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c77" noic="true" style="padding-top: 0.694em;">w</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.694em;">n</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="111" data-semantic-children="2" data-semantic-content="1" data-semantic-parent="113" data-semantic-owns="1 2"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="111" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="identifier" data-semantic-role="unit" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="2" data-semantic-parent="111"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.694em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c67" noic="true" style="padding-top: 0.694em;">g</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c64" style="padding-top: 0.694em;">d</mjx-c></mjx-mtext></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="3" data-semantic-parent="128" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="114" data-semantic-children="47,49" data-semantic-content="48" data-semantic-parent="128" data-semantic-owns="47 48 49"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="47" data-semantic-children="25,46" data-semantic-parent="114" data-semantic-owns="25 46"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="25" data-semantic-children="9,23" data-semantic-content="24" data-semantic-parent="47" data-semantic-owns="9 24 23"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="9" data-semantic-children="5,8" data-semantic-parent="25" data-semantic-owns="5 8"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="23" data-semantic-children="18,21" data-semantic-content="10" data-semantic-parent="25" data-semantic-owns="18 21"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="18" data-semantic-children="10,14,17" data-semantic-parent="23" data-semantic-owns="10 14 17"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="10" data-semantic-parent="18" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="18" data-semantic-owns="15 16"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="17"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="11,13" data-semantic-content="12" data-semantic-parent="18" data-semantic-owns="11 12 13"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="12" data-semantic-parent="14" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="23" data-semantic-owns="19 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="46" data-semantic-children="30,44" data-semantic-content="45" data-semantic-parent="47" data-semantic-owns="30 45 44"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="30" data-semantic-children="26,29" data-semantic-parent="46" data-semantic-owns="26 29"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="30"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="29" data-semantic-children="27,28" data-semantic-parent="30" data-semantic-owns="27 28"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="29"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="29"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="44" data-semantic-children="39,42" data-semantic-content="31" data-semantic-parent="46" data-semantic-owns="39 42"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="39" data-semantic-children="31,35,38" data-semantic-parent="44" data-semantic-owns="31 35 38"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="31" data-semantic-parent="39" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.288em; margin-left: 0px;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="38" data-semantic-children="36,37" data-semantic-parent="39" data-semantic-owns="36 37"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="38"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="35" data-semantic-children="32,34" data-semantic-content="33" data-semantic-parent="39" data-semantic-owns="32 33 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="35"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="33" data-semantic-parent="35" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="34" data-semantic-parent="35"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="42" data-semantic-children="40,41" data-semantic-parent="44" data-semantic-owns="40 41"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="42"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="42"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="48" data-semantic-parent="114" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="49" data-semantic-parent="114"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="semicolon" data-semantic-id="50" data-semantic-parent="128" data-semantic-operator="punctuated"><mjx-c class="mjx-c3B">;</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="126" data-semantic-children="116,125" data-semantic-content="53" data-semantic-parent="128" data-semantic-owns="116 53 125"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="116" data-semantic-children="51,52" data-semantic-content="115" data-semantic-parent="126" data-semantic-owns="51 115 52"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="51" data-semantic-parent="116"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="115" data-semantic-parent="116" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="116"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="53" data-semantic-parent="126" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="125" data-semantic-children="55,56,57,58,59,60,61,62,63" data-semantic-content="117,118,119,120,121,122,123,124" data-semantic-parent="126" data-semantic-owns="55 117 56 118 57 119 58 120 59 121 60 122 61 123 62 124 63"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="55" data-semantic-parent="125"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="117" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="56" data-semantic-parent="125"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="118" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="57" data-semantic-parent="125"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="119" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="58" data-semantic-parent="125"><mjx-c class="mjx-c1D462">𝑢</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="120" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="59" data-semantic-parent="125"><mjx-c class="mjx-c1D459">𝑙</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="121" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="60" data-semantic-parent="125"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="122" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="61" data-semantic-parent="125"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="123" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="62" data-semantic-parent="125"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="124" data-semantic-parent="125" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="63" data-semantic-parent="125"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="punctuation" data-semantic-role="colon" data-semantic-id="64" data-semantic-parent="128" data-semantic-operator="punctuated"><mjx-c class="mjx-c3A">:</mjx-c></mjx-mo><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="127" data-semantic-children="108,110" data-semantic-content="109" data-semantic-parent="128" data-semantic-owns="108 109 110"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="108" data-semantic-children="86,107" data-semantic-parent="127" data-semantic-owns="86 107"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="86" data-semantic-children="70,84" data-semantic-content="85" data-semantic-parent="108" data-semantic-owns="70 85 84"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="70" data-semantic-children="66,69" data-semantic-parent="86" data-semantic-owns="66 69"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="66" data-semantic-parent="70"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="69" data-semantic-children="67,68" data-semantic-parent="70" data-semantic-owns="67 68"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="67" data-semantic-parent="69"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="68" data-semantic-parent="69"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="85" data-semantic-parent="86" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="84" data-semantic-children="79,82" data-semantic-content="71" data-semantic-parent="86" data-semantic-owns="79 82"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="79" data-semantic-children="71,75,78" data-semantic-parent="84" data-semantic-owns="71 75 78"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="71" data-semantic-parent="79" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.297em; margin-left: 0px;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="78" data-semantic-children="76,77" data-semantic-parent="79" data-semantic-owns="76 77"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="76" data-semantic-parent="78"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="77" data-semantic-parent="78"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="75" data-semantic-children="72,74" data-semantic-content="73" data-semantic-parent="79" data-semantic-owns="72 73 74"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="72" data-semantic-parent="75"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="73" data-semantic-parent="75" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="74" data-semantic-parent="75"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="82" data-semantic-children="80,81" data-semantic-parent="84" data-semantic-owns="80 81"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="80" data-semantic-parent="82"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="81" data-semantic-parent="82"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="107" data-semantic-children="91,105" data-semantic-content="106" data-semantic-parent="108" data-semantic-owns="91 106 105"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="91" data-semantic-children="87,90" data-semantic-parent="107" data-semantic-owns="87 90"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="87" data-semantic-parent="91"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="90" data-semantic-children="88,89" data-semantic-parent="91" data-semantic-owns="88 89"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="88" data-semantic-parent="90"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="89" data-semantic-parent="90"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="106" data-semantic-parent="107" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="105" data-semantic-children="100,103" data-semantic-content="92" data-semantic-parent="107" data-semantic-owns="100 103"><mjx-msubsup data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="100" data-semantic-children="92,96,99" data-semantic-parent="105" data-semantic-owns="92 96 99"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="92" data-semantic-parent="100" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.288em; margin-left: 0px;"><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="99" data-semantic-children="97,98" data-semantic-parent="100" data-semantic-owns="97 98"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="97" data-semantic-parent="99"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="98" data-semantic-parent="99"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-spacer style="margin-top: 0.18em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="96" data-semantic-children="93,95" data-semantic-content="94" data-semantic-parent="100" data-semantic-owns="93 94 95"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="93" data-semantic-parent="96"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="94" data-semantic-parent="96" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="95" data-semantic-parent="96"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="103" data-semantic-children="101,102" data-semantic-parent="105" data-semantic-owns="101 102"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="101" data-semantic-parent="103"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="102" data-semantic-parent="103"><mjx-c class="mjx-c1D457">𝑗</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="109" data-semantic-parent="127" data-semantic-operator="relseq,>"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="110" data-semantic-parent="127"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>Where x<sub>i</sub> is the transcripts per million of a gene y across all gland libraries (n<sub>g</sub>), and x<sub>j</sub> is the transcripts per million of a gene y across all control libraries (n<sub>c</sub>). The code can be found in the R script ‘sleuth_R_file’ in the Script repository on CaltechData (<a id="intref0520" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). Based on the differential expression data, we extracted transcripts that were down or upregulated in the BQ or solvent gland cells compared to control tissue.</div></section><section id="sec-4-3-14"><h4>Preparation of single cell suspension for 10x scRNaseq</h4><div id="p0345" role="paragraph">One hundred and fifty tergal gland segments (Segment 7) and control segments (Segment 6) of <i>Dalotia</i> males were dissected in EBSS (Earle’s Balanced Salt Solution; ThermoFisher) and transferred to into 150&nbsp;μl ice-cold Schneider’s <i>Drosophila</i> medium and fetal bovine serum (SDM+FBS, 90:10 V/V; both ThermoFisher). After centrifugation at 250x g for 2&nbsp;min, subsequent washing in 300&nbsp;μl ice-cold EBSS and centrifugation for another 2&nbsp;min at 250x g, the supernatant was replaced by 150&nbsp;μl of pre-warmed (37°C) trypsin (Corning 0.25% Trypsin, 0.1% EDTA; Fisher Scientific). Cells were dissociated at 37ᵒC for 25&nbsp;min and the reaction was reinforced by pipette mixing every 5&nbsp;min. After 25&nbsp;min, trypsin was inhibited with 100&nbsp;μl soybean trypsin inhibitor (SigmaAldrich) and incubated for 5&nbsp;min. Subsequently, 500&nbsp;μl SMD+FBS was added and the cell suspension was passed through a 40&nbsp;μm cell strainer (pluriStrainer, ImTec Diagnostics). Finally, cells were pelleted at 300x g for 8&nbsp;min and resuspended in Dulbecco’s Phosphate-Buffered Saline (DPBS) 0.04% BSA (ThermoFisher) prior to 10x scRNaseq runs. For cell counting the LIVE/DEAD Viability/Cytotoxicity Kit (Invitrogen, ThermoFisher) was used. The above protocol was repeated twice for segment 6 and three times for segment 7, yielding five 10x runs.</div></section><section id="sec-4-3-15"><h4>Single-cell RNA sequencing (scRNaseq) library preparation and sequencing</h4><div id="p0350" role="paragraph">After manually determining the cell concentration using a hemocytometer, suspensions were further diluted to desired concentrations (∼1000 cells/μL) if necessary. The appropriate volume of reverse transcription mix was added to target 2,500–10,000 cells for loading into the 10x chip. The Chromium Single Cell 3′ Library &amp; Gel Bead Kit v3, Chromium Single Cell 3′Chip kit v3 (PN-120236), and Chromium i7 Multiplex Kit were used for all downstream RT, cDNA amplification (12 PCR cycles), and library preparation as instructed by the manufacturer (Chromium Single Cell 3′Reagents Kits v3&nbsp;User Guide). The quality and concentration of the resulting libraries were assessed using the Qubit High Sensitivity dsDNA kit (Thermo Scientific) and Agilent Bioanalyzer High Sensitivity DNA assay. scRNaseq libraries were sequenced on an Illumina HiSeq4000 or NovaSeq6000 (paired-end with read lengths of 150 nt) and reads were aligned to the reference transcriptome using kallisto 0.46.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref21-4" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-4" aria-controls="sref21-4">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref21-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.L.BrayH.PimentelP.MelstedL.PachterNear-optimal+probabilistic+RNA-seq+quantificationNat.+Biotechnol.342016525527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From a total of 1,669,423,289 reads, 86.3% mapped to the <i>Dalotia</i> transcriptome using kallisto | bustools (<span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Melsted et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Melsted, P. ∙ Ntranos, V. ∙ Pachter, L.</div><div><strong>The barcode, UMI, set format and BUStools</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:4472-4473</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MelstedV.NtranosL.PachterThe+barcode%2C+UMI%2C+set+format+and+BUStoolsBioinformatics35201944724473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" aria-label="Reference 2021" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">2021</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Melsted, P. ∙ Booeshaghi, A.S. ∙ Liu, L. ...</div><div><strong>Modular, efficient and constant-memory single-cell RNA-seq preprocessing</strong></div><div><em>Nat. Biotechnol.</em> 2021; <strong>39</strong>:813-818</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MelstedA.S.BooeshaghiL.LiuF.GaoL.LuK.H.J.MinE.da+Veiga+BeltrameK.E.Hj%C3%B6rleifssonJ.GehringL.PachterModular%2C+efficient+and+constant-memory+single-cell+RNA-seq+preprocessingNat.+Biotechnol.392021813818" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), giving an approximate sequencing depth of 100,000 reads per cell. The total number of transcripts detected across all cells was 26,259 (58.3%) out of&nbsp;45,044 RNAs in the transcriptome. The mean number of genes per cell ranged between 1,227 for Segment 7 and 1,219 for Segment 6 (CaltechData: <a id="intref0525" href="https://data.caltech.edu/records/1915" target="_blank">https://data.caltech.edu/records/1915</a> and <a id="intref0530" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>).</div></section><section id="sec-4-3-16"><h4>Analysis of scRNaseq data</h4><div id="p0355" role="paragraph">Kallisto alignment files were quantified with BUStools (v.0.39.3) (<span class="dropBlock reference-citations"><a id="body-ref-sref97-3" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-3" aria-controls="sref97-3">Melsted et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref97-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Melsted, P. ∙ Booeshaghi, A.S. ∙ Liu, L. ...</div><div><strong>Modular, efficient and constant-memory single-cell RNA-seq preprocessing</strong></div><div><em>Nat. Biotechnol.</em> 2021; <strong>39</strong>:813-818</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MelstedA.S.BooeshaghiL.LiuF.GaoL.LuK.H.J.MinE.da+Veiga+BeltrameK.E.Hj%C3%B6rleifssonJ.GehringL.PachterModular%2C+efficient+and+constant-memory+single-cell+RNA-seq+preprocessingNat.+Biotechnol.392021813818" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and loaded into R using BUSpaRse (<span class="dropBlock reference-citations"><a id="body-ref-sref101-2" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-2" aria-controls="sref101-2">Moses and Pachter, 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Moses, L. ∙ Pachter, L.</div><div><strong>BUSpaRse: kallisto | bustools R utilities, R Package Version 1.4.2</strong></div><div>2021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.MosesL.PachterBUSpaRse%3A+kallisto+%7C+bustools+R+utilities%2C+R+Package+Version+1.4.22021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). First, we used knee plot method<span data-style="line-through">s</span> as implemented in DropletUtils (<span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Lun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ..., 1st Human Cell Atlas Jamboree</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.L.LunS.RiesenfeldT.AndrewsT.P.DaoT.GomesJ.C.Marioni1st+Human+Cell+Atlas+JamboreeEmptyDrops%3A+distinguishing+cells+from+empty+droplets+in+droplet-based+single-cell+RNA+sequencing+dataGenome+Biol.20201963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to filter empty droplets before downstream analysis in Seurat v3.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref142-2" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-2" aria-controls="sref142-2">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref142-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.StuartA.ButlerP.HoffmanC.HafemeisterE.PapalexiW.M.Mauck3rdY.HaoM.StoeckiusP.SmibertR.SatijaComprehensive+Integration+of+Single-Cell+DataCell177201918881902.e21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To create Seurat objects for each library, we filtered cells with low (nFeature_RNA &gt; 200) or overly large (nFeature_RNA &gt; 15,000) transcript counts, merged all objects into one for quality assessment, and then split by library for normalization and scaling. From three 10x runs for Segment 7 and two 10x runs for Segment 6, we obtained 10,364 high-quality cells after stringent filtering (4,663 and 5,701 cells from Segments 6 and 7, respectively). Individual transcript counts in each cell were normalized by the total number of counts for that cell, multiplied by a scale factor (10,000), and natural log-transformed (NormalizeData function). Expression of each transcript was scaled to achieve a mean of zero and variance of 1 across cells (ScaleData function). Transcripts with high overall expression and variance across cells were identified (FindVariableFeatures function; highest standardized variance was selected by selection.method&nbsp;= ‘vst’). Subsequently, all five objects (libraries) were integrated, and shared sources of variation were identified by Seurat CCA alignment (<span class="dropBlock reference-citations"><a id="body-ref-sref29-3" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-3" aria-controls="sref29-3">Butler et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref29-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ButlerP.HoffmanP.SmibertE.PapalexiR.SatijaIntegrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+speciesNat.+Biotechnol.362018411420" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref142-3" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-3" aria-controls="sref142-3">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref142-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.StuartA.ButlerP.HoffmanC.HafemeisterE.PapalexiW.M.Mauck3rdY.HaoM.StoeckiusP.SmibertR.SatijaComprehensive+Integration+of+Single-Cell+DataCell177201918881902.e21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We selected the 3000 most variable transcripts across datasets (SelectIntegrationFeatures function) and used this set of transcripts to identify correspondences among the datasets, termed anchors (FindIntegrationAnchors function). Anchors were used to integrate the five datasets (IntegrateData function). The integrated Seurat object was again scaled and used as input for dimensionality reduction via Principal Component Analysis (PCA). Subsequently, we constructed a Shared Nearest Neighbor (SNN) graph for the dataset (FindNeighbors function) using the first 20 principal components of the PCA as well as an SNN pruning cutoff of 0.067 before performing cell clustering with the Louvain algorithm (FindClusters function; resolution of 0.25-1.0). We eventually used a resolution of 0.55, as this resulted in the separation of cell clusters that we had previously identified as different cell types by HCR (see CaltechData: <a id="intref0535" href="https://doi.org/10.22002/D1.1918" target="_blank">https://doi.org/10.22002/D1.1918</a>). Finally, we ran the Uniform Manifold Approximation and Projection (UMAP) dimensional reduction technique on the 20 principal components using default options, creating the abdominal cell type atlas (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.BechtL.McInnesJ.HealyC.-A.DutertreI.W.H.KwokL.G.NgF.GinhouxE.W.NewellDimensionality+reduction+for+visualizing+single-cell+data+using+UMAPNat.+Biotechnol.3720183844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">McInnes et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>McInnes, L. ∙ Healy, J. ∙ Melville, J.</div><div><strong>UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction</strong></div><div><em>arXiv.</em> 2018;</div><div>arXiv:1802.03426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.McInnesJ.HealyJ.MelvilleUMAP%3A+Uniform+Manifold+Approximation+and+Projection+for+Dimension+ReductionarXiv2018arXiv%3A1802.03426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-17"><h4>Cell type annotation and statistical validation of cell classes</h4><div id="p0360" role="paragraph">To annotate cell types, differentially expressed transcripts in cell clusters were identified in Seurat v3.1 (FindAllMarkers function) using both the Wilcoxon Rank Sum test as well as ROC analysis. Marker transcripts were computed using default settings for each cluster individually (FindMarkers function). A dendrogram was built in Past 4.02 (<span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">Hammer et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Hammer, O. ∙ Harper, D.A. ∙ Ryan, P.D.</div><div><strong>PAST: Paleontological statistics software package for education and data analysis</strong></div><div><em>Palaeontol. Electronica.</em> 2001; <strong>4</strong>:9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.HammerD.A.HarperP.D.RyanPAST%3A+Paleontological+statistics+software+package+for+education+and+data+analysisPalaeontol.+Electronica420019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using neighbor joining clustering (NJ) (see <span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Musser et&nbsp;al. [2021</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Musser, J.M. ∙ Schippers, K.J. ∙ Nickel, M. ...</div><div><strong>Profiling cellular diversity in sponges informs animal cell type and nervous system evolution</strong></div><div><em>Science.</em> 2021; <strong>374</strong>:717-723</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.MusserK.J.SchippersM.NickelG.MizzonA.B.KohnC.PapeJ.U.HammelF.WolfC.LiangA.Hern%C3%A1ndez-PlazaProfiling+cellular+diversity+in+sponges+informs+animal+cell+type+and+nervous+system+evolutionScience3742021717723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>] for a similar approach) on a Bray-Curtis similarity matrix (n&nbsp;= 10,000 bootstraps) based on the normalized and scaled cluster average expression profiles of the 3000 most variable transcripts. We used the list of overall differentially expressed genes, the individual cluster markers, the results of the NJ clustering and the transcript annotation, to connect cell types to known cell type classes using published marker genes from fly expression data using <a id="intref0540" href="http://flybase.org/" target="_blank">http://flybase.org/</a>, <a id="intref0545" href="https://www.uniprot.org/" target="_blank">https://www.uniprot.org/</a> and <a id="intref0550" href="https://bgee.org" target="_blank">https://bgee.org</a> (<a id="intref0555" href="https://data.caltech.edu/records/1915" target="_blank">https://data.caltech.edu/records/1915</a>).</div><div id="p0365" role="paragraph">To test the robustness of NJ clustering, we employed random forest analysis to assign cells to their pre-defined cell type classes. Random forest is a machine learning classifier that assigns samples (here cells) to predefined groups (here cell type classes) in multiple iterations and estimates the correct classification (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Breiman, 2001</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Breiman, L.</div><div><strong>Random Forests</strong></div><div><em>Mach. Learn.</em> 2001; <strong>45</strong>:5-32</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.BreimanRandom+ForestsMach.+Learn.452001532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the analysis, ntree&nbsp;= 1,000 bootstrap replicates were drawn with mtry&nbsp;= 55 variables randomly selected at each node. A confusion matrix showing the number of correctly assigned cell communities to either cell type as well as proportional class errors and total out-of-basket (OOB) estimate of error rates were computed.</div></section><section id="sec-4-3-18"><h4>Consensus non-Negative Matrix Factorization</h4><div id="p0370" role="paragraph">We used the filtered cells-by-transcripts count matrix (10,634 cells by 26,259 transcripts) to infer gene expression programs (GEPs) of single cells using consensus Non-negative Matrix Factorization (cNMF) as implemented in Python (<span class="dropBlock reference-citations"><a id="body-ref-sref78-4" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-4" aria-controls="sref78-4">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The method takes the count matrix (N cells x T transcripts) as input and produces both a gene expression program (GEP) x T matrix and a N x GEP matrix that specify the usage of each program for each cell in the data. The cNMF implementation runs multiple NMFs for each number of components (K) over a range of selected K, and combines the results of each replicate for different K to obtain more robust consensus estimates for the stability (silhouette score) and Frobenius error of each NMF model (<span class="dropBlock reference-citations"><a id="body-ref-sref78-5" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-5" aria-controls="sref78-5">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We selected the most over-dispersed transcripts (T&nbsp;= 2000) using v-scoring (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Klein et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Klein, A.M. ∙ Mazutis, L. ∙ Akartuna, I. ...</div><div><strong>Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1187-1201</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.M.KleinL.MazutisI.AkartunaN.TallapragadaA.VeresV.LiL.PeshkinD.A.WeitzM.W.KirschnerDroplet+barcoding+for+single-cell+transcriptomics+applied+to+embryonic+stem+cellsCell161201511871201" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as the input for cNMF, which is essential for performing cNMF on biologically meaningful transcriptional variation (<span class="dropBlock reference-citations"><a id="body-ref-sref78-6" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-6" aria-controls="sref78-6">Kotliar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref78-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Expression of each over-dispersed transcript T was then scaled to unit variance and subjected to cNMF over a broad range of K (K&nbsp;= 16-36). The cNMF implementation was run with default parameters with the maximum number of iterations set to 250. Each iteration was run on the same normalized dataset over the range of Ks but with different randomly selected seeds. Subsequently, the factorizations for all iterations of each K were combined and the stability and error were computed. To choose the number of components K, we followed several approaches suggested by <span class="dropBlock reference-citations"><a id="body-ref-sref78-7" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-7" aria-controls="sref78-7">Kotliar et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref78-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; i) we plotted the stability and error as a function of K (<span class="dropBlock reference-citations"><a id="body-ref-sref1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1" aria-controls="sref1">Alexandrov et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alexandrov, L.B. ∙ Nik-Zainal, S. ∙ Wedge, D.C. ...</div><div><strong>Deciphering signatures of mutational processes operative in human cancer</strong></div><div><em>Cell Rep.</em> 2013; <strong>3</strong>:246-259</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.B.AlexandrovS.Nik-ZainalD.C.WedgeP.J.CampbellM.R.StrattonDeciphering+signatures+of+mutational+processes+operative+in+human+cancerCell+Rep.32013246259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), ii) performed a scree plot analysis of variance explained per principal component for the same dataset to confirm the appropriate choice of K (Eckart–Young–Mirsky theorem; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Eckart and Young, 1936</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Eckart, C. ∙ Young, G.</div><div><strong>The approximation of one matrix by another of lower rank</strong></div><div><em>Psychometrika.</em> 1936; <strong>1</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.EckartG.YoungThe+approximation+of+one+matrix+by+another+of+lower+rankPsychometrika11936211218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and iii) re-ran the same cNMF steps with a variable number of over-dispersed T to see whether stability and error converge on a similar K (CaltechData: <a id="intref0560" href="https://data.caltech.edu/records/1915" target="_blank">https://data.caltech.edu/records/1915</a> and <a id="intref0565" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>; <a id="crosref2025" href="#mmc3">Data S3</a>).</div><div id="p0375" role="paragraph">Ultimately, we selected K&nbsp;= 20 and proceeded with the downstream cNMF workflow using default options, with density threshold r&nbsp;set to 0.05 in order to filter out replicates with low matching to a respective GEP (<a id="crosref2030" href="#mmc3">Data S3</a>). We used the K&nbsp;= 20 cNMF dataset to calculate the normalized GEP usage score for each cell, and then loaded the cell type or cell class assignment from our previous Seurat clustering/NJ+random forest analysis into Python to calculate GEP usage (in %) for each cell type/class. The resulting GEP usage matrix for each cell type was visualized as a heatmap. We also exported the normalized cells-to-GEPs (N x GEP) and GEPs-to-transcript (GEP x T) matrices (CaltechData: <a id="intref0570" href="https://data.caltech.edu/records/1915" target="_blank">https://data.caltech.edu/records/1915</a>) and loaded the N x GEP matrix into R as a matrix, integrating the GEP usage data as a new assay into the pre-existing Seurat object (CreateAssayObject function). This enabled us to plot % GEP usage across single cells, permitting visualization of GEP usage across the UMAP cell atlas.</div><div id="p0380" role="paragraph">Finally, we extracted the expression data for the GEP9 to examine its composition (<a id="intref0575" href="https://data.caltech.edu/records/1915" target="_blank">https://data.caltech.edu/records/1915</a>). We assigned GO terms to each GEP9 transcript and performed over-representation analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref160-2" href="#" role="doc-biblioref" data-xml-rid="bib160" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160-2" aria-controls="sref160-2">Yu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref160-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Yu, G. ∙ Wang, L.-G. ∙ Han, Y. ...</div><div><strong>clusterProfiler: an R package for comparing biological themes among gene clusters</strong></div><div><em>OMICS.</em> 2012; <strong>16</strong>:284-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.YuL.-G.WangY.HanQ.-Y.HeclusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clustersOMICS162012284287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of GO terms to visualize the number of genes for each GO-term after GO-slim. GO terms for the respective gene models were assigned based on the gene id with highest homology from the SwissProt database (download February 2019) or NCBI nr database (downloaded February 2019). A custom database of GO terms was created with makeOrgPackage function in the R package AnnotationForge v1.26.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref32-2" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-2" aria-controls="sref32-2">Carlson and Pages, 2019</a><div class="dropBlock__holder" data-db-target-of="sref32-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Carlson, M. ∙ Pages, H.</div><div><strong>AnnotationForge: tools for building SQLite-based annotation data packages. R package version 1.36.0</strong></div><div>2019</div><div><a id="intref0635" href="https://bioconductor.org/packages/AnnotationForge" target="_blank">https://bioconductor.org/packages/AnnotationForge</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.CarlsonH.PagesAnnotationForge%3A+tools+for+building+SQLite-based+annotation+data+packages.+R+package+version+1.36.02019https%3A%2F%2Fbioconductor.org%2Fpackages%2FAnnotationForge" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Over-representation analysis of GO terms was tested using the enrichGO function in the R package clusterProfiler v3.12.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref160-3" href="#" role="doc-biblioref" data-xml-rid="bib160" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160-3" aria-controls="sref160-3">Yu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref160-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Yu, G. ∙ Wang, L.-G. ∙ Han, Y. ...</div><div><strong>clusterProfiler: an R package for comparing biological themes among gene clusters</strong></div><div><em>OMICS.</em> 2012; <strong>16</strong>:284-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.YuL.-G.WangY.HanQ.-Y.HeclusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clustersOMICS162012284287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with a hypergeometric distribution and a Fisher’s Exact test. P values were adjusted for multiple comparisons using false discovery rate correction (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Benjamini and Hochberg, 1995</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Benjamini, Y. ∙ Hochberg, Y.</div><div><strong>Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</strong></div><div><em>J R Stat Soc Series B Stat Methodol.</em> 1995; <strong>57</strong>:289-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Enriched GO-terms were subsequently visualized using the REViGO tool available at <a id="intref0580" href="http://revigo.irb.hr" target="_blank">http://revigo.irb.hr</a> and the MonaGO tool available at <a id="interref0010" href="https://monago.erc.monash.edu/" target="_blank">https://monago.erc.monash.edu/</a>.</div></section><section id="sec-4-3-19"><h4>Genome assembly and gene prediction</h4><div id="p0385" role="paragraph">Genomic DNA was isolated using a non-destructive extraction method described by <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Maruyama and Parker (2017)</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Maruyama, M. ∙ Parker, J.</div><div><strong>Deep-Time Convergence in Rove Beetle Symbionts of Army Ants</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:920-926</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MaruyamaJ.ParkerDeep-Time+Convergence+in+Rove+Beetle+Symbionts+of+Army+AntsCurr.+Biol.272017920926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> for <i>Coproporus</i> and <i>Drusilla canaliculata</i> or, for <i>Tachinus</i>, using the QIAGEN Blood &amp; Cell Culture DNA Mini Kit following the manufacturer’s protocol. DNA quantity was assessed using the Qubit High Sensitivity dsDNA kit (Thermo Scientific) and DNA integrity was assessed with Bioanalyzer. Illumina paired-end libraries (2&nbsp;× 150bp) were prepared using the Illumina TruSeq DNA kit or NEBNext Ultra FS DNA library kit and sequenced on either the Illumina HiSeq X platform by Iridian Genomes/J. Parker and collaborators, or the HiSeq 2500 platform by the Millard and Muriel Jacobs Genetics and Genomics Laboratory at California Institute of Science and Technology. Reads for <i>Coproporus</i> and <i>Drusilla</i> were previously deposited on NCBI’s Sequence read archive (<i>Coproporus</i> SRA id: SRR4301367; Drusilla SRA id: SRR5906249). Additionally, we downloaded reads of <i>Aleochara bilineata</i> (NICBI BioProject: PRJNA378164) (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Kraaijeveld et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Kraaijeveld, K. ∙ Neleman, P. ∙ Mariën, J. ...</div><div><strong>Genomic Resources for Goniozus legneri, Aleochara bilineata and Paykullia maculata, Representing Three Independent Origins of the Parasitoid Lifestyle in Insects.</strong></div><div><em>G3 (Bethesda).</em> 2019; <strong>9</strong>:987-991</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KraaijeveldP.NelemanJ.Mari%C3%ABnE.+deMeijerJ.EllersGenomic+Resources+for+Goniozus+legneri%2C+Aleochara+bilineata+and+Paykullia+maculata%2C+Representing+Three+Independent+Origins+of+the+Parasitoid+Lifestyle+in+Insects.G3+%28Bethesda%2992019987991" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), re-assembled the genome and performed gene prediction.</div><div id="p0390" role="paragraph">Read quality for each taxon was assessed using FastQC v0.11.8 (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Andrews, S.</div><div><strong>FastQC: a quality control tool for high throughput sequence data</strong></div><div>2010</div><div><a id="intref0625" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+a+quality+control+tool+for+high+throughput+sequence+data2010https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Illumina adapters, low-quality bases (phred scores below 15) and reads shorter than 50&nbsp;bp were removed using cutadapt v1.18 (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Martin, 2011</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnet. J.</em> 2011; <strong>17</strong>:10-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MartinCutadapt+removes+adapter+sequences+from+high-throughput+sequencing+readsEMBnet.+J.1720111012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A preliminary assembly constructed from the filtered, adaptor-trimmed reads using MEGAHIT v1.1.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Li et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Li, D. ∙ Luo, R. ∙ Liu, C.-M. ...</div><div><strong>MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices.</strong></div><div><em>Methods.</em> 2016; <strong>102</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.LiR.LuoC.-M.LiuC.-M.LeungH.-F.TingK.SadakaneH.YamashitaT.-W.LamMEGAHIT+v1.0%3A+A+fast+and+scalable+metagenome+assembler+driven+by+advanced+methodologies+and+community+practices.Methods1022016311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with multiple <i>k-mers</i> (–k-list&nbsp;= 21, 29, 39, 59, 79, 99, 119) was filtered for bacterial contamination using Blobtools v1.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Laetsch and Blaxter, 2017</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Laetsch, D.R. ∙ Blaxter, M.L.</div><div><strong>BlobTools: Interrogation of genome assemblies.</strong></div><div><em>F1000Res.</em> 2017; <strong>6</strong>:1287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.R.LaetschM.L.BlaxterBlobTools%3A+Interrogation+of+genome+assemblies.F1000Res.620171287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Contigs with sequence similarity to bacterial sequences to the NCBI nt database (downloaded February 2019, Evalue 1e-25) were removed. The filtered contigs were then assembled into scaffolds with three iterations of the Redundans v0.14a (<span class="dropBlock reference-citations"><a id="body-ref-sref118-2" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-2" aria-controls="sref118-2">Pryszcz and Gabaldón, 2016</a><div class="dropBlock__holder" data-db-target-of="sref118-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Pryszcz, L.P. ∙ Gabaldón, T.</div><div><strong>Redundans: an assembly pipeline for highly heterozygous genomes</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e113</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.P.PryszczT.Gabald%C3%B3nRedundans%3A+an+assembly+pipeline+for+highly+heterozygous+genomesNucleic+Acids+Res.442016e113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) reference-based pipeline using the <i>Dalotia</i> v1 genome assembly as the reference (–iters 3,–limit 0.5,–nogapclosing). Scaffolds smaller than 1 kb were removed and gaps filled using GapCloser v1.12 (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Luo et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Luo, R. ∙ Liu, B. ∙ Xie, Y. ...</div><div><strong>SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</strong></div><div><em>Gigascience.</em> 2012; <strong>1</strong>:18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.LuoB.LiuY.XieZ.LiW.HuangJ.YuanG.HeY.ChenQ.PanY.LiuSOAPdenovo2%3A+an+empirically+improved+memory-efficient+short-read+de+novo+assemblerGigascience1201218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genome completeness was measured against the conserved Arthropoda v10 gene set (n&nbsp;= 1013 single-copy genes) using BUSCO v4.1.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref132-3" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132-3" aria-controls="sref132-3">Simão et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref132-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Simão, F.A. ∙ Waterhouse, R.M. ∙ Ioannidis, P. ...</div><div><strong>BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3210-3212</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.A.Sim%C3%A3oR.M.WaterhouseP.IoannidisE.V.KriventsevaE.M.ZdobnovBUSCO%3A+assessing+genome+assembly+and+annotation+completeness+with+single-copy+orthologsBioinformatics31201532103212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0395" role="paragraph">Species-specific repetitive elements were identified with RepeatModeler v 1.0.11 (<span class="dropBlock reference-citations"><a id="body-ref-sref134-2" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-2" aria-controls="sref134-2">Smit and Hubley, 2008</a><div class="dropBlock__holder" data-db-target-of="sref134-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Smit, A.F. ∙ Hubley, R.</div><div><strong>RepeatModeler Open-1.0</strong></div><div>2008</div><div><a id="intref0660" href="https://www.scienceopen.com/document?vid=c1050e3d-8dc1-48ba-81c8-b372af25be50" target="_blank">https://www.scienceopen.com/document?vid=c1050e3d-8dc1-48ba-81c8-b372af25be50</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.F.SmitR.HubleyRepeatModeler+Open-1.02008https%3A%2F%2Fwww.scienceopen.com%2Fdocument%3Fvid%3Dc1050e3d-8dc1-48ba-81c8-b372af25be50" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and MITE tracker (<span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Crescente et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Crescente, J.M. ∙ Zavallo, D. ∙ Helguera, M. ...</div><div><strong>MITE Tracker: an accurate approach to identify miniature inverted-repeat transposable elements in large genomes</strong></div><div><em>BMC Bioinformatics.</em> 2018; <strong>19</strong>:348</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.CrescenteD.ZavalloM.HelgueraL.S.VanzettiMITE+Tracker%3A+an+accurate+approach+to+identify+miniature+inverted-repeat+transposable+elements+in+large+genomesBMC+Bioinformatics192018348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as described by (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Brückner et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Brückner, A. ∙ Barnett, A.A. ∙ Antoshechkin, I.A. ...</div><div><strong>Molecular evolutionary trends and biosynthesis pathways in the Oribatida revealed by the genome of Archegozetes longisetosus</strong></div><div><em>Biorxiv.</em> 2021;</div><div>2020.12.10.420141</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerA.A.BarnettI.A.AntoshechkinS.A.KitchenMolecular+evolutionary+trends+and+biosynthesis+pathways+in+the+Oribatida+revealed+by+the+genome+of+Archegozetes+longisetosusBiorxiv20212020.12.10.420141" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Repeats were further filtered for genuine gene products through sequence similarity searches against a local protein database of beetle sequences (RefSeq: GCF_001937115.1,&nbsp;GCF_000699045.2, GCF_000390285.2, GCF_000355655.1, GCF_000500325.1, GCF_001412225.1, GCF_000648695.1, GCA_001443705.1, GCF_000002335.3). Unclassified repeats with sequence similarity to known TEs were retained (E-value 1e-5) whereas those with no blast homology were removed (<span class="dropBlock reference-citations"><a id="body-ref-sref110" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110" aria-controls="sref110">Petersen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref110"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Petersen, M. ∙ Armisén, D. ∙ Gibbs, R.A. ...</div><div><strong>Diversity and evolution of the transposable element repertoire in arthropods with particular reference to insects</strong></div><div><em>BMC Evol. Biol.</em> 2019; <strong>19</strong>:11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.PetersenD.Armis%C3%A9nR.A.GibbsL.HeringA.KhilaG.MayerS.RichardsO.NiehuisB.MisofDiversity+and+evolution+of+the+transposable+element+repertoire+in+arthropods+with+particular+reference+to+insectsBMC+Evol.+Biol.19201911" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Predicted repeats that were 80% identical were clustered with vsearch v 2.7.1 (–iddef 1–id 0.8–strand both) (<span class="dropBlock reference-citations"><a id="body-ref-sref124-2" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-2" aria-controls="sref124-2">Rognes et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref124-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Rognes, T. ∙ Flouri, T. ∙ Nichols, B. ...</div><div><strong>VSEARCH: a versatile open source tool for metagenomics</strong></div><div><em>PeerJ.</em> 2016; <strong>4</strong>:e2584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.RognesT.FlouriB.NicholsC.QuinceF.Mah%C3%A9VSEARCH%3A+a+versatile+open+source+tool+for+metagenomicsPeerJ42016e2584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The final repeat library was used to soft mask the assemblies using RepeatMasker v 4.07 (<span class="dropBlock reference-citations"><a id="body-ref-sref133-2" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133-2" aria-controls="sref133-2">Smit, 2004</a><div class="dropBlock__holder" data-db-target-of="sref133-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Smit, A.F.</div><div><strong>Repeat-Masker Open-3.0</strong></div><div>2004</div><div><a id="intref0655" href="http://www.Repeatmasker.Org" target="_blank">http://www.Repeatmasker.Org</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.F.SmitRepeat-Masker+Open-3.02004http%3A%2F%2Fwww.Repeatmasker.Org" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For <i>Coproporus</i>, <i>Tachinus</i> and <i>A.&nbsp;bilineata</i>, a combination of <i>ab inito</i> (GeneMark-ES v.4.33 (<span class="dropBlock reference-citations"><a id="body-ref-sref145-2" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-2" aria-controls="sref145-2">Ter-Hovhannisyan et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref145-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Ter-Hovhannisyan, V. ∙ Lomsadze, A. ∙ Chernoff, Y.O. ...</div><div><strong>Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training</strong></div><div><em>Genome Res.</em> 2008; <strong>18</strong>:1979-1990</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.Ter-HovhannisyanA.LomsadzeY.O.ChernoffM.BorodovskyGene+prediction+in+novel+fungal+genomes+using+an+ab+initio+algorithm+with+unsupervised+trainingGenome+Res.18200819791990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Augustus v3.2.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref138-2" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138-2" aria-controls="sref138-2">Stanke and Waack, 2003</a><div class="dropBlock__holder" data-db-target-of="sref138-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>Stanke, M. ∙ Waack, S.</div><div><strong>Gene prediction with a hidden Markov model and a new intron submodel</strong></div><div><em>Bioinformatics.</em> 2003; <strong>19</strong>:ii215-ii225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.StankeS.WaackGene+prediction+with+a+hidden+Markov+model+and+a+new+intron+submodelBioinformatics19Suppl+22003ii215ii225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and reference proteome tools (GeMoMA v1.6.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Keilwagen et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Keilwagen, J. ∙ Hartung, F. ∙ Grau, J.</div><div><strong>GeMoMa: Homology-Based Gene Prediction Utilizing Intron Position Conservation and RNA-seq Data</strong></div><div><em>Methods Mol. Biol.</em> 2019; <strong>1962</strong>:161-177</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.KeilwagenF.HartungJ.GrauGeMoMa%3A+Homology-Based+Gene+Prediction+Utilizing+Intron+Position+Conservation+and+RNA-seq+DataMethods+Mol.+Biol.19622019161177" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) were used to predict coding sequences. For <i>Drusilla</i>, in addition to the tools above, male and female whole body transcriptome evidence was used in two ways, through read mapping using BRAKER v2.1.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">Brůna et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Brůna, T. ∙ Hoff, K.J. ∙ Lomsadze, A. ...</div><div><strong>BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database</strong></div><div><em>Nar. Genom. Bioinform.</em> 2021; <strong>3</strong>:lqaa108</div></div><div class="external-links"><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.014&amp;key=2-s2.0-85111209000" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.Br%C5%AFnaK.J.HoffA.LomsadzeM.StankeM.BorodovskyBRAKER2%3A+automatic+eukaryotic+genome+annotation+with+GeneMark-EP%2B+and+AUGUSTUS+supported+by+a+protein+databaseNar.+Genom.+Bioinform.32021lqaa108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>de novo</i> Trinity assembled transcripts mapping using PASA v2.3.3 (<span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">Haas et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Haas, B.J. ∙ Salzberg, S.L. ∙ Zhu, W. ...</div><div><strong>Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasS.L.SalzbergW.ZhuM.PerteaJ.E.AllenJ.OrvisO.WhiteC.R.BuellJ.R.WortmanAutomated+eukaryotic+gene+structure+annotation+using+EVidenceModeler+and+the+Program+to+Assemble+Spliced+AlignmentsGenome+Biol.92008R7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Gene predictions were combined using EVidenceModeler v.1.1.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref61-4" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-4" aria-controls="sref61-4">Haas et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref61-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Haas, B.J. ∙ Salzberg, S.L. ∙ Zhu, W. ...</div><div><strong>Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasS.L.SalzbergW.ZhuM.PerteaJ.E.AllenJ.OrvisO.WhiteC.R.BuellJ.R.WortmanAutomated+eukaryotic+gene+structure+annotation+using+EVidenceModeler+and+the+Program+to+Assemble+Spliced+AlignmentsGenome+Biol.92008R7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with the following weights for <i>Corproporus</i> and <i>Tachinus</i>: ab inito evidence&nbsp;= 1 and protein evidence&nbsp;= 5; and for <i>Drusilla</i>: ab inito evidence&nbsp;= 1, protein evidence&nbsp;= 1, PASA&nbsp;= 10, and BRAKER&nbsp;= 4.</div></section><section id="sec-4-3-20"><h4>RNA extraction and transcriptome assembly</h4><div id="p0400" role="paragraph">Samples for RNaseq of <i>Lissagria laeviuscula</i> females and males (whole body) as well as <i>Platyusa sonomae</i> female and males (head and rest of the body separate) were placed in RNAlater and subsequently extracted following the Trizol protocol. Extracted RNA was shipped to Omega Bioservices (Norcross, GA, USA) on dry ice for library preparation and sequencing. The RNA for each sample was poly-A selected, cDNA was synthesized, and the final libraries were prepared following the Illumina TruSeq mRNA Stranded Kit protocol. Libraries were sequenced with ∼90 million 150&nbsp;bp paired-end reads on an Illumina HighSeq4000 platform. For each species, RNaseq reads were combined, <i>in silico</i> normalized and used for <i>de novo</i> transcriptome assembly with Trinity v2.8.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref59-3" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-3" aria-controls="sref59-3">Grabherr et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref59-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Grabherr, M.G. ∙ Haas, B.J. ∙ Yassour, M. ...</div><div><strong>Full-length transcriptome assembly from RNA-Seq data without a reference genome</strong></div><div><em>Nat. Biotechnol.</em> 2011; <strong>29</strong>:644-652</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.G.GrabherrB.J.HaasM.YassourJ.Z.LevinD.A.ThompsonI.AmitX.AdiconisL.FanR.RaychowdhuryQ.ZengFull-length+transcriptome+assembly+from+RNA-Seq+data+without+a+reference+genomeNat.+Biotechnol.292011644652" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sequences for each transcriptome were clustered and concatenated to remove duplications using CD-HIT (<span class="dropBlock reference-citations"><a id="body-ref-sref58-3" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-3" aria-controls="sref58-3">Fu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref58-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Fu, L. ∙ Niu, B. ∙ Zhu, Z. ...</div><div><strong>CD-HIT: accelerated for clustering the next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:3150-3152</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FuB.NiuZ.ZhuS.WuW.LiCD-HIT%3A+accelerated+for+clustering+the+next-generation+sequencing+dataBioinformatics28201231503152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with a sequence identity threshold of 0.95. We then ran TransDecoder (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Haas et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Haas, B.J. ∙ Papanicolaou, A. ∙ Yassour, M. ...</div><div><strong>De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:1494-1512</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasA.PapanicolaouM.YassourM.GrabherrP.D.BloodJ.BowdenM.B.CougerD.EcclesB.LiM.LieberDe+novo+transcript+sequence+reconstruction+from+RNA-seq+using+the+Trinity+platform+for+reference+generation+and+analysisNat.+Protoc.8201314941512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to predict the longest ORFs and obtained preliminary annotations of transcripts by blastx search (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Altschul et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Altschul, S.F. ∙ Gish, W. ∙ Miller, W. ...</div><div><strong>Basic local alignment search tool</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1990; <strong>215</strong>:403-410</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.AltschulW.GishW.MillerE.W.MyersD.J.LipmanBasic+local+alignment+search+toolJ.%C2%A0Mol.+Biol.2151990403410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) of all transcripts against the non-redundant NCBI protein database (<span class="dropBlock reference-citations"><a id="body-ref-sref117-2" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-2" aria-controls="sref117-2">Pruitt et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref117-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Pruitt, K.D. ∙ Tatusova, T. ∙ Maglott, D.R.</div><div><strong>NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D61-D65</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.D.PruittT.TatusovaD.R.MaglottNCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteinsNucleic+Acids+Res.352007D61D65" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-3-21"><h4>Phylogenetic analysis</h4><div id="p0405" role="paragraph">To infer enzyme relationships, we first ran TransDecoder (<span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Haas et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Haas, B.J. ∙ Papanicolaou, A. ∙ Yassour, M. ...</div><div><strong>De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:1494-1512</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasA.PapanicolaouM.YassourM.GrabherrP.D.BloodJ.BowdenM.B.CougerD.EcclesB.LiM.LieberDe+novo+transcript+sequence+reconstruction+from+RNA-seq+using+the+Trinity+platform+for+reference+generation+and+analysisNat.+Protoc.8201314941512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using default settings to predict the longest ORFs in the bulk <i>Dalotia</i> transcriptome and translate them into protein sequences. We then ran cd-hit (<span class="dropBlock reference-citations"><a id="body-ref-sref58-4" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-4" aria-controls="sref58-4">Fu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref58-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Fu, L. ∙ Niu, B. ∙ Zhu, Z. ...</div><div><strong>CD-HIT: accelerated for clustering the next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:3150-3152</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FuB.NiuZ.ZhuS.WuW.LiCD-HIT%3A+accelerated+for+clustering+the+next-generation+sequencing+dataBioinformatics28201231503152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with default options and a sequence identity threshold of 0.9 on this protein set to cluster them and remove short-length sequence artifacts. The protein sequences of eight previously published beetle genomes and <i>Drosophila melanogaster</i> were downloaded (see <a id="crosref2195" href="#sec-4-1">Key Resources Table</a>) and combined with the predicted protein sequences from the newly generated genomic data for <i>Coproporus</i> and <i>Tachinus</i> (outgroup Tachyporinae) and <i>Drusilla canaliculata</i> (Aleocharinae: Lomechusini), as well as transcriptome data for <i>Lissagria laeviuscula</i> (Aleocharinae: Falagriini) and <i>Platyusa sonomae</i> (Aleocharinae: Lomechusini) (protein datasets listed below and deposited at CaltechData: <a id="intref0590" href="https://data.caltech.edu/records/1919" target="_blank">https://data.caltech.edu/records/1919</a>). We again ran cd-hit with default options and an identity threshold of 0.98 to remove isoforms. The protein sets were clustered into orthogroups with Orthofinder v2 (<span class="dropBlock reference-citations"><a id="body-ref-sref48-2" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48-2" aria-controls="sref48-2">Emms and Kelly, 2015</a><div class="dropBlock__holder" data-db-target-of="sref48-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Emms, D.M. ∙ Kelly, S.</div><div><strong>OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:157</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.EmmsS.KellyOrthoFinder%3A+solving+fundamental+biases+in+whole+genome+comparisons+dramatically+improves+orthogroup+inference+accuracyGenome+Biol.162015157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref49-2" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" aria-label="Reference 2019" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49-2" aria-controls="sref49-2">2019</a><div class="dropBlock__holder" data-db-target-of="sref49-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Emms, D.M. ∙ Kelly, S.</div><div><strong>OrthoFinder: phylogenetic orthology inference for comparative genomics</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:238</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.EmmsS.KellyOrthoFinder%3A+phylogenetic+orthology+inference+for+comparative+genomicsGenome+Biol.202019238" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using initial Diamond sequence alignment (<span class="dropBlock reference-citations"><a id="body-ref-sref28-2" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28-2" aria-controls="sref28-2">Buchfink et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref28-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Buchfink, B. ∙ Xie, C. ∙ Huson, D.H.</div><div><strong>Fast and sensitive protein alignment using DIAMOND</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:59-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.BuchfinkC.XieD.H.HusonFast+and+sensitive+protein+alignment+using+DIAMONDNat.+Methods1220155960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and MCL clustering (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Enright et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Enright, A.J. ∙ Van Dongen, S. ∙ Ouzounis, C.A.</div><div><strong>An efficient algorithm for large-scale detection of protein families</strong></div><div><em>Nucleic Acids Res.</em> 2002; <strong>30</strong>:1575-1584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.EnrightS.Van+DongenC.A.OuzounisAn+efficient+algorithm+for+large-scale+detection+of+protein+familiesNucleic+Acids+Res.30200215751584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to assign orthogroups. This was followed by multiple sequence alignment using MAFFT (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Katoh and Standley, 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Katoh, K. ∙ Standley, D.M.</div><div><strong>MAFFT multiple sequence alignment software version 7: improvements in performance and usability</strong></div><div><em>Mol. Biol. Evol.</em> 2013; <strong>30</strong>:772-780</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KatohD.M.StandleyMAFFT+multiple+sequence+alignment+software+version+7%3A+improvements+in+performance+and+usabilityMol.+Biol.+Evol.302013772780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to align orthogroups and FastTree 2 (<span class="dropBlock reference-citations"><a id="body-ref-sref116-2" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-2" aria-controls="sref116-2">Price et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref116-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Price, M.N. ∙ Dehal, P.S. ∙ Arkin, A.P.</div><div><strong>FastTree 2--approximately maximum-likelihood trees for large alignments</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e9490</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.N.PriceP.S.DehalA.P.ArkinFastTree+2--approximately+maximum-likelihood+trees+for+large+alignmentsPLoS+ONE52010e9490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to infer Maximum Likelihood protein trees for each orthogroup. When then used the orthogroup alignments containing our target sequences and concatenated them into one master fasta-file for each enzyme family and realigned the protein sequences using MAFFT. All alignment files and tree files have been deposited at CaltechData (<a id="intref0595" href="https://data.caltech.edu/records/1916" target="_blank">https://data.caltech.edu/records/1916</a>). Master alignments were each analyzed using IQ-TREE v1.6.12 (<span class="dropBlock reference-citations"><a id="body-ref-sref103-2" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103-2" aria-controls="sref103-2">Nguyen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref103-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Nguyen, L.-T. ∙ Schmidt, H.A. ∙ von Haeseler, A. ...</div><div><strong>IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:268-274</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.-T.NguyenH.A.SchmidtA.von+HaeselerB.Q.MinhIQ-TREE%3A+a+fast+and+effective+stochastic+algorithm+for+estimating+maximum-likelihood+phylogeniesMol.+Biol.+Evol.322015268274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with 1,000 bootstrap replicates. Where appropriate, enzyme trees were rooted using <i>D.&nbsp;melanogaster</i>. Trees were visualized with FigTree v1.4.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref121-2" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121-2" aria-controls="sref121-2">Rambaut, 2012</a><div class="dropBlock__holder" data-db-target-of="sref121-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Rambaut, A.</div><div><strong>FigTree v1.4.1</strong></div><div>2012</div><div><a id="intref0645" href="http://Tree.Biol.Ed.Ac.Uk/Software/Figtree/" target="_blank">http://Tree.Biol.Ed.Ac.Uk/Software/Figtree/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.RambautFigTree+v1.4.12012http%3A%2F%2FTree.Biol.Ed.Ac.Uk%2FSoftware%2FFigtree%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For most trees, we used <i>Drosophila</i> sequences as an outgroup. The laccase tree in <a id="crosref2240" href="#fig6">Figure&nbsp;6</a>G is unrooted because the outgroup of insect laccases is unknown.</div><div id="p0410" role="paragraph">To characterize the cytochrome P450s of <i>Dalotia</i> belonging to the 4G subfamily, we downloaded a curated dataset of &gt; 350 sequences by <span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Feyereisen (2020)</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Feyereisen, R.</div><div><strong>Origin and evolution of the CYP4G subfamily in insects, cytochrome P450 enzymes involved in cuticular hydrocarbon synthesis</strong></div><div><em>Mol. Phylogenet. Evol.</em> 2020; <strong>143</strong>:106695</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.FeyereisenOrigin+and+evolution+of+the+CYP4G+subfamily+in+insects%2C+cytochrome+P450+enzymes+involved+in+cuticular+hydrocarbon+synthesisMol.+Phylogenet.+Evol.1432020106695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> and added the two <i>Dalotia</i> paralogs, performing a phylogenetic analysis with the same parameters as described in <span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Feyereisen (2020)</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Feyereisen, R.</div><div><strong>Origin and evolution of the CYP4G subfamily in insects, cytochrome P450 enzymes involved in cuticular hydrocarbon synthesis</strong></div><div><em>Mol. Phylogenet. Evol.</em> 2020; <strong>143</strong>:106695</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.FeyereisenOrigin+and+evolution+of+the+CYP4G+subfamily+in+insects%2C+cytochrome+P450+enzymes+involved+in+cuticular+hydrocarbon+synthesisMol.+Phylogenet.+Evol.1432020106695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. The tree was rooted with a <i>Panonychus citri</i> CYP450 sequence.</div></section><section id="sec-4-3-22"><h4>Protein Expression and <i>in&nbsp;vitro</i> enzymatic assays for laccase</h4><div id="p0415" role="paragraph">Synthetic cDNAs encoding the three laccases (Dmd, Lac1 and Lac2) were codon optimized for <i>E.&nbsp;coli</i> (CaltechData: <a id="intref0600" href="https://data.caltech.edu/records/1917" target="_blank">https://data.caltech.edu/records/1917</a>). Sequences were cloned into pTEV16 (<span class="dropBlock reference-citations"><a id="body-ref-opt9Cj5loRWYd-2" href="#" role="doc-biblioref" data-xml-rid="bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="opt9Cj5loRWYd-2" aria-controls="opt9Cj5loRWYd-2">VanDrisse and Escalante-Semerena, 2016</a><div class="dropBlock__holder" data-db-target-of="opt9Cj5loRWYd-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">154.</div><div>VanDrisse, C.M. ∙ Escalante-Semerena, J.C.</div><div><strong>New high-cloning-efficiency vectors for complementation studies and recombinant protein overproduction in Escherichia coli and Salmonella enterica</strong></div><div><em>Plasmid.</em> 2016; <strong>86</strong>:1-6</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.VanDrisseJ.C.Escalante-SemerenaNew+high-cloning-efficiency+vectors+for+complementation+studies+and+recombinant+protein+overproduction+in+Escherichia+coli+and+Salmonella+entericaPlasmid86201616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with a cleavable N-terminal 6xHis tag and TEV protease site. All proteins were expressed in BL21 competent <i>E.&nbsp;coli</i> cells (provided by C. VanDrisse, Caltech). Cells were grown in TB and protein expression was induced by 0.25&nbsp;mM IPTG at OD600 ≈0.7 and harvested after 12&nbsp;h at 20°C. Cells were lysed by freeze-thawing six times in lysis buffer (50&nbsp;mM MES, 150&nbsp;mM NaCl, 25&nbsp;mM imidazole, DNase, and Thermo Scientific Halt Protease Inhibitor Cocktail). Proteins were passed over a column with Ni-NTA resin at room temperature, washed with 50&nbsp;mM MES, 300&nbsp;mM NaCl, 25&nbsp;mM imidazole (pH&nbsp;= 7.5) and then eluted with 50&nbsp;mM MES, 150&nbsp;mM NaCl, 300&nbsp;mM imidazole (pH&nbsp;= 7.5). Proteins were dialyzed with Thermo Scientific SnakeSkin for 4&nbsp;h at room temperature into a final buffer of 10&nbsp;mM MES and 100&nbsp;mM CuSO<sub>4</sub> and then concentrated to 5&nbsp;mg/ml for experiments and storage at −80ᵒC.</div><div id="p0420" role="paragraph">Enzymatic activity of each protein was tested against a standard substrate, ABTS (<a id="crosref2255" href="#mmc6">Data S6</a>A). The reaction mixture was prepared as 5&nbsp;mM MES, 0.3&nbsp;M CuSO<sub>4</sub>, and 2&nbsp;mM of ABTS, with 2&nbsp;μM of laccase. The UV spectrum was traced for three minutes (<a id="crosref2260" href="#mmc6">Data S6</a>B). Subsequently, reactions were quenched with 5&nbsp;M urea. To verify the RNAi results of the gland laccase Dmd, we set up the following reaction: 2mM of each HQ (1,4-hydroquinone, 2-methyl-1,4-hydroquinone and 2-methoxy-3-methy-1,4-hydroquinone) was separately added to vials containing the same reaction buffer previously used. A pilot experiment was set up for each compound and Dmd protein was added (same concentration as above) (<a id="crosref2265" href="#mmc6">Data S6</a>C). this experiment was run for 10&nbsp;min to determine the quenching time for the enzyme reactions via UV recordings every 30 s. Subsequently, 20 samples for each compound were prepared using the same method. To half of the samples we added 0.5&nbsp;mM Dmd, while the other half served as control. Reactions were UV recorded for 1&nbsp;minute and directly quenched with 0.05&nbsp;M EDTA before being flash frozen and stored at −80ᵒC until further analysis. For the comparative analysis of Dmd, Laccase 1 and Laccase 2 we used the same buffers and substrates as previously. A total of three replicates and the same number of controls for each substrate-enzyme combination (i.e., 36 samples in total) were assayed.</div><div id="p0425" role="paragraph">To prepare samples for chemical profiling with either GC-FID or GC/MS for the Dmd versus control and comparative laccase assay, respectively, we used the following liquid-liquid extraction protocol. Frozen reaction mixtures were thawed and transferred into a clean glass vial with a sealed Teflon lid. For extraction, the aqueous phase was washed twice with 100&nbsp;μl freshly Na<sub>2</sub>SO<sub>4</sub>-dried hexane. Organic fractions were combined, dried over Na<sub>2</sub>SO<sub>4</sub> and carefully concentrated under a gentle stream of N<sub>2</sub> to a volume of ≈50&nbsp;μl. For chemical analyses sample aliquots (1&nbsp;μl) were injected by an AOC-20i autosampler system from Shimadzu, into split/splitless-injector which operated in splitless-mode at a temperature of 310°C. Either a GCMSQP2020 gas chromatography/mass-spectrometry system or a GC-2010PLUS with a flame-ionization detector (all Shimadzu, Kyōto, Japan) both equipped with a ZB-5MS fused silica capillary column (30&nbsp;m x 0.25&nbsp;mm ID, df&nbsp;= 0.25&nbsp;μm) from Phenomenex (Torrance, CA, USA) were used. Helium was used as the carrier-gas with a constant flow rate of 2.13&nbsp;ml/min. The chromatographic conditions were as follows: column temperature at the start was 40°C with a 1-minute hold after which the temperature was initially increased 50°C/min to 250°C and further increased 60°C/min to a final temperature of 300°C and held for 2&nbsp;minutes. For the MS, electron impact ionization spectra were recorded at 70 eV ion source voltage, with a scan rate of 0.2 scans/sec from m/z 40 to 450. The ion source of the mass spectrometer and the transfer line were kept at 230°C and 320°C, respectively. For the FID, detector temperature was kept at&nbsp;350°C. For quantification of the endpoint data (in ng) we used synthetic 1,4-BQ, 2-methyl-1,4-BQ (Sigma-Aldrich) and freshly synthesized 2-methoxy-3-methyl-1,4-BQ (see below).</div></section><section id="sec-4-3-23"><h4>Rheological measurements</h4><div id="p0430" role="paragraph">To measure the surface-coating ability and extensional viscosity of different artificial mixtures of synthetic secretions, we used dripping-onto-substrate (DoS) rheology. In contrast to conventional rheology which requires sample sizes on the order of milliliters, a DoS measurement can be accomplished with microliters. The instrument comprises a controlled dispensing system, a nozzle and a high-speed camera (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Dinic et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Dinic, J. ∙ Jimenez, L.N. ∙ Sharma, V.</div><div><strong>Pinch-off dynamics and dripping-onto-substrate (DoS) rheometry of complex fluids</strong></div><div><em>Lab Chip.</em> 2017; <strong>17</strong>:460-473</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DinicL.N.JimenezV.SharmaPinch-off+dynamics+and+dripping-onto-substrate+%28DoS%29+rheometry+of+complex+fluidsLab+Chip172017460473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Marshall et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Marshall, K.A. ∙ Liedtke, A.M. ∙ Todt, A.H. ...</div><div><strong>Extensional rheometry with a handheld mobile device</strong></div><div><em>Exp. Fluids.</em> 2017; <strong>58</strong>:69</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.A.MarshallA.M.LiedtkeA.H.TodtT.W.WalkerExtensional+rheometry+with+a+handheld+mobile+deviceExp.+Fluids58201769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref125" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125" aria-controls="sref125">Rosello et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Rosello, M. ∙ Sur, S. ∙ Barbet, B. ...</div><div><strong>Dripping-onto-substrate capillary breakup extensional rheometry of low-viscosity printing inks</strong></div><div><em>J.&nbsp;Non-Newton. Fluid.</em> 2019; <strong>266</strong>:160-170</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.RoselloS.SurB.BarbetJ.P.RothsteinDripping-onto-substrate+capillary+breakup+extensional+rheometry+of+low-viscosity+printing+inksJ.%C2%A0Non-Newton.+Fluid2662019160170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref137" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137" aria-controls="sref137">Sousa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref137"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>Sousa, P.C. ∙ Vega, E.J. ∙ Sousa, R.G. ...</div><div><strong>Measurement of relaxation times in extensional flow of weakly viscoelastic polymer solutions</strong></div><div><em>Rheol. Acta.</em> 2017; <strong>56</strong>:11-20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.C.SousaE.J.VegaR.G.SousaJ.M.MontaneroM.A.AlvesMeasurement+of+relaxation+times+in+extensional+flow+of+weakly+viscoelastic+polymer+solutionsRheol.+Acta5620171120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mixtures were prepared using 1,4-BQ, 2-methyl-1,4-BQ, undecane, isopropyl decanoate, ethyl decanoate and ethyl dodecanoate (all Sigma Aldrich). 2-Methoxy-3-methyl-1,4-BQ was synthesized as follows: 2&nbsp;mmol 2-Methoxy-3-methyl-1,4-hydroquinone (Sigma Aldrich) and 2&nbsp;mmol NaIO<sub>4</sub> were mixed in a 50&nbsp;mL round-bottom flask and 10&nbsp;mL CH<sub>2</sub>Cl<sub>2</sub> and 5&nbsp;mL of water were then added. The biphasic mixture was stirred at room temperature for 10&nbsp;min and the layers were subsequently separated and then the aqueous layer was washed with 10&nbsp;mL CH<sub>2</sub>Cl<sub>2</sub>. All organic fractions were combined, dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated <i>in vacuo</i>.</div><div id="p0435" role="paragraph">The all-compound mixture mimicking the beetles’ secretion was prepared by mixing 8&nbsp;mg 1,4-benzoquinone, 18&nbsp;mg 2-methyl-1,4-benzoquinone, 10&nbsp;mg 2-methoxy-3-methyl-1,4-benzoquinone, 47&nbsp;mg undecane, 7&nbsp;mg isopropyl decanoate, 13&nbsp;mg ethyl decanoate and 7&nbsp;mg ethyl dodecanoate. For some experimental mixtures, undecane was replaced by tridecane, pentadecane, heptadecane or heptacosane (all Sigma Aldrich). The ratios of compounds in all mixtures always represented the average ratios of the respective compounds in <i>Dalotia</i>’s gland secretion (<a id="crosref2290" href="#fig1">Figure&nbsp;1</a>D). Mixtures were dispensed at a rate of 0.02&nbsp;ml/min through the nozzle with an inner diameter of 0.413&nbsp;mm and outer diameter of 0.718&nbsp;mm onto an aluminum substrate a distance of 2.5&nbsp;mm below the nozzle. The fluid transitions from the nozzle to substrate via an unstable liquid bridge. The behavior of this liquid bridge was captured by the camera at 20,000 to 40,000 frames per second and analyzed to produce rheological data on the fluid (CaltechData: <a id="intref0605" href="https://data.caltech.edu/records/1905" target="_blank">https://data.caltech.edu/records/1905</a> and <a id="intref0610" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). Each measurement was repeated at least 3 times and the videos were processed using Python 3. Briefly, the videos were converted to a series of binary images and the minimum diameter at each frame was recorded. The code and parameters for each video are available in the supplement (CaltechData: <a id="intref0615" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). This yielded a dataset of diameter and time which was fit to an equation describing power-law fluids:<div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartFraction upper R Over upper R 0 EndFraction equals upper Y left parenthesis t Subscript p Baseline minus t right parenthesis Superscript n" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="(20 (4 0 (3 1 2)) 5 (19 6 18 (17 (15 7 (13 (10 8 9) 11 12) 14) 16)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="20" data-semantic-children="4,19" data-semantic-content="5" data-semantic-owns="4 5 19" data-semantic-speech="StartFraction upper R Over upper R 0 EndFraction equals upper Y left parenthesis t Subscript p Baseline minus t right parenthesis Superscript n"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="4" data-semantic-children="0,3" data-semantic-parent="20" data-semantic-owns="0 3"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-msub size="s" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D445">𝑅</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mn size="s" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn></mjx-script></mjx-msub></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="20" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-children="6,17" data-semantic-content="18" data-semantic-parent="20" data-semantic-owns="6 18 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="19"><mjx-c class="mjx-c1D44C">𝑌</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="leftright" data-semantic-id="17" data-semantic-children="15,16" data-semantic-parent="19" data-semantic-owns="15 16"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="15" data-semantic-children="13" data-semantic-content="7,14" data-semantic-parent="17" data-semantic-owns="7 13 14"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-children="10,12" data-semantic-content="11" data-semantic-parent="15" data-semantic-owns="10 11 12"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="13" data-semantic-owns="8 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="14" data-semantic-parent="15" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-script style="vertical-align: 0.475em;"><mjx-mi size="s" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="17"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-script></mjx-msup></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>Here, <i>R</i><sub><i>0</i></sub> is 0.359&nbsp;mm, <i>n</i> is the flow behavior index, and <i>t</i><sub><i>p</i></sub> is the pinch-off time, the moment the liquid bridge separates from the substrate. This equation was solved to estimate the elongational-flow consistency index Y (1/s<sup>n</sup>) and the flow behavior index n. Y describes what happens if a flow in imposed on fluid, while n describes how Y changes as the flow rate changes. Y decreases as the ratio of surface tension to viscosity decreases. When n&nbsp;= 1, mixtures behave as Newtonian fluids (no change with flow rate) and when n≠ 1 they behave as non-Newtonian fluid or power law fluids. For n&nbsp;&lt; 1, fluid flows more easily (shear thinning) as the flow rate increases. The closer n is to 1, the weaker is the dependence on flow rate. For n &gt; 1, the fluid more strongly resists flowing (shear thickening). Fluids with both low Y and low n possess higher extensional viscosity (EV). Furthermore, previous work has shown Y&nbsp;= φ(n)σ/K (<span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Doshi et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Doshi, P. ∙ Suryo, R. ∙ Yildirim, O.E. ...</div><div><strong>Scaling in pinch-off of generalized Newtonian fluids</strong></div><div><em>J Non-Newton Fluid.</em> 2003; <strong>113</strong>:1-27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.DoshiR.SuryoO.E.YildirimG.H.McKinleyO.A.BasaranScaling+in+pinch-off+of+generalized+Newtonian+fluidsJ+Non-Newton+Fluid1132003127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This is the product of the shape factor, φ(n), and surface tension divided by the power-law flow consistency index (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Blair et&nbsp;al., 1939</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Blair, G.W.S. ∙ Hening, J.C. ∙ Wagstaff, A.</div><div><strong>The Flow of Cream through Narrow Glass Tubes</strong></div><div><em>J.&nbsp;Phys. Chem.</em> 1939; <strong>43</strong>:853-864</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.W.S.BlairJ.C.HeningA.WagstaffThe+Flow+of+Cream+through+Narrow+Glass+TubesJ.%C2%A0Phys.+Chem.431939853864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The data fit this power-law function from the last frame with R/R<sub>0</sub>&nbsp;= 1 to t&nbsp;= t<sub>p</sub>, with 0 to 5 datapoints (a maximum of ∼125&nbsp;μs) excluded near t<sub>p</sub> because during this period the fluid deviates from the power-law behavior. We solved the previous equation to get K/σ which is the ratio of the flow consistency index and surface tension. We interpret this ratio as a metric of how a fluid flows over a surface at a given surface tension. K/σ can therefore be used as a proxy to describe the ability of a mixture to coat a surface, like the cuticle of other insects. We refer to this metric as the surface coating ability (SCA).</div></section></section><section id="sec-4-4"><h3>Data analysis and visualization software</h3><div id="p0440" role="paragraph">Analysis and visualization of transcriptomic data were performed using Python 3.0 (<span class="dropBlock reference-citations"><a id="body-ref-oref0020-2" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="oref0020-2" aria-controls="oref0020-2">Van Rossum and Drake, 2000</a><div class="dropBlock__holder" data-db-target-of="oref0020-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div>Van Rossum, G., &amp; Drake, F. L. (2000). <i>Python reference manual</i>. (iUniverse).</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Van+Rossum%2C+G.%2C+%26+Drake%2C+F.+L.+%282000%29.+Python+reference+manual.+%28iUniverse%29." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and R v.3.6.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref123-3" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123-3" aria-controls="sref123-3">R Core Team, 2018</a><div class="dropBlock__holder" data-db-target-of="sref123-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing. R Foundation for Statistical Computing</strong></div><div>2018</div><div><a id="intref0650" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing.+R+Foundation+for+Statistical+Computing2018https%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) assisted by JupyterLab (<span class="dropBlock reference-citations"><a id="body-ref-oref0010-2" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="oref0010-2" aria-controls="oref0010-2">Kluyver et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="oref0010-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div>Kluyver, T., Ragan-Kelley, B., Pérez, F., Granger, B.E., Bussonnier, M., Frederic, J., Kelley, K., Hamrick, J., Grout, J., Corlay, S., et al. (2016). <i>Jupyter Notebooks-a publishing format for reproducible computational workflows. In proceedings of the 20th International Conference on Electronic Publishing.</i> pp. 87-90.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Kluyver%2C+T.%2C+Ragan-Kelley%2C+B.%2C+P%C3%A9rez%2C+F.%2C+Granger%2C+B.E.%2C+Bussonnier%2C+M.%2C+Frederic%2C+J.%2C+Kelley%2C+K.%2C+Hamrick%2C+J.%2C+Grout%2C+J.%2C+Corlay%2C+S.%2C+et+al.+%282016%29.+Jupyter+Notebooks-a+publishing+format+for+reproducible+computational+workflows.+In+proceedings+of+the+20th+International+Conference+on+Electronic+Publishing.+pp.+87-90." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The following R packages were used: AnnotationForge (<span class="dropBlock reference-citations"><a id="body-ref-sref32-3" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32-3" aria-controls="sref32-3">Carlson and Pages, 2019</a><div class="dropBlock__holder" data-db-target-of="sref32-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Carlson, M. ∙ Pages, H.</div><div><strong>AnnotationForge: tools for building SQLite-based annotation data packages. R package version 1.36.0</strong></div><div>2019</div><div><a id="intref0635" href="https://bioconductor.org/packages/AnnotationForge" target="_blank">https://bioconductor.org/packages/AnnotationForge</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.CarlsonH.PagesAnnotationForge%3A+tools+for+building+SQLite-based+annotation+data+packages.+R+package+version+1.36.02019https%3A%2F%2Fbioconductor.org%2Fpackages%2FAnnotationForge" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), AnnotationHub (<span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Morgan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Morgan, M. ∙ Carlson, M. ∙ Tenenbaum, D. ...</div><div><strong>Package AnnotationHub</strong></div><div>2019</div><div><a id="intref0640" href="https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MorganM.CarlsonD.TenenbaumS.AroraPackage+AnnotationHub2019https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FAnnotationHub.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), biomart (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Durinck et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Durinck, S. ∙ Spellman, P.T. ∙ Birney, E. ...</div><div><strong>Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:1184-1191</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.DurinckP.T.SpellmanE.BirneyW.HuberMapping+identifiers+for+the+integration+of+genomic+datasets+with+the+R%2FBioconductor+package+biomaRtNat.+Protoc.4200911841191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) boot (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Canty and Ripley, 2017</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Canty, A. ∙ Ripley, B.</div><div><strong>boot: Bootstrap R (S-Plus) functions</strong></div><div><em>R Package Version.</em> 2017; <strong>1</strong>:3-20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CantyB.Ripleyboot%3A+Bootstrap+R+%28S-Plus%29+functionsR+Package+Version12017320" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), BUSpaRse (<span class="dropBlock reference-citations"><a id="body-ref-sref101-3" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101-3" aria-controls="sref101-3">Moses and Pachter, 2021</a><div class="dropBlock__holder" data-db-target-of="sref101-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Moses, L. ∙ Pachter, L.</div><div><strong>BUSpaRse: kallisto | bustools R utilities, R Package Version 1.4.2</strong></div><div>2021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.MosesL.PachterBUSpaRse%3A+kallisto+%7C+bustools+R+utilities%2C+R+Package+Version+1.4.22021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) car (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Fox et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Fox, J. ∙ Weisberg, S. ∙ Adler, D. ...</div><div><strong>Package ‘car’</strong></div><div>R Foundation for Statistical Computing, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxS.WeisbergD.AdlerD.BatesG.Baud-BovyS.EllisonD.FirthM.FriendlyG.GorjancS.GravesPackage+%E2%80%98car%E2%80%992012R+Foundation+for+Statistical+Computing" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), clusterProfiler (<span class="dropBlock reference-citations"><a id="body-ref-sref160-4" href="#" role="doc-biblioref" data-xml-rid="bib160" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160-4" aria-controls="sref160-4">Yu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref160-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Yu, G. ∙ Wang, L.-G. ∙ Han, Y. ...</div><div><strong>clusterProfiler: an R package for comparing biological themes among gene clusters</strong></div><div><em>OMICS.</em> 2012; <strong>16</strong>:284-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.YuL.-G.WangY.HanQ.-Y.HeclusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clustersOMICS162012284287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), cowplot (<span class="dropBlock reference-citations"><a id="body-ref-sref159-2" href="#" role="doc-biblioref" data-xml-rid="bib159" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref159-2" aria-controls="sref159-2">Wilke, 2017</a><div class="dropBlock__holder" data-db-target-of="sref159-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Wilke, C.</div><div><strong>Package ‘cowplot’: Streamlined Plot Theme and Plot Annotations for ‘ggplot2.’ CRAN</strong></div><div>2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.WilkePackage+%E2%80%98cowplot%E2%80%99%3A+Streamlined+Plot+Theme+and+Plot+Annotations+for+%E2%80%98ggplot2.%E2%80%99+CRAN2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), data.table (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Dowle et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Dowle, M. ∙ Srinivasan, A. ∙ Gorecki, J. ...</div><div><strong>Package ‘data. table.’ Extension of ‘data</strong></div><div>Frame, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DowleA.SrinivasanJ.GoreckiM.ChiricoP.StetsenkoT.ShortS.LianoglouE.AntonyanM.BonschH.ParsonagePackage+%E2%80%98data.+table.%E2%80%99+Extension+of+%E2%80%98data2019Frame" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), dbplyr (<span class="dropBlock reference-citations"><a id="body-ref-sref157-2" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157-2" aria-controls="sref157-2">Wickham and Ruiz, 2019</a><div class="dropBlock__holder" data-db-target-of="sref157-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>Wickham, H. ∙ Ruiz, E.</div><div><strong>dbplyr: A “dplyr” Back End for Databases. R&nbsp;package Version 1.4. 0</strong></div><div>2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamE.Ruizdbplyr%3A+A+%E2%80%9Cdplyr%E2%80%9D+Back+End+for+Databases.+R%C2%A0package+Version+1.4.+02019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), DropletUtils (<span class="dropBlock reference-citations"><a id="body-ref-sref87-3" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-3" aria-controls="sref87-3">Lun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref87-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ..., 1st Human Cell Atlas Jamboree</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.L.LunS.RiesenfeldT.AndrewsT.P.DaoT.GomesJ.C.Marioni1st+Human+Cell+Atlas+JamboreeEmptyDrops%3A+distinguishing+cells+from+empty+droplets+in+droplet-based+single-cell+RNA+sequencing+dataGenome+Biol.20201963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) effects (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Fox, 2003</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Fox, J.</div><div><strong>Effect displays in R for generalised linear models</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2003; <strong>8</strong>:1-27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxEffect+displays+in+R+for+generalised+linear+modelsJ.%C2%A0Stat.+Softw.82003127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), ggplot2 (<span class="dropBlock reference-citations"><a id="body-ref-sref156-2" href="#" role="doc-biblioref" data-xml-rid="bib156" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref156-2" aria-controls="sref156-2">Wickham, 2011</a><div class="dropBlock__holder" data-db-target-of="sref156-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Wickham, H.</div><div><strong>ggplot2</strong></div><div><em>Wiley Interdiscip. Rev. Comput. Stat.</em> 2011; <strong>3</strong>:180-185</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.Wickhamggplot2Wiley+Interdiscip.+Rev.+Comput.+Stat.32011180185" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), lattice (<span class="dropBlock reference-citations"><a id="body-ref-sref127-2" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-2" aria-controls="sref127-2">Sarkar, 2008</a><div class="dropBlock__holder" data-db-target-of="sref127-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Sarkar, D.</div><div><strong>Lattice: Multivariate Data Visualization with R</strong></div><div>Springer Science &amp; Business Media, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.SarkarLattice%3A+Multivariate+Data+Visualization+with+R2008Springer+Science+%26+Business+Media" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), lme4 (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Bates et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Bates, D. ∙ Sarkar, D. ∙ Bates, M.D. ...</div><div><strong>The lme4 package</strong></div><div><em>R Package Version.</em> 2007; <strong>2</strong>:74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.BatesD.SarkarM.D.BatesL.MatrixThe+lme4+packageR+Package+Version2200774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), MASS (<span class="dropBlock reference-citations"><a id="body-ref-sref122-2" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-2" aria-controls="sref122-2">Ripley et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref122-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Ripley, B. ∙ Venables, B. ∙ Bates, D.M. ...</div><div><strong>Package ‘mass.’</strong></div><div><em>Cran R.</em> 2013; <strong>538</strong>:113-120</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.RipleyB.VenablesD.M.BatesK.HornikA.GebhardtD.FirthM.B.RipleyPackage+%E2%80%98mass.%E2%80%99Cran+R5382013113120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) Matrix (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Bates and Maechler, 2010</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bates, D. ∙ Maechler, M.</div><div><strong>Matrix: sparse and dense matrix classes and methods. R Package Version</strong></div><div>2010</div><div><a id="intref0630" href="https://cran.r-project.org/web/packages/Matrix/index.html" target="_blank">https://cran.r-project.org/web/packages/Matrix/index.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.BatesM.MaechlerMatrix%3A+sparse+and+dense+matrix+classes+and+methods.+R+Package+Version2010https%3A%2F%2Fcran.r-project.org%2Fweb%2Fpackages%2FMatrix%2Findex.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), nlme (<span class="dropBlock reference-citations"><a id="body-ref-sref114-2" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-2" aria-controls="sref114-2">Pinheiro and Bates, 2007</a><div class="dropBlock__holder" data-db-target-of="sref114-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Pinheiro, J. ∙ Bates, D.</div><div><strong>Linear and nonlinear mixed effects models</strong></div><div><em>R Package Version.</em> 2007; <strong>3</strong>:1-89</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PinheiroD.BatesLinear+and+nonlinear+mixed+effects+modelsR+Package+Version32007189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), randomForest (<span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Liaw and Wiener, 2002</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Liaw, A. ∙ Wiener, M.</div><div><strong>Classification and regression by randomForest</strong></div><div><em>R News.</em> 2002; <strong>2/3</strong>:18-22</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.LiawM.WienerClassification+and+regression+by+randomForestR+News2%2F320021822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Seurat v3.1 (<span class="dropBlock reference-citations"><a id="body-ref-sref142-4" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142-4" aria-controls="sref142-4">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref142-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.StuartA.ButlerP.HoffmanC.HafemeisterE.PapalexiW.M.Mauck3rdY.HaoM.StoeckiusP.SmibertR.SatijaComprehensive+Integration+of+Single-Cell+DataCell177201918881902.e21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) sleuth (<span class="dropBlock reference-citations"><a id="body-ref-sref112-3" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112-3" aria-controls="sref112-3">Pimentel et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref112-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Pimentel, H. ∙ Bray, N.L. ∙ Puente, S. ...</div><div><strong>Differential analysis of RNA-seq incorporating quantification uncertainty</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:687-690</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.PimentelN.L.BrayS.PuenteP.MelstedL.PachterDifferential+analysis+of+RNA-seq+incorporating+quantification+uncertaintyNat.+Methods142017687690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), tidyverse (<span class="dropBlock reference-citations"><a id="body-ref-sref158-2" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158-2" aria-controls="sref158-2">Wickham et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref158-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the Tidyverse</strong></div><div><em>J.&nbsp;Open Source Softw.</em> 2019; <strong>4</strong>:1686</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamM.AverickJ.BryanW.ChangL.D.McGowanR.Fran%C3%A7oisG.GrolemundA.HayesL.HenryJ.HesterWelcome+to+the+TidyverseJ.%C2%A0Open+Source+Softw.420191686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), vcd (<span class="dropBlock reference-citations"><a id="body-ref-sref98-2" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-2" aria-controls="sref98-2">Meyer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref98-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Meyer, D. ∙ Zeileis, A. ∙ Hornik, K. ...</div><div><strong>Package ‘vcd.’ Visualizing Categorical Data. R Package Version 1–4</strong></div><div>2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.MeyerA.ZeileisK.HornikF.GerberM.FriendlyM.D.MeyerPackage+%E2%80%98vcd.%E2%80%99+Visualizing+Categorical+Data.+R+Package+Version+1%E2%80%9342020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0445" role="paragraph">We thank Y. Kishi, T. Naragon, J. Wagner, M. Spero, C. VanDrisse, the Bioinformatics Resource Center, and the Single Cell Profiling and Engineering Center (SPEC) in the Beckman Institute at Caltech for assistance with this work. We are grateful to M. Bronner, M. Dickinson and four anonymous reviewers for constructive feedback. A.B. is a Simons Fellow of the Life Sciences Research Foundation (LSRF). This work was supported by a Rita Allen Foundation Scholars Award, an Alfred P. Sloan Research Fellowship, a Shurl and Kay Curci Foundation grant, a Klingenstein-Simons Fellowship Award, and a National Science Foundation CAREER award (NSF 2047472 to J.P.).</div><section id="sec-5"><h3>Author contributions</h3><div id="p0450" role="paragraph">Conceptualization, J.P. and A.B.; Methodology, J.P. and A.B.; Investigation, A.B., J.M.B., R.W.L., S.A.K., M.Y., and J.P.; Formal Analysis, A.B.; Data Curation, A.B.; Writing – Original Draft, J.P. and A.B.; Writing – Review &amp; Editing, J.P., A.B., and S.A.K.; Supervision, J.P.; Project Administration, J.P.; Funding Acquisition, A.B. and J.P.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0455" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (7)</h2><div id="p0465" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/5bd46517-8fff-4eff-b676-180daa3b5184/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (502.68 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Data S1. Phylogeny of α-esterases, related to Figure&nbsp;4</div><div id="espara0010" role="paragraph">Maximum likelihood tree made with LG+F+R7 model, including the six transcripts from <i>Dalotia</i>. The topology shows that <i>TG-αEst</i> belongs to an Aleocharinae-specific clade. Violin dot plots show expression of each <i>Dalotia</i> α-esterase across abdominal cell types.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/edb53133-805a-4e39-8bbe-456b0d643998/mmc2.pdf" download="mmc2.pdf" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">PDF (13.29 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Data S2. Oenocyte CHC pathway, related to Figure&nbsp;4</div><div id="espara0015" role="paragraph">A-D: UMAPs showing expression of <i>O-CYP4G</i> (A), <i>O-FASN</i>s (B), <i>O-FAR</i>s (C) and <i>O-ELO</i>s (D) in oenocytes (insets show the oenocyte cell cluster magnified). Note that for <i>FASN</i>s, <i>FAR</i>s and <i>ELO</i>s, multiple copies are expressed, and the mean expression of all copies is shown. E-G: Levels of cuticular hydrocarbons (CHCs) extracted from beetles injected with dsRNA against <i>GFP</i> (control) and <i>O-CYP4G</i> (E), <i>O-FAS-1220</i> (F), <i>O-FAR-1142</i> and <i>O-FAR-934</i> (G) Silencing <i>O-CYP4G</i> or <i>O-FAS-1220</i> results in near-complete loss of CHCs on <i>Dalotia</i>’s body surface. Silencing <i>O-FAR-1142</i> did not affect CHC levels, while <i>O-FAR-934</i> RNAi resulted in CHC loss. H, I: Silencing expression of <i>O-ELO-708</i> and <i>O-ELO-804</i> did not affect CHC levels (H) but shifted the composition of CHCs found on <i>Dalotia</i>’s cuticle surface toward shorter-chained CHCs (I). Significance in all panels was assessed using Mann-Whitney-U tests or permutative t tests (<sup>∗∗∗</sup>&nbsp;= p ≤ 0.001; ns&nbsp;= p ≥ 0.05). J: Maximum-likelihood tree of fatty acyl-CoA reductase (FAR) transcripts in <i>Dalotia</i> and other insects (best model: LG+F+R6). The tree was rooted with the <i>Drosophila</i> FAR encoded by <i>waterproof</i>. <i>Dalotia</i> sequences in the tree are color-coded by their tissue specific expression. K: Violin dot plot showing tissue-specific expression of the <i>en</i> in the <i>Dalotia</i> abdominal cell atlas. Expression in <i>en</i> in adults is largely restricted to cuticular cells and the solvent cells. L: Immunohistochemistry shows strong labeling of solvent cell nuclei with En antibody.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/b7ebe1b3-1bbe-4c2f-b548-77ccf207d8cd/mmc3.pdf" download="mmc3.pdf" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">PDF (3.56 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Data S3. Diagnostic plots for K selection in cNMF, related to Figure&nbsp;5</div><div id="espara0020" role="paragraph">A: Clustergram showing the clustered NMF components for K&nbsp;= 20, combined across 250 replicates, after filtering (left). Number of cNMF components (K) against stability (blue) i.e., the silhouette score of the clustering, and Frobenius error (red) for a data set using 1000 overdispersed genes. B: Clustergram showing the clustered NMF components for K&nbsp;= 20, combined across 250 replicates, after filtering (left). Number of cNMF components (K) against stability (blue) i.e., the silhouette score of the clustering, and Frobenius error (red) for a data set using 3000 overdispersed genes. C: Clustergram showing the clustered NMF components for K&nbsp;= 20, combined across 250 replicates, after filtering (left). Number of cNMF components (K) against stability (blue) i.e., the silhouette score of the clustering, and Frobenius error (red) for a data set using 5000 overdispersed genes. D: Plot of the Log<sub>10</sub> proportion of variance explained per principal component for a data set using 3000 overdispersed genes. The K selection used in the manuscript is indicated by a turquoise box.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/fbe3b1d4-6a20-4e61-b011-85eb6cd21e19/mmc4.pdf" download="mmc4.pdf" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">PDF (1.28 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Data S4. GEP9 composition, related to Figure&nbsp;5</div><div id="espara0025" role="paragraph">A: List of all genes with a z-score &gt; 0.002, denoting a strong association with GEP9. A majority of genes is either related to fatty acid synthesis or modification of fatty acid derivatives. B: Gene ontology (GO) term enrichment network of the ‘Biological Process’ category based on the list of genes in A. Disc color corresponds to the significance (as assessed by Fisher’s exact test) of the respective GO-terms, while the thickness of the gray lines represents the semantic similarities between the GO-terms. Network created using REViGO. C: MonaGO plot of the genes listed in A. The inner cord plot shows which genes of the GO-term ‘fatty acid metabolism’ are shared with other GO-terms. The inner ring displays the significance (as assessed by Fisher’s exact test) of enrichment of each GO-term, while the outer cluster dendrogram is based on the pairwise similarity (Jaccard distance) of each GO-term. Collapsed clusters are depicted as red dots. BP&nbsp;= biological process, MF&nbsp;= molecular function. Plot created with MonaGO. D: Number of genes for each enriched GO-term (as assessed by Fisher’s exact test) for all three GO-categories based on the top 500 genes (by z-score) of GEP9.</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/ead1fade-aad8-47aa-b2ca-03cd3a4dda3f/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (90.09 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Data S5. RNAi survival assay without ants, extensional viscosities of synthetic secretions, and lag times of bacterial growth, related to Figure&nbsp;7</div><div id="espara0030" role="paragraph">A: Survival of adult <i>Dalotia</i> after injection of dsRNA against <i>MFASN</i>, <i>dmd</i> and <i>GFP</i> (control) over a period of twelve consecutive days without ants. The y axis shows survival (%), the x axis notes the days, and the red block shows the days important to the ant survival assay. Our data indicate that there is no significant change in survival during the experimental time window. B: Extensional viscosities (EV) of synthetic secretions. Boxes in gray show data for the elongational-flow consistency index Y (left axis), while the white boxes correspond to the flow behavior index n (right axis). The medians for Y and n of the wild-type mixtures are shown as horizontal lines. Lower values of both Y and n together correspond to higher EV. Different letters indicate significant difference in Y or n across test mixtures as assessed by a Tukey HSD posthoc test following a one-way ANOVA. C: Lag times (λ in h) of <i>P.&nbsp;fluorescens</i> culture growth after inoculation with <i>Dalotia</i> gland compounds of different mixtures. Different letters denote significant differences. For details regarding the statistical analysis see Table S1.</div></div></div><div id="mmc6" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/88e18132-b669-46f7-975f-49bbd7e1ae3a/mmc6.pdf" download="mmc6.pdf" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">PDF (501.71 KB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Data S6. <i>Dalotia</i> laccase <i>in&nbsp;vitro</i> binding assays, related to Figure&nbsp;6</div><div id="espara0035" role="paragraph">A: Scheme showing reaction of ABTS with laccase enzymes used to measure absorption in part B. One tertiary N of ABTS (λ<sub>max</sub>&nbsp;= 342&nbsp;nm) is oxidized by laccase, forming the oxidized N•+ radical with a λ<sub>max</sub> at 420&nbsp;nm while reducing Cu<sup>+2</sup> to Cu<sup>1+</sup>. B: Absorption measurements at λ&nbsp;= 420&nbsp;nm for Dmd, Lac1 and Lac2 over 180&nbsp;s using the Laccase-ABTS binding assay confirms that all three <i>in&nbsp;vitro</i>-expressed laccase enzymes are vital and bind to a control substrate. C: Monitoring absorption at λ&nbsp;= 260&nbsp;nm for the <i>in&nbsp;vitro</i> reaction of Dmd with 1,4-HQ, 2-methyl-1,4-HQ and 2-methoxy-3-methyl-1,4-HQ over 600&nbsp;s confirms that Dmd catalyzes oxidation of all three substrates to the respective BQs.</div></div></div><div id="mmc7" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.11.014/attachment/039bb8fb-9d9f-4188-a1c7-78d0dd8466f1/mmc7.xlsx" download="mmc7.xlsx" aria-labelledby="mmc7-heading mmc7-link"><span id="mmc7-link">Spreadsheet (16.74 KB)</span></a></div><div class="core-description"><div id="mmc7-heading" class="heading">Table S1. Statistical analyses of all experimental data presented in the main and supplementary figures, related to Figures 1, 2, 4, 6, and 7</div><div id="espara0040" role="paragraph">For each analysis the comparison is described, sample sizes are given, and statistical methods are named; p values or other statistical values are listed for each test.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">Raw sequence reads related to this manuscript have been deposited on NCBI under the BioProject ‘RNaseq (10x and SMARTseq) of the tergal gland of <i>Dalotia coriaria</i>’ (BioProject: PRJNA707010) and ‘Genome and transcriptome sequencing of rove beetles (Staphylinidae)’ (BioProject: PRJNA764224). All other data were uploaded to CaltechData: <a id="intref0010" href="https://data.caltech.edu/records/1915" target="_blank">https://data.caltech.edu/records/1915</a> (processed scRNaseq 10x data), <a id="intref0015" href="https://data.caltech.edu/records/1900" target="_blank">https://data.caltech.edu/records/1900</a> (processed SMARTseq data), <a id="intref0020" href="https://data.caltech.edu/records/1905" target="_blank">https://data.caltech.edu/records/1905</a> (raw rheology video data), <a id="intref0025" href="https://data.caltech.edu/records/1914" target="_blank">https://data.caltech.edu/records/1914</a> (transcriptome data of <i>Dalotia</i>), <a id="intref0030" href="https://data.caltech.edu/records/1919" target="_blank">https://data.caltech.edu/records/1919</a> (genomes and transcriptomes of other rove beetles), <a id="intref0035" href="https://data.caltech.edu/records/1917" target="_blank">https://data.caltech.edu/records/1917</a> (RNAi experiments, survival assays, <i>in&nbsp;vitro</i> enzyme data), and <a id="intref0040" href="https://data.caltech.edu/records/1916" target="_blank">https://data.caltech.edu/records/1916</a> (alignment and tree fasta files). Detailed code for scRNaseq analyses with Seurat and cNMF; video analyses of rheology data; custom R scripts for SMARTseq analyses via sleuth, GOterm assignments and survival/toxicity data analyses can be found on CaltechData (<a id="intref0045" href="https://data.caltech.edu/records/1918" target="_blank">https://data.caltech.edu/records/1918</a>). All other statistical comparisons using ANOVAs, Kruskal-Wallis tests, U-tests and simple ordinations were done in Past 3.04 (<span class="dropBlock reference-citations"><a id="body-ref-sref63-3" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-3" aria-controls="sref63-3">Hammer et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref63-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Hammer, O. ∙ Harper, D.A. ∙ Ryan, P.D.</div><div><strong>PAST: Paleontological statistics software package for education and data analysis</strong></div><div><em>Palaeontol. Electronica.</em> 2001; <strong>4</strong>:9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.HammerD.A.HarperP.D.RyanPAST%3A+Paleontological+statistics+software+package+for+education+and+data+analysisPalaeontol.+Electronica420019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1" title="View in article">1.</a></div><div>Alexandrov, L.B. ∙ Nik-Zainal, S. ∙ Wedge, D.C. ...</div><div><strong>Deciphering signatures of mutational processes operative in human cancer</strong></div><div><em>Cell Rep.</em> 2013; <strong>3</strong>:246-259</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.B.AlexandrovS.Nik-ZainalD.C.WedgeP.J.CampbellM.R.StrattonDeciphering+signatures+of+mutational+processes+operative+in+human+cancerCell+Rep.32013246259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Altschul, S.F. ∙ Gish, W. ∙ Miller, W. ...</div><div><strong>Basic local alignment search tool</strong></div><div><em>J.&nbsp;Mol. Biol.</em> 1990; <strong>215</strong>:403-410</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.F.AltschulW.GishW.MillerE.W.MyersD.J.LipmanBasic+local+alignment+search+toolJ.%C2%A0Mol.+Biol.2151990403410" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3" title="View in article">3.</a></div><div>Anderson, M.J.</div><div><strong>A new method for non-parametric multivariate analysis of variance</strong></div><div><em>Austral Ecol.</em> 2001; <strong>26</strong>:32-46</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.J.AndersonA+new+method+for+non-parametric+multivariate+analysis+of+varianceAustral+Ecol.2620013246" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Andrews, S.</div><div><strong>FastQC: a quality control tool for high throughput sequence data</strong></div><div>2010</div><div><a id="intref0625" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+a+quality+control+tool+for+high+throughput+sequence+data2010https%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Arakane, Y. ∙ Muthukrishnan, S. ∙ Beeman, R.W. ...</div><div><strong>Laccase 2 is the phenoloxidase gene required for beetle cuticle tanning</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:11337-11342</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.ArakaneS.MuthukrishnanR.W.BeemanM.R.KanostK.J.KramerLaccase+2+is+the+phenoloxidase+gene+required+for+beetle+cuticle+tanningProc.+Natl.+Acad.+Sci.+USA10220051133711342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Arendt, D. ∙ Musser, J.M. ∙ Baker, C.V.H. ...</div><div><strong>The origin and evolution of cell types</strong></div><div><em>Nat. Rev. Genet.</em> 2016; <strong>17</strong>:744-757</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.ArendtJ.M.MusserC.V.H.BakerA.BergmanC.CepkoD.H.ErwinM.PavlicevG.SchlosserS.WidderM.D.LaubichlerG.P.WagnerThe+origin+and+evolution+of+cell+typesNat.+Rev.+Genet.172016744757" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Arrese, E.L. ∙ Soulages, J.L.</div><div><strong>Insect fat body: energy, metabolism, and regulation</strong></div><div><em>Annu. Rev. Entomol.</em> 2010; <strong>55</strong>:207-225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.L.ArreseJ.L.SoulagesInsect+fat+body%3A+energy%2C+metabolism%2C+and+regulationAnnu.+Rev.+Entomol.552010207225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Asano, T. ∙ Seto, Y. ∙ Hashimoto, K. ...</div><div><strong>Mini-review an insect-specific system for terrestrialization: Laccase-mediated cuticle formation</strong></div><div><em>Insect Biochem. Mol. Biol.</em> 2019; <strong>108</strong>:61-70</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.AsanoY.SetoK.HashimotoH.KurushimaMini-review+an+insect-specific+system+for+terrestrialization%3A+Laccase-mediated+cuticle+formationInsect+Biochem.+Mol.+Biol.10820196170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Babonis, L.S. ∙ Ryan, J.F. ∙ Enjolras, C. ...</div><div><strong>Genomic analysis of the tryptome reveals molecular mechanisms of gland cell evolution</strong></div><div><em>Evodevo.</em> 2019; <strong>10</strong>:23</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.S.BabonisJ.F.RyanC.EnjolrasM.Q.MartindaleGenomic+analysis+of+the+tryptome+reveals+molecular+mechanisms+of+gland+cell+evolutionEvodevo10201923" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Barrett, R.D.H. ∙ Hoekstra, H.E.</div><div><strong>Molecular spandrels: tests of adaptation at the genetic level</strong></div><div><em>Nat. Rev. Genet.</em> 2011; <strong>12</strong>:767-780</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.D.H.BarrettH.E.HoekstraMolecular+spandrels%3A+tests+of+adaptation+at+the+genetic+levelNat.+Rev.+Genet.122011767780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Bates, D. ∙ Maechler, M.</div><div><strong>Matrix: sparse and dense matrix classes and methods. R Package Version</strong></div><div>2010</div><div><a id="intref0630" href="https://cran.r-project.org/web/packages/Matrix/index.html" target="_blank">https://cran.r-project.org/web/packages/Matrix/index.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.BatesM.MaechlerMatrix%3A+sparse+and+dense+matrix+classes+and+methods.+R+Package+Version2010https%3A%2F%2Fcran.r-project.org%2Fweb%2Fpackages%2FMatrix%2Findex.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Bates, D. ∙ Sarkar, D. ∙ Bates, M.D. ...</div><div><strong>The lme4 package</strong></div><div><em>R Package Version.</em> 2007; <strong>2</strong>:74</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.BatesD.SarkarM.D.BatesL.MatrixThe+lme4+packageR+Package+Version2200774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.BechtL.McInnesJ.HealyC.-A.DutertreI.W.H.KwokL.G.NgF.GinhouxE.W.NewellDimensionality+reduction+for+visualizing+single-cell+data+using+UMAPNat.+Biotechnol.3720183844" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Benjamini, Y. ∙ Hochberg, Y.</div><div><strong>Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</strong></div><div><em>J R Stat Soc Series B Stat Methodol.</em> 1995; <strong>57</strong>:289-300</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.2517-6161.1995.tb02031.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Betz, O. ∙ Irmler, U. ∙ Klimaszewski, J.</div><div><strong>Biology of Rove Beetles (Staphylinidae)</strong></div><div>Springer, 2018</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.BetzU.IrmlerJ.KlimaszewskiBiology+of+Rove+Beetles+%28Staphylinidae%292018Springer" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16" title="View in article">16.</a></div><div>Billeter, J.-C. ∙ Atallah, J. ∙ Krupp, J.J. ...</div><div><strong>Specialized cells tag sexual and species identity in Drosophila melanogaster</strong></div><div><em>Nature.</em> 2009; <strong>461</strong>:987-991</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.-C.BilleterJ.AtallahJ.J.KruppJ.G.MillarJ.D.LevineSpecialized+cells+tag+sexual+and+species+identity+in+Drosophila+melanogasterNature4612009987991" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Blair, G.W.S. ∙ Hening, J.C. ∙ Wagstaff, A.</div><div><strong>The Flow of Cream through Narrow Glass Tubes</strong></div><div><em>J.&nbsp;Phys. Chem.</em> 1939; <strong>43</strong>:853-864</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.W.S.BlairJ.C.HeningA.WagstaffThe+Flow+of+Cream+through+Narrow+Glass+TubesJ.%C2%A0Phys.+Chem.431939853864" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Blomquist, G.J. ∙ Ginzel, M.D.</div><div><strong>Chemical Ecology, Biochemistry, and Molecular Biology of Insect Hydrocarbons</strong></div><div><em>Annu. Rev. Entomol.</em> 2021; <strong>66</strong>:45-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.J.BlomquistM.D.GinzelChemical+Ecology%2C+Biochemistry%2C+and+Molecular+Biology+of+Insect+HydrocarbonsAnnu.+Rev.+Entomol.6620214560" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Blum, M.</div><div><strong>Chemical Defenses of Arthropods</strong></div><div>Academic Press, 1981</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.BlumChemical+Defenses+of+Arthropods1981Academic+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Bourguignon, T. ∙ Šobotník, J. ∙ Brabcová, J. ...</div><div><strong>Molecular Mechanism of the Two-Component Suicidal Weapon of Neocapritermes taracua Old Workers</strong></div><div><em>Mol. Biol. Evol.</em> 2016; <strong>33</strong>:809-819</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.BourguignonJ.%C5%A0obotn%C3%ADkJ.Brabcov%C3%A1D.Sillam-Duss%C3%A8sA.Bu%C4%8DekJ.Krasulov%C3%A1B.Vytiskov%C3%A1Z.Demianov%C3%A1M.Mare%C5%A1Y.RoisinH.VogelMolecular+Mechanism+of+the+Two-Component+Suicidal+Weapon+of+Neocapritermes+taracua+Old+WorkersMol.+Biol.+Evol.332016809819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">21.</a></div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.L.BrayH.PimentelP.MelstedL.PachterNear-optimal+probabilistic+RNA-seq+quantificationNat.+Biotechnol.342016525527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Breiman, L.</div><div><strong>Random Forests</strong></div><div><em>Mach. Learn.</em> 2001; <strong>45</strong>:5-32</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.BreimanRandom+ForestsMach.+Learn.452001532" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Brückner, A. ∙ Heethoff, M.</div><div><strong>A chemo-ecologists’ practical guide to compositional data analysis</strong></div><div><em>Chemoecology.</em> 2017; <strong>27</strong>:33-46</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerM.HeethoffA+chemo-ecologists%E2%80%99+practical+guide+to+compositional+data+analysisChemoecology2720173346" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24-1" title="View in article">24.</a></div><div>Brückner, A. ∙ Parker, J.</div><div><strong>Molecular evolution of gland cell types and chemical interactions in animals</strong></div><div><em>J.&nbsp;Exp. Biol.</em> 2020; <strong>223</strong>:jeb211938</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerJ.ParkerMolecular+evolution+of+gland+cell+types+and+chemical+interactions+in+animalsJ.%C2%A0Exp.+Biol.223Pt%2C+Suppl+12020jeb211938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Brückner, A. ∙ Kaltenpoth, M. ∙ Heethoff, M.</div><div><strong>De novo biosynthesis of simple aromatic compounds by an arthropod (<i>Archegozetes longisetosus</i>)</strong></div><div><em>Proc. Biol. Sci.</em> 2020; <strong>287</strong>:20201429</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerM.KaltenpothM.HeethoffDe+novo+biosynthesis+of+simple+aromatic+compounds+by+an+arthropod+%28Archegozetes+longisetosus%29Proc.+Biol.+Sci.287202020201429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Brückner, A. ∙ Barnett, A.A. ∙ Antoshechkin, I.A. ...</div><div><strong>Molecular evolutionary trends and biosynthesis pathways in the Oribatida revealed by the genome of Archegozetes longisetosus</strong></div><div><em>Biorxiv.</em> 2021;</div><div>2020.12.10.420141</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Br%C3%BCcknerA.A.BarnettI.A.AntoshechkinS.A.KitchenMolecular+evolutionary+trends+and+biosynthesis+pathways+in+the+Oribatida+revealed+by+the+genome+of+Archegozetes+longisetosusBiorxiv20212020.12.10.420141" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>Brůna, T. ∙ Hoff, K.J. ∙ Lomsadze, A. ...</div><div><strong>BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database</strong></div><div><em>Nar. Genom. Bioinform.</em> 2021; <strong>3</strong>:lqaa108</div></div><div class="external-links"><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.11.014&amp;key=2-s2.0-85111209000" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.Br%C5%AFnaK.J.HoffA.LomsadzeM.StankeM.BorodovskyBRAKER2%3A+automatic+eukaryotic+genome+annotation+with+GeneMark-EP%2B+and+AUGUSTUS+supported+by+a+protein+databaseNar.+Genom.+Bioinform.32021lqaa108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28-1" title="View in article">28.</a></div><div>Buchfink, B. ∙ Xie, C. ∙ Huson, D.H.</div><div><strong>Fast and sensitive protein alignment using DIAMOND</strong></div><div><em>Nat. Methods.</em> 2015; <strong>12</strong>:59-60</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.BuchfinkC.XieD.H.HusonFast+and+sensitive+protein+alignment+using+DIAMONDNat.+Methods1220155960" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Butler, A. ∙ Hoffman, P. ∙ Smibert, P. ...</div><div><strong>Integrating single-cell transcriptomic data across different conditions, technologies, and species</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>36</strong>:411-420</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.ButlerP.HoffmanP.SmibertE.PapalexiR.SatijaIntegrating+single-cell+transcriptomic+data+across+different+conditions%2C+technologies%2C+and+speciesNat.+Biotechnol.362018411420" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">30.</a></div><div>Canty, A. ∙ Ripley, B.</div><div><strong>boot: Bootstrap R (S-Plus) functions</strong></div><div><em>R Package Version.</em> 2017; <strong>1</strong>:3-20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.CantyB.Ripleyboot%3A+Bootstrap+R+%28S-Plus%29+functionsR+Package+Version12017320" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Carcamo-Noriega, E.N. ∙ Sathyamoorthi, S. ∙ Banerjee, S. ...</div><div><strong>1,4-Benzoquinone antimicrobial agents against <i>Staphylococcus aureus</i> and <i>Mycobacterium tuberculosis</i> derived from scorpion venom</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:12642-12647</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.N.Carcamo-NoriegaS.SathyamoorthiS.BanerjeeE.GnanamaniM.Mendoza-TrujilloD.Mata-EspinosaR.Hern%C3%A1ndez-PandoJ.I.Veytia-BucheliL.D.PossaniR.N.Zare1%2C4-Benzoquinone+antimicrobial+agents+against+Staphylococcus+aureus+and+Mycobacterium+tuberculosis+derived+from+scorpion+venomProc.+Natl.+Acad.+Sci.+USA11620191264212647" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32-1" title="View in article">32.</a></div><div>Carlson, M. ∙ Pages, H.</div><div><strong>AnnotationForge: tools for building SQLite-based annotation data packages. R package version 1.36.0</strong></div><div>2019</div><div><a id="intref0635" href="https://bioconductor.org/packages/AnnotationForge" target="_blank">https://bioconductor.org/packages/AnnotationForge</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.CarlsonH.PagesAnnotationForge%3A+tools+for+building+SQLite-based+annotation+data+packages.+R+package+version+1.36.02019https%3A%2F%2Fbioconductor.org%2Fpackages%2FAnnotationForge" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">33.</a></div><div>Choi, H.M.T. ∙ Schwarzkopf, M. ∙ Fornace, M.E. ...</div><div><strong>Third-generation <i>in situ</i> hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust</strong></div><div><em>Development.</em> 2018; <strong>145</strong>:dev165753</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.M.T.ChoiM.SchwarzkopfM.E.FornaceA.AcharyaG.ArtavanisJ.StegmaierA.CunhaN.A.PierceThird-generation+in+situ+hybridization+chain+reaction%3A+multiplexed%2C+quantitative%2C+sensitive%2C+versatile%2C+robustDevelopment1452018dev165753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib161" class="citations"><div id="optG10oc3bwOi" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optG10oc3bwOi-1" title="View in article">34.</a></div><div>Clifford, R.J. ∙ Milillo, M. ∙ Prestwood, J. ...</div><div><strong>Detection of bacterial 16S rRNA and identification of four clinically important bacteria by real-time PCR</strong></div><div><em>PloS one.</em> 2012; <strong>7</strong>, e48558</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.J.CliffordM.MililloJ.PrestwoodR.QuinteroD.V.ZurawskiY.I.KwakP.E.WatermanE.P.LeshoP.Mc+GannDetection+of+bacterial+16S+rRNA+and+identification+of+four+clinically+important+bacteria+by+real-time+PCRPloS+one72012e48558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">35.</a></div><div>Crescente, J.M. ∙ Zavallo, D. ∙ Helguera, M. ...</div><div><strong>MITE Tracker: an accurate approach to identify miniature inverted-repeat transposable elements in large genomes</strong></div><div><em>BMC Bioinformatics.</em> 2018; <strong>19</strong>:348</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.CrescenteD.ZavalloM.HelgueraL.S.VanzettiMITE+Tracker%3A+an+accurate+approach+to+identify+miniature+inverted-repeat+transposable+elements+in+large+genomesBMC+Bioinformatics192018348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">36.</a></div><div>Darwin, C.</div><div><strong>On the Origin of Species by Means of Natural Selection, or, the Preservation of Favoured Races in the Struggle for Life</strong></div><div>John Murray, 1859</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.5962/bhl.title.82303" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5962%2Fbhl.title.82303" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">37.</a></div><div>Dawkins, R.</div><div><strong>The Blind Watchmaker: Why the Evidence of Evolution Reveals a Universe Without Design</strong></div><div>WW Norton &amp; Company, 1986</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.DawkinsThe+Blind+Watchmaker%3A+Why+the+Evidence+of+Evolution+Reveals+a+Universe+Without+Design1986WW+Norton+%26+Company" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">38.</a></div><div>Dettner, K.</div><div><strong>Isopropylesters as Wetting Agents From the Defensive Secretion of the Rove Beetle Coprophilus striatulus F. (Coleoptera, Staphylinidae)</strong></div><div><em>Insect Biochem.</em> 1984; <strong>14</strong>:383-390</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerIsopropylesters+as+Wetting+Agents+From+the+Defensive+Secretion+of+the+Rove+Beetle+Coprophilus+striatulus+F.+%28Coleoptera%2C+Staphylinidae%29Insect+Biochem.141984383390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38" title="View in article">39.</a></div><div>Dettner, K.</div><div><strong>Solvent-dependent Variability of Effectiveness of Quinone-defensive Systems of Oxytelinae Beetles (Coleoptera: Staphylinidae)</strong></div><div><em>Entomol. Gen.</em> 1991; <strong>15</strong>:275-292</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerSolvent-dependent+Variability+of+Effectiveness+of+Quinone-defensive+Systems+of+Oxytelinae+Beetles+%28Coleoptera%3A+Staphylinidae%29Entomol.+Gen.151991275292" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">40.</a></div><div>Dettner, K.</div><div><strong>Defensive secretions and exocrine glands in free-living staphylinid beetles—their bearing on phylogeny (Coleoptera: Staphylinidae)</strong></div><div><em>Biochem. Syst. Ecol.</em> 1993; <strong>21</strong>:143-162</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerDefensive+secretions+and+exocrine+glands+in+free-living+staphylinid+beetles%E2%80%94their+bearing+on+phylogeny+%28Coleoptera%3A+Staphylinidae%29Biochem.+Syst.+Ecol.211993143162" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">41.</a></div><div>Dettner, K.</div><div><strong>Dabbing and shooting of benzo- and naphthoquinone secretions: Defensive strategies of bark-inhabiting aleocharine (col.: Staphylinidae) and tenebrionid (col.: Tenebrionidae) beetle larvae</strong></div><div><em>J.&nbsp;Chem. Ecol.</em> 1993; <strong>19</strong>:1337-1354</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerDabbing+and+shooting+of+benzo-+and+naphthoquinone+secretions%3A+Defensive+strategies+of+bark-inhabiting+aleocharine+%28col.%3A+Staphylinidae%29+and+tenebrionid+%28col.%3A+Tenebrionidae%29+beetle+larvaeJ.%C2%A0Chem.+Ecol.19199313371354" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">42.</a></div><div>Dettner, K. ∙ Schwinger, G. ∙ Wunderle, P.</div><div><strong>Sticky secretion from two pairs of defensive glands of rove beetleDeleaster dichrous (Grav.) (Coleoptera: Staphylinidae)</strong></div><div><em>J.&nbsp;Chem. Ecol.</em> 1985; <strong>11</strong>:859-883</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.DettnerG.SchwingerP.WunderleSticky+secretion+from+two+pairs+of+defensive+glands+of+rove+beetleDeleaster+dichrous+%28Grav.%29+%28Coleoptera%3A+Staphylinidae%29J.%C2%A0Chem.+Ecol.111985859883" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">43.</a></div><div>Dinic, J. ∙ Jimenez, L.N. ∙ Sharma, V.</div><div><strong>Pinch-off dynamics and dripping-onto-substrate (DoS) rheometry of complex fluids</strong></div><div><em>Lab Chip.</em> 2017; <strong>17</strong>:460-473</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.DinicL.N.JimenezV.SharmaPinch-off+dynamics+and+dripping-onto-substrate+%28DoS%29+rheometry+of+complex+fluidsLab+Chip172017460473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib167" class="citations"><div id="optMaXjtQ2q4i" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optMaXjtQ2q4i-1" title="View in article">44.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.DobinC.A.DavisF.SchlesingerJ.DrenkowC.ZaleskiS.JhaP.BatutM.ChaissonT.R.GingerasSTAR%3A+ultrafast+universal+RNA-seq+alignerBioinformatics2920131521" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">45.</a></div><div>Doshi, P. ∙ Suryo, R. ∙ Yildirim, O.E. ...</div><div><strong>Scaling in pinch-off of generalized Newtonian fluids</strong></div><div><em>J Non-Newton Fluid.</em> 2003; <strong>113</strong>:1-27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.DoshiR.SuryoO.E.YildirimG.H.McKinleyO.A.BasaranScaling+in+pinch-off+of+generalized+Newtonian+fluidsJ+Non-Newton+Fluid1132003127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">46.</a></div><div>Dowle, M. ∙ Srinivasan, A. ∙ Gorecki, J. ...</div><div><strong>Package ‘data. table.’ Extension of ‘data</strong></div><div>Frame, 2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.DowleA.SrinivasanJ.GoreckiM.ChiricoP.StetsenkoT.ShortS.LianoglouE.AntonyanM.BonschH.ParsonagePackage+%E2%80%98data.+table.%E2%80%99+Extension+of+%E2%80%98data2019Frame" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">47.</a></div><div>Durinck, S. ∙ Spellman, P.T. ∙ Birney, E. ...</div><div><strong>Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt</strong></div><div><em>Nat. Protoc.</em> 2009; <strong>4</strong>:1184-1191</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.DurinckP.T.SpellmanE.BirneyW.HuberMapping+identifiers+for+the+integration+of+genomic+datasets+with+the+R%2FBioconductor+package+biomaRtNat.+Protoc.4200911841191" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">48.</a></div><div>Dwivedi, U.N. ∙ Singh, P. ∙ Pandey, V.P. ...</div><div><strong>Structure–function relationship among bacterial, fungal and plant laccases</strong></div><div><em>J.&nbsp;Mol. Catal. B Enzym.</em> 2011; <strong>68</strong>:117-128</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=U.N.DwivediP.SinghV.P.PandeyA.KumarStructure%E2%80%93function+relationship+among+bacterial%2C+fungal+and+plant+laccasesJ.%C2%A0Mol.+Catal.+B+Enzym.682011117128" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">49.</a></div><div>Eckart, C. ∙ Young, G.</div><div><strong>The approximation of one matrix by another of lower rank</strong></div><div><em>Psychometrika.</em> 1936; <strong>1</strong>:211-218</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.EckartG.YoungThe+approximation+of+one+matrix+by+another+of+lower+rankPsychometrika11936211218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48-1" title="View in article">50.</a></div><div>Emms, D.M. ∙ Kelly, S.</div><div><strong>OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:157</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.EmmsS.KellyOrthoFinder%3A+solving+fundamental+biases+in+whole+genome+comparisons+dramatically+improves+orthogroup+inference+accuracyGenome+Biol.162015157" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49-1" title="View in article">51.</a></div><div>Emms, D.M. ∙ Kelly, S.</div><div><strong>OrthoFinder: phylogenetic orthology inference for comparative genomics</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:238</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.EmmsS.KellyOrthoFinder%3A+phylogenetic+orthology+inference+for+comparative+genomicsGenome+Biol.202019238" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">52.</a></div><div>Enright, A.J. ∙ Van Dongen, S. ∙ Ouzounis, C.A.</div><div><strong>An efficient algorithm for large-scale detection of protein families</strong></div><div><em>Nucleic Acids Res.</em> 2002; <strong>30</strong>:1575-1584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.J.EnrightS.Van+DongenC.A.OuzounisAn+efficient+algorithm+for+large-scale+detection+of+protein+familiesNucleic+Acids+Res.30200215751584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">53.</a></div><div>Erwin, D.H.</div><div><strong>Novelty and Innovation in the History of Life</strong></div><div><em>Curr. Biol.</em> 2015; <strong>25</strong>:R930-R940</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.H.ErwinNovelty+and+Innovation+in+the+History+of+LifeCurr.+Biol.252015R930R940" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">54.</a></div><div>Erwin, D.H.</div><div><strong>A conceptual framework of evolutionary novelty and innovation</strong></div><div><em>Biol. Rev. Camb. Philos. Soc.</em> 2021; <strong>96</strong>:1-15</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.H.ErwinA+conceptual+framework+of+evolutionary+novelty+and+innovationBiol.+Rev.+Camb.+Philos.+Soc.962021115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">55.</a></div><div>Feyereisen, R.</div><div><strong>Origin and evolution of the CYP4G subfamily in insects, cytochrome P450 enzymes involved in cuticular hydrocarbon synthesis</strong></div><div><em>Mol. Phylogenet. Evol.</em> 2020; <strong>143</strong>:106695</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.FeyereisenOrigin+and+evolution+of+the+CYP4G+subfamily+in+insects%2C+cytochrome+P450+enzymes+involved+in+cuticular+hydrocarbon+synthesisMol.+Phylogenet.+Evol.1432020106695" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">56.</a></div><div>Finet, C. ∙ Slavik, K. ∙ Pu, J. ...</div><div><strong>Birth-and-death evolution of the fatty acyl-CoA reductase (FAR) gene family and diversification of cuticular hydrocarbon synthesis in Drosophila</strong></div><div><em>Genome Biol. Evol.</em> 2019; <strong>11</strong>:1541-1551</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.FinetK.SlavikJ.PuS.B.CarrollH.ChungBirth-and-death+evolution+of+the+fatty+acyl-CoA+reductase+%28FAR%29+gene+family+and+diversification+of+cuticular+hydrocarbon+synthesis+in+DrosophilaGenome+Biol.+Evol.11201915411551" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">57.</a></div><div>Fox, J.</div><div><strong>Effect displays in R for generalised linear models</strong></div><div><em>J.&nbsp;Stat. Softw.</em> 2003; <strong>8</strong>:1-27</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxEffect+displays+in+R+for+generalised+linear+modelsJ.%C2%A0Stat.+Softw.82003127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">58.</a></div><div>Fox, J. ∙ Weisberg, S. ∙ Adler, D. ...</div><div><strong>Package ‘car’</strong></div><div>R Foundation for Statistical Computing, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.FoxS.WeisbergD.AdlerD.BatesG.Baud-BovyS.EllisonD.FirthM.FriendlyG.GorjancS.GravesPackage+%E2%80%98car%E2%80%992012R+Foundation+for+Statistical+Computing" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">59.</a></div><div>Francke, W. ∙ Dettner, K.</div><div><strong>Chemical Signalling in Beetles</strong></div><div><em>Top. Curr. Chem.</em> 2005; <strong>240</strong>:85-166</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.FranckeK.DettnerChemical+Signalling+in+BeetlesTop.+Curr.+Chem.240200585166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">60.</a></div><div>Fu, L. ∙ Niu, B. ∙ Zhu, Z. ...</div><div><strong>CD-HIT: accelerated for clustering the next-generation sequencing data</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:3150-3152</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.FuB.NiuZ.ZhuS.WuW.LiCD-HIT%3A+accelerated+for+clustering+the+next-generation+sequencing+dataBioinformatics28201231503152" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">61.</a></div><div>Grabherr, M.G. ∙ Haas, B.J. ∙ Yassour, M. ...</div><div><strong>Full-length transcriptome assembly from RNA-Seq data without a reference genome</strong></div><div><em>Nat. Biotechnol.</em> 2011; <strong>29</strong>:644-652</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.G.GrabherrB.J.HaasM.YassourJ.Z.LevinD.A.ThompsonI.AmitX.AdiconisL.FanR.RaychowdhuryQ.ZengFull-length+transcriptome+assembly+from+RNA-Seq+data+without+a+reference+genomeNat.+Biotechnol.292011644652" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">62.</a></div><div>Griffith, O.W. ∙ Wagner, G.P.</div><div><strong>The placenta as a model for understanding the origin and evolution of vertebrate organs</strong></div><div><em>Nat. Ecol. Evol.</em> 2017; <strong>1</strong>:0072</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.W.GriffithG.P.WagnerThe+placenta+as+a+model+for+understanding+the+origin+and+evolution+of+vertebrate+organsNat.+Ecol.+Evol.120170072" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">63.</a></div><div>Haas, B.J. ∙ Salzberg, S.L. ∙ Zhu, W. ...</div><div><strong>Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R7</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasS.L.SalzbergW.ZhuM.PerteaJ.E.AllenJ.OrvisO.WhiteC.R.BuellJ.R.WortmanAutomated+eukaryotic+gene+structure+annotation+using+EVidenceModeler+and+the+Program+to+Assemble+Spliced+AlignmentsGenome+Biol.92008R7" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">64.</a></div><div>Haas, B.J. ∙ Papanicolaou, A. ∙ Yassour, M. ...</div><div><strong>De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:1494-1512</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.J.HaasA.PapanicolaouM.YassourM.GrabherrP.D.BloodJ.BowdenM.B.CougerD.EcclesB.LiM.LieberDe+novo+transcript+sequence+reconstruction+from+RNA-seq+using+the+Trinity+platform+for+reference+generation+and+analysisNat.+Protoc.8201314941512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">65.</a></div><div>Hammer, O. ∙ Harper, D.A. ∙ Ryan, P.D.</div><div><strong>PAST: Paleontological statistics software package for education and data analysis</strong></div><div><em>Palaeontol. Electronica.</em> 2001; <strong>4</strong>:9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.HammerD.A.HarperP.D.RyanPAST%3A+Paleontological+statistics+software+package+for+education+and+data+analysisPalaeontol.+Electronica420019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">66.</a></div><div>Haunerland, N.H. ∙ Shirk, P.D.</div><div><strong>Regional and Functional Differentiation in the Insect Fact Body</strong></div><div><em>Annu. Rev. Entomol.</em> 1995; <strong>40</strong>:121-145</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.H.HaunerlandP.D.ShirkRegional+and+Functional+Differentiation+in+the+Insect+Fact+BodyAnnu.+Rev.+Entomol.401995121145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">67.</a></div><div>Hobert, O.</div><div><strong>Terminal Selectors of Neuronal Identity</strong></div><div><em>Curr. Top Dev. Biol.</em> 2016; <strong>116</strong>:455-475</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.HobertTerminal+Selectors+of+Neuronal+IdentityCurr.+Top+Dev.+Biol.1162016455475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">68.</a></div><div>Hochberg, G.K.A. ∙ Liu, Y. ∙ Marklund, E.G. ...</div><div><strong>A hydrophobic ratchet entrenches molecular complexes</strong></div><div><em>Nature.</em> 2020; <strong>588</strong>:503-508</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.K.A.HochbergY.LiuE.G.MarklundB.P.H.MetzgerA.LaganowskyJ.W.ThorntonA+hydrophobic+ratchet+entrenches+molecular+complexesNature5882020503508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">69.</a></div><div>Holze, H. ∙ Schrader, L. ∙ Buellesbach, J.</div><div><strong>Advances in deciphering the genetic basis of insect cuticular hydrocarbon biosynthesis and variation</strong></div><div><em>Heredity.</em> 2021; <strong>126</strong>:219-234</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.HolzeL.SchraderJ.BuellesbachAdvances+in+deciphering+the+genetic+basis+of+insect+cuticular+hydrocarbon+biosynthesis+and+variationHeredity1262021219234" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">70.</a></div><div>Hu, Y. ∙ Linz, D.M. ∙ Moczek, A.P.</div><div><strong>Beetle horns evolved from wing serial homologs</strong></div><div><em>Science.</em> 2019; <strong>366</strong>:1004-1007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.HuD.M.LinzA.P.MoczekBeetle+horns+evolved+from+wing+serial+homologsScience366201910041007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">71.</a></div><div>Ibarra, Y. ∙ Blair, N.T.</div><div><strong>Benzoquinone reveals a cysteine-dependent desensitization mechanism of TRPA1</strong></div><div><em>Mol. Pharmacol.</em> 2013; <strong>83</strong>:1120-1132</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.IbarraN.T.BlairBenzoquinone+reveals+a+cysteine-dependent+desensitization+mechanism+of+TRPA1Mol.+Pharmacol.83201311201132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">72.</a></div><div>Jorth, P. ∙ Spero, M.A. ∙ Livingston, J. ...</div><div><strong>Quantitative Visualization of Gene Expression in Mucoid and Nonmucoid Pseudomonas aeruginosa Aggregates Reveals Localized Peak Expression of Alginate in the Hypoxic Zone</strong></div><div><em>MBio.</em> 2019; <strong>10</strong>:e02622</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.JorthM.A.SperoJ.LivingstonD.K.NewmanQuantitative+Visualization+of+Gene+Expression+in+Mucoid+and+Nonmucoid+Pseudomonas+aeruginosa+Aggregates+Reveals+Localized+Peak+Expression+of+Alginate+in+the+Hypoxic+ZoneMBio102019e02622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">73.</a></div><div>Kang, K. ∙ Pulver, S.R. ∙ Panzano, V.C. ...</div><div><strong>Analysis of Drosophila TRPA1 reveals an ancient origin for human chemical nociception</strong></div><div><em>Nature.</em> 2010; <strong>464</strong>:597-600</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KangS.R.PulverV.C.PanzanoE.C.ChangL.C.GriffithD.L.TheobaldP.A.GarrityAnalysis+of+Drosophila+TRPA1+reveals+an+ancient+origin+for+human+chemical+nociceptionNature4642010597600" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">74.</a></div><div>Katoh, K. ∙ Standley, D.M.</div><div><strong>MAFFT multiple sequence alignment software version 7: improvements in performance and usability</strong></div><div><em>Mol. Biol. Evol.</em> 2013; <strong>30</strong>:772-780</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KatohD.M.StandleyMAFFT+multiple+sequence+alignment+software+version+7%3A+improvements+in+performance+and+usabilityMol.+Biol.+Evol.302013772780" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">75.</a></div><div>Kawamukai, M.</div><div><strong>Biosynthesis of coenzyme Q in eukaryotes</strong></div><div><em>Biosci. Biotechnol. Biochem.</em> 2016; <strong>80</strong>:23-33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.KawamukaiBiosynthesis+of+coenzyme+Q+in+eukaryotesBiosci.+Biotechnol.+Biochem.8020162333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">76.</a></div><div>Kebschull, J.M. ∙ Richman, E.B. ∙ Ringach, N. ...</div><div><strong>Cerebellar nuclei evolved by repeatedly duplicating a conserved cell-type set</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:eabd5059</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.KebschullE.B.RichmanN.RingachD.FriedmannE.AlbarranS.S.KolluruR.C.JonesW.E.AllenY.WangS.W.ChoCerebellar+nuclei+evolved+by+repeatedly+duplicating+a+conserved+cell-type+setScience3702020eabd5059" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">77.</a></div><div>Keilwagen, J. ∙ Hartung, F. ∙ Grau, J.</div><div><strong>GeMoMa: Homology-Based Gene Prediction Utilizing Intron Position Conservation and RNA-seq Data</strong></div><div><em>Methods Mol. Biol.</em> 2019; <strong>1962</strong>:161-177</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.KeilwagenF.HartungJ.GrauGeMoMa%3A+Homology-Based+Gene+Prediction+Utilizing+Intron+Position+Conservation+and+RNA-seq+DataMethods+Mol.+Biol.19622019161177" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">78.</a></div><div>Kishi, Y. ∙ Parker, J.</div><div><strong>Cell type innovation at the tips of the animal tree</strong></div><div><em>Curr. Opin. Genet. Dev.</em> 2021; <strong>69</strong>:112-121</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.KishiJ.ParkerCell+type+innovation+at+the+tips+of+the+animal+treeCurr.+Opin.+Genet.+Dev.692021112121" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">79.</a></div><div>Klein, A.M. ∙ Mazutis, L. ∙ Akartuna, I. ...</div><div><strong>Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells</strong></div><div><em>Cell.</em> 2015; <strong>161</strong>:1187-1201</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.M.KleinL.MazutisI.AkartunaN.TallapragadaA.VeresV.LiL.PeshkinD.A.WeitzM.W.KirschnerDroplet+barcoding+for+single-cell+transcriptomics+applied+to+embryonic+stem+cellsCell161201511871201" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib165" class="citations"><div id="oref0010" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-oref0010-1" title="View in article">80.</a></div>Kluyver, T., Ragan-Kelley, B., Pérez, F., Granger, B.E., Bussonnier, M., Frederic, J., Kelley, K., Hamrick, J., Grout, J., Corlay, S., et al. (2016). <i>Jupyter Notebooks-a publishing format for reproducible computational workflows. In proceedings of the 20th International Conference on Electronic Publishing.</i> pp. 87-90.</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Kluyver%2C+T.%2C+Ragan-Kelley%2C+B.%2C+P%C3%A9rez%2C+F.%2C+Granger%2C+B.E.%2C+Bussonnier%2C+M.%2C+Frederic%2C+J.%2C+Kelley%2C+K.%2C+Hamrick%2C+J.%2C+Grout%2C+J.%2C+Corlay%2C+S.%2C+et+al.+%282016%29.+Jupyter+Notebooks-a+publishing+format+for+reproducible+computational+workflows.+In+proceedings+of+the+20th+International+Conference+on+Electronic+Publishing.+pp.+87-90." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">81.</a></div><div>Kotliar, D. ∙ Veres, A. ∙ Nagy, M.A. ...</div><div><strong>Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq</strong></div><div><em>eLife.</em> 2019; <strong>8</strong>:e43803</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.KotliarA.VeresM.A.NagyS.TabriziE.HodisD.A.MeltonP.C.SabetiIdentifying+gene+expression+programs+of+cell-type+identity+and+cellular+activity+with+single-cell+RNA-SeqeLife82019e43803" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">82.</a></div><div>Koto, A. ∙ Motoyama, N. ∙ Tahara, H. ...</div><div><strong>Oxytocin/vasopressin-like peptide inotocin regulates cuticular hydrocarbon synthesis and water balancing in ants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2019; <strong>116</strong>:5597-5606</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.KotoN.MotoyamaH.TaharaS.McGregorM.MoriyamaT.OkabeM.MiuraL.KellerOxytocin%2Fvasopressin-like+peptide+inotocin+regulates+cuticular+hydrocarbon+synthesis+and+water+balancing+in+antsProc.+Natl.+Acad.+Sci.+USA116201955975606" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">83.</a></div><div>Kraaijeveld, K. ∙ Neleman, P. ∙ Mariën, J. ...</div><div><strong>Genomic Resources for Goniozus legneri, Aleochara bilineata and Paykullia maculata, Representing Three Independent Origins of the Parasitoid Lifestyle in Insects.</strong></div><div><em>G3 (Bethesda).</em> 2019; <strong>9</strong>:987-991</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.KraaijeveldP.NelemanJ.Mari%C3%ABnE.+deMeijerJ.EllersGenomic+Resources+for+Goniozus+legneri%2C+Aleochara+bilineata+and+Paykullia+maculata%2C+Representing+Three+Independent+Origins+of+the+Parasitoid+Lifestyle+in+Insects.G3+%28Bethesda%2992019987991" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">84.</a></div><div>Laetsch, D.R. ∙ Blaxter, M.L.</div><div><strong>BlobTools: Interrogation of genome assemblies.</strong></div><div><em>F1000Res.</em> 2017; <strong>6</strong>:1287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.R.LaetschM.L.BlaxterBlobTools%3A+Interrogation+of+genome+assemblies.F1000Res.620171287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">85.</a></div><div>Lamb, T.D. ∙ Collin, S.P. ∙ Pugh, Jr., E.N.</div><div><strong>Evolution of the vertebrate eye: opsins, photoreceptors, retina and eye cup.</strong></div><div><em>Nat. Rev. Neurosci.</em> 2007; <strong>8</strong>:960-976</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.D.LambS.P.CollinE.N.PughJr.Evolution+of+the+vertebrate+eye%3A+opsins%2C+photoreceptors%2C+retina+and+eye+cup.Nat.+Rev.+Neurosci.82007960976" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">86.</a></div><div>Li, D. ∙ Luo, R. ∙ Liu, C.-M. ...</div><div><strong>MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices.</strong></div><div><em>Methods.</em> 2016; <strong>102</strong>:3-11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.LiR.LuoC.-M.LiuC.-M.LeungH.-F.TingK.SadakaneH.YamashitaT.-W.LamMEGAHIT+v1.0%3A+A+fast+and+scalable+metagenome+assembler+driven+by+advanced+methodologies+and+community+practices.Methods1022016311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84-1" title="View in article">87.</a></div><div>Liaw, A. ∙ Wiener, M.</div><div><strong>Classification and regression by randomForest</strong></div><div><em>R News.</em> 2002; <strong>2/3</strong>:18-22</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.LiawM.WienerClassification+and+regression+by+randomForestR+News2%2F320021822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">88.</a></div><div>Linz, D.M. ∙ Tomoyasu, Y.</div><div><strong>Dual evolutionary origin of insect wings&nbsp;supported by an investigation of the abdominal wing serial homologs in <i>Tribolium</i></strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E658-E667</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.M.LinzY.TomoyasuDual+evolutionary+origin+of+insect+wings%C2%A0supported+by+an+investigation+of+the+abdominal+wing+serial+homologs+in+TriboliumProc.+Natl.+Acad.+Sci.+USA1152018E658E667" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib162" class="citations"><div id="optA1Pb6vGgHv" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-optA1Pb6vGgHv-1" title="View in article">89.</a></div><div>Litvaitis, M.K. ∙ Rohde, K.</div><div><strong>A molecular test of platyhelminth phylogeny: inferences from partial 28S rDNA sequences</strong></div><div><em>Invertebrate Biology.</em> 1999; <strong>118</strong>:42-56</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.K.LitvaitisK.RohdeA+molecular+test+of+platyhelminth+phylogeny%3A+inferences+from+partial+28S+rDNA+sequencesInvertebrate+Biology11819994256" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">90.</a></div><div>Lukeš, J. ∙ Archibald, J.M. ∙ Keeling, P.J. ...</div><div><strong>How a neutral evolutionary ratchet can build cellular complexity</strong></div><div><em>IUBMB Life.</em> 2011; <strong>63</strong>:528-537</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.Luke%C5%A1J.M.ArchibaldP.J.KeelingW.F.DoolittleM.W.GrayHow+a+neutral+evolutionary+ratchet+can+build+cellular+complexityIUBMB+Life632011528537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">91.</a></div><div>Lun, A.T.L. ∙ Riesenfeld, S. ∙ Andrews, T. ..., 1st Human Cell Atlas Jamboree</div><div><strong>EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:63</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.T.L.LunS.RiesenfeldT.AndrewsT.P.DaoT.GomesJ.C.Marioni1st+Human+Cell+Atlas+JamboreeEmptyDrops%3A+distinguishing+cells+from+empty+droplets+in+droplet-based+single-cell+RNA+sequencing+dataGenome+Biol.20201963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">92.</a></div><div>Luo, R. ∙ Liu, B. ∙ Xie, Y. ...</div><div><strong>SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</strong></div><div><em>Gigascience.</em> 2012; <strong>1</strong>:18</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.LuoB.LiuY.XieZ.LiW.HuangJ.YuanG.HeY.ChenQ.PanY.LiuSOAPdenovo2%3A+an+empirically+improved+memory-efficient+short-read+de+novo+assemblerGigascience1201218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89" title="View in article">93.</a></div><div>Lynch, M.</div><div><strong>The frailty of adaptive hypotheses for the origins of organismal complexity</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:8597-8604</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LynchThe+frailty+of+adaptive+hypotheses+for+the+origins+of+organismal+complexityProc.+Natl.+Acad.+Sci.+USA104Suppl+1200785978604" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">94.</a></div><div>Makki, R. ∙ Cinnamon, E. ∙ Gould, A.P.</div><div><strong>The development and functions of oenocytes</strong></div><div><em>Annu. Rev. Entomol.</em> 2014; <strong>59</strong>:405-425</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.MakkiE.CinnamonA.P.GouldThe+development+and+functions+of+oenocytesAnnu.+Rev.+Entomol.592014405425" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91-1" title="View in article">95.</a></div><div>Marioni, J.C. ∙ Arendt, D.</div><div><strong>How Single-Cell Genomics Is Changing Evolutionary and Developmental Biology</strong></div><div><em>Annu. Rev. Cell Dev. Biol.</em> 2017; <strong>33</strong>:537-553</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.C.MarioniD.ArendtHow+Single-Cell+Genomics+Is+Changing+Evolutionary+and+Developmental+BiologyAnnu.+Rev.+Cell+Dev.+Biol.332017537553" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">96.</a></div><div>Marshall, K.A. ∙ Liedtke, A.M. ∙ Todt, A.H. ...</div><div><strong>Extensional rheometry with a handheld mobile device</strong></div><div><em>Exp. Fluids.</em> 2017; <strong>58</strong>:69</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.A.MarshallA.M.LiedtkeA.H.TodtT.W.WalkerExtensional+rheometry+with+a+handheld+mobile+deviceExp.+Fluids58201769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">97.</a></div><div>Martin, M.</div><div><strong>Cutadapt removes adapter sequences from high-throughput sequencing reads</strong></div><div><em>EMBnet. J.</em> 2011; <strong>17</strong>:10-12</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MartinCutadapt+removes+adapter+sequences+from+high-throughput+sequencing+readsEMBnet.+J.1720111012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">98.</a></div><div>Maruyama, M. ∙ Parker, J.</div><div><strong>Deep-Time Convergence in Rove Beetle Symbionts of Army Ants</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:920-926</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MaruyamaJ.ParkerDeep-Time+Convergence+in+Rove+Beetle+Symbionts+of+Army+AntsCurr.+Biol.272017920926" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">99.</a></div><div>McInnes, L. ∙ Healy, J. ∙ Melville, J.</div><div><strong>UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction</strong></div><div><em>arXiv.</em> 2018;</div><div>arXiv:1802.03426</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.McInnesJ.HealyJ.MelvilleUMAP%3A+Uniform+Manifold+Approximation+and+Projection+for+Dimension+ReductionarXiv2018arXiv%3A1802.03426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">100.</a></div><div>Melsted, P. ∙ Ntranos, V. ∙ Pachter, L.</div><div><strong>The barcode, UMI, set format and BUStools</strong></div><div><em>Bioinformatics.</em> 2019; <strong>35</strong>:4472-4473</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MelstedV.NtranosL.PachterThe+barcode%2C+UMI%2C+set+format+and+BUStoolsBioinformatics35201944724473" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">101.</a></div><div>Melsted, P. ∙ Booeshaghi, A.S. ∙ Liu, L. ...</div><div><strong>Modular, efficient and constant-memory single-cell RNA-seq preprocessing</strong></div><div><em>Nat. Biotechnol.</em> 2021; <strong>39</strong>:813-818</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.MelstedA.S.BooeshaghiL.LiuF.GaoL.LuK.H.J.MinE.da+Veiga+BeltrameK.E.Hj%C3%B6rleifssonJ.GehringL.PachterModular%2C+efficient+and+constant-memory+single-cell+RNA-seq+preprocessingNat.+Biotechnol.392021813818" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98-1" title="View in article">102.</a></div><div>Meyer, D. ∙ Zeileis, A. ∙ Hornik, K. ...</div><div><strong>Package ‘vcd.’ Visualizing Categorical Data. R Package Version 1–4</strong></div><div>2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.MeyerA.ZeileisK.HornikF.GerberM.FriendlyM.D.MeyerPackage+%E2%80%98vcd.%E2%80%99+Visualizing+Categorical+Data.+R+Package+Version+1%E2%80%9342020" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">103.</a></div><div>Morata, G. ∙ Lawrence, P.A.</div><div><strong>Control of compartment development by the engrailed gene in Drosophila</strong></div><div><em>Nature.</em> 1975; <strong>255</strong>:614-617</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.MorataP.A.LawrenceControl+of+compartment+development+by+the+engrailed+gene+in+DrosophilaNature2551975614617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">104.</a></div><div>Morgan, M. ∙ Carlson, M. ∙ Tenenbaum, D. ...</div><div><strong>Package AnnotationHub</strong></div><div>2019</div><div><a id="intref0640" href="https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.MorganM.CarlsonD.TenenbaumS.AroraPackage+AnnotationHub2019https%3A%2F%2Fbioconductor.org%2Fpackages%2Frelease%2Fbioc%2Fhtml%2FAnnotationHub.html" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101-1" title="View in article">105.</a></div><div>Moses, L. ∙ Pachter, L.</div><div><strong>BUSpaRse: kallisto | bustools R utilities, R Package Version 1.4.2</strong></div><div>2021</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.MosesL.PachterBUSpaRse%3A+kallisto+%7C+bustools+R+utilities%2C+R+Package+Version+1.4.22021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">106.</a></div><div>Musser, J.M. ∙ Schippers, K.J. ∙ Nickel, M. ...</div><div><strong>Profiling cellular diversity in sponges informs animal cell type and nervous system evolution</strong></div><div><em>Science.</em> 2021; <strong>374</strong>:717-723</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.MusserK.J.SchippersM.NickelG.MizzonA.B.KohnC.PapeJ.U.HammelF.WolfC.LiangA.Hern%C3%A1ndez-PlazaProfiling+cellular+diversity+in+sponges+informs+animal+cell+type+and+nervous+system+evolutionScience3742021717723" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103-1" title="View in article">107.</a></div><div>Nguyen, L.-T. ∙ Schmidt, H.A. ∙ von Haeseler, A. ...</div><div><strong>IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:268-274</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.-T.NguyenH.A.SchmidtA.von+HaeselerB.Q.MinhIQ-TREE%3A+a+fast+and+effective+stochastic+algorithm+for+estimating+maximum-likelihood+phylogeniesMol.+Biol.+Evol.322015268274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104-1" title="View in article">108.</a></div><div>Noh, M.Y. ∙ Muthukrishnan, S. ∙ Kramer, K.J. ...</div><div><strong>Cuticle formation and pigmentation in beetles</strong></div><div><em>Curr. Opin. Insect Sci.</em> 2016; <strong>17</strong>:1-9</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.Y.NohS.MuthukrishnanK.J.KramerY.ArakaneCuticle+formation+and+pigmentation+in+beetlesCurr.+Opin.+Insect+Sci.17201619" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">109.</a></div><div>Oakley, T.H.</div><div><strong>Furcation and fusion: The phylogenetics of evolutionary novelty</strong></div><div><em>Dev. Biol.</em> 2017; <strong>431</strong>:69-76</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.H.OakleyFurcation+and+fusion%3A+The+phylogenetics+of+evolutionary+noveltyDev.+Biol.43120176976" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">110.</a></div><div>Oakley, T.H. ∙ Speiser, D.I.</div><div><strong>How Complexity Originates: The Evolution of Animal Eyes</strong></div><div><em>Annu. Rev. Ecol. Evol. Syst.</em> 2012; <strong>46</strong>:1-24</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.H.OakleyD.I.SpeiserHow+Complexity+Originates%3A+The+Evolution+of+Animal+EyesAnnu.+Rev.+Ecol.+Evol.+Syst.462012124" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">111.</a></div><div>Parker, J.</div><div><strong>Myrmecophily in beetles (Coleoptera): evolutionary patterns and biological mechanisms</strong></div><div><em>Myrmecol. News.</em> 2016; <strong>22</strong>:65-108</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerMyrmecophily+in+beetles+%28Coleoptera%29%3A+evolutionary+patterns+and+biological+mechanismsMyrmecol.+News22201665108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">112.</a></div><div>Parker, J.</div><div><strong>Staphylinids</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:R49-R51</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerStaphylinidsCurr.+Biol.272017R49R51" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">113.</a></div><div>Parker, J. ∙ Eldredge, K.T. ∙ Thomas, I. ...</div><div><strong>Hox-logic of body plan innovations for social insect symbiosis in rove beetles</strong></div><div><em>bioRxiv.</em> 2018; 198945</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.ParkerK.T.EldredgeI.ThomasR.ColemanS.DavisHox-logic+of+body+plan+innovations+for+social+insect+symbiosis+in+rove+beetlesbioRxiv2018198945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110" title="View in article">114.</a></div><div>Petersen, M. ∙ Armisén, D. ∙ Gibbs, R.A. ...</div><div><strong>Diversity and evolution of the transposable element repertoire in arthropods with particular reference to insects</strong></div><div><em>BMC Evol. Biol.</em> 2019; <strong>19</strong>:11</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.PetersenD.Armis%C3%A9nR.A.GibbsL.HeringA.KhilaG.MayerS.RichardsO.NiehuisB.MisofDiversity+and+evolution+of+the+transposable+element+repertoire+in+arthropods+with+particular+reference+to+insectsBMC+Evol.+Biol.19201911" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib166" class="citations"><div id="opt0vPPm0qdrV" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opt0vPPm0qdrV" title="View in article">115.</a></div><div>Philip, B.N. ∙ Tomoyasu, Y.</div><div><strong>Gene knockdown analysis by double-stranded RNA injection</strong></div><div><strong><i>Molecular methods for evolutionary genetics</i></strong></div><div>Humana Press, 2012; 471-497</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.N.PhilipY.TomoyasuGene+knockdown+analysis+by+double-stranded+RNA+injectionMolecular+methods+for+evolutionary+genetics2012Humana+Press471497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111" title="View in article">116.</a></div><div>Piel, J.</div><div><strong>A polyketide synthase-peptide synthetase gene cluster from an uncultured bacterial symbiont of Paederus beetles</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:14002-14007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PielA+polyketide+synthase-peptide+synthetase+gene+cluster+from+an+uncultured+bacterial+symbiont+of+Paederus+beetlesProc.+Natl.+Acad.+Sci.+USA9920021400214007" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112-1" title="View in article">117.</a></div><div>Pimentel, H. ∙ Bray, N.L. ∙ Puente, S. ...</div><div><strong>Differential analysis of RNA-seq incorporating quantification uncertainty</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:687-690</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.PimentelN.L.BrayS.PuenteP.MelstedL.PachterDifferential+analysis+of+RNA-seq+incorporating+quantification+uncertaintyNat.+Methods142017687690" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113" title="View in article">118.</a></div><div>Pineda, A. ∙ Zheng, S.-J. ∙ van Loon, J.J.A. ...</div><div><strong>Helping plants to deal with insects: the role of beneficial soil-borne microbes</strong></div><div><em>Trends Plant Sci.</em> 2010; <strong>15</strong>:507-514</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.PinedaS.-J.ZhengJ.J.A.van+LoonC.M.J.PieterseM.DickeHelping+plants+to+deal+with+insects%3A+the+role+of+beneficial+soil-borne+microbesTrends+Plant+Sci.152010507514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114-1" title="View in article">119.</a></div><div>Pinheiro, J. ∙ Bates, D.</div><div><strong>Linear and nonlinear mixed effects models</strong></div><div><em>R Package Version.</em> 2007; <strong>3</strong>:1-89</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.PinheiroD.BatesLinear+and+nonlinear+mixed+effects+modelsR+Package+Version32007189" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">120.</a></div><div>Poon, W.W. ∙ Barkovich, R.J. ∙ Hsu, A.Y. ...</div><div><strong>Yeast and rat Coq3 and Escherichia coli UbiG polypeptides catalyze both O-methyltransferase steps in coenzyme Q biosynthesis</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 1999; <strong>274</strong>:21665-21672</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=W.W.PoonR.J.BarkovichA.Y.HsuA.FrankelP.T.LeeJ.N.ShepherdD.C.MylesC.F.ClarkeYeast+and+rat+Coq3+and+Escherichia+coli+UbiG+polypeptides+catalyze+both+O-methyltransferase+steps+in+coenzyme+Q+biosynthesisJ.%C2%A0Biol.+Chem.27419992166521672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116-1" title="View in article">121.</a></div><div>Price, M.N. ∙ Dehal, P.S. ∙ Arkin, A.P.</div><div><strong>FastTree 2--approximately maximum-likelihood trees for large alignments</strong></div><div><em>PLoS ONE.</em> 2010; <strong>5</strong>:e9490</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.N.PriceP.S.DehalA.P.ArkinFastTree+2--approximately+maximum-likelihood+trees+for+large+alignmentsPLoS+ONE52010e9490" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117-1" title="View in article">122.</a></div><div>Pruitt, K.D. ∙ Tatusova, T. ∙ Maglott, D.R.</div><div><strong>NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</strong></div><div><em>Nucleic Acids Res.</em> 2007; <strong>35</strong>:D61-D65</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=K.D.PruittT.TatusovaD.R.MaglottNCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteinsNucleic+Acids+Res.352007D61D65" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118-1" title="View in article">123.</a></div><div>Pryszcz, L.P. ∙ Gabaldón, T.</div><div><strong>Redundans: an assembly pipeline for highly heterozygous genomes</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:e113</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=L.P.PryszczT.Gabald%C3%B3nRedundans%3A+an+assembly+pipeline+for+highly+heterozygous+genomesNucleic+Acids+Res.442016e113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119-1" title="View in article">124.</a></div><div>Qiu, Y. ∙ Tittiger, C. ∙ Wicker-Thomas, C. ...</div><div><strong>An insect-specific P450 oxidative decarbonylase for cuticular hydrocarbon biosynthesis</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2012; <strong>109</strong>:14858-14863</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.QiuC.TittigerC.Wicker-ThomasG.Le+GoffS.YoungE.WajnbergT.FricauxN.TaquetG.J.BlomquistR.FeyereisenAn+insect-specific+P450+oxidative+decarbonylase+for+cuticular+hydrocarbon+biosynthesisProc.+Natl.+Acad.+Sci.+USA10920121485814863" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120" title="View in article">125.</a></div><div>Raff, R.</div><div><strong>The Shape of Life: Genes, Development, and the Evolution of Animal Form</strong></div><div>University of Chicago Press, 1996</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R.RaffThe+Shape+of+Life%3A+Genes%2C+Development%2C+and+the+Evolution+of+Animal+Form1996University+of+Chicago+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121-1" title="View in article">126.</a></div><div>Rambaut, A.</div><div><strong>FigTree v1.4.1</strong></div><div>2012</div><div><a id="intref0645" href="http://Tree.Biol.Ed.Ac.Uk/Software/Figtree/" target="_blank">http://Tree.Biol.Ed.Ac.Uk/Software/Figtree/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.RambautFigTree+v1.4.12012http%3A%2F%2FTree.Biol.Ed.Ac.Uk%2FSoftware%2FFigtree%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122-1" title="View in article">127.</a></div><div>Ripley, B. ∙ Venables, B. ∙ Bates, D.M. ...</div><div><strong>Package ‘mass.’</strong></div><div><em>Cran R.</em> 2013; <strong>538</strong>:113-120</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.RipleyB.VenablesD.M.BatesK.HornikA.GebhardtD.FirthM.B.RipleyPackage+%E2%80%98mass.%E2%80%99Cran+R5382013113120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123-1" title="View in article">128.</a></div><div>R Core Team</div><div><strong>R: A language and environment for statistical computing. R Foundation for Statistical Computing</strong></div><div>2018</div><div><a id="intref0650" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=R+Core+TeamR%3A+A+language+and+environment+for+statistical+computing.+R+Foundation+for+Statistical+Computing2018https%3A%2F%2Fwww.R-project.org%2F" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124-1" title="View in article">129.</a></div><div>Rognes, T. ∙ Flouri, T. ∙ Nichols, B. ...</div><div><strong>VSEARCH: a versatile open source tool for metagenomics</strong></div><div><em>PeerJ.</em> 2016; <strong>4</strong>:e2584</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.RognesT.FlouriB.NicholsC.QuinceF.Mah%C3%A9VSEARCH%3A+a+versatile+open+source+tool+for+metagenomicsPeerJ42016e2584" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref125" title="View in article">130.</a></div><div>Rosello, M. ∙ Sur, S. ∙ Barbet, B. ...</div><div><strong>Dripping-onto-substrate capillary breakup extensional rheometry of low-viscosity printing inks</strong></div><div><em>J.&nbsp;Non-Newton. Fluid.</em> 2019; <strong>266</strong>:160-170</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.RoselloS.SurB.BarbetJ.P.RothsteinDripping-onto-substrate+capillary+breakup+extensional+rheometry+of+low-viscosity+printing+inksJ.%C2%A0Non-Newton.+Fluid2662019160170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref126" title="View in article">131.</a></div><div>Roussa, E.</div><div><strong>Channels and transporters in salivary glands</strong></div><div><em>Cell Tissue Res.</em> 2011; <strong>343</strong>:263-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.RoussaChannels+and+transporters+in+salivary+glandsCell+Tissue+Res.3432011263287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127-1" title="View in article">132.</a></div><div>Sarkar, D.</div><div><strong>Lattice: Multivariate Data Visualization with R</strong></div><div>Springer Science &amp; Business Media, 2008</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.SarkarLattice%3A+Multivariate+Data+Visualization+with+R2008Springer+Science+%26+Business+Media" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">133.</a></div><div>Scales, B.S. ∙ Dickson, R.P. ∙ LiPuma, J.J. ...</div><div><strong>Microbiology, genomics, and clinical significance of the Pseudomonas fluorescens species complex, an unappreciated colonizer of humans</strong></div><div><em>Clin. Microbiol. Rev.</em> 2014; <strong>27</strong>:927-948</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.S.ScalesR.P.DicksonJ.J.LiPumaG.B.HuffnagleMicrobiology%2C+genomics%2C+and+clinical+significance+of+the+Pseudomonas+fluorescens+species+complex%2C+an+unappreciated+colonizer+of+humansClin.+Microbiol.+Rev.272014927948" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129" title="View in article">134.</a></div><div>Schlosser, G.</div><div><strong>A Short History of Nearly Every Sense-The Evolutionary History of Vertebrate Sensory Cell Types</strong></div><div><em>Integr. Comp. Biol.</em> 2018; <strong>58</strong>:301-316</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.SchlosserA+Short+History+of+Nearly+Every+Sense-The+Evolutionary+History+of+Vertebrate+Sensory+Cell+TypesIntegr.+Comp.+Biol.582018301316" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130" title="View in article">135.</a></div><div>Schmidt-Rhaesa, A.</div><div><strong>The Evolution of Organ Systems</strong></div><div>Oxford University Press, 2007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.Schmidt-RhaesaThe+Evolution+of+Organ+Systems2007Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref131-1" title="View in article">136.</a></div><div>Shubin, N. ∙ Tabin, C. ∙ Carroll, S.</div><div><strong>Deep homology and the origins of evolutionary novelty</strong></div><div><em>Nature.</em> 2009; <strong>457</strong>:818-823</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.ShubinC.TabinS.CarrollDeep+homology+and+the+origins+of+evolutionary+noveltyNature4572009818823" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132-1" title="View in article">137.</a></div><div>Simão, F.A. ∙ Waterhouse, R.M. ∙ Ioannidis, P. ...</div><div><strong>BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</strong></div><div><em>Bioinformatics.</em> 2015; <strong>31</strong>:3210-3212</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.A.Sim%C3%A3oR.M.WaterhouseP.IoannidisE.V.KriventsevaE.M.ZdobnovBUSCO%3A+assessing+genome+assembly+and+annotation+completeness+with+single-copy+orthologsBioinformatics31201532103212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib133" class="citations"><div id="sref133" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133-1" title="View in article">138.</a></div><div>Smit, A.F.</div><div><strong>Repeat-Masker Open-3.0</strong></div><div>2004</div><div><a id="intref0655" href="http://www.Repeatmasker.Org" target="_blank">http://www.Repeatmasker.Org</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.F.SmitRepeat-Masker+Open-3.02004http%3A%2F%2Fwww.Repeatmasker.Org" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134-1" title="View in article">139.</a></div><div>Smit, A.F. ∙ Hubley, R.</div><div><strong>RepeatModeler Open-1.0</strong></div><div>2008</div><div><a id="intref0660" href="https://www.scienceopen.com/document?vid=c1050e3d-8dc1-48ba-81c8-b372af25be50" target="_blank">https://www.scienceopen.com/document?vid=c1050e3d-8dc1-48ba-81c8-b372af25be50</a></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=A.F.SmitR.HubleyRepeatModeler+Open-1.02008https%3A%2F%2Fwww.scienceopen.com%2Fdocument%3Fvid%3Dc1050e3d-8dc1-48ba-81c8-b372af25be50" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib135" class="citations"><div id="sref135" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref135" title="View in article">140.</a></div><div>Smith, S.</div><div><strong>The animal fatty acid synthase: one gene, one polypeptide, seven enzymes</strong></div><div><em>FASEB J.</em> 1994; <strong>8</strong>:1248-1259</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.SmithThe+animal+fatty+acid+synthase%3A+one+gene%2C+one+polypeptide%2C+seven+enzymesFASEB+J.8199412481259" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136-1" title="View in article">141.</a></div><div>Smith, S. ∙ Witkowski, A. ∙ Joshi, A.K.</div><div><strong>Structural and functional organization of the animal fatty acid synthase</strong></div><div><em>Prog. Lipid Res.</em> 2003; <strong>42</strong>:289-317</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.SmithA.WitkowskiA.K.JoshiStructural+and+functional+organization+of+the+animal+fatty+acid+synthaseProg.+Lipid+Res.422003289317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137" title="View in article">142.</a></div><div>Sousa, P.C. ∙ Vega, E.J. ∙ Sousa, R.G. ...</div><div><strong>Measurement of relaxation times in extensional flow of weakly viscoelastic polymer solutions</strong></div><div><em>Rheol. Acta.</em> 2017; <strong>56</strong>:11-20</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=P.C.SousaE.J.VegaR.G.SousaJ.M.MontaneroM.A.AlvesMeasurement+of+relaxation+times+in+extensional+flow+of+weakly+viscoelastic+polymer+solutionsRheol.+Acta5620171120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138-1" title="View in article">143.</a></div><div>Stanke, M. ∙ Waack, S.</div><div><strong>Gene prediction with a hidden Markov model and a new intron submodel</strong></div><div><em>Bioinformatics.</em> 2003; <strong>19</strong>:ii215-ii225</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.StankeS.WaackGene+prediction+with+a+hidden+Markov+model+and+a+new+intron+submodelBioinformatics19Suppl+22003ii215ii225" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref139" title="View in article">144.</a></div><div>Stanley-Samuelson, D.W. ∙ Jurenka, R.A. ∙ Cripps, C. ...</div><div><strong>Fatty acids in insects: Composition, metabolism, and biological significance</strong></div><div><em>Arch. Insect Biochem.</em> 1988; <strong>9</strong>:1-33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.W.Stanley-SamuelsonR.A.JurenkaC.CrippsG.J.BlomquistM.de+RenobalesFatty+acids+in+insects%3A+Composition%2C+metabolism%2C+and+biological+significanceArch.+Insect+Biochem.91988133" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib140" class="citations"><div id="sref140" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref140" title="View in article">145.</a></div><div>Stefely, J.A. ∙ Pagliarini, D.J.</div><div><strong>Biochemistry of Mitochondrial Coenzyme Q Biosynthesis</strong></div><div><em>Trends Biochem. Sci.</em> 2017; <strong>42</strong>:824-843</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.A.StefelyD.J.PagliariniBiochemistry+of+Mitochondrial+Coenzyme+Q+BiosynthesisTrends+Biochem.+Sci.422017824843" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib141" class="citations"><div id="sref141" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref141-1" title="View in article">146.</a></div><div>Steidle, J.L.M. ∙ Dettner, K.</div><div><strong>Chemistry and morphology of the tergal gland of freeliving adult Aleocharinae (Coleoptera: Staphylinidae) and its phylogenetic significance</strong></div><div><em>Syst. Entomol.</em> 1993; <strong>18</strong>:149-168</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.L.M.SteidleK.DettnerChemistry+and+morphology+of+the+tergal+gland+of+freeliving+adult+Aleocharinae+%28Coleoptera%3A+Staphylinidae%29+and+its+phylogenetic+significanceSyst.+Entomol.181993149168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib142" class="citations"><div id="sref142" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref142-1" title="View in article">147.</a></div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.StuartA.ButlerP.HoffmanC.HafemeisterE.PapalexiW.M.Mauck3rdY.HaoM.StoeckiusP.SmibertR.SatijaComprehensive+Integration+of+Single-Cell+DataCell177201918881902.e21" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib143" class="citations"><div id="sref143" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref143" title="View in article">148.</a></div><div>Surm, J.M. ∙ Moran, Y.</div><div><strong>Insights into how development and life-history dynamics shape the evolution of venom</strong></div><div><em>Evodevo.</em> 2021; <strong>12</strong>:1</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.SurmY.MoranInsights+into+how+development+and+life-history+dynamics+shape+the+evolution+of+venomEvodevo1220211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib144" class="citations"><div id="sref144" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144" title="View in article">149.</a></div><div>Suzuki, Y. ∙ Takiya, S. ∙ Suzuki, T. ...</div><div><strong>Developmental regulation of silk gene expression in Bombyx mori</strong></div><div><div>Hagedorn, H.H. ∙ Hildebrand, J.G. ∙ Kidwell, M.G. ... (Editors)</div></div><div><strong>Molecular Insect Science</strong></div><div>SpringerLink, 1990; 83-89</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.SuzukiS.TakiyaT.SuzukiC.HuiK.MatsunoM.FukutaT.NagataK.UenoDevelopmental+regulation+of+silk+gene+expression+in+Bombyx+moriH.H.HagedornJ.G.HildebrandM.G.KidwellJ.H.LawMolecular+Insect+Science1990SpringerLink8389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib145" class="citations"><div id="sref145" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145-1" title="View in article">150.</a></div><div>Ter-Hovhannisyan, V. ∙ Lomsadze, A. ∙ Chernoff, Y.O. ...</div><div><strong>Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training</strong></div><div><em>Genome Res.</em> 2008; <strong>18</strong>:1979-1990</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=V.Ter-HovhannisyanA.LomsadzeY.O.ChernoffM.BorodovskyGene+prediction+in+novel+fungal+genomes+using+an+ab+initio+algorithm+with+unsupervised+trainingGenome+Res.18200819791990" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib146" class="citations"><div id="sref146" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146" title="View in article">151.</a></div><div>Tomoyasu, Y. ∙ Denell, R.E.</div><div><strong>Larval RNAi in Tribolium (Coleoptera) for analyzing adult development</strong></div><div><em>Dev. Genes Evol.</em> 2004; <strong>214</strong>:575-578</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.TomoyasuR.E.DenellLarval+RNAi+in+Tribolium+%28Coleoptera%29+for+analyzing+adult+developmentDev.+Genes+Evol.2142004575578" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib147" class="citations"><div id="sref147" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref147" title="View in article">152.</a></div><div>Torres, J.P. ∙ Schmidt, E.W.</div><div><strong>The biosynthetic diversity of the animal world</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2019; <strong>294</strong>:17684-17692</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.P.TorresE.W.SchmidtThe+biosynthetic+diversity+of+the+animal+worldJ.%C2%A0Biol.+Chem.29420191768417692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib148" class="citations"><div id="sref148" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">153.</a></div><div>Torres, J.P. ∙ Lin, Z. ∙ Winter, J.M. ...</div><div><strong>Animal biosynthesis of complex polyketides in a photosynthetic partnership</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2882</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.P.TorresZ.LinJ.M.WinterP.J.KrugE.W.SchmidtAnimal+biosynthesis+of+complex+polyketides+in+a+photosynthetic+partnershipNat.+Commun.1120202882" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib163" class="citations"><div id="opt9Cj5loRWYd" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-opt9Cj5loRWYd-1" title="View in article">154.</a></div><div>VanDrisse, C.M. ∙ Escalante-Semerena, J.C.</div><div><strong>New high-cloning-efficiency vectors for complementation studies and recombinant protein overproduction in Escherichia coli and Salmonella enterica</strong></div><div><em>Plasmid.</em> 2016; <strong>86</strong>:1-6</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.M.VanDrisseJ.C.Escalante-SemerenaNew+high-cloning-efficiency+vectors+for+complementation+studies+and+recombinant+protein+overproduction+in+Escherichia+coli+and+Salmonella+entericaPlasmid86201616" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib164" class="citations"><div id="oref0020" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-oref0020-1" title="View in article">155.</a></div>Van Rossum, G., &amp; Drake, F. L. (2000). <i>Python reference manual</i>. (iUniverse).</div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Van+Rossum%2C+G.%2C+%26+Drake%2C+F.+L.+%282000%29.+Python+reference+manual.+%28iUniverse%29." target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib149" class="citations"><div id="sref149" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref149" title="View in article">156.</a></div><div>Vucic-Pestic, O. ∙ Birkhofer, K. ∙ Rall, B.C. ...</div><div><strong>Habitat structure and prey aggregation determine the functional response in a soil predator–prey interaction</strong></div><div><em>Pedobiologia (Jena).</em> 2010; <strong>53</strong>:307-312</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=O.Vucic-PesticK.BirkhoferB.C.RallS.ScheuU.BroseHabitat+structure+and+prey+aggregation+determine+the+functional+response+in+a+soil+predator%E2%80%93prey+interactionPedobiologia+%28Jena%29532010307312" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib150" class="citations"><div id="sref150" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref150" title="View in article">157.</a></div><div>Wagner, G.P. ∙ Lynch, V.J.</div><div><strong>Evolutionary novelties</strong></div><div><em>Curr. Biol.</em> 2010; <strong>20</strong>:R48-R52</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.P.WagnerV.J.LynchEvolutionary+noveltiesCurr.+Biol.202010R48R52" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib151" class="citations"><div id="sref151" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">158.</a></div><div>Wagner, J.M. ∙ Naragon, T.H. ∙ Brückner, A.</div><div><strong>Benzoquinones in the defensive secretion of a bug (Pamillia behrensii): a common chemical trait retrieved in the Heteroptera</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=J.M.WagnerT.H.NaragonA.Br%C3%BCcknerBenzoquinones+in+the+defensive+secretion+of+a+bug+%28Pamillia+behrensii%29%3A+a+common+chemical+trait+retrieved+in+the+HeteropterabioRxiv202010.1101%2F2020.12.11.421891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib152" class="citations"><div id="sref152" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref152" title="View in article">159.</a></div><div>Wakil, S.J. ∙ Stoops, J.K. ∙ Joshi, V.C.</div><div><strong>Fatty acid synthesis and its regulation</strong></div><div><em>Annu. Rev. Biochem.</em> 1983; <strong>52</strong>:537-579</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.J.WakilJ.K.StoopsV.C.JoshiFatty+acid+synthesis+and+its+regulationAnnu.+Rev.+Biochem.521983537579" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib153" class="citations"><div id="sref153" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref153-1" title="View in article">160.</a></div><div>Wang, Y. ∙ Hekimi, S.</div><div><strong>The Complexity of Making Ubiquinone</strong></div><div><em>Trends Endocrinol. Metab.</em> 2019; <strong>30</strong>:929-943</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Y.WangS.HekimiThe+Complexity+of+Making+UbiquinoneTrends+Endocrinol.+Metab.302019929943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib154" class="citations"><div id="sref154" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref154" title="View in article">161.</a></div><div>Way, G.P. ∙ Greene, C.S.</div><div><strong>Discovering Pathway and Cell Type Signatures in Transcriptomic Compendia with Machine Learning</strong></div><div><em>Annu. Rev. Biomed. Data Sci.</em> 2019; <strong>2</strong>:1-17</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.P.WayC.S.GreeneDiscovering+Pathway+and+Cell+Type+Signatures+in+Transcriptomic+Compendia+with+Machine+LearningAnnu.+Rev.+Biomed.+Data+Sci.22019117" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib155" class="citations"><div id="sref155" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref155" title="View in article">162.</a></div><div>West-Eberhard, M.J.</div><div><strong>Developmental Plasticity and Evolution</strong></div><div>Oxford University Press, 2003</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.J.West-EberhardDevelopmental+Plasticity+and+Evolution2003Oxford+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib156" class="citations"><div id="sref156" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref156-1" title="View in article">163.</a></div><div>Wickham, H.</div><div><strong>ggplot2</strong></div><div><em>Wiley Interdiscip. Rev. Comput. Stat.</em> 2011; <strong>3</strong>:180-185</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.Wickhamggplot2Wiley+Interdiscip.+Rev.+Comput.+Stat.32011180185" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib157" class="citations"><div id="sref157" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref157-1" title="View in article">164.</a></div><div>Wickham, H. ∙ Ruiz, E.</div><div><strong>dbplyr: A “dplyr” Back End for Databases. R&nbsp;package Version 1.4. 0</strong></div><div>2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamE.Ruizdbplyr%3A+A+%E2%80%9Cdplyr%E2%80%9D+Back+End+for+Databases.+R%C2%A0package+Version+1.4.+02019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib158" class="citations"><div id="sref158" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref158-1" title="View in article">165.</a></div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the Tidyverse</strong></div><div><em>J.&nbsp;Open Source Softw.</em> 2019; <strong>4</strong>:1686</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamM.AverickJ.BryanW.ChangL.D.McGowanR.Fran%C3%A7oisG.GrolemundA.HayesL.HenryJ.HesterWelcome+to+the+TidyverseJ.%C2%A0Open+Source+Softw.420191686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib159" class="citations"><div id="sref159" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref159-1" title="View in article">166.</a></div><div>Wilke, C.</div><div><strong>Package ‘cowplot’: Streamlined Plot Theme and Plot Annotations for ‘ggplot2.’ CRAN</strong></div><div>2017</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.WilkePackage+%E2%80%98cowplot%E2%80%99%3A+Streamlined+Plot+Theme+and+Plot+Annotations+for+%E2%80%98ggplot2.%E2%80%99+CRAN2017" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib160" class="citations"><div id="sref160" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref160-1" title="View in article">167.</a></div><div>Yu, G. ∙ Wang, L.-G. ∙ Han, Y. ...</div><div><strong>clusterProfiler: an R package for comparing biological themes among gene clusters</strong></div><div><em>OMICS.</em> 2012; <strong>16</strong>:284-287</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.YuL.-G.WangY.HanQ.-Y.HeclusterProfiler%3A+an+R+package+for+comparing+biological+themes+among+gene+clustersOMICS162012284287" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none linkback-ad widget-" id="684c6365-474d-4ce7-932a-ddffa4f64d88">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <h2>CHORUS Manuscript</h2>
<p style="margin-bottom:5px;">
<a target="_blank" href="/action/clickThrough?id=2959&amp;url=https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fam%2Fpii%2FS0092867421013295&amp;loc=%2Fcell%2Ffulltext%2FS0092-8674%2821%2901329-5&amp;pubId=60887474&amp;placeholderId=1346&amp;productId=1822"> View Open Manuscript</a>
</p>
        </div></div>
                </div>
                </div>

        
</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/b5b9ec2b-06bd-436f-a2c3-4ad5ef91b5cf/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/06e85581-153f-46ba-8eac-3cf3df5515c9/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/08a3f340-bf19-4895-ba13-aad79b1c4379/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/1eb1ea84-c9d7-43e1-8eff-a7906c8171da/main.assets/figs3.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/c9ebf03a-a86d-4e60-a6dc-bed87f77c318/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/6f703b5f-ae7c-45bf-a3f7-11e41be245af/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/d22ecbd5-e676-4add-8386-750362e20abe/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/70cfaef6-e221-48c9-b287-35dda1da7da2/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/154b76d6-11d8-422b-9226-f8efcf730795/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/cd0a7829-150c-4141-94c6-dcc4505a900f/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/ea283830-bf0a-4ff5-af59-9b1904c6008e/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/d4bba9d5-0132-435e-accc-52c8c6313687/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/9b541a00-aca9-439a-960b-31dac089d22b/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/asset/a74801d7-ace2-4ff5-8d54-d09c0ff51bc9/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867421013295%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00262%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (7)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/5bd46517-8fff-4eff-b676-180daa3b5184/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (502.68 KB)</span></a><div class="core-description">Data S1. Phylogeny of α-esterases, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/edb53133-805a-4e39-8bbe-456b0d643998/mmc2.pdf" download="mmc2.pdf"><span id="mmc2-link">PDF (13.29 MB)</span></a><div class="core-description">Data S2. Oenocyte CHC pathway, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/b7ebe1b3-1bbe-4c2f-b548-77ccf207d8cd/mmc3.pdf" download="mmc3.pdf"><span id="mmc3-link">PDF (3.56 MB)</span></a><div class="core-description">Data S3. Diagnostic plots for K selection in cNMF, related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/fbe3b1d4-6a20-4e61-b011-85eb6cd21e19/mmc4.pdf" download="mmc4.pdf"><span id="mmc4-link">PDF (1.28 MB)</span></a><div class="core-description">Data S4. GEP9 composition, related to Figure&nbsp;5</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/ead1fade-aad8-47aa-b2ca-03cd3a4dda3f/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (90.09 KB)</span></a><div class="core-description">Data S5. RNAi survival assay without ants, extensional viscosities of synthetic secretions, and lag times of bacterial growth, related to Figure&nbsp;7</div></div></div><div id="collapsible-supplementary" data-expandable="item"><div class="content" style="display:none;" role="region" id="collapsible-supplementary_content"><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/88e18132-b669-46f7-975f-49bbd7e1ae3a/mmc6.pdf" download="mmc6.pdf"><span id="mmc6-link">PDF (501.71 KB)</span></a><div class="core-description">Data S6. <i>Dalotia</i> laccase <i>in&nbsp;vitro</i> binding assays, related to Figure&nbsp;6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.11.014/attachment/039bb8fb-9d9f-4188-a1c7-78d0dd8466f1/mmc7.xlsx" download="mmc7.xlsx"><span id="mmc7-link">Spreadsheet (16.74 KB)</span></a><div class="core-description">Table S1. Statistical analyses of all experimental data presented in the main and supplementary figures, related to Figures 1, 2, 4, 6, and 7</div></div></div></div><button class="trigger" aria-controls="collapsible-supplementary_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show more</span><span class="collapse">Show less</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421013295" class="ux3DeferredOnScrollWidget axel-related-articles__body" data-show-more="true"></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.11.014","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Evolutionary assembly of cooperating cell types in an animal chemical defense system</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.014/asset/b5b9ec2b-06bd-436f-a2c3-4ad5ef91b5cf/main.assets/gr1.jpg" height="420" width="743" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/23322283-9a93-4480-b6a1-48d1b9998b5f/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.014/asset/06e85581-153f-46ba-8eac-3cf3df5515c9/main.assets/figs1.jpg" height="529" width="659" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/df2f6aa3-4cac-4d40-b87a-968b0103529e/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.014/asset/08a3f340-bf19-4895-ba13-aad79b1c4379/main.assets/figs2.jpg" height="960" width="753" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/aad40f19-2ac3-4f63-9496-df101a1241bf/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.014/asset/1eb1ea84-c9d7-43e1-8eff-a7906c8171da/main.assets/figs3.jpg" height="942" width="687" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/7023b27a-b2fe-4449-8c25-e63d6fd07d48/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.014/asset/c9ebf03a-a86d-4e60-a6dc-bed87f77c318/main.assets/gr2.jpg" height="394" width="758" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/f8ab2808-32c5-4bb9-a9d1-3f7c6789a4ee/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.014/asset/6f703b5f-ae7c-45bf-a3f7-11e41be245af/main.assets/gr3.jpg" height="596" width="771" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/6b790b50-5b30-4f75-b717-6f1f2240d23b/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.014/asset/d22ecbd5-e676-4add-8386-750362e20abe/main.assets/figs4.jpg" height="732" width="677" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/81e2f6a1-5023-4144-a6a5-44cacfea880c/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.014/asset/70cfaef6-e221-48c9-b287-35dda1da7da2/main.assets/gr4.jpg" height="821" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/fcac9bf3-f69b-4f1a-af8f-d00fe996615b/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.014/asset/154b76d6-11d8-422b-9226-f8efcf730795/main.assets/figs5.jpg" height="939" width="680" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/69afec1c-2edd-4ea1-b086-3310e602f42c/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.014/asset/cd0a7829-150c-4141-94c6-dcc4505a900f/main.assets/gr5.jpg" height="644" width="752" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/84df5429-e453-4df5-9f8f-6ea107e96036/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.014/asset/ea283830-bf0a-4ff5-af59-9b1904c6008e/main.assets/figs6.jpg" height="857" width="656" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/584ea0aa-67d4-4f3b-8ea0-a0a2eaa0127b/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.014/asset/d4bba9d5-0132-435e-accc-52c8c6313687/main.assets/gr6.jpg" height="941" width="732" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/815e9a68-dd13-4177-8b9f-fa054eae0806/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.014/asset/9b541a00-aca9-439a-960b-31dac089d22b/main.assets/gr7.jpg" height="679" width="753" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/5897f4ac-a5b3-498a-b7ba-5f06ee1774ed/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.014/asset/a74801d7-ace2-4ff5-8d54-d09c0ff51bc9/main.assets/figs7.jpg" height="845" width="769" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/6c1dde58-4ad7-4f76-9252-aab13da99284/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.11.014/asset/b5b9ec2b-06bd-436f-a2c3-4ad5ef91b5cf/main.assets/gr1.jpg" height="420" width="743" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/23322283-9a93-4480-b6a1-48d1b9998b5f/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.11.014/asset/06e85581-153f-46ba-8eac-3cf3df5515c9/main.assets/figs1.jpg" height="529" width="659" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/df2f6aa3-4cac-4d40-b87a-968b0103529e/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.11.014/asset/08a3f340-bf19-4895-ba13-aad79b1c4379/main.assets/figs2.jpg" height="960" width="753" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/aad40f19-2ac3-4f63-9496-df101a1241bf/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.11.014/asset/1eb1ea84-c9d7-43e1-8eff-a7906c8171da/main.assets/figs3.jpg" height="942" width="687" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/7023b27a-b2fe-4449-8c25-e63d6fd07d48/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.11.014/asset/c9ebf03a-a86d-4e60-a6dc-bed87f77c318/main.assets/gr2.jpg" height="394" width="758" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/f8ab2808-32c5-4bb9-a9d1-3f7c6789a4ee/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.11.014/asset/6f703b5f-ae7c-45bf-a3f7-11e41be245af/main.assets/gr3.jpg" height="596" width="771" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/6b790b50-5b30-4f75-b717-6f1f2240d23b/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.11.014/asset/d22ecbd5-e676-4add-8386-750362e20abe/main.assets/figs4.jpg" height="732" width="677" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/81e2f6a1-5023-4144-a6a5-44cacfea880c/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.11.014/asset/70cfaef6-e221-48c9-b287-35dda1da7da2/main.assets/gr4.jpg" height="821" width="753" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/fcac9bf3-f69b-4f1a-af8f-d00fe996615b/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2021.11.014/asset/154b76d6-11d8-422b-9226-f8efcf730795/main.assets/figs5.jpg" height="939" width="680" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/69afec1c-2edd-4ea1-b086-3310e602f42c/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2021.11.014/asset/cd0a7829-150c-4141-94c6-dcc4505a900f/main.assets/gr5.jpg" height="644" width="752" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/84df5429-e453-4df5-9f8f-6ea107e96036/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2021.11.014/asset/ea283830-bf0a-4ff5-af59-9b1904c6008e/main.assets/figs6.jpg" height="857" width="656" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/584ea0aa-67d4-4f3b-8ea0-a0a2eaa0127b/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2021.11.014/asset/d4bba9d5-0132-435e-accc-52c8c6313687/main.assets/gr6.jpg" height="941" width="732" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/815e9a68-dd13-4177-8b9f-fa054eae0806/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2021.11.014/asset/9b541a00-aca9-439a-960b-31dac089d22b/main.assets/gr7.jpg" height="679" width="753" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/5897f4ac-a5b3-498a-b7ba-5f06ee1774ed/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2021.11.014/asset/a74801d7-ace2-4ff5-8d54-d09c0ff51bc9/main.assets/figs7.jpg" height="845" width="769" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2021.11.014/asset/6c1dde58-4ad7-4f76-9252-aab13da99284/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01329-5';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)01329-5';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749592105417","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"25","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421013295","coverDate":"202112","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Evolutionary assembly of cooperating cell types in an animal chemical defense system","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc129bbb344298',t:'MTc0OTU5MjEwNC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc129bbb344298&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.11.014&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div></body></html>