<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="1b3ecab9-026f-4d3f-875c-bc3bfa093ba4" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;issue:issue:pii\:S0092867420X00134;article:article:pii\:S0092867421005754;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="The deep population history of northern East Asia from the Late Pleistocene to the Holocene">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(21)00575-4">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/88ef2c85-7eac-41f4-a47e-727c3741da7f/fx1.jpg">
    


        <meta property="og:description" content="Mao et al. demonstrate that the ancestry of the ca. 40 ka individual from Tianyuan
Cave near Beijing was widespread geographically and temporally before the Last Glacial
Maximum (LGM; ca. 26.5–19 ka). At the end of the LGM, the earliest northern East Asian
appeared. After 14 ka, populations in the Amur region comprised the closest East Asian
source known for Ancient Paleo-Siberians. Among adaptive genetic variants, EDAR V370A
was likely to have been elevated to high frequency after the LGM.">
    











        <meta name="twitter:title" content="The deep population history of northern East Asia from the Late Pleistocene to the Holocene">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/88ef2c85-7eac-41f4-a47e-727c3741da7f/fx1.jpg">
    


        <meta name="twitter:description" content="Mao et al. demonstrate that the ancestry of the ca. 40 ka individual from Tianyuan
Cave near Beijing was widespread geographically and temporally before the Last Glacial
Maximum (LGM; ca. 26.5–19 ka). At the end of the LGM, the earliest northern East Asian
appeared. After 14 ka, populations in the Amur region comprised the closest East Asian
source known for Ancient Paleo-Siberians. Among adaptive genetic variants, EDAR V370A
was likely to have been elevated to high frequency after the LGM.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(21)00575-4">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>The deep population history of northern East Asia from the Late Pleistocene to the Holocene: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="The deep population history of northern East Asia from the Late Pleistocene to the Holocene"> <meta name="citation_date" content="2021/06/10"> <meta name="citation_online_date" content="2021/05/27"> <meta name="citation_volume" content="184"> <meta name="citation_issue" content="12"> <meta name="citation_firstpage" content="3256"> <meta name="citation_lastpage" content="3266.e13"> <meta name="citation_pmid" content="34048699"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867421005754"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867421005754"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867421005754"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2021.04.040"> <meta name="citation_pmid" content="34048699"> <meta name="citation_author" content="Xiaowei Mao"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Hucai Zhang"><meta name="citation_author_email" content="zhanghc@ynu.edu.cn"><meta name="citation_author_institution" content="Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China"><meta name="citation_author_institution" content="CAS Center for Excellence in Tibetan Plateau Earth Sciences, Chinese Academy of Sciences (CAS), Beijing 100101, China"><meta name="citation_author" content="Shiyu Qiao"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author_institution" content="Shanghai Qi Zhi Institute, Shanghai 200232, China"><meta name="citation_author_institution" content="University of the Chinese Academy of Sciences, Beijing 100049, China"><meta name="citation_author" content="Yichen Liu"><meta name="citation_author_orcid" content="0000-0002-7187-6232"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author_institution" content="Shanghai Qi Zhi Institute, Shanghai 200232, China"><meta name="citation_author" content="Fengqin Chang"><meta name="citation_author_institution" content="Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China"><meta name="citation_author" content="Ping Xie"><meta name="citation_author_institution" content="Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China"><meta name="citation_author" content="Ming Zhang"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Tianyi Wang"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author_institution" content="Northwest University, Xi’an 710069, China"><meta name="citation_author" content="Mian Li"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author_institution" content="Shandong First Medical University &amp; Shandong Academy of Medical Sciences, Jinan 250000, China"><meta name="citation_author" content="Peng Cao"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Ruowei Yang"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Feng Liu"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Qingyan Dai"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Xiaotian Feng"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Wanjing Ping"><meta name="citation_author_orcid" content="0000-0003-4318-0673"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Chuzhao Lei"><meta name="citation_author_institution" content="Key Laboratory of Animal Genetics, Breeding and Reproduction of Shaanxi Province, College of Animal Science and Technology, Northwest A&amp;F University, Yangling 712100, China"><meta name="citation_author" content="John W. Olsen"><meta name="citation_author_orcid" content="0000-0001-5295-7451"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author_institution" content="School of Anthropology, University of Arizona, Tucson, AZ 85721-0030, USA"><meta name="citation_author" content="E. Andrew Bennett"><meta name="citation_author_orcid" content="0000-0003-3827-0711"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author" content="Qiaomei Fu"><meta name="citation_author_email" content="fuqiaomei@ivpp.ac.cn"><meta name="citation_author_institution" content="Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China"><meta name="citation_author_institution" content="Shanghai Qi Zhi Institute, Shanghai 200232, China"><meta name="citation_author_institution" content="University of the Chinese Academy of Sciences, Beijing 100049, China"><meta name="citation_keywords" content="ancient DNA; northern East Asia; Last Glacial Maximum; population dynamics"><meta name="citation_reference" content="citation_title=Fast model-based estimation of ancestry in unrelated individuals; citation_author=D.H. Alexander; citation_author=J. Novembre; citation_author=K. Lange; citation_journal_title=Genome Res.; citation_volume=19; citation_firstpage=1655; citation_lastpage=1664; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA; citation_author=R.M. Andrews; citation_author=I. Kubacka; citation_author=P.F. Chinnery; citation_author=R.N. Lightowlers; citation_author=D.M. Turnbull; citation_author=N. Howell; citation_journal_title=Nat. Genet.; citation_volume=23; citation_firstpage=147; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=A global reference for human genetic variation; citation_author=A. Auton; citation_author=L.D. Brooks; citation_author=R.M. Durbin; citation_author=E.P. Garrison; citation_author=H.M. Kang; citation_author=J.O. Korbel; citation_author=J.L. Marchini; citation_author=S. McCarthy; citation_author=G.A. McVean; citation_author=G.R. Abecasis; citation_author=1000 Genomes Project Consortium; citation_journal_title=Nature; citation_volume=526; citation_firstpage=68; citation_lastpage=74; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A concise history of world population; citation_author=M.L. Bacci; citation_publication_date=2017; citation_publisher=John Wiley &amp; Sons; "><meta name="citation_reference" content="citation_title=On the origin of modern humans: Asian perspectives; citation_author=C.J. Bae; citation_author=K. Douka; citation_author=M.D. Petraglia; citation_journal_title=Science; citation_volume=358; citation_firstpage=eaai9067; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=PHYLIP: Phylogeny Inference Package. Version 3.2. Joel Felsenstein; citation_author=B.R. Baum; citation_journal_title=Q.&nbsp;Rev. Biol.; citation_volume=64; citation_firstpage=539; citation_lastpage=541; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=The last glacial maximum; citation_author=P.U. Clark; citation_author=A.S. Dyke; citation_author=J.D. Shakun; citation_author=A.E. Carlson; citation_author=J. Clark; citation_author=B. Wohlfarth; citation_author=J.X. Mitrovica; citation_author=S.W. Hostetler; citation_author=A.M. McCabe; citation_journal_title=Science; citation_volume=325; citation_firstpage=710; citation_lastpage=714; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Database Resources of the National Genomics Data Center, China National Center for Bioinformation in 2021; citation_author=CNCB-NGDC Members and Partners; citation_journal_title=Nucleic Acids Res.; citation_volume=49; citation_number=D1; citation_firstpage=D18; citation_lastpage=D28; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The first horse herders and the impact of early Bronze Age steppe expansions into Asia; citation_author=P. de Barros Damgaard; citation_author=R. Martiniano; citation_author=J. Kamm; citation_author=J.V. Moreno-Mayar; citation_author=G. Kroonen; citation_author=M. Peyrot; citation_author=G. Barjamovic; citation_author=S. Rasmussen; citation_author=C. Zacho; citation_author=N. Baimukhanov; citation_author=et al.; citation_journal_title=Science; citation_volume=360; citation_firstpage=eaar7711; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Compound-specific radiocarbon dating and mitochondrial DNA analysis of the Pleistocene hominin from Salkhit Mongolia; citation_author=T. Devièse; citation_author=D. Massilani; citation_author=S. Yi; citation_author=D. Comeskey; citation_author=S. Nagel; citation_author=B. Nickel; citation_author=E. Ribechini; citation_author=J. Lee; citation_author=D. Tseveendorj; citation_author=B. Gunchinsuren; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=10; citation_firstpage=274; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Farmers and their languages: the first expansions; citation_author=J. Diamond; citation_author=P. Bellwood; citation_journal_title=Science; citation_volume=300; citation_firstpage=597; citation_lastpage=603; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Ancient mitogenomes show plateau populations from last 5200 years partially contributed to present-day Tibetans; citation_author=M. Ding; citation_author=T. Wang; citation_author=A.M.-S. Ko; citation_author=H. Chen; citation_author=H. Wang; citation_author=G. Dong; citation_author=H. Lu; citation_author=W. He; citation_author=S. Wangdue; citation_author=H. Yuan; citation_author=et al.; citation_journal_title=Proc. Biol. Sci.; citation_volume=287; citation_firstpage=20192968; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=DNA analysis of an early modern human from Tianyuan Cave, China; citation_author=Q. Fu; citation_author=M. Meyer; citation_author=X. Gao; citation_author=U. Stenzel; citation_author=H.A. Burbano; citation_author=J. Kelso; citation_author=S. Pääbo; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=2223; citation_lastpage=2227; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A revised timescale for human evolution based on ancient mitochondrial genomes; citation_author=Q. Fu; citation_author=A. Mittnik; citation_author=P.L.F. Johnson; citation_author=K. Bos; citation_author=M. Lari; citation_author=R. Bollongino; citation_author=C. Sun; citation_author=L. Giemsch; citation_author=R. Schmitz; citation_author=J. Burger; citation_author=et al.; citation_journal_title=Curr. Biol.; citation_volume=23; citation_firstpage=553; citation_lastpage=559; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Genome sequence of a 45,000-year-old modern human from western Siberia; citation_author=Q. Fu; citation_author=H. Li; citation_author=P. Moorjani; citation_author=F. Jay; citation_author=S.M. Slepchenko; citation_author=A.A. Bondarev; citation_author=P.L.F. Johnson; citation_author=A. Aximu-Petri; citation_author=K. Prüfer; citation_author=C. de Filippo; citation_author=et al.; citation_journal_title=Nature; citation_volume=514; citation_firstpage=445; citation_lastpage=449; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=An early modern human from Romania with a recent Neanderthal ancestor; citation_author=Q. Fu; citation_author=M. Hajdinjak; citation_author=O.T. Moldovan; citation_author=S. Constantin; citation_author=S. Mallick; citation_author=P. Skoglund; citation_author=N. Patterson; citation_author=N. Rohland; citation_author=I. Lazaridis; citation_author=B. Nickel; citation_author=et al.; citation_journal_title=Nature; citation_volume=524; citation_firstpage=216; citation_lastpage=219; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The genetic history of Ice Age Europe; citation_author=Q. Fu; citation_author=C. Posth; citation_author=M. Hajdinjak; citation_author=M. Petr; citation_author=S. Mallick; citation_author=D. Fernandes; citation_author=A. Furtwängler; citation_author=W. Haak; citation_author=M. Meyer; citation_author=A. Mittnik; citation_author=et al.; citation_journal_title=Nature; citation_volume=534; citation_firstpage=200; citation_lastpage=205; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=A replication study confirmed the EDAR gene to be a major contributor to population differentiation regarding head hair thickness in Asia; citation_author=A. Fujimoto; citation_author=J. Ohashi; citation_author=N. Nishida; citation_author=T. Miyagawa; citation_author=Y. Morishita; citation_author=T. Tsunoda; citation_author=R. Kimura; citation_author=K. Tokunaga; citation_journal_title=Hum. Genet.; citation_volume=124; citation_firstpage=179; citation_lastpage=185; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Genome flux and stasis in a five millennium transect of European prehistory; citation_author=C. Gamba; citation_author=E.R. Jones; citation_author=M.D. Teasdale; citation_author=R.L. McLaughlin; citation_author=G. Gonzalez-Fortes; citation_author=V. Mattiangeli; citation_author=L. Domboróczki; citation_author=I. Kővári; citation_author=I. Pap; citation_author=A. Anders; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=5; citation_firstpage=5257; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA; citation_author=M.-T. Gansauge; citation_author=M. Meyer; citation_journal_title=Nat. Protoc.; citation_volume=8; citation_firstpage=737; citation_lastpage=748; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Massive migration from the steppe was a source for Indo-European languages in Europe; citation_author=W. Haak; citation_author=I. Lazaridis; citation_author=N. Patterson; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=B. Llamas; citation_author=G. Brandt; citation_author=S. Nordenfelt; citation_author=E. Harney; citation_author=K. Stewardson; citation_author=et al.; citation_journal_title=Nature; citation_volume=522; citation_firstpage=207; citation_lastpage=211; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Assessing the performance of qpAdm: a statistical tool for studying population admixture; citation_author=É. Harney; citation_author=N. Patterson; citation_author=D. Reich; citation_author=J. Wakeley; citation_journal_title=Genetics; citation_volume=217; citation_firstpage=iyaa045; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=A century of feasting studies; citation_author=B. Hayden; citation_author=S. Villeneuve; citation_journal_title=Annu. Rev. Anthropol.; citation_volume=40; citation_firstpage=433; citation_lastpage=449; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The excavation of Zone III at the Xiaonanshan Site in 2015, Raohe County, Heilongjiang; citation_author=Heilongjiang Provincial Institute of Cultural Relics and Archaeology; citation_author=Commission for Preservation of Ancient Monuments, Raohe County; citation_journal_title=Chinese Archaeology; citation_volume=20; citation_firstpage=87; citation_lastpage=96; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Environmental selection during the last ice age on the mother-to-infant transmission of vitamin D and fatty acids through breast milk; citation_author=L.J. Hlusko; citation_author=J.P. Carlson; citation_author=G. Chaplin; citation_author=S.A. Elias; citation_author=J.F. Hoffecker; citation_author=M. Huffman; citation_author=N.G. Jablonski; citation_author=T.A. Monson; citation_author=D.H. O’Rourke; citation_author=M.A. Pilloud; citation_author=G.R. Scott; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=115; citation_firstpage=E4426; citation_lastpage=E4432; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Long-term genetic stability and a high-altitude East Asian origin for the peoples of the high valleys of the Himalayan arc; citation_author=C. Jeong; citation_author=A.T. Ozga; citation_author=D.B. Witonsky; citation_author=H. Malmström; citation_author=H. Edlund; citation_author=C.A. Hofman; citation_author=R.W. Hagan; citation_author=M. Jakobsson; citation_author=C.M. Lewis; citation_author=M.S. Aldenderfer; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=7485; citation_lastpage=7490; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The genetic history of admixture across inner Eurasia; citation_author=C. Jeong; citation_author=O. Balanovsky; citation_author=E. Lukianova; citation_author=N. Kahbatkyzy; citation_author=P. Flegontov; citation_author=V. Zaporozhchenko; citation_author=A. Immel; citation_author=C.-C. Wang; citation_author=O. Ixan; citation_author=E. Khussainova; citation_author=et al.; citation_journal_title=Nat. Ecol. Evol.; citation_volume=3; citation_firstpage=966; citation_lastpage=976; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Neolithic burial in Xiaonanshan, Raohe County, Heilongjiang; citation_author=Jiamusi Cultural Relics Management Station and Raohe County Cultural Relics Management Institute; citation_journal_title=Kaogu; citation_volume=1996; citation_firstpage=1; citation_lastpage=8; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Upper Palaeolithic genomes reveal deep roots of modern Eurasians; citation_author=E.R. Jones; citation_author=G. Gonzalez-Fortes; citation_author=S. Connell; citation_author=V. Siska; citation_author=A. Eriksson; citation_author=R. Martiniano; citation_author=R.L. McLaughlin; citation_author=M. Gallego Llorente; citation_author=L.M. Cassidy; citation_author=C. Gamba; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=8912; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Modeling recent human evolution in mice by expression of a selected EDAR variant; citation_author=Y.G. Kamberov; citation_author=S. Wang; citation_author=J. Tan; citation_author=P. Gerbault; citation_author=A. Wark; citation_author=L. Tan; citation_author=Y. Yang; citation_author=S. Li; citation_author=K. Tang; citation_author=H. Chen; citation_author=et al.; citation_journal_title=Cell; citation_volume=152; citation_firstpage=691; citation_lastpage=702; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Human population dynamics and Yersinia pestis in ancient northeast Asia; citation_author=G.M. Kılınç; citation_author=N. Kashuba; citation_author=D. Koptekin; citation_author=N. Bergfeldt; citation_author=H.M. Dönertaş; citation_author=R. Rodríguez-Varela; citation_author=D. Shergin; citation_author=G. Ivanov; citation_author=D. Kichigin; citation_author=K. Pestereva; citation_author=et al.; citation_journal_title=Sci. Adv.; citation_volume=7; citation_firstpage=eabc4587; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform; citation_author=M. Kircher; citation_author=S. Sawyer; citation_author=M. Meyer; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=e3; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=ANGSD: Analysis of Next Generation Sequencing Data; citation_author=T.S. Korneliussen; citation_author=A. Albrechtsen; citation_author=R. Nielsen; citation_journal_title=BMC Bioinformatics; citation_volume=15; citation_firstpage=356; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Radiocarbon dating and dietary reconstruction of the Early Neolithic Houtaomuga and Shuangta sites in the Song-Nen Plain, Northeast China; citation_author=D. Kunikita; citation_author=L. Wang; citation_author=S. Onuki; citation_author=H. Sato; citation_author=H. Matsuzaki; citation_journal_title=Quat. Int.; citation_volume=441; citation_firstpage=62; citation_lastpage=68; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Ancient human genomes suggest three ancestral populations for present-day Europeans; citation_author=I. Lazaridis; citation_author=N. Patterson; citation_author=A. Mittnik; citation_author=G. Renaud; citation_author=S. Mallick; citation_author=K. Kirsanow; citation_author=P.H. Sudmant; citation_author=J.G. Schraiber; citation_author=S. Castellano; citation_author=M. Lipson; citation_author=et al.; citation_journal_title=Nature; citation_volume=513; citation_firstpage=409; citation_lastpage=413; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Genomic insights into the origin of farming in the ancient Near East; citation_author=I. Lazaridis; citation_author=D. Nadel; citation_author=G. Rollefson; citation_author=D.C. Merrett; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=D. Fernandes; citation_author=M. Novak; citation_author=B. Gamarra; citation_author=K. Sirak; citation_author=et al.; citation_journal_title=Nature; citation_volume=536; citation_firstpage=419; citation_lastpage=424; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Historical Atlas of Northeast Asia, 1590-2010: Korea, Manchuria, Mongolia, Eastern Siberia; citation_author=N. Li; citation_author=R. Cribb; citation_publication_date=2014; citation_publisher=Columbia University Press; "><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Spatiotemporal variability of extreme precipitation in north and south of the Qinling-Huaihe region and influencing factors during 1960–2013; citation_author=S. Li; citation_author=S. Yang; citation_author=X. Liu; citation_journal_title=Prog. Geogr.; citation_volume=34; citation_firstpage=354; citation_lastpage=363; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The genetic prehistory of the Andean highlands 7000 years BP though European contact; citation_author=J. Lindo; citation_author=R. Haas; citation_author=C. Hofman; citation_author=M. Apata; citation_author=M. Moraga; citation_author=R.A. Verdugo; citation_author=J.T. Watson; citation_author=C. Viviano Llave; citation_author=D. Witonsky; citation_author=C. Beall; citation_author=et al.; citation_journal_title=Sci. Adv.; citation_volume=4; citation_firstpage=eaau4921; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Maximum Likelihood Estimation of Biological Relatedness from Low Coverage Sequencing Data; citation_author=M. Lipatov; citation_author=K. Sanjeev; citation_author=R. Patro; citation_author=K.R. Veeramah; citation_journal_title=bioRxiv; citation_firstpage=023374; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Applying f4 -statistics and admixture graphs: Theory and examples; citation_author=M. Lipson; citation_journal_title=Mol. Ecol. Resour.; citation_volume=20; citation_firstpage=1658; citation_lastpage=1667; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Ancient genomes document multiple waves of migration in Southeast Asian prehistory; citation_author=M. Lipson; citation_author=O. Cheronet; citation_author=S. Mallick; citation_author=N. Rohland; citation_author=M. Oxenham; citation_author=M. Pietrusewsky; citation_author=T.O. Pryce; citation_author=A. Willis; citation_author=H. Matsumura; citation_author=H. Buckley; citation_author=et al.; citation_journal_title=Science; citation_volume=361; citation_firstpage=92; citation_lastpage=95; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Ancient Ethiopian genome reveals extensive Eurasian admixture throughout the African continent; citation_author=M.G. Llorente; citation_author=E.R. Jones; citation_author=A. Eriksson; citation_author=V. Siska; citation_author=K.W. Arthur; citation_author=J.W. Arthur; citation_author=M.C. Curtis; citation_author=J.T. Stock; citation_author=M. Coltorti; citation_author=P. Pieruccini; citation_author=et al.; citation_journal_title=Science; citation_volume=350; citation_firstpage=820; citation_lastpage=822; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Ancestral Origins and Genetic History of Tibetan Highlanders; citation_author=D. Lu; citation_author=H. Lou; citation_author=K. Yuan; citation_author=X. Wang; citation_author=Y. Wang; citation_author=C. Zhang; citation_author=Y. Lu; citation_author=X. Yang; citation_author=L. Deng; citation_author=Y. Zhou; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=99; citation_firstpage=580; citation_lastpage=594; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The Simons Genome Diversity Project: 300 genomes from 142 diverse populations; citation_author=S. Mallick; citation_author=H. Li; citation_author=M. Lipson; citation_author=I. Mathieson; citation_author=M. Gymrek; citation_author=F. Racimo; citation_author=M. Zhao; citation_author=N. Chennagiri; citation_author=S. Nordenfelt; citation_author=A. Tandon; citation_author=et al.; citation_journal_title=Nature; citation_volume=538; citation_firstpage=201; citation_lastpage=206; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Denisovan ancestry and population history of early East Asians; citation_author=D. Massilani; citation_author=L. Skov; citation_author=M. Hajdinjak; citation_author=B. Gunchinsuren; citation_author=D. Tseveendorj; citation_author=S. Yi; citation_author=J. Lee; citation_author=S. Nagel; citation_author=B. Nickel; citation_author=T. Devièse; citation_author=et al.; citation_journal_title=Science; citation_volume=370; citation_firstpage=579; citation_lastpage=583; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Genome-wide patterns of selection in 230 ancient Eurasians; citation_author=I. Mathieson; citation_author=I. Lazaridis; citation_author=N. Rohland; citation_author=S. Mallick; citation_author=N. Patterson; citation_author=S.A. Roodenberg; citation_author=E. Harney; citation_author=K. Stewardson; citation_author=D. Fernandes; citation_author=M. Novak; citation_author=et al.; citation_journal_title=Nature; citation_volume=528; citation_firstpage=499; citation_lastpage=503; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The prehistoric peopling of Southeast Asia; citation_author=H. McColl; citation_author=F. Racimo; citation_author=L. Vinner; citation_author=F. Demeter; citation_author=T. Gakuhari; citation_author=J.V. Moreno-Mayar; citation_author=G. van Driem; citation_author=U. Gram Wilken; citation_author=A. Seguin-Orlando; citation_author=C. de la Fuente Castro; citation_author=et al.; citation_journal_title=Science; citation_volume=361; citation_firstpage=88; citation_lastpage=92; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Illumina sequencing library preparation for highly multiplexed target capture and sequencing; citation_author=M. Meyer; citation_author=M. Kircher; citation_journal_title=Cold Spring Harb. Protoc.; citation_volume=2010; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A high-coverage genome sequence from an archaic Denisovan individual; citation_author=M. Meyer; citation_author=M. Kircher; citation_author=M.-T. Gansauge; citation_author=H. Li; citation_author=F. Racimo; citation_author=S. Mallick; citation_author=J.G. Schraiber; citation_author=F. Jay; citation_author=K. Prüfer; citation_author=C. de Filippo; citation_author=et al.; citation_journal_title=Science; citation_volume=338; citation_firstpage=222; citation_lastpage=226; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Millennial-scale East Asian monsoon variability of the last glacial deduced from annually laminated sediments from Lake Sihailongwan, NE China; citation_author=J. Mingram; citation_author=M. Stebich; citation_author=G. Schettler; citation_author=Y. Hu; citation_author=P. Rioual; citation_author=N. Nowaczyk; citation_author=P. Dulski; citation_author=H. You; citation_author=S. Opitz; citation_author=Q. Liu; citation_author=et al.; citation_journal_title=Quat. Sci. Rev.; citation_volume=201; citation_firstpage=57; citation_lastpage=76; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Estimating genetic kin relationships in prehistoric populations; citation_author=J.M. Monroy Kuhn; citation_author=M. Jakobsson; citation_author=T. Günther; citation_journal_title=PLoS ONE; citation_volume=13; citation_firstpage=e0195491; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Early human dispersals within the Americas; citation_author=J.V. Moreno-Mayar; citation_author=L. Vinner; citation_author=P. de Barros Damgaard; citation_author=C. de la Fuente; citation_author=J. Chan; citation_author=J.P. Spence; citation_author=M.E. Allentoft; citation_author=T. Vimala; citation_author=F. Racimo; citation_author=T. Pinotti; citation_author=et al.; citation_journal_title=Science; citation_volume=362; citation_firstpage=eaav2621; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The promise of discovering population-specific disease-associated genes in South Asia; citation_author=N. Nakatsuka; citation_author=P. Moorjani; citation_author=N. Rai; citation_author=B. Sarkar; citation_author=A. Tandon; citation_author=N. Patterson; citation_author=G.S. Bhavani; citation_author=K.M. Girisha; citation_author=M.S. Mustak; citation_author=S. Srinivasan; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1403; citation_lastpage=1407; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The formation of human populations in South and Central Asia; citation_author=V.M. Narasimhan; citation_author=N. Patterson; citation_author=P. Moorjani; citation_author=N. Rohland; citation_author=R. Bernardos; citation_author=S. Mallick; citation_author=I. Lazaridis; citation_author=N. Nakatsuka; citation_author=I. Olalde; citation_author=M. Lipson; citation_author=et al.; citation_journal_title=Science; citation_volume=365; citation_firstpage=eaat7487; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Ancient genomes from northern China suggest links between subsistence changes and human migration; citation_author=C. Ning; citation_author=T. Li; citation_author=K. Wang; citation_author=F. Zhang; citation_author=T. Li; citation_author=X. Wu; citation_author=S. Gao; citation_author=Q. Zhang; citation_author=H. Zhang; citation_author=M.J. Hudson; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=11; citation_firstpage=2700; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Derived immune and ancestral pigmentation alleles in a 7,000-year-old Mesolithic European; citation_author=I. Olalde; citation_author=M.E. Allentoft; citation_author=F. Sánchez-Quinto; citation_author=G. Santpere; citation_author=C.W.K. Chiang; citation_author=M. DeGiorgio; citation_author=J. Prado-Martinez; citation_author=J.A. Rodríguez; citation_author=S. Rasmussen; citation_author=J. Quilez; citation_author=et al.; citation_journal_title=Nature; citation_volume=507; citation_firstpage=225; citation_lastpage=228; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Population structure and eigenanalysis; citation_author=N. Patterson; citation_author=A.L. Price; citation_author=D. Reich; citation_journal_title=PLoS Genet.; citation_volume=2; citation_firstpage=e190; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Ancient admixture in human history; citation_author=N. Patterson; citation_author=P. Moorjani; citation_author=Y. Luo; citation_author=S. Mallick; citation_author=N. Rohland; citation_author=Y. Zhan; citation_author=T. Genschoreck; citation_author=T. Webster; citation_author=D. Reich; citation_journal_title=Genetics; citation_volume=192; citation_firstpage=1065; citation_lastpage=1093; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The social context of early pottery in the Lingnan region of south China; citation_author=R. Pearson; citation_journal_title=Antiquity; citation_volume=79; citation_firstpage=819; citation_lastpage=828; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains; citation_author=B.M. Peter; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.03.13.990523; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Distinguishing between selective sweeps from standing variation and from a de novo mutation; citation_author=B.M. Peter; citation_author=E. Huerta-Sanchez; citation_author=R. Nielsen; citation_journal_title=PLoS Genet.; citation_volume=8; citation_firstpage=e1003011; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Inference of population splits and&nbsp;mixtures from genome-wide allele frequency data; citation_author=J.K. Pickrell; citation_author=J.K. Pritchard; citation_journal_title=PLoS Genet.; citation_volume=8; citation_firstpage=e1002967; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The extinction of the woolly mammoth and the archaeological record in Northeastern Asia; citation_author=V.V. Pitulko; citation_author=P.A. Nikolskiy; citation_journal_title=World Archaeol.; citation_volume=44; citation_firstpage=21; citation_lastpage=42; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=snpAD: an ancient DNA genotype caller; citation_author=K. Prüfer; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=4165; citation_lastpage=4171; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=The complete genome sequence of a Neanderthal from the Altai Mountains; citation_author=K. Prüfer; citation_author=F. Racimo; citation_author=N. Patterson; citation_author=F. Jay; citation_author=S. Sankararaman; citation_author=S. Sawyer; citation_author=A. Heinze; citation_author=G. Renaud; citation_author=P.H. Sudmant; citation_author=C. de Filippo; citation_author=et al.; citation_journal_title=Nature; citation_volume=505; citation_firstpage=43; citation_lastpage=49; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A high-coverage Neandertal genome from Vindija Cave in Croatia; citation_author=K. Prüfer; citation_author=C. de Filippo; citation_author=S. Grote; citation_author=F. Mafessoni; citation_author=P. Korlević; citation_author=M. Hajdinjak; citation_author=B. Vernot; citation_author=L. Skov; citation_author=P. Hsieh; citation_author=S. Peyrégne; citation_author=et al.; citation_journal_title=Science; citation_volume=358; citation_firstpage=655; citation_lastpage=658; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S. Purcell; citation_author=B. Neale; citation_author=K. Todd-Brown; citation_author=L. Thomas; citation_author=M.A.R. Ferreira; citation_author=D. Bender; citation_author=J. Maller; citation_author=P. Sklar; citation_author=P.I.W. de Bakker; citation_author=M.J. Daly; citation_author=P.C. Sham; citation_journal_title=Am. J. Hum. Genet.; citation_volume=81; citation_firstpage=559; citation_lastpage=575; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Denisovan Ancestry in East Eurasian and Native American Populations; citation_author=P. Qin; citation_author=M. Stoneking; citation_journal_title=Mol. Biol. Evol.; citation_volume=32; citation_firstpage=2665; citation_lastpage=2674; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans; citation_author=M. Raghavan; citation_author=P. Skoglund; citation_author=K.E. Graf; citation_author=M. Metspalu; citation_author=A. Albrechtsen; citation_author=I. Moltke; citation_author=S. Rasmussen; citation_author=T.W. Stafford; citation_author=L. Orlando; citation_author=E. Metspalu; citation_author=et al.; citation_journal_title=Nature; citation_volume=505; citation_firstpage=87; citation_lastpage=91; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Recent and planned developments of the program OxCal; citation_author=C.B. Ramsey; citation_author=S. Lee; citation_journal_title=Radiocarbon; citation_volume=55; citation_firstpage=720; citation_lastpage=730; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Denisova admixture and the first modern human dispersals into Southeast Asia and Oceania; citation_author=D. Reich; citation_author=N. Patterson; citation_author=M. Kircher; citation_author=F. Delfin; citation_author=M.R. Nandineni; citation_author=I. Pugach; citation_author=A.M. Ko; citation_author=Y.C. Ko; citation_author=T.A. Jinam; citation_author=M.E. Phipps; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=89; citation_firstpage=516; citation_lastpage=528; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP); citation_author=P.J. Reimer; citation_author=W.E.N. Austin; citation_author=E. Bard; citation_author=A. Bayliss; citation_author=P.G. Blackwell; citation_author=C.B. Ramsey; citation_author=M. Butzin; citation_author=H. Cheng; citation_author=R.L. Edwards; citation_author=M. Friedrich; citation_author=et al.; citation_journal_title=Radiocarbon; citation_volume=62; citation_firstpage=725; citation_lastpage=757; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=leeHom: adaptor trimming and merging for Illumina sequencing reads; citation_author=G. Renaud; citation_author=U. Stenzel; citation_author=J. Kelso; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=e141; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Human Parental Relatedness through Time - Detecting Runs of Homozygosity in Ancient DNA; citation_author=H. Ringbauer; citation_author=J. Novembre; citation_author=M. Steinrücken; citation_journal_title=bioRxiv; citation_doi=10.1101/2020.05.31.126912; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Partial uracil–DNA–glycosylase treatment for screening of ancient DNA; citation_author=N. Rohland; citation_author=E. Harney; citation_author=S. Mallick; citation_author=S. Nordenfelt; citation_author=D. Reich; citation_journal_title=Philos. Trans. R Soc. Lond. B Biol. Sci.; citation_volume=370; citation_firstpage=20130624; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The impact of the LGM on the development of the Upper Paleolithic in Mongolia; citation_author=E.P. Rybin; citation_author=A.M. Khatsenovich; citation_author=B. Gunchinsuren; citation_author=J.W. Olsen; citation_author=N. Zwyns; citation_journal_title=Quat. Int.; citation_volume=425; citation_firstpage=69; citation_lastpage=87; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Human behavioral responses to environmental condition and the emergence of the world’s oldest pottery in East and Northeast Asia: An overview; citation_author=H. Sato; citation_author=D. Natsuki; citation_journal_title=Quat. Int.; citation_volume=441; citation_firstpage=12; citation_lastpage=28; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Paleogenomics. Genomic structure in Europeans dating back at least 36,200 years; citation_author=A. Seguin-Orlando; citation_author=T.S. Korneliussen; citation_author=M. Sikora; citation_author=A.-S. Malaspinas; citation_author=A. Manica; citation_author=I. Moltke; citation_author=A. Albrechtsen; citation_author=A. Ko; citation_author=A. Margaryan; citation_author=V. Moiseyev; citation_author=et al.; citation_journal_title=Science; citation_volume=346; citation_firstpage=1113; citation_lastpage=1118; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=An early modern human from Tianyuan Cave, Zhoukoudian, China; citation_author=H. Shang; citation_author=H. Tong; citation_author=S. Zhang; citation_author=F. Chen; citation_author=E. Trinkaus; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=104; citation_firstpage=6573; citation_lastpage=6578; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Ancient genomes show social and reproductive behavior of early Upper Paleolithic foragers; citation_author=M. Sikora; citation_author=A. Seguin-Orlando; citation_author=V.C. Sousa; citation_author=A. Albrechtsen; citation_author=T. Korneliussen; citation_author=A. Ko; citation_author=S. Rasmussen; citation_author=I. Dupanloup; citation_author=P.R. Nigst; citation_author=M.D. Bosch; citation_author=et al.; citation_journal_title=Science; citation_volume=358; citation_firstpage=659; citation_lastpage=662; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=The population history of northeastern Siberia since the Pleistocene; citation_author=M. Sikora; citation_author=V.V. Pitulko; citation_author=V.C. Sousa; citation_author=M.E. Allentoft; citation_author=L. Vinner; citation_author=S. Rasmussen; citation_author=A. Margaryan; citation_author=P. de Barros Damgaard; citation_author=C. de la Fuente; citation_author=G. Renaud; citation_author=et al.; citation_journal_title=Nature; citation_volume=570; citation_firstpage=182; citation_lastpage=188; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago; citation_author=V. Siska; citation_author=E.R. Jones; citation_author=S. Jeon; citation_author=Y. Bhak; citation_author=H.M. Kim; citation_author=Y.S. Cho; citation_author=H. Kim; citation_author=K. Lee; citation_author=E. Veselovskaya; citation_author=T. Balueva; citation_author=et al.; citation_journal_title=Sci. Adv.; citation_volume=3; citation_firstpage=e1601877; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genomic insights into the peopling of the Southwest Pacific; citation_author=P. Skoglund; citation_author=C. Posth; citation_author=K. Sirak; citation_author=M. Spriggs; citation_author=F. Valentin; citation_author=S. Bedford; citation_author=G.R. Clark; citation_author=C. Reepmeyer; citation_author=F. Petchey; citation_author=D. Fernandes; citation_author=et al.; citation_journal_title=Nature; citation_volume=538; citation_firstpage=510; citation_lastpage=513; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Late Pleistocene spread&nbsp;of (cool-) temperate forests in Northeast China and climate changes synchronous with the North Atlantic region; citation_author=M. Stebich; citation_author=J. Mingram; citation_author=J. Han; citation_author=J. Liu; citation_journal_title=Global Planet. Change; citation_volume=65; citation_firstpage=56; citation_lastpage=70; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Human population dynamics in Europe over the Last Glacial Maximum; citation_author=M. Tallavaara; citation_author=M. Luoto; citation_author=N. Korhonen; citation_author=H. Järvinen; citation_author=H. Seppä; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=8232; citation_lastpage=8237; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The adaptive variant EDARV370A is associated with straight hair in East Asians; citation_author=J. Tan; citation_author=Y. Yang; citation_author=K. Tang; citation_author=P.C. Sabeti; citation_author=L. Jin; citation_author=S. Wang; citation_journal_title=Hum. Genet.; citation_volume=132; citation_firstpage=1187; citation_lastpage=1191; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The emergence of early pottery in East Asia: New discoveries and perspectives; citation_author=L. Wang; citation_author=P. Sebillaud; citation_journal_title=J.&nbsp;World Prehist.; citation_volume=32; citation_firstpage=73; citation_lastpage=110; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GSA: Genome Sequence Archive<sup/>; citation_author=Y. Wang; citation_author=F. Song; citation_author=J. Zhu; citation_author=S. Zhang; citation_author=Y. Yang; citation_author=T. Chen; citation_author=B. Tang; citation_author=L. Dong; citation_author=N. Ding; citation_author=Q. Zhang; citation_author=et al.; citation_journal_title=Genomics Proteomics Bioinformatics; citation_volume=15; citation_firstpage=14; citation_lastpage=18; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Genomic insights into the formation of human populations in East Asia; citation_author=C.-C. Wang; citation_author=H.-Y. Yeh; citation_author=A.N. Popov; citation_author=H.-Q. Zhang; citation_author=H. Matsumura; citation_author=K. Sirak; citation_author=O. Cheronet; citation_author=A. Kovalev; citation_author=N. Rohland; citation_author=A.M. Kim; citation_author=et al.; citation_journal_title=Nature; citation_volume=591; citation_firstpage=413; citation_lastpage=419; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia; citation_author=M.A. Yang; citation_author=X. Gao; citation_author=C. Theunert; citation_author=H. Tong; citation_author=A. Aximu-Petri; citation_author=B. Nickel; citation_author=M. Slatkin; citation_author=M. Meyer; citation_author=S. Pääbo; citation_author=J. Kelso; citation_author=Q. Fu; citation_journal_title=Curr. Biol.; citation_volume=27; citation_firstpage=3202; citation_lastpage=3208.e9; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Ancient DNA indicates human population shifts and admixture in northern and southern China; citation_author=M.A. Yang; citation_author=X. Fan; citation_author=B. Sun; citation_author=C. Chen; citation_author=J. Lang; citation_author=Y.C. Ko; citation_author=C.H. Tsang; citation_author=H. Chiu; citation_author=T. Wang; citation_author=Q. Bao; citation_author=et al.; citation_journal_title=Science; citation_volume=369; citation_firstpage=282; citation_lastpage=288; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Dietary adaptation of FADS genes in Europe varied across time and geography; citation_author=K. Ye; citation_author=F. Gao; citation_author=D. Wang; citation_author=O. Bar-Yosef; citation_author=A. Keinan; citation_journal_title=Nat. Ecol. Evol.; citation_volume=1; citation_firstpage=167; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia; citation_author=H. Yu; citation_author=M.A. Spyrou; citation_author=M. Karapetian; citation_author=S. Shnaider; citation_author=R. Radzevičiūtė; citation_author=K. Nägele; citation_author=G.U. Neumann; citation_author=S. Penske; citation_author=J. Zech; citation_author=M. Lucas; citation_author=et al.; citation_journal_title=Cell; citation_volume=181; citation_firstpage=1232; citation_lastpage=1245.e20; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Neolithisation in the southern Lesser Khingan Mountains: lithic technologies and ecological adaptation; citation_author=J.-P. Yue; citation_author=Y.-Q. Li; citation_author=S. Yang; citation_journal_title=Antiquity; citation_volume=93; citation_firstpage=1144; citation_lastpage=1160; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=OSL and AMS 14C Age of the Most Complete Mammoth Fossil Skeleton from Northeastern China and its Paleoclimate Significance; citation_author=H. Zhang; citation_author=F. Chang; citation_author=H. Li; citation_author=G. Peng; citation_author=L. Duan; citation_author=H. Meng; citation_author=X. Yang; citation_author=Z. Wei; citation_journal_title=Radiocarbon; citation_volume=61; citation_firstpage=347; citation_lastpage=358; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The Age and Characteristics of the Jade Unearthed from Xiaonanshan Tomb in Raohe County; citation_author=B. Zhao; citation_author=M. Sun; citation_author=Z. Du; citation_journal_title=Bianjiang Kaogu Yanjiu; citation_volume=2013; citation_firstpage=69; citation_lastpage=78; citation_publication_date=2013; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Northern East Asia was inhabited by modern humans as early as 40 thousand years ago (ka), as demonstrated by the Tianyuan individual. Using genome-wide data obtained from 25 individuals dated to 33.6–3.4 ka from the Amur region, we show that Tianyuan-related ancestry was widespread in northern East Asia before the Last Glacial Maximum (LGM). At the close of the LGM stadial, the earliest northern East Asian appeared in the Amur region, and this population is basal to ancient northern East Asians. Human populations in the Amur region have maintained genetic continuity from 14 ka, and these early inhabitants represent the closest East Asian source known for Ancient Paleo-Siberians. We also observed that <i>EDAR V370A</i> was likely to have been elevated to high frequency after the LGM, suggesting the possible timing for its selection. This study provides a deep look into the population dynamics of northern East Asia.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=8394&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(21)00575-4&amp;be=5885&amp;fe=7957&amp;dc=7204&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749590411567,%22n%22:0,%22f%22:5333,%22dn%22:5333,%22dne%22:5333,%22c%22:5333,%22s%22:5333,%22ce%22:5333,%22rq%22:5334,%22rp%22:5857,%22rpe%22:6631,%22dl%22:5861,%22di%22:7184,%22ds%22:7205,%22de%22:7424,%22dc%22:7957,%22l%22:7957,%22le%22:8058%7D,%22navigation%22:%7B%7D%7D&amp;fp=6516&amp;fcp=6516&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-mtable {
  display: inline-block;
  text-align: center;
  vertical-align: .25em;
  position: relative;
  box-sizing: border-box;
  border-spacing: 0;
  border-collapse: collapse;
}

mjx-mstyle[size="s"] mjx-mtable {
  vertical-align: .354em;
}

mjx-labels {
  position: absolute;
  left: 0;
  top: 0;
}

mjx-table {
  display: inline-block;
  vertical-align: -.5ex;
  box-sizing: border-box;
}

mjx-table > mjx-itable {
  vertical-align: middle;
  text-align: left;
  box-sizing: border-box;
}

mjx-labels > mjx-itable {
  position: absolute;
  top: 0;
}

mjx-mtable[justify="left"] {
  text-align: left;
}

mjx-mtable[justify="right"] {
  text-align: right;
}

mjx-mtable[justify="left"][side="left"] {
  padding-right: 0 ! important;
}

mjx-mtable[justify="left"][side="right"] {
  padding-left: 0 ! important;
}

mjx-mtable[justify="right"][side="left"] {
  padding-right: 0 ! important;
}

mjx-mtable[justify="right"][side="right"] {
  padding-left: 0 ! important;
}

mjx-mtable[align] {
  vertical-align: baseline;
}

mjx-mtable[align="top"] > mjx-table {
  vertical-align: top;
}

mjx-mtable[align="bottom"] > mjx-table {
  vertical-align: bottom;
}

mjx-mtable[side="right"] mjx-labels {
  min-width: 100%;
}

mjx-mtr {
  display: table-row;
  text-align: left;
}

mjx-mtr[rowalign="top"] > mjx-mtd {
  vertical-align: top;
}

mjx-mtr[rowalign="center"] > mjx-mtd {
  vertical-align: middle;
}

mjx-mtr[rowalign="bottom"] > mjx-mtd {
  vertical-align: bottom;
}

mjx-mtr[rowalign="baseline"] > mjx-mtd {
  vertical-align: baseline;
}

mjx-mtr[rowalign="axis"] > mjx-mtd {
  vertical-align: .25em;
}

mjx-mtd {
  display: table-cell;
  text-align: center;
  padding: .215em .4em;
}

mjx-mtd:first-child {
  padding-left: 0;
}

mjx-mtd:last-child {
  padding-right: 0;
}

mjx-mtable > * > mjx-itable > *:first-child > mjx-mtd {
  padding-top: 0;
}

mjx-mtable > * > mjx-itable > *:last-child > mjx-mtd {
  padding-bottom: 0;
}

mjx-tstrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-labels[align="left"] > mjx-mtr > mjx-mtd {
  text-align: left;
}

mjx-labels[align="right"] > mjx-mtr > mjx-mtd {
  text-align: right;
}

mjx-mtd[extra] {
  padding: 0;
}

mjx-mtd[rowalign="top"] {
  vertical-align: top;
}

mjx-mtd[rowalign="center"] {
  vertical-align: middle;
}

mjx-mtd[rowalign="bottom"] {
  vertical-align: bottom;
}

mjx-mtd[rowalign="baseline"] {
  vertical-align: baseline;
}

mjx-mtd[rowalign="axis"] {
  vertical-align: .25em;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-c.mjx-c61 {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c64 {
  padding: 0.694em 0.556em 0.011em 0;
}

mjx-c.mjx-c6D {
  padding: 0.442em 0.833em 0 0;
}

mjx-c.mjx-c69 {
  padding: 0.657em 0.278em 0 0;
}

mjx-c.mjx-c78 {
  padding: 0.431em 0.528em 0 0;
}

mjx-c.mjx-c66 {
  padding: 0.705em 0.357em 0 0;
}

mjx-c.mjx-c72 {
  padding: 0.442em 0.392em 0 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c67 {
  padding: 0.453em 0.5em 0.206em 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c62 {
  padding: 0.694em 0.556em 0.011em 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c65 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-cA0 {
  padding: 0 0.332em 0 0;
}

mjx-c.mjx-c7B {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c76 {
  padding: 0.431em 0.528em 0.011em 0;
}

mjx-c.mjx-c75 {
  padding: 0.442em 0.556em 0.011em 0;
}

mjx-c.mjx-c7D {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c74 {
  padding: 0.615em 0.389em 0.011em 0;
}

mjx-c.mjx-c68 {
  padding: 0.694em 0.556em 0 0;
}

mjx-c.mjx-c73 {
  padding: 0.448em 0.394em 0.011em 0;
}

mjx-c.mjx-c7A {
  padding: 0.431em 0.444em 0 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c35 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c70 {
  padding: 0.442em 0.556em 0.194em 0;
}

mjx-c.mjx-c71 {
  padding: 0.442em 0.528em 0.194em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c63 {
  padding: 0.448em 0.444em 0.011em 0;
}

mjx-c.mjx-c5F {
  padding: 0 0.333em 0.14em 0;
}

mjx-c.mjx-c41 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c46 {
  padding: 0.68em 0.653em 0 0;
}

mjx-c.mjx-c52 {
  padding: 0.683em 0.736em 0.022em 0;
}

mjx-c.mjx-c4E {
  padding: 0.683em 0.75em 0 0;
}

mjx-c.mjx-c45 {
  padding: 0.68em 0.681em 0 0;
}

mjx-c.mjx-c44 {
  padding: 0.683em 0.764em 0 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c50 {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c79 {
  padding: 0.431em 0.528em 0.205em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(21)00575-4"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(21)00575-4"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-995806" aria-expanded="false" aria-controls="login-links-995806" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-995806" id="login-links--995806" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-995806" data-opens="#links-dialog-995806" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-995806" aria-modal="true"><button data-closes="#links-dialog-995806" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2821%2900575-4" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2021.04.040">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(21)00575-4" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(20)X0013-4"><span>Volume 184</span>, <span>Issue 12</span></a></span><span class="meta-panel__pages"><span>p3256-3266.e13</span></span><span class="meta-panel__onlineDate">June 10, 2021</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=12&amp;pii=S0092-8674%2820%29X0013-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">The deep population history of northern East Asia from the Late Pleistocene to the Holocene</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Xiaowei</span> <span property="familyName">Mao</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xiaowei</span> <span property="familyName">Mao</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Mao/Xiaowei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Hucai</span> <span property="familyName">Zhang</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Hucai</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:zhanghc@ynu.edu.cn" property="email" title="Send email to Hucai Zhang">zhanghc@ynu.edu.cn</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China</span></div><div property="affiliation" typeof="Organization"><span property="name">CAS Center for Excellence in Tibetan Plateau Earth Sciences, Chinese Academy of Sciences (CAS), Beijing 100101, China</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Hucai" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup> <a id="eadd0010" href="mailto:zhanghc@ynu.edu.cn" property="email" title="Send email to Hucai Zhang">zhanghc@ynu.edu.cn</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Shiyu</span> <span property="familyName">Qiao</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Shiyu</span> <span property="familyName">Qiao</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Shanghai Qi Zhi Institute, Shanghai 200232, China</span></div><div property="affiliation" typeof="Organization"><span property="name">University of the Chinese Academy of Sciences, Beijing 100049, China</span></div></div><div class="core-author-link"><a href="/authored-by/Qiao/Shiyu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Yichen</span> <span property="familyName">Liu</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yichen</span> <span property="familyName">Liu</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7187-6232" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7187-6232</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Shanghai Qi Zhi Institute, Shanghai 200232, China</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Yichen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Fengqin</span> <span property="familyName">Chang</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fengqin</span> <span property="familyName">Chang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China</span></div></div><div class="core-author-link"><a href="/authored-by/Chang/Fengqin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Ping</span> <span property="familyName">Xie</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ping</span> <span property="familyName">Xie</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China</span></div></div><div class="core-author-link"><a href="/authored-by/Xie/Ping" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Ming</span> <span property="familyName">Zhang</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ming</span> <span property="familyName">Zhang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Ming" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Tianyi</span> <span property="familyName">Wang</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Tianyi</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Northwest University, Xi’an 710069, China</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Tianyi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Mian</span> <span property="familyName">Li</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mian</span> <span property="familyName">Li</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Shandong First Medical University &amp; Shandong Academy of Medical Sciences, Jinan 250000, China</span></div></div><div class="core-author-link"><a href="/authored-by/Li/Mian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Peng</span> <span property="familyName">Cao</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Peng</span> <span property="familyName">Cao</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Cao/Peng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Ruowei</span> <span property="familyName">Yang</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ruowei</span> <span property="familyName">Yang</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Yang/Ruowei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Feng</span> <span property="familyName">Liu</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Feng</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Feng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Qingyan</span> <span property="familyName">Dai</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Qingyan</span> <span property="familyName">Dai</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Dai/Qingyan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Xiaotian</span> <span property="familyName">Feng</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xiaotian</span> <span property="familyName">Feng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Feng/Xiaotian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Wanjing</span> <span property="familyName">Ping</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Wanjing</span> <span property="familyName">Ping</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4318-0673" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4318-0673</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Ping/Wanjing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Chuzhao</span> <span property="familyName">Lei</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Chuzhao</span> <span property="familyName">Lei</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Animal Genetics, Breeding and Reproduction of Shaanxi Province, College of Animal Science and Technology, Northwest A&amp;F University, Yangling 712100, China</span></div></div><div class="core-author-link"><a href="/authored-by/Lei/Chuzhao" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">John W.</span> <span property="familyName">Olsen</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">John W.</span> <span property="familyName">Olsen</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5295-7451" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5295-7451</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span property="name">School of Anthropology, University of Arizona, Tucson, AZ 85721-0030, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Olsen/John+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">E. Andrew</span> <span property="familyName">Bennett</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">E. Andrew</span> <span property="familyName">Bennett</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3827-0711" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3827-0711</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div></div><div class="core-author-link"><a href="/authored-by/Bennett/E+Andrew" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Qiaomei</span> <span property="familyName">Fu</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Qiaomei</span> <span property="familyName">Fu</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:fuqiaomei@ivpp.ac.cn" property="email" title="Send email to Qiaomei Fu">fuqiaomei@ivpp.ac.cn</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span property="name">Shanghai Qi Zhi Institute, Shanghai 200232, China</span></div><div property="affiliation" typeof="Organization"><span property="name">University of the Chinese Academy of Sciences, Beijing 100049, China</span></div></div><div class="core-author-link"><a href="/authored-by/Fu/Qiaomei" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">10</a></sup> <a id="eadd0015" href="mailto:fuqiaomei@ivpp.ac.cn" property="email" title="Send email to Qiaomei Fu">fuqiaomei@ivpp.ac.cn</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Key Laboratory of Vertebrate Evolution and Human Origins, Institute of Vertebrate Paleontology and Paleoanthropology, Center for Excellence in Life and Paleoenvironment, Chinese Academy of Sciences, Beijing 100044, China</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Institute for Ecological Research and Pollution Control of Plateau Lakes, School of Ecology and Environmental Science, Yunnan University, Kunming 650500, China</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">CAS Center for Excellence in Tibetan Plateau Earth Sciences, Chinese Academy of Sciences (CAS), Beijing 100101, China</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Shanghai Qi Zhi Institute, Shanghai 200232, China</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">University of the Chinese Academy of Sciences, Beijing 100049, China</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Northwest University, Xi’an 710069, China</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Shandong First Medical University &amp; Shandong Academy of Medical Sciences, Jinan 250000, China</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Key Laboratory of Animal Genetics, Breeding and Reproduction of Shaanxi Province, College of Animal Science and Technology, Northwest A&amp;F University, Yangling 712100, China</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">School of Anthropology, University of Arizona, Tucson, AZ 85721-0030, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">10</div><div id="fn1"><div id="ntpara0010" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received October 16, 2020</span>; <span>Revised January 20, 2021</span>; <span>Accepted April 23, 2021</span>; <span>Published online May 27, 2021</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2021.04.040" property="sameAs" target="_blank">10.1016/j.cell.2021.04.040 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867421005754" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2021 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2021.04.040" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(20)X0013-4"><img src="/cms/asset/1e6ae0ff-df88-4cfb-bcfd-261361a4a1de/cov200h.gif" alt="Cover Image - Cell, Volume 184, Issue 12"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2821%2900575-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results and discussion</a></li><li><a href="#sec-3">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (4)</a></li><li><a href="#app-2">Supporting citations</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900575-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.04.040">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2821%2900575-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2021.04.040&amp;pii=S0092-8674%2821%2900575-4" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2821%2900575-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2021.04.040">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2021.04.040" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867421005754&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867421005754&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=184&amp;issue=12&amp;pii=S0092-8674%2820%29X0013-4" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(21)00496-7" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(21)00538-9" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results and discussion</a></li><li><a href="#sec-3" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental information (4)</a></li><li><a href="#app-2" aria-current="false">Supporting citations</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Tianyuan/AR33K ancestry was widespread in northern East Asia before the LGM</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">The earliest northern East Asian appeared in the Amur region at the end of the LGM</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">The AR14K-related population is the closest East Asian source for Ancient Paleo-Siberians</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph"><i>EDAR V370A</i> likely increased to high frequency after the LGM</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Northern East Asia was inhabited by modern humans as early as 40 thousand years ago (ka), as demonstrated by the Tianyuan individual. Using genome-wide data obtained from 25 individuals dated to 33.6–3.4 ka from the Amur region, we show that Tianyuan-related ancestry was widespread in northern East Asia before the Last Glacial Maximum (LGM). At the close of the LGM stadial, the earliest northern East Asian appeared in the Amur region, and this population is basal to ancient northern East Asians. Human populations in the Amur region have maintained genetic continuity from 14 ka, and these early inhabitants represent the closest East Asian source known for Ancient Paleo-Siberians. We also observed that <i>EDAR V370A</i> was likely to have been elevated to high frequency after the LGM, suggesting the possible timing for its selection. This study provides a deep look into the population dynamics of northern East Asia.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/f0768426-e602-42bf-8d1a-397c322c6a9b/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2021.04.040/asset/88ef2c85-7eac-41f4-a47e-727c3741da7f/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/f0768426-e602-42bf-8d1a-397c322c6a9b/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;ancient+DNA&quot;" alt="Search on this keyword" rel="nofollow">ancient DNA</a></li><li><a href="/action/doSearch?AllField=&quot;northern+East+Asia&quot;" alt="Search on this keyword" rel="nofollow">northern East Asia</a></li><li><a href="/action/doSearch?AllField=&quot;Last+Glacial+Maximum&quot;" alt="Search on this keyword" rel="nofollow">Last Glacial Maximum</a></li><li><a href="/action/doSearch?AllField=&quot;population+dynamics&quot;" alt="Search on this keyword" rel="nofollow">population dynamics</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Anatomically and behaviorally modern humans were present in northern East Asia as early as 40 thousand years ago (ka) (<span class="dropBlock reference-citations"><a id="body-ref-sref81" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81" aria-controls="sref81">Shang et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref81"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Shang, H. ∙ Tong, H. ∙ Zhang, S. ...</div><div><strong>An early modern human from Tianyuan Cave, Zhoukoudian, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:6573-6578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0702169104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34249776217" target="_blank" aria-label="Open Scopus in new tab">Scopus (150)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17416672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0702169104&amp;pmid=17416672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The non-static definition of northern East Asia (<span class="dropBlock reference-citations"><a id="body-ref-sref37" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37" aria-controls="sref37">Li and Cribb, 2014</a><div class="dropBlock__holder" data-db-target-of="sref37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Li, N. ∙ Cribb, R.</div><div><strong>Historical Atlas of Northeast Asia, 1590-2010: Korea, Manchuria, Mongolia, Eastern Siberia</strong></div><div>Columbia University Press, 2014</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.LiR.CribbHistorical+Atlas+of+Northeast+Asia%2C+1590-2010%3A+Korea%2C+Manchuria%2C+Mongolia%2C+Eastern+Siberia2014Columbia+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) includes the Mongolian plateau, northern China (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Li et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Li, S. ∙ Yang, S. ∙ Liu, X.</div><div><strong>Spatiotemporal variability of extreme precipitation in north and south of the Qinling-Huaihe region and influencing factors during 1960–2013</strong></div><div><em>Prog. Geogr.</em> 2015; <strong>34</strong>:354-363</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.LiS.YangX.LiuSpatiotemporal+variability+of+extreme+precipitation+in+north+and+south+of+the+Qinling-Huaihe+region+and+influencing+factors+during+1960%E2%80%932013Prog.+Geogr.342015354363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the Japanese archipelago, the Korean peninsula, and the mountainous regions of the Russian Far East, all of which fall within a similar latitude range as that of central and southern Europe. In Europe, human population movements and size were influenced by Ice Age climatic fluctuations (<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Fu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (596)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Tallavaara et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Tallavaara, M. ∙ Luoto, M. ∙ Korhonen, N. ...</div><div><strong>Human population dynamics in Europe over the Last Glacial Maximum</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:8232-8237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1503784112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84936803076" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1503784112&amp;pmid=26100880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); e.g., the population size varied (ranging from 130,000–410,000) between 30 and 13 ka (<span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Tallavaara et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Tallavaara, M. ∙ Luoto, M. ∙ Korhonen, N. ...</div><div><strong>Human population dynamics in Europe over the Last Glacial Maximum</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:8232-8237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1503784112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84936803076" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1503784112&amp;pmid=26100880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These climatic oscillations may have had a similar effect on the population history of high-latitude and high-elevation regions in Asia, as evidenced by archaeological finds that suggest gaps in human occupation in Mongolia during marine isotope stages 3 (beginning ∼57 ka) and 2 (beginning ∼29 ka) (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Rybin et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Rybin, E.P. ∙ Khatsenovich, A.M. ∙ Gunchinsuren, B. ...</div><div><strong>The impact of the LGM on the development of the Upper Paleolithic in Mongolia</strong></div><div><em>Quat. Int.</em> 2016; <strong>425</strong>:69-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85006455434" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2016.05.001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although several studies have advanced our understanding of ancient East Asians from 9 ka to historical times (<span class="dropBlock reference-citations"><a id="body-ref-sref57-1" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-1" aria-controls="sref57-1">Ning et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref57-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Ning, C. ∙ Li, T. ∙ Wang, K. ...</div><div><strong>Ancient genomes from northern China suggest links between subsistence changes and human migration</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2700</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16557-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085859685" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16557-2&amp;pmid=32483115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Wang et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Wang, C.-C. ∙ Yeh, H.-Y. ∙ Popov, A.N. ...</div><div><strong>Genomic insights into the formation of human populations in East Asia</strong></div><div><em>Nature.</em> 2021; <strong>591</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03336-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85101273487" target="_blank" aria-label="Open Scopus in new tab">Scopus (185)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33618348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03336-2&amp;pmid=33618348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), only two human genomes earlier than the Last Glacial Maximum (LGM; approximately 26.5–19 ka) (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Clark et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Clark, P.U. ∙ Dyke, A.S. ∙ Shakun, J.D. ...</div><div><strong>The last glacial maximum</strong></div><div><em>Science.</em> 2009; <strong>325</strong>:710-714</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1172873" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-68449088262" target="_blank" aria-label="Open Scopus in new tab">Scopus (2613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19661421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1172873&amp;pmid=19661421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) have been reported (∼40 ka from Tianyuan Cave in northern China; ∼34 ka from Salkhit in northeastern Mongolia) (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Devièse et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Devièse, T. ∙ Massilani, D. ∙ Yi, S. ...</div><div><strong>Compound-specific radiocarbon dating and mitochondrial DNA analysis of the Pleistocene hominin from Salkhit Mongolia</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-08018-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060861342" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30700710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-08018-8&amp;pmid=30700710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Massilani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Massilani, D. ∙ Skov, L. ∙ Hajdinjak, M. ...</div><div><strong>Denisovan ancestry and population history of early East Asians</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:579-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc1166" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33122380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc1166&amp;pmid=33122380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">Yang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, the population structure of northern East Asia before and after the LGM stadial is largely unknown, and the lack of human fossils dating to between 40–9 ka from this area hinders our understanding of this crucial period of human prehistory.</div></section><section id="sec-2"><h2>Results and discussion</h2><section id="sec-2-1"><h3>Samples and ancient DNA generation</h3><div id="p0035" role="paragraph">To examine deep population histories in northern East Asia, genome-wide genotype data were generated from 25 ancient humans from the Songnen Plain (Heilongjiang province, northeastern China) in the Amur region, associated with direct radiocarbon dates ranging from 33,590–3,420 calibrated years before&nbsp;present (cal BP). These individuals were recovered at construction sites without archaeological context or from eroded gullies and small valleys formed in alluvial deposits of the Songhua River and Nen River during summer rainy seasons. DNA was extracted from less than 100&nbsp;mg of bone powder and used to construct single-stranded libraries that were enriched for human DNA using a panel of 1.2 million single-nucleotide polymorphisms (SNPs) (<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Fu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Enriched libraries were sequenced, resulting in SNP panel coverage from 0.002–14.45× (<a id="crosref0240" href="#tbl1">Table 1</a>). All samples showed low levels of modern human contamination (<a id="crosref0245" href="#tbl1">Table 1</a>). Removing close relatives (defined as first- or second-degree relationships) (<a id="crosref0250" href="#mmc2">Table S1</a>) and individuals with less than 25,000 SNPs resulted in a final dataset of 20 unrelated individuals (two individuals with SNP coverage lower than 50,000 are denoted with the suffix “_LowCov”) (<a id="crosref0255" href="#tbl1">Table&nbsp;1</a>). Our final samples were combined into five main chronological groups, with similar genomic profiles within each group treated as populations: (1) one upper Pleistocene individual from before the LGM (denoted AR33K; 34,324–32,360 cal BP; AR represents Amur region, followed by the approximate age in thousands of years [K]), (2) one upper Pleistocene individual dating to the end of the LGM (denoted AR19K; 19,587–19,175 cal BP), (3) two individuals dating to the period between 14,932 and 14,017 cal BP (denoted AR14K) at the end of the last glacial period, (4) four individuals from 12,735–10,302 cal BP (denoted AR13–10K), and (5) 12 individuals from 9,425–3,360 cal BP (denoted AR9.2K_o, ARpost9K (n&nbsp;=&nbsp;9), AR7.3K_LowCov, and AR3.4K_LowCov) (<a id="crosref0260" href="#fig1">Figure&nbsp;1</a>A; <a id="optZxgcB4heft" href="#figs1">Figure S1</a>; <a id="crosref0265" href="#tbl1">Table 1</a>; <a id="crosref0270" href="#mmc2">Table S1</a>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/3b9b8c1d-babf-4fed-9b30-8be284eafb65/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2021.04.040/asset/a796c76f-a63a-4cb5-91d6-f25bf090a902/main.assets/gr1.jpg" height="622" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/3b9b8c1d-babf-4fed-9b30-8be284eafb65/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Geographic and temporal distribution and population structure of newly sampled and published populations in northern East Asia</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Geographic and temporal distribution of newly sampled and published populations. Symbols as in (B).</div><div id="fspara0020" role="paragraph">(B) PCA of sampled and published East Asian and Siberian populations. The ancient populations (above) are projected on the principal components of modern populations (below). New samples are highlighted by red text and larger symbols.</div><div id="fspara0025" role="paragraph">(C) The pairwise genetic affinity among ancient East Asians using outgroup-<i>f</i><sub><i>3</i></sub> analysis. New samples are highlighted in red, and dashed rectangles show high genetic affinities.</div><div id="fspara0030" role="paragraph">See also <a id="crosref1380" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref1385" href="#tbl1">Tables 1</a> and <a id="crosref1390" href="#mmc2">S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="tbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>Individual ID</th><th>Sex</th><th>Sample ID</th><th>Population ID</th><th>Date (cal BP)<a id="crosref1480" href="#tblfn1"><sup>a</sup></a></th><th>Latitude</th><th>Longitude</th><th>Library style<a id="crosref1485" href="#tblfn2"><sup>b</sup></a></th><th>mtDNA 5′C→T (%)</th><th>1240K 5′C→T (%)</th><th>mtDNA contamination<a id="crosref1490" href="#tblfn3"><sup>c</sup></a></th><th>MT_hg<a id="crosref1495" href="#tblfn4"><sup>d</sup></a></th><th>Y_hg<a id="crosref1500" href="#tblfn5"><sup>e</sup></a></th><th>mtDNA Coverage</th><th>1240K Coverage</th><th>SNP</th></tr></thead><tbody><tr><td>NE20</td><td>F</td><td>AR33K</td><td>AR33K</td><td>34,324–32,360</td><td>45.4</td><td>127.03</td><td>SS</td><td>27</td><td>22</td><td>1.6</td><td>B</td><td>–</td><td>574</td><td>14.45</td><td>1,010,948</td></tr><tr><td>NE56</td><td>M</td><td>AR19K</td><td>AR19K</td><td>19,587–19,175</td><td>46</td><td>126.15</td><td>SS</td><td>34</td><td>19</td><td>0.5</td><td>G2</td><td>C2</td><td>628</td><td>1.583</td><td>768,551</td></tr><tr><td>NE34</td><td>M</td><td>AR14.5K</td><td>AR14K</td><td>14,932–14,176</td><td>45.98</td><td>125.58</td><td>SS</td><td>27</td><td>22</td><td>0.9</td><td>D4h3</td><td>C</td><td>488</td><td>0.716</td><td>332,628</td></tr><tr><td>NE-5</td><td>M</td><td>AR14.1K</td><td>AR14K</td><td>14,814–14,017</td><td>45.95</td><td>125.83</td><td>SS</td><td>25</td><td>19</td><td>1.1</td><td>D4h3a+@152</td><td>C</td><td>503</td><td>2.212</td><td>520,883</td></tr><tr><td>NE-1</td><td>F</td><td>AR12K</td><td>AR13-10K</td><td>12,735–12,486</td><td>46</td><td>125.73</td><td>SS</td><td>35</td><td>8</td><td>0.3</td><td>D4g</td><td>–</td><td>554</td><td>0.083</td><td>91,615</td></tr><tr><td>NE-8</td><td>M</td><td>AR11K</td><td>AR13-10K</td><td>11,601–11,176</td><td>46.01</td><td>125.82</td><td>SS</td><td>34</td><td>27</td><td>1.5</td><td>D4o</td><td>DE</td><td>274</td><td>0.421</td><td>225,066</td></tr><tr><td>NE57<a id="crosref1505" href="#tblfn6"><sup>f</sup></a></td><td>M</td><td>AR11K_deleted</td><td>–</td><td>11,206–10,765</td><td>46</td><td>126.12</td><td>DS-half</td><td>6</td><td>1</td><td>1.9</td><td>–</td><td>–</td><td>3</td><td>0.01</td><td>11,614</td></tr><tr><td>NE36<a id="crosref1510" href="#tblfn6"><sup>f</sup></a></td><td>U</td><td>AR10.6K_deleted</td><td>–</td><td>11,065–10,513</td><td>46.06</td><td>125.72</td><td>SS</td><td>16</td><td>4</td><td>0.1</td><td>D4h4</td><td>–</td><td>47</td><td>0.007</td><td>8,926</td></tr><tr><td>NE-3</td><td>M</td><td>AR10.6K</td><td>AR13-10K</td><td>10,996–10,429</td><td>45.85</td><td>126</td><td>SS</td><td>33</td><td>27</td><td>1.4</td><td>M8</td><td>C</td><td>419</td><td>0.692</td><td>348,382</td></tr><tr><td>NE-4</td><td>M</td><td>AR10.5K</td><td>AR13-10K</td><td>10,740–10,302</td><td>45.93</td><td>125.85</td><td>SS</td><td>32</td><td>26</td><td>1.8</td><td>M8</td><td>C</td><td>396</td><td>0.671</td><td>332,715</td></tr><tr><td>NE30<a id="crosref1515" href="#tblfn7"><sup>g</sup></a></td><td>M</td><td>AR9.9K_2d.rel.AR10.5K_deleted</td><td>–</td><td>10,167–9,676</td><td>45.96</td><td>126.23</td><td>SS</td><td>20</td><td>12</td><td>0.8</td><td>D4m</td><td>C</td><td>99</td><td>0.225</td><td>218,945</td></tr><tr><td>NE45</td><td>M</td><td>AR9.2K_o</td><td>AR9.2K_o</td><td>9,425–9,029</td><td>45.71</td><td>126.88</td><td>SS</td><td>17</td><td>7</td><td>1.3</td><td>G1a1a</td><td>P</td><td>163</td><td>0.228</td><td>134,344</td></tr><tr><td>NE44<a id="crosref1520" href="#tblfn6"><sup>f</sup></a></td><td>M</td><td>AR9.2K_deleted</td><td>–</td><td>9,425–9,027</td><td>45.71</td><td>126.88</td><td>SS</td><td>22</td><td>3</td><td>2.4</td><td>D4c1</td><td>–</td><td>25</td><td>0.004</td><td>5,129</td></tr><tr><td>NE35</td><td>F</td><td>AR8.9K</td><td>ARpost9K</td><td>9,131–8,770</td><td>45.91</td><td>125.95</td><td>SS</td><td>24</td><td>22</td><td>1.0</td><td>R11</td><td>–</td><td>602</td><td>4.664</td><td>739,026</td></tr><tr><td>NE-16</td><td>F</td><td>AR8.5K</td><td>ARpost9K</td><td>8,723–8,421</td><td>45.96</td><td>125.78</td><td>SS</td><td>23</td><td>14</td><td>1.2</td><td>D4o</td><td>–</td><td>271</td><td>0.534</td><td>216,989</td></tr><tr><td>NE49</td><td>F</td><td>AR8.3K</td><td>ARpost9K</td><td>8,425–8,204</td><td>45.65</td><td>126.92</td><td>SS</td><td>20</td><td>10</td><td>1.5</td><td>D4o</td><td>–</td><td>59</td><td>0.21</td><td>197,627</td></tr><tr><td>NE39</td><td>M</td><td>AR8.1K</td><td>ARpost9K</td><td>8,340–8,029</td><td>45.91</td><td>125.9</td><td>SS</td><td>15</td><td>12</td><td>1.5</td><td>D4m2</td><td>C</td><td>526</td><td>0.93</td><td>345,404</td></tr><tr><td>NE58<a id="crosref1525" href="#tblfn8"><sup>h</sup></a></td><td>M</td><td>AR7.3K_LowCov</td><td>–</td><td>7,425–7,168</td><td>45.85</td><td>126</td><td>DS-half</td><td>9</td><td>1</td><td>0.6</td><td>D4e5</td><td>–</td><td>163</td><td>0.026</td><td>30,052</td></tr><tr><td>NE-18<a id="crosref1530" href="#tblfn6"><sup>f</sup></a></td><td>M</td><td>AR7K_deleted</td><td>–</td><td>7,245–6,894</td><td>45.96</td><td>125.77</td><td>SS</td><td>22</td><td>2</td><td>0.2</td><td>D4h3</td><td>–</td><td>158</td><td>0.002</td><td>2,511</td></tr><tr><td>NE19</td><td>M</td><td>AR7K</td><td>ARpost9K</td><td>7,167–6,854</td><td>45.96</td><td>125.75</td><td>SS</td><td>28</td><td>23</td><td>2.4</td><td>F1b1+@152</td><td>C</td><td>489</td><td>0.698</td><td>349,755</td></tr><tr><td>NE29</td><td>F</td><td>AR6.87K</td><td>ARpost9K</td><td>6,993–6,747</td><td>45.93</td><td>126.37</td><td>SS</td><td>16</td><td>13</td><td>0.8</td><td>D4c1</td><td>–</td><td>528</td><td>1.074</td><td>393,142</td></tr><tr><td>NE-22</td><td>M</td><td>AR6.84K</td><td>ARpost9K</td><td>6,986–6,676</td><td>46.15</td><td>126.23</td><td>SS</td><td>20</td><td>11</td><td>1.2</td><td>D4m</td><td>C</td><td>198</td><td>0.374</td><td>183,780</td></tr><tr><td>NE-9</td><td>F</td><td>AR6.33K</td><td>ARpost9K</td><td>6,440–6,205</td><td>46.01</td><td>125.8</td><td>SS</td><td>26</td><td>23</td><td>2.7</td><td>C4a1a+195</td><td>–</td><td>569</td><td>1.097</td><td>410,575</td></tr><tr><td>NE-2</td><td>M</td><td>AR6.32K</td><td>ARpost9K</td><td>6,437–6,201</td><td>45.85</td><td>125.82</td><td>SS</td><td>19</td><td>12</td><td>1.2</td><td>D4m</td><td>C</td><td>266</td><td>0.213</td><td>173,120</td></tr><tr><td>NE61<a id="crosref1535" href="#tblfn8"><sup>h</sup></a></td><td>U</td><td>AR3.4K_LowCov</td><td>–</td><td>3,485–3,360</td><td>45.88</td><td>126.22</td><td>SS</td><td>25</td><td>5</td><td>0.1</td><td>D4b1a2a</td><td>–</td><td>574</td><td>0.022</td><td>26,464</td></tr></tbody></table></div><figcaption><div class="caption"><span class="heading">Table 1</span><div id="tspara0010" role="paragraph">Newly sampled individuals included in this study</div></div><div class="notes"><div id="tspara0015" role="paragraph">See also <a id="crosref1540" href="#mmc2">Table S1</a>.</div><div role="doc-footnote" data-has="label"><div class="label">a</div><div id="tblfn1"><div id="ntpara0015" role="paragraph">Date (cal BP), calibrated date with 99.7% confidence interval using OxCal v.4.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Ramsey and Lee, 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Ramsey, C.B. ∙ Lee, S.</div><div><strong>Recent and planned developments of the program OxCal</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:720-730</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0033822200057878" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0033822200057878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the IntCal20 calibration curve (<span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Reimer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Reimer, P.J. ∙ Austin, W.E.N. ∙ Bard, E. ...</div><div><strong>The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP)</strong></div><div><em>Radiocarbon.</em> 2020; <strong>62</strong>:725-757</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2020.41" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85092423467" target="_blank" aria-label="Open Scopus in new tab">Scopus (3774)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2020.41" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div><div role="doc-footnote" data-has="label"><div class="label">b</div><div id="tblfn2"><div id="ntpara0020" role="paragraph">SS, single-stranded DNA library; DS-half, double-stranded DNA library with half uracil-DNA-glycosylase (UDG) treatment.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">c</div><div id="tblfn3"><div id="ntpara0025" role="paragraph">Contamination was estimated as a percentage.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">d</div><div id="tblfn4"><div id="ntpara0030" role="paragraph">MT_hg, mitochondrial haplotype group.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">e</div><div id="tblfn5"><div id="ntpara0035" role="paragraph">Y_hg, Y-chromosomal haplotype group.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">f</div><div id="tblfn6"><div id="ntpara0040" role="paragraph">NE57, NE36, NE44 and NE-18 had SNP numbers of less than 25,000.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">g</div><div id="tblfn7"><div id="ntpara0045" role="paragraph">NE30 had second-degree kinship with NE-4.</div></div></div><div role="doc-footnote" data-has="label"><div class="label">h</div><div id="tblfn8"><div id="ntpara0050" role="paragraph">NE58 and NE61 had SNP numbers of less than 50,000.</div></div></div></div><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=tbl1&amp;pii=S0092-8674%2821%2900575-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Tianyuan-related ancestry was widespread geographically and temporally before the LGM</h3><div id="p0040" role="paragraph">Upper Pleistocene human population dynamics prior to the LGM stadial are largely unknown in East Asia because of limited fossil remains from this period (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Bae et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bae, C.J. ∙ Douka, K. ∙ Petraglia, M.D.</div><div><strong>On the origin of modern humans: Asian perspectives</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:eaai9067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai9067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85037608455" target="_blank" aria-label="Open Scopus in new tab">Scopus (228)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217544/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai9067&amp;pmid=29217544" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To better understand the early settlement of modern humans in East Asia, especially northern East Asia, we investigated the genomic profile of the pre-LGM (33.6 ka) AR33K specimen, a female individual. We first surveyed how this AR33K individual related to other ancient and modern populations globally using outgroup-<i>f</i><sub><i>3</i></sub> analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interestingly, outgroup-<i>f</i><sub><i>3</i></sub> results showed that AR33K and the 7,000-years-older Tianyuan individual share more genetic drift with each other than with all other tested populations (<a id="crosref0285" href="#fig1">Figures 1</a>C and <a id="crosref0290" href="#fig2">2</a>A). Symmetry tests using <i>D</i> statistics (<span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) also show that AR33K and Tianyuan form a cluster relative to all other ancient and modern populations globally (<a id="crosref0300" href="#figs2">Figure&nbsp;S2</a>), which is also supported by results from <i>qpGraph</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref60-3" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-3" aria-controls="sref60-3">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0310" href="#fig3">Figure&nbsp;3</a>A) and the maximum-likelihood tree (<span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Pickrell and Pritchard, 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Pickrell, J.K. ∙ Pritchard, J.K.</div><div><strong>Inference of population splits and&nbsp;mixtures from genome-wide allele frequency data</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1002967</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1002967" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869026948" target="_blank" aria-label="Open Scopus in new tab">Scopus (1556)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23166502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1002967&amp;pmid=23166502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0320" href="#figs3">Figure&nbsp;S3</a>). Additionally, these analyses placed AR33K and Tianyuan basal to all East Asians (<a id="crosref0325" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0330" href="#figs3">Figure&nbsp;S3</a>), and analyses for detecting archaic introgressions (<span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Peter, 2020</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Peter, B.M.</div><div><strong>100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.13.990523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098896511" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.13.990523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) show that they exhibit an excess of Denisovan ancestry compared with younger populations (<a id="crosref0340" href="#mmc1">Table S2</a>), similar to that reported by <span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Massilani et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Massilani, D. ∙ Skov, L. ∙ Hajdinjak, M. ...</div><div><strong>Denisovan ancestry and population history of early East Asians</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:579-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc1166" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33122380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc1166&amp;pmid=33122380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. It has been shown previously that the Tianyuan individual shared more alleles with an ∼35,000-year-old individual from Belgium (GoyetQ116-1; <span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Fu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (596)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) than with other contemporary western Eurasians and shared more alleles with modern Amazonian populations (i.e., Surui) than other Native Americans (<span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">Yang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Despite the high genetic similarity between AR33K and Tianyuan, we find that AR33K does not share the same genetic affinity as Tianyuan to GoyetQ116-1. Instead, AR33K presents a profile similar to other ancient East Asians, showing a trend of elevated GoyetQ116-1 ancestry compared with other ancient western Eurasians, but just at or below the cutoff for statistical significance (3 standard deviations from the mean). We find no evidence that AR33K shares more alleles with Surui than other Native Americans (<a id="crosref0360" href="#mmc3">Table S3</a>). Thus, a complex population structure may already have existed before the LGM stadial in northern East Asia.</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/cb78a1fb-c94f-4a75-81d9-ba20a37c81ff/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2021.04.040/asset/460d6feb-e7b2-46a0-ba36-6e215e149841/main.assets/gr2.jpg" height="574" width="506" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/cb78a1fb-c94f-4a75-81d9-ba20a37c81ff/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Genetic characteristics of AR33K and AR19K</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0040" role="paragraph">(A) Shared genetic drift between AR33K and published ancient populations older than 5 ka, using outgroup-<i>f</i><sub><i>3</i></sub> analysis. A higher value (represented by warmer colors) indicates a higher shared drift with AR33K. The intersection of the dotted lines indicates the location of AR33K.</div><div id="fspara0045" role="paragraph">(B) Results for <i>D</i>(North, Qihe; AR19K, Mbuti) and <i>D</i>(AR19K, Qihe; North, Mbuti). North represents northern East Asians, including the ancient Amur region populations (red), ancient Shandong populations (purple), ancient West Liao River population (orange), ancient Mongolian populations (green), ancient inner Mongolian populations (blue), and ancient Yellow River population (brown). The dashed line indicates where <i>D</i>&nbsp;= 0. A solid circle indicates that |<i>Z</i>| &gt; 3, and otherwise |<i>Z</i>|&nbsp;&lt; 3. The thick gray bar indicates one standard error, and the thin gray bar indicates two standard errors. Directions of deviations of <i>D</i> statistics are labeled with population name on the x axis at the top left and right of the panels. All SNPs were used to compute <i>D</i> statistics.</div><div id="fspara0050" role="paragraph">See also <a id="crosref1395" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref1400" href="#mmc3">Table S3</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/549534d7-7c1b-4ad4-80c2-4834f78ad6f5/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2021.04.040/asset/423b9c27-6623-413a-8eff-b719a292c582/main.assets/gr3.jpg" height="369" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/549534d7-7c1b-4ad4-80c2-4834f78ad6f5/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Admixture graph modeling with <i>qpGraph</i> and admixture modeling with <i>qpAdm</i> for Ancient Paleo-Siberians (Kolyma and UKY)</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0060" role="paragraph">(A and B) Admixture graph modeling with <i>qpGraph</i> (A) and admixture modeling with <i>qpAdm</i> (B) for Ancient Paleo-Siberians (Kolyma and UKY). Color coding for USR1 and AR14K is the same in both graphs. In <i>qpGraph</i>, yellow represents Tianyuan and AR33K, brown represents coastal northern East Asian ancestry, green represents coastal southern East Asian ancestry, and gray represents non-East Asian ancestries. In <i>qpAdm</i>, the error bars represent the standard errors of estimated ancestry proportions. See also <a id="crosref1405" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref1410" href="#mmc4">Table S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/eaa648b5-d35f-49cd-88c8-602d9b8fbe5b/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2021.04.040/asset/0f2bb9aa-f983-440f-a49f-db0e28d2cd2e/main.assets/figs1.jpg" height="663" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/eaa648b5-d35f-49cd-88c8-602d9b8fbe5b/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Related to <a id="crosref1435" href="#fig1">Figure&nbsp;1</a> and <a id="crosref1440" href="#mmc2">Tables S1</a> and <a id="crosref1445" href="#figs3">S3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0090" role="paragraph">(A) Principal components analysis (PCA) (including deleted newly sampled populations due to low quality and kinship). The ancient populations (above) are projected on the principal components of modern populations (below). New samples are highlighted in red text and larger symbols. PCA shows that AR9.9K_2d.rel.AR10.5K_deleted (second-degree relative of AR10.5K) shares a similar position with AR13-10K.</div><div id="fspara0095" role="paragraph">(B) The pairwise genetic affinity among ancient East Asians using outgroup-<i>f</i><sub><i>3</i></sub> analysis with low coverage samples included. Both PCA and outgroup-<i>f</i><sub><i>3</i></sub> analysis indicate that AR7.3K_LowCov and AR3.4K_LowCov share similar positions with AR9.2K_o in both PCA and outgroup-<i>f</i><sub><i>3</i></sub> analysis.</div><div id="fspara0100" role="paragraph">(C) Calibration of radiocarbon measurement for NE30 (AR9.9K_2d.rel.AR10.5K_deleted, who is the second-degree relative of AR10.5K). The y axis shows <sup>14</sup>C dates BP and the x axis shows the calibrated age obtained using <sup>14</sup>C dates calibrated using OxCal v4.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Ramsey and Lee, 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Ramsey, C.B. ∙ Lee, S.</div><div><strong>Recent and planned developments of the program OxCal</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:720-730</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0033822200057878" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0033822200057878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the IntCal20 calibration curve (<span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Reimer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Reimer, P.J. ∙ Austin, W.E.N. ∙ Bard, E. ...</div><div><strong>The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP)</strong></div><div><em>Radiocarbon.</em> 2020; <strong>62</strong>:725-757</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2020.41" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85092423467" target="_blank" aria-label="Open Scopus in new tab">Scopus (3774)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2020.41" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The temporal fluctuations in atmospheric <sup>14</sup>C is reflected by the progression of the <sup>14</sup>C calibration curve between approximately 10,200-9,800 cal BP.</div><div id="fspara0105" role="paragraph">(D) Analysis for the comparison of <i>D</i> statistics of AR33K (<i>D(</i>X,Y;AR33K,Mbuti)) and AR19K (<i>D(</i>X,Y;AR19K,Mbuti)) using all SNPs and transversion SNPs only. X and Y refer to the same population set as in <a id="crosref1460" href="#figs2">Figure&nbsp;S2</a>. x axis is the D or Z values using all SNPs. y axis is the D or Z values using transversion SNPs only. The regression coefficient and coefficient of determination (R<sup>2</sup>) are labeled. These results indicate that the lower Z values when using transversion SNPs is due only to the larger standard error when using fewer SNPs.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/cfe78eb5-34af-4645-9e38-5817f9e20b00/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2021.04.040/asset/50405999-da6a-4885-9dd0-343aa993a961/main.assets/figs2.jpg" height="589" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/cfe78eb5-34af-4645-9e38-5817f9e20b00/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Genetic cluster of AR33K and Tianyuan, demonstrated using <i>D</i> statistics, related to <a id="crosref1465" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0115" role="paragraph">Three forms of <i>D</i> statistics showed that <i>D(</i>X, Tianyuan; AR33K, Mbuti)&nbsp;&lt; 0 (Z&nbsp;&lt; −3), <i>D(</i>X, AR33K; Tianyuan, Mbuti)&nbsp;&lt; 0 (Z&nbsp;&lt; −3), and <i>D(</i>Tianyuan, AR33K; X, Mbuti) ~0 (−3&nbsp;&lt; Z&nbsp;&lt; 3). X represents the global ancient (i.e., older than 5 ka) and modern populations. Ancient populations are sorted by age. The dashed line indicates where D&nbsp;= 0. Solid circles indicate that |Z| &gt; 3 and otherwise |Z|&nbsp;&lt; 3. The thick gray bar indicates one standard error and the thin gray bar indicates two standard errors. Directions of deviations of <i>D</i> statistics are labeled with population name on the x axis at the top left and right of figure panels. All SNPs were used to compute <i>D</i> statistics.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/22a6ab84-2849-4a63-8c20-940ee85bd534/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2021.04.040/asset/2fef6af1-414e-4075-a26a-991d5c7b8496/main.assets/figs3.jpg" height="699" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/22a6ab84-2849-4a63-8c20-940ee85bd534/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Inferred maximum-likelihood-based trees using Treemix, related to <a id="crosref1470" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0125" role="paragraph">A maximum likelihood tree allowing four migration events is the most supported using 1,000 bootstrap runs (highlighted with red rectangle). The results from <i>Treemix</i> are consistent with <i>qpGraph</i> modeling.</div></div></div></div></figcaption></figure></div><div id="p0045" role="paragraph">AR33K (∼33 ka) is roughly contemporary with Salkhit (∼34 ka) from the Salkhit Valley (northeastern Mongolia) and located at a similar distance from Salkhit (∼1,159&nbsp;km) (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Devièse et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Devièse, T. ∙ Massilani, D. ∙ Yi, S. ...</div><div><strong>Compound-specific radiocarbon dating and mitochondrial DNA analysis of the Pleistocene hominin from Salkhit Mongolia</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-08018-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060861342" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30700710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-08018-8&amp;pmid=30700710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47-3" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-3" aria-controls="sref47-3">Massilani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref47-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Massilani, D. ∙ Skov, L. ∙ Hajdinjak, M. ...</div><div><strong>Denisovan ancestry and population history of early East Asians</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:579-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc1166" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33122380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc1166&amp;pmid=33122380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Tianyuan cave (∼1,114&nbsp;km) (<span class="dropBlock reference-citations"><a id="body-ref-sref92-3" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-3" aria-controls="sref92-3">Yang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The Salkhit individual has been proposed recently as an admixture of Tianyuan-related (∼75%) and Yana-related (∼25%) ancestries (<span class="dropBlock reference-citations"><a id="body-ref-sref47-4" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-4" aria-controls="sref47-4">Massilani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref47-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Massilani, D. ∙ Skov, L. ∙ Hajdinjak, M. ...</div><div><strong>Denisovan ancestry and population history of early East Asians</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:579-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc1166" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33122380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc1166&amp;pmid=33122380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similar to Tianyuan, Salkhit also shows a connection with GoyetQ116-1 (<span class="dropBlock reference-citations"><a id="body-ref-sref47-5" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-5" aria-controls="sref47-5">Massilani et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref47-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Massilani, D. ∙ Skov, L. ∙ Hajdinjak, M. ...</div><div><strong>Denisovan ancestry and population history of early East Asians</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:579-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc1166" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33122380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc1166&amp;pmid=33122380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, unlike Salkhit, AR33K does&nbsp;not possess more Yana-related ancestry than Tianyuan (<i>D</i>(Salkhit, Tianyuan; Yana, Mbuti) &gt; 0 [<i>Z</i>&nbsp;= 7.55]; <i>D</i>(Tianyuan, AR33K; Yana, Mbuti) ∼0 [<i>Z</i>&nbsp;= 1.88]). This probably indicates that Tianyuan-related ancestry was widespread before the LGM stadial in northern East Asia, geographically (from the North China Plain near present-day Beijing to the Salkhit Valley in northeastern Mongolia, and the Amur region of northeastern China) and temporally (from 40–33 ka). People with Tianyuan-related ancestry may have admixed with people of Yana-related ancestry in Mongolia while remaining isolated in the Amur region before the LGM.</div></section><section id="sec-2-3"><h3>The appearance of the earliest northern East Asian at the end of the LGM</h3><div id="p0050" role="paragraph">The LGM stadial had a large effect on prehistoric population dynamics in Europe (<span class="dropBlock reference-citations"><a id="body-ref-sref17-3" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-3" aria-controls="sref17-3">Fu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref17-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (596)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-3" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-3" aria-controls="sref87-3">Tallavaara et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref87-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Tallavaara, M. ∙ Luoto, M. ∙ Korhonen, N. ...</div><div><strong>Human population dynamics in Europe over the Last Glacial Maximum</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:8232-8237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1503784112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84936803076" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1503784112&amp;pmid=26100880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), however, little is known about how the profound climatic changes of this period affected populations in northern East Asia. Here we report genetic evidence from an East Asian individual who lived during the LGM (26.5–19 ka), AR19K, which is critical for understanding the population dynamics of northern East Asians at the end of this harsh period. We first assessed AR19K using principal component analysis (PCA) (<a id="crosref0400" href="#fig1">Figure&nbsp;1</a>B) and outgroup-<i>f</i><sub><i>3</i></sub> analyses (<a id="crosref0405" href="#fig1">Figure&nbsp;1</a>C), showing that AR19K has more genetic affinities with later East Asians compared with the earlier East&nbsp;Asians. The AR33K and Tianyuan individuals share a specific genetic drift that is not shared by AR19K and younger East Asians, a relationship supported by <i>qpGraph</i> modeling (<a id="crosref0410" href="#fig3">Figure&nbsp;3</a>A) and <i>Treemix</i> (<a id="crosref0415" href="#figs3">Figure&nbsp;S3</a>). This specific drift is also reflected by <i>D</i>(Tianyuan, AR33K; AR19K/AR14K/AR13-10K/ARpost9K, Mbuti) ∼0 (−0.6&nbsp;&lt; <i>Z</i>&nbsp;&lt; 1), <i>D</i>(AR19K/AR14K/AR13-10K/ARpost9K, Tianyuan; AR33K, Mbuti)&nbsp;&lt; 0 (−11.1&nbsp;&lt; <i>Z</i>&nbsp;&lt; −10.2), and <i>D</i>(AR19K/AR14K/AR13-10K/ARpost9K, AR33K; Tianyuan, Mbuti)&nbsp;&lt; 0 (−11&nbsp;&lt; <i>Z</i>&nbsp;&lt; −9.4) (<a id="crosref0420" href="#figs2">Figure&nbsp;S2</a>). These observations may suggest that a population change in northern East Asia had already occurred at the end of the LGM, earlier than the recently proposed post-LGM population change demonstrated by a 16,900-year-old subadult female (Khaiyrgas-1) found in Khaiyrgas Cave in Yakutia (Russia) (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Kılınç et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Kılınç, G.M. ∙ Kashuba, N. ∙ Koptekin, D. ...</div><div><strong>Human population dynamics and <i>Yersinia pestis</i> in ancient northeast Asia</strong></div><div><em>Sci. Adv.</em> 2021; <strong>7</strong>:eabc4587</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.abc4587" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85099219538" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33523963/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.abc4587&amp;pmid=33523963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This proposed population change may have occurred even earlier in the LGM because genetic continuity between 22 ka and 18 ka has been noted in individuals from the Mal’ta and Afontova Gora sites farther west in Siberia (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Raghavan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Raghavan, M. ∙ Skoglund, P. ∙ Graf, K.E. ...</div><div><strong>Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:87-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12736" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84892367625" target="_blank" aria-label="Open Scopus in new tab">Scopus (540)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24256729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12736&amp;pmid=24256729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Additional genetic and archaeological evidence during the LGM stadial in the Amur River Basin is needed to resolve the timing of the appearance of the lineage represented by AR19K and later individuals in this region.</div><div id="p0055" role="paragraph">It has been demonstrated recently that, by 8 ka at the latest, ancient coastal northern East Asian (aCNEA; including populations from the Chinese sites of Bianbian, Boshan, Xiaogao, and Xiaojingshan) and ancient coastal southern East Asian (aCSEA; including individuals from the Chinese sites of Qihe, Liangdao1, Liangdao2, Xitoucun, and Tanshishan) populations could already be separated into two distinct northern and southern genetic groups (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found that AR19K, along with the other newly sampled individuals reported here (from 14–6 ka), was genetically closer to aCNEA than to aCSEA (represented by Qihe) using <i>D</i>(aCNEA, Qihe; AR19K, Mbuti) (<a id="crosref0440" href="#fig2">Figure&nbsp;2</a>B). The balanced <i>D</i>(AR19K, Qihe; aCNEA, Mbuti) for Xiaojingshan and Bianbian could be explained by their possessing relatively more aCSEA ancestry than other aCNEAs (<a id="crosref0445" href="#mmc3">Table S3</a>). Interestingly, AR19K is found to be basal to ancient northern East Asians younger than 14 ka in admixture graph modeling using <i>qpGraph</i> and <i>Treemix</i> (<a id="crosref0450" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0455" href="#figs3">Figure&nbsp;S3</a>). These results reveal the existence of a north-south separation as early as 19 ka, approximately 10,000 years earlier than observed previously (<span class="dropBlock reference-citations"><a id="body-ref-sref93-3" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-3" aria-controls="sref93-3">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, ancient northern East Asians younger than 14 ka share less genetic drift with AR19K than AR14K (<a id="crosref0465" href="#mmc3">Table S3</a>), which is consistent with the basal position of AR19K for aCNEA. The analyses of AR19K, who lived toward the end of the LGM stadial, reveal this individual to be the earliest northern East Asian yet identified, thus having some degree of continuity with present-day northern East Asians and possessing a genetic ancestry distinct from that of the modern humans who occupied this region prior to the LGM (e.g., Tianyuan and AR33K).</div></section><section id="sec-2-4"><h3>Genetic continuity in the Amur region and population interactions after the LGM</h3><div id="p0060" role="paragraph">Ancient DNA samples recovered from Neolithic forager/farmers at Devil’s Gate cave (DevilsCave_N) in Primorsky Krai, far eastern Russia and the Amur River Basin reveal that genetic continuity with modern populations existed in the Amur region since 8 ka (<span class="dropBlock reference-citations"><a id="body-ref-sref57-2" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-2" aria-controls="sref57-2">Ning et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref57-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Ning, C. ∙ Li, T. ∙ Wang, K. ...</div><div><strong>Ancient genomes from northern China suggest links between subsistence changes and human migration</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2700</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16557-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085859685" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16557-2&amp;pmid=32483115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Siska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Siska, V. ∙ Jones, E.R. ∙ Jeon, S. ...</div><div><strong>Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago</strong></div><div><em>Sci. Adv.</em> 2017; <strong>3</strong>:e1601877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1601877" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85020941932" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28164156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1601877&amp;pmid=28164156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, population dynamics in the Amur region are unclear before 8 ka. Our younger samples were clustered into four subgroups based on genetic affinity and time, including AR14K, AR13-10K, ARpost9K, and AR9.2K_o (<a id="crosref0485" href="#fig1">Figures 1</a>B and 1C). This increased sampling from a large temporal range after the LGM stadial allows a better resolution of population dynamics in the Amur region and surrounding areas. Overall, PCA (<a id="crosref0490" href="#fig1">Figure&nbsp;1</a>B), outgroup-<i>f</i><sub><i>3</i></sub> (<a id="crosref0495" href="#fig1">Figure&nbsp;1</a>C), and <i>ADMIXTURE</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Alexander et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (5179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0505" href="#figs4">Figure&nbsp;S4</a>) analyses indicate that AR14K, AR13-10K, and&nbsp;ARpost9K are genetically closest to the DevilsCave_N population. This was validated because <i>D</i> statistics show that post-LGM Amur region populations share more alleles with DevilsCave_N relative to other ancient northern East Asians (<a id="crosref0510" href="#fig4">Figure&nbsp;4</a>A), which is also supported by results from <i>qpGraph</i> and <i>Treemix</i> (<a id="crosref0515" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0520" href="#figs3">Figure&nbsp;S3</a>). In addition, we observed a decrease in human background relatedness, as measured by short runs of homozygosity (ROHs) (4–8 cM) after 14 ka, which suggests an increasing local population size in the Amur region, probably because of the transition to sedentary farming (the “Neolithic transition”) (<a id="crosref0525" href="#mmc2">Table S4</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Bacci, 2017</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Bacci, M.L.</div><div><strong>A concise history of world population</strong></div><div>John Wiley &amp; Sons, 2017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/9781119406822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F9781119406822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Diamond and Bellwood, 2003</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Diamond, J. ∙ Bellwood, P.</div><div><strong>Farmers and their languages: the first expansions</strong></div><div><em>Science.</em> 2003; <strong>300</strong>:597-603</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1078208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-0037466612" target="_blank" aria-label="Open Scopus in new tab">Scopus (638)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12714734/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1078208&amp;pmid=12714734" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). One outlier population, AR9.2K_o, is genetically close to ancient populations in the Amur region after 14 ka. Interestingly, this individual also shares some genetic affinity with ancient Shandong populations (<a id="crosref0540" href="#mmc3">Table S3</a>), but this possible population interaction needs to be confirmed by further sampling. Our results demonstrate that the genetic continuity reported between modern inhabitants of the Amur River Basin and the Devil’s Gate cave population probably started as early as 14 ka, 6,000 years earlier than proposed previously (<span class="dropBlock reference-citations"><a id="body-ref-sref57-3" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-3" aria-controls="sref57-3">Ning et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref57-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Ning, C. ∙ Li, T. ∙ Wang, K. ...</div><div><strong>Ancient genomes from northern China suggest links between subsistence changes and human migration</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2700</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16557-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085859685" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16557-2&amp;pmid=32483115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Siska et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Siska, V. ∙ Jones, E.R. ∙ Jeon, S. ...</div><div><strong>Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago</strong></div><div><em>Sci. Adv.</em> 2017; <strong>3</strong>:e1601877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1601877" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85020941932" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28164156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1601877&amp;pmid=28164156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This is consistent with the archaeological record of the earliest appearance of pottery in the Amur region (∼15 ka) (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Wang, Y. ∙ Song, F. ∙ Zhu, J. ...</div><div><strong>GSA: Genome Sequence Archive&lt;sup/&gt;</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2017; <strong>15</strong>:14-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2017.01.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85012868681" target="_blank" aria-label="Open Scopus in new tab">Scopus (441)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28387199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2017.01.001&amp;pmid=28387199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96-1" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-1" aria-controls="sref96-1">Yue et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Yue, J.-P. ∙ Li, Y.-Q. ∙ Yang, S.</div><div><strong>Neolithisation in the southern Lesser Khingan Mountains: lithic technologies and ecological adaptation</strong></div><div><em>Antiquity.</em> 2019; <strong>93</strong>:1144-1160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15184/aqy.2019.92" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85073781764" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15184%2Faqy.2019.92" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In addition, the great biological diversity of the Amur region enabled humans to adopt varied, rich subsistence strategies, including hunting, fishing, and animal husbandry, to support this genetic continuity (<span class="dropBlock reference-citations"><a id="body-ref-sref57-4" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57-4" aria-controls="sref57-4">Ning et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref57-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Ning, C. ∙ Li, T. ∙ Wang, K. ...</div><div><strong>Ancient genomes from northern China suggest links between subsistence changes and human migration</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2700</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16557-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085859685" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16557-2&amp;pmid=32483115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/f612fb28-baa4-4f70-a374-9c0b744e0f52/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2021.04.040/asset/981ecd5b-36aa-431e-a069-279fb98dc540/main.assets/gr4.jpg" height="640" width="775" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/f612fb28-baa4-4f70-a374-9c0b744e0f52/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Ancient populations in the Amur region after 19 ka and time transect of <i>EDAR V370A</i></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0070" role="paragraph">(A) Genetic affinities with DevilsCave_N and post-LGM populations in the Amur region (AR14K, AR13-10K, and ARpost9K) with <i>D</i> statistics. North represents northern East Asians and Ancient Paleo-Siberians, including Ancient Paleo-Siberians (Kolyma and UKY; pink), ancient Amur region populations (red), ancient Shandong populations (purple), ancient West Liao River population (orange), ancient Mongolian populations (green), ancient inner Mongolian populations (blue), and ancient Yellow River population (brown). Directions of deviations of <i>D</i> statistics are labeled with the population name on the x axis at the top left and right. All SNPs were used to compute <i>D</i> statistics.</div><div id="fspara0075" role="paragraph">(B) Allele counts for adaptive mutations of <i>EDAR V370A</i> in Tianyuan, ancient populations in the Amur region, and recently published ancient northern East Asians (Yumin, Bianbian, Boshan, Xiaogao, and Xiaojingshan) (<span class="dropBlock reference-citations"><a id="body-ref-sref93-4" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-4" aria-controls="sref93-4">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) from 40–6 ka. Purple shading represents the period of the LGM. Dots represent genotype calls (blue, ancestral alleles; red, derived alleles; gray, missing alleles). The number at the top of the bar (separated by a comma) shows the allele coverage for the derived and ancestral allele, and the number in parentheses at the bottom of the bar shows the number of individuals included in that time column. Genotypes were called using a maximum-likelihood-based method (<i>snpAD</i>) that takes into account possible errors in the covered alleles.</div><div id="fspara0080" role="paragraph">See also <a id="crosref1420" href="#figs4">Figure&nbsp;S4</a>, <a id="crosref1425" href="#mmc1">Table S6</a>, and <a id="crosref1430" href="#sec-3">STAR Methods</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2021.04.040/asset/61fe1ca6-d816-46e3-afc3-d0478fccdd14/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2021.04.040/asset/3976b942-d42e-4628-ba54-673d5bcb77c7/main.assets/figs4.jpg" height="958" width="522" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/61fe1ca6-d816-46e3-afc3-d0478fccdd14/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Admixture plot for ancient and modern East Asians for K&nbsp;= 2 to K&nbsp;= 10, related to <a id="crosref1475" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0135" role="paragraph">(A) Admixture results for K&nbsp;= 2 to K&nbsp;= 10.</div><div id="fspara0140" role="paragraph">(B) Cross-validation results for different K values, where the lowest CV is when K&nbsp;= 4. The genetic profile of AR14K, AR13-10K, and ARpost9K are closest to the DevilsCave_N population.</div></div></div></div></figcaption></figure></div><div id="p0065" role="paragraph">An early human population defined as Ancient Paleo-Siberians (represented by the 10,000-year-old Kolyma individual from northeastern Siberia and the 14,000-year-old Ust-Kyakhta-3 or UKY specimen found near Lake Baikal in southern Siberia) have been proposed to have descended from Ancient North Eurasian (ANE)-related populations, mixing with newly arriving people carrying East Asian ancestry (<span class="dropBlock reference-citations"><a id="body-ref-sref83-3" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-3" aria-controls="sref83-3">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref83-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-1" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-1" aria-controls="sref95-1">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref95-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Yu, H. ∙ Spyrou, M.A. ∙ Karapetian, M. ...</div><div><strong>Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1232-1245.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085755497" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32437661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.037&amp;pmid=32437661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Ancient Paleo-Siberians have also been shown to be the closest relatives to Native American populations outside of the Americas and were associated with the spread of a microblade technology and post-LGM reduction of the Mammoth Steppe ecological community (<span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Pitulko and Nikolskiy, 2012</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Pitulko, V.V. ∙ Nikolskiy, P.A.</div><div><strong>The extinction of the woolly mammoth and the archaeological record in Northeastern Asia</strong></div><div><em>World Archaeol.</em> 2012; <strong>44</strong>:21-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/00438243.2012.647574" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84858129375" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F00438243.2012.647574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-4" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-4" aria-controls="sref83-4">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref83-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the modeling of Ancient Paleo-Siberians as a two-way mixture of DevilsCave_N (representing East Asian ancestry) and AfontovaGora3 (<span class="dropBlock reference-citations"><a id="body-ref-sref17-4" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-4" aria-controls="sref17-4">Fu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref17-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (596)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (representing ANE ancestry) was unsuccessful for UKY (tail probability&nbsp;= 1.45E–03) or Kolyma (tail probability&nbsp;= 3.98E–08) (<span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Yu, H. ∙ Spyrou, M.A. ∙ Karapetian, M. ...</div><div><strong>Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1232-1245.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085755497" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32437661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.037&amp;pmid=32437661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), indicating East Asian populations that could have feasibly contributed to the Ancient Paleo-Siberians have not yet been identified. We explored the relationship between the northern East Asian samples older than 13,000 years (AR19K and AR14K) and Ancient Paleo-Siberians. Using <i>qpAdm</i>, we showed that both UKY and Kolyma could be successfully modeled as a mixture of AR19K/AR14K and USR1 (<a id="crosref0605" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref0610" href="#mmc4">Table S5</a>). It was also demonstrated through <i>qpGraph</i> that the AR14K-related population could be the direct East Asian source for Ancient Paleo-Siberians (<a id="crosref0615" href="#fig3">Figure&nbsp;3</a>A). This is consistent with <i>D</i> statistics that indicate that Ancient Paleo-Siberians were genetically closer to Amur region populations after 14 ka (<i>D</i>(AR14K/AR13-10K/ARpost9K, AR19K; UKY, Mbuti) [2.5&nbsp;&lt; <i>Z</i>&nbsp;&lt; 3.9]) than to other ancient northern East Asians (i.e., <i>D</i>(aCNEA, AR19K; UKY, Mbuti) [0.7&nbsp;&lt; <i>Z</i>&nbsp;&lt; 1.4]) (<a id="crosref0620" href="#mmc3">Table S3</a>). We propose that Amur region ancestry after 14 ka is a better fit than DevilsCave_N ancestry for the East Asian component in Ancient Paleo-Siberians and, thus, that Amur region populations could have been at the forefront of interactions with ANE-related populations that likely contributed to Ancient Paleo-Siberians.</div></section><section id="sec-2-5"><h3>Adaptive genetic variants in northern East Asians from 40 to 6 ka</h3><div id="p0070" role="paragraph">Even though selection has been inferred in western Eurasians utilizing ancient DNA (<span class="dropBlock reference-citations"><a id="body-ref-sref40" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40" aria-controls="sref40">Lindo et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Lindo, J. ∙ Haas, R. ∙ Hofman, C. ...</div><div><strong>The genetic prehistory of the Andean highlands 7000 years BP though European contact</strong></div><div><em>Sci. Adv.</em> 2018; <strong>4</strong>:eaau4921</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.aau4921" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85056346614" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30417096/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.aau4921&amp;pmid=30417096" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Mathieson et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Mathieson, I. ∙ Lazaridis, I. ∙ Rohland, N. ...</div><div><strong>Genome-wide patterns of selection in 230 ancient Eurasians</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:499-503</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84951286136" target="_blank" aria-label="Open Scopus in new tab">Scopus (863)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26595274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16152&amp;pmid=26595274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Ye et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Ye, K. ∙ Gao, F. ∙ Wang, D. ...</div><div><strong>Dietary adaptation of FADS genes in Europe varied across time and geography</strong></div><div><em>Nat. Ecol. Evol.</em> 2017; <strong>1</strong>:167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-017-0167" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031759069" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29094686/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-017-0167&amp;pmid=29094686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the limited availability of ancient East Asian genomes has precluded similar research in East Asia. The data presented here, together with those from recently published ancient East Asian populations (<span class="dropBlock reference-citations"><a id="body-ref-sref93-5" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-5" aria-controls="sref93-5">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), provide a large temporal window from 40–6 ka (spanning the pre- and post-LGM periods) to understand the evolution of adaptive variants. One important adaptive variant for East Asians is the <i>EDAR V370A</i> mutation (frequency at 93.7% in Han Chinese in Beijing, China [CHB]; <span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (10306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which has been demonstrated through genome-wide association studies to be associated with thicker hair shafts, more sweat glands, and shovel-shaped incisors (<span class="dropBlock reference-citations"><a id="body-ref-sref18" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18" aria-controls="sref18">Fujimoto et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Fujimoto, A. ∙ Ohashi, J. ∙ Nishida, N. ...</div><div><strong>A replication study confirmed the EDAR gene to be a major contributor to population differentiation regarding head hair thickness in Asia</strong></div><div><em>Hum. Genet.</em> 2008; <strong>124</strong>:179-185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-008-0537-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-50649098835" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18704500/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-008-0537-1&amp;pmid=18704500" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Tan et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Tan, J. ∙ Yang, Y. ∙ Tang, K. ...</div><div><strong>The adaptive variant EDARV370A is associated with straight hair in East Asians</strong></div><div><em>Hum. Genet.</em> 2013; <strong>132</strong>:1187-1191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-013-1324-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84884900721" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23793515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-013-1324-1&amp;pmid=23793515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, the precise mechanism behind <i>EDAR V370A</i> selection is difficult to detect using evidence obtained only from modern genomes. Two selective mechanisms have been proposed: selection to increase vitamin D in breast milk in the low-UV environment around 20 ka or selection to modulate thermoregulatory sweating during a warm and humid climate around 30 ka (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Hlusko et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Hlusko, L.J. ∙ Carlson, J.P. ∙ Chaplin, G. ...</div><div><strong>Environmental selection during the last ice age on the mother-to-infant transmission of vitamin D and fatty acids through breast milk</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E4426-E4432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1711788115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85046658692" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29686092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1711788115&amp;pmid=29686092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Kamberov et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Kamberov, Y.G. ∙ Wang, S. ∙ Tan, J. ...</div><div><strong>Modeling recent human evolution in mice by expression of a selected EDAR variant</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:691-702</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2013.01.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2013.01.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84874048561" target="_blank" aria-label="Open Scopus in new tab">Scopus (209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.016&amp;pmid=23415220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here we show that mutation <i>V370A</i> in the <i>EDAR</i> gene appeared in all ancient East Asians (including AR19K), except for AR33K and Tianyuan, the only two individuals in our sample predating the LGM stadial in East Asia (<a id="crosref0670" href="#fig4">Figure&nbsp;4</a>B; <a id="crosref0675" href="#mmc2">Table S6</a>). This indicates that <i>EDAR V370A</i> was likely to be elevated to high frequency during or shortly after the LGM. These direct observations using ancient DNA make the hypothesis that <i>EDAR V370A</i> was under selection during a warm and humid environment less likely (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Kamberov et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Kamberov, Y.G. ∙ Wang, S. ∙ Tan, J. ...</div><div><strong>Modeling recent human evolution in mice by expression of a selected EDAR variant</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:691-702</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2013.01.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2013.01.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84874048561" target="_blank" aria-label="Open Scopus in new tab">Scopus (209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.016&amp;pmid=23415220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, the allelic age of <i>EDAR V370A</i> has been estimated previously to be 11,400 years ago with a large confidence interval (4,300–43,700) using modern genomes (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Peter et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Peter, B.M. ∙ Huerta-Sanchez, E. ∙ Nielsen, R.</div><div><strong>Distinguishing between selective sweeps from standing variation and from a de novo mutation</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1003011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84868152411" target="_blank" aria-label="Open Scopus in new tab">Scopus (138)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23071458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003011&amp;pmid=23071458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), whereas our direct observation demonstrated that this allele emerged as early as ∼19 ka, as observed in the AR19K individual (<a id="crosref0690" href="#fig4">Figure&nbsp;4</a>B), providing an older and more accurate upper boundary for the allelic age estimate.</div><div id="p0075" role="paragraph">In conclusion, our findings reveal the demographic changes that occurred over a long period in northern East Asia, including before the LGM stadial (∼33 ka) and after (from ∼19–6 ka). This temporal transect witnessed two major population shifts before 14 ka, followed by long-term genetic continuity in the Amur region. (1) Before the LGM, Tianyuan-related ancestry was widespread with no evidence of Yana-related ancestry in the Amur region. (2) At the close of the LGM (∼19 ka), Tianyuan-related ancestry was likely replaced by modern East Asian ancestry. Also, the earliest northern East Asian population, represented by AR19K, appeared in the Amur region during the last stage of the LGM and is basal to all ancient northern East Asians. (3) After 14 ka, populations in the Amur region maintained genetic continuity and are the closest East Asian source known for Ancient Paleo-Siberians, one branch of which represents the closest relative of Native American populations outside of the Americas. In addition to uncovering previously unknown population dynamics, our analyses provide the ancient DNA evidence to narrow the timing of the appearance of <i>EDAR V370A</i>, indicating that this genetic variant was likely elevated to high frequency during or shortly after the LGM.</div></section><section id="sec-2-6"><h3>Limitations of the study</h3><div id="p0080" role="paragraph">Although this study presents a long time transect of ancient individuals from the Late Pleistocene to the Holocene, some of the detailed population dynamics need to be verified through further sampling. First, we must collect more samples representing the LGM stadial to further elucidate the post-LGM population replacement we observed and adaptations to environmental shifts by northern East Asians. Second, we must collect additional samples dating to the period after 14 ka to verify possible population interactions between ancient populations in the Amur region and surrounding territories, such as coastal Shandong.</div></section></section><section id="sec-3"><h2>STAR★Methods</h2><section id="sec-3-1"><h3>Key resources table</h3><div id="p0085" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Biological samples</b></td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR33K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR19K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR14.5K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR14.1K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR12K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR11K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR11K_deleted</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR10.6K_deleted</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR10.6K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR10.5K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR9.9K_2d.rel.AR10.5K_deleted</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR9.2K_o</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR9.2K_deleted</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR8.9K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR8.5K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR8.3K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR8.1K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR7.3K_LowCov</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR7K_deleted</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR7K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR6.87K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR6.84K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR6.33K</td></tr><tr><td>Ancient skeletal element</td><td>This paper</td><td>AR6.32K</td></tr><tr class="rowsep"><td>Ancient skeletal element</td><td>This paper</td><td>AR3.4K_LowCov</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>Tween-20</td><td>Sigma</td><td>cat.no. P5927</td></tr><tr><td>1&nbsp;M Tris-HCl, pH 8.0</td><td>AppliChem</td><td>A4577,0500</td></tr><tr><td>0.5&nbsp;M EDTA, pH 8.0</td><td>AppliChem</td><td>A4892,1000</td></tr><tr><td>5&nbsp;M NaCl</td><td>Sigma</td><td>S5150</td></tr><tr><td>25&nbsp;mM each dNTP mix</td><td>Fermentas</td><td>cat.no. R1121</td></tr><tr><td>USER enzyme (mixture of Uracil-DNA-glycosylase, UDG, and Endonuclease, EndoVIII)</td><td>NEB</td><td>cat.no. M5505L</td></tr><tr><td><i>Bst</i> DNA Polymerase, Large Fragment</td><td>NEB</td><td>cat.no. M0275S</td></tr><tr><td>ATP, 10&nbsp;mM stock solution</td><td>NEB</td><td>cat.no. 9804</td></tr><tr><td>T4 Polynucleotide Kinase (10&nbsp;U/μL)</td><td>NEB</td><td>cat.no. M0236L or S</td></tr><tr><td>T4 DNA Polymerase (3&nbsp;U/μL)</td><td>NEB</td><td>cat.no. M0203L</td></tr><tr><td>Quick Ligation Kit</td><td>NEB</td><td>cat.no. M2200L</td></tr><tr><td>2x HI-RPM hybridization buffer</td><td>Agilent</td><td>5118-5380</td></tr><tr><td>20% SDS</td><td>Serva</td><td>39575.01</td></tr><tr><td>SSC buffer</td><td>Ambion</td><td>AM9770</td></tr><tr><td>AmpliTaq Gold 10x PCR buffer without Mg</td><td>Life Technologies</td><td>4379874</td></tr><tr><td>1M NaOH</td><td>Sigma</td><td>71463-1L</td></tr><tr><td>3M Sodium acetate pH 5.2</td><td>Sigma</td><td>S7899</td></tr><tr><td>Dynabeads MyOne C1</td><td>Life Technologies</td><td>65002</td></tr><tr><td>SeraMag Speedbeads</td><td>GE</td><td>65152105050250</td></tr><tr class="rowsep"><td>Cot-1 DNA</td><td>Invitrogen</td><td>15279011</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited data</b></td></tr><tr><td>BAM files and genotype calls of the nuclear DNA for the newly sequenced individuals are available in the BIG Data Center Genome Sequence Archive (<a id="intref0010" href="http://bigd.big.ac.cn/gsa-human/" target="_blank">http://bigd.big.ac.cn/gsa-human/</a>)</td><td>This paper</td><td>PRJCA003699</td></tr><tr class="rowsep"><td>Mitochondrial DNA (fasta format) are available in the Genome Warehouse in the National Genomics Data Center (<a id="intref0015" href="https://bigd.big.ac.cn/gwh/" target="_blank">https://bigd.big.ac.cn/gwh/</a>)</td><td>This paper</td><td>PRJCA003699</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 2a of <span class="dropBlock reference-citations"><a id="body-ref-sref21-1" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-1" aria-controls="sref21-1">Haak et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref21-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (1139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0020" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM31_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM31_ESM.zip</a></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 2b of <span class="dropBlock reference-citations"><a id="body-ref-sref21-2" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-2" aria-controls="sref21-2">Haak et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref21-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (1139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0025" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM32_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM32_ESM.zip</a></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 2c of <span class="dropBlock reference-citations"><a id="body-ref-sref21-3" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-3" aria-controls="sref21-3">Haak et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref21-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (1139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM33_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM33_ESM.zip</a></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 2d of <span class="dropBlock reference-citations"><a id="body-ref-sref21-4" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-4" aria-controls="sref21-4">Haak et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref21-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (1139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM34_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14317/MediaObjects/41586_2015_BFnature14317_MOESM34_ESM.zip</a></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 1a of <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Fu et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14558/MediaObjects/41586_2015_BFnature14558_MOESM242_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14558/MediaObjects/41586_2015_BFnature14558_MOESM242_ESM.zip</a></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 1b of <span class="dropBlock reference-citations"><a id="body-ref-sref16-3" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-3" aria-controls="sref16-3">Fu et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref16-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14558/MediaObjects/41586_2015_BFnature14558_MOESM243_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14558/MediaObjects/41586_2015_BFnature14558_MOESM243_ESM.zip</a></td></tr><tr><td>Probe for 1240K Panel</td><td>Supplemental Data 1c of <span class="dropBlock reference-citations"><a id="body-ref-sref16-4" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-4" aria-controls="sref16-4">Fu et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref16-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnature14558/MediaObjects/41586_2015_BFnature14558_MOESM244_ESM.zip" target="_blank">https://static-content.springer.com/esm/art%3A10.1038%2Fnature14558/MediaObjects/41586_2015_BFnature14558_MOESM244_ESM.zip</a></td></tr><tr><td>Phosphate-AGATCGGAAG[C3Spacer]10[TEG-biotin]<br>(TEG&nbsp;= triethylene glycol spacer)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">Gansauge and Meyer, 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gansauge, M.-T. ∙ Meyer, M.</div><div><strong>Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:737-748</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84875500588" target="_blank" aria-label="Open Scopus in new tab">Scopus (357)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.038&amp;pmid=23493070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CL78 Single-stranded adaptor</td></tr><tr><td>GTGACTGGAGTTCAGACGT<br>GTGCTCTTCC<sup>∗</sup>GA<sup>∗</sup>TC<sup>∗</sup>T</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">Gansauge and Meyer, 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gansauge, M.-T. ∙ Meyer, M.</div><div><strong>Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:737-748</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84875500588" target="_blank" aria-label="Open Scopus in new tab">Scopus (357)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.038&amp;pmid=23493070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CL130<br>Extension primer</td></tr><tr><td>CGACGCTCTTC-ddC (ddC&nbsp;= dideoxycytidine)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-3" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-3" aria-controls="sref20-3">Gansauge and Meyer, 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gansauge, M.-T. ∙ Meyer, M.</div><div><strong>Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:737-748</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84875500588" target="_blank" aria-label="Open Scopus in new tab">Scopus (357)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.038&amp;pmid=23493070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CL53 Double-stranded adaptor,<br>strand 1</td></tr><tr><td>Phosphate-GGAAGAGCGTCGTG<br>TAGGGAAAGAG<sup>∗</sup>T<sup>∗</sup>G<sup>∗</sup>T<sup>∗</sup>A</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref20-4" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-4" aria-controls="sref20-4">Gansauge and Meyer, 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gansauge, M.-T. ∙ Meyer, M.</div><div><strong>Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:737-748</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84875500588" target="_blank" aria-label="Open Scopus in new tab">Scopus (357)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.038&amp;pmid=23493070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>CL73 Double-stranded adaptor,<br>strand 2</td></tr><tr><td>A<sup>∗</sup>C<sup>∗</sup>A<sup>∗</sup>C<sup>∗</sup>TCTTTCCCTACACGA<br>CGCTCTTCCGATCT<sup>∗</sup>G<sup>∗</sup>T<sup>∗</sup>C<sup>∗</sup>T</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina sequencing library preparation for highly multiplexed target capture and sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb-prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (1425)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>NI7_P5_CR2_short</td></tr><tr><td>G<sup>∗</sup>T<sup>∗</sup>G<sup>∗</sup>A<sup>∗</sup>CTGGAGTTCAGACGTG<br>TGCTCTTCCGATCT<sup>∗</sup>G<sup>∗</sup>T<sup>∗</sup>C<sup>∗</sup>T</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina sequencing library preparation for highly multiplexed target capture and sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb-prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (1425)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>NI8_P7_CR2_short</td></tr><tr><td>A<sup>∗</sup>G<sup>∗</sup>A<sup>∗</sup>C<sup>∗</sup>AGATC<sup>∗</sup>G<sup>∗</sup>G<sup>∗</sup>A<sup>∗</sup>A</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref50-3" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-3" aria-controls="sref50-3">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref50-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina sequencing library preparation for highly multiplexed target capture and sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb-prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (1425)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>NI9_P5P7_CR2_comp</td></tr><tr><td>GTGACTGGAGTTCAGACG<br>TGTGCTCTTCCGATCT-Phosphate</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-1" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-1" aria-controls="sref13-1">Fu et&nbsp;al., 2013a</a><div class="dropBlock__holder" data-db-target-of="sref13-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-5" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-5" aria-controls="sref16-5">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>BO4.P7.part1.R</td></tr><tr><td>CAAGCAGAAGACGGCAT<br>ACGAGAT-Phosphate</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-2" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-2" aria-controls="sref13-2">Fu et&nbsp;al., 2013a</a><div class="dropBlock__holder" data-db-target-of="sref13-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-6" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-6" aria-controls="sref16-6">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>BO6.P7.part2.R</td></tr><tr><td>GTGTAGATCTCGGTGGTC<br>GCCGTATCATT-Phosphate</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-3" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-3" aria-controls="sref13-3">Fu et&nbsp;al., 2013a</a><div class="dropBlock__holder" data-db-target-of="sref13-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-7" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-7" aria-controls="sref16-7">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>BO8.P5.part1.R</td></tr><tr><td>AGATCGGAAGAGCGTC<br>GTGTAGGGAAAGAGTGT-Phosphate</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref13-4" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-4" aria-controls="sref13-4">Fu et&nbsp;al., 2013a</a><div class="dropBlock__holder" data-db-target-of="sref13-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-8" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-8" aria-controls="sref16-8">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>BO10.P5.part2.R</td></tr><tr class="rowsep"><td>GGAAGAGCGTCGTGTAGG<br>GAAAGAGTGT-Phosphate</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref92-4" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-4" aria-controls="sref92-4">Yang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td>BO11.P5.part2.R</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>leeHom</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Renaud et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Renaud, G. ∙ Stenzel, U. ∙ Kelso, J.</div><div><strong>leeHom: adaptor trimming and merging for Illumina sequencing reads</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:e141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84922394220" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25100869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku699&amp;pmid=25100869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_linkdIFrphUq4B" href="https://github.com/grenaud/leeHom/" target="_blank">https://github.com/grenaud/leeHom/</a>; RRID: <a id="interref195" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_002710" target="_blank">SCR_002710</a></td></tr><tr><td>BWA 0.6.1</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (31658)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_linklHKawh0gfM" href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net/</a>; RRID: <a id="interref200" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_010910" target="_blank">SCR_010910</a></td></tr><tr><td>ContamMix 1.0-10</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Fu et&nbsp;al., 2013b</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Fu, Q. ∙ Mittnik, A. ∙ Johnson, P.L.F. ...</div><div><strong>A revised timescale for human evolution based on ancient mitochondrial genomes</strong></div><div><em>Curr. Biol.</em> 2013; <strong>23</strong>:553-559</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2013.02.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2013.02.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84876151646" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23523248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2013.02.044&amp;pmid=23523248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="https://github.com/StanfordBioinformatics/DEFUNCT-env-modules/tree/master/contamMix/" target="_blank">https://github.com/StanfordBioinformatics/DEFUNCT-env-modules/tree/master/contamMix/</a></td></tr><tr><td>ANGSD 0.921</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Korneliussen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Korneliussen, T.S. ∙ Albrechtsen, A. ∙ Nielsen, R.</div><div><strong>ANGSD: Analysis of Next Generation Sequencing Data</strong></div><div><em>BMC Bioinformatics.</em> 2014; <strong>15</strong>:356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-014-0356-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84920729412" target="_blank" aria-label="Open Scopus in new tab">Scopus (1457)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-014-0356-4&amp;pmid=25420514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0080" href="http://popgen.dk/angsd/index.php/ANGSD/" target="_blank">http://popgen.dk/angsd/index.php/ANGSD/</a></td></tr><tr><td>READ</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Monroy Kuhn et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Monroy Kuhn, J.M. ∙ Jakobsson, M. ∙ Günther, T.</div><div><strong>Estimating genetic kin relationships in prehistoric populations</strong></div><div><em>PLoS ONE.</em> 2018; <strong>13</strong>:e0195491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0195491" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85045916532" target="_blank" aria-label="Open Scopus in new tab">Scopus (155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29684051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0195491&amp;pmid=29684051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="https://bitbucket.org/tguenther/read/src/master/" target="_blank">https://bitbucket.org/tguenther/read/src/master/</a></td></tr><tr><td>lcMLkin</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Lipatov et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Lipatov, M. ∙ Sanjeev, K. ∙ Patro, R. ...</div><div><strong>Maximum Likelihood Estimation of Biological Relatedness from Low Coverage Sequencing Data</strong></div><div><em>bioRxiv.</em> 2015; 023374</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LipatovK.SanjeevR.PatroK.R.VeeramahMaximum+Likelihood+Estimation+of+Biological+Relatedness+from+Low+Coverage+Sequencing+DatabioRxiv2015023374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0090" href="https://github.com/COMBINE-lab/maximum-likelihood-relatedness-estimation" target="_blank">https://github.com/COMBINE-lab/maximum-likelihood-relatedness-estimation</a></td></tr><tr><td>EIGENSOFT 6.1.4</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Patterson et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Patterson, N. ∙ Price, A.L. ∙ Reich, D.</div><div><strong>Population structure and eigenanalysis</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-33846006923" target="_blank" aria-label="Open Scopus in new tab">Scopus (3429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020190&amp;pmid=17194218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_linkFIYg0za8Xr" href="https://github.com/DReichLab/EIG/" target="_blank">https://github.com/DReichLab/EIG/</a>; RRID: <a id="interref205" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_004965" target="_blank">SCR_004965</a></td></tr><tr><td>ADMIXTURE 1.3.0</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Alexander et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (5179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_linklg2iSJ8tsH" href="http://dalexander.github.io/admixture/download.html" target="_blank">http://dalexander.github.io/admixture/download.html</a>; RRID: <a id="intref0105" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_001263" target="_blank">SCR_001263</a></td></tr><tr><td>PLINK v1.9</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref69-1" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-1" aria-controls="sref69-1">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref69-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (22449)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_linkXiNpeBkgjj" href="https://www.cog-genomics.org/plink/1.9" target="_blank">https://www.cog-genomics.org/plink/1.9</a>; RRID: <a id="interref210" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_001757" target="_blank">SCR_001757</a></td></tr><tr><td>ADMIXTOOLS (<i>qp3Pop, qpDstat, qpAdm, qpGraph</i>)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref60-4" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-4" aria-controls="sref60-4">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_linkWFW1D2yPDw" href="https://github.com/DReichLab/AdmixTools/" target="_blank">https://github.com/DReichLab/AdmixTools/</a>; RRID:<a id="interref215" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_018495" target="_blank">SCR_018495</a></td></tr><tr><td>Treemix 1.13</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Pickrell and Pritchard, 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Pickrell, J.K. ∙ Pritchard, J.K.</div><div><strong>Inference of population splits and&nbsp;mixtures from genome-wide allele frequency data</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1002967</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1002967" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869026948" target="_blank" aria-label="Open Scopus in new tab">Scopus (1556)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23166502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1002967&amp;pmid=23166502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0135" href="http://bitbucket.org/nygcresearch/treemix/wiki/Home/" target="_blank">http://bitbucket.org/nygcresearch/treemix/wiki/Home/</a></td></tr><tr><td>Phylip 3.697</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Baum, 1989</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Baum, B.R.</div><div><strong>PHYLIP: Phylogeny Inference Package. Version 3.2. Joel Felsenstein</strong></div><div><em>Q.&nbsp;Rev. Biol.</em> 1989; <strong>64</strong>:539-541</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1086/416571" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F416571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="PC_link6xpEEVQGEY" href="https://evolution.genetics.washington.edu/phylip.html" target="_blank">https://evolution.genetics.washington.edu/phylip.html</a>; RRID:<a id="intref0140" href="https://scicrunch.org/resources/Any/search?q=undefined&amp;l=SCR_006244" target="_blank">SCR_006244</a></td></tr><tr><td>admixfrog</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Peter, 2020</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Peter, B.M.</div><div><strong>100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.13.990523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098896511" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.13.990523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0150" href="https://github.com/benjaminpeter/admixfrog-sims/" target="_blank">https://github.com/benjaminpeter/admixfrog-sims/</a></td></tr><tr><td>hapROH</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Ringbauer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ringbauer, H. ∙ Novembre, J. ∙ Steinrücken, M.</div><div><strong>Human Parental Relatedness through Time - Detecting Runs of Homozygosity in Ancient DNA</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.05.31.126912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098902463" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.05.31.126912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0155" href="https://pypi.org/project/hapROH/" target="_blank">https://pypi.org/project/hapROH/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2821%2900575-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-3-2"><h3>Resource availability</h3><section id="sec-3-2-1"><h4>Lead contact</h4><div id="p0090" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Qiaomei Fu (<a id="intref0160" href="mailto:fuqiaomei@ivpp.ac.cn">fuqiaomei@ivpp.ac.cn</a>).</div></section><section id="sec-3-2-2"><h4>Materials availability</h4><div id="p0095" role="paragraph">This study did not generate new reagents.</div></section><section id="sec-3-2-3"><h4>Data and code availability</h4><div id="p0100" role="paragraph">Original data for the 25 newly sequenced individuals: Aligned reads (BAM format) and genotype calls (Eigenstrat format) of the nuclear DNA have been deposited in the BIG Data Center Genome Sequence Archive (<a id="intref0165" href="http://bigd.big.ac.cn/gsa-human/" target="_blank">http://bigd.big.ac.cn/gsa-human/</a>; accession number: <a id="intref0170" href="bioproject:/PRJCA003699" target="_blank">PRJCA003699</a>); Mitochondrial DNA (fasta format) have been deposited in the Genome Warehouse in the National Genomics Data Center (<a id="intref0175" href="https://bigd.big.ac.cn/gwh/" target="_blank">https://bigd.big.ac.cn/gwh/</a>; accession number: <a id="intref0180" href="bioproject:/PRJCA003699" target="_blank">PRJCA003699</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">CNCB-NGDC Members and Partners, 2021</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>CNCB-NGDC Members and Partners</div><div><strong>Database Resources of the National Genomics Data Center, China National Center for Bioinformation in 2021</strong></div><div><em>Nucleic Acids Res.</em> 2021; <strong>49</strong>:D18-D28</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa1022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85099427900" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33175170/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa1022&amp;pmid=33175170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Wang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Wang, Y. ∙ Song, F. ∙ Zhu, J. ...</div><div><strong>GSA: Genome Sequence Archive&lt;sup/&gt;</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2017; <strong>15</strong>:14-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2017.01.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85012868681" target="_blank" aria-label="Open Scopus in new tab">Scopus (441)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28387199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2017.01.001&amp;pmid=28387199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="sec-3-3"><h3>Experimental model and subject details</h3><section id="sec-3-3-1"><h4>Sites and specimen descriptions</h4><div id="p0105" role="paragraph">The Songnen Plain (45-48°N, 124-127°E) in southwest Heilongjiang Province, northeast China, includes the majority of the Amur region, located between the Daxing’an and Xiaoxing’an mountains, the Changbai Mountains and the Songliao watershed. The plain lies 120-300&nbsp;m above sea level with a relative relief of 30-100 m. The Nen and Songhua Rivers flow to the west and south, forming widely distributed floodplains. The central plain is relatively low and flat, with numerous wetlands and both large and small lakes and ponds. Sediments are mainly formed by alluvial deposits of the Songhua and Nen Rivers. The present surface of the Songnen Plain is gently undulating, thus it is also called the Wavy Plain.</div><div id="p0110" role="paragraph">Geologically, the Songnen Plain is a depression, part of the Songliao fault depression zone. The southwestern part of the depression is still sinking, while the northeast is rising. Tertiary and Quaternary sediments have risen to form platforms and uplifted hills. The upper part of the stratigraphic profile in this area of the Songnen Plain consists of loess-like loam, and the lower part is a gravel layer. Black soil (chernozëm) is widely distributed on the surface, rendering the plain a fertile agricultural zone today.</div><div id="p0115" role="paragraph">The Songnen Plain currently has a hemiboreal humid continental climate (Köppen climate classification Dwa and Dwb), with little rain in winter but is not exceptionally dry, since seasonal moisture from the Sea of Japan and the Yellow Sea reaches the area. The northwestern Songnen Plain is cold and dry because the prevailing northwest wind blows from the arid Eurasian continental interior due to the high pressure cell (the Siberian Anticyclone) established over Siberia. Summers are hot and rainy as water vapor from the sea is carried inland by subtropical high pressure effects, forming precipitation. The vegetation is mainly shrub meadowland or mesophytic weed meadowland.</div><div id="p0120" role="paragraph">Due to the impacts of climate change and human activities, groundwater levels have dropped substantially during the past few decades. Surface runoff intensely erodes soft sediments during the summer rainy season, and ravines dissect many areas of the plain. The Songnen Plain is not only a fertile land, but also one of the earliest cattle management/domestication centers in the world (<span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Zhang et al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Zhang, H. ∙ Chang, F. ∙ Li, H. ...</div><div><strong>OSL and AMS <sup>14</sup>C Age of the Most Complete Mammoth Fossil Skeleton from Northeastern China and its Paleoclimate Significance</strong></div><div><em>Radiocarbon.</em> 2019; <strong>61</strong>:347-358</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2018.48" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85061175283" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2018.48" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). It is also extremely rich in fossils of both the <i>Mammuthus-Coelodonta</i> Fauna (<span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Zhang et al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Zhang, H. ∙ Chang, F. ∙ Li, H. ...</div><div><strong>OSL and AMS <sup>14</sup>C Age of the Most Complete Mammoth Fossil Skeleton from Northeastern China and its Paleoclimate Significance</strong></div><div><em>Radiocarbon.</em> 2019; <strong>61</strong>:347-358</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2018.48" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85061175283" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2018.48" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and of human beings, especially in the drainage of the Songhua River near Harbin, the capital of China’s Heilongjiang Province.</div><div id="p0125" role="paragraph">The vegetational and climatic changes of the last glacial-Early Holocene transition have been reconstructed in detail for the Amur region using high-resolution pollen analyses from Lake Sihailongwan in Jilin Province (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Mingram et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Mingram, J. ∙ Stebich, M. ∙ Schettler, G. ...</div><div><strong>Millennial-scale East Asian monsoon variability of the last glacial deduced from annually laminated sediments from Lake Sihailongwan, NE China</strong></div><div><em>Quat. Sci. Rev.</em> 2018; <strong>201</strong>:57-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quascirev.2018.09.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85055038777" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quascirev.2018.09.023" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Stebich et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Stebich, M. ∙ Mingram, J. ∙ Han, J. ...</div><div><strong>Late Pleistocene spread&nbsp;of (cool-) temperate forests in Northeast China and climate changes synchronous with the North Atlantic region</strong></div><div><em>Global Planet. Change.</em> 2009; <strong>65</strong>:56-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gloplacha.2008.10.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-58849131825" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gloplacha.2008.10.010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Between 16,700 and 14,450 cal BP, the Amur region exhibited predominantly steppe and open taiga-like woodland communities, with abundant <i>Larix, Alnus fruticosa, Betula, Artemisia,</i> grasses and sedges, representing cold and dry conditions. During the Early Holocene, dense deciduous forests, consisting mainly of thermophilic broadleaf trees, became widespread, indicating climatic amelioration. Between 12,680-11,650 cal BP, a short-term climatic reversal to colder and/or dryer conditions has been recorded. A decrease in broadleaved trees and the reappearance of <i>Larix</i> and <i>Picea</i> signals this climatic reversal, comparable to the Younger Dryas event in the circum-Atlantic region. In general, strong atmospheric coupling between the North Atlantic region and East Asia is supported by the synchronicity of climate changes (<span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Stebich et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Stebich, M. ∙ Mingram, J. ∙ Han, J. ...</div><div><strong>Late Pleistocene spread&nbsp;of (cool-) temperate forests in Northeast China and climate changes synchronous with the North Atlantic region</strong></div><div><em>Global Planet. Change.</em> 2009; <strong>65</strong>:56-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gloplacha.2008.10.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-58849131825" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gloplacha.2008.10.010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0130" role="paragraph">Pottery appeared in the Amur region as early as 15 ka (e.g., at the Taoshan, Huayang and Houtaomuga sites) (<span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Wang and Sebillaud, 2019</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Wang, L. ∙ Sebillaud, P.</div><div><strong>The emergence of early pottery in East Asia: New discoveries and perspectives</strong></div><div><em>J.&nbsp;World Prehist.</em> 2019; <strong>32</strong>:73-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10963-018-9126-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060217290" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10963-018-9126-y" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96-2" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96-2" aria-controls="sref96-2">Yue et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref96-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Yue, J.-P. ∙ Li, Y.-Q. ∙ Yang, S.</div><div><strong>Neolithisation in the southern Lesser Khingan Mountains: lithic technologies and ecological adaptation</strong></div><div><em>Antiquity.</em> 2019; <strong>93</strong>:1144-1160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15184/aqy.2019.92" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85073781764" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15184%2Faqy.2019.92" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and is typologically similar to early ceramics discovered in the Russian Far East at archaeological sites including Goncharka-1, Novotroitskoe-10, Oshinovaya Rechika-10, Oshinovaya Rechika-16, Gasya, Khummi, and Gromatukha (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Sato and Natsuki, 2017</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Sato, H. ∙ Natsuki, D.</div><div><strong>Human behavioral responses to environmental condition and the emergence of the world’s oldest pottery in East and Northeast Asia: An overview</strong></div><div><em>Quat. Int.</em> 2017; <strong>441</strong>:12-28</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2016.12.046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85014160931" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2016.12.046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Wang and Sebillaud, 2019</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Wang, L. ∙ Sebillaud, P.</div><div><strong>The emergence of early pottery in East Asia: New discoveries and perspectives</strong></div><div><em>J.&nbsp;World Prehist.</em> 2019; <strong>32</strong>:73-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10963-018-9126-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060217290" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10963-018-9126-y" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Functionally, pottery provided a new means of food storage and cooking and improved the efficiency of food processing. Socially, pottery facilitated feasting behavior and the transport of consumables, establishing and strengthening social networks (<span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">Hayden and Villeneuve, 2011</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Hayden, B. ∙ Villeneuve, S.</div><div><strong>A century of feasting studies</strong></div><div><em>Annu. Rev. Anthropol.</em> 2011; <strong>40</strong>:433-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-anthro-081309-145740" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-80053404707" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-anthro-081309-145740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Pearson, 2005</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Pearson, R.</div><div><strong>The social context of early pottery in the Lingnan region of south China</strong></div><div><em>Antiquity.</em> 2005; <strong>79</strong>:819-828</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0003598X00114954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34347300985" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0003598X00114954" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the Amur region, isotope analyses of pottery found in the Houtaomuga (13-5 ka) (<span class="dropBlock reference-citations"><a id="body-ref-sref89-3" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-3" aria-controls="sref89-3">Wang and Sebillaud, 2019</a><div class="dropBlock__holder" data-db-target-of="sref89-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Wang, L. ∙ Sebillaud, P.</div><div><strong>The emergence of early pottery in East Asia: New discoveries and perspectives</strong></div><div><em>J.&nbsp;World Prehist.</em> 2019; <strong>32</strong>:73-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10963-018-9126-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060217290" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10963-018-9126-y" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Shuangta sites (11-7.8 ka) (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Kunikita et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Kunikita, D. ∙ Wang, L. ∙ Onuki, S. ...</div><div><strong>Radiocarbon dating and dietary reconstruction of the Early Neolithic Houtaomuga and Shuangta sites in the Song-Nen Plain, Northeast China</strong></div><div><em>Quat. Int.</em> 2017; <strong>441</strong>:62-68</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2017.01.031" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85014094153" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2017.01.031" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) indicate that fishing was an important part of the subsistence strategy of local populations. The earliest archaeological occurrence of nephrite has been identified at the Xiaonanshan site, dating to ca. 9 ka (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Jiamusi Cultural Relics Management Station and Raohe County Cultural Relics Management Institute, 1996</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Jiamusi Cultural Relics Management Station and Raohe County Cultural Relics Management Institute</div><div><strong>Neolithic burial in Xiaonanshan, Raohe County, Heilongjiang</strong></div><div><em>Kaogu.</em> 1996; <strong>1996</strong>:1-8</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Jiamusi+Cultural+Relics+Management+Station+and+Raohe+County+Cultural+Relics+Management+InstituteNeolithic+burial+in+Xiaonanshan%2C+Raohe+County%2C+HeilongjiangKaogu1996199618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Zhao et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Zhao, B. ∙ Sun, M. ∙ Du, Z.</div><div><strong>The Age and Characteristics of the Jade Unearthed from Xiaonanshan Tomb in Raohe County</strong></div><div><em>Bianjiang Kaogu Yanjiu.</em> 2013; <strong>2013</strong>:69-78</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.ZhaoM.SunZ.DuThe+Age+and+Characteristics+of+the+Jade+Unearthed+from+Xiaonanshan+Tomb+in+Raohe+CountyBianjiang+Kaogu+Yanjiu201320136978" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), also in the Amur region, signaling not only the exploitation of this new resource, but also, potentially, the establishment of trade networks, further enhancing nascent regional social connections (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Heilongjiang Provincial Institute of Cultural Relics and Archaeology Commission for Preservation of Ancient Monuments, Raohe County, 2019</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Heilongjiang Provincial Institute of Cultural Relics and Archaeology, Commission for Preservation of Ancient Monuments, Raohe County</div><div><strong>The excavation of Zone III at the Xiaonanshan Site in 2015, Raohe County, Heilongjiang</strong></div><div><em>Chinese Archaeology.</em> 2019; <strong>20</strong>:87-96</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Heilongjiang+Provincial+Institute+of+Cultural+Relics+and+ArchaeologyCommission+for+Preservation+of+Ancient+Monuments%2C+Raohe+CountyThe+excavation+of+Zone+III+at+the+Xiaonanshan+Site+in+2015%2C+Raohe+County%2C+HeilongjiangChinese+Archaeology2020198796" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0135" role="paragraph">In this study, 25 human remains, ranging from 33,590 to 3,420 cal BP (calibrated years before present, relative to 1950 CE) were selected for ancient DNA testing. All samples included in this study are under the custodianship of the co-authors and were accessed with full permission from the relevant archaeological institutes or universities. The review board of the Institute of Vertebrate Paleontology and Paleoanthropology approved the ancient genomes sampled in this project for study (Review No. 202006020007), following a protocol used in other ancient human genetic studies (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Ding et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Ding, M. ∙ Wang, T. ∙ Ko, A.M.-S. ...</div><div><strong>Ancient mitogenomes show plateau populations from last 5200 years partially contributed to present-day Tibetans</strong></div><div><em>Proc. Biol. Sci.</em> 2020; <strong>287</strong>:20192968</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32183622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=32183622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-6" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-6" aria-controls="sref93-6">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0140" role="paragraph">Samples were AMS radiocarbon (<sup>14</sup>C) dated at Peking University in Beijing. The chemical preparation and physical measurement of bone samples were similar to the standard processes described by <span class="dropBlock reference-citations"><a id="body-ref-sref97-3" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-3" aria-controls="sref97-3">Zhang et al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref97-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Zhang, H. ∙ Chang, F. ∙ Li, H. ...</div><div><strong>OSL and AMS <sup>14</sup>C Age of the Most Complete Mammoth Fossil Skeleton from Northeastern China and its Paleoclimate Significance</strong></div><div><em>Radiocarbon.</em> 2019; <strong>61</strong>:347-358</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2018.48" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85061175283" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2018.48" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. The resulting <sup>14</sup>C dates were calibrated using OxCal v4.4 (<span class="dropBlock reference-citations"><a id="body-ref-sref72-3" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-3" aria-controls="sref72-3">Ramsey and Lee, 2013</a><div class="dropBlock__holder" data-db-target-of="sref72-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Ramsey, C.B. ∙ Lee, S.</div><div><strong>Recent and planned developments of the program OxCal</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:720-730</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0033822200057878" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0033822200057878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the IntCal20 calibration curve (<span class="dropBlock reference-citations"><a id="body-ref-sref74-3" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-3" aria-controls="sref74-3">Reimer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref74-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Reimer, P.J. ∙ Austin, W.E.N. ∙ Bard, E. ...</div><div><strong>The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP)</strong></div><div><em>Radiocarbon.</em> 2020; <strong>62</strong>:725-757</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2020.41" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85092423467" target="_blank" aria-label="Open Scopus in new tab">Scopus (3774)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2020.41" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The origin sites of these samples are presented in <a id="crosref0995" href="#tbl1">Table 1</a>. Among them, specimens AR33K, AR9.2K_o, AR9.2K_deleted and AR8.3K were found on the southern bank of the Songhua River; the others were collected at Zhaodong, on the northern bank of the Songhua River. All were found at construction sites, or in eroded gullies and small valleys, especially during summer rainy seasons. Considering the ages and places of recovery of these specimens, we can see that 1) the oldest came from the Acheng District and east of Harbin on the south side of the Songhua River, where the elevation is relatively higher than surrounding areas; 2) most specimens came from the Zhaodong area on the northern Songhua River, especially the high plateau areas which are also very rich in non-human vertebrate fossils.</div></section></section><section id="sec-3-4"><h3>Method details</h3><section id="sec-3-4-1"><h4>Ancient DNA extraction</h4><div id="p0145" role="paragraph">All human remains were processed in dedicated laboratories at the Chinese Academy of Sciences’ Institute of Vertebrate Paleontology and Paleoanthropology (IVPP) in Beijing and were subjected to DNA extraction, following established protocols, from less than 100&nbsp;mg of bone (<span class="dropBlock reference-citations"><a id="body-ref-sref92-5" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-5" aria-controls="sref92-5">Yang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref93-7" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" aria-label="Reference 2020" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-7" aria-controls="sref93-7">2020</a><div class="dropBlock__holder" data-db-target-of="sref93-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Single-stranded libraries (denoted as “SS”) (<span class="dropBlock reference-citations"><a id="body-ref-sref20-5" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-5" aria-controls="sref20-5">Gansauge and Meyer, 2013</a><div class="dropBlock__holder" data-db-target-of="sref20-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Gansauge, M.-T. ∙ Meyer, M.</div><div><strong>Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:737-748</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84875500588" target="_blank" aria-label="Open Scopus in new tab">Scopus (357)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.038&amp;pmid=23493070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Kircher et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Kircher, M. ∙ Sawyer, S. ∙ Meyer, M.</div><div><strong>Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:e3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr771" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84855295457" target="_blank" aria-label="Open Scopus in new tab">Scopus (732)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22021376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr771&amp;pmid=22021376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were prepared for all samples except two individuals (NE58 and NE57), which used a uracil-DNA glycosylase partially-treated double-stranded library protocol (denoted as “DS-half” in <a id="crosref1020" href="#tbl1">Table 1</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref50-4" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-4" aria-controls="sref50-4">Meyer and Kircher, 2010</a><div class="dropBlock__holder" data-db-target-of="sref50-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina sequencing library preparation for highly multiplexed target capture and sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb-prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (1425)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Rohland et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Rohland, N. ∙ Harney, E. ∙ Mallick, S. ...</div><div><strong>Partial uracil–DNA–glycosylase treatment for screening of ancient DNA</strong></div><div><em>Philos. Trans. R Soc. Lond. B Biol. Sci.</em> 2015; <strong>370</strong>:20130624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0624" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84916210143" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25487342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0624&amp;pmid=25487342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then amplified libraries for 35 cycles using the AccuPrime Pfx polymerase to obtain enough ancient DNA for capture. The P5 and P7 primers were added last to avoid library contamination, and the amount of DNA extracted per sample was evaluated using a Thermo Scientific NanoDrop 2000 spectrometer.</div></section><section id="sec-3-4-2"><h4>Ancient DNA capture and sequencing</h4><div id="p0150" role="paragraph">We applied a DNA capture technique to enrich endogenous ancient DNA from the high levels of background environmental DNA (<span class="dropBlock reference-citations"><a id="body-ref-sref13-5" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-5" aria-controls="sref13-5">Fu et&nbsp;al., 2013a</a><div class="dropBlock__holder" data-db-target-of="sref13-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-9" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-9" aria-controls="sref16-9">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref21-5" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-5" aria-controls="sref21-5">Haak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref21-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (1139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For the mitochondrial DNA (mtDNA), oligonucleotide probes synthesized from the complete human mitochondrial genome were used to capture human mtDNA (<span class="dropBlock reference-citations"><a id="body-ref-sref13-6" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13-6" aria-controls="sref13-6">Fu et&nbsp;al., 2013a</a><div class="dropBlock__holder" data-db-target-of="sref13-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); oligonucleotide probes that targeted 1.2 million SNPs from Panels 1 and 2 (<span class="dropBlock reference-citations"><a id="body-ref-sref16-10" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-10" aria-controls="sref16-10">Fu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref16-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were used to capture human nuclear DNA. After enrichment, the Illumina MiSeq sequencing platform was used to generate 2&nbsp;× 76bp paired-end reads for mtDNA, and the Illumina Hiseq 4000 sequencing platform was used to generate 2&nbsp;× 100bp and 2&nbsp;× 150bp paired-end reads for the nuclear DNA.</div></section></section><section id="sec-3-5"><h3>Quantification and statistical analysis</h3><section id="sec-3-5-1"><h4>Read alignment</h4><div id="p0155" role="paragraph">We trimmed adapters and merged paired-end reads into a single sequence (minimum overlap of 11 base pairs) using <i>leeHom</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref75-2" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-2" aria-controls="sref75-2">Renaud et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref75-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Renaud, G. ∙ Stenzel, U. ∙ Kelso, J.</div><div><strong>leeHom: adaptor trimming and merging for Illumina sequencing reads</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:e141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84922394220" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25100869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku699&amp;pmid=25100869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and only merged reads with a length of at least 30&nbsp;bp were used. For read alignment, <i>BWA</i> (version 0.6.1) (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (31658)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was applied using the <i>aln</i> and <i>samse</i> commands with the arguments -n 0.01 and -l 16500. The mtDNA reads were aligned to the revised Cambridge Reference Sequence (rCRS) (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Andrews et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Andrews, R.M. ∙ Kubacka, I. ∙ Chinnery, P.F. ...</div><div><strong>Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA</strong></div><div><em>Nat. Genet.</em> 1999; <strong>23</strong>:147</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/13779" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-0032868141" target="_blank" aria-label="Open Scopus in new tab">Scopus (2694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10508508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F13779&amp;pmid=10508508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); Nuclear DNA reads were aligned to the human reference genome <i>hg19</i>. Duplicate reads were identified (same orientation, start, and end positions) and removed. Reads with a minimum mapping quality score of 30 were kept for analysis.</div></section><section id="sec-3-5-2"><h4>Contamination evaluation and variants calling</h4><div id="p0160" role="paragraph">Two criteria were used to evaluate contamination: 1. We used <i>ContamMix</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Fu et&nbsp;al., 2013b</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Fu, Q. ∙ Mittnik, A. ∙ Johnson, P.L.F. ...</div><div><strong>A revised timescale for human evolution based on ancient mitochondrial genomes</strong></div><div><em>Curr. Biol.</em> 2013; <strong>23</strong>:553-559</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2013.02.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2013.02.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84876151646" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23523248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2013.02.044&amp;pmid=23523248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to estimate contamination rate by comparing mtDNA fragments with the consensus mitochondrial genome for our newly sampled individuals and 311 present-day world-wide sequences. To avoid counting damaged bases as contamination, the first and last five positions of the fragments were ignored during estimation. The libraries were treated as contaminated if over 3% of the fragments matched present-day sequences better than the consensus; 2. For males, we also used <i>ANGSD</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Korneliussen et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Korneliussen, T.S. ∙ Albrechtsen, A. ∙ Nielsen, R.</div><div><strong>ANGSD: Analysis of Next Generation Sequencing Data</strong></div><div><em>BMC Bioinformatics.</em> 2014; <strong>15</strong>:356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-014-0356-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84920729412" target="_blank" aria-label="Open Scopus in new tab">Scopus (1457)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-014-0356-4&amp;pmid=25420514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to estimate contamination rate, based on the fact that one copy of the X chromosome is found in males. The libraries were treated as contaminated if the estimated contamination rate was greater than 3%. During the variant calling, the first and last five positions of the fragments were ignored. For each SNP covered at least twice in an individual, we randomly sampled one sequence to determine a single allele for that individual in order to obtain pseudo-haploid genotypes. Afterward, individuals with a number of SNPs lower than 25,000 were removed. A total of 21 out of 25 individuals were retained for further analyses.</div></section><section id="sec-3-5-3"><h4>Kinship analyses</h4><div id="p0165" role="paragraph">We used <i>READ</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Monroy Kuhn et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Monroy Kuhn, J.M. ∙ Jakobsson, M. ∙ Günther, T.</div><div><strong>Estimating genetic kin relationships in prehistoric populations</strong></div><div><em>PLoS ONE.</em> 2018; <strong>13</strong>:e0195491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0195491" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85045916532" target="_blank" aria-label="Open Scopus in new tab">Scopus (155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29684051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0195491&amp;pmid=29684051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to estimate the degrees of kinship among our 21 newly sampled individuals from the Amur region. This tool carried out kinship analyses in three steps: 1. The genome was divided into non-overlapping windows of 1 mega base pairs (Mb). For each 1 Mb window, the proportion of non-matching alleles (P0) for each pair of individuals was estimated; 2. The average P0 was normalized by the median of all average pairwise P0 across all unrelated individuals. This normalization could reduce the effect of SNP ascertainment, population diversity, and potential batch effects; 3. The degrees of kinship were categorized (unrelated, first degree, second degree, or identical individuals/identical twins) using predefined cut-offs. All unrelated individuals were kept for subsequent analyses. For relatives (identical twins, first-degree and second-degree kin), only the one with the higher SNP number was kept for subsequent analyses. Following this analysis, we removed one individual with second-degree relationship with another, keeping the individual having the higher number of SNPs.</div><div id="p0170" role="paragraph">To further validate the kinship results from <i>READ</i>, we estimated the degree of kinship using <i>lcMLkin</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Lipatov et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Lipatov, M. ∙ Sanjeev, K. ∙ Patro, R. ...</div><div><strong>Maximum Likelihood Estimation of Biological Relatedness from Low Coverage Sequencing Data</strong></div><div><em>bioRxiv.</em> 2015; 023374</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LipatovK.SanjeevR.PatroK.R.VeeramahMaximum+Likelihood+Estimation+of+Biological+Relatedness+from+Low+Coverage+Sequencing+DatabioRxiv2015023374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which infers kinship using genotype likelihoods. The method takes into account the uncertainty in genotype calling when sequence coverage is low, which is especially suitable for ancient DNA samples. <i>lcMLkin</i> estimates k<sub>0</sub>, k<sub>1</sub>, k<sub>2</sub> as the probability that no, one or two alleles are shared by identical by descent (IBD) between the pair of individuals, respectively. The coefficient of relatedness (r) can then be calculated as r&nbsp;= k1/2&nbsp;+ k2 to infer the kinship category.</div></section><section id="sec-3-5-4"><h4>Principal components analysis (PCA)</h4><div id="p0175" role="paragraph">Principal components were calculated with 64 present-day populations from the Human Origin (HO) SNP Panel (<a id="crosref1095" href="#mmc2">Table S1</a>), together with 16 present-day Han and Tibetan populations from <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Lu et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Lu, D. ∙ Lou, H. ∙ Yuan, K. ...</div><div><strong>Ancestral Origins and Genetic History of Tibetan Highlanders</strong></div><div><em>Am. J. Hum. Genet.</em> 2016; <strong>99</strong>:580-594</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85002757551" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27569548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2016.07.002&amp;pmid=27569548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, utilizing the <i>smartpca</i> program in the <i>EIGENSOFT</i> package (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Patterson et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Patterson, N. ∙ Price, A.L. ∙ Reich, D.</div><div><strong>Population structure and eigenanalysis</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-33846006923" target="_blank" aria-label="Open Scopus in new tab">Scopus (3429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020190&amp;pmid=17194218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The parameters we adopted were default settings except for lsqproject: YES, numoutlieriter: 0, and shrinkmode: YES. Our newly sequenced ancient individuals (<a id="crosref1110" href="#tbl1">Table 1</a>) and the previously reported ancient Asians (Table S1) were then projected onto the principal components calculated using present-day populations.</div></section><section id="sec-3-5-5"><h4><i>ADMIXTURE</i> analysis</h4><div id="p0180" role="paragraph">Individual ancestries were estimated using a model-based maximum likelihood clustering algorithm with <i>ADMIXTURE</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref1-3" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-3" aria-controls="sref1-3">Alexander et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref1-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (5179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The populations included in the analysis were the same as those in the PCA, except that Burmese was omitted and several present-day Russian populations were added (<a id="crosref1120" href="#mmc2">Table S1</a>). Prior to analysis, the genotypes for these populations were pruned for high linkage disequilibrium (r2&nbsp;&gt; 0.4) using <i>PLINK</i> (version v1.90) (<span class="dropBlock reference-citations"><a id="body-ref-sref69-2" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69-2" aria-controls="sref69-2">Purcell et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref69-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (22449)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with parameters “–indep-pairwise 200 25 0.4.” This left 597,573 SNPs. The <i>ADMIXTURE</i> analysis was then carried out with K from 2 to 10. For each K, the analyses were run 100 times with different seeds to estimate the cross-validation (CV) error. The best K was determined by the lowest CV error.</div></section><section id="sec-3-5-6"><h4>Outgroup-<i>f</i><sub><i>3</i></sub> and <i>D</i> statistics</h4><div id="p0185" role="paragraph">We assessed the amount of shared genetic similarity between two individuals relative to an African outgroup population (Mbuti) using an outgroup-<i>f</i><sub><i>3</i></sub> analysis (<span class="dropBlock reference-citations"><a id="body-ref-sref60-5" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-5" aria-controls="sref60-5">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In this analysis, higher <i>f</i><sub><i>3</i></sub> values indicate high genetic similarity. The pairwise outgroup-<i>f</i><sub><i>3</i></sub> was performed for our newly sampled individuals, and several published ancient and present-day Asians (<a id="crosref1135" href="#mmc2">Table S1</a>), with <i>qp3Pop</i> (version 435) software in the <i>AdmixTools</i> package (<span class="dropBlock reference-citations"><a id="body-ref-sref60-6" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-6" aria-controls="sref60-6">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The pairwise <i>f</i><sub><i>3</i></sub> results were then presented as a heatmap using R package <i>gplots</i> (<a id="intref0185" href="https://github.com/talgalili/gplots" target="_blank">https://github.com/talgalili/gplots</a>).</div><div id="p0190" role="paragraph">We also assessed the genetic affinities among three individuals using <i>D</i> statistics (<span class="dropBlock reference-citations"><a id="body-ref-sref60-7" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-7" aria-controls="sref60-7">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), in the form of <i>D</i>(P1, P2; P3, Outgroup), where Outgroup is a population that is an outgroup to populations P1, P2, and P3. The <i>D</i> statistics were carried out for our newly sampled individuals, and several published ancient and present-day Asians, with <i>qpDstat</i> (version 755) software in the <i>AdmixTools</i> package (<span class="dropBlock reference-citations"><a id="body-ref-sref60-8" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-8" aria-controls="sref60-8">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-3-5-7"><h4>Phylogeny modeling with <i>Treemix</i></h4><div id="p0195" role="paragraph">We estimated historical relationships among populations, allowing both population splits and migration events using maximum-likelihood based software <i>Treemix</i> v1.13 (<span class="dropBlock reference-citations"><a id="body-ref-sref64-3" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-3" aria-controls="sref64-3">Pickrell and Pritchard, 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Pickrell, J.K. ∙ Pritchard, J.K.</div><div><strong>Inference of population splits and&nbsp;mixtures from genome-wide allele frequency data</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1002967</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1002967" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869026948" target="_blank" aria-label="Open Scopus in new tab">Scopus (1556)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23166502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1002967&amp;pmid=23166502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The samples included in the <i>Treemix</i> analysis were: AR33K, AR19K, AR14K, ARpost14K, AR13-10K, and ARpost9K (this study) and previously published populations (Mbuti, Ust’-Ishim, Yana, Tianyuan, Ikawazu, DevilsCave_N, Shamanka_EN, Yumin, Mongolia_N_East, Early Neolithic coastal northern East Asians (coastal_nEastAsia_EN) (including Bianbian, Boshan, Xiaogao, and Xiaojingshan), Early Neolithic coastal southern East Asians (sEastAsia_EN) (including Qihe, Liangdao1, and Liangdao2), Late Neolithic coastal southern East Asians (sEastAsia_LN) (including Suogang, Xitoucun, and Tanshishan), Kolyma, UKY, and West_Siberia_N; <a id="crosref1160" href="#mmc2">Table S1</a>). The group names coastal_nEastAsia_EN, sEastAsia_EN, and sEastAsia_LN were previously defined in <span class="dropBlock reference-citations"><a id="body-ref-sref93-8" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-8" aria-controls="sref93-8">Yang et&nbsp;al. (2020)</a><div class="dropBlock__holder" data-db-target-of="sref93-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. We set Mbuti as the root with (-root Mbuti) and accounted for linkage disequilibrium by grouping sites in blocks of 500 SNPs (-k 500). We allowed for up to five migration events (-m 1-7) and ran 1,000 bootstraps for each tree (-bootstrap –q). Those 1,000 bootstrap trees were then assessed in <i>phylip</i> using the <i>consense</i> command (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Baum, 1989</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Baum, B.R.</div><div><strong>PHYLIP: Phylogeny Inference Package. Version 3.2. Joel Felsenstein</strong></div><div><em>Q.&nbsp;Rev. Biol.</em> 1989; <strong>64</strong>:539-541</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1086/416571" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F416571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), to count the number of times certain individuals grouped, relative to all others in the analysis. The inferred maximum-likelihood based trees and corresponding residuals were visualized with the in-build R script from <i>Treemix</i> v1.13.</div></section><section id="sec-3-5-8"><h4>Admixture modeling with <i>qpAdm</i></h4><div id="p0200" role="paragraph">We applied <i>qpAdm</i> (version 634) in the <i>AdmixTools</i> package (<span class="dropBlock reference-citations"><a id="body-ref-sref60-9" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-9" aria-controls="sref60-9">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to model ancestry proportions of Ancient Paleo-Siberians (<span class="dropBlock reference-citations"><a id="body-ref-sref83-5" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-5" aria-controls="sref83-5">Sikora et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref83-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95-3" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-3" aria-controls="sref95-3">Yu et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref95-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Yu, H. ∙ Spyrou, M.A. ∙ Karapetian, M. ...</div><div><strong>Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1232-1245.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085755497" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32437661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.037&amp;pmid=32437661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) related to one, two, or three different sources. <i>qpAdm</i> models ancestry of target populations using a set of source populations (left populations) and a set of reference populations (right populations), without requiring the explicit description of the phylogenetic relationship among them. A “rotating” scheme of source and reference populations was used, being suggested as the best practice to search for the most optimal model (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Harney et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Harney, É. ∙ Patterson, N. ∙ Reich, D. ...</div><div><strong>Assessing the performance of qpAdm: a statistical tool for studying population admixture</strong></div><div><em>Genetics.</em> 2021; <strong>217</strong>:iyaa045</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/genetics/iyaa045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85104899633" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33772284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgenetics%2Fiyaa045&amp;pmid=33772284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Therefore, we rotated populations from our set of source populations to a base set of reference populations to keep only one, two, or three populations as source populations, along with parameters “allsnps: YES”, “details: YES”, and “summary: YES”.</div><div id="p0205" role="paragraph">Our source populations included: AR19K, AR14K, Bianbian, Qihe, Liangdao2, AfontovaGora3, Malta1, and USR1. The “rotating” scheme could compete for the optimal sources, e.g., AR19K versus AR14K, and AfontovaGora3 versus USR1.</div><div id="p0210" role="paragraph">Our base set of reference populations, which must differ in their relationship to the target populations, included: Mota, UstIshim, Kostenki14, Iran_N, IndusPeriphery, LBK_EN, Motala12, Kotias, AR33K, Yana, and Karelia.</div><div id="p0215" role="paragraph">IndusPeriphery is a merged set of three individuals from two archaeological sites: Gonur Depe (Gonur2_BA) (∼4 ka), in the Bactria-Margiana Archaeological Complex in Turkmenistan, and Shahr_I_Sokhta_BA2 and Shahr_I_Sokhta_BA3 (5-4 ka) from Shahr-i-Sokhta in Iran (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Narasimhan et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Narasimhan, V.M. ∙ Patterson, N. ∙ Moorjani, P. ...</div><div><strong>The formation of human populations in South and Central Asia</strong></div><div><em>Science.</em> 2019; <strong>365</strong>:eaat7487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat7487" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85071752467" target="_blank" aria-label="Open Scopus in new tab">Scopus (349)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31488661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat7487&amp;pmid=31488661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0220" role="paragraph">We only considered a model with more source populations when one with fewer sources was rejected. The criteria used to reject a model with fewer sources were: 1. Tail probability of rank0 for a fewer source model &gt; 0.05; 2. The estimated admixture proportions (±standard error) were between 0 and 1.</div><div id="p0225" role="paragraph">In addition, we applied <i>qpAdm</i> to model ancestry proportions of outlier population AR9.2K_o and merged outlier populations based on PCA and outgroup-<i>f</i><sub><i>3</i></sub> analyses (denoted as AR_o, including AR9.2K_o, AR3.4K_LowCov, and AR7.3K_LowCov) related to one, two, or three different sources.</div><div id="p0230" role="paragraph">Our source populations included: AR19K, AR14K, Bianbian, Xiaojingshan, Qihe, Liangdao2.</div><div id="p0235" role="paragraph">Our base set of reference populations was the same as before, and included: Mota, UstIshim, Kostenki14, Iran_N, IndusPeriphery, LBK_EN, Motala12, Kotias, AR33K, Yana, and Karelia.</div></section><section id="sec-3-5-9"><h4>Demographic modeling with <i>qpGraph</i></h4><div id="p0240" role="paragraph">We followed a general approach for modeling admixture graphs using <i>qpGraph</i> (version 6065) in the <i>AdmixTools</i> package (<span class="dropBlock reference-citations"><a id="body-ref-sref60-10" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-10" aria-controls="sref60-10">Patterson et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref60-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which started with a basic and well-understood tree (including the central African Mbuti as an outgroup, the early western Eurasian Kostenki14, ancient North Siberian Yana, and early Asian Tianyuan). We then added extra populations (AR33K, GoyetQ116-1, Liangdao2, AR19K, AR14K, Bianbian, DevilsCave_N, AfontovaGora3, USR1, UKY, and Kolyma) one at a time in their best-fitting positions iteratively (<span class="dropBlock reference-citations"><a id="body-ref-sref93-9" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-9" aria-controls="sref93-9">Yang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref93-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An optimum tree model could be constructed based on the observed <i>f</i>-statistics (<i>f</i><sub><i>2</i></sub>, <i>f</i><sub><i>3</i></sub>, and <i>f</i><sub><i>4</i></sub> for all possible pairs of populations). A |Z|-score of less than 3 between the observed and expected values (determined by the Block Jackknife) was required. A small number (0.0001) was added to the diagonal entries of the estimated covariance matrix of the f-statistics (<b>Q</b> matrix) to stabilizing the matrix inversion. The above and other parameters applied in the <i>qpGraph</i> program are as below, following recommendations of <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Lipson (2020)</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Lipson, M.</div><div><strong>Applying f<sub>4</sub> -statistics and admixture graphs: Theory and examples</strong></div><div><em>Mol. Ecol. Resour.</em> 2020; <strong>20</strong>:1658-1667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/1755-0998.13230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85089520090" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32717097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2F1755-0998.13230&amp;pmid=32717097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>:<div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="content"><div id="p0245" role="paragraph">outpop: Mbuti</div></div></div><div id="u0035" role="listitem"><div class="content"><div id="p0250" role="paragraph">blgsize: 0.05</div></div></div><div id="u0040" role="listitem"><div class="content"><div id="p0255" role="paragraph">lsqmode: YES</div></div></div><div id="u0045" role="listitem"><div class="content"><div id="p0260" role="paragraph">diag: 0.0001</div></div></div><div id="u0050" role="listitem"><div class="content"><div id="p0265" role="paragraph">hires: YES</div></div></div><div id="u0055" role="listitem"><div class="content"><div id="p0270" role="paragraph">initmix: 1000</div></div></div><div id="u0060" role="listitem"><div class="content"><div id="p0275" role="paragraph">precision: 0.0001</div></div></div><div id="u0065" role="listitem"><div class="content"><div id="p0280" role="paragraph">zthresh: 0</div></div></div><div id="u0070" role="listitem"><div class="content"><div id="p0285" role="paragraph">terse: NO</div></div></div><div id="u0075" role="listitem"><div class="content"><div id="p0290" role="paragraph">useallsnps: NO</div></div></div></div>We first added AR33K in the above basic tree. Due to strong evidence (from outgroup-<i>f</i><sub><i>3</i></sub>, <i>D</i> statistics, and <i>Treemix</i> analyses) that AR33K shares close genetic affinity with Tianyuan, we started with a model of AR33K forming a cluster with Tianyuan to reduce the searching space of initial modeling. Next, we added GoyetQ116-1, Liangdao2, AR19K, AR14K, Bianbian, DevilsCave_N, AfontovaGora3, USR1, UKY, and Kolyma iteratively, fitting as all possible nodes or admixture between nodes.</div></section><section id="sec-3-5-10"><h4>Archaic ancestry estimation with the Hidden Markov Model</h4><div id="p0295" role="paragraph">We inferred archaic (Neanderthal and Denisovan) introgressed segments in the genomes of our newly sampled individuals with SNP numbers over 330,000 (including AR33K, AR19K, AR14.5K, AR14.1K, AR10.6K, AR10.5K, AR8.9K, AR8.1K, AR7K, AR6.87K, and AR6.33K), Tianyuan (<span class="dropBlock reference-citations"><a id="body-ref-sref92-6" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-6" aria-controls="sref92-6">Yang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and two present-day Han Chinese from the Simons Genome Diversity Project (SGDP) (<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Mallick et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Mallick, S. ∙ Li, H. ∙ Lipson, M. ...</div><div><strong>The Simons Genome Diversity Project: 300 genomes from 142 diverse populations</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:201-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18964" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84991728057" target="_blank" aria-label="Open Scopus in new tab">Scopus (837)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18964&amp;pmid=27654912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (<a id="crosref1225" href="#mmc2">Table S1</a>), using <i>admixfrog</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref62-3" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-3" aria-controls="sref62-3">Peter, 2020</a><div class="dropBlock__holder" data-db-target-of="sref62-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Peter, B.M.</div><div><strong>100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.13.990523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098896511" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.13.990523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This software adopts a Hidden Markov Model for local ancestry inference, and allows accurate detection of introgressed segments in low-coverage ancient genomes even with present-day human DNA contamination (<span class="dropBlock reference-citations"><a id="body-ref-sref62-4" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-4" aria-controls="sref62-4">Peter, 2020</a><div class="dropBlock__holder" data-db-target-of="sref62-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Peter, B.M.</div><div><strong>100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.13.990523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098896511" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.13.990523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We modeled the target individuals deriving their ancestries from three sources including two high-coverage Neanderthal genomes (NEA) (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Prüfer et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Prüfer, K. ∙ Racimo, F. ∙ Patterson, N. ...</div><div><strong>The complete genome sequence of a Neanderthal from the Altai Mountains</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:43-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12886" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84892365780" target="_blank" aria-label="Open Scopus in new tab">Scopus (1399)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24352235/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12886&amp;pmid=24352235" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">2017</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Prüfer, K. ∙ de Filippo, C. ∙ Grote, S. ...</div><div><strong>A high-coverage Neandertal genome from Vindija Cave in Croatia</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:655-658</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aao1887" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85030853901" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28982794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aao1887&amp;pmid=28982794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), one high-coverage Denisovan genome (<i>Denisova 3</i>) (DEN) (<span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Meyer et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Meyer, M. ∙ Kircher, M. ∙ Gansauge, M.-T. ...</div><div><strong>A high-coverage genome sequence from an archaic Denisovan individual</strong></div><div><em>Science.</em> 2012; <strong>338</strong>:222-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1224344" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84867331947" target="_blank" aria-label="Open Scopus in new tab">Scopus (1257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22936568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1224344&amp;pmid=22936568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and 44 genomes of present-day Sub-Saharan Africans from SGDP (AFR) (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Mallick et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Mallick, S. ∙ Li, H. ∙ Lipson, M. ...</div><div><strong>The Simons Genome Diversity Project: 300 genomes from 142 diverse populations</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:201-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18964" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84991728057" target="_blank" aria-label="Open Scopus in new tab">Scopus (837)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18964&amp;pmid=27654912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0300" role="paragraph">The modeling included two steps:<div id="olist0010" role="list"><div id="o0010" role="listitem"><div class="label">1</div><div class="content"><div id="p0305" role="paragraph">The filtered BAM files for each of the target individual were converted into the input files for <i>admixfrog.</i> The command run is:</div></div></div></div><div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="admixfrog minus bam minus bamfile StartSet individual EndSet period bam minus length minus bin minus size 35 minus minmapq 25 minus deam minus cutoff 3 minus out StartSet individual EndSet period in period xz minus ref StartSet ref bar archaicadmixture period csv period xz EndSet" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="(63 (38 0 1 2 3) 4 (8 5 6 7) 9 (43 10 11 12 13 14) 15 16 17 (46 18 19 20) 21 22 23 (27 24 25 26) 28 29 30 (34 31 32 33))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="63" data-semantic-children="38,4,8,9,43,15,16,17,46,21,22,23,27,28,29,30,34" data-semantic-collapsed="(63 (c 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62) 38 4 8 9 43 15 16 17 46 21 22 23 27 28 29 30 34)" data-semantic-owns="38 4 8 9 43 15 16 17 46 21 22 23 27 28 29 30 34" data-semantic-speech="admixfrog minus bam minus bamfile StartSet individual EndSet period bam minus length minus bin minus size 35 minus minmapq 25 minus deam minus cutoff 3 minus out StartSet individual EndSet period in period xz minus ref StartSet ref bar archaicadmixture period csv period xz EndSet"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="38" data-semantic-children="0,1,2,3" data-semantic-parent="63" data-semantic-collapsed="(38 (c 35 36 37) 0 1 2 3)" data-semantic-owns="0 1 2 3"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="38"><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.705em;">d</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.705em;">x</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.705em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.705em;">g</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="38"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="2" data-semantic-parent="38"><mjx-c class="mjx-c62" noic="true" style="padding-top: 0.694em;">b</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6D" style="padding-top: 0.694em;">m</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="38"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="4" data-semantic-parent="63"><mjx-c class="mjx-c62" noic="true" style="padding-top: 0.705em;">b</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.705em;">l</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.705em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="8" data-semantic-children="6" data-semantic-content="5,7" data-semantic-parent="63" data-semantic-owns="5 6 7"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="5" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c76" noic="true" style="padding-top: 0.694em;">v</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6C" style="padding-top: 0.694em;">l</mjx-c></mjx-mtext><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow><mjx-mtext space="2" data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-parent="63"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.694em;">.</mjx-c><mjx-c class="mjx-c62" noic="true" style="padding-top: 0.694em;">b</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6D" style="padding-top: 0.694em;">m</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="43" data-semantic-children="10,11,12,13,14" data-semantic-parent="63" data-semantic-collapsed="(43 (c 39 40 41 42) 10 11 12 13 14)" data-semantic-owns="10 11 12 13 14"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="10" data-semantic-parent="43"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-parent="43"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c67" noic="true" style="padding-top: 0.694em;">g</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c68" style="padding-top: 0.694em;">h</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="12" data-semantic-parent="43"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="13" data-semantic-parent="43"><mjx-c class="mjx-c62" noic="true" style="padding-top: 0.694em;">b</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.694em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-parent="43"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-parent="63"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.666em;">s</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.666em;">i</mjx-c><mjx-c class="mjx-c7A" noic="true" style="padding-top: 0.666em;">z</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.666em;">e</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.666em;">5</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.666em;">&nbsp;</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="16" data-semantic-parent="63"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-parent="63"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.666em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.666em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.666em;">n</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.666em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.666em;">a</mjx-c><mjx-c class="mjx-c70" noic="true" style="padding-top: 0.666em;">p</mjx-c><mjx-c class="mjx-c71" noic="true" style="padding-top: 0.666em;">q</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c35" style="padding-top: 0.666em;">5</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="46" data-semantic-children="18,19,20" data-semantic-parent="63" data-semantic-collapsed="(46 (c 44 45) 18 19 20)" data-semantic-owns="18 19 20"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="18" data-semantic-parent="46"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="19" data-semantic-parent="46"><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6D" style="padding-top: 0.694em;">m</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="20" data-semantic-parent="46"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="21" data-semantic-parent="63"><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.705em;">u</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.705em;">t</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.705em;">o</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.705em;">&nbsp;</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.705em;">3</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-parent="63"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="23" data-semantic-parent="63"><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.615em;">o</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.615em;">u</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.615em;">t</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.615em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="27" data-semantic-children="25" data-semantic-content="24,26" data-semantic-parent="63" data-semantic-owns="24 25 26"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="24" data-semantic-parent="27" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="25" data-semantic-parent="27"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c76" noic="true" style="padding-top: 0.694em;">v</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6C" style="padding-top: 0.694em;">l</mjx-c></mjx-mtext><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="26" data-semantic-parent="27" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow><mjx-mtext space="2" data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="28" data-semantic-parent="63"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.657em;">.</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.657em;">.</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.657em;">x</mjx-c><mjx-c class="mjx-c7A" style="padding-top: 0.657em;">z</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="29" data-semantic-parent="63"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="30" data-semantic-parent="63"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.705em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="34" data-semantic-children="32" data-semantic-content="31,33" data-semantic-parent="63" data-semantic-owns="31 32 33"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="31" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="32" data-semantic-parent="34"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c5F" noic="true" style="padding-top: 0.705em;">_</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c68" noic="true" style="padding-top: 0.705em;">h</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.705em;">d</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.705em;">x</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.705em;">t</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.705em;">u</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.705em;">.</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.705em;">s</mjx-c><mjx-c class="mjx-c76" noic="true" style="padding-top: 0.705em;">v</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.705em;">.</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.705em;">x</mjx-c><mjx-c class="mjx-c7A" style="padding-top: 0.705em;">z</mjx-c></mjx-mtext><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="33" data-semantic-parent="34" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>In the above command, for each target individual, the BAM files were filtered to only include fragments of at least 35 base pairs (bp) and with a mapping quality &gt;&nbsp;= 25 (–length-bin-size 35 –minmapq 25). Additionally, variant positions matching a C to T substitutions in the first three positions of the read, or a G to A substitutions at the end of a read were discarded (–deam-cutoff 3).<div id="olist0015" role="list"><div id="o0015" role="listitem"><div class="label">2</div><div class="content"><div id="p0310" role="paragraph">For inferring archaic introgressed segments using previously generated input files, the following command was run:</div></div></div></div><div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartLayout 1st Row  admixfrog minus infile StartSet individual EndSet period in period xz minus ref StartSet ref bar archaicadmixture period csv period xz EndSet minus out StartSet out bar individual EndSet minus states AFR NEA DEN minus cont minus id AFR minus ll minus tol 0.01 minus bin minus 2nd Row  size 5000 minus est negative upper F minus est minus tau minus freq minus upper F 3 minus freq minus contamination 3 minus e 0 0.01 minus ancestral PAN minus run minus penalty 0.1 minus max minus iter 250 negative n minus post minus replicates 200 minus filter minus pos 50 minus 3rd Row  filter minus map 0 EndLayout" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="(161 (62 (60 (34 0 1) 2 (6 3 4 5) 7 8 9 (35 (13 10 11 12) 14) 15 (36 (19 16 17 18) 20) 21 (39 22 23 24) 25 (42 26 27 28) 29 (45 30 31 32))) (151 (149 63 (115 (104 64 65) 66 (106 (105 67 68) 69) 70 71 72 73 74 75) 76 (118 77 78 79) 80 81 82 83 84 (121 85 86 87) 88 (123 (122 89 90) 91) 92 (128 (125 (124 93 94) 95) 96 97) 98 (131 99 100 101) 102 103)) (160 (158 (156 152 153) 154)))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtable data-semantic-type="multiline" data-semantic-role="unknown" data-semantic-id="161" data-semantic-children="62,151,160" data-semantic-owns="62 151 160" data-semantic-speech="StartLayout 1st Row  admixfrog minus infile StartSet individual EndSet period in period xz minus ref StartSet ref bar archaicadmixture period csv period xz EndSet minus out StartSet out bar individual EndSet minus states AFR NEA DEN minus cont minus id AFR minus ll minus tol 0.01 minus bin minus 2nd Row  size 5000 minus est negative upper F minus est minus tau minus freq minus upper F 3 minus freq minus contamination 3 minus e 0 0.01 minus ancestral PAN minus run minus penalty 0.1 minus max minus iter 250 negative n minus post minus replicates 200 minus filter minus pos 50 minus 3rd Row  filter minus map 0 EndLayout" justify="left" style="min-width: 74.624em;"><mjx-table><mjx-itable><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="62" data-semantic-children="60" data-semantic-parent="161" data-semantic-owns="60"><mjx-mtd style="text-align: left;"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="60" data-semantic-children="34,2,6,7,8,9,35,15,36,21,39,25,42,29,45" data-semantic-parent="62" data-semantic-collapsed="(60 (c 46 47 48 49 50 51 52 53 54 55 56 57 58 59) 34 2 6 7 8 9 35 15 36 21 39 25 42 29 45)" data-semantic-owns="34 2 6 7 8 9 35 15 36 21 39 25 42 29 45"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="34" data-semantic-children="0,1" data-semantic-parent="60" data-semantic-collapsed="(34 (c 33) 0 1)" data-semantic-owns="0 1"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-parent="34"><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.705em;">d</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.705em;">x</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.705em;">o</mjx-c><mjx-c class="mjx-c67" style="padding-top: 0.705em;">g</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="34"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="2" data-semantic-parent="60"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.705em;">n</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.705em;">l</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.705em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="6" data-semantic-children="4" data-semantic-content="3,5" data-semantic-parent="60" data-semantic-owns="3 4 5"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c76" noic="true" style="padding-top: 0.694em;">v</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6C" style="padding-top: 0.694em;">l</mjx-c></mjx-mtext><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="5" data-semantic-parent="6" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow><mjx-mtext space="2" data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-parent="60"><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.657em;">.</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.657em;">n</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.657em;">.</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.657em;">x</mjx-c><mjx-c class="mjx-c7A" style="padding-top: 0.657em;">z</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-parent="60"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-parent="60"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.705em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="35" data-semantic-children="13" data-semantic-content="14" data-semantic-parent="60" data-semantic-owns="13 14"><mjx-mrow data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="13" data-semantic-children="11" data-semantic-content="10,12" data-semantic-parent="35" data-semantic-owns="10 11 12"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="10" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c5F" noic="true" style="padding-top: 0.705em;">_</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c68" noic="true" style="padding-top: 0.705em;">h</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.705em;">d</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.705em;">x</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.705em;">t</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.705em;">u</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.705em;">.</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.705em;">c</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.705em;">s</mjx-c><mjx-c class="mjx-c76" noic="true" style="padding-top: 0.705em;">v</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.705em;">.</mjx-c><mjx-c class="mjx-c78" noic="true" style="padding-top: 0.705em;">x</mjx-c><mjx-c class="mjx-c7A" style="padding-top: 0.705em;">z</mjx-c></mjx-mtext><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="12" data-semantic-parent="13" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="14" data-semantic-parent="35" data-semantic-operator="postfixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="15" data-semantic-parent="60"><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.615em;">o</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.615em;">u</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.615em;">t</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.615em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="36" data-semantic-children="19" data-semantic-content="20" data-semantic-parent="60" data-semantic-owns="19 20"><mjx-mrow data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="19" data-semantic-children="17" data-semantic-content="16,18" data-semantic-parent="36" data-semantic-owns="16 17 18"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="16" data-semantic-parent="19" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c5F" noic="true" style="padding-top: 0.694em;">_</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c76" noic="true" style="padding-top: 0.694em;">v</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.694em;">d</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6C" style="padding-top: 0.694em;">l</mjx-c></mjx-mtext><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="18" data-semantic-parent="19" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="20" data-semantic-parent="36" data-semantic-operator="postfixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="21" data-semantic-parent="60"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.716em;">s</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.716em;">t</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.716em;">a</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.716em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.716em;">e</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.716em;">s</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.716em;">&nbsp;</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c46" noic="true" style="padding-top: 0.716em;">F</mjx-c><mjx-c class="mjx-c52" noic="true" style="padding-top: 0.716em;">R</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.716em;">&nbsp;</mjx-c><mjx-c class="mjx-c4E" noic="true" style="padding-top: 0.716em;">N</mjx-c><mjx-c class="mjx-c45" noic="true" style="padding-top: 0.716em;">E</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.716em;">&nbsp;</mjx-c><mjx-c class="mjx-c44" noic="true" style="padding-top: 0.716em;">D</mjx-c><mjx-c class="mjx-c45" noic="true" style="padding-top: 0.716em;">E</mjx-c><mjx-c class="mjx-c4E" style="padding-top: 0.716em;">N</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="39" data-semantic-children="22,23,24" data-semantic-parent="60" data-semantic-collapsed="(39 (c 37 38) 22 23 24)" data-semantic-owns="22 23 24"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="22" data-semantic-parent="39"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="23" data-semantic-parent="39"><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.615em;">c</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.615em;">o</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.615em;">n</mjx-c><mjx-c class="mjx-c74" style="padding-top: 0.615em;">t</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="24" data-semantic-parent="39"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="25" data-semantic-parent="60"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.716em;">i</mjx-c><mjx-c class="mjx-c64" noic="true" style="padding-top: 0.716em;">d</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.716em;">&nbsp;</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c46" noic="true" style="padding-top: 0.716em;">F</mjx-c><mjx-c class="mjx-c52" style="padding-top: 0.716em;">R</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="42" data-semantic-children="26,27,28" data-semantic-parent="60" data-semantic-collapsed="(42 (c 40 41) 26 27 28)" data-semantic-owns="26 27 28"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-parent="42"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="27" data-semantic-parent="42"><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c6C" style="padding-top: 0.694em;">l</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="28" data-semantic-parent="42"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="29" data-semantic-parent="60"><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.694em;">&nbsp;</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.694em;">0</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.694em;">.</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.694em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.694em;">1</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="45" data-semantic-children="30,31,32" data-semantic-parent="60" data-semantic-collapsed="(45 (c 43 44) 30 31 32)" data-semantic-owns="30 31 32"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="30" data-semantic-parent="45"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="31" data-semantic-parent="45"><mjx-c class="mjx-c62" noic="true" style="padding-top: 0.694em;">b</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.694em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="32" data-semantic-parent="45"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="151" data-semantic-children="149" data-semantic-parent="161" data-semantic-owns="149"><mjx-mtd style="text-align: left;"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="149" data-semantic-children="63,115,76,118,80,81,82,83,84,121,88,123,92,128,98,131,102,103" data-semantic-parent="151" data-semantic-collapsed="(149 (c 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148) 63 115 76 118 80 81 82 83 84 121 88 123 92 128 98 131 102 103)" data-semantic-owns="63 115 76 118 80 81 82 83 84 121 88 123 92 128 98 131 102 103"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="63" data-semantic-parent="149"><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.657em;">s</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.657em;">i</mjx-c><mjx-c class="mjx-c7A" noic="true" style="padding-top: 0.657em;">z</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.657em;">e</mjx-c><mjx-c class="mjx-cA0" style="padding-top: 0.657em;">&nbsp;</mjx-c></mjx-mtext><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="115" data-semantic-children="104,66,106,70,71,72,73,74,75" data-semantic-parent="149" data-semantic-collapsed="(115 (c 107 108 109 110 111 112 113 114) 104 66 106 70 71 72 73 74 75)" data-semantic-owns="104 66 106 70 71 72 73 74 75"><mjx-mrow data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="104" data-semantic-children="64" data-semantic-content="65" data-semantic-parent="115" data-semantic-owns="64 65"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="64" data-semantic-parent="104"><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.666em;">5</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="65" data-semantic-parent="104" data-semantic-operator="postfixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="66" data-semantic-parent="115"><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.615em;">e</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.615em;">s</mjx-c><mjx-c class="mjx-c74" style="padding-top: 0.615em;">t</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="106" data-semantic-children="105" data-semantic-content="69" data-semantic-parent="115" data-semantic-owns="105 69"><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="105" data-semantic-children="68" data-semantic-content="67" data-semantic-parent="106" data-semantic-owns="67 68"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="67" data-semantic-parent="105" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="68" data-semantic-parent="105"><mjx-c class="mjx-c46">F</mjx-c></mjx-mtext></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="69" data-semantic-parent="106" data-semantic-operator="postfixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="70" data-semantic-parent="115"><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.615em;">e</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.615em;">s</mjx-c><mjx-c class="mjx-c74" style="padding-top: 0.615em;">t</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="71" data-semantic-parent="115"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="72" data-semantic-parent="115"><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.615em;">t</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.615em;">a</mjx-c><mjx-c class="mjx-c75" style="padding-top: 0.615em;">u</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="73" data-semantic-parent="115"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="74" data-semantic-parent="115"><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c71" style="padding-top: 0.705em;">q</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="75" data-semantic-parent="115"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="76" data-semantic-parent="149"><mjx-c class="mjx-c46" noic="true" style="padding-top: 0.68em;">F</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.68em;">&nbsp;</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.68em;">3</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="118" data-semantic-children="77,78,79" data-semantic-parent="149" data-semantic-collapsed="(118 (c 116 117) 77 78 79)" data-semantic-owns="77 78 79"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="77" data-semantic-parent="118"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="78" data-semantic-parent="118"><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.705em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c71" style="padding-top: 0.705em;">q</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="79" data-semantic-parent="118"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="80" data-semantic-parent="149"><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.666em;">c</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.666em;">o</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.666em;">n</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.666em;">t</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.666em;">a</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.666em;">m</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.666em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.666em;">n</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.666em;">a</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.666em;">t</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.666em;">i</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.666em;">o</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.666em;">n</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c33" style="padding-top: 0.666em;">3</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="81" data-semantic-parent="149"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="82" data-semantic-parent="149"><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.666em;">e</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.666em;">.</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.666em;">1</mjx-c></mjx-mtext><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="83" data-semantic-parent="149"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="84" data-semantic-parent="149"><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.716em;">a</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.716em;">n</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.716em;">c</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.716em;">e</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.716em;">s</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.716em;">t</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.716em;">r</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.716em;">a</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.716em;">l</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.716em;">&nbsp;</mjx-c><mjx-c class="mjx-c50" noic="true" style="padding-top: 0.716em;">P</mjx-c><mjx-c class="mjx-c41" noic="true" style="padding-top: 0.716em;">A</mjx-c><mjx-c class="mjx-c4E" style="padding-top: 0.716em;">N</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="121" data-semantic-children="85,86,87" data-semantic-parent="149" data-semantic-collapsed="(121 (c 119 120) 85 86 87)" data-semantic-owns="85 86 87"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="85" data-semantic-parent="121"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="86" data-semantic-parent="121"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.442em;">r</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.442em;">u</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.442em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="87" data-semantic-parent="121"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="88" data-semantic-parent="149"><mjx-c class="mjx-c70" noic="true" style="padding-top: 0.694em;">p</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c79" noic="true" style="padding-top: 0.694em;">y</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.694em;">&nbsp;</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.694em;">0</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.694em;">.</mjx-c><mjx-c class="mjx-c31" style="padding-top: 0.694em;">1</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="123" data-semantic-children="122" data-semantic-content="91" data-semantic-parent="149" data-semantic-owns="122 91"><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="122" data-semantic-children="90" data-semantic-content="89" data-semantic-parent="123" data-semantic-owns="89 90"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="89" data-semantic-parent="122" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="function" data-semantic-role="limit function" data-semantic-annotation="general:text" data-semantic-id="90" data-semantic-parent="122"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.448em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.448em;">a</mjx-c><mjx-c class="mjx-c78" style="padding-top: 0.448em;">x</mjx-c></mjx-mtext></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="91" data-semantic-parent="123" data-semantic-operator="postfixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="92" data-semantic-parent="149"><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.666em;">i</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.666em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.666em;">e</mjx-c><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.666em;">r</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.666em;">2</mjx-c><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.666em;">5</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="128" data-semantic-children="125,96,97" data-semantic-parent="149" data-semantic-collapsed="(128 (c 126 127) 125 96 97)" data-semantic-owns="125 96 97"><mjx-mrow data-semantic-added="true" data-semantic-type="postfixop" data-semantic-role="subtraction" data-semantic-id="125" data-semantic-children="124" data-semantic-content="95" data-semantic-parent="128" data-semantic-owns="124 95"><mjx-mrow data-semantic-added="true" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="124" data-semantic-children="94" data-semantic-content="93" data-semantic-parent="125" data-semantic-owns="93 94"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="93" data-semantic-parent="124" data-semantic-operator="prefixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="94" data-semantic-parent="124"><mjx-c class="mjx-c6E">n</mjx-c></mjx-mtext></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="95" data-semantic-parent="125" data-semantic-operator="postfixop,-"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="96" data-semantic-parent="128"><mjx-c class="mjx-c70" noic="true" style="padding-top: 0.615em;">p</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.615em;">o</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.615em;">s</mjx-c><mjx-c class="mjx-c74" style="padding-top: 0.615em;">t</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="97" data-semantic-parent="128"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="98" data-semantic-parent="149"><mjx-c class="mjx-c72" noic="true" style="padding-top: 0.694em;">r</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c70" noic="true" style="padding-top: 0.694em;">p</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c63" noic="true" style="padding-top: 0.694em;">c</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.694em;">a</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.694em;">e</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.694em;">s</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.694em;">&nbsp;</mjx-c><mjx-c class="mjx-c32" noic="true" style="padding-top: 0.694em;">2</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.694em;">0</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.694em;">0</mjx-c></mjx-mtext><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="131" data-semantic-children="99,100,101" data-semantic-parent="149" data-semantic-collapsed="(131 (c 129 130) 99 100 101)" data-semantic-owns="99 100 101"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="99" data-semantic-parent="131"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mtext space="3" data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="100" data-semantic-parent="131"><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.705em;">l</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.705em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c72" style="padding-top: 0.705em;">r</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="101" data-semantic-parent="131"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="102" data-semantic-parent="149"><mjx-c class="mjx-c70" noic="true" style="padding-top: 0.666em;">p</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.666em;">o</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.666em;">s</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c35" noic="true" style="padding-top: 0.666em;">5</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="103" data-semantic-parent="149"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr><mjx-mtr data-semantic-type="line" data-semantic-role="multiline" data-semantic-id="160" data-semantic-children="158" data-semantic-parent="161" data-semantic-owns="158"><mjx-mtd style="text-align: left;"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="158" data-semantic-children="156,154" data-semantic-parent="160" data-semantic-collapsed="(158 (c 157) 156 154)" data-semantic-owns="156 154"><mjx-mrow data-semantic-added="true" data-semantic-type="punctuated" data-semantic-role="text" data-semantic-id="156" data-semantic-children="152,153" data-semantic-parent="158" data-semantic-collapsed="(156 (c 155) 152 153)" data-semantic-owns="152 153"><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="152" data-semantic-parent="156"><mjx-c class="mjx-c66" noic="true" style="padding-top: 0.705em;">f</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.705em;">i</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.705em;">l</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.705em;">t</mjx-c><mjx-c class="mjx-c65" noic="true" style="padding-top: 0.705em;">e</mjx-c><mjx-c class="mjx-c72" style="padding-top: 0.705em;">r</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="153" data-semantic-parent="156"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo></mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="text" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="154" data-semantic-parent="158"><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.666em;">m</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.666em;">a</mjx-c><mjx-c class="mjx-c70" noic="true" style="padding-top: 0.666em;">p</mjx-c><mjx-c class="mjx-cA0" noic="true" style="padding-top: 0.666em;">&nbsp;</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.666em;">0</mjx-c></mjx-mtext></mjx-mrow><mjx-tstrut></mjx-tstrut></mjx-mtd></mjx-mtr></mjx-itable></mjx-table></mjx-mtable></mjx-mrow></mjx-math></mjx-container></div></div></div>In the above, we used the chimpanzee (panTro4, GCA_000001515.4) reference genome to infer the ancestral state of each allele (–ancestral PAN). The window size was set to 5,000 base pairs for all individuals (–bin-size 5000). The detailed information for other parameters was suggested and explained in <span class="dropBlock reference-citations"><a id="body-ref-sref62-5" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-5" aria-controls="sref62-5">Peter (2020)</a><div class="dropBlock__holder" data-db-target-of="sref62-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Peter, B.M.</div><div><strong>100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.13.990523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098896511" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.13.990523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section><section id="sec-3-5-11"><h4>Time transect of population sizes in the Amur region</h4><div id="p0315" role="paragraph">To investigate the time transect of population size changes, we identified the ROH for the populations in the Amur region using <i>hapROH</i> software (<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Ringbauer et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Ringbauer, H. ∙ Novembre, J. ∙ Steinrücken, M.</div><div><strong>Human Parental Relatedness through Time - Detecting Runs of Homozygosity in Ancient DNA</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.05.31.126912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098902463" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.05.31.126912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which is based on a Hidden Markov Model (HMM) with ROH and non-ROH states and utilizes a panel of 5,008 phased haplotypes of the 1000 Genomes Project dataset (Phase 3, release 20130502) (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (10306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The short ROH (4-8 cM) represent patterns of past background relatedness that could be used as proxies for local population sizes. To achieve reliable results, we analyzed 11 newly sequenced ancient individuals with SNP numbers over 330,000 (including AR33K, AR19K, AR14.5K, AR14.1K, AR10.6K, AR10.5K, AR8.9K, AR8.1K, AR7K, AR6.87K, and AR6.33K) and two modern Ulchi individuals (S_Ulchi-1 and S_Ulchi-2). The effective population size was directly estimated from observed ROH lengths, using the observed lengths of ROH blocks l&nbsp;= l<sub>1</sub>, ..., l<sub>n</sub> for a given set of individuals i<sub>1</sub>, ..., i<sub>k</sub> and a maximum likelihood inference scheme. The likelihood Pr(l | Ne) was calculated and then the population size was estimated that maximizes the product of the likelihoods. To assess the potential influence of effective population size on the kinship inference, we also merged AR9.9K_2d.rel.AR10.5K_deleted and AR10.5K and estimated their effective population size.</div></section><section id="sec-3-5-12"><h4>Time transect of <i>EDAR V370A</i> in northern East Asians</h4><div id="p0320" role="paragraph">To observe the time transect of <i>EDAR V370A</i> from 40 ka to present-day, we utilized both the genotype calls and read counts from 18 newly sampled ancient individuals with SNP numbers over 50,000 (<a id="crosref1275" href="#tbl1">Table 1</a>) and 12 published ancient northern East Asians from latitudes above 30 degrees North (including populations from Tianyuan Cave, Bianbian, Boshan, Xiaogao, Xiaojingshan, Yumin, and Devil’s Gate Cave). To obtain diploid genotype calls for <i>EDAR V370A</i>, we used an alternative variant calling scheme using <i>snpAD</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Prüfer, 2018</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Prüfer, K.</div><div><strong>snpAD: an ancient DNA genotype caller</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:4165-4171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85055567295" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29931305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty507&amp;pmid=29931305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>snpAD</i> jointly estimates error rates and genotype frequencies from ancient sequences. These estimates were then used to produce genotype calls. Read counts for <i>EDAR V370A</i> were obtained from BAM files. We grouped individuals based on time windows (40,000-35,000 cal BP, 35,000-30,000 cal BP, 20,000-15,000 cal BP, 15,000-13,000 cal BP, 13,000-10,000 cal BP, 10,000-9,000 cal BP, 9,000-8,000 cal BP, and 8,000-6,000 cal BP) to observe the time transect of <i>EDAR V370A</i>.</div></section></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0325" role="paragraph">We thank Zehui Chen for designing and editing images, Xueping Ji for morphological recognition of the sampled specimen and related discussions, Benjamin Peter for providing suggestions related to the analyses of archaic introgression, Janet Kelso for commenting on the manuscript, Xiaohong Wu for discussions of radiocarbon dating, and Jianping Yue and Youqian Li for additional archaeological background. We also thank our anonymous reviewers for their helpful comments. This research was supported by the Chinese Academy of Sciences (CAS) and the Ministry of Finance of the People’s Republic of China (XDB26000000), the National Natural Science Foundation of China (41820104008, 41925009, 91731303, 41672021, and 41630102), the National Key R&amp;D Program of China (2016YFE0203700), the CAS (XDA1905010 and QYZDB-SSW-DQC003), the Research on the Roots of Chinese Civilization program of Zhengzhou University (XKZDJC202006), the Tencent Foundation through its XPLORER Prize, and the Howard Hughes Medical Institute (55008731). J.W.O.’s participation was supported by the Chinese Academy of Sciences President’s International Fellowship Initiative (PIFI; award 2018VCA0016 with 2021 extension).</div><section id="sec-4"><h3>Author contributions</h3><div id="p0330" role="paragraph">Conceptualization, X.M., H.Z., and Q.F.; formal analysis, X.M. and Q.F.; investigation, S.Q., M.L., T.W., F.C., P.X., and C.L.; resources, H.Z., P.C., R.Y., F.L., Q.D., and X.F.; writing – original draft, X.M. and Q.F.; writing – review &amp; editing, X.M., Q.F., E.A.B., H.Z., S.Q., Y.L., M.Z., J.W.O., W.P., and M.L.; supervision, Q.F.</div></section><section id="sec-5"><h3>Declaration of interests</h3><div id="p0335" role="paragraph">The authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (4)</h2><div id="p0345" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.040/attachment/935980df-b7ef-40c6-bc0a-851b1df98db4/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (229.21 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Document S1. Tables S2, S4, and S6</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.040/attachment/9bc4721d-1202-4bf8-a44d-221eb61ea22b/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (59.47 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S1. Overview of newly sequenced and published individuals, related to Figures 1 and S1, Table 1, and STAR Methods</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.040/attachment/0b476ae8-400b-4ee1-9423-af68a2975c71/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (63.08 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. D statistics for populations, including Tianyuan, AR33K, Ancient Siberians, AR9.2K_o, Qihe, ancient Shandong, AR19K, and AR14K, related to Figures 1, 2, 3, and S1</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2021.04.040/attachment/9f8953ff-2768-4105-b267-0091dd0e481c/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (17.01 KB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S5. qpAdm analyses for Ancient Paleo-Siberians and AR9.2K_o, related to Figures 1, 3, and S1</div></div></div></div></section><section id="app-2"><h2>Supporting citations</h2><div id="p0350" role="paragraph">The following references appear in the supplemental information: <span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">de Barros Damgaard et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>de Barros Damgaard, P. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref15" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15" aria-controls="sref15">Fu et al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Fu, Q. ∙ Li, H. ∙ Moorjani, P. ...</div><div><strong>Genome sequence of a 45,000-year-old modern human from western Siberia</strong></div><div><em>Nature.</em> 2014; <strong>514</strong>:445-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13810" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84914688940" target="_blank" aria-label="Open Scopus in new tab">Scopus (659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25341783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13810&amp;pmid=25341783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Gamba et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Gamba, C. ∙ Jones, E.R. ∙ Teasdale, M.D. ...</div><div><strong>Genome flux and stasis in a five millennium transect of European prehistory</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:5257</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms6257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84923357868" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25334030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms6257&amp;pmid=25334030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Jeong et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Jeong, C. ∙ Ozga, A.T. ∙ Witonsky, D.B. ...</div><div><strong>Long-term genetic stability and a high-altitude East Asian origin for the peoples of the high valleys of the Himalayan arc</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:7485-7490</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1520844113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84977279989" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27325755/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1520844113&amp;pmid=27325755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Jeong et&nbsp;al. (2019)</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Jeong, C. ∙ Balanovsky, O. ∙ Lukianova, E. ...</div><div><strong>The genetic history of admixture across inner Eurasia</strong></div><div><em>Nat. Ecol. Evol.</em> 2019; <strong>3</strong>:966-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-019-0878-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85065175002" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31036896/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-019-0878-2&amp;pmid=31036896" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Jones et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Jones, E.R. ∙ Gonzalez-Fortes, G. ∙ Connell, S. ...</div><div><strong>Upper Palaeolithic genomes reveal deep roots of modern Eurasians</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:8912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms9912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84947563586" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26567969/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms9912&amp;pmid=26567969" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Lazaridis et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Lazaridis, I. ∙ Patterson, N. ∙ Mittnik, A. ...</div><div><strong>Ancient human genomes suggest three ancestral populations for present-day Europeans</strong></div><div><em>Nature.</em> 2014; <strong>513</strong>:409-413</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84907803675" target="_blank" aria-label="Open Scopus in new tab">Scopus (852)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25230663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13673&amp;pmid=25230663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">Lazaridis et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Lazaridis, I. ∙ Nadel, D. ∙ Rollefson, G. ...</div><div><strong>Genomic insights into the origin of farming in the ancient Near East</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:419-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19310" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84984999386" target="_blank" aria-label="Open Scopus in new tab">Scopus (561)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27459054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19310&amp;pmid=27459054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Lipson et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Lipson, M. ∙ Cheronet, O. ∙ Mallick, S. ...</div><div><strong>Ancient genomes document multiple waves of migration in Southeast Asian prehistory</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:92-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat3188" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85047268535" target="_blank" aria-label="Open Scopus in new tab">Scopus (209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29773666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat3188&amp;pmid=29773666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Llorente et&nbsp;al. (2015)</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Llorente, M.G. ∙ Jones, E.R. ∙ Eriksson, A. ...</div><div><strong>Ancient Ethiopian genome reveals extensive Eurasian admixture throughout the African continent</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:820-822</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad2879" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84946962576" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26449472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad2879&amp;pmid=26449472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">McColl et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>McColl, H. ∙ Racimo, F. ∙ Vinner, L. ...</div><div><strong>The prehistoric peopling of Southeast Asia</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:88-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat3628" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85049645767" target="_blank" aria-label="Open Scopus in new tab">Scopus (243)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29976827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat3628&amp;pmid=29976827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Moreno-Mayar et&nbsp;al. (2018)</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Moreno-Mayar, J.V. ∙ Vinner, L. ∙ de Barros Damgaard, P. ...</div><div><strong>Early human dispersals within the Americas</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaav2621</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aav2621" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85058604407" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30409807/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aav2621&amp;pmid=30409807" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Nakatsuka et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Nakatsuka, N. ∙ Moorjani, P. ∙ Rai, N. ...</div><div><strong>The promise of discovering population-specific disease-associated genes in South Asia</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85028717405" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28714977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3917&amp;pmid=28714977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref58" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58" aria-controls="sref58">Olalde et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref58"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Olalde, I. ∙ Allentoft, M.E. ∙ Sánchez-Quinto, F. ...</div><div><strong>Derived immune and ancestral pigmentation alleles in a 7,000-year-old Mesolithic European</strong></div><div><em>Nature.</em> 2014; <strong>507</strong>:225-228</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12960" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84896319183" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24463515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12960&amp;pmid=24463515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Qin and Stoneking (2015)</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Qin, P. ∙ Stoneking, M.</div><div><strong>Denisovan Ancestry in East Eurasian and Native American Populations</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:2665-2674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msv141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84940935331" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsv141&amp;pmid=26104010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Reich et&nbsp;al. (2011)</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Reich, D. ∙ Patterson, N. ∙ Kircher, M. ...</div><div><strong>Denisova admixture and the first modern human dispersals into Southeast Asia and Oceania</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>89</strong>:516-528</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2011.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2011.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-80053899093" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21944045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.09.005&amp;pmid=21944045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Seguin-Orlando et&nbsp;al. (2014)</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Seguin-Orlando, A. ∙ Korneliussen, T.S. ∙ Sikora, M. ...</div><div><strong>Paleogenomics. Genomic structure in Europeans dating back at least 36,200 years</strong></div><div><em>Science.</em> 2014; <strong>346</strong>:1113-1118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84914675035" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25378462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0114&amp;pmid=25378462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Sikora et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Sikora, M. ∙ Seguin-Orlando, A. ∙ Sousa, V.C. ...</div><div><strong>Ancient genomes show social and reproductive behavior of early Upper Paleolithic foragers</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:659-662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aao1807" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85030861669" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28982795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aao1807&amp;pmid=28982795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Skoglund et&nbsp;al. (2016)</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Skoglund, P. ∙ Posth, C. ∙ Sirak, K. ...</div><div><strong>Genomic insights into the peopling of the Southwest Pacific</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:510-513</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19844" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84993661810" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27698418/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19844&amp;pmid=27698418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Alexander, D.H. ∙ Novembre, J. ∙ Lange, K.</div><div><strong>Fast model-based estimation of ancestry in unrelated individuals</strong></div><div><em>Genome Res.</em> 2009; <strong>19</strong>:1655-1664</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.094052.109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-69749099417" target="_blank" aria-label="Open Scopus in new tab">Scopus (5179)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.094052.109&amp;pmid=19648217" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Andrews, R.M. ∙ Kubacka, I. ∙ Chinnery, P.F. ...</div><div><strong>Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA</strong></div><div><em>Nat. Genet.</em> 1999; <strong>23</strong>:147</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/13779" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-0032868141" target="_blank" aria-label="Open Scopus in new tab">Scopus (2694)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10508508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F13779&amp;pmid=10508508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (10306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Bacci, M.L.</div><div><strong>A concise history of world population</strong></div><div>John Wiley &amp; Sons, 2017</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/9781119406822" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F9781119406822" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bae, C.J. ∙ Douka, K. ∙ Petraglia, M.D.</div><div><strong>On the origin of modern humans: Asian perspectives</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:eaai9067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aai9067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85037608455" target="_blank" aria-label="Open Scopus in new tab">Scopus (228)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217544/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aai9067&amp;pmid=29217544" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Baum, B.R.</div><div><strong>PHYLIP: Phylogeny Inference Package. Version 3.2. Joel Felsenstein</strong></div><div><em>Q.&nbsp;Rev. Biol.</em> 1989; <strong>64</strong>:539-541</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1086/416571" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F416571" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Clark, P.U. ∙ Dyke, A.S. ∙ Shakun, J.D. ...</div><div><strong>The last glacial maximum</strong></div><div><em>Science.</em> 2009; <strong>325</strong>:710-714</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1172873" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-68449088262" target="_blank" aria-label="Open Scopus in new tab">Scopus (2613)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19661421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1172873&amp;pmid=19661421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>CNCB-NGDC Members and Partners</div><div><strong>Database Resources of the National Genomics Data Center, China National Center for Bioinformation in 2021</strong></div><div><em>Nucleic Acids Res.</em> 2021; <strong>49</strong>:D18-D28</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkaa1022" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85099427900" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33175170/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkaa1022&amp;pmid=33175170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>de Barros Damgaard, P. ∙ Martiniano, R. ∙ Kamm, J. ...</div><div><strong>The first horse herders and the impact of early Bronze Age steppe expansions into Asia</strong></div><div><em>Science.</em> 2018; <strong>360</strong>:eaar7711</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aar7711" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85047018554" target="_blank" aria-label="Open Scopus in new tab">Scopus (240)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aar7711&amp;pmid=29743352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Devièse, T. ∙ Massilani, D. ∙ Yi, S. ...</div><div><strong>Compound-specific radiocarbon dating and mitochondrial DNA analysis of the Pleistocene hominin from Salkhit Mongolia</strong></div><div><em>Nat. Commun.</em> 2019; <strong>10</strong>:274</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-08018-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060861342" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30700710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-08018-8&amp;pmid=30700710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Diamond, J. ∙ Bellwood, P.</div><div><strong>Farmers and their languages: the first expansions</strong></div><div><em>Science.</em> 2003; <strong>300</strong>:597-603</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1078208" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-0037466612" target="_blank" aria-label="Open Scopus in new tab">Scopus (638)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12714734/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1078208&amp;pmid=12714734" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Ding, M. ∙ Wang, T. ∙ Ko, A.M.-S. ...</div><div><strong>Ancient mitogenomes show plateau populations from last 5200 years partially contributed to present-day Tibetans</strong></div><div><em>Proc. Biol. Sci.</em> 2020; <strong>287</strong>:20192968</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32183622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=32183622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13-1" title="View in article">13.</a></div><div>Fu, Q. ∙ Meyer, M. ∙ Gao, X. ...</div><div><strong>DNA analysis of an early modern human from Tianyuan Cave, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:2223-2227</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1221359110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84873426880" target="_blank" aria-label="Open Scopus in new tab">Scopus (382)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23341637/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1221359110&amp;pmid=23341637" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Fu, Q. ∙ Mittnik, A. ∙ Johnson, P.L.F. ...</div><div><strong>A revised timescale for human evolution based on ancient mitochondrial genomes</strong></div><div><em>Curr. Biol.</em> 2013; <strong>23</strong>:553-559</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2013.02.044&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2013.02.044&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84876151646" target="_blank" aria-label="Open Scopus in new tab">Scopus (413)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23523248/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2013.02.044&amp;pmid=23523248" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15" title="View in article">15.</a></div><div>Fu, Q. ∙ Li, H. ∙ Moorjani, P. ...</div><div><strong>Genome sequence of a 45,000-year-old modern human from western Siberia</strong></div><div><em>Nature.</em> 2014; <strong>514</strong>:445-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13810" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84914688940" target="_blank" aria-label="Open Scopus in new tab">Scopus (659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25341783/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13810&amp;pmid=25341783" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Fu, Q. ∙ Hajdinjak, M. ∙ Moldovan, O.T. ...</div><div><strong>An early modern human from Romania with a recent Neanderthal ancestor</strong></div><div><em>Nature.</em> 2015; <strong>524</strong>:216-219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14558" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84939496491" target="_blank" aria-label="Open Scopus in new tab">Scopus (487)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26098372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14558&amp;pmid=26098372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Fu, Q. ∙ Posth, C. ∙ Hajdinjak, M. ...</div><div><strong>The genetic history of Ice Age Europe</strong></div><div><em>Nature.</em> 2016; <strong>534</strong>:200-205</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17993" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84969820697" target="_blank" aria-label="Open Scopus in new tab">Scopus (596)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27135931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17993&amp;pmid=27135931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18" title="View in article">18.</a></div><div>Fujimoto, A. ∙ Ohashi, J. ∙ Nishida, N. ...</div><div><strong>A replication study confirmed the EDAR gene to be a major contributor to population differentiation regarding head hair thickness in Asia</strong></div><div><em>Hum. Genet.</em> 2008; <strong>124</strong>:179-185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-008-0537-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-50649098835" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18704500/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-008-0537-1&amp;pmid=18704500" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Gamba, C. ∙ Jones, E.R. ∙ Teasdale, M.D. ...</div><div><strong>Genome flux and stasis in a five millennium transect of European prehistory</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:5257</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms6257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84923357868" target="_blank" aria-label="Open Scopus in new tab">Scopus (453)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25334030/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms6257&amp;pmid=25334030" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">20.</a></div><div>Gansauge, M.-T. ∙ Meyer, M.</div><div><strong>Single-stranded DNA library preparation for the sequencing of ancient or damaged DNA</strong></div><div><em>Nat. Protoc.</em> 2013; <strong>8</strong>:737-748</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2013.038" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84875500588" target="_blank" aria-label="Open Scopus in new tab">Scopus (357)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23493070/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2013.038&amp;pmid=23493070" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21-1" title="View in article">21.</a></div><div>Haak, W. ∙ Lazaridis, I. ∙ Patterson, N. ...</div><div><strong>Massive migration from the steppe was a source for Indo-European languages in Europe</strong></div><div><em>Nature.</em> 2015; <strong>522</strong>:207-211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature14317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84930953216" target="_blank" aria-label="Open Scopus in new tab">Scopus (1139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature14317&amp;pmid=25731166" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Harney, É. ∙ Patterson, N. ∙ Reich, D. ...</div><div><strong>Assessing the performance of qpAdm: a statistical tool for studying population admixture</strong></div><div><em>Genetics.</em> 2021; <strong>217</strong>:iyaa045</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/genetics/iyaa045" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85104899633" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33772284/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fgenetics%2Fiyaa045&amp;pmid=33772284" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">23.</a></div><div>Hayden, B. ∙ Villeneuve, S.</div><div><strong>A century of feasting studies</strong></div><div><em>Annu. Rev. Anthropol.</em> 2011; <strong>40</strong>:433-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-anthro-081309-145740" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-80053404707" target="_blank" aria-label="Open Scopus in new tab">Scopus (98)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-anthro-081309-145740" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Heilongjiang Provincial Institute of Cultural Relics and Archaeology, Commission for Preservation of Ancient Monuments, Raohe County</div><div><strong>The excavation of Zone III at the Xiaonanshan Site in 2015, Raohe County, Heilongjiang</strong></div><div><em>Chinese Archaeology.</em> 2019; <strong>20</strong>:87-96</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Heilongjiang+Provincial+Institute+of+Cultural+Relics+and+ArchaeologyCommission+for+Preservation+of+Ancient+Monuments%2C+Raohe+CountyThe+excavation+of+Zone+III+at+the+Xiaonanshan+Site+in+2015%2C+Raohe+County%2C+HeilongjiangChinese+Archaeology2020198796" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25" title="View in article">25.</a></div><div>Hlusko, L.J. ∙ Carlson, J.P. ∙ Chaplin, G. ...</div><div><strong>Environmental selection during the last ice age on the mother-to-infant transmission of vitamin D and fatty acids through breast milk</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2018; <strong>115</strong>:E4426-E4432</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1711788115" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85046658692" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29686092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1711788115&amp;pmid=29686092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">26.</a></div><div>Jeong, C. ∙ Ozga, A.T. ∙ Witonsky, D.B. ...</div><div><strong>Long-term genetic stability and a high-altitude East Asian origin for the peoples of the high valleys of the Himalayan arc</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:7485-7490</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1520844113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84977279989" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27325755/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1520844113&amp;pmid=27325755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">27.</a></div><div>Jeong, C. ∙ Balanovsky, O. ∙ Lukianova, E. ...</div><div><strong>The genetic history of admixture across inner Eurasia</strong></div><div><em>Nat. Ecol. Evol.</em> 2019; <strong>3</strong>:966-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-019-0878-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85065175002" target="_blank" aria-label="Open Scopus in new tab">Scopus (103)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31036896/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-019-0878-2&amp;pmid=31036896" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>Jiamusi Cultural Relics Management Station and Raohe County Cultural Relics Management Institute</div><div><strong>Neolithic burial in Xiaonanshan, Raohe County, Heilongjiang</strong></div><div><em>Kaogu.</em> 1996; <strong>1996</strong>:1-8</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=Jiamusi+Cultural+Relics+Management+Station+and+Raohe+County+Cultural+Relics+Management+InstituteNeolithic+burial+in+Xiaonanshan%2C+Raohe+County%2C+HeilongjiangKaogu1996199618" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">29.</a></div><div>Jones, E.R. ∙ Gonzalez-Fortes, G. ∙ Connell, S. ...</div><div><strong>Upper Palaeolithic genomes reveal deep roots of modern Eurasians</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:8912</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms9912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84947563586" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26567969/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms9912&amp;pmid=26567969" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">30.</a></div><div>Kamberov, Y.G. ∙ Wang, S. ∙ Tan, J. ...</div><div><strong>Modeling recent human evolution in mice by expression of a selected EDAR variant</strong></div><div><em>Cell.</em> 2013; <strong>152</strong>:691-702</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2013.01.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2013.01.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84874048561" target="_blank" aria-label="Open Scopus in new tab">Scopus (209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23415220/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.01.016&amp;pmid=23415220" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Kılınç, G.M. ∙ Kashuba, N. ∙ Koptekin, D. ...</div><div><strong>Human population dynamics and <i>Yersinia pestis</i> in ancient northeast Asia</strong></div><div><em>Sci. Adv.</em> 2021; <strong>7</strong>:eabc4587</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.abc4587" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85099219538" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33523963/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.abc4587&amp;pmid=33523963" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Kircher, M. ∙ Sawyer, S. ∙ Meyer, M.</div><div><strong>Double indexing overcomes inaccuracies in multiplex sequencing on the Illumina platform</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:e3</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr771" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84855295457" target="_blank" aria-label="Open Scopus in new tab">Scopus (732)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22021376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr771&amp;pmid=22021376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">33.</a></div><div>Korneliussen, T.S. ∙ Albrechtsen, A. ∙ Nielsen, R.</div><div><strong>ANGSD: Analysis of Next Generation Sequencing Data</strong></div><div><em>BMC Bioinformatics.</em> 2014; <strong>15</strong>:356</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12859-014-0356-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84920729412" target="_blank" aria-label="Open Scopus in new tab">Scopus (1457)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12859-014-0356-4&amp;pmid=25420514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">34.</a></div><div>Kunikita, D. ∙ Wang, L. ∙ Onuki, S. ...</div><div><strong>Radiocarbon dating and dietary reconstruction of the Early Neolithic Houtaomuga and Shuangta sites in the Song-Nen Plain, Northeast China</strong></div><div><em>Quat. Int.</em> 2017; <strong>441</strong>:62-68</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2017.01.031" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85014094153" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2017.01.031" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">35.</a></div><div>Lazaridis, I. ∙ Patterson, N. ∙ Mittnik, A. ...</div><div><strong>Ancient human genomes suggest three ancestral populations for present-day Europeans</strong></div><div><em>Nature.</em> 2014; <strong>513</strong>:409-413</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13673" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84907803675" target="_blank" aria-label="Open Scopus in new tab">Scopus (852)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25230663/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13673&amp;pmid=25230663" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">36.</a></div><div>Lazaridis, I. ∙ Nadel, D. ∙ Rollefson, G. ...</div><div><strong>Genomic insights into the origin of farming in the ancient Near East</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:419-424</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19310" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84984999386" target="_blank" aria-label="Open Scopus in new tab">Scopus (561)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27459054/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19310&amp;pmid=27459054" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37" title="View in article">37.</a></div><div>Li, N. ∙ Cribb, R.</div><div><strong>Historical Atlas of Northeast Asia, 1590-2010: Korea, Manchuria, Mongolia, Eastern Siberia</strong></div><div>Columbia University Press, 2014</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=N.LiR.CribbHistorical+Atlas+of+Northeast+Asia%2C+1590-2010%3A+Korea%2C+Manchuria%2C+Mongolia%2C+Eastern+Siberia2014Columbia+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (31658)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Li, S. ∙ Yang, S. ∙ Liu, X.</div><div><strong>Spatiotemporal variability of extreme precipitation in north and south of the Qinling-Huaihe region and influencing factors during 1960–2013</strong></div><div><em>Prog. Geogr.</em> 2015; <strong>34</strong>:354-363</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.LiS.YangX.LiuSpatiotemporal+variability+of+extreme+precipitation+in+north+and+south+of+the+Qinling-Huaihe+region+and+influencing+factors+during+1960%E2%80%932013Prog.+Geogr.342015354363" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40" title="View in article">40.</a></div><div>Lindo, J. ∙ Haas, R. ∙ Hofman, C. ...</div><div><strong>The genetic prehistory of the Andean highlands 7000 years BP though European contact</strong></div><div><em>Sci. Adv.</em> 2018; <strong>4</strong>:eaau4921</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.aau4921" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85056346614" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30417096/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.aau4921&amp;pmid=30417096" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Lipatov, M. ∙ Sanjeev, K. ∙ Patro, R. ...</div><div><strong>Maximum Likelihood Estimation of Biological Relatedness from Low Coverage Sequencing Data</strong></div><div><em>bioRxiv.</em> 2015; 023374</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=M.LipatovK.SanjeevR.PatroK.R.VeeramahMaximum+Likelihood+Estimation+of+Biological+Relatedness+from+Low+Coverage+Sequencing+DatabioRxiv2015023374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Lipson, M.</div><div><strong>Applying f<sub>4</sub> -statistics and admixture graphs: Theory and examples</strong></div><div><em>Mol. Ecol. Resour.</em> 2020; <strong>20</strong>:1658-1667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/1755-0998.13230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85089520090" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32717097/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2F1755-0998.13230&amp;pmid=32717097" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">43.</a></div><div>Lipson, M. ∙ Cheronet, O. ∙ Mallick, S. ...</div><div><strong>Ancient genomes document multiple waves of migration in Southeast Asian prehistory</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:92-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat3188" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85047268535" target="_blank" aria-label="Open Scopus in new tab">Scopus (209)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29773666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat3188&amp;pmid=29773666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">44.</a></div><div>Llorente, M.G. ∙ Jones, E.R. ∙ Eriksson, A. ...</div><div><strong>Ancient Ethiopian genome reveals extensive Eurasian admixture throughout the African continent</strong></div><div><em>Science.</em> 2015; <strong>350</strong>:820-822</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad2879" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84946962576" target="_blank" aria-label="Open Scopus in new tab">Scopus (214)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26449472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad2879&amp;pmid=26449472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">45.</a></div><div>Lu, D. ∙ Lou, H. ∙ Yuan, K. ...</div><div><strong>Ancestral Origins and Genetic History of Tibetan Highlanders</strong></div><div><em>Am. J. Hum. Genet.</em> 2016; <strong>99</strong>:580-594</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2016.07.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2016.07.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85002757551" target="_blank" aria-label="Open Scopus in new tab">Scopus (153)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27569548/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2016.07.002&amp;pmid=27569548" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">46.</a></div><div>Mallick, S. ∙ Li, H. ∙ Lipson, M. ...</div><div><strong>The Simons Genome Diversity Project: 300 genomes from 142 diverse populations</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:201-206</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature18964" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84991728057" target="_blank" aria-label="Open Scopus in new tab">Scopus (837)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27654912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature18964&amp;pmid=27654912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">47.</a></div><div>Massilani, D. ∙ Skov, L. ∙ Hajdinjak, M. ...</div><div><strong>Denisovan ancestry and population history of early East Asians</strong></div><div><em>Science.</em> 2020; <strong>370</strong>:579-583</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.abc1166" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33122380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.abc1166&amp;pmid=33122380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Mathieson, I. ∙ Lazaridis, I. ∙ Rohland, N. ...</div><div><strong>Genome-wide patterns of selection in 230 ancient Eurasians</strong></div><div><em>Nature.</em> 2015; <strong>528</strong>:499-503</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature16152" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84951286136" target="_blank" aria-label="Open Scopus in new tab">Scopus (863)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26595274/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature16152&amp;pmid=26595274" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>McColl, H. ∙ Racimo, F. ∙ Vinner, L. ...</div><div><strong>The prehistoric peopling of Southeast Asia</strong></div><div><em>Science.</em> 2018; <strong>361</strong>:88-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat3628" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85049645767" target="_blank" aria-label="Open Scopus in new tab">Scopus (243)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29976827/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat3628&amp;pmid=29976827" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Meyer, M. ∙ Kircher, M.</div><div><strong>Illumina sequencing library preparation for highly multiplexed target capture and sequencing</strong></div><div><em>Cold Spring Harb. Protoc.</em> 2010; <strong>2010</strong></div><div>pdb-prot5448</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/pdb.prot5448" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-77956285690" target="_blank" aria-label="Open Scopus in new tab">Scopus (1425)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fpdb.prot5448" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">51.</a></div><div>Meyer, M. ∙ Kircher, M. ∙ Gansauge, M.-T. ...</div><div><strong>A high-coverage genome sequence from an archaic Denisovan individual</strong></div><div><em>Science.</em> 2012; <strong>338</strong>:222-226</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1224344" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84867331947" target="_blank" aria-label="Open Scopus in new tab">Scopus (1257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22936568/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1224344&amp;pmid=22936568" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Mingram, J. ∙ Stebich, M. ∙ Schettler, G. ...</div><div><strong>Millennial-scale East Asian monsoon variability of the last glacial deduced from annually laminated sediments from Lake Sihailongwan, NE China</strong></div><div><em>Quat. Sci. Rev.</em> 2018; <strong>201</strong>:57-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quascirev.2018.09.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85055038777" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quascirev.2018.09.023" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Monroy Kuhn, J.M. ∙ Jakobsson, M. ∙ Günther, T.</div><div><strong>Estimating genetic kin relationships in prehistoric populations</strong></div><div><em>PLoS ONE.</em> 2018; <strong>13</strong>:e0195491</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0195491" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85045916532" target="_blank" aria-label="Open Scopus in new tab">Scopus (155)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29684051/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0195491&amp;pmid=29684051" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">54.</a></div><div>Moreno-Mayar, J.V. ∙ Vinner, L. ∙ de Barros Damgaard, P. ...</div><div><strong>Early human dispersals within the Americas</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaav2621</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aav2621" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85058604407" target="_blank" aria-label="Open Scopus in new tab">Scopus (203)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30409807/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aav2621&amp;pmid=30409807" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Nakatsuka, N. ∙ Moorjani, P. ∙ Rai, N. ...</div><div><strong>The promise of discovering population-specific disease-associated genes in South Asia</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1403-1407</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3917" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85028717405" target="_blank" aria-label="Open Scopus in new tab">Scopus (93)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28714977/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3917&amp;pmid=28714977" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">56.</a></div><div>Narasimhan, V.M. ∙ Patterson, N. ∙ Moorjani, P. ...</div><div><strong>The formation of human populations in South and Central Asia</strong></div><div><em>Science.</em> 2019; <strong>365</strong>:eaat7487</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat7487" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85071752467" target="_blank" aria-label="Open Scopus in new tab">Scopus (349)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31488661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat7487&amp;pmid=31488661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57-1" title="View in article">57.</a></div><div>Ning, C. ∙ Li, T. ∙ Wang, K. ...</div><div><strong>Ancient genomes from northern China suggest links between subsistence changes and human migration</strong></div><div><em>Nat. Commun.</em> 2020; <strong>11</strong>:2700</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-020-16557-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085859685" target="_blank" aria-label="Open Scopus in new tab">Scopus (124)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-020-16557-2&amp;pmid=32483115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58" title="View in article">58.</a></div><div>Olalde, I. ∙ Allentoft, M.E. ∙ Sánchez-Quinto, F. ...</div><div><strong>Derived immune and ancestral pigmentation alleles in a 7,000-year-old Mesolithic European</strong></div><div><em>Nature.</em> 2014; <strong>507</strong>:225-228</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12960" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84896319183" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24463515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12960&amp;pmid=24463515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">59.</a></div><div>Patterson, N. ∙ Price, A.L. ∙ Reich, D.</div><div><strong>Population structure and eigenanalysis</strong></div><div><em>PLoS Genet.</em> 2006; <strong>2</strong>:e190</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.0020190" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-33846006923" target="_blank" aria-label="Open Scopus in new tab">Scopus (3429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.0020190&amp;pmid=17194218" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>Patterson, N. ∙ Moorjani, P. ∙ Luo, Y. ...</div><div><strong>Ancient admixture in human history</strong></div><div><em>Genetics.</em> 2012; <strong>192</strong>:1065-1093</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1534/genetics.112.145037" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869057650" target="_blank" aria-label="Open Scopus in new tab">Scopus (1480)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1534%2Fgenetics.112.145037&amp;pmid=22960212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">61.</a></div><div>Pearson, R.</div><div><strong>The social context of early pottery in the Lingnan region of south China</strong></div><div><em>Antiquity.</em> 2005; <strong>79</strong>:819-828</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0003598X00114954" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34347300985" target="_blank" aria-label="Open Scopus in new tab">Scopus (25)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0003598X00114954" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62-1" title="View in article">62.</a></div><div>Peter, B.M.</div><div><strong>100,000 years of gene flow between Neandertals and Denisovans in the Altai mountains</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.03.13.990523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098896511" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.03.13.990523" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Peter, B.M. ∙ Huerta-Sanchez, E. ∙ Nielsen, R.</div><div><strong>Distinguishing between selective sweeps from standing variation and from a de novo mutation</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1003011</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84868152411" target="_blank" aria-label="Open Scopus in new tab">Scopus (138)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23071458/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003011&amp;pmid=23071458" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">64.</a></div><div>Pickrell, J.K. ∙ Pritchard, J.K.</div><div><strong>Inference of population splits and&nbsp;mixtures from genome-wide allele frequency data</strong></div><div><em>PLoS Genet.</em> 2012; <strong>8</strong>:e1002967</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1002967" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84869026948" target="_blank" aria-label="Open Scopus in new tab">Scopus (1556)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23166502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1002967&amp;pmid=23166502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">65.</a></div><div>Pitulko, V.V. ∙ Nikolskiy, P.A.</div><div><strong>The extinction of the woolly mammoth and the archaeological record in Northeastern Asia</strong></div><div><em>World Archaeol.</em> 2012; <strong>44</strong>:21-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1080/00438243.2012.647574" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84858129375" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1080%2F00438243.2012.647574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">66.</a></div><div>Prüfer, K.</div><div><strong>snpAD: an ancient DNA genotype caller</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:4165-4171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty507" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85055567295" target="_blank" aria-label="Open Scopus in new tab">Scopus (35)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29931305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty507&amp;pmid=29931305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">67.</a></div><div>Prüfer, K. ∙ Racimo, F. ∙ Patterson, N. ...</div><div><strong>The complete genome sequence of a Neanderthal from the Altai Mountains</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:43-49</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12886" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84892365780" target="_blank" aria-label="Open Scopus in new tab">Scopus (1399)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24352235/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12886&amp;pmid=24352235" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">68.</a></div><div>Prüfer, K. ∙ de Filippo, C. ∙ Grote, S. ...</div><div><strong>A high-coverage Neandertal genome from Vindija Cave in Croatia</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:655-658</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aao1887" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85030853901" target="_blank" aria-label="Open Scopus in new tab">Scopus (350)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28982794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aao1887&amp;pmid=28982794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69-1" title="View in article">69.</a></div><div>Purcell, S. ∙ Neale, B. ∙ Todd-Brown, K. ...</div><div><strong>PLINK: a tool set for whole-genome association and population-based linkage analyses</strong></div><div><em>Am. J. Hum. Genet.</em> 2007; <strong>81</strong>:559-575</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1086%2F519795&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1086%2F519795&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34548292504" target="_blank" aria-label="Open Scopus in new tab">Scopus (22449)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1086%2F519795&amp;pmid=17701901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">70.</a></div><div>Qin, P. ∙ Stoneking, M.</div><div><strong>Denisovan Ancestry in East Eurasian and Native American Populations</strong></div><div><em>Mol. Biol. Evol.</em> 2015; <strong>32</strong>:2665-2674</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/molbev/msv141" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84940935331" target="_blank" aria-label="Open Scopus in new tab">Scopus (89)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26104010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fmolbev%2Fmsv141&amp;pmid=26104010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">71.</a></div><div>Raghavan, M. ∙ Skoglund, P. ∙ Graf, K.E. ...</div><div><strong>Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans</strong></div><div><em>Nature.</em> 2014; <strong>505</strong>:87-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature12736" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84892367625" target="_blank" aria-label="Open Scopus in new tab">Scopus (540)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24256729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature12736&amp;pmid=24256729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">72.</a></div><div>Ramsey, C.B. ∙ Lee, S.</div><div><strong>Recent and planned developments of the program OxCal</strong></div><div><em>Radiocarbon.</em> 2013; <strong>55</strong>:720-730</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/S0033822200057878" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FS0033822200057878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">73.</a></div><div>Reich, D. ∙ Patterson, N. ∙ Kircher, M. ...</div><div><strong>Denisova admixture and the first modern human dispersals into Southeast Asia and Oceania</strong></div><div><em>Am. J. Hum. Genet.</em> 2011; <strong>89</strong>:516-528</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2011.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.ajhg.2011.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-80053899093" target="_blank" aria-label="Open Scopus in new tab">Scopus (411)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21944045/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2011.09.005&amp;pmid=21944045" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">74.</a></div><div>Reimer, P.J. ∙ Austin, W.E.N. ∙ Bard, E. ...</div><div><strong>The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP)</strong></div><div><em>Radiocarbon.</em> 2020; <strong>62</strong>:725-757</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2020.41" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85092423467" target="_blank" aria-label="Open Scopus in new tab">Scopus (3774)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2020.41" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">75.</a></div><div>Renaud, G. ∙ Stenzel, U. ∙ Kelso, J.</div><div><strong>leeHom: adaptor trimming and merging for Illumina sequencing reads</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:e141</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gku699" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84922394220" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25100869/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgku699&amp;pmid=25100869" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">76.</a></div><div>Ringbauer, H. ∙ Novembre, J. ∙ Steinrücken, M.</div><div><strong>Human Parental Relatedness through Time - Detecting Runs of Homozygosity in Ancient DNA</strong></div><div><em>bioRxiv.</em> 2020;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/2020.05.31.126912" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85098902463" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F2020.05.31.126912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">77.</a></div><div>Rohland, N. ∙ Harney, E. ∙ Mallick, S. ...</div><div><strong>Partial uracil–DNA–glycosylase treatment for screening of ancient DNA</strong></div><div><em>Philos. Trans. R Soc. Lond. B Biol. Sci.</em> 2015; <strong>370</strong>:20130624</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1098/rstb.2013.0624" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84916210143" target="_blank" aria-label="Open Scopus in new tab">Scopus (288)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25487342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1098%2Frstb.2013.0624&amp;pmid=25487342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">78.</a></div><div>Rybin, E.P. ∙ Khatsenovich, A.M. ∙ Gunchinsuren, B. ...</div><div><strong>The impact of the LGM on the development of the Upper Paleolithic in Mongolia</strong></div><div><em>Quat. Int.</em> 2016; <strong>425</strong>:69-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2016.05.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85006455434" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2016.05.001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">79.</a></div><div>Sato, H. ∙ Natsuki, D.</div><div><strong>Human behavioral responses to environmental condition and the emergence of the world’s oldest pottery in East and Northeast Asia: An overview</strong></div><div><em>Quat. Int.</em> 2017; <strong>441</strong>:12-28</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.quaint.2016.12.046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85014160931" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.quaint.2016.12.046" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">80.</a></div><div>Seguin-Orlando, A. ∙ Korneliussen, T.S. ∙ Sikora, M. ...</div><div><strong>Paleogenomics. Genomic structure in Europeans dating back at least 36,200 years</strong></div><div><em>Science.</em> 2014; <strong>346</strong>:1113-1118</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84914675035" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25378462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0114&amp;pmid=25378462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81" title="View in article">81.</a></div><div>Shang, H. ∙ Tong, H. ∙ Zhang, S. ...</div><div><strong>An early modern human from Tianyuan Cave, Zhoukoudian, China</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2007; <strong>104</strong>:6573-6578</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0702169104" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-34249776217" target="_blank" aria-label="Open Scopus in new tab">Scopus (150)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17416672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0702169104&amp;pmid=17416672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">82.</a></div><div>Sikora, M. ∙ Seguin-Orlando, A. ∙ Sousa, V.C. ...</div><div><strong>Ancient genomes show social and reproductive behavior of early Upper Paleolithic foragers</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:659-662</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aao1807" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85030861669" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28982795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aao1807&amp;pmid=28982795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">83.</a></div><div>Sikora, M. ∙ Pitulko, V.V. ∙ Sousa, V.C. ...</div><div><strong>The population history of northeastern Siberia since the Pleistocene</strong></div><div><em>Nature.</em> 2019; <strong>570</strong>:182-188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-019-1279-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85066934053" target="_blank" aria-label="Open Scopus in new tab">Scopus (218)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-019-1279-z&amp;pmid=31168093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84-1" title="View in article">84.</a></div><div>Siska, V. ∙ Jones, E.R. ∙ Jeon, S. ...</div><div><strong>Genome-wide data from two early Neolithic East Asian individuals dating to 7700 years ago</strong></div><div><em>Sci. Adv.</em> 2017; <strong>3</strong>:e1601877</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.1601877" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85020941932" target="_blank" aria-label="Open Scopus in new tab">Scopus (83)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28164156/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.1601877&amp;pmid=28164156" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">85.</a></div><div>Skoglund, P. ∙ Posth, C. ∙ Sirak, K. ...</div><div><strong>Genomic insights into the peopling of the Southwest Pacific</strong></div><div><em>Nature.</em> 2016; <strong>538</strong>:510-513</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19844" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84993661810" target="_blank" aria-label="Open Scopus in new tab">Scopus (182)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27698418/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19844&amp;pmid=27698418" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">86.</a></div><div>Stebich, M. ∙ Mingram, J. ∙ Han, J. ...</div><div><strong>Late Pleistocene spread&nbsp;of (cool-) temperate forests in Northeast China and climate changes synchronous with the North Atlantic region</strong></div><div><em>Global Planet. Change.</em> 2009; <strong>65</strong>:56-70</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gloplacha.2008.10.010" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-58849131825" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gloplacha.2008.10.010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">87.</a></div><div>Tallavaara, M. ∙ Luoto, M. ∙ Korhonen, N. ...</div><div><strong>Human population dynamics in Europe over the Last Glacial Maximum</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:8232-8237</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1503784112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84936803076" target="_blank" aria-label="Open Scopus in new tab">Scopus (129)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26100880/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1503784112&amp;pmid=26100880" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">88.</a></div><div>Tan, J. ∙ Yang, Y. ∙ Tang, K. ...</div><div><strong>The adaptive variant EDARV370A is associated with straight hair in East Asians</strong></div><div><em>Hum. Genet.</em> 2013; <strong>132</strong>:1187-1191</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-013-1324-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-84884900721" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23793515/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-013-1324-1&amp;pmid=23793515" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">89.</a></div><div>Wang, L. ∙ Sebillaud, P.</div><div><strong>The emergence of early pottery in East Asia: New discoveries and perspectives</strong></div><div><em>J.&nbsp;World Prehist.</em> 2019; <strong>32</strong>:73-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10963-018-9126-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85060217290" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10963-018-9126-y" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">90.</a></div><div>Wang, Y. ∙ Song, F. ∙ Zhu, J. ...</div><div><strong>GSA: Genome Sequence Archive&lt;sup/&gt;</strong></div><div><em>Genomics Proteomics Bioinformatics.</em> 2017; <strong>15</strong>:14-18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.gpb.2017.01.001" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85012868681" target="_blank" aria-label="Open Scopus in new tab">Scopus (441)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28387199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.gpb.2017.01.001&amp;pmid=28387199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">91.</a></div><div>Wang, C.-C. ∙ Yeh, H.-Y. ∙ Popov, A.N. ...</div><div><strong>Genomic insights into the formation of human populations in East Asia</strong></div><div><em>Nature.</em> 2021; <strong>591</strong>:413-419</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-021-03336-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85101273487" target="_blank" aria-label="Open Scopus in new tab">Scopus (185)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33618348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-021-03336-2&amp;pmid=33618348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">92.</a></div><div>Yang, M.A. ∙ Gao, X. ∙ Theunert, C. ...</div><div><strong>40,000-Year-Old Individual from Asia Provides Insight into Early Population Structure in Eurasia</strong></div><div><em>Curr. Biol.</em> 2017; <strong>27</strong>:3202-3208.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cub.2017.09.030&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031312995" target="_blank" aria-label="Open Scopus in new tab">Scopus (140)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29033327/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2017.09.030&amp;pmid=29033327" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">93.</a></div><div>Yang, M.A. ∙ Fan, X. ∙ Sun, B. ...</div><div><strong>Ancient DNA indicates human population shifts and admixture in northern and southern China</strong></div><div><em>Science.</em> 2020; <strong>369</strong>:282-288</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aba0909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85087796796" target="_blank" aria-label="Open Scopus in new tab">Scopus (177)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba0909&amp;pmid=32409524" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">94.</a></div><div>Ye, K. ∙ Gao, F. ∙ Wang, D. ...</div><div><strong>Dietary adaptation of FADS genes in Europe varied across time and geography</strong></div><div><em>Nat. Ecol. Evol.</em> 2017; <strong>1</strong>:167</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41559-017-0167" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85031759069" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29094686/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41559-017-0167&amp;pmid=29094686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95-1" title="View in article">95.</a></div><div>Yu, H. ∙ Spyrou, M.A. ∙ Karapetian, M. ...</div><div><strong>Paleolithic to Bronze Age Siberians Reveal Connections with First Americans and across Eurasia</strong></div><div><em>Cell.</em> 2020; <strong>181</strong>:1232-1245.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=10.1016%2Fj.cell.2020.04.037&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85085755497" target="_blank" aria-label="Open Scopus in new tab">Scopus (61)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32437661/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.04.037&amp;pmid=32437661" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96-1" title="View in article">96.</a></div><div>Yue, J.-P. ∙ Li, Y.-Q. ∙ Yang, S.</div><div><strong>Neolithisation in the southern Lesser Khingan Mountains: lithic technologies and ecological adaptation</strong></div><div><em>Antiquity.</em> 2019; <strong>93</strong>:1144-1160</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15184/aqy.2019.92" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85073781764" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15184%2Faqy.2019.92" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">97.</a></div><div>Zhang, H. ∙ Chang, F. ∙ Li, H. ...</div><div><strong>OSL and AMS <sup>14</sup>C Age of the Most Complete Mammoth Fossil Skeleton from Northeastern China and its Paleoclimate Significance</strong></div><div><em>Radiocarbon.</em> 2019; <strong>61</strong>:347-358</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1017/RDC.2018.48" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2021.04.040&amp;key=2-s2.0-85061175283" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1017%2FRDC.2018.48" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">98.</a></div><div>Zhao, B. ∙ Sun, M. ∙ Du, Z.</div><div><strong>The Age and Characteristics of the Jade Unearthed from Xiaonanshan Tomb in Raohe County</strong></div><div><em>Bianjiang Kaogu Yanjiu.</em> 2013; <strong>2013</strong>:69-78</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.ZhaoM.SunZ.DuThe+Age+and+Characteristics+of+the+Jade+Unearthed+from+Xiaonanshan+Tomb+in+Raohe+CountyBianjiang+Kaogu+Yanjiu201320136978" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (8)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/a796c76f-a63a-4cb5-91d6-f25bf090a902/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/460d6feb-e7b2-46a0-ba36-6e215e149841/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/423b9c27-6623-413a-8eff-b719a292c582/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/0f2bb9aa-f983-440f-a49f-db0e28d2cd2e/main.assets/figs1.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/50405999-da6a-4885-9dd0-343aa993a961/main.assets/figs2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/2fef6af1-414e-4075-a26a-991d5c7b8496/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/981ecd5b-36aa-431e-a069-279fb98dc540/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/asset/3976b942-d42e-4628-ba54-673d5bcb77c7/main.assets/figs4.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867420X00134%3Barticle%3Aarticle%3Apii%5C%3AS0092867421005754%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">105</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">81</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">33</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">92</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2021.04.040&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (4)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/attachment/935980df-b7ef-40c6-bc0a-851b1df98db4/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (229.21 KB)</span></a><div class="core-description">Document S1. Tables S2, S4, and S6</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/attachment/9bc4721d-1202-4bf8-a44d-221eb61ea22b/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (59.47 KB)</span></a><div class="core-description">Table S1. Overview of newly sequenced and published individuals, related to Figures 1 and S1, Table 1, and STAR Methods</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/attachment/0b476ae8-400b-4ee1-9423-af68a2975c71/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (63.08 KB)</span></a><div class="core-description">Table S3. D statistics for populations, including Tianyuan, AR33K, Ancient Siberians, AR9.2K_o, Qihe, ancient Shandong, AR19K, and AR14K, related to Figures 1, 2, 3, and S1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2021.04.040/attachment/9f8953ff-2768-4105-b267-0091dd0e481c/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (17.01 KB)</span></a><div class="core-description">Table S5. qpAdm analyses for Ancient Paleo-Siberians and AR9.2K_o, related to Figures 1, 3, and S1</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related articles (40)</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867421005754" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true"><div class="section-paragraph linked-article axel-related-articles"><ul class="rlist axel-related-articles__items"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02851-7" target="_blank" title="Human dietary diversity in the Colombian Andes at the terminal Pleistocene-late Holocene sites Tequendama and Aguazuque">Human dietary diversity in the Colombian Andes at the terminal Pleistocene-late Holocene sites Tequendama and Aguazuque</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ziegler et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">January 7, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)02483-5" target="_blank" title="Relationships of Late Pleistocene giant deer as revealed by Sinomegaceros mitogenomes from East Asia">Relationships of Late Pleistocene giant deer as revealed by Sinomegaceros mitogenomes from East Asia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Xiao et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">November 6, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(22)01892-9" target="_blank" title="Middle Holocene Siberian genomes reveal highly connected gene pools throughout North Asia">Middle Holocene Siberian genomes reveal highly connected gene pools throughout North Asia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">January 12, 2023</span></div></li></ul><ul class="rlist axel-related-articles__items axel-related-articles__extra-items js--hidden"><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)01366-6" target="_blank" title="Concerns about ancient DNA sequences reported from a Late Pleistocene individual from Southeast Asia">Concerns about ancient DNA sequences reported from a Late Pleistocene individual from Southeast Asia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Tabin et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">March 24, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02630-0" target="_blank" title="Population expansion from central plain to northern coastal China inferred from ancient human genomes">Population expansion from central plain to northern coastal China inferred from ancient human genomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">November 17, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(22)01824-3" target="_blank" title="Spatial and temporal heterogeneity in human mobility patterns in Holocene Southwest Asia and the East Mediterranean">Spatial and temporal heterogeneity in human mobility patterns in Holocene Southwest Asia and the East Mediterranean</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Koptekin et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">December 8, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/ajhg/fulltext/S0002-9297(24)00457-9" target="_blank" title="Population history and admixture of the Fulani people from the Sahel">Population history and admixture of the Fulani people from the Sahel</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Fortes-Lima et al.</li></ul></div><div class="linked-article__details"><span><i>The American Journal of Human Genetics</i></span><span id="item_date">February 06, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(25)00503-2" target="_blank" title="Population histories of the Indigenous Adivasi and Sinhalese from Sri Lanka using whole genomes">Population histories of the Indigenous Adivasi and Sinhalese from Sri Lanka using whole genomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Urban Aragon et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">June 09, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(20)32714-6" target="_blank" title="Late Pleistocene-Holocene sedimentary evolution in the coastal zone of the Red River Delta">Late Pleistocene-Holocene sedimentary evolution in the coastal zone of the Red River Delta</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yen et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 18, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)08058-7" target="_blank" title="Cone-beam CT evaluation of post-extraction alveolar bone changes at the maxillary incisor sites in an East Asian population: A cross-sectional study">Cone-beam CT evaluation of post-extraction alveolar bone changes at the maxillary incisor sites in an East Asian population: A cross-sectional study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Qu et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">May 29, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(21)00635-8" target="_blank" title="Human population history at the crossroads of East and Southeast Asia since 11,000 years ago">Human population history at the crossroads of East and Southeast Asia since 11,000 years ago</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">June 24, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(21)00138-X" target="_blank" title="Middle Pleistocene genome calibrates a revised evolutionary history of extinct cave bears">Middle Pleistocene genome calibrates a revised evolutionary history of extinct cave bears</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Barlow et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">February 15, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)08494-3" target="_blank" title="Prevalence and associated factors of prediabetes in adult East African population: A systematic review and meta-analysis">Prevalence and associated factors of prediabetes in adult East African population: A systematic review and meta-analysis</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Asmelash et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">October 19, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)11950-0" target="_blank" title="Prevalence of radix accesoria dentis in a northern Peruvian population evaluated by cone-beam tomography">Prevalence of radix accesoria dentis in a northern Peruvian population evaluated by cone-beam tomography</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>León-Almanza et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 7, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(22)00928-9" target="_blank" title="A Late Pleistocene human genome from Southwest China">A Late Pleistocene human genome from Southwest China</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">July 14, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/ajhg/fulltext/S0002-9297(24)00391-4" target="_blank" title="Demographic history and genetic variation of the Armenian population">Demographic history and genetic variation of the Armenian population</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hovhannisyan et al.</li></ul></div><div class="linked-article__details"><span><i>The American Journal of Human Genetics</i></span><span id="item_date">November 25, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)01059-3" target="_blank" title="Origin and population structure of native dog breeds in the Korean peninsula and East Asia">Origin and population structure of native dog breeds in the Korean peninsula and East Asia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ahn et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">May 28, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/hgg-advances/fulltext/S2666-2477(24)00015-0" target="_blank" title="Placental expression quantitative trait loci in an East Asian population">Placental expression quantitative trait loci in an East Asian population</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Choi et al.</li></ul></div><div class="linked-article__details"><span><i>Human Genetics and Genomics Advances</i></span><span id="item_date">February 2, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)09416-1" target="_blank" title="Investigating population-specific epilepsy detection from noisy EEG signals using deep-learning models">Investigating population-specific epilepsy detection from noisy EEG signals using deep-learning models</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Islam et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">November 23, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-genomics/fulltext/S2666-979X(22)00062-3" target="_blank" title="Population dynamics and genetic connectivity in recent chimpanzee history">Population dynamics and genetic connectivity in recent chimpanzee history</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Fontsere et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Genomics</i></span><span id="item_date">June 1, 2022</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)00953-8" target="_blank" title="Childhood stunting is highly clustered in Northern Province of Rwanda: A spatial analysis of a population-based study">Childhood stunting is highly clustered in Northern Province of Rwanda: A spatial analysis of a population-based study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Ndagijimana et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">January 19, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)01239-9" target="_blank" title="Avoidable false PSMC population size peaks occur across numerous studies">Avoidable false PSMC population size peaks occur across numerous studies</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Hilgers et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">February 6, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00099-6" target="_blank" title="The amphipod genome reveals population dynamics and adaptations to hadal environment">The amphipod genome reveals population dynamics and adaptations to hadal environment</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">March 06, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(23)00101-0" target="_blank" title="Whole-genome sequencing reveals a complex African population demographic history and signatures of local adaptation">Whole-genome sequencing reveals a complex African population demographic history and signatures of local adaptation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Fan et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">March 02, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(21)00839-4" target="_blank" title="The genomic history of the Middle East">The genomic history of the Middle East</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Almarri et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">August 4, 2021</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)02145-4" target="_blank" title="A novel APA-based prognostic signature may predict the prognosis of lung adenocarcinoma in an East Asian population">A novel APA-based prognostic signature may predict the prognosis of lung adenocarcinoma in an East Asian population</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Zhang et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">September 27, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(24)02066-2" target="_blank" title="Climate and cultural evolution drove Holocene cropland change in the Huai River Valley, China">Climate and cultural evolution drove Holocene cropland change in the Huai River Valley, China</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Yu et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">September 5, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/iscience/fulltext/S2589-0042(23)01874-6" target="_blank" title="Reconstructing the population history of the Sinhalese, the major ethnic group in Śrī Laṅkā">Reconstructing the population history of the Sinhalese, the major ethnic group in Śrī Laṅkā</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Singh et al.</li></ul></div><div class="linked-article__details"><span><i>iScience</i></span><span id="item_date">August 31, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)06554-4" target="_blank" title="The first large-scale record of gelada's spatial distribution and population size in Mountain chains of Tigray, Northern Ethiopia">The first large-scale record of gelada's spatial distribution and population size in Mountain chains of Tigray, Northern Ethiopia</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Haileselasie et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 22, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(25)00559-8" target="_blank" title="Unveiling the origins and genetic makeup of the “forgotten people”: A study of the Sarmatian-period population in the Carpathian Basin">Unveiling the origins and genetic makeup of the “forgotten people”: A study of the Sarmatian-period population in the Carpathian Basin</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Schütz et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">June 10, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(20)30509-2" target="_blank" title="Genomic History of Neolithic to Bronze Age Anatolia, Northern Levant, and Southern Caucasus">Genomic History of Neolithic to Bronze Age Anatolia, Northern Levant, and Southern Caucasus</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Skourtanioti et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">May 28, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/trends/genetics/fulltext/S0168-9525(25)00050-2" target="_blank" title="Can genomic analysis actually estimate past population size?">Can genomic analysis actually estimate past population size?</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Bansal et al.</li></ul></div><div class="linked-article__details"><span><i>Trends in Genetics</i></span><span id="item_date">April 24, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)11964-0" target="_blank" title="Advancing sub-seasonal to seasonal multi-model ensemble precipitation prediction in east asia: Deep learning-based post-processing for improved accuracy">Advancing sub-seasonal to seasonal multi-model ensemble precipitation prediction in east asia: Deep learning-based post-processing for improved accuracy</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Chung et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">August 7, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(24)00093-9" target="_blank" title="Late Pleistocene stickleback environmental genomes reveal the chronology of freshwater adaptation">Late Pleistocene stickleback environmental genomes reveal the chronology of freshwater adaptation</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Laine et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">February 12, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(23)04334-7" target="_blank" title="The relationship between the serum lipid profile and hepatocellular carcinoma in east Asian population: A mendelian randomization study">The relationship between the serum lipid profile and hepatocellular carcinoma in east Asian population: A mendelian randomization study</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Pan et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">June 8, 2023</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-genomics/fulltext/S2666-979X(24)00034-X" target="_blank" title="Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria">Ancient genomes illuminate Eastern Arabian population history and adaptation against malaria</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Martiniano et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Genomics</i></span><span id="item_date">February 27, 2024</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell-reports/fulltext/S2211-1247(25)00033-6" target="_blank" title="Dynamic history of the Central Plain and Haidai region inferred from Late Neolithic to Iron Age ancient human genomes">Dynamic history of the Central Plain and Haidai region inferred from Late Neolithic to Iron Age ancient human genomes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Fang et al.</li></ul></div><div class="linked-article__details"><span><i>Cell Reports</i></span><span id="item_date">February 1, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/current-biology/fulltext/S0960-9822(25)00447-6" target="_blank" title="Loss of dance and infant-directed song among the Northern Aché">Loss of dance and infant-directed song among the Northern Aché</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Singh et al.</li></ul></div><div class="linked-article__details"><span><i>Current Biology</i></span><span id="item_date">April 29, 2025</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/cell/fulltext/S0092-8674(20)30477-3" target="_blank" title="A Paleogenomic Reconstruction of the Deep Population History of the Andes">A Paleogenomic Reconstruction of the Deep Population History of the Andes</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Nakatsuka et al.</li></ul></div><div class="linked-article__details"><span><i>Cell</i></span><span id="item_date">May 7, 2020</span></div></li><li class="linked-article__item"><a class="linked-article__title link--underlined" href="/heliyon/fulltext/S2405-8440(24)06098-5" target="_blank" title="Evolutionary profiles and complex admixture landscape in East Asia: New insights from modern and ancient Y chromosome variation perspectives">Evolutionary profiles and complex admixture landscape in East Asia: New insights from modern and ancient Y chromosome variation perspectives</a><div><ul class="meta__authors rlist--inline comma" title="list of authors"><li>Wang et al.</li></ul></div><div class="linked-article__details"><span><i>Heliyon</i></span><span id="item_date">April 29, 2024</span></div></li></ul><a href="javascript:void(0)" class="toggle-extra-items"><span>Show more</span><i aria-hidden="true" class="icon-arrow_d_n"></i></a></div></div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2021.04.040","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">The deep population history of northern East Asia from the Late Pleistocene to the Holocene</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.04.040/asset/a796c76f-a63a-4cb5-91d6-f25bf090a902/main.assets/gr1.jpg" height="622" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/3b9b8c1d-babf-4fed-9b30-8be284eafb65/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.04.040/asset/460d6feb-e7b2-46a0-ba36-6e215e149841/main.assets/gr2.jpg" height="574" width="506" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/cb78a1fb-c94f-4a75-81d9-ba20a37c81ff/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.04.040/asset/423b9c27-6623-413a-8eff-b719a292c582/main.assets/gr3.jpg" height="369" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/549534d7-7c1b-4ad4-80c2-4834f78ad6f5/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.04.040/asset/0f2bb9aa-f983-440f-a49f-db0e28d2cd2e/main.assets/figs1.jpg" height="663" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/eaa648b5-d35f-49cd-88c8-602d9b8fbe5b/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.04.040/asset/50405999-da6a-4885-9dd0-343aa993a961/main.assets/figs2.jpg" height="589" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/cfe78eb5-34af-4645-9e38-5817f9e20b00/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.04.040/asset/2fef6af1-414e-4075-a26a-991d5c7b8496/main.assets/figs3.jpg" height="699" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/22a6ab84-2849-4a63-8c20-940ee85bd534/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.04.040/asset/981ecd5b-36aa-431e-a069-279fb98dc540/main.assets/gr4.jpg" height="640" width="775" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/f612fb28-baa4-4f70-a374-9c0b744e0f52/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.04.040/asset/3976b942-d42e-4628-ba54-673d5bcb77c7/main.assets/figs4.jpg" height="958" width="522" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/61fe1ca6-d816-46e3-afc3-d0478fccdd14/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2021.04.040/asset/a796c76f-a63a-4cb5-91d6-f25bf090a902/main.assets/gr1.jpg" height="622" width="657" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/3b9b8c1d-babf-4fed-9b30-8be284eafb65/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2021.04.040/asset/460d6feb-e7b2-46a0-ba36-6e215e149841/main.assets/gr2.jpg" height="574" width="506" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/cb78a1fb-c94f-4a75-81d9-ba20a37c81ff/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2021.04.040/asset/423b9c27-6623-413a-8eff-b719a292c582/main.assets/gr3.jpg" height="369" width="506" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/549534d7-7c1b-4ad4-80c2-4834f78ad6f5/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2021.04.040/asset/0f2bb9aa-f983-440f-a49f-db0e28d2cd2e/main.assets/figs1.jpg" height="663" width="657" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/eaa648b5-d35f-49cd-88c8-602d9b8fbe5b/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2021.04.040/asset/50405999-da6a-4885-9dd0-343aa993a961/main.assets/figs2.jpg" height="589" width="657" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/cfe78eb5-34af-4645-9e38-5817f9e20b00/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2021.04.040/asset/2fef6af1-414e-4075-a26a-991d5c7b8496/main.assets/figs3.jpg" height="699" width="506" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/22a6ab84-2849-4a63-8c20-940ee85bd534/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2021.04.040/asset/981ecd5b-36aa-431e-a069-279fb98dc540/main.assets/gr4.jpg" height="640" width="775" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/f612fb28-baa4-4f70-a374-9c0b744e0f52/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2021.04.040/asset/3976b942-d42e-4628-ba54-673d5bcb77c7/main.assets/figs4.jpg" height="958" width="522" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2021.04.040/asset/61fe1ca6-d816-46e3-afc3-d0478fccdd14/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00575-4';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(21)00575-4';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749590418230","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"184","issueNumber":"12","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867421005754","coverDate":"202106","type":"XOCS-JOURNAL:SCOPE-FULL","title":"The deep population history of northern East Asia from the Late Pleistocene to the Holocene","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbe96b49ed4293',t:'MTc0OTU5MDQxNy4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbe96b49ed4293&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2021.04.040&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>