<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="b795d6a7-2d71-49c4-9f0c-272398818de9" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;article:article:pii\:S0092867422006559;website:website:cell-site;ctype:string:Journal Content;journal:journal:cell;issue:issue:pii\:S0092867421X00153;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(22)00655-9">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/007db6fa-a2e7-4f73-a807-dee5e942916c/fx1.jpg">
    


        <meta property="og:description" content="Pure populations of human artery and vein cells are rapidly generated from pluripotent
stem cells. This provides a BSL4-compatible in&nbsp;vitro model to study how the deadly
Nipah and Hendra viruses infect blood vessels.">
    











        <meta name="twitter:title" content="Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/007db6fa-a2e7-4f73-a807-dee5e942916c/fx1.jpg">
    


        <meta name="twitter:description" content="Pure populations of human artery and vein cells are rapidly generated from pluripotent
stem cells. This provides a BSL4-compatible in&nbsp;vitro model to study how the deadly
Nipah and Hendra viruses infect blood vessels.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses"> <meta name="citation_date" content="2022/07/07"> <meta name="citation_online_date" content="2022/06/22"> <meta name="citation_volume" content="185"> <meta name="citation_issue" content="14"> <meta name="citation_firstpage" content="2523"> <meta name="citation_lastpage" content="2541.e30"> <meta name="citation_pmid" content="35738284"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867422006559"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867422006559"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867422006559"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2022.05.024"> <meta name="citation_pmid" content="35738284"> <meta name="citation_author" content="Lay Teng Ang"><meta name="citation_author_email" content="layteng@stanford.edu"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Alana T. Nguyen"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Kevin J. Liu"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Angela Chen"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Xiaochen Xiong"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Matthew Curtis"><meta name="citation_author_institution" content="Department of Biomedical Engineering, University of California, Davis, Davis, CA 95616, USA"><meta name="citation_author" content="Renata M. Martin"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Brian C. Raftry"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Chun Yi Ng"><meta name="citation_author_institution" content="Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore 308232, Singapore"><meta name="citation_author" content="Uwe Vogel"><meta name="citation_author_institution" content="Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany"><meta name="citation_author" content="Angelika Lander"><meta name="citation_author_institution" content="Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany"><meta name="citation_author" content="Benjamin J. Lesch"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Jonas L. Fowler"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Alyssa R. Holman"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Timothy Chai"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Siva Vijayakumar"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Fabian P. Suchy"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Toshinobu Nishimura"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Joydeep Bhadury"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Matthew H. Porteus"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Hiromitsu Nakauchi"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Genetics, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Christine Cheung"><meta name="citation_author_institution" content="Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore 308232, Singapore"><meta name="citation_author_institution" content="Institute of Molecular and Cell Biology, A∗STAR, Singapore 138673, Singapore"><meta name="citation_author" content="Steven C. George"><meta name="citation_author_institution" content="Department of Biomedical Engineering, University of California, Davis, Davis, CA 95616, USA"><meta name="citation_author" content="Kristy Red-Horse"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author" content="Joseph B. Prescott"><meta name="citation_author_email" content="prescottj@rki.de"><meta name="citation_author_institution" content="Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany"><meta name="citation_author" content="Kyle M. Loh"><meta name="citation_author_email" content="kyleloh@stanford.edu"><meta name="citation_author_institution" content="Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA"><meta name="citation_author_institution" content="Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA"><meta name="citation_keywords" content="human pluripotent stem cells; developmental biology; endothelial cells; artery; vein; Nipah virus; Hendra virus; biosafety-level-4 virology"><meta name="citation_reference" content="citation_title=Characteristics of Nipah virus and Hendra virus replication in different cell lines and their suitability for antiviral screening; citation_author=M. Aljofan; citation_author=S. Saubern; citation_author=A.G. Meyer; citation_author=G. Marsh; citation_author=J. Meers; citation_author=B.A. Mungall; citation_journal_title=Virus Res.; citation_volume=142; citation_doi=10.1016/j.virusres.2009.01.014; citation_firstpage=92; citation_lastpage=99; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=FastQC: A quality control tool for high throughput sequence data; citation_author=S. Andrews; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Nipah virus infection; citation_author=B.S.P. Ang; citation_author=T.C.C. Lim; citation_author=L. Wang; citation_journal_title=J.&nbsp;Clin. Microbiol.; citation_volume=56; citation_doi=10.1128/JCM.01875-17; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Unraveling a novel transcription factor code determining the human arterial-specific endothelial cell signature; citation_author=X.L. Aranguren; citation_author=X. Agirre; citation_author=M. Beerens; citation_author=G. Coppiello; citation_author=M. Uriz; citation_author=I. Vandersmissen; citation_author=M. Benkheil; citation_author=J. Panadero; citation_author=N. Aguado; citation_author=A. Pascual-Montano; citation_author=et al.; citation_journal_title=Blood; citation_volume=122; citation_doi=10.1182/blood-2013-02-483255; citation_firstpage=3982; citation_lastpage=3992; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Outbreak investigation of Nipah virus disease in Kerala, India, 2018; citation_author=G. Arunkumar; citation_author=R. Chandni; citation_author=D.T. Mourya; citation_author=S.K. Singh; citation_author=R. Sadanandan; citation_author=P. Sudan; citation_author=B. Bhargava; citation_author=Nipah Investigators People and Health Study Group; citation_journal_title=J.&nbsp;Infect. Dis.; citation_volume=219; citation_doi=10.1093/infdis/jiy612; citation_firstpage=1867; citation_lastpage=1878; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Organotypic vasculature: From descriptive heterogeneity to functional pathophysiology; citation_author=H.G. Augustin; citation_author=G.Y. Koh; citation_journal_title=Science; citation_volume=357; citation_doi=10.1126/science.aal2379; citation_firstpage=eaal2379; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Heatmapper: web-enabled heat mapping for all; citation_author=S. Babicki; citation_author=D. Arndt; citation_author=A. Marcu; citation_author=Y. Liang; citation_author=J.R. Grant; citation_author=A. Maciejewski; citation_author=D.S. Wishart; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_doi=10.1093/nar/gkw419; citation_firstpage=W147; citation_lastpage=W153; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Syrian hamsters (Mesocricetus auratus) oronasally inoculated with a Nipah virus isolate from Bangladesh or Malaysia develop similar respiratory tract lesions; citation_author=L. Baseler; citation_author=E. de Wit; citation_author=D.P. Scott; citation_author=V.J. Munster; citation_author=H. Feldmann; citation_journal_title=Vet. Pathol.; citation_volume=52; citation_doi=10.1177/0300985814556189; citation_firstpage=38; citation_lastpage=45; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Dimensionality reduction for visualizing single-cell data using UMAP; citation_author=E. Becht; citation_author=L. McInnes; citation_author=J. Healy; citation_author=C.-A. Dutertre; citation_author=I.W.H. Kwok; citation_author=L.G. Ng; citation_author=F. Ginhoux; citation_author=E.W. Newell; citation_journal_title=Nat. Biotechnol.; citation_volume=37; citation_doi=10.1038/nbt.4314; citation_firstpage=38; citation_lastpage=44; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Ephrin-B2 ligand is a functional receptor for Hendra virus and Nipah virus; citation_author=M.I. Bonaparte; citation_author=A.S. Dimitrov; citation_author=K.N. Bossart; citation_author=G. Crameri; citation_author=B.A. Mungall; citation_author=K.A. Bishop; citation_author=V. Choudhry; citation_author=D.S. Dimitrov; citation_author=L.-F. Wang; citation_author=B.T. Eaton; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_doi=10.1073/pnas.0504887102; citation_firstpage=10652; citation_lastpage=10657; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Near-optimal probabilistic RNA-seq quantification; citation_author=N.L. Bray; citation_author=H. Pimentel; citation_author=P. Melsted; citation_author=L. Pachter; citation_journal_title=Nat. Biotechnol.; citation_volume=34; citation_doi=10.1038/nbt.3519; citation_firstpage=525; citation_lastpage=527; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Preparation and assay of Sendai virus; citation_author=K. Cantell; citation_author=S. Hirvonen; citation_journal_title=Methods Enzymol.; citation_volume=78; citation_doi=10.1016/0076-6879(81)78131-x; citation_firstpage=299; citation_lastpage=301; citation_publication_date=1981; "><meta name="citation_reference" content="citation_title=Angiogenesis in life, disease and medicine; citation_author=P. Carmeliet; citation_journal_title=Nature; citation_volume=438; citation_doi=10.1038/nature04478; citation_firstpage=932; citation_lastpage=936; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Vegf signaling promotes vascular endothelial differentiation by modulating etv2 expression; citation_author=S. Casie Chetty; citation_author=M.S. Rost; citation_author=J.R. Enriquez; citation_author=J.A. Schumacher; citation_author=K. Baltrunaite; citation_author=A. Rossi; citation_author=D.Y.R. Stainier; citation_author=S. Sumanas; citation_journal_title=Dev. Biol.; citation_volume=424; citation_doi=10.1016/j.ydbio.2017.03.005; citation_firstpage=147; citation_lastpage=161; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Biosafety in Microbiological and Biomedical Laboratories; citation_author=CDC; citation_author=NIH; citation_publication_date=2020; citation_publisher=United States Department of Health and Human Services; "><meta name="citation_reference" content="citation_title=Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool; citation_author=E.Y. Chen; citation_author=C.M. Tan; citation_author=Y. Kou; citation_author=Q. Duan; citation_author=Z. Wang; citation_author=G.V. Meirelles; citation_author=N.R. Clark; citation_author=A. Ma’ayan; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_doi=10.1186/1471-2105-14-128; citation_firstpage=128; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=mda-5, but not RIG-I, is a common target for paramyxovirus V proteins; citation_author=K. Childs; citation_author=N. Stock; citation_author=C. Ross; citation_author=J. Andrejeva; citation_author=L. Hilton; citation_author=M. Skinner; citation_author=R. Randall; citation_author=S. Goodbourn; citation_journal_title=Virology; citation_volume=359; citation_doi=10.1016/j.virol.2006.09.023; citation_firstpage=190; citation_lastpage=200; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Stepwise arteriovenous fate acquisition during mammalian vasculogenesis; citation_author=D.C. Chong; citation_author=Y. Koo; citation_author=K. Xu; citation_author=S. Fu; citation_author=O. Cleaver; citation_journal_title=Dev. Dyn.; citation_volume=240; citation_doi=10.1002/dvdy.22706; citation_firstpage=2153; citation_lastpage=2165; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Nipah virus: a recently emergent deadly paramyxovirus; citation_author=K.B. Chua; citation_author=W.J. Bellini; citation_author=P.A. Rota; citation_author=B.H. Harcourt; citation_author=A. Tamin; citation_author=S.K. Lam; citation_author=T.G. Ksiazek; citation_author=P.E. Rollin; citation_author=S.R. Zaki; citation_author=W. Shieh; citation_author=et al.; citation_journal_title=Science; citation_volume=288; citation_doi=10.1126/science.288.5470.1432; citation_firstpage=1432; citation_lastpage=1435; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Nipah virus sequesters inactive STAT1 in the nucleus via a P gene-encoded mechanism; citation_author=M.J. Ciancanelli; citation_author=V.A. Volchkova; citation_author=M.L. Shaw; citation_author=V.E. Volchkov; citation_author=C.F. Basler; citation_journal_title=J.&nbsp;Virol.; citation_volume=83; citation_doi=10.1128/JVI.02610-08; citation_firstpage=7828; citation_lastpage=7841; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=VEGFA-dependent and -independent pathways synergise to drive Scl expression and initiate programming of the blood stem cell lineage in Xenopus; citation_author=A. Ciau-Uitz; citation_author=P. Pinheiro; citation_author=A. Kirmizitas; citation_author=J. Zuo; citation_author=R. Patient; citation_journal_title=Development; citation_volume=140; citation_firstpage=2632; citation_lastpage=2642; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Sox17 is indispensable for acquisition and maintenance of arterial identity; citation_author=M. Corada; citation_author=F. Orsenigo; citation_author=M.F. Morini; citation_author=M.E. Pitulescu; citation_author=G. Bhat; citation_author=D. Nyqvist; citation_author=F. Breviario; citation_author=V. Conti; citation_author=A. Briot; citation_author=M.L. Iruela-Arispe; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=4; citation_doi=10.1038/ncomms3609; citation_firstpage=2609; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Endothelial cells and VEGF in vascular development; citation_author=L. Coultas; citation_author=K. Chawengsaksophak; citation_author=J. Rossant; citation_journal_title=Nature; citation_volume=438; citation_doi=10.1038/nature04479; citation_firstpage=937; citation_lastpage=945; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Cosmid: a web-based tool for identifying and validating CRISPR/Cas off-target sites; citation_author=T.J. Cradick; citation_author=P. Qiu; citation_author=C.M. Lee; citation_author=E.J. Fine; citation_author=G. Bao; citation_journal_title=Mol. Ther. Nucleic Acids; citation_volume=3; citation_doi=10.1038/mtna.2014.64; citation_firstpage=e214; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Targeting a GFP reporter gene to the MIXL1 locus of human embryonic stem cells identifies human primitive streak-like cells and enables isolation of primitive hematopoietic precursors; citation_author=R.P. Davis; citation_author=E.S. Ng; citation_author=M. Costa; citation_author=A.K. Mossman; citation_author=K. Sourris; citation_author=A.G. Elefanty; citation_author=E.G. Stanley; citation_journal_title=Blood; citation_volume=111; citation_firstpage=1876; citation_lastpage=1884; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Icy: an open BioImage informatics platform for extended reproducible research; citation_author=F. de Chaumont; citation_author=S. Dallongeville; citation_author=N. Chenouard; citation_author=N. Hervé; citation_author=S. Pop; citation_author=T. Provoost; citation_author=V. Meas-Yedid; citation_author=P. Pankajakshan; citation_author=T. Lecomte; citation_author=Y. Le Montagner; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_doi=10.1038/nmeth.2075; citation_firstpage=690; citation_lastpage=696; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Transcriptional control of endothelial cell development; citation_author=S. De Val; citation_author=B.L. Black; citation_journal_title=Dev. Cell; citation_volume=16; citation_doi=10.1016/j.devcel.2009.01.014; citation_firstpage=180; citation_lastpage=195; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Comparison of the pathogenicity of Nipah virus isolates from Bangladesh and Malaysia in the Syrian hamster; citation_author=B.L. DeBuysscher; citation_author=E. de Wit; citation_author=V.J. Munster; citation_author=D. Scott; citation_author=H. Feldmann; citation_author=J. Prescott; citation_journal_title=PLoS Negl. Trop. Dis.; citation_volume=7; citation_doi=10.1371/journal.pntd.0002024; citation_firstpage=e2024; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages; citation_author=A. Ditadi; citation_author=C.M. Sturgeon; citation_author=J. Tober; citation_author=G. Awong; citation_author=M. Kennedy; citation_author=A.D. Yzaguirre; citation_author=L. Azzola; citation_author=E.S. Ng; citation_author=E.G. Stanley; citation_author=D.L. French; citation_author=et al.; citation_journal_title=Nat. Cell Biol.; citation_volume=17; citation_doi=10.1038/ncb3161; citation_firstpage=580; citation_lastpage=591; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Stem cells: the new “model organism”; citation_author=D.G. Drubin; citation_author=A.A. Hyman; citation_journal_title=Mol. Biol. Cell; citation_volume=28; citation_doi=10.1091/mbc.E17-03-0183; citation_firstpage=1409; citation_lastpage=1411; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Dosage-sensitive requirement for mouse Dll4 in artery development; citation_author=A. Duarte; citation_author=M. Hirashima; citation_author=R. Benedito; citation_author=A. Trindade; citation_author=P. Diniz; citation_author=E. Bekman; citation_author=L. Costa; citation_author=D. Henrique; citation_author=J. Rossant; citation_journal_title=Genes Dev.; citation_volume=18; citation_doi=10.1101/gad.1239004; citation_firstpage=2474; citation_lastpage=2478; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=The growth of bacteriophage; citation_author=E.L. Ellis; citation_author=M. Delbrück; citation_journal_title=J.&nbsp;Gen. Physiol.; citation_volume=22; citation_doi=10.1085/jgp.22.3.365; citation_firstpage=365; citation_lastpage=384; citation_publication_date=1939; "><meta name="citation_reference" content="citation_title=Cytokine induction in Nipah virus-infected primary human and porcine bronchial epithelial cells; citation_author=M. Elvert; citation_author=L. Sauerhering; citation_author=A. Maisner; citation_journal_title=J.&nbsp;Infect. Dis.; citation_volume=221; citation_doi=10.1093/infdis/jiz455; citation_firstpage=S395; citation_lastpage=S400; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Selective receptor expression restricts Nipah virus infection of endothelial cells; citation_author=S. Erbar; citation_author=S. Diederich; citation_author=A. Maisner; citation_journal_title=Virol. J.; citation_volume=5; citation_doi=10.1186/1743-422X-5-142; citation_firstpage=142; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Molecular regulation of arteriovenous endothelial cell specification; citation_author=J. Fang; citation_author=K. Hirschi; citation_journal_title=F1000Res; citation_volume=8; citation_doi=10.12688/f1000research.16701.1; citation_firstpage=1208; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The molecular regulation of arteriovenous specification and maintenance; citation_author=J.E. Fish; citation_author=J.D. Wythe; citation_journal_title=Dev. Dyn.; citation_volume=244; citation_doi=10.1002/dvdy.24252; citation_firstpage=391; citation_lastpage=409; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A critical look: challenges in differentiating human pluripotent stem cells into desired cell types and organoids; citation_author=J.L. Fowler; citation_author=L.T. Ang; citation_author=K.M. Loh; citation_journal_title=Wiley Interdiscip. Rev.: Dev. Biol.; citation_volume=113; citation_doi=10.1002/wdev.368; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Generation of functional liver sinusoidal endothelial cells from human pluripotent stem-cell-derived venous angioblasts; citation_author=B.K. Gage; citation_author=J.C. Liu; citation_author=B.T. Innes; citation_author=S.A. MacParland; citation_author=I.D. McGilvray; citation_author=G.D. Bader; citation_author=G.M. Keller; citation_journal_title=Cell Stem Cell; citation_volume=27; citation_doi=10.1016/j.stem.2020.06.007; citation_firstpage=254; citation_lastpage=269.e9; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Haploinsufficiency of delta-like 4 ligand results in embryonic lethality due to major defects in arterial and vascular development; citation_author=N.W. Gale; citation_author=M.G. Dominguez; citation_author=I. Noguera; citation_author=L. Pan; citation_author=V. Hughes; citation_author=D.M. Valenzuela; citation_author=A.J. Murphy; citation_author=N.C. Adams; citation_author=H.C. Lin; citation_author=J. Holash; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=15949; citation_lastpage=15954; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Development of an acute and highly pathogenic nonhuman primate model of Nipah virus infection; citation_author=T.W. Geisbert; citation_author=K.M. Daddario-DiCaprio; citation_author=A.C. Hickey; citation_author=M.A. Smith; citation_author=Y.-P. Chan; citation_author=L.-F. Wang; citation_author=J.J. Mattapallil; citation_author=J.B. Geisbert; citation_author=K.N. Bossart; citation_author=C.C. Broder; citation_journal_title=PLoS One; citation_volume=5; citation_doi=10.1371/journal.pone.0010690; citation_firstpage=e10690; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Purification and direct transformation of epithelial progenitor cells from primary human prostate; citation_author=A.S. Goldstein; citation_author=J.M. Drake; citation_author=D.L. Burnes; citation_author=D.S. Finley; citation_author=H. Zhang; citation_author=R.E. Reiter; citation_author=J. Huang; citation_author=O.N. Witte; citation_journal_title=Nat. Protoc.; citation_volume=6; citation_doi=10.1038/nprot.2011.317; citation_firstpage=656; citation_lastpage=667; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Medical countermeasures against henipaviruses: a review and public health perspective; citation_author=R. Gómez Román; citation_author=N. Tornieporth; citation_author=N.G. Cherian; citation_author=A.C. Shurtleff; citation_author=M. L'Azou Jackson; citation_author=D. Yeskey; citation_author=A. Hacker; citation_author=E. Mungai; citation_author=T.T. Le; citation_journal_title=Lancet Infect. Dis.; citation_volume=22; citation_doi=10.1016/S1473-3099(21)00400-X; citation_firstpage=e13; citation_lastpage=e27; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression; citation_author=C. Hafemeister; citation_author=R. Satija; citation_journal_title=Genome Biol.; citation_volume=20; citation_doi=10.1186/s13059-019-1874-1; citation_firstpage=296; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Inflammatory responses and barrier function of endothelial cells derived from human induced pluripotent stem cells; citation_author=O.V. Halaidych; citation_author=C. Freund; citation_author=F. van den Hil; citation_author=D.C.F. Salvatori; citation_author=M. Riminucci; citation_author=C.L. Mummery; citation_author=V.V. Orlova; citation_journal_title=Stem Cell Rep.; citation_volume=10; citation_doi=10.1016/j.stemcr.2018.03.012; citation_firstpage=1642; citation_lastpage=1656; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Molecular characterization of Nipah virus, a newly emergent paramyxovirus; citation_author=B.H. Harcourt; citation_author=A. Tamin; citation_author=T.G. Ksiazek; citation_author=P.E. Rollin; citation_author=L.J. Anderson; citation_author=W.J. Bellini; citation_author=P.A. Rota; citation_journal_title=Virology; citation_volume=271; citation_doi=10.1006/viro.2000.0340; citation_firstpage=334; citation_lastpage=349; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Human stem cell models to study host-virus interactions in the central nervous system; citation_author=O. Harschnitz; citation_author=L. Studer; citation_journal_title=Nat. Rev. Immunol.; citation_volume=21; citation_doi=10.1038/s41577-020-00474-y; citation_firstpage=441; citation_lastpage=453; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Artery/vein specification is governed by opposing phosphatidylinositol-3 kinase and MAP kinase/ERK signaling; citation_author=C.C. Hong; citation_author=Q.P. Peterson; citation_author=J.Y. Hong; citation_author=R.T. Peterson; citation_journal_title=Curr. Biol.; citation_volume=16; citation_firstpage=1366; citation_lastpage=1372; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Single-cell bioluminescence imaging of deep tissue in freely moving animals; citation_author=S. Iwano; citation_author=M. Sugiyama; citation_author=H. Hama; citation_author=A. Watakabe; citation_author=N. Hasegawa; citation_author=T. Kuchimaru; citation_author=K.Z. Tanaka; citation_author=M. Takahashi; citation_author=Y. Ishida; citation_author=J. Hata; citation_author=et al.; citation_journal_title=Science; citation_volume=359; citation_doi=10.1126/science.aaq1067; citation_firstpage=935; citation_lastpage=939; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Paracrine and autocrine mechanisms of apelin signaling govern embryonic and tumor angiogenesis; citation_author=R.E. Kälin; citation_author=M.P. Kretz; citation_author=A.M. Meyer; citation_author=A. Kispert; citation_author=F.L. Heppner; citation_author=A.W. Brändli; citation_journal_title=Dev. Biol.; citation_volume=305; citation_doi=10.1016/j.ydbio.2007.03.004; citation_firstpage=599; citation_lastpage=614; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Single-cell transcriptome atlas of murine endothelial cells; citation_author=J. Kalucka; citation_author=L.P.M.H. de Rooij; citation_author=J. Goveia; citation_author=K. Rohlenova; citation_author=S.J. Dumas; citation_author=E. Meta; citation_author=N.V. Conchinha; citation_author=F. Taverna; citation_author=L.-A. Teuwen; citation_author=K. Veys; citation_author=et al.; citation_journal_title=Cell; citation_volume=180; citation_doi=10.1016/j.cell.2020.01.015; citation_firstpage=764; citation_lastpage=779.e20; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=APJ regulates parallel alignment of arteries and veins in the skin; citation_author=H. Kidoya; citation_author=H. Naito; citation_author=F. Muramatsu; citation_author=D. Yamakawa; citation_author=W. Jia; citation_author=M. Ikawa; citation_author=T. Sonobe; citation_author=H. Tsuchimochi; citation_author=M. Shirai; citation_author=R.H. Adams; citation_author=et al.; citation_journal_title=Dev. Cell; citation_volume=33; citation_doi=10.1016/j.devcel.2015.02.024; citation_firstpage=247; citation_lastpage=259; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Haploinsufficient lethality and formation of arteriovenous malformations in Notch pathway mutants; citation_author=L.T. Krebs; citation_author=J.R. Shutter; citation_author=K. Tanigaki; citation_author=T. Honjo; citation_author=K.L. Stark; citation_author=T. Gridley; citation_journal_title=Genes Dev.; citation_volume=18; citation_doi=10.1101/gad.1239204; citation_firstpage=2469; citation_lastpage=2473; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Trim galore; citation_author=F. Krueger; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Enrichr: a comprehensive gene set enrichment analysis web server 2016 update; citation_author=M.V. Kuleshov; citation_author=M.R. Jones; citation_author=A.D. Rouillard; citation_author=N.F. Fernandez; citation_author=Q. Duan; citation_author=Z. Wang; citation_author=S. Koplev; citation_author=S.L. Jenkins; citation_author=K.M. Jagodnik; citation_author=A. Lachmann; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_doi=10.1093/nar/gkw377; citation_firstpage=W90; citation_lastpage=W97; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Conversion of human fibroblasts to angioblast-like progenitor cells; citation_author=L. Kurian; citation_author=I. Sancho-Martinez; citation_author=E. Nivet; citation_author=A. Aguirre; citation_author=K. Moon; citation_author=C. Pendaries; citation_author=C. Volle-Challier; citation_author=F. Bono; citation_author=J.-M. Herbert; citation_author=J. Pulecio; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=77; citation_lastpage=83; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems; citation_author=Y.K. Kurokawa; citation_author=R.T. Yin; citation_author=M.R. Shang; citation_author=V.S. Shirure; citation_author=M.L. Moya; citation_author=S.C. George; citation_journal_title=Tissue Eng. Part C Methods; citation_volume=23; citation_doi=10.1089/ten.TEC.2017.0133; citation_firstpage=474; citation_lastpage=484; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing; citation_author=K. Labun; citation_author=T.G. Montague; citation_author=M. Krause; citation_author=Y.N. Torres Cleuren; citation_author=H. Tjeldnes; citation_author=E. Valen; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_doi=10.1093/nar/gkz365; citation_firstpage=W171; citation_lastpage=W174; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=voom: precision weights unlock linear model analysis tools for RNA-seq read counts; citation_author=C.W. Law; citation_author=Y. Chen; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Genome Biol.; citation_volume=15; citation_doi=10.1186/gb-2014-15-2-r29; citation_firstpage=R29; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Notch signaling is required for arterial-venous differentiation during embryonic vascular development; citation_author=N.D. Lawson; citation_author=N. Scheer; citation_author=V.N. Pham; citation_author=C.H. Kim; citation_author=A.B. Chitnis; citation_author=J.A. Campos-Ortega; citation_author=B.M. Weinstein; citation_journal_title=Development; citation_volume=128; citation_firstpage=3675; citation_lastpage=3683; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=sonic hedgehog and vascular endothelial growth factor act upstream of the Notch pathway during arterial endothelial differentiation; citation_author=N.D. Lawson; citation_author=A.M. Vogel; citation_author=B.M. Weinstein; citation_journal_title=Dev. Cell; citation_volume=3; citation_firstpage=127; citation_lastpage=136; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling; citation_author=X. Lian; citation_author=X. Bao; citation_author=A. Al-Ahmad; citation_author=J. Liu; citation_author=Y. Wu; citation_author=W. Dong; citation_author=K.K. Dunn; citation_author=E.V. Shusta; citation_author=S.P. Palecek; citation_journal_title=Stem Cell Rep.; citation_volume=3; citation_doi=10.1016/j.stemcr.2014.09.005; citation_firstpage=804; citation_lastpage=816; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Artery and vein formation: a tug of war between different forces; citation_author=F.-J. Lin; citation_author=M.-J. Tsai; citation_author=S.Y. Tsai; citation_journal_title=EMBO Rep.; citation_volume=8; citation_firstpage=920; citation_lastpage=924; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Differentiation, evaluation, and application of human induced pluripotent stem cell-derived endothelial cells; citation_author=Y. Lin; citation_author=C.-H. Gil; citation_author=M.C. Yoder; citation_journal_title=Arterioscler. Thromb. Vasc. Biol.; citation_volume=37; citation_doi=10.1161/ATVBAHA.117.309962; citation_firstpage=2014; citation_lastpage=2025; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Characterization of the antiviral and inflammatory responses against Nipah virus in endothelial cells and neurons; citation_author=M.K. Lo; citation_author=D. Miller; citation_author=M. Aljofan; citation_author=B.A. Mungall; citation_author=P.E. Rollin; citation_author=W.J. Bellini; citation_author=P.A. Rota; citation_journal_title=Virology; citation_volume=404; citation_doi=10.1016/j.virol.2010.05.005; citation_firstpage=78; citation_lastpage=88; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations; citation_author=K.M. Loh; citation_author=L.T. Ang; citation_author=J. Zhang; citation_author=V. Kumar; citation_author=J. Ang; citation_author=J.Q. Auyeong; citation_author=K.L. Lee; citation_author=S.H. Choo; citation_author=C.Y.Y. Lim; citation_author=M. Nichane; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=14; citation_firstpage=237; citation_lastpage=252; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types; citation_author=K.M. Loh; citation_author=A. Chen; citation_author=P.W. Koh; citation_author=T.Z. Deng; citation_author=R. Sinha; citation_author=J.M. Tsai; citation_author=A.A. Barkal; citation_author=K.Y. Shen; citation_author=R. Jain; citation_author=R.M. Morganti; citation_author=et al.; citation_journal_title=Cell; citation_volume=166; citation_doi=10.1016/j.cell.2016.06.011; citation_firstpage=451; citation_lastpage=467; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Sendai virus infection induces efficient adaptive immunity independently of type I interferons; citation_author=C.B. López; citation_author=J.S. Yount; citation_author=T. Hermesh; citation_author=T.M. Moran; citation_journal_title=J.&nbsp;Virol.; citation_volume=80; citation_doi=10.1128/JVI.80.9.4538-4545.2006; citation_firstpage=4538; citation_lastpage=4545; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=M.I. Love; citation_author=W. Huber; citation_author=S. Anders; citation_journal_title=Genome Biol.; citation_volume=15; citation_doi=10.1186/s13059-014-0550-8; citation_firstpage=550; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The netrin receptor UNC5B mediates guidance events controlling morphogenesis of the vascular system; citation_author=X. Lu; citation_author=F. le Noble; citation_author=L. Yuan; citation_author=Q. Jiang; citation_author=B. De Lafarge; citation_author=D. Sugiyama; citation_author=C. Bréant; citation_author=F. Claes; citation_author=F. De Smet; citation_author=J.-L. Thomas; citation_author=et al.; citation_journal_title=Nature; citation_volume=432; citation_doi=10.1038/nature03080; citation_firstpage=179; citation_lastpage=186; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Hendra virus: an emerging paramyxovirus in Australia; citation_author=S. Mahalingam; citation_author=L.J. Herrero; citation_author=E.G. Playford; citation_author=K. Spann; citation_author=B. Herring; citation_author=M.S. Rolph; citation_author=D. Middleton; citation_author=B. McCall; citation_author=H. Field; citation_author=L.F. Wang; citation_journal_title=Lancet Infect. Dis.; citation_volume=12; citation_doi=10.1016/S1473-3099(12)70158-5; citation_firstpage=799; citation_lastpage=807; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination; citation_author=R.M. Martin; citation_author=K. Ikeda; citation_author=M.K. Cromer; citation_author=N. Uchida; citation_author=T. Nishimura; citation_author=R. Romano; citation_author=A.J. Tong; citation_author=V.T. Lemgart; citation_author=J. Camarena; citation_author=M. Pavel-Dinu; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=24; citation_doi=10.1016/j.stem.2019.04.001; citation_firstpage=821; citation_lastpage=828.e5; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Inhibition of apoptosis overcomes stage-related compatibility barriers to chimera formation in mouse embryos; citation_author=H. Masaki; citation_author=M. Kato-Itoh; citation_author=Y. Takahashi; citation_author=A. Umino; citation_author=H. Sato; citation_author=K. Ito; citation_author=A. Yanagida; citation_author=T. Nishimura; citation_author=T. Yamaguchi; citation_author=M. Hirabayashi; citation_author=et al.; citation_journal_title=Cell Stem Cell; citation_volume=19; citation_doi=10.1016/j.stem.2016.10.013; citation_firstpage=587; citation_lastpage=592; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Lethal Nipah virus infection induces rapid overexpression of CXCL10; citation_author=C. Mathieu; citation_author=V. Guillaume; citation_author=A. Sabine; citation_author=K.C. Ong; citation_author=K.T. Wong; citation_author=C. Legras-Lachuer; citation_author=B. Horvat; citation_journal_title=PLoS One; citation_volume=7; citation_doi=10.1371/journal.pone.0032157; citation_firstpage=e32157; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Transcriptional dynamics of pluripotent stem cell-derived endothelial cell differentiation revealed by single-cell RNA sequencing; citation_author=I.R. McCracken; citation_author=R.S. Taylor; citation_author=F.O. Kok; citation_author=F. de la Cuesta; citation_author=R. Dobie; citation_author=B.E.P. Henderson; citation_author=J.C. Mountford; citation_author=A. Caudrillier; citation_author=N.C. Henderson; citation_author=C.P. Ponting; citation_author=A.H. Baker; citation_journal_title=Eur. Heart J.; citation_volume=385; citation_doi=10.1093/eurheartj/ehz351; citation_firstpage=9963117; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The WHO R&amp;D Blueprint: 2018 review of emerging infectious diseases requiring urgent research and development efforts; citation_author=M.S. Mehand; citation_author=F. Al-Shorbaji; citation_author=P. Millett; citation_author=B. Murgue; citation_journal_title=Antiviral Res.; citation_volume=159; citation_doi=10.1016/j.antiviral.2018.09.009; citation_firstpage=63; citation_lastpage=67; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=MAP and kinesin-dependent nuclear positioning is required for skeletal muscle function; citation_author=T. Metzger; citation_author=V. Gache; citation_author=M. Xu; citation_author=B. Cadot; citation_author=E.S. Folker; citation_author=B.E. Richardson; citation_author=E.R. Gomes; citation_author=M.K. Baylies; citation_journal_title=Nature; citation_volume=484; citation_doi=10.1038/nature10914; citation_firstpage=120; citation_lastpage=124; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Detailed morphological characterisation of Hendra virus infection of different cell types using super-resolution and conventional imaging; citation_author=P. Monaghan; citation_author=D. Green; citation_author=J. Pallister; citation_author=R. Klein; citation_author=J. White; citation_author=C. Williams; citation_author=P. McMillan; citation_author=L. Tilley; citation_author=M. Lampe; citation_author=P. Hawes; citation_author=L.-F. Wang; citation_journal_title=Virol. J.; citation_volume=11; citation_doi=10.1186/s12985-014-0200-5; citation_firstpage=200; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A morbillivirus that caused fatal disease in horses and humans; citation_author=K. Murray; citation_author=P. Selleck; citation_author=P. Hooper; citation_author=A. Hyatt; citation_author=A. Gould; citation_author=L. Gleeson; citation_author=H. Westbury; citation_author=L. Hiley; citation_author=L. Selvey; citation_author=B. Rodwell; citation_journal_title=Science; citation_volume=268; citation_doi=10.1126/science.7701348; citation_firstpage=94; citation_lastpage=97; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Optimized fibrin gel bead assay for the study of angiogenesis; citation_author=M.N. Nakatsu; citation_author=J. Davis; citation_author=C.C.W. Hughes; citation_journal_title=J.&nbsp;Vis. Exp.; citation_volume=3; citation_firstpage=186; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Single amino acid changes in the Nipah and Hendra virus attachment glycoproteins distinguish ephrinB2 from ephrinB3 usage; citation_author=O.A. Negrete; citation_author=D. Chu; citation_author=H.C. Aguilar; citation_author=B. Lee; citation_journal_title=J.&nbsp;Virol.; citation_volume=81; citation_doi=10.1128/JVI.00999-07; citation_firstpage=10804; citation_lastpage=10814; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=EphrinB2 is the entry receptor for Nipah virus, an emergent deadly paramyxovirus; citation_author=O.A. Negrete; citation_author=E.L. Levroney; citation_author=H.C. Aguilar; citation_author=A. Bertolotti-Ciarlet; citation_author=R. Nazarian; citation_author=S. Tajyar; citation_author=B. Lee; citation_journal_title=Nature; citation_volume=436; citation_doi=10.1038/nature03838; citation_firstpage=401; citation_lastpage=405; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=A comprehensive library of human transcription factors for cell fate engineering; citation_author=A.H.M. Ng; citation_author=P. Khoshakhlagh; citation_author=J.E. Rojo Arias; citation_author=G. Pasquini; citation_author=K. Wang; citation_author=A. Swiersy; citation_author=S.L. Shipman; citation_author=E. Appleton; citation_author=K. Kiaee; citation_author=R.E. Kohman; citation_author=et al.; citation_journal_title=Nat. Biotechnol.; citation_volume=51; citation_doi=10.1038/s41587-020-0742-6; citation_firstpage=987; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=The next generation of endothelial differentiation: tissue-specific ECs; citation_author=J. Nguyen; citation_author=Y.-Y. Lin; citation_author=S. Gerecht; citation_journal_title=Cell Stem Cell; citation_volume=28; citation_doi=10.1016/j.stem.2021.05.002; citation_firstpage=1188; citation_lastpage=1204; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Differentiation of human embryonic stem cells to endothelial progenitor cells on laminins in defined and xeno-free systems; citation_author=M.T.X. Nguyen; citation_author=E. Okina; citation_author=X. Chai; citation_author=K.H. Tan; citation_author=O. Hovatta; citation_author=S. Ghosh; citation_author=K. Tryggvason; citation_journal_title=Stem Cell Rep.; citation_volume=7; citation_doi=10.1016/j.stemcr.2016.08.017; citation_firstpage=802; citation_lastpage=816; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302; citation_author=K. Nishimura; citation_author=M. Ohtaka; citation_author=H. Takada; citation_author=A. Kurisaki; citation_author=N.V.K. Tran; citation_author=Y.T.H. Tran; citation_author=K. Hisatake; citation_author=M. Sano; citation_author=M. Nakanishi; citation_journal_title=Stem Cell Res.; citation_volume=23; citation_doi=10.1016/j.scr.2017.06.011; citation_firstpage=13; citation_lastpage=19; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Development of defective and persistent Sendai virus vector: a unique gene delivery/expression system ideal for cell reprogramming; citation_author=K. Nishimura; citation_author=M. Sano; citation_author=M. Ohtaka; citation_author=B. Furuta; citation_author=Y. Umemura; citation_author=Y. Nakajima; citation_author=Y. Ikehara; citation_author=T. Kobayashi; citation_author=H. Segawa; citation_author=S. Takayasu; citation_author=et al.; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=286; citation_doi=10.1074/jbc.M110.183780; citation_firstpage=4760; citation_lastpage=4771; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Differentiation of human pluripotent stem cells into functional endothelial cells in scalable suspension culture; citation_author=R. Olmer; citation_author=L. Engels; citation_author=A. Usman; citation_author=S. Menke; citation_author=M.N.H. Malik; citation_author=F. Pessler; citation_author=G. Göhring; citation_author=D. Bornhorst; citation_author=S. Bolten; citation_author=S. Abdelilah-Seyfried; citation_author=et al.; citation_journal_title=Stem Cell Rep.; citation_volume=10; citation_doi=10.1016/j.stemcr.2018.03.017; citation_firstpage=1657; citation_lastpage=1672; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=ShinyCell: simple and sharable visualisation of single-cell gene expression data; citation_author=J.F. Ouyang; citation_author=U.S. Kamaraj; citation_author=E.Y. Cao; citation_author=O.J.L. Rackham; citation_journal_title=Bioinformatics; citation_doi=10.1093/bioinformatics/btab209; citation_firstpage=btab209; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Large-scale single-cell RNA-seq reveals molecular signatures of heterogeneous populations of human induced pluripotent stem cell-derived endothelial cells; citation_author=D.T. Paik; citation_author=L. Tian; citation_author=J. Lee; citation_author=N. Sayed; citation_author=I.Y. Chen; citation_author=S. Rhee; citation_author=J.W. Rhee; citation_author=Y. Kim; citation_author=R.C. Wirka; citation_author=J.W. Buikema; citation_author=et al.; citation_journal_title=Circ. Res.; citation_volume=123; citation_doi=10.1161/CIRCRESAHA.118.312913; citation_firstpage=443; citation_lastpage=450; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations in&nbsp;vitro and bypass the liver sink when administered in&nbsp;vivo; citation_author=K. Palomares; citation_author=F. Vigant; citation_author=B. Van Handel; citation_author=O. Pernet; citation_author=K. Chikere; citation_author=P. Hong; citation_author=S.P. Sherman; citation_author=M. Patterson; citation_author=D.S. An; citation_author=W.E. Lowry; citation_author=et al.; citation_journal_title=J.&nbsp;Virol.; citation_volume=87; citation_doi=10.1128/JVI.02032-12; citation_firstpage=2094; citation_lastpage=2108; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential; citation_author=M.A. Park; citation_author=A. Kumar; citation_author=H.S. Jung; citation_author=G. Uenishi; citation_author=O.V. Moskvin; citation_author=J.A. Thomson; citation_author=I.I. Slukvin; citation_journal_title=Cell Rep.; citation_volume=23; citation_doi=10.1016/j.celrep.2018.04.092; citation_firstpage=2467; citation_lastpage=2481; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells; citation_author=C. Patsch; citation_author=L. Challet-Meylan; citation_author=E.C. Thoma; citation_author=E. Urich; citation_author=T. Heckel; citation_author=J.F. O’Sullivan; citation_author=S.J. Grainger; citation_author=F.G. Kapp; citation_author=L. Sun; citation_author=K. Christensen; citation_author=et al.; citation_journal_title=Nat. Cell Biol.; citation_volume=17; citation_doi=10.1038/ncb3205; citation_firstpage=994; citation_lastpage=1003; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Endothelial cells respond to the direction of mechanical stimuli through SMAD signaling to regulate coronary artery size; citation_author=A. Poduri; citation_author=A.H. Chang; citation_author=B. Raftrey; citation_author=S. Rhee; citation_author=M. Van; citation_author=K. Red-Horse; citation_journal_title=Development; citation_volume=144; citation_doi=10.1242/dev.150904; citation_firstpage=3241; citation_lastpage=3252; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Vascular heterogeneity and specialization in development and disease; citation_author=M. Potente; citation_author=T. Mäkinen; citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_volume=18; citation_doi=10.1038/nrm.2017.36; citation_firstpage=477; citation_lastpage=494; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Single-dose live-attenuated vesicular stomatitis virus-based vaccine protects African green monkeys from Nipah virus disease; citation_author=J. Prescott; citation_author=B.L. DeBuysscher; citation_author=F. Feldmann; citation_author=D.J. Gardner; citation_author=E. Haddock; citation_author=C. Martellaro; citation_author=D. Scott; citation_author=H. Feldmann; citation_journal_title=Vaccine; citation_volume=33; citation_doi=10.1016/j.vaccine.2015.03.089; citation_firstpage=2823; citation_lastpage=2829; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Angiocrine functions of organ-specific endothelial cells; citation_author=S. Rafii; citation_author=J.M. Butler; citation_author=B.-S. Ding; citation_journal_title=Nature; citation_volume=529; citation_doi=10.1038/nature17040; citation_firstpage=316; citation_lastpage=325; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Veins and arteries Build Hierarchical Branching Patterns Differently: Bottom-Up versus Top-Down; citation_author=K. Red-Horse; citation_author=A.F. Siekmann; citation_journal_title=BioEssays; citation_volume=41; citation_doi=10.1002/bies.201800198; citation_firstpage=e1800198; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=GenePattern 2.0; citation_author=M. Reich; citation_author=T. Liefeld; citation_author=J. Gould; citation_author=J. Lerner; citation_author=P. Tamayo; citation_author=J.P. Mesirov; citation_journal_title=Nat. Genet.; citation_volume=38; citation_doi=10.1038/ng0506-500; citation_firstpage=500; citation_lastpage=501; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=limma powers differential expression analyses for RNA-sequencing and microarray studies; citation_author=M.E. Ritchie; citation_author=B. Phipson; citation_author=D. Wu; citation_author=Y. Hu; citation_author=C.W. Law; citation_author=W. Shi; citation_author=G.K. Smyth; citation_journal_title=Nucleic Acids Res.; citation_volume=43; citation_doi=10.1093/nar/gkv007; citation_firstpage=e47; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M.D. Robinson; citation_author=D.J. McCarthy; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=26; citation_doi=10.1093/bioinformatics/btp616; citation_firstpage=139; citation_lastpage=140; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Nipah virus V protein evades alpha and gamma interferons by preventing STAT1 and STAT2 activation and nuclear accumulation; citation_author=J.J. Rodriguez; citation_author=J.-P. Parisien; citation_author=C.M. Horvath; citation_journal_title=J.&nbsp;Virol.; citation_volume=76; citation_doi=10.1128/jvi.76.22.11476-11483.2002; citation_firstpage=11476; citation_lastpage=11483; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Functional characterization of iPSC-derived arterial- and venous-like endothelial cells; citation_author=S. Rosa; citation_author=C. Praça; citation_author=P.R. Pitrez; citation_author=P.J. Gouveia; citation_author=X.L. Aranguren; citation_author=L. Ricotti; citation_author=L.S. Ferreira; citation_journal_title=Sci. Rep.; citation_volume=9; citation_doi=10.1038/s41598-019-40417-9; citation_firstpage=3826; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Redundant roles of Sox17 and Sox18 in early cardiovascular development of mouse embryos; citation_author=Y. Sakamoto; citation_author=K. Hara; citation_author=M. Kanai-Azuma; citation_author=T. Matsui; citation_author=Y. Miura; citation_author=N. Tsunekawa; citation_author=M. Kurohmaru; citation_author=Y. Saijoh; citation_author=P. Koopman; citation_author=Y. Kanai; citation_journal_title=Biochem. Biophys. Res. Commun.; citation_volume=360; citation_firstpage=539; citation_lastpage=544; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J. Schindelin; citation_author=I. Arganda-Carreras; citation_author=E. Frise; citation_author=V. Kaynig; citation_author=M. Longair; citation_author=T. Pietzsch; citation_author=S. Preibisch; citation_author=C. Rueden; citation_author=S. Saalfeld; citation_author=B. Schmid; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=9; citation_doi=10.1038/nmeth.2019; citation_firstpage=676; citation_lastpage=682; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Differential innate immune responses elicited by Nipah virus and cedar virus correlate with disparate in&nbsp;vivo pathogenesis in hamsters; citation_author=T. Schountz; citation_author=C. Campbell; citation_author=K. Wagner; citation_author=J. Rovnak; citation_author=C. Martellaro; citation_author=B.L. DeBuysscher; citation_author=H. Feldmann; citation_author=J. Prescott; citation_journal_title=Viruses; citation_volume=11; citation_doi=10.3390/v11030291; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Failure of blood-island formation and vasculogenesis in Flk-1-deficient mice; citation_author=F. Shalaby; citation_author=J. Rossant; citation_author=T.P. Yamaguchi; citation_author=M. Gertsenstein; citation_author=X.F. Wu; citation_author=M.L. Breitman; citation_author=A.C. Schuh; citation_journal_title=Nature; citation_volume=376; citation_firstpage=62; citation_lastpage=66; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Fundamental properties of the mammalian innate immune system revealed by multispecies comparison of type I interferon responses; citation_author=A.E. Shaw; citation_author=J. Hughes; citation_author=Q. Gu; citation_author=A. Behdenna; citation_author=J.B. Singer; citation_author=T. Dennis; citation_author=R.J. Orton; citation_author=M. Varela; citation_author=R.J. Gifford; citation_author=S.J. Wilson; citation_author=M. Palmarini; citation_journal_title=PLoS Biol.; citation_volume=15; citation_doi=10.1371/journal.pbio.2004086; citation_firstpage=e2004086; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Transcriptional profiling of VEGF-A and VEGF-C target genes in lymphatic endothelium reveals endothelial-specific molecule-1 as a novel mediator of lymphangiogenesis; citation_author=J.W. Shin; citation_author=R. Huggenberger; citation_author=M. Detmar; citation_journal_title=Blood; citation_volume=112; citation_doi=10.1182/blood-2008-05-156331; citation_firstpage=2318; citation_lastpage=2326; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Modeling multi-organ infection by SARS-CoV-2&nbsp;using stem cell technology; citation_author=C.R. Simoneau; citation_author=M. Ott; citation_journal_title=Cell Stem Cell; citation_volume=27; citation_doi=10.1016/j.stem.2020.11.012; citation_firstpage=859; citation_lastpage=868; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions; citation_author=G. Sriram; citation_author=J.Y. Tan; citation_author=I. Islam; citation_author=A.J. Rufaihah; citation_author=T. Cao; citation_journal_title=Stem Cell Res. Ther.; citation_volume=6; citation_doi=10.1186/s13287-015-0260-5; citation_firstpage=261; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Comprehensive integration of single-cell data; citation_author=T. Stuart; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=C. Hafemeister; citation_author=E. Papalexi; citation_author=W.M. Mauck; citation_author=Y. Hao; citation_author=M. Stoeckius; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Cell; citation_volume=177; citation_doi=10.1016/j.cell.2019.05.031; citation_firstpage=1888; citation_lastpage=1902.e21; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Placental labyrinth formation in mice requires endothelial FLRT2/UNC5B signaling; citation_author=I. Tai-Nagara; citation_author=Y. Yoshikawa; citation_author=N. Numata; citation_author=T. Ando; citation_author=K. Okabe; citation_author=Y. Sugiura; citation_author=M. Ieda; citation_author=N. Takakura; citation_author=O. Nakagawa; citation_author=B. Zhou; citation_author=et al.; citation_journal_title=Development; citation_volume=144; citation_doi=10.1242/dev.149757; citation_firstpage=2392; citation_lastpage=2401; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Resetting transcription factor control circuitry toward ground-state pluripotency in human; citation_author=Y. Takashima; citation_author=G. Guo; citation_author=R. Loos; citation_author=J. Nichols; citation_author=G. Ficz; citation_author=F. Krueger; citation_author=D. Oxley; citation_author=F. Santos; citation_author=J. Clarke; citation_author=W. Mansfield; citation_author=et al.; citation_journal_title=Cell; citation_volume=158; citation_doi=10.1016/j.cell.2014.08.029; citation_firstpage=1254; citation_lastpage=1269; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=TrackMate: an open and extensible platform for single-particle tracking; citation_author=J.Y. Tinevez; citation_author=N. Perry; citation_author=J. Schindelin; citation_author=G.M. Hoopes; citation_author=G.D. Reynolds; citation_author=E. Laplantine; citation_author=S.Y. Bednarek; citation_author=S.L. Shorte; citation_author=K.W. Eliceiri; citation_journal_title=Methods; citation_volume=115; citation_doi=10.1016/j.ymeth.2016.09.016; citation_firstpage=80; citation_lastpage=90; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A molecular cell atlas of the human lung from single-cell RNA sequencing; citation_author=K.J. Travaglini; citation_author=A.N. Nabhan; citation_author=L. Penland; citation_author=R. Sinha; citation_author=A. Gillich; citation_author=R.V. Sit; citation_author=S. Chang; citation_author=S.D. Conley; citation_author=Y. Mori; citation_author=J. Seita; citation_author=et al.; citation_journal_title=Nature; citation_volume=587; citation_doi=10.1038/s41586-020-2922-4; citation_firstpage=619; citation_lastpage=625; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Interferon signaling remains functional during Henipavirus infection of human cell lines; citation_author=E.R. Virtue; citation_author=G.A. Marsh; citation_author=L.F. Wang; citation_journal_title=J.&nbsp;Virol.; citation_volume=85; citation_doi=10.1128/JVI.02412-10; citation_firstpage=4031; citation_lastpage=4034; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Molecular distinction and angiogenic interaction between embryonic arteries and veins revealed by ephrin-B2 and its receptor Eph-B4; citation_author=H.U. Wang; citation_author=Z.F. Chen; citation_author=D.J. Anderson; citation_journal_title=Cell; citation_volume=93; citation_firstpage=741; citation_lastpage=753; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Robust differentiation of human pluripotent stem cells into endothelial cells via temporal modulation of ETV2 with modified mRNA; citation_author=K. Wang; citation_author=R.-Z. Lin; citation_author=X. Hong; citation_author=A.H. Ng; citation_author=C.N. Lee; citation_author=J. Neumeyer; citation_author=G. Wang; citation_author=X. Wang; citation_author=M. Ma; citation_author=W.T. Pu; citation_author=et al.; citation_journal_title=Sci. Adv.; citation_volume=6; citation_doi=10.1126/sciadv.aba7606; citation_firstpage=eaba7606; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis; citation_author=H. Wickham; citation_publication_date=2016; citation_publisher=Springer-Verlag New York; "><meta name="citation_reference" content="citation_title=Welcome to the Tidyverse; citation_author=H. Wickham; citation_author=M. Averick; citation_author=J. Bryan; citation_author=W. Chang; citation_author=L. McGowan; citation_author=R. François; citation_author=G. Grolemund; citation_author=A. Hayes; citation_author=L. Henry; citation_author=J. Hester; citation_author=et al.; citation_journal_title=J.&nbsp;Open Source Software; citation_volume=4; citation_doi=10.21105/joss.01686; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Dplyr: a grammar of data manipulation; citation_author=H. Wickham; citation_author=R. François; citation_author=L. Henry; citation_author=K. Müller; citation_publication_date=2022; "><meta name="citation_reference" content="citation_title=Nipah virus infection: pathology and pathogenesis of an emerging paramyxoviral zoonosis; citation_author=K.T. Wong; citation_author=W.-J. Shieh; citation_author=S. Kumar; citation_author=K. Norain; citation_author=W. Abdullah; citation_author=J. Guarner; citation_author=C.S. Goldsmith; citation_author=K.B. Chua; citation_author=S.K. Lam; citation_author=C.T. Tan; citation_author=et al.; citation_journal_title=Am. J. Pathol.; citation_volume=161; citation_doi=10.1016/S0002-9440(10)64493-8; citation_firstpage=2153; citation_lastpage=2167; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Arteries are formed by vein-derived endothelial tip cells; citation_author=C. Xu; citation_author=S.S. Hasan; citation_author=I. Schmidt; citation_author=S.F. Rocha; citation_author=M.E. Pitulescu; citation_author=J. Bussmann; citation_author=D. Meyen; citation_author=E. Raz; citation_author=R.H. Adams; citation_author=A.F. Siekmann; citation_journal_title=Nat. Commun.; citation_volume=5; citation_doi=10.1038/ncomms6758; citation_firstpage=5758; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A single amino acid substitution within the paramyxovirus Sendai virus nucleoprotein is a critical determinant for production of interferon-beta-inducing copyback-type defective interfering genomes; citation_author=A. Yoshida; citation_author=R. Kawabata; citation_author=T. Honda; citation_author=K. Sakai; citation_author=Y. Ami; citation_author=T. Sakaguchi; citation_author=T. Irie; citation_journal_title=J.&nbsp;Virol.; citation_volume=92; citation_doi=10.1128/JVI.02094-17; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing; citation_author=Y. Zeng; citation_author=J. He; citation_author=Z. Bai; citation_author=Z. Li; citation_author=Y. Gong; citation_author=C. Liu; citation_author=Y. Ni; citation_author=J. Du; citation_author=C. Ma; citation_author=L. Bian; citation_author=et al.; citation_journal_title=Cell Res.; citation_volume=29; citation_doi=10.1038/s41422-019-0228-6; citation_firstpage=881; citation_lastpage=894; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America; citation_author=J. Zhang; citation_author=L.-F. Chu; citation_author=Z. Hou; citation_author=M.P. Schwartz; citation_author=T. Hacker; citation_author=V. Vickerman; citation_author=S. Swanson; citation_author=N. Leng; citation_author=B.K. Nguyen; citation_author=A. Elwell; citation_author=et al.; citation_doi=10.1073/pnas.1702295114; citation_firstpage=201702295; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Massively parallel digital transcriptional profiling of single cells; citation_author=G.X.Y. Zheng; citation_author=J.M. Terry; citation_author=P. Belgrader; citation_author=P. Ryvkin; citation_author=Z.W. Bent; citation_author=R. Wilson; citation_author=S.B. Ziraldo; citation_author=T.D. Wheeler; citation_author=G.P. McDermott; citation_author=J. Zhu; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=8; citation_doi=10.1038/ncomms14049; citation_firstpage=14049; citation_publication_date=2017; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Stem cell research endeavors to generate specific subtypes of classically defined &quot;cell types.&quot; Here, we generate >90% pure human artery or vein endothelial cells from pluripotent stem cells within 3–4&nbsp;days. We specified artery cells by inhibiting vein-specifying signals and vice versa. These cells modeled viral infection of human vasculature by Nipah and Hendra viruses, which are extraordinarily deadly (∼57%–59% fatality rate) and require biosafety-level-4 containment. Generating pure populations of artery and vein cells highlighted that Nipah and Hendra viruses preferentially infected arteries; arteries expressed higher levels of their viral-entry receptor. Virally infected artery cells fused into syncytia containing up to 23 nuclei, which rapidly died. Despite infecting arteries and occupying ∼6%–17% of their transcriptome, Nipah and Hendra largely eluded innate immune detection, minimally eliciting interferon signaling. We thus efficiently generate artery and vein cells, introduce stem-cell-based toolkits for biosafety-level-4 virology, and explore the arterial tropism and cellular effects of Nipah and Hendra viruses.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=10150&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9&amp;be=5984&amp;fe=10102&amp;dc=9601&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749593946042,%22n%22:0,%22f%22:5350,%22dn%22:5351,%22dne%22:5351,%22c%22:5351,%22s%22:5351,%22ce%22:5351,%22rq%22:5351,%22rp%22:5964,%22rpe%22:6829,%22dl%22:5970,%22di%22:7079,%22ds%22:9600,%22de%22:9613,%22dc%22:10102,%22l%22:10102,%22le%22:10138%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><style></style><style>stream {
  position: relative;
  display: block;
}
</style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(22)00655-9"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-633170" aria-expanded="false" aria-controls="login-links-633170" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-633170" id="login-links--633170" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-633170" data-opens="#links-dialog-633170" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-633170" aria-modal="true"><button data-closes="#links-dialog-633170" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2022.05.024">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail long-title" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(22)00655-9" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(21)X0015-3"><span>Volume 185</span>, <span>Issue 14</span></a></span><span class="meta-panel__pages"><span>p2523-2541.e30</span></span><span class="meta-panel__onlineDate">July 07, 2022</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=14&amp;pii=S0092-8674%2821%29X0015-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Lay Teng</span> <span property="familyName">Ang</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lay Teng</span> <span property="familyName">Ang</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:layteng@stanford.edu" property="email" title="Send email to Lay Teng Ang">layteng@stanford.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ang/Lay+Teng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">11</a></sup> <a id="eadd0010" href="mailto:layteng@stanford.edu" property="email" title="Send email to Lay Teng Ang">layteng@stanford.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Alana T.</span> <span property="familyName">Nguyen</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alana T.</span> <span property="familyName">Nguyen</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nguyen/Alana+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">11</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Kevin J.</span> <span property="familyName">Liu</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kevin J.</span> <span property="familyName">Liu</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Liu/Kevin+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">11</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Angela</span> <span property="familyName">Chen</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Angela</span> <span property="familyName">Chen</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chen/Angela" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Xiaochen</span> <span property="familyName">Xiong</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xiaochen</span> <span property="familyName">Xiong</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Xiong/Xiaochen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Matthew</span> <span property="familyName">Curtis</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew</span> <span property="familyName">Curtis</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedical Engineering, University of California, Davis, Davis, CA 95616, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Curtis/Matthew" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Renata M.</span> <span property="familyName">Martin</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Renata M.</span> <span property="familyName">Martin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Martin/Renata+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Brian C.</span> <span property="familyName">Raftry</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Brian C.</span> <span property="familyName">Raftry</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Raftry/Brian+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Chun Yi</span> <span property="familyName">Ng</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Chun Yi</span> <span property="familyName">Ng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore 308232, Singapore</span></div></div><div class="core-author-link"><a href="/authored-by/Ng/Chun+Yi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Uwe</span> <span property="familyName">Vogel</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Uwe</span> <span property="familyName">Vogel</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Vogel/Uwe" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Angelika</span> <span property="familyName">Lander</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Angelika</span> <span property="familyName">Lander</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Lander/Angelika" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Benjamin J.</span> <span property="familyName">Lesch</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Benjamin J.</span> <span property="familyName">Lesch</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Lesch/Benjamin+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Jonas L.</span> <span property="familyName">Fowler</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jonas L.</span> <span property="familyName">Fowler</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Fowler/Jonas+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Alyssa R.</span> <span property="familyName">Holman</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Alyssa R.</span> <span property="familyName">Holman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Holman/Alyssa+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Timothy</span> <span property="familyName">Chai</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Timothy</span> <span property="familyName">Chai</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Chai/Timothy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Siva</span> <span property="familyName">Vijayakumar</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Siva</span> <span property="familyName">Vijayakumar</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Vijayakumar/Siva" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Fabian P.</span> <span property="familyName">Suchy</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fabian P.</span> <span property="familyName">Suchy</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Suchy/Fabian+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Toshinobu</span> <span property="familyName">Nishimura</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Toshinobu</span> <span property="familyName">Nishimura</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nishimura/Toshinobu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Joydeep</span> <span property="familyName">Bhadury</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joydeep</span> <span property="familyName">Bhadury</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bhadury/Joydeep" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Matthew H.</span> <span property="familyName">Porteus</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew H.</span> <span property="familyName">Porteus</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Porteus/Matthew+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Hiromitsu</span> <span property="familyName">Nakauchi</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Hiromitsu</span> <span property="familyName">Nakauchi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nakauchi/Hiromitsu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Christine</span> <span property="familyName">Cheung</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christine</span> <span property="familyName">Cheung</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore 308232, Singapore</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute of Molecular and Cell Biology, A<sup>∗</sup>STAR, Singapore 138673, Singapore</span></div></div><div class="core-author-link"><a href="/authored-by/Cheung/Christine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Steven C.</span> <span property="familyName">George</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Steven C.</span> <span property="familyName">George</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedical Engineering, University of California, Davis, Davis, CA 95616, USA</span></div></div><div class="core-author-link"><a href="/authored-by/George/Steven+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Kristy</span> <span property="familyName">Red-Horse</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kristy</span> <span property="familyName">Red-Horse</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biology, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Red-Horse/Kristy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Joseph B.</span> <span property="familyName">Prescott</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joseph B.</span> <span property="familyName">Prescott</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:prescottj@rki.de" property="email" title="Send email to Joseph B. Prescott">prescottj@rki.de</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany</span></div></div><div class="core-author-link"><a href="/authored-by/Prescott/Joseph+B" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup> <a id="eadd0015" href="mailto:prescottj@rki.de" property="email" title="Send email to Joseph B. Prescott">prescottj@rki.de</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Kyle M.</span> <span property="familyName">Loh</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kyle M.</span> <span property="familyName">Loh</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0020" href="mailto:kyleloh@stanford.edu" property="email" title="Send email to Kyle M. Loh">kyleloh@stanford.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Loh/Kyle+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">13</a></sup> <a id="eadd0020" href="mailto:kyleloh@stanford.edu" property="email" title="Send email to Kyle M. Loh">kyleloh@stanford.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Stanford Institute for Stem Cell Biology &amp; Regenerative Medicine, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Department of Developmental Biology, Stanford Cardiovascular Institute, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Department of Pediatrics, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Department of Biology, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Howard Hughes Medical Institute, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Department of Genetics, Stanford University, Stanford, CA 94305, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Department of Biomedical Engineering, University of California, Davis, Davis, CA 95616, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore 308232, Singapore</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Institute of Molecular and Cell Biology, A<sup>∗</sup>STAR, Singapore 138673, Singapore</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Center for Biological Threats and Special Pathogens, Robert Koch Institute, Berlin 13353, Germany</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">11</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">12</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">13</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received August 18, 2021</span>; <span>Revised March 26, 2022</span>; <span>Accepted May 26, 2022</span>; <span>Published online June 22, 2022</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2022.05.024" property="sameAs" target="_blank">10.1016/j.cell.2022.05.024 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867422006559" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2022 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2022.05.024" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(21)X0015-3"><img src="/cms/asset/652186d1-429d-4d6c-9697-12033e8079d0/cov200h.gif" alt="Cover Image - Cell, Volume 185, Issue 14"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2822%2900655-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (5)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900655-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.05.024">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2822%2900655-9" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2022.05.024&amp;pii=S0092-8674%2822%2900655-9" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2822%2900655-9" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2022.05.024">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2022.05.024" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867422006559&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867422006559&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=185&amp;issue=14&amp;pii=S0092-8674%2821%29X0015-3" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(22)00641-9" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(22)00649-3" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental information (5)</a></li><li><a href="#data-availability">Data and code availability</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Efficient, rapid generation of &gt;90% pure human artery and vein cells in 3–4&nbsp;days</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Precisely inducing artery cells by blocking vein-specifying cues and vice versa</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Nipah and Hendra viruses preferentially infect artery (rather than vein) cells</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Human pluripotent stem cells provide a new platform for biosafety-level-4 virology</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Stem cell research endeavors to generate specific subtypes of classically defined “cell types.” Here, we generate &gt;90% pure human artery or vein endothelial cells from pluripotent stem cells within 3–4&nbsp;days. We specified artery cells by inhibiting vein-specifying signals and vice versa. These cells modeled viral infection of human vasculature by Nipah and Hendra viruses, which are extraordinarily deadly (∼57%–59% fatality rate) and require biosafety-level-4 containment. Generating pure populations of artery and vein cells highlighted that Nipah and Hendra viruses preferentially infected arteries; arteries expressed higher levels of their viral-entry receptor. Virally infected artery cells fused into syncytia containing up to 23 nuclei, which rapidly died. Despite infecting arteries and occupying ∼6%–17% of their transcriptome, Nipah and Hendra largely eluded innate immune detection, minimally eliciting interferon signaling. We thus efficiently generate artery and vein cells, introduce stem-cell-based toolkits for biosafety-level-4 virology, and explore the arterial tropism and cellular effects of Nipah and Hendra viruses.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/32626151-f9a9-420b-aeb2-f3336177f4e1/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2022.05.024/asset/007db6fa-a2e7-4f73-a807-dee5e942916c/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/32626151-f9a9-420b-aeb2-f3336177f4e1/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;human+pluripotent+stem+cells&quot;" alt="Search on this keyword" rel="nofollow">human pluripotent stem cells</a></li><li><a href="/action/doSearch?AllField=&quot;developmental+biology&quot;" alt="Search on this keyword" rel="nofollow">developmental biology</a></li><li><a href="/action/doSearch?AllField=&quot;endothelial+cells&quot;" alt="Search on this keyword" rel="nofollow">endothelial cells</a></li><li><a href="/action/doSearch?AllField=&quot;artery&quot;" alt="Search on this keyword" rel="nofollow">artery</a></li><li><a href="/action/doSearch?AllField=&quot;vein&quot;" alt="Search on this keyword" rel="nofollow">vein</a></li><li><a href="/action/doSearch?AllField=&quot;Nipah+virus&quot;" alt="Search on this keyword" rel="nofollow">Nipah virus</a></li><li><a href="/action/doSearch?AllField=&quot;Hendra+virus&quot;" alt="Search on this keyword" rel="nofollow">Hendra virus</a></li><li><a href="/action/doSearch?AllField=&quot;biosafety-level-4+virology&quot;" alt="Search on this keyword" rel="nofollow">biosafety-level-4 virology</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0035" role="paragraph">Endothelial cells (ECs)—the innermost constituents of blood vessels—pervade all tissues and have manifold roles in health and disease. In health, they control entry and exit of bloodborne nutrients, wastes, and immune cells to and from their client tissues (<span class="dropBlock reference-citations"><a id="body-ref-sref6-1" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-1" aria-controls="sref6-1">Augustin and Koh, 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Augustin, H.G. ∙ Koh, G.Y.</div><div><strong>Organotypic vasculature: From descriptive heterogeneity to functional pathophysiology</strong></div><div><em>Science.</em> 2017; <strong>357</strong>:eaal2379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aal2379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026996599" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28775214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aal2379&amp;pmid=28775214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref23-1" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-1" aria-controls="sref23-1">Coultas et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Coultas, L. ∙ Chawengsaksophak, K. ∙ Rossant, J.</div><div><strong>Endothelial cells and VEGF in vascular development</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:937-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04479" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744432619" target="_blank" aria-label="Open Scopus in new tab">Scopus (784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04479&amp;pmid=16355211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-1" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-1" aria-controls="sref94-1">Potente and Mäkinen, 2017</a><div class="dropBlock__holder" data-db-target-of="sref94-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Potente, M. ∙ Mäkinen, T.</div><div><strong>Vascular heterogeneity and specialization in development and disease</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.36" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026911538" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28537573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.36&amp;pmid=28537573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Rafii et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Rafii, S. ∙ Butler, J.M. ∙ Ding, B.-S.</div><div><strong>Angiocrine functions of organ-specific endothelial cells</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:316-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17040" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84955286381" target="_blank" aria-label="Open Scopus in new tab">Scopus (485)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26791722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17040&amp;pmid=26791722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In disease, ECs contribute to atherosclerosis and other afflictions that account for over half of human deaths (<span class="dropBlock reference-citations"><a id="body-ref-sref6-2" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-2" aria-controls="sref6-2">Augustin and Koh, 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Augustin, H.G. ∙ Koh, G.Y.</div><div><strong>Organotypic vasculature: From descriptive heterogeneity to functional pathophysiology</strong></div><div><em>Science.</em> 2017; <strong>357</strong>:eaal2379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aal2379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026996599" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28775214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aal2379&amp;pmid=28775214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Carmeliet, 2005</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Carmeliet, P.</div><div><strong>Angiogenesis in life, disease and medicine</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:932-936</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04478" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744479430" target="_blank" aria-label="Open Scopus in new tab">Scopus (2777)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04478&amp;pmid=16355210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), including viral diseases of the vasculature.</div><div id="p0040" role="paragraph">Many risk-group-4 viruses cause deadly vascular diseases, are among the most lethal human pathogens known, and require biosafety-level-4 (BSL4) containment (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">CDC and NIH, 2020</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>CDC, NIH</div><div><strong>Biosafety in Microbiological and Biomedical Laboratories</strong></div><div>United States Department of Health and Human Services, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=CDCNIHBiosafety+in+Microbiological+and+Biomedical+Laboratories6th+edition2020United+States+Department+of+Health+and+Human+Services" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here, we focus on Nipah and Hendra viruses, which respectively cause outbreaks throughout Asia (∼59% fatality rate) and Australia (∼57% fatality rate), and are risk-group-4 paramyxoviruses in the genus <i>Henipavirus</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Ang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Ang, B.S.P. ∙ Lim, T.C.C. ∙ Wang, L.</div><div><strong>Nipah virus infection</strong></div><div><em>J.&nbsp;Clin. Microbiol.</em> 2018; <strong>56</strong></div><div>e01875–e01817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JCM.01875-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047953553" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJCM.01875-17" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref19" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19" aria-controls="sref19">Chua et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Chua, K.B. ∙ Bellini, W.J. ∙ Rota, P.A. ...</div><div><strong>Nipah virus: a recently emergent deadly paramyxovirus</strong></div><div><em>Science.</em> 2000; <strong>288</strong>:1432-1435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.288.5470.1432" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0034717054" target="_blank" aria-label="Open Scopus in new tab">Scopus (905)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10827955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.288.5470.1432&amp;pmid=10827955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Mahalingam et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Mahalingam, S. ∙ Herrero, L.J. ∙ Playford, E.G. ...</div><div><strong>Hendra virus: an emerging paramyxovirus in Australia</strong></div><div><em>Lancet Infect. Dis.</em> 2012; <strong>12</strong>:799-807</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2812%2970158-5&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2812%2970158-5&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84866619198" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22921953/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2812%2970158-5&amp;pmid=22921953" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Individual Nipah virus outbreaks epidemiologically vary due to viral variants and other factors: the 1999 Malaysian and 2018 Indian outbreaks, respectively, exhibited fatality rates of ∼40% and ∼91% (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Ang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Ang, B.S.P. ∙ Lim, T.C.C. ∙ Wang, L.</div><div><strong>Nipah virus infection</strong></div><div><em>J.&nbsp;Clin. Microbiol.</em> 2018; <strong>56</strong></div><div>e01875–e01817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JCM.01875-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047953553" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJCM.01875-17" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Arunkumar et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Arunkumar, G. ∙ Chandni, R. ∙ Mourya, D.T. ..., Nipah Investigators People and Health Study Group</div><div><strong>Outbreak investigation of Nipah virus disease in Kerala, India, 2018</strong></div><div><em>J.&nbsp;Infect. Dis.</em> 2019; <strong>219</strong>:1867-1878</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/infdis/jiy612" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066960608" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30364984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Finfdis%2Fjiy612&amp;pmid=30364984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Of exigent concern, no treatments or vaccines for either Nipah or Hendra are presently approved for human use, and these diseases are thus classified priority diseases by the World Health Organization (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Gómez Román et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Gómez Román, R. ∙ Tornieporth, N. ∙ Cherian, N.G. ...</div><div><strong>Medical countermeasures against henipaviruses: a review and public health perspective</strong></div><div><em>Lancet Infect. Dis.</em> 2022; <strong>22</strong>:e13-e27</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2821%2900400-X&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2821%2900400-X&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85121695384" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34735799/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2821%2900400-X&amp;pmid=34735799" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Mehand et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Mehand, M.S. ∙ Al-Shorbaji, F. ∙ Millett, P. ...</div><div><strong>The WHO R&amp;D Blueprint: 2018 review of emerging infectious diseases requiring urgent research and development efforts</strong></div><div><em>Antiviral Res.</em> 2018; <strong>159</strong>:63-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.antiviral.2018.09.009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85054087266" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30261226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.antiviral.2018.09.009&amp;pmid=30261226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Nipah and Hendra viruses engender severe vascular disease (multiorgan vasculitis), which can culminate in respiratory and neurological failure (<span class="dropBlock reference-citations"><a id="body-ref-sref8-1" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-1" aria-controls="sref8-1">Baseler et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Baseler, L. ∙ de Wit, E. ∙ Scott, D.P. ...</div><div><strong>Syrian hamsters (Mesocricetus auratus) oronasally inoculated with a Nipah virus isolate from Bangladesh or Malaysia develop similar respiratory tract lesions</strong></div><div><em>Vet. Pathol.</em> 2015; <strong>52</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985814556189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84920024648" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25352203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985814556189&amp;pmid=25352203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Geisbert et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Geisbert, T.W. ∙ Daddario-DiCaprio, K.M. ∙ Hickey, A.C. ...</div><div><strong>Development of an acute and highly pathogenic nonhuman primate model of Nipah virus infection</strong></div><div><em>PLoS One.</em> 2010; <strong>5</strong>:e10690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0010690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77956283436" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20502528/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0010690&amp;pmid=20502528" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref122-1" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-1" aria-controls="sref122-1">Wong et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref122-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Wong, K.T. ∙ Shieh, W.-J. ∙ Kumar, S. ...</div><div><strong>Nipah virus infection: pathology and pathogenesis of an emerging paramyxoviral zoonosis</strong></div><div><em>Am. J. Pathol.</em> 2002; <strong>161</strong>:2153-2167</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=fulltext&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=pdf&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036899143" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12466131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0002-9440%2810%2964493-8&amp;pmid=12466131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To understand the enigmatic vascular diseases caused by risk-group-4 viruses, we must precisely map their target cells and effects on such cells. This pursuit has been complicated by unique constraints of BSL4 experimentation and inadequate experimental systems.</div><div id="p0045" role="paragraph">A challenge in studying ECs is that they encompass multiple subtypes—including artery, vein, capillary, and lymphatic ECs—which are molecularly distinct and subserve different physiological functions (<span class="dropBlock reference-citations"><a id="body-ref-sref27-1" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-1" aria-controls="sref27-1">De Val and Black, 2009</a><div class="dropBlock__holder" data-db-target-of="sref27-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>De Val, S. ∙ Black, B.L.</div><div><strong>Transcriptional control of endothelial cell development</strong></div><div><em>Dev. Cell.</em> 2009; <strong>16</strong>:180-195</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2009.01.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2009.01.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-59649110012" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19217421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2009.01.014&amp;pmid=19217421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-1" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-1" aria-controls="sref36-1">Fish and Wythe, 2015</a><div class="dropBlock__holder" data-db-target-of="sref36-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Fish, J.E. ∙ Wythe, J.D.</div><div><strong>The molecular regulation of arteriovenous specification and maintenance</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:391-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924032549" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25641373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24252&amp;pmid=25641373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-2" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-2" aria-controls="sref94-2">Potente and Mäkinen, 2017</a><div class="dropBlock__holder" data-db-target-of="sref94-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Potente, M. ∙ Mäkinen, T.</div><div><strong>Vascular heterogeneity and specialization in development and disease</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.36" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026911538" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28537573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.36&amp;pmid=28537573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). ECs are well known to be targets of Nipah and Hendra <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Aljofan et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aljofan, M. ∙ Saubern, S. ∙ Meyer, A.G. ...</div><div><strong>Characteristics of Nipah virus and Hendra virus replication in different cell lines and their suitability for antiviral screening</strong></div><div><em>Virus Res.</em> 2009; <strong>142</strong>:92-99</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virusres.2009.01.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-67349241274" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19428741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virusres.2009.01.014&amp;pmid=19428741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34-1" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-1" aria-controls="sref34-1">Erbar et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Erbar, S. ∙ Diederich, S. ∙ Maisner, A.</div><div><strong>Selective receptor expression restricts Nipah virus infection of endothelial cells</strong></div><div><em>Virol. J.</em> 2008; <strong>5</strong>:142</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1743-422X-5-142" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-58149168980" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19036148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1743-422X-5-142&amp;pmid=19036148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Lo et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Lo, M.K. ∙ Miller, D. ∙ Aljofan, M. ...</div><div><strong>Characterization of the antiviral and inflammatory responses against Nipah virus in endothelial cells and neurons</strong></div><div><em>Virology.</em> 2010; <strong>404</strong>:78-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2010.05.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77953714922" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20552729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2010.05.005&amp;pmid=20552729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-1" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-1" aria-controls="sref73-1">Mathieu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref73-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Mathieu, C. ∙ Guillaume, V. ∙ Sabine, A. ...</div><div><strong>Lethal Nipah virus infection induces rapid overexpression of CXCL10</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e32157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84857681004" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22393386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032157&amp;pmid=22393386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-1" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-1" aria-controls="sref77-1">Monaghan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Monaghan, P. ∙ Green, D. ∙ Pallister, J. ...</div><div><strong>Detailed morphological characterisation of Hendra virus infection of different cell types using super-resolution and conventional imaging</strong></div><div><em>Virol. J.</em> 2014; <strong>11</strong>:200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12985-014-0200-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84936855081" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12985-014-0200-5&amp;pmid=25428656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-1" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-1" aria-controls="sref105-1">Schountz et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref105-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Schountz, T. ∙ Campbell, C. ∙ Wagner, K. ...</div><div><strong>Differential innate immune responses elicited by Nipah virus and cedar virus correlate with disparate in&nbsp;vivo pathogenesis in hamsters</strong></div><div><em>Viruses.</em> 2019; <strong>11</strong></div><div>291–211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/v11030291" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85063712957" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30909389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fv11030291&amp;pmid=30909389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref8-2" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-2" aria-controls="sref8-2">Baseler et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Baseler, L. ∙ de Wit, E. ∙ Scott, D.P. ...</div><div><strong>Syrian hamsters (Mesocricetus auratus) oronasally inoculated with a Nipah virus isolate from Bangladesh or Malaysia develop similar respiratory tract lesions</strong></div><div><em>Vet. Pathol.</em> 2015; <strong>52</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985814556189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84920024648" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25352203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985814556189&amp;pmid=25352203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Geisbert et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Geisbert, T.W. ∙ Daddario-DiCaprio, K.M. ∙ Hickey, A.C. ...</div><div><strong>Development of an acute and highly pathogenic nonhuman primate model of Nipah virus infection</strong></div><div><em>PLoS One.</em> 2010; <strong>5</strong>:e10690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0010690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77956283436" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20502528/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0010690&amp;pmid=20502528" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-3" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-3" aria-controls="sref78-3">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref122-2" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-2" aria-controls="sref122-2">Wong et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref122-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Wong, K.T. ∙ Shieh, W.-J. ∙ Kumar, S. ...</div><div><strong>Nipah virus infection: pathology and pathogenesis of an emerging paramyxoviral zoonosis</strong></div><div><em>Am. J. Pathol.</em> 2002; <strong>161</strong>:2153-2167</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=fulltext&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=pdf&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036899143" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12466131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0002-9440%2810%2964493-8&amp;pmid=12466131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, whether different EC subtypes harbor distinct functions in viral diseases remains enigmatic. Artery ECs express higher levels of <i>EFNB2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-1" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-1" aria-controls="sref117-1">Wang et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref117-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Wang, H.U. ∙ Chen, Z.F. ∙ Anderson, D.J.</div><div><strong>Molecular distinction and angiogenic interaction between embryonic arteries and veins revealed by ephrin-B2 and its receptor Eph-B4</strong></div><div><em>Cell.</em> 1998; <strong>93</strong>:741-753</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0092-8674%2800%2981436-1&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0092-8674%2800%2981436-1&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0032577446" target="_blank" aria-label="Open Scopus in new tab">Scopus (1355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9630219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2981436-1&amp;pmid=9630219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—an entry receptor for Nipah and Hendra viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Bonaparte et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bonaparte, M.I. ∙ Dimitrov, A.S. ∙ Bossart, K.N. ...</div><div><strong>Ephrin-B2 ligand is a functional receptor for Hendra virus and Nipah virus</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:10652-10657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0504887102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-23044517161" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15998730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0504887102&amp;pmid=15998730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Negrete et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Negrete, O.A. ∙ Levroney, E.L. ∙ Aguilar, H.C. ...</div><div><strong>EphrinB2 is the entry receptor for Nipah virus, an emergent deadly paramyxovirus</strong></div><div><em>Nature.</em> 2005; <strong>436</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-22944445501" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16007075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03838&amp;pmid=16007075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—leading to the hypothesis that these viruses might preferentially infect artery ECs. Nevertheless, both artery and vein ECs can reportedly be infected by Nipah and Hendra <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Aljofan et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aljofan, M. ∙ Saubern, S. ∙ Meyer, A.G. ...</div><div><strong>Characteristics of Nipah virus and Hendra virus replication in different cell lines and their suitability for antiviral screening</strong></div><div><em>Virus Res.</em> 2009; <strong>142</strong>:92-99</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virusres.2009.01.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-67349241274" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19428741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virusres.2009.01.014&amp;pmid=19428741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Lo et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Lo, M.K. ∙ Miller, D. ∙ Aljofan, M. ...</div><div><strong>Characterization of the antiviral and inflammatory responses against Nipah virus in endothelial cells and neurons</strong></div><div><em>Virology.</em> 2010; <strong>404</strong>:78-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2010.05.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77953714922" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20552729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2010.05.005&amp;pmid=20552729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-2" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-2" aria-controls="sref73-2">Mathieu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref73-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Mathieu, C. ∙ Guillaume, V. ∙ Sabine, A. ...</div><div><strong>Lethal Nipah virus infection induces rapid overexpression of CXCL10</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e32157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84857681004" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22393386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032157&amp;pmid=22393386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-2" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-2" aria-controls="sref77-2">Monaghan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Monaghan, P. ∙ Green, D. ∙ Pallister, J. ...</div><div><strong>Detailed morphological characterisation of Hendra virus infection of different cell types using super-resolution and conventional imaging</strong></div><div><em>Virol. J.</em> 2014; <strong>11</strong>:200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12985-014-0200-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84936855081" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12985-014-0200-5&amp;pmid=25428656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Yet, an important proviso is that current systems to culture ECs do not preserve arteriovenous identity: primary ECs originally isolated from arteries or veins rapidly relinquish their arteriovenous subtype identity <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref4-1" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-1" aria-controls="sref4-1">Aranguren et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref4-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aranguren, X.L. ∙ Agirre, X. ∙ Beerens, M. ...</div><div><strong>Unraveling a novel transcription factor code determining the human arterial-specific endothelial cell signature</strong></div><div><em>Blood.</em> 2013; <strong>122</strong>:3982-3992</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2013-02-483255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84891126258" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24108462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2013-02-483255&amp;pmid=24108462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This technical limitation has obscured viral tropism for, and their effects on, specific EC subtypes.</div><div id="p0050" role="paragraph">Generating well-defined populations of human artery versus vein ECs <i>en masse</i> would empower us to explore viral infection of the vasculature, and in particular which EC subtype(s) are infected. Differentiation of human pluripotent stem cells (hPSCs, including embryonic and induced pluripotent stem cells) into specific cell types (e.g., ECs) is rapidly advancing, yet generating pure populations of desired cellular “subtypes” (e.g., artery and vein ECs) remains quite challenging (<span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">Fowler et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Fowler, J.L. ∙ Ang, L.T. ∙ Loh, K.M.</div><div><strong>A critical look: challenges in differentiating human pluripotent stem cells into desired cell types and organoids</strong></div><div><em>Wiley Interdiscip. Rev.: Dev. Biol.</em> 2019; <strong>113</strong></div><div>891–823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/wdev.368" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85075149381" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fwdev.368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Starting from pluripotent cells, we must map the branching lineage choices through which they differentiate into cellular subtypes and the extracellular signals that control each step in their journey. Extant protocols often differentiate hPSCs toward mesoderm using BMP, FGF, TGF-β, and/or WNT and then bias differentiation toward ECs using VEGF and other signals. However, these signals are also permissive for the formation of other mesodermal lineages; consequently, a heterogeneous population containing a subset of ECs usually arises (e.g., <span class="dropBlock reference-citations"><a id="body-ref-sref44-1" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-1" aria-controls="sref44-1">Halaidych et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref44-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Halaidych, O.V. ∙ Freund, C. ∙ van den Hil, F. ...</div><div><strong>Inflammatory responses and barrier function of endothelial cells derived from human induced pluripotent stem cells</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1642-1656</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045106821" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29657098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.012&amp;pmid=29657098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Kurian et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Kurian, L. ∙ Sancho-Martinez, I. ∙ Nivet, E. ...</div><div><strong>Conversion of human fibroblasts to angioblast-like progenitor cells</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:77-83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84871977315" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23202434/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2255&amp;pmid=23202434" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61-1" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-1" aria-controls="sref61-1">Lian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74" aria-controls="sref74">McCracken et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref74"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>McCracken, I.R. ∙ Taylor, R.S. ∙ Kok, F.O. ...</div><div><strong>Transcriptional dynamics of pluripotent stem cell-derived endothelial cell differentiation revealed by single-cell RNA sequencing</strong></div><div><em>Eur. Heart J.</em> 2019; <strong>385</strong>:9963117</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/eurheartj/ehz351" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85080882411" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Feurheartj%2Fehz351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84" aria-controls="sref84">Nguyen et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref84"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Nguyen, M.T.X. ∙ Okina, E. ∙ Chai, X. ...</div><div><strong>Differentiation of human embryonic stem cells to endothelial progenitor cells on laminins in defined and xeno-free systems</strong></div><div><em>Stem Cell Rep.</em> 2016; <strong>7</strong>:802-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2016.08.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2016.08.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84992170620" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27693424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2016.08.017&amp;pmid=27693424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-1" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-1" aria-controls="sref87-1">Olmer et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref87-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Olmer, R. ∙ Engels, L. ∙ Usman, A. ...</div><div><strong>Differentiation of human pluripotent stem cells into functional endothelial cells in scalable suspension culture</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1657-1672</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045438454" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29681541/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.017&amp;pmid=29681541" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Paik et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Paik, D.T. ∙ Tian, L. ∙ Lee, J. ...</div><div><strong>Large-scale single-cell RNA-seq reveals molecular signatures of heterogeneous populations of human induced pluripotent stem cell-derived endothelial cells</strong></div><div><em>Circ. Res.</em> 2018; <strong>123</strong>:443-450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCRESAHA.118.312913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85055602915" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29986945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCRESAHA.118.312913&amp;pmid=29986945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-1" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-1" aria-controls="sref126-1">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). hPSC-derived ECs sometimes lack (reviewed by <span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Lin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Lin, Y. ∙ Gil, C.-H. ∙ Yoder, M.C.</div><div><strong>Differentiation, evaluation, and application of human induced pluripotent stem cell-derived endothelial cells</strong></div><div><em>Arterioscler. Thromb. Vasc. Biol.</em> 2017; <strong>37</strong>:2014-2025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/ATVBAHA.117.309962" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85032977936" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29025705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FATVBAHA.117.309962&amp;pmid=29025705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or coexpress (<span class="dropBlock reference-citations"><a id="body-ref-sref44-2" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-2" aria-controls="sref44-2">Halaidych et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref44-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Halaidych, O.V. ∙ Freund, C. ∙ van den Hil, F. ...</div><div><strong>Inflammatory responses and barrier function of endothelial cells derived from human induced pluripotent stem cells</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1642-1656</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045106821" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29657098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.012&amp;pmid=29657098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref87-2" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-2" aria-controls="sref87-2">Olmer et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref87-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Olmer, R. ∙ Engels, L. ∙ Usman, A. ...</div><div><strong>Differentiation of human pluripotent stem cells into functional endothelial cells in scalable suspension culture</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1657-1672</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045438454" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29681541/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.017&amp;pmid=29681541" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) arterial and venous markers. On the other hand, overexpression of pan-endothelial transcription factor (TF) <i>ETV2</i> efficiently differentiates hPSCs into ECs, but they are also apparently bereft of arterial or venous identity (<span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Ng et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Ng, A.H.M. ∙ Khoshakhlagh, P. ∙ Rojo Arias, J.E. ...</div><div><strong>A comprehensive library of human transcription factors for cell fate engineering</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>51</strong>:987</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0742-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096952020" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0742-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref118" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118" aria-controls="sref118">Wang et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref118"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>Wang, K. ∙ Lin, R.-Z. ∙ Hong, X. ...</div><div><strong>Robust differentiation of human pluripotent stem cells into endothelial cells via temporal modulation of ETV2 with modified mRNA</strong></div><div><em>Sci. Adv.</em> 2020; <strong>6</strong>:eaba7606</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.aba7606" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85090074756" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32832668/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.aba7606&amp;pmid=32832668" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0055" role="paragraph">Inspired by developmental knowledge of how VEGF and NOTCH specify artery versus vein subtypes of ECs <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Coultas et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Coultas, L. ∙ Chawengsaksophak, K. ∙ Rossant, J.</div><div><strong>Endothelial cells and VEGF in vascular development</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:937-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04479" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744432619" target="_blank" aria-label="Open Scopus in new tab">Scopus (784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04479&amp;pmid=16355211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref27-2" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27-2" aria-controls="sref27-2">De Val and Black, 2009</a><div class="dropBlock__holder" data-db-target-of="sref27-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>De Val, S. ∙ Black, B.L.</div><div><strong>Transcriptional control of endothelial cell development</strong></div><div><em>Dev. Cell.</em> 2009; <strong>16</strong>:180-195</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2009.01.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2009.01.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-59649110012" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19217421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2009.01.014&amp;pmid=19217421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">Fang and Hirschi, 2019</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fang, J. ∙ Hirschi, K.</div><div><strong>Molecular regulation of arteriovenous endothelial cell specification</strong></div><div><em>F1000Res.</em> 2019; <strong>8</strong>:1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.16701.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85071923734" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.16701.1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-2" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-2" aria-controls="sref36-2">Fish and Wythe, 2015</a><div class="dropBlock__holder" data-db-target-of="sref36-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Fish, J.E. ∙ Wythe, J.D.</div><div><strong>The molecular regulation of arteriovenous specification and maintenance</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:391-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924032549" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25641373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24252&amp;pmid=25641373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Lin et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Lin, F.-J. ∙ Tsai, M.-J. ∙ Tsai, S.Y.</div><div><strong>Artery and vein formation: a tug of war between different forces</strong></div><div><em>EMBO Rep.</em> 2007; <strong>8</strong>:920-924</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sj.embor.7401076" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34848924388" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17906673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsj.embor.7401076&amp;pmid=17906673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-3" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-3" aria-controls="sref94-3">Potente and Mäkinen, 2017</a><div class="dropBlock__holder" data-db-target-of="sref94-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Potente, M. ∙ Mäkinen, T.</div><div><strong>Vascular heterogeneity and specialization in development and disease</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.36" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026911538" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28537573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.36&amp;pmid=28537573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), recent protocols manipulated these two signals to bias hPSC-derived ECs toward artery or vein identity with partial efficiency <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref29-1" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-1" aria-controls="sref29-1">Ditadi et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref29-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Ditadi, A. ∙ Sturgeon, C.M. ∙ Tober, J. ...</div><div><strong>Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:580-591</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928938612" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3161&amp;pmid=25915127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">Gage et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Gage, B.K. ∙ Liu, J.C. ∙ Innes, B.T. ...</div><div><strong>Generation of functional liver sinusoidal endothelial cells from human pluripotent stem-cell-derived venous angioblasts</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:254-269.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85088364652" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640183/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.06.007&amp;pmid=32640183" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-1" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-1" aria-controls="sref91-1">Park et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref91-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Park, M.A. ∙ Kumar, A. ∙ Jung, H.S. ...</div><div><strong>Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential</strong></div><div><em>Cell Rep.</em> 2018; <strong>23</strong>:2467-2481</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047204081" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29791856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.04.092&amp;pmid=29791856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102-1" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-1" aria-controls="sref102-1">Rosa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref102-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Rosa, S. ∙ Praça, C. ∙ Pitrez, P.R. ...</div><div><strong>Functional characterization of iPSC-derived arterial- and venous-like endothelial cells</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:3826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-40417-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062643820" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30846769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-40417-9&amp;pmid=30846769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-1" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-1" aria-controls="sref110-1">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-2" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-2" aria-controls="sref126-2">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which constituted an important step forward. However, the challenges in generating artery and vein ECs <i>in&nbsp;vitro</i> indicate gaps in our understanding of how these subtypes developmentally arise and intimate that additional signals for arteriovenous specification may exist.</div><div id="p0060" role="paragraph">Here, we chart a developmental roadmap for the stepwise diversification of hPSCs into primitive streak, lateral mesoderm, and subsequently into artery and vein ECs. At each step of differentiation, we reveal the extracellular signals that need to be activated, and those that must be simultaneously inhibited to prevent emergence of undesired cell types. We created &gt;90% pure populations of either human artery or vein ECs within 3–4&nbsp;days of hPSC differentiation. We then applied these well-defined artery and vein EC populations to BSL4 virology: Nipah and Hendra viruses preferentially targeted hPSC-derived artery ECs. More broadly, hPSC-derived cell types will provide a powerful platform to explore the biology of, and countermeasures against, risk-group-4 viruses under BSL4 containment.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Efficient differentiation of hPSCs into primitive streak and lateral mesoderm by repressing paraxial mesoderm and endoderm</h3><div id="p0065" role="paragraph">Starting from hPSCs <i>in&nbsp;vitro</i>, we sought to emulate the stepwise <i>in&nbsp;vivo</i> development of pluripotent cells (corresponding to day 5.5 of mouse embryonic development [∼E5.5]) into primitive streak (∼E6.5), lateral mesoderm (∼E7–7.5), and finally artery (∼E8.25) and vein ECs (∼E9.0) (<a id="crosref0630" href="#fig1">Figure&nbsp;1</a>A; <span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). First, activating BMP, FGF, TGF-β and WNT for 24&nbsp;h differentiated hPSCs into 97.3% ± 0.3% pure MIXL1-GFP<sup>+</sup> mid primitive streak while suppressing ectoderm formation (<a id="crosref0640" href="#fig1">Figure&nbsp;1</a>B; <span class="dropBlock reference-citations"><a id="body-ref-sref66-1" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-1" aria-controls="sref66-1">Loh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/3266107b-c15e-431f-ae09-8f5b2cf99df9/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2022.05.024/asset/0c3bf538-6bb5-4679-9adf-111a8cd23e18/main.assets/gr1.jpg" height="594" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/3266107b-c15e-431f-ae09-8f5b2cf99df9/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Efficient generation of human primitive streak and lateral mesoderm from hPSCs within 1 and 2&nbsp;days, respectively</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Artery and vein development.</div><div id="fspara0020" role="paragraph">(B) Generating MIXL1<sup>+</sup> mid primitive streak within 24&nbsp;h of hPSC differentiation (<span class="dropBlock reference-citations"><a id="body-ref-sref66-2" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-2" aria-controls="sref66-2">Loh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), assayed by flow cytometry of <i>MIXL1-GFP</i> hPSCs (<span class="dropBlock reference-citations"><a id="body-ref-sref25" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25" aria-controls="sref25">Davis et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Davis, R.P. ∙ Ng, E.S. ∙ Costa, M. ...</div><div><strong>Targeting a GFP reporter gene to the MIXL1 locus of human embryonic stem cells identifies human primitive streak-like cells and enables isolation of primitive hematopoietic precursors</strong></div><div><em>Blood.</em> 2008; <strong>111</strong>:1876-1884</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2007-06-093609" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-39349097787" target="_blank" aria-label="Open Scopus in new tab">Scopus (189)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18032708/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2007-06-093609&amp;pmid=18032708" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0025" role="paragraph">(C) VEGF and BMP specify, whereas TGF-β and WNT repress, day 2 lateral mesoderm. Day 1 hPSC-derived primitive streak was treated with the indicated signals (concentrations indicated, and 10&nbsp;nM Axitinib; 200&nbsp;ng/mL NOGGIN; 1&nbsp;μM DMH1, XAV939, and C59; 2&nbsp;μM SB505124 and SB431542; base conditions contained 10&nbsp;ng/mL BMP4 and/or 100&nbsp;ng/mL VEGF) for 24 h; qPCR was performed on day 2.</div><div id="fspara0030" role="paragraph">See also <a id="crosref2490" href="#figs1">Figure&nbsp;S1</a> and <a id="crosref2495" href="#mmc5">Table&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div><div id="p0070" role="paragraph">Second, we further differentiated hPSC-derived day 1 mid primitive streak into day 2 lateral mesoderm, which contains endothelial progenitors <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">Ciau-Uitz et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Ciau-Uitz, A. ∙ Pinheiro, P. ∙ Kirmizitas, A. ...</div><div><strong>VEGFA-dependent and -independent pathways synergise to drive Scl expression and initiate programming of the blood stem cell lineage in Xenopus</strong></div><div><em>Development.</em> 2013; <strong>140</strong>:2632-2642</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.090829" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84878286239" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23637333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.090829&amp;pmid=23637333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We induced lateral mesoderm markers <i>SCL</i>, <i>LMO2</i>, and <i>FLI1</i> by (1) activating BMP, VEGF, and PKA, while (2) simultaneously inhibiting WNT, TGF-β, and PI3K to repress formation of unwanted paraxial mesoderm and endoderm (<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Loh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref66-3" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" aria-label="Reference 2016" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-3" aria-controls="sref66-3">2016</a><div class="dropBlock__holder" data-db-target-of="sref66-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0665" href="#fig1">Figures&nbsp;1</a>C and <a id="crosref0670" href="#figs1">S1</a>A–S1E). We thus generate lateral mesoderm prior to generating ECs.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/ee1b1747-8ee0-4020-9ef8-25bdf9fa5749/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2022.05.024/asset/27de4192-729f-4a7a-b26b-93cd5c836e63/main.assets/figs1.jpg" height="695" width="761" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/ee1b1747-8ee0-4020-9ef8-25bdf9fa5749/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Generation of human lateral mesoderm from hPSCs, related to <a id="crosref2620" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0285" role="paragraph">(A) BMP specifies, whereas TGF-β and WNT repress, day 2 lateral mesoderm formation; (Ai) day 1 hPSC-derived primitive streak was treated with 100&nbsp;ng/mL VEGF for 24&nbsp;h in the presence or absence of BMP4 (10–40&nbsp;ng/mL) or BMP inhibitors (1&nbsp;μM DMH1 or 200&nbsp;ng/mL NOGGIN); (Aii) day 1 hPSC-derived primitive streak was treated with 10&nbsp;ng/mL BMP4&nbsp;+ 100&nbsp;ng/mL VEGF for 24&nbsp;h in the presence or absence of a WNT agonist (CHIR99021, 1–6&nbsp;μM) or WNT inhibitors (1&nbsp;μM C59 or 1&nbsp;μM XAV939); (Aiii) day 1 hPSC-derived primitive streak was treated with 10&nbsp;ng/mL BMP4&nbsp;+ 100&nbsp;ng/mL VEGF for 24&nbsp;h in the presence or absence of a TGF-β agonist (Activin, 5–100&nbsp;ng/mL) or TGF-β inhibitors (2&nbsp;μM SB505124 or SB431542); qPCR was performed on day 2 cell populations.</div><div id="fspara0290" role="paragraph">(B) VEGF treatment induces day 2 lateral mesoderm; day 1 hPSC-derived primitive streak was treated with 10&nbsp;ng/mL BMP4&nbsp;+ 1&nbsp;μM XAV939 for 24 h, in the presence or absence of VEGF (10–100&nbsp;ng/mL) or a VEGFR inhibitor (Axitinib, 10&nbsp;nM); qPCR was performed on day 2 cell populations (data for <i>FLI1</i> is also shown in <a id="crosref2625" href="#fig1">Figure&nbsp;1</a>C).</div><div id="fspara0295" role="paragraph">(C) PI3K inhibitors specify day 2 lateral mesoderm; day 1 hPSC-derived primitive streak was treated with BMP&nbsp;+ VEGF&nbsp;+ XAV939 in the presence or absence of PI3K inhibitors (PIK90 or GDC0941) for 24&nbsp;h to generate lateral mesoderm, which was then further differentiated into artery ECs for 24 additional hours (using VEGF alone); flow cytometry was performed on day 3 artery EC populations.</div><div id="fspara0300" role="paragraph">(D) PKA agonists specify day 2 lateral mesoderm; day 1 hPSC-derived primitive streak was treated with BMP&nbsp;+ GDC0941&nbsp;+ VEGF&nbsp;+ XAV939 in the presence or absence of PKA agonists (8-bromo-cAMP [1&nbsp;mM] or Forskolin [10&nbsp;μM]) or PKA inhibitor (KT5720 [1&nbsp;μM]) for 24&nbsp;h to generate day 2 candidate lateral mesoderm, which was then further differentiated into artery ECs for 24 additional hours (using GDC0941&nbsp;+ VEGF&nbsp;+ XAV939); flow cytometry was performed on day 3 artery EC populations.</div><div id="fspara0305" role="paragraph">(E) Immunostaining of H1 hPSC-derived day 2 lateral mesoderm, with DAPI nuclear counterstain (bottom); scale bars, 100&nbsp;μm.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Efficient differentiation of hPSC-derived lateral mesoderm into artery ECs by repressing vein and heart fates</h3><div id="p0075" role="paragraph">Subsequently, we identified the signals that controlled the further segregation of hPSC-derived day 2 lateral mesoderm into artery, vein, or heart lineages (<a id="crosref0675" href="#fig2">Figure&nbsp;2</a>A). To differentiate lateral mesoderm toward ECs, we (1) activated VEGF, while (2) simultaneously inhibiting BMP and WNT, which respectively induced NKX2.5<sup>+</sup> heart precursors (<span class="dropBlock reference-citations"><a id="body-ref-sref66-4" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-4" aria-controls="sref66-4">Loh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and CDX2<sup>+</sup> non-ECs (<a id="crosref0685" href="#figs2">Figures&nbsp;S2</a>A–S2C). Having inhibited formation of non-ECs, next we screened for signals that induce arterial marker SOX17 (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Corada et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Corada, M. ∙ Orsenigo, F. ∙ Morini, M.F. ...</div><div><strong>Sox17 is indispensable for acquisition and maintenance of arterial identity</strong></div><div><em>Nat. Commun.</em> 2013; <strong>4</strong>:2609</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms3609" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84886678124" target="_blank" aria-label="Open Scopus in new tab">Scopus (157)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24153254/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms3609&amp;pmid=24153254" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Sakamoto et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Sakamoto, Y. ∙ Hara, K. ∙ Kanai-Azuma, M. ...</div><div><strong>Redundant roles of Sox17 and Sox18 in early cardiovascular development of mouse embryos</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2007; <strong>360</strong>:539-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbrc.2007.06.093" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34447265321" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17610846/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbrc.2007.06.093&amp;pmid=17610846" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) using <i>SOX17-mCherry</i> knockin reporter hPSCs (<span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Loh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/a5612dfa-af24-47c8-a5aa-466a4320749e/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2022.05.024/asset/25adc4d5-999c-46ba-ac01-7130bccb279a/main.assets/gr2.jpg" height="806" width="761" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/a5612dfa-af24-47c8-a5aa-466a4320749e/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Efficient generation of human artery endothelial cells from hPSCs within 3&nbsp;days</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0040" role="paragraph">(A) Summary of this work.</div><div id="fspara0045" role="paragraph">(B) TGF-β promotes, while PI3K inhibits, day 3 artery formation. Day 2 hPSC-derived lateral mesoderm was treated with the indicated signals, including TGF-β agonist (activin, 5–25&nbsp;ng/mL), TGF-β inhibitor (SB505124, 2&nbsp;μM) or PI3K inhibitor (GDC0941, 0.5–2&nbsp;μM), for 24 h; qPCR was performed on day 2 (base media: GDC0941&nbsp;+ VEGF&nbsp;+ XAV939&nbsp;+ AA2P&nbsp;+ TTNPB [left] and VEGF&nbsp;+ XAV939&nbsp;+ AA2P&nbsp;+ TTNPB&nbsp;+ Activin [right]). Gene expression was normalized to the base media lacking TGF-β or PI3K modulators.</div><div id="fspara0050" role="paragraph">(C) Immunostaining of H1 hPSCs either before or after differentiation into day 3 artery ECs; scale bars, 100&nbsp;μm.</div><div id="fspara0055" role="paragraph">(D) Flow cytometry of <i>SOX17-2A-mPlum</i> hPSCs before or after differentiation into day 3 artery ECs.</div><div id="fspara0060" role="paragraph">(E) H1 and SUN004.1.9 hPSC lines differentiated using the present artery EC induction method or 4 prevailing EC differentiation protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref61-2" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-2" aria-controls="sref61-2">Lian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-2" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-2" aria-controls="sref92-2">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-2" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-2" aria-controls="sref110-2">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-3" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-3" aria-controls="sref126-3">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); flow cytometry was performed on days 0, 3, 5, and 6 of differentiation; n.d.&nbsp;= not determined.</div><div id="fspara0065" role="paragraph">(F) scRNA-seq of H1 hPSCs differentiated toward artery ECs; the entire day 3 cell population was harvested without pre-selecting ECs; gene expression in log<sub>e</sub> unique molecular identifier (UMI) units.</div><div id="fspara0070" role="paragraph">(G) scRNA-seq comparison of hPSC-derived day 3 artery ECs versus Carnegie Stage 13 (CS13) human fetal artery ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref125-1" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-1" aria-controls="sref125-1">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); 6,000 variable genes (top, each dot&nbsp;= 1 gene) or selected genes (bottom) shown.</div><div id="fspara0075" role="paragraph">See also <a id="crosref2525" href="#figs2">Figure&nbsp;S2</a> and <a id="crosref2530" href="#mmc1">Tables&nbsp;S1</a> and <a id="crosref2535" href="#mmc5">S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/b14db292-7841-4eb3-8e9e-010d9c8bc2b2/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2022.05.024/asset/a43940a6-a2bf-463d-9c7e-3cfa3c0e3174/main.assets/figs2.jpg" height="933" width="761" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/b14db292-7841-4eb3-8e9e-010d9c8bc2b2/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Generation of human artery endothelial cells from hPSCs, related to <a id="crosref2630" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0315" role="paragraph">(A) BMP represses day 3 artery formation; day 2 hPSC-derived lateral mesoderm was treated with Activin&nbsp;+ GDC0941&nbsp;+ VEGF&nbsp;+ XAV939&nbsp;+ Vitamin C for 24 h, in the presence or absence of BMP4 (5–40&nbsp;ng/mL) or BMP inhibitor (DMH1); qPCR was performed on day 3 cell populations.</div><div id="fspara0320" role="paragraph">(B) TGF-β promotes, whereas WNT and PI3K repress, day 3 artery formation from <i>SOX17-mCherry</i> hPSCs; (Bi and Bii) day 2 hPSC-derived lateral mesoderm was further differentiated for 24&nbsp;h with VEGF, in the presence or absence of TGF-β agonist (Activin, 10&nbsp;ng/mL), TGF-β inhibitor (SB505124, 2&nbsp;μM), WNT agonist (CHIR99021), or WNT inhibitor (XAV939); (Biii and Biv) day 2 hPSC-derived lateral mesoderm was further differentiated for 24&nbsp;h with VEGF&nbsp;+ XAV939, in the presence or absence of TGF-β agonist (Activin, 10&nbsp;ng/mL) or PI3K inhibitor (GDC0941); flow cytometry was performed on day 3 cell populations.</div><div id="fspara0325" role="paragraph">(C) WNT inhibits day 3 artery formation; day 2 hPSC-derived lateral mesoderm was further differentiated with VEGF for 24 h, in the presence or absence of WNT agonist (CHIR99021) or WNT inhibitor (XAV939); qPCR (right) was performed on day 3 cell populations.</div><div id="fspara0330" role="paragraph">(D) Definition of the set of extracellular signals necessary for efficient artery differentiation; day 2 hPSC-derived lateral mesoderm was further differentiated for 24&nbsp;h into artery cells using the complete artery specification media (red; Activin&nbsp;+ GDC0941&nbsp;+ VEGF&nbsp;+ XAV939&nbsp;+ AA2P&nbsp;+ DMH1) or in conditions in which each factor was individually withheld; flow cytometry was performed on day 3 cell populations.</div><div id="fspara0335" role="paragraph">(E) Flow cytometry of <i>SOX17-mCherry</i> hPSCs differentiated toward artery ECs for 3&nbsp;days (top); subgating on SOX17<sup>+</sup> CD34<sup>+</sup> ECs shows that they uniformly express VE-CADHERIN and CD31 (bottom).</div><div id="fspara0340" role="paragraph">(F) Flow cytometry of H1, H7, and H9 and SUN004.1.9 hPSCs differentiated into artery ECs for 3&nbsp;days reveals that &gt;94% of cells are CD144<sup>+</sup> CD34<sup>+</sup> ECs.</div><div id="fspara0345" role="paragraph">(G) Side-by-side comparison of the artery EC differentiation protocol (described in this study) against 4 prevailing methods for endothelial differentiation (<span class="dropBlock reference-citations"><a id="body-ref-sref61-3" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-3" aria-controls="sref61-3">Lian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-3" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-3" aria-controls="sref92-3">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-3" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-3" aria-controls="sref110-3">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-4" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-4" aria-controls="sref126-4">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in the H1 and SUN004.1.9 hPSC lines; flow cytometry to assess the percentage of CD34<sup>+</sup> CD144<sup>+</sup> endothelial cells was performed on days 3, 5, and 6 of differentiation (n.d.&nbsp;= not determined).</div><div id="fspara0350" role="paragraph">(H) qPCR time course of hPSCs differentiated toward artery ECs reveals sequential expression of pluripotency, primitive streak, lateral mesoderm and finally, pan-endothelial and artery-specific markers; qPCR was performed on day 0 (undifferentiated) and differentiated day 1, 1.5, 2, 2.5, and 3 cell populations, with qPCR data shown normalized to the time point with highest expression of each given marker gene.</div><div id="fspara0355" role="paragraph">(I) qPCR of FACS-purified day 3 SOX17<sup>+</sup> CD34<sup>+</sup> artery ECs generated from the <i>SOX17-mCherry</i> hPSC line (as well as undifferentiated hPSC controls) reveals expression of artery-specific and pan-endothelial markers, while markers of alternate cell fates are minimally expressed; qPCR data are shown relative to the reference gene <i>YWHAZ</i> (100%&nbsp;= same expression as <i>YWHAZ</i>).</div><div id="fspara0360" role="paragraph">(J) Transcriptional comparison of hPSC-derived day 3 artery ECs versus Carnegie Stage 13 (CS13) human fetal artery ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref125-2" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-2" aria-controls="sref125-2">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) by scRNA-seq to determine the top 10 genes that were differentially expressed by <i>in&nbsp;vivo</i> versus <i>in&nbsp;vitro</i> artery ECs (left) and gene ontology (GO) analysis was performed on all differentially expressed genes (right).</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph">Additional signals (TGF-β and PI3K) controlled the bifurcation of day-2 lateral mesoderm into artery versus vein fates: TGF-β specified arteries, whereas PI3K conversely induced veins (<a id="crosref0705" href="#fig2">Figures&nbsp;2</a>B and <a id="crosref0710" href="#figs2">S2</a>bii–S2biv). Consequently, we efficiently generated artery ECs by (1) inducing arterial fate through TGF-β activation, while (2) repressing vein formation by simultaneously inhibiting PI3K (<span class="dropBlock reference-citations"><a id="body-ref-sref29-2" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-2" aria-controls="sref29-2">Ditadi et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref29-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Ditadi, A. ∙ Sturgeon, C.M. ∙ Tober, J. ...</div><div><strong>Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:580-591</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928938612" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3161&amp;pmid=25915127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Hong et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Hong, C.C. ∙ Peterson, Q.P. ∙ Hong, J.Y. ...</div><div><strong>Artery/vein specification is governed by opposing phosphatidylinositol-3 kinase and MAP kinase/ERK signaling</strong></div><div><em>Curr. Biol.</em> 2006; <strong>16</strong>:1366-1372</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cub.2006.05.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cub.2006.05.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33745646792" target="_blank" aria-label="Open Scopus in new tab">Scopus (162)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16824925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2006.05.046&amp;pmid=16824925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-2" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-2" aria-controls="sref91-2">Park et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref91-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Park, M.A. ∙ Kumar, A. ∙ Jung, H.S. ...</div><div><strong>Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential</strong></div><div><em>Cell Rep.</em> 2018; <strong>23</strong>:2467-2481</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047204081" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29791856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.04.092&amp;pmid=29791856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). While previous protocols to putatively generate hPSC-derived artery ECs inhibited TGF-β (<span class="dropBlock reference-citations"><a id="body-ref-sref126-5" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-5" aria-controls="sref126-5">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found that TGF-β inhibition in fact blocked artery specification and instead induced venous marker <i>NR2F2</i> (<a id="crosref0735" href="#fig2">Figures&nbsp;2</a>B and <a id="crosref0740" href="#figs2">S2</a>Bii). This thus disclosed a temporally dynamic role for TGF-β signaling in specifying arterial fate, changing every 24&nbsp;h across 3&nbsp;days of hPSC differentiation. TGF-β had to be initially activated (inducing primitive streak, day 1), then inhibited (specifying lateral mesoderm, day 2), and later re-activated (to induce arterial fate, day 3) (<a id="crosref0745" href="#fig2">Figure&nbsp;2</a>C).</div><div id="p0085" role="paragraph">Taken together, activation of TGF-β (to specify arteries) and VEGF (to specify ECs) together with simultaneous blockade of BMP, WNT, and PI3K for 24&nbsp;h differentiated day-2 lateral mesoderm into day-3 artery ECs, while inhibiting unwanted differentiation into either heart or vein lineages (<a id="crosref0750" href="#figs2">Figure&nbsp;S2</a>D). This delineates a complex signaling code for arterial specification beyond activation of VEGF and NOTCH (reviewed by <span class="dropBlock reference-citations"><a id="body-ref-sref23-3" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-3" aria-controls="sref23-3">Coultas et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Coultas, L. ∙ Chawengsaksophak, K. ∙ Rossant, J.</div><div><strong>Endothelial cells and VEGF in vascular development</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:937-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04479" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744432619" target="_blank" aria-label="Open Scopus in new tab">Scopus (784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04479&amp;pmid=16355211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">Fang and Hirschi, 2019</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fang, J. ∙ Hirschi, K.</div><div><strong>Molecular regulation of arteriovenous endothelial cell specification</strong></div><div><em>F1000Res.</em> 2019; <strong>8</strong>:1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.16701.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85071923734" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.16701.1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-3" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-3" aria-controls="sref36-3">Fish and Wythe, 2015</a><div class="dropBlock__holder" data-db-target-of="sref36-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Fish, J.E. ∙ Wythe, J.D.</div><div><strong>The molecular regulation of arteriovenous specification and maintenance</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:391-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924032549" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25641373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24252&amp;pmid=25641373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We generated SOX17<sup>+</sup> CD144<sup>+</sup> artery ECs within 3&nbsp;days of hPSC differentiation (<a id="crosref0770" href="#fig2">Figure&nbsp;2</a>C), which were &gt;90% pure, as assayed using newly engineered <i>SOX17-2A-mPlum</i> reporter hPSCs (<a id="crosref0775" href="#fig2">Figure&nbsp;2</a>D) and <i>SOX17-mCherry</i> hPSCs (<a id="crosref0780" href="#figs2">Figure&nbsp;S2</a>E). Artery differentiation was reproducible across 4 hESC and hiPSC lines, generating 97.9% ± 2.4% pure CD144<sup>+</sup> DLL4<sup>+</sup> arterial ECs within 3&nbsp;days and was more efficient and rapid than 4 published differentiation protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref61-4" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-4" aria-controls="sref61-4">Lian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-4" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-4" aria-controls="sref92-4">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-4" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-4" aria-controls="sref110-4">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-6" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-6" aria-controls="sref126-6">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0805" href="#fig2">Figures&nbsp;2</a>E, <a id="crosref0810" href="#figs2">S2</a>F, and S2G).</div></section><section id="sec-2-3"><h3>Identity and uniformity of hPSC-derived artery cells and similarities with <i>in&nbsp;vivo</i> artery cells</h3><div id="p0090" role="paragraph">Single-cell RNA sequencing (scRNA-seq) reaffirmed the efficiency of artery differentiation. After 3&nbsp;days of differentiation, 91.9% of the population formed a predominant artery EC cluster that expressed artery-specific (<i>SOX17</i>, <i>CXCR4</i>, and <i>DLL4</i>) and pan-EC (<i>VE-CADHERIN</i>, <i>CD31</i>, and <i>CD34</i>) markers (<a id="crosref0815" href="#fig2">Figure&nbsp;2</a>F), which was confirmed by qPCR (<a id="crosref0820" href="#figs2">Figures&nbsp;S2</a>H and S2I). Nevertheless, individual artery ECs varied in their cell-cycle status, as shown by <i>MKI67</i> expression (<a id="crosref0825" href="#fig2">Figure&nbsp;2</a>F). ECs were artery specific: they did not express venous marker <i>NR2F2</i> (<a id="crosref0830" href="#fig2">Figure&nbsp;2</a>F), consistent with our inhibition of vein-specifying signals (e.g., PI3K). This thus contrasts with past hPSC-derived ECs that co-expressed both arterial and venous markers (<span class="dropBlock reference-citations"><a id="body-ref-sref87-3" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87-3" aria-controls="sref87-3">Olmer et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref87-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Olmer, R. ∙ Engels, L. ∙ Usman, A. ...</div><div><strong>Differentiation of human pluripotent stem cells into functional endothelial cells in scalable suspension culture</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1657-1672</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045438454" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29681541/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.017&amp;pmid=29681541" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Paik et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Paik, D.T. ∙ Tian, L. ∙ Lee, J. ...</div><div><strong>Large-scale single-cell RNA-seq reveals molecular signatures of heterogeneous populations of human induced pluripotent stem cell-derived endothelial cells</strong></div><div><em>Circ. Res.</em> 2018; <strong>123</strong>:443-450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCRESAHA.118.312913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85055602915" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29986945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCRESAHA.118.312913&amp;pmid=29986945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, 8.1% of cells were non-ECs that expressed mesenchymal markers (<a id="crosref0845" href="#fig2">Figure&nbsp;2</a>F). Taken together, activation of artery-inducing signals with simultaneous inhibition of vein-specifying signals efficiently generated hPSC-derived artery ECs, with scarce mesenchyme-like cells.</div><div id="p0095" role="paragraph">Attesting to their lineage fidelity, hPSC-derived artery ECs showed overall transcriptional similarity (R&nbsp;= 0.74) with <i>in&nbsp;vivo</i> artery ECs within 4-week-old human fetuses (<span class="dropBlock reference-citations"><a id="body-ref-sref125-3" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-3" aria-controls="sref125-3">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and expressed comparable levels of pan-EC and arterial markers (<a id="crosref0855" href="#fig2">Figure&nbsp;2</a>G). This was promising, given that <i>in&nbsp;vivo</i> versus <i>in&nbsp;vitro</i> cells were not precisely matched in their genetic background or developmental stage. Genes that were differentially expressed by artery ECs <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> largely related to metabolism and protein translation (<a id="crosref0860" href="#figs2">Figure&nbsp;S2</a>J; <a id="crosref0865" href="#mmc1">Table&nbsp;S1</a>), and as such, may reflect metabolic adaptation to cell culture. Additionally, <i>ESM</i><i>1</i>—a VEGF-induced gene (<span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Shin et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Shin, J.W. ∙ Huggenberger, R. ∙ Detmar, M.</div><div><strong>Transcriptional profiling of VEGF-A and VEGF-C target genes in lymphatic endothelium reveals endothelial-specific molecule-1 as a novel mediator of lymphangiogenesis</strong></div><div><em>Blood.</em> 2008; <strong>112</strong>:2318-2326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-05-156331" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-55249083593" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18614759/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-05-156331&amp;pmid=18614759" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—was higher <i>in&nbsp;vitro</i> (<a id="crosref0875" href="#figs2">Figure&nbsp;S2</a>J), perhaps reflecting VEGF levels in the media. Overall, hPSC-derived artery ECs are transcriptionally similar, but are not identical, to fetal artery ECs.</div></section><section id="sec-2-4"><h3>Efficient differentiation of hPSC-derived lateral mesoderm into veins by repressing artery and heart fates</h3><div id="p0100" role="paragraph">Having defined artery-specifying signals, next we asked what signals confer vein identity (<a id="crosref0880" href="#fig3">Figure&nbsp;3</a>A). Starting from day 2 hPSC-derived lateral mesoderm, we dually inhibited the artery-specifying signals TGF-β and NOTCH to block arterial differentiation (<a id="crosref0885" href="#fig3">Figure&nbsp;3</a>B). While NOTCH inhibition promotes venous fate (<span class="dropBlock reference-citations"><a id="body-ref-sref29-3" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-3" aria-controls="sref29-3">Ditadi et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref29-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Ditadi, A. ∙ Sturgeon, C.M. ∙ Tober, J. ...</div><div><strong>Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:580-591</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928938612" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3161&amp;pmid=25915127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Duarte et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Duarte, A. ∙ Hirashima, M. ∙ Benedito, R. ...</div><div><strong>Dosage-sensitive requirement for mouse Dll4 in artery development</strong></div><div><em>Genes Dev.</em> 2004; <strong>18</strong>:2474-2478</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1239004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-5444261424" target="_blank" aria-label="Open Scopus in new tab">Scopus (445)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15466159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1239004&amp;pmid=15466159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">Gage et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Gage, B.K. ∙ Liu, J.C. ∙ Innes, B.T. ...</div><div><strong>Generation of functional liver sinusoidal endothelial cells from human pluripotent stem-cell-derived venous angioblasts</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:254-269.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85088364652" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640183/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.06.007&amp;pmid=32640183" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Gale et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Gale, N.W. ∙ Dominguez, M.G. ∙ Noguera, I. ...</div><div><strong>Haploinsufficiency of delta-like 4 ligand results in embryonic lethality due to major defects in arterial and vascular development</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:15949-15954</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0407290101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-8644290828" target="_blank" aria-label="Open Scopus in new tab">Scopus (488)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15520367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0407290101&amp;pmid=15520367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Krebs et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Krebs, L.T. ∙ Shutter, J.R. ∙ Tanigaki, K. ...</div><div><strong>Haploinsufficient lethality and formation of arteriovenous malformations in Notch pathway mutants</strong></div><div><em>Genes Dev.</em> 2004; <strong>18</strong>:2469-2473</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1239204" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-5444242184" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15466160/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1239204&amp;pmid=15466160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Lawson et&nbsp;al., 2001</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Lawson, N.D. ∙ Scheer, N. ∙ Pham, V.N. ...</div><div><strong>Notch signaling is required for arterial-venous differentiation during embryonic vascular development</strong></div><div><em>Development.</em> 2001; <strong>128</strong>:3675-3683</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.128.19.3675" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11585794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.128.19.3675&amp;pmid=11585794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-3" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-3" aria-controls="sref91-3">Park et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref91-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Park, M.A. ∙ Kumar, A. ∙ Jung, H.S. ...</div><div><strong>Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential</strong></div><div><em>Cell Rep.</em> 2018; <strong>23</strong>:2467-2481</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047204081" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29791856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.04.092&amp;pmid=29791856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found that NOTCH inhibition was not sufficient to efficiently induce vein ECs <i>in&nbsp;vitro</i>. We thus explored additional signals for vein specification.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/288eafb6-9078-44a0-9105-b40334c3d073/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2022.05.024/asset/530d0beb-8db3-4875-9f69-62c69ba21bae/main.assets/gr3.jpg" height="737" width="761" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/288eafb6-9078-44a0-9105-b40334c3d073/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Efficient generation of human vein endothelial cells from hPSCs within 4&nbsp;days</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0085" role="paragraph">(A) Summary of this work.</div><div id="fspara0090" role="paragraph">(B) Dually inhibiting TGF-β and NOTCH promotes day 3 vein gene expression. Day 2 hPSC-derived lateral mesoderm was further differentiated for 24&nbsp;h with the indicated signals, including TGF-β inhibitor (SB505124, 2&nbsp;μM) and/or NOTCH inhibitor (RO4929097, 1&nbsp;μM); qPCR performed on day 3; <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05; <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01.</div><div id="fspara0095" role="paragraph">(C) VEGF/ERK activation followed by inhibition is critical for vein formation. Day 2 hPSC-derived lateral mesoderm was further differentiated for 48&nbsp;h in the presence or absence of ERK inhibitor (PD0325901, 100&nbsp;nM) for the indicated durations; qPCR performed on day 4 (base media: VEGF&nbsp;+ SB505124&nbsp;+ RO4929097&nbsp;+ XAV939&nbsp;+ DMH1&nbsp;+ AA2P for days 3–4).</div><div id="fspara0100" role="paragraph">(D) Flow cytometry of H1 <i>NR2F2</i><i>-2A-GFP</i> hPSCs before or after differentiation into day 5 vein ECs.</div><div id="fspara0105" role="paragraph">(E) NR2F2 and VE-cadherin immunostaining of H1 hPSC-derived day 4 vein ECs; scale bars, 100&nbsp;μm.</div><div id="fspara0110" role="paragraph">(F) scRNA-seq of H1 hPSCs differentiated toward vein ECs; the entire day 4 cell population was harvested without pre-selecting ECs; gene expression in log<sub>e</sub> UMI units.</div><div id="fspara0115" role="paragraph">(G) scRNA-seq comparison of hPSC-derived day 4 vein ECs versus Carnegie Stage 13 (CS13) human fetal vein ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref125-4" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-4" aria-controls="sref125-4">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); 6,000 variable genes (top, each dot&nbsp;= 1 gene) or selected genes (bottom) shown.</div><div id="fspara0120" role="paragraph">See also <a id="crosref2545" href="#figs3">Figure&nbsp;S3</a> and <a id="crosref2550" href="#mmc2">Tables&nbsp;S2</a> and <a id="crosref2555" href="#mmc5">S5</a>.</div></div></div></div></figcaption></figure></div><div id="p0105" role="paragraph">We discovered that VEGF/ERK signaling had a temporally dynamic role in vein EC specification <i>in&nbsp;vitro</i>: on day 3, it had to be activated to generate ECs, whereas on day 4, it had to be inhibited to generate vein ECs (<a id="crosref0925" href="#fig3">Figure&nbsp;3</a>C). ERK blockade on day 3 prevented EC specification (<a id="crosref0930" href="#fig3">Figure&nbsp;3</a>C). Yet, after cells acquired EC identity by day 3, subsequently inhibiting ERK on day 4 did not impair EC identity but rather was critical to instruct vein fate (<a id="crosref0935" href="#fig3">Figure&nbsp;3</a>C). Prolonged activation or inhibition of VEGF for 48&nbsp;h did not yield vein ECs, illustrating the importance of temporal VEGF/ERK modulation <i>in&nbsp;vitro</i> (<a id="crosref0940" href="#fig3">Figure&nbsp;3</a>C). Modulating VEGF/ERK signaling with such dynamism is also crucial for vein specification <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref14-1" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-1" aria-controls="sref14-1">Chetty et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Casie Chetty, S. ∙ Rost, M.S. ∙ Enriquez, J.R. ...</div><div><strong>Vegf signaling promotes vascular endothelial differentiation by modulating etv2 expression</strong></div><div><em>Dev. Biol.</em> 2017; <strong>424</strong>:147-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2017.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85014796623" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28279709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2017.03.005&amp;pmid=28279709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-1" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-1" aria-controls="sref60-1">Lawson et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref60-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lawson, N.D. ∙ Vogel, A.M. ∙ Weinstein, B.M.</div><div><strong>sonic hedgehog and vascular endothelial growth factor act upstream of the Notch pathway during arterial endothelial differentiation</strong></div><div><em>Dev. Cell.</em> 2002; <strong>3</strong>:127-136</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1534-5807%2802%2900198-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1534-5807%2802%2900198-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036070186" target="_blank" aria-label="Open Scopus in new tab">Scopus (650)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12110173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1534-5807%2802%2900198-3&amp;pmid=12110173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Shalaby et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Shalaby, F. ∙ Rossant, J. ∙ Yamaguchi, T.P. ...</div><div><strong>Failure of blood-island formation and vasculogenesis in Flk-1-deficient mice</strong></div><div><em>Nature.</em> 1995; <strong>376</strong>:62-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/376062a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0029006696" target="_blank" aria-label="Open Scopus in new tab">Scopus (3332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7596435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F376062a0&amp;pmid=7596435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) but was not fully appreciated by earlier vein differentiation protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref29-4" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-4" aria-controls="sref29-4">Ditadi et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref29-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Ditadi, A. ∙ Sturgeon, C.M. ∙ Tober, J. ...</div><div><strong>Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:580-591</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928938612" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3161&amp;pmid=25915127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Gage et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Gage, B.K. ∙ Liu, J.C. ∙ Innes, B.T. ...</div><div><strong>Generation of functional liver sinusoidal endothelial cells from human pluripotent stem-cell-derived venous angioblasts</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:254-269.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85088364652" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640183/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.06.007&amp;pmid=32640183" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-4" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-4" aria-controls="sref91-4">Park et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref91-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Park, M.A. ∙ Kumar, A. ∙ Jung, H.S. ...</div><div><strong>Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential</strong></div><div><em>Cell Rep.</em> 2018; <strong>23</strong>:2467-2481</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047204081" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29791856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.04.092&amp;pmid=29791856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102-2" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-2" aria-controls="sref102-2">Rosa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref102-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Rosa, S. ∙ Praça, C. ∙ Pitrez, P.R. ...</div><div><strong>Functional characterization of iPSC-derived arterial- and venous-like endothelial cells</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:3826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-40417-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062643820" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30846769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-40417-9&amp;pmid=30846769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-5" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-5" aria-controls="sref110-5">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-7" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-7" aria-controls="sref126-7">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Taken together, manipulating VEGF/ERK and WNT in a temporally dynamic way, along with inhibiting artery-specifying signals, differentiated hPSCs into vein ECs (<a id="crosref0990" href="#fig3">Figures&nbsp;3</a>A, <a id="crosref0995" href="#figs3">S3</a>A, and S3B).</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/9637e7c7-72b1-4101-a761-eeb2eacd6c77/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2022.05.024/asset/ab133bf4-3032-4aa7-bf28-b6d583029a45/main.assets/figs3.jpg" height="607" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/9637e7c7-72b1-4101-a761-eeb2eacd6c77/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Generation of human vein endothelial cells from hPSCs, related to <a id="crosref2660" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0370" role="paragraph">(A) WNT activation on day 4 of differentiation specifies vein ECs; day 3 H1 <i>NR2F2</i><i>-2A-GFP</i> hPSC-derived ECs were further differentiated into day 4 vein ECs (base media: SB505124&nbsp;+ RO4929097&nbsp;+ PD0325901&nbsp;+ AA2P), in the presence or absence of WNT inhibitor (XAV939, 1&nbsp;μM) or WNT agonist (CHIR99021, 1–7&nbsp;μM) for 24 h; flow cytometry was performed on day 4 vein populations to quantify the purity of NR2F2-2A-GFP<sup>+</sup> CD144<sup>+</sup> vein ECs.</div><div id="fspara0375" role="paragraph">(B) Resplitting lateral mesoderm improves vein EC differentiation; day 2 hPSC-derived lateral mesoderm was differentiated into day 4 vein ECs without resplitting (“unsplit”) or, alternatively, was dissociated and then re-seeded at high density (∼79,000 cells/cm<sup>2</sup>) in day 3 differentiation media, followed by treatment with day 4 vein EC media. Flow cytometry was performed at day 4 of hPSC differentiation.</div><div id="fspara0380" role="paragraph">(C) Strategy to generate <i>NR2F2-2A-GFP</i> knockin reporter hPSCs using Cas9/AAV6 genome editing (<span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0385" role="paragraph">(D) After genome editing (<span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), PCR genotyping of individual hPSC lines (derived from single colonies) reveals successful integration of the <i>NR2F2-2A-GFP</i> knockin reporter allele.</div><div id="fspara0390" role="paragraph">(E) Immunostaining of undifferentiated H1 hPSCs, with DAPI nuclear counterstain (negative control for <a id="crosref2675" href="#fig3">Figure&nbsp;3</a>E); scale bars, 100&nbsp;μm.</div><div id="fspara0395" role="paragraph">(F) Transcriptional comparison of hPSC-derived day 4 vein ECs versus Carnegie Stage 13 (CS13) human fetal vein ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref125-5" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-5" aria-controls="sref125-5">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) by scRNA-seq; gene expression is quantified in UMI expression units (log<sub>e</sub> scale).</div><div id="fspara0400" role="paragraph">(G) Transcriptional comparison of hPSC-derived day 4 artery ECs versus Carnegie Stage 13 (CS13) human fetal vein ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref125-6" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-6" aria-controls="sref125-6">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) by scRNA-seq to determine the top 10 genes that were differentially expressed by <i>in&nbsp;vivo</i> versus <i>in&nbsp;vitro</i> vein ECs.</div><div id="fspara0405" role="paragraph">(H) Transcriptional comparison of hPSC-derived day 4 artery ECs versus CS13 human fetal vein ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref125-7" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-7" aria-controls="sref125-7">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) by scRNA-seq to determine gene ontology (GO) terms associated with genes differentially expressed <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i>.</div></div></div></div></figcaption></figure></div><div id="p0110" role="paragraph">Through this approach, we could efficiently generate &gt;88% pure NR2F2<sup>+</sup> CD144<sup>+</sup> vein ECs, as shown using newly engineered <i>NR2F2-2A-GFP</i> knockin reporter hPSCs (<span class="dropBlock reference-citations"><a id="body-ref-sref71-3" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-3" aria-controls="sref71-3">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref1005" href="#fig3">Figures&nbsp;3</a>D, <a id="crosref1010" href="#figs3">S3</a>C, and S3D) and wild-type hPSCs (<a id="crosref1015" href="#fig3">Figures&nbsp;3</a>E and <a id="crosref1020" href="#figs3">S3</a>E). scRNA-seq reaffirmed the efficiency of vein differentiation: 89.3% of cells formed a vein EC cluster that expressed vein (<i>NR2F2</i>, <i>FLRT2</i>, and <i>APLNR</i>) and pan-EC (<i>VE-CADHERIN</i>, <i>CD31</i>, and <i>CD34</i>) markers (<a id="crosref1025" href="#fig3">Figure&nbsp;3</a>F). ECs were vein specific: they minimally expressed arterial markers (<a id="crosref1030" href="#fig3">Figure&nbsp;3</a>F), consistent with our inhibition of artery-specifying signals (e.g., NOTCH, TGF-β and VEGF/ERK). There was a small fraction (10.7%) of contaminating, mesenchyme-like non-ECs (<a id="crosref1035" href="#fig3">Figure&nbsp;3</a>F). Taken together, we efficiently generated hPSC-derived vein ECs, with a minority of mesenchyme-like cells. hPSC-derived vein ECs showed high overall transcriptional similarity with <i>in&nbsp;vivo</i> vein ECs within the 4-week-old CS13 human fetus (<span class="dropBlock reference-citations"><a id="body-ref-sref125-8" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-8" aria-controls="sref125-8">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (R&nbsp;= 0.83); both expressed venous markers (<a id="crosref1045" href="#fig3">Figures&nbsp;3</a>G and <a id="crosref1050" href="#figs3">S3</a>F). Genes that were differentially expressed by vein ECs <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> largely related to metabolism and protein translation and trafficking (<a id="crosref1055" href="#figs3">Figures&nbsp;S3</a>G and S3H; <a id="crosref1060" href="#mmc2">Table&nbsp;S2</a>). Taken together, hPSC-derived vein ECs are transcriptionally similar to fetal vein ECs, although they are not identical.</div><div id="p0115" role="paragraph">In summary, we generate enriched populations of either artery (<a id="crosref1065" href="#fig2">Figure&nbsp;2</a>) or vein ECs (<a id="crosref1070" href="#fig3">Figure&nbsp;3</a>) within 3 or 4&nbsp;days of hPSC differentiation, respectively. We find that artery ECs arise earlier than vein ECs from hPSCs, mirroring how artery ECs (∼E8.25) precede vein ECs (∼E9.0) in mouse embryos (<span class="dropBlock reference-citations"><a id="body-ref-sref18-3" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-3" aria-controls="sref18-3">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-5"><h3>Transcriptional and functional differences between hPSC-derived artery and vein cells</h3><div id="p0120" role="paragraph">There were stark transcriptome-wide differences between hPSC-derived artery and vein ECs, including disparate expression of known arteriovenous subtype markers (<a id="crosref1080" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref1085" href="#figs4">S4</a>A; <a id="crosref1090" href="#mmc3">Table&nbsp;S3</a>). Certain EC attraction and repulsion signals showed arterial- or venous-biased expression, suggesting potential bidirectional signaling between these two EC subtypes to coordinate their spatial positions. First, hPSC-derived artery ECs expressed the chemokine APELIN, whereas vein ECs expressed its cognate receptor (<i>APLNR</i>; <a id="crosref1095" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref1100" href="#figs4">S4</a>A), which is investigated below. <i>In&nbsp;vivo</i>, artery-secreted APELIN chemoattracts APLNR<sup>+</sup> veins, thus positioning arteries and veins in parallel (<span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Kidoya et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Kidoya, H. ∙ Naito, H. ∙ Muramatsu, F. ...</div><div><strong>APJ regulates parallel alignment of arteries and veins in the skin</strong></div><div><em>Dev. Cell.</em> 2015; <strong>33</strong>:247-259</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2015.02.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2015.02.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928817344" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25920569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2015.02.024&amp;pmid=25920569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Second, hPSC-derived vein ECs expressed <i>FLRT2</i>, a repulsive ligand for the arterially expressed netrin receptor <i>UNC5B</i> (<a id="crosref1110" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref1115" href="#figs4">S4</a>A; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Lu et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Lu, X. ∙ le Noble, F. ∙ Yuan, L. ...</div><div><strong>The netrin receptor UNC5B mediates guidance events controlling morphogenesis of the vascular system</strong></div><div><em>Nature.</em> 2004; <strong>432</strong>:179-186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-8844281731" target="_blank" aria-label="Open Scopus in new tab">Scopus (426)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15510105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03080&amp;pmid=15510105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Tai-Nagara et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Tai-Nagara, I. ∙ Yoshikawa, Y. ∙ Numata, N. ...</div><div><strong>Placental labyrinth formation in mice requires endothelial FLRT2/UNC5B signaling</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:2392-2401</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.149757" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021744683" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28576770/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.149757&amp;pmid=28576770" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), potentially contributing to why arteries and veins do not intermix. However, hPSC-derived artery and vein ECs did not express markers of organ-specific ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref50-1" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-1" aria-controls="sref50-1">Kalucka et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref50-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kalucka, J. ∙ de Rooij, L.P.M.H. ∙ Goveia, J. ...</div><div><strong>Single-cell transcriptome atlas of murine endothelial cells</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:764-779.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2020.01.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2020.01.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85079395140" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32059779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.01.015&amp;pmid=32059779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref1135" href="#figs4">Figure&nbsp;S4</a>B): hence, they may correspond to the earliest ECs arising during development prior to their entry into various organs (<span class="dropBlock reference-citations"><a id="body-ref-sref18-4" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-4" aria-controls="sref18-4">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An interactive browser to explore scRNA-seq profiles of hPSC-derived artery and vein ECs is available at <a id="intref0020" href="https://anglab.shinyapps.io/arteryveinscrna/" target="_blank">https://anglab.shinyapps.io/arteryveinscrna/</a>.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/0e3117d0-c10e-448b-b226-efd470bc3500/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2022.05.024/asset/c5c3041e-527e-45b1-adac-43ce5104e33d/main.assets/gr4.jpg" height="806" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/0e3117d0-c10e-448b-b226-efd470bc3500/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Transcriptional and functional differences between hPSC-derived artery and vein endothelial cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0130" role="paragraph">(A) Bulk-population RNA-seq of H1 hPSC-derived day 3 artery versus day 4 vein ECs (CD144<sup>+</sup> FACS sorted), each dot&nbsp;= 1 gene.</div><div id="fspara0135" role="paragraph">(B) 3D vascular networks formed by H1 hPSC-derived artery and vein ECs cocultured with fibroblasts in fibrin gel for 1&nbsp;week (<span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Kurokawa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); scale bars, 200&nbsp;μm.</div><div id="fspara0140" role="paragraph">(C–E) Subcutaneous transplantation of <i>CAG-AkaLuciferase-tdTomato</i><sup>+</sup> SUN003.1 hPSC-derived artery and vein ECs, monitored by intravital imaging (to assess cell proliferation) and microscopy (4&nbsp;weeks post-transplant); scale bars, 150&nbsp;μm.</div><div id="fspara0145" role="paragraph">(F) Flow cytometry of fluorescently labeled, acetylated LDL uptake by H1 hPSC-derived artery and vein ECs within 4 h.</div><div id="fspara0150" role="paragraph">(G) SUN004.1.9 hPSC-derived artery and vein ECs exposed to shear stress or static conditions for 24&nbsp;h align to the direction of fluid flow, quantified by the angle of the long axis of CD144<sup>+</sup> cell membranes; scale bars, 100&nbsp;μm.</div><div id="fspara0155" role="paragraph">(H) Beads coated with <i>CAG-GFP</i><sup>+</sup> SUN004.1.9 hPSC-derived vein ECs preferentially form vascular sprouts after embedding in 3D fibrin gel for 1&nbsp;day, as opposed to artery EC-coated beads; scale bars, 100&nbsp;μm.</div><div id="fspara0160" role="paragraph">(I) H1 hPSC-derived artery and vein ECs were expanded for 5&nbsp;days and then treated with TNF-α (10&nbsp;ng/mL) or left untreated for 4 h, followed by qPCR to assess TNF-α-induced upregulation of immune-cell adhesion molecules; <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01.</div><div id="fspara0165" role="paragraph">(J) H1 hPSC-derived artery and vein ECs were expanded for 6&nbsp;days and then were treated with TNF-α (10&nbsp;ng/mL) or left untreated for 4 h, prior to addition of fluorescent THP1 cells for 30&nbsp;min; scale bars, 100&nbsp;μm; <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05.</div><div id="fspara0170" role="paragraph">(K) In Transwell assay, <i>CAG-GFP</i><sup>+</sup> SUN004.1.9 hPSC-derived vein ECs preferentially migrate toward Apelin-13; images of migrated cells situated on the bottom membrane surface; scale bars, 100&nbsp;μm; <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05.</div><div id="fspara0175" role="paragraph">See also <a id="crosref2565" href="#figs4">Figure&nbsp;S4</a> and <a id="crosref2570" href="#mmc3">Tables&nbsp;S3</a> and <a id="crosref2575" href="#mmc5">S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/c48620ab-231d-4deb-b2d0-3f11ca2fa51d/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2022.05.024/asset/6073da7d-a0d6-4ff8-9d5f-54e574ab7d94/main.assets/figs4.jpg" height="601" width="761" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/c48620ab-231d-4deb-b2d0-3f11ca2fa51d/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Transcriptional and functional differences between hPSC-derived artery and vein endothelial cells, related to <a id="crosref2695" href="#fig4">Figure&nbsp;4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0415" role="paragraph">(A) scRNA-seq of H1 hPSC-derived day 3 artery versus day 4 vein ECs (CD144<sup>+</sup> FACS sorted) reveals mutually exclusive expression of hallmark arteriovenous marker genes.</div><div id="fspara0420" role="paragraph">(B) hPSC-derived day 3 artery ECs and day 4 vein ECs do not express known markers of organ-specific EC identities (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Kalucka et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Kalucka, J. ∙ de Rooij, L.P.M.H. ∙ Goveia, J. ...</div><div><strong>Single-cell transcriptome atlas of murine endothelial cells</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:764-779.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2020.01.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2020.01.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85079395140" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32059779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.01.015&amp;pmid=32059779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as shown by scRNA-seq (as positive controls, expression of reference genes <i>YWHAZ</i>, <i>ACTB</i>, and <i>GAPDH</i> is shown); dot size indicates percentage of cells expressing a given marker and dot color reflects the level of gene expression.</div><div id="fspara0425" role="paragraph">(C) H1 hPSC-derived artery and vein ECs were either mixed with adult human lung fibroblasts (ratio of 1 EC:2 fibroblasts) or not mixed with fibroblasts and then subsequently cultured in a 3D fibrin gel with EGM2 media for 1&nbsp;week <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Kurokawa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Only hPSC-derived ECs mixed with fibroblasts formed 3D networks, as assessed by CD31 immunostaining, demonstrating the importance of fibroblast support for vascular network formation in this experimental system; scale bars, 500&nbsp;μm.</div><div id="fspara0430" role="paragraph">(D) SUN004.1.9 hPSC-derived artery ECs were mixed with adult human lung fibroblasts from two different individuals (#1 and #2),and then subsequently cultured in a 3D fibrin gel with EGM2 media for 1&nbsp;week <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref56-3" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-3" aria-controls="sref56-3">Kurokawa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). hPSC-derived ECs mixed with fibroblasts from both individuals formed 3D networks, as assessed by CD31 immunostaining; scale bars, 200&nbsp;μm.</div><div id="fspara0435" role="paragraph">(E) H1 hPSC-derived day 3 artery ECs and day 4 vein ECs were stained for LDLR expression (left), or cultured with LDL DyLight 550 for 4&nbsp;h in their respective arterial or venous expansion media (right), prior to flow cytometry.</div><div id="fspara0440" role="paragraph">(F) H1 hPSC-derived day 3 artery ECs and day 4 vein ECs were each expanded for 24&nbsp;h in their respective arterial or venous expansion media until they formed confluent monolayers, and then were scratched with a pipette tip. Serial imaging was performed over the next 48&nbsp;h to quantify closure of the cell monolayer; scale bars, 100&nbsp;μm.</div><div id="fspara0445" role="paragraph">(G) SUN004.1.9 hPSC-derived artery and vein ECs exposed to shear stress or static conditions for 24&nbsp;h turn on different genes, as shown by qPCR; gene expression normalized such that for each gene, the sample expressing the highest level of the gene has an expression level&nbsp;= 1.0.</div><div id="fspara0450" role="paragraph">(H) H1 hPSC-derived artery and vein ECs were thawed and expanded for 6&nbsp;days in their respective arterial or venous expansion media and then were subsequently treated with TNF-α (10&nbsp;ng/mL) or left untreated for 4 h. Flow cytometry was performed to assess TNF-α-induced changes in immune-cell adhesion molecules CD54/ICAM1, CD62E/E-SELECTIN, and CD106/VCAM1.</div></div></div></div></figcaption></figure></div><div id="p0125" role="paragraph">Importantly, both hPSC-derived artery and vein ECs executed hallmark EC functions. First, both EC subtypes formed three-dimensional vascular networks <i>in&nbsp;vitro</i> when cocultured with mesenchymal cells within fibrin hydrogels for 1&nbsp;week (<a id="crosref1145" href="#fig4">Figures&nbsp;4</a>B, <a id="crosref1150" href="#figs4">S4</a>C, and S4D; <span class="dropBlock reference-citations"><a id="body-ref-sref56-4" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-4" aria-controls="sref56-4">Kurokawa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Second, artery and vein ECs likewise formed vascular networks <i>in&nbsp;vivo</i> upon subcutaneous transplantation into adult nonobese diabetic-severe combined immunodeficiency (NOD-SCID) <i>Il2rg</i><sup><i>−/−</i></sup> mice for 1&nbsp;month (<a id="crosref1160" href="#fig4">Figures&nbsp;4</a>C–4E). Third, artery and vein ECs also incorporated low-density lipoprotein (LDL) and acetylated LDL (<a id="crosref1165" href="#fig4">Figures&nbsp;4</a>F and <a id="crosref1170" href="#figs4">S4</a>E) and sealed a deliberate gap in monolayer cultures (<a id="crosref1175" href="#figs4">Figure&nbsp;S4</a>F).</div><div id="p0130" role="paragraph">Nevertheless hPSC-derived artery and vein ECs functionally differed in various properties: (1) responses to shear stress, (2) sprouting ability, (3) responses to inflammation, and (4) migration toward chemokines, revealing differences between these two EC subtypes. First, upon 24&nbsp;h of exposure to fluid flow, both artery and vein ECs aligned themselves to the flow axis and upregulated flow-induced TF <i>KLF2</i> (<a id="crosref1180" href="#fig4">Figures&nbsp;4</a>G and <a id="crosref1185" href="#figs4">S4</a>G). However, shear stress preferentially upregulated <i>AQP1</i> and <i>CX37</i> in artery ECs (<a id="crosref1190" href="#fig4">Figures&nbsp;4</a>G and <a id="crosref1195" href="#figs4">S4</a>G), evincing different transcriptional responses to shear stress by these two EC subtypes. Second, vein (not artery) ECs coated on beads formed vascular sprouts that protruded into fibrin gels within 24&nbsp;h (<a id="crosref1200" href="#fig4">Figure&nbsp;4</a>H). This is consistent with the notion that vein ECs preferentially sprout <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Red-Horse and Siekmann, 2019</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Red-Horse, K. ∙ Siekmann, A.F.</div><div><strong>Veins and arteries Build Hierarchical Branching Patterns Differently: Bottom-Up versus Top-Down</strong></div><div><em>BioEssays.</em> 2019; <strong>41</strong>:e1800198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bies.201800198" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062109763" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30805984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbies.201800198&amp;pmid=30805984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref123" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123" aria-controls="sref123">Xu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref123"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Xu, C. ∙ Hasan, S.S. ∙ Schmidt, I. ...</div><div><strong>Arteries are formed by vein-derived endothelial tip cells</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:5758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms6758" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84923281901" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25502622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms6758&amp;pmid=25502622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Third, ECs respond to inflammation by upregulating adhesion molecules to recruit immune cells (e.g., <span class="dropBlock reference-citations"><a id="body-ref-sref44-3" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44-3" aria-controls="sref44-3">Halaidych et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref44-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Halaidych, O.V. ∙ Freund, C. ∙ van den Hil, F. ...</div><div><strong>Inflammatory responses and barrier function of endothelial cells derived from human induced pluripotent stem cells</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1642-1656</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045106821" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29657098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.012&amp;pmid=29657098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-5" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-5" aria-controls="sref92-5">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-8" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-8" aria-controls="sref126-8">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Upon treatment with inflammatory cytokine TNF-α for 4 h, both artery and vein ECs upregulated the immune-cell adhesion molecule ICAM1 and physically adhered to monocyte-like cells (<a id="crosref1230" href="#fig4">Figures&nbsp;4</a>I and 4J). Nevertheless, artery and vein ECs partially differed in their responses to inflammation: immune-cell adhesion molecules VCAM1 and E-SELECTIN were specifically upregulated in artery ECs (<a id="crosref1235" href="#fig4">Figures&nbsp;4</a>I and <a id="crosref1240" href="#figs4">S4</a>H). Fourth, we found that artery and vein ECs migrated toward different chemokines. Vein ECs preferentially migrated across a porous membrane toward the endothelial chemokine APELIN (<a id="crosref1245" href="#fig4">Figure&nbsp;4</a>K). This is consistent with how vein ECs express higher levels of <i>APLNR</i> (<a id="crosref1250" href="#fig4">Figures&nbsp;4</a>A and <a id="crosref1255" href="#figs4">S4</a>A), which is critical for vein migration <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Kidoya et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Kidoya, H. ∙ Naito, H. ∙ Muramatsu, F. ...</div><div><strong>APJ regulates parallel alignment of arteries and veins in the skin</strong></div><div><em>Dev. Cell.</em> 2015; <strong>33</strong>:247-259</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2015.02.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2015.02.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928817344" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25920569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2015.02.024&amp;pmid=25920569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Collectively, hPSC-derived artery and vein ECs exhibited important transcriptional and functional differences, reifying that they have acquired distinct subtype identities.</div></section><section id="sec-2-6"><h3>Nipah and Hendra viruses preferentially induce multinucleated syncytia from artery cells</h3><div id="p0135" role="paragraph">Next, we applied hPSC-derived artery and vein ECs to study the tropism of Nipah and Hendra viruses, which must be handled under BSL4 containment (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">CDC and NIH, 2020</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>CDC, NIH</div><div><strong>Biosafety in Microbiological and Biomedical Laboratories</strong></div><div>United States Department of Health and Human Services, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=CDCNIHBiosafety+in+Microbiological+and+Biomedical+Laboratories6th+edition2020United+States+Department+of+Health+and+Human+Services" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus hPSC-derived artery and vein ECs were cryopreserved and then thawed and briefly cultured and expanded prior to viral infection under BSL4 containment. To expand vein ECs, it was critical to continuously inhibit the artery-specifying signals TGF-β and NOTCH (<a id="crosref1270" href="#figs5">Figures&nbsp;S5</a>A and S5B). In these conditions, artery and vein ECs could be cultured for at least 6&nbsp;days, while retaining their respective subtype identities and the ability to activate interferon signaling (<a id="crosref1275" href="#figs5">Figures&nbsp;S5</a>A–S5D).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/76a57c7a-2df4-43a2-9139-947a3ec29e37/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2022.05.024/asset/28e70eed-8a6b-4131-92d8-e1aba70b0818/main.assets/figs5.jpg" height="921" width="761" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/76a57c7a-2df4-43a2-9139-947a3ec29e37/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Expansion of hPSC-derived artery and vein endothelial cells and infection by Nipah and Hendra viruses, related to <a id="crosref2715" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0460" role="paragraph">(A) hPSC-derived artery and vein ECs can be expanded for 6&nbsp;days <i>in&nbsp;vitro</i> while retaining mutually exclusive expression of arteriovenous subtype markers. SUN004.1.9 hPSCs were differentiated into day 3 artery and day 4 vein ECs (“day 0” of expansion), frozen, and which were then respectively thawed in artery expansion media (EGM2) or vein expansion media (EGM2&nbsp;+ RO4929097 [2&nbsp;μM; NOTCH inhibitor]&nbsp;+ SB505124 [2&nbsp;μM; TGF-β inhibitor]); qPCR of expanded artery and vein ECs was performed at multiple time points post-thawing, and mRNA expression is shown relative to the expression of reference gene <i>YWHAZ</i> (i.e., <i>YWHAZ</i>&nbsp;= 1.0).</div><div id="fspara0465" role="paragraph">(B) Continued inhibition of artery-specifying signals TGF-β and NOTCH signals is required to expand hPSC-derived vein ECs <i>in&nbsp;vitro</i>. SUN004.1.9 hPSCs were differentiated into artery or vein ECs (“day 0” of expansion), and then frozen; vein ECs were then thawed in EGM2, or in the absence of hydrocortisone (HC) and/or presence of RO4929097 (2&nbsp;μM; NOTCH inhibitor) and/or SB505124 (2&nbsp;μM; TGF-β inhibitor) for 4 additional days; qPCR was then performed to determine mRNA expression, which is shown relative to the expression of reference gene <i>YWHAZ</i> (i.e., <i>YWHAZ</i>&nbsp;= 1.0).</div><div id="fspara0470" role="paragraph">(C) hPSC-derived artery ECs can be expanded for 8&nbsp;days <i>in&nbsp;vitro</i>. hPSCs were differentiated into artery ECs in 3&nbsp;days (“day 0” of expansion) and then subsequently expanded in EGM2 for up to 8&nbsp;days. Flow cytometry confirmed continued expression of pan-EC markers CD34 and CD144 and arterial marker DLL4.</div><div id="fspara0475" role="paragraph">(D) hPSC-derived artery and vein ECs activate interferon signaling in response to a double-stranded RNA mimic, poly(I:C). SUN004.1.9 hPSCs were differentiated into artery or vein ECs, frozen, and then thawed and cultured in their respective expansion media (EGM2 for artery ECs and EGM2&nbsp;+ RO4929097&nbsp;+ SB505124 for vein ECs) for 5&nbsp;days. Then, poly(I:C) (50&nbsp;μg/mL) was added to the respective culture media for 24 h, and qPCR was performed, with mRNA expression shown relative to the expression of reference gene <i>YWHAZ</i> (i.e., <i>YWHAZ</i>&nbsp;= 1.0); <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01.</div><div id="fspara0480" role="paragraph">(E) H1 hPSC-derived artery and vein ECs were inoculated with Nipah or Hendra viruses for 24&nbsp;h and then fixed and stained with Diff-Quik to assess cytopathic effects; total cell number (including mononucleated and multinucleated cells) was then quantified and normalized to uninfected controls; n.s.&nbsp;= not significant.</div><div id="fspara0485" role="paragraph">(H) Cocultures of GFP<sup>+</sup> and wild-type hPSC-derived artery and vein ECs were inoculated with either Nipah or Hendra virus; live imaging was conducted over an 8-h interval. The trajectory of single GFP<sup>+</sup> cells was computationally tracked by live imaging, with each single-cell trajectory colored differently; scale bars, 200&nbsp;μm.</div><div id="fspara0490" role="paragraph">(I) Cocultures of GFP<sup>+</sup> and wild-type hPSC-derived artery and vein ECs were inoculated with either Nipah or Hendra virus; live imaging was conducted over an 8-h interval. The trajectory of single GFP<sup>+</sup> cells was computationally tracked by live imaging, revealing two features: (1) increasing numbers of GFP<sup>+</sup> cells (reflecting fusion of GFP<sup>+</sup> cells with formerly wild-type cells and leading to an apparent increase in GFP<sup>+</sup> cells) and (2) fusion (where multiple cell tracks merged).</div></div></div></div></figcaption></figure></div><div id="p0140" role="paragraph">Upon infecting artery ECs, Nipah and Hendra viruses induced them to fuse into multinucleated syncytia, including prodigious syncytia harboring up to 23 nuclei per cell (<a id="crosref1280" href="#fig5">Figure&nbsp;5</a>A). Within 24 h, Nipah virus extensively induced arterial syncytia (accounting for 44.5% of ECs), whereas vein ECs were minimally fused (1.3% of ECs; p&nbsp;= 0.0013, <a id="crosref1285" href="#fig5">Figure&nbsp;5</a>B). Likewise, Hendra virus preferentially induced syncytia among artery ECs (p&nbsp;= 0.0041, <a id="crosref1290" href="#fig5">Figure&nbsp;5</a>B). Nipah virus reduced artery EC numbers by 71.0% within 24&nbsp;h of infection; vein ECs were significantly less affected (p&nbsp;= 0.0002, <a id="crosref1295" href="#figs5">Figure&nbsp;S5</a>E).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/fbe3738a-e7cf-4f5b-ba92-9ecbdd78841b/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2022.05.024/asset/6b8eeb59-3eb7-4599-bf09-7ca290d65dbe/main.assets/gr5.jpg" height="918" width="761" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/fbe3738a-e7cf-4f5b-ba92-9ecbdd78841b/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Nipah and Hendra viruses target artery cells</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0185" role="paragraph">(A) Syncytia (arrows) in H1 hPSC-derived artery and vein ECs inoculated with Nipah or Hendra viruses for 24 h; scale bars, 25&nbsp;μm.</div><div id="fspara0190" role="paragraph">(B) Numbers of nuclei per cell after inoculation with Nipah or Hendra viruses for 24&nbsp;h (top); syncytia frequency (≥3 nuclei/cell; bottom).</div><div id="fspara0195" role="paragraph">(C) Live imaging of cocultured SUN004.1.9 GFP<sup>+</sup> and H1 wild-type artery ECs infected with Nipah virus, tracking cell fusion into GFP<sup>+</sup> syncytia, followed by death (scale bars, 200&nbsp;μm, timestamp: hours:minutes).</div><div id="fspara0200" role="paragraph">(D) Live imaging showed Nipah- and Hendra-infected artery ECs displayed significantly greater median velocity (a measure of cell fusion) than vein ECs.</div><div id="fspara0205" role="paragraph">(E) Fusion and GFP diffusion between 3 Nipah-infected artery ECs, followed by aggregation of their nuclei (scale bars, 20&nbsp;μm, timestamp: hours:minutes).</div><div id="fspara0210" role="paragraph">See also <a id="crosref2580" href="#figs5">Figure&nbsp;S5</a> and <a id="crosref2585" href="#mmc6">Video S1</a>.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">By implementing live imaging in BSL4 containment, we confirmed that Nipah and Hendra viruses preferentially induced syncytia formation in artery ECs. Cocultures of GFP<sup>+</sup> and wild-type hPSC-derived ECs were infected, and cell fusion was continuously quantified by live imaging (<a id="crosref1300" href="#figs5">Figures&nbsp;S5</a>F and S5G; <a id="crosref1305" href="#mmc6">Video S1</a>). Nipah-infected artery ECs were progressively consumed into several massive syncytia over tens of hours and eventually eliminated (<a id="crosref1310" href="#fig5">Figure&nbsp;5</a>C). Again, Nipah and Hendra viruses preferentially induced cell fusion in artery ECs, as opposed to vein ECs (<a id="crosref1315" href="#fig5">Figure&nbsp;5</a>D). Cell fusion was rapid, occurring in minutes (<a id="crosref1320" href="#fig5">Figure&nbsp;5</a>E). Strikingly, after virally induced cell fusion, nuclei subsequently aggregated together within syncytia, forming large clusters of nuclei (<a id="crosref1325" href="#fig5">Figure&nbsp;5</a>E; <a id="crosref1330" href="#mmc6">Video S1</a>) akin to a “bunch of grapes” (<a id="crosref1335" href="#fig5">Figure&nbsp;5</a>A). This contrasts with healthy syncytia (e.g., skeletal muscle), wherein nuclei are randomly distributed across the cell (<span class="dropBlock reference-citations"><a id="body-ref-sref76" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76" aria-controls="sref76">Metzger et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref76"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Metzger, T. ∙ Gache, V. ∙ Xu, M. ...</div><div><strong>MAP and kinesin-dependent nuclear positioning is required for skeletal muscle function</strong></div><div><em>Nature.</em> 2012; <strong>484</strong>:120-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84859465143" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22425998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10914&amp;pmid=22425998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Taken together, Nipah and Hendra viruses preferentially induce artery ECs to fuse together, as opposed to vein ECs.</div><div id="p0150" role="paragraph"><div id="mmc6" class="core-supplementary-material" data-category="video"><div class="core-link"><div class="cloudflare-stream-container" id="mmc6" data-type="mp4"><span id="video_stream_mmc6_src" class="video-stream-source" style="display:none">eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMzYyMDczNGRhNGMwODM4OTU5ZDc2NDIxYmUwODhlYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NTUyfQ.mBrA6sUlpShsUxPhVCS_r5X9-efiqe0gtRRJTAQ3auyOPJCqeW7ZXmCORSJtDcNlD9og0OGzQsSTYnTK_p7D4zQeYG7LcLpn5H7BE2p_R8ezfzp-Rh8sBqHmMq3NdRpDQHSWeh0P5oC_seDMGm0-L9r4vf7ODgCLJLtsz7nWhcTcr0IOzLO7i8AytbCfvNq8o4qaEx6wYuqslaD5_xuw5uchM9t8pjko4aZaxdeicx5JujOMoxPwpZUaKUH6G0hNvue-j8wjnX2OQ4eA--HttQZYrCX4B7rLQGfkntWSKotK6CtgbF8IjLfmp8_gWa761P-NwdEAaKsA8i2CpvDezg</span><stream src="eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMzYyMDczNGRhNGMwODM4OTU5ZDc2NDIxYmUwODhlYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NTUyfQ.mBrA6sUlpShsUxPhVCS_r5X9-efiqe0gtRRJTAQ3auyOPJCqeW7ZXmCORSJtDcNlD9og0OGzQsSTYnTK_p7D4zQeYG7LcLpn5H7BE2p_R8ezfzp-Rh8sBqHmMq3NdRpDQHSWeh0P5oC_seDMGm0-L9r4vf7ODgCLJLtsz7nWhcTcr0IOzLO7i8AytbCfvNq8o4qaEx6wYuqslaD5_xuw5uchM9t8pjko4aZaxdeicx5JujOMoxPwpZUaKUH6G0hNvue-j8wjnX2OQ4eA--HttQZYrCX4B7rLQGfkntWSKotK6CtgbF8IjLfmp8_gWa761P-NwdEAaKsA8i2CpvDezg" class="cloudflare-stream-player ready tracked" id="video_stream_mmc6" controls="true"><div class="player-share-button" title="Share Video"><div class="share-player-icon"></div></div><div class="player-share-container hidden"><span class="player-close-share"></span><div class="social-links-container"><ul class="social-links-list"><li class="html5-share-facebook-icon" data-share-url="https://www.facebook.com/sharer/sharer.php?u=https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9" target="_blank"></li><li class="html5-share-twitter-icon" data-share-url="https://twitter.com/share?url=https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9" target="_blank"></li><li class="html5-share-mail-icon" data-share-url="mailto:?body=https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9" target="_blank"></li></ul><input type="text" value="https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9" readonly="readonly" class="share-input"></div></div><div style="position: relative; height: 0px; width: 100%; padding-top: 56.25%;"><iframe src="https://embed.cloudflarestream.com/embed/iframe.fla9.823df22.html?videoId=eyJraWQiOiI4ZjUxYWNhY2IzYjhiNjNlNzFlYmIzYWFmYTU5NmZmYyIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJjMzYyMDczNGRhNGMwODM4OTU5ZDc2NDIxYmUwODhlYyIsImtpZCI6IjhmNTFhY2FjYjNiOGI2M2U3MWViYjNhYWZhNTk2ZmZjIiwiZXhwIjoxNzQ5NjE1NTUyfQ.mBrA6sUlpShsUxPhVCS_r5X9-efiqe0gtRRJTAQ3auyOPJCqeW7ZXmCORSJtDcNlD9og0OGzQsSTYnTK_p7D4zQeYG7LcLpn5H7BE2p_R8ezfzp-Rh8sBqHmMq3NdRpDQHSWeh0P5oC_seDMGm0-L9r4vf7ODgCLJLtsz7nWhcTcr0IOzLO7i8AytbCfvNq8o4qaEx6wYuqslaD5_xuw5uchM9t8pjko4aZaxdeicx5JujOMoxPwpZUaKUH6G0hNvue-j8wjnX2OQ4eA--HttQZYrCX4B7rLQGfkntWSKotK6CtgbF8IjLfmp8_gWa761P-NwdEAaKsA8i2CpvDezg" allowfullscreen="true" allow="autoplay; picture-in-picture" style="height: 100%; width: 100%; position: absolute; top: 0px; left: 0px; border: none;"></iframe></div></stream></div><a href="/cms/10.1016/j.cell.2022.05.024/attachment/33797c6c-c191-414b-a26d-d97da7c45df6/mmc6.mp4" download="mmc6.mp4" aria-labelledby="mmc6-heading mmc6-link"><span id="mmc6-link">Video (72.98 MB)</span></a></div><div class="core-description"><div id="mmc6-heading" class="heading">Video S1. Live-imaging of cocultured wild-type and GFP<sup>+</sup> artery or vein ECs infected by Nipah or Hendra viruses, related to Figure&nbsp;5</div></div></div></div></section><section id="sec-2-7"><h3>Nipah and Hendra viruses preferentially target artery cells in an EFNB2-dependent fashion</h3><div id="p0155" role="paragraph">Consistent with how Nipah virus preferentially induced syncytia among artery ECs, Nipah-infected artery ECs produced 11.2-fold higher levels of Nipah viral RNA relative to vein ECs, as assayed by viral RNA levels in the culture media (p&nbsp;= 0.01, <a id="crosref1345" href="#fig6">Figure&nbsp;6</a>A). Likewise, artery ECs produced 17.4-fold higher levels of Hendra viral RNA compared with vein ECs, (p&nbsp;= 0.01, <a id="crosref1350" href="#fig6">Figure&nbsp;6</a>A). Intracellular viral RNA levels similarly disclosed an arterial tropism of these viruses, with the proviso that at later time points, Nipah RNA levels in artery ECs declined due to destruction of the cells (<a id="crosref1355" href="#figs6">Figure&nbsp;S6</a>A).</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/d7bffde2-e27f-48c5-b310-0ea143a27b5a/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2022.05.024/asset/34ea8c2e-f174-4bf0-8b3f-a0c6d4ef1131/main.assets/gr6.jpg" height="637" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/d7bffde2-e27f-48c5-b310-0ea143a27b5a/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;6</span> <span class="figure__title__text">Artery cells express higher levels of EFNB2, the Nipah and Hendra virus receptor</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0220" role="paragraph">(A) Nipah and Hendra viral RNA replication in H1 hPSC-derived artery and vein ECs, as quantified by qPCR of culture media; <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05; <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01.</div><div id="fspara0225" role="paragraph">(B) scRNA-seq of human adult lung ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref115-1" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-1" aria-controls="sref115-1">Travaglini et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref115-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Travaglini, K.J. ∙ Nabhan, A.N. ∙ Penland, L. ...</div><div><strong>A molecular cell atlas of the human lung from single-cell RNA sequencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:619-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2922-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096205400" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33208946/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2922-4&amp;pmid=33208946" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0230" role="paragraph">(C) Higher EFNB2 levels in hPSC-derived artery ECs relative to vein ECs, as shown by (Ci) scRNA-seq of day 3 artery versus day 4 vein ECs and (Cii) flow cytometry for surface EFNB2 protein expression on artery versus vein ECs, 4&nbsp;days post-expansion.</div><div id="fspara0235" role="paragraph">(D) Nipah or Hendra viral RNA replication in H1 wild-type and <i>EFNB2</i><sup><i>−/−</i></sup> hPSC-derived artery and vein ECs, as assayed by qPCR of culture media.</div><div id="fspara0240" role="paragraph">(E) H1 and <i>EFNB2</i><sup><i>−/−</i></sup> hPSC-derived artery and vein ECs inoculated with <i>GFP</i>-encoding Nipah pseudovirus (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Palomares et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01.</div><div id="fspara0245" role="paragraph">See also <a id="crosref2600" href="#figs6">Figure&nbsp;S6</a> and <a id="crosref2605" href="#mmc5">Table&nbsp;S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/2459c63f-8b08-4669-b356-392fe230f9d0/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2022.05.024/asset/f4480560-709f-47dc-b8c4-b915c47bb0bc/main.assets/figs6.jpg" height="923" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/2459c63f-8b08-4669-b356-392fe230f9d0/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S6</span> <span class="figure__title__text">Nipah virus preferentially enters artery ECs in an EFNB2-dependent fashion, related to <a id="crosref2720" href="#fig6">Figure&nbsp;6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0500" role="paragraph">(A) Intracellular levels of Nipah and Hendra viral RNA after inoculating H1 hPSC-derived artery and vein ECs with Nipah and Hendra viruses, as quantified by qPCR of the cell monolayer (after removing the culture media); <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05; <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01; <sup>†</sup>extensive cell loss.</div><div id="fspara0505" role="paragraph">(B) Significantly higher expression of <i>EFNB2</i> in hPSC-derived artery ECs relative to vein ECs. hPSC-derived artery and vein ECs generated from H1, SUN004.1.9 and SUN004.2 hPSC lines were frozen and then thawed in their respective arterial or venous expansion media for 3–6&nbsp;days prior to qPCR. <i>EFNB2</i> expression is shown relative to the expression of reference gene <i>YWHAZ</i> (i.e., <i>YWHAZ</i>&nbsp;= 1.0); <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.05.</div><div id="fspara0510" role="paragraph">(C) scRNA-seq of hPSC-derived artery and vein ECs revealed that among all <i>EFNA</i> and <i>EFNB</i> family members, <i>EFNB2</i> was most highly expressed in artery ECs, whereas <i>EFNB3</i> (another candidate entry-receptor for Nipah and Hendra viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref80-1" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-1" aria-controls="sref80-1">Negrete et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref80-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Negrete, O.A. ∙ Chu, D. ∙ Aguilar, H.C. ...</div><div><strong>Single amino acid changes in the Nipah and Hendra virus attachment glycoproteins distinguish ephrinB2 from ephrinB3 usage</strong></div><div><em>J.&nbsp;Virol.</em> 2007; <strong>81</strong>:10804-10814</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.00999-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34648830167" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.00999-07&amp;pmid=17652392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) was minimally expressed.</div><div id="fspara0515" role="paragraph">(D) Primary human artery and vein ECs cultured in standard conditions relinquish arteriovenous identity <i>in&nbsp;vitro</i>. Human umbilical artery and vein ECs were expanded in EGM2 and compared against hPSC-derived day 3 artery ECs and day 4 vein ECs by qPCR.</div><div id="fspara0520" role="paragraph">(E) Cas9-mediated deletion of <i>EFNB2</i> in hPSCs, as shown by (Ei) sequencing of the <i>EFNB2</i> gene in H1 wild-type and <i>EFNB2</i><sup><i>−/−</i></sup> hPSCs, revealing a 1-base deletion in both <i>EFNB2</i> alleles in the latter hPSC line and (Eii) flow cytometry for EFNB2 expression in H1 wild-type and <i>EFNB2</i><sup><i>−/−</i></sup> hPSCs prior to and after differentiation into artery ECs.</div><div id="fspara0525" role="paragraph">(F) Nipah pseudovirus preferentially enters hPSC-derived artery ECs, by comparison with vein ECs. hPSC-derived artery and vein ECs were thawed, expanded for 3&nbsp;days in respective arterial or venous expansion media, and then transduced with <i>GFP</i>-encoding lentiviruses pseudotyped with Nipah or VSV envelope proteins for 24 h. 48&nbsp;h later, flow cytometry was performed to enumerate the percentage of GFP<sup>+</sup> ECs, revealing arterially biased tropism of Nipah-pseudotyped lentivirus. As a negative control, VSV-G-pseudotyped lentivirus did not exhibit arterially biased tropism; <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01.</div><div id="fspara0530" role="paragraph">(G) <i>EFNB2</i> is dispensable for VSV pseudovirus entry. H1 wild-type and <i>EFNB2</i><sup><i>−/−</i></sup> hPSC-derived artery, and vein ECs were thawed, expanded for 3&nbsp;days and then infected with equal infectious units of <i>GFP</i>-encoding, non-replicating lentiviruses pseudotyped with VSV envelope proteins. Flow cytometry was performed 3&nbsp;days later, showing that <i>EFNB2</i> is dispensable for infection by VSV pseudovirus; <sup>∗∗</sup>p&nbsp;&lt;&nbsp;0.01; n.s.&nbsp;= not significant.</div></div></div></div></figcaption></figure></div><div id="p0160" role="paragraph">Why do Nipah and Hendra viruses preferentially infect artery ECs, as opposed to vein ECs? EFNB2—a known receptor for these viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Bonaparte et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bonaparte, M.I. ∙ Dimitrov, A.S. ∙ Bossart, K.N. ...</div><div><strong>Ephrin-B2 ligand is a functional receptor for Hendra virus and Nipah virus</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:10652-10657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0504887102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-23044517161" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15998730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0504887102&amp;pmid=15998730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Negrete et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Negrete, O.A. ∙ Levroney, E.L. ∙ Aguilar, H.C. ...</div><div><strong>EphrinB2 is the entry receptor for Nipah virus, an emergent deadly paramyxovirus</strong></div><div><em>Nature.</em> 2005; <strong>436</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-22944445501" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16007075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03838&amp;pmid=16007075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—is expressed more highly by artery ECs <i>in&nbsp;vivo</i>, in both mice (<span class="dropBlock reference-citations"><a id="body-ref-sref18-5" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-5" aria-controls="sref18-5">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-2" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-2" aria-controls="sref117-2">Wang et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref117-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Wang, H.U. ∙ Chen, Z.F. ∙ Anderson, D.J.</div><div><strong>Molecular distinction and angiogenic interaction between embryonic arteries and veins revealed by ephrin-B2 and its receptor Eph-B4</strong></div><div><em>Cell.</em> 1998; <strong>93</strong>:741-753</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0092-8674%2800%2981436-1&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0092-8674%2800%2981436-1&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0032577446" target="_blank" aria-label="Open Scopus in new tab">Scopus (1355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9630219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2981436-1&amp;pmid=9630219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and humans (<a id="crosref1380" href="#fig6">Figure&nbsp;6</a>B; <span class="dropBlock reference-citations"><a id="body-ref-sref115-2" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115-2" aria-controls="sref115-2">Travaglini et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref115-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Travaglini, K.J. ∙ Nabhan, A.N. ∙ Penland, L. ...</div><div><strong>A molecular cell atlas of the human lung from single-cell RNA sequencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:619-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2922-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096205400" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33208946/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2922-4&amp;pmid=33208946" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). hPSC-derived ECs faithfully reproduced this arteriovenous difference in <i>EFNB2</i> expression. hPSC-derived artery ECs expressed 8.8- and 10.4-fold higher levels of EFNB2 surface protein and mRNA, respectively, compared with vein ECs (<a id="crosref1390" href="#fig6">Figures&nbsp;6</a>C, <a id="crosref1395" href="#figs6">S6</a>B, and S6C). By contrast, primary human artery and vein ECs expanded in conventional media relinquished arteriovenous differences (<span class="dropBlock reference-citations"><a id="body-ref-sref4-2" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-2" aria-controls="sref4-2">Aranguren et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref4-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aranguren, X.L. ∙ Agirre, X. ∙ Beerens, M. ...</div><div><strong>Unraveling a novel transcription factor code determining the human arterial-specific endothelial cell signature</strong></div><div><em>Blood.</em> 2013; <strong>122</strong>:3982-3992</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2013-02-483255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84891126258" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24108462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2013-02-483255&amp;pmid=24108462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): they expressed similar <i>EFNB2</i> levels (<a id="crosref1405" href="#figs6">Figure&nbsp;S6</a>D). In sum, hPSC-derived cell types afford advantages (e.g., preserved subtype identity) relative to previous cellular models used in BSL4 virology.</div><div id="p0165" role="paragraph">Preferential Nipah virus entry into artery ECs was principally driven by EFNB2. Genetically deleting <i>EFNB2</i> using Cas9 (<a id="crosref1410" href="#figs6">Figure&nbsp;S6</a>E; <span class="dropBlock reference-citations"><a id="body-ref-sref71-4" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-4" aria-controls="sref71-4">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) led to a substantial 430.3- to 6,289.2-fold reduction of Nipah and Hendra virus replication in <i>EFNB2</i><sup><i>−/−</i></sup> artery ECs, respectively (<a id="crosref1420" href="#fig6">Figure&nbsp;6</a>D). To precisely assay viral entry, we employed Nipah pseudoviruses, which enter cells but do not replicate or inflict virus-specific cellular derangements (<span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Palomares et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Indeed, Nipah pseudoviruses entered artery ECs 3.2 ± 0.2 times more efficiently than vein ECs (<a id="crosref1430" href="#fig6">Figures&nbsp;6</a>E and <a id="crosref1435" href="#figs6">S6</a>F). Ablating <i>EFNB2</i> completely abrogated Nipah pseudovirus entry into artery ECs but did not impair entry of an unrelated vesicular stomatitis virus (VSV)-based pseudovirus (<a id="crosref1440" href="#fig6">Figures&nbsp;6</a>E and <a id="crosref1445" href="#figs6">S6</a>G). Therefore, EFNB2 is necessary for Nipah and Hendra virus entry, complementing the longstanding findings that EFNB2 is sufficient for entry of these viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Bonaparte et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bonaparte, M.I. ∙ Dimitrov, A.S. ∙ Bossart, K.N. ...</div><div><strong>Ephrin-B2 ligand is a functional receptor for Hendra virus and Nipah virus</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:10652-10657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0504887102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-23044517161" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15998730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0504887102&amp;pmid=15998730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-3" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-3" aria-controls="sref81-3">Negrete et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref81-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Negrete, O.A. ∙ Levroney, E.L. ∙ Aguilar, H.C. ...</div><div><strong>EphrinB2 is the entry receptor for Nipah virus, an emergent deadly paramyxovirus</strong></div><div><em>Nature.</em> 2005; <strong>436</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-22944445501" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16007075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03838&amp;pmid=16007075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0170" role="paragraph">In sum, the developmental bifurcation between artery versus vein ECs is important for viral infection, as it culminates in differential expression of viral-entry receptors (i.e., EFNB2) between these two EC subtypes and thus differing susceptibility to Nipah and Hendra infection. Beyond EFNB2, artery and vein ECs also differ in the expression of other genes, which may contribute to preferential viral entry and replication efficiencies in artery ECs.</div></section><section id="sec-2-8"><h3>Nipah and Hendra viruses minimally alter gene expression in artery cells, largely eluding the antiviral program</h3><div id="p0175" role="paragraph">Finally, we used RNA-seq to discover how artery ECs transcriptionally react to Nipah virus, Hendra virus, or a related non-pathogenic paramyxovirus, Sendai virus (Cantell strain) (<span class="dropBlock reference-citations"><a id="body-ref-sref67-1" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-1" aria-controls="sref67-1">López et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref67-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>López, C.B. ∙ Yount, J.S. ∙ Hermesh, T. ...</div><div><strong>Sendai virus infection induces efficient adaptive immunity independently of type I interferons</strong></div><div><em>J.&nbsp;Virol.</em> 2006; <strong>80</strong>:4538-4545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.80.9.4538-4545.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646184859" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16611914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.80.9.4538-4545.2006&amp;pmid=16611914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Cells were infected with a high viral titer (multiplicity of infection [MOI]&nbsp;≈ 5), and RNA-seq was performed at 5, 10, or 15&nbsp;h post-infection (<a id="crosref1465" href="#fig7">Figure&nbsp;7</a>A; <a id="crosref1470" href="#mmc4">Table&nbsp;S4</a>). All three paramyxoviruses exhibited the expected 3′-to-5′ gene transcriptional gradient along their genomes (<a id="crosref1475" href="#figs7">Figures&nbsp;S7</a>A and S7B), authenticating our RNA-seq analysis.</div><div class="figure-wrap"><figure id="fig7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/0fe0863c-6e9c-4eae-a785-3097fc3697e2/main.assets/gr7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr7" src="/cms/10.1016/j.cell.2022.05.024/asset/83e7e01e-5cfb-49fa-9c23-79e62a887dd0/main.assets/gr7.jpg" height="810" width="761" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/0fe0863c-6e9c-4eae-a785-3097fc3697e2/main.assets/gr7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;7</span> <span class="figure__title__text">Nipah and Hendra viruses elicit minimal transcriptional changes in artery endothelial cells, despite extensive viral replication and cell fusion</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0255" role="paragraph">(A) Percentage of hPSC-derived artery EC transcriptome comprising viral reads, as quantified by RNA-seq; averaged from H1 and SUN004.1.9 hPSCs.</div><div id="fspara0260" role="paragraph">(B) Transcriptome-wide differences between infected versus uninfected artery ECs by RNA-seq; averaged from H1 and SUN004.1.9 hPSCs.</div><div id="fspara0265" role="paragraph">(C and D) RNA-seq of infected hPSC-derived artery ECs, including viral read percentage in transcriptome; averaged from H1 and SUN004.1.9 hPSCs.</div><div id="fspara0270" role="paragraph">(E) IFNβ secretion by hPSC-derived artery ECs; averaged from H1, SUN004.1.9 and SUN004.2 hPSCs; <sup>∗</sup>p&nbsp;&lt;&nbsp;0.05; n.s.&nbsp;= not significant.</div><div id="fspara0275" role="paragraph">See also <a id="crosref2610" href="#figs7">Figure&nbsp;S7</a> and <a id="crosref2615" href="#mmc4">Table&nbsp;S4</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs7" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2022.05.024/asset/adaa1a89-15fa-4daa-ab97-6d3838b66dae/main.assets/figs7_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs7" src="/cms/10.1016/j.cell.2022.05.024/asset/2d96a5bd-32b2-43c6-beb0-590e0a2017f6/main.assets/figs7.jpg" height="710" width="761" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/adaa1a89-15fa-4daa-ab97-6d3838b66dae/main.assets/figs7_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs7-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S7</span> <span class="figure__title__text">Effects of Nipah and Hendra virus infection on artery and vein ECs, related to <a id="crosref2730" href="#fig7">Figure&nbsp;7</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs7-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0540" role="paragraph">(A) RNA-seq analysis of hPSC-derived artery ECs infected with either Nipah, Hendra, or Sendai virus for 5, 10, or 15 h, followed by alignment of viral reads to viral transcripts, confirms the known 3′-to-5′ transcriptional gradient along the paramyxovirus genome, with 3′ most gene (<i>N</i>)&nbsp;expressed at the highest level, followed by decreasing expression of progressively more 5′ genes. The y axis represents viral mRNA expression relative to the entire transcriptome of infected artery ECs (100%&nbsp;= all human&nbsp;+ viral reads). Average data from H1 and SUN004.1.9 hPSCs shown.</div><div id="fspara0545" role="paragraph">(B) Expression of each individual viral gene in hPSC-derived artery ECs infected with either Nipah, Hendra, or Sendai virus. Percentage indicates the levels at which each individual viral gene (e.g., <i>N</i>) was expressed relative to the entirety of all viral gene expression (100%&nbsp;= all viral reads). Data were averaged from H1 and SUN004.1.9 hPSCs infected for 5, 10, or 15 h.</div><div id="fspara0550" role="paragraph">(C) Differentially expressed genes between infected versus uninfected artery ECs by RNA-seq, 10 and 15&nbsp;h post-infection; averages of H1 and SUN004.1.9 hPSC-derived cell types shown.</div><div id="fspara0555" role="paragraph">(D) IFNβ secretion by hPSC-derived artery ECs as quantified by ELISA; averaged from H1, SUN004.1.9, and SUN004.2 hPSCs; <sup>∗</sup>p&nbsp;&lt;&nbsp;0.01; n.s.&nbsp;= not significant.</div></div></div></div></figcaption></figure></div><div id="p0180" role="paragraph">Between 5 and 15&nbsp;h post-infection, Nipah and Hendra virus RNA replicated extensively in artery ECs (114.5- and 93.2-fold increase, respectively), relative to Sendai virus, which achieved limited (2.9-fold) RNA replication (<a id="crosref1480" href="#fig7">Figure&nbsp;7</a>A). After 15 h, Nipah occupied 17.2% of the transcriptome (∼1 in 6 mRNAs encoding Nipah genes) and Hendra occupied 6.4% of the transcriptome (<a id="crosref1485" href="#fig7">Figure&nbsp;7</a>A). Therefore, Nipah and Hendra viruses replicate significantly more efficiently than the non-pathogenic Sendai virus in artery ECs.</div><div id="p0185" role="paragraph">Despite immense viral replication and cell fusion, Nipah and Hendra viruses led to surprisingly minimal host gene expression changes: they were remarkably effective at suppressing interferon production. Nipah virus massively replicated in artery ECs, yet it did not upregulate or downregulate a single gene across the entire transcriptome to a statistically significant extent, at 5–10&nbsp;h post-infection (<a id="crosref1490" href="#fig7">Figures&nbsp;7</a>B and <a id="crosref1495" href="#figs7">S7</a>C). Similar results were observed for Hendra virus (<a id="crosref1500" href="#fig7">Figures&nbsp;7</a>B and <a id="crosref1505" href="#figs7">S7</a>C). Even when Nipah and Hendra virus RNA achieved extremely high levels at later time points (6.4%–17.2% of the transcriptome, at 15&nbsp;h post-infection), artery ECs minimally upregulated <i>IFNβ1</i>, <i>IFNλ1</i>, and interferon target genes <i>IFIT2</i> and <i>IFIT3</i> (all less than 6-fold upregulation; <a id="crosref1510" href="#fig7">Figures&nbsp;7</a>C and 7D). Despite high viral loads, Nipah- or Hendra-infected artery ECs did not secrete the antiviral cytokine IFNβ to a statistically significant degree at 12 or 24&nbsp;h post-infection (<a id="crosref1515" href="#fig7">Figure&nbsp;7</a>E). This starkly differed from Sendai virus, which induced a 100.8-fold elevation in IFNβ secretion within 24&nbsp;h (<a id="crosref1520" href="#fig7">Figure&nbsp;7</a>E). Despite low viral loads (&lt;0.8% of the transcriptome), Sendai potently induced antiviral genes, including type I/III interferons (<span class="dropBlock reference-citations"><a id="body-ref-sref67-2" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-2" aria-controls="sref67-2">López et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref67-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>López, C.B. ∙ Yount, J.S. ∙ Hermesh, T. ...</div><div><strong>Sendai virus infection induces efficient adaptive immunity independently of type I interferons</strong></div><div><em>J.&nbsp;Virol.</em> 2006; <strong>80</strong>:4538-4545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.80.9.4538-4545.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646184859" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16611914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.80.9.4538-4545.2006&amp;pmid=16611914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and interferon response genes that restrict viral replication (<span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Shaw et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Shaw, A.E. ∙ Hughes, J. ∙ Gu, Q. ...</div><div><strong>Fundamental properties of the mammalian innate immune system revealed by multispecies comparison of type I interferon responses</strong></div><div><em>PLoS Biol.</em> 2017; <strong>15</strong>:e2004086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.2004086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85040002649" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29253856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.2004086&amp;pmid=29253856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref1535" href="#fig7">Figures&nbsp;7</a>C and 7D). Additionally, Sendai-infected artery ECs elaborated immune cytokines, chemokines, and immune-cell adhesion molecules known to recruit immune cells; these antiviral and inflammatory genes were minimally induced by Nipah or Hendra infection (<a id="crosref1540" href="#fig7">Figures&nbsp;7</a>C and 7D). Normalizing to viral loads, Nipah and Hendra viruses induced <i>IFNλ1</i> to a much lower extent (69.5- and 45.1-fold, respectively) than Sendai virus (<a id="crosref1545" href="#fig7">Figure&nbsp;7</a>D). Likewise, Nipah and Hendra viruses upregulated interferon target gene <i>IFIT2</i> much less effectively (109.2- and 23.2-fold, respectively) than Sendai virus (<a id="crosref1550" href="#fig7">Figure&nbsp;7</a>D).</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0190" role="paragraph">To precisely induce desired cellular subtypes, here, we chart a stepwise road map for hPSC differentiation into artery versus vein subtypes of ECs. We mapped the temporally dynamic extracellular signals that drive or inhibit each step of differentiation. At the bifurcation between artery versus vein subtypes, we precisely induced artery ECs by repressing vein-specifying signals and vice versa. This approach generated &gt;90% pure populations of artery- versus vein-specific ECs within 3–4&nbsp;days of hPSC differentiation. The ability to efficiently and rapidly generate artery and vein ECs “on demand” should avail regenerative medicine, tissue engineering, and modeling of human vascular diseases. Importantly, these hPSC-derived artery and vein ECs were distinguished by clear transcriptional and functional differences. Here, we emphasize a pivotal distinction between artery and vein ECs: Nipah and Hendra viruses preferentially target artery ECs <i>in&nbsp;vitro</i>. In turn, this subtype-specific viral tropism emphasizes the importance of producing specific cellular subtypes <i>in&nbsp;vitro</i> and provides a vista on the target cells of deadly viruses.</div><section id="sec-3-1"><h3>A road map for human artery versus vein development</h3><div id="p0195" role="paragraph">First, we differentiated hPSCs into MIXL1<sup>+</sup> primitive streak and then SCL<sup>+</sup> HAND1<sup>+</sup> lateral mesoderm within 1 and 2&nbsp;days, respectively. Subsequently, hPSC-derived lateral mesoderm bifurcated into &gt;90% pure SOX17<sup>+</sup> artery ECs (day 3) or, alternatively, &gt;90% pure NR2F2<sup>+</sup> vein ECs (day 4). Thus, artery ECs arise prior to vein ECs <i>in&nbsp;vitro</i>, consistent with <i>in&nbsp;vivo</i> findings (<span class="dropBlock reference-citations"><a id="body-ref-sref18-6" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-6" aria-controls="sref18-6">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—these two cellular subtypes do not arise synchronously.</div></section><section id="sec-3-2"><h3>Inducing cellular subtypes: Blocking the formation of alternative subtypes</h3><div id="p0200" role="paragraph">As opposed to generating “generic” cell types of ambiguous subtype identity, we hypothesized it should be possible to produce desired cellular subtypes by explicitly <i>inhibiting</i> the signals that induce the opposing cellular subtype. At the bifurcation between artery and vein subtypes, TGF-β and NOTCH specify artery fate, whereas PI3K instructs vein identity. We induced artery ECs through a two-pronged approach: activating artery-inducing signal TGF-β, while repressing vein-specifying signal PI3K. The converse signals—TGF-β and NOTCH inhibition (together with PI3K activation)—instead induced vein ECs. This signaling logic enabled the exclusive generation of either artery or vein ECs in the absence of the opposing lineage, as quantified by scRNA-seq and other means. Our findings thus reveal a more complex signaling code for artery versus vein specification, extending beyond the current manipulation of VEGF and/or NOTCH (reviewed by <span class="dropBlock reference-citations"><a id="body-ref-sref23-4" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-4" aria-controls="sref23-4">Coultas et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref23-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Coultas, L. ∙ Chawengsaksophak, K. ∙ Rossant, J.</div><div><strong>Endothelial cells and VEGF in vascular development</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:937-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04479" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744432619" target="_blank" aria-label="Open Scopus in new tab">Scopus (784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04479&amp;pmid=16355211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">Fang and Hirschi, 2019</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Fang, J. ∙ Hirschi, K.</div><div><strong>Molecular regulation of arteriovenous endothelial cell specification</strong></div><div><em>F1000Res.</em> 2019; <strong>8</strong>:1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.16701.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85071923734" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.16701.1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36-4" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-4" aria-controls="sref36-4">Fish and Wythe, 2015</a><div class="dropBlock__holder" data-db-target-of="sref36-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Fish, J.E. ∙ Wythe, J.D.</div><div><strong>The molecular regulation of arteriovenous specification and maintenance</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:391-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924032549" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25641373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24252&amp;pmid=25641373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to bias hPSCs toward arterial or venous fates (<span class="dropBlock reference-citations"><a id="body-ref-sref29-5" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29-5" aria-controls="sref29-5">Ditadi et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref29-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Ditadi, A. ∙ Sturgeon, C.M. ∙ Tober, J. ...</div><div><strong>Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:580-591</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928938612" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3161&amp;pmid=25915127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref38-4" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-4" aria-controls="sref38-4">Gage et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref38-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Gage, B.K. ∙ Liu, J.C. ∙ Innes, B.T. ...</div><div><strong>Generation of functional liver sinusoidal endothelial cells from human pluripotent stem-cell-derived venous angioblasts</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:254-269.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85088364652" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640183/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.06.007&amp;pmid=32640183" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91-5" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91-5" aria-controls="sref91-5">Park et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref91-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Park, M.A. ∙ Kumar, A. ∙ Jung, H.S. ...</div><div><strong>Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential</strong></div><div><em>Cell Rep.</em> 2018; <strong>23</strong>:2467-2481</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047204081" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29791856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.04.092&amp;pmid=29791856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref102-3" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-3" aria-controls="sref102-3">Rosa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref102-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Rosa, S. ∙ Praça, C. ∙ Pitrez, P.R. ...</div><div><strong>Functional characterization of iPSC-derived arterial- and venous-like endothelial cells</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:3826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-40417-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062643820" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30846769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-40417-9&amp;pmid=30846769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-6" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-6" aria-controls="sref110-6">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-9" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-9" aria-controls="sref126-9">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Explicitly “inhibiting” the signals that specify alternate subtype identities may be a general approach to precisely engineer desired cellular subtypes from stem cells.</div></section><section id="sec-3-3"><h3>Inducing cellular subtypes: Temporally dynamic signaling</h3><div id="p0205" role="paragraph">hPSCs swiftly traverse alternating lineage paths every 24 h, and consequently, extracellular signals must be rapidly activated or inhibited to precisely guide differentiation down desired paths in preference to unwanted ones. Indeed, in the mouse embryo, cells rapidly segue between different states every 12–24 h: E5.5 pluripotent cells → E6.5 primitive streak → ∼E7–E7.5 lateral mesoderm → ∼E8.25 artery ECs, and ∼E9.0 vein ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref18-7" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-7" aria-controls="sref18-7">Chong et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref18-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found the same extracellular signal specified 3 different cell types over the course of 3&nbsp;days. For instance, TGF-β initially induced mid primitive streak on day 1, then repressed lateral mesoderm on day 2, and finally induced arterial ECs on day 3. By contrast, VEGF induced lateral mesoderm on day 2 and then induced and eventually inhibited vein ECs on days 3 and 4, respectively. Temporally dynamic activation and repression of extracellular signals was required to efficiently transition cells from one state to the next, with two consecutive cell types sometimes sharing diametrically opposed requirements for the same signal (e.g., VEGF) <i>in&nbsp;vitro</i>, consistent with VEGF’s temporally dynamic role <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref14-2" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14-2" aria-controls="sref14-2">Chetty et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref14-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Casie Chetty, S. ∙ Rost, M.S. ∙ Enriquez, J.R. ...</div><div><strong>Vegf signaling promotes vascular endothelial differentiation by modulating etv2 expression</strong></div><div><em>Dev. Biol.</em> 2017; <strong>424</strong>:147-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2017.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85014796623" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28279709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2017.03.005&amp;pmid=28279709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref60-2" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60-2" aria-controls="sref60-2">Lawson et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref60-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Lawson, N.D. ∙ Vogel, A.M. ∙ Weinstein, B.M.</div><div><strong>sonic hedgehog and vascular endothelial growth factor act upstream of the Notch pathway during arterial endothelial differentiation</strong></div><div><em>Dev. Cell.</em> 2002; <strong>3</strong>:127-136</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1534-5807%2802%2900198-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1534-5807%2802%2900198-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036070186" target="_blank" aria-label="Open Scopus in new tab">Scopus (650)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12110173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1534-5807%2802%2900198-3&amp;pmid=12110173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Shalaby et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Shalaby, F. ∙ Rossant, J. ∙ Yamaguchi, T.P. ...</div><div><strong>Failure of blood-island formation and vasculogenesis in Flk-1-deficient mice</strong></div><div><em>Nature.</em> 1995; <strong>376</strong>:62-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/376062a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0029006696" target="_blank" aria-label="Open Scopus in new tab">Scopus (3332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7596435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F376062a0&amp;pmid=7596435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consequently, prolonged activation or inhibition of a given signal generates a heterogeneous cell population. The previously overlooked requirement for temporally dynamic TGF-β and VEGF signals likely complicated past efforts to generate artery and vein ECs from hPSCs. It is therefore critical to map the temporal dynamics with which these signals operate, and to manipulate them with equal dynamism to precisely effect differentiation.</div></section><section id="sec-3-4"><h3>Artery ECs are target cells of Nipah and Hendra viruses</h3><div id="p0210" role="paragraph">By virtue of their high fatality rates and a general dearth of approved treatments, risk-group-4 viruses imperil global health: mapping their human target cells and their effects on such cells is crucial. Arteries and veins have largely been regarded synonymously in viral diseases. However, we found Nipah and Hendra viruses preferentially targeted artery ECs and fused them into multinucleated arterial ECs <i>in&nbsp;vitro</i>. This is consonant with a past study of Syrian hamsters, which suggested that Nipah primarily infected arteries <i>in&nbsp;vivo</i>, as morphologically inferred by elastic membrane staining (<span class="dropBlock reference-citations"><a id="body-ref-sref8-3" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-3" aria-controls="sref8-3">Baseler et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Baseler, L. ∙ de Wit, E. ∙ Scott, D.P. ...</div><div><strong>Syrian hamsters (Mesocricetus auratus) oronasally inoculated with a Nipah virus isolate from Bangladesh or Malaysia develop similar respiratory tract lesions</strong></div><div><em>Vet. Pathol.</em> 2015; <strong>52</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985814556189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84920024648" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25352203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985814556189&amp;pmid=25352203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hence, as opposed to promiscuously targeting all ECs, we demonstrate that certain viruses may preferentially target specific endothelial subtypes. Mechanistically, this arterial tropism was driven at least in part by higher expression of viral-entry receptor <i>EFNB2</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref10-4" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-4" aria-controls="sref10-4">Bonaparte et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref10-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Bonaparte, M.I. ∙ Dimitrov, A.S. ∙ Bossart, K.N. ...</div><div><strong>Ephrin-B2 ligand is a functional receptor for Hendra virus and Nipah virus</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:10652-10657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0504887102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-23044517161" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15998730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0504887102&amp;pmid=15998730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-4" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-4" aria-controls="sref81-4">Negrete et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref81-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Negrete, O.A. ∙ Levroney, E.L. ∙ Aguilar, H.C. ...</div><div><strong>EphrinB2 is the entry receptor for Nipah virus, an emergent deadly paramyxovirus</strong></div><div><em>Nature.</em> 2005; <strong>436</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-22944445501" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16007075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03838&amp;pmid=16007075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in artery ECs, which we demonstrated was required for viral entry, as shown by genetic deletion. The developmental diversification of artery versus vein ECs may thus hold hitherto-unappreciated significance in viral diseases.</div><div id="p0215" role="paragraph">Past studies infected various ECs sourced from a range of species and organs with Nipah and Hendra viruses <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref1-3" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-3" aria-controls="sref1-3">Aljofan et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref1-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Aljofan, M. ∙ Saubern, S. ∙ Meyer, A.G. ...</div><div><strong>Characteristics of Nipah virus and Hendra virus replication in different cell lines and their suitability for antiviral screening</strong></div><div><em>Virus Res.</em> 2009; <strong>142</strong>:92-99</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virusres.2009.01.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-67349241274" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19428741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virusres.2009.01.014&amp;pmid=19428741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34-2" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34-2" aria-controls="sref34-2">Erbar et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref34-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Erbar, S. ∙ Diederich, S. ∙ Maisner, A.</div><div><strong>Selective receptor expression restricts Nipah virus infection of endothelial cells</strong></div><div><em>Virol. J.</em> 2008; <strong>5</strong>:142</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1743-422X-5-142" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-58149168980" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19036148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1743-422X-5-142&amp;pmid=19036148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-3" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-3" aria-controls="sref64-3">Lo et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref64-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Lo, M.K. ∙ Miller, D. ∙ Aljofan, M. ...</div><div><strong>Characterization of the antiviral and inflammatory responses against Nipah virus in endothelial cells and neurons</strong></div><div><em>Virology.</em> 2010; <strong>404</strong>:78-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2010.05.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77953714922" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20552729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2010.05.005&amp;pmid=20552729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-3" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-3" aria-controls="sref73-3">Mathieu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref73-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Mathieu, C. ∙ Guillaume, V. ∙ Sabine, A. ...</div><div><strong>Lethal Nipah virus infection induces rapid overexpression of CXCL10</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e32157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84857681004" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22393386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032157&amp;pmid=22393386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref77-3" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77-3" aria-controls="sref77-3">Monaghan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref77-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Monaghan, P. ∙ Green, D. ∙ Pallister, J. ...</div><div><strong>Detailed morphological characterisation of Hendra virus infection of different cell types using super-resolution and conventional imaging</strong></div><div><em>Virol. J.</em> 2014; <strong>11</strong>:200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12985-014-0200-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84936855081" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12985-014-0200-5&amp;pmid=25428656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105-2" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-2" aria-controls="sref105-2">Schountz et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref105-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Schountz, T. ∙ Campbell, C. ∙ Wagner, K. ...</div><div><strong>Differential innate immune responses elicited by Nipah virus and cedar virus correlate with disparate in&nbsp;vivo pathogenesis in hamsters</strong></div><div><em>Viruses.</em> 2019; <strong>11</strong></div><div>291–211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/v11030291" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85063712957" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30909389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fv11030291&amp;pmid=30909389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, these ECs were cultured in conditions wherein arteriovenous subtype identity was lost (<span class="dropBlock reference-citations"><a id="body-ref-sref4-3" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4-3" aria-controls="sref4-3">Aranguren et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref4-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aranguren, X.L. ∙ Agirre, X. ∙ Beerens, M. ...</div><div><strong>Unraveling a novel transcription factor code determining the human arterial-specific endothelial cell signature</strong></div><div><em>Blood.</em> 2013; <strong>122</strong>:3982-3992</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2013-02-483255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84891126258" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24108462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2013-02-483255&amp;pmid=24108462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), often unbeknownst to researchers. This thus accentuates the importance of being able to generate and maintain well-defined populations of hPSC-derived artery and vein ECs <i>in&nbsp;vitro</i> to study whether viruses preferentially target specific EC subtypes and exert subtype-specific effects.</div><div id="p0220" role="paragraph">Having identified artery ECs as preferential target cells of Nipah and Hendra viruses, we systematically detailed the effects of these viruses on artery ECs using live imaging and RNA-seq. One of the greatest surprises was that Nipah and Hendra viruses inflicted surprisingly minor transcriptional effects on artery ECs, despite extensive viral replication and prodigious cell fusion. Indeed, both viruses were “stealthy” to the innate immune system of artery ECs. Despite occupying an impressive fraction (6.4%–17.2%) of an artery cell’s transcriptome, both Nipah and Hendra viruses minimally induced interferon signaling or antiviral genes. This minimal interferon response correlated with the ability of these lethal viruses to massively replicate and achieve high viral loads, in stark contrast with the non-pathogenic Sendai virus. Indeed, Nipah virus P, V, and W proteins are known to silence interferon production and signaling (<span class="dropBlock reference-citations"><a id="body-ref-sref17" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17" aria-controls="sref17">Childs et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Childs, K. ∙ Stock, N. ∙ Ross, C. ...</div><div><strong>mda-5, but not RIG-I, is a common target for paramyxovirus V proteins</strong></div><div><em>Virology.</em> 2007; <strong>359</strong>:190-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2006.09.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33846863325" target="_blank" aria-label="Open Scopus in new tab">Scopus (250)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17049367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2006.09.023&amp;pmid=17049367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Ciancanelli et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Ciancanelli, M.J. ∙ Volchkova, V.A. ∙ Shaw, M.L. ...</div><div><strong>Nipah virus sequesters inactive STAT1 in the nucleus via a P gene-encoded mechanism</strong></div><div><em>J.&nbsp;Virol.</em> 2009; <strong>83</strong>:7828-7841</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02610-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-67749142255" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19515782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02610-08&amp;pmid=19515782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Rodriguez et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Rodriguez, J.J. ∙ Parisien, J.-P. ∙ Horvath, C.M.</div><div><strong>Nipah virus V protein evades alpha and gamma interferons by preventing STAT1 and STAT2 activation and nuclear accumulation</strong></div><div><em>J.&nbsp;Virol.</em> 2002; <strong>76</strong>:11476-11483</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jvi.76.22.11476-11483.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036827851" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12388709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjvi.76.22.11476-11483.2002&amp;pmid=12388709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Past studies reached diverging conclusions regarding whether Nipah virus activates (<span class="dropBlock reference-citations"><a id="body-ref-sref33" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33" aria-controls="sref33">Elvert et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Elvert, M. ∙ Sauerhering, L. ∙ Maisner, A.</div><div><strong>Cytokine induction in Nipah virus-infected primary human and porcine bronchial epithelial cells</strong></div><div><em>J.&nbsp;Infect. Dis.</em> 2020; <strong>221</strong>:S395-S400</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/infdis/jiz455" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85084693241" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31665348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Finfdis%2Fjiz455&amp;pmid=31665348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-4" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-4" aria-controls="sref64-4">Lo et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref64-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Lo, M.K. ∙ Miller, D. ∙ Aljofan, M. ...</div><div><strong>Characterization of the antiviral and inflammatory responses against Nipah virus in endothelial cells and neurons</strong></div><div><em>Virology.</em> 2010; <strong>404</strong>:78-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2010.05.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77953714922" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20552729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2010.05.005&amp;pmid=20552729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref73-4" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-4" aria-controls="sref73-4">Mathieu et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref73-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Mathieu, C. ∙ Guillaume, V. ∙ Sabine, A. ...</div><div><strong>Lethal Nipah virus infection induces rapid overexpression of CXCL10</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e32157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84857681004" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22393386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032157&amp;pmid=22393386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), or fails to elicit (<span class="dropBlock reference-citations"><a id="body-ref-sref105-3" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105-3" aria-controls="sref105-3">Schountz et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref105-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Schountz, T. ∙ Campbell, C. ∙ Wagner, K. ...</div><div><strong>Differential innate immune responses elicited by Nipah virus and cedar virus correlate with disparate in&nbsp;vivo pathogenesis in hamsters</strong></div><div><em>Viruses.</em> 2019; <strong>11</strong></div><div>291–211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/v11030291" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85063712957" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30909389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fv11030291&amp;pmid=30909389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Virtue et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Virtue, E.R. ∙ Marsh, G.A. ∙ Wang, L.F.</div><div><strong>Interferon signaling remains functional during Henipavirus infection of human cell lines</strong></div><div><em>J.&nbsp;Virol.</em> 2011; <strong>85</strong>:4031-4034</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02412-10" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79952836894" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21289115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02412-10&amp;pmid=21289115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), interferon signaling <i>in&nbsp;vitro</i>. Here, we re-investigate this question using well-defined populations of artery and vein ECs. Additionally, we quantify the extent of interferon suppression “relative” to respective viral RNA loads, using RNA-seq. Our work also illuminates the cellular effects of Hendra virus, which hitherto were generally understudied relative to, and assumed to be similar to, Nipah virus.</div></section><section id="sec-3-5"><h3>A platform to study risk-group-4 viruses using hPSC-derived cell types</h3><div id="p0225" role="paragraph">More broadly, we introduce hPSC-derived cell types as enhanced model systems to study the target cells and cellular effects of risk-group-4 viruses. Due to BSL4 experimental constraints, <i>in&nbsp;vitro</i> studies of risk-group-4 viruses have hitherto exploited cancer cells (which poorly reflect native physiology) or primary cells (which are difficult to expand long term and to genetically edit, and often relinquish features of their <i>in&nbsp;vivo</i> counterparts), neither of which may harbor clear subtype identities (e.g., arteriovenous identity). hPSC-derived cell types can be rapidly generated in large numbers, are genetically manipulatable, and retain certain properties of their <i>in&nbsp;vivo</i> counterparts (e.g., subtype identity) (<span class="dropBlock reference-citations"><a id="body-ref-sref30" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30" aria-controls="sref30">Drubin and Hyman, 2017</a><div class="dropBlock__holder" data-db-target-of="sref30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Drubin, D.G. ∙ Hyman, A.A.</div><div><strong>Stem cells: the new “model organism”</strong></div><div><em>Mol. Biol. Cell.</em> 2017; <strong>28</strong>:1409-1411</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E17-03-0183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85020120205" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28559439/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E17-03-0183&amp;pmid=28559439" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These decisive advantages of hPSC-derived cell types have already been exploited to provide new insights into risk-group-2 and -3 viruses (<span class="dropBlock reference-citations"><a id="body-ref-sref46" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46" aria-controls="sref46">Harschnitz and Studer, 2021</a><div class="dropBlock__holder" data-db-target-of="sref46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Harschnitz, O. ∙ Studer, L.</div><div><strong>Human stem cell models to study host-virus interactions in the central nervous system</strong></div><div><em>Nat. Rev. Immunol.</em> 2021; <strong>21</strong>:441-453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-020-00474-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85098797496" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33398129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-020-00474-y&amp;pmid=33398129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">Simoneau and Ott, 2020</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Simoneau, C.R. ∙ Ott, M.</div><div><strong>Modeling multi-organ infection by SARS-CoV-2&nbsp;using stem cell technology</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:859-868</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.11.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.11.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096979593" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.11.012&amp;pmid=33275899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Indeed, hPSC-derived ECs exemplify many of these advantages: they exhibit arteriovenous differences in <i>EFNB2</i> expression (which are otherwise obscured in primary ECs cultured in conventional conditions) and are genetically tractable (as evinced by our genetic deletion of <i>EFNB2</i> to confirm its role in viral entry).</div><div id="p0230" role="paragraph">Beyond Nipah and Hendra viruses, hPSC-derived artery and vein ECs could provide a powerful platform to delineate the target cells and cellular effects of additional risk-group-4 viruses—including arenaviruses, filoviruses, flaviviruses, and nairoviruses—that cause lethal and enigmatic vascular diseases. Eventually hPSC-derived cell types could enable therapeutic and genetic screens to discover countermeasures against currently incurable risk-group-4 viruses, using cell types that are more physiologically relevant than the cancer cells often used in BSL4 experiments.</div></section><section id="sec-3-6"><h3>Limitations of the study</h3><div id="p0235" role="paragraph">The endothelial road map described here remains incomplete: it focuses on artery and vein ECs but does not encompass generation of capillaries, lymphatics, or organ-specific ECs (<span class="dropBlock reference-citations"><a id="body-ref-sref6-3" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6-3" aria-controls="sref6-3">Augustin and Koh, 2017</a><div class="dropBlock__holder" data-db-target-of="sref6-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Augustin, H.G. ∙ Koh, G.Y.</div><div><strong>Organotypic vasculature: From descriptive heterogeneity to functional pathophysiology</strong></div><div><em>Science.</em> 2017; <strong>357</strong>:eaal2379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aal2379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026996599" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28775214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aal2379&amp;pmid=28775214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83" aria-controls="sref83">Nguyen et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref83"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Nguyen, J. ∙ Lin, Y.-Y. ∙ Gerecht, S.</div><div><strong>The next generation of endothelial differentiation: tissue-specific ECs</strong></div><div><em>Cell Stem Cell.</em> 2021; <strong>28</strong>:1188-1204</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2021.05.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2021.05.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85108876397" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34081899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2021.05.002&amp;pmid=34081899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94-4" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94-4" aria-controls="sref94-4">Potente and Mäkinen, 2017</a><div class="dropBlock__holder" data-db-target-of="sref94-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Potente, M. ∙ Mäkinen, T.</div><div><strong>Vascular heterogeneity and specialization in development and disease</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.36" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026911538" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28537573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.36&amp;pmid=28537573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Creating an increasingly diverse array of ECs <i>in&nbsp;vitro</i> will, in turn, avail virology. Nipah and Hendra viruses infect ECs across multiple organs <i>in&nbsp;vivo</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref8-4" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8-4" aria-controls="sref8-4">Baseler et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref8-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Baseler, L. ∙ de Wit, E. ∙ Scott, D.P. ...</div><div><strong>Syrian hamsters (Mesocricetus auratus) oronasally inoculated with a Nipah virus isolate from Bangladesh or Malaysia develop similar respiratory tract lesions</strong></div><div><em>Vet. Pathol.</em> 2015; <strong>52</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985814556189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84920024648" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25352203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985814556189&amp;pmid=25352203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-3" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-3" aria-controls="sref40-3">Geisbert et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref40-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Geisbert, T.W. ∙ Daddario-DiCaprio, K.M. ∙ Hickey, A.C. ...</div><div><strong>Development of an acute and highly pathogenic nonhuman primate model of Nipah virus infection</strong></div><div><em>PLoS One.</em> 2010; <strong>5</strong>:e10690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0010690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77956283436" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20502528/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0010690&amp;pmid=20502528" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-4" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-4" aria-controls="sref78-4">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref122-3" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122-3" aria-controls="sref122-3">Wong et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref122-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Wong, K.T. ∙ Shieh, W.-J. ∙ Kumar, S. ...</div><div><strong>Nipah virus infection: pathology and pathogenesis of an emerging paramyxoviral zoonosis</strong></div><div><em>Am. J. Pathol.</em> 2002; <strong>161</strong>:2153-2167</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=fulltext&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=pdf&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036899143" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12466131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0002-9440%2810%2964493-8&amp;pmid=12466131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but how organ-specific ECs respond to these viruses remains a mystery. Finally, while we find Nipah and Hendra viruses preferentially target human artery ECs <i>in&nbsp;vitro</i>, <i>in&nbsp;vivo</i> verification is of paramount import. In this regard, it is critical to appreciate that cells <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> are not equivalent (<span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">Fowler et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Fowler, J.L. ∙ Ang, L.T. ∙ Loh, K.M.</div><div><strong>A critical look: challenges in differentiating human pluripotent stem cells into desired cell types and organoids</strong></div><div><em>Wiley Interdiscip. Rev.: Dev. Biol.</em> 2019; <strong>113</strong></div><div>891–823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/wdev.368" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85075149381" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fwdev.368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): hPSC-derived artery and vein ECs are transcriptionally similar, but are not identical to, their respective <i>in&nbsp;vivo</i> counterparts. Going forward, we envision that hPSC-derived cell types will provide a powerful <i>in&nbsp;vitro</i> platform to explore the tropism and biology of risk-group-4 viruses, generating hypotheses that can then be tested in <i>de rigueur</i> animal models <i>in&nbsp;vivo</i>.</div></section></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key resources table</h3><div id="p0240" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>anti-CD144 Alexa Fluor 647 antibody</td><td>BD</td><td>561567</td></tr><tr><td>anti-human CD73 (Ecto-5'-nucleotidase) PE</td><td>Biolegend</td><td>344004</td></tr><tr><td>anti-human Delta-like protein 4 (DLL4) APC</td><td>Biolegend</td><td>346508</td></tr><tr><td>Anti-<i>NR2F2</i> antibody</td><td>Abcam</td><td>ab211777 (1:100)</td></tr><tr><td>Anti-SOX17 antibody</td><td>R&amp;D Systems</td><td>AF-1924 (1:1000)</td></tr><tr><td>Anti-CD31</td><td>BD Biosciences</td><td>555444</td></tr><tr><td>Anti-ERG (ETS related gene)</td><td>Abcam</td><td>ab92513 (1:500)</td></tr><tr><td>Mouse Anti-Human CD144 (VE-Cadherin)</td><td>BD Bioscience</td><td>555661 (1:500)</td></tr><tr><td>Anti-EFNB2</td><td>R&amp;D Systems</td><td>AF496 (1:5)</td></tr><tr><td>APC Donkey Anti-Goat</td><td>R&amp;D Systems</td><td>F0108 (1:10)</td></tr><tr><td>Alexa Fluor 568 Donkey Anti-rabbit</td><td>Thermo Fisher Scientific</td><td>1:250</td></tr><tr class="rowsep"><td>Alexa Fluor 647 Donkey Anti-mouse</td><td>Thermo Fisher Scientific</td><td>1:250</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and viral strains</b></td></tr><tr><td>Nipah virus (Malaysia variant): Nipah virus/human/MY/1999/CDC</td><td>NIH/NIAID Rocky Mountain Laboratories</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref45-1" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-1" aria-controls="sref45-1">Harcourt et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref45-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Harcourt, B.H. ∙ Tamin, A. ∙ Ksiazek, T.G. ...</div><div><strong>Molecular characterization of Nipah virus, a newly emergent paramyxovirus</strong></div><div><em>Virology.</em> 2000; <strong>271</strong>:334-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/viro.2000.0340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0034608231" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10860887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fviro.2000.0340&amp;pmid=10860887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>Hendra virus (Brisbane variant): Hendra virus/horse/AU/1994/Hendra</td><td>NIH/NIAID Rocky Mountain Laboratories</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref78-5" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-5" aria-controls="sref78-5">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>Sendai virus (Cantell strain)</td><td>ATCC</td><td>VR-907</td></tr><tr><td>Sendai virus (recombinant, replication-defective, encoding <i>KLF4</i>/<i>OCT4/SOX2/MYC</i> and <i>miR-302</i> target site)</td><td>Dr. Mahito Nakanishi, National Institute of Advanced Industrial Science and Technology</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Nishimura et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Nishimura, K. ∙ Ohtaka, M. ∙ Takada, H. ...</div><div><strong>Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302</strong></div><div><em>Stem Cell Res.</em> 2017; <strong>23</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2017.06.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021304442" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28666145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2017.06.011&amp;pmid=28666145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr class="rowsep"><td>NEB® Stable Competent <i>E.&nbsp;coli</i></td><td>New England BioLabs</td><td>C3040H</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, peptides and recombinant proteins</b></td></tr><tr><td>mTesR Plus medium</td><td>STEMCELL Technologies</td><td>#100-0276</td></tr><tr><td>Penicillin/streptomycin</td><td>Thermo Fisher Scientific</td><td>15-140-122</td></tr><tr><td>KnockOut Serum Replacement</td><td>Thermo Fisher Scientific</td><td>10828028</td></tr><tr><td>MEM Non Essential Amino Acids Solution (100X)</td><td>Thermo Fisher Scientific</td><td>11-140-050</td></tr><tr><td>Geltrex LDEV-Free, hESC-Qualified, Reduced Growth Factor Basement Membrane Matrix</td><td>Thermo Fisher Scientific</td><td>A1413302</td></tr><tr><td>Recombinant Human Vitronectin Protein, Truncated (VTN-N)</td><td>Gibco</td><td>A14700</td></tr><tr><td>Matrigel Matrix Growth Factor Reduced (GFR), Phenol Red Free (RF)</td><td>BD Biosciences</td><td>356231</td></tr><tr><td>Gelatin Solution</td><td>Sigma</td><td>ES-006-B</td></tr><tr><td>Versene solution</td><td>Thermo Fisher Scientific</td><td>15040066</td></tr><tr><td>Trypsin-EDTA, 0.05%</td><td>Thermo Fisher Scientific</td><td>25300054 or 25300062</td></tr><tr><td>Accutase - Enzyme Cell Detachment Medium</td><td>Thermo Fisher Scientific</td><td>00-4555-56</td></tr><tr><td>TrypLE Express Enzyme (1X)</td><td>Thermo Fisher Scientific</td><td>12604013</td></tr><tr><td>Fetal Bovine Serum Premium Select</td><td>Atlanta Biologicals</td><td>S11550H</td></tr><tr><td>2-mercaptoethanol</td><td>Gibco</td><td>21985023</td></tr><tr><td>RPMI 1640 Medium, GlutaMAX</td><td>Thermo Fisher Scientific</td><td>61870036</td></tr><tr><td>DMEM</td><td>Sigma</td><td>D6429</td></tr><tr><td>DMEM/F12&nbsp;+ GlutaMAX</td><td>Gibco</td><td>10565042</td></tr><tr><td>IMDM&nbsp;+ GlutaMAX</td><td>Thermo Fisher</td><td>31980-097</td></tr><tr><td>F12&nbsp;+ GlutaMAX</td><td>Thermo Fisher</td><td>31765-092</td></tr><tr><td>Polyvinyl alcohol</td><td>Sigma</td><td>P8136-250G</td></tr><tr><td>Chemically defined lipid concentrate</td><td>Thermo Fisher</td><td>11905-031</td></tr><tr><td>1-thioglycerol</td><td>Sigma</td><td>M6145-100ML</td></tr><tr><td>Recombinant human insulin</td><td>Sigma</td><td>11376497001</td></tr><tr><td>Recombinant human VEGF</td><td>R&amp;D Systems</td><td>293-VE-0500</td></tr><tr><td>Recombinant human FGF2</td><td>R&amp;D Systems</td><td>233-FB-01M</td></tr><tr><td>Human transferrin</td><td>Sigma</td><td>10652202001</td></tr><tr><td>Recombinant human BMP4</td><td>R&amp;D Systems</td><td>314-BP-050</td></tr><tr><td>Penicillin/streptomycin</td><td>Thermo Fisher</td><td>15070-063</td></tr><tr><td>Recombinant Activin</td><td>R&amp;D Systems</td><td>338-AC-500/CF</td></tr><tr><td>GDC-0941</td><td>Cellagen Technology</td><td>C4321-25</td></tr><tr><td>Forskolin</td><td>Tocris</td><td>1099</td></tr><tr><td>Thiazovivin</td><td>Tocris</td><td>3845</td></tr><tr><td>XAV939</td><td>Tocris</td><td>3748</td></tr><tr><td>Ascorbic acid-2-phosphate</td><td>Sigma</td><td>49752-10G</td></tr><tr><td>DMH1</td><td>Tocris</td><td>4126</td></tr><tr><td>SB505124</td><td>Tocris</td><td>3263</td></tr><tr><td>RO4929097</td><td>Cellagen Technology</td><td>C7649-10</td></tr><tr><td>CHIR99201</td><td>Tocris</td><td>4423</td></tr><tr><td>PD0325901</td><td>Tocris</td><td>4192</td></tr><tr><td>PIK90</td><td>Calbiochem</td><td>528117</td></tr><tr><td>8-bromo-cAMP</td><td>Tocris</td><td>1140</td></tr><tr><td>KT5720</td><td>Calbiochem</td><td>420320</td></tr><tr><td>GM-CSF</td><td>Peprotech</td><td>100-11</td></tr><tr><td>IL-3</td><td>Peprotech</td><td>200-03</td></tr><tr><td>IL-6</td><td>Peprotech</td><td>200-06</td></tr><tr><td>FLT3L</td><td>Peprotech</td><td>300-19</td></tr><tr><td>TPO</td><td>R&amp;D Systems</td><td>288-TP-005/CF</td></tr><tr><td>SCF</td><td>Peprotech</td><td>300-07</td></tr><tr><td>Endothelial Cell Growth Medium 2 (EGM2)</td><td>Lonza</td><td>CC-3162</td></tr><tr><td>Endothelial Cell Growth Basal Medium 2 (EBM2)</td><td>Lonza</td><td>CC-3156</td></tr><tr><td>Fibroblast Growth Medium 2 (FGM2)</td><td>Lonza</td><td>CC-3132</td></tr><tr><td>Fibrinogen solution</td><td>Sigma Aldrich</td><td>F8630</td></tr><tr><td>Thrombin</td><td>Sigma Aldrich</td><td>T3399</td></tr><tr><td>Cytodex-3 beads</td><td>Sigma Aldrich</td><td>C3275</td></tr><tr><td>Apelin-13</td><td>Tocris</td><td>2420</td></tr><tr><td>Transwell polycarbonate membrane cell culture inserts, 6.5&nbsp;mm Transwell with 8.0&nbsp;μm pore polycarbonate membrane insert, 24-well</td><td>Corning</td><td>3422</td></tr><tr><td>Lipofectamine 3000</td><td>Thermo Fisher Scientific</td><td>L3000001</td></tr><tr><td>Low-density lipoprotein Dylight 550</td><td>Cayman Chemical</td><td>10011125</td></tr><tr><td>Acetylated Alexa Fluor 488 conjugate low-density lipoprotein</td><td>Thermo Fisher Scientific</td><td>L-23380</td></tr><tr><td>Recombinant Human TNFα</td><td>R&amp;D Systems</td><td>210-TA-020</td></tr><tr><td>FuGENE® HD Transfection Reagent</td><td>Promega</td><td>E2311</td></tr><tr><td>Poly-L-lysine</td><td>R&amp;D Systems</td><td>3438-200-01</td></tr><tr><td>LV-MAX Lentiviral Production System</td><td>Gibco</td><td>A35684</td></tr><tr><td>Bovine serum albumin (BSA)</td><td>Sigma</td><td>A2153</td></tr><tr><td>Tween-20</td><td>Sigma</td><td>P9416</td></tr><tr><td>10% Formalin</td><td>Sigma</td><td>F5554</td></tr><tr><td>Fibrinogen</td><td>Sigma-Aldrich</td><td>F8630</td></tr><tr><td>Thrombin</td><td>Sigma-Aldrich</td><td>T4648</td></tr><tr><td>Vectashield</td><td>Vector Labs</td><td>H-100</td></tr><tr><td>Glutamax</td><td>Gibco</td><td>35050061</td></tr><tr><td>STEMdiff™ APEL™</td><td>StemCell Technologies</td><td>05270</td></tr><tr><td>CP21R7</td><td>Selleckchem</td><td>S7954</td></tr><tr><td>Neurobasal medium</td><td>Thermo Fisher Scientific</td><td>21103049</td></tr><tr><td>B27</td><td>Thermo Fisher Scientific</td><td>17504044</td></tr><tr><td>N2</td><td>Thermo Fisher Scientific</td><td>17502001</td></tr><tr><td>StemPro-34 SFM medium</td><td>Thermo Fisher Scientific</td><td>10639011</td></tr><tr><td>Essential 8 media</td><td>Thermo Fisher Scientific</td><td>A1517001</td></tr><tr><td>SB431542</td><td>Tocris</td><td>1614</td></tr><tr><td>Resveratrol</td><td>R&amp;D Systems</td><td>1418/100</td></tr><tr><td>L690</td><td>R&amp;D Systems</td><td>0682/5</td></tr><tr class="rowsep"><td>Poly(I:C) sodium salt</td><td>Sigma</td><td>P1530</td></tr><tr class="rowsep"><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Human IFNβ Quantikine QuicKit ELISA</td><td>R&amp;D Systems</td><td>QK410</td></tr><tr><td>RNeasy Plus Mini Kit</td><td>Qiagen</td><td>10010049</td></tr><tr><td>QIAamp Viral RNA kit</td><td>Qiagen</td><td>52904</td></tr><tr><td>RNeasy Micro kit</td><td>Qiagen</td><td>74004</td></tr><tr><td>High-Capacity cDNA Reverse Transcription Kit</td><td>Applied Biosystems</td><td>4368814</td></tr><tr><td>AgPath-ID<sup>TM</sup> One-Step RT-PCR Kit</td><td>Thermo&nbsp;Fisher</td><td>4387391</td></tr><tr><td>SYBRGreen</td><td>Thomas Scientific</td><td>BIO-94050</td></tr><tr><td>10x Genomics Chromium single cell 3’ GEM reagent kit (v3)</td><td>10x Genomics</td><td>1000075</td></tr><tr><td>Chromium Chip B Single Cell Kit 48rxns</td><td>10x Genomics</td><td>1000073</td></tr><tr><td>LEGENDScreen PE-Conjugated Human Antibody Plates</td><td>Biolegend</td><td>700007</td></tr><tr><td>Magnetically Activated Cell Sorting (MACS)</td><td>Miltenyi Biotec</td><td>130-046-703</td></tr><tr><td>MACS LS column</td><td>Miltenyi Biotech</td><td>130-042-401</td></tr><tr><td>Fc blocking reagent</td><td>Miltenyi Biotec</td><td>130-046-703</td></tr><tr><td>Triton X-100</td><td>Sigma</td><td>X100-500ML</td></tr><tr><td>PBS, pH 7.4</td><td>Thermo Fisher</td><td>10010049</td></tr><tr><td>DPBS without ions</td><td>Thermo Fisher</td><td>14190144</td></tr><tr><td>Diff-Quik stain</td><td>Labor+Technik</td><td>LT 001</td></tr><tr><td>8-well chambered coverslip for live imaging</td><td>Ibidi</td><td>80826</td></tr><tr><td>CellTrace CFSE Cell proliferation kit</td><td>Thermo Fisher Scientific</td><td>C34554</td></tr><tr class="rowsep"><td>Molecular Probes DAPI (4',6 Diamidino 2 Phenylindole, Dihydrochloride)</td><td>Thermo Fisher Scientific</td><td>D1306</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: cell lines</b></td></tr><tr><td>H1 hESCs</td><td>WiCell</td><td>WiCell, WA01</td></tr><tr><td>H7 hESCs</td><td>WiCell</td><td>WiCell, WA07</td></tr><tr><td>H9 hESCs</td><td>WiCell</td><td>WiCell, WA09</td></tr><tr><td>H1 <i>NR2F2-2A-GFP</i> knock-in reporter hESCs</td><td>This study</td><td>N/A</td></tr><tr><td>H1 <i>SOX17-2A-mPlum</i> knock-in reporter hESCs</td><td>This study</td><td>N/A</td></tr><tr><td>H9 <i>SOX17</i>-<i>mCherry</i> knock-in reporter hESCs</td><td>This study</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref65-3" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-3" aria-controls="sref65-3">Loh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>SUN004.1.9 <i>CAG-GFP</i> hiPSCs</td><td>This study</td><td>N/A</td></tr><tr><td>SUN004.2 <i>CAG-tdTomato</i> hiPSCs</td><td>This study</td><td>N/A</td></tr><tr><td>SUN003.1 <i>CAG-AkaLuciferase-tdTomato</i> hiPSCs</td><td>This study</td><td>N/A</td></tr><tr><td>Human umbilical artery endothelial cells (HUAEC)</td><td>Sigma-Aldrich</td><td>202-05N</td></tr><tr><td>Human umbilical vein endothelial cells (HUVEC)</td><td>Sigma-Aldrich</td><td>200-05N</td></tr><tr><td>Normal human lung fibroblast cells (NHLF)</td><td>Lonza</td><td>CC-2512</td></tr><tr><td>THP-1 cells</td><td>ATCC</td><td>TIB-202</td></tr><tr><td>HEK 293T/17 cells</td><td>ATCC</td><td>CRL-11268</td></tr><tr class="rowsep"><td>Vero C1008 (Vero 76, clone E6, Vero E6)</td><td>ECACC</td><td>85020206</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental models: organisms/strains</b></td></tr><tr class="rowsep"><td><i>Mus musculus</i> (mouse): NOD-SCID <i>Il2rg</i><sup>-/-</sup> strain</td><td>The Jackson Laboratory</td><td>005557</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>Primers</td><td>See <a id="crosref1790" href="#mmc5">Table&nbsp;S5</a></td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>pCAGGS-coNiV-G-HA</td><td>Shreyas Kowdle and Benhur Lee, Icahn School of Medicine at Mount Sinai</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref90-3" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-3" aria-controls="sref90-3">Palomares et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref90-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pcDNA3.1+coNiV-F T5F-AU1</td><td>Shreyas Kowdle and Benhur Lee, Icahn School of Medicine at Mount Sinai</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref90-4" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-4" aria-controls="sref90-4">Palomares et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref90-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pCAGGS-coHeV-G-HA</td><td>Shreyas Kowdle and Benhur Lee, Icahn School of Medicine at Mount Sinai</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref80-2" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-2" aria-controls="sref80-2">Negrete et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref80-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Negrete, O.A. ∙ Chu, D. ∙ Aguilar, H.C. ...</div><div><strong>Single amino acid changes in the Nipah and Hendra virus attachment glycoproteins distinguish ephrinB2 from ephrinB3 usage</strong></div><div><em>J.&nbsp;Virol.</em> 2007; <strong>81</strong>:10804-10814</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.00999-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34648830167" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.00999-07&amp;pmid=17652392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pCAGGS-coHeV-F-AU1</td><td>Shreyas Kowdle and Benhur Lee, Icahn School of Medicine at Mount Sinai</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref80-3" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80-3" aria-controls="sref80-3">Negrete et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref80-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Negrete, O.A. ∙ Chu, D. ∙ Aguilar, H.C. ...</div><div><strong>Single amino acid changes in the Nipah and Hendra virus attachment glycoproteins distinguish ephrinB2 from ephrinB3 usage</strong></div><div><em>J.&nbsp;Virol.</em> 2007; <strong>81</strong>:10804-10814</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.00999-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34648830167" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.00999-07&amp;pmid=17652392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pVSV-G</td><td>Owen Witte, University of California Los Angeles</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref41-1" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-1" aria-controls="sref41-1">Goldstein et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref41-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Goldstein, A.S. ∙ Drake, J.M. ∙ Burnes, D.L. ...</div><div><strong>Purification and direct transformation of epithelial progenitor cells from primary human prostate</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:656-667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2011.317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79955530295" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21527922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2011.317&amp;pmid=21527922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pMDL</td><td>Owen Witte, University of California Los Angeles</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref41-2" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-2" aria-controls="sref41-2">Goldstein et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref41-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Goldstein, A.S. ∙ Drake, J.M. ∙ Burnes, D.L. ...</div><div><strong>Purification and direct transformation of epithelial progenitor cells from primary human prostate</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:656-667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2011.317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79955530295" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21527922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2011.317&amp;pmid=21527922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pREV</td><td>Owen Witte, University of California Los Angeles</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref41-3" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-3" aria-controls="sref41-3">Goldstein et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref41-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Goldstein, A.S. ∙ Drake, J.M. ∙ Burnes, D.L. ...</div><div><strong>Purification and direct transformation of epithelial progenitor cells from primary human prostate</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:656-667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2011.317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79955530295" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21527922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2011.317&amp;pmid=21527922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td>pRS348 <i>EF1A-GFP; mU6-CD34 sgRNA</i></td><td>Reuben Saunders and Jonathan Weissman, Whitehead Institute/MIT</td><td>N/A</td></tr><tr><td>PiggyBac <i>CAG-GFP</i> donor plasmid</td><td>Austin Smith, University of Cambridge</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref113-1" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-1" aria-controls="sref113-1">Takashima et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref113-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Takashima, Y. ∙ Guo, G. ∙ Loos, R. ...</div><div><strong>Resetting transcription factor control circuitry toward ground-state pluripotency in human</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:1254-1269</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2014.08.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2014.08.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84908074395" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25215486/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.08.029&amp;pmid=25215486" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td><i>CMV-PBase</i> plasmid</td><td>Austin Smith, University of Cambridge</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref113-2" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-2" aria-controls="sref113-2">Takashima et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref113-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Takashima, Y. ∙ Guo, G. ∙ Loos, R. ...</div><div><strong>Resetting transcription factor control circuitry toward ground-state pluripotency in human</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:1254-1269</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2014.08.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2014.08.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84908074395" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25215486/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.08.029&amp;pmid=25215486" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td></tr><tr><td><i>pCS-CAG-mScarlet</i> lentiviral plasmid</td><td>This study</td><td>N/A</td></tr><tr class="rowsep"><td><i>pCS-CAG-AkaLuciferase-tdTomato</i> lentiviral plasmid</td><td>This study</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td>ImageJ/Fiji</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Schindelin et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (28202)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0025" href="https://imagej.net/software/fiji/" target="_blank">https://imagej.net/software/fiji/</a></td></tr><tr><td>Seurat v3</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref111-1" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-1" aria-controls="sref111-1">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0030" href="https://satijalab.org/seurat/" target="_blank">https://satijalab.org/seurat/</a></td></tr><tr><td>Ggplot2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref119" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119" aria-controls="sref119">Wickham, 2016</a><div class="dropBlock__holder" data-db-target-of="sref119"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>Wickham, H.</div><div><strong>ggplot2: Elegant Graphics for Data Analysis</strong></div><div>Springer-Verlag New York, 2016</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-319-24277-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-319-24277-4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0035" href="https://ggplot2.tidyverse.org/" target="_blank">https://ggplot2.tidyverse.org/</a></td></tr><tr><td>Tidyverse</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref120-1" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-1" aria-controls="sref120-1">Wickham et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref120-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the Tidyverse</strong></div><div><em>J.&nbsp;Open Source Software.</em> 2019; <strong>4</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.21105/joss.01686" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.21105%2Fjoss.01686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0040" href="https://www.tidyverse.org/" target="_blank">https://www.tidyverse.org/</a></td></tr><tr><td>Dplyr</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref121" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121" aria-controls="sref121">Wickham et&nbsp;al., 2022</a><div class="dropBlock__holder" data-db-target-of="sref121"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Wickham, H. ∙ François, R. ∙ Henry, L. ...</div><div><strong>Dplyr: a grammar of data manipulation</strong></div><div><a id="intref0150" href="https://dplyr.tidyverse.org" target="_blank">https://dplyr.tidyverse.org</a><div>Date: 2022</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamR.Fran%C3%A7oisL.HenryK.M%C3%BCllerDplyr%3A+a+grammar+of+data+manipulationhttps%3A%2F%2Fdplyr.tidyverse.org2022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0045" href="https://dplyr.tidyverse.org/" target="_blank">https://dplyr.tidyverse.org/</a></td></tr><tr><td>Limma</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref99-1" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-1" aria-controls="sref99-1">Ritchie et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref99-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Ritchie, M.E. ∙ Phipson, B. ∙ Wu, D. ...</div><div><strong>limma powers differential expression analyses for RNA-sequencing and microarray studies</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:e47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84926507971" target="_blank" aria-label="Open Scopus in new tab">Scopus (13963)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv007&amp;pmid=25605792" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0050" href="https://bioconductor.org/packages/release/bioc/html/limma.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/limma.html</a></td></tr><tr><td>Voom</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Law et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Law, C.W. ∙ Chen, Y. ∙ Shi, W. ...</div><div><strong>voom: precision weights unlock linear model analysis tools for RNA-seq read counts</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:R29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2014-15-2-r29" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84896735766" target="_blank" aria-label="Open Scopus in new tab">Scopus (2673)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24485249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2014-15-2-r29&amp;pmid=24485249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0055" href="https://rdrr.io/bioc/limma/man/voom.html" target="_blank">https://rdrr.io/bioc/limma/man/voom.html</a></td></tr><tr><td>EdgeR</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0060" href="http://bioconductor.org/packages/release/bioc/html/edgeR.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/edgeR.html</a></td></tr><tr><td>RStudio</td><td>RStudio Team</td><td><a id="intref0065" href="https://www.rstudio.com/" target="_blank">https://www.rstudio.com/</a></td></tr><tr><td>Icy</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref26-1" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-1" aria-controls="sref26-1">de Chaumont et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref26-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>de Chaumont, F. ∙ Dallongeville, S. ∙ Chenouard, N. ...</div><div><strong>Icy: an open BioImage informatics platform for extended reproducible research</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:690-696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2075" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84863191618" target="_blank" aria-label="Open Scopus in new tab">Scopus (828)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2075&amp;pmid=22743774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0070" href="http://icy.bioimageanalysis.org/" target="_blank">http://icy.bioimageanalysis.org/</a></td></tr><tr><td>Kallisto</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3519" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84966283954" target="_blank" aria-label="Open Scopus in new tab">Scopus (3483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3519&amp;pmid=27043002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0075" href="https://pachterlab.github.io/kallisto/" target="_blank">https://pachterlab.github.io/kallisto/</a></td></tr><tr><td>Cell Ranger</td><td>10x Genomics</td><td><a id="intref0080" href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger" target="_blank">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger</a></td></tr><tr><td>DESeq2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref68-1" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-1" aria-controls="sref68-1">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref68-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (29293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0085" href="https://github.com/mikelove/DESeq2" target="_blank">https://github.com/mikelove/DESeq2</a></td></tr><tr><td>SCTransform</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref43-1" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-1" aria-controls="sref43-1">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (742)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0090" href="https://github.com/satijalab/sctransform" target="_blank">https://github.com/satijalab/sctransform</a></td></tr><tr><td>Enrichr</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">Chen et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chen, E.Y. ∙ Tan, C.M. ∙ Kou, Y. ...</div><div><strong>Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84876100615" target="_blank" aria-label="Open Scopus in new tab">Scopus (2789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23586463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-128&amp;pmid=23586463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Kuleshov et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Kuleshov, M.V. ∙ Jones, M.R. ∙ Rouillard, A.D. ...</div><div><strong>Enrichr: a comprehensive gene set enrichment analysis web server 2016 update</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W90-W97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw377" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84987663170" target="_blank" aria-label="Open Scopus in new tab">Scopus (3602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27141961/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw377&amp;pmid=27141961" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0095" href="https://maayanlab.cloud/Enrichr/" target="_blank">https://maayanlab.cloud/Enrichr/</a></td></tr><tr><td>FastQC</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Andrews, S.</div><div><strong>FastQC: A quality control tool for high throughput sequence data</strong></div><div><a id="intref0140" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a><div>Date: 2010</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+A+quality+control+tool+for+high+throughput+sequence+datahttp%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F2010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0100" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></td></tr><tr><td>Trim Galore</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Krueger, 2012</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Krueger, F.</div><div><strong>Trim galore</strong></div><div><a id="intref0145" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a><div>Date: 2012</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+galorehttps%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F2012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0105" href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a></td></tr><tr><td>Trackmate v6.0.2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref114-1" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-1" aria-controls="sref114-1">Tinevez et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref114-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Tinevez, J.Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: an open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2016.09.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85002002648" target="_blank" aria-label="Open Scopus in new tab">Scopus (1289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27713081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2016.09.016&amp;pmid=27713081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0110" href="https://imagej.net/plugins/trackmate/" target="_blank">https://imagej.net/plugins/trackmate/</a></td></tr><tr><td>ShinyCell</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref88-1" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-1" aria-controls="sref88-1">Ouyang et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref88-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Ouyang, J.F. ∙ Kamaraj, U.S. ∙ Cao, E.Y. ...</div><div><strong>ShinyCell: simple and sharable visualisation of single-cell gene expression data</strong></div><div><em>Bioinformatics.</em> 2021; btab209</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btab209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85134939084" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33774659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtab209&amp;pmid=33774659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0115" href="https://github.com/SGDDNB/ShinyCell" target="_blank">https://github.com/SGDDNB/ShinyCell</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2822%2900655-9" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource availability</h3><section id="sec-4-2-1"><h4>Lead contact</h4><div id="p0245" role="paragraph">Requests for further information should be directed to and will be fulfilled by the lead contact, Kyle M. Loh (<a id="intref0120" href="mailto:kyleloh@stanford.edu">kyleloh@stanford.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials availability</h4><div id="p0250" role="paragraph">Multiple human pluripotent stem cell lines were developed as part of this study: H1 <i>NR2F2-2A-GFP</i> hESCs, H1 <i>SOX17-2A-mPlum</i> hESCs, H1 <i>EFNB2</i><sup><i>-/-</i></sup> hESCs, SUN004.1.9 <i>CAG-eGFP</i> hiPSCs, SUN004.2 <i>CAG-mScarlet</i> hiPSCs, and SUN003.1 <i>CAG-AkaLuciferase-tdTomato</i> hiPSCs. All cell lines will be made freely available upon request and the completion of applicable Material Transfer Agreements.</div></section></section><section id="sec-4-3"><h3>Experimental model and subject details</h3><section id="sec-4-3-1"><h4>Cell culture</h4><div id="p0255" role="paragraph">All cells in this study were cultured in standard incubator conditions (20% O<sub>2</sub>, 5% CO<sub>2</sub> and 37 °C).</div></section><section id="sec-4-3-2"><h4>Human pluripotent stem cell lines</h4><div id="p0260" role="paragraph">The following human pluripotent stem cell (hPSC) lines were used in this study: wild-type H1, H7 and H9 hESCs (WiCell); H9 <i>SOX17-mCherry</i> knock-in reporter hESCs (<span class="dropBlock reference-citations"><a id="body-ref-sref65-4" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-4" aria-controls="sref65-4">Loh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); H1 <i>NR2F2-2A-GFP</i> knock-in reporter hESCs (generated in this study); H1 <i>SOX17-2A-mPlum</i> knock-in reporter hESCs (generated in this study); SUN004.1.9 hiPSCs carrying a constitutively-expressed <i>CAG-GFP</i> PiggyBac transgene (generated in this study); SUN004.2 hiPSCs carrying a constitutively-expressed <i>CAG-mScarlet</i> lentiviral transgene (generated in this study); and SUN003.1 hiPSCs carrying a constitutively-expressed <i>CAG-AkaLuciferase-tdTomato</i> lentiviral transgene (generated in this study). SUN004.1.9, SUN004.2 and SUN003.1 hiPSCs represent three clonal hiPSC lines originally obtained from two different individuals (SUN003 and SUN004). H1, SUN004.1.9, SUN004.2 and SUN003.1 hPSCs are of a XY genotype, whereas H7 and H9 hPSCs are of a XX genotype.</div><div id="p0265" role="paragraph">Undifferentiated hPSCs were propagated in mTeSR1 medium (StemCell Technologies)&nbsp;+ 1% penicillin/streptomycin (Thermo Fisher) or alternatively, mTeSR Plus medium (StemCell Technologies)&nbsp;+ 1% penicillin/streptomycin in monolayer cultures, on Geltrex basement membrane matrix-coated plates (described below). For mTeSR1, media was changed daily and for mTeSR Plus, media was changed every other day as per the manufacturer’s recommendations. For the sake of brevity, we refer to mTeSR1 and mTeSR Plus interchangeably as “mTeSR” for the remainder of these Methods. In order to maintain cultures of undifferentiated hPSCs, when they became partially confluent, undifferentiated hPSCs were passaged by treating them for 7&nbsp;minutes with EDTA (Versene, Thermo Fisher) at room temperature. Subsequently, EDTA was removed, mTeSR was added, and then hPSCs were manually scraped off the plate to generate clumps. hPSC clumps were then seeded onto new plates that had been precoated with Geltrex basement membrane matrix (described below) in mTeSR medium&nbsp;+ 1% penicillin/streptomycin. To reiterate, during EDTA-based maintenance passaging of undifferentiated hPSCs as clumps, ROCK inhibitor was not added.</div></section><section id="sec-4-3-3"><h4>Human pluripotent stem cell-derived endothelial cells</h4><div id="p0270" role="paragraph">After differentiation of hPSCs into artery cells (within 3&nbsp;days, as described below) or vein cells (within 4&nbsp;days, as described below), the resultant hPSC-derived artery and vein cells could be expanded for up to 6 additional days. Day 3 hPSC-derived artery cells were expanded in EGM2 (Endothelial Cell Growth Medium 2, Lonza CC-3162), which was refreshed every 24 hours. Day 4 hPSC-derived vein cells were expanded in EGM2&nbsp;+ SB505124 (2&nbsp;μM)&nbsp;+ RO4929097 (2&nbsp;μM), which was refreshed every 24 hours. Expansion of hPSC-derived artery and vein cells was performed in monolayer cultures on Geltrex-coated cell culture plates. Expanded hPSC-derived artery and vein cells were used for certain functional assays as well as viral infection experiments. The biological sex of hPSC-derived artery and vein cells corresponds to the biological sex of the parental hPSC line used to generate them.</div></section><section id="sec-4-3-4"><h4>Human umbilical artery and vein endothelial cells</h4><div id="p0275" role="paragraph">Human umbilical artery endothelial cells (HUAEC, Sigma-Aldrich, 202-05N) and human umbilical vein endothelial cells (HUVEC, Sigma-Aldrich, 200-05N) were both propagated on Geltrex-coated plates in EGM2 (Lonza CC-3162) media, which was refreshed every 1-2&nbsp;days. The biological sex of these primary human endothelial cells was not determined.</div></section><section id="sec-4-3-5"><h4>Human lung fibroblast cells</h4><div id="p0280" role="paragraph">Primary human adult lung fibroblasts (normal human lung fibroblasts [NHLF], Lonza, CC-2512) were propagated in Fibroblast Growth Medium-2 (Lonza, CC-3132), with media changes every 2-3&nbsp;days. Once confluent, fibroblasts were washed twice with DPBS without ions (Thermo Fisher, 14190144) and exposed to 0.05% Trypsin-EDTA (Thermo Fisher, 25300062) for 2&nbsp;min. Trypsin was neutralized using fresh FGM2, and the cell solution was centrifuged at 300g in order to pellet cells for additional culture or experiments. Human fibroblasts were cocultured with endothelial cells to generate 3-dimensional vascular networks (described below). For all relevant experiments, fibroblasts between passages 3-7 were used. Where indicated, NHLFs obtained from different donors were used in experiments. The biological sex of these primary human lung fibroblasts was not determined.</div></section><section id="sec-4-3-6"><h4>THP1 cells</h4><div id="p0285" role="paragraph">THP1 cells (ATCC, TIB-202) were propagated in RPMI-1640&nbsp;+ 10% fetal bovine serum (FBS) in suspension cultures, and were passaged every 2-3&nbsp;days by adding fresh media and diluting their concentration to 3x10<sup>5</sup> cells/mL. THP1 cells were used to assess the ability of immune cells to adhere to endothelial cells (described below). THP1 cells represent a transformed human monocyte-like cell line of a XY genotype.</div></section><section id="sec-4-3-7"><h4>HEK293T/17 cells</h4><div id="p0290" role="paragraph">HEK293T/17 cells (ATCC, CRL-11268) were propagated in DMEM&nbsp;+ 10% fetal bovine serum (FBS) and were passaged every 3-4&nbsp;days at a 1:10 ratio through trypsinization. HEK293T/17 cells were used to package pseudotyped lentivirus particles (described below). HEK293T/17 cells represent a transformed human cell line that is an easily-transfected derivative of the parental HEK293T line, and are of a XX genotype.</div></section><section id="sec-4-3-8"><h4>Vero E6 cells</h4><div id="p0295" role="paragraph">Vero C1008 (Vero 76, clone E6, Vero E6) cells (European Collection of Authenticated Cell Cultures (ECACC), 85020206) were propagated in DMEM containing 2% FBS, 2&nbsp;mM glutamine, and 1% penicillin/streptomycin and passaged at a 1:10 ratio through trypsinization. Vero E6 cells were used to propagate Nipah and Hendra viruses in order to generate viral stocks and to quantitatively titrate infectious virus from the supernatant of virally-infected cell cultures (described below). Vero E6 cells represent a transformed African green monkey (<i>Chlorocebus sabaeus</i>) cell line of a XX genotype.</div></section><section id="sec-4-3-9"><h4>Mouse models</h4><div id="p0300" role="paragraph">Immunodeficient NOD-SCID <i>Il2rg</i><sup>-/-</sup> (NSG) mice, obtained from The Jackson Laboratory, were housed in the Lokey Stem Cell Research Building’s Barrier Mouse Facility. Adult NSG mice were transplanted with hPSC-derived artery and vein ECs (described below). At the time of transplantation, NSG mice were 2-4&nbsp;months old and both male and female mice were used.</div></section><section id="sec-4-3-10"><h4>Nipah virus, Malaysia variant</h4><div id="p0305" role="paragraph">Nipah virus, Malaysia variant (Nipah virus/human/MY/1999/CDC; genomic sequence reported in NCBI accession number GenBank: NC_002728) was originally isolated by the CDC from the cerebrum of an infected human in Malaysia in 1999, and was passaged on Vero E6 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref45-2" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-2" aria-controls="sref45-2">Harcourt et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref45-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Harcourt, B.H. ∙ Tamin, A. ∙ Ksiazek, T.G. ...</div><div><strong>Molecular characterization of Nipah virus, a newly emergent paramyxovirus</strong></div><div><em>Virology.</em> 2000; <strong>271</strong>:334-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/viro.2000.0340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0034608231" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10860887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fviro.2000.0340&amp;pmid=10860887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We previously demonstrated the ability of Nipah virus, Malaysia variant to cause fatal disease in African green monkeys (<span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Prescott et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Prescott, J. ∙ DeBuysscher, B.L. ∙ Feldmann, F. ...</div><div><strong>Single-dose live-attenuated vesicular stomatitis virus-based vaccine protects African green monkeys from Nipah virus disease</strong></div><div><em>Vaccine.</em> 2015; <strong>33</strong>:2823-2829</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.vaccine.2015.03.089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84929297020" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25865472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.vaccine.2015.03.089&amp;pmid=25865472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Syrian hamsters (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">DeBuysscher et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>DeBuysscher, B.L. ∙ de Wit, E. ∙ Munster, V.J. ...</div><div><strong>Comparison of the pathogenicity of Nipah virus isolates from Bangladesh and Malaysia in the Syrian hamster</strong></div><div><em>PLoS Negl. Trop. Dis.</em> 2013; <strong>7</strong>:e2024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pntd.0002024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873501371" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23342177/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pntd.0002024&amp;pmid=23342177" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), thus verifying its pathogenicity. Nipah virus is a member of the species <i>Nipah henipavirus</i>; genus <i>Henipavirus</i>; family <i>Paramyxoviridae</i>; order <i>Mononegavirales</i>.</div></section><section id="sec-4-3-11"><h4>Hendra virus, Brisbane variant</h4><div id="p0310" role="paragraph">Hendra virus, Brisbane variant (Hendra virus/horse/AU/1994/Hendra; genomic sequence reported in NCBI accession number GenBank: NC_001906) was originally isolated from an infected horse in Hendra, a suburb of Brisbane, Australia, in 1994 (<span class="dropBlock reference-citations"><a id="body-ref-sref78-6" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-6" aria-controls="sref78-6">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Hendra virus is a member of the species <i>Hendra henipavirus</i>; genus <i>Henipavirus</i>; family <i>Paramyxoviridae</i>; order <i>Mononegavirales</i>.</div></section><section id="sec-4-3-12"><h4>Sendai virus, Cantell strain</h4><div id="p0315" role="paragraph">The original provenance of Sendai virus, Cantell strain is unknown, but it was passaged over 100 times in chicken embryonated eggs at the Central Public Health Laboratory in Helsinki, Finland (<span class="dropBlock reference-citations"><a id="body-ref-sref12-1" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-1" aria-controls="sref12-1">Cantell and Hirvonen, 1981</a><div class="dropBlock__holder" data-db-target-of="sref12-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cantell, K. ∙ Hirvonen, S.</div><div><strong>Preparation and assay of Sendai virus</strong></div><div><em>Methods Enzymol.</em> 1981; <strong>78</strong>:299-301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0076-6879(81)78131-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6173605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0076-6879%2881%2978131-x&amp;pmid=6173605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sendai virus, Cantell strain is known to induce interferon-β signaling <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i>: it does not cause severe disease in mice, nor is it presently known to cause human disease (<span class="dropBlock reference-citations"><a id="body-ref-sref67-3" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-3" aria-controls="sref67-3">López et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref67-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>López, C.B. ∙ Yount, J.S. ∙ Hermesh, T. ...</div><div><strong>Sendai virus infection induces efficient adaptive immunity independently of type I interferons</strong></div><div><em>J.&nbsp;Virol.</em> 2006; <strong>80</strong>:4538-4545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.80.9.4538-4545.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646184859" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16611914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.80.9.4538-4545.2006&amp;pmid=16611914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref124-1" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-1" aria-controls="sref124-1">Yoshida et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref124-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Yoshida, A. ∙ Kawabata, R. ∙ Honda, T. ...</div><div><strong>A single amino acid substitution within the paramyxovirus Sendai virus nucleoprotein is a critical determinant for production of interferon-beta-inducing copyback-type defective interfering genomes</strong></div><div><em>J.&nbsp;Virol.</em> 2018; <strong>92</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02094-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85042056136" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29237838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02094-17&amp;pmid=29237838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sendai virus is a member of the species <i>Murine respirovirus</i>; genus <i>Respirovirus</i>; family <i>Paramyxoviridae</i>; order <i>Mononegavirales</i>. Sendai virus, Cantell strain was obtained from ATCC (VR-907).</div></section></section><section id="sec-4-4"><h3>Method details</h3><section id="sec-4-4-1"><h4>Data visualization</h4><div id="p0320" role="paragraph">Standard plots were prepared with Microsoft Excel, Microsoft PowerPoint or GraphPad Prism. Heatmaps were constructed in Heatmapper (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Babicki et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Babicki, S. ∙ Arndt, D. ∙ Marcu, A. ...</div><div><strong>Heatmapper: web-enabled heat mapping for all</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W147-W153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84994283822" target="_blank" aria-label="Open Scopus in new tab">Scopus (830)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27190236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw419&amp;pmid=27190236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or, alternatively, the HeatMapViewer function of the online GenePattern platform (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Reich et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Reich, M. ∙ Liefeld, T. ∙ Gould, J. ...</div><div><strong>GenePattern 2.0</strong></div><div><em>Nat. Genet.</em> 2006; <strong>38</strong>:500-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0506-500" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646361583" target="_blank" aria-label="Open Scopus in new tab">Scopus (1493)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16642009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0506-500&amp;pmid=16642009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Flow cytometry data were visualized with FlowJo. Other types of data visualization were prepared as described in the respective sections below.</div></section><section id="sec-4-4-2"><h4>Basement membrane matrices</h4><div id="p0325" role="paragraph">hPSCs were maintained and differentiated on cell culture plates that been pre-coated with either Geltrex or vitronectin basement membrane matrices. Geltrex (Thermo Fisher) was diluted 1:100-1:200 in DMEM/F12 (Thermo Fisher) and was used to coat tissue culture plastics for at least 1 hour at 37 °C. Recombinant human truncated vitronectin (Gibco, A14700; “VTN-N”) was diluted to a 10&nbsp;μg/mL stock in PBS (lacking Ca<sup>2+</sup> or Mg<sup>2+</sup>) and was used to coat tissue culture plastics for at least 1 hour at 37 °C. After coating with either Geltrex or vitronectin, the basement membrane solution was aspirated, leaving behind a thin film; subsequently, cells were plated on the Geltrex- or vitronectin-coated cell culture plastics.</div></section><section id="sec-4-4-3"><h4>Preparing CDM2 basal media</h4><div id="p0330" role="paragraph">hPSCs were differentiated into artery and vein cells as described below in Chemically Defined Medium 2 (CDM2). The composition of CDM2 has been described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref65-5" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-5" aria-controls="sref65-5">Loh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref66-5" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" aria-label="Reference 2016" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-5" aria-controls="sref66-5">2016</a><div class="dropBlock__holder" data-db-target-of="sref66-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): 50% IMDM&nbsp;+ GlutaMAX (Thermo Fisher, 31980-097)&nbsp;+ 50% F12&nbsp;+ GlutaMAX (Thermo Fisher, 31765-092)&nbsp;+ 1&nbsp;mg/mL polyvinyl alcohol (Sigma, P8136-250G)&nbsp;+ 1% v/v chemically defined lipid concentrate (Thermo Fisher, 11905-031)&nbsp;+ 450&nbsp;μM 1-thioglycerol (Sigma, M6145-100ML)&nbsp;+ 0.7&nbsp;μg/mL recombinant human insulin (Sigma, 11376497001)&nbsp;+ 15&nbsp;μg/mL human transferrin (Sigma, 10652202001)&nbsp;+ 1% v/v penicillin/streptomycin (Thermo Fisher, 15070-063). Polyvinyl alcohol was brought into suspension by gentle warming and magnetic stirring, and the media was sterilely filtered (through a 0.22&nbsp;μm filter) prior to use.</div></section><section id="sec-4-4-4"><h4>hPSC differentiation into artery and vein cells</h4><div id="p0335" role="paragraph"><div id="ulist0015" role="list"><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0340" role="paragraph"><b>Seeding hPSCs for differentiation</b> (<b>Step 0</b>). In contrast to passaging hPSCs for maintenance, a different passaging procedure was used to plate hPSCs for differentiation; notably, sparse seeding of hPSCs as single cells is paramount for efficient differentiation. To seed hPSCs for differentiation, undifferentiated hPSCs were dissociated into single cells (Accutase, Thermo Fisher) and plated into recipient wells in mTeSR supplemented with thiazovivin (1&nbsp;μM, Tocris; a ROCK inhibitor, to enhance hPSC survival after passaging) onto plates precoated with either Geltrex basement membrane matrix or recombinant Vitronectin (described above), thus plating 25,000-50,000 hPSCs/cm<sup>2</sup> (i.e., ∼95,000-190,000 hPSCs/well of a 12-well plate). Freshly-seeded hPSCs were allowed to adhere and recover for 24 hours in mTeSR&nbsp;+ 1&nbsp;μM thiazovivin prior to initiating differentiation, during which the hPSCs re-formed small clumps. All subsequent differentiation steps were performed in defined, serum-free CDM2 basal media (<span class="dropBlock reference-citations"><a id="body-ref-sref65-6" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-6" aria-controls="sref65-6">Loh et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref65-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref66-6" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" aria-label="Reference 2016" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-6" aria-controls="sref66-6">2016</a><div class="dropBlock__holder" data-db-target-of="sref66-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To reiterate, hPSCs are maintained by passaging as clumps (to maintain normal karyotype) but are seeded for differentiation as single cells (to enable efficient differentiation).</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0345" role="paragraph"><b>Day 1</b> (<b>mid primitive streak induction, 24 hours</b> [<b>Step 1</b>]). Day 0 hPSCs were briefly washed (DMEM/F12, Thermo Fisher) to remove all traces of mTeSR&nbsp;+ thiazovivin. Then, they were differentiated towards mid primitive streak in CDM2 media supplemented with Activin A (30&nbsp;ng/mL, R&amp;D Systems), BMP4 (40&nbsp;ng/mL, R&amp;D Systems), CHIR99021 (6&nbsp;μM, Tocris), FGF2 (20&nbsp;ng/mL, Thermo Fisher) for 24 hours, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref66-7" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-7" aria-controls="sref66-7">Loh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (with the optional addition of PIK90 (100&nbsp;nM, Calbiochem)).</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0350" role="paragraph"><b>Day 2</b> (<b>lateral mesoderm induction, 24 hours</b> [<b>Step 2</b>]). Day 1 mid primitive streak cells were briefly washed (DMEM/F12) and then differentiated towards lateral mesoderm in CDM2 media supplemented with BMP4 (40&nbsp;ng/mL), GDC-0941 (2.5&nbsp;μM, Cellagen Technology), Forskolin (10&nbsp;μM, Tocris), SB-505124 (2&nbsp;μM, Tocris), VEGF (100&nbsp;ng/mL, R&amp;D Systems), XAV939 (1&nbsp;μM, Tocris) and ascorbic acid-2-phosphate (AA2P; 200&nbsp;μg/mL, Sigma) for 24 hours. Subsequently, lateral mesoderm was used for artery induction (<b>Step 3a</b>; below) or vein induction (<b>Step 3b</b>; below).</div></div></div><div id="u0045" role="listitem"><div class="label">•</div><div class="content"><div id="p0355" role="paragraph"><b>Day 3</b> (<b>artery induction, 24 hours</b> [<b>Step 3a</b>]). Day 2 lateral mesoderm cells were briefly washed (DMEM/F12) and then differentiated towards artery ECs in CDM2 media supplemented with Activin A (15&nbsp;ng/mL), DMH1 (250&nbsp;nM, Tocris), GDC-0941 (2.5&nbsp;μM), VEGF (100&nbsp;ng/mL), XAV939 (1&nbsp;μM) and AA2P (200&nbsp;μg/mL) for 24 hours, thereby yielding day 3 artery ECs.</div></div></div><div id="u0050" role="listitem"><div class="label">•</div><div class="content"><div id="p0360" role="paragraph"><b>Days 3-4</b> (<b>vein induction, 24 hours</b> [<b>Step 3b</b>]). Day 2 lateral mesoderm cells were briefly washed (DMEM/F12) and then differentiated in CDM2 media supplemented with SB505124 (2&nbsp;μM), DMH1 (250&nbsp;nM), RO4929097 (2&nbsp;μM, Cellagen Technology), VEGF (100&nbsp;ng/mL), XAV939 (1&nbsp;μM) and AA2P (200&nbsp;μg/mL) for 24 hours. Subsequently, day 3 cells were briefly washed (DMEM/F12) and then differentiated in CDM2 media supplemented with SB505124 (2&nbsp;μM), RO4929097 (2&nbsp;μM), PD0325901 (500&nbsp;nM, Tocris), CHIR99021 (1&nbsp;μM) and AA2P (200&nbsp;μg/mL) for 24 hours, thereby yielding day 4 vein ECs.</div></div></div></div></div><div id="p0365" role="paragraph">In most experiments, hPSCs were plated and then differentiated into artery and vein ECs using the aforementioned media with respective media changes every 24 hours (without dissociating and re-plating cells at intermediate timepoints). However, we subsequently found that the efficiency of vein EC differentiation was enhanced when, at the end of day 2 of differentiation, lateral mesoderm cells were dissociated and re-plated at high density (∼79,000 cells/cm<sup>2</sup>, i.e., 300,000-400,000 cells/well of a 12-well plate) in day-3 vein induction media (<a id="crosref2015" href="#figs3">Figure&nbsp;S3</a>B). No ROCK inhibitor was used when re-plating lateral mesoderm cells. After 24 hours, day-3 cells were then differentiated into vein ECs using the aforementioned day-4 vein induction media without any additional resplitting.</div></section><section id="sec-4-4-5"><h4>Maintenance of hPSC-derived artery and vein cells</h4><div id="p0370" role="paragraph">After hPSC differentiation into artery cells (within 3&nbsp;days) or vein cells (within 4&nbsp;days) as described above, they could be maintained for at least 6 additional days in&nbsp;vitro on Geltrex-coated cell culture plates. Day 3 hPSC-derived artery cells were expanded in EGM2 (Endothelial Cell Growth Medium 2, Lonza CC-3162), which was refreshed every 24 hours. Day 4 hPSC-derived vein cells were expanded in EGM2&nbsp;+ SB505124 (2&nbsp;μM)&nbsp;+ RO4929097 (2&nbsp;μM), which was refreshed every 24 hours.</div></section><section id="sec-4-4-6"><h4>Cryopreservation and thawing of hPSC-derived artery and vein cells</h4><div id="p0375" role="paragraph">After hPSC differentiation into artery cells (within 3&nbsp;days) or vein cells (within 4&nbsp;days) as described above, they were dissociated using either Accutase (Thermo Fisher) or TrypLE Express (Thermo Fisher). hPSC-derived artery cells were of sufficient purity to be frozen directly, whereas in certain experiments, hPSC-derived vein cells were first magnetically enriched for CD34<sup>+</sup> cells using CD34 microbead-based Magnetically Activated Cell Sorting (MACS, Miltenyi Biotec 130-046-703) prior to freezing. MACS was performed generally as per the manufacturer’s recommendations, with slight modifications: after dissociation, hPSC-derived vein cells were counted using a hemocytometer, and then resuspended in FACS buffer (PBS&nbsp;+ BSA&nbsp;+ EDTA) with a 6- to 7-fold excess of CD34 microbeads and Fc blocking reagent (Miltenyi Biotec 130-046-703) compared to the manufacturer’s recommendation. Vein cells were stained on ice for 30&nbsp;minutes with CD34 microbeads and Fc blocking reagent, washed, and then magnetically bound to a MACS LS column (Miltenyi Biotech 130-042-401). Unbound cells were then magnetically bound to a second MACS LS column to recover any residual CD34<sup>+</sup> cells that were not captured in the first enrichment. Subsequently, CD34<sup>+</sup> vein cells were collected, pelleted, and frozen. Flow cytometry was performed prior to and after magnetic enrichment (both bound and unbound cell fractions) to assess the precision of cell enrichment, which yielded &gt;95% pure CD144<sup>+</sup> cells throughout all MACS experiments. Both hPSC-derived artery and vein cells were frozen in freezing media (90% FBS&nbsp;+ 10% DMSO, which was sterilely filtered before use) in cryopreservation vials, and were transferred to liquid N<sub>2</sub> for long-term storage.</div><div id="p0380" role="paragraph">When desired, hPSC-derived artery and vein cells were rapidly thawed in a 37 °C water bath, washed to remove freezing media, and then thawed onto Geltrex-coated plates. As described above, hPSC-derived artery cells were maintained in EGM2, whereas hPSC-derived vein cells were maintained in EGM2&nbsp;+ SB505124 (2&nbsp;μM)&nbsp;+ RO4929097 (2&nbsp;μM). For the first 24 hours post-thawing, ROCK inhibitor (thiazovivin, 1&nbsp;μM) was added to both artery and vein cells to enhance cell survival, and was subsequently removed.</div></section><section id="sec-4-4-7"><h4>Alternate endothelial differentiation protocols for hPSCs</h4><div id="p0385" role="paragraph">hPSCs were differentiated towards ECs using the artery EC differentiation protocol developed in this study, or alternatively, 4 published methods for EC differentiation (<span class="dropBlock reference-citations"><a id="body-ref-sref61-5" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-5" aria-controls="sref61-5">Lian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref92-6" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-6" aria-controls="sref92-6">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110-7" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-7" aria-controls="sref110-7">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-10" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-10" aria-controls="sref126-10">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) in a side-by-side comparison. As per above, hPSCs were dissociated into single cells (using Accutase) and plated at a density of 25,000-50,000 hPSCs/cm<sup>2</sup> onto Geltrex-coated plates, in mTeSR media supplemented with thiazovivin (1&nbsp;μM, Tocris). Freshly-seeded hPSCs were allowed to adhere and recover for 24 hours in mTeSR&nbsp;+ 1&nbsp;μM thiazovivin prior to initiating differentiation. Immediately before differentiation, cells were briefly washed (DMEM/F12) to remove all traces of mTeSR media, and then the following differentiation media was added:<div id="ulist0020" role="list"><div id="u0055" role="listitem"><div class="label">•</div><div class="content"><div id="p0390" role="paragraph"><span class="dropBlock reference-citations"><a id="body-ref-sref61-6" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-6" aria-controls="sref61-6">Lian et al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. As described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref61-7" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61-7" aria-controls="sref61-7">Lian et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref61-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), hPSCs were treated with CHIR99021 6&nbsp;μM, Tocris) for 2&nbsp;days in LaSR basal medium (Advanced DMEM/F12 [Thermo Fisher]&nbsp;+ GlutaMAX [2.5&nbsp;mM, Gibco]&nbsp;+ AA2P [60&nbsp;μg/ml, Sigma]) for 2&nbsp;days. Then, cells were briefly washed (DMEM/F12) to remove all traces of CHIR99021-containing medium and cells were maintained in LaSR basal medium for 3-4 additional days.</div></div></div><div id="u0060" role="listitem"><div class="label">•</div><div class="content"><div id="p0395" role="paragraph"><span class="dropBlock reference-citations"><a id="body-ref-sref92-7" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-7" aria-controls="sref92-7">Patsch et al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. As described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref92-8" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-8" aria-controls="sref92-8">Patsch et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), hPSCs were treated with BMP4 (25&nbsp;ng/ml, R&amp;D Systems) and CP21R7 (1&nbsp;μM, Selleckchem) in Priming Medium (50% DMEM/F12 with Glutamax media&nbsp;+ 50% Neurobasal media&nbsp;+ N2&nbsp;+ B27 [all from Thermo Fisher]). After 3&nbsp;days, the cells were briefly washed (DMEM/F12) to remove all traces of Priming Medium and were cultured in EC Induction Medium (StemPro-34 SFM medium [Life Technologies]&nbsp;+ VEGF [200&nbsp;ng/ml, R&amp;D Systems]&nbsp;+ Forskolin [2&nbsp;μM, Tocris]). EC Induction Medium was renewed after one day and was maintained for an additional 2-3&nbsp;days.</div></div></div><div id="u0065" role="listitem"><div class="label">•</div><div class="content"><div id="p0400" role="paragraph"><span class="dropBlock reference-citations"><a id="body-ref-sref110-8" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-8" aria-controls="sref110-8">Sriram et al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. As described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref110-9" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110-9" aria-controls="sref110-9">Sriram et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref110-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), hPSCs were treated with CHIR99021 (5&nbsp;μM, Tocris) in STEMdiff APEL basal medium (StemCell Technologies) for 1&nbsp;day. After 24 hours, cells were briefly washed (DMEM/F12) to remove all traces of CHIR99021-containing medium and were treated with FGF2 (50&nbsp;ng/mL, R&amp;D Systems) in STEMdiff APEL basal medium for 1&nbsp;day. Then, cells were briefly washed (DMEM/F12, Thermo Fisher) to remove all traces of FGF2-containing medium and were cultured in STEMdiff APEL basal medium&nbsp;+ BMP4 (25&nbsp;ng/mL, R&amp;D Systems)&nbsp;+ VEGF (50&nbsp;ng/mL, R&amp;D Systems) for 3 additional days.</div></div></div><div id="u0070" role="listitem"><div class="label">•</div><div class="content"><div id="p0405" role="paragraph"><span class="dropBlock reference-citations"><a id="body-ref-sref126-11" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-11" aria-controls="sref126-11">Zhang et al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. As described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref126-12" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-12" aria-controls="sref126-12">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), hPSCs were cultured in E8BAC media (E8 media [Thermo Fisher]&nbsp;+ BMP4 [5&nbsp;ng/mL, R&amp;D Systems]&nbsp;+ Activin A [25&nbsp;ng/mL, R&amp;D Systems]&nbsp;+ CHIR99021 [1&nbsp;μM, Tocris]) for 2&nbsp;days. Subsequently, cells were briefly washed (DMEM/12), and then from day 2 to day 6, cells were differentiated in E5 media&nbsp;+ FGF2 (100&nbsp;ng/ml, R&amp;D Systems)&nbsp;+ VEGF (50&nbsp;ng/ml, R&amp;D Systems)&nbsp;+ SB431542 (10&nbsp;μM, Tocris)&nbsp;+ Resveratrol (10&nbsp;μM, R&amp;D Systems)&nbsp;+ L690 (10&nbsp;μM, R&amp;D Systems).</div></div></div></div></div><div id="p0410" role="paragraph">Flow cytometry to quantify the efficiency of EC differentiation was performed on days 0, 3, 5 and 6 of differentiation.</div></section><section id="sec-4-4-8"><h4>Generation of hiPSC lines</h4><div id="p0415" role="paragraph">Human peripheral blood mononuclear cells (PBMCs) obtained from two different healthy adult human volunteers (SUN003 and SUN004) were reprogrammed using non-integrating, replication-deficient Sendai viruses encoding the iPSC reprogramming factors (<span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Nishimura et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Nishimura, K. ∙ Ohtaka, M. ∙ Takada, H. ...</div><div><strong>Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302</strong></div><div><em>Stem Cell Res.</em> 2017; <strong>23</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2017.06.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021304442" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28666145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2017.06.011&amp;pmid=28666145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This yielded SUN004.1.9 hiPSCs (referring to donor 4, line 1, subclone 9), SUN004.2 hiPSCs (referring to donor 4, line 2) and SUN003.1 hiPSCs (referring to donor 3, line 1) used in this study.</div><div id="p0420" role="paragraph">Ficoll was used to isolate PBMCs from the whole blood of healthy human adult volunteers (SUN003 and SUN004), whose identities were kept anonymous. 2 million PBMCs from each donor were split between two wells of a 24-well plate, with each well containing 1 million cells in 1&nbsp;mL of PBMC media (RPMI&nbsp;+ 10% FBS&nbsp;+ GM-CSF (10&nbsp;ng/mL)&nbsp;+ IL-3 (10&nbsp;ng/mL)&nbsp;+ IL-6 (10&nbsp;ng/mL)&nbsp;+ FLT3L (10&nbsp;ng/mL)&nbsp;+ TPO (10&nbsp;ng/mL)&nbsp;+ SCF (10&nbsp;ng/mL)) for PBMC expansion; cultures were kept in standard incubator conditions (37 °C and 5% CO<sub>2</sub>). After 6&nbsp;days of PBMC expansion, cells were dissociated and combined into a single well of a 24-well plate in fresh PBMC media and expansion cytokines.</div><div id="p0425" role="paragraph">Subsequently, PBMCs were inoculated with replication-deficient, auto-erasing Sendai virus encoding <i>KLF4</i>, <i>OCT4</i> (<i>POU5F1</i>), <i>SOX2</i> and <i>C-MYC</i> (a gift from Dr. Mahito Nakanishi, at the National Institute of Advanced Industrial Science and Technology, Tsukuba, Japan (<span class="dropBlock reference-citations"><a id="body-ref-sref85-3" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-3" aria-controls="sref85-3">Nishimura et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref85-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Nishimura, K. ∙ Ohtaka, M. ∙ Takada, H. ...</div><div><strong>Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302</strong></div><div><em>Stem Cell Res.</em> 2017; <strong>23</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2017.06.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021304442" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28666145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2017.06.011&amp;pmid=28666145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) at multiplicity of infection (MOI)&nbsp;= 2 to initiate iPSC reprogramming. This recombinant Sendai virus has been described previously: it encodes <i>1</i>) modified versions of the <i>NP</i> and <i>P</i>/<i>C/V/W</i> genes of Sendai virus (Nagoya strain), followed by <i>2</i>) <i>KLF4</i>, <i>OCT4</i>, <i>SOX2</i> and <i>C-MYC</i> genes, and finally <i>3</i>) a modified version of the <i>L</i> gene of Sendai virus (C1.151 strain) whose 3’ UTR contains 4x repeats of the <i>miR-302</i> target sequence (<span class="dropBlock reference-citations"><a id="body-ref-sref86-1" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-1" aria-controls="sref86-1">Nishimura et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref86-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Nishimura, K. ∙ Sano, M. ∙ Ohtaka, M. ...</div><div><strong>Development of defective and persistent Sendai virus vector: a unique gene delivery/expression system ideal for cell reprogramming</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2011; <strong>286</strong>:4760-4771</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1074%2Fjbc.M110.183780&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1074%2Fjbc.M110.183780&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79953021243" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21138846/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M110.183780&amp;pmid=21138846" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref85-4" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" aria-label="Reference 2017" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-4" aria-controls="sref85-4">2017</a><div class="dropBlock__holder" data-db-target-of="sref85-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Nishimura, K. ∙ Ohtaka, M. ∙ Takada, H. ...</div><div><strong>Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302</strong></div><div><em>Stem Cell Res.</em> 2017; <strong>23</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2017.06.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021304442" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28666145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2017.06.011&amp;pmid=28666145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This recombinant Sendai virus does not encode the structural genes <i>M</i>, <i>F</i> or <i>HN</i> and thus is replication-deficient: after infecting a cell, it does not produce progeny virions (<span class="dropBlock reference-citations"><a id="body-ref-sref86-2" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86-2" aria-controls="sref86-2">Nishimura et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref86-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Nishimura, K. ∙ Sano, M. ∙ Ohtaka, M. ...</div><div><strong>Development of defective and persistent Sendai virus vector: a unique gene delivery/expression system ideal for cell reprogramming</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2011; <strong>286</strong>:4760-4771</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1074%2Fjbc.M110.183780&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1074%2Fjbc.M110.183780&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79953021243" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21138846/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M110.183780&amp;pmid=21138846" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, after infected cells have become reprogrammed into iPSCs and upregulate <i>miR-302</i>, this recombinant Sendai virus becomes auto-erased, as it contains 4x repeats of the <i>miR-302</i> target sequence (<span class="dropBlock reference-citations"><a id="body-ref-sref85-5" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-5" aria-controls="sref85-5">Nishimura et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref85-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Nishimura, K. ∙ Ohtaka, M. ∙ Takada, H. ...</div><div><strong>Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302</strong></div><div><em>Stem Cell Res.</em> 2017; <strong>23</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2017.06.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021304442" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28666145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2017.06.011&amp;pmid=28666145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0430" role="paragraph">2&nbsp;days after inoculation with Sendai virus, cells were again dissociated, pelleted and resuspended in 12&nbsp;mL of 50% PBMC media and 50% hiPSC media (DMEM/F12, 20% KnockOut Serum Replacement [KOSR], 1% non-essential amino acids [NEAA], 1% penicillin/streptomycin [P/S], 100&nbsp;μM 2-mercaptoethanol, and 5&nbsp;ng/ml FGF2) supplemented with GM-CSF (10&nbsp;ng/mL), IL-3 (10&nbsp;ng/mL), IL-6 (10&nbsp;ng/mL), FLT3L (10&nbsp;ng/mL), TPO (10&nbsp;ng/mL) and SCF (10&nbsp;ng/mL). The cell suspension was split among three wells of a 6-well plate that contained mouse embryonic fibroblasts (MEFs). MEFs had been seeded on the previous day by first coating wells with 0.1% porcine gelatin (which had been pre-coated for 1 hour at 37 °C), after which mitomycin-C-treated, mitotically-inactivated C57BL/6 MEFs were plated at 15,000 cells/cm<sup>2</sup> in MEF media (DMEM with 10% FBS). After plating hiPSCs on MEFs in 50% PBMC media&nbsp;+ 50% hiPSC media, half-media changes were performed for the next 7-10&nbsp;days with hiPSC media, thereby gradually shifting the overall media composition towards hiPSC media.</div><div id="p0435" role="paragraph">Morphologically-conspicuous hiPSC colonies were then manually picked, mechanically separated by pipette trituration and placed in hiPSC media on a new MEF-coated plate with 10&nbsp;μM Y-27632 (ROCK inhibitor, to enhance cell survival) for first 24 hours. After 5&nbsp;days of daily media changes (using hiPSC media), hiPSCs were dissociated into single cells with TrypLE and transitioned onto a vitronectin-coated plate in E8 media supplemented with 10&nbsp;μM Y-27632 for 24 hours. Undifferentiated hiPSCs were routinely passaged every 4-6&nbsp;days by re-seeding them at a density of 5,000-10,000 cells/cm<sup>2</sup>. Subsequently, hiPSCs were adapted to culture on Geltrex-coated plates in mTeSR media. This procedure was used to generate three distinct, clonal hiPSC lines referred to as SUN003.1, SUN004.1 and SUN004.2 (see next section).</div></section><section id="sec-4-4-9"><h4>Fluorescent labeling of hiPSC lines</h4><div id="p0440" role="paragraph">SUN003.1, SUN004.1 and SUN004.2 hiPSC lines were engineered to carry constitutively-expressed reporter transgenes to enable the tracking of differentiated progeny cells.</div><div id="p0445" role="paragraph">SUN003.1 hiPSCs carry a constitutively-expressed <i>CAG-AkaLuciferase-tdTomato</i> lentiviral transgene. The <i>pCS</i>-<i>CAG-AkaLuciferase-tdTomato</i> lentiviral plasmid was generated by cloning <i>AkaLuciferase</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Iwano et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Iwano, S. ∙ Sugiyama, M. ∙ Hama, H. ...</div><div><strong>Single-cell bioluminescence imaging of deep tissue in freely moving animals</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:935-939</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaq1067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85042300288" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29472486/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaq1067&amp;pmid=29472486" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) into the <i>pCS</i>-<i>CAG-tdTomato</i> plasmid (<span class="dropBlock reference-citations"><a id="body-ref-sref72-1" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-1" aria-controls="sref72-1">Masaki et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref72-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Masaki, H. ∙ Kato-Itoh, M. ∙ Takahashi, Y. ...</div><div><strong>Inhibition of apoptosis overcomes stage-related compatibility barriers to chimera formation in mouse embryos</strong></div><div><em>Cell Stem Cell.</em> 2016; <strong>19</strong>:587-592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2016.10.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2016.10.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84995691206" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27814480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.10.013&amp;pmid=27814480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), such that AkaLuciferase and tdTomato are expressed together as an in-frame fusion protein. Lentivirus was then generated using the LV-MAX Lentiviral Production System (Gibco, A35684) as per the manufacturer’s protocol. SUN003.1 hiPSCs were transduced with the <i>CAG-AkaLuciferase-tdTomato</i> lentivirus, cultured for 7-10&nbsp;days, and then tdTomato+ hiPSCs were isolated by FACS, thus yielding <i>CAG-AkaLuciferase-tdTomato</i><sup>+</sup> SUN003.1 hiPSCs (referring to referring to donor 3, line 1).</div><div id="p0450" role="paragraph">SUN004.1.9 hiPSCs carry a constitutively-expressed <i>CAG-eGFP</i> PiggyBac transgene. To integrate the PiggyBac transposon, SUN004.1 hiPSCs were seeded into 1 well of a 6-well plate at a density of 20,000 cells/cm<sup>2</sup>, and the following day, they were then transfected with 250&nbsp;μL of transfection mixture, which contained 1.25&nbsp;μg of donor plasmid (<i>CAG-GFP</i> PiggyBac donor plasmid), 1.25&nbsp;μg of helper plasmid (<i>CMV-PBase</i>, encoding the PiggyBac transposase enzyme) and 7.5&nbsp;μL of Lipofectamine 3000 (L3000001, Thermo Fisher). 5 to 10&nbsp;days post-transposition, GFP<sup>+</sup> hiPSCs that had been successfully transposed were isolated by fluorescence-activated cell sorting (FACS). The <i>CAG-GFP</i><sup>+</sup> SUN004.1.9 hiPSC subclonal line (referring to donor 4, clone 1, subclone 9) was isolated by manually picking and expanding a single GFP<sup>+</sup> colony.</div><div id="p0455" role="paragraph">SUN004.2 hiPSCs carry a constitutively-expressed <i>CAG-mScarlet</i> lentiviral transgene. The <i>pCS-CAG-mScarlet</i> lentiviral plasmid was generated by cloning <i>mScarlet</i> into the <i>pCS-CAG-eGFP</i> plasmid (<span class="dropBlock reference-citations"><a id="body-ref-sref72-2" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72-2" aria-controls="sref72-2">Masaki et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref72-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Masaki, H. ∙ Kato-Itoh, M. ∙ Takahashi, Y. ...</div><div><strong>Inhibition of apoptosis overcomes stage-related compatibility barriers to chimera formation in mouse embryos</strong></div><div><em>Cell Stem Cell.</em> 2016; <strong>19</strong>:587-592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2016.10.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2016.10.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84995691206" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27814480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.10.013&amp;pmid=27814480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Lentivirus production and transduction of SUN004.2 hiPSCs was performed as described above, and then 7-10&nbsp;days post-transduction, mScarlet<sup>+</sup> hiPSCs were isolated by FACS, thus yielding <i>CAG-mScarlet</i><sup>+</sup> SUN004.2 hiPSCs (referring to referring to donor 4, line 2).</div></section><section id="sec-4-4-10"><h4>Flow cytometry</h4><div id="p0460" role="paragraph">Undifferentiated and differentiated hPSCs were dissociated by incubation in TrypLE Express (Gibco) for 5&nbsp;minutes at 37°C. Subsequently, dissociated cells in TrypLE Express were diluted 1:5-1:10 in DMEM/F12 and centrifuged (pelleted) at 500g for 5&nbsp;minutes. Each cell pellet was resuspended in FACS buffer (PBS&nbsp;+ 1&nbsp;mM EDTA [Invitrogen]&nbsp;+ 2% v/v FBS [Atlanta Bio]&nbsp;+ 1% Penicillin/Streptomycin [Gibco]) supplemented with fluorescently-conjugated primary antibodies, and antibody staining occurred for 30&nbsp;minutes on ice protected from light. After staining, cells were washed twice with FACS buffer and resuspended in 200&nbsp;μL FACS buffer with DAPI (1:10,000, Biolegend) for live/dead discrimination. Flow cytometry was performed on a BD Biosciences FACSAria II flow cytometer or a Beckman Coulter CytoFlex analyzer (Stanford Stem Cell Institute FACS Core). For data analysis, cells were gated based on forward and side scatter area parameter, followed by height and width parameters for doublet discrimination. Subsequently, live cells that were negative for DAPI were gated for all marker analyses and calculations of population frequency.</div></section><section id="sec-4-4-11"><h4>Quantitative PCR</h4><div id="p0465" role="paragraph">Undifferentiated or differentiated hPSCs were lysed in 350&nbsp;μL of RLT Plus Buffer and RNA was extracted using the RNeasy Plus Mini Kit (Qiagen) according to the manufacturer’s protocol. 300&nbsp;ng of total RNA was reverse transcribed into cDNA for qPCR using the High-Capacity cDNA Reverse Transcription Kit (Applied Biosystems) according to the manufacturer’s protocol. qPCR was performed in 384-well format as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref66-8" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-8" aria-controls="sref66-8">Loh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using gene-specific forward and reverse primers on a QuantStudio 5 qPCR machine (Thermo Fisher). qPCR primer sequences are provided in <a id="crosref2130" href="#mmc5">Table&nbsp;S5</a>. Expression of all genes was normalized to the levels of the reference gene <i>YWHAZ</i>. This qPCR procedure was only used for biosafety level 2 (BSL2) cell cultures. A separate qPCR procedure was used for biosafety level 4 (BSL4) materials, and is detailed further below.</div></section><section id="sec-4-4-12"><h4>Immunostaining</h4><div id="p0470" role="paragraph">Cultured monolayer cells were fixed, permeabilized and immunostained as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref66-9" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66-9" aria-controls="sref66-9">Loh et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref66-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Imaging was conducted using an FV3000 confocal microscope (Olympus).</div></section><section id="sec-4-4-13"><h4>Genome editing to create knock-in fluorescent reporter hPSCs</h4><div id="p0475" role="paragraph">H1 <i>NR2F2-2A-GFP</i> knock-in hPSCs and H1 <i>SOX17-2A-mPlum</i> knock-in hPSCs were genetically engineered using the Cas9/AAV6 system (<span class="dropBlock reference-citations"><a id="body-ref-sref71-5" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-5" aria-controls="sref71-5">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This genomic knock-in strategy entails the electroporation of hPSCs with a Cas9-sgRNA ribonucleoprotein (RNP) complex (to create a double-stranded break at a desired genomic locus), and delivery of a donor template for homologous recombination, supplied by AAV6 (<span class="dropBlock reference-citations"><a id="body-ref-sref71-6" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-6" aria-controls="sref71-6">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0480" role="paragraph">In brief, hPSCs were treated with 10&nbsp;μM ROCK inhibitor (Y-27632) 24 hours prior to genome editing. hPSCs at 70-80% confluence were then dissociated using Accutase (Life Technologies) followed by neutralization with ROCK inhibitor-supplemented mTeSR1 media. Prior to electroporation, RNP complex was formed by combining 5&nbsp;μg of HiFi SpCas9 (Integrated DNA Technologies) and&nbsp;1.75&nbsp;μg of sgRNA for 10&nbsp;minutes at room temperature, which was then diluted with 20&nbsp;μL of P3 Primary Cell Solution buffer (Lonza). For each electroporation reaction, 500,000 hPSCs were mixed with the Cas9/sgRNA RNP-containing nucleofection solution. Nucleofection was performed using a 16-well Nucleocuvette Strip with the 4D Nucleofector system (Lonza) using the CA137 electroporation code. Following electroporation, cells were transferred into one well of a Matrigel-coated 24-well plate containing 500&nbsp;μL of mTeSR1 media supplemented with 10&nbsp;μM Y-27632. AAV6 donor vector (encoding <i>NR2F2-2A-GFP</i> and <i>SOX17-2A-mPlum</i> donor templates for homologous recombination) was added at 100,000 multiplicity of infection (MOI) directly to hPSCs immediately after plating. Cells were then incubated at 37°C for 24 hours. Media was changed 24 hours post-editing and 10&nbsp;μM Y-27632 was removed 48 hours after.</div><div id="p0485" role="paragraph">The <i>NR2F2</i> and <i>SOX17</i> synthetic sgRNAs were purchased from Synthego with chemically-modified nucleotides at the three terminal positions at both the 5′ and 3′ ends. Modified nucleotides contained 2′-O-methyl 3′-phosphorothioate:<div id="ulist0025" role="list"><div id="u0075" role="listitem"><div class="label">•</div><div class="content"><div id="p0490" role="paragraph"><i>NR2F2</i> sgRNA: AUUUAUUGAAUUGCCAUAUA</div></div></div><div id="u0080" role="listitem"><div class="label">•</div><div class="content"><div id="p0495" role="paragraph"><i>SOX17</i> sgRNA: CUGCAGGCUGGGGCGGAUCA</div></div></div></div></div></section><section id="sec-4-4-14"><h4>Genome editing to create <i>EFNB2</i>-knockout hPSCs</h4><div id="p0500" role="paragraph">The <i>EFNB2</i> gene was deleted in H1 hPSCs using the aforementioned Cas9 protocol (section “Genome editing to create knock-in fluorescent reporter hPSCs”; <span class="dropBlock reference-citations"><a id="body-ref-sref71-7" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-7" aria-controls="sref71-7">Martin et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref71-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with the exception that Cas9 RNPs were used to generate indels in the <i>EFNB2</i> locus, but no templates for homologous recombination were provided. This procedure was repeated twice: the initial round of gene targeting produced EFNB2<sup>+/-</sup> hPSCs, which were then re-electroporated with Cas9 RNP targeting <i>EFNB2</i>, thereby generating EFNB2<sup>-/-</sup> hPSCs.</div><div id="p0505" role="paragraph">sgRNAs targeting the <i>EFNB2</i> locus were designed using CHOPCHOP (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Labun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Labun, K. ∙ Montague, T.G. ∙ Krause, M. ...</div><div><strong>CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:W171-W174</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85069235112" target="_blank" aria-label="Open Scopus in new tab">Scopus (403)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31106371/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz365&amp;pmid=31106371" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and verified using the COSMID program (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Cradick et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Cradick, T.J. ∙ Qiu, P. ∙ Lee, C.M. ...</div><div><strong>Cosmid: a web-based tool for identifying and validating CRISPR/Cas off-target sites</strong></div><div><em>Mol. Ther. Nucleic Acids.</em> 2014; <strong>3</strong>:e214</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1038%2Fmtna.2014.64&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1038%2Fmtna.2014.64&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84921598229" target="_blank" aria-label="Open Scopus in new tab">Scopus (223)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25462530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmtna.2014.64&amp;pmid=25462530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The <i>EFNB2</i> synthetic sgRNA was purchased from Synthego with chemically-modified nucleotides at the three terminal positions at both the 5' and 3' ends. Modified nucleotides contained 2'-O-methyl 3'-phosphorothioate. The genomic sgRNA target sequence was:<div id="ulist0030" role="list"><div id="u0085" role="listitem"><div class="label">•</div><div class="content"><div id="p0510" role="paragraph"><i>EFNB2</i> sgRNA: AGGCUCUAAAACUAUCGAUU</div></div></div></div></div></section><section id="sec-4-4-15"><h4>3D network formation from hPSC-derived endothelial cells</h4><div id="p0515" role="paragraph">Three different types of endothelial cells were used in the formation of <i>in&nbsp;vitro</i> vascular networks in fibrin hydrogels. Where indicated, <i>1</i>) hPSC-derived artery endothelial cells, <i>2</i>) hPSC-derived vein endothelial cells, or <i>3</i>) umbilical cord blood-derived endothelial colony forming cells (CB-ECFC, a positive control (<span class="dropBlock reference-citations"><a id="body-ref-sref56-5" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-5" aria-controls="sref56-5">Kurokawa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)) were utilized.</div><div id="p0520" role="paragraph">hPSC-derived artery and vein cells were frozen and thawed (as described above), and after thawing, were cultured in EGM2 medium for 3&nbsp;days (supplemented with 2&nbsp;μM Thiazovivin for the first 24 hours). EGM2 media was refreshed every 24 hours for the 3-day period. After 3&nbsp;days of expansion, hPSC-derived artery and vein endothelial cells and normal human lung fibroblasts (NHLFs) were used to produce 3D endothelial networks as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref56-6" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-6" aria-controls="sref56-6">Kurokawa et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref56-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, hPSC-derived endothelial cells and normal human lung fibroblasts (NHLFs) were respectively dissociated using 0.05% Trypsin-EDTA (Thermo Fisher, 25300062). The cells were then mixed at a ratio of 1 endothelial cell:2 fibroblasts. The resultant cell mixture was centrifuged at 300g for 5&nbsp;min, and resuspended in 10&nbsp;mg/mL of bovine fibrinogen (Sigma-Aldrich, F8630). 50&nbsp;μL of the cell-fibrinogen solution was then placed inside of a small chamber consisting of cured polydimethylsiloxane (Sylgard 184) on glass, and clotted using thrombin from bovine plasma (Sigma-Aldrich T4648). The final thrombin concentration in the 10&nbsp;mg/mL fibrin hydrogel was 2 unit/mL (which allowed for clotting of the fibrinogen into fibrin). After complete clotting of the hydrogels, the resultant fibrin hydrogels were submerged in 500&nbsp;μL EGM2 and cultured in a 37 °C, 5% CO2 humidified incubator. EGM2 media changes occurred every 1-2&nbsp;days with fresh, warm EGM2. After 7&nbsp;days of culture to allow for network formation in EGM2, hydrogels were fixed using 10% formalin (Sigma F5554) in order to undergo immunofluorescent analysis. Briefly, hydrogels were blocked using 2% BSA (Sigma A2153) in PBS+0.1% Tween-20 (Sigma P9416), after which antibodies were used to stain the tissues overnight at 4 °C. Images were then taken on a fluorescent inverted microscope (Olympus IX83). Resultant image analysis and image preparation was performed using Fiji.</div></section><section id="sec-4-4-16"><h4>hPSC-derived endothelial cell migration <i>in&nbsp;vitro</i> in scratch assay</h4><div id="p0525" role="paragraph">hPSCs were differentiated into artery and vein endothelial cells as described above. Then, hPSC-derived artery cells were cultured in EGM2 for 1&nbsp;day while vein endothelial cells were cultured in EGM2&nbsp;+ SB505124 (2&nbsp;μM)&nbsp;+ RO4929097 (2&nbsp;μM) for 1&nbsp;day, until they formed a confluent monolayer. Then, media was removed and cells were washed with DMEM/F12 before being scratched with a P200 pipette tip affixed to the end of a P200 pipette. After cells were scratched, floating cells were washed away with DMEM/F12. Then, artery expansion media (EGM2) was added to artery endothelial cells and vein expansion media (EGM2&nbsp;+ SB505124 (2&nbsp;μM)&nbsp;+ RO4929097 (2&nbsp;μM)) was added to vein endothelial cells. Media was replaced every 24 hours. Cells were imaged 0, 6, 12, 24 and 48 hours after scratching, using the Leica Paula Microscope. For image acquisition, tape was used on the microscope stage to mark the perimeter of the cell culture plate after scratching such that for images taken at different timepoints, the image was acquired at the same location every time. To quantify the closure of the gap, each image was set to scale using ImageJ. Then, using the ImageJ measurement tool, the length of the gap between both borders of the closing cell monolayer were measured at 6 different locations.</div></section><section id="sec-4-4-17"><h4>Shear stress-induced polarization of hPSC-derived endothelial cells</h4><div id="p0530" role="paragraph">hPSC-derived day-3 artery endothelial cells were cultured in a 6-well plate with EGM2 medium for 24 hours, either in static culture (standard conditions) or a rotator (shear stress conditions). hPSC-derived day-4 vein endothelial cells were magnetically enriched for CD34<sup>+</sup> cells (Miltenyi Biotec) and cultured for 1&nbsp;week in EGM2 media prior to exposure to shear stress for 24 hours as per above. Shear stress conditions were described previously and entailed culturing cells on a 6-well plate on an orbital shaker (<span class="dropBlock reference-citations"><a id="body-ref-sref93" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93" aria-controls="sref93">Poduri et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref93"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Poduri, A. ∙ Chang, A.H. ∙ Raftrey, B. ...</div><div><strong>Endothelial cells respond to the direction of mechanical stimuli through SMAD signaling to regulate coronary artery size</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:3241-3252</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.150904" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85029803848" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28760815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.150904&amp;pmid=28760815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After 24 hours of static or shear stress conditions, both immunostaining and qPCR was performed.</div><div id="p0535" role="paragraph">After exposure to static media or shear stress for 24 hours, immunostaining was performed to quantify the degree of cell alignment. To this end, cells were fixed in 4% paraformaldehyde (PFA) for 15&nbsp;minutes at 4 °C. Then, the cells were washed with PBS for three times, waiting 5&nbsp;minutes between each wash. After washing, cells were incubated in 4 °C overnight with primary antibody in 0.5% Triton X-100/PBS solution. Next, cells were washed with PBS for three times, waiting 15&nbsp;minutes between each wash. Afterwards, the cells were incubated with secondary antibody in 0.5% Triton X-100/PBS solution. Secondary antibody was removed and DAPI in PBS was added and incubated for 5&nbsp;minutes. Then, the cells were washed with 0.5% Triton X-100/PBS three times, waiting 15&nbsp;minutes between each wash. Finally, PBS was added to the cells, and imaging was conducted using a Keyence BZ-9000 microscope.</div><div id="p0540" role="paragraph">After imaging, quantification of cell alignment was performed using ImageJ. Cell directionality analysis was done using the “Directionality” plugin, using Fourier components analysis with Nbins=90, producing a directionality histogram which starts at -90 degrees and ends at&nbsp;+90 degrees.</div></section><section id="sec-4-4-18"><h4>LDL and acetylated LDL uptake assay</h4><div id="p0545" role="paragraph">hPSC-derived day 3 artery and day 4 vein endothelial cells were washed with DMEM/F12 (Thermo Fisher) and incubated at 37°C for 4 hours with either a 1:25 concentration of low-density lipoprotein (LDL) conjugated with DyLight550 (Cayman) or a 5&nbsp;μg/mL concentration of acetylated LDL conjugated to Alexa Fluor 488 (Fisher Scientific) in their respective differentiation media. As a negative control, hPSC-derived day 3 artery and day 4 vein endothelial cells were incubated in their respective differentiation media without LDL DyLight 550 or acetylated LDL Alexa 488. After incubation, cells were dissociated by incubating them in TrypLE Express (Gibco) for 5&nbsp;minutes at 37 °C. Subsequently, the dissociated cells were diluted 1:10 in DMEM/F12 and centrifuged at 500g for 5&nbsp;minutes. Each cell pellet was then resuspended in FACS buffer (DPBS&nbsp;+ 5&nbsp;mM EDTA [Invitrogen]&nbsp;+ 0.5% BSA) supplemented with anti-CD144 antibody conjugated to Alexa Fluor 647 (BD Biosciences) and incubated on ice for 30&nbsp;minutes, while being protected from light. After antibody staining, cells were washed twice with FACS buffer and resuspended in 300&nbsp;μL FACS buffer containing DAPI (1:10,000, Biolegend). Then the resuspended cells were passed through a 100&nbsp;μm strainer (BD Biosciences) and analyzed using a FACSAria II flow cytometer (Stanford Stem Cell Institute FACS Core). During flow cytometry data analysis, we gated on CD144<sup>+</sup> endothelial cells before quantifying fluorescent LDL or acetylated LDL uptake by endothelial cells.</div></section><section id="sec-4-4-19"><h4>Immune cell adhesion to TNFα-treated endothelial cells</h4><div id="p0550" role="paragraph">The THP1 human monocyte-like cell line was fluorescently labelled with CFSE and then cocultured with hPSC-derived endothelial cells to assess immune cell adhesion to endothelial cells. Methods to assess adhesion of immune cells to endothelial cells were modified from previously-published protocols (<span class="dropBlock reference-citations"><a id="body-ref-sref102-4" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102-4" aria-controls="sref102-4">Rosa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref102-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Rosa, S. ∙ Praça, C. ∙ Pitrez, P.R. ...</div><div><strong>Functional characterization of iPSC-derived arterial- and venous-like endothelial cells</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:3826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-40417-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062643820" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30846769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-40417-9&amp;pmid=30846769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126-13" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126-13" aria-controls="sref126-13">Zhang et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref126-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0555" role="paragraph">First, THP1 cells were fluorescently labelled with CFSE, largely as per the manufacturer’s recommendations. THP1 cells were resuspended in DMEM/F12 at a concentration of 1 million cells/mL and then incubated with 5&nbsp;μM of CellTrace CFSE (Thermo Fisher Scientific, C34554) for 20&nbsp;minutes at room temperature, protected from light, and with occasional pipetting to keep cells resuspended during the staining process. After 20&nbsp;minutes of incubation, the cell suspension was diluted 1:5 in RPMI1640&nbsp;+ 10% FBS for 5&nbsp;minutes. Then, cells were centrifuged at 300g for 5&nbsp;minutes, the supernatant was aspirated, and then fresh RPMI1640&nbsp;+ 10% FBS was added.</div><div id="p0560" role="paragraph">Separately, hPSC-derived artery and vein endothelial cells cultured in 12-well plates were stimulated with the inflammatory cytokine TNFα (10&nbsp;ng/mL) for 4 hours in their respective culture media (EGM2 for artery cells and EGM2&nbsp;+ RO4929097 (2μM)&nbsp;+ SB505124 (2μM) for vein cells) at 37 °C. As a negative control, artery and vein cells were cultured in their respective culture media in the absence of TNFα.</div><div id="p0565" role="paragraph">Subsequently, Hoechst dye (1&nbsp;μg/mL) was added in the dark for 5&nbsp;minutes to fluorescently label the nuclei of live artery and vein cells. Then, cells were washed with DMEM/F12. After washing, CFSE-labelled THP-1 immune cells (described above) were added to each well in a 12-well plate at 1 million cells/well in EGM2 media for 30&nbsp;minutes at 37 °C. After this 30-minute coculture of THP1 and endothelial cells, the wells were washed 3 times with DMEM/F12 to remove any non-adhered THP1 cells. Images were taken using a Keyence BZ-9000 fluorescence microscope and quantification of the number of adhered THP1 cells was performed using ImageJ/Fiji.</div></section><section id="sec-4-4-20"><h4>Endothelial sprouting assay</h4><div id="p0570" role="paragraph">The sprouting assay was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Nakatsu et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Nakatsu, M.N. ∙ Davis, J. ∙ Hughes, C.C.W.</div><div><strong>Optimized fibrin gel bead assay for the study of angiogenesis</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2007; <strong>3</strong>:186</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18978935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=18978935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with modifications. First, <i>CAG-GFP</i><sup>+</sup> SUN004.1.9 hPSC-derived artery or vein ECs were thawed and respectively maintained in artery or vein expansion media (EGM2 for artery ECs and EGM2&nbsp;+ RO4929097 (2&nbsp;μM)&nbsp;+ SB505124 (2&nbsp;μM) for vein ECs) for 24 hours until they were fully confluent. Second, ECs were cultured with EGM2&nbsp;+ 10% heat-inactivated FBS for 24 hours, prior to dissociating them for the sprouting assay. Third, hPSC-derived artery and vein ECs were dissociated using trypsin and counted. In parallel, Cytodex-3 beads (Sigma Aldrich, C3275) were allowed to settle for 5&nbsp;minutes, and were washed briefly in 1&nbsp;mL of warm EGM2 media before removing the media. Fourth, Cytodex-3 beads were coated with hPSC-derived ECs by incubating them at a concentration of ∼400 cells/bead in 1.5&nbsp;mL of warm EGM2 media. The bead/cell suspension was incubated for 4 hours at 37°C, with gentle pipetting every 20&nbsp;minutes. After 4 hours, the EC-coated beads were transferred to a well of a non-tissue-culture-treated 24-well plate and left overnight in EGM2&nbsp;+ 10% heat-inactivated FBS. The EC-coated beads were then transferred to a 15&nbsp;mL conical tube and left to settle for 5&nbsp;minutes, followed by three washes with warm EGM2, and then the beads were counted on a coverslip.</div><div id="p0575" role="paragraph">To embed EC-coated beads in a 3D fibrin gel, beads were resuspended in sterilely-filtered fibrinogen solution (2.0&nbsp;mg/mL, Sigma Aldrich, F8630) at a concentration of ∼500 beads/mL, and 0.5&nbsp;mL of the fibrinogen/bead suspension was added to each well of a 24-well plate. Then, 0.5&nbsp;mL of thrombin (0.625 units/mL, Sigma Aldrich, T3399) was added and the bead-fibrinogen-thrombin mixture was triturated ∼4-5 times, with care to avoid the formation of bubbles. The plate was left in the hood for 5&nbsp;minutes, and then incubated at 37° C for 10&nbsp;minutes to generate fibrin gels. After the gels formed, 1&nbsp;mL of EGM2 media&nbsp;+ 10% FBS was added dropwise. Gels were incubated at 37°C for 24 hours in order for GFP<sup>+</sup> endothelial sprouts to emerge. At the end of the assay, EGM2 was removed and the gels were washed with DPBS prior to overnight fixation in 4% paraformaldyde. Gels were imaged using a fluorescence microscope. The number of sprouts per bead and the length of each sprout was quantitated on at least five beads using MetaMorph software.</div></section><section id="sec-4-4-21"><h4>Endothelial chemotaxis towards Apelin-13</h4><div id="p0580" role="paragraph">The chemotaxis assay was performed as previously reported (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Kälin et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Kälin, R.E. ∙ Kretz, M.P. ∙ Meyer, A.M. ...</div><div><strong>Paracrine and autocrine mechanisms of apelin signaling govern embryonic and tumor angiogenesis</strong></div><div><em>Dev. Biol.</em> 2007; <strong>305</strong>:599-614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2007.03.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34247591454" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17412318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2007.03.004&amp;pmid=17412318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with modifications. The polycarbonate membrane of a Transwell filter (with 8&nbsp;μm pores; Corning, 3422) was coated with 0.1% gelatin overnight. Separately, <i>CAG-GFP</i><sup>+</sup> SUN004.1.9 hPSC-derived artery or vein ECs were respectively maintained in artery or vein expansion media (EGM2 for artery ECs and EGM2&nbsp;+ RO4929097 (2&nbsp;μM)&nbsp;+ SB505124 (2&nbsp;μM) for vein ECs) until they were fully confluent. hPSC-derived artery or vein ECs were then dissociated using trypsin and resuspended in Endothelial Basal Media 2 media (EBM2, Lonza) at concentration of 1.5&nbsp;×&nbsp;10<sup>5</sup> cells/mL. Then, 100&nbsp;μL of the cell suspension was added to the upper chamber of the gelatin-coated Transwell filter, and 600&nbsp;μL of EBM2 media&nbsp;+ Apelin-13 (100&nbsp;ng/mL, Tocris, 2420) was added to the lower chamber. Cells were incubated at 37°C for 4 hours to allow for cell migration towards the lower chamber. Then, non-migratory cells were removed with a cotton swab. Migratory cells on the bottom of the Transwell membrane were fixed and stained with DAPI. Cells were imaged under a fluorescence microscope and counted using ImageJ software.</div></section><section id="sec-4-4-22"><h4>Subcutaneous transplantation of hPSC-derived endothelial cells</h4><div id="p0585" role="paragraph">SUN003.1 <i>CAG-AkaLuciferase-tdTomato</i> hPSC-derived ECs were subcutaneously transplanted into adult, immunodeficient NOD-SCID <i>Il2rg</i><sup>-/-</sup> (NSG) mice. In brief, hPSC-derived endothelial cells were suspended in a 1:1 mixture of Matrigel (phenol-free, growth factor reduced)&nbsp;+ F12 medium (supplemented with 100&nbsp;ng/mL FGF2 and 50&nbsp;ng/mL VEGF). NSG mice were then anesthetized with 2% isoflurane and 100% oxygen flowing at 2 liters/minute until they were unconscious and unresponsive to stimuli (i.e., a toe pinch). A fold of skin directly lateral to the spine and near the neck was swabbed with ethanol in order to disinfect it, and then was gently pinched. A volume of 50-200&nbsp;μL containing ∼10<sup>6</sup> cells suspended in the Matrigel&nbsp;+ F12&nbsp;+ FGF2&nbsp;+ VEGF solution was then injected under the skin flap using a 27-gauge syringe. Anesthesia was then discontinued, and mice were allowed to recover in a warm cage until they became alert and awake.</div><div id="p0590" role="paragraph">1&nbsp;month post-transplantation, NSG mice were sacrificed, and subcutaneous gels containing hPSC-derived ECs were recovered and then fixed in 4% formaldehyde overnight at 4 °C. Fixed gels were then placed in PBT buffer (PBS with 0.5% Triton-X) with primary antibodies and incubated overnight at 4 °C on a shaker. The next day, samples were washed every 1.5 hours in PBT buffer for approximately 8 hours at room&nbsp;temperature, and with gentle agitation. Subsequently, Alexa Fluor 488-conjugated secondary antibodies (Life Technologies) were then added in PBT buffer at a 1:250 concentration and incubated overnight at 4 °C on a shaker. Finally, the stained samples were washed every 1.5 hours with PBT buffer for approximately 8 hours and cleared using Vectashield (Vector Labs, H-1000). Samples were then placed in a microscope chamber and imaged using a&nbsp;Zeiss LSM-700 confocal microscope. Image processing was done in ImageJ.</div></section><section id="sec-4-4-23"><h4>Single-cell RNA-sequencing</h4><div id="p0595" role="paragraph">We executed single-cell RNA-seq on H1 hPSC-derived day 3 artery ECs and day 4 vein ECs using the 10x Genomics Chromium v3 platform (<span class="dropBlock reference-citations"><a id="body-ref-sref127-1" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-1" aria-controls="sref127-1">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref127-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Zheng, G.X.Y. ∙ Terry, J.M. ∙ Belgrader, P. ...</div><div><strong>Massively parallel digital transcriptional profiling of single cells</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14049</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85009446777" target="_blank" aria-label="Open Scopus in new tab">Scopus (2096)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14049&amp;pmid=28091601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Libraries were prepared using the 10x Genomics Chromium 3’ Kit v3 following the manufacturer’s guidelines. Libraries were prepared with i7 indices (compatible with Illumina sequencers) and then multiplexed. Multiplexed libraries were sequenced across multiple lanes of an Illumina HiSeq 4000 sequencer by Novogene. Unique sequences in each i7 index were used for demultiplexing. Cell Ranger (10x Genomics) was used to perform read alignment to the hg38 reference genome, filtering, barcode counting and UMI counting.</div><div id="p0600" role="paragraph">Cell matrix files generated from Cell Ranger were imported into RStudio using the function “Read10x_h5” in Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then implemented well-established quality control procedures to exclude dying/dead cells that <i>1</i>) expressed low numbers of genes, <i>2</i>) displayed too low or too high mitochondrial counts, or <i>3</i>) did not express reference/housekeeping genes <i>ACTB</i> or <i>YWHAZ</i>; additionally, we also <i>4</i>) computationally excluded doublets by removing cells that had expressed markedly higher numbers of genes. Only high-quality single-cell transcriptomes that passed these quality control metrics were used for subsequent analyses. scRNAseq data were normalized using the “LogNormalized” function and scaled using a linear transformation in Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-3" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-3" aria-controls="sref111-3">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Expression of marker genes was denoted by different color shades and superimposed over the entire cell population, as visualized on a Uniform Manifold Approximation and Projection (UMAP) projection (<span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (1416)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Importantly, all cells within the entire day 3 and day 4 differentiated populations were shown on the UMAP projection, without any pre-selection of ECs.</div><div id="p0605" role="paragraph">An interactive web portal (<span class="dropBlock reference-citations"><a id="body-ref-sref88-2" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88-2" aria-controls="sref88-2">Ouyang et&nbsp;al., 2021</a><div class="dropBlock__holder" data-db-target-of="sref88-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Ouyang, J.F. ∙ Kamaraj, U.S. ∙ Cao, E.Y. ...</div><div><strong>ShinyCell: simple and sharable visualisation of single-cell gene expression data</strong></div><div><em>Bioinformatics.</em> 2021; btab209</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btab209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85134939084" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33774659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtab209&amp;pmid=33774659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to explore scRNAseq profiles of hPSC-derived artery and vein ECs is available at <a id="intref0125" href="https://anglab.shinyapps.io/arteryveinscrna/" target="_blank">https://anglab.shinyapps.io/arteryveinscrna/</a>.</div></section><section id="sec-4-4-24"><h4>Comparing <i>in&nbsp;vivo</i> versus <i>in&nbsp;vivo</i> artery and vein cells using single-cell RNA-seq</h4><div id="p0610" role="paragraph">We used scRNAseq to quantify the degree of transcriptional similarity between hPSC-derived artery and vein ECs (<i>in&nbsp;vitro</i>; described above) as well as human artery and vein ECs obtained from a Carnegie Stage 13 (CS13) human fetus (<i>in&nbsp;vivo</i>). scRNAseq data of CS13 human fetal artery and vein ECs were generated previously (<span class="dropBlock reference-citations"><a id="body-ref-sref125-9" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-9" aria-controls="sref125-9">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and were downloaded from the Gene Expression Omnibus (GSE135202). Importantly, both <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> EC datasets were generated using the 10x Genomics Chromium platform (<span class="dropBlock reference-citations"><a id="body-ref-sref127-2" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127-2" aria-controls="sref127-2">Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref127-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Zheng, G.X.Y. ∙ Terry, J.M. ∙ Belgrader, P. ...</div><div><strong>Massively parallel digital transcriptional profiling of single cells</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14049</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85009446777" target="_blank" aria-label="Open Scopus in new tab">Scopus (2096)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14049&amp;pmid=28091601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), thus mitigating concerns that would otherwise arise from a cross-platform comparison.</div><div id="p0615" role="paragraph">We performed quality control and normalization for CS13 human fetal scRNAseq data using the methods described above for hPSC-derived artery and vein ECs (section “Single-cell RNA-sequencing”). In brief, dead/dying cells as well as doublets were computationally excluded and then CS13 human fetal scRNAseq data were normalized using the “LogNormalized” function and scaled using a linear transformation in Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-4" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-4" aria-controls="sref111-4">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0620" role="paragraph">To decompose the CS13 human fetal scRNAseq dataset into its constituent cell-types, we used Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-5" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-5" aria-controls="sref111-5">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to perform Louvain clustering (resolution 0.05), which separated the fetal cell population into 4 subsets: artery ECs (<i>CDH5+, PECAM1+, DLL4+, CXCR4+, HEY1+</i>); vein ECs (<i>CDH5+, PECAM1+, NT5E+, APLNR+, NR2F2+, NRP2+</i>); mesenchymal (<i>TWIST2+ CXCL12</i>+ <i>DLK1+ HAND1+ FBLN5+ WT1+</i>), and epithelial cells (<i>EPCAM+, CDX2+</i>). The existence of these 4 clusters—and the marker genes used to discriminate them—are consistent with what was reported in the original report that described these scRNAseq data (<span class="dropBlock reference-citations"><a id="body-ref-sref125-10" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125-10" aria-controls="sref125-10">Zeng et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref125-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Next, we focused our analysis only on the fetal ECs: we selected cells that expressed pan-endothelial genes (<i>PECAM1</i> and <i>CDH5</i>) at a level of UMI&nbsp;&gt;&nbsp;1.221 (i.e., UMI e<sup>0</sup><sup>.</sup><sup>2</sup>) in both the CS13 artery and CS13 vein subpopulations for downstream analyses.</div><div id="p0625" role="paragraph">In parallel, we computationally isolated ECs from our scRNAseq datasets of hPSC-derived day 3 artery and day 4 vein ECs <i>in&nbsp;vitro</i>. Louvain clustering (resolution&nbsp;= 0.1) was applied using Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-6" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-6" aria-controls="sref111-6">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which decomposed the day 3 population into artery ECs (<i>CDH5</i>+, <i>PECAM1</i>+, CD34+, <i>SOX17</i>+, <i>CXCR4</i>+, <i>DLL4</i>+) vs. non-ECs; similarly, this procedure decomposed the day 4 population into vein ECs (<i>CDH5</i>+, <i>PECAM1</i>+, CD34+, <i>NR2F2</i>+, <i>NT5E</i>+, <i>FLRT2</i>+) vs. non-ECs. The hPSC-derived artery ECs and vein ECs were computationally isolated and then used for subsequent analyses.</div><div id="p0630" role="paragraph">We then merged our <i>in&nbsp;vitro</i> scRNAseq dataset—composed of hPSC-derived day 3 artery ECs and day 4 vein ECs, in addition to other <i>in&nbsp;vitro</i> cell-types—with the CS13 human fetal artery and vein EC dataset. The combined scRNAseq dataset was normalized and scaled using SCTransform (<span class="dropBlock reference-citations"><a id="body-ref-sref43-2" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43-2" aria-controls="sref43-2">Hafemeister and Satija, 2019</a><div class="dropBlock__holder" data-db-target-of="sref43-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (742)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to remove technical variation while retaining biological variation. Next, we calculated the average expression of each gene in the SCTransform-normalized dataset (containing both <i>in&nbsp;vivo</i> and <i>in&nbsp;vitro</i> ECs) using the “AverageExpression” function in Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-7" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-7" aria-controls="sref111-7">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then identified the top 6000 variable genes in this merged dataset using the “CellScatter” function.</div><div id="p0635" role="paragraph">On the basis of the expression of these 6000 variable genes, we calculated the Pearson correlation coefficient 1) between our hPSC-derived artery ECs vs. CS13 human fetal artery ECs and 2) between our hPSC-derived vein ECs vs. CS13 human fetal vein ECs.</div><div id="p0640" role="paragraph">To find differentially-expressed genes between hPSC-derived vs. CS13 fetal artery and vein ECs, we used the function “FindMarkers” in Seurat v3 (<span class="dropBlock reference-citations"><a id="body-ref-sref111-8" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-8" aria-controls="sref111-8">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref111-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), with parameters set to only identify genes that were expressed in at least 25% of cells within that subcluster. This identified the most differentially-expressed genes between <i>1</i>) hPSC-derived vs. human fetal artery ECs and <i>2</i>) hPSC-derived vs. human fetal vein ECs. Of these genes that strongly differed in expression levels between <i>in&nbsp;vitro</i>-derived vs. <i>in&nbsp;vivo</i> ECs, we manually annotated those that had been previously linked to ribosome translation (<i>GNB2L1</i>, <i>EEF1G</i>, <i>RPL17</i>, <i>RPS17</i>) and mitochondrial respiration (<i>ATP5G2</i>, <i>ATP5E</i>, <i>ATP5L</i>, <i>MT-ND2</i>, <i>MT-ND1</i>, <i>MT-ND4</i>, <i>MT-CTB</i>, <i>MT-CO1</i>, <i>MT-ATP6</i>, <i>MT-CO2</i>, <i>MT-CO3</i>, <i>MT-CTB</i>). Finally, we performed gene ontology analysis on the lists of genes differentially expressed between hPSC-derived vs. CS13 fetal ECs using the online Enrichr platform (<span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">Chen et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Chen, E.Y. ∙ Tan, C.M. ∙ Kou, Y. ...</div><div><strong>Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84876100615" target="_blank" aria-label="Open Scopus in new tab">Scopus (2789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23586463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-128&amp;pmid=23586463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Kuleshov et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Kuleshov, M.V. ∙ Jones, M.R. ∙ Rouillard, A.D. ...</div><div><strong>Enrichr: a comprehensive gene set enrichment analysis web server 2016 update</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W90-W97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw377" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84987663170" target="_blank" aria-label="Open Scopus in new tab">Scopus (3602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27141961/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw377&amp;pmid=27141961" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-25"><h4>Bulk-population RNA-seq of uninfected cells</h4><div id="p0645" role="paragraph">We performed bulk-population RNA-seq on H1 hPSC-derived day 3 artery ECs and day 4 vein ECs, both of which were FACS purified for CD144<sup>+</sup> ECs to exclude any spurious gene expression from contaminating non-ECs. Cells were lysed using Zymo RNA lysis buffer and RNA was extracted using the Zymo Quick RNA kit according to the manufacturer’s instructions. We determined the RNA integrity number (RIN) of each sample using Agilent Bioanalyzer on-chip electrophoresis, and high-quality RNA samples with RIN ≥ 8 were enriched for poly(A) sequences and used to construct RNA-seq libraries, which were then ligated to adaptors for multiplexing. RNA-seq libraries were then sequenced on the DNBSEQ-G400 sequencer using DNA nanoball and combinational probe anchor synthesis technology by BGI Global Genomic Services to generate 150-bp paired-end reads. To minimize batch effects, all libraries were pooled together into a single sample, which was then sequenced on multiple lanes.</div><div id="p0650" role="paragraph">After obtaining RNA-seq reads, we performed quality control using FastQC (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Andrews, S.</div><div><strong>FastQC: A quality control tool for high throughput sequence data</strong></div><div><a id="intref0140" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a><div>Date: 2010</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+A+quality+control+tool+for+high+throughput+sequence+datahttp%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F2010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Next, we used Trim Galore (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Krueger, 2012</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Krueger, F.</div><div><strong>Trim galore</strong></div><div><a id="intref0145" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a><div>Date: 2012</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+galorehttps%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F2012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to remove index adaptors, low-quality base-calls or reads less than 20bp, using a read quality cutoff Phred score of 33. Trimmed and filtered reads from each single library (which was originally sequenced on multiple lanes) were then merged into a pooled file, each representing a single library. Subsequently reads were aligned with the GRCh38 (hg38) <i>Homo sapiens</i> reference genome and gene-level RNA-seq counts were quantified using Kallisto (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3519" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84966283954" target="_blank" aria-label="Open Scopus in new tab">Scopus (3483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3519&amp;pmid=27043002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then filtered and log<sub>2</sub> normalized counts per million (Tidyverse, BaseR, and EdgeR) (<span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120-2" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120-2" aria-controls="sref120-2">Wickham et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref120-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the Tidyverse</strong></div><div><em>J.&nbsp;Open Source Software.</em> 2019; <strong>4</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.21105/joss.01686" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.21105%2Fjoss.01686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Then, we performed dimensional reduction and differential gene expression analyses (Limma and EdgeR) (<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Law et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Law, C.W. ∙ Chen, Y. ∙ Shi, W. ...</div><div><strong>voom: precision weights unlock linear model analysis tools for RNA-seq read counts</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:R29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2014-15-2-r29" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84896735766" target="_blank" aria-label="Open Scopus in new tab">Scopus (2673)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24485249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2014-15-2-r29&amp;pmid=24485249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99-2" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-2" aria-controls="sref99-2">Ritchie et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref99-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Ritchie, M.E. ∙ Phipson, B. ∙ Wu, D. ...</div><div><strong>limma powers differential expression analyses for RNA-sequencing and microarray studies</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:e47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84926507971" target="_blank" aria-label="Open Scopus in new tab">Scopus (13963)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv007&amp;pmid=25605792" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref100-3" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-3" aria-controls="sref100-3">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref100-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to identify genes differentially expressed between hPSC-derived artery vs. vein ECs. These data were visualized on volcano plots: genes that showed a log<sub>2</sub> fold differences less than 3 (i.e., less than 8-fold differential expression) were represented in gray, while differentially expressed genes (i.e., those with greater than 8-fold expression differences) were highlighted in distinct colors.</div></section><section id="sec-4-4-26"><h4>Generation of Nipah-pseudotyped lentivirus</h4><div id="p0655" role="paragraph">Non-replicating, <i>GFP</i>-expressing lentiviruses pseudotyped to solely display Nipah or vesicular stomatitis virus (VSV) envelope proteins were generated from HEK293T/17 packaging cells using a 3<sup>rd</sup> generation lentiviral packaging system. In brief, cell-culture plates were coated with poly-L-lysine (R&amp;D Systems, 3438-200-01) for 5&nbsp;minutes at room temperature, which was then aspirated. Then 1 million HEK293T/17 cells (ATCC, CRL-11268) were seeded onto each poly-L-lysine-coated well of a 6-well plate. 24 hours after seeding, HEK293T/17 cells were ∼80-90% confluent and they were transfected using FuGENE HD (Promega, E2311).</div><div id="p0660" role="paragraph">The following lentiviral packaging plasmids encoding Nipah or VSV envelope proteins were used:<div id="ulist0035" role="list"><div id="u0090" role="listitem"><div class="label">•</div><div class="content"><div id="p0665" role="paragraph"><b>pCAGGS-coNiV-G-HA</b>: Encodes a codon-optimized <i>NiV-G</i> gene whose amino acid product is identical to the protein encoded by the <i>NiV-G</i> reference gene (GenBank: <a id="interref155" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=NC_002728.1" target="_blank">NC_002728.1</a>: 8710-11255 nucleotides, from Nipah virus/human/MY/1999/CDC), and additionally carries a hemagglutinin (HA) tag (<span class="dropBlock reference-citations"><a id="body-ref-sref90-5" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-5" aria-controls="sref90-5">Palomares et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref90-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Provided by Dr. Benhur Lee’s laboratory (Icahn School of Medicine).</div></div></div><div id="u0095" role="listitem"><div class="label">•</div><div class="content"><div id="p0670" role="paragraph"><b>pcDNA3.1-coNiV-F T5F-Au1</b>: Encodes a codon-optimized <i>NiV-F</i> gene whose amino acid product is a truncated version of the protein encoded by the <i>NiV-F</i> reference gene (GenBank: <a id="interref160" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=NC_002728.1" target="_blank">NC_002728.1</a>: 6370-8706 nucleotides, from Nipah virus/human/MY/1999/CDC). However <i>1</i>) the cytoplasmic tail of NiV-F was truncated to only carry 5 amino acid residues (a change that improves the production of pseudotyped lentiviruses) and <i>2</i>) additionally it carries an AU1 tag (<span class="dropBlock reference-citations"><a id="body-ref-sref90-6" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-6" aria-controls="sref90-6">Palomares et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref90-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Provided by Dr. Benhur Lee’s laboratory (Icahn School of Medicine).</div></div></div><div id="u0100" role="listitem"><div class="label">•</div><div class="content"><div id="p0675" role="paragraph"><b>VSV-G</b>: Encodes vesicular stomatitis virus envelope protein (VSV-G). Provided by Dr. Owen Witte’s laboratory (University of California Los Angeles) (<span class="dropBlock reference-citations"><a id="body-ref-sref41-4" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41-4" aria-controls="sref41-4">Goldstein et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref41-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Goldstein, A.S. ∙ Drake, J.M. ∙ Burnes, D.L. ...</div><div><strong>Purification and direct transformation of epithelial progenitor cells from primary human prostate</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:656-667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2011.317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79955530295" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21527922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2011.317&amp;pmid=21527922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div></div></div><div id="p0680" role="paragraph">HEK293T/17 cells were transfected using FuGENE HD with the following amounts of lentiviral packaging plasmids per well of a 6-well plate: 1.38&nbsp;μg pMDL plasmid&nbsp;+ 0.53&nbsp;μg pREV plasmid&nbsp;+ 0.76&nbsp;μg envelope protein-encoding plasmid(s)&nbsp;+ 2.13&nbsp;μg pRS348 <i>EF1A-GFP</i> plasmid. For the envelope protein-encoding plasmid(s), the following amounts were used for: <i>1</i>) VSV pseudovirus (0.76&nbsp;μg VSV-G plasmid) or <i>2</i>) Nipah pseudovirus (0.37&nbsp;μg NiV-G plasmid&nbsp;+ 0.37&nbsp;μg NiV-F plasmid). For transfection, room temperature DMEM medium was first mixed with FuGENE, and then mixed with plasmid DNA; after 15&nbsp;minute incubation at room temperature, the transfection solution was added dropwise to HEK293T/17 cells.</div><div id="p0685" role="paragraph">16-20 hours post-transfection, the culture media was replaced with 1mL of pre-warmed DMEM&nbsp;+ 10% FBS media. 24 hours media later, the media (containing pseudotyped lentivirus particles encoding a constitutively-expressed <i>EF1A-GFP</i> reporter) was harvested and either immediately used for transduction experiments or frozen at -80 °C.</div></section><section id="sec-4-4-27"><h4>Transduction of endothelial cells with Nipah-pseudotyped lentivirus</h4><div id="p0690" role="paragraph">After Nipah- and VSV-pseudotyped lentiviruses were generated as described above, they were used to transduce hPSC-derived artery and vein endothelial cells. First, wells in a 96-well, flat-bottom plate were coated with Geltrex (50-100&nbsp;μL/well), which was then aspirated. Subsequently, hPSC-derived artery or vein cells (generated from either wild-type or EFNB2<sup>-/-</sup> hPSCs) were thawed and 16,800-67,200 cells were seeded per Geltrex-coated well of a 96-well plate in 100&nbsp;μL of respective artery or vein expansion media supplemented with the ROCK inhibitor thiazovivin for 24 hours. Artery or vein expansion media was changed every 24 hours, and thiazovivin was withheld after the initial 24 hour seeding period.</div><div id="p0695" role="paragraph">72 hours post-seeding, media was removed from artery and vein cells, and then 100&nbsp;μL of fresh artery or vein expansion media was added, in addition to 0.3-100&nbsp;μL of viral supernatant containing pseudotyped lentiviruses (or control media lacking pseudotyped viruses). Pseudotyped lentiviruses were added for 24 hours prior to removal of culture media, and subsequently, media was replaced with fresh artery and vein media every 24 hours.</div><div id="p0700" role="paragraph">6&nbsp;days post-seeding (i.e., 3&nbsp;days post-transduction), each well of artery or vein cells was dissociated through the addition of 50&nbsp;μL TrypLE Express (incubated at 37 °C, 3-6&nbsp;minutes) and trituration. Then, 150&nbsp;μL of FACS buffer was added per well, and cells were transferred into a 96-well V-bottom plate before centrifugation (at 4 °C, 5&nbsp;minutes). Subsequently, supernatant was aspirated to remove the dissociation buffer, and cells were resuspended in 150&nbsp;μL of FACS buffer&nbsp;+ DAPI per well. Flow cytometry was performed using a CytoFLEX flow cytometer (Beckman Coulter Life Sciences) to enumerate the percentage of GFP<sup>+</sup> (i.e., virally-transduced) viable cells in each well.</div><div id="p0705" role="paragraph">The percentage of GFP<sup>+</sup> cells was used to quantify the relative efficiency with which a given pseudotyped lentivirus transduces artery vs. vein endothelial cells (i.e., the arteriovenous tropism of a given pseudotyped lentivirus). In brief, artery and vein cells were inoculated with an <i>equal</i> volume of lentiviral supernatant containing Nipah- or VSV-pseudotyped lentivirus and the percentage of GFP<sup>+</sup> cells were quantified by flow cytometry, as described above. In parallel, wells of artery and vein cells were dissociated and counted to determine cell numbers, in order to correct for any differences in transduction efficiency attributable to the use of different numbers of artery vs. vein cells.</div><div id="p0710" role="paragraph">Next, the multiplicity of infection (i.e., after inoculation with a given volume of pseudotyped lentiviruses, the <i>number</i> of pseudotyped lentiviruses per each single cell) was calculated using Poisson statistics as follows. Assuming that, after viral inoculation, the number of viruses that transduces a single cell follows a Poisson distribution (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Ellis and Delbrück, 1939</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Ellis, E.L. ∙ Delbrück, M.</div><div><strong>The growth of bacteriophage</strong></div><div><em>J.&nbsp;Gen. Physiol.</em> 1939; <strong>22</strong>:365-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1085/jgp.22.3.365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0001489791" target="_blank" aria-label="Open Scopus in new tab">Scopus (354)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19873108/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1085%2Fjgp.22.3.365&amp;pmid=19873108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), then: P(n)&nbsp;= (m<sup>n</sup> <sup>∗</sup> e<sup>-m</sup>)/n!, where P(n) is the probability that a cell will be transduced by <i>n</i> number of viruses and <i>m</i> is the multiplicity of infection.</div><div id="p0715" role="paragraph">The frequency of non-transduced cells in a given cell population is described by P(n&nbsp;= 0)&nbsp;= e<sup>-m</sup>. The frequency of transduced cells is described by P(n&nbsp;&gt;&nbsp;0)&nbsp;= 1 – P(0). Consequently, P(n&nbsp;&gt;&nbsp;0)&nbsp;= 1 – e<sup>-m</sup>.</div><div id="p0720" role="paragraph">In this case, after transduction with <i>GFP</i>-encoding lentiviruses, P(n&nbsp;&gt;&nbsp;0) is equivalent to the percentage of GFP<sup>+</sup> cells in a given cell population (henceforth “GFP<sup>+</sup>”). In summary, the multiplicity of infection (<i>m</i>) can be described by: m&nbsp;= –ln(1 – GFP<sup>+</sup>). In this study, the percentage of GFP<sup>+</sup> cells was always greater than 60% (i.e., the multiplicity of infection was always less than 0.916), thereby enabling the multiplicity of infection to be accurately determined within the linear range of this quantitative assay. In this manner, the percentage of GFP<sup>+</sup> artery or vein cells (enumerated by flow cytometry) was used to calculate the multiplicity of infection, which was then multiplied by the number of artery or vein cells to yield the inferred number of pseudotyped lentivirus particles that were originally used to inoculate the artery or vein cells. Given that the same absolute <i>volume</i> of pseudotyped lentivirus was used to inoculate artery or vein cells, any differences in the inferred <i>number</i> of pseudotyped lentivirus particles for artery vs. vein cells must be attributable to differences in the <i>tropism</i> of the pseudotyped lentivirus for artery vs. vein cells. Finally, the inferred number of pseudotyped lentivirus particles used to inoculate artery cells was divided by the inferred number of pseudotyped lentivirus particles used to inoculate vein cells, thus calculating the <i>arteriovenous tropism</i> of Nipah- or VSV-pseudotyped lentiviruses.</div></section><section id="sec-4-4-28"><h4>Generating stocks of Nipah and Hendra viruses and viral quantification</h4><div id="p0725" role="paragraph">Authentic Nipah and Hendra viruses are classified in Risk Group 4 (<span class="dropBlock reference-citations"><a id="body-ref-sref15-3" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-3" aria-controls="sref15-3">CDC and NIH, 2020</a><div class="dropBlock__holder" data-db-target-of="sref15-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>CDC, NIH</div><div><strong>Biosafety in Microbiological and Biomedical Laboratories</strong></div><div>United States Department of Health and Human Services, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=CDCNIHBiosafety+in+Microbiological+and+Biomedical+Laboratories6th+edition2020United+States+Department+of+Health+and+Human+Services" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). All experiments with Risk Group 4 viruses were conducted under maximum containment conditions in the biosafety level 4 (BSL4) laboratory of the Robert Koch Institute, in accordance with standard operating procedures institutionally approved by the Robert Koch Institute.</div><div id="p0730" role="paragraph">For this study, Nipah virus, Malaysia variant (Nipah virus/human/MY/1999/CDC) (<span class="dropBlock reference-citations"><a id="body-ref-sref45-3" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45-3" aria-controls="sref45-3">Harcourt et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref45-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Harcourt, B.H. ∙ Tamin, A. ∙ Ksiazek, T.G. ...</div><div><strong>Molecular characterization of Nipah virus, a newly emergent paramyxovirus</strong></div><div><em>Virology.</em> 2000; <strong>271</strong>:334-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/viro.2000.0340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0034608231" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10860887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fviro.2000.0340&amp;pmid=10860887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Hendra virus, Brisbane variant (Hendra virus/horse/AU/1994/Hendra) (<span class="dropBlock reference-citations"><a id="body-ref-sref78-7" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-7" aria-controls="sref78-7">Murray et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref78-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were obtained from CDC Viral Special Pathogens Branch reference stocks. These viruses were then subsequently propagated on Vero E6 cells (Vero C1008 (Vero 76, clone E6, Vero E6), from the European Collection of Authenticated Cell Cultures (ECACC)) in DMEM&nbsp;+ 2% FBS to generate archive stocks, which were frozen at -80°C. As needed, archive stocks were thawed and passaged once more on Vero E6 to generate working stocks of these viruses, which were frozen at -80°C. Working stocks of these viruses were used for all subsequent experiments.</div><div id="p0735" role="paragraph">These viral stocks were quantified using the tissue culture infectious dose 50% (TCID<sub>50</sub>) assay. For viral quantitation, Vero E6 cells were seeded into 96-well tissue culture plates in DMEM containing 10% FBS, 2&nbsp;mM glutamine, and 1% penicillin/streptomycin, at such a density that after overnight plating the cells became 95-100% confluent the following day. Separately, serial 10-fold dilutions of virus stocks were then diluted in DMEM. Then, the Vero E6 culture media was aspirated and 100&nbsp;μL of viral supernatant was added to triplicate wells of Vero E6 cells. This was incubated for 1 hour at 37 °C, 5% CO<sub>2</sub>. Subsequently, 100&nbsp;μL of DMEM containing 4% FBS, 4&nbsp;mM glutamine, and 2% penicillin/streptomycin was added to the wells (such that the final FBS concentration was 2%) and this was incubated for 4-5&nbsp;days at 37 °C, 5% CO<sub>2</sub>. The cells were then scored for the presence of cytopathic effects (CPE). The endpoint dilution of the viral stock in which 50% of the wells exhibited CPE was calculated using the Spearman-Karber method, therefore quantifying the viral dose present in the original viral stocks (expressed as TCID<sub>50</sub> units/mL). These titrated viral stocks were then used to infect hPSC-derived artery and vein ECs, as described below.</div><div id="p0740" role="paragraph">In certain experiments, Sendai virus, Cantell strain (<span class="dropBlock reference-citations"><a id="body-ref-sref12-2" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12-2" aria-controls="sref12-2">Cantell and Hirvonen, 1981</a><div class="dropBlock__holder" data-db-target-of="sref12-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Cantell, K. ∙ Hirvonen, S.</div><div><strong>Preparation and assay of Sendai virus</strong></div><div><em>Methods Enzymol.</em> 1981; <strong>78</strong>:299-301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0076-6879(81)78131-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6173605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0076-6879%2881%2978131-x&amp;pmid=6173605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—a paramyxovirus not known to cause disease (<span class="dropBlock reference-citations"><a id="body-ref-sref67-4" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67-4" aria-controls="sref67-4">López et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref67-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>López, C.B. ∙ Yount, J.S. ∙ Hermesh, T. ...</div><div><strong>Sendai virus infection induces efficient adaptive immunity independently of type I interferons</strong></div><div><em>J.&nbsp;Virol.</em> 2006; <strong>80</strong>:4538-4545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.80.9.4538-4545.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646184859" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16611914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.80.9.4538-4545.2006&amp;pmid=16611914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref124-2" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-2" aria-controls="sref124-2">Yoshida et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref124-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Yoshida, A. ∙ Kawabata, R. ∙ Honda, T. ...</div><div><strong>A single amino acid substitution within the paramyxovirus Sendai virus nucleoprotein is a critical determinant for production of interferon-beta-inducing copyback-type defective interfering genomes</strong></div><div><em>J.&nbsp;Virol.</em> 2018; <strong>92</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02094-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85042056136" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29237838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02094-17&amp;pmid=29237838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)—was used as a control for Nipah and Hendra viruses, which are highly-lethal paramyxoviruses. Sendai virus is classified in Risk Group 2, but was handled in the BSL4 laboratory for consistency. Sendai virus, Cantell strain was obtained from ATCC (VR-907).</div></section><section id="sec-4-4-29"><h4>Infection of hPSC-derived artery and vein cells with Nipah or Hendra viruses</h4><div id="p0745" role="paragraph">Prior to viral infection, frozen stocks of hPSC-derived artery and vein ECs were thawed as described above and propagated on Geltrex-coated 24-well plates, 48-well plates or 8-well chamber slides (Ibidi, 80826) in their respective expansion media (EGM2 for artery ECs and EGM2&nbsp;+ RO4929097&nbsp;+ SB505124 for vein ECs) until they became fully confluent, at 4-5&nbsp;days post-seeding. At that point, they were then transferred into BSL4 containment.</div><div id="p0750" role="paragraph">In each experiment, hPSC-derived artery or vein ECs were then inoculated with the same infectious quantity of a given virus. This enabled a side-by-side comparison of the responses of artery vs. vein ECs. Artery and vein ECs were inoculated with viruses for 1 hour (37°C, 5% CO<sub>2</sub>) in EGM2 medium. Different amounts of virus (titrated as described above) were used in various experiments depending on the objective of the experiment:<div id="ulist0040" role="list"><div id="u0105" role="listitem"><div class="label">•</div><div class="content"><div id="p0755" role="paragraph"><b>Visualization of syncytia formation</b> <b>and cytopathic effects at a fixed timepoint</b> (<a id="crosref2375" href="#fig5">Figures&nbsp;5</a>A, 5B, and <a id="crosref2380" href="#figs5">S5</a>E): 10<sup>4</sup> TCID<sub>50</sub> units of Nipah or Hendra virus were added per well of a 24-well plate</div></div></div><div id="u0110" role="listitem"><div class="label">•</div><div class="content"><div id="p0760" role="paragraph"><b>Visualization of syncytia formation through live-imaging</b> (<a id="crosref2385" href="#fig5">Figures&nbsp;5</a>C–5E, <a id="crosref2390" href="#figs5">S5</a>H, and S5I): 10<sup>4</sup> TCID<sub>50</sub> units of Nipah or Hendra virus were added per well of an 8-well chamber slide</div></div></div><div id="u0115" role="listitem"><div class="label">•</div><div class="content"><div id="p0765" role="paragraph"><b>Quantification of viral replication kinetics</b> (<a id="crosref2395" href="#fig6">Figures&nbsp;6</a>A and <a id="crosref2400" href="#figs6">S6</a>A): 5×10<sup>3</sup> TCID<sub>50</sub> units of Nipah or Hendra virus were added per well of a 48-well plate</div></div></div><div id="u0120" role="listitem"><div class="label">•</div><div class="content"><div id="p0770" role="paragraph"><b>RNA-seq analysis of infected cells</b> (<a id="crosref2405" href="#fig7">Figures&nbsp;7</a>A–7D and <a id="crosref2410" href="#figs7">S7</a>A–S7C): 10<sup>5</sup> TCID<sub>50</sub> units of Nipah or Hendra virus, or 30 HA units of Sendai virus, were added per well of a 24-well plate</div></div></div><div id="u0125" role="listitem"><div class="label">•</div><div class="content"><div id="p0775" role="paragraph"><b>IFNβ secretion by infected cells</b> (<a id="crosref2415" href="#fig7">Figures&nbsp;7</a>E and <a id="crosref2420" href="#figs7">S7</a>D): 5×10<sup>4</sup> TCID<sub>50</sub> units of Nipah or Hendra virus, or 15 HA units of Sendai virus, were added per well of a 48-well plate</div></div></div></div></div><div id="p0780" role="paragraph">After 1 hour of incubation with virus, cells were washed twice in EGM2 to remove residual virus in the media. Infected artery and vein ECs were then cultured in their respective arterial or venous expansion media for various lengths of time, as described for each individual experiment. Media was changed every 24 hours.</div></section><section id="sec-4-4-30"><h4>Assessing cytopathic effects in virally-infected hPSC-derived artery and vein cells</h4><div id="p0785" role="paragraph">To visualize cytopathic effects, Diff-Quik staining (Labor+Technik, LT 001) was performed on hPSC-derived ECs that had been infected with Nipah virus, Hendra virus or left uninfected for 24 hours. For each condition, 5 fields of view were randomly selected and imaged. In each image, the number of total nuclei as well as syncytia were quantified using ImageJ/Fiji. To count multinucleated cells, ImageJ/Fiji’s cell counter was used to label and tally the total number of cells with 1 nucleus or ≥2 nuclei. To count the total number of cells, multiple ImageJ/Fiji plugins were used to process the images automatically. Images were converted to 8-bit format, and then thresholded, smoothened and filtered to remove noise (despeckled). In each image, overlapping objects were separated using the function “watershed” and finally, cells/particles were counted using the function “analyze particles”. In summary, both <i>1</i>) the number of nuclei per cell and <i>2</i>) the frequency of multinucleated cells relative to total cell numbers were quantified.</div></section><section id="sec-4-4-31"><h4>Live imaging of Nipah and Hendra infection in BSL4 conditions</h4><div id="p0790" role="paragraph">To visualize the kinetics and extent of Nipah and Hendra virus-induced cell fusion and cell death in real time, live imaging was performed under maximum containment in the BSL4 laboratory. To quantify cell fusion, GFP<sup>+</sup> ECs (obtained from the SUN004.1.9 hiPSC line) and wild-type ECs (obtained from the H1 hESC line) were cocultured at a 4:1 ratio, such that fusion between cells of these two genotypes would manifest as diffusion of GFP from one cell to the other.</div><div id="p0795" role="paragraph">To this end, frozen stocks of wild-type H1-derived and GFP<sup>+</sup> SUN004.1.9-derived artery and vein ECs were thawed and cocultured as described above, on Geltrex-coated 8-well chamber slides compatible with microscopy (Ibidi, 80826) in 200&nbsp;μL of their respective arterial or venous expansion media. After infection with Nipah virus (10<sup>4</sup> TCID<sub>50</sub> units/well), Hendra virus (10<sup>4</sup> TCID<sub>50</sub> units/well) or a negative control (uninfected), live imaging of artery and vein ECs was performed on a spinning disk confocal microscope (CorrSight, FEI) for 24 hours, with a frame rate of 5&nbsp;minutes, at 37°C, 5% CO<sub>2</sub>, using a 5x objective lens. For these live imaging experiments, a high virus titer was used in order to rapidly induce cytopathic effects to visualize virally-induced cell fusion and death.</div></section><section id="sec-4-4-32"><h4>Computational analysis of virally-infected live imaging data</h4><div id="p0800" role="paragraph">Time-lapse videos were imported into Fiji and then analyzed using Mastodon or Trackmate v6.0.2 (<span class="dropBlock reference-citations"><a id="body-ref-sref114-2" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114-2" aria-controls="sref114-2">Tinevez et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref114-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Tinevez, J.Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: an open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2016.09.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85002002648" target="_blank" aria-label="Open Scopus in new tab">Scopus (1289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27713081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2016.09.016&amp;pmid=27713081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As described above, cocultures of wild-type and GFP<sup>+</sup> cells were used for live-imaging experiments. GFP<sup>+</sup> cells were identified as spots or blobs with an estimated blob/spot size of 20 pixels (using the Laplacian of Gaussian [LoG] filter). Then, the 20-pixel spots were filtered based on a quality score above 0.26-0.28. Next, the spots were then linked from frame to frame to build track segments and linking costs of the tracks were calculated (using LAP tracker, based on the Linear Assignment Problem mathematical framework). The parameters for frame-to-frame linking, track segment gap closing, and track segment merging were all set to a maximum distance of 15 pixels. The outputs of this analysis include the median velocity of spots and cell tracks. Cell tracks were exported as .xml files and imported into Icy (<span class="dropBlock reference-citations"><a id="body-ref-sref26-2" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26-2" aria-controls="sref26-2">de Chaumont et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref26-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>de Chaumont, F. ∙ Dallongeville, S. ∙ Chenouard, N. ...</div><div><strong>Icy: an open BioImage informatics platform for extended reproducible research</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:690-696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2075" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84863191618" target="_blank" aria-label="Open Scopus in new tab">Scopus (828)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2075&amp;pmid=22743774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to obtain images whereby cell tracks were overlaid over the microscope field. To compare the median velocities of infected artery vs. vein cells, the Mann Whitney non-parametric t-test was performed and two-tailed p-values calculated to determine statistical significance.</div></section><section id="sec-4-4-33"><h4>Quantification of viral replication</h4><div id="p0805" role="paragraph">hPSC-derived artery and vein ECs were inoculated with Nipah and Hendra viruses for 1 hour and then washed twice in EGM2 to remove excess virus (as described above). After viral inoculation, both the extracellular levels of viral RNA (encompassing viral particles budding into the culture media) and the intracellular levels of viral RNA (encompassing intracellular viral replication) were quantified across different timepoints.</div><div id="p0810" role="paragraph">To quantify viral genomes present in the media, 140&nbsp;μL of the culture media (i.e., supernatant) was placed in 560&nbsp;μL of lysis buffer AVL (Qiagen QIAamp Viral RNA extraction kit, Qiagen, 52904), in addition to 560&nbsp;μL of 100% ethanol, in order to inactivate it for transfer out of the BSL4 laboratory. Then, RNA was extracted following the manufacturer’s instructions. To quantify intracellular viral genomes, RNA was first extracted from cellular monolayers as described below (in the section “Bulk-population RNA-seq of virally-infected cells”). Subsequently, qRT-PCR was performed using primers directed against the viral genome and the AgPath-ID<sup>TM</sup> One-Step RT-PCR Kit (Thermo&nbsp;Fisher, 4387391).</div></section><section id="sec-4-4-34"><h4>Bulk-population RNA-seq of virally-infected cells</h4><div id="p0815" role="paragraph">H1 hPSC-derived and SUN004.1.9 hPSC-derived artery and vein ECs in 24-well plates were inoculated with Nipah virus (10<sup>5</sup> TCID<sub>50</sub> units/well), Hendra virus (10<sup>5</sup> TCID<sub>50</sub> units/well) or Sendai virus, Cantell strain (30 HA units/well) or left uninfected (negative control). For RNA-seq experiments, a high viral dose (multiplicity of infection [MOI]&nbsp;≈ 5) was used in order to synchronize infection in a large proportion of cells. In the same experiment, hPSC-derived artery and vein ECs from two separate genetic backgrounds (H1 hESC-derived and SUN004.1.9 hiPSC-derived) were infected to ensure the consistency of any transcriptional changes that were identified. The timepoints of 5, 10 and 15 hours post-infection were chosen because they encompassed viral entry, viral replication and virus-induced cell fusion prior to extensive cellular destruction at 24 hours post-infection induced by Nipah virus (at the viral dose used in this experiment). For each timepoint, Nipah-, Hendra- or Sendai-infected cells were transcriptionally compared against uninfected cells (negative control) harvested at the same timepoint.</div><div id="p0820" role="paragraph">Cells were harvested at 5 hours, 10 hours, and 15 hours post-infection; at each timepoint, cells were lysed in 350&nbsp;μL of buffer RLT from the RNeasy Micro Kit (Qiagen, 74004) and then triturated with 350&nbsp;μL of 70% ethanol to inactivate infectious material for transfer out of the BSL4 laboratory. Then, RNA was extracted as per the manufacturer’s instructions. After quality control procedures, polyadenylated mRNA was enriched using oligo(dT) beads. The mRNA was then fragmented randomly in fragmentation buffer, followed by cDNA synthesis using random hexamers and reverse transcriptase. After first-strand synthesis, a second-strand synthesis buffer (Illumina) was added with dNTPs, RNase H and E.&nbsp;coli polymerase I to generate the second strand by nick-translation. The final cDNA library was then subjected to a round of purification, terminal repair, A-tailing, ligation of sequencing adapters, size selection and PCR enrichment. RNA-seq libraries were sequenced on an Illumina NovaSeq 6000 sequencer by Novogene.</div><div id="p0825" role="paragraph">After obtaining RNA-seq reads, quality control was performed using FastQC (<span class="dropBlock reference-citations"><a id="body-ref-sref2-3" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-3" aria-controls="sref2-3">Andrews, 2010</a><div class="dropBlock__holder" data-db-target-of="sref2-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Andrews, S.</div><div><strong>FastQC: A quality control tool for high throughput sequence data</strong></div><div><a id="intref0140" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a><div>Date: 2010</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+A+quality+control+tool+for+high+throughput+sequence+datahttp%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F2010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and Trim Galore (<span class="dropBlock reference-citations"><a id="body-ref-sref53-3" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-3" aria-controls="sref53-3">Krueger, 2012</a><div class="dropBlock__holder" data-db-target-of="sref53-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Krueger, F.</div><div><strong>Trim galore</strong></div><div><a id="intref0145" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a><div>Date: 2012</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+galorehttps%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F2012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as described above (section “Bulk-population RNA-seq of uninfected cells”). Trimmed and filtered reads from each individual library (which was originally sequenced on multiple lanes) were then merged into a single pooled file for each library.</div><div id="p0830" role="paragraph">RNA-seq profiles of virus-infected cells contained both human and viral reads. To analyze human (host cell) transcripts, reads were aligned to the GRCh38 (hg38) <i>Homo sapiens</i> reference genome and gene-level RNA-seq counts were quantified using Kallisto (<span class="dropBlock reference-citations"><a id="body-ref-sref11-3" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-3" aria-controls="sref11-3">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3519" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84966283954" target="_blank" aria-label="Open Scopus in new tab">Scopus (3483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3519&amp;pmid=27043002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These counts were imported into RStudio and differential gene expression analyses of these counts were performed using DESeq2 (<span class="dropBlock reference-citations"><a id="body-ref-sref68-2" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68-2" aria-controls="sref68-2">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref68-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (29293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For all samples, regularized log transformation was performed on the counts to remove the dependence of variance on the mean.</div><div id="p0835" role="paragraph">To determine the overall percentage of viral reads within a given RNA-seq sample, Kallisto (<span class="dropBlock reference-citations"><a id="body-ref-sref11-4" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-4" aria-controls="sref11-4">Bray et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref11-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3519" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84966283954" target="_blank" aria-label="Open Scopus in new tab">Scopus (3483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3519&amp;pmid=27043002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) was used to align reads to the reference genomes of Nipah virus (NCBI accession number GenBank: NC_002728), Hendra virus (NCBI accession number GenBank: NC_001906) or Sendai virus (NCBI accession number GenBank: NC_001552). This enabled the detection of all viral reads within a given sample. Then the number of total viral counts was divided over the number of total human&nbsp;+ viral counts to estimate what proportion of the cellular transcriptome comprised viral transcripts.</div><div id="p0840" role="paragraph">To quantify the expression of specific viral genes, Kallisto was used to align reads to the following viral genes:<div id="ulist0045" role="list"><div id="u0130" role="listitem"><div class="label">•</div><div class="content"><div id="p0845" role="paragraph">For Nipah virus, reads were aligned to Nipah F (NC_002728.1:6370-8706 nucleotides), Nipah G (NC_002728.1:8710-11255 nucleotides), Nipah L (NC_002728.1:11259-18213 nucleotides), Nipah M (NC_002728.1:5008-6366 nucleotides), Nipah N (NC_002728.1:56-2297 nucleotides) and Nipah P/V/W/C (otherwise known as Nvgp2; NC_002728.1:2301-5004 nucleotides).</div></div></div><div id="u0135" role="listitem"><div class="label">•</div><div class="content"><div id="p0850" role="paragraph">For Hendra virus, reads were aligned to Hendra F (NC_001906.3:6346-8676 nucleotides), Hendra G (NC_001906.3:8680-11243 nucleotides), Hendra P/V/W/C (otherwise known as Hvgp2; NC_001906.3:2283-4980 nucleotides), Hendra L (NC_001906.3:11247-18201 nucleotides), Hendra M (NC_001906.3:4984-6342 nucleotides) and Hendra N (NC_001906.3:56-2279 nucleotides).</div></div></div><div id="u0140" role="listitem"><div class="label">•</div><div class="content"><div id="p0855" role="paragraph">For Sendai virus, reads were aligned to Sendai F (NC_001552.1:4866-6563 nucleotides), Sendai HN (NC_001552.1:6693-8420 nucleotides), Sendai L (NC_001552.1:8556-15242 nucleotides), Sendai M (NC_001552.1:3669-4715 nucleotides), Sendai NP (NC_001552.1:120-1694 nucleotides) and Sendai P/V/W/C (NC_001552.1:1844-3550 nucleotides).</div></div></div></div></div></section><section id="sec-4-4-35"><h4>Quantification of IFNβ secretion by virally-infected cells</h4><div id="p0860" role="paragraph">H1 hPSC-derived, SUN004.1.9 hPSC-derived and SUN004.2 hPSC-derived artery and vein ECs in 48-well plates were inoculated with Nipah virus (5x10<sup>4</sup> TCID<sub>50</sub> units/well), Hendra virus (5x10<sup>4</sup> TCID<sub>50</sub> units/well), Sendai virus (15 HA units/well), or left uninfected (negative control), for 1 hour. For this experiment, a high viral dose (multiplicity of infection [MOI]&nbsp;≈ 5) was used in order to synchronize infection in a large proportion of cells.</div><div id="p0865" role="paragraph">After 1 hour of viral infection, cells were washed twice in EGM2 medium, and then artery and vein cells were cultured in their respective growth media in standard incubator conditions (37°C, 5% CO<sub>2</sub>). 12 and 24 hours later, supernatants (culture media) were harvested, and stored at -80 °C. Media was then thawed, and the concentration of secreted interferon-β (IFNβ) protein in the undiluted media was quantified using ELISA (enzyme-linked immunosorbent assay), specifically using the Human IFNβ Quantikine QuicKit ELISA Kit (R&amp;D Systems, QK410) following the manufacturer’s instructions. Absorbance was quantified using a TECAN Sunrise plate reader (Tecan Trading AG, Switzerland) with a spectral filter. The concentration of IFNβ in the samples was calculated by comparing the absorbance to that of a standard curve generated with human IFNβ, after subtracting background absorbance, using TECAN Magellan software. Data shown represent the average of results from three separate cell lines (H1, SUN004.1.9 and SUN004.2 hPSCs).</div></section><section id="sec-4-4-36"><h4>Calculation of case fatality rates</h4><div id="p0870" role="paragraph">The case fatality rates of both Nipah and Hendra viruses are currently estimated at ∼57-59%, and are referred to as such in this manuscript.<div id="ulist0050" role="list"><div id="u0145" role="listitem"><div class="label">•</div><div class="content"><div id="p0875" role="paragraph"><b>Nipah virus</b>: In a recent review (<span class="dropBlock reference-citations"><a id="body-ref-sref3-3" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-3" aria-controls="sref3-3">Ang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref3-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Ang, B.S.P. ∙ Lim, T.C.C. ∙ Wang, L.</div><div><strong>Nipah virus infection</strong></div><div><em>J.&nbsp;Clin. Microbiol.</em> 2018; <strong>56</strong></div><div>e01875–e01817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JCM.01875-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047953553" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJCM.01875-17" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), Nipah virus was recognized to have caused 326 fatalities among 573 human cases in the following regions: Malaysia and Singapore (1998-1999), India and Bangladesh (2001-2012) and the Philippines (2014). Additionally, the World Health Organization has recognized 48 fatalities among 65 new human cases in Bangladesh (2013-2015) and India (2018) (<a id="intref0130" href="https://web.archive.org/web/20180613144643/http://www.searo.who.int/entity/emerging_diseases/links/morbidity-and-mortality-nipah-sear-2001-2018.pdf?ua=1" target="_blank">https://web.archive.org/web/20180613144643/http://www.searo.who.int/entity/emerging_diseases/links/morbidity-and-mortality-nipah-sear-2001-2018.pdf?ua=1</a>). Taken together, Nipah virus is currently recognized to have caused 374 fatalities among 638 human cases (∼58.6% case fatality rate).</div></div></div><div id="u0150" role="listitem"><div class="label">•</div><div class="content"><div id="p0880" role="paragraph"><b>Hendra virus</b>: Hendra virus is currently recognized to have caused 4 fatalities among 7 human cases in Australia (∼57.1% case fatality rate) (<span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Mahalingam et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Mahalingam, S. ∙ Herrero, L.J. ∙ Playford, E.G. ...</div><div><strong>Hendra virus: an emerging paramyxovirus in Australia</strong></div><div><em>Lancet Infect. Dis.</em> 2012; <strong>12</strong>:799-807</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2812%2970158-5&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2812%2970158-5&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84866619198" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22921953/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2812%2970158-5&amp;pmid=22921953" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div></div></div></section></section><section id="sec-4-5"><h3>Quantification and statistical analysis</h3><div id="p0885" role="paragraph">Throughout this study, all error bars represent the standard error of the mean (s.e.m.) and a two-tailed t test was employed to determine statistical significance. Statistical significance values (P values) were reported in the figures and accompanying main text. Throughout all figures in this manuscript, wherever asterisks are used to denote statistical significance, <sup>∗</sup>P&lt;0.05 and <sup>∗∗</sup>P&lt;0.01, as defined in the figure legends. Statistical tests were implemented in Microsoft Excel and GraphPad Prism.</div><div id="p0890" role="paragraph">For qPCR analyses of differentiated hPSCs, for each experimental condition, two wells of cells in the same cell culture plate were analyzed (representing biological replicates) and for each of these wells, qPCR was performed twice in two separate wells of a 384-well qPCR plate (representing technical replicates). The average (mean) of these four datapoints is reported in histograms, and error bars represent the standard error of the mean.</div><div id="p0895" role="paragraph">For Diff-Quik staining of virally-infected hPSC-derived artery and vein cells, for each experimental condition, microscope images of five randomly-selected fields-of-view were generated. The number of nuclei per cell was quantified to determine syncytia formation: the average (mean) of data obtained from these five fields-of-view is reported in histograms, and error bars represent the standard error of the mean.</div><div id="p0900" role="paragraph">For qPCR analyses of viral replication in wild-type hPSC-derived artery and vein cells, for each experimental condition, three independent wells of cells in the same cell culture plate were analyzed (representing biological replicates). The average (mean) of these three datapoints is reported in histograms, and error bars represent the standard error of the mean.</div><div id="p0905" role="paragraph">For RNA-seq analyses of gene expression in virally-infected hPSC-derived artery and vein cells, for each experimental condition, two independent wells of cells in the same cell culture plate were analyzed (representing biological replicates). The average (mean) of these two datapoints is reported in histograms and heatmaps, and error bars represent the standard error of the mean.</div><div id="p0910" role="paragraph">For interferon-β secretion from virally-infected hPSC-derived artery and vein cells, for each experimental condition, three independent wells of cells in the same cell culture plate were analyzed (representing biological replicates). The average (mean) of these three datapoints is reported in histograms and heatmaps, and error bars represent the standard error of the mean.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0915" role="paragraph">We thank A. Elefanty, E. Stanley, and E. Ng for MIXL1-GFP and SOX17-mCherry hPSCs; H. Feldmann and M. Nakanishi for viruses; B. Lee, J. Weissman, A. Smith, and O. Witte for plasmids; P. Rios, T. Nishimura, N. Pek, S. Miao, M. Gu, and M. Rabinovitch for contributing to preliminary experiments; and M. Nichane, M.R. Corces, A. Kershner, A. Chang, I. Williams, P. Beachy, R. Nusse, K. Davis, B. Azizoglu, R. Saunders, W. Allen, S. Zheng, M. Blewett, I. Weissman, A. McCarty, and J. Choi for reagents and advice. Infrastructure support was provided by the Stanford Stem Cell FACS Core and Animal Facility, Stanford Diabetes Genomics Analysis Core Facility (NIH P30DK116074), C. Carswell-Crumpton, M. Alvarez, and L. Dunkin-Hubby. This work was supported by NIH (DP5OD024558 and R01HL128503), JDRF Northern California Center of Excellence, Additional Ventures Innovation Fund, Stanford Ludwig and Beckman Centers, Stanford Maternal and Child Health Research Institute, A<sup>∗</sup>STAR Industrial Alignment Fund (H18/01/a0/017), Robert Koch Institute intramural funds, and the Anonymous, Fickel, and Weintz families. This work was also supported by CIRM Bridges Program TB1-01195 (A.C. and A.T.N.); NIH T32GM119995 (R.M.M. and J.L.F.) and T32GM007365 (T.C.); NDSEG and Stanford Bio-X Fellowships (J.L.F.); and the Assar Gabrielsson Foundation and Swedish Research Council (2017-0034, J.B.). L.T.A. is a Siebel Investigator. K.R.-H. is an HHMI Investigator. K.M.L. and C.C. are Human Frontier Science Program Young Investigators (RGY0069/2019). K.M.L. is a Packard Foundation fellow, Pew Scholar, Baxter Foundation faculty scholar, and the Anthony DiGenova endowed faculty scholar.</div><section id="sec-5"><h3>Author contributions</h3><div id="p0920" role="paragraph">L.T.A., A.T.N., A.C., K.J.L., X.X., J.L.F., A.R.H., and K.M.L. differentiated hPSCs and, with M.C., B.C.R., C.Y.N., C.C., S.C.G., and K.R.-H., performed functional assays. L.T.A., K.J.L., X.X., J.L.F., S.V., and K.M.L. performed genomics analyses. R.M.M., B.J.L., M.H.P., F.P.S., T.N., J.B., and H.N. provided reagents. U.V., A.L., and J.B.P. performed BSL4 experiments. L.T.A., J.B.P., and K.M.L. supervised the study.</div></section><section id="sec-6"><h3>Declaration of interests</h3><div id="p0925" role="paragraph">K.M.L., L.T.A., A.C., A.T.N., and J.L.F. have filed patent applications related to endothelial differentiation.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental information (5)</h2><div id="p0935" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.05.024/attachment/ec5a3b6a-88ad-4e91-ae5a-f4edd54a110b/mmc1.csv" download="mmc1.csv" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">File (393.87 KB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table&nbsp;S1. Gene expression differences between hPSC-derived versus human fetal artery ECs, as assayed by single-cell RNA-seq, related to Figure&nbsp;2</div><div id="espara0010" role="paragraph">Single-cell RNA-seq of H1 hPSC-derived artery ECs <i>in&nbsp;vitro</i> vs. Carnegie Stage 13 human fetal artery ECs <i>in&nbsp;vivo</i>. Genes that are differentially expressed between <i>in&nbsp;vitro</i> vs. <i>in&nbsp;vivo</i> artery ECs are shown, and log<sub>e</sub> fold change is shown (i.e., positive fold change indicates higher expression in hPSC-derived cells, relative to <i>in&nbsp;vivo</i> cells). For each gene, “pct.1” and “pct.2” shows the percentage of cells that detectably express that gene, among H1 hPSC-derived artery ECs and human fetal artery ECs, respectively.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="file"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.05.024/attachment/93db3841-7932-4e65-8757-f9972dd15dd8/mmc2.csv" download="mmc2.csv" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">File (264.36 KB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table&nbsp;S2. Gene expression differences between hPSC-derived versus human fetal vein ECs, as assayed by single-cell RNA-seq, related to Figure&nbsp;3</div><div id="espara0015" role="paragraph">Single-cell RNA-seq of H1 hPSC-derived vein ECs in&nbsp;vitro vs. Carnegie Stage 13 human fetal vein ECs in&nbsp;vivo. Genes that are differentially expressed between in&nbsp;vitro vs. in&nbsp;vivo vein ECs are shown, and loge fold change is shown (i.e., positive fold change indicates higher expression in hPSC-derived cells, relative to in&nbsp;vivo cells.) For each gene, “pct.1” and “pct.2” shows the percentage of cells that detectably express that gene, among H1 hPSC-derived vein ECs and human vein artery ECs, respectively.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.05.024/attachment/4d8deffb-bdac-42c4-bd17-d471aafbd92c/mmc3.xls" download="mmc3.xls" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (570.00 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table&nbsp;S3. Differentially expressed genes between hPSC-derived artery versus vein ECs, as assayed by bulk-population RNA-seq, related to Figure&nbsp;4</div><div id="espara0020" role="paragraph">Bulk-population RNA-seq of H1 hPSC-derived day 3 artery ECs vs. day 4 vein ECs (both FACS sorted for CD144+ cells). All counts&nbsp;&gt;&nbsp;1 were normalized to the total read count and log2 transformed using Limma-Voom.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.05.024/attachment/8d3d4556-c2e9-4d01-bfeb-b6e42a79e361/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (7.41 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table&nbsp;S4. Bulk-population RNA-seq of hPSC-derived artery and vein cells infected by Nipah, Hendra, or Sendai viruses, related to Figure&nbsp;7</div><div id="espara0025" role="paragraph">H1 and SUN004.1.9 hPSC-derived artery and vein ECs were inoculated with Nipah, Hendra or Sendai viruses for 5, 10 or 15&nbsp;hr, and bulk-population RNA-seq was performed. Replicate 1&nbsp;= H1 and replicate 2&nbsp;= SUN004.1.9. Counts were regularized, log<sub>2</sub> transformed and normalized by size factors using DESeq2.</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2022.05.024/attachment/82e7f0de-0c6f-4470-839d-d9c041ee052b/mmc5.pdf" download="mmc5.pdf" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">PDF (47.39 KB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table&nbsp;S5. Quantitative PCR primers, related to Figures&nbsp;1, 2, 3, and 4</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="data-availability" class="core-data-availability"><h2>Data and code availability</h2><div id="p0030" role="paragraph">Bulk-population RNA-seq and single-cell RNA-seq datasets generated as part of this study are available at the NCBI Sequence Read Archive, accession number GenBank: PRJNA837932. Single-cell RNA-seq datasets of hPSC-derived artery and vein ECs that were generated as part of this study can be interactively browsed at a custom web portal: <a id="intref0010" href="https://anglab.shinyapps.io/arteryveinscrna/" target="_blank">https://anglab.shinyapps.io/arteryveinscrna/</a>. Computational scripts used for genomics analyses conducted as part of this study are available at Github: <a id="intref0015" href="https://github.com/LayTengAngLab/ArteryVeinEndothelial" target="_blank">https://github.com/LayTengAngLab/ArteryVeinEndothelial</a>.</div></section><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Aljofan, M. ∙ Saubern, S. ∙ Meyer, A.G. ...</div><div><strong>Characteristics of Nipah virus and Hendra virus replication in different cell lines and their suitability for antiviral screening</strong></div><div><em>Virus Res.</em> 2009; <strong>142</strong>:92-99</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virusres.2009.01.014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-67349241274" target="_blank" aria-label="Open Scopus in new tab">Scopus (30)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19428741/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virusres.2009.01.014&amp;pmid=19428741" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Andrews, S.</div><div><strong>FastQC: A quality control tool for high throughput sequence data</strong></div><div><a id="intref0140" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a><div>Date: 2010</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=S.AndrewsFastQC%3A+A+quality+control+tool+for+high+throughput+sequence+datahttp%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ffastqc%2F2010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Ang, B.S.P. ∙ Lim, T.C.C. ∙ Wang, L.</div><div><strong>Nipah virus infection</strong></div><div><em>J.&nbsp;Clin. Microbiol.</em> 2018; <strong>56</strong></div><div>e01875–e01817</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JCM.01875-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047953553" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJCM.01875-17" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4-1" title="View in article">4.</a></div><div>Aranguren, X.L. ∙ Agirre, X. ∙ Beerens, M. ...</div><div><strong>Unraveling a novel transcription factor code determining the human arterial-specific endothelial cell signature</strong></div><div><em>Blood.</em> 2013; <strong>122</strong>:3982-3992</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2013-02-483255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84891126258" target="_blank" aria-label="Open Scopus in new tab">Scopus (63)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24108462/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2013-02-483255&amp;pmid=24108462" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Arunkumar, G. ∙ Chandni, R. ∙ Mourya, D.T. ..., Nipah Investigators People and Health Study Group</div><div><strong>Outbreak investigation of Nipah virus disease in Kerala, India, 2018</strong></div><div><em>J.&nbsp;Infect. Dis.</em> 2019; <strong>219</strong>:1867-1878</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/infdis/jiy612" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066960608" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30364984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Finfdis%2Fjiy612&amp;pmid=30364984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6-1" title="View in article">6.</a></div><div>Augustin, H.G. ∙ Koh, G.Y.</div><div><strong>Organotypic vasculature: From descriptive heterogeneity to functional pathophysiology</strong></div><div><em>Science.</em> 2017; <strong>357</strong>:eaal2379</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aal2379" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026996599" target="_blank" aria-label="Open Scopus in new tab">Scopus (312)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28775214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aal2379&amp;pmid=28775214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Babicki, S. ∙ Arndt, D. ∙ Marcu, A. ...</div><div><strong>Heatmapper: web-enabled heat mapping for all</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W147-W153</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw419" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84994283822" target="_blank" aria-label="Open Scopus in new tab">Scopus (830)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27190236/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw419&amp;pmid=27190236" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8-1" title="View in article">8.</a></div><div>Baseler, L. ∙ de Wit, E. ∙ Scott, D.P. ...</div><div><strong>Syrian hamsters (Mesocricetus auratus) oronasally inoculated with a Nipah virus isolate from Bangladesh or Malaysia develop similar respiratory tract lesions</strong></div><div><em>Vet. Pathol.</em> 2015; <strong>52</strong>:38-45</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1177/0300985814556189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84920024648" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25352203/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1177%2F0300985814556189&amp;pmid=25352203" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (1416)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Bonaparte, M.I. ∙ Dimitrov, A.S. ∙ Bossart, K.N. ...</div><div><strong>Ephrin-B2 ligand is a functional receptor for Hendra virus and Nipah virus</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:10652-10657</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0504887102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-23044517161" target="_blank" aria-label="Open Scopus in new tab">Scopus (310)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15998730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0504887102&amp;pmid=15998730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Bray, N.L. ∙ Pimentel, H. ∙ Melsted, P. ...</div><div><strong>Near-optimal probabilistic RNA-seq quantification</strong></div><div><em>Nat. Biotechnol.</em> 2016; <strong>34</strong>:525-527</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.3519" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84966283954" target="_blank" aria-label="Open Scopus in new tab">Scopus (3483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.3519&amp;pmid=27043002" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12-1" title="View in article">12.</a></div><div>Cantell, K. ∙ Hirvonen, S.</div><div><strong>Preparation and assay of Sendai virus</strong></div><div><em>Methods Enzymol.</em> 1981; <strong>78</strong>:299-301</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/0076-6879(81)78131-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6173605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2F0076-6879%2881%2978131-x&amp;pmid=6173605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Carmeliet, P.</div><div><strong>Angiogenesis in life, disease and medicine</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:932-936</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04478" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744479430" target="_blank" aria-label="Open Scopus in new tab">Scopus (2777)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04478&amp;pmid=16355210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14-1" title="View in article">14.</a></div><div>Casie Chetty, S. ∙ Rost, M.S. ∙ Enriquez, J.R. ...</div><div><strong>Vegf signaling promotes vascular endothelial differentiation by modulating etv2 expression</strong></div><div><em>Dev. Biol.</em> 2017; <strong>424</strong>:147-161</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2017.03.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85014796623" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28279709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2017.03.005&amp;pmid=28279709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>CDC, NIH</div><div><strong>Biosafety in Microbiological and Biomedical Laboratories</strong></div><div>United States Department of Health and Human Services, 2020</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=CDCNIHBiosafety+in+Microbiological+and+Biomedical+Laboratories6th+edition2020United+States+Department+of+Health+and+Human+Services" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Chen, E.Y. ∙ Tan, C.M. ∙ Kou, Y. ...</div><div><strong>Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84876100615" target="_blank" aria-label="Open Scopus in new tab">Scopus (2789)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23586463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-128&amp;pmid=23586463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17" title="View in article">17.</a></div><div>Childs, K. ∙ Stock, N. ∙ Ross, C. ...</div><div><strong>mda-5, but not RIG-I, is a common target for paramyxovirus V proteins</strong></div><div><em>Virology.</em> 2007; <strong>359</strong>:190-200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2006.09.023" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33846863325" target="_blank" aria-label="Open Scopus in new tab">Scopus (250)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17049367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2006.09.023&amp;pmid=17049367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Chong, D.C. ∙ Koo, Y. ∙ Xu, K. ...</div><div><strong>Stepwise arteriovenous fate acquisition during mammalian vasculogenesis</strong></div><div><em>Dev. Dyn.</em> 2011; <strong>240</strong>:2153-2165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.22706" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-80051685993" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21793101/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.22706&amp;pmid=21793101" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19" title="View in article">19.</a></div><div>Chua, K.B. ∙ Bellini, W.J. ∙ Rota, P.A. ...</div><div><strong>Nipah virus: a recently emergent deadly paramyxovirus</strong></div><div><em>Science.</em> 2000; <strong>288</strong>:1432-1435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.288.5470.1432" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0034717054" target="_blank" aria-label="Open Scopus in new tab">Scopus (905)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10827955/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.288.5470.1432&amp;pmid=10827955" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Ciancanelli, M.J. ∙ Volchkova, V.A. ∙ Shaw, M.L. ...</div><div><strong>Nipah virus sequesters inactive STAT1 in the nucleus via a P gene-encoded mechanism</strong></div><div><em>J.&nbsp;Virol.</em> 2009; <strong>83</strong>:7828-7841</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02610-08" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-67749142255" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19515782/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02610-08&amp;pmid=19515782" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Ciau-Uitz, A. ∙ Pinheiro, P. ∙ Kirmizitas, A. ...</div><div><strong>VEGFA-dependent and -independent pathways synergise to drive Scl expression and initiate programming of the blood stem cell lineage in Xenopus</strong></div><div><em>Development.</em> 2013; <strong>140</strong>:2632-2642</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.090829" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84878286239" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23637333/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.090829&amp;pmid=23637333" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">22.</a></div><div>Corada, M. ∙ Orsenigo, F. ∙ Morini, M.F. ...</div><div><strong>Sox17 is indispensable for acquisition and maintenance of arterial identity</strong></div><div><em>Nat. Commun.</em> 2013; <strong>4</strong>:2609</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms3609" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84886678124" target="_blank" aria-label="Open Scopus in new tab">Scopus (157)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24153254/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms3609&amp;pmid=24153254" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23-1" title="View in article">23.</a></div><div>Coultas, L. ∙ Chawengsaksophak, K. ∙ Rossant, J.</div><div><strong>Endothelial cells and VEGF in vascular development</strong></div><div><em>Nature.</em> 2005; <strong>438</strong>:937-945</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature04479" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-30744432619" target="_blank" aria-label="Open Scopus in new tab">Scopus (784)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16355211/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature04479&amp;pmid=16355211" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">24.</a></div><div>Cradick, T.J. ∙ Qiu, P. ∙ Lee, C.M. ...</div><div><strong>Cosmid: a web-based tool for identifying and validating CRISPR/Cas off-target sites</strong></div><div><em>Mol. Ther. Nucleic Acids.</em> 2014; <strong>3</strong>:e214</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1038%2Fmtna.2014.64&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1038%2Fmtna.2014.64&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_24_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84921598229" target="_blank" aria-label="Open Scopus in new tab">Scopus (223)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25462530/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmtna.2014.64&amp;pmid=25462530" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">25.</a></div><div>Davis, R.P. ∙ Ng, E.S. ∙ Costa, M. ...</div><div><strong>Targeting a GFP reporter gene to the MIXL1 locus of human embryonic stem cells identifies human primitive streak-like cells and enables isolation of primitive hematopoietic precursors</strong></div><div><em>Blood.</em> 2008; <strong>111</strong>:1876-1884</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2007-06-093609" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-39349097787" target="_blank" aria-label="Open Scopus in new tab">Scopus (189)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18032708/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2007-06-093609&amp;pmid=18032708" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26-1" title="View in article">26.</a></div><div>de Chaumont, F. ∙ Dallongeville, S. ∙ Chenouard, N. ...</div><div><strong>Icy: an open BioImage informatics platform for extended reproducible research</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:690-696</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2075" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84863191618" target="_blank" aria-label="Open Scopus in new tab">Scopus (828)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743774/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2075&amp;pmid=22743774" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27-1" title="View in article">27.</a></div><div>De Val, S. ∙ Black, B.L.</div><div><strong>Transcriptional control of endothelial cell development</strong></div><div><em>Dev. Cell.</em> 2009; <strong>16</strong>:180-195</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2009.01.014&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2009.01.014&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-59649110012" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19217421/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2009.01.014&amp;pmid=19217421" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">28.</a></div><div>DeBuysscher, B.L. ∙ de Wit, E. ∙ Munster, V.J. ...</div><div><strong>Comparison of the pathogenicity of Nipah virus isolates from Bangladesh and Malaysia in the Syrian hamster</strong></div><div><em>PLoS Negl. Trop. Dis.</em> 2013; <strong>7</strong>:e2024</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pntd.0002024" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873501371" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23342177/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pntd.0002024&amp;pmid=23342177" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29-1" title="View in article">29.</a></div><div>Ditadi, A. ∙ Sturgeon, C.M. ∙ Tober, J. ...</div><div><strong>Human definitive haemogenic endothelium and arterial vascular endothelium represent distinct lineages</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:580-591</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3161" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928938612" target="_blank" aria-label="Open Scopus in new tab">Scopus (170)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25915127/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3161&amp;pmid=25915127" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30" title="View in article">30.</a></div><div>Drubin, D.G. ∙ Hyman, A.A.</div><div><strong>Stem cells: the new “model organism”</strong></div><div><em>Mol. Biol. Cell.</em> 2017; <strong>28</strong>:1409-1411</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1091/mbc.E17-03-0183" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85020120205" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28559439/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1091%2Fmbc.E17-03-0183&amp;pmid=28559439" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">31.</a></div><div>Duarte, A. ∙ Hirashima, M. ∙ Benedito, R. ...</div><div><strong>Dosage-sensitive requirement for mouse Dll4 in artery development</strong></div><div><em>Genes Dev.</em> 2004; <strong>18</strong>:2474-2478</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1239004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-5444261424" target="_blank" aria-label="Open Scopus in new tab">Scopus (445)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15466159/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1239004&amp;pmid=15466159" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">32.</a></div><div>Ellis, E.L. ∙ Delbrück, M.</div><div><strong>The growth of bacteriophage</strong></div><div><em>J.&nbsp;Gen. Physiol.</em> 1939; <strong>22</strong>:365-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1085/jgp.22.3.365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0001489791" target="_blank" aria-label="Open Scopus in new tab">Scopus (354)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19873108/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1085%2Fjgp.22.3.365&amp;pmid=19873108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33" title="View in article">33.</a></div><div>Elvert, M. ∙ Sauerhering, L. ∙ Maisner, A.</div><div><strong>Cytokine induction in Nipah virus-infected primary human and porcine bronchial epithelial cells</strong></div><div><em>J.&nbsp;Infect. Dis.</em> 2020; <strong>221</strong>:S395-S400</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/infdis/jiz455" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85084693241" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31665348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Finfdis%2Fjiz455&amp;pmid=31665348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34-1" title="View in article">34.</a></div><div>Erbar, S. ∙ Diederich, S. ∙ Maisner, A.</div><div><strong>Selective receptor expression restricts Nipah virus infection of endothelial cells</strong></div><div><em>Virol. J.</em> 2008; <strong>5</strong>:142</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1743-422X-5-142" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-58149168980" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19036148/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1743-422X-5-142&amp;pmid=19036148" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">35.</a></div><div>Fang, J. ∙ Hirschi, K.</div><div><strong>Molecular regulation of arteriovenous endothelial cell specification</strong></div><div><em>F1000Res.</em> 2019; <strong>8</strong>:1208</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.12688/f1000research.16701.1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85071923734" target="_blank" aria-label="Open Scopus in new tab">Scopus (19)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.12688%2Ff1000research.16701.1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36-1" title="View in article">36.</a></div><div>Fish, J.E. ∙ Wythe, J.D.</div><div><strong>The molecular regulation of arteriovenous specification and maintenance</strong></div><div><em>Dev. Dyn.</em> 2015; <strong>244</strong>:391-409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/dvdy.24252" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924032549" target="_blank" aria-label="Open Scopus in new tab">Scopus (94)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25641373/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fdvdy.24252&amp;pmid=25641373" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">37.</a></div><div>Fowler, J.L. ∙ Ang, L.T. ∙ Loh, K.M.</div><div><strong>A critical look: challenges in differentiating human pluripotent stem cells into desired cell types and organoids</strong></div><div><em>Wiley Interdiscip. Rev.: Dev. Biol.</em> 2019; <strong>113</strong></div><div>891–823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/wdev.368" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85075149381" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fwdev.368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">38.</a></div><div>Gage, B.K. ∙ Liu, J.C. ∙ Innes, B.T. ...</div><div><strong>Generation of functional liver sinusoidal endothelial cells from human pluripotent stem-cell-derived venous angioblasts</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:254-269.e9</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.06.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85088364652" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32640183/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.06.007&amp;pmid=32640183" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">39.</a></div><div>Gale, N.W. ∙ Dominguez, M.G. ∙ Noguera, I. ...</div><div><strong>Haploinsufficiency of delta-like 4 ligand results in embryonic lethality due to major defects in arterial and vascular development</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:15949-15954</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0407290101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-8644290828" target="_blank" aria-label="Open Scopus in new tab">Scopus (488)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15520367/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0407290101&amp;pmid=15520367" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">40.</a></div><div>Geisbert, T.W. ∙ Daddario-DiCaprio, K.M. ∙ Hickey, A.C. ...</div><div><strong>Development of an acute and highly pathogenic nonhuman primate model of Nipah virus infection</strong></div><div><em>PLoS One.</em> 2010; <strong>5</strong>:e10690</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0010690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77956283436" target="_blank" aria-label="Open Scopus in new tab">Scopus (111)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20502528/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0010690&amp;pmid=20502528" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41-1" title="View in article">41.</a></div><div>Goldstein, A.S. ∙ Drake, J.M. ∙ Burnes, D.L. ...</div><div><strong>Purification and direct transformation of epithelial progenitor cells from primary human prostate</strong></div><div><em>Nat. Protoc.</em> 2011; <strong>6</strong>:656-667</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2011.317" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79955530295" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21527922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2011.317&amp;pmid=21527922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">42.</a></div><div>Gómez Román, R. ∙ Tornieporth, N. ∙ Cherian, N.G. ...</div><div><strong>Medical countermeasures against henipaviruses: a review and public health perspective</strong></div><div><em>Lancet Infect. Dis.</em> 2022; <strong>22</strong>:e13-e27</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2821%2900400-X&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2821%2900400-X&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85121695384" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34735799/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2821%2900400-X&amp;pmid=34735799" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43-1" title="View in article">43.</a></div><div>Hafemeister, C. ∙ Satija, R.</div><div><strong>Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression</strong></div><div><em>Genome Biol.</em> 2019; <strong>20</strong>:296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-019-1874-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85076966458" target="_blank" aria-label="Open Scopus in new tab">Scopus (742)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-019-1874-1&amp;pmid=31870423" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44-1" title="View in article">44.</a></div><div>Halaidych, O.V. ∙ Freund, C. ∙ van den Hil, F. ...</div><div><strong>Inflammatory responses and barrier function of endothelial cells derived from human induced pluripotent stem cells</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1642-1656</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045106821" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29657098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.012&amp;pmid=29657098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45-1" title="View in article">45.</a></div><div>Harcourt, B.H. ∙ Tamin, A. ∙ Ksiazek, T.G. ...</div><div><strong>Molecular characterization of Nipah virus, a newly emergent paramyxovirus</strong></div><div><em>Virology.</em> 2000; <strong>271</strong>:334-349</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1006/viro.2000.0340" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0034608231" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10860887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1006%2Fviro.2000.0340&amp;pmid=10860887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46" title="View in article">46.</a></div><div>Harschnitz, O. ∙ Studer, L.</div><div><strong>Human stem cell models to study host-virus interactions in the central nervous system</strong></div><div><em>Nat. Rev. Immunol.</em> 2021; <strong>21</strong>:441-453</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-020-00474-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85098797496" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33398129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-020-00474-y&amp;pmid=33398129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">47.</a></div><div>Hong, C.C. ∙ Peterson, Q.P. ∙ Hong, J.Y. ...</div><div><strong>Artery/vein specification is governed by opposing phosphatidylinositol-3 kinase and MAP kinase/ERK signaling</strong></div><div><em>Curr. Biol.</em> 2006; <strong>16</strong>:1366-1372</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cub.2006.05.046&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cub.2006.05.046&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33745646792" target="_blank" aria-label="Open Scopus in new tab">Scopus (162)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16824925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cub.2006.05.046&amp;pmid=16824925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">48.</a></div><div>Iwano, S. ∙ Sugiyama, M. ∙ Hama, H. ...</div><div><strong>Single-cell bioluminescence imaging of deep tissue in freely moving animals</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:935-939</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaq1067" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85042300288" target="_blank" aria-label="Open Scopus in new tab">Scopus (190)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29472486/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaq1067&amp;pmid=29472486" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">49.</a></div><div>Kälin, R.E. ∙ Kretz, M.P. ∙ Meyer, A.M. ...</div><div><strong>Paracrine and autocrine mechanisms of apelin signaling govern embryonic and tumor angiogenesis</strong></div><div><em>Dev. Biol.</em> 2007; <strong>305</strong>:599-614</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ydbio.2007.03.004" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34247591454" target="_blank" aria-label="Open Scopus in new tab">Scopus (154)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17412318/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ydbio.2007.03.004&amp;pmid=17412318" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50-1" title="View in article">50.</a></div><div>Kalucka, J. ∙ de Rooij, L.P.M.H. ∙ Goveia, J. ...</div><div><strong>Single-cell transcriptome atlas of murine endothelial cells</strong></div><div><em>Cell.</em> 2020; <strong>180</strong>:764-779.e20</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2020.01.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2020.01.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85079395140" target="_blank" aria-label="Open Scopus in new tab">Scopus (317)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32059779/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2020.01.015&amp;pmid=32059779" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">51.</a></div><div>Kidoya, H. ∙ Naito, H. ∙ Muramatsu, F. ...</div><div><strong>APJ regulates parallel alignment of arteries and veins in the skin</strong></div><div><em>Dev. Cell.</em> 2015; <strong>33</strong>:247-259</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2015.02.024&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.devcel.2015.02.024&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84928817344" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25920569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.devcel.2015.02.024&amp;pmid=25920569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">52.</a></div><div>Krebs, L.T. ∙ Shutter, J.R. ∙ Tanigaki, K. ...</div><div><strong>Haploinsufficient lethality and formation of arteriovenous malformations in Notch pathway mutants</strong></div><div><em>Genes Dev.</em> 2004; <strong>18</strong>:2469-2473</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gad.1239204" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-5444242184" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15466160/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.1239204&amp;pmid=15466160" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">53.</a></div><div>Krueger, F.</div><div><strong>Trim galore</strong></div><div><a id="intref0145" href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a><div>Date: 2012</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=F.KruegerTrim+galorehttps%3A%2F%2Fwww.bioinformatics.babraham.ac.uk%2Fprojects%2Ftrim_galore%2F2012" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">54.</a></div><div>Kuleshov, M.V. ∙ Jones, M.R. ∙ Rouillard, A.D. ...</div><div><strong>Enrichr: a comprehensive gene set enrichment analysis web server 2016 update</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W90-W97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw377" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84987663170" target="_blank" aria-label="Open Scopus in new tab">Scopus (3602)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27141961/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw377&amp;pmid=27141961" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">55.</a></div><div>Kurian, L. ∙ Sancho-Martinez, I. ∙ Nivet, E. ...</div><div><strong>Conversion of human fibroblasts to angioblast-like progenitor cells</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:77-83</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2255" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84871977315" target="_blank" aria-label="Open Scopus in new tab">Scopus (123)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23202434/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2255&amp;pmid=23202434" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#fig4" title="View in article">56.</a></div><div>Kurokawa, Y.K. ∙ Yin, R.T. ∙ Shang, M.R. ...</div><div><strong>Human induced pluripotent stem cell-derived endothelial cells for three-dimensional microphysiological systems</strong></div><div><em>Tissue Eng. Part C Methods.</em> 2017; <strong>23</strong>:474-484</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1089/ten.TEC.2017.0133" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85027454424" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28622076/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1089%2Ften.TEC.2017.0133&amp;pmid=28622076" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">57.</a></div><div>Labun, K. ∙ Montague, T.G. ∙ Krause, M. ...</div><div><strong>CHOPCHOP v3: expanding the CRISPR web toolbox beyond genome editing</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:W171-W174</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkz365" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85069235112" target="_blank" aria-label="Open Scopus in new tab">Scopus (403)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31106371/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkz365&amp;pmid=31106371" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">58.</a></div><div>Law, C.W. ∙ Chen, Y. ∙ Shi, W. ...</div><div><strong>voom: precision weights unlock linear model analysis tools for RNA-seq read counts</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:R29</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2014-15-2-r29" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84896735766" target="_blank" aria-label="Open Scopus in new tab">Scopus (2673)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24485249/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2014-15-2-r29&amp;pmid=24485249" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">59.</a></div><div>Lawson, N.D. ∙ Scheer, N. ∙ Pham, V.N. ...</div><div><strong>Notch signaling is required for arterial-venous differentiation during embryonic vascular development</strong></div><div><em>Development.</em> 2001; <strong>128</strong>:3675-3683</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.128.19.3675" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11585794/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.128.19.3675&amp;pmid=11585794" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60-1" title="View in article">60.</a></div><div>Lawson, N.D. ∙ Vogel, A.M. ∙ Weinstein, B.M.</div><div><strong>sonic hedgehog and vascular endothelial growth factor act upstream of the Notch pathway during arterial endothelial differentiation</strong></div><div><em>Dev. Cell.</em> 2002; <strong>3</strong>:127-136</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1534-5807%2802%2900198-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1534-5807%2802%2900198-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036070186" target="_blank" aria-label="Open Scopus in new tab">Scopus (650)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12110173/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1534-5807%2802%2900198-3&amp;pmid=12110173" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61-1" title="View in article">61.</a></div><div>Lian, X. ∙ Bao, X. ∙ Al-Ahmad, A. ...</div><div><strong>Efficient differentiation of human pluripotent stem cells to endothelial progenitors via small-molecule activation of WNT signaling</strong></div><div><em>Stem Cell Rep.</em> 2014; <strong>3</strong>:804-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2014.09.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84922668186" target="_blank" aria-label="Open Scopus in new tab">Scopus (225)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25418725/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2014.09.005&amp;pmid=25418725" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">62.</a></div><div>Lin, F.-J. ∙ Tsai, M.-J. ∙ Tsai, S.Y.</div><div><strong>Artery and vein formation: a tug of war between different forces</strong></div><div><em>EMBO Rep.</em> 2007; <strong>8</strong>:920-924</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/sj.embor.7401076" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34848924388" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17906673/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fsj.embor.7401076&amp;pmid=17906673" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">63.</a></div><div>Lin, Y. ∙ Gil, C.-H. ∙ Yoder, M.C.</div><div><strong>Differentiation, evaluation, and application of human induced pluripotent stem cell-derived endothelial cells</strong></div><div><em>Arterioscler. Thromb. Vasc. Biol.</em> 2017; <strong>37</strong>:2014-2025</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/ATVBAHA.117.309962" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85032977936" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29025705/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FATVBAHA.117.309962&amp;pmid=29025705" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">64.</a></div><div>Lo, M.K. ∙ Miller, D. ∙ Aljofan, M. ...</div><div><strong>Characterization of the antiviral and inflammatory responses against Nipah virus in endothelial cells and neurons</strong></div><div><em>Virology.</em> 2010; <strong>404</strong>:78-88</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.virol.2010.05.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-77953714922" target="_blank" aria-label="Open Scopus in new tab">Scopus (52)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20552729/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.virol.2010.05.005&amp;pmid=20552729" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65-1" title="View in article">65.</a></div><div>Loh, K.M. ∙ Ang, L.T. ∙ Zhang, J. ...</div><div><strong>Efficient endoderm induction from human pluripotent stem cells by logically directing signals controlling lineage bifurcations</strong></div><div><em>Cell Stem Cell.</em> 2014; <strong>14</strong>:237-252</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2013.12.007&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84893794554" target="_blank" aria-label="Open Scopus in new tab">Scopus (224)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24412311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2013.12.007&amp;pmid=24412311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66-1" title="View in article">66.</a></div><div>Loh, K.M. ∙ Chen, A. ∙ Koh, P.W. ...</div><div><strong>Mapping the pairwise choices leading from pluripotency to human bone, heart, and other mesoderm cell types</strong></div><div><em>Cell.</em> 2016; <strong>166</strong>:451-467</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2016.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84978924170" target="_blank" aria-label="Open Scopus in new tab">Scopus (215)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27419872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2016.06.011&amp;pmid=27419872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67-1" title="View in article">67.</a></div><div>López, C.B. ∙ Yount, J.S. ∙ Hermesh, T. ...</div><div><strong>Sendai virus infection induces efficient adaptive immunity independently of type I interferons</strong></div><div><em>J.&nbsp;Virol.</em> 2006; <strong>80</strong>:4538-4545</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.80.9.4538-4545.2006" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646184859" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16611914/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.80.9.4538-4545.2006&amp;pmid=16611914" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68-1" title="View in article">68.</a></div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (29293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">69.</a></div><div>Lu, X. ∙ le Noble, F. ∙ Yuan, L. ...</div><div><strong>The netrin receptor UNC5B mediates guidance events controlling morphogenesis of the vascular system</strong></div><div><em>Nature.</em> 2004; <strong>432</strong>:179-186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03080" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_69_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-8844281731" target="_blank" aria-label="Open Scopus in new tab">Scopus (426)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15510105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03080&amp;pmid=15510105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70-1" title="View in article">70.</a></div><div>Mahalingam, S. ∙ Herrero, L.J. ∙ Playford, E.G. ...</div><div><strong>Hendra virus: an emerging paramyxovirus in Australia</strong></div><div><em>Lancet Infect. Dis.</em> 2012; <strong>12</strong>:799-807</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2812%2970158-5&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS1473-3099%2812%2970158-5&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84866619198" target="_blank" aria-label="Open Scopus in new tab">Scopus (82)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22921953/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1473-3099%2812%2970158-5&amp;pmid=22921953" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#figs3" title="View in article">71.</a></div><div>Martin, R.M. ∙ Ikeda, K. ∙ Cromer, M.K. ...</div><div><strong>Highly efficient and marker-free genome editing of human pluripotent stem cells by CRISPR-Cas9 RNP and AAV6 donor-mediated homologous recombination</strong></div><div><em>Cell Stem Cell.</em> 2019; <strong>24</strong>:821-828.e5</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2019.04.001&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85064323495" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31051134/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2019.04.001&amp;pmid=31051134" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72-1" title="View in article">72.</a></div><div>Masaki, H. ∙ Kato-Itoh, M. ∙ Takahashi, Y. ...</div><div><strong>Inhibition of apoptosis overcomes stage-related compatibility barriers to chimera formation in mouse embryos</strong></div><div><em>Cell Stem Cell.</em> 2016; <strong>19</strong>:587-592</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2016.10.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2016.10.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84995691206" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27814480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2016.10.013&amp;pmid=27814480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73-1" title="View in article">73.</a></div><div>Mathieu, C. ∙ Guillaume, V. ∙ Sabine, A. ...</div><div><strong>Lethal Nipah virus infection induces rapid overexpression of CXCL10</strong></div><div><em>PLoS One.</em> 2012; <strong>7</strong>:e32157</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0032157" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84857681004" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22393386/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0032157&amp;pmid=22393386" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74" title="View in article">74.</a></div><div>McCracken, I.R. ∙ Taylor, R.S. ∙ Kok, F.O. ...</div><div><strong>Transcriptional dynamics of pluripotent stem cell-derived endothelial cell differentiation revealed by single-cell RNA sequencing</strong></div><div><em>Eur. Heart J.</em> 2019; <strong>385</strong>:9963117</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/eurheartj/ehz351" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85080882411" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Feurheartj%2Fehz351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">75.</a></div><div>Mehand, M.S. ∙ Al-Shorbaji, F. ∙ Millett, P. ...</div><div><strong>The WHO R&amp;D Blueprint: 2018 review of emerging infectious diseases requiring urgent research and development efforts</strong></div><div><em>Antiviral Res.</em> 2018; <strong>159</strong>:63-67</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.antiviral.2018.09.009" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85054087266" target="_blank" aria-label="Open Scopus in new tab">Scopus (139)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30261226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.antiviral.2018.09.009&amp;pmid=30261226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76" title="View in article">76.</a></div><div>Metzger, T. ∙ Gache, V. ∙ Xu, M. ...</div><div><strong>MAP and kinesin-dependent nuclear positioning is required for skeletal muscle function</strong></div><div><em>Nature.</em> 2012; <strong>484</strong>:120-124</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10914" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84859465143" target="_blank" aria-label="Open Scopus in new tab">Scopus (171)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22425998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10914&amp;pmid=22425998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77-1" title="View in article">77.</a></div><div>Monaghan, P. ∙ Green, D. ∙ Pallister, J. ...</div><div><strong>Detailed morphological characterisation of Hendra virus infection of different cell types using super-resolution and conventional imaging</strong></div><div><em>Virol. J.</em> 2014; <strong>11</strong>:200</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s12985-014-0200-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84936855081" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25428656/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs12985-014-0200-5&amp;pmid=25428656" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">78.</a></div><div>Murray, K. ∙ Selleck, P. ∙ Hooper, P. ...</div><div><strong>A morbillivirus that caused fatal disease in horses and humans</strong></div><div><em>Science.</em> 1995; <strong>268</strong>:94-97</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7701348" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0028943779" target="_blank" aria-label="Open Scopus in new tab">Scopus (565)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7701348/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7701348&amp;pmid=7701348" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">79.</a></div><div>Nakatsu, M.N. ∙ Davis, J. ∙ Hughes, C.C.W.</div><div><strong>Optimized fibrin gel bead assay for the study of angiogenesis</strong></div><div><em>J.&nbsp;Vis. Exp.</em> 2007; <strong>3</strong>:186</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18978935/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=18978935" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#figs6" title="View in article">80.</a></div><div>Negrete, O.A. ∙ Chu, D. ∙ Aguilar, H.C. ...</div><div><strong>Single amino acid changes in the Nipah and Hendra virus attachment glycoproteins distinguish ephrinB2 from ephrinB3 usage</strong></div><div><em>J.&nbsp;Virol.</em> 2007; <strong>81</strong>:10804-10814</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.00999-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34648830167" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17652392/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.00999-07&amp;pmid=17652392" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">81.</a></div><div>Negrete, O.A. ∙ Levroney, E.L. ∙ Aguilar, H.C. ...</div><div><strong>EphrinB2 is the entry receptor for Nipah virus, an emergent deadly paramyxovirus</strong></div><div><em>Nature.</em> 2005; <strong>436</strong>:401-405</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature03838" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-22944445501" target="_blank" aria-label="Open Scopus in new tab">Scopus (347)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16007075/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature03838&amp;pmid=16007075" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">82.</a></div><div>Ng, A.H.M. ∙ Khoshakhlagh, P. ∙ Rojo Arias, J.E. ...</div><div><strong>A comprehensive library of human transcription factors for cell fate engineering</strong></div><div><em>Nat. Biotechnol.</em> 2020; <strong>51</strong>:987</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41587-020-0742-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096952020" target="_blank" aria-label="Open Scopus in new tab">Scopus (44)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41587-020-0742-6" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83" title="View in article">83.</a></div><div>Nguyen, J. ∙ Lin, Y.-Y. ∙ Gerecht, S.</div><div><strong>The next generation of endothelial differentiation: tissue-specific ECs</strong></div><div><em>Cell Stem Cell.</em> 2021; <strong>28</strong>:1188-1204</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2021.05.002&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2021.05.002&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85108876397" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/34081899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2021.05.002&amp;pmid=34081899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84" title="View in article">84.</a></div><div>Nguyen, M.T.X. ∙ Okina, E. ∙ Chai, X. ...</div><div><strong>Differentiation of human embryonic stem cells to endothelial progenitor cells on laminins in defined and xeno-free systems</strong></div><div><em>Stem Cell Rep.</em> 2016; <strong>7</strong>:802-816</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2016.08.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2016.08.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84992170620" target="_blank" aria-label="Open Scopus in new tab">Scopus (31)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27693424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2016.08.017&amp;pmid=27693424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">85.</a></div><div>Nishimura, K. ∙ Ohtaka, M. ∙ Takada, H. ...</div><div><strong>Simple and effective generation of transgene-free induced pluripotent stem cells using an auto-erasable Sendai virus vector responding to microRNA-302</strong></div><div><em>Stem Cell Res.</em> 2017; <strong>23</strong>:13-19</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.scr.2017.06.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021304442" target="_blank" aria-label="Open Scopus in new tab">Scopus (40)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28666145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.scr.2017.06.011&amp;pmid=28666145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86-1" title="View in article">86.</a></div><div>Nishimura, K. ∙ Sano, M. ∙ Ohtaka, M. ...</div><div><strong>Development of defective and persistent Sendai virus vector: a unique gene delivery/expression system ideal for cell reprogramming</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2011; <strong>286</strong>:4760-4771</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1074%2Fjbc.M110.183780&amp;cf=fulltext&amp;site=jbc-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1074%2Fjbc.M110.183780&amp;cf=pdf&amp;site=jbc-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79953021243" target="_blank" aria-label="Open Scopus in new tab">Scopus (251)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21138846/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M110.183780&amp;pmid=21138846" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87-1" title="View in article">87.</a></div><div>Olmer, R. ∙ Engels, L. ∙ Usman, A. ...</div><div><strong>Differentiation of human pluripotent stem cells into functional endothelial cells in scalable suspension culture</strong></div><div><em>Stem Cell Rep.</em> 2018; <strong>10</strong>:1657-1672</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stemcr.2018.03.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85045438454" target="_blank" aria-label="Open Scopus in new tab">Scopus (46)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29681541/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stemcr.2018.03.017&amp;pmid=29681541" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88-1" title="View in article">88.</a></div><div>Ouyang, J.F. ∙ Kamaraj, U.S. ∙ Cao, E.Y. ...</div><div><strong>ShinyCell: simple and sharable visualisation of single-cell gene expression data</strong></div><div><em>Bioinformatics.</em> 2021; btab209</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btab209" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85134939084" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33774659/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtab209&amp;pmid=33774659" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">89.</a></div><div>Paik, D.T. ∙ Tian, L. ∙ Lee, J. ...</div><div><strong>Large-scale single-cell RNA-seq reveals molecular signatures of heterogeneous populations of human induced pluripotent stem cell-derived endothelial cells</strong></div><div><em>Circ. Res.</em> 2018; <strong>123</strong>:443-450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1161/CIRCRESAHA.118.312913" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85055602915" target="_blank" aria-label="Open Scopus in new tab">Scopus (74)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29986945/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1161%2FCIRCRESAHA.118.312913&amp;pmid=29986945" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#fig6" title="View in article">90.</a></div><div>Palomares, K. ∙ Vigant, F. ∙ Van Handel, B. ...</div><div><strong>Nipah virus envelope-pseudotyped lentiviruses efficiently target ephrinB2-positive stem cell populations <i>in&nbsp;vitro</i> and bypass the liver sink when administered <i>in&nbsp;vivo</i></strong></div><div><em>J.&nbsp;Virol.</em> 2013; <strong>87</strong>:2094-2108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02032-12" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84873834467" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23192877/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02032-12&amp;pmid=23192877" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91-1" title="View in article">91.</a></div><div>Park, M.A. ∙ Kumar, A. ∙ Jung, H.S. ...</div><div><strong>Activation of the arterial program drives development of definitive hemogenic endothelium with lymphoid potential</strong></div><div><em>Cell Rep.</em> 2018; <strong>23</strong>:2467-2481</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.celrep.2018.04.092&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85047204081" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29791856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.celrep.2018.04.092&amp;pmid=29791856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">92.</a></div><div>Patsch, C. ∙ Challet-Meylan, L. ∙ Thoma, E.C. ...</div><div><strong>Generation of vascular endothelial and smooth muscle cells from human pluripotent stem cells</strong></div><div><em>Nat. Cell Biol.</em> 2015; <strong>17</strong>:994-1003</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb3205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84938743523" target="_blank" aria-label="Open Scopus in new tab">Scopus (306)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26214132/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb3205&amp;pmid=26214132" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93" title="View in article">93.</a></div><div>Poduri, A. ∙ Chang, A.H. ∙ Raftrey, B. ...</div><div><strong>Endothelial cells respond to the direction of mechanical stimuli through SMAD signaling to regulate coronary artery size</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:3241-3252</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.150904" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85029803848" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28760815/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.150904&amp;pmid=28760815" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94-1" title="View in article">94.</a></div><div>Potente, M. ∙ Mäkinen, T.</div><div><strong>Vascular heterogeneity and specialization in development and disease</strong></div><div><em>Nat. Rev. Mol. Cell Biol.</em> 2017; <strong>18</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nrm.2017.36" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85026911538" target="_blank" aria-label="Open Scopus in new tab">Scopus (257)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28537573/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnrm.2017.36&amp;pmid=28537573" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">95.</a></div><div>Prescott, J. ∙ DeBuysscher, B.L. ∙ Feldmann, F. ...</div><div><strong>Single-dose live-attenuated vesicular stomatitis virus-based vaccine protects African green monkeys from Nipah virus disease</strong></div><div><em>Vaccine.</em> 2015; <strong>33</strong>:2823-2829</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.vaccine.2015.03.089" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84929297020" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25865472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.vaccine.2015.03.089&amp;pmid=25865472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">96.</a></div><div>Rafii, S. ∙ Butler, J.M. ∙ Ding, B.-S.</div><div><strong>Angiocrine functions of organ-specific endothelial cells</strong></div><div><em>Nature.</em> 2016; <strong>529</strong>:316-325</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17040" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84955286381" target="_blank" aria-label="Open Scopus in new tab">Scopus (485)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26791722/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17040&amp;pmid=26791722" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">97.</a></div><div>Red-Horse, K. ∙ Siekmann, A.F.</div><div><strong>Veins and arteries Build Hierarchical Branching Patterns Differently: Bottom-Up versus Top-Down</strong></div><div><em>BioEssays.</em> 2019; <strong>41</strong>:e1800198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/bies.201800198" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062109763" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30805984/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fbies.201800198&amp;pmid=30805984" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">98.</a></div><div>Reich, M. ∙ Liefeld, T. ∙ Gould, J. ...</div><div><strong>GenePattern 2.0</strong></div><div><em>Nat. Genet.</em> 2006; <strong>38</strong>:500-501</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng0506-500" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-33646361583" target="_blank" aria-label="Open Scopus in new tab">Scopus (1493)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16642009/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng0506-500&amp;pmid=16642009" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99-1" title="View in article">99.</a></div><div>Ritchie, M.E. ∙ Phipson, B. ∙ Wu, D. ...</div><div><strong>limma powers differential expression analyses for RNA-sequencing and microarray studies</strong></div><div><em>Nucleic Acids Res.</em> 2015; <strong>43</strong>:e47</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84926507971" target="_blank" aria-label="Open Scopus in new tab">Scopus (13963)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv007&amp;pmid=25605792" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">100.</a></div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (19590)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">101.</a></div><div>Rodriguez, J.J. ∙ Parisien, J.-P. ∙ Horvath, C.M.</div><div><strong>Nipah virus V protein evades alpha and gamma interferons by preventing STAT1 and STAT2 activation and nuclear accumulation</strong></div><div><em>J.&nbsp;Virol.</em> 2002; <strong>76</strong>:11476-11483</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/jvi.76.22.11476-11483.2002" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036827851" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12388709/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2Fjvi.76.22.11476-11483.2002&amp;pmid=12388709" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102-1" title="View in article">102.</a></div><div>Rosa, S. ∙ Praça, C. ∙ Pitrez, P.R. ...</div><div><strong>Functional characterization of iPSC-derived arterial- and venous-like endothelial cells</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:3826</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-40417-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85062643820" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30846769/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-40417-9&amp;pmid=30846769" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">103.</a></div><div>Sakamoto, Y. ∙ Hara, K. ∙ Kanai-Azuma, M. ...</div><div><strong>Redundant roles of Sox17 and Sox18 in early cardiovascular development of mouse embryos</strong></div><div><em>Biochem. Biophys. Res. Commun.</em> 2007; <strong>360</strong>:539-544</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.bbrc.2007.06.093" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-34447265321" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17610846/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.bbrc.2007.06.093&amp;pmid=17610846" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">104.</a></div><div>Schindelin, J. ∙ Arganda-Carreras, I. ∙ Frise, E. ...</div><div><strong>Fiji: an open-source platform for biological-image analysis</strong></div><div><em>Nat. Methods.</em> 2012; <strong>9</strong>:676-682</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2019" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84862520770" target="_blank" aria-label="Open Scopus in new tab">Scopus (28202)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2019&amp;pmid=22743772" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105-1" title="View in article">105.</a></div><div>Schountz, T. ∙ Campbell, C. ∙ Wagner, K. ...</div><div><strong>Differential innate immune responses elicited by Nipah virus and cedar virus correlate with disparate in&nbsp;vivo pathogenesis in hamsters</strong></div><div><em>Viruses.</em> 2019; <strong>11</strong></div><div>291–211</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3390/v11030291" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85063712957" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30909389/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3390%2Fv11030291&amp;pmid=30909389" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106-1" title="View in article">106.</a></div><div>Shalaby, F. ∙ Rossant, J. ∙ Yamaguchi, T.P. ...</div><div><strong>Failure of blood-island formation and vasculogenesis in Flk-1-deficient mice</strong></div><div><em>Nature.</em> 1995; <strong>376</strong>:62-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/376062a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0029006696" target="_blank" aria-label="Open Scopus in new tab">Scopus (3332)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7596435/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F376062a0&amp;pmid=7596435" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">107.</a></div><div>Shaw, A.E. ∙ Hughes, J. ∙ Gu, Q. ...</div><div><strong>Fundamental properties of the mammalian innate immune system revealed by multispecies comparison of type I interferon responses</strong></div><div><em>PLoS Biol.</em> 2017; <strong>15</strong>:e2004086</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pbio.2004086" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85040002649" target="_blank" aria-label="Open Scopus in new tab">Scopus (137)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29253856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pbio.2004086&amp;pmid=29253856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">108.</a></div><div>Shin, J.W. ∙ Huggenberger, R. ∙ Detmar, M.</div><div><strong>Transcriptional profiling of VEGF-A and VEGF-C target genes in lymphatic endothelium reveals endothelial-specific molecule-1 as a novel mediator of lymphangiogenesis</strong></div><div><em>Blood.</em> 2008; <strong>112</strong>:2318-2326</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-05-156331" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-55249083593" target="_blank" aria-label="Open Scopus in new tab">Scopus (110)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18614759/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-05-156331&amp;pmid=18614759" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">109.</a></div><div>Simoneau, C.R. ∙ Ott, M.</div><div><strong>Modeling multi-organ infection by SARS-CoV-2&nbsp;using stem cell technology</strong></div><div><em>Cell Stem Cell.</em> 2020; <strong>27</strong>:859-868</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.11.012&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.stem.2020.11.012&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096979593" target="_blank" aria-label="Open Scopus in new tab">Scopus (17)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33275899/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.stem.2020.11.012&amp;pmid=33275899" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110-1" title="View in article">110.</a></div><div>Sriram, G. ∙ Tan, J.Y. ∙ Islam, I. ...</div><div><strong>Efficient differentiation of human embryonic stem cells to arterial and venous endothelial cells under feeder- and serum- free conditions</strong></div><div><em>Stem Cell Res. Ther.</em> 2015; <strong>6</strong>:261</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13287-015-0260-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84951950036" target="_blank" aria-label="Open Scopus in new tab">Scopus (73)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26718617/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13287-015-0260-5&amp;pmid=26718617" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111-1" title="View in article">111.</a></div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive integration of single-cell data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902.e21</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (3719)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">112.</a></div><div>Tai-Nagara, I. ∙ Yoshikawa, Y. ∙ Numata, N. ...</div><div><strong>Placental labyrinth formation in mice requires endothelial FLRT2/UNC5B signaling</strong></div><div><em>Development.</em> 2017; <strong>144</strong>:2392-2401</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.149757" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85021744683" target="_blank" aria-label="Open Scopus in new tab">Scopus (16)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28576770/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.149757&amp;pmid=28576770" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">113.</a></div><div>Takashima, Y. ∙ Guo, G. ∙ Loos, R. ...</div><div><strong>Resetting transcription factor control circuitry toward ground-state pluripotency in human</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:1254-1269</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2014.08.029&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2Fj.cell.2014.08.029&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84908074395" target="_blank" aria-label="Open Scopus in new tab">Scopus (586)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25215486/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.08.029&amp;pmid=25215486" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114-1" title="View in article">114.</a></div><div>Tinevez, J.Y. ∙ Perry, N. ∙ Schindelin, J. ...</div><div><strong>TrackMate: an open and extensible platform for single-particle tracking</strong></div><div><em>Methods.</em> 2017; <strong>115</strong>:80-90</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.ymeth.2016.09.016" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85002002648" target="_blank" aria-label="Open Scopus in new tab">Scopus (1289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27713081/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ymeth.2016.09.016&amp;pmid=27713081" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#fig6" title="View in article">115.</a></div><div>Travaglini, K.J. ∙ Nabhan, A.N. ∙ Penland, L. ...</div><div><strong>A molecular cell atlas of the human lung from single-cell RNA sequencing</strong></div><div><em>Nature.</em> 2020; <strong>587</strong>:619-625</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2922-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85096205400" target="_blank" aria-label="Open Scopus in new tab">Scopus (286)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33208946/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2922-4&amp;pmid=33208946" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">116.</a></div><div>Virtue, E.R. ∙ Marsh, G.A. ∙ Wang, L.F.</div><div><strong>Interferon signaling remains functional during Henipavirus infection of human cell lines</strong></div><div><em>J.&nbsp;Virol.</em> 2011; <strong>85</strong>:4031-4034</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02412-10" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-79952836894" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21289115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02412-10&amp;pmid=21289115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117-1" title="View in article">117.</a></div><div>Wang, H.U. ∙ Chen, Z.F. ∙ Anderson, D.J.</div><div><strong>Molecular distinction and angiogenic interaction between embryonic arteries and veins revealed by ephrin-B2 and its receptor Eph-B4</strong></div><div><em>Cell.</em> 1998; <strong>93</strong>:741-753</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0092-8674%2800%2981436-1&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0092-8674%2800%2981436-1&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0032577446" target="_blank" aria-label="Open Scopus in new tab">Scopus (1355)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9630219/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2981436-1&amp;pmid=9630219" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118" title="View in article">118.</a></div><div>Wang, K. ∙ Lin, R.-Z. ∙ Hong, X. ...</div><div><strong>Robust differentiation of human pluripotent stem cells into endothelial cells via temporal modulation of ETV2 with modified mRNA</strong></div><div><em>Sci. Adv.</em> 2020; <strong>6</strong>:eaba7606</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciadv.aba7606" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85090074756" target="_blank" aria-label="Open Scopus in new tab">Scopus (29)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32832668/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciadv.aba7606&amp;pmid=32832668" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119" title="View in article">119.</a></div><div>Wickham, H.</div><div><strong>ggplot2: Elegant Graphics for Data Analysis</strong></div><div>Springer-Verlag New York, 2016</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/978-3-319-24277-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2F978-3-319-24277-4" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120-1" title="View in article">120.</a></div><div>Wickham, H. ∙ Averick, M. ∙ Bryan, J. ...</div><div><strong>Welcome to the Tidyverse</strong></div><div><em>J.&nbsp;Open Source Software.</em> 2019; <strong>4</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.21105/joss.01686" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.21105%2Fjoss.01686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121" title="View in article">121.</a></div><div>Wickham, H. ∙ François, R. ∙ Henry, L. ...</div><div><strong>Dplyr: a grammar of data manipulation</strong></div><div><a id="intref0150" href="https://dplyr.tidyverse.org" target="_blank">https://dplyr.tidyverse.org</a><div>Date: 2022</div></div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=H.WickhamR.Fran%C3%A7oisL.HenryK.M%C3%BCllerDplyr%3A+a+grammar+of+data+manipulationhttps%3A%2F%2Fdplyr.tidyverse.org2022" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122-1" title="View in article">122.</a></div><div>Wong, K.T. ∙ Shieh, W.-J. ∙ Kumar, S. ...</div><div><strong>Nipah virus infection: pathology and pathogenesis of an emerging paramyxoviral zoonosis</strong></div><div><em>Am. J. Pathol.</em> 2002; <strong>161</strong>:2153-2167</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=fulltext&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=10.1016%2FS0002-9440%2810%2964493-8&amp;cf=pdf&amp;site=ajpa-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-0036899143" target="_blank" aria-label="Open Scopus in new tab">Scopus (283)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12466131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0002-9440%2810%2964493-8&amp;pmid=12466131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123" title="View in article">123.</a></div><div>Xu, C. ∙ Hasan, S.S. ∙ Schmidt, I. ...</div><div><strong>Arteries are formed by vein-derived endothelial tip cells</strong></div><div><em>Nat. Commun.</em> 2014; <strong>5</strong>:5758</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms6758" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-84923281901" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25502622/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms6758&amp;pmid=25502622" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124-1" title="View in article">124.</a></div><div>Yoshida, A. ∙ Kawabata, R. ∙ Honda, T. ...</div><div><strong>A single amino acid substitution within the paramyxovirus Sendai virus nucleoprotein is a critical determinant for production of interferon-beta-inducing copyback-type defective interfering genomes</strong></div><div><em>J.&nbsp;Virol.</em> 2018; <strong>92</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/JVI.02094-17" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_124_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85042056136" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29237838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FJVI.02094-17&amp;pmid=29237838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#fig2" title="View in article">125.</a></div><div>Zeng, Y. ∙ He, J. ∙ Bai, Z. ...</div><div><strong>Tracing the first hematopoietic stem cell generation in human embryo by single-cell RNA sequencing</strong></div><div><em>Cell Res.</em> 2019; <strong>29</strong>:881-894</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41422-019-0228-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85073937796" target="_blank" aria-label="Open Scopus in new tab">Scopus (77)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31501518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41422-019-0228-6&amp;pmid=31501518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref126-1" title="View in article">126.</a></div><div>Zhang, J. ∙ Chu, L.-F. ∙ Hou, Z. ...</div><div><strong>Functional characterization of human pluripotent stem cell-derived arterial endothelial cells. Proceedings of the National Academy of Sciences of The United States of America</strong></div><div>2017 201702295</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1702295114" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1702295114" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127-1" title="View in article">127.</a></div><div>Zheng, G.X.Y. ∙ Terry, J.M. ∙ Belgrader, P. ...</div><div><strong>Massively parallel digital transcriptional profiling of single cells</strong></div><div><em>Nat. Commun.</em> 2017; <strong>8</strong>:14049</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms14049" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2022.05.024&amp;key=2-s2.0-85009446777" target="_blank" aria-label="Open Scopus in new tab">Scopus (2096)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms14049&amp;pmid=28091601" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none linkback-ad widget-" id="684c6365-474d-4ce7-932a-ddffa4f64d88">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <h2>CHORUS Manuscript</h2>
<p style="margin-bottom:5px;">
<a target="_blank" href="/action/clickThrough?id=2959&amp;url=https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fam%2Fpii%2FS0092867422006559&amp;loc=%2Fcell%2Ffulltext%2FS0092-8674%2822%2900655-9&amp;pubId=61755115&amp;placeholderId=1346&amp;productId=1822"> View Open Manuscript</a>
</p>
        </div></div>
                </div>
                </div>

        
</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (14)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/0c3bf538-6bb5-4679-9adf-111a8cd23e18/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/27de4192-729f-4a7a-b26b-93cd5c836e63/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/25adc4d5-999c-46ba-ac01-7130bccb279a/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/a43940a6-a2bf-463d-9c7e-3cfa3c0e3174/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/530d0beb-8db3-4875-9f69-62c69ba21bae/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/ab133bf4-3032-4aa7-bf28-b6d583029a45/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/c5c3041e-527e-45b1-adac-43ce5104e33d/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/6073da7d-a0d6-4ff8-9d5f-54e574ab7d94/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/28e70eed-8a6b-4131-92d8-e1aba70b0818/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/6b8eeb59-3eb7-4599-bf09-7ca290d65dbe/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure&amp;nbsp;6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/34ea8c2e-f174-4bf0-8b3f-a0c6d4ef1131/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure&amp;nbsp;S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/f4480560-709f-47dc-b8c4-b915c47bb0bc/main.assets/figs6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig7" aria-label="Go to Figure&amp;nbsp;7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/83e7e01e-5cfb-49fa-9c23-79e62a887dd0/main.assets/gr7.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs7" aria-label="Go to Figure&amp;nbsp;S7"><img src="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/asset/2d96a5bd-32b2-43c6-beb0-590e0a2017f6/main.assets/figs7.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Barticle%3Aarticle%3Apii%5C%3AS0092867422006559%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Bjournal%3Ajournal%3Acell%3Bissue%3Aissue%3Apii%5C%3AS0092867421X00153%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false"><li aria-label="loading articles" role="alert" class="js--loading plum-x--spinner hidden"></li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (5)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/attachment/ec5a3b6a-88ad-4e91-ae5a-f4edd54a110b/mmc1.csv" download="mmc1.csv"><span id="mmc1-link">File (393.87 KB)</span></a><div class="core-description">Table&nbsp;S1. Gene expression differences between hPSC-derived versus human fetal artery ECs, as assayed by single-cell RNA-seq, related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="file"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/attachment/93db3841-7932-4e65-8757-f9972dd15dd8/mmc2.csv" download="mmc2.csv"><span id="mmc2-link">File (264.36 KB)</span></a><div class="core-description">Table&nbsp;S2. Gene expression differences between hPSC-derived versus human fetal vein ECs, as assayed by single-cell RNA-seq, related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/attachment/4d8deffb-bdac-42c4-bd17-d471aafbd92c/mmc3.xls" download="mmc3.xls"><span id="mmc3-link">Spreadsheet (570.00 KB)</span></a><div class="core-description">Table&nbsp;S3. Differentially expressed genes between hPSC-derived artery versus vein ECs, as assayed by bulk-population RNA-seq, related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/attachment/8d3d4556-c2e9-4d01-bfeb-b6e42a79e361/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (7.41 MB)</span></a><div class="core-description">Table&nbsp;S4. Bulk-population RNA-seq of hPSC-derived artery and vein cells infected by Nipah, Hendra, or Sendai viruses, related to Figure&nbsp;7</div></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2022.05.024/attachment/82e7f0de-0c6f-4470-839d-d9c041ee052b/mmc5.pdf" download="mmc5.pdf"><span id="mmc5-link">PDF (47.39 KB)</span></a><div class="core-description">Table&nbsp;S5. Quantitative PCR primers, related to Figures&nbsp;1, 2, 3, and 4</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867422006559" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2022.05.024","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.05.024/asset/0c3bf538-6bb5-4679-9adf-111a8cd23e18/main.assets/gr1.jpg" height="594" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/3266107b-c15e-431f-ae09-8f5b2cf99df9/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.05.024/asset/27de4192-729f-4a7a-b26b-93cd5c836e63/main.assets/figs1.jpg" height="695" width="761" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/ee1b1747-8ee0-4020-9ef8-25bdf9fa5749/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.05.024/asset/25adc4d5-999c-46ba-ac01-7130bccb279a/main.assets/gr2.jpg" height="806" width="761" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/a5612dfa-af24-47c8-a5aa-466a4320749e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.05.024/asset/a43940a6-a2bf-463d-9c7e-3cfa3c0e3174/main.assets/figs2.jpg" height="933" width="761" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/b14db292-7841-4eb3-8e9e-010d9c8bc2b2/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.05.024/asset/530d0beb-8db3-4875-9f69-62c69ba21bae/main.assets/gr3.jpg" height="737" width="761" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/288eafb6-9078-44a0-9105-b40334c3d073/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.05.024/asset/ab133bf4-3032-4aa7-bf28-b6d583029a45/main.assets/figs3.jpg" height="607" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/9637e7c7-72b1-4101-a761-eeb2eacd6c77/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.05.024/asset/c5c3041e-527e-45b1-adac-43ce5104e33d/main.assets/gr4.jpg" height="806" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/0e3117d0-c10e-448b-b226-efd470bc3500/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.05.024/asset/6073da7d-a0d6-4ff8-9d5f-54e574ab7d94/main.assets/figs4.jpg" height="601" width="761" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/c48620ab-231d-4deb-b2d0-3f11ca2fa51d/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.05.024/asset/28e70eed-8a6b-4131-92d8-e1aba70b0818/main.assets/figs5.jpg" height="921" width="761" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/76a57c7a-2df4-43a2-9139-947a3ec29e37/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.05.024/asset/6b8eeb59-3eb7-4599-bf09-7ca290d65dbe/main.assets/gr5.jpg" height="918" width="761" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/fbe3738a-e7cf-4f5b-ba92-9ecbdd78841b/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.05.024/asset/34ea8c2e-f174-4bf0-8b3f-a0c6d4ef1131/main.assets/gr6.jpg" height="637" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/d7bffde2-e27f-48c5-b310-0ea143a27b5a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.05.024/asset/f4480560-709f-47dc-b8c4-b915c47bb0bc/main.assets/figs6.jpg" height="923" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/2459c63f-8b08-4669-b356-392fe230f9d0/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.05.024/asset/83e7e01e-5cfb-49fa-9c23-79e62a887dd0/main.assets/gr7.jpg" height="810" width="761" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/0fe0863c-6e9c-4eae-a785-3097fc3697e2/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.05.024/asset/2d96a5bd-32b2-43c6-beb0-590e0a2017f6/main.assets/figs7.jpg" height="710" width="761" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/adaa1a89-15fa-4daa-ab97-6d3838b66dae/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle fw-loaded" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2022.05.024/asset/0c3bf538-6bb5-4679-9adf-111a8cd23e18/main.assets/gr1.jpg" height="594" width="761" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/3266107b-c15e-431f-ae09-8f5b2cf99df9/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2022.05.024/asset/27de4192-729f-4a7a-b26b-93cd5c836e63/main.assets/figs1.jpg" height="695" width="761" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/ee1b1747-8ee0-4020-9ef8-25bdf9fa5749/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2022.05.024/asset/25adc4d5-999c-46ba-ac01-7130bccb279a/main.assets/gr2.jpg" height="806" width="761" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/a5612dfa-af24-47c8-a5aa-466a4320749e/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2022.05.024/asset/a43940a6-a2bf-463d-9c7e-3cfa3c0e3174/main.assets/figs2.jpg" height="933" width="761" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/b14db292-7841-4eb3-8e9e-010d9c8bc2b2/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2022.05.024/asset/530d0beb-8db3-4875-9f69-62c69ba21bae/main.assets/gr3.jpg" height="737" width="761" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/288eafb6-9078-44a0-9105-b40334c3d073/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2022.05.024/asset/ab133bf4-3032-4aa7-bf28-b6d583029a45/main.assets/figs3.jpg" height="607" width="761" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/9637e7c7-72b1-4101-a761-eeb2eacd6c77/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2022.05.024/asset/c5c3041e-527e-45b1-adac-43ce5104e33d/main.assets/gr4.jpg" height="806" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/0e3117d0-c10e-448b-b226-efd470bc3500/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2022.05.024/asset/6073da7d-a0d6-4ff8-9d5f-54e574ab7d94/main.assets/figs4.jpg" height="601" width="761" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/c48620ab-231d-4deb-b2d0-3f11ca2fa51d/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2022.05.024/asset/28e70eed-8a6b-4131-92d8-e1aba70b0818/main.assets/figs5.jpg" height="921" width="761" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/76a57c7a-2df4-43a2-9139-947a3ec29e37/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2022.05.024/asset/6b8eeb59-3eb7-4599-bf09-7ca290d65dbe/main.assets/gr5.jpg" height="918" width="761" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/fbe3738a-e7cf-4f5b-ba92-9ecbdd78841b/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2022.05.024/asset/34ea8c2e-f174-4bf0-8b3f-a0c6d4ef1131/main.assets/gr6.jpg" height="637" width="761" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/d7bffde2-e27f-48c5-b310-0ea143a27b5a/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2022.05.024/asset/f4480560-709f-47dc-b8c4-b915c47bb0bc/main.assets/figs6.jpg" height="923" width="761" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/2459c63f-8b08-4669-b356-392fe230f9d0/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;7</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr7" src="/cms/10.1016/j.cell.2022.05.024/asset/83e7e01e-5cfb-49fa-9c23-79e62a887dd0/main.assets/gr7.jpg" height="810" width="761" aria-labelledby="fig7-title fig7-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/0fe0863c-6e9c-4eae-a785-3097fc3697e2/main.assets/gr7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S7</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs7" src="/cms/10.1016/j.cell.2022.05.024/asset/2d96a5bd-32b2-43c6-beb0-590e0a2017f6/main.assets/figs7.jpg" height="710" width="761" aria-labelledby="figs7-title figs7-content" data-viewer-src="/cms/10.1016/j.cell.2022.05.024/asset/adaa1a89-15fa-4daa-ab97-6d3838b66dae/main.assets/figs7_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(22)00655-9';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749593952721","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"185","issueNumber":"14","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867422006559","coverDate":"202207","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Generating human artery and vein cells from pluripotent stem cells highlights the arterial tropism of Nipah and Hendra viruses","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dc3fb4bee8428c',t:'MTc0OTU5Mzk1Mi4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dc3fb4bee8428c&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2022.05.024&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>