<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="addf1033-998d-40a3-b00c-6f0192237a0a" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867420314537;issue:issue:pii\:S0092867419X00276;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Human T-bet Governs Innate and Innate-like Adaptive IFN-γ Immunity against Mycobacteria">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(20)31453-7">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/asset/d136c7af-cd66-417c-b48b-0a321f215a4f/fx1.jpg">
    


        <meta property="og:description" content="Human T-bet is involved in the regulation of distinct leukocyte subsets, when compared
with its mouse counterpart, with its deficiency underlying severe mycobacterial disease">
    











        <meta name="twitter:title" content="Human T-bet Governs Innate and Innate-like Adaptive IFN-γ Immunity against Mycobacteria">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/asset/d136c7af-cd66-417c-b48b-0a321f215a4f/fx1.jpg">
    


        <meta name="twitter:description" content="Human T-bet is involved in the regulation of distinct leukocyte subsets, when compared
with its mouse counterpart, with its deficiency underlying severe mycobacterial disease">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(20)31453-7">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Human T-bet Governs Innate and Innate-like Adaptive IFN-γ Immunity against Mycobacteria: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Human T-bet Governs Innate and Innate-like Adaptive IFN-γ Immunity against Mycobacteria"> <meta name="citation_date" content="2020/12/23"> <meta name="citation_online_date" content="2020/12/08"> <meta name="citation_volume" content="183"> <meta name="citation_issue" content="7"> <meta name="citation_firstpage" content="1826"> <meta name="citation_lastpage" content="1847.e31"> <meta name="citation_pmid" content="33296702"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867420314537"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867420314537"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867420314537"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2020.10.046"> <meta name="citation_pmid" content="33296702"> <meta name="citation_author" content="Rui Yang"><meta name="citation_author_email" content="ryang@rockefeller.edu"><meta name="citation_author_orcid" content="0000-0003-4427-2158"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Federico Mele"><meta name="citation_author_institution" content="Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Lisa Worley"><meta name="citation_author_institution" content="Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia"><meta name="citation_author_institution" content="St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia"><meta name="citation_author" content="David Langlais"><meta name="citation_author_orcid" content="0000-0003-4429-0110"><meta name="citation_author_institution" content="Department of Human Genetics, Department of Microbiology and Immunology, McGill University, Montreal, QC H3A 0G1, Canada"><meta name="citation_author_institution" content="McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada"><meta name="citation_author" content="Jérémie Rosain"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Ibithal Benhsaien"><meta name="citation_author_institution" content="Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco"><meta name="citation_author_institution" content="Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco"><meta name="citation_author" content="Houda Elarabi"><meta name="citation_author_institution" content="Pediatrics Department, Hassan II Hospital, 80030 Dakhla, Morocco"><meta name="citation_author" content="Carys A. Croft"><meta name="citation_author_institution" content="Innate Immunity Unit, Institut Pasteur, 75724 Paris, France"><meta name="citation_author_institution" content="INSERM U1223, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, 75006 Paris, France"><meta name="citation_author" content="Jean-Marc Doisne"><meta name="citation_author_orcid" content="0000-0001-6714-1308"><meta name="citation_author_institution" content="Innate Immunity Unit, Institut Pasteur, 75724 Paris, France"><meta name="citation_author_institution" content="INSERM U1223, 75015 Paris, France"><meta name="citation_author" content="Peng Zhang"><meta name="citation_author_orcid" content="0000-0002-6129-567X"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Marc Weisshaar"><meta name="citation_author_orcid" content="0000-0002-7769-1730"><meta name="citation_author_institution" content="Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland"><meta name="citation_author" content="David Jarrossay"><meta name="citation_author_institution" content="Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland"><meta name="citation_author" content="Daniela Latorre"><meta name="citation_author_orcid" content="0000-0003-1031-1534"><meta name="citation_author_institution" content="Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland"><meta name="citation_author" content="Yichao Shen"><meta name="citation_author_orcid" content="0000-0002-5257-6808"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Jing Han"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Masato Ogishi"><meta name="citation_author_orcid" content="0000-0003-2421-7389"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Conor Gruber"><meta name="citation_author_institution" content="Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"><meta name="citation_author_institution" content="The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"><meta name="citation_author" content="Janet Markle"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Fatima Al Ali"><meta name="citation_author_orcid" content="0000-0001-6451-7357"><meta name="citation_author_institution" content="Research Branch, Sidra Medicine, Doha, PO 26999, Qatar"><meta name="citation_author" content="Mahbuba Rahman"><meta name="citation_author_orcid" content="0000-0003-4586-4067"><meta name="citation_author_institution" content="Research Branch, Sidra Medicine, Doha, PO 26999, Qatar"><meta name="citation_author" content="Taushif Khan"><meta name="citation_author_orcid" content="0000-0002-7917-8965"><meta name="citation_author_institution" content="Research Branch, Sidra Medicine, Doha, PO 26999, Qatar"><meta name="citation_author" content="Yoann Seeleuthner"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Gaspard Kerner"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Lucas T. Husquin"><meta name="citation_author_institution" content="Human Evolutionary Genetics Unit, CNRS UMR2000, Institut Pasteur, 75015 Paris, France"><meta name="citation_author" content="Julia L. Maclsaac"><meta name="citation_author_institution" content="BC Children’s Hospital Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, BC V5Z 4H4, Canada"><meta name="citation_author" content="Mohamed Jeljeli"><meta name="citation_author_orcid" content="0000-0001-9561-8847"><meta name="citation_author_institution" content="University of Paris, 75006 Paris, France"><meta name="citation_author_institution" content="Immunology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France"><meta name="citation_author" content="Abderrahmane Errami"><meta name="citation_author_orcid" content="0000-0002-2916-5007"><meta name="citation_author_institution" content="Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco"><meta name="citation_author" content="Fatima Ailal"><meta name="citation_author_institution" content="Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco"><meta name="citation_author_institution" content="Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco"><meta name="citation_author" content="Michael S. Kobor"><meta name="citation_author_institution" content="BC Children’s Hospital Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, BC V5Z 4H4, Canada"><meta name="citation_author" content="Carmen Oleaga-Quintas"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Manon Roynard"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Mathieu Bourgey"><meta name="citation_author_orcid" content="0000-0002-8432-834X"><meta name="citation_author_institution" content="McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada"><meta name="citation_author_institution" content="Canadian Centre for Computational Genomics, Montreal, QC H3A 0G1, Canada"><meta name="citation_author" content="Jamila El Baghdadi"><meta name="citation_author_institution" content="Genetics Unit, Military Hospital Mohamed V, 10100 Rabat, Morocco"><meta name="citation_author" content="Stéphanie Boisson-Dupuis"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Anne Puel"><meta name="citation_author_orcid" content="0000-0003-2603-0323"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Fréderic Batteux"><meta name="citation_author_institution" content="University of Paris, 75006 Paris, France"><meta name="citation_author_institution" content="Immunology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France"><meta name="citation_author" content="Flore Rozenberg"><meta name="citation_author_institution" content="University of Paris, 75006 Paris, France"><meta name="citation_author_institution" content="Virology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France"><meta name="citation_author" content="Nico Marr"><meta name="citation_author_orcid" content="0000-0002-1927-7072"><meta name="citation_author_institution" content="Research Branch, Sidra Medicine, Doha, PO 26999, Qatar"><meta name="citation_author_institution" content="College of Health and Life Sciences, Hamad Bin Khalifa University, Doha, PO 34110, Qatar"><meta name="citation_author" content="Qiang Pan-Hammarström"><meta name="citation_author_institution" content="Department of Biosciences and Nutrition, Karolinska Institutet, 17177 Stockholm, Sweden"><meta name="citation_author" content="Dusan Bogunovic"><meta name="citation_author_institution" content="Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"><meta name="citation_author_institution" content="Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"><meta name="citation_author_institution" content="The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA"><meta name="citation_author" content="Lluis Quintana-Murci"><meta name="citation_author_institution" content="Human Evolutionary Genetics Unit, CNRS UMR2000, Institut Pasteur, 75015 Paris, France"><meta name="citation_author_institution" content="Chair of Human Genomics and Evolution, Collège de France, 75005 Paris, France"><meta name="citation_author" content="Thomas Carroll"><meta name="citation_author_institution" content="Bioinformatics Resource Center, The Rockefeller University, New York, NY 10065, USA"><meta name="citation_author" content="Cindy S. Ma"><meta name="citation_author_orcid" content="0000-0001-5387-8413"><meta name="citation_author_institution" content="Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia"><meta name="citation_author_institution" content="St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia"><meta name="citation_author" content="Laurent Abel"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author" content="Aziz Bousfiha"><meta name="citation_author_orcid" content="0000-0002-5011-9873"><meta name="citation_author_institution" content="Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco"><meta name="citation_author_institution" content="Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco"><meta name="citation_author" content="James P. Di Santo"><meta name="citation_author_institution" content="Innate Immunity Unit, Institut Pasteur, 75724 Paris, France"><meta name="citation_author_institution" content="INSERM U1223, 75015 Paris, France"><meta name="citation_author" content="Laurie H. Glimcher"><meta name="citation_author_institution" content="Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA"><meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA"><meta name="citation_author_institution" content="Department of Immunology, Harvard Medical School, Boston, MA 02115, USA"><meta name="citation_author" content="Philippe Gros"><meta name="citation_author_orcid" content="0000-0002-0114-1978"><meta name="citation_author_institution" content="McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada"><meta name="citation_author_institution" content="Department of Biochemistry, McGill University, Montreal, QC H3G 1Y6, Canada"><meta name="citation_author" content="Stuart G. Tangye"><meta name="citation_author_orcid" content="0000-0002-5360-5180"><meta name="citation_author_institution" content="Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia"><meta name="citation_author_institution" content="St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia"><meta name="citation_author" content="Federica Sallusto"><meta name="citation_author_orcid" content="0000-0003-3750-2752"><meta name="citation_author_institution" content="Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland"><meta name="citation_author_institution" content="Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland"><meta name="citation_author" content="Jacinta Bustamante"><meta name="citation_author_orcid" content="0000-0002-3439-2482"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author_institution" content="Study Center for Primary Immunodeficiencies, Necker Children Hospital, AP-HP, 75015 Paris, France"><meta name="citation_author" content="Jean-Laurent Casanova"><meta name="citation_author_email" content="casanova@rockefeller.edu"><meta name="citation_author_institution" content="St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA"><meta name="citation_author_institution" content="Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France"><meta name="citation_author_institution" content="University of Paris, Imagine Institute, 75015 Paris, France"><meta name="citation_author_institution" content="Pediatric Hematology-Immunology Unit, Necker Hospital for Sick Children, AP-HP, 75015 Paris, France"><meta name="citation_author_institution" content="Howard Hughes Medical Institute, New York, NY, USA"><meta name="citation_keywords" content="T-bet; Mendelian susceptibility to mycobacterial disease; mycobacterium; IFN-γ; innate lymphocyte; innate-like adaptive lymphocyte; inborn errors of immunity; immunodeficiency"><meta name="citation_reference" content="citation_title=Merlin--rapid analysis of dense genetic maps using sparse gene flow trees; citation_author=G.R. Abecasis; citation_author=S.S. Cherny; citation_author=W.O. Cookson; citation_author=L.R. Cardon; citation_journal_title=Nat. Genet.; citation_volume=30; citation_firstpage=97; citation_lastpage=101; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Surface phenotype and antigenic specificity of human interleukin 17-producing T helper memory cells; citation_author=E.V. Acosta-Rodriguez; citation_author=L. Rivino; citation_author=J. Geginat; citation_author=D. Jarrossay; citation_author=M. Gattorno; citation_author=A. Lanzavecchia; citation_author=F. Sallusto; citation_author=G. Napolitani; citation_journal_title=Nat. Immunol.; citation_volume=8; citation_firstpage=639; citation_lastpage=646; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update; citation_author=E. Afgan; citation_author=D. Baker; citation_author=M. van den Beek; citation_author=D. Blankenberg; citation_author=D. Bouvier; citation_author=M. Čech; citation_author=J. Chilton; citation_author=D. Clements; citation_author=N. Coraor; citation_author=C. Eberhard; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=W1; citation_firstpage=W3; citation_lastpage=W10; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Clinical, immunological, and molecular findings in five patients with Major histocompatibility complex class II deficiency from India; citation_author=J. Aluri; citation_author=M. Gupta; citation_author=A. Dalvi; citation_author=S. Mhatre; citation_author=M. Kulkarni; citation_author=G. Hule; citation_author=M. Desai; citation_author=N. Shah; citation_author=P. Taur; citation_author=R. Vedam; citation_author=M. Madkaikar; citation_journal_title=Front. Immunol.; citation_volume=9; citation_firstpage=188; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia; citation_author=A.D. Amir el; citation_author=K.L. Davis; citation_author=M.D. Tadmor; citation_author=E.F. Simonds; citation_author=J.H. Levine; citation_author=S.C. Bendall; citation_author=D.K. Shenfeld; citation_author=S. Krishnaswamy; citation_author=G.P. Nolan; citation_author=D. Pe’er; citation_journal_title=Nat. Biotechnol.; citation_volume=31; citation_firstpage=545; citation_lastpage=552; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=β2-Microglobulin deficiency causes a complex immunodeficiency of the innate and adaptive immune system; citation_author=Ö. Ardeniz; citation_author=S. Unger; citation_author=H. Onay; citation_author=S. Ammann; citation_author=C. Keck; citation_author=C. Cianga; citation_author=B. Gerçeker; citation_author=B. Martin; citation_author=I. Fuchs; citation_author=U. Salzer; citation_author=et al.; citation_journal_title=J.&nbsp;Allergy Clin. Immunol.; citation_volume=136; citation_firstpage=392; citation_lastpage=401; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A global reference for human genetic variation; citation_author=A. Auton; citation_author=L.D. Brooks; citation_author=R.M. Durbin; citation_author=E.P. Garrison; citation_author=H.M. Kang; citation_author=J.O. Korbel; citation_author=J.L. Marchini; citation_author=S. McCarthy; citation_author=G.A. McVean; citation_author=G.R. Abecasis; citation_author=1000 Genomes Project Consortium; citation_journal_title=Nature; citation_volume=526; citation_firstpage=68; citation_lastpage=74; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=T-bet deficiency facilitates airway colonization by Mycoplasma pulmonis in a murine model of asthma; citation_author=C.S. Bakshi; citation_author=M. Malik; citation_author=P.M. Carrico; citation_author=T.J. Sellati; citation_journal_title=J.&nbsp;Immunol.; citation_volume=177; citation_firstpage=1786; citation_lastpage=1795; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=T cell immunity. Functional heterogeneity of human memory CD4(+) T&nbsp;cell clones primed by pathogens or vaccines; citation_author=S. Becattini; citation_author=D. Latorre; citation_author=F. Mele; citation_author=M. Foglierini; citation_author=C. De Gregorio; citation_author=A. Cassotta; citation_author=B. Fernandez; citation_author=S. Kelderman; citation_author=T.N. Schumacher; citation_author=D. Corti; citation_author=et al.; citation_journal_title=Science; citation_volume=347; citation_firstpage=400; citation_lastpage=406; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Dimensionality reduction for visualizing single-cell data using UMAP; citation_author=E. Becht; citation_author=L. McInnes; citation_author=J. Healy; citation_author=C.-A. Dutertre; citation_author=I.W.H. Kwok; citation_author=L.G. Ng; citation_author=F. Ginhoux; citation_author=E.W. Newell; citation_journal_title=Nat. Biotechnol.; citation_volume=37; citation_firstpage=38; citation_lastpage=44; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Loss of T-bet, but not STAT1, prevents the development of experimental autoimmune encephalomyelitis; citation_author=E. Bettelli; citation_author=B. Sullivan; citation_author=S.J. Szabo; citation_author=R.A. Sobel; citation_author=L.H. Glimcher; citation_author=V.K. Kuchroo; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=200; citation_firstpage=79; citation_lastpage=87; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=CD56 negative NK cells: origin, function, and role in chronic viral disease; citation_author=N.K. Björkström; citation_author=H.-G. Ljunggren; citation_author=J.K. Sandberg; citation_journal_title=Trends Immunol.; citation_volume=31; citation_firstpage=401; citation_lastpage=406; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Mycobacterial disease and impaired IFN-gamma immunity in humans with inherited ISG15 deficiency; citation_author=D. Bogunovic; citation_author=M. Byun; citation_author=L.A. Durfee; citation_author=A. Abhyankar; citation_author=O. Sanal; citation_author=D. Mansouri; citation_author=S. Salem; citation_author=I. Radovanovic; citation_author=A.V. Grant; citation_author=P. Adimi; citation_author=et al.; citation_journal_title=Science; citation_volume=337; citation_firstpage=1684; citation_lastpage=1688; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Tuberculosis and impaired IL-23-dependent IFN-γuberculosis and impaired IL-23-dependent IFN-mmunity in humans wit; citation_author=S. Boisson-Dupuis; citation_author=N. Ramirez-Alejo; citation_author=Z. Li; citation_author=E. Patin; citation_author=G. Rao; citation_author=G. Kerner; citation_author=C.K. Lim; citation_author=D.N. Krementsov; citation_author=N. Hernandez; citation_author=C.S. Ma; citation_author=et al.; citation_journal_title=Sci. Immunol.; citation_volume=3; citation_firstpage=eaau8714; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position; citation_author=J.D. Buenrostro; citation_author=P.G. Giresi; citation_author=L.C. Zaba; citation_author=H.Y. Chang; citation_author=W.J. Greenleaf; citation_journal_title=Nat. Methods; citation_volume=10; citation_firstpage=1213; citation_lastpage=1218; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide; citation_author=J.D. Buenrostro; citation_author=B. Wu; citation_author=H.Y. Chang; citation_author=W.J. Greenleaf; citation_journal_title=Curr. Protoc. Mol. Biol.; citation_volume=109; citation_firstpage=21.29.1; citation_lastpage=21.29.9; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Mendelian susceptibility to mycobacterial disease: recent discoveries; citation_author=J. Bustamante; citation_journal_title=Hum. Genet.; citation_volume=139; citation_firstpage=993; citation_lastpage=1000; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=profileplyr: Visualization and annotation of read signal over genomic ranges with profileplyr; citation_author=T. Carroll; citation_author=D. Barrows; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Impact of artifact removal on ChIP quality metrics in ChIP-seq and ChIP-exo data; citation_author=T.S. Carroll; citation_author=Z. Liang; citation_author=R. Salama; citation_author=R. Stark; citation_author=I. de Santiago; citation_journal_title=Front. Genet.; citation_volume=5; citation_firstpage=75; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Human genetic basis of interindividual variability in the course of infection; citation_author=J.L. Casanova; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E7118; citation_lastpage=E7127; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Severe infectious diseases of childhood as monogenic inborn errors of immunity; citation_author=J.L. Casanova; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=E7128; citation_lastpage=E7137; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The human genetic determinism of life-threatening infectious diseases: genetic heterogeneity and physiological homogeneity?; citation_author=J.L. Casanova; citation_journal_title=Hum. Genet.; citation_volume=139; citation_firstpage=681; citation_lastpage=694; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Lethal Infectious Diseases as Inborn Errors of Immunity: Toward a Synthesis of the Germ and Genetic Theories.; citation_author=J.L. Casanova; citation_journal_title=Annu. Rev. Pathol.; citation_volume=16; citation_doi=10.1146/annurev-pathol-031920-101429; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=The human model: a genetic dissection of immunity to infection in natural conditions; citation_author=J.L. Casanova; citation_author=L. Abel; citation_journal_title=Nat. Rev. Immunol.; citation_volume=4; citation_firstpage=55; citation_lastpage=66; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Human genetics of infectious diseases: Unique insights into immunological redundancy; citation_author=J.L. Casanova; citation_author=L. Abel; citation_journal_title=Semin. Immunol.; citation_volume=36; citation_firstpage=1; citation_lastpage=12; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Immunological conditions of children with BCG disseminated infection; citation_author=J.L. Casanova; citation_author=E. Jouanguy; citation_author=S. Lamhamedi; citation_author=S. Blanche; citation_author=A. Fischer; citation_journal_title=Lancet; citation_volume=346; citation_firstpage=581; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Guidelines for genetic studies in single patients: lessons from primary immunodeficiencies; citation_author=J.-L. Casanova; citation_author=M.E. Conley; citation_author=S.J. Seligman; citation_author=L. Abel; citation_author=L.D. Notarangelo; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=211; citation_firstpage=2137; citation_lastpage=2149; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Recessive mutations in MCM4/PRKDC cause a novel syndrome involving a primary immunodeficiency and a disorder of DNA repair; citation_author=J.P. Casey; citation_author=M. Nobbs; citation_author=P. McGettigan; citation_author=S. Lynch; citation_author=S. Ennis; citation_journal_title=J.&nbsp;Med. Genet.; citation_volume=49; citation_firstpage=242; citation_lastpage=245; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Immune regulation of gammadelta T&nbsp;cell responses in mycobacterial infections; citation_author=Z.W. Chen; citation_journal_title=Clin. Immunol.; citation_volume=116; citation_firstpage=202; citation_lastpage=207; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Epigenetic and transcriptional programs lead to default IFN-γ production by gammadelta T&nbsp;cells; citation_author=L. Chen; citation_author=W. He; citation_author=S.T. Kim; citation_author=J. Tao; citation_author=Y. Gao; citation_author=H. Chi; citation_author=A.M. Intlekofer; citation_author=B. Harvey; citation_author=S.L. Reiner; citation_author=Z. Yin; citation_author=et al.; citation_journal_title=J.&nbsp;Immunol.; citation_volume=178; citation_firstpage=2730; citation_lastpage=2736; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=γδ T&nbsp;cells: first line of defense and beyond; citation_author=Y.H. Chien; citation_author=C. Meyer; citation_author=M. Bonneville; citation_journal_title=Annu. Rev. Immunol.; citation_volume=32; citation_firstpage=121; citation_lastpage=155; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3; citation_author=P. Cingolani; citation_author=A. Platts; citation_author=L. Wang; citation_author=M. Coon; citation_author=T. Nguyen; citation_author=L. Wang; citation_author=S.J. Land; citation_author=X. Lu; citation_author=D.M. Ruden; citation_journal_title=Fly (Austin); citation_volume=6; citation_firstpage=80; citation_lastpage=92; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues; citation_author=M.R. Corces; citation_author=A.E. Trevino; citation_author=E.G. Hamilton; citation_author=P.G. Greenside; citation_author=N.A. Sinnott-Armstrong; citation_author=S. Vesuna; citation_author=A.T. Satpathy; citation_author=A.J. Rubin; citation_author=K.S. Montine; citation_author=B. Wu; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=959; citation_lastpage=962; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Inherited GINS1 deficiency underlies growth retardation along with neutropenia and NK cell deficiency; citation_author=J. Cottineau; citation_author=M.C. Kottemann; citation_author=F.P. Lach; citation_author=Y.-H. Kang; citation_author=F. Vély; citation_author=E.K. Deenick; citation_author=T. Lazarov; citation_author=L. Gineau; citation_author=Y. Wang; citation_author=A. Farina; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=127; citation_firstpage=1991; citation_lastpage=2006; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Tissue-specific functions of invariant natural killer T&nbsp;cells; citation_author=C.M. Crosby; citation_author=M. Kronenberg; citation_journal_title=Nat. Rev. Immunol.; citation_volume=18; citation_firstpage=559; citation_lastpage=574; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=T-bet and Eomes instruct the development of two distinct natural killer cell lineages in the liver and in the bone marrow; citation_author=C. Daussy; citation_author=F. Faure; citation_author=K. Mayol; citation_author=S. Viel; citation_author=G. Gasteiger; citation_author=E. Charrier; citation_author=J. Bienvenu; citation_author=T. Henry; citation_author=E. Debien; citation_author=U.A. Hasan; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=211; citation_firstpage=563; citation_lastpage=577; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Unprecedented diversity of genotypic revertants in lymphocytes of a patient with Wiskott-Aldrich syndrome; citation_author=B.R. Davis; citation_author=M.J. Dicola; citation_author=N.L. Prokopishyn; citation_author=J.B. Rosenberg; citation_author=D. Moratto; citation_author=L.M. Muul; citation_author=F. Candotti; citation_author=R. Michael Blaese; citation_journal_title=Blood; citation_volume=111; citation_firstpage=5064; citation_lastpage=5067; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Homozygous human TAP peptide transporter mutation in HLA class I deficiency; citation_author=H. De La Salle; citation_author=D. Hanau; citation_author=D. Fricker; citation_author=A. Urlacher; citation_author=A. Kelly; citation_author=J. Salamero; citation_author=S.H. Powis; citation_author=L. Donato; citation_author=H. Bausinger; citation_author=M. Laforet; citation_author=et al.; citation_journal_title=Science; citation_volume=265; citation_firstpage=237; citation_lastpage=241; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Analysis of ChIP-seq Data in R/Bioconductor; citation_author=I. de Santiago; citation_author=T. Carroll; citation_firstpage=195; citation_lastpage=226; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data; citation_author=M.A. DePristo; citation_author=E. Banks; citation_author=R. Poplin; citation_author=K.V. Garimella; citation_author=J.R. Maguire; citation_author=C. Hartl; citation_author=A.A. Philippakis; citation_author=G. del Angel; citation_author=M.A. Rivas; citation_author=M. Hanna; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=43; citation_firstpage=491; citation_lastpage=498; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Major histocompatibility complex class II deficiency complicated by Mycobacterium avium complex in a boy of mixed ethnicity; citation_author=D. Dimitrova; citation_author=P.Y. Ong; citation_author=M.R.G. O’Gorman; citation_author=J.A. Church; citation_journal_title=J.&nbsp;Clin. Immunol.; citation_volume=34; citation_firstpage=677; citation_lastpage=680; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A. Dobin; citation_author=C.A. Davis; citation_author=F. Schlesinger; citation_author=J. Drenkow; citation_author=C. Zaleski; citation_author=S. Jha; citation_author=P. Batut; citation_author=M. Chaisson; citation_author=T.R. Gingeras; citation_journal_title=Bioinformatics; citation_volume=29; citation_firstpage=15; citation_lastpage=21; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=MIP-1alpha, MIP-1beta, RANTES, and ATAC/lymphotactin function together with IFN-gamma as type 1 cytokines; citation_author=B.G. Dorner; citation_author=A. Scheffold; citation_author=M.S. Rolph; citation_author=M.B. Huser; citation_author=S.H.E. Kaufmann; citation_author=A. Radbruch; citation_author=I.E.A. Flesch; citation_author=R.A. Kroczek; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=99; citation_firstpage=6181; citation_lastpage=6186; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Single-cell analysis of the murine chemokines MIP-1alpha, MIP-1beta, RANTES and ATAC/lymphotactin by flow cytometry; citation_author=B.G. Dorner; citation_author=S. Steinbach; citation_author=M.B. Hüser; citation_author=R.A. Kroczek; citation_author=A. Scheffold; citation_journal_title=J.&nbsp;Immunol. Methods; citation_volume=274; citation_firstpage=83; citation_lastpage=91; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Coordinate expression of cytokines and chemokines by NK cells during murine cytomegalovirus infection; citation_author=B.G. Dorner; citation_author=H.R.C. Smith; citation_author=A.R. French; citation_author=S. Kim; citation_author=J. Poursine-Laurent; citation_author=D.L. Beckman; citation_author=J.T. Pingel; citation_author=R.A. Kroczek; citation_author=W.M. Yokoyama; citation_journal_title=J.&nbsp;Immunol.; citation_volume=172; citation_firstpage=3119; citation_lastpage=3131; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Human interferon-gamma-mediated immunity is a genetically controlled continuous trait that determines the outcome of mycobacterial invasion; citation_author=S. Dupuis; citation_author=R. Döffinger; citation_author=C. Picard; citation_author=C. Fieschi; citation_author=F. Altare; citation_author=E. Jouanguy; citation_author=L. Abel; citation_author=J.L. Casanova; citation_journal_title=Immunol. Rev.; citation_volume=178; citation_firstpage=129; citation_lastpage=137; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Major histocompatibility complex class II deficiency: a clinical review; citation_author=R. Elhasid; citation_author=A. Etzioni; citation_journal_title=Blood Rev.; citation_volume=10; citation_firstpage=242; citation_lastpage=248; citation_publication_date=1996; "><meta name="citation_reference" content="citation_title=Identifying ChIP-seq enrichment using MACS; citation_author=J. Feng; citation_author=T. Liu; citation_author=B. Qin; citation_author=Y. Zhang; citation_author=X.S. Liu; citation_journal_title=Nat. Protoc.; citation_volume=7; citation_firstpage=1728; citation_lastpage=1740; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data; citation_author=G. Finak; citation_author=A. McDavid; citation_author=M. Yajima; citation_author=J. Deng; citation_author=V. Gersuk; citation_author=A.K. Shalek; citation_author=C.K. Slichter; citation_author=H.W. Miller; citation_author=M.J. McElrath; citation_author=M. Prlic; citation_author=et al.; citation_journal_title=Genome Biol.; citation_volume=16; citation_firstpage=278; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Development of spontaneous airway changes consistent with human asthma in mice lacking T-bet; citation_author=S. Finotto; citation_author=M.F. Neurath; citation_author=J.N. Glickman; citation_author=S. Qin; citation_author=H.A. Lehr; citation_author=F.H. Green; citation_author=K. Ackerman; citation_author=K. Haley; citation_author=P.R. Galle; citation_author=S.J. Szabo; citation_author=et al.; citation_journal_title=Science; citation_volume=295; citation_firstpage=336; citation_lastpage=338; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=GENCODE reference annotation for the human and mouse genomes; citation_author=A. Frankish; citation_author=M. Diekhans; citation_author=A.-M. Ferreira; citation_author=R. Johnson; citation_author=I. Jungreis; citation_author=J. Loveland; citation_author=J.M. Mudge; citation_author=C. Sisu; citation_author=J. Wright; citation_author=J. Armstrong; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=47; citation_number=D1; citation_firstpage=D766; citation_lastpage=D773; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Communicable ulcerative colitis induced by T-bet deficiency in the innate immune system; citation_author=W.S. Garrett; citation_author=G.M. Lord; citation_author=S. Punit; citation_author=G. Lugo-Villarino; citation_author=S.K. Mazmanian; citation_author=S. Ito; citation_author=J.N. Glickman; citation_author=L.H. Glimcher; citation_journal_title=Cell; citation_volume=131; citation_firstpage=33; citation_lastpage=45; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=T-bet controls intestinal mucosa immune responses via repression of type 2 innate lymphoid cell function; citation_author=N. Garrido-Mesa; citation_author=J.-H. Schroeder; citation_author=E. Stolarczyk; citation_author=A.L. Gallagher; citation_author=J.W. Lo; citation_author=C. Bailey; citation_author=L. Campbell; citation_author=V. Sexl; citation_author=T.T. MacDonald; citation_author=J.K. Howard; citation_author=et al.; citation_journal_title=Mucosal Immunol.; citation_volume=12; citation_firstpage=51; citation_lastpage=63; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Annexin A1 and the regulation of innate and adaptive immunity; citation_author=F.N.E. Gavins; citation_author=M.J. Hickey; citation_journal_title=Front. Immunol.; citation_volume=3; citation_firstpage=354; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Human naive and memory CD4+ T&nbsp;cell repertoires specific for naturally processed antigens analyzed using libraries of amplified T&nbsp;cells; citation_author=R. Geiger; citation_author=T. Duhen; citation_author=A. Lanzavecchia; citation_author=F. Sallusto; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=206; citation_firstpage=1525; citation_lastpage=1534; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Extremely randomized trees; citation_author=P. Geurts; citation_author=D. Ernst; citation_author=L. Wehenkel; citation_journal_title=Mach. Learn.; citation_volume=63; citation_firstpage=3; citation_lastpage=42; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=The biology and functional importance of MAIT cells; citation_author=D.I. Godfrey; citation_author=H.-F. Koay; citation_author=J. McCluskey; citation_author=N.A. Gherardin; citation_journal_title=Nat. Immunol.; citation_volume=20; citation_firstpage=1110; citation_lastpage=1128; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genome-wide regulatory analysis reveals that T-bet controls Th17 lineage differentiation through direct suppression of IRF4; citation_author=M.R. Gökmen; citation_author=R. Dong; citation_author=A. Kanhere; citation_author=N. Powell; citation_author=E. Perucha; citation_author=I. Jackson; citation_author=J.K. Howard; citation_author=M. Hernandez-Fuentes; citation_author=R.G. Jenner; citation_author=G.M. Lord; citation_journal_title=J.&nbsp;Immunol.; citation_volume=191; citation_firstpage=5925; citation_lastpage=5932; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=CXCR3 in T&nbsp;cell function; citation_author=J.R. Groom; citation_author=A.D. Luster; citation_journal_title=Exp. Cell Res.; citation_volume=317; citation_firstpage=620; citation_lastpage=631; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Complex heatmaps reveal patterns and correlations in multidimensional genomic data; citation_author=Z. Gu; citation_author=R. Eils; citation_author=M. Schlesner; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=2847; citation_lastpage=2849; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Butyrophilin3A proteins and Vγ9Vδ2 T&nbsp;cell activation; citation_author=S. Gu; citation_author=M.T. Borowska; citation_author=C.T. Boughter; citation_author=E.J. Adams; citation_journal_title=Semin. Cell Dev. Biol.; citation_volume=84; citation_firstpage=65; citation_lastpage=74; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=IRF8 mutations and human dendritic-cell immunodeficiency; citation_author=S. Hambleton; citation_author=S. Salem; citation_author=J. Bustamante; citation_author=V. Bigley; citation_author=S. Boisson-Dupuis; citation_author=J. Azevedo; citation_author=A. Fortin; citation_author=M. Haniffa; citation_author=L. Ceron-Gutierrez; citation_author=C.M. Bacon; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=365; citation_firstpage=127; citation_lastpage=138; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=A novel mutation in TAP1 gene leading to MHC class I deficiency: Report of two cases and review of the literature; citation_author=D. Hanalioglu; citation_author=D.C. Ayvaz; citation_author=T.T. Ozgur; citation_author=M. van der Burg; citation_author=O. Sanal; citation_author=I. Tezcan; citation_journal_title=Clin. Immunol.; citation_volume=178; citation_firstpage=74; citation_lastpage=78; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=MHC class I and II deficiencies; citation_author=S. Hanna; citation_author=A. Etzioni; citation_journal_title=J.&nbsp;Allergy Clin. Immunol.; citation_volume=134; citation_firstpage=269; citation_lastpage=275; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Detection of genome-edited mutant clones by a simple competition-based PCR method; citation_author=T. Harayama; citation_author=H. Riezman; citation_journal_title=PLoS ONE; citation_volume=12; citation_firstpage=e0179165; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Key implication of CD277/butyrophilin-3 (BTN3A) in cellular stress sensing by a major human γδ T-cell subset; citation_author=C. Harly; citation_author=Y. Guillaume; citation_author=S. Nedellec; citation_author=C.M. Peigné; citation_author=H. Mönkkönen; citation_author=J. Mönkkönen; citation_author=J. Li; citation_author=J. Kuball; citation_author=E.J. Adams; citation_author=S. Netzer; citation_author=et al.; citation_journal_title=Blood; citation_volume=120; citation_firstpage=2269; citation_lastpage=2279; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Diverse roles for T-bet in the effector responses required for resistance to infection; citation_author=G. Harms Pritchard; citation_author=A.O. Hall; citation_author=D.A. Christian; citation_author=S. Wagage; citation_author=Q. Fang; citation_author=G. Muallem; citation_author=B. John; citation_author=A. Glatman Zaretsky; citation_author=W.G. Dunn; citation_author=J. Perrigoue; citation_author=et al.; citation_journal_title=J.&nbsp;Immunol.; citation_volume=194; citation_firstpage=1131; citation_lastpage=1140; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Life-threatening influenza pneumonitis in a child with inherited IRF9 deficiency; citation_author=N. Hernandez; citation_author=I. Melki; citation_author=H. Jing; citation_author=T. Habib; citation_author=S.S.Y. Huang; citation_author=J. Danielson; citation_author=T. Kula; citation_author=S. Drutman; citation_author=S. Belkaya; citation_author=V. Rattina; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=215; citation_firstpage=2567; citation_lastpage=2585; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Bacille Calmette-Guérin vaccination enhances human gamma delta T&nbsp;cell responsiveness to mycobacteria suggestive of a memory-like phenotype; citation_author=D.F. Hoft; citation_author=R.M. Brown; citation_author=S.T. Roodman; citation_journal_title=J.&nbsp;Immunol.; citation_volume=161; citation_firstpage=1045; citation_lastpage=1054; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=MCM4 mutation causes adrenal failure, short stature, and natural killer cell deficiency in humans; citation_author=C.R. Hughes; citation_author=L. Guasti; citation_author=E. Meimaridou; citation_author=C.H. Chuang; citation_author=J.C. Schimenti; citation_author=P.J. King; citation_author=C. Costigan; citation_author=A.J.L. Clark; citation_author=L.A. Metherell; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=122; citation_firstpage=814; citation_lastpage=820; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=T-box transcription-factor-deficient mice display increased joint pathology and failure of infection control during staphylococcal arthritis; citation_author=O.H. Hultgren; citation_author=M. Verdrengh; citation_author=A. Tarkowski; citation_journal_title=Microbes Infect.; citation_volume=6; citation_firstpage=529; citation_lastpage=535; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Requirement for T-bet in the aberrant differentiation of unhelped memory CD8+ T&nbsp;cells; citation_author=A.M. Intlekofer; citation_author=N. Takemoto; citation_author=C. Kao; citation_author=A. Banerjee; citation_author=F. Schambach; citation_author=J.K. Northrop; citation_author=H. Shen; citation_author=E.J. Wherry; citation_author=S.L. Reiner; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=204; citation_firstpage=2015; citation_lastpage=2021; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The human gene connectome as a map of short cuts for morbid allele discovery; citation_author=Y. Itan; citation_author=S.-Y. Zhang; citation_author=G. Vogt; citation_author=A. Abhyankar; citation_author=M. Herman; citation_author=P. Nitschke; citation_author=D. Fried; citation_author=L. Quintana-Murci; citation_author=L. Abel; citation_author=J.-L. Casanova; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=110; citation_firstpage=5558; citation_lastpage=5563; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=HGCS: an online tool for prioritizing disease-causing gene variants by biological distance; citation_author=Y. Itan; citation_author=M. Mazel; citation_author=B. Mazel; citation_author=A. Abhyankar; citation_author=P. Nitschke; citation_author=L. Quintana-Murci; citation_author=S. Boisson-Dupuis; citation_author=B. Boisson; citation_author=L. Abel; citation_author=S.-Y. Zhang; citation_author=J.L. Casanova; citation_journal_title=BMC Genomics; citation_volume=15; citation_firstpage=256; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The human gene damage index as a gene-level approach to prioritizing exome variants; citation_author=Y. Itan; citation_author=L. Shang; citation_author=B. Boisson; citation_author=E. Patin; citation_author=A. Bolze; citation_author=M. Moncada-Vélez; citation_author=E. Scott; citation_author=M.J. Ciancanelli; citation_author=F.G. Lafaille; citation_author=J.G. Markle; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=112; citation_firstpage=13615; citation_lastpage=13620; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The mutation significance cutoff: gene-level thresholds for variant predictions; citation_author=Y. Itan; citation_author=L. Shang; citation_author=B. Boisson; citation_author=M.J. Ciancanelli; citation_author=J.G. Markle; citation_author=R. Martinez-Barricarte; citation_author=E. Scott; citation_author=I. Shah; citation_author=P.D. Stenson; citation_author=J. Gleeson; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=13; citation_firstpage=109; citation_lastpage=110; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The Transcription Factor T-bet Limits Amplification of Type I IFN Transcriptome and Circuitry in T Helper 1 Cells; citation_author=S. Iwata; citation_author=Y. Mikami; citation_author=H.-W. Sun; citation_author=S.R. Brooks; citation_author=D. Jankovic; citation_author=K. Hirahara; citation_author=A. Onodera; citation_author=H.-Y. Shih; citation_author=T. Kawabe; citation_author=K. Jiang; citation_author=et al.; citation_journal_title=Immunity; citation_volume=46; citation_firstpage=983; citation_lastpage=991; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=RIPPLY3 is a retinoic acid-inducible repressor required for setting the borders of the pre-placodal ectoderm; citation_author=A. Janesick; citation_author=J. Shiotsugu; citation_author=M. Taketani; citation_author=B. Blumberg; citation_journal_title=Development; citation_volume=139; citation_firstpage=1213; citation_lastpage=1224; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Lysine 313 of T-box is crucial for modulation of protein stability, DNA binding, and threonine phosphorylation of T-bet; citation_author=E.J. Jang; citation_author=H.R. Park; citation_author=J.H. Hong; citation_author=E.S. Hwang; citation_journal_title=J.&nbsp;Immunol.; citation_volume=190; citation_firstpage=5764; citation_lastpage=5770; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The transcription factors T-bet and GATA-3 control alternative pathways of T-cell differentiation through a shared set of target genes; citation_author=R.G. Jenner; citation_author=M.J. Townsend; citation_author=I. Jackson; citation_author=K. Sun; citation_author=R.D. Bouwman; citation_author=R.A. Young; citation_author=L.H. Glimcher; citation_author=G.M. Lord; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=106; citation_firstpage=17876; citation_lastpage=17881; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=IL-12 and IFN-γ in host defense against mycobacteria and salmonella in mice and men; citation_author=E. Jouanguy; citation_author=R. Döffinger; citation_author=S. Dupuis; citation_author=A. Pallier; citation_author=F. Altare; citation_author=J.L. Casanova; citation_journal_title=Curr. Opin. Immunol.; citation_volume=11; citation_firstpage=346; citation_lastpage=351; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=T-bet controls autoaggressive CD8 lymphocyte responses in type 1 diabetes; citation_author=A.E. Juedes; citation_author=E. Rodrigo; citation_author=L. Togher; citation_author=L.H. Glimcher; citation_author=M.G. von Herrath; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=199; citation_firstpage=1153; citation_lastpage=1162; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=T-bet and GATA3 orchestrate Th1 and Th2 differentiation through lineage-specific targeting of distal regulatory elements; citation_author=A. Kanhere; citation_author=A. Hertweck; citation_author=U. Bhatia; citation_author=M.R. Gökmen; citation_author=E. Perucha; citation_author=I. Jackson; citation_author=G.M. Lord; citation_author=R.G. Jenner; citation_journal_title=Nat. Commun.; citation_volume=3; citation_firstpage=1268; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Transcription factor T-bet represses expression of the inhibitory receptor PD-1 and sustains virus-specific CD8+ T&nbsp;cell responses during chronic infection; citation_author=C. Kao; citation_author=K.J. Oestreich; citation_author=M.A. Paley; citation_author=A. Crawford; citation_author=J.M. Angelosanto; citation_author=M.A. Ali; citation_author=A.M. Intlekofer; citation_author=J.M. Boss; citation_author=S.L. Reiner; citation_author=A.S. Weinmann; citation_author=E.J. Wherry; citation_journal_title=Nat. Immunol.; citation_volume=12; citation_firstpage=663; citation_lastpage=671; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The mutational constraint spectrum quantified from variation in 141,456 humans; citation_author=K.J. Karczewski; citation_author=L.C. Francioli; citation_author=G. Tiao; citation_author=B.B. Cummings; citation_author=J. Alföldi; citation_author=Q. Wang; citation_author=R.L. Collins; citation_author=K.M. Laricchia; citation_author=A. Ganna; citation_author=D.P. Birnbaum; citation_author=Genome Aggregation Database Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=581; citation_firstpage=434; citation_lastpage=443; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Inherited human IFN-γ deficiency underlies mycobacterial disease; citation_author=G. Kerner; citation_author=J. Rosain; citation_author=A. Guérin; citation_author=A. Al-Khabaz; citation_author=C. Oleaga-Quintas; citation_author=F. Rapaport; citation_author=M.J. Massaad; citation_author=J.Y. Ding; citation_author=T. Khan; citation_author=F.A. Ali; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=130; citation_firstpage=3158; citation_lastpage=3171; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=A general framework for estimating the relative pathogenicity of human genetic variants; citation_author=M. Kircher; citation_author=D.M. Witten; citation_author=P. Jain; citation_author=B.J. O’Roak; citation_author=G.M. Cooper; citation_author=J. Shendure; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=310; citation_lastpage=315; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=MR1 presents microbial vitamin B metabolites to MAIT cells; citation_author=L. Kjer-Nielsen; citation_author=O. Patel; citation_author=A.J. Corbett; citation_author=J. Le Nours; citation_author=B. Meehan; citation_author=L. Liu; citation_author=M. Bhati; citation_author=Z. Chen; citation_author=L. Kostenko; citation_author=R. Reantragoon; citation_author=et al.; citation_journal_title=Nature; citation_volume=491; citation_firstpage=717; citation_lastpage=723; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=The transcription factor T-bet controls regulatory T&nbsp;cell homeostasis and function during type 1 inflammation; citation_author=M.A. Koch; citation_author=G. Tucker-Heard; citation_author=N.R. Perdue; citation_author=J.R. Killebrew; citation_author=K.B. Urdahl; citation_author=D.J. Campbell; citation_journal_title=Nat. Immunol.; citation_volume=10; citation_firstpage=595; citation_lastpage=602; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Disruption of an antimycobacterial circuit between dendritic and helper T&nbsp;cells in human SPPL2a deficiency; citation_author=X.F. Kong; citation_author=R. Martinez-Barricarte; citation_author=J. Kennedy; citation_author=F. Mele; citation_author=T. Lazarov; citation_author=E.K. Deenick; citation_author=C.S. Ma; citation_author=G. Breton; citation_author=K.B. Lucero; citation_author=D. Langlais; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=19; citation_firstpage=973; citation_lastpage=985; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Fast, sensitive and accurate integration of single-cell data with Harmony; citation_author=I. Korsunsky; citation_author=N. Millard; citation_author=J. Fan; citation_author=K. Slowikowski; citation_author=F. Zhang; citation_author=K. Wei; citation_author=Y. Baglaenko; citation_author=M. Brenner; citation_author=P.R. Loh; citation_author=S. Raychaudhuri; citation_journal_title=Nat. Methods; citation_volume=16; citation_firstpage=1289; citation_lastpage=1296; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The Role of XCR1 and its Ligand XCL1 in Antigen Cross-Presentation by Murine and Human Dendritic Cells; citation_author=R.A. Kroczek; citation_author=V. Henn; citation_journal_title=Front. Immunol.; citation_volume=3; citation_firstpage=14; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Natural killer T&nbsp;cells and X-linked lymphoproliferative syndrome; citation_author=S. Latour; citation_journal_title=Curr. Opin. Allergy Clin. Immunol.; citation_volume=7; citation_firstpage=510; citation_lastpage=514; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=T-bet represses T(H)17 differentiation by preventing Runx1-mediated activation of the gene encoding RORγt; citation_author=V. Lazarevic; citation_author=X. Chen; citation_author=J.H. Shim; citation_author=E.S. Hwang; citation_author=E. Jang; citation_author=A.N. Bolm; citation_author=M. Oukka; citation_author=V.K. Kuchroo; citation_author=L.H. Glimcher; citation_journal_title=Nat. Immunol.; citation_volume=12; citation_firstpage=96; citation_lastpage=104; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=T-bet: a bridge between innate and adaptive immunity; citation_author=V. Lazarevic; citation_author=L.H. Glimcher; citation_author=G.M. Lord; citation_journal_title=Nat. Rev. Immunol.; citation_volume=13; citation_firstpage=777; citation_lastpage=789; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=T-bet Is Required for the Rapid Clearance of Attenuated Rabies Virus from Central Nervous System Tissue; citation_author=A. Lebrun; citation_author=C. Portocarrero; citation_author=R.B. Kean; citation_author=D.A. Barkhouse; citation_author=M. Faber; citation_author=D.C. Hooper; citation_journal_title=J.&nbsp;Immunol.; citation_volume=195; citation_firstpage=4358; citation_lastpage=4368; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Stability and function of regulatory T&nbsp;cells expressing the transcription factor T-bet; citation_author=A.G. Levine; citation_author=A. Mendoza; citation_author=S. Hemmers; citation_author=B. Moltedo; citation_author=R.E. Niec; citation_author=M. Schizas; citation_author=B.E. Hoyos; citation_author=E.V. Putintseva; citation_author=A. Chaudhry; citation_author=S. Dikiy; citation_author=et al.; citation_journal_title=Nature; citation_volume=546; citation_firstpage=421; citation_lastpage=425; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=T-bet’s ability to regulate individual target genes requires the conserved T-box domain to recruit histone methyltransferase activity and a separate family member-specific transactivation domain; citation_author=M.D. Lewis; citation_author=S.A. Miller; citation_author=M.M. Miazgowicz; citation_author=K.M. Beima; citation_author=A.S. Weinmann; citation_journal_title=Mol. Cell. Biol.; citation_volume=27; citation_firstpage=8510; citation_lastpage=8521; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=H. Li; citation_author=B. Handsaker; citation_author=A. Wysoker; citation_author=T. Fennell; citation_author=J. Ruan; citation_author=N. Homer; citation_author=G. Marth; citation_author=G. Abecasis; citation_author=R. Durbin; citation_author=1000 Genome Project Data Processing Subgroup; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=2078; citation_lastpage=2079; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=featureCounts: an efficient general purpose program for assigning sequence reads to genomic features; citation_author=Y. Liao; citation_author=G.K. Smyth; citation_author=W. Shi; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=923; citation_lastpage=930; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A. Liberzon; citation_author=C. Birger; citation_author=H. Thorvaldsdóttir; citation_author=M. Ghandi; citation_author=J.P. Mesirov; citation_author=P. Tamayo; citation_journal_title=Cell Syst.; citation_volume=1; citation_firstpage=417; citation_lastpage=425; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Systemic Human ILC Precursors Provide a Substrate for Tissue ILC Differentiation; citation_author=A.I. Lim; citation_author=Y. Li; citation_author=S. Lopez-Lastra; citation_author=R. Stadhouders; citation_author=F. Paul; citation_author=A. Casrouge; citation_author=N. Serafini; citation_author=A. Puel; citation_author=J. Bustamante; citation_author=L. Surace; citation_author=et al.; citation_journal_title=Cell; citation_volume=168; citation_firstpage=1086; citation_lastpage=1100; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=A Quantitative Analysis of Complexity of Human Pathogen-Specific CD4&nbsp;T Cell Responses in Healthy M. tuberculosis Infected South Africans; citation_author=C.S. Lindestam Arlehamn; citation_author=D.M. McKinney; citation_author=C. Carpenter; citation_author=S. Paul; citation_author=V. Rozot; citation_author=E. Makgotlho; citation_author=Y. Gregg; citation_author=M. van Rooyen; citation_author=J.D. Ernst; citation_author=M. Hatherill; citation_author=et al.; citation_journal_title=PLoS Pathog.; citation_volume=12; citation_firstpage=e1005760; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Gain-of-function human STAT1 mutations impair IL-17 immunity and underlie chronic mucocutaneous candidiasis; citation_author=L. Liu; citation_author=S. Okada; citation_author=X.-F. Kong; citation_author=A.Y. Kreins; citation_author=S. Cypowyj; citation_author=A. Abhyankar; citation_author=J. Toubiana; citation_author=Y. Itan; citation_author=M. Audry; citation_author=P. Nitschke; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=208; citation_firstpage=1635; citation_lastpage=1648; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The Wiskott-Aldrich syndrome protein is required for iNKT cell maturation and function; citation_author=M. Locci; citation_author=E. Draghici; citation_author=F. Marangoni; citation_author=M. Bosticardo; citation_author=M. Catucci; citation_author=A. Aiuti; citation_author=C. Cancrini; citation_author=L. Marodi; citation_author=T. Espanol; citation_author=R.G.M. Bredius; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=206; citation_firstpage=735; citation_lastpage=742; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=CCR5 is characteristic of Th1 lymphocytes; citation_author=P. Loetscher; citation_author=M. Uguccioni; citation_author=L. Bordoli; citation_author=M. Baggiolini; citation_author=B. Moser; citation_author=C. Chizzolini; citation_author=J.-M. Dayer; citation_journal_title=Nature; citation_volume=391; citation_firstpage=344; citation_lastpage=345; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=M.I. Love; citation_author=W. Huber; citation_author=S. Anders; citation_journal_title=Genome Biol.; citation_volume=15; citation_firstpage=550; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=T-bet is required for optimal production of IFN-gamma and antigen-specific T&nbsp;cell activation by dendritic cells; citation_author=G. Lugo-Villarino; citation_author=R. Maldonado-Lopez; citation_author=R. Possemato; citation_author=C. Penaranda; citation_author=L.H. Glimcher; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=100; citation_firstpage=7749; citation_lastpage=7754; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Hematopoietic Cell Transplantation for MHC Class II Deficiency; citation_author=S.H. Lum; citation_author=B. Neven; citation_author=M.A. Slatter; citation_author=A.R. Gennery; citation_journal_title=Front Pediatr.; citation_volume=7; citation_firstpage=516; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Improved transplant survival and long-term disease outcome in children with MHC class II deficiency; citation_author=S.H. Lum; citation_author=C. Anderson; citation_author=P. McNaughton; citation_author=K.R. Engelhardt; citation_author=B. MacKenzie; citation_author=H. Watson; citation_author=H. Al-Mousa; citation_author=W. Al-Herz; citation_author=B. Al-Saud; citation_author=R. Mohammed; citation_author=et al.; citation_journal_title=Blood; citation_volume=135; citation_firstpage=954; citation_lastpage=973; citation_publication_date=2020; "><meta name="citation_reference" content="citation_title=Identification of genes involved in the initiation of human Th1 or Th2 cell commitment; citation_author=R. Lund; citation_author=H. Ahlfors; citation_author=E. Kainonen; citation_author=A.-M. Lahesmaa; citation_author=C. Dixon; citation_author=R. Lahesmaa; citation_journal_title=Eur. J. Immunol.; citation_volume=35; citation_firstpage=3307; citation_lastpage=3319; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Motif-based analysis of large nucleotide data sets using MEME-ChIP; citation_author=W. Ma; citation_author=W.S. Noble; citation_author=T.L. Bailey; citation_journal_title=Nat. Protoc.; citation_volume=9; citation_firstpage=1428; citation_lastpage=1450; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Monogenic mutations differentially affect the quantity and quality of T follicular helper cells in patients with human primary immunodeficiencies; citation_author=C.S. Ma; citation_author=N. Wong; citation_author=G. Rao; citation_author=D.T. Avery; citation_author=J. Torpy; citation_author=T. Hambridge; citation_author=J. Bustamante; citation_author=S. Okada; citation_author=J.L. Stoddard; citation_author=E.K. Deenick; citation_author=et al.; citation_journal_title=J.&nbsp;Allergy Clin. Immunol.; citation_volume=136; citation_firstpage=993; citation_lastpage=1006; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest; citation_author=R. Martínez-Barricarte; citation_author=S.J. de Jong; citation_author=J. Markle; citation_author=R. de Paus; citation_author=S. Boisson-Dupuis; citation_author=J. Bustamante; citation_author=E. van de Vosse; citation_author=B. Fleckenstein; citation_author=J.-L. Casanova; citation_journal_title=Curr. Protoc. Immunol.; citation_volume=115; citation_firstpage=7.21C.1; citation_lastpage=7.21C.12; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Human IFN-γ immunity to mycobacteria is governed by both IL-12 and IL-23; citation_author=R. Martínez-Barricarte; citation_author=J.G. Markle; citation_author=C.S. Ma; citation_author=E.K. Deenick; citation_author=N. Ramírez-Alejo; citation_author=F. Mele; citation_author=D. Latorre; citation_author=S.A. Mahdaviani; citation_author=C. Aytekin; citation_author=D. Mansouri; citation_author=et al.; citation_journal_title=Sci. Immunol.; citation_volume=3; citation_firstpage=eaau6759; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Role of Th1/Th17 balance regulated by T-bet in a mouse model of Mycobacterium avium complex disease; citation_author=M. Matsuyama; citation_author=Y. Ishii; citation_author=Y. Yageta; citation_author=S. Ohtsuka; citation_author=S. Ano; citation_author=Y. Matsuno; citation_author=Y. Morishima; citation_author=K. Yoh; citation_author=S. Takahashi; citation_author=K. Ogawa; citation_author=et al.; citation_journal_title=J.&nbsp;Immunol.; citation_volume=192; citation_firstpage=1707; citation_lastpage=1717; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors; citation_author=C.S. McGinnis; citation_author=L.M. Murrow; citation_author=Z.J. Gartner; citation_journal_title=Cell Syst.; citation_volume=8; citation_firstpage=329; citation_lastpage=337; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A. McKenna; citation_author=M. Hanna; citation_author=E. Banks; citation_author=A. Sivachenko; citation_author=K. Cibulskis; citation_author=A. Kernytsky; citation_author=K. Garimella; citation_author=D. Altshuler; citation_author=S. Gabriel; citation_author=M. Daly; citation_author=M.A. DePristo; citation_journal_title=Genome Res.; citation_volume=20; citation_firstpage=1297; citation_lastpage=1303; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Molecular mechanisms by which T-bet regulates T-helper cell commitment; citation_author=S.A. Miller; citation_author=A.S. Weinmann; citation_journal_title=Immunol. Rev.; citation_volume=238; citation_firstpage=233; citation_lastpage=246; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Lymphocytes bearing antigen-specific γ δ T-cell receptors accumulate in human infectious disease lesions; citation_author=R.L. Modlin; citation_author=C. Pirmez; citation_author=F.M. Hofman; citation_author=V. Torigian; citation_author=K. Uyemura; citation_author=T.H. Rea; citation_author=B.R. Bloom; citation_author=M.B. Brenner; citation_journal_title=Nature; citation_volume=339; citation_firstpage=544; citation_lastpage=548; citation_publication_date=1989; "><meta name="citation_reference" content="citation_title=Mutation in the TCRα subunit constant gene (TRAC) leads to a human immunodeficiency disorder characterized by a lack of TCRαβ+ T&nbsp;cells; citation_author=N.V. Morgan; citation_author=S. Goddard; citation_author=T.S. Cardno; citation_author=D. McDonald; citation_author=F. Rahman; citation_author=D. Barge; citation_author=A. Ciupek; citation_author=A. Straatman-Iwanowska; citation_author=S. Pasha; citation_author=M. Guckian; citation_author=et al.; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=121; citation_firstpage=695; citation_lastpage=702; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Identification of interferon-gamma as the lymphokine that activates human macrophage oxidative metabolism and antimicrobial activity; citation_author=C.F. Nathan; citation_author=H.W. Murray; citation_author=M.E. Wiebe; citation_author=B.Y. Rubin; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=158; citation_firstpage=670; citation_lastpage=689; citation_publication_date=1983; "><meta name="citation_reference" content="citation_title=Activated STAT4 has an essential role in Th1 differentiation and proliferation that is independent of its role in the maintenance of IL-12R beta 2 chain expression and signaling; citation_author=R. Nishikomori; citation_author=T. Usui; citation_author=C.-Y. Wu; citation_author=A. Morinobu; citation_author=J.J. O’Shea; citation_author=W. Strober; citation_journal_title=J.&nbsp;Immunol.; citation_volume=169; citation_firstpage=4388; citation_lastpage=4398; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=The transcription factor T-bet regulates parasitemia and promotes pathogenesis during Plasmodium berghei ANKA murine malaria; citation_author=M.S. Oakley; citation_author=B.R. Sahu; citation_author=L. Lotspeich-Cole; citation_author=N.R. Solanki; citation_author=V. Majam; citation_author=P.T. Pham; citation_author=R. Banerjee; citation_author=Y. Kozakai; citation_author=S.C. Derrick; citation_author=S. Kumar; citation_author=S.L. Morris; citation_journal_title=J.&nbsp;Immunol.; citation_volume=191; citation_firstpage=4699; citation_lastpage=4708; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Multibatch Cytometry Data Integration for Optimal Immunophenotyping.; citation_author=M. Ogishi; citation_journal_title=J. Immunol.; citation_volume=206; citation_doi=10.4049/jimmunol.2000854; citation_publication_date=2021; "><meta name="citation_reference" content="citation_title=Impairment of immunity to Candida and Mycobacterium in humans with bi-allelic RORC mutations; citation_author=S. Okada; citation_author=J.G. Markle; citation_author=E.K. Deenick; citation_author=F. Mele; citation_author=D. Averbuch; citation_author=M. Lagos; citation_author=M. Alzahrani; citation_author=S. Al-Muhsen; citation_author=R. Halwani; citation_author=C.S. Ma; citation_author=et al.; citation_journal_title=Science; citation_volume=349; citation_firstpage=606; citation_lastpage=613; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A predominance of the T&nbsp;cell receptor V gamma 2/V delta 2 subset in human mycobacteria-responsive T&nbsp;cells suggests germline gene encoded recognition; citation_author=G. Panchamoorthy; citation_author=J. McLean; citation_author=R.L. Modlin; citation_author=C.T. Morita; citation_author=S. Ishikawa; citation_author=M.B. Brenner; citation_author=H. Band; citation_journal_title=J.&nbsp;Immunol.; citation_volume=147; citation_firstpage=3360; citation_lastpage=3369; citation_publication_date=1991; "><meta name="citation_reference" content="citation_title=Evidence for extrathymic changes in the T&nbsp;cell receptor gamma/delta repertoire; citation_author=C.M. Parker; citation_author=V. Groh; citation_author=H. Band; citation_author=S.A. Porcelli; citation_author=C. Morita; citation_author=M. Fabbi; citation_author=D. Glass; citation_author=J.L. Strominger; citation_author=M.B. Brenner; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=171; citation_firstpage=1597; citation_lastpage=1612; citation_publication_date=1990; "><meta name="citation_reference" content="citation_title=Control of Effector CD8+ T Cell Function by the Transcription Factor Eomesodermin; citation_author=E.L. Pearce; citation_author=A.C. Mullen; citation_author=G.A. Martins; citation_author=C.M. Krawczyk; citation_author=A.S. Hutchins; citation_author=V.P. Zediak; citation_author=M. Banica; citation_author=C.B. DiCioccio; citation_author=D.A. Gross; citation_author=C.-A. Mao; citation_author=et al.; citation_journal_title=Science; citation_volume=302; citation_firstpage=1041; citation_lastpage=1043; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=The Intolerance of Regulatory Sequence to Genetic Variation Predicts Gene Dosage Sensitivity; citation_author=S. Petrovski; citation_author=A.B. Gussow; citation_author=Q. Wang; citation_author=M. Halvorsen; citation_author=Y. Han; citation_author=W.H. Weir; citation_author=A.S. Allen; citation_author=D.B. Goldstein; citation_journal_title=PLoS Genet.; citation_volume=11; citation_firstpage=e1005492; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=The transcription factor T-bet regulates intestinal inflammation mediated by interleukin-7 receptor+ innate lymphoid cells; citation_author=N. Powell; citation_author=A.W. Walker; citation_author=E. Stolarczyk; citation_author=J.B. Canavan; citation_author=M.R. Gökmen; citation_author=E. Marks; citation_author=I. Jackson; citation_author=A. Hashim; citation_author=M.A. Curtis; citation_author=R.G. Jenner; citation_author=et al.; citation_journal_title=Immunity; citation_volume=37; citation_firstpage=674; citation_lastpage=684; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=deepTools2: a next generation web server for deep-sequencing data analysis; citation_author=F. Ramírez; citation_author=D.P. Ryan; citation_author=B. Grüning; citation_author=V. Bhardwaj; citation_author=F. Kilpert; citation_author=A.S. Richter; citation_author=S. Heyne; citation_author=F. Dündar; citation_author=T. Manke; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=W1; citation_firstpage=W160-5; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The transcription factor T-bet is essential for the development of NKp46+ innate lymphocytes via the Notch pathway; citation_author=L.C. Rankin; citation_author=J.R. Groom; citation_author=M. Chopin; citation_author=M.J. Herold; citation_author=J.A. Walker; citation_author=L.A. Mielke; citation_author=A.N.J. McKenzie; citation_author=S. Carotta; citation_author=S.L. Nutt; citation_author=G.T. Belz; citation_journal_title=Nat. Immunol.; citation_volume=14; citation_firstpage=389; citation_lastpage=395; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Expression of T-bet by CD4 T&nbsp;cells is essential for resistance to Salmonella infection; citation_author=R. Ravindran; citation_author=J. Foley; citation_author=T. Stoklasek; citation_author=L.H. Glimcher; citation_author=S.J. McSorley; citation_journal_title=J.&nbsp;Immunol.; citation_volume=175; citation_firstpage=4603; citation_lastpage=4610; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Somatic genetic rescue in Mendelian haematopoietic diseases; citation_author=P. Revy; citation_author=C. Kannengiesser; citation_author=A. Fischer; citation_journal_title=Nat. Rev. Genet.; citation_volume=20; citation_firstpage=582; citation_lastpage=598; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M.D. Robinson; citation_author=D.J. McCarthy; citation_author=G.K. Smyth; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=139; citation_lastpage=140; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Integrative genomics viewer; citation_author=J.T. Robinson; citation_author=H. Thorvaldsdóttir; citation_author=W. Winckler; citation_author=M. Guttman; citation_author=E.S. Lander; citation_author=G. Getz; citation_author=J.P. Mesirov; citation_journal_title=Nat. Biotechnol.; citation_volume=29; citation_firstpage=24; citation_lastpage=26; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Mendelian susceptibility to mycobacterial disease: 2014-2018 update; citation_author=J. Rosain; citation_author=X.F. Kong; citation_author=R. Martinez-Barricarte; citation_author=C. Oleaga-Quintas; citation_author=N. Ramirez-Alejo; citation_author=J. Markle; citation_author=S. Okada; citation_author=S. Boisson-Dupuis; citation_author=J.L. Casanova; citation_author=J. Bustamante; citation_journal_title=Immunol. Cell Biol.; citation_volume=97; citation_firstpage=360; citation_lastpage=367; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Cutting edge: STAT1 and T-bet play distinct roles in determining outcome of visceral leishmaniasis caused by Leishmania donovani; citation_author=L.E. Rosas; citation_author=H.M. Snider; citation_author=J. Barbi; citation_author=A.A. Satoskar; citation_author=G. Lugo-Villarino; citation_author=T. Keiser; citation_author=T. Papenfuss; citation_author=J.E. Durbin; citation_author=D. Radzioch; citation_author=L.H. Glimcher; citation_author=A.R. Satoskar; citation_journal_title=J.&nbsp;Immunol.; citation_volume=177; citation_firstpage=22; citation_lastpage=25; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Clinical course of patients with major histocompatibility complex class II deficiency; citation_author=M.A. Saleem; citation_author=P.D. Arkwright; citation_author=E.G. Davies; citation_author=A.J. Cant; citation_author=P.A. Veys; citation_journal_title=Arch. Dis. Child.; citation_volume=83; citation_firstpage=356; citation_lastpage=359; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Flexible programs of chemokine receptor expression on human polarized T helper 1 and 2 lymphocytes; citation_author=F. Sallusto; citation_author=D. Lenig; citation_author=C.R. Mackay; citation_author=A. Lanzavecchia; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=187; citation_firstpage=875; citation_lastpage=883; citation_publication_date=1998; "><meta name="citation_reference" content="citation_title=Improved vectors and genome-wide libraries for CRISPR screening; citation_author=N.E. Sanjana; citation_author=O. Shalem; citation_author=F. Zhang; citation_journal_title=Nat. Methods; citation_volume=11; citation_firstpage=783; citation_lastpage=784; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Regulation of interferon-γ during innate and adaptive immune responses; citation_author=J.R. Schoenborn; citation_author=C.B. Wilson; citation_journal_title=Adv. Immunol.; citation_volume=96; citation_firstpage=41; citation_lastpage=101; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Characterization of Greater Middle Eastern genetic variation for enhanced disease gene discovery; citation_author=E.M. Scott; citation_author=A. Halees; citation_author=Y. Itan; citation_author=E.G. Spencer; citation_author=Y. He; citation_author=M.A. Azab; citation_author=S.B. Gabriel; citation_author=A. Belkadi; citation_author=B. Boisson; citation_author=L. Abel; citation_author=Greater Middle East Variome Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=48; citation_firstpage=1071; citation_lastpage=1076; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Adaptive immune response of Vgamma2Vdelta2+ T&nbsp;cells during mycobacterial infections; citation_author=Y. Shen; citation_author=D. Zhou; citation_author=L. Qiu; citation_author=X. Lai; citation_author=M. Simon; citation_author=L. Shen; citation_author=Z. Kou; citation_author=Q. Wang; citation_author=L. Jiang; citation_author=J. Estep; citation_author=et al.; citation_journal_title=Science; citation_volume=295; citation_firstpage=2255; citation_lastpage=2258; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=A minimal IFN-gamma promoter confers Th1 selective expression; citation_author=M. Soutto; citation_author=F. Zhang; citation_author=B. Enerson; citation_author=Y. Tong; citation_author=M. Boothby; citation_author=T.M. Aune; citation_journal_title=J.&nbsp;Immunol.; citation_volume=169; citation_firstpage=4205; citation_lastpage=4212; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Comprehensive Integration of Single-Cell Data; citation_author=T. Stuart; citation_author=A. Butler; citation_author=P. Hoffman; citation_author=C. Hafemeister; citation_author=E. Papalexi; citation_author=W.M. Mauck; citation_author=Y. Hao; citation_author=M. Stoeckius; citation_author=P. Smibert; citation_author=R. Satija; citation_journal_title=Cell; citation_volume=177; citation_firstpage=1888; citation_lastpage=1902; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Antigen-driven effector CD8 T&nbsp;cell function regulated by T-bet; citation_author=B.M. Sullivan; citation_author=A. Juedes; citation_author=S.J. Szabo; citation_author=M. von Herrath; citation_author=L.H. Glimcher; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=100; citation_firstpage=15818; citation_lastpage=15823; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Increased susceptibility of mice lacking T-bet to infection with Mycobacterium tuberculosis correlates with increased IL-10 and decreased IFN-gamma production; citation_author=B.M. Sullivan; citation_author=O. Jobe; citation_author=V. Lazarevic; citation_author=K. Vasquez; citation_author=R. Bronson; citation_author=L.H. Glimcher; citation_author=I. Kramnik; citation_journal_title=J.&nbsp;Immunol.; citation_volume=175; citation_firstpage=4593; citation_lastpage=4602; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Protective immunity to genital herpes simplex [correction of simpex] virus type 2 infection is mediated by T-bet; citation_author=A. Svensson; citation_author=I. Nordström; citation_author=J.B. Sun; citation_author=K. Eriksson; citation_journal_title=J.&nbsp;Immunol.; citation_volume=174; citation_firstpage=6266; citation_lastpage=6273; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=A novel transcription factor, T-bet, directs Th1 lineage commitment; citation_author=S.J. Szabo; citation_author=S.T. Kim; citation_author=G.L. Costa; citation_author=X. Zhang; citation_author=C.G. Fathman; citation_author=L.H. Glimcher; citation_journal_title=Cell; citation_volume=100; citation_firstpage=655; citation_lastpage=669; citation_publication_date=2000; "><meta name="citation_reference" content="citation_title=Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells; citation_author=S.J. Szabo; citation_author=B.M. Sullivan; citation_author=C. Stemmann; citation_author=A.R. Satoskar; citation_author=B.P. Sleckman; citation_author=L.H. Glimcher; citation_journal_title=Science; citation_volume=295; citation_firstpage=338; citation_lastpage=342; citation_publication_date=2002; "><meta name="citation_reference" content="citation_title=Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program; citation_author=D. Taliun; citation_author=D.N. Harris; citation_author=M.D. Kessler; citation_author=J. Carlson; citation_author=Z.A. Szpiech; citation_author=R. Torres; citation_author=S.A.G. Taliun; citation_author=A. Corvelo; citation_author=S.M. Gogarten; citation_author=H.M. Kang; citation_author=et al.; citation_journal_title=BioRxiv; citation_firstpage=563866; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Singular role for T-BET+CXCR3+ regulatory T&nbsp;cells in protection from autoimmune diabetes; citation_author=T.G. Tan; citation_author=D. Mathis; citation_author=C. Benoist; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=14103; citation_lastpage=14108; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The role of the Th1 transcription factor T-bet in a mouse model of immune-mediated bone-marrow failure; citation_author=Y. Tang; citation_author=M.J. Desierto; citation_author=J. Chen; citation_author=N.S. Young; citation_journal_title=Blood; citation_volume=115; citation_firstpage=541; citation_lastpage=548; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Human immunity against EBV-lessons from the clinic; citation_author=S.G. Tangye; citation_author=U. Palendira; citation_author=E.S.J. Edwards; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=214; citation_firstpage=269; citation_lastpage=283; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=TBX21: a functional variant predicts improvement in asthma with the use of inhaled corticosteroids; citation_author=K.G. Tantisira; citation_author=E.S. Hwang; citation_author=B.A. Raby; citation_author=E.S. Silverman; citation_author=S.L. Lake; citation_author=B.G. Richter; citation_author=S.L. Peng; citation_author=J.M. Drazen; citation_author=L.H. Glimcher; citation_author=S.T. Weiss; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=101; citation_firstpage=18099; citation_lastpage=18104; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Signal transducer and activator of transcription 4 is required for the transcription factor T-bet to promote T helper 1 cell-fate determination; citation_author=V.T. Thieu; citation_author=Q. Yu; citation_author=H.-C. Chang; citation_author=N. Yeh; citation_author=E.T. Nguyen; citation_author=S. Sehra; citation_author=M.H. Kaplan; citation_journal_title=Immunity; citation_volume=29; citation_firstpage=679; citation_lastpage=690; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=T-bet antagonizes mSin3a recruitment and transactivates a fully methylated IFN-gamma promoter via a conserved T-box half-site; citation_author=Y. Tong; citation_author=T. Aune; citation_author=M. Boothby; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=102; citation_firstpage=2034; citation_lastpage=2039; citation_publication_date=2005; "><meta name="citation_reference" content="citation_title=Lamin A/C augments Th1 differentiation and response against vaccinia virus and Leishmania major; citation_author=R. Toribio-Fernández; citation_author=V. Zorita; citation_author=V. Rocha-Perugini; citation_author=S. Iborra; citation_author=G. Martínez Del Hoyo; citation_author=R. Chevre; citation_author=B. Dorado; citation_author=D. Sancho; citation_author=F. Sanchez-Madrid; citation_author=V. Andrés; citation_author=J.M. Gonzalez-Granado; citation_journal_title=Cell Death Dis.; citation_volume=9; citation_firstpage=9; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=T-bet regulates the terminal maturation and homeostasis of NK and Valpha14i NKT cells; citation_author=M.J. Townsend; citation_author=A.S. Weinmann; citation_author=J.L. Matsuda; citation_author=R. Salomon; citation_author=P.J. Farnham; citation_author=C.A. Biron; citation_author=L. Gapin; citation_author=L.H. Glimcher; citation_journal_title=Immunity; citation_volume=20; citation_firstpage=477; citation_lastpage=494; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=CD4+ alpha beta T&nbsp;cell and gamma delta T&nbsp;cell responses to Mycobacterium tuberculosis. Similarities and differences in Ag recognition, cytotoxic effector function, and cytokine production; citation_author=K. Tsukaguchi; citation_author=K.N. Balaji; citation_author=W.H. Boom; citation_journal_title=J.&nbsp;Immunol.; citation_volume=154; citation_firstpage=1786; citation_lastpage=1796; citation_publication_date=1995; "><meta name="citation_reference" content="citation_title=Antisense-mediated exon skipping to correct IL-12Rbeta1 deficiency in T&nbsp;cells; citation_author=E. van de Vosse; citation_author=E.M. Verhard; citation_author=R.A. de Paus; citation_author=G.J. Platenburg; citation_author=J.C.T. van Deutekom; citation_author=A. Aartsma-Rus; citation_author=J.T. van Dissel; citation_journal_title=Blood; citation_volume=113; citation_firstpage=4548; citation_lastpage=4555; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=FlowSOM: Using self-organizing maps for visualization and interpretation of cytometry data; citation_author=S. Van Gassen; citation_author=B. Callebaut; citation_author=M.J. Van Helden; citation_author=B.N. Lambrecht; citation_author=P. Demeester; citation_author=T. Dhaene; citation_author=Y. Saeys; citation_journal_title=Cytometry A; citation_volume=87; citation_firstpage=636; citation_lastpage=645; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Butyrophilin 3A1 binds phosphorylated antigens and stimulates human γδ T&nbsp;cells; citation_author=S. Vavassori; citation_author=A. Kumar; citation_author=G.S. Wan; citation_author=G.S. Ramanjaneyulu; citation_author=M. Cavallari; citation_author=S. El Daker; citation_author=T. Beddoe; citation_author=A. Theodossis; citation_author=N.K. Williams; citation_author=E. Gostick; citation_author=et al.; citation_journal_title=Nat. Immunol.; citation_volume=14; citation_firstpage=908; citation_lastpage=916; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Induction of Th1-Biased T Follicular Helper (Tfh) Cells in Lymphoid Tissues during Chronic Simian Immunodeficiency Virus Infection Defines Functionally Distinct Germinal Center Tfh Cells; citation_author=V. Velu; citation_author=G.H. Mylvaganam; citation_author=S. Gangadhara; citation_author=J.J. Hong; citation_author=S.S. Iyer; citation_author=S. Gumber; citation_author=C.C. Ibegbu; citation_author=F. Villinger; citation_author=R.R. Amara; citation_journal_title=J.&nbsp;Immunol.; citation_volume=197; citation_firstpage=1832; citation_lastpage=1842; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=In&nbsp;vitro differentiation of human macrophages with enhanced antimycobacterial activity; citation_author=G. Vogt; citation_author=C. Nathan; citation_journal_title=J.&nbsp;Clin. Invest.; citation_volume=121; citation_firstpage=3889; citation_lastpage=3901; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=RSeQC: quality control of RNA-seq experiments; citation_author=L. Wang; citation_author=S. Wang; citation_author=W. Li; citation_journal_title=Bioinformatics; citation_volume=28; citation_firstpage=2184; citation_lastpage=2185; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Dual T&nbsp;cell- and B cell-intrinsic deficiency in humans with biallelic RLTPR mutations; citation_author=Y. Wang; citation_author=C.S. Ma; citation_author=Y. Ling; citation_author=A. Bousfiha; citation_author=Y. Camcioglu; citation_author=S. Jacquot; citation_author=K. Payne; citation_author=E. Crestani; citation_author=R. Roncagalli; citation_author=A. Belkadi; citation_author=et al.; citation_journal_title=J.&nbsp;Exp. Med.; citation_volume=213; citation_firstpage=2413; citation_lastpage=2435; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking; citation_author=M.D. Wilkerson; citation_author=D.N. Hayes; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=1572; citation_lastpage=1573; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Mucosal-associated invariant T&nbsp;cells: New insights into antigen recognition and activation; citation_author=X. Xiao; citation_author=J. Cai; citation_journal_title=Front. Immunol.; citation_volume=8; citation_firstpage=1540; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Comprehensive serological profiling of human populations using a synthetic human virome; citation_author=G.J. Xu; citation_author=T. Kula; citation_author=Q. Xu; citation_author=M.Z. Li; citation_author=S.D. Vernon; citation_author=T. Ndung’u; citation_author=K. Ruxrungtham; citation_author=J. Sanchez; citation_author=C. Brander; citation_author=R.T. Chung; citation_author=et al.; citation_journal_title=Science; citation_volume=348; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Human Sox4 facilitates the development of CXCL13-producing helper T&nbsp;cells in inflammatory environments; citation_author=H. Yoshitomi; citation_author=S. Kobayashi; citation_author=A. Miyagawa-Hayashino; citation_author=A. Okahata; citation_author=K. Doi; citation_author=K. Nishitani; citation_author=K. Murata; citation_author=H. Ito; citation_author=T. Tsuruyama; citation_author=H. Haga; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=9; citation_firstpage=3762; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Pro- and antiinflammatory cytokine signaling: reciprocal antagonism regulates interferon-gamma production by human natural killer cells; citation_author=J. Yu; citation_author=M. Wei; citation_author=B. Becknell; citation_author=R. Trotta; citation_author=S. Liu; citation_author=Z. Boyd; citation_author=M.S. Jaung; citation_author=B.W. Blaser; citation_author=J. Sun; citation_author=D.M. Benson; citation_author=et al.; citation_journal_title=Immunity; citation_volume=24; citation_firstpage=575; citation_lastpage=590; citation_publication_date=2006; "><meta name="citation_reference" content="citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y. Zhang; citation_author=T. Liu; citation_author=C.A. Meyer; citation_author=J. Eeckhoute; citation_author=D.S. Johnson; citation_author=B.E. Bernstein; citation_author=C. Nusbaum; citation_author=R.M. Myers; citation_author=M. Brown; citation_author=W. Li; citation_author=X.S. Liu; citation_journal_title=Genome Biol.; citation_volume=9; citation_firstpage=R137; citation_publication_date=2008; "><meta name="citation_reference" content="citation_title=Human intracellular ISG15 prevents interferon-α/β over-amplification and auto-inflammation; citation_author=X. Zhang; citation_author=D. Bogunovic; citation_author=B. Payelle-Brogard; citation_author=V. Francois-Newton; citation_author=S.D. Speer; citation_author=C. Yuan; citation_author=S. Volpi; citation_author=Z. Li; citation_author=O. Sanal; citation_author=D. Mansouri; citation_author=et al.; citation_journal_title=Nature; citation_volume=517; citation_firstpage=89; citation_lastpage=93; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=PopViz: a webserver for visualizing minor allele frequencies and damage prediction scores of human genetic variations; citation_author=P. Zhang; citation_author=B. Bigio; citation_author=F. Rapaport; citation_author=S.-Y. Zhang; citation_author=J.-L. Casanova; citation_author=L. Abel; citation_author=B. Boisson; citation_author=Y. Itan; citation_journal_title=Bioinformatics; citation_volume=34; citation_firstpage=4307; citation_lastpage=4309; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Circulating CXCR3+ Tfh cells positively correlate with neutralizing antibody responses in HCV-infected patients; citation_author=J. Zhang; citation_author=W. Liu; citation_author=B. Wen; citation_author=T. Xie; citation_author=P. Tang; citation_author=Y. Hu; citation_author=L. Huang; citation_author=K. Jin; citation_author=P. Zhang; citation_author=Z. Liu; citation_author=et al.; citation_journal_title=Sci. Rep.; citation_volume=9; citation_firstpage=10090; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Human TH9 differentiation is dependent on signal transducer and activator of transcription (STAT) 3 to restrain STAT1-mediated inhibition; citation_author=Y. Zhang; citation_author=A.M. Siegel; citation_author=G. Sun; citation_author=T. Dimaggio; citation_author=A.F. Freeman; citation_author=J.D. Milner; citation_journal_title=J.&nbsp;Allergy Clin. Immunol.; citation_volume=143; citation_firstpage=1108; citation_lastpage=1118; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=The transcription factor GATA-3 is necessary and sufficient for Th2 cytokine gene expression in CD4 T&nbsp;cells; citation_author=W. Zheng; citation_author=R.A. Flavell; citation_journal_title=Cell; citation_volume=89; citation_firstpage=587; citation_lastpage=596; citation_publication_date=1997; "><meta name="citation_reference" content="citation_title=Differentiation of effector CD4 T&nbsp;cell populations (∗); citation_author=J. Zhu; citation_author=H. Yamane; citation_author=W.E. Paul; citation_journal_title=Annu. Rev. Immunol.; citation_volume=28; citation_firstpage=445; citation_lastpage=489; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Clinical and immunological aspects of HLA class I deficiency; citation_author=J. Zimmer; citation_author=E. Andrès; citation_author=L. Donato; citation_author=D. Hanau; citation_author=F. Hentges; citation_author=H. de la Salle; citation_journal_title=QJM; citation_volume=98; citation_firstpage=719; citation_lastpage=727; citation_publication_date=2005; "><meta name="citation_abstract" content="<h2>Summary</h2><p>Inborn errors of human interferon gamma (IFN-γ) immunity underlie mycobacterial disease. We report a patient with mycobacterial disease due to inherited deficiency of the transcription factor T-bet. The patient has extremely low counts of circulating <i>Mycobacterium</i>-reactive natural killer (NK), invariant NKT (iNKT), mucosal-associated invariant T (MAIT), and Vδ2<sup>+</sup> γδ T lymphocytes, and of <i>Mycobacterium</i>-non reactive classic T<sub>H</sub>1 lymphocytes, with the residual populations of these cells also producing abnormally small amounts of IFN-γ. Other lymphocyte subsets develop normally but produce low levels of IFN-γ, with the exception of CD8<sup>+</sup> αβ T and non-classic CD4<sup>+</sup> αβ T<sub>H</sub>1<sup>∗</sup> lymphocytes, which produce IFN-γ normally in response to mycobacterial antigens. Human T-bet deficiency thus underlies mycobacterial disease by preventing the development of innate (NK) and innate-like adaptive lymphocytes (iNKT, MAIT, and Vδ2<sup>+</sup> γδ T&nbsp;cells) and IFN-γ production by them, with mycobacterium-specific, IFN-γ-producing, purely adaptive CD8<sup>+</sup> αβ T, and CD4<sup>+</sup> αβ T<sub>H</sub>1<sup>∗</sup> cells unable to compensate for this deficit.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=5019&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(20)31453-7&amp;be=3359&amp;fe=4974&amp;dc=4599&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749588649588,%22n%22:0,%22f%22:2804,%22dn%22:2805,%22dne%22:2805,%22c%22:2805,%22s%22:2805,%22ce%22:2805,%22rq%22:2805,%22rp%22:3334,%22rpe%22:4221,%22dl%22:3340,%22di%22:4536,%22ds%22:4599,%22de%22:4619,%22dc%22:4973,%22l%22:4973,%22le%22:5005%7D,%22navigation%22:%7B%7D%7D&amp;fp=3856&amp;fcp=3856&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(20)31453-7"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(20)31453-7"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-805243" aria-expanded="false" aria-controls="login-links-805243" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-805243" id="login-links--805243" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-805243" data-opens="#links-dialog-805243" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-805243" aria-modal="true"><button data-closes="#links-dialog-805243" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2820%2931453-7" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2020.10.046">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(20)31453-7" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0027-6"><span>Volume 183</span>, <span>Issue 7</span></a></span><span class="meta-panel__pages"><span>p1826-1847.e31</span></span><span class="meta-panel__onlineDate">December 23, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=183&amp;issue=7&amp;pii=S0092-8674%2819%29X0027-6" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Human T-bet Governs Innate and Innate-like Adaptive IFN-γ Immunity against Mycobacteria</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">Rui</span> <span property="familyName">Yang</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rui</span> <span property="familyName">Yang</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4427-2158" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4427-2158</a></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:ryang@rockefeller.edu" property="email" title="Send email to Rui Yang">ryang@rockefeller.edu</a></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Yang/Rui" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup> <a id="eadd0010" href="mailto:ryang@rockefeller.edu" property="email" title="Send email to Rui Yang">ryang@rockefeller.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Federico</span> <span property="familyName">Mele</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Federico</span> <span property="familyName">Mele</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Mele/Federico" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a role="doc-noteref">37</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Lisa</span> <span property="familyName">Worley</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lisa</span> <span property="familyName">Worley</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia</span></div><div property="affiliation" typeof="Organization"><span property="name">St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Worley/Lisa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">37</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">David</span> <span property="familyName">Langlais</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David</span> <span property="familyName">Langlais</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4429-0110" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4429-0110</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, Department of Microbiology and Immunology, McGill University, Montreal, QC H3A 0G1, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Langlais/David" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a role="doc-noteref">37</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Jérémie</span> <span property="familyName">Rosain</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jérémie</span> <span property="familyName">Rosain</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Rosain/J%C3%A9r%C3%A9mie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a role="doc-noteref">37</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Ibithal</span> <span property="familyName">Benhsaien</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ibithal</span> <span property="familyName">Benhsaien</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco</span></div></div><div class="core-author-link"><a href="/authored-by/Benhsaien/Ibithal" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a role="doc-noteref">38</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Houda</span> <span property="familyName">Elarabi</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Houda</span> <span property="familyName">Elarabi</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Pediatrics Department, Hassan II Hospital, 80030 Dakhla, Morocco</span></div></div><div class="core-author-link"><a href="/authored-by/Elarabi/Houda" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">11</a>,<a role="doc-noteref">38</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Carys A.</span> <span property="familyName">Croft</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Carys A.</span> <span property="familyName">Croft</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Innate Immunity Unit, Institut Pasteur, 75724 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">INSERM U1223, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, 75006 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Croft/Carys+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a role="doc-noteref">38</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Jean-Marc</span> <span property="familyName">Doisne</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jean-Marc</span> <span property="familyName">Doisne</span> <a class="orcid-id" href="https://orcid.org/0000-0001-6714-1308" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-6714-1308</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Innate Immunity Unit, Institut Pasteur, 75724 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">INSERM U1223, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Doisne/Jean-Marc" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a role="doc-noteref">38</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Peng</span> <span property="familyName">Zhang</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Peng</span> <span property="familyName">Zhang</span> <a class="orcid-id" href="https://orcid.org/0000-0002-6129-567X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-6129-567X</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zhang/Peng" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a role="doc-noteref">38</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Marc</span> <span property="familyName">Weisshaar</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Marc</span> <span property="familyName">Weisshaar</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7769-1730" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7769-1730</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Weisshaar/Marc" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a>,<a role="doc-noteref">38</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">David</span> <span property="familyName">Jarrossay</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David</span> <span property="familyName">Jarrossay</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Jarrossay/David" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Daniela</span> <span property="familyName">Latorre</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Daniela</span> <span property="familyName">Latorre</span> <a class="orcid-id" href="https://orcid.org/0000-0003-1031-1534" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-1031-1534</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Latorre/Daniela" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Yichao</span> <span property="familyName">Shen</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yichao</span> <span property="familyName">Shen</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5257-6808" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5257-6808</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Shen/Yichao" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Jing</span> <span property="familyName">Han</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jing</span> <span property="familyName">Han</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Han/Jing" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Masato</span> <span property="familyName">Ogishi</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Masato</span> <span property="familyName">Ogishi</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2421-7389" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2421-7389</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Ogishi/Masato" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Conor</span> <span property="familyName">Gruber</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Conor</span> <span property="familyName">Gruber</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gruber/Conor" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">18</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Janet</span> <span property="familyName">Markle</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Janet</span> <span property="familyName">Markle</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Markle/Janet" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Fatima</span> <span property="familyName">Al Ali</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fatima</span> <span property="familyName">Al Ali</span> <a class="orcid-id" href="https://orcid.org/0000-0001-6451-7357" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-6451-7357</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Research Branch, Sidra Medicine, Doha, PO 26999, Qatar</span></div></div><div class="core-author-link"><a href="/authored-by/Al+Ali/Fatima" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Mahbuba</span> <span property="familyName">Rahman</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mahbuba</span> <span property="familyName">Rahman</span> <a class="orcid-id" href="https://orcid.org/0000-0003-4586-4067" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-4586-4067</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Research Branch, Sidra Medicine, Doha, PO 26999, Qatar</span></div></div><div class="core-author-link"><a href="/authored-by/Rahman/Mahbuba" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Taushif</span> <span property="familyName">Khan</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Taushif</span> <span property="familyName">Khan</span> <a class="orcid-id" href="https://orcid.org/0000-0002-7917-8965" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-7917-8965</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Research Branch, Sidra Medicine, Doha, PO 26999, Qatar</span></div></div><div class="core-author-link"><a href="/authored-by/Khan/Taushif" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Yoann</span> <span property="familyName">Seeleuthner</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Yoann</span> <span property="familyName">Seeleuthner</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Seeleuthner/Yoann" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Gaspard</span> <span property="familyName">Kerner</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Gaspard</span> <span property="familyName">Kerner</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Kerner/Gaspard" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Lucas T.</span> <span property="familyName">Husquin</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lucas T.</span> <span property="familyName">Husquin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Genetics Unit, CNRS UMR2000, Institut Pasteur, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Husquin/Lucas+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au25" aria-controls="axel_author_au25"><span property="givenName">Julia L.</span> <span property="familyName">Maclsaac</span></a><div id="au25" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au25"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Julia L.</span> <span property="familyName">Maclsaac</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">BC Children’s Hospital Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, BC V5Z 4H4, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Maclsaac/Julia+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">21</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au26" aria-controls="axel_author_au26"><span property="givenName">Mohamed</span> <span property="familyName">Jeljeli</span></a><div id="au26" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mohamed</span> <span property="familyName">Jeljeli</span> <a class="orcid-id" href="https://orcid.org/0000-0001-9561-8847" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-9561-8847</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, 75006 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Immunology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Jeljeli/Mohamed" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">22</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au27" aria-controls="axel_author_au27"><span property="givenName">Abderrahmane</span> <span property="familyName">Errami</span></a><div id="au27" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Abderrahmane</span> <span property="familyName">Errami</span> <a class="orcid-id" href="https://orcid.org/0000-0002-2916-5007" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-2916-5007</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco</span></div></div><div class="core-author-link"><a href="/authored-by/Errami/Abderrahmane" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au28" aria-controls="axel_author_au28"><span property="givenName">Fatima</span> <span property="familyName">Ailal</span></a><div id="au28" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fatima</span> <span property="familyName">Ailal</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco</span></div></div><div class="core-author-link"><a href="/authored-by/Ailal/Fatima" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au29" aria-controls="axel_author_au29"><span property="givenName">Michael S.</span> <span property="familyName">Kobor</span></a><div id="au29" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael S.</span> <span property="familyName">Kobor</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">BC Children’s Hospital Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, BC V5Z 4H4, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Kobor/Michael+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">21</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au30" aria-controls="axel_author_au30"><span property="givenName">Carmen</span> <span property="familyName">Oleaga-Quintas</span></a><div id="au30" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au30"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Carmen</span> <span property="familyName">Oleaga-Quintas</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Oleaga-Quintas/Carmen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au31" aria-controls="axel_author_au31"><span property="givenName">Manon</span> <span property="familyName">Roynard</span></a><div id="au31" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Manon</span> <span property="familyName">Roynard</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Roynard/Manon" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au32" aria-controls="axel_author_au32"><span property="givenName">Mathieu</span> <span property="familyName">Bourgey</span></a><div id="au32" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mathieu</span> <span property="familyName">Bourgey</span> <a class="orcid-id" href="https://orcid.org/0000-0002-8432-834X" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-8432-834X</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">Canadian Centre for Computational Genomics, Montreal, QC H3A 0G1, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Bourgey/Mathieu" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">23</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au33" aria-controls="axel_author_au33"><span property="givenName">Jamila</span> <span property="familyName">El Baghdadi</span></a><div id="au33" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au33"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jamila</span> <span property="familyName">El Baghdadi</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Genetics Unit, Military Hospital Mohamed V, 10100 Rabat, Morocco</span></div></div><div class="core-author-link"><a href="/authored-by/El+Baghdadi/Jamila" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">24</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au34" aria-controls="axel_author_au34"><span property="givenName">Stéphanie</span> <span property="familyName">Boisson-Dupuis</span></a><div id="au34" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stéphanie</span> <span property="familyName">Boisson-Dupuis</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Boisson-Dupuis/St%C3%A9phanie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au35" aria-controls="axel_author_au35"><span property="givenName">Anne</span> <span property="familyName">Puel</span></a><div id="au35" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anne</span> <span property="familyName">Puel</span> <a class="orcid-id" href="https://orcid.org/0000-0003-2603-0323" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-2603-0323</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Puel/Anne" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au36" aria-controls="axel_author_au36"><span property="givenName">Fréderic</span> <span property="familyName">Batteux</span></a><div id="au36" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Fréderic</span> <span property="familyName">Batteux</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, 75006 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Immunology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Batteux/Fr%C3%A9deric" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">22</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au37" aria-controls="axel_author_au37"><span property="givenName">Flore</span> <span property="familyName">Rozenberg</span></a><div id="au37" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au37"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Flore</span> <span property="familyName">Rozenberg</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, 75006 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Virology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Rozenberg/Flore" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">25</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au38" aria-controls="axel_author_au38"><span property="givenName">Nico</span> <span property="familyName">Marr</span></a><div id="au38" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au38"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Nico</span> <span property="familyName">Marr</span> <a class="orcid-id" href="https://orcid.org/0000-0002-1927-7072" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-1927-7072</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Research Branch, Sidra Medicine, Doha, PO 26999, Qatar</span></div><div property="affiliation" typeof="Organization"><span property="name">College of Health and Life Sciences, Hamad Bin Khalifa University, Doha, PO 34110, Qatar</span></div></div><div class="core-author-link"><a href="/authored-by/Marr/Nico" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">26</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au39" aria-controls="axel_author_au39"><span property="givenName">Qiang</span> <span property="familyName">Pan-Hammarström</span></a><div id="au39" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Qiang</span> <span property="familyName">Pan-Hammarström</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Biosciences and Nutrition, Karolinska Institutet, 17177 Stockholm, Sweden</span></div></div><div class="core-author-link"><a href="/authored-by/Pan-Hammarstr%C3%B6m/Qiang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">27</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au40" aria-controls="axel_author_au40"><span property="givenName">Dusan</span> <span property="familyName">Bogunovic</span></a><div id="au40" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au40"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Dusan</span> <span property="familyName">Bogunovic</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bogunovic/Dusan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">17</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">18</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au41" aria-controls="axel_author_au41"><span property="givenName">Lluis</span> <span property="familyName">Quintana-Murci</span></a><div id="au41" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lluis</span> <span property="familyName">Quintana-Murci</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Human Evolutionary Genetics Unit, CNRS UMR2000, Institut Pasteur, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Chair of Human Genomics and Evolution, Collège de France, 75005 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Quintana-Murci/Lluis" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">28</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au42" aria-controls="axel_author_au42"><span property="givenName">Thomas</span> <span property="familyName">Carroll</span></a><div id="au42" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Thomas</span> <span property="familyName">Carroll</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Bioinformatics Resource Center, The Rockefeller University, New York, NY 10065, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Carroll/Thomas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">29</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au43" aria-controls="axel_author_au43"><span property="givenName">Cindy S.</span> <span property="familyName">Ma</span></a><div id="au43" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Cindy S.</span> <span property="familyName">Ma</span> <a class="orcid-id" href="https://orcid.org/0000-0001-5387-8413" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0001-5387-8413</a></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia</span></div><div property="affiliation" typeof="Organization"><span property="name">St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Ma/Cindy+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au44" aria-controls="axel_author_au44"><span property="givenName">Laurent</span> <span property="familyName">Abel</span></a><div id="au44" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Laurent</span> <span property="familyName">Abel</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">39</div><div id="fn3"><div id="ntpara0020" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Abel/Laurent" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a role="doc-noteref">39</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au45" aria-controls="axel_author_au45"><span property="givenName">Aziz</span> <span property="familyName">Bousfiha</span></a><div id="au45" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aziz</span> <span property="familyName">Bousfiha</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5011-9873" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5011-9873</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">39</div><div id="fn3"><div id="ntpara0020" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco</span></div><div property="affiliation" typeof="Organization"><span property="name">Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco</span></div></div><div class="core-author-link"><a href="/authored-by/Bousfiha/Aziz" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a role="doc-noteref">39</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au46" aria-controls="axel_author_au46"><span property="givenName">James P.</span> <span property="familyName">Di Santo</span></a><div id="au46" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au46"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James P.</span> <span property="familyName">Di Santo</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">39</div><div id="fn3"><div id="ntpara0020" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Innate Immunity Unit, Institut Pasteur, 75724 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">INSERM U1223, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Di+Santo/James+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">12</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a role="doc-noteref">39</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au47" aria-controls="axel_author_au47"><span property="givenName">Laurie H.</span> <span property="familyName">Glimcher</span></a><div id="au47" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Laurie H.</span> <span property="familyName">Glimcher</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">40</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Immunology, Harvard Medical School, Boston, MA 02115, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Glimcher/Laurie+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">30</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">31</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">32</a>,<a role="doc-noteref">40</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au48" aria-controls="axel_author_au48"><span property="givenName">Philippe</span> <span property="familyName">Gros</span></a><div id="au48" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Philippe</span> <span property="familyName">Gros</span> <a class="orcid-id" href="https://orcid.org/0000-0002-0114-1978" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-0114-1978</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">40</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biochemistry, McGill University, Montreal, QC H3G 1Y6, Canada</span></div></div><div class="core-author-link"><a href="/authored-by/Gros/Philippe" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">33</a>,<a role="doc-noteref">40</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au49" aria-controls="axel_author_au49"><span property="givenName">Stuart G.</span> <span property="familyName">Tangye</span></a><div id="au49" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stuart G.</span> <span property="familyName">Tangye</span> <a class="orcid-id" href="https://orcid.org/0000-0002-5360-5180" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-5360-5180</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">40</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia</span></div><div property="affiliation" typeof="Organization"><span property="name">St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia</span></div></div><div class="core-author-link"><a href="/authored-by/Tangye/Stuart+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a role="doc-noteref">40</a></sup></span></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au50" aria-controls="axel_author_au50"><span property="givenName">Federica</span> <span property="familyName">Sallusto</span></a><div id="au50" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Federica</span> <span property="familyName">Sallusto</span> <a class="orcid-id" href="https://orcid.org/0000-0003-3750-2752" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0003-3750-2752</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">40</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland</span></div></div><div class="core-author-link"><a href="/authored-by/Sallusto/Federica" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">15</a>,<a role="doc-noteref">40</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au51" aria-controls="axel_author_au51"><span property="givenName">Jacinta</span> <span property="familyName">Bustamante</span></a><div id="au51" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jacinta</span> <span property="familyName">Bustamante</span> <a class="orcid-id" href="https://orcid.org/0000-0002-3439-2482" property="identifier" aria-label="ORCID identifier" target="_blank">0000-0002-3439-2482</a></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">41</div><div id="fn5"><div id="ntpara0030" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Study Center for Primary Immunodeficiencies, Necker Children Hospital, AP-HP, 75015 Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Bustamante/Jacinta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">34</a>,<a role="doc-noteref">41</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au52" aria-controls="axel_author_au52"><span property="givenName">Jean-Laurent</span> <span property="familyName">Casanova</span></a><div id="au52" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jean-Laurent</span> <span property="familyName">Casanova</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0015" href="mailto:casanova@rockefeller.edu" property="email" title="Send email to Jean-Laurent Casanova">casanova@rockefeller.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">41</div><div id="fn5"><div id="ntpara0030" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">42</div><div id="fn6"><div id="ntpara0035" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Pediatric Hematology-Immunology Unit, Necker Hospital for Sick Children, AP-HP, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span property="name">Howard Hughes Medical Institute, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Casanova/Jean-Laurent" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">35</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">36</a>,<a role="doc-noteref">41</a>,<a role="doc-noteref">42</a></sup> <a id="eadd0015" href="mailto:casanova@rockefeller.edu" property="email" title="Send email to Jean-Laurent Casanova">casanova@rockefeller.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">St Giles Laboratory of Human Genetics of Infectious Diseases, Rockefeller Branch, Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Center of Medical Immunology, Institute for Research in Biomedicine, Faculty of Biomedical Sciences, University of Italian Switzerland (USI), 6500 Bellinzona, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Garvan Institute of Medical Research, Darlinghurst 2010, NSW, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">St Vincent’s Clinical School, Faculty of Medicine, UNSW Sydney, Darlinghurst 2010, NSW, Australia</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Department of Human Genetics, Department of Microbiology and Immunology, McGill University, Montreal, QC H3A 0G1, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">McGill University Genome Center, McGill Research Centre on Complex Traits, Montreal, QC H3A 0G1, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Laboratory of Human Genetics of Infectious Diseases, Necker Branch, INSERM UMR 1163, Necker Hospital for Sick Children, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">University of Paris, Imagine Institute, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">Laboratory of Clinical Immunology, Inflammation and Allergy, Faculty of Medicine and Pharmacy of Casablanca, King Hassan II University, 20460 Casablanca, Morocco</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Clinical Immunology Unit, Department of Pediatric Infectious Diseases, Children’s Hospital, CHU Averroes, 20460 Casablanca, Morocco</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">Pediatrics Department, Hassan II Hospital, 80030 Dakhla, Morocco</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Innate Immunity Unit, Institut Pasteur, 75724 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">INSERM U1223, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">University of Paris, 75006 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">Institute of Microbiology, ETH Zurich, 8093 Zurich, Switzerland</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Department of Microbiology, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">17</span><span id="aff17" property="name">Department of Pediatrics, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">18</span><span id="aff18" property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY 10029, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">19</span><span id="aff19" property="name">Research Branch, Sidra Medicine, Doha, PO 26999, Qatar</span></div><div property="affiliation" typeof="Organization"><span class="label">20</span><span id="aff20" property="name">Human Evolutionary Genetics Unit, CNRS UMR2000, Institut Pasteur, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">21</span><span id="aff21" property="name">BC Children’s Hospital Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, BC V5Z 4H4, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">22</span><span id="aff22" property="name">Immunology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">23</span><span id="aff23" property="name">Canadian Centre for Computational Genomics, Montreal, QC H3A 0G1, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">24</span><span id="aff24" property="name">Genetics Unit, Military Hospital Mohamed V, 10100 Rabat, Morocco</span></div><div property="affiliation" typeof="Organization"><span class="label">25</span><span id="aff25" property="name">Virology Laboratory, Cochin Hospital, AH-HP, 75014 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">26</span><span id="aff26" property="name">College of Health and Life Sciences, Hamad Bin Khalifa University, Doha, PO 34110, Qatar</span></div><div property="affiliation" typeof="Organization"><span class="label">27</span><span id="aff27" property="name">Department of Biosciences and Nutrition, Karolinska Institutet, 17177 Stockholm, Sweden</span></div><div property="affiliation" typeof="Organization"><span class="label">28</span><span id="aff28" property="name">Chair of Human Genomics and Evolution, Collège de France, 75005 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">29</span><span id="aff29" property="name">Bioinformatics Resource Center, The Rockefeller University, New York, NY 10065, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">30</span><span id="aff30" property="name">Department of Cancer Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA 02215, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">31</span><span id="aff31" property="name">Department of Medicine, Brigham and Women’s Hospital, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">32</span><span id="aff32" property="name">Department of Immunology, Harvard Medical School, Boston, MA 02115, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">33</span><span id="aff33" property="name">Department of Biochemistry, McGill University, Montreal, QC H3G 1Y6, Canada</span></div><div property="affiliation" typeof="Organization"><span class="label">34</span><span id="aff34" property="name">Study Center for Primary Immunodeficiencies, Necker Children Hospital, AP-HP, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">35</span><span id="aff35" property="name">Pediatric Hematology-Immunology Unit, Necker Hospital for Sick Children, AP-HP, 75015 Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">36</span><span id="aff36" property="name">Howard Hughes Medical Institute, New York, NY, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">39</div><div id="fn3"><div id="ntpara0020" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">40</div><div id="fn4"><div id="ntpara0025" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">41</div><div id="fn5"><div id="ntpara0030" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">42</div><div id="fn6"><div id="ntpara0035" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received January 16, 2020</span>; <span>Revised June 25, 2020</span>; <span>Accepted October 26, 2020</span>; <span>Published online December 8, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2020.10.046" property="sameAs" target="_blank">10.1016/j.cell.2020.10.046 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867420314537" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2020.10.046" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0027-6"><img src="/cms/asset/88b41c5c-ac5d-4de0-a328-98a62b805db6/cov200h.gif" alt="Cover Image - Cell, Volume 183, Issue 7"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2820%2931453-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (3)</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2931453-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.10.046">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2820%2931453-7" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2020.10.046&amp;pii=S0092-8674%2820%2931453-7" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2820%2931453-7" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2020.10.046">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2020.10.046" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867420314537&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867420314537&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=183&amp;issue=7&amp;pii=S0092-8674%2819%29X0027-6" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)31531-2" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)31526-9" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (3)</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">Human inherited T-bet deficiency underlies severe mycobacterial disease</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">The development of a variety of IFN-γ-producing lymphocyte subsets is impaired</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">The production of IFN-γ by innate and innate-like lymphocytes is impaired</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph"><i>Mycobacterium</i>-specific αβ CD4 T<sub>H</sub>1<sup>∗</sup> and CD8 cells produce IFN-γ normally</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">Inborn errors of human interferon gamma (IFN-γ) immunity underlie mycobacterial disease. We report a patient with mycobacterial disease due to inherited deficiency of the transcription factor T-bet. The patient has extremely low counts of circulating <i>Mycobacterium</i>-reactive natural killer (NK), invariant NKT (iNKT), mucosal-associated invariant T (MAIT), and Vδ2<sup>+</sup> γδ T lymphocytes, and of <i>Mycobacterium</i>-non reactive classic T<sub>H</sub>1 lymphocytes, with the residual populations of these cells also producing abnormally small amounts of IFN-γ. Other lymphocyte subsets develop normally but produce low levels of IFN-γ, with the exception of CD8<sup>+</sup> αβ T and non-classic CD4<sup>+</sup> αβ T<sub>H</sub>1<sup>∗</sup> lymphocytes, which produce IFN-γ normally in response to mycobacterial antigens. Human T-bet deficiency thus underlies mycobacterial disease by preventing the development of innate (NK) and innate-like adaptive lymphocytes (iNKT, MAIT, and Vδ2<sup>+</sup> γδ T&nbsp;cells) and IFN-γ production by them, with mycobacterium-specific, IFN-γ-producing, purely adaptive CD8<sup>+</sup> αβ T, and CD4<sup>+</sup> αβ T<sub>H</sub>1<sup>∗</sup> cells unable to compensate for this deficit.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/7311ba78-9d2e-492d-b13f-4e53cb13fd35/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2020.10.046/asset/d136c7af-cd66-417c-b48b-0a321f215a4f/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/7311ba78-9d2e-492d-b13f-4e53cb13fd35/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;T-bet&quot;" alt="Search on this keyword" rel="nofollow">T-bet</a></li><li><a href="/action/doSearch?AllField=&quot;Mendelian+susceptibility+to+mycobacterial+disease&quot;" alt="Search on this keyword" rel="nofollow">Mendelian susceptibility to mycobacterial disease</a></li><li><a href="/action/doSearch?AllField=&quot;mycobacterium&quot;" alt="Search on this keyword" rel="nofollow">mycobacterium</a></li><li><a href="/action/doSearch?AllField=&quot;IFN-%CE%B3&quot;" alt="Search on this keyword" rel="nofollow">IFN-γ</a></li><li><a href="/action/doSearch?AllField=&quot;innate+lymphocyte&quot;" alt="Search on this keyword" rel="nofollow">innate lymphocyte</a></li><li><a href="/action/doSearch?AllField=&quot;innate-like+adaptive+lymphocyte&quot;" alt="Search on this keyword" rel="nofollow">innate-like adaptive lymphocyte</a></li><li><a href="/action/doSearch?AllField=&quot;inborn+errors+of+immunity&quot;" alt="Search on this keyword" rel="nofollow">inborn errors of immunity</a></li><li><a href="/action/doSearch?AllField=&quot;immunodeficiency&quot;" alt="Search on this keyword" rel="nofollow">immunodeficiency</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">In the course of primary infection, life-threatening disease in otherwise healthy children, adolescents, and even adults, can result from monogenic inborn errors of immunity (IEI), which display genetic heterogeneity and physiological homogeneity (<span class="dropBlock reference-citations"><a id="body-ref-sref20" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20" aria-controls="sref20">Casanova, 2015a</a><div class="dropBlock__holder" data-db-target-of="sref20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Casanova, J.L.</div><div><strong>Human genetic basis of interindividual variability in the course of infection</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E7118-E7127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1521644112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84952683321" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26621739/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1521644112&amp;pmid=26621739" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref21" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21" aria-controls="sref21">2015b</a><div class="dropBlock__holder" data-db-target-of="sref21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Casanova, J.L.</div><div><strong>Severe infectious diseases of childhood as monogenic inborn errors of immunity</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E7128-E7137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1521651112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84952685581" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26621750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1521651112&amp;pmid=26621750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref182" href="#" role="doc-biblioref" data-xml-rid="bib182" href-manipulated="true" aria-label="Reference 2020" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref182" aria-controls="sref182">2020</a><div class="dropBlock__holder" data-db-target-of="sref182"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Casanova, J.L.</div><div><strong>The human genetic determinism of life-threatening infectious diseases: genetic heterogeneity and physiological homogeneity?</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:681-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02184-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085371245" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32462426/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02184-w&amp;pmid=32462426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref181" href="#" role="doc-biblioref" data-xml-rid="bib181" href-manipulated="true" aria-label="Reference 2021" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref181" aria-controls="sref181">2021</a><div class="dropBlock__holder" data-db-target-of="sref181"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Casanova, J.L.</div><div><strong>Lethal Infectious Diseases as Inborn Errors of Immunity: Toward a Synthesis of the Germ and Genetic Theories.</strong></div><div><em>Annu. Rev. Pathol.</em> 2021; <strong>16</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-pathol-031920-101429" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-pathol-031920-101429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mendelian susceptibility to mycobacterial disease (MSMD) is characterized by a selective, inherited predisposition to clinical disease caused by weakly virulent mycobacteria, such as <i>Mycobacterium bovis</i> Bacille Calmette-Guérin (BCG) vaccines and environmental mycobacteria (EM) (<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Bustamante, 2020</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Bustamante, J.</div><div><strong>Mendelian susceptibility to mycobacterial disease: recent discoveries</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:993-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02120-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85079165775" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02120-y&amp;pmid=32025907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-1" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-1" aria-controls="sref84-1">Kerner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref84-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Kerner, G. ∙ Rosain, J. ∙ Guérin, A. ...</div><div><strong>Inherited human IFN-γ deficiency underlies mycobacterial disease</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2020; <strong>130</strong>:3158-3171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI135460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085294382" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32163377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI135460&amp;pmid=32163377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Patients are also vulnerable to <i>bona fide</i> tuberculosis. Patients with typical, “isolated” MSMD are rarely prone to other infectious agents, with the exception of <i>Salmonella</i> and occasionally other intra-macrophagic bacteria, fungi, and parasites. Patients with atypical, “syndromic” MSMD often display other clinical phenotypes, infectious or otherwise. MSMD, both “isolated” and “syndromic”, displays a high level of genetic heterogeneity, with causal mutations in 16 genes, and additional allelic heterogeneity, resulting in 31 different disorders. However, there is also physiological homogeneity, as all known genetic causes of MSMD affect interferon gamma (IFN-γ)-dependent immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref17-2" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-2" aria-controls="sref17-2">Bustamante, 2020</a><div class="dropBlock__holder" data-db-target-of="sref17-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Bustamante, J.</div><div><strong>Mendelian susceptibility to mycobacterial disease: recent discoveries</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:993-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02120-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85079165775" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02120-y&amp;pmid=32025907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Kerner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Kerner, G. ∙ Rosain, J. ∙ Guérin, A. ...</div><div><strong>Inherited human IFN-γ deficiency underlies mycobacterial disease</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2020; <strong>130</strong>:3158-3171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI135460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085294382" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32163377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI135460&amp;pmid=32163377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref136" href="#" role="doc-biblioref" data-xml-rid="bib136" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref136" aria-controls="sref136">Rosain et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref136"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">139.</div><div>Rosain, J. ∙ Kong, X.F. ∙ Martinez-Barricarte, R. ...</div><div><strong>Mendelian susceptibility to mycobacterial disease: 2014-2018 update</strong></div><div><em>Immunol. Cell Biol.</em> 2019; <strong>97</strong>:360-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/imcb.12210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85055711153" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30264912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fimcb.12210&amp;pmid=30264912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Mutations of <i>IFNG</i>, <i>IL12B</i>, <i>IL12RB1</i>, <i>IL12RB2</i>, <i>IL23R</i>, <i>TYK2</i>, <i>ISG15</i>, <i>RORC</i>, <i>IKBKG</i> (NEMO), <i>IRF8</i>, and <i>SPPL2A</i> impede IFN-γ production by innate and adaptive immune cells, whereas mutations of <i>IFNGR1</i>, <i>IFNGR2</i>, <i>STAT1</i>, <i>JAK1</i>, and <i>CYBB</i> impair cellular responses to IFN-γ (<a id="crosref0630" href="#figs1">Figure&nbsp;S1</a>A). The clinical penetrance and severity of MSMD depend strongly on genetic etiology and increase with decreasing levels of IFN-γ activity (<span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Dupuis et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Dupuis, S. ∙ Döffinger, R. ∙ Picard, C. ...</div><div><strong>Human interferon-gamma-mediated immunity is a genetically controlled continuous trait that determines the outcome of mycobacterial invasion</strong></div><div><em>Immunol. Rev.</em> 2000; <strong>178</strong>:129-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1034/j.1600-065X.2000.17810.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11213797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1034%2Fj.1600-065X.2000.17810.x&amp;pmid=11213797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Seven etiologies also result in an impairment of immunological circuits other than the IFN-γ circuit, accounting for “syndromic” MSMD in the corresponding patients (<span class="dropBlock reference-citations"><a id="body-ref-sref17-3" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-3" aria-controls="sref17-3">Bustamante, 2020</a><div class="dropBlock__holder" data-db-target-of="sref17-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Bustamante, J.</div><div><strong>Mendelian susceptibility to mycobacterial disease: recent discoveries</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:993-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02120-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85079165775" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02120-y&amp;pmid=32025907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Collectively, these studies revealed the crucial role of human IFN-γ in antimycobacterial immunity and its redundancy for immunity against many other pathogens.</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/5d665e17-0534-45e2-a940-67c414b718f4/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2020.10.046/asset/822188d1-872a-4a66-b301-156310189f10/main.assets/figs1.jpg" height="910" width="718" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/5d665e17-0534-45e2-a940-67c414b718f4/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Clinical and Genetic Features of a Patient with MSMD Carrying a Homozygous <i>TBX21</i> Variant Leading to the Substitution of Two Amino Acids and the Characterization of this Variant, Related to <a id="crosref2800" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0385" role="paragraph"><b>(A)</b> Schematic view of the 16 MSMD-causing genes within myeloid and lymphoid cells in the context of host antimycobacterial immunity. (<b>B</b>) We tested for serum IgG against various viral peptides or peptides derived from bacteria or other pathogens by phage immunoprecipitation-sequencing. Serum samples from an unrelated healthy three-year-old boy (CTL1), two healthy adults (CTL2 and CTL3), and a <i>TBX21</i> M/M P, and IgG-depleted serum and intravenous immunoglobulin (IVIg) were assayed. The heatmap (left) shows the counts of peptides significantly enriched for a given species with less than a continuous seven-residue subsequence, the estimated size of a linear epitope, in common. The species for which the patient had a score greater than our set threshold (&gt; 3) are shown. Count values above the set threshold are shown on a color scale from green (≥4) to red (≥40) and dark-red (≥80). (<b>C</b>) Genomic DNA from P (M/M) and his parents (WT/M) was subjected to Sanger sequencing to confirm the <i>TBX21</i> variant. (<b>D</b>) Comparison between the WT and Mut sequences of the region in which the variant is located within exon 1 of <i>TBX21</i>. An alignment of the WT and Mut sequences revealed a juxtaposed region with a sequence identical to that of the variant sequence. (<b>E</b>) Alignment of peptide fragments from human T-box domain-containing transcription factors. (<b>F</b>) pCMV6 plasmids containing EV, WT T-bet (WT), the K314R variant of T-bet (K314R), T-bet with a deletion of the T-box domain (T-del), and mutant T-bet (Mut) were used to transfect HEK293T cells. T-bet expression was characterized by immunoblotting with anti-T-bet 4B10 monoclonal Ab and anti-Flag Ab. Anti-GAPDH antibody was used as an endogenous control. (<b>G</b>) Mutations of individual residues in the T-bet cDNA (E156S, M157L, E156A, and E157A) were generated in pCMV6 plasmids by site-directed mutagenesis. The plasmids were used to transfect HEK293T cells for overexpression. T-bet levels were measured by immunoblotting with anti-T-bet 4B10 Ab and anti-Flag Ab. Anti-GAPDH Ab was used as endogenous control. (<b>H</b>) The plasmids described in (G) were used to transfect HEK293T cells, together with the <i>IFNG</i> reporter plasmids and <i>Renilla</i> luciferase pRL-SV40 plasmids. After 3&nbsp;days, the luciferase signal was assessed in a Dual-Glo assay. (<b>I</b>) Single-cell CRISPR/Cas9-edited NK-92 clones were stimulated with 50 pg/mL IL-12 and 10&nbsp;ng/mL IL-18 or were left untreated (NS). Intracellular IFN-γ production was determined by ICS flow cytometry. (<b>J</b>) T-bet-deficient or pLenti-Crispr-V2-EV-transduced NK-92 single-cell clones were transduced with retroviruses generated from pLZRS-ires-ΔNGFR plasmids containing no insert (EV), or the WT, K314R or Mut <i>TBX21</i> cDNA with a C-terminal Flag-tag. They were stimulated with 50 pg/mL IL-12 and 10&nbsp;ng/mL IL-18 or were left untreated. Intracellular IFN-γ production was determined by ICS flow cytometry. (<b>K</b>) Naive CD4<sup>+</sup> T&nbsp;cells were expanded under T<sub>H</sub>0 conditions, and were then transduced with retroviruses generated from pLZRS-ires-ΔNGFR plasmids containing no insert (EV), or the WT, or Mut <i>TBX21</i> cDNA with a C-terminal Flag-tag. Transduced cells were positive for the bicistronic marker CD271. Cells were surface-stained for CD271 and intracellularly stained for IFN-γ and TNF-α and subjected to flow cytometry analysis. (<b>L</b>) All variants (including the only homozygous variant – Mut, and 36 heterozygous variants, as indicated) from the HGID in-house database are represented on the schematic illustration. (<b>M</b>) All in-house <i>TBX21</i> variants, as in (L), were tested for transcriptional activity in the human <i>IFNG</i> luciferase reporter assay. pCMV6 plasmids containing variants were used to transfect cells, together with the human <i>IFNG</i> reporter plasmids or the control EV pGL4.10-EV luciferase reporter plasmid, and the pRL-SV40 <i>Renilla</i> plasmid for luciferase activity measurement.</div><div id="fspara0390" role="paragraph">In (H, I and M), bars represent the mean and the standard deviation. Dots represent individual samples or technical replicates. One-way ANOVA was used for analysis in (H). Two-way ANOVA was used for analysis in (I). A non-parametric analysis for comparison with pGL4.10-p<i>IFNG</i>&nbsp;+ pCMV6-WT <i>TBX21</i> transfection condition was performed in (M). In (H, I and M), <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001.</div></div></div></div></figcaption></figure></div><div id="p0035" role="paragraph">The cellular basis of MSMD in patients with impaired responses to IFN-γ involves mononuclear phagocytes. The ability of these cells to contain the ingested mycobacteria depends on their activation by IFN-γ (<span class="dropBlock reference-citations"><a id="body-ref-sref121" href="#" role="doc-biblioref" data-xml-rid="bib121" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref121" aria-controls="sref121">Nathan et&nbsp;al., 1983</a><div class="dropBlock__holder" data-db-target-of="sref121"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">123.</div><div>Nathan, C.F. ∙ Murray, H.W. ∙ Wiebe, M.E. ...</div><div><strong>Identification of interferon-gamma as the lymphokine that activates human macrophage oxidative metabolism and antimicrobial activity</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1983; <strong>158</strong>:670-689</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.158.3.670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0020502358" target="_blank" aria-label="Open Scopus in new tab">Scopus (1291)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6411853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.158.3.670&amp;pmid=6411853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The cellular basis of MSMD in patients with impaired IFN-γ production is poorly understood, as most types of lymphocytes can produce IFN-γ (<span class="dropBlock reference-citations"><a id="body-ref-sref141-1" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-1" aria-controls="sref141-1">Schoenborn and Wilson, 2007</a><div class="dropBlock__holder" data-db-target-of="sref141-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Schoenborn, J.R. ∙ Wilson, C.B.</div><div><strong>Regulation of interferon-γ during innate and adaptive immune responses</strong></div><div><em>Adv. Immunol.</em> 2007; <strong>96</strong>:41-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0065-2776(07)96002-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-35549003579" target="_blank" aria-label="Open Scopus in new tab">Scopus (968)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17981204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0065-2776%2807%2996002-2&amp;pmid=17981204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Some genetic etiologies of MSMD affect some lymphocyte subsets more than others. ISG15 deficiency preferentially impairs the production of IFN-γ by natural killer (NK) cells (<span class="dropBlock reference-citations"><a id="body-ref-sref13" href="#" role="doc-biblioref" data-xml-rid="bib13" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref13" aria-controls="sref13">Bogunovic et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Bogunovic, D. ∙ Byun, M. ∙ Durfee, L.A. ...</div><div><strong>Mycobacterial disease and impaired IFN-gamma immunity in humans with inherited ISG15 deficiency</strong></div><div><em>Science.</em> 2012; <strong>337</strong>:1684-1688</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1224026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866748115" target="_blank" aria-label="Open Scopus in new tab">Scopus (279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22859821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1224026&amp;pmid=22859821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref174" href="#" role="doc-biblioref" data-xml-rid="bib174" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref174" aria-controls="sref174">Zhang et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref174"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">177.</div><div>Zhang, X. ∙ Bogunovic, D. ∙ Payelle-Brogard, B. ...</div><div><strong>Human intracellular ISG15 prevents interferon-α/β over-amplification and auto-inflammation</strong></div><div><em>Nature.</em> 2015; <strong>517</strong>:89-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13801" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84922880395" target="_blank" aria-label="Open Scopus in new tab">Scopus (241)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25307056/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13801&amp;pmid=25307056" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interleukin (IL)-12Rβ2 deficiency preferentially impairs the production of IFN-γ by NK, B, γδ T, classic αβ T&nbsp;cells, type 1 innate lymphoid cells (ILC1s), and ILC2s, whereas IL-23R deficiency preferentially impairs the production of this cytokine by invariant NKT (iNKT) and mucosal-associated invariant T (MAIT) cells, and both IL-12Rβ1 and TYK2 deficiencies impair both the IL-12- and IL-23-dependent subsets (<span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Boisson-Dupuis et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Boisson-Dupuis, S. ∙ Ramirez-Alejo, N. ∙ Li, Z. ...</div><div><strong>Tuberculosis and impaired IL-23-dependent IFN-γuberculosis and impaired IL-23-dependent IFN-mmunity in humans wit</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaau8714</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau8714" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85058915562" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30578352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau8714&amp;pmid=30578352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref114" href="#" role="doc-biblioref" data-xml-rid="bib114" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref114" aria-controls="sref114">Martínez-Barricarte et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref114"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">116.</div><div>Martínez-Barricarte, R. ∙ Markle, J.G. ∙ Ma, C.S. ...</div><div><strong>Human IFN-γ immunity to mycobacteria is governed by both IL-12 and IL-23</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaau6759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau6759" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85057006182" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30578351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau6759&amp;pmid=30578351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). SPPL2a and IRF8 deficiencies selectively impair the production of IFN-γ by CD4<sup>+</sup> CCR6<sup>+</sup> T<sub>H</sub>1 (T<sub>H</sub>1<sup>∗</sup>) cells (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Hambleton et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Hambleton, S. ∙ Salem, S. ∙ Bustamante, J. ...</div><div><strong>IRF8 mutations and human dendritic-cell immunodeficiency</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2011; <strong>365</strong>:127-138</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1100066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79960219807" target="_blank" aria-label="Open Scopus in new tab">Scopus (401)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21524210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1100066&amp;pmid=21524210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref88" href="#" role="doc-biblioref" data-xml-rid="bib88" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref88" aria-controls="sref88">Kong et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref88"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Kong, X.F. ∙ Martinez-Barricarte, R. ∙ Kennedy, J. ...</div><div><strong>Disruption of an antimycobacterial circuit between dendritic and helper T&nbsp;cells in human SPPL2a deficiency</strong></div><div><em>Nat. Immunol.</em> 2018; <strong>19</strong>:973-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0178-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85051985294" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30127434/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0178-z&amp;pmid=30127434" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), a T<sub>H</sub>1 cell subset enriched in <i>Mycobacterium</i>-specific effector cells, whereas CCR6<sup>–</sup> T<sub>H</sub>1 cells do not respond to mycobacteria (<span class="dropBlock reference-citations"><a id="body-ref-sref2-1" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-1" aria-controls="sref2-1">Acosta-Rodriguez et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref2-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Acosta-Rodriguez, E.V. ∙ Rivino, L. ∙ Geginat, J. ...</div><div><strong>Surface phenotype and antigenic specificity of human interleukin 17-producing T helper memory cells</strong></div><div><em>Nat. Immunol.</em> 2007; <strong>8</strong>:639-646</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34249079176" target="_blank" aria-label="Open Scopus in new tab">Scopus (1293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17486092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1467&amp;pmid=17486092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). RORγ/RORγT deficiency impairs the development of iNKT and MAIT cells and also compromises the production of IFN-γ by γδ T and αβ T<sub>H</sub>1<sup>∗</sup> cells (<span class="dropBlock reference-citations"><a id="body-ref-sref124-1" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-1" aria-controls="sref124-1">Okada et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref124-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Okada, S. ∙ Markle, J.G. ∙ Deenick, E.K. ...</div><div><strong>Impairment of immunity to Candida and Mycobacterium in humans with bi-allelic RORC mutations</strong></div><div><em>Science.</em> 2015; <strong>349</strong>:606-613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa4282" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84939154723" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26160376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa4282&amp;pmid=26160376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Interestingly, although the lack of both αβ T and γδ T&nbsp;cells in patients with severe combined immunodeficiency (SCID) underlies BCG disease (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Casanova et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Casanova, J.L. ∙ Jouanguy, E. ∙ Lamhamedi, S. ...</div><div><strong>Immunological conditions of children with BCG disseminated infection</strong></div><div><em>Lancet.</em> 1995; <strong>346</strong>:581</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.5555%2Furi%3Apii%3AS0140673695914218&amp;site=lancet-site" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7658805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5555%2Furi%3Apii%3AS0140673695914218&amp;pmid=7658805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), most, if not all other deficits of antigen-specific αβ T&nbsp;cell responses, whether affecting only CD4<sup>+</sup> or CD8<sup>+</sup> T&nbsp;cells, such as MHC class II and MHC class I deficiencies, respectively, typically do not (<span class="dropBlock reference-citations"><a id="body-ref-sref4" href="#" role="doc-biblioref" data-xml-rid="bib4" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref4" aria-controls="sref4">Aluri et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Aluri, J. ∙ Gupta, M. ∙ Dalvi, A. ...</div><div><strong>Clinical, immunological, and molecular findings in five patients with Major histocompatibility complex class II deficiency from India</strong></div><div><em>Front. Immunol.</em> 2018; <strong>9</strong>:188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2018.00188" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85042098241" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29527204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2018.00188&amp;pmid=29527204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Ardeniz et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Ardeniz, Ö. ∙ Unger, S. ∙ Onay, H. ...</div><div><strong>β2-Microglobulin deficiency causes a complex immunodeficiency of the innate and adaptive immune system</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2015; <strong>136</strong>:392-401</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.12.1937&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.12.1937&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84938746200" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25702838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2014.12.1937&amp;pmid=25702838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Dimitrova et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Dimitrova, D. ∙ Ong, P.Y. ∙ O’Gorman, M.R.G. ...</div><div><strong>Major histocompatibility complex class II deficiency complicated by Mycobacterium avium complex in a boy of mixed ethnicity</strong></div><div><em>J.&nbsp;Clin. Immunol.</em> 2014; <strong>34</strong>:677-680</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10875-014-0048-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84939872978" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24789686/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10875-014-0048-x&amp;pmid=24789686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Elhasid and Etzioni, 1996</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Elhasid, R. ∙ Etzioni, A.</div><div><strong>Major histocompatibility complex class II deficiency: a clinical review</strong></div><div><em>Blood Rev.</em> 1996; <strong>10</strong>:242-248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0268-960X(96)90008-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0030465562" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9012922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0268-960X%2896%2990008-9&amp;pmid=9012922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Hanalioglu et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Hanalioglu, D. ∙ Ayvaz, D.C. ∙ Ozgur, T.T. ...</div><div><strong>A novel mutation in TAP1 gene leading to MHC class I deficiency: Report of two cases and review of the literature</strong></div><div><em>Clin. Immunol.</em> 2017; <strong>178</strong>:74-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.clim.2017.01.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85013123103" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28161407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.clim.2017.01.011&amp;pmid=28161407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref62" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62" aria-controls="sref62">Hanna and Etzioni, 2014</a><div class="dropBlock__holder" data-db-target-of="sref62"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Hanna, S. ∙ Etzioni, A.</div><div><strong>MHC class I and II deficiencies</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2014; <strong>134</strong>:269-275</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.06.001&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.06.001&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25001848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2014.06.001&amp;pmid=25001848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref36" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36" aria-controls="sref36">De La Salle et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref36"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>De La Salle, H. ∙ Hanau, D. ∙ Fricker, D. ...</div><div><strong>Homozygous human TAP peptide transporter mutation in HLA class I deficiency</strong></div><div><em>Science.</em> 1994; <strong>265</strong>:237-241</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7517574" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7517574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7517574&amp;pmid=7517574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Lum et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Lum, S.H. ∙ Neven, B. ∙ Slatter, M.A. ...</div><div><strong>Hematopoietic Cell Transplantation for MHC Class II Deficiency</strong></div><div><em>Front Pediatr.</em> 2019; <strong>7</strong>:516</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fped.2019.00516" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85077320039" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31921728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffped.2019.00516&amp;pmid=31921728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref109" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" aria-label="Reference 2020" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109" aria-controls="sref109">2020</a><div class="dropBlock__holder" data-db-target-of="sref109"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Lum, S.H. ∙ Anderson, C. ∙ McNaughton, P. ...</div><div><strong>Improved transplant survival and long-term disease outcome in children with MHC class II deficiency</strong></div><div><em>Blood.</em> 2020; <strong>135</strong>:954-973</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood.2019002690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31932845/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood.2019002690&amp;pmid=31932845" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref120" href="#" role="doc-biblioref" data-xml-rid="bib120" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref120" aria-controls="sref120">Morgan et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref120"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">122.</div><div>Morgan, N.V. ∙ Goddard, S. ∙ Cardno, T.S. ...</div><div><strong>Mutation in the TCRα subunit constant gene (TRAC) leads to a human immunodeficiency disorder characterized by a lack of TCRαβ+ T&nbsp;cells</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2011; <strong>121</strong>:695-702</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI41931" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79551544623" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21206088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI41931&amp;pmid=21206088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref138" href="#" role="doc-biblioref" data-xml-rid="bib138" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref138" aria-controls="sref138">Saleem et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref138"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">141.</div><div>Saleem, M.A. ∙ Arkwright, P.D. ∙ Davies, E.G. ...</div><div><strong>Clinical course of patients with major histocompatibility complex class II deficiency</strong></div><div><em>Arch. Dis. Child.</em> 2000; <strong>83</strong>:356-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/adc.83.4.356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0033807808" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10999878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fadc.83.4.356&amp;pmid=10999878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref180" href="#" role="doc-biblioref" data-xml-rid="bib180" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref180" aria-controls="sref180">Zimmer et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref180"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">183.</div><div>Zimmer, J. ∙ Andrès, E. ∙ Donato, L. ...</div><div><strong>Clinical and immunological aspects of HLA class I deficiency</strong></div><div><em>QJM.</em> 2005; <strong>98</strong>:719-727</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/qjmed/hci112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-27644538518" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16087697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fqjmed%2Fhci112&amp;pmid=16087697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, selective deficiencies of NK or iNKT cells do not confer a predisposition to mycobacterial disease (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Casey et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Casey, J.P. ∙ Nobbs, M. ∙ McGettigan, P. ...</div><div><strong>Recessive mutations in MCM4/PRKDC cause a novel syndrome involving a primary immunodeficiency and a disorder of DNA repair</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2012; <strong>49</strong>:242-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2012-100803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84864097350" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22499342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2012-100803&amp;pmid=22499342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Cottineau et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Cottineau, J. ∙ Kottemann, M.C. ∙ Lach, F.P. ...</div><div><strong>Inherited GINS1 deficiency underlies growth retardation along with neutropenia and NK cell deficiency</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2017; <strong>127</strong>:1991-2006</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI90727" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85018993845" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28414293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI90727&amp;pmid=28414293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref68" href="#" role="doc-biblioref" data-xml-rid="bib68" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref68" aria-controls="sref68">Hughes et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref68"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Hughes, C.R. ∙ Guasti, L. ∙ Meimaridou, E. ...</div><div><strong>MCM4 mutation causes adrenal failure, short stature, and natural killer cell deficiency in humans</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2012; <strong>122</strong>:814-820</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI60224" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857875264" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354170/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI60224&amp;pmid=22354170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Latour, 2007</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Latour, S.</div><div><strong>Natural killer T&nbsp;cells and X-linked lymphoproliferative syndrome</strong></div><div><em>Curr. Opin. Allergy Clin. Immunol.</em> 2007; <strong>7</strong>:510-514</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/ACI.0b013e3282f1bad6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17989527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FACI.0b013e3282f1bad6&amp;pmid=17989527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Locci et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Locci, M. ∙ Draghici, E. ∙ Marangoni, F. ...</div><div><strong>The Wiskott-Aldrich syndrome protein is required for iNKT cell maturation and function</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2009; <strong>206</strong>:735-742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20081773" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-65549150035" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19307326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20081773&amp;pmid=19307326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref154" href="#" role="doc-biblioref" data-xml-rid="bib154" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref154" aria-controls="sref154">Tangye et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref154"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">157.</div><div>Tangye, S.G. ∙ Palendira, U. ∙ Edwards, E.S.J.</div><div><strong>Human immunity against EBV-lessons from the clinic</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:269-283</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161846" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85012292927" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28108590/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161846&amp;pmid=28108590" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The nature of the IFN-γ-producing innate, innate-like adaptive, and purely adaptive lymphocyte subsets indispensable for antimycobacterial immunity, either alone or in combination, therefore remains largely unknown. No genetic cause has yet been identified for half the MSMD patients. We therefore sought to discover a new genetic etiology of MSMD that would expand the molecular circuit controlling human IFN-γ immunity while better delineating the cellular network involved.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Identification of an MSMD Patient Homozygous for an Indel Variant of <i>TBX21</i></h3><div id="p0040" role="paragraph">We studied a 4-year-old boy (“P,” II.2) born to first-cousin Moroccan parents (<a id="crosref0790" href="#fig1">Figure&nbsp;1</a>A). He suffered from disseminated BCG disease (BCG-osis) following vaccination at the age of 3&nbsp;months. He also had persistent reactive airway disease (RAD) but was otherwise healthy (see <a id="crosref0795" href="#sec-4">STAR Methods</a> - Case Report). He did not suffer from any other severe clinical infectious diseases despite documented (VirScan) infection with various viruses, including Epstein-Barr virus (EBV), human cytomegalovirus (CMV), roseola virus, adenoviruses A, B, C, and D, influenza virus A, rhinovirus A, and bacteria, such as <i>Streptococcus pneumoniae</i> and <i>Staphylococcus aureus</i> (<a id="crosref0800" href="#figs1">Figure&nbsp;S1</a>B). We hypothesized that P had an autosomal recessive (AR) defect. We performed whole-exome sequencing (WES) on P, his unaffected brother, and both parents. Genome-wide linkage (GWL) analysis revealed 32 linked regions (LOD score &gt;1.3 and size &gt;500 kb) under a model of complete penetrance (<a id="crosref0805" href="#mmc1">Data S1</a>). In these linked regions, there were 15 rare homozygous non-synonymous or essential splicing variants of 15 different genes (minor allele frequency, MAF &lt;0.003 in gnomAD v2.1 and 1000 Genomes Project, including for each major ancestry) with a combined annotation-dependent depletion (CADD) score above their mutation significance cutoffs (MSCs) (<span class="dropBlock reference-citations"><a id="body-ref-sref7" href="#" role="doc-biblioref" data-xml-rid="bib7" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref7" aria-controls="sref7">Auton et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (5078)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref74-1" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-1" aria-controls="sref74-1">Itan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Itan, Y. ∙ Shang, L. ∙ Boisson, B. ...</div><div><strong>The mutation significance cutoff: gene-level thresholds for variant predictions</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:109-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3739" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84961309083" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26820543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3739&amp;pmid=26820543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref83-1" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-1" aria-controls="sref83-1">Karczewski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref83-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-1" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-1" aria-controls="sref85-1">Kircher et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref85-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Kircher, M. ∙ Witten, D.M. ∙ Jain, P. ...</div><div><strong>A general framework for estimating the relative pathogenicity of human genetic variants</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:310-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84895858942" target="_blank" aria-label="Open Scopus in new tab">Scopus (2683)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24487276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2892&amp;pmid=24487276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref175-1" href="#" role="doc-biblioref" data-xml-rid="bib175" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref175-1" aria-controls="sref175-1">Zhang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref175-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">178.</div><div>Zhang, P. ∙ Bigio, B. ∙ Rapaport, F. ...</div><div><strong>PopViz: a webserver for visualizing minor allele frequencies and damage prediction scores of human genetic variations</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:4307-4309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85058604691" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30535305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty536&amp;pmid=30535305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0835" href="#mmc1">Table S1</a>). After the exclusion of genes with other predicted loss-of-function (LOF) variants with a frequency greater than 0.5% in gnomAD and of genes not expressed in leukocytes, only four candidate genes (<i>TBX21</i>, <i>PSMD9</i>, <i>ARHGAP27</i>, and <i>ERN1</i>) remained. The c.466_471delGAGATGinsAGTTTA insertion and/or deletion (indel) variant of <i>TBX21</i> (T-box protein 21, or T-box, expressed in T&nbsp;cells, T-bet) was the candidate variant predicted to be the most damaging (<span class="dropBlock reference-citations"><a id="body-ref-sref85-2" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-2" aria-controls="sref85-2">Kircher et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref85-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Kircher, M. ∙ Witten, D.M. ∙ Jain, P. ...</div><div><strong>A general framework for estimating the relative pathogenicity of human genetic variants</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:310-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84895858942" target="_blank" aria-label="Open Scopus in new tab">Scopus (2683)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24487276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2892&amp;pmid=24487276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, based on connectivity to <i>IFNG</i>, the central gene of the entire network of all known MSMD-causing genes (<span class="dropBlock reference-citations"><a id="body-ref-sref71" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71" aria-controls="sref71">Itan et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref71"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Itan, Y. ∙ Zhang, S.-Y. ∙ Vogt, G. ...</div><div><strong>The human gene connectome as a map of short cuts for morbid allele discovery</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:5558-5563</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1218167110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84875826166" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23509278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1218167110&amp;pmid=23509278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref72" href="#" role="doc-biblioref" data-xml-rid="bib72" href-manipulated="true" aria-label="Reference 2014" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref72" aria-controls="sref72">2014</a><div class="dropBlock__holder" data-db-target-of="sref72"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Itan, Y. ∙ Mazel, M. ∙ Mazel, B. ...</div><div><strong>HGCS: an online tool for prioritizing disease-causing gene variants by biological distance</strong></div><div><em>BMC Genomics.</em> 2014; <strong>15</strong>:256</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-15-256" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84899103596" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24694260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-15-256&amp;pmid=24694260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>TBX21</i> was the most plausible candidate gene (<a id="crosref0855" href="#mmc1">Table S1</a>). T-bet is a transcription factor that governs the development or function of several IFN-γ-producing lymphocytes in mice, including T<sub>H</sub>1 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Lazarevic et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Lazarevic, V. ∙ Glimcher, L.H. ∙ Lord, G.M.</div><div><strong>T-bet: a bridge between innate and adaptive immunity</strong></div><div><em>Nat. Rev. Immunol.</em> 2013; <strong>13</strong>:777-789</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri3536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84886790466" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24113868/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri3536&amp;pmid=24113868" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref149-1" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149-1" aria-controls="sref149-1">Szabo et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref149-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Szabo, S.J. ∙ Kim, S.T. ∙ Costa, G.L. ...</div><div><strong>A novel transcription factor, T-bet, directs Th1 lineage commitment</strong></div><div><em>Cell.</em> 2000; <strong>100</strong>:655-669</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10761931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980702-3&amp;pmid=10761931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref150-1" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" aria-label="Reference 2002" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-1" aria-controls="sref150-1">2002</a><div class="dropBlock__holder" data-db-target-of="sref150-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Szabo, S.J. ∙ Sullivan, B.M. ∙ Stemmann, C. ...</div><div><strong>Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:338-342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059462" target="_blank" aria-label="Open Scopus in new tab">Scopus (920)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065543&amp;pmid=11786644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These findings suggested that homozygosity for the rare indel variant c.466_471delGAGATGinsAGTTTA of <i>TBX21</i> causes MSMD. We investigated this variant according to the guidelines for genetic studies of single patients (<span class="dropBlock reference-citations"><a id="body-ref-sref25-1" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-1" aria-controls="sref25-1">Casanova et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref25-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Casanova, J.-L. ∙ Conley, M.E. ∙ Seligman, S.J. ...</div><div><strong>Guidelines for genetic studies in single patients: lessons from primary immunodeficiencies</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2014; <strong>211</strong>:2137-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20140520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84908247576" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25311508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20140520&amp;pmid=25311508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sanger sequencing confirmed that P was homozygous for the indel variant of exon 1 of <i>TBX21</i>, whereas his unaffected brother was homozygous wild-type (WT/WT) and both parents were heterozygous (WT/M) (<a id="crosref0880" href="#figs1">Figure&nbsp;S1</a>C). A closely juxtaposed 12-nucleotide (nt) region identical to this variant sequence was detected 8-nt upstream of the variant and may have served as a template for its generation (<a id="crosref0885" href="#figs1">Figure&nbsp;S1</a>D). The variant did not alter the exon 1-exon 2 junction of the <i>TBX21</i> mRNA in EBV-transformed B (EBV-B) cells or peripheral blood mononuclear cells (PBMCs) (<a id="crosref0890" href="#mmc1">Data S1</a>). The variant present in P thus resulted in the replacement of E156 and M157, two amino acids that are highly conserved across different species and among other paralogs of T-box transcription factors, with S156 and L157 (<a id="crosref0895" href="#fig1">Figure&nbsp;1</a>B and <a id="crosref0900" href="#figs1">Figure&nbsp;S1</a>E).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/f4effde4-1ebd-4886-a33a-bc94f589a951/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2020.10.046/asset/0c9d586f-71c8-4bbd-8854-0d2e8bfec582/main.assets/gr1.jpg" height="949" width="720" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/f4effde4-1ebd-4886-a33a-bc94f589a951/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Discovery of an MSMD Patient with a Homozygous <i>TBX21</i> Variant and the Corresponding Molecular Characterization</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) Pedigree of a consanguineous family with the mutant <i>TBX21</i> allele.</div><div id="fspara0020" role="paragraph">(B) Schematic representation of the mutation.</div><div id="fspara0025" role="paragraph">(C) Nuclear and cytoplasmic fractions from HEK293T cells transfected with indicated <i>TBX21</i> cDNA-containing vectors subjected to immunoblotting against indicated proteins.</div><div id="fspara0030" role="paragraph">(D) EMSA on nuclear extracts of HEK293T cells transfected with empty vector (EV), WT, or Mut <i>TBX21</i> alleles.</div><div id="fspara0035" role="paragraph">(E) TBRE-reporter luciferase assay testing WT or Mut T-bet in HEK293T cells.</div><div id="fspara0040" role="paragraph">(F) Human <i>IFNG</i> reporter luciferase assay testing HEK293T cells transfected with WT or Mut <i>TBX21</i> cDNA-containing vectors, at the indicated concentrations.</div><div id="fspara0045" role="paragraph">(G) IFN-γ production in response to stimulation with IL-12 and IL-18 in T-bet knockout (KO) NK-92 cells.</div><div id="fspara0050" role="paragraph">(H) IFN-γ production in response to stimulation with IL-12 and IL-18 in T-bet KO NK-92 cells complemented with EV, WT, K314R, or Mut <i>TBX21</i>-containing plasmids.</div><div id="fspara0055" role="paragraph">(I) Intracellular IFN-γ production in response to phorbol 12-myristate 13-acetate and ionomycin (P/I) in expanding T<sub>H</sub>0 cells transduced with EV, WT, or Mut <i>TBX21</i> cDNA.</div><div id="fspara0060" role="paragraph">(J) IFN-γ production from cells transduced as in (I) in response to stimulation with anti-CD3/CD28 Ab.</div><div id="fspara0065" role="paragraph">(K) Graph of CADD score against minor allele frequency (MAF), for <i>TBX21</i> variants reported in the gnomAD. Mutation significance cutoff (MSC) was shown.</div><div id="fspara0070" role="paragraph">(L) TBRE-reporter luciferase assay testing indicated variants of <i>TBX21</i> as in (E).</div><div id="fspara0075" role="paragraph">In (E)–(J) and (L), bars represent the mean and standard deviation. Dots represent individual samples or technical replicates. Two-way ANOVA was used for analysis in (E)–(I). One-way ANOVA was used for analysis in (J) and (L). In (E)–(J) and (L), <sup>∗</sup>p&nbsp;&lt; 0.05, <sup>∗∗</sup>p&nbsp;&lt; 0.01, <sup>∗∗∗</sup>p&nbsp;&lt; 0.001, <sup>∗∗∗∗</sup>p&nbsp;&lt; 0.0001; ns, not significant.</div><div id="fspara0080" role="paragraph">See also <a id="crosref2725" href="#figs1">Figure&nbsp;S1</a>, <a id="crosref2730" href="#mmc1">Table S1</a>, and <a id="crosref2735" href="#mmc1">Data S1</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-2"><h3>Overexpressed Mutant T-bet Is LOF</h3><div id="p0045" role="paragraph">We investigated the expression of the mutant allele (Mut), by overexpressing an empty vector (EV), or vectors containing the WT or Mut allele, or negative controls with the T-box domain deleted (T-del) or K314R, the human ortholog of a known LOF mouse mutant, in human embryonic kidney (HEK) 293T cells (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Jang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Jang, E.J. ∙ Park, H.R. ∙ Hong, J.H. ...</div><div><strong>Lysine 313 of T-box is crucial for modulation of protein stability, DNA binding, and threonine phosphorylation of T-bet</strong></div><div><em>J.&nbsp;Immunol.</em> 2013; <strong>190</strong>:5764-5770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1203403" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84878073297" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23616576/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1203403&amp;pmid=23616576" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The production and nuclear translocation of Mut T-bet were impaired (<a id="crosref0910" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0915" href="#figs1">Figure&nbsp;S1</a>F). An assessment of its binding to consensus T-box regulatory elements (TBRE) in DNA revealed that nuclear proteins from WT-transfected cells bound the WT-TBRE, but not the Mut-TBRE. Furthermore, this specific complex was super-shifted by an anti-T-bet antibody (Ab) and inhibited by a competitor probe (CP). However, Mut T-bet did not bind WT-TBRE DNA (<a id="crosref0920" href="#fig1">Figure&nbsp;1</a>D). We also assessed the ability of WT and Mut T-bet to induce a luciferase transgene under the control of the TBRE or human <i>IFNG</i> proximal promoter (<span class="dropBlock reference-citations"><a id="body-ref-sref76-1" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-1" aria-controls="sref76-1">Janesick et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref76-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Janesick, A. ∙ Shiotsugu, J. ∙ Taketani, M. ...</div><div><strong>RIPPLY3 is a retinoic acid-inducible repressor required for setting the borders of the pre-placodal ectoderm</strong></div><div><em>Development.</em> 2012; <strong>139</strong>:1213-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.071456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857300258" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.071456&amp;pmid=22354841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref144-1" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144-1" aria-controls="sref144-1">Soutto et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref144-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Soutto, M. ∙ Zhang, F. ∙ Enerson, B. ...</div><div><strong>A minimal IFN-gamma promoter confers Th1 selective expression</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:4205-4212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.8.4205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12370350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.8.4205&amp;pmid=12370350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). WT T-bet induced high levels of luciferase activity with WT-TBRE but not with Mut-TBRE. Mut T-bet and the negative controls (T-del and K314R) did not induce luciferase activity (<a id="crosref0935" href="#fig1">Figure&nbsp;1</a>E). Mut T-bet was also LOF for transactivation of the <i>IFNG</i> promoter, whereas the negative control, K314R, was markedly hypomorphic (<a id="crosref0940" href="#fig1">Figure&nbsp;1</a>F). We investigated the amino-acid substitution responsible for the abolition of transcriptional activity. We tested the effects of the WT and Mut forms of T-bet and of T-bet forms with single-residue substitutions (E156S and M157L) or alanine substitutions (E156A and M157A) on T-bet protein production and transcriptional activity. The loss of E156 (E156S and E156A) abolished transcriptional activity, but the production of the T-bet protein was unaffected. By contrast, a loss of methionine residues (M157L and M157A) preserved transcriptional activity but decreased the levels of T-bet protein (<a id="crosref0945" href="#figs1">Figures S1</a>G and S1H). T-bet is required for optimal IFN-γ production in NK, ILCs, γδ, and CD4<sup>+</sup> T&nbsp;cells in mice and is sufficient for IFN-γ production in NK and CD4<sup>+</sup> T&nbsp;cells in humans (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Chen et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Chen, L. ∙ He, W. ∙ Kim, S.T. ...</div><div><strong>Epigenetic and transcriptional programs lead to default IFN-γ production by gammadelta T&nbsp;cells</strong></div><div><em>J.&nbsp;Immunol.</em> 2007; <strong>178</strong>:2730-2736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.178.5.2730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17312115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.178.5.2730&amp;pmid=17312115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Lazarevic et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Lazarevic, V. ∙ Glimcher, L.H. ∙ Lord, G.M.</div><div><strong>T-bet: a bridge between innate and adaptive immunity</strong></div><div><em>Nat. Rev. Immunol.</em> 2013; <strong>13</strong>:777-789</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri3536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84886790466" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24113868/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri3536&amp;pmid=24113868" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref129" href="#" role="doc-biblioref" data-xml-rid="bib129" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref129" aria-controls="sref129">Powell et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref129"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">132.</div><div>Powell, N. ∙ Walker, A.W. ∙ Stolarczyk, E. ...</div><div><strong>The transcription factor T-bet regulates intestinal inflammation mediated by interleukin-7 receptor+ innate lymphoid cells</strong></div><div><em>Immunity.</em> 2012; <strong>37</strong>:674-684</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2012.09.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2012.09.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84867856710" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23063332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2012.09.008&amp;pmid=23063332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref150-2" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-2" aria-controls="sref150-2">Szabo et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref150-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Szabo, S.J. ∙ Sullivan, B.M. ∙ Stemmann, C. ...</div><div><strong>Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:338-342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059462" target="_blank" aria-label="Open Scopus in new tab">Scopus (920)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065543&amp;pmid=11786644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref172" href="#" role="doc-biblioref" data-xml-rid="bib172" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref172" aria-controls="sref172">Yu et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref172"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">175.</div><div>Yu, J. ∙ Wei, M. ∙ Becknell, B. ...</div><div><strong>Pro- and antiinflammatory cytokine signaling: reciprocal antagonism regulates interferon-gamma production by human natural killer cells</strong></div><div><em>Immunity.</em> 2006; <strong>24</strong>:575-590</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2006.03.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2006.03.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33646563695" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16713975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2006.03.016&amp;pmid=16713975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We investigated the impact of the T-bet variant on the induction of <i>IFNG</i> expression, by generating CRISPR/Cas9 gene-edited human NK-92 cell lines lacking <i>TBX21</i> (<a id="crosref0975" href="#mmc1">Data S1</a>). Upon stimulation with IL-12&nbsp;+ IL-18, these <i>TBX21</i> knockout (KO) NK-92 cells displayed a strong impairment of IFN-γ production (<a id="crosref0980" href="#fig1">Figure&nbsp;1</a>G; <a id="crosref0985" href="#figs1">Figure&nbsp;S1</a>I). We re-introduced the WT or Mut <i>TBX21</i>-containing plasmid into <i>TBX21</i> KO NK-92 cells. We found that the WT <i>TBX21</i> rescued IFN-γ production, whereas the Mut <i>TBX21</i> did not (<a id="crosref0990" href="#fig1">Figure&nbsp;1</a>H; <a id="crosref0995" href="#figs1">Figure&nbsp;S1</a>J). Finally, the overexpression of WT but not of Mut T-bet increased IFN-γ levels to values above those for endogenous production in expanding naive CD4<sup>+</sup> T&nbsp;cells from healthy donors (<a id="crosref1000" href="#fig1">Figures 1</a>I and 1J; <a id="crosref1005" href="#figs1">Figure&nbsp;S1</a>K). Thus, overexpression of Mut T-bet abolished DNA binding, and the mutant protein had no transactivation activity and could not induce IFN-γ production in an NK cell line or CD4<sup>+</sup> T&nbsp;cells. It can, therefore, reasonably be considered a LOF allele.</div></section><section id="sec-2-3"><h3><i>TBX21</i> Variants in the General Population Are Functional</h3><div id="p0050" role="paragraph">The <i>TBX21</i> indel variant in P was not found in the gnomAD v2.1.1, v3, Bravo, or Middle Eastern cohort databases, or our in-house database of more than 8,000 exomes, including &gt;1,000 for individuals of North African origin (<span class="dropBlock reference-citations"><a id="body-ref-sref83-2" href="#" role="doc-biblioref" data-xml-rid="bib83" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref83-2" aria-controls="sref83-2">Karczewski et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref83-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref142" href="#" role="doc-biblioref" data-xml-rid="bib142" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref142" aria-controls="sref142">Scott et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref142"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">145.</div><div>Scott, E.M. ∙ Halees, A. ∙ Itan, Y. ..., Greater Middle East Variome Consortium</div><div><strong>Characterization of Greater Middle Eastern genetic variation for enhanced disease gene discovery</strong></div><div><em>Nat. Genet.</em> 2016; <strong>48</strong>:1071-1076</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3592" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84978705937" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27428751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3592&amp;pmid=27428751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref151" href="#" role="doc-biblioref" data-xml-rid="bib151" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref151" aria-controls="sref151">Taliun et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref151"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">154.</div><div>Taliun, D. ∙ Harris, D.N. ∙ Kessler, M.D. ...</div><div><strong>Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program</strong></div><div><em>BioRxiv.</em> 2019; 563866</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.TaliunD.N.HarrisM.D.KesslerJ.CarlsonZ.A.SzpiechR.TorresS.A.G.TaliunA.CorveloS.M.GogartenH.M.KangSequencing+of+53%2C831+diverse+genomes+from+the+NHLBI+TOPMed+ProgramBioRxiv2019563866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The CADD score of 34 obtained for this allele is well above the MSC of 3.313 (<a id="crosref1025" href="#fig1">Figure&nbsp;1</a>K; <span class="dropBlock reference-citations"><a id="body-ref-sref74-2" href="#" role="doc-biblioref" data-xml-rid="bib74" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref74-2" aria-controls="sref74-2">Itan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref74-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Itan, Y. ∙ Shang, L. ∙ Boisson, B. ...</div><div><strong>The mutation significance cutoff: gene-level thresholds for variant predictions</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:109-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3739" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84961309083" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26820543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3739&amp;pmid=26820543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref85-3" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85-3" aria-controls="sref85-3">Kircher et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref85-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Kircher, M. ∙ Witten, D.M. ∙ Jain, P. ...</div><div><strong>A general framework for estimating the relative pathogenicity of human genetic variants</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:310-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84895858942" target="_blank" aria-label="Open Scopus in new tab">Scopus (2683)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24487276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2892&amp;pmid=24487276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref175-2" href="#" role="doc-biblioref" data-xml-rid="bib175" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref175-2" aria-controls="sref175-2">Zhang et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref175-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">178.</div><div>Zhang, P. ∙ Bigio, B. ∙ Rapaport, F. ...</div><div><strong>PopViz: a webserver for visualizing minor allele frequencies and damage prediction scores of human genetic variations</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:4307-4309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85058604691" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30535305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty536&amp;pmid=30535305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The <i>TBX21</i> gene has a low tolerance of deleterious variations, with a low gene damage index (GDI) score of 3.493 (<span class="dropBlock reference-citations"><a id="body-ref-sref73" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73" aria-controls="sref73">Itan et&nbsp;al.,&nbsp;2015</a><div class="dropBlock__holder" data-db-target-of="sref73"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Itan, Y. ∙ Shang, L. ∙ Boisson, B. ...</div><div><strong>The human gene damage index as a gene-level approach to prioritizing exome variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:13615-13620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1518646112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84946595515" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26483451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1518646112&amp;pmid=26483451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and a low residual variation intolerance score (RVIS: −0.74) (<span class="dropBlock reference-citations"><a id="body-ref-sref128" href="#" role="doc-biblioref" data-xml-rid="bib128" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref128" aria-controls="sref128">Petrovski et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref128"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">131.</div><div>Petrovski, S. ∙ Gussow, A.B. ∙ Wang, Q. ...</div><div><strong>The Intolerance of Regulatory Sequence to Genetic Variation Predicts Gene Dosage Sensitivity</strong></div><div><em>PLoS Genet.</em> 2015; <strong>11</strong>:e1005492</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1005492" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26332131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1005492&amp;pmid=26332131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Moreover, only two predicted LOF variants (variants: 17:47745045 A/AGCTG, and 17:47744913 C/G) were found in the heterozygous state in gnomAD. As their MAFs were &lt;5&nbsp;× 10<sup>–6</sup>, homozygosity rates for any of these three variants are well below the prevalence of MSMD (about 1/50,000). In the general population covered by the gnomAD database, seven variants have been identified in the homozygous state: A109G has a low CADD score, below the MSC, whereas H33Q, I339V, Y395C, M331I, R218H, and A29P have CADD scores above the MSC (<a id="crosref1055" href="#fig1">Figure&nbsp;1</a>K; <a id="crosref1060" href="#mmc1">Data S1</a>). The mouse ortholog of H33Q (H32Q) has been shown to be functionally neutral (<span class="dropBlock reference-citations"><a id="body-ref-sref155" href="#" role="doc-biblioref" data-xml-rid="bib155" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref155" aria-controls="sref155">Tantisira et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref155"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">158.</div><div>Tantisira, K.G. ∙ Hwang, E.S. ∙ Raby, B.A. ...</div><div><strong>TBX21: a functional variant predicts improvement in asthma with the use of inhaled corticosteroids</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:18099-18104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0408532102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-19944427616" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15604153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0408532102&amp;pmid=15604153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). None of these seven alleles affected transactivation of the WT-TBRE promoter (<a id="crosref1070" href="#fig1">Figure&nbsp;1</a>L). Thus, all the <i>TBX21</i> variants present in the homozygous state in gnomAD are functionally neutral. In our in-house cohort of &gt;8,000 exomes from patients with various infectious phenotypes, P is the only patient carrying a rare biallelic variant at the <i>TBX21</i> locus. We investigated whether any of the other 36 non-synonymous variants of <i>TBX21</i> in our in-house database could underlie infections in the heterozygous state, by testing each of them experimentally (<a id="crosref1075" href="#mmc1">Data S1</a>). None had any functional impact on transcriptional activity (<a id="crosref1080" href="#figs1">Figures S1</a>L and S1M). Therefore, the data for P and his family, our in-house cohort, and the general population suggest that inherited T-bet deficiency, whether complete or partial, is exceedingly rare in the general population (&lt;5.8&nbsp;× 10<sup>–8</sup>). These findings also suggest that homozygosity for the Mut LOF variant of <i>TBX21</i> is responsible for MSMD in P.</div></section><section id="sec-2-4"><h3>Homozygosity for the <i>TBX21</i> Mutation Underlies Complete T-bet Deficiency</h3><div id="p0055" role="paragraph">We investigated the production and function of endogenous Mut T-bet in <i>Herpes saimiri</i> virus-transformed T&nbsp;cells (HVS-T) and primary CD4<sup>+</sup> T&nbsp;cells from P. Levels of <i>TBX21</i> mRNA were normal, but endogenous T-bet protein levels were low in P’s cells (<a id="crosref1085" href="#fig2">Figures 2</a>A and 2B; <a id="crosref1090" href="#figs2">Figures S2</a>A and S2B). Together with the observation of low levels of Mut T-bet protein on overexpression (<a id="crosref1095" href="#figs1">Figure&nbsp;S1</a>F), these findings suggest that the <i>TBX21</i> mutation decreases T-bet protein levels by a post-transcriptional mechanism. T-bet transactivates <i>IFNG</i> and <i>TNF</i> by directly binding to their regulatory promoter or enhancer (<span class="dropBlock reference-citations"><a id="body-ref-sref50" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50" aria-controls="sref50">Garrett et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref50"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Garrett, W.S. ∙ Lord, G.M. ∙ Punit, S. ...</div><div><strong>Communicable ulcerative colitis induced by T-bet deficiency in the innate immune system</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:33-45</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2007.08.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2007.08.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34848889673" target="_blank" aria-label="Open Scopus in new tab">Scopus (687)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17923086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.08.017&amp;pmid=17923086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-1" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-1" aria-controls="sref81-1">Kanhere et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref81-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Kanhere, A. ∙ Hertweck, A. ∙ Bhatia, U. ...</div><div><strong>T-bet and GATA3 orchestrate Th1 and Th2 differentiation through lineage-specific targeting of distal regulatory elements</strong></div><div><em>Nat. Commun.</em> 2012; <strong>3</strong>:1268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms2260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871783406" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms2260&amp;pmid=23232398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref144-2" href="#" role="doc-biblioref" data-xml-rid="bib144" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref144-2" aria-controls="sref144-2">Soutto et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref144-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">147.</div><div>Soutto, M. ∙ Zhang, F. ∙ Enerson, B. ...</div><div><strong>A minimal IFN-gamma promoter confers Th1 selective expression</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:4205-4212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.8.4205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12370350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.8.4205&amp;pmid=12370350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref149-2" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149-2" aria-controls="sref149-2">Szabo et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref149-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Szabo, S.J. ∙ Kim, S.T. ∙ Costa, G.L. ...</div><div><strong>A novel transcription factor, T-bet, directs Th1 lineage commitment</strong></div><div><em>Cell.</em> 2000; <strong>100</strong>:655-669</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10761931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980702-3&amp;pmid=10761931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref157-1" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157-1" aria-controls="sref157-1">Tong et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref157-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Tong, Y. ∙ Aune, T. ∙ Boothby, M.</div><div><strong>T-bet antagonizes mSin3a recruitment and transactivates a fully methylated IFN-gamma promoter via a conserved T-box half-site</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:2034-2039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0409510102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-13844320399" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15684083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0409510102&amp;pmid=15684083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Levels of spontaneous IFN-γ and tumor necrosis factor (TNF)-α production by P’s HVS-T cells were much lower than those for HVS-T cells from all the healthy donors and heterozygous relatives tested (<a id="crosref1125" href="#fig2">Figures 2</a>C–2F). This cytokine production defect in <i>TBX21</i> mutant HVS-T cells was rescued by WT T-bet complementation (<a id="crosref1130" href="#fig2">Figure&nbsp;2</a>G; <a id="crosref1135" href="#figs2">Figure&nbsp;S2</a>C). We investigated whether homozygous variants of <i>PSMD9</i>, <i>ARHGAP27</i>, or <i>ERN1</i> contributed to the low levels of IFN-γ production, by transducing HVS-T cells from P with the WT cDNAs of <i>PSMD9</i>, <i>ARHGAP27</i>, or <i>ERN1</i>. Overexpression of the WT <i>PSMD9</i>, <i>ARHGAP27</i>, or <i>ERN1</i> did not rescue the IFN-γ deficit (<a id="crosref1140" href="#figs2">Figures S2</a>D–S2G). Moreover, the overexpression of these genes in HVS-T cells from P that had already been transduced with WT <i>TBX21</i> did not further enhance IFN-γ production. These data suggest that <i>PSMD9</i>, <i>ARHGAP27</i>, and <i>ERN1</i> variants make no significant contribution to the patient’s immunological and clinical phenotype (<a id="crosref1145" href="#figs2">Figures S2</a>H and S2I).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/db795497-4517-4945-851a-73b50d5162ee/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2020.10.046/asset/5400cb79-17d1-41bd-ac4b-3f567d284df6/main.assets/gr2.jpg" height="791" width="759" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/db795497-4517-4945-851a-73b50d5162ee/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure 2</span> <span class="figure__title__text">The T-bet Variant in Patient-Derived Cells Is Loss of Function</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0090" role="paragraph">(A) Whole-cell lysates from expanding CD4<sup>+</sup> T cells from healthy donors (CTL), a heterozygous parent (WT/M), and the patient (M/M) subjected to immunoblotting.</div><div id="fspara0095" role="paragraph">(B) Immunoblotting against indicated proteins of whole-cell lysates of HVS-T cells derived from CTL, M/M, and WT/M.</div><div id="fspara0100" role="paragraph">(C and D) Spontaneous production of IFN-γ (C) and TNF-α (D) from cultures of HVS-T cells from CTL, M/M, and WT/M.</div><div id="fspara0105" role="paragraph">(E and F) The <i>IFNG</i> (E) and <i>TNF</i> (F) expression of HVS-T cells, as in (C), was assessed by RT-qPCR.</div><div id="fspara0110" role="paragraph">(G) HVS-T cells from CTL, M/M, and WT/M were rescued with EV or WT <i>TBX21</i>. The percentage of intracellular IFN-γ-producing cells with or without P/I stimulation is shown.</div><div id="fspara0115" role="paragraph">(H) CD4<sup>+</sup> T cells from CTL, M/M, and WT/M were expanded under T<sub>H</sub>0 or T<sub>H</sub>1 conditions. M/M cells were transduced as described in (G). Cells were subjected to ICS for IFN-γ and TNF-α in response to P/I.</div><div id="fspara0120" role="paragraph">(I) Percentage of IFN-γ- and TNF-α-producing cells in (H).</div><div id="fspara0125" role="paragraph">(J) Transduced T<sub>H</sub>0 cells, as in (H), were isolated and restimulated and were subjected to RNA-seq, along with non-transduced cells. Pathway enrichment among differentially regulated genes is shown.</div><div id="fspara0130" role="paragraph">(K) The numbers of T-bet-dependent differentially expressed immune genes in RNA-seq are shown.</div><div id="fspara0135" role="paragraph">(L) Heatmap of selected T-bet-dependent downstream genes, as in (K).</div><div id="fspara0140" role="paragraph">In (C)–(G) and (I), bars represent the mean and the standard error of the mean. Dots represent individual samples. One-way ANOVA was used for analysis in (C) and (D). Mann-Whitney tests were used for analysis in (E) and (F), comparing WT/M or M/M to CTL. In (C)–(F), <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001; ns, not significant.</div><div id="fspara0145" role="paragraph">See also <a id="crosref2740" href="#figs2">Figure S2</a>, <a id="crosref2745" href="#mmc2">Table S2</a>, and <a id="crosref2750" href="#mmc1">Data S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/94800d20-0495-4a68-94e9-6602d0b7688a/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2020.10.046/asset/a769eb99-98be-4102-8f0f-94fe9d10f907/main.assets/figs2.jpg" height="927" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/94800d20-0495-4a68-94e9-6602d0b7688a/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">The Patient Has Autosomal Recessive Complete T-bet Deficiency, Related to <a id="crosref2805" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0400" role="paragraph"><b>(A)</b> CD4<sup>+</sup> T&nbsp;cells from healthy donors (CTL), the heterozygous father (WT/M) or P (M/M) were expanded with anti-CD3/CD28 antibody-coated beads. They were subjected to RT-qPCR for <i>TBX21</i> with two different probes. Data are displayed as 2<sup>-ΔΔCt</sup> after normalization relative to <i>GUS</i> (endogenous control) expression, using the control mean as a calibrator. (<b>B</b>) <i>TBX21</i> expression was assessed in immortalized T&nbsp;cells (HVS-T) from healthy donors (CTL), the heterozygous father (WT/M) and the patient (M/M), by RT-qPCR, as in (A). (<b>C</b>) P-derived HVS-T cells (M/M) were retrovirally transduced with EV or WT <i>TBX21</i> containing a bicistronic CD271 surface marker. Intracellular T-bet and IFN-γ production by these cells and by CTL and heterozygous <i>TBX21</i> WT/M HVS-T cells was assessed by flow cytometry in the presence and absence of P/I stimulation. (<b>D</b>) Expression of PSMD9 and ARHGAP27 (SH3 domain-containing 20 protein) in HEK293T cells transfected with pTRIP-mCherry-2A-<i>PSMD9</i> or <i>ARHGAP27</i>-V5 plasmids along with VSVG and Pax2 plasmids, as assessed by immunoblotting with an anti-V5 tag Ab. (<b>E</b>) Expression of ERN1 in HEK293T cells transfected with pLenti-EV or WT-<i>ERN1</i>-Cter-GFP plasmids along with VSVG and Pax2 plasmids, as assessed by immunoblotting with anti-GFP or anti-ERN1Ab. (<b>F</b>) Flow cytometry analysis of the intracellular production of IFN-γ in control (CTL) HVS-T cells, M/M HVS-T cells from P, and in M/M HVS-T cells transduced with pTRIP-mCherry-2A-plasmids containing WT <i>PSMD9</i> or WT <i>ARHGAP27,</i> or empty vector. (<b>G</b>) Flow cytometry analysis of IFN-γ intracellular production in control (CTL) cells, P’s HVS-T cells (M/M), and P’s HVS-T cells transduced with pLenti-Cter-GFP-2A-puro containing the WT <i>ERN1</i> or empty vector. (<b>H</b>) and (<b>I</b>) Flow cytometry analysis of intracellular IFN-γ production in control (CTL) cells, M/M HVS-T cells from P, or M/M HVS-T cells retrovirally transduced with EV or WT T-bet (MM+EV or MM+WT) or MM+WT HVS-T cells lentivirally transduced with the cDNA of EV or WT <i>PSMD9</i>, <i>ARHGAP27</i> (H) and <i>ERN1</i> (I). (<b>J</b>) Isolated CD4<sup>+</sup> T&nbsp;cells from a healthy donor, a <i>TBX21</i> WT/M parent and the <i>TBX21</i> homozygous M/M P were expanded with anti-CD3/CD28 antibody-coated beads under T<sub>H</sub>0 or T<sub>H</sub>1 conditions. After 7d, EV or WT T-bet plasmids were used to transduce P’s cells in the presence of a bicistronic CD271 surface reporter. Cells were subjected to ICS for IFN-γ and TNF-α in response to P/I restimulation. The gating strategy for the experiment is shown. (<b>K</b>) Cells transduced as in (J) were isolated by MACS with anti-CD271 antibody-coated beads. Together with non-transduced cells, they were re-stimulated with anti-CD3/CD28 antibody-coated beads, and their culture supernatants were subjected to ELISA for IFN-γ and TNF-α.</div><div id="fspara0405" role="paragraph">In (A and B), bars represent the mean and the standard deviation; in (K), bars represent the mean and the standard error of the mean. Dots represent individual samples. One-way ANOVA was used for analysis in (B), comparing WT/M or M/M to CTL. ns&nbsp;= not significant.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-5"><h3>T-bet Deficiency Alters the Expression of Downstream Target Genes</h3><div id="p0060" role="paragraph">The functional impact of the T-bet mutation was also investigated in primary CD4<sup>+</sup> T&nbsp;cells. Upon stimulation with phorbol 12-myristate 13-acetate (PMA) and ionomycin (P/I), IFN-γ production was almost entirely abolished in P’s expanded T<sub>H</sub>0 cells and TNF-α production was impaired; the production of both these molecules was rescued by WT T-bet (<a id="crosref1150" href="#fig2">Figures 2</a>H and 2I; <a id="crosref1155" href="#figs2">Figures S2</a>J and S2K). In T<sub>H</sub>1 conditions, exogenous IL-12 bypassed T-bet and induced moderate IFN-γ production by P’s cells. However, the levels of IFN-γ were still ∼60%–70% lower than those in healthy donors (<a id="crosref1160" href="#fig2">Figures 2</a>H and 2I). We investigated other T-bet-dependent transcriptional targets, by performing RNA sequencing (RNA-seq) to compare T<sub>H</sub>0 cells from 4 healthy donors, P, and P’s cells complemented with WT T-bet after incubation with anti-CD3/28 Ab beads. We found that 415 genes were downregulated and 489 genes were upregulated in P’s cells relative to healthy donors (<a id="crosref1165" href="#mmc1">Data S1</a>; <a id="crosref1170" href="#mmc2">Table S2</a>). The complementation of P’s cells with WT T-bet reversed the differential expression of 99 of the downregulated and 161 of the upregulated genes (<a id="crosref1175" href="#mmc1">Data S1</a>). Thus, only a subset of differentially regulated genes were rescued, possibly due to differences in genetic variants other than <i>TBX21</i> between controls and P. Alternatively, T-bet may control the differential expression of some genes early in activation, and the genetic rescue may have occurred too late to reverse this effect. These targets were enriched in cytokine signaling pathway genes (<a id="crosref1180" href="#fig2">Figure&nbsp;2</a>J). We therefore decided to focus on genes involved in immunological signaling. Only 32 such genes were upregulated or downregulated in P’s cells, but these differences in expression relative to controls were reversed by WT T-bet (<a id="crosref1185" href="#fig2">Figure&nbsp;2</a>K). Known T-bet-dependent targets, such as <i>IFNG</i>, <i>CCL3</i>, and <i>CXCR3</i>, were downregulated in CD4<sup>+</sup> T&nbsp;cells from the T-bet-deficient patient, whereas <i>IFNGR2</i> expression was upregulated (<a id="crosref1190" href="#fig2">Figure&nbsp;2</a>L; <span class="dropBlock reference-citations"><a id="body-ref-sref75" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75" aria-controls="sref75">Iwata et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref75"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Iwata, S. ∙ Mikami, Y. ∙ Sun, H.-W. ...</div><div><strong>The Transcription Factor T-bet Limits Amplification of Type I IFN Transcriptome and Circuitry in T Helper 1 Cells</strong></div><div><em>Immunity.</em> 2017; <strong>46</strong>:983-991</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2017.05.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2017.05.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85020642282" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28623086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.05.005&amp;pmid=28623086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref78-1" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-1" aria-controls="sref78-1">Jenner et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref78-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Jenner, R.G. ∙ Townsend, M.J. ∙ Jackson, I. ...</div><div><strong>The transcription factors T-bet and GATA-3 control alternative pathways of T-cell differentiation through a shared set of target genes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:17876-17881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0909357106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-70449571945" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19805038/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0909357106&amp;pmid=19805038" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A set of new T-bet target genes was also identified, including <i>CCL1</i>, <i>CCL13</i>, <i>CCL4</i>, <i>CSF2</i>, <i>CXCR5</i>, <i>GZMM</i>, <i>CD40</i>, <i>CD86</i>, <i>IL7R</i>, <i>IL10</i>, <i>LTB</i>, <i>ITGA5</i>, and <i>ITGB5</i> (<a id="crosref1205" href="#fig2">Figure&nbsp;2</a>L). Collectively, our data indicate that the patient had AR complete T-bet deficiency, affecting the expression of a set of T-bet-dependent target genes, including <i>IFNG</i>.</div></section><section id="sec-2-6"><h3>T-bet Induces Permissive Chromatin Accessibility and CpG Methylation in <i>IFNG</i></h3><div id="p0065" role="paragraph">We analyzed the molecular mechanisms by which T-bet controls transcription. Epigenetically, T-bet is known to induce a permissive environment for transcription at the <i>IFNG</i> promoter through histone modifications and the suppression of CpG methylation (<span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Lewis et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Lewis, M.D. ∙ Miller, S.A. ∙ Miazgowicz, M.M. ...</div><div><strong>T-bet’s ability to regulate individual target genes requires the conserved T-box domain to recruit histone methyltransferase activity and a separate family member-specific transactivation domain</strong></div><div><em>Mol. Cell. Biol.</em> 2007; <strong>27</strong>:8510-8521</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01615-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-37549032110" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17923685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01615-07&amp;pmid=17923685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref118-1" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-1" aria-controls="sref118-1">Miller and Weinmann, 2010</a><div class="dropBlock__holder" data-db-target-of="sref118-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Miller, S.A. ∙ Weinmann, A.S.</div><div><strong>Molecular mechanisms by which T-bet regulates T-helper cell commitment</strong></div><div><em>Immunol. Rev.</em> 2010; <strong>238</strong>:233-246</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1600-065X.2010.00952.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77958610119" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20969596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1600-065X.2010.00952.x&amp;pmid=20969596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref157-2" href="#" role="doc-biblioref" data-xml-rid="bib157" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref157-2" aria-controls="sref157-2">Tong et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref157-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">160.</div><div>Tong, Y. ∙ Aune, T. ∙ Boothby, M.</div><div><strong>T-bet antagonizes mSin3a recruitment and transactivates a fully methylated IFN-gamma promoter via a conserved T-box half-site</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:2034-2039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0409510102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-13844320399" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15684083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0409510102&amp;pmid=15684083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, it remains unknown whether T-bet directly regulates chromatin accessibility in mice or humans. The regulation of CpG methylation at the genome-wide scale by T-bet has never been studied. We performed omni-ATAC sequencing (omni-ATAC-seq) analysis and EPIC DNA CpG methylation array analysis with T<sub>H</sub>0 cells derived from P and controls. In P’s cells, a gain of chromatin accessibility was observed at 1,661 loci and a loss of chromatin accessibility was observed at 3,675 loci (<a id="crosref1225" href="#fig3">Figures 3</a>A and 3B; <a id="crosref1230" href="#mmc1">Data S1</a>; <a id="crosref1235" href="#mmc3">Table S3</a>). We found that 662 and 1,642 of&nbsp;these loci, respectively, were subject to strict regulation by T-bet, as their gains and losses of chromatin accessibility were reversed by WT T-bet (<a id="crosref1240" href="#fig3">Figure&nbsp;3</a>B; <a id="crosref1245" href="#mmc3">Table S3</a>). The chromatin regions opened up by T-bet were heavily occupied by bound T-bet, whereas those closed by T-bet did not typically require T-bet binding (<a id="crosref1250" href="#fig3">Figure&nbsp;3</a>C; <span class="dropBlock reference-citations"><a id="body-ref-sref81-2" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-2" aria-controls="sref81-2">Kanhere et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref81-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Kanhere, A. ∙ Hertweck, A. ∙ Bhatia, U. ...</div><div><strong>T-bet and GATA3 orchestrate Th1 and Th2 differentiation through lineage-specific targeting of distal regulatory elements</strong></div><div><em>Nat. Commun.</em> 2012; <strong>3</strong>:1268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms2260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871783406" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms2260&amp;pmid=23232398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). An enrichment in T-box binding elements was observed in loci for which chromatin accessibility was increased by T-bet, whereas an enrichment in Forkhead box elements was observed at loci for which chromatin accessibility was decreased by T-bet (<a id="crosref1260" href="#fig3">Figures 3</a>D and 3E). The chromatin accessibility of 181 immunological genes (247 loci), including <i>IFNG</i>, <i>CYBB</i>, and <i>TYK2</i>, three known MSMD-causing genes, was increased by T-bet, whereas that of 73 immunological genes (82 loci) was decreased by T-bet (<a id="crosref1265" href="#fig3">Figure&nbsp;3</a>F; <a id="crosref1270" href="#mmc1">Data S1</a>; <a id="crosref1275" href="#mmc3">Table S3</a>). Three known T-bet-dependent targets, <i>IFNG</i>, <i>TNF</i>, and <i>CXCR3</i>, were among the top hits for the differentially regulated loci. The transcription start sites (TSSs) of <i>IFNG</i> and <i>TNF</i>, the proximal promoter of <i>IFNG</i>, and the enhancers of <i>CXCR3</i> were inaccessible in T-bet-deficient cells, and this inaccessibility was rescued by WT T-bet (<a id="crosref1280" href="#fig3">Figures 3</a>G–3I). The EPIC DNA CpG methylation array analysis identified 644,236 CpG sites that were differentially regulated (<a id="crosref1285" href="#mmc3">Table S3</a>). Three CpG loci within <i>IFNG</i> were hypermethylated in conditions of T-bet deficiency, whereas their methylation was reduced to levels similar to those in controls on complementation with WT T-bet (<a id="crosref1290" href="#fig3">Figures 3</a>J and 3K). <i>NR5A2</i>, <i>TIMD4</i>, <i>ATXN2</i>, <i>ZAK</i>, <i>SLAMF8</i>, <i>TBKBP1</i>, <i>CD247</i>, <i>HDAC4</i>, and several other genes were also regulated in a similar manner (<a id="crosref1295" href="#fig3">Figure&nbsp;3</a>J; <a id="crosref1300" href="#mmc3">Table S3</a>). Interestingly, the methylation of six CpG loci within <i>ENTPD1</i> not previously linked to T-bet&nbsp;also increased in a T-bet-dependent manner (<a id="crosref1305" href="#fig3">Figures 3</a>J and 3L). By contrast, <i>IL10</i> was a top target for which CpG methylation was drastically reduced in T-bet deficiency but rescued by WT T-bet (<a id="crosref1310" href="#fig3">Figure&nbsp;3</a>M). Taken together, these results demonstrate that T-bet orchestrates the expression of target genes by modulating both their chromatin accessibility and CpG methylation. Genome-wide omni-ATAC-seq and CpG methylation array analyses identified new epigenetic targets of T-bet (<a id="crosref1315" href="#mmc3">Table S3</a>). They also showed that chromatin accessibility at <i>IFNG</i> was increased by T-bet at both the TSSs and promoter sites, whereas the CpG methylation of <i>IFNG</i> was decreased by T-bet at three different positions.</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/fe99329d-681f-4fb0-ad79-9bb8bf766810/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2020.10.046/asset/f5e1b6b9-3d3d-4216-90d7-765e795511fe/main.assets/gr3.jpg" height="949" width="684" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/fe99329d-681f-4fb0-ad79-9bb8bf766810/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure 3</span> <span class="figure__title__text">An Altered Spectrum of Epigenetic Regulation in CD4<sup>+</sup> T Cells in T-bet Deficiency</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">(A) Expanded CD4<sup>+</sup> T<sub>H</sub>0 cells from CTL, IL-12Rβ1-deficient MSMD patients (<i>IL12RB1</i> M/M), WT/M, M/M, and M/M cells complemented with empty vector (M/M +EV) or with WT T-bet (M/M +WT) were subjected to omni-ATAC-seq. PCA analysis of the peaks called in the various samples, plotted in ggplot2.</div><div id="fspara0160" role="paragraph">(B) Number of called peaks differentially regulated as indicated.</div><div id="fspara0165" role="paragraph">(C) Heatmap of all loci opening or closing in a T-bet-dependent manner as in (B).</div><div id="fspara0170" role="paragraph">(D and E) The most significantly enriched DNA binding motifs at loci displaying T-bet-dependent increases (D) and loci displaying T-bet-dependent decreases (E), as in (B and C).</div><div id="fspara0175" role="paragraph">(F) Chromatin accessibility heatmap for all the loci displaying the most significant differential regulation of immune genes (CTL – M/M, adjusted p &lt; 1 × 10<sup>−5</sup>), abolished by WT T-bet (M/M+EV – M/M+WT, adjusted p &lt; 1 × 10<sup>−5</sup>).</div><div id="fspara0180" role="paragraph">(G–I) Chromatin accessibility of transcription start site (TSS), proximal promoter, and distal enhancer regions of <i>IFNG</i>, <i>TNF</i>, and <i>CXCR3</i>.</div><div id="fspara0185" role="paragraph">(J) The differential methylation (beta-value) of CpG sites between M/M and CTL or between M/M+EV and M/M+WT is shown.</div><div id="fspara0190" role="paragraph">(K–M) CpG methylation status of CpG sites in <i>IFNG</i> (K), <i>ENTPD1</i> (L), and <i>IL10</i> (M).</div><div id="fspara0195" role="paragraph">See also <a id="crosref2755" href="#mmc3">Table S3</a> and <a id="crosref2760" href="#mmc1">Data S1</a>.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-7"><h3>T-bet Deficiency Impairs NK Cell Maturation</h3><div id="p0070" role="paragraph">We then investigated the role of T-bet in the development of leukocyte lineages. Complete blood counts for fresh samples from P showed that the numbers of lymphocytes, neutrophils, and monocytes were normal. We studied the PBMC subsets of P after recovery from mycobacterial disease, by mass cytometry (cytometry by time-of-flight, CyTOF). A simultaneous analysis of the expression of 38 different cell-surface markers was performed to compare the leukocyte subsets of P and his parents, healthy donors, and patients with complete IL-12Rβ1 deficiency, the most common genetic etiology of MSMD, as controls (<a id="crosref1320" href="#fig4">Figure&nbsp;4</a>A; <a id="crosref1325" href="#figs3">Figure&nbsp;S3</a>A; <span class="dropBlock reference-citations"><a id="body-ref-sref162-1" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref162-1" aria-controls="sref162-1">Van Gassen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref162-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Van Gassen, S. ∙ Callebaut, B. ∙ Van Helden, M.J. ...</div><div><strong>FlowSOM: Using self-organizing maps for visualization and interpretation of cytometry data</strong></div><div><em>Cytometry A.</em> 2015; <strong>87</strong>:636-645</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.22625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84932198501" target="_blank" aria-label="Open Scopus in new tab">Scopus (278)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25573116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.22625&amp;pmid=25573116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref89-1" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-1" aria-controls="sref89-1">Korsunsky et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref89-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Korsunsky, I. ∙ Millard, N. ∙ Fan, J. ...</div><div><strong>Fast, sensitive and accurate integration of single-cell data with Harmony</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1289-1296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0619-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85070698712" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0619-0&amp;pmid=31740819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The frequencies of the plasmacytoid DC (pDC) and conventional DC 1 and 2 (cDC1 and cDC2) subsets were not affected by human T-bet deficiency (<a id="crosref1340" href="#figs3">Figure&nbsp;S3</a>B). All major myeloid lineages were normal in P. We therefore focused on the development of lymphoid lineages. Total NK cells (defined as Lin<sup>–</sup>CD7<sup>+</sup>CD16<sup>+</sup> or CD94<sup>+</sup>) were present in normal numbers in P (<a id="crosref1345" href="#figs3">Figures&nbsp;S3</a>C and S3D). However, CD16<sup>+</sup> and CD56<sup>bright</sup> NK cells levels were much lower in P than in controls (<a id="crosref1350" href="#fig4">Figures 4</a>A and 4B). Moreover, P had an abnormally high frequency of CD56<sup>–</sup>CD127<sup>–</sup> NK cells (<a id="crosref1355" href="#figs3">Figure&nbsp;S3</a>D); this NK cell subset has low levels of cytotoxicity and is rare in healthy and normal individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Björkström et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Björkström, N.K. ∙ Ljunggren, H.-G. ∙ Sandberg, J.K.</div><div><strong>CD56 negative NK cells: origin, function, and role in chronic viral disease</strong></div><div><em>Trends Immunol.</em> 2010; <strong>31</strong>:401-406</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.it.2010.08.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.it.2010.08.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-78049252250" target="_blank" aria-label="Open Scopus in new tab">Scopus (132)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20829113/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2010.08.003&amp;pmid=20829113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The frequencies of ILC precursors (ILCPs) and ILC2s in P were similar to those in healthy donors and IL-12Rβ1-deficient patients (<a id="crosref1365" href="#figs3">Figures S3</a>E and S3F). In stringent analyses, ILC1 and ILC3 are too rare for quantification in human peripheral blood (<span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Lim et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Lim, A.I. ∙ Li, Y. ∙ Lopez-Lastra, S. ...</div><div><strong>Systemic Human ILC Precursors Provide a Substrate for Tissue ILC Differentiation</strong></div><div><em>Cell.</em> 2017; <strong>168</strong>:1086-1100</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2017.02.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2017.02.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85014819484" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28283063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.02.021&amp;pmid=28283063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, human T-bet is required for the correct development or maturation of NK cells, but not monocytes, DCs, ILC2, or ILCP.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/3095b237-a2e6-4496-a46a-4ddec6e3c70d/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2020.10.046/asset/90192588-803d-4fde-9096-783211eceae3/main.assets/gr4.jpg" height="922" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/3095b237-a2e6-4496-a46a-4ddec6e3c70d/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure 4</span> <span class="figure__title__text">Impaired <i>In Vivo</i> Development of NK, Invariant NKT, MAIT, Vδ2<sup>+</sup> γδ T, and T<sub>H</sub>1 Cells in T-bet Deficiency</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0205" role="paragraph">(A) UMAP representation shows immunophenotyping of 40,000 CD45<sup>+</sup>CD66b<sup>–</sup>DNA<sup>+</sup> live PBMCs from age-matched controls (Age CTL) and <i>TBX21</i> M/M, by CyTOF.</div><div id="fspara0210" role="paragraph">(B) Frequencies of CD56<sup>bright</sup> NK cells and CD56<sup>dim</sup>CD16<sup>+</sup> NK cells by CyTOF.</div><div id="fspara0215" role="paragraph">(C–E) Percentages of iNKT (C), MAIT (D), and Vδ2<sup>+</sup> γδ T cells (E) among live single cells from indicated individuals, including P’s healthy brother (WT/WT), by flow cytometry.</div><div id="fspara0220" role="paragraph">(F) viSNE map of 10,000 CD45RA<sup>–</sup> memory CD4<sup>+</sup> T cells from indicated individuals, with expression of CXCR3 and CCR5 shown.</div><div id="fspara0225" role="paragraph">(G and H) Frequencies of CCR6<sup>–</sup> T<sub>H</sub>1 (CCR6<sup>–</sup>CXCR3<sup>+</sup>CCR4<sup>–</sup>) (G) and CCR6<sup>+</sup> T<sub>H</sub>1<sup>∗</sup> (CCR6<sup>+</sup>CXCR3<sup>+</sup>CCR4<sup>–</sup>) cells (H) among live CD45<sup>+</sup>CD66b<sup>–</sup> cells are shown.</div><div id="fspara0230" role="paragraph">(I) A bubble graph is presented to show genes for which the proportion of cells displaying expression is altered in the <i>TBX21</i> M/M cells in comparison with PBMCs from the <i>TBX21</i> WT/M father, as a control, by scRNA-seq. The size of the bubble indicates the proportion of cells expressing the gene and the color scale indicates the log2-transformed ratios. The genes highlighted in red were identified as T-bet-regulated by ATAC-seq and/or RNA-seq. LD, low level of detection; ND, not detected.</div><div id="fspara0235" role="paragraph">In (B)–(E), (G), and (H), bars represent the mean and the standard error of the mean. Dots represent individual samples or technical replicates. Mann-Whitney nonparametric tests were used for analysis in (C)–(E). In (C)–(E), <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001; ns, not significant.</div><div id="fspara0240" role="paragraph">See also <a id="crosref2765" href="#figs3">Figures S3</a> and <a id="crosref2770" href="#figs4">S4</a> and <a id="crosref2775" href="#mmc1">Data S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/5c417311-5be6-4294-9830-dadfd57935ca/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2020.10.046/asset/5124747d-482f-42db-951d-88fcfc5e1cfa/main.assets/figs3.jpg" height="948" width="729" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/5c417311-5be6-4294-9830-dadfd57935ca/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure S3</span> <span class="figure__title__text">T-bet Deficiency Results in a Normal Distribution of Dendritic Cell Subsets, ILCs, and Vδ1<sup>+</sup> γδ T Cells but Alters the Development of NK cells, iNKT, MAIT, and Vδ2<sup>+</sup> γδ T cells, Related to <a id="crosref2810" href="#fig4">Figure 4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0415" role="paragraph"><b>(A)</b> PBMCs were immunophenotyped with CyTOF. The gating strategy for different immune subsets in a healthy control is indicated. (<b>B</b>) Percentages of pDCs, cDC1 cells and cDC2 cells among adult healthy donors (CTL), age-matched controls (Age-matched CTL), IL-12Rβ1-deficient MSMD patients (<i>IL12RB1</i> M/M) and P (<i>TBX21</i> M/M), as indicated in (A). (<b>C</b>) PBMCs from P and a healthy control were subjected to the FACS immunophenotyping of innate lymphoid cells (ILCs) and NK cells. CD117<sup>+</sup> ILCs, consisting principally of ILC precursors (ILCP) and type 2 ILC (ILC2) cells, were gated manually on conventional flow cytometry. The gating strategy is shown. (<b>D</b>) Total NK cells (CD16<sup>+</sup> or CD94<sup>+</sup>Lin<sup>-</sup>CD7<sup>+</sup> cells) were plotted with CD56 against CD127 for manual gating on different NK cell subsets. (<b>E</b>) Percentage of ILC precursor and ILC2 cells in adult healthy donors (CTL), age-matched controls (Age CTL), IL-12Rβ1-deficient MSMD patients (<i>IL12RB1</i> M/M) and P (<i>TBX21</i> M/M), summarized from CyTOF data, as in (A). (<b>F</b>) Immunophenotyping of ILC2 and ILCP cells, analyzed by fluorescence flow cytometry. Plot of ILC2 and ILCP, and the percentage of ILC2 and ILCP (within live CD45<sup>+</sup> cells) in a healthy donor (CTL), the T-bet-deficient patient (M/M) and his age/ethnicity-matched healthy brother (WT/WT bro). (<b>G</b>) PBMCs from a CTL, IL-12Rβ1-deficient MSMD patients (<i>IL12RB1</i> M/M), the TBX21 WT/M parents (WT/M mo or fa), the T-bet-deficient P (M/M) and his age/ethnicity-matched healthy brother (WT/WT bro) were stained with surface markers for lymphocytes involved innate immunity (MAIT and iNKT cells) or both adaptive and innate immunity (γδ T cells, in particular Vδ1 and Vδ2 γδ T cells). The gating strategy for flow cytometry is shown. (<b>H</b>) - (<b>J</b>) iNKT (H), MAIT (I) and Vδ2<sup>+</sup> γδ T (J) cells are shown for P and his relatives. (<b>K</b>) Percentages of Vδ1<sup>+</sup> γδ T cells in healthy adult donors (CTL), age-matched controls (Age CTL), IL-12Rβ1-deficient MSMD patients (<i>IL12RB1</i> M/M) and P (<i>TBX21</i> M/M), his age/ethnicity-matched healthy brother (<i>TBX21</i> WT/WT) and his healthy parents (<i>TBX21</i> WT/M).</div><div id="fspara0420" role="paragraph">In (B and E), bars represent the mean and the standard deviation. In (K), bars represent the mean and the standard error of the mean. Dots represent individual samples. Mann-Whitney nonparametric tests were used for analysis in (K). In (K), <sup>∗∗</sup>p &lt; 0.01.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-8"><h3>Low Levels of the iNKT, MAIT, and Vδ2<sup>+</sup> γδ T Cell Lineages in T-bet Deficiency</h3><div id="p0075" role="paragraph">iNKT, MAIT, and γδ T&nbsp;cells are “innate-like” adaptive T&nbsp;cell lineages with less T&nbsp;cell receptor (TCR) diversity than conventional, “purely” adaptive αβ T&nbsp;cells (<span class="dropBlock reference-citations"><a id="body-ref-sref29" href="#" role="doc-biblioref" data-xml-rid="bib29" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref29" aria-controls="sref29">Chien et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref29"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Chien, Y.H. ∙ Meyer, C. ∙ Bonneville, M.</div><div><strong>γδ T&nbsp;cells: first line of defense and beyond</strong></div><div><em>Annu. Rev. Immunol.</em> 2014; <strong>32</strong>:121-155</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-032713-120216" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84896966923" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24387714/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-032713-120216&amp;pmid=24387714" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Crosby and Kronenberg, 2018</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Crosby, C.M. ∙ Kronenberg, M.</div><div><strong>Tissue-specific functions of invariant natural killer T&nbsp;cells</strong></div><div><em>Nat. Rev. Immunol.</em> 2018; <strong>18</strong>:559-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-018-0034-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85049559053" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29967365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-018-0034-2&amp;pmid=29967365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref55" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55" aria-controls="sref55">Godfrey et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref55"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Godfrey, D.I. ∙ Koay, H.-F. ∙ McCluskey, J. ...</div><div><strong>The biology and functional importance of MAIT cells</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:1110-1128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0444-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85070817379" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31406380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0444-8&amp;pmid=31406380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). iNKT cells constitute a group of T&nbsp;cells with invariant TCRs combining properties from both T&nbsp;cells and NK cells (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Crosby and Kronenberg, 2018</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Crosby, C.M. ∙ Kronenberg, M.</div><div><strong>Tissue-specific functions of invariant natural killer T&nbsp;cells</strong></div><div><em>Nat. Rev. Immunol.</em> 2018; <strong>18</strong>:559-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-018-0034-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85049559053" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29967365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-018-0034-2&amp;pmid=29967365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The iNKT cell levels of P were low (present at a level ∼27-fold lower than that in controls) (<a id="crosref1395" href="#fig4">Figure&nbsp;4</a>C; <a id="crosref1400" href="#figs3">Figures S3</a>G and S3H). MAIT cells express invariant Vα7.2-Jα33 TCR-α restricted by a monomorphic class I-related MHC molecule, along with ligands derived from vitamin B synthesis (<span class="dropBlock reference-citations"><a id="body-ref-sref86" href="#" role="doc-biblioref" data-xml-rid="bib86" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref86" aria-controls="sref86">Kjer-Nielsen et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref86"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Kjer-Nielsen, L. ∙ Patel, O. ∙ Corbett, A.J. ...</div><div><strong>MR1 presents microbial vitamin B metabolites to MAIT cells</strong></div><div><em>Nature.</em> 2012; <strong>491</strong>:717-723</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11605" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84870284140" target="_blank" aria-label="Open Scopus in new tab">Scopus (659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23051753/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11605&amp;pmid=23051753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref169" href="#" role="doc-biblioref" data-xml-rid="bib169" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref169" aria-controls="sref169">Xiao and Cai, 2017</a><div class="dropBlock__holder" data-db-target-of="sref169"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">172.</div><div>Xiao, X. ∙ Cai, J.</div><div><strong>Mucosal-associated invariant T&nbsp;cells: New insights into antigen recognition and activation</strong></div><div><em>Front. Immunol.</em> 2017; <strong>8</strong>:1540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2017.01540" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85034088916" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29176983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2017.01540&amp;pmid=29176983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). P also had a lower frequency of MAIT cells than healthy donors (∼15-fold) (<a id="crosref1415" href="#fig4">Figure&nbsp;4</a>D; <a id="crosref1420" href="#figs3">Figure&nbsp;S3</a>I). Total γδ T&nbsp;cell frequency was normal in P. However, the frequency of the Vδ2<sup>+</sup> subset, a group of γδ T&nbsp;cells that recognize phosphoantigen (pAgs) (<span class="dropBlock reference-citations"><a id="body-ref-sref59-1" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-1" aria-controls="sref59-1">Gu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref59-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Gu, S. ∙ Borowska, M.T. ∙ Boughter, C.T. ...</div><div><strong>Butyrophilin3A proteins and Vγ9Vδ2 T&nbsp;cell activation</strong></div><div><em>Semin. Cell Dev. Biol.</em> 2018; <strong>84</strong>:65-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.semcdb.2018.02.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85043373095" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29471037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.semcdb.2018.02.007&amp;pmid=29471037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-1" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-1" aria-controls="sref64-1">Harly et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Harly, C. ∙ Guillaume, Y. ∙ Nedellec, S. ...</div><div><strong>Key implication of CD277/butyrophilin-3 (BTN3A) in cellular stress sensing by a major human γδ T-cell subset</strong></div><div><em>Blood.</em> 2012; <strong>120</strong>:2269-2279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2012-05-430470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866308872" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22767497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2012-05-430470&amp;pmid=22767497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref163-1" href="#" role="doc-biblioref" data-xml-rid="bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref163-1" aria-controls="sref163-1">Vavassori et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref163-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Vavassori, S. ∙ Kumar, A. ∙ Wan, G.S. ...</div><div><strong>Butyrophilin 3A1 binds phosphorylated antigens and stimulates human γδ T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:908-916</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2665" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84883147590" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23872678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2665&amp;pmid=23872678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), was low (∼6-fold lower than control levels) in P, whereas the levels of the Vδ1<sup>+</sup> subset of γδ T&nbsp;cells seemed to be slightly high (<a id="crosref1440" href="#fig4">Figure&nbsp;4</a>E; <a id="crosref1445" href="#figs3">Figures S3</a>J and S3K). Mild abnormalities of B cell development and antibody production unrelated to the patient’s mycobacterial disease were observed and will be reported in a separate study (unpublished data). CD4<sup>+</sup> and CD8<sup>+</sup> αβ T&nbsp;cells are the two most prevalent blood lineages of adaptive lymphocytes expressing a highly diverse αβ TCR repertoire. Antigen-driven CD8<sup>+</sup> T&nbsp;cell effector responses and the optimal induction of memory CD8<sup>+</sup> T&nbsp;cells in mice are controlled by T-bet (<span class="dropBlock reference-citations"><a id="body-ref-sref11-1" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-1" aria-controls="sref11-1">Bettelli et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref11-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bettelli, E. ∙ Sullivan, B. ∙ Szabo, S.J. ...</div><div><strong>Loss of T-bet, but not STAT1, prevents the development of experimental autoimmune encephalomyelitis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2004; <strong>200</strong>:79-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20031819" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-3142724732" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15238607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20031819&amp;pmid=15238607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Juedes et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Juedes, A.E. ∙ Rodrigo, E. ∙ Togher, L. ...</div><div><strong>T-bet controls autoaggressive CD8 lymphocyte responses in type 1 diabetes</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2004; <strong>199</strong>:1153-1162</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20031873" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-2442509626" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15096540/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20031873&amp;pmid=15096540" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref146-1" href="#" role="doc-biblioref" data-xml-rid="bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146-1" aria-controls="sref146-1">Sullivan et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref146-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">149.</div><div>Sullivan, B.M. ∙ Juedes, A. ∙ Szabo, S.J. ...</div><div><strong>Antigen-driven effector CD8 T&nbsp;cell function regulated by T-bet</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:15818-15823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2636938100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0347994894" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14673093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2636938100&amp;pmid=14673093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the T-bet-deficient patient, total CD8<sup>+</sup> T&nbsp;cells and the composition of naive, central memory, effector memory, and T<sub>EMRA</sub> cells were normal (<a id="crosref1465" href="#figs4">Figure&nbsp;S4</a>A). We further investigated CD8<sup>+</sup> T&nbsp;cells in an unbiased manner, by automatic viSNE clustering with a panel of surface markers, including chemokine receptors (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Amir el et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Amir el, A.D. ∙ Davis, K.L. ∙ Tadmor, M.D. ...</div><div><strong>viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:545-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84880280631" target="_blank" aria-label="Open Scopus in new tab">Scopus (816)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23685480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2594&amp;pmid=23685480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We found no apparent difference between the memory CD8<sup>+</sup> T&nbsp;cells of P and controls. However, a small subset of naive CD8<sup>+</sup> T&nbsp;cells (CD45RA<sup>+</sup>CD38<sup>int</sup>CXCR3<sup>int</sup>CCR6<sup>–</sup>CCR5<sup>–</sup>CD27<sup>high</sup>CD127<sup>high</sup>) was absent from P (<a id="crosref1475" href="#figs4">Figure&nbsp;S4</a>B). Thus, the development of iNKT, MAIT cells, and Vδ2<sup>+</sup> γδ T&nbsp;cells and a small subset of naive CD8<sup>+</sup> T&nbsp;cells is impaired in T-bet deficiency.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/510bbbaf-b9ba-4228-b615-3fd0629407ba/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2020.10.046/asset/c135f75b-ee81-495e-9266-0cd27200b920/main.assets/figs4.jpg" height="886" width="696" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/510bbbaf-b9ba-4228-b615-3fd0629407ba/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure S4</span> <span class="figure__title__text">Preserved T<sub>H</sub>2, T<sub>H</sub>17, and Circulating T<sub>FH</sub> Memory CD4<sup>+</sup> T Cells in T-bet Deficiency, Related to <a id="crosref2815" href="#fig4">Figure 4</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0430" role="paragraph"><b>(A)</b> Naive (CD45RA<sup>+</sup>CCR7<sup>+</sup>), central memory (CM, CD45RA<sup>-</sup>CCR7<sup>+</sup>), effector memory (EM, CD45RA<sup>-</sup>CCR7<sup>-</sup>) and T<sub>EMRA</sub> (CD45RA<sup>+</sup>CCR7<sup>-</sup>) cells were manually gated from CD8<sup>+</sup> T cells on CyTOF data. The percentages of these cells are shown. (<b>B</b>) 10,000 total CD8<sup>+</sup> T cells were clustered by viSNE, based on a selection of surface markers. Manual gating of naive and memory subsets overlaid on clusters of CD8<sup>+</sup> T cells. CXCR3 and CD38 expression are indicated on this plot. (<b>C</b>) Naive (CD45RA<sup>+</sup>CCR7<sup>+</sup>), central memory (CM, CD45RA<sup>-</sup>CCR7<sup>+</sup>), effector memory (EM, CD45RA<sup>-</sup>CCR7<sup>-</sup>) and T<sub>EMRA</sub> (CD45RA<sup>+</sup>CCR7<sup>-</sup>) cells were manually gated from CD4<sup>+</sup> T cells on the basis of CyTOF data. The percentages of these cells are shown. (<b>D</b>) The gating strategy for the CyTOF analysis of memory CD4<sup>+</sup> T cells and T<sub>H</sub>1, T<sub>H</sub>2, T<sub>H</sub>17 and T<sub>H</sub>1<sup>∗</sup> cells is shown. (<b>E</b>) Memory CD4<sup>+</sup> T cells (CD45RA<sup>-</sup>) from indicated individuals were gated out manually, and further subjected to viSNE deconvolution with T cell chemokine markers for 10,000 randomly selected events. The expression of CXCR3, CCR5, CCR6 and CCR4 is shown. (<b>F</b>) and (<b>G</b>) Percentages of T<sub>H</sub>2 (CCR6<sup>-</sup>CXCR3<sup>-</sup>CCR4<sup>+</sup>) (F) and T<sub>H</sub>17 cells (CCR6<sup>+</sup>CXCR3<sup>-</sup>CCR4<sup>+</sup>) (G) among memory CD4<sup>+</sup> T cells. (<b>H</b>) Frequencies of the circulating follicular T helper (cT<sub>FH</sub>) cells and their subsets (CXCR3<sup>+</sup>, double-negative as DN, CCR6<sup>+</sup>, or CXCR3<sup>+</sup>CCR6<sup>+</sup> T<sub>FH</sub>) from healthy donors (CTL) and the T-bet-deficient patient (M/M) gated from conventional flow cytometry data. (<b>I</b>) Percentages of Treg cells gated from CyTOF data. (<b>J</b>) Percentages of CXCR3<sup>+</sup> and CCR5<sup>+</sup>CXCR3<sup>-</sup> Tregs gated from CyTOF data.</div><div id="fspara0435" role="paragraph">In (A, C, F, G, H, I and J), bars represent the mean and the standard error of the mean. Dots represent individual samples.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-9"><h3>Selective Depletion of CCR6<sup>–</sup> T<sub>H</sub>1 Cells from CD4<sup>+</sup> T Cells in T-bet Deficiency</h3><div id="p0080" role="paragraph">Both P and his heterozygous parents had normal distributions of naive and memory CD4<sup>+</sup> T&nbsp;cells (<a id="crosref1480" href="#figs4">Figure&nbsp;S4</a>C). We further analyzed individual CD4<sup>+</sup> T&nbsp;cell subsets by viSNE clustering on antigen-experienced memory cells. Several memory CD4<sup>+</sup> T&nbsp;cell subsets typically present in healthy donors were missing in P. In particular, most of the CXCR3<sup>+</sup> cells, corresponding to T<sub>H</sub>1 cells in humans, and CCR5<sup>+</sup> cells (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Groom and Luster, 2011</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Groom, J.R. ∙ Luster, A.D.</div><div><strong>CXCR3 in T&nbsp;cell function</strong></div><div><em>Exp. Cell Res.</em> 2011; <strong>317</strong>:620-631</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.yexcr.2010.12.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79952091851" target="_blank" aria-label="Open Scopus in new tab">Scopus (400)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21376175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.yexcr.2010.12.017&amp;pmid=21376175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Loetscher et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Loetscher, P. ∙ Uguccioni, M. ∙ Bordoli, L. ...</div><div><strong>CCR5 is characteristic of Th1 lymphocytes</strong></div><div><em>Nature.</em> 1998; <strong>391</strong>:344-345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/34814" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9450746/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F34814&amp;pmid=9450746" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref139" href="#" role="doc-biblioref" data-xml-rid="bib139" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref139" aria-controls="sref139">Sallusto et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref139"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">142.</div><div>Sallusto, F. ∙ Lenig, D. ∙ Mackay, C.R. ...</div><div><strong>Flexible programs of chemokine receptor expression on human polarized T helper 1 and 2 lymphocytes</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1998; <strong>187</strong>:875-883</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.187.6.875" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0032536869" target="_blank" aria-label="Open Scopus in new tab">Scopus (1325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9500790/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.187.6.875&amp;pmid=9500790" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), were missing in P (<a id="crosref1500" href="#fig4">Figure&nbsp;4</a>F; <a id="crosref1505" href="#figs4">Figures S4</a>D and S4E). The frequency of classic CXCR3<sup>+</sup>CCR6<sup>–</sup> T<sub>H</sub>1 cells was lower than that in all healthy adult and age-matched donors examined, whereas the frequency of CXCR3<sup>+</sup>CCR6<sup>+</sup> non-classic T<sub>H</sub>1<sup>∗</sup> cells, which are known to be mostly <i>Mycobacterium</i>-specific, was comparable to that of age-matched healthy donors (<a id="crosref1510" href="#fig4">Figures 4</a>G and 4H). The frequencies of human T<sub>H</sub>2, T<sub>H</sub>17, and follicular helper (T<sub>FH</sub>) cells in peripheral blood were normal (<a id="crosref1515" href="#figs4">Figures S4</a>F–S4H). However, levels of CXCR3<sup>+</sup> T<sub>FH</sub> cells, a group of T<sub>H</sub>1-biased T<sub>FH</sub> cells that produce IFN-γ together with IL-21 in germinal centers (<span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib112" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Ma et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">114.</div><div>Ma, C.S. ∙ Wong, N. ∙ Rao, G. ...</div><div><strong>Monogenic mutations differentially affect the quantity and quality of T follicular helper cells in patients with human primary immunodeficiencies</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2015; <strong>136</strong>:993-1006</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2015.05.036&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2015.05.036&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84943454615" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26162572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2015.05.036&amp;pmid=26162572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref164" href="#" role="doc-biblioref" data-xml-rid="bib164" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref164" aria-controls="sref164">Velu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref164"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">167.</div><div>Velu, V. ∙ Mylvaganam, G.H. ∙ Gangadhara, S. ...</div><div><strong>Induction of Th1-Biased T Follicular Helper (Tfh) Cells in Lymphoid Tissues during Chronic Simian Immunodeficiency Virus Infection Defines Functionally Distinct Germinal Center Tfh Cells</strong></div><div><em>J.&nbsp;Immunol.</em> 2016; <strong>197</strong>:1832-1842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1600143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84983776546" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27481845/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1600143&amp;pmid=27481845" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref176" href="#" role="doc-biblioref" data-xml-rid="bib176" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref176" aria-controls="sref176">Zhang et&nbsp;al., 2019a</a><div class="dropBlock__holder" data-db-target-of="sref176"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">179.</div><div>Zhang, J. ∙ Liu, W. ∙ Wen, B. ...</div><div><strong>Circulating CXCR3<sup>+</sup> Tfh cells positively correlate with neutralizing antibody responses in HCV-infected patients</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:10090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-46533-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85068921679" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31300682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-46533-w&amp;pmid=31300682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), were low in P (<a id="crosref1535" href="#figs4">Figure&nbsp;S4</a>H). CXCR3<sup>+</sup> regulatory T&nbsp;cells (Tregs), a group of T<sub>H</sub>1-skewed Tregs (<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Koch et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Koch, M.A. ∙ Tucker-Heard, G. ∙ Perdue, N.R. ...</div><div><strong>The transcription factor T-bet controls regulatory T&nbsp;cell homeostasis and function during type 1 inflammation</strong></div><div><em>Nat. Immunol.</em> 2009; <strong>10</strong>:595-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1731" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-65749103365" target="_blank" aria-label="Open Scopus in new tab">Scopus (805)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19412181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1731&amp;pmid=19412181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref95" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95" aria-controls="sref95">Levine et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref95"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Levine, A.G. ∙ Mendoza, A. ∙ Hemmers, S. ...</div><div><strong>Stability and function of regulatory T&nbsp;cells expressing the transcription factor T-bet</strong></div><div><em>Nature.</em> 2017; <strong>546</strong>:421-425</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22360" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85020935637" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28607488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22360&amp;pmid=28607488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref152" href="#" role="doc-biblioref" data-xml-rid="bib152" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref152" aria-controls="sref152">Tan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref152"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">155.</div><div>Tan, T.G. ∙ Mathis, D. ∙ Benoist, C.</div><div><strong>Singular role for T-BET+CXCR3+ regulatory T&nbsp;cells in protection from autoimmune diabetes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:14103-14108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1616710113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85002497847" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27872297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1616710113&amp;pmid=27872297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), were also present at abnormally low levels, but the proportion of total Tregs was normal (<a id="crosref1555" href="#figs4">Figures S4</a>I and S4J). Thus, human T-bet deficiency selectively impairs development of the classic CXCR3<sup>+</sup>CCR6<sup>–</sup> T<sub>H</sub>1, CXCR3<sup>+</sup> T<sub>FH</sub>, and CXCR3<sup>+</sup> Treg CD4<sup>+</sup> T&nbsp;cell subsets but has no effect on the T<sub>FH</sub>, T<sub>H</sub>2, T<sub>H</sub>17, CCR6<sup>+</sup> T<sub>H</sub>1<sup>∗</sup>, and total Treg subsets, as shown by CyTOF and flow cytometry.</div></section><section id="sec-2-10"><h3>Single-Cell Transcriptomic Profile <i>In&nbsp;Vivo</i> Is Altered by T-bet Deficiency</h3><div id="p0085" role="paragraph">We investigated the development and phenotype of leukocyte subsets in the patient further, by performing single-cell RNA-seq (scRNA-seq) with PBMCs from P and his father. The clustering of the various immune subsets yielded eight distinct major subsets: NK cells, pDCs, monocytes, B cells, CD8<sup>+</sup> cytotoxic T lymphocytes (CD8<sup>+</sup> CTLs), CD8<sup>+</sup> naive, CD4<sup>+</sup> naive, and CD4<sup>+</sup> effector/memory T (T<sub>EM</sub>) cells (<span class="dropBlock reference-citations"><a id="body-ref-sref10-1" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-1" aria-controls="sref10-1">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref10-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (449)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with the CyTOF results, normal frequencies of pDC, CD8<sup>+</sup> CTLs, CD4<sup>+</sup> naive, and T<sub>EM</sub> cells were obtained, together with a low frequency of NK cells, on scRNA-seq (<a id="crosref1565" href="#mmc1">Data S1</a>). We investigated the transcriptomic changes at single-cell level associated with T-bet deficiency, by filtering to select all genes with expression detected in &gt;5% of cells in at least one cluster, with at least a 4-fold change in expression. We identified 34 genes as differentially regulated in T-bet-deficient cells relative to a heterozygous control. As for the RNA-seq data, some targets with expression known to be dependent on T-bet, including <i>CXCR3</i> in T<sub>EM</sub>, CD8<sup>+</sup> CTLs, and NK cells, and <i>CCL4</i> and <i>CCL3</i> in all cell types, were downregulated in T-bet-deficient cells (<a id="crosref1570" href="#fig4">Figure&nbsp;4</a>I). The expression of <i>XCL1</i>, <i>STAT4</i>, <i>SOX4</i>, <i>LMNA</i>, and <i>ANXA1</i> was also impaired in at least one subset of T-bet-deficient cells (<a id="crosref1575" href="#fig4">Figure&nbsp;4</a>I). In humans and mice, XCL1, STAT4, SOX4, LMNA, and ANXA1 are known to be involved in T<sub>H</sub>1 immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Dorner et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Dorner, B.G. ∙ Scheffold, A. ∙ Rolph, M.S. ...</div><div><strong>MIP-1alpha, MIP-1beta, RANTES, and ATAC/lymphotactin function together with IFN-gamma as type 1 cytokines</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:6181-6186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.092141999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037197261" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11972057/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.092141999&amp;pmid=11972057" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" aria-label="Reference 2003" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">2003</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Dorner, B.G. ∙ Steinbach, S. ∙ Hüser, M.B. ...</div><div><strong>Single-cell analysis of the murine chemokines MIP-1alpha, MIP-1beta, RANTES and ATAC/lymphotactin by flow cytometry</strong></div><div><em>J.&nbsp;Immunol. Methods.</em> 2003; <strong>274</strong>:83-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-1759(02)00498-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037374062" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12609535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-1759%2802%2900498-2&amp;pmid=12609535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" aria-label="Reference 2004" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">2004</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Dorner, B.G. ∙ Smith, H.R.C. ∙ French, A.R. ...</div><div><strong>Coordinate expression of cytokines and chemokines by NK cells during murine cytomegalovirus infection</strong></div><div><em>J.&nbsp;Immunol.</em> 2004; <strong>172</strong>:3119-3131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.172.5.3119" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-1342303341" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14978118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.172.5.3119&amp;pmid=14978118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Gavins and Hickey, 2012</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Gavins, F.N.E. ∙ Hickey, M.J.</div><div><strong>Annexin A1 and the regulation of innate and adaptive immunity</strong></div><div><em>Front. Immunol.</em> 2012; <strong>3</strong>:354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2012.00354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84874215363" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23230437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2012.00354&amp;pmid=23230437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref90" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90" aria-controls="sref90">Kroczek and Henn, 2012</a><div class="dropBlock__holder" data-db-target-of="sref90"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Kroczek, R.A. ∙ Henn, V.</div><div><strong>The Role of XCR1 and its Ligand XCL1 in Antigen Cross-Presentation by Murine and Human Dendritic Cells</strong></div><div><em>Front. Immunol.</em> 2012; <strong>3</strong>:14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2012.00014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84874067035" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22566900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2012.00014&amp;pmid=22566900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref122" href="#" role="doc-biblioref" data-xml-rid="bib122" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref122" aria-controls="sref122">Nishikomori et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref122"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">124.</div><div>Nishikomori, R. ∙ Usui, T. ∙ Wu, C.-Y. ...</div><div><strong>Activated STAT4 has an essential role in Th1 differentiation and proliferation that is independent of its role in the maintenance of IL-12R beta 2 chain expression and signaling</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:4388-4398</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.8.4388" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12370372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.8.4388&amp;pmid=12370372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref156" href="#" role="doc-biblioref" data-xml-rid="bib156" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref156" aria-controls="sref156">Thieu et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref156"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">159.</div><div>Thieu, V.T. ∙ Yu, Q. ∙ Chang, H.-C. ...</div><div><strong>Signal transducer and activator of transcription 4 is required for the transcription factor T-bet to promote T helper 1 cell-fate determination</strong></div><div><em>Immunity.</em> 2008; <strong>29</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2008.08.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2008.08.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-55349117409" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18993086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2008.08.017&amp;pmid=18993086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref158" href="#" role="doc-biblioref" data-xml-rid="bib158" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref158" aria-controls="sref158">Toribio-Fernández et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref158"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">161.</div><div>Toribio-Fernández, R. ∙ Zorita, V. ∙ Rocha-Perugini, V. ...</div><div><strong>Lamin A/C augments Th1 differentiation and response against vaccinia virus and Leishmania major</strong></div><div><em>Cell Death Dis.</em> 2018; <strong>9</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41419-017-0007-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85042799473" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29311549/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41419-017-0007-6&amp;pmid=29311549" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref171" href="#" role="doc-biblioref" data-xml-rid="bib171" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref171" aria-controls="sref171">Yoshitomi et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref171"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">174.</div><div>Yoshitomi, H. ∙ Kobayashi, S. ∙ Miyagawa-Hayashino, A. ...</div><div><strong>Human Sox4 facilitates the development of CXCL13-producing helper T&nbsp;cells in inflammatory environments</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:3762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06187-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85053538844" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30232328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06187-0&amp;pmid=30232328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). NKG7 is involved in the initiation of human T<sub>H</sub>1 commitment, and its genetic locus is tightly occupied by T-bet (<span class="dropBlock reference-citations"><a id="body-ref-sref78-2" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78-2" aria-controls="sref78-2">Jenner et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref78-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Jenner, R.G. ∙ Townsend, M.J. ∙ Jackson, I. ...</div><div><strong>The transcription factors T-bet and GATA-3 control alternative pathways of T-cell differentiation through a shared set of target genes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:17876-17881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0909357106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-70449571945" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19805038/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0909357106&amp;pmid=19805038" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref81-3" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-3" aria-controls="sref81-3">Kanhere et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref81-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Kanhere, A. ∙ Hertweck, A. ∙ Bhatia, U. ...</div><div><strong>T-bet and GATA3 orchestrate Th1 and Th2 differentiation through lineage-specific targeting of distal regulatory elements</strong></div><div><em>Nat. Commun.</em> 2012; <strong>3</strong>:1268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms2260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871783406" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms2260&amp;pmid=23232398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref110" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110" aria-controls="sref110">Lund et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref110"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Lund, R. ∙ Ahlfors, H. ∙ Kainonen, E. ...</div><div><strong>Identification of genes involved in the initiation of human Th1 or Th2 cell commitment</strong></div><div><em>Eur. J. Immunol.</em> 2005; <strong>35</strong>:3307-3319</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/eji.200526079" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-27944450018" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16220538/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Feji.200526079&amp;pmid=16220538" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). <i>NKG7</i> expression in naive CD4<sup>+</sup>, CD8<sup>+</sup>, and B cells was dependent on functional T-bet, whereas <i>PRDM1</i> was downregulated in CD4<sup>+</sup> T and CD8<sup>+</sup> CTLs cells from P (<a id="crosref1640" href="#fig4">Figure&nbsp;4</a>I). The <i>IFNG</i> gene was weakly expressed across lymphocyte populations, as shown by scRNA-seq, and its expression did not seem to be dependent on T-bet in basal conditions (data not shown). In addition, the expression of several genes not previously linked to T-bet was also altered in at least one cell subset in P (<a id="crosref1645" href="#fig4">Figure&nbsp;4</a>I). Thus, in addition to <i>CXCR3</i>, <i>NKG7</i>, <i>CCL3</i>, and <i>CCL4</i>, which were weakly expressed in at least one leukocyte subset from this patient with human T-bet deficiency, consistent with the findings of RNA-seq and omni-ATAC-seq, the expression of a set of previously unknown target genes in immune subsets is also controlled by T-bet (<a id="crosref1650" href="#fig4">Figure&nbsp;4</a>I).</div></section><section id="sec-2-11"><h3>Impaired IFN-γ Production by NK, MAIT, Vδ2<sup>+</sup> γδ T, and CD4<sup>+</sup> T&nbsp;cells</h3><div id="p0090" role="paragraph">Human IFN-γ is essential for antimycobacterial immunity, as all 31 known genetic etiologies of MSMD affect IFN-γ-dependent immunity. The <i>in&nbsp;vivo</i> development of NK, iNKT, MAIT, Vδ2<sup>+</sup> γδ T, and classic T<sub>H</sub>1 cells was found to be impaired in P, but it remained possible that the IFN-γ production capacity of the remaining lymphocytes could compensate, thereby contributing to antimycobacterial immunity. We assessed the response of P’s PBMCs to <i>ex&nbsp;vivo</i> stimulation with P/I. They secreted significantly less IFN-γ secretion than control cells (<a id="crosref1655" href="#fig5">Figure&nbsp;5</a>A). The frequencies of IFN-γ- or TNF-α-producing total lymphocytes were also low in P (<a id="crosref1660" href="#fig5">Figures 5</a>B–5D). We assessed the IFN-γ and TNF-α production of lymphocyte subsets by spectral flow cytometry (<a id="crosref1665" href="#figs5">Figure&nbsp;S5</a>A). Upon P/I stimulation, the frequencies of IFN-γ- or TNF-α-producing CD56<sup>+</sup> NK cells were low in P (<a id="crosref1670" href="#fig5">Figure&nbsp;5</a>E; <a id="crosref1675" href="#figs5">Figures S5</a>B and S5C). We then assessed of the response of P’s NK cells to <i>ex&nbsp;vivo</i> stimulation with IL-12, IL-15, and IL-18. Total NK cells from P displayed impaired degranulation, with low levels of CD107a expression, and almost no IFN-γ production (<a id="crosref1680" href="#fig5">Figure&nbsp;5</a>F; <a id="crosref1685" href="#figs5">Figures S5</a>D and S5E). T-bet deficiency not only reduced the overall frequencies of iNKT, MAIT, and Vδ2<sup>+</sup> γδ T&nbsp;cells but also compromised the production of IFN-γ and TNF-α <i>ex&nbsp;vivo</i> by the residual activated iNKT, MAIT, and Vδ2<sup>+</sup> γδ T&nbsp;cells detected in P (<a id="crosref1690" href="#fig5">Figures 5</a>G–5K; <a id="crosref1695" href="#figs5">Figure&nbsp;S5</a>F). Despite an increase in the frequency of Vδ1<sup>+</sup> γδ T&nbsp;cells <i>in&nbsp;vivo</i>, the levels of IFN-γ- or TNF-α-producing-Vδ1<sup>+</sup> γδ T&nbsp;cells were also low <i>ex&nbsp;vivo</i> (<a id="crosref1700" href="#fig5">Figures 5</a>G, 5L, and 5M). By contrast, we observed no difference in IFN-γ and TNF-α production by Vδ1<sup>–</sup>Vδ2<sup>–</sup> γδ T&nbsp;cells between P and controls (<a id="crosref1705" href="#fig5">Figures 5</a>G; <a id="crosref1710" href="#figs5">Figures S5</a>G and S5H). IFN-γ-producing cells from the various lymphocyte subsets examined in healthy donors expressed high levels of T-bet, but all the residual IFN-γ-producing cells from the various subsets in P tested negative for T-bet (<a id="crosref1715" href="#fig5">Figure&nbsp;5</a>G). We also analyzed the response of αβ T&nbsp;cells to P/I. CD8<sup>+</sup> T&nbsp;cells secrete substantial amounts of IFN-γ upon microbial challenge (<span class="dropBlock reference-citations"><a id="body-ref-sref141-2" href="#" role="doc-biblioref" data-xml-rid="bib141" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref141-2" aria-controls="sref141-2">Schoenborn and Wilson, 2007</a><div class="dropBlock__holder" data-db-target-of="sref141-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">144.</div><div>Schoenborn, J.R. ∙ Wilson, C.B.</div><div><strong>Regulation of interferon-γ during innate and adaptive immune responses</strong></div><div><em>Adv. Immunol.</em> 2007; <strong>96</strong>:41-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0065-2776(07)96002-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-35549003579" target="_blank" aria-label="Open Scopus in new tab">Scopus (968)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17981204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0065-2776%2807%2996002-2&amp;pmid=17981204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The frequencies of CD8<sup>+</sup> T&nbsp;cells producing IFN-γ or TNF-α in response to P/I did not differ between P and age-matched healthy donors (<a id="crosref1725" href="#figs5">Figures S5</a>I and S5J). By contrast, the frequencies of IFN-γ- or TNF-α-producing CD4<sup>+</sup> T&nbsp;cells were slightly low in P (<a id="crosref1730" href="#fig5">Figures 5</a>N and 5O). Thus, among the residual circulating lymphocytes of P, NK, iNKT, MAIT, Vδ2<sup>+</sup>, Vδ1<sup>+</sup> γδ T, and CD4<sup>+</sup> T&nbsp;cells had impaired IFN-γ production <i>ex&nbsp;vivo</i> in response to stimulation with IL-12, IL-15, IL-18, or P/I, whereas Vδ1<sup>–</sup>Vδ2<sup>–</sup> γδ and CD8<sup>+</sup> αβ T&nbsp;cells did not.</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/a8408118-2af8-4ac3-a800-fabf5e4610e4/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2020.10.046/asset/d8680554-2771-4510-ad79-25392b88fbc9/main.assets/gr5.jpg" height="937" width="714" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/a8408118-2af8-4ac3-a800-fabf5e4610e4/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure 5</span> <span class="figure__title__text">Impaired IFN-γ <i>Ex Vivo</i> Production from the Remaining Lymphocytes of the T-bet-Deficient Patient</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0250" role="paragraph">(A) IFN-γ production in culture supernatants of PBMCs from indicated individuals in response to stimulation with P/I is shown.</div><div id="fspara0255" role="paragraph">(B) Dot plots showing the expression of T-bet and IFN-γ in total live lymphocytes as in (A).</div><div id="fspara0260" role="paragraph">(C and D) Frequency of IFN-γ (C) or TNF-α-producing (D) cells in response to P/I as in (B).</div><div id="fspara0265" role="paragraph">(E) Frequency of IFN-γ-producing cells among NK cells in response to P/I.</div><div id="fspara0270" role="paragraph">(F) PBMCs were stimulated with IL-12, IL-15, and IL-18. The expression of CD107a and IFN-γ among NK cells were assessed by flow cytometry.</div><div id="fspara0275" role="paragraph">(G) Dot plots showing the expression of T-bet and IFN-γ by indicated subsets in response to P/I.</div><div id="fspara0280" role="paragraph">(H–O) Frequency of IFN-γ- (H, J, L, and N) or TNF-α (I, K, M, and O)-producing cells among MAIT cells, Vδ2<sup>+</sup> γδ T cells, V1<sup>+</sup> γδ T cells, and CD4<sup>+</sup> T cells in response to P/I.</div><div id="fspara0285" role="paragraph">(P–U) Fluorescence-activated cell sorted (FACS) naive and memory CD4<sup>+</sup> T cells were activated/expanded. The production of IFN-γ was assessed intracellularly (P) and in culture supernatants (Q). The production of TNF-α was assessed intracellularly (R) and in culture supernatants (S). The production of IL-22 (T) and IL-17A (U) was assessed in culture supernatants.</div><div id="fspara0290" role="paragraph">(V–Y) Naive CD4<sup>+</sup> T cells were subjected to activation under T<sub>H</sub>0 or T<sub>H</sub>1 conditions. The production of IFN-γ (V and W) and TNF-α (X and Y) was assessed.</div><div id="fspara0295" role="paragraph">(Z) Memory CD4<sup>+</sup> T cells were activated under T<sub>H</sub>0 or T<sub>H</sub>1 conditions. The production of IFN-γ and TNF-α was assessed.</div><div id="fspara0300" role="paragraph">In (A), (C)–(F), and (H)–(Z), bars represent the mean and the standard error of the mean. Dots represent individual samples or technical replicates. Mann-Whitney nonparametric tests were used for analysis in (A), (C)–(E), and (H)–(O). In (A), (C)–(E), and (H)–(O), <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001; ns, not significant.</div><div id="fspara0305" role="paragraph">See also <a id="crosref2780" href="#figs5">Figure S5</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/2bc74197-3a6e-4e91-9406-1dd94da80393/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2020.10.046/asset/6698186b-bcde-442b-9ff4-e87e7067b345/main.assets/figs5.jpg" height="949" width="766" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/2bc74197-3a6e-4e91-9406-1dd94da80393/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure S5</span> <span class="figure__title__text"><i>Ex Vivo</i> and <i>In Vitro</i> Non-mycobacterial Stimulation of the Residual Lymphocytes from the Patient with T-bet Deficiency, Related to <a id="crosref2820" href="#fig5">Figure 5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0445" role="paragraph"><b>(A)</b> Gating strategy for the surface and intracellular staining of PBMCs <i>ex vivo</i> stimulated with P/I, for an age-matched control sample. (<b>B</b>) Intracellular staining of IFN-γ and T-bet in NK cells, as shown in (A), from an age-matched control (Age CTL), P (M/M) and P’s healthy brother (WT/WT) stimulated with P/I. (<b>C</b>) Frequencies of TNF-α-producing cells among CD56<sup>+</sup> NK cells, as gated in (A), in the indicated individuals. (<b>D</b>) The gating strategy for gating total NK cells from a travel control (CTL), P (M/M), P’s brother (WT/WT) and P’s heterozygous mother (WT/M), in the presence and absence of stimulation with IL-12, IL-15 and IL-18, is shown. (<b>E</b>) Expression of CD107a and IFN-γ among total NK cells, as in (D). (<b>F</b>) Frequencies of IFN-γ-producing cells among iNKT cells, as gated in (A) for the indicated individuals. (<b>G</b>) and (<b>H</b>) Vδ1<sup>-</sup>Vδ2<sup>-</sup> γδ T cells were gated from PBMCs prepared as shown in (A). Intracellular staining for IFN-γ (G) and TNF-α (H) is shown. (<b>I</b>) and (<b>J</b>) CD8<sup>+</sup> T cells were gated from PBMCs prepared as shown in (A). Intracellular staining for IFN-γ (I) and TNF-α (J) is shown. (<b>K - R</b>) Naive and memory CD4<sup>+</sup> T cells from controls (CTL) and P (M/M) were sorted by FACS and activated/expanded. Intracellular staining of IL-22 (K), IL-9 (L) and IL-21 (R). Production of IL-9 (M), IL-4 (N), IL-5 (O), IL-10 (P) and IL-13 (Q), as measured in culture supernatants. (<b>S</b>) and (<b>T</b>) Naive CD4<sup>+</sup> T cells were isolated by FACS and subjected to activation under T<sub>H</sub>0, T<sub>H</sub>9, T<sub>H</sub>17 or T<sub>H</sub>9 conditions combined with T<sub>H</sub>17 conditions. The production of IL-9 was assessed intracellularly (S) and in culture supernatants (T). (<b>U</b>) Naive CD4<sup>+</sup> T cells were isolated by FACS and subjected to activation under T<sub>H</sub>0, T<sub>H</sub>1 or T<sub>H</sub>2 conditions. IL-10 production, determined with culture supernatants, is shown. (<b>V</b>) Naive CD4<sup>+</sup> T cells were isolated by FACS and subjected to activation under T<sub>H</sub>0, T<sub>H</sub>2 or T<sub>H</sub>9 conditions. IL-13 production, as measured on culture supernatants.</div><div id="fspara0450" role="paragraph">In (C, F - V), bars represent the mean and the standard error of the mean. Dots represent individual samples. Mann-Whitney non-parametric tests were used for analysis in (C, F - J). In (C, F - J), <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001, ns = not significant.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-12"><h3>Selective Impairment of IFN-γ Production by T<sub>H</sub> Cells in T-bet Deficiency</h3><div id="p0095" role="paragraph">T-bet was first discovered and has been most extensively studied in CD4<sup>+</sup> T&nbsp;cells in the context of mouse T<sub>H</sub>1 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref149-3" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149-3" aria-controls="sref149-3">Szabo et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref149-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Szabo, S.J. ∙ Kim, S.T. ∙ Costa, G.L. ...</div><div><strong>A novel transcription factor, T-bet, directs Th1 lineage commitment</strong></div><div><em>Cell.</em> 2000; <strong>100</strong>:655-669</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10761931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980702-3&amp;pmid=10761931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref150-3" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" aria-label="Reference 2002" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-3" aria-controls="sref150-3">2002</a><div class="dropBlock__holder" data-db-target-of="sref150-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Szabo, S.J. ∙ Sullivan, B.M. ∙ Stemmann, C. ...</div><div><strong>Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:338-342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059462" target="_blank" aria-label="Open Scopus in new tab">Scopus (920)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065543&amp;pmid=11786644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This discovery, together with that of GATA3 (<span class="dropBlock reference-citations"><a id="body-ref-sref178" href="#" role="doc-biblioref" data-xml-rid="bib178" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref178" aria-controls="sref178">Zheng and Flavell, 1997</a><div class="dropBlock__holder" data-db-target-of="sref178"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">181.</div><div>Zheng, W. ∙ Flavell, R.A.</div><div><strong>The transcription factor GATA-3 is necessary and sufficient for Th2 cytokine gene expression in CD4 T&nbsp;cells</strong></div><div><em>Cell.</em> 1997; <strong>89</strong>:587-596</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980240-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980240-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9160750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980240-8&amp;pmid=9160750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), revealed the molecular determinism of T<sub>H</sub>1/T<sub>H</sub>2 CD4<sup>+</sup> T&nbsp;cell differentiation and paved the way for an understanding of T<sub>H</sub>17, iTreg, T<sub>H</sub>22, T<sub>FH</sub>, and T<sub>H</sub>9 cell lineage determination (<span class="dropBlock reference-citations"><a id="body-ref-sref48" href="#" role="doc-biblioref" data-xml-rid="bib48" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref48" aria-controls="sref48">Finotto et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref48"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Finotto, S. ∙ Neurath, M.F. ∙ Glickman, J.N. ...</div><div><strong>Development of spontaneous airway changes consistent with human asthma in mice lacking T-bet</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:336-338</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065544" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059509" target="_blank" aria-label="Open Scopus in new tab">Scopus (513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786643/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065544&amp;pmid=11786643" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref179" href="#" role="doc-biblioref" data-xml-rid="bib179" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref179" aria-controls="sref179">Zhu et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref179"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">182.</div><div>Zhu, J. ∙ Yamane, H. ∙ Paul, W.E.</div><div><strong>Differentiation of effector CD4 T&nbsp;cell populations (<sup>∗</sup>)</strong></div><div><em>Annu. Rev. Immunol.</em> 2010; <strong>28</strong>:445-489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-030409-101212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77952313777" target="_blank" aria-label="Open Scopus in new tab">Scopus (1795)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20192806/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-030409-101212&amp;pmid=20192806" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We therefore investigated the impact of T-bet deficiency on primary CD4<sup>+</sup> T&nbsp;cells. IFN-γ production by memory CD4<sup>+</sup> T&nbsp;cells was impaired by T-bet deficiency (<a id="crosref1760" href="#fig5">Figures 5</a>P and 5Q). Another T<sub>H</sub>1 cytokine, TNF-α, was also produced in smaller amounts by memory CD4<sup>+</sup> T&nbsp;cells from P than by those of most of the healthy donors (<a id="crosref1765" href="#fig5">Figures 5</a>R and 5S). By contrast, the memory CD4<sup>+</sup> T&nbsp;cells of P produced larger amounts of the T<sub>H</sub>17 effector cytokines IL-17A and IL-22 than did those of healthy donors (<a id="crosref1770" href="#fig5">Figures 5</a>T and 5U; <a id="crosref1775" href="#figs5">Figure&nbsp;S5</a>K). Unlike previous studies (<span class="dropBlock reference-citations"><a id="body-ref-sref56" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56" aria-controls="sref56">Gökmen et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref56"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Gökmen, M.R. ∙ Dong, R. ∙ Kanhere, A. ...</div><div><strong>Genome-wide regulatory analysis reveals that T-bet controls Th17 lineage differentiation through direct suppression of IRF4</strong></div><div><em>J.&nbsp;Immunol.</em> 2013; <strong>191</strong>:5925-5932</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1202254" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84890418697" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24249732/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1202254&amp;pmid=24249732" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref177" href="#" role="doc-biblioref" data-xml-rid="bib177" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref177" aria-controls="sref177">Zhang et&nbsp;al., 2019b</a><div class="dropBlock__holder" data-db-target-of="sref177"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">180.</div><div>Zhang, Y. ∙ Siegel, A.M. ∙ Sun, G. ...</div><div><strong>Human T<sub>H</sub>9 differentiation is dependent on signal transducer and activator of transcription (STAT) 3 to restrain STAT1-mediated inhibition</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2019; <strong>143</strong>:1108-1118</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2018.06.036&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2018.06.036&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30030006/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2018.06.036&amp;pmid=30030006" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we found that T-bet deficiency had no effect on IL-9 production <i>ex&nbsp;vivo</i> (<a id="crosref1790" href="#figs5">Figures S5</a>L and S5M). Surprisingly, <i>ex&nbsp;vivo</i> T<sub>H</sub>2 cytokines from memory CD4<sup>+</sup> T&nbsp;cells were also not affected by human T-bet deficiency (<a id="crosref1795" href="#figs5">Figures S5</a>N–S5Q). P’s memory CD4<sup>+</sup> cells produced less IL-21 <i>ex&nbsp;vivo</i> than the memory CD4<sup>+</sup> cells of most of the healthy donors (<a id="crosref1800" href="#figs5">Figure&nbsp;S5</a>R). We then investigated the role of T-bet in human T<sub>H</sub> cell differentiation <i>in&nbsp;vitro</i>. Naive CD4<sup>+</sup> T&nbsp;cells from P or healthy donors were allowed to differentiate in T<sub>H</sub>0, T<sub>H</sub>1, T<sub>H</sub>2, T<sub>H</sub>9, or T<sub>H</sub>17 conditions. The induction of IFN-γ production in naive CD4<sup>+</sup> T&nbsp;cells was abolished by T-bet deficiency under T<sub>H</sub>1 conditions (<a id="crosref1805" href="#fig5">Figures 5</a>V and 5W). Similarly, T-bet-deficient naive CD4<sup>+</sup> T&nbsp;cells produced less TNF-α than the corresponding cells from most controls (<a id="crosref1810" href="#fig5">Figures 5</a>X and 5Y). Furthermore, the induction of IL-9 under various conditions <i>in&nbsp;vitro</i> was weaker in naive CD4<sup>+</sup> T&nbsp;cells from P than in the corresponding cells from most controls (<a id="crosref1815" href="#figs5">Figures S5</a>S and S5T). <i>In&nbsp;vitro</i>-induced T<sub>H</sub>2 cells from P produced more IL-10, but not IL-13, than control cells (<a id="crosref1820" href="#figs5">Figures S5</a>U and S5V). Even memory CD4<sup>+</sup> T&nbsp;cells from P displayed impaired IFN-γ and TNF-α production under T<sub>H</sub>1 polarizing conditions (<a id="crosref1825" href="#fig5">Figure&nbsp;5</a>Z). Thus, AR T-bet deficiency leads not only to defective IFN-γ and TNF-α production <i>ex&nbsp;vivo</i> and <i>in&nbsp;vitro</i> but also to a moderate upregulation of the production of IL-17A and IL-22, two cytokines characteristic of T<sub>H</sub>17 cells (<span class="dropBlock reference-citations"><a id="body-ref-sref92" href="#" role="doc-biblioref" data-xml-rid="bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92" aria-controls="sref92">Lazarevic et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref92"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Lazarevic, V. ∙ Chen, X. ∙ Shim, J.H. ...</div><div><strong>T-bet represses T(H)17 differentiation by preventing Runx1-mediated activation of the gene encoding RORγt</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:96-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-78650306567" target="_blank" aria-label="Open Scopus in new tab">Scopus (243)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21151104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1969&amp;pmid=21151104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-13"><h3>Poor Cellular Response to BCG Infection <i>In&nbsp;Vitro</i> in T-bet Deficiency</h3><div id="p0100" role="paragraph">We investigated the molecular and cellular basis of BCG disease in P, by identifying the leukocyte subsets producing the largest amounts of IFN-γ in a T-bet-dependent manner during acute BCG infection <i>in&nbsp;vitro</i>. The infection of PBMCs with BCG induced IFN-γ production, which was further increased by stimulation with exogenous IL-12 (<a id="crosref1835" href="#fig6">Figure&nbsp;6</a>A). PBMCs from P had low levels of IFN-γ and TNF-α production, but the level of IL-5 production in response to BCG infection was higher than that in most of the controls <b>(</b><a id="crosref1840" href="#fig6">Figure&nbsp;6</a>A; <a id="crosref1845" href="#figs6">Figures S6</a>A and S6B). Almost all the IFN-γ-producing cells in controls had high levels of T-bet expression (<a id="crosref1850" href="#figs6">Figure&nbsp;S6</a>C). Thus, T-bet<sup>+</sup> IFN-γ<sup>+</sup> double-positive (DP) cells were the major <i>Mycobacterium</i>-responsive cells, with a function potentially dependent on T-bet. However, T-bet<sup>+</sup> IFN-γ DP cell levels were low during acute infection in P (<a id="crosref1855" href="#fig6">Figure&nbsp;6</a>B; <a id="crosref1860" href="#figs6">Figure&nbsp;S6</a>C). T-bet<sup>+</sup> IFN-γ<sup>+</sup> DP cells induced by exogenous IL-12 alone comprised predominantly NK cells, whereas those induced by IL-23 alone also contained a large proportion of MAIT cells, in addition to NK cells (<a id="crosref1865" href="#figs6">Figure&nbsp;S6</a>D). Among the T-bet<sup>+</sup> IFN-γ<sup>+</sup> cells of healthy donors, NK, MAIT, Vδ2<sup>+</sup> γδ T, CD4<sup>+</sup>, and CD8<sup>+</sup> αβ T&nbsp;cells were the dominant responders during BCG infection, whereas iNKT, Vδ1<sup>+</sup> γδ T, Vδ1<sup>–</sup>Vδ2<sup>–</sup> γδ T, and B cells were minority responders (<a id="crosref1870" href="#fig6">Figures 6</a>C and 6D). However, these subsets of T-bet<sup>+</sup> IFN-γ<sup>+</sup> cells were almost entirely depleted from P’s PBMCs following BCG infection (<a id="crosref1875" href="#fig6">Figure&nbsp;6</a>D). We then investigated each leukocyte subset separately (<a id="crosref1880" href="#mmc1">Data S1</a>). Fewer than 2% of Vδ1<sup>+</sup> γδ T&nbsp;cells, Vδ1<sup>–</sup>Vδ2<sup>–</sup> γδ T&nbsp;cells, B&nbsp;cells, CD4<sup>+</sup>, or CD8<sup>+</sup> αβ T&nbsp;cells became T-bet<sup>+</sup> IFN-γ<sup>+</sup> during BCG infection. However, among innate or innate-like lymphocytes, up to 10% of NK cells, 40% of iNKT, and 50% of Vδ2<sup>+</sup> γδ T&nbsp;cells from healthy donors, but not from P, became T-bet<sup>+</sup> IFN-γ<sup>+</sup> in response to BCG infection, and the frequency of these cells was further increased by exogenous IL-12 (<a id="crosref1885" href="#fig6">Figures 6</a>E–6G; <a id="crosref1890" href="#figs6">Figures S6</a>E and S6F). Up to 50% of MAIT cells became T-bet<sup>+</sup> IFN-γ<sup>+</sup> during BCG infection, but no difference was observed between age-matched healthy donors and P (<a id="crosref1895" href="#fig6">Figure&nbsp;6</a>E; <a id="crosref1900" href="#figs6">Figure&nbsp;S6</a>E). Upon acute BCG infection <i>in&nbsp;vitro</i>, despite a less robust activation of CD4<sup>+</sup> T&nbsp;cells than of innate or innate-like lymphocytes, the frequency of T-bet<sup>+</sup> IFN-γ<sup>+</sup> cells was low among CD4<sup>+</sup> but not among CD8<sup>+</sup> αβ T&nbsp;cells of P (<a id="crosref1905" href="#fig6">Figure&nbsp;6</a>H; <a id="crosref1910" href="#figs6">Figures S6</a>G and S6H). Thus, the IFN-γ production controlled by T-bet during acute BCG infection <i>in&nbsp;vitro</i> takes place mostly in NK, iNKT, Vδ2<sup>+</sup> γδ T, and CD4<sup>+</sup> αβ T&nbsp;cells but not in Vδ1<sup>+</sup> γδ T&nbsp;cells, Vδ1<sup>–</sup>Vδ2<sup>–</sup> γδ T&nbsp;cells, or CD8<sup>+</sup> αβ T&nbsp;cells. The development of MAIT cells is also dependent on T-bet, whereas the IFN-γ production of these cells during BCG infection is not. These experimental findings <i>in&nbsp;vitro</i> do not exclude a contribution of other subsets <i>in&nbsp;vivo</i>. NK, iNKT, MAIT, and Vδ2<sup>+</sup> γδ T&nbsp;cells responded robustly to acute BCG infection <i>in&nbsp;vitro</i>, but these subsets were absent or functionally deficient in the patient with inherited T-bet deficiency.</div><div class="figure-wrap"><figure id="fig6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/01baddba-5c1c-4178-aa63-1e818c14be72/main.assets/gr6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr6" src="/cms/10.1016/j.cell.2020.10.046/asset/abe52ef8-1df5-4303-a9e5-354e3381677f/main.assets/gr6.jpg" height="937" width="689" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/01baddba-5c1c-4178-aa63-1e818c14be72/main.assets/gr6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig6-title" class="figure__title"><span class="label figure__label">Figure 6</span> <span class="figure__title__text">T-bet Deficiency Leads to Defective NK, iNKT, and Vδ2<sup>+</sup> γδT Cells, Resulting in Susceptibility to Mycobacteria</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0315" role="paragraph">(A) IFN-γ secretion by PBMCs from indicated individuals stimulated with and without live <i>M. bovis</i> BCG in the presence and absence of IL-12 and IL-23.</div><div id="fspara0320" role="paragraph">(B) Percentages of T-bet<sup>+</sup> IFN-γ<sup>+</sup> double-positive (DP) cells in the indicated samples, cultured as in (A).</div><div id="fspara0325" role="paragraph">(C) Percentages of indicated immune subsets among the DP cells of both CTL and age CTL individuals.</div><div id="fspara0330" role="paragraph">(D) viSNE plots showing different clusters of immune cells among DP cells.</div><div id="fspara0335" role="paragraph">(E) Percentages of DP cells within each immune subset, as indicated.</div><div id="fspara0340" role="paragraph">(F and G) Dot plots showing the expression of T-bet and IFN-γ in NK (F) and Vδ2<sup>+</sup> γδ T cells (G), cultured as in (A).</div><div id="fspara0345" role="paragraph">(H) Percentages of DP cells within each indicated adaptive immune subset.</div><div id="fspara0350" role="paragraph">(I and J) PBMCs from indicated individuals were stimulated with lysates of <i>M. bovis</i> BCG (BCG-lysates) for 14 days. Dot plots of Vγ9δ2 γδT cells (I) and the total number of Vγ9δ2 γδT cells (J) are shown.</div><div id="fspara0355" role="paragraph">(K) The level of IFN-γ in culture supernatants from (J) is shown.</div><div id="fspara0360" role="paragraph">(L–N) The intracellular IFN-γ production of antigen-specific CD4<sup>+</sup> or CD8<sup>+</sup> T cells expanded from PPD or BCG-lysates was measured. The percentages of IFN-γ-producing cells among PPD-responsive CD4<sup>+</sup> T (L), BCG-lysate-responsive CD4<sup>+</sup> (M), and CD8<sup>+</sup> T (N) cells are shown.</div><div id="fspara0365" role="paragraph">(O) Memory CCR6<sup>+</sup>CD4<sup>+</sup> T cell clones responding to the peptide pool from <i>M. bovis</i> BCG were identified. IFN-γ production in culture supernatants from these BCG-specific clones was shown.</div><div id="fspara0370" role="paragraph">In (A), (B), (E), (H), and (K)–(N), bars represent the mean and the standard error of the mean. Dots represent individual samples and technical replicates. In (O), bars represent the mean and the standard deviation. Dots represent individual T cell clones. In (C), the numbers indicate the mean and the standard error of the mean. Mann-Whitney nonparametric tests were used for analysis in (A), (B), (E), and (H). In (A), (B), (E), and (H), <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001; ns, not significant.</div><div id="fspara0375" role="paragraph">See also <a id="crosref2785" href="#figs6">Figure S6</a>, <a id="crosref2830" href="#mmc1">Table S4</a>, and <a id="crosref2795" href="#mmc1">Data S1</a>.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs6" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2020.10.046/asset/ecdb1012-6fb2-4219-ba9b-b4baa7c0a6c6/main.assets/figs6_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs6" src="/cms/10.1016/j.cell.2020.10.046/asset/1e40b6ed-39e0-4984-a517-4d4bc57a9c16/main.assets/figs6.jpg" height="878" width="759" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/ecdb1012-6fb2-4219-ba9b-b4baa7c0a6c6/main.assets/figs6_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs6-title" class="figure__title"><span class="label figure__label">Figure S6</span> <span class="figure__title__text">T-bet Deficiency Alters Immunity to <i>M. bovis</i> BCG, Related to <a id="crosref2825" href="#fig6">Figure 6</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs6-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0460" role="paragraph"><b>(A</b> and (<b>B</b>) PBMCs from adult controls (CTL), age-matched controls (Age CTL), P (M/M), P’s healthy brother (WT/WT), and P’s healthy parents (WT/M) were left unstimulated or stimulated with live <i>M. bovis</i> BCG in the presence and absence of IL-12 and IL-23. The production of TNF-α (A) and IL-5 (B) in culture supernatants was assessed with Legendplex cytometric bead arrays. (<b>C</b>) PBMCs from P (M/M), P’s healthy brother (WT/WT) and one of P’s healthy parents (WT/M), stimulated as in (A), were analyzed for the intracellular expression of T-bet and IFN-γ by flow cytometry. Percentages of T-bet<sup>+</sup> IFN-γ<sup>+</sup> cells within live single lymphocytes are highlighted. (<b>D</b>) Pie-charts of the percentages of the indicated immune subsets in PBMCs from healthy controls stmulated with IL-12 or IL-23 alone. (<b>E</b>) and (<b>F</b>) PBMCs from adult controls (CTL), age-matched controls (Age CTL), P (M/M), P’s healthy brother (WT/WT), and P’s healthy parents (WT/M) were left unstimulated or were stimulated with live <i>M. bovis</i> BCG in the presence and absence of IL-12 and IL-23. Frequencies of T-bet<sup>+</sup> IFN-γ<sup>+</sup> cells gated on each individual immune subset, including NK, iNKT, MAIT, Vδ2<sup>+</sup> γδ T (Vd2), Vδ1<sup>+</sup> γδ T (Vd1) and Vδ1<sup>-</sup>Vδ2<sup>-</sup> γδ T (Vd1-Vd1- gdT) cells from different groups of individuals, stimulated with BCG (E) and BCG + IL-23 (F), are shown. (<b>G</b>) Frequencies of T-bet<sup>+</sup> IFN-γ<sup>+</sup> cells gated on adaptive immune subsets, including B, CD4<sup>+</sup> T and CD8<sup>+</sup> T cells from different groups of individuals, stimulated with BCG + IL-23. (<b>H</b>) Dot-plots showing the expression of intracellular T-bet and IFN-γ gated on CD4<sup>+</sup> T cells with and without BCG stimulation in the presence and absence of IL-12 and IL-23. (<b>I</b>) Frequency of Vδ2<sup>+</sup> γδ T cells in PBMCs from different groups of individuals expanded with BCG-lysates for 14 d. (<b>J</b>) Dot-plots showing intracellular T-bet and IFN-γ levels in non-responsive and PPD-responsive CD4<sup>+</sup> T cells following PPD stimulation. (<b>K</b>) and (<b>L</b>) Dot-plots showing intracellular T-bet and IFN-γ levels in non-responsive and BCG-lysates-responsive CD4<sup>+</sup> T (K) and CD8<sup>+</sup> T cells (L).</div><div id="fspara0465" role="paragraph">In (A, B, E, F, G and I), bars represent the mean and the standard error of the mean. In (D), numbers represent the mean and the standard deviation. Dots represent individual samples. Mann-Whitney nonparametric tests were used for analysis in (A, B, E, F and G), <sup>∗</sup>p &lt; 0.05, <sup>∗∗</sup>p &lt; 0.01, <sup>∗∗∗</sup>p &lt; 0.001, <sup>∗∗∗∗</sup>p &lt; 0.0001, ns = not significant.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-14"><h3>Defective Prolonged Anti-BCG Immunity Mediated by Vδ2<sup>+</sup> γδ T Cells</h3><div id="p0105" role="paragraph">The stimulation of PBMCs <i>in&nbsp;vitro</i> with live BCG involves both antigens specifically recognized by <i>Mycobacterium</i>-specific cognate αβ and γδ T&nbsp;cells and many other stimuli. BCG infection <i>in&nbsp;vitro</i> mimics acute infection <i>in&nbsp;vivo</i> but may not be robust enough for investigations of the antigen-specific adaptive immune response, particularly as concerns prolonged adaptive immunity. We therefore studied Vδ2<sup>+</sup> γδ T&nbsp;cells and CD4<sup>+</sup> αβ T&nbsp;cells, two adaptive immune lymphocyte subsets that produced substantial amounts of IFN-γ during BCG infection <i>in&nbsp;vitro</i> and are known to function in an antigen-specific manner. In PBMCs from healthy donors, Vδ2<sup>+</sup> γδ T&nbsp;cells responded most robustly during BCG infection (<a id="crosref1915" href="#fig6">Figure&nbsp;6</a>E). Vδ2<sup>+</sup> γδ T&nbsp;cells, a major subset of γδ T&nbsp;cells recognizing pAg from microbial sources (<span class="dropBlock reference-citations"><a id="body-ref-sref59-2" href="#" role="doc-biblioref" data-xml-rid="bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59-2" aria-controls="sref59-2">Gu et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref59-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>Gu, S. ∙ Borowska, M.T. ∙ Boughter, C.T. ...</div><div><strong>Butyrophilin3A proteins and Vγ9Vδ2 T&nbsp;cell activation</strong></div><div><em>Semin. Cell Dev. Biol.</em> 2018; <strong>84</strong>:65-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.semcdb.2018.02.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85043373095" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29471037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.semcdb.2018.02.007&amp;pmid=29471037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref64-2" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64-2" aria-controls="sref64-2">Harly et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref64-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Harly, C. ∙ Guillaume, Y. ∙ Nedellec, S. ...</div><div><strong>Key implication of CD277/butyrophilin-3 (BTN3A) in cellular stress sensing by a major human γδ T-cell subset</strong></div><div><em>Blood.</em> 2012; <strong>120</strong>:2269-2279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2012-05-430470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866308872" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22767497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2012-05-430470&amp;pmid=22767497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref163-2" href="#" role="doc-biblioref" data-xml-rid="bib163" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref163-2" aria-controls="sref163-2">Vavassori et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref163-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">166.</div><div>Vavassori, S. ∙ Kumar, A. ∙ Wan, G.S. ...</div><div><strong>Butyrophilin 3A1 binds phosphorylated antigens and stimulates human γδ T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:908-916</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2665" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84883147590" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23872678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2665&amp;pmid=23872678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), are also known to play an essential role in the recall response to mycobacterial re-infection in humans and non-human primates (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Chen, 2005</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Chen, Z.W.</div><div><strong>Immune regulation of gammadelta T&nbsp;cell responses in mycobacterial infections</strong></div><div><em>Clin. Immunol.</em> 2005; <strong>116</strong>:202-207</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.clim.2005.04.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-23444455796" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16087145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.clim.2005.04.005&amp;pmid=16087145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref143" href="#" role="doc-biblioref" data-xml-rid="bib143" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref143" aria-controls="sref143">Shen et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref143"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">146.</div><div>Shen, Y. ∙ Zhou, D. ∙ Qiu, L. ...</div><div><strong>Adaptive immune response of Vgamma2Vdelta2+ T&nbsp;cells during mycobacterial infections</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:2255-2258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1068819" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037155586" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11910108/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1068819&amp;pmid=11910108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Vδ2<sup>+</sup> γδ T&nbsp;cells proliferate vigorously in response to mycobacterial infection <i>in&nbsp;vivo</i> and can expand robustly in response to pAg-rich lysates of mycobacterial species <i>in&nbsp;vitro</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Hoft et&nbsp;al., 1998</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>Hoft, D.F. ∙ Brown, R.M. ∙ Roodman, S.T.</div><div><strong>Bacille Calmette-Guérin vaccination enhances human gamma delta T&nbsp;cell responsiveness to mycobacteria suggestive of a memory-like phenotype</strong></div><div><em>J.&nbsp;Immunol.</em> 1998; <strong>161</strong>:1045-1054</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9670986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=9670986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref119" href="#" role="doc-biblioref" data-xml-rid="bib119" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref119" aria-controls="sref119">Modlin et&nbsp;al., 1989</a><div class="dropBlock__holder" data-db-target-of="sref119"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">121.</div><div>Modlin, R.L. ∙ Pirmez, C. ∙ Hofman, F.M. ...</div><div><strong>Lymphocytes bearing antigen-specific γ δ T-cell receptors accumulate in human infectious disease lesions</strong></div><div><em>Nature.</em> 1989; <strong>339</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/339544a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2471938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F339544a0&amp;pmid=2471938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref125" href="#" role="doc-biblioref" data-xml-rid="bib125" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref125" aria-controls="sref125">Panchamoorthy et&nbsp;al., 1991</a><div class="dropBlock__holder" data-db-target-of="sref125"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">128.</div><div>Panchamoorthy, G. ∙ McLean, J. ∙ Modlin, R.L. ...</div><div><strong>A predominance of the T&nbsp;cell receptor V gamma 2/V delta 2 subset in human mycobacteria-responsive T&nbsp;cells suggests germline gene encoded recognition</strong></div><div><em>J.&nbsp;Immunol.</em> 1991; <strong>147</strong>:3360-3369</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1658147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=1658147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref126" href="#" role="doc-biblioref" data-xml-rid="bib126" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref126" aria-controls="sref126">Parker et&nbsp;al., 1990</a><div class="dropBlock__holder" data-db-target-of="sref126"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">129.</div><div>Parker, C.M. ∙ Groh, V. ∙ Band, H. ...</div><div><strong>Evidence for extrathymic changes in the T&nbsp;cell receptor gamma/delta repertoire</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1990; <strong>171</strong>:1597-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.171.5.1597" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2185330/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.171.5.1597&amp;pmid=2185330" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref160" href="#" role="doc-biblioref" data-xml-rid="bib160" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref160" aria-controls="sref160">Tsukaguchi et&nbsp;al., 1995</a><div class="dropBlock__holder" data-db-target-of="sref160"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">163.</div><div>Tsukaguchi, K. ∙ Balaji, K.N. ∙ Boom, W.H.</div><div><strong>CD4+ alpha beta T&nbsp;cell and gamma delta T&nbsp;cell responses to Mycobacterium tuberculosis. Similarities and differences in Ag recognition, cytotoxic effector function, and cytokine production</strong></div><div><em>J.&nbsp;Immunol.</em> 1995; <strong>154</strong>:1786-1796</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7836763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=7836763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We investigated whether the function of Vδ2<sup>+</sup> γδ T&nbsp;cells was affected in the patient with T-bet deficiency, as these cells represented a small, but important proportion (∼0.2%) of P’s peripheral lymphocytes. The populations of Vδ2<sup>+</sup> T&nbsp;cells from all controls and relatives of P expanded vigorously following prolonged stimulation with BCG lysates. By contrast, no such expansion was observed for T-bet-deficient Vδ2<sup>+</sup> T&nbsp;cells (<a id="crosref1970" href="#fig6">Figures 6</a>I and 6J; <a id="crosref1975" href="#figs6">Figure&nbsp;S6</a>I). After 2&nbsp;weeks of expansion, the levels of IFN-γ production by T-bet-deficient cells were lower than those for healthy donors’ cells (<a id="crosref1980" href="#fig6">Figure&nbsp;6</a>K).</div></section><section id="sec-2-15"><h3>Redundant Role of T-bet in IFN-γ Production by BCG-Specific Cognate T<sub>H</sub>1<sup>∗</sup> Cells</h3><div id="p0110" role="paragraph">IFN-γ production by CD4<sup>+</sup> αβ T&nbsp;cells upon BCG infection <i>ex&nbsp;vivo</i> was subtly affected by T-bet deficiency, but it remains unclear whether there is a difference in the mycobacterial antigen-specific CD4<sup>+</sup> αβ T&nbsp;cell response. It also remains unclear whether the memory adaptive immunity to mycobacteria elicited by antigen-specific CD4<sup>+</sup> T&nbsp;cells is dependent on T-bet. We addressed both these unresolved issues by analyzing <i>Mycobacterium</i>-responsive CD4<sup>+</sup> T&nbsp;cells expanded through prolonged stimulation with BCG-lysates or tuberculin purified protein derivative (PPD) (<a id="crosref1985" href="#mmc1">Data S1</a>). Like those of BCG-vaccinated healthy donors, P’s CD4<sup>+</sup> T&nbsp;cells proliferated equally well in response to both BCG-lysates and PPD (<a id="crosref1990" href="#mmc1">Data S1</a>). No difference in IFN-γ production from PPD-responsive CD4<sup>+</sup> T&nbsp;cells was observed between healthy donors and P (<a id="crosref1995" href="#fig6">Figure&nbsp;6</a>L; <a id="crosref2000" href="#figs6">Figure&nbsp;S6</a>J). Unlike PPD, which activated only CD4<sup>+</sup> T&nbsp;cells, BCG-lysates activated both CD4<sup>+</sup> T&nbsp;cells and CD8<sup>+</sup> T&nbsp;cells, albeit to a lesser extent (<a id="crosref2005" href="#mmc1">Data S1</a>). Like PPD stimulation, IFN-γ production from both BCG-lysate-responsive CD4<sup>+</sup> and CD8<sup>+</sup> T&nbsp;cells was similar between controls and P (<a id="crosref2010" href="#fig6">Figures 6</a>M and 6N; <a id="crosref2015" href="#figs6">Figures S6</a>K and S6L). We confirmed these findings by screening antigen-reactive T&nbsp;cell libraries established from the CD4<sup>+</sup>CCR6<sup>–</sup> (containing classic T<sub>H</sub>1 cells) and CD4<sup>+</sup>CCR6<sup>+</sup> (containing T<sub>H</sub>1<sup>∗</sup> <i>Mycobacterium</i>-responsive cells) memory subsets (<span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Geiger et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Geiger, R. ∙ Duhen, T. ∙ Lanzavecchia, A. ...</div><div><strong>Human naive and memory CD4+ T&nbsp;cell repertoires specific for naturally processed antigens analyzed using libraries of amplified T&nbsp;cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2009; <strong>206</strong>:1525-1534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20090504" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-67650469829" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19564353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20090504&amp;pmid=19564353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with our <i>in&nbsp;vivo</i> findings, the T&nbsp;cells in the CD4<sup>+</sup>CCR6<sup>–</sup> and CD4<sup>+</sup>CCR6<sup>+</sup> libraries had low levels of CXCR3 or IFN-γ (<a id="crosref2025" href="#mmc1">Data S1</a>). P’s CD4<sup>+</sup>CCR6<sup>+</sup> T&nbsp;cell library responded robustly to BCG, tetanus toxoid and <i>C.&nbsp;albicans</i>, and his CD4<sup>+</sup>CCR6<sup>–</sup> T&nbsp;cell library responded normally to influenza virus, CMV, and EBV. Despite intact proliferation, IFN-γ production from T-bet-deficient T&nbsp;cells responding to influenza virus, EBV, tetanus toxoid, and <i>C.&nbsp;albicans</i> was weak. However, P’s CCR6<sup>+</sup> T&nbsp;cells, consisting almost entirely of <i>Mycobacterium</i>-specific memory T<sub>H</sub>1<sup>∗</sup> cells (<span class="dropBlock reference-citations"><a id="body-ref-sref2-2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2-2" aria-controls="sref2-2">Acosta-Rodriguez et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref2-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Acosta-Rodriguez, E.V. ∙ Rivino, L. ∙ Geginat, J. ...</div><div><strong>Surface phenotype and antigenic specificity of human interleukin 17-producing T helper memory cells</strong></div><div><em>Nat. Immunol.</em> 2007; <strong>8</strong>:639-646</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34249079176" target="_blank" aria-label="Open Scopus in new tab">Scopus (1293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17486092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1467&amp;pmid=17486092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref9" href="#" role="doc-biblioref" data-xml-rid="bib9" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref9" aria-controls="sref9">Becattini et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Becattini, S. ∙ Latorre, D. ∙ Mele, F. ...</div><div><strong>T cell immunity. Functional heterogeneity of human memory CD4(+) T&nbsp;cell clones primed by pathogens or vaccines</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:400-406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260668" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84921753054" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25477212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260668&amp;pmid=25477212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), proliferated robustly in response to BCG peptides. Moreover, their IFN-γ production was normal, and their levels of IL-10 production were slightly higher (<a id="crosref2040" href="#fig6">Figure&nbsp;6</a>O; <a id="crosref2045" href="#mmc1">Data S1</a>). The normal levels of IFN-γ production could not be attributed to cells with a revertant genotype, as reported in other T&nbsp;cell primary immunodeficiency diseases (PIDs) (<span class="dropBlock reference-citations"><a id="body-ref-sref35" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35" aria-controls="sref35">Davis et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref35"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Davis, B.R. ∙ Dicola, M.J. ∙ Prokopishyn, N.L. ...</div><div><strong>Unprecedented diversity of genotypic revertants in lymphocytes of a patient with Wiskott-Aldrich syndrome</strong></div><div><em>Blood.</em> 2008; <strong>111</strong>:5064-5067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2007-06-095299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-46749091756" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18332229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2007-06-095299&amp;pmid=18332229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref133" href="#" role="doc-biblioref" data-xml-rid="bib133" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref133" aria-controls="sref133">Revy et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref133"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">136.</div><div>Revy, P. ∙ Kannengiesser, C. ∙ Fischer, A.</div><div><strong>Somatic genetic rescue in Mendelian haematopoietic diseases</strong></div><div><em>Nat. Rev. Genet.</em> 2019; <strong>20</strong>:582-598</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-019-0139-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85067013592" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31186537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-019-0139-x&amp;pmid=31186537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), because the IFN-γ<sup>+</sup> BCG-specific T&nbsp;cell clones still carried the <i>TBX21</i> indel variant. Thus, the prolonged immunity to BCG infection mediated by Vδ2<sup>+</sup> γδ T and memory αβ T&nbsp;cells was divergently controlled by T-bet, as T-bet was required for the generation of long-term immunity due to Vδ2<sup>+</sup> γδ T&nbsp;cells, but redundant for IFN-γ production by BCG-specific cognate CD4<sup>+</sup> T<sub>H</sub>1<sup>∗</sup> cells or <i>Mycobacterium</i>-responsive CD8<sup>+</sup> αβ T&nbsp;cells (<a id="crosref2835" href="#mmc1">Table S4</a>).</div></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0115" role="paragraph">We report here the identification and study of the first patient discovered to have inherited complete T-bet deficiency resulting clinically in MSMD. Key observations made in T-bet-deficient mice were validated in this human patient with T-bet deficiency: (1) the development of T<sub>H</sub>1 cells and their production of effector cytokines, including IFN-γ in particular, requires T-bet (<span class="dropBlock reference-citations"><a id="body-ref-sref149-4" href="#" role="doc-biblioref" data-xml-rid="bib149" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref149-4" aria-controls="sref149-4">Szabo et&nbsp;al., 2000</a><div class="dropBlock__holder" data-db-target-of="sref149-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">152.</div><div>Szabo, S.J. ∙ Kim, S.T. ∙ Costa, G.L. ...</div><div><strong>A novel transcription factor, T-bet, directs Th1 lineage commitment</strong></div><div><em>Cell.</em> 2000; <strong>100</strong>:655-669</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10761931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980702-3&amp;pmid=10761931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref150-4" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" aria-label="Reference 2002" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-4" aria-controls="sref150-4">2002</a><div class="dropBlock__holder" data-db-target-of="sref150-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Szabo, S.J. ∙ Sullivan, B.M. ∙ Stemmann, C. ...</div><div><strong>Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:338-342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059462" target="_blank" aria-label="Open Scopus in new tab">Scopus (920)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065543&amp;pmid=11786644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); (2) the development of NK and iNKT cells is dependent on T-bet (<span class="dropBlock reference-citations"><a id="body-ref-sref159" href="#" role="doc-biblioref" data-xml-rid="bib159" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref159" aria-controls="sref159">Townsend et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref159"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">162.</div><div>Townsend, M.J. ∙ Weinmann, A.S. ∙ Matsuda, J.L. ...</div><div><strong>T-bet regulates the terminal maturation and homeostasis of NK and Valpha14i NKT cells</strong></div><div><em>Immunity.</em> 2004; <strong>20</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS1074-7613%2804%2900076-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS1074-7613%2804%2900076-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-1842785209" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15084276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1074-7613%2804%2900076-7&amp;pmid=15084276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); and (3) the regulation of T-bet-dependent targets, including <i>CXCR3</i>, <i>TNF</i>, and <i>IFNG</i>, involves both direct transactivation and epigenetic modulation (<span class="dropBlock reference-citations"><a id="body-ref-sref118-2" href="#" role="doc-biblioref" data-xml-rid="bib118" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118-2" aria-controls="sref118-2">Miller and Weinmann, 2010</a><div class="dropBlock__holder" data-db-target-of="sref118-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">120.</div><div>Miller, S.A. ∙ Weinmann, A.S.</div><div><strong>Molecular mechanisms by which T-bet regulates T-helper cell commitment</strong></div><div><em>Immunol. Rev.</em> 2010; <strong>238</strong>:233-246</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1600-065X.2010.00952.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77958610119" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20969596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1600-065X.2010.00952.x&amp;pmid=20969596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Accordingly, T-bet-deficient mice are highly vulnerable to mycobacteria, including <i>M.&nbsp;tuberculosis</i> and <i>M.&nbsp;avium</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Matsuyama et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">117.</div><div>Matsuyama, M. ∙ Ishii, Y. ∙ Yageta, Y. ...</div><div><strong>Role of Th1/Th17 balance regulated by T-bet in a mouse model of Mycobacterium avium complex disease</strong></div><div><em>J.&nbsp;Immunol.</em> 2014; <strong>192</strong>:1707-1717</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1302258" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84894237503" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24446514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1302258&amp;pmid=24446514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref147" href="#" role="doc-biblioref" data-xml-rid="bib147" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref147" aria-controls="sref147">Sullivan et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref147"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">150.</div><div>Sullivan, B.M. ∙ Jobe, O. ∙ Lazarevic, V. ...</div><div><strong>Increased susceptibility of mice lacking T-bet to infection with Mycobacterium tuberculosis correlates with increased IL-10 and decreased IFN-gamma production</strong></div><div><em>J.&nbsp;Immunol.</em> 2005; <strong>175</strong>:4593-4602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.175.7.4593" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16177104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.175.7.4593&amp;pmid=16177104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), like mice deficient for other genes that govern IFN-γ immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref79" href="#" role="doc-biblioref" data-xml-rid="bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref79" aria-controls="sref79">Jouanguy et&nbsp;al., 1999</a><div class="dropBlock__holder" data-db-target-of="sref79"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Jouanguy, E. ∙ Döffinger, R. ∙ Dupuis, S. ...</div><div><strong>IL-12 and IFN-γ in host defense against mycobacteria and salmonella in mice and men</strong></div><div><em>Curr. Opin. Immunol.</em> 1999; <strong>11</strong>:346-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0952-7915(99)80055-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0033057576" target="_blank" aria-label="Open Scopus in new tab">Scopus (275)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10375558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0952-7915%2899%2980055-7&amp;pmid=10375558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By contrast, despite the requirement of T-bet for immunity to a broad spectrum of pathogens following experimental inoculation in mice (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Bakshi et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Bakshi, C.S. ∙ Malik, M. ∙ Carrico, P.M. ...</div><div><strong>T-bet deficiency facilitates airway colonization by Mycoplasma pulmonis in a murine model of asthma</strong></div><div><em>J.&nbsp;Immunol.</em> 2006; <strong>177</strong>:1786-1795</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.177.3.1786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33746215188" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16849489/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.177.3.1786&amp;pmid=16849489" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref65" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65" aria-controls="sref65">Harms Pritchard et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref65"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Harms Pritchard, G. ∙ Hall, A.O. ∙ Christian, D.A. ...</div><div><strong>Diverse roles for T-bet in the effector responses required for resistance to infection</strong></div><div><em>J.&nbsp;Immunol.</em> 2015; <strong>194</strong>:1131-1140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1401617" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84921456080" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25556247/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1401617&amp;pmid=25556247" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref69" href="#" role="doc-biblioref" data-xml-rid="bib69" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref69" aria-controls="sref69">Hultgren et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref69"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Hultgren, O.H. ∙ Verdrengh, M. ∙ Tarkowski, A.</div><div><strong>T-box transcription-factor-deficient mice display increased joint pathology and failure of infection control during staphylococcal arthritis</strong></div><div><em>Microbes Infect.</em> 2004; <strong>6</strong>:529-535</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.micinf.2004.02.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-2542424828" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15158185/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.micinf.2004.02.005&amp;pmid=15158185" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref82" href="#" role="doc-biblioref" data-xml-rid="bib82" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref82" aria-controls="sref82">Kao et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref82"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Kao, C. ∙ Oestreich, K.J. ∙ Paley, M.A. ...</div><div><strong>Transcription factor T-bet represses expression of the inhibitory receptor PD-1 and sustains virus-specific CD8+ T&nbsp;cell responses during chronic infection</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:663-671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79959375167" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21623380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2046&amp;pmid=21623380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref94" href="#" role="doc-biblioref" data-xml-rid="bib94" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref94" aria-controls="sref94">Lebrun et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref94"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Lebrun, A. ∙ Portocarrero, C. ∙ Kean, R.B. ...</div><div><strong>T-bet Is Required for the Rapid Clearance of Attenuated Rabies Virus from Central Nervous System Tissue</strong></div><div><em>J.&nbsp;Immunol.</em> 2015; <strong>195</strong>:4358-4368</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1501274" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84945143808" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26408670/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1501274&amp;pmid=26408670" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref123" href="#" role="doc-biblioref" data-xml-rid="bib123" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref123" aria-controls="sref123">Oakley et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref123"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">125.</div><div>Oakley, M.S. ∙ Sahu, B.R. ∙ Lotspeich-Cole, L. ...</div><div><strong>The transcription factor T-bet regulates parasitemia and promotes pathogenesis during Plasmodium berghei ANKA murine malaria</strong></div><div><em>J.&nbsp;Immunol.</em> 2013; <strong>191</strong>:4699-4708</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1300396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84886017430" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24078698/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1300396&amp;pmid=24078698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref132" href="#" role="doc-biblioref" data-xml-rid="bib132" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref132" aria-controls="sref132">Ravindran et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref132"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">135.</div><div>Ravindran, R. ∙ Foley, J. ∙ Stoklasek, T. ...</div><div><strong>Expression of T-bet by CD4 T&nbsp;cells is essential for resistance to Salmonella infection</strong></div><div><em>J.&nbsp;Immunol.</em> 2005; <strong>175</strong>:4603-4610</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.175.7.4603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-25444439804" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16177105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.175.7.4603&amp;pmid=16177105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref137" href="#" role="doc-biblioref" data-xml-rid="bib137" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref137" aria-controls="sref137">Rosas et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref137"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">140.</div><div>Rosas, L.E. ∙ Snider, H.M. ∙ Barbi, J. ...</div><div><strong>Cutting edge: STAT1 and T-bet play distinct roles in determining outcome of visceral leishmaniasis caused by Leishmania donovani</strong></div><div><em>J.&nbsp;Immunol.</em> 2006; <strong>177</strong>:22-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.177.1.22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33745305544" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16785492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.177.1.22&amp;pmid=16785492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref148" href="#" role="doc-biblioref" data-xml-rid="bib148" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref148" aria-controls="sref148">Svensson et&nbsp;al., 2005</a><div class="dropBlock__holder" data-db-target-of="sref148"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">151.</div><div>Svensson, A. ∙ Nordström, I. ∙ Sun, J.B. ...</div><div><strong>Protective immunity to genital herpes simplex [correction of simpex] virus type 2 infection is mediated by T-bet</strong></div><div><em>J.&nbsp;Immunol.</em> 2005; <strong>174</strong>:6266-6273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.174.10.6266" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-18744373400" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15879125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.174.10.6266&amp;pmid=15879125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref150-5" href="#" role="doc-biblioref" data-xml-rid="bib150" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref150-5" aria-controls="sref150-5">Szabo et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref150-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">153.</div><div>Szabo, S.J. ∙ Sullivan, B.M. ∙ Stemmann, C. ...</div><div><strong>Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:338-342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059462" target="_blank" aria-label="Open Scopus in new tab">Scopus (920)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065543&amp;pmid=11786644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), the only apparent infectious phenotype of the T-bet-deficient patient is MSMD. Our study provides compelling evidence that inherited T-bet deficiency is a genetic etiology of MSMD due to the disruption of IFN-γ immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref25-2" href="#" role="doc-biblioref" data-xml-rid="bib25" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref25-2" aria-controls="sref25-2">Casanova et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref25-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Casanova, J.-L. ∙ Conley, M.E. ∙ Seligman, S.J. ...</div><div><strong>Guidelines for genetic studies in single patients: lessons from primary immunodeficiencies</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2014; <strong>211</strong>:2137-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20140520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84908247576" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25311508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20140520&amp;pmid=25311508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This experiment of nature suggests that T-bet is required for protective immunity to intramacrophagic mycobacteria but largely redundant for immunity to most intracellular pathogens, including viruses in particular. This is at odds with findings in mice but consistent with other genetic etiologies of MSMD, all of which are inborn errors of IFN-γ immunity (<span class="dropBlock reference-citations"><a id="body-ref-sref17-4" href="#" role="doc-biblioref" data-xml-rid="bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-4" aria-controls="sref17-4">Bustamante, 2020</a><div class="dropBlock__holder" data-db-target-of="sref17-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Bustamante, J.</div><div><strong>Mendelian susceptibility to mycobacterial disease: recent discoveries</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:993-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02120-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85079165775" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02120-y&amp;pmid=32025907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref84-3" href="#" role="doc-biblioref" data-xml-rid="bib84" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-3" aria-controls="sref84-3">Kerner et&nbsp;al., 2020</a><div class="dropBlock__holder" data-db-target-of="sref84-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Kerner, G. ∙ Rosain, J. ∙ Guérin, A. ...</div><div><strong>Inherited human IFN-γ deficiency underlies mycobacterial disease</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2020; <strong>130</strong>:3158-3171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI135460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085294382" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32163377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI135460&amp;pmid=32163377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Our observation further suggests that the functions of T-bet unrelated to IFN-γ are redundant in humans. The identification of additional T-bet-deficient patients is required to draw firm conclusions. However, it is striking that humans genetically deprived of key immunological molecules other than T-bet or IFN-γ often show a much greater redundancy than the corresponding mutant mice (<span class="dropBlock reference-citations"><a id="body-ref-sref22" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22" aria-controls="sref22">Casanova and Abel, 2004</a><div class="dropBlock__holder" data-db-target-of="sref22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Casanova, J.L. ∙ Abel, L.</div><div><strong>The human model: a genetic dissection of immunity to infection in natural conditions</strong></div><div><em>Nat. Rev. Immunol.</em> 2004; <strong>4</strong>:55-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri1264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14704768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri1264&amp;pmid=14704768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref23" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" aria-label="Reference 2018" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23" aria-controls="sref23">2018</a><div class="dropBlock__holder" data-db-target-of="sref23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Casanova, J.L. ∙ Abel, L.</div><div><strong>Human genetics of infectious diseases: Unique insights into immunological redundancy</strong></div><div><em>Semin. Immunol.</em> 2018; <strong>36</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.smim.2017.12.008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85039455920" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29254755/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.smim.2017.12.008&amp;pmid=29254755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0120" role="paragraph">Our study also suggests unexpected immunological abnormalities not documented in T-bet-deficient mice that also contribute to the development of MSMD: (1) T-bet was apparently required for the optimal development of two innate-like adaptive lineages of immune cells, MAIT and Vδ2<sup>+</sup> γδ T&nbsp;cells; (2) T-bet was also apparently required for the optimal production of IFN-γ by the few NK, iNKT, MAIT, and Vδ2<sup>+</sup> γδ T lymphocytes that were able to develop (<span class="dropBlock reference-citations"><a id="body-ref-sref11-2" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11-2" aria-controls="sref11-2">Bettelli et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref11-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Bettelli, E. ∙ Sullivan, B. ∙ Szabo, S.J. ...</div><div><strong>Loss of T-bet, but not STAT1, prevents the development of experimental autoimmune encephalomyelitis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2004; <strong>200</strong>:79-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20031819" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-3142724732" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15238607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20031819&amp;pmid=15238607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Daussy et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Daussy, C. ∙ Faure, F. ∙ Mayol, K. ...</div><div><strong>T-bet and Eomes instruct the development of two distinct natural killer cell lineages in the liver and in the bone marrow</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2014; <strong>211</strong>:563-577</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20131560" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84896847858" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24516120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20131560&amp;pmid=24516120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref51" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51" aria-controls="sref51">Garrido-Mesa et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref51"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Garrido-Mesa, N. ∙ Schroeder, J.-H. ∙ Stolarczyk, E. ...</div><div><strong>T-bet controls intestinal mucosa immune responses via repression of type 2 innate lymphoid cell function</strong></div><div><em>Mucosal Immunol.</em> 2019; <strong>12</strong>:51-63</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41385-018-0092-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85055560183" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30356098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41385-018-0092-6&amp;pmid=30356098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref70" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70" aria-controls="sref70">Intlekofer et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref70"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Intlekofer, A.M. ∙ Takemoto, N. ∙ Kao, C. ...</div><div><strong>Requirement for T-bet in the aberrant differentiation of unhelped memory CD8+ T&nbsp;cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:2015-2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20070841" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34548402098" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17698591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20070841&amp;pmid=17698591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref107" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref107" aria-controls="sref107">Lugo-Villarino et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref107"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Lugo-Villarino, G. ∙ Maldonado-Lopez, R. ∙ Possemato, R. ...</div><div><strong>T-bet is required for optimal production of IFN-gamma and antigen-specific T&nbsp;cell activation by dendritic cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:7749-7754</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1332767100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037934497" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12802010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1332767100&amp;pmid=12802010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref127" href="#" role="doc-biblioref" data-xml-rid="bib127" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref127" aria-controls="sref127">Pearce et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref127"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">130.</div><div>Pearce, E.L. ∙ Mullen, A.C. ∙ Martins, G.A. ...</div><div><strong>Control of Effector CD8+ T Cell Function by the Transcription Factor Eomesodermin</strong></div><div><em>Science.</em> 2003; <strong>302</strong>:1041-1043</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1090148" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0242410712" target="_blank" aria-label="Open Scopus in new tab">Scopus (652)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14605368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1090148&amp;pmid=14605368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref131" href="#" role="doc-biblioref" data-xml-rid="bib131" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref131" aria-controls="sref131">Rankin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref131"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">134.</div><div>Rankin, L.C. ∙ Groom, J.R. ∙ Chopin, M. ...</div><div><strong>The transcription factor T-bet is essential for the development of NKp46+ innate lymphocytes via the Notch pathway</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:389-395</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2545" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84875445419" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23455676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2545&amp;pmid=23455676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref146-2" href="#" role="doc-biblioref" data-xml-rid="bib146" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref146-2" aria-controls="sref146-2">Sullivan et&nbsp;al., 2003</a><div class="dropBlock__holder" data-db-target-of="sref146-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">149.</div><div>Sullivan, B.M. ∙ Juedes, A. ∙ Szabo, S.J. ...</div><div><strong>Antigen-driven effector CD8 T&nbsp;cell function regulated by T-bet</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:15818-15823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2636938100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0347994894" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14673093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2636938100&amp;pmid=14673093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref153" href="#" role="doc-biblioref" data-xml-rid="bib153" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref153" aria-controls="sref153">Tang et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref153"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">156.</div><div>Tang, Y. ∙ Desierto, M.J. ∙ Chen, J. ...</div><div><strong>The role of the Th1 transcription factor T-bet in a mouse model of immune-mediated bone-marrow failure</strong></div><div><em>Blood.</em> 2010; <strong>115</strong>:541-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2009-03-211383" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77449092131" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19903901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2009-03-211383&amp;pmid=19903901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Unexpectedly, IFN-γ production by cognate purely adaptive <i>Mycobacterium</i>-specific T<sub>H</sub>1<sup>∗</sup> CD4<sup>+</sup> T&nbsp;cells, and by PPD-, or BCG-lysate-responsive CD4<sup>+</sup> or CD8<sup>+</sup> αβ T&nbsp;cells, was unaffected by T-bet deficiency. The compensatory mechanisms underlying intact IFN-γ production by these antigen-specific CD4<sup>+</sup> T&nbsp;cells remain to be discovered. Taken together, impaired IFN-γ production by NK and iNKT cells, as in mice, and by MAIT and Vδ2<sup>+</sup> γδ T&nbsp;cells, as shown here, accounts for MSMD in this patient with T-bet deficiency, despite normal T<sub>H</sub>1<sup>∗</sup> development and function. By contrast, inborn errors of immunity that disrupt IFN-γ production by the&nbsp;selective depletion of NK, iNKT, CD4<sup>+</sup>, or CD8<sup>+</sup> αβ T cells do not underlie mycobacterial disease, because of the compensation provided by other subsets. Conversely, the loss of all T&nbsp;cell subsets in SCID patients does result in predisposition to mycobacterial disease. Interestingly, a different combination of deficits accounts for MSMD in patients with RORγT deficiency, who lack iNKT and MAIT cells and whose γδ T and T<sub>H</sub>1<sup>∗</sup> cells do not produce IFN-γ, and whose NK cells are unable to compensate (<span class="dropBlock reference-citations"><a id="body-ref-sref124-2" href="#" role="doc-biblioref" data-xml-rid="bib124" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref124-2" aria-controls="sref124-2">Okada et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref124-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">127.</div><div>Okada, S. ∙ Markle, J.G. ∙ Deenick, E.K. ...</div><div><strong>Impairment of immunity to Candida and Mycobacterium in humans with bi-allelic RORC mutations</strong></div><div><em>Science.</em> 2015; <strong>349</strong>:606-613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa4282" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84939154723" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26160376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa4282&amp;pmid=26160376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). T-bet and RORγT deficiencies are characterized by iNKT, MAIT, and γδ T&nbsp;cell deficiencies, whereas an NK deficit is observed only in T-bet deficiency and a deficit of T<sub>H</sub>1<sup>∗</sup> cells is observed only in RORγT deficiency. We found that human T-bet was essential for both innate (NK cells) and innate-like (iNKT, MAIT, and Vδ2<sup>+</sup> γδ T&nbsp;cells) adaptive immunity to mycobacteria, but, surprisingly, redundant for classic, purely adaptive immunity (CD4<sup>+</sup> T<sub>H</sub>1<sup>∗</sup> and CD8<sup>+</sup> αβ T&nbsp;cells) to mycobacteria.</div></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0130" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Anti-mouse/human T-bet (Clone 4B10)</td><td>Bio X Cell</td><td>Cat# BE0100, RRID:<a id="intref0010" href="https://antibodyregistry.org/AB_10950173" target="_blank">AB_10950173</a></td></tr><tr><td>Anti-FLAG (M2)-HRP conjugated</td><td>Sigma-Aldrich</td><td>Cat# A8592, RRID:<a id="intref0015" href="https://antibodyregistry.org/AB_439702" target="_blank">AB_439702</a></td></tr><tr><td>Anti-alpha Tubulin-HRP conjugated</td><td>Proteintech</td><td>Cat# HRP-66031, RRID:<a id="intref0020" href="https://antibodyregistry.org/AB_2687491" target="_blank">AB_2687491</a></td></tr><tr><td>Anti-Lamin A/C (rabbit polyclonal H-110)</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-20681, RRID:<a id="intref0025" href="https://antibodyregistry.org/AB_648154" target="_blank">AB_648154</a></td></tr><tr><td>InVivoMab mouse IgG1 isotype control antibody</td><td>Bio X Cell</td><td>Cat# BE0083, RRID:<a id="intref0030" href="https://antibodyregistry.org/AB_1107784" target="_blank">AB_1107784</a></td></tr><tr><td>Anti-Stat1 (N terminus) purified mouse antibody (Clone 1/Stat1)</td><td>BD Biosciences</td><td>Cat# 610116, RRID:<a id="intref0035" href="https://antibodyregistry.org/AB_397522" target="_blank">AB_397522</a></td></tr><tr><td>Mouse anti-GAPDH (clone 0411)-HRP conjugated</td><td>Santa Cruz Biotechnology</td><td>Cat# sc-47724, RRID:<a id="intref0040" href="https://antibodyregistry.org/AB_627678" target="_blank">AB_627678</a></td></tr><tr><td>Rabbit anti-T-bet polyclonal antibody</td><td>Proteintech</td><td>Cat# 13700-1-AP, RRID:<a id="intref0045" href="https://antibodyregistry.org/AB_10666436" target="_blank">AB_10666436</a></td></tr><tr><td>Anti-GFP tag Mouse Monoclonal Antibody</td><td>Proteintech</td><td>Cat# HRP-66002,</td></tr><tr><td>Anti-V5-HRP antibody</td><td>Thermo Fisher Scientific</td><td>Cat# R961-25, RRID:<a id="intref0050" href="https://antibodyregistry.org/AB_2556565" target="_blank">AB_2556565</a></td></tr><tr><td>Anti-ERN1 antibody</td><td>Proteintech</td><td>Cat# 27528-1-AP</td></tr><tr><td>BV750 Mouse Anti-Human CD4</td><td>BD Biosciences</td><td>Cat# 566356, RRID:<a id="intref0055" href="https://antibodyregistry.org/AB_2744426" target="_blank">AB_2744426</a></td></tr><tr><td>BD Horizon BV650 Mouse Anti-Human CD3</td><td>BD Biosciences</td><td>Cat# 563999, RRID:<a id="intref0060" href="https://antibodyregistry.org/AB_2738535" target="_blank">AB_2738535</a></td></tr><tr><td>Pacific Blue anti-human CD8a [RPA-T8]</td><td>BioLegend</td><td>Cat# 301033, RRID:<a id="intref0065" href="https://antibodyregistry.org/AB_1595443" target="_blank">AB_1595443</a></td></tr><tr><td>FITC anti-human CD271 (NGFR) [ME20.4]</td><td>BioLegend</td><td>Cat# 345104, RRID:<a id="intref0070" href="https://antibodyregistry.org/AB_2282828" target="_blank">AB_2282828</a></td></tr><tr><td>PE/Dazzle594 anti-human CD271 (NGFR)</td><td>BioLegend</td><td>Cat# 345120, RRID:<a id="intref0075" href="https://antibodyregistry.org/AB_2750274" target="_blank">AB_2750274</a></td></tr><tr><td>BUV563 Mouse Anti-Human CD4 Clone SK3 (also known as Leu3a)</td><td>BD Biosciences</td><td>Cat# 612912</td></tr><tr><td>BUV737 Mouse Anti-Human CD8 Clone SK1</td><td>BD Biosciences</td><td>Cat# 612754</td></tr><tr><td>Anti-T-bet Mouse Monoclonal Antibody (PE/Cy7) [clone: 4B10]</td><td>BioLegend</td><td>Cat# 644823, RRID:<a id="intref0080" href="https://antibodyregistry.org/AB_2561760" target="_blank">AB_2561760</a></td></tr><tr><td>Alexa Fluor 647 Mouse anti-Human CD271</td><td>BD PharMingen</td><td>Cat# 560326, RRID:<a id="intref0085" href="https://antibodyregistry.org/AB_1645403" target="_blank">AB_1645403</a></td></tr><tr><td>Anti-CD45 89Y-conjugated (Clone HI30)</td><td>Fluidigm</td><td>Cat# 3089003B, RRID:<a id="intref0090" href="https://antibodyregistry.org/AB_2661851" target="_blank">AB_2661851</a></td></tr><tr><td>Anti-CD57 113In-conjudated (Clone HCD57)</td><td>BioLegend</td><td>Cat# 322302, RRID:<a id="intref0095" href="https://antibodyregistry.org/AB_2661815" target="_blank">AB_2661815</a></td></tr><tr><td>Anti-CD11c 115In-conjudated (Clone Bu15)</td><td>BioLegend</td><td>Cat# 337202, RRID:<a id="intref0100" href="https://antibodyregistry.org/AB_1236381" target="_blank">AB_1236381</a></td></tr><tr><td>Anti-CD33 141Pr-conjugated (Clone WM53)</td><td>BioLegend</td><td>Cat# 303410, RRID:<a id="intref0105" href="https://antibodyregistry.org/AB_2074243" target="_blank">AB_2074243</a></td></tr><tr><td>Anti-CD19 142Nd-conjugated (Clone HIB19)</td><td>BioLegend</td><td>Cat# 302202, RRID:<a id="intref0110" href="https://antibodyregistry.org/AB_2661817" target="_blank">AB_2661817</a></td></tr><tr><td>Anti-CD45RA 143Nd-conjudated (Clone HI100)</td><td>BioLegend</td><td>Cat# 304102, RRID:<a id="intref0115" href="https://antibodyregistry.org/AB_314406" target="_blank">AB_314406</a></td></tr><tr><td>Anti-CD141 144Nd-conjugated (Clone M80)</td><td>BioLegend</td><td>Cat# 344102, RRID:<a id="intref0120" href="https://antibodyregistry.org/AB_2661788" target="_blank">AB_2661788</a></td></tr><tr><td>Anti-CD4 145Nd-conjugated (Clone RPA-T4)</td><td>BioLegend</td><td>Cat# 300502, RRID:<a id="intref0125" href="https://antibodyregistry.org/AB_314070" target="_blank">AB_314070</a></td></tr><tr><td>Anti-CD8 146Nd-conjugated (Clone RPA-T8)</td><td>BioLegend</td><td>Cat# 301002, RRID:<a id="intref0130" href="https://antibodyregistry.org/AB_2661818" target="_blank">AB_2661818</a></td></tr><tr><td>Anti-CD20 147Sm-conjugated (Clone 2H7)</td><td>BioLegend</td><td>Cat# 302302, RRID:<a id="intref0135" href="https://antibodyregistry.org/AB_314250" target="_blank">AB_314250</a></td></tr><tr><td>Anti-CD16 148Nd-conjugated (Clone 3G8)</td><td>BioLegend</td><td>Cat# 302014, RRID:<a id="intref0140" href="https://antibodyregistry.org/AB_314214" target="_blank">AB_314214</a></td></tr><tr><td>Anti-CD127 149Sm-conjugated (Clone A019D5)</td><td>Fluidigm</td><td>Cat# 3149011B, RRID:<a id="intref0145" href="https://antibodyregistry.org/AB_2661792" target="_blank">AB_2661792</a></td></tr><tr><td>Anti-CD1c 150Nd-conjugated (Clone L161)</td><td>BioLegend</td><td>Cat# 331502, RRID:<a id="intref0150" href="https://antibodyregistry.org/AB_2661820" target="_blank">AB_2661820</a></td></tr><tr><td>Anti-CD123 151Eu-conjugated (Clone 6H6)</td><td>BioLegend</td><td>Cat# 306002, RRID:<a id="intref0155" href="https://antibodyregistry.org/AB_2661822" target="_blank">AB_2661822</a></td></tr><tr><td>Anti-CD66b 152Sm-conjugated (Clone G10F5)</td><td>BioLegend</td><td>Cat# 305102, RRID:<a id="intref0160" href="https://antibodyregistry.org/AB_2661823" target="_blank">AB_2661823</a></td></tr><tr><td>Anti-PD-1 153Eu-conjugated (Clone EH12.2H7)</td><td>BioLegend</td><td>Cat# 329926, RRID:<a id="intref0165" href="https://antibodyregistry.org/AB_11147365" target="_blank">AB_11147365</a></td></tr><tr><td>Anti-CD86 154Sm-conjugated (Clone IT2.2)</td><td>BioLegend</td><td>Cat# 305410, RRID:<a id="intref0170" href="https://antibodyregistry.org/AB_314530" target="_blank">AB_314530</a></td></tr><tr><td>Anti-CD27 155Gd-conjugated (Clone O323)</td><td>BioLegend</td><td>Cat# 302802, RRID:<a id="intref0175" href="https://antibodyregistry.org/AB_2661825" target="_blank">AB_2661825</a></td></tr><tr><td>Anti-CCR5 156Gd-conjugated (Clone J418F1)</td><td>BioLegend</td><td>Cat# 359102, RRID:<a id="intref0180" href="https://antibodyregistry.org/AB_2562457" target="_blank">AB_2562457</a></td></tr><tr><td>Anti-CD117 158Gd-conjugated (Clone 104D2)</td><td>BioLegend</td><td>Cat# 313202, RRID:<a id="intref0185" href="https://antibodyregistry.org/AB_313202" target="_blank">AB_313202</a></td></tr><tr><td>Anti-CD24 159Tb-conjugated (Clone ML5)</td><td>BioLegend</td><td>Cat# 311102, RRID:<a id="intref0190" href="https://antibodyregistry.org/AB_314851" target="_blank">AB_314851</a></td></tr><tr><td>Anti-CD14 160Gd-conjugated (Clone M5E2)</td><td>BioLegend</td><td>Cat# 301810, RRID:<a id="intref0195" href="https://antibodyregistry.org/AB_314192" target="_blank">AB_314192</a></td></tr><tr><td>Anti-CD56 161Dy-conjugated (Clone B159)</td><td>BD Biosciences</td><td>Cat# 555513, RRID:<a id="intref0200" href="https://antibodyregistry.org/AB_395903" target="_blank">AB_395903</a></td></tr><tr><td>Anti-gdTCR 162Dy-conjugated (Clone REA591)</td><td>Miltenyi</td><td>Cat# 048-785</td></tr><tr><td>Anti-CRTH2 163Dy-conjugated (Clone REA598)</td><td>Milentyi</td><td>Cat# 130-122-305, RRID:<a id="intref0205" href="https://antibodyregistry.org/AB_2801886" target="_blank">AB_2801886</a></td></tr><tr><td>Anti-CLEC12A 164Dy-conjugated (Clone 50C1)</td><td>BioLegend</td><td>Cat# 353602, RRID:<a id="intref0210" href="https://antibodyregistry.org/AB_10962440" target="_blank">AB_10962440</a></td></tr><tr><td>Anti-CCR6 165Ho-conjugated (Clone G034E3)</td><td>BioLegend</td><td>Cat# 353402, RRID:<a id="intref0215" href="https://antibodyregistry.org/AB_10918625" target="_blank">AB_10918625</a></td></tr><tr><td>Anti-CD25 166Er-conjugated (Clone M-A251)</td><td>BioLegend</td><td>Cat# 356102, RRID:<a id="intref0220" href="https://antibodyregistry.org/AB_2661833" target="_blank">AB_2661833</a></td></tr><tr><td>Anti-CCR7 167Er-conjugated (Clone G043H7)</td><td>BioLegend</td><td>Cat# 353256, RRID:<a id="intref0225" href="https://antibodyregistry.org/AB_2814291" target="_blank">AB_2814291</a></td></tr><tr><td>Anti-CD3 168Er-conjugated (Clone UCHT1)</td><td>BioLegend</td><td>Cat# 300402, RRID:<a id="intref0230" href="https://antibodyregistry.org/AB_2661835" target="_blank">AB_2661835</a></td></tr><tr><td>Anti-CX3CR1 169Tm-conjugated (Clone 2A9-1)</td><td>BioLegend</td><td>Cat# 341602, RRID:<a id="intref0235" href="https://antibodyregistry.org/AB_1595422" target="_blank">AB_1595422</a></td></tr><tr><td>Anti-CD38 170Er-conjugated (Clone HB-7)</td><td>BioLegend</td><td>Cat# 356602, RRID:<a id="intref0240" href="https://antibodyregistry.org/AB_2661836" target="_blank">AB_2661836</a></td></tr><tr><td>Anti-CD161 171Yb-conjugated (Clone HP-3G10)</td><td>BioLegend</td><td>Cat# 339902, RRID:<a id="intref0245" href="https://antibodyregistry.org/AB_2661837" target="_blank">AB_2661837</a></td></tr><tr><td>Anti-CD209 172Yb-conjugated (Clone 9E9A8)</td><td>BioLegend</td><td>Cat# 330102, RRID:<a id="intref0250" href="https://antibodyregistry.org/AB_1134253" target="_blank">AB_1134253</a></td></tr><tr><td>Anti-CXCR3 173Yb-conjugated (Clone REA232)</td><td>Miltenyi</td><td>Cat# 130-108-022, RRID:<a id="intref0255" href="https://antibodyregistry.org/AB_2655743" target="_blank">AB_2655743</a></td></tr><tr><td>Anti-HLADR 174Yb-conjugated (Clone L243)</td><td>BioLegend</td><td>Cat# 307602, RRID:<a id="intref0260" href="https://antibodyregistry.org/AB_314680" target="_blank">AB_314680</a></td></tr><tr><td>Anti-CCR4 176Yb-conjugated (Clone 205410)</td><td>R&amp;D Systems</td><td>Cat# MAB1567, RRID:<a id="intref0265" href="https://antibodyregistry.org/AB_2074395" target="_blank">AB_2074395</a></td></tr><tr><td>Anti-CD11b 208Bi-conjugated (Clone ICRF44)</td><td>Fluidigm</td><td>Cat# 3209003B, RRID:<a id="intref0270" href="https://antibodyregistry.org/AB_2687654" target="_blank">AB_2687654</a></td></tr><tr><td>Live/dead 103Rh-conjugated</td><td>Fluidigm</td><td>Cat# 201103A</td></tr><tr><td>DNA1 191Ir-conjugated</td><td>Fluidigm</td><td>Cat# 201192A</td></tr><tr><td>DNA2 193Ir-conjugated</td><td>Fluidigm</td><td>Cat# 201192A</td></tr><tr><td>APC/Cy7 anti-human CD4 Antibody (Clone RPA-T4)</td><td>BioLegend</td><td>Cat# 300518, RRID:<a id="intref0275" href="https://antibodyregistry.org/AB_314086" target="_blank">AB_314086</a></td></tr><tr><td>Alexa Fluor 488 anti-human CD45RA (Clone HI100)</td><td>BioLegend</td><td>Cat# 304114, RRID:<a id="intref0280" href="https://antibodyregistry.org/AB_528816" target="_blank">AB_528816</a></td></tr><tr><td>Alexa Fluor 647 anti-human CD197 (CCR7) (Clone G043H7)</td><td>BioLegend</td><td>Cat# 353218, RRID:<a id="intref0285" href="https://antibodyregistry.org/AB_10917385" target="_blank">AB_10917385</a></td></tr><tr><td>LIVE/DEAD Fixable Aqua Dead Cell Stain Kit</td><td>Thermo Fisher Scientific</td><td>Cat# L34957</td></tr><tr><td>Zombie-NIR live-dead exclusion dye</td><td>BioLegend</td><td>Cat# 423105</td></tr><tr><td>FITC anti-human CD271 (NGFR) (Clone ME20.4)</td><td>BioLegend</td><td>Cat# 345104, RRID:<a id="intref0290" href="https://antibodyregistry.org/AB_2282828" target="_blank">AB_2282828</a></td></tr><tr><td>Brilliant Violet 510 anti-human TNF-α (Clone MAb11)</td><td>BioLegend</td><td>Cat# 502950, RRID:<a id="intref0295" href="https://antibodyregistry.org/AB_2565860" target="_blank">AB_2565860</a></td></tr><tr><td>PerCP/Cy5.5 anti-human IFN-γ (Clone 4S.B3)</td><td>BioLegend</td><td>Cat# 502526, RRID:<a id="intref0300" href="https://antibodyregistry.org/AB_961355" target="_blank">AB_961355</a></td></tr><tr><td>PE anti-human IL-4 (Clone 8D4-8)</td><td>BioLegend</td><td>Cat# 500704, RRID:<a id="intref0305" href="https://antibodyregistry.org/AB_315116" target="_blank">AB_315116</a></td></tr><tr><td>APC anti-human IL-13 (Clone JES10-5A2)</td><td>BioLegend</td><td>Cat# 501908, RRID:<a id="intref0310" href="https://antibodyregistry.org/AB_315203" target="_blank">AB_315203</a></td></tr><tr><td>PE anti-T-bet (Clone 4B10)</td><td>BioLegend</td><td>Cat# 644810, RRID:<a id="intref0315" href="https://antibodyregistry.org/AB_2200542" target="_blank">AB_2200542</a></td></tr><tr><td>Brilliant Violet 711 anti-human IFN-γ (Clone 4S.B3)</td><td>BioLegend</td><td>Cat# 502540, RRID:<a id="intref0320" href="https://antibodyregistry.org/AB_2563506" target="_blank">AB_2563506</a></td></tr><tr><td>PE/Dazzle594 anti-human IL-10 (Clone JES3-19F1)</td><td>BioLegend</td><td>Cat# 506812, RRID:<a id="intref0325" href="https://antibodyregistry.org/AB_2632783" target="_blank">AB_2632783</a></td></tr><tr><td>Brilliant Violet 785 anti-human IL-17A (Clone BL168)</td><td>BioLegend</td><td>Cat# 512338, RRID:<a id="intref0330" href="https://antibodyregistry.org/AB_2566765" target="_blank">AB_2566765</a></td></tr><tr><td>Brilliant Violet 421 anti-mouse/human IL-5 (Clone TRFK5)</td><td>BioLegend</td><td>Cat# 504311, RRID:<a id="intref0335" href="https://antibodyregistry.org/AB_2563161" target="_blank">AB_2563161</a></td></tr><tr><td>Alexa Fluor 532, Anti-CD3 Monoclonal Antibody (Clone UCHT1)</td><td>Thermo Fisher Scientific</td><td>Cat# 58-0038-42, RRID:<a id="intref0340" href="https://antibodyregistry.org/AB_11218675" target="_blank">AB_11218675</a></td></tr><tr><td>Anti-Human gamma delta TCR FITC</td><td>Thermo Fisher Scientific</td><td>Cat# 11-9959-42, RRID:<a id="intref0345" href="https://antibodyregistry.org/AB_10669049" target="_blank">AB_10669049</a></td></tr><tr><td>APC/Fire 750 anti-human TCR Vδ2 Antibody</td><td>BioLegend</td><td>Cat# 331420, RRID:<a id="intref0350" href="https://antibodyregistry.org/AB_2687326" target="_blank">AB_2687326</a></td></tr><tr><td>Brilliant Violet 605 anti-human CD56 (NCAM) (Clone 5.1H11)</td><td>BioLegend</td><td>Cat# 362538, RRID:<a id="intref0355" href="https://antibodyregistry.org/AB_2565856" target="_blank">AB_2565856</a></td></tr><tr><td>Brilliant Violet 510 anti-human CD8a (Clone RPA-T8)</td><td>BioLegend</td><td>Cat# 301047, RRID:<a id="intref0360" href="https://antibodyregistry.org/AB_2561378" target="_blank">AB_2561378</a></td></tr><tr><td>Brilliant Violet 711 anti-human TCR Vα7.2 (Clone 3C10)</td><td>BioLegend</td><td>Cat# 351731, RRID:<a id="intref0365" href="https://antibodyregistry.org/AB_2629679" target="_blank">AB_2629679</a></td></tr><tr><td>PE/Cy7 anti-human TCR Vα24-Jα18 (iNKT cell) (Clone 6B11)</td><td>BioLegend</td><td>Cat# 342912, RRID:<a id="intref0370" href="https://antibodyregistry.org/AB_2562230" target="_blank">AB_2562230</a></td></tr><tr><td>Anti-TCR Vδ1-VioBlue, human</td><td>Miltenyi Biotec</td><td>Cat# 130-100-555, RRID:<a id="intref0375" href="https://antibodyregistry.org/AB_2653957" target="_blank">AB_2653957</a></td></tr><tr><td>PE anti-human CD161 (Clone HP-3G10)</td><td>BioLegend</td><td>Cat# 339938, RRID:<a id="intref0380" href="https://antibodyregistry.org/AB_2564141" target="_blank">AB_2564141</a></td></tr><tr><td>Anti-TCR Vβ11-APC, human (clone REA559)</td><td>Miltenyi Biotec</td><td>Cat# 130-108-733, RRID:<a id="intref0385" href="https://antibodyregistry.org/AB_2653741" target="_blank">AB_2653741</a></td></tr><tr><td>Anti-Human gamma delta TCR – Alexa 647</td><td>BioLegend</td><td>Cat# 331214, RRID:<a id="intref0390" href="https://antibodyregistry.org/AB_1089210" target="_blank">AB_1089210</a></td></tr><tr><td>Anti-human CD3 – V450</td><td>BD Biosciences</td><td>Cat# 560365, RRID:<a id="intref0395" href="https://antibodyregistry.org/AB_1645570" target="_blank">AB_1645570</a></td></tr><tr><td>Anti-TCR Vδ1-FITC, human</td><td>Miltenyi Biotec</td><td>Cat# 130-118-362, RRID:<a id="intref0400" href="https://antibodyregistry.org/AB_2751495" target="_blank">AB_2751495</a></td></tr><tr><td>Brilliant Violet 785 anti-human CD20 [2H7]</td><td>BioLegend</td><td>Cat# 302356, RRID:<a id="intref0405" href="https://antibodyregistry.org/AB_2566316" target="_blank">AB_2566316</a></td></tr><tr><td>MR1-5-OP-RU-tetramer tetramer</td><td>NIH tetramer core facility</td><td>N/A</td></tr><tr><td>Alexa Fluor 700 anti-human TCR Vα7.2 [3C10]</td><td>BioLegend</td><td>Cat# 351728, RRID:<a id="intref0410" href="https://antibodyregistry.org/AB_2566337" target="_blank">AB_2566337</a></td></tr><tr><td>BV480 Mouse Anti-Human Invariant NK T Cell</td><td>BD Biosciences</td><td>Cat# 746788, RRID:<a id="intref0415" href="https://antibodyregistry.org/AB_2744044" target="_blank">AB_2744044</a></td></tr><tr><td>PerCP/Cy5.5 anti-human IL-17A [BL168]</td><td>BioLegend</td><td>Cat# 512313, RRID:<a id="intref0420" href="https://antibodyregistry.org/AB_961397" target="_blank">AB_961397</a></td></tr><tr><td>Rorγt Mouse anti Human, PE, Clone: Q21 559</td><td>BD Biosciences</td><td>Cat# 563081, RRID:<a id="intref0425" href="https://antibodyregistry.org/AB_2686896" target="_blank">AB_2686896</a></td></tr><tr><td>FITC anti-mouse/human Ki-67 Antibody</td><td>BioLegend</td><td>Cat# 151211, RRID:<a id="intref0430" href="https://antibodyregistry.org/AB_2814054" target="_blank">AB_2814054</a></td></tr><tr><td>anti-CCR6-PE (clone 11A9)</td><td>BD Biosciences</td><td>Cat# 561019, RRID:<a id="intref0435" href="https://antibodyregistry.org/AB_10565962" target="_blank">AB_10565962</a></td></tr><tr><td>anti-CXCR3-APC (clone 1C6)</td><td>BD Biosciences</td><td>Cat# 550967, RRID:<a id="intref0440" href="https://antibodyregistry.org/AB_398481" target="_blank">AB_398481</a></td></tr><tr><td>anti-IFN-γ-FITC (clone B27)</td><td>BD Biosciences</td><td>Cat# 554700, RRID:<a id="intref0445" href="https://antibodyregistry.org/AB_395517" target="_blank">AB_395517</a></td></tr><tr><td>anti-IFN-γ-PE (clone B27)</td><td>BD Biosciences</td><td>Cat# 554701, RRID:<a id="intref0450" href="https://antibodyregistry.org/AB_395518" target="_blank">AB_395518</a></td></tr><tr><td>anti-IL-17A-eFluor660 (Clone eBio64DEC17)</td><td>Thermo Fisher Scientific</td><td>Cat# 50-7179-42, RRID:<a id="intref0455" href="https://antibodyregistry.org/AB_11149126" target="_blank">AB_11149126</a></td></tr><tr><td>anti-IL-4-PE (Clone MP4-25D2)</td><td>BD Biosciences</td><td>Cat# 554485, RRID:<a id="intref0460" href="https://antibodyregistry.org/AB_395424" target="_blank">AB_395424</a></td></tr><tr><td>anti-IL-22-PerCP-eFluor-710 (Clone 22URTI)</td><td>Thermo Fisher Scientific</td><td>Cat# 46-7229-42, RRID:<a id="intref0465" href="https://antibodyregistry.org/AB_10596639" target="_blank">AB_10596639</a></td></tr><tr><td>anti-CD1a-biotin (HI149)</td><td>BioLegend</td><td>Cat# 300112, RRID:<a id="intref0470" href="https://antibodyregistry.org/AB_389344" target="_blank">AB_389344</a></td></tr><tr><td>anti-CD3-biotin (OKT3)</td><td>Thermo Fisher Scientific</td><td>Cat# 13-0037-82, RRID:<a id="intref0475" href="https://antibodyregistry.org/AB_1234955" target="_blank">AB_1234955</a></td></tr><tr><td>anti-CD5-biotin (UCHT2)</td><td>BD Biosciences</td><td>Cat# 555351, RRID:<a id="intref0480" href="https://antibodyregistry.org/AB_395755" target="_blank">AB_395755</a></td></tr><tr><td>anti-CD14-biotin (61D3)</td><td>Thermo Fisher Scientific</td><td>Cat # 13-0149-82, RRID:<a id="intref0485" href="https://antibodyregistry.org/AB_466373" target="_blank">AB_466373</a></td></tr><tr><td>anti-CD19-biotin (HIB19)</td><td>Thermo Fisher Scientific</td><td>Cat# 13-0199-82, RRID:<a id="intref0490" href="https://antibodyregistry.org/AB_466388" target="_blank">AB_466388</a></td></tr><tr><td>anti-CD34-biotin (4H11)</td><td>BioLegend</td><td>Cat# 316404</td></tr><tr><td>anti-CD123-biotin (6H6)</td><td>BioLegend</td><td>Cat# 306004, RRID:<a id="intref0495" href="https://antibodyregistry.org/AB_314578" target="_blank">AB_314578</a></td></tr><tr><td>anti-CD203c-biotin (FR3-16A11)</td><td>Miltenyi Biotec</td><td>Cat# 130-092-345, RRID:<a id="intref0500" href="https://antibodyregistry.org/AB_615067" target="_blank">AB_615067</a></td></tr><tr><td>anti-CD303-biotin (AC144)</td><td>Miltenyi Biotec</td><td>Cat# 130-090-961</td></tr><tr><td>anti-TCRαβ-biotin (IP26)</td><td>BioLegend</td><td>Cat# 306704, RRID:<a id="intref0505" href="https://antibodyregistry.org/AB_314632" target="_blank">AB_314632</a></td></tr><tr><td>anti-TCRγδ-biotin (B1)</td><td>BD Biosciences</td><td>Cat# 555716, RRID:<a id="intref0510" href="https://antibodyregistry.org/AB_396060" target="_blank">AB_396060</a></td></tr><tr><td>anti-FcεRIα-biotin (AER-37 (CRA-1))</td><td>BioLegend</td><td>Cat# 334606, RRID:<a id="intref0515" href="https://antibodyregistry.org/AB_2571885" target="_blank">AB_2571885</a></td></tr><tr><td>streptavidin BUV661</td><td>BD Biosciences</td><td>Cat# 565081</td></tr><tr><td>anti-CD62L-FITC (DREG-56)</td><td>Thermo Fisher Scientific</td><td>Cat# 11-0629-42, RRID:<a id="intref0520" href="https://antibodyregistry.org/AB_10667774" target="_blank">AB_10667774</a></td></tr><tr><td>anti-CD94-PerCP-Vio700 (REA113)</td><td>Miltenyi Biotec</td><td>Cat# 130-098-947, RRID:<a id="intref0525" href="https://antibodyregistry.org/AB_2659633" target="_blank">AB_2659633</a></td></tr><tr><td>anti-TCRVα24Jα18-PE (6B11)</td><td>Thermo Fisher Scientific</td><td>Cat# 12-5806-42, RRID:<a id="intref0530" href="https://antibodyregistry.org/AB_1724172" target="_blank">AB_1724172</a></td></tr><tr><td>anti-CD26-PE-CF594 (M-A261)</td><td>BD Biosciences</td><td>Cat# 565158, RRID:<a id="intref0535" href="https://antibodyregistry.org/AB_2739085" target="_blank">AB_2739085</a></td></tr><tr><td>anti-CD127-PE-Cy7 (eBioRDR5)</td><td>Thermo Fisher Scientific</td><td>Cat# 25-1278-42, RRID:<a id="intref0540" href="https://antibodyregistry.org/AB_1659672" target="_blank">AB_1659672</a></td></tr><tr><td>anti-CRTH2-Alexa Fluor 647 (BM16)</td><td>BD Biosciences</td><td>Cat# 558042, RRID:<a id="intref0545" href="https://antibodyregistry.org/AB_2112699" target="_blank">AB_2112699</a></td></tr><tr><td>anti-CD161-Alexa Fluor 700 (HP-3G10)</td><td>BioLegend</td><td>Cat# 339942, RRID:<a id="intref0550" href="https://antibodyregistry.org/AB_2565870" target="_blank">AB_2565870</a></td></tr><tr><td>anti-EOMES-APC-eFluor 780 (WD1928)</td><td>Thermo Fisher Scientific</td><td>Cat# 47-4877-42, RRID:<a id="intref0555" href="https://antibodyregistry.org/AB_2573982" target="_blank">AB_2573982</a></td></tr><tr><td>anti-NKp46-BV421 (9E2/NKp46)</td><td>BD Biosciences</td><td>Cat# 564065, RRID:<a id="intref0560" href="https://antibodyregistry.org/AB_2738572" target="_blank">AB_2738572</a></td></tr><tr><td>anti-CD45RA-BV570 (HI100)</td><td>BioLegend</td><td>Cat# 304132, RRID:<a id="intref0565" href="https://antibodyregistry.org/AB_2563813" target="_blank">AB_2563813</a></td></tr><tr><td>anti-CD117-BV605 (104D2)</td><td>BioLegend</td><td>Cat# 313218, RRID:<a id="intref0570" href="https://antibodyregistry.org/AB_2562025" target="_blank">AB_2562025</a></td></tr><tr><td>anti-RORγt-BV650 (Q21-559)</td><td>BD Biosciences</td><td>Cat# 624156</td></tr><tr><td>anti-CD7-BV711(M-T701)</td><td>BD Biosciences</td><td>Cat# 564018, RRID:<a id="intref0575" href="https://antibodyregistry.org/AB_2738544" target="_blank">AB_2738544</a></td></tr><tr><td>anti-CD16-BUV496 (3G8)</td><td>BD Biosciences</td><td>Cat# 564653, RRID:<a id="intref0580" href="https://antibodyregistry.org/AB_2744294" target="_blank">AB_2744294</a></td></tr><tr><td>anti-CD25-BUV563 (2A3)</td><td>BD Biosciences</td><td>Cat# 565699, RRID:<a id="intref0585" href="https://antibodyregistry.org/AB_2744341" target="_blank">AB_2744341</a></td></tr><tr><td>anti-CD56-BUV737 (NCAM16.2)</td><td>BD Biosciences</td><td>Cat# 564447, RRID:<a id="intref0590" href="https://antibodyregistry.org/AB_2744432" target="_blank">AB_2744432</a></td></tr><tr><td>anti-CD45-BUV805 (HI30)</td><td>BD Biosciences</td><td>Cat# 612891</td></tr><tr><td>anti-T-bet-BV786 (O4-46)</td><td>BD Biosciences</td><td>Cat# 564141, RRID:<a id="intref0595" href="https://antibodyregistry.org/AB_2738615" target="_blank">AB_2738615</a></td></tr><tr><td>anti-GATA3-BUV395 (L50-823)</td><td>BD Biosciences</td><td>Cat# 565448, RRID:<a id="intref0600" href="https://antibodyregistry.org/AB_2739241" target="_blank">AB_2739241</a></td></tr><tr><td>Fixable Viability Dye eFluor 506</td><td>Thermo Fisher Scientific</td><td>Cat# 65-0866-14</td></tr><tr><td>anti-CD107a-PE (eBioH4A3)</td><td>Thermo Fisher Scientific</td><td>Cat# 12-1079-42, RRID:<a id="intref0605" href="https://antibodyregistry.org/AB_10853326" target="_blank">AB_10853326</a></td></tr><tr><td>anti-CD1a-FITC (HI149)</td><td>BioLegend</td><td>Cat# 300104, RRID:<a id="intref0610" href="https://antibodyregistry.org/AB_314018" target="_blank">AB_314018</a></td></tr><tr><td>anti-CD3-FITC (OKT3)</td><td>Thermo Fisher Scientific</td><td>Cat# 11-0037-42, RRID:<a id="intref0615" href="https://antibodyregistry.org/AB_2016669" target="_blank">AB_2016669</a></td></tr><tr><td>anti-CD5-FITC (UCHT2)</td><td>Thermo Fisher Scientific</td><td>Cat# 11-0059-42, RRID:<a id="intref0620" href="https://antibodyregistry.org/AB_2572433" target="_blank">AB_2572433</a></td></tr><tr><td>anti-CD14-FITC (TUK4)</td><td>Miltenyi Biotec</td><td>Cat# 130-080-701, RRID:<a id="intref0625" href="https://antibodyregistry.org/AB_244303" target="_blank">AB_244303</a></td></tr><tr><td>anti-CD19-FITC (HIB19)</td><td>BioLegend</td><td>Cat# 302206, RRID:<a id="intref0630" href="https://antibodyregistry.org/AB_314236" target="_blank">AB_314236</a></td></tr><tr><td>anti-CD34-FITC (581)</td><td>BioLegend</td><td>Cat# 343504, RRID:<a id="intref0635" href="https://antibodyregistry.org/AB_1731852" target="_blank">AB_1731852</a></td></tr><tr><td>anti-CD123-FITC (6H6)</td><td>BioLegend</td><td>Cat# 306014, RRID:<a id="intref0640" href="https://antibodyregistry.org/AB_2124259" target="_blank">AB_2124259</a></td></tr><tr><td>anti-CD203c-FITC (NP4D6)</td><td>BioLegend</td><td>Cat# 324614, RRID:<a id="intref0645" href="https://antibodyregistry.org/AB_11218991" target="_blank">AB_11218991</a></td></tr><tr><td>anti-CD303-FITC (201A)</td><td>BioLegend</td><td>Cat# 354208, RRID:<a id="intref0650" href="https://antibodyregistry.org/AB_2561364" target="_blank">AB_2561364</a></td></tr><tr><td>anti-FcεRIɑ-FITC (AER-37 (CRA-1))</td><td>BioLegend</td><td>Cat# 334608, RRID:<a id="intref0655" href="https://antibodyregistry.org/AB_1227653" target="_blank">AB_1227653</a></td></tr><tr><td>anti-TCRɑβ-FITC (IP26)</td><td>Thermo Fisher Scientific</td><td>Cat# 11-9986-42, RRID:<a id="intref0660" href="https://antibodyregistry.org/AB_10853662" target="_blank">AB_10853662</a></td></tr><tr><td>anti-TCRγδ-FITC (B1)</td><td>BioLegend</td><td>Cat# 331208, RRID:<a id="intref0665" href="https://antibodyregistry.org/AB_1575108" target="_blank">AB_1575108</a></td></tr><tr><td>anti-NKp46-APC (9E2/NKp46)</td><td>BD Biosciences</td><td>Cat# 558051, RRID:<a id="intref0670" href="https://antibodyregistry.org/AB_398653" target="_blank">AB_398653</a></td></tr><tr><td>anti-CD56-BV786 (NCAM16.2)</td><td>BD Biosciences</td><td>Cat# 564058, RRID:<a id="intref0675" href="https://antibodyregistry.org/AB_2738569" target="_blank">AB_2738569</a></td></tr><tr><td>anti-CD16-BUV737 (3G8)</td><td>BD Biosciences</td><td>Cat# 564434, RRID:<a id="intref0680" href="https://antibodyregistry.org/AB_2744295" target="_blank">AB_2744295</a></td></tr><tr><td>anti-granzyme B-PE-CF594 (GB11)</td><td>BD Biosciences</td><td>Cat# 562462, RRID:<a id="intref0685" href="https://antibodyregistry.org/AB_2737618" target="_blank">AB_2737618</a></td></tr><tr><td>anti-EOMES-APC-eFluor 780 (WD1928)</td><td>Thermo Fisher Scientific</td><td>Cat# 47-4877-42, RRID:<a id="intref0690" href="https://antibodyregistry.org/AB_2573982" target="_blank">AB_2573982</a></td></tr><tr><td>anti-perforin-BV421 (δG9)</td><td>BD Biosciences</td><td>Cat# 563393, RRID:<a id="intref0695" href="https://antibodyregistry.org/AB_2738178" target="_blank">AB_2738178</a></td></tr><tr><td>anti-TNF-α-BV605 (Mab11)</td><td>BioLegend</td><td>Cat# 502936, RRID:<a id="intref0700" href="https://antibodyregistry.org/AB_2563884" target="_blank">AB_2563884</a></td></tr><tr class="rowsep"><td>anti-IFN-γ-BUV395 (B27)</td><td>BD Biosciences</td><td>Cat# 624163</td></tr><tr class="rowsep"><td colspan="3"><b>Bacterial and Virus Strains</b></td></tr><tr class="rowsep"><td><i>Mycobacterium bovis</i> BCG</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref165-1" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref165-1" aria-controls="sref165-1">Vogt and Nathan, 2011</a><div class="dropBlock__holder" data-db-target-of="sref165-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">168.</div><div>Vogt, G. ∙ Nathan, C.</div><div><strong>In&nbsp;vitro differentiation of human macrophages with enhanced antimycobacterial activity</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2011; <strong>121</strong>:3889-3901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI57235" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-80053413551" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21911939/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI57235&amp;pmid=21911939" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Biological Samples</b></td></tr><tr class="rowsep"><td>Peripheral blood mononuclear cells from indicated individuals</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Chemicals, Peptides, and Recombinant Proteins</b></td></tr><tr><td>Recombinant human Interleukin-12</td><td>R&amp;D Systems</td><td>Cat# 219-IL-025</td></tr><tr><td>Recombinant human Interleukin-23</td><td>R&amp;D Systems</td><td>Cat# 1290-IL-010</td></tr><tr><td>Recombinant human Interleukin-15</td><td>Miltenyi Biotec</td><td>Cat# 130-093-955</td></tr><tr><td>Recombinant human Interleukin-18</td><td>MBL/CliniSciences</td><td>Cat# B001-5</td></tr><tr><td>Recombinant human Interleukin-12</td><td>Miltenyi Biotec</td><td>Cat# 130-096-704</td></tr><tr><td>Recombinant human Interleukin-2</td><td>Roche Diagnostics Corporation</td><td>Cat# 11147528001</td></tr><tr><td>Proleukin (aldesleukin)</td><td>PROMETHEUS</td><td>N/A</td></tr><tr><td>Tuberculin/PPD</td><td>STATEN SERUM INSTITUT</td><td>N/A</td></tr><tr><td>Phorbol 12-myristate 13-acetate</td><td>MedChem Express</td><td>Cat# HY-18739</td></tr><tr class="rowsep"><td>Ionomycin calcium salt</td><td>Sigma-Aldrich</td><td>Cat# I3909-1ML</td></tr><tr class="rowsep"><td colspan="3"><b>Critical Commercial Assays</b></td></tr><tr><td>Dual-Glo Luciferase Assay system</td><td>Promega</td><td>Cat# E2940</td></tr><tr><td>SureSelect Human All Exon V4+UTR</td><td>Agilent Technologies</td><td>N/A</td></tr><tr><td>SureSelect Human All Exon V6</td><td>Agilent Technologies</td><td>Cat# 5190-8863</td></tr><tr><td>Human IFN-gamma ELISA Ready-SET-Go! Kit</td><td>Thermo Fisher Scientific</td><td>Cat# 88-7316-88, RRID:<a id="intref0705" href="https://antibodyregistry.org/AB_2575075" target="_blank">AB_2575075</a></td></tr><tr><td>Taqman Universal PCR Master Mix for qPCR</td><td>Thermo Fisher Scientific</td><td>Cat# 4304437</td></tr><tr><td>Taqman Gene Expression Assays - <i>TBX21</i> (Probe 1: Hs00894393 and Probe 2: Hs00894391), <i>IFNG</i> (Hs99999041), <i>TNF</i> (Hs01113624), <i>IL5</i> (Hs01548712) and <i>GUSB</i>.</td><td>Thermo Fisher Scientific</td><td>Cat# 4351372, 4448892, 4331182, 4310888E</td></tr><tr><td>QIAprep Spin Miniprep Kit</td><td>QIAGEN</td><td>Cat# 27106</td></tr><tr><td>Genome-Wide Human SNP Array 6.0</td><td>Thermo Fisher Scientific</td><td>Cat# 901150</td></tr><tr><td>HISPEED PLASMID MIDI KIT</td><td>QIAGEN</td><td>Cat# 810500</td></tr><tr><td>HiSpeed Plasmid Maxi Kit</td><td>QIAGEN</td><td>Cat# 12663</td></tr><tr><td>SuperSignal West Femto Maximum Sensitivity Substrate</td><td>Life Technologies</td><td>Cat# 34096</td></tr><tr><td>MinElute PCR Purification Kit</td><td>QIAGEN</td><td>Cat# 28004</td></tr><tr><td>EPIC DNA methylation array</td><td>Illumina</td><td>Cat# WG-317-1001</td></tr><tr><td>LIVE/DEAD Viability kit</td><td>Thermo Fisher Scientific</td><td>Cat# L3224</td></tr><tr><td>Chromium Single Cell 3′ Reagent Kits (v2)</td><td>10X Genomics</td><td>Cat# 120237</td></tr><tr><td>Chromium Single Cell A Chip Kit</td><td>10X Genomics</td><td>Cat# 120236</td></tr><tr><td>Dynabeads Human T-Activator CD3/CD28 for T Cell Expansion and Activation</td><td>Thermo Fisher Scientific</td><td>Cat# 11132D</td></tr><tr><td>Illumina TruSeq stranded mRNA LT kit</td><td>Illumina</td><td>Cat# RS-122-2101</td></tr><tr><td>CD4+ T Cell Isolation Kit, human</td><td>Miltenyi Biotec</td><td>Cat# 130-096-533</td></tr><tr><td>CD271 Microbead Kit</td><td>Miltenyi Biotec</td><td>Cat# 130-099-023</td></tr><tr><td>LEGENDplex Human Th Panel (13-plex) with V-bottom Plate</td><td>BioLegend</td><td>Cat# 740722</td></tr><tr><td>Foxp3 / Transcription Factor Staining Buffer Set</td><td>Thermo Fisher Scientific</td><td>Cat# 00-5523-00</td></tr><tr class="rowsep"><td>T Cell Activation/Expansion Kit, human</td><td>Miltenyi Biotec</td><td>Cat# 130-091-441</td></tr><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>Whole-exome sequencing of the patient</td><td>This manuscript</td><td>NCBI PRJNA641463</td></tr><tr><td>RNA-seq of expanding CD4<sup>+</sup> T&nbsp;cells</td><td>This manuscript</td><td>NCBI PRJNA641463</td></tr><tr><td>Omni-ATAC-seq of expanding CD4<sup>+</sup> T&nbsp;cells</td><td>This manuscript</td><td>NCBI PRJNA641463</td></tr><tr class="rowsep"><td>Single-cell RNA-seq of PBMC of the patient or heterozygous control</td><td>This manuscript</td><td>EGA EGAS00001004504</td></tr><tr class="rowsep"><td colspan="3"><b>Experimental Models: Cell Lines</b></td></tr><tr><td>HEK293T cells</td><td>ATCC</td><td>Cat# CRL-11268, RRID:CVCL_1926</td></tr><tr><td>NK-92 cell line</td><td>ATCC</td><td>Cat# CRL-2407, RRID:CVCL_2142</td></tr><tr><td>Expanded CD4+ T&nbsp;cells from healthy controls or P’s family</td><td>This manuscript</td><td>N/A</td></tr><tr><td>HVS-T cell lines from healthy controls or P’s family</td><td>This manuscript</td><td>N/A</td></tr><tr><td>EBV-B cell lines from healthy controls or P’s family</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td>Phoenix-Ampho cells</td><td>ATCC</td><td>Cat# CRL-3213, RRID:CVCL_H716</td></tr><tr class="rowsep"><td colspan="3"><b>Oligonucleotides</b></td></tr><tr class="rowsep"><td>Table S5</td><td>Table S5</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Recombinant DNA</b></td></tr><tr><td>pCMV6-human <i>TBX21</i> (NM_013351) with C-terminal Myc-DDK Tag</td><td>ORIGENE</td><td>Cat# RC207902</td></tr><tr><td>Mutant pCMV6-<i>TBX21</i> with C-terminal Myc-DDK Tag (various genetic variants)</td><td>This manuscript</td><td>N/A</td></tr><tr><td>pCMV6-human <i>ERN1</i> with C-terminal Myc-DDK Tag</td><td>OriGene</td><td>Cat# RC215023</td></tr><tr><td>pLenti-C-mGFP-P2A-puro lentiviral plasmid</td><td>OriGene</td><td>Cat# PS100093</td></tr><tr><td>pTRIP-SFFV-mCherry-2A plasmid containing ORFs of <i>PSMD9</i> or <i>ARHGAP27</i></td><td>This manuscript</td><td>N/A</td></tr><tr><td rowspan="3">TBRE-TK-Luc-WT</td><td rowspan="3">(<span class="dropBlock reference-citations"><a id="body-ref-sref76-2" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-2" aria-controls="sref76-2">Janesick et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref76-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Janesick, A. ∙ Shiotsugu, J. ∙ Taketani, M. ...</div><div><strong>RIPPLY3 is a retinoic acid-inducible repressor required for setting the borders of the pre-placodal ectoderm</strong></div><div><em>Development.</em> 2012; <strong>139</strong>:1213-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.071456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857300258" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.071456&amp;pmid=22354841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene: TBRE-TK-Luc-WT</td></tr><tr><td>Cat# 36246</td></tr><tr><td>RRID:Addgene_36246</td></tr><tr><td rowspan="3">TBRE-TK-Luc-Mut</td><td rowspan="3">(<span class="dropBlock reference-citations"><a id="body-ref-sref76-3" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-3" aria-controls="sref76-3">Janesick et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref76-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Janesick, A. ∙ Shiotsugu, J. ∙ Taketani, M. ...</div><div><strong>RIPPLY3 is a retinoic acid-inducible repressor required for setting the borders of the pre-placodal ectoderm</strong></div><div><em>Development.</em> 2012; <strong>139</strong>:1213-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.071456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857300258" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.071456&amp;pmid=22354841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene: TBRE-TK-Luc-Mut</td></tr><tr><td>Cat# 36245</td></tr><tr><td>RRID:Addgene_36245</td></tr><tr><td>pGL4.10-luciferase entry vector</td><td>Promega</td><td>Cat# E6651</td></tr><tr><td>pGL4.10-human <i>IFNG</i> proximal promoter-luciferase</td><td>This manuscript</td><td>N/A</td></tr><tr><td rowspan="3">pLZRS-IRES-deltaNGFR (EV-ires-CD271)</td><td rowspan="3">(<span class="dropBlock reference-citations"><a id="body-ref-sref161-1" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref161-1" aria-controls="sref161-1">van de Vosse et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref161-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>van de Vosse, E. ∙ Verhard, E.M. ∙ de Paus, R.A. ...</div><div><strong>Antisense-mediated exon skipping to correct IL-12Rbeta1 deficiency in T&nbsp;cells</strong></div><div><em>Blood.</em> 2009; <strong>113</strong>:4548-4555</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-12-196220" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-66549092636" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19258592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-12-196220&amp;pmid=19258592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene: pLZRS-IRES-deltaNGFR</td></tr><tr><td>Cat# 72930</td></tr><tr><td>RRID:Addgene_72930</td></tr><tr><td>pLZRS-WT or mutant <i>TBX21</i>-IRES-deltaNGFR (WT, K314R, Mut TBX21)</td><td>This manuscript</td><td>N/A</td></tr><tr><td rowspan="3">pLenti-CRISPR-V2</td><td rowspan="3">(<span class="dropBlock reference-citations"><a id="body-ref-sref140-1" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-1" aria-controls="sref140-1">Sanjana et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref140-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>Sanjana, N.E. ∙ Shalem, O. ∙ Zhang, F.</div><div><strong>Improved vectors and genome-wide libraries for CRISPR screening</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:783-784</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84905262730" target="_blank" aria-label="Open Scopus in new tab">Scopus (1702)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25075903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3047&amp;pmid=25075903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>Addgene: pLenti-CRISPR-V2</td></tr><tr><td>Cat# 52961</td></tr><tr><td>RRID:Addgene_52961</td></tr><tr class="rowsep"><td>pLenti-CRISPR-V2 for gRNA9 and gRNA15 (<i>TBX21</i>)</td><td>This manuscript</td><td>N/A</td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>BWA</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref97-1" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-1" aria-controls="sref97-1">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref97-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (18663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0710" href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net/</a></td></tr><tr><td>Genome Analysis Toolkit</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref117-1" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-1" aria-controls="sref117-1">McKenna et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref117-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (10353)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0715" href="https://gatk.broadinstitute.org/hc/en-us" target="_blank">https://gatk.broadinstitute.org/hc/en-us</a></td></tr><tr><td>SAMtools</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref98-1" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-1" aria-controls="sref98-1">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref98-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (21003)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0720" href="http://samtools.sourceforge.net/" target="_blank">http://samtools.sourceforge.net/</a></td></tr><tr><td>Picard</td><td><a id="intref0725" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a></td><td><a id="intref0730" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a></td></tr><tr><td>GATK HaplotypeCaller</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref38-1" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-1" aria-controls="sref38-1">DePristo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref38-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5537)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-2" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-2" aria-controls="sref117-2">McKenna et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref117-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (10353)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0735" href="https://www.broadinstitute.org/gatk" target="_blank">https://www.broadinstitute.org/gatk</a></td></tr><tr><td>SnpEff</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref30-1" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-1" aria-controls="sref30-1">Cingolani et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref30-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cingolani, P. ∙ Platts, A. ∙ Wang, L. ...</div><div><strong>A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</strong></div><div><em>Fly (Austin).</em> 2012; <strong>6</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/fly.19695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84862506964" target="_blank" aria-label="Open Scopus in new tab">Scopus (3490)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Ffly.19695&amp;pmid=22728672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0740" href="http://snpeff.sourceforge.net/" target="_blank">snpeff.sourceforge.net/</a></td></tr><tr><td>Affymetrix Power Tools Software Package</td><td>Affymetrix</td><td><a id="intref0745" href="https://www.affymetrix.com/support/developer/powertools/changelog/index.html" target="_blank">https://www.affymetrix.com/support/developer/powertools/changelog/index.html</a></td></tr><tr><td>MERLIN 1.1.2 software</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Abecasis et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abecasis, G.R. ∙ Cherny, S.S. ∙ Cookson, W.O. ...</div><div><strong>Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</strong></div><div><em>Nat. Genet.</em> 2002; <strong>30</strong>:97-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0036338150" target="_blank" aria-label="Open Scopus in new tab">Scopus (2652)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11731797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng786&amp;pmid=11731797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0750" href="http://bioinformatics.well.ox.ac.uk/Merlin" target="_blank">http://bioinformatics.well.ox.ac.uk/Merlin</a></td></tr><tr><td>Galaxy</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Afgan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Afgan, E. ∙ Baker, D. ∙ van den Beek, M. ...</div><div><strong>The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W3-W10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27137889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw343&amp;pmid=27137889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0755" href="https://galaxyproject.org" target="_blank">https://galaxyproject.org</a></td></tr><tr><td>STAR aligner</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (9649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0760" href="https://github.com/alexdobin/STAR/releases" target="_blank">https://github.com/alexdobin/STAR/releases</a></td></tr><tr><td>RSeQC</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref166-1" href="#" role="doc-biblioref" data-xml-rid="bib166" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref166-1" aria-controls="sref166-1">Wang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref166-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">169.</div><div>Wang, L. ∙ Wang, S. ∙ Li, W.</div><div><strong>RSeQC: quality control of RNA-seq experiments</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2184-2185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84865074382" target="_blank" aria-label="Open Scopus in new tab">Scopus (721)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts356&amp;pmid=22743226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0765" href="https://sourceforge.net/projects/rseqc/" target="_blank">https://sourceforge.net/projects/rseqc/</a></td></tr><tr><td>featureCounts</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref99-1" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-1" aria-controls="sref99-1">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref99-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (4007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0770" href="http://subread.sourceforge.net/" target="_blank">http://subread.sourceforge.net/</a></td></tr><tr><td>Molecular Signature Database (MSigDB)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Liberzon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Liberzon, A. ∙ Birger, C. ∙ Thorvaldsdóttir, H. ...</div><div><strong>The Molecular Signatures Database (MSigDB) hallmark gene set collection</strong></div><div><em>Cell Syst.</em> 2015; <strong>1</strong>:417-425</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84955328286" target="_blank" aria-label="Open Scopus in new tab">Scopus (1439)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.12.004&amp;pmid=26771021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0775" href="https://www.gsea-msigdb.org/gsea/msigdb" target="_blank">https://www.gsea-msigdb.org/gsea/msigdb</a></td></tr><tr><td>edgeR</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref134-1" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-1" aria-controls="sref134-1">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref134-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (12880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0780" href="https://bioconductor.org/packages/release/bioc/html/edgeR.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/edgeR.html</a></td></tr><tr><td>ComplexHeatmap</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref58-1" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-1" aria-controls="sref58-1">Gu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref58-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M.</div><div><strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:2847-2849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw313" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84992187637" target="_blank" aria-label="Open Scopus in new tab">Scopus (843)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw313&amp;pmid=27207943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0785" href="http://www.bioconductor.org/packages/devel/bioc/html/ComplexHeatmap.html" target="_blank">http://www.bioconductor.org/packages/devel/bioc/html/ComplexHeatmap.html</a></td></tr><tr><td>MACS2 software in BAMPE mode</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref46-1" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-1" aria-controls="sref46-1">Feng et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref46-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Feng, J. ∙ Liu, T. ∙ Qin, B. ...</div><div><strong>Identifying ChIP-seq enrichment using MACS</strong></div><div><em>Nat. Protoc.</em> 2012; <strong>7</strong>:1728-1740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2012.101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866127830" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22936215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2012.101&amp;pmid=22936215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref173-1" href="#" role="doc-biblioref" data-xml-rid="bib173" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref173-1" aria-controls="sref173-1">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref173-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">176.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (5850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td>N/A</td></tr><tr><td>MEME-ChIP</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref111-1" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-1" aria-controls="sref111-1">Ma et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref111-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Ma, W. ∙ Noble, W.S. ∙ Bailey, T.L.</div><div><strong>Motif-based analysis of large nucleotide data sets using MEME-ChIP</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:1428-1450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84901830680" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24853928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.083&amp;pmid=24853928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0790" href="https://meme.nbcr.net/" target="_blank">https://meme.nbcr.net/</a></td></tr><tr><td>DESeq2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref106-1" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-1" aria-controls="sref106-1">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref106-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (14777)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0795" href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a></td></tr><tr><td>DeepTools2</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref130-1" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130-1" aria-controls="sref130-1">Ramírez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref130-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85017133693" target="_blank" aria-label="Open Scopus in new tab">Scopus (955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw257&amp;pmid=27079975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0800" href="https://deeptools.readthedocs.io/en/develop/" target="_blank">https://deeptools.readthedocs.io/en/develop/</a></td></tr><tr><td>profileplyr package</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref18-1" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-1" aria-controls="sref18-1">Carroll and Barrows, 2019</a><div class="dropBlock__holder" data-db-target-of="sref18-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Carroll, T. ∙ Barrows, D.</div><div><strong>profileplyr: Visualization and annotation of read signal over genomic ranges with profileplyr</strong></div><div>2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.CarrollD.Barrowsprofileplyr%3A+Visualization+and+annotation+of+read+signal+over+genomic+ranges+with+profileplyr2019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0805" href="https://www.bioconductor.org/packages/release/bioc/html/profileplyr.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/profileplyr.html</a></td></tr><tr><td>ChIPQC package</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref19-1" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-1" aria-controls="sref19-1">Carroll et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref19-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Carroll, T.S. ∙ Liang, Z. ∙ Salama, R. ...</div><div><strong>Impact of artifact removal on ChIP quality metrics in ChIP-seq and ChIP-exo data</strong></div><div><em>Front. Genet.</em> 2014; <strong>5</strong>:75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fgene.2014.00075" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84898030206" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24782889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffgene.2014.00075&amp;pmid=24782889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99-2" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-2" aria-controls="sref99-2">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref99-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (4007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-1" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-1" aria-controls="sref37-1">de Santiago and Carroll, 2018</a><div class="dropBlock__holder" data-db-target-of="sref37-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>de Santiago, I. ∙ Carroll, T.</div><div><strong>Analysis of ChIP-seq Data in R/Bioconductor</strong></div><div>2018 195-226</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.de+SantiagoT.CarrollAnalysis+of+ChIP-seq+Data+in+R%2FBioconductor2018195226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0810" href="http://bioconductor.org/packages/release/bioc/html/ChIPQC.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/ChIPQC.html</a></td></tr><tr><td rowspan="2">FlowSOM and ConsensusClusterPlus packages</td><td rowspan="2">(<span class="dropBlock reference-citations"><a id="body-ref-sref162-2" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref162-2" aria-controls="sref162-2">Van Gassen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref162-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Van Gassen, S. ∙ Callebaut, B. ∙ Van Helden, M.J. ...</div><div><strong>FlowSOM: Using self-organizing maps for visualization and interpretation of cytometry data</strong></div><div><em>Cytometry A.</em> 2015; <strong>87</strong>:636-645</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.22625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84932198501" target="_blank" aria-label="Open Scopus in new tab">Scopus (278)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25573116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.22625&amp;pmid=25573116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref168-1" href="#" role="doc-biblioref" data-xml-rid="bib168" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref168-1" aria-controls="sref168-1">Wilkerson and Hayes, 2010</a><div class="dropBlock__holder" data-db-target-of="sref168-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">171.</div><div>Wilkerson, M.D. ∙ Hayes, D.N.</div><div><strong>ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:1572-1573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77954182362" target="_blank" aria-label="Open Scopus in new tab">Scopus (544)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20427518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq170&amp;pmid=20427518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0815" href="https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html</a></td></tr><tr><td><a id="intref0820" href="https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html</a></td></tr><tr><td>extraTrees</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref54-1" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-1" aria-controls="sref54-1">Geurts et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref54-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Geurts, P. ∙ Ernst, D. ∙ Wehenkel, L.</div><div><strong>Extremely randomized trees</strong></div><div><em>Mach. Learn.</em> 2006; <strong>63</strong>:3-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10994-006-6226-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33646430006" target="_blank" aria-label="Open Scopus in new tab">Scopus (1993)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10994-006-6226-1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0825" href="https://cran.r-project.org/web/packages/extraTrees/index.html" target="_blank">https://cran.r-project.org/web/packages/extraTrees/index.html</a></td></tr><tr><td>Cell Ranger</td><td>10X Genomics</td><td><a id="intref0830" href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger" target="_blank">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger</a></td></tr><tr><td>Doublet Finder</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref116-1" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-1" aria-controls="sref116-1">McGinnis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref116-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>McGinnis, C.S. ∙ Murrow, L.M. ∙ Gartner, Z.J.</div><div><strong>DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors</strong></div><div><em>Cell Syst.</em> 2019; <strong>8</strong>:329-337</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85064396876" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.03.003&amp;pmid=30954475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0835" href="https://github.com/chris-mcginnis-ucsf/DoubletFinder" target="_blank">https://github.com/chris-mcginnis-ucsf/DoubletFinder</a></td></tr><tr><td>Seurat R package</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref145-1" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-1" aria-controls="sref145-1">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref145-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">148.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (766)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0840" href="https://cran.r-project.org/web/packages/Seurat/index.html" target="_blank">https://cran.r-project.org/web/packages/Seurat/index.html</a></td></tr><tr><td>Uniform Manifold Approximation and Projection (UMAP)</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref10-2" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-2" aria-controls="sref10-2">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref10-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (449)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0845" href="https://cran.r-project.org/web/packages/uwot/index.html" target="_blank">https://cran.r-project.org/web/packages/uwot/index.html</a></td></tr><tr><td>MAST</td><td>(<span class="dropBlock reference-citations"><a id="body-ref-sref47-1" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-1" aria-controls="sref47-1">Finak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref47-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Finak, G. ∙ McDavid, A. ∙ Yajima, M. ...</div><div><strong>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0844-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84951574149" target="_blank" aria-label="Open Scopus in new tab">Scopus (404)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0844-5&amp;pmid=26653891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>)</td><td><a id="intref0850" href="https://github.com/RGLab/MAST" target="_blank">https://github.com/RGLab/MAST</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2820%2931453-7" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Resource Availability</h3><section id="sec-4-2-1"><h4>Lead Contact</h4><div id="p0135" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Jean-Laurent Casanova (<a id="intref0855" href="mailto:casanova@mail.rockefeller.edu">casanova@mail.rockefeller.edu</a>).</div></section><section id="sec-4-2-2"><h4>Materials Availability</h4><div id="p0140" role="paragraph">All unique/stable reagents generated in this study are available from the Lead Contact with a completed Materials Transfer Agreement with Inserm or the Rockefeller University.</div></section><section id="sec-4-2-3"><h4>Data and Code Availability</h4><div id="p0145" role="paragraph">The whole-exome sequencing, RNA-seq and omni-ATAC-seq datasets generated during this study are available at the Sequence Read Archive (SRA). The accession number for the whole exome sequencing, RNA-seq, and omni-ATAC-seq reported in this paper is SRA: PRJNA641463. The scRNA-seq datasets generated during this study are available at the EGA European Genome-Phenome Archive. The accession number for the scRNA-seq reported in this paper is EGA: EGAS00001004504. This study did not generate any unique code. Any other piece of data will be available upon reasonable request.</div></section></section><section id="sec-4-3"><h3>Experimental Model and Subject Details</h3><section id="sec-4-3-1"><h4>Human subjects</h4><div id="p0150" role="paragraph">The patient and the relatives studied here were living in and followed up in Morocco. The study was approved by and performed in accordance with the requirements of the institutional ethics committee of Necker Hospital for Sick Children, Paris, France, and the Rockefeller University, New York, USA. Informed consent was obtained for the patient, his relatives, and healthy control volunteers enrolled in the study. This study was also approved by the Sydney Local Health District RPAH Zone Human Research Ethics Committee and Research Governance Office, Royal Prince Alfred Hospital, Camperdown, New South Wales, Australia (protocol X16-0210/LNR/16/RPAH/257). Written informed consent was obtained from participants or their guardians. Informed consent from participants in Switzerland was approved by the local ethical committee (CE3428 authorized by Comitato Etico Cantonale, <a id="intref0860" href="https://www4.ti.ch/index.php?id=29269" target="_blank">https://www4.ti.ch/index.php?id=29269</a>). Experiments using samples from human subjects were conducted in the United States, France, Australia and Switzerland, in accordance with local regulations and with the approval of the IRBs of corresponding institutions. Plasma samples from unrelated healthy subjects used as controls for antibody profiling by phage immunoprecipitation-sequencing (PhIP-Seq) were collected at Sidra Medicine in accordance with a study protocol approved by the Clinical Research Ethics Board of Sidra Medicine.</div><div id="p0155" role="paragraph">Case report of the patient (P) - P was born in 2015 to Moroccan first-cousin parents. He was vaccinated with BCG at the age of 3&nbsp;months. After vaccination, he developed fever and left axillary lymphadenopathy accompanied by a cutaneous eruption. He received amoxicillin for 10&nbsp;days, resulting in clinical improvement. At the age of 6&nbsp;months, he was hospitalized for persistent fever, weight loss, cutaneous erythema, ear drainage and persistent oral thrush. Axillary adenopathy was present, with fistulation associated with hepatosplenomegaly and multiple abdominal adenopathies. The patient was treated with four antimycobacterial drugs; he displayed a good clinical response to 18&nbsp;months of treatment and has been off antibiotics for 15&nbsp;months. He was also found to have mild cytomegalovirus (CMV) viremia at the time (CMV PCR 3.74 log/mL, threshold&nbsp;&lt; 2.4 log/mL), and was treated with ganciclovir. Serological PCR tests for CMV were negative at the age of 2 years, but weakly positive at the age of 3 years (IgM anti-CMV weakly positive, CMV PCR 3.62 log/mL, threshold&nbsp;&lt; 2.4 log/mL). However, the patient showed no characteristic features of CMV infection and did not present clinically confirmed CMV disease. At the age of 11&nbsp;months, he was hospitalized for four days due to airway hyperresponsiveness requiring treatment with inhaled steroids and albuterol. The patient had a high blood eosinophil count, documented since the age of since months (1,460/mm<sup>3</sup>). By the age of 17&nbsp;months, he had persistent upper respiratory tract inflammation, with manifestations of persistent rhinorrhea, difficult respiration, wheezing and coughing that improved on treatment with oral steroid and salbutamol. The patient has since been treated with inhaled steroids for persistent upper respiratory tract inflammation. The clinical manifestations affecting the upper respiratory tract persisted at the age of 3 years. No trigger was identified for these symptoms and signs, which were not triggered by exercise, activity, feeding or parasitic infections. A serum IgE allergen screen was negative for more than 250 allergens tested. The patient requires inhaled steroid therapy (fluticasone spray) every morning and evening. He also frequently visits the emergency room, and is hospitalized about once per month for exacerbations requiring oral steroid (beta-methasone) and inhaled salbutamol. The patient has been vaccinated for <i>Tetanus</i>, <i>Diphtheria</i>, and <i>Haemophilus</i> B, for all of which positive serum IgG result have been obtained. He also tested positive for anti-HSV-1 IgG, anti-EBV IgG, anti-CMV IgG, anti-mumps IgG and anti-parainfluenza IgG in clinical serology examinations. No anatomical abnormalities of the airways were noted. The patient has a brother who is healthy, with no history of severe infectious disease or airway hyperresponsiveness. The genetic cause of the patient’s airway hyperresponsiveness will be studied separately.</div></section><section id="sec-4-3-2"><h4>Cell lines</h4><div id="p0160" role="paragraph">HEK293T and Phoenix A retroviral packaging cells were cultured in IMDM (GIBCO) supplemented with 10% fetal bovine serum (FBS). B cells from the patient or controls were immortalized in-house with Epstein-Barr virus (EBV-B cells) and cultured in RPMI 1640 (GIBCO) supplemented with 10% FBS. NK-92 cells were cultured in RPMI 1640 (GIBCO) supplemented with 10% FBS, 1&nbsp;mM sodium pyruvate, 200&nbsp;U/mL recombinant proleukin IL-2 (PROMETHEUS). <i>Herpesvirus saimiri</i>-transformed T (HVS-T) cells were generated with either <i>H.&nbsp;saimiri</i> strain C488 for transformation or the TERT transformation system (<span class="dropBlock reference-citations"><a id="body-ref-sref167" href="#" role="doc-biblioref" data-xml-rid="bib167" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref167" aria-controls="sref167">Wang et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref167"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">170.</div><div>Wang, Y. ∙ Ma, C.S. ∙ Ling, Y. ...</div><div><strong>Dual T&nbsp;cell- and B cell-intrinsic deficiency in humans with biallelic RLTPR mutations</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2016; <strong>213</strong>:2413-2435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20160576" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84994667641" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27647349/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20160576&amp;pmid=27647349" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). HVS-T cells were cultured in Panserin/RPMI 1640 (ratio 1:1) supplemented with 20% FBS, L-glutamine, gentamycin and 20&nbsp;U/mL human rIL-2 (Roche). Isolated CD4<sup>+</sup> T&nbsp;cells were cultured in X-vivo 15/gentamycin/L-glutamine (Lonza) supplemented with human AB Serum (GemCell). Peripheral blood mononuclear cells (PBMCs) were cultured in RPMI 1640 supplemented with 10% FBS for stimulation experiments. All cell lines used tested negative for mycoplasma. HEK293T, Phoenix A and NK-92 cells were purchased from the ATCC.</div></section></section><section id="sec-4-4"><h3>Method Details</h3><section id="sec-4-4-1"><h4>Phage immunoprecipitation-sequencing (PhIP-Seq)</h4><div id="p0165" role="paragraph">Patient serum and control samples were analyzed in duplicate by phage immunoprecipitation-sequencing (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Hernandez et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Hernandez, N. ∙ Melki, I. ∙ Jing, H. ...</div><div><strong>Life-threatening influenza pneumonitis in a child with inherited IRF9 deficiency</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2018; <strong>215</strong>:2567-2585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20180628" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85054094895" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30143481/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20180628&amp;pmid=30143481" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref170-1" href="#" role="doc-biblioref" data-xml-rid="bib170" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref170-1" aria-controls="sref170-1">Xu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref170-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Xu, G.J. ∙ Kula, T. ∙ Xu, Q. ...</div><div><strong>Comprehensive serological profiling of human populations using a synthetic human virome</strong></div><div><em>Science.</em> 2015; <strong>348</strong></div><div>aaa0698–aaa0698</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0698" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84930635082" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We used 10% liquid IVIg from pooled human plasma (Privigen® CSL Behring AG), human IgG-depleted serum (Supplier No HPLASERGFA5ML, Molecular Innovations, Inc.), and plasma samples from two unrelated healthy adult subjects and one unrelated healthy three-year-old boy for comparison. PhIP-Seq was carried out as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref170-2" href="#" role="doc-biblioref" data-xml-rid="bib170" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref170-2" aria-controls="sref170-2">Xu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref170-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Xu, G.J. ∙ Kula, T. ∙ Xu, Q. ...</div><div><strong>Comprehensive serological profiling of human populations using a synthetic human virome</strong></div><div><em>Science.</em> 2015; <strong>348</strong></div><div>aaa0698–aaa0698</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0698" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84930635082" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but with the following modifications. We determined levels of total IgG in the serum or plasma samples with the Human IgG total ELISA Ready-SET-Go kit (Thermo Fisher Scientific) and incubated diluted samples containing approximately 4&nbsp;μg of total IgG at 4°C overnight with 2&nbsp;× 10<sup>10</sup> plaque-forming units (PFUs) of a modified version of the original VirScan phage library (<span class="dropBlock reference-citations"><a id="body-ref-sref170-3" href="#" role="doc-biblioref" data-xml-rid="bib170" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref170-3" aria-controls="sref170-3">Xu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref170-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Xu, G.J. ∙ Kula, T. ∙ Xu, Q. ...</div><div><strong>Comprehensive serological profiling of human populations using a synthetic human virome</strong></div><div><em>Science.</em> 2015; <strong>348</strong></div><div>aaa0698–aaa0698</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0698" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84930635082" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Specifically, the T7 phage library used here for peptide display contained the same viral peptides as the original VirScan phage library plus additional peptides derived from protein sequences of various microbial B cell antigens available from the IEDB (<a id="intref0865" href="http://www.iedb.org" target="_blank">www.iedb.org</a>). For the computational analysis and background correction, we also sequenced the phage library before (input library sample) and after immunoprecipitation with beads alone (mock IP). We performed a number (n&nbsp;= 46) pf technical repeats. Single-end sequencing was performed with the NextSeq500 system (Illumina), to generate approximately two million reads per sample and ∼20 million reads for the input library samples. As previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref170-4" href="#" role="doc-biblioref" data-xml-rid="bib170" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref170-4" aria-controls="sref170-4">Xu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref170-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">173.</div><div>Xu, G.J. ∙ Kula, T. ∙ Xu, Q. ...</div><div><strong>Comprehensive serological profiling of human populations using a synthetic human virome</strong></div><div><em>Science.</em> 2015; <strong>348</strong></div><div>aaa0698–aaa0698</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0698" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84930635082" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), reads were mapped onto the original library sequences with Bowtie 2 and read counts were adjusted according to library size. A zero-inflated generalized Poisson model was used to estimate the <i>p-value</i>s, to reflect enrichment for each of the peptides. We considered peptides to be significantly enriched only if the –log10 <i>p-value</i> was at least 2.3 in all replicates. Species-specific score values were computed for each serum or plasma sample, by counting the significantly enriched peptides for a given species with less than a continuous seven-residue subsequence, the estimated size of a linear epitope, in common. We corrected for unspecific binding of peptides to the capture matrix, by also calculating species-specific background score values by counting the peptides displaying enrichment to the 90th percentile of the mock IP samples. These peptides were used for background subtraction.</div></section><section id="sec-4-4-2"><h4>Whole-exome sequencing (WES), genotyping and genome-wide linkage (GWL) analysis</h4><div id="p0170" role="paragraph">All four members of the family tested were subjected to genomic DNA extraction followed by WES. Exome capture was performed with the SureSelect Human All Exon V4+UTR and SureSelect Human All Exon V6 (Agilent Technologies). Paired-end sequencing was performed on a HiSeq 2000 (Illumina), generating 100-base reads. We used the Genome Analysis Software Kit (GATK) (version 3.4–46) best-practice pipeline to analyze our WES data (<span class="dropBlock reference-citations"><a id="body-ref-sref98-2" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98-2" aria-controls="sref98-2">Li et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref98-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (21003)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-3" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-3" aria-controls="sref117-3">McKenna et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref117-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (10353)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Reads were aligned with the human reference genome hg19, with BWA (<span class="dropBlock reference-citations"><a id="body-ref-sref97-2" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97-2" aria-controls="sref97-2">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref97-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (18663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). PCR duplicates were removed with Picard tools (<a id="intref1010" href="https://picard.sourceforge.net/" target="_blank">https://picard.sourceforge.net/</a>). The GATK base quality score recalibrator was applied to correct sequencing artifacts. The GATK HaplotypeCaller was used to identify variant calls (<span class="dropBlock reference-citations"><a id="body-ref-sref38-2" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-2" aria-controls="sref38-2">DePristo et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref38-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5537)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref117-4" href="#" role="doc-biblioref" data-xml-rid="bib117" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref117-4" aria-controls="sref117-4">McKenna et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref117-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">119.</div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (10353)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) Variants were annotated with SnpEff (<a id="intref1110" href="https://snpeff.sourceforge.net/" target="_blank">https://snpeff.sourceforge.net/</a>) (<span class="dropBlock reference-citations"><a id="body-ref-sref30-2" href="#" role="doc-biblioref" data-xml-rid="bib30" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref30-2" aria-controls="sref30-2">Cingolani et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref30-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Cingolani, P. ∙ Platts, A. ∙ Wang, L. ...</div><div><strong>A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</strong></div><div><em>Fly (Austin).</em> 2012; <strong>6</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/fly.19695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84862506964" target="_blank" aria-label="Open Scopus in new tab">Scopus (3490)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Ffly.19695&amp;pmid=22728672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0175" role="paragraph">All four members of the family were genotyped with Genome-Wide Human SNP Array 6.0 data. Genotype calling was achieved with the Affymetrix Power Tools Software Package (<a id="intref0870" href="https://www.affymetrix.com/support/developer/powertools/changelog/index.html" target="_blank">https://www.affymetrix.com/support/developer/powertools/changelog/index.html</a>). Parametric multipoint linkage analysis was performed with MERLIN 1.1.2 software (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Abecasis et&nbsp;al., 2002</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Abecasis, G.R. ∙ Cherny, S.S. ∙ Cookson, W.O. ...</div><div><strong>Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</strong></div><div><em>Nat. Genet.</em> 2002; <strong>30</strong>:97-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0036338150" target="_blank" aria-label="Open Scopus in new tab">Scopus (2652)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11731797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng786&amp;pmid=11731797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), assuming autosomal recessive inheritance with complete penetrance and a damaging allele frequency of 1&nbsp;× 10<sup>−4</sup>. Allele frequencies were estimated for 562,685 SNP markers, with family founders and HapMap CEU trios. Markers were clustered using an r<sup>2</sup> threshold (–rsq parameter) of 0.4. The genetic variant of interest was confirmed by PCR amplification of the region (forward primer: 5′- gtgaggactacgcgctacc-3′, and reverse primer: 5′-cagaagcattgtcgagccag-3′) followed by Sanger sequencing.</div></section><section id="sec-4-4-3"><h4>cDNA sequencing</h4><div id="p0180" role="paragraph">We used cDNA sequencing to characterize the consequences of the patients’ variant. In brief, total RNA was extracted from EBV-B cells or PBMCs from the patient and healthy donors (QIAGEN). We synthesized cDNA from mRNA, using Superscript III (Thermo Fisher Scientific). A region spanning the mutation site was amplified by PCR with the forward (5′- tagaagccaggcgtcagagc-3′) and reverse R4 (5′- ctcggcattctggtaggcag-3′), R5 (5′- gcaatgaactgggtttcttgg-3′), or R6 (5′- gactcaaagttctcccggaatcc-3′) primers. PCR amplicons were sequenced with the BigDye Terminator Cycle Sequencing Kit (Thermo Fisher Scientific). Alternatively, cDNA generated from patient or control PBMCs was amplified by PCR with the forward primer 5′- tctacactctttccctacacgacgctcttccgatctgtgaggactacgcgctacc-3′ and reverse primer 5′- gtgactggagttcagacgtgtgctcttccgatctgcaatgaactgggtttcttgg-3′, to amplify a region spanning the mutation in a first round of PCR. A second round of PCR was then performed on the PCR products from the first PCR as a template. We used the forward primer 5′- aatgatacggcgaccaccgagatctacactctttccctacacgac −3′ and the reverse primers 5′- caagcagaagacggcatacgagatcgtattcggtgactggagttcagacgtgtg −3′, 5′- caagcagaagacggcatacgagatctcctagagtgactggagttcagacgtgtg −3′, and 5′- caagcagaagacggcatacgagattagttgcggtgactggagttcagacgtgtg −3′, to barcode two healthy control cDNAs and the patient’s cDNA. Adaptor and barcoded PCR amplicons were mixed and purified, and used for next-generation sequencing by Nano Format 250 paired-end MiSeq sequencing (illumina). We aligned MiSeq data to the reference genome with RNA-STAR aligner (<span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Afgan et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Afgan, E. ∙ Baker, D. ∙ van den Beek, M. ...</div><div><strong>The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W3-W10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27137889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw343&amp;pmid=27137889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (9649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The aligned sequencing data were then viewed with Integrative Genome Browser (<span class="dropBlock reference-citations"><a id="body-ref-sref135" href="#" role="doc-biblioref" data-xml-rid="bib135" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref135" aria-controls="sref135">Robinson et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref135"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">138.</div><div>Robinson, J.T. ∙ Thorvaldsdóttir, H. ∙ Winckler, W. ...</div><div><strong>Integrative genomics viewer</strong></div><div><em>Nat. Biotechnol.</em> 2011; <strong>29</strong>:24-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1754" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-78651271733" target="_blank" aria-label="Open Scopus in new tab">Scopus (5120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21221095/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1754&amp;pmid=21221095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-4-4-4"><h4>Reverse transcription-quantitative PCR (RT-qPCR)</h4><div id="p0185" role="paragraph">Total RNA was extracted from cells with the RNeasy Kit (QIAGEN) or the Quick-RNA kit (Zymo Research). Reverse transcription was performed with the SuperScript III enzyme (Thermo Fisher Scientific). Messenger RNAs were quantified with the cDNA as a template and the <i>TBX21</i> (Probe 1: Hs00894393 and Probe 2: Hs00894391), <i>IFNG</i> (Hs99999041), <i>TNF</i> (Hs01113624), <i>IL5</i> (Hs01548712) (Thermo Fisher), <i>GUSB</i> (Thermo Fisher) probes.</div></section><section id="sec-4-4-5"><h4>T-bet/TBX21 overexpression</h4><div id="p0190" role="paragraph">A wild-type (WT) <i>TBX21</i> plasmid with a pCMV6 backbone was purchased from Origene (RC207902). The genetic variants of <i>TBX21</i>studied here, and the variants described in gnomAD (<a id="intref0875" href="https://gnomad.broadinstitute.org/" target="_blank">https://gnomad.broadinstitute.org/</a>) or in our in-house database were introduced into pCMV6-WT <i>TBX21</i> by site-directed mutagenesis PCR with the CloneAmp HiFi PCR Premix (Takara Bio). <i>TBX21</i> WT plasmids or plasmids containing variants were used to transfect HEK293T cells in the presence of Lipofectamine 2000 reagent (Thermo Fisher Scientific) to achieve overexpression. The WT or mutant (Mut) <i>TBX21</i> was overexpressed in <i>in&nbsp;vitro</i>-expanded T&nbsp;cells, HVS-T cells, or NK-92 cells, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref113-1" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-1" aria-controls="sref113-1">Martínez-Barricarte et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref113-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Martínez-Barricarte, R. ∙ de Jong, S.J. ∙ Markle, J. ...</div><div><strong>Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest</strong></div><div><em>Curr. Protoc. Immunol.</em> 2016; <strong>115</strong>:7.21C.1-7.21C.12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpim.15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85029409873" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpim.15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, the <i>TBX21</i> plasmid was subcloned into pLZRSires-ΔNGFR (Addgene) (<span class="dropBlock reference-citations"><a id="body-ref-sref161-2" href="#" role="doc-biblioref" data-xml-rid="bib161" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref161-2" aria-controls="sref161-2">van de Vosse et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref161-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">164.</div><div>van de Vosse, E. ∙ Verhard, E.M. ∙ de Paus, R.A. ...</div><div><strong>Antisense-mediated exon skipping to correct IL-12Rbeta1 deficiency in T&nbsp;cells</strong></div><div><em>Blood.</em> 2009; <strong>113</strong>:4548-4555</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-12-196220" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-66549092636" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19258592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-12-196220&amp;pmid=19258592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then used pLZRS containing the WT or Mut <i>TBX21</i> to transfect Phoenix A cells, and retroviruses were produced and concentrated with Retro-X (Takara Bio). Concentrated retrovirus preparations were used for retroviral transduction.</div></section><section id="sec-4-4-6"><h4>Immunoblotting</h4><div id="p0195" role="paragraph">Proteins were solubilized in RIPA buffer containing 10&nbsp;mM Tris-Cl (pH 8.0), 1&nbsp;mM EDTA, 0.5&nbsp;mM EGTA, 1% Triton X-100, 0.1% sodium deoxycholate, 0.1% SDS and 150&nbsp;mM NaCl supplemented with protease-inhibitor cocktail (Complete mini, Roche). Alternatively, nuclear proteins from HEK293T transfectants were purified with the NE-PER Nuclear Extraction kit in accordance with the manufacturer’s procedures (Thermo Fisher Scientific). Proteins were quantified with the Bradford assay. Equal amounts of lysate (5&nbsp;μg to 50&nbsp;μg of) were mixed with SDS sample buffer and boiled for 5min at 95°C before being subjected to electrophoresis in 12% acrylamide SDS-PAGE gels. The following antibodies were used for immunoblotting: mouse anti-human T-bet 4B10 monoclonal Ab (BioxCell), rabbit anti-human T-bet polyclonal Ab (Proteintech), anti-Flag-M2 HRP-conjugated Ab (Sigma-Aldrich), anti-α-tubulin HRP-conjugated Ab (Proteintech), anti-lamin A/C Ab (Santa Cruz), anti-GAPDH HRP-conjugated Ab (Santa Cruz), anti-STAT1 monoclonal Ab (BD Biosciences), sheep ECL anti-mouse IgG HRP-conjugated Ab (GE Healthcare), and ECL anti-rabbit IgG HRP-conjugated Ab (GE Healthcare).</div></section><section id="sec-4-4-7"><h4>Electrophoretic mobility shift assays (EMSA)</h4><div id="p0200" role="paragraph">HEK293T cells were transfected with empty plasmid (EV), WT or mutant <i>TBX21</i> pCMV6 plasmids. After 2&nbsp;days, nuclear extracts were obtained from the cells with the NE-PER Nuclear Extraction Kit (Thermo Fisher Scientific). EMSA was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref103" href="#" role="doc-biblioref" data-xml-rid="bib103" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref103" aria-controls="sref103">Liu et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref103"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Liu, L. ∙ Okada, S. ∙ Kong, X.-F. ...</div><div><strong>Gain-of-function human STAT1 mutations impair IL-17 immunity and underlie chronic mucocutaneous candidiasis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2011; <strong>208</strong>:1635-1648</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20110958" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79961154447" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21727188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20110958&amp;pmid=21727188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, <sup>32</sup>P was used to label consensus T-box responsive element (TBRE) duplexes of DNA oligos (WT oligo: 5′-gataatttcacacctaggtgtgaaatt-3′, Mut Oligo as control: 5′-gataatttcacgtctaggtacgaaatt-3′ and 5′-gataatttcgtacctagacgtgaaatt-3′). Unlabeled WT TBRE duplex was used as non-radioactive competitive probe. We mixed 10&nbsp;μg of nuclear extract from each condition or 80&nbsp;μg (8x) of nuclear extract from Mut <i>TBX21</i> transfectants with or without unlabeled probe, 2&nbsp;μg mouse IgG1 control (BioxCell) or 2&nbsp;μg mouse anti-T-bet 4B10 monoclonal Ab (BioxCell) and incubated for 30&nbsp;min. <sup>32</sup>P-labeled WT or Mut TBRE duplex was then added and the mixture was incubated for 30&nbsp;min on ice in the presence of poly-dIdC (Sigma-Aldrich). Samples were subjected to polyacrylamide gel electrophoresis. The gel was dried and placed against X-ray film for 2&nbsp;days, after which the radioactive signal was read on a Typhoon platform.</div></section><section id="sec-4-4-8"><h4>Luciferase reporter assays</h4><div id="p0205" role="paragraph">HEK293T cells were transfected with the indicated expression plasmids, firefly luciferase plasmids under the control of WT or Mut <i>TBRE</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref76-4" href="#" role="doc-biblioref" data-xml-rid="bib76" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref76-4" aria-controls="sref76-4">Janesick et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref76-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Janesick, A. ∙ Shiotsugu, J. ∙ Taketani, M. ...</div><div><strong>RIPPLY3 is a retinoic acid-inducible repressor required for setting the borders of the pre-placodal ectoderm</strong></div><div><em>Development.</em> 2012; <strong>139</strong>:1213-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.071456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857300258" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.071456&amp;pmid=22354841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as previously described, or with human <i>IFNG</i> promoters in a pGL4.10 backbone (−565 -&nbsp;+85 of <i>IFNG</i>), and with a constitutively expressed <i>Renilla</i> luciferase plasmid for normalization (pRL-SV40). Cells were transfected in the presence of Lipofectamine 2000 (Thermo Fisher Scientific) for 3&nbsp;days. Luciferase levels were measured with Dual-Glo reagent, according to the manufacturer’s protocol (Promega). Firefly luciferase values were normalized against <i>Renilla</i> luciferase values, and fold-induction is reported relative to controls transfect with empty plasmid.</div></section><section id="sec-4-4-9"><h4>Generation and analysis of T-bet-deficient NK-92 cells</h4><div id="p0210" role="paragraph">T-bet-deficient NK-92 cells were generated according to a protocol described elsewhere (<span class="dropBlock reference-citations"><a id="body-ref-sref140-2" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-2" aria-controls="sref140-2">Sanjana et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref140-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>Sanjana, N.E. ∙ Shalem, O. ∙ Zhang, F.</div><div><strong>Improved vectors and genome-wide libraries for CRISPR screening</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:783-784</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84905262730" target="_blank" aria-label="Open Scopus in new tab">Scopus (1702)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25075903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3047&amp;pmid=25075903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We used oligonucleotides for gRNA9 and gRNA15 (gRNA9_Forward: 5′- caccgtccaacaatgtgacccaggt-3′, gRNA_Reverse: 5′-aaacacctgggtcacattgttggac-3′; gRNA15_Forward: 5′-caccgccgcactcaccgtccctgct-3′, gRNA15_reverse: 5′- aaacagcagggacggtgagtgcggc-3′). DNA duplex was annealed from the oligonucleotides above, and was inserted into pLenti-CRISPR-V2 plasmids (Addgene) (<span class="dropBlock reference-citations"><a id="body-ref-sref140-3" href="#" role="doc-biblioref" data-xml-rid="bib140" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref140-3" aria-controls="sref140-3">Sanjana et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref140-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">143.</div><div>Sanjana, N.E. ∙ Shalem, O. ∙ Zhang, F.</div><div><strong>Improved vectors and genome-wide libraries for CRISPR screening</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:783-784</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84905262730" target="_blank" aria-label="Open Scopus in new tab">Scopus (1702)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25075903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3047&amp;pmid=25075903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We then used pLenti-CRISPR plasmids, VSV-G envelope, and psPAX2 plasmids to transfect HEK293T cells in the presence of Lipofectamine 2000. Supernatants containing lentiviruses were harvested and concentrated with Lenti-X concentrator (Takara Bio). We resuspended 10<sup>6</sup> NK-92 cells with 20x concentrated lentiviruses for gRNA9, gRNA15 or empty plasmid, in the presence of proleukin IL-2 (200&nbsp;U/mL) (PROMETHEUS). After incubation for 2&nbsp;days, we added 3.75&nbsp;μg/mL puromycin to the culture for the selection of stably transduced NK-92 cells. After a further two weeks, cells were harvested for the Surveyor Nuclease Assay (Integrated DNA Tech) and a three-primer PCR-based screening system (<span class="dropBlock reference-citations"><a id="body-ref-sref63" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63" aria-controls="sref63">Harayama and Riezman, 2017</a><div class="dropBlock__holder" data-db-target-of="sref63"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Harayama, T. ∙ Riezman, H.</div><div><strong>Detection of genome-edited mutant clones by a simple competition-based PCR method</strong></div><div><em>PLoS ONE.</em> 2017; <strong>12</strong>:e0179165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0179165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85020204183" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28586390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0179165&amp;pmid=28586390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to confirm successful DNA editing. Cells were plated at a density of 0.7 cells in 200μL per well in a U-bottomed 96-well plate. Two weeks later, single-cell clones were screened with a three-primer PCR-based system, with Phire Direct PCR Master Mix (Thermo Fisher Scientific). The primers used for this screening assay were gRNA9_Forward: 5′-&nbsp;agcccctacccctaattcct-3′, gRNA9_Reverse: 5′-ggcatctattccctgggacc-4’, gRNA9_Reverse_in: 5′-cttttgaagagcaggtcctacct-3′ and gRNA15_Forward: 5′-GCCTGAATATGACCCCCGTC-3′, gRNA15_Reverse: 5′-caccactaccaccactaaagc-3′; gRNA15_Forward_in: 5′-acagagatgatcatcaccaagca-3′. Single-cell clones with biallelic disruptions were selected for validation by RT-qPCR and western blotting. EV-transduced single-cell clones were randomly picked as controls. Cells were then transduced with retroviruses generated from pLZRS-ires-ΔCD271 plasmids containing EV, WT, Mut or K314R <i>TBX21,</i> as described above (<span class="dropBlock reference-citations"><a id="body-ref-sref113-2" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-2" aria-controls="sref113-2">Martínez-Barricarte et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref113-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Martínez-Barricarte, R. ∙ de Jong, S.J. ∙ Markle, J. ...</div><div><strong>Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest</strong></div><div><em>Curr. Protoc. Immunol.</em> 2016; <strong>115</strong>:7.21C.1-7.21C.12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpim.15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85029409873" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpim.15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Stably transduced cells were isolated with anti-CD271 antibody-coated beads (Miltenyi Biotec). Cells were stimulated with 50 pg/mL IL-12 (R&amp;D) and 10&nbsp;ng/mL IL-18 (InvivoGen), or were left unstimulated, and all cells were incubated for one day. The IFN-γ content of the supernatants was determined by ELISA. Intracellular IFN-γ production was measured with flow cytometry with intracellular staining (ICS). In brief, cells were fixed and permeabilized (BioLegend), and were subjected to ICS staining with FcBlock (Miltenyi Biotec), anti-T-bet BV421 (BioLegend) and anti-IFN-γ PE (BioLegend) before their use for ICS flow cytometry analysis.</div></section><section id="sec-4-4-10"><h4>Cytokine determinations in <i>TBX21</i>-transduced primary CD4<sup>+</sup> T&nbsp;cells</h4><div id="p0215" role="paragraph">PBMCs from three healthy donors were stained with FcBlock (Miltenyi Biotec), anti-CD4-APC-Cy7, anti-CD45RA-Alexa488, anti-CCR7-Alexa647 and Live-Dead exclusion Aqua. Naive CD4<sup>+</sup> T&nbsp;cells were isolated by FACS. Cells were stimulated with anti-CD3/CD28 Ab Dynabeads (Thermo Fisher Scientific) at a 1:1 cell:bead ratio under T<sub>H</sub>0 or T<sub>H</sub>2 (recombinant human IL-4 12.5&nbsp;ng/mL, R&amp;D) polarizing conditions, in the presence of 100&nbsp;U/mL proleukin IL-2 (PROMETHEUS). Fresh medium (X-vivo 15, 5% human AB serum) and cytokines were added every 3&nbsp;days. After 7&nbsp;days, cells were restimulated with Dynabeads at a 1:2 ratio (bead:cell). One day after restimulation, cells were transduced with retroviruses generated from pLZRS-ires-ΔCD271 plasmids containing WT or Mut <i>TBX21,</i> as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref113-3" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-3" aria-controls="sref113-3">Martínez-Barricarte et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref113-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Martínez-Barricarte, R. ∙ de Jong, S.J. ∙ Markle, J. ...</div><div><strong>Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest</strong></div><div><em>Curr. Protoc. Immunol.</em> 2016; <strong>115</strong>:7.21C.1-7.21C.12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpim.15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85029409873" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpim.15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Transduced cells were isolated with anti-CD271 antibody-coated beads (Miltenyi Biotec) 14&nbsp;days after stimulation. Intracellular cytokine production was determined by ICS. In brief, cells were restimulated with 25&nbsp;ng/mL phorbol 12-myristate 13-acetate (PMA) and 0.5&nbsp;μM ionomycin for 6&nbsp;h in the presence of GolgiPlug (BD Bioscences). Cells were stained with Zombie-NIR live-dead exclusion dye (BioLegend), FcBlock (Miltenyi Biotec), and FITC-conjugated anti-CD271 antibody (BioLegend). Cells were then fixed, permeabilized and subjected to ICS staining with anti-TNF-α-BV510 (BioLegend), anti-IFN-γ-PERCP/Cy5.5 (BioLegend), anti-IL-4-PE, anti-IL-13-APC antibodies, followed by flow cytometry analysis with an Aurora cytometer (Cytek).</div></section><section id="sec-4-4-11"><h4>HVS-T cell analysis</h4><div id="p0220" role="paragraph">We plated 200,000 HVS-T cells from healthy donors or P in a 96-well U-bottomed plate at a density of 1x10<sup>6</sup> cells/mL. The plate was incubated for 2&nbsp;days, and the culture supernatants and cells were harvested for ELISA or RT-qPCR for <i>IFNG</i> and <i>TNF</i>. HVS-T cells from P were transduced with retroviruses generated from pLZRS-ires-ΔCD271 plasmids containing WT <i>TBX21</i> or EV, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref113-4" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-4" aria-controls="sref113-4">Martínez-Barricarte et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref113-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Martínez-Barricarte, R. ∙ de Jong, S.J. ∙ Markle, J. ...</div><div><strong>Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest</strong></div><div><em>Curr. Protoc. Immunol.</em> 2016; <strong>115</strong>:7.21C.1-7.21C.12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpim.15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85029409873" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpim.15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After incubation for 5&nbsp;days, the HVS-T cells were stimulated with 25&nbsp;ng/mL PMA and 500&nbsp;nM ionomycin or were left untreated; they were then incubated in the presence of GolgiPlug (BD Biosciences) for 6-8 h. Cells were stained with Zombie-NIR live-dead dye(BioLegend), FcBlock (Miltenyi Biotec), anti-CD4-BV750 (BD Biosciences), anti-CD3-BV650 (BD Biosciences), anti-CD8a-Pacific Blue (BioLegend), and anti-CD271-FITC (BioLegend) antibodies, fixed and permeabilized for ICS staining with anti-T-bet-PE (BioLegend), anti-IFN-γ-BV711 (BioLegend), anti-TNF-α-BV510 (BioLegend), anti-IL-13-APC (BioLegend), anti-IL-10-PE-Dazzle594 (BioLegend) antibodies. ICS analysis was performed on an Aurora cytometer (Cytek).</div><div id="p0225" role="paragraph">HVS-T cells were also analyzed for IFN-γ production in response to <i>PSMD9</i>, <i>ARHGAP27</i> and <i>ERN1</i> cDNA overexpression. In brief,&nbsp;pTRIP-mCherry-2A-PSMD9 and ARHGAP27 were generated by subcloning from MGC clones BC004184 and BC067345 (transOMIC). A C-terminal V5 tag was added upstream of the stop codons. The pLenti-Cter-GFP-2A-puro empty vector plasmid was obtained from OriGene. The pCMV6-ERN1-Cter-Myc-Flag plasmid was obtained from OriGene. pLenti-ERN1-Cter-GFP (fusion)-2A-puro was generated by the double digestion, with <i>EcoR</i>I and <i>Xho</i>I, of pCMV6-ERN1 and pLenti-Cter-GFP-2A-puro, followed by the isolation of the ERN1 fragment by gel extraction, and T4 ligation. Lentiviruses containing cDNAs encoding EV, PSMD9, ARHGAP27, ERN1-GFP were generated in HEK293T cells by transfection with pTRIP or pLenti-GFP-2A-puro, VSV.G and Pax2 packaging plasmids. Transfected HEK293T cells were analyzed for the expression of PSMD9 and ARHGAP27 by assessing the expression of the C-terminal V5 tag and ERN1 by western blotting with anti-GFP Ab or anti-ERN1 Ab (Proteintech). Lentiviruses containing the cDNAs of EV, <i>PSMD9</i>, <i>ARHGAP27</i> and <i>ERN1</i>-GFP were used to transduce HVS-T cells derived from the cells of P (M/M), or M/M HVS-T cells retrovirally transduced with WT T-bet (M/M+WT). After 2 – 4&nbsp;days of transduction, cells were stimulated with 100&nbsp;ng/mL PMA and 2&nbsp;μM ionomycin for 8&nbsp;h and harvested for staining for flow cytometry. ERN1-transduced HVS-T cells and their control HVS-T cells were stained with Zombie NIR Fixable Viability Dye (BioLegend) and surface staining was then performed with FcBlock, anti-CD271-PE/Dazzle594 (BioLegend), anti-CD4-BUV563 (BD Biosciences) and anti-CD8-BUV737 (BD Biosciences) antibodies. Cells were fixed and permeabilized with the FOXP3/Transcription Factor Buffer kit (Thermo Fisher Scientific) and stained intracellularly with anti-T-bet-PE/Cy7 (BioLegend), anti-IFN-γ-BV711 (BioLegend) and anti-TNF-α-BV510 (BioLegend) antibodies. For PSMD9- and ARHGAP27-transduced HVS-T cells and their control HVS-T cells, cells were stained with Zombie NIR Fixable Viability Dye (BioLegend) and then surface stained with FcBlock and anti-CD271-Alexa647 antibody (BD PharMingen). Cells were fixed with 2.8% PFA and permeabilized with the FOXP3/Transcription Factor Buffer kit (Thermo Fisher Scientific). They were then subjected to intracellular staining with anti-T-bet-PE (BioLegend) and anti-IFN-γ-BV711 (BioLegend) antibodies. ICS analysis was performed on an Aurora cytometer (Cytek).</div></section><section id="sec-4-4-12"><h4>Functional analysis of expanded T-bet-deficient CD4<sup>+</sup> T&nbsp;cells <i>in&nbsp;vitro</i></h4><div id="p0230" role="paragraph">CD4<sup>+</sup> T&nbsp;cells were isolated from PBMCs from five healthy controls (CTL), IL-12Rβ1-deficient (<i>IL12RB1</i> M/M) patients, the T-bet WT/M heterozygous father (<i>TBX21</i> WT/M), and the T-bet M/M patient (<i>TBX21</i> M/M), with anti-CD4 antibody-coated beads (Miltenyi Biotec). They were then expanded with anti-CD3/CD28 antibody-coated Dynabeads (Thermo Fisher Scientific) at a 1:1 (cell:bead) ratio under T<sub>H</sub>0 or T<sub>H</sub>1 (2.5&nbsp;ng/mL IL-12 R&amp;D and anti-human IL-4 neutralizing Ab InVivoMab, BioXcell) conditions, in the presence of 100&nbsp;U/mL proleukin IL-2 (PROMETHEUS) and anti-human IL-10 neutralizing Ab (Thermo Fisher Scientific). Medium (X-vivo15 5% human AB serum) and cytokine cocktails were refreshed every 3 – 4&nbsp;days. After 7&nbsp;days, the culture consisted of &gt; 95% CD4<sup>+</sup> T&nbsp;cells. Cells were restimulated with anti-CD3/CD28 Ab-coated Dynabeads every 7-8&nbsp;days. P’s CD4<sup>+</sup> T&nbsp;cells were transduced with retroviruses obtained from pLZRS-ires-ΔCD271 plasmids with or without WT <i>TBX21</i> or EV, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref113-5" href="#" role="doc-biblioref" data-xml-rid="bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-5" aria-controls="sref113-5">Martínez-Barricarte et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref113-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">115.</div><div>Martínez-Barricarte, R. ∙ de Jong, S.J. ∙ Markle, J. ...</div><div><strong>Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest</strong></div><div><em>Curr. Protoc. Immunol.</em> 2016; <strong>115</strong>:7.21C.1-7.21C.12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpim.15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85029409873" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpim.15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Six days after transduction, cells were restimulated with 25&nbsp;ng/mL PMA and 500&nbsp;nM ionomycin in the presence of GolgiPlug (BD Biosciences) for 6 h. Cells were stained with Zombie NIR Live-Dead exclusion reagents (BioLegend), anti-CD271-FITC antibody and were then fixed and permeabilized. They were stained with anti-TNF-α-BV510, anti-IL-13-APC, anti-IL-4-PE, anti-IL-17A-BV785, anti-IL-10-PE-Dazzle594, anti-IL-5-BV421 and anti-IFN-γ-PerCp-Cy5.5 antibodies and subjected to flow cytometry in an Aurora cytometer (Cytek). Three weeks after their initial expansion, transduced CD271<sup>+</sup> CD4<sup>+</sup> T&nbsp;cells from P were isolated with anti-CD271 antibody-coated beads (Miltenyi Biotec). The cells isolated, and expanded CD4<sup>+</sup> T<sub>H</sub>0 cells from healthy donors or T-bet wt/m individuals were restimulated with anti-CD3/CD28 antibody-coated beads in the absence of cytokine for 16&nbsp;h or 48 h. Cytokine levels were determined in 13-Legendplex assays (BioLegend) on culture supernatants from cultures restimulated for 48 h. Total RNA was extracted from cells restimulated for 16&nbsp;h for RNA-seq analysis (QIAGEN). In brief, 100&nbsp;ng of total RNA was used to generate RNA-Seq libraries with the Illumina TruSeq stranded mRNA LT kit (Cat# RS-122-2101). Libraries prepared with unique barcodes were pooled at equal molar ratios. The pool was denatured and sequenced on an Illumina NextSeq 500 sequencer with high-output V2 reagents and NextSeq Control Software v1.4 to generate 75&nbsp;bp single reads, according to the manufacturer’s protocol (Illumina, Cat# 15048776).</div><div id="p0235" role="paragraph">RNA-seq FASTQ files were obtained, and inspected by fastqc for quality control of the raw data. The sequencing reads of each FASTQ file were then mapped to the GENCODE human reference genome GRCh38.p13 (<span class="dropBlock reference-citations"><a id="body-ref-sref49" href="#" role="doc-biblioref" data-xml-rid="bib49" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref49" aria-controls="sref49">Frankish et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref49"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Frankish, A. ∙ Diekhans, M. ∙ Ferreira, A.-M. ...</div><div><strong>GENCODE reference annotation for the human and mouse genomes</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D766-D773</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059554160" target="_blank" aria-label="Open Scopus in new tab">Scopus (366)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30357393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky955&amp;pmid=30357393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with STAR aligner v2.7 (<span class="dropBlock reference-citations"><a id="body-ref-sref40-3" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-3" aria-controls="sref40-3">Dobin et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref40-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (9649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and the quality of each mapped alignment in the BAM file was evaluated by RSeQC (<span class="dropBlock reference-citations"><a id="body-ref-sref166-2" href="#" role="doc-biblioref" data-xml-rid="bib166" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref166-2" aria-controls="sref166-2">Wang et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref166-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">169.</div><div>Wang, L. ∙ Wang, S. ∙ Li, W.</div><div><strong>RSeQC: quality control of RNA-seq experiments</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2184-2185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84865074382" target="_blank" aria-label="Open Scopus in new tab">Scopus (721)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts356&amp;pmid=22743226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Reads were quantified to generate gene-level read counts from the read alignment, with featureCounts v1.6.0 (<span class="dropBlock reference-citations"><a id="body-ref-sref99-3" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-3" aria-controls="sref99-3">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref99-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (4007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), based on GENCODE GRCh38.p13 gene annotation. The gene-level read counts were then normalized and log2-transformed by DESeq2 (<span class="dropBlock reference-citations"><a id="body-ref-sref106-2" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-2" aria-controls="sref106-2">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref106-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (14777)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), to obtain the gene expression value for all genes and all samples. An analysis of the differential expression of all genes and immune genes was performed to compare gene expression in the four TBET samples (<i>TBX21</i> WT/M, <i>TBX21</i> M/M, M/M+EV, and M/M+WT) with that in five control samples, with each sample having two technical replicates. The 1,984 immune-related genes were identified from the immune system process gene set in the Molecular Signature Database (MSigDB) (<span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Liberzon et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Liberzon, A. ∙ Birger, C. ∙ Thorvaldsdóttir, H. ...</div><div><strong>The Molecular Signatures Database (MSigDB) hallmark gene set collection</strong></div><div><em>Cell Syst.</em> 2015; <strong>1</strong>:417-425</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84955328286" target="_blank" aria-label="Open Scopus in new tab">Scopus (1439)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.12.004&amp;pmid=26771021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). By applying the trimmed mean of M values (TMM), normalization and gene-wise generalized linear model regression were performed in edgeR (<span class="dropBlock reference-citations"><a id="body-ref-sref134-2" href="#" role="doc-biblioref" data-xml-rid="bib134" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref134-2" aria-controls="sref134-2">Robinson et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref134-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">137.</div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (12880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The genes displaying significant differential expression were selected on the basis of a <i>p-value</i> ≤ 0.05 and a FDR ≤ 0.05. The differential gene expression data were plotted as a heatmap by ComplexHeatmap (<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Gu et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M.</div><div><strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:2847-2849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw313" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84992187637" target="_blank" aria-label="Open Scopus in new tab">Scopus (843)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw313&amp;pmid=27207943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and genes and samples were clustered on the basis of complete linkage and Euclidean distance of gene expression values.</div></section><section id="sec-4-4-13"><h4>Omni-ATAC-seq on CD4<sup>+</sup> T&nbsp;cells and analysis</h4><div id="p0240" role="paragraph">CD4<sup>+</sup> T&nbsp;cells derived from healthy controls, <i>IL12RB1</i> M/M patients, a <i>TBX21</i> WT/M parent and the <i>TBX21</i> M/M P were expanded under T<sub>H</sub>0 conditions, as described above. Nine days after the retroviral transduction of CD4<sup>+</sup> T&nbsp;cells from P, as described above, CD271<sup>+</sup> cells were isolated with anti-CD271 antibody-coated beads (Miltenyi Biotec). Omni-ATAC-seq library preparation was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Corces et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>Corces, M.R. ∙ Trevino, A.E. ∙ Hamilton, E.G. ...</div><div><strong>An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:959-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85030994641" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28846090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4396&amp;pmid=28846090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, 50,000 cells from three healthy donors (CTL), two <i>IL12RB1</i> M/M patients, technical duplicates for a <i>TBX21</i> WT/M individual, the <i>TBX21</i> M/M P and P cells complemented with EV (M/M+EV) or WT (M/M+WT) <i>TBX21</i> were harvested, washed with 50&nbsp;μl cold PBS, and centrifuged to obtain a pellet. Pellets were lysed with 50&nbsp;μl cold lysis buffer consisting of 48.5&nbsp;μl resuspension buffer (10&nbsp;mM Tris-HCl pH 7.5, 10&nbsp;mM NaCl, 3&nbsp;mM MgCl<sub>2</sub> in water), 0.5&nbsp;μl 10% NP-40, 0.5&nbsp;μl 10% Tween-20, and 0.5&nbsp;μl 1% digitonin. Lysates were incubated on ice and then washed with 0.1% Tween-20 resuspension buffer. Lysates were centrifuged and the nuclei in the pellet were subjected to Tn5 transposition with 50&nbsp;μL of a mixture of 25&nbsp;μL 2 x TD buffer, 16.5&nbsp;μL PBS, 0.5&nbsp;μL 10% Tween-20, 0.5&nbsp;μL 1% digitonin, 2.5&nbsp;μL Tn5 transposase, 5&nbsp;μL nuclease-free water at 37°C in a thermomixer operating at 1,000&nbsp;rpm for 30&nbsp;min. DNA fragments were extracted with the MinElute PCR purification kit and eluted in 30&nbsp;μl EB buffer (QIAGEN). We used 10&nbsp;μL of eluted DNA for the amplification of DNA fragments with the NEBNext PCR master mix (New England BioLab), over six cycles, with the Ad1_forward and 15 indexed Ad2_reverse primers, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref15-1" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-1" aria-controls="sref15-1">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1949)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-1" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-1" aria-controls="sref16-1">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Buenrostro, J.D. ∙ Wu, B. ∙ Chang, H.Y. ...</div><div><strong>ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide</strong></div><div><em>Curr. Protoc. Mol. Biol.</em> 2015; <strong>109</strong>:21.29.1-21.29.9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/0471142727.mb2129s109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84949212255" target="_blank" aria-label="Open Scopus in new tab">Scopus (502)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25559105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F0471142727.mb2129s109&amp;pmid=25559105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The partially amplified library was subjected to quantitative PCR analysis with the Ad1 and Ad2 primers, Sybr Green reagents and NEBNext PCR master mix for 25 cycles. Additional PCR cycles were performed, based on 1/3 the maximal fluorescence for each sample, as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref15-2" href="#" role="doc-biblioref" data-xml-rid="bib15" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref15-2" aria-controls="sref15-2">Buenrostro et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref15-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1949)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>, <span class="dropBlock reference-citations"><a id="body-ref-sref16-2" href="#" role="doc-biblioref" data-xml-rid="bib16" href-manipulated="true" aria-label="Reference 2015" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref16-2" aria-controls="sref16-2">2015</a><div class="dropBlock__holder" data-db-target-of="sref16-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Buenrostro, J.D. ∙ Wu, B. ∙ Chang, H.Y. ...</div><div><strong>ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide</strong></div><div><em>Curr. Protoc. Mol. Biol.</em> 2015; <strong>109</strong>:21.29.1-21.29.9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/0471142727.mb2129s109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84949212255" target="_blank" aria-label="Open Scopus in new tab">Scopus (502)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25559105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F0471142727.mb2129s109&amp;pmid=25559105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A left-sided isolation of PCR products was performed with AMPure beads (Beckman Coulter). DNA was quantified with a Qubit fluorimeter. Equal amounts of each sample were pooled and subjected to paired-end sequencing on a NextSeq high-output sequencer generating 75&nbsp;bp reads.</div><div id="p0245" role="paragraph">The ATAC-seq reads were aligned with the hg38 genome from the BSgenome.Hsapiens.UCSC.hg38 Bioconductor package (version 1.4.1), with Rsubread’s alignment method in paired-end mode, with fragments of 1 to 5,000 base pairs in length considered correctly paired (<span class="dropBlock reference-citations"><a id="body-ref-sref99-4" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-4" aria-controls="sref99-4">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref99-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (4007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Normalized, fragment signal bigWigs were created with the rtracklayer package. Peak calls were made with MACS2 software in BAMPE mode (<span class="dropBlock reference-citations"><a id="body-ref-sref46-2" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-2" aria-controls="sref46-2">Feng et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref46-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Feng, J. ∙ Liu, T. ∙ Qin, B. ...</div><div><strong>Identifying ChIP-seq enrichment using MACS</strong></div><div><em>Nat. Protoc.</em> 2012; <strong>7</strong>:1728-1740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2012.101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866127830" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22936215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2012.101&amp;pmid=22936215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref173-2" href="#" role="doc-biblioref" data-xml-rid="bib173" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref173-2" aria-controls="sref173-2">Zhang et&nbsp;al., 2008</a><div class="dropBlock__holder" data-db-target-of="sref173-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">176.</div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (5850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and peak summits were used in MEME-ChIP software the identification of known and novel motifs (<span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Ma et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">113.</div><div>Ma, W. ∙ Noble, W.S. ∙ Bailey, T.L.</div><div><strong>Motif-based analysis of large nucleotide data sets using MEME-ChIP</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:1428-1450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84901830680" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24853928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.083&amp;pmid=24853928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Differential ATAC-seq signals were identified with the DESeq2 package (<span class="dropBlock reference-citations"><a id="body-ref-sref106-3" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106-3" aria-controls="sref106-3">Love et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref106-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (14777)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Fragment length distribution plots and nucleosome-free/mono-nucleosome meta-TSS plots were produced with the profileplyr package (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Carroll and Barrows, 2019</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>Carroll, T. ∙ Barrows, D.</div><div><strong>profileplyr: Visualization and annotation of read signal over genomic ranges with profileplyr</strong></div><div>2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.CarrollD.Barrowsprofileplyr%3A+Visualization+and+annotation+of+read+signal+over+genomic+ranges+with+profileplyr2019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Heatmaps of ChIP-seq and ATAC-seq signals were created from the normalized bigWigs signal, with DeepTools2 software (<span class="dropBlock reference-citations"><a id="body-ref-sref130-2" href="#" role="doc-biblioref" data-xml-rid="bib130" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref130-2" aria-controls="sref130-2">Ramírez et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref130-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">133.</div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85017133693" target="_blank" aria-label="Open Scopus in new tab">Scopus (955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw257&amp;pmid=27079975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0250" role="paragraph">T-bet ChIP-seq data (SRR332104) and its input control (SRR332102) were retrieved as unaligned sequences from the European Nucleotide Archive (<span class="dropBlock reference-citations"><a id="body-ref-sref81-4" href="#" role="doc-biblioref" data-xml-rid="bib81" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref81-4" aria-controls="sref81-4">Kanhere et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref81-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Kanhere, A. ∙ Hertweck, A. ∙ Bhatia, U. ...</div><div><strong>T-bet and GATA3 orchestrate Th1 and Th2 differentiation through lineage-specific targeting of distal regulatory elements</strong></div><div><em>Nat. Commun.</em> 2012; <strong>3</strong>:1268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms2260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871783406" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms2260&amp;pmid=23232398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Sequences were aligned with the hg38 genome from the Bsgenome.Hsapiens.UCSC.hg38 Bioconductor package (version 1.4.1), with Rsubread’s alignment method, and predicted fragment lengths calculated with the ChIPQC package (<span class="dropBlock reference-citations"><a id="body-ref-sref19-2" href="#" role="doc-biblioref" data-xml-rid="bib19" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref19-2" aria-controls="sref19-2">Carroll et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref19-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>Carroll, T.S. ∙ Liang, Z. ∙ Salama, R. ...</div><div><strong>Impact of artifact removal on ChIP quality metrics in ChIP-seq and ChIP-exo data</strong></div><div><em>Front. Genet.</em> 2014; <strong>5</strong>:75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fgene.2014.00075" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84898030206" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24782889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffgene.2014.00075&amp;pmid=24782889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref99-5" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99-5" aria-controls="sref99-5">Liao et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref99-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (4007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref37-2" href="#" role="doc-biblioref" data-xml-rid="bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref37-2" aria-controls="sref37-2">de Santiago and Carroll, 2018</a><div class="dropBlock__holder" data-db-target-of="sref37-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>de Santiago, I. ∙ Carroll, T.</div><div><strong>Analysis of ChIP-seq Data in R/Bioconductor</strong></div><div>2018 195-226</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.de+SantiagoT.CarrollAnalysis+of+ChIP-seq+Data+in+R%2FBioconductor2018195226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Normalized, fragment-extended bigWigs signals were created with the rtracklayer package.</div></section><section id="sec-4-4-14"><h4>EPIC DNA methylation arrays and quantification of DNA methylation</h4><div id="p0255" role="paragraph">CD4<sup>+</sup> T&nbsp;cells derived from healthy controls, a <i>TBX21</i> WT/M individual and the <i>TBX21</i> M/M P were expanded under T<sub>H</sub>0 conditions, as described above. Seventeen days after the retroviral transduction of CD4<sup>+</sup> T&nbsp;cells from P as described above, CD271<sup>+</sup> cells were isolated with anti-CD271 antibody-coated beads (Miltenyi Biotec). DNA from non-transduced or CD271<sup>+</sup> isolated CD4<sup>+</sup> T&nbsp;cells was extracted with a DNA mini kit (QIAGEN). DNA was subjected to bisulfite conversion with the EZ DNA Methylation Kit (ZymoResearch). We quantified the DNA in all samples by performing EPIC arrays (Illumina), following the manufacturer’s instructions. Technical replicates were included for several samples during EPIC processing, to ensure that the measurements were reliable. All microarrays were scanned with the Illumina HiScan system. Data files were processed and converted to beta-values, corresponding to the percentage CpG methylation for a given CpG site. We wrote scripts to filter all loci, retaining those with higher levels of methylation in M/M and M/M+EV than the maximum methylation level in controls, and lower levels of methylation in M/M+WT than in controls (group 1), and those with lower levels of methylation in M/M and M/M+EV than the minimum level of methylation in controls, and higher levels of methylation in M/M+WT than the minimum level in controls (group 2). We then measured and plotted the difference in CpG methylation between the patient and controls, by subtracting the mean methylation level for the five controls from that for M/M and comparing the value obtained with the level of restored CpG methylation by subtracting the level of methylation in M/M+WT from the methylation in M/M+EV. On the plot, red dots represent the loci with higher levels of methylation in the patient than in controls, and green dots represent loci with lower levels of methylation in the patient than in controls. Dot size is proportional to the fold difference in methylation in patients relative to the mean value for controls. The genes of our interest are highlighted on the scatterplot.</div></section><section id="sec-4-4-15"><h4>Immunophenotyping of leukocytes</h4><div id="p0260" role="paragraph">In the CyTOF experiment, PBMCs from 26 healthy adults of diverse ethnicities, nine child or adolescent controls (age range: 1 to 17 years, with diverse ancestries, including one age-matched child from Morocco), six IL-12Rβ1-deficient patients (recruited from previous studies), and the T-bet-deficient patient (M/M). We recruited 26 healthy adult controls locally in New York City. We analyzed the PBMCs of four individuals on two separate dates, as duplicates. We obtained 30 vials of cells from these adult controls, 10 of which were frozen after being allowed to rest at room temperature for 24 – 48&nbsp;h to mimic travel conditions. The vials of cells from the six IL-12Rβ1-deficient patients and the age-matched controls were typically frozen after centrifugation on Ficoll following the overnight shipment of the blood sample. For this experiment, the PBMCs of P (M/M) were prepared from blood that had traveled overnight from Morocco to Paris. Metal-conjugated mAbs were obtained from Fluidigm according to the manufacturer’s instructions. The following antibodies were included in the staining panel: anti-CD45-89Y, anti-CD57-113In, anti-CD11c-115In, anti-CD33-141Pr, anti-CD19-142Nd, anti-CD45RA-143Nd, anti-CD141-144Nd, anti-CD4-145Nd, anti-CD8-146Nd, anti-CD20-147Sm, anti-CD16-148Nd, anti-CD127-149Sm, anti-CD1c-150Nd, anti-CD123-151Eu, anti-CD66b-152Sm, anti-PD-1-153Eu, anti-CD86-154Sm, anti-CD27-155Gd, anti-CCR5-156Gd, anti-CD117-158Gd, anti-CD24-159Tb, anti-CD14-160Gd, anti-CD56-161Dy, anti-gdTCR-162Dy, anti-DRTH2-163Dy, anti-CLEC12A-164Dy, anti-CCR6-165Ho, anti-CD25-166Er, anti-CCR7-167Er, anti-CD3-168Er, anti-CX3CR1-169Tm, anti-CD38-170Er, anti-CD161-171Yb, anti-CD209-172Yb, anti-CXCR3-173Yb, anti-HLA-DR-174Yb, anti-CCR4-176Yb, anti-CD11b-209Bi.</div><div id="p0265" role="paragraph">For the identification of immunophenotypes unique to the <i>TBX21</i> patient, we compared data from six batches of CyTOF experiments including age-matched pediatric controls. In addition to manually inspecting data from each batch separately, we also compared immunophenotypes across batches in an objective and unbiased manner, with a computerized system (<span class="dropBlock reference-citations"><a id="body-ref-sref183-1" href="#" role="doc-biblioref" data-xml-rid="bib183" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref183-1" aria-controls="sref183-1">Ogishi, 2021</a><div class="dropBlock__holder" data-db-target-of="sref183-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Ogishi, M.</div><div><strong>Multibatch Cytometry Data Integration for Optimal Immunophenotyping.</strong></div><div><em>J. Immunol.</em> 2021; <strong>206</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.2000854" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85097887715" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33229441/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.2000854&amp;pmid=33229441" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). This method involved a combination of high-dimensional batch-correction, unsupervised clustering, and batch-specific cell-type classification through machine learning. We first used a computational approach to normalize batch effects and identify phenotypes unique to the <i>TBX21</i> patient through an unbiased clustering approach (<span class="dropBlock reference-citations"><a id="body-ref-sref183-2" href="#" role="doc-biblioref" data-xml-rid="bib183" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref183-2" aria-controls="sref183-2">Ogishi, 2021</a><div class="dropBlock__holder" data-db-target-of="sref183-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">126.</div><div>Ogishi, M.</div><div><strong>Multibatch Cytometry Data Integration for Optimal Immunophenotyping.</strong></div><div><em>J. Immunol.</em> 2021; <strong>206</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.2000854" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85097887715" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33229441/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.2000854&amp;pmid=33229441" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Each of the FCS files was processed to exclude dead cells and doublets automatically. Cells from healthy local controls were used for batch correction with <i>Harmony</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref89-2" href="#" role="doc-biblioref" data-xml-rid="bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref89-2" aria-controls="sref89-2">Korsunsky et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref89-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Korsunsky, I. ∙ Millard, N. ∙ Fan, J. ...</div><div><strong>Fast, sensitive and accurate integration of single-cell data with Harmony</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1289-1296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0619-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85070698712" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0619-0&amp;pmid=31740819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Data were down-sampled to 200,000 cells per batch before batch correction. Unsupervised clustering was then performed with batch-corrected marker expression values and the <i>FlowSOM</i> and <i>ConsensusClusterPlus</i> packages in <i>Bioconductor</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref162-3" href="#" role="doc-biblioref" data-xml-rid="bib162" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref162-3" aria-controls="sref162-3">Van Gassen et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref162-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">165.</div><div>Van Gassen, S. ∙ Callebaut, B. ∙ Van Helden, M.J. ...</div><div><strong>FlowSOM: Using self-organizing maps for visualization and interpretation of cytometry data</strong></div><div><em>Cytometry A.</em> 2015; <strong>87</strong>:636-645</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.22625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84932198501" target="_blank" aria-label="Open Scopus in new tab">Scopus (278)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25573116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.22625&amp;pmid=25573116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <span class="dropBlock reference-citations"><a id="body-ref-sref168-2" href="#" role="doc-biblioref" data-xml-rid="bib168" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref168-2" aria-controls="sref168-2">Wilkerson and Hayes, 2010</a><div class="dropBlock__holder" data-db-target-of="sref168-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">171.</div><div>Wilkerson, M.D. ∙ Hayes, D.N.</div><div><strong>ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:1572-1573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77954182362" target="_blank" aria-label="Open Scopus in new tab">Scopus (544)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20427518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq170&amp;pmid=20427518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We retained 40 clusters after metaclustering. Each cluster was manually inspected and annotated on the basis of its marker expression profile. A batch-specific cell-type classifier was then trained on the uncorrected expression values and cell-type annotations. We sampled a maximum of 100 cells per cell type at random, to improve computation efficiency and to alleviate class imbalance issues, as some of the cell types are much rarer than others. A highly randomized tree algorithm was used, with tuning of the hyperparameters <i>mtry</i> and <i>numRandomCuts</i> for each batch (<span class="dropBlock reference-citations"><a id="body-ref-sref54-2" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54-2" aria-controls="sref54-2">Geurts et&nbsp;al., 2006</a><div class="dropBlock__holder" data-db-target-of="sref54-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Geurts, P. ∙ Ernst, D. ∙ Wehenkel, L.</div><div><strong>Extremely randomized trees</strong></div><div><em>Mach. Learn.</em> 2006; <strong>63</strong>:3-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10994-006-6226-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33646430006" target="_blank" aria-label="Open Scopus in new tab">Scopus (1993)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10994-006-6226-1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Classifiers were trained through five repeats of 10-fold cross-validation with internal upsampling. Internal accuracy during cross-validation always exceeded ∼95%. Finally, the trained classifier was applied to the rest of the cells, including the patient’s cells, to predict their cell types. With this approach, any cell could be classified probabilistically as belonging to one of the predefined cell types, making it possible to avoid a strong skewing of patient data, which would affect the batch-correction and clustering steps. For the purposes of visualization, batch correction was performed with Harmony on cells from all controls and patients. Data were down-sampled to 40,000 cells per batch before batch correction. Finally, the trained classifier was applied to all cells, including the patient’s cells, to predict their cell types and, thus, to identify immunophenotypes for each individual. After harmonization, dimension reduction was performed by Uniform Manifold Approximation and Projection (UMAP) (<span class="dropBlock reference-citations"><a id="body-ref-sref10-3" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-3" aria-controls="sref10-3">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref10-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (449)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The figures are color-coded on the basis of the cell type identified by the unsupervised clustering approach described above. All analyses were performed with R ver.3.6.3.</div><div id="p0270" role="paragraph">We also deconvoluted 10,000 memory CD4<sup>+</sup> T&nbsp;cells (DNA<sup>+</sup>Live_CD45<sup>+</sup>CD66b<sup>-</sup>CD123<sup>-</sup>CD3<sup>+</sup>CD20<sup>-</sup>CD56<sup>-</sup>γδTCR<sup>-</sup>CD4<sup>+</sup>CD8<sup>-</sup>CD45RA<sup>-</sup>) into viSNE plots based on 20 surface markers (113ln_CD57, 115ln_CD11c, 149sm_CD127, 150Nd_CD1c, 154Sm_CD86, 155Gd_CD27, 156Gd_CCR5, 158Gd_CD117, 159Tb_CD24, 163Dy_CRTH2, 165Ho_CCR6, 166Er_CD25, 167Er_CCR7, 169Tm_CX3CR1, 170Er_CD38, 171Yb_CD161, 173Yb_CXCR3, 174Yb_HLA-DR, 176Yb_CCR4, 209Bi_CD11b) (Amir el et&nbsp;al., 2013). Analyses were performed by manually gating populations for NK cells, naive and memory subsets of CD4<sup>+</sup> and CD8<sup>+</sup> T&nbsp;cells, innate lymphoid cells (ILCs), Tregs and dendritic cell subsets. For T helper cell analysis, we included an additional four healthy donors from a separate experiment (same staining panel) in the analysis. Manual gating was used for T<sub>H</sub> analysis. We also deconvoluted 10,000 CD8<sup>+</sup> T&nbsp;cells into viSNE plots based on 113ln_CD57, 115ln_CD11c, 143Nd-CD45RA, 144Nd-CD141, 149sm_CD127, 150Nd_CD1c, 154Sm_CD86, 155Gd_CD27, 156Gd_CCR5, 158Gd_CD117, 159Tb_CD24, 163Dy_CRTH2, 165Ho_CCR6, 166Er_CD25, 167Er_CCR7, 169Tm_CX3CR1, 170Er_CD38, 171Yb_CD161, 173Yb_CXCR3, 174Yb_HLA-DR, 176Yb_CCR4, 209Bi_CD11b (Amir el et&nbsp;al., 2013).</div><div id="p0275" role="paragraph">Immunophenotyping for MAIT, invariant NKT cells, Vδ2 and Vδ1 cells was performed in three different batches. In the first batch, PBMCs from 18 healthy controls (CTL), three IL-12Rβ1-deficient individuals (<i>IL12RB1</i> M/M), two T-bet-heterozygous parents (<i>TBX21</i> WT/M), P (<i>TBX21</i> M/M) and P’s healthy brother (<i>TBX21</i> WT/WT) were included. They were stained with the following panel: FcBlock (Miltenyi Biotec), Zombie-NIR live-dead exclusion dye (BioLegend), anti-CD3-Alexa532 (Thermo Fisher Scientific), anti-γδTCR-FITC (Thermo Fisher Scientific), anti-Vδ2-APC-Fire750 (BioLegend), anti-CD56-BV605 (BioLegend), anti-CD4-BV750 (BD Biosciences), anti-CD8a-BV510 (BioLegend), anti-Vα7.2-BV711 (BioLegend), anti-Vα24-Jα18-PE-Cy7 (BioLegend), anti-Vδ1-Vioblue (Miltenyi Biotec), anti-CD161-PE (BioLegend) and anti-Vβ11-APC (Miltenyi Biotec) antibodies. Cells were analyzed with an Aurora cytometer (Cytek). In the 2<sup>nd</sup> batch, 31 adult control samples, seven samples from age-matched controls (age CTL), including a Moroccan child, and samples from P (M/M) and his mother sample were included. In the 3<sup>rd</sup> batch, 16 adult control samples, five samples from age-matched controls (age CTL) and a sample from P were included. In the 3<sup>rd</sup> batch, 16 adult control samples, five samples from age-matched controls (age CTL) and P’s sample were included. For the three batches considered together, we obtained 65 data points for healthy adults, with PBMCs from 20 healthy adults undergoing two analyses in different batches, PBMCs from three samples undergoing analyses three times, in different batches and PBMCs from 16 individuals analyzed only once. All 12 age-matched controls were different individuals. Three vials of PBMCs from P, isolated from fresh blood and frozen in the same batch, were analyzed in three separate batches. P’s parents and P’s brother’s PBMCs were isolated from fresh blood and frozen. The age matched controls were of various ethnic backgrounds, and all were aged 1 – 17 years at the time of blood sampling. The PBMCs of most of these age-matched controls were prepared from blood that had been transported overnight. PBMCs from three IL-12Rβ1-deficient individuals were prepared from blood transported overnight. In the second and third batches, immunophenotyping for MAIT, invariant NKT cells, Vδ2 and Vδ1 cells was performed with a flow cytometry staining panel containing FcBlock (Miltenyi Biotec), Zombie-NIR live-dead exclusion dye (BioLegend), anti-γδTCR-alexa 647 (BioLegend), anti-CD3-V450 (BD Biosciences), anti-CD56-BV605 (BioLegend), anti-CD4-BUV563 (BD Biosciences), anti-Vδ1TCR-FITC (Miltenyi), anti-CD8-BUV737 (BD Biosciences), anti-Vδ2TCR-APC/Fire750 (BioLegend), anti-CD20-BV785 (BioLegend), anti-Vα7.2-BV711 (BioLegend), anti-CD161-PE (BioLegend), MR1-5-OP-RU-tetramer (NIH tetramer core facility), anti-Vβ11-APC (Miltenyi), anti-iNKT-BV480 (BD Biosciences) antibodies. Cells were analyzed with an Aurora cytometer (Cytek). Three batches of data were compiled for the final analysis. The percentages of the various immune subsets were determined by manual gating.</div><div id="p0280" role="paragraph">Immunophenotyping for innate lymphoid cells and NK cells was confirmed separately, by conventional flow cytometry. In brief, biotinylated anti-human anti-CD1a (HI149), anti-CD3 (OKT3), anti-CD5 (UCHT2), anti-CD14 (61D3), anti-CD19 (HIB19), anti-CD34 (4H11), anti-CD123 (6H6), anti-CD203c (FR3-16A11), anti-CD303 (AC144), anti-TCRαβ (IP26), anti-TCRγδ (B1) and anti-FcεRIα (AER-37(CRA-1)) antibodies were used in combination with streptavidin BUV661, and conjugated anti-CD62L-FITC (Dreg-56), anti-CD94-PerCP-Vio700 (REA113), anti-TCRVα24Jα18-PE (6B11), anti-CD26-PE-CF594 (M-A261), anti-CD127-PE-Cy7 (eBioRDR5), anti-CRTH2-Alexa Fluor 647 (BM16), anti-CD161-Alexa Fluor 700 (HP-3G10), anti-EOMES-APC-eFluor 780 (WD1928), anti-NKp46-BV421 (9E2/NKp46), anti-CD45RA-BV570 (HI100), anti-CD117-BV605 (104D2), anti-RORγt-BV650 (Q21-559), anti-CD7-BV711(M-T701), anti-CD16-BUV496 (3G8), anti-CD25-BUV563 (2A3), anti-CD56-BUV737 (NCAM16.2) and anti-CD45-BUV805 (HI30); anti-T-bet-BV786 (O4-46), anti-GATA3-BUV395 (L50-823) antibodies. Antibodies were purchased from BioLegend, Thermo Fisher Scientific, BD Biosciences, or Miltenyi Biotec. Fc receptors were blocked with IgG from human serum (Sigma-Aldrich). Surface membrane staining was performed in Brilliant Stain Buffer (BD Biosciences). Transcription factors were stained with the Foxp3 staining kit (Thermo Fisher Scientific), according to the manufacturer’s instructions. The fixable viability dye eFluor 506 (Thermo Fisher Scientific) was used to exclude dead cells. Samples were acquired on a BD Symphony A5 cytometer (BD Biosciences) with FACSDiva 8 software and were analyzed with FlowJo 10 (BD Biosciences).</div><div id="p0285" role="paragraph">Immunophenotyping was confirmed separately for the T- and B cell subsets by conventional flow cytometry. In brief, PBMCs from healthy controls and P were labeled with mAbs against CD3, CD4, CD8, CD45RA, CCR7, CD127, CD25, CXCR5, CXCR3, CCR6, PD-1, and CD56. The CD4<sup>+</sup> and CD8<sup>+</sup> T&nbsp;cell subsets were separated into naive (T<sub>Naive</sub>: CCR7<sup>+</sup>CD45RA<sup>+</sup>), central memory (T<sub>CM</sub>: CCR7<sup>+</sup>CD45RA<sup>−</sup>), effector memory (T<sub>EM</sub>: CCR7<sup>−</sup>CD45RA<sup>−</sup>), and terminally differentiated effector memory cells (T<sub>EMRA</sub>: CCR7<sup>−</sup>CD45RA<sup>+</sup>) after initial gating on CD8<sup>−</sup>CD4<sup>+</sup> or CD8<sup>+</sup>CD4<sup>−</sup> cells. Proportions of regulatory T&nbsp;cells (CD4<sup>+</sup>CD127<sup>lo</sup>CD25<sup>hi</sup>), total memory (CD4<sup>+</sup>CD45RA<sup>−</sup>), and circulating Tfh (cTfh; CD4<sup>+</sup>CD45RA<sup>−</sup>CXCR5<sup>+</sup>) cells, and subsets of total memory and cTfh cells defined on the basis of CXCR3 and CCR6 expression (T<sub>H</sub>1: CCR6<sup>−</sup>CXCR3<sup>+</sup>, T<sub>H</sub>17: CCR6<sup>+</sup>CXCR3<sup>−</sup>, T<sub>H</sub>1<sup>∗</sup>/T<sub>H</sub>1–17: CCR6<sup>+</sup>CXCR3<sup>+</sup>, T<sub>H</sub>2: CCR6<sup>−</sup>CXCR3<sup>−</sup>) were also determined.</div></section><section id="sec-4-4-16"><h4>Single-cell (sc) RNA-seq and analysis</h4><div id="p0290" role="paragraph">We performed scRNA-seq on PBMCs obtained from the <i>TBX21</i> WT/M heterozygous father and the <i>TBX21</i> M/M patient. PBMCs were quickly thawed at 37°C and gently resuspended by serial additions of DMEM&nbsp;+ 10% heat-inactivated (HI) FBS, according to the recommendations from the 10x Genomics “Sample preparation Fresh Frozen Human PBMC” protocol Rev C. Cells were centrifuged at 400 x <i>g</i> for 5&nbsp;min, washed with DMEM&nbsp;+ 10% HI-FBS, and cells were then counted and viability was assessed with the LIVE/DEA Viability kit (Thermo Fisher Scientific) according to the manufacturer’s guidelines. Following centrifugation at&nbsp;400 x <i>g</i> for 5&nbsp;min, cells were resuspended at a concentration of 1000 cells/μl in PBS&nbsp;+ 0.04% BSA and loaded onto a 10&nbsp;x&nbsp;Genomics Chromium chip for single-cell capture. Reverse transcription and library preparation were performed with Chromium Single Cell 3′ Reagent Kits (v2) in accordance with the manufacturer’s guidelines, and library quality was assessed with a Bioanalyzer DNA chip. Each library was sequenced on two lanes of an Illumina HiSeq 4000 sequencer in a 28&nbsp;bp/98&nbsp;bp paired-end configuration.</div><div id="p0295" role="paragraph">The sequence-read quality of individual sequencing lanes was assessed with BVAtools (<a id="intref0880" href="https://bitbucket.org/mugqic/bvatools" target="_blank">https://bitbucket.org/mugqic/bvatools</a>). Cell Ranger v3.0.1 was used to aggregate the sequences from two lanes/samples, to map reads to the hg38 human reference genome assembly, for filtering, and counting barcodes and UMIs; a list of UMI counts was thus obtained for each gene and each cell. Cells outside the [5%,95%] interval for these metrics or with &gt; 10% mitochondrial genes were excluded. The DoubletFinder package was used to identify and filter out cell doublets (<span class="dropBlock reference-citations"><a id="body-ref-sref116-2" href="#" role="doc-biblioref" data-xml-rid="bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116-2" aria-controls="sref116-2">McGinnis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref116-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">118.</div><div>McGinnis, C.S. ∙ Murrow, L.M. ∙ Gartner, Z.J.</div><div><strong>DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors</strong></div><div><em>Cell Syst.</em> 2019; <strong>8</strong>:329-337</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85064396876" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.03.003&amp;pmid=30954475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). After the removal of dead cells and doublets, 10,409 cells for the T-bet wt/m heterozygous father and 11,741 cells for the T-bet m/m P were analyzed together with the Seurat v3&nbsp;R package (<span class="dropBlock reference-citations"><a id="body-ref-sref145-2" href="#" role="doc-biblioref" data-xml-rid="bib145" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref145-2" aria-controls="sref145-2">Stuart et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref145-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">148.</div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (766)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and cell clustering was performed by the Uniform Manifold Approximation and Projection dimension reduction method (<span class="dropBlock reference-citations"><a id="body-ref-sref10-4" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10-4" aria-controls="sref10-4">Becht et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref10-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (449)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), using the most variable genes, but excluding mitochondrial and ribosomal protein genes. This analysis identified eight distinct cell clusters; the marker genes for each cluster were identified with Seurat and used to determine cell types. Differential expression was analyzed by comparing UMI gene counts between the patient and the control for each cluster, with the MAST approach (<span class="dropBlock reference-citations"><a id="body-ref-sref47-2" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47-2" aria-controls="sref47-2">Finak et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref47-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Finak, G. ∙ McDavid, A. ∙ Yajima, M. ...</div><div><strong>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0844-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84951574149" target="_blank" aria-label="Open Scopus in new tab">Scopus (404)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0844-5&amp;pmid=26653891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Genes with a <i>p-value</i>&nbsp;&lt; 10<sup>−25</sup> for differential expression were analyzed in more detail. For each selected gene, cells in which the expression of the gene was detectable were counted for the index case and the control in each cluster, to reduce the dropout bias inherent to droplet scRNA approaches. The number of cells expressing the gene was normalized according to the number of cells in each cluster and the fold-difference in expression was calculated by comparing the index case and the control. For the figures, cluster annotations and the levels of expression of genes of interest were projected onto the UMAP clustering, but with the cells of the patient and the control shown separately.</div></section><section id="sec-4-4-17"><h4>Stimulation of NK cells <i>ex&nbsp;vivo</i></h4><div id="p0300" role="paragraph">PBMCs from five healthy donors including a travel control, the T-bet WT/M mother, the T-bet WT/WT brother and the T-bet M/M P were left untreated or stimulated with IL-12 (Miltenyi Biotec), IL-15 (Miltenyi Biotec) and IL-18 (MBL/CliniSciences) for 18 h. Cells were treated with GolgiPlug and GolgiStop (BD Biosciences), and stained with anti-CD107a-PE antibody (eBioH4A4, Thermo Fisher Scientific) for 4 h. They were then stained with eFluor506 live-dead reagents (Thermo Fisher Scientific), followed by antibodies binding to surface antigens, including anti-CD1a-FITC (H1149), anti-CD3-FITC (OKT3), anti-CD5-FITC (UCHT2), anti-CD14-FITC (TUK4), anti-CD19-FITC (HIB19), anti-CD34-FITC (581), anti-CD123-FITC (6H6), anti-CD203c (NP4D6), anti-CD303-FITC (201A), anti-FcεRIα-FITC (AER-37(CRA-1)), anti-TCRɑβ-FITC (IP26), anti-TCRγδ-FITC (B1), anti-CD94-PerCP-Vio700 (REA113), anti-CD127-PE-Cy7 (eBioRDR5), anti-NKp46-APC (9E2/NKp46), anti-CD7-BV711 (M-T701), anti-CD56-BV786 (NCAM16.2), anti-CD16-BUV737 (3G8), and anti-CD45-BUV805 (HI30) antibodies. All the markers targeted by FITC-conjugated Abs were considered to be Lineage-specific markers (Lin). Cells were fixed and permeabilized (Fix &amp; Perm; Thermo Fisher Scientific) and subjected to intracellular staining with anti-granzyme B-PE-CF594 (GB11), anti-EOMES-APC-eFluor 780 (WD1928), anti-perforin-BV421 (δG9), anti-TNF-α-BV605 (Mab11) and anti-IFN-γ-BUV395 (B27) antibodies. Antibodies were purchased from BioLegend, Thermo Fisher Scientific, BD Biosciences, or Miltenyi Biotec. We analyzed ICS and surface staining with a BD LSRFortessa flow cytometer (BD Biosciences).</div></section><section id="sec-4-4-18"><h4>CD4<sup>+</sup> T&nbsp;cell isolation and functional characterization <i>ex&nbsp;vivo</i> and T<sub>H</sub> differentiation <i>in&nbsp;vitro</i></h4><div id="p0305" role="paragraph">The healthy controls (CTL) for this experiment were adults recruited locally in Australia, and PBMCs for P (M/M) were prepared from two separate overnight shipments of blood from Casablanca to Paris and frozen. The two dots in these figure panels represent data from two independent experiments on two batches. PBMCs from healthy donors and P were incubated with mAbs against CD4, CD45RA, CCR7, CXCR5, CD127 and CD25. Naive and memory CD4<sup>+</sup> T&nbsp;cells were isolated by first excluding Tregs (CD25<sup>hi</sup>CD127<sup>lo</sup>) and then sorting CD4<sup>+</sup>CD45RA<sup>+</sup>CCR7<sup>+</sup>CXCR5<sup>−</sup> and CD4<sup>+</sup>CD45RA<sup>−</sup>CCR7<sup>±</sup> cells, respectively. Isolated naive and memory CD4<sup>+</sup> T&nbsp;cells were then cultured in 96-well round-bottomed well plates (30-40 x10<sup>3</sup> cells/well) with T&nbsp;cell activation and expansion (TAE) beads (anti-CD2/CD3/CD28; Miltenyi Biotech) alone (T<sub>H</sub>0) or under T<sub>H</sub>1 (50&nbsp;ng/mL IL-12), T<sub>H</sub>2 (100&nbsp;U/mL, IL-4), T<sub>H</sub>9 (2.5&nbsp;ng/mL TGF-β, 100&nbsp;U/mL IL-4), or T<sub>H</sub>17 (2.5&nbsp;ng/mL TGF-β, 50&nbsp;ng/mL IL-1-β, 50&nbsp;ng/mL IL-6, 50&nbsp;ng/mL IL-21, 50&nbsp;ng/mL IL-23, 50&nbsp;ng/mL PGE2) polarizing conditions. After 5&nbsp;days, supernatants were harvested and the production of IL-4, IL-5, IL-9, IL-10, IL-13, IL-17A, IL-17F, IFN-γ and TNF-β was assessed with cytometric bead arrays (Becton Dickinson); IL-22 secretion was measured by ELISA (eBioscience). For cytokine expression, activated CD4<sup>+</sup> T&nbsp;cells were restimulated with PMA (100&nbsp;ng/mL)/ionomycin (750&nbsp;ng/mL) for 6 h, with brefeldin A (10&nbsp;μg/mL) added after two hours. Cells were then fixed and the intracellular expression of IL-4, IL-9, IL-13, IL-10, IL-17A, IL-17F, IL-22, IL-21 and IFN-γ was determined.</div></section><section id="sec-4-4-19"><h4>Stimulation of PBMCs with PMA/ionomycin for intracellular staining and flow cytometry analysis</h4><div id="p0310" role="paragraph">This experiment was performed twice, on two different dates. The two batches of data were compiled for the final analysis. Fourteen healthy adult donors were recruited at the Rockefeller University (New York, USA). Two healthy adult controls, four age-matched controls, P and his relatives were recruited at Necker Hospital (Paris, France). PBMCs from two of 16 adult controls were frozen after being allowed to rest at room temperature for 24 – 48&nbsp;h to mimic transport conditions. The PBMCs from the other 14 adult controls were prepared directly from freshly collected blood and frozen. Two vials of PBMCs from P were used in this experiment: one prepared directly from fresh blood before freezing, and the other prepared and frozen after overnight shipment of the blood samples. The PBMCs of four age-matched controls were prepared from blood that had been shipped overnight. The PBMCs of P’s relatives were prepared from blood that had been shipped overnight. PBMCs from six healthy adult controls were stimulated twice, in two separate batches, as technical replicates. The second vial of P’s PBMCs was stimulated in two sets of wells, as technical duplicates. The PBMCs of three age-matched controls were stimulated in two sets of wells, as technical duplicates. The PBMCs of P’s WT/M mother were stimulated in two sets of wells, as technical duplicates.</div><div id="p0315" role="paragraph">In both batches of experiments, we plated 300,000 cells per well in 96-well U-bottomed plates, at a density of 1.5&nbsp;× 10<sup>6</sup> cells/ml. Cells were cultured for 40&nbsp;h in RPMI. They were then stimulated with 50&nbsp;ng/mL PMA and 1&nbsp;μM ionomycin. After 1 h, Golgiplug was added to the culture, in accordance with the manufacturer’s protocol. After another 7 h, cells were harvested for staining and flow cytometry. In brief, cells were first stained with Zombie NIR Viability kit (BioLegend) for 15&nbsp;min. They were then stained with FcBlock (Miltenyi), anti-γδTCR-alexa 647 (BioLegend), anti-CD3-V450 (BD Biosciences), anti-CD56-BV605 (BioLegend), anti-CD4-BUV563 (BD Biosciences), anti-Vδ1TCR-FITC (Miltenyi), anti-CD8-BUV737 (BD Biosciences), anti-Vδ2TCR-APC/Fire750 (BioLegend), anti-CD20-BV785 (BioLegend), anti-Vα7.2-Alexa 700 (BioLegend), MR1-5-OP-RU-tetramer (NIH tetramer core facility), anti-Vβ11-APC (Miltenyi), and anti-iNKT-BV480 (BD Biosciences) antibodies for 30&nbsp;min. The stained cells were fixed with the FOXP3/Transcription Factor kit (Thermo Fisher Scientific) and stained intracellularly with anti-T-bet-PE/Cy7 (BioLegend), anti-IFN-γ-BV711 (BioLegend), anti-TNF-α-BV510 (BioLegend), anti-IL-17A-PERCP/Cy5.5, anti-RORγT-PE (BD Biosciences), anti-CD3-V450 (BD Biosciences), anti-CD4-BUV563 (BD Biosciences) and anti-CD8-BUV737 (BD Biosciences) antibodies in Perm/Wash buffer. Cells were then analyzed in a CyTek Aurora spectral flow cytometer.</div></section><section id="sec-4-4-20"><h4>Stimulation of PBMCs with PMA/ionomycin for cytokine secretion</h4><div id="p0320" role="paragraph">This experiment was done in three batches, on three separate dates. The data from the three batches were compiled for the final analysis. Thirty healthy adult controls were recruited at the Rockefeller University (New York, USA) and one healthy adult control was recruited at Necker Hospital (Paris, France). PBMCs from eight of the 31 adult controls were frozen after being allowed to rest at room temperature for 24 – 48&nbsp;h to mimic travel conditions. The PBMCs of all the other 23 adult controls were isolated from fresh blood and frozen. PBMCs from 18 adult controls were analyzed once. Those from 13 adult controls were analyzed twice, in two separate batches. Three vials of PBMCs from P were used in this experiment: two vials of PBMCs isolated from fresh blood and then frozen, and a vial of PBMCs isolated from blood that had been transported overnight, which were then frozen. These three vials of PBMCs from P were used separately, in three batches. One vial of PBMCs prepared from fresh blood was plated in two sets of wells as technical duplicates. Thus, four data points were measured for P. The PBMCs of P’s relatives were prepared from blood that had been shipped overnight. PBMCs from P’s healthy brother, mother and father were each analyzed once. Five age-matched controls were recruited from around the world, including one matched for both age and ethnicity from Morocco. PBMCs were isolated from the blood of these patients after overnight shipment and frozen. PBMCs from three age-matched controls were plated in two sets of wells, as technical duplicates. The PBMCs of the age- and ethnicity-matched control were plated in technical triplicates. In three batches of experiments, we plated 300,000 cells per well in 96-well U-bottomed plates, at a density of 1.5&nbsp;× 10<sup>6</sup> cells/mL. Cells were cultured for 40&nbsp;h in RPMI. They were then stimulated by incubation with 50&nbsp;ng/mL PMA and 1&nbsp;μM ionomycin for 8 h. Supernatants were harvested for human T<sub>H</sub> cytokine analysis with the 13-plex LegendPlex kit (BioLegend).</div></section><section id="sec-4-4-21"><h4>Stimulation of PBMCs with live BCG infection for cytokine production</h4><div id="p0325" role="paragraph">This experiment was performed in three batches, on three dates. The data from the three batches were compiled for the final analysis. Thirty healthy adult controls were recruited at the Rockefeller University (New York, USA) and one healthy adult control was recruited at Necker Hospital (Paris, France). PBMCs from eight of the 31 adult controls were frozen after being allowed to rest at room temperature for 24 – 48&nbsp;h to mimic travel conditions. The PBMCs of all the other 23 adult controls were isolated from fresh blood and frozen. PBMCs from 19 adult controls were analyzed once, and those from 10 adult controls were assayed twice, in two separate batches. PBMCs from one control were plated in three sets, as technical triplicates, in one batch. PBMCs from one control were assayed five times in three separate batches, as technical replicates. In total, 47 data points were obtained for adult controls. Five age-matched controls were recruited globally, including one matched for both age and ethnicity from Morocco. Their PBMCs were obtained from blood that had been shipped overnight and were frozen. PBMCs from three age-matched controls were plated in two sets of wells, as technical duplicates. PBMCs from the other age-matched controls were plated in single sets. Eight data points in total were obtained for age-matched controls. Three vials of PBMCs from P were used in this experiment: two vials of PBMCs isolated from fresh blood and then frozen, and a vial of PBMCs isolated from blood that had been transported overnight, which were then frozen. These three vials of PBMCs from P were used separately, in three batches. In total three data points were generated for P. PBMCs from P’s relatives were isolated from blood that had been shipped overnight. PBMCs from P’s healthy brother, mother and father were each analyzed once each.</div><div id="p0330" role="paragraph">In all three batches of experiments, we plated 300,000 cells per well in 96-well U-bottomed plates, at a density of 1.5&nbsp;× 10<sup>6</sup> cells/ml. For each sample, cells were plated in the presence and absence of live <i>M.&nbsp;bovis-</i>BCG at a MOI&nbsp;= 1 (gift from Carl Nathan), and in the presence and absence of recombinant IL-12 (5&nbsp;ng/mL, R&amp;D) or recombinant IL-23 (10&nbsp;ng/mL, R&amp;D) (<span class="dropBlock reference-citations"><a id="body-ref-sref165-2" href="#" role="doc-biblioref" data-xml-rid="bib165" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref165-2" aria-controls="sref165-2">Vogt and Nathan, 2011</a><div class="dropBlock__holder" data-db-target-of="sref165-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">168.</div><div>Vogt, G. ∙ Nathan, C.</div><div><strong>In&nbsp;vitro differentiation of human macrophages with enhanced antimycobacterial activity</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2011; <strong>121</strong>:3889-3901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI57235" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-80053413551" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21911939/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI57235&amp;pmid=21911939" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Golgiplug (BD Biosciences) was added to each well after 40&nbsp;h of stimulation. Eight hours later, supernatants were harvested for cytokine determinations in a 13-plex Legendplex assay (BioLegend), and cells were collected by centrifugation for FACS staining. In brief, cells were first stained with the Zombie NIR Viability kit (BioLegend) for 15&nbsp;minutes. They were then stained with FcBlock (Miltenyi Biotec), anti-γδTCR-alexa 647 (BioLegend), anti-CD3-V450 (BD Biosciences), anti-CD56-BV605 (BioLegend), anti-CD4-BUV563 (BD Biosciences), anti-Vδ1TCR-FITC (Miltenyi Biotec), anti-CD8-BUV737 (BD Biosciences), anti-Vδ2TCR-APC/Fire750 (BioLegend), anti-CD20-BV785 (BioLegend), anti-Vα7.2-Alexa 700 (BioLegend), MR1-5-OP-RU-tetramer (NIH tetramer core facility), anti-Vβ11-APC (Miltenyi Biotec), and anti-iNKT-BV480 (BD Biosciences) antibodies for 30&nbsp;minutes. The stained cells were fixed with the FOXP3/Transcription Factor Buffer set (Thermo Fisher Scientific) and intracellularly stained with anti-T-bet-PE/Cy7 (BioLegend), anti-IFN-γ-BV711 (BioLegend), anti-TNF-α-BV510 (BioLegend), anti-IL-17A-PERCP/Cy5.5 and anti-RORγT-PE (BD Biosciences) antibodies in Perm/Wash buffer. Cells were acquired on a CyTek Aurora spectral flow cytometer and the data were analyzed with Cytobank. T-bet<sup>+</sup> IFN-γ<sup>+</sup> double-positive cells were gated manually from the live single-lymphocyte population. The viSNE algorithm was then used to cluster these T-bet<sup>+</sup> IFN-γ<sup>+</sup> double-positive cells with surface lineage-determining markers (Amir el et&nbsp;al., 2013). Automated viSNE clusters were plotted with tSNE1 and tSNE2. Clusters were then overlaid on manually gated immune cell subsets for illustration.</div></section><section id="sec-4-4-22"><h4>Vδ2<sup>+</sup> γδ T&nbsp;cell stimulation</h4><div id="p0335" role="paragraph">Lysates of <i>M.&nbsp;bovis-</i>BCG were prepared and provided by Dr. Carl F. Nathan. PBMCs from adult controls, IL-12Rβ1 deficient patients, the T-bet M/M P and his relatives were resuspended at a density of 1x10<sup>6</sup> cells/mL. We plated 200&nbsp;μL cells per well for each condition, in a 96-well U-bottomed plate. We added BCG lysate to each well at a final concentration of 5&nbsp;μg/mL. Recombinant IL-2 (Roche) was added to a final concentration of 10&nbsp;U/mL. Fresh medium and IL-2 were added to the culture every three to four days. Eleven days after stimulation, the cells were fully confluent in the 96-well plate. Cells were then transferred to a 48-well plate and each well was topped up to500&nbsp;μL per well with medium. After 14&nbsp;days of stimulation, the cells were stained with FcBlock (Miltenyi Biotec), Zombie-NIR live/dead (BioLegend), anti-CD3-Alexa532 (Thermo Fisher Scientific), anti-γδTCR-FITC (Thermo Fisher Scientific), anti-Vδ2 TCR-APCFire750 (BioLegend), anti-CD56-BV605 (BioLegend), anti-CD4-BV750 (BD Biosciences), anti-CD8a-BV510 (BioLegend), anti-β11TCR-APC (Miltenyi Biotec), anti-CD161-PE (BioLegend), anti- Vα7.2 TCR-BV711 (BioLegend), anti-Vα24-Jα18 (iNKT)-PE/Cy7 (BioLegend), anti-Vδ1-Vioblue (Miltenyi Biotec), anti-CD16-BV650 (BioLegend), anti-CD20-BV785 (BioLegend), anti-αβTCR-Alexa700 (BioLegend) antibodies and analyzed by flow cytometry on an Aurora cytometer (Cytek). Supernatants were harvested for human T<sub>H</sub> cytokine analysis with the13-plex LegendPlex kit (BioLegend).</div></section><section id="sec-4-4-23"><h4>Stimulation of αβ T&nbsp;cells with PPD and BCG-lysate</h4><div id="p0340" role="paragraph">PBMCs from four BCG-vaccinated healthy controls and P were stained with the CellTrace Far-Red Cell Proliferation Kit (Thermo Fisher Scientific) at a dilution of 1:25,000. Cells were washed and resuspended at a density of 2&nbsp;× 10<sup>6</sup> cells/mL in RPMI 10% human AB serum (Gemini). Cells were plated in 96-well U-bottomed plates and stimulated with BCG-lysate at a final concentration of 5&nbsp;μg/mL or with tuberculin purified protein derivative (PPD) at a final concentration of 5&nbsp;μg/mL (STATEN SERUM INSTITUT). Cells were cultured for 6&nbsp;days and then harvested and stained with the Zombie NIR Viability kit (BioLegend) for 15&nbsp;minutes. Cells were then surface-stained with FcBlock, anti-CD3-V450 (BD Biosciences), anti-CD4-BUV563 (BD Biosciences), anti-CD8-BUV737 (BD Biosciences), anti-Vδ2TCR-APC/Fire750 (BioLegend), anti-CD56-BV605 (BioLegend) and anti-CD20-BV785 (BioLegend) antibodies for 30&nbsp;minutes. Cells were fixed with the FOXP3/Transcription Factor Buffer kit (Thermo Fisher Scientific) and stained by overnight incubation with anti-T-bet-PE/Cy7 (BioLegend), anti-IFN-γ-BV711 (BioLegend), anti-TNF-α-BV510 (BioLegend) and anti-human/mouse-Ki-67 (BioLegend) antibodies in Perm/Wash buffer. Data were acquired in a CyTek Aurora spectral flow cytometer.</div></section><section id="sec-4-4-24"><h4>Isolation and screening of CD4<sup>+</sup> T&nbsp;cell lines</h4><div id="p0345" role="paragraph">The T&nbsp;cell clone experiment was performed as previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Geiger et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Geiger, R. ∙ Duhen, T. ∙ Lanzavecchia, A. ...</div><div><strong>Human naive and memory CD4+ T&nbsp;cell repertoires specific for naturally processed antigens analyzed using libraries of amplified T&nbsp;cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2009; <strong>206</strong>:1525-1534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20090504" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-67650469829" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19564353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20090504&amp;pmid=19564353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In brief, CCR6<sup>+</sup> and CCR6<sup>–</sup> CD4<sup>+</sup> memory T&nbsp;cells were sorted from CD14<sup>–</sup> PBMCs with a FACS Aria (BD Biosciences), excluding CD45RA<sup>+</sup>CCR7<sup>+</sup> naive cells, and CD25<sup>+</sup>, CD19<sup>+</sup>, and CD8<sup>+</sup> cells. Sorted T&nbsp;cells (500 cells/well) were polyclonally stimulated with 1&nbsp;μg/mL PHA (Remel) in the presence of irradiated (45 Gy) allogeneic feeder cells (10<sup>4</sup> per well) and IL-2 (500 IU/mL) in RPMI complete medium. After expansion for at least 20&nbsp;days, the T&nbsp;cell lines were analyzed by surface or intracellular staining with the following monoclonal antibodies: anti-CCR6-PE (BD Biosciences; clone: 11A9); anti-CXCR3-APC (BD Biosciences; clone: 1C6); anti-IFN-γ-FITC or anti-IFN-γ-PE (BD Biosciences, clone: B27); anti-IL-17A-eFluor660 FITC (eBiosciences, clone: eBio64DEC17); anti-IL-4-PE (BD Biosciences, clone: MP4-25D2); anti-IL-22-PerCP-eFluor-710 (Thermo Fisher Scientific, clone: 22URTI). For identification of the antigen-responding T&nbsp;cell clones, the T&nbsp;cell lines were also screened by culturing thoroughly washed cells (2.5&nbsp;× 10<sup>5</sup> cells/well) with autologous irradiated B cells (2.5&nbsp;× 10<sup>4</sup> cells/well), with or without a three-hour pulse of the following antigens: <i>M.&nbsp;tuberculosis</i> peptide pool (0.5&nbsp;μg/mL/peptide, comprising 207 15-mer peptides), BCG peptide pool (0.5&nbsp;μg/mL/peptide, comprising 211 15-mer peptides), HCMV peptide pool (0.5&nbsp;μg/mL/peptide, comprising 76 15-20-mer peptides), EBV peptide pool (0.5&nbsp;μg/mL/peptide, comprising 46 15-20-mer peptides), Influenza virus HA peptide pool (2&nbsp;μg/mL/peptide, comprising 351 15-mer peptides), <i>Candida albicans</i> peptide pool (0.5&nbsp;μg mL/peptide, comprising 252 15-mer peptides) or tetanus toxoid peptide pool (1&nbsp;μg/mL/peptide, comprising 125 15-20-mer peptides). These peptide pools were kindly provided by Cecilia Lindestam Arlehamn and Alessandro Sette at the La Jolla Institute for Immunology (<span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Lindestam Arlehamn et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Lindestam Arlehamn, C.S. ∙ McKinney, D.M. ∙ Carpenter, C. ...</div><div><strong>A Quantitative Analysis of Complexity of Human Pathogen-Specific CD4&nbsp;T Cell Responses in Healthy M. tuberculosis Infected South Africans</strong></div><div><em>PLoS Pathog.</em> 2016; <strong>12</strong>:e1005760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1005760" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84982943706" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27409590/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1005760&amp;pmid=27409590" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Proliferation was assessed on day 4, after incubation for 16&nbsp;h with 1&nbsp;μCi/mL [<sup>3</sup>H]-thymidine (GE Healthcare). The concentrations of cytokines in the culture supernatants produced by the antigen-responding T&nbsp;cell clones were measured after 48&nbsp;h of stimulation with cytometric bead arrays (eBioscience).</div></section></section><section id="sec-4-5"><h3>Quantification and Statistical Analysis</h3><div id="p0350" role="paragraph">Statistical analysis was performed using PRISM software (Graphpad). All values are expressed as Mean ± SD or SEM as indicated in the figure legends. Student’s t test, one-way ANOVA, two-way ANOVA, or non-parametric tests were performed tailored to the hypotheses tested, and were explained in figure legends. Statistical significance was accepted at α&nbsp;&lt; 0.05.</div></section></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0355" role="paragraph">We thank the patients and families, members of the laboratory, Cecilia Lindestam Arlehamn and Alessandro Sette, NIH Tetramer Core Facility (NTCF), James McCluskey, Jamie Rossjohn, and David Fairlie, the Genomics and the Flow Cytometry Resource Center of Rockefeller University, National Center for Advancing Translational Sciences of the National Institutes of Health (UL1TR001866 to Rockefeller University), Steven Elledge, National Institute of Allergy and Infectious Diseases (R37AI095983 to J.L.C. and U19AI118626 to F.S.), Sackler Center for Biomedicine and Nutrition at the Center for Clinical and Translational Science (CCTS), Shapiro-Silverberg Fund for the Advancement of Translational Research at CCTS, Rockefeller University (to R.Y.), Research Grant Program of the Immune Deficiency Foundation (to R.Y.), Integrative Biology of Emerging Infectious Diseases Laboratory of Excellence (ANR-10-LABX-62-IBEID), French National Research Agency under the “Investments for the future” program (ANR-10-IAHU-01), ANR-GENMSMD (ANR-16-CE17-0005-01 to J.B.), ANR-LTh-MSMD-CMCD (ANR-18-CE93-0008-01 to A.P and F.S.), Fonds de Recherche en Santé Respiratoire (SRC2017 to J.B.), French Foundation for Medical Research (FRM) (EQU201903007798), the SCOR Corporate Foundation for Science, ECOS Nord (C19S01-63407 to J.B.), Swiss National Science Foundation (310030L_182728 to F.S.), French Foundation for Medical Research (FRM) (EQU201903007798), Helmut Horten Foundation, Canadian Center for Computational Genomics (C3G), Genomics Technology Platform (GTP), and Genome Canada. J. R. was supported by INSERM Poste d’accueil and and Imagine Institute, T.K. by Qatar National Research Fund (PPM1-1220-150017). S.G.T. by Program grant (1113904), Principal Research Fellowship (1042925), and Leadership 3 Investigator grant (1176665) from the National Health and Medical Research Council of Australia, C.S.M. by an Early/Mid-Career Development Research Fellowship, Ministry of Health, NSW Government, R.Y. by Stony Wold-Herbert Fund.</div><section id="sec-5"><h3>Author Contributions</h3><div id="p0360" role="paragraph">R.Y., J.B., and J.-L.C. conceptualized the study, designed the experiments, interpreted the data, and wrote the manuscript. R.Y., J.R., D. Langlais, M.B., P.G., L.W., C.S.M., S.G.T., J.L.M., M.S.K., F.M., D.J., M.W., D. Latorre, F.S., C.A.C., J.-M.D., J.P.D., J.M., Y. Shen, J.H., M.O., C.G., D.B., F.A.A., M. Rahman, T.K., N.M., C.O.-Q., M. Roynard, and J.B. conducted experiments and analyzed data. R.Y., P.Z., L.T.H., L.Q.-M., Y. Seeleuthner, G.K., L.A., and T.C. analyzed high-throughput sequencing data. M.J., F.B., and F.R. performed clinical serological tests. H.E., I.B., A.E., F.A., A.B., and J.B. took care of the patient. J.E.B. contributed North African exomes. S.B.-D., A.P., Q.P.-H, L.H.G., P.G., S.G.T., F.S., J. B., and J.-L.C. supervised the study. All authors provided input on the final manuscript.</div></section><section id="sec-6"><h3>Declaration of Interests</h3><div id="p0365" role="paragraph">L.H.G. serves on the Board of Directors of GlaxoSmithKline Pharmaceutical Company and the Analog Device Corporation and formerly served on the Boards of Bristol Myers Squibb Pharmaceutical Company and the Waters Corporation. She is also on the Scientific Advisory Boards of Abpro, Kaleido, and Repare biotechnology companies. J.-L.C. serves on the Scientific Advisory Boards of ADMA Biologics Inc., Celgene, and Kymera Therapeutics, Inc. He also consults for Elixiron Immunotherapeutics. Other authors declare no competing interests.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (3)</h2><div id="p0375" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.10.046/attachment/6419240d-687a-45d2-91ce-9b13a3a1ed61/mmc1.pdf" download="mmc1.pdf" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">PDF (4.14 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Document S1. Tables S1, S4, and S5 and Data S1</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.10.046/attachment/cebc24df-c879-4c00-9806-3dc74d665451/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (1.36 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. RNA-Seq Analysis of Expanded CD4 T<sub>H</sub>0 Cells from Healthy Donors (CTL), a T-bet Heterozygous parent (WT/M), the T-bet Homozygous P (M/M), P Complemented with Empty Vector (M/M+EV), or P complemented with WT T-bet (M/M+WT), Related to Figure&nbsp;2</div><div id="espara0010" role="paragraph">Cells were restimulated with anti-CD3/CD28 Ab-coated beads for 16 h. Lists of DEG_Status, DEG_FoldChange and DEG_ZScore identifying differentially regulated target genes genome-wide. Slim_DEG identifies differentially regulated target immune genes.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2020.10.046/attachment/fc84ce93-3b2e-43be-893d-78907758c096/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (3.84 MB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Chromatin Accessibility of Loci Differentially Regulated in a T-bet-Dependent Manner and Beta-Value for the Differential Methylation of CpG Sites between T-bet m/m (M/M) and Controls (CTL) Abolished by WT T-bet Complementation, Related to Figure&nbsp;3</div><div id="espara0015" role="paragraph">Related data were divided into five sub-sheets. Sheets titled ATAC_Closed by T-bet, ATAC_Opened by T-bet, and ATAC_61 genes above cutoff represent related data from Omni-ATAC-seq dataset generated from CD4+ T&nbsp;cells. Sheets titled Epic_array_MM&nbsp;&lt; CTL and Epic_array_MM &gt; CTL represent related data from the Epic-methylation CpG array.</div></div></div></div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Abecasis, G.R. ∙ Cherny, S.S. ∙ Cookson, W.O. ...</div><div><strong>Merlin--rapid analysis of dense genetic maps using sparse gene flow trees</strong></div><div><em>Nat. Genet.</em> 2002; <strong>30</strong>:97-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0036338150" target="_blank" aria-label="Open Scopus in new tab">Scopus (2652)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11731797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng786&amp;pmid=11731797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2-1" title="View in article">2.</a></div><div>Acosta-Rodriguez, E.V. ∙ Rivino, L. ∙ Geginat, J. ...</div><div><strong>Surface phenotype and antigenic specificity of human interleukin 17-producing T helper memory cells</strong></div><div><em>Nat. Immunol.</em> 2007; <strong>8</strong>:639-646</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni1467" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34249079176" target="_blank" aria-label="Open Scopus in new tab">Scopus (1293)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17486092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni1467&amp;pmid=17486092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Afgan, E. ∙ Baker, D. ∙ van den Beek, M. ...</div><div><strong>The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W3-W10</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw343" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27137889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw343&amp;pmid=27137889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref4" title="View in article">4.</a></div><div>Aluri, J. ∙ Gupta, M. ∙ Dalvi, A. ...</div><div><strong>Clinical, immunological, and molecular findings in five patients with Major histocompatibility complex class II deficiency from India</strong></div><div><em>Front. Immunol.</em> 2018; <strong>9</strong>:188</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2018.00188" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85042098241" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29527204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2018.00188&amp;pmid=29527204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Amir el, A.D. ∙ Davis, K.L. ∙ Tadmor, M.D. ...</div><div><strong>viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia</strong></div><div><em>Nat. Biotechnol.</em> 2013; <strong>31</strong>:545-552</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.2594" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84880280631" target="_blank" aria-label="Open Scopus in new tab">Scopus (816)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23685480/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.2594&amp;pmid=23685480" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Ardeniz, Ö. ∙ Unger, S. ∙ Onay, H. ...</div><div><strong>β2-Microglobulin deficiency causes a complex immunodeficiency of the innate and adaptive immune system</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2015; <strong>136</strong>:392-401</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.12.1937&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.12.1937&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84938746200" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25702838/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2014.12.1937&amp;pmid=25702838" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref7" title="View in article">7.</a></div><div>Auton, A. ∙ Brooks, L.D. ∙ Durbin, R.M. ..., 1000 Genomes Project Consortium</div><div><strong>A global reference for human genetic variation</strong></div><div><em>Nature.</em> 2015; <strong>526</strong>:68-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature15393" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84943171338" target="_blank" aria-label="Open Scopus in new tab">Scopus (5078)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature15393&amp;pmid=26432245" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Bakshi, C.S. ∙ Malik, M. ∙ Carrico, P.M. ...</div><div><strong>T-bet deficiency facilitates airway colonization by Mycoplasma pulmonis in a murine model of asthma</strong></div><div><em>J.&nbsp;Immunol.</em> 2006; <strong>177</strong>:1786-1795</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.177.3.1786" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33746215188" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16849489/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.177.3.1786&amp;pmid=16849489" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref9" title="View in article">9.</a></div><div>Becattini, S. ∙ Latorre, D. ∙ Mele, F. ...</div><div><strong>T cell immunity. Functional heterogeneity of human memory CD4(+) T&nbsp;cell clones primed by pathogens or vaccines</strong></div><div><em>Science.</em> 2015; <strong>347</strong>:400-406</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1260668" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84921753054" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25477212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1260668&amp;pmid=25477212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10-1" title="View in article">10.</a></div><div>Becht, E. ∙ McInnes, L. ∙ Healy, J. ...</div><div><strong>Dimensionality reduction for visualizing single-cell data using UMAP</strong></div><div><em>Nat. Biotechnol.</em> 2018; <strong>37</strong>:38-44</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.4314" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059628259" target="_blank" aria-label="Open Scopus in new tab">Scopus (449)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.4314" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11-1" title="View in article">11.</a></div><div>Bettelli, E. ∙ Sullivan, B. ∙ Szabo, S.J. ...</div><div><strong>Loss of T-bet, but not STAT1, prevents the development of experimental autoimmune encephalomyelitis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2004; <strong>200</strong>:79-87</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20031819" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-3142724732" target="_blank" aria-label="Open Scopus in new tab">Scopus (361)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15238607/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20031819&amp;pmid=15238607" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Björkström, N.K. ∙ Ljunggren, H.-G. ∙ Sandberg, J.K.</div><div><strong>CD56 negative NK cells: origin, function, and role in chronic viral disease</strong></div><div><em>Trends Immunol.</em> 2010; <strong>31</strong>:401-406</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.it.2010.08.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.it.2010.08.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-78049252250" target="_blank" aria-label="Open Scopus in new tab">Scopus (132)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20829113/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.it.2010.08.003&amp;pmid=20829113" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref13" title="View in article">13.</a></div><div>Bogunovic, D. ∙ Byun, M. ∙ Durfee, L.A. ...</div><div><strong>Mycobacterial disease and impaired IFN-gamma immunity in humans with inherited ISG15 deficiency</strong></div><div><em>Science.</em> 2012; <strong>337</strong>:1684-1688</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1224026" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866748115" target="_blank" aria-label="Open Scopus in new tab">Scopus (279)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22859821/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1224026&amp;pmid=22859821" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref14" title="View in article">14.</a></div><div>Boisson-Dupuis, S. ∙ Ramirez-Alejo, N. ∙ Li, Z. ...</div><div><strong>Tuberculosis and impaired IL-23-dependent IFN-γuberculosis and impaired IL-23-dependent IFN-mmunity in humans wit</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaau8714</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau8714" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_14_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85058915562" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30578352/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau8714&amp;pmid=30578352" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib15" class="citations"><div id="sref15" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref15-1" title="View in article">15.</a></div><div>Buenrostro, J.D. ∙ Giresi, P.G. ∙ Zaba, L.C. ...</div><div><strong>Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position</strong></div><div><em>Nat. Methods.</em> 2013; <strong>10</strong>:1213-1218</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.2688" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84888877924" target="_blank" aria-label="Open Scopus in new tab">Scopus (1949)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24097267/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.2688&amp;pmid=24097267" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref16-1" title="View in article">16.</a></div><div>Buenrostro, J.D. ∙ Wu, B. ∙ Chang, H.Y. ...</div><div><strong>ATAC-seq: A Method for Assaying Chromatin Accessibility Genome-Wide</strong></div><div><em>Curr. Protoc. Mol. Biol.</em> 2015; <strong>109</strong>:21.29.1-21.29.9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/0471142727.mb2129s109" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84949212255" target="_blank" aria-label="Open Scopus in new tab">Scopus (502)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25559105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2F0471142727.mb2129s109&amp;pmid=25559105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref17-1" title="View in article">17.</a></div><div>Bustamante, J.</div><div><strong>Mendelian susceptibility to mycobacterial disease: recent discoveries</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:993-1000</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02120-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85079165775" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32025907/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02120-y&amp;pmid=32025907" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref18-1" title="View in article">18.</a></div><div>Carroll, T. ∙ Barrows, D.</div><div><strong>profileplyr: Visualization and annotation of read signal over genomic ranges with profileplyr</strong></div><div>2019</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=T.CarrollD.Barrowsprofileplyr%3A+Visualization+and+annotation+of+read+signal+over+genomic+ranges+with+profileplyr2019" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib19" class="citations"><div id="sref19" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref19-1" title="View in article">19.</a></div><div>Carroll, T.S. ∙ Liang, Z. ∙ Salama, R. ...</div><div><strong>Impact of artifact removal on ChIP quality metrics in ChIP-seq and ChIP-exo data</strong></div><div><em>Front. Genet.</em> 2014; <strong>5</strong>:75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fgene.2014.00075" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84898030206" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24782889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffgene.2014.00075&amp;pmid=24782889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20" title="View in article">20.</a></div><div>Casanova, J.L.</div><div><strong>Human genetic basis of interindividual variability in the course of infection</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E7118-E7127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1521644112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84952683321" target="_blank" aria-label="Open Scopus in new tab">Scopus (84)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26621739/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1521644112&amp;pmid=26621739" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref21" title="View in article">21.</a></div><div>Casanova, J.L.</div><div><strong>Severe infectious diseases of childhood as monogenic inborn errors of immunity</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:E7128-E7137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1521651112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84952685581" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26621750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1521651112&amp;pmid=26621750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib182" class="citations"><div id="sref182" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref182" title="View in article">22.</a></div><div>Casanova, J.L.</div><div><strong>The human genetic determinism of life-threatening infectious diseases: genetic heterogeneity and physiological homogeneity?</strong></div><div><em>Hum. Genet.</em> 2020; <strong>139</strong>:681-694</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s00439-020-02184-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_22_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085371245" target="_blank" aria-label="Open Scopus in new tab">Scopus (3)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32462426/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs00439-020-02184-w&amp;pmid=32462426" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib181" class="citations"><div id="sref181" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref181" title="View in article">23.</a></div><div>Casanova, J.L.</div><div><strong>Lethal Infectious Diseases as Inborn Errors of Immunity: Toward a Synthesis of the Germ and Genetic Theories.</strong></div><div><em>Annu. Rev. Pathol.</em> 2021; <strong>16</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-pathol-031920-101429" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-pathol-031920-101429" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22" title="View in article">24.</a></div><div>Casanova, J.L. ∙ Abel, L.</div><div><strong>The human model: a genetic dissection of immunity to infection in natural conditions</strong></div><div><em>Nat. Rev. Immunol.</em> 2004; <strong>4</strong>:55-66</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri1264" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14704768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri1264&amp;pmid=14704768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref23" title="View in article">25.</a></div><div>Casanova, J.L. ∙ Abel, L.</div><div><strong>Human genetics of infectious diseases: Unique insights into immunological redundancy</strong></div><div><em>Semin. Immunol.</em> 2018; <strong>36</strong>:1-12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.smim.2017.12.008" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_25_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85039455920" target="_blank" aria-label="Open Scopus in new tab">Scopus (34)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29254755/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.smim.2017.12.008&amp;pmid=29254755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">26.</a></div><div>Casanova, J.L. ∙ Jouanguy, E. ∙ Lamhamedi, S. ...</div><div><strong>Immunological conditions of children with BCG disseminated infection</strong></div><div><em>Lancet.</em> 1995; <strong>346</strong>:581</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="/servlet/linkout?suffix=e_1_5_1_2_26_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.5555%2Furi%3Apii%3AS0140673695914218&amp;site=lancet-site" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7658805/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.5555%2Furi%3Apii%3AS0140673695914218&amp;pmid=7658805" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref25-1" title="View in article">27.</a></div><div>Casanova, J.-L. ∙ Conley, M.E. ∙ Seligman, S.J. ...</div><div><strong>Guidelines for genetic studies in single patients: lessons from primary immunodeficiencies</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2014; <strong>211</strong>:2137-2149</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20140520" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84908247576" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25311508/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20140520&amp;pmid=25311508" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">28.</a></div><div>Casey, J.P. ∙ Nobbs, M. ∙ McGettigan, P. ...</div><div><strong>Recessive mutations in MCM4/PRKDC cause a novel syndrome involving a primary immunodeficiency and a disorder of DNA repair</strong></div><div><em>J.&nbsp;Med. Genet.</em> 2012; <strong>49</strong>:242-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/jmedgenet-2012-100803" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84864097350" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22499342/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fjmedgenet-2012-100803&amp;pmid=22499342" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">29.</a></div><div>Chen, Z.W.</div><div><strong>Immune regulation of gammadelta T&nbsp;cell responses in mycobacterial infections</strong></div><div><em>Clin. Immunol.</em> 2005; <strong>116</strong>:202-207</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.clim.2005.04.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-23444455796" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16087145/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.clim.2005.04.005&amp;pmid=16087145" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">30.</a></div><div>Chen, L. ∙ He, W. ∙ Kim, S.T. ...</div><div><strong>Epigenetic and transcriptional programs lead to default IFN-γ production by gammadelta T&nbsp;cells</strong></div><div><em>J.&nbsp;Immunol.</em> 2007; <strong>178</strong>:2730-2736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.178.5.2730" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17312115/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.178.5.2730&amp;pmid=17312115" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref29" title="View in article">31.</a></div><div>Chien, Y.H. ∙ Meyer, C. ∙ Bonneville, M.</div><div><strong>γδ T&nbsp;cells: first line of defense and beyond</strong></div><div><em>Annu. Rev. Immunol.</em> 2014; <strong>32</strong>:121-155</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-032713-120216" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84896966923" target="_blank" aria-label="Open Scopus in new tab">Scopus (280)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24387714/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-032713-120216&amp;pmid=24387714" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref30-1" title="View in article">32.</a></div><div>Cingolani, P. ∙ Platts, A. ∙ Wang, L. ...</div><div><strong>A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</strong></div><div><em>Fly (Austin).</em> 2012; <strong>6</strong>:80-92</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4161/fly.19695" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84862506964" target="_blank" aria-label="Open Scopus in new tab">Scopus (3490)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4161%2Ffly.19695&amp;pmid=22728672" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">33.</a></div><div>Corces, M.R. ∙ Trevino, A.E. ∙ Hamilton, E.G. ...</div><div><strong>An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:959-962</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85030994641" target="_blank" aria-label="Open Scopus in new tab">Scopus (256)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28846090/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4396&amp;pmid=28846090" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">34.</a></div><div>Cottineau, J. ∙ Kottemann, M.C. ∙ Lach, F.P. ...</div><div><strong>Inherited GINS1 deficiency underlies growth retardation along with neutropenia and NK cell deficiency</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2017; <strong>127</strong>:1991-2006</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI90727" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85018993845" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28414293/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI90727&amp;pmid=28414293" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">35.</a></div><div>Crosby, C.M. ∙ Kronenberg, M.</div><div><strong>Tissue-specific functions of invariant natural killer T&nbsp;cells</strong></div><div><em>Nat. Rev. Immunol.</em> 2018; <strong>18</strong>:559-574</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41577-018-0034-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_35_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85049559053" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29967365/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41577-018-0034-2&amp;pmid=29967365" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">36.</a></div><div>Daussy, C. ∙ Faure, F. ∙ Mayol, K. ...</div><div><strong>T-bet and Eomes instruct the development of two distinct natural killer cell lineages in the liver and in the bone marrow</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2014; <strong>211</strong>:563-577</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20131560" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84896847858" target="_blank" aria-label="Open Scopus in new tab">Scopus (294)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24516120/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20131560&amp;pmid=24516120" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35" title="View in article">37.</a></div><div>Davis, B.R. ∙ Dicola, M.J. ∙ Prokopishyn, N.L. ...</div><div><strong>Unprecedented diversity of genotypic revertants in lymphocytes of a patient with Wiskott-Aldrich syndrome</strong></div><div><em>Blood.</em> 2008; <strong>111</strong>:5064-5067</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2007-06-095299" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-46749091756" target="_blank" aria-label="Open Scopus in new tab">Scopus (24)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18332229/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2007-06-095299&amp;pmid=18332229" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref36" title="View in article">38.</a></div><div>De La Salle, H. ∙ Hanau, D. ∙ Fricker, D. ...</div><div><strong>Homozygous human TAP peptide transporter mutation in HLA class I deficiency</strong></div><div><em>Science.</em> 1994; <strong>265</strong>:237-241</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.7517574" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7517574/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.7517574&amp;pmid=7517574" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref37-1" title="View in article">39.</a></div><div>de Santiago, I. ∙ Carroll, T.</div><div><strong>Analysis of ChIP-seq Data in R/Bioconductor</strong></div><div>2018 195-226</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=I.de+SantiagoT.CarrollAnalysis+of+ChIP-seq+Data+in+R%2FBioconductor2018195226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref38-1" title="View in article">40.</a></div><div>DePristo, M.A. ∙ Banks, E. ∙ Poplin, R. ...</div><div><strong>A framework for variation discovery and genotyping using next-generation DNA sequencing data</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:491-498</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.806" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79955483667" target="_blank" aria-label="Open Scopus in new tab">Scopus (5537)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21478889/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.806&amp;pmid=21478889" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">41.</a></div><div>Dimitrova, D. ∙ Ong, P.Y. ∙ O’Gorman, M.R.G. ...</div><div><strong>Major histocompatibility complex class II deficiency complicated by Mycobacterium avium complex in a boy of mixed ethnicity</strong></div><div><em>J.&nbsp;Clin. Immunol.</em> 2014; <strong>34</strong>:677-680</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10875-014-0048-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84939872978" target="_blank" aria-label="Open Scopus in new tab">Scopus (5)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24789686/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10875-014-0048-x&amp;pmid=24789686" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">42.</a></div><div>Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F. ...</div><div><strong>STAR: ultrafast universal RNA-seq aligner</strong></div><div><em>Bioinformatics.</em> 2013; <strong>29</strong>:15-21</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871809302" target="_blank" aria-label="Open Scopus in new tab">Scopus (9649)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts635&amp;pmid=23104886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref41" title="View in article">43.</a></div><div>Dorner, B.G. ∙ Scheffold, A. ∙ Rolph, M.S. ...</div><div><strong>MIP-1alpha, MIP-1beta, RANTES, and ATAC/lymphotactin function together with IFN-gamma as type 1 cytokines</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2002; <strong>99</strong>:6181-6186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.092141999" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037197261" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11972057/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.092141999&amp;pmid=11972057" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">44.</a></div><div>Dorner, B.G. ∙ Steinbach, S. ∙ Hüser, M.B. ...</div><div><strong>Single-cell analysis of the murine chemokines MIP-1alpha, MIP-1beta, RANTES and ATAC/lymphotactin by flow cytometry</strong></div><div><em>J.&nbsp;Immunol. Methods.</em> 2003; <strong>274</strong>:83-91</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0022-1759(02)00498-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037374062" target="_blank" aria-label="Open Scopus in new tab">Scopus (18)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12609535/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0022-1759%2802%2900498-2&amp;pmid=12609535" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">45.</a></div><div>Dorner, B.G. ∙ Smith, H.R.C. ∙ French, A.R. ...</div><div><strong>Coordinate expression of cytokines and chemokines by NK cells during murine cytomegalovirus infection</strong></div><div><em>J.&nbsp;Immunol.</em> 2004; <strong>172</strong>:3119-3131</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.172.5.3119" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-1342303341" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14978118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.172.5.3119&amp;pmid=14978118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">46.</a></div><div>Dupuis, S. ∙ Döffinger, R. ∙ Picard, C. ...</div><div><strong>Human interferon-gamma-mediated immunity is a genetically controlled continuous trait that determines the outcome of mycobacterial invasion</strong></div><div><em>Immunol. Rev.</em> 2000; <strong>178</strong>:129-137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1034/j.1600-065X.2000.17810.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11213797/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1034%2Fj.1600-065X.2000.17810.x&amp;pmid=11213797" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">47.</a></div><div>Elhasid, R. ∙ Etzioni, A.</div><div><strong>Major histocompatibility complex class II deficiency: a clinical review</strong></div><div><em>Blood Rev.</em> 1996; <strong>10</strong>:242-248</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0268-960X(96)90008-9" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0030465562" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9012922/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0268-960X%2896%2990008-9&amp;pmid=9012922" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref46-1" title="View in article">48.</a></div><div>Feng, J. ∙ Liu, T. ∙ Qin, B. ...</div><div><strong>Identifying ChIP-seq enrichment using MACS</strong></div><div><em>Nat. Protoc.</em> 2012; <strong>7</strong>:1728-1740</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2012.101" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866127830" target="_blank" aria-label="Open Scopus in new tab">Scopus (528)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22936215/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2012.101&amp;pmid=22936215" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47-1" title="View in article">49.</a></div><div>Finak, G. ∙ McDavid, A. ∙ Yajima, M. ...</div><div><strong>MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data</strong></div><div><em>Genome Biol.</em> 2015; <strong>16</strong>:278</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-015-0844-5" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_49_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84951574149" target="_blank" aria-label="Open Scopus in new tab">Scopus (404)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26653891/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-015-0844-5&amp;pmid=26653891" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref48" title="View in article">50.</a></div><div>Finotto, S. ∙ Neurath, M.F. ∙ Glickman, J.N. ...</div><div><strong>Development of spontaneous airway changes consistent with human asthma in mice lacking T-bet</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:336-338</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065544" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059509" target="_blank" aria-label="Open Scopus in new tab">Scopus (513)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786643/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065544&amp;pmid=11786643" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref49" title="View in article">51.</a></div><div>Frankish, A. ∙ Diekhans, M. ∙ Ferreira, A.-M. ...</div><div><strong>GENCODE reference annotation for the human and mouse genomes</strong></div><div><em>Nucleic Acids Res.</em> 2019; <strong>47</strong>:D766-D773</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gky955" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85059554160" target="_blank" aria-label="Open Scopus in new tab">Scopus (366)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30357393/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgky955&amp;pmid=30357393" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref50" title="View in article">52.</a></div><div>Garrett, W.S. ∙ Lord, G.M. ∙ Punit, S. ...</div><div><strong>Communicable ulcerative colitis induced by T-bet deficiency in the innate immune system</strong></div><div><em>Cell.</em> 2007; <strong>131</strong>:33-45</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2007.08.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2007.08.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34848889673" target="_blank" aria-label="Open Scopus in new tab">Scopus (687)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17923086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2007.08.017&amp;pmid=17923086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51" title="View in article">53.</a></div><div>Garrido-Mesa, N. ∙ Schroeder, J.-H. ∙ Stolarczyk, E. ...</div><div><strong>T-bet controls intestinal mucosa immune responses via repression of type 2 innate lymphoid cell function</strong></div><div><em>Mucosal Immunol.</em> 2019; <strong>12</strong>:51-63</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41385-018-0092-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85055560183" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30356098/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41385-018-0092-6&amp;pmid=30356098" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref52" title="View in article">54.</a></div><div>Gavins, F.N.E. ∙ Hickey, M.J.</div><div><strong>Annexin A1 and the regulation of innate and adaptive immunity</strong></div><div><em>Front. Immunol.</em> 2012; <strong>3</strong>:354</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2012.00354" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84874215363" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23230437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2012.00354&amp;pmid=23230437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref53-1" title="View in article">55.</a></div><div>Geiger, R. ∙ Duhen, T. ∙ Lanzavecchia, A. ...</div><div><strong>Human naive and memory CD4+ T&nbsp;cell repertoires specific for naturally processed antigens analyzed using libraries of amplified T&nbsp;cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2009; <strong>206</strong>:1525-1534</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20090504" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-67650469829" target="_blank" aria-label="Open Scopus in new tab">Scopus (148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19564353/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20090504&amp;pmid=19564353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54-1" title="View in article">56.</a></div><div>Geurts, P. ∙ Ernst, D. ∙ Wehenkel, L.</div><div><strong>Extremely randomized trees</strong></div><div><em>Mach. Learn.</em> 2006; <strong>63</strong>:3-42</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10994-006-6226-1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33646430006" target="_blank" aria-label="Open Scopus in new tab">Scopus (1993)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10994-006-6226-1" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55" title="View in article">57.</a></div><div>Godfrey, D.I. ∙ Koay, H.-F. ∙ McCluskey, J. ...</div><div><strong>The biology and functional importance of MAIT cells</strong></div><div><em>Nat. Immunol.</em> 2019; <strong>20</strong>:1110-1128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-019-0444-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85070817379" target="_blank" aria-label="Open Scopus in new tab">Scopus (66)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31406380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-019-0444-8&amp;pmid=31406380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56" title="View in article">58.</a></div><div>Gökmen, M.R. ∙ Dong, R. ∙ Kanhere, A. ...</div><div><strong>Genome-wide regulatory analysis reveals that T-bet controls Th17 lineage differentiation through direct suppression of IRF4</strong></div><div><em>J.&nbsp;Immunol.</em> 2013; <strong>191</strong>:5925-5932</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1202254" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84890418697" target="_blank" aria-label="Open Scopus in new tab">Scopus (27)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24249732/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1202254&amp;pmid=24249732" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">59.</a></div><div>Groom, J.R. ∙ Luster, A.D.</div><div><strong>CXCR3 in T&nbsp;cell function</strong></div><div><em>Exp. Cell Res.</em> 2011; <strong>317</strong>:620-631</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.yexcr.2010.12.017" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79952091851" target="_blank" aria-label="Open Scopus in new tab">Scopus (400)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21376175/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.yexcr.2010.12.017&amp;pmid=21376175" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref58-1" title="View in article">60.</a></div><div>Gu, Z. ∙ Eils, R. ∙ Schlesner, M.</div><div><strong>Complex heatmaps reveal patterns and correlations in multidimensional genomic data</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:2847-2849</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw313" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84992187637" target="_blank" aria-label="Open Scopus in new tab">Scopus (843)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw313&amp;pmid=27207943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59-1" title="View in article">61.</a></div><div>Gu, S. ∙ Borowska, M.T. ∙ Boughter, C.T. ...</div><div><strong>Butyrophilin3A proteins and Vγ9Vδ2 T&nbsp;cell activation</strong></div><div><em>Semin. Cell Dev. Biol.</em> 2018; <strong>84</strong>:65-74</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.semcdb.2018.02.007" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85043373095" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29471037/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.semcdb.2018.02.007&amp;pmid=29471037" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">62.</a></div><div>Hambleton, S. ∙ Salem, S. ∙ Bustamante, J. ...</div><div><strong>IRF8 mutations and human dendritic-cell immunodeficiency</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2011; <strong>365</strong>:127-138</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1100066" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79960219807" target="_blank" aria-label="Open Scopus in new tab">Scopus (401)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21524210/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1100066&amp;pmid=21524210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref61" title="View in article">63.</a></div><div>Hanalioglu, D. ∙ Ayvaz, D.C. ∙ Ozgur, T.T. ...</div><div><strong>A novel mutation in TAP1 gene leading to MHC class I deficiency: Report of two cases and review of the literature</strong></div><div><em>Clin. Immunol.</em> 2017; <strong>178</strong>:74-78</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.clim.2017.01.011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85013123103" target="_blank" aria-label="Open Scopus in new tab">Scopus (9)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28161407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.clim.2017.01.011&amp;pmid=28161407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref62" title="View in article">64.</a></div><div>Hanna, S. ∙ Etzioni, A.</div><div><strong>MHC class I and II deficiencies</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2014; <strong>134</strong>:269-275</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.06.001&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_64_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2014.06.001&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25001848/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2014.06.001&amp;pmid=25001848" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63" title="View in article">65.</a></div><div>Harayama, T. ∙ Riezman, H.</div><div><strong>Detection of genome-edited mutant clones by a simple competition-based PCR method</strong></div><div><em>PLoS ONE.</em> 2017; <strong>12</strong>:e0179165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pone.0179165" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_65_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85020204183" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28586390/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pone.0179165&amp;pmid=28586390" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64-1" title="View in article">66.</a></div><div>Harly, C. ∙ Guillaume, Y. ∙ Nedellec, S. ...</div><div><strong>Key implication of CD277/butyrophilin-3 (BTN3A) in cellular stress sensing by a major human γδ T-cell subset</strong></div><div><em>Blood.</em> 2012; <strong>120</strong>:2269-2279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2012-05-430470" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84866308872" target="_blank" aria-label="Open Scopus in new tab">Scopus (282)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22767497/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2012-05-430470&amp;pmid=22767497" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref65" title="View in article">67.</a></div><div>Harms Pritchard, G. ∙ Hall, A.O. ∙ Christian, D.A. ...</div><div><strong>Diverse roles for T-bet in the effector responses required for resistance to infection</strong></div><div><em>J.&nbsp;Immunol.</em> 2015; <strong>194</strong>:1131-1140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1401617" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84921456080" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25556247/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1401617&amp;pmid=25556247" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">68.</a></div><div>Hernandez, N. ∙ Melki, I. ∙ Jing, H. ...</div><div><strong>Life-threatening influenza pneumonitis in a child with inherited IRF9 deficiency</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2018; <strong>215</strong>:2567-2585</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20180628" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85054094895" target="_blank" aria-label="Open Scopus in new tab">Scopus (33)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30143481/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20180628&amp;pmid=30143481" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">69.</a></div><div>Hoft, D.F. ∙ Brown, R.M. ∙ Roodman, S.T.</div><div><strong>Bacille Calmette-Guérin vaccination enhances human gamma delta T&nbsp;cell responsiveness to mycobacteria suggestive of a memory-like phenotype</strong></div><div><em>J.&nbsp;Immunol.</em> 1998; <strong>161</strong>:1045-1054</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9670986/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=9670986" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref68" title="View in article">70.</a></div><div>Hughes, C.R. ∙ Guasti, L. ∙ Meimaridou, E. ...</div><div><strong>MCM4 mutation causes adrenal failure, short stature, and natural killer cell deficiency in humans</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2012; <strong>122</strong>:814-820</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI60224" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_70_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857875264" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354170/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI60224&amp;pmid=22354170" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref69" title="View in article">71.</a></div><div>Hultgren, O.H. ∙ Verdrengh, M. ∙ Tarkowski, A.</div><div><strong>T-box transcription-factor-deficient mice display increased joint pathology and failure of infection control during staphylococcal arthritis</strong></div><div><em>Microbes Infect.</em> 2004; <strong>6</strong>:529-535</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/j.micinf.2004.02.005" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-2542424828" target="_blank" aria-label="Open Scopus in new tab">Scopus (22)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15158185/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.micinf.2004.02.005&amp;pmid=15158185" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref70" title="View in article">72.</a></div><div>Intlekofer, A.M. ∙ Takemoto, N. ∙ Kao, C. ...</div><div><strong>Requirement for T-bet in the aberrant differentiation of unhelped memory CD8+ T&nbsp;cells</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2007; <strong>204</strong>:2015-2021</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20070841" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-34548402098" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17698591/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20070841&amp;pmid=17698591" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71" title="View in article">73.</a></div><div>Itan, Y. ∙ Zhang, S.-Y. ∙ Vogt, G. ...</div><div><strong>The human gene connectome as a map of short cuts for morbid allele discovery</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2013; <strong>110</strong>:5558-5563</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1218167110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84875826166" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23509278/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1218167110&amp;pmid=23509278" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref72" title="View in article">74.</a></div><div>Itan, Y. ∙ Mazel, M. ∙ Mazel, B. ...</div><div><strong>HGCS: an online tool for prioritizing disease-causing gene variants by biological distance</strong></div><div><em>BMC Genomics.</em> 2014; <strong>15</strong>:256</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2164-15-256" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84899103596" target="_blank" aria-label="Open Scopus in new tab">Scopus (23)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24694260/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2164-15-256&amp;pmid=24694260" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref73" title="View in article">75.</a></div><div>Itan, Y. ∙ Shang, L. ∙ Boisson, B. ...</div><div><strong>The human gene damage index as a gene-level approach to prioritizing exome variants</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2015; <strong>112</strong>:13615-13620</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1518646112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84946595515" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26483451/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1518646112&amp;pmid=26483451" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref74-1" title="View in article">76.</a></div><div>Itan, Y. ∙ Shang, L. ∙ Boisson, B. ...</div><div><strong>The mutation significance cutoff: gene-level thresholds for variant predictions</strong></div><div><em>Nat. Methods.</em> 2016; <strong>13</strong>:109-110</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3739" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84961309083" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26820543/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3739&amp;pmid=26820543" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75" title="View in article">77.</a></div><div>Iwata, S. ∙ Mikami, Y. ∙ Sun, H.-W. ...</div><div><strong>The Transcription Factor T-bet Limits Amplification of Type I IFN Transcriptome and Circuitry in T Helper 1 Cells</strong></div><div><em>Immunity.</em> 2017; <strong>46</strong>:983-991</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2017.05.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2017.05.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85020642282" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28623086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2017.05.005&amp;pmid=28623086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib76" class="citations"><div id="sref76" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref76-1" title="View in article">78.</a></div><div>Janesick, A. ∙ Shiotsugu, J. ∙ Taketani, M. ...</div><div><strong>RIPPLY3 is a retinoic acid-inducible repressor required for setting the borders of the pre-placodal ectoderm</strong></div><div><em>Development.</em> 2012; <strong>139</strong>:1213-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1242/dev.071456" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84857300258" target="_blank" aria-label="Open Scopus in new tab">Scopus (21)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22354841/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1242%2Fdev.071456&amp;pmid=22354841" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">79.</a></div><div>Jang, E.J. ∙ Park, H.R. ∙ Hong, J.H. ...</div><div><strong>Lysine 313 of T-box is crucial for modulation of protein stability, DNA binding, and threonine phosphorylation of T-bet</strong></div><div><em>J.&nbsp;Immunol.</em> 2013; <strong>190</strong>:5764-5770</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1203403" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84878073297" target="_blank" aria-label="Open Scopus in new tab">Scopus (26)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23616576/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1203403&amp;pmid=23616576" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78-1" title="View in article">80.</a></div><div>Jenner, R.G. ∙ Townsend, M.J. ∙ Jackson, I. ...</div><div><strong>The transcription factors T-bet and GATA-3 control alternative pathways of T-cell differentiation through a shared set of target genes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2009; <strong>106</strong>:17876-17881</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0909357106" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-70449571945" target="_blank" aria-label="Open Scopus in new tab">Scopus (142)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19805038/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0909357106&amp;pmid=19805038" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref79" title="View in article">81.</a></div><div>Jouanguy, E. ∙ Döffinger, R. ∙ Dupuis, S. ...</div><div><strong>IL-12 and IFN-γ in host defense against mycobacteria and salmonella in mice and men</strong></div><div><em>Curr. Opin. Immunol.</em> 1999; <strong>11</strong>:346-351</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0952-7915(99)80055-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0033057576" target="_blank" aria-label="Open Scopus in new tab">Scopus (275)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10375558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0952-7915%2899%2980055-7&amp;pmid=10375558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">82.</a></div><div>Juedes, A.E. ∙ Rodrigo, E. ∙ Togher, L. ...</div><div><strong>T-bet controls autoaggressive CD8 lymphocyte responses in type 1 diabetes</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2004; <strong>199</strong>:1153-1162</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20031873" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-2442509626" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15096540/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20031873&amp;pmid=15096540" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref81-1" title="View in article">83.</a></div><div>Kanhere, A. ∙ Hertweck, A. ∙ Bhatia, U. ...</div><div><strong>T-bet and GATA3 orchestrate Th1 and Th2 differentiation through lineage-specific targeting of distal regulatory elements</strong></div><div><em>Nat. Commun.</em> 2012; <strong>3</strong>:1268</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms2260" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84871783406" target="_blank" aria-label="Open Scopus in new tab">Scopus (161)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23232398/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms2260&amp;pmid=23232398" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref82" title="View in article">84.</a></div><div>Kao, C. ∙ Oestreich, K.J. ∙ Paley, M.A. ...</div><div><strong>Transcription factor T-bet represses expression of the inhibitory receptor PD-1 and sustains virus-specific CD8+ T&nbsp;cell responses during chronic infection</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:663-671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2046" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79959375167" target="_blank" aria-label="Open Scopus in new tab">Scopus (276)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21623380/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2046&amp;pmid=21623380" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref83-1" title="View in article">85.</a></div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ..., Genome Aggregation Database Consortium</div><div><strong>The mutational constraint spectrum quantified from variation in 141,456 humans</strong></div><div><em>Nature.</em> 2020; <strong>581</strong>:434-443</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41586-020-2308-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085542423" target="_blank" aria-label="Open Scopus in new tab">Scopus (373)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41586-020-2308-7&amp;pmid=32461654" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref84-1" title="View in article">86.</a></div><div>Kerner, G. ∙ Rosain, J. ∙ Guérin, A. ...</div><div><strong>Inherited human IFN-γ deficiency underlies mycobacterial disease</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2020; <strong>130</strong>:3158-3171</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI135460" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85085294382" target="_blank" aria-label="Open Scopus in new tab">Scopus (8)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/32163377/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI135460&amp;pmid=32163377" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85-1" title="View in article">87.</a></div><div>Kircher, M. ∙ Witten, D.M. ∙ Jain, P. ...</div><div><strong>A general framework for estimating the relative pathogenicity of human genetic variants</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:310-315</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2892" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84895858942" target="_blank" aria-label="Open Scopus in new tab">Scopus (2683)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24487276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2892&amp;pmid=24487276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib86" class="citations"><div id="sref86" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref86" title="View in article">88.</a></div><div>Kjer-Nielsen, L. ∙ Patel, O. ∙ Corbett, A.J. ...</div><div><strong>MR1 presents microbial vitamin B metabolites to MAIT cells</strong></div><div><em>Nature.</em> 2012; <strong>491</strong>:717-723</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11605" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84870284140" target="_blank" aria-label="Open Scopus in new tab">Scopus (659)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23051753/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11605&amp;pmid=23051753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">89.</a></div><div>Koch, M.A. ∙ Tucker-Heard, G. ∙ Perdue, N.R. ...</div><div><strong>The transcription factor T-bet controls regulatory T&nbsp;cell homeostasis and function during type 1 inflammation</strong></div><div><em>Nat. Immunol.</em> 2009; <strong>10</strong>:595-602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1731" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-65749103365" target="_blank" aria-label="Open Scopus in new tab">Scopus (805)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19412181/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1731&amp;pmid=19412181" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref88" title="View in article">90.</a></div><div>Kong, X.F. ∙ Martinez-Barricarte, R. ∙ Kennedy, J. ...</div><div><strong>Disruption of an antimycobacterial circuit between dendritic and helper T&nbsp;cells in human SPPL2a deficiency</strong></div><div><em>Nat. Immunol.</em> 2018; <strong>19</strong>:973-985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41590-018-0178-z" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85051985294" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30127434/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41590-018-0178-z&amp;pmid=30127434" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref89" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref89-1" title="View in article">91.</a></div><div>Korsunsky, I. ∙ Millard, N. ∙ Fan, J. ...</div><div><strong>Fast, sensitive and accurate integration of single-cell data with Harmony</strong></div><div><em>Nat. Methods.</em> 2019; <strong>16</strong>:1289-1296</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41592-019-0619-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85070698712" target="_blank" aria-label="Open Scopus in new tab">Scopus (112)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41592-019-0619-0&amp;pmid=31740819" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90" title="View in article">92.</a></div><div>Kroczek, R.A. ∙ Henn, V.</div><div><strong>The Role of XCR1 and its Ligand XCL1 in Antigen Cross-Presentation by Murine and Human Dendritic Cells</strong></div><div><em>Front. Immunol.</em> 2012; <strong>3</strong>:14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2012.00014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84874067035" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22566900/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2012.00014&amp;pmid=22566900" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">93.</a></div><div>Latour, S.</div><div><strong>Natural killer T&nbsp;cells and X-linked lymphoproliferative syndrome</strong></div><div><em>Curr. Opin. Allergy Clin. Immunol.</em> 2007; <strong>7</strong>:510-514</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1097/ACI.0b013e3282f1bad6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17989527/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1097%2FACI.0b013e3282f1bad6&amp;pmid=17989527" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92" title="View in article">94.</a></div><div>Lazarevic, V. ∙ Chen, X. ∙ Shim, J.H. ...</div><div><strong>T-bet represses T(H)17 differentiation by preventing Runx1-mediated activation of the gene encoding RORγt</strong></div><div><em>Nat. Immunol.</em> 2011; <strong>12</strong>:96-104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.1969" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-78650306567" target="_blank" aria-label="Open Scopus in new tab">Scopus (243)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21151104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.1969&amp;pmid=21151104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">95.</a></div><div>Lazarevic, V. ∙ Glimcher, L.H. ∙ Lord, G.M.</div><div><strong>T-bet: a bridge between innate and adaptive immunity</strong></div><div><em>Nat. Rev. Immunol.</em> 2013; <strong>13</strong>:777-789</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nri3536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84886790466" target="_blank" aria-label="Open Scopus in new tab">Scopus (244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24113868/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnri3536&amp;pmid=24113868" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref94" title="View in article">96.</a></div><div>Lebrun, A. ∙ Portocarrero, C. ∙ Kean, R.B. ...</div><div><strong>T-bet Is Required for the Rapid Clearance of Attenuated Rabies Virus from Central Nervous System Tissue</strong></div><div><em>J.&nbsp;Immunol.</em> 2015; <strong>195</strong>:4358-4368</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1501274" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_96_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84945143808" target="_blank" aria-label="Open Scopus in new tab">Scopus (15)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26408670/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1501274&amp;pmid=26408670" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref95" title="View in article">97.</a></div><div>Levine, A.G. ∙ Mendoza, A. ∙ Hemmers, S. ...</div><div><strong>Stability and function of regulatory T&nbsp;cells expressing the transcription factor T-bet</strong></div><div><em>Nature.</em> 2017; <strong>546</strong>:421-425</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature22360" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85020935637" target="_blank" aria-label="Open Scopus in new tab">Scopus (128)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28607488/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature22360&amp;pmid=28607488" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">98.</a></div><div>Lewis, M.D. ∙ Miller, S.A. ∙ Miazgowicz, M.M. ...</div><div><strong>T-bet’s ability to regulate individual target genes requires the conserved T-box domain to recruit histone methyltransferase activity and a separate family member-specific transactivation domain</strong></div><div><em>Mol. Cell. Biol.</em> 2007; <strong>27</strong>:8510-8521</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1128/MCB.01615-07" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-37549032110" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17923685/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1128%2FMCB.01615-07&amp;pmid=17923685" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97-1" title="View in article">99.</a></div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (18663)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98-1" title="View in article">100.</a></div><div>Li, H. ∙ Handsaker, B. ∙ Wysoker, A. ..., 1000 Genome Project Data Processing Subgroup</div><div><strong>The Sequence Alignment/Map format and SAMtools</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:2078-2079</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp352" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_100_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-68549104404" target="_blank" aria-label="Open Scopus in new tab">Scopus (21003)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp352&amp;pmid=19505943" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99-1" title="View in article">101.</a></div><div>Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.</div><div><strong>featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:923-930</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btt656" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_101_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84897397058" target="_blank" aria-label="Open Scopus in new tab">Scopus (4007)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtt656&amp;pmid=24227677" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">102.</a></div><div>Liberzon, A. ∙ Birger, C. ∙ Thorvaldsdóttir, H. ...</div><div><strong>The Molecular Signatures Database (MSigDB) hallmark gene set collection</strong></div><div><em>Cell Syst.</em> 2015; <strong>1</strong>:417-425</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2015.12.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84955328286" target="_blank" aria-label="Open Scopus in new tab">Scopus (1439)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2015.12.004&amp;pmid=26771021" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">103.</a></div><div>Lim, A.I. ∙ Li, Y. ∙ Lopez-Lastra, S. ...</div><div><strong>Systemic Human ILC Precursors Provide a Substrate for Tissue ILC Differentiation</strong></div><div><em>Cell.</em> 2017; <strong>168</strong>:1086-1100</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2017.02.021&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2017.02.021&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85014819484" target="_blank" aria-label="Open Scopus in new tab">Scopus (195)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28283063/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2017.02.021&amp;pmid=28283063" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref102" title="View in article">104.</a></div><div>Lindestam Arlehamn, C.S. ∙ McKinney, D.M. ∙ Carpenter, C. ...</div><div><strong>A Quantitative Analysis of Complexity of Human Pathogen-Specific CD4&nbsp;T Cell Responses in Healthy M. tuberculosis Infected South Africans</strong></div><div><em>PLoS Pathog.</em> 2016; <strong>12</strong>:e1005760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.ppat.1005760" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84982943706" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27409590/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.ppat.1005760&amp;pmid=27409590" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref103" title="View in article">105.</a></div><div>Liu, L. ∙ Okada, S. ∙ Kong, X.-F. ...</div><div><strong>Gain-of-function human STAT1 mutations impair IL-17 immunity and underlie chronic mucocutaneous candidiasis</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2011; <strong>208</strong>:1635-1648</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20110958" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79961154447" target="_blank" aria-label="Open Scopus in new tab">Scopus (483)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21727188/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20110958&amp;pmid=21727188" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">106.</a></div><div>Locci, M. ∙ Draghici, E. ∙ Marangoni, F. ...</div><div><strong>The Wiskott-Aldrich syndrome protein is required for iNKT cell maturation and function</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2009; <strong>206</strong>:735-742</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20081773" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-65549150035" target="_blank" aria-label="Open Scopus in new tab">Scopus (43)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19307326/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20081773&amp;pmid=19307326" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">107.</a></div><div>Loetscher, P. ∙ Uguccioni, M. ∙ Bordoli, L. ...</div><div><strong>CCR5 is characteristic of Th1 lymphocytes</strong></div><div><em>Nature.</em> 1998; <strong>391</strong>:344-345</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/34814" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9450746/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F34814&amp;pmid=9450746" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106-1" title="View in article">108.</a></div><div>Love, M.I. ∙ Huber, W. ∙ Anders, S.</div><div><strong>Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</strong></div><div><em>Genome Biol.</em> 2014; <strong>15</strong>:550</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-014-0550-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84924629414" target="_blank" aria-label="Open Scopus in new tab">Scopus (14777)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-014-0550-8&amp;pmid=25516281" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref107" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref107" title="View in article">109.</a></div><div>Lugo-Villarino, G. ∙ Maldonado-Lopez, R. ∙ Possemato, R. ...</div><div><strong>T-bet is required for optimal production of IFN-gamma and antigen-specific T&nbsp;cell activation by dendritic cells</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:7749-7754</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1332767100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037934497" target="_blank" aria-label="Open Scopus in new tab">Scopus (194)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12802010/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1332767100&amp;pmid=12802010" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">110.</a></div><div>Lum, S.H. ∙ Neven, B. ∙ Slatter, M.A. ...</div><div><strong>Hematopoietic Cell Transplantation for MHC Class II Deficiency</strong></div><div><em>Front Pediatr.</em> 2019; <strong>7</strong>:516</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fped.2019.00516" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85077320039" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31921728/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffped.2019.00516&amp;pmid=31921728" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109" title="View in article">111.</a></div><div>Lum, S.H. ∙ Anderson, C. ∙ McNaughton, P. ...</div><div><strong>Improved transplant survival and long-term disease outcome in children with MHC class II deficiency</strong></div><div><em>Blood.</em> 2020; <strong>135</strong>:954-973</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood.2019002690" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31932845/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood.2019002690&amp;pmid=31932845" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110" title="View in article">112.</a></div><div>Lund, R. ∙ Ahlfors, H. ∙ Kainonen, E. ...</div><div><strong>Identification of genes involved in the initiation of human Th1 or Th2 cell commitment</strong></div><div><em>Eur. J. Immunol.</em> 2005; <strong>35</strong>:3307-3319</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/eji.200526079" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-27944450018" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16220538/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Feji.200526079&amp;pmid=16220538" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref111-1" title="View in article">113.</a></div><div>Ma, W. ∙ Noble, W.S. ∙ Bailey, T.L.</div><div><strong>Motif-based analysis of large nucleotide data sets using MEME-ChIP</strong></div><div><em>Nat. Protoc.</em> 2014; <strong>9</strong>:1428-1450</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nprot.2014.083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_113_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84901830680" target="_blank" aria-label="Open Scopus in new tab">Scopus (91)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24853928/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnprot.2014.083&amp;pmid=24853928" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">114.</a></div><div>Ma, C.S. ∙ Wong, N. ∙ Rao, G. ...</div><div><strong>Monogenic mutations differentially affect the quantity and quality of T follicular helper cells in patients with human primary immunodeficiencies</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2015; <strong>136</strong>:993-1006</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2015.05.036&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2015.05.036&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_114_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84943454615" target="_blank" aria-label="Open Scopus in new tab">Scopus (106)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26162572/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2015.05.036&amp;pmid=26162572" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">115.</a></div><div>Martínez-Barricarte, R. ∙ de Jong, S.J. ∙ Markle, J. ...</div><div><strong>Transduction of Herpesvirus saimiri-Transformed T Cells with Exogenous Genes of Interest</strong></div><div><em>Curr. Protoc. Immunol.</em> 2016; <strong>115</strong>:7.21C.1-7.21C.12</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cpim.15" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_115_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85029409873" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcpim.15" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref114" title="View in article">116.</a></div><div>Martínez-Barricarte, R. ∙ Markle, J.G. ∙ Ma, C.S. ...</div><div><strong>Human IFN-γ immunity to mycobacteria is governed by both IL-12 and IL-23</strong></div><div><em>Sci. Immunol.</em> 2018; <strong>3</strong>:eaau6759</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/sciimmunol.aau6759" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_116_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85057006182" target="_blank" aria-label="Open Scopus in new tab">Scopus (50)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30578351/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fsciimmunol.aau6759&amp;pmid=30578351" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">117.</a></div><div>Matsuyama, M. ∙ Ishii, Y. ∙ Yageta, Y. ...</div><div><strong>Role of Th1/Th17 balance regulated by T-bet in a mouse model of Mycobacterium avium complex disease</strong></div><div><em>J.&nbsp;Immunol.</em> 2014; <strong>192</strong>:1707-1717</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1302258" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_117_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84894237503" target="_blank" aria-label="Open Scopus in new tab">Scopus (12)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24446514/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1302258&amp;pmid=24446514" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116-1" title="View in article">118.</a></div><div>McGinnis, C.S. ∙ Murrow, L.M. ∙ Gartner, Z.J.</div><div><strong>DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors</strong></div><div><em>Cell Syst.</em> 2019; <strong>8</strong>:329-337</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cels.2019.03.003&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_118_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85064396876" target="_blank" aria-label="Open Scopus in new tab">Scopus (85)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cels.2019.03.003&amp;pmid=30954475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib117" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref117-1" title="View in article">119.</a></div><div>McKenna, A. ∙ Hanna, M. ∙ Banks, E. ...</div><div><strong>The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</strong></div><div><em>Genome Res.</em> 2010; <strong>20</strong>:1297-1303</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.107524.110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_119_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77956295988" target="_blank" aria-label="Open Scopus in new tab">Scopus (10353)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20644199/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.107524.110&amp;pmid=20644199" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib118" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118-1" title="View in article">120.</a></div><div>Miller, S.A. ∙ Weinmann, A.S.</div><div><strong>Molecular mechanisms by which T-bet regulates T-helper cell commitment</strong></div><div><em>Immunol. Rev.</em> 2010; <strong>238</strong>:233-246</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1600-065X.2010.00952.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_120_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77958610119" target="_blank" aria-label="Open Scopus in new tab">Scopus (53)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20969596/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1600-065X.2010.00952.x&amp;pmid=20969596" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib119" class="citations"><div id="sref119" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref119" title="View in article">121.</a></div><div>Modlin, R.L. ∙ Pirmez, C. ∙ Hofman, F.M. ...</div><div><strong>Lymphocytes bearing antigen-specific γ δ T-cell receptors accumulate in human infectious disease lesions</strong></div><div><em>Nature.</em> 1989; <strong>339</strong>:544-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/339544a0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2471938/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F339544a0&amp;pmid=2471938" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib120" class="citations"><div id="sref120" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref120" title="View in article">122.</a></div><div>Morgan, N.V. ∙ Goddard, S. ∙ Cardno, T.S. ...</div><div><strong>Mutation in the TCRα subunit constant gene (TRAC) leads to a human immunodeficiency disorder characterized by a lack of TCRαβ+ T&nbsp;cells</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2011; <strong>121</strong>:695-702</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI41931" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_122_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-79551544623" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21206088/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI41931&amp;pmid=21206088" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib121" class="citations"><div id="sref121" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref121" title="View in article">123.</a></div><div>Nathan, C.F. ∙ Murray, H.W. ∙ Wiebe, M.E. ...</div><div><strong>Identification of interferon-gamma as the lymphokine that activates human macrophage oxidative metabolism and antimicrobial activity</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1983; <strong>158</strong>:670-689</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.158.3.670" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_123_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0020502358" target="_blank" aria-label="Open Scopus in new tab">Scopus (1291)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/6411853/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.158.3.670&amp;pmid=6411853" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib122" class="citations"><div id="sref122" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref122" title="View in article">124.</a></div><div>Nishikomori, R. ∙ Usui, T. ∙ Wu, C.-Y. ...</div><div><strong>Activated STAT4 has an essential role in Th1 differentiation and proliferation that is independent of its role in the maintenance of IL-12R beta 2 chain expression and signaling</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:4388-4398</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.8.4388" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12370372/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.8.4388&amp;pmid=12370372" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib123" class="citations"><div id="sref123" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref123" title="View in article">125.</a></div><div>Oakley, M.S. ∙ Sahu, B.R. ∙ Lotspeich-Cole, L. ...</div><div><strong>The transcription factor T-bet regulates parasitemia and promotes pathogenesis during Plasmodium berghei ANKA murine malaria</strong></div><div><em>J.&nbsp;Immunol.</em> 2013; <strong>191</strong>:4699-4708</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1300396" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_125_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84886017430" target="_blank" aria-label="Open Scopus in new tab">Scopus (37)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24078698/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1300396&amp;pmid=24078698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib183" class="citations"><div id="sref183" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref183-1" title="View in article">126.</a></div><div>Ogishi, M.</div><div><strong>Multibatch Cytometry Data Integration for Optimal Immunophenotyping.</strong></div><div><em>J. Immunol.</em> 2021; <strong>206</strong></div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.2000854" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_126_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85097887715" target="_blank" aria-label="Open Scopus in new tab">Scopus (1)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/33229441/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.2000854&amp;pmid=33229441" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib124" class="citations"><div id="sref124" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref124-1" title="View in article">127.</a></div><div>Okada, S. ∙ Markle, J.G. ∙ Deenick, E.K. ...</div><div><strong>Impairment of immunity to Candida and Mycobacterium in humans with bi-allelic RORC mutations</strong></div><div><em>Science.</em> 2015; <strong>349</strong>:606-613</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa4282" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_127_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84939154723" target="_blank" aria-label="Open Scopus in new tab">Scopus (206)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26160376/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa4282&amp;pmid=26160376" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib125" class="citations"><div id="sref125" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref125" title="View in article">128.</a></div><div>Panchamoorthy, G. ∙ McLean, J. ∙ Modlin, R.L. ...</div><div><strong>A predominance of the T&nbsp;cell receptor V gamma 2/V delta 2 subset in human mycobacteria-responsive T&nbsp;cells suggests germline gene encoded recognition</strong></div><div><em>J.&nbsp;Immunol.</em> 1991; <strong>147</strong>:3360-3369</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/1658147/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=1658147" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib126" class="citations"><div id="sref126" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref126" title="View in article">129.</a></div><div>Parker, C.M. ∙ Groh, V. ∙ Band, H. ...</div><div><strong>Evidence for extrathymic changes in the T&nbsp;cell receptor gamma/delta repertoire</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1990; <strong>171</strong>:1597-1612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.171.5.1597" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/2185330/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.171.5.1597&amp;pmid=2185330" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib127" class="citations"><div id="sref127" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref127" title="View in article">130.</a></div><div>Pearce, E.L. ∙ Mullen, A.C. ∙ Martins, G.A. ...</div><div><strong>Control of Effector CD8+ T Cell Function by the Transcription Factor Eomesodermin</strong></div><div><em>Science.</em> 2003; <strong>302</strong>:1041-1043</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1090148" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_130_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0242410712" target="_blank" aria-label="Open Scopus in new tab">Scopus (652)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14605368/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1090148&amp;pmid=14605368" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib128" class="citations"><div id="sref128" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref128" title="View in article">131.</a></div><div>Petrovski, S. ∙ Gussow, A.B. ∙ Wang, Q. ...</div><div><strong>The Intolerance of Regulatory Sequence to Genetic Variation Predicts Gene Dosage Sensitivity</strong></div><div><em>PLoS Genet.</em> 2015; <strong>11</strong>:e1005492</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1005492" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26332131/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1005492&amp;pmid=26332131" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib129" class="citations"><div id="sref129" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref129" title="View in article">132.</a></div><div>Powell, N. ∙ Walker, A.W. ∙ Stolarczyk, E. ...</div><div><strong>The transcription factor T-bet regulates intestinal inflammation mediated by interleukin-7 receptor+ innate lymphoid cells</strong></div><div><em>Immunity.</em> 2012; <strong>37</strong>:674-684</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2012.09.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2012.09.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_132_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84867856710" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23063332/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2012.09.008&amp;pmid=23063332" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib130" class="citations"><div id="sref130" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref130-1" title="View in article">133.</a></div><div>Ramírez, F. ∙ Ryan, D.P. ∙ Grüning, B. ...</div><div><strong>deepTools2: a next generation web server for deep-sequencing data analysis</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:W160-5</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkw257" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_133_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85017133693" target="_blank" aria-label="Open Scopus in new tab">Scopus (955)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkw257&amp;pmid=27079975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib131" class="citations"><div id="sref131" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref131" title="View in article">134.</a></div><div>Rankin, L.C. ∙ Groom, J.R. ∙ Chopin, M. ...</div><div><strong>The transcription factor T-bet is essential for the development of NKp46+ innate lymphocytes via the Notch pathway</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:389-395</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2545" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_134_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84875445419" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23455676/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2545&amp;pmid=23455676" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib132" class="citations"><div id="sref132" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref132" title="View in article">135.</a></div><div>Ravindran, R. ∙ Foley, J. ∙ Stoklasek, T. ...</div><div><strong>Expression of T-bet by CD4 T&nbsp;cells is essential for resistance to Salmonella infection</strong></div><div><em>J.&nbsp;Immunol.</em> 2005; <strong>175</strong>:4603-4610</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.175.7.4603" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_135_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-25444439804" target="_blank" aria-label="Open Scopus in new tab">Scopus (113)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16177105/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.175.7.4603&amp;pmid=16177105" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib133" class="citations"><div id="sref133" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref133" title="View in article">136.</a></div><div>Revy, P. ∙ Kannengiesser, C. ∙ Fischer, A.</div><div><strong>Somatic genetic rescue in Mendelian haematopoietic diseases</strong></div><div><em>Nat. Rev. Genet.</em> 2019; <strong>20</strong>:582-598</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41576-019-0139-x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_136_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85067013592" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31186537/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41576-019-0139-x&amp;pmid=31186537" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib134" class="citations"><div id="sref134" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref134-1" title="View in article">137.</a></div><div>Robinson, M.D. ∙ McCarthy, D.J. ∙ Smyth, G.K.</div><div><strong>edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:139-140</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp616" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_137_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-75249087100" target="_blank" aria-label="Open Scopus in new tab">Scopus (12880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp616&amp;pmid=19910308" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib135" class="citations"><div id="sref135" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref135" title="View in article">138.</a></div><div>Robinson, J.T. ∙ Thorvaldsdóttir, H. ∙ Winckler, W. ...</div><div><strong>Integrative genomics viewer</strong></div><div><em>Nat. Biotechnol.</em> 2011; <strong>29</strong>:24-26</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nbt.1754" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_138_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-78651271733" target="_blank" aria-label="Open Scopus in new tab">Scopus (5120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21221095/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnbt.1754&amp;pmid=21221095" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib136" class="citations"><div id="sref136" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref136" title="View in article">139.</a></div><div>Rosain, J. ∙ Kong, X.F. ∙ Martinez-Barricarte, R. ...</div><div><strong>Mendelian susceptibility to mycobacterial disease: 2014-2018 update</strong></div><div><em>Immunol. Cell Biol.</em> 2019; <strong>97</strong>:360-367</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/imcb.12210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_139_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85055711153" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30264912/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fimcb.12210&amp;pmid=30264912" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib137" class="citations"><div id="sref137" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref137" title="View in article">140.</a></div><div>Rosas, L.E. ∙ Snider, H.M. ∙ Barbi, J. ...</div><div><strong>Cutting edge: STAT1 and T-bet play distinct roles in determining outcome of visceral leishmaniasis caused by Leishmania donovani</strong></div><div><em>J.&nbsp;Immunol.</em> 2006; <strong>177</strong>:22-25</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.177.1.22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_140_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33745305544" target="_blank" aria-label="Open Scopus in new tab">Scopus (41)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16785492/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.177.1.22&amp;pmid=16785492" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib138" class="citations"><div id="sref138" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref138" title="View in article">141.</a></div><div>Saleem, M.A. ∙ Arkwright, P.D. ∙ Davies, E.G. ...</div><div><strong>Clinical course of patients with major histocompatibility complex class II deficiency</strong></div><div><em>Arch. Dis. Child.</em> 2000; <strong>83</strong>:356-359</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1136/adc.83.4.356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_141_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0033807808" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10999878/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1136%2Fadc.83.4.356&amp;pmid=10999878" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib139" class="citations"><div id="sref139" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref139" title="View in article">142.</a></div><div>Sallusto, F. ∙ Lenig, D. ∙ Mackay, C.R. ...</div><div><strong>Flexible programs of chemokine receptor expression on human polarized T helper 1 and 2 lymphocytes</strong></div><div><em>J.&nbsp;Exp. Med.</em> 1998; <strong>187</strong>:875-883</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.187.6.875" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_142_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0032536869" target="_blank" aria-label="Open Scopus in new tab">Scopus (1325)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9500790/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.187.6.875&amp;pmid=9500790" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib140" class="citations"><div id="sref140" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref140-1" title="View in article">143.</a></div><div>Sanjana, N.E. ∙ Shalem, O. ∙ Zhang, F.</div><div><strong>Improved vectors and genome-wide libraries for CRISPR screening</strong></div><div><em>Nat. Methods.</em> 2014; <strong>11</strong>:783-784</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.3047" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_143_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84905262730" target="_blank" aria-label="Open Scopus in new tab">Scopus (1702)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25075903/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.3047&amp;pmid=25075903" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib141" class="citations"><div id="sref141" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref141-1" title="View in article">144.</a></div><div>Schoenborn, J.R. ∙ Wilson, C.B.</div><div><strong>Regulation of interferon-γ during innate and adaptive immune responses</strong></div><div><em>Adv. Immunol.</em> 2007; <strong>96</strong>:41-101</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1016/S0065-2776(07)96002-2" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_144_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-35549003579" target="_blank" aria-label="Open Scopus in new tab">Scopus (968)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17981204/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0065-2776%2807%2996002-2&amp;pmid=17981204" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib142" class="citations"><div id="sref142" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref142" title="View in article">145.</a></div><div>Scott, E.M. ∙ Halees, A. ∙ Itan, Y. ..., Greater Middle East Variome Consortium</div><div><strong>Characterization of Greater Middle Eastern genetic variation for enhanced disease gene discovery</strong></div><div><em>Nat. Genet.</em> 2016; <strong>48</strong>:1071-1076</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3592" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_145_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84978705937" target="_blank" aria-label="Open Scopus in new tab">Scopus (197)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27428751/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3592&amp;pmid=27428751" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib143" class="citations"><div id="sref143" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref143" title="View in article">146.</a></div><div>Shen, Y. ∙ Zhou, D. ∙ Qiu, L. ...</div><div><strong>Adaptive immune response of Vgamma2Vdelta2+ T&nbsp;cells during mycobacterial infections</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:2255-2258</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1068819" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_146_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037155586" target="_blank" aria-label="Open Scopus in new tab">Scopus (285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11910108/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1068819&amp;pmid=11910108" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib144" class="citations"><div id="sref144" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref144-1" title="View in article">147.</a></div><div>Soutto, M. ∙ Zhang, F. ∙ Enerson, B. ...</div><div><strong>A minimal IFN-gamma promoter confers Th1 selective expression</strong></div><div><em>J.&nbsp;Immunol.</em> 2002; <strong>169</strong>:4205-4212</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.169.8.4205" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/12370350/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.169.8.4205&amp;pmid=12370350" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib145" class="citations"><div id="sref145" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref145-1" title="View in article">148.</a></div><div>Stuart, T. ∙ Butler, A. ∙ Hoffman, P. ...</div><div><strong>Comprehensive Integration of Single-Cell Data</strong></div><div><em>Cell.</em> 2019; <strong>177</strong>:1888-1902</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.cell.2019.05.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_148_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85066448459" target="_blank" aria-label="Open Scopus in new tab">Scopus (766)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.05.031&amp;pmid=31178118" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib146" class="citations"><div id="sref146" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref146-1" title="View in article">149.</a></div><div>Sullivan, B.M. ∙ Juedes, A. ∙ Szabo, S.J. ...</div><div><strong>Antigen-driven effector CD8 T&nbsp;cell function regulated by T-bet</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2003; <strong>100</strong>:15818-15823</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.2636938100" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_149_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0347994894" target="_blank" aria-label="Open Scopus in new tab">Scopus (289)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14673093/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2636938100&amp;pmid=14673093" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib147" class="citations"><div id="sref147" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref147" title="View in article">150.</a></div><div>Sullivan, B.M. ∙ Jobe, O. ∙ Lazarevic, V. ...</div><div><strong>Increased susceptibility of mice lacking T-bet to infection with Mycobacterium tuberculosis correlates with increased IL-10 and decreased IFN-gamma production</strong></div><div><em>J.&nbsp;Immunol.</em> 2005; <strong>175</strong>:4593-4602</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.175.7.4593" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16177104/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.175.7.4593&amp;pmid=16177104" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib148" class="citations"><div id="sref148" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref148" title="View in article">151.</a></div><div>Svensson, A. ∙ Nordström, I. ∙ Sun, J.B. ...</div><div><strong>Protective immunity to genital herpes simplex [correction of simpex] virus type 2 infection is mediated by T-bet</strong></div><div><em>J.&nbsp;Immunol.</em> 2005; <strong>174</strong>:6266-6273</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.174.10.6266" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_151_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-18744373400" target="_blank" aria-label="Open Scopus in new tab">Scopus (42)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15879125/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.174.10.6266&amp;pmid=15879125" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib149" class="citations"><div id="sref149" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref149-1" title="View in article">152.</a></div><div>Szabo, S.J. ∙ Kim, S.T. ∙ Costa, G.L. ...</div><div><strong>A novel transcription factor, T-bet, directs Th1 lineage commitment</strong></div><div><em>Cell.</em> 2000; <strong>100</strong>:655-669</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_152_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980702-3&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/10761931/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980702-3&amp;pmid=10761931" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib150" class="citations"><div id="sref150" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref150-1" title="View in article">153.</a></div><div>Szabo, S.J. ∙ Sullivan, B.M. ∙ Stemmann, C. ...</div><div><strong>Distinct effects of T-bet in TH1 lineage commitment and IFN-gamma production in CD4 and CD8 T&nbsp;cells</strong></div><div><em>Science.</em> 2002; <strong>295</strong>:338-342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1065543" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_153_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-0037059462" target="_blank" aria-label="Open Scopus in new tab">Scopus (920)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11786644/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1065543&amp;pmid=11786644" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib151" class="citations"><div id="sref151" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref151" title="View in article">154.</a></div><div>Taliun, D. ∙ Harris, D.N. ∙ Kessler, M.D. ...</div><div><strong>Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program</strong></div><div><em>BioRxiv.</em> 2019; 563866</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.TaliunD.N.HarrisM.D.KesslerJ.CarlsonZ.A.SzpiechR.TorresS.A.G.TaliunA.CorveloS.M.GogartenH.M.KangSequencing+of+53%2C831+diverse+genomes+from+the+NHLBI+TOPMed+ProgramBioRxiv2019563866" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib152" class="citations"><div id="sref152" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref152" title="View in article">155.</a></div><div>Tan, T.G. ∙ Mathis, D. ∙ Benoist, C.</div><div><strong>Singular role for T-BET+CXCR3+ regulatory T&nbsp;cells in protection from autoimmune diabetes</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:14103-14108</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1616710113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_155_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85002497847" target="_blank" aria-label="Open Scopus in new tab">Scopus (36)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27872297/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1616710113&amp;pmid=27872297" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib153" class="citations"><div id="sref153" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref153" title="View in article">156.</a></div><div>Tang, Y. ∙ Desierto, M.J. ∙ Chen, J. ...</div><div><strong>The role of the Th1 transcription factor T-bet in a mouse model of immune-mediated bone-marrow failure</strong></div><div><em>Blood.</em> 2010; <strong>115</strong>:541-548</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2009-03-211383" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_156_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77449092131" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19903901/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2009-03-211383&amp;pmid=19903901" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib154" class="citations"><div id="sref154" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref154" title="View in article">157.</a></div><div>Tangye, S.G. ∙ Palendira, U. ∙ Edwards, E.S.J.</div><div><strong>Human immunity against EBV-lessons from the clinic</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2017; <strong>214</strong>:269-283</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20161846" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_157_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85012292927" target="_blank" aria-label="Open Scopus in new tab">Scopus (68)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28108590/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20161846&amp;pmid=28108590" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib155" class="citations"><div id="sref155" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref155" title="View in article">158.</a></div><div>Tantisira, K.G. ∙ Hwang, E.S. ∙ Raby, B.A. ...</div><div><strong>TBX21: a functional variant predicts improvement in asthma with the use of inhaled corticosteroids</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2004; <strong>101</strong>:18099-18104</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0408532102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_158_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-19944427616" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15604153/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0408532102&amp;pmid=15604153" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib156" class="citations"><div id="sref156" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref156" title="View in article">159.</a></div><div>Thieu, V.T. ∙ Yu, Q. ∙ Chang, H.-C. ...</div><div><strong>Signal transducer and activator of transcription 4 is required for the transcription factor T-bet to promote T helper 1 cell-fate determination</strong></div><div><em>Immunity.</em> 2008; <strong>29</strong>:679-690</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2008.08.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2008.08.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_159_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-55349117409" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18993086/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2008.08.017&amp;pmid=18993086" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib157" class="citations"><div id="sref157" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref157-1" title="View in article">160.</a></div><div>Tong, Y. ∙ Aune, T. ∙ Boothby, M.</div><div><strong>T-bet antagonizes mSin3a recruitment and transactivates a fully methylated IFN-gamma promoter via a conserved T-box half-site</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2005; <strong>102</strong>:2034-2039</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.0409510102" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_160_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-13844320399" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15684083/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.0409510102&amp;pmid=15684083" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib158" class="citations"><div id="sref158" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref158" title="View in article">161.</a></div><div>Toribio-Fernández, R. ∙ Zorita, V. ∙ Rocha-Perugini, V. ...</div><div><strong>Lamin A/C augments Th1 differentiation and response against vaccinia virus and Leishmania major</strong></div><div><em>Cell Death Dis.</em> 2018; <strong>9</strong>:9</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41419-017-0007-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_161_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85042799473" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29311549/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41419-017-0007-6&amp;pmid=29311549" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib159" class="citations"><div id="sref159" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref159" title="View in article">162.</a></div><div>Townsend, M.J. ∙ Weinmann, A.S. ∙ Matsuda, J.L. ...</div><div><strong>T-bet regulates the terminal maturation and homeostasis of NK and Valpha14i NKT cells</strong></div><div><em>Immunity.</em> 2004; <strong>20</strong>:477-494</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS1074-7613%2804%2900076-7&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS1074-7613%2804%2900076-7&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_162_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-1842785209" target="_blank" aria-label="Open Scopus in new tab">Scopus (524)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15084276/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS1074-7613%2804%2900076-7&amp;pmid=15084276" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib160" class="citations"><div id="sref160" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref160" title="View in article">163.</a></div><div>Tsukaguchi, K. ∙ Balaji, K.N. ∙ Boom, W.H.</div><div><strong>CD4+ alpha beta T&nbsp;cell and gamma delta T&nbsp;cell responses to Mycobacterium tuberculosis. Similarities and differences in Ag recognition, cytotoxic effector function, and cytokine production</strong></div><div><em>J.&nbsp;Immunol.</em> 1995; <strong>154</strong>:1786-1796</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7836763/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=7836763" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib161" class="citations"><div id="sref161" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref161-1" title="View in article">164.</a></div><div>van de Vosse, E. ∙ Verhard, E.M. ∙ de Paus, R.A. ...</div><div><strong>Antisense-mediated exon skipping to correct IL-12Rbeta1 deficiency in T&nbsp;cells</strong></div><div><em>Blood.</em> 2009; <strong>113</strong>:4548-4555</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1182/blood-2008-12-196220" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_164_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-66549092636" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19258592/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1182%2Fblood-2008-12-196220&amp;pmid=19258592" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib162" class="citations"><div id="sref162" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref162-1" title="View in article">165.</a></div><div>Van Gassen, S. ∙ Callebaut, B. ∙ Van Helden, M.J. ...</div><div><strong>FlowSOM: Using self-organizing maps for visualization and interpretation of cytometry data</strong></div><div><em>Cytometry A.</em> 2015; <strong>87</strong>:636-645</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/cyto.a.22625" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_165_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84932198501" target="_blank" aria-label="Open Scopus in new tab">Scopus (278)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25573116/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fcyto.a.22625&amp;pmid=25573116" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib163" class="citations"><div id="sref163" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref163-1" title="View in article">166.</a></div><div>Vavassori, S. ∙ Kumar, A. ∙ Wan, G.S. ...</div><div><strong>Butyrophilin 3A1 binds phosphorylated antigens and stimulates human γδ T&nbsp;cells</strong></div><div><em>Nat. Immunol.</em> 2013; <strong>14</strong>:908-916</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ni.2665" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_166_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84883147590" target="_blank" aria-label="Open Scopus in new tab">Scopus (242)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23872678/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fni.2665&amp;pmid=23872678" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib164" class="citations"><div id="sref164" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref164" title="View in article">167.</a></div><div>Velu, V. ∙ Mylvaganam, G.H. ∙ Gangadhara, S. ...</div><div><strong>Induction of Th1-Biased T Follicular Helper (Tfh) Cells in Lymphoid Tissues during Chronic Simian Immunodeficiency Virus Infection Defines Functionally Distinct Germinal Center Tfh Cells</strong></div><div><em>J.&nbsp;Immunol.</em> 2016; <strong>197</strong>:1832-1842</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.4049/jimmunol.1600143" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_167_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84983776546" target="_blank" aria-label="Open Scopus in new tab">Scopus (56)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27481845/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.4049%2Fjimmunol.1600143&amp;pmid=27481845" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib165" class="citations"><div id="sref165" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref165-1" title="View in article">168.</a></div><div>Vogt, G. ∙ Nathan, C.</div><div><strong>In&nbsp;vitro differentiation of human macrophages with enhanced antimycobacterial activity</strong></div><div><em>J.&nbsp;Clin. Invest.</em> 2011; <strong>121</strong>:3889-3901</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1172/JCI57235" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_168_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-80053413551" target="_blank" aria-label="Open Scopus in new tab">Scopus (62)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21911939/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1172%2FJCI57235&amp;pmid=21911939" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib166" class="citations"><div id="sref166" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref166-1" title="View in article">169.</a></div><div>Wang, L. ∙ Wang, S. ∙ Li, W.</div><div><strong>RSeQC: quality control of RNA-seq experiments</strong></div><div><em>Bioinformatics.</em> 2012; <strong>28</strong>:2184-2185</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bts356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_169_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84865074382" target="_blank" aria-label="Open Scopus in new tab">Scopus (721)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22743226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbts356&amp;pmid=22743226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib167" class="citations"><div id="sref167" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref167" title="View in article">170.</a></div><div>Wang, Y. ∙ Ma, C.S. ∙ Ling, Y. ...</div><div><strong>Dual T&nbsp;cell- and B cell-intrinsic deficiency in humans with biallelic RLTPR mutations</strong></div><div><em>J.&nbsp;Exp. Med.</em> 2016; <strong>213</strong>:2413-2435</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1084/jem.20160576" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_170_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84994667641" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27647349/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1084%2Fjem.20160576&amp;pmid=27647349" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib168" class="citations"><div id="sref168" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref168-1" title="View in article">171.</a></div><div>Wilkerson, M.D. ∙ Hayes, D.N.</div><div><strong>ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:1572-1573</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq170" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_171_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77954182362" target="_blank" aria-label="Open Scopus in new tab">Scopus (544)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20427518/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq170&amp;pmid=20427518" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib169" class="citations"><div id="sref169" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref169" title="View in article">172.</a></div><div>Xiao, X. ∙ Cai, J.</div><div><strong>Mucosal-associated invariant T&nbsp;cells: New insights into antigen recognition and activation</strong></div><div><em>Front. Immunol.</em> 2017; <strong>8</strong>:1540</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.3389/fimmu.2017.01540" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_172_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85034088916" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29176983/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.3389%2Ffimmu.2017.01540&amp;pmid=29176983" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib170" class="citations"><div id="sref170" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref170-1" title="View in article">173.</a></div><div>Xu, G.J. ∙ Kula, T. ∙ Xu, Q. ...</div><div><strong>Comprehensive serological profiling of human populations using a synthetic human virome</strong></div><div><em>Science.</em> 2015; <strong>348</strong></div><div>aaa0698–aaa0698</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aaa0698" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_173_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84930635082" target="_blank" aria-label="Open Scopus in new tab">Scopus (135)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aaa0698" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib171" class="citations"><div id="sref171" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref171" title="View in article">174.</a></div><div>Yoshitomi, H. ∙ Kobayashi, S. ∙ Miyagawa-Hayashino, A. ...</div><div><strong>Human Sox4 facilitates the development of CXCL13-producing helper T&nbsp;cells in inflammatory environments</strong></div><div><em>Nat. Commun.</em> 2018; <strong>9</strong>:3762</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41467-018-06187-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_174_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85053538844" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30232328/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41467-018-06187-0&amp;pmid=30232328" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib172" class="citations"><div id="sref172" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref172" title="View in article">175.</a></div><div>Yu, J. ∙ Wei, M. ∙ Becknell, B. ...</div><div><strong>Pro- and antiinflammatory cytokine signaling: reciprocal antagonism regulates interferon-gamma production by human natural killer cells</strong></div><div><em>Immunity.</em> 2006; <strong>24</strong>:575-590</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2006.03.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.immuni.2006.03.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_175_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-33646563695" target="_blank" aria-label="Open Scopus in new tab">Scopus (141)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16713975/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.immuni.2006.03.016&amp;pmid=16713975" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib173" class="citations"><div id="sref173" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref173-1" title="View in article">176.</a></div><div>Zhang, Y. ∙ Liu, T. ∙ Meyer, C.A. ...</div><div><strong>Model-based analysis of ChIP-Seq (MACS)</strong></div><div><em>Genome Biol.</em> 2008; <strong>9</strong>:R137</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/gb-2008-9-9-r137" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_176_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-53849146020" target="_blank" aria-label="Open Scopus in new tab">Scopus (5850)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fgb-2008-9-9-r137&amp;pmid=18798982" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib174" class="citations"><div id="sref174" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref174" title="View in article">177.</a></div><div>Zhang, X. ∙ Bogunovic, D. ∙ Payelle-Brogard, B. ...</div><div><strong>Human intracellular ISG15 prevents interferon-α/β over-amplification and auto-inflammation</strong></div><div><em>Nature.</em> 2015; <strong>517</strong>:89-93</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13801" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_177_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-84922880395" target="_blank" aria-label="Open Scopus in new tab">Scopus (241)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25307056/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13801&amp;pmid=25307056" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib175" class="citations"><div id="sref175" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref175-1" title="View in article">178.</a></div><div>Zhang, P. ∙ Bigio, B. ∙ Rapaport, F. ...</div><div><strong>PopViz: a webserver for visualizing minor allele frequencies and damage prediction scores of human genetic variations</strong></div><div><em>Bioinformatics.</em> 2018; <strong>34</strong>:4307-4309</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/bty536" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_178_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85058604691" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30535305/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbty536&amp;pmid=30535305" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib176" class="citations"><div id="sref176" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref176" title="View in article">179.</a></div><div>Zhang, J. ∙ Liu, W. ∙ Wen, B. ...</div><div><strong>Circulating CXCR3<sup>+</sup> Tfh cells positively correlate with neutralizing antibody responses in HCV-infected patients</strong></div><div><em>Sci. Rep.</em> 2019; <strong>9</strong>:10090</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41598-019-46533-w" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_179_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-85068921679" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31300682/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41598-019-46533-w&amp;pmid=31300682" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib177" class="citations"><div id="sref177" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref177" title="View in article">180.</a></div><div>Zhang, Y. ∙ Siegel, A.M. ∙ Sun, G. ...</div><div><strong>Human T<sub>H</sub>9 differentiation is dependent on signal transducer and activator of transcription (STAT) 3 to restrain STAT1-mediated inhibition</strong></div><div><em>J.&nbsp;Allergy Clin. Immunol.</em> 2019; <strong>143</strong>:1108-1118</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2018.06.036&amp;cf=fulltext&amp;site=ymai-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_180_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2Fj.jaci.2018.06.036&amp;cf=pdf&amp;site=ymai-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30030006/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaci.2018.06.036&amp;pmid=30030006" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib178" class="citations"><div id="sref178" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref178" title="View in article">181.</a></div><div>Zheng, W. ∙ Flavell, R.A.</div><div><strong>The transcription factor GATA-3 is necessary and sufficient for Th2 cytokine gene expression in CD4 T&nbsp;cells</strong></div><div><em>Cell.</em> 1997; <strong>89</strong>:587-596</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980240-8&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_181_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=10.1016%2FS0092-8674%2800%2980240-8&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/9160750/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0092-8674%2800%2980240-8&amp;pmid=9160750" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib179" class="citations"><div id="sref179" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref179" title="View in article">182.</a></div><div>Zhu, J. ∙ Yamane, H. ∙ Paul, W.E.</div><div><strong>Differentiation of effector CD4 T&nbsp;cell populations (<sup>∗</sup>)</strong></div><div><em>Annu. Rev. Immunol.</em> 2010; <strong>28</strong>:445-489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1146/annurev-immunol-030409-101212" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_182_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-77952313777" target="_blank" aria-label="Open Scopus in new tab">Scopus (1795)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20192806/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1146%2Fannurev-immunol-030409-101212&amp;pmid=20192806" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib180" class="citations"><div id="sref180" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref180" title="View in article">183.</a></div><div>Zimmer, J. ∙ Andrès, E. ∙ Donato, L. ...</div><div><strong>Clinical and immunological aspects of HLA class I deficiency</strong></div><div><em>QJM.</em> 2005; <strong>98</strong>:719-727</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/qjmed/hci112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_183_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2020.10.046&amp;key=2-s2.0-27644538518" target="_blank" aria-label="Open Scopus in new tab">Scopus (71)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/16087697/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fqjmed%2Fhci112&amp;pmid=16087697" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (12)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/822188d1-872a-4a66-b301-156310189f10/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/0c9d586f-71c8-4bbd-8854-0d2e8bfec582/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure 2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/5400cb79-17d1-41bd-ac4b-3f567d284df6/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/a769eb99-98be-4102-8f0f-94fe9d10f907/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure 3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/f5e1b6b9-3d3d-4216-90d7-765e795511fe/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure 4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/90192588-803d-4fde-9096-783211eceae3/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/5124747d-482f-42db-951d-88fcfc5e1cfa/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/c135f75b-ee81-495e-9266-0cd27200b920/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure 5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/d8680554-2771-4510-ad79-25392b88fbc9/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/6698186b-bcde-442b-9ff4-e87e7067b345/main.assets/figs5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig6" aria-label="Go to Figure 6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/abe52ef8-1df5-4303-a9e5-354e3381677f/main.assets/gr6.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs6" aria-label="Go to Figure S6"><img src="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/asset/1e40b6ed-39e0-4984-a517-4d4bc57a9c16/main.assets/figs6.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867420314537%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00276%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">100</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">187</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">3</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">2</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2020.10.046&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (3)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="pdf"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/attachment/6419240d-687a-45d2-91ce-9b13a3a1ed61/mmc1.pdf" download="mmc1.pdf"><span id="mmc1-link">PDF (4.14 MB)</span></a><div class="core-description">Document S1. Tables S1, S4, and S5 and Data S1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/attachment/cebc24df-c879-4c00-9806-3dc74d665451/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (1.36 MB)</span></a><div class="core-description">Table S2. RNA-Seq Analysis of Expanded CD4 T<sub>H</sub>0 Cells from Healthy Donors (CTL), a T-bet Heterozygous parent (WT/M), the T-bet Homozygous P (M/M), P Complemented with Empty Vector (M/M+EV), or P complemented with WT T-bet (M/M+WT), Related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2020.10.046/attachment/fc84ce93-3b2e-43be-893d-78907758c096/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (3.84 MB)</span></a><div class="core-description">Table S3. Chromatin Accessibility of Loci Differentially Regulated in a T-bet-Dependent Manner and Beta-Value for the Differential Methylation of CpG Sites between T-bet m/m (M/M) and Controls (CTL) Abolished by WT T-bet Complementation, Related to Figure&nbsp;3</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867420314537" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2020.10.046","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Human T-bet Governs Innate and Innate-like Adaptive IFN-γ Immunity against Mycobacteria</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.10.046/asset/822188d1-872a-4a66-b301-156310189f10/main.assets/figs1.jpg" height="910" width="718" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/5d665e17-0534-45e2-a940-67c414b718f4/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.10.046/asset/0c9d586f-71c8-4bbd-8854-0d2e8bfec582/main.assets/gr1.jpg" height="949" width="720" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/f4effde4-1ebd-4886-a33a-bc94f589a951/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.10.046/asset/5400cb79-17d1-41bd-ac4b-3f567d284df6/main.assets/gr2.jpg" height="791" width="759" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/db795497-4517-4945-851a-73b50d5162ee/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.10.046/asset/a769eb99-98be-4102-8f0f-94fe9d10f907/main.assets/figs2.jpg" height="927" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/94800d20-0495-4a68-94e9-6602d0b7688a/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.10.046/asset/f5e1b6b9-3d3d-4216-90d7-765e795511fe/main.assets/gr3.jpg" height="949" width="684" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/fe99329d-681f-4fb0-ad79-9bb8bf766810/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.10.046/asset/90192588-803d-4fde-9096-783211eceae3/main.assets/gr4.jpg" height="922" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/3095b237-a2e6-4496-a46a-4ddec6e3c70d/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.10.046/asset/5124747d-482f-42db-951d-88fcfc5e1cfa/main.assets/figs3.jpg" height="948" width="729" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/5c417311-5be6-4294-9830-dadfd57935ca/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.10.046/asset/c135f75b-ee81-495e-9266-0cd27200b920/main.assets/figs4.jpg" height="886" width="696" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/510bbbaf-b9ba-4228-b615-3fd0629407ba/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.10.046/asset/d8680554-2771-4510-ad79-25392b88fbc9/main.assets/gr5.jpg" height="937" width="714" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/a8408118-2af8-4ac3-a800-fabf5e4610e4/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.10.046/asset/6698186b-bcde-442b-9ff4-e87e7067b345/main.assets/figs5.jpg" height="949" width="766" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/2bc74197-3a6e-4e91-9406-1dd94da80393/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.10.046/asset/abe52ef8-1df5-4303-a9e5-354e3381677f/main.assets/gr6.jpg" height="937" width="689" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/01baddba-5c1c-4178-aa63-1e818c14be72/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.10.046/asset/1e40b6ed-39e0-4984-a517-4d4bc57a9c16/main.assets/figs6.jpg" height="878" width="759" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/ecdb1012-6fb2-4219-ba9b-b4baa7c0a6c6/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2020.10.046/asset/822188d1-872a-4a66-b301-156310189f10/main.assets/figs1.jpg" height="910" width="718" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/5d665e17-0534-45e2-a940-67c414b718f4/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2020.10.046/asset/0c9d586f-71c8-4bbd-8854-0d2e8bfec582/main.assets/gr1.jpg" height="949" width="720" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/f4effde4-1ebd-4886-a33a-bc94f589a951/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2020.10.046/asset/5400cb79-17d1-41bd-ac4b-3f567d284df6/main.assets/gr2.jpg" height="791" width="759" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/db795497-4517-4945-851a-73b50d5162ee/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2020.10.046/asset/a769eb99-98be-4102-8f0f-94fe9d10f907/main.assets/figs2.jpg" height="927" width="750" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/94800d20-0495-4a68-94e9-6602d0b7688a/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2020.10.046/asset/f5e1b6b9-3d3d-4216-90d7-765e795511fe/main.assets/gr3.jpg" height="949" width="684" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/fe99329d-681f-4fb0-ad79-9bb8bf766810/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2020.10.046/asset/90192588-803d-4fde-9096-783211eceae3/main.assets/gr4.jpg" height="922" width="761" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/3095b237-a2e6-4496-a46a-4ddec6e3c70d/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2020.10.046/asset/5124747d-482f-42db-951d-88fcfc5e1cfa/main.assets/figs3.jpg" height="948" width="729" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/5c417311-5be6-4294-9830-dadfd57935ca/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2020.10.046/asset/c135f75b-ee81-495e-9266-0cd27200b920/main.assets/figs4.jpg" height="886" width="696" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/510bbbaf-b9ba-4228-b615-3fd0629407ba/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2020.10.046/asset/d8680554-2771-4510-ad79-25392b88fbc9/main.assets/gr5.jpg" height="937" width="714" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/a8408118-2af8-4ac3-a800-fabf5e4610e4/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2020.10.046/asset/6698186b-bcde-442b-9ff4-e87e7067b345/main.assets/figs5.jpg" height="949" width="766" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/2bc74197-3a6e-4e91-9406-1dd94da80393/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure 6</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr6" src="/cms/10.1016/j.cell.2020.10.046/asset/abe52ef8-1df5-4303-a9e5-354e3381677f/main.assets/gr6.jpg" height="937" width="689" aria-labelledby="fig6-title fig6-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/01baddba-5c1c-4178-aa63-1e818c14be72/main.assets/gr6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure S6</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs6" src="/cms/10.1016/j.cell.2020.10.046/asset/1e40b6ed-39e0-4984-a517-4d4bc57a9c16/main.assets/figs6.jpg" height="878" width="759" aria-labelledby="figs6-title figs6-content" data-viewer-src="/cms/10.1016/j.cell.2020.10.046/asset/ecdb1012-6fb2-4219-ba9b-b4baa7c0a6c6/main.assets/figs6_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)31453-7';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(20)31453-7';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749588653608","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"183","issueNumber":"7","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867420314537","coverDate":"202012","type":"XOCS-JOURNAL:SCOPE-FULL","title":"Human Tbet Governs Innate and Innatelike Adaptive IFNx3b3 Immunity against Mycobacteria","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94dbbe555bd04eef',t:'MTc0OTU4ODY1Mi4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94dbbe555bd04eef&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2020.10.046&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>