<html lang="en" class="pb-page fonts-loaded fonts-loaded" data-request-id="9b6b84c9-bdf2-4c53-91b7-d7dca1490051" style="--article-sticky-top: 82.00019999999999px; --scroll-padding: 163.594px auto auto;"><head data-pb-dropzone="head"><meta name="pbContext" content=";product:product:elsevier\:product\:cell;website:website:cell-site;ctype:string:Journal Content;article:article:pii\:S0092867419313984;issue:issue:pii\:S0092867419X00045;journal:journal:cell;pageGroup:string:Article View;page:string:Show Full Text;requestedJournal:journal:cell;wgroup:string:Default Website Group;subPage:string:Full Text">




<!-- if there is any customization for Responsive Project widget  -->


























    
    







    
            
        



        
                <meta name="robots" content="noarchive,nocache">
            
    






    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1, user-scalable=1">











        <meta property="og:title" content="Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism">
    


        <meta property="og:type" content="Article">
    


        <meta property="og:url" content="https://www.cell.com/cell/abstract/S0092-8674(19)31398-4">
    



        <meta property="og:site_name" content="Cell">
    


        <meta property="og:image" content="https://www.cell.com/cms/attachment/9e9d3ab2-3cb6-46aa-8e12-041aff8e5a0a/fx1.jpg">
    


        <meta property="og:description" content="Large-scale sequencing of patients with autism allows identification of over 100 putative
ASD-associated genes, the majority of which are neuronally expressed, and investigation
of distinct genetic influences on ASD compared with other neurodevelopmental disorders.">
    











        <meta name="twitter:title" content="Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism">
    




        <meta name="twitter:image" content="https://www.cell.com/cms/attachment/9e9d3ab2-3cb6-46aa-8e12-041aff8e5a0a/fx1.jpg">
    


        <meta name="twitter:description" content="Large-scale sequencing of patients with autism allows identification of over 100 putative
ASD-associated genes, the majority of which are neuronally expressed, and investigation
of distinct genetic influences on ASD compared with other neurodevelopmental disorders.">
    

        <meta name="twitter:card" content="summary_large_image">
    











    
    
            
            
        <link rel="canonical" href="https://www.cell.com/cell/fulltext/S0092-8674(19)31398-4">
    


<meta name="tdm-reservation" content="1">
<meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">

<title>Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism: Cell</title><meta name="citation_journal_title" content="Cell"> <meta name="citation_journal_abbrev" content="Cell"> <meta name="citation_publisher" content="Elsevier"> <meta name="citation_title" content="Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism"> <meta name="citation_date" content="2020/02/06"> <meta name="citation_online_date" content="2020/01/23"> <meta name="citation_volume" content="180"> <meta name="citation_issue" content="3"> <meta name="citation_firstpage" content="568"> <meta name="citation_lastpage" content="584.e23"> <meta name="citation_pmid" content="31981491"> <meta name="citation_abstract_html_url" content="https://www.cell.com/action/showAbstract?pii=S0092867419313984"> <meta name="citation_fulltext_html_url" content="https://www.cell.com/action/showFullText?pii=S0092867419313984"> <meta name="citation_pdf_url" content="https://www.cell.com/action/showPdf?pii=S0092867419313984"> <meta name="citation_issn" content="0092-8674"> <meta name="citation_issn" content="1097-4172"> <meta name="citation_language" content="English"> <meta name="citation_doi" content="10.1016/j.cell.2019.12.036"> <meta name="citation_pmid" content="31981491"> <meta name="citation_author" content="F. Kyle Satterstrom"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author" content="Jack A. Kosmicki"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author" content="Jiebiao Wang"><meta name="citation_author_institution" content="Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA"><meta name="citation_author" content="Michael S. Breen"><meta name="citation_author_institution" content="Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author" content="Silvia De Rubeis"><meta name="citation_author_institution" content="Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author" content="Joon-Yong An"><meta name="citation_author_institution" content="Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author_institution" content="School of Biosystem and Biomedical Science, College of Health Science, Korea University, Seoul, Republic of Korea"><meta name="citation_author" content="Minshi Peng"><meta name="citation_author_institution" content="Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA"><meta name="citation_author" content="Ryan Collins"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Program in Bioinformatics and Integrative Genomics, Harvard Medical School, Boston, MA, USA"><meta name="citation_author" content="Jakob Grove"><meta name="citation_author_institution" content="The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark"><meta name="citation_author_institution" content="Center for Genomics and Personalized Medicine, Aarhus, Denmark"><meta name="citation_author_institution" content="Department of Biomedicine – Human Genetics, Aarhus University, Aarhus, Denmark"><meta name="citation_author" content="Lambertus Klei"><meta name="citation_author_institution" content="Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA"><meta name="citation_author" content="Christine Stevens"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author" content="Jennifer Reichert"><meta name="citation_author_institution" content="Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author" content="Maureen S. Mulhern"><meta name="citation_author_institution" content="Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author" content="Mykyta Artomov"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author" content="Sherif Gerges"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author" content="Brooke Sheppard"><meta name="citation_author_institution" content="Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Xinyi Xu"><meta name="citation_author_institution" content="Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author" content="Aparna Bhaduri"><meta name="citation_author_institution" content="Department of Neurology, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author_institution" content="The Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Utku Norman"><meta name="citation_author_institution" content="Computer Engineering Department, Bilkent University, Ankara, Turkey"><meta name="citation_author" content="Harrison Brand"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author" content="Grace Schwartz"><meta name="citation_author_institution" content="Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Rachel Nguyen"><meta name="citation_author_institution" content="Center for Autism Research and Translation, University of California, Irvine, Irvine, CA, USA"><meta name="citation_author" content="Elizabeth E. Guerrero"><meta name="citation_author_institution" content="MIND (Medical Investigation of Neurodevelopmental Disorders) Institute, University of California, Davis, Davis, CA, USA"><meta name="citation_author" content="Caroline Dias"><meta name="citation_author_institution" content="Division of Genetics, Boston Children’s Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Division of Developmental Medicine, Boston Children’s Hospital, Boston, MA, USA"><meta name="citation_author" content="Branko Aleksic"><meta name="citation_author" content="Richard Anney"><meta name="citation_author" content="Mafalda Barbosa"><meta name="citation_author" content="Somer Bishop"><meta name="citation_author" content="Alfredo Brusco"><meta name="citation_author" content="Jonas Bybjerg-Grauholm"><meta name="citation_author" content="Angel Carracedo"><meta name="citation_author" content="Marcus C.Y. Chan"><meta name="citation_author" content="Andreas G. Chiocchetti"><meta name="citation_author" content="Brian H.Y. Chung"><meta name="citation_author" content="Hilary Coon"><meta name="citation_author" content="Michael L. Cuccaro"><meta name="citation_author" content="Aurora Curró"><meta name="citation_author" content="Bernardo Dalla Bernardina"><meta name="citation_author" content="Ryan Doan"><meta name="citation_author" content="Enrico Domenici"><meta name="citation_author" content="Shan Dong"><meta name="citation_author" content="Chiara Fallerini"><meta name="citation_author" content="Montserrat Fernández-Prieto"><meta name="citation_author" content="Giovanni Battista Ferrero"><meta name="citation_author" content="Christine M. Freitag"><meta name="citation_author" content="Menachem Fromer"><meta name="citation_author" content="J. Jay Gargus"><meta name="citation_author" content="Daniel Geschwind"><meta name="citation_author" content="Elisa Giorgio"><meta name="citation_author" content="Javier González-Peñas"><meta name="citation_author" content="Stephen Guter"><meta name="citation_author" content="Danielle Halpern"><meta name="citation_author" content="Emily Hansen-Kiss"><meta name="citation_author" content="Xin He"><meta name="citation_author" content="Gail E. Herman"><meta name="citation_author" content="Irva Hertz-Picciotto"><meta name="citation_author" content="David M. Hougaard"><meta name="citation_author" content="Christina M. Hultman"><meta name="citation_author" content="Iuliana Ionita-Laza"><meta name="citation_author" content="Suma Jacob"><meta name="citation_author" content="Jesslyn Jamison"><meta name="citation_author" content="Astanand Jugessur"><meta name="citation_author" content="Miia Kaartinen"><meta name="citation_author" content="Gun Peggy Knudsen"><meta name="citation_author" content="Alexander Kolevzon"><meta name="citation_author" content="Itaru Kushima"><meta name="citation_author" content="So Lun Lee"><meta name="citation_author" content="Terho Lehtimäki"><meta name="citation_author" content="Elaine T. Lim"><meta name="citation_author" content="Carla Lintas"><meta name="citation_author" content="W. Ian Lipkin"><meta name="citation_author" content="Diego Lopergolo"><meta name="citation_author" content="Fátima Lopes"><meta name="citation_author" content="Yunin Ludena"><meta name="citation_author" content="Patricia Maciel"><meta name="citation_author" content="Per Magnus"><meta name="citation_author" content="Behrang Mahjani"><meta name="citation_author" content="Nell Maltman"><meta name="citation_author" content="Dara S. Manoach"><meta name="citation_author" content="Gal Meiri"><meta name="citation_author" content="Idan Menashe"><meta name="citation_author" content="Judith Miller"><meta name="citation_author" content="Nancy Minshew"><meta name="citation_author" content="Eduarda M.S. Montenegro"><meta name="citation_author" content="Danielle Moreira"><meta name="citation_author" content="Eric M. Morrow"><meta name="citation_author" content="Ole Mors"><meta name="citation_author" content="Preben Bo Mortensen"><meta name="citation_author" content="Matthew Mosconi"><meta name="citation_author" content="Pierandrea Muglia"><meta name="citation_author" content="Benjamin M. Neale"><meta name="citation_author" content="Merete Nordentoft"><meta name="citation_author" content="Norio Ozaki"><meta name="citation_author" content="Aarno Palotie"><meta name="citation_author" content="Mara Parellada"><meta name="citation_author" content="Maria Rita Passos-Bueno"><meta name="citation_author" content="Margaret Pericak-Vance"><meta name="citation_author" content="Antonio M. Persico"><meta name="citation_author" content="Isaac Pessah"><meta name="citation_author" content="Kaija Puura"><meta name="citation_author" content="Abraham Reichenberg"><meta name="citation_author" content="Alessandra Renieri"><meta name="citation_author" content="Evelise Riberi"><meta name="citation_author" content="Elise B. Robinson"><meta name="citation_author" content="Kaitlin E. Samocha"><meta name="citation_author" content="Sven Sandin"><meta name="citation_author" content="Susan L. Santangelo"><meta name="citation_author" content="Gerry Schellenberg"><meta name="citation_author" content="Stephen W. Scherer"><meta name="citation_author" content="Sabine Schlitt"><meta name="citation_author" content="Rebecca Schmidt"><meta name="citation_author" content="Lauren Schmitt"><meta name="citation_author" content="Isabela M.W. Silva"><meta name="citation_author" content="Tarjinder Singh"><meta name="citation_author" content="Paige M. Siper"><meta name="citation_author" content="Moyra Smith"><meta name="citation_author" content="Gabriela Soares"><meta name="citation_author" content="Camilla Stoltenberg"><meta name="citation_author" content="Pål Suren"><meta name="citation_author" content="Ezra Susser"><meta name="citation_author" content="John Sweeney"><meta name="citation_author" content="Peter Szatmari"><meta name="citation_author" content="Lara Tang"><meta name="citation_author" content="Flora Tassone"><meta name="citation_author" content="Karoline Teufel"><meta name="citation_author" content="Elisabetta Trabetti"><meta name="citation_author" content="Maria del Pilar Trelles"><meta name="citation_author" content="Christopher A. Walsh"><meta name="citation_author" content="Lauren A. Weiss"><meta name="citation_author" content="Thomas Werge"><meta name="citation_author" content="Donna M. Werling"><meta name="citation_author" content="Emilie M. Wigdor"><meta name="citation_author" content="Emma Wilkinson"><meta name="citation_author" content="A. Jeremy Willsey"><meta name="citation_author" content="Timothy W. Yu"><meta name="citation_author" content="Mullin H.C. Yu"><meta name="citation_author" content="Ryan Yuen"><meta name="citation_author" content="Elaine Zachi"><meta name="citation_author" content="Esben Agerbo"><meta name="citation_author" content="Thomas Damm Als"><meta name="citation_author" content="Vivek Appadurai"><meta name="citation_author" content="Marie Bækvad-Hansen"><meta name="citation_author" content="Rich Belliveau"><meta name="citation_author" content="Alfonso Buil"><meta name="citation_author" content="Caitlin E. Carey"><meta name="citation_author" content="Felecia Cerrato"><meta name="citation_author" content="Kimberly Chambert"><meta name="citation_author" content="Claire Churchhouse"><meta name="citation_author" content="Søren Dalsgaard"><meta name="citation_author" content="Ditte Demontis"><meta name="citation_author" content="Ashley Dumont"><meta name="citation_author" content="Jacqueline Goldstein"><meta name="citation_author" content="Christine S. Hansen"><meta name="citation_author" content="Mads Engel Hauberg"><meta name="citation_author" content="Mads V. Hollegaard"><meta name="citation_author" content="Daniel P. Howrigan"><meta name="citation_author" content="Hailiang Huang"><meta name="citation_author" content="Julian Maller"><meta name="citation_author" content="Alicia R. Martin"><meta name="citation_author" content="Joanna Martin"><meta name="citation_author" content="Manuel Mattheisen"><meta name="citation_author" content="Jennifer Moran"><meta name="citation_author" content="Jonatan Pallesen"><meta name="citation_author" content="Duncan S. Palmer"><meta name="citation_author" content="Carsten Bøcker Pedersen"><meta name="citation_author" content="Marianne Giørtz Pedersen"><meta name="citation_author" content="Timothy Poterba"><meta name="citation_author" content="Jesper Buchhave Poulsen"><meta name="citation_author" content="Stephan Ripke"><meta name="citation_author" content="Andrew J. Schork"><meta name="citation_author" content="Wesley K. Thompson"><meta name="citation_author" content="Patrick Turley"><meta name="citation_author" content="Raymond K. Walters"><meta name="citation_author" content="Catalina Betancur"><meta name="citation_author_institution" content="Sorbonne Université, INSERM, CNRS, Neuroscience Paris Seine, Institut de Biologie Paris Seine, Paris, France"><meta name="citation_author" content="Edwin H. Cook"><meta name="citation_author_institution" content="Institute for Juvenile Research, Department of Psychiatry, University of Illinois at Chicago, Chicago, IL, USA"><meta name="citation_author" content="Louise Gallagher"><meta name="citation_author_institution" content="Department of Psychiatry, School of Medicine, Trinity College Dublin, Dublin, Ireland"><meta name="citation_author" content="Michael Gill"><meta name="citation_author_institution" content="Department of Psychiatry, School of Medicine, Trinity College Dublin, Dublin, Ireland"><meta name="citation_author" content="James S. Sutcliffe"><meta name="citation_author_institution" content="Vanderbilt Genetics Institute, Vanderbilt University School of Medicine, Nashville, TN, USA"><meta name="citation_author_institution" content="Department of Molecular Physiology and Biophysics and Psychiatry, Vanderbilt University School of Medicine, Nashville, TN, USA"><meta name="citation_author" content="Audrey Thurm"><meta name="citation_author_institution" content="National Institute of Mental Health, NIH, Bethesda, MD, USA"><meta name="citation_author" content="Michael E. Zwick"><meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA"><meta name="citation_author" content="Anders D. Børglum"><meta name="citation_author_institution" content="The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark"><meta name="citation_author_institution" content="Center for Genomics and Personalized Medicine, Aarhus, Denmark"><meta name="citation_author_institution" content="Department of Biomedicine – Human Genetics, Aarhus University, Aarhus, Denmark"><meta name="citation_author_institution" content="Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark"><meta name="citation_author" content="Matthew W. State"><meta name="citation_author_institution" content="Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="A. Ercument Cicek"><meta name="citation_author_institution" content="Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA"><meta name="citation_author_institution" content="Computer Engineering Department, Bilkent University, Ankara, Turkey"><meta name="citation_author" content="Michael E. Talkowski"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author" content="David J. Cutler"><meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA"><meta name="citation_author" content="Bernie Devlin"><meta name="citation_author_institution" content="Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA"><meta name="citation_author" content="Stephan J. Sanders"><meta name="citation_author_email" content="stephan.sanders@ucsf.edu"><meta name="citation_author_institution" content="Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA"><meta name="citation_author" content="Kathryn Roeder"><meta name="citation_author_email" content="roeder@andrew.cmu.edu"><meta name="citation_author_institution" content="Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA"><meta name="citation_author_institution" content="Computational Biology Department, Carnegie Mellon University, Pittsburgh, PA, USA"><meta name="citation_author" content="Mark J. Daly"><meta name="citation_author_email" content="mjdaly@broadinstitute.org"><meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA"><meta name="citation_author_institution" content="Harvard Medical School, Boston, MA, USA"><meta name="citation_author_institution" content="Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA"><meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland"><meta name="citation_author" content="Joseph D. Buxbaum"><meta name="citation_author_email" content="joseph.buxbaum@mssm.edu"><meta name="citation_author_institution" content="Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_author_institution" content="Department of Neuroscience, Icahn School of Medicine at Mount Sinai, New York, NY, USA"><meta name="citation_keywords" content="autism spectrum disorder; exome sequencing; genetics; neurodevelopment; liability; cell type; cytoskeleton; excitatory neurons; inhibitory neurons; excitatory-inhibitory balance"><meta name="citation_reference" content="citation_title=A method and server for predicting damaging missense mutations; citation_author=I.A. Adzhubei; citation_author=S. Schmidt; citation_author=L. Peshkin; citation_author=V.E. Ramensky; citation_author=A. Gerasimova; citation_author=P. Bork; citation_author=A.S. Kondrashov; citation_author=S.R. Sunyaev; citation_journal_title=Nat. Methods; citation_volume=7; citation_firstpage=248; citation_lastpage=249; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Prevalence of Autism Spectrum Disorder Among Children Aged 8 Years - Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2014; citation_author=J. Baio; citation_author=L. Wiggins; citation_author=D.L. Christensen; citation_author=M.J. Maenner; citation_author=J. Daniels; citation_author=Z. Warren; citation_author=M. Kurzius-Spencer; citation_author=W. Zahorodny; citation_author=C. Robinson Rosenberg; citation_author=T. White; citation_author=et al.; citation_journal_title=MMWR Surveill Summ.; citation_volume=67; citation_firstpage=1; citation_lastpage=23; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genetic effects on gene expression across human tissues; citation_author=A. Battle; citation_author=C.D. Brown; citation_author=B.E. Engelhardt; citation_author=S.B. Montgomery; citation_author=GTEx Consortium; citation_author=Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group; citation_author=Statistical Methods groups—Analysis Working Group; citation_author=Enhancing GTEx (eGTEx) groups; citation_author=NIH Common Fund; citation_author=NIH/NCI; citation_author=NIH/NHGRI; citation_author=NIH/NIMH; citation_author=NIH/NIDA; citation_author=Biospecimen Collection Source Site—NDRI; citation_author=Biospecimen Collection Source Site—RPCI; citation_author=Biospecimen Core Resource—VARI; citation_author=Brain Bank Repository—University of Miami Brain Endowment Bank; citation_author=Leidos Biomedical—Project Management; citation_author=ELSI Study; citation_author=Genome Browser Data Integration &amp;Visualization—EBI; citation_author=Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz; citation_author=Lead analysts; citation_author=Laboratory, Data Analysis &amp;Coordinating Center (LDACC); citation_author=NIH program management; citation_author=Biospecimen collection; citation_author=Pathology; citation_author=eQTL manuscript working group; citation_journal_title=Nature; citation_volume=550; citation_firstpage=204; citation_lastpage=213; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Opposing effects on NaV1.2 function underlie differences between SCN2A variants observed in individuals with autism spectrum disorder or infantile seizures; citation_author=R. Ben-Shalom; citation_author=C.M. Keeshen; citation_author=K.N. Berrios; citation_author=J.Y. An; citation_author=S.J. Sanders; citation_author=K.J. Bender; citation_journal_title=Biol. Psychiatry; citation_volume=82; citation_firstpage=224; citation_lastpage=232; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Disruptive CHD8 mutations define a subtype of autism early in development; citation_author=R. Bernier; citation_author=C. Golzio; citation_author=B. Xiong; citation_author=H.A. Stessman; citation_author=B.P. Coe; citation_author=O. Penn; citation_author=K. Witherspoon; citation_author=J. Gerdts; citation_author=C. Baker; citation_author=A.T. Vulto-van Silfhout; citation_author=et al.; citation_journal_title=Cell; citation_volume=158; citation_firstpage=263; citation_lastpage=276; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project; citation_author=E. Birney; citation_author=J.A. Stamatoyannopoulos; citation_author=A. Dutta; citation_author=R. Guigó; citation_author=T.R. Gingeras; citation_author=E.H. Margulies; citation_author=Z. Weng; citation_author=M. Snyder; citation_author=E.T. Dermitzakis; citation_author=R.E. Thurman; citation_author=ENCODE Project Consortium; citation_author=NISC Comparative Sequencing Program; citation_author=Baylor College of Medicine Human Genome Sequencing Center; citation_author=Washington University Genome Sequencing Center; citation_author=Broad Institute; citation_author=Children’s Hospital Oakland Research Institute; citation_author=et al.; citation_journal_title=Nature; citation_volume=447; citation_firstpage=799; citation_lastpage=816; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=The SAND domain structure defines a novel DNA-binding fold in transcriptional regulation; citation_author=M.J. Bottomley; citation_author=M.W. Collard; citation_author=J.I. Huggenvik; citation_author=Z. Liu; citation_author=T.J. Gibson; citation_author=M. Sattler; citation_journal_title=Nat. Struct. Biol.; citation_volume=8; citation_firstpage=626; citation_lastpage=633; citation_publication_date=2001; "><meta name="citation_reference" content="citation_title=The autism sequencing consortium: large-scale, high-throughput sequencing in autism spectrum disorders; citation_author=J.D. Buxbaum; citation_author=M.J. Daly; citation_author=B. Devlin; citation_author=T. Lehner; citation_author=K. Roeder; citation_author=M.W. State; citation_author=Autism Sequencing Consortium; citation_journal_title=Neuron; citation_volume=76; citation_firstpage=1052; citation_lastpage=1056; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Genotype to phenotype relationships in autism spectrum disorders; citation_author=J. Chang; citation_author=S.R. Gilman; citation_author=A.H. Chiang; citation_author=S.J. Sanders; citation_author=D. Vitkup; citation_journal_title=Nat. Neurosci.; citation_volume=18; citation_firstpage=191; citation_lastpage=198; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=A genome-wide association study of autism using the Simons Simplex Collection: Does reducing phenotypic heterogeneity in autism increase genetic homogeneity?; citation_author=P. Chaste; citation_author=L. Klei; citation_author=S.J. Sanders; citation_author=V. Hus; citation_author=M.T. Murtha; citation_author=J.K. Lowe; citation_author=A.J. Willsey; citation_author=D. Moreno-De-Luca; citation_author=T.W. Yu; citation_author=E. Fombonne; citation_author=et al.; citation_journal_title=Biol. Psychiatry; citation_volume=77; citation_firstpage=775; citation_lastpage=784; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=ToppGene Suite for gene list enrichment analysis and candidate gene prioritization; citation_author=J. Chen; citation_author=E.E. Bardes; citation_author=B.J. Aronow; citation_author=A.G. Jegga; citation_journal_title=Nucleic Acids Res.; citation_volume=37; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool; citation_author=E.Y. Chen; citation_author=C.M. Tan; citation_author=Y. Kou; citation_author=Q. Duan; citation_author=Z. Wang; citation_author=G.V. Meirelles; citation_author=N.R. Clark; citation_author=A. Ma’ayan; citation_journal_title=BMC Bioinformatics; citation_volume=14; citation_firstpage=128; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Functional analysis of novel DEAF1 variants identified through clinical exome sequencing expands DEAF1-associated neurodevelopmental disorder (DAND) phenotype; citation_author=L. Chen; citation_author=P.J. Jensik; citation_author=J.T. Alaimo; citation_author=M. Walkiewicz; citation_author=S. Berger; citation_author=E. Roeder; citation_author=E.A. Faqeih; citation_author=J.A. Bernstein; citation_author=A.C.M. Smith; citation_author=S.V. Mullegama; citation_author=et al.; citation_journal_title=Hum. Mutat.; citation_volume=38; citation_firstpage=1774; citation_lastpage=1785; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Prevalence and Characteristics of Autism Spectrum Disorder Among Children Aged 8 Years--Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2012; citation_author=D.L. Christensen; citation_author=J. Baio; citation_author=K. Van Naarden Braun; citation_author=D. Bilder; citation_author=J. Charles; citation_author=J.N. Constantino; citation_author=J. Daniels; citation_author=M.S. Durkin; citation_author=R.T. Fitzgerald; citation_author=M. Kurzius-Spencer; citation_author=Centers for Disease Control and Prevention (CDC); citation_author=et al.; citation_journal_title=MMWR Surveill. Summ.; citation_volume=65; citation_firstpage=1; citation_lastpage=23; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Refining analyses of copy number variation identifies specific genes associated with developmental delay; citation_author=B.P. Coe; citation_author=K. Witherspoon; citation_author=J.A. Rosenfeld; citation_author=B.W. van Bon; citation_author=A.T. Vulto-van Silfhout; citation_author=P. Bosco; citation_author=K.L. Friend; citation_author=C. Baker; citation_author=S. Buono; citation_author=L.E. Vissers; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=1063; citation_lastpage=1071; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=A copy number variation morbidity map of developmental delay; citation_author=G.M. Cooper; citation_author=B.P. Coe; citation_author=S. Girirajan; citation_author=J.A. Rosenfeld; citation_author=T.H. Vu; citation_author=C. Baker; citation_author=C. Williams; citation_author=H. Stalker; citation_author=R. Hamid; citation_author=V. Hannig; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=43; citation_firstpage=838; citation_lastpage=846; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment; citation_author=J. Cotney; citation_author=R.A. Muhle; citation_author=S.J. Sanders; citation_author=L. Liu; citation_author=A.J. Willsey; citation_author=W. Niu; citation_author=W. Liu; citation_author=L. Klei; citation_author=J. Lei; citation_author=J. Yin; citation_author=et al.; citation_journal_title=Nat. Commun.; citation_volume=6; citation_firstpage=6404; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=MAGMA: generalized gene-set analysis of GWAS data; citation_author=C.A. de Leeuw; citation_author=J.M. Mooij; citation_author=T. Heskes; citation_author=D. Posthuma; citation_journal_title=PLoS Comput. Biol.; citation_volume=11; citation_firstpage=e1004219; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Synaptic, transcriptional and chromatin genes disrupted in autism; citation_author=S. De Rubeis; citation_author=X. He; citation_author=A.P. Goldberg; citation_author=C.S. Poultney; citation_author=K. Samocha; citation_author=A.E. Cicek; citation_author=Y. Kou; citation_author=L. Liu; citation_author=M. Fromer; citation_author=S. Walker; citation_author=DDD Study; citation_author=Homozygosity Mapping Collaborative for Autism; citation_author=UK10K Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=515; citation_firstpage=209; citation_lastpage=215; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Prevalence and architecture of de novo mutations in developmental disorders; citation_author=Deciphering Developmental Disorders Study; citation_journal_title=Nature; citation_volume=542; citation_firstpage=433; citation_lastpage=438; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder; citation_author=D. Demontis; citation_author=R.K. Walters; citation_author=J. Martin; citation_author=M. Mattheisen; citation_author=T.D. Als; citation_author=E. Agerbo; citation_author=G. Baldursson; citation_author=R. Belliveau; citation_author=J. Bybjerg-Grauholm; citation_author=M. Bækvad-Hansen; citation_author=ADHD Working Group of the Psychiatric Genomics Consortium (PGC); citation_author=Early Lifecourse &amp; Genetic Epidemiology (EAGLE) Consortium; citation_author=23andMe Research Team; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=63; citation_lastpage=75; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Genomic control for association studies; citation_author=B. Devlin; citation_author=K. Roeder; citation_journal_title=Biometrics; citation_volume=55; citation_firstpage=997; citation_lastpage=1004; citation_publication_date=1999; "><meta name="citation_reference" content="citation_title=NAHR-mediated copy-number variants in a clinical population: mechanistic insights into both genomic disorders and Mendelizing traits; citation_author=P. Dittwald; citation_author=T. Gambin; citation_author=P. Szafranski; citation_author=J. Li; citation_author=S. Amato; citation_author=M.Y. Divon; citation_author=L.X. Rodríguez Rojas; citation_author=L.E. Elton; citation_author=D.A. Scott; citation_author=C.P. Schaaf; citation_author=et al.; citation_journal_title=Genome Res.; citation_volume=23; citation_firstpage=1395; citation_lastpage=1409; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Large-scale inference: empirical Bayes methods for estimation, testing, and prediction; citation_author=B. Efron; citation_publication_date=2012; citation_publisher=Cambridge University Press; "><meta name="citation_reference" content="citation_title=Differential Ability Scales; citation_author=C.D. Elliott; citation_publication_date=2007; citation_publisher=Harcourt Assessment; "><meta name="citation_reference" content="citation_title=The inheritance of liability to certain diseases, estimated from the incidence among relatives; citation_author=D.S. Falconer; citation_journal_title=Ann. Hum. Genet.; citation_volume=29; citation_firstpage=51; citation_lastpage=76; citation_publication_date=1965; "><meta name="citation_reference" content="citation_title=Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap; citation_author=M.J. Gandal; citation_author=J.R. Haney; citation_author=N.N. Parikshak; citation_author=V. Leppa; citation_author=G. Ramaswami; citation_author=C. Hartl; citation_author=A.J. Schork; citation_author=V. Appadurai; citation_author=A. Buil; citation_author=T.M. Werge; citation_author=CommonMind Consortium; citation_author=PsychENCODE Consortium; citation_author=iPSYCH-BROAD Working Group; citation_author=et al.; citation_journal_title=Science; citation_volume=359; citation_firstpage=693; citation_lastpage=697; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder; citation_author=M.J. Gandal; citation_author=P. Zhang; citation_author=E. Hadjimichael; citation_author=R.L. Walker; citation_author=C. Chen; citation_author=S. Liu; citation_author=H. Won; citation_author=H. Van Bakel; citation_author=M. Varghese; citation_author=Y. Wang; citation_author=A.W. Shieh; citation_author=et al.; citation_journal_title=Science; citation_volume=362; citation_firstpage=eaat8127; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Most genetic risk for autism resides with common variation; citation_author=T. Gaugler; citation_author=L. Klei; citation_author=S.J. Sanders; citation_author=C.A. Bodea; citation_author=A.P. Goldberg; citation_author=A.B. Lee; citation_author=M. Mahajan; citation_author=D. Manaa; citation_author=Y. Pawitan; citation_author=J. Reichert; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=881; citation_lastpage=885; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests; citation_author=B. Gel; citation_author=A. Díez-Villanueva; citation_author=E. Serra; citation_author=M. Buschbeck; citation_author=M.A. Peinado; citation_author=R. Malinverni; citation_journal_title=Bioinformatics; citation_volume=32; citation_firstpage=289; citation_lastpage=291; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Identification of common genetic risk variants for autism spectrum disorder; citation_author=J. Grove; citation_author=S. Ripke; citation_author=T.D. Als; citation_author=M. Mattheisen; citation_author=R.K. Walters; citation_author=H. Won; citation_author=J. Pallesen; citation_author=E. Agerbo; citation_author=O.A. Andreassen; citation_author=R. Anney; citation_author=Autism Spectrum Disorder Working Group of the Psychiatric Genomics Consortium; citation_author=BUPGEN; citation_author=Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium; citation_author=23andMe Research Team; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=431; citation_lastpage=444; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A map of constrained coding regions in the human genome; citation_author=J.M. Havrilla; citation_author=B.S. Pedersen; citation_author=R.M. Layer; citation_author=A.R. Quinlan; citation_journal_title=Nat. Genet.; citation_volume=51; citation_firstpage=88; citation_lastpage=95; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes; citation_author=X. He; citation_author=S.J. Sanders; citation_author=L. Liu; citation_author=S. De Rubeis; citation_author=E.T. Lim; citation_author=J.S. Sutcliffe; citation_author=G.D. Schellenberg; citation_author=R.A. Gibbs; citation_author=M.J. Daly; citation_author=J.D. Buxbaum; citation_author=et al.; citation_journal_title=PLoS Genet.; citation_volume=9; citation_firstpage=e1003671; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=De novo variants in neurodevelopmental disorders with epilepsy; citation_author=H.O. Heyne; citation_author=T. Singh; citation_author=H. Stamberger; citation_author=R. Abou Jamra; citation_author=H. Caglayan; citation_author=D. Craiu; citation_author=P. De Jonghe; citation_author=R. Guerrini; citation_author=K.L. Helbig; citation_author=B.P.C. Koeleman; citation_author=EuroEPINOMICS RES Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=1048; citation_lastpage=1053; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=A network of epigenetic regulators guides developmental haematopoiesis in&nbsp;vivo; citation_author=H.T. Huang; citation_author=K.L. Kathrein; citation_author=A. Barton; citation_author=Z. Gitlin; citation_author=Y.H. Huang; citation_author=T.P. Ward; citation_author=O. Hofmann; citation_author=A. Dibiase; citation_author=A. Song; citation_author=S. Tyekucheva; citation_author=et al.; citation_journal_title=Nat. Cell Biol.; citation_volume=15; citation_firstpage=1516; citation_lastpage=1525; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The contribution of de novo coding mutations to autism spectrum disorder; citation_author=I. Iossifov; citation_author=B.J. O’Roak; citation_author=S.J. Sanders; citation_author=M. Ronemus; citation_author=N. Krumm; citation_author=D. Levy; citation_author=H.A. Stessman; citation_author=K.T. Witherspoon; citation_author=L. Vives; citation_author=K.E. Patterson; citation_author=et al.; citation_journal_title=Nature; citation_volume=515; citation_firstpage=216; citation_lastpage=221; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Identification of a nuclear export signal and protein interaction domains in deformed epidermal autoregulatory factor-1 (DEAF-1); citation_author=P.J. Jensik; citation_author=J.I. Huggenvik; citation_author=M.W. Collard; citation_journal_title=J.&nbsp;Biol. Chem.; citation_volume=279; citation_firstpage=32692; citation_lastpage=32699; citation_publication_date=2004; "><meta name="citation_reference" content="citation_title=Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands; citation_author=S.C. Jin; citation_author=J. Homsy; citation_author=S. Zaidi; citation_author=Q. Lu; citation_author=S. Morton; citation_author=S.R. DePalma; citation_author=X. Zeng; citation_author=H. Qi; citation_author=W. Chang; citation_author=M.C. Sierant; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=1593; citation_lastpage=1601; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Defining the phenotypic spectrum of SLC6A1 mutations; citation_author=K.M. Johannesen; citation_author=E. Gardella; citation_author=T. Linnankivi; citation_author=C. Courage; citation_author=A. de Saint Martin; citation_author=A.-E. Lehesjoki; citation_author=C. Mignot; citation_author=A. Afenjar; citation_author=G. Lesca; citation_author=M.-T. Abi-Warde; citation_author=et al.; citation_journal_title=Epilepsia; citation_volume=59; citation_firstpage=389; citation_lastpage=402; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Detecting and estimating contamination of human DNA samples in sequencing and array-based genotype data; citation_author=G. Jun; citation_author=M. Flickinger; citation_author=K.N. Hetrick; citation_author=J.M. Romm; citation_author=K.F. Doheny; citation_author=G.R. Abecasis; citation_author=M. Boehnke; citation_author=H.M. Kang; citation_journal_title=Am. J. Hum. Genet.; citation_volume=91; citation_firstpage=839; citation_lastpage=848; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Spatio-temporal transcriptome of the human brain; citation_author=H.J. Kang; citation_author=Y.I. Kawasawa; citation_author=F. Cheng; citation_author=Y. Zhu; citation_author=X. Xu; citation_author=M. Li; citation_author=A.M. Sousa; citation_author=M. Pletikos; citation_author=K.A. Meyer; citation_author=G. Sedmak; citation_author=et al.; citation_journal_title=Nature; citation_volume=478; citation_firstpage=483; citation_lastpage=489; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Variation across 141,456 human exomes and genomes reveals the spectrum of loss-of-function intolerance across human protein-coding genes; citation_author=K.J. Karczewski; citation_author=L.C. Francioli; citation_author=G. Tiao; citation_author=B.B. Cummings; citation_author=J. Alföldi; citation_author=Q. Wang; citation_author=R.L. Collins; citation_author=K.M. Laricchia; citation_author=A. Ganna; citation_author=D.P. Birnbaum; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/531210; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=HIstome--a relational knowledgebase of human histone proteins and histone modifying enzymes; citation_author=S.P. Khare; citation_author=F. Habib; citation_author=R. Sharma; citation_author=N. Gadewal; citation_author=S. Gupta; citation_author=S. Galande; citation_journal_title=Nucleic Acids Res.; citation_volume=40; citation_firstpage=D337; citation_lastpage=D342; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples; citation_author=J.A. Kosmicki; citation_author=K.E. Samocha; citation_author=D.P. Howrigan; citation_author=S.J. Sanders; citation_author=K. Slowikowski; citation_author=M. Lek; citation_author=K.J. Karczewski; citation_author=D.J. Cutler; citation_author=B. Devlin; citation_author=K. Roeder; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=49; citation_firstpage=504; citation_lastpage=510; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=ChEA: transcription factor regulation inferred from integrating genome-wide ChIP-X experiments; citation_author=A. Lachmann; citation_author=H. Xu; citation_author=J. Krishnan; citation_author=S.I. Berger; citation_author=A.R. Mazloom; citation_author=A. Ma’ayan; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=2438; citation_lastpage=2444; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=ClinVar: public archive of relationships among sequence variation and human phenotype; citation_author=M.J. Landrum; citation_author=J.M. Lee; citation_author=G.R. Riley; citation_author=W. Jang; citation_author=W.S. Rubinstein; citation_author=D.M. Church; citation_author=D.R. Maglott; citation_journal_title=Nucleic Acids Res.; citation_volume=42; citation_firstpage=D980; citation_lastpage=D985; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Validity of childhood autism in the Danish Psychiatric Central Register: findings from a cohort sample born 1990-1999; citation_author=M.B. Lauritsen; citation_author=M. Jørgensen; citation_author=K.M. Madsen; citation_author=S. Lemcke; citation_author=S. Toft; citation_author=J. Grove; citation_author=D.E. Schendel; citation_author=P. Thorsen; citation_journal_title=J.&nbsp;Autism Dev. Disord.; citation_volume=40; citation_firstpage=139; citation_lastpage=148; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals; citation_author=J.J. Lee; citation_author=R. Wedow; citation_author=A. Okbay; citation_author=E. Kong; citation_author=O. Maghzian; citation_author=M. Zacher; citation_author=T.A. Nguyen-Viet; citation_author=P. Bowers; citation_author=J. Sidorenko; citation_author=R. Karlsson Linnér; citation_author=23andMe Research Team; citation_author=COGENT (Cognitive Genomics Consortium); citation_author=Social Science Genetic Association Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=1112; citation_lastpage=1121; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Analysis of protein-coding genetic variation in 60,706 humans; citation_author=M. Lek; citation_author=K.J. Karczewski; citation_author=E.V. Minikel; citation_author=K.E. Samocha; citation_author=E. Banks; citation_author=T. Fennell; citation_author=A.H. O’Donnell-Luria; citation_author=J.S. Ware; citation_author=A.J. Hill; citation_author=B.B. Cummings; citation_author=Exome Aggregation Consortium; citation_author=et al.; citation_journal_title=Nature; citation_volume=536; citation_firstpage=285; citation_lastpage=291; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=The 2q37-deletion syndrome: an update of the clinical spectrum including overweight, brachydactyly and behavioural features in 14 new patients; citation_author=C. Leroy; citation_author=E. Landais; citation_author=S. Briault; citation_author=A. David; citation_author=O. Tassy; citation_author=N. Gruchy; citation_author=B. Delobel; citation_author=M.J. Grégoire; citation_author=B. Leheup; citation_author=L. Taine; citation_author=et al.; citation_journal_title=Eur. J. Hum. Genet.; citation_volume=21; citation_firstpage=602; citation_lastpage=612; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Toward better understanding of artifacts in variant calling from high-coverage samples; citation_author=H. Li; citation_journal_title=Bioinformatics; citation_volume=30; citation_firstpage=2843; citation_lastpage=2851; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H. Li; citation_author=R. Durbin; citation_journal_title=Bioinformatics; citation_volume=25; citation_firstpage=1754; citation_lastpage=1760; citation_publication_date=2009; "><meta name="citation_reference" content="citation_title=A scored human protein-protein interaction network to catalyze genomic interpretation; citation_author=T. Li; citation_author=R. Wernersson; citation_author=R.B. Hansen; citation_author=H. Horn; citation_author=J. Mercer; citation_author=G. Slodkowicz; citation_author=C.T. Workman; citation_author=O. Rigina; citation_author=K. Rapacki; citation_author=H.H. Stærfeldt; citation_author=et al.; citation_journal_title=Nat. Methods; citation_volume=14; citation_firstpage=61; citation_lastpage=64; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Integrative functional genomic analysis of human brain development and neuropsychiatric risks; citation_author=M. Li; citation_author=G. Santpere; citation_author=Y. Imamura Kawasawa; citation_author=O.V. Evgrafov; citation_author=F.O. Gulden; citation_author=S. Pochareddy; citation_author=S.M. Sunkin; citation_author=Z. Li; citation_author=Y. Shin; citation_author=Y. Zhu; citation_author=BrainSpan Consortium; citation_author=PsychENCODE Consortium; citation_author=PsychENCODE Developmental Subgroup; citation_author=et al.; citation_journal_title=Science; citation_volume=362; citation_firstpage=eaat7615; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Rates, distribution and implications of postzygotic mosaic mutations in autism spectrum disorder; citation_author=E.T. Lim; citation_author=M. Uddin; citation_author=S. De Rubeis; citation_author=Y. Chan; citation_author=A.S. Kamumbu; citation_author=X. Zhang; citation_author=A.M. D’Gama; citation_author=S.N. Kim; citation_author=R.S. Hill; citation_author=A.P. Goldberg; citation_author=Autism Sequencing Consortium; citation_author=et al.; citation_journal_title=Nat. Neurosci.; citation_volume=20; citation_firstpage=1217; citation_lastpage=1224; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=DAWN: a framework to identify autism genes and subnetworks using gene expression and genetics; citation_author=L. Liu; citation_author=J. Lei; citation_author=S.J. Sanders; citation_author=A.J. Willsey; citation_author=Y. Kou; citation_author=A.E. Cicek; citation_author=L. Klei; citation_author=C. Lu; citation_author=X. He; citation_author=M. Li; citation_author=et al.; citation_journal_title=Mol. Autism; citation_volume=5; citation_firstpage=22; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Network assisted analysis to reveal the genetic basis of autism; citation_author=L. Liu; citation_author=J. Lei; citation_author=K. Roeder; citation_journal_title=Ann. Appl. Stat.; citation_volume=9; citation_firstpage=1571; citation_lastpage=1600; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Autism Diagnostic Interview-Revised: a revised version of a diagnostic interview for caregivers of individuals with possible pervasive developmental disorders; citation_author=C. Lord; citation_author=M. Rutter; citation_author=A. Le Couteur; citation_journal_title=J.&nbsp;Autism Dev. Disord.; citation_volume=24; citation_firstpage=659; citation_lastpage=685; citation_publication_date=1994; "><meta name="citation_reference" content="citation_title=Novel KCNQ3 mutation in a large family with benign familial neonatal epilepsy: A rare cause of neonatal seizures; citation_author=S. Maljevic; citation_author=S. Vejzovic; citation_author=M.K. Bernhard; citation_author=A. Bertsche; citation_author=S. Weise; citation_author=M. Döcker; citation_author=H. Lerche; citation_author=J.R. Lemke; citation_author=A. Merkenschlager; citation_author=S. Syrbe; citation_journal_title=Mol. Syndromol.; citation_volume=7; citation_firstpage=189; citation_lastpage=196; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Robust relationship inference in genome-wide association studies; citation_author=A. Manichaikul; citation_author=J.C. Mychaleckyj; citation_author=S.S. Rich; citation_author=K. Daly; citation_author=M. Sale; citation_author=W.M. Chen; citation_journal_title=Bioinformatics; citation_volume=26; citation_firstpage=2867; citation_lastpage=2873; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=The Ensembl Variant Effect Predictor; citation_author=W. McLaren; citation_author=L. Gil; citation_author=S.E. Hunt; citation_author=H.S. Riat; citation_author=G.R. Ritchie; citation_author=A. Thormann; citation_author=P. Flicek; citation_author=F. Cunningham; citation_journal_title=Genome Biol.; citation_volume=17; citation_firstpage=122; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Protocol Update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0); citation_author=H. Mi; citation_author=A. Muruganujan; citation_author=X. Huang; citation_author=D. Ebert; citation_author=C. Mills; citation_author=X. Guo; citation_author=P.D. Thomas; citation_journal_title=Nat. Protoc.; citation_volume=14; citation_firstpage=703; citation_lastpage=721; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Early-onset epileptic encephalopathy caused by gain-of-function mutations in the voltage sensor of Kv7.2 and Kv7.3 potassium channel subunits; citation_author=F. Miceli; citation_author=M.V. Soldovieri; citation_author=P. Ambrosino; citation_author=M. De Maria; citation_author=M. Migliore; citation_author=R. Migliore; citation_author=M. Taglialatela; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=35; citation_firstpage=3782; citation_lastpage=3793; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Analysis of family resemblance. 3. Complex segregation of quantitative traits; citation_author=N.E. Morton; citation_author=C.J. MacLean; citation_journal_title=Am. J. Hum. Genet.; citation_volume=26; citation_firstpage=489; citation_lastpage=503; citation_publication_date=1974; "><meta name="citation_reference" content="citation_title=Mullen Scales of Early Learning Manual; citation_author=E.M. Mullen; citation_publication_date=1995; citation_publisher=American Guidance Service; "><meta name="citation_reference" content="citation_title=Meta-analysis of genome-wide association studies of attention-deficit/hyperactivity disorder; citation_author=B.M. Neale; citation_author=S.E. Medland; citation_author=S. Ripke; citation_author=P. Asherson; citation_author=B. Franke; citation_author=K.P. Lesch; citation_author=S.V. Faraone; citation_author=T.T. Nguyen; citation_author=H. Schäfer; citation_author=P. Holmans; citation_author=Psychiatric GWAS Consortium: ADHD Subgroup; citation_author=et al.; citation_journal_title=J.&nbsp;Am. Acad. Child Adolesc. Psychiatry; citation_volume=49; citation_firstpage=884; citation_lastpage=897; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Patterns and rates of exonic de novo mutations in autism spectrum disorders; citation_author=B.M. Neale; citation_author=Y. Kou; citation_author=L. Liu; citation_author=A. Ma’ayan; citation_author=K.E. Samocha; citation_author=A. Sabo; citation_author=C.F. Lin; citation_author=C. Stevens; citation_author=L.S. Wang; citation_author=V. Makarov; citation_author=et al.; citation_journal_title=Nature; citation_volume=485; citation_firstpage=242; citation_lastpage=245; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex; citation_author=T.J. Nowakowski; citation_author=A. Bhaduri; citation_author=A.A. Pollen; citation_author=B. Alvarado; citation_author=M.A. Mostajo-Radji; citation_author=E. Di Lullo; citation_author=M. Haeussler; citation_author=C. Sandoval-Espinosa; citation_author=S.J. Liu; citation_author=D. Velmeshev; citation_author=et al.; citation_journal_title=Science; citation_volume=358; citation_firstpage=1318; citation_lastpage=1323; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation; citation_author=N.A. O’Leary; citation_author=M.W. Wright; citation_author=J.R. Brister; citation_author=S. Ciufo; citation_author=D. Haddad; citation_author=R. McVeigh; citation_author=B. Rajput; citation_author=B. Robbertse; citation_author=B. Smith-White; citation_author=D. Ako-Adjei; citation_author=et al.; citation_journal_title=Nucleic Acids Res.; citation_volume=44; citation_number=D1; citation_firstpage=D733; citation_lastpage=D745; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Genome-wide association study identifies 74 loci associated with educational attainment; citation_author=A. Okbay; citation_author=J.P. Beauchamp; citation_author=M.A. Fontana; citation_author=J.J. Lee; citation_author=T.H. Pers; citation_author=C.A. Rietveld; citation_author=P. Turley; citation_author=G.-B. Chen; citation_author=V. Emilsson; citation_author=S.F.W. Meddens; citation_author=LifeLines Cohort Study; citation_author=et al.; citation_journal_title=Nature; citation_volume=533; citation_firstpage=539; citation_lastpage=542; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism; citation_author=N.N. Parikshak; citation_author=R. Luo; citation_author=A. Zhang; citation_author=H. Won; citation_author=J.K. Lowe; citation_author=V. Chandran; citation_author=S. Horvath; citation_author=D.H. Geschwind; citation_journal_title=Cell; citation_volume=155; citation_firstpage=1008; citation_lastpage=1021; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=The iPSYCH2012 case-cohort sample: new directions for unravelling genetic and environmental architectures of severe mental disorders; citation_author=C.B. Pedersen; citation_author=J. Bybjerg-Grauholm; citation_author=M.G. Pedersen; citation_author=J. Grove; citation_author=E. Agerbo; citation_author=M. Bækvad-Hansen; citation_author=J.B. Poulsen; citation_author=C.S. Hansen; citation_author=J.J. McGrath; citation_author=T.D. Als; citation_author=et al.; citation_journal_title=Mol. Psychiatry; citation_volume=23; citation_firstpage=6; citation_lastpage=14; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Functional impact of global rare copy number variation in autism spectrum disorders; citation_author=D. Pinto; citation_author=A.T. Pagnamenta; citation_author=L. Klei; citation_author=R. Anney; citation_author=D. Merico; citation_author=R. Regan; citation_author=J. Conroy; citation_author=T.R. Magalhaes; citation_author=C. Correia; citation_author=B.S. Abrahams; citation_author=et al.; citation_journal_title=Nature; citation_volume=466; citation_firstpage=368; citation_lastpage=372; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation; citation_author=D. Polioudakis; citation_author=L. de la Torre-Ubieta; citation_author=J. Langerman; citation_author=A.G. Elkins; citation_author=X. Shi; citation_author=J.L. Stein; citation_author=C.K. Vuong; citation_author=S. Nichterwitz; citation_author=M. Gevorgian; citation_author=C.K. Opland; citation_author=et al.; citation_journal_title=Neuron; citation_volume=103; citation_firstpage=785; citation_lastpage=801.e8; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=Fecundity of patients with schizophrenia, autism, bipolar disorder, depression, anorexia nervosa, or substance abuse vs their unaffected siblings; citation_author=R.A. Power; citation_author=S. Kyaga; citation_author=R. Uher; citation_author=J.H. MacCabe; citation_author=N. Långström; citation_author=M. Landen; citation_author=P. McGuffin; citation_author=C.M. Lewis; citation_author=P. Lichtenstein; citation_author=A.C. Svensson; citation_journal_title=JAMA Psychiatry; citation_volume=70; citation_firstpage=22; citation_lastpage=30; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=ClinGen--the Clinical Genome Resource; citation_author=H.L. Rehm; citation_author=J.S. Berg; citation_author=L.D. Brooks; citation_author=C.D. Bustamante; citation_author=J.P. Evans; citation_author=M.J. Landrum; citation_author=D.H. Ledbetter; citation_author=D.R. Maglott; citation_author=C.L. Martin; citation_author=R.L. Nussbaum; citation_author=ClinGen; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=372; citation_firstpage=2235; citation_lastpage=2242; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Discontinuity in the genetic and environmental causes of the intellectual disability spectrum; citation_author=A. Reichenberg; citation_author=M. Cederlöf; citation_author=A. McMillan; citation_author=M. Trzaskowski; citation_author=O. Kapra; citation_author=E. Fruchter; citation_author=K. Ginat; citation_author=M. Davidson; citation_author=M. Weiser; citation_author=H. Larsson; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=113; citation_firstpage=1098; citation_lastpage=1103; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=GWAS of 126,559 individuals identifies genetic variants associated with educational attainment; citation_author=C.A. Rietveld; citation_author=S.E. Medland; citation_author=J. Derringer; citation_author=J. Yang; citation_author=T. Esko; citation_author=N.W. Martin; citation_author=H.J. Westra; citation_author=K. Shakhbazov; citation_author=A. Abdellaoui; citation_author=A. Agrawal; citation_author=LifeLines Cohort Study; citation_author=et al.; citation_journal_title=Science; citation_volume=340; citation_firstpage=1467; citation_lastpage=1471; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Genome-wide association study identifies five new schizophrenia loci; citation_author=S. Ripke; citation_author=A.R. Sanders; citation_author=K.S. Kendler; citation_author=D.F. Levinson; citation_author=P. Sklar; citation_author=P.A. Holmans; citation_author=D.-Y.Y. Lin; citation_author=J. Duan; citation_author=R.A. Ophoff; citation_author=O.A. Andreassen; citation_author=Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=43; citation_firstpage=969; citation_lastpage=976; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Genome-wide association analysis identifies 13 new risk loci for schizophrenia; citation_author=S. Ripke; citation_author=C. O’Dushlaine; citation_author=K. Chambert; citation_author=J.L. Moran; citation_author=A.K. Kähler; citation_author=S. Akterin; citation_author=S.E. Bergen; citation_author=A.L. Collins; citation_author=J.J. Crowley; citation_author=M. Fromer; citation_author=Multicenter Genetic Studies of Schizophrenia Consortium; citation_author=Psychosis Endophenotypes International Consortium; citation_author=Wellcome Trust Case Control Consortium 2; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=45; citation_firstpage=1150; citation_lastpage=1159; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=A mega-analysis of genome-wide association studies for major depressive disorder; citation_author=S. Ripke; citation_author=N.R. Wray; citation_author=C.M. Lewis; citation_author=S.P. Hamilton; citation_author=M.M. Weissman; citation_author=G. Breen; citation_author=E.M. Byrne; citation_author=D.H. Blackwood; citation_author=D.I. Boomsma; citation_author=S. Cichon; citation_author=Major Depressive Disorder Working Group of the Psychiatric GWAS Consortium; citation_author=et al.; citation_journal_title=Mol. Psychiatry; citation_volume=18; citation_firstpage=497; citation_lastpage=511; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Autism spectrum disorder severity reflects the average contribution of de novo and familial influences; citation_author=E.B. Robinson; citation_author=K.E. Samocha; citation_author=J.A. Kosmicki; citation_author=L. McGrath; citation_author=B.M. Neale; citation_author=R.H. Perlis; citation_author=M.J. Daly; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=15161; citation_lastpage=15165; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Model of autism: increased ratio of excitation/inhibition in key neural systems; citation_author=J.L. Rubenstein; citation_author=M.M. Merzenich; citation_journal_title=Genes Brain Behav.; citation_volume=2; citation_firstpage=255; citation_lastpage=267; citation_publication_date=2003; "><meta name="citation_reference" content="citation_title=Inherited and de novo genetic risk for autism impacts shared networks; citation_author=E.K. Ruzzo; citation_author=L. Pérez-Cano; citation_author=J.Y. Jung; citation_author=L.K. Wang; citation_author=D. Kashef-Haghighi; citation_author=C. Hartl; citation_author=C. Singh; citation_author=J. Xu; citation_author=J.N. Hoekstra; citation_author=O. Leventhal; citation_author=et al.; citation_journal_title=Cell; citation_volume=178; citation_firstpage=850; citation_lastpage=866.e26; citation_publication_date=2019; "><meta name="citation_reference" content="citation_title=A framework for the interpretation of de novo mutation in human disease; citation_author=K.E. Samocha; citation_author=E.B. Robinson; citation_author=S.J. Sanders; citation_author=C. Stevens; citation_author=A. Sabo; citation_author=L.M. McGrath; citation_author=J.A. Kosmicki; citation_author=K. Rehnström; citation_author=S. Mallick; citation_author=A. Kirby; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=944; citation_lastpage=950; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Regional missense constraint improves variant deleteriousness prediction; citation_author=K.E. Samocha; citation_author=J.A. Kosmicki; citation_author=K.J. Karczewski; citation_author=A.H. O’Donnell-Luria; citation_author=E. Pierce-Hoffman; citation_author=D.G. MacArthur; citation_author=B.M. Neale; citation_author=M.J. Daly; citation_journal_title=bioRxiv; citation_doi=10.1101/148353; citation_publication_date=2017; "><meta name="citation_reference" content="citation_title=De novo mutations revealed by whole-exome sequencing are strongly associated with autism; citation_author=S.J. Sanders; citation_author=M.T. Murtha; citation_author=A.R. Gupta; citation_author=J.D. Murdoch; citation_author=M.J. Raubeson; citation_author=A.J. Willsey; citation_author=A.G. Ercan-Sencicek; citation_author=N.M. DiLullo; citation_author=N.N. Parikshak; citation_author=J.L. Stein; citation_author=et al.; citation_journal_title=Nature; citation_volume=485; citation_firstpage=237; citation_lastpage=241; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci; citation_author=S.J. Sanders; citation_author=X. He; citation_author=A.J. Willsey; citation_author=A.G. Ercan-Sencicek; citation_author=K.E. Samocha; citation_author=A.E. Cicek; citation_author=M.T. Murtha; citation_author=V.H. Bal; citation_author=S.L. Bishop; citation_author=S. Dong; citation_author=Autism Sequencing Consortium; citation_author=et al.; citation_journal_title=Neuron; citation_volume=87; citation_firstpage=1215; citation_lastpage=1233; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=ASD and ADHD have a similar burden of rare protein-truncating variants; citation_author=F.K. Satterstrom; citation_author=R.K. Walters; citation_author=T. Singh; citation_author=E.M. Wigdor; citation_author=F. Lescai; citation_author=D. Demontis; citation_author=J.A. Kosmicki; citation_author=J. Grove; citation_author=C. Stevens; citation_author=J. Bybjerg-Grauholm; citation_author=et al.; citation_journal_title=bioRxiv; citation_doi=10.1101/277707; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Clinical genetics evaluation in identifying the etiology of autism spectrum disorders: 2013 guideline revisions; citation_author=G.B. Schaefer; citation_author=N.J. Mendelsohn; citation_author=Professional Practice and Guidelines Committee; citation_journal_title=Genet. Med.; citation_volume=15; citation_firstpage=399; citation_lastpage=407; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Biological insights from 108 schizophrenia-associated genetic loci; citation_author=Schizophrenia Working Group of the Psychiatric Genomics Consortium; citation_journal_title=Nature; citation_volume=511; citation_firstpage=421; citation_lastpage=427; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Strong association of de novo copy number mutations with autism; citation_author=J. Sebat; citation_author=B. Lakshmi; citation_author=D. Malhotra; citation_author=J. Troge; citation_author=C. Lese-Martin; citation_author=T. Walsh; citation_author=B. Yamrom; citation_author=S. Yoon; citation_author=A. Krasnitz; citation_author=J. Kendall; citation_author=et al.; citation_journal_title=Science; citation_volume=316; citation_firstpage=445; citation_lastpage=449; citation_publication_date=2007; "><meta name="citation_reference" content="citation_title=Prospective investigation of autism and genotype-phenotype correlations in 22q13 deletion syndrome and SHANK3 deficiency; citation_author=L. Soorya; citation_author=A. Kolevzon; citation_author=J. Zweifach; citation_author=T. Lim; citation_author=Y. Dobry; citation_author=L. Schwartz; citation_author=Y. Frank; citation_author=A.T. Wang; citation_author=G. Cai; citation_author=E. Parkhomenko; citation_author=et al.; citation_journal_title=Mol. Autism; citation_volume=4; citation_firstpage=18; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=PRIMUS: rapid reconstruction of pedigrees from genome-wide estimates of identity by descent; citation_author=J. Staples; citation_author=D. Qiao; citation_author=M.H. Cho; citation_author=E.K. Silverman; citation_author=D.A. Nickerson; citation_author=J.E. Below; citation_author=University of Washington Center for Mendelian Genomics; citation_journal_title=Am. J. Hum. Genet.; citation_volume=95; citation_firstpage=553; citation_lastpage=564; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=CHD8 regulates neurodevelopmental pathways associated with autism spectrum disorder in neural progenitors; citation_author=A. Sugathan; citation_author=M. Biagioli; citation_author=C. Golzio; citation_author=S. Erdin; citation_author=I. Blumenthal; citation_author=P. Manavalan; citation_author=A. Ragavendran; citation_author=H. Brand; citation_author=D. Lucente; citation_author=J. Miles; citation_author=et al.; citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_volume=111; citation_firstpage=E4468; citation_lastpage=E4477; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline; citation_author=G.A. Van der Auwera; citation_author=M.O. Carneiro; citation_author=C. Hartl; citation_author=R. Poplin; citation_author=G. Del Angel; citation_author=A. Levy-Moonshine; citation_author=T. Jordan; citation_author=K. Shakir; citation_author=D. Roazen; citation_author=J. Thibault; citation_author=et al.; citation_journal_title=Curr. Protoc. Bioinformatics; citation_volume=11; citation_firstpage=11.10.1; citation_lastpage=11.10.33; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Transcriptomic analysis of autistic brain reveals convergent molecular pathology; citation_author=I. Voineagu; citation_author=X. Wang; citation_author=P. Johnston; citation_author=J.K. Lowe; citation_author=Y. Tian; citation_author=S. Horvath; citation_author=J. Mill; citation_author=R.M. Cantor; citation_author=B.J. Blencowe; citation_author=D.H. Geschwind; citation_journal_title=Nature; citation_volume=474; citation_firstpage=380; citation_lastpage=384; citation_publication_date=2011; "><meta name="citation_reference" content="citation_title=Mutations affecting the SAND domain of DEAF1 cause intellectual disability with severe speech impairment and behavioral problems; citation_author=A.T. Vulto-van Silfhout; citation_author=S. Rajamanickam; citation_author=P.J. Jensik; citation_author=S. Vergult; citation_author=N. de Rocker; citation_author=K.J. Newhall; citation_author=R. Raghavan; citation_author=S.N. Reardon; citation_author=K. Jarrett; citation_author=T. McIntyre; citation_author=et al.; citation_journal_title=Am. J. Hum. Genet.; citation_volume=94; citation_firstpage=649; citation_lastpage=661; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Chromosomal microarray versus karyotyping for prenatal diagnosis; citation_author=R.J. Wapner; citation_author=C.L. Martin; citation_author=B. Levy; citation_author=B.C. Ballif; citation_author=C.M. Eng; citation_author=J.M. Zachary; citation_author=M. Savage; citation_author=L.D. Platt; citation_author=D. Saltzman; citation_author=W.A. Grobman; citation_author=et al.; citation_journal_title=N.&nbsp;Engl. J. Med.; citation_volume=367; citation_firstpage=2175; citation_lastpage=2184; citation_publication_date=2012; "><meta name="citation_reference" content="citation_title=WISC III (Wechsler Intelligence Scale for Children); citation_author=D. Wechsler; citation_publication_date=1992; citation_publisher=The Psychological Corporation; "><meta name="citation_reference" content="citation_title=Wechsler Abbreviated Scale of Intelligence; citation_author=D. Wechsler; citation_publication_date=1999; citation_publisher=The Psychological Corporation; "><meta name="citation_reference" content="citation_title=The role of sex-differential biology in risk for autism spectrum disorder; citation_author=D.M. Werling; citation_journal_title=Biol. Sex Differ.; citation_volume=7; citation_firstpage=58; citation_publication_date=2016; "><meta name="citation_reference" content="citation_title=An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder; citation_author=D.M. Werling; citation_author=H. Brand; citation_author=J.Y. An; citation_author=M.R. Stone; citation_author=L. Zhu; citation_author=J.T. Glessner; citation_author=R.L. Collins; citation_author=S. Dong; citation_author=R.M. Layer; citation_author=E. Markenscoff-Papadimitriou; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=26; citation_firstpage=727; citation_lastpage=736; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Haploinsufficiency of HDAC4 causes brachydactyly mental retardation syndrome, with brachydactyly type E, developmental delays, and behavioral problems; citation_author=S.R. Williams; citation_author=M.A. Aldred; citation_author=V.M. Der Kaloustian; citation_author=F. Halal; citation_author=G. Gowans; citation_author=D.R. McLeod; citation_author=S. Zondag; citation_author=H.V. Toriello; citation_author=R.E. Magenis; citation_author=S.H. Elsea; citation_journal_title=Am. J. Hum. Genet.; citation_volume=87; citation_firstpage=219; citation_lastpage=228; citation_publication_date=2010; "><meta name="citation_reference" content="citation_title=Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism; citation_author=A.J. Willsey; citation_author=S.J. Sanders; citation_author=M. Li; citation_author=S. Dong; citation_author=A.T. Tebbenkamp; citation_author=R.A. Muhle; citation_author=S.K. Reilly; citation_author=L. Lin; citation_author=S. Fertuzinhos; citation_author=J.A. Miller; citation_author=et al.; citation_journal_title=Cell; citation_volume=155; citation_firstpage=997; citation_lastpage=1007; citation_publication_date=2013; "><meta name="citation_reference" content="citation_title=Defining the role of common variation in the genomic and biological architecture of adult human height; citation_author=A.R. Wood; citation_author=T. Esko; citation_author=J. Yang; citation_author=S. Vedantam; citation_author=T.H. Pers; citation_author=S. Gustafsson; citation_author=A.Y. Chu; citation_author=K. Estrada; citation_author=J. Luan; citation_author=Z. Kutalik; citation_author=Electronic Medical Records and Genomics (eMEMERGEGE) Consortium; citation_author=MIGen Consortium; citation_author=PAGEGE Consortium; citation_author=LifeLines Cohort Study; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=46; citation_firstpage=1173; citation_lastpage=1186; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression; citation_author=N.R. Wray; citation_author=S. Ripke; citation_author=M. Mattheisen; citation_author=M. Trzaskowski; citation_author=E.M. Byrne; citation_author=A. Abdellaoui; citation_author=M.J. Adams; citation_author=E. Agerbo; citation_author=T.M. Air; citation_author=T.M.F. Andlauer; citation_author=eQTLGen; citation_author=23andMe; citation_author=Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium; citation_author=et al.; citation_journal_title=Nat. Genet.; citation_volume=50; citation_firstpage=668; citation_lastpage=681; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data; citation_author=C.F. Wright; citation_author=T.W. Fitzgerald; citation_author=W.D. Jones; citation_author=S. Clayton; citation_author=J.F. McRae; citation_author=M. van Kogelenberg; citation_author=D.A. King; citation_author=K. Ambridge; citation_author=D.M. Barrett; citation_author=T. Bayzetinova; citation_author=DDD study; citation_author=et al.; citation_journal_title=Lancet; citation_volume=385; citation_firstpage=1305; citation_lastpage=1314; citation_publication_date=2015; "><meta name="citation_reference" content="citation_title=Cell type-specific expression analysis to identify putative cellular mechanisms for neurogenetic disorders; citation_author=X. Xu; citation_author=A.B. Wells; citation_author=D.R. O’Brien; citation_author=A. Nehorai; citation_author=J.D. Dougherty; citation_journal_title=J.&nbsp;Neurosci.; citation_volume=34; citation_firstpage=1420; citation_lastpage=1431; citation_publication_date=2014; "><meta name="citation_reference" content="citation_title=Meta-analysis of genome-wide association studies for height and body mass index in ∼700000 individuals of European ancestry; citation_author=L. Yengo; citation_author=J. Sidorenko; citation_author=K.E. Kemper; citation_author=Z. Zheng; citation_author=A.R. Wood; citation_author=M.N. Weedon; citation_author=T.M. Frayling; citation_author=J. Hirschhorn; citation_author=J. Yang; citation_author=P.M. Visscher; citation_author=GIANT Consortium; citation_journal_title=Hum. Mol. Genet.; citation_volume=27; citation_firstpage=3641; citation_lastpage=3649; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=Heritable variation, with little or no maternal effect, accounts for recurrence risk to autism spectrum disorder in Sweden; citation_author=B.H.K. Yip; citation_author=D. Bai; citation_author=B. Mahjani; citation_author=L. Klei; citation_author=Y. Pawitan; citation_author=C.M. Hultman; citation_author=D.E. Grice; citation_author=K. Roeder; citation_author=J.D. Buxbaum; citation_author=B. Devlin; citation_author=et al.; citation_journal_title=Biol. Psychiatry; citation_volume=83; citation_firstpage=589; citation_lastpage=597; citation_publication_date=2018; "><meta name="citation_reference" content="citation_title=LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis; citation_author=J. Zheng; citation_author=A.M. Erzurumluoglu; citation_author=B.L. Elsworth; citation_author=J.P. Kemp; citation_author=L. Howe; citation_author=P.C. Haycock; citation_author=G. Hemani; citation_author=K. Tansey; citation_author=C. Laurin; citation_author=B.S. Pourcain; citation_author=Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium; citation_author=et al.; citation_journal_title=Bioinformatics; citation_volume=33; citation_firstpage=272; citation_lastpage=279; citation_publication_date=2017; "><meta name="citation_abstract" content="<h2>Summary</h2><p>We present the largest exome sequencing study of autism spectrum disorder (ASD) to date (n&nbsp;= 35,584 total samples, 11,986 with ASD). Using an enhanced analytical framework to integrate <i>de novo</i> and case-control rare variation, we identify 102 risk genes at a false discovery rate of 0.1 or less. Of these genes, 49 show higher frequencies of disruptive <i>de novo</i> variants in individuals ascertained to have severe neurodevelopmental delay, whereas 53 show higher frequencies in individuals ascertained to have ASD; comparing ASD cases with mutations in these groups reveals phenotypic differences. Expressed early in brain development, most risk genes have roles in regulation of gene expression or neuronal communication (i.e., mutations effect neurodevelopmental and neurophysiological changes), and 13 fall within loci recurrently hit by copy number variants. In cells from the human cortex, expression of risk genes is enriched in excitatory and inhibitory neuronal lineages, consistent with multiple paths to an excitatory-inhibitory imbalance underlying ASD.</p>"><meta charset="UTF-8"><!--

    Pagebuild admin UI
-->



<script type="text/javascript" src="https://bam.nr-data.net/1/16e2192192?a=916957862&amp;sa=1&amp;v=1177.96a4d39&amp;t=Unnamed%20Transaction&amp;rst=6385&amp;ck=1&amp;ref=https://www.cell.com/cell/fulltext/S0092-8674(19)31398-4&amp;be=3366&amp;fe=6192&amp;dc=5434&amp;af=err,xhr,stn,ins,spa&amp;perf=%7B%22timing%22:%7B%22of%22:1749585506943,%22n%22:0,%22f%22:2610,%22dn%22:2611,%22dne%22:2611,%22c%22:2611,%22s%22:2611,%22ce%22:2611,%22rq%22:2611,%22rp%22:3338,%22rpe%22:4709,%22dl%22:3343,%22di%22:5425,%22ds%22:5433,%22de%22:5766,%22dc%22:6191,%22l%22:6191,%22le%22:6313%7D,%22navigation%22:%7B%7D%7D&amp;fp=3856&amp;fcp=3856&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-spa-1177.min.js"></script><script type="text/javascript">
    ;window.NREUM||(NREUM={});NREUM.init={privacy:{cookies_enabled:true}};
    window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{c.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(26),c={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(c.console=!0,o.indexOf("dev")!==-1&&(c.dev=!0),o.indexOf("nr_dev")!==-1&&(c.nrDev=!0))}catch(s){}c.nrDev&&i.on("internal-error",function(t){r(t.stack)}),c.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),c.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(c,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,c){try{l?l-=1:o(c||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,s.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:s.now();i("err",[t,n])}var i=t("handle"),a=t(27),c=t("ee"),s=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;s.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),s.xhrWrappable&&t(14),d=!0)}c.on("fn-start",function(t,e,n){d&&(l+=1)}),c.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),c.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),c.on("internal-error",function(t){i("ierr",[t,s.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){_++,T=g.hash,this[u]=y.now()}function o(){_--,g.hash!==T&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var c="-start",s="-end",f="-body",u="fn"+c,d="fn"+s,p="cb"+c,l="cb"+s,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable){var x=t(10),b=t(11),E=t(8),R=t(6),O=t(13),N=t(7),M=t(14),P=t(9),S=t("ee"),C=S.get("tracer");t(16),y.features.spa=!0;var T,_=0;S.on(u,r),S.on(p,r),S.on(d,o),S.on(l,o),S.buffer([u,d,"xhr-done","xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+s,"clearTimeout"+c,u]),M.buffer([u,"new-xhr","send-xhr"+c]),N.buffer([m+c,m+"-done",m+f+c,m+f+s]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+c]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(M,"send-xhr"+c),a(S,"xhr-resolved"),a(S,"xhr-done"),a(N,m+c),a(N,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,_>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),c=t(12),s="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",g="pushState",y=t("loader");y.features.stn=!0,t(8),"addEventListener"in window&&t(6);var x=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof x&&(this.bstStart=y.now())}),o.on(v,function(t,e){var n=t[0];n instanceof x&&i("bst",[n,e,this.bstStart,y.now()])}),a.on(m,function(t,e,n){this.bstStart=y.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),this.bstType])}),c.on(m,function(){this.bstStart=y.now()}),c.on(v,function(t,e){i(w,[e,this.bstStart,y.now(),"requestAnimationFrame"])}),o.on(g+l,function(t){this.time=y.now(),this.startPath=location.pathname+location.hash}),o.on(g+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+s]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+s]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+s]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){c.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),c=t("wrap-function")(a,!0),s=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=s(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?c(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var c=r.apply(this,t);return o.emit(n+"start",[t,a],c),c.then(function(t){return o.emit(n+"end",[null,t],c),t},function(t){throw o.emit(n+"end",[t],c),t})})}var o=t("ee").get("fetch"),i=t(27),a=t(26);e.exports=o;var c=window,s="fetch-",f=s+"body-",u=["arrayBuffer","blob","json","text","formData"],d=c.Request,p=c.Response,l=c.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(c,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){s.emit("jsonp-error",[],p),s.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=c(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),s.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function c(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var s=t("ee").get("jsonp"),f=t("wrap-function")(s);if(e.exports=s,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?f.inPlace(Node.prototype,l,"dom-"):(f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-")),s.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=c(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t("wrap-function"),a=t("ee").get("promise"),c=i(a),s=t(26),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;s(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),c=f.resolve(i);return c}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),s(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=c(t[0],"resolve-",this),t[1]=c(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),c.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=c(t[0],"cb-",this),t[1]=c(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),c="setTimeout",s="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[c,"setImmediate"],c+d),a.inPlace(window,[s],s+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(s+u,r),i.on(c+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",c)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,g,"fn-",c)}function i(t){y.push(t),h&&(b?b.then(a):v?v(a):(E=-E,R.data=E))}function a(){for(var t=0;t<y.length;t++)r([],y[t]);y.length&&(y=[])}function c(t,e){return e}function s(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",g=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],y=[];e.exports=u;var x=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(s(l,x),x.prototype=l.prototype,d.inPlace(x.prototype,["open","send"],"-xhr-",c),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var b=m&&m.resolve();if(!v&&!m){var E=1,R=document.createTextNode(E);new h(a).observe(R,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){if(!c(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||s(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&s(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",c=1,s="",f="";return o+"@nr="+i+"-"+c+"-"+n+"-"+r+"-"+t+"-"+a+"-"+s+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var c={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(c.d.tk=i),btoa(JSON.stringify(c))}function c(t){return f()&&s(t)}function s(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(23),h=t(17);e.exports={generateTracePayload:r,shouldGenerateTrace:c}},{}],16:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,u.emit("xhr-done",[t],t),c("xhr",[e,n,this.startTime]))}}function o(t,e){var n=s(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=s(e),t.sameOrigin=t.parsedOrigin.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable){var c=t("handle"),s=t(17),f=t(15).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(21),m=t(20),v=t(18),w=window.XMLHttpRequest;a.features.xhr=!0,t(14),t(7),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,t.addEventListener("load",function(n){i(e,t)},!1),h&&(h>34||h<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var c=0;c<p;c++)e.addEventListener(d[c],this.listener,!1)}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("addEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof w&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof w&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url&&(n=t[0].url),n&&(this.parsedOrigin=s(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)})}},{}],17:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,c=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||c),"/"===o.pathname&&(r[t]=o),o}},{}],18:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(20);e.exports=r},{}],19:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(c(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(26),c=t(27),s=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(s.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw s.emit("fn-err",[arguments,this,t],n),t}finally{s.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,e])}},{}],20:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],21:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],22:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?d("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&d("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();n.length>0&&d("lcp",[n[n.length-1]])}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||d("cls",[t])})}function a(t){if(t instanceof h&&!v){var e=Math.round(t.timeStamp),n={type:t.type};e<=p.now()?n.fid=p.now()-e:e>p.offset&&e<=Date.now()?(e-=p.offset,n.fid=p.now()-e):e=p.now(),v=!0,d("timing",["fi",e,n])}}function c(t){d("pageHide",[p.now(),t])}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var s,f,u,d=t("handle"),p=t("loader"),l=t(25),h=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){s=new PerformanceObserver(r);try{s.observe({entryTypes:["paint"]})}catch(m){}f=new PerformanceObserver(o);try{f.observe({entryTypes:["largest-contentful-paint"]})}catch(m){}u=new PerformanceObserver(i);try{u.observe({type:"layout-shift",buffered:!0})}catch(m){}}if("addEventListener"in document){var v=!1,w=["click","keydown","mousedown","pointerdown","touchstart"];w.forEach(function(t){document.addEventListener(t,a,!1)})}l(c)}},{}],23:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",c=0;c<i.length;c++)o=i[c],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(31)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],24:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var c=navigator.userAgent,s=c.match(a);s&&c.indexOf("Chrome")===-1&&c.indexOf("Chromium")===-1&&(o="Safari",i=s[1])}e.exports={agent:o,version:i,match:r}},{}],25:[function(t,e,n){function r(t){function e(){t(a&&document[a]?document[a]:document[o]?"hidden":"visible")}"addEventListener"in document&&i&&document.addEventListener(i,e,!1)}e.exports=r;var o,i,a;"undefined"!=typeof document.hidden?(o="hidden",i="visibilitychange",a="visibilityState"):"undefined"!=typeof document.msHidden?(o="msHidden",i="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(o="webkitHidden",i="webkitvisibilitychange",a="webkitVisibilityState")},{}],26:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],27:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],28:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?s(t,c,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),c=m(n),s=c.length,f=0;f<s;f++)c[f].apply(a,r);var d=u[y[n]];return d&&d.push([x,n,r,a]),a}}function l(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",y[n]=e,e in u||(u[e]=[])})}var g={},y={},x={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return x}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var c="nr@context",s=t("gos"),f=t(26),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=b.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(y,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+b.offset],null,"api"),s("timing",["load",n]);var r=l.createElement("script");r.src="https://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===l.readyState&&i()}function i(){s("mark",["domContent",a()+b.offset],null,"api")}function a(){return R.exists&&performance.now?Math.round(performance.now()):(c=Math.max((new Date).getTime(),c))-b.offset}var c=(new Date).getTime(),s=t("handle"),f=t(26),u=t("ee"),d=t(24),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var g=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1177.min.js"},x=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),b=e.exports={offset:c,now:a,origin:g,features:{},xhrWrappable:x,userAgent:d};t(19),t(22),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),s("mark",["firstbyte",c],null,"api");var E=0,R=t(28)},{}],"wrap-function":[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(27),a="nr@original",c=Object.prototype.hasOwnProperty,s=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,c,s;try{a=this,r=i(arguments),c="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],c])}u(e+"start",[r,a,o],c);try{return s=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],c),d}finally{u(e+"end",[r,a,s],c)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,c,s,f="-"===o.charAt(0);for(s=0;s<e.length;s++)c=e[s],a=t[c],r(a)||(t[c]=n(a,f?c+o:o,i,c))}function u(n,r,o){if(!s||e){var i=s;s=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}s=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)c.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}]},{},["loader",2,16,5,3,4]);
    ;NREUM.loader_config={accountID:"2095297",trustKey:"2038175",agentID:"916957862",licenseKey:"16e2192192",applicationID:"916957862"}
    ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"16e2192192",applicationID:"916957862",sa:1}
</script>
    <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js" type="text/javascript" charset="UTF-8" data-domain-script="dada9834-04e2-4bcd-b5ba-cae44d498dfa"></script>
    <script type="text/javascript">
        function getCookie(name) {
            var value = "; " + document.cookie;
            var parts = value.split("; " + name + "=");
            if (parts.length == 2) {
                return true;
            }
        }
            function OptanonWrapper() {
                var optCookie = getCookie("OptanonAlertBoxClosed");
                var isActiveCategory = "false";
                var isFoundCookieBox = "false";
                if(isActiveCategory == 'false' && isFoundCookieBox == 'false' && optCookie) {
                    OneTrust.InsertScript('/pb-assets/adobe/prod/adobedtm-1616606406233.js', 'head', null, null, '2');
                }
            }
    </script>


<noscript>
    <style>.requiresJS {
        display: none
    }

    .dartAd {
        display: block !important
    }</style>
</noscript>
        <script type="text/javascript" src="/wro/o4ny~selected-legacy.js"></script>
        <link rel="stylesheet" type="text/css" href="/wro/o4ny~selected-legacy.css">
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
        <script>window.MSInputMethodContext && document.documentMode && document.write('<script src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js"><\x2fscript>');</script><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/build-b26f3342b722959c7931.css"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/print-b26f3342b722959c7931.css" media="print"><link rel="stylesheet" href="/products/marlin/cell/releasedAssets/css/themes/theme-cell.css">







        
        
        <script class="optanon-category-4" type="text/plain" async="" src="https://www.youtube.com/iframe_api"></script>
        
    





<!---->
<meta http-equiv="X-UA-Compatible" content="IE=edge">












<script src="https://cdn.cookielaw.org/scripttemplates/202411.2.0/otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk:focus{outline:2px solid #000;outline-offset:-2px}#onetrust-banner-sdk a:focus{outline:2px solid #000}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{outline-offset:1px}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{height:64px;width:64px}#onetrust-banner-sdk .ot-tcf2-vendor-count.ot-text-bold{font-weight:bold}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon,#ot-sync-ntfy .ot-close-icon{background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a,#ot-sync-ntfy .powered-by-logo,#ot-sync-ntfy .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block;text-decoration:none;font-size:.75em}#onetrust-banner-sdk .powered-by-logo:hover,#onetrust-banner-sdk .ot-pc-footer-logo a:hover,#onetrust-pc-sdk .powered-by-logo:hover,#onetrust-pc-sdk .ot-pc-footer-logo a:hover,#ot-sync-ntfy .powered-by-logo:hover,#ot-sync-ntfy .ot-pc-footer-logo a:hover{color:#565656}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *,#ot-sync-ntfy h3 *,#ot-sync-ntfy h4 *,#ot-sync-ntfy h6 *,#ot-sync-ntfy button *,#ot-sync-ntfy a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide,#ot-sync-ntfy .ot-hide{display:none !important}#onetrust-banner-sdk button.ot-link-btn:hover,#onetrust-pc-sdk button.ot-link-btn:hover,#ot-sync-ntfy button.ot-link-btn:hover{text-decoration:underline;opacity:1}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type=checkbox]:checked,#onetrust-pc-sdk [type=checkbox]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type=checkbox]:disabled+label::before,#onetrust-pc-sdk [type=checkbox]:disabled+label:after,#onetrust-pc-sdk [type=checkbox]:disabled+label{pointer-events:none;opacity:.8}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type=checkbox]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type=checkbox]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px .1ex}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat;display:inline-flex;justify-content:center;align-items:center}#onetrust-pc-sdk .pc-logo img,#onetrust-pc-sdk .ot-pc-logo img{max-height:100%;max-width:100%}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr,.ot-sdk-cookie-policy .screen-reader-only,.ot-sdk-cookie-policy .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in,#onetrust-banner-sdk.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#fff;background-color:#468254;height:auto;white-space:normal;word-wrap:break-word;padding:.8em 2em;font-size:.8em;line-height:1.2;cursor:pointer;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#2c6415}#ot-sdk-btn.ot-sdk-show-settings:active,#ot-sdk-btn.optanon-show-settings:active{color:#fff;background-color:#2c6415;border:1px solid rgba(162,192,169,.5)}.onetrust-pc-dark-filter{background:rgba(0,0,0,.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}.ot-cookie-label{text-decoration:underline}@media only screen and (min-width: 426px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk p{font-size:.75em}}#onetrust-banner-sdk .banner-option-input:focus+label{outline:1px solid #000;outline-style:auto}.category-vendors-list-handler+a:focus,.category-vendors-list-handler+a:focus-visible{outline:2px solid #000}#onetrust-pc-sdk .ot-userid-title{margin-top:10px}#onetrust-pc-sdk .ot-userid-title>span,#onetrust-pc-sdk .ot-userid-timestamp>span{font-weight:700}#onetrust-pc-sdk .ot-userid-desc{font-style:italic}#onetrust-pc-sdk .ot-host-desc a{pointer-events:initial}#onetrust-pc-sdk .ot-ven-hdr>p a{position:relative;z-index:2;pointer-events:initial}#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info a{margin-right:auto}#onetrust-pc-sdk .ot-pc-footer-logo img{width:136px;height:16px}#onetrust-pc-sdk .ot-pur-vdr-count{font-weight:400;font-size:.7rem;padding-top:3px;display:block}#onetrust-banner-sdk .ot-optout-signal,#onetrust-pc-sdk .ot-optout-signal{border:1px solid #32ae88;border-radius:3px;padding:5px;margin-bottom:10px;background-color:#f9fffa;font-size:.85rem;line-height:2}#onetrust-banner-sdk .ot-optout-signal .ot-optout-icon,#onetrust-pc-sdk .ot-optout-signal .ot-optout-icon{display:inline;margin-right:5px}#onetrust-banner-sdk .ot-optout-signal svg,#onetrust-pc-sdk .ot-optout-signal svg{height:20px;width:30px;transform:scale(0.5)}#onetrust-banner-sdk .ot-optout-signal svg path,#onetrust-pc-sdk .ot-optout-signal svg path{fill:#32ae88}#onetrust-consent-sdk .ot-general-modal{overflow:hidden;position:fixed;margin:0 auto;top:50%;left:50%;width:40%;padding:1.5rem;max-width:575px;min-width:575px;z-index:2147483647;border-radius:2.5px;transform:translate(-50%, -50%)}#onetrust-consent-sdk .ot-signature-health-group{margin-top:1rem;padding-left:1.25rem;padding-right:1.25rem;margin-bottom:.625rem;width:calc(100% - 2.5rem)}#onetrust-consent-sdk .ot-signature-health-group .ot-signature-health-form{gap:.5rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:70%;gap:.35rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-input{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-consent-sdk .ot-signature-health .ot-signature-subtitle{font-size:1.125rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-group-title{font-size:1.25rem;font-weight:bold}#onetrust-consent-sdk .ot-signature-health,#onetrust-consent-sdk .ot-signature-health-group{display:flex;flex-direction:column;gap:1rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-cont{display:flex;flex-direction:column;gap:.25rem}#onetrust-consent-sdk .ot-signature-health .ot-signature-paragraph,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-paragraph{margin:0;line-height:20px;font-size:max(14px,.875rem)}#onetrust-consent-sdk .ot-signature-health .ot-health-signature-error,#onetrust-consent-sdk .ot-signature-health-group .ot-health-signature-error{color:#4d4d4d;font-size:min(12px,.75rem)}#onetrust-consent-sdk .ot-signature-health .ot-signature-buttons-cont,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-buttons-cont{margin-top:max(.75rem,2%);gap:1rem;display:flex;justify-content:flex-end}#onetrust-consent-sdk .ot-signature-health .ot-signature-button,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button{flex:1;height:auto;color:#fff;cursor:pointer;line-height:1.2;min-width:125px;font-weight:600;font-size:.813em;border-radius:2px;padding:12px 10px;white-space:normal;word-wrap:break-word;word-break:break-word;background-color:#68b631;border:2px solid #68b631}#onetrust-consent-sdk .ot-signature-health .ot-signature-button.reject,#onetrust-consent-sdk .ot-signature-health-group .ot-signature-button.reject{background-color:#fff}#onetrust-consent-sdk .ot-input-field-cont{display:flex;flex-direction:column;gap:.5rem}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:65%}#onetrust-consent-sdk .ot-signature-health-form{display:flex;flex-direction:column}#onetrust-consent-sdk .ot-signature-health-form .ot-signature-label{margin-bottom:0;line-height:20px;font-size:max(14px,.875rem)}@media only screen and (max-width: 600px){#onetrust-consent-sdk .ot-general-modal{min-width:100%}#onetrust-consent-sdk .ot-signature-health .ot-signature-health-form{width:100%}#onetrust-consent-sdk .ot-input-field-cont .ot-signature-input{width:100%}}#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy,#ot-sync-ntfy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before,#ot-sync-ntfy *,#ot-sync-ntfy ::after,#ot-sync-ntfy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox,#ot-sync-ntfy div,#ot-sync-ntfy span,#ot-sync-ntfy h1,#ot-sync-ntfy h2,#ot-sync-ntfy h3,#ot-sync-ntfy h4,#ot-sync-ntfy h5,#ot-sync-ntfy h6,#ot-sync-ntfy p,#ot-sync-ntfy img,#ot-sync-ntfy svg,#ot-sync-ntfy button,#ot-sync-ntfy section,#ot-sync-ntfy a,#ot-sync-ntfy label,#ot-sync-ntfy input,#ot-sync-ntfy ul,#ot-sync-ntfy li,#ot-sync-ntfy nav,#ot-sync-ntfy table,#ot-sync-ntfy thead,#ot-sync-ntfy tr,#ot-sync-ntfy td,#ot-sync-ntfy tbody,#ot-sync-ntfy .ot-main-content,#ot-sync-ntfy .ot-toggle,#ot-sync-ntfy #ot-content,#ot-sync-ntfy #ot-pc-content,#ot-sync-ntfy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before,#ot-sync-ntfy label:before,#ot-sync-ntfy label:after,#ot-sync-ntfy .checkbox:after,#ot-sync-ntfy .checkbox:before{content:"";content:none}#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media(min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media(min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media(min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:.9em;font-weight:400;line-height:38px;letter-spacing:.01em;text-decoration:none;white-space:nowrap;background-color:rgba(0,0,0,0);border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:not(.ot-link-btn):focus{color:#333;border-color:#888;opacity:.9}#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus{outline:2px solid #000}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type=button].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=submit].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=reset].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type=button].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=submit].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=reset].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type=button].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type=text],#onetrust-pc-sdk input[type=text],#ot-sdk-cookie-policy input[type=text]{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk input[type=text]:focus,#onetrust-pc-sdk input[type=text]:focus,#ot-sdk-cookie-policy input[type=text]:focus{border:1px solid #000;outline:0}#onetrust-banner-sdk label,#onetrust-pc-sdk label,#ot-sdk-cookie-policy label{display:block;margin-bottom:.5rem;font-weight:600}#onetrust-banner-sdk input[type=checkbox],#onetrust-pc-sdk input[type=checkbox],#ot-sdk-cookie-policy input[type=checkbox]{display:inline}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-pc-sdk ul ul,#ot-sdk-cookie-policy ul ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk.otFlat.top{top:0px;bottom:auto}#onetrust-banner-sdk.otRelFont{font-size:1rem}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy .ot-gv-list-handler{float:left;font-size:.82em;padding:0;margin-bottom:0;border:0;line-height:normal;height:auto;width:auto}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-left:0;padding-right:0}#onetrust-banner-sdk .onetrust-close-btn-ui{width:44px;height:44px;background-size:12px;border:none;position:relative;margin:auto;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk.ot-bnr-w-logo .ot-bnr-logo{position:absolute;top:50%;transform:translateY(-50%);left:0px}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-policy{margin-left:65px}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk .banner-option-input{cursor:pointer;width:auto;height:auto;border:none;padding:0;padding-right:3px;margin:0 0 10px;font-size:.82em;line-height:1.4}#onetrust-banner-sdk .banner-option-input *{pointer-events:none;font-size:inherit;line-height:inherit}#onetrust-banner-sdk .banner-option-input[aria-expanded=true]~.banner-option-details{display:block;height:auto}#onetrust-banner-sdk .banner-option-input[aria-expanded=true] .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option{margin-bottom:12px;margin-left:0;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:first-child{padding-left:2px}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-header{cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid rgba(0,0,0,0);border-bottom:6px solid rgba(0,0,0,0);border-left:6px solid dimgray;margin-left:10px;vertical-align:middle}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both;margin-bottom:0;padding:0;border:0;height:auto;width:auto}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk.ot-close-btn-link{padding-top:25px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container{top:15px;transform:none;right:15px}#onetrust-banner-sdk.ot-close-btn-link #onetrust-close-btn-container button{padding:0;white-space:pre-wrap;border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:6px;right:2px}#onetrust-banner-sdk #onetrust-policy{margin-left:0;margin-top:3em}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{top:auto;transform:none}#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-close-btn-container{position:absolute;top:0;right:0}#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui{top:10px;right:10px}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk.ot-bnr-w-logo #onetrust-button-group-parent{padding-left:50px}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-accept-btn-handler,#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group #onetrust-reject-all-handler{float:left}}@media only screen and (min-width: 769px){#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}}@media only screen and (min-width: 897px)and (max-width: 1023px){#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:75%;transform:translateY(-50%)}#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{position:relative;margin:0;right:-22px;top:2px}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-close-btn-container{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;right:0}#onetrust-banner-sdk #onetrust-close-btn-container button{right:-12px}#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk .ot-optout-signal{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:30%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:3%;padding-right:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #FFF;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options *,
                    #onetrust-banner-sdk .ot-cat-header,
                    #onetrust-banner-sdk .ot-optout-signal
                    {
                        color: #2E2E2E;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
             #onetrust-consent-sdk #onetrust-banner-sdk a[href],
                    #onetrust-consent-sdk #onetrust-banner-sdk a[href] font,
                    #onetrust-consent-sdk #onetrust-banner-sdk .ot-link-btn
                        {
                            color: #007398;
                        }#onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-banner-sdk *:focus,
            #onetrust-consent-sdk #onetrust-banner-sdk:focus {
               outline-color: #000000;
               outline-width: 1px;
            }
            #onetrust-consent-sdk #onetrust-pc-btn-handler,
            #onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                color: #6CC04A; border-color: #6CC04A;
                background-color:
                #FFF;
            }/*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
#onetrust-pc-sdk.otPcCenter{overflow:hidden;position:fixed;margin:0 auto;top:5%;right:0;left:0;width:40%;max-width:575px;min-width:575px;border-radius:2.5px;z-index:2147483647;background-color:#fff;-webkit-box-shadow:0px 2px 10px -3px #999;-moz-box-shadow:0px 2px 10px -3px #999;box-shadow:0px 2px 10px -3px #999}#onetrust-pc-sdk.otPcCenter[dir=rtl]{right:0;left:0}#onetrust-pc-sdk.otRelFont{font-size:1rem}#onetrust-pc-sdk .ot-optout-signal{margin-top:.625rem}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus,#onetrust-pc-sdk .ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk #ot-addtl-venlst .ot-arw-cntr *,#onetrust-pc-sdk #ot-addtl-venlst .ot-plus-minus *,#onetrust-pc-sdk .ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk #ot-gn-venlst .ot-ven-item .ot-acc-hdr{min-height:40px}#onetrust-pc-sdk .ot-pc-header{height:39px;padding:10px 0 10px 30px;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk #ot-pc-title,#onetrust-pc-sdk #ot-category-title,#onetrust-pc-sdk .ot-cat-header,#onetrust-pc-sdk #ot-lst-title,#onetrust-pc-sdk .ot-ven-hdr .ot-ven-name,#onetrust-pc-sdk .ot-always-active{font-weight:bold;color:dimgray}#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:55%;font-weight:700}#onetrust-pc-sdk .ot-cat-item p{clear:both;float:left;margin-top:10px;margin-bottom:5px;line-height:1.5;font-size:.812em;color:dimgray}#onetrust-pc-sdk .ot-close-icon{height:44px;width:44px;background-size:10px}#onetrust-pc-sdk #ot-pc-title{float:left;font-size:1em;line-height:1.5;margin-bottom:10px;margin-top:10px;width:100%}#onetrust-pc-sdk #accept-recommended-btn-handler{margin-right:10px;margin-bottom:25px;outline-offset:-1px}#onetrust-pc-sdk #ot-pc-desc{clear:both;width:100%;font-size:.812em;line-height:1.5;margin-bottom:25px}#onetrust-pc-sdk #ot-pc-desc *{font-size:inherit;line-height:inherit}#onetrust-pc-sdk #ot-pc-desc ul li{padding:10px 0px}#onetrust-pc-sdk a{color:#656565;cursor:pointer}#onetrust-pc-sdk a:hover{color:#3860be}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk #vdr-lst-dsc{font-size:.812em;line-height:1.5;padding:10px 15px 5px 15px}#onetrust-pc-sdk button{max-width:394px;padding:12px 30px;line-height:1;word-break:break-word;word-wrap:break-word;white-space:normal;font-weight:bold;height:auto}#onetrust-pc-sdk .ot-link-btn{padding:0;margin-bottom:0;border:0;font-weight:normal;line-height:normal;width:auto;height:auto}#onetrust-pc-sdk #ot-pc-content{position:absolute;overflow-y:scroll;padding-left:2px;padding-right:30px;top:60px;bottom:110px;margin:1px 3px 0 30px;width:calc(100% - 63px)}#onetrust-pc-sdk .ot-vs-list .ot-always-active,#onetrust-pc-sdk .ot-cat-grp .ot-always-active{float:right;clear:none;color:#3860be;margin:0;font-size:.813em;line-height:1.3}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-track{margin-right:20px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .ot-pc-scrollbar::-webkit-scrollbar-thumb{border-radius:10px;background:#d8d8d8}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-pc-scrollbar{scrollbar-arrow-color:#d8d8d8;scrollbar-darkshadow-color:#d8d8d8;scrollbar-face-color:#d8d8d8;scrollbar-shadow-color:#d8d8d8}#onetrust-pc-sdk .save-preference-btn-handler{margin-right:20px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-right:10px}#onetrust-pc-sdk #ot-pc-desc .privacy-notice-link{margin-left:0;margin-right:8px}#onetrust-pc-sdk #ot-pc-desc .ot-imprint-handler{margin-left:0;margin-right:8px}#onetrust-pc-sdk .ot-subgrp-cntr{display:inline-block;clear:both;width:100%;padding-top:15px}#onetrust-pc-sdk .ot-switch+.ot-subgrp-cntr{padding-top:10px}#onetrust-pc-sdk ul.ot-subgrps{margin:0;font-size:initial}#onetrust-pc-sdk ul.ot-subgrps li p,#onetrust-pc-sdk ul.ot-subgrps li h5{font-size:.813em;line-height:1.4;color:dimgray}#onetrust-pc-sdk ul.ot-subgrps .ot-switch{min-height:auto}#onetrust-pc-sdk ul.ot-subgrps .ot-switch-nob{top:0}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr{display:inline-block;width:100%}#onetrust-pc-sdk ul.ot-subgrps .ot-acc-txt{margin:0}#onetrust-pc-sdk ul.ot-subgrps li{padding:0;border:none}#onetrust-pc-sdk ul.ot-subgrps li h5{position:relative;top:5px;font-weight:bold;margin-bottom:0;float:left}#onetrust-pc-sdk li.ot-subgrp{margin-left:20px;overflow:auto}#onetrust-pc-sdk li.ot-subgrp>h5{width:calc(100% - 100px)}#onetrust-pc-sdk .ot-cat-item p>ul,#onetrust-pc-sdk li.ot-subgrp p>ul{margin:0px;list-style:disc;margin-left:15px;font-size:inherit}#onetrust-pc-sdk .ot-cat-item p>ul li,#onetrust-pc-sdk li.ot-subgrp p>ul li{font-size:inherit;padding-top:10px;padding-left:0px;padding-right:0px;border:none}#onetrust-pc-sdk .ot-cat-item p>ul li:last-child,#onetrust-pc-sdk li.ot-subgrp p>ul li:last-child{padding-bottom:10px}#onetrust-pc-sdk .ot-pc-logo{height:40px;width:120px}#onetrust-pc-sdk .ot-pc-footer{position:absolute;bottom:0px;width:100%;max-height:160px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-refuse-all-handler{margin-bottom:0px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:160px}#onetrust-pc-sdk.ot-ftr-stacked .ot-pc-footer button{width:100%;max-width:none}#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:0 30px;width:calc(100% - 60px);padding-right:0}#onetrust-pc-sdk .ot-pc-footer-logo{height:30px;width:100%;text-align:right;background:#f4f4f4}#onetrust-pc-sdk .ot-pc-footer-logo a{display:inline-block;margin-top:5px;margin-right:10px}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo{direction:rtl}#onetrust-pc-sdk[dir=rtl] .ot-pc-footer-logo a{margin-right:25px}#onetrust-pc-sdk .ot-tgl{float:right;position:relative;z-index:1}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background-color:#468254;border:1px solid #fff}#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{-webkit-transform:translateX(20px);-ms-transform:translateX(20px);transform:translateX(20px);background-color:#fff;border-color:#fff}#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch{outline:#000 solid 1px}#onetrust-pc-sdk .ot-switch{position:relative;display:inline-block;width:45px;height:25px}#onetrust-pc-sdk .ot-switch-nob{position:absolute;cursor:pointer;top:0;left:0;right:0;bottom:0;background-color:#767676;border:1px solid #ddd;transition:all .2s ease-in 0s;-moz-transition:all .2s ease-in 0s;-o-transition:all .2s ease-in 0s;-webkit-transition:all .2s ease-in 0s;border-radius:20px}#onetrust-pc-sdk .ot-switch-nob:before{position:absolute;content:"";height:18px;width:18px;bottom:3px;left:3px;background-color:#fff;-webkit-transition:.4s;transition:.4s;border-radius:20px}#onetrust-pc-sdk .ot-chkbox input:checked~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]~label::before{background-color:#3860be}#onetrust-pc-sdk .ot-chkbox input[aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox label{position:relative;display:inline-block;padding-left:30px;cursor:pointer;font-weight:500}#onetrust-pc-sdk .ot-chkbox label::before,#onetrust-pc-sdk .ot-chkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-chkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px;top:auto}#onetrust-pc-sdk .ot-chkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-label-txt{display:none}#onetrust-pc-sdk .ot-chkbox input,#onetrust-pc-sdk .ot-tgl input{position:absolute;opacity:0;width:0;height:0}#onetrust-pc-sdk .ot-arw-cntr{float:right;position:relative;pointer-events:none}#onetrust-pc-sdk .ot-arw-cntr .ot-arw{width:16px;height:16px;margin-left:5px;color:dimgray;display:inline-block;vertical-align:middle;-webkit-transition:all 150ms ease-in 0s;-moz-transition:all 150ms ease-in 0s;-o-transition:all 150ms ease-in 0s;transition:all 150ms ease-in 0s}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw-cntr svg{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:#000 1px solid}#onetrust-pc-sdk .ot-tgl-cntr,#onetrust-pc-sdk .ot-arw-cntr{display:inline-block}#onetrust-pc-sdk .ot-tgl-cntr{width:45px;float:right;margin-top:2px}#onetrust-pc-sdk #ot-lst-cnt .ot-tgl-cntr{margin-top:10px}#onetrust-pc-sdk .ot-always-active-subgroup{width:auto;padding-left:0px !important;top:3px;position:relative}#onetrust-pc-sdk .ot-label-status{font-size:.75em;display:none;font-size:.75em;position:relative;top:2px;padding-right:5px;float:left}#onetrust-pc-sdk .ot-arw-cntr{margin-top:-1px}#onetrust-pc-sdk .ot-arw-cntr svg{-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s;transition:all 300ms ease-in 0s;height:10px;width:10px}#onetrust-pc-sdk input:checked~.ot-acc-hdr .ot-arw{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk .ot-arw{width:10px;margin-left:15px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0}#onetrust-pc-sdk .ot-hlst-cntr{margin-top:5px;display:inline-block;width:100%}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{clear:both;color:#3860be;margin-left:0;font-size:.813em;text-decoration:none;float:left;overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover,#onetrust-pc-sdk .category-host-list-handler:hover{text-decoration-line:underline}#onetrust-pc-sdk .category-vendors-list-handler+a{clear:none}#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{display:inline-block;height:13px;width:13px;background-repeat:no-repeat;margin-left:1px;margin-top:6px;cursor:pointer}#onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{margin-bottom:-1px}#onetrust-pc-sdk .back-btn-handler{font-size:1em;text-decoration:none}#onetrust-pc-sdk .back-btn-handler:hover{opacity:.6}#onetrust-pc-sdk #ot-lst-title h3{display:inline-block;word-break:break-word;word-wrap:break-word;margin-bottom:0;color:#656565;font-size:1em;font-weight:bold;margin-left:15px}#onetrust-pc-sdk #ot-lst-title{margin:10px 0 10px 0px;font-size:1em;text-align:left}#onetrust-pc-sdk #ot-pc-hdr{margin:0 0 0 30px;height:auto;width:auto}#onetrust-pc-sdk #ot-pc-hdr input::placeholder{color:#707070;font-style:italic}#onetrust-pc-sdk #vendor-search-handler{height:31px;width:100%;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left;margin-left:15px}#onetrust-pc-sdk .ot-ven-name{display:block;width:auto;padding-right:5px}#onetrust-pc-sdk #ot-lst-cnt{overflow-y:auto;margin-left:20px;margin-right:7px;width:calc(100% - 27px);max-height:calc(100% - 80px);height:100%;transform:translate3d(0, 0, 0)}#onetrust-pc-sdk #ot-pc-lst{width:100%;bottom:100px;position:absolute;top:60px}#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr .ot-arw-cntr *{visibility:hidden}#onetrust-pc-sdk #ot-pc-lst .ot-tgl-cntr{right:12px;position:absolute}#onetrust-pc-sdk #ot-pc-lst .ot-arw-cntr{float:right;position:relative}#onetrust-pc-sdk #ot-pc-lst .ot-arw{margin-left:10px}#onetrust-pc-sdk #ot-pc-lst .ot-acc-hdr{overflow:hidden;cursor:pointer}#onetrust-pc-sdk .ot-vlst-cntr{overflow:hidden}#onetrust-pc-sdk #ot-sel-blk{overflow:hidden;width:100%;position:sticky;position:-webkit-sticky;top:0;z-index:3}#onetrust-pc-sdk #ot-back-arw{height:12px;width:12px}#onetrust-pc-sdk .ot-lst-subhdr{width:100%;display:inline-block}#onetrust-pc-sdk .ot-search-cntr{float:left;width:78%;position:relative}#onetrust-pc-sdk .ot-search-cntr>svg{width:30px;height:30px;position:absolute;float:left;right:-15px}#onetrust-pc-sdk .ot-fltr-cntr{float:right;right:50px;position:relative}#onetrust-pc-sdk #filter-btn-handler{background-color:#3860be;border-radius:17px;display:inline-block;position:relative;width:32px;height:32px;-moz-transition:.1s ease;-o-transition:.1s ease;-webkit-transition:1s ease;transition:.1s ease;padding:0;margin:0}#onetrust-pc-sdk #filter-btn-handler:hover{background-color:#3860be}#onetrust-pc-sdk #filter-btn-handler svg{width:12px;height:12px;margin:3px 10px 0 10px;display:block;position:static;right:auto;top:auto}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{color:#3860be;text-decoration:none;font-weight:100;display:inline-block;padding-top:10px;transform:translate(0, 1%);-o-transform:translate(0, 1%);-ms-transform:translate(0, 1%);-webkit-transform:translate(0, 1%);position:relative;z-index:2}#onetrust-pc-sdk .ot-ven-link *,#onetrust-pc-sdk .ot-ven-legclaim-link *{font-size:inherit}#onetrust-pc-sdk .ot-ven-link:hover,#onetrust-pc-sdk .ot-ven-legclaim-link:hover{text-decoration:underline}#onetrust-pc-sdk .ot-ven-hdr{width:calc(100% - 160px);height:auto;float:left;word-break:break-word;word-wrap:break-word;vertical-align:middle;padding-bottom:3px}#onetrust-pc-sdk .ot-ven-link,#onetrust-pc-sdk .ot-ven-legclaim-link{letter-spacing:.03em;font-size:.75em;font-weight:400}#onetrust-pc-sdk .ot-ven-dets{border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-ven-dets li:first-child p:first-child{border-top:none}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:not(:first-child){border-top:1px solid #ddd !important}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(odd){width:30%}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc:nth-child(n+3) p:nth-of-type(even){width:50%;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p,#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{padding-top:5px;padding-bottom:5px;display:block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc h5{display:inline-block}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-last-child(-n+1){padding-bottom:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc p:nth-child(-n+2):not(.disc-pur){padding-top:10px}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur-cont{display:inline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur{position:relative;width:50% !important;word-break:break-word;word-wrap:break-word;left:calc(30% + 17px)}#onetrust-pc-sdk .ot-ven-dets .ot-ven-disc .disc-pur:nth-child(-n+1){position:static}#onetrust-pc-sdk .ot-ven-dets p,#onetrust-pc-sdk .ot-ven-dets h5,#onetrust-pc-sdk .ot-ven-dets span{font-size:.69em;text-align:left;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-bottom:10px;padding-left:15px;color:#2e3644}#onetrust-pc-sdk .ot-ven-dets h5{padding-top:5px}#onetrust-pc-sdk .ot-ven-dets span{color:dimgray;padding:0;vertical-align:baseline}#onetrust-pc-sdk .ot-ven-dets .ot-ven-pur h5{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;padding-bottom:5px;margin-bottom:5px;font-weight:bold}#onetrust-pc-sdk #ot-host-lst .ot-sel-all{float:right;position:relative;margin-right:42px;top:10px}#onetrust-pc-sdk #ot-host-lst .ot-sel-all input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #ot-host-lst .ot-sel-all label{height:20px;width:20px;padding-left:0px}#onetrust-pc-sdk #ot-host-lst .ot-acc-txt{overflow:hidden;width:95%}#onetrust-pc-sdk .ot-host-hdr{position:relative;z-index:1;pointer-events:none;width:calc(100% - 125px);float:left}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-desc{display:inline-block;width:90%}#onetrust-pc-sdk .ot-host-name{pointer-events:none}#onetrust-pc-sdk .ot-host-hdr>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;margin-bottom:5px;pointer-events:initial}#onetrust-pc-sdk .ot-host-name+a{margin-top:5px}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a,#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{color:dimgray;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-name,#onetrust-pc-sdk .ot-host-name a{font-weight:bold;font-size:.82em;line-height:1.3}#onetrust-pc-sdk .ot-host-name a{font-size:1em}#onetrust-pc-sdk .ot-host-expand{margin-top:3px;margin-bottom:3px;clear:both;display:block;color:#3860be;font-size:.72em;font-weight:normal}#onetrust-pc-sdk .ot-host-expand *{font-size:inherit}#onetrust-pc-sdk .ot-host-desc,#onetrust-pc-sdk .ot-host-info{font-size:.688em;line-height:1.4;font-weight:normal}#onetrust-pc-sdk .ot-host-desc{margin-top:10px}#onetrust-pc-sdk .ot-host-opt{margin:0;font-size:inherit;display:inline-block;width:100%}#onetrust-pc-sdk .ot-host-opt li>div div{font-size:.8em;padding:5px 0}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk .ot-host-opt li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-host-info{border:none;display:inline-block;width:calc(100% - 10px);padding:10px;margin-bottom:10px;background-color:#f8f8f8}#onetrust-pc-sdk .ot-host-info>div{overflow:auto}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk #ot-fltr-modal{width:100%;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-fltr-modal .ot-label-txt{display:inline-block;font-size:.85em;color:dimgray}#onetrust-pc-sdk #ot-fltr-cnt{z-index:2147483646;background-color:#fff;position:absolute;height:90%;max-height:300px;width:325px;left:210px;margin-top:10px;margin-bottom:20px;padding-right:10px;border-radius:3px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-scrlcnt{overflow-y:auto;overflow-x:hidden;clear:both;max-height:calc(100% - 60px)}#onetrust-pc-sdk #ot-anchor{border:12px solid rgba(0,0,0,0);display:none;position:absolute;z-index:2147483647;right:55px;top:75px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk .ot-fltr-btns{margin-left:15px}#onetrust-pc-sdk #filter-apply-handler{margin-right:15px}#onetrust-pc-sdk .ot-fltr-opt{margin-bottom:5px;margin-left:15px;min-height:20px;width:75%;position:relative}#onetrust-pc-sdk .ot-fltr-opt p{display:inline-block;margin:0;font-size:.9em;color:#2e3644}#onetrust-pc-sdk .ot-chkbox label span{font-size:.85em;color:dimgray}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox][aria-checked=mixed]+label::after{content:""}#onetrust-pc-sdk .ot-chkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #ot-selall-vencntr,#onetrust-pc-sdk #ot-selall-adtlvencntr,#onetrust-pc-sdk #ot-selall-hostcntr,#onetrust-pc-sdk #ot-selall-licntr,#onetrust-pc-sdk #ot-selall-gnvencntr{right:15px;position:relative;width:20px;height:20px;float:right}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label,#onetrust-pc-sdk #ot-selall-gnvencntr label{float:left;padding-left:0}#onetrust-pc-sdk #ot-ven-lst:first-child{border-top:1px solid #e2e2e2}#onetrust-pc-sdk ul{list-style:none;padding:0}#onetrust-pc-sdk ul li{position:relative;margin:0;padding:15px 15px 15px 10px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk ul li h3,#onetrust-pc-sdk ul li h4{font-size:.75em;color:#656565;margin:0;display:inline-block;width:70%;height:auto;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk ul li p{margin:0;font-size:.7em}#onetrust-pc-sdk ul li input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk .ot-cat-item>button:focus,#onetrust-pc-sdk .ot-acc-cntr>button:focus,#onetrust-pc-sdk li>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-cat-item>button,#onetrust-pc-sdk .ot-acc-cntr>button,#onetrust-pc-sdk li>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=false]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-txt,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-cat-item>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk .ot-acc-cntr>button[aria-expanded=true]~.ot-acc-grpcntr,#onetrust-pc-sdk li>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-host-item>button:focus,#onetrust-pc-sdk .ot-ven-item>button:focus{outline:0;border:2px solid #000}#onetrust-pc-sdk .ot-hide-acc>button{pointer-events:none}#onetrust-pc-sdk .ot-hide-acc .ot-plus-minus>*,#onetrust-pc-sdk .ot-hide-acc .ot-arw-cntr>*{visibility:hidden}#onetrust-pc-sdk .ot-hide-acc .ot-acc-hdr{min-height:30px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt){padding-right:10px;width:calc(100% - 37px);margin-top:10px;max-height:calc(100% - 90px)}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk{background-color:#f9f9fc;border:1px solid #e2e2e2;width:calc(100% - 2px);padding-bottom:5px;padding-top:5px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt{border:unset;background-color:unset}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all-hdr{display:none}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all{padding-right:.5rem}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) #ot-sel-blk.ot-vnd-list-cnt .ot-sel-all .ot-chkbox{right:0}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all{padding-right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-sel-all-chkbox{width:auto}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) ul li{border:1px solid #e2e2e2;margin-bottom:10px}#onetrust-pc-sdk.ot-addtl-vendors #ot-lst-cnt:not(.ot-host-cnt) .ot-acc-cntr>.ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk.ot-addtl-vendors .ot-sel-all-chkbox{float:right}#onetrust-pc-sdk.ot-addtl-vendors .ot-plus-minus~.ot-sel-all-chkbox{right:34px}#onetrust-pc-sdk.ot-addtl-vendors #ot-ven-lst:first-child{border-top:none}#onetrust-pc-sdk .ot-acc-cntr{position:relative;border-left:1px solid #e2e2e2;border-right:1px solid #e2e2e2;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr input{z-index:1}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr{background-color:#f9f9fc;padding:5px 0 5px 15px;width:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-plus-minus{vertical-align:middle;top:auto}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr .ot-arw-cntr{right:10px}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-hdr input{z-index:2}#onetrust-pc-sdk .ot-acc-cntr.ot-add-tech .ot-acc-hdr{padding:10px 0 10px 15px}#onetrust-pc-sdk .ot-acc-cntr>input[type=checkbox]:checked~.ot-acc-hdr{border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-acc-cntr>.ot-acc-txt{padding-left:10px;padding-right:10px}#onetrust-pc-sdk .ot-acc-cntr button[aria-expanded=true]~.ot-acc-txt{width:auto}#onetrust-pc-sdk .ot-acc-cntr .ot-addtl-venbox{display:none}#onetrust-pc-sdk .ot-vlst-cntr{margin-bottom:0;width:100%}#onetrust-pc-sdk .ot-vensec-title{font-size:.813em;vertical-align:middle;display:inline-block}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{margin-left:0;margin-top:10px}#onetrust-pc-sdk #ot-selall-vencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-adtlvencntr.line-through label::after,#onetrust-pc-sdk #ot-selall-licntr.line-through label::after,#onetrust-pc-sdk #ot-selall-hostcntr.line-through label::after,#onetrust-pc-sdk #ot-selall-gnvencntr.line-through label::after{height:auto;border-left:0;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none;left:5px;top:9px}#onetrust-pc-sdk #ot-category-title{float:left;padding-bottom:10px;font-size:1em;width:100%}#onetrust-pc-sdk .ot-cat-grp{margin-top:10px}#onetrust-pc-sdk .ot-cat-item{line-height:1.1;margin-top:10px;display:inline-block;width:100%}#onetrust-pc-sdk .ot-btn-container{text-align:right}#onetrust-pc-sdk .ot-btn-container button{display:inline-block;font-size:.75em;letter-spacing:.08em;margin-top:19px}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{position:absolute;top:10px;right:0;z-index:1;padding:0;background-color:rgba(0,0,0,0);border:none}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon svg{display:block;height:10px;width:10px}#onetrust-pc-sdk #clear-filters-handler{margin-top:20px;margin-bottom:10px;float:right;max-width:200px;text-decoration:none;color:#3860be;font-size:.9em;font-weight:bold;background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,0);padding:1px}#onetrust-pc-sdk #clear-filters-handler:hover{color:#2285f7}#onetrust-pc-sdk #clear-filters-handler:focus{outline:#000 solid 1px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl,#onetrust-pc-sdk .ot-enbl-chr h4~.ot-always-active{right:45px}#onetrust-pc-sdk .ot-enbl-chr h4~.ot-tgl+.ot-tgl{right:120px}#onetrust-pc-sdk .ot-enbl-chr .ot-pli-hdr.ot-leg-border-color span:first-child{width:90px}#onetrust-pc-sdk .ot-enbl-chr li.ot-subgrp>h5+.ot-tgl-cntr{padding-right:25px}#onetrust-pc-sdk .ot-plus-minus{width:20px;height:20px;font-size:1.5em;position:relative;display:inline-block;margin-right:5px;top:3px}#onetrust-pc-sdk .ot-plus-minus span{position:absolute;background:#27455c;border-radius:1px}#onetrust-pc-sdk .ot-plus-minus span:first-of-type{top:25%;bottom:25%;width:10%;left:45%}#onetrust-pc-sdk .ot-plus-minus span:last-of-type{left:25%;right:25%;height:10%;top:45%}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-arw,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:first-of-type,#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{transform:rotate(90deg)}#onetrust-pc-sdk button[aria-expanded=true]~.ot-acc-hdr .ot-plus-minus span:last-of-type{left:50%;right:50%}#onetrust-pc-sdk #ot-selall-vencntr label,#onetrust-pc-sdk #ot-selall-adtlvencntr label,#onetrust-pc-sdk #ot-selall-hostcntr label,#onetrust-pc-sdk #ot-selall-licntr label{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk .ot-host-item .ot-plus-minus,#onetrust-pc-sdk .ot-ven-item .ot-plus-minus{float:left;margin-right:8px;top:10px}#onetrust-pc-sdk .ot-ven-item ul{list-style:none inside;font-size:100%;margin:0}#onetrust-pc-sdk .ot-ven-item ul li{margin:0 !important;padding:0;border:none !important}#onetrust-pc-sdk .ot-pli-hdr{color:#77808e;overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;width:calc(100% - 2px);border-top-left-radius:3px;border-top-right-radius:3px}#onetrust-pc-sdk .ot-pli-hdr span:first-child{top:50%;transform:translateY(50%);max-width:90px}#onetrust-pc-sdk .ot-pli-hdr span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk .ot-li-title{float:right;font-size:.813em}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color{background-color:#f4f4f4;border:1px solid #d8d8d8}#onetrust-pc-sdk .ot-pli-hdr.ot-leg-border-color span:first-child{text-align:left;width:70px}#onetrust-pc-sdk li.ot-subgrp>h5,#onetrust-pc-sdk .ot-cat-header{width:calc(100% - 130px)}#onetrust-pc-sdk li.ot-subgrp>h5+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-acc-grpdesc{margin-bottom:5px}#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-acc-grpcntr .ot-vlst-cntr+.ot-subgrp-cntr{border-top:none}#onetrust-pc-sdk .ot-acc-hdr .ot-arw-cntr+.ot-tgl-cntr,#onetrust-pc-sdk .ot-acc-txt h4+.ot-tgl-cntr{padding-left:13px}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-subgrp>h5,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header{width:calc(100% - 145px)}#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item h5+.ot-tgl-cntr,#onetrust-pc-sdk .ot-pli-hdr~.ot-cat-item .ot-cat-header+.ot-tgl{padding-left:28px}#onetrust-pc-sdk .ot-sel-all-hdr,#onetrust-pc-sdk .ot-sel-all-chkbox{display:inline-block;width:100%;position:relative}#onetrust-pc-sdk .ot-sel-all-chkbox{z-index:1}#onetrust-pc-sdk .ot-sel-all{margin:0;position:relative;padding-right:23px;float:right}#onetrust-pc-sdk .ot-consent-hdr,#onetrust-pc-sdk .ot-li-hdr{float:right;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-li-hdr{max-width:100px;padding-right:10px}#onetrust-pc-sdk .ot-consent-hdr{max-width:55px}#onetrust-pc-sdk #ot-selall-licntr{display:block;width:21px;height:auto;float:right;position:relative;right:80px}#onetrust-pc-sdk #ot-selall-licntr label{position:absolute}#onetrust-pc-sdk .ot-ven-ctgl{margin-left:66px}#onetrust-pc-sdk .ot-ven-litgl+.ot-arw-cntr{margin-left:81px}#onetrust-pc-sdk .ot-enbl-chr .ot-host-cnt .ot-tgl-cntr{width:auto}#onetrust-pc-sdk #ot-lst-cnt:not(.ot-host-cnt) .ot-tgl-cntr{width:auto;top:auto;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox{position:relative;display:inline-block;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-chkbox label{position:absolute;padding:0;width:20px;height:20px}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-acc-grpdesc+.ot-leg-btn-container{padding-left:20px;padding-right:20px;width:calc(100% - 40px);margin-bottom:5px}#onetrust-pc-sdk .ot-subgrp .ot-leg-btn-container{margin-bottom:5px}#onetrust-pc-sdk #ot-ven-lst .ot-leg-btn-container{margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:auto;padding:6.5px 8px;margin-bottom:0;letter-spacing:0;font-size:.75em;line-height:normal}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{text-decoration:underline;padding:0;font-size:.75em;font-weight:600;line-height:1;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:inherit;line-height:1.5}#onetrust-pc-sdk.ot-close-btn-link #close-pc-btn-handler{border:none;height:auto;line-height:1.5;text-decoration:underline;font-size:.69em;background:none;right:15px;top:15px;width:auto;font-weight:normal}#onetrust-pc-sdk .ot-pgph-link{font-size:.813em !important;margin-top:5px;position:relative}#onetrust-pc-sdk .ot-pgph-link.ot-pgph-link-subgroup{margin-bottom:1rem}#onetrust-pc-sdk .ot-pgph-contr{margin:0 2.5rem}#onetrust-pc-sdk .ot-pgph-title{font-size:1.18rem;margin-bottom:2rem}#onetrust-pc-sdk .ot-pgph-desc{font-size:1rem;font-weight:400;margin-bottom:2rem;line-height:1.5rem}#onetrust-pc-sdk .ot-pgph-desc:not(:last-child):after{content:"";width:96%;display:block;margin:0 auto;padding-bottom:2rem;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk .ot-cat-header{float:left;font-weight:600;font-size:.875em;line-height:1.5;max-width:90%;vertical-align:middle}#onetrust-pc-sdk .ot-vnd-item>button:focus{outline:#000 solid 2px}#onetrust-pc-sdk .ot-vnd-item>button{position:absolute;cursor:pointer;width:100%;height:100%;margin:0;top:0;left:0;z-index:1;max-width:none;border:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=false]~.ot-acc-txt{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-txt{transition:.1s ease-in;margin-top:10px;width:100%;overflow:auto;display:block}#onetrust-pc-sdk .ot-vnd-item>button[aria-expanded=true]~.ot-acc-grpcntr{width:auto;margin-top:0px;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{position:relative;border-radius:2px;margin:0;padding:0;border:1px solid #d8d8d8;border-top:none;width:calc(100% - 2px);float:left}#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{margin-top:10px;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc{padding-left:20px;padding-right:20px;width:calc(100% - 40px);font-size:.812em;margin-bottom:10px;margin-top:15px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul{padding-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpdesc>ul li{padding-top:0;line-height:1.5;padding-bottom:10px}#onetrust-pc-sdk .ot-accordion-layout div+.ot-acc-grpdesc{margin-top:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:first-child{margin-top:10px}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr:last-child,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr:last-child{margin-bottom:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding-top:11.5px;padding-bottom:11.5px;padding-left:20px;padding-right:20px;width:calc(100% - 40px);display:inline-block}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-txt{width:100%;padding:0}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp-cntr{padding-left:20px;padding-right:15px;padding-bottom:0;width:calc(100% - 35px)}#onetrust-pc-sdk .ot-accordion-layout .ot-subgrp{padding-right:5px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-grpcntr{z-index:1;position:relative}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr{position:absolute;top:50%;transform:translateY(-50%);right:20px;margin-top:-2px}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header+.ot-arw-cntr .ot-arw{width:15px;height:20px;margin-left:5px;color:dimgray}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{float:none;color:#2e3644;margin:0;display:inline-block;height:auto;word-wrap:break-word;min-height:inherit}#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr,#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding-left:20px;width:calc(100% - 20px);display:inline-block;margin-top:0;padding-bottom:2px}#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{position:relative;min-height:25px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl,#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{position:absolute;top:50%;transform:translateY(-50%);right:20px}#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl+.ot-tgl{right:95px}#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler,#onetrust-pc-sdk .ot-accordion-layout .category-vendors-list-handler+a{margin-top:5px}#onetrust-pc-sdk #ot-lst-cnt{margin-top:1rem;max-height:calc(100% - 96px)}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 2rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list,#onetrust-pc-sdk .ot-vnd-serv{width:auto;padding:1rem 1.25rem;padding-bottom:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:600;font-size:.95em;line-height:2;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item{border:none;margin:0;padding:0}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button{outline:none;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item button[aria-expanded=true],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item button[aria-expanded=true]{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:first-child{margin-top:.25rem;border-top:unset}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child{margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item:last-child button,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item:last-child button{border-bottom:none}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info-cntr{border:1px solid #d8d8d8;padding:.75rem 1.75rem;padding-bottom:0;width:auto;margin-top:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info{margin-bottom:1rem;padding-left:.75rem;padding-right:.75rem;display:flex;flex-direction:column}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOEmail]{border-top:1px solid #d8d8d8;padding-top:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink],#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info[data-vnd-info-key*=DPOLink]{border-bottom:1px solid #d8d8d8;padding-bottom:1rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-lbl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-lbl{font-weight:bold;font-size:.85em;margin-bottom:.5rem}#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-vnd-info .ot-vnd-cnt,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info .ot-vnd-cnt{margin-left:.5rem;font-weight:500;font-size:.85rem}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt{padding-left:40px}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-size:.8em}#onetrust-pc-sdk .ot-vs-list.ot-vnd-subgrp-cnt .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv.ot-vnd-subgrp-cnt .ot-cat-header{font-size:.8em}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv{margin-bottom:1rem;padding:1rem .95rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr{padding-bottom:.75rem;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .ot-subgrp-cntr .ot-vnd-serv .ot-vnd-serv-hdr-cntr .ot-vnd-serv-hdr{font-weight:700;font-size:.8em;line-height:20px;margin-left:.82rem}#onetrust-pc-sdk .ot-subgrp-cntr .ot-cat-header{font-weight:700;font-size:.8em;line-height:20px}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-vnd-serv .ot-vnd-lst-cont .ot-accordion-layout .ot-acc-hdr div.ot-chkbox{margin-left:.82rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr{padding:.7rem 0;margin:0;display:flex;width:100%;align-items:center;justify-content:space-between}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:first-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:first-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:first-child{margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr div:last-child,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr div:last-child,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr div:last-child{margin-right:.5rem;margin-left:.5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-always-active,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-always-active{position:relative;right:unset;top:unset;transform:unset}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-plus-minus,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-plus-minus{top:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-arw-cntr,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-arw-cntr{float:none;top:unset;right:unset;transform:unset;margin-top:-2px;position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-cat-header,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-cat-header{flex:1;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-tgl,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-tgl{position:relative;transform:none;right:0;top:0;float:none}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox{position:relative;margin:0 .5rem}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label{padding:0}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox label::before,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox label::before{position:relative}#onetrust-pc-sdk .ot-vs-config .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk #ot-pc-lst .ot-vs-list .ot-vnd-item .ot-acc-hdr .ot-chkbox input,#onetrust-pc-sdk .ot-accordion-layout.ot-checkbox-consent .ot-acc-hdr .ot-chkbox input{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0;z-index:1}#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h5.ot-cat-header,#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp .ot-acc-hdr h4.ot-cat-header{margin:0}#onetrust-pc-sdk .ot-vs-config .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp h5{top:0;line-height:20px}#onetrust-pc-sdk .ot-vs-list{display:flex;flex-direction:column;padding:0;margin:.5rem 4px}#onetrust-pc-sdk .ot-vs-selc-all{display:flex;padding:0;float:unset;align-items:center;justify-content:flex-start}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf{justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf.ot-caret-conf .ot-sel-all-chkbox{margin-right:48px}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf .ot-sel-all-chkbox{margin:0;padding:0;margin-right:14px;justify-content:flex-end}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-chkbox,#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr.ot-tgl{display:inline-block;right:unset;width:auto;height:auto;float:none}#onetrust-pc-sdk .ot-vs-selc-all.ot-toggle-conf #ot-selall-vencntr label{width:45px;height:25px}#onetrust-pc-sdk .ot-vs-selc-all .ot-sel-all-chkbox{margin-right:11px;margin-left:.75rem;display:flex;align-items:center}#onetrust-pc-sdk .ot-vs-selc-all .sel-all-hdr{margin:0 1.25rem;font-size:.812em;line-height:normal;text-align:center;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .ot-vnd-list-cnt #ot-selall-vencntr.ot-chkbox{float:unset;right:0}#onetrust-pc-sdk[dir=rtl] #ot-back-arw,#onetrust-pc-sdk[dir=rtl] input~.ot-acc-hdr .ot-arw{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk[dir=rtl] input:checked~.ot-acc-hdr .ot-arw{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk[dir=rtl] .ot-chkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk[dir=rtl] .ot-search-cntr>svg{right:0}@media only screen and (max-width: 600px){#onetrust-pc-sdk.otPcCenter{left:0;min-width:100%;height:100%;top:0;border-radius:0}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk.ot-ftr-stacked .ot-btn-container{margin:1px 3px 0 10px;padding-right:10px;width:calc(100% - 23px)}#onetrust-pc-sdk .ot-btn-container button{max-width:none;letter-spacing:.01em}#onetrust-pc-sdk #close-pc-btn-handler{top:10px;right:17px}#onetrust-pc-sdk p{font-size:.7em}#onetrust-pc-sdk #ot-pc-hdr{margin:10px 10px 0 5px;width:calc(100% - 15px)}#onetrust-pc-sdk .vendor-search-handler{font-size:1em}#onetrust-pc-sdk #ot-back-arw{margin-left:12px}#onetrust-pc-sdk #ot-lst-cnt{margin:0;padding:0 5px 0 10px;min-width:95%}#onetrust-pc-sdk .switch+p{max-width:80%}#onetrust-pc-sdk .ot-ftr-stacked button{width:100%}#onetrust-pc-sdk #ot-fltr-cnt{max-width:320px;width:90%;border-top-right-radius:0;border-bottom-right-radius:0;margin:0;margin-left:15px;left:auto;right:40px;top:85px}#onetrust-pc-sdk .ot-fltr-opt{margin-left:25px;margin-bottom:10px}#onetrust-pc-sdk .ot-pc-refuse-all-handler{margin-bottom:0}#onetrust-pc-sdk #ot-fltr-cnt{right:40px}}@media only screen and (max-width: 476px){#onetrust-pc-sdk .ot-fltr-cntr,#onetrust-pc-sdk #ot-fltr-cnt{right:10px}#onetrust-pc-sdk #ot-anchor{right:25px}#onetrust-pc-sdk button{width:100%}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-sel-all{padding-right:9px}#onetrust-pc-sdk:not(.ot-addtl-vendors) #ot-pc-lst:not(.ot-enbl-chr) .ot-tgl-cntr{right:0}}@media(min-width: 768px){#onetrust-pc-sdk.ot-tgl-with-label .ot-label-status{display:inline}#onetrust-pc-sdk.ot-tgl-with-label #ot-pc-lst .ot-label-status{display:none}}@media only screen and (max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcCenter{left:0;top:0;min-width:100%;height:100%;border-radius:0}#onetrust-pc-sdk .ot-pc-header{height:auto;min-height:20px}#onetrust-pc-sdk .ot-pc-header .ot-pc-logo{max-height:30px}#onetrust-pc-sdk .ot-pc-footer{max-height:60px;overflow-y:auto}#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk #ot-pc-lst{bottom:70px}#onetrust-pc-sdk.ot-ftr-stacked #ot-pc-content{bottom:70px}#onetrust-pc-sdk #ot-anchor{left:initial;right:50px}#onetrust-pc-sdk #ot-lst-title{margin-top:12px}#onetrust-pc-sdk #ot-lst-title *{font-size:inherit}#onetrust-pc-sdk #ot-pc-hdr input{margin-right:0;padding-right:45px}#onetrust-pc-sdk .switch+p{max-width:85%}#onetrust-pc-sdk #ot-sel-blk{position:static}#onetrust-pc-sdk #ot-pc-lst{overflow:auto}#onetrust-pc-sdk #ot-lst-cnt{max-height:none;overflow:initial}#onetrust-pc-sdk #ot-lst-cnt.no-results{height:auto}#onetrust-pc-sdk input{font-size:1em !important}#onetrust-pc-sdk p{font-size:.6em}#onetrust-pc-sdk #ot-fltr-modal{width:100%;top:0}#onetrust-pc-sdk ul li p,#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a,#onetrust-pc-sdk .category-host-list-handler{font-size:.6em}#onetrust-pc-sdk.ot-shw-fltr #ot-anchor{display:none !important}#onetrust-pc-sdk.ot-shw-fltr #ot-pc-lst{height:100% !important;overflow:hidden;top:0px}#onetrust-pc-sdk.ot-shw-fltr #ot-fltr-cnt{margin:0;height:100%;max-height:none;padding:10px;top:0;width:calc(100% - 20px);position:absolute;right:0;left:0;max-width:none}#onetrust-pc-sdk.ot-shw-fltr .ot-fltr-scrlcnt{max-height:calc(100% - 65px)}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #ot-search-cntr,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk ot-grp-hdr1 .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title:after
                ,#onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-cnt,
                        #onetrust-consent-sdk #onetrust-pc-sdk #ot-anchor {
                    background-color: #FFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-ven-lst .ot-ven-opts p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-li-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-fltr-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-sel-blk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-lst-title h3,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst .ot-ven-name,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-lst #ot-ven-lst .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-optout-signal
                {
                    color: #2E2E2E;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-pgph-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-ven-legclaim-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-name a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-acc-hdr .ot-host-expand,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-content #ot-pc-desc .ot-link-btn,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-vnd-serv .ot-vnd-item .ot-vnd-info a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #ot-lst-cnt .ot-vnd-info a
                    {
                        color: #007398;
                    }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { text-decoration: underline;}
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-subgrp-tgl .ot-switch.ot-toggle
             {
                background-color: #F8F8F8;
            }
             #onetrust-consent-sdk #onetrust-pc-sdk #ot-host-lst .ot-host-info,
                    #onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-txt .ot-ven-dets
                            {
                                background-color: #F8F8F8;
                            }
        #onetrust-consent-sdk #onetrust-pc-sdk
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #007398;border-color: #007398;
                color: #FFF;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-active-menu {
                border-color: #007398;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border: 1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #4D4D4D; border-color: #4D4D4D;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-tgl input:focus + .ot-switch, .ot-switch .ot-switch-nob, .ot-switch .ot-switch-nob:before,
            #onetrust-pc-sdk .ot-checkbox input[type="checkbox"]:focus + label::before,
            #onetrust-pc-sdk .ot-chkbox input[type="checkbox"]:focus + label::before {
                outline-color: #000000;
                outline-width: 1px;
                outline-offset: 1px;
            }
            #onetrust-pc-sdk .ot-host-item > button:focus, #onetrust-pc-sdk .ot-ven-item > button:focus {
                border: 1px solid #000000;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk *:focus,
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-vlst-cntr > a:focus {
               outline: 1px solid #000000;
               outline-offset: 1px;
            }#onetrust-pc-sdk .ot-vlst-cntr .ot-ext-lnk,  #onetrust-pc-sdk .ot-ven-hdr .ot-ext-lnk{
                    background-image: url('https://cdn.cookielaw.org/logos/static/ot_external_link.svg');
                }
            /*! v6.10.0 2021-01-18 */
div#onetrust-consent-sdk #onetrust-banner-sdk{border-top:2px solid #d0d3d4!important;box-shadow:none;padding:24px}div#onetrust-consent-sdk button{border-radius:0!important;box-shadow:none!important;box-sizing:border-box!important;font-size:20px!important;font-weight:400!important;letter-spacing:0!important;max-width:none!important;white-space:nowrap!important}div#onetrust-consent-sdk button:not(.ot-link-btn){background-color:#007dbc!important;border:2px solid #007dbc!important;color:#fff!important;height:48px!important;padding:0 1em!important;width:auto!important}div#onetrust-consent-sdk button:hover{background-color:#fff!important;border-color:#949494!important;color:#2e2e2e!important}div#onetrust-consent-sdk button.ot-link-btn{color:#007dbc!important;font-size:16px!important;text-decoration:underline}div#onetrust-consent-sdk button.ot-link-btn:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-consent-sdk a,div#onetrust-pc-sdk a{color:#007dbc!important;text-decoration:underline!important}div#onetrust-consent-sdk a,div#onetrust-consent-sdk button,div#onetrust-consent-sdk p:hover{opacity:1!important}div#onetrust-consent-sdk a:focus,div#onetrust-consent-sdk button:focus,div#onetrust-consent-sdk input:focus{outline:2px solid #949494!important;outline-offset:1px!important}div#onetrust-banner-sdk .ot-sdk-container{padding:0;width:auto}div#onetrust-banner-sdk .ot-sdk-row{align-items:flex-start;box-sizing:border-box;display:flex;flex-direction:column;justify-content:space-between;margin:auto;max-width:1152px}div#onetrust-banner-sdk .ot-sdk-row:after{display:none}div#onetrust-banner-sdk #onetrust-group-container,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-group-container,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{flex-grow:1;width:auto}div#onetrust-banner-sdk #onetrust-policy,div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy{margin:0;overflow:visible}div#onetrust-banner-sdk.ot-bnr-flift #onetrust-policy-text,div#onetrust-consent-sdk #onetrust-policy-text{font-size:16px;line-height:24px;max-width:44em;margin:0}div#onetrust-consent-sdk #onetrust-policy-text a[href]{font-weight:400;margin-left:8px}div#onetrust-banner-sdk #onetrust-button-group-parent{flex:0 0 auto;margin:32px 0 0;width:100%}div#onetrust-banner-sdk #onetrust-button-group{display:flex;flex-direction:row;flex-wrap:wrap;justify-content:flex-end;margin:-8px}div#onetrust-banner-sdk .banner-actions-container{display:flex;flex:1 0 auto}div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type,div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto;margin:8px;width:auto}div#onetrust-consent-sdk #onetrust-pc-btn-handler{background-color:#fff!important;color:inherit!important}div#onetrust-banner-sdk #onetrust-close-btn-container{display:none}@media only screen and (min-width:556px){div#onetrust-consent-sdk #onetrust-banner-sdk{padding:40px}div#onetrust-banner-sdk #onetrust-policy{margin:0 40px 0 0}div#onetrust-banner-sdk .ot-sdk-row{align-items:center;flex-direction:row}div#onetrust-banner-sdk #onetrust-button-group-parent,div#onetrust-banner-sdk.ot-bnr-flift:not(.ot-iab-2) #onetrust-button-group-parent,div#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:0;padding:0;width:auto}div#onetrust-banner-sdk #onetrust-button-group,div#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{align-items:stretch;flex-direction:column-reverse;margin:0}div#onetrust-consent-sdk #onetrust-accept-btn-handler,div#onetrust-consent-sdk #onetrust-pc-btn-handler{flex:1 0 auto}}@media only screen and (min-width:768px){div#onetrust-banner-sdk #onetrust-policy{margin:0 48px 0 0}div#onetrust-consent-sdk #onetrust-banner-sdk{padding:48px}}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-size:16px;line-height:24px}div#onetrust-consent-sdk #onetrust-pc-sdk p,div#onetrust-pc-sdk #ot-pc-desc,div#onetrust-pc-sdk .category-host-list-handler,div#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{font-size:16px;font-weight:400;line-height:24px}div#onetrust-consent-sdk a:hover,div#onetrust-pc-sdk a:hover{color:inherit!important;text-decoration-color:#949494!important}div#onetrust-pc-sdk{border-radius:0;bottom:0;height:auto;left:0;margin:auto;max-width:100%;overflow:hidden;right:0;top:0;width:512px;max-height:800px}div#onetrust-pc-sdk .ot-pc-header{display:none}div#onetrust-pc-sdk #ot-pc-content{overscroll-behavior:contain;padding:0 12px 0 24px;margin:16px 4px 0 0;top:0;right:16px;left:0;width:auto}div#onetrust-pc-sdk #ot-category-title,div#onetrust-pc-sdk #ot-pc-title{font-size:24px;font-weight:400;line-height:32px;margin:0 0 16px}div#onetrust-pc-sdk #ot-pc-desc{padding:0}div#onetrust-pc-sdk #ot-pc-desc a{display:inline}div#onetrust-pc-sdk #accept-recommended-btn-handler{display:none!important}div#onetrust-pc-sdk input[type=checkbox]:focus+.ot-acc-hdr{outline:2px solid #007dbc;outline-offset:-1px;transition:none}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item{border-width:0 0 2px}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item:first-of-type{border-width:2px 0}div#onetrust-pc-sdk .ot-accordion-layout .ot-acc-hdr{padding:8px 0;width:100%}div#onetrust-pc-sdk .ot-plus-minus{transform:translateY(2px)}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item>button{background:0 0!important;border:0!important;height:44px!important;max-width:none!important;width:calc(100% - 48px)!important}div#onetrust-consent-sdk #onetrust-pc-sdk h5{font-weight:700}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr{padding:0}div#onetrust-pc-sdk .ot-accordion-layout.ot-cat-item .ot-acc-grpdesc{padding:0;width:100%}div#onetrust-pc-sdk .ot-acc-grpcntr .ot-subgrp-cntr{border:0;padding:0}div#onetrust-pc-sdk .ot-subgrp-cntr ul.ot-subgrps li.ot-subgrp{margin:0}div#onetrust-pc-sdk .ot-always-active-group .ot-cat-header{width:calc(100% - 160px)}#onetrust-pc-sdk .ot-accordion-layout .ot-cat-header{width:calc(100% - 88px)}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active{color:inherit;font-size:12px;font-weight:400;line-height:1.5;padding-right:48px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:before{border-radius:12px;position:absolute;right:0;top:0;content:'';background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active:after{border-radius:50%;position:absolute;right:5px;top:4px;content:'';background-color:#007dbc;height:12px;width:12px}div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-always-active,div#onetrust-pc-sdk .ot-accordion-layout h4~.ot-tgl{right:2px}div#onetrust-pc-sdk .ot-switch{display:block;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob,div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob{background:#fff;border:2px solid #939393;box-sizing:border-box;height:20px;width:40px}div#onetrust-pc-sdk .ot-tgl input+.ot-switch .ot-switch-nob:before{background-color:#737373;height:8px;left:4px;top:4px;width:8px}div#onetrust-pc-sdk .ot-tgl input:checked+.ot-switch .ot-switch-nob:before{background-color:#007dbc;height:12px;left:0;top:2px;width:12px}div#onetrust-pc-sdk .ot-tgl input:focus+.ot-switch .ot-switch-nob{box-shadow:0 0;outline:2px solid #007dbc!important;outline-offset:1px;transition:none}div#onetrust-consent-sdk #onetrust-pc-sdk .ot-acc-grpcntr.ot-acc-txt{background-color:transparent;padding-left:3px}div#onetrust-pc-sdk .ot-accordion-layout .ot-hlst-cntr,div#onetrust-pc-sdk .ot-accordion-layout .ot-vlst-cntr{overflow:visible;width:100%}div#onetrust-pc-sdk .ot-pc-footer{border-top:0 solid}div#onetrust-pc-sdk .ot-btn-container{padding-top:24px;text-align:center}div#onetrust-pc-sdk .ot-pc-footer button{margin:8px 0;background-color:#fff}div#onetrust-pc-sdk .ot-pc-footer-logo{background-color:#fff}div#onetrust-pc-sdk #ot-lst-title span{font-size:24px;font-weight:400;line-height:32px}div#onetrust-pc-sdk #ot-host-lst .ot-host-desc,div#onetrust-pc-sdk #ot-host-lst .ot-host-expand,div#onetrust-pc-sdk #ot-host-lst .ot-host-name,div#onetrust-pc-sdk #ot-host-lst .ot-host-name a,div#onetrust-pc-sdk .back-btn-handler,div#onetrust-pc-sdk .ot-host-opt li>div div{font-size:16px;font-weight:400;line-height:24px}div#onetrust-pc-sdk #ot-pc-lst{top:0}div#onetrust-pc-sdk .back-btn-handler{text-decoration:none!important}div#onetrust-pc-sdk #filter-btn-handler:hover svg{filter:invert(1)}div#onetrust-pc-sdk .back-btn-handler svg{width:16px;height:16px}div#onetrust-pc-sdk .ot-host-item>button{background:0 0!important;border:0!important;height:66px!important;max-width:none!important;width:calc(100% - 5px)!important;transform:translate(2px,2px)}div#onetrust-pc-sdk .ot-host-item{border-bottom:2px solid #b9b9b9;padding:0}div#onetrust-pc-sdk .ot-host-item .ot-acc-hdr{margin:0 0 -6px;padding:8px 0}div#onetrust-pc-sdk ul li:first-child{border-top:2px solid #b9b9b9}div#onetrust-pc-sdk .ot-host-item .ot-plus-minus{margin:0 8px 0 0}div#onetrust-pc-sdk .ot-search-cntr{width:calc(100% - 48px)}div#onetrust-pc-sdk .ot-host-opt .ot-host-info{background-color:transparent}div#onetrust-pc-sdk .ot-host-opt li>div div{padding:0}div#onetrust-pc-sdk #vendor-search-handler{border-radius:0;border-color:#939393;border-style:solid;border-width:2px 0 2px 2px;font-size:20px;height:48px;margin:0}div#onetrust-pc-sdk .ot-lst-subhdr svg{right:0;top:8px}div#onetrust-pc-sdk .ot-fltr-cntr{box-sizing:border-box;right:0;width:48px}div#onetrust-pc-sdk #filter-btn-handler{width:48px!important;padding:8px!important}div#onetrust-pc-sdk #ot-fltr-modal{max-height:400px}
.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy.otRelFont{font-size:1rem}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:1rem}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            .ot-floating-button__front{background-image:url('https://cdn.cookielaw.org/logos/static/ot_persistent_cookie_icon.png')}</style><script src="https://static.addtoany.com/menu/modules/core.pt5ow5lr.js" type="module"></script><style></style><style type="text/css">.a2a_hide{display:none}.a2a_logo_color{background-color:#0166ff}.a2a_menu,.a2a_menu *{-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box;float:none;margin:0;padding:0;position:static;height:auto;width:auto}.a2a_menu{border-radius:6px;display:none;direction:ltr;background:#FFF;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;color:#000;line-height:12px;border:1px solid #CCC;vertical-align:baseline;overflow:hidden}.a2a_mini{min-width:200px;position:absolute;width:300px;z-index:9999997}.a2a_overlay{display:none;background:#616c7deb;backdrop-filter:blur(4px);-webkit-backdrop-filter:blur(4px);position:fixed;top:0;right:0;left:0;bottom:0;z-index:9999998;-webkit-tap-highlight-color:transparent;transition:opacity .14s,backdrop-filter .14s}.a2a_full{background:#FFF;border:1px solid #FFF;box-shadow:#2a2a2a1a 0 0 20px 10px;height:auto;height:calc(320px);top:15%;left:50%;margin-left:-320px;position:fixed;text-align:center;width:640px;z-index:9999999;transition:transform .14s,opacity .14s}.a2a_full_footer,.a2a_full_header,.a2a_full_services{border:0;margin:0;padding:12px;box-sizing:border-box}.a2a_full_header{padding-bottom:8px}.a2a_full_services{height:280px;overflow-y:scroll;padding:0 12px;-webkit-overflow-scrolling:touch}.a2a_full_services .a2a_i{display:inline-block;float:none;width:181px;width:calc(33.334% - 18px)}div.a2a_full_footer{font-size:12px;text-align:center;padding:8px 14px}div.a2a_full_footer a,div.a2a_full_footer a:visited{display:inline;font-size:12px;line-height:14px;padding:8px 14px}div.a2a_full_footer a:focus,div.a2a_full_footer a:hover{background:0 0;border:0;color:#0166FF}div.a2a_full_footer a span.a2a_s_a2a,div.a2a_full_footer a span.a2a_w_a2a{background-size:14px;border-radius:3px;display:inline-block;height:14px;line-height:14px;margin:0 3px 0 0;vertical-align:top;width:14px}.a2a_modal{height:0;left:50%;margin-left:-320px;position:fixed;text-align:center;top:15%;width:640px;z-index:9999999;transition:transform .14s,opacity .14s;-webkit-tap-highlight-color:transparent}.a2a_modal_body{background:0 0;border:0;font:24px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;position:relative;height:auto;width:auto}.a2a_thanks{color:#fff;height:auto;margin-top:20px;width:auto}.a2a_thanks>div:first-child{margin:0 0 40px 0}.a2a_thanks div *{height:inherit}#a2a_copy_link{background:#FFF;border:1px solid #FFF;cursor:pointer;margin-top:15%}label.a2a_s_link#a2a_copy_link_icon,label.a2a_w_link#a2a_copy_link_icon{background-size:48px;border-radius:0;display:inline-block;height:48px;left:0;line-height:48px;margin:0 3px 0 0;position:absolute;vertical-align:top;width:48px}#a2a_modal input#a2a_copy_link_text{background-color:transparent;border:0;color:#2A2A2A;cursor:pointer;font:inherit;height:48px;left:62px;max-width:initial;min-height:auto;padding:0;position:relative;width:564px;width:calc(100% - 76px)}#a2a_copy_link_copied{background-color:#0166ff;color:#fff;display:none;font:inherit;font-size:16px;margin-top:1px;padding:3px 8px}@media (forced-colors:active){.a2a_color_buttons a,.a2a_svg{forced-color-adjust:none}}@media (prefers-color-scheme:dark){.a2a_menu a,.a2a_menu a.a2a_i,.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more,i.a2a_i{border-color:#2a2a2a!important;color:#fff!important}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover,.a2a_menu_find_container{border-color:#444!important;background-color:#444!important}.a2a_menu:not(.a2a_thanks){background-color:#2a2a2a;border-color:#2a2a2a}.a2a_menu_find{color:#fff!important}.a2a_menu label.a2a_s_find svg{background-color:transparent!important}.a2a_menu label.a2a_s_find svg path{fill:#fff!important}.a2a_full{box-shadow:#00000066 0 0 20px 10px}.a2a_overlay{background-color:#373737eb}}@media print{.a2a_floating_style,.a2a_menu,.a2a_overlay{visibility:hidden}}@keyframes a2aFadeIn{from{opacity:0}to{opacity:1}}.a2a_starting{opacity:0}.a2a_starting.a2a_full,.a2a_starting.a2a_modal{transform:scale(.8)}@media (max-width:639px){.a2a_full{border-radius:0;top:15%;left:0;margin-left:auto;width:100%}.a2a_modal{left:0;margin-left:10px;width:calc(100% - 20px)}}@media (min-width:318px) and (max-width:437px){.a2a_full .a2a_full_services .a2a_i{width:calc(50% - 18px)}}@media (max-width:317px){.a2a_full .a2a_full_services .a2a_i{width:calc(100% - 18px)}}@media (max-height:436px){.a2a_full{bottom:40px;height:auto;top:40px}}@media (max-height:550px){.a2a_modal{top:30px}}@media (max-height:360px){.a2a_modal{top:20px}.a2a_thanks>div:first-child{margin-bottom:20px}}.a2a_menu a{color:#0166FF;text-decoration:none;font:16px sans-serif-light,HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Arial,Helvetica,"Liberation Sans",sans-serif;line-height:14px;height:auto;width:auto;outline:0}.a2a_menu a.a2a_i:visited,.a2a_menu a.a2a_more{color:#0166FF}.a2a_menu a.a2a_i:active,.a2a_menu a.a2a_i:focus,.a2a_menu a.a2a_i:hover,.a2a_menu a.a2a_more:active,.a2a_menu a.a2a_more:focus,.a2a_menu a.a2a_more:hover{color:#2A2A2A;border-color:#EEE;border-style:solid;background-color:#EEE;text-decoration:none}.a2a_menu label.a2a_s_find{background-size:24px;height:24px;left:8px;pointer-events:auto;position:absolute;top:7px;width:24px}.a2a_menu label.a2a_s_find svg{background-color:#FFF}.a2a_menu label.a2a_s_find svg path{fill:#CCC}#a2a_menu_container{display:inline-block}.a2a_menu_find_container{border:1px solid #CCC;border-radius:6px;padding:2px 24px 2px 0;position:relative;text-align:left}.a2a_cols_container .a2a_col1{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}#a2a_modal input,#a2a_modal input[type=text],.a2a_menu input,.a2a_menu input[type=text]{display:block;background-image:none;box-shadow:none;line-height:100%;margin:0;outline:0;overflow:hidden;padding:0;-moz-box-shadow:none;-webkit-box-shadow:none;-webkit-appearance:none}#a2afeed_find_container input,#a2afeed_find_container input[type=text],#a2apage_find_container input,#a2apage_find_container input[type=text]{background-color:transparent;border:0;box-sizing:content-box;color:#2A2A2A;float:none;font:inherit;font-size:16px;height:28px;line-height:20px;left:38px;outline:0;margin:0;max-width:initial;min-height:initial;padding:2px 0;position:relative;width:99%}.a2a_clear{clear:both}.a2a_svg{background-repeat:no-repeat;display:block;overflow:hidden;height:32px;line-height:32px;padding:0;pointer-events:none;width:32px}.a2a_svg svg{background-repeat:no-repeat;background-position:50% 50%;border:none;display:block;left:0;margin:0 auto;overflow:hidden;padding:0;position:relative;top:0;width:auto;height:auto}a.a2a_i,i.a2a_i{display:block;float:left;border:1px solid #FFF;line-height:24px;padding:6px 8px;text-align:left;white-space:nowrap;overflow:hidden;text-overflow:ellipsis;width:132px}a.a2a_i span,a.a2a_more span{display:inline-block;overflow:hidden;vertical-align:top}a.a2a_i .a2a_svg{margin:0 6px 0 0}a.a2a_i .a2a_svg,a.a2a_more .a2a_svg{background-size:24px;height:24px;line-height:24px;width:24px}a.a2a_sss:hover{border-left:1px solid #CCC}a.a2a_more{border-bottom:1px solid #FFF;border-left:0;border-right:0;line-height:24px;margin:6px 0 0;padding:6px;-webkit-touch-callout:none}a.a2a_more span{height:24px;margin:0 6px 0 0}.a2a_kit .a2a_svg{background-repeat:repeat}.a2a_default_style a:empty,.a2a_flex_style a:empty,.a2a_floating_style a:empty,.a2a_overlay_style a:empty{display:none}.a2a_color_buttons a,.a2a_floating_style a{text-decoration:none}.a2a_default_style:not(.a2a_flex_style) a{float:left;line-height:16px;padding:0 2px}.a2a_default_style a:hover .a2a_svg,.a2a_floating_style a:hover .a2a_svg,.a2a_overlay_style a:hover .a2a_svg svg{opacity:.7}.a2a_overlay_style.a2a_default_style a:hover .a2a_svg{opacity:1}.a2a_default_style .a2a_count,.a2a_default_style .a2a_svg,.a2a_floating_style .a2a_svg,.a2a_menu .a2a_svg,.a2a_vertical_style .a2a_count,.a2a_vertical_style .a2a_svg{border-radius:4px}.a2a_default_style .a2a_counter img,.a2a_default_style .a2a_dd,.a2a_default_style .a2a_svg{float:left}.a2a_default_style .a2a_img_text{margin-right:4px}.a2a_default_style .a2a_divider{border-left:1px solid #000;display:inline;float:left;height:16px;line-height:16px;margin:0 5px}.a2a_kit a{cursor:pointer;transition:none}.a2a_floating_style{background-color:#fff;border-radius:6px;position:fixed;z-index:9999995}.a2a_overlay_style{z-index:2147483647}.a2a_floating_style,.a2a_overlay_style{animation:a2aFadeIn .2s ease-in;padding:4px}.a2a_vertical_style:not(.a2a_flex_style) a{clear:left;display:block;overflow:hidden;padding:4px}.a2a_floating_style.a2a_default_style{bottom:0}.a2a_floating_style.a2a_default_style a,.a2a_overlay_style.a2a_default_style a{padding:4px}.a2a_count{background-color:#fff;border:1px solid #ccc;box-sizing:border-box;color:#2a2a2a;display:block;float:left;font:12px Arial,Helvetica,sans-serif;height:16px;margin-left:4px;position:relative;text-align:center;width:50px}.a2a_count:after,.a2a_count:before{border:solid transparent;border-width:4px 4px 4px 0;content:"";height:0;left:0;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:50%;width:0}.a2a_count:before{border-right-color:#ccc}.a2a_count:after{border-right-color:#fff;margin-left:-3px}.a2a_count span{animation:a2aFadeIn .14s ease-in}.a2a_vertical_style .a2a_counter img{display:block}.a2a_vertical_style .a2a_count{float:none;margin-left:0;margin-top:6px}.a2a_vertical_style .a2a_count:after,.a2a_vertical_style .a2a_count:before{border:solid transparent;border-width:0 4px 4px 4px;content:"";height:0;left:50%;line-height:0;margin:-4px 0 0 -4px;position:absolute;top:0;width:0}.a2a_vertical_style .a2a_count:before{border-bottom-color:#ccc}.a2a_vertical_style .a2a_count:after{border-bottom-color:#fff;margin-top:-3px}.a2a_color_buttons .a2a_count,.a2a_color_buttons .a2a_count:after,.a2a_color_buttons .a2a_count:before,.a2a_color_buttons.a2a_vertical_style .a2a_count:after,.a2a_color_buttons.a2a_vertical_style .a2a_count:before{background-color:transparent;border:none;color:#fff;float:none;width:auto}.a2a_color_buttons.a2a_vertical_style .a2a_count{margin-top:0}.a2a_flex_style{display:flex;align-items:flex-start;gap:0}.a2a_default_style.a2a_flex_style{left:0;right:0;width:100%}.a2a_vertical_style.a2a_flex_style{flex-direction:column;top:0;bottom:0}.a2a_flex_style a{display:flex;justify-content:center;flex:1;padding:4px}.a2a_flex_style.a2a_vertical_style a{flex-direction:column}.a2a_floating_style.a2a_color_buttons,.a2a_floating_style.a2a_flex_style{border-radius:0;padding:0}.a2a_floating_style.a2a_default_style.a2a_flex_style{bottom:0}.a2a_kit.a2a_flex_style .a2a_counter img,.a2a_kit.a2a_flex_style .a2a_dd,.a2a_kit.a2a_flex_style .a2a_svg{float:none}.a2a_nowrap{white-space:nowrap}.a2a_note{margin:0 auto;padding:9px;font-size:12px;text-align:center}.a2a_note .a2a_note_note{margin:0;color:#2A2A2A}.a2a_wide a{display:block;margin-top:3px;border-top:1px solid #EEE;text-align:center}.a2a_label{position:absolute!important;clip-path:polygon(0px 0px,0px 0px,0px 0px);-webkit-clip-path:polygon(0px 0px,0px 0px,0px 0px);overflow:hidden;height:1px;width:1px}.a2a_kit,.a2a_menu,.a2a_modal,.a2a_overlay{-ms-touch-action:manipulation;touch-action:manipulation;outline:0}.a2a_dd{-webkit-user-drag:none}.a2a_dd img{border:0}.a2a_button_facebook_like iframe{max-width:none}</style><script type="text/javascript" src="https://www.cell.com/products/marlin/cell/releasedAssets/public/mathjax-511ffcc670c4d93d695bf67802ba7aad/tex-mml-chtml.js" async=""></script><link href="https://crossmark-cdn.crossref.org/widget/v2.0/style.css" type="text/css" rel="stylesheet"><style type="text/css">.CtxtMenu_InfoClose {  top:.2em; right:.2em;}
.CtxtMenu_InfoContent {  overflow:auto; text-align:left; font-size:80%;  padding:.4em .6em; border:1px inset; margin:1em 0px;  max-height:20em; max-width:30em; background-color:#EEEEEE;  white-space:normal;}
.CtxtMenu_Info.CtxtMenu_MousePost {outline:none;}
.CtxtMenu_Info {  position:fixed; left:50%; width:auto; text-align:center;  border:3px outset; padding:1em 2em; background-color:#DDDDDD;  color:black;  cursor:default; font-family:message-box; font-size:120%;  font-style:normal; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 15px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius:15px;               /* Safari and Chrome */  -moz-border-radius:15px;                  /* Firefox */  -khtml-border-radius:15px;                /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */  filter:progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color="gray", Positive="true"); /* IE */}
</style><style type="text/css">.CtxtMenu_MenuClose {  position:absolute;  cursor:pointer;  display:inline-block;  border:2px solid #AAA;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  font-family: "Courier New", Courier;  font-size:24px;  color:#F0F0F0}
.CtxtMenu_MenuClose span {  display:block; background-color:#AAA; border:1.5px solid;  border-radius:18px;  -webkit-border-radius: 18px;             /* Safari and Chrome */  -moz-border-radius: 18px;                /* Firefox */  -khtml-border-radius: 18px;              /* Konqueror */  line-height:0;  padding:8px 0 6px     /* may need to be browser-specific */}
.CtxtMenu_MenuClose:hover {  color:white!important;  border:2px solid #CCC!important}
.CtxtMenu_MenuClose:hover span {  background-color:#CCC!important}
.CtxtMenu_MenuClose:hover:focus {  outline:none}
</style><style type="text/css">.CtxtMenu_Menu {  position:absolute;  background-color:white;  color:black;  width:auto; padding:5px 0px;  border:1px solid #CCCCCC; margin:0; cursor:default;  font: menu; text-align:left; text-indent:0; text-transform:none;  line-height:normal; letter-spacing:normal; word-spacing:normal;  word-wrap:normal; white-space:nowrap; float:none; z-index:201;  border-radius: 5px;                     /* Opera 10.5 and IE9 */  -webkit-border-radius: 5px;             /* Safari and Chrome */  -moz-border-radius: 5px;                /* Firefox */  -khtml-border-radius: 5px;              /* Konqueror */  box-shadow:0px 10px 20px #808080;         /* Opera 10.5 and IE9 */  -webkit-box-shadow:0px 10px 20px #808080; /* Safari 3 & Chrome */  -moz-box-shadow:0px 10px 20px #808080;    /* Forefox 3.5 */  -khtml-box-shadow:0px 10px 20px #808080;  /* Konqueror */}
.CtxtMenu_MenuItem {  padding: 1px 2em;  background:transparent;}
.CtxtMenu_MenuArrow {  position:absolute; right:.5em; padding-top:.25em; color:#666666;  font-family: null; font-size: .75em}
.CtxtMenu_MenuActive .CtxtMenu_MenuArrow {color:white}
.CtxtMenu_MenuArrow.CtxtMenu_RTL {left:.5em; right:auto}
.CtxtMenu_MenuCheck {  position:absolute; left:.7em;  font-family: null}
.CtxtMenu_MenuCheck.CtxtMenu_RTL { right:.7em; left:auto }
.CtxtMenu_MenuRadioCheck {  position:absolute; left: .7em;}
.CtxtMenu_MenuRadioCheck.CtxtMenu_RTL {  right: .7em; left:auto}
.CtxtMenu_MenuInputBox {  padding-left: 1em; right:.5em; color:#666666;  font-family: null;}
.CtxtMenu_MenuInputBox.CtxtMenu_RTL {  left: .1em;}
.CtxtMenu_MenuComboBox {  left:.1em; padding-bottom:.5em;}
.CtxtMenu_MenuSlider {  left: .1em;}
.CtxtMenu_SliderValue {  position:absolute; right:.1em; padding-top:.25em; color:#333333;  font-size: .75em}
.CtxtMenu_SliderBar {  outline: none; background: #d3d3d3}
.CtxtMenu_MenuLabel {  padding: 1px 2em 3px 1.33em;  font-style:italic}
.CtxtMenu_MenuRule {  border-top: 1px solid #DDDDDD;  margin: 4px 3px;}
.CtxtMenu_MenuDisabled {  color:GrayText}
.CtxtMenu_MenuActive {  background-color: #606872;  color: white;}
.CtxtMenu_MenuDisabled:focus {  background-color: #E8E8E8}
.CtxtMenu_MenuLabel:focus {  background-color: #E8E8E8}
.CtxtMenu_ContextMenu:focus {  outline:none}
.CtxtMenu_ContextMenu .CtxtMenu_MenuItem:focus {  outline:none}
.CtxtMenu_SelectionMenu {  position:relative; float:left;  border-bottom: none; -webkit-box-shadow:none; -webkit-border-radius:0px; }
.CtxtMenu_SelectionItem {  padding-right: 1em;}
.CtxtMenu_Selection {  right: 40%; width:50%; }
.CtxtMenu_SelectionBox {  padding: 0em; max-height:20em; max-width: none;  background-color:#FFFFFF;}
.CtxtMenu_SelectionDivider {  clear: both; border-top: 2px solid #000000;}
.CtxtMenu_Menu .CtxtMenu_MenuClose {  top:-10px; left:-10px}
</style><style id="MJX-CHTML-styles">
mjx-container[overflow="scroll"][display] {
  overflow-x: auto;
  min-width: initial !important;
}

mjx-container[overflow="truncate"][display] {
  overflow-x: hidden;
  min-width: initial !important;
}

mjx-container[jax="CHTML"] {
  white-space: nowrap;
}

mjx-container [space="1"] {
  margin-left: .111em;
}

mjx-container [space="2"] {
  margin-left: .167em;
}

mjx-container [space="3"] {
  margin-left: .222em;
}

mjx-container [space="4"] {
  margin-left: .278em;
}

mjx-container [space="5"] {
  margin-left: .333em;
}

mjx-container [rspace="1"] {
  margin-right: .111em;
}

mjx-container [rspace="2"] {
  margin-right: .167em;
}

mjx-container [rspace="3"] {
  margin-right: .222em;
}

mjx-container [rspace="4"] {
  margin-right: .278em;
}

mjx-container [rspace="5"] {
  margin-right: .333em;
}

mjx-container [size="s"] {
  font-size: 70.7%;
}

mjx-container [size="ss"] {
  font-size: 50%;
}

mjx-container [size="Tn"] {
  font-size: 60%;
}

mjx-container [size="sm"] {
  font-size: 85%;
}

mjx-container [size="lg"] {
  font-size: 120%;
}

mjx-container [size="Lg"] {
  font-size: 144%;
}

mjx-container [size="LG"] {
  font-size: 173%;
}

mjx-container [size="hg"] {
  font-size: 207%;
}

mjx-container [size="HG"] {
  font-size: 249%;
}

mjx-container [width="full"] {
  width: 100%;
}

mjx-box {
  display: inline-block;
}

mjx-block {
  display: block;
}

mjx-itable {
  display: inline-table;
}

mjx-row {
  display: table-row;
}

mjx-row > * {
  display: table-cell;
}

mjx-container [inline-breaks] {
  display: inline;
}

mjx-mtext {
  display: inline-block;
  text-align: left;
}

mjx-mstyle {
  display: inline-block;
}

mjx-merror {
  display: inline-block;
  color: red;
  background-color: yellow;
}

mjx-mphantom {
  visibility: hidden;
}

mjx-math {
  display: inline-block;
  text-align: left;
  line-height: 0;
  text-indent: 0;
  font-style: normal;
  font-weight: normal;
  font-size: 100%;
  font-size-adjust: none;
  letter-spacing: normal;
  word-wrap: normal;
  word-spacing: normal;
  direction: ltr;
  padding: 1px 0;
}

mjx-container[jax="CHTML"][display="true"] {
  display: block;
  text-align: center;
  justify-content: center;
  margin: 1em 0;
}

mjx-container[jax="CHTML"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="CHTML"][display="true"] mjx-math {
  padding: 0;
}

mjx-container[jax="CHTML"][justify="left"] {
  text-align: left;
  justify-content: left;
}

mjx-container[jax="CHTML"][justify="right"] {
  text-align: right;
  justify-content: right;
}

mjx-container[jax="CHTML"] mjx-break::after {
  content: " ";
  white-space: normal;
  font-family: MJX-BRK;
}

mjx-break[size="0"] {
  letter-spacing: -0.999em;
}

mjx-break[size="1"] {
  letter-spacing: -0.889em;
}

mjx-break[size="2"] {
  letter-spacing: -0.833em;
}

mjx-break[size="3"] {
  letter-spacing: -0.778em;
}

mjx-break[size="4"] {
  letter-spacing: -0.722em;
}

mjx-break[size="5"] {
  letter-spacing: -0.667em;
}

mjx-math[breakable] {
  display: inline;
}

mjx-mrow {
  display: inline-block;
  text-align: left;
}

mjx-linestack, mjx-mrow[break-top] {
  display: inline-table;
  width: 100%;
}

mjx-linestack[break-align="bottom"], mjx-mrow[break-top][break-align="bottom"] {
  display: inline-block;
}

mjx-linestack[break-align="middle"], mjx-mrow[break-top][break-align="middle"] {
  vertical-align: middle;
}

mjx-linestack[break-align="center"], mjx-mrow[break-top][break-align="center"] {
  vertical-align: middle;
}

mjx-linestack[breakable] {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox {
  display: inline;
}

mjx-linestack[breakable] > mjx-linebox::before {
  white-space: pre;
  content: "\A";
}

mjx-linestack[breakable] > mjx-linebox::after {
  white-space: normal;
  content: " ";
  letter-spacing: -.999em;
  font-family: MJX-BRK;
}

mjx-linestack[breakable] > mjx-linebox:first-of-type::before {
  display: none;
}

mjx-linestack[breakable] > mjx-linebox:last-of-type::after {
  display: none;
}

mjx-linebox {
  display: block;
}

mjx-linebox[align="left"] {
  text-align: left;
}

mjx-linebox[align="center"] {
  text-align: center;
}

mjx-linebox[align="right"] {
  text-align: right;
}

mjx-linestrut {
  display: inline-block;
  height: 1em;
  vertical-align: -.25em;
}

mjx-mi {
  display: inline-block;
  text-align: left;
}

mjx-c {
  display: inline-block;
  width: 0;
  text-align: right;
}

mjx-utext {
  display: inline-block;
  padding: .75em 0 .2em 0;
}

mjx-mo {
  display: inline-block;
  text-align: left;
}

mjx-stretchy-h {
  display: inline-block;
}

mjx-stretchy-h > * {
  display: inline-block;
  width: 0;
}

mjx-stretchy-h > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: clip visible;
  width: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
  text-align: center;
}

mjx-stretchy-h > mjx-ext > mjx-c {
  transform: scalex(500);
  width: 0;
}

mjx-stretchy-v {
  display: inline-block;
  text-align: center;
}

mjx-stretchy-v > * {
  display: block;
  height: 0;
}

mjx-stretchy-v > mjx-ext {
  /* IE */ overflow: hidden;
  /* others */ overflow: visible clip;
  height: 100%;
  border: 0px solid transparent;
  box-sizing: border-box;
}

mjx-mark {
  display: inline-block;
  height: 0;
}

mjx-msub {
  display: inline-block;
  text-align: left;
}

mjx-msubsup {
  display: inline-block;
  text-align: left;
}

mjx-script {
  display: inline-block;
  padding-right: .05em;
  padding-left: .033em;
}

mjx-script > mjx-spacer {
  display: block;
}

mjx-mn {
  display: inline-block;
  text-align: left;
}

mjx-msup {
  display: inline-block;
  text-align: left;
}

mjx-mfrac {
  display: inline-block;
  text-align: left;
}

mjx-frac {
  display: inline-block;
  vertical-align: 0.17em;
  padding: 0 .22em;
}

mjx-frac[type="d"] {
  vertical-align: .04em;
}

mjx-frac[delims] {
  padding: 0 .1em;
}

mjx-frac[atop] {
  padding: 0 .12em;
}

mjx-frac[atop][delims] {
  padding: 0;
}

mjx-dtable {
  display: inline-table;
  width: 100%;
}

mjx-dtable > * {
  font-size: 2000%;
}

mjx-dbox {
  display: block;
  font-size: 5%;
}

mjx-num {
  display: block;
  text-align: center;
}

mjx-den {
  display: block;
  text-align: center;
}

mjx-mfrac[bevelled] > mjx-num {
  display: inline-block;
}

mjx-mfrac[bevelled] > mjx-den {
  display: inline-block;
}

mjx-den[align="right"], mjx-num[align="right"] {
  text-align: right;
}

mjx-den[align="left"], mjx-num[align="left"] {
  text-align: left;
}

mjx-nstrut {
  display: inline-block;
  height: .054em;
  width: 0;
  vertical-align: -.054em;
}

mjx-nstrut[type="d"] {
  height: .217em;
  vertical-align: -.217em;
}

mjx-dstrut {
  display: inline-block;
  height: .505em;
  width: 0;
}

mjx-dstrut[type="d"] {
  height: .726em;
}

mjx-line {
  display: block;
  box-sizing: border-box;
  min-height: 1px;
  height: .06em;
  border-top: .075em solid;
  margin: .06em -.1em;
  overflow: hidden;
}

mjx-line[type="d"] {
  margin: .18em -.1em;
}

mjx-munderover {
  display: inline-block;
  text-align: left;
}

mjx-munderover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-munderover:not([limits="false"]) > * {
  display: block;
}

mjx-msqrt {
  display: inline-block;
  text-align: left;
}

mjx-root {
  display: inline-block;
  white-space: nowrap;
}

mjx-surd {
  display: inline-block;
  vertical-align: top;
}

mjx-sqrt {
  display: inline-block;
  padding-top: .075em;
}

mjx-sqrt > mjx-box {
  border-top: .075em solid;
  padding-left: .03em;
  margin-left: -.03em;
}

mjx-sqrt.mjx-tall > mjx-box {
  padding-left: .3em;
  margin-left: -.3em;
}

mjx-mspace {
  display: inline-block;
  text-align: left;
}

mjx-mover {
  display: inline-block;
  text-align: left;
}

mjx-mover:not([limits="false"]) {
  padding-top: .1em;
}

mjx-mover:not([limits="false"]) > * {
  display: block;
  text-align: left;
}

mjx-munder {
  display: inline-block;
  text-align: left;
}

mjx-over {
  text-align: left;
}

mjx-munder:not([limits="false"]) {
  display: inline-table;
}

mjx-munder > mjx-row {
  text-align: left;
}

mjx-under {
  padding-bottom: .1em;
}

.MM-N {
  font-family: MJX-MM-ZERO, MJX-MM-N;
}

.MM-B {
  font-family: MJX-MM-ZERO, MJX-MM-B;
}

.MM-I {
  font-family: MJX-MM-ZERO, MJX-MM-I;
}

.MM-BI {
  font-family: MJX-MM-ZERO, MJX-MM-BI;
}

.MM-DS {
  font-family: MJX-MM-ZERO, MJX-MM-DS;
}

.MM-F {
  font-family: MJX-MM-ZERO, MJX-MM-F;
}

.MM-FB {
  font-family: MJX-MM-ZERO, MJX-MM-FB;
}

.MM-S {
  font-family: MJX-MM-ZERO, MJX-MM-S;
}

.MM-SB {
  font-family: MJX-MM-ZERO, MJX-MM-SB;
}

.MM-SS {
  font-family: MJX-MM-ZERO, MJX-MM-SS;
}

.MM-SSB {
  font-family: MJX-MM-ZERO, MJX-MM-SSB;
}

.MM-SSI {
  font-family: MJX-MM-ZERO, MJX-MM-SSI;
}

.MM-SSBI {
  font-family: MJX-MM-ZERO, MJX-MM-SSBI;
}

.MM-M {
  font-family: MJX-MM-ZERO, MJX-MM-M;
}

.MM-SO {
  font-family: MJX-MM-ZERO, MJX-MM-SO;
}

.MM-LO {
  font-family: MJX-MM-ZERO, MJX-MM-LO;
}

.MM-S3 {
  font-family: MJX-MM-ZERO, MJX-MM-S3;
}

.MM-S4 {
  font-family: MJX-MM-ZERO, MJX-MM-S4;
}

.MM-S5 {
  font-family: MJX-MM-ZERO, MJX-MM-S5;
}

.MM-S6 {
  font-family: MJX-MM-ZERO, MJX-MM-S6;
}

.MM-S7 {
  font-family: MJX-MM-ZERO, MJX-MM-S7;
}

.MM-MI {
  font-family: MJX-MM-ZERO, MJX-MM-MI;
}

.MM-C {
  font-family: MJX-MM-ZERO, MJX-MM-C;
}

.MM-CB {
  font-family: MJX-MM-ZERO, MJX-MM-CB;
}

.MM-OS {
  font-family: MJX-MM-ZERO, MJX-MM-OS;
}

.MM-OB {
  font-family: MJX-MM-ZERO, MJX-MM-OB;
}

.MM-V {
  font-family: MJX-MM-ZERO, MJX-MM-V;
}

.MM-LT {
  font-family: MJX-MM-ZERO, MJX-MM-LT;
}

.MM-RB {
  font-family: MJX-MM-ZERO, MJX-MM-RB;
}

.MM-EM {
  font-family: MJX-MM-ZERO, MJX-MM-EM;
}

@font-face /* MJX-MM-ZERO */ {
  font-family: MJX-MM-ZERO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-zero.woff") format("woff");
}

@font-face /* MJX-BRK */ {
  font-family: MJX-BRK;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-brk.woff") format("woff");
}

@font-face /* MJX-MM-N */ {
  font-family: MJX-MM-N;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-n.woff") format("woff");
}

@font-face /* MJX-MM-B */ {
  font-family: MJX-MM-B;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-b.woff") format("woff");
}

@font-face /* MJX-MM-I */ {
  font-family: MJX-MM-I;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-i.woff") format("woff");
}

@font-face /* MJX-MM-BI */ {
  font-family: MJX-MM-BI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-bi.woff") format("woff");
}

@font-face /* MJX-MM-DS */ {
  font-family: MJX-MM-DS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ds.woff") format("woff");
}

@font-face /* MJX-MM-F */ {
  font-family: MJX-MM-F;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-f.woff") format("woff");
}

@font-face /* MJX-MM-FB */ {
  font-family: MJX-MM-FB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-fb.woff") format("woff");
}

@font-face /* MJX-MM-S */ {
  font-family: MJX-MM-S;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s.woff") format("woff");
}

@font-face /* MJX-MM-SB */ {
  font-family: MJX-MM-SB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-sb.woff") format("woff");
}

@font-face /* MJX-MM-SS */ {
  font-family: MJX-MM-SS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ss.woff") format("woff");
}

@font-face /* MJX-MM-SSB */ {
  font-family: MJX-MM-SSB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssb.woff") format("woff");
}

@font-face /* MJX-MM-SSI */ {
  font-family: MJX-MM-SSI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssi.woff") format("woff");
}

@font-face /* MJX-MM-SSBI */ {
  font-family: MJX-MM-SSBI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ssbi.woff") format("woff");
}

@font-face /* MJX-MM-M */ {
  font-family: MJX-MM-M;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-m.woff") format("woff");
}

@font-face /* MJX-MM-SO */ {
  font-family: MJX-MM-SO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-so.woff") format("woff");
}

@font-face /* MJX-MM-LO */ {
  font-family: MJX-MM-LO;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lo.woff") format("woff");
}

@font-face /* MJX-MM-S3 */ {
  font-family: MJX-MM-S3;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s3.woff") format("woff");
}

@font-face /* MJX-MM-S4 */ {
  font-family: MJX-MM-S4;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s4.woff") format("woff");
}

@font-face /* MJX-MM-S5 */ {
  font-family: MJX-MM-S5;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s5.woff") format("woff");
}

@font-face /* MJX-MM-S6 */ {
  font-family: MJX-MM-S6;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s6.woff") format("woff");
}

@font-face /* MJX-MM-S7 */ {
  font-family: MJX-MM-S7;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-s7.woff") format("woff");
}

@font-face /* MJX-MM-MI */ {
  font-family: MJX-MM-MI;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-mi.woff") format("woff");
}

@font-face /* MJX-MM-C */ {
  font-family: MJX-MM-C;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-c.woff") format("woff");
}

@font-face /* MJX-MM-CB */ {
  font-family: MJX-MM-CB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-cb.woff") format("woff");
}

@font-face /* MJX-MM-OS */ {
  font-family: MJX-MM-OS;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-os.woff") format("woff");
}

@font-face /* MJX-MM-OB */ {
  font-family: MJX-MM-OB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-ob.woff") format("woff");
}

@font-face /* MJX-MM-V */ {
  font-family: MJX-MM-V;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-v.woff") format("woff");
}

@font-face /* MJX-MM-LT */ {
  font-family: MJX-MM-LT;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-lt.woff") format("woff");
}

@font-face /* MJX-MM-RB */ {
  font-family: MJX-MM-RB;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-rb.woff") format("woff");
}

@font-face /* MJX-MM-EM */ {
  font-family: MJX-MM-EM;
  src: url("https://cdn.jsdelivr.net/npm/mathjax-modern-font/chtml/woff/mjx-mm-em.woff") format("woff");
}

mjx-stretchy-h.mjx-c305 mjx-ext mjx-c {
  padding: 0.67em 0 0 0;
  width: 0.19em;
}

mjx-c.mjx-c1D43F {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c1D461 {
  padding: 0.626em 0.361em 0.011em 0;
}

mjx-c.mjx-c3C {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c1D45A {
  padding: 0.442em 0.878em 0.011em 0;
}

mjx-c.mjx-c1D453 {
  padding: 0.705em 0.552em 0.205em 0;
}

mjx-c.mjx-c3A8 {
  padding: 0.683em 0.778em 0 0;
}

mjx-c.mjx-c223C {
  padding: 0.366em 0.773em 0 0;
}

mjx-c.mjx-c1D434 {
  padding: 0.716em 0.75em 0 0;
}

mjx-c.mjx-c31 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2212 {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D707 {
  padding: 0.442em 0.603em 0.216em 0;
}

mjx-c.mjx-c1D6FC {
  padding: 0.442em 0.64em 0.011em 0;
}

mjx-c.mjx-c32 {
  padding: 0.666em 0.5em 0 0;
}

mjx-c.mjx-c2062 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c1D45D {
  padding: 0.442em 0.503em 0.194em 0;
}

mjx-c.mjx-c3D {
  padding: 0.367em 0.778em 0 0;
}

mjx-c.mjx-c2B {
  padding: 0.583em 0.778em 0.083em 0;
}

mjx-c.mjx-c1D45E {
  padding: 0.442em 0.452em 0.194em 0;
}

mjx-c.mjx-c1D441 {
  padding: 0.683em 0.881em 0 0;
}

mjx-c.mjx-c1D440 {
  padding: 0.683em 1.044em 0 0;
}

mjx-c.mjx-c1D450 {
  padding: 0.442em 0.433em 0.011em 0;
}

mjx-c.mjx-c1D44E {
  padding: 0.442em 0.529em 0.011em 0;
}

mjx-c.mjx-c1D45C {
  padding: 0.442em 0.485em 0.011em 0;
}

mjx-c.mjx-c1D449 {
  padding: 0.683em 0.769em 0.022em 0;
}

mjx-c.mjx-c1D443 {
  padding: 0.683em 0.754em 0 0;
}

mjx-c.mjx-c1D45F {
  padding: 0.442em 0.451em 0.011em 0;
}

mjx-c.mjx-c2061 {
  padding: 0 0 0 0;
}

mjx-c.mjx-c7B {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c7C {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c1D437 {
  padding: 0.683em 0.828em 0 0;
}

mjx-c.mjx-c7D {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-cAC {
  padding: 0.367em 0.667em 0 0;
}

mjx-c.mjx-c1D454 {
  padding: 0.442em 0.477em 0.205em 0;
}

mjx-c.mjx-c28 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c29 {
  padding: 0.748em 0.389em 0.248em 0;
}

mjx-c.mjx-c221E.MM-I {
  padding: 0.456em 0.942em 0.025em 0;
}

mjx-c.mjx-c222B {
  padding: 0.805em 0.665em 0.306em 0;
}

mjx-c.mjx-c221A {
  padding: 0.04em 0.853em 0.96em 0;
}

mjx-c.mjx-c1D70B {
  padding: 0.431em 0.57em 0.011em 0;
}

mjx-c.mjx-c1D452 {
  padding: 0.442em 0.466em 0.011em 0;
}

mjx-c.mjx-c1D465 {
  padding: 0.442em 0.572em 0.011em 0;
}

mjx-c.mjx-c1D451 {
  padding: 0.694em 0.52em 0.011em 0;
}

mjx-c.mjx-c1D6F7 {
  padding: 0.683em 0.667em 0 0;
}

mjx-c.mjx-c3A6 {
  padding: 0.683em 0.722em 0 0;
}

mjx-c.mjx-c2E {
  padding: 0.106em 0.278em 0 0;
}

mjx-c.mjx-c56 {
  padding: 0.683em 0.75em 0.022em 0;
}

mjx-c.mjx-c2F {
  padding: 0.75em 0.5em 0.25em 0;
}

mjx-c.mjx-c1D44F {
  padding: 0.694em 0.429em 0.011em 0;
}

mjx-c.mjx-c1D467 {
  padding: 0.442em 0.467em 0.011em 0;
}

mjx-c.mjx-c1D6FE {
  padding: 0.442em 0.543em 0.215em 0;
}

mjx-c.mjx-c1D456 {
  padding: 0.661em 0.345em 0.011em 0;
}

mjx-c.mjx-c1D44B {
  padding: 0.683em 0.851em 0 0;
}

mjx-c.mjx-c50 {
  padding: 0.683em 0.681em 0 0;
}

mjx-c.mjx-c6F {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c69 {
  padding: 0.657em 0.278em 0 0;
}

mjx-c.mjx-c73 {
  padding: 0.448em 0.394em 0.011em 0;
}

mjx-c.mjx-c6E {
  padding: 0.442em 0.556em 0 0;
}

mjx-c.mjx-c47 {
  padding: 0.705em 0.785em 0.022em 0;
}

mjx-c.mjx-c61 {
  padding: 0.448em 0.5em 0.011em 0;
}

mjx-c.mjx-c6D {
  padding: 0.442em 0.833em 0 0;
}

mjx-c.mjx-c2C {
  padding: 0.106em 0.278em 0.193em 0;
}

mjx-c.mjx-c1D6FD {
  padding: 0.706em 0.574em 0.194em 0;
}

mjx-c.mjx-c1D446 {
  padding: 0.705em 0.645em 0.022em 0;
}

mjx-c.mjx-c1D417 {
  padding: 0.686em 0.869em 0 0;
}

mjx-c.mjx-c4D {
  padding: 0.683em 0.917em 0 0;
}

mjx-c.mjx-c75 {
  padding: 0.442em 0.556em 0.011em 0;
}

mjx-c.mjx-c6C {
  padding: 0.694em 0.278em 0 0;
}

mjx-c.mjx-c74 {
  padding: 0.615em 0.389em 0.011em 0;
}

mjx-c.mjx-c1D429 {
  padding: 0.45em 0.639em 0.194em 0;
}

mjx-c.mjx-c2026 {
  padding: 0.106em 0.837em 0 0;
}

mjx-c.mjx-c37 {
  padding: 0.676em 0.5em 0.022em 0;
}

mjx-c.mjx-c34 {
  padding: 0.677em 0.5em 0 0;
}

mjx-c.mjx-c38 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c2208 {
  padding: 0.543em 0.667em 0.043em 0;
}

mjx-c.mjx-c2211 {
  padding: 0.75em 1.056em 0.25em 0;
}

mjx-c.mjx-c3E {
  padding: 0.547em 0.778em 0.047em 0;
}

mjx-c.mjx-c2265 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c1D706 {
  padding: 0.694em 0.583em 0.013em 0;
}

mjx-c.mjx-c1D43A {
  padding: 0.705em 0.786em 0.022em 0;
}

mjx-c.mjx-c1D436 {
  padding: 0.705em 0.76em 0.022em 0;
}

mjx-c.mjx-c30 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c1D460 {
  padding: 0.442em 0.469em 0.011em 0;
}

mjx-c.mjx-c1D45B {
  padding: 0.442em 0.6em 0.011em 0;
}

mjx-c.mjx-c2264 {
  padding: 0.631em 0.778em 0.119em 0;
}

mjx-c.mjx-c1D435 {
  padding: 0.683em 0.759em 0 0;
}

mjx-c.mjx-c28.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c29.MM-LO {
  padding: 0.847em 0.458em 0.347em 0;
}

mjx-c.mjx-c1D447 {
  padding: 0.677em 0.704em 0 0;
}

mjx-c.mjx-c36 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c33 {
  padding: 0.666em 0.5em 0.022em 0;
}

mjx-c.mjx-c7B.MM-LO {
  padding: 0.85em 0.583em 0.35em 0;
}

mjx-c.mjx-c7D.MM-LO {
  padding: 0.85em 0.583em 0.35em 0;
}

mjx-c.mjx-c4E {
  padding: 0.683em 0.75em 0 0;
}

mjx-c.mjx-c45 {
  padding: 0.68em 0.681em 0 0;
}

mjx-c.mjx-c1D719 {
  padding: 0.694em 0.596em 0.205em 0;
}

mjx-c.mjx-c1D442 {
  padding: 0.705em 0.763em 0.022em 0;
}

mjx-c.mjx-c1D438 {
  padding: 0.68em 0.764em 0 0;
}

mjx-c.mjx-c5B {
  padding: 0.75em 0.278em 0.25em 0;
}

mjx-c.mjx-c5D {
  padding: 0.75em 0.278em 0.25em 0;
}
</style></head>
<body class="pb-ui">
    <div class="skip">
        <a href="#page-body-id" id="skipNavigationLink" title="Skip to Main Content" tabindex="0" class="skipToContent">Skip to Main Content</a>
        <a href="#mega-menu-nav" id="skipToMainMenu" title="Skip to Main Menu" tabindex="0" class="skipToContent">Skip to Main Menu</a>
    </div>



    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="ux3-layout-widget" data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" class="ux3">
        
    

        


    

            

            
                <div class="cell fonts-loaded">
            

            <div data-widget-id="832ad3f3-97ac-45ab-9af1-90c6f7bff54a" data-widget-name="ux3-layout-widget"><div>
    

        


    

            

            

            <div id="a1f1ba25-79af-4194-aaa9-66206104a800" class="widget loginPopup none widget-none widget-compact-all"><div class="wrapped"><div class="widget-body body body-none body-compact-all"><div role="dialog" aria-modal="true" aria-label="Login" class="popup login-popup hidden"><div class="content"><a href="#" aria-label="Close login popup" title="Close login popup" role="button" class="close"><i class="icon-close_thin"></i></a><h2 class="login-title">Login to your account</h2><div class="login-form"><p tabindex="-1" class="message error"></p><form action="/action/doLogin?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" method="post"><input type="hidden" name="redirectUri" value="https://www.cell.com/cell/fulltext/S0092-8674(19)31398-4"><input type="hidden" name="referer" value="/cell/fulltext/S0092-8674(19)31398-4"><input type="hidden" name="ecommRedirect"><input type="hidden" name="code" value="cell-site"><input type="hidden" name="UX3LoginWidget" value="true"><input type="hidden" name="isPopup" value="true"><div class="input-group login-input--wrapper"><div class="label"><label for="login">Email/Username</label></div><input id="login" type="text" name="login" value="" size="15" class="login"><div role="alert" class="error-hint js--hidden"><div class="error-hint__text"><div class="error-text">Your email address is a required field.</div><div class="email-example">E.g., j.smith@mail.com</div></div></div></div><div class="input-group login-input--wrapper"><div class="label"><label for="password">Password</label></div><input id="password" type="password" name="password" value="" autocomplete="off" class="password"><a href="#" class="login__field__action js-show-password">Show</a><div role="alert" class="error-hint js--hidden"><span class="error-hint__text">Your password is a required field.</span></div><div class="actions"><a href="https://www.cell.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a></div></div><div class="remember"><input type="checkbox" name="savePassword" id="a1f1ba25-79af-4194-aaa9-66206104a800-remember" value="1" checked="checked"><label for="a1f1ba25-79af-4194-aaa9-66206104a800-remember">Remember me</label></div><div class="submit"><input type="submit" name="submit" value="Log in" class="button submit primary"></div></form></div><div class="create-account"><div>Don’t have an account?</div><a href="/action/registration?redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" class="show-registration">Create a Free Account</a></div></div></div></div></div></div>

            

        

    

        


    

            

            

            <div role="dialog" aria-modal="true" aria-label="Reset password" class="popup top-drawer request-reset-password-drawer hidden"><div class="content"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" role="button" class="close"><i class="icon-close_thin"></i></a><div class="request-reset"><div class="form"><p class="sub">If you don't remember your password, you can reset it by entering your email address and clicking the Reset Password button. You will then receive an email that contains a secure link for resetting your password</p><div tabindex="-1" class="message error"></div><form action="/action/requestResetPassword" method="post" class="request-reset-password-form"><input type="hidden" name="platform"><input type="hidden" name="requestResetPassword" value="true"><input type="hidden" name="codeSite" value="cell-site"><div class="input-group"><div class="label"><label for="reset-password-email">Email*</label></div><input id="reset-password-email" type="text" name="email" value="" size="15" class="email"></div><div class="submit" disabled="disabled"><input type="submit" name="submit" value="Submit" disabled="disabled" class="button primary submit"></div></form></div><div class="success-template hidden"><p>If the address matches a valid account an email will be sent to __email__ with instructions for resetting your password</p></div></div><div class="request-reset-cancel"><a href="#" aria-label="Close reset password popup" title="Close reset password popup" class="cancel">Cancel</a></div></div></div>

            

        

    

        


    

            

            
                <div class="ux3-ads">
            

            <div class="pb-ad">
            <!-- Do Not Modify without contacting ELS-HSOnlineCampaign@elsevier.com -->
<!-- To be kept in sync with Global Commercial Publishing Operations-->
<!-- Start: GPT Async -->
<div id="gpt-Variables" data-adfile="CELLJOURNAL">
	<input type="hidden" id="gptSite" value="cell">
	<input type="hidden" id="gptLoggedIn" value="no">
	<input type="hidden" id="gptAccessType" value="ae:ANON_GUEST|ae:ANON_GUEST">
	<input type="hidden" id="gptDefaultSite" value="cell">
</div>
<div id="gpt-Interscroller" class="gpt-InScrollBox" style="display:none;">
	<div class="gpt-InScrollLabel">ADVERTISEMENT</div>
	<div class="gpt-InScrollContainer">
		<div class="gpt-InScrollWrapper">
			<div class="gpt-InScrollInner">
				<div class="gpt-adinterscroller" data-adsize="interscroller"></div>
			</div>
		</div>
	</div>
	<div class="gpt-InScrollLabel">SCROLL TO CONTINUE WITH CONTENT</div>
</div>
<div id="gpt-DebugDiv" class="gpt-DebugDiv" style="display:none;">
	<div style="width: 100%; display: table;">
		<div style="display: table-row">
			<div style="width: 200px; display: table-cell;">
				<button class="gpt-DebugBtn" onclick="gpt_ShowConsole()">Open GPT Console</button><br>
				<button id="gpt_GrapeShotDebugBtn" class="gpt-DebugBtn" onclick="gpt_GrapeShotDebug();">Open Oracle Keywords</button><br>
				<br>
				<button class="gpt-DebugBtn" onclick="gpt_LoadDebugValues()">Refresh Values</button>
			</div>
			<div style="display: table-cell;">
				<table class="gpt-DebugTable">
					<thead>
						<tr>
							<th>Property</th>
							<th>Value</th>
						</tr>
					</thead>
					<tbody>
						<tr><td class="gpt-DebugTitle">Status</td><td><div class="gptStatus_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Version</td><td><div class="gptVersion_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad File</td><td><div class="gptAdFile_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Disable Ads Flag</td><td><div class="gptDisableAds_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Environment</td><td><div class="gptTestENV_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Init</td><td><div class="gptMoatInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Moat Ready</td><td><div class="gptMoatReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Ready</td><td><div class="gptContextualReady_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual URL</td><td><div class="gptContextualURL_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Initial Segments</td><td><div class="gptContextualInit_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Contextual Used Segments</td><td><div class="gptContextualResult_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">AdUnit</td><td><div class="gptSite_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">SubAdUnit</td><td><div class="gptPage_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Custom Targeting</td><td><div class="gptCustomTargeting_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Ad Events</td><td><div class="gptEvent_Debug"></div></td></tr>
						<tr><td class="gpt-DebugTitle">Invalid Ad Sizes</td><td><div class="gptInvalidAdSize_Debug"></div></td></tr>
					</tbody>
				</table>
			</div>
		</div>
	</div>
</div>
<style type="text/css">
	.gpt-DebugBtn {
		display: inline-block;
		text-align: center;
		text-decoration: none;
		margin: 2px 0;
		border: solid 1px #122b40;
		border-radius: 0.4em;
		padding: 0.5em 1em;
		color: #ffffff;
		background-color: #204d74;
		font-weight: normal;
		font-size: 12px;
	}

	.gpt-DebugDiv {
		margin: 5px;
	}

	.gpt-DebugTitle {
		font-weight: bold;
	}

	.gpt-DebugTable {
		border: 1px solid #dddddd;
		border-collapse: collapse;
		margin: 5px;
		font-size: 0.9em;
		font-family: sans-serif;
		min-width: 400px;
		margin-bottom: 25px;
	}

		.gpt-DebugTable td:first-child {
			padding: 5px;
		}

		.gpt-DebugTable thead tr th {
			background-color: #d9edf7;
			color: #ffffff;
			text-align: left;
			padding: 5px;
			border-bottom: 2px solid #204d74;
		}

		.gpt-DebugTable th,
		.gpt-DebugTable td {
			padding: 5px;
		}

		.gpt-DebugTable tbody tr {
			border-bottom: 1px solid #dddddd;
		}

			.gpt-DebugTable tbody tr:nth-of-type(even) {
				background-color: #f3f3f3;
			}

			.gpt-DebugTable tbody tr:last-of-type {
				border-bottom: 2px solid #204d74;
			}

	.gpt-InScrollLabel {
		background-color: rgb(0, 0, 0);
		color: rgb(255, 255, 255);
		position: relative;
		max-width: 100vw;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		padding: 0;
		text-align: center;
		width: 100vw !important;
		font-size: 10px;
	}

	.gpt-InScrollBox {
		height: 100vh !important;
		width: 100vw;
		cursor: pointer;
		max-width: 100vw;
		position: relative;
		left: 50%;
		right: 50%;
		margin-left: -50vw;
		margin-right: -50vw;
		margin-bottom: 50px;
		padding-top: 5px;
		padding-bottom: 5px;
		z-index: 10001;
		background-color: white;
	}

		.gpt-InScrollBox .gpt-InScrollContainer {
			height: 95vh !important;
			position: relative;
		}

		.gpt-InScrollBox .gpt-InScrollWrapper {
			position: absolute !important;
			top: 0 !important;
			left: 0;
			width: 100%;
			height: 95vh !important;
			border: 0 !important;
			margin: 0 !important;
			padding: 0 !important;
			clip: rect(0, auto, auto, 0) !important;
			-webkit-clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
			clip-path: polygon(0px 0px, 100% 0px, 100% 100%, 0px 100%) !important;
		}

		.gpt-InScrollBox .gpt-InScrollInner {
			position: fixed !important;
			top: 0 !important;
			left: 0 !important;
			bottom: 0;
			margin: auto;
			width: 100vw;
			height: 95vh;
			-webkit-transform: translateZ(0) !important;
			display: -webkit-box;
			display: -ms-flexbox;
			display: flex;
			-webkit-box-orient: vertical;
			-webkit-box-direction: normal;
			-ms-flex-direction: column;
			flex-direction: column;
			-webkit-box-align: center;
			-ms-flex-align: center;
			align-items: center;
			-webkit-box-pack: center;
			-ms-flex-pack: center;
			justify-content: center;
			text-align: center;
		}
</style>
<script class="optanon-category-4" type="text/plain">
	//--------------------------------------------------------------------------
	var gpt_Version = '04072023';
	var gpt_AdFile = $('#gpt-Variables').data('adfile');
	var gpt_MaxMOATInitAttempts = 11;
	var gpt_MaxMOATReadyAttempts = 11;
	var gpt_MaxContextualReadyAttempts = 21;
	var gpt_MOATInitAttempts = 0;
	var gpt_MOATReadyAttempts = 0;
	var gpt_ContextualReadyAttempts = 0;
	var gpt_IsProd = true;
	var gpt_StartTime;
	var gs_channels = 'DEFAULT';
	var gpt_ContextualReady = false;
	var gpt_ContextualURL = 'Not enabled outside of Production';
	var gpt_DisableAdsFlagFound = false;

	var gptEvent_Debug = '';
	var gptCustomTargeting_Debug = '';
	var gptSite_Debug = "";
	var gptPage_Debug = "";
	var gptContextualEnabled = false;
	var gptContextualResult_Debug = '';
	var gptContextualInit_Debug = '';
	var gptContextualURL_Debug = '';
	var gptContextualReady_Debug = '';
	var gptMoatReady_Debug = '';
	var gptMoatInit_Debug = '';
	var gptTestENV_Debug = '';
	var gptDisableAds_Debug = '';
	var gptStatus_Debug = 'GPT Still Loading - Refresh Values';
	var gptInvalidAdSize_Debug = '';
	//--------------------------------------------------------------------------
	$(document).ready(function () {
		//Check to see if we are on a development env
		gpt_CheckEnvironment();

		if (!gpt_DisableAdsFlagFound) {
			gpt_StartTime = new Date();
			if (gpt_AdFile !== "PU") {
				//Once the dom has loaded, begin the process to init the ads.
				gpt_InitScripts();
				gpt_CheckForMOATInit();
			}
		}
	});
</script>
<script type="text/javascript">
	function gpt_CheckEnvironment() {
		if ($('#gptDisableAds').length > 0) {
			gptDisableAds_Debug = 'Flag Detected - Ads Disabled';
			gpt_DisableAdsFlagFound = true;
		}
		else {
			gptDisableAds_Debug = 'Ads Enabled';
			gpt_DisableAdsFlagFound = false;
		}
		switch (gpt_AdFile) {
			case 'PU':
				gptContextualEnabled = true;
				if ($("#Environment").val() !== "Prod") {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			case 'JBS':
			case 'LANCET':
				gptContextualEnabled = true;
				if ($(".gpt-ad").length > 0) {
					//Other ads exist, allow the Prestitial
					$(document.body).append("<div class='gpt-ad' data-adsize='prestitial'></div>");
				}
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
			default:
				//Cell
				gptContextualEnabled = false;
				if (window.location.hostname.toLowerCase().indexOf('.literatumonline.') >= 0) {
					gpt_IsProd = false;
					gptTestENV_Debug = 'Test Environment Detected';
				}
				else {
					gptTestENV_Debug = 'Production Environment Detected';
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
    function gpt_GetNonCMEArticle() {
        var url;
        $('.articleCitation').each(function (i, obj) {
            if ($(obj).find('.toc__item__detials').length > 0) {
                var itemDetails = $(obj).first().find('.toc__item__detials');
                var cmeFound = 0;
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='CME']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='(CME)']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='Education']").length; }
                if (cmeFound == 0) { cmeFound = $(itemDetails).find("img[title~='education']").length; }
                if (cmeFound == 0) {
                    var itemLink = $(itemDetails).first().find('.toc__item__title');
                    var linkCount = $(itemLink).first().find("a:contains('CME')").length;
                    if (linkCount == 0) {
                        url = $(itemLink).first().find('a').attr('href');
                        return false;   //Found a URL, break the loop
                    }
                }
            }
        });
        return url;
    }
	//--------------------------------------------------------------------------
	function gpt_GetElapsedTime() {
		var endTime = new Date();
		var timeDiff = endTime - gpt_StartTime; //in ms
		// strip the ms
		timeDiff /= 1000;

		// get seconds
		return seconds = Math.round(timeDiff) + " seconds";
	}
	//--------------------------------------------------------------------------
	function gpt_CheckSSC() {
		//Check for ScreenShot Client
		var gpt_SSC = gpt_GetQuerystring('gpt_ssc', '');
		if (gpt_SSC == '1') {
			//it's the ScreenShotClient, disable MOAT IVT Detection
			$('#gpt-Variables').append('<input type="hidden" id="gptm_data">');
			$('#gptm_data').val(0);
			gpt_Target("m_data");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_LocalSS(adTypeId) {
		window.scrollTo(0, 0);
		gpt_Sleep(1000);
		gpt_ScrollToAd(adTypeId);
		gpt_PreTearsheet(adTypeId);
	}
	//--------------------------------------------------------------------------
	function gpt_InitScripts() {
		if (gpt_AdFile == "PU") {
			if (gpt_IsProd) {
				gpt_ContextualURL = gpt_CheckForNULL($('#canonicalurl').val());
				if (gpt_ContextualURL === null) {
					gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				}
				else {
					gpt_ContextualURL = window.location.origin + gpt_ContextualURL;
				}
				var useSSL = 'https:' == document.location.protocol;
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}
		}
		else {
			//JBS-Cell-Lancet
			var useSSL = 'https:' == document.location.protocol;

			var gads = document.createElement('script');
			gads.async = true;
			gads.type = 'text/javascript';
			gads.src = (useSSL ? 'https:' : 'http:') + '//securepubads.g.doubleclick.net/tag/js/gpt.js';
			document.head.appendChild(gads);

			if (gpt_IsProd && gptContextualEnabled) {
				gpt_ContextualURL = window.location.href.split('?')[0].split('#')[0];
				var gs = document.createElement('script');
				gs.async = true;
				gs.type = 'text/javascript';
				gs.src = (useSSL ? 'https:' : 'http:') + '//elsevier.gscontxt.net/multizone/channels.cgi?url=' + encodeURIComponent(gpt_ContextualURL);
				gs.onload = function () {
					gpt_ContextualReady = true;
				};
				document.head.appendChild(gs);
			}

			var elsevierMoatHeader = document.createElement('link');
			elsevierMoatHeader.rel = 'preload';
			elsevierMoatHeader.as = 'script';
			elsevierMoatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(elsevierMoatHeader);

			var elsevierMoatAds = document.createElement('link');
			elsevierMoatAds.rel = 'preconnect';
			elsevierMoatAds.src = (useSSL ? 'https:' : 'http:') + '//mb.moatads.com';
			document.head.appendChild(elsevierMoatAds);

			var moatHeader = document.createElement('script');
			moatHeader.async = true; gads.type = 'text/javascript';
			moatHeader.src = (useSSL ? 'https:' : 'http:') + '//z.moatads.com/elsevierheader150204004183/moatheader.js';
			document.head.appendChild(moatHeader);
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForMOATInit() {
		//Check every 1/10 of a second until MOAT is Loaded or 1 second timeout
		if (gpt_MOATInitAttempts < gpt_MaxMOATInitAttempts) {
			gpt_MOATInitAttempts += 1;
			if (typeof window.moatPrebidApi === "object") {
				//MOAT LOADED
				gptMoatInit_Debug = '0.0' + gpt_MOATInitAttempts + ' seconds';
				gpt_CheckForMOATReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATInit(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not LOADED
			gptMoatInit_Debug = 'Timeout Expired';
			var googletag = googletag || {};
			googletag.cmd = googletag.cmd || [];
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForMOATReady() {
		//Check every 1/10 of a second until MOAT is Ready or 1 second timeout
		if (gpt_MOATReadyAttempts < gpt_MaxMOATReadyAttempts) {
			gpt_MOATReadyAttempts += 1;
			if (window.moatPrebidApi.safetyDataAvailable()) {
				//MOAT Ready
				gptMoatReady_Debug = '0.0' + gpt_MOATReadyAttempts + " seconds";
				gpt_CheckForContextualReady();
			}
			else {
				setTimeout(() => { gpt_CheckForMOATReady(); }, 100);
			}
		}
		else {
			//TIMEOUT: MOAT Not Ready
			gptMoatReady_Debug = 'Timeout Expired';
			gpt_CheckForContextualReady();
		}
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForContextualReady() {
		//Check every 1/10 of a second until Contextual is Ready or 2 second timeout
		if (gpt_IsProd && gptContextualEnabled) {
			//Enabled
			if (gpt_ContextualReadyAttempts < gpt_MaxContextualReadyAttempts) {
				gpt_ContextualReadyAttempts += 1;
				if (gpt_ContextualReady) {
					//Contextual Ready
					gptContextualURL_Debug = gpt_ContextualURL;
					gptContextualReady_Debug = '0.0' + gpt_ContextualReadyAttempts + " seconds";
					gptContextualInit_Debug = gs_channels;
					gpt_InitAds();
				}
				else {
					setTimeout(() => { gpt_CheckForContextualReady(); }, 100);
				}
			}
			else {
				//LOG TIMEOUT error
				//REMOVE FOR NOW (SHOULD LOG TO A DIFF LOCATION)
				//TIMEOUT: Contextual Not Ready
				$('#gpt_GrapeShotDebugBtn').remove();
				gptContextualURL_Debug = gpt_ContextualURL;
				gptContextualReady_Debug = 'Timeout Expired';
				gptContextualInit_Debug = "";
				gpt_InitAds();
			}
		}
		else {
			//Not Enabled
			$('#gpt_GrapeShotDebugBtn').remove();
			switch (gpt_AdFile) {
				case 'PU':
				case 'LANCET':
				case 'JBS':
					gptContextualURL_Debug = 'Not enabled outside of Production';
					gptContextualReady_Debug = 'Not enabled outside of Production';
					gptContextualInit_Debug = 'Not enabled outside of Production';
					break;
				case "CELLJOURNAL":
				case "CELLBUCKET":
					gptContextualReady_Debug = 'Not enabled on Cell';
					gptContextualInit_Debug = 'Not enabled on Cell';
					gptContextualURL_Debug = 'Not enabled on Cell';
					break;
			}
			gpt_InitAds();
		}
	}
	//--------------------------------------------------------------------------
	function GPT_GetPageByURL() {
		var host = window.location.host.toLocaleLowerCase()
		var localpath = window.location.pathname.toLocaleLowerCase();

		///Check for Cell.com
		if (host.indexOf('cell.com') >= 0) {
			return null;
		}

		///Check for Cell.com (marlin-stag)
		if (host.indexOf('www-cell-com') >= 0) {
			return null;
		}

		///Check for Lancet
		if (host.indexOf('thelancet') >= 0) {
			return null;
		}

		if (localpath.indexOf('/article/') >= 0) {
			return 'article.fulltext';
		}

		if (localpath.indexOf('/action/dosearch') >= 0) {
			return 'search.searchResults';
		}

		if (localpath.indexOf('/content/') >= 0) {
			return 'periodical.editorial';
		}

		return 'periodical.home';
	}
	//--------------------------------------------------------------------------
	function gpt_CheckForNULL(x, valueifnull) {
		if (typeof valueifnull == "undefined") {
			valueifnull = null;
		}

		if (typeof x == 'undefined') {
			return valueifnull;
		}
		if (x == null) {
			return valueifnull;
		}
		if (x == "null") {
			return valueifnull;
		}
		if ($.trim(x) == '') {
			return valueifnull;
		}

		return x;
	}
	//--------------------------------------------------------------------------
	function gpt_GetAdditionalTargeting() {
		//adds page level targeting of custom variables
		//the id (minus gpt) and the value will be used as a key-value pair
		//eg: <input type="hidden" id="gptissuepii" value="x" class="gpt-Target">
		//eg: target issuepii=x
		$(".gpt-Target").each(function (index) {
			var gptTargetId = $(this).attr('id');
			gptTargetId = gptTargetId.replace('gpt', '').toLowerCase();
			if (gptTargetId !== '') {
				gpt_Target(gptTargetId);
			}
		})
	}
	//--------------------------------------------------------------------------
	function gpt_GetQuerystring(key, default_) {
		//function to read querystring variables
		if (default_ === null) default_ = '';
		key = key.replace(/[\[]/, "\\\[").replace(/[\]]/, "\\\]");
		var regex = new RegExp("[\\?&]" + key + "=([^&#]*)");
		var qs = regex.exec(window.location.href);
		if (qs === null)
			return default_;
		else
			return qs[1];
	}
	//--------------------------------------------------------------------------
	function gpt_GetDFPTestId() {
		//Get any request for a test id from querystring and set custom targeting parameter
		var gptDFPTestId = gpt_GetQuerystring('dfptestid', '');
		if (gptDFPTestId !== '') {
			$('#gpt-Variables').append('<input type="hidden" id="gptDFPTestId">');
			$('#gptDFPTestId').val(gptDFPTestId);
			gpt_Target("DFPTestId");
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Target(targetName) {
		//Set page level target if value is found
		if ($('#gpt' + targetName).length > 0) {
			var targetValue = $('#gpt' + targetName).val();

			if (targetValue !== '') {
				targetValue = targetValue.replace(/[^-a-z0-9]/ig, '');	//Only Allow a-z, 0-9
				googletag.pubads().setTargeting(targetName.toLowerCase(), targetValue.toLowerCase());
				gptCustomTargeting_Debug = gptCustomTargeting_Debug + targetName.toLowerCase() + ' = ' + targetValue.toLowerCase() + '<br>';
			}
		}
	}
	//--------------------------------------------------------------------------
	function gpt_TargetAccessType() {
		var currentAccessType = $('#gptAccessType').val();
		if (typeof currentAccessType == "undefined") {
			currentAccessType = "";
		}
		if (currentAccessType.indexOf('ANON_IP') >= 0) {
			googletag.pubads().setTargeting("loggedin", "yes");
		}
		else {
			gpt_Target("LoggedIn");
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_PreTearsheet(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		//Remove Elsevier Cookie Policy Notice
		if ($("#onetrust-consent-sdk").length > 0) {
			$("#onetrust-consent-sdk").remove();
		}

        //Mobile Nav
        $('#skip').remove();
        $('skipNavigationLink').remove();

		//JBS Modals
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.leadinModal').length > 0) {
			$('.leadinModal').remove();
		}
		if ($('.usabilla__overlay').length > 0) {
			$('.usabilla__overlay').remove();
		}

		//Pendo
		if ($('._pendo-step-container').length > 0) {
			$('._pendo-step-container').remove();
		}
		if ($('._pendo-guide-tt_').length > 0) {
			$('._pendo-guide-tt_').remove();
		}
		if ($('._pendo-image').length > 0) {
			$('._pendo-image').remove();
		}
		if ($('._pendo-badge-image').length > 0) {
			$('._pendo-badge-image').remove();
		}

		switch (gpt_AdFile) {
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Mobile Bug Fix on Cell
				if (parseInt(adTypeId) == 5) {
					$('.widget-horizontalAd').css('position', 'static');
					$('.body-horizontalAd').css('display', 'block');
				}
				break;
			case "JBS":
			case "LANCET":
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 3: //MPU
					case 4: //SkyScraper
						//BUGFIX: Show Sidebar ad for JBS
						//skyscraper & mpu
						if ($('.widget-verticalAd').length > 0) {
							$('.widget-verticalAd').css('display', 'block');
						}
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
					case 6:     //Prestitial
						$('#backgroundLayer').css('display', 'block');
						break;
					default:
						$('#backgroundLayer').css('display', 'none');   //Hide any Prestitial
						break;
				}
				break;
		}
	}
	//--------------------------------------------------------------------------
	function gpt_Sleep(ms) {
		return new Promise(resolve => setTimeout(resolve, ms));
	}
	//--------------------------------------------------------------------------
	async function gpt_ScrollToAd(adTypeId) {
		if (typeof adTypeId === "undefined") {
			adTypeId = 0;
		}
		var adId = "";
		var offset = 0;
		var objLeaderboard = "";
		var gpt_ScrollToAd = true;

		switch (gpt_AdFile) {
			case "PU": //PracticeUpdate
				var hideOtherAds = true;
				$('#modalPrestitial').modal('hide');
				$('.jumbotron-welcome').hide();
				objLeaderboard = ".container-leaderboard";
				offset = 53;
				switch (parseInt(adTypeId)) {
					case 2:
						//LeaderBoard, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 3:
						//MPU
						adId = "adSlot2";
						break;
					case 4:
						//SkyScraper
						adId = "adSlot3";
						$('.most-read').hide();
						break;
					case 5:
						//Mobile, no Scroll Needed
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
					case 6:
						//Interstitial
						gpt_ScrollToAd = false;
						adId = "adSlot15";
						await gpt_Sleep(1000);
						$(objLeaderboard).hide();
						$('#modalPrestitial').unbind();
						$('#modalPrestitial').modal();
						break;
					case 7:
						//Native Ad
						break;
					case 10:
						//Master Companion
						hideOtherAds = false;
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						adId = "adSlot1";
						break;
				}
				if (hideOtherAds) {
					//Hide all other ads
					$('.ad-container').each(function (i, obj) {
						if (this.id !== adId) {
							var tempParent = this.id;
							tempParent = tempParent.replace("adSlot", "adParent");
							$("#" + tempParent).addClass('hide');
						}
					});
				}
				break;
			case "JBS":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$(".widget-verticalAd").hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="sidebar"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					case 13:	    //Phone - Portrait
					case 14:        //Phone - Landscape
						$(objLeaderboard).hide();
						offset = 60;
						adId = 'gpt-Interscroller';
						gpt_ScrollToAd = true;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
			case "CELLJOURNAL":
			case "CELLBUCKET":
				objLeaderboard = ".js__adToHide";
				offset = 260;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="boombox"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				//Hide the second ad if found
				var tCount = 0;
				$(".widget.literatumAd.none.GPT-ad-placeholder.widget-horizontalAd.widget-compact-all").each(function (index) {
					tCount += 1;
					if (tCount !== 1) {
						$(this).hide();
					}
				});
				break;
			case "LANCET":
				objLeaderboard = ".js__adToHide";
				offset = 80;
				switch (parseInt(adTypeId)) {
					case 2:
						//Leaderboard
						gpt_ScrollToAd = false;
						$('.body-verticalAd').hide();
						break;
					case 5:
						//Mobile
						//Scroll to Top
						gpt_ScrollToAd = false;
						break;
					case 3: //MPU
					case 4: //SkyScraper
						$(objLeaderboard).hide();
						adId = $('.gpt-ad-defined[data-adsize="skyscraper"').attr('id');
						break;
					case 10:
						//MasterCompanion
						gpt_ScrollToAd = false;
						break;
					default:
						//Unknown Ad Type
						gpt_ScrollToAd = false;
						break;
				}
				break;
		}

		//Scroll to Top
		var gpt_container = $("html,body");
		gpt_container.animate({ scrollTop: 0, scrollLeft: 0 });

		if (gpt_ScrollToAd) {
			//If Adexists, scroll to it
			var gpt_scrollTo = $('#' + adId);
			if (gpt_scrollTo.length > 0) {
				//Scroll to the Ad
				var gpt_topPosition = gpt_scrollTo.offset().top - gpt_container.offset().top + gpt_container.scrollTop();
				if (gpt_topPosition > 0) {
					gpt_container.animate({ scrollTop: gpt_topPosition, scrollLeft: 0 });
					if (gpt_topPosition - offset > 0 && offset > 0) {
						gpt_container.animate({ scrollTop: gpt_topPosition - offset, scrollLeft: 0 });
					}
				}
			}
		}
	}
	//--------------------------------------------------------------------------
	async function gpt_CheckForCookiePolicy() {
		var returnVal = false;
		if ($('#onetrust-accept-btn-handler').length > 0) {
			$('#onetrust-accept-btn-handler').click();
			returnVal = true;
		}
		return returnVal;
	}
	//--------------------------------------------------------------------------
	function gpt_InitAds() {
		var gpt_ResponsiveHomepage = false;

		switch (gpt_AdFile) {
			case "JBS":
			case "CELLJOURNAL":
			case "CELLBUCKET":
				//Responsive Check
				if ($('#hdrHomeBrand').length > 0) {
					gpt_ResponsiveHomepage = true;
				}
				break;
		}

		//InterScroller Check
		if (gpt_AdFile === "JBS") {
			var gpt_InterScrollerEnabled = false;
			var gpt_InterScrollerInsertObj;

			if ($("#articleHeader").length > 0) {
				//It's an article Page
				if ($('.article__sections > section').length > 0) {
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".article__sections > section");
				}
			}
			else {
				if ($('.toc__section').length > 0) {
					//It's a TOC Page
					gpt_InterScrollerEnabled = true;
					gpt_InterScrollerInsertObj = $(".toc__section").first();
				}
			}

			//Add the Interscroller
			if (gpt_InterScrollerEnabled) {
				gpt_InterScrollerInsertObj.first().append($("#gpt-Interscroller").clone());
				$("#gpt-Interscroller").remove();
				$(".gpt-adinterscroller").addClass("gpt-ad");
			}
		}

		googletag.cmd.push(function () {
			var gptBoomBoxSizeMapping;
			var gptFluidMapping;
			var gptHomePageSidebarMapping;
			var gptCellHomePageSidebarMapping;
			var gptInterscrollerMapping;
			var gptLeaderboardSizeMapping;
			var gptMobileSizeMapping;
			var gptPrestitialMapping;
			var gptRectangleShortSizeMapping;
			var gptRectangleSizeMapping;
			var gptSideBarSizeMapping;
			var gptTallSizeMapping;
			var gptWideSizeMapping;
			var gptWideRectangleSizeMapping;

			//***********************************************
			//Start Define Size mapping based on client Viewport
			//***********************************************
			if (gpt_AdFile === 'PU') {
				gptTallSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 600], [160, 600], [120, 600]]).
					addSize([0, 0], []).
					build();

				gptRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250]]).
					addSize([0, 0], []).
					build();

				gptRectangleShortSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[300, 250], [320, 50], [300, 50]]).
					addSize([0, 0], [[300, 250], [320, 50], [300, 50]]).
					build();

				gptWideSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([760, 0], [[728, 90], [320, 50], [300, 50]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptWideRectangleSizeMapping = googletag.sizeMapping().
					addSize([992, 0], []).
					addSize([760, 0], [[320, 50], [300, 50], [300, 250]]).
					addSize([0, 0], [[320, 50], [300, 50], [300, 250]]).
					build();

				gptMobileSizeMapping = googletag.sizeMapping().
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([760, 0], [[1, 1]]).
					build();
			}
			else {
				//JBS-CELL-LANCET
				gptLeaderboardSizeMapping = googletag.sizeMapping().
					addSize([992, 0], [[728, 90]]).
					addSize([768, 0], [[728, 90]]).
					addSize([0, 0], [[320, 50], [300, 50]]).
					build();

				gptHomePageSidebarMapping = googletag.sizeMapping().
					addSize([2090, 795], [[300, 600], [300, 250], [160, 600], [120, 600]]).
					addSize([1910, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptCellHomePageSidebarMapping = googletag.sizeMapping().
                    addSize([2180, 795], [[300, 600], [160, 600], [120, 600], [300, 250], [336, 280]]).
					addSize([2180, 445], [[336, 280]]).
                    addSize([2090, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
                    addSize([2090, 445], [[300, 250]]).
					addSize([1830, 795], [[160, 600], [120, 600]]).
                    addSize([1740, 795], [[120, 600]]).
					addSize([0, 0], []).
					build();

				gptSideBarSizeMapping = googletag.sizeMapping().
					addSize([1140, 795], [[300, 600], [160, 600], [120, 600], [300, 250]]).
					addSize([1140, 445], [[300, 250]]).
					addSize([0, 0], []). 				
					build();

				gptBoomBoxSizeMapping = googletag.sizeMapping().
					addSize([768, 0], [[300, 250], [336, 280]]).
					addSize([0, 0], []). 						
					build();

				gptFluidMapping = googletag.sizeMapping().
					addSize([0, 0], ['fluid']).
					build();

				gptPrestitialMapping = googletag.sizeMapping().
					addSize([600, 700], [[1, 1]]).
					addSize([0, 0], []).
					build();

				gptInterscrollerMapping = googletag.sizeMapping().
					addSize([1140, 445], []).
					addSize([1024, 768], [[1024, 768]]).
					addSize([768, 1024], [[768, 1024]]).
					addSize([480, 320], [[480, 320]]).
					addSize([320, 480], [[320, 480]]).
					build();
			}
			//***********************************************
			//End Define Size mapping based on client Viewport
			//***********************************************

			//***********************************************
			//Start detection for ad slots on page
			//***********************************************
			var gptMasterPage = gpt_CheckForNULL($('#gptPage').val());
			var gptMasterSite;

			if (gpt_AdFile === 'PU') {
				gptMasterSite = $('#gptSite').val();
			}
			else {
				//JBS-CELL-LANCET
				gptMasterSite = "dev";

				if (gpt_IsProd) {
					gptMasterSite = gpt_CheckForNULL($('#gptSite').val());

					//If there is no master site, check for a defaultsite
					if (gptMasterSite == null) {
						gptMasterSite = gpt_CheckForNULL($('#gptDefaultSite').val());
					}
				}
				else {
					//Dev ENV
					//If the page has no gptSite Value, assume no value which will disable ads
					if (gpt_CheckForNULL($('#gptSite').val()) === null) {
						gptMasterSite = null;
					}
				}

				//If there is no master Page, check for a Page using the URL
				if (gptMasterPage === null) {
					gptMasterPage = GPT_GetPageByURL();
				}
			}

			$(".gpt-ad").each(function (index) {
				var gptPage = '';
				var adSizeMapping = '';
				var gptPOS;
				var gptTC;
				var gptNative;
				var adId;
				var gptSite;
				var tempSlot = null;


                if (gpt_IsProd) {
					if (gpt_AdFile !== "CELLJOURNAL") {
						//Check if the ad slot has it's own site defined
						if (typeof $(this).data('site') === 'undefined') {
							gptSite = gptMasterSite;
						}
						else {
							gptSite = gpt_CheckForNULL($(this).data('site'));
						}
					}
					else {
						gptSite = gptMasterSite;
					}
                }
                else {
					gptSite = gptMasterSite;
                }



				if (gpt_AdFile === 'PU') {
					adId = $(this).attr('id');
				}
				else {
					//JBS-CELL-LANCET
					adId = 'gpt-ad-' + index + 1;
				}

				if (gpt_CheckForNULL(gptSite) !== null) {
					gptSite_Debug = gptSite;

					gptPOS = parseInt($(this).data('pos')) || -1;
					if (gptPOS === -1) {
						gptPOS = index + 1;
					}

					//Check if the ad slot has it's own page defined
					gptPage = $(this).data('page');
					if (gpt_CheckForNULL(gptPage) == null) {
						//Use the Master Page
						gptPage = gptMasterPage;
					}

					//If the page is not null, preceed it with a slash
					if (gptPage !== null) {
						gptPage = '/' + gptPage;
					}
					else {
						gptPage = '';
					}
					gptPage_Debug = gptPage;


					gptTC = $(this).data('tc');
					if (typeof gptTC === "undefined") {
						gptTC = '';
					}

					gptNative = $(this).data('native');
					if (typeof gptNative === "undefined") {
						gptNative = '';
					}

					//Determine sizemapping based on data-adsize
					if (gpt_AdFile === 'PU') {
						switch ($(this).data('adsize')) {
							case 'tall':
								adSizeMapping = gptTallSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600]], adId);
								break;
							case 'rectangle':
								adSizeMapping = gptRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								break;
							case 'rectangleshort':
								adSizeMapping = gptRectangleShortSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [320, 50], [300, 50]], adId);
								break;
							case 'wide':
								adSizeMapping = gptWideSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'widerectangle':
								adSizeMapping = gptWideRectangleSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50], [300, 250]], adId);
								break;
							case 'mobile':
								adSizeMapping = gptMobileSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 50], [300, 50]], adId);
								break;
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[1, 1]], adId);
								break;
						}
					}
					else {
						//JBS-CELL-LANCET
						switch ($(this).data('adsize')) {
							case 'cell-leaderboard':
							case 'leaderboard':
								adSizeMapping = gptLeaderboardSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[728, 90], [320, 50], [300, 50]], adId);
								break;
							case 'bottom':
								if (gpt_ResponsiveHomepage) {
									adSizeMapping = gptHomePageSidebarMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'boombox':
								if (gpt_AdFile === "CELLJOURNAL" || gpt_AdFile == "CELLBUCKET") {
									if (gpt_ResponsiveHomepage) {
										adSizeMapping = gptCellHomePageSidebarMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250], [336, 280]], adId);
									}
									else {
										adSizeMapping = gptBoomBoxSizeMapping;
										tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250], [336, 280]], adId);
									}
								}
								else {
									adSizeMapping = gptBoomBoxSizeMapping;
									tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[300, 250]], adId);
								}
								break;
							case 'interscroller':
								adSizeMapping = gptInterscrollerMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[320, 480], [480, 320], [768, 1024], [1024, 768]], adId);
								break;
							case 'sidebar':
							case 'skyscraper':
								adSizeMapping = gptSideBarSizeMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, [[120, 600], [160, 600], [300, 600], [300, 250]], adId);
								break;
							case 'native':
							case 'fluid':
								adSizeMapping = gptFluidMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
							case 'prestitial':
								adSizeMapping = gptPrestitialMapping;
								tempSlot = googletag.defineSlot('/6053/els.' + gptSite + gptPage, 'fluid', adId);
								break;
						}
					}
				}

				//Ignore any invalid adsizes that could be on the page
				if (adSizeMapping !== '') {
					$(this).addClass('gpt-ad-defined');
					if (gpt_AdFile !== 'PU') {
						$(this).attr('id', adId);
					}
					tempSlot.defineSizeMapping(adSizeMapping).addService(googletag.pubads()).setTargeting("pos", gptPOS);

					if (gptTC !== '') {
						tempSlot.setTargeting("tc", gptTC);
					}
					if (gptNative !== '') {
						tempSlot.setTargeting("native", gptNative);
					}
					gptEvent_Debug = gptEvent_Debug + $(this).data('adsize') + ' Requested<br>';
				}
				else {
					gptInvalidAdSize_Debug = gptInvalidAdSize_Debug + "Invalid Size Found, Id: " + $(this).prop('id') + ', adSize: ' + $(this).data('adsize') + '<br>';
					$(this).remove();
				}
			});
			if (gpt_CheckForNULL(gptInvalidAdSize_Debug) === null) {
				gptInvalidAdSize_Debug = "None Detected";
			}
			//***********************************************
			//End detection for ad slots on page
			//***********************************************

			//***********************************************
			//Start page level targeting
			//***********************************************
			if (gpt_AdFile !== 'PU') {
				//Logged in or IP Access
				gpt_TargetAccessType();

				//Check for Screenshot Client
				gpt_CheckSSC();
			}

			//Check for DFP Test
			gpt_GetDFPTestId();

			//Check for Additional Targeting
			gpt_GetAdditionalTargeting();

			if (gpt_IsProd && gptContextualEnabled) {
				var gpt_ContextualRetry = false;
				//Check for Contextual Targeting
				if (gs_channels !== "DEFAULT") {
					var gpt_ContextualCategories = Array.from(gs_channels);
					for (var i = 0; i < gpt_ContextualCategories.length; i++) {
						if (gpt_ContextualCategories[i] == 'gx_retry') {
							gpt_ContextualRetry = true;
						}
						if (!gpt_ContextualCategories[i].startsWith("custom_")) {
							//Remove any tag that is not custom
							gpt_ContextualCategories.splice(i, 1);
							i--;
						}
						else {
							//Strip the 'custom_' from the tag
							gpt_ContextualCategories[i] = gpt_ContextualCategories[i].replace("custom_", "")
						}
					}
					if (gpt_ContextualCategories.length > 0) {
						googletag.pubads().setTargeting("gs_cat", gpt_ContextualCategories);
					}
					else {
						gpt_ContextualCategories = 'none';
					}
					if (gpt_ContextualRetry) {
						gpt_ContextualCategories = "*Indexing*";
					}
					gptContextualResult_Debug = gpt_ContextualCategories;
				}
				else {
					gptContextualResult_Debug = 'No segments found';
				}
			}
			else {
				//Not Prod || Contextual not enabled
				gptContextualResult_Debug = 'Not enabled';
				
				if (!gpt_IsProd) {
                    var gpt_DevSite = gpt_CheckForNULL($('#gptSite').val(), "dev");
                    $('#gpt-Variables').append('<input type="hidden" id="gptEnvironment" value="' + gpt_DevSite + '">');
                    gpt_Target("Environment");
                }

			}
			//***********************************************
			//End page level targeting
			//***********************************************

			//Enable google dfp services
			if (parseInt($('#gptLazyLoading').val()) === 1) {
				googletag.pubads().enableLazyLoad({
					fetchMarginPercent: 200,
					renderMarginPercent: 0,
					mobileScaling: 2.0
				});
			}
			else {
				googletag.pubads().enableSingleRequest();
			}

			//Add Listener for RenderEnded
			googletag.pubads().addEventListener('slotRenderEnded', function (event) {
				if (!event.isEmpty) {
					if (typeof LocalAdLoaded === "function") {
						LocalAdLoaded(event);
					}
					if (gpt_ResponsiveHomepage) {
						if (typeof positionAd === "function") {
							positionAd();
						}
					}
					if (event.size[0] == "480" || event.size[1] == "480") {
						$('.gpt-InScrollBox').css('display', 'block');
						gptEvent_Debug = gptEvent_Debug + 'Interscroller ' + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}
					else {
						gptEvent_Debug = gptEvent_Debug + event.size + ' rendered: ' + gpt_GetElapsedTime() + '<br>';
					}

				}
			});

			if (gpt_AdFile !== 'PU') {
				googletag.pubads().collapseEmptyDivs();
			}
			googletag.enableServices();

			//Populate Defined Ad Slots
			$(".gpt-ad-defined").each(function (index) {
				var adId = $(this).attr('id');
				googletag.cmd.push(function () { googletag.display(adId); });
			});

			gptStatus_Debug = 'GPT Complete';
		});
	}
	//--------------------------------------------------------------------------
	function gpt_DisplayDebug() {
		//Clone the template and append to Body
		$(document.body).append($('#gpt-DebugDiv').clone());
		$('#gpt-DebugDiv').remove();

		//Populate the Values
		gpt_LoadDebugValues();

		//Display and scroll to debug section
		$('.gpt-DebugDiv').css('display', 'block');
		window.scrollTo(0, document.body.scrollHeight);
	}
	//--------------------------------------------------------------------------
	function gpt_LoadDebugValues() {
		$('.gpt-DebugBtn').blur();
		$('.gptVersion_Debug').text(gpt_Version);
		$('.gptAdFile_Debug').html(gpt_AdFile);
		$('.gptEvent_Debug').html(gptEvent_Debug);
		$('.gptCustomTargeting_Debug').html(gptCustomTargeting_Debug);
		$('.gptSite_Debug').text(gptSite_Debug);
		$('.gptContextualResult_Debug').text(gptContextualResult_Debug);
		$('.gptContextualInit_Debug').text(gptContextualInit_Debug);
		$('.gptContextualURL_Debug').text(gptContextualURL_Debug);
		$('.gptContextualReady_Debug').text(gptContextualReady_Debug);
		$('.gptMoatReady_Debug').text(gptMoatReady_Debug);
		$('.gptMoatInit_Debug').text(gptMoatInit_Debug);
		$('.gptTestENV_Debug').text(gptTestENV_Debug);
		$('.gptDisableAds_Debug').text(gptDisableAds_Debug);
		$('.gptStatus_Debug').text(gptStatus_Debug);
		$('.gptPage_Debug').text(gptPage_Debug);
		$('.gptInvalidAdSize_Debug').html(gptInvalidAdSize_Debug);
	}
	//--------------------------------------------------------------------------
	function gpt_GrapeShotDebug() {
		$('.gpt-DebugBtn').blur();
		window.open("http://admin-elsevier.grapeshot.co.uk/custom/bookmarklet.cgi?blocking=1&vn=2&url=" + encodeURIComponent(document.location), "GRAPESHOT_POPUP", "width=650,height=600,scrollbars=yes,screenX=40,screenY=50,menubar=no,toolbar=no,resizable=yes", "Custom").focus();
	}
	//--------------------------------------------------------------------------
	function gpt_ShowConsole() {
		$('.gpt-DebugBtn').blur();
		googletag.openConsole();
	}
	//--------------------------------------------------------------------------
</script>
        </div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="912135d6-7135-4d45-b457-b503cacc3c8b" data-widget-name="ux3-layout-widget"><header class="header base fixed">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header js__adToHide widget-horizontalAd" id="9d6062d7-fad6-4e64-81cc-7dd25d99b23d">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 1 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="1"></div>
<!-- End AdSlot 1 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" class="header__wrapper clearfix" data-db-parent-of="quickSearch"><div id="brandingRow"><a href="/" data-aa-name="science that inspires" data-aa-region="cp global header" title="Visit Cell.com">Science that inspires</a><p id="brandingRowRight"><a href="/" data-aa-name="cell press endorsement" data-aa-region="cp global header" title="Visit Cell.com">A Cell Press journal</a></p></div>
        
    

        


    

            

            

            <div data-widget-id="3a83e2a6-2e7e-4931-892c-4ef43b6e3312" data-widget-name="ux3-layout-widget"><div class="col-xs-6 header__left-side">
    

        


    

            

            
                <a href="/home" id="cpIconLnk" data-aa-name="cell press logo" data-aa-region="cp global header" title="Cell Press" target="_blank"><svg id="cp_logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 98.1" style="enable-background:new 0 0 100 98.1;" xml:space="preserve" height="100" width="98.1"><path style="fill:#FFFFFF;" d="M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z"></path></svg></a><div class="journal-logos">
            

            <span data-widget-id="35d8d734-3aa0-4ec8-9e2d-14babe803594" data-widget-name="ux3-general-image"><a href="/cell/home" title="Cell"><img alt="Cell" src="/pb-assets/ux3/logos/cell/Cell_white-1532516611593.svg"></a></span>

            
                </div>
            

        

    

        


    

            

            
                <div class="journal-logos">
            

            <span data-widget-id="3ffcbf1f-79fd-4ba3-85b5-bef10a63475f" data-widget-name="ux3-general-image"><a href="https://www.cell.com/open-access?utm_source=oa-label&amp;__hstc=109691226.e4e812b53ff324eccbdbdb2e4fb9a6ab.1644503737405.1644524234622.1644587109507.4&amp;__hssc=109691226.7.1644587109507&amp;__hsfp=3253153019" title="This journal offers authors two options (open access or subscription) to publish research"><img alt="This journal offers authors two options (open access or subscription) to publish research" src="/pb-assets/ux3/logos/cell/Supports_open_access_border-1645554937897.svg"></a></span>

            
                </div>
            

        
</div><div class="col-xs-6">
    

        


    

            

            

            <div data-widget-id="2555f6db-b333-4edb-8b15-ef9213f1f0a5" data-widget-name="ux3-layout-widget"><div class="header__nav row">
    

        


    

            

            

            <ul data-widget-id="52432d24-0331-4c9d-85f3-2033edc7208b" data-widget-name="UX3ProfileLinksWidget" class="rlist rlist--inline profile-links"><li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login-dropdown"><div class="profile-links__user-list__wrapper"><button data-db-target-for="loginmenu-330514" aria-expanded="false" aria-controls="login-links-330514" title="Log in links dropdown" class="profile-links__username"><span>Log in</span></button><div data-db-target-of="loginmenu-330514" id="login-links--330514" class="profile-links__user-list__holder dropBlock__holder"><ul class="rlist profile-links__list"><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" target="_self" data-profile-link="true">Get Institutional Access</a></li></ul></div></div></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" target="_self" data-profile-link="true">Register</a></li><li class="hidden-lg"> <div class="profile-links__controls"> <button aria-expanded="false" aria-controls="links-dialog-330514" data-opens="#links-dialog-330514" class="profile-links__btn--open"><span class="sr-only">open links dialog</span><i aria-hidden="true" class="icon-gizmo-person"></i></button></div><dialog id="links-dialog-330514" aria-modal="true"><button data-closes="#links-dialog-330514" class="profile-links__btn--close"><span class="sr-only">close links dialog</span><i aria-hidden="true" class="icon-close-solid"></i></button><ul>   <li class="profile-links__submit"><a data-adobe-name="Publish" class="profile-links__submit__btn" title="Submit" href="https://www.editorialmanager.com/cell" target="_self" data-aa-conversiondriver="jbs:global-header:submitpage" data-profile-link="true">Submit</a></li><li class="profile-links__login"><a data-adobe-name="Log In" class="profile-links__login__btn" title="Log in to Cell.com" href="https://www.cell.com/action/idLogin?type=login&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" target="_self" data-profile-link="true">Log in</a></li><li class="profile-links__institution"><a data-adobe-name="Get institutional access" class="profile-links__institution__btn" title="Institutional access" href="https://www.cell.com/action/idLogin?type=institution&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" target="_self" data-profile-link="true">Get Institutional Access</a></li><li class="profile-links__register"><a data-adobe-name="Register" class="profile-links__register__btn" title="Register a username and password for free" href="https://www.cell.com/action/idLogin?type=registration&amp;redirectUri=https%3A%2F%2Fwww.cell.com%2Fcell%2Ffulltext%2FS0092-8674%2819%2931398-4" target="_self" data-profile-link="true">Register</a></li></ul></dialog></li></ul>

            

        
</div></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasInstitutionBanner none  widget-none" id="51b7fa33-2137-4f58-9e98-9746c4400513" style="display: none;">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><div class="ui-state-highlight ui-corner-all csas-institution-banner hidden" data-doi="10.1016/j.cell.2019.12.036">
    <span class="ui-icon ui-icon-close banner-close"></span>
    <p>
        Access provided by <span class="accountName"></span>
    </p>
</div>

<style>
    .csas-institution-banner {
        padding: 10px 0 0 10px;
        margin: 5px;
    }

    .csas-institution-banner.ui-state-highlight .ui-icon {
        background-image: url("/templates/jsp/css/jquery-ui-1.10.2/base/images/ui-icons_454545_256x240.png");
        margin-right: 5px;
        margin-top: -4px;
    }

    .csas-institution-banner .banner-close {
        float: right;
    }
</style>

<script>

    function resetBanner(doi) {
        $.get('/action/getAccount?doi=' + doi, function(data) {
            if (data) {
                $(".csas-institution-banner p .accountName").html(data);
                $(".csasInstitutionBanner").fadeIn();
                $(".csas-institution-banner").fadeIn();
            }
        });
    }

    var flag = 0;
    $(document).ready(function() {
        if(!$(".csas-institution-banner").is(":visible")){
            $(".csasInstitutionBanner").hide();
        }

        $(".banner-close").click(function() {
            $.get("/action/dismissBanner", function(data) {
                $(".csasInstitutionBanner").fadeOut(function () {
                    if(typeof UX !== 'undefined') {
                        UX.pageBody.init();
                    }
                });
            });
        });

        $(".articleContent .tabs li[role='tab'].showFullText").click(function() {
           flag = 1;
        });

        $(document).on("reloadTabsContent", function() {
            var isSelected = $(".articleContent .tabs li[role='tab'].showFullText").hasClass("active");
            if (isSelected) {
                var isHidden =  $(".csas-institution-banner").is(".hidden");
                if (!isHidden) {
                    resetBanner($(".csas-institution-banner").data("doi"));
                    console.log("fadeIn");
                }
            } else {
                $(".csas-institution-banner").fadeOut();
                console.log("fadeOut");
            }
            flag = 0;
        });

        $(document).on("click", ".articleContent .tabs li[role='tab']", function() {

        });
    });
</script></div>
                </div>
                </div>

        
</header></div>

            

        










        <div data-widget-def="graphQueryWidget" data-widget-id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" class="ux3">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget graphQueryWidget none top-padding-handler noMargin axel-mega-menu widget-none widget-compact-all fonts-loaded" id="2a125dae-6dbc-4e76-a75e-3e563f1d3a1d" style="padding-top: 115.594px;">
                
                
                    
                

                <div class="wrapped " id="CP TOP LEVEL NAV MENU">
                    
                    <div class="widget-body body body-none  body-compact-all"><nav id="mega-menu-nav" role="navigation" class="mega-menu-nav">
  <!--.skip//a(href='#searchText' aria-label='Skip menu') Skip menu
  -->
  <div class="mega-menu-nav__wrapper">
    <button class="mega-menu__toggle"><span class="sr-only">Main menu</span><i aria-hidden="true" class="icon-gizmo-hamburger"></i></button>
    <ul role="menubar" class="mega-menu rlist">
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-1" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-1" data-item-link="main-menu-link-1" data-index="0" class="mega-menu__item-link menu-link"><span>This journal</span></a>
        <div id="main-menu-panel-1" aria-labelledby="main-menu-link-1" data-item-wrapper="main-menu-link-1" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <div aria-label="Menu dropzone section" tabindex="-1" class="listDropZone">
    

        


    

            

            

            <div class="cover-image"><div class="cover-image__thumb"><a href="/issue/S0092-8674(24)X0012-4" class="menu-link"><img src="/cms/asset/atypon:cms:attachment:img:d5e6:rev:1748270185670-389:pii:S0092867424X00124/cover.tif.jpg" alt="Cover Image - Cell, Volume 188, Issue 11"></a></div><div class="cover-image__meta"></div></div>

            

        
</div>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-2" class="sub-menu__heading">Read
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/newarticles" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Online now</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/current" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Current issue</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/archive" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Archive</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/action/showPreferences?menuTab=Alerts" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Publication alerts</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sneakpeek" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Sneak Peek - preprints</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-1-3" class="sub-menu__heading">About
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-2">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/advisory-board" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Advisory board</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/aims" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Aims and scope</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/contact" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/editors-and-staff" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Editors and staff</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/editorial-ethics" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Ethics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/permissions" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Permissions</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Reviewer instructions</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-2" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-2" data-item-link="main-menu-link-2" data-index="1" class="mega-menu__item-link menu-link"><span>Journals</span></a>
        <div id="main-menu-panel-2" aria-labelledby="main-menu-link-2" data-item-wrapper="main-menu-link-2" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/5" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-1" class="sub-menu__heading">Life &amp; medical sciences
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-4">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cancer-cell/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cancer Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-chemical-biology" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Chemical Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-genomics/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Genomics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-host-microbe/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Host &amp; Microbe</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-metabolism/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-medicine/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-stem-cell/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Stem Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-systems/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Systems</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/current-biology/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Current Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/developmental-cell/home" data-index="11" role="menuitem" class="sub-menu__item-link menu-link"><span>Developmental Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/immunity/home" data-index="12" role="menuitem" class="sub-menu__item-link menu-link"><span>Immunity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/med/home" data-index="13" role="menuitem" class="sub-menu__item-link menu-link"><span>Med</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-cell/home" data-index="14" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Cell</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/neuron/home" data-index="15" role="menuitem" class="sub-menu__item-link menu-link"><span>Neuron</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/structure/home" data-index="16" role="menuitem" class="sub-menu__item-link menu-link"><span>Structure</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/ajhg/home" data-index="17" role="menuitem" class="sub-menu__item-link menu-link"><span>American Journal of Human Genetics (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysj/home" data-index="18" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Journal (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/biophysreports/home" data-index="19" role="menuitem" class="sub-menu__item-link menu-link"><span>Biophysical Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/hgg-advances/home" data-index="20" role="menuitem" class="sub-menu__item-link menu-link"><span>Human Genetics and Genomics Advances (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-plant/home" data-index="21" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Plant (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/molecular-therapy/home" data-index="22" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/methods/home" data-index="23" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Methods &amp; Clinical Development (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/nucleic-acids/home" data-index="24" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Nucleic Acids (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/molecular-therapy-family/oncology/home" data-index="25" role="menuitem" class="sub-menu__item-link menu-link"><span>Molecular Therapy Oncology (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/plant-communications/home" data-index="26" role="menuitem" class="sub-menu__item-link menu-link"><span>Plant Communications (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/stem-cell-reports/home" data-index="27" role="menuitem" class="sub-menu__item-link menu-link"><span>Stem Cell Reports (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biochemical-sciences/home" data-index="28" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biochemical Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cancer/home" data-index="29" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cancer</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cell-biology/home" data-index="30" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cell Biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/ecology-evolution/home" data-index="31" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Ecology &amp; Evolution</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/endocrinology-metabolism/home" data-index="32" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Endocrinology &amp; Metabolism</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/genetics/home" data-index="33" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Genetics</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/immunology/home" data-index="34" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Immunology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/microbiology/home" data-index="35" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Microbiology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/molecular-medicine/home" data-index="36" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Molecular Medicine</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/neurosciences/home" data-index="37" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Neurosciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/parasitology/home" data-index="38" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Parasitology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/pharmacological-sciences/home" data-index="39" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Pharmacological Sciences</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/plant-science/home" data-index="40" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Plant Science</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 5/6" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-2" class="sub-menu__heading">Physical sciences &amp; engineering
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-biomaterials/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Biomaterials</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-physical-science/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Physical Science</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/chem-catalysis/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Chem Catalysis</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/device/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Device</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/joule/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Joule</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/matter/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>Matter</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/newton/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Newton</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/chemistry/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Chemistry</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 6/7" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-2-3" class="sub-menu__heading">Multidisciplinary
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-methods/home" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Methods</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-reports-sustainability/home" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Reports Sustainability</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/heliyon/home" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Heliyon</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/iscience/home" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>iScience</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/one-earth/home" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>One Earth</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/patterns/home" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Patterns</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/star-protocols/home" data-index="6" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Protocols</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/nexus/home" data-index="7" role="menuitem" class="sub-menu__item-link menu-link"><span>Nexus (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/the-innovation/home" data-index="8" role="menuitem" class="sub-menu__item-link menu-link"><span>The Innovation (<em>partner</em>)</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/biotechnology/home" data-index="9" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Biotechnology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/trends/cognitive-sciences/home" data-index="10" role="menuitem" class="sub-menu__item-link menu-link"><span>Trends in Cognitive Sciences</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-3" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-3" data-item-link="main-menu-link-3" data-index="2" class="mega-menu__item-link menu-link"><span>Publish</span></a>
        <div id="main-menu-panel-3" aria-labelledby="main-menu-link-3" data-item-wrapper="main-menu-link-3" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-1" class="sub-menu__heading">Authors
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://www.editorialmanager.com/cell?__hstc=25856994.7201670a4397fa03148684920d4a8d38.1672762013831.1694716659709.1694718909620.146&amp;__hssc=25856994.15.1694718909620&amp;__hsfp=1366844671" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Submit article
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell/information-for-authors" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for authors</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/multi-journal-submission" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Multi-Journal Submission</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/information-for-authors/star-authors-guide" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>STAR Methods</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-3-2" class="sub-menu__heading">Reviewers
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/reviewers" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Information for reviewers</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-4" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-4" data-item-link="main-menu-link-4" data-index="3" class="mega-menu__item-link menu-link"><span>News &amp; events</span></a>
        <div id="main-menu-panel-4" aria-labelledby="main-menu-link-4" data-item-wrapper="main-menu-link-4" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/newsroom" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Newsroom</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/events" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Events</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/symposia" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Symposia</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/webinars" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Webinars</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/consortia" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Consortia Hub</span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
      <li role="presentation" class="mega-menu__item"><a id="main-menu-link-5" aria-haspopup="true" href="javascript:void(0)" role="menuitem" aria-controls="main-menu-panel-5" data-item-link="main-menu-link-5" data-index="4" class="mega-menu__item-link menu-link"><span>About Cell Press</span></a>
        <div id="main-menu-panel-5" aria-labelledby="main-menu-link-5" data-item-wrapper="main-menu-link-5" aria-expanded="false" class="mega-menu__item-wrapper">
          <ul class="sub-menu rlist mega-menu__sub-menu-heading">
            <li style="grid-column: 1/2" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-1" class="sub-menu__heading">Cell Press
              </h4>
              <ul role="menubar" data-index="0" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/about" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>About Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/contact" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Contact us</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="https://service.elsevier.com/app/overview/cell-press/" data-index="2" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Help &amp; support
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/open-access" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>Open access at Cell Press</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/diversity" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Inclusion and diversity</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-50" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>50th anniversary</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 2/3" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-2" class="sub-menu__heading">Hubs
              </h4>
              <ul role="menubar" data-index="1" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="/biotechnology-bioengineering" data-index="0" role="menuitem" class="sub-menu__item-link menu-link"><span>Biotechnology and bioengineering</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/cell-press-chemistry" data-index="1" role="menuitem" class="sub-menu__item-link menu-link"><span>Chemistry</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/clinical-translational" data-index="2" role="menuitem" class="sub-menu__item-link menu-link"><span>Clinical and translational</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/covid-19" data-index="3" role="menuitem" class="sub-menu__item-link menu-link"><span>COVID-19</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/fundamental-biology" data-index="4" role="menuitem" class="sub-menu__item-link menu-link"><span>Fundamental biology</span></a></li>
                <li role="presentation" class="menu-link__wrapper"><a href="/sustainability" data-index="5" role="menuitem" class="sub-menu__item-link menu-link"><span>Sustainability</span></a></li>
              </ul>
            </li>
            <li style="grid-column: 3/4" class="double-width sub-menu__item">
              <h4 data-item-link="main-menu-link-5-3" class="sub-menu__heading">Careers
              </h4>
              <ul role="menubar" data-index="2" class="rlist heading-list col-count-1">
                <li role="presentation" class="menu-link__wrapper"><a href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" data-index="0" target="_blank" role="menuitem" class="sub-menu__item-link menu-link"><span>Cell Press careers
                      <div class="external-icon"></div><span class="sr-only">External Link</span></span></a></li>
              </ul>
            </li>
          </ul>
        </div>
      </li>
    </ul>
    <div class="mega-menu-nav__quick-search">
    

        


    

            

            

            <div data-widget-id="a1f05748-166f-41e0-b01a-ca8ac1cc95d4" data-widget-name="UX3QuickSearchWidget" class="quick-search"><div class="dropBlock"><form action="https://www.cell.com/action/doSearch?journalCode=cell" name="quickSearchBar" method="get" title="Quick Search"><input id="type" type="hidden" value="quicksearch" name="type"><div class="quick-search__toggle autoCompleteWrapper"><input id="searchText" type="text" name="text1" placeholder="Search for..." enterkeyhint="search" autocomplete="off" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" data-group-titles-conf="3" data-publication-items-conf="3" aria-label="Enter a search term" class="searchText searchText-autocomplete ui-autocomplete-input"><button type="submit" title="Search" aria-label="Submit the search form" class="btn quick-search__button icon-search"><span>Go search</span></button><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a><ul id="ui-id-1" tabindex="0" class="ui-menu ui-widget ui-widget-content ui-autocomplete ui-front" unselectable="on" style="display: none;"></ul></div><div class="quick-search__box"><div class="quick-search__box-inner"><div class="quick-search__box-wrapper"><select name="field1" aria-label="Select a search area"><option value="AllField">All content</option><option value="Title">Article title</option><option value="Contrib">Authors</option><option value="Keyword">Keywords</option><option value="Abstract">Abstract</option><option value="AbstractTitleKeywordFilterField">Article title, abstract, keywords</option></select><input type="hidden" name="journalCode" value="cell"><div class="quick-search__scope-options"><span>Search within:</span><label title="This journal" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeSeries-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" title="This journal" value="cell" checked="checked" aria-label="Search within current journal"><span>This journal</span></label><label title="Full site" class="radio--primary"><input type="radio" name="SeriesKey" id="searchScopeFullSite-a1f05748-166f-41e0-b01a-ca8ac1cc95d4" value="" title="Full site" aria-label="Search within full site"><span>Full site</span></label></div><a href="https://www.cell.com/search/advanced?SeriesKey=cell&amp;ISSN=0092-8674&amp;journalCode=cell" title="Advanced search" class="quick-search__advanced-link">Advanced search</a></div><a href="javascript:void(0)" aria-label="Close search options" title="Close search options" class="quick-search__box-close"><i aria-hidden="true" class="icon-tools_close"></i></a></div></div><div class="quick-search__alert hidden"><p>Please enter a term before submitting your search.</p><a href="#" class="quick-search__alert--close"> Ok</a></div></form></div></div>

            

        
</div>
  </div>
</nav>
<div id="page-body-id" class="page-body-flag"></div></div>
                </div>
                </div>

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="f8c6daf9-9df1-4fd9-ba3a-7f5d7b3d4466" data-widget-name="UX3HTMLWidget"><style>
.journal-logos:nth-of-type(2) img {
max-height: 30px !important;
}
.journal-logos:nth-of-type(3) img {
max-height: 30px !important;
}
.ux3 .journal-logos + .journal-logos {
margin-left: 30px;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 55%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 45%;
}
@media screen and (max-width: 1000px) {
.ux3 .journal-logos + .journal-logos {
display: none;
}
.header__wrapper.clearfix .col-xs-6:first-of-type {
width: 50%;
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
width: 50%;
}
}

.ux3 .journal-logos:first-of-type {
   max-width: 500px;
}

.profile-links {
   white-space: nowrap;
}

@media only screen and (min-width: 740px) and (max-width: 1600px) {
   .journal-logos:nth-of-type(2) img {
      max-width: 250px !important;
   }
}

@media only screen and (min-width: 1024px) and (max-width: 1600px) {
   .header__wrapper.clearfix .col-xs-6:first-of-type {
      width: 60% !important;
   }
   .header__wrapper.clearfix .col-xs-6:nth-of-type(2) {
      width: 40% !important;
   }
}

@media only screen and (max-width: 1240px) {
   .ux3 .journal-logos + .journal-logos {
      display: none !important;
   }
}

</style></div>

            

        
</div></div>

            
                </div>
            

        

        </div>
    

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget axelPublicationContent none  widget-none" id="cadc9b41-a542-448a-9de4-8d0803320174">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none "><link id="build-style-article" rel="stylesheet" type="text/css" href="/products/marlin/cell/releasedAssets/css/build-article-b26f3342b722959c7931.css"><article xmlns="http://www.w3.org/1999/xhtml" data-design="pill" data-has="right-rail long-title" vocab="http://schema.org/" typeof="ScholarlyArticle" data-pii="S0092-8674(19)31398-4" lang="en" dir="ltr"><div data-core-wrapper="header"><header data-extent="frontmatter"><div class="core-container"><div data-article-access="free" data-article-access-type="open" class="meta-panel"><span class="meta-panel__left-content"><span class="meta-panel__type"><span>Article</span></span></span><span class="meta-panel__middle-content"><span class="meta-panel__volumeIssue"><a href="/cell/issue?pii=S0092-8674(19)X0004-5"><span>Volume 180</span>, <span>Issue 3</span></a></span><span class="meta-panel__pages"><span>p568-584.e23</span></span><span class="meta-panel__onlineDate">February 06, 2020</span></span><span class="meta-panel__right-content"><span class="meta-panel__access meta-panel__access--archive"><span>Open Archive</span></span></span><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=180&amp;issue=3&amp;pii=S0092-8674%2819%29X0004-5" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div><h1 property="name">Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism</h1><div class="contributors"><span class="authors"><span role="list"><span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au1" aria-controls="axel_author_au1"><span property="givenName">F. Kyle</span> <span property="familyName">Satterstrom</span></a><div id="au1" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">F. Kyle</span> <span property="familyName">Satterstrom</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Satterstrom/F+Kyle" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a role="doc-noteref">37</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au2" aria-controls="axel_author_au2"><span property="givenName">Jack A.</span> <span property="familyName">Kosmicki</span></a><div id="au2" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jack A.</span> <span property="familyName">Kosmicki</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Kosmicki/Jack+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a role="doc-noteref">37</a></sup></span> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au3" aria-controls="axel_author_au3"><span property="givenName">Jiebiao</span> <span property="familyName">Wang</span></a><div id="au3" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jiebiao</span> <span property="familyName">Wang</span></div><div class="content"><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Wang/Jiebiao" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a role="doc-noteref">37</a></sup></span><span data-displayed-on="all"> ∙ … </span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au4" aria-controls="axel_author_au4"><span property="givenName">Michael S.</span> <span property="familyName">Breen</span></a><div id="au4" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael S.</span> <span property="familyName">Breen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Breen/Michael+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au5" aria-controls="axel_author_au5"><span property="givenName">Silvia</span> <span property="familyName">De Rubeis</span></a><div id="au5" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Silvia</span> <span property="familyName">De Rubeis</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/De+Rubeis/Silvia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au6" aria-controls="axel_author_au6"><span property="givenName">Joon-Yong</span> <span property="familyName">An</span></a><div id="au6" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joon-Yong</span> <span property="familyName">An</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">School of Biosystem and Biomedical Science, College of Health Science, Korea University, Seoul, Republic of Korea</span></div></div><div class="core-author-link"><a href="/authored-by/An/Joon-Yong" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">11</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au7" aria-controls="axel_author_au7"><span property="givenName">Minshi</span> <span property="familyName">Peng</span></a><div id="au7" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Minshi</span> <span property="familyName">Peng</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Peng/Minshi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au8" aria-controls="axel_author_au8"><span property="givenName">Ryan</span> <span property="familyName">Collins</span></a><div id="au8" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Ryan</span> <span property="familyName">Collins</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Bioinformatics and Integrative Genomics, Harvard Medical School, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Collins/Ryan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">12</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au9" aria-controls="axel_author_au9"><span property="givenName">Jakob</span> <span property="familyName">Grove</span></a><div id="au9" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au9"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jakob</span> <span property="familyName">Grove</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomics and Personalized Medicine, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedicine – Human Genetics, Aarhus University, Aarhus, Denmark</span></div></div><div class="core-author-link"><a href="/authored-by/Grove/Jakob" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">15</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au10" aria-controls="axel_author_au10"><span property="givenName">Lambertus</span> <span property="familyName">Klei</span></a><div id="au10" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Lambertus</span> <span property="familyName">Klei</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Klei/Lambertus" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au11" aria-controls="axel_author_au11"><span property="givenName">Christine</span> <span property="familyName">Stevens</span></a><div id="au11" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Christine</span> <span property="familyName">Stevens</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Stevens/Christine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au12" aria-controls="axel_author_au12"><span property="givenName">Jennifer</span> <span property="familyName">Reichert</span></a><div id="au12" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Jennifer</span> <span property="familyName">Reichert</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Reichert/Jennifer" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au13" aria-controls="axel_author_au13"><span property="givenName">Maureen S.</span> <span property="familyName">Mulhern</span></a><div id="au13" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Maureen S.</span> <span property="familyName">Mulhern</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Mulhern/Maureen+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au14" aria-controls="axel_author_au14"><span property="givenName">Mykyta</span> <span property="familyName">Artomov</span></a><div id="au14" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mykyta</span> <span property="familyName">Artomov</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Artomov/Mykyta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au15" aria-controls="axel_author_au15"><span property="givenName">Sherif</span> <span property="familyName">Gerges</span></a><div id="au15" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au15"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Sherif</span> <span property="familyName">Gerges</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Gerges/Sherif" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au16" aria-controls="axel_author_au16"><span property="givenName">Brooke</span> <span property="familyName">Sheppard</span></a><div id="au16" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au16"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Brooke</span> <span property="familyName">Sheppard</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sheppard/Brooke" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au17" aria-controls="axel_author_au17"><span property="givenName">Xinyi</span> <span property="familyName">Xu</span></a><div id="au17" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au17"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Xinyi</span> <span property="familyName">Xu</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Xu/Xinyi" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au18" aria-controls="axel_author_au18"><span property="givenName">Aparna</span> <span property="familyName">Bhaduri</span></a><div id="au18" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au18"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Aparna</span> <span property="familyName">Bhaduri</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Neurology, University of California, San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">The Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Bhaduri/Aparna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">17</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">18</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au19" aria-controls="axel_author_au19"><span property="givenName">Utku</span> <span property="familyName">Norman</span></a><div id="au19" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au19"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Utku</span> <span property="familyName">Norman</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Computer Engineering Department, Bilkent University, Ankara, Turkey</span></div></div><div class="core-author-link"><a href="/authored-by/Norman/Utku" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au20" aria-controls="axel_author_au20"><span property="givenName">Harrison</span> <span property="familyName">Brand</span></a><div id="au20" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au20"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Harrison</span> <span property="familyName">Brand</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Brand/Harrison" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au21" aria-controls="axel_author_au21"><span property="givenName">Grace</span> <span property="familyName">Schwartz</span></a><div id="au21" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au21"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Grace</span> <span property="familyName">Schwartz</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Schwartz/Grace" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au22" aria-controls="axel_author_au22"><span property="givenName">Rachel</span> <span property="familyName">Nguyen</span></a><div id="au22" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au22"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Rachel</span> <span property="familyName">Nguyen</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Autism Research and Translation, University of California, Irvine, Irvine, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Nguyen/Rachel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">20</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au23" aria-controls="axel_author_au23"><span property="givenName">Elizabeth E.</span> <span property="familyName">Guerrero</span></a><div id="au23" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au23"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Elizabeth E.</span> <span property="familyName">Guerrero</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">MIND (Medical Investigation of Neurodevelopmental Disorders) Institute, University of California, Davis, Davis, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Guerrero/Elizabeth+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">21</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au24" aria-controls="axel_author_au24"><span property="givenName">Caroline</span> <span property="familyName">Dias</span></a><div id="au24" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Caroline</span> <span property="familyName">Dias</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Division of Genetics, Boston Children’s Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Division of Developmental Medicine, Boston Children’s Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Dias/Caroline" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">22</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">23</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_collab0010" aria-controls="axel_author_collab0010"><span property="name">Autism Sequencing Consortium</span></a><div id="collab0010" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_collab0010"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="name">Autism Sequencing Consortium</span></div><div class="content"><div class="core-author-link"><a href="/authored-by/group/Autism+Sequencing+Consortium" rel="nofollow" target="_blank">Search for articles by this author</a></div><div class="core-collaboration dropBlock__slider"><div class="label">Authors List</div><div class="contributors"><span><a href="#" class="dropBlock__slider__ctrl">Branko Aleksic</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Branko</span> <span property="familyName">Aleksic</span></div><div class="core-author-link"><a href="/authored-by/Aleksic/Branko" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Richard Anney</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Richard</span> <span property="familyName">Anney</span></div><div class="core-author-link"><a href="/authored-by/Anney/Richard" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Mafalda Barbosa</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Mafalda</span> <span property="familyName">Barbosa</span></div><div class="core-author-link"><a href="/authored-by/Barbosa/Mafalda" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Somer Bishop</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Somer</span> <span property="familyName">Bishop</span></div><div class="core-author-link"><a href="/authored-by/Bishop/Somer" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Alfredo Brusco</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Alfredo</span> <span property="familyName">Brusco</span></div><div class="core-author-link"><a href="/authored-by/Brusco/Alfredo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Jonas Bybjerg-Grauholm</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Jonas</span> <span property="familyName">Bybjerg-Grauholm</span></div><div class="core-author-link"><a href="/authored-by/Bybjerg-Grauholm/Jonas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Angel Carracedo</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Angel</span> <span property="familyName">Carracedo</span></div><div class="core-author-link"><a href="/authored-by/Carracedo/Angel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Marcus C.Y. Chan</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Marcus C.Y.</span> <span property="familyName">Chan</span></div><div class="core-author-link"><a href="/authored-by/Chan/Marcus+C+Y" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Andreas G. Chiocchetti</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Andreas G.</span> <span property="familyName">Chiocchetti</span></div><div class="core-author-link"><a href="/authored-by/Chiocchetti/Andreas+G" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Brian H.Y. Chung</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Brian H.Y.</span> <span property="familyName">Chung</span></div><div class="core-author-link"><a href="/authored-by/Chung/Brian+H+Y" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Hilary Coon</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Hilary</span> <span property="familyName">Coon</span></div><div class="core-author-link"><a href="/authored-by/Coon/Hilary" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Michael L. Cuccaro</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Michael L.</span> <span property="familyName">Cuccaro</span></div><div class="core-author-link"><a href="/authored-by/Cuccaro/Michael+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Aurora Curró</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Aurora</span> <span property="familyName">Curró</span></div><div class="core-author-link"><a href="/authored-by/Curr%C3%B3/Aurora" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Bernardo Dalla Bernardina</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Bernardo</span> <span property="familyName">Dalla Bernardina</span></div><div class="core-author-link"><a href="/authored-by/Dalla+Bernardina/Bernardo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Ryan Doan</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Ryan</span> <span property="familyName">Doan</span></div><div class="core-author-link"><a href="/authored-by/Doan/Ryan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Enrico Domenici</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Enrico</span> <span property="familyName">Domenici</span></div><div class="core-author-link"><a href="/authored-by/Domenici/Enrico" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Shan Dong</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Shan</span> <span property="familyName">Dong</span></div><div class="core-author-link"><a href="/authored-by/Dong/Shan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Chiara Fallerini</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Chiara</span> <span property="familyName">Fallerini</span></div><div class="core-author-link"><a href="/authored-by/Fallerini/Chiara" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Montserrat Fernández-Prieto</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Montserrat</span> <span property="familyName">Fernández-Prieto</span></div><div class="core-author-link"><a href="/authored-by/Fern%C3%A1ndez-Prieto/Montserrat" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Giovanni Battista Ferrero</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Giovanni Battista</span> <span property="familyName">Ferrero</span></div><div class="core-author-link"><a href="/authored-by/Ferrero/Giovanni+Battista" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Christine M. Freitag</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Christine M.</span> <span property="familyName">Freitag</span></div><div class="core-author-link"><a href="/authored-by/Freitag/Christine+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Menachem Fromer</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Menachem</span> <span property="familyName">Fromer</span></div><div class="core-author-link"><a href="/authored-by/Fromer/Menachem" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">J. Jay Gargus</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">J. Jay</span> <span property="familyName">Gargus</span></div><div class="core-author-link"><a href="/authored-by/Gargus/J+Jay" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Daniel Geschwind</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Daniel</span> <span property="familyName">Geschwind</span></div><div class="core-author-link"><a href="/authored-by/Geschwind/Daniel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Elisa Giorgio</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Elisa</span> <span property="familyName">Giorgio</span></div><div class="core-author-link"><a href="/authored-by/Giorgio/Elisa" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Javier González-Peñas</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Javier</span> <span property="familyName">González-Peñas</span></div><div class="core-author-link"><a href="/authored-by/Gonz%C3%A1lez-Pe%C3%B1as/Javier" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Stephen Guter</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Stephen</span> <span property="familyName">Guter</span></div><div class="core-author-link"><a href="/authored-by/Guter/Stephen" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Danielle Halpern</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Danielle</span> <span property="familyName">Halpern</span></div><div class="core-author-link"><a href="/authored-by/Halpern/Danielle" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Emily Hansen-Kiss</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Emily</span> <span property="familyName">Hansen-Kiss</span></div><div class="core-author-link"><a href="/authored-by/Hansen-Kiss/Emily" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Xin He</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Xin</span> <span property="familyName">He</span></div><div class="core-author-link"><a href="/authored-by/He/Xin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Gail E. Herman</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Gail E.</span> <span property="familyName">Herman</span></div><div class="core-author-link"><a href="/authored-by/Herman/Gail+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Irva Hertz-Picciotto</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Irva</span> <span property="familyName">Hertz-Picciotto</span></div><div class="core-author-link"><a href="/authored-by/Hertz-Picciotto/Irva" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">David M. Hougaard</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">David M.</span> <span property="familyName">Hougaard</span></div><div class="core-author-link"><a href="/authored-by/Hougaard/David+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Christina M. Hultman</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Christina M.</span> <span property="familyName">Hultman</span></div><div class="core-author-link"><a href="/authored-by/Hultman/Christina+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Iuliana Ionita-Laza</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Iuliana</span> <span property="familyName">Ionita-Laza</span></div><div class="core-author-link"><a href="/authored-by/Ionita-Laza/Iuliana" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Suma Jacob</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Suma</span> <span property="familyName">Jacob</span></div><div class="core-author-link"><a href="/authored-by/Jacob/Suma" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Jesslyn Jamison</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Jesslyn</span> <span property="familyName">Jamison</span></div><div class="core-author-link"><a href="/authored-by/Jamison/Jesslyn" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Astanand Jugessur</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Astanand</span> <span property="familyName">Jugessur</span></div><div class="core-author-link"><a href="/authored-by/Jugessur/Astanand" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Miia Kaartinen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Miia</span> <span property="familyName">Kaartinen</span></div><div class="core-author-link"><a href="/authored-by/Kaartinen/Miia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Gun Peggy Knudsen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Gun Peggy</span> <span property="familyName">Knudsen</span></div><div class="core-author-link"><a href="/authored-by/Knudsen/Gun+Peggy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Alexander Kolevzon</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Alexander</span> <span property="familyName">Kolevzon</span></div><div class="core-author-link"><a href="/authored-by/Kolevzon/Alexander" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Itaru Kushima</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Itaru</span> <span property="familyName">Kushima</span></div><div class="core-author-link"><a href="/authored-by/Kushima/Itaru" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">So Lun Lee</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">So Lun</span> <span property="familyName">Lee</span></div><div class="core-author-link"><a href="/authored-by/Lee/So+Lun" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Terho Lehtimäki</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Terho</span> <span property="familyName">Lehtimäki</span></div><div class="core-author-link"><a href="/authored-by/Lehtim%C3%A4ki/Terho" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Elaine T. Lim</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Elaine T.</span> <span property="familyName">Lim</span></div><div class="core-author-link"><a href="/authored-by/Lim/Elaine+T" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Carla Lintas</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Carla</span> <span property="familyName">Lintas</span></div><div class="core-author-link"><a href="/authored-by/Lintas/Carla" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">W. Ian Lipkin</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">W. Ian</span> <span property="familyName">Lipkin</span></div><div class="core-author-link"><a href="/authored-by/Lipkin/W+Ian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Diego Lopergolo</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Diego</span> <span property="familyName">Lopergolo</span></div><div class="core-author-link"><a href="/authored-by/Lopergolo/Diego" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Fátima Lopes</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Fátima</span> <span property="familyName">Lopes</span></div><div class="core-author-link"><a href="/authored-by/Lopes/F%C3%A1tima" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Yunin Ludena</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Yunin</span> <span property="familyName">Ludena</span></div><div class="core-author-link"><a href="/authored-by/Ludena/Yunin" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Patricia Maciel</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Patricia</span> <span property="familyName">Maciel</span></div><div class="core-author-link"><a href="/authored-by/Maciel/Patricia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Per Magnus</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Per</span> <span property="familyName">Magnus</span></div><div class="core-author-link"><a href="/authored-by/Magnus/Per" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Behrang Mahjani</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Behrang</span> <span property="familyName">Mahjani</span></div><div class="core-author-link"><a href="/authored-by/Mahjani/Behrang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Nell Maltman</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Nell</span> <span property="familyName">Maltman</span></div><div class="core-author-link"><a href="/authored-by/Maltman/Nell" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Dara S. Manoach</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Dara S.</span> <span property="familyName">Manoach</span></div><div class="core-author-link"><a href="/authored-by/Manoach/Dara+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Gal Meiri</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Gal</span> <span property="familyName">Meiri</span></div><div class="core-author-link"><a href="/authored-by/Meiri/Gal" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Idan Menashe</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Idan</span> <span property="familyName">Menashe</span></div><div class="core-author-link"><a href="/authored-by/Menashe/Idan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Judith Miller</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Judith</span> <span property="familyName">Miller</span></div><div class="core-author-link"><a href="/authored-by/Miller/Judith" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Nancy Minshew</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Nancy</span> <span property="familyName">Minshew</span></div><div class="core-author-link"><a href="/authored-by/Minshew/Nancy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Eduarda M.S. Montenegro</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Eduarda M.S.</span> <span property="familyName">Montenegro</span></div><div class="core-author-link"><a href="/authored-by/Montenegro/Eduarda+M+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Danielle Moreira</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Danielle</span> <span property="familyName">Moreira</span></div><div class="core-author-link"><a href="/authored-by/Moreira/Danielle" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Eric M. Morrow</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Eric M.</span> <span property="familyName">Morrow</span></div><div class="core-author-link"><a href="/authored-by/Morrow/Eric+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Ole Mors</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Ole</span> <span property="familyName">Mors</span></div><div class="core-author-link"><a href="/authored-by/Mors/Ole" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Preben Bo Mortensen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Preben Bo</span> <span property="familyName">Mortensen</span></div><div class="core-author-link"><a href="/authored-by/Mortensen/Preben+Bo" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Matthew Mosconi</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Matthew</span> <span property="familyName">Mosconi</span></div><div class="core-author-link"><a href="/authored-by/Mosconi/Matthew" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Pierandrea Muglia</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Pierandrea</span> <span property="familyName">Muglia</span></div><div class="core-author-link"><a href="/authored-by/Muglia/Pierandrea" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Benjamin M. Neale</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Benjamin M.</span> <span property="familyName">Neale</span></div><div class="core-author-link"><a href="/authored-by/Neale/Benjamin+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Merete Nordentoft</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Merete</span> <span property="familyName">Nordentoft</span></div><div class="core-author-link"><a href="/authored-by/Nordentoft/Merete" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Norio Ozaki</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Norio</span> <span property="familyName">Ozaki</span></div><div class="core-author-link"><a href="/authored-by/Ozaki/Norio" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Aarno Palotie</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Aarno</span> <span property="familyName">Palotie</span></div><div class="core-author-link"><a href="/authored-by/Palotie/Aarno" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Mara Parellada</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Mara</span> <span property="familyName">Parellada</span></div><div class="core-author-link"><a href="/authored-by/Parellada/Mara" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Maria Rita Passos-Bueno</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Maria Rita</span> <span property="familyName">Passos-Bueno</span></div><div class="core-author-link"><a href="/authored-by/Passos-Bueno/Maria+Rita" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Margaret Pericak-Vance</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Margaret</span> <span property="familyName">Pericak-Vance</span></div><div class="core-author-link"><a href="/authored-by/Pericak-Vance/Margaret" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Antonio M. Persico</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Antonio M.</span> <span property="familyName">Persico</span></div><div class="core-author-link"><a href="/authored-by/Persico/Antonio+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Isaac Pessah</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Isaac</span> <span property="familyName">Pessah</span></div><div class="core-author-link"><a href="/authored-by/Pessah/Isaac" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Kaija Puura</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Kaija</span> <span property="familyName">Puura</span></div><div class="core-author-link"><a href="/authored-by/Puura/Kaija" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Abraham Reichenberg</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Abraham</span> <span property="familyName">Reichenberg</span></div><div class="core-author-link"><a href="/authored-by/Reichenberg/Abraham" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Alessandra Renieri</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Alessandra</span> <span property="familyName">Renieri</span></div><div class="core-author-link"><a href="/authored-by/Renieri/Alessandra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Evelise Riberi</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Evelise</span> <span property="familyName">Riberi</span></div><div class="core-author-link"><a href="/authored-by/Riberi/Evelise" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Elise B. Robinson</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Elise B.</span> <span property="familyName">Robinson</span></div><div class="core-author-link"><a href="/authored-by/Robinson/Elise+B" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Kaitlin E. Samocha</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Kaitlin E.</span> <span property="familyName">Samocha</span></div><div class="core-author-link"><a href="/authored-by/Samocha/Kaitlin+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Sven Sandin</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Sven</span> <span property="familyName">Sandin</span></div><div class="core-author-link"><a href="/authored-by/Sandin/Sven" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Susan L. Santangelo</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Susan L.</span> <span property="familyName">Santangelo</span></div><div class="core-author-link"><a href="/authored-by/Santangelo/Susan+L" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Gerry Schellenberg</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Gerry</span> <span property="familyName">Schellenberg</span></div><div class="core-author-link"><a href="/authored-by/Schellenberg/Gerry" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Stephen W. Scherer</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Stephen W.</span> <span property="familyName">Scherer</span></div><div class="core-author-link"><a href="/authored-by/Scherer/Stephen+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Sabine Schlitt</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Sabine</span> <span property="familyName">Schlitt</span></div><div class="core-author-link"><a href="/authored-by/Schlitt/Sabine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Rebecca Schmidt</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Rebecca</span> <span property="familyName">Schmidt</span></div><div class="core-author-link"><a href="/authored-by/Schmidt/Rebecca" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Lauren Schmitt</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Lauren</span> <span property="familyName">Schmitt</span></div><div class="core-author-link"><a href="/authored-by/Schmitt/Lauren" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Isabela M.W. Silva</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Isabela M.W.</span> <span property="familyName">Silva</span></div><div class="core-author-link"><a href="/authored-by/Silva/Isabela+M+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Tarjinder Singh</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Tarjinder</span> <span property="familyName">Singh</span></div><div class="core-author-link"><a href="/authored-by/Singh/Tarjinder" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Paige M. Siper</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Paige M.</span> <span property="familyName">Siper</span></div><div class="core-author-link"><a href="/authored-by/Siper/Paige+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Moyra Smith</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Moyra</span> <span property="familyName">Smith</span></div><div class="core-author-link"><a href="/authored-by/Smith/Moyra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Gabriela Soares</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Gabriela</span> <span property="familyName">Soares</span></div><div class="core-author-link"><a href="/authored-by/Soares/Gabriela" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Camilla Stoltenberg</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Camilla</span> <span property="familyName">Stoltenberg</span></div><div class="core-author-link"><a href="/authored-by/Stoltenberg/Camilla" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Pål Suren</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Pål</span> <span property="familyName">Suren</span></div><div class="core-author-link"><a href="/authored-by/Suren/P%C3%A5l" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Ezra Susser</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Ezra</span> <span property="familyName">Susser</span></div><div class="core-author-link"><a href="/authored-by/Susser/Ezra" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">John Sweeney</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">John</span> <span property="familyName">Sweeney</span></div><div class="core-author-link"><a href="/authored-by/Sweeney/John" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Peter Szatmari</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Peter</span> <span property="familyName">Szatmari</span></div><div class="core-author-link"><a href="/authored-by/Szatmari/Peter" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Lara Tang</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Lara</span> <span property="familyName">Tang</span></div><div class="core-author-link"><a href="/authored-by/Tang/Lara" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Flora Tassone</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Flora</span> <span property="familyName">Tassone</span></div><div class="core-author-link"><a href="/authored-by/Tassone/Flora" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Karoline Teufel</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Karoline</span> <span property="familyName">Teufel</span></div><div class="core-author-link"><a href="/authored-by/Teufel/Karoline" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Elisabetta Trabetti</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Elisabetta</span> <span property="familyName">Trabetti</span></div><div class="core-author-link"><a href="/authored-by/Trabetti/Elisabetta" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Maria del Pilar Trelles</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Maria del Pilar</span> <span property="familyName">Trelles</span></div><div class="core-author-link"><a href="/authored-by/Trelles/Maria+del+Pilar" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Christopher A. Walsh</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Christopher A.</span> <span property="familyName">Walsh</span></div><div class="core-author-link"><a href="/authored-by/Walsh/Christopher+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Lauren A. Weiss</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Lauren A.</span> <span property="familyName">Weiss</span></div><div class="core-author-link"><a href="/authored-by/Weiss/Lauren+A" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Thomas Werge</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Thomas</span> <span property="familyName">Werge</span></div><div class="core-author-link"><a href="/authored-by/Werge/Thomas" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Donna M. Werling</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Donna M.</span> <span property="familyName">Werling</span></div><div class="core-author-link"><a href="/authored-by/Werling/Donna+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Emilie M. Wigdor</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Emilie M.</span> <span property="familyName">Wigdor</span></div><div class="core-author-link"><a href="/authored-by/Wigdor/Emilie+M" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Emma Wilkinson</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Emma</span> <span property="familyName">Wilkinson</span></div><div class="core-author-link"><a href="/authored-by/Wilkinson/Emma" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">A. Jeremy Willsey</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">A. Jeremy</span> <span property="familyName">Willsey</span></div><div class="core-author-link"><a href="/authored-by/Willsey/A+Jeremy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Timothy W. Yu</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Timothy W.</span> <span property="familyName">Yu</span></div><div class="core-author-link"><a href="/authored-by/Yu/Timothy+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Mullin H.C. Yu</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Mullin H.C.</span> <span property="familyName">Yu</span></div><div class="core-author-link"><a href="/authored-by/Yu/Mullin+H+C" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Ryan Yuen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Ryan</span> <span property="familyName">Yuen</span></div><div class="core-author-link"><a href="/authored-by/Yuen/Ryan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Elaine Zachi</a><div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Elaine</span> <span property="familyName">Zachi</span></div><div class="core-author-link"><a href="/authored-by/Zachi/Elaine" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span></div></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_collab0015" aria-controls="axel_author_collab0015"><span property="name">iPSYCH-Broad Consortium</span></a><div id="collab0015" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_collab0015"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="name">iPSYCH-Broad Consortium</span></div><div class="content"><div class="core-author-link"><a href="/authored-by/group/iPSYCH-Broad+Consortium" rel="nofollow" target="_blank">Search for articles by this author</a></div><div class="core-collaboration dropBlock__slider"><div class="label">Authors List</div><div class="contributors"><span><a href="#" class="dropBlock__slider__ctrl">Esben Agerbo</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Esben</span> <span property="familyName">Agerbo</span></div><div class="core-author-link"><a href="/authored-by/Agerbo/Esben" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Thomas Damm Als</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Thomas Damm</span> <span property="familyName">Als</span></div><div class="core-author-link"><a href="/authored-by/Als/Thomas+Damm" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Vivek Appadurai</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Vivek</span> <span property="familyName">Appadurai</span></div><div class="core-author-link"><a href="/authored-by/Appadurai/Vivek" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Marie Bækvad-Hansen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Marie</span> <span property="familyName">Bækvad-Hansen</span></div><div class="core-author-link"><a href="/authored-by/B%C3%A6kvad-Hansen/Marie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Rich Belliveau</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Rich</span> <span property="familyName">Belliveau</span></div><div class="core-author-link"><a href="/authored-by/Belliveau/Rich" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Alfonso Buil</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Alfonso</span> <span property="familyName">Buil</span></div><div class="core-author-link"><a href="/authored-by/Buil/Alfonso" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Caitlin E. Carey</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Caitlin E.</span> <span property="familyName">Carey</span></div><div class="core-author-link"><a href="/authored-by/Carey/Caitlin+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Felecia Cerrato</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Felecia</span> <span property="familyName">Cerrato</span></div><div class="core-author-link"><a href="/authored-by/Cerrato/Felecia" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Kimberly Chambert</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Kimberly</span> <span property="familyName">Chambert</span></div><div class="core-author-link"><a href="/authored-by/Chambert/Kimberly" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Claire Churchhouse</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Claire</span> <span property="familyName">Churchhouse</span></div><div class="core-author-link"><a href="/authored-by/Churchhouse/Claire" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Søren Dalsgaard</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Søren</span> <span property="familyName">Dalsgaard</span></div><div class="core-author-link"><a href="/authored-by/Dalsgaard/S%C3%B8ren" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Ditte Demontis</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Ditte</span> <span property="familyName">Demontis</span></div><div class="core-author-link"><a href="/authored-by/Demontis/Ditte" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Ashley Dumont</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Ashley</span> <span property="familyName">Dumont</span></div><div class="core-author-link"><a href="/authored-by/Dumont/Ashley" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Jacqueline Goldstein</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Jacqueline</span> <span property="familyName">Goldstein</span></div><div class="core-author-link"><a href="/authored-by/Goldstein/Jacqueline" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Christine S. Hansen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Christine S.</span> <span property="familyName">Hansen</span></div><div class="core-author-link"><a href="/authored-by/Hansen/Christine+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Mads Engel Hauberg</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Mads Engel</span> <span property="familyName">Hauberg</span></div><div class="core-author-link"><a href="/authored-by/Hauberg/Mads+Engel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Mads V. Hollegaard</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Mads V.</span> <span property="familyName">Hollegaard</span></div><div class="core-author-link"><a href="/authored-by/Hollegaard/Mads+V" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Daniel P. Howrigan</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Daniel P.</span> <span property="familyName">Howrigan</span></div><div class="core-author-link"><a href="/authored-by/Howrigan/Daniel+P" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Hailiang Huang</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Hailiang</span> <span property="familyName">Huang</span></div><div class="core-author-link"><a href="/authored-by/Huang/Hailiang" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Julian Maller</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Julian</span> <span property="familyName">Maller</span></div><div class="core-author-link"><a href="/authored-by/Maller/Julian" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Alicia R. Martin</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Alicia R.</span> <span property="familyName">Martin</span></div><div class="core-author-link"><a href="/authored-by/Martin/Alicia+R" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Joanna Martin</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Joanna</span> <span property="familyName">Martin</span></div><div class="core-author-link"><a href="/authored-by/Martin/Joanna" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Manuel Mattheisen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Manuel</span> <span property="familyName">Mattheisen</span></div><div class="core-author-link"><a href="/authored-by/Mattheisen/Manuel" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Jennifer Moran</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Jennifer</span> <span property="familyName">Moran</span></div><div class="core-author-link"><a href="/authored-by/Moran/Jennifer" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Jonatan Pallesen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Jonatan</span> <span property="familyName">Pallesen</span></div><div class="core-author-link"><a href="/authored-by/Pallesen/Jonatan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Duncan S. Palmer</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Duncan S.</span> <span property="familyName">Palmer</span></div><div class="core-author-link"><a href="/authored-by/Palmer/Duncan+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Carsten Bøcker Pedersen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Carsten Bøcker</span> <span property="familyName">Pedersen</span></div><div class="core-author-link"><a href="/authored-by/Pedersen/Carsten+B%C3%B8cker" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Marianne Giørtz Pedersen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Marianne Giørtz</span> <span property="familyName">Pedersen</span></div><div class="core-author-link"><a href="/authored-by/Pedersen/Marianne+Gi%C3%B8rtz" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Timothy Poterba</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Timothy</span> <span property="familyName">Poterba</span></div><div class="core-author-link"><a href="/authored-by/Poterba/Timothy" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Jesper Buchhave Poulsen</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Jesper Buchhave</span> <span property="familyName">Poulsen</span></div><div class="core-author-link"><a href="/authored-by/Poulsen/Jesper+Buchhave" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Stephan Ripke</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Stephan</span> <span property="familyName">Ripke</span></div><div class="core-author-link"><a href="/authored-by/Ripke/Stephan" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Andrew J. Schork</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Andrew J.</span> <span property="familyName">Schork</span></div><div class="core-author-link"><a href="/authored-by/Schork/Andrew+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Wesley K. Thompson</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Wesley K.</span> <span property="familyName">Thompson</span></div><div class="core-author-link"><a href="/authored-by/Thompson/Wesley+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Patrick Turley</a>, <div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Patrick</span> <span property="familyName">Turley</span></div><div class="core-author-link"><a href="/authored-by/Turley/Patrick" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span><span><a href="#" class="dropBlock__slider__ctrl">Raymond K. Walters</a><div property="author" typeof="Person" class="dropBlock__slider__wrapper"><a class="dropBlock__slider__back" href="#" title="Back to list"><i class="icon-arrow_l" aria-hidden="true"></i><span>Back to list</span></a><div property="name" class="heading"><span property="givenName">Raymond K.</span> <span property="familyName">Walters</span></div><div class="core-author-link"><a href="/authored-by/Walters/Raymond+K" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></span></div></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au174" aria-controls="axel_author_au174"><span property="givenName">Catalina</span> <span property="familyName">Betancur</span></a><div id="au174" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au174"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Catalina</span> <span property="familyName">Betancur</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Sorbonne Université, INSERM, CNRS, Neuroscience Paris Seine, Institut de Biologie Paris Seine, Paris, France</span></div></div><div class="core-author-link"><a href="/authored-by/Betancur/Catalina" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">24</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au175" aria-controls="axel_author_au175"><span property="givenName">Edwin H.</span> <span property="familyName">Cook</span></a><div id="au175" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au175"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Edwin H.</span> <span property="familyName">Cook</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Institute for Juvenile Research, Department of Psychiatry, University of Illinois at Chicago, Chicago, IL, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cook/Edwin+H" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">25</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au176" aria-controls="axel_author_au176"><span property="givenName">Louise</span> <span property="familyName">Gallagher</span></a><div id="au176" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au176"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Louise</span> <span property="familyName">Gallagher</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, School of Medicine, Trinity College Dublin, Dublin, Ireland</span></div></div><div class="core-author-link"><a href="/authored-by/Gallagher/Louise" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">26</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au177" aria-controls="axel_author_au177"><span property="givenName">Michael</span> <span property="familyName">Gill</span></a><div id="au177" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au177"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael</span> <span property="familyName">Gill</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, School of Medicine, Trinity College Dublin, Dublin, Ireland</span></div></div><div class="core-author-link"><a href="/authored-by/Gill/Michael" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">26</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au178" aria-controls="axel_author_au178"><span property="givenName">James S.</span> <span property="familyName">Sutcliffe</span></a><div id="au178" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au178"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">James S.</span> <span property="familyName">Sutcliffe</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Vanderbilt Genetics Institute, Vanderbilt University School of Medicine, Nashville, TN, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Molecular Physiology and Biophysics and Psychiatry, Vanderbilt University School of Medicine, Nashville, TN, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sutcliffe/James+S" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">27</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">28</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au179" aria-controls="axel_author_au179"><span property="givenName">Audrey</span> <span property="familyName">Thurm</span></a><div id="au179" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au179"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Audrey</span> <span property="familyName">Thurm</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">National Institute of Mental Health, NIH, Bethesda, MD, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Thurm/Audrey" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">29</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au180" aria-controls="axel_author_au180"><span property="givenName">Michael E.</span> <span property="familyName">Zwick</span></a><div id="au180" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au180"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael E.</span> <span property="familyName">Zwick</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Zwick/Michael+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">30</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au181" aria-controls="axel_author_au181"><span property="givenName">Anders D.</span> <span property="familyName">Børglum</span></a><div id="au181" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au181"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Anders D.</span> <span property="familyName">Børglum</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomics and Personalized Medicine, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Biomedicine – Human Genetics, Aarhus University, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span property="name">Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark</span></div></div><div class="core-author-link"><a href="/authored-by/B%C3%B8rglum/Anders+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">13</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">14</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">15</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">31</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au182" aria-controls="axel_author_au182"><span property="givenName">Matthew W.</span> <span property="familyName">State</span></a><div id="au182" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au182"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Matthew W.</span> <span property="familyName">State</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/State/Matthew+W" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au183" aria-controls="axel_author_au183"><span property="givenName">A. Ercument</span> <span property="familyName">Cicek</span></a><div id="au183" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au183"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">A. Ercument</span> <span property="familyName">Cicek</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Computer Engineering Department, Bilkent University, Ankara, Turkey</span></div></div><div class="core-author-link"><a href="/authored-by/Cicek/A+Ercument" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">19</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au184" aria-controls="axel_author_au184"><span property="givenName">Michael E.</span> <span property="familyName">Talkowski</span></a><div id="au184" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au184"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Michael E.</span> <span property="familyName">Talkowski</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Talkowski/Michael+E" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">5</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au185" aria-controls="axel_author_au185"><span property="givenName">David J.</span> <span property="familyName">Cutler</span></a><div id="au185" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au185"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">David J.</span> <span property="familyName">Cutler</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Cutler/David+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">30</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au186" aria-controls="axel_author_au186"><span property="givenName">Bernie</span> <span property="familyName">Devlin</span></a><div id="au186" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au186"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Bernie</span> <span property="familyName">Devlin</span></div><div class="content"><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Devlin/Bernie" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">16</a></sup></span></span><span data-hidden-on="all"> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au187" aria-controls="axel_author_au187"><span property="givenName">Stephan J.</span> <span property="familyName">Sanders</span></a><div id="au187" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au187"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Stephan J.</span> <span property="familyName">Sanders</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0510" href="mailto:stephan.sanders@ucsf.edu" property="email" title="Send email to Stephan J. Sanders">stephan.sanders@ucsf.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Sanders/Stephan+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">10</a>,<a role="doc-noteref">38</a></sup> <a id="eadd0510" href="mailto:stephan.sanders@ucsf.edu" property="email" title="Send email to Stephan J. Sanders">stephan.sanders@ucsf.edu</a></span></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au188" aria-controls="axel_author_au188"><span property="givenName">Kathryn</span> <span property="familyName">Roeder</span></a><div id="au188" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au188"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Kathryn</span> <span property="familyName">Roeder</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0410" href="mailto:roeder@andrew.cmu.edu" property="email" title="Send email to Kathryn Roeder">roeder@andrew.cmu.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Computational Biology Department, Carnegie Mellon University, Pittsburgh, PA, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Roeder/Kathryn" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">6</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">32</a>,<a role="doc-noteref">38</a></sup> <a id="eadd0410" href="mailto:roeder@andrew.cmu.edu" property="email" title="Send email to Kathryn Roeder">roeder@andrew.cmu.edu</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au189" aria-controls="axel_author_au189"><span property="givenName">Mark J.</span> <span property="familyName">Daly</span></a><div id="au189" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au189"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Mark J.</span> <span property="familyName">Daly</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0110" href="mailto:mjdaly@broadinstitute.org" property="email" title="Send email to Mark J. Daly">mjdaly@broadinstitute.org</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</span></div></div><div class="core-author-link"><a href="/authored-by/Daly/Mark+J" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">1</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">2</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">3</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">4</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">5</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">33</a>,<a role="doc-noteref">38</a></sup> <a id="eadd0110" href="mailto:mjdaly@broadinstitute.org" property="email" title="Send email to Mark J. Daly">mjdaly@broadinstitute.org</a></span> ∙ <span property="author" typeof="Person" role="listitem" class="corresponding-author"><span class="dropBlock"><a href="#" aria-expanded="false" data-db-target-for="axel_author_au190" aria-controls="axel_author_au190"><span property="givenName">Joseph D.</span> <span property="familyName">Buxbaum</span></a><div id="au190" property="author" typeof="Person" class="dropBlock__holder" data-db-target-of="axel_author_au190"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="heading"><span property="givenName">Joseph D.</span> <span property="familyName">Buxbaum</span></div><div class="content"><div class="core-author-emails"><div class="label">Correspondence</div><div>Corresponding author</div><div><a id="eadd0010" href="mailto:joseph.buxbaum@mssm.edu" property="email" title="Send email to Joseph D. Buxbaum">joseph.buxbaum@mssm.edu</a></div></div><div class="core-author-notes"><div class="label">Footnotes</div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">39</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead Contact</div></div></div></div><div class="core-author-affiliations"><div class="label">Affiliations</div><div property="affiliation" typeof="Organization"><span property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span property="name">Department of Neuroscience, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-author-link"><a href="/authored-by/Buxbaum/Joseph+D" rel="nofollow" target="_blank">Search for articles by this author</a></div></div></div><button class="dropBlock__ctrl--close" title="Close contributor details" aria-label="Close contributor details"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span><sup class="xref"><a property="affiliation" typeof="Organization" role="doc-noteref">7</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">8</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">9</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">34</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">35</a>,<a property="affiliation" typeof="Organization" role="doc-noteref">36</a>,<a role="doc-noteref">38</a>,<a role="doc-noteref">39</a></sup> <a id="eadd0010" href="mailto:joseph.buxbaum@mssm.edu" property="email" title="Send email to Joseph D. Buxbaum">joseph.buxbaum@mssm.edu</a></span><span data-displayed-on="all"> … <span><span data-action="reveal" tabindex="0" role="listitem">Show more</span></span> </span><span data-displayed-on="none" data-on-display="all" aria-hidden="true"> <span data-action="hide" tabindex="0" role="listitem">Show less</span></span></span></span></div><div class="core-self-citation"><div class="expandable-triggers"><button aria-controls="core-affiliations-notes" aria-expanded="false" tabindex="0" role="button"><span>Affiliations &amp; Notes</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button><button aria-controls="core-content-info" aria-expanded="false" tabindex="0" role="button"><span>Article Info</span><i class="icon-arrow_d_n" aria-hidden="true"></i></button></div><div id="core-linked-content">
    

        

</div><div class="expandable-content"><div id="core-affiliations-notes" role="region" data-expandable="item" style="display: none;"><div class="affiliations"><div property="affiliation" typeof="Organization"><span class="label">1</span><span id="aff1" property="name">Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">2</span><span id="aff2" property="name">Analytic and Translational Genetics Unit, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">3</span><span id="aff3" property="name">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">4</span><span id="aff4" property="name">Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">5</span><span id="aff5" property="name">Center for Genomic Medicine, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">6</span><span id="aff6" property="name">Department of Statistics, Carnegie Mellon University, Pittsburgh, PA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">7</span><span id="aff7" property="name">Seaver Autism Center for Research and Treatment, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">8</span><span id="aff8" property="name">Department of Psychiatry, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">9</span><span id="aff9" property="name">The Mindich Child Health and Development Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">10</span><span id="aff10" property="name">Department of Psychiatry, UCSF Weill Institute for Neurosciences, University of California, San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">11</span><span id="aff11" property="name">School of Biosystem and Biomedical Science, College of Health Science, Korea University, Seoul, Republic of Korea</span></div><div property="affiliation" typeof="Organization"><span class="label">12</span><span id="aff12" property="name">Program in Bioinformatics and Integrative Genomics, Harvard Medical School, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">13</span><span id="aff13" property="name">The Lundbeck Foundation Initiative for Integrative Psychiatric Research, iPSYCH, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span class="label">14</span><span id="aff14" property="name">Center for Genomics and Personalized Medicine, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span class="label">15</span><span id="aff15" property="name">Department of Biomedicine – Human Genetics, Aarhus University, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span class="label">16</span><span id="aff16" property="name">Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, PA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">17</span><span id="aff17" property="name">Department of Neurology, University of California, San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">18</span><span id="aff18" property="name">The Eli and Edythe Broad Center of Regeneration Medicine and Stem Cell Research, University of California, San Francisco, San Francisco, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">19</span><span id="aff19" property="name">Computer Engineering Department, Bilkent University, Ankara, Turkey</span></div><div property="affiliation" typeof="Organization"><span class="label">20</span><span id="aff20" property="name">Center for Autism Research and Translation, University of California, Irvine, Irvine, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">21</span><span id="aff21" property="name">MIND (Medical Investigation of Neurodevelopmental Disorders) Institute, University of California, Davis, Davis, CA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">22</span><span id="aff22" property="name">Division of Genetics, Boston Children’s Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">23</span><span id="aff23" property="name">Division of Developmental Medicine, Boston Children’s Hospital, Boston, MA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">24</span><span id="aff24" property="name">Sorbonne Université, INSERM, CNRS, Neuroscience Paris Seine, Institut de Biologie Paris Seine, Paris, France</span></div><div property="affiliation" typeof="Organization"><span class="label">25</span><span id="aff25" property="name">Institute for Juvenile Research, Department of Psychiatry, University of Illinois at Chicago, Chicago, IL, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">26</span><span id="aff26" property="name">Department of Psychiatry, School of Medicine, Trinity College Dublin, Dublin, Ireland</span></div><div property="affiliation" typeof="Organization"><span class="label">27</span><span id="aff27" property="name">Vanderbilt Genetics Institute, Vanderbilt University School of Medicine, Nashville, TN, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">28</span><span id="aff28" property="name">Department of Molecular Physiology and Biophysics and Psychiatry, Vanderbilt University School of Medicine, Nashville, TN, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">29</span><span id="aff29" property="name">National Institute of Mental Health, NIH, Bethesda, MD, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">30</span><span id="aff30" property="name">Department of Human Genetics, Emory University School of Medicine, Atlanta, GA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">31</span><span id="aff31" property="name">Bioinformatics Research Centre, Aarhus University, Aarhus, Denmark</span></div><div property="affiliation" typeof="Organization"><span class="label">32</span><span id="aff32" property="name">Computational Biology Department, Carnegie Mellon University, Pittsburgh, PA, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">33</span><span id="aff33" property="name">Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</span></div><div property="affiliation" typeof="Organization"><span class="label">34</span><span id="aff34" property="name">Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">35</span><span id="aff35" property="name">Friedman Brain Institute, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div><div property="affiliation" typeof="Organization"><span class="label">36</span><span id="aff36" property="name">Department of Neuroscience, Icahn School of Medicine at Mount Sinai, New York, NY, USA</span></div></div><div class="core-authors-notes"><div role="doc-footnote" data-has="label"><div class="label">37</div><div id="fn1"><div id="ntpara0010" role="paragraph">These authors contributed equally</div></div></div><div role="doc-footnote" data-has="label"><div class="label">38</div><div id="fn2"><div id="ntpara0015" role="paragraph">Senior author</div></div></div><div role="doc-footnote" data-has="label"><div class="label">39</div><div id="fn3"><div id="ntpara0020" role="paragraph">Lead Contact</div></div></div></div></div><div id="core-content-info" role="region" data-expandable="item" style="display: none;"><div class="history"><div>Publication History:</div><div><span>Received March 20, 2019</span>; <span>Revised July 8, 2019</span>; <span>Accepted December 24, 2019</span>; <span>Published online January 23, 2020</span></div></div><div class="core-links"><span class="doi">DOI: <a href="https://doi.org/10.1016/j.cell.2019.12.036" property="sameAs" target="_blank">10.1016/j.cell.2019.12.036 <span class="sr-only">External Link</span></a></span><span class="science-direct">Also available on <a href="https://www.sciencedirect.com/science/article/pii/S0092867419313984" target="_blank">ScienceDirect <span class="sr-only">External Link</span></a></span></div><div role="paragraph">Copyright: © 2020 Elsevier Inc.</div><div class="core-links"><span>User License: <a href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank">Elsevier user license</a> | <a href="https://www.elsevier.com/open-access" target="_blank">Elsevier's open access license policy</a></span></div><div class="core-crossmark-badge"><a data-target="crossmark" href="#" title="Check for updates on crossmark" data-doi="10.1016/j.cell.2019.12.036" data-id="article-info-crossmark" class="crossmark__link" style="cursor: pointer;"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 290 58" height="22px" role="presentation"><path fill="url(#horizontal_crossmark)" stroke="#948F8F" stroke-miterlimit="10" stroke-width=".75" d="M287 1H3a2 2 0 0 0-2 2v52a2 2 0 0 0 2 2h284a2 2 0 0 0 2-2V3a2 2 0 0 0-2-2Z"></path><path fill="#535353" d="M80.3 23.58a6.395 6.395 0 0 0-.55-1.43 3.92 3.92 0 0 0-2.19-1.86 5.099 5.099 0 0 0-1.71-.29 5 5 0 0 0-2.46.58 4.831 4.831 0 0 0-1.69 1.54 6.921 6.921 0 0 0-1 2.19 10.16 10.16 0 0 0 0 5 6.921 6.921 0 0 0 1 2.19 4.83 4.83 0 0 0 1.69 1.5 5 5 0 0 0 2.46.58 4.44 4.44 0 0 0 1.85-.36 4.12 4.12 0 0 0 1.38-1c.39-.436.695-.942.9-1.49A7.094 7.094 0 0 0 80.4 29h3a8.46 8.46 0 0 1-.69 2.92 7.179 7.179 0 0 1-1.59 2.29 7 7 0 0 1-2.35 1.49 8.14 8.14 0 0 1-3 .52 8.43 8.43 0 0 1-3.59-.74 7.88 7.88 0 0 1-2.69-2 9.001 9.001 0 0 1-1.69-3 11.41 11.41 0 0 1 0-7.23 9.001 9.001 0 0 1 1.69-3 8.001 8.001 0 0 1 2.69-2 8.3 8.3 0 0 1 3.59-.75 9.44 9.44 0 0 1 2.83.41c.866.242 1.68.642 2.4 1.18A6.229 6.229 0 0 1 82.65 21a6.66 6.66 0 0 1 .78 2.58H80.3ZM85.9 17.9h2.85v6.63h.05a4.15 4.15 0 0 1 1.56-1.46 4.61 4.61 0 0 1 2.31-.59 4.83 4.83 0 0 1 3.33 1.1 4.21 4.21 0 0 1 1.22 3.3v8.88H94.4v-8.13a3.51 3.51 0 0 0-.65-2.21 2.36 2.36 0 0 0-1.87-.69 3.1 3.1 0 0 0-1.3.26 2.821 2.821 0 0 0-1 .73A3.41 3.41 0 0 0 89 26.8a3.88 3.88 0 0 0-.23 1.33v7.63H85.9V17.9ZM102.47 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7A5.349 5.349 0 0 1 109.7 35a5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4A5.953 5.953 0 0 1 100 32a8.748 8.748 0 0 1-.41-2.75 7.768 7.768 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.264.905.352 1.852.26 2.79h-9.53Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.594 3.594 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM122.88 27.15a2.666 2.666 0 0 0-1-1.81 3.14 3.14 0 0 0-2-.61c-.4.005-.798.07-1.18.19-.435.136-.83.376-1.15.7a3.92 3.92 0 0 0-.87 1.43 6.783 6.783 0 0 0-.35 2.39c.002.539.065 1.076.19 1.6.113.509.316.993.6 1.43.273.41.635.752 1.06 1a3 3 0 0 0 1.58.39 2.872 2.872 0 0 0 2.06-.77 3.596 3.596 0 0 0 1-2.18h2.85a6.26 6.26 0 0 1-1.91 3.86 5.83 5.83 0 0 1-4 1.34 6.61 6.61 0 0 1-2.69-.51 5.695 5.695 0 0 1-2-1.4 6.001 6.001 0 0 1-1.21-2.11 8.279 8.279 0 0 1-.41-2.65 8.987 8.987 0 0 1 .4-2.72 6.237 6.237 0 0 1 1.2-2.21 5.614 5.614 0 0 1 2-1.47 6.7 6.7 0 0 1 2.79-.54 7.801 7.801 0 0 1 2.14.29 5.634 5.634 0 0 1 1.8.86c.529.386.969.88 1.29 1.45a5 5 0 0 1 .6 2.07h-2.85l.06-.02ZM128.13 17.9H131V28l5.15-5.22h3.5l-5 4.75 5.42 8.17h-3.47l-4-6.27-1.6 1.7v4.63h-2.85V17.9h-.02ZM147.15 22.83h2.13v-1.08a5.505 5.505 0 0 1 .3-2c.161-.456.436-.862.8-1.18.33-.273.721-.462 1.14-.55.44-.094.89-.141 1.34-.14a8.158 8.158 0 0 1 2 .18v2.24a4.113 4.113 0 0 0-.59-.11 6.343 6.343 0 0 0-.74 0 1.74 1.74 0 0 0-1 .27 1.204 1.204 0 0 0-.41 1.08v1.33h2.42V25h-2.42v10.8h-2.85V25h-2.12v-2.17ZM162.23 36.1a6.999 6.999 0 0 1-2.76-.51 5.877 5.877 0 0 1-2-1.41 5.998 5.998 0 0 1-1.33-2.18 8.775 8.775 0 0 1 0-5.48 6.008 6.008 0 0 1 1.27-2.15 5.894 5.894 0 0 1 2-1.41A7.71 7.71 0 0 1 165 23a5.894 5.894 0 0 1 2 1.41 6.003 6.003 0 0 1 1.28 2.15c.585 1.78.585 3.7 0 5.48a5.99 5.99 0 0 1-3.28 3.54 7.002 7.002 0 0 1-2.77.52Zm0-2.25a3.268 3.268 0 0 0 1.65-.4c.447-.262.83-.621 1.12-1.05.304-.444.528-.938.66-1.46a6.581 6.581 0 0 0 0-3.31 4.416 4.416 0 0 0-.66-1.46 3.526 3.526 0 0 0-1.15-1 3.6 3.6 0 0 0-3.3 0 3.526 3.526 0 0 0-1.15 1 4.414 4.414 0 0 0-.66 1.46 6.558 6.558 0 0 0 0 3.31c.132.522.356 1.016.66 1.46a3.47 3.47 0 0 0 1.15 1.05 3.26 3.26 0 0 0 1.68.4ZM171.13 22.83h2.67v2.5c.097-.362.264-.701.49-1 .251-.341.547-.647.88-.91a4.404 4.404 0 0 1 1.14-.66 3.41 3.41 0 0 1 1.28-.25c.23-.013.46-.013.69 0h.39v2.79l-.61-.09a5.173 5.173 0 0 0-.61 0 3.278 3.278 0 0 0-1.36.29 3.131 3.131 0 0 0-1.11.85 4.171 4.171 0 0 0-.75 1.39 6 6 0 0 0-.28 1.9v6.15h-2.85V22.83h.03ZM198.22 35.75h-2.8V34a3.907 3.907 0 0 1-1.56 1.56 4.25 4.25 0 0 1-2.11.59 4.742 4.742 0 0 1-3.75-1.31 5.537 5.537 0 0 1-1.12-3.84v-8.2h2.85v7.92a3.503 3.503 0 0 0 .65 2.4 2.38 2.38 0 0 0 1.83.7 3.599 3.599 0 0 0 1.5-.27 2.66 2.66 0 0 0 1-.74c.259-.326.444-.705.54-1.11a5.79 5.79 0 0 0 .16-1.4v-7.5h2.85v12.95h-.04ZM201.3 22.83h2.7v1.75a3.523 3.523 0 0 1 1.72-1.58 5.568 5.568 0 0 1 2.33-.49 6.17 6.17 0 0 1 2.66.54 5.246 5.246 0 0 1 1.89 1.47c.515.65.899 1.394 1.13 2.19.257.87.385 1.773.38 2.68.001.851-.114 1.7-.34 2.52a6.568 6.568 0 0 1-1 2.16 5.002 5.002 0 0 1-4.21 2.06c-.419 0-.837-.036-1.25-.11a5.39 5.39 0 0 1-1.2-.36A4.737 4.737 0 0 1 205 35a3.567 3.567 0 0 1-.81-.91v6.45h-2.89V22.83Zm10 6.47c0-.574-.078-1.146-.23-1.7a4.606 4.606 0 0 0-.67-1.46 3.475 3.475 0 0 0-1.12-1 3.06 3.06 0 0 0-1.55-.39A3.141 3.141 0 0 0 205 26a5.518 5.518 0 0 0-1 3.3c-.004.611.077 1.22.24 1.81.138.523.38 1.012.71 1.44a3.55 3.55 0 0 0 1.14 1 3.242 3.242 0 0 0 1.54.35 3.18 3.18 0 0 0 1.65-.4c.438-.25.817-.59 1.11-1 .299-.44.513-.931.63-1.45a7.38 7.38 0 0 0 .23-1.75h.05ZM228.67 35.75H226V34a3.38 3.38 0 0 1-1.67 1.61 5.668 5.668 0 0 1-2.33.49 6.17 6.17 0 0 1-2.66-.54 5.307 5.307 0 0 1-1.89-1.46 6.254 6.254 0 0 1-1.12-2.19 9.48 9.48 0 0 1-.37-2.71 8.67 8.67 0 0 1 .47-3 6.058 6.058 0 0 1 1.26-2.1 4.89 4.89 0 0 1 1.8-1.21 5.738 5.738 0 0 1 2.06-.39c.412 0 .824.037 1.23.11.413.073.815.194 1.2.36.38.166.736.381 1.06.64.32.255.594.562.81.91V17.9h2.85v17.85h-.03Zm-10-6.33c-.001.55.069 1.098.21 1.63.13.511.35.996.65 1.43.288.414.668.757 1.11 1a3.241 3.241 0 0 0 1.6.38 3.199 3.199 0 0 0 1.64-.4 3.398 3.398 0 0 0 1.13-1.05c.297-.447.517-.94.65-1.46.14-.542.211-1.1.21-1.66a5.227 5.227 0 0 0-1-3.35 3.175 3.175 0 0 0-2.61-1.2 3.23 3.23 0 0 0-1.69.41 3.478 3.478 0 0 0-1.12 1.07 4.494 4.494 0 0 0-.62 1.5 7.871 7.871 0 0 0-.11 1.7h-.05ZM242.6 32.88c-.018.258.03.516.14.75a.577.577 0 0 0 .54.23h.3c.135-.002.269-.019.4-.05v2l-.39.11-.49.08-.5.08h-.42a2.75 2.75 0 0 1-1.45-.35 1.731 1.731 0 0 1-.75-1.23 4.998 4.998 0 0 1-2.09 1.2 8.224 8.224 0 0 1-2.39.38 5.861 5.861 0 0 1-1.67-.24 4.394 4.394 0 0 1-1.41-.7 3.35 3.35 0 0 1-1-1.18 3.624 3.624 0 0 1-.36-1.66 3.84 3.84 0 0 1 .44-2 3.246 3.246 0 0 1 1.15-1.17 5.195 5.195 0 0 1 1.6-.61c.593-.127 1.19-.223 1.79-.29a14.77 14.77 0 0 1 1.47-.21 6.896 6.896 0 0 0 1.24-.21c.318-.077.611-.236.85-.46a1.286 1.286 0 0 0 .31-.94 1.458 1.458 0 0 0-.26-.9A1.759 1.759 0 0 0 239 25a2.807 2.807 0 0 0-.86-.25 6.941 6.941 0 0 0-.9-.06 3.58 3.58 0 0 0-2 .5 2 2 0 0 0-.87 1.55h-2.85c.021-.73.227-1.442.6-2.07.338-.54.797-.996 1.34-1.33a5.475 5.475 0 0 1 1.84-.7c.692-.134 1.395-.2 2.1-.2.632 0 1.262.067 1.88.2a5.337 5.337 0 0 1 1.66.65 3.6 3.6 0 0 1 1.19 1.16c.314.525.47 1.129.45 1.74v6.65l.02.04Zm-2.85-3.6a3.45 3.45 0 0 1-1.6.51c-.667.06-1.3.147-1.9.26a6.137 6.137 0 0 0-.87.21 2.64 2.64 0 0 0-.75.38c-.22.164-.395.38-.51.63a2.129 2.129 0 0 0-.19.94c-.011.29.085.575.27.8.182.215.407.388.66.51.269.128.555.216.85.26.277.05.558.076.84.08a4.63 4.63 0 0 0 1-.12 3.418 3.418 0 0 0 1-.43 2.73 2.73 0 0 0 .81-.76 1.93 1.93 0 0 0 .32-1.14v-2.14l.07.01ZM244.37 22.83h2.15V19h2.85v3.88h2.58V25h-2.58v6.9a7.064 7.064 0 0 0 0 .77c.016.194.074.381.17.55a.84.84 0 0 0 .41.34 2 2 0 0 0 .75.11h.6c.203-.01.404-.04.6-.09v2.2l-.93.1a8.403 8.403 0 0 1-.92 0 6.33 6.33 0 0 1-1.81-.21 2.421 2.421 0 0 1-1.08-.62 2.15 2.15 0 0 1-.52-1 7.791 7.791 0 0 1-.16-1.42V25h-2.15v-2.17h.04ZM256.22 30c-.002.491.069.98.21 1.45.131.448.348.866.64 1.23a3.08 3.08 0 0 0 1.08.84 3.543 3.543 0 0 0 1.55.31 3.431 3.431 0 0 0 2-.54c.541-.402.94-.966 1.14-1.61h2.7a5.352 5.352 0 0 1-2.1 3.26 5.715 5.715 0 0 1-1.74.85 7 7 0 0 1-2 .29 6.848 6.848 0 0 1-2.7-.5 5.577 5.577 0 0 1-2-1.4 5.947 5.947 0 0 1-1.21-2.18 8.754 8.754 0 0 1-.41-2.75 7.758 7.758 0 0 1 .44-2.61 6.67 6.67 0 0 1 1.25-2.17 5.863 5.863 0 0 1 4.56-2 5.79 5.79 0 0 1 2.74.64c.783.41 1.466.986 2 1.69a6.824 6.824 0 0 1 1.16 2.41c.243.916.308 1.87.19 2.81h-9.5V30Zm6.65-1.87a4.605 4.605 0 0 0-.29-1.3c-.148-.4-.368-.77-.65-1.09a3.22 3.22 0 0 0-1-.75 3 3 0 0 0-1.31-.29 3.431 3.431 0 0 0-1.36.26 3.108 3.108 0 0 0-1 .73 3.575 3.575 0 0 0-.7 1.09 3.798 3.798 0 0 0-.29 1.35h6.65-.05ZM270 31.6a2.154 2.154 0 0 0 .95 1.75c.607.35 1.3.524 2 .5.304-.002.608-.022.91-.06a3.943 3.943 0 0 0 1-.24c.286-.105.54-.28.74-.51a1.23 1.23 0 0 0 .26-.89 1.261 1.261 0 0 0-.4-.9 2.779 2.779 0 0 0-1-.56 9.315 9.315 0 0 0-1.34-.36l-1.5-.33a14.219 14.219 0 0 1-1.54-.42 4.937 4.937 0 0 1-1.33-.68 3.12 3.12 0 0 1-.94-1.09 3.472 3.472 0 0 1-.35-1.64 2.94 2.94 0 0 1 .51-1.76 3.913 3.913 0 0 1 1.3-1.15 5.66 5.66 0 0 1 1.75-.61 10.316 10.316 0 0 1 1.84-.17 8.418 8.418 0 0 1 1.91.21 5.19 5.19 0 0 1 1.65.69c.496.318.916.741 1.23 1.24.341.559.55 1.188.61 1.84h-3a1.783 1.783 0 0 0-.94-1.37 3.892 3.892 0 0 0-1.69-.35 6.093 6.093 0 0 0-.71 0 3.327 3.327 0 0 0-.78.19 1.642 1.642 0 0 0-.61.4 1 1 0 0 0-.25.69 1.093 1.093 0 0 0 .36.85c.294.245.634.428 1 .54.437.153.885.274 1.34.36l1.55.33a20.9 20.9 0 0 1 1.53.42c.479.157.931.386 1.34.68.41.277.751.643 1 1.07.255.494.379 1.045.36 1.6a3.48 3.48 0 0 1-.52 2 4.15 4.15 0 0 1-1.36 1.3 6.19 6.19 0 0 1-1.86.73 9.435 9.435 0 0 1-2 .23 8.823 8.823 0 0 1-2.26-.27 5.265 5.265 0 0 1-1.8-.84 4 4 0 0 1-1.2-1.4 4.445 4.445 0 0 1-.46-2h2.7v-.02Z"></path><path fill="#C72914" d="m21.94 38.74 18.12-12.08v-14.5H21.94v26.58Z"></path><path fill="#EF3340" d="M40.06 38.74 21.94 26.66v-14.5h18.12v26.58Z"></path><path fill="#3EB1C8" d="M31 9a20 20 0 1 0 0 40 20 20 0 0 0 0-40Zm0 35.1a15.1 15.1 0 1 1 0-30.2 15.1 15.1 0 0 1 0 30.2Z"></path><path fill="#FFC72C" d="M42.09 39.23A15.09 15.09 0 0 1 17 23.38l-4.2-2.64a20 20 0 0 0 33.49 21.14l-4.2-2.65Z"></path><defs><linearGradient id="horizontal_crossmark" x1="145" x2="145" y1="7.33" y2="56.82" gradientUnits="userSpaceOnUse"><stop stop-color="#fff"></stop><stop offset="1" stop-color="#c4c4c4"></stop></linearGradient></defs></svg></a></div><div class="core-content-info"></div></div></div></div></div></header><div class="core-header-aside">
    

        


    

            

            

            <div class="cover-image cover-image--axel"><div class="cover-image__thumb"><a href="https://www.cell.com/cell/issue?pii=S0092-8674(19)X0004-5"><img src="/cms/asset/625c1d51-4245-4cb2-8066-7e93c7c4b35f/cov200h.gif" alt="Cover Image - Cell, Volume 180, Issue 3"></a></div></div>

            

        
</div></div><div data-core-wrapper="content"><div id="dummy-sticky-observer"></div><div data-core-nav="header" data-extent="frontmatter"><div class="core-nav-wrapper core-container" aria-label="Article navigation"><div class="navigation-menu-pdf">
    

        


    

            

            

            <div data-widget-id="ecff07e8-0bc4-4533-b1f1-49365e2ead20" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><a href="/action/showPdf?pii=S0092-8674%2819%2931398-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" role="menuitem" class="article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></div>

            

        
</div><div class="core-sections-menu"><button data-id="article-toolbar-showhide" aria-controls="article_sections_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle section navigation menu" style="margin-top: 0px;"><i class="icon-list" aria-hidden="true"></i><span>Outline</span></button><nav id="article_sections_menu" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><div class="heading">Outline<button aria-haspopup="true" aria-controls="article_sections_menu" title="Close outline menu"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div><ul><li><a href="#author-highlights-abstract">Highlights</a></li><li><a href="#author-abstract">Summary</a></li><li><a href="#graphical-abstract">Graphical Abstract</a></li><li><a href="#keywords">Keywords</a></li><li><a href="#sec-1">Introduction</a></li><li><a href="#sec-2">Results</a></li><li><a href="#sec-3">Discussion</a></li><li><a href="#sec-4">STAR★Methods</a></li><li><a href="#sec-5">Consortia</a></li><li><a href="#acknowledgments">Acknowledgments</a></li><li><a href="#supplementary-material">Supplemental Information (5)</a></li><li><a href="#app-2">Supporting Citations</a></li><li><a href="#references">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div><div class="navigation-menu-share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="5f978446-c80a-450c-a185-94d633df2db2" data-widget-name="UX3share" aria-hidden="false" role="group" class="article-tools__dropblock article-tools__share"><button aria-label="Toggle share menu" aria-expanded="false" aria-haspopup="true" role="menuitem" aria-controls="navigation_menu_share_block" aria-labelledby="share-label" class="article-tools__item__share"><i aria-hidden="true" class="icon-article-share"></i><span id="share-label">Share</span></button><div id="navigation_menu_share_block" role="menu" class="share__block axel-share article-tools__dropblock__holder"><h4 class="axel-share__title heading">Share<button title="Close share menu" aria-expanded="false" aria-haspopup="true" aria-controls="navigation_menu_share_block"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2819%2931398-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="f1a208d3-ac5e-4db2-9c3a-40ef97ddfa10" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2019.12.036">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</div><div><button data-id="article-nav-menubar-more" aria-controls="article_more_menu" aria-expanded="false" aria-haspopup="true" aria-label="Toggle more menu"><i aria-hidden="true" class="icon-more-mobile"></i><span>More</span></button><div class="info-panel"><div class="info-panel__left-content"><div class="info-panel__article_tools info-panel__item"><div id="article_more_menu" aria-label="More menu" data-core-nav="more"><div class="heading">More<button title="Close more menu" aria-controls="article_more_menu" aria-haspopup="true"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div>
    

        


    

            

            

            <ul data-widget-id="183a17a2-87b5-4e6f-bb85-6b5cda068c89" data-widget-name="UX3ArticleToolsWidget" class="article-tools"><li class="article-tools__item article-tools__pdf"><a href="/action/showPdf?pii=S0092-8674%2819%2931398-4" title="Standard PDF (opens in a new tab)" target="_blank" aria-hidden="false" class="article-tools__ctrl article-tools__item__pdf"><svg focusable="false" viewBox="0 0 32 32" width="20" height="20" title="PDF" class="pdf-icon"><path id="pdfwhite" d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path id="pdfda0000" d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path id="pdffff9f9" fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path id="pdff91d0a" d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="basic">Download PDF</span><span class="alternative">Download PDF</span></a></li><li><a href="https://www.cell.com/action/showCitFormats?doi=10.1016%2Fj.cell.2019.12.036&amp;pii=S0092-8674%2819%2931398-4" title="Cite" target="_blank" aria-label="Open Cite in new tab" class="article-tools__item__displayExportCitation"><i aria-hidden="true" class="icon-export-Citation"></i><span>Cite</span></a></li><li id="article_tools_share">
    

        


    

            

            

            <!-- Go to https://www.addtoany.com/buttons/customize/ to customize your tools --><script type="text/javascript" defer="defer" src="https://static.addtoany.com/menu/page.js"></script><script type="text/javascript">var a2a_config = a2a_config || {};
a2a_config.thanks = {
    postShare: false,
    ad: false,
};</script><script>var a2a_config = a2a_config || {};
let res = window.location.href.replace(/\(/g, "%28").replace(/\)/g, "%29");
a2a_config.templates = a2a_config.templates || {};

a2a_config.templates.email = {
    body: res
};
</script><div data-widget-id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" data-widget-name="UX3share" class="dropBlock article-tools__dropblock article-tools__share"><button data-db-target-for="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" aria-expanded="false" title="open Share menu" class="article-tools__ctrl share__ctrl"><i aria-hidden="true" class="icon-article-share"></i><span>Share</span></button><div data-db-target-of="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da" id="c3faf0bb-84d9-4c61-a6d8-93bfc77be8da_Pop" class="share__block axel-share dropBlock__holder article-tools__dropblock__holder"><h4 class="axel-share__title">Share</h4><div class="axel-share__inner"><span class="sr-only">Share on</span><ul class="a2a__share-list rlist w-slide--list a2a a2a_kit a2a_kit_size_32 a2a_default_style" data-a2a-url="https://www.cell.com/cell/fulltext/S0092-8674%2819%2931398-4" style="line-height: 32px;"><li class="a2a_listitem_custom"><a role="link" title="Email" class="a2a_button_email" target="_blank" rel="nofollow noopener" href="/#email"><i aria-hidden="true" class="at-icon-wrapper icon-gizmo-email-plus"></i><span>Email</span></a></li><li class="a2a_listitem_custom"><a role="link" title="formerly known as Twitter" class="a2a_button_twitter" target="_blank" rel="nofollow noopener" href="/#twitter"><i aria-hidden="true" class="at-icon-wrapper icon-x-default"></i><span>X</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Facebook" class="a2a_button_facebook" target="_blank" rel="nofollow noopener" href="/#facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook-square-2"></i><span>Facebook</span></a></li><li class="a2a_listitem_custom"><a role="link" title="LinkedIn" class="a2a_button_linkedin" target="_blank" rel="nofollow noopener" href="/#linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin-square"></i><span>LinkedIn</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Sina Weibo" class="a2a_button_sina_weibo" target="_blank" rel="nofollow noopener" href="/#sina_weibo"><i aria-hidden="true" class="at-icon-wrapper icon-weibo"></i><span>Sina Weibo</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Mendeley" class="a2a_button_mendeley" target="_blank" rel="nofollow noopener" href="/#mendeley"><i aria-hidden="true" class="at-icon-wrapper icon-mendeley-2"></i><span>Mendeley</span></a></li><li class="a2a_listitem_custom"><a role="link" title="Bluesky" class="a2a_button_bluesky" target="_blank" rel="nofollow noopener" href="/#bluesky"><i aria-hidden="true" class="at-icon-wrapper icon-bluesky-logo"></i><span>Bluesky</span></a></li><div style="clear: both;"></div></ul><div class="extra-links-drop-zone">
    

        


    

            

            

            <div data-widget-id="ae9bf5c5-a2ee-4f5c-8e41-9b21638b1a9f" data-widget-name="UX3HTMLWidget"><a class="axel-share__extra" target="_blank" href="/personalize/addFavoritePublication?doi=10.1016/j.cell.2019.12.036">
	<i class="icon-reading-list" aria-hidden="true"></i>
	<span>Add to my reading list</span>
</a></div>

            

        
</div></div></div></div>

            

        
</li><li><a href="https://www.cell.com/action/addCitationAlert?doi=10.1016%2Fj.cell.2019.12.036" title="Set Alert" target="_blank" aria-label="Open Set Alert in new tab" class="article-tools__item__displayCreateCitationAlert"><i aria-hidden="true" class="icon-set-alert"></i><span>Set Alert</span></a></li><li><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0092867419313984&amp;orderBeanReset=true&amp;orderSource=Phoenix" title="Get Rights" target="_blank" aria-label="Open Get Rights in new tab" class="article-tools__item__request"><i aria-hidden="true" class="icon-get-rights"></i><span>Get Rights</span></a></li><li><a href="https://www.contentednet.com/salesforce-form/?pii=S0092867419313984&amp;issn=00928674&amp;source=jbs" title="Reprints" target="_blank" aria-label="Open Reprints in new tab" class="article-tools__item__reprints"><i aria-hidden="true" class="icon-article-reprints"></i><span>Reprints</span></a></li></ul>

            

        
</div></div><div class="info-panel__download-issue info-panel__item"><div class="article-header__download-full-issue"><button title="Download full issue" type="submit" formaction="/action/downloadMultiplePdf?action=downloadIssuePdf&amp;journal=cell&amp;volume=180&amp;issue=3&amp;pii=S0092-8674%2819%29X0004-5" class="btn--download-issue btn--cancel download-full-issue"><i class="icon-download-new"></i><span>Download Full Issue</span></button><div class="table-of-content__alert table-of-content__alert--download"><p>Download started</p><a href="#" class="table-of-content__alert--close">Ok</a></div></div></div></div><div class="info-panel__right-content"><div class="info-panel__navigation info-panel__item"><div class="content-navigation"><a href="/cell/fulltext/S0092-8674(20)30061-1" aria-label="Previous article" aria-disabled="false" class="content-navigation__prev"><i aria-hidden="true" class="icon-arrow_l"></i><span>Previous article</span></a><a href="/cell/fulltext/S0092-8674(20)30056-8" aria-label="Next article" aria-disabled="false" class="content-navigation__next"><span>Next article</span><i aria-hidden="true" class="icon-arrow_r"></i></a></div></div></div></div></div></div></div><div class="core-sections-menu-outer"><div class="core-sections-menu"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-show-outline" aria-hidden="true"></i><span>Show Outline</span></button><nav id="article_sections_menu-desktop" aria-label="Contents" data-core-nav="article" style="margin-top: 0px;"><button aria-controls="article_sections_menu-desktop" aria-haspopup="true" aria-expanded="false" aria-label="Toggle sections menu" data-id="article-toolbar-showhide"><i class="icon-hide-outline" aria-hidden="true"></i><span>Hide Outline</span></button><ul><li><a href="#author-highlights-abstract" aria-current="true">Highlights</a></li><li><a href="#author-abstract" aria-current="false">Summary</a></li><li><a href="#graphical-abstract" aria-current="false">Graphical Abstract</a></li><li><a href="#keywords" aria-current="false">Keywords</a></li><li><a href="#sec-1" aria-current="false">Introduction</a></li><li><a href="#sec-2" aria-current="false">Results</a></li><li><a href="#sec-3" aria-current="false">Discussion</a></li><li><a href="#sec-4" aria-current="false">STAR★Methods</a></li><li><a href="#sec-5" aria-current="false">Consortia</a></li><li><a href="#acknowledgments" aria-current="false">Acknowledgments</a></li><li><a href="#supplementary-material" aria-current="false">Supplemental Information (5)</a></li><li><a href="#app-2" aria-current="false">Supporting Citations</a></li><li><a href="#references" aria-current="false">References</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-metrics">Article metrics</a></li><li role="none" class="mobile-only"><a role="menuitem" href="#core-collateral-relatedArticles">Related Articles</a></li></ul></nav></div></div><div><div class="core-content-inner"><div class="core-linked-content"></div><div id="abstracts" data-extent="frontmatter"><div class="core-container"><section id="author-highlights-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Highlights</h2><div id="abspara0020" role="paragraph"><div id="ulist0010" role="list"><div id="u0010" role="listitem"><div class="label">•</div><div class="content"><div id="p0010" role="paragraph">102 genes implicated in risk for autism spectrum disorder (ASD genes, FDR&nbsp;≤ 0.1)</div></div></div><div id="u0015" role="listitem"><div class="label">•</div><div class="content"><div id="p0015" role="paragraph">Most are expressed and enriched early in excitatory and inhibitory neuronal lineages</div></div></div><div id="u0020" role="listitem"><div class="label">•</div><div class="content"><div id="p0020" role="paragraph">Most affect synapses or regulate other genes; how these roles dovetail is unknown</div></div></div><div id="u0025" role="listitem"><div class="label">•</div><div class="content"><div id="p0025" role="paragraph">Some ASD genes alter early development broadly, others appear more specific to ASD</div></div></div></div></div></section><section id="author-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Summary</h2><div id="abspara0010" role="paragraph">We present the largest exome sequencing study of autism spectrum disorder (ASD) to date (n&nbsp;= 35,584 total samples, 11,986 with ASD). Using an enhanced analytical framework to integrate <i>de novo</i> and case-control rare variation, we identify 102 risk genes at a false discovery rate of 0.1 or less. Of these genes, 49 show higher frequencies of disruptive <i>de novo</i> variants in individuals ascertained to have severe neurodevelopmental delay, whereas 53 show higher frequencies in individuals ascertained to have ASD; comparing ASD cases with mutations in these groups reveals phenotypic differences. Expressed early in brain development, most risk genes have roles in regulation of gene expression or neuronal communication (i.e., mutations effect neurodevelopmental and neurophysiological changes), and 13 fall within loci recurrently hit by copy number variants. In cells from the human cortex, expression of risk genes is enriched in excitatory and inhibitory neuronal lineages, consistent with multiple paths to an excitatory-inhibitory imbalance underlying ASD.</div></section><section id="graphical-abstract" property="abstract" typeof="Text" role="doc-abstract"><h2 property="name">Graphical Abstract</h2><div id="abspara0015" role="paragraph"><div class="figure-wrap"><figure id="undfig1" class="graphic" data-figureviewer-ignore="true"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/3a39d5bd-cfb9-44a3-9ee7-a308a7cf1103/main.assets/fx1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="fx1" alt="Graphical abstract undfig1" src="/cms/10.1016/j.cell.2019.12.036/asset/9e9d3ab2-3cb6-46aa-8e12-041aff8e5a0a/main.assets/fx1.jpg" height="375" width="375" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/3a39d5bd-cfb9-44a3-9ee7-a308a7cf1103/main.assets/fx1_lrg.jpg" loading="lazy"></a></figure></div></div></section></div></div><div class="core-linked-content"></div><section id="keywords" property="keywords"><h2>Keywords</h2><ol><li><a href="/action/doSearch?AllField=&quot;autism+spectrum+disorder&quot;" alt="Search on this keyword" rel="nofollow">autism spectrum disorder</a></li><li><a href="/action/doSearch?AllField=&quot;exome+sequencing&quot;" alt="Search on this keyword" rel="nofollow">exome sequencing</a></li><li><a href="/action/doSearch?AllField=&quot;genetics&quot;" alt="Search on this keyword" rel="nofollow">genetics</a></li><li><a href="/action/doSearch?AllField=&quot;neurodevelopment&quot;" alt="Search on this keyword" rel="nofollow">neurodevelopment</a></li><li><a href="/action/doSearch?AllField=&quot;liability&quot;" alt="Search on this keyword" rel="nofollow">liability</a></li><li><a href="/action/doSearch?AllField=&quot;cell+type&quot;" alt="Search on this keyword" rel="nofollow">cell type</a></li><li><a href="/action/doSearch?AllField=&quot;cytoskeleton&quot;" alt="Search on this keyword" rel="nofollow">cytoskeleton</a></li><li><a href="/action/doSearch?AllField=&quot;excitatory+neurons&quot;" alt="Search on this keyword" rel="nofollow">excitatory neurons</a></li><li><a href="/action/doSearch?AllField=&quot;inhibitory+neurons&quot;" alt="Search on this keyword" rel="nofollow">inhibitory neurons</a></li><li><a href="/action/doSearch?AllField=&quot;excitatory-inhibitory+balance&quot;" alt="Search on this keyword" rel="nofollow">excitatory-inhibitory balance</a></li></ol></section><div class="core-linked-content"></div><div class="core-linked-content"></div><section id="bodymatter" data-extent="bodymatter" property="articleBody" typeof="Text"><div class="core-container"><section id="sec-1"><h2>Introduction</h2><div id="p0030" role="paragraph">Rare inherited and <i>de novo</i> variants are major contributors to individual risk for autism spectrum disorder (ASD) (<span class="dropBlock reference-citations"><a id="body-ref-sref92-1" href="#" role="doc-biblioref" data-xml-rid="bib21 bib38 bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-1" aria-controls="sref92-1">De Rubeis et&nbsp;al., 2014; Iossifov et&nbsp;al., 2014; Sanders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Iossifov, I. ∙ O’Roak, B.J. ∙ Sanders, S.J. ...</div><div><strong>The contribution of de novo coding mutations to autism spectrum disorder</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:216-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13908" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912101541" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13908&amp;pmid=25363768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). When such rare variation disrupts a gene in individuals with ASD more often than expected by chance, it implicates that gene in risk (<span class="dropBlock reference-citations"><a id="body-ref-sref35-1" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-1" aria-controls="sref35-1">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These risk genes provide insight into the underpinnings of ASD both individually (<span class="dropBlock reference-citations"><a id="body-ref-sref5" href="#" role="doc-biblioref" data-xml-rid="bib4 bib5" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref5" aria-controls="sref5">Ben-Shalom et&nbsp;al., 2017; Bernier et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">4.</div><div>Ben-Shalom, R. ∙ Keeshen, C.M. ∙ Berrios, K.N. ...</div><div><strong>Opposing effects on NaV1.2 function underlie differences between SCN2A variants observed in individuals with autism spectrum disorder or infantile seizures</strong></div><div><em>Biol. Psychiatry.</em> 2017; <strong>82</strong>:224-232</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.01.009&amp;cf=fulltext&amp;site=bps-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.01.009&amp;cf=pdf&amp;site=bps-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85014039999" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28256214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biopsych.2017.01.009&amp;pmid=28256214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">5.</div><div>Bernier, R. ∙ Golzio, C. ∙ Xiong, B. ...</div><div><strong>Disruptive CHD8 mutations define a subtype of autism early in development</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:263-276</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2014.06.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2014.06.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84904635209" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24998929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.06.017&amp;pmid=24998929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and <i>en masse</i> (<span class="dropBlock reference-citations"><a id="body-ref-sref109-1" href="#" role="doc-biblioref" data-xml-rid="bib21 bib88 bib92 bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-1" aria-controls="sref109-1">De Rubeis et&nbsp;al., 2014; Ruzzo et&nbsp;al., 2019; Sanders et&nbsp;al., 2015; Willsey et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref109-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Ruzzo, E.K. ∙ Pérez-Cano, L. ∙ Jung, J.Y. ...</div><div><strong>Inherited and de novo genetic risk for autism impacts shared networks</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:850-866.e26</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2019.07.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2019.07.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85070184891" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.015&amp;pmid=31398340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Willsey, A.J. ∙ Sanders, S.J. ∙ Li, M. ...</div><div><strong>Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:997-1007</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889583293" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.020&amp;pmid=24267886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). However, fundamental questions about the altered neurodevelopment and altered neurophysiology in ASD—including when it occurs, where, and in what cell types—remain poorly resolved.</div><div id="p0035" role="paragraph">Here we present the largest exome sequencing study in ASD to date. We assembled a cohort of 35,584 samples, including 11,986 with ASD. We introduce an enhanced Bayesian analytic framework that incorporates recently developed gene- and variant-level scores of evolutionary constraint of genetic variation, and we use it to identify 102 ASD-associated genes (false discovery rate [FDR] ≤ 0.1). Because ASD is often one of a constellation of symptoms of neurodevelopmental delay (NDD), we identify subsets of the 102 ASD-associated genes that have disruptive <i>de novo</i> variants more often in NDD-ascertained or ASD-ascertained cohorts. We also consider the cellular function of ASD-associated genes and, by examining extant data from single cells in the developing human cortex, (1) show that their expression is enriched in maturing and mature excitatory and inhibitory neurons from midfetal development onward, (2) confirm their role in neuronal communication or regulation of gene expression, and (3) show that these functions are separable. Together, these insights form an important step forward in elucidating the neurobiology of ASD.</div></section><section id="sec-2"><h2>Results</h2><section id="sec-2-1"><h3>Dataset</h3><div id="p0040" role="paragraph">We analyzed whole-exome sequence (WES) data from 35,584 samples that passed our quality control procedures (<a id="crosref0485" href="#sec-4">STAR Methods</a>): 21,219 family-based samples (6,430 ASD cases, 2,179 unaffected siblings, and both parents) and 14,365 case-control samples (5,556 ASD cases, 8,809 controls) (<a id="crosref0490" href="#figs1">Figure&nbsp;S1</a>; <a id="crosref0495" href="#mmc1">Table S1</a>). Of these, 6,197 samples were newly sequenced by our consortium (1,908 cases with parents, 274 additional cases, 25 controls) and 11,265 samples were newly incorporated (416&nbsp;cases with parents, plus 4,811 additional cases and 5,214 controls from the Danish iPSYCH study; <span class="dropBlock reference-citations"><a id="body-ref-sref93-1" href="#" role="doc-biblioref" data-xml-rid="bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-1" aria-controls="sref93-1">Satterstrom et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref93-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Satterstrom, F.K. ∙ Walters, R.K. ∙ Singh, T. ...</div><div><strong>ASD and ADHD have a similar burden of rare protein-truncating variants</strong></div><div><em>bioRxiv.</em> 2018;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/277707" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85095617614" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F277707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div class="figure-wrap"><figure id="figs1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/826d7e93-97ca-424e-beb4-f548a879b2b5/main.assets/figs1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs1" src="/cms/10.1016/j.cell.2019.12.036/asset/7cbb96f5-ec83-4489-a5ad-df093a4a3147/main.assets/figs1.jpg" height="942" width="668" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/826d7e93-97ca-424e-beb4-f548a879b2b5/main.assets/figs1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S1</span> <span class="figure__title__text">Data Processing for Samples with Read-Level Exome Data, Related to <a id="crosref1830" href="#fig1">Figure&nbsp;1</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0210" role="paragraph">This figure describes the variant, genotype, and sample quality control steps that were applied to the raw joint-genotyped variant calls (VCF) to generate the <i>de&nbsp;novo</i>, case-control, and inherited variants used in downstream analyses. Low-complexity regions were defined by <span class="dropBlock reference-citations"><a id="body-ref-sref53-1" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-1" aria-controls="sref53-1">Li (2014)</a><div class="dropBlock__holder" data-db-target-of="sref53-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Li, H.</div><div><strong>Toward better understanding of artifacts in variant calling from high-coverage samples</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:2843-2851</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84913546864" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24974202/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu356&amp;pmid=24974202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. VQSR, estimated chimeric reads, and estimated contamination were calculated by the Genome Analysis Toolkit (GATK); relatedness values were calculated by KING (<span class="dropBlock reference-citations"><a id="body-ref-sref62-1" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-1" aria-controls="sref62-1">Manichaikul et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref62-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Manichaikul, A. ∙ Mychaleckyj, J.C. ∙ Rich, S.S. ...</div><div><strong>Robust relationship inference in genome-wide association studies</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2867-2873</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq559" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-78149265272" target="_blank" aria-label="Open Scopus in new tab">Scopus (539)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20926424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq559&amp;pmid=20926424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); most data handling occurred in Hail 0.1 (<a id="intref0175" href="http://hail.is" target="_blank">http://hail.is</a>; <a id="intref0180" href="https://github.com/hail-is/hail" target="_blank">https://github.com/hail-is/hail</a>). Metrics from ExAC and gnomAD were used to exclude common variants. Abbreviations: VCF: Variant Call Format; QC: Quality control; VQSR: Variant Quality Score Recalibration; DP: read depth; AB: allele balance; GQ: genotype quality; PL: Phred-scaled likelihood of the genotype; PAR: pseudoautosomal region; SD: standard deviation; WB: whole-blood derived DNA; CL: cell-line derived DNA; AF: allele frequency; AC: allele count; VQSLOD: variant quality score log-odds.</div></div></div></div></figcaption></figure></div><div id="p0045" role="paragraph">From the family-based data, we identified 9,345 rare <i>de novo</i> variants in protein-coding exons (allele frequency ≤ 0.1% in our dataset and non-psychiatric subsets of reference databases): 63% of cases and 59% of unaffected siblings carried at least one such variant (4,073 of 6,430 and 1,294 of 2,179, respectively; <a id="crosref0505" href="#mmc1">Table S1</a>; <a id="crosref0510" href="#figs1">Figure&nbsp;S1</a>). For inherited and case-control analyses, we included variants with an allele count of no more than five in our dataset or a reference database (<a id="crosref0515" href="#sec-4">STAR Methods</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref51-1" href="#" role="doc-biblioref" data-xml-rid="bib46 bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-1" aria-controls="sref51-1">Kosmicki et&nbsp;al., 2017; Lek et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-2"><h3>Effect of Genetic Variants on ASD Risk</h3><div id="p0050" role="paragraph">Because protein-truncating variants (PTVs; nonsense, frameshift, and essential splice site variants) show a greater difference in burden between ASD cases and controls than missense variants, their average effect on liability must be larger (<span class="dropBlock reference-citations"><a id="body-ref-sref35-2" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-2" aria-controls="sref35-2">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Measures of functional severity assessing evolutionary constraint against deleterious genetic variation, such as the “probability of loss-of-function intolerance” (pLI) score (<span class="dropBlock reference-citations"><a id="body-ref-sref51-2" href="#" role="doc-biblioref" data-xml-rid="bib46 bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-2" aria-controls="sref51-2">Kosmicki et&nbsp;al., 2017; Lek et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and the integrated “missense badness, PolyPhen-2, constraint” (MPC) score (<span class="dropBlock reference-citations"><a id="body-ref-sref90-1" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-1" aria-controls="sref90-1">Samocha et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref90-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Samocha, K.E. ∙ Kosmicki, J.A. ∙ Karczewski, K.J. ...</div><div><strong>Regional missense constraint improves variant deleteriousness prediction</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/148353" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85093588144" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F148353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), can further delineate variant classes with higher burden. Therefore, we divided the list of rare autosomal genetic variants into seven tiers of predicted functional severity: three tiers for PTVs by pLI score (≥0.995, 0.5–0.995, 0–0.5) in order of decreasing expected effect; likewise, three tiers for missense variants by MPC score (≥2, 1–2, 0–1); and a single tier for synonymous variants, expected to have minimal effect. We further divided variants by their inheritance pattern: <i>de novo</i>, inherited, and case-control. Because ASD is associated with reduced fecundity (<span class="dropBlock reference-citations"><a id="body-ref-sref78" href="#" role="doc-biblioref" data-xml-rid="bib78" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref78" aria-controls="sref78">Power et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref78"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">75.</div><div>Power, R.A. ∙ Kyaga, S. ∙ Uher, R. ...</div><div><strong>Fecundity of patients with schizophrenia, autism, bipolar disorder, depression, anorexia nervosa, or substance abuse vs their unaffected siblings</strong></div><div><em>JAMA Psychiatry.</em> 2013; <strong>70</strong>:22-30</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamapsychiatry.2013.268" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84871964302" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23147713/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Fjamapsychiatry.2013.268&amp;pmid=23147713" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), variation associated with it is subject to natural selection. Inherited variation has survived at least one generation of viability and fecundity selection in the parental generation whereas <i>de novo</i> variation in offspring has not. Thus, on average, <i>de novo</i> mutations are exposed to less selective pressure and could mediate substantial risk for ASD. This expectation is borne out by the substantially higher proportions of all three PTV tiers and the two most severe missense variant tiers in <i>de novo</i> compared with inherited variants (<a id="crosref0535" href="#fig1">Figure&nbsp;1</a>A).</div><div class="figure-wrap"><figure id="fig1" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/379b31ff-ee89-4304-959d-e4f0e361a995/main.assets/gr1_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr1" src="/cms/10.1016/j.cell.2019.12.036/asset/64046fc9-1e40-49ed-ad5f-3c80f01b296a/main.assets/gr1.jpg" height="946" width="708" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/379b31ff-ee89-4304-959d-e4f0e361a995/main.assets/gr1_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig1-title" class="figure__title"><span class="label figure__label">Figure&nbsp;1</span> <span class="figure__title__text">Distribution of Rare Autosomal Protein-Coding Variants in ASD Cases and Controls</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig1-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0015" role="paragraph">(A) The proportion of rare autosomal genetic variants split by predicted functional consequences, represented by color, is displayed for family-based (split into <i>de novo</i> and inherited variants) and case-control data. PTVs and missense variants are split into three tiers of predicted functional severity, represented by shade, based on the pLI and MPC metrics, respectively.</div><div id="fspara0020" role="paragraph">(B) The relative difference in variant frequency (i.e., burden) between ASD cases and controls (top and bottom) or transmitted and untransmitted parental variants (center) is shown for the top two tiers of functional severity for PTVs (left and center) and the top tier of functional severity for missense variants (right). Next to the bar plot, the same data are shown divided by sex.</div><div id="fspara0025" role="paragraph">(C) The relative difference in variant frequency shown in (B) is converted to a trait liability <i>Z</i> score, split by the same subsets used in (A). For context, a <i>Z</i> score of 2.18 would shift an individual from the population mean to the top 1.69% of the population (equivalent to an ASD threshold based on 1 in 68 children; <span class="dropBlock reference-citations"><a id="body-ref-sref14" href="#" role="doc-biblioref" data-xml-rid="bib14" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref14" aria-controls="sref14">Christensen et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref14"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">14.</div><div>Christensen, D.L. ∙ Baio, J. ∙ Van Naarden Braun, K. ..., Centers for Disease Control and Prevention (CDC)</div><div><strong>Prevalence and Characteristics of Autism Spectrum Disorder Among Children Aged 8 Years--Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2012</strong></div><div><em>MMWR Surveill. Summ.</em> 2016; <strong>65</strong>:1-23</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15585/mmwr.ss6503a1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27031587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15585%2Fmmwr.ss6503a1&amp;pmid=27031587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). No significant difference in liability was observed between males and females for any analysis.</div><div id="fspara0030" role="paragraph">Statistical tests: (B) and (C), binomial exact test (BET) for most contrasts; exceptions were “both” and “case-control,” for which Fisher’s method for combining BET p values for each sex and, for case-control, each population was used; p values corrected for 168 tests are shown.</div></div></div></div></figcaption></figure></div><div id="p0055" role="paragraph">Comparing family-based cases with unaffected siblings in the 1,447 genes with pLI ≥ 0.995, there is a 3.5-fold enrichment of <i>de novo</i> PTVs (366 in 6,430 cases versus 35 in 2,179 controls; 0.057 versus 0.016 variants per sample (vps); p&nbsp;= 4&nbsp;× 10<sup>−17</sup>, two-sided Poisson exact test; <a id="crosref0540" href="#fig1">Figure&nbsp;1</a>B) and 1.2-fold enrichment of rare inherited PTVs (695 transmitted versus 557 untransmitted in 5,869 parents; 0.12 versus 0.10 vps; p&nbsp;= 0.07, binomial exact test; <a id="crosref0545" href="#fig1">Figure&nbsp;1</a>B). The same genes in the case-control data show an intermediate 1.8-fold enrichment of PTVs (874 in 5,556 cases versus 759 in 8,809 controls; 0.16 versus 0.09 vps; p&nbsp;= 4&nbsp;× 10<sup>−24</sup>, binomial exact test; <a id="crosref0550" href="#fig1">Figure&nbsp;1</a>B). Analysis of the middle tier of PTVs (0.5&nbsp;≤ pLI&nbsp;&lt; 0.995) shows a similar but muted pattern (<a id="crosref0555" href="#fig1">Figure&nbsp;1</a>B), whereas the lowest tier of PTVs (pLI&nbsp;&lt; 0.5) shows no enrichment (<a id="crosref0560" href="#mmc1">Table S1</a>).</div><div id="p0060" role="paragraph"><i>De novo</i> missense variants occur more frequently than <i>de novo</i> PTVs. Collectively, they show only marginal enrichment over the rate expected by chance (<span class="dropBlock reference-citations"><a id="body-ref-sref21-3" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-3" aria-controls="sref21-3">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref0570" href="#fig1">Figure&nbsp;1</a>). The most severe <i>de novo</i> missense variants (MPC ≥ 2), however, show a frequency similar to the most severe tier of <i>de novo</i> PTVs. They yield 2.1-fold case enrichment (354 in 6,430 cases versus 58 in 2,179 controls; 0.055 versus 0.027 vps; p&nbsp;= 3&nbsp;× 10<sup>−8</sup>, two-sided Poisson exact test; <a id="crosref0575" href="#fig1">Figure&nbsp;1</a>B) with consistent 1.2-fold enrichment in case-control data (4,277 in 5,556 cases versus 6,149 in 8,809 controls; 0.80 versus 0.68 vps; p&nbsp;= 4&nbsp;× 10<sup>−7</sup>, binomial exact test; <a id="crosref0580" href="#fig1">Figure&nbsp;1</a>B). These variants show stronger enrichment than the middle tier of PTVs, whereas the other two tiers of missense variation are not significantly enriched (<a id="crosref0585" href="#mmc1">Table S1</a>).</div><div id="p0065" role="paragraph">From our data, the proportion of the variance explained by <i>de&nbsp;novo</i> PTVs is 1.3%, 1.2% of it from the highest pLI category. The proportion of the variance explained by <i>de novo</i> MPC ≥ 2 missense variants is 0.5%, whereas all remaining missense variation explains 0.12%. Thus, in total, all exome <i>de novo</i> variants in the autosomes explain 1.92% of the variance of ASD.</div></section><section id="sec-2-3"><h3>Sex Differences in ASD Risk</h3><div id="p0070" role="paragraph">ASD is more prevalent in males than females. In line with previous observations (<span class="dropBlock reference-citations"><a id="body-ref-sref21-4" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-4" aria-controls="sref21-4">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we observe a 2-fold enrichment of <i>de novo</i> PTVs in highly constrained genes in affected females (n&nbsp;= 1,097) versus affected males (n&nbsp;= 5,333) (p&nbsp;= 3&nbsp;× 10<sup>−6</sup>, two-sided Poisson exact test; <a id="crosref0595" href="#fig1">Figure&nbsp;1</a>B; <a id="crosref0600" href="#mmc1">Table&nbsp;S1</a>). This result is consistent with the female protective effect model, which postulates that females require an increased genetic load to reach the threshold for ASD diagnosis (<span class="dropBlock reference-citations"><a id="body-ref-sref106" href="#" role="doc-biblioref" data-xml-rid="bib106" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref106" aria-controls="sref106">Werling, 2016</a><div class="dropBlock__holder" data-db-target-of="sref106"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">102.</div><div>Werling, D.M.</div><div><strong>The role of sex-differential biology in risk for autism spectrum disorder</strong></div><div><em>Biol. Sex Differ.</em> 2016; <strong>7</strong>:58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13293-016-0112-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84995459627" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27891212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13293-016-0112-8&amp;pmid=27891212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The converse hypothesis is that risk variation has larger effects in males than in females so that females require a higher burden to reach the same diagnostic threshold as males. Across all classes of genetic variants, we observed no significant sex differences in trait liability, consistent with the female protective effect model (<a id="crosref0610" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0615" href="#sec-4">STAR Methods</a>). Thus, we estimated the liability <i>Z</i> scores for different classes of variants from both sexes together (<a id="crosref0620" href="#fig1">Figure&nbsp;1</a>C; <a id="crosref0625" href="#mmc1">Table S1</a>) and leveraged them to enhance gene discovery.</div></section><section id="sec-2-4"><h3>ASD Gene Discovery</h3><div id="p0075" role="paragraph">In previous risk gene discovery efforts, we used the transmitted and <i>de novo</i> association (TADA) model (<span class="dropBlock reference-citations"><a id="body-ref-sref35-3" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-3" aria-controls="sref35-3">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) to integrate protein-truncating and missense variants that are <i>de novo</i>, inherited, or from case-control populations and to stratify autosomal genes by FDR for association. Here we update the TADA model to include pLI score as a continuous metric for PTVs and MPC score as a two-tiered metric (≥2, 1–2) for missense variants (<a id="crosref0635" href="#sec-4">STAR Methods</a>; <a id="crosref0640" href="#figs2">Figure&nbsp;S2</a>). From family data, we include <i>de novo</i> PTVs as well as <i>de novo</i> missense variants, whereas from the case-control, we include only PTVs; we do not include inherited variants because of the limited liabilities observed (<a id="crosref0645" href="#fig1">Figure&nbsp;1</a>C). Our analyses reveal that these modifications result in an enhanced TADA model with greater sensitivity and accuracy than the original model (<a id="crosref0650" href="#fig2">Figure&nbsp;2</a>A); no other covariates examined were important after accounting for these factors (<a id="crosref0655" href="#sec-4">STAR Methods</a>).</div><div class="figure-wrap"><figure id="fig2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/63a1aa54-d45f-4e0c-b3d0-18e33d5eb512/main.assets/gr2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr2" src="/cms/10.1016/j.cell.2019.12.036/asset/d7d3b23d-21c9-4c78-9be4-31a6d0d43cb4/main.assets/gr2.jpg" height="688" width="770" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/63a1aa54-d45f-4e0c-b3d0-18e33d5eb512/main.assets/gr2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;2</span> <span class="figure__title__text">Gene Discovery in the ASC Cohort</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0040" role="paragraph">(A) WES data from 35,584 samples are entered into a Bayesian analysis framework (TADA) that incorporates pLI score for PTVs and MPC score for missense variants.</div><div id="fspara0045" role="paragraph">(B) The model identifies 102 autosomal genes associated with ASD at a false discovery rate (FDR) threshold of 0.1 or less, which is shown on the y axis of this Manhattan plot, with each point representing a gene. Of these, 78 pass the threshold FDR of 0.05 or less, and 26 pass the threshold family-wise error rate (FWER) of 0.05 or less.</div><div id="fspara0050" role="paragraph">(C) Repeating our ASD trait liability analysis (<a id="crosref1800" href="#fig1">Figure&nbsp;1</a>C) for variants observed within the 102 ASD-associated genes only.</div><div id="fspara0055" role="paragraph">Statistical tests: (B), TADA; (C), BET for most contrasts; exceptions were “both” and “case-control,” for which Fisher’s method for combining BET p values for each sex and, for case-control, each population was used; p values corrected for 168 tests are shown.</div></div></div></div></figcaption></figure></div><div class="figure-wrap"><figure id="figs2" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/7d96eace-13bd-40da-84d4-74d7d4a8294b/main.assets/figs2_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs2" src="/cms/10.1016/j.cell.2019.12.036/asset/7b882646-6021-489a-9ea3-88bc568a941b/main.assets/figs2.jpg" height="941" width="738" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/7d96eace-13bd-40da-84d4-74d7d4a8294b/main.assets/figs2_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs2-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S2</span> <span class="figure__title__text">Enhancing the TADA Model and Evaluating Its Accuracy, Related to <a id="crosref1845" href="#fig2">Figure&nbsp;2</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs2-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0220" role="paragraph">A) Distribution of p-values from the TADA analysis of protein truncating variants (PTVs) and Mis3 (PolyPhen-2 probably damaging) variants in the data. B) Empirical-known signal experiments (EKSE) to evaluate the control of FDR. The bold line shows the FDR, averaged over 20 iterations versus q-values. For these iterations, a true signal is superimposed on a common background of benign mutations. Error bars are obtained from the “pure” simulation (<a id="crosref1850" href="#sec-4">STAR Methods</a>). C) Observed vs. simulated distribution of synonymous and Mis1 (PolyPhen-2 benign) variants. The solid line shows the distribution of 500 simulation results, compared to the observed shown in the dashed line. Top: the number of genes with synonymous events ≥3. The average over simulations is 16.6, while the observed is 19. Bottom: the number of genes with Mis1 (PolyPhen-2 benign) events ≥2. The average over simulations is 160.6, while the observed is 179. D) To visualize potential deviations from the model in the TADA analysis of the true ASD sample, TADA p-values are converted to z-values, which are assumed to follow a standard normal density (for null genes). Here a normal density with mean zero and variance&nbsp;= 1.02 (in red) is superimposed on the histogram of results from the real data, illustrating how closely the data follow the assumed model (1.02&nbsp;= variance&nbsp;= λGC for these results). E) Underestimating the per-gene mutation rate could lead to false discoveries. To assess the impact of model misspecification, the genomic control factor, λGC, is plotted for various misspecifications of μ. Each line corresponds to a different fraction of genes with underestimated μ, and the x-axis is the magnitude of the underestimation: μused /μtrue. F) True discoveries (TD) and false discoveries (FD) by critical value, and for various levels of model discrepancy, with λ<sub>GC</sub> as the genomic control factor. FWER is set at the correction rate. FDR is set at 0.05, 0.10, and 0.30. The circumference of each pie chart equals the cube root of the total number of discoveries. We observe that the q-values of both non-signal (G) and signal (H) borderline genes decreases as π increases, but the impact on the non-signal borderline genes is not substantial and q-values for only a few of these genes cross the threshold. I) The average relative risk (γ) for variants in the data. We model the relative risk for <i>de novo</i> protein truncating variants (PTVs) as a continuous function of pLI. J) For <i>de novo</i> missense variants, we estimate the curve using a moving window over probands’ missense variants ordered by MPC score. The MPC score can be cut into three categories, and we use the last two in the TADA analysis, since γ in the first category is close to the null value of one. K) For case-control PTVs, we model the relative risk as a continuous function of pLI. L) The empirical FDR versus q-values, averaged over 100 simulations using the original Bayes Factors. M) The empirical FDR versus q-values, averaged over 100 simulations using the Bayes Factors with a lower limit. N) The q-values for three types of TADA analyses. We present the 114 genes with q-value less than 0.1 in at least one analysis.</div></div></div></div></figcaption></figure></div><div id="p0080" role="paragraph">Our refined TADA model identifies 102 ASD risk genes at FDR ≤ 0.1, of which 78 pass FDR ≤ 0.05 and 26 pass Bonferroni-corrected (p ≤ 0.05) thresholds (<a id="crosref0660" href="#fig2">Figure&nbsp;2</a>B; <a id="crosref0665" href="#mmc2">Table S2</a>). Simulation experiments (<a id="crosref0670" href="#sec-4">STAR Methods</a>) show that the FDR is properly calibrated and relatively insensitive to estimates of the total number of ASD-related genes in the genome (<a id="crosref0675" href="#figs2">Figure&nbsp;S2</a>). Of the 102 ASD-associated genes, 60 were not discovered by our earlier analyses (<span class="dropBlock reference-citations"><a id="body-ref-sref92-3" href="#" role="doc-biblioref" data-xml-rid="bib21 bib38 bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-3" aria-controls="sref92-3">De Rubeis et&nbsp;al., 2014; Iossifov et&nbsp;al., 2014; Sanders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Iossifov, I. ∙ O’Roak, B.J. ∙ Sanders, S.J. ...</div><div><strong>The contribution of de novo coding mutations to autism spectrum disorder</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:216-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13908" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912101541" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13908&amp;pmid=25363768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). These include 30 considered truly novel because they have not been implicated in autosomal dominant neurodevelopmental disorders (ASD, developmental delay, epilepsy, and intellectual disability) and were not significantly enriched for <i>de novo</i> and/or rare variants in previous studies (<a id="crosref0680" href="#mmc2">Table S2</a>). The patterns of liability seen for the 102 genes are similar to that seen over all genes (compare <a id="crosref0685" href="#fig2">Figure&nbsp;2</a>C with <a id="crosref0690" href="#fig1">Figure&nbsp;1</a>C), although the effects of variants are uniformly larger, as would be expected for this selected list.</div><div id="p0085" role="paragraph">We did not analyze <i>de novo</i> mutations on chromosome X because they are rare, which reduces power for gene discovery from these data; the majority of <i>de novo</i> mutations are of paternal origin, and only females—who represent a minority of ASD diagnoses—receive an X chromosome from their fathers. Moreover, many of the known ASD genes identified on chromosome X show recessive-like inheritance, in which males inherit risk variation from an unaffected mother, and, with our current sample size, we are underpowered for inherited variation. Complementing these observations, when we assessed variants from chromosome X using sex-stratified case-control analyses, no gene had a significant excess of PTV and MPC&nbsp;≥ 2 variants after Bonferroni correction (<a id="crosref0695" href="#mmc2">Table S2</a>). Five genes did show evidence of increased <i>de novo</i> variants (<i>ARHGEF9</i>, <i>IQSEC2</i>, <i>SLC25A6</i>, <i>PCDH19</i>, and <i>OFD1</i>); all but <i>SLC25A6</i> are already implicated in X-linked intellectual disability. Of these variants, 43% are&nbsp;in females (which make up 17% of the cohort), underscoring the challenges of analyzing <i>de novo</i> mutations on chromosome X.</div></section><section id="sec-2-5"><h3>Patterns of Mutations in ASD Genes</h3><div id="p0090" role="paragraph">The ratio of PTVs to missense mutations varies substantially between genes (<a id="crosref0700" href="#fig3">Figure&nbsp;3</a>A). Some genes reach our association threshold through PTVs alone (e.g., <i>ADNP</i>), and three genes have a significant excess of PTVs relative to missense mutations, accounting for gene mutability: <i>SYNGAP1</i>, <i>DYRK1A</i>, and <i>ARID1B</i> (p&nbsp;&lt; 0.0005, binomial test). Because of the increased cohort size and availability of the MPC metric, we are also able, for the first time, to associate genes with ASD based primarily on <i>de novo</i> missense variation. Four genes carry four or more <i>de novo</i> missense variants (MPC ≥ 1) in ASD cases and one or no PTVs: <i>DEAF1</i>, <i>KCNQ3</i>, <i>SCN1A</i>, and <i>SLC6A1</i> (<a id="crosref0705" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0710" href="#mmc3">Table S3</a>).</div><div class="figure-wrap"><figure id="fig3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/71c982c4-299f-46fc-a363-5ece27ee76a5/main.assets/gr3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr3" src="/cms/10.1016/j.cell.2019.12.036/asset/1c8bd6d3-02e5-46c0-a433-1b103ec78307/main.assets/gr3.jpg" height="548" width="770" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/71c982c4-299f-46fc-a363-5ece27ee76a5/main.assets/gr3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;3</span> <span class="figure__title__text">Genetic Characterization of ASD Genes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0065" role="paragraph">(A) Count of PTVs versus missense variants (MPC ≥ 1) in cases for each ASD-associated gene (red points, selected genes labeled). These counts reflect the data used by TADA for association analysis: <i>de novo</i> and case-control data for PTVs; <i>de novo</i> only for missense.</div><div id="fspara0070" role="paragraph">(B) Location of ASD <i>de novo</i> missense variants in <i>DEAF1</i>. The five ASD variants (marked in red) are in the SAND (Sp100, AIRE-1, NucP41/75, DEAF-1) DNA-binding domain (amino acids 193–273, spirals show α helices, arrows show β sheets, KDWK is the DNA-binding motif) alongside 10 variants observed in NDD, several of which have been shown to reduce DNA binding, including Q264P and Q264R (<span class="dropBlock reference-citations"><a id="body-ref-sref102" href="#" role="doc-biblioref" data-xml-rid="bib13 bib36 bib102" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref102" aria-controls="sref102">Chen et&nbsp;al., 2017; Heyne et&nbsp;al., 2018; Vulto-van Silfhout et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref102"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">13.</div><div>Chen, L. ∙ Jensik, P.J. ∙ Alaimo, J.T. ...</div><div><strong>Functional analysis of novel DEAF1 variants identified through clinical exome sequencing expands DEAF1-associated neurodevelopmental disorder (DAND) phenotype</strong></div><div><em>Hum. Mutat.</em> 2017; <strong>38</strong>:1774-1785</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/humu.23339" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85030106221" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28940898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fhumu.23339&amp;pmid=28940898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Heyne, H.O. ∙ Singh, T. ∙ Stamberger, H. ..., EuroEPINOMICS RES Consortium</div><div><strong>De novo variants in neurodevelopmental disorders with epilepsy</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1048-1053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0143-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85049011462" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942082/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0143-7&amp;pmid=29942082" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">98.</div><div>Vulto-van Silfhout, A.T. ∙ Rajamanickam, S. ∙ Jensik, P.J. ...</div><div><strong>Mutations affecting the SAND domain of DEAF1 cause intellectual disability with severe speech impairment and behavioral problems</strong></div><div><em>Am. J. Hum. Genet.</em> 2014; <strong>94</strong>:649-661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84899899835" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24726472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2014.03.013&amp;pmid=24726472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0075" role="paragraph">(C) Location of ASD missense variants in <i>KCNQ3</i>. All four ASD variants are located in the voltage sensor (fourth of six transmembrane domains), with three in the same residue (R230), including the gain-of-function R230C mutation observed in NDD (<span class="dropBlock reference-citations"><a id="body-ref-sref65-1" href="#" role="doc-biblioref" data-xml-rid="bib36 bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-1" aria-controls="sref65-1">Heyne et&nbsp;al., 2018; Miceli et&nbsp;al, 2015</a><div class="dropBlock__holder" data-db-target-of="sref65-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Heyne, H.O. ∙ Singh, T. ∙ Stamberger, H. ..., EuroEPINOMICS RES Consortium</div><div><strong>De novo variants in neurodevelopmental disorders with epilepsy</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1048-1053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0143-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85049011462" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942082/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0143-7&amp;pmid=29942082" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Miceli, F. ∙ Soldovieri, M.V. ∙ Ambrosino, P. ...</div><div><strong>Early-onset epileptic encephalopathy caused by gain-of-function mutations in the voltage sensor of Kv7.2 and Kv7.3 potassium channel subunits</strong></div><div><em>J.&nbsp;Neurosci.</em> 2015; <strong>35</strong>:3782-3793</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.4423-14.2015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84924787957" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25740509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.4423-14.2015&amp;pmid=25740509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Five inherited variants observed in benign infantile seizures are shown in the pore loop (<span class="dropBlock reference-citations"><a id="body-ref-sref61" href="#" role="doc-biblioref" data-xml-rid="bib48 bib61" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref61" aria-controls="sref61">Landrum et&nbsp;al., 2014; Maljevic et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref61"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">46.</div><div>Landrum, M.J. ∙ Lee, J.M. ∙ Riley, G.R. ...</div><div><strong>ClinVar: public archive of relationships among sequence variation and human phenotype</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D980-D985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84891809093" target="_blank" aria-label="Open Scopus in new tab">Scopus (959)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24234437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1113&amp;pmid=24234437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">59.</div><div>Maljevic, S. ∙ Vejzovic, S. ∙ Bernhard, M.K. ...</div><div><strong>Novel KCNQ3 mutation in a large family with benign familial neonatal epilepsy: A rare cause of neonatal seizures</strong></div><div><em>Mol. Syndromol.</em> 2016; <strong>7</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000447461" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84978056091" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27781029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000447461&amp;pmid=27781029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0080" role="paragraph">(D) Location of ASD missense variants in <i>SCN1A</i> alongside 17 <i>de novo</i> variants in NDD and epilepsy (<span class="dropBlock reference-citations"><a id="body-ref-sref36-3" href="#" role="doc-biblioref" data-xml-rid="bib36" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref36-3" aria-controls="sref36-3">Heyne et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref36-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Heyne, H.O. ∙ Singh, T. ∙ Stamberger, H. ..., EuroEPINOMICS RES Consortium</div><div><strong>De novo variants in neurodevelopmental disorders with epilepsy</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1048-1053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0143-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85049011462" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942082/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0143-7&amp;pmid=29942082" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0085" role="paragraph">(E) Location of ASD missense variants in <i>SLC6A1</i> alongside 31 <i>de novo</i> variants in NDD and epilepsy (<span class="dropBlock reference-citations"><a id="body-ref-sref41" href="#" role="doc-biblioref" data-xml-rid="bib36 bib41" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref41" aria-controls="sref41">Heyne et&nbsp;al., 2018; Johannesen et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref41"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">34.</div><div>Heyne, H.O. ∙ Singh, T. ∙ Stamberger, H. ..., EuroEPINOMICS RES Consortium</div><div><strong>De novo variants in neurodevelopmental disorders with epilepsy</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1048-1053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0143-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85049011462" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942082/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0143-7&amp;pmid=29942082" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">39.</div><div>Johannesen, K.M. ∙ Gardella, E. ∙ Linnankivi, T. ...</div><div><strong>Defining the phenotypic spectrum of SLC6A1 mutations</strong></div><div><em>Epilepsia.</em> 2018; <strong>59</strong>:389-402</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/epi.13986" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85041378864" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29315614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fepi.13986&amp;pmid=29315614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0090" role="paragraph">(F) Subtelomeric 2q37 deletions are associated with facial dysmorphisms, brachydactyly, high BMI, NDD, and ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref52" href="#" role="doc-biblioref" data-xml-rid="bib52" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref52" aria-controls="sref52">Leroy et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref52"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">50.</div><div>Leroy, C. ∙ Landais, E. ∙ Briault, S. ...</div><div><strong>The 2q37-deletion syndrome: an update of the clinical spectrum including overweight, brachydactyly and behavioural features in 14 new patients</strong></div><div><em>Eur. J. Hum. Genet.</em> 2013; <strong>21</strong>:602-612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2012.230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84867413401" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23073310/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fejhg.2012.230&amp;pmid=23073310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Although three genes within the locus have a pLI score of 0.995 or higher, only <i>HDLBP</i> is associated with ASD.</div><div id="fspara0095" role="paragraph">(G) Deletions at the 11q13.2−q13.4 locus have been observed in NDD, ASD, and otodental dysplasia (<span class="dropBlock reference-citations"><a id="body-ref-sref17-1" href="#" role="doc-biblioref" data-xml-rid="bib16 bib17" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref17-1" aria-controls="sref17-1">Coe et&nbsp;al., 2014; Cooper et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref17-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Coe, B.P. ∙ Witherspoon, K. ∙ Rosenfeld, J.A. ...</div><div><strong>Refining analyses of copy number variation identifies specific genes associated with developmental delay</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:1063-1071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3092" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84922012902" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3092&amp;pmid=25217958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Cooper, G.M. ∙ Coe, B.P. ∙ Girirajan, S. ...</div><div><strong>A copy number variation morbidity map of developmental delay</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:838-846</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80052260252" target="_blank" aria-label="Open Scopus in new tab">Scopus (755)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21841781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.909&amp;pmid=21841781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Five genes within the locus have a pLI score of 0.995 or higher, including two ASD genes: <i>KMT5B</i> and <i>SHANK2</i>.</div><div id="fspara0100" role="paragraph">(H) Assessment of gene-based enrichment, via MAGMA, of 102 ASD genes against genome-wide significant common variants from six GWASs.</div><div id="fspara0105" role="paragraph">(I) Gene-based enrichment of 102 ASD genes in multiple GWASs as a function of effective cohort size. The GWAS used for each disorder in (I) has a black outline.</div><div id="fspara0110" role="paragraph">Statistical tests: (F) and (G), TADA; (H) and (I), MAGMA.</div></div></div></div></figcaption></figure></div><div id="p0095" role="paragraph">For <i>DEAF1</i>, five <i>de novo</i> missense variants were observed, and all reside in the SAND (Sp100, AIRE-1, NucP41/75, DEAF-1) domain (<a id="crosref0715" href="#fig3">Figure&nbsp;3</a>B), which is critical for dimerization and DNA binding (<span class="dropBlock reference-citations"><a id="body-ref-sref39" href="#" role="doc-biblioref" data-xml-rid="bib7 bib39" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref39" aria-controls="sref39">Bottomley et&nbsp;al., 2001; Jensik et&nbsp;al., 2004</a><div class="dropBlock__holder" data-db-target-of="sref39"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">7.</div><div>Bottomley, M.J. ∙ Collard, M.W. ∙ Huggenvik, J.I. ...</div><div><strong>The SAND domain structure defines a novel DNA-binding fold in transcriptional regulation</strong></div><div><em>Nat. Struct. Biol.</em> 2001; <strong>8</strong>:626-633</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/89675" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-0034957535" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11427895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F89675&amp;pmid=11427895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">37.</div><div>Jensik, P.J. ∙ Huggenvik, J.I. ∙ Collard, M.W.</div><div><strong>Identification of a nuclear export signal and protein interaction domains in deformed epidermal autoregulatory factor-1 (DEAF-1)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2004; <strong>279</strong>:32692-32699</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M400946200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-3543033387" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15161925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M400946200&amp;pmid=15161925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For <i>KCNQ3,</i> all four <i>de novo</i> missense variants modify arginine residues in the voltage-sensing fourth transmembrane domain, with three at a single residue previously characterized as gain of function in NDD (R230C; <a id="crosref0720" href="#fig3">Figure&nbsp;3</a>C; <span class="dropBlock reference-citations"><a id="body-ref-sref65-2" href="#" role="doc-biblioref" data-xml-rid="bib65" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref65-2" aria-controls="sref65-2">Miceli et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref65-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">63.</div><div>Miceli, F. ∙ Soldovieri, M.V. ∙ Ambrosino, P. ...</div><div><strong>Early-onset epileptic encephalopathy caused by gain-of-function mutations in the voltage sensor of Kv7.2 and Kv7.3 potassium channel subunits</strong></div><div><em>J.&nbsp;Neurosci.</em> 2015; <strong>35</strong>:3782-3793</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.4423-14.2015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84924787957" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25740509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.4423-14.2015&amp;pmid=25740509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Of the four <i>de novo</i> missense variants identified in <i>SCN1A</i> (<a id="crosref0730" href="#fig3">Figure&nbsp;3</a>A; <a id="crosref0735" href="#mmc3">Table S3</a>), three occur in the C terminus (<a id="crosref0740" href="#fig3">Figure&nbsp;3</a>D), and all four carriers have seizures. Finally, we observe eight <i>de novo</i> missense variants in <i>SLC6A1</i> (<a id="crosref0745" href="#fig3">Figure&nbsp;3</a>E), with four in the sixth transmembrane domain and one recurring in two independent cases (A288V). Five of the six subjects with available information on history of seizure have seizures; all four subjects assessed have intellectual disability.</div></section><section id="sec-2-6"><h3>ASD Genes within Recurrent Copy Number Variants (CNVs)</h3><div id="p0100" role="paragraph">Large CNVs represent another important source of risk for ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref96" href="#" role="doc-biblioref" data-xml-rid="bib96" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref96" aria-controls="sref96">Sebat et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref96"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">92.</div><div>Sebat, J. ∙ Lakshmi, B. ∙ Malhotra, D. ...</div><div><strong>Strong association of de novo copy number mutations with autism</strong></div><div><em>Science.</em> 2007; <strong>316</strong>:445-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1138659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-34247481814" target="_blank" aria-label="Open Scopus in new tab">Scopus (1835)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17363630/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1138659&amp;pmid=17363630" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), but these genomic disorder segments can include dozens of genes, complicating the identification of driver gene(s) within these regions. To determine whether the 102 ASD genes could nominate driver genes within genomic disorder regions, we first curated a consensus list from nine sources, totaling 823 protein-coding genes in 51 autosomal genomic disorder loci associated with ASD or ASD-related phenotypes, including NDD (<a id="crosref0755" href="#mmc3">Table S3</a>). Of the 51 loci, 12 encompassed a total of 13 ASD-associated genes (<a id="crosref0760" href="#mmc3">Table S3</a>), which is greater than expected by chance when controlling for number of genes, PTV mutation rate, and brain expression levels per gene (2.3-fold increase; p&nbsp;= 2.3&nbsp;× 10<sup>−3</sup>, permutation). These 12 loci were divided into three groups: (1) the overlapping ASD gene matched the consensus driver gene (e.g., <i>SHANK3</i> for Phelan-McDermid syndrome; <span class="dropBlock reference-citations"><a id="body-ref-sref97" href="#" role="doc-biblioref" data-xml-rid="bib97" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref97" aria-controls="sref97">Soorya et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref97"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">93.</div><div>Soorya, L. ∙ Kolevzon, A. ∙ Zweifach, J. ...</div><div><strong>Prospective investigation of autism and genotype-phenotype correlations in 22q13 deletion syndrome and SHANK3 deficiency</strong></div><div><em>Mol. Autism.</em> 2013; <strong>4</strong>:18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2040-2392-4-18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84878743755" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23758760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2040-2392-4-18&amp;pmid=23758760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); (2) an ASD gene emerged that did not match the previously predicted driver gene(s) within the region, such as <i>HDLBP</i> at 2q37.3 (<a id="crosref0770" href="#fig3">Figure&nbsp;3</a>F), where <i>HDAC4</i> has been hypothesized as a driver gene (<span class="dropBlock reference-citations"><a id="body-ref-sref108" href="#" role="doc-biblioref" data-xml-rid="bib108" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref108" aria-controls="sref108">Williams et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref108"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">104.</div><div>Williams, S.R. ∙ Aldred, M.A. ∙ Der Kaloustian, V.M. ...</div><div><strong>Haploinsufficiency of HDAC4 causes brachydactyly mental retardation syndrome, with brachydactyly type E, developmental delays, and behavioral problems</strong></div><div><em>Am. J. Hum. Genet.</em> 2010; <strong>87</strong>:219-228</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2010.07.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2010.07.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77955584378" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20691407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.07.011&amp;pmid=20691407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); and (3) no previous driver gene had been established within the locus, such as <i>BCL11A</i> at 2p15-p16.1. One locus, 11q13.2-q13.4, had two of our 102 genes (<i>SHANK2</i> and <i>KMT5B</i>; <a id="crosref0780" href="#fig3">Figure&nbsp;3</a>G), highlighting that genomic disorder loci can result from risk conferred by multiple genes, potentially including genes with small effect sizes that we are underpowered to detect.</div></section><section id="sec-2-7"><h3>Relationship of ASD Genes with GWAS Signals</h3><div id="p0105" role="paragraph">Common variation plays an important role in ASD risk (<span class="dropBlock reference-citations"><a id="body-ref-sref31" href="#" role="doc-biblioref" data-xml-rid="bib31" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref31" aria-controls="sref31">Gaugler et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref31"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">29.</div><div>Gaugler, T. ∙ Klei, L. ∙ Sanders, S.J. ...</div><div><strong>Most genetic risk for autism resides with common variation</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:881-885</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84905582433" target="_blank" aria-label="Open Scopus in new tab">Scopus (436)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25038753/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3039&amp;pmid=25038753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and recent genome-wide association studies (GWASs) reveal a handful of ASD-associated loci (<span class="dropBlock reference-citations"><a id="body-ref-sref33-1" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-1" aria-controls="sref33-1">Grove et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Grove, J. ∙ Ripke, S. ∙ Als, T.D. ..., Autism Spectrum Disorder Working Group of the Psychiatric Genomics Consortium, BUPGEN, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, 23andMe Research Team</div><div><strong>Identification of common genetic risk variants for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:431-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0344-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062110842" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0344-8&amp;pmid=30804558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Notably, among the five GWAS-significant ASD hits (<span class="dropBlock reference-citations"><a id="body-ref-sref33-2" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-2" aria-controls="sref33-2">Grove et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Grove, J. ∙ Ripke, S. ∙ Als, T.D. ..., Autism Spectrum Disorder Working Group of the Psychiatric Genomics Consortium, BUPGEN, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, 23andMe Research Team</div><div><strong>Identification of common genetic risk variants for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:431-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0344-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062110842" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0344-8&amp;pmid=30804558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), <i>KMT2E</i> is implicated by both GWAS and the list of 102 FDR ≤ 0.1 genes described here (Fisher’s exact test, p&nbsp;= 0.029). Thus, using MAGMA (multi-marker analysis of genomic annotation; <span class="dropBlock reference-citations"><a id="body-ref-sref20-1" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-1" aria-controls="sref20-1">de Leeuw et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref20-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>de Leeuw, C.A. ∙ Mooij, J.M. ∙ Heskes, T. ...</div><div><strong>MAGMA: generalized gene-set analysis of GWAS data</strong></div><div><em>PLoS Comput. Biol.</em> 2015; <strong>11</strong>:e1004219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84929486000" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25885710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004219&amp;pmid=25885710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we asked whether common genetic variation in or near the 102 identified genes (within 10 kb) influences ASD risk or other related traits. For these associated genes, MAGMA integrates GWAS summary statistics to determine whether their signal is enriched over background; namely, brain-expressed protein-coding genes. We used results from six GWAS datasets: ASD, schizophrenia, major depressive disorder, and attention deficit hyperactivity disorder (ADHD), which are all positively genetically correlated with ASD and with each other; educational attainment, which is positively correlated with ASD and negatively correlated with schizophrenia and ADHD; and human height as a negative control (<a id="crosref0805" href="#mmc3">Table S3</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref116" href="#" role="doc-biblioref" data-xml-rid="bib23 bib33 bib50 bib68 bib72 bib81 bib82 bib83 bib84 bib95 bib111 bib114 bib116" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref116" aria-controls="sref116">Demontis et&nbsp;al., 2019; Grove et&nbsp;al., 2019; Lee et&nbsp;al., 2018; Neale et&nbsp;al., 2010; Okbay et&nbsp;al., 2016; Rietveld et&nbsp;al., 2013; Ripke et&nbsp;al., 2011, 2013a, 2013b; Schizophrenia Working Group of the Psychiatric Genomics Consortium, 2014; Wray et&nbsp;al., 2018; Yengo et&nbsp;al., 2018; Zheng et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref116"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Demontis, D. ∙ Walters, R.K. ∙ Martin, J. ..., ADHD Working Group of the Psychiatric Genomics Consortium (PGC), Early Lifecourse &amp; Genetic Epidemiology (EAGLE) Consortium, 23andMe Research Team</div><div><strong>Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:63-75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0269-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85057436335" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0269-7&amp;pmid=30478444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Grove, J. ∙ Ripke, S. ∙ Als, T.D. ..., Autism Spectrum Disorder Working Group of the Psychiatric Genomics Consortium, BUPGEN, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, 23andMe Research Team</div><div><strong>Identification of common genetic risk variants for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:431-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0344-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062110842" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0344-8&amp;pmid=30804558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Lee, J.J. ∙ Wedow, R. ∙ Okbay, A. ..., 23andMe Research Team, COGENT (Cognitive Genomics Consortium), Social Science Genetic Association Consortium</div><div><strong>Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1112-1121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0147-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85050543338" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30038396/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0147-3&amp;pmid=30038396" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Neale, B.M. ∙ Medland, S.E. ∙ Ripke, S. ..., Psychiatric GWAS Consortium: ADHD Subgroup</div><div><strong>Meta-analysis of genome-wide association studies of attention-deficit/hyperactivity disorder</strong></div><div><em>J.&nbsp;Am. Acad. Child Adolesc. Psychiatry.</em> 2010; <strong>49</strong>:884-897</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.jaac.2010.06.008&amp;cf=fulltext&amp;site=jaac-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.jaac.2010.06.008&amp;cf=pdf&amp;site=jaac-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77956178166" target="_blank" aria-label="Open Scopus in new tab">Scopus (291)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20732625/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaac.2010.06.008&amp;pmid=20732625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Okbay, A. ∙ Beauchamp, J.P. ∙ Fontana, M.A. ..., LifeLines Cohort Study</div><div><strong>Genome-wide association study identifies 74 loci associated with educational attainment</strong></div><div><em>Nature.</em> 2016; <strong>533</strong>:539-542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27225129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17671&amp;pmid=27225129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Rietveld, C.A. ∙ Medland, S.E. ∙ Derringer, J. ..., LifeLines Cohort Study</div><div><strong>GWAS of 126,559 individuals identifies genetic variants associated with educational attainment</strong></div><div><em>Science.</em> 2013; <strong>340</strong>:1467-1471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1235488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84879271608" target="_blank" aria-label="Open Scopus in new tab">Scopus (379)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23722424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1235488&amp;pmid=23722424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Ripke, S. ∙ Sanders, A.R. ∙ Kendler, K.S. ..., Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium</div><div><strong>Genome-wide association study identifies five new schizophrenia loci</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:969-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.940" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80053384370" target="_blank" aria-label="Open Scopus in new tab">Scopus (1244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21926974/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.940&amp;pmid=21926974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Ripke, S. ∙ O’Dushlaine, C. ∙ Chambert, K. ..., Multicenter Genetic Studies of Schizophrenia Consortium, Psychosis Endophenotypes International Consortium, Wellcome Trust Case Control Consortium 2</div><div><strong>Genome-wide association analysis identifies 13 new risk loci for schizophrenia</strong></div><div><em>Nat. Genet.</em> 2013; <strong>45</strong>:1150-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84885020424" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23974872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2742&amp;pmid=23974872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Ripke, S. ∙ Wray, N.R. ∙ Lewis, C.M. ..., Major Depressive Disorder Working Group of the Psychiatric GWAS Consortium</div><div><strong>A mega-analysis of genome-wide association studies for major depressive disorder</strong></div><div><em>Mol. Psychiatry.</em> 2013; <strong>18</strong>:497-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2012.21" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84879892380" target="_blank" aria-label="Open Scopus in new tab">Scopus (635)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22472876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2012.21&amp;pmid=22472876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Schizophrenia Working Group of the Psychiatric Genomics Consortium</div><div><strong>Biological insights from 108 schizophrenia-associated genetic loci</strong></div><div><em>Nature.</em> 2014; <strong>511</strong>:421-427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13595" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84904804929" target="_blank" aria-label="Open Scopus in new tab">Scopus (3226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13595&amp;pmid=25056061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Wray, N.R. ∙ Ripke, S. ∙ Mattheisen, M. ..., eQTLGen, 23andMe, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium</div><div><strong>Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:668-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0090-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85046022254" target="_blank" aria-label="Open Scopus in new tab">Scopus (394)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29700475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0090-3&amp;pmid=29700475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">110.</div><div>Yengo, L. ∙ Sidorenko, J. ∙ Kemper, K.E. ..., GIANT Consortium</div><div><strong>Meta-analysis of genome-wide association studies for height and body mass index in ∼700000 individuals of European ancestry</strong></div><div><em>Hum. Mol. Genet.</em> 2018; <strong>27</strong>:3641-3649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddy271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85054297174" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30124842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddy271&amp;pmid=30124842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">112.</div><div>Zheng, J. ∙ Erzurumluoglu, A.M. ∙ Elsworth, B.L. ..., Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium</div><div><strong>LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:272-279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85028296791" target="_blank" aria-label="Open Scopus in new tab">Scopus (223)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27663502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw613&amp;pmid=27663502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Correcting for six analyses, only the schizophrenia and educational attainment GWAS signals show significant enrichment in ASD genes (<a id="crosref0810" href="#fig3">Figure&nbsp;3</a>H). The ASD GWAS signal was not enriched, potentially because common and rare variation contributing to ASD risk affect distinct genes or potentially because we currently lack the sample sizes to detect the convergence of the two. We conjecture that the second hypothesis is more likely because of three results: the known genetic correlation of schizophrenia and educational attainment with ASD, the enrichment of common variation conferring risk for both found in the 102 ASD genes, and the statistically significant overlap we demonstrate for <i>KMT2E</i>. In addition, effective cohort sizes for schizophrenia, educational attainment, and height dwarf that for ASD (<a id="crosref0815" href="#fig3">Figure&nbsp;3</a>I), and the quality of the GWAS signal strongly increases with sample size. Thus, for results from well-powered GWASs, it is reassuring that there is no signal for height but a clearly detectable signal for two traits genetically correlated with ASD.</div></section><section id="sec-2-8"><h3>Relationship between ASD and Other Neurodevelopmental Disorders</h3><div id="p0110" role="paragraph">Family studies yield high heritability estimates in ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref115" href="#" role="doc-biblioref" data-xml-rid="bib115" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref115" aria-controls="sref115">Yip et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref115"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">111.</div><div>Yip, B.H.K. ∙ Bai, D. ∙ Mahjani, B. ...</div><div><strong>Heritable variation, with little or no maternal effect, accounts for recurrence risk to autism spectrum disorder in Sweden</strong></div><div><em>Biol. Psychiatry.</em> 2018; <strong>83</strong>:589-597</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.09.007&amp;cf=fulltext&amp;site=bps-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.09.007&amp;cf=pdf&amp;site=bps-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85032932944" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29100626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biopsych.2017.09.007&amp;pmid=29100626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), whereas estimates of heritability in severe NDD are lower (<span class="dropBlock reference-citations"><a id="body-ref-sref80" href="#" role="doc-biblioref" data-xml-rid="bib80" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref80" aria-controls="sref80">Reichenberg et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref80"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">77.</div><div>Reichenberg, A. ∙ Cederlöf, M. ∙ McMillan, A. ...</div><div><strong>Discontinuity in the genetic and environmental causes of the intellectual disability spectrum</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:1098-1103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1508093112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84955494277" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26711998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1508093112&amp;pmid=26711998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Consistent with these observations, exome studies identify a higher frequency of disruptive <i>de novo</i> variants in severe NDD than in ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref22-1" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-1" aria-controls="sref22-1">Deciphering Developmental Disorders Study, 2017</a><div class="dropBlock__holder" data-db-target-of="sref22-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Deciphering Developmental Disorders Study</div><div><strong>Prevalence and architecture of de novo mutations in developmental disorders</strong></div><div><em>Nature.</em> 2017; <strong>542</strong>:433-438</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature21062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85015375341" target="_blank" aria-label="Open Scopus in new tab">Scopus (371)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28135719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature21062&amp;pmid=28135719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because 30%–50% of ASD individuals have comorbid intellectual disability and/or NDD, many genes are associated with both disorders (<span class="dropBlock reference-citations"><a id="body-ref-sref75-1" href="#" role="doc-biblioref" data-xml-rid="bib75" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref75-1" aria-controls="sref75-1">Pinto et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref75-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Pinto, D. ∙ Pagnamenta, A.T. ∙ Klei, L. ...</div><div><strong>Functional impact of global rare copy number variation in autism spectrum disorders</strong></div><div><em>Nature.</em> 2010; <strong>466</strong>:368-372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77954657070" target="_blank" aria-label="Open Scopus in new tab">Scopus (1285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20531469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09146&amp;pmid=20531469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Distinguishing genes that, when disrupted, lead to ASD more frequently than NDD could shed new light on how atypical neurodevelopment maps onto the core deficits of ASD.</div><div id="p0115" role="paragraph">To partition the 102 ASD genes in this manner, we compiled data from 5,264 trios ascertained for severe NDD (<a id="crosref0840" href="#mmc4">Table S4</a>) and compared the relative frequency, <i>R</i>, of disruptive <i>de novo</i> variants (which we define as PTVs or missense variants with MPC ≥ 1) in ASD- or NDD-ascertained trios. Genes with <i>R</i> &gt; 1 were classified as ASD-predominant (ASD<sub>P</sub>, 50 genes), whereas those with <i>R</i>&nbsp;&lt; 1 were classified as ASD with NDD (ASD<sub>NDD</sub>, 49 genes). Based on case-control data, the three other genes were assigned to the ASD<sub>P</sub> group (<a id="crosref0845" href="#fig4">Figure&nbsp;4</a>A). Thirteen of the genes demonstrate nominally significant heterogeneity between samples ascertained for ASD versus NDD (Fisher’s exact test, p&nbsp;&lt; 0.05) with only <i>ANKRD11</i> and <i>ASXL3</i> significant after correction for 102 genes; these and other heterogeneity analyses are described in <a id="crosref0850" href="#sec-4">STAR Methods</a> and <a id="crosref0855" href="#mmc4">Table S4</a>.</div><div class="figure-wrap"><figure id="fig4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/9c295865-705d-4b14-9b5f-19064d31bf64/main.assets/gr4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr4" src="/cms/10.1016/j.cell.2019.12.036/asset/0d271389-ea57-4ce6-89bd-402b4d70b79e/main.assets/gr4.jpg" height="672" width="770" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/9c295865-705d-4b14-9b5f-19064d31bf64/main.assets/gr4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;4</span> <span class="figure__title__text">Phenotypic and Functional Categories of ASD-Associated Genes</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0120" role="paragraph">(A) Frequency of disruptive <i>de novo</i> variants (e.g., PTVs or missense variants with MPC ≥ 1) in ASD-ascertained and NDD-ascertained cohorts (<a id="crosref1820" href="#mmc4">Table S4</a>) is shown for the 102 ASD-associated genes (selected genes labeled). Fifty genes with a higher frequency in ASD are designated ASD-predominant (ASD<sub>P</sub>), whereas the 49 genes more frequently mutated in NDD are designated as ASD<sub>NDD</sub>. Three genes marked with a star (<i>UBR1</i>, <i>MAP1A</i>, and <i>NUP155</i>) are included in the ASD<sub>P</sub> category on the basis of case-control data (<a id="crosref1825" href="#mmc4">Table S4</a>), which are not shown here. Of the 26 FWER genes, 10 are ASD<sub>P</sub> and 16 are ASD<sub>NDD</sub>. Of the 102 genes, 13 demonstrate nominally significant heterogeneity between samples ascertained for ASD versus NDD (<a id="crosref1885" href="#mmc4">Table S4</a>).</div><div id="fspara0125" role="paragraph">(B) ASD cases with disruptive <i>de novo</i> variants in ASD genes show delayed walking compared with ASD cases without such <i>de novo</i> variants, and the effect is greater for those with disruptive <i>de novo</i> variants in ASD<sub>NDD</sub> genes.</div><div id="fspara0130" role="paragraph">(C) Similarly, cases with disruptive <i>de novo</i> variants in ASD<sub>NDD</sub> genes and, to a lesser extent, ASD<sub>P</sub> genes have a lower full-scale IQ (FSIQ) than other ASD cases.</div><div id="fspara0135" role="paragraph">(D) Despite the association between <i>de novo</i> variants in ASD genes and cognitive impairment shown in (C), an excess of disruptive <i>de novo</i> variants is observed in cases without intellectual disability (FSIQ ≥ 70) or with an IQ above the cohort mean (FSIQ ≥ 82).</div><div id="fspara0140" role="paragraph">(E) Along with the phenotypic division (A), genes can also be classified functionally into four groups (gene expression regulation [GER], neuronal communication [NC], cytoskeleton, and other) based on Gene Ontology and research literature. The 102 ASD risk genes are shown in a mosaic plot divided by gene function and, from (A), the ASD versus NDD variant frequency, with the area of each box proportional to the number of genes.</div><div id="fspara0145" role="paragraph">Statistical tests: (B) and (C), t test; (D), chi-square test with 1° of freedom.</div></div></div></div></figcaption></figure></div><div id="p0120" role="paragraph">For ASD<sub>P</sub> genes and transmission of rare PTVs (relative frequency&nbsp;&lt; 0.001) from parents to their affected offspring, 44 PTVs were transmitted and 18 were not (p&nbsp;= 0.001, transmission disequilibrium test [TDT]), whereas, for ASD<sub>NDD</sub> genes, 14 were&nbsp;transmitted and 8 were not (p&nbsp;= 0.29; TDT). The frequency of PTVs in parents is significantly greater in ASD<sub>P</sub> genes (1.17&nbsp;per gene) than in ASD<sub>NDD</sub> genes (0.45 per gene; p&nbsp;= 6.6&nbsp;× 10<sup>−6</sup>, binomial test), whereas the frequency of <i>de novo</i> PTVs in cases is not markedly different between the two groups (95 in ASD<sub>P</sub> genes, 121 in ASD<sub>NDD</sub> genes; p&nbsp;= 0.07, binomial test with probability of success&nbsp;= 0.503 [PTV in ASD<sub>P</sub> genes]). The paucity of inherited PTVs in ASD<sub>NDD</sub> genes is consistent with greater selective pressure acting against disruptive variants in these genes and highlights fundamental differences between these two classes.</div><div id="p0125" role="paragraph">In addition, ASD subjects who carry disruptive <i>de novo</i> variants in ASD<sub>NDD</sub> genes walk 2.6 ± 1.2&nbsp;months later (<a id="crosref0860" href="#fig4">Figure&nbsp;4</a>B; p&nbsp;= 2.3&nbsp;× 10<sup>−5</sup>, t test, df&nbsp;= 251) and have an IQ 11.9 ± 6.0 points lower (<a id="crosref0865" href="#fig4">Figure&nbsp;4</a>C; p&nbsp;= 1.1&nbsp;× 10<sup>−4</sup>, two-sided t test, df&nbsp;= 278), on average, than ASD subjects with disruptive <i>de&nbsp;novo</i> variants in ASD<sub>P</sub> genes (<a id="crosref0870" href="#mmc4">Table S4</a>). Both sets of subjects differ significantly from the rest of the cohort with respect to IQ and age of walking (<a id="crosref0875" href="#fig4">Figures 4</a>B and 4C; <a id="crosref0880" href="#mmc4">Table S4</a>).</div><div id="p0130" role="paragraph">The data thus support an overall distinction between ASD<sub>P</sub> and ASD<sub>NDD</sub> genes <i>en masse</i>, although it is a matter of degree; disruptive <i>de novo</i> variants in both categories affect IQ and age of walking. Moreover, the smaller average effect of mutations on cognitive function in ASD<sub>P</sub> genes relative to ASD<sub>NDD</sub> genes does not mean that any individual carrying a disruptive <i>de novo</i> variant in an ASD<sub>P</sub> gene necessarily has an IQ of 70 or higher; likewise, not all individuals carrying a disruptive <i>de novo</i> variant in an ASD<sub>NDD</sub> gene have an IQ of less than 70. In addition, <i>de novo</i> variation plays an important role in ASD risk for both IQ groups. If we partition ASD cases into those with an IQ of 70 or higher (69.4%) versus those with an IQ of less than 70 (30.6%), individuals in the higher-IQ group still carry a greater burden of <i>de novo</i> variants relative to expectation, and this remains true when partitioning the IQ at the cohort mean (full-scale IQ [FSIQ] ≥ 82; <a id="crosref0885" href="#fig4">Figure&nbsp;4</a>D; 3,010 of 6,430 have FSIQ information) or when considering the 102 ASD genes only (<a id="crosref0890" href="#sec-4">STAR Methods</a>). Thus, excess burden is not limited to low-IQ cases, supporting the idea that <i>de novo</i> variants do not solely impair cognition (<span class="dropBlock reference-citations"><a id="body-ref-sref85" href="#" role="doc-biblioref" data-xml-rid="bib85" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref85" aria-controls="sref85">Robinson et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref85"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">82.</div><div>Robinson, E.B. ∙ Samocha, K.E. ∙ Kosmicki, J.A. ...</div><div><strong>Autism spectrum disorder severity reflects the average contribution of de novo and familial influences</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:15161-15165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1409204111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84908030450" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25288738/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1409204111&amp;pmid=25288738" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section><section id="sec-2-9"><h3>Functional Dissection of ASD Genes</h3><div id="p0135" role="paragraph">Past analyses have identified two major functional groups of ASD genes: those involved in gene expression regulation (GER), including chromatin regulators and transcription factors, and those involved in neuronal communication (NC), including synaptic function (<span class="dropBlock reference-citations"><a id="body-ref-sref21-6" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-6" aria-controls="sref21-6">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Similarly, Gene Ontology enrichment analysis with the 102 ASD genes identifies 16 genes in the “regulation of transcription from RNA polymerase II promoter” category (GO:0006357, 5.7-fold enrichment, FDR&nbsp;= 6.2&nbsp;× 10<sup>−6</sup>) and 9 in the “synaptic transmission” category (GO:0007268, 5.0-fold enrichment, FDR&nbsp;= 3.8&nbsp;× 10<sup>−3</sup>). For further analyses, we used a combination of Gene Ontology and&nbsp;primary literature to assign genes to GER (n&nbsp;= 58), NC (n&nbsp;=&nbsp;24), “cytoskeleton organization” (n&nbsp;= 9, GO:0007010), or “other” categories (<a id="crosref0905" href="#sec-4">STAR Methods</a>; <a id="crosref0910" href="#mmc4">Table S4</a>; <a id="crosref0915" href="#fig4">Figure&nbsp;4</a>E). Interestingly, ASD subjects who carry disruptive <i>de novo</i> variants in either GER or NC genes showed delayed age of walking and reduced IQ compared with those with no mutations in the 102 genes (<a id="crosref0920" href="#figs3">Figure&nbsp;S3</a>; <a id="crosref0925" href="#sec-4">STAR Methods</a>), yet carriers of disruptive variants in GER genes show significantly greater delays in age of walking compared with those with disruptive variants in NC genes.</div><div class="figure-wrap"><figure id="figs3" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/b9ba4526-627e-4893-9b64-561c447dc79b/main.assets/figs3_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs3" src="/cms/10.1016/j.cell.2019.12.036/asset/197a43ad-4045-410e-89d3-7d7eaaa5d91b/main.assets/figs3.jpg" height="530" width="375" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/b9ba4526-627e-4893-9b64-561c447dc79b/main.assets/figs3_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs3-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S3</span> <span class="figure__title__text">Phenotypic Distribution of Age of Walking and FSIQ in ASD Individuals, Split by Those Carrying <i>De Novo</i> Variants in GER Genes and NC Genes, Related to <a id="crosref1855" href="#fig3">Figure&nbsp;3</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs3-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0230" role="paragraph">A) Age of walking distribution for 4,456 ASD probands for whom age of walking data was available split into three groups: those carrying a <i>de novo</i> missense (MPC&nbsp;≥ 1) variant or PTV in a GER gene (N=140, purple), those carrying a <i>de novo</i> missense (MPC ≥ 1) variant or PTV in an NC gene (N=71, green), and the rest of the ASD population without such a <i>de novo</i> variant in the 102 FDR&lt;0.1 genes (N=4,204, grey). B) Full-scale IQ (FSIQ) distribution for 4,821 ASD probands for whom FSIQ was available split into three groups: those carrying a <i>de novo</i> missense (MPC ≥ 1) variant or PTV in a GER gene (N=159, purple), those carrying a <i>de novo</i> missense (MPC ≥ 1) variant or PTV in an NC gene (N=77, green), and the rest of the ASD population without such a <i>de novo</i> variant in the 102 FDR&lt;0.1 genes (N=4,542, grey).</div></div></div></div></figcaption></figure></div></section><section id="sec-2-10"><h3>ASD Genes Are Expressed Early in Brain Development</h3><div id="p0140" role="paragraph">The 102 ASD genes can be subdivided by phenotypic effect (53 ASD<sub>P</sub> genes, 49 ASD<sub>NDD</sub> genes) and functional role (58 GER genes, 24 NC genes) to give five gene sets (including all 102). We first evaluated enrichment of these five gene sets in the 53 tissues with bulk RNA sequencing (RNA-seq) data in the Genotype-Tissue Expression (GTEx) resource (<span class="dropBlock reference-citations"><a id="body-ref-sref3-1" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-1" aria-controls="sref3-1">Battle et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref3-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Battle, A. ∙ Brown, C.D. ∙ Engelhardt, B.E. ..., GTEx Consortium, Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group, Statistical Methods groups—Analysis Working Group, Enhancing GTEx (eGTEx) groups, NIH Common Fund, NIH/NCI, NIH/NHGRI, NIH/NIMH, NIH/NIDA, Biospecimen Collection Source Site—NDRI, Biospecimen Collection Source Site—RPCI, Biospecimen Core Resource—VARI, Brain Bank Repository—University of Miami Brain Endowment Bank, Leidos Biomedical—Project Management, ELSI Study, Genome Browser Data Integration &amp;Visualization—EBI, Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz, Lead analysts, Laboratory, Data Analysis &amp;Coordinating Center (LDACC), NIH program management, Biospecimen collection, Pathology, eQTL manuscript working group</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). To enhance tissue-specific resolution, we selected genes that were expressed in one tissue at a significantly higher level than the remaining 52 tissues; specifically, log<sub>2</sub> fold change of 0.5 or more and FDR of less than 0.05 (t test). Subsequently, we assessed over-representation of each ASD gene set within each of the 53 tissue-specific gene sets relative to a background of all other tissue-specific gene sets. Correcting for 53 tests, enrichment was observed in 11 of 13 brain regions, with the strongest enrichment in the cortex (30 genes, p&nbsp;= 3&nbsp;× 10<sup>−6</sup>, odds ratio [OR]&nbsp;= 3.7; <a id="crosref0935" href="#fig5">Figure&nbsp;5</a>A) and cerebellar hemisphere (48&nbsp;genes, p&nbsp;= 3&nbsp;× 10<sup>−6</sup>, OR&nbsp;= 2.9; <a id="crosref0940" href="#fig5">Figure&nbsp;5</a>A). Of the four gene subsets, NC genes were the most highly enriched in the cortex (17 of 23, p&nbsp;= 3&nbsp;× 10<sup>−11</sup>, OR&nbsp;= 25; <a id="crosref0945" href="#fig5">Figure&nbsp;5</a>A), whereas GER genes were the least enriched (10 of 58, p&nbsp;= 0.36, OR&nbsp;= 1.5; <a id="crosref0950" href="#fig5">Figure&nbsp;5</a>A; <a id="crosref0955" href="#mmc5">Table S5</a>). Notably, of the 102 ASD genes, only the cerebellar transcription factor <i>PAX5</i> (FDR&nbsp;= 0.005, TADA) was not expressed in the cortex (78 expected; p&nbsp;= 1&nbsp;× 10<sup>−9</sup>, binomial test).</div><div class="figure-wrap"><figure id="fig5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/03f8ce1b-2195-449a-b797-f4cbe8e92fb1/main.assets/gr5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="gr5" src="/cms/10.1016/j.cell.2019.12.036/asset/917a7389-dde6-457a-8163-a7a726128835/main.assets/gr5.jpg" height="831" width="770" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/03f8ce1b-2195-449a-b797-f4cbe8e92fb1/main.assets/gr5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="fig5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;5</span> <span class="figure__title__text">Analysis of 102 ASD-Associated Genes in the Context of Gene Expression Data</span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="fig5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0155" role="paragraph">(A) GTEx bulk RNA-seq data from 53 tissues were processed to identify genes enriched in specific tissues. Gene set enrichment was performed for the 102 ASD genes and four subsets (ASD<sub>P</sub>, ASD<sub>NDD</sub>, GER, and NC) for each tissue. Five representative tissues are shown here, including cortex, which has the greatest degree of enrichment (OR&nbsp;= 3.7; p&nbsp;= 2.6&nbsp;× 10<sup>−6</sup>).</div><div id="fspara0160" role="paragraph">(B) BrainSpan bulk RNA-seq data across 10 developmental stages was used to plot the normalized expression of the 101 cortically expressed ASD genes (excluding <i>PAX5</i>, which is not expressed in the cortex) across development, split by the four subsets.</div><div id="fspara0165" role="paragraph">(C) A t-statistic was calculated, comparing prenatal with postnatal expression in the BrainSpan data. The t-statistic distribution of 101 ASD-associated genes shows a prenatal bias (p&nbsp;= 8&nbsp;× 10<sup>−8</sup>) for GER genes (p&nbsp;= 9&nbsp;× 10<sup>−15</sup>), whereas NC genes are postnatally biased (p&nbsp;= 0.03).</div><div id="fspara0170" role="paragraph">(D) The cumulative number of ASD-associated genes expressed in RNA-seq data for 4,261 cells collected from human forebrain across prenatal development (<span class="dropBlock reference-citations"><a id="body-ref-sref70-1" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-1" aria-controls="sref70-1">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref70-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="fspara0175" role="paragraph">(E) t-SNE analysis identifies 19 clusters with unambiguous cell type in these single-cell expression data.</div><div id="fspara0180" role="paragraph">(F) The enrichment of the 102 ASD-associated genes within cells of each type is represented by color. The most consistent enrichment is observed in maturing and mature excitatory (bottom center) and inhibitory (top right) neurons.</div><div id="fspara0185" role="paragraph">(G) The developmental relationships of the 19 clusters are indicated by black arrows, with the inhibitory lineage shown on the left (cyan), excitatory lineage in the middle (magenta), and non-neuronal cell types on the right (gray). The proportion of the 102 ASD-associated genes observed in at least 25% of cells within the cluster is shown by the pie chart, whereas the log-transformed Bonferroni-corrected p value of gene set enrichment is shown by the size of the red circle.</div><div id="fspara0190" role="paragraph">(H) The relationship between the number of cells in the cluster (x axis) and the p value for ASD gene enrichment (y axis) is shown for the 19 cell type clusters. Linear regression indicates that clusters with few expressed genes (e.g., C23 newborn inhibitory neurons) have higher p values than clusters with many genes (e.g., C25 radial glia).</div><div id="fspara0195" role="paragraph">(I) The relationship between the 19 cell type clusters using hierarchical clustering based on the 10% of genes with the greatest variability among cell types.</div><div id="fspara0200" role="paragraph">Statistical tests: (A), t test; (C), Wilcoxon test; (E), (F), (H), and (I), FET.</div></div></div></div></figcaption></figure></div><div id="p0145" role="paragraph">Next, we developed a t-statistic that assesses the relative prenatal versus postnatal expression bias for each gene (<a id="crosref0960" href="#sec-4">STAR&nbsp;Methods</a>). Cortically expressed ASD genes are enriched prenatally (p&nbsp;= 8&nbsp;× 10<sup>−8</sup>, Wilcoxon test; <a id="crosref0965" href="#fig5">Figures 5</a>B and 5C). The ASD<sub>P</sub> and ASD<sub>NDD</sub> gene sets show similar patterns (<a id="crosref0970" href="#fig5">Figure&nbsp;5</a>B), although ASD<sub>NDD</sub> genes show more prenatal bias (p&nbsp;= 5&nbsp;× 10<sup>−6</sup>, Wilcoxon test; <a id="crosref0975" href="#fig5">Figure&nbsp;5</a>C). The GER genes display a marked prenatal bias (p&nbsp;= 9&nbsp;× 10<sup>−15</sup>, Wilcoxon test; <a id="crosref0980" href="#fig5">Figure&nbsp;5</a>C), reaching their highest levels during early to late fetal development (<a id="crosref0985" href="#fig5">Figure&nbsp;5</a>B), whereas the NC genes show postnatal bias&nbsp;(p&nbsp;= 0.03, Wilcoxon test; <a id="crosref0990" href="#fig5">Figure&nbsp;5</a>C), having their highest expression between late midfetal development and infancy (<a id="crosref0995" href="#fig5">Figure&nbsp;5</a>B). Applying unsupervised co-expression network analysis (weighted gene co-expression network analysis; WGCNA) to the BrainSpan gene expression data yielded enrichment for cortically-expressed ASD genes within discretely co-expressed groups of genes (i.e., modules) across development (<a id="crosref1000" href="#sec-4">STAR Methods</a>); however, GER and NC genes co-clustered separately (<a id="crosref1005" href="#figs4">Figure&nbsp;S4</a>; <a id="crosref1010" href="#mmc5">Table S5</a>). Thus, in keeping with prior analyses (<span class="dropBlock reference-citations"><a id="body-ref-sref113-1" href="#" role="doc-biblioref" data-xml-rid="bib9 bib73 bib109 bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-1" aria-controls="sref113-1">Chang et&nbsp;al., 2015; Parikshak et&nbsp;al., 2013; Willsey et&nbsp;al., 2013; Xu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref113-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Chang, J. ∙ Gilman, S.R. ∙ Chiang, A.H. ...</div><div><strong>Genotype to phenotype relationships in autism spectrum disorders</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:191-198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.3907" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84923186910" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25531569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.3907&amp;pmid=25531569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Parikshak, N.N. ∙ Luo, R. ∙ Zhang, A. ...</div><div><strong>Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1008-1021</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889561601" target="_blank" aria-label="Open Scopus in new tab">Scopus (434)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.031&amp;pmid=24267887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Willsey, A.J. ∙ Sanders, S.J. ∙ Li, M. ...</div><div><strong>Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:997-1007</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889583293" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.020&amp;pmid=24267886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Xu, X. ∙ Wells, A.B. ∙ O’Brien, D.R. ...</div><div><strong>Cell type-specific expression analysis to identify putative cellular mechanisms for neurogenetic disorders</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:1420-1431</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.4488-13.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84892720695" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24453331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.4488-13.2014&amp;pmid=24453331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), ASD genes are expressed at high levels in the human cortex and early in development. The differing expression patterns of GER and NC genes could reflect two distinct periods of ASD susceptibility during development or a single susceptibility period when both functional gene sets are highly expressed in mid-to-late fetal development.</div><div class="figure-wrap"><figure id="figs4" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/0415bdb6-a255-4e86-954d-2f83c91cc578/main.assets/figs4_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs4" src="/cms/10.1016/j.cell.2019.12.036/asset/1409390a-ef6f-4f3d-848f-7e2fc6853df4/main.assets/figs4.jpg" height="935" width="688" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/0415bdb6-a255-4e86-954d-2f83c91cc578/main.assets/figs4_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs4-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S4</span> <span class="figure__title__text">Enrichment of 102 ASD Genes in WGCNA Modules from Human Cortex and in Single-Cell Gene Expression Data, Related to <a id="crosref1860" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs4-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0240" role="paragraph">A) BrainSpan bulk RNA-sequencing data underwent extensive data quality control and data pre-processing. B) A total of 299 postmortem brain samples were used to assess prenatal versus postnatal bias in gene expression patterns for each individual gene. C) Next, a subset of 177 high-quality samples aged 8 postconception weeks (pcw) to 1 year of age were used to construct early developmental coexpression networks (i.e. modules). A total of 27 modules were identified, clustered by module eigengene (ME) values and assessed for enrichment for the 102 ASD risk genes and selected functional subsets. Node color intensity indicates significance level. Module number and the total number of genes within each module are labeled to the right of the enrichment plot. Splines were applied to capture non-linear ME effects while ensuring patterns of gene expression are continuous across early development. D) Peaking in expression during mid-fetal development, module M25 was enriched for ASD and GER genes and E) was enriched for chromatin modification/organization and histone modification/acetylation terms. F) Peaking in expression during neonatal/infancy, module M4 was enriched for NC genes and G) was implicated in transsynaptic signaling and cell-cell signaling. H) The cumulative number of ASD-associated genes expressed in RNA-seq data for different definitions of expression (0.01-0.25 of cells contain at least one transcript of the gene) and for 4,261 cells collected from human forebrain across prenatal development (<a id="crosref1865" href="#fig5">Figure&nbsp;5</a>D) in all cells from the excitatory lineage. I) The plot in ‘H’ is repeated for all cells in the inhibitory lineage. J) The enrichment of ASD associated genes in single cells arranged using tSNE (<a id="crosref1870" href="#fig5">Figure&nbsp;5</a>F) divided into the ASDP, ASDNDD, GER, and NC gene sets.</div></div></div></div></figcaption></figure></div></section><section id="sec-2-11"><h3>ASD Genes Are Enriched in Maturing Inhibitory and Excitatory Neurons</h3><div id="p0150" role="paragraph">Prior analyses have implicated excitatory glutamatergic neurons in the cortex and medium spiny neurons in the striatum in ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref113-2" href="#" role="doc-biblioref" data-xml-rid="bib9 bib73 bib109 bib113" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref113-2" aria-controls="sref113-2">Chang et&nbsp;al., 2015; Parikshak et&nbsp;al., 2013; Willsey et&nbsp;al., 2013; Xu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref113-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">9.</div><div>Chang, J. ∙ Gilman, S.R. ∙ Chiang, A.H. ...</div><div><strong>Genotype to phenotype relationships in autism spectrum disorders</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:191-198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.3907" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84923186910" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25531569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.3907&amp;pmid=25531569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Parikshak, N.N. ∙ Luo, R. ∙ Zhang, A. ...</div><div><strong>Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1008-1021</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889561601" target="_blank" aria-label="Open Scopus in new tab">Scopus (434)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.031&amp;pmid=24267887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Willsey, A.J. ∙ Sanders, S.J. ∙ Li, M. ...</div><div><strong>Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:997-1007</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889583293" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.020&amp;pmid=24267886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">109.</div><div>Xu, X. ∙ Wells, A.B. ∙ O’Brien, D.R. ...</div><div><strong>Cell type-specific expression analysis to identify putative cellular mechanisms for neurogenetic disorders</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:1420-1431</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.4488-13.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84892720695" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24453331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.4488-13.2014&amp;pmid=24453331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Here we perform a more direct assessment, examining expression of the 102 ASD-associated genes in an existing single-cell RNA-seq dataset of 4,261 cells from the prenatal human forebrain (<span class="dropBlock reference-citations"><a id="body-ref-sref70-2" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-2" aria-controls="sref70-2">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref70-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), ranging from 6 to 37 post-conception weeks (pcw) with an average of 16.3 pcw (<a id="crosref1020" href="#mmc5">Table S5</a>). We divided the cells into 17 developmental stages to assess the cumulative distribution of expressed genes by developmental endpoint (<a id="crosref1025" href="#fig5">Figure&nbsp;5</a>D). For each endpoint, a gene was defined as expressed when at least one transcript mapped to this gene in 25% or more of cells for 1 or more pcw stage. By definition, more genes were expressed as fetal development progressed, 4,481 by 13 pcw and 7,171 by 37 pcw. Although the majority of ASD genes (68) were expressed by 13 pcw, the number increased to 81 by 23 pcw, consistent with the BrainSpan data (<a id="crosref1030" href="#fig5">Figures 5</a>B and 5C). More liberal thresholds for expression resulted in higher numbers of ASD genes expressed (<a id="crosref1035" href="#fig5">Figure&nbsp;5</a>D), but the patterns were similar across thresholds and when considering gene function or cell type (<a id="crosref1040" href="#figs4">Figure&nbsp;S4</a>).</div><div id="p0155" role="paragraph">To investigate the cell types implicated in ASD, we considered 25 cell type clusters identified by t-distributed stochastic neighbor embedding (t-SNE) analysis, of which 19 clusters containing 3,839 cells were unambiguously associated with a cell type (<span class="dropBlock reference-citations"><a id="body-ref-sref70-3" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-3" aria-controls="sref70-3">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref70-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="crosref1050" href="#fig5">Figure&nbsp;5</a>E; <a id="crosref1055" href="#mmc5">Table S5</a>) and were used for enrichment analysis. Within each cell type cluster, a gene was considered expressed when at least one of its transcripts was detected in 25% or more of cells; 7,867 protein-coding genes met this criterion. Contrasting one cell type with the others, ASD genes are enriched in maturing and mature neurons of excitatory and inhibitory lineages (<a id="crosref1060" href="#fig5">Figures 5</a>F and 5G). Early excitatory neurons (C3) expressed the most ASD genes (72; OR&nbsp;= 5.0, p&nbsp;&lt; 1&nbsp;× 10<sup>−10</sup>, Fisher’s exact test [FET]), whereas the choroid plexus (C20) and microglia (C19) expressed the fewest (39; p&nbsp;= 0.09 and 0.14, respectively; FET); 14 genes were not expressed in any cluster (<a id="crosref1065" href="#fig5">Figure&nbsp;5</a>G). Within the major neuronal lineages, early excitatory neurons (C3) and striatal interneurons (C1) showed the greatest degree of enrichment (72&nbsp;and 51 genes, respectively; p&nbsp;&lt; 1&nbsp;× 10<sup>−10</sup>, FET; <a id="crosref1070" href="#fig5">Figures 5</a>F and 5G; <a id="crosref1075" href="#mmc5">Table S5</a>). Overall, maturing and mature neurons in the excitatory and inhibitory lineages showed a similar degree of enrichment, whereas the excitatory lineage expressed the most ASD genes, paralleling the larger numbers of genes expressed in excitatory lineage cells (<a id="crosref1080" href="#fig5">Figure&nbsp;5</a>H). The only non-neuronal cell type with significant enrichment was oligodendrocyte progenitor cells (OPCs) and astrocytes (C4; 62 genes, OR&nbsp;= 2.8, p&nbsp;= 8&nbsp;× 10<sup>−5</sup>, FET). Of the 62 genes expressed, 57 overlapped with radial glia, which share developmental origins with OPCs. These results are consistent with previous studies in post-mortem brain that identified dysregulation of gene expression in microglia but enriched expression of ASD risk genes only in neuronal cells (<span class="dropBlock reference-citations"><a id="body-ref-sref101" href="#" role="doc-biblioref" data-xml-rid="bib88 bib29 bib30 bib101" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref101" aria-controls="sref101">Ruzzo et&nbsp;al., 2019; Gandal et&nbsp;al., 2018a, 2018b; Voineagu et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref101"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">27.</div><div>Gandal, M.J. ∙ Haney, J.R. ∙ Parikshak, N.N. ..., CommonMind Consortium, PsychENCODE Consortium, iPSYCH-BROAD Working Group</div><div><strong>Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:693-697</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad6469" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85041925301" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29439242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad6469&amp;pmid=29439242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">28.</div><div>Gandal, M.J. ∙ Zhang, P. ∙ Hadjimichael, E. ...</div><div><strong>Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat8127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat8127" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058422390" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat8127&amp;pmid=30545856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">84.</div><div>Ruzzo, E.K. ∙ Pérez-Cano, L. ∙ Jung, J.Y. ...</div><div><strong>Inherited and de novo genetic risk for autism impacts shared networks</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:850-866.e26</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2019.07.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2019.07.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85070184891" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.015&amp;pmid=31398340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">97.</div><div>Voineagu, I. ∙ Wang, X. ∙ Johnston, P. ...</div><div><strong>Transcriptomic analysis of autistic brain reveals convergent molecular pathology</strong></div><div><em>Nature.</em> 2011; <strong>474</strong>:380-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-79959262465" target="_blank" aria-label="Open Scopus in new tab">Scopus (962)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21614001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10110&amp;pmid=21614001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Furthermore, recent results for single-cell analysis in mid-gestation human brain development also highlight enrichment for ASD gene expression in both excitatory and inhibitory lineages (<span class="dropBlock reference-citations"><a id="body-ref-sref77" href="#" role="doc-biblioref" data-xml-rid="bib77" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref77" aria-controls="sref77">Polioudakis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref77"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">74.</div><div>Polioudakis, D. ∙ de la Torre-Ubieta, L. ∙ Langerman, J. ...</div><div><strong>A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation</strong></div><div><em>Neuron.</em> 2019; <strong>103</strong>:785-801.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85070182242" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31303374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.06.011&amp;pmid=31303374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), along with some expression in non-neural cells without enrichment, as observed here. To validate the t-SNE clusters, we selected 10% of the expressed genes showing the greatest variability among the cell types and performed hierarchical clustering (<a id="crosref1090" href="#fig5">Figure&nbsp;5</a>I). This recaptured the division of these clusters by lineage (excitatory versus inhibitory) and by development stage (radial glia and progenitors versus neurons).</div><section id="sec-2-11-1"><h4>Prediction of Novel Risk Genes and Functional Relationships among ASD Genes</h4><div id="p0160" role="paragraph">ASD genes show convergent functional roles (<a id="crosref1095" href="#fig4">Figure&nbsp;4</a>E) and expression patterns in the cortex (<a id="crosref1100" href="#fig5">Figure&nbsp;5</a>B). Genes that are co-expressed with these ASD genes, interact with them, or are regulated by them could lend insight into convergent or auxiliary functions related to risk. In particular, we examined whether <i>in silico</i> network analyses would highlight additional risk genes and clarify the regulatory relationships between GER and NC genes. Three additional analyses were performed: the discovering association with networks (DAWN) approach to integrate TADA scores and gene co-expression data, enrichment analysis using protein-protein interaction (PPI) networks, and analyses using results from chromatin and cross-linked immunoprecipitation sequence assays to evaluate regulatory networks (<a id="crosref1105" href="#sec-4">STAR Methods</a>; <a id="crosref1110" href="#figs5">Figure&nbsp;S5</a>; <a id="crosref1115" href="#mmc5">Table S5</a>). Using the TADA results and BrainSpan gene co-expression data from the midfetal human cortex, DAWN yields 138 genes (FDR ≤ 0.005), including 83 genes that are not captured by TADA, with 69 of these 83 correlated with many other genes. Notably, 12 of the genes DAWN previously predicted as plausibly contributing to risk (<span class="dropBlock reference-citations"><a id="body-ref-sref21-7" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-7" aria-controls="sref21-7">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) were identified as new TADA genes here (enrichment p&nbsp;= 8.4&nbsp;× 10<sup>−11</sup>; OR&nbsp;= 16.4). To explore whether GER and NC gene sets interact more than would be expected by chance, we analyzed PPI networks and found that they do not; there was an excess of interactions among all ASD genes (82 genes, p&nbsp;= 0.02, FET), GER genes (49 genes, p&nbsp;= 0.006), and NC genes (12 genes, p&nbsp;= 0.03) but not among GER and NC genes (2 genes, p&nbsp;= 1.00). GER genes did not regulate the NC genes, according to our analyses, although GER-GER regulation was enriched. Even <i>CHD8</i>, a prominent and well-characterized ASD GER gene, did not regulate NC genes more than expected by chance (<a id="crosref1125" href="#figs5">Figure&nbsp;S5</a>).</div><div class="figure-wrap"><figure id="figs5" class="graphic"><a class="icon-full-screen" href="/cms/10.1016/j.cell.2019.12.036/asset/d482847c-bfef-4e07-91cd-8d0b65fd7209/main.assets/figs5_lrg.jpg" target="_blank" title="View full size image in a new tab"><img id="figs5" src="/cms/10.1016/j.cell.2019.12.036/asset/7989e58f-7719-4424-8751-5400a28b88fd/main.assets/figs5.jpg" height="794" width="770" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/d482847c-bfef-4e07-91cd-8d0b65fd7209/main.assets/figs5_lrg.jpg" loading="lazy"></a><figcaption class="figure__caption__body"><div id="figs5-title" class="figure__title"><span class="label figure__label">Figure&nbsp;S5</span> <span class="figure__title__text">Functional Relationships of ASD Risk Genes, Related to <a id="crosref1875" href="#fig5">Figure&nbsp;5</a></span></div><div class="accordion"><div><button class="accordion__control" aria-expanded="false" title="Expand"><span>Show full caption</span><i class="icon-section_arrow_d"></i></button><button class="figure__open__ctrl">Figure viewer</button></div><div id="figs5-content" class="accordion__content" style="display: none;"><div class="figure__caption__text__content"><div id="fspara0250" role="paragraph">A) ASD association data from TADA (<a id="crosref1880" href="#mmc2">Table S2</a>) is integrated with co-expression data from the midfetal human brain to implicate additional genes in ASD using DAWN (Discovering Association With Networks). The top 138 genes that share edges are shown (FDR ≤ 0.005). B) ASD-associated genes form a single protein-protein interaction network with more edges than expected by chance (p=0.01 using permutation). C) Experimental data, obtained using ChIP and CLIP methods across multiple species and a wide range of neuronal and non-neuronal tissues types, identified the regulatory targets of 26 GER genes (top circle). These data were used to assess whether three functionally defined groups of ASD-associated genes were enriched for regulatory targets, represented as arrows, weighted by the total number of regulatory targets for the GER genes. The expected number of targets in each functional group was estimated by permutation, controlling for brain expression, de novo PTV mutation rate, and pLI. For CHD8 enrichment analysis, genomic coordinates for GER and NC genes were assessed for enrichment for human brain-specific CHD8 binding sites derived from (D-E) human mid-fetal brain tissue and (F-G) human neural progenitor cells (NPCs). The regioneR R package was used to test overlaps of genomic regions based on permutation sampling. We sampled random regions from the genome 1,000 times, matching size and chromosomal distribution of the region set under study. By recomputing the overlap with CHD8 binding sites in each permutation, statistical significance of the observed overlap was computed. We observed significant enrichment for GER genes and CHD8 binding sites derived from (D) human midfetal brain tissue and (F) human NPCs. However, no significant enrichment was observed for NC genes from either study.</div></div></div></div></figcaption></figure></div></section></section></section><section id="sec-3"><h2>Discussion</h2><div id="p0165" role="paragraph">By characterizing rare <i>de novo</i> and inherited coding variation from 35,584 individuals, including 11,986 with ASD, we implicate 102 genes in risk for ASD at an FDR of 0.1 or less (<a id="crosref1130" href="#fig2">Figure&nbsp;2</a>), of which 30 are novel risk genes. Notably, analyses of the 102 risk genes led to novel genetic, phenotypic, and functional findings. Evidence of several of the genes is driven by missense variants, including confirmed gain-of-function mutations in the potassium channel <i>KCNQ3</i> and possible gain-of-function mutations in <i>DEAF1</i>, <i>SCN1A</i>, and <i>SLC6A1</i> (<a id="crosref1135" href="#fig3">Figure&nbsp;3</a>). Further, we strengthen evidence for driver genes in genomic disorder loci and propose a new driver gene, <i>BCL11A</i>, for the recurrent CNV at 2p15-p16.1. By evaluating GWAS results for ASD and related phenotypes and asking whether their common variant association signals overlap significantly with the 102 risk genes, we find substantial enrichment of GWAS signals for two traits genetically correlated with ASD—schizophrenia and educational attainment. For ASD itself, however, this enrichment is not significant, likely because of the limited power of the ASD GWAS. Despite this, <i>KMT2E</i> is significantly associated with ASD by both common and rare risk variation.</div><div id="p0170" role="paragraph">We performed a genetic partition between genes predominantly conferring liability for ASD (ASD<sub>P</sub>) and genes imparting risk to both ASD and NDD (ASD<sub>NDD</sub>). Three lines of evidence support the partition. First, cognitive impairment and motor delay are more severe in ASD subjects carrying mutations in ASD<sub>NDD</sub> than in ASD<sub>P</sub> genes (<a id="crosref1140" href="#fig4">Figures 4</a>B and 4C); second, inherited variation plays a lesser role in ASD<sub>NDD</sub> than in ASD<sub>P</sub> genes; and third, heterogeneity analysis demonstrates clear distinctions between the two groups of genes. Thus, ASD-associated genes are distributed across a spectrum of phenotypes and selective pressure. At one extreme, gene haploinsufficiency leads to global developmental delay with impaired cognitive, social, and gross motor skills, leading to strong negative selection (e.g., <i>ANKRD11</i>, <i>ARID1B</i>). At the other extreme, gene haploinsufficiency leads to ASD, and there is more modest involvement of other developmental phenotypes and selective pressure (e.g., <i>GIGYF1</i>, <i>ANK2</i>). This distinction has important ramifications for clinicians, geneticists, and neuroscientists because it suggests that clearly delineating the effect of these genes across neurodevelopmental dimensions could offer a route to deconvolve the social dysfunction and repetitive behaviors that define ASD from more general neurodevelopmental impairment. Larger cohorts will be required to reliably identify specific genes as being enriched in ASD compared with NDD.</div><div id="p0175" role="paragraph">Single-cell gene expression data from the developing human cortex implicate mid-to-late fetal development and maturing and mature neurons in both excitatory and inhibitory lineages in ASD risk (<a id="crosref1145" href="#fig5">Figure&nbsp;5</a>). Expression of GER genes shows a prenatal bias whereas expression of NC genes does not. Placing these results in the context of multiple non-exclusive hypotheses around the origins of ASD, it is intriguing to speculate that the NC ASD genes provide compelling support for excitatory-inhibitory imbalance in ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref87" href="#" role="doc-biblioref" data-xml-rid="bib87" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref87" aria-controls="sref87">Rubenstein and Merzenich, 2003</a><div class="dropBlock__holder" data-db-target-of="sref87"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">83.</div><div>Rubenstein, J.L. ∙ Merzenich, M.M.</div><div><strong>Model of autism: increased ratio of excitation/inhibition in key neural systems</strong></div><div><em>Genes Brain Behav.</em> 2003; <strong>2</strong>:255-267</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1034/j.1601-183X.2003.00037.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-0242291090" target="_blank" aria-label="Open Scopus in new tab">Scopus (1284)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14606691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1034%2Fj.1601-183X.2003.00037.x&amp;pmid=14606691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) through direct effects on neurotransmission. However, because there was no support for a regulatory role for GER ASD genes on either NC or cytoskeletal ASD genes, additional mechanisms having to do with cell migration and neurodevelopment also appear to be at play. This might suggest that GER ASD genes affect the excitatory-inhibitory balance by altering the numbers of excitatory and inhibitory neurons in given regions of the brain. ASD must arise by phenotypic convergence among these diverse neurobiological trajectories, and further dissecting the nature of this convergence, especially in the genes we identified here, is likely to hold the key to understanding the developmental neurobiology that underlies the ASD phenotype.</div></section><section id="sec-4"><h2>STAR★Methods</h2><section id="sec-4-1"><h3>Key Resources Table</h3><div id="p0185" role="paragraph"><div class="figure-wrap"><figure id="undtbl1" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true" tabindex="0"><thead><tr class="rowsep"><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr class="rowsep"><td colspan="3"><b>Deposited Data</b></td></tr><tr><td>ASC-generated WES sequencing data</td><td>This paper</td><td>dbGaP Study Accession: phs000298.v4.p3</td></tr><tr><td>SFARI-generated WES sequencing data</td><td>SFARI</td><td><a id="intref0010" href="https://www.sfari.org/resource/sfari-base/" target="_blank">https://www.sfari.org/resource/sfari-base/</a></td></tr><tr><td>iPSYCH-generated WES sequencing data</td><td>iPSYCH-Broad consortium</td><td><a id="intref0015" href="http://ipsych.genome.au.dk/" target="_blank">http://ipsych.genome.au.dk/</a></td></tr><tr><td>Human reference genome NCBI build 37, GRCh37</td><td>Genome Reference Consortium</td><td><a id="intref0020" href="https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/" target="_blank">https://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/</a></td></tr><tr><td>Exome aggregation consortium (ExAC)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref51-3" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-3" aria-controls="sref51-3">Lek et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0025" href="http://exac.broadinstitute.org/" target="_blank">http://exac.broadinstitute.org/</a></td></tr><tr><td>Genome aggregation database (gnomAD)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref44" href="#" role="doc-biblioref" data-xml-rid="bib44" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref44" aria-controls="sref44">Karczewski et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref44"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">42.</div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ...</div><div><strong>Variation across 141,456 human exomes and genomes reveals the spectrum of loss-of-function intolerance across human protein-coding genes</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/531210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85094417987" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F531210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0030" href="https://gnomad.broadinstitute.org/" target="_blank">https://gnomad.broadinstitute.org/</a></td></tr><tr><td>Deciphering Developmental Disorders (DDD)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref22-2" href="#" role="doc-biblioref" data-xml-rid="bib22" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref22-2" aria-controls="sref22-2">Deciphering Developmental Disorders Study, 2017</a><div class="dropBlock__holder" data-db-target-of="sref22-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">20.</div><div>Deciphering Developmental Disorders Study</div><div><strong>Prevalence and architecture of de novo mutations in developmental disorders</strong></div><div><em>Nature.</em> 2017; <strong>542</strong>:433-438</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature21062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85015375341" target="_blank" aria-label="Open Scopus in new tab">Scopus (371)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28135719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature21062&amp;pmid=28135719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0035" href="https://www.ddduk.org/" target="_blank">https://www.ddduk.org/</a></td></tr><tr><td>Genotype-Tissue Expression (GTEx) resource</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref3-2" href="#" role="doc-biblioref" data-xml-rid="bib3" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref3-2" aria-controls="sref3-2">Battle et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref3-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">3.</div><div>Battle, A. ∙ Brown, C.D. ∙ Engelhardt, B.E. ..., GTEx Consortium, Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group, Statistical Methods groups—Analysis Working Group, Enhancing GTEx (eGTEx) groups, NIH Common Fund, NIH/NCI, NIH/NHGRI, NIH/NIMH, NIH/NIDA, Biospecimen Collection Source Site—NDRI, Biospecimen Collection Source Site—RPCI, Biospecimen Core Resource—VARI, Brain Bank Repository—University of Miami Brain Endowment Bank, Leidos Biomedical—Project Management, ELSI Study, Genome Browser Data Integration &amp;Visualization—EBI, Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz, Lead analysts, Laboratory, Data Analysis &amp;Coordinating Center (LDACC), NIH program management, Biospecimen collection, Pathology, eQTL manuscript working group</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0040" href="https://gtexportal.org/home/" target="_blank">https://gtexportal.org/home/</a></td></tr><tr><td>BrainSpan</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref56-1" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-1" aria-controls="sref56-1">Li et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref56-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Li, M. ∙ Santpere, G. ∙ Imamura Kawasawa, Y. ..., BrainSpan Consortium, PsychENCODE Consortium, PsychENCODE Developmental Subgroup</div><div><strong>Integrative functional genomic analysis of human brain development and neuropsychiatric risks</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat7615</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat7615" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058377382" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545854/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat7615&amp;pmid=30545854" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0045" href="http://www.brainspan.org/" target="_blank">http://www.brainspan.org/</a></td></tr><tr><td>Single-cell RNA-seq data from developing cortex</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref70-4" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-4" aria-controls="sref70-4">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref70-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0050" href="https://cells.ucsc.edu/?ds=cortex-dev" target="_blank">https://cells.ucsc.edu/?ds=cortex-dev</a></td></tr><tr class="rowsep"><td>InWeb_IM (protein-protein interaction data)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref55-1" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-1" aria-controls="sref55-1">Li et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref55-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Li, T. ∙ Wernersson, R. ∙ Hansen, R.B. ...</div><div><strong>A scored human protein-protein interaction network to catalyze genomic interpretation</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:61-64</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84997794652" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27892958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4083&amp;pmid=27892958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0055" href="http://www.lagelab.org/resources/" target="_blank">http://www.lagelab.org/resources/</a></td></tr><tr class="rowsep"><td colspan="3"><b>Software and Algorithms</b></td></tr><tr><td>Genome Analysis Toolkit (GATK)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref100-1" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-1" aria-controls="sref100-1">Van der Auwera et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref100-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Van der Auwera, G.A. ∙ Carneiro, M.O. ∙ Hartl, C. ...</div><div><strong>From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline</strong></div><div><em>Curr. Protoc. Bioinformatics.</em> 2013; <strong>11</strong>:11.10.1-11.10.33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.A.Van+der+AuweraM.O.CarneiroC.HartlR.PoplinG.Del+AngelA.Levy-MoonshineT.JordanK.ShakirD.RoazenJ.ThibaultFrom+FastQ+data+to+high+confidence+variant+calls%3A+the+Genome+Analysis+Toolkit+best+practices+pipelineCurr.+Protoc.+Bioinformatics11201311.10.111.10.33" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0060" href="https://software.broadinstitute.org/gatk/" target="_blank">https://software.broadinstitute.org/gatk/</a></td></tr><tr><td>Hail</td><td><a id="intref0065" href="https://hail.is/" target="_blank">https://hail.is/</a></td><td><a id="intref0070" href="https://github.com/hail-is/hail/" target="_blank">https://github.com/hail-is/hail/</a></td></tr><tr><td>Variant Effect Predictor (VEP)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref63-1" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-1" aria-controls="sref63-1">McLaren et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref63-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>McLaren, W. ∙ Gil, L. ∙ Hunt, S.E. ...</div><div><strong>The Ensembl Variant Effect Predictor</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0974-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84973338712" target="_blank" aria-label="Open Scopus in new tab">Scopus (900)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27268795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0974-4&amp;pmid=27268795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0075" href="http://grch37.ensembl.org/Homo_sapiens/Tools/VEP" target="_blank">http://grch37.ensembl.org/Homo_sapiens/Tools/VEP</a></td></tr><tr><td>TADA</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref35-4" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-4" aria-controls="sref35-4">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0080" href="http://www.compgen.pitt.edu/TADA/TADA_guide.html" target="_blank">http://www.compgen.pitt.edu/TADA/TADA_guide.html</a></td></tr><tr><td>Gene Ontology (via Panther)</td><td><span class="dropBlock reference-citations"><a id="body-ref-sref64" href="#" role="doc-biblioref" data-xml-rid="bib64" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref64" aria-controls="sref64">Mi et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref64"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">62.</div><div>Mi, H. ∙ Muruganujan, A. ∙ Huang, X. ...</div><div><strong>Protocol Update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0)</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:703-721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0128-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062109631" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0128-8&amp;pmid=30804569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span></td><td><a id="intref0085" href="http://www.pantherdb.org/" target="_blank">http://www.pantherdb.org/</a></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl1&amp;pii=S0092-8674%2819%2931398-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div></section><section id="sec-4-2"><h3>Lead Contact and Materials Availability</h3><section id="sec-4-2-1"><h4>Lead Contact</h4><div id="p0190" role="paragraph">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Joseph D. Buxbaum (<a id="intref0090" href="mailto:joseph.buxbaum@mssm.edu">joseph.buxbaum@mssm.edu</a>).</div></section></section><section id="sec-4-3"><h3>Materials Availability</h3><div id="p0195" role="paragraph">This study did not generate new unique reagents.</div></section><section id="sec-4-4"><h3>Experimental Model and Subject Details</h3><section id="sec-4-4-1"><h4>Overview of the Autism Sequencing Consortium cohort</h4><div id="p0200" role="paragraph">The Autism Sequencing Consortium (ASC) is a large-scale international genomic consortium integrating ASD cohorts and sequencing data from over one hundred investigators (<span class="dropBlock reference-citations"><a id="body-ref-sref8" href="#" role="doc-biblioref" data-xml-rid="bib8" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref8" aria-controls="sref8">Buxbaum et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">8.</div><div>Buxbaum, J.D. ∙ Daly, M.J. ∙ Devlin, B. ..., Autism Sequencing Consortium</div><div><strong>The autism sequencing consortium: large-scale, high-throughput sequencing in autism spectrum disorders</strong></div><div><em>Neuron.</em> 2012; <strong>76</strong>:1052-1056</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2012.12.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2012.12.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84871371945" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23259942/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.12.008&amp;pmid=23259942" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; <a id="intref0095" href="https://genome.emory.edu/ASC/" target="_blank">https://genome.emory.edu/ASC/</a>).</div><div id="p0205" role="paragraph">At the outset, the ASC aggregated data from ASC sites, but over the past several years it has also been able to sequence samples at the Broad Institute/Massachusetts General Hospital through the Broad Center for Common Disease Genomics (UM1HG008895, Mark Daly, PI).</div><div id="p0210" role="paragraph">The analysis presented here is based on 35,584 unique human samples collected from 32 distinct sample sets (<a id="crosref1215" href="#mmc1">Table S1</a>). These include cohorts sequenced by the Autism Sequencing Consortium (ASC) and published in our first (<span class="dropBlock reference-citations"><a id="body-ref-sref21-8" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-8" aria-controls="sref21-8">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or second study (<span class="dropBlock reference-citations"><a id="body-ref-sref57" href="#" role="doc-biblioref" data-xml-rid="bib57" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref57" aria-controls="sref57">Lim et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref57"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">55.</div><div>Lim, E.T. ∙ Uddin, M. ∙ De Rubeis, S. ..., Autism Sequencing Consortium</div><div><strong>Rates, distribution and implications of postzygotic mosaic mutations in autism spectrum disorder</strong></div><div><em>Nat. Neurosci.</em> 2017; <strong>20</strong>:1217-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4598" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85028454831" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28714951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4598&amp;pmid=28714951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) (Germany, Japan, PAGES, Pittsburgh, Seaver, Spain, TASC, and UCSF), as well as new collections (Boston, Brazil, CHARGE, Chicago, Hong Kong, Miami, Portugal, Rome, Siena, Turin, UC Irvine, and Utah), with a total of 6,197 newly collected and sequenced samples included in our final analysis. We also sequenced samples from the Autism Genetic Resource Exchange (AGRE), the Boston Autism Consortium, two sites in Finland, and Swedish controls from epidemiological studies in schizophrenia and bipolar disorder. We imported exome sequence data from the Simons Simplex Collection (<span class="dropBlock reference-citations"><a id="body-ref-sref38-3" href="#" role="doc-biblioref" data-xml-rid="bib38" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref38-3" aria-controls="sref38-3">Iossifov et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref38-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">36.</div><div>Iossifov, I. ∙ O’Roak, B.J. ∙ Sanders, S.J. ...</div><div><strong>The contribution of de novo coding mutations to autism spectrum disorder</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:216-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13908" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912101541" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13908&amp;pmid=25363768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as well as an unpublished Norwegian cohort, and included them in our dataset&nbsp;alongside ASC-sequenced samples.</div><div id="p0215" role="paragraph">In addition, we incorporated published <i>de novo</i> variants from the UK10K consortium, the University of Pennsylvania, Vanderbilt University, and a collection of samples from the Middle East. Finally, we integrated gene-level variant counts from autism cases and matched controls from the iPSYCH research initiative (<span class="dropBlock reference-citations"><a id="body-ref-sref93-2" href="#" role="doc-biblioref" data-xml-rid="bib49 bib74 bib93" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref93-2" aria-controls="sref93-2">Lauritsen et&nbsp;al., 2010; Pedersen et&nbsp;al., 2018; Satterstrom et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref93-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">47.</div><div>Lauritsen, M.B. ∙ Jørgensen, M. ∙ Madsen, K.M. ...</div><div><strong>Validity of childhood autism in the Danish Psychiatric Central Register: findings from a cohort sample born 1990-1999</strong></div><div><em>J.&nbsp;Autism Dev. Disord.</em> 2010; <strong>40</strong>:139-148</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10803-009-0818-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-75949111242" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19728067/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10803-009-0818-0&amp;pmid=19728067" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">72.</div><div>Pedersen, C.B. ∙ Bybjerg-Grauholm, J. ∙ Pedersen, M.G. ...</div><div><strong>The iPSYCH2012 case-cohort sample: new directions for unravelling genetic and environmental architectures of severe mental disorders</strong></div><div><em>Mol. Psychiatry.</em> 2018; <strong>23</strong>:6-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2017.196" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85040051369" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28924187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2017.196&amp;pmid=28924187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">89.</div><div>Satterstrom, F.K. ∙ Walters, R.K. ∙ Singh, T. ...</div><div><strong>ASD and ADHD have a similar burden of rare protein-truncating variants</strong></div><div><em>bioRxiv.</em> 2018;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/277707" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85095617614" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F277707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A description of each cohort, with the number of samples sequenced and the number used in our analyses, its ascertainment and diagnostic strategy, and associated references is found in Table S1. This table also contains details on the family relationships, sex, and phenotypic status of all samples.</div></section><section id="sec-4-4-2"><h4>Informed consent and study approval</h4><div id="p0220" role="paragraph">The ASC and ASC sites are approved by appropriate Institutional Review Boards or Ethical Committees and informed consent was obtained from all subjects. The individual studies that contribute to the ASC may have directly ascertained and interviewed human clinical subjects, along with controls, in accord with the ethical principles and practices of modern biomedical research. In contrast to contributing sites, the ASC as a group has a different relationship to human subjects. The ASC has no direct contact with the research subjects and no identifying information is provided by the primary sites to the ASC. From the perspective of the ASC, all data is de-identified and effectively anonymized.</div><div id="p0225" role="paragraph">For samples provided to the ASC sequencing site (Broad-MGH through their funding from NHGRI), a copy of the consent(s) and a signed IRB Data Use Letter (DUL) was received before samples were accepted. The DUL confirms that the samples can be shared and are suitable for upload to dbGaP and NDAR.</div><div id="p0230" role="paragraph">The iPSYCH study was approved by the Regional Scientific Ethics Committee in Denmark and the Danish Data Protection Agency.</div></section></section><section id="sec-4-5"><h3>Method Details</h3><section id="sec-4-5-1"><h4>Exome sequencing and data processing</h4><div id="p0235" role="paragraph">The bulk of new ASC samples were sequenced at the Broad Institute on Illumina HiSeq sequencers using the Illumina Nextera exome capture kit. The remainder were sequenced at three other sites: the University of California, San Franciso (N&nbsp;= 495), the Sanger Institute (N&nbsp;= 443), and Johns Hopkins University (N&nbsp;= 302), all using similar methods. Sequencing reads were aligned to human genome build 37 (GRCh37/hg19) using the Burrows-Wheeler Aligner (BWA, <span class="dropBlock reference-citations"><a id="body-ref-sref54" href="#" role="doc-biblioref" data-xml-rid="bib54" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref54" aria-controls="sref54">Li and Durbin, 2009</a><div class="dropBlock__holder" data-db-target-of="sref54"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">52.</div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (15889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), aggregated into a BAM file. Picard (<a id="intref0100" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>) was used for sorting by chromosome coordinates and marking duplicates. Single nucleotide variants (SNVs) and insertions / deletions (indels) were jointly called across all samples using the Genome Analysis Toolkit (GATK; <span class="dropBlock reference-citations"><a id="body-ref-sref100-2" href="#" role="doc-biblioref" data-xml-rid="bib100" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref100-2" aria-controls="sref100-2">Van&nbsp;der Auwera et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref100-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">96.</div><div>Van der Auwera, G.A. ∙ Carneiro, M.O. ∙ Hartl, C. ...</div><div><strong>From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline</strong></div><div><em>Curr. Protoc. Bioinformatics.</em> 2013; <strong>11</strong>:11.10.1-11.10.33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.A.Van+der+AuweraM.O.CarneiroC.HartlR.PoplinG.Del+AngelA.Levy-MoonshineT.JordanK.ShakirD.RoazenJ.ThibaultFrom+FastQ+data+to+high+confidence+variant+calls%3A+the+Genome+Analysis+Toolkit+best+practices+pipelineCurr.+Protoc.+Bioinformatics11201311.10.111.10.33" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) HaplotypeCaller package version 3.4. Variant call accuracy was estimated using the GATK Variant Quality Score Recalibration (VQSR) approach. The VCF file (format v4.1) was produced by the Broad sequencing and calling pipeline with GATK version 3.4 (g3c929b0).</div></section></section><section id="sec-4-6"><h3>Quantification and Statistical Analysis</h3><section id="sec-4-6-1"><h4>Dataset Quality Control</h4><div id="p0240" role="paragraph">The VCF file, containing approximately 29,000 exomes, was loaded into Hail 0.1 (<a id="intref0105" href="https://hail.is/" target="_blank">https://hail.is/</a>; <a id="intref0110" href="https://github.com/hail-is/hail" target="_blank">https://github.com/hail-is/hail</a>) to perform basic quality control steps. Multi-allelic sites were split into bi-allelic sites and each variant was then annotated with the Variant Effect Predictor (VEP, <span class="dropBlock reference-citations"><a id="body-ref-sref63-2" href="#" role="doc-biblioref" data-xml-rid="bib63" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref63-2" aria-controls="sref63-2">McLaren et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref63-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">61.</div><div>McLaren, W. ∙ Gil, L. ∙ Hunt, S.E. ...</div><div><strong>The Ensembl Variant Effect Predictor</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0974-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84973338712" target="_blank" aria-label="Open Scopus in new tab">Scopus (900)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27268795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0974-4&amp;pmid=27268795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) by prioritizing coding canonical transcripts. VEP assigned properties such as gene name and consequence to each variant. An overview of the quality control and data cleaning process is shown in <a id="crosref1250" href="#figs1">Figure&nbsp;S1</a>.</div><section id="sec-4-6-1-1"><h5>Round 1, sample-level quality control</h5><div id="p0245" role="paragraph">To check the accuracy of the reported pedigree information, relatedness was calculated between each pair of samples using Hail’s ibd() function and sex was imputed for each sample using Hail’s impute_sex() function. The relatedness values were input into the program PRIMUS (<span class="dropBlock reference-citations"><a id="body-ref-sref98" href="#" role="doc-biblioref" data-xml-rid="bib98" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref98" aria-controls="sref98">Staples et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref98"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">94.</div><div>Staples, J. ∙ Qiao, D. ∙ Cho, M.H. ..., University of Washington Center for Mendelian Genomics</div><div><strong>PRIMUS: rapid reconstruction of pedigrees from genome-wide estimates of identity by descent</strong></div><div><em>Am. J. Hum. Genet.</em> 2014; <strong>95</strong>:553-564</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.10.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.10.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25439724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2014.10.005&amp;pmid=25439724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which inferred pedigree structure. Combined with the imputed sex, these inferred pedigrees were compared to reported pedigrees and checked for discrepancies. Obvious errors in reporting were fixed (e.g., swapped mother/father or parent/child labels in the same family), and samples with a discrepancy that could not be resolved (∼200) were dropped. Parents without a child in the dataset (∼250) were also dropped, resulting in 28,547 samples and 5,420,608 unique variants.</div></section><section id="sec-4-6-1-2"><h5>Round 1, variant-level quality control</h5><div id="p0250" role="paragraph">Low-complexity regions were removed (110,963 variants), as were SNVs that failed VQSR (265,130 variants), leaving 5,044,515 unique variants. For genotype quality control, several filters were applied: we filtered calls with a depth less than 10 or greater than 1,000; for homozygous reference calls, we filtered genotypes with less than 90% of the read depth supporting the reference allele or with a genotype quality less than 25; for homozygous variant calls, we filtered genotypes with less than 90% of the read depth supporting the alternate allele or with a Phred-scaled likelihood (PL) of being homozygous reference less than 25; and for heterozygous calls, we filtered genotypes with less than 90% of the read depth supporting either the reference or alternate allele, with a PL of being homozygous reference less than 25, with less than 25% of the read depth supporting the alternate allele (i.e., an allele balance less than 0.25), or with a probability of the allele balance (calculated from a binomial distribution centered on 0.5) less than 1x10<sup>−9</sup>. We additionally filtered any heterozygous call in the X or Y non-pseudoautosomal regions in a sample that imputed as male. For samples imputed as female, calls from the Y chromosome were removed. After applying these filters and removing sites that were no longer variant, the dataset contained 28,547 samples and 4,755,048 unique variants.</div></section><section id="sec-4-6-1-3"><h5>Round 2, sample-level quality control</h5><div id="p0255" role="paragraph">We applied further sample-level quality control filters using the filtered variants. We removed samples with estimated contamination levels using FREEMIX &gt; 7.5% (20 samples) (<span class="dropBlock reference-citations"><a id="body-ref-sref42" href="#" role="doc-biblioref" data-xml-rid="bib42" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref42" aria-controls="sref42">Jun et&nbsp;al., 2012</a><div class="dropBlock__holder" data-db-target-of="sref42"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">40.</div><div>Jun, G. ∙ Flickinger, M. ∙ Hetrick, K.N. ...</div><div><strong>Detecting and estimating contamination of human DNA samples in sequencing and array-based genotype data</strong></div><div><em>Am. J. Hum. Genet.</em> 2012; <strong>91</strong>:839-848</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2012.09.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2012.09.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84868490540" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23103226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2012.09.004&amp;pmid=23103226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) or chimeric reads &gt; 7.5% (121 samples). Stratifying samples into 18 different batches (by exome capture/year/cohort/sequencing center, see <a id="crosref1265" href="#mmc1">Table S1</a>), samples were filtered if their call rate was greater than 3 standard deviations below the group mean (300 samples). Duplicate samples were then removed (761 samples), as were samples for which the imputed sex did not match the reported sex (59 samples). Following these sample filters, family structures were re-evaluated: if one or more parents of a case in the family-based data (which we also refer to as a “proband”) had been filtered, the proband was reclassified as a case in the case-control data and the remaining parent (if any) was dropped; if the proband had an unaffected sibling, the sibling was kept as a “sibling of case” (not used in this study); if one or more parents were filtered and no proband remained, then data for remaining family members were removed; and relatives not relevant for calling <i>de novo</i> variants (such as aunts or uncles) were removed. After applying these rules, the dataset contained 5,833 complete families, with 5,924 affected probands, 2,007 unaffected offspring, 5,834 fathers, and 5,833 mothers (one family contained two probands, two fathers, and one mother).</div><div id="p0260" role="paragraph">The dataset&nbsp;also contained 2,388 cases, 106 siblings of cases, and 4,324 controls, none of whom were part of a complete trio. To prevent complicated patterns of dependency, we excluded all but one sample (or one case-sibling pair) from each group of related samples within these categories. We defined related samples using a KING (<span class="dropBlock reference-citations"><a id="body-ref-sref62-2" href="#" role="doc-biblioref" data-xml-rid="bib62" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref62-2" aria-controls="sref62-2">Manichaikul et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref62-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">60.</div><div>Manichaikul, A. ∙ Mychaleckyj, J.C. ∙ Rich, S.S. ...</div><div><strong>Robust relationship inference in genome-wide association studies</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2867-2873</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq559" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-78149265272" target="_blank" aria-label="Open Scopus in new tab">Scopus (539)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20926424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq559&amp;pmid=20926424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) kinship value of 0.1 or greater, approximately equivalent to a Pi-Hat of 0.2 or greater. After this filtering, the dataset contained 2,353 cases, 100 siblings of cases, and 4,316 controls, for a total of 26,367 samples.</div></section><section id="sec-4-6-1-4"><h5>Round 2, variant-level quality control</h5><div id="p0265" role="paragraph">After filtering sites that were no longer variant due to sample exclusion, there were 4,605,130 unique variants. For a second round of variant quality control, variants with call rate&nbsp;&lt; 10% (17,083 variants) or a Hardy-Weinberg equilibrium p value less than 1x10<sup>−12</sup> (27,862 variants) were excluded, leaving 26,367 samples and 4,560,185 unique variants. This dataset was then used as the starting point for the <i>de novo</i>, inherited, and case-control workflows.</div></section></section><section id="sec-4-6-2"><h4>Defining rare and <i>de novo</i> variants</h4><section id="sec-4-6-2-1"><h5>De novo variation</h5><div id="p0270" role="paragraph"><i>De novo</i> variants were called from the 26,367-sample dataset described above, including 5,924 affected probands and 2,007 unaffected offspring (7,931 total children). After filtering any genotype with a GQ&nbsp;&lt; 25, <i>de novo</i> variants were called using the de_novo() function of Hail 0.1, which implements the caller used in previous ASC work (<a id="intref0115" href="https://github.com/ksamocha/de_novo_scripts" target="_blank">https://github.com/ksamocha/de_novo_scripts</a>). Population allele frequencies for variants were obtained from the non-psychiatric subset of gnomAD (<a id="intref0120" href="https://gnomad.broadinstitute.org/" target="_blank">https://gnomad.broadinstitute.org/</a>) and these frequencies were used as the input priors. As additional parameters, parents’ homozygous reference genotypes were required to have no more than 3% of reads supporting the alternate allele, children’s heterozygous calls were required to have at least 30% of reads supporting the alternate allele, and the ratio of child read depth to parental read depth was required to be at least 0.3.</div><div id="p0275" role="paragraph">This process identified 44,562 putative <i>de novo</i> variants at 26,577 distinct genomic locations in the 7,931 children in the dataset. Of the 7,931 children, 519 were also part of a whole-genome sequencing project (<span class="dropBlock reference-citations"><a id="body-ref-sref118" href="#" role="doc-biblioref" data-xml-rid="bib107" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref118" aria-controls="sref118">Werling et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref118"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">103.</div><div>Werling, D.M. ∙ Brand, H. ∙ An, J.Y. ...</div><div><strong>An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2018; <strong>26</strong>:727-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0107-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85046026156" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0107-y" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and we added a further 168 <i>de&nbsp;novo</i> variants called in protein-coding regions of these samples from the whole-genome sequencing that were not called in the exome sequencing. We also incorporated 338 previously published and validated <i>de novo</i> variants in our samples that were not identified by our caller (<span class="dropBlock reference-citations"><a id="body-ref-sref46-3" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-3" aria-controls="sref46-3">Kosmicki et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref46-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Thus, in total, we had 45,068 putative <i>de novo</i> variants at 27,083 distinct loci in 7,931 children. For quality control on the <i>de novo</i> variants, we retained variants if they were high confidence as indicated by the calling algorithm, medium confidence and a singleton in the dataset, or previously experimentally validated (20,862 putative <i>de novo</i> variants included). To remove calls stemming from cell line artifacts, an allele balance of at least 0.4 was required for the 773 probands and 40 siblings for whom data were generated from immortalized cell line DNA (2,171 putative <i>de novo</i> calls excluded). Next, a call was removed if it had an allele frequency &gt; 0.1% across the samples in our dataset, in the non-psychiatric subset of ExAC (r0.3,&nbsp;<a id="intref0125" href="http://exac.broadinstitute.org/" target="_blank">http://exac.broadinstitute.org/</a>), or in the non-psychiatric subset of gnomAD (5,068 putative <i>de novo</i> variants excluded). Calls were excluded if they appeared more than twice in the remaining list of putative <i>de novo</i> variants (403 putative <i>de novo</i> variants excluded) and were then limited to one variant per person per gene (570 putative <i>de novo</i> variants excluded), retaining variants with the most severe consequence when selecting which one to keep. Finally, samples whose DNA source was whole-blood or saliva were excluded if they had more than seven protein-coding putative <i>de novo</i> variants (20 out of 5,143 probands and 13 out of 1,967 unaffected children excluded). Samples whose DNA source was immortalized cell lines were dropped if they had more than five protein-coding putative <i>de novo</i> variants (35 out of 773 probands and 1 out of 40 unaffected children excluded). After applying these filters, the remaining list of high confidence <i>de novo</i> variants included 14,569 <i>de novo</i> variants from 5,869 probands and 1,993 unaffected children. To maximize power and improve consistency with prior analyses, we supplemented this set with 933 and 287 published <i>de novo</i> variants in 561 probands and 186 siblings (<span class="dropBlock reference-citations"><a id="body-ref-sref92-4" href="#" role="doc-biblioref" data-xml-rid="bib21 bib92 bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-4" aria-controls="sref92-4">De Rubeis et&nbsp;al., 2014; Sanders et&nbsp;al., 2015; Kosmicki et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref92-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), respectively, for whom original sequence data were not available. The final list of high confidence <i>de novo</i> variants included 15,789 <i>de novo</i> variants from 6,430 probands and 2,179 unaffected children (<a id="crosref1285" href="#mmc1">Table S1</a>; <a id="crosref1290" href="#figs1">Figure&nbsp;S1</a>).</div></section><section id="sec-4-6-2-2"><h5>Rare inherited variation</h5><div id="p0280" role="paragraph">As with <i>de novo</i> variation, we used the dataset of 26,367 samples and 4,560,185 unique variants described above as a starting point to identify high confidence rare inherited variants. Any genotype call with a GQ&nbsp;&lt; 25 was removed and heterozygous genotypes were required to have an allele balance ≥ 0.3. Variants were required to have a call rate ≥ 90%, insertions and deletions were required to pass VQSR, and SNVs were required to have a VQSLOD (variant quality score log odds) ≥ −2.085. The VQSLOD threshold for SNVs was determined by identifying the threshold at which synonymous variants with an allele count of 1 among parents in the dataset were transmitted to the child 50% of the time, as described previously (<span class="dropBlock reference-citations"><a id="body-ref-sref51-4" href="#" role="doc-biblioref" data-xml-rid="bib51 bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-4" aria-controls="sref51-4">Lek et&nbsp;al., 2016; Kosmicki et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref51-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Protein-truncating variants were required to be high confidence (“HC”) by the LOFTEE plugin for VEP and to have no LOFTEE flags other than “SINGLE_EXON.”</div><div id="p0285" role="paragraph">For the purposes of gene-level counts, variants were tallied in the 5,869 probands and 1,993 unaffected children with exome sequencing data available who passed quality control for <i>de novo</i> variation above. Variants were required to have an allele count ≤ 5 in the combined parents, cases, and controls (18,153 people) in our dataset, as well as an allele count ≤ 5 in the non-psychiatric subset of ExAC (see <a id="crosref1295" href="#figs1">Figure&nbsp;S1</a>).</div></section><section id="sec-4-6-2-3"><h5>Rare case-control variation</h5><div id="p0290" role="paragraph">For samples that were not in complete trios, rare variants were filtered using the same metrics and thresholds as rare inherited variants (above). For purposes of gene-level counts, rare variants were defined using the same allele frequency thresholds as rare inherited variants: allele count ≤ 5 in the 18,153 combined parents, cases, and controls in the dataset, as well as an allele count ≤ 5 in the non-psychiatric subset of ExAC (see <a id="crosref1300" href="#figs1">Figure&nbsp;S1</a>).</div><div id="p0295" role="paragraph">To ensure well-matched cases and controls, probable ancestry was calculated by merging our raw dataset with genotypes from the 1000 Genomes Project and conducting principal components analysis (PCA) in Hail on a set of ∼5,000 common SNPs. A naive Bayes classifier was trained (using the naiveBayes function from the R package e1071) on the 1000 Genomes samples and used to predict which of our samples clustered with the populations labeled as European or East Asian. Rates of synonymous variants were well-matched between cases and controls from the Swedish contributing site, which were classified European (745 cases and 3,595 controls), as well as between cases and controls from the Japanese contributing site which were classified East Asian (196 cases and 298 controls). For inclusion in TADA, we counted variants from the 4,340 Swedish samples. Overall variant rates were higher in the Japanese samples than the Swedish samples, possibly because our filtering was based on allele counts in ExAC, and ExAC has less representation from East Asian samples than European ones.</div></section></section><section id="sec-4-6-3"><h4>Analysis of variant classes</h4><div id="p0300" role="paragraph">To model a qualitative trait—in this case, the presence or absence of ASD—using standard quantitative genetics concepts, we imagine that there is an unobserved, normally distributed variable called “liability” that determines whether or not an individual is diagnosed with ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref28" href="#" role="doc-biblioref" data-xml-rid="bib28" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref28" aria-controls="sref28">Falconer, 1965</a><div class="dropBlock__holder" data-db-target-of="sref28"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">26.</div><div>Falconer, D.S.</div><div><strong>The inheritance of liability to certain diseases, estimated from the incidence among relatives</strong></div><div><em>Ann. Hum. Genet.</em> 1965; <strong>29</strong>:51-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1469-1809.1965.tb00500.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1469-1809.1965.tb00500.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We assume that liability, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L" tabindex="0" ctxtmenu_counter="0" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper L"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, has mean 0 and variance 1 in the general population. Individuals with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L" tabindex="0" ctxtmenu_counter="1" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper L"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> greater than some threshold <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t" tabindex="0" ctxtmenu_counter="2" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="t"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> are diagnosed with ASD and individuals with <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper L less than t" tabindex="0" ctxtmenu_counter="3" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper L less than t"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D43F">𝐿</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> are considered “typical.” Under this model, the prevalence difference between males and females is viewed as a difference in thresholds for males and females. For a male to be diagnosed with ASD, his liability must be larger than <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t Subscript m" tabindex="0" ctxtmenu_counter="4" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="t Subscript m"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. For a female to be diagnosed with ASD her liability must be larger than <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t Subscript f" tabindex="0" ctxtmenu_counter="5" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="t Subscript f"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. Since ASD is more common in males than females, we conclude that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t Subscript m Baseline less than t Subscript f" tabindex="0" ctxtmenu_counter="6" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (2 0 1) 3 (6 4 5))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="7" data-semantic-children="2,6" data-semantic-content="3" data-semantic-owns="2 3 6" data-semantic-speech="t Subscript m Baseline less than t Subscript f"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="7" data-semantic-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="3" data-semantic-parent="7" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="7" data-semantic-owns="4 5"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. For all that follows we will assume that the prevalence of ASD in males, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper Psi Subscript m" tabindex="0" ctxtmenu_counter="7" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper Psi Subscript m"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c3A8">Ψ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, is 1 in 42 (implying <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t Subscript m Baseline tilde" tabindex="0" ctxtmenu_counter="8" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 (2 0 1) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-owns="2 3 4" data-semantic-speech="t Subscript m Baseline tilde"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="5" data-semantic-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 1.98), and the prevalence of ASD in females, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="normal upper Psi Subscript f" tabindex="0" ctxtmenu_counter="9" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="normal upper Psi Subscript f"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c3A8">Ψ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, is 1 in 189 (implying <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="t Subscript f Baseline tilde" tabindex="0" ctxtmenu_counter="10" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 (2 0 1) 3 4)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-owns="2 3 4" data-semantic-speech="t Subscript f Baseline tilde"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="5" data-semantic-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="4" data-semantic-parent="5"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 2.56) (<span class="dropBlock reference-citations"><a id="body-ref-sref2" href="#" role="doc-biblioref" data-xml-rid="bib2" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref2" aria-controls="sref2">Baio et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">2.</div><div>Baio, J. ∙ Wiggins, L. ∙ Christensen, D.L. ...</div><div><strong>Prevalence of Autism Spectrum Disorder Among Children Aged 8 Years - Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2014</strong></div><div><em>MMWR Surveill Summ.</em> 2018; <strong>67</strong>:1-23</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15585/mmwr.ss6706a1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85047968753" target="_blank" aria-label="Open Scopus in new tab">Scopus (813)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29701730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15585%2Fmmwr.ss6706a1&amp;pmid=29701730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We model ASD+ID similarly, but with lower prevalence than all ASD (male prevalence 0.00499; female prevalence 0.00138).</div><div id="p0305" role="paragraph">When considering the effects of individual alleles on liability, we employ an elaboration to the standard quantitative genetics model, which is sometimes called the “mixed model of inheritance” (<span class="dropBlock reference-citations"><a id="body-ref-sref66" href="#" role="doc-biblioref" data-xml-rid="bib66" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref66" aria-controls="sref66">Morton &amp; MacLean, 1974</a><div class="dropBlock__holder" data-db-target-of="sref66"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">64.</div><div>Morton, N.E. ∙ MacLean, C.J.</div><div><strong>Analysis of family resemblance. 3. Complex segregation of quantitative traits</strong></div><div><em>Am. J. Hum. Genet.</em> 1974; <strong>26</strong>:489-503</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4842773/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=4842773" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We assume that individual alleles make additive contributions to liability, so that for some allele, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A 1" tabindex="0" ctxtmenu_counter="11" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper A 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, individuals with 0 copies of the allele have mean <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="negative mu" tabindex="0" ctxtmenu_counter="12" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-children="1" data-semantic-content="0" data-semantic-owns="0 1" data-semantic-speech="negative mu"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="0" data-semantic-parent="2" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, variance 1 liability, but individuals with 1 copy have mean <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha minus mu" tabindex="0" ctxtmenu_counter="13" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="alpha minus mu"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, variance 1, and individuals with 2 copies have mean <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 alpha minus mu" tabindex="0" ctxtmenu_counter="14" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 0 4 1) 2 3)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-children="5,3" data-semantic-content="2" data-semantic-owns="5 2 3" data-semantic-speech="2 alpha minus mu"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="0,1" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="0 4 1"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="2" data-semantic-parent="6" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, variance 1 liability. Assuming Hardy-Weinberg equilibrium for genotypes, and the frequency of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper A 1" tabindex="0" ctxtmenu_counter="15" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper A 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> equaling <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="p" tabindex="0" ctxtmenu_counter="16" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="p"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu equals 2 alpha p squared plus alpha Baseline 2 p q equals 2 alpha p" tabindex="0" ctxtmenu_counter="17" style="font-size: 130%;"><mjx-math data-semantic-structure="(27 0 1 (23 (18 2 16 3 17 (6 4 5)) 7 (22 8 19 9 20 10 21 11)) 12 (26 13 24 14 25 15))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="27" data-semantic-children="0,23,26" data-semantic-content="1,12" data-semantic-owns="0 1 23 12 26" data-semantic-speech="mu equals 2 alpha p squared plus alpha Baseline 2 p q equals 2 alpha p"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="27"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="27" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="23" data-semantic-children="18,22" data-semantic-content="7" data-semantic-parent="27" data-semantic-owns="18 7 22"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="2,3,6" data-semantic-content="16,17" data-semantic-parent="23" data-semantic-owns="2 16 3 17 6"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="18"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="18"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="4,5" data-semantic-parent="18" data-semantic-owns="4 5"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="7" data-semantic-parent="23" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="8,9,10,11" data-semantic-content="19,20,21" data-semantic-parent="23" data-semantic-owns="8 19 9 20 10 21 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="22"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="22"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="22"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="22"><mjx-c class="mjx-c1D45E">𝑞</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="12" data-semantic-parent="27" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="26" data-semantic-children="13,14,15" data-semantic-content="24,25" data-semantic-parent="27" data-semantic-owns="13 24 14 25 15"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="26"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="26"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="26"><mjx-c class="mjx-c1D45D">𝑝</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. Here <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu" tabindex="0" ctxtmenu_counter="18" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="mu"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is a normalizing factor to ensure the overall population has mean liability 0.</div><div id="p0310" role="paragraph">For several of our analyses we are interested in the effect, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha" tabindex="0" ctxtmenu_counter="19" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="alpha"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, for variants of a particular type in a collection of genes, for instance <i>de novo</i> PTVs in genes with pLI scores ≥ 0.995. If a variant is individually exceptionally rare, we have virtually no power to estimate its individual effect size, but over a large collection of such variants average properties are estimable. To do so, we model the entire collection of variants as if there were a single allele with frequency equal to the sum of the individual variant frequencies. This approach makes little sense for common variants, but for sufficiently rare variants, where single individuals seldom harbor more than one, this is a reasonable and helpful approximation. For some variant types, however, such as silent variants, the count of alleles can be substantial. For this reason, rather than standardize by <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 upper N" tabindex="0" ctxtmenu_counter="20" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 2 1)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="3" data-semantic-children="0,1" data-semantic-content="2" data-semantic-owns="0 2 1" data-semantic-speech="2 upper N"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="2" data-semantic-parent="3" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="21" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the number of subjects, we standardize by <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 upper N upper M" tabindex="0" ctxtmenu_counter="22" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 3 1 4 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="0,1,2" data-semantic-content="3,4" data-semantic-owns="0 3 1 4 2" data-semantic-speech="2 upper N upper M"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M" tabindex="0" ctxtmenu_counter="23" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper M"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>&nbsp;= 17,484 is the number of autosomal protein-coding genes analyzed herein. This standardization has no material impact on calculations of parameters of interest. To distinguish between cases and controls, we write <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N Subscript c a" tabindex="0" ctxtmenu_counter="24" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper N Subscript c a"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N Subscript c o" tabindex="0" ctxtmenu_counter="25" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="upper N Subscript c o"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> respectively.</div><div id="p0315" role="paragraph">Thus, for each type of variant we are interested in studying, e.g., <i>de novo</i> PTV mutations, we count the number of observations of this class of variant in cases (our probands in trios), and the number of observations of this class of variant in controls (our siblings in trios). For a given type of variant, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper V" tabindex="0" ctxtmenu_counter="26" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper V"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, we call <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V vertical bar upper D right brace" tabindex="0" ctxtmenu_counter="27" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 13 (12 1 11 (10 2 (8 (5 3 4) 7 6) 9)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="14" data-semantic-children="0,12" data-semantic-content="13" data-semantic-owns="0 13 12" data-semantic-speech="upper P r left brace upper V vertical bar upper D right brace"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="12" data-semantic-children="1,10" data-semantic-content="11,1" data-semantic-parent="14" data-semantic-owns="1 11 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="12" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="2,9" data-semantic-parent="12" data-semantic-owns="2 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="5,6" data-semantic-content="7" data-semantic-parent="10" data-semantic-owns="5 7 6"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="5" data-semantic-children="3,4" data-semantic-content="4" data-semantic-parent="8" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> the frequency of this type of variant in cases (observed number of variants divided by <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 upper N upper M" tabindex="0" ctxtmenu_counter="28" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 3 1 4 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="0,1,2" data-semantic-content="3,4" data-semantic-owns="0 3 1 4 2" data-semantic-speech="2 upper N upper M"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>), and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V vertical bar not sign upper D right brace" tabindex="0" ctxtmenu_counter="29" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 0 11 1 12 (10 2 (8 (5 3 4) 6 7) 9))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="13" data-semantic-children="0,1,10" data-semantic-content="11,12" data-semantic-owns="0 11 1 12 10" data-semantic-speech="upper P r left brace upper V vertical bar not sign upper D right brace"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="13"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="13"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="2,9" data-semantic-parent="13" data-semantic-owns="2 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="prefix operator" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="5 6 7"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="5" data-semantic-children="3,4" data-semantic-content="4" data-semantic-parent="8" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="4" data-semantic-parent="5" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="prefix operator" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="infixop,¬"><mjx-c class="mjx-cAC">¬</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> the corresponding value in controls. We make these calculations separately in males and females, which we denote as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V Subscript m Baseline vertical bar upper D Subscript m Baseline right brace" tabindex="0" ctxtmenu_counter="30" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 17 (16 1 15 (14 2 (12 (7 (5 3 4) 6) 11 (10 8 9)) 13)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="0,16" data-semantic-content="17" data-semantic-owns="0 17 16" data-semantic-speech="upper P r left brace upper V Subscript m Baseline vertical bar upper D Subscript m Baseline right brace"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="18"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="16" data-semantic-children="1,14" data-semantic-content="15,1" data-semantic-parent="18" data-semantic-owns="1 15 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="16" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="2,13" data-semantic-parent="16" data-semantic-owns="2 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="7,10" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="7 11 10"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="7" data-semantic-children="5,6" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="12" data-semantic-owns="8 9"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V Subscript f Baseline vertical bar upper D Subscript f Baseline right brace" tabindex="0" ctxtmenu_counter="31" style="font-size: 130%;"><mjx-math data-semantic-structure="(18 0 17 (16 1 15 (14 2 (12 (7 (5 3 4) 6) 11 (10 8 9)) 13)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="0,16" data-semantic-content="17" data-semantic-owns="0 17 16" data-semantic-speech="upper P r left brace upper V Subscript f Baseline vertical bar upper D Subscript f Baseline right brace"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="18"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="16" data-semantic-children="1,14" data-semantic-content="15,1" data-semantic-parent="18" data-semantic-owns="1 15 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="16" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="2,13" data-semantic-parent="16" data-semantic-owns="2 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="7,10" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="7 11 10"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="7" data-semantic-children="5,6" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="12" data-semantic-owns="8 9"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V Subscript m Baseline vertical bar not sign upper D Subscript m Baseline right brace" tabindex="0" ctxtmenu_counter="32" style="font-size: 130%;"><mjx-math data-semantic-structure="(17 0 15 1 16 (14 2 (12 (7 (5 3 4) 6) 8 (11 9 10)) 13))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-children="0,1,14" data-semantic-content="15,16" data-semantic-owns="0 15 1 16 14" data-semantic-speech="upper P r left brace upper V Subscript m Baseline vertical bar not sign upper D Subscript m Baseline right brace"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="17"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="17"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="2,13" data-semantic-parent="17" data-semantic-owns="2 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="prefix operator" data-semantic-id="12" data-semantic-children="7,11" data-semantic-content="8" data-semantic-parent="14" data-semantic-owns="7 8 11"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="7" data-semantic-children="5,6" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="prefix operator" data-semantic-id="8" data-semantic-parent="12" data-semantic-operator="infixop,¬"><mjx-c class="mjx-cAC">¬</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="12" data-semantic-owns="9 10"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V Subscript f Baseline vertical bar not sign upper D Subscript f Baseline right brace" tabindex="0" ctxtmenu_counter="33" style="font-size: 130%;"><mjx-math data-semantic-structure="(17 0 15 1 16 (14 2 (12 (7 (5 3 4) 6) 8 (11 9 10)) 13))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="17" data-semantic-children="0,1,14" data-semantic-content="15,16" data-semantic-owns="0 15 1 16 14" data-semantic-speech="upper P r left brace upper V Subscript f Baseline vertical bar not sign upper D Subscript f Baseline right brace"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="17"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="17"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="17" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="2,13" data-semantic-parent="17" data-semantic-owns="2 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="prefix operator" data-semantic-id="12" data-semantic-children="7,11" data-semantic-content="8" data-semantic-parent="14" data-semantic-owns="7 8 11"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="7" data-semantic-children="5,6" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="prefix operator" data-semantic-id="8" data-semantic-parent="12" data-semantic-operator="infixop,¬"><mjx-c class="mjx-cAC">¬</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="11" data-semantic-children="9,10" data-semantic-parent="12" data-semantic-owns="9 10"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="11"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="11"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m" tabindex="0" ctxtmenu_counter="34" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="m"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f" tabindex="0" ctxtmenu_counter="35" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="f"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> subscripts distinguish male and females. The overall frequency of the variant class can be found by:<div id="ufd1" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V Subscript g Baseline right brace equals upper P r left brace upper V Subscript g Baseline vertical bar upper D Subscript g Baseline right brace normal upper Psi Subscript g Baseline plus upper P r left brace upper V Subscript g Baseline vertical bar not sign upper D Subscript g Baseline right brace left parenthesis 1 minus normal upper Psi Subscript g Baseline right parenthesis" tabindex="0" ctxtmenu_counter="36" style="font-size: 130%;"><mjx-math data-semantic-structure="(67 (63 0 62 (61 1 60 (7 2 (5 3 4) 6))) 8 (66 9 64 (59 10 58 (48 11 (47 (44 (21 (19 (14 12 13) 15 (18 16 17)) 20) 43 (24 22 23)) 25 (46 26 45 (42 27 41 (39 28 (38 (33 (31 29 30) 32) 34 (37 35 36)))))) 40)) 65 (57 49 (55 50 51 (54 52 53)) 56)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="67" data-semantic-children="63,66" data-semantic-content="8" data-semantic-owns="63 8 66" data-semantic-speech="upper P r left brace upper V Subscript g Baseline right brace equals upper P r left brace upper V Subscript g Baseline vertical bar upper D Subscript g Baseline right brace normal upper Psi Subscript g Baseline plus upper P r left brace upper V Subscript g Baseline vertical bar not sign upper D Subscript g Baseline right brace left parenthesis 1 minus normal upper Psi Subscript g Baseline right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="63" data-semantic-children="0,61" data-semantic-content="62" data-semantic-parent="67" data-semantic-owns="0 62 61"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="63"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="62" data-semantic-parent="63" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="61" data-semantic-children="1,7" data-semantic-content="60,1" data-semantic-parent="63" data-semantic-owns="1 60 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="61" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="60" data-semantic-parent="61" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="7" data-semantic-children="5" data-semantic-content="2,6" data-semantic-parent="61" data-semantic-owns="2 5 6"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="67" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="66" data-semantic-children="9,59,57" data-semantic-content="64,65" data-semantic-parent="67" data-semantic-owns="9 64 59 65 57"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="66"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="64" data-semantic-parent="66" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="59" data-semantic-children="10,48" data-semantic-content="58,10" data-semantic-parent="66" data-semantic-owns="10 58 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="59" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="58" data-semantic-parent="59" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="48" data-semantic-children="47" data-semantic-content="11,40" data-semantic-parent="59" data-semantic-owns="11 47 40"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="48" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="47" data-semantic-children="44,46" data-semantic-content="25" data-semantic-parent="48" data-semantic-owns="44 25 46"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="44" data-semantic-children="21,24" data-semantic-content="43" data-semantic-parent="47" data-semantic-owns="21 43 24"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="21" data-semantic-children="19,20" data-semantic-content="20" data-semantic-parent="44" data-semantic-owns="19 20"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="19" data-semantic-children="14,15,18" data-semantic-content="15" data-semantic-parent="21" data-semantic-owns="14 15 18"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="12,13" data-semantic-parent="19" data-semantic-owns="12 13"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="15" data-semantic-parent="19" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="18" data-semantic-children="16,17" data-semantic-parent="19" data-semantic-owns="16 17"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="18"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="closefence" data-semantic-id="20" data-semantic-parent="21" data-semantic-operator="punctuated"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="43" data-semantic-parent="44" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="24" data-semantic-children="22,23" data-semantic-parent="44" data-semantic-owns="22 23"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24"><mjx-c class="mjx-c3A8">Ψ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="25" data-semantic-parent="47" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="46" data-semantic-children="26,42" data-semantic-content="45" data-semantic-parent="47" data-semantic-owns="26 45 42"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="46"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="46" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="42" data-semantic-children="27,39" data-semantic-content="41,27" data-semantic-parent="46" data-semantic-owns="27 41 39"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="42" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="39" data-semantic-children="28,38" data-semantic-content="28" data-semantic-parent="42" data-semantic-owns="28 38"><mjx-mo data-semantic-type="punctuation" data-semantic-role="openfence" data-semantic-id="28" data-semantic-parent="39" data-semantic-operator="punctuated"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="prefix operator" data-semantic-id="38" data-semantic-children="33,37" data-semantic-content="34" data-semantic-parent="39" data-semantic-owns="33 34 37"><mjx-mrow><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="33" data-semantic-children="31,32" data-semantic-content="32" data-semantic-parent="38" data-semantic-owns="31 32"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="31" data-semantic-children="29,30" data-semantic-parent="33" data-semantic-owns="29 30"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="31"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="32" data-semantic-parent="33" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="prefix operator" data-semantic-id="34" data-semantic-parent="38" data-semantic-operator="infixop,¬"><mjx-c class="mjx-cAC">¬</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="37" data-semantic-children="35,36" data-semantic-parent="38" data-semantic-owns="35 36"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="35" data-semantic-parent="37"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="37"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="40" data-semantic-parent="48" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="65" data-semantic-parent="66" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="57" data-semantic-children="55" data-semantic-content="49,56" data-semantic-parent="66" data-semantic-owns="49 55 56"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="49" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="55" data-semantic-children="50,54" data-semantic-content="51" data-semantic-parent="57" data-semantic-owns="50 51 54"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="50" data-semantic-parent="55"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="51" data-semantic-parent="55" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="54" data-semantic-children="52,53" data-semantic-parent="55" data-semantic-owns="52 53"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="52" data-semantic-parent="54"><mjx-c class="mjx-c3A8">Ψ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="53" data-semantic-parent="54"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="56" data-semantic-parent="57" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g" tabindex="0" ctxtmenu_counter="37" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="g"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> can be either <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="f" tabindex="0" ctxtmenu_counter="38" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="f"><mjx-c class="mjx-c1D453">𝑓</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> or <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="m" tabindex="0" ctxtmenu_counter="39" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="m"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, for females and males, respectively. From this the Penetrance (probability of disorder given variant) of the variant class can be found immediately by Bayes rule:<div id="ufd2" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper D Subscript g Baseline vertical bar upper V Subscript g Baseline right brace equals StartFraction upper P r left brace upper V Subscript g Baseline vertical bar upper D Subscript g Baseline right brace upper Psi Subscript g Baseline Over upper P r left brace upper V Subscript g Baseline right brace EndFraction" tabindex="0" ctxtmenu_counter="40" style="font-size: 130%;"><mjx-math data-semantic-structure="(56 (55 0 54 (53 1 52 (14 2 (12 (7 (5 3 4) 6) 11 (10 8 9)) 13))) 15 (51 (38 16 36 (35 17 34 (30 18 (28 (23 (21 19 20) 22) 27 (26 24 25)) 29)) 37 (33 31 32)) (50 39 49 (48 40 47 (46 41 (44 42 43) 45)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="56" data-semantic-children="55,51" data-semantic-content="15" data-semantic-owns="55 15 51" data-semantic-speech="upper P r left brace upper D Subscript g Baseline vertical bar upper V Subscript g Baseline right brace equals StartFraction upper P r left brace upper V Subscript g Baseline vertical bar upper D Subscript g Baseline right brace upper Psi Subscript g Baseline Over upper P r left brace upper V Subscript g Baseline right brace EndFraction"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="55" data-semantic-children="0,53" data-semantic-content="54" data-semantic-parent="56" data-semantic-owns="0 54 53"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="55"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="54" data-semantic-parent="55" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="53" data-semantic-children="1,14" data-semantic-content="52,1" data-semantic-parent="55" data-semantic-owns="1 52 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="53" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="52" data-semantic-parent="53" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="2,13" data-semantic-parent="53" data-semantic-owns="2 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="7,10" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="7 11 10"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="7" data-semantic-children="5,6" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="12" data-semantic-owns="8 9"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="15" data-semantic-parent="56" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mfrac space="4" data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="51" data-semantic-children="38,50" data-semantic-parent="56" data-semantic-owns="38 50"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="38" data-semantic-children="16,35,33" data-semantic-content="36,37" data-semantic-parent="51" data-semantic-owns="16 36 35 37 33"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="38"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="36" data-semantic-parent="38" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="35" data-semantic-children="17,30" data-semantic-content="34,17" data-semantic-parent="38" data-semantic-owns="17 34 30"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="35" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="34" data-semantic-parent="35" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="18,29" data-semantic-parent="35" data-semantic-owns="18 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="18" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="28" data-semantic-children="23,26" data-semantic-content="27" data-semantic-parent="30" data-semantic-owns="23 27 26"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="23" data-semantic-children="21,22" data-semantic-content="22" data-semantic-parent="28" data-semantic-owns="21 22"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="21" data-semantic-children="19,20" data-semantic-parent="23" data-semantic-owns="19 20"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="24,25" data-semantic-parent="28" data-semantic-owns="24 25"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="26"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="26"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="37" data-semantic-parent="38" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="unknown" data-semantic-id="33" data-semantic-children="31,32" data-semantic-parent="38" data-semantic-owns="31 32"><mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="31" data-semantic-parent="33"><mjx-c class="mjx-c3A8">Ψ</mjx-c></mjx-mtext></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="32" data-semantic-parent="33"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="50" data-semantic-children="39,48" data-semantic-content="49" data-semantic-parent="51" data-semantic-owns="39 49 48"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="50"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="49" data-semantic-parent="50" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="48" data-semantic-children="40,46" data-semantic-content="47,40" data-semantic-parent="50" data-semantic-owns="40 47 46"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="40" data-semantic-parent="48" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="47" data-semantic-parent="48" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="46" data-semantic-children="44" data-semantic-content="41,45" data-semantic-parent="48" data-semantic-owns="41 44 45"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="41" data-semantic-parent="46" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="44" data-semantic-children="42,43" data-semantic-parent="46" data-semantic-owns="42 43"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="42" data-semantic-parent="44"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="44"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="45" data-semantic-parent="46" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac></mjx-mrow></mjx-math></mjx-container></div></div></div>To find the average effect, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g" tabindex="0" ctxtmenu_counter="41" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="alpha Subscript upper V Sub Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, of this variant class we note:<div id="ufd3" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper D Subscript g Baseline vertical bar upper V Subscript g Baseline right brace equals integral Subscript t Subscript g Baseline minus alpha Subscript upper V Sub Subscript g Baseline Superscript infinity Baseline StartFraction 1 Over StartRoot 2 pi EndRoot EndFraction e Superscript minus StartFraction x squared Over 2 EndFraction d x Baseline" tabindex="0" ctxtmenu_counter="42" style="font-size: 130%;"><mjx-math data-semantic-structure="(60 (59 0 58 (57 1 56 (14 2 (12 (7 (5 3 4) 6) 11 (10 8 9)) 13))) 15 (55 (28 16 (26 (19 17 18) 20 (25 21 (24 22 23))) 27) (53 (35 29 (34 (33 30 32 31))) 52 (49 36 (48 37 (47 (42 (40 38 39) 41) 45 43 46 44)))) 54))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="60" data-semantic-children="59,55" data-semantic-content="15" data-semantic-owns="59 15 55" data-semantic-speech="upper P r left brace upper D Subscript g Baseline vertical bar upper V Subscript g Baseline right brace equals integral Subscript t Subscript g Baseline minus alpha Subscript upper V Sub Subscript g Baseline Superscript infinity Baseline StartFraction 1 Over StartRoot 2 pi EndRoot EndFraction e Superscript minus StartFraction x squared Over 2 EndFraction d x Baseline"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="59" data-semantic-children="0,57" data-semantic-content="58" data-semantic-parent="60" data-semantic-owns="0 58 57"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="59"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="58" data-semantic-parent="59" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="57" data-semantic-children="1,14" data-semantic-content="56,1" data-semantic-parent="59" data-semantic-owns="1 56 14"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="57" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="56" data-semantic-parent="57" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="14" data-semantic-children="12" data-semantic-content="2,13" data-semantic-parent="57" data-semantic-owns="2 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="7,10" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="7 11 10"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="7" data-semantic-children="5,6" data-semantic-content="6" data-semantic-parent="12" data-semantic-owns="5 6"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="7" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="10" data-semantic-children="8,9" data-semantic-parent="12" data-semantic-owns="8 9"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="15" data-semantic-parent="60" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="integral" data-semantic-role="integral" data-semantic-id="55" data-semantic-children="28,53,54" data-semantic-content="16" data-semantic-parent="60" data-semantic-owns="28 53 54"><mjx-munderover data-semantic-type="limboth" data-semantic-role="integral" data-semantic-id="28" data-semantic-children="16,26,27" data-semantic-parent="55" data-semantic-owns="16 26 27"><mjx-over style="padding-bottom: 0.182em; padding-left: 0.94em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="infty" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c221E MM-I">∞</mjx-c></mjx-mi></mjx-mrow></mjx-over><mjx-box><mjx-munder><mjx-row><mjx-base style="padding-left: 0.85em;"><mjx-mo data-semantic-type="largeop" data-semantic-role="integral" data-semantic-id="16" data-semantic-parent="28" data-semantic-operator="integral"><mjx-c class="mjx-c222B">∫</mjx-c></mjx-mo></mjx-base></mjx-row><mjx-row><mjx-under style="padding-top: 0.167em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="26" data-semantic-children="19,25" data-semantic-content="20" data-semantic-parent="28" data-semantic-owns="19 20 25"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="19" data-semantic-children="17,18" data-semantic-parent="26" data-semantic-owns="17 18"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="20" data-semantic-parent="26" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="25" data-semantic-children="21,24" data-semantic-parent="26" data-semantic-owns="21 24"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="25"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="24" data-semantic-children="22,23" data-semantic-parent="25" data-semantic-owns="22 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="24"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="24"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-under></mjx-row></mjx-munder></mjx-box></mjx-munderover><mjx-mrow space="2" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="53" data-semantic-children="35,49" data-semantic-content="52" data-semantic-parent="55" data-semantic-owns="35 52 49"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="35" data-semantic-children="29,34" data-semantic-parent="53" data-semantic-owns="29 34"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="35"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-msqrt data-semantic-type="sqrt" data-semantic-role="unknown" data-semantic-id="34" data-semantic-children="33" data-semantic-parent="35" data-semantic-owns="33"><mjx-sqrt><mjx-surd><mjx-mo><mjx-c class="mjx-c221A">√</mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.169em;"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="33" data-semantic-children="30,31" data-semantic-content="32" data-semantic-parent="34" data-semantic-owns="30 32 31"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="30" data-semantic-parent="33"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="31" data-semantic-parent="33"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi></mjx-mrow></mjx-box></mjx-sqrt></mjx-msqrt></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="52" data-semantic-parent="53" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="49" data-semantic-children="36,48" data-semantic-parent="53" data-semantic-owns="36 48"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="36" data-semantic-parent="49"><mjx-c class="mjx-c1D452">𝑒</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-id="48" data-semantic-children="47" data-semantic-content="37" data-semantic-parent="49" data-semantic-owns="37 47"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="37" data-semantic-parent="48" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="47" data-semantic-children="42,43,44" data-semantic-content="45,46" data-semantic-parent="48" data-semantic-owns="42 45 43 46 44"><mjx-mfrac data-semantic-type="fraction" data-semantic-role="division" data-semantic-id="42" data-semantic-children="40,41" data-semantic-parent="47" data-semantic-owns="40 41"><mjx-frac><mjx-num><mjx-nstrut></mjx-nstrut><mjx-mrow size="s"><mjx-msup data-semantic-type="superscript" data-semantic-role="latinletter" data-semantic-id="40" data-semantic-children="38,39" data-semantic-parent="42" data-semantic-owns="38 39"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="38" data-semantic-parent="40"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.363em;"><mjx-mrow><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="39" data-semantic-parent="40"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup></mjx-mrow></mjx-num><mjx-dbox><mjx-dtable><mjx-line></mjx-line><mjx-row><mjx-den><mjx-dstrut></mjx-dstrut><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="41" data-semantic-parent="42"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn></mjx-mrow></mjx-den></mjx-row></mjx-dtable></mjx-dbox></mjx-frac></mjx-mfrac><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="45" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="43" data-semantic-parent="47"><mjx-c class="mjx-c1D451">𝑑</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="46" data-semantic-parent="47" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="44" data-semantic-parent="47"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-script></mjx-msup></mjx-mrow><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="54" data-semantic-parent="55"></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container></div></div></div>This is the area under the curve of a normal distribution with mean <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g" tabindex="0" ctxtmenu_counter="43" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="alpha Subscript upper V Sub Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and variance 1, from the threshold <i>t</i><sub><i>g</i></sub> to infinity. In other words, the penetrance of a genotype is the fraction of the genotype’s liability distribution that exceeds the disorder threshold. We can find the effect size (average liability of the genotype) by inverting a standard normal cumulative distribution, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper Phi left parenthesis x right parenthesis" tabindex="0" ctxtmenu_counter="44" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 5 (4 1 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-children="0,4" data-semantic-content="5,0" data-semantic-owns="0 5 4" data-semantic-speech="upper Phi left parenthesis x right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6" data-semantic-operator="appl"><mjx-c class="mjx-c1D6F7">𝛷</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="4" data-semantic-children="2" data-semantic-content="1,3" data-semantic-parent="6" data-semantic-owns="1 2 3"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>:<div id="ufd4" class="display-formula"><div class="equation" role="math"><div class="inner"><mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g Subscript Baseline equals t Subscript g Baseline minus normal upper Phi Superscript negative 1 Baseline left parenthesis 1 minus upper P r left brace upper D Subscript g Baseline vertical bar upper V Subscript g Baseline right brace right parenthesis period" tabindex="0" ctxtmenu_counter="45" style="font-size: 130%;"><mjx-math data-semantic-structure="(45 (44 (4 0 (3 1 2)) 5 (43 (8 6 7) 9 (42 (14 10 (13 11 12)) 41 (39 15 (37 16 17 (36 18 35 (34 19 33 (32 20 (30 (25 (23 21 22) 24) 29 (28 26 27)) 31)))) 38)))) 40)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="45" data-semantic-children="44,40" data-semantic-content="40" data-semantic-owns="44 40" data-semantic-speech="alpha Subscript upper V Sub Subscript g Subscript Baseline equals t Subscript g Baseline minus normal upper Phi Superscript negative 1 Baseline left parenthesis 1 minus upper P r left brace upper D Subscript g Baseline vertical bar upper V Subscript g Baseline right brace right parenthesis period"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="44" data-semantic-children="4,43" data-semantic-content="5" data-semantic-parent="45" data-semantic-owns="4 5 43"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-parent="44" data-semantic-owns="0 3"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="44" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="43" data-semantic-children="8,42" data-semantic-content="9" data-semantic-parent="44" data-semantic-owns="8 9 42"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="43" data-semantic-owns="6 7"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D461">𝑡</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="9" data-semantic-parent="43" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="42" data-semantic-children="14,39" data-semantic-content="41" data-semantic-parent="43" data-semantic-owns="14 41 39"><mjx-msup data-semantic-type="superscript" data-semantic-role="greekletter" data-semantic-id="14" data-semantic-children="10,13" data-semantic-parent="42" data-semantic-owns="10 13"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="14"><mjx-c class="mjx-c3A6">Φ</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: 0.363em;"><mjx-mrow size="s" data-semantic-type="prefixop" data-semantic-role="negative" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-children="12" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="11 12"><mjx-mo data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="prefixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="41" data-semantic-parent="42" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="39" data-semantic-children="37" data-semantic-content="15,38" data-semantic-parent="42" data-semantic-owns="15 37 38"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="39" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="37" data-semantic-children="16,36" data-semantic-content="17" data-semantic-parent="39" data-semantic-owns="16 17 36"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="37"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="17" data-semantic-parent="37" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="36" data-semantic-children="18,34" data-semantic-content="35" data-semantic-parent="37" data-semantic-owns="18 35 34"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="36"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="35" data-semantic-parent="36" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="34" data-semantic-children="19,32" data-semantic-content="33,19" data-semantic-parent="36" data-semantic-owns="19 33 32"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="34" data-semantic-operator="appl"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="32" data-semantic-children="30" data-semantic-content="20,31" data-semantic-parent="34" data-semantic-owns="20 30 31"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="20" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="30" data-semantic-children="25,28" data-semantic-content="29" data-semantic-parent="32" data-semantic-owns="25 29 28"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="25" data-semantic-children="23,24" data-semantic-content="24" data-semantic-parent="30" data-semantic-owns="23 24"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="23" data-semantic-children="21,22" data-semantic-parent="25" data-semantic-owns="21 22"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="23"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="23"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="24" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="29" data-semantic-parent="30" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="28" data-semantic-children="26,27" data-semantic-parent="30" data-semantic-owns="26 27"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="28"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="28"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="31" data-semantic-parent="32" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="38" data-semantic-parent="39" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="punctuation" data-semantic-role="fullstop" data-semantic-id="40" data-semantic-parent="45" data-semantic-operator="punctuated"><mjx-c class="mjx-c2E">.</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container></div></div></div>In this manner, we go from observable quantities (prevalence, genotype frequency, and frequency of genotype among cases only) to a variant type’s unobserved but inferred effect on the liability scale. Empirically, the relative risk for the variant type is calculated as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper P r left brace upper V vertical bar upper D right brace divided by upper P r left brace upper V vertical bar not sign upper D right brace" tabindex="0" ctxtmenu_counter="46" style="font-size: 130%;"><mjx-math data-semantic-structure="(29 (28 0 26 1 27 (9 2 (8 (6 3 4 5) 7))) 10 (25 (24 11 22 12 23 (20 13 (19 (16 14 15) 17 18))) 21))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="29" data-semantic-children="28,25" data-semantic-content="10" data-semantic-owns="28 10 25" data-semantic-speech="upper P r left brace upper V vertical bar upper D right brace divided by upper P r left brace upper V vertical bar not sign upper D right brace"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="28" data-semantic-children="0,1,9" data-semantic-content="26,27" data-semantic-parent="29" data-semantic-owns="0 26 1 27 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="28"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="28"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="9" data-semantic-children="2,8" data-semantic-content="2" data-semantic-parent="28" data-semantic-owns="2 8"><mjx-mo data-semantic-type="punctuation" data-semantic-role="openfence" data-semantic-id="2" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="8" data-semantic-children="6,7" data-semantic-content="7" data-semantic-parent="9" data-semantic-owns="6 7"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="6" data-semantic-children="3,4,5" data-semantic-content="4" data-semantic-parent="8" data-semantic-owns="3 4 5"><mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c56">V</mjx-c></mjx-mtext></mjx-mrow><mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="4" data-semantic-parent="6" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="6"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="closefence" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="punctuated"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="10" data-semantic-parent="29" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="25" data-semantic-children="24,21" data-semantic-content="21" data-semantic-parent="29" data-semantic-owns="24 21"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="24" data-semantic-children="11,12,20" data-semantic-content="22,23" data-semantic-parent="25" data-semantic-owns="11 22 12 23 20"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="24"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="22" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="24"><mjx-c class="mjx-c1D45F">𝑟</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="24" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="startpunct" data-semantic-id="20" data-semantic-children="13,19" data-semantic-content="13" data-semantic-parent="24" data-semantic-owns="13 19"><mjx-mo data-semantic-type="punctuation" data-semantic-role="openfence" data-semantic-id="13" data-semantic-parent="20" data-semantic-operator="punctuated"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="prefix operator" data-semantic-id="19" data-semantic-children="16,18" data-semantic-content="17" data-semantic-parent="20" data-semantic-owns="16 17 18"><mjx-mrow><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="16" data-semantic-children="14,15" data-semantic-content="15" data-semantic-parent="19" data-semantic-owns="14 15"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="15" data-semantic-parent="16" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="prefix operator" data-semantic-id="17" data-semantic-parent="19" data-semantic-operator="infixop,¬"><mjx-c class="mjx-cAC">¬</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="18" data-semantic-parent="19"><mjx-c class="mjx-c1D437">𝐷</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="closefence" data-semantic-id="21" data-semantic-parent="25" data-semantic-operator="punctuated"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> for the contrast of cases versus controls. To assess whether or not there is any difference in this variant class between cases and controls, we perform an exact binomial test on the underlying observed counts, where the probability of success is given by <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N Subscript c a Baseline divided by left parenthesis upper N Subscript c a Baseline plus upper N Subscript c o Baseline right parenthesis" tabindex="0" ctxtmenu_counter="47" style="font-size: 130%;"><mjx-math data-semantic-structure="(24 (5 0 (4 1 3 2)) 6 (23 7 (21 (13 8 (12 9 11 10)) 14 (20 15 (19 16 18 17))) 22))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="24" data-semantic-children="5,23" data-semantic-content="6" data-semantic-owns="5 6 23" data-semantic-speech="upper N Subscript c a Baseline divided by left parenthesis upper N Subscript c a Baseline plus upper N Subscript c o Baseline right parenthesis"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="24" data-semantic-owns="0 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="6" data-semantic-parent="24" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="23" data-semantic-children="21" data-semantic-content="7,22" data-semantic-parent="24" data-semantic-owns="7 21 22"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="7" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="21" data-semantic-children="13,20" data-semantic-content="14" data-semantic-parent="23" data-semantic-owns="13 14 20"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="13" data-semantic-children="8,12" data-semantic-parent="21" data-semantic-owns="8 12"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="13"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="12" data-semantic-children="9,10" data-semantic-content="11" data-semantic-parent="13" data-semantic-owns="9 11 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="12"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="12"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="14" data-semantic-parent="21" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-msub space="3" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="20" data-semantic-children="15,19" data-semantic-parent="21" data-semantic-owns="15 19"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="19" data-semantic-children="16,17" data-semantic-content="18" data-semantic-parent="20" data-semantic-owns="16 18 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="19"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="19" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="19"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. The odds ratio is computed from four observations, the number of variants of the risk class in cases, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="a" tabindex="0" ctxtmenu_counter="48" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="a"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>; the number of variants of the risk class in controls, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="49" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>; the number of alleles not in the risk class in cases <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 upper N Subscript c a Baseline upper M minus a" tabindex="0" ctxtmenu_counter="50" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 (12 0 10 (6 1 (5 2 4 3)) 11 7) 8 9)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-children="12,9" data-semantic-content="8" data-semantic-owns="12 8 9" data-semantic-speech="2 upper N Subscript c a Baseline upper M minus a"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="0,6,7" data-semantic-content="10,11" data-semantic-parent="13" data-semantic-owns="0 10 6 11 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="12"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="1,5" data-semantic-parent="12" data-semantic-owns="1 5"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-parent="13" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="13"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>; and the parallel calculation for controls, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 upper N Subscript c o Baseline upper M minus b" tabindex="0" ctxtmenu_counter="51" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 (12 0 10 (6 1 (5 2 4 3)) 11 7) 8 9)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="13" data-semantic-children="12,9" data-semantic-content="8" data-semantic-owns="12 8 9" data-semantic-speech="2 upper N Subscript c o Baseline upper M minus b"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="0,6,7" data-semantic-content="10,11" data-semantic-parent="13" data-semantic-owns="0 10 6 11 7"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="12"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="6" data-semantic-children="1,5" data-semantic-parent="12" data-semantic-owns="1 5"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.078em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D45C">𝑜</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="12"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-parent="13" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="13"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0320" role="paragraph">To estimate a confidence interval of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g" tabindex="0" ctxtmenu_counter="52" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="alpha Subscript upper V Sub Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, we note that in a very formal sense <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g" tabindex="0" ctxtmenu_counter="53" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="alpha Subscript upper V Sub Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is the average effect on the liability scale of the variant. Were we able to observe those effects directly, we could have calculated the observed mean and standard error of those effects. Because we cannot observe liability directly here, we infer the standard error of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g" tabindex="0" ctxtmenu_counter="54" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="alpha Subscript upper V Sub Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> by the following procedure: map the p value from the binomial test, described above, onto an equivalent z-value from the normal distribution, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="z" tabindex="0" ctxtmenu_counter="55" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="z"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>; then <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g Baseline divided by z" tabindex="0" ctxtmenu_counter="56" style="font-size: 130%;"><mjx-math data-semantic-structure="(7 (4 0 (3 1 2)) 5 6)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="7" data-semantic-children="4,6" data-semantic-content="5" data-semantic-owns="4 5 6" data-semantic-speech="alpha Subscript upper V Sub Subscript g Baseline divided by z"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-parent="7" data-semantic-owns="0 3"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D467">𝑧</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> is a reasonable estimator for the standard error of the estimator for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="alpha Subscript upper V Sub Subscript g" tabindex="0" ctxtmenu_counter="57" style="font-size: 130%;"><mjx-math data-semantic-structure="(4 0 (3 1 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,3" data-semantic-owns="0 3" data-semantic-speech="alpha Subscript upper V Sub Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FC">𝛼</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="3" data-semantic-children="1,2" data-semantic-parent="4" data-semantic-owns="1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.186em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0325" role="paragraph">For <a id="crosref1320" href="#mmc1">Table S1</a>, calculations for “All Genes” and for “Other Genes” were performed separately for males and females and also separately for the PAGES and DBS (iPSYCH) samples. Inherited analysis calculations were also separated by male and female and by proband and sibling. To combine effects between males and females, we took inverse-variance weighted averages of male and female effect sizes. We performed analogous calculations for the populations of case-control samples. For these calculations for the 102 ASD genes, however, because the counts of events were often small, we combined data over males and females and over PAGES and DBS samples to compute overall parameters (i.e., performed mega- versus meta-analysis). When parameters could not be estimated, this is noted as NA. Selected results are shown in <a id="crosref1325" href="#fig1">Figure&nbsp;1</a> and <a id="crosref1330" href="#fig2">Figure&nbsp;2</a>.</div></section><section id="sec-4-6-4"><h4>Transmission and De Novo Association Test (TADA)</h4><section id="sec-4-6-4-1"><h5>Background</h5><div id="p0330" role="paragraph">Published analyses of whole exome sequencing (WES) data using TADA have evaluated two categories of rare variation, namely protein-truncating variants (PTVs; i.e., frameshift, stop gained, canonical splice site disruption) and “probably damaging” missense variants according to PolyPhen-2 (Mis3) (<span class="dropBlock reference-citations"><a id="body-ref-sref1-1" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-1" aria-controls="sref1-1">Adzhubei et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref1-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Adzhubei, I.A. ∙ Schmidt, S. ∙ Peshkin, L. ...</div><div><strong>A method and server for predicting damaging missense mutations</strong></div><div><em>Nat. Methods.</em> 2010; <strong>7</strong>:248-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth0410-248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77951640946" target="_blank" aria-label="Open Scopus in new tab">Scopus (6943)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20354512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth0410-248&amp;pmid=20354512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), in the context of three categories of inheritance pattern: <i>de novo</i>, inherited, and case-control. TADA requires a mutational model (<span class="dropBlock reference-citations"><a id="body-ref-sref91" href="#" role="doc-biblioref" data-xml-rid="bib91 bib69 bib89" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref91" aria-controls="sref91">Sanders et&nbsp;al., 2012; Neale et&nbsp;al., 2012; Samocha et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref91"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">67.</div><div>Neale, B.M. ∙ Kou, Y. ∙ Liu, L. ...</div><div><strong>Patterns and rates of exonic de novo mutations in autism spectrum disorders</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:242-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84860712363" target="_blank" aria-label="Open Scopus in new tab">Scopus (1063)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11011&amp;pmid=22495311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">85.</div><div>Samocha, K.E. ∙ Robinson, E.B. ∙ Sanders, S.J. ...</div><div><strong>A framework for the interpretation of de novo mutation in human disease</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:944-950</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3050" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84905309176" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25086666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3050&amp;pmid=25086666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">87.</div><div>Sanders, S.J. ∙ Murtha, M.T. ∙ Gupta, A.R. ...</div><div><strong>De novo mutations revealed by whole-exome sequencing are strongly associated with autism</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:237-241</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10945" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84860780495" target="_blank" aria-label="Open Scopus in new tab">Scopus (1148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495306/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10945&amp;pmid=22495306" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) which accounts for gene size and sequence composition to obtain an expectation for mutations per gene, given sample size. It treats all PTVs within a gene as equivalent, although their impact on risk is allowed to vary across genes and inheritance patterns (likewise for Mis3). TADA first computes a gene-specific Bayes factor for each mutation category and inheritance pattern, and then it multiplies these Bayes factors to generate a statistic that summarizes all evidence of association for each gene. The total Bayes factor is finally converted to a q-value to control false discovery rate (<span class="dropBlock reference-citations"><a id="body-ref-sref21-10" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-10" aria-controls="sref21-10">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). As a Bayesian model, TADA requires prior parameters or hyperparameters, namely the fraction of genes in the genome affecting risk, thus far taken to be 0.05, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="58" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, the relative risk for a particular mutation category. See <span class="dropBlock reference-citations"><a id="body-ref-sref35-5" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-5" aria-controls="sref35-5">He et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref35-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> for estimators.</div></section><section id="sec-4-6-4-2"><h5>Evaluating TADA and False Discovery Rate (FDR)</h5><div id="p0335" role="paragraph">For downstream analysis it is critical to ensure reliable performance of TADA so that risk gene lists, such as those with FDR <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than" tabindex="0" ctxtmenu_counter="59" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="less than"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 10%, are properly calibrated. Such guarantees are straightforward to prove in many settings (<span class="dropBlock reference-citations"><a id="body-ref-sref26" href="#" role="doc-biblioref" data-xml-rid="bib26" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref26" aria-controls="sref26">Efron, 2012</a><div class="dropBlock__holder" data-db-target-of="sref26"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">24.</div><div>Efron, B.</div><div><strong>Large-scale inference: empirical Bayes methods for estimation, testing, and prediction</strong></div><div>Cambridge University Press, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.EfronLarge-scale+inference%3A+empirical+Bayes+methods+for+estimation%2C+testing%2C+and+prediction2012Cambridge+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In the WES setting, however, and especially for the relatively discrete counts of <i>de novo</i> events, a demonstration that the FDR rate holds is warranted. It is worth noting that, even though there are many genes that contain no mutations, the mutation rate is gene-specific and varies with gene length. Consequently, with the exception of the genes with a signal, the p values from the TADA analysis of PTV and Mis3 mutations are almost uniformly distributed (<a id="crosref1355" href="#figs2">Figure&nbsp;S2</a>A).</div><div id="p0340" role="paragraph">To evaluate the validity of the FDR framework in the context of TADA analysis, we conduct “empirical-known signal experiments” (EKSE). The idea is to perform TADA analyses in which the true signal is known <i>a priori</i>. To make the simulation as real as possible, it is performed using real <i>de novo</i> mutation counts as a base. These mutations are chosen to carry no detectable signal (i.e., mimicking the null distribution because they are believed to be non-functional). Simulated signals for association are then generated for randomly selected genes. Once the data are generated, TADA is used to analyze them and the resulting FDR and other features of the method are examined.</div></section><section id="sec-4-6-4-3"><h5>EKSE Simulations to Assess the Properties of FDR</h5><div id="p0345" role="paragraph">For these empirically-known signal experiments, we let synonymous variants play the role of Mis3 (denoted as Mis3new) and “benign” missense variants according to PolyPhen-2 (Mis1) play the role of PTV (denoted as PTVnew). Signals are layered onto genes that are randomly chosen. Below is the detailed procedure:<div id="olist0010" role="list"><div id="o0010" role="listitem"><div class="label">1)</div><div class="content"><div id="p0350" role="paragraph">Divide all 17,484 autosomal protein-coding genes into 20 bins of equal size. Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b equals" tabindex="0" ctxtmenu_counter="60" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="b equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 1, …, 20.</div></div></div><div id="o0015" role="listitem"><div class="label">2)</div><div class="content"><div id="p0355" role="paragraph">For each of the 20 bins, iteratively generate a signal for all genes in the bin; the remaining 19 bins, with no signal, represent the&nbsp;null genes. The extra signal in the <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i" tabindex="0" ctxtmenu_counter="61" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="i"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>th gene for both new kinds of <i>de novo</i> variants is simulated using <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper X Subscript i Baseline vertical bar gamma Subscript i Baseline tilde Poisson left parenthesis 2 mu Subscript i Baseline left parenthesis gamma Subscript i Baseline minus 1 right parenthesis upper N right parenthesis" tabindex="0" ctxtmenu_counter="62" style="font-size: 130%;"><mjx-math data-semantic-structure="(35 (34 (4 (2 0 1) 3) 33 (7 5 6)) 8 (32 9 31 (30 10 (28 11 25 (14 12 13) 26 (23 15 (21 (18 16 17) 19 20) 22) 27 24) 29)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="35" data-semantic-children="34,32" data-semantic-content="8" data-semantic-owns="34 8 32" data-semantic-speech="upper X Subscript i Baseline vertical bar gamma Subscript i Baseline tilde Poisson left parenthesis 2 mu Subscript i Baseline left parenthesis gamma Subscript i Baseline minus 1 right parenthesis upper N right parenthesis"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="34" data-semantic-children="4,7" data-semantic-content="33" data-semantic-parent="35" data-semantic-owns="4 33 7"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="endpunct" data-semantic-id="4" data-semantic-children="2,3" data-semantic-content="3" data-semantic-parent="34" data-semantic-owns="2 3"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="4" data-semantic-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D44B">𝑋</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.023em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="vbar" data-semantic-id="3" data-semantic-parent="4" data-semantic-operator="punctuated"><mjx-c class="mjx-c7C">|</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="7" data-semantic-children="5,6" data-semantic-parent="34" data-semantic-owns="5 6"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="8" data-semantic-parent="35" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="32" data-semantic-children="9,30" data-semantic-content="31,9" data-semantic-parent="35" data-semantic-owns="9 31 30"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="9" data-semantic-parent="32" data-semantic-operator="appl"><mjx-c class="mjx-c50" noic="true" style="padding-top: 0.683em;">P</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.683em;">o</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.683em;">i</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.683em;">s</mjx-c><mjx-c class="mjx-c73" noic="true" style="padding-top: 0.683em;">s</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.683em;">o</mjx-c><mjx-c class="mjx-c6E" style="padding-top: 0.683em;">n</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="30" data-semantic-children="28" data-semantic-content="10,29" data-semantic-parent="32" data-semantic-owns="10 28 29"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="10" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="28" data-semantic-children="11,14,23,24" data-semantic-content="25,26,27" data-semantic-parent="30" data-semantic-owns="11 25 14 26 23 27 24"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="28"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="14" data-semantic-children="12,13" data-semantic-parent="28" data-semantic-owns="12 13"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="14"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="23" data-semantic-children="21" data-semantic-content="15,22" data-semantic-parent="28" data-semantic-owns="15 21 22"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="15" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="21" data-semantic-children="18,20" data-semantic-content="19" data-semantic-parent="23" data-semantic-owns="18 19 20"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="18" data-semantic-children="16,17" data-semantic-parent="21" data-semantic-owns="16 17"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="18"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="19" data-semantic-parent="21" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="27" data-semantic-parent="28" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="28"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="29" data-semantic-parent="30" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript i Baseline tilde Gamma left parenthesis gamma comma beta right parenthesis" tabindex="0" ctxtmenu_counter="63" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (2 0 1) 3 (13 4 12 (11 5 (9 6 7 8) 10)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="2,13" data-semantic-content="3" data-semantic-owns="2 3 13" data-semantic-speech="gamma Subscript i Baseline tilde Gamma left parenthesis gamma comma beta right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="14" data-semantic-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="14" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="13" data-semantic-children="4,11" data-semantic-content="12,4" data-semantic-parent="14" data-semantic-owns="4 12 11"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="4" data-semantic-parent="13" data-semantic-operator="appl"><mjx-c class="mjx-c47" noic="true" style="padding-top: 0.705em;">G</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c61" style="padding-top: 0.705em;">a</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="11" data-semantic-children="9" data-semantic-content="5,10" data-semantic-parent="13" data-semantic-owns="5 9 10"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="6,7,8" data-semantic-content="7" data-semantic-parent="11" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="10" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. The hyperparameters are selected to yield signals similar to the real data: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta equals" tabindex="0" ctxtmenu_counter="64" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="beta equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.2 and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="65" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is set to be 2.4 and 5.4 for Mis3new and PTVnew respectively, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N equals" tabindex="0" ctxtmenu_counter="66" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper N equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 6,430. The “-1” in the Poisson equation is to account for the observed <i>de novo</i> variants already included from the real data. The simulated <i>de novo</i> events are added to the observed Mis3new and PTVnew to create each of the 20 datasets.</div></div></div><div id="o0020" role="listitem"><div class="label">3)</div><div class="content"><div id="p0360" role="paragraph">Perform TADA analysis for each of the 20 datasets.</div></div></div><div id="o0025" role="listitem"><div class="label">4)</div><div class="content"><div id="p0365" role="paragraph">Display the resulting q-FDR curves for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b equals" tabindex="0" ctxtmenu_counter="67" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="b equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 1, …, 20, and q-FDR averaged over <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="b" tabindex="0" ctxtmenu_counter="68" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="b"><mjx-c class="mjx-c1D44F">𝑏</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>.</div></div></div></div></div></section><section id="sec-4-6-4-4"><h5>Pure Simulations to Assess the Properties of FDR</h5><div id="p0370" role="paragraph">This simulation is closely related to EKSE. The only difference is that the null mutations are generated randomly from a multinomial distribution instead of adopted directly from the synonymous and Mis1 variants. The procedure is described below:<div id="olist0015" role="list"><div id="o0030" role="listitem"><div class="label">1)</div><div class="content"><div id="p0375" role="paragraph">Randomly sample a fraction of all 17,484 autosomal protein-coding genes as signal genes, denoted as set <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper S" tabindex="0" ctxtmenu_counter="69" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper S"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. We set the fraction as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="pi equals" tabindex="0" ctxtmenu_counter="70" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="pi equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.05. The number of trios is <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N equals" tabindex="0" ctxtmenu_counter="71" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper N equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 6,430.</div></div></div><div id="o0035" role="listitem"><div class="label">2)</div><div class="content"><div id="p0380" role="paragraph">For both new types of variants, Mis3new and PTVnew, the mutations of all the genes are randomly generated from a multinomial distribution, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="bold upper X tilde Multinom left parenthesis upper M comma bold p right parenthesis" tabindex="0" ctxtmenu_counter="72" style="font-size: 130%;"><mjx-math data-semantic-structure="(12 0 1 (11 2 10 (9 3 (7 4 5 6) 8)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="12" data-semantic-children="0,11" data-semantic-content="1" data-semantic-owns="0 1 11" data-semantic-speech="bold upper X tilde Multinom left parenthesis upper M comma bold p right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="12"><mjx-c class="mjx-c1D417">𝐗</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="12" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="11" data-semantic-children="2,9" data-semantic-content="10,2" data-semantic-parent="12" data-semantic-owns="2 10 9"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="2" data-semantic-parent="11" data-semantic-operator="appl"><mjx-c class="mjx-c4D" noic="true" style="padding-top: 0.694em;">M</mjx-c><mjx-c class="mjx-c75" noic="true" style="padding-top: 0.694em;">u</mjx-c><mjx-c class="mjx-c6C" noic="true" style="padding-top: 0.694em;">l</mjx-c><mjx-c class="mjx-c74" noic="true" style="padding-top: 0.694em;">t</mjx-c><mjx-c class="mjx-c69" noic="true" style="padding-top: 0.694em;">i</mjx-c><mjx-c class="mjx-c6E" noic="true" style="padding-top: 0.694em;">n</mjx-c><mjx-c class="mjx-c6F" noic="true" style="padding-top: 0.694em;">o</mjx-c><mjx-c class="mjx-c6D" style="padding-top: 0.694em;">m</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="9" data-semantic-children="7" data-semantic-content="3,8" data-semantic-parent="11" data-semantic-owns="3 7 8"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="3" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="7" data-semantic-children="4,5,6" data-semantic-content="5" data-semantic-parent="9" data-semantic-owns="4 5 6"><mjx-mtext data-semantic-type="text" data-semantic-role="annotation" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c4D">M</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="5" data-semantic-parent="7" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="7"><mjx-c class="mjx-c1D429">𝐩</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="8" data-semantic-parent="9" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where the probability vector <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="bold p" tabindex="0" ctxtmenu_counter="73" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="bold p"><mjx-c class="mjx-c1D429">𝐩</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is proportional to <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="bold p equals StartSet mu Subscript i Baseline gamma Subscript i Baseline EndSet Subscript i equals 1 comma ellipsis comma 17484" tabindex="0" ctxtmenu_counter="74" style="font-size: 130%;"><mjx-math data-semantic-structure="(23 0 1 (22 (12 2 (10 (5 3 4) 9 (8 6 7)) 11) (21 (20 13 14 15) 16 17 18 19)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="23" data-semantic-children="0,22" data-semantic-content="1" data-semantic-owns="0 1 22" data-semantic-speech="bold p equals StartSet mu Subscript i Baseline gamma Subscript i Baseline EndSet Subscript i equals 1 comma ellipsis comma 17484"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="bold" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="23"><mjx-c class="mjx-c1D429">𝐩</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="23" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="set singleton" data-semantic-id="22" data-semantic-children="12,21" data-semantic-parent="23" data-semantic-owns="12 21"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="12" data-semantic-children="10" data-semantic-content="2,11" data-semantic-parent="22" data-semantic-owns="2 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c7B">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="10" data-semantic-children="5,8" data-semantic-content="9" data-semantic-parent="12" data-semantic-owns="5 9 8"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="3,4" data-semantic-parent="10" data-semantic-owns="3 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="10" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="8" data-semantic-children="6,7" data-semantic-parent="10" data-semantic-owns="6 7"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c7D">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s" data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="21" data-semantic-children="20,16,17,18,19" data-semantic-content="16,18" data-semantic-parent="22" data-semantic-owns="20 16 17 18 19"><mjx-mrow data-semantic-added="true" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="20" data-semantic-children="13,15" data-semantic-content="14" data-semantic-parent="21" data-semantic-owns="13 14 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="14" data-semantic-parent="20" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="16" data-semantic-parent="21" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="ellipsis" data-semantic-id="17" data-semantic-parent="21"><mjx-c class="mjx-c2026">…</mjx-c></mjx-mo><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="18" data-semantic-parent="21" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.677em;">1</mjx-c><mjx-c class="mjx-c37" noic="true" style="padding-top: 0.677em;">7</mjx-c><mjx-c class="mjx-c34" noic="true" style="padding-top: 0.677em;">4</mjx-c><mjx-c class="mjx-c38" noic="true" style="padding-top: 0.677em;">8</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript i Baseline tilde Gamma left parenthesis gamma comma beta right parenthesis" tabindex="0" ctxtmenu_counter="75" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 (2 0 1) 3 (13 4 12 (11 5 (9 6 7 8) 10)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="14" data-semantic-children="2,13" data-semantic-content="3" data-semantic-owns="2 3 13" data-semantic-speech="gamma Subscript i Baseline tilde Gamma left parenthesis gamma comma beta right parenthesis"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="14" data-semantic-owns="0 1"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="3" data-semantic-parent="14" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="appl" data-semantic-role="prefix function" data-semantic-id="13" data-semantic-children="4,11" data-semantic-content="12,4" data-semantic-parent="14" data-semantic-owns="4 12 11"><mjx-mtext data-semantic-type="function" data-semantic-role="prefix function" data-semantic-font="normal" data-semantic-id="4" data-semantic-parent="13" data-semantic-operator="appl"><mjx-c class="mjx-c47" noic="true" style="padding-top: 0.705em;">G</mjx-c><mjx-c class="mjx-c61" noic="true" style="padding-top: 0.705em;">a</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c6D" noic="true" style="padding-top: 0.705em;">m</mjx-c><mjx-c class="mjx-c61" style="padding-top: 0.705em;">a</mjx-c></mjx-mtext><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="11" data-semantic-children="9" data-semantic-content="5,10" data-semantic-parent="13" data-semantic-owns="5 9 10"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="5" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="9" data-semantic-children="6,7,8" data-semantic-content="7" data-semantic-parent="11" data-semantic-owns="6 7 8"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="7" data-semantic-parent="9" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="9"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="10" data-semantic-parent="11" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> if <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="i element of upper S" tabindex="0" ctxtmenu_counter="76" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="i element of upper S"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-mi space="4" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, otherwise equals 1. The total number of mutations is <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 2 upper N sigma summation Underscript i equals 1 Overscript 17484 Endscripts mu Subscript i Baseline gamma Subscript i" tabindex="0" ctxtmenu_counter="77" style="font-size: 130%;"><mjx-math data-semantic-structure="(23 0 1 (22 2 20 3 21 (19 (10 4 (8 5 6 7) 9) (18 (13 11 12) 17 (16 14 15)))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="23" data-semantic-children="0,22" data-semantic-content="1" data-semantic-owns="0 1 22" data-semantic-speech="upper M equals 2 upper N sigma summation Underscript i equals 1 Overscript 17484 Endscripts mu Subscript i Baseline gamma Subscript i"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="23"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="23" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="2,3,19" data-semantic-content="20,21" data-semantic-parent="23" data-semantic-owns="2 20 3 21 19"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="22"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="22"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="19" data-semantic-children="10,18" data-semantic-content="4" data-semantic-parent="22" data-semantic-owns="10 18"><mjx-munderover data-semantic-type="limboth" data-semantic-role="sum" data-semantic-id="10" data-semantic-children="4,8,9" data-semantic-parent="19" data-semantic-owns="4 8 9" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="4" data-semantic-parent="10" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em; margin-left: 0px;"><mjx-mrow size="s"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="10"><mjx-c class="mjx-c31" noic="true" style="padding-top: 0.677em;">1</mjx-c><mjx-c class="mjx-c37" noic="true" style="padding-top: 0.677em;">7</mjx-c><mjx-c class="mjx-c34" noic="true" style="padding-top: 0.677em;">4</mjx-c><mjx-c class="mjx-c38" noic="true" style="padding-top: 0.677em;">8</mjx-c><mjx-c class="mjx-c34" style="padding-top: 0.677em;">4</mjx-c></mjx-mn></mjx-mrow><mjx-spacer style="margin-top: 0.276em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-munderover><mjx-mrow space="2" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="13,16" data-semantic-content="17" data-semantic-parent="19" data-semantic-owns="13 17 16"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="13" data-semantic-children="11,12" data-semantic-parent="18" data-semantic-owns="11 12"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="16" data-semantic-children="14,15" data-semantic-parent="18" data-semantic-owns="14 15"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="16"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="16"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. The mutation rates of Mis3new are taken from synonymous variants, and the mutation rates of PTVnew are taken from Mis1. The hyperparameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma comma beta" tabindex="0" ctxtmenu_counter="78" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="3" data-semantic-children="0,1,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="gamma comma beta"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> are set to be the same as in EKSE.</div></div></div><div id="o0040" role="listitem"><div class="label">3)</div><div class="content"><div id="p0385" role="paragraph">Perform TADA analysis on the two generated types of variants. Display the resulting q-FDR curve.</div></div></div><div id="o0045" role="listitem"><div class="label">4)</div><div class="content"><div id="p0390" role="paragraph">Repeat steps 1-3 one hundred times.</div></div></div></div></div></section><section id="sec-4-6-4-5"><h5>Results of simulations</h5><div id="p0395" role="paragraph"><a id="crosref1360" href="#figs2">Figure&nbsp;S2</a>B shows the averaged actual FDR versus the q-value over the 20 EKSE experiments. The error bars are obtained from the pure simulation. For q <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than" tabindex="0" ctxtmenu_counter="79" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="less than"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.1 the average curve follows the diagonal line (roughly), which indicates that the actual FDR is well controlled in the region of primary interest. We do detect a slight bump in the actual FDR for q <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than" tabindex="0" ctxtmenu_counter="80" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.1. To understand this deviation, we compared the observed counts for synonymous (Mis3new) and Mis1 (PTVnew) to simulated counts generated from the model.</div><div id="p0400" role="paragraph">The distribution of the number of genes with synonymous counts <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="81" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>3 and Mis1 counts <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="82" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>2 is contrasted with the observed counts (<a id="crosref1365" href="#figs2">Figure&nbsp;S2</a>C). The contrasts show that there is a slight excess of multiple hits in the observed counts compared to the model. Adding counts of synonymous and Mis1 mutations we obtain a single distribution of mutations per gene and find that there is an excess of counts of 0, 2, 3, and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than" tabindex="0" ctxtmenu_counter="83" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container>3 and a relative lack of counts of 1; overall the counts are fairly similar, but they differ significantly from expectations (chi-square p value&nbsp;= 0.012). The 8 null genes with the strongest TADA signal are <i>GNS, LRRFIP1, GALC, GRN, MYH9, FOXK2, AP1B1</i>, and <i>UNC45B</i>, and these are the genes that contribute to the bump in the FDR. However, none of these genes are significant (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="q less than" tabindex="0" ctxtmenu_counter="84" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="inequality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="q less than"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D45E">𝑞</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,<"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.1) in the EKSE analysis or in the actual data analysis of Mis3 and PTV mutations. From this EKSE experiment we conclude that the TADA model does not perfectly capture reality and the actual FDR deviates slightly from reported value for values of q <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than" tabindex="0" ctxtmenu_counter="85" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.1. This deviation is likely due to inexact estimates of the per gene mutation rate.</div><div id="p0405" role="paragraph">TADA relies on a mutation rate model for genes, which is an estimated quantity. Hence, we evaluate the impact of misspecification of mutation rates. To quantify the deviation from the expected null distribution due to mutation rate misspecification, we use the theory of genomic control (<span class="dropBlock reference-citations"><a id="body-ref-sref24" href="#" role="doc-biblioref" data-xml-rid="bib24" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref24" aria-controls="sref24">Devlin and Roeder, 1999</a><div class="dropBlock__holder" data-db-target-of="sref24"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">22.</div><div>Devlin, B. ∙ Roeder, K.</div><div><strong>Genomic control for association studies</strong></div><div><em>Biometrics.</em> 1999; <strong>55</strong>:997-1004</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.0006-341X.1999.00997.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11315092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.0006-341X.1999.00997.x&amp;pmid=11315092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), specifically estimating the inflation factor <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda Subscript upper G upper C" tabindex="0" ctxtmenu_counter="86" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="lamda Subscript upper G upper C"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>. In this experiment, we randomly select 10%–50% of genes and artificially make the nominal mutation rates increasingly lower than their actual estimated rates. This will make the observed mutation count larger than the expected count for a subset of genes. The result is that test statistics for association will tend to be increased for some genes, and the larger the discrepancy, the larger the set of test statistics that do not follow the expected null distribution. The genomic control factor, based on the z-statistics from the TADA analysis (<a id="crosref1375" href="#figs2">Figure&nbsp;S2</a>D), quantifies this inflation. As expected, the genomic control factor increases as more genes are analyzed with lowered nominal mutation rates (<a id="crosref1380" href="#figs2">Figure&nbsp;S2</a>E). The inflation for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda Subscript upper G upper C" tabindex="0" ctxtmenu_counter="87" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="lamda Subscript upper G upper C"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> is modest, however, even for these fairly notable misspecifications of the mutation rates.</div><div id="p0410" role="paragraph">Because TADA is a Bayesian method, it is more natural to use FDR than a Family-Wise Error Rate (FWER) cutoff to determine significance. In this gene discovery setting it is informative to compare the numbers of true discoveries (TD), false discoveries (FD), and FDR for different p value and FDR thresholds and to examine the impact of model mis-specifications on FDR (<a id="crosref1385" href="#figs2">Figure&nbsp;S2</a>F). We measure discrepancies via the genomic control factor <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis lamda Subscript upper G upper C Baseline right parenthesis" tabindex="0" ctxtmenu_counter="88" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 (6 1 (5 2 4 3)) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="0,7" data-semantic-owns="0 6 7" data-semantic-speech="left parenthesis lamda Subscript upper G upper C Baseline right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="1,5" data-semantic-parent="8" data-semantic-owns="1 5"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. We simulate the Z-value of 20,000 genes, 5% with a signal from <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N left parenthesis mu comma lamda Subscript upper G upper C Baseline right parenthesis" tabindex="0" ctxtmenu_counter="89" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 13 (12 1 (10 2 3 (9 4 (8 5 7 6))) 11))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="14" data-semantic-children="0,12" data-semantic-content="13,0" data-semantic-owns="0 13 12" data-semantic-speech="upper N left parenthesis mu comma lamda Subscript upper G upper C Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="1,11" data-semantic-parent="14" data-semantic-owns="1 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="2,3,9" data-semantic-content="3" data-semantic-parent="12" data-semantic-owns="2 3 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="10"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="4,8" data-semantic-parent="10" data-semantic-owns="4 8"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="8" data-semantic-children="5,6" data-semantic-content="7" data-semantic-parent="9" data-semantic-owns="5 7 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> and 95% from the null <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N left parenthesis 0 comma lamda Subscript upper G upper C Baseline right parenthesis" tabindex="0" ctxtmenu_counter="90" style="font-size: 130%;"><mjx-math data-semantic-structure="(14 0 13 (12 1 (10 2 3 (9 4 (8 5 7 6))) 11))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="14" data-semantic-children="0,12" data-semantic-content="13,0" data-semantic-owns="0 13 12" data-semantic-speech="upper N left parenthesis 0 comma lamda Subscript upper G upper C Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="14" data-semantic-operator="appl"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="13" data-semantic-parent="14" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="12" data-semantic-children="10" data-semantic-content="1,11" data-semantic-parent="14" data-semantic-owns="1 10 11"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="10" data-semantic-children="2,3,9" data-semantic-content="3" data-semantic-parent="12" data-semantic-owns="2 3 9"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="10"><mjx-c class="mjx-c30">0</mjx-c></mjx-mn><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="3" data-semantic-parent="10" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-msub space="2" data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="4,8" data-semantic-parent="10" data-semantic-owns="4 8"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="8" data-semantic-children="5,6" data-semantic-content="7" data-semantic-parent="9" data-semantic-owns="5 7 6"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="8"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="11" data-semantic-parent="12" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda Subscript upper G upper C" tabindex="0" ctxtmenu_counter="91" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="lamda Subscript upper G upper C"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> varies from 1 to 1.2. The value of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu" tabindex="0" ctxtmenu_counter="92" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="mu"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is chosen to be 2 to approximately mimic the real data. Based on 1,000 replications, we calculate the average TD, FD, and FDR for a Bonferroni-adjusted p value threshold and different FDR thresholds. As expected, FWER has considerably fewer FD but also notably fewer TD than FDR, and the observed FDR is well calibrated when <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda Subscript upper G upper C Baseline equals 1" tabindex="0" ctxtmenu_counter="93" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (5 0 (4 1 3 2)) 6 7)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-owns="5 6 7" data-semantic-speech="lamda Subscript upper G upper C Baseline equals 1"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="8" data-semantic-owns="0 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container> (Figure&nbsp;S2F). (For <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda Subscript upper G upper C Baseline equals 1" tabindex="0" ctxtmenu_counter="94" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 (5 0 (4 1 3 2)) 6 7)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-owns="5 6 7" data-semantic-speech="lamda Subscript upper G upper C Baseline equals 1"><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-parent="8" data-semantic-owns="0 4"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, TD&nbsp;= 5, 52, 113, 334, and FD&nbsp;= 0.1, 3, 13, and 144 for the four thresholds examined. In each case the error rate is controlled at the expected rate.) However, as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda Subscript upper G upper C" tabindex="0" ctxtmenu_counter="95" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 (4 1 3 2))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="0,4" data-semantic-owns="0 4" data-semantic-speech="lamda Subscript upper G upper C"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="4" data-semantic-children="1,2" data-semantic-content="3" data-semantic-parent="5" data-semantic-owns="1 3 2"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="3" data-semantic-parent="4" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D436">𝐶</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> increases the actual FDR increases rapidly, especially for larger q-values. In contrast, FWER is fairly well controlled even for model discrepancies.</div><div id="p0415" role="paragraph">Next, again by using simulations, we investigate how varying the number of risk genes impacts q-values for genes near but not over the FDR threshold. The key idea is to examine how the q-values of the borderline genes—defined to be genes with 0.05&nbsp;&lt; q&nbsp;&lt; 0.1—vary as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="pi" tabindex="0" ctxtmenu_counter="96" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="pi"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, the fraction of signal genes, changes. We expect that, as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="pi" tabindex="0" ctxtmenu_counter="97" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="pi"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> increases, q-values of genes near the threshold will decrease. Indeed, we observe that the q-values of both non-signal (<a id="crosref1390" href="#figs2">Figure&nbsp;S2</a>G) and signal (<a id="crosref1395" href="#figs2">Figure&nbsp;S2</a>H) borderline genes decrease as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="pi" tabindex="0" ctxtmenu_counter="98" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="pi"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> increases, but the impact on the non-signal borderline genes is not substantial and q-values for only a few of these genes cross the threshold.</div></section></section><section id="sec-4-6-5"><h4>A more powerful TADA model (TADA<sup>+</sup>)</h4><div id="p0420" role="paragraph">TADA requires input of several parameters, most notably the relative risk, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="99" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. To estimate the relative risk for a category of mutations, we use the burden-relative risk relationship derived in <span class="dropBlock reference-citations"><a id="body-ref-sref35-6" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-6" aria-controls="sref35-6">He et&nbsp;al. (2013)</a><div class="dropBlock__holder" data-db-target-of="sref35-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>: <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma equals 1 plus left parenthesis lamda minus 1 right parenthesis divided by pi" tabindex="0" ctxtmenu_counter="100" style="font-size: 130%;"><mjx-math data-semantic-structure="(15 0 1 (14 2 3 (13 (10 4 (8 5 6 7) 9) 11 12)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="15" data-semantic-children="0,14" data-semantic-content="1" data-semantic-owns="0 1 14" data-semantic-speech="gamma equals 1 plus left parenthesis lamda minus 1 right parenthesis divided by pi"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="15"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="15" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="addition" data-semantic-id="14" data-semantic-children="2,13" data-semantic-content="3" data-semantic-parent="15" data-semantic-owns="2 3 13"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="14"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="addition" data-semantic-id="3" data-semantic-parent="14" data-semantic-operator="infixop,+"><mjx-c class="mjx-c2B">+</mjx-c></mjx-mo><mjx-mrow space="3" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="13" data-semantic-children="10,12" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="10 11 12"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="4,9" data-semantic-parent="13" data-semantic-owns="4 8 9"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="4" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="8" data-semantic-children="5,7" data-semantic-content="6" data-semantic-parent="10" data-semantic-owns="5 6 7"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="8"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="6" data-semantic-parent="8" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mn space="3" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="8"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="11" data-semantic-parent="13" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="13"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="pi equals" tabindex="0" ctxtmenu_counter="101" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="pi equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D70B">𝜋</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.05 is the estimated fraction of risk genes and the burden <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda" tabindex="0" ctxtmenu_counter="102" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="lamda"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is calculated by comparing mutation counts in probands and unaffected siblings. Because differences in sequencing depths and variant calling procedures may lead to systematic differences in mutation rates, we normalize the counts using synonymous mutations counts. Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="x" tabindex="0" ctxtmenu_counter="103" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="x"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper S" tabindex="0" ctxtmenu_counter="104" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper S"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> be the number of mutations in the category of interest and compare the counts in cases (cs) and controls (cn) as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="lamda equals left parenthesis x Subscript c s Baseline upper S Subscript c n Baseline right parenthesis divided by left parenthesis x Subscript c n Baseline upper S Subscript c s Baseline right parenthesis" tabindex="0" ctxtmenu_counter="105" style="font-size: 130%;"><mjx-math data-semantic-structure="(38 0 1 (37 (18 2 (16 (8 3 (7 4 6 5)) 15 (14 9 (13 10 12 11))) 17) 19 (36 20 (34 (26 21 (25 22 24 23)) 33 (32 27 (31 28 30 29))) 35)))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="38" data-semantic-children="0,37" data-semantic-content="1" data-semantic-owns="0 1 37" data-semantic-speech="lamda equals left parenthesis x Subscript c s Baseline upper S Subscript c n Baseline right parenthesis divided by left parenthesis x Subscript c n Baseline upper S Subscript c s Baseline right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="38"><mjx-c class="mjx-c1D706">𝜆</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="38" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="37" data-semantic-children="18,36" data-semantic-content="19" data-semantic-parent="38" data-semantic-owns="18 19 36"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="18" data-semantic-children="16" data-semantic-content="2,17" data-semantic-parent="37" data-semantic-owns="2 16 17"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="8,14" data-semantic-content="15" data-semantic-parent="18" data-semantic-owns="8 15 14"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="8" data-semantic-children="3,7" data-semantic-parent="16" data-semantic-owns="3 7"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="4,5" data-semantic-content="6" data-semantic-parent="8" data-semantic-owns="4 6 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="7"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="14" data-semantic-children="9,13" data-semantic-parent="16" data-semantic-owns="9 13"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="14"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.032em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="13" data-semantic-children="10,11" data-semantic-content="12" data-semantic-parent="14" data-semantic-owns="10 12 11"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="13"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="12" data-semantic-parent="13" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="13"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="17" data-semantic-parent="18" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-mo space="2" data-semantic-type="operator" data-semantic-role="division" data-semantic-id="19" data-semantic-parent="37" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow space="2"><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="36" data-semantic-children="34" data-semantic-content="20,35" data-semantic-parent="37" data-semantic-owns="20 34 35"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="20" data-semantic-parent="36" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="34" data-semantic-children="26,32" data-semantic-content="33" data-semantic-parent="36" data-semantic-owns="26 33 32"><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="26" data-semantic-children="21,25" data-semantic-parent="34" data-semantic-owns="21 25"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="21" data-semantic-parent="26"><mjx-c class="mjx-c1D465">𝑥</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="25" data-semantic-children="22,23" data-semantic-content="24" data-semantic-parent="26" data-semantic-owns="22 24 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="22" data-semantic-parent="25"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="23" data-semantic-parent="25"><mjx-c class="mjx-c1D45B">𝑛</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="33" data-semantic-parent="34" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="32" data-semantic-children="27,31" data-semantic-parent="34" data-semantic-owns="27 31"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="32"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.032em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="31" data-semantic-children="28,29" data-semantic-content="30" data-semantic-parent="32" data-semantic-owns="28 30 29"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="31"><mjx-c class="mjx-c1D450">𝑐</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="31" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="31"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="35" data-semantic-parent="36" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>.</div><div id="p0425" role="paragraph">Previous TADA analyses (<span class="dropBlock reference-citations"><a id="body-ref-sref92-5" href="#" role="doc-biblioref" data-xml-rid="bib21 bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-5" aria-controls="sref92-5">De Rubeis et&nbsp;al., 2014; Sanders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) used two annotation categories, PTV and Mis3. Here we develop a more powerful version of TADA, which uses additional annotation information. For clarity we will label the original version TADA<sup>O</sup> and the refined model TADA<sup>+</sup>.</div><div id="p0430" role="paragraph">Recent studies have refined our understanding of what variation is likely to be meaningful for risk in two ways. Regarding PTVs, <span class="dropBlock reference-citations"><a id="body-ref-sref46-6" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-6" aria-controls="sref46-6">Kosmicki et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref46-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> showed that signals carried by PTVs involve a subset of genes that are evolutionarily constrained. For these genes, the population tends to have far fewer PTVs than would be expected based on gene size, base-pair content, and evolutionary models. This constraint feature of genes is embodied in pLI (the probability of being loss-of-function [aka PTV] intolerant) (<span class="dropBlock reference-citations"><a id="body-ref-sref51-5" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-5" aria-controls="sref51-5">Lek et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), which is a metric ranging from zero to one, with a larger pLI representing a greater dearth of PTV variation. <span class="dropBlock reference-citations"><a id="body-ref-sref46-7" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-7" aria-controls="sref46-7">Kosmicki et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref46-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> found that genes with pLI <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than" tabindex="0" ctxtmenu_counter="106" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than"><mjx-c class="mjx-c3E">&gt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 0.9 tend to harbor most of the ASD association signal from PTVs. In this work, we model the relative risk <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis gamma right parenthesis" tabindex="0" ctxtmenu_counter="107" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="3" data-semantic-children="1" data-semantic-content="0,2" data-semantic-owns="0 1 2" data-semantic-speech="left parenthesis gamma right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="3"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="2" data-semantic-parent="3" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container> of <i>de novo</i> PTVs as a continuous function of pLI. Figure&nbsp;S2I shows <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="108" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> as a function of pLI, where the x axis has been converted using the inverse normal transformation, but the original values of pLI are given. We create seven bins of data and fit a logistic curve to the data. The dots are the data and the black line is the fitted curve. Then we compute error bars based on the 95% prediction interval around the fitted curve. In the upcoming implementation we truncate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="109" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> at the null value of one.</div><div id="p0435" role="paragraph">More refined information is also available for missense variants. <span class="dropBlock reference-citations"><a id="body-ref-sref90-2" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-2" aria-controls="sref90-2">Samocha et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref90-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Samocha, K.E. ∙ Kosmicki, J.A. ∙ Karczewski, K.J. ...</div><div><strong>Regional missense constraint improves variant deleteriousness prediction</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/148353" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85093588144" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F148353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> recently introduced the MPC score, a missense deleteriousness metric composed of “Missense badness,” PolyPhen-2 (<span class="dropBlock reference-citations"><a id="body-ref-sref1-2" href="#" role="doc-biblioref" data-xml-rid="bib1" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref1-2" aria-controls="sref1-2">Adzhubei et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref1-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">1.</div><div>Adzhubei, I.A. ∙ Schmidt, S. ∙ Peshkin, L. ...</div><div><strong>A method and server for predicting damaging missense mutations</strong></div><div><em>Nat. Methods.</em> 2010; <strong>7</strong>:248-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth0410-248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77951640946" target="_blank" aria-label="Open Scopus in new tab">Scopus (6943)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20354512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth0410-248&amp;pmid=20354512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and Constraint. This metric also uses the concept of evolutionary constraint and seeks to quantify the degree of constraint for all missense variation in the genome. To determine how MPC might be used in TADA, we compute the average relative risk (<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="110" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, the hyperparameters for TADA) for a moving window of MPC in the ASC data (<a id="crosref1430" href="#figs2">Figure&nbsp;S2</a>J). Using a window over probands’ missense variants ordered by MPC score, and with a width of 7.5% of the variants, we obtain the curve showing the average relative risk as a function of MPC score. Three levels of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="111" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> naturally emerge from this relationship, with the first level (MPC <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than" tabindex="0" ctxtmenu_counter="112" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="less than"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 1) being close to marginal relative risk and two levels showing evidence for excess burden in ASD (<a id="crosref1435" href="#figs2">Figure&nbsp;S2</a>J). Based on the nature of these results, we chose to group missense mutations into two categories for TADA, using established thresholds of MPC (<span class="dropBlock reference-citations"><a id="body-ref-sref90-3" href="#" role="doc-biblioref" data-xml-rid="bib90" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref90-3" aria-controls="sref90-3">Samocha et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref90-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">86.</div><div>Samocha, K.E. ∙ Kosmicki, J.A. ∙ Karczewski, K.J. ...</div><div><strong>Regional missense constraint improves variant deleteriousness prediction</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/148353" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85093588144" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F148353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>): 1<mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than or equals" tabindex="0" ctxtmenu_counter="113" style="font-size: 130%;"><mjx-math data-semantic-structure="1" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mspace style="width: 0.25em;"></mjx-mspace><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="1" data-semantic-speech="less than or equals"><mjx-c class="mjx-c2264">≤</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> MPC <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than" tabindex="0" ctxtmenu_counter="114" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="less than"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 2 (MisA) and MPC <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="greater than or equals" tabindex="0" ctxtmenu_counter="115" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="greater than or equals"><mjx-c class="mjx-c2265">≥</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 2 (MisB). Note that missense variation with MPC <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="less than" tabindex="0" ctxtmenu_counter="116" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mo data-semantic-type="relation" data-semantic-role="inequality" data-semantic-id="0" data-semantic-speech="less than"><mjx-c class="mjx-c3C">&lt;</mjx-c></mjx-mo></mjx-mrow></mjx-math></mjx-container> 1 is treated as benign. The relative risk for each of the two missense categories is computed directly from the data (<span class="dropBlock reference-citations"><a id="body-ref-sref35-7" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-7" aria-controls="sref35-7">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper M i s upper A" tabindex="0" ctxtmenu_counter="117" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="gamma Subscript upper M i s upper A"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.153em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;= 4.18 and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript upper M i s upper B" tabindex="0" ctxtmenu_counter="118" style="font-size: 130%;"><mjx-math data-semantic-structure="(9 0 (8 1 5 2 6 3 7 4))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="0,8" data-semantic-owns="0 8" data-semantic-speech="gamma Subscript upper M i s upper B"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="9" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>&nbsp;= 22.15. We note that other tools for variant classification have been recently published (<span class="dropBlock reference-citations"><a id="body-ref-sref34" href="#" role="doc-biblioref" data-xml-rid="bib34" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref34" aria-controls="sref34">Havrilla et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref34"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">32.</div><div>Havrilla, J.M. ∙ Pedersen, B.S. ∙ Layer, R.M. ...</div><div><strong>A map of constrained coding regions in the human genome</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:88-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0294-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058142396" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30531870/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0294-6&amp;pmid=30531870" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and MPC score has been shown to have a comparable performance to these methods.</div><div id="p0440" role="paragraph">Besides the <i>de novo</i> variants, we also consider PTVs from case-control data by aggregating the iPSYCH (Danish) data and PAGES (Swedish) data. Following the same procedure as for <i>de novo</i> PTVs, within seven bins, we estimate the relative risks for the two case-control datasets separately and combine them with a precision weight. We then fit a logistic curve using the seven points to smooth <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="119" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> as a continuous function of pLI (<a id="crosref1455" href="#figs2">Figure&nbsp;S2</a>K). In the TADA analysis, we treat <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="120" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> of each gene as fixed for case-control data to achieve closed-form solutions and thus facilitate computation.</div><div id="p0445" role="paragraph">These analyses define three categories of variation that are potentially meaningful for risk. The gene-specific mutation rates for PTVs and missense variants have been reported previously (<span class="dropBlock reference-citations"><a id="body-ref-sref51-6" href="#" role="doc-biblioref" data-xml-rid="bib51" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref51-6" aria-controls="sref51-6">Lek et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref51-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">49.</div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and we further estimated the mutation rates for MisA and MisB. With mutation rates and hyperparameters estimated above, the refined TADA model can be applied to the data to identify risk genes for ASD. To allow for more variability in the prior for <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="121" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, we set <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta equals" tabindex="0" ctxtmenu_counter="122" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="beta equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.2.</div><div id="p0450" role="paragraph">To resolve an emerging issue with the model’s Bayes factor (BF) values, we implement a floor adjustment that imposes a lower bound of 1 on all BF. The issue is that for some genes with larger mutation rates and zero <i>de novo</i> MisB mutations, the MisB BF is substantially lower than 1. Multiplying this with the other evidence renders those genes not significant. Indeed, with the mutation rates provided and the high relative risk of MisB, the model clearly expected to observe at least one <i>de novo</i> MisB variant. (This happens for other categories as well, but most notably for MisB.) We think the problem is heterogeneity of genes—some genes with a <i>de novo</i> PTV just do not have MisB mutations in the data, even though these mutations are expected. It does not make sense to have the observation of no mutations drive the model. To circumvent the problem, we made a modification of the method so that BF is replaced by max(1,BF). We tested this in simulations and the size of the modified test is satisfactory (see the discussion in the next section).</div><div id="p0455" role="paragraph">TADA<sup>+</sup> incorporates all of the refinements delineated here. Using TADA<sup>+</sup>, 102 genes with q-value less than 0.1 are identified, including three genes that have excessive PTVs in siblings (<i>EIF3G</i>, <i>KDM5B</i>, <i>RAI1</i>). By contrast, TADA<sup>O</sup> identifies only 79 genes when applied to the same data. Clearly, the new relevant functional information embodied in the pLI and MPC scores improves the power of TADA by refining the model.</div><section id="sec-4-6-5-1"><h5>Simulations to evaluate TADA<sup>+</sup></h5><div id="p0460" role="paragraph">Simulations illustrate the performance of TADA<sup>+</sup> when applied to <i>de novo</i> mutations only. In this setting, we simulate three types of <i>de&nbsp;novo</i> variants: PTV, MisA, and MisB, using the mean risks and mutation rates from real data. Below is the detailed procedure.<div id="olist0020" role="list"><div id="o0050" role="listitem"><div class="label">1)</div><div class="content"><div id="p0465" role="paragraph">Randomly select 5% of 17,484 autosomal protein-coding genes as the signal genes; denote the set of signal genes as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper G Subscript upper S" tabindex="0" ctxtmenu_counter="123" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper G Subscript upper S"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> and the null genes as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper G Subscript upper N" tabindex="0" ctxtmenu_counter="124" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="upper G Subscript upper N"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>.</div></div></div><div id="o0055" role="listitem"><div class="label">2)</div><div class="content"><div id="p0470" role="paragraph">For each signal gene <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g element of upper G Subscript upper S" tabindex="0" ctxtmenu_counter="125" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="g element of upper G Subscript upper S"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D446">𝑆</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, generate risk <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript g" tabindex="0" ctxtmenu_counter="126" style="font-size: 130%;"><mjx-math data-semantic-structure="(2 0 1)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msub data-semantic-type="subscript" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-owns="0 1" data-semantic-speech="gamma Subscript g"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em; margin-left: -0.025em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="2"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container> for each three types of variants from a Gamma distribution, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript g Superscript a Baseline tilde upper G a m m a left parenthesis gamma overbar Subscript g Superscript a Baseline comma beta right parenthesis" tabindex="0" ctxtmenu_counter="127" style="font-size: 130%;"><mjx-math data-semantic-structure="(33 (4 (3 0 1) 2) 5 (32 6 30 (29 7 28 (27 8 26 9)) 31 (25 10 24 (23 11 (21 (18 (17 (14 12 13) 15) 16) 19 20) 22))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="33" data-semantic-children="4,32" data-semantic-content="5" data-semantic-owns="4 5 32" data-semantic-speech="gamma Subscript g Superscript a Baseline tilde upper G a m m a left parenthesis gamma overbar Subscript g Superscript a Baseline comma beta right parenthesis"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="4" data-semantic-children="0,1,2" data-semantic-parent="33" data-semantic-collapsed="(4 (3 0 1) 2)" data-semantic-owns="0 1 2"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="4"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="4"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="5" data-semantic-parent="33" data-semantic-operator="relseq,∼"><mjx-c class="mjx-c223C">∼</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="32" data-semantic-children="6,29,25" data-semantic-content="30,31" data-semantic-parent="33" data-semantic-owns="6 30 29 31 25"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="32"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-children="7,27" data-semantic-content="28,7" data-semantic-parent="32" data-semantic-owns="7 28 27"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="29" data-semantic-operator="appl"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="28" data-semantic-parent="29" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="27" data-semantic-children="8,9" data-semantic-content="26" data-semantic-parent="29" data-semantic-owns="8 26 9"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="27"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="26" data-semantic-parent="27" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="27"><mjx-c class="mjx-c1D45A">𝑚</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="32" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="appl" data-semantic-role="simple function" data-semantic-id="25" data-semantic-children="10,23" data-semantic-content="24,10" data-semantic-parent="32" data-semantic-owns="10 24 23"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="25" data-semantic-operator="appl"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo data-semantic-type="punctuation" data-semantic-role="application" data-semantic-id="24" data-semantic-parent="25" data-semantic-added="true" data-semantic-operator="appl"><mjx-c class="mjx-c2061">⁡</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="23" data-semantic-children="21" data-semantic-content="11,22" data-semantic-parent="25" data-semantic-owns="11 21 22"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="11" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c28 MM-LO">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="21" data-semantic-children="18,19,20" data-semantic-content="19" data-semantic-parent="23" data-semantic-owns="18 19 20"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="18" data-semantic-children="14,15,16" data-semantic-parent="21" data-semantic-collapsed="(18 (17 14 15) 16)" data-semantic-owns="14 15 16"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="14" data-semantic-children="12,13" data-semantic-parent="18" data-semantic-owns="12 13"><mjx-over style="padding-bottom: 0.105em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="13" data-semantic-parent="14"><mjx-stretchy-h class="mjx-c305" style="width: 0.543em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base style="padding-left: 0.013em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="simple function" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="18"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="18"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="19" data-semantic-parent="21" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="20" data-semantic-parent="21"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="22" data-semantic-parent="23" data-semantic-operator="fenced"><mjx-c class="mjx-c29 MM-LO">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="a equals left parenthesis upper P upper T upper V comma upper M i s upper A comma upper M i s upper B right parenthesis" tabindex="0" ctxtmenu_counter="128" style="font-size: 130%;"><mjx-math data-semantic-structure="(30 0 1 (29 2 (27 (18 3 16 4 17 5) 6 (22 7 19 8 20 9 21 10) 11 (26 12 23 13 24 14 25 15)) 28))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="30" data-semantic-children="0,29" data-semantic-content="1" data-semantic-owns="0 1 29" data-semantic-speech="a equals left parenthesis upper P upper T upper V comma upper M i s upper A comma upper M i s upper B right parenthesis"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="30"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="30" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="29" data-semantic-children="27" data-semantic-content="2,28" data-semantic-parent="30" data-semantic-owns="2 27 28"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="2" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="punctuated" data-semantic-role="sequence" data-semantic-id="27" data-semantic-children="18,6,22,11,26" data-semantic-content="6,11" data-semantic-parent="29" data-semantic-owns="18 6 22 11 26"><mjx-mrow data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="18" data-semantic-children="3,4,5" data-semantic-content="16,17" data-semantic-parent="27" data-semantic-owns="3 16 4 17 5"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="18"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="16" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="18"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="18" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="18"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="6" data-semantic-parent="27" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="22" data-semantic-children="7,8,9,10" data-semantic-content="19,20,21" data-semantic-parent="27" data-semantic-owns="7 19 8 20 9 21 10"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="22"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="22"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="20" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="9" data-semantic-parent="22"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="21" data-semantic-parent="22" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="22"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="punctuation" data-semantic-role="comma" data-semantic-id="11" data-semantic-parent="27" data-semantic-operator="punctuated"><mjx-c class="mjx-c2C">,</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="26" data-semantic-children="12,13,14,15" data-semantic-content="23,24,25" data-semantic-parent="27" data-semantic-owns="12 23 13 24 14 25 15"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="26"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="23" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="26"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="24" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="26"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="25" data-semantic-parent="26" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="26"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="28" data-semantic-parent="29" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="beta equals" tabindex="0" ctxtmenu_counter="129" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="beta equals"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D6FD">𝛽</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow data-semantic-added="true" data-semantic-type="empty" data-semantic-role="unknown" data-semantic-id="2" data-semantic-parent="3"></mjx-mrow></mjx-mrow></mjx-math></mjx-container> 0.2 and the hyperparameters <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma overbar Subscript g Superscript a" tabindex="0" ctxtmenu_counter="130" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 (5 (2 0 1) 3) 4)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="6" data-semantic-children="2,3,4" data-semantic-collapsed="(6 (5 2 3) 4)" data-semantic-owns="2 3 4" data-semantic-speech="gamma overbar Subscript g Superscript a"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="6" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.543em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base style="padding-left: 0.013em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="6"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>s are set to match the empirical counts. Note that <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma overbar Subscript g Superscript upper M i s upper A" tabindex="0" ctxtmenu_counter="131" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 (12 (2 0 1) 3) (11 4 8 5 9 6 10 7))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="13" data-semantic-children="2,3,11" data-semantic-collapsed="(13 (12 2 3) 11)" data-semantic-owns="2 3 11" data-semantic-speech="gamma overbar Subscript g Superscript upper M i s upper A"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="13" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.543em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base style="padding-left: 0.013em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="4,5,6,7" data-semantic-content="8,9,10" data-semantic-parent="13" data-semantic-owns="4 8 5 9 6 10 7" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="11"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="13"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma overbar Subscript g Superscript upper M i s upper B" tabindex="0" ctxtmenu_counter="132" style="font-size: 130%;"><mjx-math data-semantic-structure="(13 (12 (2 0 1) 3) (11 4 8 5 9 6 10 7))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="13" data-semantic-children="2,3,11" data-semantic-collapsed="(13 (12 2 3) 11)" data-semantic-owns="2 3 11" data-semantic-speech="gamma overbar Subscript g Superscript upper M i s upper B"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="13" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.543em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base style="padding-left: 0.013em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="4,5,6,7" data-semantic-content="8,9,10" data-semantic-parent="13" data-semantic-owns="4 8 5 9 6 10 7" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="11"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="11"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="11"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="11"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="13"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> are the same across all genes, but <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma overbar Subscript g Superscript upper P upper T upper V" tabindex="0" ctxtmenu_counter="133" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 (10 (2 0 1) 3) (9 4 7 5 8 6))" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="11" data-semantic-children="2,3,9" data-semantic-collapsed="(11 (10 2 3) 9)" data-semantic-owns="2 3 9" data-semantic-speech="gamma overbar Subscript g Superscript upper P upper T upper V"><mjx-mrow><mjx-mover data-semantic-type="overscore" data-semantic-role="greekletter" data-semantic-id="2" data-semantic-children="0,1" data-semantic-parent="11" data-semantic-owns="0 1"><mjx-over style="padding-bottom: 0.105em; margin-bottom: -0.63em;"><mjx-mo data-semantic-type="operator" data-semantic-role="overaccent" data-semantic-annotation="accent:bar" data-semantic-id="1" data-semantic-parent="2"><mjx-stretchy-h class="mjx-c305" style="width: 0.543em;"><mjx-ext><mjx-c class="MM-EM">̅</mjx-c></mjx-ext></mjx-stretchy-h></mjx-mo></mjx-over><mjx-base style="padding-left: 0.013em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="2"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-base></mjx-mover></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="9" data-semantic-children="4,5,6" data-semantic-content="7,8" data-semantic-parent="11" data-semantic-owns="4 7 5 8 6" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="9"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="8" data-semantic-parent="9" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="11"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container> are different.</div></div></div><div id="o0060" role="listitem"><div class="label">3)</div><div class="content"><div id="p0475" role="paragraph">For the null genes <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="g element of upper G Subscript upper N" tabindex="0" ctxtmenu_counter="134" style="font-size: 130%;"><mjx-math data-semantic-structure="(5 0 1 (4 2 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="element" data-semantic-id="5" data-semantic-children="0,4" data-semantic-content="1" data-semantic-owns="0 1 4" data-semantic-speech="g element of upper G Subscript upper N"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="operator" data-semantic-role="element" data-semantic-id="1" data-semantic-parent="5" data-semantic-operator="infixop,∈"><mjx-c class="mjx-c2208">∈</mjx-c></mjx-mo><mjx-msub space="4" data-semantic-type="subscript" data-semantic-role="latinletter" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="5" data-semantic-owns="2 3"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="4"><mjx-c class="mjx-c1D43A">𝐺</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.15em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msub></mjx-mrow></mjx-math></mjx-container>, set <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma Subscript g Superscript upper P upper T upper V Baseline equals gamma Subscript g Superscript upper M i s upper A Baseline equals gamma Subscript g Superscript upper M i s upper B Baseline equals 1" tabindex="0" ctxtmenu_counter="135" style="font-size: 130%;"><mjx-math data-semantic-structure="(38 (9 (8 0 1) (7 2 5 3 6 4)) 10 (22 (21 11 12) (20 13 17 14 18 15 19 16)) 23 (35 (34 24 25) (33 26 30 27 31 28 32 29)) 36 37)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="38" data-semantic-children="9,22,35,37" data-semantic-content="10,23,36" data-semantic-owns="9 10 22 23 35 36 37" data-semantic-speech="gamma Subscript g Superscript upper P upper T upper V Baseline equals gamma Subscript g Superscript upper M i s upper A Baseline equals gamma Subscript g Superscript upper M i s upper B Baseline equals 1"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="0,1,7" data-semantic-parent="38" data-semantic-collapsed="(9 (8 0 1) 7)" data-semantic-owns="0 1 7"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="9"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="9" data-semantic-owns="2 5 3 6 4" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c1D443">𝑃</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c1D447">𝑇</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D449">𝑉</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.282em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="9"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="10" data-semantic-parent="38" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="22" data-semantic-children="11,12,20" data-semantic-parent="38" data-semantic-collapsed="(22 (21 11 12) 20)" data-semantic-owns="11 12 20"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="22"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="13,14,15,16" data-semantic-content="17,18,19" data-semantic-parent="22" data-semantic-owns="13 17 14 18 15 19 16" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="13" data-semantic-parent="20"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="17" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="14" data-semantic-parent="20"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="20"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="16" data-semantic-parent="20"><mjx-c class="mjx-c1D434">𝐴</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="22"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="23" data-semantic-parent="38" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-msubsup space="4" data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="35" data-semantic-children="24,25,33" data-semantic-parent="38" data-semantic-collapsed="(35 (34 24 25) 33)" data-semantic-owns="24 25 33"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="24" data-semantic-parent="35"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="33" data-semantic-children="26,27,28,29" data-semantic-content="30,31,32" data-semantic-parent="35" data-semantic-owns="26 30 27 31 28 32 29" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="26" data-semantic-parent="33"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="30" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="27" data-semantic-parent="33"><mjx-c class="mjx-c1D456">𝑖</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="31" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="28" data-semantic-parent="33"><mjx-c class="mjx-c1D460">𝑠</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="32" data-semantic-parent="33" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="29" data-semantic-parent="33"><mjx-c class="mjx-c1D435">𝐵</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="25" data-semantic-parent="35"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="36" data-semantic-parent="38" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="37" data-semantic-parent="38"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>.</div></div></div><div id="o0065" role="listitem"><div class="label">4)</div><div class="content"><div id="p0480" role="paragraph">For each variant, generate the counts from a Multinomial distribution, where the total number is the expected total counts <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 upper N sigma summation Underscript g Endscripts mu Subscript g Superscript a Baseline gamma Subscript g Superscript a" tabindex="0" ctxtmenu_counter="136" style="font-size: 130%;"><mjx-math data-semantic-structure="(20 0 18 1 19 (17 (4 2 3) (16 (9 (8 5 6) 7) 15 (14 (13 10 11) 12))))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="20" data-semantic-children="0,1,17" data-semantic-content="18,19" data-semantic-owns="0 18 1 19 17" data-semantic-speech="2 upper N sigma summation Underscript g Endscripts mu Subscript g Superscript a Baseline gamma Subscript g Superscript a"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="20"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="18" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="20"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="19" data-semantic-parent="20" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-added="true" data-semantic-type="bigop" data-semantic-role="sum" data-semantic-id="17" data-semantic-children="4,16" data-semantic-content="2" data-semantic-parent="20" data-semantic-owns="4 16"><mjx-munder data-semantic-type="limlower" data-semantic-role="sum" data-semantic-id="4" data-semantic-children="2,3" data-semantic-parent="17" data-semantic-owns="2 3" limits="false"><mjx-mo data-semantic-type="largeop" data-semantic-role="sum" data-semantic-id="2" data-semantic-parent="4" data-semantic-operator="bigop"><mjx-c class="mjx-c2211">∑</mjx-c></mjx-mo><mjx-script style="vertical-align: -0.285em;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="4"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-munder><mjx-mrow space="2" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="16" data-semantic-children="9,14" data-semantic-content="15" data-semantic-parent="17" data-semantic-owns="9 15 14"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="9" data-semantic-children="5,6,7" data-semantic-parent="16" data-semantic-collapsed="(9 (8 5 6) 7)" data-semantic-owns="5 6 7"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="5" data-semantic-parent="9"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="9"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="9"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="15" data-semantic-parent="16" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="14" data-semantic-children="10,11,12" data-semantic-parent="16" data-semantic-collapsed="(14 (13 10 11) 12)" data-semantic-owns="10 11 12"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="10" data-semantic-parent="14"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="12" data-semantic-parent="14"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="11" data-semantic-parent="14"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N equals 6430" tabindex="0" ctxtmenu_counter="137" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper N equals 6430"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn space="4" data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c36" noic="true" style="padding-top: 0.677em;">6</mjx-c><mjx-c class="mjx-c34" noic="true" style="padding-top: 0.677em;">4</mjx-c><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.677em;">3</mjx-c><mjx-c class="mjx-c30" style="padding-top: 0.677em;">0</mjx-c></mjx-mn></mjx-mrow></mjx-math></mjx-container>, and the probability is proportional to <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="StartSet mu Subscript g Superscript a Baseline gamma Subscript g Superscript a Baseline EndSet Subscript g equals 1 Superscript upper M" tabindex="0" ctxtmenu_counter="138" style="font-size: 130%;"><mjx-math data-semantic-structure="(21 (20 (14 0 (12 (5 (4 1 2) 3) 11 (10 (9 6 7) 8)) 13) (18 15 16 17)) 19)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-msubsup data-semantic-type="subsup" data-semantic-role="set singleton" data-semantic-id="21" data-semantic-children="14,18,19" data-semantic-collapsed="(21 (20 14 18) 19)" data-semantic-owns="14 18 19" data-semantic-speech="StartSet mu Subscript g Superscript a Baseline gamma Subscript g Superscript a Baseline EndSet Subscript g equals 1 Superscript upper M"><mjx-mrow data-semantic-parent="21"><mjx-mrow data-semantic-type="fenced" data-semantic-role="set singleton" data-semantic-id="14" data-semantic-children="12" data-semantic-content="0,13" data-semantic-parent="20" data-semantic-owns="0 12 13"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7B MM-LO">{</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="12" data-semantic-children="5,10" data-semantic-content="11" data-semantic-parent="14" data-semantic-owns="5 11 10"><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="5" data-semantic-children="1,2,3" data-semantic-parent="12" data-semantic-collapsed="(5 (4 1 2) 3)" data-semantic-owns="1 2 3"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="5"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: 0px;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="11" data-semantic-parent="12" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-msubsup data-semantic-type="subsup" data-semantic-role="greekletter" data-semantic-id="10" data-semantic-children="6,7,8" data-semantic-parent="12" data-semantic-collapsed="(10 (9 6 7) 8)" data-semantic-owns="6 7 8"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="6" data-semantic-parent="10"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow><mjx-script style="vertical-align: -0.247em; margin-left: -0.025em;"><mjx-mrow size="s" style="margin-left: 0.108em;"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="10"><mjx-c class="mjx-c1D44E">𝑎</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.29em;"></mjx-spacer><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="7" data-semantic-parent="10"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="13" data-semantic-parent="14" data-semantic-operator="fenced"><mjx-c class="mjx-c7D MM-LO">}</mjx-c></mjx-mo></mjx-mrow></mjx-mrow><mjx-script style="vertical-align: -0.427em; margin-left: 0px;"><mjx-mrow size="s"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="19" data-semantic-parent="21"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow><mjx-spacer style="margin-top: 0.533em;"></mjx-spacer><mjx-mrow size="s" data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="18" data-semantic-children="15,17" data-semantic-content="16" data-semantic-parent="21" data-semantic-owns="15 16 17"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="15" data-semantic-parent="18"><mjx-c class="mjx-c1D454">𝑔</mjx-c></mjx-mi><mjx-mo data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="16" data-semantic-parent="18" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="17" data-semantic-parent="18"><mjx-c class="mjx-c31">1</mjx-c></mjx-mn></mjx-mrow></mjx-script></mjx-msubsup></mjx-mrow></mjx-math></mjx-container>. The mutation rates are taken from the real data.</div></div></div><div id="o0070" role="listitem"><div class="label">5)</div><div class="content"><div id="p0485" role="paragraph">Apply TADA<sup>+</sup> with Bayes Factors having a lower limit (floor) of 1, and calculate the empirical FDR.</div></div></div><div id="o0075" role="listitem"><div class="label">6)</div><div class="content"><div id="p0490" role="paragraph">Repeat steps 1-5 one hundred times.</div></div></div></div></div><div id="p0495" role="paragraph">Figure&nbsp;S2L (original BF) and Figure&nbsp;S2M (floor BF) show that the TADA<sup>+</sup> model controls FDR. Applying the floor principle increases the FDR by a modest amount. In practice we found that there was considerable heterogeneity across genes and this adjustment was necessary.</div></section><section id="sec-4-6-5-2"><h5>TADA<sup>O</sup> versus TADA<sup>+</sup> analyses</h5><div id="p0500" role="paragraph">We explore the performance of TADA<sup>O</sup> and TADA<sup>+</sup> with four analyses:<div id="olist0025" role="list"><div id="o0080" role="listitem"><div class="label">A)</div><div class="content"><div id="p0505" role="paragraph">TADA<sup>O</sup> applied to the previous ASC cohort (ASC2015), <i>de novo</i> variants only.</div></div></div><div id="o0085" role="listitem"><div class="label">B)</div><div class="content"><div id="p0510" role="paragraph">TADA<sup>O</sup> applied to the new ASC cohort (ASC2018), <i>de novo</i> variants only.</div></div></div><div id="o0090" role="listitem"><div class="label">C)</div><div class="content"><div id="p0515" role="paragraph">TADA<sup>+</sup> applied to ASC2018, <i>de novo</i> variants only.</div></div></div><div id="o0095" role="listitem"><div class="label">D)</div><div class="content"><div id="p0520" role="paragraph">TADA<sup>+</sup> applied to ASC2018, <i>de novo</i> and case-control variants.</div></div></div></div></div><div id="p0525" role="paragraph">By moving through the four analyses, we change one variable at a time and analyze the consequences. From A to B, we evaluate the impact of the new <i>de novo</i> data introduced as part of ASC2018. From B to C, we compare the improvements in the model by contrasting TADA<sup>O</sup> and TADA<sup>+</sup>. From C to D, we assess the impact of adding in the case-control data.</div><div id="p0530" role="paragraph">With additional data and a more powerful TADA model, we obtain substantial new discoveries. We identify 31 genes in A, 65 genes in B, 85 genes in C and 102 genes in D. Contrasting genes discovered in A with the new ones discovered in B, we compare the mutation rates and find that the newly discovered genes are 8.9% smaller for PTV (Wilcoxon test p value&nbsp;= 0.35) and 5.8% smaller for Mis3 (Wilcoxon test p value&nbsp;= 0.47) on average; likewise we find that the average fraction of Mis3 over all <i>de novo</i> mutations (Mis3&nbsp;+ PTV) is 0.04 larger (Wilcoxon test p value&nbsp;= 0.63). The similarity in mutation rate and types of mutations found in the newly discovered genes suggests that they are primarily due to the larger sample size. The additional genes discovered in C are due to the more powerful TADA model.</div><div id="p0535" role="paragraph">Evaluating the q-values from these analyses—for the genes with a q-value less than 0.1 in at least one analysis—q-values typically decrease in sequence from analysis A to B to C to D (<a id="crosref1465" href="#figs2">Figure&nbsp;S2</a>N, showing B as “TADA<sup>O</sup>, dn,” C as “TADA<sup>+</sup>, dn,” and D as “TADA<sup>+</sup>, dncc”), with the q-value of analysis D being the smallest. Of the genes with a q-value greater than 0.1 in D but less than 0.1 in at least one other analysis, most are downgraded in analysis D because of refinements in the new TADA model (with the genes or variants having, for instance, low pLI score or low MPC score, particularly MPC less than 1 or missing and thus not categorized as MisA or MisB). A Manhattan plot of the 102 genes identified in D is shown in <a id="crosref1470" href="#fig2">Figure&nbsp;2</a>.</div></section></section><section id="sec-4-6-6"><h4>ASD<sub>P</sub>, ASD<sub>NDD</sub> and DDD gene heterogeneity analyses</h4><div id="p0540" role="paragraph">Heterogeneity analyses were performed using a chi-square approximation test and a Fisher’s exact test (FET) based on <i>de novo</i> counts in ASD and neurodevelopmental delay (NDD) for different subsets of genes. Variants were counted if they were disruptive, i.e., a protein-truncating variant or a missense variant with MPC score ≥ 1. Variant counts in ASD (“dn ASD”) were based on this study, while variant counts in NDD (“dn NDD”) were based on 5,264 trios ascertained for severe NDD (<a id="crosref1475" href="#mmc4">Table S4</a>).</div><div id="p0545" role="paragraph">We started out with the list of 102 TADA ASD genes, which we classified as either ASD<sub>P</sub> or ASD<sub>NDD</sub> (<a id="crosref1480" href="#fig4">Figure&nbsp;4</a>). We also considered the 49 ASD<sub>NDD</sub> genes and the 102 TADA genes in combination with an “NDD set” of genes containing disruptive variants in the 5,264 NDD trios (avoiding double-counting of genes disrupted in both ASD and NDD). A table of these genes is provided below.<div class="figure-wrap"><figure id="undtbl2" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th>Gene set</th><th># genes</th><th>dn ASD</th><th>dn NDD</th></tr></thead><tbody><tr><td>TADA 102</td><td>102</td><td>391</td><td>514</td></tr><tr><td>ASD<sub>P</sub></td><td>53</td><td>176</td><td>43</td></tr><tr><td>ASD<sub>NDD</sub></td><td>49</td><td>215</td><td>471</td></tr><tr><td>ASD<sub>NDD</sub> (including NDD set)</td><td>90</td><td>258</td><td>825</td></tr><tr><td>All (including NDD set)</td><td>143</td><td>434</td><td>868</td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl2&amp;pii=S0092-8674%2819%2931398-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0550" role="paragraph">For the 102 TADA ASD genes, we can evaluate whether the count of disruptive <i>de novo</i> events is homogeneous for the ASC (“dn&nbsp;ASD”) versus NDD (“dn NDD”) samples: the answer is no (p&nbsp;= 5&nbsp;× 10<sup>−12</sup>), the data are highly heterogeneous. If we consider the ASD<sub>P</sub> and ASD<sub>NDD</sub> genes by themselves, however, neither subsample shows significant heterogeneity. We can also include the NDD gene set, bringing the overall total to 143 (eliminates genes on chromosome X and <i>FOXG1</i>; <i>NUP155</i> was eliminated in&nbsp;the previous analysis, due to a lack of <i>de novo</i> mutations). With this expanded list, heterogeneity naturally increases (p&nbsp;= 4&nbsp;× 10<sup>−26</sup>). If we reasonably put the newly added genes into the ASD<sub>NDD</sub> list, we find that there is now heterogeneity in this set (p&nbsp;= 0.001), and it is driven by the lack of mutations in ASD subjects for the newly added NDD genes (see below).<div class="figure-wrap"><figure id="undtbl3" class="table" data-figureviewer-ignore="true"><div class="table-wrap"><table data-figureviewer-ignore="true"><thead><tr class="rowsep"><th rowspan="2">Gene set</th><th rowspan="2"># genes</th><th colspan="2"><span>P value</span></th></tr><tr class="rowsep"><th>Chi-square approximation</th><th>FET<sup>1</sup></th></tr></thead><tbody><tr><td>TADA 102</td><td>102</td><td>5.1&nbsp;× 10<sup>−12</sup></td><td>5.5&nbsp;× 10<sup>−9</sup></td></tr><tr><td>ASD<sub>P</sub></td><td>53</td><td>0.971</td><td>0.916</td></tr><tr><td>ASD<sub>NDD</sub></td><td>49</td><td>0.389</td><td>0.259</td></tr><tr><td>ASD<sub>NDD</sub> (including NDD set)</td><td>90</td><td>0.001</td><td>0.004</td></tr><tr><td>All genes (including NDD set)</td><td>143</td><td>4.3&nbsp;× 10<sup>−26</sup></td><td>3.2&nbsp;× 10<sup>−24</sup></td></tr></tbody></table></div><figcaption><div class="core-table-tools"><ul><li><a class="core-table-show-action" href="/action/showFullTableHTML?isHtml=true&amp;tableId=undtbl3&amp;pii=S0092-8674%2819%2931398-4" target="_blank">Open table in a new tab</a></li></ul></div></figcaption></figure></div></div><div id="p0555" role="paragraph">Because of the total counts involved in the analyses, the FET was performed in independent (random) subsets and then the statistics were combined, and results represent the average over 100 repetitions. For the chi-square test, the genes were required to have at least one <i>de novo</i> event in either ASD or DDD. This removed 1 ASD<sub>P</sub> gene from the analysis.</div></section><section id="sec-4-6-7"><h4>Genes in Recurrent Genomic Disorders (GD)</h4><section id="sec-4-6-7-1"><h5>Curation of reported GD loci</h5><div id="p0560" role="paragraph">We constructed a list of genomic loci previously reported to be associated with ASD- or NDD-related phenotypes due to rare copy number variants (CNVs). We first collated coordinates of pathogenic “genomic disorder” (GD) regions as reported by nine previous studies (<span class="dropBlock reference-citations"><a id="body-ref-sref112" href="#" role="doc-biblioref" data-xml-rid="bib92 bib17 bib103 bib94 bib25 bib16 bib75 bib112 bib79" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref112" aria-controls="sref112">Sanders et&nbsp;al., 2015; Cooper et&nbsp;al., 2011; Wapner et&nbsp;al., 2012; Schaefer et&nbsp;al., 2013; Dittwald et&nbsp;al., 2013; Coe et&nbsp;al., 2014; Pinto et&nbsp;al., 2010; Wright et&nbsp;al., 2015; Rehm et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref112"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">15.</div><div>Coe, B.P. ∙ Witherspoon, K. ∙ Rosenfeld, J.A. ...</div><div><strong>Refining analyses of copy number variation identifies specific genes associated with developmental delay</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:1063-1071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3092" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84922012902" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3092&amp;pmid=25217958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">16.</div><div>Cooper, G.M. ∙ Coe, B.P. ∙ Girirajan, S. ...</div><div><strong>A copy number variation morbidity map of developmental delay</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:838-846</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80052260252" target="_blank" aria-label="Open Scopus in new tab">Scopus (755)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21841781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.909&amp;pmid=21841781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">23.</div><div>Dittwald, P. ∙ Gambin, T. ∙ Szafranski, P. ...</div><div><strong>NAHR-mediated copy-number variants in a clinical population: mechanistic insights into both genomic disorders and Mendelizing traits</strong></div><div><em>Genome Res.</em> 2013; <strong>23</strong>:1395-1409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.152454.112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84883679009" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23657883/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.152454.112&amp;pmid=23657883" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">73.</div><div>Pinto, D. ∙ Pagnamenta, A.T. ∙ Klei, L. ...</div><div><strong>Functional impact of global rare copy number variation in autism spectrum disorders</strong></div><div><em>Nature.</em> 2010; <strong>466</strong>:368-372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77954657070" target="_blank" aria-label="Open Scopus in new tab">Scopus (1285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20531469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09146&amp;pmid=20531469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">76.</div><div>Rehm, H.L. ∙ Berg, J.S. ∙ Brooks, L.D. ..., ClinGen</div><div><strong>ClinGen--the Clinical Genome Resource</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2015; <strong>372</strong>:2235-2242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMsr1406261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84930526399" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26014595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMsr1406261&amp;pmid=26014595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">90.</div><div>Schaefer, G.B. ∙ Mendelsohn, N.J., Professional Practice and Guidelines Committee</div><div><strong>Clinical genetics evaluation in identifying the etiology of autism spectrum disorders: 2013 guideline revisions</strong></div><div><em>Genet. Med.</em> 2013; <strong>15</strong>:399-407</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1038%2Fgim.2013.32&amp;cf=fulltext&amp;site=gim-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1038%2Fgim.2013.32&amp;cf=pdf&amp;site=gim-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84877264570" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23519317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fgim.2013.32&amp;pmid=23519317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">99.</div><div>Wapner, R.J. ∙ Martin, C.L. ∙ Levy, B. ...</div><div><strong>Chromosomal microarray versus karyotyping for prenatal diagnosis</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2012; <strong>367</strong>:2175-2184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1203382" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84870549609" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23215555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1203382&amp;pmid=23215555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">108.</div><div>Wright, C.F. ∙ Fitzgerald, T.W. ∙ Jones, W.D. ..., DDD study</div><div><strong>Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data</strong></div><div><em>Lancet.</em> 2015; <strong>385</strong>:1305-1314</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2FS0140-6736%2814%2961705-0&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2FS0140-6736%2814%2961705-0&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84926522440" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25529582/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2814%2961705-0&amp;pmid=25529582" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and converted all coordinates to human reference genome build GRCh37/hg19 with the UCSC liftOver tool, as necessary. We next clustered the coordinates of all overlapping CNV regions using svtk bedcluster and a minimum 50% reciprocal overlap between segments, retaining the median clustered coordinates of all CNV regions appearing in at least two of the nine studies considered. After clustering, we excluded any CNV segments &gt; 5Mb in size and all segments on sex chromosomes. Finally, we annotated each CNV segment passing all filters with all overlapping genes drawn from the list of 17,484 autosomal protein-coding genes considered during TADA analyses.</div></section><section id="sec-4-6-7-2"><h5>Assessment of overlap between ASD-associated genes and GD loci</h5><div id="p0565" role="paragraph">We designed three permutation-based approaches to benchmark null expectations for the overlap of ASD-associated genes and GD loci. All approaches involved randomly drawing new sets of collinear genes for each GD locus from the list of all 17,484 autosomal protein-coding genes considered in TADA analyses, but differed in how these new genes were selected. These sampling approaches are summarized as follows:<div id="olist0030" role="list"><div id="o0100" role="listitem"><div class="label">1)</div><div class="content"><div id="p0570" role="paragraph">Matched on number of genes: a new collinear list of genes was drawn for each GD locus, where the number of genes was matched to the number of genes in the original GD locus.</div></div></div><div id="o0105" role="listitem"><div class="label">2)</div><div class="content"><div id="p0575" role="paragraph">Matched on PTV mutation rates: a new collinear list of genes was drawn for each GD, where the number of genes was determined such that the sum of their estimated PTV mutation rates was at least as large as the sum of the estimated PTV mutation rates of the original list of genes in that GD locus.</div></div></div><div id="o0110" role="listitem"><div class="label">3)</div><div class="content"><div id="p0580" role="paragraph">Matched on brain expression, PTV mutation rates, and number of genes: prior to permutation, all genes were assigned a PTV mutation rate quintile and a brain expression quintile determined by the median brain expression value for that gene across all samples and all brain regions present in GTEx release v7 calculated after excluding genes with non-zero median brain expression. During permutation, a new collinear list of genes was drawn for each GD such that the number of genes matched the original GD locus, with the additional requirements that the distribution of these genes across brain expression quintiles and PTV mutation rate quintiles were also preserved.</div></div></div></div></div><div id="p0585" role="paragraph">For each permutation, we performed one of the three above approaches for all 51 GD loci to obtain a new set of sampled genes, and we then counted the number of newly sampled genes that matched the TADA thresholds for ASD association in this study. We performed 1,000,000 permutations for each approach and computed p values based on the fraction of all permutations where the number of GD loci with at least one randomly sampled ASD-associated gene matched or exceed the empirical observation in the original data. Fold-changes (FCs) were determined as the observed number of GD loci with at least one ASD-associated gene divided by the mean number of GDs with at least one ASD-associated gene across all 1,000,000 permutations.</div><div id="p0590" role="paragraph">Finally, we titrated additional parameters to examine the variability of results from this permutation approach. For each of the three gene-sampling schemes above, we performed a separate 1,000,000 independent permutations for each combination of two additional factors, as follows:<div id="olist0035" role="list"><div id="o0115" role="listitem"><div class="label">1)</div><div class="content"><div id="p0595" role="paragraph">ASD-associated gene list: we considered two different significance levels for ASD-associated genes: 102 genes identified with TADA<sup>+</sup> (FDR ≤ 0.1) and 26 genes identified as reaching Bonferroni-corrected significance (Table S2).</div></div></div><div id="o0120" role="listitem"><div class="label">2)</div><div class="content"><div id="p0600" role="paragraph">Chromosome sampling weights: for each GD locus in each permutation, an autosomal chromosome was selected based on one of two weighting schemes prior to randomly sampling a new set of collinear genes. These weights were either (1) determined by the fraction of all autosomal genes located on each chromosome, or (2) determined by the fraction of GD loci located on each chromosome.</div></div></div></div></div><div id="p0605" role="paragraph">All results were consistent across gene sampling strategies and the additional parameters had limited influence on our individual results or overall conclusions (<a id="crosref1490" href="#fig3">Figure&nbsp;3</a>).</div></section></section><section id="sec-4-6-8"><h4>Enrichment of Common Variants in the Detected Genes</h4><div id="p0610" role="paragraph">To investigate whether the 102 ASD-associated genes were enriched for common variants associated with ASD and genetically correlated traits, we ran competitive gene-set enrichment analyses on the set of 102 genes using MAGMA (<span class="dropBlock reference-citations"><a id="body-ref-sref20-2" href="#" role="doc-biblioref" data-xml-rid="bib20" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref20-2" aria-controls="sref20-2">de Leeuw et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref20-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">18.</div><div>de Leeuw, C.A. ∙ Mooij, J.M. ∙ Heskes, T. ...</div><div><strong>MAGMA: generalized gene-set analysis of GWAS data</strong></div><div><em>PLoS Comput. Biol.</em> 2015; <strong>11</strong>:e1004219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84929486000" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25885710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004219&amp;pmid=25885710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) with brain-expressed genes from BrainSpan (see section below on developmental expression data) as background. We used summary statistics from the latest GWAS of ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref33-4" href="#" role="doc-biblioref" data-xml-rid="bib33" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref33-4" aria-controls="sref33-4">Grove et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref33-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">31.</div><div>Grove, J. ∙ Ripke, S. ∙ Als, T.D. ..., Autism Spectrum Disorder Working Group of the Psychiatric Genomics Consortium, BUPGEN, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, 23andMe Research Team</div><div><strong>Identification of common genetic risk variants for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:431-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0344-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062110842" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0344-8&amp;pmid=30804558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), ADHD (<span class="dropBlock reference-citations"><a id="body-ref-sref23-2" href="#" role="doc-biblioref" data-xml-rid="bib23" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref23-2" aria-controls="sref23-2">Demontis et&nbsp;al., 2019</a><div class="dropBlock__holder" data-db-target-of="sref23-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">21.</div><div>Demontis, D. ∙ Walters, R.K. ∙ Martin, J. ..., ADHD Working Group of the Psychiatric Genomics Consortium (PGC), Early Lifecourse &amp; Genetic Epidemiology (EAGLE) Consortium, 23andMe Research Team</div><div><strong>Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:63-75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0269-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85057436335" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0269-7&amp;pmid=30478444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), major depression without the 23andMe contribution (<span class="dropBlock reference-citations"><a id="body-ref-sref111-2" href="#" role="doc-biblioref" data-xml-rid="bib111" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref111-2" aria-controls="sref111-2">Wray et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref111-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">107.</div><div>Wray, N.R. ∙ Ripke, S. ∙ Mattheisen, M. ..., eQTLGen, 23andMe, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium</div><div><strong>Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:668-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0090-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85046022254" target="_blank" aria-label="Open Scopus in new tab">Scopus (394)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29700475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0090-3&amp;pmid=29700475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), schizophrenia (<span class="dropBlock reference-citations"><a id="body-ref-sref95-2" href="#" role="doc-biblioref" data-xml-rid="bib95" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref95-2" aria-controls="sref95-2">Schizophrenia Working Group of the Psychiatric Genomics Consortium, 2014</a><div class="dropBlock__holder" data-db-target-of="sref95-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">91.</div><div>Schizophrenia Working Group of the Psychiatric Genomics Consortium</div><div><strong>Biological insights from 108 schizophrenia-associated genetic loci</strong></div><div><em>Nature.</em> 2014; <strong>511</strong>:421-427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13595" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84904804929" target="_blank" aria-label="Open Scopus in new tab">Scopus (3226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13595&amp;pmid=25056061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), and educational attainment (<span class="dropBlock reference-citations"><a id="body-ref-sref50-2" href="#" role="doc-biblioref" data-xml-rid="bib50" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref50-2" aria-controls="sref50-2">Lee et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref50-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">48.</div><div>Lee, J.J. ∙ Wedow, R. ∙ Okbay, A. ..., 23andMe Research Team, COGENT (Cognitive Genomics Consortium), Social Science Genetic Association Consortium</div><div><strong>Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1112-1121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0147-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85050543338" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30038396/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0147-3&amp;pmid=30038396" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), as well as a GWAS of height (<span class="dropBlock reference-citations"><a id="body-ref-sref110" href="#" role="doc-biblioref" data-xml-rid="bib110" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref110" aria-controls="sref110">Wood et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref110"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">106.</div><div>Wood, A.R. ∙ Esko, T. ∙ Yang, J. ..., Electronic Medical Records and Genomics (eMEMERGEGE) Consortium, MIGen Consortium, PAGEGE Consortium, LifeLines Cohort Study</div><div><strong>Defining the role of common variation in the genomic and biological architecture of adult human height</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:1173-1186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84908890496" target="_blank" aria-label="Open Scopus in new tab">Scopus (833)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25282103/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3097&amp;pmid=25282103" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) as a control. In addition, to illustrate the effect of statistical power in the GWAS, we ran enrichment analyses also for historical GWAS for these phenotypes (<span class="dropBlock reference-citations"><a id="body-ref-sref84-2" href="#" role="doc-biblioref" data-xml-rid="bib68 bib82 bib83 bib84 bib81 bib72" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref84-2" aria-controls="sref84-2">Neale et&nbsp;al., 2010; Ripke et&nbsp;al., 2011, 2013a, 2013b; Rietveld et&nbsp;al., 2013; Okbay et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref84-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">66.</div><div>Neale, B.M. ∙ Medland, S.E. ∙ Ripke, S. ..., Psychiatric GWAS Consortium: ADHD Subgroup</div><div><strong>Meta-analysis of genome-wide association studies of attention-deficit/hyperactivity disorder</strong></div><div><em>J.&nbsp;Am. Acad. Child Adolesc. Psychiatry.</em> 2010; <strong>49</strong>:884-897</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.jaac.2010.06.008&amp;cf=fulltext&amp;site=jaac-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.jaac.2010.06.008&amp;cf=pdf&amp;site=jaac-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77956178166" target="_blank" aria-label="Open Scopus in new tab">Scopus (291)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20732625/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaac.2010.06.008&amp;pmid=20732625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">70.</div><div>Okbay, A. ∙ Beauchamp, J.P. ∙ Fontana, M.A. ..., LifeLines Cohort Study</div><div><strong>Genome-wide association study identifies 74 loci associated with educational attainment</strong></div><div><em>Nature.</em> 2016; <strong>533</strong>:539-542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27225129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17671&amp;pmid=27225129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">78.</div><div>Rietveld, C.A. ∙ Medland, S.E. ∙ Derringer, J. ..., LifeLines Cohort Study</div><div><strong>GWAS of 126,559 individuals identifies genetic variants associated with educational attainment</strong></div><div><em>Science.</em> 2013; <strong>340</strong>:1467-1471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1235488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84879271608" target="_blank" aria-label="Open Scopus in new tab">Scopus (379)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23722424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1235488&amp;pmid=23722424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">79.</div><div>Ripke, S. ∙ Sanders, A.R. ∙ Kendler, K.S. ..., Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium</div><div><strong>Genome-wide association study identifies five new schizophrenia loci</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:969-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.940" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80053384370" target="_blank" aria-label="Open Scopus in new tab">Scopus (1244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21926974/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.940&amp;pmid=21926974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">80.</div><div>Ripke, S. ∙ O’Dushlaine, C. ∙ Chambert, K. ..., Multicenter Genetic Studies of Schizophrenia Consortium, Psychosis Endophenotypes International Consortium, Wellcome Trust Case Control Consortium 2</div><div><strong>Genome-wide association analysis identifies 13 new risk loci for schizophrenia</strong></div><div><em>Nat. Genet.</em> 2013; <strong>45</strong>:1150-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84885020424" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23974872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2742&amp;pmid=23974872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">81.</div><div>Ripke, S. ∙ Wray, N.R. ∙ Lewis, C.M. ..., Major Depressive Disorder Working Group of the Psychiatric GWAS Consortium</div><div><strong>A mega-analysis of genome-wide association studies for major depressive disorder</strong></div><div><em>Mol. Psychiatry.</em> 2013; <strong>18</strong>:497-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2012.21" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84879892380" target="_blank" aria-label="Open Scopus in new tab">Scopus (635)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22472876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2012.21&amp;pmid=22472876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). From the summary statistics, gene-based p values were estimated using test statistics defined as the sum of -log<sub>10</sub>(SNP p values) for SNPs located within the transcribed region plus a padding of 10kb flanking regions on either side. The gene-set enrichment was conducted by regressing gene-based Z-scores on a dichotomous gene-set indicator and covariates, which included gene size, gene density, sample size, the reciprocal of the minimal allele count, and the logarithm of these variables. We applied the default settings in MAGMA (<a id="crosref1530" href="#fig3">Figure&nbsp;3</a>).</div></section><section id="sec-4-6-9"><h4>Defining Gene Groups</h4><div id="p0615" role="paragraph">Past analyses have identified two major groups of ASD-associated genes: those involved in gene expression regulation (GER) and those involved in neuronal communication (NC) (<span class="dropBlock reference-citations"><a id="body-ref-sref92-7" href="#" role="doc-biblioref" data-xml-rid="bib21 bib92" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref92-7" aria-controls="sref92-7">De Rubeis et&nbsp;al., 2014; Sanders et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref92-7"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">88.</div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A simple gene ontology analysis with our list of 102 ASD genes replicates this finding, identifying 16 genes in category GO:0006357 “regulation of transcription from RNA&nbsp;polymerase II promoter” (5.7-fold enrichment, FDR&nbsp;= 6.2x10<sup>−6</sup>) and 9 genes in category GO:0007268: “synaptic transmission” (5.0-fold enrichment, FDR&nbsp;= 3.8x10<sup>−3</sup>). To assign genes to the GER and NC categories, we used a combination of gene ontology, gene descriptions, and primary research.</div><div id="p0620" role="paragraph">Fifty-eight genes were assigned to the GER group based on one of:<div id="olist0040" role="list"><div id="o0125" role="listitem"><div class="label">1)</div><div class="content"><div id="p0625" role="paragraph">Clear description of a role as a chromatin modifier, transcription factor, or DNA/RNA binding protein on RefSeq (<span class="dropBlock reference-citations"><a id="body-ref-sref71-1" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-1" aria-controls="sref71-1">O’Leary et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref71-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>O’Leary, N.A. ∙ Wright, M.W. ∙ Brister, J.R. ...</div><div><strong>Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:D733-D745</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1189&amp;pmid=26553804" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div><div id="o0130" role="listitem"><div class="label">2)</div><div class="content"><div id="p0630" role="paragraph">Evidence of role as a chromatin modifier, transcription factor, or DNA/RNA binding protein in primary research identified on PubMed (<a id="intref0130" href="https://www.ncbi.nlm.nih.gov/pubmed" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed</a>).</div></div></div><div id="o0135" role="listitem"><div class="label">3)</div><div class="content"><div id="p0635" role="paragraph">Located in the nucleus (GO:0005634) and annotated with at least two of the following gene ontology groups or their children:</div></div></div><div id="u0030" role="listitem"><div class="label">•</div><div class="content"><div id="p0640" role="paragraph">GO:0000122:negative regulation of transcription by RNA polymerase II</div></div></div><div id="u0035" role="listitem"><div class="label">•</div><div class="content"><div id="p0645" role="paragraph">GO:0000785:chromatin</div></div></div><div id="u0040" role="listitem"><div class="label">•</div><div class="content"><div id="p0650" role="paragraph">GO:0000981:RNA polymerase II transcription factor activity, sequence-specific DNA binding</div></div></div><div id="u0045" role="listitem"><div class="label">•</div><div class="content"><div id="p0655" role="paragraph">GO:0000988:transcription factor activity, protein binding</div></div></div><div id="u0050" role="listitem"><div class="label">•</div><div class="content"><div id="p0660" role="paragraph">GO:0003682:chromatin binding</div></div></div><div id="u0055" role="listitem"><div class="label">•</div><div class="content"><div id="p0665" role="paragraph">GO:0003700:DNA-binding transcription factor activity</div></div></div><div id="u0060" role="listitem"><div class="label">•</div><div class="content"><div id="p0670" role="paragraph">GO:0006325:chromatin organization</div></div></div><div id="u0065" role="listitem"><div class="label">•</div><div class="content"><div id="p0675" role="paragraph">GO:0010468:regulation of gene expression</div></div></div><div id="u0070" role="listitem"><div class="label">•</div><div class="content"><div id="p0680" role="paragraph">GO:0045944:positive regulation of transcription by RNA polymerase II</div></div></div></div></div><div id="p0685" role="paragraph">Twenty-four genes were assigned to the GER group based on one of:<div id="olist0045" role="list"><div id="o0140" role="listitem"><div class="label">1)</div><div class="content"><div id="p0690" role="paragraph">Clear description of a role in the synapse or regulating membrane potential on RefSeq (<span class="dropBlock reference-citations"><a id="body-ref-sref71-2" href="#" role="doc-biblioref" data-xml-rid="bib71" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref71-2" aria-controls="sref71-2">O’Leary et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref71-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">69.</div><div>O’Leary, N.A. ∙ Wright, M.W. ∙ Brister, J.R. ...</div><div><strong>Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:D733-D745</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1189&amp;pmid=26553804" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></div></div><div id="o0145" role="listitem"><div class="label">2)</div><div class="content"><div id="p0695" role="paragraph">Evidence of a role in the synapse or regulating membrane potential in primary research identified on PubMed (<a id="intref0135" href="https://www.ncbi.nlm.nih.gov/pubmed" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed</a>).</div></div></div><div id="o0150" role="listitem"><div class="label">3)</div><div class="content"><div id="p0700" role="paragraph">Located in the cytoplasm (GO:0005737) and annotated with at least two of the following gene ontology groups or their children:</div></div></div><div id="u0075" role="listitem"><div class="label">•</div><div class="content"><div id="p0705" role="paragraph">GO:0007267:cell-cell signaling</div></div></div><div id="u0080" role="listitem"><div class="label">•</div><div class="content"><div id="p0710" role="paragraph">GO:0042391:regulation of membrane potential</div></div></div><div id="u0085" role="listitem"><div class="label">•</div><div class="content"><div id="p0715" role="paragraph">GO:0045202:synapse</div></div></div></div></div><div id="p0720" role="paragraph">Of the remaining 20 genes, 9 are annotated with GO:0007010:cytoskeleton organization or child terms of this ontology term; these genes are classified as “Cytoskeleton genes.” The remaining 11 genes are described as “Other.” See <a id="crosref1545" href="#mmc4">Table S4</a> and <a id="crosref1550" href="#fig4">Figure&nbsp;4</a>.</div></section><section id="sec-4-6-10"><h4>Comorbid Phenotypes</h4><div id="p0725" role="paragraph">Full-scale IQ scores were measured using several tests including, but not limited to, the Differential Ability Scales, Second Edition (<span class="dropBlock reference-citations"><a id="body-ref-sref27" href="#" role="doc-biblioref" data-xml-rid="bib27" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref27" aria-controls="sref27">Elliott, 2007</a><div class="dropBlock__holder" data-db-target-of="sref27"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">25.</div><div>Elliott, C.D.</div><div><strong>Differential Ability Scales</strong></div><div>Harcourt Assessment, 2007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.D.ElliottDifferential+Ability+ScalesSecond+Edition2007Harcourt+Assessment" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); the Mullen Scales of Early Learning (<span class="dropBlock reference-citations"><a id="body-ref-sref67" href="#" role="doc-biblioref" data-xml-rid="bib67" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref67" aria-controls="sref67">Mullen, 1995</a><div class="dropBlock__holder" data-db-target-of="sref67"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">65.</div><div>Mullen, E.M.</div><div><strong>Mullen Scales of Early Learning Manual</strong></div><div>American Guidance Service, 1995</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.M.MullenMullen+Scales+of+Early+Learning+Manual1995American+Guidance+Service" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); the Wechsler Intelligence Scale for Children (<span class="dropBlock reference-citations"><a id="body-ref-sref104" href="#" role="doc-biblioref" data-xml-rid="bib104" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref104" aria-controls="sref104">Wechsler, 1992</a><div class="dropBlock__holder" data-db-target-of="sref104"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">100.</div><div>Wechsler, D.</div><div><strong>WISC III (Wechsler Intelligence Scale for Children)</strong></div><div>The Psychological Corporation, 1992</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.WechslerWISC+III+%28Wechsler+Intelligence+Scale+for+Children%291992The+Psychological+Corporation" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); and the Wechsler Abbreviated Scale of Intelligence (<span class="dropBlock reference-citations"><a id="body-ref-sref105" href="#" role="doc-biblioref" data-xml-rid="bib105" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref105" aria-controls="sref105">Wechsler, 1999</a><div class="dropBlock__holder" data-db-target-of="sref105"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">101.</div><div>Wechsler, D.</div><div><strong>Wechsler Abbreviated Scale of Intelligence</strong></div><div>The Psychological Corporation, 1999</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.WechslerWechsler+Abbreviated+Scale+of+Intelligence1999The+Psychological+Corporation" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The full-scale IQ estimates were taken from the full-scale deviation IQ variable when available and full-scale ratio IQ when it was not (<span class="dropBlock reference-citations"><a id="body-ref-sref10" href="#" role="doc-biblioref" data-xml-rid="bib10" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref10" aria-controls="sref10">Chaste et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref10"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">10.</div><div>Chaste, P. ∙ Klei, L. ∙ Sanders, S.J. ...</div><div><strong>A genome-wide association study of autism using the Simons Simplex Collection: Does reducing phenotypic heterogeneity in autism increase genetic homogeneity?</strong></div><div><em>Biol. Psychiatry.</em> 2015; <strong>77</strong>:775-784</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2014.09.017&amp;cf=fulltext&amp;site=bps-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2014.09.017&amp;cf=pdf&amp;site=bps-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84926358552" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25534755/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biopsych.2014.09.017&amp;pmid=25534755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Full-scale IQ is normally distributed with a mean of 100 and a standard deviation of 15, by definition. We defined intellectual disability to be if a subject met one of the following conditions: a full-scale IQ (FSIQ)&nbsp;&lt; 70 (i.e., two standard deviations below the mean), if the proband was administered but could not complete an IQ test, indicated by the subject having a date for their IQ test but no IQ score, or if the subject had a human phenotype ontology (HPO) term or International Classification of Diseases (ICD) code indicating intellectual disability or mental retardation. Age of walking unaided (in months) was taken from question 5A from the Autism Diagnostic Interview (ADI) (<span class="dropBlock reference-citations"><a id="body-ref-sref60" href="#" role="doc-biblioref" data-xml-rid="bib60" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref60" aria-controls="sref60">Lord et&nbsp;al., 1994</a><div class="dropBlock__holder" data-db-target-of="sref60"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">58.</div><div>Lord, C. ∙ Rutter, M. ∙ Le Couteur, A.</div><div><strong>Autism Diagnostic Interview-Revised: a revised version of a diagnostic interview for caregivers of individuals with possible pervasive developmental disorders</strong></div><div><em>J.&nbsp;Autism Dev. Disord.</em> 1994; <strong>24</strong>:659-685</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF02172145" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-0027997172" target="_blank" aria-label="Open Scopus in new tab">Scopus (5772)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7814313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF02172145&amp;pmid=7814313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We divided individuals into three possible categories for seizure status: yes, no, and unknown. A subject was put into the yes bin if he or she had a diagnosis of seizures or epilepsy, or a value of 2 on question 85 from the ADI (indicating a diagnosis of epilepsy). A subject was put into the no bin if no seizure/epilepsy diagnosis was indicated or if ADI question 85 had a value of 0. All remaining subjects were put into the unknown bin (<a id="crosref1585" href="#fig4">Figure&nbsp;4</a>).</div><section id="sec-4-6-10-1"><h5>Burden of mutations in ASD as a function of IQ and additional comorbid phenotypes</h5><div id="p0730" role="paragraph"><i>Burden of mutations over all genes.</i> We used full-scale IQ to separate subjects into groups. Of the 5,298 probands with any <i>de&nbsp;novo</i> variant, 3,010 have FSIQ information; of these, 2,055 (68.3%) have a FSIQ &gt; 70 and 1,586 (52.7%) have a FSIQ &gt; 82. For a sample size <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper N" tabindex="0" ctxtmenu_counter="139" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-speech="upper N"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mtext></mjx-mrow></mjx-math></mjx-container>, the expected number of mutations within genes is computed as <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper E equals 2 upper N phi" tabindex="0" ctxtmenu_counter="140" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 1 (7 2 5 3 6 4))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="8" data-semantic-children="0,7" data-semantic-content="1" data-semantic-owns="0 1 7" data-semantic-speech="upper E equals 2 upper N phi"><mjx-mtext data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="0" data-semantic-parent="8"><mjx-c class="mjx-c45">E</mjx-c></mjx-mtext><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="8" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="7" data-semantic-children="2,3,4" data-semantic-content="5,6" data-semantic-parent="8" data-semantic-owns="2 5 3 6 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="7"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mtext data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="normal" data-semantic-annotation="general:text;clearspeak:simple" data-semantic-id="3" data-semantic-parent="7"><mjx-c class="mjx-c4E">N</mjx-c></mjx-mtext><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="7" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="7"><mjx-c class="mjx-c1D719">𝜙</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="phi" tabindex="0" ctxtmenu_counter="141" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="phi"><mjx-c class="mjx-c1D719">𝜙</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the sum of the mutation rate, per variant type, over all relevant genes. (For example, to calculate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="phi" tabindex="0" ctxtmenu_counter="142" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="phi"><mjx-c class="mjx-c1D719">𝜙</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> for PTVs in genes with pLI &gt; 0.995, we compute the sum of the PTV mutation rates for these genes.) We then compare <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper E" tabindex="0" ctxtmenu_counter="143" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mtext data-semantic-type="text" data-semantic-role="unknown" data-semantic-font="normal" data-semantic-annotation="clearspeak:unit" data-semantic-id="0" data-semantic-speech="upper E"><mjx-c class="mjx-c45">E</mjx-c></mjx-mtext></mjx-mrow></mjx-math></mjx-container> to the observed count for this mutation class, <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper O" tabindex="0" ctxtmenu_counter="144" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="upper O"><mjx-c class="mjx-c1D442">𝑂</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>, and evaluate the distribution of <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper O divided by upper E" tabindex="0" ctxtmenu_counter="145" style="font-size: 130%;"><mjx-math data-semantic-structure="(3 0 1 2)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="infixop" data-semantic-role="division" data-semantic-id="3" data-semantic-children="0,2" data-semantic-content="1" data-semantic-owns="0 1 2" data-semantic-speech="upper O divided by upper E"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="3"><mjx-c class="mjx-c1D442">𝑂</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="division" data-semantic-id="1" data-semantic-parent="3" data-semantic-operator="infixop,/"><mjx-c class="mjx-c2F">/</mjx-c></mjx-mo><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="3"><mjx-c class="mjx-c1D438">𝐸</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-mrow></mjx-math></mjx-container> as a chi-square statistic with 1 degree of freedom.</div><div id="p0735" role="paragraph"><i>Burden of mutations over 102 TADA ASD genes.</i> This analysis addresses the question of whether the signal found in the 102 genes with q&nbsp;&lt; 0.1 could have arisen solely from low IQ subjects, such that any mutations found in higher IQ subjects occurred by chance. To answer this question, we must address the bias inherent in choosing 102 genes because they have q&nbsp;&lt; 0.1. To do so, we performed model-based simulations, similar to those used to evaluate the properties of the TADA model. We first select 874 genes with the smallest q-values from the real data and label them “signal genes.” Let <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="upper M equals 0.306 upper N" tabindex="0" ctxtmenu_counter="146" style="font-size: 130%;"><mjx-math data-semantic-structure="(6 0 1 (5 2 4 3))" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="relseq" data-semantic-role="equality" data-semantic-id="6" data-semantic-children="0,5" data-semantic-content="1" data-semantic-owns="0 1 5" data-semantic-speech="upper M equals 0.306 upper N"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="6"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo space="4" data-semantic-type="relation" data-semantic-role="equality" data-semantic-id="1" data-semantic-parent="6" data-semantic-operator="relseq,="><mjx-c class="mjx-c3D">=</mjx-c></mjx-mo><mjx-mrow space="4" data-semantic-added="true" data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="5" data-semantic-children="2,3" data-semantic-content="4" data-semantic-parent="6" data-semantic-owns="2 4 3"><mjx-mn data-semantic-type="number" data-semantic-role="float" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c2E" noic="true" style="padding-top: 0.666em;">.</mjx-c><mjx-c class="mjx-c33" noic="true" style="padding-top: 0.666em;">3</mjx-c><mjx-c class="mjx-c30" noic="true" style="padding-top: 0.666em;">0</mjx-c><mjx-c class="mjx-c36" style="padding-top: 0.666em;">6</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="5" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi></mjx-mrow></mjx-mrow></mjx-math></mjx-container> be the number of subjects with IQ&nbsp;&lt; 70 (as&nbsp;seen above) who accumulate mutations at rates greater than chance. We generate mutations for the signal genes using the TADA model and Poisson rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis 2 upper M gamma mu right parenthesis" tabindex="0" ctxtmenu_counter="147" style="font-size: 130%;"><mjx-math data-semantic-structure="(10 0 (8 1 5 2 6 3 7 4) 9)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="10" data-semantic-children="8" data-semantic-content="0,9" data-semantic-owns="0 8 9" data-semantic-speech="left parenthesis 2 upper M gamma mu right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="8" data-semantic-children="1,2,3,4" data-semantic-content="5,6,7" data-semantic-parent="10" data-semantic-owns="1 5 2 6 3 7 4"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="8"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="8"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="6" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="8"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="7" data-semantic-parent="8" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="8"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="9" data-semantic-parent="10" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>, where <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="mu" tabindex="0" ctxtmenu_counter="148" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="mu"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the gene-specific mutation rate and <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="gamma" tabindex="0" ctxtmenu_counter="149" style="font-size: 130%;"><mjx-math data-semantic-structure="0" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-speech="gamma"><mjx-c class="mjx-c1D6FE">𝛾</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container> is the increased rate of mutations due to this being a risk gene and the mutation of a particular type, and we generate additional mutations at a Poisson rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="2 left bracket upper N minus upper M right bracket mu" tabindex="0" ctxtmenu_counter="150" style="font-size: 130%;"><mjx-math data-semantic-structure="(11 0 9 (7 1 (5 2 3 4) 6) 10 8)" class="MM-N" aria-hidden="true"><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:unit" data-semantic-id="11" data-semantic-children="0,7,8" data-semantic-content="9,10" data-semantic-owns="0 9 7 10 8" data-semantic-speech="2 left bracket upper N minus upper M right bracket mu"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="0" data-semantic-parent="11"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="9" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mrow space="2" data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="7" data-semantic-children="5" data-semantic-content="1,6" data-semantic-parent="11" data-semantic-owns="1 5 6"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="1" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c5B">[</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="subtraction" data-semantic-id="5" data-semantic-children="2,4" data-semantic-content="3" data-semantic-parent="7" data-semantic-owns="2 3 4"><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="5"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo space="3" data-semantic-type="operator" data-semantic-role="subtraction" data-semantic-id="3" data-semantic-parent="5" data-semantic-operator="infixop,−"><mjx-c class="mjx-c2212">−</mjx-c></mjx-mo><mjx-mi space="3" data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="4" data-semantic-parent="5"><mjx-c class="mjx-c1D440">𝑀</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="6" data-semantic-parent="7" data-semantic-operator="fenced"><mjx-c class="mjx-c5D">]</mjx-c></mjx-mo></mjx-mrow><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="10" data-semantic-parent="11" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi space="2" data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="8" data-semantic-parent="11"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow></mjx-math></mjx-container>. We generate mutations in non-signal genes at a Poisson rate <mjx-container class="MathJax CtxtMenu_Attached_0" jax="CHTML" overflow="overflow" aria-label="left parenthesis 2 upper N mu right parenthesis" tabindex="0" ctxtmenu_counter="151" style="font-size: 130%;"><mjx-math data-semantic-structure="(8 0 (6 1 4 2 5 3) 7)" class="MM-N" aria-hidden="true"><mjx-mrow><mjx-mrow data-semantic-type="fenced" data-semantic-role="leftright" data-semantic-id="8" data-semantic-children="6" data-semantic-content="0,7" data-semantic-owns="0 6 7" data-semantic-speech="left parenthesis 2 upper N mu right parenthesis"><mjx-mo data-semantic-type="fence" data-semantic-role="open" data-semantic-id="0" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c28">(</mjx-c></mjx-mo><mjx-mrow data-semantic-type="infixop" data-semantic-role="implicit" data-semantic-annotation="clearspeak:simple;clearspeak:unit" data-semantic-id="6" data-semantic-children="1,2,3" data-semantic-content="4,5" data-semantic-parent="8" data-semantic-owns="1 4 2 5 3"><mjx-mn data-semantic-type="number" data-semantic-role="integer" data-semantic-font="normal" data-semantic-annotation="clearspeak:simple" data-semantic-id="1" data-semantic-parent="6"><mjx-c class="mjx-c32">2</mjx-c></mjx-mn><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="4" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="latinletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="2" data-semantic-parent="6"><mjx-c class="mjx-c1D441">𝑁</mjx-c></mjx-mi><mjx-mo data-semantic-type="operator" data-semantic-role="multiplication" data-semantic-id="5" data-semantic-parent="6" data-semantic-added="true" data-semantic-operator="infixop,⁢"><mjx-c class="mjx-c2062">⁢</mjx-c></mjx-mo><mjx-mi data-semantic-type="identifier" data-semantic-role="greekletter" data-semantic-font="italic" data-semantic-annotation="clearspeak:simple" data-semantic-id="3" data-semantic-parent="6"><mjx-c class="mjx-c1D707">𝜇</mjx-c></mjx-mi></mjx-mrow><mjx-mo data-semantic-type="fence" data-semantic-role="close" data-semantic-id="7" data-semantic-parent="8" data-semantic-operator="fenced"><mjx-c class="mjx-c29">)</mjx-c></mjx-mo></mjx-mrow></mjx-mrow></mjx-math></mjx-container>. We run TADA to get the new top 102 genes and the new signal genes, and we record counts occurring in new signal genes by chance (i.e., for individuals with high IQ). We perform the simulation 500 times to obtain the distribution of counts in signal genes for individuals with high IQ and compare this to the observed data. We then repeat the analysis, splitting on an IQ of 82 instead of 70.</div><div id="p0740" role="paragraph">For all four informative <i>de novo</i> mutation types (two missense categories and two PTV categories), the expected counts were consistently lower than the observed count; only for missense mutations with MPC between 1 and 2 does the expected count, 13.54 (±4.2) approach the observed value, 23 (p&nbsp;= 0.03). For all other mutation types, the empirical p value is far smaller, based on 500 simulations (MPC ≥ 2: 13.9 ± 3.9 expected versus 28 observed; PTV for pLI ≥ 0.995: 8.3 ± 3.0 versus 48; and PTV for 0.5&nbsp;≤ pLI&nbsp;&lt; 0.995: 3.0 ± 1.9 versus 15). We also performed these simulations for a split on IQ at 82 and reached the same conclusion, i.e., that the mutations in the higher IQ ASD subjects accumulate at a rate far greater than chance (e.g., <a id="crosref1590" href="#fig4">Figure&nbsp;4</a>).</div></section><section id="sec-4-6-10-2"><h5>GER and NC mutations in ASD as a function of age of walking and IQ</h5><div id="p0745" role="paragraph">As with the ASD<sub>P</sub> and ASD<sub>NDD</sub> genes (<a id="crosref1595" href="#fig4">Figure&nbsp;4</a>), we similarly compared whether ascertained ASD individuals harboring <i>de novo</i> missense (MPC ≥ 1) or <i>de novo</i> PTVs in GER and NC genes differed from one another and the remaining ASD individuals with respect to age of walking and full-scale IQ when such phenotype data was available (4,456 ASD individuals had available age of walking data; 4,821 individuals had available full-scale IQ data). The 140 ASD individuals with these groups of <i>de novo</i> variants in GER genes walked 1.84&nbsp;months later than the 71 ASD individuals with such <i>de novo</i> variants in NC genes (p&nbsp;= 0.01; 95% CI: 0.37-3.31; two-sided, two-sample t test) and 3.4&nbsp;months later than the remaining 4,204 ASD individuals (p&nbsp;= 7.54x10ˆ-21; 95% CI: 2.69-4.11; two-sided, two-sample t test) (Figure&nbsp;S3A). Similarly, the 71 ASD individuals harboring <i>de novo</i> missense (MPC ≥ 1) or <i>de novo</i> PTVs in NC genes also walked 1.56&nbsp;months later than the remaining 4,204 ASD individuals (p&nbsp;= 0.002; 95% CI: 0.59-2.53; two-sided, two-sample t test). A similar trend was also observed with respect to full-scale IQ, although we failed to observe any difference between the 159 ASD individuals with these groups of <i>de novo</i> variants in GER genes and the 77 ASD individuals with such <i>de novo</i> variants in NC genes (p&nbsp;= 0.84; two-sided, two-sample t test). The 159 ASD individuals with <i>de novo</i> missense (MPC ≥ 1) and <i>de novo</i> PTVs in GER genes had a full-scale IQ 15.23 lower (95% CI: 11.02-19.46) than the remaining 4,204 ASD individuals (p&nbsp;= 1.6x10ˆ-12; two-sided, two-sample t test) (<a id="crosref1600" href="#figs3">Figure&nbsp;S3</a>B). Similarly, the 77 ASD individuals with <i>de novo</i> missense (MPC ≥ 1) and <i>de novo</i> PTVs in NC genes had a full-scale IQ 15.97 lower (95% CI: 9.95-21.99) than the remaining 4,204 ASD individuals (p&nbsp;= 2.1x10ˆ-7; two-sided, two-sample t test).</div></section></section><section id="sec-4-6-11"><h4>Expression Analysis</h4><section id="sec-4-6-11-1"><h5>Tissue-specific expression</h5><div id="p0750" role="paragraph">Genotype-Tissue Expression (GTEx) RNA-seq data (<a id="intref0140" href="https://www.gtexportal.org/home/" target="_blank">https://www.gtexportal.org/home/</a>) were summarized to GENCODE10 and gene-level reads per kilobase million mapped reads (RPKM) values were used across 53 tissue types, including 13 distinct brain regions. Samples with an RNA integrity number (RIN) ≤ 7 were removed from subsequent analyses. Genes were defined as brain-expressed if they were present at an RPKM of 0.5 in 80% of the samples from at least one tissue type, resulting in 27,546 genes. Finally, expression values were log-transformed (log<sub>2</sub>[RPKM+1]).</div><div id="p0755" role="paragraph">To determine tissue-specific gene expression signatures (i.e., genes which are significantly more expressed in a given tissue type compared to all other tissues), a linear regression model was applied for each gene for each tissue against all other tissues. Models were adjusted for age, RIN, gender, individual as a repeated-measure, and surrogate variables to account for potential batch effects and other unwanted technical and biological variation. Significance values were adjusted for multiple testing using the Benjamini and Hochberg (BH) method to estimate FDR. After the BH correction, genes with q<i>-</i>value&nbsp;&lt; 0.05 and log<sub>2</sub> FC &gt; 0.5 are defined as “tissue-specific” in a given tissue, though we note that a gene may be listed as tissue-specific in more than one tissue, especially if those tissues are closely related. These curated data formed the basis of our tissue-specific gene set enrichment analysis.</div><div id="p0760" role="paragraph">To test for over-representation of tissue-specific expression within a given gene set, a modified version of the GeneOverlap function in R was used, so that all pairwise tests were corrected for multiple testing using the BH method. The Fisher’s exact test function also provides an estimated odds ratio in comparison to a tissue- and genome-wide background set of 27,546 genes.</div></section><section id="sec-4-6-11-2"><h5>Developmental neocortex expression</h5><div id="p0765" role="paragraph">BrainSpan developmental RNA-seq data (<a id="intref0145" href="http://www.brainspan.org" target="_blank">http://www.brainspan.org</a>) were summarized to GENCODE10 and gene-level RPKMs were used across 528 samples from 40 individuals (<span class="dropBlock reference-citations"><a id="body-ref-sref56-2" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-2" aria-controls="sref56-2">Li et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref56-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Li, M. ∙ Santpere, G. ∙ Imamura Kawasawa, Y. ..., BrainSpan Consortium, PsychENCODE Consortium, PsychENCODE Developmental Subgroup</div><div><strong>Integrative functional genomic analysis of human brain development and neuropsychiatric risks</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat7615</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat7615" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058377382" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545854/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat7615&amp;pmid=30545854" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Only the neocortical regions were used in our analysis—dorsolateral prefrontal cortex (DFC), ventrolateral prefrontal cortex (VFC), medial prefrontal cortex (MFC), orbitofrontal cortex (OFC), primary motor cortex (M1C), primary somatosensory cortex (S1C), primary association cortex (A1C), inferior parietal cortex (IPC), superior temporal cortex (STC), inferior temporal cortex (ITC), and primary visual cortex (V1C). Samples with RIN ≤ 7 were removed from subsequent analyses. Genes were defined as expressed if they were present at an RPKM of 0.5 in 80% of the samples from at least one neocortical region at one major temporal epoch, resulting in 22,141 genes across 299 high-quality samples ranging from 8 post-conceptual weeks to 40 years of age (<a id="crosref1610" href="#figs4">Figure&nbsp;S4</a>A). Finally, expression values were log-transformed (log<sub>2</sub>[RPKM+1]).</div><div id="p0770" role="paragraph">Linear regression was performed for each of the 22,141 genes, modeling gene expression as a continuous dependent variable, as a function of a binary ‘prenatal’ stage variable. Similar to tissue-specific linear models (as above), each regression analysis included gender, individual as a repeated-measure, ethnicity, and surrogate variables as adjustment variables. The regression model generated a ‘prenatal effect’ (<i>t-</i>statistic) of the log<sub>2</sub> FC of prenatal versus postnatal transcript abundance. A BH multiple test&nbsp;correction was used to estimate FDR. Genes were defined as either prenatally or postnatally biased (log<sub>2</sub> FC &gt; 0.1 and q-value&nbsp;&lt;&nbsp;0.05) or unbiased in expression (q &gt; 0.05). Under this paradigm, we measured the concordance of the prenatal effect across each of the 11 neocortical brain regions against the combination of all 11 areas to ensure consistent gene-based effects; we observed an average <i>r</i> of 0.956. Subsequently, a total of five gene sets (102 ASD genes, 53 ASD<sub>P</sub> genes, 49 ASD<sub>NDD</sub> genes, 58 GER genes, and 24 NC genes) were evaluated by a Wilcoxon signed rank test to determine if the fetal effect distribution of the set differed significantly from the entire neocortical background, using the reduced statistic of one fetal effect per gene. The neocortical background was defined as genes which were simultaneously detected by WES in the current study as well as genes found to be expressed in the neocortex following quality control procedures (described above).</div></section><section id="sec-4-6-11-3"><h5>Early developmental neocortical co-expression modules from BrainSpan expression data</h5><div id="p0775" role="paragraph">Weighted gene co-expression network analysis (WGCNA) was used to build a signed co-expression network from all early developmental samples passing our QC standards (as above). This resulted in 177 high-quality samples ranging from 8 post-conception weeks to 1 year of age that were used to build an early developmental network. The absolute values of the biweight midcorrelation coefficients were computed for all possible gene pairs and resulting values were transformed with an exponential weight (β). We used a β threshold power of 21 so the subsequent network satisfied scale-free topology (R<sup>2</sup> &gt; 0.8) and had a high mean connectivity and sufficient information for module detection.</div><div id="p0780" role="paragraph">Module robustness was ensured by randomly sampling (2/3 of the total) from the initial set of samples 1000 times followed by consensus network analysis to define modules. The dynamic tree-cut algorithm was used to detect network modules with a minimum module size set to 200 and a cut tree height set to 0.9999. Singular value decomposition of each module’s expression matrix was performed and the resulting module eigengene (ME), equivalent to the first principal component, was used to represent the overall expression profile for each module per sample. Pairs of modules were merged when the correlation of their ME values exceeded 0.90. A total of 27 early developmental neocortical co-expression modules were identified. Each module was assessed for over-representation of five gene sets (102 ASD genes, 53 ASD<sub>P</sub> genes, 49 ASD<sub>NDD</sub> genes, 58 GER genes, and 24 NC genes) using a one-sided Fisher’s exact test and adjusting all pairwise tests for multiple testing using the BH method. Functional annotation of candidate modules was performed using ToppGene (<span class="dropBlock reference-citations"><a id="body-ref-sref11" href="#" role="doc-biblioref" data-xml-rid="bib11" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref11" aria-controls="sref11">Chen et&nbsp;al., 2009</a><div class="dropBlock__holder" data-db-target-of="sref11"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">11.</div><div>Chen, J. ∙ Bardes, E.E. ∙ Aronow, B.J. ...</div><div><strong>ToppGene Suite for gene list enrichment analysis and candidate gene prioritization</strong></div><div><em>Nucleic Acids Res.</em> 2009; <strong>37</strong></div><div>W305-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkp427" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-67849130563" target="_blank" aria-label="Open Scopus in new tab">Scopus (1034)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkp427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div><div id="p0785" role="paragraph"><i>Gene expression in cell types from the fetal human neocortex.</i> To assess cell type enrichment in the developing human cortex, we used data from <span class="dropBlock reference-citations"><a id="body-ref-sref70-5" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-5" aria-controls="sref70-5">Nowakowski et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref70-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span> consisting of 4,216 cells, which were divided into 25 cell type clusters and expressed 58,865 identified genes. We excluded genes that were not on the list of 17,484 autosomal protein-coding genes used for the TADA analysis, resulting in 17,116 genes which included all 102 TADA ASD-associated genes. The 4,261 cells were divided into 17 bins by developmental stage, measured in post-conception weeks (<span class="dropBlock reference-citations"><a id="body-ref-sref70-6" href="#" role="doc-biblioref" data-xml-rid="bib70" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref70-6" aria-controls="sref70-6">Nowakowski et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref70-6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">68.</div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). For each of the 17 developmental stages, a gene was considered expressed if at least one transcript mapped to this gene in 25% or more of cells for at least one post-conception weeks stage.</div><div id="p0790" role="paragraph">For cell type-specific analyses, six cell type clusters were excluded because they could not be unambiguously associated with a cell type. The 19 remaining cell types contained 3,839 cells. Within each cell type cluster, a gene was considered expressed if one or more of its transcripts were detected in 25% or more cells, which resulted in 7,867 protein-coding genes being expressed in one or more cell type cluster.</div><div id="p0795" role="paragraph">To determine enrichment for expression of the 102 TADA ASD genes, the universe of genes expressed was determined by the cells in this experiment; e.g., for the cell type-specific enrichment, the universe was U&nbsp;= 7,867 genes. The counts of the two-by-two table for enrichment of ASD genes, by cell type cluster, were: X, the number of TADA genes expressed in the cell type; Y, the number of TADA genes not expressed in the cell type; Z, the total number of genes expressed in the cell type minus X; and U – (X+Y+Z). Given this table, the enrichment odds ratio was calculated and significance judged by Fisher’s exact test.</div><div id="p0800" role="paragraph">To evaluate how well cell types clustered, we first found 10% of genes showing the largest variation among cell types, as judged by analysis of variance. Performing hierarchical clustering using dissimilarity of expression for these genes revealed commonalities about the common types of cells (hclust package from R, centroid method) (<a id="crosref1630" href="#fig5">Figure&nbsp;5</a>).</div><div id="p0805" role="paragraph">To evaluate whether the enrichment odds ratio was a function of the number of genes expressed per cell type, we used linear regression. Because the odds ratio decreases as the number of genes increases, we next asked what might cause this phenomenon. It is reasonable to conjecture that the diversity of genes expressed (i.e., captured in our sequenced transcripts) would likely be a function of the evenness of expression of observed transcripts. In other words, if some transcripts were expressed at high levels, they will tend to lower the chance of capturing other genes in the sequence run for the cell, because the number of reads per cell was limited to fall between 1 and 2 million reads. This relative evenness should be captured by the mean expression over genes, and it is for these data: there is a very strong relationship between mean expression, taken over all expressed genes per cell type, and the number of genes expressed (R<sup>2</sup>&nbsp;= 94.5%, <a id="crosref1635" href="#fig5">Figure&nbsp;5</a>).</div><div id="p0810" role="paragraph"><i>Weighted gene co-expression network analysis of BrainSpan bulk cortex gene expression.</i> To interpret gene co-expression and enrichment across a broader range of early developmental samples, we used Weighted Gene Coexpression Network Analysis (WGCNA) to assess spatiotemporal co-expression from 177 high-quality BrainSpan samples aged 8 pcw to 1 year. WGCNA yielded 27 early developmental co-expression modules, two of which show significant over-representation of 101 ASD genes (<i>PAX5</i> went undetected in the BrainSpan data and so was not considered) after correction for multiple testing (<a id="crosref1640" href="#figs4">Figure&nbsp;S4</a>; <a id="crosref1645" href="#mmc5">Table S5</a>): M4 for the NC gene set (∩&nbsp;= 5 genes, OR&nbsp;= 13.7, p&nbsp;= 0.002, FET); and M25 for all 101 ASD genes (∩&nbsp;= 17 genes, OR&nbsp;= 12.1, p&nbsp;= 3x10<sup>−11</sup>, FET), although driven by the GER gene set (∩&nbsp;= 17 genes, OR&nbsp;= 26.2, p&nbsp;= 9x10<sup>−16</sup>, FET). With regard to single-cell gene&nbsp;expression, genes in the NC-specific M4 showed greatest enrichment in maturing neurons, both excitatory and inhibitory (p&nbsp;&lt; 0.001 for each of 6 neuronal cell types, FET), whereas genes in M25 showed enrichment across all 19 cell types (p&nbsp;&lt; 0.001 for all&nbsp;cell types, FET). The DAWN associated genes are enriched in M3 (∩&nbsp;= 10, OR&nbsp;= 10.1, p&nbsp;= 5x10<sup>−6</sup>, FET) and M25 (∩&nbsp;= 7, OR&nbsp;= 5.9, p&nbsp;= 0.004, FET) but not M4, although expression of genes in M4 are highly correlated with those of M3 during early development, and both are highly expressed prenatally. Comparing our gene modules to previously published candidate ASD gene networks obtained using WGCNA (<span class="dropBlock reference-citations"><a id="body-ref-sref73-3" href="#" role="doc-biblioref" data-xml-rid="bib73" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref73-3" aria-controls="sref73-3">Parikshak et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref73-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">71.</div><div>Parikshak, N.N. ∙ Luo, R. ∙ Zhang, A. ...</div><div><strong>Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1008-1021</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889561601" target="_blank" aria-label="Open Scopus in new tab">Scopus (434)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.031&amp;pmid=24267887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) shows that our M4 strongly overlaps with previously identified M16 (p&nbsp;= 5.5x10<sup>−69</sup>, FET), and our M24 overlaps with previously identified M2 (p&nbsp;= 1.3x10<sup>−239</sup>, FET) (note that both studies make use of BrainSpan data so the overlap is not unexpected but does help to relate the results from the two studies).</div></section></section><section id="sec-4-6-12"><h4>Detecting Association with Networks (DAWN)</h4><section id="sec-4-6-12-1"><h5>Extended gene discovery with co-expression data</h5><div id="p0815" role="paragraph">The DAWN (Detecting Association With Networks) algorithm is a network-based gene discovery algorithm. The main assumption of the algorithm, related to this research, is that ASD-related genes are working as a functional group and should be co-expressed in a relevant spatio-temporal window during neurodevelopment. The algorithm predicts ASD-related genes using their interactions with other genes in the gene partial co-expression network, which is constructed using a Partial Neighborhood Selection method. A hidden Markov random field model assigns network-adjusted posterior risk scores to each gene (<span class="dropBlock reference-citations"><a id="body-ref-sref59" href="#" role="doc-biblioref" data-xml-rid="bib58 bib59" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref59" aria-controls="sref59">Liu et&nbsp;al., 2014; Liu et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref59"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Liu, L. ∙ Lei, J. ∙ Sanders, S.J. ...</div><div><strong>DAWN: a framework to identify autism genes and subnetworks using gene expression and genetics</strong></div><div><em>Mol. Autism.</em> 2014; <strong>5</strong>:22</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2040-2392-5-22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84899472749" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24602502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2040-2392-5-22&amp;pmid=24602502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">57.</div><div>Liu, L. ∙ Lei, J. ∙ Roeder, K.</div><div><strong>Network assisted analysis to reveal the genetic basis of autism</strong></div><div><em>Ann. Appl. Stat.</em> 2015; <strong>9</strong>:1571-1600</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1214/15-AOAS844" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84947255657" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27134692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1214%2F15-AOAS844&amp;pmid=27134692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). A gene is assigned a higher posterior risk score if the gene (1) has a high prior risk assigned using exome sequencing studies (i.e., TADA) and (2) is highly interacting with other risk genes.</div><div id="p0820" role="paragraph">To estimate the partial co-expression network, we used the BrainSpan microarray dataset (<a id="intref0150" href="https://developinghumanbrain.org/" target="_blank">https://developinghumanbrain.org/</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref43" href="#" role="doc-biblioref" data-xml-rid="bib43" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref43" aria-controls="sref43">Kang et&nbsp;al., 2011</a><div class="dropBlock__holder" data-db-target-of="sref43"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">41.</div><div>Kang, H.J. ∙ Kawasawa, Y.I. ∙ Cheng, F. ...</div><div><strong>Spatio-temporal transcriptome of the human brain</strong></div><div><em>Nature.</em> 2011; <strong>478</strong>:483-489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80054993342" target="_blank" aria-label="Open Scopus in new tab">Scopus (880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22031440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10523&amp;pmid=22031440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). We opted for this dataset over the RNA-seq from an overlapping group of samples (<span class="dropBlock reference-citations"><a id="body-ref-sref56-3" href="#" role="doc-biblioref" data-xml-rid="bib56" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref56-3" aria-controls="sref56-3">Li et&nbsp;al., 2018</a><div class="dropBlock__holder" data-db-target-of="sref56-3"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">54.</div><div>Li, M. ∙ Santpere, G. ∙ Imamura Kawasawa, Y. ..., BrainSpan Consortium, PsychENCODE Consortium, PsychENCODE Developmental Subgroup</div><div><strong>Integrative functional genomic analysis of human brain development and neuropsychiatric risks</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat7615</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat7615" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058377382" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545854/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat7615&amp;pmid=30545854" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) due to the larger number of samples and individuals in the microarray data. We extracted the subset of the data for a spatio-temporal window that was previously implicated for ASD risk: midfetal development in the prefrontal cortex (PFC) (<span class="dropBlock reference-citations"><a id="body-ref-sref109-4" href="#" role="doc-biblioref" data-xml-rid="bib109" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-4" aria-controls="sref109-4">Willsey et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref109-4"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Willsey, A.J. ∙ Sanders, S.J. ∙ Li, M. ...</div><div><strong>Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:997-1007</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889583293" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.020&amp;pmid=24267886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). Because the microarray probes for <i>CHD8</i>, one of the genes with the lowest p values for ASD association (<span class="dropBlock reference-citations"><a id="body-ref-sref21-13" href="#" role="doc-biblioref" data-xml-rid="bib21" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref21-13" aria-controls="sref21-13">De Rubeis et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref21-13"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">19.</div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), do not perform well (<span class="dropBlock reference-citations"><a id="body-ref-sref109-5" href="#" role="doc-biblioref" data-xml-rid="bib109 bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref109-5" aria-controls="sref109-5">Willsey et&nbsp;al., 2013; Cotney et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref109-5"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cotney, J. ∙ Muhle, R.A. ∙ Sanders, S.J. ...</div><div><strong>The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:6404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms7404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84924567722" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25752243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms7404&amp;pmid=25752243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">105.</div><div>Willsey, A.J. ∙ Sanders, S.J. ∙ Li, M. ...</div><div><strong>Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:997-1007</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889583293" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.020&amp;pmid=24267886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we elected to impute the expression of <i>CHD8</i> in microarray data from the RNA-seq data. To do so, we picked the top genes that are highly correlated with <i>CHD8</i> in the RNA-seq data. Using the multivariate normal data imputation method, we imputed the corresponding measurements of <i>CHD8</i> in the microarray data for DAWN analysis. We input the TADA p values as the prior risk scores. DAWN works with p values less than, but not equal to, 1.0. Thus, we replaced the p values of 7,994 genes (45.7% of total genes) with a p value equal to 1.0 with 0.999. We fixed the top 10 ASD-associated genes with respect to TADA q-values as seed genes in the program. No additional covariates were supplied. The input hyperparameters for the DAWN algorithm were lambda&nbsp;= 0.24, p value cutoff&nbsp;= 0.1, and partial correlation threshold&nbsp;= 0.7. DAWN yielded 138 genes (FDR ≤ 0.005), including 83 genes that are not captured by TADA, with 69 of these 83 correlated with many other genes (<a id="crosref1675" href="#mmc5">Table S5</a>). Of the 83, 19 are implicated in neurodevelopmental disorders, and seven of these have autosomal recessive inheritance (<a id="crosref1680" href="#mmc5">Table S5</a>). Of the 138 genes, 38 are expressed in excitatory cell types (p&nbsp;&lt; 1.6x10-4, FET), 25 are also expressed in inhibitory cell types (p&nbsp;&lt; 7.9&nbsp;× 10-4, FET), and many play a role in GER or NC.</div></section><section id="sec-4-6-12-2"><h5>Assessment of interactions by gene function</h5><div id="p0825" role="paragraph">We used the DAWN results for a downstream network analysis of the interactions between synaptic genes and chromatin genes on a network of (1) ASD risk genes and (2) additional genes chosen by DAWN as tightly interacting with ASD risk genes. The list of synaptic genes was obtained from Genes to Cognition (<a id="intref0155" href="http://www.genes2cognition.org/db/GeneList" target="_blank">http://www.genes2cognition.org/db/GeneList</a>). Specifically, we used lists L09-L16, which include the human orthologs of various synaptic complexes in mouse (1,123 genes). The list of chromatin modifiers was obtained from the HIstome Database (<a id="intref0160" href="http://www.actrec.gov.in/histome/" target="_blank">http://www.actrec.gov.in/histome/</a>; <span class="dropBlock reference-citations"><a id="body-ref-sref45" href="#" role="doc-biblioref" data-xml-rid="bib45 bib37" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref45" aria-controls="sref45">Khare et&nbsp;al., 2012; Huang et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref45"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">35.</div><div>Huang, H.T. ∙ Kathrein, K.L. ∙ Barton, A. ...</div><div><strong>A network of epigenetic regulators guides developmental haematopoiesis in&nbsp;vivo</strong></div><div><em>Nat. Cell Biol.</em> 2013; <strong>15</strong>:1516-1525</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2870" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24240475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2870&amp;pmid=24240475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">43.</div><div>Khare, S.P. ∙ Habib, F. ∙ Sharma, R. ...</div><div><strong>HIstome--a relational knowledgebase of human histone proteins and histone modifying enzymes</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D337-D342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr1125" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84862156464" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22140112/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr1125&amp;pmid=22140112" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>; 173 genes total). We picked a threshold of FDR&nbsp;&lt; 0.025, which yielded 100 genes in a DAWN ranking and evaluated interactions among them to form our subnetwork. Of these 100 genes, 40 genes have a TADA q-value&nbsp;&lt; 0.1.</div></section></section><section id="sec-4-6-13"><h4>Protein-Protein Connectivity Among ASD Genes</h4><div id="p0830" role="paragraph">We used the InWeb_IM (<span class="dropBlock reference-citations"><a id="body-ref-sref55-2" href="#" role="doc-biblioref" data-xml-rid="bib55" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref55-2" aria-controls="sref55-2">Li et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref55-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">53.</div><div>Li, T. ∙ Wernersson, R. ∙ Hansen, R.B. ...</div><div><strong>A scored human protein-protein interaction network to catalyze genomic interpretation</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:61-64</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84997794652" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27892958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4083&amp;pmid=27892958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) database of direct protein-protein interactions (PPI) to investigate whether the number of connections among ASD<sub>P</sub> and ASD<sub>NDD</sub> genes exceeds expectation, suggesting significant functional relatedness. All candidate genes were brain-expressed as confirmed by the Allen Human Brain Atlas RNA-sequencing data (<a id="intref0165" href="http://portal.brain-map.org/" target="_blank">http://portal.brain-map.org/</a>), as this could otherwise serve as a potential confounding factor for excessive connectivity, compared to the general pool of genes present in the reference PPI network. We binned genes in each tested gene list into deciles of mean brain expression (in TPM) and used this distribution as a further reference for construction of random gene lists for significance analysis. To assess the significance of observed connectivity within each candidate gene list, we performed 1000 random draws of gene sets matching the candidate gene list in number of genes and distribution of expression levels. Empirical p values were estimated as the proportion of random gene sets with greater or equal connectivity compared to the candidate gene list. Subsequently, to estimate the significance of connections with a known list of genes associated with neurodevelopmental delay (<span class="dropBlock reference-citations"><a id="body-ref-sref46-8" href="#" role="doc-biblioref" data-xml-rid="bib46" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref46-8" aria-controls="sref46-8">Kosmicki et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref46-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">44.</div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>), we performed 1000 random draws of gene sets matching the candidate gene lists in number of genes and brain expression distribution. Further, the number of direct connections was estimated between a random gene set and neurodevelopmental delay-associated genes. Empirical p values were estimated as the proportion of random gene sets with a greater or equal number of connections to neurodevelopmental delay-associated genes compared to the candidate gene list.</div><div id="p0835" role="paragraph">To explore whether GER and NC gene sets interact more than would be expected by chance, we analyzed protein-protein interaction (PPI) networks (<a id="crosref1695" href="#figs5">Figure&nbsp;S5</a>B; <a id="crosref1700" href="#mmc5">Table S5</a>) and found they do not: there was a significant excess of interactions among all ASD genes (82 genes, p&nbsp;= 0.02, FET), GER genes (49 genes, p&nbsp;= 0.006, FET), and NC genes (12 genes, p&nbsp;= 0.03, FET), but not among GER and NC genes (2 genes, p&nbsp;= 1.00, FET). Nor do GER genes regulate the NC genes, according to our analyses, although GER-GER regulation was enriched (<a id="crosref1705" href="#figs5">Figure&nbsp;S5</a>C; <a id="crosref1710" href="#mmc5">Table S5</a>).</div></section><section id="sec-4-6-14"><h4>Enrichment of Transcription Factor Regulatory Networks for GER Genes</h4><div id="p0840" role="paragraph">Starting with the 58 GER genes, we performed a systematic search for regulatory targets identified by protein-DNA interactions (e.g., ChIP-Seq) or protein-RNA interactions (e.g., iCLIP). First, we searched for target sets in available databases: ChEA (<span class="dropBlock reference-citations"><a id="body-ref-sref47" href="#" role="doc-biblioref" data-xml-rid="bib47" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref47" aria-controls="sref47">Lachmann et&nbsp;al., 2010</a><div class="dropBlock__holder" data-db-target-of="sref47"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">45.</div><div>Lachmann, A. ∙ Xu, H. ∙ Krishnan, J. ...</div><div><strong>ChEA: transcription factor regulation inferred from integrating genome-wide ChIP-X experiments</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2438-2444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq466" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77957243164" target="_blank" aria-label="Open Scopus in new tab">Scopus (315)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20709693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq466&amp;pmid=20709693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and ENCODE (<span class="dropBlock reference-citations"><a id="body-ref-sref6" href="#" role="doc-biblioref" data-xml-rid="bib6" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref6" aria-controls="sref6">Birney et&nbsp;al., 2007</a><div class="dropBlock__holder" data-db-target-of="sref6"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">6.</div><div>Birney, E. ∙ Stamatoyannopoulos, J.A. ∙ Dutta, A. ..., ENCODE Project Consortium, NISC Comparative Sequencing Program, Baylor College of Medicine Human Genome Sequencing Center, Washington University Genome Sequencing Center, Broad Institute, Children’s Hospital Oakland Research Institute</div><div><strong>Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</strong></div><div><em>Nature.</em> 2007; <strong>447</strong>:799-816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05874" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-34250305146" target="_blank" aria-label="Open Scopus in new tab">Scopus (3593)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17571346/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05874&amp;pmid=17571346" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The target lists were obtained from the Enrichr libraries (ChEA 2016 and ENCODE ChIP-seq 2015; <span class="dropBlock reference-citations"><a id="body-ref-sref12" href="#" role="doc-biblioref" data-xml-rid="bib12" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref12" aria-controls="sref12">Chen et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref12"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">12.</div><div>Chen, E.Y. ∙ Tan, C.M. ∙ Kou, Y. ...</div><div><strong>Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84876100615" target="_blank" aria-label="Open Scopus in new tab">Scopus (1156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23586463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-128&amp;pmid=23586463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and from a literature search for available ChIP or CLIP experiments, which are immunoprecipitation-based techniques to identify protein interactions with DNA and RNA, respectively. We found target data for 26 of 55 GER genes, with targets ranging from 5 to 8,189 per gene. The data were generated in a wide range of tissues ranging from cell lines to liver to cortex and stemmed from 2 species (mouse and human). In total, we identified 21,514 gene targets from the 26 genes, which included 14,925 protein-coding genes from Ensembl. We constructed a network of 48,932 interactions with genes as the nodes and directed edges as transcription factor regulation relationships (<a id="crosref1730" href="#mmc5">Table S5</a>).</div><div id="p0845" role="paragraph">To assess the significance of the connection between GER genes and a set of targets (i.e., NC genes), we generated random gene sets that matched genes with respect to brain expression, <i>de novo</i> PTV mutation rate, and pLI. For mutation rate, we used 5 bins such&nbsp;that each bin contained an equal number of genes (∼3,497). For pLI, we split genes into 3 groups: 0 to&nbsp;&lt; 0.5, ≥ 0.5 to&nbsp;&lt;&nbsp;0.995, and ≥ 0.995 to 1. They contained 12,152, 4,506, and 1,583 genes, respectively.</div><div id="p0850" role="paragraph">First, we checked whether GER genes are significantly connected to ASD-associated genes. There are 409 connections between 58 GER genes and the 102 ASD-associated genes (which include the 58 GER genes). Eight self-loops (a protein binding near to the gene that encodes it) were ignored. We generated 1,000 random gene sets of size 102. While the expected number of connections was 314, GER genes have 361 links to ASD-associated genes (p&nbsp;= 0.006). Then, we checked the significance of the connectivity within GER genes (only GER-GER connections). There are 229 connections within GER genes. Repeating the same analysis, we found the expected number of connections to be 175, demonstrating significant connectivity (p&nbsp;&lt; 0.001). On the other hand, the connectivity between GER genes and NC genes was weaker—there are 132 connections, whereas the expected was 140 (p&nbsp;= 0.72).</div><div id="p0855" role="paragraph">As a further control, we checked whether GER genes were significantly connected to congenital heart disease risk genes, which is known to have an overlapping genetic component with ASD (<span class="dropBlock reference-citations"><a id="body-ref-sref40-1" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-1" aria-controls="sref40-1">Jin et&nbsp;al., 2017</a><div class="dropBlock__holder" data-db-target-of="sref40-1"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). The list of 253 congenital heart disease risk genes was obtained from <span class="dropBlock reference-citations"><a id="body-ref-sref40-2" href="#" role="doc-biblioref" data-xml-rid="bib40" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref40-2" aria-controls="sref40-2">Jin et&nbsp;al. (2017)</a><div class="dropBlock__holder" data-db-target-of="sref40-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">38.</div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>. Of the 253 genes, 19 were excluded because they are either not autosomal protein-coding genes and/or lack known mutations rates, as used in our analysis. Specifically, 11 are on chromosome X (<i>HCCS, NSDHL, RBM10, PQBP1, ZIC3, GPC3, BCOR, FLNA, OFD1, COX7B, MID1</i>); 6 have no data in ExAC (<i>DNAH11, C1ORF127, IRX5, GATA6, FOXC2, FOXC1</i>); and 2 are noncoding (<i>RNU4ATAC</i> and <i>RPS17</i>). There are 685 links between GER genes and 234 congenital heart disease genes compared to an expected value of 622 (p&nbsp;= 0.007; genes were matched with respect to pLI and mutation rate, but not brain expression). Thus, despite a strong regulatory relationship existing within GER genes and even between GER genes and congenital heart disease genes, this is missing between GER genes and NC genes, suggesting that these functional circuitries act independently rather than as a coherent unit, which is also seen in our DAWN analysis. We note that many of the ChIP and CLIP datasets were not generated in brain tissue and that NC genes are more likely to be brain-specific than GER genes.</div><section id="sec-4-6-14-1"><h5>Enrichment of CHD8 targets for the GER genes</h5><div id="p0860" role="paragraph">To assess whether ASD-associated genes relate to known genome-wide CHD8 binding sites, we tested our previously defined GER and NC gene sets for enrichment with human brain-specific sequences from two independent ChIP-seq studies covering: 1) 3,281 CHD8-binding sites in the human mid-fetal brain at 16-19 post-conception weeks (<span class="dropBlock reference-citations"><a id="body-ref-sref18-2" href="#" role="doc-biblioref" data-xml-rid="bib18" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref18-2" aria-controls="sref18-2">Cotney et&nbsp;al., 2015</a><div class="dropBlock__holder" data-db-target-of="sref18-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">17.</div><div>Cotney, J. ∙ Muhle, R.A. ∙ Sanders, S.J. ...</div><div><strong>The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:6404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms7404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84924567722" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25752243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms7404&amp;pmid=25752243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>); and 2) 6,860 CHD8-binding sites in human neural progenitor cells using the intersection of signal-enriched regions detected by all three CHD8 antibodies used in the study (<span class="dropBlock reference-citations"><a id="body-ref-sref99" href="#" role="doc-biblioref" data-xml-rid="bib99" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref99" aria-controls="sref99">Sugathan et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref99"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">95.</div><div>Sugathan, A. ∙ Biagioli, M. ∙ Golzio, C. ...</div><div><strong>CHD8 regulates neurodevelopmental pathways associated with autism spectrum disorder in neural progenitors</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:E4468-E4477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1405266111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84908065133" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25294932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1405266111&amp;pmid=25294932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>). In order to assess overlap with these binding sites, genomic coordinates were defined as the start and end positions for each GER and NC gene (analogous to gene length). A permutation-based approach with 1,000 random permutations was used to determine statistical significance of the overlap between genomic coordinates for GER and NC genes with CHD8-binding sites using the R package regioneR (<span class="dropBlock reference-citations"><a id="body-ref-sref32" href="#" role="doc-biblioref" data-xml-rid="bib32" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref32" aria-controls="sref32">Gel et&nbsp;al., 2016</a><div class="dropBlock__holder" data-db-target-of="sref32"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">30.</div><div>Gel, B. ∙ Díez-Villanueva, A. ∙ Serra, E. ...</div><div><strong>regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:289-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv562" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84959886262" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26424858/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv562&amp;pmid=26424858" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section></section><section id="sec-4-7"><h3>Data and Code Availability</h3><div id="p0865" role="paragraph">All data generated as part of the ASC is transferred to dbGaP with Study Accession: phs000298.v4.p3. Data generated previously are detailed in <a id="crosref1760" href="#mmc1">Table S1</a> and in the <a id="crosref1765" href="#sec-4-1">Key Resources Table</a>. TADA has been previously described (<span class="dropBlock reference-citations"><a id="body-ref-sref35-8" href="#" role="doc-biblioref" data-xml-rid="bib35" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref35-8" aria-controls="sref35-8">He et&nbsp;al., 2013</a><div class="dropBlock__holder" data-db-target-of="sref35-8"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">33.</div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>) and enhancements to TADA are described in detail in the main text and the <a id="crosref1775" href="#sec-4">STAR Methods</a>. DAWN has also been described (<span class="dropBlock reference-citations"><a id="body-ref-sref58-2" href="#" role="doc-biblioref" data-xml-rid="bib58" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref58-2" aria-controls="sref58-2">Liu et&nbsp;al., 2014</a><div class="dropBlock__holder" data-db-target-of="sref58-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">56.</div><div>Liu, L. ∙ Lei, J. ∙ Sanders, S.J. ...</div><div><strong>DAWN: a framework to identify autism genes and subnetworks using gene expression and genetics</strong></div><div><em>Mol. Autism.</em> 2014; <strong>5</strong>:22</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2040-2392-5-22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84899472749" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24602502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2040-2392-5-22&amp;pmid=24602502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>).</div></section></section><section id="sec-5"><h2>Consortia</h2><div id="p0870" role="paragraph">The members of the Autism Sequencing Consortium (ASC) are Branko Aleksic, Richard Anney, Mafalda Barbosa, Somer Bishop, Alfredo Brusco, Jonas Bybjerg-Grauholm, Angel Carracedo, Marcus C.Y. Chan, Andreas G. Chiocchetti, Brian H.Y. Chung, Hilary Coon, Michael L. Cuccaro, Aurora Curró, Bernardo Dalla Bernardina, Ryan Doan, Enrico Domenici, Shan Dong, Chiara Fallerini, Montserrat Fernández-Prieto, Giovanni Battista Ferrero, Christine M. Freitag, Menachem Fromer, J. Jay Gargus, Daniel Geschwind, Elisa Giorgio, Javier González-Peñas, Stephen Guter, Danielle Halpern, Emily Hansen-Kiss, Xin He, Gail E. Herman, Irva Hertz-Picciotto, David M. Hougaard, Christina M. Hultman, Iuliana Ionita-Laza, Suma Jacob, Jesslyn Jamison, Astanand Jugessur, Miia Kaartinen, Gun Peggy Knudsen, Alexander Kolevzon, Itaru Kushima, So Lun Lee, Terho Lehtimäki, Elaine T. Lim, Carla Lintas, W. Ian Lipkin, Diego Lopergolo, Fátima Lopes, Yunin Ludena, Patricia Maciel, Per Magnus, Behrang Mahjani, Nell Maltman, Dara S. Manoach, Gal Meiri, Idan Menashe, Judith Miller, Nancy Minshew, Eduarda M.S. Montenegro, Danielle Moreira, Eric M. Morrow, Ole Mors, Preben Bo Mortensen, Matthew Mosconi, Pierandrea Muglia, Benjamin M. Neale, Merete Nordentoft, Norio Ozaki, Aarno Palotie, Mara Parellada, Maria Rita Passos-Bueno, Margaret Pericak-Vance, Antonio M. Persico, Isaac Pessah, Kaija Puura, Abraham Reichenberg, Alessandra Renieri, Evelise Riberi, Elise B. Robinson, Kaitlin E. Samocha, Sven Sandin, Susan L. Santangelo, Gerry Schellenberg, Stephen W. Scherer, Sabine Schlitt, Rebecca Schmidt, Lauren Schmitt, Isabela M.W. Silva, Tarjinder Singh, Paige M. Siper, Moyra Smith, Gabriela Soares, Camilla Stoltenberg, Pål Suren, Ezra Susser, John Sweeney, Peter Szatmari, Lara Tang, Flora Tassone, Karoline Teufel, Elisabetta Trabetti, Maria del Pilar Trelles, Christopher A. Walsh, Lauren A. Weiss, Thomas Werge, Donna M. Werling, Emilie M. Wigdor, Emma Wilkinson, A. Jeremy Willsey, Timothy W. Yu, Mullin H.C. Yu, Ryan Yuen, and Elaine Zachi.</div><div id="p0875" role="paragraph">The members of the iPSYCH-Broad Consortium are Esben Agerbo, Thomas Damm Als, Vivek Appadurai, Marie Bækvad-Hansen, Rich Belliveau, Alfonso Buil, Caitlin E. Carey, Felecia Cerrato, Kimberly Chambert, Claire Churchhouse, Søren Dalsgaard, Ditte Demontis, Ashley Dumont, Jacqueline Goldstein, Christine S. Hansen, Mads Engel Hauberg, Mads V. Hollegaard, Daniel P. Howrigan, Hailiang Huang, Julian Maller, Alicia R. Martin, Joanna Martin, Manuel Mattheisen, Jennifer Moran, Jonatan Pallesen, Duncan S. Palmer, Carsten Bøcker Pedersen, Marianne Giørtz Pedersen, Timothy Poterba, Jesper Buchhave Poulsen, Stephan Ripke, Andrew J. Schork, Wesley K. Thompson, Patrick Turley, and Raymond K. Walters.</div></section><section id="acknowledgments" role="doc-acknowledgments"><h2>Acknowledgments</h2><div id="p0880" role="paragraph">We thank the families who participated in this research, without whose contributions genetic studies would be impossible. This study was supported by the AMED (JP19dm0107087 to B.A. and N.O.), Autism Science Foundation (to S.J.S., S.L.B., and E.B.R.), NHGRI (HG008895 to M.J.D. and HG002295 to R.C.), NIMH (MH111658 and MH057881 to B.D., MH111661 and MH100233-03S1 to J.D.B., R01 MH109900 to K.R., MH115957 to M.E.T., MH111660 to M.J.D., and MH111662 to S.J.S. and M.W.S.), NSF (GRFP 2017240332 to R.C.), the Seaver Foundation (to J.D.B. and S.D.R.), and the Simons Foundation (SF402281 to S.J.S., M.W.S., B.D., and K.R. and SF573206 to M.E.T.). Funding for individual cohorts is detailed further in the <a id="crosref1785" href="#sec-4">STAR Methods</a>. We thank Tom Nowakowski (UCSF) for facilitating access to the single-cell gene expression data.</div><section id="sec-6"><h3>Author Contributions</h3><div id="p0885" role="paragraph">Resources, C. Stevens, J.R., S. Gerges, G. Schwartz, R.N., E.E.G., C.D., B.A., M.B., A. Brusco, J.B., A. Carracedo, M.C.Y.C., A.G.C., B.H.Y.C., H.C., M.L.C., A. Currò, B.D.B., E.D., S.D., C.F., M. Fernández-Prieto, G.B.F., C.M.F., J.J.G., E.G., J. González-Peñas, D.H., E.H., G.E.H., I.H., D.M.H., C.M.H., I.I., S.J., J.J., A.J., M.K., G.P.K., A.K., I.K., S.L.L., T.L., E.T.L., C.L., W.I.L., D.L., F.L., P. Maciel, P. Magnus, D.S.M., G.M., I.M., J.M., N. Minshew, E.M.M.d.S., D.M., E.M.M., O.M., P.B.M., M.M., P. Muglia, B.N., M.N., N.O., A. Palotie, M. Parellada, M.R.P., M. Pericak-Vance, A. Persico, I.P., K.P., A. Reichenberg, A. Renieri, E.R., E.B.R., S. Sandin, G. Schellenberg, S.W.S., S. Schlitt, R.S., I.M.W.S., P.M.S., M.S., G. Soares, C. Stolenberg, P. Suren, E.S., J.S., P. Szatmari, F.T., K.T., E.T., M.d.P.T., C.A.W., L.A.W., T.W., D.W., E.W., J.A.W., T.W.Y., M.H.C.Y., R.Y., E.Z., E.H.C., J.S.S., A.D.B., M.E.T., S.J.S., M.J.D., and J.D.B.; Investigation, F.K.S., J.A.K., J.W., M.S.B., S.D.R., J.A., M. Peng, R.C., J. Grove, L.K., C. Stevens, J.R., M.S.M., M.A., B.S., X.X., A. Bhaduri, H.B., R.N., R.A., B.H.Y.C., R.D., C.M.F., M. Fromer, S. Guter, X.H., S.J., S.L.L., T.L., Y.L., B.M., N. Maltman, K.P., E.B.R., L.S., T.S., P.M.S., J.S., T.W.Y., C.B., E.H.C., M.E.Z., A.D.B., A.E.C., M.E.T., D.J.C., B.D., S.J.S., K.R., M.J.D., and J.D.B.; Data Curation: F.K.S., J.A.K., S.D.R., J.A., R.C., C. Stevens, M.S.M., B.S., A.G.C., S.D., C.M.F., S. Guter, K.E.S., E.M.W., E.H.C., S.J.S., and J.D.B.; Formal Analysis: F.K.S., J.A.K., J.W., M.S.B., J.A., R.C., J. Grove, L.K., A. Bhaduri, U.N., H.B., J.J.G., X.H., I.I., B.N., C.B., A.E.C., D.J.C., B.D., S.J.S., and K.R.; Visualization, J.A.K., J.W., M.S.B., J.A., R.C., A. Bhaduri, A.E.C., B.D., S.J.S., and K.R.; Writing – Original Draft, F.K.S., J.A.K., J.W., M.S.B., R.C., J.J.G., M.E.T., D.J.C., B.D., S.J.S., K.R., M.J.D., and J.D.B.; Writing – Review and Editing, F.K.S., M.S.B., S.D.R., J.A., R.C., J. Grove, A. Bhaduri, S.B., I.I., T.L., B.N., K.P., A. Renieri, C.A.W., D.W., T.W.Y., C.B., E.H.C., J.S.S., A.D.B., M.E.T., D.J.C., B.D., S.J.S., K.R., M.J.D., and J.D.B.; Funding Acquisition, C. Stevens, A. Brusco, C.M.F., D.G., N.O., J.S., C.A.W., J.A.W., M.E.Z., A.D.B., M.W.S., M.E.T., D.J.C., B.D., S.J.S., K.R., M.J.D., and J.D.B.; Project Administration, S.D.R., C. Stevens, J.R., A.G.C., P.M.S., J.S., L.T., C.A.W., C.B., E.H.C., L.G., M.G., J.S.S., A.T., M.E.Z., A.D.B., M.W.S., M.E.T., D.J.C., B.D., S.J.S., K.R., M.J.D., and J.D.B.</div></section><section id="sec-7"><h3>Declaration of Interests</h3><div id="p0890" role="paragraph">B.M.N. is a member of the scientific advisory board at Deep Genomics and consults for Biogen, Camp4 Therapeutics Corporation, Takeda Pharmaceutical, and Biogen. During the last 3 years, C.M. Freitag has been consultant to Desitin and Roche and receives royalties for books on ASD, ADHD, and MDD.</div></section></section><section id="supplementary-material" data-type="supplementary-materials"><h2>Supplemental Information (5)</h2><div id="p0900" role="paragraph"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div id="mmc1" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2019.12.036/attachment/44aca411-6be3-4158-b1d6-6b339d60136d/mmc1.xlsx" download="mmc1.xlsx" aria-labelledby="mmc1-heading mmc1-link"><span id="mmc1-link">Spreadsheet (5.83 MB)</span></a></div><div class="core-description"><div id="mmc1-heading" class="heading">Table S1. Cohorts, Samples, <i>De Novo</i> Variants, and Liability Analysis, Related to Figure&nbsp;1</div><div id="espara0010" role="paragraph">This file contains sheets that (1) list the cohorts that contributed samples to this study, (2) list the 35,584 samples used in the analysis, as well as basic sample information, (3) list the <i>de novo</i> variants called in the 6,430 individuals with ASD and 2,179 individuals without ASD (a total of 15,789 <i>de novo</i> variants were identified covering 9,345 coding variants and 6,444 non-coding variants), and (4) display the variant counts and calculations used in the liability analysis in Figure&nbsp;1 in the main manuscript.</div></div></div><div id="mmc2" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2019.12.036/attachment/cc634b34-b459-4aff-acf0-cb58a3013bd1/mmc2.xlsx" download="mmc2.xlsx" aria-labelledby="mmc2-heading mmc2-link"><span id="mmc2-link">Spreadsheet (5.94 MB)</span></a></div><div class="core-description"><div id="mmc2-heading" class="heading">Table S2. TADA FDR Values, Related to Figure&nbsp;2</div><div id="espara0015" role="paragraph">This file contains sheets that (1) list the association data for 17,484 autosomal genes along with TADA false discovery rates, including the 102 genes identified from TADA at an FDR&nbsp;&lt; 0.1 used for Figures 2B, 3A, 4A, and 4E and (2) list chromosome X gene-level p values.</div></div></div><div id="mmc3" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2019.12.036/attachment/712fc849-cd3e-4ec9-820a-c23443608a80/mmc3.xlsx" download="mmc3.xlsx" aria-labelledby="mmc3-heading mmc3-link"><span id="mmc3-link">Spreadsheet (174.55 KB)</span></a></div><div class="core-description"><div id="mmc3-heading" class="heading">Table S3. Missense Variants, Genomic Disorder Loci, and GWAS Enrichment Analysis, Related to Figure&nbsp;3</div><div id="espara0020" role="paragraph">This file contains sheets that (1) list <i>de novo</i> missense variants identified in <i>DEAF1</i>, <i>KCNQ3</i>, <i>SCN1A</i>, and <i>SLC6A1</i> in this and other studies, used to make Figures 3B–3E, (2) list GD loci and their overlap with ASD risk genes identified by TADA, used to make Figures 3F and 3G, and (3) show the gene set enrichment results from the analysis to determine if the 102 ASD-associated genes were enriched for common variants associated with ASD and genetically correlated traits, used to make Figures 3H and 3I.</div></div></div><div id="mmc4" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2019.12.036/attachment/45f1baed-a2ec-4128-be82-a50ae08adc38/mmc4.xlsx" download="mmc4.xlsx" aria-labelledby="mmc4-heading mmc4-link"><span id="mmc4-link">Spreadsheet (1.20 MB)</span></a></div><div class="core-description"><div id="mmc4-heading" class="heading">Table S4. Neurodevelopmental Delay Variants, ASD Phenotype Data, and Functional Categorization of ASD-Associated Genes, Related to Figure&nbsp;4</div><div id="espara0025" role="paragraph">This file contains sheets that (1) list 8,087 published <i>de novo</i> variants in 5,264 individuals ascertained for severe intellectual disability / developmental delay, used in Figure&nbsp;4A, (2) give gene-level results for heterogeneity tests between mutations in ascertained ASD and ascertained NDD in the 102 ASD genes, (3) list the IQ and age of walking phenotype data (when available) for the 6,430 individuals with ASD and 2,079 unaffected siblings used to construct Figures 4B–4D, and (4) show specific Gene Ontology terms attached to the ASD-associated genes and give our functional classification of each gene, used in Figures 4E, 5A, 5C, and S5.</div></div></div><div id="mmc5" class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="/cms/10.1016/j.cell.2019.12.036/attachment/0b26abc3-a77a-4f26-806f-109797aec47e/mmc5.xlsx" download="mmc5.xlsx" aria-labelledby="mmc5-heading mmc5-link"><span id="mmc5-link">Spreadsheet (8.42 MB)</span></a></div><div class="core-description"><div id="mmc5-heading" class="heading">Table S5. Integration of functional data, Related to Figure&nbsp;5</div><div id="espara0030" role="paragraph">This file contains sheets that (1) display information related to the GTEx and BrainSpan analyses shown in Figure&nbsp;5 and Figure&nbsp;S4, (2) display results from weighted gene coexpression network analysis (WGCNA), highlighting which genes are in which coexpression module, (3) show the genes expressed in each cell type cluster and the enrichment of ASD genes in each cluster, used in Figures 5D–5I, (4) contain full results for the Detecting Association With Networks (DAWN) analysis, which uses coexpression with known ASD-related genes to predict further ASD-related genes, used in Figure S5A, (5) list the protein-protein interactions depicted in Figure S5B, and (6) list the interactions displayed in Figure S5C.</div></div></div></div></section><section id="app-2"><h2>Supporting Citations</h2><div id="p0905" role="paragraph">The following reference appears in the Supplemental Information: <span class="dropBlock reference-citations"><a id="body-ref-sref53-2" href="#" role="doc-biblioref" data-xml-rid="bib53" href-manipulated="true" class="reference-citations__ctrl" aria-expanded="false" data-db-target-for="sref53-2" aria-controls="sref53-2">Li (2014)</a><div class="dropBlock__holder" data-db-target-of="sref53-2"><div class="dropBlock__body__outer"><div class="dropBlock__body"><div class="citation"><div class="citation-content"><div class="citation-content"><div class="label">51.</div><div>Li, H.</div><div><strong>Toward better understanding of artifacts in variant calling from high-coverage samples</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:2843-2851</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84913546864" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24974202/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu356&amp;pmid=24974202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><button class="dropBlock__ctrl--close" title="Close info" aria-label="close info"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 70 128" height="24" width="14"><path d="M68.94 36.12L62 29 35 56 8 29l-7 7 27 27L1 90l7 7 27-27 27 27 7-7-27-27 26.94-26.88"></path></svg></button></div></div></span>.</div></section></div></section><div class="core-linked-content">
    

        

</div><section id="backmatter" data-extent="backmatter"><div class="core-container"><section id="references"><h2>References</h2><div id="bibliography" role="doc-bibliography"><div role="list"><div role="listitem" data-has="label"><div id="bib1" class="citations"><div id="sref1" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref1-1" title="View in article">1.</a></div><div>Adzhubei, I.A. ∙ Schmidt, S. ∙ Peshkin, L. ...</div><div><strong>A method and server for predicting damaging missense mutations</strong></div><div><em>Nat. Methods.</em> 2010; <strong>7</strong>:248-249</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth0410-248" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_1_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77951640946" target="_blank" aria-label="Open Scopus in new tab">Scopus (6943)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20354512/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth0410-248&amp;pmid=20354512" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib2" class="citations"><div id="sref2" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref2" title="View in article">2.</a></div><div>Baio, J. ∙ Wiggins, L. ∙ Christensen, D.L. ...</div><div><strong>Prevalence of Autism Spectrum Disorder Among Children Aged 8 Years - Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2014</strong></div><div><em>MMWR Surveill Summ.</em> 2018; <strong>67</strong>:1-23</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15585/mmwr.ss6706a1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_2_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85047968753" target="_blank" aria-label="Open Scopus in new tab">Scopus (813)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29701730/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15585%2Fmmwr.ss6706a1&amp;pmid=29701730" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib3" class="citations"><div id="sref3" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref3-1" title="View in article">3.</a></div><div>Battle, A. ∙ Brown, C.D. ∙ Engelhardt, B.E. ..., GTEx Consortium, Laboratory, Data Analysis &amp;Coordinating Center (LDACC)—Analysis Working Group, Statistical Methods groups—Analysis Working Group, Enhancing GTEx (eGTEx) groups, NIH Common Fund, NIH/NCI, NIH/NHGRI, NIH/NIMH, NIH/NIDA, Biospecimen Collection Source Site—NDRI, Biospecimen Collection Source Site—RPCI, Biospecimen Core Resource—VARI, Brain Bank Repository—University of Miami Brain Endowment Bank, Leidos Biomedical—Project Management, ELSI Study, Genome Browser Data Integration &amp;Visualization—EBI, Genome Browser Data Integration &amp;Visualization—UCSC Genomics Institute, University of California Santa Cruz, Lead analysts, Laboratory, Data Analysis &amp;Coordinating Center (LDACC), NIH program management, Biospecimen collection, Pathology, eQTL manuscript working group</div><div><strong>Genetic effects on gene expression across human tissues</strong></div><div><em>Nature.</em> 2017; <strong>550</strong>:204-213</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature24277" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_3_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85031313737" target="_blank" aria-label="Open Scopus in new tab">Scopus (674)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature24277&amp;pmid=29022597" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib4" class="citations"><div id="sref4" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">4.</a></div><div>Ben-Shalom, R. ∙ Keeshen, C.M. ∙ Berrios, K.N. ...</div><div><strong>Opposing effects on NaV1.2 function underlie differences between SCN2A variants observed in individuals with autism spectrum disorder or infantile seizures</strong></div><div><em>Biol. Psychiatry.</em> 2017; <strong>82</strong>:224-232</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.01.009&amp;cf=fulltext&amp;site=bps-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.01.009&amp;cf=pdf&amp;site=bps-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_4_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85014039999" target="_blank" aria-label="Open Scopus in new tab">Scopus (51)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28256214/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biopsych.2017.01.009&amp;pmid=28256214" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib5" class="citations"><div id="sref5" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref5" title="View in article">5.</a></div><div>Bernier, R. ∙ Golzio, C. ∙ Xiong, B. ...</div><div><strong>Disruptive CHD8 mutations define a subtype of autism early in development</strong></div><div><em>Cell.</em> 2014; <strong>158</strong>:263-276</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2014.06.017&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2014.06.017&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_5_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84904635209" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24998929/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2014.06.017&amp;pmid=24998929" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib6" class="citations"><div id="sref6" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref6" title="View in article">6.</a></div><div>Birney, E. ∙ Stamatoyannopoulos, J.A. ∙ Dutta, A. ..., ENCODE Project Consortium, NISC Comparative Sequencing Program, Baylor College of Medicine Human Genome Sequencing Center, Washington University Genome Sequencing Center, Broad Institute, Children’s Hospital Oakland Research Institute</div><div><strong>Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</strong></div><div><em>Nature.</em> 2007; <strong>447</strong>:799-816</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature05874" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_6_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-34250305146" target="_blank" aria-label="Open Scopus in new tab">Scopus (3593)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17571346/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature05874&amp;pmid=17571346" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib7" class="citations"><div id="sref7" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">7.</a></div><div>Bottomley, M.J. ∙ Collard, M.W. ∙ Huggenvik, J.I. ...</div><div><strong>The SAND domain structure defines a novel DNA-binding fold in transcriptional regulation</strong></div><div><em>Nat. Struct. Biol.</em> 2001; <strong>8</strong>:626-633</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/89675" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_7_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-0034957535" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11427895/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2F89675&amp;pmid=11427895" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib8" class="citations"><div id="sref8" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref8" title="View in article">8.</a></div><div>Buxbaum, J.D. ∙ Daly, M.J. ∙ Devlin, B. ..., Autism Sequencing Consortium</div><div><strong>The autism sequencing consortium: large-scale, high-throughput sequencing in autism spectrum disorders</strong></div><div><em>Neuron.</em> 2012; <strong>76</strong>:1052-1056</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2012.12.008&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2012.12.008&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_8_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84871371945" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23259942/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2012.12.008&amp;pmid=23259942" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib9" class="citations"><div id="sref9" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">9.</a></div><div>Chang, J. ∙ Gilman, S.R. ∙ Chiang, A.H. ...</div><div><strong>Genotype to phenotype relationships in autism spectrum disorders</strong></div><div><em>Nat. Neurosci.</em> 2015; <strong>18</strong>:191-198</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.3907" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_9_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84923186910" target="_blank" aria-label="Open Scopus in new tab">Scopus (95)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25531569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.3907&amp;pmid=25531569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib10" class="citations"><div id="sref10" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref10" title="View in article">10.</a></div><div>Chaste, P. ∙ Klei, L. ∙ Sanders, S.J. ...</div><div><strong>A genome-wide association study of autism using the Simons Simplex Collection: Does reducing phenotypic heterogeneity in autism increase genetic homogeneity?</strong></div><div><em>Biol. Psychiatry.</em> 2015; <strong>77</strong>:775-784</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2014.09.017&amp;cf=fulltext&amp;site=bps-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2014.09.017&amp;cf=pdf&amp;site=bps-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_10_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84926358552" target="_blank" aria-label="Open Scopus in new tab">Scopus (70)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25534755/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biopsych.2014.09.017&amp;pmid=25534755" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib11" class="citations"><div id="sref11" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref11" title="View in article">11.</a></div><div>Chen, J. ∙ Bardes, E.E. ∙ Aronow, B.J. ...</div><div><strong>ToppGene Suite for gene list enrichment analysis and candidate gene prioritization</strong></div><div><em>Nucleic Acids Res.</em> 2009; <strong>37</strong></div><div>W305-11</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkp427" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_11_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-67849130563" target="_blank" aria-label="Open Scopus in new tab">Scopus (1034)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkp427" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib12" class="citations"><div id="sref12" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref12" title="View in article">12.</a></div><div>Chen, E.Y. ∙ Tan, C.M. ∙ Kou, Y. ...</div><div><strong>Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool</strong></div><div><em>BMC Bioinformatics.</em> 2013; <strong>14</strong>:128</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/1471-2105-14-128" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_12_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84876100615" target="_blank" aria-label="Open Scopus in new tab">Scopus (1156)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23586463/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F1471-2105-14-128&amp;pmid=23586463" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib13" class="citations"><div id="sref13" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">13.</a></div><div>Chen, L. ∙ Jensik, P.J. ∙ Alaimo, J.T. ...</div><div><strong>Functional analysis of novel DEAF1 variants identified through clinical exome sequencing expands DEAF1-associated neurodevelopmental disorder (DAND) phenotype</strong></div><div><em>Hum. Mutat.</em> 2017; <strong>38</strong>:1774-1785</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1002/humu.23339" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_13_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85030106221" target="_blank" aria-label="Open Scopus in new tab">Scopus (7)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28940898/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1002%2Fhumu.23339&amp;pmid=28940898" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib14" class="citations"><div id="sref14" class="citation"><div class="citation-content"><div class="label"><a href="#fig1" title="View in article">14.</a></div><div>Christensen, D.L. ∙ Baio, J. ∙ Van Naarden Braun, K. ..., Centers for Disease Control and Prevention (CDC)</div><div><strong>Prevalence and Characteristics of Autism Spectrum Disorder Among Children Aged 8 Years--Autism and Developmental Disabilities Monitoring Network, 11 Sites, United States, 2012</strong></div><div><em>MMWR Surveill. Summ.</em> 2016; <strong>65</strong>:1-23</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.15585/mmwr.ss6503a1" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27031587/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.15585%2Fmmwr.ss6503a1&amp;pmid=27031587" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib16" class="citations"><div id="sref16" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">15.</a></div><div>Coe, B.P. ∙ Witherspoon, K. ∙ Rosenfeld, J.A. ...</div><div><strong>Refining analyses of copy number variation identifies specific genes associated with developmental delay</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:1063-1071</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3092" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_15_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84922012902" target="_blank" aria-label="Open Scopus in new tab">Scopus (253)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25217958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3092&amp;pmid=25217958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib17" class="citations"><div id="sref17" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">16.</a></div><div>Cooper, G.M. ∙ Coe, B.P. ∙ Girirajan, S. ...</div><div><strong>A copy number variation morbidity map of developmental delay</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:838-846</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.909" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_16_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80052260252" target="_blank" aria-label="Open Scopus in new tab">Scopus (755)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21841781/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.909&amp;pmid=21841781" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib18" class="citations"><div id="sref18" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-5" title="View in article">17.</a></div><div>Cotney, J. ∙ Muhle, R.A. ∙ Sanders, S.J. ...</div><div><strong>The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment</strong></div><div><em>Nat. Commun.</em> 2015; <strong>6</strong>:6404</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncomms7404" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_17_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84924567722" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25752243/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncomms7404&amp;pmid=25752243" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib20" class="citations"><div id="sref20" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref20-1" title="View in article">18.</a></div><div>de Leeuw, C.A. ∙ Mooij, J.M. ∙ Heskes, T. ...</div><div><strong>MAGMA: generalized gene-set analysis of GWAS data</strong></div><div><em>PLoS Comput. Biol.</em> 2015; <strong>11</strong>:e1004219</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pcbi.1004219" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_18_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84929486000" target="_blank" aria-label="Open Scopus in new tab">Scopus (372)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25885710/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pcbi.1004219&amp;pmid=25885710" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib21" class="citations"><div id="sref21" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">19.</a></div><div>De Rubeis, S. ∙ He, X. ∙ Goldberg, A.P. ..., DDD Study, Homozygosity Mapping Collaborative for Autism, UK10K Consortium</div><div><strong>Synaptic, transcriptional and chromatin genes disrupted in autism</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:209-215</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13772" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_19_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912144889" target="_blank" aria-label="Open Scopus in new tab">Scopus (1001)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13772&amp;pmid=25363760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib22" class="citations"><div id="sref22" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref22-1" title="View in article">20.</a></div><div>Deciphering Developmental Disorders Study</div><div><strong>Prevalence and architecture of de novo mutations in developmental disorders</strong></div><div><em>Nature.</em> 2017; <strong>542</strong>:433-438</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature21062" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_20_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85015375341" target="_blank" aria-label="Open Scopus in new tab">Scopus (371)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28135719/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature21062&amp;pmid=28135719" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib23" class="citations"><div id="sref23" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">21.</a></div><div>Demontis, D. ∙ Walters, R.K. ∙ Martin, J. ..., ADHD Working Group of the Psychiatric Genomics Consortium (PGC), Early Lifecourse &amp; Genetic Epidemiology (EAGLE) Consortium, 23andMe Research Team</div><div><strong>Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:63-75</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0269-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_21_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85057436335" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30478444/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0269-7&amp;pmid=30478444" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib24" class="citations"><div id="sref24" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref24" title="View in article">22.</a></div><div>Devlin, B. ∙ Roeder, K.</div><div><strong>Genomic control for association studies</strong></div><div><em>Biometrics.</em> 1999; <strong>55</strong>:997-1004</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.0006-341X.1999.00997.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/11315092/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.0006-341X.1999.00997.x&amp;pmid=11315092" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib25" class="citations"><div id="sref25" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">23.</a></div><div>Dittwald, P. ∙ Gambin, T. ∙ Szafranski, P. ...</div><div><strong>NAHR-mediated copy-number variants in a clinical population: mechanistic insights into both genomic disorders and Mendelizing traits</strong></div><div><em>Genome Res.</em> 2013; <strong>23</strong>:1395-1409</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/gr.152454.112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_23_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84883679009" target="_blank" aria-label="Open Scopus in new tab">Scopus (76)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23657883/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgr.152454.112&amp;pmid=23657883" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib26" class="citations"><div id="sref26" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref26" title="View in article">24.</a></div><div>Efron, B.</div><div><strong>Large-scale inference: empirical Bayes methods for estimation, testing, and prediction</strong></div><div>Cambridge University Press, 2012</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=B.EfronLarge-scale+inference%3A+empirical+Bayes+methods+for+estimation%2C+testing%2C+and+prediction2012Cambridge+University+Press" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib27" class="citations"><div id="sref27" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref27" title="View in article">25.</a></div><div>Elliott, C.D.</div><div><strong>Differential Ability Scales</strong></div><div>Harcourt Assessment, 2007</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=C.D.ElliottDifferential+Ability+ScalesSecond+Edition2007Harcourt+Assessment" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib28" class="citations"><div id="sref28" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref28" title="View in article">26.</a></div><div>Falconer, D.S.</div><div><strong>The inheritance of liability to certain diseases, estimated from the incidence among relatives</strong></div><div><em>Ann. Hum. Genet.</em> 1965; <strong>29</strong>:51-76</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/j.1469-1809.1965.tb00500.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fj.1469-1809.1965.tb00500.x" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib29" class="citations"><div id="sref29" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">27.</a></div><div>Gandal, M.J. ∙ Haney, J.R. ∙ Parikshak, N.N. ..., CommonMind Consortium, PsychENCODE Consortium, iPSYCH-BROAD Working Group</div><div><strong>Shared molecular neuropathology across major psychiatric disorders parallels polygenic overlap</strong></div><div><em>Science.</em> 2018; <strong>359</strong>:693-697</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aad6469" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_27_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85041925301" target="_blank" aria-label="Open Scopus in new tab">Scopus (216)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29439242/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aad6469&amp;pmid=29439242" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib30" class="citations"><div id="sref30" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">28.</a></div><div>Gandal, M.J. ∙ Zhang, P. ∙ Hadjimichael, E. ...</div><div><strong>Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat8127</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat8127" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_28_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058422390" target="_blank" aria-label="Open Scopus in new tab">Scopus (79)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545856/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat8127&amp;pmid=30545856" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib31" class="citations"><div id="sref31" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref31" title="View in article">29.</a></div><div>Gaugler, T. ∙ Klei, L. ∙ Sanders, S.J. ...</div><div><strong>Most genetic risk for autism resides with common variation</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:881-885</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3039" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_29_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84905582433" target="_blank" aria-label="Open Scopus in new tab">Scopus (436)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25038753/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3039&amp;pmid=25038753" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib32" class="citations"><div id="sref32" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref32" title="View in article">30.</a></div><div>Gel, B. ∙ Díez-Villanueva, A. ∙ Serra, E. ...</div><div><strong>regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests</strong></div><div><em>Bioinformatics.</em> 2016; <strong>32</strong>:289-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btv562" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_30_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84959886262" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26424858/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtv562&amp;pmid=26424858" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib33" class="citations"><div id="sref33" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref33-1" title="View in article">31.</a></div><div>Grove, J. ∙ Ripke, S. ∙ Als, T.D. ..., Autism Spectrum Disorder Working Group of the Psychiatric Genomics Consortium, BUPGEN, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, 23andMe Research Team</div><div><strong>Identification of common genetic risk variants for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:431-444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-019-0344-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_31_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062110842" target="_blank" aria-label="Open Scopus in new tab">Scopus (115)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-019-0344-8&amp;pmid=30804558" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib34" class="citations"><div id="sref34" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref34" title="View in article">32.</a></div><div>Havrilla, J.M. ∙ Pedersen, B.S. ∙ Layer, R.M. ...</div><div><strong>A map of constrained coding regions in the human genome</strong></div><div><em>Nat. Genet.</em> 2019; <strong>51</strong>:88-95</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0294-6" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_32_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058142396" target="_blank" aria-label="Open Scopus in new tab">Scopus (20)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30531870/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0294-6&amp;pmid=30531870" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib35" class="citations"><div id="sref35" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref35-1" title="View in article">33.</a></div><div>He, X. ∙ Sanders, S.J. ∙ Liu, L. ...</div><div><strong>Integrated model of de novo and inherited genetic variants yields greater power to identify risk genes</strong></div><div><em>PLoS Genet.</em> 2013; <strong>9</strong>:e1003671</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1371/journal.pgen.1003671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_33_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84884620245" target="_blank" aria-label="Open Scopus in new tab">Scopus (108)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23966865/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1371%2Fjournal.pgen.1003671&amp;pmid=23966865" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib36" class="citations"><div id="sref36" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">34.</a></div><div>Heyne, H.O. ∙ Singh, T. ∙ Stamberger, H. ..., EuroEPINOMICS RES Consortium</div><div><strong>De novo variants in neurodevelopmental disorders with epilepsy</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1048-1053</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0143-7" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_34_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85049011462" target="_blank" aria-label="Open Scopus in new tab">Scopus (45)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29942082/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0143-7&amp;pmid=29942082" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib37" class="citations"><div id="sref37" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">35.</a></div><div>Huang, H.T. ∙ Kathrein, K.L. ∙ Barton, A. ...</div><div><strong>A network of epigenetic regulators guides developmental haematopoiesis in&nbsp;vivo</strong></div><div><em>Nat. Cell Biol.</em> 2013; <strong>15</strong>:1516-1525</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ncb2870" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24240475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fncb2870&amp;pmid=24240475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib38" class="citations"><div id="sref38" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">36.</a></div><div>Iossifov, I. ∙ O’Roak, B.J. ∙ Sanders, S.J. ...</div><div><strong>The contribution of de novo coding mutations to autism spectrum disorder</strong></div><div><em>Nature.</em> 2014; <strong>515</strong>:216-221</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13908" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_36_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84912101541" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13908&amp;pmid=25363768" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib39" class="citations"><div id="sref39" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref39" title="View in article">37.</a></div><div>Jensik, P.J. ∙ Huggenvik, J.I. ∙ Collard, M.W.</div><div><strong>Identification of a nuclear export signal and protein interaction domains in deformed epidermal autoregulatory factor-1 (DEAF-1)</strong></div><div><em>J.&nbsp;Biol. Chem.</em> 2004; <strong>279</strong>:32692-32699</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1074/jbc.M400946200" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_37_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-3543033387" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/15161925/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1074%2Fjbc.M400946200&amp;pmid=15161925" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib40" class="citations"><div id="sref40" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref40-1" title="View in article">38.</a></div><div>Jin, S.C. ∙ Homsy, J. ∙ Zaidi, S. ...</div><div><strong>Contribution of rare inherited and de novo variants in 2,871 congenital heart disease probands</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:1593-1601</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3970" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_38_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85032451853" target="_blank" aria-label="Open Scopus in new tab">Scopus (126)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28991257/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3970&amp;pmid=28991257" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib41" class="citations"><div id="sref41" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">39.</a></div><div>Johannesen, K.M. ∙ Gardella, E. ∙ Linnankivi, T. ...</div><div><strong>Defining the phenotypic spectrum of SLC6A1 mutations</strong></div><div><em>Epilepsia.</em> 2018; <strong>59</strong>:389-402</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1111/epi.13986" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_39_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85041378864" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29315614/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1111%2Fepi.13986&amp;pmid=29315614" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib42" class="citations"><div id="sref42" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref42" title="View in article">40.</a></div><div>Jun, G. ∙ Flickinger, M. ∙ Hetrick, K.N. ...</div><div><strong>Detecting and estimating contamination of human DNA samples in sequencing and array-based genotype data</strong></div><div><em>Am. J. Hum. Genet.</em> 2012; <strong>91</strong>:839-848</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2012.09.004&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2012.09.004&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_40_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84868490540" target="_blank" aria-label="Open Scopus in new tab">Scopus (134)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23103226/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2012.09.004&amp;pmid=23103226" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib43" class="citations"><div id="sref43" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref43" title="View in article">41.</a></div><div>Kang, H.J. ∙ Kawasawa, Y.I. ∙ Cheng, F. ...</div><div><strong>Spatio-temporal transcriptome of the human brain</strong></div><div><em>Nature.</em> 2011; <strong>478</strong>:483-489</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10523" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_41_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80054993342" target="_blank" aria-label="Open Scopus in new tab">Scopus (880)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22031440/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10523&amp;pmid=22031440" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib44" class="citations"><div id="sref44" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref44" title="View in article">42.</a></div><div>Karczewski, K.J. ∙ Francioli, L.C. ∙ Tiao, G. ...</div><div><strong>Variation across 141,456 human exomes and genomes reveals the spectrum of loss-of-function intolerance across human protein-coding genes</strong></div><div><em>bioRxiv.</em> 2019;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/531210" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_42_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85094417987" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F531210" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib45" class="citations"><div id="sref45" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref45" title="View in article">43.</a></div><div>Khare, S.P. ∙ Habib, F. ∙ Sharma, R. ...</div><div><strong>HIstome--a relational knowledgebase of human histone proteins and histone modifying enzymes</strong></div><div><em>Nucleic Acids Res.</em> 2012; <strong>40</strong>:D337-D342</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkr1125" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_43_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84862156464" target="_blank" aria-label="Open Scopus in new tab">Scopus (72)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22140112/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkr1125&amp;pmid=22140112" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib46" class="citations"><div id="sref46" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">44.</a></div><div>Kosmicki, J.A. ∙ Samocha, K.E. ∙ Howrigan, D.P. ...</div><div><strong>Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples</strong></div><div><em>Nat. Genet.</em> 2017; <strong>49</strong>:504-510</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3789" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_44_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85012277777" target="_blank" aria-label="Open Scopus in new tab">Scopus (97)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28191890/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3789&amp;pmid=28191890" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib47" class="citations"><div id="sref47" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref47" title="View in article">45.</a></div><div>Lachmann, A. ∙ Xu, H. ∙ Krishnan, J. ...</div><div><strong>ChEA: transcription factor regulation inferred from integrating genome-wide ChIP-X experiments</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2438-2444</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq466" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_45_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77957243164" target="_blank" aria-label="Open Scopus in new tab">Scopus (315)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20709693/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq466&amp;pmid=20709693" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib48" class="citations"><div id="sref48" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">46.</a></div><div>Landrum, M.J. ∙ Lee, J.M. ∙ Riley, G.R. ...</div><div><strong>ClinVar: public archive of relationships among sequence variation and human phenotype</strong></div><div><em>Nucleic Acids Res.</em> 2014; <strong>42</strong>:D980-D985</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkt1113" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_46_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84891809093" target="_blank" aria-label="Open Scopus in new tab">Scopus (959)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24234437/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkt1113&amp;pmid=24234437" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib49" class="citations"><div id="sref49" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-2" title="View in article">47.</a></div><div>Lauritsen, M.B. ∙ Jørgensen, M. ∙ Madsen, K.M. ...</div><div><strong>Validity of childhood autism in the Danish Psychiatric Central Register: findings from a cohort sample born 1990-1999</strong></div><div><em>J.&nbsp;Autism Dev. Disord.</em> 2010; <strong>40</strong>:139-148</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/s10803-009-0818-0" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_47_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-75949111242" target="_blank" aria-label="Open Scopus in new tab">Scopus (133)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19728067/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2Fs10803-009-0818-0&amp;pmid=19728067" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib50" class="citations"><div id="sref50" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">48.</a></div><div>Lee, J.J. ∙ Wedow, R. ∙ Okbay, A. ..., 23andMe Research Team, COGENT (Cognitive Genomics Consortium), Social Science Genetic Association Consortium</div><div><strong>Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:1112-1121</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0147-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_48_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85050543338" target="_blank" aria-label="Open Scopus in new tab">Scopus (210)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30038396/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0147-3&amp;pmid=30038396" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib51" class="citations"><div id="sref51" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref51-1" title="View in article">49.</a></div><div>Lek, M. ∙ Karczewski, K.J. ∙ Minikel, E.V. ..., Exome Aggregation Consortium</div><div><strong>Analysis of protein-coding genetic variation in 60,706 humans</strong></div><div><em>Nature.</em> 2016; <strong>536</strong>:285-291</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature19057" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature19057&amp;pmid=27535533" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib52" class="citations"><div id="sref52" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">50.</a></div><div>Leroy, C. ∙ Landais, E. ∙ Briault, S. ...</div><div><strong>The 2q37-deletion syndrome: an update of the clinical spectrum including overweight, brachydactyly and behavioural features in 14 new patients</strong></div><div><em>Eur. J. Hum. Genet.</em> 2013; <strong>21</strong>:602-612</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ejhg.2012.230" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_50_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84867413401" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23073310/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fejhg.2012.230&amp;pmid=23073310" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib53" class="citations"><div id="sref53" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">51.</a></div><div>Li, H.</div><div><strong>Toward better understanding of artifacts in variant calling from high-coverage samples</strong></div><div><em>Bioinformatics.</em> 2014; <strong>30</strong>:2843-2851</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btu356" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_51_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84913546864" target="_blank" aria-label="Open Scopus in new tab">Scopus (272)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24974202/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtu356&amp;pmid=24974202" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib54" class="citations"><div id="sref54" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref54" title="View in article">52.</a></div><div>Li, H. ∙ Durbin, R.</div><div><strong>Fast and accurate short read alignment with Burrows-Wheeler transform</strong></div><div><em>Bioinformatics.</em> 2009; <strong>25</strong>:1754-1760</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btp324" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_52_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-67649884743" target="_blank" aria-label="Open Scopus in new tab">Scopus (15889)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtp324&amp;pmid=19451168" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib55" class="citations"><div id="sref55" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref55-1" title="View in article">53.</a></div><div>Li, T. ∙ Wernersson, R. ∙ Hansen, R.B. ...</div><div><strong>A scored human protein-protein interaction network to catalyze genomic interpretation</strong></div><div><em>Nat. Methods.</em> 2017; <strong>14</strong>:61-64</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nmeth.4083" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_53_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84997794652" target="_blank" aria-label="Open Scopus in new tab">Scopus (235)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27892958/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnmeth.4083&amp;pmid=27892958" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib56" class="citations"><div id="sref56" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref56-1" title="View in article">54.</a></div><div>Li, M. ∙ Santpere, G. ∙ Imamura Kawasawa, Y. ..., BrainSpan Consortium, PsychENCODE Consortium, PsychENCODE Developmental Subgroup</div><div><strong>Integrative functional genomic analysis of human brain development and neuropsychiatric risks</strong></div><div><em>Science.</em> 2018; <strong>362</strong>:eaat7615</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aat7615" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_54_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85058377382" target="_blank" aria-label="Open Scopus in new tab">Scopus (58)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30545854/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aat7615&amp;pmid=30545854" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib57" class="citations"><div id="sref57" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref57" title="View in article">55.</a></div><div>Lim, E.T. ∙ Uddin, M. ∙ De Rubeis, S. ..., Autism Sequencing Consortium</div><div><strong>Rates, distribution and implications of postzygotic mosaic mutations in autism spectrum disorder</strong></div><div><em>Nat. Neurosci.</em> 2017; <strong>20</strong>:1217-1224</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nn.4598" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_55_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85028454831" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28714951/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnn.4598&amp;pmid=28714951" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib58" class="citations"><div id="sref58" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">56.</a></div><div>Liu, L. ∙ Lei, J. ∙ Sanders, S.J. ...</div><div><strong>DAWN: a framework to identify autism genes and subnetworks using gene expression and genetics</strong></div><div><em>Mol. Autism.</em> 2014; <strong>5</strong>:22</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2040-2392-5-22" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_56_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84899472749" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24602502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2040-2392-5-22&amp;pmid=24602502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib59" class="citations"><div id="sref59" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref59" title="View in article">57.</a></div><div>Liu, L. ∙ Lei, J. ∙ Roeder, K.</div><div><strong>Network assisted analysis to reveal the genetic basis of autism</strong></div><div><em>Ann. Appl. Stat.</em> 2015; <strong>9</strong>:1571-1600</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1214/15-AOAS844" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_57_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84947255657" target="_blank" aria-label="Open Scopus in new tab">Scopus (14)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27134692/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1214%2F15-AOAS844&amp;pmid=27134692" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib60" class="citations"><div id="sref60" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref60" title="View in article">58.</a></div><div>Lord, C. ∙ Rutter, M. ∙ Le Couteur, A.</div><div><strong>Autism Diagnostic Interview-Revised: a revised version of a diagnostic interview for caregivers of individuals with possible pervasive developmental disorders</strong></div><div><em>J.&nbsp;Autism Dev. Disord.</em> 1994; <strong>24</strong>:659-685</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1007/BF02172145" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_58_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-0027997172" target="_blank" aria-label="Open Scopus in new tab">Scopus (5772)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/7814313/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1007%2FBF02172145&amp;pmid=7814313" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib61" class="citations"><div id="sref61" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">59.</a></div><div>Maljevic, S. ∙ Vejzovic, S. ∙ Bernhard, M.K. ...</div><div><strong>Novel KCNQ3 mutation in a large family with benign familial neonatal epilepsy: A rare cause of neonatal seizures</strong></div><div><em>Mol. Syndromol.</em> 2016; <strong>7</strong>:189-196</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1159/000447461" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_59_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84978056091" target="_blank" aria-label="Open Scopus in new tab">Scopus (4)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27781029/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1159%2F000447461&amp;pmid=27781029" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib62" class="citations"><div id="sref62" class="citation"><div class="citation-content"><div class="label"><a href="#figs1" title="View in article">60.</a></div><div>Manichaikul, A. ∙ Mychaleckyj, J.C. ∙ Rich, S.S. ...</div><div><strong>Robust relationship inference in genome-wide association studies</strong></div><div><em>Bioinformatics.</em> 2010; <strong>26</strong>:2867-2873</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btq559" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_60_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-78149265272" target="_blank" aria-label="Open Scopus in new tab">Scopus (539)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20926424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtq559&amp;pmid=20926424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib63" class="citations"><div id="sref63" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref63-1" title="View in article">61.</a></div><div>McLaren, W. ∙ Gil, L. ∙ Hunt, S.E. ...</div><div><strong>The Ensembl Variant Effect Predictor</strong></div><div><em>Genome Biol.</em> 2016; <strong>17</strong>:122</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13059-016-0974-4" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_61_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84973338712" target="_blank" aria-label="Open Scopus in new tab">Scopus (900)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27268795/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13059-016-0974-4&amp;pmid=27268795" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib64" class="citations"><div id="sref64" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref64" title="View in article">62.</a></div><div>Mi, H. ∙ Muruganujan, A. ∙ Huang, X. ...</div><div><strong>Protocol Update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0)</strong></div><div><em>Nat. Protoc.</em> 2019; <strong>14</strong>:703-721</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41596-019-0128-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_62_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85062109631" target="_blank" aria-label="Open Scopus in new tab">Scopus (48)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30804569/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41596-019-0128-8&amp;pmid=30804569" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib65" class="citations"><div id="sref65" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">63.</a></div><div>Miceli, F. ∙ Soldovieri, M.V. ∙ Ambrosino, P. ...</div><div><strong>Early-onset epileptic encephalopathy caused by gain-of-function mutations in the voltage sensor of Kv7.2 and Kv7.3 potassium channel subunits</strong></div><div><em>J.&nbsp;Neurosci.</em> 2015; <strong>35</strong>:3782-3793</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.4423-14.2015" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_63_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84924787957" target="_blank" aria-label="Open Scopus in new tab">Scopus (67)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25740509/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.4423-14.2015&amp;pmid=25740509" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib66" class="citations"><div id="sref66" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref66" title="View in article">64.</a></div><div>Morton, N.E. ∙ MacLean, C.J.</div><div><strong>Analysis of family resemblance. 3. Complex segregation of quantitative traits</strong></div><div><em>Am. J. Hum. Genet.</em> 1974; <strong>26</strong>:489-503</div></div><div class="external-links"><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/4842773/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?pmid=4842773" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib67" class="citations"><div id="sref67" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref67" title="View in article">65.</a></div><div>Mullen, E.M.</div><div><strong>Mullen Scales of Early Learning Manual</strong></div><div>American Guidance Service, 1995</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=E.M.MullenMullen+Scales+of+Early+Learning+Manual1995American+Guidance+Service" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib68" class="citations"><div id="sref68" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">66.</a></div><div>Neale, B.M. ∙ Medland, S.E. ∙ Ripke, S. ..., Psychiatric GWAS Consortium: ADHD Subgroup</div><div><strong>Meta-analysis of genome-wide association studies of attention-deficit/hyperactivity disorder</strong></div><div><em>J.&nbsp;Am. Acad. Child Adolesc. Psychiatry.</em> 2010; <strong>49</strong>:884-897</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.jaac.2010.06.008&amp;cf=fulltext&amp;site=jaac-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.jaac.2010.06.008&amp;cf=pdf&amp;site=jaac-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_66_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77956178166" target="_blank" aria-label="Open Scopus in new tab">Scopus (291)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20732625/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.jaac.2010.06.008&amp;pmid=20732625" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib69" class="citations"><div id="sref69" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">67.</a></div><div>Neale, B.M. ∙ Kou, Y. ∙ Liu, L. ...</div><div><strong>Patterns and rates of exonic de novo mutations in autism spectrum disorders</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:242-245</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature11011" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_67_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84860712363" target="_blank" aria-label="Open Scopus in new tab">Scopus (1063)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495311/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature11011&amp;pmid=22495311" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib70" class="citations"><div id="sref70" class="citation"><div class="citation-content"><div class="label"><a href="#fig5" title="View in article">68.</a></div><div>Nowakowski, T.J. ∙ Bhaduri, A. ∙ Pollen, A.A. ...</div><div><strong>Spatiotemporal gene expression trajectories reveal developmental hierarchies of the human cortex</strong></div><div><em>Science.</em> 2017; <strong>358</strong>:1318-1323</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.aap8809" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_68_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85037631415" target="_blank" aria-label="Open Scopus in new tab">Scopus (120)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29217575/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aap8809&amp;pmid=29217575" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib71" class="citations"><div id="sref71" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref71-1" title="View in article">69.</a></div><div>O’Leary, N.A. ∙ Wright, M.W. ∙ Brister, J.R. ...</div><div><strong>Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</strong></div><div><em>Nucleic Acids Res.</em> 2016; <strong>44</strong>:D733-D745</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/nar/gkv1189" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fnar%2Fgkv1189&amp;pmid=26553804" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib72" class="citations"><div id="sref72" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">70.</a></div><div>Okbay, A. ∙ Beauchamp, J.P. ∙ Fontana, M.A. ..., LifeLines Cohort Study</div><div><strong>Genome-wide association study identifies 74 loci associated with educational attainment</strong></div><div><em>Nature.</em> 2016; <strong>533</strong>:539-542</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature17671" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27225129/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature17671&amp;pmid=27225129" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib73" class="citations"><div id="sref73" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">71.</a></div><div>Parikshak, N.N. ∙ Luo, R. ∙ Zhang, A. ...</div><div><strong>Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:1008-1021</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.031&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_71_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889561601" target="_blank" aria-label="Open Scopus in new tab">Scopus (434)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267887/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.031&amp;pmid=24267887" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib74" class="citations"><div id="sref74" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-2" title="View in article">72.</a></div><div>Pedersen, C.B. ∙ Bybjerg-Grauholm, J. ∙ Pedersen, M.G. ...</div><div><strong>The iPSYCH2012 case-cohort sample: new directions for unravelling genetic and environmental architectures of severe mental disorders</strong></div><div><em>Mol. Psychiatry.</em> 2018; <strong>23</strong>:6-14</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2017.196" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_72_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85040051369" target="_blank" aria-label="Open Scopus in new tab">Scopus (54)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/28924187/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2017.196&amp;pmid=28924187" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib75" class="citations"><div id="sref75" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref75-1" title="View in article">73.</a></div><div>Pinto, D. ∙ Pagnamenta, A.T. ∙ Klei, L. ...</div><div><strong>Functional impact of global rare copy number variation in autism spectrum disorders</strong></div><div><em>Nature.</em> 2010; <strong>466</strong>:368-372</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature09146" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_73_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77954657070" target="_blank" aria-label="Open Scopus in new tab">Scopus (1285)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20531469/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature09146&amp;pmid=20531469" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib77" class="citations"><div id="sref77" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref77" title="View in article">74.</a></div><div>Polioudakis, D. ∙ de la Torre-Ubieta, L. ∙ Langerman, J. ...</div><div><strong>A Single-Cell Transcriptomic Atlas of Human Neocortical Development during Mid-gestation</strong></div><div><em>Neuron.</em> 2019; <strong>103</strong>:785-801.e8</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2019.06.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_74_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85070182242" target="_blank" aria-label="Open Scopus in new tab">Scopus (6)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31303374/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2019.06.011&amp;pmid=31303374" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib78" class="citations"><div id="sref78" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref78" title="View in article">75.</a></div><div>Power, R.A. ∙ Kyaga, S. ∙ Uher, R. ...</div><div><strong>Fecundity of patients with schizophrenia, autism, bipolar disorder, depression, anorexia nervosa, or substance abuse vs their unaffected siblings</strong></div><div><em>JAMA Psychiatry.</em> 2013; <strong>70</strong>:22-30</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1001/jamapsychiatry.2013.268" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_75_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84871964302" target="_blank" aria-label="Open Scopus in new tab">Scopus (151)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23147713/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1001%2Fjamapsychiatry.2013.268&amp;pmid=23147713" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib79" class="citations"><div id="sref79" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">76.</a></div><div>Rehm, H.L. ∙ Berg, J.S. ∙ Brooks, L.D. ..., ClinGen</div><div><strong>ClinGen--the Clinical Genome Resource</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2015; <strong>372</strong>:2235-2242</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMsr1406261" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_76_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84930526399" target="_blank" aria-label="Open Scopus in new tab">Scopus (356)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26014595/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMsr1406261&amp;pmid=26014595" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib80" class="citations"><div id="sref80" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref80" title="View in article">77.</a></div><div>Reichenberg, A. ∙ Cederlöf, M. ∙ McMillan, A. ...</div><div><strong>Discontinuity in the genetic and environmental causes of the intellectual disability spectrum</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2016; <strong>113</strong>:1098-1103</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1508093112" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_77_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84955494277" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26711998/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1508093112&amp;pmid=26711998" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib81" class="citations"><div id="sref81" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">78.</a></div><div>Rietveld, C.A. ∙ Medland, S.E. ∙ Derringer, J. ..., LifeLines Cohort Study</div><div><strong>GWAS of 126,559 individuals identifies genetic variants associated with educational attainment</strong></div><div><em>Science.</em> 2013; <strong>340</strong>:1467-1471</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1235488" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_78_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84879271608" target="_blank" aria-label="Open Scopus in new tab">Scopus (379)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23722424/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1235488&amp;pmid=23722424" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib82" class="citations"><div id="sref82" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">79.</a></div><div>Ripke, S. ∙ Sanders, A.R. ∙ Kendler, K.S. ..., Schizophrenia Psychiatric Genome-Wide Association Study (GWAS) Consortium</div><div><strong>Genome-wide association study identifies five new schizophrenia loci</strong></div><div><em>Nat. Genet.</em> 2011; <strong>43</strong>:969-976</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.940" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_79_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-80053384370" target="_blank" aria-label="Open Scopus in new tab">Scopus (1244)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21926974/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.940&amp;pmid=21926974" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib83" class="citations"><div id="sref83" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">80.</a></div><div>Ripke, S. ∙ O’Dushlaine, C. ∙ Chambert, K. ..., Multicenter Genetic Studies of Schizophrenia Consortium, Psychosis Endophenotypes International Consortium, Wellcome Trust Case Control Consortium 2</div><div><strong>Genome-wide association analysis identifies 13 new risk loci for schizophrenia</strong></div><div><em>Nat. Genet.</em> 2013; <strong>45</strong>:1150-1159</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.2742" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_80_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84885020424" target="_blank" aria-label="Open Scopus in new tab">Scopus (884)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23974872/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.2742&amp;pmid=23974872" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib84" class="citations"><div id="sref84" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">81.</a></div><div>Ripke, S. ∙ Wray, N.R. ∙ Lewis, C.M. ..., Major Depressive Disorder Working Group of the Psychiatric GWAS Consortium</div><div><strong>A mega-analysis of genome-wide association studies for major depressive disorder</strong></div><div><em>Mol. Psychiatry.</em> 2013; <strong>18</strong>:497-511</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/mp.2012.21" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_81_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84879892380" target="_blank" aria-label="Open Scopus in new tab">Scopus (635)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22472876/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fmp.2012.21&amp;pmid=22472876" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib85" class="citations"><div id="sref85" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref85" title="View in article">82.</a></div><div>Robinson, E.B. ∙ Samocha, K.E. ∙ Kosmicki, J.A. ...</div><div><strong>Autism spectrum disorder severity reflects the average contribution of de novo and familial influences</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:15161-15165</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1409204111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_82_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84908030450" target="_blank" aria-label="Open Scopus in new tab">Scopus (59)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25288738/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1409204111&amp;pmid=25288738" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib87" class="citations"><div id="sref87" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref87" title="View in article">83.</a></div><div>Rubenstein, J.L. ∙ Merzenich, M.M.</div><div><strong>Model of autism: increased ratio of excitation/inhibition in key neural systems</strong></div><div><em>Genes Brain Behav.</em> 2003; <strong>2</strong>:255-267</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1034/j.1601-183X.2003.00037.x" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_83_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-0242291090" target="_blank" aria-label="Open Scopus in new tab">Scopus (1284)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/14606691/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1034%2Fj.1601-183X.2003.00037.x&amp;pmid=14606691" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib88" class="citations"><div id="sref88" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">84.</a></div><div>Ruzzo, E.K. ∙ Pérez-Cano, L. ∙ Jung, J.Y. ...</div><div><strong>Inherited and de novo genetic risk for autism impacts shared networks</strong></div><div><em>Cell.</em> 2019; <strong>178</strong>:850-866.e26</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2019.07.015&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2019.07.015&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_84_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85070184891" target="_blank" aria-label="Open Scopus in new tab">Scopus (10)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/31398340/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2019.07.015&amp;pmid=31398340" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib89" class="citations"><div id="sref117" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">85.</a></div><div>Samocha, K.E. ∙ Robinson, E.B. ∙ Sanders, S.J. ...</div><div><strong>A framework for the interpretation of de novo mutation in human disease</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:944-950</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3050" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_85_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84905309176" target="_blank" aria-label="Open Scopus in new tab">Scopus (425)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25086666/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3050&amp;pmid=25086666" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib90" class="citations"><div id="sref90" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref90-1" title="View in article">86.</a></div><div>Samocha, K.E. ∙ Kosmicki, J.A. ∙ Karczewski, K.J. ...</div><div><strong>Regional missense constraint improves variant deleteriousness prediction</strong></div><div><em>bioRxiv.</em> 2017;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/148353" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_86_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85093588144" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F148353" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib91" class="citations"><div id="sref91" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref91" title="View in article">87.</a></div><div>Sanders, S.J. ∙ Murtha, M.T. ∙ Gupta, A.R. ...</div><div><strong>De novo mutations revealed by whole-exome sequencing are strongly associated with autism</strong></div><div><em>Nature.</em> 2012; <strong>485</strong>:237-241</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10945" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_87_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84860780495" target="_blank" aria-label="Open Scopus in new tab">Scopus (1148)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/22495306/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10945&amp;pmid=22495306" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib92" class="citations"><div id="sref92" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref92-1" title="View in article">88.</a></div><div>Sanders, S.J. ∙ He, X. ∙ Willsey, A.J. ..., Autism Sequencing Consortium</div><div><strong>Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci</strong></div><div><em>Neuron.</em> 2015; <strong>87</strong>:1215-1233</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.neuron.2015.09.016&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_88_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84942113437" target="_blank" aria-label="Open Scopus in new tab">Scopus (429)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.neuron.2015.09.016&amp;pmid=26402605" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib93" class="citations"><div id="sref93" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref93-1" title="View in article">89.</a></div><div>Satterstrom, F.K. ∙ Walters, R.K. ∙ Singh, T. ...</div><div><strong>ASD and ADHD have a similar burden of rare protein-truncating variants</strong></div><div><em>bioRxiv.</em> 2018;</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1101/277707" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_89_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85095617614" target="_blank" aria-label="Open Scopus in new tab">Scopus (0)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1101%2F277707" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib94" class="citations"><div id="sref94" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">90.</a></div><div>Schaefer, G.B. ∙ Mendelsohn, N.J., Professional Practice and Guidelines Committee</div><div><strong>Clinical genetics evaluation in identifying the etiology of autism spectrum disorders: 2013 guideline revisions</strong></div><div><em>Genet. Med.</em> 2013; <strong>15</strong>:399-407</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1038%2Fgim.2013.32&amp;cf=fulltext&amp;site=gim-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1038%2Fgim.2013.32&amp;cf=pdf&amp;site=gim-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_90_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84877264570" target="_blank" aria-label="Open Scopus in new tab">Scopus (180)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23519317/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fgim.2013.32&amp;pmid=23519317" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib95" class="citations"><div id="sref95" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">91.</a></div><div>Schizophrenia Working Group of the Psychiatric Genomics Consortium</div><div><strong>Biological insights from 108 schizophrenia-associated genetic loci</strong></div><div><em>Nature.</em> 2014; <strong>511</strong>:421-427</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature13595" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_91_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84904804929" target="_blank" aria-label="Open Scopus in new tab">Scopus (3226)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25056061/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature13595&amp;pmid=25056061" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib96" class="citations"><div id="sref96" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref96" title="View in article">92.</a></div><div>Sebat, J. ∙ Lakshmi, B. ∙ Malhotra, D. ...</div><div><strong>Strong association of de novo copy number mutations with autism</strong></div><div><em>Science.</em> 2007; <strong>316</strong>:445-449</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1126/science.1138659" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_92_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-34247481814" target="_blank" aria-label="Open Scopus in new tab">Scopus (1835)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/17363630/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.1138659&amp;pmid=17363630" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib97" class="citations"><div id="sref97" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref97" title="View in article">93.</a></div><div>Soorya, L. ∙ Kolevzon, A. ∙ Zweifach, J. ...</div><div><strong>Prospective investigation of autism and genotype-phenotype correlations in 22q13 deletion syndrome and SHANK3 deficiency</strong></div><div><em>Mol. Autism.</em> 2013; <strong>4</strong>:18</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/2040-2392-4-18" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_93_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84878743755" target="_blank" aria-label="Open Scopus in new tab">Scopus (118)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23758760/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2F2040-2392-4-18&amp;pmid=23758760" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib98" class="citations"><div id="sref98" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref98" title="View in article">94.</a></div><div>Staples, J. ∙ Qiao, D. ∙ Cho, M.H. ..., University of Washington Center for Mendelian Genomics</div><div><strong>PRIMUS: rapid reconstruction of pedigrees from genome-wide estimates of identity by descent</strong></div><div><em>Am. J. Hum. Genet.</em> 2014; <strong>95</strong>:553-564</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.10.005&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_94_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.10.005&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25439724/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2014.10.005&amp;pmid=25439724" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib99" class="citations"><div id="sref99" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref99" title="View in article">95.</a></div><div>Sugathan, A. ∙ Biagioli, M. ∙ Golzio, C. ...</div><div><strong>CHD8 regulates neurodevelopmental pathways associated with autism spectrum disorder in neural progenitors</strong></div><div><em>Proc. Natl. Acad. Sci. USA.</em> 2014; <strong>111</strong>:E4468-E4477</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1073/pnas.1405266111" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_95_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84908065133" target="_blank" aria-label="Open Scopus in new tab">Scopus (125)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25294932/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.1405266111&amp;pmid=25294932" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib100" class="citations"><div id="sref100" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref100-1" title="View in article">96.</a></div><div>Van der Auwera, G.A. ∙ Carneiro, M.O. ∙ Hartl, C. ...</div><div><strong>From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline</strong></div><div><em>Curr. Protoc. Bioinformatics.</em> 2013; <strong>11</strong>:11.10.1-11.10.33</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=G.A.Van+der+AuweraM.O.CarneiroC.HartlR.PoplinG.Del+AngelA.Levy-MoonshineT.JordanK.ShakirD.RoazenJ.ThibaultFrom+FastQ+data+to+high+confidence+variant+calls%3A+the+Genome+Analysis+Toolkit+best+practices+pipelineCurr.+Protoc.+Bioinformatics11201311.10.111.10.33" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib101" class="citations"><div id="sref101" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref101" title="View in article">97.</a></div><div>Voineagu, I. ∙ Wang, X. ∙ Johnston, P. ...</div><div><strong>Transcriptomic analysis of autistic brain reveals convergent molecular pathology</strong></div><div><em>Nature.</em> 2011; <strong>474</strong>:380-384</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/nature10110" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_97_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-79959262465" target="_blank" aria-label="Open Scopus in new tab">Scopus (962)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/21614001/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fnature10110&amp;pmid=21614001" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib102" class="citations"><div id="sref102" class="citation"><div class="citation-content"><div class="label"><a href="#fig3" title="View in article">98.</a></div><div>Vulto-van Silfhout, A.T. ∙ Rajamanickam, S. ∙ Jensik, P.J. ...</div><div><strong>Mutations affecting the SAND domain of DEAF1 cause intellectual disability with severe speech impairment and behavioral problems</strong></div><div><em>Am. J. Hum. Genet.</em> 2014; <strong>94</strong>:649-661</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.03.013&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2014.03.013&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_98_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84899899835" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24726472/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2014.03.013&amp;pmid=24726472" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib103" class="citations"><div id="sref103" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">99.</a></div><div>Wapner, R.J. ∙ Martin, C.L. ∙ Levy, B. ...</div><div><strong>Chromosomal microarray versus karyotyping for prenatal diagnosis</strong></div><div><em>N.&nbsp;Engl. J. Med.</em> 2012; <strong>367</strong>:2175-2184</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1056/NEJMoa1203382" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_99_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84870549609" target="_blank" aria-label="Open Scopus in new tab">Scopus (589)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/23215555/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1056%2FNEJMoa1203382&amp;pmid=23215555" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib104" class="citations"><div id="sref104" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref104" title="View in article">100.</a></div><div>Wechsler, D.</div><div><strong>WISC III (Wechsler Intelligence Scale for Children)</strong></div><div>The Psychological Corporation, 1992</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.WechslerWISC+III+%28Wechsler+Intelligence+Scale+for+Children%291992The+Psychological+Corporation" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib105" class="citations"><div id="sref105" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref105" title="View in article">101.</a></div><div>Wechsler, D.</div><div><strong>Wechsler Abbreviated Scale of Intelligence</strong></div><div>The Psychological Corporation, 1999</div></div><div class="external-links"><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar?q=D.WechslerWechsler+Abbreviated+Scale+of+Intelligence1999The+Psychological+Corporation" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib106" class="citations"><div id="sref106" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref106" title="View in article">102.</a></div><div>Werling, D.M.</div><div><strong>The role of sex-differential biology in risk for autism spectrum disorder</strong></div><div><em>Biol. Sex Differ.</em> 2016; <strong>7</strong>:58</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1186/s13293-016-0112-8" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_102_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84995459627" target="_blank" aria-label="Open Scopus in new tab">Scopus (28)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27891212/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1186%2Fs13293-016-0112-8&amp;pmid=27891212" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib107" class="citations"><div id="sref118" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref118" title="View in article">103.</a></div><div>Werling, D.M. ∙ Brand, H. ∙ An, J.Y. ...</div><div><strong>An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder</strong></div><div><em>Nat. Genet.</em> 2018; <strong>26</strong>:727-736</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0107-y" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_103_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85046026156" target="_blank" aria-label="Open Scopus in new tab">Scopus (38)</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0107-y" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib108" class="citations"><div id="sref108" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref108" title="View in article">104.</a></div><div>Williams, S.R. ∙ Aldred, M.A. ∙ Der Kaloustian, V.M. ...</div><div><strong>Haploinsufficiency of HDAC4 causes brachydactyly mental retardation syndrome, with brachydactyly type E, developmental delays, and behavioral problems</strong></div><div><em>Am. J. Hum. Genet.</em> 2010; <strong>87</strong>:219-228</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2010.07.011&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.ajhg.2010.07.011&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_104_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-77955584378" target="_blank" aria-label="Open Scopus in new tab">Scopus (188)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/20691407/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.ajhg.2010.07.011&amp;pmid=20691407" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib109" class="citations"><div id="sref109" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref109-1" title="View in article">105.</a></div><div>Willsey, A.J. ∙ Sanders, S.J. ∙ Li, M. ...</div><div><strong>Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism</strong></div><div><em>Cell.</em> 2013; <strong>155</strong>:997-1007</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=fulltext&amp;site=cell-site">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.cell.2013.10.020&amp;cf=pdf&amp;site=cell-site">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_105_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84889583293" target="_blank" aria-label="Open Scopus in new tab">Scopus (387)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24267886/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.cell.2013.10.020&amp;pmid=24267886" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib110" class="citations"><div id="sref110" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref110" title="View in article">106.</a></div><div>Wood, A.R. ∙ Esko, T. ∙ Yang, J. ..., Electronic Medical Records and Genomics (eMEMERGEGE) Consortium, MIGen Consortium, PAGEGE Consortium, LifeLines Cohort Study</div><div><strong>Defining the role of common variation in the genomic and biological architecture of adult human height</strong></div><div><em>Nat. Genet.</em> 2014; <strong>46</strong>:1173-1186</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/ng.3097" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_106_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84908890496" target="_blank" aria-label="Open Scopus in new tab">Scopus (833)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25282103/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fng.3097&amp;pmid=25282103" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib111" class="citations"><div id="sref111" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">107.</a></div><div>Wray, N.R. ∙ Ripke, S. ∙ Mattheisen, M. ..., eQTLGen, 23andMe, Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium</div><div><strong>Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression</strong></div><div><em>Nat. Genet.</em> 2018; <strong>50</strong>:668-681</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1038/s41588-018-0090-3" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_107_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85046022254" target="_blank" aria-label="Open Scopus in new tab">Scopus (394)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29700475/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1038%2Fs41588-018-0090-3&amp;pmid=29700475" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib112" class="citations"><div id="sref112" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref112" title="View in article">108.</a></div><div>Wright, C.F. ∙ Fitzgerald, T.W. ∙ Jones, W.D. ..., DDD study</div><div><strong>Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data</strong></div><div><em>Lancet.</em> 2015; <strong>385</strong>:1305-1314</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2FS0140-6736%2814%2961705-0&amp;cf=fulltext&amp;site=lancet-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2FS0140-6736%2814%2961705-0&amp;cf=pdf&amp;site=lancet-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_108_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84926522440" target="_blank" aria-label="Open Scopus in new tab">Scopus (296)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/25529582/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2FS0140-6736%2814%2961705-0&amp;pmid=25529582" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib113" class="citations"><div id="sref113" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref113-1" title="View in article">109.</a></div><div>Xu, X. ∙ Wells, A.B. ∙ O’Brien, D.R. ...</div><div><strong>Cell type-specific expression analysis to identify putative cellular mechanisms for neurogenetic disorders</strong></div><div><em>J.&nbsp;Neurosci.</em> 2014; <strong>34</strong>:1420-1431</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1523/JNEUROSCI.4488-13.2014" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_109_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-84892720695" target="_blank" aria-label="Open Scopus in new tab">Scopus (80)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/24453331/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1523%2FJNEUROSCI.4488-13.2014&amp;pmid=24453331" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib114" class="citations"><div id="sref114" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">110.</a></div><div>Yengo, L. ∙ Sidorenko, J. ∙ Kemper, K.E. ..., GIANT Consortium</div><div><strong>Meta-analysis of genome-wide association studies for height and body mass index in ∼700000 individuals of European ancestry</strong></div><div><em>Hum. Mol. Genet.</em> 2018; <strong>27</strong>:3641-3649</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/hmg/ddy271" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_110_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85054297174" target="_blank" aria-label="Open Scopus in new tab">Scopus (130)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/30124842/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fhmg%2Fddy271&amp;pmid=30124842" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib115" class="citations"><div id="sref115" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref115" title="View in article">111.</a></div><div>Yip, B.H.K. ∙ Bai, D. ∙ Mahjani, B. ...</div><div><strong>Heritable variation, with little or no maternal effect, accounts for recurrence risk to autism spectrum disorder in Sweden</strong></div><div><em>Biol. Psychiatry.</em> 2018; <strong>83</strong>:589-597</div></div><div class="external-links"><div class="core-xlink-fulltext"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.09.007&amp;cf=fulltext&amp;site=bps-site" target="_blank" aria-label="Open Full Text in new tab">Full Text</a></div><div class="core-xlink-pdf"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=4&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=10.1016%2Fj.biopsych.2017.09.007&amp;cf=pdf&amp;site=bps-site" target="_blank" aria-label="Open Full Text (PDF) in new tab">Full Text (PDF)</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_111_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85032932944" target="_blank" aria-label="Open Scopus in new tab">Scopus (11)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/29100626/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1016%2Fj.biopsych.2017.09.007&amp;pmid=29100626" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div><div role="listitem" data-has="label"><div id="bib116" class="citations"><div id="sref116" class="citation"><div class="citation-content"><div class="label"><a href="#body-ref-sref116" title="View in article">112.</a></div><div>Zheng, J. ∙ Erzurumluoglu, A.M. ∙ Elsworth, B.L. ..., Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium</div><div><strong>LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis</strong></div><div><em>Bioinformatics.</em> 2017; <strong>33</strong>:272-279</div></div><div class="external-links"><div class="core-xlink-crossref"><a href="https://doi.org/10.1093/bioinformatics/btw613" target="_blank" aria-label="Open Crossref in new tab">Crossref</a></div><div class="core-xlink-scopus"><a href="/servlet/linkout?suffix=e_1_5_1_2_112_2&amp;dbid=137438953472&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;key=2-s2.0-85028296791" target="_blank" aria-label="Open Scopus in new tab">Scopus (223)</a></div><div class="core-xlink-pubmed"><a href="https://pubmed.ncbi.nlm.nih.gov/27663502/" target="_blank" aria-label="Open PubMed in new tab">PubMed</a></div><div class="core-xlink-google-scholar"><a href="https://scholar.google.com/scholar_lookup?doi=10.1093%2Fbioinformatics%2Fbtw613&amp;pmid=27663502" target="_blank" aria-label="Open Google Scholar in new tab">Google Scholar</a></div></div></div></div></div></div></div></section></div></section><div class="core-linked-content">
    

        


    

            

            

            <div data-widget-id="8633d695-b258-4908-b86f-e2c268ec3c6b" data-widget-name="UX3HTMLWidget"><script type="text/javascript">
if($('.content-navigation__body__title').length){
$('.content-navigation__body__title').each(function(){
var html=this.innerHTML;
var div = document.createElement("div");
div.innerHTML = html;
var text = div.textContent || div.innerText || "";
this.innerHTML=text;
});
}
</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="393effff-5b44-4681-a9b4-4f029b3540b7" data-widget-name="UX3HTMLWidget"><style>
.linkback-ad .pb-ad {
    display: block;
   height: auto;
    margin-left: 0px;
    margin-right: auto;
}
</style></div>

            

        

    

        

</div></div></div><div class="core-collateral-aside"><section id="core-collateral-figures"><h2>Figures (10)<button class="js--openFigureViewer"><span>Figure Viewer</span></button></h2><div class="core-collateral-aside-wrapper"><div class="grid"><a class="figure__open__ctrl" href="#figs1" aria-label="Go to Figure&amp;nbsp;S1"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/7cbb96f5-ec83-4489-a5ad-df093a4a3147/main.assets/figs1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig1" aria-label="Go to Figure&amp;nbsp;1"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/64046fc9-1e40-49ed-ad5f-3c80f01b296a/main.assets/gr1.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig2" aria-label="Go to Figure&amp;nbsp;2"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/d7d3b23d-21c9-4c78-9be4-31a6d0d43cb4/main.assets/gr2.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs2" aria-label="Go to Figure&amp;nbsp;S2"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/7b882646-6021-489a-9ea3-88bc568a941b/main.assets/figs2.jpg" aria-hidden="true"></a></div><div id="collapsible-figures" data-expandable="item"><div class="content grid" style="display:none;" role="region" id="collapsible-figures_content"><a class="figure__open__ctrl" href="#fig3" aria-label="Go to Figure&amp;nbsp;3"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/1c8bd6d3-02e5-46c0-a433-1b103ec78307/main.assets/gr3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig4" aria-label="Go to Figure&amp;nbsp;4"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/0d271389-ea57-4ce6-89bd-402b4d70b79e/main.assets/gr4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs3" aria-label="Go to Figure&amp;nbsp;S3"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/197a43ad-4045-410e-89d3-7d7eaaa5d91b/main.assets/figs3.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#fig5" aria-label="Go to Figure&amp;nbsp;5"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/917a7389-dde6-457a-8163-a7a726128835/main.assets/gr5.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs4" aria-label="Go to Figure&amp;nbsp;S4"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/1409390a-ef6f-4f3d-848f-7e2fc6853df4/main.assets/figs4.jpg" aria-hidden="true"></a><a class="figure__open__ctrl" href="#figs5" aria-label="Go to Figure&amp;nbsp;S5"><img src="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/asset/7989e58f-7719-4424-8751-5400a28b88fd/main.assets/figs5.jpg" aria-hidden="true"></a></div><button class="trigger" aria-controls="collapsible-figures_content" aria-expanded="false" tabindex="0" role="button"><span class="expand">Show all figures</span><span class="collapse">Hide figures</span><i class="icon-section_arrow_d" aria-hidden="true"></i></button></div></div></section>
    

        


    

            

            

            <section id="core-collateral-metrics" class="publication-list__axel-section"><h2>Article metrics</h2><div><ul class="publication-list rlist publication-list--list plum-x--lazy-load publication-list--axel js--loaded" data-plum-x-url="/pb/widgets/ux3/publicationList/plumx?widgetId=66dd7e11-7768-4006-93d8-a72b60e2a264&amp;pbContext=%3Bproduct%3Aproduct%3Aelsevier%5C%3Aproduct%5C%3Acell%3Bwebsite%3Awebsite%3Acell-site%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3Apii%5C%3AS0092867419313984%3Bissue%3Aissue%3Apii%5C%3AS0092867419X00045%3Bjournal%3Ajournal%3Acell%3BpageGroup%3Astring%3AArticle+View%3Bpage%3Astring%3AShow+Full+Text%3BrequestedJournal%3Ajournal%3Acell%3Bwgroup%3Astring%3ADefault+Website+Group%3BsubPage%3Astring%3AFull+Text" data-pb-mode-enabled="false">
                <li class="publication-list__box publication-list__box-axel">
                    <ul class="rlist axel-list__box">
                        
                        
            <li class="plum-x-wrapper">
                <div class="plum-x plum-x--axel">
                    <div class="plum-x__items">
                        
                    <div class="plum-x__item plum-x-citation">
                        <div class="plum-x__item-num">1,170</div>
                        <div class="plum-x__item-title">Citations</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-capture">
                        <div class="plum-x__item-num">1,710</div>
                        <div class="plum-x__item-title">Captures</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-mention">
                        <div class="plum-x__item-num">41</div>
                        <div class="plum-x__item-title">Mentions</div>
                    </div>    
                
                    <div class="plum-x__item plum-x-socialMedia">
                        <div class="plum-x__item-num">66</div>
                        <div class="plum-x__item-title">Social Media</div>
                    </div>    
                
                    </div>
                    <div class="plum-x__details">
                        <div class="plum-x__logo">
                            <img src="/products/marlin/releasedAssets/images/plumx-logo.png" alt="Plum X logo">
                        </div>
                        <a class="plum-x__view-details" href="https://plu.mx/plum/a/?doi=10.1016/j.cell.2019.12.036&amp;theme=plum-jbs-theme&amp;hideUsage=true" target="_blank">
                            <span>View details</span>
                            <span class="sr-only">opens in a new tab</span>
                            <i class="icon-external-arrow" aria-hidden="true"></i>
                        </a>
                    </div>
                </div>  
            </li>
        
                    </ul>
                </li></ul></div></section>

            

        
<section id="core-collateral-supplementary"><h2>Supplemental information (5)</h2><div class="core-collateral-aside-wrapper"><div class="core-supplementary-material"><div class="core-link"><button class="downloadAllSupplementary"><span>Download all</span></button></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/attachment/44aca411-6be3-4158-b1d6-6b339d60136d/mmc1.xlsx" download="mmc1.xlsx"><span id="mmc1-link">Spreadsheet (5.83 MB)</span></a><div class="core-description">Table S1. Cohorts, Samples, <i>De Novo</i> Variants, and Liability Analysis, Related to Figure&nbsp;1</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/attachment/cc634b34-b459-4aff-acf0-cb58a3013bd1/mmc2.xlsx" download="mmc2.xlsx"><span id="mmc2-link">Spreadsheet (5.94 MB)</span></a><div class="core-description">Table S2. TADA FDR Values, Related to Figure&nbsp;2</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/attachment/712fc849-cd3e-4ec9-820a-c23443608a80/mmc3.xlsx" download="mmc3.xlsx"><span id="mmc3-link">Spreadsheet (174.55 KB)</span></a><div class="core-description">Table S3. Missense Variants, Genomic Disorder Loci, and GWAS Enrichment Analysis, Related to Figure&nbsp;3</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/attachment/45f1baed-a2ec-4128-be82-a50ae08adc38/mmc4.xlsx" download="mmc4.xlsx"><span id="mmc4-link">Spreadsheet (1.20 MB)</span></a><div class="core-description">Table S4. Neurodevelopmental Delay Variants, ASD Phenotype Data, and Functional Categorization of ASD-Associated Genes, Related to Figure&nbsp;4</div></div></div><div class="core-supplementary-material" data-category="spreadsheet"><div class="core-link"><a href="https://www.cell.com/cms/10.1016/j.cell.2019.12.036/attachment/0b26abc3-a77a-4f26-806f-109797aec47e/mmc5.xlsx" download="mmc5.xlsx"><span id="mmc5-link">Spreadsheet (8.42 MB)</span></a><div class="core-description">Table S5. Integration of functional data, Related to Figure&nbsp;5</div></div></div></div></section>
    

        


    

            

            

            <section id="core-collateral-relatedArticles" class="axel-related-articles"><h2 id="relatedArticles" class="top axel-related-articles__title" data-title="Related articles">Related Articles</h2><div data-pb-widget-type="ux3-relatedArticles" data-widget-id="8d3844d1-450e-497c-ab75-a63fc41750ac" data-widget-name="AtyponTempUX3RelatedArticlesWidget" data-article-pii="S0092867419313984" class="ux3DeferredOnScrollWidget axel-related-articles__body ux3DeferredLoaded" data-show-more="true">We haven't found any related articles.</div></section>

            

        
</div></div><aside data-core-aside="right-rail"><div class="inner-wrapper">

    

        


    

            

            

            <div data-widget-id="dca7470d-9085-4d21-a6f4-a66397718146" data-widget-name="UX3HTMLWidget"><input type="hidden" id="gptPage" value="cell-videos"></div>

            

        











        <div data-widget-def="literatumAd" data-widget-id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc" class="adplaceholders" style="right: 0px; position: static;">
        
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd alignCenter  widget-verticalAd" id="481d10c2-bbe0-470e-a5ba-207a81ecc8dc">
                
                
                    
                

                <div class="wrapped " id="Marketplace-Recs">
                    
                    <div class="widget-body body body-verticalAd "><div class="verticalAdContainer">
    <div class="verticalAdLabelContainer">
        <div class="verticalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adVert.gif" alt="Advertisement">
        </div>
    </div>
    <div class="verticalAd">
        





        <div class="pb-ad">
            <script src="//cdn.pbgrd.com/core-cell.js" async=""></script>
<div id="pbgrd-skyr-c"><div id="pbgrd-skyr" style="width: inherit; position: relative; background: transparent;"></div></div>
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

        </div>
    

</div></aside><div class="core-authors-details" style="display:none"><section class="core-authors"></section></div></article><div data-extent="article-wrapper"><div class="core-container"></div></div><script id="axel-publication-metadata" type="application/json">{"doi":"10.1016/j.cell.2019.12.036","data-is-retraction":false,"data-is-retired":false}
</script><script type="text/javascript" data-ot-ignore="data-ot-ignore" defer="defer" src="https://static.addtoany.com/menu/page.js" class="optanon-category-C0004 ot-vscat-C0004"></script><template id="figure_nav_template"><a href="#" class="open-in-viewer btn btn-dark figure-pop-btn"><i aria-hidden="true" class="icon-export_citation"></i><span>Open in viewer</span></a></template><div id="figure-viewer" role="dialog" aria-modal="true" aria-label="Figure Viewer" class="showCaption showThumbstrip figure-viewer"><nav class="nav-bar figure-viewer__top"><a href="#" title="Close Figure Viewer" role="button" class="figure-viewer__close"><i aria-hidden="true" class="icon-tools_close"></i></a><div class="figure-viewer__title">Large-Scale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism</div></nav><div class="frames"><ul class="frames__list"><li><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2019.12.036/asset/7cbb96f5-ec83-4489-a5ad-df093a4a3147/main.assets/figs1.jpg" height="942" width="668" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/826d7e93-97ca-424e-beb4-f548a879b2b5/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2019.12.036/asset/64046fc9-1e40-49ed-ad5f-3c80f01b296a/main.assets/gr1.jpg" height="946" width="708" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/379b31ff-ee89-4304-959d-e4f0e361a995/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2019.12.036/asset/d7d3b23d-21c9-4c78-9be4-31a6d0d43cb4/main.assets/gr2.jpg" height="688" width="770" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/63a1aa54-d45f-4e0c-b3d0-18e33d5eb512/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2019.12.036/asset/7b882646-6021-489a-9ea3-88bc568a941b/main.assets/figs2.jpg" height="941" width="738" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/7d96eace-13bd-40da-84d4-74d7d4a8294b/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2019.12.036/asset/1c8bd6d3-02e5-46c0-a433-1b103ec78307/main.assets/gr3.jpg" height="548" width="770" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/71c982c4-299f-46fc-a363-5ece27ee76a5/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2019.12.036/asset/0d271389-ea57-4ce6-89bd-402b4d70b79e/main.assets/gr4.jpg" height="672" width="770" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/9c295865-705d-4b14-9b5f-19064d31bf64/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2019.12.036/asset/197a43ad-4045-410e-89d3-7d7eaaa5d91b/main.assets/figs3.jpg" height="530" width="375" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/b9ba4526-627e-4893-9b64-561c447dc79b/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2019.12.036/asset/917a7389-dde6-457a-8163-a7a726128835/main.assets/gr5.jpg" height="831" width="770" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/03f8ce1b-2195-449a-b797-f4cbe8e92fb1/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2019.12.036/asset/1409390a-ef6f-4f3d-848f-7e2fc6853df4/main.assets/figs4.jpg" height="935" width="688" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/0415bdb6-a255-4e86-954d-2f83c91cc578/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li><li><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2019.12.036/asset/7989e58f-7719-4424-8751-5400a28b88fd/main.assets/figs5.jpg" height="794" width="770" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/d482847c-bfef-4e07-91cd-8d0b65fd7209/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0" class="ui-draggable ui-draggable-handle" style="position: relative;"></div></li></ul><a href="#" title="Previous Frame" class="frameButton previousFrameButton frames__prev"><!--span Previous Frame--><i class="icon-arrow_l"></i></a><a href="#" title="Next Frame" class="frameButton nextFrameButton frames__next"><!--span Next Frame--><i class="icon-arrow_r"></i></a></div><div class="caption"><div class="figure-viewer__caption-body"></div></div><div class="toolbar figure-viewer__toolbar"><span class="figure-viewer__counter"></span><a href="#" class="button toggle-caption-button figure-viewer__caption__ctrl"><i aria-hidden="true" class="icon icon-info"></i><span class="label">Hide Caption</span></a><a href="#" class="button download-options-button"><i aria-hidden="true" class="icon icon-gizmo-folder-in download-size-icon"></i><span class="label">Download</span></a><a href="#figures" class="button see-figures figure-viewer__see-figure__ctrl"><i aria-hidden="true" class="icon icon-gizmo-image_figure viewer-size-icon"></i><span class="label">See figure in Article</span></a></div><div class="thumbstrip"><a href="#" class="toggleButton thumbstrip__ctrl"><span>Toggle Thumbstrip</span></a><a href="#" title="Previous Thumbnails" class="thumbstripButton previousThumbstripButton thumbstrip__prev"><i aria-hidden="true" class="icon-arrow_l"></i></a><a href="#" title="Next Thumbnails" class="thumbstripButton nextThumbstripButton thumbstrip__next"><i aria-hidden="true" class="icon-arrow_r"></i></a><div class="thumbstrip__wrapper"><ul class="rlist figure-viewer__thumbslist"><li><div class="thumbnail__label">Figure&nbsp;S1</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs1" src="/cms/10.1016/j.cell.2019.12.036/asset/7cbb96f5-ec83-4489-a5ad-df093a4a3147/main.assets/figs1.jpg" height="942" width="668" aria-labelledby="figs1-title figs1-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/826d7e93-97ca-424e-beb4-f548a879b2b5/main.assets/figs1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;1</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr1" src="/cms/10.1016/j.cell.2019.12.036/asset/64046fc9-1e40-49ed-ad5f-3c80f01b296a/main.assets/gr1.jpg" height="946" width="708" aria-labelledby="fig1-title fig1-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/379b31ff-ee89-4304-959d-e4f0e361a995/main.assets/gr1_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;2</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr2" src="/cms/10.1016/j.cell.2019.12.036/asset/d7d3b23d-21c9-4c78-9be4-31a6d0d43cb4/main.assets/gr2.jpg" height="688" width="770" aria-labelledby="fig2-title fig2-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/63a1aa54-d45f-4e0c-b3d0-18e33d5eb512/main.assets/gr2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S2</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs2" src="/cms/10.1016/j.cell.2019.12.036/asset/7b882646-6021-489a-9ea3-88bc568a941b/main.assets/figs2.jpg" height="941" width="738" aria-labelledby="figs2-title figs2-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/7d96eace-13bd-40da-84d4-74d7d4a8294b/main.assets/figs2_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;3</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr3" src="/cms/10.1016/j.cell.2019.12.036/asset/1c8bd6d3-02e5-46c0-a433-1b103ec78307/main.assets/gr3.jpg" height="548" width="770" aria-labelledby="fig3-title fig3-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/71c982c4-299f-46fc-a363-5ece27ee76a5/main.assets/gr3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;4</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr4" src="/cms/10.1016/j.cell.2019.12.036/asset/0d271389-ea57-4ce6-89bd-402b4d70b79e/main.assets/gr4.jpg" height="672" width="770" aria-labelledby="fig4-title fig4-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/9c295865-705d-4b14-9b5f-19064d31bf64/main.assets/gr4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S3</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs3" src="/cms/10.1016/j.cell.2019.12.036/asset/197a43ad-4045-410e-89d3-7d7eaaa5d91b/main.assets/figs3.jpg" height="530" width="375" aria-labelledby="figs3-title figs3-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/b9ba4526-627e-4893-9b64-561c447dc79b/main.assets/figs3_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;5</div><div class="thumbnail"><div class="thumbnail__image"><img id="gr5" src="/cms/10.1016/j.cell.2019.12.036/asset/917a7389-dde6-457a-8163-a7a726128835/main.assets/gr5.jpg" height="831" width="770" aria-labelledby="fig5-title fig5-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/03f8ce1b-2195-449a-b797-f4cbe8e92fb1/main.assets/gr5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S4</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs4" src="/cms/10.1016/j.cell.2019.12.036/asset/1409390a-ef6f-4f3d-848f-7e2fc6853df4/main.assets/figs4.jpg" height="935" width="688" aria-labelledby="figs4-title figs4-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/0415bdb6-a255-4e86-954d-2f83c91cc578/main.assets/figs4_lrg.jpg" loading="lazy" tabindex="0"></div></div></li><li><div class="thumbnail__label">Figure&nbsp;S5</div><div class="thumbnail"><div class="thumbnail__image"><img id="figs5" src="/cms/10.1016/j.cell.2019.12.036/asset/7989e58f-7719-4424-8751-5400a28b88fd/main.assets/figs5.jpg" height="794" width="770" aria-labelledby="figs5-title figs5-content" data-viewer-src="/cms/10.1016/j.cell.2019.12.036/asset/d482847c-bfef-4e07-91cd-8d0b65fd7209/main.assets/figs5_lrg.jpg" loading="lazy" tabindex="0"></div></div></li></ul></div></div><div class="download figure-viewer__download"><div class="pointer"></div><ul class="rlist downlad__wrapper"><li class="hidden-xs hidden-sm"><a href="/action/downloadFigures?pii=e00489&amp;id=" class="downloadAllButton"><i class="icon icon-gizmo-ppt figure-viewer__download__viewer-size-icon figure-viewer__download__viewer-size-icon--ppt"></i><span class="figure-viewer__download__text"> Download .PPT</span></a></li></ul></div></div><template id="toCitationLink" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></a></div></template><template id="toCitationButton" data-citation="Citation" data-footnote="Footnote"><div class="to-citation__wrapper"><button class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span>Go to</span></button></div></template><template id="toCitationAccordion" data-collapse-citation="Hide Citations" data-expand-citation="Show Citations" data-collapse-footnote="Hide Footnotes" data-expand-footnote="Show Footnotes"><div class="to-citation__wrapper"><button aria-controls="aria-controls" aria-expanded="false" aria-label="Toggle citations menu" class="to-citation__toggle"><i aria-hidden="true" class="icon-plus"></i><span data-expand-title="data-expand-title" data-collapsed-title="data-collapsed-title" class="accordion__toggle__title"></span></button><div role="menu" aria-label="links" class="to-citation__accordion no-separator"><ul></ul></div></div></template><template id="toCitationAccordionItem"><li><a class="to-citation"><i aria-hidden="true" class="icon-subdirectory_arrow_left"></i><span></span></a></li></template><template id="citations_truncate_template"><div class="citations-truncation"><button data-label-expand="Show all references" data-label-collapse="Show fewer" data-label-remaining="references remaining" class="btn btn--inverse"><span>Show all references</span></button></div></template><template id="collateral_texts_template" data-do="Digital Object" data-figures="Figures" data-metrics="Article Metrics" data-related="Related articles" data-supplementary="Supplemental information"></template><template id="collapsible_authors_template"><button aria-expanded="false" data-expandable="all" data-label-expand="Expand All" data-label-collapse="Collapse All" class="collateral-contributors-control"><span>Expand All</span></button></template>
<template id="collapsible_tables_collapse_template"><div class="collapsible-figure-btn__wrapper expanded"><button aria-expanded="true" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-up"></i><span class="text-uppercase">Collapse</span></button></div></template>
<template id="collapsible_tables_expand_template"><div class="collapsible-figure-btn__wrapper collapsed"><button aria-expanded="false" class="btn collapsible-figure-btn btn--inverse"><i aria-hidden="true" class="icon-arrow-down"></i><span class="text-uppercase">Expand Table</span></button></div></template><template id="authorsAffiliationsLink"><a href="#tab-contributors" class="to-authors-affiliations">Authors Info &amp; Affiliations</a></template></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-  widget-compact-all" id="756cf18d-e910-4468-936c-0225ae7c669f">
                
                
                    
                

                <div class="wrapped " id="ad for nucleus-single-molecule-technologies">
                    
                    <div class="widget-body body body-  body-compact-all"><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(19)31398-4';

if(URL_var.indexOf("S0165-6147(17)30205-5") > -1
|| URL_var.indexOf("S0167-7799(18)30204-X") > -1 
|| URL_var.indexOf("S0092-8674(18)31649-0") > -1 
|| URL_var.indexOf("S0092-8674(19)30404-0") > -1 
|| URL_var.indexOf("S1097-2765(18)30756-1") > -1 
|| URL_var.indexOf("S0092-8674(18)31452-1") > -1 
|| URL_var.indexOf("S1097-2765(19)30175-3") > -1 
|| URL_var.indexOf("S0092-8674(18)30852-3") > -1 
|| URL_var.indexOf("S0006-3495(18)30295-9") > -1 
|| URL_var.indexOf("S1097-2765(19)30063-2") > -1 
|| URL_var.indexOf("S0006-3495(17)33208-3") > -1 
|| URL_var.indexOf("S2211-1247(18)31639-5") > -1 
|| URL_var.indexOf("S1097-2765(18)30978-X") > -1 
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-single-molecule-technologies" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none  widget-" id="754b1780-1117-4371-9285-9cd7020963d5">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body- "><div class="pb-ad">
            <script type="text/javascript">
URL_var = 'https://www.cell.com/cell/fulltext/S0092-8674(19)31398-4';

if(URL_var.indexOf("S1535-6108(19)30104-7") > -1
|| URL_var.indexOf("S0092-8674(18)31394-1") > -1
|| URL_var.indexOf("S1074-7613(19)30033-0") > -1
|| URL_var.indexOf("S1535-6108(18)30473-2") > -1
|| URL_var.indexOf("S0092-8674(19)30154-0") > -1
|| URL_var.indexOf("S1550-4131(18)30178-5") > -1
|| URL_var.indexOf("S1535-6108(19)30242-9") > -1
|| URL_var.indexOf("S1074-7613(18)30431-X") > -1
|| URL_var.indexOf("S0092-8674(19)30618-X") > -1
|| URL_var.indexOf("S1535-6108(18)30472-0") > -1
|| URL_var.indexOf("S1535-6108(18)30120-X") > -1
|| URL_var.indexOf("S0092-8674(18)31247-9") > -1
|| URL_var.indexOf("S1074-7613(18)30085-2") > -1
|| URL_var.indexOf("S1074-7613(18)30080-3") > -1
|| URL_var.indexOf("S1471-4906(18)30210-2") > -1
|| URL_var.indexOf("S1550-4131(19)30304-3") > -1
|| URL_var.indexOf("S1934-5909(18)30599-X") > -1
|| URL_var.indexOf("S2405-8033(19)30024-X") > -1
|| URL_var.indexOf("S1535-6108(18)30109-0") > -1
|| URL_var.indexOf("S0962-8924(18)30127-2") > -1
)
    {
document.write('<input type="hidden" id="gpttc" value="nucleus-tumor-immune-microenvironment" class="gpt-Target">')
}
</script>
        </div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="9c9ff7a1-a035-49f6-a336-600c9419c518" data-widget-name="UX3HTMLWidget"><!-- Start of HubSpot Embed Code --> 

<script type="text/plain" class="optanon-category-4" id="hs-script-loader" async="" defer="" src="//js.hs-scripts.com/519224.js"></script> 

<!-- End of HubSpot Embed Code --></div>

            

        

    

        


    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget sessionValidationWidget none  widget-none  widget-compact-all" id="0257aef7-6074-4898-a989-1c3f693225a2">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><script>
    $.ajax({
        url: "https://www.cell.com/ssoSessionCheck",
        type: "GET",
        dataType: "jsonp",
        success: function (data) {
            if(data.RefreshSession) {
                location.reload();
            }
        },
        error : function (){
            console.warn("can't validate CSAS session.");
        }
    });
</script></div>
                </div>
                </div>

        

    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                            
                        
                        
                            
                            
                            
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget csasSsoUrls none  widget-none  widget-compact-all" id="948bc5c4-8b0d-47be-b5ad-6570559802d1">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-none  body-compact-all"><!-- clear SSO URLs after all iFrames are rendered or after passing a time out-->
<script>
    setTimeout(clearSsoUrlsFun,10000);
    var iFramesList = document.getElementById("sso-urls-div").getElementsByTagName("iframe");
    var renderedIFramesCount = 0;
    var numberOfIFrames = iFramesList.length;
    var skipRefresh = false;
    for (var i = 0; i < iFramesList.length; i++) {
        var iFrame = iFramesList[i];
        iFrame.addEventListener("load", function() {
            renderedIFramesCount = renderedIFramesCount + 1;
            if (renderedIFramesCount >= numberOfIFrames)
            {
                clearSsoUrlsFun();
            }
        });
    }

    function clearSsoUrlsFun() {
        if (skipRefresh == false && numberOfIFrames > 0)
        {
            skipRefresh = true;
            $.ajax({
                url: "https://www.cell.com/action/clearSsoUrls",
                type: "GET",
                dataType: "jsonp",
                error : errorCallingAjx
            });
        }
    }

    function errorCallingAjx(){
        console.log("can't delete sso urls .")
    }
</script></div>
                </div>
                </div>

        










        <div data-widget-def="ux3-layout-widget" data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" class="ux3 ">
        
    

        


    

            

            

            <div data-widget-id="d95b4a46-dc4d-4ca8-83f9-8b6724c649ec" data-widget-name="ux3-layout-widget" class="fonts-loaded"><div class="footer clearfix">
    

        


    

                
                
                
                
                
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                    
                
                    
                
                    
                        
                        
                        
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                    
                
                
                
                <div class="widget literatumAd none adplaceholder adplaceholder--header widget-horizontalAd" id="a07ee92b-3942-49f8-9186-a2a08fc3fc56">
                
                

                <div class="wrapped ">
                    
                    <div class="widget-body body body-horizontalAd "><div class="horizontalAdContainer">
    <div class="horizontalAdLabelContainer">
        <div class="horizontalAdLabel">
                <img src="/templates/jsp/_style2/_marlin/images/bg_adHoriz.gif" alt="Advertisement">
        </div>
    </div>
    <div class="horizontalAd">
        





        <div class="pb-ad">
            <!-- Beginning Async AdSlot 4 for Ad unit els.cellpress/cell  ### size: [[728,90]] -->
<div class="gpt-ad" data-adsize="cell-leaderboard" data-page="" data-site="cell" data-pos="4"></div>
<!-- End AdSlot 4 -->
        </div>
    

    </div>
</div></div>
                </div>
                </div>

        

    

        


    

            

            

            <div data-widget-id="23703bbc-af46-4f3f-a5bb-dd52891ff0ff" data-widget-name="UX3HTMLWidget"><div class="footer__top">
    <div class="footer__logo centered"><a href="/home" title="Cell Press"><!-- Generator: Adobe Illustrator 25.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 300 72" style="enable-background:new 0 0 300 72;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#007DBC;}
</style>
<g>
	<path class="st0" d="M39.4,7.5c-2.3-0.3-4.4,1.1-4.9,3.3l-1.8,7.4c-0.4,1.8-1.9,3.1-3.8,3.3l-10.5,1.2C7.1,24-1,34.3,0.7,45.6
		c1.3,9,8.6,15.7,17.2,17c2.3,0.3,4.4-1.1,4.9-3.3l1.8-7.4c0.4-1.8,1.9-3.1,3.8-3.3l10.5-1.2c11.3-1.2,19.4-11.6,17.8-22.9
		C55.3,15.4,48.1,8.8,39.4,7.5z M37.1,34l-6.6,0.8c-1.1,0.1-2,0.9-2.3,2l-1.9,8.1c-0.6,2.4-2.5,4.2-5,4.5c-3.8,0.4-7.1-2.4-7.4-6.2
		c-0.3-3.7,2.6-6.8,6.3-7.2l6.6-0.8c1.1-0.1,2-0.9,2.3-2l1.9-8.1c0.6-2.4,2.5-4.2,5-4.5c3.8-0.4,7.1,2.4,7.4,6.2
		C43.7,30.5,40.8,33.6,37.1,34z"></path>
	<rect x="153.1" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M90.2,20.4c7.3,0,8.6,4.8,9.4,7.3h8.8c-0.5-6.2-5.6-14.9-18.2-14.9c-10.9,0-19.4,7.5-19.4,22.3
		c0,14.6,8.1,22.2,19.3,22.2c10.6,0,16.6-6.4,18.4-15.6h-8.8c-1.1,5.2-4.5,7.9-9.4,7.9c-7.5,0-10.5-6.8-10.5-14.4
		C79.7,23.3,85.9,20.4,90.2,20.4z"></path>
	<path class="st0" d="M125.2,24c-13.7,0-15,12.1-15,15.8c0,12.9,6.8,17.1,15.6,17.1c5.4,0,11.6-2.5,13.8-10H131
		c-0.7,2.2-3,3.4-5.3,3.4c-6.6,0-7.1-5.3-7.3-7.7h21.5v-1.5C139.9,27,131.3,24,125.2,24z M118.6,37.4c0.5-4.1,2.5-6.6,6.4-6.6
		c2.6,0,5.9,1.5,6.4,6.6H118.6z"></path>
	<rect x="142" y="13.9" class="st0" width="8.2" height="42.3"></rect>
	<path class="st0" d="M174,13.8h-9.8V21h7.2c4.3,0,8.2,0.9,8.2,6.4c0,5.4-3.9,6.4-8.2,6.4h-7.2V41h9.8c10.6,0,14.7-6.7,14.7-13.6
		C188.7,20.5,184.6,13.8,174,13.8z"></path>
	<path class="st0" d="M210.5,24.9c-0.5-0.2-1.2-0.3-1.8-0.3c-4.1,0-8.3,2.7-9.8,6.5h-0.1v-5.7h-8v30.7h8.4V42.3
		c0-5.4,2.1-9.9,8.3-9.9c1,0,2.3,0.1,3,0.3V24.9z"></path>
	<path class="st0" d="M225.6,24.6c-9.3,0-15.7,7-15.7,16.2c0,9.5,6,16.1,15.7,16.1c6.9,0,12-3.1,14.4-10.3h-7.4
		c-0.5,1.9-3.3,4-6.6,4c-4.7,0-7.3-2.4-7.5-7.8h22.1C241.1,33.4,236,24.6,225.6,24.6z M218.4,37.5c0.1-2.4,1.7-6.5,7-6.5
		c4.1,0,5.9,2.3,6.7,6.5H218.4z"></path>
	<path class="st0" d="M260.3,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		C242,54.3,248.9,57,256,57c7,0,14-2.6,14-10.7C270.1,40.6,265.2,38.8,260.3,37.7z"></path>
	<path class="st0" d="M289.8,37.7c-4.8-1.1-9.6-1.5-9.6-4.6c0-2.5,2.8-2.9,4.7-2.9c3,0,5.5,0.9,5.8,4.2h8c-0.7-7.5-6.8-9.8-13.5-9.8
		c-6.6,0-13.4,2-13.4,9.9c0,5.4,4.9,7.1,9.7,8.2c6.1,1.3,9.6,2.2,9.6,4.8c0,3-3.1,3.9-5.5,3.9c-3.3,0-6.4-1.5-6.5-5.2h-8
		c0.4,8.1,7.3,10.8,14.4,10.8c7,0,14-2.6,14-10.7C299.6,40.6,294.7,38.8,289.8,37.7z"></path>
	<path class="st0" d="M34.1,57.6c0.3-0.2,0.4-0.6,0.4-1c0-0.5-0.2-0.9-0.5-1.1c-0.3-0.2-0.8-0.3-1.4-0.3h-1.9v5.1h0.8V58h0.8
		l1.3,2.2h0.9l-1.4-2.2C33.6,57.9,33.9,57.8,34.1,57.6z M32.5,57.3h-0.9v-1.5h1c0.1,0,0.3,0,0.4,0c0.1,0,0.3,0,0.4,0.1
		c0.1,0,0.2,0.1,0.3,0.2c0.1,0.1,0.1,0.2,0.1,0.4c0,0.2,0,0.3-0.1,0.4c-0.1,0.1-0.2,0.2-0.3,0.2c-0.1,0.1-0.2,0.1-0.4,0.1
		C32.8,57.3,32.6,57.3,32.5,57.3z"></path>
	<path class="st0" d="M35.6,54.2c-0.4-0.4-0.9-0.7-1.4-0.9c-0.5-0.2-1.1-0.3-1.7-0.3c-0.6,0-1.2,0.1-1.7,0.3c-0.6,0.2-1,0.5-1.4,0.9
		c-0.4,0.4-0.7,0.9-1,1.5c-0.2,0.6-0.4,1.2-0.4,1.9c0,0.7,0.1,1.3,0.3,1.8c0.2,0.6,0.5,1,0.9,1.5c0.4,0.4,0.9,0.7,1.4,1
		c0.6,0.2,1.2,0.4,1.8,0.4c0.6,0,1.2-0.1,1.7-0.3c0.5-0.2,1-0.5,1.4-0.9c0.4-0.4,0.7-0.9,1-1.5c0.2-0.6,0.4-1.2,0.4-1.9
		c0-0.7-0.1-1.3-0.4-1.9C36.4,55.1,36,54.6,35.6,54.2z M35.7,59.1c-0.2,0.5-0.4,0.9-0.8,1.2c-0.3,0.3-0.7,0.6-1.1,0.8
		c-0.4,0.2-0.9,0.3-1.4,0.3c-0.6,0-1-0.1-1.5-0.3c-0.4-0.2-0.8-0.5-1.1-0.8c-0.3-0.3-0.5-0.7-0.7-1.2c-0.2-0.5-0.2-1-0.2-1.5
		c0-0.6,0.1-1.1,0.3-1.5c0.2-0.5,0.4-0.9,0.8-1.2c0.3-0.3,0.7-0.6,1.1-0.8c0.4-0.2,0.9-0.3,1.4-0.3c0.5,0,1,0.1,1.4,0.3
		c0.4,0.2,0.8,0.4,1.1,0.8c0.3,0.3,0.6,0.7,0.8,1.2c0.2,0.5,0.3,1,0.3,1.5C36,58.1,35.9,58.6,35.7,59.1z"></path>
</g>
</svg></a></div>
    <div class="footer__social-links">
        <ul class="rlist rlist--inline centered">
            <li><a href="http://on.fb.me/1tgHakh" title="Facebook" target="_blank"><i aria-hidden="true" class="icon-Icon_Facebook"></i></a></li>

            <li><a href="https://www.instagram.com/cellpress/" title="Instagram" target="_blank"><svg width="18px" height="18px" viewBox="0 0 18 18" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-147.000000, -16.000000)" fill-rule="nonzero" fill="#969696"> <g id="Instagram-Logo" transform="translate(147.000000, 16.000000)"> <path d="M9,1.62580645 C11.4096774,1.62580645 11.7,1.62580645 12.6290323,1.68387097 C13.5,1.71290323 13.9935484,1.85806452 14.3129032,2.00322581 C14.7193548,2.17741935 15.0387097,2.3516129 15.3580645,2.67096774 C15.6774194,2.99032258 15.8806452,3.28064516 16.0258065,3.71612903 C16.1419355,4.03548387 16.2870968,4.5 16.3451613,5.4 C16.3741935,6.35806452 16.4032258,6.61935484 16.4032258,9.02903226 C16.4032258,11.4387097 16.4032258,11.7290323 16.3451613,12.6580645 C16.316129,13.5290323 16.1709677,14.0225806 16.0258065,14.3419355 C15.8516129,14.7483871 15.6774194,15.0677419 15.3580645,15.3870968 C15.0387097,15.7064516 14.7483871,15.9096774 14.3129032,16.0548387 C13.9935484,16.1709677 13.5290323,16.316129 12.6290323,16.3741935 C11.6709677,16.4032258 11.4096774,16.4322581 9,16.4322581 C6.59032258,16.4322581 6.3,16.4322581 5.37096774,16.3741935 C4.5,16.3451613 4.00645161,16.2 3.68709677,16.0548387 C3.28064516,15.8806452 2.96129032,15.7064516 2.64193548,15.3870968 C2.32258065,15.0677419 2.11935484,14.7774194 1.97419355,14.3419355 C1.85806452,14.0225806 1.71290323,13.5580645 1.65483871,12.6580645 C1.62580645,11.7 1.62580645,11.4096774 1.62580645,9 C1.62580645,6.59032258 1.62580645,6.3 1.68387097,5.37096774 C1.71290323,4.5 1.85806452,4.00645161 2.00322581,3.68709677 C2.17741935,3.28064516 2.3516129,2.96129032 2.67096774,2.64193548 C2.99032258,2.32258065 3.28064516,2.11935484 3.71612903,1.97419355 C4.03548387,1.85806452 4.5,1.71290323 5.4,1.65483871 C6.3,1.62580645 6.59032258,1.62580645 9,1.62580645 L9,1.62580645 Z M9,0 C6.56129032,0 6.24193548,0 5.28387097,0.0580645161 C4.32580645,0.116129032 3.65806452,0.261290323 3.10645161,0.464516129 C2.52580645,0.696774194 2.00322581,1.01612903 1.50967742,1.50967742 C1.01612903,2.00322581 0.696774194,2.52580645 0.464516129,3.10645161 C0.232258065,3.68709677 0.0870967742,4.32580645 0.0580645161,5.28387097 C0,6.24193548 0,6.56129032 0,9 C0,11.4387097 0,11.7580645 0.0580645161,12.716129 C0.116129032,13.6741935 0.261290323,14.3419355 0.464516129,14.8935484 C0.696774194,15.4741935 1.01612903,15.9967742 1.50967742,16.4903226 C2.00322581,16.983871 2.52580645,17.3032258 3.10645161,17.5354839 C3.68709677,17.7677419 4.32580645,17.9129032 5.28387097,17.9419355 C6.24193548,18 6.56129032,18 9,18 C11.4387097,18 11.7580645,18 12.716129,17.9419355 C13.6741935,17.883871 14.3419355,17.7387097 14.8935484,17.5354839 C15.4741935,17.3032258 15.9967742,16.983871 16.4903226,16.4903226 C16.983871,15.9967742 17.3032258,15.4741935 17.5354839,14.8935484 C17.7677419,14.3129032 17.9129032,13.6741935 17.9419355,12.716129 C18,11.7580645 18,11.4387097 18,9 C18,6.56129032 18,6.24193548 17.9419355,5.28387097 C17.883871,4.32580645 17.7387097,3.65806452 17.5354839,3.10645161 C17.3032258,2.52580645 16.983871,2.00322581 16.4903226,1.50967742 C15.9967742,1.01612903 15.4741935,0.696774194 14.8935484,0.464516129 C14.3129032,0.232258065 13.6741935,0.0870967742 12.716129,0.0580645161 C11.7580645,0 11.4387097,0 9,0 L9,0 Z"></path> <path d="M9,4.38387097 C6.44516129,4.38387097 4.38387097,6.44516129 4.38387097,9 C4.38387097,11.5548387 6.44516129,13.616129 9,13.616129 C11.5548387,13.616129 13.616129,11.5548387 13.616129,9 C13.616129,6.44516129 11.5548387,4.38387097 9,4.38387097 Z M9,11.9903226 C7.34516129,11.9903226 6.00967742,10.6548387 6.00967742,9 C6.00967742,7.34516129 7.34516129,6.00967742 9,6.00967742 C10.6548387,6.00967742 11.9903226,7.34516129 11.9903226,9 C11.9903226,10.6548387 10.6548387,11.9903226 9,11.9903226 Z"></path> <circle id="Oval" cx="13.7903226" cy="4.20967742" r="1.07419355"></circle> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.linkedin.com/company/cell-press/" title="LinkedIn" target="_blank"><i aria-hidden="true" class="icon-linkedin"></i></a></li>
            
                        <li><a href="http://bit.ly/1zq7Q5T" title="Twitter" target="_blank">
                <!--<i aria-hidden="true" class="icon-Icon_Twitter"></i></a>-->
                
                <svg id="xicon" viewBox="0 0 1200 1227" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="none"><path fill="#969696" d="M714.163 519.284L1160.89 0H1055.03L667.137 450.887L357.328 0H0L468.492 681.821L0 1226.37H105.866L515.491 750.218L842.672 1226.37H1200L714.137 519.284H714.163ZM569.165 687.828L521.697 619.934L144.011 79.6944H306.615L611.412 515.685L658.88 583.579L1055.08 1150.3H892.476L569.165 687.854V687.828Z"></path></svg> 

                    <style>
                    .footer svg#xicon {
                      height: 16px;
                    }
                    </style>
            </a></li>

            
            <li><a href="/pb-assets/marketing/WeChat/WeChat-1557753222727.jpg" title="WeChat" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-122.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="wechat" transform="translate(122.000000, 17.000000)"> <path d="M12.375,4.5 L12.375,4.5 L12.4875,4.5 L12.4875,4.5 C12.6898579,4.48604473 12.8520408,4.32704187 12.87,4.125 C12.8705905,4.04725572 12.8470025,3.9712499 12.8025,3.9075 C11.94,1.5675 9.4425,0 6.5625,0 C2.9475,0 0,2.52 0,5.625 C0.0313772094,7.34410122 0.895513317,8.94093526 2.3175,9.9075 L1.5675,11.46 C1.49009368,11.6087816 1.52088865,11.7908702 1.64291494,11.9059236 C1.76494122,12.0209769 1.94852501,12.0410168 2.0925,11.955 L4.08,10.8225 C4.61861992,11.0129999 5.17724298,11.1413322 5.745,11.205 C5.86564124,11.2178337 5.98503738,11.1715098 6.06545426,11.0806685 C6.14587113,10.9898272 6.17737153,10.865694 6.15,10.7475 C6.06148727,10.4219419 6.01114891,10.0871918 6,9.75 C6,6.855 8.8575,4.5 12.375,4.5 Z M8.25,3 C8.66421356,3 9,3.33578644 9,3.75 C9,4.16421356 8.66421356,4.5 8.25,4.5 C7.83578644,4.5 7.5,4.16421356 7.5,3.75 C7.5,3.33578644 7.83578644,3 8.25,3 Z M4.5,4.5 C4.08578644,4.5 3.75,4.16421356 3.75,3.75 C3.75,3.33578644 4.08578644,3 4.5,3 C4.91421356,3 5.25,3.33578644 5.25,3.75 C5.25,4.16421356 4.91421356,4.5 4.5,4.5 Z"></path> <path d="M18,9.75 C18,7.2675 15.48,5.25 12.375,5.25 C9.27,5.25 6.75,7.2675 6.75,9.75 C6.75,12.2325 9.27,14.25 12.375,14.25 C12.9568557,14.2502837 13.5364258,14.1772075 14.1,14.0325 L15.96,14.9625 C16.0114248,14.9874074 16.0678613,15.0002339 16.125,15 C16.2497193,14.9993787 16.3659682,14.9367912 16.4351508,14.8330172 C16.5043335,14.7292432 16.5173955,14.5978644 16.47,14.4825 L15.96,13.2075 C17.1784092,12.4716325 17.9450305,11.1723221 18,9.75 L18,9.75 Z M10.5,9 C10.0857864,9 9.75,8.66421356 9.75,8.25 C9.75,7.83578644 10.0857864,7.5 10.5,7.5 C10.9142136,7.5 11.25,7.83578644 11.25,8.25 C11.25,8.66421356 10.9142136,9 10.5,9 Z M14.25,9 C13.8357864,9 13.5,8.66421356 13.5,8.25 C13.5,7.83578644 13.8357864,7.5 14.25,7.5 C14.6642136,7.5 15,7.83578644 15,8.25 C15,8.66421356 14.6642136,9 14.25,9 Z" opacity="0.959999979"></path> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.weibo.com/CellPress" title="Weibo" target="_blank"><svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-99.000000, -17.000000)" fill-rule="nonzero" fill="#969696"> <g id="weibo" transform="translate(99.000000, 14.000000)"> <g transform="translate(0.000000, 3.000000)"> <path d="M13.2402211,7.47054172 C12.9918392,7.39866484 12.821608,7.34902326 12.9514975,7.03290287 C13.232804,6.34602189 13.2622915,5.75359781 12.9571055,5.33095486 C12.3847236,4.5380684 10.8191759,4.58064295 9.02514573,5.30975376 C9.02514573,5.30889193 8.46144724,5.54882627 8.60544724,5.11515185 C8.88150754,4.25435294 8.83953769,3.53386047 8.41043216,3.11759508 C7.43716583,2.17285363 4.84824121,3.15344733 2.62745729,5.3059617 C0.964944724,6.9186238 0,8.62798358 0,10.105855 C0,12.9324952 3.73748744,14.8235294 7.39429146,14.8235294 C12.1877186,14.8235294 15.3768844,11.9501778 15.3768844,9.80576471 C15.3768844,8.50974008 14.2514774,7.77442408 13.2402211,7.47054172 Z M7.40424121,13.5561176 C4.48625126,13.8315595 1.9678794,12.5686293 1.77811055,10.7327524 C1.58870352,8.89773735 3.80098492,7.18579207 6.71861307,6.90948837 C9.63678392,6.63266758 12.1556985,7.89663201 12.3447437,9.73164706 C12.533608,11.5675239 10.3227739,13.2796416 7.40424121,13.5561176 Z M18,5.16875787 C17.9985528,2.31402189 15.5695477,0 12.5728643,0 C12.222995,0 11.9396985,0.270098495 11.9396985,0.603283174 C11.9396985,0.936640219 12.222995,1.20656635 12.5728643,1.20656635 C14.8708945,1.20656635 16.7336683,2.98159781 16.7336683,5.17099863 C16.7336683,5.50435568 17.0173266,5.77428181 17.3668342,5.77428181 C17.7167035,5.77428181 18,5.50435568 18,5.17099863 C18,5.17099863 18,5.1696197 18,5.16875787 Z M15.4841608,5.08016142 C15.1948945,3.73415048 14.0812462,2.67323393 12.6683819,2.39744733 C12.3262915,2.33039672 11.9914372,2.54051163 11.9212462,2.86680164 C11.8506935,3.19309166 12.0712161,3.51179754 12.4133065,3.57919289 C13.3319397,3.75845417 14.0559196,4.44774829 14.2438794,5.32302599 C14.3146131,5.64931601 14.6491055,5.85925855 14.9915578,5.7923803 C15.3338291,5.72498495 15.5541709,5.40627907 15.4841608,5.08016142 Z M5.90761809,8.7069275 C4.7321005,8.9320383 3.93901508,9.87126402 4.1358392,10.8044569 C4.33248241,11.7378222 5.44504523,12.311803 6.62056281,12.0865198 C7.79571859,11.861409 8.58916583,10.922528 8.3921608,9.98899042 C8.19569849,9.0559699 7.08277387,8.48181669 5.90761809,8.7069275 Z"></path> </g> </g> </g> </g></svg></a></li>
            
            <li><a href="https://www.youtube.com/user/cellvideoabstracts" title="YouTube" target="_blank"><svg width="18px" height="13px" viewBox="0 0 18 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g transform="translate(-76.000000, -19.000000)" fill-rule="nonzero" fill="#969696"> <g id="youtube" transform="translate(76.000000, 14.000000)"> <path d="M17.2984219,6.41974219 C16.6486641,5.64735937 15.4490625,5.33232422 13.1580703,5.33232422 L4.84175391,5.33232422 C2.49834375,5.33232422 1.27838672,5.66771484 0.631089844,6.49001953 C0,7.29172266 0,8.47304297 0,10.1080547 L0,13.2243047 C0,16.3917773 0.748792969,18 4.84175391,18 L13.1581055,18 C15.1448555,18 16.2457383,17.7219844 16.9579688,17.040375 C17.688375,16.3413984 18,15.2001211 18,13.2243047 L18,10.1080547 C18,8.38381641 17.951168,7.19553516 17.2984219,6.41974219 Z M11.5560703,12.0963867 L7.77965625,14.0700586 C7.69524609,14.1141797 7.60289062,14.136082 7.51071094,14.136082 C7.40629687,14.136082 7.30212891,14.1079922 7.20987891,14.0521289 C7.03613672,13.9468711 6.93007031,13.7585391 6.93007031,13.5554414 L6.93007031,9.62075391 C6.93007031,9.41800781 7.03585547,9.22985156 7.20924609,9.12452344 C7.38267187,9.01919531 7.59839062,9.01198828 7.77832031,9.10546875 L11.5547344,11.0664141 C11.7468633,11.1661523 11.8675195,11.3645039 11.8678008,11.5809258 C11.8680469,11.7975586 11.747918,11.9961914 11.5560703,12.0963867 Z"></path> </g> </g> </g></svg></a></li>
            
        </ul>
    </div>
</div>

<style>

.ux3 .footer__logo svg {
    max-width: 160px !important;
}

.ux3 .footer__social-links a svg {
    max-width: 26px;
    height: 1.15rem;
    width: auto;
}

.ux3 .footer__social-links a:hover svg path {
    fill: #353535;
}

</style></div>

            

        

    

        


    

            

            
                <div class="footer__bottom text-onDark">
            

            <div data-widget-id="06227714-c621-4651-9879-b0ce5176cccc" data-widget-name="ux3-layout-widget"><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="07e1d276-b4dd-4870-b858-ed74e70d64ef" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Life &amp; medical sciences journals</h2>
		<ul class="rlist clearfix sided">
		<li>
			<a alt="Cell" href="/cell/home">Cell</a>
		</li>
		<li>
			<a alt="Cancer Cell" href="/cancer-cell/home">Cancer Cell</a>
		</li>
		<li>
			<a alt="Cell Chemical Biology" href="/cell-chemical-biology">Cell Chemical Biology</a>
		</li>
		<li>
			<a alt="Cell Genomics" href="/cell-genomics/home">Cell Genomics</a>
		</li>
		<li>
			<a alt="Cell Host and Microbe" href="/cell-host-microbe/home">Cell Host &amp; Microbe</a>
		</li>
		<li>
			<a alt="Cell Metabolism" href="/cell-metabolism/home">Cell Metabolism</a>
		</li>
		<li>
			<a alt="Cell Reports" href="/cell-reports/home">Cell Reports</a>
		</li>
		<li>
			<a alt="Cell Reports Medicine" href="/cell-reports-medicine/home">Cell Reports Medicine</a>
		</li>
		<li>
			<a alt="Cell Stem Cell" href="/cell-stem-cell/home">Cell Stem Cell</a>
		</li>
		<li>
			<a alt="Cell Systems" href="/cell-systems/home">Cell Systems</a>
		</li>
		<li>
			<a alt="Current Biology" href="/current-biology/home">Current Biology</a>
		</li>
		<li>
			<a alt="Developmental Cell" href="/developmental-cell/home">Developmental Cell</a>
		</li>
		<li>
			<a alt="Immunity" href="/immunity/home">Immunity</a>
		</li>
		<li>
			<a alt="Med" href="/med/home">Med</a>
		</li>
		<li>
			<a alt="Molecular Cell" href="/molecular-cell/home">Molecular Cell</a>
		</li>
		<li>
			<a alt="Neuron" href="/neuron/home">Neuron</a>
		</li>
		<li>
			<a alt="Structure" href="/structure/home">Structure</a>
		</li>
		<li>
			<a alt="American Journal of Human Genetics" href="/ajhg/home">American Journal of Human Genetics (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Journal" href="/biophysj/home">Biophysical Journal (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Biophysical Reports" href="/biophysreports/home">Biophysical Reports (<i>partner</i>)</a>
		</li>	
		<li>
			<a alt="HGG Advances" href="/hgg-advances/home">Human Genetics and Genomics Advances (<i>partner</i>)</a></li>
		<li>
			<a alt="Molecular Plant" href="/molecular-plant/home">Molecular Plant (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy" href="/molecular-therapy-family/molecular-therapy/home">Molecular Therapy (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Methods and Clinical Development" href="/molecular-therapy-family/methods/home">Molecular Therapy Methods &amp; Clinical Development (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Nucleic Acids" href="/molecular-therapy-family/nucleic-acids/home">Molecular Therapy Nucleic Acids (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Molecular Therapy Oncology" href="/molecular-therapy-family/oncology/home">Molecular Therapy Oncology (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Plant Communications" href="/plant-communications/home">Plant Communications (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Stem Cell Reports" href="/stem-cell-reports/home">Stem Cell Reports (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biochemical Sciences" href="/trends/biochemical-sciences/home">Trends in Biochemical Sciences</a>
		</li>
		<li>
		</li><li>
			<a alt="Trends in Cancer" href="/trends/cancer/home">Trends in Cancer</a>
		</li>
		<li>
			<a alt="Trends in Cell Biology" href="/trends/cell-biology/home">Trends in Cell Biology</a>
		</li>
		<li>
			<a alt="Trends in Ecology and Evolution" href="/trends/ecology-evolution/home">Trends in Ecology &amp; Evolution</a>
		</li>
		<li>
			<a alt="Trends in Endocrinology and Metabolism" href="/trends/endocrinology-metabolism/home">Trends in Endocrinology &amp; Metabolism</a>
		</li>
		<li>
			<a alt="Trends in Genetics" href="/trends/genetics/home">Trends in Genetics</a>
		</li>
		<li>
			<a alt="Trends in Immunology" href="/trends/immunology/home">Trends in Immunology</a>
		</li>
		<li>
			<a alt="Trends in Microbiology" href="/trends/microbiology/home">Trends in Microbiology</a>
		</li>
		<li>
			<a alt="Trends in Molecular Medicine" href="/trends/molecular-medicine/home">Trends in Molecular Medicine</a>
		</li>
		<li>
			<a alt="Trends in Neurosciences" href="/trends/neurosciences/home">Trends in Neurosciences</a>
		</li>
		<li>
			<a alt="Trends in Parasitology" href="/trends/parasitology/home">Trends in Parasitology</a>
		</li>
		<li>
			<a alt="Trends in Pharmacological Sciences" href="/trends/pharmacological-sciences/home">Trends in Pharmacological Sciences</a>
		</li>
		<li>
			<a alt="Trends in Plant Science" href="/trends/plant-science/home">Trends in Plant Science</a>
		</li>
		</ul>
	</div>

	<div class="group">
		<h2 class="group__title">Physical sciences &amp; engineering journals</h2>
		<ul class="rlist clearfix sided">
		  <li><a alt="Cell Biomaterials" href="/cell-biomaterials/home">Cell Biomaterials</a></li>
		  <li>
			<a alt="Cell Reports Physical Science" href="/cell-reports-physical-science/home">Cell Reports Physical Science</a>
		</li>
		<li>
			<a alt="Chem" href="/chem/home">Chem</a>
		</li>
		<li>
			<a alt="Chem Catalysis" href="/chem-catalysis/home">Chem Catalysis</a>
		</li>
		<li>
			<a alt="Device" href="/device/home">Device</a>
		</li>
		<li>
			<a alt="Joule" href="/joule/home">Joule</a>
		</li>
		<li>
			<a alt="Matter" href="/matter/home">Matter</a>
		</li>	
		<li><a alt="Matter" href="/newton/home">Newton</a></li>
		<li>
			<a alt="Trends in Chemistry" href="/trends/chemistry/home">Trends in Chemistry</a>
		</li>			
		</ul>
	</div>

<div class="group">
	<h2 class="group__title">Multidisciplinary journals</h2>
	<ul class="rlist clearfix sided">
	  		<li>
			<a alt="Cell Reports Methods" href="/cell-reports-methods/home">Cell Reports Methods</a>
		</li>
		<li>
			<a alt="Cell Reports Sustainability" href="/cell-reports-sustainability/home">Cell Reports Sustainability</a>
		</li>
		<li>
			<a alt="Heliyon" href="/heliyon/home">Heliyon</a>
		</li>	
		<li>
			<a alt="iScience" href="/iscience/home">iScience</a>
		</li>
		<li>
			<a alt="One Earth" href="/one-earth/home">One Earth</a>
		</li>
		<li>
			<a alt="Patterns" href="/patterns/home">Patterns</a>
		</li>
		<li>
			<a alt="STAR Protocols" href="/star-protocols/home">STAR Protocols</a>
		</li>
		<li>
			<a alt="Nexus" href="/nexus/home">Nexus (<i>partner</i>)</a>
		</li>			
        <li>
			<a alt="The Innovation" href="/the-innovation/home">The Innovation (<i>partner</i>)</a>
		</li>
		<li>
			<a alt="Trends in Biotechnology" href="/trends/biotechnology/home">Trends in Biotechnology</a>
		</li>
		<li>
			<a alt="Trends in Cognitive Sciences" href="/trends/cognitive-sciences/home">Trends in Cognitive Sciences</a>
		</li>
	</ul>
</div>

</div>
</div>

            

        
</div><div class="col-sm-4 col-md-5 clearfix">
    

        


    

            

            

            <div data-widget-id="850476de-00cb-4c0d-943c-bf1881da24f9" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="clearfix">
		<div class="group sided">
			<h2 class="group__title">Authors</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Submit article" href="/submit-to-cell-press">Submit article</a>
				</li>
				<li>
					<a alt="Multi-Journal Submission" href="/multi-journal-submission">Multi-Journal Submission</a>
				</li>
				<li>
					<a alt="STAR Methods" href="/star-authors-guide">STAR Methods</a>
				</li>
				<li>
					<a alt="Sneak Peek – Preprints" href="/sneakpeek">Sneak Peek – Preprints</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Reviewers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Reviewers" href="/reviewers">Information for reviewers</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">News &amp; events</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Newsroom" href="/newsroom">Newsroom</a>
				</li>
				<li>
					<a alt="Events" href="/events">Events</a>
				</li>
				<li>
					<a alt="Cell Symposia" href="/symposia">Cell Symposia</a>
				</li>
				<li>
					<a alt="Consortia Hub" href="/consortia">Consortia Hub</a>
				</li>
				<li>
					<a alt="Webinars" href="/webinars">Webinars</a>
				</li>
								<li>
					&nbsp;
				</li>
			</ul>
		<h2 class="group__title">Multimedia</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Podcast" href="/podcast">Cell Press Podcast</a>
				</li>
				<li>
					<a alt="Cell Press Videos" href="/video">Cell Press Videos</a>
				</li>
				<li>
					<a alt="Color and Comics" href="/comics">Coloring and Comics</a>
				</li>
				<li>
					<a alt="Figure360" href="/video-guidelines">Figure360</a>
				</li>
				<li>
					<a alt="Cell Picture Show" href="/pictureshow">Cell Picture Show</a>
				</li>
				<li>
					<a alt="Research Arc" href="/research-arc">Research Arc</a>
				</li>
			</ul>
		</div>

		<div class="group sided">
			<h2 class="group__title">About</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="About Cell Press" href="/about">About Cell Press</a>
				</li>

				<li>
					<a alt="Open access" href="/open-access">Open access</a>
				</li>
				<li>
					<a alt="COVID Hub" href="/covid-19">COVID Hub</a>
				</li>
				<li>
					<a alt="Sustainability" href="/sustainability">Sustainability hub</a>
				</li>
				<li>
					<a alt="Inclusion and diversity" href="/diversity">Inclusion and diversity</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Contact</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Contact us" href="/contact">Contact us</a>
				</li>
				<li>
					<a alt="Help and Support" href="https://service.elsevier.com/app/overview/cell-press/" target="_blank">Help &amp; Support</a>
				</li>
				<li>
					&nbsp;
				</li>
			</ul>
			<h2 class="group__title">Careers</h2>
			<ul class="rlist clearfix">
				<li>
					<a alt="Cell Press Careers" href="https://relx.wd3.myworkdayjobs.com/en-US/ElsevierJobs" target="_blank">Cell Press Careers</a>
				</li>
			</ul>
		</div>
	</div>
</div></div>

            

        
</div><div class="col-sm-4 col-md-2 clearfix">
    

        


    

            

            

            <div data-widget-id="beafac41-1cfc-440b-9b55-74aba4a5612c" data-widget-name="UX3HTMLWidget"><div class="groups">
	<div class="group">
		<h2 class="group__title">Access</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Subscribe" href="/subscribe">Subscribe</a>
			</li>
			<li>
				<a alt="Claim" href="/activate">Claim</a>
			</li>
			<li>
				<a alt="Read It Now" href="/read-it-now">Read-It-Now</a>
			</li>
			<li>
				<a alt="Recommend to Librarian" href="http://info.cell.com/request-access-to-cell-press-content" target="_blank">Recommend to Librarian</a>
			</li>
			<li>
				<a alt="Publication Alerts" href="/action/showPreferences?menuTab=Alerts">Publication Alerts</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Collections</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="Best of Cell Press" href="/bestof">Best of Cell Press</a>
			</li>
			<li>
				<a alt="Cell Press Reviews" href="/reviews">Cell Press Reviews</a>
			</li>
			<li>
				<a alt="Cell Selections" href="/selections">Cell Press Selections</a>
			</li>
			<li>
				<a alt="Nucleus Collections" href="/nucleus">Nucleus Collections</a>
			</li>
			<li>
				<a alt="SnapShots" href="/snapshots">SnapShot Archive</a>
			</li>
		</ul>
	</div>
	<div class="group">
		<h2 class="group__title">Information</h2>
		<ul class="rlist clearfix">
			<li>
				<a alt="For Advertisters" href="/media-kit">For Advertisers</a>
			</li>
			<li>
				<a alt="For Recruiters" target="_blank" href="/pb-assets/Cell/common/CP-RecruitmentKit-US-2018-LowRes-1712070648.pdf">For Recruiters</a>
			</li>
			<li>
				<a alt="For Librarians" target="_blank" href="https://www.elsevier.com/products/sciencedirect/journals/direct-subscription-journals">For Librarians</a>
			</li>
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms and Conditions</a>
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>
			</li>
		</ul>
	</div>
</div></div>

            

        
</div></div>

            
                </div>
            

        

    

        


    

            

            

            <div data-widget-id="779ad11f-7bcd-4a2a-a2cd-8c67b649cbb2" data-widget-name="UX3HTMLWidget"><style>
 
button#cookieBtn {
   text-decoration: none;
   background-color: transparent;
   color: #005789;
   font-weight: bold;
   padding: 0;
   margin: 0;
   font-size: 14px;
}
 
button#cookieBtn:hover {
   text-decoration: underline;
}

</style>
<script>
// HIDE ONETRUST BACKGROUND AFTER USER PRESSES ESCAPE KEY
setTimeout(function(){ 
   $("#onetrust-pc-sdk").on("keydown", function(e) { 
      if (e.keyCode == 27) {  // ESC key
         setTimeout(function(){ 
            $(".onetrust-pc-dark-filter").css("display","none");
         }, 100); 
      }
   });
}, 5000);
</script>

<span class="align-center copy-right">
<hr>
<p align="center"><b>The content on this site is intended for healthcare professionals and researchers across all fields of science.</b></p>
<hr>
<div>
<p>We use cookies to help provide and enhance our service and tailor content. To update your cookie settings, please visit the 
      <b><button id="cookieBtn" class="ot-sdk-show-settings" type="button" aria-label="Manage cookie consent preferences">Cookie Settings</button></b> for this site.<br>

All content on this site: Copyright © 2025 Elsevier Inc., its licensors, and contributors. <br>All rights are reserved, including those for text and data mining, AI training, and similar technologies. <br>For all open access content, the relevant licensing terms apply.
</p>
<ul class="rlist--inline">
			<li>
				<a alt="Privacy Policy" target="_blank" href="https://www.elsevier.com/legal/privacy-policy">Privacy Policy</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Terms and Conditions" target="_blank" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions">Terms &amp; Conditions</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Accessibility" href="/accessibility">Accessibility</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Help and Support" target="_blank" href="https://service.elsevier.com/app/overview/cell-press/">Help &amp; Support</a>&nbsp;&nbsp;
			</li>
			<li>
				<a alt="Contact" href="/contact">Contact</a>
			</li>
			</ul>
<p>
<a href="https://www.relx.com"><img src="/pb/assets/raw/Health%20Advance/images/relx-1554393210983.png" alt="RELX" rel="nofollow" target="_new"></a></p>
</div>

</span></div>

            

        

    

        


    

            

            

            <div data-widget-id="c5e179c1-156e-41e3-af7f-5e307efa8acf" data-widget-name="UX3HTMLWidget"><style>#brandingRow{
    background-color:#fff;
    padding:7px 26px 7px 30px;
    color:#007DBC;
    font-size:14px;
    clear:both;
    width:100%
}
#brandingRow a{
    text-decoration:none;
    color:#007DBC
}
#brandingRow a:hover{
    text-decoration:underline
}
#brandingRowRight{
    display:inline;
    float:right;
    width:auto;
    padding:0;
    margin:0
}
svg#cp_logo {
  float: none;
  height: 36px;
  width: 36px;
  margin-left: 0px;
  margin-top: 0px;
}

footer hr {
  display: block;
}

@media screen and (max-width: 600px){
    #brandingRow{
        font-size:12px
    }
}
@media screen and (max-width: 750px){
    #brandingRow{
        padding:7px 18px 7px 16px
    }
    svg#cp_logo{
        margin-right:10px
    }
}
@media screen and (max-width: 800px) {
   a#cpIconLnk {
      display: none;
   }
}

.header__wrapper.clearfix .col-xs-6:first-of-type{
    width:55%
}
.header__wrapper.clearfix .col-xs-6:nth-of-type(2){
    width:45%
}
@media screen and (min-width: 800px) and (max-width: 1000px){
    .header__wrapper.clearfix .col-xs-6:first-of-type{
        width:75%
    }
    .header__wrapper.clearfix .col-xs-6:nth-of-type(2){
        width:25%
    }
}

</style>

<script>
var partnerx = false;
var overrideEndorsement = "";
var tmpj = $("input[name='journalCode']").attr("value");
	
if (window.location.pathname == "/molecular-therapy-family/home") {
	tmpj = "ymthe";
}
	
var ribbonx = "";
if (tmpj == "ajhg" || tmpj == "bpj" || tmpj == "molp" || tmpj == "xplc" || tmpj == "stemcr" || tmpj == "ymthe" || tmpj == "xhgg" || tmpj == "omtn" || tmpj == "omto" || tmpj == "omtm" || tmpj == "biophysreports" ||  tmpj == "bpr" || tmpj == "xinn" || tmpj == "omton") {
	partnerx = true;
	ribbonx = "<a href='https://www.cell.com' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press partner journal</a>";
} else if (tmpj != "" && tmpj != "undefined" && tmpj != undefined) {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press journal</a>";
} else {
	ribbonx = "<a href='/' data-aa-name='cell press endorsement' data-aa-region='cp global header' title='Visit Cell.com'>A Cell Press resource</a>";
}
	
if (window.location.pathname == "/") {
	ribbonx = "";
}
if (overrideEndorsement != "") {
	ribbonx = overrideEndorsement;
}

$(".header__wrapper.clearfix").prepend("<div id='brandingRow'><a href='/' data-aa-name='science that inspires' data-aa-region='cp global header' title='Visit Cell.com'>Science that inspires</a><p id='brandingRowRight'>" + ribbonx + "</p></div>");
	
// handle scrolling effect on article pages
if ($("article").length > 0) {
	var scrollx;
	$(window).scroll(function() {
		scrollx = $(window).scrollTop();
		if (scrollx > 0) { 
			$("#brandingRow").css("display","none"); 
		} else {
			$("#brandingRow").css("display","block");
		}
	});
}

if (window.location.pathname == "/" || partnerx == false) {
               $("<a href='/home' id='cpIconLnk' data-aa-name='cell press logo' data-aa-region='cp global header' title='Cell Press' target='_blank'><svg id='cp_logo' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='0 0 100 98.1' style='enable-background:new 0 0 100 98.1;' xml:space='preserve' height='100' width='98.1'><path style='fill:#FFFFFF;' d='M69.09,0.09c-4.01-0.6-7.83,1.94-8.78,5.89l-3.14,13.11c-0.76,3.19-3.44,5.56-6.7,5.93l-18.6,2.13          C11.83,29.31-2.53,47.66,0.37,67.78c2.3,15.96,15.18,27.77,30.54,30.08c4.01,0.6,7.83-1.94,8.78-5.89l3.14-13.11 c0.76-3.19,3.44-5.56,6.69-5.93l18.6-2.13c20.04-2.17,34.41-20.52,31.5-40.65C97.32,14.21,84.45,2.39,69.09,0.09z M65.06,47.2  l-11.75,1.4c-1.98,0.21-3.62,1.65-4.08,3.59l-3.43,14.3c-1.01,4.22-4.5,7.39-8.8,8c-6.66,0.72-12.63-4.2-13.15-10.92                c-0.51-6.5,4.6-12.11,11.08-12.81l11.75-1.39c1.98-0.22,3.62-1.65,4.08-3.59l3.43-14.3c1.01-4.22,4.5-7.39,8.8-8          c6.66-0.72,12.63,4.19,13.15,10.92C76.66,40.89,71.55,46.5,65.06,47.2z'/></svg></a>").insertBefore(".header__wrapper.clearfix .journal-logos:eq(0)");
}


</script></div>

            

        

    

        


    

            

            

            <div data-widget-id="1af759b7-9bb6-4413-ada9-51a8333e8fb8" data-widget-name="UX3HTMLWidget"><style>
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-444969 - c#4]*/
    [data-tab-id="changelogTab"] {
        display: none !important;
    }
    
    /*This was added temporarily until the PB changelog tab issue got fixed. [LIT-518839]*/
    @media (min-width: 769px) {
        .ux3 .quick-search .dropBlock [data-db-target-of],
        .quick-search .dropBlock [data-db-target-of] {
            width: 100vw !important;
        }
    }
</style></div>

            

        
</div></div>

            

        

        </div>
    

    

        


    

            

            

            <div data-widget-id="d7e0813b-a300-497a-ab37-9c3ddaf03e71" data-widget-name="UX3HTMLWidget"><!--
<div class="session-reminder hidden">
    <div class="session-reminder__overlay">
        <div class="session-reminder__dialog">
            <div class="session-reminder__dialog-inner">
                <div class="dialog-header" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-desc">
                    <h2 class="dialog-header-title" id="dialog-title">
                        <span>Session Timeout</span>
                        <div class="dialog-timer">(2:00)</div>
                    </h2>
                    <button class="dialog-close" type="button" aria-label="close dialog">
                        <i class="icon-tools_close" aria-hidden="true"></i>
                    </button>
                </div>
                <div class="dialog-body">
                    <p class="dialog-desc" id="dialog-desc">Your session will expire shortly. If you are still working, click the ‘Keep Me Logged In’ button below. If you do not respond within the next minute, you will be automatically logged out.</p>
                    <p class="dialog-expired sr-only" aria-live="polite"></p>
                </div>
                <div class="dialog-footer">
                    <button class="session-reset" type="button">Keep Me Logged In</button>
                </div>
            </div>
        </div>
    </div>
</div>
<a class="sr-only" href="javascript:void(0)" aria-hidden="true"></a>
--></div>

            

        
        </div>
    </div>


































    



















        
        
            
                <script src="/products/marlin/cell/releasedAssets/js/main.bundle-90475670808d5d3eee41.js"></script>
            
        
        
        

    




<script type="text/javascript">
    var pageData =  {"journal":{"specialty":"Cell Press","issn":"0092-8674","name":"Cell"},"page":{"loadTimestamp":"1749585511573","server":"cell","environment":"prod","businessUnit":"ELS:RAP:JBS","noTracking":"","name":"fulltext:html","section":"Cell","language":"en","type":"CP-CA","productName":"JB"},"visitor":{"departmentName":"|ae:ScienceDirect Guests","accessType":"ae:ANON_GUEST|ae:ANON_GUEST","accountId":"ae:838949|ae:228598","accountName":"ae:Cell Guest Account|ae:ScienceDirect Guests","departmentId":"ae:977212|ae:291352","ipAddress":"189.120.74.179","loginStatus":"anonymous","userId":"ae:246228266|ae:12975512"},"content":[{"accessType":"article:free:openarchive","volumeNumber":"180","issueNumber":"3","viewState":"fulltext","format":"MIME-HTML","id":"pii:S0092867419313984","coverDate":"202002","type":"XOCS-JOURNAL:SCOPE-FULL","title":"LargeScale Exome Sequencing Study Implicates Both Developmental and Functional Changes in the Neurobiology of Autism","status":"S300"}]};
</script>

<script type="text/javascript">
    if(typeof pageDataTracker !== "undefined") {
        pageDataTracker.trackPageLoad();
    }
</script>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'94db719b0b0552eb',t:'MTc0OTU4NTUxMC4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script><iframe height="1" width="1" style="position: absolute; top: 0px; left: 0px; border: none; visibility: hidden;"></iframe><script defer="" src="https://static.cloudflareinsights.com/beacon.min.js/vcd15cbe7772f49c399c6a5babf22c1241717689176015" integrity="sha512-ZpsOmlRQV6y907TI0dKBHq9Md29nnaEIPlkf84rnaERnq6zvWvPUqr2ft8M1aS28oN72PdrCzSjY4U6VaAw1EQ==" data-cf-beacon="{&quot;rayId&quot;:&quot;94db719b0b0552eb&quot;,&quot;serverTiming&quot;:{&quot;name&quot;:{&quot;cfExtPri&quot;:true,&quot;cfEdge&quot;:true,&quot;cfOrigin&quot;:true,&quot;cfL4&quot;:true,&quot;cfSpeedBrain&quot;:true,&quot;cfCacheStatus&quot;:true}},&quot;version&quot;:&quot;2025.5.0&quot;,&quot;token&quot;:&quot;57a02315956f4b2e82817b3dfa99b1a2&quot;}" crossorigin="anonymous"></script>


<div id="addtoany" style="position: static;"><div class="a2a_overlay" id="a2a_overlay"></div><div id="a2a_modal" class="a2a_modal a2a_hide" role="dialog" tabindex="-1" aria-label=""><div class="a2a_modal_body a2a_menu a2a_hide" id="a2a_copy_link"><label for="a2a_copy_link_text" id="a2a_copy_link_icon" class="a2a_svg a2a_s_link a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>Copy link</title><path fill="#FFF" d="M7.591 21.177c0-.36.126-.665.377-.917l2.804-2.804a1.235 1.235 0 0 1 .913-.378c.377 0 .7.144.97.43-.026.028-.11.11-.255.25-.144.14-.24.236-.29.29a2.82 2.82 0 0 0-.2.256 1.056 1.056 0 0 0-.177.344 1.43 1.43 0 0 0-.046.37c0 .36.126.666.377.918a1.25 1.25 0 0 0 .918.377c.126.001.251-.015.373-.047.125-.037.242-.096.345-.175.09-.06.176-.127.256-.2.1-.094.196-.19.29-.29.14-.142.223-.23.25-.254.297.28.445.607.445.984 0 .36-.126.664-.377.916l-2.778 2.79a1.242 1.242 0 0 1-.917.364c-.36 0-.665-.118-.917-.35l-1.982-1.97a1.223 1.223 0 0 1-.378-.9l-.001-.004Zm9.477-9.504c0-.36.126-.665.377-.917l2.777-2.79a1.235 1.235 0 0 1 .913-.378c.35 0 .656.12.917.364l1.984 1.968c.254.252.38.553.38.903 0 .36-.126.665-.38.917l-2.802 2.804a1.238 1.238 0 0 1-.916.364c-.377 0-.7-.14-.97-.418.026-.027.11-.11.255-.25a7.5 7.5 0 0 0 .29-.29c.072-.08.139-.166.2-.255.08-.103.14-.22.176-.344.032-.12.048-.245.047-.37 0-.36-.126-.662-.377-.914a1.247 1.247 0 0 0-.917-.377c-.136 0-.26.015-.37.046-.114.03-.23.09-.346.175a3.868 3.868 0 0 0-.256.2c-.054.05-.15.148-.29.29-.14.146-.222.23-.25.258-.294-.278-.442-.606-.442-.983v-.003ZM5.003 21.177c0 1.078.382 1.99 1.146 2.736l1.982 1.968c.745.75 1.658 1.12 2.736 1.12 1.087 0 2.004-.38 2.75-1.143l2.777-2.79c.75-.747 1.12-1.66 1.12-2.737 0-1.106-.392-2.046-1.183-2.818l1.186-1.185c.774.79 1.708 1.186 2.805 1.186 1.078 0 1.995-.376 2.75-1.13l2.803-2.81c.751-.754 1.128-1.671 1.128-2.748 0-1.08-.382-1.993-1.146-2.738L23.875 6.12C23.13 5.372 22.218 5 21.139 5c-1.087 0-2.004.382-2.75 1.146l-2.777 2.79c-.75.747-1.12 1.66-1.12 2.737 0 1.105.392 2.045 1.183 2.817l-1.186 1.186c-.774-.79-1.708-1.186-2.805-1.186-1.078 0-1.995.377-2.75 1.132L6.13 18.426c-.754.755-1.13 1.672-1.13 2.75l.003.001Z"></path></svg></label><input id="a2a_copy_link_text" type="text" title="Copy link" readonly=""><div id="a2a_copy_link_copied">✓</div></div><div class="a2a_modal_body a2a_menu a2a_thanks a2a_hide" id="a2a_thanks"><div class="a2a_localize" data-a2a-localize="inner,ThanksForSharing">Thanks for sharing!</div></div></div><div class="a2a_menu a2a_full a2a_localize" id="a2apage_full" role="dialog" tabindex="-1" aria-label="Share" data-a2a-localize="title,Share"><div class="a2a_full_header"><div id="a2apage_find_container" class="a2a_menu_find_container"><label for="a2apage_find" id="a2apage_find_icon" class="a2a_svg a2a_s_find"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true"><title>Find any service</title><path fill="#CCC" d="M19.7 18.2l-4.5-4.5c.7-1.1 1.2-2.3 1.2-3.6 0-3.5-2.8-6.3-6.3-6.3s-6.3 2.8-6.3 6.3 2.8 6.3 6.3 6.3c1.4 0 2.6-.4 3.6-1.2l4.5 4.5c.6.6 1.3.7 1.7.2.5-.4.4-1.1-.2-1.7zm-9.6-3.6c-2.5 0-4.5-2.1-4.5-4.5 0-2.5 2.1-4.5 4.5-4.5 2.5 0 4.5 2.1 4.5 4.5s-2 4.5-4.5 4.5z"></path></svg></label><input id="a2apage_find" class="a2a_menu_find a2a_localize" type="text" autocomplete="off" title="Find any service" data-a2a-localize="title,FindAnyServiceToAddTo"></div></div><div class="a2a_full_services" id="a2apage_full_services" role="presentation"></div><div class="a2a_full_footer"><a href="https://www.addtoany.com" title="Share Buttons" rel="noopener" target="_blank"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span>AddToAny</a></div></div><div id="a2apage_dropdown" class="a2a_menu a2a_mini a2a_localize a2a_hide" tabindex="-1" aria-label="Share" data-a2a-localize="label,Share"><div class="a2a_mini_services" id="a2apage_mini_services"></div><div id="a2apage_cols_container" class="a2a_cols_container"><div class="a2a_col1" id="a2apage_col1"></div><div id="a2apage_2_col1" class="a2a_hide"></div><div class="a2a_clear"></div></div><div class="a2apage_wide a2a_wide"><a href="#addtoany" id="a2apage_show_more_less" class="a2a_more a2a_localize" title="Show all" data-a2a-localize="title,ShowAll"><span class="a2a_svg a2a_s__default a2a_s_a2a a2a_logo_color"><svg focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><g fill="#FFF"><path d="M14 7h4v18h-4z"></path><path d="M7 14h18v4H7z"></path></g></svg></span><span class="a2a_localize" data-a2a-localize="inner,More">More…</span></a></div></div><div style="height: 1px; width: 1px; position: absolute; z-index: 100000; top: 0px; visibility: hidden;"><iframe id="a2a_sm_ifr" title="AddToAny Utility Frame" aria-hidden="true" src="https://static.addtoany.com/menu/sm.25.html#type=core&amp;event=load" style="height: 1px; width: 1px; border: 0px; left: 0px; top: 0px; position: absolute; z-index: 100000; display: none;"></iframe></div></div><div role="status" aria-live="assertive" aria-relevant="additions" class="ui-helper-hidden-accessible"></div><div id="onetrust-consent-sdk" data-nosnippet="true"><div class="onetrust-pc-dark-filter ot-fade-in" style="z-index:2147483645;"></div><div id="onetrust-banner-sdk" class="otFlat bottom ot-wo-title ot-buttons-fw" tabindex="0" role="region" aria-label="Cookie banner"><div role="dialog" aria-modal="true" aria-label="Privacy"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><div id="onetrust-policy-text">We use cookies that are necessary to make our site work. We may also use additional cookies to analyze, improve, and personalize our content and your digital experience. For more information, see our<a class="ot-cookie-policy-link" href="https://www.elsevier.com/legal/cookienotice" target="_blank" aria-label=", opens in a new tab" rel="noopener">Cookie Policy</a></div></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" class="cookie-setting-link">Cookie Settings</button>  <button id="onetrust-accept-btn-handler">Accept all cookies</button></div></div></div></div><!-- Close Button --><div id="onetrust-close-btn-container"></div><!-- Close Button END--></div></div><div id="onetrust-pc-sdk" class="otPcCenter ot-hide ot-fade-in" lang="en" aria-label="Preference center" role="region"><div role="dialog" aria-modal="true" style="height: 100%;" aria-label="Cookie Preference Center"><!-- Close Button --><div class="ot-pc-header"><!-- Logo Tag --><div class="ot-pc-logo" role="img" aria-label="Company Logo"><img alt="Company Logo" src="https://cdn.cookielaw.org/logos/static/ot_company_logo.png"></div></div><!-- Close Button --><div id="ot-pc-content" class="ot-pc-scrollbar"><div class="ot-optout-signal ot-hide"><div class="ot-optout-icon"><svg xmlns="http://www.w3.org/2000/svg"><path class="ot-floating-button__svg-fill" d="M14.588 0l.445.328c1.807 1.303 3.961 2.533 6.461 3.688 2.015.93 4.576 1.746 7.682 2.446 0 14.178-4.73 24.133-14.19 29.864l-.398.236C4.863 30.87 0 20.837 0 6.462c3.107-.7 5.668-1.516 7.682-2.446 2.709-1.251 5.01-2.59 6.906-4.016zm5.87 13.88a.75.75 0 00-.974.159l-5.475 6.625-3.005-2.997-.077-.067a.75.75 0 00-.983 1.13l4.172 4.16 6.525-7.895.06-.083a.75.75 0 00-.16-.973z" fill="#FFF" fill-rule="evenodd"></path></svg></div><span></span></div><h2 id="ot-pc-title">Cookie Preference Center</h2><div id="ot-pc-desc">We use cookies which are necessary to make our site work. We may also use additional cookies to analyse, improve and personalise our content and your digital experience. For more information, see our <a href="https://www.elsevier.com/legal/cookienotice/_nocache" target="_blank">Cookie Policy</a> and the list of <a href="https://support.google.com/admanager/answer/9012903" target="_blank">Google Ad-Tech Vendors</a>.
<br>
<br>
You may choose not to allow some types of cookies. However, blocking some types may impact your experience of our site and the services we are able to offer. See the different category headings below to find out more or change your settings.
<br>
</div><button id="accept-recommended-btn-handler">Allow all</button><section class="ot-sdk-row ot-cat-grp"><h3 id="ot-category-title"> Manage Consent Preferences</h3><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="1"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-1" aria-labelledby="ot-header-id-1 ot-status-id-1"></button><!-- Accordion header --><div class="ot-acc-hdr ot-always-active-group"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-1">Strictly Necessary Cookies</h4><div id="ot-status-id-1" class="ot-always-active">Always active</div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-1">These cookies are necessary for the website to function and cannot be switched off in our systems. They are usually only set in response to actions made by you which amount to a request for services, such as setting your privacy preferences, logging in or filling in forms. You can set your browser to block or alert you about these cookies, but some parts of the site will not then work. These cookies do not store any personally identifiable information.
<br><br></p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="1">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="3"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-3" aria-labelledby="ot-header-id-3"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-3">Functional Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-3" id="ot-group-id-3" role="switch" class="category-switch-handler" data-optanongroupid="3" aria-labelledby="ot-header-id-3"> <label class="ot-switch" for="ot-group-id-3"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Functional Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-3">These cookies enable the website to provide enhanced functionality and personalisation. They may be set by us or by third party providers whose services we have added to our pages. If you do not allow these cookies then some or all of these services may not function properly.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="3">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="2"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-2" aria-labelledby="ot-header-id-2"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-2">Performance Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-2" id="ot-group-id-2" role="switch" class="category-switch-handler" data-optanongroupid="2" aria-labelledby="ot-header-id-2"> <label class="ot-switch" for="ot-group-id-2"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Performance Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-2">These cookies allow us to count visits and traffic sources so we can measure and improve the performance of our site. They help us to know which pages are the most and least popular and see how visitors move around the site.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="2">Cookie Details List‎</button></div></div></div><div class="ot-accordion-layout ot-cat-item ot-vs-config" data-optanongroupid="4"><button aria-expanded="false" ot-accordion="true" aria-controls="ot-desc-id-4" aria-labelledby="ot-header-id-4"></button><!-- Accordion header --><div class="ot-acc-hdr"><div class="ot-plus-minus"><span></span><span></span></div><h4 class="ot-cat-header" id="ot-header-id-4">Targeting Cookies</h4><div class="ot-tgl"><input type="checkbox" name="ot-group-id-4" id="ot-group-id-4" role="switch" class="category-switch-handler" data-optanongroupid="4" aria-labelledby="ot-header-id-4"> <label class="ot-switch" for="ot-group-id-4"><span class="ot-switch-nob"></span> <span class="ot-label-txt">Targeting Cookies</span></label> </div></div><!-- accordion detail --><div class="ot-acc-grpcntr ot-acc-txt"><p class="ot-acc-grpdesc ot-category-desc" id="ot-desc-id-4">These cookies may be set through our site by our advertising partners. They may be used by those companies to build a profile of your interests and show you relevant adverts on other sites. If you do not allow these cookies, you will experience less targeted advertising.</p><div class="ot-hlst-cntr"><button class="ot-link-btn category-host-list-handler" aria-label="Cookie Details List" data-parent-id="4">Cookie Details List‎</button></div></div></div><!-- Groups sections starts --><!-- Group section ends --><!-- Accordion Group section starts --><!-- Accordion Group section ends --></section></div><section id="ot-pc-lst" class="ot-hide ot-hosts-ui ot-pc-scrollbar"><div id="ot-pc-hdr"><div id="ot-lst-title"><button class="ot-link-btn back-btn-handler" aria-label="Back"><svg id="ot-back-arw" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#656565" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></button><h3>Cookie List</h3></div><div class="ot-lst-subhdr"><div class="ot-search-cntr"><p role="status" class="ot-scrn-rdr"></p><input id="vendor-search-handler" type="text" name="vendor-search-handler" placeholder="Search…" aria-label="Cookie list search"> <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110" aria-hidden="true"><title>Search Icon</title><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
            s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
            c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
            s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div class="ot-fltr-cntr"><button id="filter-btn-handler" aria-label="Filter" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 402.577 402.577" xml:space="preserve"><title>Filter Icon</title><g><path fill="#fff" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
      c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
      c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
      C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></button></div><div id="ot-anchor"></div><section id="ot-fltr-modal"><div id="ot-fltr-cnt"><button id="clear-filters-handler">Clear</button><div class="ot-fltr-scrlcnt ot-pc-scrollbar"><div class="ot-fltr-opts" role="list"><div class="ot-fltr-opt" role="listitem"><div class="ot-chkbox"><input id="chkbox-id" type="checkbox" class="category-filter-handler"> <label for="chkbox-id"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div><div class="ot-fltr-btns"><button id="filter-apply-handler">Apply</button> <button id="filter-cancel-handler">Cancel</button></div></div></div></section></div></div><section id="ot-lst-cnt" class="ot-host-cnt ot-pc-scrollbar"><div id="ot-sel-blk"><div class="ot-sel-all"><div class="ot-sel-all-hdr"><span class="ot-consent-hdr">Consent</span> <span class="ot-li-hdr">Leg.Interest</span></div><div class="ot-sel-all-chkbox"><div class="ot-chkbox" id="ot-selall-hostcntr"><input id="select-all-hosts-groups-handler" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-vencntr"><input id="select-all-vendor-groups-handler" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div><div class="ot-chkbox" id="ot-selall-licntr"><input id="select-all-vendor-leg-handler" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="ot-label-txt">checkbox label</span></label> <span class="ot-label-status">label</span></div></div></div></div><div class="ot-sdk-row"><div class="ot-sdk-column"><ul id="ot-host-lst"></ul></div></div></section></section><div class="ot-pc-footer ot-pc-scrollbar"><div class="ot-btn-container"> <button class="save-preference-btn-handler onetrust-close-btn-handler">Confirm my choices</button></div><!-- Footer logo --><div class="ot-pc-footer-logo"><a href="https://www.onetrust.com/products/cookie-consent/" target="_blank" rel="noopener noreferrer" aria-label="Powered by OneTrust Opens in a new Tab"><img alt="Powered by Onetrust" src="https://cdn.cookielaw.org/logos/static/powered_by_logo.svg" title="Powered by OneTrust Opens in a new Tab"></a></div></div><!-- Cookie subgroup container --><!-- Vendor list link --><!-- Cookie lost link --><!-- Toggle HTML element --><!-- Checkbox HTML --><!-- plus minus--><!-- Arrow SVG element --><!-- Accordion basic element --><span class="ot-scrn-rdr" aria-atomic="true" aria-live="polite"></span><!-- Vendor Service container and item template --></div><iframe class="ot-text-resize" sandbox="allow-same-origin" title="onetrust-text-resize" style="position: absolute; top: -50000px; width: 100em;" aria-hidden="true"></iframe></div></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">__("articleCrossmark.closePopup")</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content" title="Crossmark service - Intentionally blank" aria-hidden="false" tabindex="0" src="https://crossmark.crossref.org/dialog?doi=10.1016%2Fj.cell.2019.12.036&amp;domain=www.cell.com&amp;uri_scheme=https%3A&amp;cm_version=v2.0"></iframe></div></div></div></div></div></body></html>